
<html lang="en"     class="pb-page"  data-request-id="c360346b-b595-4858-887d-8a2e50cc515d"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2014.57.issue-1;wgroup:string:ACHS website Group;article:article:10.1021/jm4012802;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Pan-Histone Demethylase Inhibitors Simultaneously Targeting Jumonji C and Lysine-Specific Demethylases Display High Anticancer Activities" /></meta><meta name="dc.Creator" content="Dante  Rotili" /></meta><meta name="dc.Creator" content="Stefano  Tomassi" /></meta><meta name="dc.Creator" content="Mariarosaria  Conte" /></meta><meta name="dc.Creator" content="Rosaria  Benedetti" /></meta><meta name="dc.Creator" content="Marcello  Tortorici" /></meta><meta name="dc.Creator" content="Giuseppe  Ciossani" /></meta><meta name="dc.Creator" content="Sergio  Valente" /></meta><meta name="dc.Creator" content="Biagina  Marrocco" /></meta><meta name="dc.Creator" content="Donatella  Labella" /></meta><meta name="dc.Creator" content="Ettore  Novellino" /></meta><meta name="dc.Creator" content="Andrea  Mattevi" /></meta><meta name="dc.Creator" content="Lucia  Altucci" /></meta><meta name="dc.Creator" content="Anthony  Tumber" /></meta><meta name="dc.Creator" content="Clarence  Yapp" /></meta><meta name="dc.Creator" content="Oliver N. F.  King" /></meta><meta name="dc.Creator" content="Richard J.  Hopkinson" /></meta><meta name="dc.Creator" content="Akane  Kawamura" /></meta><meta name="dc.Creator" content="Christopher J.  Schofield" /></meta><meta name="dc.Creator" content="Antonello  Mai" /></meta><meta name="dc.Description" content="In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid..." /></meta><meta name="Description" content="In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 19, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm4012802" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm4012802" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm4012802" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm4012802" /></link>
        
    
    

<title>Pan-Histone Demethylase Inhibitors Simultaneously Targeting Jumonji C and Lysine-Specific Demethylases Display High Anticancer Activities | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm4012802" /></meta><meta property="og:title" content="Pan-Histone Demethylase Inhibitors Simultaneously Targeting Jumonji C and Lysine-Specific Demethylases Display High Anticancer Activities" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0011.jpeg" /></meta><meta property="og:description" content="In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid LSD1/JmjC or “pan-KDM” inhibitors 1–6 by coupling the skeleton of tranylcypromine 7, a known LSD1 inhibitor, with 4-carboxy-4′-carbomethoxy-2,2′-bipyridine 8 or 5-carboxy-8-hydroxyquinoline 9, two 2-oxoglutarate competitive templates developed for JmjC inhibition. Hybrid compounds 1–6 are able to simultaneously target both KDM families and have been validated as potential antitumor agents in cells. Among them, 2 and 3 increase H3K4 and H3K9 methylation levels in cells and cause growth arrest and substantial apoptosis in LNCaP prostate and HCT116 colon cancer cells. When tested in noncancer mesenchymal progenitor (MePR) cells, 2 and 3 induced little and no apoptosis, respectively, thus showing cancer-selective inhibiting action." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm4012802"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm4012802">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm4012802&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm4012802&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm4012802&amp;href=/doi/10.1021/jm4012802" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2014</span><span class="cit-fg-volume">, 57</span><span class="cit-fg-issue">, 1</span><span class="cit-fg-pageRange">, 42-55</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/57/1" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm400633d" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm401555v" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Pan-Histone Demethylase Inhibitors Simultaneously Targeting Jumonji C and Lysine-Specific Demethylases Display High Anticancer Activities</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dante++Rotili">Dante Rotili</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Stefano++Tomassi">Stefano Tomassi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mariarosaria++Conte">Mariarosaria Conte</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rosaria++Benedetti">Rosaria Benedetti</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marcello++Tortorici">Marcello Tortorici</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Giuseppe++Ciossani">Giuseppe Ciossani</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sergio++Valente">Sergio Valente</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Biagina++Marrocco">Biagina Marrocco</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Donatella++Labella">Donatella Labella</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ettore++Novellino">Ettore Novellino</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andrea++Mattevi">Andrea Mattevi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lucia++Altucci">Lucia Altucci</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Anthony++Tumber">Anthony Tumber</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▽</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Clarence++Yapp">Clarence Yapp</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▽</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Oliver+N.+F.++King">Oliver N. F. King</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▲</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Richard+J.++Hopkinson">Richard J. Hopkinson</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▲</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Akane++Kawamura">Akane Kawamura</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▲</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+J.++Schofield">Christopher J. Schofield</a></span><span class="author-xref-symbol author-aff-symbol"><sup>▲</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Antonello++Mai">Antonello Mai</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Drug Chemistry and Technologies, Sapienza University of Rome, P. le A. Moro 5, 00185 Rome, Italy</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Department of Pharmacy, University of Naples “Federico II”, Via D. Montesano 49, 80131 Naples, Italy</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Biochemistry, Biophysics and General Pathology, Second University of Naples, Vico L. De Crecchio 7, 80138 Naples, Italy</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Biology and Biotechnology, University of Pavia, Via Ferrata 9, 27100 Pavia, Italy</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Institute of Genetics and Biophysics, Consiglio Nazionale delle Ricerche, Via P. Castellino, 80100 Naples, Italy</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">▽</span> <span class="aff-text">Structural Genomics Consortium, University of Oxford, Old Road Campus, Roosvelt Drive, Headington OX3 7DU, United Kingdom</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">▲</span> <span class="aff-text">Chemistry Research Laboratory University of Oxford, 12 Mansfield Road, Oxford OX1 3TA, United Kingdom</span></div><div class="corresp-info"><strong>*</strong>Telephone +39 06 49913392; fax +39 06 491491; e-mail <a href="/cdn-cgi/l/email-protection#8aebe4fee5e4efe6e6e5a4e7ebe3caffe4e3f8e5e7ebbba4e3fe"><span class="__cf_email__" data-cfemail="f899968c97969d949497d6959991b88d96918a979599c9d6918c">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm4012802&amp;href=/doi/10.1021%2Fjm4012802" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2014</span></span><span class="cit-volume">, 57</span><span class="cit-issue">, 1</span><span class="cit-pageRange">, 42–55</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 10, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 August 2013</li><li><span class="item_label"><b>Published</b> online</span>19 December 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 9 January 2014</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm4012802" title="DOI URL">https://doi.org/10.1021/jm4012802</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D42%26pageCount%3D14%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDante%2BRotili%252C%2BStefano%2BTomassi%252C%2BMariarosaria%2BConte%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D57%26issueNum%3D1%26contentID%3Djm4012802%26title%3DPan-Histone%2BDemethylase%2BInhibitors%2BSimultaneously%2BTargeting%2BJumonji%2BC%2Band%2BLysine-Specific%2BDemethylases%2BDisplay%2BHigh%2BAnticancer%2BActivities%26numPages%3D14%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D55%26publicationDate%3DJanuary%2B2014">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm4012802"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4769</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">84</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm4012802" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Pan-Histone Demethylase Inhibitors Simultaneously Targeting Jumonji C and Lysine-Specific Demethylases Display High Anticancer Activities&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dante&quot;,&quot;last_name&quot;:&quot;Rotili&quot;},{&quot;first_name&quot;:&quot;Stefano&quot;,&quot;last_name&quot;:&quot;Tomassi&quot;},{&quot;first_name&quot;:&quot;Mariarosaria&quot;,&quot;last_name&quot;:&quot;Conte&quot;},{&quot;first_name&quot;:&quot;Rosaria&quot;,&quot;last_name&quot;:&quot;Benedetti&quot;},{&quot;first_name&quot;:&quot;Marcello&quot;,&quot;last_name&quot;:&quot;Tortorici&quot;},{&quot;first_name&quot;:&quot;Giuseppe&quot;,&quot;last_name&quot;:&quot;Ciossani&quot;},{&quot;first_name&quot;:&quot;Sergio&quot;,&quot;last_name&quot;:&quot;Valente&quot;},{&quot;first_name&quot;:&quot;Biagina&quot;,&quot;last_name&quot;:&quot;Marrocco&quot;},{&quot;first_name&quot;:&quot;Donatella&quot;,&quot;last_name&quot;:&quot;Labella&quot;},{&quot;first_name&quot;:&quot;Ettore&quot;,&quot;last_name&quot;:&quot;Novellino&quot;},{&quot;first_name&quot;:&quot;Andrea&quot;,&quot;last_name&quot;:&quot;Mattevi&quot;},{&quot;first_name&quot;:&quot;Lucia&quot;,&quot;last_name&quot;:&quot;Altucci&quot;},{&quot;first_name&quot;:&quot;Anthony&quot;,&quot;last_name&quot;:&quot;Tumber&quot;},{&quot;first_name&quot;:&quot;Clarence&quot;,&quot;last_name&quot;:&quot;Yapp&quot;},{&quot;first_name&quot;:&quot;Oliver&quot;,&quot;last_name&quot;:&quot;N. F. King&quot;},{&quot;first_name&quot;:&quot;Richard&quot;,&quot;last_name&quot;:&quot;J. Hopkinson&quot;},{&quot;first_name&quot;:&quot;Akane&quot;,&quot;last_name&quot;:&quot;Kawamura&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;J. Schofield&quot;},{&quot;first_name&quot;:&quot;Antonello&quot;,&quot;last_name&quot;:&quot;Mai&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;19&quot;,&quot;issue&quot;:&quot;1&quot;,&quot;volume&quot;:&quot;57&quot;,&quot;pages&quot;:&quot;42-55&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm4012802&quot;},&quot;abstract&quot;:&quot;In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid LSD1/JmjC or “pan-KDM” inhibitors 1–6 by coupling the skeleton of tranylcypromine 7, a known LSD1 inhibitor, with 4-carboxy-4′-carbomethoxy-2,2′-bipyridine 8 or 5-carboxy-8-hydroxyquinoline 9, two 2-oxoglutarate competitive templates developed for JmjC inhibition. Hybrid compounds 1–6 are able to simultaneously target both KDM families and have been validated as potential antitumor agents in cells. Among them, 2 and 3 increase H3K4 and H3K9 methylation levels in cells and cause growth arrest and substantial apoptosis in LNCaP prostate and HCT116 colon cancer cells. When tested in noncancer mesenchymal progenitor (MePR) cells, 2 and 3 induced little and no apoptosis, respectively, thus showing cancer-selective inhibiting action.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4012802&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4012802" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4012802&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4012802" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm4012802&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4012802" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm4012802&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm4012802&amp;href=/doi/10.1021/jm4012802" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm4012802" /></input><a href="/doi/pdf/10.1021/jm4012802" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm4012802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm4012802%26sid%3Dliteratum%253Aachs%26pmid%3D24325601%26genre%3Darticle%26aulast%3DRotili%26date%3D2014%26atitle%3DPan-Histone%2BDemethylase%2BInhibitors%2BSimultaneously%2BTargeting%2BJumonji%2BC%2Band%2BLysine-Specific%2BDemethylases%2BDisplay%2BHigh%2BAnticancer%2BActivities%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D57%26issue%3D1%26spage%3D42%26epage%3D55%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291304" title="Organic reactions">Organic reactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/57/1" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jmcmar.2014.57.issue-1/production/jmcmar.2014.57.issue-1.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">In prostate cancer, two different types of histone lysine demethylases (KDM), LSD1/KDM1 and JMJD2/KDM4, are coexpressed and colocalize with the androgen receptor. We designed and synthesized hybrid LSD1/JmjC or “pan-KDM” inhibitors <b>1</b>–<b>6</b> by coupling the skeleton of tranylcypromine <b>7</b>, a known LSD1 inhibitor, with 4-carboxy-4′-carbomethoxy-2,2′-bipyridine <b>8</b> or 5-carboxy-8-hydroxyquinoline <b>9</b>, two 2-oxoglutarate competitive templates developed for JmjC inhibition. Hybrid compounds <b>1</b>–<b>6</b> are able to simultaneously target both KDM families and have been validated as potential antitumor agents in cells. Among them, <b>2</b> and <b>3</b> increase H3K4 and H3K9 methylation levels in cells and cause growth arrest and substantial apoptosis in LNCaP prostate and HCT116 colon cancer cells. When tested in noncancer mesenchymal progenitor (MePR) cells, <b>2</b> and <b>3</b> induced little and no apoptosis, respectively, thus showing cancer-selective inhibiting action.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80852" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80852" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The covalent modifications of histones, including acetylation, methylation, and phosphorylation, play central roles in DNA replication/repair and in the regulation of transcription.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> In contrast to lysyl-acetylation, which normally correlates with transcriptional activation, lysyl-methylation correlates with both activation and repression depending on the context, including lysine residue location and the extent (me1, me2, or me3) of methylation. In general, methylation of histone H3 at Lys residues 9 or 27 (H3K9 or H3K27) or histone H4 at Lys 20 (H4K20) is associated with repression, whereas H3K4, H3K36, and H3K79 methylation correlates with enhanced transcription.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Lysine methylation was once thought to be irreversible; however, the identification first of the lysine-specific demethylases (LSDs) followed by the Jumonji C (JmjC) demethylases, has led to the proposal that it is dynamic.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a></div><div class="NLM_p">The human LSDs (LSD1 and LSD2, KDM1) demethylate histone substrates via a flavin adenine dinucleotide (FAD)-dependent amine oxidase type mechanism. They catalyze the overall transfer of hydride from the methyl amino group to FAD, thus generating a methyliminium ion, which is in turn hydrolyzed to yield a carbinolamine that spontaneously degrades to give formaldehyde and the demethylated amine (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>A).<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4, 5)</a> For mechanistic reasons, the LSDs demethylate only mono- and dimethylated lysines. LSD1 is specific for H3K4me1/me2 as a component of the CoREST corepressor complex,<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> but in the presence of the androgen receptor (AR), its specificity changes from H3K4 to H3K9.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Thus, LSD1 can act either as a transcriptional corepressor or coactivator depending on its binding partners and substrates. Pathologically, LSD1 was found to be oncogenic and/or overexpressed in several cancers ranging from leukemias to solid tumors.<a onclick="showRef(event, 'ref5 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref5 ref7 ref8 ref9">(5, 7-9)</a> In addition, LSD1 also regulates gene transcription of viruses such as hepatitis B virus (HBV),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> herpes simplex virus (HSV), and varicella zoster virus (VZV).<a onclick="showRef(event, 'ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref11 ref12">(11, 12)</a></div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reactions Catalyzed by (A) Lysine-Specific Demethylases and (B) Jumonji C Demethylases</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The JmjC enzymes are Fe(II) and 2-oxoglutarate (2OG) -dependent oxygenases and represent the largest lysyl-demethylase (KDM) superfamily, comprising ∼20 human enzymes grouped into five subfamilies (KDM2/7, KDM3, KDM4, KDM5, and KDM6). Mechanistically, the reaction between Fe(II)/2OG complex and molecular oxygen converts 2OG into carbon dioxide, succinate, and an enzyme-bound Fe(IV)═O species. The Fe(IV)═O species reacts with the methyl group of the methylated histone lysine residue to produce an hemiaminal intermediate that spontaneously generates formaldehyde and demethylated lysine while regenerating the active Fe(II) center (Scheme <a class="ref internalNav" href="#sch1" aria-label="1">1</a>B). In contrast to the LSDs, the JmjC KDMs can demethylate all three lysine-methylation states.<a onclick="showRef(event, 'ref5 ref13'); return false;" href="javascript:void(0);" class="ref ref5 ref13">(5, 13)</a> JmjC demethylases are overexpressed in multiple types of cancer cells,<a onclick="showRef(event, 'ref2 ref3 ref5 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref5 ref13 ref14">(2, 3, 5, 13, 14)</a> and some JmjC demethylases have been involved in neural development and/or function and are associated with noncancer diseases including X-linked mental retardation, autism, and midline defects.<a onclick="showRef(event, 'ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref15 ref16 ref17">(15-17)</a></div><div class="NLM_p">In addition to histones, LSD1 and some JmjCs interact with nonhistone proteins that also contribute to transcriptional and/or nontranscriptional related processes. Specifically, LSD1 demethylates the tumor suppressor p53 at its K370me2 residue, repressing p53-mediated transcriptional activation and p53-induced apoptosis,<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> and negatively regulates the signal transducer and activator of transcription 3 (STAT3) by demethylation of its K140me2 residue.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Demethylation of the transcription factor E2F1 in K185 by LSD1 regulates E2F1-induced cell death, but in this case opposing effects are reported<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21, 22)</a> and further studies are required. LSD1 is also able to demethylate myosin phosphatase MYPT1,<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> enhancing its ubiquitin-mediated degradation and increasing the amount of phosphorylated retinoblastoma protein RB1 with subsequent effects on the cell cycle. Moreover, K1096 in the DNA methyltransferase DNMT1 seems to be a substrate for LSD1,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> thus establishing cross-talk between DNA and histone methylation. With respect to nonhistone JmjC substrates, mouse KDM6B interacts with p53 during neural stem cell differentiation and stabilizes p53 by demethylation, allowing its accumulation in the nucleus.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> The K218 and K221 residues of the p65 subunit of the nuclear factor κB (NF-κB) may also be targets of KDM2A, that in this way negatively regulates NF-κB activity and transcription.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></div><div class="NLM_p">In human prostate cancers, LSD1 and JmjC KDM4A/C are coexpressed and colocalize with androgen receptor (AR).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Recently, KDM4A/C inhibitors were found not to inhibit prostate (LNCaP and PC3) and colon (HCT116) cancer cell growth in isolation, but they displayed antiproliferative effects in combination with NCL-2, an LSD1 inhibitor,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> suggesting a potential for synergy in LSD and JmjC KDM inhibition.</div><div class="NLM_p">It is now clear that in multifactorial diseases such as cancer, central nervous system disorders, diabetes, or immunoinflammatory diseases, which involve multiple altered cellular pathways and signals, the use of single-target drugs (the “one drug, one target” concept) can result in unsatisfactory treatment outcomes.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a> Combination therapies employing two or more drugs can be more efficient in controlling such complex disease systems and less prone to resistance.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> An alternative strategy to combination therapy is to develop a single chemical entity that is able to modulate multiple targets simultaneously (designed multiple ligands, DMLs).<a onclick="showRef(event, 'ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref36">(34-36)</a> The overall goal of the DML approach is to enhance the efficacy and/or improve the safety of therapy with respect to drug combination. Additional advantages of DMLs are the reduction of pill burden for patients, due to the administration of a single compound, a lower risk of drug–drug interactions, and improvement in medication adherence.</div><div class="NLM_p">Building on previous works,<a onclick="showRef(event, 'ref37 ref38 ref39 ref40'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39 ref40">(37-40)</a> we describe here pan-demethylase inhibitors <b>1</b>–<b>6</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) that simultaneously inhibit both LSD1 and JmjC KDMs. Compounds <b>1</b>–<b>6</b> were designed by coupling tranylcypromine (<b>7</b>), a known LSD1 inhibitor,<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and 4-carboxy-4′-carbomethoxy-2,2′-bipyridine <b>8</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> or 5-carboxy-8-hydroxyquinoline (<b>9</b>, 5-carboxy-8HQ),<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> two 2OG competitive templates developed for JmjC inhibition (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). The molecular weight of these pan-demethylase inhibitors is acceptable because they result from combination of low molecular weight molecules.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of pan-demethylase inhibitors <b>1</b>–<b>6</b> described here and single-family target inhibitors <b>7</b>–<b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The compounds were selected on the basis of known inhibition results and crystallographic analyses of both the LSDs and JmjC KDMs. On the basis of manual docking studies (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), we proposed that conjugation of tranylcypromine to the carboxylate group of either 4-carboxy-4′-carbomethoxy-2,2′-bipyridine <b>8</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (to form <b>1</b> and <b>2</b>) or 5-carboxy-8-hydroxyquinoline <b>9</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> (to form <b>3</b>–<b>6</b>), would give compounds that are capable of binding to both the LSD and KDM4 active sites (for description of docking, see <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). On the basis of the LSD1 structure (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XAJ">2XAJ</a>),<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> the bipyridyl (for <b>1</b>) and hydroxyquinoline (for <b>3</b>) rings are predicted to protrude into the LSD1 substrate binding pocket, with the tranylcypromine moiety conjugating to FAD, as occurs during inhibition by tranylcypromine <b>7</b> alone (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A).<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Analysis of the active site of KDM4A (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ">3PDQ</a>)<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> leads to the proposal that both <b>1</b> and <b>3</b> can chelate the protein-bound iron via their bipyridyl and hydroxyquinoline groups respectively, as observed for a known bipyridyl inhibitor<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and for 5-carboxy-8HQ <b>9</b>.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Moreover, the primary amines of <b>1</b> and <b>3</b> may be involved in hydrogen bonding and/or electrostatic interactions with KDM4A (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B).</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Manual docking of compounds <b>1</b> (yellow) and <b>3</b> (blue) in (A) LSD1 and (B) KDM4A active sites. (A) In LSD1, <b>1</b> and <b>3</b> are proposed to react with active-site-bound FAD, opening the cyclopropyl ring. The bipyridyl (in the case of <b>1</b>) and 8-hydroxyquinolyl (in the case of <b>3</b>) groups protrude toward the histone substrate-binding pocket. (B) In KDM4A, <b>1</b> and <b>3</b> are proposed to chelate to the active-site-bound Fe(II) via the bipyridyl and 8-hydroxyquinolyl groups, respectively. The tranylcypromine moiety in both compounds likely protrudes toward the histone substrate binding pocket, potentially forming hydrogen bonding and/or electrostatic interactions via the primary amine. Models of both enantiomers of <b>1</b> and <b>3</b> bound to KDM4A are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22812" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22812" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Compounds <b>1</b> and <b>2</b> (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, route A) were prepared by coupling 4-carboxy-4′-carbomethoxy-2,2′-bipyridine <b>8</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> with <i>trans</i>-<i>N</i>-Boc-2-(4-aminophenyl)cyclopropylamine <b>10</b><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> by use of HOBt/EDCI to give amide <b>11</b>; ester hydrolysis to give <b>12</b> followed by N-Boc deprotection with CF<sub>3</sub>CO<sub>2</sub>H gave initial target <b>1</b> (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>; all hybrid compounds were prepared as <i>trans</i> racemates). Alternatively, <b>11</b> was directly hydrolyzed in 4 N HCl to give ester <b>2</b>. To prepare 8-hydroxyquinoline compounds <b>3</b>–<b>6</b> (Scheme <a class="ref internalNav" href="#sch2" aria-label="2">2</a>, route B), 5-carboxy-8HQ <b>9</b><a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> as its O-methoxymethyl-protected derivative (<b>13</b>) was condensed with <b>10</b>, and intermediate <b>14</b> was treated with 4 N HCl to provide <b>3</b>. Reaction of <b>13</b> with the requisite methyl-ω-aminoalkanoate hydrochlorides<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> gave methyl esters <b>15</b>–<b>17</b>, which after alkaline hydrolysis were condensed with <b>10</b> to give the intermediates <b>18</b>–<b>20</b>. Subsequent acid deprotection of <b>18</b>–<b>20</b> afforded final compounds <b>4</b>–<b>6</b>. Alternatively, intermediates <b>15</b>–<b>17</b> were MOM-group-deprotected to give the 8HQ methyl esters <b>21</b>–<b>23</b>, which were hydrolyzed to give acids <b>24</b>–<b>26</b>.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>1</b>–<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Racemic <i>trans</i>-<i>N</i>-Boc-2-(4-aminophenyl)cyclopropylamine <b>10</b>, HOBt, EDCI, Et<sub>3</sub>N, DMF, RT; (b) 0.2 N NaOH, MeOH, RT; (c) CF<sub>3</sub>COOH, 0 °C to RT; (d) 4 N HCl in 1,4-dioxane, THF/MeOH, RT; (e) methyl ω-aminoalkanoate hydrochloride, HOBT, EDCI, Et<sub>3</sub>N, DCM, RT; (f) 2 N LiOH, THF, RT.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i5">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05573" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05573" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> KDM Inhibitory Assays</h3><div class="NLM_p">Compounds <b>1</b>–<b>6</b> were tested against LSD1 and a subfamily representative panel of JmjC KDMs including KDM4. In the LSD1 inhibition assays, the tested compounds all showed single-digit (<b>1</b>, <b>5</b>, <b>6</b>) or submicromolar (<b>2</b>–<b>4</b>) IC<sub>50</sub> values, with similar or increased potencies with respect to the reference compound <b>7</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Compounds <b>1</b>–<b>3</b> were also tested against MAO-A and MAO-B to assess their selectivity toward LSD1: at the tested conditions, <b>1</b> and <b>3</b> were definitively more effective in inhibiting LSD1 than MAOs, while <b>2</b> was less potent against MAO-B but showed similar inhibitory potency against MAO-A with respect to LSD1, analogously to what was observed with other tranylcypromine-based compounds.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Inhibition of JmjC KDMs by hybrid compounds <b>1</b>–<b>6</b> (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) and 8HQ esters (<b>21</b>–<b>23</b>) and acids (<b>24</b>–<b>26</b>) (Table S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>) was then investigated, with <b>8</b> and <b>9</b> as reference compounds. Interestingly, both 8HQ (<b>3</b>–<b>6</b>) and bipyridine (<b>1</b>) hybrid inhibitors showed similar potencies against JmjC KDMs as the reference compounds <b>8</b> and <b>9</b>, demonstrating that the conjugation of tranylcypromine does not adversely affect activity. Compound <b>1</b> displayed low or submicromolar activity against all the tested JmjCs, thus being the most potent of the series. Importantly, when tested against the hypoxia inducible factor hydroxylases FIH and PHD2, the former of which is closely related to the JmjC KDMs, <b>1</b> showed no/weak inhibition, revealing high KDM selectivity. Consistent with previous SAR for the bipyridine derivatives, the free acid is required for high potency against JmjC KDMs, as the ester “pro-drug” form (<b>2</b>) weakens the potency. Single-digit (9 out of 10) values were obtained for <b>2</b>–<b>6</b> against KDM4, the KDM subfamily that has been shown to work synergistically with LSD1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>).<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Inhibition of Lysine-Specific Demethylase 1, Monoamine Oxidases, Jumonji C, and Other 2-Oxoglutarate-Dependent Enzymes by Pan-Demethylase Inhibitors <b>1</b>–<b>6</b><a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="11" align="center" char=".">IC<sub>50</sub>, μM</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">LSD1 (KDM1)</th><th class="colsep0 rowsep0" align="center" char=".">MAO-A</th><th class="colsep0 rowsep0" align="center" char=".">MAO-B</th><th class="colsep0 rowsep0" align="center" char=".">FBXL11 (KDM2/7)</th><th class="colsep0 rowsep0" align="center" char=".">JMJD1A (KDM3)</th><th class="colsep0 rowsep0" align="center" char=".">JMJD2C (KDM4)</th><th class="colsep0 rowsep0" align="center" char=".">JMJD2E (KDM4)</th><th class="colsep0 rowsep0" align="center" char=".">JARID1C (KDM5)</th><th class="colsep0 rowsep0" align="center" char=".">JMJD3 (KDM6)</th><th class="colsep0 rowsep0" align="center" char=".">FIH</th><th class="colsep0 rowsep0" align="center" char=".">PHD2</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b></td><td class="colsep0 rowsep0" align="char" char=".">2.2</td><td class="colsep0 rowsep0" align="char" char=".">35.4</td><td class="colsep0 rowsep0" align="char" char=".">47.0</td><td class="colsep0 rowsep0" align="char" char=".">0.22</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char=".">0.07</td><td class="colsep0 rowsep0" align="char" char=".">0.42</td><td class="colsep0 rowsep0" align="char" char=".">0.19</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">278</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char=".">43.3</td><td class="colsep0 rowsep0" align="char" char=".">12.2</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">2.7</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td><td class="colsep0 rowsep0" align="char" char=".">76</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char=".">8.9</td><td class="colsep0 rowsep0" align="char" char=".">81.0</td><td class="colsep0 rowsep0" align="char" char=".">7.8</td><td class="colsep0 rowsep0" align="char" char=".">31</td><td class="colsep0 rowsep0" align="char" char=".">1.2</td><td class="colsep0 rowsep0" align="char" char=".">3.9</td><td class="colsep0 rowsep0" align="char" char=".">26</td><td class="colsep0 rowsep0" align="char" char=".">27</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">8.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="char" char="."><1</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">5.5</td><td class="colsep0 rowsep0" align="char" char=".">35</td><td class="colsep0 rowsep0" align="char" char=".">18</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="char" char=".">1.6</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">8.2</td><td class="colsep0 rowsep0" align="char" char=".">9.7</td><td class="colsep0 rowsep0" align="char" char=".">3.1</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="char" char=".">1.0</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">12</td><td class="colsep0 rowsep0" align="char" char=".">9.1</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">5.1</td><td class="colsep0 rowsep0" align="char" char=".">37</td><td class="colsep0 rowsep0" align="char" char=".">16</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="char" char=".">2.1</td><td class="colsep0 rowsep0" align="char" char=".">4.5</td><td class="colsep0 rowsep0" align="char" char=".">2.5</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="char" char=".">>100</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="char" char=".">1.1</td><td class="colsep0 rowsep0" align="char" char=".">3.5</td><td class="colsep0 rowsep0" align="char" char=".">5.0</td><td class="colsep0 rowsep0" align="char" char=".">0.03</td><td class="colsep0 rowsep0" align="char" char=".">11.2</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="char" char=".">ND<a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char=".">15</td><td class="colsep0 rowsep0" align="char" char=".">0.17</td><td class="colsep0 rowsep0" align="char" char=".">0.6</td><td class="colsep0 rowsep0" align="char" char=".">0.3</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="char" char=".">0.14</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">The KDM subfamily of each demethylase enzyme is shown in parentheses. Family-specific target inhibitors <b>7</b>–<b>9</b> were used as reference compounds. Inhibition assays were performed in duplicate. The errors in determinations of IC<sub>50</sub> are within ±10% of their values.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">ND, not detectable. Compound <b>9</b> interferes with the peroxidase used in the coupled enzymatic assay and the inhibition could not be reliably measured.</p></div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Effects of <b>2</b> and <b>3</b> on Histone Methylation in HeLa Cells</h3><div class="NLM_p">We then investigated whether the pan-demethylase inhibitors <b>1</b>, <b>2</b>, and <b>3</b> affect selected global histone methylation states in HeLa cells, using immunofluorescence-based methods (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). Compounds <b>7</b> for LSD1 and <b>8a</b><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> (a pro-drug form of <b>8</b>,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) and <b>9</b> for JmjC enzymes were used as references. The hybrid inhibitors <b>2</b> and <b>3</b> caused substantial, dose-dependent increases of methyl marks at lower concentrations than <b>9</b>, whereas <b>7</b> caused essentially no changes in the overall methylation levels (including H3K4 methylations), in agreement with previous studies.<a onclick="showRef(event, 'ref46 ref47 ref48'); return false;" href="javascript:void(0);" class="ref ref46 ref47 ref48">(46-48)</a> The lack of effects on H3K4me3 may reflect the weaker inhibition of H3K4 demethylases (JARID/KDM5) over H3K9me3 KDMs (KDM4), coupled to the simultaneous effect of inhibiting the H3K4me2-selective demethylase LSD1 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) or more complex effects due to inhibition of multiple targets. The 2,2′-bipyridine derivative <b>2</b> caused a H3K9me2/3-specific dose-dependent hypermethylation effect, whereas <b>8a</b> and <b>7</b> alone had no effect. This observation suggests that the “hybridization” of bipyridyl and tranylcypromine motifs has a synergistic effect with respect to inhibition of LSD1 and KDM4/3 families. There was a general decrease in the immunofluorescence signal at higher doses where toxicity was observed (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>I).</div><figure id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A–H) Detection of changes in the global histone methylation levels in HeLa cells after treatment with <b>1</b>, <b>2</b>, <b>3</b>, <b>7</b>, <b>8a</b>, and <b>9</b> over 72 h by use of an immunofluorescence-based assay. (I) Cytotoxicity of <b>2</b>, <b>3</b>, and <b>8a</b> at high doses is shown as number of cells per fixed field of view.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Effects of Pan-KDM Inhibitors in Human Prostate LNCaP and Colon HCT116 Cancer Cells</h3><div class="NLM_p">Compounds <b>1</b>–<b>6</b> (50 μM) were tested in human prostate LNCaP and colon HCT116 cancer cells to investigate their effects on cell cycle (after 30 h), apoptosis (after 30 h), and differentiation (after 48 h), using <b>7</b>–<b>9</b> as reference compounds. In LNCaP cells, <b>2</b>, <b>3</b>, and <b>6</b> caused a strong G1 phase arrest (≥90%) with a substantial fraction of the cells in pre-G1 stage consistent with apoptosis induction, especially for <b>2</b> and <b>3</b> (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>A). In HCT116 cells, the pan-inhibitors <b>3</b> and <b>5</b> and the JmjC-selective inhibitor <b>9</b> displayed cell cycle alteration when compared to controls; however, as with the LNCaP cells, only <b>2</b> and <b>3</b> induced pre-G1 phase accumulation (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>B). Differentiation data for <b>1</b>–<b>9</b> are reported in Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>. Dose–response curves for apoptosis in LNCaP and HCT116 cells were detected after treatment of cells for 48 h with <b>2</b> and <b>3</b> at doses from 10 to 100 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>C). When compared in independent experiments to a combination of the single-family target inhibitors <b>7</b> and <b>8</b>, <b>2</b> and <b>3</b> showed much more efficacy in inducing apoptosis in both LNCaP and HCT116 cells (at 50 μM for 24 h; Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). To assess their differential toxicity, <b>2</b> and <b>3</b> were also tested (50 μM, 24 h) in mesenchymal progenitor (MePR) cells:<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> in this noncancer cell line, <b>2</b> increased the pre-G1 peak percentage from 8.4% to 12.8% with respect to the control, while with <b>3</b> the same value was under the control value (4.2%), suggesting a cancer-cell-selective apoptotic induction for the two pan-KDM inhibitors (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). The cell cycle phase effects of <b>2</b>, <b>3</b>, <b>7</b>, <b>8</b> and <b>7</b> + <b>8</b> combination (50 μM, 24 h) in LNCaP, HCT116, and MePR cell lines are reported in Figure S2, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>.</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of pan-demethylase inhibitors <b>1</b>–<b>6</b> in LNCaP and in HCT116 cells. (A, B) Cell cycle effect (left) and percentage of cells at pre-G1 peak (right) of <b>1</b>–<b>9</b> (50 μM, 30 h) in LNCaP (A) and HCT116 (B) cells are shown. (C) Dose–response curves with <b>2</b> and <b>3</b> for apoptosis (48 h) in LNCaP (left) and HCT116 (right) cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of effects on pre-G1 peak accumulation for <b>2</b>, <b>3</b>, <b>7</b>, <b>8</b>, and combination of <b>7</b> + <b>8</b> (50 μM, 24 h) in LNCaP, HCT116, and MePR cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Despite differences in number of cells in pre-G1 phase in Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>, <b>2</b> and <b>3</b> induced cell death in a cancer-specific manner. These differences, however, could be due to cell populations (i.e., the percentage of pre-G1 for the untreated HCT116 cells was around 2% in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and around 8% in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>). Also times of induction (30 h in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and 24 h in Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>) could justify the differences among percentages in data.</div><div class="NLM_p">Effects on cell proliferation and migration were detected by treating LNCaP and HCT116 cells with 50 μM <b>2</b> and <b>3</b> up to 72 h, using the histone deacetylase (HDAC) inhibitor suberoylanilide hydroxamic acid (SAHA) as a positive control. In LNCaP cells, <b>2</b> showed similar growth arrest as SAHA, while <b>3</b> was less effective (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A, left; in Figure S3, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>, the whole 72-h assay is shown). Nevertheless, differently from SAHA, neither <b>2</b> or <b>3</b> was able to stop migration in this cell line (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A, right). In HCT116 cells, strong antiproliferative effects for <b>2</b> and <b>3</b> were detected even after 6 h (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, left), while the corresponding SAHA effect appeared only at 72 h (Figure S4, <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a>). In this cell line, <b>2</b> and SAHA strongly hampered migration, while <b>3</b> had no effect (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, right).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Proliferation curves and cell migration relative to (A) LNCaP and (B) HCT116 cell lines after 24 h. The HDAC inhibitor SAHA was used as a reference drug. (Left) Proliferation curves after 24 h. Control (untreated cells) in red, cells treated with SAHA (5 μM) in green, cells treated with <b>2</b> (50 μM) in blue, cells treated with <b>3</b> (50 μM) in violet, and base-lines of cells treated with LNCaP (A) in sky-blue or with HCT116 (B) in yellow-green are shown. (Right) Cell migration after 24 h. Histograms have been represented by slope relative to the control (untreated cells), cells treated with SAHA (5 μM), cells treated with <b>2</b> (50 μM), and cells treated with <b>3</b> (50 μM). Data show the mean value from three parallel experiments with error bars showing the standard deviations on top of each column.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Western blot analysis using mark-specific antibodies supported the proposal that the effects of <b>2</b> and <b>3</b> in LNCaP and HCT116 cells are due to both LSD1 and JmjC inhibition. Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a> shows increased H3K4me2/3 and H3K9me3 methylation after treatment with 50 μM <b>2</b> and <b>3</b>.</div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blot analyses of H3K4me2/3 and H3K9me2/3 methylation after treatment of LNCaP (left) and HCT116 (right) cells with <b>2</b>, <b>3</b>, and the reference compounds <b>7</b>–<b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63200" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63200" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In conclusion, we have shown that “pan-KDM” inhibitors can be obtained by coupling the chemical features of tranylcypromine, a known LSD1 inhibitor, with the 2,2′-bipyridine or 5-carboxy-8HQ scaffolds, two 2OG competitive moieties developed for JmjC inhibition. Such compounds are able to inhibit LSD1 as well as JmjC enzymes while having little or no effect against other tested 2OG enzymes lacking KDM activity. It is also interesting that the tranylcypromine moieties act as covalently binding inhibitors, whereas the JmjC inhibitor scaffolds bind reversibly (note the excellent safety record of covalent clinical candidates and marketed agents that has been recently described).<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> The application of such LSD/JmjC hybrid inhibitors to cells enables simultaneous increases in levels of H3K4me2/3 and H3K9me2/3 as well as high growth arrest and apoptosis (<b>2</b> and <b>3</b>) in LNCaP prostate and HCT116 colon cancer cells, whereas the related family-specific single-target inhibitors <b>7</b>–<b>9</b>, as well as a combination of <b>7</b> + <b>8</b>, were inactive. When tested in MePR noncancer cells to assess their differential toxicity, <b>2</b> and <b>3</b> showed very low (<b>2</b>) or no (<b>3</b>) ability to increase the pre-G1 accumulation. It should be noted that since a single KDM may target multiple substrates, different biological effects depending on the context could manifest.</div><div class="NLM_p last">To the best of our knowledge, these are the first pan-KDM inhibitors to be described. More generally, the results demonstrate that hybrid molecules inhibiting different classes of histone-modifying enzymes have substantial potential as functional probes or histone methylation, and we suggest that combining other types of inhibitors targeting histone-modifying enzymes may be productive with respect to regulating the expression of specific (sets of) genes. Our pan-demethylase inhibitors may also be useful for medicinal chemistry efforts relating to cancer, in line with what happened with SAHA, which likely owes its anticancer activity to its pan-HDAC inhibitor profile.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74132" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74132" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Chemistry</h3><div class="NLM_p last">Melting points were determined on a Buchi 530 melting point apparatus. <sup>1</sup>H NMR and <sup>13</sup>C NMR spectra were recorded at 400 MHz on a Bruker AC 400 spectrometer; chemical shifts are reported in δ (parts per million, ppm) units relative to the internal reference tetramethylsilane (Me<sub>4</sub>Si). Mass spectra were recorded on a API-TOF Mariner by Perspective Biosystem (Stratford, TX); samples were injected by an Harvard pump at a flow rate of 5–10 μL/min, infused in the electrospray system. All compounds were routinely checked by thin-layer chromatography (TLC) and <sup>1</sup>H NMR. TLC was performed on aluminum-backed silica gel plates (Merck DC, Alufolien Kieselgel 60 F<sub>254</sub>) with spots visualized by UV light or by use of a KMnO<sub>4</sub> alkaline solution. All solvents were reagent-grade and, when necessary, were purified and dried by standard methods. Concentration of solutions after reactions and extractions involved the use of a rotary evaporator operating at reduced pressure of ∼20 Torr. Organic solutions were dried over anhydrous sodium sulfate. Elemental analysis was used to determine purity of the described compounds, that is, >95%. Analytical results are within 0.40% of theoretical values. All chemicals were purchased from Sigma Aldrich s.r.l. (Milan, Italy) or from TCI Europe N.V. (Zwijndrecht, Belgium) and were of the highest purity. As a rule, samples prepared for physical and biological studies were dried in high vacuum over P<sub>2</sub>O<sub>5</sub> for 20 h at temperatures ranging from 25 to 40 °C, depending on the sample melting point.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> <i>trans</i>-Methyl 4′-[(4-{2-[(<i>tert</i>-Butoxycarbonyl)amino]cyclopropyl}phenyl)carbamoyl]-[2,2′-bipyridine]-4-carboxylate (<b>11</b>)</h3><div class="NLM_p last">The reagents HOBT (62 mg, 0.39 mmol, 1.2 equiv), EDCI (75 mg, 0.39 mmol, 1.2 equiv), and Et<sub>3</sub>N (0.15 mL, 1.11 mmol, 3.4 equiv) were added in sequence to a mixture of <b>8</b><a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> (101 mg, 0.39 mmol, 1.2 equiv) and <i>trans</i>-<i>tert</i>-butyl 2-(4-aminophenyl)cyclopropylcarbamate <b>10</b><a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> (81 mg, 0.33 mmol, 1.0 equiv) in dry <i>N</i>,<i>N</i>-dimethylformamide (DMF) (2.1 mL), and the resulting solution was stirred at room temperature overnight. The reaction was quenched with brine (5 mL) and concentrated under reduced pressure, and the aqueous phase was extracted with AcOEt (4 × 30 mL). The combined organic phases were washed with NaHCO<sub>3</sub> saturated solution (2 × 7 mL) and brine (5 mL). The organic phases were collected, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a crude product that was purified by column chromatography on silica gel (eluent CHCl<sub>3</sub>/MeOH 100/1) to give <b>11</b> (127 mg, 80%) as an off-white solid, mp 196–199 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.10 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.43 [s, 9H, NHCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 1.91 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.62 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.98 (s, 3H, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 7.12 (d, 2H, benzene ring), 7.24 (br s, 1H, CHN<i>H</i>CO<sub>2</sub>tBu), 7.70 (d, 2H, benzene ring), 7.97 (m, 2H, bipyridine rings), 8.89 (s, 2H, bipyridine rings), 8.93 (m, 1H, bipyridine ring), 8.98 (m, 1H, bipyridine ring), 10.64 (br s, 1H, ArN<i>H</i>CO). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 15.6, 22.4, 29.3, 31.3, 49.7, 81.2, 117.8, 118.2, 121.1, 121.8, 122.6, 126.5, 132.4, 137.6, 142.8, 149.2, 150.0, 154.8, 155.3, 156.0, 159.2, 166.7, 168.8. MS, <i>m</i>/<i>z</i> 489 [M + H]<sup>+</sup>.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> General Procedure for Synthesis of <i>trans</i>-Methyl 4′-{[4-(2-Aminocyclopropyl)phenyl]carbamoyl}-[2,2′-bipyridine]-4-carboxylate Hydrochloride (<b>2</b>), <i>trans</i>-<i>N</i>-[4-(2-Aminocyclopropyl)phenyl]-8-hydroxyquinoline-5-carboxamide Hydrochloride (<b>3</b>), and <i>trans</i>-<i>N</i>-(ω-{[4-(2-Aminocyclopropyl)phenyl]amino}-ω-oxoalkyl)-8-hydroxyquinoline-5-carboxamide Hydrochlorides (<b>4</b>–<b>6</b>)</h3><div id="sec5_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i15" class="anchor-spacer"></div><h4 class="article-section__title" id="_i15"> Example: <i>trans</i>-<i>N</i>-(4-{[4-(2-Aminocyclopropyl)phenyl]amino}-4-oxobutyl)-8-hydroxyquinoline-5-carboxamide Hydrochloride (<b>4</b>)</h4><div class="NLM_p last">Compound <b>18</b> (100 mg, 0.18 mmol, 1.0 equiv) was dissolved in a mixture of dry tetrahydrofuran (THF)/MeOH (1.4 mL/1.4 mL) and the solution was stirred at 0 °C. Then 4 N HCl in 1,4-dioxane (3.65 mL, 14.58 mmol, 80 equiv) was added dropwise and the mixture was allowed to warm at room temperature. After 3 h, the suspension was cooled at 0 °C, and 4 N HCl in 1,4-dioxane (1.82 mL, 7.29 mmol, 40 equiv) was added dropwise again. After 8 h, when conversion was complete, the suspension was filtered and washed with dry THF and then with dry Et<sub>2</sub>O to afford <b>4</b> (68 mg, 85%) as a yellow hygroscopic solid, mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.15 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.35 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.90 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.28 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.42 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO), 2.74 (m, 1H, -CH- cyclopropane ring), 7.06 (d, 2H, benzene ring), 7.30 (d, 1H, quinoline ring), 7.52 (d, 2H, benzene ring), 7.87 (m, 2H, quinoline ring), 8.47 (br s, 3H, CHN<i>H</i><sub>2</sub>·<i>H</i>Cl), 8.65 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 9.02 (d, 1H, quinoline ring), 9.24 (d, 1H, quinoline ring), 10.01 (s, 1H, ArN<i>H</i>CO), 11.40 (br s, 1H, quinoline OH). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.9, 22.8, 26.2, 28.5, 34.1, 40.1, 112.4, 119.7, 124.5, 126.7, 128.2, 128.5, 130.5, 131.1, 133.6, 135.3, 138.9, 150.3, 157.7, 162.1, 180.6. MS (relative to free amine), <i>m</i>/<i>z</i> 405 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>trans</i>-Methyl 4′-{[4-(2-Aminocyclopropyl)phenyl]carbamoyl}-[2,2′-bipyridine]-4-carboxylate Hydrochloride (<b>2</b>)</h4><div class="NLM_p last">Yield 61%; mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm: 1.22 (m, 1H, -CH<i>H</i>- cyclopropane ring), 1.39 (m, 1H, -CH<i>H</i>- cyclopropane ring), 2.34 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.81 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.97 (s, 3H, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 7.19 (d, 2H, benzene ring), 7.76 (d, 2H, benzene ring), 7.99 (m, 2H, bipyridine rings), 8.43 (br s, 3H, CHN<i>H</i><sub>2</sub>.<i>H</i>Cl), 8.89 (s, 2H, bipyridine rings), 8.94 (d, 1H, bipyridine ring), 8.98 (d, 1H, bipyridine ring), 10.71 (br s, 1H, ArN<i>H</i>CO). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.5, 22.4, 28.3, 52.0, 117.6, 118.1, 120.8, 122.1, 123.0, 125.8, 133.4, 137.5, 144.1, 149.9, 150.5, 155.2, 156.8, 158.2, 165.3, 166.4. MS (relative to free amine), <i>m</i>/<i>z</i> 389 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i17" class="anchor-spacer"></div><h4 class="article-section__title" id="_i17"> <i>trans</i>-<i>N</i>-[4-(2-Aminocyclopropyl)phenyl]-8-hydroxyquinoline-5-carboxamide Hydrochloride (<b>3</b>)</h4><div class="NLM_p last">Yield 71%; mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.23 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.38 (m, 1H, -C<i>H</i>H- cyclopropane ring), 2.32 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.80 (m, 1H, -CH- cyclopropane ring), 7.17 (d, 2H, benzene ring), 7.28 (d, 1H, quinoline ring), 7.72 (d, 2H, benzene ring), 7.79 (m, 1H, quinoline ring), 7.94 (d, 1H, quinoline ring), 8.38 (br s, 3H, CHN<i>H</i><sub>2</sub>·<i>H</i>Cl), 9.00 (m, 2H, quinoline ring), 10.47 (s, 1H, ArN<i>H</i>CO). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.8, 22.6, 28.2, 113.8, 121.7, 125.7, 126.8, 128.0, 129.1, 130.2, 130.9, 134.1, 135.8, 137.9, 152.1, 159.3, 165.1. MS (relative to free amine), <i>m</i>/<i>z</i> 320 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> <i>trans</i>-<i>N</i>-(5-{[4-(2-Aminocyclopropyl)phenyl]amino}-5-oxopentyl)-8-hydroxyquinoline-5-carboxamide Hydrochloride (<b>5</b>)</h4><div class="NLM_p last">Yield 65%; mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.17 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.33 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.64 (m, 4H, CH<sub>2</sub>C<i>H</i><sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.26 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.36 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO), 2.75 (m, 1H, -CH- cyclopropane ring), 7.07 (d, 2H, benzene ring), 7.21 (d, 1H, quinoline ring), 7.53 (d, 2H, benzene ring), 7.78 (m, 2H, quinoline ring), 8.35 (br s, 3H, CHN<i>H</i><sub>2</sub>·<i>H</i>Cl), 8.57 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 8.97 (d, 1H, quinoline ring), 9.06 (d, 1H, quinoline ring), 9.92 (s, 1H, ArN<i>H</i>CO), 11.10 (br s, 1H, quinoline OH). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.5, 22.3, 23.1, 28.0, 29.4, 38.3, 39.0, 114.4, 121.9, 125.4, 126.6, 127.9, 128.8, 130.5, 131.1, 134.7, 135.4, 138.9, 151.7, 159.3, 162.4, 179.7. MS (relative to free amine), <i>m</i>/<i>z</i> 419 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i19" class="anchor-spacer"></div><h4 class="article-section__title" id="_i19"> <i>trans</i>-<i>N</i>-(6-{[4-(2-Aminocyclopropyl)phenyl]amino}-6-oxohexyl)-8-hydroxyquinoline-5-carboxamide Hydrochloride (<b>6</b>)</h4><div class="NLM_p last">Yield 89%; mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.18 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.38 (m, 3H, -C<i>H</i>H- cyclopropane ring and -CH<sub>2</sub>- aliphatic chain), 1.61 (m, 4H, aliphatic chain), 2.32 (m, 3H, -C<i>H</i>- cyclopropane ring and CH<sub>2</sub>C<i>H</i><sub>2</sub>CO), 2.75 (m, 1H, -CH- cyclopropane ring), 3.33 (t, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 7.06 (d, 2H, benzene ring), 7.33 (d, 1H, quinoline ring), 7.52 (d, 2H, benzene ring), 7.85 (d, 1H, quinoline ring), 7.89 (m, 1H, quinoline ring), 8.43 (br s, 3H, CHN<i>H</i><sub>2</sub>·<i>H</i>Cl), 8.62 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 9.03 (d, 1H, quinoline ring), 9.24 (d, 1H, quinoline ring), 9.92 (s, 1H, ArN<i>H</i>CO). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.3, 22.2, 25.2, 25.7, 28.1, 29.5, 38.4, 39.5, 114.3, 121.5, 125.8, 126.9, 127.7, 128.7, 130.4, 131.3, 134.5, 135.3, 138.8, 151.7, 159.5, 162.7, 179.8. MS (relative to free amine), <i>m</i>/<i>z</i> 433 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i20" class="anchor-spacer"></div><h4 class="article-section__title" id="_i20"> 4′-[(4-{2-[(<i>tert</i>-Butoxycarbonyl)amino]cyclopropyl}phenyl)carbamoyl]-[2,2′-bipyridine]-4-carboxylic Acid (<b>12</b>)</h4><div class="NLM_p last">Compound <b>11</b> (111 mg, 0.227 mmol, 1.0 equiv) was suspended in 4 mL of MeOH and cooled in an ice bath. NaOH (27.3 mg, 0.682 mmol, 3.0 equiv) in water (0.7 mL) was added dropwise, and the mixture was allowed to stir at room temperature. After 9 h the organic solvent was removed in vacuo and the aqueous phase, cooled at 0 °C, was acidified to pH 2 with a solution of 0.5 N KHSO<sub>4</sub>. The obtained suspension was filtered to afford the desired compound <b>12</b> (96 mg, 89%) as a yellow solid, mp >250 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.11 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.39 [s, 9H, NHCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 1.90 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.62 (m, 1H, -C<i>H</i>- cyclopropane ring), 7.12 (d, 2H, benzene ring), 7.24 (br s, 1H, CHN<i>H</i>CO<sub>2</sub>tBu), 7.70 (d, 2H, benzene ring), 7.95 (m, 2H, bipyridine rings), 8.88 (s, 2H, bipyridine rings), 8.94 (m, 2H, bipyridine ring), 10.64 (br s, 1H, ArN<i>H</i>CO), 13.89 (br s, 1H, COO<i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.1, 21.9, 27.8, 32.7, 80.3, 117.6, 118.2, 118.4, 120.1, 121.5, 124.4, 125.3, 134.0, 137.4, 144.9, 149.1, 149.9, 154.9, 155.4, 155.9, 159.5, 164.8, 170.1. MS, <i>m</i>/<i>z</i> 475 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> <i>trans</i>-4′-{[4-(2-Aminocyclopropyl)phenyl]carbamoyl}-[2,2′-bipyridine]-4-carboxylic Acid Trifluoroacetate (<b>1</b>)</h4><div class="NLM_p last">Trifluoroacetic acid (1.04 mL, 13.54 mmol, 230 equiv) was added to <b>12</b> (28.2 mg, 0.059 mmol, 1.0 equiv) placed in an ice bath, and the mixture was allowed to stir at room temperature. After 5 h the solvent was removed at low temperature under reduced pressure, and then the solid was stirred as a suspension in dry Et<sub>2</sub>O for 1 h and filtered to provide the desired compound <b>1</b> (26 mg, 90%) as a dark red solid, mp >250 °C (THF/MeOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.23 (m, 1H, -C<i>H</i>H- cyclopropane ring), 1.33 (m, 1H, -C<i>H</i>H- cyclopropane ring), 2.28 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.84 (m, 1H, -C<i>H</i>- cyclopropane ring), 7.20 (d, 2H, benzene ring), 7.74 (d, 2H, benzene ring), 7.96 (m, 2H, bipyridine rings), 8.26 (br s, 3H, N<i>H</i><sub>2</sub>·CF<sub>3</sub>COO<i>H</i>), 8.88 (s, 2H, bipyridine rings), 8.94 (m, 2H, bipyridine ring), 10.69 (br s, 1H, ArN<i>H</i>CO). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 14.9, 22.6, 28.2, 117.3, 117.9, 118.7, 120.7, 121.2, 124.1, 125.5, 134.5, 138.7, 144.9, 149.2, 149.8, 155.2, 156.1, 159.8, 162.2, 164.9, 170.4. MS (relative to free amine), <i>m</i>/<i>z</i> 375 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> 8-(Methoxymethoxy)quinoline-5-carboxylic Acid (<b>13</b>)</h4><div class="NLM_p last">8-Hydroxyquinoline-5-carboxylic acid (2.41 g, 12.8 mmol, 1 equiv) was added portionwise to a stirring suspension of 60% NaH (1.28 g, 31.9 mmol, 2.5 equiv) in dry THF (30 mL) at 0 °C, and the resulting mixture was stirred under inert atmosphere (N<sub>2</sub>) for 30 min. Then methoxymethyl bromide (MOM-Br; 2.5 mL, 30.6 mmol, 2.4 equiv) dissolved in dry THF (20 mL) was added dropwise, and the suspension was allowed to warm to room temperature under inert atmosphere (N<sub>2</sub>). After 4 h, a solution of LiOH (1.22 g, 51.0 mmol, 4.0 equiv) in water (25.5 mL) was added at 0 °C, and the resulting mixture was stirred at room temperature overnight. The reaction mixture was acidified with glacial acetic acid to pH 4 and then extracted with dichloromethane (DCM; 5 × 90 mL); the organic phases were collected, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, and filtered, and the solvent was removed under reduced pressure to give <b>13</b> (2.92 g, 98%) as a white solid, mp 157–159 °C (CH<sub>3</sub>CN/MeOH). <sup>1</sup>H NMR (CD<sub>3</sub>OD) δ<sub>H</sub>/ppm 2.09 (s, 3H, CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 4.08 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 5.97 (d, 1H, quinoline ring), 6.18 (m, 1H, quinoline ring), 6.83 (d, 1H, quinoline ring), 7.39 (d, 1H, quinoline ring), 8.06 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CD<sub>3</sub>OD) δ<sub>C</sub>/ppm 55.1, 95.3, 104.5, 119.0, 122.7, 126.5, 131.3, 133.1, 140.0, 148.1, 159.7, 167.5. MS, <i>m</i>/<i>z</i> 234 [M + H]<sup>+</sup>.</div></div><div id="sec5_3_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> <i>trans</i>-<i>tert</i>-Butyl-(2-{4-[8-(Methoxymethoxy)quinoline-5-carboxamido]phenyl}cyclopropyl)carbamate (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared following the same procedure used for <b>11</b>, starting from <b>13</b> and <b>10</b>. Yield 67%; mp 188–191 °C (CH<sub>3</sub>CN). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.19 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.49 [s, 9H, NHCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 2.07 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.74 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.61 (s, 3H, CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 5.58 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 7.21 (d, 2H, benzene ring), 7.44 (d, 1H, quinoline ring), 7.56 (m, 3H, quinoline ring and benzene ring), 7.67 (s, 1H, CON<i>H</i>Ar), 7.80 (d, 1H, quinoline ring), 8.93 (d, 1H, quinoline ring), 9.04 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 15.2, 23.3, 29.0, 32.8, 55.4, 80.5, 95.1, 107.8, 121.8, 125.3, 126.0, 127.7, 128.3, 130.0, 131.4, 133.8, 136.5, 138.1, 151.5, 156.4, 161.2, 164.4. MS, <i>m</i>/<i>z</i> 464 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> General Procedure for Synthesis of 8-(Methoxymethoxy)quinoline-5-carboxyamidoalkyl Methyl Esters (<b>15</b>–<b>17</b>)</h3><div id="sec5_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> Example: Methyl 4-({[8-(Methoxymethoxy)quinolin-5-yl]carbonyl}amino)butanoate (<b>15</b>)</h4><div class="NLM_p last">The reagents HOBT (382 mg, 2.40 mmol, 1.2 equiv), EDCI (460 mg, 2.40 mmol, 1.2 equiv), and Et<sub>3</sub>N (1.12 mL, 8 mmol, 4.0 equiv) were added in sequence to a mixture of <b>13</b> (560 mg, 2.40 mmol, 1.2 equiv) and methyl 4-aminobutanoate hydrochloride (306 mg, 2 mmol, 1 equiv) in dry DCM (12 mL), and the solution was stirred at room temperature. After overnight stirring, the reaction mixture was diluted with DCM (90 mL) and the organic phase was washed with Na<sub>2</sub>CO<sub>3</sub> (2 × 50 mL) and brine (2 × 50 mL), dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a residue that was purified by column chromatography on silica gel (eluent DCM/MeOH 35/1). The collected fractions were evaporated and the resulting crude product was finally recrystallized from benzene/cyclohexane to afford <b>15</b> (550 mg, 83%) as a white solid, mp 115–117 °C (benzene/cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 2.04 (m, 2H, NCH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.52 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.61 (m, 5H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub> and CO<sub>2</sub>C<i>H</i><sub>3</sub>), 3.70 (s, 3H, OCH<sub>2</sub>OC<i>H</i><sub>3</sub>), 5.56 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.32 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.39 (d, 1H, quinoline ring), 7.53 (m, 1H, quinoline ring), 7.66 (d, 1H, quinoline ring), 8.91 (d, 1H, quinoline ring), 9.01 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 22.8, 29.5, 40.3, 50.7, 55.4, 92.7, 107.8, 126.3, 129.7, 130.1, 131.3, 131.9, 137.5, 151.3, 163.4, 165.0, 176.1. MS, <i>m</i>/<i>z</i> 333 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> Methyl 5-({[8-(Methoxymethoxy)quinolin-5-yl]carbonyl}amino)pentanoate (<b>16</b>)</h4><div class="NLM_p last">Yield 52%; mp 96–98 °C (benzene/cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.77 (m, 4H, CH<sub>2</sub>C<i>H</i><sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.44 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.53 (m, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 3.60 (s, 3H, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 3.71 (s, 3H, OCH<sub>2</sub>OC<i>H</i><sub>3</sub>), 5.56 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.16 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.39 (d, 1H, quinoline ring), 7.53 (m, 1H, quinoline ring), 7.67 (d, 1H, quinoline ring), 8.89 (d, 1H, quinoline ring), 9.00 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 22.3, 30.1, 32.4, 39.7, 52.6, 56.6, 92.7, 111.2, 128.1, 128.7, 130.2, 132.1, 133.7, 137.9, 155.3, 163.4, 164.1, 176.7. MS, <i>m</i>/<i>z</i> 347 [M + H]<sup>+</sup>.</div></div><div id="sec5_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Methyl 6-({[8-(Methoxymethoxy)quinolin-5-yl]carbonyl}amino)hexanoate (<b>17</b>)</h4><div class="NLM_p last">Yield 66%; mp 121–123 °C (benzene/cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.50 (m, 2H, aliphatic chain), 1.72 (m, 4H, aliphatic chain), 2.38 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.55 (m, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 3.60 (s, 3H, CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 3.69 (s, 3H, CO<sub>2</sub>C<i>H</i>3), 5.56 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.06 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.39 (d, 1H, quinoline ring), 7.53 (m, 1H, quinoline ring), 7.65 (d, 1H, quinoline ring), 8.87 (d, 1H, quinoline ring), 9.01 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 24.3, 25.2, 30.1, 32.9, 41.6, 51.7, 57.6, 93.2, 109.8, 123.2, 128.9, 129.6, 131.2, 132.3, 135.2, 153.8, 161.1, 162.8, 175.8. MS, <i>m</i>/<i>z</i> 361 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> General Procedure for Synthesis of <i>trans</i>-<i>tert</i>-Butyl [2-(4-{ω-([8-(Methoxymethoxy)quinoline-5-carboxamido]alkylamido}phenyl)cyclopropyl]carbamate (<b>18</b>–<b>20</b>)</h3><div id="sec5_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Example: <i>trans</i>-<i>tert</i>-Butyl [2-(4-{6-[8-(Methoxymethoxy)quinoline-5-carboxamido]hexanamido}phenyl)cyclopropyl]carbamate (<b>20</b>)</h4><div class="NLM_p last">LiOH (102 mg, 4.26 mmol, 4.0 equiv) dissolved in water (3 mL) was added to a stirred solution of <b>17</b> (360 mg, 1.07 mmol, 1.0 equiv) in THF (4 mL) placed on an ice bath, and the resulting mixture was stirred at room temperature overnight. The reaction was acidified with glacial acetic acid to pH 4–5, then diluted with DCM (50 mL) and extracted; the aqueous phase was further extracted with DCM (9 × 50 mL). The organic phases were collected, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford 5-({[8-(methoxymethoxy)quinolin-5-yl]carbonyl}amino)hexanoic acid (300 mg, 85%) as a white solid that was used in the next step without further purification. HOBT (92 mg, 0.58 mmol, 1.1 equiv), EDCI (110 mg, 0.58 mmol, 1.1 equiv), and Et<sub>3</sub>N (0.24 mL, 1.70 mmol, 3.2 equiv) were added in sequence to a mixture of 5-({[8-(methoxymethoxy)quinolin-5-yl]carbonyl}amino)hexanoic acid (190 mg, 0.58 mmol, 1.1 equiv) and <i>trans</i>-<i>tert</i>-butyl 2-(4-aminophenyl)cyclopropylcarbamate <b>10</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> (132 mg, 0.53 mmol, 1.0 equiv) in dry DMF (4.2 mL), and the resulting solution was stirred at room temperature. After 18 h the solvent was evaporated under reduced pressure and the residue was partitioned between AcOEt (100 mL) and brine (20 mL). The organic phase was further washed with NaHCO<sub>3</sub> saturated solution (2 × 40 mL) and the aqueous phases were back-extracted with AcOEt (40 mL). The organic phases were collected, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to give a crude product that was purified by column chromatography on silica gel (eluent CHCl<sub>3</sub>/MeOH 30/1) to give <b>20</b> (226 mg, 74%) as a yellow foam, mp 69–72 °C (cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.15 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.47 [s, 9H, NCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 1.53 (m, 2H, aliphatic chain), 1.72 (m, 2H, aliphatic chain), 1.81 (m, 2H, aliphatic chain), 2.02 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.40 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO), 2.68 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.57 (m, 5H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub> and CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 4.88 (br s, 1H, CHN<i>H</i>CO<sub>2</sub>tBu), 5.55 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.28 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.07 (d, 2H, benzene ring), 7.35 (m, 2H, quinoline ring and ArN<i>H</i>CO), 7.40 (d, 2H, benzene ring), 7.50 (m, 1H, quinoline ring), 7.65 (d, 1H, quinoline ring), 8.87 (d, 1H, quinoline ring), 9.00 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 16.3, 21.7, 24.6, 26.7, 29.1, 29.7, 32.5, 37.6, 39.3, 53.8, 79.1, 89.4, 107.5, 124.6, 127.3, 127.8, 128.2, 129.3, 132,5, 133.6, 135.0, 139.2, 140.6, 151.0, 153.3, 159.8, 162.7, 182.9. MS, <i>m</i>/<i>z</i> 577 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> <i>trans</i>-<i>tert</i>-Butyl [2-(4-{4-[8-(Methoxymethoxy)quinoline-5-carboxamido]butanamido}phenyl)cyclopropyl]carbamate (<b>18</b>)</h4><div class="NLM_p last">Yield 88%; mp 184–186 °C (benzene/cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.14 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.47 [s, 9H, NCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 2.02 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.09 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.53 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CONH), 2.69 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.59 (s, 3H, CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 3.66 (m, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 4.89 (br s, 1H, CHN<i>H</i>CO<sub>2</sub>tBu), 5.54 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.65 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.08 (d, 2H, benzene ring), 7.32 (d, 1H, quinoline ring), 7.50 (m, 3H, benzene ring and quinoline ring), 7.65 (d, 1H, quinoline ring), 8.46 (br s, 1H, ArN<i>H</i>CO), 8.89 (d, 1H, quinoline ring), 9.00 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 16.0, 22.1, 26.8, 27.5, 32.3, 35.8, 39.0, 53.5, 80.2, 95.5, 107.4, 119.6, 126.6, 126.9, 128.3, 129.3, 131.5, 133.9, 135.1, 137.7, 141.8, 148.8, 155.9, 159.5, 165.6, 184.4. MS, <i>m</i>/<i>z</i> 549 [M + H]<sup>+</sup>.</div></div><div id="sec5_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> <i>trans</i>-<i>tert</i>-Butyl [2-(4-{5-[8-(Methoxymethoxy)quinoline-5-carboxamido]pentanamido}phenyl)cyclopropyl]carbamate (<b>19</b>)</h4><div class="NLM_p last">Yield 74%; mp 111–114 °C (cyclohexane). <sup>1</sup>H NMR (CDCl<sub>3</sub>) δ<sub>H</sub>/ppm 1.15 (m, 2H, -C<i>H</i><sub>2</sub>- cyclopropane ring), 1.47 [s, 9H, NHCO<sub>2</sub>C(C<i>H</i><sub>3</sub>)<sub>3</sub>], 1.78 (m, 2H, aliphatic chain) 1.87 (m, 2H, aliphatic chain), 2.02 (m, 1H, -C<i>H</i>- cyclopropane ring), 2.09 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.49 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO), 2.68 (m, 1H, -C<i>H</i>- cyclopropane ring), 3.58 (m, 5H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub> and CH<sub>2</sub>OC<i>H</i><sub>3</sub>), 4.87 (br s, 1H, N<i>H</i>CO<sub>2</sub>tBu), 5.56 (s, 2H, OC<i>H</i><sub>2</sub>OCH<sub>3</sub>), 6.41 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 7.07 (d, 2H, benzene ring), 7.38 (d, 1H, quinoline ring), 7.45 (m, 3H, benzene ring and quinoline ring), 7.64 (br s, 1H, ArN<i>H</i>CO), 7.68 (d, 1H, quinoline ring), 8.85 (d, 1H, quinoline ring), 9.00 (d, 1H, quinoline ring). <sup>13</sup>C NMR (CDCl<sub>3</sub>) δ<sub>C</sub>/ppm 15.8, 21.7, 21.9, 27.8, 29.1, 29.8, 37.7, 39.4, 53.0, 80.1, 93.8, 106.9, 123.4, 126.6, 127.0, 127.5, 128.4, 130.0, 131.8, 136.1, 137.7, 138.1, 149.7, 156.5, 161.2, 164.4, 182.2. MS, <i>m</i>/<i>z</i> 563 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> General Procedure for Synthesis of ω-(8-Hydroxyquinoline-5-carboxamido)alkyl Methyl Esters (<b>21</b>–<b>23</b>)</h3><div id="sec5_6_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> Example: Methyl 5-(8-Hydroxyquinoline-5-carboxamido)pentanoate (<b>22</b>)</h4><div class="NLM_p last">Compound <b>16</b> (150 mg, 0.43 mmol, 1.0 equiv) was dissolved in a mixture of dry THF/MeOH (2 mL/2 mL) and the solution was stirred at 0 °C. Then 4 N HCl in 1,4-dioxane (6.5 mL, 25.98 mmol, 60 equiv) was added dropwise, and the mixture was allowed to warm to room temperature. After 6 h, when conversion was complete, the suspension was filtered and washed with dry Et<sub>2</sub>O to give <b>22</b> (125 mg, 95%) as a yellow solid, mp 175–177 °C (CH<sub>3</sub>CN). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.60 (m, 4H, CH<sub>2</sub>C<i>H</i><sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.38 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.32 (m, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 3.60 (s, 3H, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 7.44 (d, 1H, quinoline ring), 7.92 (d, 1H, quinoline ring), 8.00 (m, 1H, quinoline ring), 8.70 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 9.08 (d, 1H, quinoline ring), 9.39 (m, 1H, quinoline ring), 12.17 (br s, 1H, ArO<i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 22.9, 29.7, 32.8, 40.1, 50.8, 111.6, 126.5, 128.5, 131.6, 132.0, 134.9, 140.2, 153.0, 161.2, 163.8, 175.9. MS, <i>m</i>/<i>z</i> 303 [M + H]<sup>+</sup>.</div></div><div id="sec5_6_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Methyl 6-(8-Hydroxyquinoline-5-carboxamido)hexanoate (<b>23</b>)</h4><div class="NLM_p last">Yield 97%; mp 164–166 °C (CH<sub>3</sub>CN). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.37 (m, 2H, aliphatic chain), 1.59 (m, 4H, aliphatic chain), 2.33 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.30 (m, 2H, NHC<i>H</i><sub>2</sub>CH<sub>2</sub>), 7.28 (d, 1H, quinoline ring), 7.82 (d, 1H, quinoline ring), 7.87 (m, 1H, quinoline ring), 8.57 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 9.02 (d, 1H, quinoline ring), 9.18 (d, 1H, quinoline ring), 11.35 (br s, 1H, ArO<i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 24.5, 26.4, 30.0, 32.7, 40.4, 51.5, 109.5, 122.6, 130.4, 130.8, 133.0, 133.7, 134.0, 152.5, 160.8, 163.0, 174.1. MS, <i>m</i>/<i>z</i> 317 [M + H]<sup>+</sup>.</div></div><div id="sec5_6_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> Methyl 4-(8-Hydroxyquinoline-5-carboxamido)butanoate (<b>21</b>)</h4><div class="NLM_p last">Yield 77%; mp 185–186 °C (CH<sub>3</sub>CN). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.84 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.42 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>Me), 3.61 (s, 3H, CO<sub>2</sub>C<i>H</i><sub>3</sub>), 7.09 (d, 1H, quinoline ring), 7.62 (m, 1H, quinoline ring), 7.69 (d, 1H, quinoline ring), 8.43 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 8.87 (m, 2H, quinoline ring), 10.21 (br s, 1H, ArO<i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 23.6, 30.0, 39.7, 50.2, 107.7, 126.8, 129.8, 130.0, 133.1, 133.8, 136.8, 151.2, 162.7, 164.9, 176.0. MS, <i>m</i>/<i>z</i> 289 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> General Procedure for Synthesis of ω-(8-Hydroxyquinoline-5-carboxamido)alkanoic Acids (<b>24</b>–<b>26</b>)</h3><div id="sec5_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> Example: 5-(8-Hydroxyquinoline-5-carboxamido)pentanoic Acid (<b>25</b>)</h4><div class="NLM_p last">A solution of LiOH (47 mg, 1.11 mmol, 3.0 equiv) in water (0.6 mL) was added to a stirred solution of <b>22</b> (112 mg, 0.37 mmol, 1.0 equiv) in THF (1.3 mL) placed on an ice bath, and the resulting mixture was stirred at room temperature overnight. The solvent was removed, NaHCO<sub>3</sub> saturated solution (10 mL) was added, and the aqueous phase was washed with AcOEt (3 × 10 mL). The aqueous solution was acidified with glacial acetic acid to pH 4 and then extracted with AcOEt (5 × 20 mL). The organic phases were collected, dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated in vacuo to afford <b>25</b> (102 mg, 96%) as an off-yellow solid, mp 207–209 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.58 (m, 4H, aliphatic chain), 2.30 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>H), 7.08 (d, 1H, quinoline ring), 7.62 (m, 1H, quinoline ring), 7.68 (d, 1H, quinoline ring), 8.41 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 8.84 (d, 1H, quinoline ring), 8.88 (d, 1H, quinoline ring), 10.17 (br s, 1H, ArO<i>H</i>), 11.98 (br s, 1H, CH<sub>2</sub>CO<sub>2</sub><i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 22.6, 29.8, 33.3, 39.9, 111.1, 126.3, 128.4, 128.7, 131.8, 132.7, 136.0, 151.0, 160.1, 162.9, 177.9. MS, <i>m</i>/<i>z</i> 289 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 5-(8-Hydroxyquinoline-5-carboxamido)hexanoic Acid (<b>26</b>)</h4><div class="NLM_p last">Yield 66%; mp 187–189 °C (CH<sub>3</sub>CN). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.38 (m, 2H, aliphatic chain), 1.57 (m, 4H, aliphatic chain), 2.23 (t, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>H), 7.08 (d, 1H, quinoline ring), 7.62 (m, 1H, quinoline ring), 7.69 (d, 1H, quinoline ring), 8.41 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 8.83 (d, 1H, quinoline ring), 8.88 (d, 1H, quinoline ring), 10.16 (br s, 1H, ArO<i>H</i>), 12.00 (br s, 1H, CH<sub>2</sub>CO<sub>2</sub><i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 24.3, 27.2, 29.0, 34.5, 40.1, 110.4, 125.8, 128.8, 129.1, 131.9, 132.5, 133.9, 153.4, 158.5, 162.3, 175.5. MS, <i>m</i>/<i>z</i> 303 [M + H]<sup>+</sup>.</div></div><div id="sec5_7_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 5-(8-Hydroxyquinoline-5-carboxamido)butanoic Acid (<b>24</b>)</h4><div class="NLM_p last">Yield 86%; mp 217–218 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>H</sub>/ppm 1.79 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>), 2.32 (m, 2H, CH<sub>2</sub>C<i>H</i><sub>2</sub>CO<sub>2</sub>H), 7.08 (d, 1H, quinoline ring), 7.62 (m, 1H, quinoline ring), 7.69 (d, 1H, quinoline ring), 8.44 (br s, 1H, CON<i>H</i>CH<sub>2</sub>), 8.84 (d, 1H, quinoline ring), 8.89 (d, 1H, quinoline ring), 10.20 (br s, 1H, ArO<i>H</i>), 12.05 (br s, 1H, CH<sub>2</sub>CO<sub>2</sub><i>H</i>). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>) δ<sub>C</sub>/ppm 23.2, 33.2, 39.5, 110.8, 126.0, 128.5, 128.7, 131.9, 132.2, 135.5, 151.0, 159.7, 162.9, 178.8. MS, <i>m</i>/<i>z</i> 275 [M + H]<sup>+</sup>.</div></div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Lysine-Specific Demethylase 1 Assay</h3><div class="NLM_p last">His-tagged recombinant form of human LSD1 comprising residues 171–836 was copurified with a glutathione transferase-tagged CoREST protein (residues 308–440) as described.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> The potency of the inhibitors was evaluated by measuring their IC<sub>50</sub> values by a coupled enzymatic assay monitoring hydrogen peroxide formation. A peptide corresponding to the N-terminal 21 amino acids of H3 monomethylated on lysine 4 was used as substrate at the fixed concentration of 29 μM (5 times <i>K</i><sub>m</sub>). The reaction mixture contained 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid (Hepes), pH 7.5, 0.1 mM 4-aminoantipyrine, 1 mM 3,5-dichloro-2-hydroxybenzenesulfonic acid, 2.8 μM horseradish peroxidase, and 1 μM LSD1/CoREST. The enzyme was incubated for 5 min at room temperature before the enzymatic activity was measured.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Monoamine Oxidase A and B Assays</h3><div class="NLM_p last">Recombinant human MAO-A and -B were expressed in <i>Pichia pastoris</i> and purified as described.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> IC<sub>50</sub> values were measured with the peroxidase-coupled assay using benzylamine (MAO-B) and kynuramine (MAO-A) as substrates, at the fixed concentration of 333 μM (1.5 times <i>K</i><sub>m</sub>). The protein (16.7 nM) was titrated with increasing concentrations of inhibitor (5-min incubation at room temperature). The reaction mixture contained 50 mM Hepes, pH 7.5, 0.025% (w/v) reduced Triton-X100, 0.1 mM Amplex Red, 2.8 μM horseradish peroxidase, and 16 nM enzyme.</div></div><div id="sec5_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Jumonji C Inhibitory Assays</h3><div id="sec5_10_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> Materials</h4><div class="NLM_p last">Anti-histone H3 (dimethyl K9) (Abcam, catalog no. ab1220), anti-histone H3 (trimethyl K9) (Abcam, catalog no. ab8988), anti-histone H3 (dimethyl K4) (Abcam, catalog no. ab32356), antihistone H3 (trimethyl K4) (Diagenode, catalog no. pAb-003-010), goat anti-mouse Alexafluor 488 (Invitrogen, catalog no. A21121) or goat anti-rabbit Alexafluor 488 (Invitrogen, catalog no. A11034) were purchased from the indicated sources.</div></div><div id="sec5_10_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 2-Oxoglutarate Oxygenase Enzyme Assay</h4><div class="NLM_p last">All recombinant 2OG oxygenase enzymes were produced as described.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> The IC<sub>50</sub> values of inhibitors against JmjC containing KDMs were determined by use of AlphaScreen as described.<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> RapidFire mass spectrometry/assay methods used for PHD2 and FIH will be reported elsewhere.</div></div><div id="sec5_10_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> Cell Culturing</h4><div class="NLM_p last">HeLa cells were maintained in OptiMEM medium supplemented with 0.5% fetal calf serum and 1% penicillin-–treptomycin. Cells (1500 cells/well) were seeded into 96-well optical-grade plates (Becton Dickinson) and left overnight to adhere. Test compounds were diluted in culture medium at a concentration of 100 μM, further serially diluted at a ratio of 1:2 (1% DMSO final), and incubated on the adhered HeLa cells. Medium containing inhibitors was replaced every 24 h for a 3-day period.</div></div><div id="sec5_10_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> Immunostaining</h4><div class="NLM_p last">Cells were rinsed with phosphate-buffered saline (PBS), fixed in 4% paraformaldehyde (20 min), and permeabilized with 0.5% Triton X-100 (10 min), followed by another PBS rinse. The cells were blocked (30 min) with 3% fetal calf serum diluted in PBS and further incubated overnight in primary antibody (1:500) diluted in blocking solution. Cells were rinsed three times with PBS, followed by incubation with the secondary antibody for 1 h. After PBS rinses (×3), cells were stained with DAPI.</div></div><div id="sec5_10_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> Image Acquisition and Analysis</h4><div class="NLM_p last">The Pathway (Beckton Dickinson), an automated high-content imaging platform, was used to image the immunostained cells in the 96-well plate configuration. For each well, the system acquired a 3-by-3 tile-scanned image for Alexafluor 488 and DAPI. During analysis, the Pathway’s software used the DAPI staining to identify nuclei as regions-of-interests (ROI). For each nucleus, the software extracted the average intensity of the histone staining, followed by the average intensity of all the nuclei in that particular well. The number of remaining cells and average nuclear size of each well was also calculated from the DAPI image. These three parameters from each well were then plotted against the corresponding concentration to obtain a dose–response curve, which was plotted in GraphPad Prism 5.</div></div></div><div id="sec5_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> LNCaP Prostate Cancer, HCT116 Colon Cancer, and MePR Mesenchymal Progenitor Cell Assays</h3><div id="sec5_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> Cell Lines</h4><div class="NLM_p last">HCT116 (human colorectal carcinoma cell line, ATCC) were grown at 37 °C in air and 5% CO<sub>2</sub> in DMEM medium (Euroclone), and LNCaP (human prostate cancer cell line, ATCC) and MePR (mesenchymal progenitor non-cancer-cell line) were grown at 37 °C in air and 5% CO<sub>2</sub> in RPMI medium (Euroclone). Both media were supplemented with 10% heat-inactivated FBS (Euroclone), 1% glutamine (Lonza), 1% penicillin/streptomycin (Euroclone), and 0.1% gentamycin (Lonza).</div></div><div id="sec5_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> Cell Cycle Analysis</h4><div class="NLM_p last">Cells (2.5 × 10<sup>5</sup>, from cell lines HCT116, LNCaP, and MePR) were collected by centrifugation after stimulation at several times with reference or testing compounds at 50 μM. The cells were resuspended in 500 μL of hypotonic buffer (0.1% NP-40, 0.1% sodium citrate, 50 μg/mL PI, and RNase A) and incubated in the dark for 30 min. The analysis was performed by FACS-Calibur (Becton Dickinson) using the Cell Quest Pro software (Becton Dickinson) and ModFit LT version 3 software (Verity). Pre-G1 picks were analyzed as indicative of sub-G1 apoptotic population. The experiments were performed in triplicate, and data shown represent independent media values.</div></div><div id="sec5_11_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> Dose-Dependent Apoptosis Evaluation</h4><div class="NLM_p last">HCT116 and LNCaP cells (2.5 × 10<sup>5</sup>) were treated with increasing doses (10, 25, 50, and 100 μM) of <b>2</b> and <b>3</b>. Cell cycle distribution of 10 000 cells was analyzed with a FACS-Calibur flow cytometer (Becton Dickinson, Franklin Lakes, NJ) by ModFit version 3 Technology (Verity). Pre-G1 picks were analyzed as indicative of sub-G1 apoptotic population. All the experiments were performed at least 3 times, and values are expressed as mean ± standard deviation (SD).</div></div><div id="sec5_11_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> Histone Extraction</h4><div class="NLM_p last">After stimulation with compounds, cells (HCT116 and LNCaP) were collected by centrifugation and washed two times with PBS. Then the samples were resuspended in Triton extraction buffer [TEB; PBS containing 0.5% Triton X-100 (v/v), 2 mM PMSF, and 0.02% (w/v) NaN<sub>3</sub>), and the lysis was performed for 10 min at 4 °C. Next, samples were centrifugated at 2000 rpm for 10 min at 4 °C, and the pellets were washed in TEB (half the volume). After a new centrifugation under the same conditions, the samples were resuspended in 0.2 N HCl and the acidic histone extraction was carried out overnight at 4 °C. The supernatant was recovered by centrifugation and the protein content was quantified by bicinchoninic (BCA) protein assay (Pierce).</div></div><div id="sec5_11_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> Western Blot Analysis</h4><div class="NLM_p">Histone extracts (10 μg) were denatured and boiled in buffer (0.25 M Tris-HCl, pH 6.8, 8% SDS, 40% glycerol, 5% 2-mercaptoethanol, and 0.05% bromophenol blue) for 3 min before electrophoresis. Proteins were subjected to sodium dodecyl sulfate–polyacrylamide gel electrophoresis (SDS–PAGE; 15% polyacrylamide gels) in Tris–glycine–SDS buffer (25 mM Tris, 192 mM glycine, and 0.1% SDS). After electrophoresis, proteins were transferred to nitrocellulose membranes (Bio-Rad mini-protean gel and Transblot Turbo transfer system, Bio-Rad). The membranes were stained with Ponceau red before blocking (with 5% nonfat dry milk in 1× TBS/ 0.1% Tween) and then incubated with the primary antibodies overnight at 4 °C.</div><div class="NLM_p last">The employed antibodies were H3K4me2 and H3K4me3, H3K9me2 and H3K9me 3 (Abcam); total H4 (Cell Signaling) was used to normalize for equal loading of histone extracts.</div></div><div id="sec5_11_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> Determining Cell Proliferation with the xCELLigence System</h4><div class="NLM_p last">Tumor cell proliferation was monitored with the xCELLigence system (Roche, Mannheim, Germany). LNCaP and HCT116 cells were suspended in DMEM and RPMI media, respectively, and added into a 96-well microtiter plate that is specifically designed to measure cellular impedance (E-Plate, Roche, Mannheim, Germany). The measured impedance, which is dependent on the level of confluence, was expressed as an arbitrary unit called cell index (CI) . The cell index at each time point is defined as (<i>R</i><sub>n</sub> – <i>R</i><sub>b</sub>)/(15X), where <i>R</i><sub>n</sub> is the cell-electrode impedance of the well when it contains cells and <i>R</i><sub>b</sub> is the background impedance of the well with medium alone. xCELLigence monitors cellular events in real time, measuring electrical impedance across interdigitated microelectrodes integrated on the bottom of tissue culture E-Plates. The impedance measurement provides quantitative information about the biological status of the cells, including cell number, viability, and morphology. For experiments, LNCaP and HCT116 cell lines were starved in DMEM/10% FBS and RPMI/10% FBS, respectively, overnight before being seeded on an E-Plate 96. Two hours after seeding, scalar cell concentrations were added in triplicate. Dynamic CI values were monitored in 30-min intervals from the time of plating until the end of the experiment. CI values were calculated and plotted on the graph. Standard deviation of tetraplicates of wells for the two types of cells with different treatments were analyzed with RTCA software.</div></div><div id="sec5_11_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> Cell Migration Assay</h4><div class="NLM_p last">Kinetic information about cell migration by dynamically recording the whole cell migration process in real time, without labeling cells, has been performed with the Roche xCELLigence real-time cell analyzer (RTCA) DP instrument. The RTCA DP instrument uses the CIM (cellular invasion/migration)-Plate 16 featuring microelectronic sensors integrated into the underside of the microporous poly(ethylene terephthalate) (PET) membrane of a Boyden-like chamber. In this way cells migrate from the upper chamber through the membrane into the bottom chamber in response to the chemoattractant (we used fetal bovine serum), thus contacting and adhering to the electronic sensors on the underside of the membrane, resulting in an increase of the impedance. The impedance increase is proportional to increasing numbers of migrated cells on the underside of the membrane. Moreover, cell-index values reflecting impedance changes are recorded by the RTCA DP instrument. The CIM-Plate was assembled by placing the top chamber into the bottom chamber and snapping the two together. Serum-free medium was placed in the top chamber to hydrate and preincubate the membrane for 1 h in the CO<sub>2</sub> incubator at 37 °C before a background measurement was obtained. LNCaP and HCT116 cells were resuspended at the indicated concentration in serum-free medium. Once the CIM-Plate was equilibrated, it was placed in the RTCA DP station and the background cell index values were measured. The CIM-Plate was then removed from the RTCA DP station, and cells were added to the top chamber at the desired concentration. The CIM-Plate was placed in the RTCA DP station and migration was monitored every 2 min for several hours. Cells were analyzed in the absence or presence of 10% FBS in the bottom chamber. Cell migration was detected by automated real-time monitoring, and the low and high seeding densities were quantitatively monitored and reflected by the cell index values.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i56"><a href="/doi/suppl/10.1021/jm4012802">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53075" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53075" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Additional text describing manual docking and granulocytic differentiation; <sup>1</sup>H NMR spectra for <b>1</b>–<b>3</b>; two tables listing elemental analyses of <b>1</b>–<b>6</b> and <b>11</b>–<b>26</b> and JmjCs’ IC<sub>50</sub> values for <b>21</b>–<b>26</b>; and four figures showing pro-differentiative effects of <b>1</b>–<b>9</b> in LNCaP and HCT116 cells; cell cycle effects of <b>2</b>, <b>3</b>, <b>7</b>, <b>8</b>, and <b>7</b> + <b>8</b> combination (50 μM, 24 h) in LNCaP, HCT116, and MePR cells; proliferative curves of LNCaP and HCT116 cells treated with 50 μM <b>2</b> and <b>3</b> for 72 h; and migration assay in HCT116 cells with <b>2</b> and <b>3</b> (50 μM, 72 h). This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm4012802/suppl_file/jm4012802_si_001.pdf">jm4012802_si_001.pdf (1.76 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm4012802" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52773" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52773" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antonello Mai</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.
le A. Moro 5, 00185 Rome, Italy</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#066768726968636a6a69286b676f4673686f74696b6737286f72"><span class="__cf_email__" data-cfemail="771619031819121b1b18591a161e3702191e05181a1646591e03">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dante Rotili</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.
le A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefano Tomassi</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Naples “Federico
II”, Via D. Montesano
49, 80131 Naples, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mariarosaria Conte</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Biophysics and General Pathology, Second University of Naples, Vico L. De Crecchio 7, 80138 Naples, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rosaria Benedetti</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Biophysics and General Pathology, Second University of Naples, Vico L. De Crecchio 7, 80138 Naples, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcello Tortorici</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
9, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Giuseppe Ciossani</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
9, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sergio Valente</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.
le A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Biagina Marrocco</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.
le A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Donatella Labella</span> - <span class="hlFld-Affiliation affiliation">Department
of Drug Chemistry and Technologies, Sapienza
University of Rome, P.
le A. Moro 5, 00185 Rome, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ettore Novellino</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmacy, University of Naples “Federico
II”, Via D. Montesano
49, 80131 Naples, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andrea Mattevi</span> - <span class="hlFld-Affiliation affiliation">Department
of Biology and Biotechnology, University
of Pavia, Via Ferrata
9, 27100 Pavia, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lucia Altucci</span> - <span class="hlFld-Affiliation affiliation">Department
of Biochemistry, Biophysics and General Pathology, Second University of Naples, Vico L. De Crecchio 7, 80138 Naples, Italy</span>; 
    <span class="hlFld-Affiliation affiliation">Institute
of Genetics and Biophysics, Consiglio Nazionale
delle Ricerche, Via P.
Castellino, 80100 Naples, Italy</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Anthony Tumber</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Old Road Campus, Roosvelt Drive, Headington OX3 7DU, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Clarence Yapp</span> - <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium, University of Oxford, Old Road Campus, Roosvelt Drive, Headington OX3 7DU, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Oliver N. F. King</span> - <span class="hlFld-Affiliation affiliation">Chemistry
Research Laboratory University of Oxford, 12 Mansfield Road, Oxford OX1 3TA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Richard J. Hopkinson</span> - <span class="hlFld-Affiliation affiliation">Chemistry
Research Laboratory University of Oxford, 12 Mansfield Road, Oxford OX1 3TA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Akane Kawamura</span> - <span class="hlFld-Affiliation affiliation">Chemistry
Research Laboratory University of Oxford, 12 Mansfield Road, Oxford OX1 3TA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher J. Schofield</span> - <span class="hlFld-Affiliation affiliation">Chemistry
Research Laboratory University of Oxford, 12 Mansfield Road, Oxford OX1 3TA, United Kingdom</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.R. and S.T. have contributed equally to the work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d1617e4278-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i58">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47823" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47823" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by FIRB RBFR10ZJQT, Sapienza Ateneo Project 2012, IIT-Sapienza Project, Fondazione Cariplo (2010.0778), AIRC (IG-11342 and IG-11812), MIUR (Progetto Epigen), FP7 Project BLUEPRINT/282510, and COST/TD0905.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i59" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i59"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i60" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i60"> Abbreviations Used</h2><tr><td class="NLM_term">2OG</td><td class="NLM_def"><p class="first last">2-oxoglutarate</p></td></tr><tr><td class="NLM_term">8HQ</td><td class="NLM_def"><p class="first last">8-hydroxyquinoline</p></td></tr><tr><td class="NLM_term">AR</td><td class="NLM_def"><p class="first last">androgen receptor</p></td></tr><tr><td class="NLM_term">CIM</td><td class="NLM_def"><p class="first last">cellular invasion/migration</p></td></tr><tr><td class="NLM_term">CoREST</td><td class="NLM_def"><p class="first last">corepressor of RE1 silencing transcription factor</p></td></tr><tr><td class="NLM_term">DAPI</td><td class="NLM_def"><p class="first last">4′,6-diamidino-2-phenylindole</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle medium</p></td></tr><tr><td class="NLM_term">DMLs</td><td class="NLM_def"><p class="first last">designed multiple ligands</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA methyltransferase 1</p></td></tr><tr><td class="NLM_term">EDCI</td><td class="NLM_def"><p class="first last">1-ethyl-3-(3-dimethylaminopropyl)carbodiimide</p></td></tr><tr><td class="NLM_term">FACS</td><td class="NLM_def"><p class="first last">fluorescence-activated cell sorting</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">FIH</td><td class="NLM_def"><p class="first last">factor inhibiting HIF</p></td></tr><tr><td class="NLM_term">HCT116</td><td class="NLM_def"><p class="first last">human colorectal carcinoma cell line</p></td></tr><tr><td class="NLM_term">HeLa</td><td class="NLM_def"><p class="first last">Henrietta Lacks cells</p></td></tr><tr><td class="NLM_term">HIF</td><td class="NLM_def"><p class="first last">hypoxia inducible factor</p></td></tr><tr><td class="NLM_term">HOBT</td><td class="NLM_def"><p class="first last">1-hydroxybenzotriazole</p></td></tr><tr><td class="NLM_term">KDMs</td><td class="NLM_def"><p class="first last">lysine demethylases</p></td></tr><tr><td class="NLM_term">JmjC</td><td class="NLM_def"><p class="first last">Jumonji C</p></td></tr><tr><td class="NLM_term">LNCaP</td><td class="NLM_def"><p class="first last">lymph node carcinoma of the prostate</p></td></tr><tr><td class="NLM_term">LSDs</td><td class="NLM_def"><p class="first last">lysine-specific demethylases</p></td></tr><tr><td class="NLM_term">MAOs</td><td class="NLM_def"><p class="first last">monoamine oxidases</p></td></tr><tr><td class="NLM_term">MePR</td><td class="NLM_def"><p class="first last">mesenchymal progenitor cells</p></td></tr><tr><td class="NLM_term">MYPT1</td><td class="NLM_def"><p class="first last">myosin phosphatase 1</p></td></tr><tr><td class="NLM_term">PC3</td><td class="NLM_def"><p class="first last">prostate cancer cell line</p></td></tr><tr><td class="NLM_term">PHD2</td><td class="NLM_def"><p class="first last">prolyl hydroxylase domain 2</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr><tr><td class="NLM_term">PMSF</td><td class="NLM_def"><p class="first last">phenylmethanesulfonyl fluoride</p></td></tr><tr><td class="NLM_term">RB1</td><td class="NLM_def"><p class="first last">retinoblastoma protein 1</p></td></tr><tr><td class="NLM_term">RMPI</td><td class="NLM_def"><p class="first last">Roswell Park Memorial Institute medium</p></td></tr><tr><td class="NLM_term">ROI</td><td class="NLM_def"><p class="first last">regions-of-interests</p></td></tr><tr><td class="NLM_term">RTCA</td><td class="NLM_def"><p class="first last">real-time cell analyzer</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal trasducer and activator of transcription 3</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i61">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20515" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20515" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 52 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0ljOZV8YtFVfuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1815</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Kooistra, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0ljOZV8YtFVfuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: Oxidative chemistry for multifaceted functions in chromatin regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+Oxidative+chemistry+for+multifaceted+functions+in+chromatin+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520Oxidative%2520chemistry%2520for%2520multifaceted%2520functions%2520in%2520chromatin%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Targeting histone demethylases: A new avenue for the fight against cancer</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1177%2F1947601911417976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21941621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCgu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=663-679&author=D.+Rotiliauthor=A.+Mai&title=Targeting+histone+demethylases%3A+A+new+avenue+for+the+fight+against+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases: a new avenue for the fight against cancer</span></div><div class="casAuthors">Rotili, Dante; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-679</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  In addn. to genetic disorders, epigenetic alterations have been shown to be involved in cancer, through misregulation of histone modifications.  Miswriting, misreading, and mis-erasing of histone acetylation as well as methylation marks can be actually assocd. with oncogenesis and tumor proliferation.  Historically, methylation of Arg and Lys residues has been considered a stable, irreversible process due to the slow turnover of Me groups in chromatin.  The discovery in recent years of a large no. of histone Lys demethylases (KDMs, belonging to either the amino oxidase or the JmjC family) totally changed this point of view and suggested a new role for dynamic histone methylation in biol. processes.  Since overexpression, alteration, or mutation of a no. of KDMs has been found in many types of cancers, such enzymes could represent diagnostic tools as well as epigenetic targets to modulate for obtaining novel therapeutic weapons against cancer.  The first little steps in this direction are described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXByA6M4P0brVg90H21EOLACvtfcHk0ljVAVy8kwvWxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCgu7jI&md5=0c5fcc8d7b5d5a07431dd26a85b6200a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F1947601911417976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911417976%26sid%3Dliteratum%253Aachs%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DTargeting%2520histone%2520demethylases%253A%2520A%2520new%2520avenue%2520for%2520the%2520fight%2520against%2520cancer%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D663%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lg3M9TgiwSW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Crea, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrar, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danesi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helgason, C. D.</span><span> </span><span class="NLM_article-title">The emerging role of histone lysine demethylases in prostate cancer</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1186%2F1476-4598-11-52" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=52&author=F.+Creaauthor=L.+Sunauthor=A.+Maiauthor=Y.+T.+Chiangauthor=W.+L.+Farrarauthor=R.+Danesiauthor=C.+D.+Helgason&title=The+emerging+role+of+histone+lysine+demethylases+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of histone lysine demethylases in prostate cancer</span></div><div class="casAuthors">Crea, Francesco; Sun, Lei; Mai, Antonello; Chiang, Yan Ting; Farrar, William L.; Danesi, Romano; Helgason, Cheryl D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Early prostate cancer (PCa) is generally treatable and assocd. with good prognosis.  After a variable time, PCa evolves into a highly metastatic and treatment-refractory disease: castration-resistant PCa (CRPC).  Currently, few prognostic factors are available to predict the emergence of CRPC, and no curative option is available.  Epigenetic gene regulation has been shown to trigger PCa metastasis and androgen-independence.  Most epigenetic studies have focused on DNA and histone methyltransferases.  While DNA methylation leads to gene silencing, histone methylation can trigger gene activation or inactivation, depending on the target amino acid residues and the extent of methylation (me1, me2, or me3).  Interestingly, some histone modifiers are essential for PCa tumor-initiating cell (TIC) self-renewal.  TICs are considered the seeds responsible for metastatic spreading and androgen-independence.  Histone Lysine Demethylases (KDMs) are a novel class of epigenetic enzymes which can remove both repressive and activating histone marks.  KDMs are currently grouped into 7 major classes, each one targeting a specific methylation site.  Since their discovery, KDM expression has been found to be deregulated in several neoplasms.  In PCa, KDMs may act as either tumor suppressors or oncogenes, depending on their gene regulatory function.  For example, KDM1A and KDM4C are essential for PCa androgen-dependent proliferation, while PHF8 is involved in PCa migration and invasion.  Interestingly, the possibility of pharmacol. targeting KDMs has been demonstrated.  In the present paper, we summarize the emerging role of KDMs in regulating the metastatic potential and androgen-dependence of PCa.  In addn., we speculate on the possible interaction between KDMs and other epigenetic effectors relevant for PCa TICs.  Finally, we explore the role of KDMs as novel prognostic factors and therapeutic targets.  We believe that studies on histone demethylation may add a novel perspective in our efforts to prevent and cure advanced PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrllNqXZaFkN7Vg90H21EOLACvtfcHk0lg3M9TgiwSW5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSrsbrM&md5=8a8d12d7d2d788da032a668fd8c66ab9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-11-52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-11-52%26sid%3Dliteratum%253Aachs%26aulast%3DCrea%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DY.%2BT.%26aulast%3DFarrar%26aufirst%3DW.%2BL.%26aulast%3DDanesi%26aufirst%3DR.%26aulast%3DHelgason%26aufirst%3DC.%2BD.%26atitle%3DThe%2520emerging%2520role%2520of%2520histone%2520lysine%2520demethylases%2520in%2520prostate%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2012%26volume%3D11%26spage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Amente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lania, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majello, B.</span><span> </span><span class="NLM_article-title">The histone LSD1 demethylase in stemness and cancer transcription programs</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1829</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=981-986&author=S.+Amenteauthor=L.+Laniaauthor=B.+Majello&title=The+histone+LSD1+demethylase+in+stemness+and+cancer+transcription+programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmente%26aufirst%3DS.%26aulast%3DLania%26aufirst%3DL.%26aulast%3DMajello%26aufirst%3DB.%26atitle%3DThe%2520histone%2520LSD1%2520demethylase%2520in%2520stemness%2520and%2520cancer%2520transcription%2520programs%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1829%26spage%3D981%26epage%3D986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Andrisani, O. M.</span><span> </span><span class="NLM_article-title">Deregulation of epigenetic mechanisms by the hepatitis B virus X protein in hepatocarcinogenesis</span> <span class="citation_source-journal">Viruses</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">858</span><span class="NLM_x">–</span> <span class="NLM_lpage">872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.3390%2Fv5030858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltFSmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=858-872&author=O.+M.+Andrisani&title=Deregulation+of+epigenetic+mechanisms+by+the+hepatitis+B+virus+X+protein+in+hepatocarcinogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of epigenetic mechanisms by the hepatitis B virus X protein in hepatocarcinogenesis</span></div><div class="casAuthors">Andrisani, Ourania M.</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">858-872</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This review focuses on the significance of deregulation of epigenetic mechanisms by the hepatitis B virus (HBV) X protein in hepatocarcinogenesis and HBV replication.  Epigenetic mechanisms, DNA methylation, and specific histone modifications, e.g., trimethylation of H3 on lysine-27 or lysine-4, maintain 'cellular memory' by silencing expression of lineage-inducing factors in stem cells and conversely, of pluripotency factors in differentiated cells.  The X protein has been reported to induce expression of DNA methyltransferases (DNMTs), likely promoting epigenetic changes during hepatocarcinogenesis.  Furthermore, in cellular and animal models of X-mediated oncogenic transformation, protein levels of chromatin modifying proteins Suz12 and Znf198 are down-regulated.  Suz12 is essential for the Polycomb Repressive Complex 2 (PRC2) mediating the repressive trimethylation of H3 on lysine-27 (H3K27me3).  Znf198, stabilizes the LSD1-CoREST-HDAC complex that removes, via lysine demethylase1 (LSD1), the activating trimethylation of H3 on lysine-4 (H3K4me3).  Down-regulation of Suz12 also occurs in liver tumors of woodchucks chronically infected by woodchuck hepatitis virus, an animal model recapitulating HBV-mediated hepatocarcinogenesis in humans.  Significantly, subgroups of HBV-induced liver cancer re-express hepatoblast and fetal markers, and imprinted genes, suggesting hepatocyte reprogramming during oncogenic transformation.  Lastly, down-regulation of Suz12 and Znf198 enhances HBV replication.  Collectively, these observations suggest deregulation of epigenetic mechanisms by HBV X protein influences both the viral cycle and the host cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYAyyFUp_UgrVg90H21EOLACvtfcHk0lgLfgJdjDFGMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltFSmsLY%253D&md5=5ebf5dcab8e9a1771cf4c2b78f786eee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fv5030858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv5030858%26sid%3Dliteratum%253Aachs%26aulast%3DAndrisani%26aufirst%3DO.%2BM.%26atitle%3DDeregulation%2520of%2520epigenetic%2520mechanisms%2520by%2520the%2520hepatitis%2520B%2520virus%2520X%2520protein%2520in%2520hepatocarcinogenesis%26jtitle%3DViruses%26date%3D2013%26volume%3D5%26spage%3D858%26epage%3D872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase LSD1 blocks α-herpesvirus lytic replication and reactivation from latency</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1312</span><span class="NLM_x">–</span> <span class="NLM_lpage">1317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnm.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=19855399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=1312-1317&author=Y.+Liangauthor=J.+L.+Vogelauthor=A.+Narayananauthor=H.+Pengauthor=T.+M.+Kristie&title=Inhibition+of+the+histone+demethylase+LSD1+blocks+%CE%B1-herpesvirus+lytic+replication+and+reactivation+from+latency"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase LSD1 blocks α-herpesvirus lytic replication and reactivation from latency</span></div><div class="casAuthors">Liang, Yu; Vogel, Jodi L.; Narayanan, Aarthi; Peng, Hua; Kristie, Thomas M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1312-1317</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible methylation of histone tails serves as either a pos. signal recognized by transcriptional assemblies or a neg. signal that result in repression.  Invading viral pathogens that depend upon the host cell's transcriptional app. are also subject to the regulatory impact of chromatin assembly and modifications.  Here we show that infection by the α-herpesviruses, herpes simplex virus (HSV) and varicella zoster virus (VZV), results in the rapid accumulation of chromatin bearing repressive histone H3 Lys9 methylation.  To enable expression of viral immediate early (IE) genes, both viruses use the cellular transcriptional coactivator host cell factor-1 (HCF-1) to recruit the lysine-specific demethylase-1 (LSD1) to the viral immediate early promoters.  Depletion of LSD1 or inhibition of its activity with monoamine oxidase inhibitors (MAOIs) results in the accumulation of repressive chromatin and a block to viral gene expression.  As HCF-1 is a component of the Set1 and MLL1 histone H3 Lys4 methyltransferase complexes, it thus coordinates modulation of repressive H3 Lys9 methylation levels with addn. of activating H3 Lys4 trimethylation marks.  Strikingly, MAOIs also block the reactivation of HSV from latency in sensory neurons, indicating that the HCF-1 complex is a crucial component of the reactivation mechanism.  The results support pharmaceutical control of histone modifying enzymes as a strategy for controlling herpesvirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppT5vH8lPRnbVg90H21EOLACvtfcHk0lgLfgJdjDFGMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF&md5=bd9aacfdbcba54e30f9482f8fcc96777</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2051%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520blocks%2520%25CE%25B1-herpesvirus%2520lytic%2520replication%2520and%2520reactivation%2520from%2520latency%26jtitle%3DNat.%2520Med.%26date%3D2009%26volume%3D15%26spage%3D1312%26epage%3D1317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyechan, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">The coactivator host cell factor-1 mediates Set1 and MLL1 H3K4 trimethylation at herpesvirus immediate early promoters for initiation of infection</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">10835</span><span class="NLM_x">–</span> <span class="NLM_lpage">10840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=10835-10840&author=A.+Narayananauthor=W.+T.+Ruyechanauthor=T.+M.+Kristie&title=The+coactivator+host+cell+factor-1+mediates+Set1+and+MLL1+H3K4+trimethylation+at+herpesvirus+immediate+early+promoters+for+initiation+of+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DRuyechan%26aufirst%3DW.%2BT.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DThe%2520coactivator%2520host%2520cell%2520factor-1%2520mediates%2520Set1%2520and%2520MLL1%2520H3K4%2520trimethylation%2520at%2520herpesvirus%2520immediate%2520early%2520promoters%2520for%2520initiation%2520of%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D10835%26epage%3D10840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0ljnxOKAfAqlWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Conte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Molecular pathways: the complexity of the epigenome in cancer and recent clinical advances</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">5526</span><span class="NLM_x">–</span> <span class="NLM_lpage">5534</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1158%2F1078-0432.CCR-12-2037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22904103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5526-5534&author=M.+Conteauthor=L.+Altucci&title=Molecular+pathways%3A+the+complexity+of+the+epigenome+in+cancer+and+recent+clinical+advances"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Pathways: The Complexity of the Epigenome in Cancer and Recent Clinical Advances</span></div><div class="casAuthors">Conte, Mariarosaria; Altucci, Lucia</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5526-5534</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Human cancer is causally linked to genomic and epigenomic deregulations.  Epigenetic abnormalities occurring within signaling pathways regulating proliferation, migration, growth, differentiation, transcription, and death signals may be crit. in the progression of malignancies.  Consequently, identification of epigenetic marks and their bioimplications in tumors represents a crucial step toward defining new therapeutic strategies both in cancer treatment and prevention.  Alterations of writers, readers, and erasers in cancer may affect, for example, the methylation and acetylation state of huge areas of chromatin, suggesting that epi-based treatments may require "distinct" therapeutic strategies compared with "canonical" targeted treatments.  Whereas anticancer treatments targeting histone deacetylase and DNA methylation have entered the clinic, addnl. chromatin modification enzymes have not yet been pharmacol. targeted for clin. use in patients.  Thus, a greater insight into alterations occurring on chromatin modifiers and their impact in tumorigenesis represents a crucial advancement in exploiting epigenetic targeting in cancer prevention and treatment.  Here, the interplay of the best known epi-mutations and how their targeting might be optimized are addressed.  Clin Cancer Res; 18(20); 5526-34. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0GJKNWI0QMbVg90H21EOLACvtfcHk0ljnxOKAfAqlWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu73I&md5=866f7001a59f17ba545b2a193687d89f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-2037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-2037%26sid%3Dliteratum%253Aachs%26aulast%3DConte%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DMolecular%2520pathways%253A%2520the%2520complexity%2520of%2520the%2520epigenome%2520in%2520cancer%2520and%2520recent%2520clinical%2520advances%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5526%26epage%3D5534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Abidi, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simensen, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, C. E.</span><span> </span><span class="NLM_article-title">Mutations in JARID1C are associated with X-linked mental retardation, short stature and hyperreflexia</span> <span class="citation_source-journal">J. Med. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1136%2Fjmg.2008.058990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWktbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2008&pages=787-93&author=F.+E.+Abidiauthor=L.+Hollowayauthor=C.+A.+Mooreauthor=D.+D.+Weaverauthor=R.+J.+Simensenauthor=R.+E.+Stevensonauthor=R.+C.+Rogersauthor=C.+E.+Schwartz&title=Mutations+in+JARID1C+are+associated+with+X-linked+mental+retardation%2C+short+stature+and+hyperreflexia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in JARID1C are associated with X-linked mental retardation, short stature and hyperreflexia</span></div><div class="casAuthors">Abidi, F. E.; Holloway, L.; Moore, C. A.; Weaver, D. D.; Simensen, R. J.; Stevenson, R. E.; Rogers, R. C.; Schwartz, C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">787-793</span>CODEN:
                <span class="NLM_cas:coden">JMDGAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-2593</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Mutations in the JARID1C (Jumonji AT-rich interactive domain 1C) gene were recently assocd. with X-linked mental retardation (XLMR).  Mutations in this gene are reported to be one of the relatively more common causes of XLMR with a frequency of approx. 3% in males with proven or probable XLMR.  The JARID1C protein functions as a histone 3 lysine 4 (H3K4) demethylase and is involved in the demethylation of H3K4me3 and H3K4me2.  Mutation anal. of the JARID1C gene was conducted in the following cohorts: probands from 23 XLMR families linked to Xp11.2, 92 males with mental retardation and short stature, and 172 probands from small XLMR families with no linkage information.  Four novel mutations consisting of two missense mutations, p.A77T and p.V504M, and two frame shift mutations, p.E468fsX2 and p.R1481fsX9, were identified in males with mental retardation.  Two of the mutations, p.V504M and p.E468fsX2, are located in the JmjC domain of the JARID1C gene where no previous mutations have been reported.  Addnl. studies showed that the missense mutation, p.V504M, was a de novo event on the grandpaternal X chromosome of the family.  Clin. findings of the nine affected males from the four different families included mental retardation (100%), short stature (55%), hyperreflexia (78%), seizures (33%) and aggressive behavior (44%).  The degree of mental retardation consisted of mild (25%), moderate (12%) and severe (63%).  Based on the clin. observations, male patients with mental retardation, short stature and hyperreflexia should be considered candidates for mutations in the JARID1C gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1xtwZdctwWbVg90H21EOLACvtfcHk0lj8z3JQQG-mJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWktbY%253D&md5=9624349b128f3429d01effd5ba926bfc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fjmg.2008.058990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjmg.2008.058990%26sid%3Dliteratum%253Aachs%26aulast%3DAbidi%26aufirst%3DF.%2BE.%26aulast%3DHolloway%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DC.%2BA.%26aulast%3DWeaver%26aufirst%3DD.%2BD.%26aulast%3DSimensen%26aufirst%3DR.%2BJ.%26aulast%3DStevenson%26aufirst%3DR.%2BE.%26aulast%3DRogers%26aufirst%3DR.%2BC.%26aulast%3DSchwartz%26aufirst%3DC.%2BE.%26atitle%3DMutations%2520in%2520JARID1C%2520are%2520associated%2520with%2520X-linked%2520mental%2520retardation%252C%2520short%2520stature%2520and%2520hyperreflexia%26jtitle%3DJ.%2520Med.%2520Genet.%26date%3D2008%26volume%3D45%26spage%3D787%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Adegbola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browning, M.</span><span> </span><span class="NLM_article-title">A novel mutation in JARID1C/SMCX in a patient with autism spectrum disorder (ASD)</span> <span class="citation_source-journal">Am. J. Med. Genet. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">146A</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1002%2Fajmg.a.32142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislCksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146A&publication_year=2008&pages=505-511&author=A.+Adegbolaauthor=H.+Gaoauthor=S.+Sommerauthor=M.+Browning&title=A+novel+mutation+in+JARID1C%2FSMCX+in+a+patient+with+autism+spectrum+disorder+%28ASD%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A novel mutation in JARID1C/SMCX in a patient with austism spectrum disorder (ASD)</span></div><div class="casAuthors">Adegbola, Abidemi; Gao, Hanlin; Sommer, Steve; Browning, Marsha</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Medical Genetics, Part A</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">146A</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-511</span>CODEN:
                <span class="NLM_cas:coden">AJMGB8</span>;
        ISSN:<span class="NLM_cas:issn">1552-4825</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We describe a nondysmorphic patient with developmental delay and autism spectrum disorder who has a missense mutation in the Jumonji AT-rich interactive domain 1C (JARID1C) gene.  This child first presented at 30 mo of age with stereotyped and repetitive behaviors, impairment in social reciprocity and in the use of multiple nonverbal behaviors, and developmental delay primarily in the language domain.  A diagnosis of autism was made and subsequently confirmed at the current age of 47 mo.  Cytogenetic and fragile X studies were normal.  Mutational anal. revealed a novel missense mutation in exon 16 of the JARID1C gene that results in an arginine to tryptophan substitution at amino acid 766 (R766W).  Sequence alignment anal. with multiple available eukaryotic sequences including the homologous proteins of mouse and zebrafish demonstrated that the affected amino acid is conserved.  JARID1C has not previously been implicated in autism susceptibility.  Recent novel mol. evidence suggests that it is a histone demethylase specific for di- and trimethylated histone 3 lysine 4 (H3K4) and functions as a transcriptional repressor by fostering REST-mediated neuronal gene regulation.  The JARID1C-regulated genes SCN2A, CACNA1H, BDNF, and SLC18A1 have previously been assocd. with autism and cognitive dysfunction.  This patient brings the total no. of reported JARID1C mutations to 14.  This presentation both extends the range of neurocognitive phenotypes attributable to mutations in this gene and illustrates the importance of mol. studies and DNA sequence anal. for accurate diagnosis of monogenic causes of autism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphl01Md4iStrVg90H21EOLACvtfcHk0lj8z3JQQG-mJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislCksrk%253D&md5=a6dcc2325e9abe3e1e108fdc20b99cc9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fajmg.a.32142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fajmg.a.32142%26sid%3Dliteratum%253Aachs%26aulast%3DAdegbola%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSommer%26aufirst%3DS.%26aulast%3DBrowning%26aufirst%3DM.%26atitle%3DA%2520novel%2520mutation%2520in%2520JARID1C%252FSMCX%2520in%2520a%2520patient%2520with%2520autism%2520spectrum%2520disorder%2520%2528ASD%2529%26jtitle%3DAm.%2520J.%2520Med.%2520Genet.%2520A%26date%3D2008%26volume%3D146A%26spage%3D505%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Santos-Rebouças, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fintelman-Rodrigues, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuss, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pimentel, M. M.</span><span> </span><span class="NLM_article-title">A novel nonsense mutation in KDM5C/JARID1C gene causing intellectual disability, short stature and speech delay</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">498</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2011&pages=67-71&author=C.+B.+Santos-Rebou%C3%A7asauthor=N.+Fintelman-Rodriguesauthor=L.+R.+Jensenauthor=A.+W.+Kussauthor=M.+G.+Ribeiroauthor=M.+Camposauthor=J.+M.+Santosauthor=M.+M.+Pimentel&title=A+novel+nonsense+mutation+in+KDM5C%2FJARID1C+gene+causing+intellectual+disability%2C+short+stature+and+speech+delay"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSantos-Rebou%25C3%25A7as%26aufirst%3DC.%2BB.%26aulast%3DFintelman-Rodrigues%26aufirst%3DN.%26aulast%3DJensen%26aufirst%3DL.%2BR.%26aulast%3DKuss%26aufirst%3DA.%2BW.%26aulast%3DRibeiro%26aufirst%3DM.%2BG.%26aulast%3DCampos%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DJ.%2BM.%26aulast%3DPimentel%26aufirst%3DM.%2BM.%26atitle%3DA%2520novel%2520nonsense%2520mutation%2520in%2520KDM5C%252FJARID1C%2520gene%2520causing%2520intellectual%2520disability%252C%2520short%2520stature%2520and%2520speech%2520delay%26jtitle%3DNeurosci.%2520Lett.%26date%3D2011%26volume%3D498%26spage%3D67%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">p53 is regulated by the lysine demethylase LSD1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=105-108&author=J.+Huangauthor=R.+Senguptaauthor=A.+B.+Espejoauthor=M.+G.+Leeauthor=J.+A.+Dorseyauthor=M.+Richterauthor=S.+Opravilauthor=R.+Shiekhattarauthor=M.+T.+Bedfordauthor=T.+Jenuweinauthor=S.+L.+Berger&title=p53+is+regulated+by+the+lysine+demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DEspejo%26aufirst%3DA.%2BB.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3Dp53%2520is%2520regulated%2520by%2520the%2520lysine%2520demethylase%2520LSD1%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D105%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Scoumanne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">The lysine-specific demethylase 1 is required for cell proliferation in both p53-dependent and -independent manners</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">15471</span><span class="NLM_x">–</span> <span class="NLM_lpage">15475</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=15471-15475&author=A.+Scoumanneauthor=X.+Chen&title=The+lysine-specific+demethylase+1+is+required+for+cell+proliferation+in+both+p53-dependent+and+-independent+manners"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DScoumanne%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DThe%2520lysine-specific%2520demethylase%25201%2520is%2520required%2520for%2520cell%2520proliferation%2520in%2520both%2520p53-dependent%2520and%2520-independent%2520manners%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D15471%26epage%3D15475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">21504</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-21504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lgvYujqWQlKNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D21504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Kontaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Lysine methylation regulates E2F1-induced cell death</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.molcel.2010.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=20603083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=152-160&author=H.+Kontakiauthor=I.+Talianidis&title=Lysine+methylation+regulates+E2F1-induced+cell+death"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methylation regulates E2F1-induced cell death</span></div><div class="casAuthors">Kontaki, Haroula; Talianidis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">152-160</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone-modifying enzymes can regulate DNA damage-induced apoptosis through modulation of p53 function.  Here, we show that, in p53-deficient tumor cells, Set9 and LSD1 regulate DNA damage-induced cell death in a manner opposite to that obsd. in p53+/+ cells, via modulation of E2F1 stabilization.  Set9 methylates E2F1 at lysine-185, which prevents E2F1 accumulation during DNA damage and activation of its proapoptotic target gene p73.  This Me mark is removed by LSD1, which is required for E2F1 stabilization and apoptotic function.  The mol. mechanism involves cross-talks between Lys methylation and other covalent modifications that affect E2F1 stability.  Methylation at Lys-185 inhibits acetylation and phosphorylation at distant positions and, in parallel, stimulates ubiquitination and degrdn. of the protein.  The findings illustrate that the function of methyltransferases can have opposing biol. outcomes depending on the specificity of transcription factor targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXEj98lICfQ7Vg90H21EOLACvtfcHk0lgvYujqWQlKNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D&md5=12b34f33cde33be3f37a53f8115aecde</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DKontaki%26aufirst%3DH.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DLysine%2520methylation%2520regulates%2520E2F1-induced%2520cell%2520death%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D152%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Xie, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Z.</span><span> </span><span class="NLM_article-title">Methylation-mediated regulation of E2F1 in DNA damage-induced cell death</span> <span class="citation_source-journal">J. Recept. Signal Transduct. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=139-146&author=Q.+Xieauthor=Y.+Baiauthor=J.+Wuauthor=Y.+Sunauthor=Y.+Wangauthor=Y.+Zhangauthor=P.+Meiauthor=Z.+Yuan&title=Methylation-mediated+regulation+of+E2F1+in+DNA+damage-induced+cell+death"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DP.%26aulast%3DYuan%26aufirst%3DZ.%26atitle%3DMethylation-mediated%2520regulation%2520of%2520E2F1%2520in%2520DNA%2520damage-induced%2520cell%2520death%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduct.%2520Res.%26date%3D2011%26volume%3D31%26spage%3D139%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Demethylation of RB regulator MYPT1 by histone demethylase LSD1 promotes cell cycle progression in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1158%2F0008-5472.CAN-10-2446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21115810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=655-660&author=H.+S.+Choauthor=T.+Suzukiauthor=N.+Dohmaeauthor=S.+Hayamiauthor=M.+Unokiauthor=M.+Yoshimatsuauthor=G.+Toyokawaauthor=M.+Takawaauthor=T.+Chenauthor=J.+K.+Kurashauthor=H.+I.+Fieldauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Demethylation+of+RB+regulator+MYPT1+by+histone+demethylase+LSD1+promotes+cell+cycle+progression+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylation of RB Regulator MYPT1 by Histone Demethylase LSD1 Promotes Cell Cycle Progression in Cancer Cells</span></div><div class="casAuthors">Cho, Hyun-Soo; Suzuki, Takehiro; Dohmae, Naoshi; Hayami, Shinya; Unoki, Motoko; Yoshimatsu, Masanori; Toyokawa, Gouji; Takawa, Masashi; Chen, Taiping; Kurash, Julia K.; Field, Helen I.; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-660</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 (also known as KDM1 and AOF2) is active in various cancer cells, but its biol. significance in human carcinogenesis is unexplored.  In this study, we explored hypothesized interactions between LSD1 and MYPT1, a known regulator of RB1 phosphorylation.  We found that MYPT1 was methylated in vitro and in vivo by histone lysine methyltransferase SETD7 and demethylated by LSD1, identifying Lys 442 of MYPT1 as a target for methylation/demethylation by these enzymes.  LSD1 silencing increased MYPT1 protein levels, decreasing the steady state level of phosphorylated RB1 (Ser 807/811) and reducing E2F activity.  MYPT1 methylation status influenced the affinity of MYPT1 for the ubiquitin-proteasome pathway of protein turnover.  MYPT1 was unstable in murine cells deficient in SETD7, supporting the concept that MYPT1 protein stability is physiol. regulated by methylation status.  LSD1 overexpression could activate RB1 phosphorylation by inducing a destabilization of MYPT1 protein.  Taken together, our results comprise a novel cell cycle regulatory mechanism mediated by methylation/demethylation dynamics, and they reveal the significance of LSD1 overexpression in human carcinogenesis.  Cancer Res; 71(3); 655-60.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh4fxBs2QNSrVg90H21EOLACvtfcHk0ljndkUW3yMOZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D&md5=4812d562b79453755b7c2eb8f74cc3eb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2446%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDemethylation%2520of%2520RB%2520regulator%2520MYPT1%2520by%2520histone%2520demethylase%2520LSD1%2520promotes%2520cell%2520cycle%2520progression%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D655%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lgO6mIK_Kr3sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Solá, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xavier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aranha, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgado, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jepsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, C. M.</span><span> </span><span class="NLM_article-title">p53 interaction with JMJD3 results in its nuclear distribution during mouse neural stem cell differentiation</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">e18421</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+Sol%C3%A1author=J.+M.+Xavierauthor=D.+M.+Santosauthor=M.+M.+Aranhaauthor=A.+L.+Morgadoauthor=K.+Jepsenauthor=C.+M.+Rodrigues&title=p53+interaction+with+JMJD3+results+in+its+nuclear+distribution+during+mouse+neural+stem+cell+differentiation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSol%25C3%25A1%26aufirst%3DS.%26aulast%3DXavier%26aufirst%3DJ.%2BM.%26aulast%3DSantos%26aufirst%3DD.%2BM.%26aulast%3DAranha%26aufirst%3DM.%2BM.%26aulast%3DMorgado%26aufirst%3DA.%2BL.%26aulast%3DJepsen%26aufirst%3DK.%26aulast%3DRodrigues%26aufirst%3DC.%2BM.%26atitle%3Dp53%2520interaction%2520with%2520JMJD3%2520results%2520in%2520its%2520nuclear%2520distribution%2520during%2520mouse%2520neural%2520stem%2520cell%2520differentiation%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudkov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Regulation of NF-kappaB by NSD1/FBXL11-dependent reversible lysine methylation of p65</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=46-51&author=T.+Luauthor=M.+W.+Jacksonauthor=B.+Wangauthor=M.+Yangauthor=M.+R.+Chanceauthor=M.+Miyagiauthor=A.+V.+Gudkovauthor=G.+R.+Stark&title=Regulation+of+NF-kappaB+by+NSD1%2FFBXL11-dependent+reversible+lysine+methylation+of+p65"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJackson%26aufirst%3DM.%2BW.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DGudkov%26aufirst%3DA.%2BV.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DRegulation%2520of%2520NF-kappaB%2520by%2520NSD1%252FFBXL11-dependent%2520reversible%2520lysine%2520methylation%2520of%2520p65%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D46%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greschik, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fncb1546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17277772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=347-353&author=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=H.+Greschikauthor=B.+D.+Fodorauthor=T.+Jenuweinauthor=C.+Voglerauthor=R.+Schneiderauthor=T.+Guntherauthor=R.+Buettnerauthor=E.+Metzgerauthor=R.+Schule&title=Cooperative+demethylation+by+JMJD2C+and+LSD1+promotes+androgen+receptor-dependent+gene+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span></div><div class="casAuthors">Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Greschik, Holger; Fodor, Barna D.; Jenuwein, Thomas; Vogler, Christine; Schneider, Robert; Guenther, Thomas; Buettner, Reinhard; Metzger, Eric; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-353</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Posttranslational modifications of histones, such as methylation, regulate chromatin structure and gene expression.  Recently, lysine-specific demethylase 1 (LSD1), the first histone demethylase, was identified.  LSD1 interacts with the androgen receptor and promotes androgen-dependent transcription of target genes by ligand-induced demethylation of mono- and dimethylated histone H3 at Lys 9 (H3K9) only.  Here, we identify the Jumonji C (JMJC) domain-contg. protein JMJD2C as the first histone tridemethylase regulating androgen receptor function.  JMJD2C interacts with androgen receptor in vitro and in vivo.  Assembly of ligand-bound androgen receptor and JMJD2C on androgen receptor-target genes results in demethylation of tri-Me H3K9 and in stimulation of androgen receptor-dependent transcription.  Conversely, knockdown of JMJD2C inhibits androgen-induced removal of tri-Me H3K9, transcriptional activation and tumor cell proliferation.  Importantly, JMJD2C colocalizes with androgen receptor and LSD1 in normal prostate and in prostate carcinomas.  JMJD2C and LSD1 interact and both demethylases cooperatively stimulate androgen receptor-dependent gene transcription.  In addn., androgen receptor, JMJD2C and LSD1 assemble on chromatin to remove Me groups from mono, di and trimethylated H3K9.  Thus, our data suggest that specific gene regulation requires the assembly and coordinate action of demethylases with distinct substrate specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2b5sXBcoCRrVg90H21EOLACvtfcHk0lgO6mIK_Kr3sw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D&md5=d1f7ec6fbaadf301372ebe644a2e978a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fncb1546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1546%26sid%3Dliteratum%253Aachs%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DCooperative%2520demethylation%2520by%2520JMJD2C%2520and%2520LSD1%2520promotes%2520androgen%2520receptor-dependent%2520gene%2520expression%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D347%26epage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oehme, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamme, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komaarashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, H.</span><span> </span><span class="NLM_article-title">Design, synthesis, enzyme-inhibitory activity, and effect on human cancer cells of a novel series of Jumonji domain-containing protein 2 histone demethylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5629</span><span class="NLM_x">–</span> <span class="NLM_lpage">5638</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1003655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5629-5638&author=S.+Hamadaauthor=T.+Suzukiauthor=K.+Minoauthor=K.+Kosekiauthor=F.+Oehmeauthor=I.+Flammeauthor=H.+Ozasaauthor=Y.+Itohauthor=D.+Ogasawaraauthor=H.+Komaarashiauthor=A.+Katoauthor=H.+Tsumotoauthor=M.+Nakagawaauthor=R.+Hasegawaauthor=T.+Sasakiauthor=N.+Mizukamiauthor=H.+Miyata&title=Design%2C+synthesis%2C+enzyme-inhibitory+activity%2C+and+effect+on+human+cancer+cells+of+a+novel+series+of+Jumonji+domain-containing+protein+2+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Enzyme-Inhibitory Activity, and Effect on Human Cancer Cells of a Novel Series of Jumonji Domain-Containing Protein 2 Histone Demethylase Inhibitors</span></div><div class="casAuthors">Hamada, Shohei; Suzuki, Takayoshi; Mino, Koshiki; Koseki, Koichi; Oehme, Felix; Flamme, Ingo; Ozasa, Hiroki; Itoh, Yukihiro; Ogasawara, Daisuke; Komaarashi, Haruka; Kato, Aiko; Tsumoto, Hiroki; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5629-5638</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of Jumonji domain-contg. protein (JMJD) histone demethylases are candidate anticancer agents as well as potential tools for elucidating the biol. functions of JMJDs.  On the basis of the crystal structure of JMJD2A and a homol. model of JMJD2C, we designed and prepd. a series of hydroxamate analogs bearing a tertiary amine.  Enzyme assays using JMJD2C, JMJD2A, and prolyl hydroxylases revealed that hydroxamate analog 8 is a potent and selective JMJD2 inhibitor, showing 500-fold greater JMJD2C-inhibitory activity and more than 9100-fold greater JMJD2C-selectivity compared with the lead compd. N-oxalylglycine 2.  Compds. 17 and 18, prodrugs of compd. 8, each showed synergistic growth inhibition of cancer cells in combination with an inhibitor of lysine-specific demethylase 1 (LSD1).  These findings suggest that combination treatment with JMJD2 inhibitors and LSD1 inhibitors may represent a novel strategy for anticancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT6G3YdmcF_bVg90H21EOLACvtfcHk0liG7QlpUVf_IA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCis7o%253D&md5=f4ab3a75cb995d88668dd8527f554ea0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm1003655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1003655%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DOehme%26aufirst%3DF.%26aulast%3DFlamme%26aufirst%3DI.%26aulast%3DOzasa%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DKomaarashi%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DHasegawa%26aufirst%3DR.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DMizukami%26aufirst%3DN.%26aulast%3DMiyata%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%252C%2520enzyme-inhibitory%2520activity%252C%2520and%2520effect%2520on%2520human%2520cancer%2520cells%2520of%2520a%2520novel%2520series%2520of%2520Jumonji%2520domain-containing%2520protein%25202%2520histone%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5629%26epage%3D5638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Sams-Dodd, F.</span><span> </span><span class="NLM_article-title">Target-based drug discovery: is something wrong?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=139-147&author=F.+Sams-Dodd&title=Target-based+drug+discovery%3A+is+something+wrong%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSams-Dodd%26aufirst%3DF.%26atitle%3DTarget-based%2520drug%2520discovery%253A%2520is%2520something%2520wrong%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D139%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Keith, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borisy, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, B. R.</span><span> </span><span class="NLM_article-title">Multicomponent therapeutics for networked systems</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=461-467&author=C.+T.+Keithauthor=A.+A.+Borisyauthor=B.+R.+Stockwell&title=Multicomponent+therapeutics+for+networked+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeith%26aufirst%3DC.%2BT.%26aulast%3DBorisy%26aufirst%3DA.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DMulticomponent%2520therapeutics%2520for%2520networked%2520systems%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D461%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Kamb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span> </span><span class="NLM_article-title">Why is cancer drug discovery so difficult?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=115-120&author=A.+Kambauthor=S.+Weeauthor=C.+Lengauer&title=Why+is+cancer+drug+discovery+so+difficult%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26aulast%3DWee%26aufirst%3DS.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DWhy%2520is%2520cancer%2520drug%2520discovery%2520so%2520difficult%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D115%26epage%3D120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Daub, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Specht, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullrich, A.</span><span> </span><span class="NLM_article-title">Strategies to overcome resistance to targeted protein kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnrd1579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=15573099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=1001-1010&author=H.+Daubauthor=K.+Spechtauthor=A.+Ullrich&title=Strategies+to+overcome+resistance+to+targeted+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to overcome resistance to targeted protein kinase inhibitors</span></div><div class="casAuthors">Daub, Henrik; Specht, Katja; Ullrich, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1001-1010</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Selective inhibition of protein tyrosine kinases is gaining importance as an effective therapeutic approach for the treatment of a wide range of human cancers.  However, as extensively documented for the BCR-ABL oncogene in imatinib-treated leukemia patients, clin. resistance caused by mutations in the targeted oncogene has been obsd.  Here, we look at how structural and mechanistic insights from imatinib-insensitive Bcr-Abl have been exploited to identify second-generation drugs that override acquired target resistance.  These insights have created a rationale for the development of either multi-targeted protein kinase inhibitors or cocktails of selective antagonists as antitumor drugs that combine increased therapeutic potency with a reduced risk of the emergence of mol. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_U6mWwYG307Vg90H21EOLACvtfcHk0liXOaRhO6a9qQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsL3K&md5=49c2ce8b0150e562e382d5246b3c0a42</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd1579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1579%26sid%3Dliteratum%253Aachs%26aulast%3DDaub%26aufirst%3DH.%26aulast%3DSpecht%26aufirst%3DK.%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DStrategies%2520to%2520overcome%2520resistance%2520to%2520targeted%2520protein%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D1001%26epage%3D1010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Rather, M. A.; Bhat, B. A.; Qurishi, M. A.</span><span> </span><span class="NLM_article-title">Multicomponent phytotherapeutic approach gaining momentum: Is the “one drug to fit all” model breaking down?</span> <span class="citation_source-journal">Phytomedicine</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span><a href="http://dx.doi.org/10.1016/j.phymed.2013.07.015" class="extLink">http://dx.doi.org/10.1016/j.phymed.2013.07.015</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=M.+A.+Rather&author=B.+A.+Bhat&author=M.+A.+Qurishi&title=Multicomponent+phytotherapeutic+approach+gaining+momentum%3A+Is+the+%E2%80%9Cone+drug+to+fit+all%E2%80%9D+model+breaking+down%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRather%26aufirst%3DM.%2BA.%26atitle%3DMulticomponent%2520phytotherapeutic%2520approach%2520gaining%2520momentum%253A%2520Is%2520the%2520%25E2%2580%259Cone%2520drug%2520to%2520fit%2520all%25E2%2580%259D%2520model%2520breaking%2520down%253F%26jtitle%3DPhytomedicine%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Morphy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankovic, Z.</span><span> </span><span class="NLM_article-title">Designed multiple ligands. An emerging drug discovery paradigm</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6523</span><span class="NLM_x">–</span> <span class="NLM_lpage">6543</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm058225d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6523-6543&author=R.+Morphyauthor=Z.+Rankovic&title=Designed+multiple+ligands.+An+emerging+drug+discovery+paradigm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Designed Multiple Ligands.</span></div><div class="casAuthors">Morphy, Richard; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6523-6543</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The review discusses main areas in designed multiple ligands (DMLs) discovery that include 5HT transporter-based DMLs for depression, dopamine D2-receptor based DMLs for schizophrenia, histamine H1 receptor-based DMLs for allergies, DMLs targeting the angiotensin system for hypertension, nonsteroidal anti-inflammatory drugs, and PPAR-based DMLs for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgouWcjsiKgbVg90H21EOLACvtfcHk0lhc7ZlGnjznjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL&md5=00e3bf5a521fc91996ce95bb379a0b5c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm058225d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm058225d%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DDesigned%2520multiple%2520ligands.%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6523%26epage%3D6543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Zimmerman, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehàr, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keith, C. T.</span><span> </span><span class="NLM_article-title">Multi-target therapeutics: When the whole is greater than the sum of the parts</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=34-42&author=G.+R.+Zimmermanauthor=J.+Leh%C3%A0rauthor=C.+T.+Keith&title=Multi-target+therapeutics%3A+When+the+whole+is+greater+than+the+sum+of+the+parts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerman%26aufirst%3DG.%2BR.%26aulast%3DLeh%25C3%25A0r%26aufirst%3DJ.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26atitle%3DMulti-target%2520therapeutics%253A%2520When%2520the%2520whole%2520is%2520greater%2520than%2520the%2520sum%2520of%2520the%2520parts%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D34%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Fortin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bérubé, G.</span><span> </span><span class="NLM_article-title">Advances in the development of hybrid anticancer drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1047</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1517%2F17460441.2013.798296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=23646979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOitL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1029-1047&author=S.+Fortinauthor=G.+B%C3%A9rub%C3%A9&title=Advances+in+the+development+of+hybrid+anticancer+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of hybrid anticancer drugs</span></div><div class="casAuthors">Fortin, Sebastien; Berube, Gervais</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1029-1047</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Hybrid anticancer drugs are of great therapeutic interests as they can potentially overcome most of the pharmacokinetic drawbacks encountered when using conventional anticancer drugs.  In fact, the future of hybrid anticancer drugs is very bright for the discovery of highly potent and selective mols. that triggers two or more cytocidal pharmacol. mechanisms of action acting in synergy to inhibit cancer tumor growth.  Areas covered: This review represents the most advanced and recent data in the field of hybrid anticancer agents covering mainly the past 5 years of research.  It also accounts for other significant reviews already published on the topic of anticancer hybrids.  The review showcases the research that is at the leading edge of hybrid anticancer drug discovery.  The main areas covered by the present review are: DNA alkylating agent hybrids (e.g., platinum(II), nitrogen mustard, etc.), vitamin-D receptor, agonist-histone deacetylase inhibitors, combi-mol. therapies and other types of hybrid anticancer agents.  Expert opinion: The current development in the field describes strategies that have never been used before for the design of hybrid anticancer drugs.  The information currently available and described in this section allows us to identify the main parameters required to design such mols.  It also provides a clear view of the future directions that must be explored for the successful development and discovery of useful hybrid anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1FoQV3stg57Vg90H21EOLACvtfcHk0lhc7ZlGnjznjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOitL7E&md5=d5157ee2d3cc8778f9129b051bf52e0d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.798296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.798296%26sid%3Dliteratum%253Aachs%26aulast%3DFortin%26aufirst%3DS.%26aulast%3DB%25C3%25A9rub%25C3%25A9%26aufirst%3DG.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520hybrid%2520anticancer%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D1029%26epage%3D1047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: mixed histone/protein methyltransferase, acetyltransferase, and class III deacetylase (sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+mixed+histone%2Fprotein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28sirtuin%29+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520mixed%2520histone%252Fprotein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Upadhyay, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">An analog of BIX-01294 selectively inhibits a family of histone H3 lysine 9 Jumonji demethylases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">416</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2012&pages=319-327&author=K.+Upadhyayauthor=D.+Rotiliauthor=J.+W.+Hanauthor=R.+Huauthor=Y.+Changauthor=D.+Labellaauthor=X.+Zhangauthor=Y.+S.+Yoonauthor=A.+Maiauthor=X.+Cheng&title=An+analog+of+BIX-01294+selectively+inhibits+a+family+of+histone+H3+lysine+9+Jumonji+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DUpadhyay%26aufirst%3DK.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DJ.%2BW.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYoon%26aufirst%3DY.%2BS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAn%2520analog%2520of%2520BIX-01294%2520selectively%2520inhibits%2520a%2520family%2520of%2520histone%2520H3%2520lysine%25209%2520Jumonji%2520demethylases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D416%26spage%3D319%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altun, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackeen, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">A photoreactive small-molecule probe for 2-oxoglutarate oxygenases</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">642</span><span class="NLM_x">–</span> <span class="NLM_lpage">654</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.chembiol.2011.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21609845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=642-654&author=D.+Rotiliauthor=M.+Altunauthor=A.+Kawamuraauthor=A.+Wolfauthor=R.+Fischerauthor=I.+K.+Leungauthor=M.+M.+Mackeenauthor=Y.+M.+Tianauthor=P.+J.+Ratcliffeauthor=A.+Maiauthor=B.+M.+Kesslerauthor=C.+J.+Schofield&title=A+photoreactive+small-molecule+probe+for+2-oxoglutarate+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A Photoreactive Small-Molecule Probe for 2-Oxoglutarate Oxygenases</span></div><div class="casAuthors">Rotili, Dante; Altun, Mikael; Kawamura, Akane; Wolf, Alexander; Fischer, Roman; Leung, Ivanhoe K. H.; MacKeen, Mukram M.; Tian, Ya-min; Ratcliffe, Peter J.; Mai, Antonello; Kessler, Benedikt M.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">642-654</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">2-Oxoglutarate (2-OG)-dependent oxygenases have diverse roles in human biol.  The inhibition of several 2-OG oxygenases is being targeted for therapeutic intervention, including for cancer, anemia, and ischemic diseases.  We report a small-mol. probe for 2-OG oxygenases that employs a hydroxyquinoline template coupled to a photoactivable crosslinking group and an affinity-purifn. tag.  Following studies with recombinant proteins, the probe was shown to crosslink to 2-OG oxygenases in human crude cell exts., including to proteins at endogenous levels.  This approach is useful for inhibitor profiling, as demonstrated by crosslinking to the histone demethylase FBXL11 (KDM2A) in HEK293T nuclear exts.  The results also suggest that small-mol. probes may be suitable for substrate identification studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq76D2hRKWJjbVg90H21EOLACvtfcHk0lguYFJnaVEE3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslOksLk%253D&md5=cd8f7d35948c618d6b21e9c62602b90c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DMackeen%26aufirst%3DM.%2BM.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DA%2520photoreactive%2520small-molecule%2520probe%2520for%25202-oxoglutarate%2520oxygenases%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D642%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperman, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">7056</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-7056&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+R.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermanauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0lguYFJnaVEE3A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%2BR.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOpperman%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D7056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aik, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, J. B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperman, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3110</span><span class="NLM_x">–</span> <span class="NLM_lpage">3117</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1039%2Fc3sc51122g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3110-3117&author=R.+J.+Hopkinsonauthor=A.+Tumberauthor=C.+Yappauthor=R.+Chowdhuryauthor=W.+Aikauthor=K.+H.+Cheauthor=X.+S.+Liauthor=J.+B.+L.+Kristensenauthor=O.+N.+F.+Kingauthor=M.+C.+Chanauthor=K.+K.+Yeohauthor=H.+Choiauthor=L.+J.+Walportauthor=C.+C.+Thinnesauthor=J.+T.+Bushauthor=C.+Lejeuneauthor=A.+M.+Rydzikauthor=N.+R.+Roseauthor=E.+A.+Baggauthor=M.+A.+McDonoughauthor=T.+J.+Krojerauthor=W.+W.+Yueauthor=S.+S.+Ngauthor=L.+Olsenauthor=E.+S.+Butlerauthor=U.+Oppermanauthor=S.+Mullerauthor=R.+J.+Kloseauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=5-Carboxy-8-hydroxyquinoline+is+a+broad+spectrum+2-oxoglutarate+oxygenase+inhibitor+which+causes+iron+translocation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span></div><div class="casAuthors">Hopkinson, Richard J.; Tumber, Anthony; Yapp, Clarence; Chowdhury, Rasheduzzaman; Aik, WeiShen; Che, Ka Hing; Li, Xuan Shirley; Kristensen, Jan B. L.; King, Oliver N. F.; Chan, Mun Chiang; Yeoh, Kar Kheng; Choi, Hwanho; Walport, Louise J.; Thinnes, Cyrille C.; Bush, Jacob T.; Lejeune, Clarisse; Rydzik, Anna M.; Rose, Nathan R.; Bagg, Eleanor A.; McDonough, Michael A.; Krojer, Tobias J.; Yue, Wyatt W.; Ng, Stanley S.; Olsen, Lars; Brennan, Paul E.; Oppermann, Udo; Mueller, Susanne; Klose, Robert J.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3110-3117</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate and iron dependent oxygenases are therapeutic targets for human diseases.  Using a representative 2OG oxygenase panel, the authors compare the inhibitory activities of 5-carboxy-8-hydroxyquinoline (IOX1) and 4-carboxy-8-hydroxyquinoline (4C8HQ) with that of two other commonly used 2OG oxygenase inhibitors, N-oxalylglycine (NOG) and 2,4-pyridinedicarboxylic acid (2,4-PDCA).  IOX1 has a broad spectrum of activity, as demonstrated by the inhibition of transcription factor hydroxylases, representatives of all 2OG dependent histone demethylase subfamilies, nucleic acid demethylases and γ-butyrobetaine hydroxylase.  Cellular assays show that, unlike NOG and 2,4-PDCA, IOX1 is active against both cytosolic and nuclear 2OG oxygenases without ester derivatization.  Unexpectedly, crystallog. studies on these oxygenases demonstrate that IOX1, but not 4C8HQ, can cause translocation of the active site metal, revealing a rare example of protein ligand-induced metal movement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_NBtJvNEBOLVg90H21EOLACvtfcHk0ljck3KyToz-Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L&md5=506fb4e75971c57a7a968f4d37805856</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2Fc3sc51122g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc51122g%26sid%3Dliteratum%253Aachs%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DAik%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKristensen%26aufirst%3DJ.%2BB.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKrojer%26aufirst%3DT.%2BJ.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DButler%26aufirst%3DE.%2BS.%26aulast%3DOpperman%26aufirst%3DU.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3D5-Carboxy-8-hydroxyquinoline%2520is%2520a%2520broad%2520spectrum%25202-oxoglutarate%2520oxygenase%2520inhibitor%2520which%2520causes%2520iron%2520translocation%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D3110%26epage%3D3117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheinost, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=87-91&author=S.+S.+Ngauthor=K.+L.+Kavanaghauthor=M.+A.+McDonoughauthor=D.+Butlerauthor=E.+S.+Pilkaauthor=B.+M.+Lienardauthor=J.+E.+Brayauthor=P.+Savitskyauthor=C.+Gileadiauthor=F.+Von+Delftauthor=N.+R.+Roseauthor=J.+Offerauthor=J.+C.+Scheinostauthor=T.+Borowskiauthor=M.+Sundstromauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Crystal+structures+of+histone+demethylase+JMJD2A+reveal+basis+for+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DButler%26aufirst%3DD.%26aulast%3DPilka%26aufirst%3DE.%2BS.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DOffer%26aufirst%3DJ.%26aulast%3DScheinost%26aufirst%3DJ.%2BC.%26aulast%3DBorowski%26aufirst%3DT.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520demethylase%2520JMJD2A%2520reveal%2520basis%2520for%2520substrate%2520specificity%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D87%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span> </span><span class="NLM_article-title">Inhibition of histone demethylases by 4-carboxy-2,2′-bipyridyl compounds</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">764</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=759-764&author=K.+H.+Changauthor=O.+N.+Kingauthor=A.+Tumberauthor=E.+C.+Woonauthor=T.+D.+Heightmanauthor=M.+A.+McDonoughauthor=C.+J.+Schofieldauthor=N.+R.+Rose&title=Inhibition+of+histone+demethylases+by+4-carboxy-2%2C2%E2%80%B2-bipyridyl+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DRose%26aufirst%3DN.%2BR.%26atitle%3DInhibition%2520of%2520histone%2520demethylases%2520by%25204-carboxy-2%252C2%25E2%2580%25B2-bipyridyl%2520compounds%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D759%26epage%3D764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Gros, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorente, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yardley, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wharton, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Little, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Perez, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Pacanowska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span> </span><span class="NLM_article-title">Evaluation of azasterols as anti-parasitics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6094</span><span class="NLM_x">–</span> <span class="NLM_lpage">6103</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060290f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6094-6103&author=L.+Grosauthor=S.+O.+Lorenteauthor=C.+J.+Jimenezauthor=V.+Yardleyauthor=L.+Rattrayauthor=H.+Whartonauthor=S.+Littleauthor=S.+L.+Croftauthor=L.+M.+Ruiz-Perezauthor=D.+Gonzalez-Pacanowskaauthor=I.+H.+Gilbert&title=Evaluation+of+azasterols+as+anti-parasitics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm060290f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060290f%26sid%3Dliteratum%253Aachs%26aulast%3DGros%26aufirst%3DL.%26aulast%3DLorente%26aufirst%3DS.%2BO.%26aulast%3DJimenez%26aufirst%3DC.%2BJ.%26aulast%3DYardley%26aufirst%3DV.%26aulast%3DRattray%26aufirst%3DL.%26aulast%3DWharton%26aufirst%3DH.%26aulast%3DLittle%26aufirst%3DS.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DRuiz-Perez%26aufirst%3DL.%2BM.%26aulast%3DGonzalez-Pacanowska%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DEvaluation%2520of%2520azasterols%2520as%2520anti-parasitics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6094%26epage%3D6103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span> <span class="citation_source-journal">Elife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_elocation-id">e00633</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DElife%26date%3D2013%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0liwOwF831LheA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Schenk, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göllner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebestreit, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popescu, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller-Tidow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelent, A.</span><span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+G%C3%B6llnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+M%C3%BCller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID γ (with interleukin-2 (IL-2) receptor γ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0lg_4krL7TIF5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DG%25C3%25B6llner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D605%26epage%3D611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dell’Aversana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricciardiello, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petraglia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carissimo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruotti, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savarese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinelli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancemi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">MePR: a novel human mesenchymal progenitor model with characteristics of pluripotency</span> <span class="citation_source-journal">Stem Cells Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2368</span><span class="NLM_x">–</span> <span class="NLM_lpage">2383</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=2368-2383&author=M.+Miceliauthor=G.+Franciauthor=C.+Dell%E2%80%99Aversanaauthor=F.+Ricciardielloauthor=F.+Petragliaauthor=A.+Carissimoauthor=L.+Peroneauthor=G.+M.+Maruottiauthor=M.+Savareseauthor=P.+Martinelliauthor=M.+Cancemiauthor=L.+Altucci&title=MePR%3A+a+novel+human+mesenchymal+progenitor+model+with+characteristics+of+pluripotency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DFranci%26aufirst%3DG.%26aulast%3DDell%25E2%2580%2599Aversana%26aufirst%3DC.%26aulast%3DRicciardiello%26aufirst%3DF.%26aulast%3DPetraglia%26aufirst%3DF.%26aulast%3DCarissimo%26aufirst%3DA.%26aulast%3DPerone%26aufirst%3DL.%26aulast%3DMaruotti%26aufirst%3DG.%2BM.%26aulast%3DSavarese%26aufirst%3DM.%26aulast%3DMartinelli%26aufirst%3DP.%26aulast%3DCancemi%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DMePR%253A%2520a%2520novel%2520human%2520mesenchymal%2520progenitor%2520model%2520with%2520characteristics%2520of%2520pluripotency%26jtitle%3DStem%2520Cells%2520Dev.%26date%3D2013%26volume%3D22%26spage%3D2368%26epage%3D2383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D. K.</span><span> </span><span class="NLM_article-title">Drug discovery for a new generation of covalent drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1517%2F17460441.2012.688744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22607458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=561-581&author=A.+S.+Kalgutkarauthor=D.+K.+Dalvie&title=Drug+discovery+for+a+new+generation+of+covalent+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery for a new generation of covalent drugs</span></div><div class="casAuthors">Kalgutkar, Amit S.; Dalvie, Deepak K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">561-581</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The design of target-specific covalent inhibitors is conceptually attractive because of increased biochem. efficiency through covalency and increased duration of action that outlasts the pharmacokinetics of the agent.  Although many covalent inhibitors have been approved or are in advanced clin. trials to treat indications such as cancer and hepatitis C, there is a general tendency to avoid them as drug candidates because of concerns regarding immune-mediated toxicity that can arise from indiscriminate reactivity with off-target proteins.Areas covered: The review examines potential reason(s) for the excellent safety record of marketed covalent agents and advanced clin. candidates for emerging therapeutic targets.  A significant emphasis is placed on proteomic techniques and chem./biochem. reactivity assays that aim to provide a systematic rank ordering of pharmacol. selectivity relative to off-target protein reactivity of covalent inhibitors.Expert opinion: While tactics to examine selective covalent modification of the pharmacol. target are broadly applicable in drug discovery, it is unclear whether the output from such studies can prospectively predict idiosyncratic immune-mediated drug toxicity.  Opinions regarding an acceptable threshold of protein reactivity/body burden for a toxic electrophile and a non-toxic electrophilic covalent drug have not been defined.  Increasing confidence in proteomic and chem./biochem. reactivity screens will require a retrospective side-by-side profiling of marketed covalent drugs and electrophiles known to cause deleterious toxic effects via non-selective covalent binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmXy20mDTPbVg90H21EOLACvtfcHk0lg_4krL7TIF5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D&md5=b7fc8e60b263dda5765dd8380f52df17</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688744%26sid%3Dliteratum%253Aachs%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26atitle%3DDrug%2520discovery%2520for%2520a%2520new%2520generation%2520of%2520covalent%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D561%26epage%3D581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Marks, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span> </span><span class="NLM_article-title">Dimethyl sulfoxide to vorinostat: Development of this histone deacetylase inhibitor as an anticancer drug</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnbt1272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17211407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=84-90&author=P.+A.+Marksauthor=R.+Breslow&title=Dimethyl+sulfoxide+to+vorinostat%3A+Development+of+this+histone+deacetylase+inhibitor+as+an+anticancer+drug"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl sulfoxide to vorinostat: development of this histone deacetylase inhibitor as an anticancer drug</span></div><div class="casAuthors">Marks, Paul A.; Breslow, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-90</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In our quest to understand why DMSO (DMSO) can cause growth arrest and terminal differentiation of transformed cells, we followed a path that led us to discover suberoylanilide hydroxamic acid (SAHA; vorinostat (Zolinza)), which is a histone deacetylase inhibitor.  SAHA reacts with and blocks the catalytic site of these enzymes.  Extensive structure-activity studies were done along the path from DMSO to SAHA.  SAHA can cause growth arrest and death of a broad variety of transformed cells both in vitro and in tumor-bearing animals at concns. not toxic to normal cells.  SAHA has many protein targets whose structure and function are altered by acetylation, including chromatin-assocd. histones, nonhistone gene transcription factors and proteins involved in regulation of cell proliferation, migration and death.  In clin. trials, SAHA has shown significant anticancer activity against both hematol. and solid tumors at doses well tolerated by patients.  A new drug application was approved by the US Food and Drug Administration for vorinostat for treatment of cutaneous T-cell lymphoma.  More potent analogs of SAHA have shown unacceptable toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK3endE54rQ7Vg90H21EOLACvtfcHk0lgDJqgc51ls0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D&md5=cc2a39d80d5d5bea0a1377bdf48a957d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnbt1272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1272%26sid%3Dliteratum%253Aachs%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBreslow%26aufirst%3DR.%26atitle%3DDimethyl%2520sulfoxide%2520to%2520vorinostat%253A%2520Development%2520of%2520this%2520histone%2520deacetylase%2520inhibitor%2520as%2520an%2520anticancer%2520drug%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D84%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strain-Damerell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kochan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">Plant growth regulator daminozide is a selective inhibitor of human KDM2/7 histone demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6639</span><span class="NLM_x">–</span> <span class="NLM_lpage">6643</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300677j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6639-6643&author=N.+R.+Roseauthor=E.+C.+Y.+Woonauthor=A.+Tumberauthor=L.+J.+Walportauthor=R.+Chowdhuryauthor=X.+S.+Liauthor=O.+N.+Kingauthor=C.+Lejeuneauthor=S.+S.+Ngauthor=T.+Krojerauthor=M.+C.+Chanauthor=A.+M.+Rydzikauthor=R.+J.+Hopkinsonauthor=K.+H.+Cheauthor=M.+Danielauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=G.+Kochanauthor=I.+K.+Leungauthor=J.+Dunfordauthor=K.+K.+Yeohauthor=P.+J.+Ratcliffeauthor=N.+Burgess-Brownauthor=F.+von+Delftauthor=S.+Mullerauthor=B.+Marsdenauthor=P.+E.+Brennanauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=R.+J.+Kloseauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=Plant+growth+regulator+daminozide+is+a+selective+inhibitor+of+human+KDM2%2F7+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Plant Growth Regulator Daminozide Is a Selective Inhibitor of Human KDM2/7 Histone Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Tumber, Anthony; Walport, Louise J.; Chowdhury, Rasheduzzaman; Li, Xuan Shirley; King, Oliver N. F.; Lejeune, Clarisse; Ng, Stanley S.; Krojer, Tobias; Chan, Mun Chiang; Rydzik, Anna M.; Hopkinson, Richard J.; Che, Ka Hing; Daniel, Michelle; Strain-Damerell, Claire; Gileadi, Carina; Kochan, Grazyna; Leung, Ivanhoe K. H.; Dunford, James; Yeoh, Kar Kheng; Ratcliffe, Peter J.; Burgess-Brown, Nicola; von Delft, Frank; Muller, Susanne; Marsden, Brian; Brennan, Paul E.; McDonough, Michael A.; Oppermann, Udo; Klose, Robert J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6639-6643</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The JmjC oxygenases catalyze the N-demethylation of Nε-Me lysine residues in histones and are current therapeutic targets.  A set of human 2-oxoglutarate analogs were screened using a unified assay platform for JmjC demethylases and related oxygenases.  Results led to the finding that daminozide (N-(dimethylamino)succinamic acid, 160 Da), a plant growth regulator, selectively inhibits the KDM2/7 JmjC subfamily.  Kinetic and crystallog. studies reveal that daminozide chelates the active site metal via its hydrazide carbonyl and dimethylamino groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwDtqe21TmpbVg90H21EOLACvtfcHk0lgDJqgc51ls0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFSrur0%253D&md5=25072edc07e7e039e6e742dc215a6b61</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm300677j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300677j%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%2BY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDaniel%26aufirst%3DM.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DKochan%26aufirst%3DG.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DDunford%26aufirst%3DJ.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DPlant%2520growth%2520regulator%2520daminozide%2520is%2520a%2520selective%2520inhibitor%2520of%2520human%2520KDM2%252F7%2520histone%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6639%26epage%3D6643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 84 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Xing-Jie Dai, Ying Liu, Xiao-Peng Xiong, Lei-Peng Xue, Yi-Chao Zheng, <span class="NLM_string-name hlFld-ContribAuthor">Hong-Min Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Tranylcypromine Based Lysine-Specific Demethylase 1 Inhibitor: Summary and Perspective. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (23)
                                     , 14197-14215. <a href="https://doi.org/10.1021/acs.jmedchem.0c00919" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00919</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00919&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00919%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTranylcypromine%252BBased%252BLysine-Specific%252BDemethylase%252B1%252BInhibitor%25253A%252BSummary%252Band%252BPerspective%26aulast%3DDai%26aufirst%3DXing-Jie%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30052020%26date%3D30092020%26date%3D15092020%26volume%3D63%26issue%3D23%26spage%3D14197%26epage%3D14215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Martin Roatsch, Inga Hoffmann, Martine I. Abboud, Rebecca L. Hancock, Hanna Tarhonskaya, Kuo-Feng Hsu, Sarah E. Wilkins, Tzu-Lan Yeh, Kerstin Lippl, Kerstin Serrer, Isabelle Moneke, Theresa D. Ahrens, Dina Robaa, Sandra Wenzler, Nicolas P. F. Barthes, Henriette Franz, Wolfgang Sippl, Silke Lassmann, Sven Diederichs, Erik Schleicher, Christopher J. Schofield, Akane Kawamura, Roland Schüle, <span class="NLM_string-name hlFld-ContribAuthor">Manfred Jung</span>. </span><span class="cited-content_cbyCitation_article-title">The Clinically Used Iron Chelator Deferasirox Is an Inhibitor of Epigenetic JumonjiC Domain-Containing Histone Demethylases. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (8)
                                     , 1737-1750. <a href="https://doi.org/10.1021/acschembio.9b00289" title="DOI URL">https://doi.org/10.1021/acschembio.9b00289</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00289%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DThe%252BClinically%252BUsed%252BIron%252BChelator%252BDeferasirox%252BIs%252Ban%252BInhibitor%252Bof%252BEpigenetic%252BJumonjiC%252BDomain-Containing%252BHistone%252BDemethylases%26aulast%3DRoatsch%26aufirst%3DMartin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12042019%26date%3D09072019%26date%3D19072019%26date%3D09072019%26volume%3D14%26issue%3D8%26spage%3D1737%26epage%3D1750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tian-Shu Kang, Chung-Nga Ko, Jia-Tong Zhang, Chun Wu, Chun-Yuen Wong, Dik-Lung Ma, <span class="NLM_string-name hlFld-ContribAuthor">Chung-Hang Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Rhodium(III)-Based Inhibitor of the JMJD3-H3K27me3 Interaction and Modulator of the Inflammatory Response. </span><span class="cited-content_cbyCitation_journal-name">Inorganic Chemistry</span><span> <strong>2018,</strong> <em>57 </em>
                                    (22)
                                     , 14023-14026. <a href="https://doi.org/10.1021/acs.inorgchem.8b02256" title="DOI URL">https://doi.org/10.1021/acs.inorgchem.8b02256</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.inorgchem.8b02256&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.inorgchem.8b02256%26sid%3Dliteratum%253Aachs%26jtitle%3DInorganic%2520Chemistry%26atitle%3DRhodium%252528III%252529-Based%252BInhibitor%252Bof%252Bthe%252BJMJD3-H3K27me3%252BInteraction%252Band%252BModulator%252Bof%252Bthe%252BInflammatory%252BResponse%26aulast%3DKang%26aufirst%3DTian-Shu%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10082018%26date%3D30102018%26volume%3D57%26issue%3D22%26spage%3D14023%26epage%3D14026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tom E.  McAllister</span>, <span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Richard J.  Hopkinson</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, and <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>  . </span><span class="cited-content_cbyCitation_article-title">Recent Progress in Histone Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (4)
                                     , 1308-1329. <a href="https://doi.org/10.1021/acs.jmedchem.5b01758" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01758</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01758&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01758%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252BProgress%252Bin%252BHistone%252BDemethylase%252BInhibitors%26aulast%3DMcAllister%26aufirst%3DTom%2BE.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D10112015%26date%3D02022016%26date%3D25022016%26date%3D28122015%26volume%3D59%26issue%3D4%26spage%3D1308%26epage%3D1329" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yukihiro  Itoh</span>, <span class="hlFld-ContribAuthor ">Hideyuki  Sawada</span>, <span class="hlFld-ContribAuthor ">Miki  Suzuki</span>, <span class="hlFld-ContribAuthor ">Toshifumi  Tojo</span>, <span class="hlFld-ContribAuthor ">Ryuzo  Sasaki</span>, <span class="hlFld-ContribAuthor ">Makoto  Hasegawa</span>, <span class="hlFld-ContribAuthor ">Tamio  Mizukami</span>, and <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of Jumonji AT-Rich Interactive Domain 1A Inhibitors and Their Effect on Cancer Cells. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>6 </em>
                                    (6)
                                     , 665-670. <a href="https://doi.org/10.1021/acsmedchemlett.5b00083" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.5b00083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.5b00083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.5b00083%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252BJumonji%252BAT-Rich%252BInteractive%252BDomain%252B1A%252BInhibitors%252Band%252BTheir%252BEffect%252Bon%252BCancer%252BCells%26aulast%3DItoh%26aufirst%3DYukihiro%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D17022015%26date%3D23042015%26date%3D28042015%26date%3D11062015%26date%3D23042015%26volume%3D6%26issue%3D6%26spage%3D665%26epage%3D670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Li-Ying  Ma</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Sai-Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Ru  Wang</span>, <span class="hlFld-ContribAuthor ">Lu-Ping  Pang</span>, <span class="hlFld-ContribAuthor ">Miao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jun-Wei  Wang</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Juan  Li</span>, <span class="hlFld-ContribAuthor ">Cong  Wang</span>, <span class="hlFld-ContribAuthor ">Biao  Hu</span>, <span class="hlFld-ContribAuthor ">Ying  Liu</span>, <span class="hlFld-ContribAuthor ">Xiao-Dan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jia-Jia  Wang</span>, <span class="hlFld-ContribAuthor ">Zhi-Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, and <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>  . </span><span class="cited-content_cbyCitation_article-title">Design, Synthesis, and Structure–Activity Relationship of Novel LSD1 Inhibitors Based on Pyrimidine–Thiourea Hybrids As Potent, Orally Active Antitumor Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>58 </em>
                                    (4)
                                     , 1705-1716. <a href="https://doi.org/10.1021/acs.jmedchem.5b00037" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00037%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252BSynthesis%25252C%252Band%252BStructure%2525E2%252580%252593Activity%252BRelationship%252Bof%252BNovel%252BLSD1%252BInhibitors%252BBased%252Bon%252BPyrimidine%2525E2%252580%252593Thiourea%252BHybrids%252BAs%252BPotent%25252C%252BOrally%252BActive%252BAntitumor%252BAgents%26aulast%3DMa%26aufirst%3DLi-Ying%26date%3D2015%26date%3D2015%26date%3D2014%26date%3D09092014%26date%3D06022015%26date%3D26022015%26date%3D22012015%26volume%3D58%26issue%3D4%26spage%3D1705%26epage%3D1716" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Veronica  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Bruna  Saponara</span>, <span class="hlFld-ContribAuthor ">Roberto  Cirilli</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Donatella  Labella</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Giovanni  Ruoppolo</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, and <span class="hlFld-ContribAuthor ">Antonello  Mai</span>  . </span><span class="cited-content_cbyCitation_article-title">Pure Diastereomers of a Tranylcypromine-Based LSD1 Inhibitor: Enzyme Selectivity and In-Cell Studies. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2015,</strong> <em>6 </em>
                                    (2)
                                     , 173-177. <a href="https://doi.org/10.1021/ml500424z" title="DOI URL">https://doi.org/10.1021/ml500424z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ml500424z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fml500424z%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DPure%252BDiastereomers%252Bof%252Ba%252BTranylcypromine-Based%252BLSD1%252BInhibitor%25253A%252BEnzyme%252BSelectivity%252Band%252BIn-Cell%252BStudies%26aulast%3DValente%26aufirst%3DSergio%26date%3D2015%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D18102014%26date%3D08122014%26date%3D12122014%26date%3D12022015%26date%3D08122014%26volume%3D6%26issue%3D2%26spage%3D173%26epage%3D177" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kunal  Nepali</span>, <span class="hlFld-ContribAuthor ">Jing-Ping  Liou</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in epigenetic cancer therapeutics: clinical advancement and emerging trends. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomedical Science</span><span> <strong>2021,</strong> <em>28 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12929-021-00721-x" title="DOI URL">https://doi.org/10.1186/s12929-021-00721-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12929-021-00721-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12929-021-00721-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomedical%2520Science%26atitle%3DRecent%252Bdevelopments%252Bin%252Bepigenetic%252Bcancer%252Btherapeutics%25253A%252Bclinical%252Badvancement%252Band%252Bemerging%252Btrends%26aulast%3DNepali%26aufirst%3DKunal%26date%3D2021%26date%3D2021%26volume%3D28%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Shao-Jie  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Jing  Shi</span>, <span class="hlFld-ContribAuthor ">Lin-Feng  Jin</span>, <span class="hlFld-ContribAuthor ">Tong  Yu</span>, <span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>. </span><span class="cited-content_cbyCitation_article-title">Research progress of dual inhibitors targeting crosstalk between histone epigenetic modulators for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113588. <a href="https://doi.org/10.1016/j.ejmech.2021.113588" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113588</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113588&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113588%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DResearch%252Bprogress%252Bof%252Bdual%252Binhibitors%252Btargeting%252Bcrosstalk%252Bbetween%252Bhistone%252Bepigenetic%252Bmodulators%252Bfor%252Bcancer%252Btherapy%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2021%26volume%3D222%26spage%3D113588" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kathryn  Davis</span>, <span class="hlFld-ContribAuthor ">Presley  Azarcon</span>, <span class="hlFld-ContribAuthor ">Samuel  Hickenlooper</span>, <span class="hlFld-ContribAuthor ">Ryan  Bia</span>, <span class="hlFld-ContribAuthor ">Emilee  Horiuchi</span>, <span class="hlFld-ContribAuthor ">Marta W.  Szulik</span>, <span class="hlFld-ContribAuthor ">Sarah  Franklin</span>. </span><span class="cited-content_cbyCitation_article-title">The role of demethylases in cardiac development and disease. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular and Cellular Cardiology</span><span> <strong>2021,</strong> <em>158 </em>, 89-100. <a href="https://doi.org/10.1016/j.yjmcc.2021.05.018" title="DOI URL">https://doi.org/10.1016/j.yjmcc.2021.05.018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.yjmcc.2021.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.yjmcc.2021.05.018%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520and%2520Cellular%2520Cardiology%26atitle%3DThe%252Brole%252Bof%252Bdemethylases%252Bin%252Bcardiac%252Bdevelopment%252Band%252Bdisease%26aulast%3DDavis%26aufirst%3DKathryn%26date%3D2021%26volume%3D158%26spage%3D89%26epage%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Megan H  Trager</span>, <span class="hlFld-ContribAuthor ">Bindeshwar  Sah</span>, <span class="hlFld-ContribAuthor ">Zhongming  Chen</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Control of Breast Cancer Pathogenesis by Histone Methylation and the Hairless Histone Demethylase. </span><span class="cited-content_cbyCitation_journal-name">Endocrinology</span><span> <strong>2021,</strong> <em>162 </em>
                                    (8)
                                     <a href="https://doi.org/10.1210/endocr/bqab088" title="DOI URL">https://doi.org/10.1210/endocr/bqab088</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1210/endocr/bqab088&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1210%2Fendocr%2Fbqab088%26sid%3Dliteratum%253Aachs%26jtitle%3DEndocrinology%26atitle%3DControl%252Bof%252BBreast%252BCancer%252BPathogenesis%252Bby%252BHistone%252BMethylation%252Band%252Bthe%252BHairless%252BHistone%252BDemethylase%26aulast%3DTrager%26aufirst%3DMegan%2BH%26date%3D2021%26date%3D2021%26volume%3D162%26issue%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stephin  Baby</span>, <span class="hlFld-ContribAuthor ">Durgesh  Gurukkala Valapil</span>, <span class="hlFld-ContribAuthor ">Nagula  Shankaraiah</span>. </span><span class="cited-content_cbyCitation_article-title">Unravelling KDM4 histone demethylase inhibitors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2021,</strong> <em>21 </em><a href="https://doi.org/10.1016/j.drudis.2021.05.015" title="DOI URL">https://doi.org/10.1016/j.drudis.2021.05.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2021.05.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2021.05.015%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DUnravelling%252BKDM4%252Bhistone%252Bdemethylase%252Binhibitors%252Bfor%252Bcancer%252Btherapy%26aulast%3DBaby%26aufirst%3DStephin%26date%3D2021%26volume%3D21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosaria  Benedetti</span>, <span class="hlFld-ContribAuthor ">Chiara  Papulino</span>, <span class="hlFld-ContribAuthor ">Giulia  Sgueglia</span>, <span class="hlFld-ContribAuthor ">Ugo  Chianese</span>, <span class="hlFld-ContribAuthor ">Tommaso  De Marchi</span>, <span class="hlFld-ContribAuthor ">Francesco  Iovino</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Emma  Niméus</span>, <span class="hlFld-ContribAuthor ">Carmela  Dell’ Aversana</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Regulatory Interplay between miR-181a-5p and Estrogen Receptor Signaling Cascade in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2021,</strong> <em>13 </em>
                                    (3)
                                     , 543. <a href="https://doi.org/10.3390/cancers13030543" title="DOI URL">https://doi.org/10.3390/cancers13030543</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers13030543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers13030543%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DRegulatory%252BInterplay%252Bbetween%252BmiR-181a-5p%252Band%252BEstrogen%252BReceptor%252BSignaling%252BCascade%252Bin%252BBreast%252BCancer%26aulast%3DBenedetti%26aufirst%3DRosaria%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D3%26spage%3D543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anna  Sanchez</span>, <span class="hlFld-ContribAuthor ">Fatma Zohra  Houfaf Khoufaf</span>, <span class="hlFld-ContribAuthor ">Mouhamed  Idrissou</span>, <span class="hlFld-ContribAuthor ">Frédérique  Penault-Llorca</span>, <span class="hlFld-ContribAuthor ">Yves-Jean  Bignon</span>, <span class="hlFld-ContribAuthor ">Laurent  Guy</span>, <span class="hlFld-ContribAuthor ">Dominique  Bernard-Gallon</span>. </span><span class="cited-content_cbyCitation_article-title">The Functions of the Demethylase JMJD3 in Cancer. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (2)
                                     , 968. <a href="https://doi.org/10.3390/ijms22020968" title="DOI URL">https://doi.org/10.3390/ijms22020968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22020968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22020968%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DThe%252BFunctions%252Bof%252Bthe%252BDemethylase%252BJMJD3%252Bin%252BCancer%26aulast%3DSanchez%26aufirst%3DAnna%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D2%26spage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qi-Yuan  Yang</span>, <span class="hlFld-ContribAuthor ">Qian-Qian  Cao</span>, <span class="hlFld-ContribAuthor ">Yun-Liang  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiao-Fang  Xu</span>, <span class="hlFld-ContribAuthor ">Cai-Xing  Deng</span>, <span class="hlFld-ContribAuthor ">Rajesh  Kumar</span>, <span class="hlFld-ContribAuthor ">Xiao-Min  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiu-Jian  Wang</span>, <span class="hlFld-ContribAuthor ">Hong  Liang</span>, <span class="hlFld-ContribAuthor ">Zhen-Feng  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis, structural characterization and antitumor activity of six rare earth metal complexes with 8-hydroxyquinoline derivatives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Inorganic Biochemistry</span><span> <strong>2020,</strong> <em>211 </em>, 111175. <a href="https://doi.org/10.1016/j.jinorgbio.2020.111175" title="DOI URL">https://doi.org/10.1016/j.jinorgbio.2020.111175</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jinorgbio.2020.111175&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jinorgbio.2020.111175%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Inorganic%2520Biochemistry%26atitle%3DSynthesis%25252C%252Bstructural%252Bcharacterization%252Band%252Bantitumor%252Bactivity%252Bof%252Bsix%252Brare%252Bearth%252Bmetal%252Bcomplexes%252Bwith%252B8-hydroxyquinoline%252Bderivatives%26aulast%3DYang%26aufirst%3DQi-Yuan%26date%3D2020%26volume%3D211%26spage%3D111175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sarder  Arifuzzaman</span>, <span class="hlFld-ContribAuthor ">Mst Reshma  Khatun</span>, <span class="hlFld-ContribAuthor ">Rabeya  Khatun</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging of lysine demethylases (KDMs): From pathophysiological insights to novel therapeutic opportunities. </span><span class="cited-content_cbyCitation_journal-name">Biomedicine & Pharmacotherapy</span><span> <strong>2020,</strong> <em>129 </em>, 110392. <a href="https://doi.org/10.1016/j.biopha.2020.110392" title="DOI URL">https://doi.org/10.1016/j.biopha.2020.110392</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biopha.2020.110392&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biopha.2020.110392%26sid%3Dliteratum%253Aachs%26jtitle%3DBiomedicine%2520%2526%2520Pharmacotherapy%26atitle%3DEmerging%252Bof%252Blysine%252Bdemethylases%252B%252528KDMs%252529%25253A%252BFrom%252Bpathophysiological%252Binsights%252Bto%252Bnovel%252Btherapeutic%252Bopportunities%26aulast%3DArifuzzaman%26aufirst%3DSarder%26date%3D2020%26volume%3D129%26spage%3D110392" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vitor  Coutinho Carneiro</span>, <span class="hlFld-ContribAuthor ">Isabel Caetano  de Abreu da Silva</span>, <span class="hlFld-ContribAuthor ">Murilo Sena  Amaral</span>, <span class="hlFld-ContribAuthor ">Adriana S. A.  Pereira</span>, <span class="hlFld-ContribAuthor ">Gilbert Oliveira  Silveira</span>, <span class="hlFld-ContribAuthor ">David da Silva  Pires</span>, <span class="hlFld-ContribAuthor ">Sergio  Verjovski-Almeida</span>, <span class="hlFld-ContribAuthor ">Frank J.  Dekker</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Eduardo José  Lopes-Torres</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Raymond J.  Pierce</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Julien  Lancelot</span>, <span class="hlFld-ContribAuthor ">Silvana  Thiengo</span>, <span class="hlFld-ContribAuthor ">Monica Ammon  Fernandez</span>, <span class="hlFld-ContribAuthor ">Amanda Roberta Revoredo  Vicentino</span>, <span class="hlFld-ContribAuthor ">Marina Moraes  Mourão</span>, <span class="hlFld-ContribAuthor ">Fernanda Sales  Coelho</span>, <span class="hlFld-ContribAuthor ">Marcelo Rosado  Fantappié</span>, . </span><span class="cited-content_cbyCitation_article-title">Pharmacological inhibition of lysine-specific demethylase 1 (LSD1) induces global transcriptional deregulation and ultrastructural alterations that impair viability in Schistosoma mansoni. </span><span class="cited-content_cbyCitation_journal-name">PLOS Neglected Tropical Diseases</span><span> <strong>2020,</strong> <em>14 </em>
                                    (7)
                                     , e0008332. <a href="https://doi.org/10.1371/journal.pntd.0008332" title="DOI URL">https://doi.org/10.1371/journal.pntd.0008332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pntd.0008332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pntd.0008332%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520Neglected%2520Tropical%2520Diseases%26atitle%3DPharmacological%252Binhibition%252Bof%252Blysine-specific%252Bdemethylase%252B1%252B%252528LSD1%252529%252Binduces%252Bglobal%252Btranscriptional%252Bderegulation%252Band%252Bultrastructural%252Balterations%252Bthat%252Bimpair%252Bviability%252Bin%252BSchistosoma%252Bmansoni%26aulast%3DCoutinho%2BCarneiro%26aufirst%3DVitor%26date%3D2020%26date%3D2020%26volume%3D14%26issue%3D7%26spage%3De0008332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyun  Liang</span>, <span class="hlFld-ContribAuthor ">Haiwen  Wang</span>, <span class="hlFld-ContribAuthor ">Yongliang  Du</span>, <span class="hlFld-ContribAuthor ">Bingling  Luo</span>, <span class="hlFld-ContribAuthor ">Ning  Meng</span>, <span class="hlFld-ContribAuthor ">Meifeng  Cen</span>, <span class="hlFld-ContribAuthor ">Peng  Huang</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>, <span class="hlFld-ContribAuthor ">Shijun  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">New tranylcypromine derivatives containing sulfonamide motif as potent LSD1 inhibitors to target acute myeloid leukemia: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2020,</strong> <em>99 </em>, 103808. <a href="https://doi.org/10.1016/j.bioorg.2020.103808" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.103808</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.103808&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.103808%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Btranylcypromine%252Bderivatives%252Bcontaining%252Bsulfonamide%252Bmotif%252Bas%252Bpotent%252BLSD1%252Binhibitors%252Bto%252Btarget%252Bacute%252Bmyeloid%252Bleukemia%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DLiang%26aufirst%3DLiyun%26date%3D2020%26volume%3D99%26spage%3D103808" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hilal  Saraç</span>, <span class="hlFld-ContribAuthor ">Tunç  Morova</span>, <span class="hlFld-ContribAuthor ">Elisabete  Pires</span>, <span class="hlFld-ContribAuthor ">James  McCullagh</span>, <span class="hlFld-ContribAuthor ">Anıl  Kaplan</span>, <span class="hlFld-ContribAuthor ">Ahmet  Cingöz</span>, <span class="hlFld-ContribAuthor ">Tugba  Bagci-Onder</span>, <span class="hlFld-ContribAuthor ">Tamer  Önder</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, <span class="hlFld-ContribAuthor ">Nathan A.  Lack</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic characterization of chromatin modifying enzymes identifies KDM3B as a critical regulator in castration resistant prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">Oncogene</span><span> <strong>2020,</strong> <em>39 </em>
                                    (10)
                                     , 2187-2201. <a href="https://doi.org/10.1038/s41388-019-1116-8" title="DOI URL">https://doi.org/10.1038/s41388-019-1116-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41388-019-1116-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41388-019-1116-8%26sid%3Dliteratum%253Aachs%26jtitle%3DOncogene%26atitle%3DSystematic%252Bcharacterization%252Bof%252Bchromatin%252Bmodifying%252Benzymes%252Bidentifies%252BKDM3B%252Bas%252Ba%252Bcritical%252Bregulator%252Bin%252Bcastration%252Bresistant%252Bprostate%252Bcancer%26aulast%3DSara%25C3%25A7%26aufirst%3DHilal%26date%3D2020%26date%3D2019%26volume%3D39%26issue%3D10%26spage%3D2187%26epage%3D2201" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashraf F.  Fouad</span>, <span class="hlFld-ContribAuthor ">Asma A.  Khan</span>, <span class="hlFld-ContribAuthor ">Renato M.  Silva</span>, <span class="hlFld-ContribAuthor ">Mo K.  Kang</span>. </span><span class="cited-content_cbyCitation_article-title">Genetic and Epigenetic Characterization of Pulpal and Periapical Inflammation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Physiology</span><span> <strong>2020,</strong> <em>11 </em><a href="https://doi.org/10.3389/fphys.2020.00021" title="DOI URL">https://doi.org/10.3389/fphys.2020.00021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphys.2020.00021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphys.2020.00021%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Physiology%26atitle%3DGenetic%252Band%252BEpigenetic%252BCharacterization%252Bof%252BPulpal%252Band%252BPeriapical%252BInflammation%26aulast%3DFouad%26aufirst%3DAshraf%2BF.%26date%3D2020%26date%3D2020%26volume%3D11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gerhard  Jung</span>, <span class="hlFld-ContribAuthor ">Eva  Hernández-Illán</span>, <span class="hlFld-ContribAuthor ">Leticia  Moreira</span>, <span class="hlFld-ContribAuthor ">Francesc  Balaguer</span>, <span class="hlFld-ContribAuthor ">Ajay  Goel</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics of colorectal cancer: biomarker and therapeutic potential. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Gastroenterology & Hepatology</span><span> <strong>2020,</strong> <em>17 </em>
                                    (2)
                                     , 111-130. <a href="https://doi.org/10.1038/s41575-019-0230-y" title="DOI URL">https://doi.org/10.1038/s41575-019-0230-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41575-019-0230-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41575-019-0230-y%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Gastroenterology%2520%2526%2520Hepatology%26atitle%3DEpigenetics%252Bof%252Bcolorectal%252Bcancer%25253A%252Bbiomarker%252Band%252Btherapeutic%252Bpotential%26aulast%3DJung%26aufirst%3DGerhard%26date%3D2020%26date%3D2020%26volume%3D17%26issue%3D2%26spage%3D111%26epage%3D130" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: A new frontier for epi‐drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2020,</strong> <em>40 </em>
                                    (1)
                                     , 190-244. <a href="https://doi.org/10.1002/med.21600" title="DOI URL">https://doi.org/10.1002/med.21600</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21600&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21600%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252BA%252Bnew%252Bfrontier%252Bfor%252Bepi%2525E2%252580%252590drug%252Bdiscovery%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26date%3D2019%26volume%3D40%26issue%3D1%26spage%3D190%26epage%3D244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Federica  Sarno</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Histone demethylase inhibitors and their potential in cancer treatment. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 143-177. <a href="https://doi.org/10.1016/B978-0-12-816422-8.00007-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-816422-8.00007-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-816422-8.00007-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-816422-8.00007-6%26sid%3Dliteratum%253Aachs%26atitle%3DHistone%252Bdemethylase%252Binhibitors%252Band%252Btheir%252Bpotential%252Bin%252Bcancer%252Btreatment%26aulast%3DSarno%26aufirst%3DFederica%26date%3D2020%26spage%3D143%26epage%3D177%26pub%3DElsevier%26atitle%3DHistone%252BModifications%252Bin%252BTherapy%26date%3D2020%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosaria  Benedetti</span>, <span class="hlFld-ContribAuthor ">Carmela  Dell’Aversana</span>, <span class="hlFld-ContribAuthor ">Tommaso  De Marchi</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Ning Qing  Liu</span>, <span class="hlFld-ContribAuthor ">Boris  Novakovic</span>, <span class="hlFld-ContribAuthor ">Serena  Boccella</span>, <span class="hlFld-ContribAuthor ">Salvatore  Di Maro</span>, <span class="hlFld-ContribAuthor ">Sandro  Cosconati</span>, <span class="hlFld-ContribAuthor ">Alfonso  Baldi</span>, <span class="hlFld-ContribAuthor ">Emma  Niméus</span>, <span class="hlFld-ContribAuthor ">Johan  Schultz</span>, <span class="hlFld-ContribAuthor ">Urban  Höglund</span>, <span class="hlFld-ContribAuthor ">Sabatino  Maione</span>, <span class="hlFld-ContribAuthor ">Chiara  Papulino</span>, <span class="hlFld-ContribAuthor ">Ugo  Chianese</span>, <span class="hlFld-ContribAuthor ">Francesco  Iovino</span>, <span class="hlFld-ContribAuthor ">Antonio  Federico</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Hendrik G.  Stunnenberg</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibition of Histone Demethylases LSD1 and UTX Regulates ERα Signaling in Breast Cancer. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (12)
                                     , 2027. <a href="https://doi.org/10.3390/cancers11122027" title="DOI URL">https://doi.org/10.3390/cancers11122027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11122027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11122027%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DInhibition%252Bof%252BHistone%252BDemethylases%252BLSD1%252Band%252BUTX%252BRegulates%252BER%2525CE%2525B1%252BSignaling%252Bin%252BBreast%252BCancer%26aulast%3DBenedetti%26aufirst%3DRosaria%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D12%26spage%3D2027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christophe  Blanquart</span>, <span class="hlFld-ContribAuthor ">Camille  Linot</span>, <span class="hlFld-ContribAuthor ">Pierre-François  Cartron</span>, <span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Philippe  Bertrand</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Metalloenzymes. </span><span class="cited-content_cbyCitation_journal-name">Current Medicinal Chemistry</span><span> <strong>2019,</strong> <em>26 </em>
                                    (15)
                                     , 2748-2785. <a href="https://doi.org/10.2174/0929867325666180706105903" title="DOI URL">https://doi.org/10.2174/0929867325666180706105903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/0929867325666180706105903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F0929867325666180706105903%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Medicinal%2520Chemistry%26atitle%3DEpigenetic%252BMetalloenzymes%26aulast%3DBlanquart%26aufirst%3DChristophe%26date%3D2019%26volume%3D26%26issue%3D15%26spage%3D2748%26epage%3D2785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adam  Lee</span>, <span class="hlFld-ContribAuthor ">M. Teresa  Borrello</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">LSD
              (Lysine‐Specific Demethylase): A Decade‐Long Trip from Discovery to Clinical Trials. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 221-261. <a href="https://doi.org/10.1002/9783527809257.ch10" title="DOI URL">https://doi.org/10.1002/9783527809257.ch10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch10%26sid%3Dliteratum%253Aachs%26atitle%3DLSD%252B%252528Lysine%2525E2%252580%252590Specific%252BDemethylase%252529%25253A%252BA%252BDecade%2525E2%252580%252590Long%252BTrip%252Bfrom%252BDiscovery%252Bto%252BClinical%252BTrials%26aulast%3DLee%26aufirst%3DAdam%26date%3D2019%26date%3D2019%26spage%3D221%26epage%3D261%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lijun  Li</span>, <span class="hlFld-ContribAuthor ">Ruizhe  Li</span>, <span class="hlFld-ContribAuthor ">Yumei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of selective and reversible LSD1 inhibitors with anti-metastasis activity by high-throughput docking. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 544-548. <a href="https://doi.org/10.1016/j.bmcl.2018.12.067" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.067</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.067%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252Bselective%252Band%252Breversible%252BLSD1%252Binhibitors%252Bwith%252Banti-metastasis%252Bactivity%252Bby%252Bhigh-throughput%252Bdocking%26aulast%3DLi%26aufirst%3DLijun%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D544%26epage%3D548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine-Specific Histone Demethylases 1/2 (LSD1/2) and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 197-219. <a href="https://doi.org/10.1007/7355_2019_74" title="DOI URL">https://doi.org/10.1007/7355_2019_74</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_74&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_74%26sid%3Dliteratum%253Aachs%26atitle%3DLysine-Specific%252BHistone%252BDemethylases%252B1%25252F2%252B%252528LSD1%25252F2%252529%252Band%252BTheir%252BInhibitors%26aulast%3DSuzuki%26aufirst%3DTakayoshi%26date%3D2019%26date%3D2020%26spage%3D197%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aksinija A.  Kogan</span>, <span class="hlFld-ContribAuthor ">Rena G.  Lapidus</span>, <span class="hlFld-ContribAuthor ">Maria R.  Baer</span>, <span class="hlFld-ContribAuthor ">Feyruz V.  Rassool</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting epigenetically mediated changes: Acute myeloid leukemia, leukemia stem cells and the bone marrow microenvironment. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 213-253. <a href="https://doi.org/10.1016/bs.acr.2018.12.005" title="DOI URL">https://doi.org/10.1016/bs.acr.2018.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.acr.2018.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.acr.2018.12.005%26sid%3Dliteratum%253Aachs%26atitle%3DExploiting%252Bepigenetically%252Bmediated%252Bchanges%25253A%252BAcute%252Bmyeloid%252Bleukemia%25252C%252Bleukemia%252Bstem%252Bcells%252Band%252Bthe%252Bbone%252Bmarrow%252Bmicroenvironment%26aulast%3DKogan%26aufirst%3DAksinija%2BA.%26date%3D2019%26spage%3D213%26epage%3D253%26pub%3DElsevier%26atitle%3DCancer%252BStem%252BCells%26date%3D2019%26volume%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anne  Bouchut</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Christine  Pierrot</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Sophia  Lafitte</span>, <span class="hlFld-ContribAuthor ">Johan  Schultz</span>, <span class="hlFld-ContribAuthor ">Urban  Hoglund</span>, <span class="hlFld-ContribAuthor ">Roberta  Mazzone</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Giancarlo  Fabrizi</span>, <span class="hlFld-ContribAuthor ">Dany  Pechalrieu</span>, <span class="hlFld-ContribAuthor ">Paola B.  Arimondo</span>, <span class="hlFld-ContribAuthor ">Tina S.  Skinner-Adams</span>, <span class="hlFld-ContribAuthor ">Ming Jang  Chua</span>, <span class="hlFld-ContribAuthor ">Kathy T.  Andrews</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Jamal  Khalife</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel quinazoline derivatives as potent antiplasmodial agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>161 </em>, 277-291. <a href="https://doi.org/10.1016/j.ejmech.2018.10.041" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.10.041</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.10.041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.10.041%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252Bquinazoline%252Bderivatives%252Bas%252Bpotent%252Bantiplasmodial%252Bagents%26aulast%3DBouchut%26aufirst%3DAnne%26date%3D2019%26volume%3D161%26spage%3D277%26epage%3D291" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xudong  Sun</span>, <span class="hlFld-ContribAuthor ">Lina  Ding</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the binding mode and unbinding mechanism of LSD1 inhibitors by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2018,</strong> <em>20 </em>
                                    (47)
                                     , 29833-29846. <a href="https://doi.org/10.1039/C8CP03090A" title="DOI URL">https://doi.org/10.1039/C8CP03090A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8CP03090A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8CP03090A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DProbing%252Bthe%252Bbinding%252Bmode%252Band%252Bunbinding%252Bmechanism%252Bof%252BLSD1%252Binhibitors%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DSun%26aufirst%3DXudong%26date%3D2018%26date%3D2018%26volume%3D20%26issue%3D47%26spage%3D29833%26epage%3D29846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yosuke  Ota</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>. </span><span class="cited-content_cbyCitation_article-title">Drug Design Concepts for LSD1‐Selective Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1782-1791. <a href="https://doi.org/10.1002/tcr.201810031" title="DOI URL">https://doi.org/10.1002/tcr.201810031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201810031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201810031%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DDrug%252BDesign%252BConcepts%252Bfor%252BLSD1%2525E2%252580%252590Selective%252BInhibitors%26aulast%3DOta%26aufirst%3DYosuke%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1782%26epage%3D1791" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tayaba  Ismail</span>, <span class="hlFld-ContribAuthor ">Hyun-Kyung  Lee</span>, <span class="hlFld-ContribAuthor ">Chowon  Kim</span>, <span class="hlFld-ContribAuthor ">Taejoon  Kwon</span>, <span class="hlFld-ContribAuthor ">Tae Joo  Park</span>, <span class="hlFld-ContribAuthor ">Hyun-Shik  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">KDM1A microenvironment, its oncogenic potential, and therapeutic significance. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics & Chromatin</span><span> <strong>2018,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13072-018-0203-3" title="DOI URL">https://doi.org/10.1186/s13072-018-0203-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13072-018-0203-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13072-018-0203-3%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%2520%2526%2520Chromatin%26atitle%3DKDM1A%252Bmicroenvironment%25252C%252Bits%252Boncogenic%252Bpotential%25252C%252Band%252Btherapeutic%252Bsignificance%26aulast%3DIsmail%26aufirst%3DTayaba%26date%3D2018%26date%3D2018%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan‐Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Simon Wing Fai  Mok</span>, <span class="hlFld-ContribAuthor ">Chun  Wu</span>, <span class="hlFld-ContribAuthor ">Betty Yuen Kwan  Law</span>, <span class="hlFld-ContribAuthor ">Xiang‐Min  Miao</span>, <span class="hlFld-ContribAuthor ">Ke‐Jia  Wu</span>, <span class="hlFld-ContribAuthor ">Hai‐Jing  Zhong</span>, <span class="hlFld-ContribAuthor ">Chun‐Yuen  Wong</span>, <span class="hlFld-ContribAuthor ">Vincent Kam Wai  Wong</span>, <span class="hlFld-ContribAuthor ">Dik‐Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Chung‐Hang  Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Inhibition of Lysine‐Specific Demethylase 5A (KDM5A) Using a Rhodium(III) Complex for Triple‐Negative Breast Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2018,</strong> <em>130 </em>
                                    (40)
                                     , 13275-13279. <a href="https://doi.org/10.1002/ange.201807305" title="DOI URL">https://doi.org/10.1002/ange.201807305</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201807305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201807305%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DSelective%252BInhibition%252Bof%252BLysine%2525E2%252580%252590Specific%252BDemethylase%252B5A%252B%252528KDM5A%252529%252BUsing%252Ba%252BRhodium%252528III%252529%252BComplex%252Bfor%252BTriple%2525E2%252580%252590Negative%252BBreast%252BCancer%252BTherapy%26aulast%3DYang%26aufirst%3DGuan%25E2%2580%2590Jun%26date%3D2018%26date%3D2018%26volume%3D130%26issue%3D40%26spage%3D13275%26epage%3D13279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guan‐Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Simon Wing Fai  Mok</span>, <span class="hlFld-ContribAuthor ">Chun  Wu</span>, <span class="hlFld-ContribAuthor ">Betty Yuen Kwan  Law</span>, <span class="hlFld-ContribAuthor ">Xiang‐Min  Miao</span>, <span class="hlFld-ContribAuthor ">Ke‐Jia  Wu</span>, <span class="hlFld-ContribAuthor ">Hai‐Jing  Zhong</span>, <span class="hlFld-ContribAuthor ">Chun‐Yuen  Wong</span>, <span class="hlFld-ContribAuthor ">Vincent Kam Wai  Wong</span>, <span class="hlFld-ContribAuthor ">Dik‐Lung  Ma</span>, <span class="hlFld-ContribAuthor ">Chung‐Hang  Leung</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Inhibition of Lysine‐Specific Demethylase 5A (KDM5A) Using a Rhodium(III) Complex for Triple‐Negative Breast Cancer Therapy. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2018,</strong> <em>57 </em>
                                    (40)
                                     , 13091-13095. <a href="https://doi.org/10.1002/anie.201807305" title="DOI URL">https://doi.org/10.1002/anie.201807305</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201807305&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201807305%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DSelective%252BInhibition%252Bof%252BLysine%2525E2%252580%252590Specific%252BDemethylase%252B5A%252B%252528KDM5A%252529%252BUsing%252Ba%252BRhodium%252528III%252529%252BComplex%252Bfor%252BTriple%2525E2%252580%252590Negative%252BBreast%252BCancer%252BTherapy%26aulast%3DYang%26aufirst%3DGuan%25E2%2580%2590Jun%26date%3D2018%26volume%3D57%26issue%3D40%26spage%3D13091%26epage%3D13095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qi-Yuan  Yang</span>, <span class="hlFld-ContribAuthor ">Qian-Qian  Cao</span>, <span class="hlFld-ContribAuthor ">Qi-Pin  Qin</span>, <span class="hlFld-ContribAuthor ">Cai-Xing  Deng</span>, <span class="hlFld-ContribAuthor ">Hong  Liang</span>, <span class="hlFld-ContribAuthor ">Zhen-Feng  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Syntheses, Crystal Structures, and Antitumor Activities of Copper(II) and Nickel(II) Complexes with 2-((2-(Pyridin-2-yl)hydrazono)methyl)quinolin-8-ol. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2018,</strong> <em>19 </em>
                                    (7)
                                     , 1874. <a href="https://doi.org/10.3390/ijms19071874" title="DOI URL">https://doi.org/10.3390/ijms19071874</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms19071874&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms19071874%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DSyntheses%25252C%252BCrystal%252BStructures%25252C%252Band%252BAntitumor%252BActivities%252Bof%252BCopper%252528II%252529%252Band%252BNickel%252528II%252529%252BComplexes%252Bwith%252B2-%252528%2525282-%252528Pyridin-2-yl%252529hydrazono%252529methyl%252529quinolin-8-ol%26aulast%3DYang%26aufirst%3DQi-Yuan%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D7%26spage%3D1874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Petraglia</span>, <span class="hlFld-ContribAuthor ">Abhishek A.  Singh</span>, <span class="hlFld-ContribAuthor ">Vincenzo  Carafa</span>, <span class="hlFld-ContribAuthor ">Angela  Nebbioso</span>, <span class="hlFld-ContribAuthor ">Mariarosaria  Conte</span>, <span class="hlFld-ContribAuthor ">Lucia  Scisciola</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Alfonso  Baldi</span>, <span class="hlFld-ContribAuthor ">Amit  Mandoli</span>, <span class="hlFld-ContribAuthor ">Valeria Belsito  Petrizzi</span>, <span class="hlFld-ContribAuthor ">Concetta  Ingenito</span>, <span class="hlFld-ContribAuthor ">Sandro  De Falco</span>, <span class="hlFld-ContribAuthor ">Valeria  Cicatiello</span>, <span class="hlFld-ContribAuthor ">Ivana  Apicella</span>, <span class="hlFld-ContribAuthor ">Eva M.  Janssen-Megens</span>, <span class="hlFld-ContribAuthor ">Bowon  Kim</span>, <span class="hlFld-ContribAuthor ">Guoqiang  Yi</span>, <span class="hlFld-ContribAuthor ">Colin  Logie</span>, <span class="hlFld-ContribAuthor ">Simon  Heath</span>, <span class="hlFld-ContribAuthor ">Menotti  Ruvo</span>, <span class="hlFld-ContribAuthor ">Albertus T.J.  Wierenga</span>, <span class="hlFld-ContribAuthor ">Paul  Flicek</span>, <span class="hlFld-ContribAuthor ">Marie Laure  Yaspo</span>, <span class="hlFld-ContribAuthor ">Veronique  Della Valle</span>, <span class="hlFld-ContribAuthor ">Olivier  Bernard</span>, <span class="hlFld-ContribAuthor ">Stefano  Tomassi</span>, <span class="hlFld-ContribAuthor ">Ettore  Novellino</span>, <span class="hlFld-ContribAuthor ">Alessandra  Feoli</span>, <span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>, <span class="hlFld-ContribAuthor ">Ivo  Gut</span>, <span class="hlFld-ContribAuthor ">Edo  Vellenga</span>, <span class="hlFld-ContribAuthor ">Hendrik G.  Stunnenberg</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Joost H.A.  Martens</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Combined HAT/EZH2 modulation leads to cancer-selective cell death. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (39)
                                     , 25630-25646. <a href="https://doi.org/10.18632/oncotarget.25428" title="DOI URL">https://doi.org/10.18632/oncotarget.25428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.25428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.25428%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DCombined%252BHAT%25252FEZH2%252Bmodulation%252Bleads%252Bto%252Bcancer-selective%252Bcell%252Bdeath%26aulast%3DPetraglia%26aufirst%3DFrancesca%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D39%26spage%3D25630%26epage%3D25646" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oscar  Méndez-Lucio</span>, <span class="hlFld-ContribAuthor ">J. Jesús  Naveja</span>, <span class="hlFld-ContribAuthor ">Hugo  Vite-Caritino</span>, <span class="hlFld-ContribAuthor ">Fernando D.  Prieto-Martínez</span>, <span class="hlFld-ContribAuthor ">José L.  Medina-Franco</span>. </span><span class="cited-content_cbyCitation_article-title">Polypharmacology in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 1-29. <a href="https://doi.org/10.1002/9783527674381.ch1" title="DOI URL">https://doi.org/10.1002/9783527674381.ch1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527674381.ch1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527674381.ch1%26sid%3Dliteratum%253Aachs%26atitle%3DPolypharmacology%252Bin%252BDrug%252BDiscovery%26aulast%3DM%25C3%25A9ndez-Lucio%26aufirst%3DOscar%26date%3D2017%26date%3D2017%26spage%3D1%26epage%3D29%26pub%3DWiley-VCH%2520Verlag%2520GmbH%2520%2526%2520Co.%2520KGaA%26atitle%3DDrug%252BSelectivity%26aulast%3DHandler%26aufirst%3DNorbert%26date%3D2017%26volume%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insight into inhibitors of flavin adenine dinucleotide-dependent lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics</span><span> <strong>2017,</strong> <em>12 </em>
                                    (5)
                                     , 340-352. <a href="https://doi.org/10.1080/15592294.2017.1290032" title="DOI URL">https://doi.org/10.1080/15592294.2017.1290032</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/15592294.2017.1290032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F15592294.2017.1290032%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%26atitle%3DStructural%252Binsight%252Binto%252Binhibitors%252Bof%252Bflavin%252Badenine%252Bdinucleotide-dependent%252Blysine%252Bdemethylases%26aulast%3DNiwa%26aufirst%3DHideaki%26date%3D2017%26date%3D2017%26volume%3D12%26issue%3D5%26spage%3D340%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Oliveri</span>, <span class="hlFld-ContribAuthor ">Francesco  Bellia</span>, <span class="hlFld-ContribAuthor ">Graziella  Vecchio</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Isomers of Cyclodextrin-Bearing IOX1 Compound as Inhibitors of Aβ Aggregation. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2017,</strong> <em>2 </em>
                                    (2)
                                     , 655-659. <a href="https://doi.org/10.1002/slct.201601651" title="DOI URL">https://doi.org/10.1002/slct.201601651</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.201601651&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.201601651%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DStructural%252BIsomers%252Bof%252BCyclodextrin-Bearing%252BIOX1%252BCompound%252Bas%252BInhibitors%252Bof%252BA%2525CE%2525B2%252BAggregation%26aulast%3DOliveri%26aufirst%3DValentina%26date%3D2017%26date%3D2017%26volume%3D2%26issue%3D2%26spage%3D655%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kexin  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">DNA and Histone Methylation in Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 489-529. <a href="https://doi.org/10.1007/978-3-319-59786-7_18" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_18%26sid%3Dliteratum%253Aachs%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bin%252BProstate%252BCancer%26aulast%3DXu%26aufirst%3DKexin%26date%3D2017%26date%3D2017%26spage%3D489%26epage%3D529%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John R.  Horton</span>, <span class="hlFld-ContribAuthor ">Molly  Gale</span>, <span class="hlFld-ContribAuthor ">Qin  Yan</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Cheng</span>. </span><span class="cited-content_cbyCitation_article-title">The Molecular Basis of Histone Demethylation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 151-219. <a href="https://doi.org/10.1007/978-3-319-59786-7_7" title="DOI URL">https://doi.org/10.1007/978-3-319-59786-7_7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-319-59786-7_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-319-59786-7_7%26sid%3Dliteratum%253Aachs%26atitle%3DThe%252BMolecular%252BBasis%252Bof%252BHistone%252BDemethylation%26aulast%3DHorton%26aufirst%3DJohn%2BR.%26date%3D2017%26date%3D2017%26spage%3D151%26epage%3D219%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DDNA%252Band%252BHistone%252BMethylation%252Bas%252BCancer%252BTargets%26aulast%3DKaneda%26aufirst%3DAtsushi%26date%3D2017%26volume%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ying-Chao  Duan</span>, <span class="hlFld-ContribAuthor ">Yuan-Yuan  Guan</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Zhai</span>, <span class="hlFld-ContribAuthor ">Li-Na  Ding</span>, <span class="hlFld-ContribAuthor ">Wen-Ping  Qin</span>, <span class="hlFld-ContribAuthor ">Dan-Dan  Shen</span>, <span class="hlFld-ContribAuthor ">Xue-Qi  Liu</span>, <span class="hlFld-ContribAuthor ">Xu-Dong  Sun</span>, <span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of resveratrol derivatives as novel LSD1 inhibitors: Design, synthesis and their biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>126 </em>, 246-258. <a href="https://doi.org/10.1016/j.ejmech.2016.11.035" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.11.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.11.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.11.035%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bresveratrol%252Bderivatives%252Bas%252Bnovel%252BLSD1%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Btheir%252Bbiological%252Bevaluation%26aulast%3DDuan%26aufirst%3DYing-Chao%26date%3D2017%26volume%3D126%26spage%3D246%26epage%3D258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashwini  Jambhekar</span>, <span class="hlFld-ContribAuthor ">Jamie N.  Anastas</span>, <span class="hlFld-ContribAuthor ">Yang  Shi</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Lysine Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Cold Spring Harbor Perspectives in Medicine</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     , a026484. <a href="https://doi.org/10.1101/cshperspect.a026484" title="DOI URL">https://doi.org/10.1101/cshperspect.a026484</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/cshperspect.a026484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fcshperspect.a026484%26sid%3Dliteratum%253Aachs%26jtitle%3DCold%2520Spring%2520Harbor%2520Perspectives%2520in%2520Medicine%26atitle%3DHistone%252BLysine%252BDemethylase%252BInhibitors%26aulast%3DJambhekar%26aufirst%3DAshwini%26date%3D2017%26date%3D2016%26volume%3D7%26issue%3D1%26spage%3Da026484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shebli  Mehrazarin</span>, <span class="hlFld-ContribAuthor ">Abdullah  Alshaikh</span>, <span class="hlFld-ContribAuthor ">Mo K.  Kang</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Mechanisms of Apical Periodontitis. </span><span class="cited-content_cbyCitation_journal-name">Dental Clinics of North America</span><span> <strong>2017,</strong> <em>61 </em>
                                    (1)
                                     , 17-35. <a href="https://doi.org/10.1016/j.cden.2016.08.003" title="DOI URL">https://doi.org/10.1016/j.cden.2016.08.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cden.2016.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cden.2016.08.003%26sid%3Dliteratum%253Aachs%26jtitle%3DDental%2520Clinics%2520of%2520North%2520America%26atitle%3DMolecular%252BMechanisms%252Bof%252BApical%252BPeriodontitis%26aulast%3DMehrazarin%26aufirst%3DShebli%26date%3D2017%26volume%3D61%26issue%3D1%26spage%3D17%26epage%3D35" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Marabelli</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>. </span><span class="cited-content_cbyCitation_article-title">The growing structural and functional complexity of the LSD1/KDM1A histone demethylase. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2016,</strong> <em>41 </em>, 135-144. <a href="https://doi.org/10.1016/j.sbi.2016.07.011" title="DOI URL">https://doi.org/10.1016/j.sbi.2016.07.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2016.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2016.07.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DThe%252Bgrowing%252Bstructural%252Band%252Bfunctional%252Bcomplexity%252Bof%252Bthe%252BLSD1%25252FKDM1A%252Bhistone%252Bdemethylase%26aulast%3DMarabelli%26aufirst%3DChiara%26date%3D2016%26volume%3D41%26spage%3D135%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emily L.  Ricq</span>, <span class="hlFld-ContribAuthor ">Jacob M.  Hooker</span>, <span class="hlFld-ContribAuthor ">Stephen J.  Haggarty</span>. </span><span class="cited-content_cbyCitation_article-title">Toward development of epigenetic drugs for central nervous system disorders: Modulating neuroplasticity
              via
              H3K4 methylation. </span><span class="cited-content_cbyCitation_journal-name">Psychiatry and Clinical Neurosciences</span><span> <strong>2016,</strong> <em>70 </em>
                                    (12)
                                     , 536-550. <a href="https://doi.org/10.1111/pcn.12426" title="DOI URL">https://doi.org/10.1111/pcn.12426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/pcn.12426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fpcn.12426%26sid%3Dliteratum%253Aachs%26jtitle%3DPsychiatry%2520and%2520Clinical%2520Neurosciences%26atitle%3DToward%252Bdevelopment%252Bof%252Bepigenetic%252Bdrugs%252Bfor%252Bcentral%252Bnervous%252Bsystem%252Bdisorders%25253A%252BModulating%252Bneuroplasticity%252Bvia%252BH3K4%252Bmethylation%26aulast%3DRicq%26aufirst%3DEmily%2BL.%26date%3D2016%26date%3D2016%26volume%3D70%26issue%3D12%26spage%3D536%26epage%3D550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Inês  Graça</span>, <span class="hlFld-ContribAuthor ">Eva  Pereira-Silva</span>, <span class="hlFld-ContribAuthor ">Rui  Henrique</span>, <span class="hlFld-ContribAuthor ">Graham  Packham</span>, <span class="hlFld-ContribAuthor ">Simon J.  Crabb</span>, <span class="hlFld-ContribAuthor ">Carmen  Jerónimo</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic modulators as therapeutic targets in prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0264-8" title="DOI URL">https://doi.org/10.1186/s13148-016-0264-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0264-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0264-8%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DEpigenetic%252Bmodulators%252Bas%252Btherapeutic%252Btargets%252Bin%252Bprostate%252Bcancer%26aulast%3DGra%25C3%25A7a%26aufirst%3DIn%25C3%25AAs%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Angel R.  de Lera</span>, <span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic polypharmacology: from combination therapy to multitargeted drugs. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-016-0271-9" title="DOI URL">https://doi.org/10.1186/s13148-016-0271-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-016-0271-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-016-0271-9%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DEpigenetic%252Bpolypharmacology%25253A%252Bfrom%252Bcombination%252Btherapy%252Bto%252Bmultitargeted%252Bdrugs%26aulast%3Dde%2BLera%26aufirst%3DAngel%2BR.%26date%3D2016%26date%3D2016%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiang  Yuan</span>, <span class="hlFld-ContribAuthor ">Jinyu  Kong</span>, <span class="hlFld-ContribAuthor ">Zhikun  Ma</span>, <span class="hlFld-ContribAuthor ">Na  Li</span>, <span class="hlFld-ContribAuthor ">Ruinuo  Jia</span>, <span class="hlFld-ContribAuthor ">Yiwen  Liu</span>, <span class="hlFld-ContribAuthor ">Fuyou  Zhou</span>, <span class="hlFld-ContribAuthor ">Qimin  Zhan</span>, <span class="hlFld-ContribAuthor ">Gang  Liu</span>, <span class="hlFld-ContribAuthor ">Shegan  Gao</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4C, a H3K9me3 Histone Demethylase, is Involved in the Maintenance of Human ESCC-Initiating Cells by Epigenetically Enhancing SOX2 Expression. </span><span class="cited-content_cbyCitation_journal-name">Neoplasia</span><span> <strong>2016,</strong> <em>18 </em>
                                    (10)
                                     , 594-609. <a href="https://doi.org/10.1016/j.neo.2016.08.005" title="DOI URL">https://doi.org/10.1016/j.neo.2016.08.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.neo.2016.08.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.neo.2016.08.005%26sid%3Dliteratum%253Aachs%26jtitle%3DNeoplasia%26atitle%3DKDM4C%25252C%252Ba%252BH3K9me3%252BHistone%252BDemethylase%25252C%252Bis%252BInvolved%252Bin%252Bthe%252BMaintenance%252Bof%252BHuman%252BESCC-Initiating%252BCells%252Bby%252BEpigenetically%252BEnhancing%252BSOX2%252BExpression%26aulast%3DYuan%26aufirst%3DXiang%26date%3D2016%26volume%3D18%26issue%3D10%26spage%3D594%26epage%3D609" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ludovica  Morera</span>, <span class="hlFld-ContribAuthor ">Martin  Roatsch</span>, <span class="hlFld-ContribAuthor ">Michael C. D.  Fürst</span>, <span class="hlFld-ContribAuthor ">Inga  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Johanna  Senger</span>, <span class="hlFld-ContribAuthor ">Mirjam  Hau</span>, <span class="hlFld-ContribAuthor ">Henriette  Franz</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Markus R.  Heinrich</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">4-Biphenylalanine- and 3-Phenyltyrosine-Derived Hydroxamic Acids as Inhibitors of the JumonjiC-Domain-Containing Histone Demethylase KDM4A. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (18)
                                     , 2063-2083. <a href="https://doi.org/10.1002/cmdc.201600218" title="DOI URL">https://doi.org/10.1002/cmdc.201600218</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201600218&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201600218%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3D4-Biphenylalanine-%252Band%252B3-Phenyltyrosine-Derived%252BHydroxamic%252BAcids%252Bas%252BInhibitors%252Bof%252Bthe%252BJumonjiC-Domain-Containing%252BHistone%252BDemethylase%252BKDM4A%26aulast%3DMorera%26aufirst%3DLudovica%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D18%26spage%3D2063%26epage%3D2083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Oliveri</span>, <span class="hlFld-ContribAuthor ">Graziella  Vecchio</span>. </span><span class="cited-content_cbyCitation_article-title">8-Hydroxyquinolines in medicinal chemistry: A structural perspective. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>120 </em>, 252-274. <a href="https://doi.org/10.1016/j.ejmech.2016.05.007" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.05.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.05.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.05.007%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3D8-Hydroxyquinolines%252Bin%252Bmedicinal%252Bchemistry%25253A%252BA%252Bstructural%252Bperspective%26aulast%3DOliveri%26aufirst%3DValentina%26date%3D2016%26volume%3D120%26spage%3D252%26epage%3D274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martin  Roatsch</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Martin  Pippel</span>, <span class="hlFld-ContribAuthor ">Joanne E  Nettleship</span>, <span class="hlFld-ContribAuthor ">Yamini  Reddivari</span>, <span class="hlFld-ContribAuthor ">Louise E  Bird</span>, <span class="hlFld-ContribAuthor ">Inga  Hoffmann</span>, <span class="hlFld-ContribAuthor ">Henriette  Franz</span>, <span class="hlFld-ContribAuthor ">Raymond J  Owens</span>, <span class="hlFld-ContribAuthor ">Roland  Schüle</span>, <span class="hlFld-ContribAuthor ">Ralf  Flaig</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Substituted 2-(2-aminopyrimidin-4-yl)pyridine-4-carboxylates as potent inhibitors of JumonjiC domain-containing histone demethylases. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (13)
                                     , 1553-1571. <a href="https://doi.org/10.4155/fmc.15.188" title="DOI URL">https://doi.org/10.4155/fmc.15.188</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.188&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.188%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSubstituted%252B2-%2525282-aminopyrimidin-4-yl%252529pyridine-4-carboxylates%252Bas%252Bpotent%252Binhibitors%252Bof%252BJumonjiC%252Bdomain-containing%252Bhistone%252Bdemethylases%26aulast%3DRoatsch%26aufirst%3DMartin%26date%3D2016%26volume%3D8%26issue%3D13%26spage%3D1553%26epage%3D1571" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">A.  Ganesan</span>. </span><span class="cited-content_cbyCitation_article-title">Multitarget Drugs: an Epigenetic Epiphany. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2016,</strong> <em>11 </em>
                                    (12)
                                     , 1227-1241. <a href="https://doi.org/10.1002/cmdc.201500394" title="DOI URL">https://doi.org/10.1002/cmdc.201500394</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201500394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201500394%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DMultitarget%252BDrugs%25253A%252Ban%252BEpigenetic%252BEpiphany%26aulast%3DGanesan%26aufirst%3DA.%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D12%26spage%3D1227%26epage%3D1241" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryan  Lillico</span>, <span class="hlFld-ContribAuthor ">Nicholas  Stesco</span>, <span class="hlFld-ContribAuthor ">Tina  Khorshid Amhad</span>, <span class="hlFld-ContribAuthor ">Claudia  Cortes</span>, <span class="hlFld-ContribAuthor ">Mike P  Namaka</span>, <span class="hlFld-ContribAuthor ">Ted M  Lakowski</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitors of enzymes catalyzing modifications to histone lysine residues: structure, function and activity. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2016,</strong> <em>8 </em>
                                    (8)
                                     , 879-897. <a href="https://doi.org/10.4155/fmc-2016-0021" title="DOI URL">https://doi.org/10.4155/fmc-2016-0021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc-2016-0021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc-2016-0021%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DInhibitors%252Bof%252Benzymes%252Bcatalyzing%252Bmodifications%252Bto%252Bhistone%252Blysine%252Bresidues%25253A%252Bstructure%25252C%252Bfunction%252Band%252Bactivity%26aulast%3DLillico%26aufirst%3DRyan%26date%3D2016%26volume%3D8%26issue%3D8%26spage%3D879%26epage%3D897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">LING-LING  KONG</span>, <span class="hlFld-ContribAuthor ">DONG-MEI  MAN</span>, <span class="hlFld-ContribAuthor ">TIAN  WANG</span>, <span class="hlFld-ContribAuthor ">GUO-AN  ZHANG</span>, <span class="hlFld-ContribAuthor ">WEN  CUI</span>. </span><span class="cited-content_cbyCitation_article-title">Downregulation of LSD1 suppresses the proliferation, tumorigenicity and invasion of papillary thyroid carcinoma K1 cells. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2016,</strong> <em>11 </em>
                                    (4)
                                     , 2475-2480. <a href="https://doi.org/10.3892/ol.2016.4244" title="DOI URL">https://doi.org/10.3892/ol.2016.4244</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2016.4244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2016.4244%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DDownregulation%252Bof%252BLSD1%252Bsuppresses%252Bthe%252Bproliferation%25252C%252Btumorigenicity%252Band%252Binvasion%252Bof%252Bpapillary%252Bthyroid%252Bcarcinoma%252BK1%252Bcells%26aulast%3DKONG%26aufirst%3DLING-LING%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D4%26spage%3D2475%26epage%3D2480" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gervais  Bérubé</span>. </span><span class="cited-content_cbyCitation_article-title">An overview of molecular hybrids in drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2016,</strong> <em>11 </em>
                                    (3)
                                     , 281-305. <a href="https://doi.org/10.1517/17460441.2016.1135125" title="DOI URL">https://doi.org/10.1517/17460441.2016.1135125</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/17460441.2016.1135125&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F17460441.2016.1135125%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DAn%252Boverview%252Bof%252Bmolecular%252Bhybrids%252Bin%252Bdrug%252Bdiscovery%26aulast%3DB%25C3%25A9rub%25C3%25A9%26aufirst%3DGervais%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D3%26spage%3D281%26epage%3D305" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>, <span class="hlFld-ContribAuthor ">Adi F  Gazdar</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibiting the Jumonji family: a potential new clinical approach to targeting aberrant epigenetic mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2016,</strong> <em>8 </em>
                                    (3)
                                     , 313-316. <a href="https://doi.org/10.2217/epi.15.115" title="DOI URL">https://doi.org/10.2217/epi.15.115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.115%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DInhibiting%252Bthe%252BJumonji%252Bfamily%25253A%252Ba%252Bpotential%252Bnew%252Bclinical%252Bapproach%252Bto%252Btargeting%252Baberrant%252Bepigenetic%252Bmechanisms%26aulast%3DMartinez%26aufirst%3DElisabeth%2BD%26date%3D2016%26volume%3D8%26issue%3D3%26spage%3D313%26epage%3D316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wilian A.  Cortopassi</span>, <span class="hlFld-ContribAuthor ">Robert  Simion</span>, <span class="hlFld-ContribAuthor ">Charles E.  Honsby</span>, <span class="hlFld-ContribAuthor ">Tanos C. C.  França</span>, <span class="hlFld-ContribAuthor ">Robert S.  Paton</span>. </span><span class="cited-content_cbyCitation_article-title">Dioxygen Binding in the Active Site of Histone Demethylase JMJD2A and the Role of the Protein Environment. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2015,</strong> <em>21 </em>
                                    (52)
                                     , 18983-18992. <a href="https://doi.org/10.1002/chem.201502983" title="DOI URL">https://doi.org/10.1002/chem.201502983</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201502983&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201502983%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DDioxygen%252BBinding%252Bin%252Bthe%252BActive%252BSite%252Bof%252BHistone%252BDemethylase%252BJMJD2A%252Band%252Bthe%252BRole%252Bof%252Bthe%252BProtein%252BEnvironment%26aulast%3DCortopassi%26aufirst%3DWilian%2BA.%26date%3D2015%26date%3D2015%26volume%3D21%26issue%3D52%26spage%3D18983%26epage%3D18992" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao  Huang</span>, <span class="hlFld-ContribAuthor ">Bin  Wen</span>. </span><span class="cited-content_cbyCitation_article-title">“Identification Card”: Sites on Histone Modification of Cancer Cell. </span><span class="cited-content_cbyCitation_journal-name">Chinese Medical Sciences Journal</span><span> <strong>2015,</strong> <em>30 </em>
                                    (4)
                                     , 203-209. <a href="https://doi.org/10.1016/S1001-9294(16)30001-3" title="DOI URL">https://doi.org/10.1016/S1001-9294(16)30001-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/S1001-9294(16)30001-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FS1001-9294%2816%2930001-3%26sid%3Dliteratum%253Aachs%26jtitle%3DChinese%2520Medical%2520Sciences%2520Journal%26atitle%3D%2525E2%252580%25259CIdentification%252BCard%2525E2%252580%25259D%25253A%252BSites%252Bon%252BHistone%252BModification%252Bof%252BCancer%252BCell%26aulast%3DHuang%26aufirst%3DChao%26date%3D2015%26volume%3D30%26issue%3D4%26spage%3D203%26epage%3D209" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosaria  Benedetti</span>, <span class="hlFld-ContribAuthor ">Mariarosaria  Conte</span>, <span class="hlFld-ContribAuthor ">Concetta  Iside</span>, <span class="hlFld-ContribAuthor ">Lucia  Altucci</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic-based therapy: From single- to multi-target approaches. </span><span class="cited-content_cbyCitation_journal-name">The International Journal of Biochemistry & Cell Biology</span><span> <strong>2015,</strong> <em>69 </em>, 121-131. <a href="https://doi.org/10.1016/j.biocel.2015.10.016" title="DOI URL">https://doi.org/10.1016/j.biocel.2015.10.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biocel.2015.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biocel.2015.10.016%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520International%2520Journal%2520of%2520Biochemistry%2520%2526%2520Cell%2520Biology%26atitle%3DEpigenetic-based%252Btherapy%25253A%252BFrom%252Bsingle-%252Bto%252Bmulti-target%252Bapproaches%26aulast%3DBenedetti%26aufirst%3DRosaria%26date%3D2015%26volume%3D69%26spage%3D121%26epage%3D131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xueshun  Wang</span>, <span class="hlFld-ContribAuthor ">Boshi  Huang</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Suzuki</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Medicinal chemistry insights in the discovery of novel LSD1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (8)
                                     , 1379-1396. <a href="https://doi.org/10.2217/epi.15.86" title="DOI URL">https://doi.org/10.2217/epi.15.86</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.86&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.86%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DMedicinal%252Bchemistry%252Binsights%252Bin%252Bthe%252Bdiscovery%252Bof%252Bnovel%252BLSD1%252Binhibitors%26aulast%3DWang%26aufirst%3DXueshun%26date%3D2015%26volume%3D7%26issue%3D8%26spage%3D1379%26epage%3D1396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Natalia J.  Martinez</span>, <span class="hlFld-ContribAuthor ">Anton  Simeonov</span>. </span><span class="cited-content_cbyCitation_article-title">Cell-based assays to support the profiling of small molecules with histone methyltransferase and demethylase modulatory activity. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today: Technologies</span><span> <strong>2015,</strong> <em>18 </em>, 9-17. <a href="https://doi.org/10.1016/j.ddtec.2015.10.004" title="DOI URL">https://doi.org/10.1016/j.ddtec.2015.10.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ddtec.2015.10.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ddtec.2015.10.004%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technologies%26atitle%3DCell-based%252Bassays%252Bto%252Bsupport%252Bthe%252Bprofiling%252Bof%252Bsmall%252Bmolecules%252Bwith%252Bhistone%252Bmethyltransferase%252Band%252Bdemethylase%252Bmodulatory%252Bactivity%26aulast%3DMartinez%26aufirst%3DNatalia%2BJ.%26date%3D2015%26volume%3D18%26spage%3D9%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jana S.  Burchfield</span>, <span class="hlFld-ContribAuthor ">Qingtian  Li</span>, <span class="hlFld-ContribAuthor ">Helen Y.  Wang</span>, <span class="hlFld-ContribAuthor ">Rong-Fu  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">JMJD3 as an epigenetic regulator in development and disease. </span><span class="cited-content_cbyCitation_journal-name">The International Journal of Biochemistry & Cell Biology</span><span> <strong>2015,</strong> <em>67 </em>, 148-157. <a href="https://doi.org/10.1016/j.biocel.2015.07.006" title="DOI URL">https://doi.org/10.1016/j.biocel.2015.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.biocel.2015.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.biocel.2015.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520International%2520Journal%2520of%2520Biochemistry%2520%2526%2520Cell%2520Biology%26atitle%3DJMJD3%252Bas%252Ban%252Bepigenetic%252Bregulator%252Bin%252Bdevelopment%252Band%252Bdisease%26aulast%3DBurchfield%26aufirst%3DJana%2BS.%26date%3D2015%26volume%3D67%26spage%3D148%26epage%3D157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Bayo</span>, <span class="hlFld-ContribAuthor ">Maithili P  Dalvi</span>, <span class="hlFld-ContribAuthor ">Elisabeth D  Martinez</span>. </span><span class="cited-content_cbyCitation_article-title">Successful strategies in the discovery of small-molecule epigenetic modulators with anticancer potential. </span><span class="cited-content_cbyCitation_journal-name">Future Medicinal Chemistry</span><span> <strong>2015,</strong> <em>7 </em>
                                    (16)
                                     , 2243-2261. <a href="https://doi.org/10.4155/fmc.15.140" title="DOI URL">https://doi.org/10.4155/fmc.15.140</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4155/fmc.15.140&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4155%2Ffmc.15.140%26sid%3Dliteratum%253Aachs%26jtitle%3DFuture%2520Medicinal%2520Chemistry%26atitle%3DSuccessful%252Bstrategies%252Bin%252Bthe%252Bdiscovery%252Bof%252Bsmall-molecule%252Bepigenetic%252Bmodulators%252Bwith%252Banticancer%252Bpotential%26aulast%3DBayo%26aufirst%3DJuan%26date%3D2015%26volume%3D7%26issue%3D16%26spage%3D2243%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yi-Chao  Zheng</span>, <span class="hlFld-ContribAuthor ">Jinlian  Ma</span>, <span class="hlFld-ContribAuthor ">Zhiru  Wang</span>, <span class="hlFld-ContribAuthor ">Jinfeng  Li</span>, <span class="hlFld-ContribAuthor ">Bailing  Jiang</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiaojing  Shi</span>, <span class="hlFld-ContribAuthor ">Xixin  Wang</span>, <span class="hlFld-ContribAuthor ">Wen  Zhao</span>, <span class="hlFld-ContribAuthor ">Hong-Min  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">A Systematic Review of Histone Lysine-Specific Demethylase 1 and Its Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2015,</strong> <em>35 </em>
                                    (5)
                                     , 1032-1071. <a href="https://doi.org/10.1002/med.21350" title="DOI URL">https://doi.org/10.1002/med.21350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21350%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DA%252BSystematic%252BReview%252Bof%252BHistone%252BLysine-Specific%252BDemethylase%252B1%252Band%252BIts%252BInhibitors%26aulast%3DZheng%26aufirst%3DYi-Chao%26date%3D2015%26date%3D2015%26volume%3D35%26issue%3D5%26spage%3D1032%26epage%3D1071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tamara  Maes</span>, <span class="hlFld-ContribAuthor ">Elena  Carceller</span>, <span class="hlFld-ContribAuthor ">Jordi  Salas</span>, <span class="hlFld-ContribAuthor ">Alberto  Ortega</span>, <span class="hlFld-ContribAuthor ">Carlos  Buesa</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in the development of histone lysine demethylase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Pharmacology</span><span> <strong>2015,</strong> <em>23 </em>, 52-60. <a href="https://doi.org/10.1016/j.coph.2015.05.009" title="DOI URL">https://doi.org/10.1016/j.coph.2015.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.coph.2015.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.coph.2015.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Pharmacology%26atitle%3DAdvances%252Bin%252Bthe%252Bdevelopment%252Bof%252Bhistone%252Blysine%252Bdemethylase%252Binhibitors%26aulast%3DMaes%26aufirst%3DTamara%26date%3D2015%26volume%3D23%26spage%3D52%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rebecca L  Hancock</span>, <span class="hlFld-ContribAuthor ">Kate  Dunne</span>, <span class="hlFld-ContribAuthor ">Louise J  Walport</span>, <span class="hlFld-ContribAuthor ">Emily  Flashman</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic regulation by histone demethylases in hypoxia. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2015,</strong> <em>7 </em>
                                    (5)
                                     , 791-811. <a href="https://doi.org/10.2217/epi.15.24" title="DOI URL">https://doi.org/10.2217/epi.15.24</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi.15.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi.15.24%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DEpigenetic%252Bregulation%252Bby%252Bhistone%252Bdemethylases%252Bin%252Bhypoxia%26aulast%3DHancock%26aufirst%3DRebecca%2BL%26date%3D2015%26volume%3D7%26issue%3D5%26spage%3D791%26epage%3D811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jonathan M.  Burg</span>, <span class="hlFld-ContribAuthor ">Jennifer E.  Link</span>, <span class="hlFld-ContribAuthor ">Brittany S.  Morgan</span>, <span class="hlFld-ContribAuthor ">Frederick J.  Heller</span>, <span class="hlFld-ContribAuthor ">Amanda E.  Hargrove</span>, <span class="hlFld-ContribAuthor ">Dewey G.  McCafferty</span>. </span><span class="cited-content_cbyCitation_article-title">KDM1 class flavin-dependent protein lysine demethylases. </span><span class="cited-content_cbyCitation_journal-name">Biopolymers</span><span> <strong>2015,</strong> <em>104 </em>
                                    (4)
                                     , 213-246. <a href="https://doi.org/10.1002/bip.22643" title="DOI URL">https://doi.org/10.1002/bip.22643</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bip.22643&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbip.22643%26sid%3Dliteratum%253Aachs%26jtitle%3DBiopolymers%26atitle%3DKDM1%252Bclass%252Bflavin-dependent%252Bprotein%252Blysine%252Bdemethylases%26aulast%3DBurg%26aufirst%3DJonathan%2BM.%26date%3D2015%26date%3D2015%26volume%3D104%26issue%3D4%26spage%3D213%26epage%3D246" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ke  Liu</span>, <span class="hlFld-ContribAuthor ">Yanli  Liu</span>, <span class="hlFld-ContribAuthor ">Johnathan L.  Lau</span>, <span class="hlFld-ContribAuthor ">Jinrong  Min</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic targets and drug discovery Part 2: Histone demethylation and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2015,</strong> <em>151 </em>, 121-140. <a href="https://doi.org/10.1016/j.pharmthera.2015.04.001" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2015.04.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2015.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2015.04.001%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DEpigenetic%252Btargets%252Band%252Bdrug%252Bdiscovery%252BPart%252B2%25253A%252BHistone%252Bdemethylation%252Band%252BDNA%252Bmethylation%26aulast%3DLiu%26aufirst%3DKe%26date%3D2015%26volume%3D151%26spage%3D121%26epage%3D140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simone  de Brot</span>, <span class="hlFld-ContribAuthor ">Atara  Ntekim</span>, <span class="hlFld-ContribAuthor ">Ryan  Cardenas</span>, <span class="hlFld-ContribAuthor ">Victoria  James</span>, <span class="hlFld-ContribAuthor ">Cinzia  Allegrucci</span>, <span class="hlFld-ContribAuthor ">David M  Heery</span>, <span class="hlFld-ContribAuthor ">David O  Bates</span>, <span class="hlFld-ContribAuthor ">Niels  Ødum</span>, <span class="hlFld-ContribAuthor ">Jenny L  Persson</span>, <span class="hlFld-ContribAuthor ">Nigel P  Mongan</span>. </span><span class="cited-content_cbyCitation_article-title">Regulation of vascular endothelial growth factor in prostate cancer. </span><span class="cited-content_cbyCitation_journal-name">Endocrine-Related Cancer</span><span> <strong>2015,</strong> <em>22 </em>
                                    (3)
                                     , R107-R123. <a href="https://doi.org/10.1530/ERC-15-0123" title="DOI URL">https://doi.org/10.1530/ERC-15-0123</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1530/ERC-15-0123&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1530%2FERC-15-0123%26sid%3Dliteratum%253Aachs%26jtitle%3DEndocrine-Related%2520Cancer%26atitle%3DRegulation%252Bof%252Bvascular%252Bendothelial%252Bgrowth%252Bfactor%252Bin%252Bprostate%252Bcancer%26aulast%3Dde%2BBrot%26aufirst%3DSimone%26date%3D2015%26volume%3D22%26issue%3D3%26spage%3DR107%26epage%3DR123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Veronica  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Bruna  Saponara</span>, <span class="hlFld-ContribAuthor ">Roberto  Cirilli</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Donatella  Labella</span>, <span class="hlFld-ContribAuthor ">Biagina  Marrocco</span>, <span class="hlFld-ContribAuthor ">Daria  Monaldi</span>, <span class="hlFld-ContribAuthor ">Giovanni  Ruoppolo</span>, <span class="hlFld-ContribAuthor ">Mats  Tilset</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Pure enantiomers of benzoylamino-tranylcypromine: LSD1 inhibition, gene modulation in human leukemia cells and effects on clonogenic potential of murine promyelocytic blasts. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2015,</strong> <em>94 </em>, 163-174. <a href="https://doi.org/10.1016/j.ejmech.2015.02.060" title="DOI URL">https://doi.org/10.1016/j.ejmech.2015.02.060</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2015.02.060&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2015.02.060%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPure%252Benantiomers%252Bof%252Bbenzoylamino-tranylcypromine%25253A%252BLSD1%252Binhibition%25252C%252Bgene%252Bmodulation%252Bin%252Bhuman%252Bleukemia%252Bcells%252Band%252Beffects%252Bon%252Bclonogenic%252Bpotential%252Bof%252Bmurine%252Bpromyelocytic%252Bblasts%26aulast%3DValente%26aufirst%3DSergio%26date%3D2015%26volume%3D94%26spage%3D163%26epage%3D174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Young-Won  Chin</span>, <span class="hlFld-ContribAuthor ">Sun-Young  Han</span>. </span><span class="cited-content_cbyCitation_article-title">KDM4 histone demethylase inhibitors for anti-cancer agents: a patent review. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2015,</strong> <em>25 </em>
                                    (2)
                                     , 135-144. <a href="https://doi.org/10.1517/13543776.2014.991310" title="DOI URL">https://doi.org/10.1517/13543776.2014.991310</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1517/13543776.2014.991310&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1517%2F13543776.2014.991310%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DKDM4%252Bhistone%252Bdemethylase%252Binhibitors%252Bfor%252Banti-cancer%252Bagents%25253A%252Ba%252Bpatent%252Breview%26aulast%3DChin%26aufirst%3DYoung-Won%26date%3D2015%26date%3D2014%26volume%3D25%26issue%3D2%26spage%3D135%26epage%3D144" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander-Thomas  Hauser</span>, <span class="hlFld-ContribAuthor ">Martin  Roatsch</span>, <span class="hlFld-ContribAuthor ">Johannes  Schulz-Fincke</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Histone Demethylase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 397-424. <a href="https://doi.org/10.1016/B978-0-12-801080-8.00018-1" title="DOI URL">https://doi.org/10.1016/B978-0-12-801080-8.00018-1</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801080-8.00018-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801080-8.00018-1%26sid%3Dliteratum%253Aachs%26atitle%3DDiscovery%252Bof%252BHistone%252BDemethylase%252BInhibitors%26aulast%3DHauser%26aufirst%3DAlexander-Thomas%26date%3D2015%26spage%3D397%26epage%3D424%26pub%3DElsevier%26atitle%3DEpigenetic%252BTechnological%252BApplications%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu'ning  Song</span>, <span class="hlFld-ContribAuthor ">Hao  Xu</span>, <span class="hlFld-ContribAuthor ">Wenmin  Chen</span>, <span class="hlFld-ContribAuthor ">Peng  Zhan</span>, <span class="hlFld-ContribAuthor ">Xinyong  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">8-Hydroxyquinoline: a privileged structure with a broad-ranging pharmacological potential. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (1)
                                     , 61-74. <a href="https://doi.org/10.1039/C4MD00284A" title="DOI URL">https://doi.org/10.1039/C4MD00284A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00284A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00284A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3D8-Hydroxyquinoline%25253A%252Ba%252Bprivileged%252Bstructure%252Bwith%252Ba%252Bbroad-ranging%252Bpharmacological%252Bpotential%26aulast%3DSong%26aufirst%3DYu%2527ning%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D1%26spage%3D61%26epage%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Rodriguez</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Stefano  Rovida</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Alessia  Lucidi</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Ciossani</span>, <span class="hlFld-ContribAuthor ">Andrea  Mattevi</span>, <span class="hlFld-ContribAuthor ">Oronza A.  Botrugno</span>, <span class="hlFld-ContribAuthor ">Paola  Dessanti</span>, <span class="hlFld-ContribAuthor ">Ciro  Mercurio</span>, <span class="hlFld-ContribAuthor ">Paola  Vianello</span>, <span class="hlFld-ContribAuthor ">Saverio  Minucci</span>, <span class="hlFld-ContribAuthor ">Mario  Varasi</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Pyrrole- and indole-containing tranylcypromine derivatives as novel lysine-specific demethylase 1 inhibitors active on cancer cells. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2015,</strong> <em>6 </em>
                                    (4)
                                     , 665-670. <a href="https://doi.org/10.1039/C4MD00507D" title="DOI URL">https://doi.org/10.1039/C4MD00507D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00507D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00507D%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DPyrrole-%252Band%252Bindole-containing%252Btranylcypromine%252Bderivatives%252Bas%252Bnovel%252Blysine-specific%252Bdemethylase%252B1%252Binhibitors%252Bactive%252Bon%252Bcancer%252Bcells%26aulast%3DRodriguez%26aufirst%3DVeronica%26date%3D2015%26date%3D2015%26volume%3D6%26issue%3D4%26spage%3D665%26epage%3D670" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Santanu  Paul</span>, <span class="hlFld-ContribAuthor ">Satish  Ramalingam</span>, <span class="hlFld-ContribAuthor ">Dharmalingam  Subramaniam</span>, <span class="hlFld-ContribAuthor ">Joaquina  Baranda</span>, <span class="hlFld-ContribAuthor ">Shrikant  Anant</span>, <span class="hlFld-ContribAuthor ">Animesh  Dhar</span>. </span><span class="cited-content_cbyCitation_article-title">Histone Demethylases in Colon Cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Colorectal Cancer Reports</span><span> <strong>2014,</strong> <em>10 </em>
                                    (4)
                                     , 417-424. <a href="https://doi.org/10.1007/s11888-014-0253-2" title="DOI URL">https://doi.org/10.1007/s11888-014-0253-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11888-014-0253-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11888-014-0253-2%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Colorectal%2520Cancer%2520Reports%26atitle%3DHistone%252BDemethylases%252Bin%252BColon%252BCancer%26aulast%3DPaul%26aufirst%3DSantanu%26date%3D2014%26date%3D2014%26volume%3D10%26issue%3D4%26spage%3D417%26epage%3D424" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Cyrille C.  Thinnes</span>, <span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, <span class="hlFld-ContribAuthor ">Rasheduzzaman  Chowdhury</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>, <span class="hlFld-ContribAuthor ">Richard J.  Hopkinson</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting histone lysine demethylases — Progress, challenges, and the future. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2014,</strong> <em>1839 </em>
                                    (12)
                                     , 1416-1432. <a href="https://doi.org/10.1016/j.bbagrm.2014.05.009" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2014.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2014.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2014.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DTargeting%252Bhistone%252Blysine%252Bdemethylases%252B%2525E2%252580%252594%252BProgress%25252C%252Bchallenges%25252C%252Band%252Bthe%252Bfuture%26aulast%3DThinnes%26aufirst%3DCyrille%2BC.%26date%3D2014%26volume%3D1839%26issue%3D12%26spage%3D1416%26epage%3D1432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dashyant  Dhanak</span>, <span class="hlFld-ContribAuthor ">Paul  Jackson</span>. </span><span class="cited-content_cbyCitation_article-title">Development and classes of epigenetic drugs for cancer. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2014,</strong> <em>455 </em>
                                    (1-2)
                                     , 58-69. <a href="https://doi.org/10.1016/j.bbrc.2014.07.006" title="DOI URL">https://doi.org/10.1016/j.bbrc.2014.07.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2014.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2014.07.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DDevelopment%252Band%252Bclasses%252Bof%252Bepigenetic%252Bdrugs%252Bfor%252Bcancer%26aulast%3DDhanak%26aufirst%3DDashyant%26date%3D2014%26volume%3D455%26issue%3D1-2%26spage%3D58%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katherine S.  England</span>, <span class="hlFld-ContribAuthor ">Anthony  Tumber</span>, <span class="hlFld-ContribAuthor ">Tobias  Krojer</span>, <span class="hlFld-ContribAuthor ">Giuseppe  Scozzafava</span>, <span class="hlFld-ContribAuthor ">Stanley S.  Ng</span>, <span class="hlFld-ContribAuthor ">Michelle  Daniel</span>, <span class="hlFld-ContribAuthor ">Aleksandra  Szykowska</span>, <span class="hlFld-ContribAuthor ">KaHing  Che</span>, <span class="hlFld-ContribAuthor ">Frank  von Delft</span>, <span class="hlFld-ContribAuthor ">Nicola A.  Burgess-Brown</span>, <span class="hlFld-ContribAuthor ">Akane  Kawamura</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Schofield</span>, <span class="hlFld-ContribAuthor ">Paul E.  Brennan</span>. </span><span class="cited-content_cbyCitation_article-title">Optimisation of a triazolopyridine based histone demethylase inhibitor yields a potent and selective KDM2A (FBXL11) inhibitor. </span><span class="cited-content_cbyCitation_journal-name">MedChemComm</span><span> <strong>2014,</strong> <em>5 </em>
                                    (12)
                                     , 1879-1886. <a href="https://doi.org/10.1039/C4MD00291A" title="DOI URL">https://doi.org/10.1039/C4MD00291A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C4MD00291A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC4MD00291A%26sid%3Dliteratum%253Aachs%26jtitle%3DMedChemComm%26atitle%3DOptimisation%252Bof%252Ba%252Btriazolopyridine%252Bbased%252Bhistone%252Bdemethylase%252Binhibitor%252Byields%252Ba%252Bpotent%252Band%252Bselective%252BKDM2A%252B%252528FBXL11%252529%252Binhibitor%26aulast%3DEngland%26aufirst%3DKatherine%2BS.%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D12%26spage%3D1879%26epage%3D1886" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alena  Shmakova</span>, <span class="hlFld-ContribAuthor ">Michael  Batie</span>, <span class="hlFld-ContribAuthor ">Jimena  Druker</span>, <span class="hlFld-ContribAuthor ">Sonia  Rocha</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin and oxygen sensing in the context of JmjC histone demethylases. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Journal</span><span> <strong>2014,</strong> <em>462 </em>
                                    (3)
                                     , 385-395. <a href="https://doi.org/10.1042/BJ20140754" title="DOI URL">https://doi.org/10.1042/BJ20140754</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BJ20140754&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBJ20140754%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Journal%26atitle%3DChromatin%252Band%252Boxygen%252Bsensing%252Bin%252Bthe%252Bcontext%252Bof%252BJmjC%252Bhistone%252Bdemethylases%26aulast%3DShmakova%26aufirst%3DAlena%26date%3D2014%26date%3D2014%26volume%3D462%26issue%3D3%26spage%3D385%26epage%3D395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David P.  Gavin</span>, <span class="hlFld-ContribAuthor ">Christina  Floreani</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetics of Schizophrenia. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2014,</strong>,, 155-201. <a href="https://doi.org/10.1016/B978-0-12-801311-3.00005-6" title="DOI URL">https://doi.org/10.1016/B978-0-12-801311-3.00005-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801311-3.00005-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801311-3.00005-6%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetics%252Bof%252BSchizophrenia%26aulast%3DGavin%26aufirst%3DDavid%2BP.%26date%3D2014%26spage%3D155%26epage%3D201%26pub%3DElsevier%26atitle%3DEpigenetics%26date%3D2014%26volume%3D115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reactions Catalyzed by (A) Lysine-Specific Demethylases and (B) Jumonji C Demethylases</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of pan-demethylase inhibitors <b>1</b>–<b>6</b> described here and single-family target inhibitors <b>7</b>–<b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Manual docking of compounds <b>1</b> (yellow) and <b>3</b> (blue) in (A) LSD1 and (B) KDM4A active sites. (A) In LSD1, <b>1</b> and <b>3</b> are proposed to react with active-site-bound FAD, opening the cyclopropyl ring. The bipyridyl (in the case of <b>1</b>) and 8-hydroxyquinolyl (in the case of <b>3</b>) groups protrude toward the histone substrate-binding pocket. (B) In KDM4A, <b>1</b> and <b>3</b> are proposed to chelate to the active-site-bound Fe(II) via the bipyridyl and 8-hydroxyquinolyl groups, respectively. The tranylcypromine moiety in both compounds likely protrudes toward the histone substrate binding pocket, potentially forming hydrogen bonding and/or electrostatic interactions via the primary amine. Models of both enantiomers of <b>1</b> and <b>3</b> bound to KDM4A are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>1</b>–<b>6</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>(a) Racemic <i>trans</i>-<i>N</i>-Boc-2-(4-aminophenyl)cyclopropylamine <b>10</b>, HOBt, EDCI, Et<sub>3</sub>N, DMF, RT; (b) 0.2 N NaOH, MeOH, RT; (c) CF<sub>3</sub>COOH, 0 °C to RT; (d) 4 N HCl in 1,4-dioxane, THF/MeOH, RT; (e) methyl ω-aminoalkanoate hydrochloride, HOBT, EDCI, Et<sub>3</sub>N, DCM, RT; (f) 2 N LiOH, THF, RT.</p></p></figure><figure data-id="fig3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. (A–H) Detection of changes in the global histone methylation levels in HeLa cells after treatment with <b>1</b>, <b>2</b>, <b>3</b>, <b>7</b>, <b>8a</b>, and <b>9</b> over 72 h by use of an immunofluorescence-based assay. (I) Cytotoxicity of <b>2</b>, <b>3</b>, and <b>8a</b> at high doses is shown as number of cells per fixed field of view.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Effects of pan-demethylase inhibitors <b>1</b>–<b>6</b> in LNCaP and in HCT116 cells. (A, B) Cell cycle effect (left) and percentage of cells at pre-G1 peak (right) of <b>1</b>–<b>9</b> (50 μM, 30 h) in LNCaP (A) and HCT116 (B) cells are shown. (C) Dose–response curves with <b>2</b> and <b>3</b> for apoptosis (48 h) in LNCaP (left) and HCT116 (right) cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Comparison of effects on pre-G1 peak accumulation for <b>2</b>, <b>3</b>, <b>7</b>, <b>8</b>, and combination of <b>7</b> + <b>8</b> (50 μM, 24 h) in LNCaP, HCT116, and MePR cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Proliferation curves and cell migration relative to (A) LNCaP and (B) HCT116 cell lines after 24 h. The HDAC inhibitor SAHA was used as a reference drug. (Left) Proliferation curves after 24 h. Control (untreated cells) in red, cells treated with SAHA (5 μM) in green, cells treated with <b>2</b> (50 μM) in blue, cells treated with <b>3</b> (50 μM) in violet, and base-lines of cells treated with LNCaP (A) in sky-blue or with HCT116 (B) in yellow-green are shown. (Right) Cell migration after 24 h. Histograms have been represented by slope relative to the control (untreated cells), cells treated with SAHA (5 μM), cells treated with <b>2</b> (50 μM), and cells treated with <b>3</b> (50 μM). Data show the mean value from three parallel experiments with error bars showing the standard deviations on top of each column.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/medium/jm-2013-012802_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Western blot analyses of H3K4me2/3 and H3K9me2/3 methylation after treatment of LNCaP (left) and HCT116 (right) cells with <b>2</b>, <b>3</b>, and the reference compounds <b>7</b>–<b>9</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2014/jmcmar.2014.57.issue-1/jm4012802/production/images/large/jm-2013-012802_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm4012802&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i61">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00160" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00160" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 52 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Kouzarides, T.</span><span> </span><span class="NLM_article-title">Chromatin modifications and their function</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">128</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">–</span> <span class="NLM_lpage">705</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.cell.2007.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17320507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=128&publication_year=2007&pages=693-705&author=T.+Kouzarides&title=Chromatin+modifications+and+their+function"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin modifications and their function</span></div><div class="casAuthors">Kouzarides, Tony</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">128</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">693-705</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The surface of nucleosomes is studded with a multiplicity of modifications.  At least eight different classes have been characterized to date and many different sites have been identified for each class.  Operationally, modifications function either by disrupting chromatin contacts or by affecting the recruitment of nonhistone proteins to chromatin.  Their presence on histones can dictate the higher-order chromatin structure in which DNA is packaged and can orchestrate the ordered recruitment of enzyme complexes to manipulate DNA.  In this way, histone modifications have the potential to influence many fundamental biol. processes, some of which may be epigenetically inherited.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAH9AX2dGicbVg90H21EOLACvtfcHk0lhq2WUWG24oSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis12ju7Y%253D&md5=c5995fe68fd9473f4268d59c01d3b9de</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2007.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2007.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DKouzarides%26aufirst%3DT.%26atitle%3DChromatin%2520modifications%2520and%2520their%2520function%26jtitle%3DCell%26date%3D2007%26volume%3D128%26spage%3D693%26epage%3D705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1815</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Kooistra, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Molecular mechanisms and potential functions of histone demethylases</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">–</span> <span class="NLM_lpage">311</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnrm3327" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22473470" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=297-311&author=S.+M.+Kooistraauthor=K.+Helin&title=Molecular+mechanisms+and+potential+functions+of+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular mechanisms and potential functions of histone demethylases</span></div><div class="casAuthors">Kooistra, Susanne Marije; Helin, Kristian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">297-311</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Histone modifications are thought to regulate chromatin structure, transcription and other nuclear processes.  Histone methylation was originally believed to be an irreversible modification that could only be removed by histone eviction or by diln. during DNA replication.  However, the isolation of two families of enzymes that can demethylate histones has changed this notion.  The biochem. activities of these histone demethylases towards specific Lys residues on histones, and in some cases non-histone substrates, have highlighted their importance in developmental control, cell-fate decisions and disease.  Their ability to be regulated through protein-targeting complexes and post-translational modifications is also beginning to shed light on how they provide dynamic control during transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrolbbZ5rmYmrVg90H21EOLACvtfcHk0ljgcYzhXzs0gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFWhu78%253D&md5=a5f5ffdac5c5658bc262d584faf5216f</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm3327&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm3327%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DS.%2BM.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DMolecular%2520mechanisms%2520and%2520potential%2520functions%2520of%2520histone%2520demethylases%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2012%26volume%3D13%26spage%3D297%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battaglioli, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span> </span><span class="NLM_article-title">LSD1: Oxidative chemistry for multifaceted functions in chromatin regulation</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">189</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=181-189&author=F.+Fornerisauthor=C.+Bindaauthor=E.+Battaglioliauthor=A.+Mattevi&title=LSD1%3A+Oxidative+chemistry+for+multifaceted+functions+in+chromatin+regulation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DBattaglioli%26aufirst%3DE.%26aulast%3DMattevi%26aufirst%3DA.%26atitle%3DLSD1%253A%2520Oxidative%2520chemistry%2520for%2520multifaceted%2520functions%2520in%2520chromatin%2520regulation%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D181%26epage%3D189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Targeting histone demethylases: A new avenue for the fight against cancer</span> <span class="citation_source-journal">Genes Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">663</span><span class="NLM_x">–</span> <span class="NLM_lpage">679</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1177%2F1947601911417976" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21941621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlCgu7jI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=663-679&author=D.+Rotiliauthor=A.+Mai&title=Targeting+histone+demethylases%3A+A+new+avenue+for+the+fight+against+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting histone demethylases: a new avenue for the fight against cancer</span></div><div class="casAuthors">Rotili, Dante; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">663-679</span>CODEN:
                <span class="NLM_cas:coden">GCEAAY</span>;
        ISSN:<span class="NLM_cas:issn">1947-6019</span>.
    
            (<span class="NLM_cas:orgname">Sage Publications</span>)
        </div><div class="casAbstract">A review.  In addn. to genetic disorders, epigenetic alterations have been shown to be involved in cancer, through misregulation of histone modifications.  Miswriting, misreading, and mis-erasing of histone acetylation as well as methylation marks can be actually assocd. with oncogenesis and tumor proliferation.  Historically, methylation of Arg and Lys residues has been considered a stable, irreversible process due to the slow turnover of Me groups in chromatin.  The discovery in recent years of a large no. of histone Lys demethylases (KDMs, belonging to either the amino oxidase or the JmjC family) totally changed this point of view and suggested a new role for dynamic histone methylation in biol. processes.  Since overexpression, alteration, or mutation of a no. of KDMs has been found in many types of cancers, such enzymes could represent diagnostic tools as well as epigenetic targets to modulate for obtaining novel therapeutic weapons against cancer.  The first little steps in this direction are described here.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXByA6M4P0brVg90H21EOLACvtfcHk0ljgcYzhXzs0gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlCgu7jI&md5=0c5fcc8d7b5d5a07431dd26a85b6200a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1177%2F1947601911417976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1177%252F1947601911417976%26sid%3Dliteratum%253Aachs%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DTargeting%2520histone%2520demethylases%253A%2520A%2520new%2520avenue%2520for%2520the%2520fight%2520against%2520cancer%26jtitle%3DGenes%2520Cancer%26date%3D2011%26volume%3D2%26spage%3D663%26epage%3D679" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, A. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Günther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schüle, R.</span><span> </span><span class="NLM_article-title">LSD1 demethylates repressive histone marks to promote androgen-receptor-dependent transcription</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">437</span><span class="NLM_x">, </span> <span class="NLM_fpage">436</span><span class="NLM_x">–</span> <span class="NLM_lpage">439</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=437&publication_year=2005&pages=436-439&author=E.+Metzgerauthor=M.+Wissmannauthor=N.+Yinauthor=J.+M.+M%C3%BCllerauthor=R.+Schneiderauthor=A.+H.+Petersauthor=T.+G%C3%BCntherauthor=R.+Buettnerauthor=R.+Sch%C3%BCle&title=LSD1+demethylates+repressive+histone+marks+to+promote+androgen-receptor-dependent+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DM%25C3%25BCller%26aufirst%3DJ.%2BM.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DPeters%26aufirst%3DA.%2BH.%26aulast%3DG%25C3%25BCnther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DSch%25C3%25BCle%26aufirst%3DR.%26atitle%3DLSD1%2520demethylates%2520repressive%2520histone%2520marks%2520to%2520promote%2520androgen-receptor-dependent%2520transcription%26jtitle%3DNature%26date%3D2005%26volume%3D437%26spage%3D436%26epage%3D439" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine demethylases inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+demethylases+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lhjRuSmxA71kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520demethylases%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Crea, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrar, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danesi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helgason, C. D.</span><span> </span><span class="NLM_article-title">The emerging role of histone lysine demethylases in prostate cancer</span> <span class="citation_source-journal">Mol. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">52</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1186%2F1476-4598-11-52" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVSrsbrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=52&author=F.+Creaauthor=L.+Sunauthor=A.+Maiauthor=Y.+T.+Chiangauthor=W.+L.+Farrarauthor=R.+Danesiauthor=C.+D.+Helgason&title=The+emerging+role+of+histone+lysine+demethylases+in+prostate+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">The emerging role of histone lysine demethylases in prostate cancer</span></div><div class="casAuthors">Crea, Francesco; Sun, Lei; Mai, Antonello; Chiang, Yan Ting; Farrar, William L.; Danesi, Romano; Helgason, Cheryl D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">52</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Early prostate cancer (PCa) is generally treatable and assocd. with good prognosis.  After a variable time, PCa evolves into a highly metastatic and treatment-refractory disease: castration-resistant PCa (CRPC).  Currently, few prognostic factors are available to predict the emergence of CRPC, and no curative option is available.  Epigenetic gene regulation has been shown to trigger PCa metastasis and androgen-independence.  Most epigenetic studies have focused on DNA and histone methyltransferases.  While DNA methylation leads to gene silencing, histone methylation can trigger gene activation or inactivation, depending on the target amino acid residues and the extent of methylation (me1, me2, or me3).  Interestingly, some histone modifiers are essential for PCa tumor-initiating cell (TIC) self-renewal.  TICs are considered the seeds responsible for metastatic spreading and androgen-independence.  Histone Lysine Demethylases (KDMs) are a novel class of epigenetic enzymes which can remove both repressive and activating histone marks.  KDMs are currently grouped into 7 major classes, each one targeting a specific methylation site.  Since their discovery, KDM expression has been found to be deregulated in several neoplasms.  In PCa, KDMs may act as either tumor suppressors or oncogenes, depending on their gene regulatory function.  For example, KDM1A and KDM4C are essential for PCa androgen-dependent proliferation, while PHF8 is involved in PCa migration and invasion.  Interestingly, the possibility of pharmacol. targeting KDMs has been demonstrated.  In the present paper, we summarize the emerging role of KDMs in regulating the metastatic potential and androgen-dependence of PCa.  In addn., we speculate on the possible interaction between KDMs and other epigenetic effectors relevant for PCa TICs.  Finally, we explore the role of KDMs as novel prognostic factors and therapeutic targets.  We believe that studies on histone demethylation may add a novel perspective in our efforts to prevent and cure advanced PCa.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrllNqXZaFkN7Vg90H21EOLACvtfcHk0lhjRuSmxA71kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVSrsbrM&md5=8a8d12d7d2d788da032a668fd8c66ab9</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-11-52&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-11-52%26sid%3Dliteratum%253Aachs%26aulast%3DCrea%26aufirst%3DF.%26aulast%3DSun%26aufirst%3DL.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DChiang%26aufirst%3DY.%2BT.%26aulast%3DFarrar%26aufirst%3DW.%2BL.%26aulast%3DDanesi%26aufirst%3DR.%26aulast%3DHelgason%26aufirst%3DC.%2BD.%26atitle%3DThe%2520emerging%2520role%2520of%2520histone%2520lysine%2520demethylases%2520in%2520prostate%2520cancer%26jtitle%3DMol.%2520Cancer%26date%3D2012%26volume%3D11%26spage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Amente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lania, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majello, B.</span><span> </span><span class="NLM_article-title">The histone LSD1 demethylase in stemness and cancer transcription programs</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">1829</span><span class="NLM_x">, </span> <span class="NLM_fpage">981</span><span class="NLM_x">–</span> <span class="NLM_lpage">986</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1829&publication_year=2013&pages=981-986&author=S.+Amenteauthor=L.+Laniaauthor=B.+Majello&title=The+histone+LSD1+demethylase+in+stemness+and+cancer+transcription+programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAmente%26aufirst%3DS.%26aulast%3DLania%26aufirst%3DL.%26aulast%3DMajello%26aufirst%3DB.%26atitle%3DThe%2520histone%2520LSD1%2520demethylase%2520in%2520stemness%2520and%2520cancer%2520transcription%2520programs%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2013%26volume%3D1829%26spage%3D981%26epage%3D986" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Andrisani, O. M.</span><span> </span><span class="NLM_article-title">Deregulation of epigenetic mechanisms by the hepatitis B virus X protein in hepatocarcinogenesis</span> <span class="citation_source-journal">Viruses</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">858</span><span class="NLM_x">–</span> <span class="NLM_lpage">872</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.3390%2Fv5030858" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltFSmsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=858-872&author=O.+M.+Andrisani&title=Deregulation+of+epigenetic+mechanisms+by+the+hepatitis+B+virus+X+protein+in+hepatocarcinogenesis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of epigenetic mechanisms by the hepatitis B virus X protein in hepatocarcinogenesis</span></div><div class="casAuthors">Andrisani, Ourania M.</div><div class="citationInfo"><span class="NLM_cas:title">Viruses</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">858-872</span>CODEN:
                <span class="NLM_cas:coden">VIRUBR</span>;
        ISSN:<span class="NLM_cas:issn">1999-4915</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">This review focuses on the significance of deregulation of epigenetic mechanisms by the hepatitis B virus (HBV) X protein in hepatocarcinogenesis and HBV replication.  Epigenetic mechanisms, DNA methylation, and specific histone modifications, e.g., trimethylation of H3 on lysine-27 or lysine-4, maintain 'cellular memory' by silencing expression of lineage-inducing factors in stem cells and conversely, of pluripotency factors in differentiated cells.  The X protein has been reported to induce expression of DNA methyltransferases (DNMTs), likely promoting epigenetic changes during hepatocarcinogenesis.  Furthermore, in cellular and animal models of X-mediated oncogenic transformation, protein levels of chromatin modifying proteins Suz12 and Znf198 are down-regulated.  Suz12 is essential for the Polycomb Repressive Complex 2 (PRC2) mediating the repressive trimethylation of H3 on lysine-27 (H3K27me3).  Znf198, stabilizes the LSD1-CoREST-HDAC complex that removes, via lysine demethylase1 (LSD1), the activating trimethylation of H3 on lysine-4 (H3K4me3).  Down-regulation of Suz12 also occurs in liver tumors of woodchucks chronically infected by woodchuck hepatitis virus, an animal model recapitulating HBV-mediated hepatocarcinogenesis in humans.  Significantly, subgroups of HBV-induced liver cancer re-express hepatoblast and fetal markers, and imprinted genes, suggesting hepatocyte reprogramming during oncogenic transformation.  Lastly, down-regulation of Suz12 and Znf198 enhances HBV replication.  Collectively, these observations suggest deregulation of epigenetic mechanisms by HBV X protein influences both the viral cycle and the host cell.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYAyyFUp_UgrVg90H21EOLACvtfcHk0lik-t7JmHikyQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltFSmsLY%253D&md5=5ebf5dcab8e9a1771cf4c2b78f786eee</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.3390%2Fv5030858&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fv5030858%26sid%3Dliteratum%253Aachs%26aulast%3DAndrisani%26aufirst%3DO.%2BM.%26atitle%3DDeregulation%2520of%2520epigenetic%2520mechanisms%2520by%2520the%2520hepatitis%2520B%2520virus%2520X%2520protein%2520in%2520hepatocarcinogenesis%26jtitle%3DViruses%26date%3D2013%26volume%3D5%26spage%3D858%26epage%3D872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogel, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">Inhibition of the histone demethylase LSD1 blocks α-herpesvirus lytic replication and reactivation from latency</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">1312</span><span class="NLM_x">–</span> <span class="NLM_lpage">1317</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnm.2051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=19855399" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=1312-1317&author=Y.+Liangauthor=J.+L.+Vogelauthor=A.+Narayananauthor=H.+Pengauthor=T.+M.+Kristie&title=Inhibition+of+the+histone+demethylase+LSD1+blocks+%CE%B1-herpesvirus+lytic+replication+and+reactivation+from+latency"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the histone demethylase LSD1 blocks α-herpesvirus lytic replication and reactivation from latency</span></div><div class="casAuthors">Liang, Yu; Vogel, Jodi L.; Narayanan, Aarthi; Peng, Hua; Kristie, Thomas M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1312-1317</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Reversible methylation of histone tails serves as either a pos. signal recognized by transcriptional assemblies or a neg. signal that result in repression.  Invading viral pathogens that depend upon the host cell's transcriptional app. are also subject to the regulatory impact of chromatin assembly and modifications.  Here we show that infection by the α-herpesviruses, herpes simplex virus (HSV) and varicella zoster virus (VZV), results in the rapid accumulation of chromatin bearing repressive histone H3 Lys9 methylation.  To enable expression of viral immediate early (IE) genes, both viruses use the cellular transcriptional coactivator host cell factor-1 (HCF-1) to recruit the lysine-specific demethylase-1 (LSD1) to the viral immediate early promoters.  Depletion of LSD1 or inhibition of its activity with monoamine oxidase inhibitors (MAOIs) results in the accumulation of repressive chromatin and a block to viral gene expression.  As HCF-1 is a component of the Set1 and MLL1 histone H3 Lys4 methyltransferase complexes, it thus coordinates modulation of repressive H3 Lys9 methylation levels with addn. of activating H3 Lys4 trimethylation marks.  Strikingly, MAOIs also block the reactivation of HSV from latency in sensory neurons, indicating that the HCF-1 complex is a crucial component of the reactivation mechanism.  The results support pharmaceutical control of histone modifying enzymes as a strategy for controlling herpesvirus infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppT5vH8lPRnbVg90H21EOLACvtfcHk0lj49WnV0id5RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSqsLbF&md5=bd9aacfdbcba54e30f9482f8fcc96777</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnm.2051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2051%26sid%3Dliteratum%253Aachs%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DVogel%26aufirst%3DJ.%2BL.%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DPeng%26aufirst%3DH.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DInhibition%2520of%2520the%2520histone%2520demethylase%2520LSD1%2520blocks%2520%25CE%25B1-herpesvirus%2520lytic%2520replication%2520and%2520reactivation%2520from%2520latency%26jtitle%3DNat.%2520Med.%26date%3D2009%26volume%3D15%26spage%3D1312%26epage%3D1317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Narayanan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruyechan, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristie, T. M.</span><span> </span><span class="NLM_article-title">The coactivator host cell factor-1 mediates Set1 and MLL1 H3K4 trimethylation at herpesvirus immediate early promoters for initiation of infection</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">10835</span><span class="NLM_x">–</span> <span class="NLM_lpage">10840</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=10835-10840&author=A.+Narayananauthor=W.+T.+Ruyechanauthor=T.+M.+Kristie&title=The+coactivator+host+cell+factor-1+mediates+Set1+and+MLL1+H3K4+trimethylation+at+herpesvirus+immediate+early+promoters+for+initiation+of+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DA.%26aulast%3DRuyechan%26aufirst%3DW.%2BT.%26aulast%3DKristie%26aufirst%3DT.%2BM.%26atitle%3DThe%2520coactivator%2520host%2520cell%2520factor-1%2520mediates%2520Set1%2520and%2520MLL1%2520H3K4%2520trimethylation%2520at%2520herpesvirus%2520immediate%2520early%2520promoters%2520for%2520initiation%2520of%2520infection%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D10835%26epage%3D10840" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibition of 2-oxoglutarate dependent oxygenases</span> <span class="citation_source-journal">Chem. Soc. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4364</span><span class="NLM_x">–</span> <span class="NLM_lpage">4397</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1039%2Fc0cs00203h" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21390379" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2011&pages=4364-4397&author=N.+R.+Roseauthor=M.+A.+McDonoughauthor=O.+N.+Kingauthor=A.+Kawamuraauthor=C.+J.+Schofield&title=Inhibition+of+2-oxoglutarate+dependent+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of 2-oxoglutarate dependent oxygenases</span></div><div class="casAuthors">Rose, Nathan R.; McDonough, Michael A.; King, Oliver N. F.; Kawamura, Akane; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4364-4397</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  2-Oxoglutarate (2OG) dependent oxygenases are ubiquitous iron enzymes that couple substrate oxidn. to the conversion of 2OG to succinate and carbon dioxide.  In humans their roles include collagen biosynthesis, fatty acid metab., DNA repair, RNA and chromatin modifications, and hypoxic sensing.  Com. applications of 2OG oxygenase inhibitors began with plant growth retardants, and now extend to a clin. used pharmaceutical compd. for cardioprotection.  Several 2OG oxygenases are now being targeted for therapeutic intervention for diseases including anemia, inflammation and cancer.  In this crit. review, we describe studies on the inhibition of 2OG oxygenases, focusing on small mols., and discuss the potential of 2OG oxygenases as therapeutic targets (295 refs.).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqp4Z13BJAo17Vg90H21EOLACvtfcHk0lj49WnV0id5RA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVemur8%253D&md5=f7a87e0bc9392d9a29f4ffb7d6f92b44</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1039%2Fc0cs00203h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc0cs00203h%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibition%2520of%25202-oxoglutarate%2520dependent%2520oxygenases%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2011%26volume%3D40%26spage%3D4364%26epage%3D4397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Conte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Molecular pathways: the complexity of the epigenome in cancer and recent clinical advances</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">5526</span><span class="NLM_x">–</span> <span class="NLM_lpage">5534</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1158%2F1078-0432.CCR-12-2037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22904103" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFahu73I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=5526-5534&author=M.+Conteauthor=L.+Altucci&title=Molecular+pathways%3A+the+complexity+of+the+epigenome+in+cancer+and+recent+clinical+advances"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Pathways: The Complexity of the Epigenome in Cancer and Recent Clinical Advances</span></div><div class="casAuthors">Conte, Mariarosaria; Altucci, Lucia</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5526-5534</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  Human cancer is causally linked to genomic and epigenomic deregulations.  Epigenetic abnormalities occurring within signaling pathways regulating proliferation, migration, growth, differentiation, transcription, and death signals may be crit. in the progression of malignancies.  Consequently, identification of epigenetic marks and their bioimplications in tumors represents a crucial step toward defining new therapeutic strategies both in cancer treatment and prevention.  Alterations of writers, readers, and erasers in cancer may affect, for example, the methylation and acetylation state of huge areas of chromatin, suggesting that epi-based treatments may require "distinct" therapeutic strategies compared with "canonical" targeted treatments.  Whereas anticancer treatments targeting histone deacetylase and DNA methylation have entered the clinic, addnl. chromatin modification enzymes have not yet been pharmacol. targeted for clin. use in patients.  Thus, a greater insight into alterations occurring on chromatin modifiers and their impact in tumorigenesis represents a crucial advancement in exploiting epigenetic targeting in cancer prevention and treatment.  Here, the interplay of the best known epi-mutations and how their targeting might be optimized are addressed.  Clin Cancer Res; 18(20); 5526-34. ©2012 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0GJKNWI0QMbVg90H21EOLACvtfcHk0lisGcVYXsexCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFahu73I&md5=866f7001a59f17ba545b2a193687d89f</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-12-2037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-12-2037%26sid%3Dliteratum%253Aachs%26aulast%3DConte%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DMolecular%2520pathways%253A%2520the%2520complexity%2520of%2520the%2520epigenome%2520in%2520cancer%2520and%2520recent%2520clinical%2520advances%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D5526%26epage%3D5534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Abidi, F. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weaver, D. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simensen, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, C. E.</span><span> </span><span class="NLM_article-title">Mutations in JARID1C are associated with X-linked mental retardation, short stature and hyperreflexia</span> <span class="citation_source-journal">J. Med. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">787</span><span class="NLM_x">–</span> <span class="NLM_lpage">93</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1136%2Fjmg.2008.058990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWktbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2008&pages=787-93&author=F.+E.+Abidiauthor=L.+Hollowayauthor=C.+A.+Mooreauthor=D.+D.+Weaverauthor=R.+J.+Simensenauthor=R.+E.+Stevensonauthor=R.+C.+Rogersauthor=C.+E.+Schwartz&title=Mutations+in+JARID1C+are+associated+with+X-linked+mental+retardation%2C+short+stature+and+hyperreflexia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in JARID1C are associated with X-linked mental retardation, short stature and hyperreflexia</span></div><div class="casAuthors">Abidi, F. E.; Holloway, L.; Moore, C. A.; Weaver, D. D.; Simensen, R. J.; Stevenson, R. E.; Rogers, R. C.; Schwartz, C. E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medical Genetics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">787-793</span>CODEN:
                <span class="NLM_cas:coden">JMDGAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-2593</span>.
    
            (<span class="NLM_cas:orgname">BMJ Publishing Group</span>)
        </div><div class="casAbstract">Mutations in the JARID1C (Jumonji AT-rich interactive domain 1C) gene were recently assocd. with X-linked mental retardation (XLMR).  Mutations in this gene are reported to be one of the relatively more common causes of XLMR with a frequency of approx. 3% in males with proven or probable XLMR.  The JARID1C protein functions as a histone 3 lysine 4 (H3K4) demethylase and is involved in the demethylation of H3K4me3 and H3K4me2.  Mutation anal. of the JARID1C gene was conducted in the following cohorts: probands from 23 XLMR families linked to Xp11.2, 92 males with mental retardation and short stature, and 172 probands from small XLMR families with no linkage information.  Four novel mutations consisting of two missense mutations, p.A77T and p.V504M, and two frame shift mutations, p.E468fsX2 and p.R1481fsX9, were identified in males with mental retardation.  Two of the mutations, p.V504M and p.E468fsX2, are located in the JmjC domain of the JARID1C gene where no previous mutations have been reported.  Addnl. studies showed that the missense mutation, p.V504M, was a de novo event on the grandpaternal X chromosome of the family.  Clin. findings of the nine affected males from the four different families included mental retardation (100%), short stature (55%), hyperreflexia (78%), seizures (33%) and aggressive behavior (44%).  The degree of mental retardation consisted of mild (25%), moderate (12%) and severe (63%).  Based on the clin. observations, male patients with mental retardation, short stature and hyperreflexia should be considered candidates for mutations in the JARID1C gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1xtwZdctwWbVg90H21EOLACvtfcHk0lisGcVYXsexCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWktbY%253D&md5=9624349b128f3429d01effd5ba926bfc</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1136%2Fjmg.2008.058990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1136%252Fjmg.2008.058990%26sid%3Dliteratum%253Aachs%26aulast%3DAbidi%26aufirst%3DF.%2BE.%26aulast%3DHolloway%26aufirst%3DL.%26aulast%3DMoore%26aufirst%3DC.%2BA.%26aulast%3DWeaver%26aufirst%3DD.%2BD.%26aulast%3DSimensen%26aufirst%3DR.%2BJ.%26aulast%3DStevenson%26aufirst%3DR.%2BE.%26aulast%3DRogers%26aufirst%3DR.%2BC.%26aulast%3DSchwartz%26aufirst%3DC.%2BE.%26atitle%3DMutations%2520in%2520JARID1C%2520are%2520associated%2520with%2520X-linked%2520mental%2520retardation%252C%2520short%2520stature%2520and%2520hyperreflexia%26jtitle%3DJ.%2520Med.%2520Genet.%26date%3D2008%26volume%3D45%26spage%3D787%26epage%3D93" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Adegbola, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sommer, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Browning, M.</span><span> </span><span class="NLM_article-title">A novel mutation in JARID1C/SMCX in a patient with autism spectrum disorder (ASD)</span> <span class="citation_source-journal">Am. J. Med. Genet. A</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">146A</span><span class="NLM_x">, </span> <span class="NLM_fpage">505</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1002%2Fajmg.a.32142" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXislCksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146A&publication_year=2008&pages=505-511&author=A.+Adegbolaauthor=H.+Gaoauthor=S.+Sommerauthor=M.+Browning&title=A+novel+mutation+in+JARID1C%2FSMCX+in+a+patient+with+autism+spectrum+disorder+%28ASD%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A novel mutation in JARID1C/SMCX in a patient with austism spectrum disorder (ASD)</span></div><div class="casAuthors">Adegbola, Abidemi; Gao, Hanlin; Sommer, Steve; Browning, Marsha</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Medical Genetics, Part A</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">146A</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-511</span>CODEN:
                <span class="NLM_cas:coden">AJMGB8</span>;
        ISSN:<span class="NLM_cas:issn">1552-4825</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">We describe a nondysmorphic patient with developmental delay and autism spectrum disorder who has a missense mutation in the Jumonji AT-rich interactive domain 1C (JARID1C) gene.  This child first presented at 30 mo of age with stereotyped and repetitive behaviors, impairment in social reciprocity and in the use of multiple nonverbal behaviors, and developmental delay primarily in the language domain.  A diagnosis of autism was made and subsequently confirmed at the current age of 47 mo.  Cytogenetic and fragile X studies were normal.  Mutational anal. revealed a novel missense mutation in exon 16 of the JARID1C gene that results in an arginine to tryptophan substitution at amino acid 766 (R766W).  Sequence alignment anal. with multiple available eukaryotic sequences including the homologous proteins of mouse and zebrafish demonstrated that the affected amino acid is conserved.  JARID1C has not previously been implicated in autism susceptibility.  Recent novel mol. evidence suggests that it is a histone demethylase specific for di- and trimethylated histone 3 lysine 4 (H3K4) and functions as a transcriptional repressor by fostering REST-mediated neuronal gene regulation.  The JARID1C-regulated genes SCN2A, CACNA1H, BDNF, and SLC18A1 have previously been assocd. with autism and cognitive dysfunction.  This patient brings the total no. of reported JARID1C mutations to 14.  This presentation both extends the range of neurocognitive phenotypes attributable to mutations in this gene and illustrates the importance of mol. studies and DNA sequence anal. for accurate diagnosis of monogenic causes of autism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphl01Md4iStrVg90H21EOLACvtfcHk0liLFqQC8lZhng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXislCksrk%253D&md5=a6dcc2325e9abe3e1e108fdc20b99cc9</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fajmg.a.32142&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fajmg.a.32142%26sid%3Dliteratum%253Aachs%26aulast%3DAdegbola%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DH.%26aulast%3DSommer%26aufirst%3DS.%26aulast%3DBrowning%26aufirst%3DM.%26atitle%3DA%2520novel%2520mutation%2520in%2520JARID1C%252FSMCX%2520in%2520a%2520patient%2520with%2520autism%2520spectrum%2520disorder%2520%2528ASD%2529%26jtitle%3DAm.%2520J.%2520Med.%2520Genet.%2520A%26date%3D2008%26volume%3D146A%26spage%3D505%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Santos-Rebouças, C. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fintelman-Rodrigues, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuss, A. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribeiro, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campos, M.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pimentel, M. M.</span><span> </span><span class="NLM_article-title">A novel nonsense mutation in KDM5C/JARID1C gene causing intellectual disability, short stature and speech delay</span> <span class="citation_source-journal">Neurosci. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">498</span><span class="NLM_x">, </span> <span class="NLM_fpage">67</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=498&publication_year=2011&pages=67-71&author=C.+B.+Santos-Rebou%C3%A7asauthor=N.+Fintelman-Rodriguesauthor=L.+R.+Jensenauthor=A.+W.+Kussauthor=M.+G.+Ribeiroauthor=M.+Camposauthor=J.+M.+Santosauthor=M.+M.+Pimentel&title=A+novel+nonsense+mutation+in+KDM5C%2FJARID1C+gene+causing+intellectual+disability%2C+short+stature+and+speech+delay"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSantos-Rebou%25C3%25A7as%26aufirst%3DC.%2BB.%26aulast%3DFintelman-Rodrigues%26aufirst%3DN.%26aulast%3DJensen%26aufirst%3DL.%2BR.%26aulast%3DKuss%26aufirst%3DA.%2BW.%26aulast%3DRibeiro%26aufirst%3DM.%2BG.%26aulast%3DCampos%26aufirst%3DM.%26aulast%3DSantos%26aufirst%3DJ.%2BM.%26aulast%3DPimentel%26aufirst%3DM.%2BM.%26atitle%3DA%2520novel%2520nonsense%2520mutation%2520in%2520KDM5C%252FJARID1C%2520gene%2520causing%2520intellectual%2520disability%252C%2520short%2520stature%2520and%2520speech%2520delay%26jtitle%3DNeurosci.%2520Lett.%26date%3D2011%26volume%3D498%26spage%3D67%26epage%3D71" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sengupta, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dorsey, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richter, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opravil, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shiekhattar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, S. L.</span><span> </span><span class="NLM_article-title">p53 is regulated by the lysine demethylase LSD1</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">449</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">108</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=449&publication_year=2007&pages=105-108&author=J.+Huangauthor=R.+Senguptaauthor=A.+B.+Espejoauthor=M.+G.+Leeauthor=J.+A.+Dorseyauthor=M.+Richterauthor=S.+Opravilauthor=R.+Shiekhattarauthor=M.+T.+Bedfordauthor=T.+Jenuweinauthor=S.+L.+Berger&title=p53+is+regulated+by+the+lysine+demethylase+LSD1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DEspejo%26aufirst%3DA.%2BB.%26aulast%3DLee%26aufirst%3DM.%2BG.%26aulast%3DDorsey%26aufirst%3DJ.%2BA.%26aulast%3DRichter%26aufirst%3DM.%26aulast%3DOpravil%26aufirst%3DS.%26aulast%3DShiekhattar%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DBerger%26aufirst%3DS.%2BL.%26atitle%3Dp53%2520is%2520regulated%2520by%2520the%2520lysine%2520demethylase%2520LSD1%26jtitle%3DNature%26date%3D2007%26volume%3D449%26spage%3D105%26epage%3D108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Scoumanne, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span> </span><span class="NLM_article-title">The lysine-specific demethylase 1 is required for cell proliferation in both p53-dependent and -independent manners</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">15471</span><span class="NLM_x">–</span> <span class="NLM_lpage">15475</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=15471-15475&author=A.+Scoumanneauthor=X.+Chen&title=The+lysine-specific+demethylase+1+is+required+for+cell+proliferation+in+both+p53-dependent+and+-independent+manners"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DScoumanne%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DX.%26atitle%3DThe%2520lysine-specific%2520demethylase%25201%2520is%2520required%2520for%2520cell%2520proliferation%2520in%2520both%2520p53-dependent%2520and%2520-independent%2520manners%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D15471%26epage%3D15475" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">21504</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-21504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lhASnUI4sgpeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D21504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Kontaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Lysine methylation regulates E2F1-induced cell death</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">160</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.molcel.2010.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=20603083" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2010&pages=152-160&author=H.+Kontakiauthor=I.+Talianidis&title=Lysine+methylation+regulates+E2F1-induced+cell+death"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine methylation regulates E2F1-induced cell death</span></div><div class="casAuthors">Kontaki, Haroula; Talianidis, Iannis</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">152-160</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Histone-modifying enzymes can regulate DNA damage-induced apoptosis through modulation of p53 function.  Here, we show that, in p53-deficient tumor cells, Set9 and LSD1 regulate DNA damage-induced cell death in a manner opposite to that obsd. in p53+/+ cells, via modulation of E2F1 stabilization.  Set9 methylates E2F1 at lysine-185, which prevents E2F1 accumulation during DNA damage and activation of its proapoptotic target gene p73.  This Me mark is removed by LSD1, which is required for E2F1 stabilization and apoptotic function.  The mol. mechanism involves cross-talks between Lys methylation and other covalent modifications that affect E2F1 stability.  Methylation at Lys-185 inhibits acetylation and phosphorylation at distant positions and, in parallel, stimulates ubiquitination and degrdn. of the protein.  The findings illustrate that the function of methyltransferases can have opposing biol. outcomes depending on the specificity of transcription factor targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXEj98lICfQ7Vg90H21EOLACvtfcHk0lhQFC9jQiqljA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFejs7w%253D&md5=12b34f33cde33be3f37a53f8115aecde</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2010.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2010.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DKontaki%26aufirst%3DH.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DLysine%2520methylation%2520regulates%2520E2F1-induced%2520cell%2520death%26jtitle%3DMol.%2520Cell%26date%3D2010%26volume%3D39%26spage%3D152%26epage%3D160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Xie, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mei, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Z.</span><span> </span><span class="NLM_article-title">Methylation-mediated regulation of E2F1 in DNA damage-induced cell death</span> <span class="citation_source-journal">J. Recept. Signal Transduct. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">146</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2011&pages=139-146&author=Q.+Xieauthor=Y.+Baiauthor=J.+Wuauthor=Y.+Sunauthor=Y.+Wangauthor=Y.+Zhangauthor=P.+Meiauthor=Z.+Yuan&title=Methylation-mediated+regulation+of+E2F1+in+DNA+damage-induced+cell+death"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DQ.%26aulast%3DBai%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMei%26aufirst%3DP.%26aulast%3DYuan%26aufirst%3DZ.%26atitle%3DMethylation-mediated%2520regulation%2520of%2520E2F1%2520in%2520DNA%2520damage-induced%2520cell%2520death%26jtitle%3DJ.%2520Recept.%2520Signal%2520Transduct.%2520Res.%26date%3D2011%26volume%3D31%26spage%3D139%26epage%3D146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Cho, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dohmae, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hayami, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Unoki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshimatsu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Demethylation of RB regulator MYPT1 by histone demethylase LSD1 promotes cell cycle progression in cancer cells</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">655</span><span class="NLM_x">–</span> <span class="NLM_lpage">660</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1158%2F0008-5472.CAN-10-2446" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21115810" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=655-660&author=H.+S.+Choauthor=T.+Suzukiauthor=N.+Dohmaeauthor=S.+Hayamiauthor=M.+Unokiauthor=M.+Yoshimatsuauthor=G.+Toyokawaauthor=M.+Takawaauthor=T.+Chenauthor=J.+K.+Kurashauthor=H.+I.+Fieldauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Demethylation+of+RB+regulator+MYPT1+by+histone+demethylase+LSD1+promotes+cell+cycle+progression+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Demethylation of RB Regulator MYPT1 by Histone Demethylase LSD1 Promotes Cell Cycle Progression in Cancer Cells</span></div><div class="casAuthors">Cho, Hyun-Soo; Suzuki, Takehiro; Dohmae, Naoshi; Hayami, Shinya; Unoki, Motoko; Yoshimatsu, Masanori; Toyokawa, Gouji; Takawa, Masashi; Chen, Taiping; Kurash, Julia K.; Field, Helen I.; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">655-660</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Histone demethylase LSD1 (also known as KDM1 and AOF2) is active in various cancer cells, but its biol. significance in human carcinogenesis is unexplored.  In this study, we explored hypothesized interactions between LSD1 and MYPT1, a known regulator of RB1 phosphorylation.  We found that MYPT1 was methylated in vitro and in vivo by histone lysine methyltransferase SETD7 and demethylated by LSD1, identifying Lys 442 of MYPT1 as a target for methylation/demethylation by these enzymes.  LSD1 silencing increased MYPT1 protein levels, decreasing the steady state level of phosphorylated RB1 (Ser 807/811) and reducing E2F activity.  MYPT1 methylation status influenced the affinity of MYPT1 for the ubiquitin-proteasome pathway of protein turnover.  MYPT1 was unstable in murine cells deficient in SETD7, supporting the concept that MYPT1 protein stability is physiol. regulated by methylation status.  LSD1 overexpression could activate RB1 phosphorylation by inducing a destabilization of MYPT1 protein.  Taken together, our results comprise a novel cell cycle regulatory mechanism mediated by methylation/demethylation dynamics, and they reveal the significance of LSD1 overexpression in human carcinogenesis.  Cancer Res; 71(3); 655-60.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqh4fxBs2QNSrVg90H21EOLACvtfcHk0lhQFC9jQiqljA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVCis7c%253D&md5=4812d562b79453755b7c2eb8f74cc3eb</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-10-2446&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-10-2446%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DH.%2BS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DDohmae%26aufirst%3DN.%26aulast%3DHayami%26aufirst%3DS.%26aulast%3DUnoki%26aufirst%3DM.%26aulast%3DYoshimatsu%26aufirst%3DM.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DDemethylation%2520of%2520RB%2520regulator%2520MYPT1%2520by%2520histone%2520demethylase%2520LSD1%2520promotes%2520cell%2520cycle%2520progression%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D655%26epage%3D660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lj44ubF0_qNhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Solá, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xavier, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santos, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aranha, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgado, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jepsen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodrigues, C. M.</span><span> </span><span class="NLM_article-title">p53 interaction with JMJD3 results in its nuclear distribution during mouse neural stem cell differentiation</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">e18421</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=S.+Sol%C3%A1author=J.+M.+Xavierauthor=D.+M.+Santosauthor=M.+M.+Aranhaauthor=A.+L.+Morgadoauthor=K.+Jepsenauthor=C.+M.+Rodrigues&title=p53+interaction+with+JMJD3+results+in+its+nuclear+distribution+during+mouse+neural+stem+cell+differentiation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSol%25C3%25A1%26aufirst%3DS.%26aulast%3DXavier%26aufirst%3DJ.%2BM.%26aulast%3DSantos%26aufirst%3DD.%2BM.%26aulast%3DAranha%26aufirst%3DM.%2BM.%26aulast%3DMorgado%26aufirst%3DA.%2BL.%26aulast%3DJepsen%26aufirst%3DK.%26aulast%3DRodrigues%26aufirst%3DC.%2BM.%26atitle%3Dp53%2520interaction%2520with%2520JMJD3%2520results%2520in%2520its%2520nuclear%2520distribution%2520during%2520mouse%2520neural%2520stem%2520cell%2520differentiation%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jackson, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gudkov, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Regulation of NF-kappaB by NSD1/FBXL11-dependent reversible lysine methylation of p65</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">46</span><span class="NLM_x">–</span> <span class="NLM_lpage">51</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=46-51&author=T.+Luauthor=M.+W.+Jacksonauthor=B.+Wangauthor=M.+Yangauthor=M.+R.+Chanceauthor=M.+Miyagiauthor=A.+V.+Gudkovauthor=G.+R.+Stark&title=Regulation+of+NF-kappaB+by+NSD1%2FFBXL11-dependent+reversible+lysine+methylation+of+p65"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJackson%26aufirst%3DM.%2BW.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DGudkov%26aufirst%3DA.%2BV.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DRegulation%2520of%2520NF-kappaB%2520by%2520NSD1%252FFBXL11-dependent%2520reversible%2520lysine%2520methylation%2520of%2520p65%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2010%26volume%3D107%26spage%3D46%26epage%3D51" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Wissmann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greschik, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fodor, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenuwein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vogler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gunther, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buettner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span> </span><span class="NLM_article-title">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span> <span class="citation_source-journal">Nat. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">347</span><span class="NLM_x">–</span> <span class="NLM_lpage">353</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fncb1546" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17277772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=347-353&author=M.+Wissmannauthor=N.+Yinauthor=J.+M.+Mullerauthor=H.+Greschikauthor=B.+D.+Fodorauthor=T.+Jenuweinauthor=C.+Voglerauthor=R.+Schneiderauthor=T.+Guntherauthor=R.+Buettnerauthor=E.+Metzgerauthor=R.+Schule&title=Cooperative+demethylation+by+JMJD2C+and+LSD1+promotes+androgen+receptor-dependent+gene+expression"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Cooperative demethylation by JMJD2C and LSD1 promotes androgen receptor-dependent gene expression</span></div><div class="casAuthors">Wissmann, Melanie; Yin, Na; Mueller, Judith M.; Greschik, Holger; Fodor, Barna D.; Jenuwein, Thomas; Vogler, Christine; Schneider, Robert; Guenther, Thomas; Buettner, Reinhard; Metzger, Eric; Schuele, Roland</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-353</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Posttranslational modifications of histones, such as methylation, regulate chromatin structure and gene expression.  Recently, lysine-specific demethylase 1 (LSD1), the first histone demethylase, was identified.  LSD1 interacts with the androgen receptor and promotes androgen-dependent transcription of target genes by ligand-induced demethylation of mono- and dimethylated histone H3 at Lys 9 (H3K9) only.  Here, we identify the Jumonji C (JMJC) domain-contg. protein JMJD2C as the first histone tridemethylase regulating androgen receptor function.  JMJD2C interacts with androgen receptor in vitro and in vivo.  Assembly of ligand-bound androgen receptor and JMJD2C on androgen receptor-target genes results in demethylation of tri-Me H3K9 and in stimulation of androgen receptor-dependent transcription.  Conversely, knockdown of JMJD2C inhibits androgen-induced removal of tri-Me H3K9, transcriptional activation and tumor cell proliferation.  Importantly, JMJD2C colocalizes with androgen receptor and LSD1 in normal prostate and in prostate carcinomas.  JMJD2C and LSD1 interact and both demethylases cooperatively stimulate androgen receptor-dependent gene transcription.  In addn., androgen receptor, JMJD2C and LSD1 assemble on chromatin to remove Me groups from mono, di and trimethylated H3K9.  Thus, our data suggest that specific gene regulation requires the assembly and coordinate action of demethylases with distinct substrate specificities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2b5sXBcoCRrVg90H21EOLACvtfcHk0liSDXAyd9XAmg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXit12rsrw%253D&md5=d1f7ec6fbaadf301372ebe644a2e978a</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1038%2Fncb1546&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1546%26sid%3Dliteratum%253Aachs%26aulast%3DWissmann%26aufirst%3DM.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMuller%26aufirst%3DJ.%2BM.%26aulast%3DGreschik%26aufirst%3DH.%26aulast%3DFodor%26aufirst%3DB.%2BD.%26aulast%3DJenuwein%26aufirst%3DT.%26aulast%3DVogler%26aufirst%3DC.%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DGunther%26aufirst%3DT.%26aulast%3DBuettner%26aufirst%3DR.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DSchule%26aufirst%3DR.%26atitle%3DCooperative%2520demethylation%2520by%2520JMJD2C%2520and%2520LSD1%2520promotes%2520androgen%2520receptor-dependent%2520gene%2520expression%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D347%26epage%3D353" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Hamada, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mino, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koseki, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oehme, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flamme, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozasa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoh, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogasawara, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Komaarashi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kato, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsumoto, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakagawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sasaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mizukami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, H.</span><span> </span><span class="NLM_article-title">Design, synthesis, enzyme-inhibitory activity, and effect on human cancer cells of a novel series of Jumonji domain-containing protein 2 histone demethylase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">5629</span><span class="NLM_x">–</span> <span class="NLM_lpage">5638</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm1003655" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovVCis7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=5629-5638&author=S.+Hamadaauthor=T.+Suzukiauthor=K.+Minoauthor=K.+Kosekiauthor=F.+Oehmeauthor=I.+Flammeauthor=H.+Ozasaauthor=Y.+Itohauthor=D.+Ogasawaraauthor=H.+Komaarashiauthor=A.+Katoauthor=H.+Tsumotoauthor=M.+Nakagawaauthor=R.+Hasegawaauthor=T.+Sasakiauthor=N.+Mizukamiauthor=H.+Miyata&title=Design%2C+synthesis%2C+enzyme-inhibitory+activity%2C+and+effect+on+human+cancer+cells+of+a+novel+series+of+Jumonji+domain-containing+protein+2+histone+demethylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Design, Synthesis, Enzyme-Inhibitory Activity, and Effect on Human Cancer Cells of a Novel Series of Jumonji Domain-Containing Protein 2 Histone Demethylase Inhibitors</span></div><div class="casAuthors">Hamada, Shohei; Suzuki, Takayoshi; Mino, Koshiki; Koseki, Koichi; Oehme, Felix; Flamme, Ingo; Ozasa, Hiroki; Itoh, Yukihiro; Ogasawara, Daisuke; Komaarashi, Haruka; Kato, Aiko; Tsumoto, Hiroki; Nakagawa, Hidehiko; Hasegawa, Makoto; Sasaki, Ryuzo; Mizukami, Tamio; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">5629-5638</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Selective inhibitors of Jumonji domain-contg. protein (JMJD) histone demethylases are candidate anticancer agents as well as potential tools for elucidating the biol. functions of JMJDs.  On the basis of the crystal structure of JMJD2A and a homol. model of JMJD2C, we designed and prepd. a series of hydroxamate analogs bearing a tertiary amine.  Enzyme assays using JMJD2C, JMJD2A, and prolyl hydroxylases revealed that hydroxamate analog 8 is a potent and selective JMJD2 inhibitor, showing 500-fold greater JMJD2C-inhibitory activity and more than 9100-fold greater JMJD2C-selectivity compared with the lead compd. N-oxalylglycine 2.  Compds. 17 and 18, prodrugs of compd. 8, each showed synergistic growth inhibition of cancer cells in combination with an inhibitor of lysine-specific demethylase 1 (LSD1).  These findings suggest that combination treatment with JMJD2 inhibitors and LSD1 inhibitors may represent a novel strategy for anticancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT6G3YdmcF_bVg90H21EOLACvtfcHk0lg29pLI-stYvA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovVCis7o%253D&md5=f4ab3a75cb995d88668dd8527f554ea0</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Fjm1003655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1003655%26sid%3Dliteratum%253Aachs%26aulast%3DHamada%26aufirst%3DS.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMino%26aufirst%3DK.%26aulast%3DKoseki%26aufirst%3DK.%26aulast%3DOehme%26aufirst%3DF.%26aulast%3DFlamme%26aufirst%3DI.%26aulast%3DOzasa%26aufirst%3DH.%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DOgasawara%26aufirst%3DD.%26aulast%3DKomaarashi%26aufirst%3DH.%26aulast%3DKato%26aufirst%3DA.%26aulast%3DTsumoto%26aufirst%3DH.%26aulast%3DNakagawa%26aufirst%3DM.%26aulast%3DHasegawa%26aufirst%3DR.%26aulast%3DSasaki%26aufirst%3DT.%26aulast%3DMizukami%26aufirst%3DN.%26aulast%3DMiyata%26aufirst%3DH.%26atitle%3DDesign%252C%2520synthesis%252C%2520enzyme-inhibitory%2520activity%252C%2520and%2520effect%2520on%2520human%2520cancer%2520cells%2520of%2520a%2520novel%2520series%2520of%2520Jumonji%2520domain-containing%2520protein%25202%2520histone%2520demethylase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D5629%26epage%3D5638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Sams-Dodd, F.</span><span> </span><span class="NLM_article-title">Target-based drug discovery: is something wrong?</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">139</span><span class="NLM_x">–</span> <span class="NLM_lpage">147</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=139-147&author=F.+Sams-Dodd&title=Target-based+drug+discovery%3A+is+something+wrong%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSams-Dodd%26aufirst%3DF.%26atitle%3DTarget-based%2520drug%2520discovery%253A%2520is%2520something%2520wrong%253F%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2005%26volume%3D10%26spage%3D139%26epage%3D147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Keith, C. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borisy, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, B. R.</span><span> </span><span class="NLM_article-title">Multicomponent therapeutics for networked systems</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2005&pages=461-467&author=C.+T.+Keithauthor=A.+A.+Borisyauthor=B.+R.+Stockwell&title=Multicomponent+therapeutics+for+networked+systems"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKeith%26aufirst%3DC.%2BT.%26aulast%3DBorisy%26aufirst%3DA.%2BA.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DMulticomponent%2520therapeutics%2520for%2520networked%2520systems%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2005%26volume%3D4%26spage%3D461%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Kamb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span> </span><span class="NLM_article-title">Why is cancer drug discovery so difficult?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=115-120&author=A.+Kambauthor=S.+Weeauthor=C.+Lengauer&title=Why+is+cancer+drug+discovery+so+difficult%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26aulast%3DWee%26aufirst%3DS.%26aulast%3DLengauer%26aufirst%3DC.%26atitle%3DWhy%2520is%2520cancer%2520drug%2520discovery%2520so%2520difficult%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2007%26volume%3D6%26spage%3D115%26epage%3D120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Daub, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Specht, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullrich, A.</span><span> </span><span class="NLM_article-title">Strategies to overcome resistance to targeted protein kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1001</span><span class="NLM_x">–</span> <span class="NLM_lpage">1010</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnrd1579" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=15573099" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVarsL3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2004&pages=1001-1010&author=H.+Daubauthor=K.+Spechtauthor=A.+Ullrich&title=Strategies+to+overcome+resistance+to+targeted+protein+kinase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies to overcome resistance to targeted protein kinase inhibitors</span></div><div class="casAuthors">Daub, Henrik; Specht, Katja; Ullrich, Axel</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1001-1010</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Selective inhibition of protein tyrosine kinases is gaining importance as an effective therapeutic approach for the treatment of a wide range of human cancers.  However, as extensively documented for the BCR-ABL oncogene in imatinib-treated leukemia patients, clin. resistance caused by mutations in the targeted oncogene has been obsd.  Here, we look at how structural and mechanistic insights from imatinib-insensitive Bcr-Abl have been exploited to identify second-generation drugs that override acquired target resistance.  These insights have created a rationale for the development of either multi-targeted protein kinase inhibitors or cocktails of selective antagonists as antitumor drugs that combine increased therapeutic potency with a reduced risk of the emergence of mol. resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_U6mWwYG307Vg90H21EOLACvtfcHk0lgUJThDdKvMzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVarsL3K&md5=49c2ce8b0150e562e382d5246b3c0a42</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrd1579&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd1579%26sid%3Dliteratum%253Aachs%26aulast%3DDaub%26aufirst%3DH.%26aulast%3DSpecht%26aufirst%3DK.%26aulast%3DUllrich%26aufirst%3DA.%26atitle%3DStrategies%2520to%2520overcome%2520resistance%2520to%2520targeted%2520protein%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2004%26volume%3D3%26spage%3D1001%26epage%3D1010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Rather, M. A.; Bhat, B. A.; Qurishi, M. A.</span><span> </span><span class="NLM_article-title">Multicomponent phytotherapeutic approach gaining momentum: Is the “one drug to fit all” model breaking down?</span> <span class="citation_source-journal">Phytomedicine</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span><a href="http://dx.doi.org/10.1016/j.phymed.2013.07.015" class="extLink">http://dx.doi.org/10.1016/j.phymed.2013.07.015</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=M.+A.+Rather&author=B.+A.+Bhat&author=M.+A.+Qurishi&title=Multicomponent+phytotherapeutic+approach+gaining+momentum%3A+Is+the+%E2%80%9Cone+drug+to+fit+all%E2%80%9D+model+breaking+down%3F"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRather%26aufirst%3DM.%2BA.%26atitle%3DMulticomponent%2520phytotherapeutic%2520approach%2520gaining%2520momentum%253A%2520Is%2520the%2520%25E2%2580%259Cone%2520drug%2520to%2520fit%2520all%25E2%2580%259D%2520model%2520breaking%2520down%253F%26jtitle%3DPhytomedicine%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Morphy, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rankovic, Z.</span><span> </span><span class="NLM_article-title">Designed multiple ligands. An emerging drug discovery paradigm</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">48</span><span class="NLM_x">, </span> <span class="NLM_fpage">6523</span><span class="NLM_x">–</span> <span class="NLM_lpage">6543</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm058225d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=48&publication_year=2005&pages=6523-6543&author=R.+Morphyauthor=Z.+Rankovic&title=Designed+multiple+ligands.+An+emerging+drug+discovery+paradigm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Designed Multiple Ligands.</span></div><div class="casAuthors">Morphy, Richard; Rankovic, Zoran</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">48</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6523-6543</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The review discusses main areas in designed multiple ligands (DMLs) discovery that include 5HT transporter-based DMLs for depression, dopamine D2-receptor based DMLs for schizophrenia, histamine H1 receptor-based DMLs for allergies, DMLs targeting the angiotensin system for hypertension, nonsteroidal anti-inflammatory drugs, and PPAR-based DMLs for metabolic disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgouWcjsiKgbVg90H21EOLACvtfcHk0lgUJThDdKvMzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVeisr%252FL&md5=00e3bf5a521fc91996ce95bb379a0b5c</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fjm058225d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm058225d%26sid%3Dliteratum%253Aachs%26aulast%3DMorphy%26aufirst%3DR.%26aulast%3DRankovic%26aufirst%3DZ.%26atitle%3DDesigned%2520multiple%2520ligands.%2520An%2520emerging%2520drug%2520discovery%2520paradigm%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2005%26volume%3D48%26spage%3D6523%26epage%3D6543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Zimmerman, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehàr, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keith, C. T.</span><span> </span><span class="NLM_article-title">Multi-target therapeutics: When the whole is greater than the sum of the parts</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">34</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2007&pages=34-42&author=G.+R.+Zimmermanauthor=J.+Leh%C3%A0rauthor=C.+T.+Keith&title=Multi-target+therapeutics%3A+When+the+whole+is+greater+than+the+sum+of+the+parts"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZimmerman%26aufirst%3DG.%2BR.%26aulast%3DLeh%25C3%25A0r%26aufirst%3DJ.%26aulast%3DKeith%26aufirst%3DC.%2BT.%26atitle%3DMulti-target%2520therapeutics%253A%2520When%2520the%2520whole%2520is%2520greater%2520than%2520the%2520sum%2520of%2520the%2520parts%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2007%26volume%3D12%26spage%3D34%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Fortin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bérubé, G.</span><span> </span><span class="NLM_article-title">Advances in the development of hybrid anticancer drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1029</span><span class="NLM_x">–</span> <span class="NLM_lpage">1047</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1517%2F17460441.2013.798296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=23646979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOitL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1029-1047&author=S.+Fortinauthor=G.+B%C3%A9rub%C3%A9&title=Advances+in+the+development+of+hybrid+anticancer+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in the development of hybrid anticancer drugs</span></div><div class="casAuthors">Fortin, Sebastien; Berube, Gervais</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1029-1047</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: Hybrid anticancer drugs are of great therapeutic interests as they can potentially overcome most of the pharmacokinetic drawbacks encountered when using conventional anticancer drugs.  In fact, the future of hybrid anticancer drugs is very bright for the discovery of highly potent and selective mols. that triggers two or more cytocidal pharmacol. mechanisms of action acting in synergy to inhibit cancer tumor growth.  Areas covered: This review represents the most advanced and recent data in the field of hybrid anticancer agents covering mainly the past 5 years of research.  It also accounts for other significant reviews already published on the topic of anticancer hybrids.  The review showcases the research that is at the leading edge of hybrid anticancer drug discovery.  The main areas covered by the present review are: DNA alkylating agent hybrids (e.g., platinum(II), nitrogen mustard, etc.), vitamin-D receptor, agonist-histone deacetylase inhibitors, combi-mol. therapies and other types of hybrid anticancer agents.  Expert opinion: The current development in the field describes strategies that have never been used before for the design of hybrid anticancer drugs.  The information currently available and described in this section allows us to identify the main parameters required to design such mols.  It also provides a clear view of the future directions that must be explored for the successful development and discovery of useful hybrid anticancer drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1FoQV3stg57Vg90H21EOLACvtfcHk0liHaqqbrwBcKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOitL7E&md5=d5157ee2d3cc8778f9129b051bf52e0d</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1517%2F17460441.2013.798296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2013.798296%26sid%3Dliteratum%253Aachs%26aulast%3DFortin%26aufirst%3DS.%26aulast%3DB%25C3%25A9rub%25C3%25A9%26aufirst%3DG.%26atitle%3DAdvances%2520in%2520the%2520development%2520of%2520hybrid%2520anticancer%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2013%26volume%3D8%26spage%3D1029%26epage%3D1047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: mixed histone/protein methyltransferase, acetyltransferase, and class III deacetylase (sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+mixed+histone%2Fprotein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28sirtuin%29+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520mixed%2520histone%252Fprotein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Binda, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanenghi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cirilli, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karytinos, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciossani, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Botrugno, O. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forneris, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tardugno, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edmondson, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minucci, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattevi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Biochemical, structural, and biological evaluation of tranylcypromine derivatives as inhibitors of histone demethylases LSD1 and LSD2</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">6827</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja101557k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=6827-6833&author=C.+Bindaauthor=S.+Valenteauthor=M.+Romanenghiauthor=S.+Pilottoauthor=R.+Cirilliauthor=A.+Karytinosauthor=G.+Ciossaniauthor=O.+A.+Botrugnoauthor=F.+Fornerisauthor=M.+Tardugnoauthor=D.+E.+Edmondsonauthor=S.+Minucciauthor=A.+Matteviauthor=A.+Mai&title=Biochemical%2C+structural%2C+and+biological+evaluation+of+tranylcypromine+derivatives+as+inhibitors+of+histone+demethylases+LSD1+and+LSD2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fja101557k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101557k%26sid%3Dliteratum%253Aachs%26aulast%3DBinda%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DRomanenghi%26aufirst%3DM.%26aulast%3DPilotto%26aufirst%3DS.%26aulast%3DCirilli%26aufirst%3DR.%26aulast%3DKarytinos%26aufirst%3DA.%26aulast%3DCiossani%26aufirst%3DG.%26aulast%3DBotrugno%26aufirst%3DO.%2BA.%26aulast%3DForneris%26aufirst%3DF.%26aulast%3DTardugno%26aufirst%3DM.%26aulast%3DEdmondson%26aufirst%3DD.%2BE.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DMattevi%26aufirst%3DA.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DBiochemical%252C%2520structural%252C%2520and%2520biological%2520evaluation%2520of%2520tranylcypromine%2520derivatives%2520as%2520inhibitors%2520of%2520histone%2520demethylases%2520LSD1%2520and%2520LSD2%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D6827%26epage%3D6833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Upadhyay, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labella, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, Y. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span> </span><span class="NLM_article-title">An analog of BIX-01294 selectively inhibits a family of histone H3 lysine 9 Jumonji demethylases</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">416</span><span class="NLM_x">, </span> <span class="NLM_fpage">319</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=416&publication_year=2012&pages=319-327&author=K.+Upadhyayauthor=D.+Rotiliauthor=J.+W.+Hanauthor=R.+Huauthor=Y.+Changauthor=D.+Labellaauthor=X.+Zhangauthor=Y.+S.+Yoonauthor=A.+Maiauthor=X.+Cheng&title=An+analog+of+BIX-01294+selectively+inhibits+a+family+of+histone+H3+lysine+9+Jumonji+demethylases"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DUpadhyay%26aufirst%3DK.%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DHan%26aufirst%3DJ.%2BW.%26aulast%3DHu%26aufirst%3DR.%26aulast%3DChang%26aufirst%3DY.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DYoon%26aufirst%3DY.%2BS.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DX.%26atitle%3DAn%2520analog%2520of%2520BIX-01294%2520selectively%2520inhibits%2520a%2520family%2520of%2520histone%2520H3%2520lysine%25209%2520Jumonji%2520demethylases%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2012%26volume%3D416%26spage%3D319%26epage%3D327" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Rotili, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altun, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackeen, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, Y. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">A photoreactive small-molecule probe for 2-oxoglutarate oxygenases</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">642</span><span class="NLM_x">–</span> <span class="NLM_lpage">654</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.chembiol.2011.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=21609845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmslOksLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2011&pages=642-654&author=D.+Rotiliauthor=M.+Altunauthor=A.+Kawamuraauthor=A.+Wolfauthor=R.+Fischerauthor=I.+K.+Leungauthor=M.+M.+Mackeenauthor=Y.+M.+Tianauthor=P.+J.+Ratcliffeauthor=A.+Maiauthor=B.+M.+Kesslerauthor=C.+J.+Schofield&title=A+photoreactive+small-molecule+probe+for+2-oxoglutarate+oxygenases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A Photoreactive Small-Molecule Probe for 2-Oxoglutarate Oxygenases</span></div><div class="casAuthors">Rotili, Dante; Altun, Mikael; Kawamura, Akane; Wolf, Alexander; Fischer, Roman; Leung, Ivanhoe K. H.; MacKeen, Mukram M.; Tian, Ya-min; Ratcliffe, Peter J.; Mai, Antonello; Kessler, Benedikt M.; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">642-654</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">2-Oxoglutarate (2-OG)-dependent oxygenases have diverse roles in human biol.  The inhibition of several 2-OG oxygenases is being targeted for therapeutic intervention, including for cancer, anemia, and ischemic diseases.  We report a small-mol. probe for 2-OG oxygenases that employs a hydroxyquinoline template coupled to a photoactivable crosslinking group and an affinity-purifn. tag.  Following studies with recombinant proteins, the probe was shown to crosslink to 2-OG oxygenases in human crude cell exts., including to proteins at endogenous levels.  This approach is useful for inhibitor profiling, as demonstrated by crosslinking to the histone demethylase FBXL11 (KDM2A) in HEK293T nuclear exts.  The results also suggest that small-mol. probes may be suitable for substrate identification studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq76D2hRKWJjbVg90H21EOLACvtfcHk0liHaqqbrwBcKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmslOksLk%253D&md5=cd8f7d35948c618d6b21e9c62602b90c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2011.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2011.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRotili%26aufirst%3DD.%26aulast%3DAltun%26aufirst%3DM.%26aulast%3DKawamura%26aufirst%3DA.%26aulast%3DWolf%26aufirst%3DA.%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DMackeen%26aufirst%3DM.%2BM.%26aulast%3DTian%26aufirst%3DY.%2BM.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DA%2520photoreactive%2520small-molecule%2520probe%2520for%25202-oxoglutarate%2520oxygenases%26jtitle%3DChem.%2520Biol.%26date%3D2011%26volume%3D18%26spage%3D642%26epage%3D654" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mecinovic, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bello, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperman, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span> </span><span class="NLM_article-title">Inhibitor scaffolds for 2-oxoglutarate-dependent histone lysine demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">7053</span><span class="NLM_x">–</span> <span class="NLM_lpage">7056</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm800936s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=7053-7056&author=N.+R.+Roseauthor=S.+S.+Ngauthor=J.+Mecinovicauthor=B.+M.+R.+Lienardauthor=S.+H.+Belloauthor=Z.+Sunauthor=M.+A.+McDonoughauthor=U.+Oppermanauthor=C.+J.+Schofield&title=Inhibitor+scaffolds+for+2-oxoglutarate-dependent+histone+lysine+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitor Scaffolds for 2-Oxoglutarate-Dependent Histone Lysine Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Ng, Stanley S.; Mecinovic, Jasmin; Lienard, Benoit M. R.; Bello, Simon H.; Sun, Zhe; McDonough, Michael A.; Oppermann, Udo; Schofield, Christopher J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7053-7056</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamic methylation of histone lysyl residues plays an important role in biol. by regulating transcription, maintaining genomic integrity, and by contributing to epigenetic effects.  Here we describe a variety of inhibitor scaffolds that inhibit the human 2-oxoglutarate-dependent JMJD2 subfamily of histone demethylases.  Combined with structural data, these chem. starting points will be useful to generate small-mol. probes to analyze the physiol. roles of these enzymes in epigenetic signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr90HQ8nfBLFrVg90H21EOLACvtfcHk0lj-2cRrUXEvsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Ors7rI&md5=155c830b0544a6c1d5e6eb81d14389de</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm800936s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm800936s%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DMecinovic%26aufirst%3DJ.%26aulast%3DLienard%26aufirst%3DB.%2BM.%2BR.%26aulast%3DBello%26aufirst%3DS.%2BH.%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOpperman%26aufirst%3DU.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26atitle%3DInhibitor%2520scaffolds%2520for%25202-oxoglutarate-dependent%2520histone%2520lysine%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D7053%26epage%3D7056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yapp, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aik, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kristensen, J. B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thinnes, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bush, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bagg, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, W. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olsen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Opperman, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span> <span class="citation_source-journal">Chem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">3110</span><span class="NLM_x">–</span> <span class="NLM_lpage">3117</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1039%2Fc3sc51122g" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=3110-3117&author=R.+J.+Hopkinsonauthor=A.+Tumberauthor=C.+Yappauthor=R.+Chowdhuryauthor=W.+Aikauthor=K.+H.+Cheauthor=X.+S.+Liauthor=J.+B.+L.+Kristensenauthor=O.+N.+F.+Kingauthor=M.+C.+Chanauthor=K.+K.+Yeohauthor=H.+Choiauthor=L.+J.+Walportauthor=C.+C.+Thinnesauthor=J.+T.+Bushauthor=C.+Lejeuneauthor=A.+M.+Rydzikauthor=N.+R.+Roseauthor=E.+A.+Baggauthor=M.+A.+McDonoughauthor=T.+J.+Krojerauthor=W.+W.+Yueauthor=S.+S.+Ngauthor=L.+Olsenauthor=E.+S.+Butlerauthor=U.+Oppermanauthor=S.+Mullerauthor=R.+J.+Kloseauthor=P.+J.+Ratcliffeauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=5-Carboxy-8-hydroxyquinoline+is+a+broad+spectrum+2-oxoglutarate+oxygenase+inhibitor+which+causes+iron+translocation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">5-Carboxy-8-hydroxyquinoline is a broad spectrum 2-oxoglutarate oxygenase inhibitor which causes iron translocation</span></div><div class="casAuthors">Hopkinson, Richard J.; Tumber, Anthony; Yapp, Clarence; Chowdhury, Rasheduzzaman; Aik, WeiShen; Che, Ka Hing; Li, Xuan Shirley; Kristensen, Jan B. L.; King, Oliver N. F.; Chan, Mun Chiang; Yeoh, Kar Kheng; Choi, Hwanho; Walport, Louise J.; Thinnes, Cyrille C.; Bush, Jacob T.; Lejeune, Clarisse; Rydzik, Anna M.; Rose, Nathan R.; Bagg, Eleanor A.; McDonough, Michael A.; Krojer, Tobias J.; Yue, Wyatt W.; Ng, Stanley S.; Olsen, Lars; Brennan, Paul E.; Oppermann, Udo; Mueller, Susanne; Klose, Robert J.; Ratcliffe, Peter J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3110-3117</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">2-Oxoglutarate and iron dependent oxygenases are therapeutic targets for human diseases.  Using a representative 2OG oxygenase panel, the authors compare the inhibitory activities of 5-carboxy-8-hydroxyquinoline (IOX1) and 4-carboxy-8-hydroxyquinoline (4C8HQ) with that of two other commonly used 2OG oxygenase inhibitors, N-oxalylglycine (NOG) and 2,4-pyridinedicarboxylic acid (2,4-PDCA).  IOX1 has a broad spectrum of activity, as demonstrated by the inhibition of transcription factor hydroxylases, representatives of all 2OG dependent histone demethylase subfamilies, nucleic acid demethylases and γ-butyrobetaine hydroxylase.  Cellular assays show that, unlike NOG and 2,4-PDCA, IOX1 is active against both cytosolic and nuclear 2OG oxygenases without ester derivatization.  Unexpectedly, crystallog. studies on these oxygenases demonstrate that IOX1, but not 4C8HQ, can cause translocation of the active site metal, revealing a rare example of protein ligand-induced metal movement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_NBtJvNEBOLVg90H21EOLACvtfcHk0lj-2cRrUXEvsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVehs73L&md5=506fb4e75971c57a7a968f4d37805856</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1039%2Fc3sc51122g&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3sc51122g%26sid%3Dliteratum%253Aachs%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DYapp%26aufirst%3DC.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DAik%26aufirst%3DW.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKristensen%26aufirst%3DJ.%2BB.%2BL.%26aulast%3DKing%26aufirst%3DO.%2BN.%2BF.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DThinnes%26aufirst%3DC.%2BC.%26aulast%3DBush%26aufirst%3DJ.%2BT.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DBagg%26aufirst%3DE.%2BA.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DKrojer%26aufirst%3DT.%2BJ.%26aulast%3DYue%26aufirst%3DW.%2BW.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DOlsen%26aufirst%3DL.%26aulast%3DButler%26aufirst%3DE.%2BS.%26aulast%3DOpperman%26aufirst%3DU.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3D5-Carboxy-8-hydroxyquinoline%2520is%2520a%2520broad%2520spectrum%25202-oxoglutarate%2520oxygenase%2520inhibitor%2520which%2520causes%2520iron%2520translocation%26jtitle%3DChem.%2520Sci.%26date%3D2013%26volume%3D4%26spage%3D3110%26epage%3D3117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kavanagh, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pilka, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lienard, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bray, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savitsky, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Offer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheinost, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borowski, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sundstrom, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span> </span><span class="NLM_article-title">Crystal structures of histone demethylase JMJD2A reveal basis for substrate specificity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">448</span><span class="NLM_x">, </span> <span class="NLM_fpage">87</span><span class="NLM_x">–</span> <span class="NLM_lpage">91</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=87-91&author=S.+S.+Ngauthor=K.+L.+Kavanaghauthor=M.+A.+McDonoughauthor=D.+Butlerauthor=E.+S.+Pilkaauthor=B.+M.+Lienardauthor=J.+E.+Brayauthor=P.+Savitskyauthor=C.+Gileadiauthor=F.+Von+Delftauthor=N.+R.+Roseauthor=J.+Offerauthor=J.+C.+Scheinostauthor=T.+Borowskiauthor=M.+Sundstromauthor=C.+J.+Schofieldauthor=U.+Oppermann&title=Crystal+structures+of+histone+demethylase+JMJD2A+reveal+basis+for+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKavanagh%26aufirst%3DK.%2BL.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DButler%26aufirst%3DD.%26aulast%3DPilka%26aufirst%3DE.%2BS.%26aulast%3DLienard%26aufirst%3DB.%2BM.%26aulast%3DBray%26aufirst%3DJ.%2BE.%26aulast%3DSavitsky%26aufirst%3DP.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DVon%2BDelft%26aufirst%3DF.%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DOffer%26aufirst%3DJ.%26aulast%3DScheinost%26aufirst%3DJ.%2BC.%26aulast%3DBorowski%26aufirst%3DT.%26aulast%3DSundstrom%26aufirst%3DM.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DOppermann%26aufirst%3DU.%26atitle%3DCrystal%2520structures%2520of%2520histone%2520demethylase%2520JMJD2A%2520reveal%2520basis%2520for%2520substrate%2520specificity%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D87%26epage%3D91" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Chang, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rose, N. R.</span><span> </span><span class="NLM_article-title">Inhibition of histone demethylases by 4-carboxy-2,2′-bipyridyl compounds</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">759</span><span class="NLM_x">–</span> <span class="NLM_lpage">764</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=759-764&author=K.+H.+Changauthor=O.+N.+Kingauthor=A.+Tumberauthor=E.+C.+Woonauthor=T.+D.+Heightmanauthor=M.+A.+McDonoughauthor=C.+J.+Schofieldauthor=N.+R.+Rose&title=Inhibition+of+histone+demethylases+by+4-carboxy-2%2C2%E2%80%B2-bipyridyl+compounds"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DK.%2BH.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWoon%26aufirst%3DE.%2BC.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DRose%26aufirst%3DN.%2BR.%26atitle%3DInhibition%2520of%2520histone%2520demethylases%2520by%25204-carboxy-2%252C2%25E2%2580%25B2-bipyridyl%2520compounds%26jtitle%3DChemMedChem%26date%3D2011%26volume%3D6%26spage%3D759%26epage%3D764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Gros, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lorente, S. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jimenez, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yardley, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rattray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wharton, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Little, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Croft, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruiz-Perez, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Pacanowska, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, I. H.</span><span> </span><span class="NLM_article-title">Evaluation of azasterols as anti-parasitics</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6094</span><span class="NLM_x">–</span> <span class="NLM_lpage">6103</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm060290f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6094-6103&author=L.+Grosauthor=S.+O.+Lorenteauthor=C.+J.+Jimenezauthor=V.+Yardleyauthor=L.+Rattrayauthor=H.+Whartonauthor=S.+Littleauthor=S.+L.+Croftauthor=L.+M.+Ruiz-Perezauthor=D.+Gonzalez-Pacanowskaauthor=I.+H.+Gilbert&title=Evaluation+of+azasterols+as+anti-parasitics"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm060290f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm060290f%26sid%3Dliteratum%253Aachs%26aulast%3DGros%26aufirst%3DL.%26aulast%3DLorente%26aufirst%3DS.%2BO.%26aulast%3DJimenez%26aufirst%3DC.%2BJ.%26aulast%3DYardley%26aufirst%3DV.%26aulast%3DRattray%26aufirst%3DL.%26aulast%3DWharton%26aufirst%3DH.%26aulast%3DLittle%26aufirst%3DS.%26aulast%3DCroft%26aufirst%3DS.%2BL.%26aulast%3DRuiz-Perez%26aufirst%3DL.%2BM.%26aulast%3DGonzalez-Pacanowska%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DI.%2BH.%26atitle%3DEvaluation%2520of%2520azasterols%2520as%2520anti-parasitics%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6094%26epage%3D6103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Kerenyi, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">Histone demethylase Lsd1 represses hematopoietic stem and progenitor cell signatures during blood cell maturation</span> <span class="citation_source-journal">Elife</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_elocation-id">e00633</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2013&author=M.+A.+Kerenyiauthor=Z.+Shaoauthor=Y.+J.+Hsuauthor=G.+Guoauthor=S.+Lucauthor=K.+O%E2%80%99Brienauthor=Y.+Fujiwaraauthor=C.+Pengauthor=M.+Nguyenauthor=S.+H.+Orkin&title=Histone+demethylase+Lsd1+represses+hematopoietic+stem+and+progenitor+cell+signatures+during+blood+cell+maturation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKerenyi%26aufirst%3DM.%2BA.%26aulast%3DShao%26aufirst%3DZ.%26aulast%3DHsu%26aufirst%3DY.%2BJ.%26aulast%3DGuo%26aufirst%3DG.%26aulast%3DLuc%26aufirst%3DS.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DK.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DPeng%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DM.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DHistone%2520demethylase%2520Lsd1%2520represses%2520hematopoietic%2520stem%2520and%2520progenitor%2520cell%2520signatures%2520during%2520blood%2520cell%2520maturation%26jtitle%3DElife%26date%3D2013%26volume%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Harris, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lynch, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spencer, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hitchin, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ciceri, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blaser, J. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greystoke, B. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogilvie, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Somervaille, T. C.</span><span> </span><span class="NLM_article-title">The histone demethylase KDM1A sustains the oncogenic potential of MLL-AF9 leukemia stem cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">473</span><span class="NLM_x">–</span> <span class="NLM_lpage">487</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1016%2Fj.ccr.2012.03.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22464800" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=473-487&author=W.+J.+Harrisauthor=X.+Huangauthor=J.+T.+Lynchauthor=G.+J.+Spencerauthor=J.+R.+Hitchinauthor=Y.+Liauthor=F.+Ciceriauthor=J.+G.+Blaserauthor=B.+F.+Greystokeauthor=A.+M.+Jordanauthor=C.+J.+Millerauthor=D.+J.+Ogilvieauthor=T.+C.+Somervaille&title=The+histone+demethylase+KDM1A+sustains+the+oncogenic+potential+of+MLL-AF9+leukemia+stem+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Histone Demethylase KDM1A Sustains the Oncogenic Potential of MLL-AF9 Leukemia Stem Cells</span></div><div class="casAuthors">Harris, William J.; Huang, Xu; Lynch, James T.; Spencer, Gary J.; Hitchin, James R.; Li, Yaoyong; Ciceri, Filippo; Blaser, Julian G.; Greystoke, Brigit F.; Jordan, Allan M.; Miller, Crispin J.; Ogilvie, Donald J.; Somervaille, Tim C. P.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">473-487</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Using a mouse model of human MLL-AF9 leukemia, we identified the lysine-specific demethylase KDM1A (LSD1 or AOF2) as an essential regulator of leukemia stem cell (LSC) potential.  KDM1A acts at genomic loci bound by MLL-AF9 to sustain expression of the assocd. oncogenic program, thus preventing differentiation and apoptosis.  In vitro and in vivo pharmacol. targeting of KDM1A using tranylcypromine analogs active in the nanomolar range phenocopied Kdm1a knockdown in both murine and primary human AML cells exhibiting MLL translocations.  By contrast, the clonogenic and repopulating potential of normal hematopoietic stem and progenitor cells was spared.  These data establish KDM1A as a key effector of the differentiation block in MLL leukemia, which may be selectively targeted to therapeutic effect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiZ65foej58rVg90H21EOLACvtfcHk0liq1zN8KBxpEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyqsrc%253D&md5=3227f2ac24a884a8ebadbce71198830c</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2012.03.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2012.03.014%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DW.%2BJ.%26aulast%3DHuang%26aufirst%3DX.%26aulast%3DLynch%26aufirst%3DJ.%2BT.%26aulast%3DSpencer%26aufirst%3DG.%2BJ.%26aulast%3DHitchin%26aufirst%3DJ.%2BR.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DCiceri%26aufirst%3DF.%26aulast%3DBlaser%26aufirst%3DJ.%2BG.%26aulast%3DGreystoke%26aufirst%3DB.%2BF.%26aulast%3DJordan%26aufirst%3DA.%2BM.%26aulast%3DMiller%26aufirst%3DC.%2BJ.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DSomervaille%26aufirst%3DT.%2BC.%26atitle%3DThe%2520histone%2520demethylase%2520KDM1A%2520sustains%2520the%2520oncogenic%2520potential%2520of%2520MLL-AF9%2520leukemia%2520stem%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2012%26volume%3D21%26spage%3D473%26epage%3D487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Schenk, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Göllner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hebestreit, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klein, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Popescu, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burnett, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mills, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Casero, R. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marton, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minden, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dugas, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dick, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Müller-Tidow, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zelent, A.</span><span> </span><span class="NLM_article-title">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">605</span><span class="NLM_x">–</span> <span class="NLM_lpage">611</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnm.2661" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22406747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=605-611&author=T.+Schenkauthor=W.+C.+Chenauthor=S.+G%C3%B6llnerauthor=L.+Howellauthor=L.+Jinauthor=K.+Hebestreitauthor=H.+U.+Kleinauthor=A.+C.+Popescuauthor=A.+Burnettauthor=K.+Millsauthor=R.+A.+Caseroauthor=L.+Martonauthor=P.+Wosterauthor=M.+D.+Mindenauthor=M.+Dugasauthor=J.+C.+Wangauthor=J.+E.+Dickauthor=C.+M%C3%BCller-Tidowauthor=K.+Petrieauthor=A.+Zelent&title=Inhibition+of+the+LSD1+%28KDM1A%29+demethylase+reactivates+the+all-trans-retinoic+acid+differentiation+pathway+in+acute+myeloid+leukemia"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of the LSD1 (KDM1A) demethylase reactivates the all-trans-retinoic acid differentiation pathway in acute myeloid leukemia</span></div><div class="casAuthors">Schenk, Tino; Chen, Weihsu Claire; Goellner, Stefanie; Howell, Louise; Jin, Liqing; Hebestreit, Katja; Klein, Hans-Ulrich; Popescu, Andreea C.; Burnett, Alan; Mills, Ken; Casero, Robert A.; Marton, Laurence; Woster, Patrick; Minden, Mark D.; Dugas, Martin; Wang, Jean C. Y.; Dick, John E.; Mueller-Tidow, Carsten; Petrie, Kevin; Zelent, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">605-611</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Acute promyelocytic leukemia (APL), a cytogenetically distinct subtype of acute myeloid leukemia (AML), characterized by the t(15;17)-assocd. PML-RARA fusion, has been successfully treated with therapy utilizing all-trans-retinoic acid (ATRA) to differentiate leukemic blasts.  However, among patients with non-APL AML, ATRA-based treatment has not been effective.  Here we show that, through epigenetic reprogramming, inhibitors of lysine-specific demethylase 1 (LSD1, also called KDM1A), including tranylcypromine (TCP), unlocked the ATRA-driven therapeutic response in non-APL AML.  LSD1 inhibition did not lead to a large-scale increase in histone 3 Lys4 dimethylation (H3K4me2) across the genome, but it did increase H3K4me2 and expression of myeloid-differentiation-assocd. genes.  Notably, treatment with ATRA plus TCP markedly diminished the engraftment of primary human AML cells in vivo in nonobese diabetic (NOD)-severe combined immunodeficient (SCID) mice, suggesting that ATRA in combination with TCP may target leukemia-initiating cells.  Furthermore, initiation of ATRA plus TCP treatment 15 d after engraftment of human AML cells in NOD-SCID γ (with interleukin-2 (IL-2) receptor γ chain deficiency) mice also revealed the ATRA plus TCP drug combination to have a potent anti-leukemic effect that was superior to treatment with either drug alone.  These data identify LSD1 as a therapeutic target and strongly suggest that it may contribute to AML pathogenesis by inhibiting the normal pro-differentiative function of ATRA, paving the way for new combinatorial therapies for AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpU7ebz8Ar4AbVg90H21EOLACvtfcHk0liq1zN8KBxpEQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1WhtL8%253D&md5=57d968de189123149a2714df0208725f</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1038%2Fnm.2661&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2661%26sid%3Dliteratum%253Aachs%26aulast%3DSchenk%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DG%25C3%25B6llner%26aufirst%3DS.%26aulast%3DHowell%26aufirst%3DL.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DHebestreit%26aufirst%3DK.%26aulast%3DKlein%26aufirst%3DH.%2BU.%26aulast%3DPopescu%26aufirst%3DA.%2BC.%26aulast%3DBurnett%26aufirst%3DA.%26aulast%3DMills%26aufirst%3DK.%26aulast%3DCasero%26aufirst%3DR.%2BA.%26aulast%3DMarton%26aufirst%3DL.%26aulast%3DWoster%26aufirst%3DP.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDugas%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DJ.%2BC.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DM%25C3%25BCller-Tidow%26aufirst%3DC.%26aulast%3DPetrie%26aufirst%3DK.%26aulast%3DZelent%26aufirst%3DA.%26atitle%3DInhibition%2520of%2520the%2520LSD1%2520%2528KDM1A%2529%2520demethylase%2520reactivates%2520the%2520all-trans-retinoic%2520acid%2520differentiation%2520pathway%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DNat.%2520Med.%26date%3D2012%26volume%3D18%26spage%3D605%26epage%3D611" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dell’Aversana, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ricciardiello, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petraglia, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carissimo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perone, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maruotti, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Savarese, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinelli, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cancemi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">MePR: a novel human mesenchymal progenitor model with characteristics of pluripotency</span> <span class="citation_source-journal">Stem Cells Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2368</span><span class="NLM_x">–</span> <span class="NLM_lpage">2383</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2013&pages=2368-2383&author=M.+Miceliauthor=G.+Franciauthor=C.+Dell%E2%80%99Aversanaauthor=F.+Ricciardielloauthor=F.+Petragliaauthor=A.+Carissimoauthor=L.+Peroneauthor=G.+M.+Maruottiauthor=M.+Savareseauthor=P.+Martinelliauthor=M.+Cancemiauthor=L.+Altucci&title=MePR%3A+a+novel+human+mesenchymal+progenitor+model+with+characteristics+of+pluripotency"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DFranci%26aufirst%3DG.%26aulast%3DDell%25E2%2580%2599Aversana%26aufirst%3DC.%26aulast%3DRicciardiello%26aufirst%3DF.%26aulast%3DPetraglia%26aufirst%3DF.%26aulast%3DCarissimo%26aufirst%3DA.%26aulast%3DPerone%26aufirst%3DL.%26aulast%3DMaruotti%26aufirst%3DG.%2BM.%26aulast%3DSavarese%26aufirst%3DM.%26aulast%3DMartinelli%26aufirst%3DP.%26aulast%3DCancemi%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DMePR%253A%2520a%2520novel%2520human%2520mesenchymal%2520progenitor%2520model%2520with%2520characteristics%2520of%2520pluripotency%26jtitle%3DStem%2520Cells%2520Dev.%26date%3D2013%26volume%3D22%26spage%3D2368%26epage%3D2383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Kalgutkar, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D. K.</span><span> </span><span class="NLM_article-title">Drug discovery for a new generation of covalent drugs</span> <span class="citation_source-journal">Expert Opin. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">561</span><span class="NLM_x">–</span> <span class="NLM_lpage">581</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1517%2F17460441.2012.688744" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=22607458" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2012&pages=561-581&author=A.+S.+Kalgutkarauthor=D.+K.+Dalvie&title=Drug+discovery+for+a+new+generation+of+covalent+drugs"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Drug discovery for a new generation of covalent drugs</span></div><div class="casAuthors">Kalgutkar, Amit S.; Dalvie, Deepak K.</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">561-581</span>CODEN:
                <span class="NLM_cas:coden">EODDBX</span>;
        ISSN:<span class="NLM_cas:issn">1746-0441</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Introduction: The design of target-specific covalent inhibitors is conceptually attractive because of increased biochem. efficiency through covalency and increased duration of action that outlasts the pharmacokinetics of the agent.  Although many covalent inhibitors have been approved or are in advanced clin. trials to treat indications such as cancer and hepatitis C, there is a general tendency to avoid them as drug candidates because of concerns regarding immune-mediated toxicity that can arise from indiscriminate reactivity with off-target proteins.Areas covered: The review examines potential reason(s) for the excellent safety record of marketed covalent agents and advanced clin. candidates for emerging therapeutic targets.  A significant emphasis is placed on proteomic techniques and chem./biochem. reactivity assays that aim to provide a systematic rank ordering of pharmacol. selectivity relative to off-target protein reactivity of covalent inhibitors.Expert opinion: While tactics to examine selective covalent modification of the pharmacol. target are broadly applicable in drug discovery, it is unclear whether the output from such studies can prospectively predict idiosyncratic immune-mediated drug toxicity.  Opinions regarding an acceptable threshold of protein reactivity/body burden for a toxic electrophile and a non-toxic electrophilic covalent drug have not been defined.  Increasing confidence in proteomic and chem./biochem. reactivity screens will require a retrospective side-by-side profiling of marketed covalent drugs and electrophiles known to cause deleterious toxic effects via non-selective covalent binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowmXy20mDTPbVg90H21EOLACvtfcHk0lhMgHQBwcKedw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFWmt7k%253D&md5=b7fc8e60b263dda5765dd8380f52df17</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1517%2F17460441.2012.688744&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F17460441.2012.688744%26sid%3Dliteratum%253Aachs%26aulast%3DKalgutkar%26aufirst%3DA.%2BS.%26aulast%3DDalvie%26aufirst%3DD.%2BK.%26atitle%3DDrug%2520discovery%2520for%2520a%2520new%2520generation%2520of%2520covalent%2520drugs%26jtitle%3DExpert%2520Opin.%2520Drug%2520Discovery%26date%3D2012%26volume%3D7%26spage%3D561%26epage%3D581" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Marks, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Breslow, R.</span><span> </span><span class="NLM_article-title">Dimethyl sulfoxide to vorinostat: Development of this histone deacetylase inhibitor as an anticancer drug</span> <span class="citation_source-journal">Nat. Biotechnol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">84</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm4012802&amp;key=10.1038%2Fnbt1272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm4012802&amp;key=17211407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2007&pages=84-90&author=P.+A.+Marksauthor=R.+Breslow&title=Dimethyl+sulfoxide+to+vorinostat%3A+Development+of+this+histone+deacetylase+inhibitor+as+an+anticancer+drug"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Dimethyl sulfoxide to vorinostat: development of this histone deacetylase inhibitor as an anticancer drug</span></div><div class="casAuthors">Marks, Paul A.; Breslow, Ronald</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">84-90</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In our quest to understand why DMSO (DMSO) can cause growth arrest and terminal differentiation of transformed cells, we followed a path that led us to discover suberoylanilide hydroxamic acid (SAHA; vorinostat (Zolinza)), which is a histone deacetylase inhibitor.  SAHA reacts with and blocks the catalytic site of these enzymes.  Extensive structure-activity studies were done along the path from DMSO to SAHA.  SAHA can cause growth arrest and death of a broad variety of transformed cells both in vitro and in tumor-bearing animals at concns. not toxic to normal cells.  SAHA has many protein targets whose structure and function are altered by acetylation, including chromatin-assocd. histones, nonhistone gene transcription factors and proteins involved in regulation of cell proliferation, migration and death.  In clin. trials, SAHA has shown significant anticancer activity against both hematol. and solid tumors at doses well tolerated by patients.  A new drug application was approved by the US Food and Drug Administration for vorinostat for treatment of cutaneous T-cell lymphoma.  More potent analogs of SAHA have shown unacceptable toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoK3endE54rQ7Vg90H21EOLACvtfcHk0lhMgHQBwcKedw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXis1GrsQ%253D%253D&md5=cc2a39d80d5d5bea0a1377bdf48a957d</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1038%2Fnbt1272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1272%26sid%3Dliteratum%253Aachs%26aulast%3DMarks%26aufirst%3DP.%2BA.%26aulast%3DBreslow%26aufirst%3DR.%26atitle%3DDimethyl%2520sulfoxide%2520to%2520vorinostat%253A%2520Development%2520of%2520this%2520histone%2520deacetylase%2520inhibitor%2520as%2520an%2520anticancer%2520drug%26jtitle%3DNat.%2520Biotechnol.%26date%3D2007%26volume%3D25%26spage%3D84%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Rose, N. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woon, E. C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tumber, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walport, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">King, O. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lejeune, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ng, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krojer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rydzik, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopkinson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Che, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daniel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strain-Damerell, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gileadi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kochan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leung, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunford, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeoh, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ratcliffe, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess-Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Delft, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsden, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDonough, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klose, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawamura, A.</span><span> </span><span class="NLM_article-title">Plant growth regulator daminozide is a selective inhibitor of human KDM2/7 histone demethylases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6639</span><span class="NLM_x">–</span> <span class="NLM_lpage">6643</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm300677j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Fjm4012802&amp;key=1%3ACAS%3A528%3ADC%252BC38XptFSrur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6639-6643&author=N.+R.+Roseauthor=E.+C.+Y.+Woonauthor=A.+Tumberauthor=L.+J.+Walportauthor=R.+Chowdhuryauthor=X.+S.+Liauthor=O.+N.+Kingauthor=C.+Lejeuneauthor=S.+S.+Ngauthor=T.+Krojerauthor=M.+C.+Chanauthor=A.+M.+Rydzikauthor=R.+J.+Hopkinsonauthor=K.+H.+Cheauthor=M.+Danielauthor=C.+Strain-Damerellauthor=C.+Gileadiauthor=G.+Kochanauthor=I.+K.+Leungauthor=J.+Dunfordauthor=K.+K.+Yeohauthor=P.+J.+Ratcliffeauthor=N.+Burgess-Brownauthor=F.+von+Delftauthor=S.+Mullerauthor=B.+Marsdenauthor=P.+E.+Brennanauthor=M.+A.+McDonoughauthor=U.+Oppermannauthor=R.+J.+Kloseauthor=C.+J.+Schofieldauthor=A.+Kawamura&title=Plant+growth+regulator+daminozide+is+a+selective+inhibitor+of+human+KDM2%2F7+histone+demethylases"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Plant Growth Regulator Daminozide Is a Selective Inhibitor of Human KDM2/7 Histone Demethylases</span></div><div class="casAuthors">Rose, Nathan R.; Woon, Esther C. Y.; Tumber, Anthony; Walport, Louise J.; Chowdhury, Rasheduzzaman; Li, Xuan Shirley; King, Oliver N. F.; Lejeune, Clarisse; Ng, Stanley S.; Krojer, Tobias; Chan, Mun Chiang; Rydzik, Anna M.; Hopkinson, Richard J.; Che, Ka Hing; Daniel, Michelle; Strain-Damerell, Claire; Gileadi, Carina; Kochan, Grazyna; Leung, Ivanhoe K. H.; Dunford, James; Yeoh, Kar Kheng; Ratcliffe, Peter J.; Burgess-Brown, Nicola; von Delft, Frank; Muller, Susanne; Marsden, Brian; Brennan, Paul E.; McDonough, Michael A.; Oppermann, Udo; Klose, Robert J.; Schofield, Christopher J.; Kawamura, Akane</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6639-6643</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The JmjC oxygenases catalyze the N-demethylation of Nε-Me lysine residues in histones and are current therapeutic targets.  A set of human 2-oxoglutarate analogs were screened using a unified assay platform for JmjC demethylases and related oxygenases.  Results led to the finding that daminozide (N-(dimethylamino)succinamic acid, 160 Da), a plant growth regulator, selectively inhibits the KDM2/7 JmjC subfamily.  Kinetic and crystallog. studies reveal that daminozide chelates the active site metal via its hydrazide carbonyl and dimethylamino groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwDtqe21TmpbVg90H21EOLACvtfcHk0lhMgHQBwcKedw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XptFSrur0%253D&md5=25072edc07e7e039e6e742dc215a6b61</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm300677j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300677j%26sid%3Dliteratum%253Aachs%26aulast%3DRose%26aufirst%3DN.%2BR.%26aulast%3DWoon%26aufirst%3DE.%2BC.%2BY.%26aulast%3DTumber%26aufirst%3DA.%26aulast%3DWalport%26aufirst%3DL.%2BJ.%26aulast%3DChowdhury%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DX.%2BS.%26aulast%3DKing%26aufirst%3DO.%2BN.%26aulast%3DLejeune%26aufirst%3DC.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DKrojer%26aufirst%3DT.%26aulast%3DChan%26aufirst%3DM.%2BC.%26aulast%3DRydzik%26aufirst%3DA.%2BM.%26aulast%3DHopkinson%26aufirst%3DR.%2BJ.%26aulast%3DChe%26aufirst%3DK.%2BH.%26aulast%3DDaniel%26aufirst%3DM.%26aulast%3DStrain-Damerell%26aufirst%3DC.%26aulast%3DGileadi%26aufirst%3DC.%26aulast%3DKochan%26aufirst%3DG.%26aulast%3DLeung%26aufirst%3DI.%2BK.%26aulast%3DDunford%26aufirst%3DJ.%26aulast%3DYeoh%26aufirst%3DK.%2BK.%26aulast%3DRatcliffe%26aufirst%3DP.%2BJ.%26aulast%3DBurgess-Brown%26aufirst%3DN.%26aulast%3Dvon%2BDelft%26aufirst%3DF.%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DMarsden%26aufirst%3DB.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DMcDonough%26aufirst%3DM.%2BA.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DKlose%26aufirst%3DR.%2BJ.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DKawamura%26aufirst%3DA.%26atitle%3DPlant%2520growth%2520regulator%2520daminozide%2520is%2520a%2520selective%2520inhibitor%2520of%2520human%2520KDM2%252F7%2520histone%2520demethylases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6639%26epage%3D6643" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XAJ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2XAJ','PDB','2XAJ'); return false;">PDB: 2XAJ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PDQ','PDB','3PDQ'); return false;">PDB: 3PDQ</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i56"><a href="/doi/suppl/10.1021/jm4012802">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_12285"></div></div></div></div></div><hr /></hr><p class="last">Additional text describing manual docking and granulocytic differentiation; <sup>1</sup>H NMR spectra for <b>1</b>–<b>3</b>; two tables listing elemental analyses of <b>1</b>–<b>6</b> and <b>11</b>–<b>26</b> and JmjCs’ IC<sub>50</sub> values for <b>21</b>–<b>26</b>; and four figures showing pro-differentiative effects of <b>1</b>–<b>9</b> in LNCaP and HCT116 cells; cell cycle effects of <b>2</b>, <b>3</b>, <b>7</b>, <b>8</b>, and <b>7</b> + <b>8</b> combination (50 μM, 24 h) in LNCaP, HCT116, and MePR cells; proliferative curves of LNCaP and HCT116 cells treated with 50 μM <b>2</b> and <b>3</b> for 72 h; and migration assay in HCT116 cells with <b>2</b> and <b>3</b> (50 μM, 72 h). This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm4012802/suppl_file/jm4012802_si_001.pdf">jm4012802_si_001.pdf (1.76 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm4012802&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2014.57.issue-1%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Fjm4012802%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm4012802" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0fd3bffe1d203","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
