
<html lang="en"     class="pb-page"  data-request-id="ef7ccb47-8927-4632-a44a-6c2f3539c57f"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2020.63.issue-24;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.0c01647;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="HER2 Kinase-Targeted Breast Cancer Therapy: Design, Synthesis, and In Vitro and In Vivo Evaluation of Novel Lapatinib Congeners as Selective and Potent HER2 Inhibitors with Favorable Metabolic Stability" /></meta><meta name="dc.Creator" content="Tamer A.  Elwaie" /></meta><meta name="dc.Creator" content="Safinaz E.  Abbas" /></meta><meta name="dc.Creator" content="Enayat I.  Aly" /></meta><meta name="dc.Creator" content="Riham F.  George" /></meta><meta name="dc.Creator" content="Hamdy  Ali" /></meta><meta name="dc.Creator" content="Nikolai  Kraiouchkine" /></meta><meta name="dc.Creator" content="Khaldoun S.  Abdelwahed" /></meta><meta name="dc.Creator" content="Tamer E.  Fandy" /></meta><meta name="dc.Creator" content="Khalid A.  El Sayed" /></meta><meta name="dc.Creator" content="Zakaria Y.  Abd Elmageed" /></meta><meta name="dc.Creator" content="Hamed I.  Ali" /></meta><meta name="dc.Description" content="HER2 kinase as a well-established target for breast cancer (BC) therapy is associated with aggressive clinical outcomes; thus, herein we present structural optimization for HER2-selective targeting..." /></meta><meta name="Description" content="HER2 kinase as a well-established target for breast cancer (BC) therapy is associated with aggressive clinical outcomes; thus, herein we present structural optimization for HER2-selective targeting..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="December 14, 2020" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c01647" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2020 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01647" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c01647" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c01647" /></link>
        
    
    

<title>HER2 Kinase-Targeted Breast Cancer Therapy: Design, Synthesis, and In Vitro and In Vivo Evaluation of Novel Lapatinib Congeners as Selective and Potent HER2 Inhibitors with Favorable Metabolic Stability | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c01647" /></meta><meta property="og:title" content="HER2 Kinase-Targeted Breast Cancer Therapy: Design, Synthesis, and In Vitro and In Vivo Evaluation of Novel Lapatinib Congeners as Selective and Potent HER2 Inhibitors with Favorable Metabolic Stability" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0025.jpeg" /></meta><meta property="og:description" content="HER2 kinase as a well-established target for breast cancer (BC) therapy is associated with aggressive clinical outcomes; thus, herein we present structural optimization for HER2-selective targeting. HER2 profiling of the developed derivatives demonstrated potent and selective inhibitions (IC50: 5.4–12 nM) compared to lapatinib (IC50: 95.5 nM). Favorably, 17d exhibited minimum off-target kinase activation. NCI-5-dose screening revealed broad-spectrum activities (GI50: 1.43–2.09 μM) and 17d had a remarkable selectivity toward BC. Our compounds revealed significant selective and potent antiproliferative activities (∼20-fold) against HER2+ (AU565, BT474) compared to HER2(−) cells. At 0.1 IC50, 15i, 17d, and 25b inhibited pERK1/2 and pAkt by immunoblotting. Furthermore, 17d demonstrated potent in vivo tumor regression against the BT474 xenograft model. Notably, a metastasis case was observed in the vehicle but not in the test mice groups. CD-1 mice metabolic stability assay revealed high stability and low intrinsic clearance of 17d (T1/2 &gt; 145 min and CLint(mic) &lt; 9.6 mL/min/kg)." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c01647"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01647">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c01647&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c01647&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c01647&amp;href=/doi/10.1021/acs.jmedchem.0c01647" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2020</span><span class="cit-fg-volume">, 63</span><span class="cit-fg-issue">, 24</span><span class="cit-fg-pageRange">, 15906-15945</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/63/24" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01609" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c01684" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">HER2 Kinase-Targeted Breast Cancer Therapy: Design, Synthesis, and <i>In Vitro</i> and <i>In Vivo</i> Evaluation of Novel Lapatinib Congeners as Selective and Potent HER2 Inhibitors with Favorable Metabolic Stability</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Tamer A. Elwaie</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tamer A. Elwaie</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</div><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tamer+A.++Elwaie">Tamer A. Elwaie</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Safinaz E. Abbas</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Safinaz E. Abbas</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Safinaz+E.++Abbas">Safinaz E. Abbas</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Enayat I. Aly</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Enayat I. Aly</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Enayat+I.++Aly">Enayat I. Aly</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Riham F. George</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Riham F. George</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Riham+F.++George">Riham F. George</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Hamdy Ali</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hamdy Ali</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hamdy++Ali">Hamdy Ali</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nikolai Kraiouchkine</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nikolai Kraiouchkine</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Physical and Environmental Sciences, Texas A&M University, Corpus Christi, Texas 78412, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nikolai++Kraiouchkine">Nikolai Kraiouchkine</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Khaldoun S. Abdelwahed</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Khaldoun S. Abdelwahed</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Basic Pharmaceutical and Toxicological Sciences, College of Pharmacy, University of Louisiana at Monroe, 1800 Bienville Drive, Monroe, Louisiana 71201, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Khaldoun+S.++Abdelwahed">Khaldoun S. Abdelwahed</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Tamer E. Fandy</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Tamer E. Fandy</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical & Administrative Sciences, School of Pharmacy, University of Charleston, Charleston, West Virginia 25304, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Tamer+E.++Fandy">Tamer E. Fandy</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Khalid A. El Sayed</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Khalid A. El Sayed</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Basic Pharmaceutical and Toxicological Sciences, College of Pharmacy, University of Louisiana at Monroe, 1800 Bienville Drive, Monroe, Louisiana 71201, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Khalid+A.++El+Sayed">Khalid A. El Sayed</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1456-4064" title="Orcid link">http://orcid.org/0000-0002-1456-4064</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Zakaria Y. Abd Elmageed</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Zakaria Y. Abd Elmageed</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</div><div class="loa-info-affiliations-info">Department of Pharmacology, Edward Via College of Osteopathic Medicine, University of Louisiana at Monroe, Monroe, Louisiana 71203, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#91cbf4fdfcf0f6f4f4f5d1e4fdfcbfe7f2fefcbff4f5e4"><span class="__cf_email__" data-cfemail="540e3138393533313130142138397a22373b397a313021">[email protected]</span></a>. Phone: (318) 342-7185. Fax: (318) 342-7280.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Zakaria+Y.++Abd+Elmageed">Zakaria Y. Abd Elmageed</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Hamed I. Ali</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Hamed I. Ali</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#84e5e8fdedf7e9e5ede8c4f0e5e9f1aae1e0f1"><span class="__cf_email__" data-cfemail="b0d1dcc9d9c3ddd1d9dcf0c4d1ddc59ed5d4c5">[email protected]</span></a>. Phone: (361) 221-0758. Fax: (361) 221-0793.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Hamed+I.++Ali">Hamed I. Ali</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6939-8462" title="Orcid link">http://orcid.org/0000-0002-6939-8462</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01647&amp;href=/doi/10.1021%2Facs.jmedchem.0c01647" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2020</span></span><span class="cit-volume">, 63</span><span class="cit-issue">, 24</span><span class="cit-pageRange">, 15906–15945</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">December 14, 2020</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 September 2020</li><li><span class="item_label"><b>Published</b> online</span>14 December 2020</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 December 2020</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c01647" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01647</a></div><div class="article_header-article-copyright"><strong>Copyright © 2020 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D15906%26pageCount%3D40%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DTamer%2BA.%2BElwaie%252C%2BSafinaz%2BE.%2BAbbas%252C%2BEnayat%2BI.%2BAly%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D63%26issueNum%3D24%26contentID%3Dacs.jmedchem.0c01647%26title%3DHER2%2BKinase-Targeted%2BBreast%2BCancer%2BTherapy%253A%2BDesign%252C%2BSynthesis%252C%2Band%2BIn%2BVitro%2Band%2BIn%2BVivo%2BEvaluation%2Bof%2BNovel%2BLapatinib%2BCongeners%2Bas%2BSelective%2Band%2BPotent%2BHER2%2BInhibitors%2Bwith%2BFavorable%2BMetabolic%2BStability%26numPages%3D40%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D15945%26publicationDate%3DDecember%2B2020">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c01647"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1257</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c01647" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;HER2 Kinase-Targeted Breast Cancer Therapy: Design, Synthesis, and In Vitro and In Vivo Evaluation of Novel Lapatinib Congeners as Selective and Potent HER2 Inhibitors with Favorable Metabolic Stability&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Tamer&quot;,&quot;last_name&quot;:&quot;A. Elwaie&quot;},{&quot;first_name&quot;:&quot;Safinaz&quot;,&quot;last_name&quot;:&quot;E. Abbas&quot;},{&quot;first_name&quot;:&quot;Enayat&quot;,&quot;last_name&quot;:&quot;I. Aly&quot;},{&quot;first_name&quot;:&quot;Riham&quot;,&quot;last_name&quot;:&quot;F. George&quot;},{&quot;first_name&quot;:&quot;Hamdy&quot;,&quot;last_name&quot;:&quot;Ali&quot;},{&quot;first_name&quot;:&quot;Nikolai&quot;,&quot;last_name&quot;:&quot;Kraiouchkine&quot;},{&quot;first_name&quot;:&quot;Khaldoun&quot;,&quot;last_name&quot;:&quot;S. Abdelwahed&quot;},{&quot;first_name&quot;:&quot;Tamer&quot;,&quot;last_name&quot;:&quot;E. Fandy&quot;},{&quot;first_name&quot;:&quot;Khalid&quot;,&quot;last_name&quot;:&quot;A. El Sayed&quot;},{&quot;first_name&quot;:&quot;Zakaria&quot;,&quot;last_name&quot;:&quot;Y. Abd Elmageed&quot;},{&quot;first_name&quot;:&quot;Hamed&quot;,&quot;last_name&quot;:&quot;I. Ali&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2020&quot;,&quot;month&quot;:&quot;12&quot;,&quot;day&quot;:&quot;14&quot;,&quot;issue&quot;:&quot;24&quot;,&quot;volume&quot;:&quot;63&quot;,&quot;pages&quot;:&quot;15906-15945&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c01647&quot;},&quot;abstract&quot;:&quot;HER2 kinase as a well-established target for breast cancer (BC) therapy is associated with aggressive clinical outcomes; thus, herein we present structural optimization for HER2-selective targeting. HER2 profiling of the developed derivatives demonstrated potent and selective inhibitions (IC50: 5.4–12 nM) compared to lapatinib (IC50: 95.5 nM). Favorably, 17d exhibited minimum off-target kinase activation. NCI-5-dose screening revealed broad-spectrum activities (GI50: 1.43–2.09 μM) and 17d had a remarkable selectivity toward BC. Our compounds revealed significant selective and potent antiproliferative activities (∼20-fold) against HER2+ (AU565, BT474) compared to HER2(−) cells. At 0.1 IC50, 15i, 17d, and 25b inhibited pERK1/2 and pAkt by immunoblotting. Furthermore, 17d demonstrated potent in vivo tumor regression against the BT474 xenograft model. Notably, a metastasis case was observed in the vehicle but not in the test mice groups. CD-1 mice metabolic stability assay revealed high stability and low intrinsi&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01647&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01647" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01647&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01647" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c01647&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01647" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c01647&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c01647&amp;href=/doi/10.1021/acs.jmedchem.0c01647" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c01647" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c01647" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01647&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c01647%26sid%3Dliteratum%253Aachs%26pmid%3D33314925%26genre%3Darticle%26aulast%3DElwaie%26date%3D2020%26atitle%3DHER2%2BKinase-Targeted%2BBreast%2BCancer%2BTherapy%253A%2BDesign%252C%2BSynthesis%252C%2Band%2BIn%2BVitro%2Band%2BIn%2BVivo%2BEvaluation%2Bof%2BNovel%2BLapatinib%2BCongeners%2Bas%2BSelective%2Band%2BPotent%2BHER2%2BInhibitors%2Bwith%2BFavorable%2BMetabolic%2BStability%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D63%26issue%3D24%26spage%3D15906%26epage%3D15945%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (6)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=291788" title="Reaction products">Reaction products</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/63/24" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/jmcmar.2020.63.issue-24/20201224/jmcmar.2020.63.issue-24.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">HER2 kinase as a well-established target for breast cancer (BC) therapy is associated with aggressive clinical outcomes; thus, herein we present structural optimization for HER2-selective targeting. HER2 profiling of the developed derivatives demonstrated potent and selective inhibitions (IC<sub>50</sub>: 5.4–12 nM) compared to lapatinib (IC<sub>50</sub>: 95.5 nM). Favorably, <b>17d</b> exhibited minimum off-target kinase activation. NCI-5-dose screening revealed broad-spectrum activities (GI<sub>50</sub>: 1.43–2.09 μM) and <b>17d</b> had a remarkable selectivity toward BC. Our compounds revealed significant selective and potent antiproliferative activities (∼20-fold) against HER2+ (AU565, BT474) compared to HER2(−) cells. At 0.1 IC<sub>50</sub>, <b>15i</b>, <b>17d</b>, and <b>25b</b> inhibited pERK1/2 and pAkt by immunoblotting. Furthermore, <b>17d</b> demonstrated potent <i>in vivo</i> tumor regression against the BT474 xenograft model. Notably, a metastasis case was observed in the vehicle but not in the test mice groups. CD-1 mice metabolic stability assay revealed high stability and low intrinsic clearance of <b>17d</b> (<i>T</i><sub>1/2</sub> > 145 min and CL<sub>int(mic)</sub> < 9.6 mL/min/kg).</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74667" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74667" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Cancer is the second main cause of death in the United States and worldwide.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4 ref5 ref6'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4 ref5 ref6">(1−6)</a> The most common cancers among men and women are prostate and breast cancers (BCs), respectively.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> BC is the most commonly diagnosed cancer and the main cause of cancer deaths among women globally. Approximately, two million patients are diagnosed with BC and more than 600,000 deaths occur each year.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In the United States, 276,480 new cases of female BC have been estimated and 42,170 deaths were reported in 2020.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> The ErbB (HER) proteins are related to subclass I of the superfamily of receptor tyrosine kinases (RTKs) which plays a vital role in signal transduction pathways and the initiation of malignancy of several solid tumors. The ErbB family contains epidermal growth factor receptor (EGFR; also named ErbB1/HER1), ErbB2/HER2, ErbB3/HER3, and ErbB4/HER4.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> All ErbBs are closely similar in their structures and consist of three domains, an extracellular ligand-binding domain, a transmembrane domain, and an intracellular kinase domain.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Unlike EGFR (ErbB1/HER1), ErbB2 (HER2) does not have a recognized activating ligand (ligandless receptor),<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> accordingly its overexpression can lead to cell transformation without the need of ligand interaction.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> HER2 signalings are initiated by homo- or hetero-dimerization and these dimers have a lower dissociation rate with persistent signal transduction compared to other dimers. Subsequently, these intracellular cascades activate cell survival, proliferation, motility, and differentiation.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Dysregulation of the <i>HER2</i> gene disrupts cell proliferation and leads to the formation of aggressive tumor cells.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> The HER2 RTK is amplified in approximately 15–30% of BC and is associated with worse aggressive tumor forms.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> HER2 overexpression has also been reported with high frequencies in the lung (NSCLC), gastroesophageal, ovarian, melanoma, and oral cancers.<a onclick="showRef(event, 'ref18 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref18 ref19 ref20">(18−20)</a> HER2-positive (HER2+) BC often has a more aggressive phenotype than HER2-negative (HER2−) BC.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Worldwide, approximately 15–20% of BC cases are HER2+.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> Because of the identification of the HER2 receptor amplification as an adverse prognostic factor in a special subtype of the metastatic BC, there has been a substantial improvement in the survival of these patients because of the development of anti-HER2-targeted therapies.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> HER2 is a well-established therapeutic target for the treatment of BC.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Currently, there is a variety of FDA-approved HER2-targeted therapies for the treatment of HER2+ BCs including anti-HER2 monoclonal antibodies, such as trastuzumab and pertuzumab, antibody–drug conjugate such as trastuzumab-emtansine (T-DM1), and small-molecule pan-HER tyrosine kinase inhibitors (TKIs) such as lapatinib and neratinib.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The small-molecule TKIs have potential advantages over monoclonal antibody therapies, including their ability to inhibit multiple kinases and p95 HER2 (highly active truncated HER2 lacking most of the extracellular domain).<a onclick="showRef(event, 'ref23 ref25'); return false;" href="javascript:void(0);" class="ref ref23 ref25">(23,25)</a> Likewise, TKIs have lower cardiopulmonary toxicities commonly associated with monoclonal antibody therapies<a onclick="showRef(event, 'ref26 ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref26 ref27 ref28">(26−28)</a> and have a higher ability to penetrate the blood–brain barrier with a greater therapeutic potential against brain metastases.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In addition, the majority of BC patients who have an initial therapeutic response to the HER2-targeted antibody trastuzumab reveal poor prognosis within one year.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Unlike lapatinib, neratinib is a potent irreversible pan-HER kinase inhibitor but its clinical use may result in hepatotoxicity.<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> In favor of this strategy, four of the 25 FDA-approved medications in 2019 are kinase inhibitors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The development of kinase inhibitors is based on the fundamental understanding of the general pharmacophoric features of these inhibitors (summarized in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) which are composed of (a) core scaffold serving as a hinge binder; (b) hydrophobic bulky moiety extended into the back pocket revealing HER2 kinase selectivity; and (c) hydrophilic solubilizing fragment harbored into the solvent-accessible region to conduct its pharmacokinetic role through balancing the logP of the inhibitors. The solvent-accessible region can tolerate a wide range of solubilizing groups modulating the cellular activity without affecting the enzymatic activity. Moreover, introducing particular functionalities into the solubilizing moiety would optimize the potency and selectivity profiles.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The 4-anilinoquinazoline is the most commonly used hinge binder scaffolds for EGFR kinase inhibitors, represented by the clinically approved drugs: gefitinib, erlotinib, and lapatinib (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref33 ref34 ref35'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35">(33−35)</a> Lapatinib was developed by GSK as the first dual EGFR/HER2 inhibitor and approved by the FDA in 2007 for treatment of BC. Its bulky 4-benzyloxyaniline substituent is essential for EGFR/HER2 dual inhibition. However, smaller anilines such as 3-ethynylaniline in erlotinib and 3-chloro-4-fluoro-aniline in gefitinib only inhibit EGFR.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Lapatinib binds to the inactive EGFR closed conformation displacing the αC-helix to “out” and gives the 3-fluorobenzyloxy group access to the hydrophobic back pocket.<a onclick="showRef(event, 'ref32 ref37'); return false;" href="javascript:void(0);" class="ref ref32 ref37">(32,37)</a> This extra binding in the back pocket contributes to the dual EGFR/HER2 inhibition and has longer inhibition time when compared to gefitinib and erlotinib which do not have access into the back pocket.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> In this regard, Abouzid et al. reported enhanced EGFR/HER2 activities of quinazoline derivatives through incorporating bulkier anilino moieties at position 4 in a manner similar to lapatinib. These moieties include 4-(4-aryl-2-imino pyridino) anilines, which could be oriented deep in the back of the ATP-binding site and mimicking the 3-chloro-4-[(3-fluorobenzyl)oxy]aniline group of lapatinib.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> The HER2<sup>T798I(M)</sup> gatekeeper mutation and its analogous EGFR<sup>T790M</sup> have been shown to confer the most common clinical resistance against lapatinib.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> This resistance emerges from replacing a smaller polar Thr<sup>798</sup> with bulkier Ile<sup>798</sup> or Met<sup>798</sup> residues, which causes unfavorable steric clashes between the gatekeeper mutant Ile<sup>798</sup> amino acid and lapatinib.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Targeting other HER family members, such as EGFR (HER1), does not functionally suppress HER2 signaling and may introduce off-target toxicities such as diarrhea and rash.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> Moreover, in BC cells that overexpress HER2, the inhibition of HER family members EGFR or HER4 does not functionally suppress HER2 signaling and consequently does not prevent tumor cell proliferation or survival.<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42,43)</a> Noteworthy, lapatinib, as an EGFR inhibitor, has another serious drawback by causing severe skin rash and diarrhea. Skin toxicity is typically manifested as mild to severe papulopustular rash in the majority of patients (45–100%) receiving EGFR inhibitors.<a onclick="showRef(event, 'ref44 ref45 ref46'); return false;" href="javascript:void(0);" class="ref ref44 ref45 ref46">(44−46)</a> These adverse effects reveal serious physical and psycho-social complications, particularly for female patients with BC. For all the aforementioned limitations of current medications, there is an eventual need for developing novel selective HER2 inhibitors with enhanced efficacy against mutant isoforms to overcome the reported limitations of lapatinib. Based on the reported key binding features of lapatinib as a dual EGFR/HER2 inhibitor, guided by its binding mode through flexible docking into the HER2 kinase domain (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and inspired by the importance of HER2 overexpression in BC progression, in this study, we present the design and development of new series of 6-substituted-4-anilinoquinazolines (<b>10a–c</b>, <b>15a–l</b>, <b>17a–d</b>, and, <b>25a–h</b>) as promising candidates for HER2 inhibition with improved potency and selectivity. The developed candidates can preferentially inhibit HER2 kinase and reduce the potential of EGFR-associated toxicities. In addition, our strategy is oriented to developing a variety of lapatinib analogues with some structural flexibility, this flexibility of the designed compounds may diminish the steric clashes within the gatekeeper amino acids of the HER2<sup>T798I</sup> mutant BC and consequently provide better treatment options.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of EGFR/HER2 inhibitors into their corresponding kinases and the fragments used for their design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structure for the FDA-approved EGFR and dual EGFR/HER2 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking mode of lapatinib (ball and sticks) and TAK285 (orange lines) into the HER2 kinase domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). Hydrophobic region 1 (L726, A751, L800, G804, L852, and M801); hydrophobic selectivity pocket (M774, L785, L796, T798, D863, and F864); solvent-accessible region for TAK285 (G727, S728, and V734); solvent-accessible region for lapatinib (C805 and D808); α-C helix (P761-V777), and the gatekeeper amino acid (T798).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52867" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52867" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Rational Design</h3><div class="NLM_p">The rational design of our compounds was based on the fundamental pharmacophoric features for the kinase inhibitors summarized in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a> and according to the analysis of the binding pattern of the FDA-approved kinase inhibitors. Our template kinase inhibitors include the dual EGFR/HER2 inhibitors (lapatinib and neratinib) and the investigational selective HER2 inhibitor (CP-724,714).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Our efforts were focused on the design and synthesis of new hybrid molecules with the 6-substituted 4-anilinoquinazoline core to potentiate the selectivity against the target HER2 kinase by reversible inhibition and tackling the emergence of lapatinib acquired resistance. Guided by the reported EGFR/HER2 kinase inhibitors, we initially designed and synthesized the lead compound (<b>10a</b>; <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>) via incorporating a flexible spacer at position 6 of the quinazoline ring, namely, 6-acetamido, to mimic neratinib. This flexible design is aimed to avoid the steric clashes within the gatekeeper T798I(M) mutant kinase. A hydrophilic ethanolamine tail was introduced to the flexible spacer at position 6 through nucleophilic substitution. At position 4 “<i>N</i>-(4-(3-chloro-4-(3-fluorobenzyl)oxy)phenyl)amino” moiety of lapatinib was retained. The lead compound (<b>10a</b>) revealed superior inhibition against HER2 kinase (IC<sub>50</sub> = 10.4 nM) compared to lapatinib which exhibited inferior inhibition (IC<sub>50</sub> = 95.5 nM). WST-8 assay for the lead compound was conducted using AU565 (only HER2+) and BT474 {triple positive, HER2+, estrogen (ER+), and progesterone (PR+)} BC cell lines. The lead compound (<b>10a</b>) exhibited promising antiproliferative activities (IC<sub>50</sub> = 0.22 and 0.24 μM, respectively). These preliminary results have prompted us to enhance the efficiency of the lead compound through lead optimization, as depicted in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. To increase the selectivity of HER2 inhibition, the “<i>N</i>-(4-(3-chloro-4-(3-fluorobenzyl)oxy)phenyl)amino” moiety of lapatinib was replaced by phenoxy phenyl analogues that fit well into the back pocket of HER2. Optimization of HER2 inhibitory activity was conducted through replacing ethanolamine by morpholine, <i>N</i>-methyl piperazine, diethylamine, and dimethylamine at position 6 of the quinazoline scaffold to improve the pharmacokinetic properties. This hydrophilic tail was designed to fit into the ribose and phosphate region of the kinase ATP-binding site and improve the druggability. Different flexible spacers of acetamido, benzamido, and propoxy moieties were bridged between quinazoline and the hydrophilic tail. The flexibility of the used spacers is engendered by replacing the rigid 6-C–C bond in lapatinib by more flexible C–O or C–N bonds in the newly synthesized compounds, as shown in detail in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>. The structural optimization was conducted using computer-aided drug design (CADD). A small library of 189 compounds (data are presented in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Table S1</a>) was designed by changing the substituents at position 4 and 6 of the quinazoline nucleus based on the literature database. At position 4, the lipophilic moiety was either substituted benzyloxy or phenoxy anilino moieties with the variables, including <i>n</i> = 0 or 1, X = H, Cl, or Br and R = o, m, or p CH<sub>3</sub>, CF<sub>3</sub>, CN, and F. At position 6, the flexible spacer was the acetamido, benzamido, or propoxy moiety and the hydrophilic terminal was the ethanolamino, morpholino, <i>N</i>-methyl piperazino, diethylamino, dimethylamino, methylsulfonylethylamino, immidazo, or pyrrolidino moiety. Then, the 3D structures of the developed compounds were energetically minimized to be explored as kinase inhibitors by docking into the crystal structure of HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>),<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> in complex with its cocrystallized kinase inhibitor. The GOLD software package version 5.2.2 (Cambridge Crystallographic Data Centre, Cambridge, U.K.)<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> was used in this study, which is described in detail under the <a class="ref internalNav" href="#sec2_8" aria-label="Molecular Docking">Molecular Docking</a> part. The newly synthesized candidates were biologically evaluated in enzyme inhibitor-based assays and cell antiproliferative activity ones. The following are the biological screening tests that had been carried out in the present work; <i>in vitro</i> protein kinase profiling was conducted to figure out the degree of selectivity obtained against the target HER2 using 20 different types of kinases, IC<sub>50</sub> determination against wild and HER2 L755S mutant kinases, screening against National Cancer Institute-NCI 60 human cell panel, and the WST-8 antiproliferative assay using BT-474 and AU-565 BC cell lines (HER2+), as well as MCF7 and MDA-MB-231 BC cell lines (HER2−), were included. In addition, we also investigated the apoptotic inducing effect in BT-474 cells; western blot analysis was also established to study the effect of some selected candidates on inhibition of the HER2 downstream targets (ERK and AKT). <i>In vivo</i> antitumor efficacy and microsomal stability assay were also established for the most promising candidates.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Rational design enhanced using CADD of the newly synthesized compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Chemistry</h3><div class="NLM_p">In this part, we described the preparation of novel 6-substituted-4-anilinoquinazolines (<b>10a–c</b>, <b>15a–I</b>, <b>17a–d</b>, and <b>25a–h</b>) which were purified by flash column chromatography. The structures of the newly synthesized compounds were elucidated by different spectroscopic methods, including <sup>1</sup>H NMR, <sup>13</sup>C DEPT, <sup>13</sup>C NMR, and high-resolution mass spectrometry (HRMS).</div><div class="NLM_p">The synthesis of the different substituted aniline derivatives (<b>4a–d</b>) is described in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Reaction of the commercially available 2-chloro-4-nitrophenol (<b>1a</b>) or 2-chloro-1-fluoro-4-nitrobenzene (<b>1b</b>) with 1-(bromomethyl)-3-fluorobenzene (<b>2a</b>) or the appropriate phenol derivative (<b>2b–d</b>) was carried out in dimethylformamide (DMF) or acetonitrile in the presence of potassium carbonate to afford the corresponding nitro derivatives (<b>3a–d</b>) in good yields of 73–89%.<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> Reduction of the nitro derivatives (<b>3a–d</b>) into their amino analogues (<b>4a–d</b>) were achieved by either heating with Fe/CaCl<sub>2</sub> in aqueous ethanolic solution to prepare (<b>4a</b>) intermediate or via reflux with SnCl<sub>2</sub> for 2.5 h in methanol under nitrogen to afford intermediates (<b>4b–d</b>). The reduction methods gave the desired intermediates in very good yields of 77–82%, in shorter reaction times, and with higher purity than the reported methods using H<sub>2</sub>/Pt–C,<a onclick="showRef(event, 'ref50 ref51'); return false;" href="javascript:void(0);" class="ref ref50 ref51">(50,51)</a> SnCl<sub>2</sub>/EtOAc/reflux/24 h,<a onclick="showRef(event, 'ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref52 ref53">(52,53)</a> Na<sub>2</sub>S/dioxane/70–80 °C/24 h (Zinin reaction),<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> or Fe/NH<sub>4</sub>Cl/EtOH/reflux/2 h.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> The <sup>1</sup>H NMR spectrum of <b>4c</b> revealed a broad signal at δ 5.42, exchanged with D<sub>2</sub>O, representing the two protons of the NH<sub>2</sub> group. The impact of the acetamido linker at position 6 was investigated by applying <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, and the 3-chloro-4-((3-fluorobenzyl)oxy)aniline (<b>4a</b>) was incorporated into position 4 of the quinazoline nucleus through Dimroth rearrangement, as depicted in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. The reaction of 2-amino-5-nitrobenzonitrile (<b>5</b>) with DMF-DMA (dimethylformamide-dimethylacetal) afforded the formimidamide intermediate (<b>6</b>) in a 90% yield, which was then cyclized upon reflux with compound <b>4a</b> in glacial acetic acid into 6-nitro-4-anilinoquinazoline derivative (<b>7</b>) through Dimroth rearrangement (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>) as reported for similar derivatives.<a onclick="showRef(event, 'ref55'); return false;" href="javascript:void(0);" class="ref ref55">(55)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed mechanism of Dimroth rearrangement for the preparation of 6-nitroquinazoline derivative (<b>7</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0019.jpeg" id="gr18" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 2-Chloro-4-nitro-1-(3-(substituted)phenoxy/benzyloxy)benzene (<b>3a–d</b>) and 3-Chloro-4-(3-(substituted)phenoxy/benzyloxy)aniline (<b>4a–d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF, K<sub>2</sub>CO<sub>3</sub>, 85 °C, 3 h; (b) Fe, CaCl<sub>2</sub>, EtOH, H<sub>2</sub>O, 80 °C, 8 h; (c) acetonitrile, K<sub>2</sub>CO<sub>3</sub>, 85 °C, 5 h; (d) SnCl<sub>2</sub>, MeOH, reflux, under N<sub>2</sub>, 2.5 h.</p></p></figure><figure id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0020.jpeg" id="gr19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2-Substituted Acetamide (<b>10a–c</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF-DMA, reflux 2 h; (b) glacial HOAc, reflux, 3 h; (c) Fe, CaCl<sub>2</sub>, EtOH, H<sub>2</sub>O, 90 °C, 12 h; (d) ClCH<sub>2</sub>COCl, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 1 h; (e) ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h or appropriate secondary amine (50 equiv), reflux, 2 h.</p></p></figure><div class="NLM_p">Reduction of (<b>7</b>) could be attained by a sodium dithionite/ammonia mixture or SnCl<sub>2</sub>/methanol, but the yield was very low (30%) and the product requires purification by flash column chromatography. In the present study, reflux of iron in an alcoholic/aqueous solution of calcium chloride was implemented to efficiently afford a 6-amino-4-anilinoquinazoline derivative (<b>8</b>) in an excellent yield of 97% with high purity without further purification. Subsequently, compound <b>8</b> was acylated by 2-chloroacetyl chloride in acetone to afford the corresponding 2-chloroacetamido derivative (<b>9</b>) in 61% yield. The target compounds <b>10a–c</b> were prepared through a nucleophilic substitution on the chloroacetamido moiety of compound <b>9</b> using ethanolamine or the appropriate cyclic secondary amines (morpholine and <i>N</i>-methyl piperazine) according to the procedure described in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a> to afford pure compounds in 57–87% yields. Compounds <b>10a–c</b> were confirmed by multiple spectroscopic data including their <sup>1</sup>H NMR spectra which exhibited two singlet signals at δ 5.25 and δ 3.19–3.35 for benzyloxy CH<sub>2</sub> protons and acetamido CH<sub>2</sub> protons, respectively. Besides, a quartet signal at δ 3.54–3.49 and a triplet signal at δ 2.69–2.66 were representing the four protons of the ethanolamine terminal in the target compound (<b>10a</b>), and two triplet signals at δ 2.55–3.69 were detected for the eight protons of the morpholino ring in compound (<b>10b</b>). Another two broad singlet signals at δ 2.41–2.57 were assigned for the eight protons of the piperazino ring in compound (<b>10c</b>). An additional panel of novel target compounds (<b>15a–l</b> and <b>17a–d</b>) was prepared to study the structure–activity relationship (SAR) and to enhance the selectivity, as shown in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. The lapatinib anilino moiety at position 4, namely, 3-chloro-4-((3-fluorobenzyl)oxy) aniline in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>, was replaced by substituted phenoxyaniline fragments to enhance the selectivity against HER2 kinase, whereas the 6-acetamido linker was kept in compounds (<b>15a–l</b>) or replaced by a bulkier 6-methyl benzamido linker in compounds (<b>17a–d</b>). Reaction of 2-amino-5-nitrobenzonitrile (<b>5</b>) with triethylorthoformate in the presence of acetic anhydride produced the formimidate derivative (<b>11</b>), which was then cyclized upon reflux with the appropriately substituted phenoxyaniline (<b>4b–d</b>) to afford 6-nitro-4-anilinoquinazoline derivatives (<b>12a–c</b>) through Dimroth rearrangement. It was noteworthy that reduction of the 6-nitro-quinazolione derivatives (<b>12a–c</b>) to the corresponding 6-amino analogues (<b>13a–c</b>) was adequately accomplished using SnCl<sub>2</sub>/MeOH, where they were obtained within short reaction time (1.5 h), in high purities and excellent yields (86–94%). In comparison, the other reported conditions using iron/CaCl<sub>2</sub>/EtOH afforded these compounds within a longer reaction time (12 h) and in inferior purities and lower yields (52–65%). Also, a longer reaction time (24 h) is needed on using Na<sub>2</sub>S/dioxane.</div><figure id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0021.jpeg" id="gr20" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Target Compounds <b>15a–l</b> and <b>17a–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) triethylorthoformate, (Ac)<sub>2</sub>O, reflux, 30 h; (b) glacial HOAc, reflux, 7 h; (c) SnCl<sub>2</sub>, MeOH, reflux, under N<sub>2</sub>, 1.5 h; (d) ClCH<sub>2</sub>COCl, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 1 h; (e) appropriate secondary amine (50 equiv), reflux, 2 h or ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h; (f) using <b>13a</b>, 4-(chloromethyl)benzoyl chloride, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 0.5 h; (g) excess appropriate secondary amine (50 equiv), reflux, 2 h or ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h.</p></p></figure><div class="NLM_p">Compounds (<b>13a–c</b>) were subsequently acylated with 2-chloroacetyl chloride in acetone to yield the corresponding 2-chloroacetamido derivatives (<b>14a–c</b>), while compound (<b>16</b>) was prepared by acylation of intermediate (<b>13a</b>) with 4-(chloromethyl)benzoyl chloride to afford the corresponding 4-(chloromethyl)benzamido derivative (<b>16</b>). Two series of the target compounds (<b>15a–l</b>, 43–92% yields) and (<b>17a–d</b>, 48–78% yields) were synthesized through nucleophilic substitution of compounds (<b>14a–c</b>) and (<b>16</b>), respectively, with the appropriate secondary amines or ethanolamine, as described in <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>. Partial hydrolysis was detected in compounds (<b>17a–d</b>) bearing a benzamido linker to their corresponding 6-amino quinazoline analogues as a result of the nucleophilic attack of the reacting amines on the reactive carbonyl group of the benzamido moiety. Therefore, the hydrolytic products were removed by flash column chromatography to get the target compounds in a pure form. Compounds (<b>15a–l</b>) were confirmed by <sup>1</sup>H NMR spectra exhibiting a singlet signal at δ 3.15–3.38 for the acetamido CH<sub>2</sub> protons, in addition to, <sup>13</sup>C DEPT revealed an inverted signal at δ 53.17–63.69, representing the acetamido CH<sub>2</sub> carbon. On the other hand, <sup>1</sup>H NMR spectra of (<b>17a–d</b>) revealed the characteristic AB doublet spin system signals of the phenylene residue at δ 7.46–8.03 and a characteristic <sup>13</sup>C DEPT inverted signal for benzyl CH<sub>2</sub> at δ 52.97–63.42.</div><div class="NLM_p">It is noteworthy that the success of Dimroth rearrangement for the preparation of 6-nitroquinazoline derivatives (<b>7</b>) and (<b>12a–c</b>) depends on the used precursors of formimidamide (<b>6</b>) or the formimidate (<b>11</b>), as shown in <a class="ref internalNav" href="#sch2" aria-label="Schemes 2">Schemes 2</a> and <a class="ref internalNav" href="#sch3" aria-label="3">3</a>, respectively. We have observed that intermediate (<b>6</b>) could be used properly to afford the desired 6-nitroquinazoline derivatives (<b>7</b>) and (<b>12a–c</b>). However, the intermediate (<b>11</b>) was not the appropriate precursor for the preparation of 6-nitroquinazoline derivative (<b>7</b>), as shown in <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Two pathways conducted for the preparation of 6-nitroquinazoline derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthesis of 4-substituted phenoxy/benzyloxy anilino quinazolin-6-ol and their 6-substituted propoxy analogues (<b>25a–h</b>) is depicted in detail in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The quinazoline-4,6-diol (<b>19</b>) was prepared using the Niementowski synthesis,<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> by heating the commercially available 5-hydroxy anthranilic acid (<b>18</b>) in formamide.</div><figure id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0022.jpeg" id="gr21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Target Compounds <b>25a–h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NH<sub>2</sub>CHO, 190 °C, 1 h; (b) AC<sub>2</sub>O, pyridine, 100 °C, 2 h; (c) SOCl<sub>2</sub>, DMF, 95 °C, 5 h; (d) acetonitrile, K<sub>2</sub>CO<sub>3</sub>, rt, 8 h (with <b>4a</b>); acetonitrile, K<sub>2</sub>CO<sub>3</sub>, reflux, 12 h (with <b>4b</b> or <b>4c</b>); (e) NH<sub>4</sub>OH, MeOH, rt, 2 h, (f) Cl(CH<sub>2</sub>)<sub>3</sub>Br, K<sub>2</sub>CO<sub>3</sub>, TBAB, acetonitrile, reflux, 2 h; (g) appropriate amine (100 equiv), reflux, 2 h or ethanolamine (10 equiv), MeOH, TEA, reflux, 8 h.</p></p></figure><div class="NLM_p">The <sup>1</sup>H NMR spectrum of (<b>19</b>) showed a characteristic singlet signal of the C-2 proton of the quinazoline nucleus at δ 7.90, whereas the two OH protons were detected at δ 10.05 and 12.02. In addition, the melting point was consistent with the reported one.<a onclick="showRef(event, 'ref57 ref58'); return false;" href="javascript:void(0);" class="ref ref57 ref58">(57,58)</a> The phenolic OH at the C-6 position of (<b>19</b>) was selectively protected by acetylation via heating with acetic anhydride at 100 °C for 2 h in the presence of pyridine as a basic catalyst to afford 4-hydroxy-6-quinazolinyl acetate (<b>20</b>).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The <sup>1</sup>H NMR spectrum of compound (<b>20</b>) exhibited a singlet signal at δ 2.31 attributed to the three protons of the CH<sub>3</sub> group accompanied by the disappearance of the C-6 phenolic OH signal at δ 10.05. Chlorination of (<b>20</b>) was carried out by heating with thionyl chloride.<a onclick="showRef(event, 'ref57 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref59">(57,59)</a> The freshly prepared 4-chloro derivative (<b>21</b>) was unstable; therefore, it was immediately coupled with the appropriate aniline derivative (<b>4a–c</b>) by long-term reflux in isopropanol, in the presence of trimethylamine (TEA),<a onclick="showRef(event, 'ref44 ref57'); return false;" href="javascript:void(0);" class="ref ref44 ref57">(44,57)</a> or in <i>N</i>-methyl pyrrolidinone.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> However, using the reported methods, the desired compounds (<b>22a–c</b>) were obtained in very low yields (20–42%) with multispot side products. Thus, we significantly optimized the coupling reaction using acetonitrile/K<sub>2</sub>CO<sub>3</sub>, either at room temperature (for <b>22a</b>) or under reflux (for <b>22b</b>,<b>c</b>), to afford the desired intermediates in good purity and with higher yields (60–90%). The melting point of compound <b>22a</b> was consistent with the reported one (290–292 °C).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Compounds <b>22b</b> and <b>22c</b> were confirmed by their <sup>1</sup>H NMR spectral data with the appearance of the signals of eleven aromatic protons at δ 6.74–8.69. Hydrolysis of the acetylated intermediates (<b>22a–c</b>) to the corresponding quinazolin-6-ol derivatives (<b>23a–c</b>) was conducted using aqueous ammonia in methanol (1:3) at room temperature up to 12 h. The melting point of the compound (<b>23a</b>) was consistent with the reported one (>300 °C).<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> The new compounds (<b>23b</b> and <b>23c</b>) were confirmed by their <sup>1</sup>H NMR spectra showing a singlet exchangeable signal at δ 10.14 of the 6-OH proton with a concurrent disappearance of the singlet signal at δ 2.38 and the <sup>13</sup>C DEPT signal at δ 21.21–21.22 of the CH<sub>3</sub> group. Alkylation of compounds (<b>23a–c</b>) was carried out by reflux with 1-bromo-3-chloropropane in the presence of K<sub>2</sub>CO<sub>3</sub> and a catalytic amount of tetrabutylammonium bromide (TBAB).<a onclick="showRef(event, 'ref57 ref60'); return false;" href="javascript:void(0);" class="ref ref57 ref60">(57,60)</a></div><div class="NLM_p last">The new compounds (<b>24b</b> and <b>24c</b>) were elucidated spectroscopically through the appearance of two triplets at δ 4.26–4.28 and δ 3.85 and one pentet at δ 2.25 corresponding to the six protons of the chloropropoxy moiety, along with the disappearance of a singlet signal at δ 10.14 of the 6-OH proton. In addition, their <sup>13</sup>C DEPT spectra revealed three inverted signals at δ 32.13–32.15, δ 42.43–42.46, and δ 65.59–65.61 which represent three CH<sub>2</sub> groups of the chloropropoxy moiety. The target compounds (<b>25a–g</b>) could be prepared through nucleophilic substitutions on (<b>24a–c</b>) with the appropriate secondary amines in dry acetonitrile, by heating under reflux in the presence of anhydrous K<sub>2</sub>CO<sub>3</sub> a catalytic amount of potassium iodide.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> In the current study, the target compounds (<b>25a–g</b>) were synthesized by refluxing the mixture of compounds (<b>24a–c</b>) with 100 equiv of appropriate secondary amine, including <i>N</i>-methyl piperazine, morpholine, and diethylamine. Compounds (<b>25a–g</b>) were afforded in excellent yields (76–94%) and high purity. On the other hand, because of the high nucleophilic reactivity of ethanolamine, compound <b>25h</b> could not be prepared under the same reaction condition adopted for (<b>25a–g</b>), where multispot degradation byproducts were obtained. Therefore, compound <b>25h</b> was prepared by heating under reflux a mixture of intermediate <b>24c</b> and excess ethanolamine (10 equiv) in MeOH, rather than the solvent-free reaction, in the presence of a catalytic amount of TEA to afford compound (<b>25h</b>) with 42% yield. <sup>1</sup>H NMR spectra of compounds <b>25a</b> and <b>25b</b> exhibited singlet signals at δ 5.14 and δ 5.25, respectively, for their characteristic benzylic protons (OC<i>H</i><sub>2</sub>C<sub>6</sub>H<sub>4</sub>F) with a concurrent appearance of two broad singlets at δ 2.48 and δ 3.74 for the eight protons of the morpholino moiety or at δ 2.30 and δ 2.38 for the eight protons of the piperazino moiety, respectively. In addition, the six protons of the propoxy linker were detected as two triplets and one pentet at δ 1.90–4.18. Moreover, <sup>13</sup>C DEPT spectra revealed six signals for the CH<sub>2</sub> groups, whereas compounds <b>25c–h</b> (missing benzylic protons) were confirmed by their aliphatic signals of the corresponding secondary or primary amines.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> <i>In Vitro</i> Protein Kinase Inhibition Assay</h3><div class="NLM_p">The protein kinase profiles of seven compounds (<b>10a</b>, <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b>) against a panel of 20 protein kinases using the [γ-<sup>32</sup>P]ATP radiometric assay method were evaluated by Kinexus Bioinformatics Corporation, Vancouver, British Columbia, Canada. Two reference kinase inhibitors on the phosphotransferase activity were used including lapatinib (EGFR and HER2 dual selective TKI) and sorafenib (nonselective potent multikinase inhibitor). This assay was performed at 10 μM concentration in a single measurement by employing the standardized assay methodology outlined in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> part.</div><div id="sec2_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> Rationale for Selection of the 20 Kinase Panels</h4><div class="NLM_p last">The EGFR (HER1), HER2, and HER4 kinases were selected as the most common targets in therapy of aggressive BC. ABL kinase was selected as an antitarget kinase in BC which was reported to negatively modulate HER2+ BC progression <i>in vivo</i>.<a onclick="showRef(event, 'ref62 ref63'); return false;" href="javascript:void(0);" class="ref ref62 ref63">(62,63)</a> Meanwhile, BRAF, ERK1, and ERK2 kinases were picked out, being a downstream signal transducer in the EGFR family. The SRC activity was reported to be the main mode of resistance to Herceptin as the first-line therapy for HER2+ BC.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> The FAK is overexpressed and activated in several advanced-stage solid cancers including multidrug-resistance BC.<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a> JAK2 was selected because the JAK2-STAT3 signaling pathway is known as a crucial signaling pathway for regulating cancer progression and metastasis.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> C-kit, c-Met, FLT1,4, KDR, and PDGFRα kinases are involved in angiogenesis and were reported as effective targets in the treatment of solid cancers. MAPKAPK2, ROCK2, and p38α kinases were used as examples of different target serine/threonine kinases to explore selectivity to our target tyrosine kinase.</div></div><div id="sec2_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i12" class="anchor-spacer"></div><h4 class="article-section__title" id="_i12"> Selection Criteria for the Test Compounds for Kinase Profiling</h4><div class="NLM_p">Seven compounds were screened for kinase profiling including the lead compound (<b>10a</b>) and its morpholino analogue (<b>10b</b>). Other different compounds bearing an acetamido spacer with a 4-phenoxyaniline moiety (<b>15a</b>, <b>15e</b>, and <b>15l</b>), compound <b>17d</b> bearing a benzamido spacer and a 4-phenoxyaniline moiety, and compound <b>25f</b> with a propoxy spacer and a 4-phenoxyaniline moiety. The selected candidates are representative of the optimized analogues of the lead compound (<b>10a</b>; generated by CADD).</div><div class="NLM_p">Throughout this study, the inhibition of target activity by the compound was given negative (−) values while activation of target activity was given positive (+) values, and only >25% changes in activity compared to the control were considered to be significant. Six out of 20 kinases tested by compound <b>17d</b> exhibited moderate-to-high inhibition scores ranging from −40 to −98% including HER4 (−98%), HER2 (−92%), EGFR (HER1) (−70%), ROCK2 (−47%), JAK2 (−47%), and ABL1 (−40%), whereas two kinases showed an activation, including c-Kit (31%) and FLT4 (57%), as shown in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>.</div><figure id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Protein kinase profiling for compounds (<b>10a</b>, <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b>) at 10 μM single dose concentration, represented by % activity or % inhibition, against a panel of 20 kinases in comparison with two reference compounds lapatinib and sorafenib using the [γ-<sup>32</sup>P]ATP radiometric assay method. (−) values: inhibition of target activity; (+) values: activation of target activity; >−25% changes in activity compared to the control were considered to be significant inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">For compound <b>15a</b>, four kinases revealed moderate-to-high inhibition scores ranging from −33 to −97% including HER4 (−97%), HER2 (−92%), EGFR (−67%), and JAK2 (−33%). Similar to <b>17d</b>, c-Kit (46%) and FLT4 (47%) were moderately activated in the presence of the compound <b>15a</b> as well as ERK1 (32%). In the presence of <b>15e</b>, five kinases were inhibited ranging from −37 to −94% including HER4 (−94%), HER2 (−91%), EGFR (−71%), JAK2 (−58%), and ABL1 (−37%). The other five kinases were activated, including KDR (38%), SRC (38%), FLT1 (51%), PDGFR alpha (82%), and c-Kit (115%). Compound <b>15l</b> inhibited five kinases from moderate-to-high level ranging from −49 to −99% inhibition, including HER4 (−99%), HER2 (−93%), EGFR (−78%), JAK2 (−70%), and ABL1 (−49%), while two kinases were activated, including c-Kit (43%) and PDGFR-α (86%). Testing of compound <b>25f</b> revealed moderate-to-high inhibitions ranging from −65 to −98% against 5 kinases including HER4 (−99%), HER2 (−92%), JAK2 (−87%), EGFR (−65%), and ALB1 (−65%), with activations noted with c-Kit (39%), KDR (40%), ERK1 (41%), and PDGFR-α (115%). Compound <b>10a</b> exhibited high inhibition against HER4 (−98%), HER2 (−93%), ABL1 (−86%), JAK2 (−86%), and EGFR (−78%), whereas it activated KDR (32%) and PDGFR-α (54%). Similarly, compound <b>10b</b> showed high inhibition of HER4 (−97%), HER2 (−92%), ABL1 (−86%), EGFR (−81%), and JAK2 (−74%) and moderate activation toward KDR (46%), FLT1 (49%), and PDGFR-α (55%). Regarding the reference compounds used in this study, sorafenib as a potent nonselective multikinase inhibitor was tested at 10 μM against the same panel of 20 protein kinases. It showed strong inhibitions against ABL1 (−97%), FLT4 (−97%), JAK2 (−94%), and B-RAF (−92%), whereas it revealed moderate inhibitions against FLT1 (−78%), EGFR (−74%), c-Kit (−63%), HER4 (−60%), PDGFR-α (−55%), and P38-α (−46%). Lapatinib was used as a selective dual kinase inhibitor, which interrupts the HER2/neu and EGFR pathways. Five out of 20 tested kinases were moderately to strongly inhibited including HER4 (−99%), JAK2 (−97%), ABL1 (−90%), EGFR (−80%), and HER2 (−52%), while c-Kit, FLT1, KDR, and PDGFR-α kinases were moderately to highly activated within 37–86%, and these activations increase the possibility that in some cancer patients, lapatinib could in principle exacerbate their cancer.</div><div class="NLM_p">The abovementioned results indicated that our newly synthesized compounds revealed a significant selective inhibition against HER2 and HER4 kinases within −92 to −99% and to a moderate extent against the subfamily closely related to RTK EGFR (HER1) within −67 to −81%. These results highly support our design for selective HER2 kinase inhibitors for the treatment of aggressive BC. Furthermore, the test compounds exhibited moderate-to-substantial kinase inhibitory effects against ABL1 and JAK2 kinases within −25 to −86% and −33 to −87%, respectively. In comparison with lapatinib as a dual EGFR/HER2 kinase inhibitor, compounds <b>10a</b>, <b>15a</b>, and <b>17d</b> and to a lower extent <b>10b</b> are considered as more safe compounds in activating c-KIT, FLT1, KDR, and PDGFR-α kinases. Various studies reported that PDGFR expression (including PDGFR-α and β and both show similar kind of functions) is associated with poor prognosis of BC patients via activating of Stat3 to support BC initiation and progression.<a onclick="showRef(event, 'ref68 ref69 ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref68 ref69 ref70 ref71">(68−71)</a> Accordingly and in comparison with lapatinib (activating PDGFR-α by 56%), compounds <b>15a</b> and <b>17d</b> (affecting PDGFR-α by 18% and −4, respectively) are considered as the safest and potent compounds for the treatment of aggressive HER2+ BC.</div><div class="NLM_p">The potent inhibitory activity of compound <b>17d</b> against HER2 kinase (−92%) is highly correlated with its robust binding affinity into human HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>) with a GoldScore fitness of 92.63 compared to 84.59 for the native bound ligand. Compound <b>17d</b> was docked into HER2 kinase by two hydrogen bonds between its quinazoline-N<sup>1</sup>H and terminal ethanolamine-OH and NH moiety of M801 and NH of S728 amino acids, respectively. Besides, hydrophobic interaction plays a crucial role in the binding of compound <b>17d</b> into the intensively hydrophobic HER2 kinase domain, where its planar quinazoline ring interacts hydrophobically with L726, L852, Ala751, and M801 by π–σ and π–alkyl interactions, as clearly depicted in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>. Moreover, its <i>m</i>-chloro-6-aniline moiety interacts comprehensively by π–alkyl interactions with the V734 and K753 bond, whereas the <i>m</i>-chloro atom interacts hydrophobically by the halogen acceptor with L796 and K753 residues. Noticeably, the terminal phenoxy moiety was deeply harbored into the hydrophobic HER2 selectivity pocket through four π–π and π–alkyl interactions with F864, M774, L785, and L796. Accordingly, compound <b>17d</b> demonstrated a GoldScore (external vdw) of 62.94.</div><figure id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking of compound <b>17d</b> (ball and stick, colored by element) into human HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>) within an rmsd of 3.1 Å from the native cocrystallized ligand (03Q). It exhibited two hydrogen bonds (green-dotted lines) with M801 and S728 in addition to multiple hydrophobic interactions. The color codes are hydrophobic (π–alkyl and alkyl–alkyl): ---; hydrophobic (π–π): ---; hydrophobic (π–σ): ---; halogen acceptor bonding: --- <i>m</i>-chloro atom interacts hydrophobically by a halogen acceptor with L796 and K753 residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> <i>In Vitro</i> HER2 Kinase Inhibitory Activity (IC<sub>50</sub>) against Wild and L755S Mutant Kinases</h3><div class="NLM_p">The ADP-Glo Assay method was performed for kinase profiling of five compounds (<b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>) to evaluate their IC<sub>50</sub> using concentrations 0.5 nM to 25.0 μM. Two protein kinases (HER2 wild-type and HER2 L755S mutant) were used in a single measurement. Lapatinib was used as a positive control at the same used concentrations.</div><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i14" class="anchor-spacer"></div><h4 class="article-section__title" id="_i14"> <i>In Vitro</i> HER2 Kinase Inhibitory Activity against Wild Type</h4><div class="NLM_p">Compounds <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> revealed the most potent activities against HER2+ BC cell lines (BT474 and AU565; IC<sub>50</sub>: 0.08–0.20 μM), in comparison with the lead compound <b>10a</b> (IC<sub>50</sub>: 0.22–0.24 μM), respectively.</div><div class="NLM_p">Out of these compounds, <b>15i</b>, <b>17d</b>, and <b>25b</b> were further selected by NCI for 5-dose assay against NCI60 cancer cell lines. Therefore, these compounds were selected for further evaluation of their IC<sub>50</sub> against wild-type HER2 kinase. At the initially used concentrations (0.01–25 μM), all the test compounds exhibited high inhibitions (50–91%) greater than lapatinib (14–88%; IC<sub>50</sub> = 95.5 nM) against HER2 wild-type kinase, as shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>A. Therefore, concentrations of 0.5–100 nM were further used to calculate the IC<sub>50</sub> of the test compounds. At 100 nM, compound <b>25b</b> revealed the highest inhibition (91%) and at 0.5 nM, all the compounds exhibited inhibition of HER2 wild-type kinase at 3–25%, as shown in <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>B. All five test compounds (<b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>) have IC<sub>50</sub> scores ranging from 5.4 to 12.0 nM. In all cases, IC<sub>50</sub> values for the test compounds were far lower than the IC<sub>50</sub> value for lapatinib 95.5 nM, as listed in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</div><figure id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. % IC<sub>50</sub> graph and calculations for the five compounds (<b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>) and lapatinib tested against HER2 wild-type kinase using the ADP-Glo assay method at (A) concentrations 0.01–25.0 μM and (B) concentrations 0.5–100 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> of Compounds <b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> in Comparison with Lapatinib Tested against HER2 Wild-Type Kinase at Concentrations Ranging from 0.5 nM to 25 μM Using the ADP-Glo Assay Method</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">IC<sub>50</sub> (nM)</th><th class="colsep0 rowsep0" align="center">95% confidence intervals</th><th class="colsep0 rowsep0" align="center">SEM (nM)</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center" char=".">DOF</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10a</b></td><td class="colsep0 rowsep0" align="char" char=".">10.4</td><td class="colsep0 rowsep0" align="left">5.512–19.430</td><td class="colsep0 rowsep0" align="left">±1.159</td><td class="colsep0 rowsep0" align="char" char=".">0.9958</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10c</b></td><td class="colsep0 rowsep0" align="char" char=".">6.1</td><td class="colsep0 rowsep0" align="left">4.679–7.969</td><td class="colsep0 rowsep0" align="left">±1.064</td><td class="colsep0 rowsep0" align="char" char=".">0.9993</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15i</b></td><td class="colsep0 rowsep0" align="char" char=".">5.4</td><td class="colsep0 rowsep0" align="left">4.291–6.666</td><td class="colsep0 rowsep0" align="left">±1.052</td><td class="colsep0 rowsep0" align="char" char=".">0.9995</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17d</b></td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="left">8.412–17.114</td><td class="colsep0 rowsep0" align="left">±1.085</td><td class="colsep0 rowsep0" align="char" char=".">0.9989</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25b</b></td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="left">3.757–18.404</td><td class="colsep0 rowsep0" align="left">±1.202</td><td class="colsep0 rowsep0" align="char" char=".">0.9922</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lapatinib</td><td class="colsep0 rowsep0" align="char" char=".">95.5</td><td class="colsep0 rowsep0" align="left">43.97–207.40</td><td class="colsep0 rowsep0" align="left">±1.276</td><td class="colsep0 rowsep0" align="char" char=".">0.9939</td><td class="colsep0 rowsep0" align="char" char=".">5</td></tr></tbody></table></div></div></div><div id="sec2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i16" class="anchor-spacer"></div><h4 class="article-section__title" id="_i16"> <i>In Vitro</i> HER2 Kinase Inhibitory Activity against L755S Mutant Type</h4><div class="NLM_p">HER2 pathway reactivation by the HER2/L755S mutation was found to be the common somatic mutation which constitutes a mechanism of acquired resistance to lapatinib-containing HER2-targeted therapies in preclinical HER2+ BC models.<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> The most frequently altered codon in HER2 exon 19 that encodes the tyrosine kinase intracellular domain was L755 (10 mutations, L755S);<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> therefore, the kinase domain mutant HER2 L755S was used in this study in which HER2 is mutated in 14.05% of breast carcinoma patients with ERBB2 L755S present in 0.74% of all breast carcinoma patients,<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74,75)</a> as depicted in <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>. However, none of the compounds showed a strong inhibition with the HER2-L755S mutant kinase, and the inhibitions ranged from 12% at the lowest concentration of 0.1 μM of the test compounds up to the highest inhibition of 39% at a concentration of 25 μM (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). The lapatinib followed the same pattern of inhibition as the test compounds. According to these results, there were no IC<sub>50</sub> values determined for the compounds tested with HER2 L755S mutant kinase as the highest inhibition score was only 39%.</div><figure id="fig10" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Top disease cases with HER2 L755S (data retrieved from refs <a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and <a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. % Activity Change in Compounds <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> in Comparison with Lapatinib Tested against HER2 L755S Mutant Kinase at Concentrations Ranging from 0.1 to 25 μM Using the ADP-Glo Assay Method</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">0.1 μM</th><th class="colsep0 rowsep0" align="center" char=".">0.5 μM</th><th class="colsep0 rowsep0" align="center" char=".">1.0 μM</th><th class="colsep0 rowsep0" align="center" char=".">5.0 μM</th><th class="colsep0 rowsep0" align="center" char=".">10 μM</th><th class="colsep0 rowsep0" align="center" char=".">25 μM</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>10c</b></td><td class="colsep0 rowsep0" align="char" char=".">–16</td><td class="colsep0 rowsep0" align="char" char=".">–21</td><td class="colsep0 rowsep0" align="char" char=".">–26</td><td class="colsep0 rowsep0" align="char" char=".">–29</td><td class="colsep0 rowsep0" align="char" char=".">–32</td><td class="colsep0 rowsep0" align="char" char=".">–33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15i</b></td><td class="colsep0 rowsep0" align="char" char=".">–12</td><td class="colsep0 rowsep0" align="char" char=".">–20</td><td class="colsep0 rowsep0" align="char" char=".">–20</td><td class="colsep0 rowsep0" align="char" char=".">–24</td><td class="colsep0 rowsep0" align="char" char=".">–33</td><td class="colsep0 rowsep0" align="char" char=".">–28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17d</b></td><td class="colsep0 rowsep0" align="char" char=".">–15</td><td class="colsep0 rowsep0" align="char" char=".">–19</td><td class="colsep0 rowsep0" align="char" char=".">–20</td><td class="colsep0 rowsep0" align="char" char=".">–22</td><td class="colsep0 rowsep0" align="char" char=".">–27</td><td class="colsep0 rowsep0" align="char" char=".">–29</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25b</b></td><td class="colsep0 rowsep0" align="char" char=".">–12</td><td class="colsep0 rowsep0" align="char" char=".">–19</td><td class="colsep0 rowsep0" align="char" char=".">–22</td><td class="colsep0 rowsep0" align="char" char=".">–25</td><td class="colsep0 rowsep0" align="char" char=".">–31</td><td class="colsep0 rowsep0" align="char" char=".">–39</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">lapatinib</td><td class="colsep0 rowsep0" align="char" char=".">–14</td><td class="colsep0 rowsep0" align="char" char=".">–18</td><td class="colsep0 rowsep0" align="char" char=".">–23</td><td class="colsep0 rowsep0" align="char" char=".">–31</td><td class="colsep0 rowsep0" align="char" char=".">–35</td><td class="colsep0 rowsep0" align="char" char=".">–37</td></tr></tbody></table></div></div><div class="NLM_p last">It is known that the L755S mutation correlates with activated HER2 signaling and transformation, but this may require complementary mutations.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> It has also been previously reported that HER2 T733I and L755S mutations are associated with resistance to inhibition by lapatinib <i>in vitro</i>.<a onclick="showRef(event, 'ref77 ref78 ref79'); return false;" href="javascript:void(0);" class="ref ref77 ref78 ref79">(77−79)</a></div></div><div id="sec2_4_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i18" class="anchor-spacer"></div><h4 class="article-section__title" id="_i18"> Antiproliferative Activity Assay</h4><div class="NLM_p">The antiproliferative activity of the newly synthesized compounds was conducted by WST-8 assay against a panel of four different HER2-expressing BC cell lines. The selected panel has different genetic backgrounds including wild-type luminal AU565 cell line with abnormally high expressing HER2,<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> TPBC (HER2+/ER+/PR+) BT474 cells, estrogen and progesterone receptor-positive (ER+/PR+) MCF7 cells, and triple-negative (MDA-MB-231) cells. These four BC cell lines represent two different genetic backgrounds of tumor suppressor p53, including AU565/p53 wild type (wt), BT474/p53 mutant (mt), MCF-7/p53 (wt), and MDA-MB-231/p53 (mt). Phenotyping and molecular features of the cell lines used in our study are listed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Characterization, Molecular Information, Culture Conditions, and Clinical Features for the Four BC Cell Lines Used in Our Study; the Information was Retrieved from Ref <a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a><a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">cell line</th><th class="colsep0 rowsep0" align="center">HER2</th><th class="colsep0 rowsep0" align="center">ER</th><th class="colsep0 rowsep0" align="center">PR</th><th class="colsep0 rowsep0" align="center">P53</th><th class="colsep0 rowsep0" align="center">origin</th><th class="colsep0 rowsep0" align="center">subtype</th><th class="colsep0 rowsep0" align="center">tumor</th><th class="colsep0 rowsep0" align="center">culture medium</th><th class="colsep0 rowsep0" align="center">references</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AU565</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">NA</td><td class="colsep0 rowsep0" align="left">metastasis</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">AC</td><td class="colsep0 rowsep0" align="left">DMEM/F12</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref81 ref83'); return false;" href="javascript:void(0);" class="ref ref81 ref83">(81,83)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BT474</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">mt</td><td class="colsep0 rowsep0" align="left">primary</td><td class="colsep0 rowsep0" align="left">LB</td><td class="colsep0 rowsep0" align="left">IDC</td><td class="colsep0 rowsep0" align="left">DMEM/F12</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref81 ref82 ref84 ref85 ref86 ref87'); return false;" href="javascript:void(0);" class="ref ref81 ref82 ref84 ref85 ref86 ref87">(81,82,84−87)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCF7</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">+</td><td class="colsep0 rowsep0" align="left">wt</td><td class="colsep0 rowsep0" align="left">metastasis</td><td class="colsep0 rowsep0" align="left">LA</td><td class="colsep0 rowsep0" align="left">IDC</td><td class="colsep0 rowsep0" align="left">RPMI, DMEM</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref7 ref9 ref81 ref83'); return false;" href="javascript:void(0);" class="ref ref7 ref9 ref81 ref83">(7,9,81,83)</a></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MDA-MB-231</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">–</td><td class="colsep0 rowsep0" align="left">mt</td><td class="colsep0 rowsep0" align="left">metastasis</td><td class="colsep0 rowsep0" align="left">TNB</td><td class="colsep0 rowsep0" align="left">AC</td><td class="colsep0 rowsep0" align="left">RPMI, DMEM</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref3 ref4 ref7 ref9 ref86'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref7 ref9 ref86">(3,4,7,9,86)</a></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">HER2: human epidermal growth factor receptor 2; ER: estrogen receptor; PR: progesterone receptor; NA: not available; wt: wild type; mt: mutant type; H: HER2 positive; LA: luminal A; LB: luminal B; TNB: triple-negative B; RPMI: Roswell Park Memorial Institute (RPMI) 1640 medium; DMEM: Dulbecco’s modified Eagle medium; AC: adenocarcinoma; and IDC: invasive ductal carcinoma.</p></div></div></div><div class="NLM_p">Lapatinib is an FDA-approved dual EGFR/HER2 TKI for the treatment of HER2+ metastatic BC was used as a positive control.</div><div class="NLM_p">Our compounds revealed statistically significant selective and potent antiproliferative activities against HER2+ BC cells (AU565 and BT474) of mean ± SEM (0.295 ± 0.03207 μM) compared to HER2– BC cells (MCF7 and MDA-MB-231) of mean ± SEM (5.817 ± 0.9121 μM). Notably, all the synthesized compounds were tested against human metastatic high expressing HER2 BC cells (AU565) and revealed potent antiproliferative activities with an IC<sub>50</sub> of 0.099–0.923 μM range. Compounds <b>10b</b>, <b>15h</b>, <b>15i</b>, <b>15k</b>, <b>25a</b>, and <b>25b</b> are highly potent with IC<sub>50</sub> less than 0.2 μM, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>. Remarkably, compound <b>25b</b> (IC<sub>50</sub> = 0.099) showed the most potent activity and most comparable one to lapatinib (IC<sub>50</sub> = 0.064 μM). To a lower extent, 75% of other compounds exhibited antiproliferative activity less than 0.5 μM in the range of 0.205–0.456 μM against the same cell line.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Antiproliferative Activity (IC<sub>50</sub>; μM) of Compounds <b>10a–25h</b> against Four BC AU565, BT474, MCF7, and MDA-MB-231 Cell Lines</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0024.gif" alt="" id="gr22" /></img><div></div></div><div class="NLM_p">These results indicate that our designed compounds significantly target HER2 kinase, which is highly expressed in AU565 BC cells. Interestingly, 23 out of 27 (85%) of the designed compounds showed an improved IC<sub>50</sub> of 0.081–0.540 μM against triple-positive BT474 BC cells with more potency than HER2 BC cells (AU565). These results also indicate that the quinazoline derivatives selectively target HER2 kinase in addition to other oncologic pathways, which have been investigated and discussed under apoptosis assay and western blotting. Noticeably, multitargeting of the triple-positive BT474 BC cell was detected by 85% of the tested compounds. This augmented effect was noticed to a higher extent for compounds <b>15d</b>, <b>15l</b>, <b>17a</b>, <b>25d</b>, <b>25f</b>, and <b>25h</b> up to 6 times antiproliferative activity compared to their IC<sub>50</sub> toward AU565 BC cells (HER2+), whereas compounds <b>10b</b>,<b>c</b>, <b>15a–c</b>, <b>15e</b>,<b>f</b>,<b>h</b>,<b>i</b>,<b>k</b>, <b>17b–d</b>, <b>25a–c</b>, <b>25e</b>, and <b>25g</b> exhibited a lower boost (up to 2.0 times) of antiproliferative activity by targeting other receptors/pathways. Likewise to their effects against AU565 cells, compounds <b>10b–c</b>, <b>15d</b>,<b>h</b>,<b>i</b>,<b>k</b>,<b>l</b>, and <b>25a</b>,<b>b</b>,<b>f</b> exhibited high antiproliferative activity toward BT474 cells with IC<sub>50</sub> ≤ 0.2 μM, as depicted in <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>. Strikingly, compounds <b>15h</b>, <b>15i</b>, <b>25a</b>, and <b>25b</b> revealed the most potent and comparable activities to lapatinib with an IC<sub>50</sub> of 0.091, 0.081, 0.081, and 0.083 μM, respectively. The potent antiproliferative activity of compound <b>25b</b> against HER2+ BC cells is a reflection of its high binding affinity (GoldScore fitness 90.70) into human HER2 kinase compared to 84.59 for the native bound ligand. Compound <b>25b</b> was docked proximately superimposed on the native cocrystallized 03Q ligand within a root-mean-square deviation (rmsd) of 1.1 Å. It bound to the binding site of the titled kinase by one hydrogen bond between its quinazoline-N<sup>1</sup>H and HN moiety of M801 as a key amino acid, as shown in <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>. Despite compounds <b>15h</b>, <b>15i</b>, <b>25a</b>, and <b>25b</b> that showed the lowest IC<sub>50</sub>, compound <b>15d</b> displayed the highest augmentation of antiproliferative activity by targeting triple-positive BT474 BC cells (IC<sub>50</sub> = 0.107 μM) of ∼6.0 times more potency than AU565 (0.663 μM). The antiproliferative activities of the tested compounds were noticeably diminished using other BC cells lacking HER2 kinase such as estrogen and progesterone receptor-positive (ER+/PR+) MCF7 cells and triple-negative (MDA-MB-231) cells (IC<sub>50</sub> ranges from 0.321 to 18.119 and 0.737 to 30.162 μM, respectively). In contrast, the IC<sub>50</sub> ranges of the HER2+ BC cells (AU565 and BT474) are from 0.099 to 0.923 and 0.081 to 0.540 μM, respectively. This means that MCF7 and MDA-MB-231 cells are less sensitive to the inhibitory effects of the tested compounds than the BC cells harboring HER2 kinase. Nevertheless, the majority (74–89%) of our designed quinazoline derivatives exhibited superior inhibition of MCF7 and MDA-MB231 cells than lapatinib. Compounds <b>10c</b>, <b>15d</b>, <b>25b</b>, <b>17b</b>, <b>25d</b>, and <b>25f–h</b> inhibited cell proliferation of MCF7 cells with IC<sub>50</sub>: 0.321–1.933 μM, implying their greater potency (up to 20 times) more than lapatinib (IC<sub>50</sub>: 6.301 μM), and specifically, compounds <b>15d</b> (IC<sub>50</sub>: 0.321 μM) and <b>25d</b> (IC<sub>50</sub>: 0.913 μM) exhibited the lowest IC<sub>50</sub> values. Remarkably, the antiproliferative activities of compounds <b>15d</b>, <b>15f</b>, <b>15i</b>, <b>15k</b>, <b>17b</b>, <b>25b</b>, <b>25d</b>, and <b>25f–h</b> against MCF7 cells (IC<sub>50</sub>: 0.321–2.647 μM) are highly consistent with their augmented antiproliferative effects shown toward AU565 and BT474 breast cell lines. These results support the suggestion that our compounds are selectively targeting HER2+ (in AU565 cells) and get boosted by targeting other receptors as shown in triple-positive (HER2+/ER+/PR+) BT474 cells, and to a lower extent, they can influence the estrogen and progesterone receptors (ER+/PR+) as depicted in MCF7 cells in the absence of HER2 as a main target. On the other hand, the majority of the tested compounds (∼67%) displayed poor antiproliferative activities against triple-negative (MDA-MB-231) BC cell lines with IC<sub>50</sub>: 4.503–30.162 μM, which represents 15–100% times higher IC<sub>50</sub> than that against HER2-expressing cells (AU565 and BT474). This is clearly explained by the absence of HER2 and other boosting targets such as ER/PR expressed in these cells. To a lower extent, compounds <b>10c</b>, <b>15c</b>, <b>15f</b>, <b>15g</b>, <b>15j</b>, <b>17b</b>, <b>25b</b>, <b>25d</b>, and <b>25f–h</b> exhibited an acceptable IC<sub>50</sub> in the range of 0.737–3.924 μM, that is, up to 13 times less active than their corresponding activities against AU565 and BT474 breast cell lines. The reasonable explanation of this slight activity against triple-negative (MDA-MB-231) BC cell lines is the expression of 3% of HER2 kinase in comparison with 100% in triple-positive (HER2+/ER+/PR+) BT474 cells, whereas MDA-MB-231 cells expressed about 9 times EGFR (also known as HER1) more than BT474 cells.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> Furthermore, HER1 which is overexpressed in 15–30% of breast carcinoma<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> is frequently associated with poor prognosis in triple-negative BC.<a onclick="showRef(event, 'ref91 ref92 ref93'); return false;" href="javascript:void(0);" class="ref ref91 ref92 ref93">(91−93)</a> The aforementioned facts might explain the slight activities of some compounds toward triple-negative (MDA-MB-231) BC cell lines.</div><figure id="fig11" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Dose–response curves for cell viability by WST-8 assay of AU565 and BT474 cells and their corresponding IC<sub>50</sub> values (standard error are given). Cells were treated with compounds <b>10c</b>, <b>15d</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> at concentrations 0.01–100 μM for 72 h. Compounds were tested in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig12" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Docking of compound <b>25b</b> (ball and stick, colored by element) into human HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). It exhibited one hydrogen bond (green dotted lines) with M801. It was docked within an rmsd of 1.10 Å from the native cocrystallized 03Q ligand (shown as yellow sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> <i>In Vitro</i> Primary Single-Dose (10 μM Concentration) Screening against the NCI 60 Human Cell Panel</h3><div class="NLM_p">The detailed discussion of single-dose and 5-dose screening toward NCI 60 human cancer cell lines and the cytotoxicity mean graphs from NCI is presented in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Supporting Information</a>. Single-dose testing was performed at 10 μM concentration against 60 cell lines, representing nine tumor subpanels. As depicted in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S1</a>, many of the test-substituted quinazoline compounds exhibited distinctive patterns of selectivity and sensitivity against different NCI cell panels. Remarkably, compounds <b>15f</b>, <b>15i</b>, <b>17c</b>, <b>17d</b>, <b>25b</b>, <b>25d</b>, and <b>25f–h</b> exhibited mean cell growth and lethality in the range of 11.85–19.30% and −1.30 to −31.03%, respectively. Therefore, these potent compounds in addition to <b>15g</b>, <b>15k</b>, <b>17a</b>, and <b>25e</b> were selected by the DTP program in NCI for further 5-dose screening using the mean growth percent of 47.7% as a minimum criterion for inhibition. Compounds <b>15f</b>, <b>15i</b>, <b>17c</b>, <b>17d</b>, <b>25b</b>, <b>25d</b>, and <b>25f–h</b> exhibited broad spectrum cell growth inhibition and lethality against BC up to −78.7%. All test compounds exhibited significant inhibition and lethality in the range of 18.8 to −66.0% against MDA-MB-468 cells. Compounds <b>15f</b>, <b>17c</b>, <b>25b</b>, and <b>25g</b> exhibited significant lethality in the range of −11.7 to −78.7% against MCF7, MDA-MB-231, HS578T, and MDA-MB-468 cells, whereas compounds <b>15i</b>, <b>17d</b>, <b>25f</b>, and <b>25h</b> showed lethality in the range of −18 to −74.6% against MCF7, MDA-MB-231, and MDA-MB-468 cells. Unfortunately, none of the six human BC cells used by the DTP in NCI was the HER2+ cell. Among these six cancer cells, four cells, namely, MDA-MB-231, HS578T, BT549, and MDA-MB-468, are characterized by the lack of expression of the three main molecular markers (ER, PR, and HER2) and referred as triple-negative,<a onclick="showRef(event, 'ref84 ref94'); return false;" href="javascript:void(0);" class="ref ref84 ref94">(84,94)</a> whereas MCF-7 and T-47D breast cell lines are ER+/PR+, making them an ideal model to study hormone response.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div><div id="sec2_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> <i>In Vitro</i> Five-Dose Full NCI 60 Cell Panel Screening</h4><div class="NLM_p last">Surprisingly, out of 20 compounds tested initially at a single high dose (10<sup>–5</sup> M), 13 compounds were selected to be screened by five-dose concentration against the full NCI 60 cell panel. These compounds, namely, <b>15f</b>, <b>15g</b>, <b>15i</b>, <b>15k</b>, <b>17a</b>, <b>17c</b>, <b>17d</b>, <b>25b</b>, <b>25d</b>, <b>25e</b>, <b>25f</b>, <b>25g</b>, and <b>25h</b>, exhibited significant growth inhibitions in the single-dose screen. The criteria for efficiently capturing these thirteen compounds and progressing them to the 5-dose antiproliferative activity were based on careful analysis of historical DTP screening data. In this assay, the test compounds were incubated at five different concentrations (0.01, 0.1, 1.0, 10, and 100 μM). Three dose–response parameters (GI<sub>50</sub>, TGI, and LC<sub>50</sub>) were calculated for each compound against each cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Table S2</a>). Compounds <b>15i</b>, <b>25b</b>, and <b>25g</b> demonstrated the most potent antiproliferative activities against the tested NCI 60 cell panel. Compounds <b>15i</b> and <b>25b</b> exhibited the same pattern of potency against MDA-MB-468, the triple-negative BC cells with GI<sub>50</sub> = 0.4 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S2</a>).</div></div><div id="sec2_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> Sensitivity of the Test Compounds toward <b>9</b> Subpanels of NCI 60 Cell Lines</h4><div class="NLM_p last">To a higher extent, the test compounds <b>15i</b> and <b>25b</b> exhibited impressive anticancer activity against most of the NCI 60 cell lines representing nine different subpanels with GI<sub>50</sub> values between 0.4 and 2.1 μM. The best inhibitions were revealed against BC cell lines with average GI<sub>50</sub> values of 1.21 and 1.37 μM, respectively. Noticeably, BC MDA-MB-468 cells were inhibited selectively by many compounds including <b>15i</b>, <b>15k</b>, <b>17d</b>, <b>25b</b>, <b>25e</b>, <b>25g</b> and <b>25h</b> with GI<sub>50</sub> equal to 0.4, 0.2, 0.6, 0.4, 0.5, 0.9, and 0.6 μM, respectively. The selectivity ratio (<i>S</i>) of the screening results of the selected test 13 compounds against NCI-60 human tumor cell lines at the five-dose level was calculated as reported<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> by dividing its average GI<sub>50</sub> toward a full panel of NCI 60 cell lines (MID-all) by its average GI<sub>50</sub> toward cell lines of a particular subpanel (MID), that is, higher value indicates better selectivity. As per this criterion, the majority (77–85%) of the test compounds exhibited a remarkable selectivity profile toward BC (<i>S</i>: 1.05–1.44), as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Table S3</a> and depicted in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S3</a>. Regarding the selectivity against a particular subpanel of NCI-60 cell lines, compounds <b>15k</b>, <b>17a</b>, <b>17d</b>, <b>25e</b>, and <b>25f</b> were found to be the most significantly selective toward BC cells with a selectivity ratio (<i>S</i>) of 1.24, 1.27, 1.44, 1.24, and 1.26, respectively, as obviously shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S3</a>.</div></div><div id="sec2_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> Annexin V PI/FITC Apoptosis Assay</h4><div class="NLM_p last">Based on the previous reports<a onclick="showRef(event, 'ref29 ref96 ref97'); return false;" href="javascript:void(0);" class="ref ref29 ref96 ref97">(29,96,97)</a> that show that lapatinib induces cell cycle arrest and apoptosis in HER2-positive BC cells, Annexin V fluorescein isothiocyanate/propidium iodide (FITC/PI) apoptosis assay was conducted for the most potent compounds revealing augmented effects in BT474 cells (<b>10c</b>, <b>15d</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>). These compounds were chosen for the apoptosis assay based on their powerful antiproliferative activities and/or augmented effect on targeting triple+ BT474 cells in comparison with their effects on HER2+ AU565 cells. The BT474 cells were treated with the selected compounds and labeled by Annexin V PI/FITC, followed by flow cytometric analysis. Compounds <b>10c</b>, <b>15d</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> induced early-phase apoptosis of BT474 in a dose-dependent manner by 3–10-fold compared to the control cells, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S4</a>. Remarkably, compound <b>10c</b> at the lowest concentration (0.2 μM) showed significant early-phase apoptosis of about sevenfold induction compared to the control cells. This activity was increased to 10-fold compared to the control at 0.5, 1.0, and 5 μM. To a lower extent, compounds <b>25b</b>, <b>15i</b>, <b>15d</b>, and <b>17d</b> exhibited early-phase apoptosis up to sixfold at 0.5–5.0 μM concentration compared to the control. Regarding the late apoptotic changes as identified by PI staining, compound <b>10c</b> showed a notable increase (2–5-folds) in late apoptosis at 0.5–5.0 μM concentrations compared to their counterparts. At the same concentration, compounds <b>25b</b> and <b>15i</b> induced late-phase apoptosis to a lower level (≤3-fold) compared to the control cells, whereas compounds <b>15d</b> and <b>17d</b> have insignificant changes in late-phase apoptosis of the BT474 cell lines. To some extent, these results can partially explain the preferential effects, particularly for compounds <b>15d</b> and <b>15i</b> on HER2+ AU565 cells (IC<sub>50</sub>: 0.663 and 0.135 μM) in comparison with their effects on the triple-positive BT474 cells (IC<sub>50</sub>: 0.107 and 0.081 μM, respectively). Despite some of these apoptotic results are partially consistent with the WST-8 and NCI 60 cell line antiproliferative assays, we cannot speculate to correlate these assays to each other. Additionally, these compounds may exhibit their antitumor effects by downregulating other different hallmarks of cancer<a onclick="showRef(event, 'ref98 ref99'); return false;" href="javascript:void(0);" class="ref ref98 ref99">(98,99)</a> without affecting apoptosis. Hence, further assays were conducted to investigate the mechanism by which our compounds reveal their antitumor effects through other pathways than the apoptosis. Further Annexin V apoptosis assays using MCF7 and MDA-MB-231 cells were conducted with the potent compounds <b>10c</b>, <b>15i</b>, and <b>25b</b> which revealed antiproliferative activities of IC<sub>50</sub> ranges of 1.78–2.65 and 0.74–4.50 μM, respectively, as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S5</a>. In contrast to their apoptotic results using BT474 cells, the tested compounds (<b>10c</b>, <b>15i</b>, and <b>25b</b> at 5.0 μM) exhibited insignificant effects on the early- and late-phase apoptosis of MCF7 and MDA-MB-231 cell lines. These results anticipate the specificity of our optimized compounds toward HER+ harboring cells.</div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Cell Growth Inhibition by Modulation of HER2 Downstream Signaling Pathways</h3><div class="NLM_p">Compounds <b>15d</b>, <b>15i</b>, and <b>25b</b> were selected to investigate their effects on the expression of HER2 kinase and its downstream signaling proteins. These three compounds were selected for further assessment at the molecular level because they exhibited the most potent antiproliferative activities against the HER2+ BC cells, in addition to the significant augmented activity of compound <b>15d</b> in the triple (+) BT-474 cells. Moreover, the apoptosis induction effect of these compounds was proved to be enhanced. Furthermore, these compounds were the best representatives for the derivatives bearing a 4-substituted phenoxy anilino moiety (<b>15d</b> and <b>15i</b>) and those bearing a 4-substituted benzyloxy anilino moiety (<b>25b</b>). It is well-known that the activation of HER signaling triggers several downstream signaling pathways, mainly mitogen-activated protein kinase/extracellular-related kinase (MAPK/ERK) and phosphatidylinositol-4,5-bisphosphate 3-kinase (PI3K)/Akt pathways.<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> The effects of the three test compounds as well as lapatinib, as a positive control, on HER2 and its main downstream signaling pathways were assessed by immunoblotting. The expression level of HER2 and the activities of ERK1/2 and Akt were compared in cellular lysates prepared from HER2+ BC cell lines (AU565 and BT474) at different time points and concentrations. The AU565 and BT474 cells were treated with IC<sub>50</sub>, 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> concentrations for each compound including lapatinib. The treated cells were harvested from each concentration after 24 and 72 h, and HER expression and activities of ERK1/2 and Akt were determined by western blot analysis as described.<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> Using AU565 BC cells, our results indicated that IC<sub>50</sub> concentrations of compound <b>25b</b> and lapatinib dramatically reduced the Akt activity after 24 and 72 h of treatment. This inhibitory effect of ERK kinase was significantly exhibited by lapatinib after 24 h and to a lower extent after 72 h. Compound <b>25b</b> and lapatinib demonstrated mild inhibition of HER2 expression after 24 h, whereas compounds <b>15d</b> and <b>15i</b> did not show any detectable inhibitions on these kinases (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). At a lower concentration of 0.5 IC<sub>50</sub>, compound <b>15i</b> showed a significant reduction in HER2 expression compared to lapatinib at 24 h but not at 72 h. Peculiarly, the tested compounds, as well as lapatinib, inhibited Akt expression (total Akt) after 72 h. However, only compound <b>25b</b> and lapatinib were able to demonstrate the inhibitory effect on Akt activity. At a concentration of 0.5 IC<sub>50</sub>, compound <b>15i</b> showed inhibition of pERK expression almost similar to that of lapatinib after 24 and 72 h. Remarkably, compounds <b>15i</b> and <b>25b</b> reduced pAKT competitively to lapatinib at 0.1 IC<sub>50</sub> concentration after 24 h. At the same concentration and time point, compound <b>25b</b> inhibited HER2 expression comparable to lapatinib, as depicted in <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A.</div><figure id="fig13" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Effect of optimized compounds on the HER2 signaling pathway in BC cells. AU565 (A) and BT474 (B) cells were treated with compounds <b>15d</b>, <b>15i</b>, and <b>25b</b> at concentrations of their IC<sub>50</sub>, 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> for each compound at 24 and 72 h. Lapatinib was used as an authenticated control for the test compounds. Western blot analysis was performed and GAPDH was used as a reference loading control. Data represented results collected from at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The effect of the three test compounds and lapatinib on HER2 expression as well as its downstream targets was further investigated using the BT474 triple-positive BC cell line. Our results showed that at IC<sub>50</sub> concentrations, all compounds, specifically compound <b>15i</b>, and lapatinib revealed a substantial reduction in HER2 expression after 24 h of treatment. In the same way, compounds <b>15i</b>, <b>25b</b>, and lapatinib significantly inhibited the activity of Akt as indicated by the notable reduction of phosphorylated AKT after 72 h (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B). At 0.5 IC<sub>50</sub> and 0.1 IC<sub>50</sub> concentrations, most of the compounds particularly compound <b>25b</b> and lapatinib revealed a dramatic inhibition in the activity of both Akt and ERK1/2 after 72 h (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B). A close look at the overall immunoblotting analysis using AU565 and BT474 cells indicates that all the three tested compounds were able to inhibit the expression/activity of HER2 and/or at least one of its downstream targets in at least at a one-time point or three used concentrations.</div><div class="NLM_p">Furthermore, the HER2 signaling pathway in BC BT-474 cells was evaluated using western blot analysis for compound <b>17d</b>. The cells were treated at IC<sub>50</sub> (0.204 μM), 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> concentrations within two-time points (24 and 48 h). GAPDH was used as a loading control. Using BT474 BC cells, our results indicated that at 0.1 and 0.5 IC<sub>50</sub> concentrations, compound <b>17d</b> has particularly reduced HER2 and pAKT expressions compared to the control 24 h, whereas this effect was exhibited at IC<sub>50</sub> at 48 h. To a lower extent, compound <b>17d</b> exhibited a mild inhibitory effect against Akt and pERK1/2 after 24 and 48 h, as shown in <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig14" id="" class=" internalNav">14</a></a>.</div><figure id="fig14" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0015.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Western blot analysis for compound <b>17d</b> to evaluate the HER2 signaling pathway in BC cells (BT-474). The cells were treated with different concentrations at IC<sub>50</sub> (0.204 μM), 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> at 24 and 48 h. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>In Vivo</i> Antitumor Efficacy for the Most Potent Optimized Scaffolds Using the BT474 Athymic Nude Mice (Foxn<sup>1nu</sup>/Foxn<sup>1+</sup>) Xenograft Mouse Model</h3><div class="NLM_p">Encouraged by the potent inhibitory activity of compounds <b>17d</b> and <b>25b</b> regarding their low IC<sub>50</sub> against BT474 cell lines as demonstrated by the WST-8 assay data, we were excited to know whether this inhibition can be reproduced in the preclinical BC model.</div><div id="sec2_7_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Rationale for the Selection of Compounds <b>17d</b> and <b>25b</b> for the <i>In Vivo</i> Study</h4><div class="NLM_p">The selection criteria of compounds <b>17d</b> and <b>25b</b> for evaluation of their <i>in vivo</i> anticancer activity depended on the overall activity of these compounds and NOT limited to the antiproliferative activities. Regarding the selection of compound <b>25b</b> for <i>in vivo</i> testing, compounds <b>15i</b>, <b>25a</b>, and <b>25b</b> revealed COMPARABLE or almost SAME potency against BT474 cells, IC<sub>50</sub> values (0.08 μM; <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>). Compound <b>25b</b> is considered as the best candidate in this series (1) with similar activities, (2) being previously selected by NCI as a promising candidate for 5-dose assay, (3) with its potent inhibitory activity against HER2 kinase (IC<sub>50</sub> of 8.3 nM), and (4) it shares the 4-benzyloxyaniline moiety of lapatinib and could be considered as a well-optimized analogue by a comparative <i>in vivo</i> study.</div><div class="NLM_p">On the other hand, compound <b>17d</b> was selected for <i>in vivo</i> testing being; (1) revealing superior kinase profiling selectivity over the targeted HER2 kinase, with a particular insignificant effect on the off-targeted PDGFRα kinase (−4.0%), which was overactivated by lapatinib (56%) and other test compounds (18–115%), (2) <b>17d</b> was selected by NCI for 5-dose assay and revealed the highest sensitivity toward BC cell lines within NCI60 cell lines, and (3) it has the 4-phenoxyaniline moiety and could be considered as the best example for studying the beneficial effect of this moiety on the <i>in vivo</i> efficacy, compared to the 4-benzyloxyaniline moiety of lapatinib.</div><div class="NLM_p">In this study, female athymic nude mice (Foxn<sup>1nu</sup>/Foxn<sup>1+</sup>) were used to examine these two compounds <b>17d</b> and <b>25b</b>. When the volume of the engrafted tumors reached around 100–150 mm<sup>3</sup>, the tested compounds were administered at 50 mg/kg, by oral gavage 5×/week, and continued over 55 days.<a onclick="showRef(event, 'ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref80 ref81">(80,81)</a> To evaluate the treatment response of the BT474 xenograft growth in mice, the tumor growth curves of the three tested groups were analyzed and compared among each other using the R package TumGrowth according to Enot et al. (2018).<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> The tumor growth curves of the three groups were subjected to type II ANOVA and pairwise comparisons across groups using this statistical package. According to the statistical data and as depicted in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>A, oral treatment of the nude female mice with 50 mg/kg of compound <b>17d</b> substantially inhibited the BT474 tumor growth compared to either the vehicle control (<i>p</i> = 0.0013) or compound <b>25b</b> treated group (<i>p</i> = 0.01). Mice treated with compound <b>25b</b> demonstrated mild growth inhibition at the same dose; however, this reduction did not reach a significant value when compared to the vehicle control group (<i>p</i> = 0.5352), as shown in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>B. More details were illustrated in <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Figure S6</a> illustrating the average tumor growth rate (tumor size with error bars vs time) for the three mice groups that received the vehicle, compound <b>17d</b> and <b>25b</b>, respectively. These data were further confirmed when using the average tumor weight of the three treatment groups after animals were sacrificed. The average tumor weights for the <b>17d</b> group (0.572 g) were significantly less than that of the vehicle control group (2.046 g) and the <i>p</i>-value of the comparison was 0.024 (ANOVA/Tukey test). The average tumor weight of the vehicle control group (2.046 g) is still higher than that of the <b>25b</b> compound group (1.223 g). However, there was no significant difference recorded when both groups were compared (<i>p</i> = 0.155, ANOVA/Tukey), as depicted in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>D. Histologically, the H&E stain of the xenograft tumor tissues indicated that the cells are originated from the xenograft and had the typical morphology of cancer cells. This was unequivocally observed in the vehicle-treated tissues with round cells and enlarged nuclei. Interestingly, large areas of necrosis were observed in compounds <b>17d-</b> and <b>25b</b>-sensitive tumors, as depicted by red arrows in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>E (upper panel), the higher magnification showing fibrosis and inflammation in compounds <b>17d-</b> and <b>25b</b>-treated tumors. On the other hand, the xenograft tissues stained by the ki67 antibody indicated that the proliferative cells are most abundant in the vehicle control group, whereas they are dramatically reduced on treatment by compound <b>17d</b> and to a less extent by compound <b>25b</b>, as clearly depicted in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>E (lower panel). Interestingly, a metastasis case was observed in the vehicle group but not detected in the mice treated by the test compounds, as obviously pointed out with red arrows in <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig15" id="" class=" internalNav">15</a></a>A, whereas no metastasis was detected in the other groups treated by the compounds <b>17d</b> and <b>25b</b>. This study anticipates that compound <b>17d</b> is a potential replacement therapy for lapatinib to sensitize HER2-overexpressing BCs and significantly reduce the tumorigenesis and metastasis of the HER2+ BC. Nevertheless, the <i>in vivo</i> efficacy of compound <b>25b</b> is less active as compared to compound <b>17d</b>.</div><figure id="fig15" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0016.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Effect of compounds <b>17d</b> and <b>25b</b> on tumor growth of BT474 cells in the preclinical mouse model. (A) Represented images of mice implanted with 2 × 10<sup>6</sup> BT474 cells on the dorsal flank. After growing of the nodules, mice were randomized 5 mice per group and received the vehicle and compound <b>17d</b> and <b>25b</b>, respectively; then, all mice were sacrificed 6 weeks after the initial treatment. The remarkable metastases in the vehicle group are pointed out by red arrows. At the end of the experiment, tumor tissues were resected and preserved. (B) Tumor growth was measured twice a week for 6 weeks after treatments. (C,D) Whole-body weight per each mouse was recorded twice a week and the resected tumors were weighed per each animal. (E) Formalin-fixed paraffin-embedded xenograft tissues were stained with hematoxylin (H) and eosin (E), and proliferative index Ki67 for the three groups as indicated. Significance considered at <i>p</i> < 0.05 regarding mice that received vehicle. The magnification for H&E and Ki67 staining was 200× and for inserts was 400×.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Fortunately, our findings are fit in-line with the reported postulations<a onclick="showRef(event, 'ref103 ref104'); return false;" href="javascript:void(0);" class="ref ref103 ref104">(103,104)</a> that the superior <i>in vitro</i> activity of the lipophilic 3-fluoro benzyl derivatives (as <b>25b</b>) would not translate into a better <i>in vivo</i> efficacy because of their higher protein binding.</div><div class="NLM_p last">On the other hand, Mohamed et al. reported the O-dealkylation on the 3-fluorobenzyl moiety of lapatinib as an abundant metabolic pathway.<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> Similarly, the metabolic study of pyrotinib indicated that the O-dealkylated metabolite exhibited weak activity against HER2 kinase because of the loss of interaction with the HER2 back pocket.<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> Accordingly, to investigate the metabolic stability of compounds <b>17d</b> (bearing 3-fluorophenyl moiety) and <b>25b</b> (bearing 3-fluorobenzyl moiety), a microsomal stability assay was conducted.</div></div><div id="sec2_7_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> CD-1 Mouse Microsomal Stability Assay</h4><div class="NLM_p">Compounds <b>17d</b> and <b>25b</b>, 5 μL at a final concentration of 100 μM, were incubated with pooled mouse liver microsomes (final concentration 0.56 mg/mL) in phosphate buffer 0.1 M (pH 7.4) in a 37 °C water bath, and the reaction was initiated by the addition of an NADPH cofactor. The reaction was allowed for different time points, 5, 15, 30, 45, and 60 min and after quenching of the reactions, each bioanalysis plate was sealed and shaken for 10 min prior to LC–MS/MS analysis. The equation of first-order kinetics was used to calculate <i>T</i><sub>1/2</sub> and CL<sub>int(mic)</sub> (μL/min/mg). Testosterone, propafenone, and diclofenac were used as positive controls as reference compounds with high and medium clearance, respectively.</div><div class="NLM_p">As shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, compound <b>17d</b> that revealed higher <i>T</i><sub>1/2</sub> = >145 min compared to compound <b>25b</b><i>T</i><sub>1/2</sub> = 57.4 min along with lower intrinsic clearance was observed with compound <b>17d</b>, CL<sub>int(mic)</sub> = <9.6 mL/min/kg compared to <b>25b</b>, CL<sub>int(mic)</sub> = 24.2 mL/min/kg. Accordingly, compound <b>25b</b> is likely to be rapidly cleared <i>in vivo</i> resulting in a short duration of action. This underpins the less sufficient antitumor activity of <b>25b</b> compared to <b>17d</b> in the <i>in vivo</i> animal model.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Mouse Microsomal Stability Assay Result of the Test Compounds<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">test compound</th><th class="colsep0 rowsep0" align="center" char="."><i>R</i><sup>2</sup></th><th class="colsep0 rowsep0" align="center"><i>T</i><sub>1/2</sub> (min)</th><th class="colsep0 rowsep0" align="center">CL<sub>int(mic)</sub> (μL/min/mg)</th><th class="colsep0 rowsep0" align="center">CL<sub>int(liver)</sub> (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char=".">remaining (<i>T</i> = 60 min) (%)</th><th class="colsep0 rowsep0" align="center" char=".">remaining (NCF = 60 min) (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17d</b></td><td class="colsep0 rowsep0" align="char" char=".">0.9822</td><td class="colsep0 rowsep0" align="left"><b>>145</b></td><td class="colsep0 rowsep0" align="left"><b><9.6</b></td><td class="colsep0 rowsep0" align="left"><38.0</td><td class="colsep0 rowsep0" align="char" char=".">102.7</td><td class="colsep0 rowsep0" align="char" char=".">100.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25b</b></td><td class="colsep0 rowsep0" align="char" char=".">0.9749</td><td class="colsep0 rowsep0" align="left"><b>57.4</b></td><td class="colsep0 rowsep0" align="left"><b>24.2</b></td><td class="colsep0 rowsep0" align="left">95.7</td><td class="colsep0 rowsep0" align="char" char=".">46.4</td><td class="colsep0 rowsep0" align="char" char=".">93.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">testosterone</td><td class="colsep0 rowsep0" align="char" char=".">0.9991</td><td class="colsep0 rowsep0" align="left">5.0</td><td class="colsep0 rowsep0" align="left">279.1</td><td class="colsep0 rowsep0" align="left">1105.4</td><td class="colsep0 rowsep0" align="char" char=".">0.0</td><td class="colsep0 rowsep0" align="char" char=".">103.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">diclofenac</td><td class="colsep0 rowsep0" align="char" char=".">0.9978</td><td class="colsep0 rowsep0" align="left">60.0</td><td class="colsep0 rowsep0" align="left">23.1</td><td class="colsep0 rowsep0" align="left">91.5</td><td class="colsep0 rowsep0" align="char" char=".">50.2</td><td class="colsep0 rowsep0" align="char" char=".">101.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">propafenone</td><td class="colsep0 rowsep0" align="char" char=".">0.9968</td><td class="colsep0 rowsep0" align="left">2.1</td><td class="colsep0 rowsep0" align="left">668.1</td><td class="colsep0 rowsep0" align="left">2645.6</td><td class="colsep0 rowsep0" align="char" char=".">0.0</td><td class="colsep0 rowsep0" align="char" char=".">105.5</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">NCF: no cofactor. No NADPH regenerating system is added to NCF samples (replaced by buffer) during the 60 min incubation. If the NCF remaining is less than 60%, then non-NADPH-dependent metabolism occurs; <i>R</i><sup>2</sup>: correlation coefficient; <i>T</i><sub>1/2</sub>: half-life; CL<sub>int(mic)</sub>: intrinsic clearance; CL<sub>int(mic)</sub> = 0.693/<i>T</i><sub>1/2</sub>/mg microsome protein per mL; CL<sub>int(liver)</sub> = CL<sub>int(mic)</sub> × mg microsomal protein/g liver weight × g liver weight/kg body weight mg microsomal protein/g liver weight: 45 mg/g for all species; and liver weights: 88 g/kg for a mouse.</p></div></div></div></div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Molecular Docking</h3><div id="sec2_8_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Docking Validation and Protocol Optimization</h4><div class="NLM_p">The scoring function of the GOLD program was validated by checking the extent of the resemblance of the best-docked conformation of the cocrystallized ligand with the reported biological method.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> Therefore, we initiated this study by redocking of the cocrystallized ligand (03Q: 2-(2-[4-((5-chloro-6-[3-(trifluoromethyl)phenoxy]pyridin-3-ylamino)-5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-5-yl]ethoxy)ethanol)) into chain B (75% overall quality) of the HER2 kinase domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). The docking protocol was optimized by running five trials of docking including rigid and flexible docking, as listed in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Trial 4, involving a flexible docking with M801 residue and 50,000 operations, exhibited the best-optimized protocol with an rmsd of 1.54 Å and four hydrogen bonds, as shown in <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig16" id="" class=" internalNav">16</a></a>. The binding modes for the 03Q native ligand obtained in trial 4 were greatly consistent with its reported cocrystal structure bound to HER2 kinase.</div><figure id="fig16" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0017.jpeg" id="gr16" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Optimization of the docking protocol of the cocrystallized ligand (03Q) into HER2 kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). The five docked trials are labeled (1–5) in comparison with the native bound ligand (03Q, shown as yellow sticks) in which trial 4 (shown as ball and sticks) utilizing M801 as a flexible residue with 50,000 operations exhibited the best-optimized protocol with an rmsd of 1.54 Å and four hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Optimization of the Docking into HER2 Kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>)</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center" char="."> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">hydrogen bonds between compounds and HER2 kinase</th><th class="colsep0 rowsep0" align="center" char="."> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">GoldScore</th><th class="colsep0 rowsep0" align="center" char="."><i>S</i>(hb-ext)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center" char="."><i>S</i>(vdw-ext)<a class="ref internalNav" href="#t6fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">flexible residues</th><th class="colsep0 rowsep0" align="center">atom of compd</th><th class="colsep0 rowsep0" align="center">amino acid</th><th class="colsep0 rowsep0" align="center" char=".">rmsd<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a> (Å)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trial 1</td><td class="colsep0 rowsep0" align="char" char=".">87.29</td><td class="colsep0 rowsep0" align="char" char=".">3.8604</td><td class="colsep0 rowsep0" align="char" char=".">63.3455</td><td class="colsep0 rowsep0" align="left">M801</td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char=".">3.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">D863</td><td class="colsep0 rowsep0" align="left">terminal CH<sub>2</sub>O</td><td class="colsep0 rowsep0" align="left">HN of C805</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">T798</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trial 2</td><td class="colsep0 rowsep0" align="char" char=".">94.26</td><td class="colsep0 rowsep0" align="char" char=".">3.9375</td><td class="colsep0 rowsep0" align="char" char=".">68.4816</td><td class="colsep0 rowsep0" align="left">M801</td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char=".">3.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">D863</td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HN of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">terminal CH<sub>2</sub>O</td><td class="colsep0 rowsep0" align="left">HN of C805</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trial 3</td><td class="colsep0 rowsep0" align="char" char=".">89.75</td><td class="colsep0 rowsep0" align="char" char=".">2.0682</td><td class="colsep0 rowsep0" align="char" char=".">66.9045</td><td class="colsep0 rowsep0" align="left">M801</td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char=".">2.44</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HO of T862</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HN of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trial 4</td><td class="colsep0 rowsep0" align="char" char=".">91.40</td><td class="colsep0 rowsep0" align="char" char=".">6.4519</td><td class="colsep0 rowsep0" align="char" char=".">64.8467</td><td class="colsep0 rowsep0" align="left">M801</td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char=".">1.54</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HN of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HO of T862</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">terminal CH<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">O═C of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">trial 5</td><td class="colsep0 rowsep0" align="char" char=".">85.52</td><td class="colsep0 rowsep0" align="char" char=".">6.8702</td><td class="colsep0 rowsep0" align="char" char=".">61.0202</td><td class="colsep0 rowsep0" align="left">rigid</td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char=".">0.99</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">terminal CH<sub>2</sub>OH</td><td class="colsep0 rowsep0" align="left">O═C of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">native ligand<a class="ref internalNav" href="#t6fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">quinazoline-N<sup>1</sup>H</td><td class="colsep0 rowsep0" align="left">HN of M801</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">pyridine-N</td><td class="colsep0 rowsep0" align="left">HN of D863</td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>S</i>(hb_ext)<a class="ref internalNav" href="#t6fn1" aria-label="a">a</a> GoldScore protein–ligand hydrogen bonding.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last"><i>S</i>(vdw_ext): GoldScore external Vdw (van der Waals interactions between the protein and ligand).</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">rmsd: Root-mean-square deviation.</p></div><div class="footnote" id="t6fn4"><sup><sup>d</sup></sup><p class="last">03Q: 2-(2-[4-((5-chloro-6-[3-(trifluoromethyl)phenoxy]pyridin-3-ylamino)-5<i>H</i>-pyrrolo[3,2-<i>d</i>]pyrimidin-5-yl]ethoxy)ethanol).</p></div></div></div></div><div id="sec2_8_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> GoldScore Fitness of the Designed Quinazoline Derivatives into HER2 Kinase</h4><div class="NLM_p last">The GoldScore algorithmic function was applied in this molecular docking because it affords superior docking results than the ChemScore function in GOLD as a powerful molecular docking program.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> All the docking results, including GoldScore fitness, <i>S</i>(hb-ext), <i>S</i>(vdw-ext), hydrogen bonds between the ligand and HER2 kinase, and rmsd between the docked pose and the native cocrystallized ligand, are listed in the Supporting Information, <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Table S4</a>. Our compounds were docked into the human HER2 kinase domain revealing Gold fitness scores ranging from 77.38 to 93.29. Compounds <b>10c</b>, <b>15f</b>, <b>15i</b>, <b>15j</b>, <b>17d</b>, <b>25b</b>, and <b>25h</b> exhibited the most noteworthy GoldScore fitness of 90.14–93.29 in comparison with the cocrystallized compound 03Q of 91.40. The majority of the test compounds (17 out of 27) were docked proximately superimposed on the native ligand within ≤2.50 Å. They docked into the ATP-binding site through up to three hydrogen bonds to a higher extent with M801 (NH) and C805 (NH) amino acids.</div></div><div id="sec2_8_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> Structure–Activity Relationship</h4><div class="NLM_p">In this study, three key structural features of the designed compounds were discussed. These include the polar terminal residues at position 6, the anilino substitution at position 4, and the flexible spacer between the quinazoline ring and the terminal hydrophilic moieties, as depicted in <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig17" id="" class=" internalNav">17</a></a>. Regarding the antiproliferative assay, compounds <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15h</b>, <b>17a</b>, <b>25a</b>, <b>25c</b>, and <b>25e</b> bearing a morpholine terminal residue revealed potent and selective activity against HER2+ BC cells (BT-474 and AU-565). These compounds exhibited IC<sub>50</sub> in sub-micromolar concentrations (IC<sub>50</sub> = 0.08–0.9 μM) with a noticeably diminished activity against HER2– BC cells MCF7 and MDA-MB-231 (IC<sub>50</sub> = 5.5–30 μM). Meanwhile, compounds having an <i>N</i>-methylpiperazine residue (<b>10c</b>, <b>15f</b>, <b>15i</b>, <b>17b</b>, <b>25b</b>, <b>25d</b>, and <b>25f</b>) exhibited a broad spectrum of activity against BC cell lines. Their antiproliferative strengths vary ranging from strong (IC<sub>50</sub> = 0.08–0.7 μM) against HER2+ BC cells to moderate activity (IC<sub>50</sub> = 0.7–2.5 μM) against HER2– BC cells. Compounds having acyclic polar residues (ethanolamine, dimethylamine, and diethylamine) along with an acetamido or propoxy spacer exhibited inferior activity of 2–3 times less than compounds bearing a terminal morpholine or <i>N</i>-methyl piperazine moiety. However, the reverse was observed in the presence of a bulkier benzamide linker, where compounds with acyclic polar residues exhibited 2–5 times more potential than compounds bearing cyclic terminal residues.</div><figure id="fig17" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0018.jpeg" id="gr17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. SAR studies of the newly synthesized 4,6 disubstituted quinazolines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Thus, the steric effect of the substituents at position 6 of the quinazoline scaffold has a significant impact on the antiproliferative activity, where these substituents will accommodate into the solvent-accessible region of HER2 kinase. It is noteworthy that the methyl substitution on the phenoxyaniline moiety at position 4 dramatically enhanced the cytotoxic activity of compounds within 4 to 6 times higher than substitution by a trifluoromethyl or fluoro moiety. Considering that compounds have the same spacer and terminal residues, compounds <b>15h</b>, <b>15i</b>, and <b>15k</b> with a methyl substituent on the phenoxyaniline ring at position 4 exhibited IC<sub>50</sub> with an average of 0.1 μM against AU565 BC cells. Meanwhile, compounds <b>15e–15g</b> and compounds <b>15a</b>, <b>15b</b>, and <b>15d</b> bearing a trifluoromethyl and fluoro group, respectively, revealed an average of IC<sub>50</sub> = 0.4 and 0.6 μM, respectively.</div><div class="NLM_p last">Regarding the protein kinase profiling assay, compounds <b>10a</b>, <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b> demonstrated the same pattern of inhibitory activity and selectivity against the HER kinase family (EGFR, HER2, and HER4) with higher particular potency against the target HER2 kinase (−91 to 93%) and with moderate-to-weak inhibition on other nontarget 17 kinases. Regardless of the type of the spacer or the polar terminal residue, the test quinazoline derivatives bearing 4-substituted phenoxyaniline moieties (<b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b>) showed remarkable selectivity against the target HER2 kinase (−91 to −92%) and moderate inhibition against EGFR (−65 to −78%). Compound <b>17d</b> having a benzamide spacer and ethanolamine terminal residue demonstrated the best selectivity among the test compounds with −92% inhibition of HER2, −70% for EGFR, −47% for ROCK2, −47% for JAK-2, and −40% for ABL1 kinase. Nevertheless, the test quinazoline derivatives (<b>10a</b> and <b>10b</b>) substituted with a benzyloxyaniline moiety at position 4 exhibited inferior selectivity against HER2 of −92 to −93%, EGFR of −78 to −81%, JAK-2 of −74 to −86%, and ABL of −86%. However, the tested compounds <b>10a</b>, <b>10c</b>, and <b>25b</b> or <b>15i</b> and <b>17d</b> bearing benzyloxyaniline or phenoxyaniline moieties at position 4 revealed superior HER2 inhibition with an IC<sub>50</sub> of 10.4, 6.1, and 8.3 nM or 5.4 and 12 nM, respectively, compared to lapatinib IC<sub>50</sub> of 95.5 nM. It is noteworthy that the benzyloxyaniline moiety is metabolically labile to O-dealkylation which may result in high clearance and low <i>in vivo</i> activity, as observed in the <i>in vivo</i> efficacy of <b>25b</b>. Accordingly, we can conclude that the substitution at position 4 with a phenoxyaniline or benzyloxyaniline moiety mainly affects the selectivity against HER2 kinase and the metabolic stability <i>in vivo</i>. Additionally, the polar substitution at position 6 can also modulate the cellular activity of the tested compounds.</div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i35">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64343" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64343" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, novel 27 4,6-disubstituted quinazoline derivatives were designed and synthesized guided by the binding mode of the current dual EGFR/HER2 inhibitor, lapatinib. This design was conducted through multistep reactions to incorporate different flexible spacers at position 6 of the quinazoline ring, namely, acetamido, methylbenzamido, or propoxy. This was followed by a nucleophilic substitution with several polar fragments. At position 4, the <i>N</i>-(4-(3-chloro-4-(3-fluorobenzyl)oxy)phenyl) amino moiety of lapatinib was replaced by the <i>N</i>-(4-(3-chloro-4-(3-substituted-phenoxy)phenyl)amino) moiety to properly fit into the lipophilic back pocket with an enhanced selectivity against the target HER2 kinase. Increasing the selectivity against HER2 kinase was assumed to minimize the EGFR inhibition-associated side effects such as severe rash and diarrhea. The synthetic methodologies involved in this project have been optimized to report the most appropriate reaction conditions affording the targeted and intermediate compounds at a higher yield and purity. The <i>in vitro</i> protein kinase profiling for the lead compound (<b>10a</b>) and the other representative analogues (<b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b>) at 10 μM single-dose concentration against 20 kinases demonstrated a superior significant selectivity than lapatinib. Our compounds revealed inhibitions within a range of −92 to −99% against HER2 and HER4 kinases and within −67 to −81% against EGFR (HER1). Remarkably, the screened compounds specifically <b>17d</b> exhibited negligible activation or even mild inhibition of other nontarget kinases which were activated by lapatinib. These results anticipate lower side effects and toxicity of compound <b>17d</b>. This preferential selectivity of the test compounds was an encouraging result to evaluate the cytotoxic activity of the newly synthesized quinazolines against HER2 BC cell lines and HER2-negative cells for comparison. The majority of synthesized compounds demonstrated potent and selective antiproliferative activities toward HER2+ BC cell line AU-565. Furthermore, many compounds revealed augmented activity against triple-positive cells (BT-474) with noticeably diminished activity against HER2-negative BC cells (MCF7 and MDA-MB-231). Against wild-type HER2 kinase, the privileged compounds (<b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>) demonstrated higher inhibitory activity against HER2 kinase with IC<sub>50</sub> values in the range of 5.4–12 nM compared to lapatinib (95.5 nM). On the other hand, the western blot analysis indicated that the phosphorylation of HER2 downstream signaling through AKT and ERK pathways was impaired in BT-474 and AU-565 cells treated with compounds <b>15d</b>, <b>15i</b>, and <b>25b</b> at different concentration points of IC<sub>50</sub>, 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> for each compound. The apoptosis assay for compounds <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> revealed induction of the early-phase apoptosis in BT-474 cells 3–10-fold compared to the negative control. Furthermore, out of 20 compounds tested at 10 μM single dose in the full NCI 60 cell panel, 13 compounds (<b>15f</b>, <b>15g</b>, <b>15i</b>, <b>15k</b>, <b>17a</b>, <b>17c</b>, <b>17d</b>, <b>25b</b>, <b>25d</b>, <b>25e</b>, <b>25f</b>, <b>25g</b>, and <b>25h</b>) were selected to be evaluated at five concentration levels, where compound <b>17d</b> demonstrated the highest selectivity against BC. The SAR study expounded that the terminal polar moieties not only affect the drug-likeness and pharmacokinetic properties of the compound but also modulate the cellular activity. We have noticed that compounds bearing a morpholino moiety or its acyclic analogue ethanolamine revealed potent and selective antiproliferative activities against HER2(+) BC cell lines (BT-474 and AU-565). Meanwhile, the <i>N</i>-methylpiperazino-bearing compounds exhibited extended moderate activity against the other HER2(−ve) BC cell lines. Inspired by the potent antiproliferative and enzymatic activities of compounds <b>17d</b> and <b>25b</b> bearing the 4-substituted phenoxyanilino and 4-substituted benzyloxyanilino moiety, respectively, were selected to be tested for their <i>in vivo</i> antitumor efficacy using the BT-474 athymic nude mice (Foxn<sup>1nu</sup>/Foxn<sup>1+</sup>) xenograft model. Even though compound <b>25b</b> exhibited higher antiproliferative activity in WST-8 assay (IC<sub>50</sub> = 0.083 μM) than compound <b>17d</b> (IC<sub>50</sub> = 0.204 μM), both compounds behaved differently in <i>in vivo</i> evaluation. Compound <b>17d</b> substantially inhibited the BT-474 tumor growth compared to either the vehicle control or compound <b>25b</b>-treated groups. This was attributed to the higher <i>T</i><sub>1/2</sub> and lower intrinsic clearance of compound <b>17d</b> (<i>T</i><sub>1/2</sub> = >145, CL<sub>int(mic)</sub> = <9.6 mL/min/kg), compared to compound <b>25b</b> (<i>T</i><sub>1/2</sub> = 57.4 min, CL<sub>int(mic)</sub> = 24.2 mL/min/kg). Interestingly, metastasis was observed in the nude mice treated by the vehicle which was not detected in the other groups treated by compounds <b>17d</b> and <b>25b</b>. Altogether, we eventually identified compound <b>17d</b> as a promising candidate for HER2+ BC therapy deserving further investigations.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08600" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08600" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Chemistry</h3><div class="NLM_p last">This part is concerned with the synthetic schemes adopted for the synthesis of the designed compounds and the description of the synthetic chemical steps used, aided by a literature survey for each route. All the synthesized compounds were purified using flash column chromatography and then subjected to a comprehensive spectrometric analysis including <sup>1</sup>H NMR, <sup>13</sup>C DEPT, and <sup>13</sup>C NMR. In addition, all 27 key target compounds have been further analyzed by HRMS. Using integrated analytical methods, the purity of the synthesized compounds was confirmed at ≥95% by ultrahigh-performance liquid chromatography/MS (UHPLC/MS) and NMR methods. All different spectra of our compounds have been uploaded in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Supporting Information</a> for publication. Thin-layer chromatography (TLC) analysis of reaction mixtures was performed using Sigma-Aldrich silica gel 60 Å, F254 TLC plates, and visualized under UV. Flash column chromatography was carried out on RediSep <i>R</i><sub><i>f</i></sub>-Teledyne ISCO cartilages, 230–450 mesh using CombiFlash <i>R</i><sub><i>f</i></sub> 200-Teledyne ISCO. The reaction progress was monitored using an Advion Expression-L Mass spectrophotometer using TLC plate extraction or ASAP techniques. <sup>1</sup>H, <sup>13</sup>C DEPT, and <sup>13</sup>C NMR spectra were recorded in CDCl<sub>3</sub> or dimethyl sulfoxide (DMSO) at ambient temperature using a Bruker 300 MHz spectrometer. Coupling constants were expressed in Hz and the abbreviations used for multiplicity are as follows: s = singlet, d = doublet, t = triplet, q = quartet, p = pentet, m = multiplet, and br = broad. <sup>1</sup>H and <sup>13</sup>C NMR chemical shifts are reported in ppm downfield of tetramethylsilane and are, respectively, referenced to the residual solvent (δ = 7.26 and 77.36 ppm for CDCl<sub>3</sub>, δ = 2.54 and 40.45 ppm for DMSO-<i>d</i><sub>6</sub>). High-resolution mass spectra (HRMS) were obtained on LC/MS-TOFAgilent 6210 using electrospray ionization (ESI). Melting points were recorded on a Stuart melting point smp20 apparatus using Kimax Glass Capillary Tubes, 100 mm.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> 2-Chloro-1-((3-fluorobenzyl)oxy)-4-nitrobenzene (<b>3a</b>)</h3><div class="NLM_p last">To a mixture of 2-chloro-4-nitrophenol (<b>1a</b>) (3.47 g, 20.0 mmol) and potassium carbonate (2.76 g, 20.0 mmol) in DMF (20 mL), 3-fluorobenzyl bromide (<b>2a</b>) (3.81 g, 2.47 mL, 20.2 mmol) was added. After being stirred at 85 °C for 3 h, water (40 mL) was added and the mixture was extracted with ethyl acetate (EtOAc). The organic layer was washed with saturated brine (30 mL), dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and concentrated in vacuum to yield <b>3a</b> (4.10 g, 73% yield) as a light-yellow solid.<a onclick="showRef(event, 'ref108 ref109'); return false;" href="javascript:void(0);" class="ref ref108 ref109">(108,109)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> 2-Chloro-4-nitro-1-(3-(substituted)phenoxy)benzene (<b>3b–d</b>)</h3><div class="NLM_p">To a mixture of 2-chloro-1-fluoro-4-nitrobenzene (<b>1b</b>) (3.50 g, 20.0 mmol) and potassium carbonate (2.76 g, 20.0 mmol) in acetonitrile (20 mL) was added appropriate phenol <b>2b–d</b> (20.0 mmol). After being stirred at 85 °C for 5 h, the reaction mixture was filtered while hot and the filtrate was concentrated under reduced pressure to afford <b>3b–d</b>.<a onclick="showRef(event, 'ref110 ref111'); return false;" href="javascript:void(0);" class="ref ref110 ref111">(110,111)</a></div><div id="sec4_3_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 2-Chloro-1-(3-fluorophenoxy)-4-nitrobenzene (<b>3b</b>)</h4><div class="NLM_p last">(4.7 g, yield 89%), yellow solid.</div></div><div id="sec4_3_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 2-Chloro-4-nitro-1-(3-(trifluoromethyl)phenoxy)benzene (<b>3c</b>)</h4><div class="NLM_p last">(5.2 g, yield 83%), yellow solid.</div></div><div id="sec4_3_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 2-Chloro-1-(3-methylphenoxy)-4-nitrobenzene (<b>3d</b>)</h4><div class="NLM_p last">(4.1 g, yield 78%), faint yellow oil.</div></div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> 3-Chloro-4-((3-fluorobenzyl)oxy)aniline (<b>4a</b>)</h3><div class="NLM_p">To a suspension of (<b>3a</b>) (3.6 g, 12.8 mmol) and reduced iron [2.3 g, 38.4 mmol (3 mol equiv)] in ethanol (50 mL), a solution of calcium chloride (1.5 g, 12.8 mmol, 5 mL water) was added. The mixture was stirred at 85 °C for 8 h. The formed black solid was removed by filtration.</div><div class="NLM_p last">Then, the filtrate was centrifuged for 30 min and then filtered again. The clear filtrate was concentrated under vacuum. The residue was washed with water, filtered, dried, and purified by silica gel column chromatography (eluent, EtOAc/hexane = 1:3) to yield <b>4a</b> (2.5 g, 77% yield) as yellowish white crystals.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> General Procedure for the Synthesis of 3-Chloro-4-(3-(substituted)phenoxy)aniline (<b>4b–d</b>)</h3><div class="NLM_p">A mixture of the appropriate nitro derivative (<b>3b–d</b>) (5 mmol) and stannous chloride (4.7 g, 25 mmol) in methanol (20 mL) was stirred at reflux for 2.5 h under a nitrogen atmosphere. Excess methanol was removed under reduced pressure, the remaining residue was dissolved in ethyl acetate (200 mL), and the obtained solution was neutralized with an aqueous saturated solution of sodium bicarbonate and stirred for 1 h. The resulting gelatinous precipitate was centrifuged for 15 min and then discarded by filtration. The clear filtrate was transferred into a separating funnel and the organic phase was separated. The aqueous phase was extracted with ethyl acetate (2 × 20 mL), and the organic fractions were combined, dried over anhydrous sodium sulfate, and concentrated under vacuum to afford the corresponding aniline derivatives <b>4b–d</b>.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 3-Chloro-4-(3-fluorophenoxy)aniline (<b>4b</b>)</h4><div class="NLM_p last">(0.97 g, yield 82%), yellow oil solidifies on cooling.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 3-Chloro-4-(3-(trifluoromethyl)phenoxy)aniline (<b>4c</b>)</h4><div class="NLM_p last">(1.13 g, yield 79%) brown oil solidifies on cooling.<a onclick="showRef(event, 'ref108 ref109'); return false;" href="javascript:void(0);" class="ref ref108 ref109">(108,109)</a><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 7.57–7.52 (t, <i>J</i> = 7.9 Hz, 1H, Ar-5′-H), 7.38–7.36 (d, <i>J</i> = 7.8 Hz, 1H, Ar-4′-H), 7.10–7.06 (m, 2H, Ar-2′,6′-H), 7.00–6.97 (d, <i>J</i> = 8.6 Hz, 1H, Ar-5-H), 6.75–6.76 (d (m-coupling), <i>J</i> = 1.6 Hz, 1H, Ar-2-H), 6.61–6.57 (dd, <i>J</i> = 8.6, 1.7 Hz, 1H, Ar-6-H), 5.42 (s, 2H, NH<sub>2</sub>, exchanged with D<sub>2</sub>O).</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 3-Chloro-4-(3-methylphenoxy)aniline (<b>4d</b>)</h4><div class="NLM_p last">(0.87 g, 75%), colorless oil.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Synthesis of <i>N</i>′-(2-Cyano-4-nitrophenyl)-<i>N</i>,<i>N</i>-dimethylformimidamide (<b>6</b>)</h3><div class="NLM_p last">A stirred mixture of 2-amino-5-nitrobenzonitrile (<b>5</b>) (2.5 g, 15.3 mmol) and <i>N</i>,<i>N</i>-dimethylformamide dimethylacetal (5 mL, 37.6 mmol, 2.4 equiv) was heated under reflux for 2 h and then cooled to room temperature. The resulting precipitate was filtered, washed with ether (3 × 25 mL), and then dried to afford <i>N</i>′-(2-cyano-4-nitro-phenyl)-<i>N</i>,<i>N</i>-dimethyl-formamidine (<b>6</b>) as a yellow solid (3 g, 90% yield, mp 153–155 °C, as reported).<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Synthesis of 4-(3-Chloro-4-(3-fluorobenzyloxyl)phenylamino)-6-nitroquinazoline (<b>7</b>)</h3><div class="NLM_p last">Compound (<b>6</b>; 2 g, 9.17 mmol) was held at reflux for 3 h with 3-chloro-4-((3-fluorobenzyl)oxy)aniline (<b>4a</b>) (2.3 g, 9.17 mmol) in glacial acetic acid (10 mL). The formed precipitate was filtered while hot and washed with diethylether (3 × 10 mL) to afford the corresponding nitroquinazoline derivative (<b>7</b>) as a yellow to orange solid (3.6 g, 92.5% yield, mp; 280 °C, as reported).<a onclick="showRef(event, 'ref110 ref113'); return false;" href="javascript:void(0);" class="ref ref110 ref113">(110,113)</a></div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Synthesis of 4-(3-Chloro-4-(3-fluorobenzyloxyl)phenylamino)-6-aminoquinazoline (<b>8</b>)</h3><div class="NLM_p last">To a suspension of the 6-nitro derivative (<b>15</b>; 2.12 g, 5 mmol) and reduced iron (0.85 g, 15 mmol) in ethanol (50 mL), a solution of calcium chloride (0.6 g, 5 mmol, 5 mL water) was added. The mixture was stirred at 90 °C for 12 h. The formed black solid was removed by filtration, and the filtrate was centrifuged for 30 min and then was filtered. The clear filtrate was concentrated under vacuum to afford <b>8</b> as a pure yellow solid (97% yield, mp; 190 °C, as reported).<a onclick="showRef(event, 'ref100 ref110'); return false;" href="javascript:void(0);" class="ref ref100 ref110">(100,110)</a></div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> 2-Chloro-<i>N</i>-(4-((3-chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl) Acetamide (<b>9</b>)</h3><div class="NLM_p">To a solution of 6-aminoquinazoline (<b>8</b>) (1.3 g, 3.2 mmol) in acetone (20 mL), NaHCO<sub>3</sub> (0.3 g, 3.5 mmol) was added. The reaction mixture was stirred under nitrogen at 0 °C. This was followed by the dropwise addition of solution of 2-chloroacetylchloride (0.4 g, 3.4 mmol) in acetone (2 mL). The reaction mixture was stirred for 1 h at 0 °C.</div><div class="NLM_p last">The excess solvent was removed under reduced pressure. Cold water (20 mL) was added to the remaining residue and the resulting suspension was neutralized with a saturated solution of NaHCO<sub>3</sub>, if necessary. The formed solid was then filtered, dried, and washed with diethylether (3 × 15 mL) to afford compound <b>9</b> as a pure yellow solid. (0.9 g, yield 61%, mp >300 °C); <sup>1</sup>H NMR (300 MHz, DMSO): δ 11.31 (s, 1H, NH, exchanged with D<sub>2</sub>O), 11.21 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.97 (s, 1H, Ar-2-H), 8.82 (s, 1H, Ar-5-H), 8.06–8.03 (d, <i>J</i> = 8.6 Hz, 1H, Ar-8-H), 7.96–7.93 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.86 (s, 1H, Ar-2′-H), 7.61–7.59 (d, <i>J</i> = 7.9 Hz, 1H, Ar-6″-H), 7.49–7.47 (d, <i>J</i> = 7.2 Hz, 1H, Ar-6′-H), 7.34–7.31 (m 3H, Ar-5′,2″, 4″-H), 7.22–7.16 (t, <i>J</i> = 7.8 Hz, 1H, Ar-5″-H), 5.30 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 4.44 (s, 2H, CH<sub>2</sub>, acetamido). <sup>13</sup>C DEPT (75 MHz, DMSO): δ 150.81 (1C, Ar-2-CH), 131.04 (1C, Ar-5″-CH), 129.50 (1C, Ar-8-CH), 126.69 (1C, Ar-7-CH), 124.95 (1C, Ar-6″-CH), 123.82 (1C, Ar-6′-CH), 122.53 (1C, Ar-2′-CH), 115.40 (1C, Ar-5′-CH), 114.59 (1C, Ar-4″-CH), 114.39 (1C, Ar-2″-CH), 113.45 (1C, Ar-5-CH), 69.83 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 43.89 (1C, CH<sub>2</sub>, acetamido); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 471.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Synthesis of <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2-((2-hydroxyethyl)amino)acetamide (<b>10a</b>)</h3><div class="NLM_p">A mixture of the chloroacetamido intermediate (<b>9</b>) (0.47 g, 1 mmol) and 2-aminoethanol (0.25 g, 0.3 mL, 5 mmol) in methanol (20 mL) was held at reflux for 4 h in the presence of triethylamine (15 drops). Methanol was then removed under reduced pressure, and the remaining residue was dissolved in dichloromethane (DCM) and purified by flash column chromatography with ethylacetate/MeOH (90:10%) as an eluent to yield compound <b>10a</b>.</div><div class="NLM_p last">White solid, mp 140–145 °C, (0.28 g, yield; 57%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.15 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.76 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.60 (s, 1H, Ar-2-H), 8.51 (s, 1H, Ar-5-H), 8.07–8.04 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.99 (d, <i>J</i> = 2.5 Hz (m-coupling), 1H, Ar-2′-H), 7.78–7.75 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.73–7.69 (dd, <i>J</i> = 9.0, 2.5 Hz, 1H, Ar-6′-H), 7.51–7.44 (m, 1H, Ar-4″-H), 7.34–7.29 (m, 2H, Ar-5′,2″-H), 7.27–7.24 (d, <i>J</i> = 9, Ar-6″-H), 7.21–7.16 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 5.25 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 4.68–4.65 (t, <i>J</i> = 4.9 Hz, 1H, OH, exchanged with D<sub>2</sub>O), 3.54–4.49 (q, <i>J</i> = 5 Hz, 1H, CH<sub>2</sub>, OH<b>C<i>H</i></b><sub><b>2</b></sub>CH<sub>2</sub>NH), 3.37 (s, 2H, CH<sub>2</sub>, acetamido), 2.69–2.66 (t, <i>J</i> = 5.3 Hz, 2H, CH<sub>2</sub>, OHCH<sub>2</sub><b>C<i>H</i></b><sub><b>2</b></sub>NH); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.67 (1C, Ar-2-CH), 131.07 (1C, Ar-5″-CH), 130.96 (1C, Ar-8-CH), 128.84 (1C, Ar-7-CH), 127.24 (1C, Ar-6″-CH), 124.54 (1C, Ar-6′-CH), 123.77 (1C, Ar-2′-CH), 115.00 (1C, Ar-5′-CH), 114.69 (1C, Ar-4″-CH), 114.34 (1C, Ar-2″-CH), 112.13 (1C, Ar-5-CH), 69.83 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 60.83 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH), 53.16 (1C, CH<sub>2</sub>, acetamido), 52.18 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH); <sup>13</sup>C NMR (75 MHz, DMSO): δ 171.18 (1C, C═O), 167.28 (1C, Ar-4-C), 161.05 (1C, Ar-3″-C) 157.77 (1C, Ar-4′-C), 153.67 (1C, Ar-2-CH), 150.01 (1C, Ar-6-C), 146.91 (1C, Ar-8a-C), 140.11 (1C, Ar-1″-C), 136.81 (1C, Ar-1′-C), 133.86 (1C, Ar-3′-C), 131.06 (1C, Ar-5″-CH), 130.96 (1C, Ar-8-CH), 128.84 (1C, Ar-7-CH), 127.24 (1C, Ar-6″-CH), 124.54 (1C, Ar-6′-CH), 123.76 (1C, Ar-2′-CH), 115.72 (1C, Ar-4a-C), 115.28 (1C, Ar-5′-CH), 114.72 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 112.13 (1C, Ar-5-CH), 69.85 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 60.84 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH), 53.16 (1C, CH<sub>2</sub>, acetamido), 52.17 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>24</sub>ClFN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 496.1552; found, 496.1547.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> General Procedure for the Synthesis of Compounds <b>10b</b> and <b>10c</b></h3><div class="NLM_p">A mixture of compound <b>9</b> (0.47 g, 1 mmol) and the appropriate secondary amine, 1-methyl piperazine or morpholine (50 mmol), was heated under reflux temperature for 2 h. The reaction mixture was then cooled to room temperature, poured onto iced-water (30 mL) portion wise while stirring, and 0.2 g of NaCl was added, where a precipitate was formed. The resultant precipitate was filtered, washed with water, and dried. Trituration of the dried precipitate with diethylether (3 × 15 mL) afforded the desired products (<b>10b</b> and <b>10c</b>) as pure compounds with no need for further purification.</div><div id="sec4_11_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2 Morpholino Acetamide (<b>10b</b>)</h4><div class="NLM_p last">Yellowish white solid, mp 175–180 °C, (0.4 g, yield; 87%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.01 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.77 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.62 (s, 1H, Ar-2-H), 8.51 (s, 1H, Ar-5-H), 8.01–7.97 (dd, <i>J</i> = 8.7, 1.9 Hz, 2H, Ar-8, 2′-H) 7.77–7.74 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.72–7.68 (dd, <i>J</i> = 8.9, 2.3 Hz, 1H, Ar-6′-H), 7.50–7.43 (m, 1H, Ar-4″-H), 7.34–7.30 (m, 2H, Ar-5′, 2″-H), 7.27–7.24 (d, <i>J</i> = 9, Ar-6″-H), 7.21–7.15 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 5.25 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 3.69–3.66 (t, <i>J</i> = 6, 4H, 2CH<sub>2</sub>, morpholino), 3.21 (s, 2H, CH<sub>2</sub>, acetamido), 2.58–2.55 (t, <i>J</i> = 6, 4H, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C DEPT (75 MHz, DMSO): δ 153.77 (1C, Ar-2-CH), 131.05 (1C, Ar-5″-CH), 130.95 (1C, Ar-8-CH), 128.73 (1C, Ar-7-CH), 127.81 (1C, Ar-6″-CH), 124.67 (1C, Ar-6′-CH), 123.76 (1C, Ar-2′-CH), 115.27 (1C, Ar-5′-CH), 114.74 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 112.85 (1C, Ar-5-CH), 69.89 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 66.57 (2C, 2CH<sub>2</sub>, morpholino), 62.38 (1C, CH<sub>2</sub>, acetamido), 53.72 (2C, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C NMR (75 MHz, DMSO): δ 168.80 (1C, C═O), 164.29 (1C, Ar-4-C), 161.06 (1C, Ar-3″-C), 157.83 (1C, Ar-4′-C), 153.76 (1C, Ar-2-CH), 150.09 (1C, Ar-6-C), 147.06 (1C, Ar-8a-C), 140.21 (1C, Ar-1″-C), 136.63 (1C, Ar-1′-C), 133.84 (1C, Ar-3′-C), 131.05 (1C, Ar-5″-CH), 130.94 (1C, Ar-8-CH), 128.73 (1C, Ar-7-CH), 127.80 (1C, Ar-6″-CH), 124.67 (1C, Ar-6′-CH), 123.74 (1C, Ar-2′-CH), 115.65 (1C, Ar-4a-C), 115.27 (1C, Ar-5′-CH), 114.78 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 112.85 (1C, Ar-5-CH), 69.92 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 66.57 (2C, 2CH<sub>2</sub>, morpholino), 62.38 (1C, CH<sub>2</sub>, acetamido), 53.73 (2C, 2CH<sub>2</sub>, morpholino); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>26</sub>ClFN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 522.1708; found, 522.1705.</div></div><div id="sec4_11_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2-(4-methylpiperazin-1-yl)acetamide (<b>10c</b>)</h4><div class="NLM_p last">Grayish brown solid, mp 155–158 °C, (0.4 g, yield; 78%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.93 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.77 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.59 (s, 1H, Ar-2-H), 8.51 (s, 1H, Ar-5-H), 8.02–7.99 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.98–7.97 (d, <i>J</i> = 3 Hz (m-coupling) 1H, Ar-2′-H), 7.77–7.74 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.72–7.68 (dd, <i>J</i> = 9, 3 Hz, 1H, Ar-6′-H), 7.50–7.43 (m, 1H, Ar-4″-H), 7.34–7.29 (m, 2H, Ar-5′, 2″-H), 7.27–7.24 (d, <i>J</i> = 9, Ar-6″-H), 7.21–7.15 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 5.25 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 3.19 (s, 2H, CH<sub>2</sub>, acetamido), 2.57 (br s, 4H, 2CH<sub>2</sub>, piperazino), 2.41 (br s, 4H, 2CH<sub>2</sub> piperazino), 2.18 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO): δ 153.76 (1C, Ar-2-CH), 131.07 (1C, Ar-5″-CH), 130.96 (1C, Ar-8-CH), 128.77 (1C, Ar-7-CH), 127.65 (1C, Ar-6″-CH), 124.67 (1C, Ar-6′-CH), 123.79 (1C, Ar-2′-CH), 115.29 (1C, Ar-5′-CH), 114.69 (1C, Ar-4″-CH), 114.34 (1C, Ar-2″-CH), 112.63 (1C, Ar-5-CH), 69.83 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 62.14 (1C, CH<sub>2</sub>, acetamido), 54.98 (2C, 2CH<sub>2</sub>, piperazino), 53.28 (2C, 2CH<sub>2</sub>, piperazino), 46.20 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO): δ 169.01 (1C, C═O), 164.28 (1C, Ar-4-C), 161.05 (1C, Ar-3″-C), 157.81 (1C, Ar-4′-C), 153.76 (1C, Ar-2-CH), 150.06 (1C, Ar-6-C), 147.02 (1C, Ar-8a-C), 140.19 (1C, Ar-1″-C), 136.64 (1C, Ar-1′-C), 133.78 (1C, Ar-3′-C), 131.07 (1C, Ar-5″-CH), 130.96 (1C, Ar-8-CH), 128.77 (1C, Ar-7-CH), 127.65 (1C, Ar-6″-CH), 124.83 (1C, Ar-6′-CH), 123.79 (1C, Ar-2′-CH), 115.65 (1C, Ar-4a-C), 115.28 (1C, Ar-5′-CH), 114.70 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 112.63 (1C, Ar-5-CH), 69.83 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 62.14 (1C, CH<sub>2</sub>, acetamido), 54.99 (2C, 2CH<sub>2</sub>, piperazino), 53.28 (2C, 2CH<sub>2</sub>, piperazino), 46.20 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>29</sub>ClFN<sub>6</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 535.2025; found, 535.2024.</div></div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Ethyl-<i>N</i>-(2-cyano-4-nitrophenyl)formimidate (<b>11</b>)</h3><div class="NLM_p last">2-Amino-5-nitrobenzonitrile (<b>5</b>; 5 g, 30.6 mmol) was held at reflux in triethyl orthoformate (50 mL) for 30 h in the presence of acetic anhydride (10 drops). The reaction mixture was cooled to room temperature, poured onto ice water, and stirred for half an hour. The formed precipitate was filtered under vacuum and left to dry to yield compound <b>11</b>. Yellow solid, 5.6 g, yield 88%.<a onclick="showRef(event, 'ref114'); return false;" href="javascript:void(0);" class="ref ref114">(114)</a></div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> General Procedure for the Synthesis of Compounds (<b>12a–c</b>)</h3><div class="NLM_p">Compound (<b>11</b>; 2.19 g, 10 mmol) was held at reflux for 7 h with the appropriate aniline derivative (<b>4b–d</b>) (10.2 mmol) in glacial acetic acid (20 mL). The reaction mixture was then poured onto the ice/water mixture and stirred for 30 min with the addition of 0.2 g of NaCl, and the precipitate formed was filtered, washed with diethylether, and dried. The obtained solid was recrystallized from methanol to afford the corresponding nitroquinazoline derivative (<b>12a–c</b>) as a fluffy yellow solid.</div><div id="sec4_13_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> <i>N</i>-(3-Chloro-4-(3-fluorophenoxy)phenyl)-6-nitroquinazolin-4-amine (<b>12a</b>)</h4><div class="NLM_p last">(3.1 g, yield 76%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.47 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.61 (s, 1H, Ar-5-H), 8.75 (s, 1H, Ar-2-H), 8.55–8.52 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 8.23 (s, 1H, Ar-2′-H), 7.94–7.88 (m, 2H, Ar-2″, 8-H), 7.46–7.38 (m, 1H, Ar-6′-H), 7.31–7.28 (d, <i>J</i> = 8.5 Hz, 1H, Ar-4″-H), 6.99–6.93 (t, <i>J</i> = 8.4 Hz, 1H, Ar-5″-H), 6.87–6.83 (d, <i>J</i> = 10.5 Hz, 1H, Ar-5′-H), 6.79–6.76 (d, <i>J</i> = 8.5 Hz, 1H, Ar-6″-H); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 157.90 (1C, Ar-2-CH), 131.89 (1C, Ar-8-CH), 130.06 (1C, Ar-5″-CH), 127.15 (1C, Ar-7-CH), 124.5 6 (1C, Ar-5′-CH), 123.15 (1C, Ar-6′-CH), 122.60 (1C, Ar-5-CH), 121.15 (1C, Ar-2′-CH), 113.02 (1C, Ar-6″-CH), 110.19 (1C, Ar-4″-CH), 104.72 (1C, Ar-2″-CH); MS (APCI<sup>–</sup>): <i>m</i>/<i>z</i> 411.5[M + 1]<sup>+</sup>.</div></div><div id="sec4_13_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> <i>N</i>-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)-6-nitroquinazolin-4-amine (<b>12b</b>)</h4><div class="NLM_p last">(3.5 g, yield; 78%), mp 185–187 °C, as reported.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a></div></div><div id="sec4_13_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> <i>N</i>-(3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)-6-nitroquinazolin-4-amine (<b>12c</b>)</h4><div class="NLM_p last">(3.3 g, yield: 82%), <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.48 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.63 (s, 1H, Ar-5-H), 8.75 (s, 1H, Ar-2-H), 8.57–8.54 (dd, <i>J</i> = 9.1, 1.9 Hz, 1H, Ar-7-H), 8.19 (s, 1H, Ar-2′-H), 7.95–7.92 (d, <i>J</i> = 9.1 Hz, 1H, Ar-8-H), 7.79–7.76 (d, <i>J</i> = 9.1 Hz, 1H, Ar-6′-H), 7.35–7.32 (d, <i>J</i> = 9.1 Hz, 1H, Ar-5′-H), 7.24–7.19 (m, 1H, Ar-2″), 7.12–7.07 (t, <i>J</i> = 7.4 Hz, 1H, Ar-5″-H), 7.00–6.97 (d, <i>J</i> = 7.7 Hz, 1H, Ar-6″-H), 6.80–6.78 (d, <i>J</i> = 7.5 Hz, 1H, Ar-4″-H), 2.25 (s, 3H, CH<sub>3</sub>). <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.03 (1C, Ar-2-CH), 132.02 (1C, Ar-8-CH), 130.03 (1C, Ar-5″-CH), 127.93 (1C, Ar-7-CH), 127.17 (1C, Ar-4″-CH), 124.88 (1C, Ar-5′-CH), 124.38 (1C, Ar-6′-CH), 123.24 (1C, Ar-5-CH), 121.20 (1C, Ar-2′-CH), 119.92 (1C, Ar-2″-CH), 117.83 (1C, Ar-6″-CH). MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 407.1 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> General Procedure for the Synthesis of Compounds (<b>13a–c</b>)</h3><div class="NLM_p">A mixture of the 6-nitro quinazoline derivatives (<b>12a–c</b>) (5 mmol) and stannous chloride (25 mmol) in methanol (20 mL) was stirred at reflux for 1.5 h, under a nitrogen atmosphere. Excess methanol was removed under reduced pressure, the remaining residue was dissolved in ethyl acetate (200 mL), and the resulting solution was neutralized with an aqueous saturated solution of NaHCO<sub>3</sub> and stirred for 1 h. The resulting gelatinous precipitate was centrifuged for 45 min and then discarded by filtration. The clear filtrate was transferred into a separating funnel, and the organic layer was separated from the aqueous one. The aqueous phase was extracted with ethyl acetate (2 × 20 mL), and the organic fractions were combined, dried over anhydrous sodium sulfate, and concentrated under vacuum to afford the corresponding 6-amino quinazoline derivatives <b>13a–c</b>.</div><div id="sec4_14_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> <i>N</i>-4-(3-Chloro-4-(3-fluorophenoxy)phenyl)quinazoline-4,6-diamine (<b>13a</b>)</h4><div class="NLM_p last">Brown oil, (1.63 g, yield; 86%), <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.54 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.38 (s, 1H, Ar-2-H), 8.27 (s, 1H, Ar-2″-H), 7.91–7.88 (d, <i>J</i> = 8.7 Hz, 1H, Ar-8-H), 7.57–7.54 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.43–7.38 (m, 1H, AR-5-H), 7.34 (s, 1H, Ar-2′-H), 7.28–7.25 (d, <i>J</i> = 8.5 Hz, 2H, Ar-4″, 6′-H), 6.96–6.91 (t, <i>J</i> = 8.1 Hz, 1H, Ar-5″-H), 6.82–6.79 (d, <i>J</i> = 10.5 Hz, 1H, Ar-5′-H), 6.75–6.73 (d, <i>J</i> = 8.2 Hz, 1H, Ar-6″-H), 5.65 (s, 2H, NH<sub>2</sub>, exchanged with D<sub>2</sub>O). MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 381.2 [M + 1]<sup>+</sup>.</div></div><div id="sec4_14_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> <i>N</i>-4-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)quinazoline-4,6-diamine (<b>13b</b>)</h4><div class="NLM_p last">Yellow solid, 1.97 g, yield; 86%, mp 104–106 °C, as reported.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a></div></div><div id="sec4_14_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> <i>N</i>-4-(3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)quinazoline-4,6-diamine (<b>13c</b>)</h4><div class="NLM_p last">Brown solid, (1.77 g, yield; 94%), <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>):δ 9.46 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.37 (s, 1H, Ar-2-H), 8.23 (s, 1H, Ar-2′-H), 7.81–7.78 (d, <i>J</i> = 8.6 Hz, 1H, Ar-8-H), 7.57–7.54 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.34 (s, 1H, Ar-5-H), 7.31–7.25 (m, 2H, Ar-6″, 6′), 7.20–7.15 (m, 1H, Ar-2″-H), 7.07–7.02 (t, <i>J</i> = 7.3 Hz, 1H, Ar-5″-H), 7.00–6.97 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 6.73–6.70 (d, <i>J</i> = 7.5 Hz, 1H, Ar-4″-H), 5.61 (s, 2H, NH<sub>2</sub>, exchanged with D<sub>2</sub>O), 2.26 (s, 3H, CH<sub>3</sub>). <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.10 (1C, Ar-2-CH), 131.89 (1C, Ar-8-CH), 129.17 (1C, Ar-5″-CH), 127.78 (1C, Ar-7-CH), 124.23 (1C, Ar-4″-CH), 123.90 (1C, Ar-5′-CH), 123.49 (1C, Ar-6′-CH), 122.05 (1C, Ar-2′-CH), 120.53 (1C, Ar-2″-CH), 117.07 (1C, Ar-6″-CH), 101.34 (1C, Ar-5-CH), 16.21 (1C, CH<sub>3</sub>). MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 377.1 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i65" class="anchor-spacer"></div><h3 class="article-section__title" id="_i65"> General Procedure for the Synthesis of Compounds <b>14a–c</b></h3><div class="NLM_p">To the solution of intermediate <b>13a–c</b> (5 mmol) in acetone (20 mL), NaHCO<sub>3</sub> (0.44 g, 5.2 mmol) was added. The reaction mixture was stirred under nitrogen at 0 °C, followed by the dropwise addition of 2-choroacetyl chloride (0.4 mL, 5.2 mmol) and stirring was continued for 1 h at 0 °C. Excess solvent was then removed under reduced pressure.</div><div class="NLM_p">Cold water was added (20 mL) to the remaining residue and the resulting suspension was neutralized with a solution of NaHCO<sub>3</sub>, if necessary (if pH still acidic). The formed solid was then filtered and purified by column chromatography with EtOAc as an eluent to afford compounds <b>14a–c</b> as a yellow solid.</div><div id="sec4_15_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 2-Chloro-<i>N</i>-(4-((3-chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)acetamide (<b>14a</b>)</h4><div class="NLM_p last">(1.41 g, yield; 62%), <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.93 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.83 (s, 1H, Ar-2-H), 8.69 (s, 1H, Ar-5-H), 8.14 (s, 1H, Ar-2″-H), 7.9–7.94 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8-H), 7.87–7.80 (m, 2H, Ar-7, 6′-H), 7.45–7.38 (dd, 1H, Ar-4″-H), 7.30–7.28 (d, <i>J</i> = 8.8 Hz, 1H, Ar-5′-H), 6.99–6.93 (t, <i>J</i> = 7.7 Hz, 1H, Ar-5″-H), 6.86–6.76 (m, 2H, Ar-2′,6″-H), 4.40 (s, 2H, CH<sub>2</sub>). <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.59 (1C, Ar-2-CH), 131.90 (1C, Ar-5″-CH), 128.40 (1C, Ar-8-CH), 126.65 (1C, Ar-7-CH), 125.12 (1C, Ar-5′-CH), 123.76 (1C, Ar-6′-CH), 122.55 (1C, Ar-2′-CH), 113.11 (1C, Ar-6″-CH), 112.99 (1C, Ar-4″-CH), 110.19 (1C, Ar-2″-CH), 104.71 (1C, Ar-5-CH), 43.90 (1C, CH<sub>2</sub>). MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 457.2 [M + 1]<sup>+</sup>.</div></div><div id="sec4_15_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 2-Chloro-<i>N</i>-(4-((3-chloro-4-(3-(trifluoromethyl) phenoxy)phenyl)amino)quinazolin-6-yl)acetamide (<b>14b</b>)</h4><div class="NLM_p last">1.36 g, yield; 54%, <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.94 (s, 2H, NH, exchanged with D<sub>2</sub>O), 8.87 (s, 1H, Ar-2-H), 8.73 (s, 1H, Ar-5-H), 8.17–8.18 (d (m-coupling), <i>J</i> = 1.2 Hz, 1H, Ar-2′-H), 7.97–7.94 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.89–7.82 (m, 2H, Ar-7, 6′-H), 7.66–7.61 (t, <i>J</i> = 7.8 Hz, 1H, Ar-5″-H), 7.51–7.48 (d, <i>J</i> = 7.7 Hz, 1H, Ar-4″-H), 7.36–7.33 (d, <i>J</i> = 7.7 Hz, 1H, Ar-6″-H), 7.28–7.24 (m, 2H, Ar-2″, 5′), 4.41 (s, 2H, CH<sub>2</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.40 (1C, Ar-2-CH), 131.98 (1C, Ar-5″-CH), 128.60 (1C, Ar-8-CH), 126.16 (1C, Ar-7-CH), 125.40 (1C, Ar-6″-CH), 124.10 (1C, Ar-5′-CH), 122.74 (1C, Ar-6′-CH), 120.96 (1C, Ar-4″-CH), 120.16 (1C, Ar-2′-CH), 113.56 (1C, Ar-2″-CH), 113.12 (1C, Ar-5-CH), 43.90 (1C, CH<sub>2</sub>); MS (ESI<sup>+</sup>): <i>m</i>/<i>z</i> 507.1 [M + 1]<sup>+</sup>.</div></div><div id="sec4_15_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> 2-Chloro-<i>N</i>-(4-((3-chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl) Acetamide (<b>14c</b>)</h4><div class="NLM_p last">(2.01 g, yield; 89%); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.22 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.97 (s, 1H, Ar-2-H), 8.83 (s, 1H, Ar-5-H), 8.08–7.95 (m, 3H, Ar-Ar-2′,8,7-H), 7.65–7.62 (d, <i>J</i> = 8.8 Hz, 1H, Ar-6′-H), 7.36–7.34 (d, <i>J</i> = 7.8 Hz, 1H, Ar-6″-CH), 7.26–7.21 (m, 1H, Ar-2″-CH), 7.14–7.09 (t, <i>J</i> = 7.8 Hz, 1H, Ar-5″-CH), 6.97–6.94 (d, <i>J</i> = 8.8 Hz, 1H, Ar-5′-H), 6.86–6.83 (d, <i>J</i> = 7.9 Hz, 1H, Ar-4″-CH), 4.44 (s, 2H, CH<sub>2</sub>), 2.24 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 150.93 (1C, Ar-2-CH), 132.10 (1C, Ar-8-CH), 129.47 (1C, Ar-5″-CH), 128.01 (1C, Ar-7-CH), 126.71 (1C, Ar-4″-CH), 124.98 (1C, Ar-5′-CH), 124.7 (1C, Ar-6′-CH)<sub>3</sub>, 122.94 (1C, Ar-2′-CH), 119.29 (1C, Ar-2″-CH), 118.40 (1C, Ar-6″-CH), 113.44 (1C, Ar-5-CH), 43.86 (1C, CH<sub>2</sub>), 16.12 (1C, CH<sub>3</sub>); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 453.6 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i69" class="anchor-spacer"></div><h3 class="article-section__title" id="_i69"> General Procedure for the Synthesis of Compounds (<b>15a–l</b>)</h3><div class="NLM_p">A mixture of appropriate 6-chloro-acetamido-quinazoline derivative (<b>14a–c</b>) (1 mmol) and the appropriate secondary amine, 1-methyl piperazine or morpholine (50 mmol), was placed into a 25 mL round-bottomed flask or into a capped vial in the case of low boiling point secondary amine including dimethylamine or diethylamine (50 mmol). The reaction mixture was heated under reflux temperature and monitored by TLC. After 2 h, the TLC of reaction mass indicated the absence of intermediate (<b>14a–c</b>). The reaction mixture was cooled to room temperature and poured onto iced-water (30 mL) portion wise while stirring, and the resultant precipitate was collected by filtration, washed with water, dried, and purified by flash column chromatography using DCM/methanol (95:5%) as an eluent to afford the desired products (<b>15a-1</b>).</div><div id="sec4_16_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-2-morpholino Acetamide (<b>15a</b>)</h4><div class="NLM_p last">Yellow solid, 0.39 g, yield; 77%, mp 122–125 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.05 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.95 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.68 (s, 1H, Ar-2-H), 8.59 (s, 1H, Ar-5-H), 8.22 (s, 1H, Ar-2″-H), 8.03–8.00 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.90–7.87 (d, 1H, <i>J</i> = 8.5, Ar-6′-H), 7.81–7.78 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.44, 7.37 (dd, 1H, Ar-4″-H), 7.30–7.27 (d, 1H, <i>J</i> = 8.7, Ar-5′-H), 6.97–6.92 (t, <i>J</i> = 7.1 Hz, 1H, Ar-5″-H), 6.84–6.74 (m, 2H, Ar-2′, 6″-H), 3.67 (br s, 4H, 2CH<sub>2</sub>, morpholino), 3.22 (s, 2H, CH<sub>2</sub>, acetamido), 2.57 (br s, 4H, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.58 (1C, Ar-2-CH), 131.86 (1C, Ar-5″-CH), 128.84 (1C, Ar-8-CH), 128.00 (1C, Ar-7-CH), 124.08 (1C, Ar-5′-CH), 122.83 (1C, Ar-6′-CH), 112.78 (1C, Ar-2′-CH), 110.28 (1C, Ar-6″-CH), 110.00 (1C, Ar-4″-CH), 104.80 (1C, Ar-2″-CH), 104.47 (1C, Ar-5-CH), 66.57 (2C, 2CH<sub>2</sub>, morpholine), 62.40 (1C, CH<sub>2</sub>, acetamido), 53.72 (2C, 2CH<sub>2</sub>, morpholine); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 173.77 (1C, C═O), 163.85 (1C, Ar-4-C), 162.42 (1C, Ar-3″-C), 158.34 (1C, Ar-1″-C), 151.81 (1C, Ar-2-CH), 150.97 (1C, Ar-6-C), 142.58 (1C, Ar-8a-C), 141.52 (1C, Ar-4′-C), 136.62 (1C, Ar-1′-C), 136.49 (1C, Ar-5″-CH), 133.54 (1C, Ar-8-CH), 132.73 (1C, Ar-3′-C), 130.09 (1C, Ar-7-CH), 128.90 (1C, Ar-5′-CH), 127.66 (1C, Ar-6′-CH), 120.45 (1C, Ar-2′-C), 117.52 (1C, Ar-4a-C), 115.03 (1C, Ar-6″-CH), 114.75 (1C, Ar-4″-CH), 109.49 (1C, Ar-2″-CH), 109.16 (1C, Ar-5-CH), 71.29 (2C, 2CH<sub>2</sub>, morpholino), 67.10 (2C, 2CH<sub>2</sub>, morpholino), 58.42 (2C, 2CH<sub>2</sub>, morpholino); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>24</sub>ClFN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 508.1552; found, 508.1559.</div></div><div id="sec4_16_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-2-(4-methylpiperazin-1-yl)acetamide (<b>15b</b>)</h4><div class="NLM_p last">Light-brown solid, 0.35 g, yield; 68%, mp 163–165 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.97 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.95 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.65 (s, 1H, Ar-2-H), 8.59 (s, 1H, Ar-5-H), 8.22 (s, 1H, Ar-2′-H), 8.05–8.02 (d, <i>J</i> = 8.7 Hz, 1H, Ar-8-H), 7.90–7.89, 7.87–7.86 (dd, <i>J</i> = 8.8, 1.07 Hz, 1H, Ar-7-H), 7.81–7.78 (d, 1H, <i>J</i> = 9, Ar-6′-H), 7.45–7.37 (m, 1H, Ar-2″-H), 7.30–7.28 (d, 1H, <i>J</i> = 8.9, Ar-5′-H), 6.98–6.92 (t, <i>J</i> = 8.3 Hz, 1H, Ar-5″-H), 6.84–6.81 (d, <i>J</i> = 8.5 Hz, 1H, Ar-4″-H), 6.78–6.75 (d, <i>J</i> = 8.2 Hz, 1H, Ar-6″-H), 3.20 (s, 2H, CH<sub>2</sub>, acetamido), 2.57 (br s, 4H, 2CH<sub>2</sub>, piperazino), 2.41 (br s, 4H, 2CH<sub>2</sub>, piperazino), 2.18 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.54 (1C, Ar-2-CH), 131.75 (1C, Ar-5″-CH), 128.86 (1C, Ar-8-CH), 127.84 (1C, Ar-7-CH), 125.01 (1C, Ar-5′-CH), 124.07 (1C, Ar-6′-CH), 122.79 (1C, Ar-2′-CH), 112.75 (1C, Ar-6″-CH), 112.58 (1C, Ar-4″-CH), 110.01 (1C, Ar-2″-CH), 104.46 (1C, Ar-5-CH), 62.17 (1C, CH<sub>2</sub>, acetamido), 54.99 (2C, 2CH<sub>2</sub>, piperazine), 53.29 (2C, 2CH<sub>2</sub>, piperazino), 46.23 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.09 (1C, C═O), 159.12 (1C, Ar-4-C), 157.63 (1C, Ar-1″-C), 153.57 (1C, Ar-2-CH), 147.12 (1C, Ar-6-C), 146.17 (1C, Ar-8a-C), 137.91 (1C, Ar-4′-C), 136.84 (2C, Ar-1′, 3″-C), 131.74 (1C, Ar-5″-CH), 128.87 (1C, Ar-8-CH), 127.86 (1C, Ar-7-CH), 125.29 (2C, Ar-3′-C, Ar-6″-CH), 125.00 (1C, Ar-5′-CH), 124.10 (2C, Ar-6′, 4″-CH), 122.86 (1C, Ar-2′-CH), 115.76 (1C, Ar-4a-C), 110.00 (1C, Ar-2″-CH), 104.46 (1C, Ar-5-CH), 62.15 (1C, CH<sub>2</sub>, acetamido), 54.98 (2C, 2CH<sub>2</sub>, piperazino), 53.28 (2C, 2CH<sub>2</sub>, piperazino), 46.21 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>27</sub>ClFN<sub>6</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 521.1868; found, 521.1868.</div></div><div id="sec4_16_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-2-(diethylamino) Acetamide (<b>15c</b>)</h4><div class="NLM_p last">Brown solid, 0.4 g, yield; 86%, mp 110–115 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.95 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.62 (s, 1H, Ar-2-H), 8.59 (s, 1H, Ar-5-H), 8.24 (s, 1H, Ar-2″-H), 8.21–8.18 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.92–7.89 (d, 1H, <i>J</i> = 9 Hz, Ar-7-H), 7.81–7.78 (d, 1H, <i>J</i> = 9, Ar-5′-H), 7.45–7.37 (dd, 1H, Ar-6′-H), 7.30–7.27 (d, 1H, <i>J</i> = 8.8, Ar-2′-H), 6.98–6.92 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 6.84–6.81 (d, <i>J</i> = 9 Hz, 1H, Ar-4″-H), 6.77–6.74 (d, 1H, <i>J</i> = 9 Hz, Ar-6″-H), 3.23 (s, 2H, CH<sub>2</sub>, acetamido), 2.68–2.61 (q, <i>J</i> = 7 Hz, 4H, 2CH<sub>2</sub>, (CH<sub>3</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub>2</sub>N−), 1.09–1.04 (t, <i>J</i> = 7 Hz, 6H, 2CH<sub>3</sub>, (<b>C<i>H</i></b><sub><b><i>3</i></b></sub>CH<sub>2</sub>)<sub>2</sub>N−); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.53 (1C, Ar-2-CH), 131.85 (1C, Ar-8-CH), 128.92 (1C, Ar-5″-CH), 127.38 (1C, Ar-7-CH), 124.04 (1C, Ar-5′-CH), 122.79 (1C, Ar-6′-CH), 112.77 (1C, Ar-2′-CH), 112.18 (1C, Ar-6″-CH), 110.28 (1C, Ar-4″-CH), 104.81 (1C, Ar-2″-CH), 104.46 (1C, Ar-5-CH), 57.78 (1C, CH<sub>2</sub>, acetamido), 48.48 (2C, 2CH<sub>2</sub>, (CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub>)<sub>2</sub>N−), 12.44 (2C, 2CH<sub>3</sub>, (<b><i>C</i>H</b><sub><b>3</b></sub>CH<sub>2</sub>)<sub>2</sub>N−); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.98 (1C, C═O), 159.06 (1C, Ar-4-C), 158.92 (1C, Ar-3″-C), 157.65 (1C, Ar-1″-C), 153.55 (1C, Ar-2-CH), 146.86 (1C, Ar-6-C), 146.29 (1C, Ar-8a-C), 137.69 (1C, Ar-4′-C), 136.69 (1C, Ar-1′-C), 131.89 (1C, Ar-8-CH), 128.80 (1C, Ar-5″-CH), 127.51 (1C, Ar-7-CH), 125.38 (1C, Ar-3′-C), 124.20 (1C, Ar-5′-CH), 122.83 (1C, Ar-6′-CH), 115.68 (1C, Ar-4a-C), 112.72 (1C, Ar-2′-CH), 112.05 (1C, Ar-6″-CH), 110.29 (1C, Ar-4″-CH), 104.67 (1C, Ar-2″-CH), 104.34 (1C, Ar-5-CH), 57.68 (1C, CH<sub>2</sub>, acetamido), 48.47 (2C, 2CH<sub>2</sub>, (CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub>)<sub>2</sub>N−), 12.35 (2C, 2CH<sub>3</sub>, (<b><i>C</i>H</b><sub><b>3</b></sub>CH<sub>2</sub>)<sub>2</sub>N−); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>26</sub>ClFN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 494.1759; found, 494.1756.</div></div><div id="sec4_16_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-2-(dimethylamino)acetamide (<b>15d</b>)</h4><div class="NLM_p last">Brown solid, 0.3 g, yield: 72%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.00 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.91 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.65 (s, 1H, Ar-2-H), 8.58 (s, 1H, Ar-5-H), 8.23 (s, 1H, Ar-2″-H), 8.13–8.10 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.90–7.87 (d, 1H, <i>J</i> = 8.6, Ar-6′-H), 7.80–7.77 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.44–7.37 (dd, 1H, Ar-4″-H), 7.30–7.27 (d, 1H, <i>J</i> = 8.8, Ar-5′-H), 6.98–6.92 (t, <i>J</i> = 8.5 Hz, 1H, Ar-5″-H), 6.85–6.75 (m, 2H, Ar-2′,6″-H), 3.16 (s, 2H, CH<sub>2</sub>, acetamido), 2.33 (s, 6H, 2CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.46 (1C, Ar-2-CH), 131.73 (1C, Ar-5″-CH), 128.82 (1C, Ar-8-CH), 127.64 (1C, Ar-7-CH), 123.93 (1C, Ar-5′-CH), 122.79 (1C, Ar-6′-CH), 112.77 (1C, Ar-2′-CH), 110.27 (1C, Ar-6″-CH), 109.98 (1C, Ar-4″-CH), 104.80 (1C, Ar-2″-CH), 104.46 (1C, Ar-5-CH), 63.69 (1C, CH<sub>2</sub>), 45.95 (2C, 2CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.44 (1C, C═O), 165.00 (1C, Ar-4-C), 161.76 (1C, Ar-3″-C), 159.14 (1C, Ar-1″-C), 153.50 (1C, Ar-2-CH), 146.11 (1C, Ar-6-C), 136.96 (1C, Ar-8a-C), 131.86 (1C, Ar-4′-C), 131.73 (1C, Ar-5″-CH), 128.82 (1C, Ar-8-CH), 127.65 (1C, Ar-7-CH), 125.31 (1C, Ar-1′-C), 123.94 (1C, Ar-5′-CH), 122.80 (1C, Ar-6′-CH), 122.69 (1C, Ar-3′-C), 112.75 (1C, Ar-2′-CH), 112.45 (1C, Ar-4a-C), 110.27 (1C, Ar-6″-CH), 109.99 (1C, Ar-4″-CH), 104.79 (1C, Ar-2″-CH), 104.46 (1C, Ar-5-CH), 63.68 (1C, CH<sub>2</sub>), 45.94 (2C, 2CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>22</sub>ClFN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 466.1446; found, 466.1441.</div></div><div id="sec4_16_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i74" class="anchor-spacer"></div><h4 class="article-section__title" id="_i74"> <i>N</i>-(4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-yl)-2-morpholino-acetamide (<b>15e</b>)</h4><div class="NLM_p last">Yellow solid, 0.42 g, yield; 77%, mp 105–108; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.05 (s, H, NH, exchanged with D<sub>2</sub>O), 9.96 (s, H, NH, exchanged with D<sub>2</sub>O), 8.67 (s, 1H, Ar-2-H), 8.60 (s, 1H, Ar-5-H), 8.25 (s, 1H, Ar-2′-H), 8.03–8.00 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.92–7.89 (d, 1H, <i>J</i> = 9 Hz, Ar-7-H), 7.81–7.78 (d, 1H, <i>J</i> = 9, Ar-6′-H), 7.65–7.59 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 7.49–7.46 (d, 1H, <i>J</i> = 9 Hz, Ar-4″-H), 7.35–7.32 (d, 1H, <i>J</i> = 9 Hz, Ar-6″-H), 7.25–7.21 (m, 2H, Ar-2″, 5′), 3.67 (br s, 4H, 2CH<sub>2</sub>, morpholino) 3.22 (s, 2H, CH<sub>2</sub>), 2.57 (br s, 4H, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.58 (1C, Ar-2-CH), 131.91 (1C, Ar-5″-CH), 128.87 (1C, Ar-8-CH), 128.02 (1C, Ar-7-CH), 124.12 (1C, Ar-6″-CH), 122.83 (2C, Ar-5′, 6′-CH), 120.67 (1C, Ar-4″-CH), 119.90 (1C, Ar-2′-CH), 113.27 (1C, Ar-2″-CH), 112.74 (1C, Ar-5-CH), 66.57 (2C, 2CH<sub>2</sub>, morpholino), 62.41 (1C, CH<sub>2</sub>), 53.72 (2C, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.90 (1C, C═O), 158.12 (1C, Ar-4-C), 157.63 (1C, Ar-1″-C), 153.57 (1C, Ar-2-CH), 147.16 (1C, Ar-6-C), 145.84 (1C, Ar-8a-C), 138.15 (1C, Ar-4′-C), 136.83 (2C, Ar-1′, 3″-C), 131.91 (1C, Ar-5″-CH), 128.85 (1C, Ar-8-CH), 128.01 (1C, Ar-7-CH), 125.37 (1C, Ar-3′-C), 125.22 (1C, CF<sub>3</sub>), 124.12 (1C, Ar-6″-CH), 122.95 (2C, Ar-5′, 6′-CH), 120.69 (1C, Ar-4″-CH), 119.91 (1C, Ar-2′-CH), 115.76 (1C, Ar-4a-C), 113.26 (1C, Ar-2″-CH), 112.75 (1C, Ar-5-CH), 66.56 (2C, 2CH<sub>2</sub>, morpholino), 62.40 (1C, CH<sub>2</sub>), 53.72 (2C, 2CH<sub>2</sub>, morpholino); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>24</sub>ClF<sub>3</sub>N<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 558.1520; found, 558.1528.</div></div><div id="sec4_16_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> <i>N</i>-(4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-yl)-2-(4-methylpiperazin-1-yl)acetamide (<b>15f</b>)</h4><div class="NLM_p last">Brown oil solidifies when kept in the fridge overnight, 0.27 g, yield; 48%: <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.96 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.65 (s, 1H, Ar-2-H), 8.59 (s, 1H, Ar-5-H), 8.25 (s, 1H, Ar-2′-H), 8.06–8.03 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.92–7.89 (d, 1H, <i>J</i> = 9 Hz, Ar-7-H), 7.81–7.78 (d, 1H, <i>J</i> = 9, Ar-6′-H), 7.64–7.59 (t, <i>J</i> = 6 Hz, 1H, Ar-5″-H), 7.48–7.46 (d, 1H, <i>J</i> = 6 Hz, Ar-4″-H), 7.34–7.32 (d, 1H, <i>J</i> = 6 Hz, Ar-6″-H), 7.25–7.21 (m, 2H, Ar-2″, 5′), 3.20 (s, 2H, CH<sub>2</sub>, acetamido), 2.57 (br s, 4H, 2CH<sub>2</sub>, piperazino), 2.41 (br s, 4H, 2CH<sub>2</sub>, piperazine), 2.18 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.54 (1C, Ar-2-CH), 131.90 (1C, Ar-5″-CH), 128.87 (1C, Ar-8-CH), 127.85 (1C, Ar-7-CH), 124.12 (1C, Ar-6″-CH), 122.94 (2C, Ar-5′, 6′-CH), 120.66 (1C, Ar-4″-CH), 119.90 (1C, Ar-2′-CH), 113.31 (1C, Ar-2″-CH), 112.54 (1C, Ar-5-CH), 62.15 (1C, CH<sub>2</sub>, acetamido), 54.97 (2C, 2CH<sub>2</sub>, piperazino), 53.26 (2C, 2CH<sub>2</sub>, piperazino), 46.18 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.08 (1C, C═O), 158.12 (1C, Ar-4-C), 157.62 (1C, Ar-1″-C), 153.54 (1C, Ar-2-CH), 147.14 (1C, Ar-6-C), 145.84 (1C, Ar-8a-C), 138.15 (1C, Ar-4′-C), 136.85 (2C, Ar-1′, 3″-C), 131.90 (1C, Ar-5″-CH), 128.87 (1C, Ar-8-CH), 127.85 (1C, Ar-7-CH), 125.81 (1C, Ar-3′-C), 125.40 (1C, CF<sub>3</sub>), 124.12 (1C, Ar-6″-CH), 122.94 (2C, Ar-5′, 6′-CH), 120.67 (1C, Ar-4″-CH), 119.95 (1C, Ar-2′-CH), 115.78 (1C, Ar-4a-C), 113.32 (1C, Ar-2″-CH), 112.54 (1C, Ar-5-CH), 62.15 (1C, CH<sub>2</sub>, acetamido), 54.97 (2C, 2CH<sub>2</sub>, piperazino), 53.26 (2C, 2CH<sub>2</sub>, piperazino), 46.18 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>27</sub>ClF<sub>3</sub>N<sub>6</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 571.1836; found, 571.1837.</div></div><div id="sec4_16_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76"> <i>N-</i>(4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-yl)-2-(dimethylamino)acetamide (<b>15g</b>)</h4><div class="NLM_p last">Brown solid, 0.42 g, yield; 83%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.99 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.65 (s, 1H, Ar-2-H), 8.59 (s, 1H, Ar-5-H), 8.25 (s, 1H, Ar-2′-H), 8.13–8.10 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.92–7.89 (d, 1H, <i>J</i> = 9 Hz, Ar-7-H), 7.80–7.77 (d, 1H, <i>J</i> = 9 Hz, Ar-6′-H), 7.64–7.59 (t, <i>J</i> = 6 Hz, 1H, Ar-5″-H), 7.48–7.46 (d, 1H, <i>J</i> = 6 Hz, Ar-4″-H), 7.34–7.32 (d, 1H, <i>J</i> = 6 Hz, Ar-6″-H), 7.25–7.21 (m, 2H, Ar-2″, 5′), 3.15 (s, 2H, CH<sub>2</sub>), 2.33 (s, 6H, 2CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.45 (1C, Ar-2-CH), 131.89 (1C, Ar-5″-CH), 128.82 (1C, Ar-8-CH), 127.66 (1C, Ar-7-CH), 123.97 (1C, Ar-6″-CH), 122.95 (1C, Ar-5′-CH), 122.77 (1C, Ar-6′-CH), 120.69 (1C, Ar-4″-CH), 119.94 (1C, Ar-2′-CH), 113.29 (1C, Ar-2″-CH), 112.44 (1C, Ar-5-CH), 63.68 (1C, CH<sub>2</sub>), 45.93 (2C, 2CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO): δ 169.41 (1C, C═O), 158.13 (1C, Ar-4-C), 157.51 (1C, Ar-1″-C), 153.44 (1C, Ar-2-CH), 147.02 (1C, Ar-6-C), 145.82 (1C, Ar-8a-C), 138.27 (1C, Ar-4′-C), 136.98 (1C, Ar-1′-C), 131.88 (1C, Ar-5″-CH), 131.40 (1C, Ar-3″-C), 130.99 (1C, Ar-3′-C), 128.82 (1C, Ar-8-CH), 127.64 (1C, Ar-7-CH), 125.41 (1C, CF<sub>3</sub>), 123.97 (1C, Ar-6″-CH), 122.92 (1C, Ar-5′-CH), 122.77 (1C, Ar-6′-CH), 120.69 (1C, Ar-4″-CH), 119.88 (1C, Ar-2′-CH), 115.82 (1C, Ar-4a-C), 113.28 (1C, Ar-2″-CH), 112.44 (1C, Ar-5-CH), 63.68 (1C, CH<sub>2</sub>), 45.93 (2C, 2CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>22</sub>ClF<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 516.1414; found, 516.1424.</div></div><div id="sec4_16_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77"> <i>N</i>-(4-((3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl)-2-morpholino Acetamide (<b>15h</b>)</h4><div class="NLM_p last">Off-white solid, 0.42 g, yield; 84%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.04 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.89 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.64 (s, 1H, Ar-2-H), 8.55 (s, 1H, Ar-5-H), 8.16 (s, 1H, Ar-2′-H), 8.03–8.00 (d, <i>J</i> = 8.7 Hz, 1H, Ar-8-H), 7.79–7.76 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.75 (s, 1H, Ar-2″-H), 7.32–7.30 (d, 1H, <i>J</i> = 7.1 Hz, Ar-6′-H), 7.21–7.19 (d, 1H, <i>J</i> = 7.3 Hz, Ar-5′-H), 7.09–7.04 (t, <i>J</i> = 8.8 Hz, 1H, Ar-5″-H), 6.99–6.96 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6″-H), 6.76–6.73 (d, <i>J</i> = 8.8 Hz, 1H, Ar-4″-H), 3.67 (br s, 4H, 2CH<sub>2</sub>, morpholino), 3.21 (s, 2H, CH<sub>2</sub>, acetamido), 2.56 (br s, 4H, 2CH<sub>2</sub>, morpholino), 2.26 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.66 (1C, Ar-2-CH), 131.95 (1C, Ar-8-CH), 128.78 (1C, Ar-5″-CH), 127.85 (2C, Ar-7, 4″-CH), 124.40 (1C, Ar-5′-CH), 124.12 (1C, Ar-6′-CH), 122.90 (1C, Ar-2′-CH), 120.22 (1C, Ar-2″-CH), 117.39 (1C, Ar-6″-CH), 112.78 (1C, Ar-5-CH), 66.57 (2C, 2CH<sub>2</sub>, morpholino), 62.38 (1C, CH<sub>2</sub>, acetamido), 53.70 (2C, 2CH<sub>2</sub>, morpholino), 16.19 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.92 (1C, C═O), 157.73 (1C, Ar-4-C), 155.01 (1C, Ar-1″-C), 153.66 (1C, Ar-2-CH), 148.08 (1C, Ar-6-C), 147.09 (1C, Ar-8a-C), 136.73 (1C, Ar-4′-C), 136.37 (1C, Ar-3″-C), 131.95 (1C, Ar-8-CH), 128.77 (1C, Ar-5″-CH), 128.26 (1C, Ar-1′-C), 127.84 (2C, Ar-7, 4″-CH), 124.41 (1C, Ar-5′-CH), 124.11 (1C, Ar-6′-CH), 123.79 (1C, Ar-3′-C), 122.90 (1C, Ar-2′-CH), 120.20 (1C, Ar-2″-CH), 117.41 (1C, Ar-6″-CH), 115.70 (1C, Ar-4a-C), 112.78 (1C, Ar-5-CH), 66.57 (2C, 2CH<sub>2</sub>, morpholino), 62.38 (1C, CH<sub>2</sub>, acetamido), 53.71 (2C, 2CH<sub>2</sub>, morpholino), 16.18 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>27</sub>ClN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 504.1802; found, 504.1799.</div></div><div id="sec4_16_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78"> <i>N</i>-(4-((3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl)-2-(4-methylpiperazin-1-yl)acetamide (<b>15i</b>)</h4><div class="NLM_p last">Light-brown solid, 0.47 g, yield; 92%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.94 (s, 1H, 1NH, exchanged with D<sub>2</sub>O), 9.87 (s, 1H, 1NH, exchanged with D<sub>2</sub>O), 8.62 (s, 1H, Ar-2-H), 8.56 (s, 1H, Ar-5-H), 8.17–8.16 (d (m-coupling), <i>J</i> = 2.1 Hz, 1H, Ar-2′-H), 8.04–8.01 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.79–7.76 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.78–7.74 (dd, <i>J</i> = 9 Hz, 2.1 Hz, 1H, Ar-6′-H), 7.34–7.31 (d, <i>J</i> = 7.3 Hz, 1H, Ar-6″-H), 7.22–7.17 (m, 1H, Ar-2″-H), 7.10–7.05 (t, <i>J</i> = 7.3 Hz, 1H, Ar-5″-H), 7.00–6.97 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 6.77–6.74 (d, <i>J</i> = 7.4 Hz, 1H, Ar-4″-H), 3.19 (s, 2H, CH<sub>2</sub>, acetamido), 2.57 (br s, 4H, 2-CH<sub>2</sub>, piperazino), 2.41 (br s, 4H, 2-CH<sub>2</sub>, piperazino), 2.26 (s, 3H, CH<sub>3</sub>, piperazino), 2.18 (s, 3H, CH<sub>3</sub>, tolyl); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.63 (1C, Ar-2-CH), 131.94 (1C, Ar-8-CH), 128.82 (1C, Ar-5″-CH), 127.82 (2C, Ar-7, 4″-CH), 124.39 (1C, Ar-5′-CH), 124.10 (1C, Ar-6′-CH), 122.89 (1C, Ar-2′-CH), 120.23 (1C, Ar-2″-CH), 117.39 (1C, Ar-6″-CH), 112.61 (1C, Ar-5-CH), 62.15 (1C, CH<sub>2</sub>, acetamido), 54.99 (2C, 2-CH<sub>2</sub>, piperazino), 53.29 (2C, 2-CH<sub>2</sub>, piperazino), 46.21 (1C, CH<sub>3</sub>, piperazino), 16.21 (1C, CH<sub>3</sub>, tolyl); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.05 (1C, C═O), 157.72 (1C, Ar-4-C), 155.03 (1C, Ar-1″-C), 153.63 (1C, Ar-2-CH), 148.06 (1C, Ar-6-C), 147.12 (1C, Ar-8a-C), 136.77 (1C, Ar-4′-C), 136.41 (1C, Ar-3″-C), 131.95 (1C, Ar-8-CH), 128.83 (1C, Ar-5″-CH), 128.27 (1C, Ar-1′-C), 127.84 (2C, Ar-7, 4″-CH), 124.40 (1C, Ar-5′-CH), 124.11 (1C, Ar-6′-CH), 123.78 (1C, Ar-3′-C), 122.90 (1C, Ar-2′-CH), 120.21 (1C, Ar-2″-CH), 117.42 (1C, Ar-6″-CH), 115.73 (1C, Ar-4a-C), 112.62 (1C, Ar-5-CH), 62.15 (1C, CH<sub>2</sub>, acetamido), 55.00 (2C, 2-CH<sub>2</sub>, piperazino), 53.29 (2C, 2-CH<sub>2</sub>, piperazino), 46.20 (1C, CH<sub>3</sub>, piperazino), 16.18 (1C, CH<sub>3</sub>, tolyl); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>30</sub>ClN<sub>6</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 517.2119; found, 517.2120.</div></div><div id="sec4_16_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79"> <i>N</i>-(4-((3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl)-2-(diethylamino) Acetamide (<b>15j</b>)</h4><div class="NLM_p last">Yellow solid, 0.37 g, yield; 77%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.92 (s, 1H, NH, exchanged with D<sub>2</sub>O), 9.86 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.57 (s, 1H, Ar-2-H), 8.56 (s, 1H, Ar-5-H), 8.20–8.17 (m, 2H, Ar-8, 2′-H), 7.80–7.77 (d, <i>J</i> = 9 Hz, 2H, Ar-7, 2″-H), 7.33–7.31 (d, 1H, <i>J</i> = 7.4, Ar-6′-H), 7.19–7.17 (d, 1H, <i>J</i> = 7.3, Ar-5′-H), 7.09–7.05 (t, <i>J</i> = 8.9 Hz, 1H, Ar-5″-H), 7.00–6.97 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6″-H), 6.77–6.74 (d, <i>J</i> = 8.8 Hz, 1H, Ar-4″-H), 3.23 (s, 2H, CH<sub>2</sub>, acetamido), 2.68–2.61 (q, <i>J</i> = 7.1 Hz, 4H, 2-CH<sub>2</sub>, (CH<sub>3</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub>2</sub>N−), 2.26 (s, 3H, CH<sub>3</sub>, tolyl), 1.08–1.04 (t, <i>J</i> = 7.1 Hz, 6H, 2-CH<sub>3</sub>, (<b>C<i>H</i></b><sub><b><i>3</i></b></sub>CH<sub>2</sub>)<sub>2</sub>N−); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.60 (1C, Ar-2-CH), 131.95 (1C, Ar-8-CH), 128.89 (1C, Ar-5″-CH), 127.84 (1C, Ar-7-CH), 127.32 (1C, Ar-4″-CH), 124.34 (1C, Ar-5′-CH), 124.10 (1C, Ar-6′-CH), 122.82 (1C, Ar-2′-CH), 120.24 (1C, Ar-2″-CH), 117.40 (1C, Ar-6″-CH), 112.16 (1C, Ar-5-CH), 57.82 (1C, CH<sub>2</sub>, acetamido), 48.50 (2C, 2-CH<sub>2</sub>, (CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub>)<sub>2</sub>N−) 16.20 (1C, CH<sub>3</sub>, tolyl), 12.44 (2C, 2-CH<sub>3</sub>, (<b><i>C</i>H</b><sub><b><i>3</i></b></sub>CH<sub>2</sub>)<sub>2</sub>N−); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.60 (1C, C═O), 157.64 (1C, Ar-4-C), 155.03 (1C, Ar-1″-C), 153.59 (1C, Ar-2-CH), 148.05 (1C, Ar-6-C), 147.01 (1C, Ar-8a-C), 136.74 (1C, Ar-4′-C), 136.41 (1C, Ar-3″-C), 131.94 (1C, Ar-8-CH), 128.88 (1C, Ar-5″-CH), 128.26 (1C, Ar-1′-C), 127.84 (1C, Ar-7-CH), 127.31 (1C, Ar-4″-CH), 124.34 (1C, Ar-5′-CH), 124.10 (1C, Ar-6′-CH), 123.80 (1C, Ar-3′-C), 122.83 (1C, Ar-2′-CH), 120.23 (1C, Ar-2″-CH), 117.41 (1C, Ar-6″-CH), 115.79 (1C, Ar-4a-C), 112.17 (1C, Ar-5-CH), 57.83 (1C, CH<sub>2</sub>, acetamido), 48.51 (2C, 2-CH<sub>2</sub>, (CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub>)<sub>2</sub>N−), 16.19 (1C, CH<sub>3</sub>, tolyl), 12.44 (2C, 2-CH<sub>3</sub>, (<b><i>C</i>H</b><sub><b><i>3</i></b></sub>CH<sub>2</sub>)<sub>2</sub>N); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>29</sub>ClN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 490.2010; found, 490.2015.</div></div><div id="sec4_16_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80"> <i>N</i>-(4-((3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl)-2-(dimethylamino) Acetamide (<b>15k</b>)</h4><div class="NLM_p last">Yellow solid, 0.22 g, yield; 84%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.96 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.62 (s, 1H, Ar-2-H), 8.55 (s, 1H, Ar-5-H), 8.17 (s, 1H, Ar-2′-H), 8.13–8.10 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8-H), 7.79–7.76 (d, <i>J</i> = 8.9 Hz, 2H, Ar-7, 6′-H), 7.33–7.30 (d, <i>J</i> = 7.1 Hz, 1H, Ar-6″-H), 7.21–7.16 (m, 1H, Ar-2″-H), 7.09–7.04 (t, <i>J</i> = 7.1 Hz, 1H, Ar-5″-H), 7.00–6.97 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 6.77–6.74 (d, <i>J</i> = 7.2 Hz, 1H, Ar-4″-H), 3.15 (s, 2H, CH<sub>2</sub>), 2.33 (s, 6H, 2CH<sub>3</sub>), 2.26 (s, 3H, CH<sub>3</sub>, tolyl); <sup>13</sup>C DEPT 135 (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.52 (1C, Ar-2-CH), 131.93 (1C, Ar-8-CH), 128.78 (1C, Ar-5″-CH), 127.83 (1C, Ar-7-CH), 127.56 (1C, Ar-4″-CH), 124.21 (1C, Ar-5′-CH), 124.08 (1C, Ar-6′-CH), 122.69 (1C, Ar-2′-CH), 120.24 (1C, Ar-2″-CH), 117.39 (1C, Ar-6″-CH), 112.49 (1C, Ar-5-CH), 63.68 (1C, CH<sub>2</sub>), 45.93 (2C, 2-CH<sub>3</sub>), 16.19 (1C, CH<sub>3</sub>, tolyl); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.39 (1C, C═O), 157.59 (1C, Ar-4-C), 155.05 (1C, Ar-1″-C), 153.52 (1C, Ar-2-CH), 148.00 (1C, Ar-6-C), 146.99 (1C, Ar-8a-C), 136.89 (1C, Ar-4′-C), 136.55 (1C, Ar-3″-C), 131.93 (1C, Ar-8-CH), 128.78 (1C, Ar-5″-CH), 128.26 (1C, Ar-1′-C), 127.82 (1C, Ar-7-CH), 127.56 (1C, Ar-4″-CH), 124.22 (1C, Ar-5′-CH), 124.08 (1C, Ar-6′-CH), 123.82 (1C, Ar-3′-C), 122.69 (1C, Ar-2′-CH), 120.23 (1C, Ar-2″-CH), 117.40 (1C, Ar-6″-CH), 115.80 (1C, Ar-4a-C), 112.49 (1C, Ar-5-CH), 63.69 (1C, CH<sub>2</sub>), 45.93 (2C, 2-CH<sub>3</sub>), 16.18 (1C, CH<sub>3</sub>, tolyl); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>25</sub>ClN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 462.1697; found, 462.1693.</div></div><div id="sec4_16_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81"> <i>N</i>-(4-((3-Chloro-4-(<i>m</i>-tolyloxy)phenyl)amino)quinazolin-6-yl)-2-((2-hydroxyethyl)amino)acetamide (<b>15l</b>)</h4><div class="NLM_p">A mixture of intermediate <b>14c</b> (0.45 g, 1 mmol) and 2-aminoethanol (0.25 g, 0.3 mL, 5 mmol) in methanol (20 mL) was held at reflux for 6 h in the presence of triethylamine (15 drops). Methanol was then removed under reduced pressure, and the remaining residue was recrystallized from acetone to yield compound <b>15l</b>.</div><div class="NLM_p last">White solid, 0.2 g, yield: 43%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.88 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 8.63 (s, 1H, Ar-2-H), 8.56 (s, 1H, Ar-5-H), 8.18 (s, 1H, Ar-2′-H), 8.09–8.05 (d, <i>J</i> = 9.1 Hz, 1H, Ar-8-H), 7.80–7.77 (dd, <i>J</i> = 9.1 Hz, 3.1 Hz, 2H, Ar-7, 6′-H), 7.33–7.31 (d, <i>J</i> = 6 Hz, 1H, Ar-6″-H), 7.19–7.17 (d, <i>J</i> = 6 Hz, 1H, Ar-4″-H), 7.10–7.05 (t, <i>J</i> = 6 Hz, 1H, Ar-5″), 7.00–6.97 (d, <i>J</i> = 9 Hz, 1H, Ar-5′-H), 6.76–6.74 (s, 1H, Ar-2″-H), 4.67 (s, 1H, OH, exchanged with D<sub>2</sub>O), 4.66 (s, 1H, 1NH, exchanged with D<sub>2</sub>O), 3.51 (br s, 2H, CH<sub>2</sub>, OH<b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>NH), 3.38 (s, 2H, CH<sub>2</sub>, acetamido), 2.68 (br s, 2H, CH<sub>2</sub>, OHCH<sub>2</sub><b>C<i>H</i></b><sub><b>2</b></sub>NH), 2.27 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.55 (1C, Ar-2-CH), 131.94 (1C, Ar-8-CH), 128.89 (1C, Ar-5″-CH), 127.85 (1C, Ar-7-CH), 127.36 (1C, Ar-4″-CH), 124.26 (1C, Ar-5′-CH), 124.09 (1C, Ar-6′-CH), 122.76 (1C, Ar-2′-CH), 120.26 (1C, Ar-2″-CH), 117.37 (1C, Ar-6″-CH), 112.11 (1C, Ar-5-CH), 60.85 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH), 53.17 (1C, CH<sub>2</sub>, acetamido), 52.18 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH), 16.20 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 171.18 (1C, C═O), 157.66 (1C, Ar-4-C), 155.04 (1C, Ar-1″-C), 153.55 (1C, Ar-2-CH), 147.97 (1C, Ar-6-C), 146.98 (1C, Ar-8a-C), 136.91 (1C, Ar-4′-C), 136.47 (1C, Ar-3″-C), 131.94 (1C, Ar-8-CH), 128.90 (1C, Ar-5″-CH), 128.24 (1C, Ar-1′-C), 127.84 (1C, Ar-7-CH), 127.37 (1C, Ar-4″-CH), 124.27 (1C, Ar-5′-CH), 124.09 (1C, Ar-6′-CH), 122.76 (1C, Ar-2′-CH), 120.26 (1C, Ar-2″-CH), 117.37 (1C, Ar-6″-CH), 115.80 (1C, Ar-4a-C), 112.11 (1C, Ar-5-CH), 60.81 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH), 53.14 (1C, CH<sub>2</sub>, acetamido), 52.15 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH), 16.21 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>25</sub>ClN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 478.1646; found, 478.1642.</div></div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i82" class="anchor-spacer"></div><h3 class="article-section__title" id="_i82"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-4-(chloromethyl)benzamide (<b>16</b>)</h3><div class="NLM_p">To the solution of intermediate <b>13a</b> (1.5 g, 4 mmol) in acetone (20 mL), NaHCO<sub>3</sub> (0.35 g, 4.2 mmol) was added. The reaction mixture was stirred under nitrogen at 0 °C, followed by the dropwise addition of a solution of 4(choromethyl) benzoyl chloride (0.8 g, 4.2 mmol) in acetone (2 mL). The reaction mixture was stirred for 30 min at 0 °C. Excess solvent was removed under reduced pressure. Cold water (20 mL) was added to the remaining residue and the resulting suspension was neutralized with a saturated solution of NaHCO<sub>3</sub>, if necessary. The solid obtained was filtered, dried, and washed with DCM and diethylether to afford compound <b>16</b> as pure greenish-yellow solid. 1.1 g, yield: 51%.</div><div class="NLM_p last"><sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.81 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 9.05 (s, 1H, Ar-2′-H), 8.74 (s, 1H, Ar-5′-H), 8.18 (s, 1H, Ar-2‴-H), 8.15–8.12 (d, <i>J</i> = 9 Hz, 1H, Ar-8′-H), 8.10–8.07 (d, <i>J</i> = 7.9 Hz, 2H, Ar-2,6-H), 7.92–7.89 (d, <i>J</i> = 9 Hz, 1H, Ar-7′-H), 7.86–7.83 (d, <i>J</i> = 9 Hz, 1H, Ar-6″-H), 7.65–7.62 (d, <i>J</i> = 7.8 Hz, 2H, Ar-3,5-H), 7.46–7.38 (m, 1H, Ar-2″-H), 7.32–7.29 (d, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 7.00–6.94 (t, <i>J</i> = 8.4 Hz, 1H, Ar-5‴-H), 6.88–6.85 (d, <i>J</i> = 8.4, 1H, Ar-4‴-H), 6.80–6.77 (d, <i>J</i> = 8.3 Hz, 1H, Ar-6‴-H), 4.87 (s, 2H, CH<sub>2</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.37 (1C, Ar-2′-CH), 131.92 (1C, Ar-5‴-CH), 129.76 (1C, Ar-8′-CH), 129.37 (2C, Ar-2, 6-CH), 128.63 (2C, Ar-3,5-CH), 125.92 (1C, Ar-7′-CH), 125.20 (1C, Ar-5″-CH), 123.81 (1C, Ar-6″-CH), 122.54 (1C, Ar-2″-CH), 114.35 (1C, Ar-6‴-CH), 113.07 (1C, Ar-4‴-CH), 110.24 (1C, Ar-2‴-CH), 104.78 (1C, Ar-5′-CH), 45.83 (1C, CH<sub>2</sub>). MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 533.2 [M + 1]<sup>+</sup>.</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i83" class="anchor-spacer"></div><h3 class="article-section__title" id="_i83"> General Procedure for the Synthesis of Compounds (<b>17a–c</b>)</h3><div class="NLM_p">A mixture of intermediate <b>16</b> (0.53 g, 1 mmol) and the appropriate secondary amine, 1-methyl piperazine or morpholine (50 mmol), was placed into a 25 mL round-bottomed flask or into a capped vial in the case of low boiling point secondary amine, dimethylamine (100 mmol). The reaction mixture was heated under reflux and monitored by TLC. After 2 h, the TLC of the reaction mass indicated the absence of intermediate <b>16</b>. The reaction mixture was cooled to room temperature and poured onto iced-water (30 mL) portion wise with stirring. The resultant precipitate was filtered, washed with water, and dried. The obtained product was purified by flash column chromatography using hexane/ethylacetate (90:10%) as an eluent to afford the desired products (<b>17a–c</b>).</div><div id="sec4_18_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i84" class="anchor-spacer"></div><h4 class="article-section__title" id="_i84"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-4-(morpholinomethyl)benzamide (<b>17a</b>)</h4><div class="NLM_p last">Light-yellow solid, 0.27 g, yield; 48%, mp 134–137 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.59 (s, 1H, NH, exchanged with D<sub>2</sub>O), 10.01 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.94 (s, 1H, Ar-2′-H), 8.62 (s, 1H, Ar-5′-H), 8.26 (s, 1H, Ar-2‴-H), 8.03–8.01 (ds, <i>J</i> = 7.5 Hz, 3H, Ar-2″,2,6-H), 7.93–7.90 (d, <i>J</i> = 9 Hz, 1H, Ar-8′-H), 7.85–7.82 (d, <i>J</i> = 9 Hz, 1H, Ar-7′-H), 7.51–7.49 (d, <i>J</i> = 7.8 Hz, 2H, Ar-3,5-H), 7.44–7.36 (dd, <i>J</i> = 8.1 Hz, 1.2 Hz, 1H, Ar-6″-H), 7.30–7.27 (d, <i>J</i> = 9 Hz, 1H, Ar-4‴-H), 6.97–6.92 (t, <i>J</i> = 9 Hz, 1H, Ar-5‴-H), 6.85–6.81 (d, <i>J</i> = 8.4 Hz, 1H, Ar-5″-H), 6.78–6.75 (d, <i>J</i> = 9 Hz, 1H, Ar-6‴-H), 3.59 (br s, 4H, 2CH<sub>2</sub>, morpholino), 3.55 (s, 2H, CH<sub>2</sub>, −<b>C<i>H</i></b><sub><b><i>2</i></b></sub>C<sub>6</sub>H<sub>4</sub>), 2.38 (br s, 4H, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.67 (1C, Ar-2′-CH), 131.84 (1C, Ar-5‴-CH), 129.34 (2C, Ar-2, 6-CH), 128.76 (1C, Ar-8′-CH), 128.15 (2C, Ar-3,5-CH), 123.99 (1C, Ar-7′-CH), 122.74 (1C, Ar-5″-CH), 113.91 (1C, Ar-6″-CH), 112.76 (1C, Ar-2″-CH), 110.26 (1C, Ar-6‴-CH), 109.98 (1C, Ar-4‴-CH), 104.80 (1C, Ar-2‴-CH), 104.47 (1C, Ar-5′-CH), 66.65 (2C, 2CH<sub>2</sub>, morpholino), 62.43 (1C, CH<sub>2</sub>, −<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>), 53.66 (2C, 2CH<sub>2</sub>, morpholino); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.93 (1C, Ar-4′-C), 165.01 (1C, C═O), 161.77 (1C, Ar-3‴-C), 159.15 (1C, Ar-1‴-C), 159.00 (1C, Ar-4″-C), 157.73 (1C, Ar-8′a-C), 153.68 (1C, Ar-2′-CH), 147.34 (1C, Ar-4-C), 146.12 (1C, Ar-6′-C), 142.57 (1C, Ar-1″-C), 138.02 (1C, Ar-1-C), 137.36 (1C, Ar-4′a-C), 131.84 (1C, Ar-5‴-CH), 129.34 (2C, Ar-2, 6-CH), 128.84 (1C, Ar-8′-CH), 128.14 (2C, Ar-3,5-CH), 125.29 (1C, Ar-3″-C), 124.00 (1C, Ar-7′-CH), 122.74 (1C, Ar-5″-CH), 113.91 (1C, Ar-6″-CH), 112.76 (1C, Ar-2″-CH), 110.26 (1C, Ar-6‴-CH), 109.98 (1C, Ar-4‴-CH), 104.81 (1C, Ar-2‴-CH), 104.47 (1C, Ar-5′-CH), 66.65 (2C, 2CH<sub>2</sub>, morpholino), 62.43 (1C, CH<sub>2</sub>, −<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>), 53.66 (2C, 2CH<sub>2</sub>, morpholino); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>32</sub>H<sub>28</sub>ClFN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 584.1865; found, 584.1869.</div></div><div id="sec4_18_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i85" class="anchor-spacer"></div><h4 class="article-section__title" id="_i85"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-4-((4-methylpip erazin-1-yl)methyl)benzamide (<b>17b</b>)</h4><div class="NLM_p last">Brown solid, 0.39 g, yield; 67%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.58 (s, 1H, NH, exchanged with D<sub>2</sub>O), 10.00 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.93 (s, 1H, Ar-2′-H), 8.61 (s, 1H, Ar-5′-H), 8.25 (s, 1H, Ar-2‴-H), 8.02–8.00 (ds, <i>J</i> = 7.1 Hz, 3H, Ar-2″,2,6-H), 7.92–7.90 (d, <i>J</i> = 8.4 Hz, 1H, Ar-8′-H), 7.84–7.81 (d, <i>J</i> = 8.5 Hz, 1H, Ar-7′-H), 7.48–7.46 (d, <i>J</i> = 7.0 Hz, 2H, Ar-3,5-H), 7.41–7.36 (m, 1H, Ar-6″-H), 7.29–7.26 (d, <i>J</i> = 7.6 Hz, 1H, Ar-4‴-H), 6.97–6.92 (t, <i>J</i> = 7.7 Hz, 1H, Ar-5‴-H), 6.84–6.80 (d, <i>J</i> = 10.4 Hz, 1H, Ar-5″-H), 6.77–6.75 (d, <i>J</i> = 7.7 Hz, 1H, Ar-6‴-H), 3.54 (s, 2H, CH<sub>2</sub>, −<b>C<i>H</i></b><sub><b><i>2</i></b></sub>C<sub>6</sub>H<sub>4</sub>), 2.38 (br s, 8H, 4-CH<sub>2</sub>, piperazino), 2.17 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.66 (1C, Ar-2′-CH), 131.84 (1C, Ar-5‴-CH), 129.22 (2C, Ar-2, 6-CH), 128.73 (1C, Ar-8′-CH), 128.13 (2C, Ar-3,5-CH), 124.00 (1C, Ar-7′-CH), 122.74 (1C, Ar-5″-CH), 113.93 (1C, Ar-6″-CH), 112.79 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 109.99 (1C, Ar-4‴-CH), 104.81 (1C, Ar-2‴-CH), 104.47 (1C, Ar-5′-CH), 62.00 (1C, CH<sub>2</sub>, −<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>), 55.06 (2C, 2CH<sub>2</sub>, piperazino), 52.89 (2C, 2CH<sub>2</sub>, piperazino), 46.01 (1C, CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.96 (1C, Ar-4′-C), 165.01 (1C, C═O), 161.77 (1C, Ar-3‴-C), 159.14 (1C, Ar-1‴-C), 159.00 (1C, Ar-4″-C), 157.74 (1C, Ar-8′a-C), 153.65 (1C, Ar-2′-CH), 147.32 (1C, Ar-4-C), 146.15 (1C, Ar-6′-C), 142.99 (1C, Ar-1″-C), 138.02 (1C, Ar-1-C), 137.39 (1C, Ar-4′a-C), 131.71 (1C, Ar-5‴-CH), 129.22 (2C, Ar-2, 6-CH), 128.72 (1C, Ar-8′-CH), 128.12 (2C, Ar-3,5-CH), 125.29 (1C, Ar-3″-C), 124.01 (1C, Ar-7′-CH), 122.73 (1C, Ar-5″-CH), 113.93 (1C, Ar-6″-CH), 112.79 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 109.99 (1C, Ar-4‴-CH), 104.81 (1C, Ar-2‴-CH), 104.48 (1C, Ar-5′-CH), 62.01 (1C, CH<sub>2</sub>, −<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>), 55.06 (2C, 2CH<sub>2</sub>, piperazino), 52.89 (2C, 2CH<sub>2</sub>, piperazino), 46.01 (1C, CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>33</sub>H<sub>31</sub>ClFN<sub>6</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 597.2181; found, 597.2184.</div></div><div id="sec4_18_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i86" class="anchor-spacer"></div><h4 class="article-section__title" id="_i86"> <i>N</i>-(4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl)-4-((dimethyl amino)methyl) Benzamide (<b>17c</b>)</h4><div class="NLM_p last">Yellow solid, 0.42 g, yield; 78%, mp 135–138 °C (dec); <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.60 (s, 1H, NH, exchanged with D<sub>2</sub>O), 10.01 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.94 (s, 1H, Ar-2′-H), 8.62 (s, 1H, Ar-5′-H), 8.25 (s, 1H, Ar-2‴-H), 8.03–8.00 (ds, <i>J</i> = 7.4 Hz, 3H, Ar-2″,2,6H), 7.93–7.90 (d, <i>J</i> = 8.8 Hz, 1H, Ar-8′-H), 7.85–7.82 (d, <i>J</i> = 8.9 Hz, 1H, Ar-7′-H), 7.50–7.47 (d, <i>J</i> = 7.6 Hz, 2H, Ar-3,5-H), 7.42–7.39 (d, <i>J</i> = 9 Hz, 1H, Ar-6″-H), 7.31–7.28 (d, <i>J</i> = 8.5 Hz, 1H, Ar-4‴-H), 6.98–6.92 (t, <i>J</i> = 8.4 Hz, 1H, Ar-5‴-H), 6.85–6.82 (d, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 6.78–6.75 (d, <i>J</i> = 8.4 Hz, 1H, Ar-6‴-H), 3.48 (s, 2H, CH<sub>2</sub>), 2.17 (s, 6H, 2CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.68 (1C, Ar-2′-CH), 131.86 (1C, Ar-5‴-CH), 129.17 (2C, Ar-2, 6-CH), 128.74 (1C, Ar-8′-CH), 128.12 (2C, Ar-3,5-CH), 124.01 (1C, Ar-7′-CH), 122.77 (1C, Ar-5″-CH), 113.91 (1C, Ar-6″-CH), 112.77 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 109.99 (1C, Ar-4‴-CH), 104.79 (1C, Ar-2‴-CH), 104.46 (1C, Ar-5′-CH), 63.42 (1C, CH<sub>2</sub>), 45.48 (2C, 2CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.97 (1C, Ar-4′-C), 165.00 (1C, C═O), 161.76 (1C, Ar-3‴-C), 159.14 (1C, Ar-1‴-C), 158.99 (1C, Ar-4″-C), 157.73 (1C, Ar-8′a-C), 153.68 (1C, Ar-2′-CH), 147.30 (1C, Ar-4-C), 146.13 (1C, Ar-6′-C), 143.65 (1C, Ar-1″-C), 138.00 (1C, Ar-1-C), 137.37 (1C, Ar-4′a-C), 131.86 (1C, Ar-5‴-CH), 129.17 (2C, Ar-2, 6-CH), 128.74 (1C, Ar-8′-CH), 128.11 (2C, Ar-3,5-CH), 125.30 (1C, Ar-3″-C), 124.01 (1C, Ar-7′-CH), 122.77 (1C, Ar-5″-CH), 113.92 (1C, Ar-6″-CH), 112.77 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 110.00 (1C, Ar-4‴-CH), 104.80 (1C, Ar-2‴-CH), 104.46 (1C, Ar-5′-CH), 63.43 (1C, CH<sub>2</sub>), 45.48 (2C, 2CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>26</sub>ClFN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 542.1759; found, 542.1755.</div></div><div id="sec4_18_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i87" class="anchor-spacer"></div><h4 class="article-section__title" id="_i87"> <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2-((2-hydroxyethyl)amino) Acetamide (<b>17d</b>)</h4><div class="NLM_p">A mixture of intermediate <b>17</b> (0.53 g, 1 mmol) and 2-aminoethanol (0.25 g, 0.3 mL, 5 mmol) in methanol (20 mL) was held at reflux for 4 h in the presence of triethylamine (15 drops). Methanol was removed under reduced pressure, and the remaining residue was dissolved in DCM and purified by flash column chromatography with ethylacetate/MeOH (90:10%) as an eluent to afford compound <b>17d</b>.</div><div class="NLM_p last">Brown solid, 0.37 g, yield: 68%; <sup>1</sup>H NMR (300 MHz, DMSO): δ 10.59 (s, 2H, 2NH, exchanged with D<sub>2</sub>O), 10.03 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.95 (s, 1H, Ar-2′-H), 8.61 (s, 1H, Ar-5′-H), 8.26 (s, 1H, Ar-2‴-H), 8.06 (s, 1H, Ar-2″-H), 8.03–8.00 (d, <i>J</i> = 8 Hz, 2H, Ar-2,6-H), 7.93–7.90 (d, <i>J</i> = 8.4 Hz, 1H Ar-4‴-H), 7.85–7.82 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8′-H), 7.54–7.51 (d, <i>J</i> = 7.9 Hz, 2H, Ar-3,5-H), 7.45–7.37 (m, 1H, Ar-6″-H), 7.30–7.28 (d, <i>J</i> = 8.9 Hz, 1H, Ar-7′-H), 6.98–6.92 (t, <i>J</i> = 8.4 Hz,, 1H, Ar-5‴), 6.85–6.82 (d, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 6.78–6.75 (d, <i>J</i> = 8.3 Hz, 1H, Ar-6‴-H), 4.50 (s, 1H, OH, exchanged with D<sub>2</sub>O), 3.81 (s, 2H, CH<sub>2</sub>, −NH<b>C<i>H</i></b><sub><b><i>2</i></b></sub>C<sub>6</sub>H<sub>4</sub>), 3.51–3.48 (t, <i>J</i> = 5.6 Hz, 2H, CH<sub>2</sub>, OH<b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>NH−), 2.61–2.58 (t, <i>J</i> = 5.7 Hz, 2H, CH<sub>2</sub>, OHCH<sub>2</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>NH−); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 153.65 (1C, Ar-2′-CH), 131.86 (1C, Ar-5‴-CH), 128.85 (1C, Ar-8′-CH), 128.32 (2C, Ar-2,6-CH), 128.07 (2C, Ar-3,5-CH), 123.98 (1C, Ar-7′-CH), 122.72 (1C, Ar-5″-CH), 113.97 (1C, Ar-6″-CH), 112.77 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 109.99 (1C, Ar-4‴-CH), 104.81 (1C, Ar-2‴-CH), 104.48 (1C, Ar-5′-CH), 60.87 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH−), 52.97 (1C, CH<sub>2</sub>, NH<b><i>C</i>H</b><sub><b>2</b></sub>C6H4), 51.57 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH−); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.96 (1C, Ar-4′-C), 165.01 (1C, C═O), 161.77 (1C, Ar-3‴-C), 159.15 (1C, Ar-1‴-C), 159.01 (1C, Ar-4″-C), 157.69 (1C, Ar-8′a-C), 153.64 (1C, Ar-2′-CH), 147.27 (1C, Ar-4-C), 146.10 (1C, Ar-6′-C), 145.82 (1C, Ar-1″-C), 137.42 (1C, Ar-1-C), 133.07 (1C, Ar-4′a-C), 131.85 (1C, Ar-5‴-CH), 128.87 (1C, Ar-8′-CH), 128.32 (2C, Ar-2, 6-CH), 128.06 (2C, Ar-3,5-CH), 125.30 (1C, Ar-3″-C), 123.98 (1C, Ar-7′-CH), 122.72 (1C, Ar-5″-CH), 113.98 (1C, Ar-6″-CH), 112.80 (1C, Ar-2″-CH), 110.27 (1C, Ar-6‴-CH), 109.99 (1C, Ar-4‴-CH), 104.81 (1C, Ar-2‴-CH),,104.48 (1C, Ar-5′-CH), 60.87 (1C, CH<sub>2</sub>, OH<b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>NH−), 52.97 (1C, CH<sub>2</sub>, NH<b><i>C</i>H</b><sub><b>2</b></sub>C6H<sub>4</sub>), 51.57 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>NH−). HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>30</sub>H<sub>26</sub>ClFN<sub>5</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 558.1708; found, 558.1703.</div></div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i88" class="anchor-spacer"></div><h3 class="article-section__title" id="_i88"> Quinazoline-4,6-diol (<b>19</b>)</h3><div class="NLM_p">2-Amino-5-hydroxybenzoic acid (<b>18</b>; 2.00 g, 13.1 mmol) and formamide (9.37 mL, 235 mmol) were mixed and stirred for 1 h at 190 °C, a precipitate was formed while hot, and then, the reaction mixture was cooled to room temperature. The solid was filtered, triturated with H<sub>2</sub>O, and washed with diethyl ether to afford quinazoline-4,6-diol (<b>19</b>).</div><div class="NLM_p last">Brown solid, mp >300 (dec),<a onclick="showRef(event, 'ref109 ref115'); return false;" href="javascript:void(0);" class="ref ref109 ref115">(109,115)</a> 1.5 g, yield: 71%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.02 (s, 1H, OH, exchanged with D<sub>2</sub>O), 10.05 (s, 1H, OH, exchanged with D<sub>2</sub>O), 7.90 (s, 1H, Ar-2-H), 7.54–7.51 (d, <i>J</i> = 8.8 Hz, 1H, Ar-8-H), 7.41–7.40 (d (m-coupling), <i>J</i> = 2.3 Hz, 1H, Ar-5-H), 7.27–7.23 (dd, <i>J</i> = 8.7, 2.5 Hz, 1H, Ar-7-H).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i89" class="anchor-spacer"></div><h3 class="article-section__title" id="_i89"> 4-Hydroxyquinazolin-6-yl Acetate (<b>20</b>)</h3><div class="NLM_p">A mixture of (<b>19</b>) (2.00 g, 12.3 mmol) and pyridine (2.5 mL) in acetic anhydride (18 mL) was stirred at 100 °C for 2 h. The reaction mixture was poured into ice-cold water (100 mL). The formed precipitate was filtered, washed with water, and dried to obtain the product (<b>20</b>).<a onclick="showRef(event, 'ref109 ref115'); return false;" href="javascript:void(0);" class="ref ref109 ref115">(109,115)</a></div><div class="NLM_p last">Off-white solid, 1.5 g, yield: 60%, mp >300 °C; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.34 (s, 1H, OH, exchanged with D<sub>2</sub>O), 8.10 (s, 1H, Ar-2-H), 7.83–7.82 (d (m-coupling), <i>J</i> = 1.8 Hz, 1H, Ar-5-H), 7.74–7.71 (d, <i>J</i> = 8.8 Hz, 1H, Ar-8-H), 7.60–7.57(dd, <i>J</i> = 8.8, 1.8 Hz, 1H, Ar-7-H), 2.31 (s, 3H, CH<sub>3</sub>).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i90" class="anchor-spacer"></div><h3 class="article-section__title" id="_i90"> General Procedure for the Synthesis of Compounds (<b>22a–c</b>)</h3><div class="NLM_p">A mixture of (<b>20</b>) (2.00 g, 9.8 mmol), SOCl<sub>2</sub> (15 mL), and DMF (0.5 mL) was heated under reflux (95 °C) for 5 h. Excess SOCl<sub>2</sub> was removed under reduced pressure. The residue was treated with toluene (10 mL) and the solvent was evaporated under reduced pressure to yield 4-chloroquinazolin-6-yl acetate (<b>21</b>) in a pure form. Immediately, to a mixture of intermediate <b>21</b> and the appropriate aniline derivative (<b>4a–c</b>) (9.8 mmol) dissolved in acetonitrile (25 mL), K<sub>2</sub>CO<sub>3</sub> (1.4 g, 9.8 mmol) was added. For compound <b>22a</b>, the reaction mixture was stirred at room temperature under a nitrogen atmosphere for 8 h, and the formed precipitate was filtered and washed with acetonitrile, water, and ether in order and dried to provide product (<b>22a</b>) in a pure form. For compounds <b>22b</b>,<b>c</b>, the reaction mixture was refluxed under a nitrogen atmosphere for 12 h, cooled to room temperature, and filtered. The filtrate was evaporated under reduced pressure, and the produced oily mass was dissolved in DCM and purified by flash column chromatography using hexane/ethyl acetate (80:20%) as an eluent to afford compound <b>22b</b> or using DCM/methanol (90:10%) to afford compound <b>22c</b> in a pure form.</div><div id="sec4_21_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i91" class="anchor-spacer"></div><h4 class="article-section__title" id="_i91"> 4-(3-Chloro-4-(3-fluorobenzyloxy)phenylamino)quinazolin-6-yl Acetate (<b>22a</b>)</h4><div class="NLM_p last">Yellow solid, 3.8 g, yield: 90%, mp 290–292 °C, as reported.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a></div></div><div id="sec4_21_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i92" class="anchor-spacer"></div><h4 class="article-section__title" id="_i92"> 4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-yl Acetate (<b>22b</b>)</h4><div class="NLM_p last">Brown solid, 3.1 g, yield: 76%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.88 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.68 (s, 1H, Ar-2-H), 8.35–8.36 (d (m-coupling), <i>J</i> = 2.2 Hz, 1H, Ar-5-H), 8.30–8.29 (d (m-coupling), <i>J</i> = 2.5 Hz, 1H, Ar-2′-H), 7.93–7.90 (dd, <i>J</i> = 8.9, 2.5 Hz, 1H, Ar-6′-H), 7.89–7.86 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.72–7.69 (dd, <i>J</i> = 9.0, 2.2 Hz, 1H, Ar-7-H), 7.44–7.36 (m, 1H, Ar-2″-H), 7.31–7.28 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 6.97–6.92 (t, <i>J</i> = 8.3 Hz, 1H, Ar-5″-H), 6.85–6.81 (dd, <i>J</i> = 8.3, 2.1 Hz, 1H, Ar-4″-H), 6.77–6.74 (dd, <i>J</i> = 8.3 Hz, 1.9 Hz, 1H, Ar-6″-H), 2.38 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.59 (1C, Ar-2-CH), 131.85 (1C, Ar-8-CH), 129.88 (1C, Ar-7-CH), 129.06 (1C, Ar-5″-CH), 123.83 (1C, Ar-5′-CH), 122.80 (1C, Ar-6′-CH), 122.50 (1C, Ar-2′-CH), 115.10 (1C, Ar-6″-CH), 112.85 (1C, Ar-5-CH), 110.32 (1C, Ar-4″-CH), 104.87 (1C, Ar-2″-CH), 21.22 (1C, CH<sub>3</sub>); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 424.1 [M + 1]<sup>+</sup>.</div></div><div id="sec4_21_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i93" class="anchor-spacer"></div><h4 class="article-section__title" id="_i93"> 4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-ylacetate (<b>22c</b>)</h4><div class="NLM_p last">Yellow solid, 2.7 g, yield: 60%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.89 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.69 (s, 1H, Ar-2-H), 8.35–8.32 (d, <i>J</i> = 8.3 Hz, 2H, Ar-5,8-H), 7.95–7.92 (d, <i>J</i> = 8.5 Hz, 1H, Ar-7-H), 7.89–7.86 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6′-H), 7.72–7.69 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 7.64–7.59 (t, <i>J</i> = 7.9 Hz, 1H, Ar-5″-H), 7.48–7.46 (d, <i>J</i> = 7.8 Hz, 1H, Ar-4″-H), 7.35–7.32 (d, <i>J</i> = 8 Hz, 1H, Ar-6″-H), 7.25–7.2 (m, 2H, Ar-2′,2″-H), 2.38 (s, 3H, CH<sub>3</sub>); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 154.57 (1C, Ar-2-CH), 131.89 (1C, Ar-8-CH), 129.87 (1C, Ar-7-CH), 129.07 (1C, Ar-5″-CH), 123.86 (1C, Ar-6″-CH), 122.97 (1C, Ar-5′-CH), 122.59 (1C, Ar-6′-CH), 120.71 (1C, Ar-4″-CH), 119.93 (1C, Ar-2′-CH), 115.10 (1C, Ar-2″-CH), 113.35 (1C, Ar-5-CH), 21.2 (1C, CH<sub>3</sub>); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 474.1 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i94" class="anchor-spacer"></div><h3 class="article-section__title" id="_i94"> General Procedure for the Synthesis of Compounds (<b>23a–c</b>)</h3><div class="NLM_p">A mixture of compound <b>22a</b>, <b>22b</b>, or <b>22c</b> (5 mmol) and ammonium hydroxide (20 mL) in methanol (60 mL) was stirred at room temperature for 3–12 h. The clear solution formed was evaporated under reduced pressure, and the residue was diluted with water (30 mL) and stirred for 30 min. The resulting solid was filtered, washed with water, and dried at 50 °C to afford compounds <b>23a–c</b>.</div><div id="sec4_22_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i95" class="anchor-spacer"></div><h4 class="article-section__title" id="_i95"> 4-(3-Chloro-4-(3-fluorobenzyloxy)phenylamino)quinazolin-6-ol (<b>23a</b>)</h4><div class="NLM_p last">Reaction time: 12 h, yellow solid, 1.9 g, yield 71%; mp >300 °C, as reported.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a></div></div><div id="sec4_22_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i96" class="anchor-spacer"></div><h4 class="article-section__title" id="_i96"> 4-((3-Chloro-4-(3-fluorophenoxy)phenyl)amino)quinazolin-6-ol (<b>23b</b>)</h4><div class="NLM_p last">Reaction time: 4 h, yellow solid, 1.75 g, yield: 92%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.14 (s, 1H, OH, exchanged with D<sub>2</sub>O), 9.65 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.5 (s, 1H, Ar-2-H), 8.31 (s, 1H, Ar-2″-H), 7.95–7.93 (d, <i>J</i> = 8.2 Hz, 1H, Ar-8-H), 7.80 (s, 1H, Ar-2′-H), 7.72–7.69 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6′-H), 7.47–7.43 (m, 1H, Ar-6″-H), 7.41–7.38 (d, <i>J</i> = 7.5 Hz, 1H, Ar-4″-H), 7.29–7.26 (d, <i>J</i> = 8.8 Hz, 1H, Ar-5′-H), 6.96–6.91 (t, <i>J</i> = 7.7 Hz, 1H, Ar-5″-H), 6.82–6.81 (d (m-coupling), <i>J</i> = 2.3 Hz, 1H, Ar-5-H), 6.76–6.73 (d, <i>J</i> = 8.3 Hz, 1H, Ar-7-H); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.74 (1C, Ar-2-CH), 131.69 (1C, Ar-8-CH), 129.87 (1C, Ar-5″-CH), 124.81 (1C, Ar-7-CH), 123.49 (1C, Ar-5′-CH), 122.86 (1C, Ar-6′-CH), 122.28 (1C, Ar-2′-CH), 112.68 (1C, Ar-6″-CH), 110.19 (1C, Ar-4″-CH), 105.43 (1C, Ar-2″-CH), 104.71 (1C, Ar-5-CH); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 382.1 [M + 1]<sup>+</sup>.</div></div><div id="sec4_22_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i97" class="anchor-spacer"></div><h4 class="article-section__title" id="_i97"> 4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-ol (<b>23c</b>)</h4><div class="NLM_p last">Reaction time: 3 h, yellowish white solid, 1.89 g, yield: 88%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 10.14 (s, 1H, OH, exchanged with D<sub>2</sub>O), 9.67 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.52 (s, 1H, Ar-2-H), 8.34–8.33 (d (m-coupling), <i>J</i> = 1.8 Hz, 1H, Ar-2′-H), 7.98–7.95 (dd, <i>J</i> = 8.8 Hz, 1.9 Hz, 1H, Ar-6′-H), 7.80 (s, 1H, Ar-5-H), 7.73–7.70 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8-H), 7.64–7.59 (t, <i>J</i> = 7.9 Hz, 1H, Ar-5″-H), 7.48–7.45 (d, <i>J</i> = 8.8 Hz, 2H, Ar-7, 5′-H), 7.34–7.32 (d, <i>J</i> = 7.9 Hz, 1H, Ar-4″-H), 7.24–7.20 (m, 2H, Ar-6″, 2″-H); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 151.72 (1C, Ar-2-CH), 131.88 (1C, Ar-8-CH), 129.88 (1C, Ar-5″-CH), 124.83 (1C, Ar-7-CH), 123.04 (1C, Ar-6″-CH), 122.37 (1C, Ar-5′-CH), 120.60 (1C, Ar-6′-CH), 119.89 (1C, Ar-4″-CH), 119.84 (1C, Ar-2′-CH), 113.18 (1C, Ar-2″-CH), 105.42 (1C, Ar-5-CH); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 432.1 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i98" class="anchor-spacer"></div><h3 class="article-section__title" id="_i98"> General Procedure for the Synthesis of Compounds (<b>24a–c</b>)</h3><div class="NLM_p">A mixture of compound <b>24a</b>, <b>24b</b>, or <b>24c</b> (5 mmol), K<sub>2</sub>CO<sub>3</sub> (1.4 g, 10 mmol), and catalytic amount of TBAB (81 mg, 0.25 mmol, 0.05 equiv) in acetonitrile (30 mL) was stirred at room temperature for 30 min, and then, 1-bromo-3-chloropropane (3.14 g, 2mL, 20 mmol) was added. The reaction mixture was refluxed at 85 °C for 2 h and then cooled to room temperature and filtered. The filtrate was evaporated under reduced pressure, and the residue obtained was dissolved in ethyl acetate and loaded on silica gel 60 (230–450 mesh) to be purified by flash column chromatography (DCM/methanol = 97:3%) to afford pure products (<b>24a–c</b>).</div><div id="sec4_23_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i99" class="anchor-spacer"></div><h4 class="article-section__title" id="_i99"> <i>N</i>-(3-Chloro-4-(3-fluorobenzyloxy)phenyl)-6-(3-chloropropoxy)quinazolin-4-amine (<b>24a</b>)</h4><div class="NLM_p last">Yellow solid, 0.9 g, yield: 42%; mp 144–146 °C, as reported.<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a></div></div><div id="sec4_23_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i100" class="anchor-spacer"></div><h4 class="article-section__title" id="_i100"> <i>N</i>-(3-Chloro-4-(3-fluorophenoxy)phenyl)-6-(3-chloropropoxy)quinazolin-4-amine (<b>24b</b>)</h4><div class="NLM_p last">Yellowish white solid, 2 g, yield: 88%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.73 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.57 (s, 1H, Ar-2-H), 8.24 (d (m-coupling), <i>J</i> = 2.3 Hz, 1H, Ar-5-H), 7.95–7.94 (d (m-coupling), <i>J</i> = 2.1 Hz, 1H, Ar-2′-H), 7.93–7.89 (dd, <i>J</i> = 9.0, 2.3 Hz, 1H, Ar-7-H), 7.78–7.75 (d, <i>J</i> = 9.0 Hz, 1H, Ar-8-H), 7.57–7.53 (dd, <i>J</i> = 9.1, 2.1 Hz, 1H, Ar-6′-H), 7.45–7.37 (m, 1H, Ar-2″-H), 7.32–7.29 (d, <i>J</i> = 9 Hz, 1H, Ar-5′-H), 6.98–6.93 (t, <i>J</i> = 8.4 Hz, 1H, Ar-5″-H), 6.85–6.82 (d, <i>J</i> = 8.4 Hz, 1H, Ar-4″-H), 6.78–6.75 (dd, <i>J</i> = 8.4, 1.7 Hz, 1H, Ar-6″-H), 4.31–4.28 (t, <i>J</i> = 5.8 Hz, 2H, CH<sub>2</sub>, Cl(CH<sub>2</sub>)<sub>2</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>O), 3.89–3.85 (t, <i>J</i> = 6.4 Hz, 2H, CH<sub>2</sub>, Cl<b>C<i>H</i></b><sub><b><i>2</i></b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.33–2.25 (p, 2H, CH<sub>2</sub>, ClCH<sub>2</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>O); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.66 (1C, Ar-2-CH), 131.75 (1C, Ar-8-CH), 130.07 (1C, Ar-5″-CH), 124.90 (1C, Ar-7-CH), 124.01 (1C, Ar-5′-CH), 122.81 (1C, Ar-6′-CH), 122.73 (1C, Ar-2′-CH), 112.84 (1C, Ar-6″-CH), 110.31 (1C, Ar-4″-CH), 104.84 (1C, Ar-2″-CH), 103.49 (1C, Ar-5-CH), 65.61 (1C, CH<sub>2</sub>, Cl(CH<sub>2</sub>)<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>O), 42.46 (1C, CH<sub>2</sub>, Cl<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 32.13 (1C, CH<sub>2</sub>, ClCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>O); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 458.1 [M + 1]<sup>+</sup>.</div></div><div id="sec4_23_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i101" class="anchor-spacer"></div><h4 class="article-section__title" id="_i101"> <i>N</i>-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)-6-(3-chloropropoxy)quinazolin-4-amine (<b>24c</b>)</h4><div class="NLM_p last">Yellow solid, 1.55 g, yield: 62%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.72 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.57 (s, 1H, Ar-2-H), 8.27–8.26 (d (m-coupling), <i>J</i> = 1.8 Hz, 1H, Ar-5-H), 7.92 (m, 2H, Ar-2′, 7-H), 7.77–7.74 (d, <i>J</i> = 9.1 Hz, 1H, Ar-8-H), 7.64–7.59 (t, <i>J</i> = 7.8 Hz, 1H, Ar-5″-H), 7.54–7.51 (dd, <i>J</i> = 9.1, 1.7 Hz, 1H, Ar-6′-H), 7.48–7.45 (d, <i>J</i> = 7.7 Hz, 1H, Ar-4″-H), 7.36–7.33 (d, <i>J</i> = 9 Hz, 1H, Ar-5′-H), 7.25–7.22 (m, 2H, Ar-2″, 6″-H), 4.30–4.26 (t, <i>J</i> = 5.8 Hz, 2H, CH<sub>2</sub>, Cl(CH<sub>2</sub>)<sub>2</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>O), 3.89–3.85 (t, <i>J</i> = 6.3 Hz, 2H, CH<sub>2</sub>, Cl<b>C<i>H</i></b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.33–2.25 (p, 2H, CH<sub>2</sub>, ClCH<sub>2</sub><b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>O); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.61 (1C, Ar-2-CH), 131.87 (1C, Ar-8-CH), 130.04 (1C, Ar-5″-CH), 124.84 (1C, Ar-7-CH), 123.99 (1C, Ar-6″-CH), 122.95 (1C, Ar-5′-CH), 122.75 (1C, Ar-6′-CH), 120.70 (1C, Ar-4″-CH), 119.89 (1C, Ar-2′-CH), 113.27 (1C, Ar-2″-CH), 103.45 (1C, Ar-5-CH), 65.59 (1C, CH<sub>2</sub>, Cl(CH<sub>2</sub>)<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>O), 42.43 (1C, CH<sub>2</sub>, Cl<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 32.15 (1C, CH<sub>2</sub>, ClCH<sub><i>2</i></sub><b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>O); MS (APCI<sup>+</sup>): <i>m</i>/<i>z</i> 508.2 [M + 1]<sup>+</sup>.</div></div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> General Procedure for the Synthesis of Compounds (<b>25a–h</b>)</h3><div class="NLM_p">A mixture of <b>24a</b>, <b>24b</b>, or <b>24c</b> (1 mmol) and the appropriate secondary amine, 1-methyl piperazine or morpholine (100 mmol), was placed in a 25 mL round-bottomed flask or in a capped vial in the case of secondary amine of low boiling point as diethylamine (100 mmol). The reaction mixture was heated under reflux and monitored by TLC. After 2–4 h, the TLC of the reaction mass indicated the absence of the 6-chloropropoxy quinazoline intermediate (<b>24a–c</b>). The reaction mixture was cooled to room temperature and poured onto iced-water (30 mL). The resultant precipitate was filtered, washed with water, and dried. Trituration of the dried precipitate with diethyl ether yielded the desired products in a pure form.</div><div id="sec4_24_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i103" class="anchor-spacer"></div><h4 class="article-section__title" id="_i103"> <i>N</i>-(3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)-6-(3-morpholinopropoxy)quinazolin-4-amine (<b>25a</b>)</h4><div class="NLM_p last">Reaction time: 2 h, yellowish white solid, 0.4 g, yield: 92%; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.67 (s, 1H, Ar-2-H), 7.86–7.83 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.79 (s, 1H, Ar-5-H), 7.57 (s, 1H, NH), 7.55–7.52 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.46–7.43 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6′-H), 7.40–7.33 (m, 1H, Ar-4″-H), 7.25–7.23 (d, <i>J</i> = 8.6 Hz, 1H, Ar-6″-H), 7.18 (br s, 2H, Ar-2′, 2″-H), 7.06–7.00 (t, <i>J</i> = 8.6 Hz, 1H, Ar-5″-H), 6.97–6.94 (d, <i>J</i> = 8.8 Hz, 1H, Ar-5′-H), 5.14 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b><i>2</i></b></sub>C<sub>6</sub>H<sub>4</sub>F), 4.14–4.09 (t, <i>J</i> = 6.2 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 3.74 (br s, 4H, 2-CH<sub>2</sub>, O(<b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 2.57–2.52 (t, <i>J</i> = 7.1 Hz, <i>N</i>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>CH<sub>2</sub>O), 2.48 (br s, 4H, 2-CH<sub>2</sub>, N(<b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 2.08–2.01 (m, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C DEPT (75 MHz, CDCl<sub>3</sub>): δ 152.83 (1C, Ar-2-CH), 130.39 (1C, Ar-8-CH), 130.14 (1C, Ar-5″-CH), 124.75 (1C, Ar-7-CH), 124.54 (1C, Ar-6″-CH), 122.44 (1C, Ar-6′-CH), 121.83 (1C, Ar-2′-CH), 115.24 (1C, Ar-5′-CH), 114.43 (1C, Ar-4″-CH), 113.83 (1C, Ar-2″-CH), 100.59 (1C, Ar-5-CH), 70.45 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 66.94 (2C, 2-CH<sub>2</sub>, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 66.74 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.39 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.75 (2C, 2-CH<sub>2</sub>, N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 26.28 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 157.44 (1C, Ar-4-C), 156.90 (1C, Ar-6-C), 152.83 (1C, Ar-2-CH), 150.97 (1C, Ar-3″-C), 145.43 (1C, Ar-8a-C), 139.10 (1C, Ar-4′-C), 132.46 (2C, Ar-1″, 1′-C), 130.26 (2C, Ar-8, 5″-CH), 124.74 (1C, Ar-7-CH), 124.53 (1C, Ar-6″-CH), 123.55 (1C, Ar-3′-C), 122.45 (1C, Ar-6′-CH), 121.83 (1C, Ar-2′-CH), 115.50 (1C, Ar-5′-CH), 114.79 (1C, Ar-4a-C), 114.44 (1C, Ar-4″-CH), 113.84 (1C, Ar-2″-CH), 100.60 (1C, Ar-5-CH), 70.46 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 66.93 (2C, 2-CH<sub>2</sub>, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 66.76 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.38 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.75 (2C, 2-CH<sub>2</sub>, N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 26.28 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>29</sub>ClFN<sub>4</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 523.1912; found, 523.1912.</div></div><div id="sec4_24_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i104" class="anchor-spacer"></div><h4 class="article-section__title" id="_i104"> <i>N</i>-(3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)-6-(3-(4-methylpiperazin-1-yl)propoxy)quinazolin-4-amine (<b>25b</b>)</h4><div class="NLM_p last">Reaction time; 3 h, light-brown solid, 0.4 g, yield: 77%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.61 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.48 (s, 1H, Ar-2-H), 8.00 (d, 1H, Ar-8-H), 7.88 (s, 1H, Ar-5-H), 7.75–7.69 (m, 2H, Ar-7, 6′-H), 7.48 (s, 1H, Ar-2′-H), 7.45–7.43 (d, <i>J</i> = 8.8 Hz, 1H, Ar-4″-H), 7.34 (s, 1H, Ar-2″-H), 7.32–7.29 (d, <i>J</i> = 7.5 Hz, 1H, Ar-5′-H), 7.28–7.25 (d, <i>J</i> = 9 Hz, 1H, Ar-6″-H), 7.20–7.15 (t, <i>J</i> = 9 Hz, 1H, Ar-5″-H), 5.25 (s, 2H, CH<sub>2</sub>, O<b>C<i>H</i></b><sub><b><i>2</i></b></sub>C<sub>6</sub>H<sub>4</sub>F), 4.18–4.14 (t, <i>J</i> = 6 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 2.47–2.43 (t, <i>J</i> = 6.1 Hz, <i>N</i>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>CH<sub>2</sub>CH<sub>2</sub>O), 2.38 (br s, 4H, 2-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>, piperazino), 2.30 (br s, 4H, 2-<b>C<i>H</i></b><sub><b>2</b></sub>, piperazino), 2.13 (s, 3H, <b>C<i>H</i></b><sub><b><i>3</i></b></sub>), 1.98–1.90 (p, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C DEPT (75 MHz, DMSO): δ 152.71 (1C, Ar-2-CH), 131.06 (1C, Ar-8-CH), 129.85 (1C, Ar-5″-CH), 124.69 (1C, Ar-7-CH), 124.54 (1C, Ar-6″-CH), 123.76 (1C, Ar-6′-CH), 122.72 (1C, Ar-2′-CH), 115.28 (1C, Ar-5′-CH), 114.73 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 103.36 (1C, Ar-5-CH), 69.8 4 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 67.12 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.19 (2C, 2-<b><i>C</i>H</b><sub><b>2</b></sub>, piperazino), 54.91 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.25 (2C, 2-<b><i>C</i>H</b><sub><b>2</b></sub>, piperazino), 46.20 (1C, CH<sub>3</sub>), 26.70 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); <sup>13</sup>C NMR (75 MHz, DMSO): δ 157.28 (1C, Ar-4-C), 157.20 (1C, Ar-6-C), 152.71 (1C, Ar-2-CH), 150.01 (1C, Ar-3″-C), 145.38 (1C, Ar-8a-C), 140.18 (1C, Ar-4′-C), 140.08 (1C, Ar-1″-C), 133.81 (1C, Ar-1′-C), 131.05 (1C, Ar-8-CH), 129.85 (1C, Ar-5″-CH), 124.68 (1C, Ar-7-CH), 124.53 (1C, Ar-6″-CH), 123.78 (1C, Ar-6′-CH), 122.71 (1C, Ar-2′-CH), 121.50 (1C, Ar-3′-C), 116.03 (1C, Ar-4a-C), 115.27 (1C, Ar-5′-CH), 114.76 (1C, Ar-4″-CH), 114.33 (1C, Ar-2″-CH), 103.39 (1C, Ar-5-CH), 69.87 (1C, CH<sub>2</sub>, O<b><i>C</i>H</b><sub><b>2</b></sub>C<sub>6</sub>H<sub>4</sub>F), 67.13 (1C, CH<sub>2</sub>, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.18 (2C, 2-<b><i>C</i>H</b><sub><b>2</b></sub>, piperazino), 54.91 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O), 53.24 (2C, 2-<b><i>C</i>H</b><sub><b>2</b></sub>, piperazino), 46.18 (1C, CH<sub>3</sub>), 26.70 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>32</sub>ClFN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 536.2229; found, 536.2226.</div></div><div id="sec4_24_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i105" class="anchor-spacer"></div><h4 class="article-section__title" id="_i105"> <i>N</i>-(3-Chloro-4-(3-fluorophenoxy)phenyl)-6-(3-morpholinopropoxy)quinazolin-4-amine (<b>25c</b>)</h4><div class="NLM_p last">Reaction time; 3 h, white solid, mp 172–175 °C, 0.4 g, yield: 94%; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.70 (s, 1H, NH), 8.34 (s, 1H, Ar-2-H), 8.00 (s, 1H, Ar-5-H), 7.84–7.81 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8-H), 7.68–7.65 (d, <i>J</i> = 8.9 Hz, 1H, Ar-7-H), 7.43–7.38 (m, 2H, Ar-6′, 2′-H), 7.27–7.21 (m, 1H, Ar-4″-H), 7.07–7.04 (d, <i>J</i> = 8.8 Hz, 1H, Ar-5′-H), 6.77–6.71 (m, 2H, Ar-2″, 5″-H), 6.67–6.63 (dd, 1H, Ar-6″-H), 4.08–4.04 (t, <i>J</i> = 6.0 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 3.72 (br s, 4H, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b>2</b></sub>, morpholino), 2.52–2.46 (m, 6H, 3-CH<sub>2</sub>, 2-CH<sub>2</sub> of N(<b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub><b><i>2</i></b></sub>, morpholino, &N-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.03–1.95 (p, <i>J</i> = 6.0 Hz, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C DEPT (75 MHz, CDCl<sub>3</sub>): δ 152.63 (1C, Ar-2-CH), 130.58 (1C, Ar-8-CH), 130.08 (1C, Ar-5″-CH), 124.81 (1C, Ar-7-CH), 124.06 (1C, Ar-5′-CH), 122.01 (1C, Ar-6′-CH), 121.59 (1C, Ar-2′-CH), 112.66 (1C, Ar-6″-CH), 109.95 (1C, Ar-4″-CH), 104.96 (1C, Ar-2″-CH), 101.07 (1C, Ar-5-CH), 66.86 (2C, 2-CH<sub>2</sub>, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 66.74 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.38 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.71 (2C, 2-CH<sub>2</sub>, N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b>2</b></sub>, morpholino), 26.18 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 157.54 (1C, Ar-4-C), 156.71 (1C, Ar-6-C), 152.64 (1C, Ar-2-CH), 147.48 (1C, Ar-3″-C), 145.49 (1C, Ar-1″-C), 136.11 (1C, Ar-8a-C), 130.59 (1C, Ar-8-CH), 130.46 (1C, Ar-4′-C), 130.28 (1C, Ar-5″-CH), 126.66 (1C, Ar-1′-C), 124.74 (1C, Ar-7-CH), 123.98 (1C, Ar-5′-CH), 122.07 (2C, Ar-6′, 3′-CH), 121.49 (1C, Ar-2′-CH), 115.70 (1C, Ar-4a-C), 112.63 (1C, Ar-6″-CH), 109.97 (1C, Ar-4″-CH), 104.97 (1C, Ar-2″-CH), 100.86 (1C, Ar-5-CH), 66.89 (2C, 2-CH<sub>2</sub>, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub> morpholino), 66.76 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.39 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.74 (2C, 2-CH<sub>2</sub>, N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b>2</b></sub>, morpholino), 26.22 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>27</sub>H<sub>27</sub>ClFN<sub>4</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 509.1756; found, 509.1755.</div></div><div id="sec4_24_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i106" class="anchor-spacer"></div><h4 class="article-section__title" id="_i106"> <i>N</i>-(3-Chloro-4-(3-fluorophenoxy)phenyl)-6-(3-(4-methylpiperazin-1-yl)propoxy)quinazolin-4-amine (<b>25d</b>)</h4><div class="NLM_p last">Reaction time: 4 h, brown solid, 0.39 g, yield: 76%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.73 (s, 1H, NH, exchanged with D<sub>2</sub>O), 8.54 (s, 1H, Ar-2-H), 8.24 (s, 1H, Ar-2″-H), 7.90 (br s, 2H, Ar-2′,5-H), 7.75–7.72 (d, <i>J</i> = 9.1 Hz, 1H, Ar-8-H), 7.52–7.49 (d, <i>J</i> = 9 Hz, 1H, Ar-7-H), 7.44–7.36 (m, 1H, Ar-4″-H), 7.31–7.28 (d, <i>J</i> = 8.9 Hz, 1H, Ar-6′-H), 6.97–6.92 (t, <i>J</i> = 8.1 Hz, 1H, Ar-5″-H), 6.83–6.80 (d, <i>J</i> = 8.9 Hz, 1H, Ar-5′-H), 6.77–6.74 (d, <i>J</i> = 8.2 Hz, 1H, Ar-6″-H), 4.19–4.15 (t, <i>J</i> = 5.4 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 2.45–2.31 (m, 10H, 4-CH<sub>2</sub> of piperazine & <i>N</i>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 1.97–1.93 (m, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C DEPT (75 MHz, DMSO): δ 152.50 (1C, Ar-2-CH), 131.73 (1C, Ar-8-CH), 129.96 (1C, Ar-5″-CH), 124.83 (1C, Ar-7-CH), 123.93 (1C, Ar-5′-CH), 122.80 (1C, Ar-6′-CH), 122.66 (1C, Ar-2′-CH), 112.74 (1C, Ar-6″-CH), 110.27 (1C, Ar-4″-CH), 104.78 (1C, Ar-2″-CH), 103.43 (1C, Ar-5-CH), 67.15 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.18 (2C, 2-CH<sub>2</sub>, piperazino), 54.89 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>-(CH<sub>2</sub>)<sub>2</sub>-O), 53.24 (2C, 2-CH<sub>2</sub>, piperazino), 46.18 (1C, CH<sub>3</sub>), 26.70 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); <sup>13</sup>C NMR (75 MHz, DMSO): δ 161.76 (1C, Ar-4-C), 159.13 (1C, Ar-3″-CH), 157.34 (1C, Ar-1″-C), 157.09 (1C, Ar-6-C), 152.50 (1C, Ar-2-CH), 146.16 (1C, Ar-8a-C), 145.50 (1C, Ar-4′-C), 137.91 (1C, Ar-1′-C), 131.72 (1C, Ar-8-CH), 129.95 (1C, Ar-5″-CH), 125.35 (1C, Ar-3′-C), 124.84 (1C, Ar-7-CH), 123.96 (1C, Ar-5′-CH), 122.78 (1C, Ar-6′-CH), 122.68 (1C, Ar-2′-CH), 116.13 (1C, Ar-4a-C), 112.81 (1C, Ar-6″-CH), 110.27 (1C, Ar-4″-CH), 104.78 (1C, Ar-2″-CH), 103.49 (1C, Ar-5-CH), 67.18 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.17 (2C, 2-CH<sub>2</sub>, piperazino), 54.89 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 53.22 (2C, 2-CH<sub>2</sub>, piperazino), 46.15 (1C, CH<sub>3</sub>), 26.70 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub>2</sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>30</sub>ClFN<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 522.2072; found, 522.2071.</div></div><div id="sec4_24_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i107" class="anchor-spacer"></div><h4 class="article-section__title" id="_i107"> <i>N</i>-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)-6-(3-morpholinopropoxy)quinazolin-4-amine (<b>25e</b>)</h4><div class="NLM_p last">Reaction time, yellow solid, 3 h, mp 95–100 °C, 0.46 g, yield: 84%; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.71 (s, 1H, NH), 8.20 (s, 1H, Ar-2-H), 8.04 (s, 1H, Ar-5-H), 7.86–7.83 (d, <i>J</i> = 9 Hz, 1H, Ar-8-H), 7.69–7.67 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.45–7.40 (m, 2H, Ar-6′, 2′-H), 7.34 (br s, 2H, Ar-4″,5′-H), 7.20 (s, 1H, Ar-2″-H), 7.12–7.05 (m, 2H, Ar-6″, 5″-H), 4.08–4.05 (t, <i>J</i> = 5.6 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 3.74 (br s, 4H, O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 2.59–2.48 (m, 6H, 3-CH<sub>2</sub>, 2-CH<sub>2</sub> of N(<b>C<i>H</i></b><sub><b><i>2</i></b></sub>)<sub><b><i>2</i></b></sub> morpholino, &N-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.02–1.98 (p, <i>J</i> = 5.6 Hz, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C DEPT (75 MHz, CDCl<sub>3</sub>): δ 152.61 (1C, Ar-2-CH), 130.32 (1C, Ar-8-CH), 130.19 (1C, Ar-5″-CH), 124.85 (1C, Ar-7-CH), 124.04 (1C, Ar-6″-CH), 121.98 (1C, Ar-5′-CH), 121.55 (1C, Ar-6′-CH), 120.17 (1C, Ar-4″-CH), 119.61 (1C, Ar-2′-CH), 113.95 (1C, Ar-2″-CH), 100.83 (1C, Ar-5-CH), 66.85 (3C, 3-CH<sub>2</sub>, 2-CH<sub>2</sub> of O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.37 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 53.70 (2C, 2-CH<sub>2</sub> of N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b>2</b></sub> morpholino), 26.14 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-CH<sub>2</sub>-O); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 157.64 (1C, Ar-4-C), 157.58 (1C, Ar-6-C), 156.75 (1C, Ar-1″-C), 152.61 (1C, Ar-2-CH), 147.24 (1C, Ar-8a-C), 145.45 (1C, Ar-4′-C), 136.31 (1C, Ar-1′-C), 130.33 (2C, Ar-8-CH, <i>C</i>F<sub>3</sub>), 130.20 (2C, Ar-5″-CH, Ar-3″-C), 126.68 (1C, Ar-3′-C), 124.85 (1C, Ar-7-CH), 124.04 (1C, Ar-6″-CH), 121.98 (1C, Ar-5′-CH), 121.54 (1C, Ar-6′-CH), 120.17 (1C, Ar-4″-CH), 119.64 (1C, Ar-2′-CH), 115.73 (1C, Ar-4a-C), 113.96 (1C, Ar-2″-CH), 100.83 (1C, Ar-5-CH), 66.86 (2C, 2-CH<sub>2</sub> of O(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b><i>2</i></b></sub>, morpholino), 66.74 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 55.37 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 53.71 (2C, 2-CH<sub>2</sub> of N(<b><i>C</i>H</b><sub><b>2</b></sub>)<sub><b>2</b></sub> morpholino), 26.16 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>27</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 559.1724; found, 559.1726.</div></div><div id="sec4_24_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i108" class="anchor-spacer"></div><h4 class="article-section__title" id="_i108"> <i>N</i>-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)-6-(3-(4-methylpiperazin-1-yl)propoxy)quinazolin-4-amine (<b>25f</b>)</h4><div class="NLM_p last">Reaction time: 4 h, brown solid, 0.46 g, yield: 82%, <sup>1</sup>H NMR (300 MHz, CHCl<sub>3</sub>): δ 8.71 (s, 1H, NH), 8.49 (s, 1H, Ar-2-H), 8.09 (s, 1H, Ar-5-H), 7.84–7.81 (d, <i>J</i> = 8.9 Hz, 1H, Ar-8-H), 7.76–7.73 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.43–7.40 (m, 3H, Ar-6′, 2′,4″-H), 7.34–7.32 (d, 1H, Ar-5′-H), 7.20 (s, 1H, Ar-2″-H), 7.12–7.06 (m, 2H, Ar-6″, 5″-H), 4.05 (br s, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-O), 2.83 (br s, 2H, CH<sub>2</sub>, <i>N</i>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.49 (br s, 8H, 4CH<sub>2</sub>, piperazino), 2.29 (s, 3H, CH<sub>3</sub>), 2.00 (br s, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-C<b><i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O). <sup>13</sup>C DEPT (75 MHz, CHCl<sub>3</sub>): δ 152.61 (1C, Ar-2-CH), 130.31 (1C, Ar-8-CH), 130.07 (1C, Ar-5″-CH), 125.85 (1C, Ar-7-CH), 124.85 (1C, Ar-6″-CH), 122.00 (1C, Ar-5′-CH), 121.56 (1C, Ar-6′-CH), 120.11 (1C, Ar-4″-CH), 119.54 (1C, Ar-2′-CH), 113.88 (1C, Ar-2″-CH), 101.26 (1C, Ar-5-CH), 66.89 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 54.89 (3C, 3-CH<sub>2</sub>, 2-CH<sub>2</sub> of piperazino, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 53.08 (2C, 2-CH<sub>2</sub>, piperazino), 45.89 (1C, CH<sub>3</sub>), 26.37 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-CH<sub>2</sub>-O). <sup>13</sup>C NMR (75 MHz, CHCl<sub>3</sub>): δ 157.73 (1C, Ar-4-C), 157.44 (1C, Ar-6-C), 156.86 (1C, Ar-1″-C), 152.64 (1C, Ar-2-CH), 147.03 (1C, Ar-8a-C), 145.48 (1C, Ar-4′-C), 136.62 (1C, Ar-1′-C), 130.31 (2C, Ar-8-CH, CF<sub>3</sub>), 130.10 (2C, Ar-5″-CH, Ar-3″-C), 126.63 (1C, Ar-3′-C), 124.81 (1C, Ar-7-CH), 123.99 (1C, Ar-6″-CH), 122.01 (1C, Ar-5′-CH), 121.54 (1C, Ar-6′-CH), 120.12 (1C, Ar-4″-CH), 119.52 (1C, Ar-2′-CH), 115.86 (1C, Ar-4a-C), 113.90 (1C, Ar-2″-CH), 101.31 (1C, Ar-5-CH), 66.78 (1C, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-O), 54.87 (1C, CH<sub>2</sub>, <i>N</i>-<b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 54.72 (2C, 2-CH<sub>2</sub>, piperazine), 52.89 (2C, 2-CH<sub>2</sub>, piperazine), 45.75 (1C, CH<sub>3</sub>), 26.33 (1C, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b><i>C</i>H</b><sub><b>2</b></sub>-CH<sub>2</sub>-O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>29</sub>H<sub>30</sub>ClF<sub>3</sub>N<sub>5</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 572.2040; found, 572.2042.</div></div><div id="sec4_24_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i109" class="anchor-spacer"></div><h4 class="article-section__title" id="_i109"> <i>N</i>-(3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)-6-(3-(diethylamino)propoxy)quinazolin-4-amine (<b>25g</b>)</h4><div class="NLM_p last">Reaction time; 2 h, yellow solid, 0.41 g, yield: 76%; <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>): δ 8.72 (s, 1H, NH), 8.56 (s, 1H, Ar-2-H), 8.12 (s, 1H, Ar-5-H), 7.85–7.82 (d, <i>J</i> = 8.8 Hz, 1H, Ar-8-H), 7.74–7.72 (d, <i>J</i> = 8.8 Hz, 1H, Ar-7-H), 7.44–7.40 (m, 3H, Ar-6′, 2′,4″-H), 7.34–7.32 (d, 1H, Ar-5′-H), 7.21 (s, 1H, Ar-2″-H), 7.12–7.05 (m, 2H, Ar-6″, 5″-H), 4.04–4.00 (t, <i>J</i> = 6.3 Hz, 2H, CH<sub>2</sub>, <i>N</i>-(CH<sub>2</sub>)<sub>2</sub>-<b>C<i>H</i></b><sub><b>2</b></sub>-O), 2.61–2.54 (m, 6H, 3(CH<sub>2</sub>)), 1.99–1.91 (p, 2H, CH<sub>2</sub>, <i>N</i>-CH<sub>2</sub>-<b>C<i>H</i></b><sub><b><i>2</i></b></sub>-CH<sub>2</sub>-O), 1.07–1.02 (t, <i>J</i> = 7.1 Hz, 6H, 2(CH<sub>3</sub>)); <sup>13</sup>C DEPT (75 MHz, CDCl<sub>3</sub>): δ 152.61 (1C, Ar-2-CH), 130.30 (1C, Ar-8-CH), 130.02 (1C, Ar-5″-CH), 125.10 (1C, Ar-7-CH), 124.05 (1C, Ar-6″-CH), 121.97 (1C, Ar-5′-CH), 121.56 (1C, Ar-6′-CH), 120.11 (1C, Ar-4″-CH), 119.57 (1C, Ar-2′-CH), 113.97 (1C, Ar-2″-CH), 101.04 (1C, Ar-5-CH), 66.91 (1C, CH<sub>2</sub>, <b><i>C</i>H</b><sub><b>2</b></sub>O), 49.13 (1C, CH<sub>2</sub>, NH <b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 46.89 (2C, 2CH<sub>2</sub>, (CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub><b>-</b>)<sub>2</sub>), 26.46 (1C, CH<sub>2</sub>, NHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>O), 11.28 (2C, 2CH<sub>3</sub>); <sup>13</sup>C NMR (75 MHz, CDCl<sub>3</sub>): δ 157.72 (1C, Ar-4-C), 157.52 (1C, Ar-6-C), 156.89 (1C, Ar-1″-C), 152.61 (1C, Ar-2-CH), 147.09 (1C, Ar-8a-C), 145.43 (1C, Ar-4′-C), 136.54 (1C, Ar-1′-C), 130.29 (2C, Ar-8-CH, CF<sub>3</sub>), 130.03 (2C, Ar-5″-CH, Ar-3″-C), 126.64 (1C, Ar-3′-C), 125.09 (1C, Ar-7-CH), 124.04 (1C, Ar-6″-CH), 121.96 (1C, Ar-5′-CH), 121.56 (1C, Ar-6′-CH), 120.11 (1C, Ar-4″-CH), 119.57 (1C, Ar-2′-CH), 115.84 (1C, Ar-4a-C), 113.97 (1C, Ar-2″-CH), 101.03 (1C, Ar-5-CH), 66.91 (1C, CH<sub>2</sub>, <b><i>C</i>H</b><sub><b>2</b></sub>O), 49.14 (1C, CH<sub>2</sub>, NH <b><i>C</i>H</b><sub><b>2</b></sub>(CH<sub>2</sub>)<sub>2</sub>O), 46.90 (2C, 2CH<sub>2</sub>, 2(CH<sub>3</sub><b><i>C</i>H</b><sub><b>2</b></sub><b>-</b>)<sub>2</sub>), 26.49 (1C, CH<sub>2</sub>, NHCH<sub>2</sub><b><i>C</i>H</b><sub><b>2</b></sub>CH<sub>2</sub>O), 11.29 (2C, 2CH<sub>3</sub>); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>29</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>2</sub> [M + 1]<sup>+</sup>, 545.1931; found, 545.1933.</div></div><div id="sec4_24_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i110" class="anchor-spacer"></div><h4 class="article-section__title" id="_i110"> 2-((3-((4-((3-Chloro-4-(3-(trifluoromethyl)phenoxy)phenyl)amino)quinazolin-6-yl)oxy)propyl) amino)ethan-1-ol (<b>25h</b>)</h4><div class="NLM_p">A mixture of intermediate <b>24c</b> (0.5 g, 1 mmol) and 2-aminoethanol (0.5 g, 0.6 mL, 10 mmol) in methanol (25 mL) was held at reflux for 8 h in the presence of triethylamine (15 drops). Methanol was removed under reduced pressure, and the remaining residue was dissolved in DCM and purified by flash column chromatography with DCM/MeOH (95:5%) as an eluent to yield compound <b>25h</b>.</div><div class="NLM_p last">Yellow semisolid mass, 0.22 g, yield: 42%; <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>): δ 9.76 (1H, NH, exchanged with D<sub>2</sub>O), 8.56 (s, 1H, Ar-2-H), 8.27–8.26 (d (m-coupling), <i>J</i> = 2.1, 1H, Ar-5-H), 7.96–7.92 (dd, <i>J</i> = 8.7 Hz, 2 Hz, 2H, Ar-7, 8-H), 7.77–7.74 (d, <i>J</i> = 9 Hz, Ar-5′-H), 7.65–7.60 (t, <i>J</i> = 7.8 Hz, 1H, Ar-5″-H), 7.54–7.50 (dd, <i>J</i> = 9 Hz, 2.1 Hz, 1H, Ar-6′-H), 7.49–7.46 (d, <i>J</i> = 7.9 Hz, 1H, Ar-4″-H), 7.36–7.33 (d (m-coupling), <i>J</i> = 2.1 Hz, 1H, Ar-2′-H), 7.25–7.22 (m, 2H, Ar-6″, 2″), 4.24–4.20 (t, <i>J</i> = 6.2 Hz, 2H, NH + OH, exchanged with D<sub>2</sub>O), 3.49–3.45 (t, <i>J</i> = 5.7 Hz, 2H, CH<sub>2</sub>, −C<i>H</i><sub>2</sub>O), 3.37–3.33 (t, <i>J</i> = 6.7 Hz, 2H, CH<sub>2</sub>, OHC<i>H</i><sub>2</sub>CH<sub>2</sub>NH−), 2.76–2.71 (t, <i>J</i> = 6.7 Hz, 2H, CH<sub>2</sub>, OHCH<sub>2</sub>C<i>H</i><sub>2</sub>NH−), 2.62–2.56 (m, 2H, CH<sub>2</sub>, NHC<i>H</i><sub>2</sub>(CH<sub>2</sub>)<sub>2</sub>O), 2.00–1.92 (p, 2H, CH<sub>2</sub>, NHCH<sub>2</sub>C<i>H</i><sub>2</sub>CH<sub>2</sub>O); <sup>13</sup>C DEPT (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 152.47 (1C, Ar-2-CH), 131.92 (1C, Ar-8-CH), 129.96 (1C, Ar-5″-CH), 124.92 (1C, Ar-7-CH), 123.98 (1C, Ar-6″-CH), 123.00 (1C, Ar-5′-CH), 122.78 (1C, Ar-6′-CH), 120.70 (1C, Ar-4″-CH), 119.91 (1C, Ar-2′-CH), 113.30 (1C, Ar-2″-CH), 103.35 (1C, Ar-5-CH), 67.25 (1C, CH<sub>2</sub>, −<i>C</i>H<sub>2</sub>O−), 60.84 (1C, CH<sub>2</sub>, OH<i>C</i>H<sub>2</sub>−), 52.29 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><i>C</i>H<sub>2</sub>NH−), 46.41 (1C, CH<sub>2</sub>, −NH<i>C</i>H<sub>2</sub>(CH<sub>2</sub>)<sub>2</sub>O), 29.82 (1C, CH<sub>2</sub>, NHCH<sub>2</sub><i>C</i>H<sub>2</sub>CH<sub>2</sub>O); <sup>13</sup>C NMR (75 MHz, DMSO-<i>d</i><sub>6</sub>): δ 158.14 (1C, Ar-4-C), 157.42 (1C, Ar-6-C), 157.04 (1C, Ar-1″-C), 152.47 (1C, Ar-2-CH), 145.81 (1C, Ar-8a-C), 145.51 (1C, Ar-4′-C), 138.20 (1C, Ar-1′-C), 131.92 (1C, Ar-8-CH), 131.39 (1C, CF<sub>3</sub>), 130.97 (1C, Ar-3″-C), 129.97 (1C, Ar-5″-CH), 125.46 (1C, Ar-3′-C), 124.91 (1C, Ar-7-CH), 123.98 (1C, Ar-6″-CH), 123.00 (1C, Ar-5′-CH), 122.78 (1C, Ar-6′-CH), 120.70 (2C, Ar-2′,4″-CH), 116.17 (1C, Ar-4a-C), 113.31 (1C, Ar-2″-CH), 103.36 (1C, Ar-5-CH), 67.25 (1C, CH<sub>2</sub>, <i>C</i>H<sub>2</sub>O), 60.84 (1C, CH<sub>2</sub>, OH<i>C</i>H<sub>2</sub>-), 52.29 (1C, CH<sub>2</sub>, OHCH<sub>2</sub><i>C</i>H<sub>2</sub>NH), 46.41 (1C, CH<sub>2</sub>, NH <i>C</i>H<sub>2</sub>(CH<sub>2</sub>)<sub>2</sub>O), 29.83 (1C, CH<sub>2</sub>, NHCH<sub>2</sub><i>C</i>H<sub>2</sub>CH<sub>2</sub>O); HRMS (ES<sup>+</sup>): <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>25</sub>ClF<sub>3</sub>N<sub>4</sub>O<sub>3</sub> [M + 1]<sup>+</sup>, 533.1567; found, 533.1563.</div></div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i111" class="anchor-spacer"></div><h3 class="article-section__title" id="_i111"> Pharmacology</h3><div id="sec4_25_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i112" class="anchor-spacer"></div><h4 class="article-section__title" id="_i112"> Material Quality Control and Reagents</h4><div class="NLM_p last">The recombinant protein kinases employed in the compound profiling process were cloned, expressed, and purified using proprietary methods. Quality control testing is routinely performed to ensure compliance to acceptable standards. The [γ-<sup>32</sup>P]ATP was purchased from PerkinElmer. All other materials were of standard laboratory grade. The stock solutions of the tested compounds were prepared and 5 μL were incorporated into each assay and tested against the protein kinase panel selected. Quality control testing is routinely performed to ensure compliance with acceptable standards. The ADP-GloTM assay kit was purchased from Promega, and all other materials were of standard laboratory grade.</div></div><div id="sec4_25_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i113" class="anchor-spacer"></div><h4 class="article-section__title" id="_i113"> [γ-<sup>32</sup>P]ATP Radiometric Protein Kinase Assay Method</h4><div class="NLM_p last">The protein kinase profiling was conducted by Kinexus Bioinformatics Corporation, Vancouver, British Columbia, Canada, according to the following protocols. The kinase profiling of 7 test compounds, namely, <b>10a</b>, <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b> by [γ-<sup>32</sup>P]ATP radioisotope was carried out in a single measurement at 10 μM concentration against a panel of 20 protein kinases. Sorafenib and lapatinib were used as positive references at a final concentration of 10 μM. All experiments have been conducted in a designated radioactive environment and at ambient temperature for 30 min in a total volume of 25 μL according to the following protocol. Component 1: 5.0 μL of the active kinase (∼10–50 nM final concentration), component 2: 5.0 μL of the substrate solution, component 3: 5.0 μL of buffer, component 4: 5.0 μL of test and reference at various concentrations or 10% DMSO for the blank control, and component 5: 5.0 μL of [γ-<sup>32</sup>P]ATP (from 50 μM stock solution, 0.8 μCi). The research was launched by adding [γ-<sup>32</sup>P]ATP to a mixture of other components and incubating for 30 min. Then, the assay was terminated by adding 10 μL of the mixture onto a MultiScreen phosphocellulose P81 plate, which was then washed 3× with 1% solution of phosphoric acid solution. Using a Trilux scintillation counter, the radioactivity was measured for the test samples in comparison with the blank control which contains all components except replacing the kinase substrate by an equal volume of assay buffer. The protein kinase activity was corrected by removing the value of the blank control. The protocol of these assays was optimized to afford a high signal-to-noise ratio.</div></div><div id="sec4_25_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i114" class="anchor-spacer"></div><h4 class="article-section__title" id="_i114"> ADP-Glo Protein Kinase Assay Method</h4><div class="NLM_p last">This assay was conducted using the ADP-GloTM assay kit, to check the generated ADP by specific kinase to be detected through the increase in the luminescence signal. All experiments have been conducted at 30 °C temperature for 30 min in a total volume of 25 μL according to the formula: Component 1: 5.0 μL of active kinase, component 2: 5.0 μL of substrate solution (from 125 μM stock), component 3: 5.0 μL of buffer, component 4: 5.0 μL of test and reference at various concentrations or 10% DMSO for the blank control, and component 5: 5.0 μL of ATP stock solution. The research was launched by mixing all the 5 components in a 96-well plate and incubating it for 30 min. Then, the assay was terminated by adding 25 μL of ADP-Glo (Promega) and then was shaken at 30 °C for 40 min. A total of 50 μL of Kinase Detection Reagent was added and then the 96-well plate was shaken and then incubated for an additional 0.5 h at 30 °C. A GloMax plate reader (Promega; Cat# E7031) was used to read the 96-well reaction plate compared with the blank control which contains all components except replacing the kinase substrate by an equal volume of assay buffer. The protein kinase activity was corrected by removing the value of the blank control. The protocol of these assays was optimized to afford a high signal-to-noise ratio.</div></div><div id="sec4_25_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i115" class="anchor-spacer"></div><h4 class="article-section__title" id="_i115"> <i>In Vitro</i> HER2 Kinase Inhibitory Activity (IC<sub>50</sub>) against Wild and L755S Mutant Types</h4><div class="NLM_p last">The recombinant protein kinases employed in the compound profiling process were cloned, expressed, and purified using proprietary methods. The stock solutions of the test compounds were prepared and tested against HER2 kinases. The assay was carried out as described in detail under the ADP-Glo protein kinase assay method, where the intra-assay variability of each experiment was determined to be less than 10%.</div></div><div id="sec4_25_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i116" class="anchor-spacer"></div><h4 class="article-section__title" id="_i116"> Cell Culture</h4><div class="NLM_p last">Four human BC cell lines were utilized in this study to carry out cell proliferation assay, including luminal HER2-positive AU565 cell line, triple-positive BT474 cells, (ER/PR)-positive MCF7 cells, and triple-negative (MDA-MB-231) cells. The four BC cell lines have been purchased from ATCC (Manassas, VA). The cell culture was conducted at 37 °C, at 5% CO<sub>2</sub>, 95% air, and 100% relative humidity. 10% heat-inactivated fetal bovine serum, <span class="smallcaps smallerCapital">l</span>-glutamine, and 1% penicillin/streptomycin were added to RPMI-1640 media and RPMI/DMEM. The use of the BC cells was limited to 20 passages and was checked regularly (every 6 months) for mycoplasma by detecting the bioluminescence using a MycoWarning sample detection kit (Lonza, Switzerland).</div></div><div id="sec4_25_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i117" class="anchor-spacer"></div><h4 class="article-section__title" id="_i117"> Cell Proliferation Assay</h4><div class="NLM_p last">The <i>in vitro</i> cell viability assay was used to determine cellular proliferation, inhibitory activity, and cytotoxicity of 27 test compounds using a WST-8 reagent as per manufacturer’s instructions (Cell Counting Kit-8, Dojindo Molecular Technologies, Inc., Rockville, MD). About 70–80% of cell confluency was used in all experiments. Cells were seeded in 96-well plates (Nalgene–Nunc, Thermo Fisher Scientific, USA) at a density of 5 × 10<sup>3</sup> cells per each well in 100 μL of culture medium. Cells were incubated at 37 °C overnight to allow cell attachment. The final concentrations of each compound in wells were 0.0, 0.005, 0.05, 0.5, 5, 25, and 50 μM/mL in 100 μL of media (0.1% DMSO for the blank control). After incubation for 72 h, 10 μL of the WST-8 reagent was added into each well and then the plates were incubated for 3 h. Plates were shaken for 1 min and the absorbance was examined using a multiplate reader at λ<sub>450</sub> nm. The IC<sub>50</sub> (50% growth inhibition) for the test compounds against each BC cell was determined using GraphPad Prism version 5.00 for Windows (GraphPad Software, San Diego, CA, <a href="http://www.graphpad.com" class="extLink">www.graphpad.com</a>).</div></div><div id="sec4_25_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i118" class="anchor-spacer"></div><h4 class="article-section__title" id="_i118"> NCI-60 Screening Assay</h4><div class="NLM_p last">The screening methodology is described in detail at <a href="http://dtp.nci.nih.gov/branches/btb/ivclsp.html" class="extLink">http://dtp.nci.nih.gov/branches/btb/ivclsp.html</a>.</div></div><div id="sec4_25_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i119" class="anchor-spacer"></div><h4 class="article-section__title" id="_i119"> NCI <b>60 Cell One-Dose Screen</b><a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a></h4><div class="NLM_p last">All compounds submitted to the NCI 60 cell screen were initially evaluated at a single high dose (10 μM) in the full NCI 60 cell panel including 9 subpanels (hematopoietic system, lung cancer, colon, brain, skin melanoma, ovary, kidney, prostate, and BCs) as described in detail.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Briefly, different cancer cells (100 μL) were inoculated into 96-well microtiter plates at plating densities varying from 5000 to 40,000 cells/well depending on the replication time of each cell line. In the case of suspension cells, the protocol was the same except that 50 μL of 80% TCA (final concentration, 16% TCA) was added to terminate the experiment with fixing the settled cells at the bottom of the wells. The % growth was calculated at each of the drug concentrations levels using the absorbance measurements [time zero, <i>T</i><sub>z</sub>, control growth, <i>C</i>, and test growth in the presence of drugs at 10 μM concentration (<i>T</i><sub>i</sub>)].<span class="NLM_disp-formula" id="ueq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_m001.gif" alt="" id="_i120" /></img></span><span class="NLM_disp-formula" id="ueq2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_m002.gif" alt="" id="_i121" /></img></span></div></div><div id="sec4_25_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i122" class="anchor-spacer"></div><h4 class="article-section__title" id="_i122"> NCI 60 Cell Five-Dose Screen<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a></h4><div class="NLM_p last">Out of 20 compounds tested by one-dose screen, namely, <b>10a</b>,<b>b</b>, <b>15a</b>,<b>c</b>,<b>e</b>,<b>f</b>,<b>g</b>,<b>i</b>,<b>k</b>, <b>17a</b>,<b>c</b>,<b>d</b>, and <b>25b</b>,<b>c</b>,<b>d</b>,<b>e</b>,<b>f</b>,<b>g</b>,<b>h</b>, 13 compounds were selected to be screened by five-dose concentration. These compounds, namely, <b>15f</b>, <b>15g</b>, <b>15i</b>, <b>15k</b>, <b>17a</b>, <b>17c</b>, <b>17d</b>, <b>25b</b>, <b>25d</b>, <b>25e</b>, <b>25f</b>, <b>25g</b>, and <b>25h</b> exhibited significant growth inhibition and further screened at five concentration levels (0.01, 0.1, 1.0, 10, and 100 μM) against the full NCI 60 cell panel. Five-dose NCI 60 cell screening was conducted as described under one-dose screen but at different concentrations.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a> Three dose–response parameters were calculated for each experimental agent. Growth inhibition of 50% (GI<sub>50</sub>) was calculated by <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_m003.gif" alt="" id="_i123" /></img></span>, which is the drug concentration resulting in a 50% reduction in the net protein increase in control cells during the drug incubation. The drug concentration resulting in total growth inhibition (<b>TGI</b>) was calculated by <i>T</i><sub>i</sub> = <i>T</i><sub>z</sub>. The <b>LC</b><sub><b>50</b></sub> (concentration of the drug resulting in a 50% reduction in the measured protein at the end of the drug treatment as compared to that at the beginning) indicating a net loss of cells following treatment was calculated by <span class="NLM_inline-formula"><img class="imgWithoutFigViewer inline-formula-styles" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_m004.gif" alt="" id="_i124" /></img></span>.</div></div><div id="sec4_25_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i125" class="anchor-spacer"></div><h4 class="article-section__title" id="_i125"> Effect of the Optimized compounds in Inducing Cell Apoptosis in BT474 cells</h4><div class="NLM_p last">Cell apoptosis assay was conducted by flow cytometry using BT474 breast cell lines as described.<a onclick="showRef(event, 'ref117'); return false;" href="javascript:void(0);" class="ref ref117">(117)</a> Briefly, BT474 cells were harvested and then washed twice and resuspended in ice-cold phosphate-buffered saline (PBS). The cells were stained by Guava Nexin (Millipore) reagent (Annexin V-PE and 7-AAD), followed by incubation for 20 min in the dark at room temperature. Samples were acquired using Guava easyCyte 5 flow cytometer. The dot plots were generated using CellQuest software (BD Biosciences, San Jose, CA, USA).<a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a> Data represented the mean of duplicates ± SD.</div></div><div id="sec4_25_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i126" class="anchor-spacer"></div><h4 class="article-section__title" id="_i126"> Apoptosis Assay Using MCF7 and MDA-MB-231 Cells</h4><div class="NLM_p last">Cell apoptosis assay using MCF7 and MDA-MB-231 BC cells was conducted by flow cytometry as described.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> Annexin V-FITC apoptosis detection kit (Abcam Inc., Cambridge Science Park, Cambridge, UK) was used to determine the apoptosis and necrosis of the cell populations. Briefly, after treatment with test compounds for 72 h, BC cells (MCF7 and MDA-MB231; 10<sup>5</sup> cells) were harvested by trypsinization and washed twice with ice-cold PBS (pH 7.4). Then, cells were incubated with 0.5 mL of Annexin V-FITC/PI (1.0 mg/mL) solution for 30 min in the dark at room temperature following the manufacturer’s protocol.<a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a> The stained cells were injected via ACEA Novocyte flow cytometer (ACEA Biosciences Inc., San Diego, CA, USA) and analyzed for FITC and PI fluorescent signals using an FL1 and FL2 signal detector, respectively (λ<sub>ex/em</sub> 488/530 nm for FITC and λ<sub>ex/em</sub> 535/617 nm for PI). For each sample, 12,000 events were acquired and positive FITC and/or PI cells were quantified by quadrant analysis and calculated using ACEA NovoExpress software (ACEA Biosciences Inc., San Diego, CA, USA).</div></div><div id="sec4_25_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i127" class="anchor-spacer"></div><h4 class="article-section__title" id="_i127"> Western Blot Analysis</h4><div class="NLM_p last">Immunoblotting was used to determine the antitumor effects of the optimized compounds on the molecular level through the detection of downstream signaling targets of the HER2 pathway. The analysis was conducted according to which we previously described.<a onclick="showRef(event, 'ref101 ref120'); return false;" href="javascript:void(0);" class="ref ref101 ref120">(101,120)</a> Briefly, about 25 μg of cellular protein lysates prepared from the control and the treated AU565 and BT474 cells were resolved onto a 4–20% SDS-PAGE gel (Bio-Rad, Hercules, CA) under reducing conditions. The fractionated proteins are transferred onto a nitrocellulose membrane (Bio-Rad, Hercules, CA) and subsequently blocked with 3% bovine serum albumin for 1 h at room temperature. The blocked membranes were incubated overnight at 4 °C with appropriate primary antibodies raised against HER2, total and phosphorylated Akt, and ERK1/2 (Cell Signaling, Beverly, USA). Lapatinib was used as an authenticated control for the tested compounds and GAPDH was used as a reference loading control (Santa Cruz Biotechnology, Dallas, TX). Cells were treated with different concentrations at IC<sub>50</sub>, 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> for each compound at two different time points (24 and 72 h). After serial washing, the membranes were incubated with the appropriate secondary antibodies for 1 h at room temperature, and the signal was developed by Amersham ECL Prime WB Detection Reagent (GE Healthcare Life Sciences, Pittsburgh, PA) using C-Digit and Odyssey CLX Imaging System (LI-COR Biosciences, Lincoln, NE).</div></div><div id="sec4_25_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i128" class="anchor-spacer"></div><h4 class="article-section__title" id="_i128"> Antitumor Effects of the Most Potent Optimized Scaffolds Using the Preclinical BC Model</h4><div class="NLM_p last">All animal experiments, surgical and dosing procedures, were approved by Texas A&M University, Rangel College of Pharmacy, and consistent with the Institutional Animal Care and Use Committee (IACUC#091604). Two of the most active compounds with the lowest IC<sub>50</sub> against BT474 cell lines were subjected to preclinical studies. Approximately 21 female athymic nude mice (Foxn<sup>1nu</sup>/Foxn<sup>1+</sup>) of 6-week old were obtained from Envigo (Indianapolis, IN, USA); code: 069(nu)/070(nu/+) and maintained with sterilized food and water. These nude mice were used to assess the <i>in vivo</i> anticancer activities as described.<a onclick="showRef(event, 'ref25 ref100'); return false;" href="javascript:void(0);" class="ref ref25 ref100">(25,100)</a> Briefly, 2 × 10<sup>6</sup> triple-positive BT474 cells in a 1:1 serum-free medium were mixed with an equal amount of Matrigel on ice and were subcutaneously inoculated into dorsal flanks of each athymic nude mouse. Then, mice were randomly divided into three groups (<i>n</i> = 7); group I received the vehicle, group II and III received compounds <b>17d</b> and <b>25b</b>, respectively, in sterile saline, 50 mg/kg, by oral gavage 5×/week. Based on an overall statistical significance level of 5%, the power of 80% and effect size of 1.06284, a balanced one-way ANOVA, comparing these three groups would require at least 5 animals per group as previously described.<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> Tumor volume was measured with digital calipers every other day and calculated using the formula: (length × width × depth × 0.5236) and body weight daily. The experiment was terminated, and the animal was sacrificed when the tumor volume reached the maximum allowed size. Treatments were started after the tumor reached 100–150 mm<sup>3</sup> and continued for 55 days. Each animal received about 50 doses.<a onclick="showRef(event, 'ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref80 ref81">(80,81)</a> Animals were then sacrificed, and tumors were collected. H&E and Ki67 immunostaining in tumor sections was performed to assess the effect of the most active candidate.</div></div><div id="sec4_25_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i129" class="anchor-spacer"></div><h4 class="article-section__title" id="_i129"> CD-1 Mouse Microsomal Stability Assay</h4><div class="NLM_p last">The assay was conducted using liver microsomes of CD-1 Mouse Cat no. M1000 Xenotech MLM and according to the protocol of creative bioarray, as shown in detail in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">Supporting Information</a>. Briefly, compounds <b>17d</b> and <b>25b</b>, 5 μL at a final concentration of 100 μM, were incubated with pooled mouse liver microsomes (final concentration 0.56 mg/mL) in phosphate buffer 0.1 M (pH 7.4) in a 37 °C water bath and the reaction was initiated by the addition of an NADPH cofactor. The reaction was allowed for different time points, 5, 15, 30, 45, and 60 min and after quenching of the reactions, each bioanalysis plate was sealed and shaken for 10 min prior to LC–MS/MS analysis. The equation of first-order kinetics was used to calculate <i>T</i><sub>1/2</sub> and CL<sub>int(mic)</sub> (μL/min/mg).</div></div><div id="sec4_25_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i130" class="anchor-spacer"></div><h4 class="article-section__title" id="_i130"> Confirmation of the Compounds’ Purity by the UHPLC/MS Method</h4><div class="NLM_p last">The UHPLC/MS method was developed for the evaluation of final compounds’ purity, using an UltiMate 3000 UHPLC system by Thermo Fisher Scientific. UHPLC-grade methanol and acetonitrile (Thermo Scientific) were used. The mobile phase was filtered through a 0.45 μm membrane filter and then used. Chromatography was carried out on an Accucore C-18 RP column (4.6 mm diameter, 50 mm length, 2.6 μm bead size, Thermo Scientific) using a mixture of methanol and acetonitrile in the ratio of 50:50 (v/v) as mobile phase at a rate of 0.3 mL/min. The detection was carried out at a wavelength of 240 and 254 nm with a run time of 8 min. Stock solutions of the analyzed compounds were prepared by dissolving 10 mg of each in 1 mL of methanol and filtered through a Whatman Syringe Filter (PVDF membrane, pore size 0.45 μm) to get 10 mg/mL solutions. These stock solutions were diluted with methanol to get the working solutions of a concentration of 1 mg/mL. 0.5–2 μL of each working solution was injected into the UPLC system to obtain the chromatogram, and the injection volume used for each sample was selected with consideration of the linear region of relatively uniform response. The area under curve (AUC) value for each analyte was measured, and the relative area percentage was considered as an indicator for sample purity.</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i131"><a href="/doi/suppl/10.1021/acs.jmedchem.0c01647" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48737" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48737" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c01647?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01647</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings for the target compounds: <b>10a–25h</b>, additional discussion of <i>in vitro</i> single-dose and five-dose screening against the NCI 60 cell panel and the selectivity ratio, Annexin V FITC/PI apoptosis assays, average tumor growth rate for mice groups that received the vehicle, compound <b>17d</b>, and <b>25b</b>, the structural optimization using CADD and the docking results into HER2 kinase, and the spectra of <sup>1</sup>H NMR, <sup>13</sup>C DEPT, <sup>13</sup>C NMR, and HRMS for compounds <b>4c–25h</b> and UPLC/MS chromatograms and final analytical reports of compounds (1<b>0a–c</b>,<b>15a</b>,<b>b</b>,<b>c</b>,<b>e</b>,<b>f</b>,<b>i</b>,<b>k</b>,<b>l</b>, <b>17d</b>, <b>and 25a</b>,<b>b</b>,<b>c</b>,<b>e</b>,<b>f</b>,<b>g</b>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Antiproliferative activity of compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_002.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a> dimer (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_003.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a> monomer (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_004.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Optimization of docking validation (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_005.pdb" class="ext-link">PDB</a>)</p></li><li><p class="inline">Analysis of 27 compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_006.pdb" class="ext-link">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf">jm0c01647_si_001.pdf (37.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_002.csv">jm0c01647_si_002.csv (2.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_003.pdb">jm0c01647_si_003.pdb (391.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_004.pdb">jm0c01647_si_004.pdb (191.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_005.pdb">jm0c01647_si_005.pdb (406.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_006.pdb">jm0c01647_si_006.pdb (526.02 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c01647" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35307" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35307" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zakaria Y. Abd Elmageed</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmacology, Edward Via College of Osteopathic Medicine, University of Louisiana at Monroe, Monroe, Louisiana 71203, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#550f3039383432303031152039387b23363a387b303120"><span class="__cf_email__" data-cfemail="a1fbc4cdccc0c6c4c4c5e1d4cdcc8fd7c2cecc8fc4c5d4">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hamed I. Ali</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6939-8462" title="Orcid link">http://orcid.org/0000-0002-6939-8462</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#0564697c6c7668646c6945716468702b606170"><span class="__cf_email__" data-cfemail="ed8c8194849e808c8481ad998c8098c3888998">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tamer A. Elwaie</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Safinaz E. Abbas</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Enayat I. Aly</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Riham F. George</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Chemistry, Faculty of Pharmacy, Cairo University, Cairo 11562, Egypt</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hamdy Ali</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical Sciences, Irma Lerma Rangel College of Pharmacy, Texas A&M University, College Station, Kingsville, Texas 78363, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nikolai Kraiouchkine</span> - <span class="hlFld-Affiliation affiliation">Department
of Physical and Environmental Sciences, Texas A&M University, Corpus
Christi, Texas 78412, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Khaldoun S. Abdelwahed</span> - <span class="hlFld-Affiliation affiliation">School
of Basic Pharmaceutical and Toxicological Sciences, College of Pharmacy, University of Louisiana at Monroe, 1800 Bienville Drive, Monroe, Louisiana 71201, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tamer E. Fandy</span> - <span class="hlFld-Affiliation affiliation">Department
of Pharmaceutical & Administrative Sciences, School of Pharmacy, University of Charleston, Charleston, West Virginia 25304, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Khalid A. El Sayed</span> - <span class="hlFld-Affiliation affiliation">School
of Basic Pharmaceutical and Toxicological Sciences, College of Pharmacy, University of Louisiana at Monroe, 1800 Bienville Drive, Monroe, Louisiana 71201, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1456-4064" title="Orcid link">http://orcid.org/0000-0002-1456-4064</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>Participated in research design: H.I.A. and S.E.A. Conducted experiments: T.A.E., Z.Y.A.E., T.E.F., H.A., and N.K. Performed data analysis: H.I.A., S.E.A., Z.Y.A.E., T.A.E., R.F.G., N.K., Z.Y.A.E., T.E.F., and H.A. Wrote or contributed to the writing of the manuscript: H.I.A., S.E.A., Z.Y.A.E., T.A.E., E.I.A., T.E.F., and H.A.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i133">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62231" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62231" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Start-up Fund by Texas A&M Health Sciences Center (to H.I.A. grant number 121500-35558). It was also supported in part by the scholarship offered to T.A.E. by the Department of Missions, Ministry of higher education in Egypt. The authors wish to offer their deep gratitude to Mohamed Gaballah and Rofaida Gaballa for their generous help in conducting the antiproliferative and Western blot assays. We would also like to thank the staff at the vivarium of Texas A&M-Rangel College of pharmacy with special gratitude to Jennifer Santos (Veterinary Technician IV). Special gratitude to Mr. Daniel Freeman, Multimedia Project Coordinator for improving the quality of the figures in this manuscript.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">BC</td><td class="NLM_def"><p class="first last">breast cancer</p></td></tr><tr><td class="NLM_term">EGFR</td><td class="NLM_def"><p class="first last">epidermal growth factor receptor-2</p></td></tr><tr><td class="NLM_term">HER2</td><td class="NLM_def"><p class="first last">human epidermal growth factor receptor-2</p></td></tr><tr><td class="NLM_term">TKIs</td><td class="NLM_def"><p class="first last">tyrosine kinase inhibitors</p></td></tr><tr><td class="NLM_term">NSCLC</td><td class="NLM_def"><p class="first last">nonsmall-cell lung carcinoma</p></td></tr><tr><td class="NLM_term">RTKs</td><td class="NLM_def"><p class="first last">receptor tyrosine kinases</p></td></tr><tr><td class="NLM_term">ADP-Glo</td><td class="NLM_def"><p class="first last">adenosine diphosphate assay using ADP-Glo kits (ADP-Glo, Promega Corp.)</p></td></tr><tr><td class="NLM_term">[γ-<sup>32</sup>P]ATP</td><td class="NLM_def"><p class="first last">adenosine 5′-triphosphate-gamma-<sup>32</sup>P</p></td></tr><tr><td class="NLM_term">TBAB</td><td class="NLM_def"><p class="first last">tetra-<i>n</i>-butylammonium bromide</p></td></tr><tr><td class="NLM_term"><sup>13</sup>C DEPT</td><td class="NLM_def"><p class="first last"><sup>13</sup>C distortionless enhancement by polarization transfer</p></td></tr><tr><td class="NLM_term">HRMS</td><td class="NLM_def"><p class="first last">high-resolution mass spectrometry</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">CADD</td><td class="NLM_def"><p class="first last">computer-aided drug design</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i135">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83310" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83310" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 120 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fitzmaurice, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dicker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamavid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradi-Lakeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodbrook, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamadeh, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werdecker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessner, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Te Ao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karimkhani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwebel, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Leo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plass, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ukwaja, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurston, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun Jin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catalá-López, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deVeber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosgood, H. D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leasher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leigh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanabria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beardsley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronfani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naldi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geleijnse, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrime, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitborde, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourmalek, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotufo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteghamati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankey, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunevicius, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malekzadeh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellavalle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas-Rueda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepanlou, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichenthal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abera, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fereshtehnejad, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiue, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasankari, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsharif, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahimi-Movaghar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlassov, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcenes, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mekonnen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melaku, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLachlan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trillini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshrati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandona, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amare, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castañeda-Orjuela, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gool, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Violante, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, I.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deribe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soreide, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knibbs, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kereselidze, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monasta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikhbahaei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafezi-Nejad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreeramareddy, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandona, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollset, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mokdad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salomon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forouzanfar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naghavi, M.</span></span> <span> </span><span class="NLM_article-title">The global burden of cancer 2013</span>. <i>JAMA Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1001/jamaoncol.2015.0735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1001%2Fjamaoncol.2015.0735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26181261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC28%252Fkt1Wksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=505-527&author=C.+Fitzmauriceauthor=D.+Dickerauthor=A.+Painauthor=H.+Hamavidauthor=M.+Moradi-Lakehauthor=M.+F.+MacIntyreauthor=C.+Allenauthor=G.+Hansenauthor=R.+Woodbrookauthor=C.+Wolfeauthor=R.+R.+Hamadehauthor=A.+Mooreauthor=A.+Werdeckerauthor=B.+D.+Gessnerauthor=B.+Te+Aoauthor=B.+McMahonauthor=C.+Karimkhaniauthor=C.+Yuauthor=G.+S.+Cookeauthor=D.+C.+Schwebelauthor=D.+O.+Carpenterauthor=D.+M.+Pereiraauthor=D.+Nashauthor=D.+S.+Kaziauthor=D.+De+Leoauthor=D.+Plassauthor=K.+N.+Ukwajaauthor=G.+D.+Thurstonauthor=K.+Yun+Jinauthor=E.+P.+Simardauthor=E.+Millsauthor=E.-K.+Parkauthor=F.+Catal%C3%A1-L%C3%B3pezauthor=G.+deVeberauthor=C.+Gotayauthor=G.+Khanauthor=H.+D.+Hosgoodauthor=I.+S.+Santosauthor=J.+L.+Leasherauthor=J.+Singhauthor=J.+Leighauthor=J.+B.+Jonasauthor=J.+Sanabriaauthor=J.+Beardsleyauthor=K.+H.+Jacobsenauthor=K.+Takahashiauthor=R.+C.+Franklinauthor=L.+Ronfaniauthor=M.+Monticoauthor=L.+Naldiauthor=M.+Tonelliauthor=J.+Geleijnseauthor=M.+Petzoldauthor=M.+G.+Shrimeauthor=M.+Younisauthor=N.+Yonemotoauthor=N.+Breitbordeauthor=P.+Yipauthor=F.+Pourmalekauthor=P.+A.+Lotufoauthor=A.+Esteghamatiauthor=G.+J.+Hankeyauthor=R.+Aliauthor=R.+Luneviciusauthor=R.+Malekzadehauthor=R.+Dellavalleauthor=R.+Weintraubauthor=R.+Lucasauthor=R.+Hayauthor=D.+Rojas-Ruedaauthor=R.+Westermanauthor=S.+G.+Sepanlouauthor=S.+Nolteauthor=S.+Pattenauthor=S.+Weichenthalauthor=S.+F.+Aberaauthor=S.-M.+Fereshtehnejadauthor=I.+Shiueauthor=T.+Driscollauthor=T.+Vasankariauthor=U.+Alsharifauthor=V.+Rahimi-Movagharauthor=V.+V.+Vlassovauthor=W.+S.+Marcenesauthor=W.+Mekonnenauthor=Y.+A.+Melakuauthor=Y.+Yanoauthor=A.+Artamanauthor=I.+Camposauthor=J.+MacLachlanauthor=U.+Muellerauthor=D.+Kimauthor=M.+Trilliniauthor=B.+Eshratiauthor=H.+C.+Williamsauthor=K.+Shibuyaauthor=R.+Dandonaauthor=K.+Murthyauthor=B.+Cowieauthor=A.+T.+Amareauthor=C.+A.+Antonioauthor=C.+Casta%C3%B1eda-Orjuelaauthor=C.+H.+van+Goolauthor=F.+Violanteauthor=I.-H.+Ohauthor=K.+Deribeauthor=K.+Soreideauthor=L.+Knibbsauthor=M.+Kereselidzeauthor=M.+Greenauthor=R.+Cardenasauthor=N.+Royauthor=T.+Tillmannauthor=Y.+Liauthor=H.+Kruegerauthor=L.+Monastaauthor=S.+Deyauthor=S.+Sheikhbahaeiauthor=N.+Hafezi-Nejadauthor=G.+A.+Kumarauthor=C.+T.+Sreeramareddyauthor=L.+Dandonaauthor=H.+Wangauthor=S.+E.+Vollsetauthor=A.+Mokdadauthor=J.+A.+Salomonauthor=R.+Lozanoauthor=T.+Vosauthor=M.+Forouzanfarauthor=A.+Lopezauthor=C.+Murrayauthor=M.+Naghavi&title=The+global+burden+of+cancer+2013&doi=10.1001%2Fjamaoncol.2015.0735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Global Burden of Cancer 2013</span></div><div class="casAuthors">Jonas Jost; Tillman Taavi; Fitzmaurice Christina; Dicker Daniel; Pain Amanda; Hamavid Hannah; MacIntyre Michael F; Allen Christine; Hansen Gillian; Woodbrook Rachel; Wang Haidong; Mokdad Ali; Vos Theo; Forouzanfar Mohammad; Murray Christopher; Naghavi Mohsen; Moradi-Lakeh Maziar; Wolfe Charles; Hamadeh Randah R; Moore Ami; Werdecker Andrea; Gessner Bradford D; Te Ao Braden; McMahon Brian; Karimkhani Chante; Yu Chuanhua; Cooke Graham S; Schwebel David C; Carpenter David O; Pereira David M; Nash Denis; Kazi Dhruv S; De Leo Diego; Plass Dietrich; Ukwaja Kingsley N; Thurston George D; Yun Jin Kim; Simard Edgar P; Mills Edward; Park Eun-Kee; Catala-Lopez Ferran; deVeber Gabrielle; Gotay Carolyn; Krueger Hans; Khan Gulfaraz; Hosgood H Dean 3rd; Santos Itamar S; Lotufo Paulo A; Leasher Janet L; Singh Jasvinder; Leigh James; Jonas Jost B; Sanabria Juan; Beardsley Justin; Jacobsen Kathryn H; Takahashi Ken; Franklin Richard C; Ronfani Luca; Montico Marcella; Monasta Lorenzo; Naldi Luigi; Tonelli Marcello; Geleijnse Johanna; Petzold Max; Shrime Mark G; Campos Ismael; Salomon Joshua A; Younis Mustafa; Yonemoto Naohiro; Breitborde Nicholas; Yip Paul; Pourmalek Farshad; Esteghamati Alireza; Sheikhbahaei Sara; Hafezi-Nejad Nima; Hankey Graeme J; Ali Raghib; Lunevicius Raimundas; Malekzadeh Reza; Dellavalle Robert; Weintraub Robert; Lucas Robyn; Hay Roderick; Rojas-Rueda David; Westerman Ronny; Sepanlou Sadaf G; Nolte Sandra; Patten Scott; Weichenthal Scott; Abera Semaw Ferede; Fereshtehnejad Seyed-Mohammad; Shiue Ivy; Driscoll Tim; Vasankari Tommi; Alsharif Ubai; Rahimi-Movaghar Vafa; Vlassov Vasiliy V; Marcenes W S; Mekonnen Wubegzier; Melaku Yohannes Adama; Yano Yuichiro; Artaman Al; MacLachlan Jennifer; Cowie Benjamin; Mueller Ulrich; Kim Daniel; Trillini Matias; Eshrati Babak; Williams Hywel C; Shibuya Kenji; Dandona Rakhi; Murthy Kinnari; Kumar G Anil; Amare Azmeraw T; Antonio Carl Abelardo; Castaneda-Orjuela Carlos; van Gool Coen H; Violante Francesco; Oh In-Hwan; Deribe Kedede; Soreide Kjetil; Knibbs Luke; Kereselidze Maia; Green Mark; Cardenas Rosario; Roy Nobhojit; Tillmann Taavi; Li Yongmei; Dey Subhojit; Sreeramareddy Chandrashekhar T; Dandona Lalit; Vollset Stein Emil; Lozano Rafael; Lopez Alan</div><div class="citationInfo"><span class="NLM_cas:title">JAMA oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">IMPORTANCE:  Cancer is among the leading causes of death worldwide.  Current estimates of cancer burden in individual countries and regions are necessary to inform local cancer control strategies.  OBJECTIVE:  To estimate mortality, incidence, years lived with disability (YLDs), years of life lost (YLLs), and disability-adjusted life-years (DALYs) for 28 cancers in 188 countries by sex from 1990 to 2013.  EVIDENCE REVIEW:  The general methodology of the Global Burden of Disease (GBD) 2013 study was used.  Cancer registries were the source for cancer incidence data as well as mortality incidence (MI) ratios.  Sources for cause of death data include vital registration system data, verbal autopsy studies, and other sources.  The MI ratios were used to transform incidence data to mortality estimates and cause of death estimates to incidence estimates.  Cancer prevalence was estimated using MI ratios as surrogates for survival data; YLDs were calculated by multiplying prevalence estimates with disability weights, which were derived from population-based surveys; YLLs were computed by multiplying the number of estimated cancer deaths at each age with a reference life expectancy; and DALYs were calculated as the sum of YLDs and YLLs.  FINDINGS:  In 2013 there were 14.9 million incident cancer cases, 8.2 million deaths, and 196.3 million DALYs.  Prostate cancer was the leading cause for cancer incidence (1.4 million) for men and breast cancer for women (1.8 million).  Tracheal, bronchus, and lung (TBL) cancer was the leading cause for cancer death in men and women, with 1.6 million deaths.  For men, TBL cancer was the leading cause of DALYs (24.9 million).  For women, breast cancer was the leading cause of DALYs (13.1 million).  Age-standardized incidence rates (ASIRs) per 100 000 and age-standardized death rates (ASDRs) per 100 000 for both sexes in 2013 were higher in developing vs developed countries for stomach cancer (ASIR, 17 vs 14; ASDR, 15 vs 11), liver cancer (ASIR, 15 vs 7; ASDR, 16 vs 7), esophageal cancer (ASIR, 9 vs 4; ASDR, 9 vs 4), cervical cancer (ASIR, 8 vs 5; ASDR, 4 vs 2), lip and oral cavity cancer (ASIR, 7 vs 6; ASDR, 2 vs 2), and nasopharyngeal cancer (ASIR, 1.5 vs 0.4; ASDR, 1.2 vs 0.3).  Between 1990 and 2013, ASIRs for all cancers combined (except nonmelanoma skin cancer and Kaposi sarcoma) increased by more than 10% in 113 countries and decreased by more than 10% in 12 of 188 countries.  CONCLUSIONS AND RELEVANCE:  Cancer poses a major threat to public health worldwide, and incidence rates have increased in most countries since 1990.  The trend is a particular threat to developing nations with health systems that are ill-equipped to deal with complex and expensive cancer treatments.  The annual update on the Global Burden of Cancer will provide all stakeholders with timely estimates to guide policy efforts in cancer prevention, screening, treatment, and palliation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQIAyANjE4GzBfKb6ELK8t0fW6udTcc2eaH6hfmgu5mh7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252Fkt1Wksw%253D%253D&md5=7466761111f16c35c0953901258d0587</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1001%2Fjamaoncol.2015.0735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjamaoncol.2015.0735%26sid%3Dliteratum%253Aachs%26aulast%3DFitzmaurice%26aufirst%3DC.%26aulast%3DDicker%26aufirst%3DD.%26aulast%3DPain%26aufirst%3DA.%26aulast%3DHamavid%26aufirst%3DH.%26aulast%3DMoradi-Lakeh%26aufirst%3DM.%26aulast%3DMacIntyre%26aufirst%3DM.%2BF.%26aulast%3DAllen%26aufirst%3DC.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DWoodbrook%26aufirst%3DR.%26aulast%3DWolfe%26aufirst%3DC.%26aulast%3DHamadeh%26aufirst%3DR.%2BR.%26aulast%3DMoore%26aufirst%3DA.%26aulast%3DWerdecker%26aufirst%3DA.%26aulast%3DGessner%26aufirst%3DB.%2BD.%26aulast%3DTe%2BAo%26aufirst%3DB.%26aulast%3DMcMahon%26aufirst%3DB.%26aulast%3DKarimkhani%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DCooke%26aufirst%3DG.%2BS.%26aulast%3DSchwebel%26aufirst%3DD.%2BC.%26aulast%3DCarpenter%26aufirst%3DD.%2BO.%26aulast%3DPereira%26aufirst%3DD.%2BM.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DKazi%26aufirst%3DD.%2BS.%26aulast%3DDe%2BLeo%26aufirst%3DD.%26aulast%3DPlass%26aufirst%3DD.%26aulast%3DUkwaja%26aufirst%3DK.%2BN.%26aulast%3DThurston%26aufirst%3DG.%2BD.%26aulast%3DYun%2BJin%26aufirst%3DK.%26aulast%3DSimard%26aufirst%3DE.%2BP.%26aulast%3DMills%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DE.-K.%26aulast%3DCatal%25C3%25A1-L%25C3%25B3pez%26aufirst%3DF.%26aulast%3DdeVeber%26aufirst%3DG.%26aulast%3DGotay%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DG.%26aulast%3DHosgood%26aufirst%3DH.%2BD.%26aulast%3DSantos%26aufirst%3DI.%2BS.%26aulast%3DLeasher%26aufirst%3DJ.%2BL.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DLeigh%26aufirst%3DJ.%26aulast%3DJonas%26aufirst%3DJ.%2BB.%26aulast%3DSanabria%26aufirst%3DJ.%26aulast%3DBeardsley%26aufirst%3DJ.%26aulast%3DJacobsen%26aufirst%3DK.%2BH.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DFranklin%26aufirst%3DR.%2BC.%26aulast%3DRonfani%26aufirst%3DL.%26aulast%3DMontico%26aufirst%3DM.%26aulast%3DNaldi%26aufirst%3DL.%26aulast%3DTonelli%26aufirst%3DM.%26aulast%3DGeleijnse%26aufirst%3DJ.%26aulast%3DPetzold%26aufirst%3DM.%26aulast%3DShrime%26aufirst%3DM.%2BG.%26aulast%3DYounis%26aufirst%3DM.%26aulast%3DYonemoto%26aufirst%3DN.%26aulast%3DBreitborde%26aufirst%3DN.%26aulast%3DYip%26aufirst%3DP.%26aulast%3DPourmalek%26aufirst%3DF.%26aulast%3DLotufo%26aufirst%3DP.%2BA.%26aulast%3DEsteghamati%26aufirst%3DA.%26aulast%3DHankey%26aufirst%3DG.%2BJ.%26aulast%3DAli%26aufirst%3DR.%26aulast%3DLunevicius%26aufirst%3DR.%26aulast%3DMalekzadeh%26aufirst%3DR.%26aulast%3DDellavalle%26aufirst%3DR.%26aulast%3DWeintraub%26aufirst%3DR.%26aulast%3DLucas%26aufirst%3DR.%26aulast%3DHay%26aufirst%3DR.%26aulast%3DRojas-Rueda%26aufirst%3DD.%26aulast%3DWesterman%26aufirst%3DR.%26aulast%3DSepanlou%26aufirst%3DS.%2BG.%26aulast%3DNolte%26aufirst%3DS.%26aulast%3DPatten%26aufirst%3DS.%26aulast%3DWeichenthal%26aufirst%3DS.%26aulast%3DAbera%26aufirst%3DS.%2BF.%26aulast%3DFereshtehnejad%26aufirst%3DS.-M.%26aulast%3DShiue%26aufirst%3DI.%26aulast%3DDriscoll%26aufirst%3DT.%26aulast%3DVasankari%26aufirst%3DT.%26aulast%3DAlsharif%26aufirst%3DU.%26aulast%3DRahimi-Movaghar%26aufirst%3DV.%26aulast%3DVlassov%26aufirst%3DV.%2BV.%26aulast%3DMarcenes%26aufirst%3DW.%2BS.%26aulast%3DMekonnen%26aufirst%3DW.%26aulast%3DMelaku%26aufirst%3DY.%2BA.%26aulast%3DYano%26aufirst%3DY.%26aulast%3DArtaman%26aufirst%3DA.%26aulast%3DCampos%26aufirst%3DI.%26aulast%3DMacLachlan%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DTrillini%26aufirst%3DM.%26aulast%3DEshrati%26aufirst%3DB.%26aulast%3DWilliams%26aufirst%3DH.%2BC.%26aulast%3DShibuya%26aufirst%3DK.%26aulast%3DDandona%26aufirst%3DR.%26aulast%3DMurthy%26aufirst%3DK.%26aulast%3DCowie%26aufirst%3DB.%26aulast%3DAmare%26aufirst%3DA.%2BT.%26aulast%3DAntonio%26aufirst%3DC.%2BA.%26aulast%3DCasta%25C3%25B1eda-Orjuela%26aufirst%3DC.%26aulast%3Dvan%2BGool%26aufirst%3DC.%2BH.%26aulast%3DViolante%26aufirst%3DF.%26aulast%3DOh%26aufirst%3DI.-H.%26aulast%3DDeribe%26aufirst%3DK.%26aulast%3DSoreide%26aufirst%3DK.%26aulast%3DKnibbs%26aufirst%3DL.%26aulast%3DKereselidze%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DCardenas%26aufirst%3DR.%26aulast%3DRoy%26aufirst%3DN.%26aulast%3DTillmann%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKrueger%26aufirst%3DH.%26aulast%3DMonasta%26aufirst%3DL.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DSheikhbahaei%26aufirst%3DS.%26aulast%3DHafezi-Nejad%26aufirst%3DN.%26aulast%3DKumar%26aufirst%3DG.%2BA.%26aulast%3DSreeramareddy%26aufirst%3DC.%2BT.%26aulast%3DDandona%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DVollset%26aufirst%3DS.%2BE.%26aulast%3DMokdad%26aufirst%3DA.%26aulast%3DSalomon%26aufirst%3DJ.%2BA.%26aulast%3DLozano%26aufirst%3DR.%26aulast%3DVos%26aufirst%3DT.%26aulast%3DForouzanfar%26aufirst%3DM.%26aulast%3DLopez%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DNaghavi%26aufirst%3DM.%26atitle%3DThe%2520global%2520burden%2520of%2520cancer%25202013%26jtitle%3DJAMA%2520Oncol.%26date%3D2015%26volume%3D1%26spage%3D505%26epage%3D527%26doi%3D10.1001%2Fjamaoncol.2015.0735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heron, M.</span></span> <span> </span><span class="NLM_article-title">Deaths : leading causes for 2008</span>. <i>Natl. Vital Stat. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22827019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC38fisFOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=1-94&author=M.+Heron&title=Deaths+%3A+leading+causes+for+2008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Deaths: leading causes for 2008</span></div><div class="casAuthors">Heron Melonie</div><div class="citationInfo"><span class="NLM_cas:title">National vital statistics reports : from the Centers for Disease Control and Prevention, National Center for Health Statistics, National Vital Statistics System</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1-94</span>
        ISSN:<span class="NLM_cas:issn">1551-8922</span>.
    </div><div class="casAbstract">OBJECTIVES:  This report presents final 2008 data on the 10 leading causes of death in the United States by age, sex, race, and Hispanic origin.  Leading causes of infant, neonatal, and postneonatal death are also presented.  This report supplements the Division of Vital Statistics' annual report of final mortality statistics.  METHODS:  Data in this report are based on information from all death certificates filed in the 50 states and the District of Columbia in 2008.  Causes of death classified by the International Classification of Diseases, Tenth Revision (ICD-10) are ranked according to the number of deaths assigned to rankable causes.  Cause-of-death statistics are based on the underlying cause of death.  RESULTS:  in 2008, the 10 leading causes of death were, in rank order: Diseases of heart; Malignant neoplasms; Chronic lower respiratory diseases; Cerebrovascular diseases; Accidents (unintentional injuries); Alzheimer's disease; Diabetes mellitus; Influenza and pneumonia; Nephritis, nephrotic syndrome and nephrosis; and Intentional self-harm (suicide).  They accounted for approximately 76 percent of all deaths occurring in the United States.  Differences in the rankings are evident by age, sex, race, and Hispanic origin.  Leading causes of infant death for 2008 were, in rank order: Congenital malformations, deformations and chromosomal abnormalities; Disorders related to short gestation and low birth weight, not elsewhere classified; Sudden infant death syndrome; Newborn affected by maternal complications of pregnancy; Accidents (unintentional injuries); Newborn affected by complications of placenta, cord and membranes; Bacterial sepsis of newborn; Respiratory distress of newborn; Diseases of the circulatory system; and Neonatal hemorrhage.  Important variations in the leading causes of infant death are noted for the neonatal and postneonatal periods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRrmI3hG59C2Zyxfu64m4ffW6udTcc2eblVc3KL5owU7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fisFOjsQ%253D%253D&md5=024b685faf95755d47dc8f80ffce28a3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeron%26aufirst%3DM.%26atitle%3DDeaths%2520%253A%2520leading%2520causes%2520for%25202008%26jtitle%3DNatl.%2520Vital%2520Stat.%2520Rep.%26date%3D2012%26volume%3D60%26spage%3D1%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fekry, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezzat, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salama, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshehri, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span> <span> </span><span class="NLM_article-title">Bioactive glycoalkaloides isolated from Solanum melongena fruit peels with potential anticancer properties against hepatocellular carcinoma cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1746</span>– <span class="NLM_lpage">1756</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-36089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-36089-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30741973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3cfht1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1746-1756&author=M.+I.+Fekryauthor=S.+M.+Ezzatauthor=M.+M.+Salamaauthor=O.+Y.+Alshehriauthor=A.+M.+Al-Abd&title=Bioactive+glycoalkaloides+isolated+from+Solanum+melongena+fruit+peels+with+potential+anticancer+properties+against+hepatocellular+carcinoma+cells&doi=10.1038%2Fs41598-018-36089-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive glycoalkaloides isolated from Solanum melongena fruit peels with potential anticancer properties against hepatocellular carcinoma cells</span></div><div class="casAuthors">Fekry Mostafa I; Ezzat Shahira M; Salama Maha M; Fekry Mostafa I; Ezzat Shahira M; Salama Maha M; Alshehri Ohoud Y; Al-Abd Ahmed M; Al-Abd Ahmed M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1746</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is progressively increasing tumor with lack of accurate prognosis and inadequate systemic treatment approaches.  Solanum sp. (such as Solanum melongena) is a folk herb which is reported to possess anticancer properties.  In a continuity for our interest in pursuing the anticancer activity of compounds isolated from the fruit peels of Solanum melongena, the HPLC profiling and ESI-MS assessment for the methanolic extract evidenced the presence of bioactive glycoalkaloids (solasonine, solasodine and solamargine).  These glycoalkaloids were isolated, purified and proved to possess in vitro cytotoxicity against human liver cancer cell lines (Huh7 and HepG2).  Herein, we investigated the potential mechanism of action of these compounds using DNA content flow-cytometry and apoptosis/necrosis differential anaylsis using annexin-V/FITC staining.  Solasonine, solasodine and solamargine inducd significant antiproliferative effect against liver cancer cells (Huh7 and HepG2) which was attributed to cell cycle arrest at S-phase.  Solamargine, solasodine and solasonine induced significant apoptosis in Huh7 cells.  Only solamargine-induced cell cycle arrest, was reflected as apoptotic cell killing effect against HepG2 cells.  In conclusion, glycoalkaloids derived from Solanum melongena and particularly, solamargine are promising antiproliferative agents with potential anticancer effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVXWE2RUz1qENoizSOkG_FfW6udTcc2eYkkgetMDs1W7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfht1GmtA%253D%253D&md5=caa0c90a3d84a3a920915973a4736e58</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-36089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-36089-6%26sid%3Dliteratum%253Aachs%26aulast%3DFekry%26aufirst%3DM.%2BI.%26aulast%3DEzzat%26aufirst%3DS.%2BM.%26aulast%3DSalama%26aufirst%3DM.%2BM.%26aulast%3DAlshehri%26aufirst%3DO.%2BY.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26atitle%3DBioactive%2520glycoalkaloides%2520isolated%2520from%2520Solanum%2520melongena%2520fruit%2520peels%2520with%2520potential%2520anticancer%2520properties%2520against%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D1746%26epage%3D1756%26doi%3D10.1038%2Fs41598-018-36089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bashmail, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamoudi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noorwali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegazy, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span> <span> </span><span class="NLM_article-title">Thymoquinone synergizes gemcitabine anti-breast cancer activity via modulating its apoptotic and autophagic activities</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">11674</span>– <span class="NLM_lpage">11684</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-30046-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-30046-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30076320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3c7kvFOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=11674-11684&author=H.+A.+Bashmailauthor=A.+A.+Alamoudiauthor=A.+Noorwaliauthor=G.+A.+Hegazyauthor=H.+Choudhryauthor=A.+M.+Al-Abd&title=Thymoquinone+synergizes+gemcitabine+anti-breast+cancer+activity+via+modulating+its+apoptotic+and+autophagic+activities&doi=10.1038%2Fs41598-018-30046-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Thymoquinone synergizes gemcitabine anti-breast cancer activity via modulating its apoptotic and autophagic activities</span></div><div class="casAuthors">Bashmail Hanan A; Alamoudi Aliaa A; Noorwali Abdulwahab; Hegazy Gehan A; AJabnoor Ghada; Alamoudi Aliaa A; Noorwali Abdulwahab; Hegazy Gehan A; Choudhry Hani; Al-Abd Ahmed M; Al-Abd Ahmed M; Al-Abd Ahmed M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11674</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of anti-cancer adjuvant therapy is rationalized by potentiating the efficacy, and/or protecting from major side effects of chemotherapeutics.  Thymoquinone (TQ) is a naturally occurring compound with cumulative evidence of anti-cancer properties.  In this study, we assessed the chemomodulatory potential of TQ to gemcitabine (GCB) against human breast adenocarcinoma (MCF-7), and ductal carcinoma (T47D) cells.  TQ showed cytotoxic effects against MCF-7 and T47D with IC50's of 64.9 ± 14 μM and 165 ± 2 μM, respectively.  The IC50's of GCB against MCF-7 and T47D were 0.9 ± 0.18 μM and 14.3 ± 2.8 μM and were significantly reduced after combination with TQ to 0.058 ± 12 μM and 2.3 ± 0.2 μM, respectively.  The CI- values were indicative of synergism in MCF-7 and T47D cells (0.15 and 0.30, respectively).  Further investigation showed that GCB caused significant anti-proliferative effect reflected by increasing cell population in S-phase in both cell lines.  TQ potentiated GCB-induced anti-proliferative activity in both cell lines.  GCB induced considerable apoptosis in T47D cell line, and TQ significantly increased GCB-induced apoptotic effects by 1.5 to 3.6 folds.  Interestingly, GCB, TQ and their combination induced significant autophagic cell death in the apoptosis defected MCF-7 cells.  In addition, TQ, GCB and their combination depleted breast cancer associated stem cell (CD44((+))/CD24((-)/(low))) clone within MCF-7 and T47D cells by 3.8% to 27.5%.  In conclusion, TQ showed promising chemomodulatory effects to GCB against breast cancer cells via inducing apoptosis, necrosis and autophagy, in addition to depleting tumor associated resistant stem cell fraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTe4YmZ7rOGXN0Q2ypr7jHkfW6udTcc2eYkkgetMDs1W7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7kvFOlug%253D%253D&md5=5a30945cde5e3c49b7051e6b85577752</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-30046-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-30046-z%26sid%3Dliteratum%253Aachs%26aulast%3DBashmail%26aufirst%3DH.%2BA.%26aulast%3DAlamoudi%26aufirst%3DA.%2BA.%26aulast%3DNoorwali%26aufirst%3DA.%26aulast%3DHegazy%26aufirst%3DG.%2BA.%26aulast%3DChoudhry%26aufirst%3DH.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26atitle%3DThymoquinone%2520synergizes%2520gemcitabine%2520anti-breast%2520cancer%2520activity%2520via%2520modulating%2520its%2520apoptotic%2520and%2520autophagic%2520activities%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D11674%26epage%3D11684%26doi%3D10.1038%2Fs41598-018-30046-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Halawany, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdallah, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, S. R. M.</span></span> <span> </span><span class="NLM_article-title">New xanthones and cytotoxic constituents from Garcinia mangostana fruit hulls against human hepatocellular, breast, and colorectal cancer cell lines</span>. <i>J. Ethnopharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">302</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1016/j.jep.2017.01.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jep.2017.01.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28108382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=302-312&author=G.+A.+Mohamedauthor=A.+M.+Al-Abdauthor=A.+M.+El-Halawanyauthor=H.+M.+Abdallahauthor=S.+R.+M.+Ibrahim&title=New+xanthones+and+cytotoxic+constituents+from+Garcinia+mangostana+fruit+hulls+against+human+hepatocellular%2C+breast%2C+and+colorectal+cancer+cell+lines&doi=10.1016%2Fj.jep.2017.01.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">New xanthones and cytotoxic constituents from Garcinia mangostana fruit hulls against human hepatocellular, breast, and colorectal cancer cell lines</span></div><div class="casAuthors">Mohamed, Gamal A.; Al-Abd, Ahmed M.; El-Halawany, Ali M.; Abdallah, Hossam M.; Ibrahim, Sabrin R. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Ethnopharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">302-312</span>CODEN:
                <span class="NLM_cas:coden">JOETD7</span>;
        ISSN:<span class="NLM_cas:issn">0378-8741</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Cancer has proceeded to surpass one of the most chronic illnesses to be the major cause of mortality in both the developing and developed world.  Garcinia mangostana L. (mangosteen, family Guttiferae) known as the queen of fruits, is one of the most popular tropical fruits.  It is cultivated in Southeast Asian countries: Malaysia, Indonesia, Sri Lanka, Burma, Thailand, and Philippines.  Traditionally, numerous parts of G. mangostana have been utilized to treat various ailments such as abdominal pain, haemorrhoids, food allergies, arthritis, leucorrhoea, gonorrhea, diarrhea, dysentery, wound infection, suppuration, and chronic ulcer.  Although anticancer activity has been reported for the plant, the goal of the study was designed to isolate and characterize the active metabolites from G. mangostana and measure their cytotoxic properties.  In this research, the mechanism of antiproliferative/cytotoxic effects of the tested compds. was investigated.  The CHCl3 fraction of the air-dried fruit hulls was repeatedly chromatographed on SiO2, RP18, Diaion HP-20, and polyamide columns to furnish fourteen compds.  The structures of these metabolites were proven by UV, IR, 1D, and 2D NMR measurements and HRESIMS.  Addnl., the cytotoxic potential of all compds. was assessed against MCF-7, HCT-116, and HepG2 cell lines using SRB-U assay.  Antiproliferative and cell cycle interference effects of potentially potent compds. were tested using DNA content flow cytometry.  The mechanism of cell death induction was also studied using annexin-V/PI differential staining coupled with flow cytometry.  The CHCl3 sol. fraction afforded two new xanthones: mangostanaxanthones V (1) and VI (2), along with twelve known compds.: mangostanaxanthone IV (3), β-mangostin (4), garcinone E (5), α-mangostin (6), nor-mangostin (7), garcimangosone D (8), aromadendrin-8-C-β-D-glucopyranoside (9), 1,2,4,5-tetrahydroxybenzene (10), 2,4,3'-trihydroxybenzophenone-6-O-β-glucopyranoside (11), maclurin-6-O-β-D-glucopyranoside (rhodanthenone) (12), epicatechin (13), and 2,4,6,3',5'-pentahydroxybenzophenone (14).  Only compd. 5 showed considerable antiproliferative/cytotoxic effects with IC50's ranging from 15.8 to 16.7 μM.  Compds. 3, 4, and 6 showed moderate to weak cytotoxic effects (IC50's ranged from 45.7 to 116.4 μM).  Using DNA content flow cytometry, it was found that only 5 induced significant cell cycle arrest at G0/G1-phase which is indicative of its antiproliferative properties.  Addnl., by using annexin V-FITC/PI differential staining, 5 induced cells killing effect via the induction of apoptosis and necrosis in both HepG2 and HCT116 cells.  Compd. 3 produce necrosis and apoptosis only in HCT116 cells.  On contrary, 6 induced apoptosis and necrosis in HepG2 cells and moderate necrosis in HCT116 cells.  Fourteen compds. were isolated from chloroform fraction of G. mangostana fruit hulls.  Cytotoxic properties exhibited by the isolated xanthones from G. mangostana reinforce the avail of it as a natural cytotoxic agent against various cancers.  These evidences could provide relevant bases for the scientific rationale of using G. mangostana in anti-cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbDuWzpsHds7Vg90H21EOLACvtfcHk0lju5OjQRPcklw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GktL0%253D&md5=30b3e0075ece49cdeb641a341bb29ffd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jep.2017.01.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jep.2017.01.030%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DG.%2BA.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26aulast%3DEl-Halawany%26aufirst%3DA.%2BM.%26aulast%3DAbdallah%26aufirst%3DH.%2BM.%26aulast%3DIbrahim%26aufirst%3DS.%2BR.%2BM.%26atitle%3DNew%2520xanthones%2520and%2520cytotoxic%2520constituents%2520from%2520Garcinia%2520mangostana%2520fruit%2520hulls%2520against%2520human%2520hepatocellular%252C%2520breast%252C%2520and%2520colorectal%2520cancer%2520cell%2520lines%26jtitle%3DJ.%2520Ethnopharmacol.%26date%3D2017%26volume%3D198%26spage%3D302%26epage%3D312%26doi%3D10.1016%2Fj.jep.2017.01.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapierre, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heys, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malone, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wintringer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farhat, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soolingen, D. v.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supply, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahseen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arandjelovic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peto, T. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crook, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, Z.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe</span>. <i>Wellcome Open Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.12688/wellcomeopenres.15603.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.12688%2Fwellcomeopenres.15603.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=32055708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB387hvVOntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=191-222&author=M.+Huntauthor=P.+Bradleyauthor=S.+G.+Lapierreauthor=S.+Heysauthor=M.+Thomsitauthor=M.+B.+Hallauthor=K.+M.+Maloneauthor=P.+Wintringerauthor=T.+M.+Walkerauthor=D.+M.+Cirilloauthor=I.+Comasauthor=M.+R.+Farhatauthor=P.+Fowlerauthor=J.+Gardyauthor=N.+Ismailauthor=T.+A.+Kohlauthor=V.+Mathysauthor=M.+Merkerauthor=S.+Niemannauthor=S.+V.+Omarauthor=V.+Sintchenkoauthor=G.+Smithauthor=D.+v.+Soolingenauthor=P.+Supplyauthor=S.+Tahseenauthor=M.+Wilcoxauthor=I.+Arandjelovicauthor=T.+E.+A.+Petoauthor=D.+W.+Crookauthor=Z.+Iqbal&title=Antibiotic+resistance+prediction+for+Mycobacterium+tuberculosis+from+genome+sequence+data+with+Mykrobe&doi=10.12688%2Fwellcomeopenres.15603.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe</span></div><div class="casAuthors">Hunt Martin; Bradley Phelim; Heys Simon; Thomsit Mark; Hall Michael B; Malone Kerri M; Wintringer Penelope; Iqbal Zamin; Hunt Martin; Walker Timothy M; Fowler Phillip; Peto Tim E A; Crook Derrick W; Lapierre Simon Grandjean; Lapierre Simon Grandjean; Walker Timothy M; Cirillo Daniela M; Comas Inaki; Comas Inaki; Comas Inaki; Farhat Maha R; Gardy Jennifer; Gardy Jennifer; Ismail Nazir; Omar Shaheed Vally; Kohl Thomas A; Merker Matthias; Niemann Stefan; Mathys Vanessa; Niemann Stefan; Sintchenko Vitali; Smith Grace; van Soolingen Dick; Supply Philip; Tahseen Sabira; Wilcox Mark; Wilcox Mark; Arandjelovic Irena; Crook Derrick W</div><div class="citationInfo"><span class="NLM_cas:title">Wellcome open research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">191</span>
        ISSN:<span class="NLM_cas:issn">2398-502X</span>.
    </div><div class="casAbstract">Two billion people are infected with Mycobacterium tuberculosis, leading to 10 million new cases of active tuberculosis and 1.5 million deaths annually.  Universal access to drug susceptibility testing (DST) has become a World Health Organization priority.  We previously developed a software tool, Mykrobe predictor, which provided offline species identification and drug resistance predictions for M. tuberculosis from whole genome sequencing (WGS) data.  Performance was insufficient to support the use of WGS as an alternative to conventional phenotype-based DST, due to mutation catalogue limitations.  Here we present a new tool, Mykrobe, which provides the same functionality based on a new software implementation.  Improvements include i) an updated mutation catalogue giving greater sensitivity to detect pyrazinamide resistance, ii) support for user-defined resistance catalogues, iii) improved identification of non-tuberculous mycobacterial species, and iv) an updated statistical model for Oxford Nanopore Technologies sequencing data.  Mykrobe is released under MIT license at https://github.com/mykrobe-tools/mykrobe.  We incorporate mutation catalogues from the CRyPTIC consortium et al. (2018) and from Walker et al. (2015), and make improvements based on performance on an initial set of 3206 and an independent set of 5845 M. tuberculosis Illumina sequences.  To give estimates of error rates, we use a prospectively collected dataset of 4362 M. tuberculosis isolates.  Using culture based DST as the reference, we estimate Mykrobe to be 100%, 95%, 82%, 99% sensitive and 99%, 100%, 99%, 99% specific for rifampicin, isoniazid, pyrazinamide and ethambutol resistance prediction respectively.  We benchmark against four other tools on 10207 (=5845+4362) samples, and also show that Mykrobe gives concordant results with nanopore data.  We measure the ability of Mykrobe-based DST to guide personalized therapeutic regimen design in the context of complex drug susceptibility profiles, showing 94% concordance of implied regimen with that driven by phenotypic DST, higher than all other benchmarked tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD36sZ4K3pbf69nzq6467DfW6udTcc2ebergM3Btm1Brntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387hvVOntA%253D%253D&md5=d45c4de8438022a8878aa309865f5be0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.12688%2Fwellcomeopenres.15603.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Fwellcomeopenres.15603.1%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DM.%26aulast%3DBradley%26aufirst%3DP.%26aulast%3DLapierre%26aufirst%3DS.%2BG.%26aulast%3DHeys%26aufirst%3DS.%26aulast%3DThomsit%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DM.%2BB.%26aulast%3DMalone%26aufirst%3DK.%2BM.%26aulast%3DWintringer%26aufirst%3DP.%26aulast%3DWalker%26aufirst%3DT.%2BM.%26aulast%3DCirillo%26aufirst%3DD.%2BM.%26aulast%3DComas%26aufirst%3DI.%26aulast%3DFarhat%26aufirst%3DM.%2BR.%26aulast%3DFowler%26aufirst%3DP.%26aulast%3DGardy%26aufirst%3DJ.%26aulast%3DIsmail%26aufirst%3DN.%26aulast%3DKohl%26aufirst%3DT.%2BA.%26aulast%3DMathys%26aufirst%3DV.%26aulast%3DMerker%26aufirst%3DM.%26aulast%3DNiemann%26aufirst%3DS.%26aulast%3DOmar%26aufirst%3DS.%2BV.%26aulast%3DSintchenko%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DG.%26aulast%3DSoolingen%26aufirst%3DD.%2Bv.%26aulast%3DSupply%26aufirst%3DP.%26aulast%3DTahseen%26aufirst%3DS.%26aulast%3DWilcox%26aufirst%3DM.%26aulast%3DArandjelovic%26aufirst%3DI.%26aulast%3DPeto%26aufirst%3DT.%2BE.%2BA.%26aulast%3DCrook%26aufirst%3DD.%2BW.%26aulast%3DIqbal%26aufirst%3DZ.%26atitle%3DAntibiotic%2520resistance%2520prediction%2520for%2520Mycobacterium%2520tuberculosis%2520from%2520genome%2520sequence%2520data%2520with%2520Mykrobe%26jtitle%3DWellcome%2520Open%2520Res.%26date%3D2019%26volume%3D4%26spage%3D191%26epage%3D222%26doi%3D10.12688%2Fwellcomeopenres.15603.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2ebergM3Btm1Brntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520cancer%2520statistics%25202018%253A%2520GLOBOCAN%2520estimates%2520of%2520incidence%2520and%2520mortality%2520worldwide%2520for%252036%2520cancers%2520in%2520185%2520countries%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">National
Cancer Institute</span>.  <i>Cancer Stat Facts:
Female Breast Cancer</i>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=National%0ACancer+Institute&title=Cancer+Stat+Facts%3A%0AFemale+Breast+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DCancer%2520Stat%2520Facts%253A%250AFemale%2520Breast%2520Cancer%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olayioye, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neve, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span> <span> </span><span class="NLM_article-title">The ErbB signaling network : receptor heterodimerization in development and cancer</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3159</span>– <span class="NLM_lpage">3167</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.13.3159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Femboj%2F19.13.3159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10880430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3159-3167&author=M.+A.+Olayioyeauthor=R.+M.+Neveauthor=H.+A.+Laneauthor=N.+E.+Hynes&title=The+ErbB+signaling+network+%3A+receptor+heterodimerization+in+development+and+cancer&doi=10.1093%2Femboj%2F19.13.3159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The ErbB signaling network: receptor heterodimerization in development and cancer</span></div><div class="casAuthors">Olayioye, Monilola A.; Neve, Richard M.; Lane, Heidi A.; Hynes, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review, with ∼126 refs., on individual ErbB receptors and their signaling pot. and examples of specific ErbB heterodimers in signaling and development.  The authors particular emphasis is on their role in mammary gland biol. , an organ in which the ErbB family and its ligands are critically involved in development, differentiation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr03qEN815kibVg90H21EOLACvtfcHk0liTlkf5yhAdPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtLY%253D&md5=3e70478dda68f1cfd5ab5bdb910e96a2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.13.3159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.13.3159%26sid%3Dliteratum%253Aachs%26aulast%3DOlayioye%26aufirst%3DM.%2BA.%26aulast%3DNeve%26aufirst%3DR.%2BM.%26aulast%3DLane%26aufirst%3DH.%2BA.%26aulast%3DHynes%26aufirst%3DN.%2BE.%26atitle%3DThe%2520ErbB%2520signaling%2520network%2520%253A%2520receptor%2520heterodimerization%2520in%2520development%2520and%2520cancer%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D3159%26epage%3D3167%26doi%3D10.1093%2Femboj%2F19.13.3159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-s.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, T. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, R.</span></span> <span> </span><span class="NLM_article-title">An open-and-shut case ? recent insights into the activation of EGF / ErbB receptors csiro health science and nutrition</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(03)00350-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs1097-2765%2803%2900350-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=14527402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFGrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=541-552&author=A.+W.+Burgessauthor=H.-s.+Choauthor=C.+Eigenbrotauthor=K.+M.+Fergusonauthor=T.+P.+J.+Garrettauthor=D.+J.+Leahyauthor=M.+A.+Lemmonauthor=M.+X.+Sliwkowskiauthor=C.+W.+Wardauthor=S.+Yokoyamaauthor=R.+Paradeauthor=R.+Parade&title=An+open-and-shut+case+%3F+recent+insights+into+the+activation+of+EGF+%2F+ErbB+receptors+csiro+health+science+and+nutrition&doi=10.1016%2Fs1097-2765%2803%2900350-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An open-and-shut case? Recent insights into the activation of EGF/ErbB receptors</span></div><div class="casAuthors">Burgess, Antony W.; Cho, Hyun-Soo; Eigenbrot, Charles; Ferguson, Kathryn M.; Garrett, Thomas P. J.; Leahy, Daniel J.; Lemmon, Mark A.; Sliwkowski, Mark X.; Ward, Colin W.; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">541-552</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Recent crystallog. studies have provided significant new insight into how receptor tyrosine kinases from the EGF receptor or ErbB family are regulated by their growth factor ligands.  EGF receptor dimerization is mediated by a unique dimerization arm, which becomes exposed only after a dramatic domain rearrangement is promoted by growth factor binding.  ErbB2, a family member that has no ligand, has its dimerization arm constitutively exposed, and this explains several of its unique properties.  The authors outline a mechanistic view of ErbB receptor homo- and hetero-dimerization, which suggests new approaches for interfering with these processes when they are implicated in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXA0vpzHMgJrVg90H21EOLACvtfcHk0liTlkf5yhAdPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFGrsbc%253D&md5=e2e2d07fc67ccdb7eedebeb73e511ae4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2803%2900350-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252803%252900350-2%26sid%3Dliteratum%253Aachs%26aulast%3DBurgess%26aufirst%3DA.%2BW.%26aulast%3DCho%26aufirst%3DH.-s.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26aulast%3DGarrett%26aufirst%3DT.%2BP.%2BJ.%26aulast%3DLeahy%26aufirst%3DD.%2BJ.%26aulast%3DLemmon%26aufirst%3DM.%2BA.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DWard%26aufirst%3DC.%2BW.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DParade%26aufirst%3DR.%26aulast%3DParade%26aufirst%3DR.%26atitle%3DAn%2520open-and-shut%2520case%2520%253F%2520recent%2520insights%2520into%2520the%2520activation%2520of%2520EGF%2520%252F%2520ErbB%2520receptors%2520csiro%2520health%2520science%2520and%2520nutrition%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D541%26epage%3D552%26doi%3D10.1016%2Fs1097-2765%2803%2900350-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">ErbB/HER protein-tyrosine kinases: structures and small molecule inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2014.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.phrs.2014.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24928736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCjsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2014&pages=42-59&author=R.+Roskoski&title=ErbB%2FHER+protein-tyrosine+kinases%3A+structures+and+small+molecule+inhibitors&doi=10.1016%2Fj.phrs.2014.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB/HER protein-tyrosine kinases: Structures and small molecule inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-59</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) family consists of four members that belong to the ErbB lineage of proteins (ErbB1-4).  These receptors consist of an extracellular domain, a single hydrophobic transmembrane segment, and an intracellular portion with a juxtamembrane segment, a protein kinase domain, and a carboxyterminal tail.  The ErbB proteins function as homo and heterodimers.  Growth factor binding to EGFR induces a large conformational change in the extracellular domain.  Two ligand-EGFR complexes unite to form a back-to-back dimer in which the ligands are on opposite sides of the aggregate.  Following ligand binding, EGFR intracellular kinase domains form an asym. dimer.  The carboxyterminal lobe of the activator kinase of the dimer interacts with the amino-terminal lobe of the receiver kinase thereby leading to its allosteric stimulation.  Several malignancies are assocd. with the mutation or increased expression of members of the ErbB family including lung, breast, stomach, colorectal, head and neck, and pancreatic carcinomas.  Gefitinib, erlotinib, and afatinib are orally effective protein-kinase targeted quinazoline derivs. that are used in the treatment of ERBB1-mutant lung cancer and lapatinib is an orally effective quinazoline deriv. used in the treatment of ErbB2-overexpressing breast cancer.  Moreover, monoclonal antibodies that target the extracellular domain of ErbB2 are used for the treatment of ErbB2-pos. breast cancer and monoclonal antibodies that target ErbB1 and are used for the treatment of colorectal cancer.  Cancers treated with these targeted drugs eventually become resistant to them, and a current goal of research is to develop drugs that are effective against drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLydngYGBiMLVg90H21EOLACvtfcHk0liTlkf5yhAdPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCjsLfF&md5=bc670e77f2b34592d168e6f60f56a548</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2014.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2014.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DErbB%252FHER%2520protein-tyrosine%2520kinases%253A%2520structures%2520and%2520small%2520molecule%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2014%26volume%3D87%26spage%3D42%26epage%3D59%26doi%3D10.1016%2Fj.phrs.2014.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">Biology of HER2 and its importance in breast cancer</span>. <i>Oncology</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1159/000055396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1159%2F000055396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11694782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFWjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=1-13&author=Y.+Yarden&title=Biology+of+HER2+and+its+importance+in+breast+cancer&doi=10.1159%2F000055396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Biology of HER2 and Its Importance in Breast Cancer</span></div><div class="casAuthors">Yarden, Yosef</div><div class="citationInfo"><span class="NLM_cas:title">Oncology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">ONCOBS</span>;
        ISSN:<span class="NLM_cas:issn">0030-2414</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">A review.  Human epidermal growth factor receptor-2 (HER2/erbB-2) belongs to a family of four transmembrane receptors involved in signal transduction pathways that regulate cell growth and differentiation.  Overexpression/amplification of HER2 is assocd. with malignancy and a poor prognosis in breast cancer.  HER2 acts as a networking receptor that mediates signaling to cancer cells, causing them to proliferate.  HER receptors exist as monomers but dimerize on ligand binding.  HER ligands are bivalent growth factor mols. whose low-affinity site binds to HER2.  No HER2-specific ligand has been identified but HER2 is the preferred heterodimerization partner for other HER receptors.  HER2-contg. heterodimers are relatively long-lived and potent.  HER3 has no inherent activity and is the major and most potent dimerization partner of HER2.  HER2 overexpression biases the formation of HER2-contg. heterodimers, leading to enhanced responsiveness to stromal growth factors and oncogenic transformation.  Removal of HER2 from the cell surface or inhibition of its intrinsic enzymic activity may reduce oncogenicity.  Our research suggests that the antitumor efficacy of HER2-specific antibodies such as Herceptin relates to their ability to direct HER2 to a Cbl- dependent endocytosis and degrdn. pathway.  The reported clin. therapeutic efficacy of anti-HER2 monoclonal antibodies in breast cancer highlights the importance of understanding the biol. of HER2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtc627aN6-eLVg90H21EOLACvtfcHk0liNE3RESxVSlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFWjsLw%253D&md5=fb9f4c11080bb151edc30e62bb8869a8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1159%2F000055396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000055396%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DBiology%2520of%2520HER2%2520and%2520its%2520importance%2520in%2520breast%2520cancer%26jtitle%3DOncology%26date%3D2001%26volume%3D61%26spage%3D1%26epage%3D13%26doi%3D10.1159%2F000055396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotaling, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendly, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J. A.</span></span> <span> </span><span class="NLM_article-title">Nonclinical studies addressing the mechanism of action of trastuzumab (Herceptin)</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">70</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10482195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1SisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1999&pages=60-70&author=M.+X.+Sliwkowskiauthor=J.+A.+Lofgrenauthor=G.+D.+Lewisauthor=T.+E.+Hotalingauthor=B.+M.+Fendlyauthor=J.+A.+Fox&title=Nonclinical+studies+addressing+the+mechanism+of+action+of+trastuzumab+%28Herceptin%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nonclinical studies addressing the mechanism of action of trastuzumab (Herceptin)</span></div><div class="casAuthors">Sliwkowski, Mark X.; Lofgren, Julie A.; Lewis, Gail D.; Hotaling, Timothy E.; Fendly, Brian M.; Fox, Judith A.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4, Suppl. 12</span>),
    <span class="NLM_cas:pages">60-70</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">W. B. Saunders Co.</span>)
        </div><div class="casAbstract">A review with 75 refs.  HER2 is a ligand-less member of the human epidermal growth factor receptor or ErbB family of tyrosine kinases.  In normal biol. systems, HER2 functions as a co-receptor for a multitude of epidermal growth factor-like ligands that bind and activate other HER family members.  HER2 overexpression is obsd. in a no. of human adenocarcinomas and results in constitutive HER2 activation.  Specific targeting of these tumors can be accomplished with antibodies directed against the extracellular domain of the HER2 protein.  One of these antibodies, 4D5, has been fully humanized and is termed trastuzumab (Herceptin; Genentech, San Francisco, CA).  Treatment of HER2-overexpressing breast cancer cell lines with trastuzumab results in induction of p27KIPI and the Rb-related protein, p130, which in turn significantly reduces the no. of cells undergoing S-phase.  A no. of other phenotypic changes are obsd. in vitro as a consequence of trastuzumab binding to HER2-overexpressing cells.  These phenotypic changes include down-modulation of the HER2 receptor, inhibition of tumor cell growth, reversed cytokine resistance, restored E-cadherin expression levels, and reduced vascular endothelial growth factor prodn.  Interaction of trastuzumab with the human immune system via its human IgG1 Fc domain may potentiate its antitumor activities.  In vitro studies demonstrate that trastuzumab is very effective in mediating antibody-dependent cell-mediated cytotoxicity against HER2-overexpressing tumor targets.  Trastuzumab treatment of mouse xenograft models results in marked suppression of tumor growth.  When given in combination with std. cytotoxic chemotherapeutic agents, trastuzumab treatment generally results in statistically superior antitumor efficacy compared with either agent given alone.  Taken together, these studies suggest that the mechanism of action of trastuzumab includes antagonizing the constitutive growth-signaling properties of the HER2 system, enlisting immune cells to attack and kill the tumor target, and augmenting chemotherapy-induced cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp46ajd3v71YbVg90H21EOLACvtfcHk0liNE3RESxVSlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1SisbY%253D&md5=daeda0d8548a188580d9b90d15a44175</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DLofgren%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DG.%2BD.%26aulast%3DHotaling%26aufirst%3DT.%2BE.%26aulast%3DFendly%26aufirst%3DB.%2BM.%26aulast%3DFox%26aufirst%3DJ.%2BA.%26atitle%3DNonclinical%2520studies%2520addressing%2520the%2520mechanism%2520of%2520action%2520of%2520trastuzumab%2520%2528Herceptin%2529%26jtitle%3DSemin.%2520Oncol.%26date%3D1999%26volume%3D26%26spage%3D60%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, M. I.</span></span> <span> </span><span class="NLM_article-title">Erratum: HER2/Neu: mechanisms of dimerization/oligomerization</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">328</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1205119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fsj.onc.1205119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11840330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=328&author=P.+J.+Brennanauthor=T.+Kumagaiauthor=A.+Berezovauthor=R.+Muraliauthor=M.+I.+Greene&title=Erratum%3A+HER2%2FNeu%3A+mechanisms+of+dimerization%2Foligomerization&doi=10.1038%2Fsj.onc.1205119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HER2/Neu: mechanisms of dimerization/oligomerization. [Erratum to document cited in CA134:205690]</span></div><div class="casAuthors">Brennan, Patrick J.; Kumagai, Toru; Berezov, Alan; Murali, Ramachandran; Greene, Mark I.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The correct name of the second author is Toru Kumagai, not Kumogai.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRpC9GeIShsrVg90H21EOLACvtfcHk0liNE3RESxVSlg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVejtL8%253D&md5=de1143f8d79b7e239957dca452565172</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205119%26sid%3Dliteratum%253Aachs%26aulast%3DBrennan%26aufirst%3DP.%2BJ.%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DBerezov%26aufirst%3DA.%26aulast%3DMurali%26aufirst%3DR.%26aulast%3DGreene%26aufirst%3DM.%2BI.%26atitle%3DErratum%253A%2520HER2%252FNeu%253A%2520mechanisms%2520of%2520dimerization%252Foligomerization%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D328%26doi%3D10.1038%2Fsj.onc.1205119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mernaugh, R.</span></span> <span> </span><span class="NLM_article-title">Resistance to trastuzumab in breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7479</span>– <span class="NLM_lpage">7491</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-09-0636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-09-0636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=20008848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGgur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7479-7491&author=P.+R.+Pohlmannauthor=I.+A.+Mayerauthor=R.+Mernaugh&title=Resistance+to+trastuzumab+in+breast+cancer&doi=10.1158%2F1078-0432.ccr-09-0636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Trastuzumab in Breast Cancer</span></div><div class="casAuthors">Pohlmann, Paula R.; Mayer, Ingrid A.; Mernaugh, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7479-7491</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  HER2 is a transmembrane oncoprotein encoded by the HER2/neu gene and is overexpressed in approx. 20 to 25% of invasive breast cancers.  It can be therapeutically targeted by trastuzumab, a humanized IgG1 kappa light chain monoclonal antibody.  Although trastuzumab is currently considered one of the most effective treatments in oncol., a significant no. of patients with HER2-overexpressing breast cancer do not benefit from it.  Understanding the mechanisms of action and resistance to trastuzumab is therefore crucial for the development of new therapeutic strategies.  This review discusses proposed trastuzumab mode of action as well as proposed mechanisms for resistance.  Mechanisms for resistance are grouped into four main categories: (1) obstacles preventing trastuzumab binding to HER2; (2) upregulation of HER2 downstream signaling pathways; (3) signaling through alternate pathways; and (4) failure to trigger an immune-mediated mechanism to destroy tumor cells.  These potential mechanisms through which trastuzumab resistance may arise have been used as a guide to develop drugs, presently in clin. trials, to overcome resistance.  The mechanisms conferring trastuzumab resistance, when completely understood, will provide insight on how best to treat HER2-overexpressing breast cancer.  The understanding of each mechanism of resistance is therefore crit. for the educated development of strategies to overcome it, as well as for the development of tools that would allow definitive and efficient patient selection for each therapy. (Clin Cancer Res 2009;15(24):7479-91).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMog_W0HxkOrVg90H21EOLACvtfcHk0lix5aRgivlrDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGgur3F&md5=704b281777724d433a8d75ac22491d8f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-09-0636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-09-0636%26sid%3Dliteratum%253Aachs%26aulast%3DPohlmann%26aufirst%3DP.%2BR.%26aulast%3DMayer%26aufirst%3DI.%2BA.%26aulast%3DMernaugh%26aufirst%3DR.%26atitle%3DResistance%2520to%2520trastuzumab%2520in%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7479%26epage%3D7491%26doi%3D10.1158%2F1078-0432.ccr-09-0636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevray, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning of the neu gene: Absence of gross structural alteration in oncogenic alleles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1073/pnas.83.2.261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1073%2Fpnas.83.2.261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=3001730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaL28XotlSnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=261-264&author=M.+C.+Hungauthor=A.+L.+Schechterauthor=P.+Y.+Chevrayauthor=D.+F.+Sternauthor=R.+A.+Weinberg&title=Molecular+cloning+of+the+neu+gene%3A+Absence+of+gross+structural+alteration+in+oncogenic+alleles&doi=10.1073%2Fpnas.83.2.261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning of the neu gene:  absence of gross structural alteration in oncogenic alleles</span></div><div class="casAuthors">Hung, Mien Chie; Schechter, Alan L.; Chevray, Pierre Yves M.; Stern, David F.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The rat neu gene is distantly related to the erbB gene and encodes a cell surface protein that appears to function as a growth factor receptor.  To study the mechanisms that caused the conversion of the normal neu gene to an oncogenic allele, mol. clones of the neu oncogene as well as a clone of the corresponding protooncogene were isolated.  The transforming neu oncogene and the proton-neu gene clones exhibit identical restriction enzyme patterns.  Amplification of the proto-neu gene in NIH 3T3 cells by means of cotransfection with a dihydrofolate reductase gene resulted in methotrexate-resistant colonies that produce high levels of normal neu-encoded p185 protein.  In contrast to cells carrying low levels of the oncogene-encoded protein, these cells appeared normal.  The results suggest that the lesion that led to activation of the neu gene is a minor change in DNA sequence and is apparently located in the protein-encoding region of the gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj3ThZdPzQl7Vg90H21EOLACvtfcHk0lix5aRgivlrDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XotlSnug%253D%253D&md5=27357da2ccf1aa2ec3ca6c28ce276a59</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.2.261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.2.261%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DM.%2BC.%26aulast%3DSchechter%26aufirst%3DA.%2BL.%26aulast%3DChevray%26aufirst%3DP.%2BY.%26aulast%3DStern%26aufirst%3DD.%2BF.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DMolecular%2520cloning%2520of%2520the%2520neu%2520gene%253A%2520Absence%2520of%2520gross%2520structural%2520alteration%2520in%2520oncogenic%2520alleles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1986%26volume%3D83%26spage%3D261%26epage%3D264%26doi%3D10.1073%2Fpnas.83.2.261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funkhouser, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser-Rogers, K.</span></span> <span> </span><span class="NLM_article-title">Review: Significance of, and optimal screening for, HER-2 gene amplification and protein overexpression in breast carcinoma</span>. <i>Ann. Clin. Lab. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">349</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11688845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot1Smt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2001&pages=349&author=W.+K.+Funkhouserauthor=K.+Kaiser-Rogers&title=Review%3A+Significance+of%2C+and+optimal+screening+for%2C+HER-2+gene+amplification+and+protein+overexpression+in+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Review: Significance of, and optimal screening for, HER-2 gene amplification and protein overexpression in breast carcinoma</span></div><div class="casAuthors">Funkhouser, William K.; Kaiser-Rogers, Kathleen</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Clinical and Laboratory Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-358</span>CODEN:
                <span class="NLM_cas:coden">ACLSCP</span>;
        ISSN:<span class="NLM_cas:issn">0091-7370</span>.
    
            (<span class="NLM_cas:orgname">Association of Clinical Scientists</span>)
        </div><div class="casAbstract">A review.  Breast carcinoma is a common disease, with an estd. 183,000 new cases expected in the USA during 2000.  Whereas early stage patients have high likelihood of cure, only 20-40% of patients with metastatic breast carcinoma respond to presently available chemotherapy.  A need exists to identify the underlying biol. subsets of morphol. similar carcinomas in order to develop customized therapies for patients who require chemotherapy.  The HER-2 receptor tyrosine kinase is overexpressed in 15-30% of breast carcinomas, and is assocd. with a worse prognosis stage-for-stage.  Humanized monoclonal antibody therapy appears to benefit this subset of patients by improving their response rate and survival following anthracycline- or taxane-based chemotherapeutic regimens.  Both HER-2 gene amplification and protein overexpression correlate with clin. outcomes, and screening for HER-2 gene amplification appears to be the more informative test.  This article reviews data on the HER-2 gene and protein, discusses their assocn. with clin. outcomes, and proposes a strategy for screening for HER-2 excess in formalin-fixed specimens of breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpalKrvVXs2gbVg90H21EOLACvtfcHk0lix5aRgivlrDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXot1Smt7o%253D&md5=eadfb2d29a99cef461d55b11c26881bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFunkhouser%26aufirst%3DW.%2BK.%26aulast%3DKaiser-Rogers%26aufirst%3DK.%26atitle%3DReview%253A%2520Significance%2520of%252C%2520and%2520optimal%2520screening%2520for%252C%2520HER-2%2520gene%2520amplification%2520and%2520protein%2520overexpression%2520in%2520breast%2520carcinoma%26jtitle%3DAnn.%2520Clin.%2520Lab.%2520Sci.%26date%3D2001%26volume%3D31%26spage%3D349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brabender, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salonga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hölscher, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, P. V.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor and HER2-neu mRNA expression in non-small cell lung cancer is correlated with survival</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1850</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11448895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlslaltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=1850&author=J.+Brabenderauthor=K.+D.+Danenbergauthor=R.+Metzgerauthor=P.+M.+Schneiderauthor=J.+Parkauthor=D.+Salongaauthor=A.+H.+H%C3%B6lscherauthor=P.+V.+Danenberg&title=Epidermal+growth+factor+receptor+and+HER2-neu+mRNA+expression+in+non-small+cell+lung+cancer+is+correlated+with+survival"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor and HER2-neu mRNA expression in non-small cell lung cancer is correlated with survival</span></div><div class="casAuthors">Brabender, Jan; Danenberg, Kathleen D.; Metzger, Ralf; Schneider, Paul M.; Park, Ji Min; Salonga, Dennis; Holscher, Arnulf H.; Danenberg, Peter V.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1850-1855</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The prognostic role of epidermal growth factor receptor (EGFR) and HER2-neu remains controversial in patients with non-small cell lung cancer (NSCLC).  We studied the assocn. between the mRNA expression of EGFR, HER2-neu, and survival in primary tumor and matching nonmalignant tissues from 83 patients with NSCLC.  Anal. was performed using a quant. real-time PCR system (Taqman).  EGFR and HER2-neu mRNA expression was detectable in all (100%) specimens analyzed.  Twenty-nine (34.9%) patients had high HER2-neu expression, and 28 (33.7%) patients had high EGFR expression.  A high HER2-neu and EGFR coexpression was detectable in 14 (16.9%) patients.  High HER2-neu expression was assocd. with inferior survival (P = 0.004), whereas high EGFR expression showed a trend toward inferior survival (P = 0.176).  The impact of HER2-neu and EGFR coexpression on patients' survival was additive (P = 0.003).  Multivariate anal. detd. high HER2-neu expression (P = 0.041), and high EGFR/HER2-neu coexpression (P = 0.030) as significant and independent unfavorable prognostic factors.  These findings indicate that HER2-neu and EGFR play a crucial role in the biol. behavior of NSCLCs.  Testing of mol. marker coexpression (EGFR and HER2-neu) improves the estn. of prognosis and appears to define low- and high-risk groups for treatment failure in curatively resected NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokEzxS4tTL-7Vg90H21EOLACvtfcHk0lix5aRgivlrDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlslaltb8%253D&md5=ae41d3db82e5ff88fe10f246429f087d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrabender%26aufirst%3DJ.%26aulast%3DDanenberg%26aufirst%3DK.%2BD.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DP.%2BM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DSalonga%26aufirst%3DD.%26aulast%3DH%25C3%25B6lscher%26aufirst%3DA.%2BH.%26aulast%3DDanenberg%26aufirst%3DP.%2BV.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520and%2520HER2-neu%2520mRNA%2520expression%2520in%2520non-small%2520cell%2520lung%2520cancer%2520is%2520correlated%2520with%2520survival%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D1850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, D.</span></span> <span> </span><span class="NLM_article-title">Oral cancer progression and c-erbB-2/neu proto-oncogene expression</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1016/0304-3835(92)90234-m</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2F0304-3835%2892%2990234-m" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1355405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADyaK38zos1CltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1992&pages=215-220&author=L.+Houauthor=D.+Shiauthor=S.-M.+Tuauthor=H.-Z.+Zhangauthor=M.-C.+Hungauthor=D.+Ling&title=Oral+cancer+progression+and+c-erbB-2%2Fneu+proto-oncogene+expression&doi=10.1016%2F0304-3835%2892%2990234-m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Oral cancer progression and c-erbB-2/neu proto-oncogene expression</span></div><div class="casAuthors">Hou L; Shi D; Tu S M; Zhang H Z; Hung M C; Ling D</div><div class="citationInfo"><span class="NLM_cas:title">Cancer letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-20</span>
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    </div><div class="casAbstract">Monoclonal antibody PAb3 to c-erbB-2/neu protein was utilized in the immunoperoxidase staining of 86 human specimens from oral mucosa.  These tissue specimens represented a spectrum from 7 normal to 9 simple hyperplasia, 15 mild dysplasia, 14 moderate dysplasia, 20 severe dysplasia and 21 squamous cell carcinoma.  Our study indicated that as the cells acquire a more malignant phenotype, there was a progressive increase in neu expression.  It also suggested that neu may be involved in the development of oral cancers and that its evaluation in the early stages may assist in the diagnosis and management of oral cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8-GGKG5txDokFb3mqPUYDfW6udTcc2ebZE-k9dmS6mbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38zos1CltQ%253D%253D&md5=fee3cd5310b27f28a0bf471bf91b79b6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2F0304-3835%2892%2990234-m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3835%252892%252990234-m%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DD.%26aulast%3DTu%26aufirst%3DS.-M.%26aulast%3DZhang%26aufirst%3DH.-Z.%26aulast%3DHung%26aufirst%3DM.-C.%26aulast%3DLing%26aufirst%3DD.%26atitle%3DOral%2520cancer%2520progression%2520and%2520c-erbB-2%252Fneu%2520proto-oncogene%2520expression%26jtitle%3DCancer%2520Lett.%26date%3D1992%26volume%3D65%26spage%3D215%26epage%3D220%26doi%3D10.1016%2F0304-3835%2892%2990234-m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span> <span> </span><span class="NLM_article-title">HER2 as a promising target for cytotoxicity T cells in human melanoma therapy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e73261</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0073261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0073261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24015299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e73261&author=J.+Maauthor=H.+Hanauthor=D.+Liuauthor=W.+liauthor=H.+Fengauthor=X.+Xueauthor=X.+Wuauthor=G.+Niuauthor=G.+Zhangauthor=Y.+Zhaoauthor=C.+Liuauthor=H.+Taoauthor=B.+Gao&title=HER2+as+a+promising+target+for+cytotoxicity+T+cells+in+human+melanoma+therapy&doi=10.1371%2Fjournal.pone.0073261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 as a promising target for cytotoxicity T cells in human melanoma therapy</span></div><div class="casAuthors">Ma, Juan; Han, Huamin; Liu, Deruo; Li, Wei; Feng, Hongxiang; Xue, Xin; Wu, Xiaoran; Niu, Ge; Zhang, Ge; Zhao, Yunfeng; Liu, Changzhen; Tao, Hua; Gao, Bin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e73261</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Anti-HER2/neu antibody therapy has been reported to mediate tumor regression of HER2/neu+ tumors.  Here we demonstrated the expression of HER2 in a wide range of human melanoma cells including a primary culture and seven cell lines, and we further investigated whether HER2 could be served as a target for T cell mediated immunotherapy of human melanoma.  Specific cytolytic activity of activated T cells (ATC) armed with anti-CD3 × anti-HER2 bispecific antibody (HER2Bi-Ab) against Malme-3M-luc cells was evaluated by bioluminescent signal generated by luciferase reporter which did not alter HER2 expression or proliferation ability of Malme-3M cells.  Contrast with unarmed ATC, increased cytotoxic activity of HER2Bi-armed ATC against Malme-3M-luc cells was obsd. at effector/target (E/T) ratios of 1:1, 5:1, and 20:1.  Moreover, HER2Bi-armed ATC expressed higher level of activation marker CD69 and secreted significantly higher level of IFN-γ than unarmed ATC counterpart at the E/T ratio of 20:1.  In addn., compared with anti-HER2 mAb (Herceptin) or unarmed ATC, HER2Bi-armed ATC showed remarkable suppression effect on Malme-3M-luc tumor cells.  Furthermore, in melanoma tumor cell xenograft mice, infusion of HER2Bi-armed ATC successfully inhibited the growth of melanoma tumors.  The anti-tumor effect of HER2Bi-armed ATC may provide a promising immunotherapy for melanoma in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo99ILu7lrNebVg90H21EOLACvtfcHk0lipDxH6vrMPSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrvI&md5=8d0f508736ee30ac02bb4b69a045c4b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0073261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0073261%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3Dli%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DB.%26atitle%3DHER2%2520as%2520a%2520promising%2520target%2520for%2520cytotoxicity%2520T%2520cells%2520in%2520human%2520melanoma%2520therapy%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De73261%26doi%3D10.1371%2Fjournal.pone.0073261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musolino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccolallo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panebianco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozzetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michiara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silini, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardizzoni, A.</span></span> <span> </span><span class="NLM_article-title">Multifactorial central nervous system recurrence susceptibility in patients with HER2-positive breast cancer</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1837</span>– <span class="NLM_lpage">1846</span>, <span class="refDoi"> DOI: 10.1002/cncr.25771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcncr.25771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21509760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvlt12rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=1837-1846&author=A.+Musolinoauthor=L.+Ciccolalloauthor=M.+Panebiancoauthor=E.+Fontanaauthor=D.+Zanoniauthor=C.+Bozzettiauthor=M.+Michiaraauthor=E.+M.+Siliniauthor=A.+Ardizzoni&title=Multifactorial+central+nervous+system+recurrence+susceptibility+in+patients+with+HER2-positive+breast+cancer&doi=10.1002%2Fcncr.25771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Multifactorial central nervous system recurrence susceptibility in patients with HER2-positive breast cancer: epidemiological and clinical data from a population-based cancer registry study</span></div><div class="casAuthors">Musolino Antonino; Ciccolallo Laura; Panebianco Michele; Fontana Elisa; Zanoni Daniele; Bozzetti Cecilia; Michiara Maria; Silini Enrico Maria; Ardizzoni Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1837-46</span>
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    </div><div class="casAbstract">BACKGROUND:  A series of retrospective studies have reported that patients with human epidermal growth factor receptor 2(HER2)-positive breast cancer are at a greater risk of central nervous system (CNS) metastases.  Trastuzumab, which does not cross the blood-brain barrier, has been associated with this increased risk.  METHODS:  The authors evaluated incidence, survival, and risk factors for CNS metastases in the incident breast cancer population systematically collected by the Parma Province Cancer Registry over the 4-year period between 2004 and 2007.  RESULTS:  A total of 1458 patients with a diagnosis of stage I to III invasive breast cancer were analyzed for study purposes.  At a median follow-up of 4.1 years, CNS events were observed in 1.3% and 5% of HER2-negative patients and HER2-positive patients, respectively (P < .0001).  The administration of trastuzumab either as adjuvant therapy or for metastatic disease was associated with a significantly increased risk of CNS involvement at first disease recurrence and after first extracranial recurrence, respectively.  According to multivariate analysis, HER2-positive status and trastuzumab treatment, high Ki-67 index, and hormone receptor negativity remained independent risk factors for the development of CNS metastasis.  CONCLUSIONS:  To the authors' knowledge, this is the first population-based cancer registry study analyzing factors associated with CNS recurrence in a general population of newly diagnosed breast cancer patients with known HER2 status.  The data from the current study provide evidence that patients with HER2-positive breast cancer have a significantly higher incidence of CNS metastasis after treatment with trastuzumab.  Improvements in systemic control and overall survival associated with trastuzumab-based therapy may lead to an "unmasking" of CNS disease recurrence that would otherwise remain clinically silent before a patient's death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzAwMcs1oehCM8MEzWGjS9fW6udTcc2ebZE-k9dmS6mbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvlt12rsA%253D%253D&md5=8f3833e5f32b9534a231886a90b747d1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcncr.25771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.25771%26sid%3Dliteratum%253Aachs%26aulast%3DMusolino%26aufirst%3DA.%26aulast%3DCiccolallo%26aufirst%3DL.%26aulast%3DPanebianco%26aufirst%3DM.%26aulast%3DFontana%26aufirst%3DE.%26aulast%3DZanoni%26aufirst%3DD.%26aulast%3DBozzetti%26aufirst%3DC.%26aulast%3DMichiara%26aufirst%3DM.%26aulast%3DSilini%26aufirst%3DE.%2BM.%26aulast%3DArdizzoni%26aufirst%3DA.%26atitle%3DMultifactorial%2520central%2520nervous%2520system%2520recurrence%2520susceptibility%2520in%2520patients%2520with%2520HER2-positive%2520breast%2520cancer%26jtitle%3DCancer%26date%3D2011%26volume%3D117%26spage%3D1837%26epage%3D1846%26doi%3D10.1002%2Fcncr.25771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loibl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, L.</span></span> <span> </span><span class="NLM_article-title">HER2-positive breast cancer</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">2415</span>– <span class="NLM_lpage">2429</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(16)32417-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs0140-6736%2816%2932417-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=27939064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVegurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=2415-2429&author=S.+Loiblauthor=L.+Gianni&title=HER2-positive+breast+cancer&doi=10.1016%2Fs0140-6736%2816%2932417-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-positive breast cancer</span></div><div class="casAuthors">Loibl, Sibylle; Gianni, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10087</span>),
    <span class="NLM_cas:pages">2415-2429</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Anti-HER2 treatment for HER2-pos. breast cancer has changed the natural biol. of this disease.  This Series article reviews the main achievements so far in the treatment of both metastatic and early HER2-pos. breast cancer.  The success of neoadjuvant therapy in HER2-pos. early breast cancer is esp. acknowledged, as pertuzumab has been approved on the basis of a higher proportion of patients achieving a pathol. complete response with pertuzumab and trastuzumab than with trastuzumab alone in a neoadjuvant study.  Event-free survival after the confirmatory adjuvant trial completed recruitment was numerically better with pertuzumab plus trastuzumab than with trastuzumab alone.  With survival rates of almost 5 years in women with metastatic HER2-pos. breast cancer and 75% of patients achieving a pathol. complete response, new treatments in the past decade have clearly improved the prognosis of HER2-pos. breast cancer.  Despite these achievements, however, the persisting high toll of deaths resulting from HER2-pos. breast cancer calls for continued, intensive clin. research of newer therapies and combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogw25S2Ui-abVg90H21EOLACvtfcHk0lh0_G46TkO1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVegurbF&md5=65fcd259f84cfb308e78ddacec0eb692</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fs0140-6736%2816%2932417-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0140-6736%252816%252932417-5%26sid%3Dliteratum%253Aachs%26aulast%3DLoibl%26aufirst%3DS.%26aulast%3DGianni%26aufirst%3DL.%26atitle%3DHER2-positive%2520breast%2520cancer%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D2415%26epage%3D2429%26doi%3D10.1016%2Fs0140-6736%2816%2932417-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Escrivá-de-Romaní, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arumí, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saura, C.</span></span> <span> </span><span class="NLM_article-title">HER2-positive breast cancer: Current and new therapeutic strategies</span>. <i>Breast</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2018.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.breast.2018.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29631097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjgt1ygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=80-88&author=S.+Escriv%C3%A1-de-Roman%C3%ADauthor=M.+Arum%C3%ADauthor=M.+Belletauthor=C.+Saura&title=HER2-positive+breast+cancer%3A+Current+and+new+therapeutic+strategies&doi=10.1016%2Fj.breast.2018.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-positive breast cancer: Current and new therapeutic strategies</span></div><div class="casAuthors">Escriva-de-Romani Santiago; Arumi Miriam; Bellet Meritxell; Saura Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Breast (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-88</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the identification of the HER2 receptor amplification as an adverse prognostic factor that defined a special subtype of metastatic breast cancer, there has been a substantial improvement in survival of patients affected with this disease due to the development of anti-HER2 targeted therapies.  The approval of trastuzumab and pertuzumab associated to a taxane in first line and subsequent treatment with the antibody-drug conjugate T-DM1 has certainly contributed to achieve these outcomes.  The Tyrosine Kinase Inhibitor lapatinib was also approved in the basis of an improvement in progression free survival, becoming another commonly used treatment in combination with capecitabine.  Inevitably, despite these therapeutic advances most patients progress on therapy due to primary or acquired resistance or because of an incorrect HER2 positivity assessment.  Hence, it is crucial to correctly categorize HER2 amplified tumors and define mechanisms of resistance to design effective new treatment approaches.  In addition, identifying biomarkers of response or resistance permits to tailor the therapeutic options for each patient sparing them from unnecessary toxicity as well as improving their outcomes.  The aim of this review is to examine new strategies in development to treat HER2-positive metastatic breast cancer referring to the mechanisms of action of new drugs and new combinations including results reported so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3a48Zp0Bo9Nj_Vg2vObihfW6udTcc2eaIHhaX2hW767ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjgt1ygsQ%253D%253D&md5=36ea3f9eec7f253d0a0f35ebb812acbc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2018.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2018.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DEscriv%25C3%25A1-de-Roman%25C3%25AD%26aufirst%3DS.%26aulast%3DArum%25C3%25AD%26aufirst%3DM.%26aulast%3DBellet%26aufirst%3DM.%26aulast%3DSaura%26aufirst%3DC.%26atitle%3DHER2-positive%2520breast%2520cancer%253A%2520Current%2520and%2520new%2520therapeutic%2520strategies%26jtitle%3DBreast%26date%3D2018%26volume%3D39%26spage%3D80%26epage%3D88%26doi%3D10.1016%2Fj.breast.2018.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, Y.-J.</span></span> <span> </span><span class="NLM_article-title">HER2-targeted therapies - a role beyond breast cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1038/s41571-019-0268-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41571-019-0268-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=31548601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVarsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=33-48&author=D.-Y.+Ohauthor=Y.-J.+Bang&title=HER2-targeted+therapies+-+a+role+beyond+breast+cancer&doi=10.1038%2Fs41571-019-0268-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-targeted therapies - a role beyond breast cancer</span></div><div class="casAuthors">Oh, Do-Youn; Bang, Yung-Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-48</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  HER2 is an established therapeutic target in a large subset of women with breast cancer; a variety of agents including trastuzumab, pertuzumab, lapatinib, neratinib and trastuzumab emtansine (T-DM1) have been approved for the treatment of HER2-pos. breast cancer.  HER2 is also overexpressed in subsets of patients with other solid tumors.  Notably, the addn. of trastuzumab to first-line chemotherapy has improved the overall survival of patients with HER2-pos. gastric cancer, and has become the std.-of-care treatment for this group of patients.  However, trials involving pertuzumab, lapatinib and T-DM1 have failed to provide significant improvements in the outcomes of patients with HER2-pos. gastric cancer.  HER2-targeted therapies are also being tested in patients with other solid tumors harbouring HER2 overexpression, and/or amplifications or other mutations of the gene encoding HER2 (ERBB2), including biliary tract, colorectal, non-small-cell lung and bladder cancers.  The experience with gastric cancer suggests that the successes obsd. in HER2-pos. breast cancer might not be replicated in these other tumor types, owing to differences in the level of HER2 overexpression and other aspects of disease biol.  In this Review, we describe the current role of HER2-targeted therapies beyond breast cancer and also highlight the potential of novel HER2-targeted agents that are currently in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWFUgr5GY13bVg90H21EOLACvtfcHk0lh0_G46TkO1hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVarsLfF&md5=8fc3d80ab470b5e01686cae99f944885</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41571-019-0268-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-019-0268-3%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DD.-Y.%26aulast%3DBang%26aufirst%3DY.-J.%26atitle%3DHER2-targeted%2520therapies%2520-%2520a%2520role%2520beyond%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D17%26spage%3D33%26epage%3D48%26doi%3D10.1038%2Fs41571-019-0268-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arribas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra-Palau, J. L.</span></span> <span> </span><span class="NLM_article-title">p95HER2 and breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-3795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-10-3795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21343397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFCgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1515-1519&author=J.+Arribasauthor=J.+Baselgaauthor=K.+Pedersenauthor=J.+L.+Parra-Palau&title=p95HER2+and+breast+cancer&doi=10.1158%2F0008-5472.can-10-3795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">p95HER2 and Breast Cancer</span></div><div class="casAuthors">Arribas, Joaquin; Baselga, Jose; Pedersen, Kim; Parra-Palau, Josep Lluis</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1515-1519</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  A subtype of HER2-pos. tumors with distinct biol. and clin. features expresses a series of carboxy-terminal fragments collectively known as p95HER2.  One of these fragments, named 100- to 115-kDa p95HER2 or 611-CTF, is hyperactive because of its ability to form homodimers maintained by intermol. disulfide bonds.  Despite lacking the majority of the extracellular domain, this HER2 fragment drives breast cancer progression in vivo.  The recent availability of specific anti-p95 antibodies has confirmed previous results indicating that the expression of p95HER2 is predictive of poor prognosis and correlates with resistance to the treatment with trastuzumab, a therapeutic antibody directed against the extracellular domain of HER2.  Cancer Res; 71(5); 1515-9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZSd5rzePjJ7Vg90H21EOLACvtfcHk0ljlFzn6GEJthA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFCgu7w%253D&md5=79b01b5f32271d0d833d804828787358</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-10-3795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-10-3795%26sid%3Dliteratum%253Aachs%26aulast%3DArribas%26aufirst%3DJ.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DPedersen%26aufirst%3DK.%26aulast%3DParra-Palau%26aufirst%3DJ.%2BL.%26atitle%3Dp95HER2%2520and%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D1515%26epage%3D1519%26doi%3D10.1158%2F0008-5472.can-10-3795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokmanovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W. J.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab-mediated cardiotoxicity: current understanding, challenges, and frontiers</span>. <i>Antibody Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1093/abt/tby003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Fabt%2Ftby003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30215054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVOit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=13-17&author=N.+Mohanauthor=J.+Jiangauthor=M.+Dokmanovicauthor=W.+J.+Wu&title=Trastuzumab-mediated+cardiotoxicity%3A+current+understanding%2C+challenges%2C+and+frontiers&doi=10.1093%2Fabt%2Ftby003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab-mediated cardiotoxicity: current understanding, challenges, and frontiers</span></div><div class="casAuthors">Mohan, Nishant; Jiang, Jiangsong; Dokmanovic, Milos; Wu, Wen Jin</div><div class="citationInfo"><span class="NLM_cas:title">Antibody Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">ATNHAH</span>;
        ISSN:<span class="NLM_cas:issn">2516-4236</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Trastuzumab, an epidermal growth factor receptor 2 (HER2) targeting humanized monoclonal antibody, has been approved for the treatment HER2-pos. breast cancer and HER2-positve metastatic gastric cancer.  However, cardiotoxicity assocd. with its clin. application poses challenges for clinicians and patients, mechanisms of which are still evolving.  This review will summarize the current mechanistic understanding of trastuzumab-mediated cardiotoxicity, discuss the novel role of DNA topoisomerase IIB as a shared target for enhanced cardiotoxicity induced by trastuzumab and anthracyclines-based combination regimens, and speculate the potential impact of trastuzumab intervention in immune checkpoint inhibitors-based therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm7vff0kdlbVg90H21EOLACvtfcHk0ljlFzn6GEJthA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVOit7Y%253D&md5=2bcca83b7fb6ac29fdc06717626c04f6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fabt%2Ftby003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabt%252Ftby003%26sid%3Dliteratum%253Aachs%26aulast%3DMohan%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DDokmanovic%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.%2BJ.%26atitle%3DTrastuzumab-mediated%2520cardiotoxicity%253A%2520current%2520understanding%252C%2520challenges%252C%2520and%2520frontiers%26jtitle%3DAntibody%2520Ther.%26date%3D2018%26volume%3D1%26spage%3D13%26epage%3D17%26doi%3D10.1093%2Fabt%2Ftby003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diéras, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegram, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welslau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, L.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab emtansine versus capecitabine plus lapatinib in patients with previously treated HER2-positive advanced breast cancer (EMILIA): a descriptive analysis of final overall survival results from a randomised, open-label, phase 3 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">742</span>, <span class="refDoi"> DOI: 10.1016/s1470-2045(17)30312-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs1470-2045%2817%2930312-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28526536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFKiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=732-742&author=V.+Di%C3%A9rasauthor=D.+Milesauthor=S.+Vermaauthor=M.+Pegramauthor=M.+Welslauauthor=J.+Baselgaauthor=I.+E.+Kropauthor=K.+Blackwellauthor=S.+Hoerschauthor=J.+Xuauthor=M.+Greenauthor=L.+Gianni&title=Trastuzumab+emtansine+versus+capecitabine+plus+lapatinib+in+patients+with+previously+treated+HER2-positive+advanced+breast+cancer+%28EMILIA%29%3A+a+descriptive+analysis+of+final+overall+survival+results+from+a+randomised%2C+open-label%2C+phase+3+trial&doi=10.1016%2Fs1470-2045%2817%2930312-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab emtansine versus capecitabine plus lapatinib in patients with previously treated HER2-positive advanced breast cancer (EMILIA): a descriptive analysis of final overall survival results from a randomised, open-label, phase 3 trial</span></div><div class="casAuthors">Dieras, Veronique; Miles, David; Verma, Sunil; Pegram, Mark; Welslau, Manfred; Baselga, Jose; Krop, Ian E.; Blackwell, Kim; Hoersch, Silke; Xu, Jin; Green, Marjorie; Gianni, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">732-742</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The antibody-drug conjugate trastuzumab emtansine is indicated for the treatment of patients with HER2-pos. metastatic breast cancer previously treated with trastuzumab and a taxane.  Approval of this drug was based on progression-free survival and interim overall survival data from the phase 3 EMILIA study.  In this report, we present a descriptive anal. of the final overall survival data from that trial.  EMILIA was a randomized, international, open-label, phase 3 study of men and women aged 18 years or older with HER2-pos. unresectable, locally advanced or metastatic breast cancer previously treated with trastuzumab and a taxane.  Enrolled patients were randomly assigned (1:1) via a hierarchical, dynamic randomization scheme and an interactive voice response system to trastuzumab emtansine (3·6 mg/kg i.v. every 3 wk) or control (capecitabine 1000 mg/m2 self-administered orally twice daily on days 1-14 on each 21-day cycle, plus lapatinib 1250 mg orally once daily on days 1-21).  Randomisation was stratified by world region (USA vs western Europe vs or other), no. of previous chemotherapy regimens for unresectable, locally advanced, or metastatic disease (0 or 1 vs >1), and disease involvement (visceral vs non-visceral).  The coprimary efficacy endpoints were progression-free survival (per independent review committee assessment) and overall survival.  Efficacy was analyzed in the intention-to-treat population; safety was analyzed in all patients who received at least one dose of study treatment, with patients analyzed according to the treatment actually received.  On May 30, 2012, the study protocol was amended to allow crossover from control to trastuzumab emtansine after the second interim overall survival anal. crossed the prespecified overall survival efficacy boundary.  This study is registered with ClinicalTrials.gov, no. NCT00829166.  Between Feb 23, 2009, and Oct 13, 2011, 991 eligible patients were enrolled and randomly assigned to either trastuzumab emtansine (n=495) or capecitabine and lapatinib (control; n=496).  In this final descriptive anal., median overall survival was longer with trastuzumab emtansine than with control (29·9 mo [95% CI 26·3-34·1] vs 25·9 mo [95% CI 22·7-28·3]; hazard ratio 0·75 [95% CI 0·64-0·88]). 136 (27%) of 496 patients crossed over from control to trastuzumab emtansine after the second interim overall survival anal. (median follow-up duration 24·1 mo [IQR 19·5-26·1]).  Of those patients originally randomly assigned to trastuzumab emtansine, 254 (51%) of 495 received capecitabine and 241 [49%] of 495 received lapatinib (sep. or in combination) after study drug discontinuation.  In the safety population (488 patients treated with capecitabine plus lapatinib, 490 patients treated with trastuzumab emtansine), fewer grade 3 or worse adverse events occurred with trastuzumab emtansine (233 [48%] of 490) than with capecitabine plus lapatinib control treatment (291 [60%] of 488).  In the control group, the most frequently reported grade 3 or worse adverse events were diarrhea (103 [21%] of 488 patients) followed by palmar-plantar erythrodysaesthesia syndrome (87 [18%]), and vomiting (24 [5%]).  The safety profile of trastuzumab emtansine was similar to that reported previously; the most frequently reported grade 3 or worse adverse events in the trastuzumab emtansine group were thrombocytopenia (70 [14%] of 490), increased aspartate aminotransferase levels (22 [5%]), and anemia (19 [4%]).  Nine patients died from adverse events; five of these deaths were judged to be related to treatment (two in the control group [coronary artery disease and multiorgan failure] and three in the trastuzumab emtansine group [metabolic encephalopathy, neutropenic sepsis, and acute myeloid leukemia]).  This descriptive anal. of final overall survival in the EMILIA trial shows that trastuzumab emtansine improved overall survival in patients with previously treated HER2-pos. metastatic breast cancer even in the presence of crossover treatment.  The safety profile was similar to that reported in previous analyses, reaffirming trastuzumab emtansine as an efficacious and tolerable treatment in this patient population.F Hoffmann-La Roche/Genentech.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnbUnIygFBbVg90H21EOLACvtfcHk0ljlFzn6GEJthA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFKiur0%253D&md5=5d3663e6a753ddb9a2f23f67a7f3a9a4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs1470-2045%2817%2930312-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1470-2045%252817%252930312-1%26sid%3Dliteratum%253Aachs%26aulast%3DDi%25C3%25A9ras%26aufirst%3DV.%26aulast%3DMiles%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%26aulast%3DPegram%26aufirst%3DM.%26aulast%3DWelslau%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DBlackwell%26aufirst%3DK.%26aulast%3DHoersch%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DGianni%26aufirst%3DL.%26atitle%3DTrastuzumab%2520emtansine%2520versus%2520capecitabine%2520plus%2520lapatinib%2520in%2520patients%2520with%2520previously%2520treated%2520HER2-positive%2520advanced%2520breast%2520cancer%2520%2528EMILIA%2529%253A%2520a%2520descriptive%2520analysis%2520of%2520final%2520overall%2520survival%2520results%2520from%2520a%2520randomised%252C%2520open-label%252C%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2017%26volume%3D18%26spage%3D732%26epage%3D742%26doi%3D10.1016%2Fs1470-2045%2817%2930312-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneeweiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eniu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tausch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratnayake, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés, J.</span></span> <span> </span><span class="NLM_article-title">Pertuzumab plus trastuzumab in combination with standard neoadjuvant anthracycline-containing and anthracycline-free chemotherapy regimens in patients with HER2-positive early breast cancer: A randomized phase II cardiac safety study (TRYPHAENA)</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2278</span>– <span class="NLM_lpage">2284</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdt182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Fannonc%2Fmdt182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23704196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3snmsVOqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=2278-2284&author=A.+Schneeweissauthor=S.+Chiaauthor=T.+Hickishauthor=V.+Harveyauthor=A.+Eniuauthor=R.+Heggauthor=C.+Tauschauthor=J.+H.+Seoauthor=Y.-F.+Tsaiauthor=J.+Ratnayakeauthor=V.+McNallyauthor=G.+Rossauthor=J.+Cort%C3%A9s&title=Pertuzumab+plus+trastuzumab+in+combination+with+standard+neoadjuvant+anthracycline-containing+and+anthracycline-free+chemotherapy+regimens+in+patients+with+HER2-positive+early+breast+cancer%3A+A+randomized+phase+II+cardiac+safety+study+%28TRYPHAENA%29&doi=10.1093%2Fannonc%2Fmdt182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pertuzumab plus trastuzumab in combination with standard neoadjuvant anthracycline-containing and anthracycline-free chemotherapy regimens in patients with HER2-positive early breast cancer: a randomized phase II cardiac safety study (TRYPHAENA)</span></div><div class="casAuthors">Schneeweiss A; Chia S; Hickish T; Harvey V; Eniu A; Hegg R; Tausch C; Seo J H; Tsai Y-F; Ratnayake J; McNally V; Ross G; Cortes J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2278-84</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Pertuzumab (P) combined with trastuzumab (H)-based chemotherapy improves efficacy in early and advanced HER2-positive breast cancer.  We assessed the tolerability, with particular focus on cardiac safety, of H and P with chemotherapy in the neoadjuvant treatment of HER2-positive early breast cancer.  PATIENTS AND METHODS:  In this multicenter, open-label phase II study, patients with operable, locally advanced, or inflammatory breast cancer were randomized 1 : 1 : 1 to receive six neoadjuvant cycles q3w (Arm A: 5-fluorouracil, epirubicin, cyclophosphamide [FEC] + H + P ×3 → docetaxel [T] + H + P ×3; Arm B: FEC ×3 → T + H + P ×3; Arm C: T + carboplatin + H [TCH]+P ×6). pCR was assessed at surgery and adjuvant therapy given to complete 1 year of H.  RESULTS:  Two hundred twenty-five patients were randomized.  During neoadjuvant treatment, two patients (2.7%; Arm B) experienced symptomatic left ventricular systolic dysfunction (LVSD) and 11 patients (Arm A: 4 [5.6%]; Arm B: 4 [5.3%]; Arm C: 3 [3.9%]) had declines in left ventricular ejection fraction of ≥10% points from baseline to <50%.  Diarrhea was the most common adverse event. pCR (ypT0/is) was reported for 61.6% (Arm A), 57.3% (Arm B), and 66.2% (Arm C) of patients.  CONCLUSION:  The combination of P with H and standard chemotherapy resulted in low rates of symptomatic LVSD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_ysMXSzWrcw20zIbnCCI3fW6udTcc2ebiIe8zPvZM_rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snmsVOqsg%253D%253D&md5=921bf406259fbe2207740b5a19bba16c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdt182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdt182%26sid%3Dliteratum%253Aachs%26aulast%3DSchneeweiss%26aufirst%3DA.%26aulast%3DChia%26aufirst%3DS.%26aulast%3DHickish%26aufirst%3DT.%26aulast%3DHarvey%26aufirst%3DV.%26aulast%3DEniu%26aufirst%3DA.%26aulast%3DHegg%26aufirst%3DR.%26aulast%3DTausch%26aufirst%3DC.%26aulast%3DSeo%26aufirst%3DJ.%2BH.%26aulast%3DTsai%26aufirst%3DY.-F.%26aulast%3DRatnayake%26aufirst%3DJ.%26aulast%3DMcNally%26aufirst%3DV.%26aulast%3DRoss%26aufirst%3DG.%26aulast%3DCort%25C3%25A9s%26aufirst%3DJ.%26atitle%3DPertuzumab%2520plus%2520trastuzumab%2520in%2520combination%2520with%2520standard%2520neoadjuvant%2520anthracycline-containing%2520and%2520anthracycline-free%2520chemotherapy%2520regimens%2520in%2520patients%2520with%2520HER2-positive%2520early%2520breast%2520cancer%253A%2520A%2520randomized%2520phase%2520II%2520cardiac%2520safety%2520study%2520%2528TRYPHAENA%2529%26jtitle%3DAnn.%2520Oncol.%26date%3D2013%26volume%3D24%26spage%3D2278%26epage%3D2284%26doi%3D10.1093%2Fannonc%2Fmdt182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nahta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L. X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, F. J.</span></span> <span> </span><span class="NLM_article-title">Lapatinib induces apoptosis in trastuzumab-resistant breast cancer cells: effects on insulin-like growth factor I signaling</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-06-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1535-7163.mct-06-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17308062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=667-674&author=R.+Nahtaauthor=L.+X.+H.+Yuanauthor=Y.+Duauthor=F.+J.+Esteva&title=Lapatinib+induces+apoptosis+in+trastuzumab-resistant+breast+cancer+cells%3A+effects+on+insulin-like+growth+factor+I+signaling&doi=10.1158%2F1535-7163.mct-06-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lapatinib induces apoptosis in trastuzumab-resistant breast cancer cells: Effects on insulin-like growth factor I signaling</span></div><div class="casAuthors">Nahta, Rita; Yuan, Linda X. H.; Du, Yi; Esteva, Francisco J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">667-674</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The majority of breast cancer patients who achieve an initial therapeutic response to the HER2-targeted antibody trastuzumab will show disease progression within 1 yr.  Thus, the identification of novel agents that effectively inhibit survival of cancer cells that have progressed on trastuzumab is crit.  In the current study, we show that the dual epidermal growth factor receptor (EGFR)/human EGFR-2 (HER2) kinase inhibitor lapatinib induces apoptosis in trastuzumab-resistant cells derived from the HER2-overexpressing SKBR3 breast cancer line.  Lapatinib inhibited EGFR and HER2 signaling in resistant cells, blocking activation of downstream Akt, mitogen-activated protein kinase, and S6 kinases and inducing expression of p27kip1.  Importantly, lapatinib also inhibited insulin-like growth factor I (IGF-I) signaling and growth-promoting effects in parental and resistant cells, and the cytotoxic effects of lapatinib were further enhanced by the IGF-I receptor-blocking antibody αIR3.  As increased IGF-I receptor signaling has been implicated in trastuzumab resistance, our data strongly support further study of lapatinib as a potential therapeutic in breast cancers that have progressed on trastuzumab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgA2XSBOQGErVg90H21EOLACvtfcHk0lj16Bkg1wHICg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWrsbc%253D&md5=4bee464a1d992a25a4c5f832cb4b0086</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-06-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-06-0423%26sid%3Dliteratum%253Aachs%26aulast%3DNahta%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DL.%2BX.%2BH.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DEsteva%26aufirst%3DF.%2BJ.%26atitle%3DLapatinib%2520induces%2520apoptosis%2520in%2520trastuzumab-resistant%2520breast%2520cancer%2520cells%253A%2520effects%2520on%2520insulin-like%2520growth%2520factor%2520I%2520signaling%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2007%26volume%3D6%26spage%3D667%26epage%3D674%26doi%3D10.1158%2F1535-7163.mct-06-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eli, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalani, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J.</span></span> <span> </span><span class="NLM_article-title">Preclinical characteristics of the irreversible pan- her kinase inhibitor neratinib compared with lapatinib: implications for the treatment of HER2- positive and HER2-mutated breast cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">737</span>, <span class="refDoi"> DOI: 10.3390/cancers11060737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3390%2Fcancers11060737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtlCktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=737&author=D.+M.+Collinsauthor=N.+T.+Conlonauthor=S.+Kannanauthor=C.+S.+Vermaauthor=L.+D.+Eliauthor=A.+S.+Lalaniauthor=J.+Crown&title=Preclinical+characteristics+of+the+irreversible+pan-+her+kinase+inhibitor+neratinib+compared+with+lapatinib%3A+implications+for+the+treatment+of+HER2-+positive+and+HER2-mutated+breast+cancer&doi=10.3390%2Fcancers11060737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical characteristics of the irreversible pan-HER kinase inhibitor neratinib compared with lapatinib: implications for the treatment of HER2-positive and HER2-mutated breast cancer</span></div><div class="casAuthors">Collins, Denis M.; Conlon, Neil T.; Kannan, Srinivasaraghavan; Verma, Chandra S.; Eli, Lisa D.; Lalani, Alshad S.; Crown, John</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">737</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">An estd. 15-20% of breast cancers overexpress human epidermal growth factor receptor 2 (HER2/ERBB2/neu).  Two small-mol. tyrosine kinase inhibitors (TKIs), lapatinib and neratinib, have been approved for the treatment of HER2-pos. (HER2+) breast cancer.  Lapatinib, a reversible epidermal growth factor receptor (EGFR/ERBB1/HER1) and HER2 TKI, is used for the treatment of advanced HER2+ breast cancer in combination with capecitabine, in combination with trastuzumab in patients with hormone receptor-neg. metastatic breast cancer, and in combination with an aromatase inhibitor for the first-line treatment of HER2+ breast cancer.  Neratinib, a next-generation, irreversible pan-HER TKI, is used in the US for extended adjuvant treatment of adult patients with early-stage HER2+ breast cancer following 1 yr of trastuzumab.  In Europe, neratinib is used in the extended adjuvant treatment of adult patients with early-stage hormone receptor-pos. HER2+ breast cancer who are less than 1 yr from the completion of prior adjuvant trastuzumab-based therapy.  Preclin. studies have shown that these agents have distinct properties that may impact their clin. activity.  This review describes the preclin. characterization of lapatinib and neratinib, with a focus on the differences between these two agents that may have implications for patient management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4tzLmkuc0rVg90H21EOLACvtfcHk0lhHW8wymNm4xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtlCktrg%253D&md5=c1d2bb10e99b843b9bf028734994868d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fcancers11060737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11060737%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DD.%2BM.%26aulast%3DConlon%26aufirst%3DN.%2BT.%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DEli%26aufirst%3DL.%2BD.%26aulast%3DLalani%26aufirst%3DA.%2BS.%26aulast%3DCrown%26aufirst%3DJ.%26atitle%3DPreclinical%2520characteristics%2520of%2520the%2520irreversible%2520pan-%2520her%2520kinase%2520inhibitor%2520neratinib%2520compared%2520with%2520lapatinib%253A%2520implications%2520for%2520the%2520treatment%2520of%2520HER2-%2520positive%2520and%2520HER2-mutated%2520breast%2520cancer%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D737%26doi%3D10.3390%2Fcancers11060737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><a href="https://www.fda.gov/drugs/new-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products/new-drug-therapy-approvals-2019" class="extLink">https://www.fda.gov/drugs/new-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products/new-drug-therapy-approvals-2019</a>. (accessed Jan 18, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fnew-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products%2Fnew-drug-therapy-approvals-2019.+%28accessed+Jan+18%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">How to train your inhibitor: design strategies to overcome resistance to epidermal growth factor receptor inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.07.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2017.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28754471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GktLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=131-151&author=S.+N.+Milikauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=K.+A.+M.+Abouzid&title=How+to+train+your+inhibitor%3A+design+strategies+to+overcome+resistance+to+epidermal+growth+factor+receptor+inhibitors&doi=10.1016%2Fj.ejmech.2017.07.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">How to train your inhibitor: Design strategies to overcome resistance to Epidermal Growth Factor Receptor inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Lasheen, Deena S.; Serya, Rabah A. T.; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-151</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Epidermal Growth Factor Receptor (EGFR) stands out as a key player in the development of many cancers.  Its dysregulation is assocd. with a vast no. of tumors such as nonsmall-cell lung cancer, colon cancer, head-and-neck cancer, breast and ovarian cancer.  Being implicated in the development of a no. of the most lethal cancers worldwide, EGFR has long been considered as a focal target for cancer therapies, ever since the FDA approval of "Gefitinib" in 2003 and up to the last FDA approved small mol. EGFR kinase inhibitor "Osimertinib" in 2015.  Studies are still going on to find more efficient EGFR inhibitors due to the continuous emergence of resistance to the current inhibitors.  Cancerous cells resist EGFR tyrosine kinase inhibitors (TKIs) through various mechanisms, the most commonly reported ones are the T790M mutation and HER2 amplification.  Therefore, tackling EGFR TKIs-resistant tumors through a multitargeting approach comprising a dual EGFR/HER2 inhibitor that is also capable of inhibiting the mutant T790M EGFR is anticipated to overcome drug resistance.  In this review, the authors will survey the structural aspects of EGFR family and the structure-activity relationship of representative dual EGFR/HER2 inhibitors.  To follow, the authors will discuss the structural aspects of the mutation-driven resistance and various design strategies to overcome it.  Finally, the authors will review the SAR of exemplary irreversible dual EGFR/HER2 inhibitors that can overcome the mutation-driven resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY7kCQB9fXZbVg90H21EOLACvtfcHk0lhHW8wymNm4xA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GktLjK&md5=689e6f4e9a377f847da751187b1875bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DHow%2520to%2520train%2520your%2520inhibitor%253A%2520design%2520strategies%2520to%2520overcome%2520resistance%2520to%2520epidermal%2520growth%2520factor%2520receptor%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D131%26epage%3D151%26doi%3D10.1016%2Fj.ejmech.2017.07.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: gefitinib (ZD1839) (iressa(R)) tablets</span>. <i>Oncologista</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.8-4-303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.8-4-303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12897327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFSqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=303-306&author=M.+H.+Cohenauthor=G.+A.+Williamsauthor=R.+Sridharaauthor=G.+Chenauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+gefitinib+%28ZD1839%29+%28iressa%28R%29%29+tablets&doi=10.1634%2Ftheoncologist.8-4-303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: gefitinib (ZD1839) (Iressa) tablets</span></div><div class="casAuthors">Cohen, Martin H.; Williams, Grant A.; Sridhara, Rajeshwari; Chen, Gang; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">303-306</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  On May 5, 2003, gefitinib (Iressa, ZD1839) 250-mg tablets received accelerated approval by the U.S. Food and Drug Administration as monotherapy treatment for patients with locally advanced or metastatic non-small cell lung cancer (NSCLC) after failure of both platinum-based and docetaxel chemotherapies.  Information provided in this summary includes efficacy and safety results of relevant clin. trials.  Effectiveness was demonstrated in a randomized, double-blind, phase II, multicenter trial comparing two oral doses of gefitinib (250 mg/day vs. 500 mg/day).  Two hundred sixteen patients were enrolled.  The 142 patients who were refractory to or intolerant of a platinum and docetaxel comprised the evaluable population for the efficacy anal.  A partial tumor response occurred in 14% (9 of 66) of patients receiving gefitinib 250 mg/day and in 8% (6 of 76) of patients receiving gefitinib 500 mg/day.  The overall objective response rate for both doses combined was 10.6% (15 of 142 patients) (95% confidence interval 6.0%-16.8%).  Responses were more frequent in females and in nonsmokers.  The median duration of response was 7.0 mo (range 4.6-18.6+ months).  Other submitted data included the results of two large trials conducted in chemotherapy-naive, stage III and IV NSCLC patients.  Patients were randomized to receive gefitinib (250 mg or 500 mg daily) or placebo, in combination with either gemcitabine plus cisplatin (n = 1,093) or carboplatin plus paclitaxel (n = 1,037).  Results from those studies showed no benefit (response rate, time to progression, or survival) from adding gefitinib to chemotherapy.  Consequently, gefitinib is only recommended for use as monotherapy.  Common adverse events assocd. with gefitinib treatment included diarrhea, rash, acne, dry skin, nausea, and vomiting.  Most toxicities were Common Toxicity Criteria grade 1 or 2.  Interstitial lung disease (ILD) has been obsd. in patients receiving gefitinib.  Worldwide, the incidence of ILD is about 1% (2% in the Japanese postmarketing experience and about 0.3% in a U.S. expanded access program).  Approx. one-third of the cases were fatal.  Physicians should promptly evaluate new or worsening pulmonary symptoms.  If ILD is confirmed, appropriate management includes discontinuation of gefitinib.  Gefitinib was approved under accelerated approval regulations on the basis of a surrogate end point response rate.  No controlled gefitinib trials, to date, demonstrate a clin. benefit, such as improvement in disease-related symptoms or greater survival.  Accelerated approval regulations require the sponsor to conduct further studies to verify that gefitinib therapy produces such a benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqLYgwUCCwY7Vg90H21EOLACvtfcHk0lilkc9npufNGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFSqtrc%253D&md5=b05bb79ab3ac78dbb34f617311339ae0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.8-4-303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.8-4-303%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DWilliams%26aufirst%3DG.%2BA.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520gefitinib%2520%2528ZD1839%2529%2520%2528iressa%2528R%2529%2529%2520tablets%26jtitle%3DOncologista%26date%3D2003%26volume%3D8%26spage%3D303%26epage%3D306%26doi%3D10.1634%2Ftheoncologist.8-4-303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: erlotinib (Tarceva) tablets</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.10-7-461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.10-7-461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16079312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=461-466&author=M.+H.+Cohenauthor=J.+R.+Johnsonauthor=Y.+F.+Chenauthor=R.+Sridharaauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+erlotinib+%28Tarceva%29+tablets&doi=10.1634%2Ftheoncologist.10-7-461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: erlotinib (Tarceva) tablets</span></div><div class="casAuthors">Cohen, Martin H.; Johnson, John R.; Chen, Yeh-Fong; Sridhara, Rajeshwari; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">461-466</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">On Nov. 18, 2004, erlotinib (Tarceva; OSI Pharmaceuticals, Inc., Melville, NY, http://www.osip.com, and Genentech, Inc., South San Francisco, CA, http://www.gene.com) received regular approval as monotherapy for the treatment of patients with locally advanced or metastatic non-small cell lung cancer (NSCLC) after failure of at least one prior chemotherapy regimen.  Survival of erlotinib-treated patients was superior to that of placebo-treated patients.  The median survival duration of erlotinib-treated patients was 6.67 mo, compared with 4.70 mo for placebo-treated patients.  Exploratory univariate analyses showed a larger survival prolongation in two subsets of patients: those who never smoked and those with epidermal growth factor receptor (EGFR)-pos. tumors.  Patients who never smoked and were EGFR-pos. had a large erlotinib survival benefit.  Erlotinib was also superior to placebo for progression-free survival and a response rate of 8.9% vs. 0.9%.  Skin rash and diarrhea were the most common erlotinib adverse events.  Severe rash occurred in 8%, and severe diarrhea occurred in 6% of erlotinib-treated patients.  In the first-line treatment of NSCLC, two large, controlled, randomized trials showed no benefit from adding erlotinib to doublet, platinum-based chemotherapy.  Therefore, erlotinib is not indicated for use in this setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG6dyEamjGgrVg90H21EOLACvtfcHk0lilkc9npufNGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltrnJ&md5=e613a64fdb4a24a5e98ade5855477086</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.10-7-461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.10-7-461%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DJ.%2BR.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520erlotinib%2520%2528Tarceva%2529%2520tablets%26jtitle%3DOncologist%26date%3D2005%26volume%3D10%26spage%3D461%26epage%3D466%26doi%3D10.1634%2Ftheoncologist.10-7-461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C. w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justice, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: lapatinib in combination with capecitabine for previously treated metastatic breast cancer that overexpresses HER-2</span>. <i>Oncologista</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2008-0816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.2008-0816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18849320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGgur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=1114-1119&author=Q.+Ryanauthor=A.+Ibrahimauthor=M.+H.+Cohenauthor=J.+Johnsonauthor=C.+w.+Koauthor=R.+Sridharaauthor=R.+Justiceauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+lapatinib+in+combination+with+capecitabine+for+previously+treated+metastatic+breast+cancer+that+overexpresses+HER-2&doi=10.1634%2Ftheoncologist.2008-0816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: lapatinib in combination with capecitabine for previously treated metastatic breast cancer that overexpresses HER-2</span></div><div class="casAuthors">Ryan, Qin; Ibrahim, Amna; Cohen, Martin H.; Johnson, John; Ko, Chia-Wen; Sridhara, Rajeshwari; Justice, Robert; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1114-1119</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">On March 13, 2007, the U.S. Food and Drug Administration approved lapatinib (Tykerb tablets; Glaxo-SmithKline, Philadelphia), an oral, small mol., dual tyrosine kinase inhibitor of ErbB-2 and ErbB-1, for use in combination with capecitabine for the treatment of patients with human epidermal growth factor receptor (HER)-2-overexpressing metastatic breast cancer who had received prior therapy including an anthracycline, a taxane, and trastuzumab.  One multicenter, open-label, randomized trial was submitted.  Eligible patients had stage IIIb or IV breast cancer, ErbB-2 overexpression (immunohistochem. 3+ or 2+ with fluorescence in situ hybridization confirmation), measurable disease, a 0 or 1 Eastern Cooperative Oncol. Group performance status score, a cardiac ejection fraction within the institutional normal range, and adequate lab. function.  Patients received either lapatinib (1,250 mg once daily on days 1-21) plus capecitabine (1,000 mg/m2 every 12 h on days 1-14) every 21 days or capecitabine alone (1,250 mg/m2 every 12 h on days 1-14) every 21 days.  The primary endpoint was time to progression (TTP) detd. by a blinded independent review panel.  After TTP results of a prespecified interim anal. were made available, study enrollment was discontinued (399 patients enrolled).  The median TTP was 27.1 vs. 18.6 wk (hazard ratio, 0.57; p = .00013) favoring the lapatinib plus capecitabine arm.  Response rates were 23.7% (lapatinib plus capecitabine) vs. 13.9% (capecitabine alone).  Survival data were not mature.  Although the toxicities obsd. in the lapatinib and capecitabine combination arm were generally similar to those in the capecitabine alone arm, a higher incidence of diarrhea and rash was noted with the combination.  Grade 3 or 4 adverse reactions that occurred with a frequency of >5% in patients on the combination arm were diarrhea (13%) and palmar-plantar erythrodysesthesia (12%).  There was a 2% incidence of reversible decreased left ventricular function in the combination arm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_M2JQVn8WCbVg90H21EOLACvtfcHk0ljhwYOEdFqLRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGgur7I&md5=618f9677ef731dcc593be4a45504470c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2008-0816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2008-0816%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DQ.%26aulast%3DIbrahim%26aufirst%3DA.%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DKo%26aufirst%3DC.%2Bw.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DJustice%26aufirst%3DR.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520lapatinib%2520in%2520combination%2520with%2520capecitabine%2520for%2520previously%2520treated%2520metastatic%2520breast%2520cancer%2520that%2520overexpresses%2520HER-2%26jtitle%3DOncologista%26date%3D2008%26volume%3D13%26spage%3D1114%26epage%3D1119%26doi%3D10.1634%2Ftheoncologist.2008-0816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affleck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubberfield, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntrip, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stables, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brignola, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadwell, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reep, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span> <span> </span><span class="NLM_article-title">The characterization of novel, dual ErbB-2/EGFR, tyrosine kinase inhibitors: Potential therapy for cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11585755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsVeht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=7196&author=D.+W.+Rusnakauthor=K.+Affleckauthor=S.+G.+Cockerillauthor=C.+Stubberfieldauthor=R.+Harrisauthor=M.+Pageauthor=K.+J.+Smithauthor=S.+B.+Guntripauthor=M.+C.+Carterauthor=R.+J.+Shawauthor=A.+Jowettauthor=J.+Stablesauthor=P.+Topleyauthor=E.+R.+Woodauthor=P.+S.+Brignolaauthor=S.+H.+Kadwellauthor=B.+R.+Reepauthor=R.+J.+Mullinauthor=K.+J.+Alligoodauthor=B.+R.+Keithauthor=R.+M.+Crosbyauthor=D.+M.+Murrayauthor=W.+B.+Knightauthor=T.+M.+Gilmerauthor=K.+Lackey&title=The+characterization+of+novel%2C+dual+ErbB-2%2FEGFR%2C+tyrosine+kinase+inhibitors%3A+Potential+therapy+for+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The characterization of novel, dual ErbB-2/EGFR, tyrosine kinase inhibitors: potential therapy for cancer</span></div><div class="casAuthors">Rusnak, David W.; Affleck, Karen; Cockerill, Stuart G.; Stubberfield, Colin; Harris, Robert; Page, Martin; Smith, Kathryn J.; Guntrip, Stephen B.; Carter, Malcolm C.; Shaw, Robert J.; Jowett, Amanda; Stables, Jeremy; Topley, Peter; Wood, Edgar R.; Brignola, Perry S.; Kadwell, Sue H.; Reep, Bryan R.; Mullin, Robert J.; Alligood, Krystal J.; Keith, Barry R.; Crosby, Renae M.; Murray, Doris M.; Knight, W. Blaine; Gilmer, Tona M.; Lackey, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7196-7203</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The type 1 receptor tyrosine kinases constitute a family of transmembrane proteins involved in various aspects of cell growth and survival and have been implicated in the initiation and progression of several types of human malignancies.  The best characterized of these proteins are the epidermal growth factor receptor (EGFR) and ErbB-2 (HER-2/neu).  We have developed potent quinazoline and pyrido-[3,4-d]-pyrimidine small mols. that are dual inhibitors of ErbB-2 and EGFR.  The compds. demonstrate potent in vitro inhibition of the ErbB-2 and EGFR kinase domains with IC50s <80 nM.  Growth of ErbB-2- and EGFR-expressing tumor cell lines is inhibited at concns. <0.5 μM.  Selectivity for tumor cell growth inhibition vs. normal human fibroblast growth inhibition ranges from 10- to >75-fold.  Tumor growth in mouse s.c. xenograft models of the BT474 and HN5 cell lines is inhibited in a dose-responsive manner using oral doses of 10 and 30 mg/kg twice per day.  In addn., the tested compds. caused a redn. of ErbB-2 and EGFR autophosphorylation in tumor fragments from these xenograft models.  These data indicate that these compds. have potential use as therapy in the broad population of cancer patients overexpressing ErbB-2 and/or EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoifUxgOrEfMLVg90H21EOLACvtfcHk0ljhwYOEdFqLRA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsVeht70%253D&md5=374cb1b4451356866a5060c5ff39a1d1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DAffleck%26aufirst%3DK.%26aulast%3DCockerill%26aufirst%3DS.%2BG.%26aulast%3DStubberfield%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DPage%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DGuntrip%26aufirst%3DS.%2BB.%26aulast%3DCarter%26aufirst%3DM.%2BC.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26aulast%3DJowett%26aufirst%3DA.%26aulast%3DStables%26aufirst%3DJ.%26aulast%3DTopley%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DBrignola%26aufirst%3DP.%2BS.%26aulast%3DKadwell%26aufirst%3DS.%2BH.%26aulast%3DReep%26aufirst%3DB.%2BR.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DCrosby%26aufirst%3DR.%2BM.%26aulast%3DMurray%26aufirst%3DD.%2BM.%26aulast%3DKnight%26aufirst%3DW.%2BB.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26aulast%3DLackey%26aufirst%3DK.%26atitle%3DThe%2520characterization%2520of%2520novel%252C%2520dual%2520ErbB-2%252FEGFR%252C%2520tyrosine%2520kinase%2520inhibitors%253A%2520Potential%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2001%26volume%3D61%26spage%3D7196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truesdale, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickerson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennisi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shewchuk, L.</span></span> <span> </span><span class="NLM_article-title">A unique structure for epidermal growth factor receptor bound to GW572016 (Lapatinib): Relationships among protein conformation, inhibitor off-rate, and receptor activity in tumor cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">6652</span>– <span class="NLM_lpage">6659</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-04-1168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-04-1168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=15374980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFOlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=6652-6659&author=E.+R.+Woodauthor=A.+T.+Truesdaleauthor=O.+B.+McDonaldauthor=D.+Yuanauthor=A.+Hassellauthor=S.+H.+Dickersonauthor=B.+Ellisauthor=C.+Pennisiauthor=E.+Horneauthor=K.+Lackeyauthor=K.+J.+Alligoodauthor=D.+W.+Rusnakauthor=T.+M.+Gilmerauthor=L.+Shewchuk&title=A+unique+structure+for+epidermal+growth+factor+receptor+bound+to+GW572016+%28Lapatinib%29%3A+Relationships+among+protein+conformation%2C+inhibitor+off-rate%2C+and+receptor+activity+in+tumor+cells&doi=10.1158%2F0008-5472.can-04-1168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A unique structure for epidermal growth factor receptor bound to GW572016 (Lapatinib): relationships among protein conformation, inhibitor off-rate, and receptor activity in tumor cells</span></div><div class="casAuthors">Wood, Edgar R.; Truesdale, Anne T.; McDonald, Octerloney B.; Yuan, Derek; Hassell, Anne; Dickerson, Scott H.; Ellis, Byron; Pennisi, Christopher; Horne, Earnest; Lackey, Karen; Alligood, Krystal J.; Rusnak, David W.; Gilmer, Tona M.; Shewchuk, Lisa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6652-6659</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">GW572016 (Lapatinib) is a tyrosine kinase inhibitor in clin. development for cancer that is a potent dual inhibitor of epidermal growth factor receptor (EGFR, ErbB-1) and ErbB-2.  The authors detd. the crystal structure of EGFR bound to GW572016.  The compd. is bound to an inactive-like conformation of EGFR that is very different from the active-like structure bound by the selective EGFR inhibitor OSI-774 (Tarceva) described previously.  Surprisingly, the authors found that GW572016 has a very slow off-rate from the purified intracellular domains of EGFR and ErbB-2 compared with OSI-774 and another EGFR selective inhibitor, ZD-1839 (Iressa).  Treatment of tumor cells with these inhibitors results in down-regulation of receptor tyrosine phosphorylation.  The authors evaluated the duration of the drug effect after washing away free compd. and found that the rate of recovery of receptor phosphorylation in the tumor cells reflected the inhibitor off-rate from the purified intracellular domain.  The slow off-rate of GW572016 correlates with a prolonged down-regulation of receptor tyrosine phosphorylation in tumor cells.  The differences in the off-rates of these drugs and the ability of GW572016 to inhibit ErbB-2 can be explained by the enzyme-inhibitor structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXjJPy6XYk4rVg90H21EOLACvtfcHk0li70Bhe3AL9Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFOlu7g%253D&md5=788d92fc5db9f1ecf9d3fbf3142e5c9f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-04-1168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-04-1168%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DTruesdale%26aufirst%3DA.%2BT.%26aulast%3DMcDonald%26aufirst%3DO.%2BB.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DHassell%26aufirst%3DA.%26aulast%3DDickerson%26aufirst%3DS.%2BH.%26aulast%3DEllis%26aufirst%3DB.%26aulast%3DPennisi%26aufirst%3DC.%26aulast%3DHorne%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26aulast%3DShewchuk%26aufirst%3DL.%26atitle%3DA%2520unique%2520structure%2520for%2520epidermal%2520growth%2520factor%2520receptor%2520bound%2520to%2520GW572016%2520%2528Lapatinib%2529%253A%2520Relationships%2520among%2520protein%2520conformation%252C%2520inhibitor%2520off-rate%252C%2520and%2520receptor%2520activity%2520in%2520tumor%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D6652%26epage%3D6659%26doi%3D10.1158%2F0008-5472.can-04-1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0li70Bhe3AL9Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadek, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafafy, A.-H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of new HER2/EGFR dual kinase inhibitors based on the anilinoquinazoline scaffold as potential anti-cancer agents</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.3109/14756366.2013.765417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3109%2F14756366.2013.765417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23402383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktVSmur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=215-222&author=M.+M.+Sadekauthor=R.+A.+Serryaauthor=A.-H.+N.+Kafafyauthor=M.+Ahmedauthor=F.+Wangauthor=K.+A.+M.+Abouzid&title=Discovery+of+new+HER2%2FEGFR+dual+kinase+inhibitors+based+on+the+anilinoquinazoline+scaffold+as+potential+anti-cancer+agents&doi=10.3109%2F14756366.2013.765417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new HER2/EGFR dual kinase inhibitors based on the anilinoquinazoline scaffold as potential anti-cancer agents</span></div><div class="casAuthors">Sadek, Maiada M.; Serrya, Rabah A.; Kafafy, Abdel-Hamid N.; Ahmed, Marawan; Wang, Feng; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">215-222</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Herein, we designed and synthesized certain anilinoquinazoline derivs. bearing bulky arylpyridinyl, arylpropenoyl and arylpyrazolyl moieties at the 4' position of the anilinoquinazoline, as potential dual HER2/EGFR kinase inhibitors.  A detailed mol. modeling study was performed by docking the synthesized compds. in the active site of the epidermal growth factor receptor (EGFR).  The synthesized compds. were further tested for their inhibitory activity on EGFR and HER2 tyrosine kinases.  The aryl 2-imino-1,2-dihydropyridine derivs. 5d and 5e displayed the most potent inhibitory activity on EGFR with IC50 equal to 2.09 and 1.94 μM, resp., and with IC50 equal to 3.98 and 1.04 μM on HER2, resp.  Furthermore, the anti-proliferative activity of these most active compds. on MDA-MB-231 breast cancer cell lines, known to overexpress EGFR, showed an IC50 range of 2.4 and 2.5 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_wzITyW3RbVg90H21EOLACvtfcHk0li70Bhe3AL9Uw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktVSmur4%253D&md5=47495022661acb01543e95b85e0cf1a2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3109%2F14756366.2013.765417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F14756366.2013.765417%26sid%3Dliteratum%253Aachs%26aulast%3DSadek%26aufirst%3DM.%2BM.%26aulast%3DSerrya%26aufirst%3DR.%2BA.%26aulast%3DKafafy%26aufirst%3DA.-H.%2BN.%26aulast%3DAhmed%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DDiscovery%2520of%2520new%2520HER2%252FEGFR%2520dual%2520kinase%2520inhibitors%2520based%2520on%2520the%2520anilinoquinazoline%2520scaffold%2520as%2520potential%2520anti-cancer%2520agents%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2014%26volume%3D29%26spage%3D215%26epage%3D222%26doi%3D10.3109%2F14756366.2013.765417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukouvala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gendreau, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfrey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vysotskaia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, T. S.</span></span> <span> </span><span class="NLM_article-title">EXEL-7647 inhibits mutant forms of ErbB2 associated with lapatinib resistance and neoplastic transformation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-07-4367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-07-4367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18413839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=2465-2475&author=T.+Troweauthor=S.+Boukouvalaauthor=K.+Calkinsauthor=R.+E.+Cutlerauthor=R.+Fongauthor=R.+Funkeauthor=S.+B.+Gendreauauthor=Y.+D.+Kimauthor=N.+Millerauthor=J.+R.+Woolfreyauthor=V.+Vysotskaiaauthor=J.+P.+Yangauthor=M.+E.+Gerritsenauthor=D.+J.+Matthewsauthor=P.+Lambauthor=T.+S.+Heuer&title=EXEL-7647+inhibits+mutant+forms+of+ErbB2+associated+with+lapatinib+resistance+and+neoplastic+transformation&doi=10.1158%2F1078-0432.ccr-07-4367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">EXEL-7647 Inhibits Mutant Forms of ErbB2 Associated with Lapatinib Resistance and Neoplastic Transformation</span></div><div class="casAuthors">Trowe, Torsten; Boukouvala, Sotiria; Calkins, Keith; Cutler, Richard E., Jr.; Fong, Ryan; Funke, Roel; Gendreau, Steven B.; Kim, Yong D.; Miller, Nicole; Woolfrey, John R.; Vysotskaia, Valentina; Yang, Jing Ping; Gerritsen, Mary E.; Matthews, David J.; Lamb, Peter; Heuer, Timothy S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2465-2475</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations assocd. with resistance to kinase inhibition are an important mechanism of intrinsic or acquired loss of clin. efficacy for kinase-targeted therapeutics.  We report the prospective discovery of ErbB2 mutations that confer resistance to the small-mol. inhibitor lapatinib.  We did in vitro screening using a randomly mutagenized ErbB2 expression library in Ba/F3 cells, which were dependent on ErbB2 activity for survival and growth.  Lapatinib resistance screens identified mutations at 16 different ErbB2 amino acid residues, with 12 mutated amino acids mapping to the kinase domain.  Mutations conferring the greatest lapatinib resistance cluster in the NH2-terminal kinase lobe and hinge region.  Structural computer modeling studies suggest that lapatinib resistance is caused by multiple mechanisms; including direct steric interference and restriction of conformational flexibility (the inactive state required for lapatinib binding is energetically unfavorable).  ErbB2 T798I imparts the strongest lapatinib resistance effect and is analogous to the epidermal growth factor receptor T790M, ABL T315I, and cKIT T670I gatekeeper mutations that are assocd. with clin. drug resistance.  ErbB2 mutants assocd. with lapatinib resistance transformed NIH-3T3 cells, including L755S and T733I mutations known to occur in human breast and gastric carcinomas, supporting a direct mechanism for lapatinib resistance in ErbB2-driven human cancers.  The epidermal growth factor receptor/ErbB2/vascular endothelial growth factor receptor inhibitor EXEL-7647 was found to inhibit almost all lapatinib resistance-assocd. mutations.  Furthermore, no ErbB2 mutations were found to be assocd. with EXEL-7647 resistance and lapatinib sensitivity.  Taken together, these data suggest potential target-based mechanisms of resistance to lapatinib and suggest that EXEL-7647 may be able to circumvent these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIufvHezxJo7Vg90H21EOLACvtfcHk0lhBQO5bVjvEQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D&md5=c51774479df7be7c06d2cc22e8c58afc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-07-4367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-07-4367%26sid%3Dliteratum%253Aachs%26aulast%3DTrowe%26aufirst%3DT.%26aulast%3DBoukouvala%26aufirst%3DS.%26aulast%3DCalkins%26aufirst%3DK.%26aulast%3DCutler%26aufirst%3DR.%2BE.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGendreau%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BD.%26aulast%3DMiller%26aufirst%3DN.%26aulast%3DWoolfrey%26aufirst%3DJ.%2BR.%26aulast%3DVysotskaia%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DJ.%2BP.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DMatthews%26aufirst%3DD.%2BJ.%26aulast%3DLamb%26aufirst%3DP.%26aulast%3DHeuer%26aufirst%3DT.%2BS.%26atitle%3DEXEL-7647%2520inhibits%2520mutant%2520forms%2520of%2520ErbB2%2520associated%2520with%2520lapatinib%2520resistance%2520and%2520neoplastic%2520transformation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D2465%26epage%3D2475%26doi%3D10.1158%2F1078-0432.ccr-07-4367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Diarrhea With HER2-Targeted Agents in Cancer Patients: A Systematic Review and Meta-Analysis</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">946</span>, <span class="refDoi"> DOI: 10.1002/jcph.1382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fjcph.1382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30707455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyjs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=935-946&author=J.+Li&title=Diarrhea+With+HER2-Targeted+Agents+in+Cancer+Patients%3A+A+Systematic+Review+and+Meta-Analysis&doi=10.1002%2Fjcph.1382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Diarrhea With HER2-Targeted Agents in Cancer Patients: A Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Li, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7, VIII International Conference Nanomaterials and Technologies, 2019</span>),
    <span class="NLM_cas:pages">935-946</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To fully investigate the diarrhea of human epidermal growth factor receptor 2 (HER2)-targeted agents in cancer patients.  The relevant studies of the randomized, controlled trials (RCTs) in cancer patients treated with HER2-targeted agents were retrieved, and the systematic evaluation was conducted.  EMBASE, MEDLINE, and PubMed were searched for articles published until August 2018.  Forty-two RCTs and 21 633 patients were included.  The current meta-anal. suggested that the use of HER2-targeted agents significantly increases the risk of developing all-grade diarrhea (RR, 2.78; 95%CI, 2.37-3.25; P < .00001) and high-grade diarrhea (RR, 4.89; 95%CI, 3.09-7.75; P < .00001).  The RRs of all-grade diarrhea and high-grade diarrhea varied significantly according to drug type, control group, and treatment regimen.  The RR of all-grade diarrhea varied significantly according to tumor type.  Afatinib and neratinib tended to assoc. with the highest risk of all-grade diarrhea and high-grade diarrhea, resp.  Trastuzumab was assocd. with the lowest risk of diarrhea.  Breast cancer patients tended to have a higher risk of all-grade diarrhea than patients with nonbreast cancer when receiving a HER2-targeted agent.  The available data suggested that the use of HER2-targeted agents is assocd. with a significantly increased risk of diarrhea in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTXA94GBpC27Vg90H21EOLACvtfcHk0lhBQO5bVjvEQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyjs7jL&md5=5052d8511a449cd2581b848fc165f511</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcph.1382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.1382%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DDiarrhea%2520With%2520HER2-Targeted%2520Agents%2520in%2520Cancer%2520Patients%253A%2520A%2520Systematic%2520Review%2520and%2520Meta-Analysis%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2019%26volume%3D59%26spage%3D935%26epage%3D946%26doi%3D10.1002%2Fjcph.1382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee-Hoeflich, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crocker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munroe, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, H. M.</span></span> <span> </span><span class="NLM_article-title">A central role for HER3 in HER2-amplified breast cancer: Implications for targeted therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">5878</span>– <span class="NLM_lpage">5887</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-08-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-08-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18632642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovVCnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=5878-5887&author=S.+T.+Lee-Hoeflichauthor=L.+Crockerauthor=E.+Yaoauthor=T.+Phamauthor=X.+Munroeauthor=K.+P.+Hoeflichauthor=M.+X.+Sliwkowskiauthor=H.+M.+Stern&title=A+central+role+for+HER3+in+HER2-amplified+breast+cancer%3A+Implications+for+targeted+therapy&doi=10.1158%2F0008-5472.can-08-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Central Role for HER3 in HER2-Amplified Breast Cancer: Implications for Targeted Therapy</span></div><div class="casAuthors">Lee-Hoeflich, Si Tuen; Crocker, Lisa; Yao, Evelyn; Pham, Thinh; Munroe, Xander; Hoeflich, Klaus P.; Sliwkowski, Mark X.; Stern, Howard M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5878-5887</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) and HER3 each form heterodimers with HER2 and have independently been implicated as key coreceptors that drive HER2-amplified breast cancer.  Some studies suggest a dominant role for EGFR, a notion of renewed interest given the development of dual HER2/EGFR small-mol. inhibitors.  Other studies point to HER3 as the primary coreceptor.  To clarify the relative contributions of EGFR and HER3 to HER2 signaling, the authors studied receptor knockdown via small interfering RNA technol. across a panel of 6 HER2-overexpressing cell lines.  Interestingly, HER3 was as crit. as HER2 for maintaining cell proliferation in most cell lines, whereas EGFR was dispensable.  Induction of HER3 knockdown in the HER2-overexpressing BT474M1 cell line was found to inhibit growth in 3-dimensional culture and induce rapid tumor regression of in vivo xenografts.  Furthermore, preferential phosphorylation of HER3, but not EGFR, was obsd. in HER2-amplified breast cancer tissues.  Given these data suggesting HER3 as an important therapeutic target, the authors examd. the activity of pertuzumab, a HER2 antibody that inhibits HER3 signaling by blocking ligand-induced HER2/HER3 heterodimerization.  Pertuzumab inhibited ligand-dependent morphogenesis in 3-dimensional culture and induced tumor regression in the heregulin-dependent MDA-MB-175 xenograft model.  Importantly, these activities of pertuzumab were distinct from those of trastuzumab, a monoclonal antibody currently used for treatment of HER2-amplified breast cancer patients.  Thus, inhibition of HER3 may be more clin. relevant than inhibition of EGFR in HER2-amplified breast cancer and also adding pertuzumab to trastuzumab may augment therapeutic benefit by blocking HER2/HER3 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9dxNZ1FF4bVg90H21EOLACvtfcHk0lhBQO5bVjvEQQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovVCnsbs%253D&md5=de1600e07012542e98eea06cc2db60d8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-08-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-08-0380%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Hoeflich%26aufirst%3DS.%2BT.%26aulast%3DCrocker%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DE.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DMunroe%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DStern%26aufirst%3DH.%2BM.%26atitle%3DA%2520central%2520role%2520for%2520HER3%2520in%2520HER2-amplified%2520breast%2520cancer%253A%2520Implications%2520for%2520targeted%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D5878%26epage%3D5887%26doi%3D10.1158%2F0008-5472.can-08-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson-Fisher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent Lloyd, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, D. F.</span></span> <span> </span><span class="NLM_article-title">Formation of Neu/ErbB2-induced mammary tumors is unaffected by loss of ErbB4</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5664</span>– <span class="NLM_lpage">5672</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1209574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fsj.onc.1209574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16652155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVOrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=5664-5672&author=A.+J.+Jackson-Fisherauthor=G.+Bellingerauthor=E.+Shumauthor=J.+K.+Duongauthor=A.+S.+Perkinsauthor=M.+Gassmannauthor=W.+Mullerauthor=K.+C.+Kent+Lloydauthor=D.+F.+Stern&title=Formation+of+Neu%2FErbB2-induced+mammary+tumors+is+unaffected+by+loss+of+ErbB4&doi=10.1038%2Fsj.onc.1209574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of Neu/ErbB2-induced mammary tumors is unaffected by loss of ErbB4</span></div><div class="casAuthors">Jackson-Fisher, A. J.; Bellinger, G.; Shum, E.; Duong, J. K.; Perkins, A. S.; Gassmann, M.; Muller, W.; Kent Lloyd, K. C.; Stern, D. F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5664-5672</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The four members of the ErbB family of receptor tyrosine kinases are involved in development and tumorigenesis of the mammary gland.  Whereas the epidermal growth factor receptor, ErbB2 and ErbB3 are pos. assocd. with various cancers, clin. studies of ErbB4 in breast cancer are contradictory.  Results from tissue culture analyses and some clin. studies suggested that ErbB4 is either a tumor suppressor or is a neg. regulator of ErbB2-driven tumors.  Neu-Cre-ErbB4flox/null mice in which ErbB4 was inactivated by Cre-lox-mediated recombination in the mammary gland developed MMTV-Neu-driven mammary tumors with a similar latency period to mice with one or two wild-type ErbB4 alleles.  Moreover, there was no difference in the histologies of tumors that developed, nor in the propensity to form lung metastases.  Taken together these results suggest that ErbB4 is not a potent, highly penetrant tumor suppressor, nor is it a factor in Neu-mediated tumorigenesis in this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6EvUyXV8xJLVg90H21EOLACvtfcHk0li9Ij6CYHY6Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVOrsrg%253D&md5=12fc4fac982975e27b5f88be36cd74f6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1209574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1209574%26sid%3Dliteratum%253Aachs%26aulast%3DJackson-Fisher%26aufirst%3DA.%2BJ.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DShum%26aufirst%3DE.%26aulast%3DDuong%26aufirst%3DJ.%2BK.%26aulast%3DPerkins%26aufirst%3DA.%2BS.%26aulast%3DGassmann%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DW.%26aulast%3DKent%2BLloyd%26aufirst%3DK.%2BC.%26aulast%3DStern%26aufirst%3DD.%2BF.%26atitle%3DFormation%2520of%2520Neu%252FErbB2-induced%2520mammary%2520tumors%2520is%2520unaffected%2520by%2520loss%2520of%2520ErbB4%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D5664%26epage%3D5672%26doi%3D10.1038%2Fsj.onc.1209574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Soler, R.</span></span> <span> </span><span class="NLM_article-title">Skin toxicities associated with epidermal growth factor receptor inhibitors</span>. <i>Target. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/s11523-009-0114-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs11523-009-0114-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19452131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1Mzos1KhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=107-119&author=T.+Liauthor=R.+Perez-Soler&title=Skin+toxicities+associated+with+epidermal+growth+factor+receptor+inhibitors&doi=10.1007%2Fs11523-009-0114-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Skin toxicities associated with epidermal growth factor receptor inhibitors</span></div><div class="casAuthors">Li Tianhong; Perez-Soler Roman</div><div class="citationInfo"><span class="NLM_cas:title">Targeted oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">107-19</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of epidermal growth factor receptor (EGFR) inhibitors in several epithelial tumors has increased considerably in recent years.  Currently, they are approved in non-small cell lung cancer (NSCLC), pancreatic cancer, colorectal cancer and head and neck cancer.  Skin toxicity is a class-specific side effect that is typically manifested as a papulopustular rash in the majority (45-100%) of patients receiving EGFR inhibitors.  The skin toxicity is related to the inhibition of EGFR in the skin, which is crucial for the normal development and physiology of the epidermis.  Although rarely life-threatening, skin toxicity may cause significant physical and psycho-social discomfort.  Nevertheless, the presence and severity of skin rash is associated with improved clinical efficacy in patients receiving EGFR inhibitors.  The goal of managing EGFR inhibitor-associated skin toxicity is to minimize the detrimental effects of the rash on patients' quality of life and treatment course without antagonizing the clinical efficacy of EGFR inhibitors.  There is currently no evidence-based treatment guideline to prevent or treat the EGFR inhibitor-associated skin toxicities.  Expert panels recommend a proactive, multidisciplinary approach that includes patient education and the use of a grade-based treatment algorithm.  Elucidation of the mechanisms of EGFR inhibitor-associated skin toxicity and development of mechanism-based novel therapies are urgently needed.  Preclinical data suggest topical application of a potent phosphatase inhibitor menadione (Vitamin K3) can rescue the inhibition of EGFR and downstream signaling molecules in the skin of mice receiving systemic EGFR inhibitor erlotinib or cetuximab.  A randomized, double-blinded, placebo-controlled study has been initiated to evaluate the clinical efficacy of menadione topical cream, in the treatment or prevention of EGFR inhibitor-induced skin toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBjiE5iFNI-Nts4TFFB11gfW6udTcc2eY_GYwU21N4OLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mzos1KhtA%253D%253D&md5=bd52132174e886f4a5d63368881e1bdb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs11523-009-0114-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11523-009-0114-0%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DPerez-Soler%26aufirst%3DR.%26atitle%3DSkin%2520toxicities%2520associated%2520with%2520epidermal%2520growth%2520factor%2520receptor%2520inhibitors%26jtitle%3DTarget.%2520Oncol.%26date%3D2009%26volume%3D4%26spage%3D107%26epage%3D119%26doi%3D10.1007%2Fs11523-009-0114-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bratslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span> <span> </span><span class="NLM_article-title">Sarcomatoid renal cell carcinoma: A comprehensive review of the biology and current treatment strategies</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2011-0227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.2011-0227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22234634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XjslGqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=46-54&author=B.+Shuchauthor=G.+Bratslavskyauthor=W.+M.+Linehanauthor=R.+Srinivasan&title=Sarcomatoid+renal+cell+carcinoma%3A+A+comprehensive+review+of+the+biology+and+current+treatment+strategies&doi=10.1634%2Ftheoncologist.2011-0227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Sarcomatoid renal cell carcinoma: a comprehensive review of the biology and current treatment strategies</span></div><div class="casAuthors">Shuch, Brian; Bratslavsky, Gennady; Linehan, W. Marston; Srinivasan, Ramaprasad</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-54</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  Recent advancements in the mol. characterization of renal cell carcinoma altered the classification system and now kidney cancer is divided into several distinct histol. subtypes.  Although once a sep. histol. category, sarcomatoid renal cell carcinoma is no longer considered a sep. tumor type because it can occur with all histol. subtypes.  Limited research on tumors with sarcomatoid change has led to minimal progress in the understanding and treatment of these tumors.  Because the sarcomatoid variant of renal cell carcinoma can account for approx. one in six cases of advanced kidney cancer, we hope to familiarize clinicians with these tumors by describing the historic background, histol. features, mol. characterization, diagnosis, prognosis, treatment strategies, and active clin. trials of this aggressive type of tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAitmhpDnN7Vg90H21EOLACvtfcHk0li9Ij6CYHY6Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjslGqsLw%253D&md5=822e5c9520c4e7ed429fff02c0237f8f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2011-0227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2011-0227%26sid%3Dliteratum%253Aachs%26aulast%3DShuch%26aufirst%3DB.%26aulast%3DBratslavsky%26aufirst%3DG.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26aulast%3DSrinivasan%26aufirst%3DR.%26atitle%3DSarcomatoid%2520renal%2520cell%2520carcinoma%253A%2520A%2520comprehensive%2520review%2520of%2520the%2520biology%2520and%2520current%2520treatment%2520strategies%26jtitle%3DOncologist%26date%3D2012%26volume%3D17%26spage%3D46%26epage%3D54%26doi%3D10.1634%2Ftheoncologist.2011-0227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0lgaH5wAZFAk0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jani, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerson, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harriman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pustilnik, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafidi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyn, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S. K.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacologic characterization of CP-724,714, a selective ErbB2 tyrosine kinase inhibitor</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">9887</span>– <span class="NLM_lpage">9893</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-06-3559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-06-3559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17942920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOnsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=9887-9893&author=J.+P.+Janiauthor=R.+S.+Finnauthor=M.+Campbellauthor=K.+G.+Colemanauthor=R.+D.+Connellauthor=N.+Currierauthor=E.+O.+Emersonauthor=E.+Floydauthor=S.+Harrimanauthor=J.+C.+Kathauthor=J.+Morrisauthor=J.+D.+Moyerauthor=L.+R.+Pustilnikauthor=K.+Rafidiauthor=S.+Ralstonauthor=A.+M.+K.+Rossiauthor=S.+J.+Steynauthor=L.+Wagnerauthor=S.+M.+Winterauthor=S.+K.+Bhattacharya&title=Discovery+and+pharmacologic+characterization+of+CP-724%2C714%2C+a+selective+ErbB2+tyrosine+kinase+inhibitor&doi=10.1158%2F0008-5472.can-06-3559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pharmacologic Characterization of CP-724,714, a Selective ErbB2 Tyrosine Kinase Inhibitor</span></div><div class="casAuthors">Jani, Jitesh P.; Finn, Richard S.; Campbell, Mary; Coleman, Kevin G.; Connell, Richard D.; Currier, Nicolas; Emerson, Erling O.; Floyd, Eugenia; Harriman, Shawn; Kath, John C.; Morris, Joel; Moyer, James D.; Pustilnik, Leslie R.; Rafidi, Kristina; Ralston, Sherry; Rossi, Ann Marie K.; Steyn, Stefanus J.; Wagner, Larry; Winter, Steven M.; Bhattacharya, Samit K.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9887-9893</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Amplification and overexpression of erbB2 (Her-2/neu) proto-oncogene has been linked to human malignancies including tumors of the breast, ovary, and stomach.  It has been implicated in tumor growth, sensitivity to std. chemotherapy, prognosis of patients, and disease-free survival.  Although the clin. use of trastuzumab (Herceptin) has prolonged the survival of breast cancer patients with erbB2-overexpressing tumors, there is an urgent need for more potent and orally bioavailable small-mol. inhibitors.  CP-724,714 is a potent inhibitor of erbB2 receptor autophosphorylation in intact cells and is currently undergoing phase I clin. trials.  Here, we describe the effects of CP-724,714 in vitro and in vivo in human breast cancer models.  CP-724,714 is selective for inhibiting growth of HER2-driven cell lines.  In addn., we show that it induces G1 cell cycle block in erbB2-overexpressing BT-474 human breast carcinoma cells and inhibits erbB2 autophosphorylation in xenografts when administered p.o. to athymic mice.  It induces a marked redn. of extracellular signal-regulated kinase and Akt phosphorylation, tumor cell apoptosis, and release of caspase-3.  P.o. administration (q.d. or b.i.d.) of CP-724,714 inhibits the growth of erbB2-overexpressing tumors in athymic mice without overt adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_a6qjvEZI-rVg90H21EOLACvtfcHk0lgaH5wAZFAk0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOnsbvE&md5=21dc688108dd46aba15694d7e3159321</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-06-3559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-06-3559%26sid%3Dliteratum%253Aachs%26aulast%3DJani%26aufirst%3DJ.%2BP.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DConnell%26aufirst%3DR.%2BD.%26aulast%3DCurrier%26aufirst%3DN.%26aulast%3DEmerson%26aufirst%3DE.%2BO.%26aulast%3DFloyd%26aufirst%3DE.%26aulast%3DHarriman%26aufirst%3DS.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DRafidi%26aufirst%3DK.%26aulast%3DRalston%26aufirst%3DS.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DWinter%26aufirst%3DS.%2BM.%26aulast%3DBhattacharya%26aufirst%3DS.%2BK.%26atitle%3DDiscovery%2520and%2520pharmacologic%2520characterization%2520of%2520CP-724%252C714%252C%2520a%2520selective%2520ErbB2%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D9887%26epage%3D9893%26doi%3D10.1158%2F0008-5472.can-06-3559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the mechanism of inhibition and allosteric activation of the kinase domain of HER2 protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18756</span>– <span class="NLM_lpage">18765</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.206193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1074%2Fjbc.m110.206193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21454582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18756-18765&author=K.+Aertgeertsauthor=R.+Skeneauthor=J.+Yanoauthor=B.-C.+Sangauthor=H.+Zouauthor=G.+Snellauthor=A.+Jenningsauthor=K.+Iwamotoauthor=N.+Habukaauthor=A.+Hirokawaauthor=T.+Ishikawaauthor=T.+Tanakaauthor=H.+Mikiauthor=Y.+Ohtaauthor=S.+Sogabe&title=Structural+analysis+of+the+mechanism+of+inhibition+and+allosteric+activation+of+the+kinase+domain+of+HER2+protein&doi=10.1074%2Fjbc.m110.206193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Mechanism of Inhibition and Allosteric Activation of the Kinase Domain of HER2 Protein</span></div><div class="casAuthors">Aertgeerts, Kathleen; Skene, Robert; Yano, Jason; Sang, Bi-Ching; Zou, Hua; Snell, Gyorgy; Jennings, Andy; Iwamoto, Keiji; Habuka, Noriyuki; Hirokawa, Aki; Ishikawa, Tomoyasu; Tanaka, Toshimasa; Miki, Hiroshi; Ohta, Yoshikazu; Sogabe, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18756-18765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aberrant signaling of ErbB family members human epidermal growth factor 2 (HER2) and epidermal growth factor receptor (EGFR) is implicated in many human cancers, and HER2 expression is predictive of human disease recurrence and prognosis.  Small mol. kinase inhibitors of EGFR and of both HER2 and EGFR have received approval for the treatment of cancer.  We present the first high resoln. crystal structure of the kinase domain of HER2 in complex with a selective inhibitor to understand protein activation, inhibition, and function at the mol. level.  HER2 kinase domain crystallizes as a dimer and suggests evidence for an allosteric mechanism of activation comparable with previously reported activation mechanisms for EGFR and HER4.  A unique Gly-rich region in HER2 following the α-helix C is responsible for increased conformational flexibility within the active site and could explain the low intrinsic catalytic activity previously reported for HER2.  In addn., we solved the crystal structure of the kinase domain of EGFR in complex with a HER2/EGFR dual inhibitor (TAK-285).  Comparison with previously reported inactive and active EGFR kinase domain structures gave insight into the mechanism of HER2 and EGFR inhibition and may help guide the design and development of new cancer drugs with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomtAlJ0FGUX7Vg90H21EOLACvtfcHk0lgaH5wAZFAk0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D&md5=c7897f5363d6452f7b60a8ced0f2bc30</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m110.206193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m110.206193%26sid%3Dliteratum%253Aachs%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DJennings%26aufirst%3DA.%26aulast%3DIwamoto%26aufirst%3DK.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26atitle%3DStructural%2520analysis%2520of%2520the%2520mechanism%2520of%2520inhibition%2520and%2520allosteric%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520HER2%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18756%26epage%3D18765%26doi%3D10.1074%2Fjbc.m110.206193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved protein-ligand docking using GOLD</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein-ligand+docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0lgJ4xf7oJYzLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkamhawy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanath, A. N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedair, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leem, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pae, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, E. J.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5147</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26475520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5147-5154&author=A.+Elkamhawyauthor=A.+K.+Faragauthor=A.+N.+I.+Viswanathauthor=T.+M.+Bedairauthor=D.+G.+Leemauthor=K.-T.+Leeauthor=A.+N.+Paeauthor=E.+J.+Roh&title=Targeting+EGFR%2FHER2+tyrosine+kinases+with+a+new+potent+series+of+6-substituted+4-anilinoquinazoline+hybrids%3A+Design%2C+synthesis%2C+kinase+assay%2C+cell-based+assay%2C+and+molecular+docking&doi=10.1016%2Fj.bmcl.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span></div><div class="casAuthors">Elkamhawy, Ahmed; Farag, Ahmed Karam; Viswanath, Ambily Nath Indu; Bedair, Tarek M.; Leem, Dong Gyu; Lee, Kyung-Tae; Pae, Ae Nim; Roh, Eun Joo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5147-5154</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coexpression of EGFR and HER2 has been found in many tumors such as breast, ovarian, colon and prostate cancers, with poor prognosis of the patients.  Herein, our team has designed and synthesized new eighteen compds. with 6-substituted 4-anilinoquinazoline core to selectively inhibit EGFR/HER2 tyrosine kinases.  Twelve compds. showed nanomolar range of IC50 values on EGFR and/or HER2 kinases.  Accordingly, a detailed structure activity relationship (SAR) was established.  A mol. docking study demonstrated the favorable binding modes of I (X = Me, NHEt) at the ATP active site of both kinases.  A kinase selectivity profile performed for compd. 8d showed great selectivity for EGFR and HER2.  In addn., compd. I (X = Me, NHEt) and II exerted selective promising cytotoxic activity over BT-474 cell line with IC50 values of 2.70, 1.82 and 1.95 μM, resp.  From these results, we report analogs I (X = Me, NHEt) and II as promising candidates for the discovery of well-balanced compds. in terms of the kinase inhibitory potency and antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_Ln0rUhX7bVg90H21EOLACvtfcHk0lgJ4xf7oJYzLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP&md5=0d7c9e4896863934f0190537c10c42ae</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DElkamhawy%26aufirst%3DA.%26aulast%3DFarag%26aufirst%3DA.%2BK.%26aulast%3DViswanath%26aufirst%3DA.%2BN.%2BI.%26aulast%3DBedair%26aufirst%3DT.%2BM.%26aulast%3DLeem%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DK.-T.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRoh%26aufirst%3DE.%2BJ.%26atitle%3DTargeting%2520EGFR%252FHER2%2520tyrosine%2520kinases%2520with%2520a%2520new%2520potent%2520series%2520of%25206-substituted%25204-anilinoquinazoline%2520hybrids%253A%2520Design%252C%2520synthesis%252C%2520kinase%2520assay%252C%2520cell-based%2520assay%252C%2520and%2520molecular%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5147%26epage%3D5154%26doi%3D10.1016%2Fj.bmcl.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0liJvWAVZo8dFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellamy, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, K.</span></span> <span> </span><span class="NLM_article-title">Selective reduction of aromatic nitro compounds with stannous chloride in non acidic and non aqueous medium</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(01)80041-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs0040-4039%2801%2980041-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaL2cXlslSrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1984&pages=839-842&author=F.+D.+Bellamyauthor=K.+Ou&title=Selective+reduction+of+aromatic+nitro+compounds+with+stannous+chloride+in+non+acidic+and+non+aqueous+medium&doi=10.1016%2Fs0040-4039%2801%2980041-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Selective reduction of aromatic nitro compounds with stannous chloride in nonacidic and nonaqueous medium</span></div><div class="casAuthors">Bellamy, F. D.; Ou, K.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">839-42</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Arom. nitro compds. are reduced by SnCl2.2H2O in EtOH or EtOAc or by anhyd. SnCl2 in EtOH and other reducible or acid sensitive groups such as aldehyde, ketone, ester, cyano, halogen and O-benzyl remain unaffected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqxnQ2t7MilbVg90H21EOLACvtfcHk0liJvWAVZo8dFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXlslSrsro%253D&md5=db13aa6ac466d0a9bb0998184183dab2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2801%2980041-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252801%252980041-1%26sid%3Dliteratum%253Aachs%26aulast%3DBellamy%26aufirst%3DF.%2BD.%26aulast%3DOu%26aufirst%3DK.%26atitle%3DSelective%2520reduction%2520of%2520aromatic%2520nitro%2520compounds%2520with%2520stannous%2520chloride%2520in%2520non%2520acidic%2520and%2520non%2520aqueous%2520medium%26jtitle%3DTetrahedron%2520Lett.%26date%3D1984%26volume%3D25%26spage%3D839%26epage%3D842%26doi%3D10.1016%2Fs0040-4039%2801%2980041-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0liJvWAVZo8dFw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, H. K.</span></span> <i>The Zinin Reduction of Nitroarenes</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_year">2011</span>; Vol.  <span class="NLM_volume">20</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2F0471264180.or020.04" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=H.+K.+Porter&title=The+Zinin+Reduction+of+Nitroarenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F0471264180.or020.04&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471264180.or020.04%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DH.%2BK.%26btitle%3DThe%2520Zinin%2520Reduction%2520of%2520Nitroarenes%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2011%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foucourt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubouilh-Benard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chosson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblond, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, T.</span></span> <span> </span><span class="NLM_article-title">Microwave-accelerated Dimroth rearrangement for the synthesis of 4-anilino-6-nitroquinazolines. Application to an efficient synthesis of a microtubule destabilizing agent</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">4495</span>– <span class="NLM_lpage">4502</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2010.04.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.tet.2010.04.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVSksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=4495-4502&author=A.+Foucourtauthor=C.+Dubouilh-Benardauthor=E.+Chossonauthor=C.+Corbi%C3%A8reauthor=C.+Buquetauthor=M.+Iannelliauthor=B.+Leblondauthor=F.+Marsaisauthor=T.+Besson&title=Microwave-accelerated+Dimroth+rearrangement+for+the+synthesis+of+4-anilino-6-nitroquinazolines.+Application+to+an+efficient+synthesis+of+a+microtubule+destabilizing+agent&doi=10.1016%2Fj.tet.2010.04.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-accelerated Dimroth rearrangement for the synthesis of 4-anilino-6-nitroquinazolines. Application to an efficient synthesis of a microtubule destabilizing agent</span></div><div class="casAuthors">Foucourt, Alicia; Dubouilh-Benard, Carole; Chosson, Elizabeth; Corbiere, Cecile; Buquet, Catherine; Iannelli, Mauro; Leblond, Bertrand; Marsais, Francis; Besson, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4495-4502</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Useful and rapid access to 4-anilino-6-nitroquinazolines was investigated on a multi-gram scale via a microwave-accelerated condensation and Dimroth rearrangement of anilines with imines, obtained by the reaction of anthranilonitriles with formamide dimethylacetal.  A novel short and efficient route to Azixa (EPi28495, MPC-6827), I, a microtubule destabilizing agent and apoptosis inducer, was performed successfully and demonstrates that well controlled parameters offer comfortable, safe and environmentally friendly usage of microwave technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBk2zlyCgBXLVg90H21EOLACvtfcHk0lh487pyLPe3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVSksLY%253D&md5=68d8109f726976e656002b8dc3aa0bad</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2010.04.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2010.04.066%26sid%3Dliteratum%253Aachs%26aulast%3DFoucourt%26aufirst%3DA.%26aulast%3DDubouilh-Benard%26aufirst%3DC.%26aulast%3DChosson%26aufirst%3DE.%26aulast%3DCorbi%25C3%25A8re%26aufirst%3DC.%26aulast%3DBuquet%26aufirst%3DC.%26aulast%3DIannelli%26aufirst%3DM.%26aulast%3DLeblond%26aufirst%3DB.%26aulast%3DMarsais%26aufirst%3DF.%26aulast%3DBesson%26aufirst%3DT.%26atitle%3DMicrowave-accelerated%2520Dimroth%2520rearrangement%2520for%2520the%2520synthesis%2520of%25204-anilino-6-nitroquinazolines.%2520Application%2520to%2520an%2520efficient%2520synthesis%2520of%2520a%2520microtubule%2520destabilizing%2520agent%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D4495%26epage%3D4502%26doi%3D10.1016%2Fj.tet.2010.04.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Von
Niementowski, S.</span></span> <span> </span><span class="NLM_article-title">Synthesen von chinazolinverbindungen</span>. <i>J. Prakt. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1894</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1002/prac.18950510150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fprac.18950510150" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1894&pages=564-572&author=S.+Von%0ANiementowski&title=Synthesen+von+chinazolinverbindungen&doi=10.1002%2Fprac.18950510150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fprac.18950510150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprac.18950510150%26sid%3Dliteratum%253Aachs%26aulast%3DVon%2BNiementowski%26aufirst%3DS.%26atitle%3DSynthesen%2520von%2520chinazolinverbindungen%26jtitle%3DJ.%2520Prakt.%2520Chem.%26date%3D1894%26volume%3D51%26spage%3D564%26epage%3D572%26doi%3D10.1002%2Fprac.18950510150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.-T.</span></span> <span> </span><span class="NLM_article-title">Novel EGFR inhibitors prepared by combination of dithiocarbamic acid esters and 4-anilinoquinazolines</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3637</span>– <span class="NLM_lpage">3640</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.04.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2011.04.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21570843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntFKhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=3637-3640&author=R.-D.+Liauthor=X.+Zhangauthor=Q.-Y.+Liauthor=Z.-M.+Geauthor=R.-T.+Li&title=Novel+EGFR+inhibitors+prepared+by+combination+of+dithiocarbamic+acid+esters+and+4-anilinoquinazolines&doi=10.1016%2Fj.bmcl.2011.04.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel EGFR inhibitors prepared by combination of dithiocarbamic acid esters and 4-anilinoquinazolines</span></div><div class="casAuthors">Li, Ri-Dong; Zhang, Xin; Li, Qiao-Yan; Ge, Ze-Mei; Li, Run-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3637-3640</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">On the basis of combination strategy, a novel series of EGFR inhibitors were designed and synthesized by combination of dithiocarbamic acid esters and 4-anilinoquinazolines.  The effect of the synthesized compds. on cell proliferation was evaluated by MTT assay in three human cancer cell lines: MDA-MB-468, SK-BR-3 and HCT-116.  Some compds. were found more potent against all three cell lines and other compds. were found more potent against both MDA-MB-468 and SK-BR-3 than Lapatinib.  SAR studies revealed that the substituents on C6 and C7 positions of quinazoline, the amine component of dithiocarbamate moiety and the linker greatly affected the activity.  This work provides a promising new strategy for the prepn. of potent tyrosine kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo92wKhvesdvrVg90H21EOLACvtfcHk0lh487pyLPe3dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntFKhs7s%253D&md5=e76bffa5c3d8b62e357442dfcf74686c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.04.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.04.096%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DR.-D.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.-Y.%26aulast%3DGe%26aufirst%3DZ.-M.%26aulast%3DLi%26aufirst%3DR.-T.%26atitle%3DNovel%2520EGFR%2520inhibitors%2520prepared%2520by%2520combination%2520of%2520dithiocarbamic%2520acid%2520esters%2520and%25204-anilinoquinazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D3637%26epage%3D3640%26doi%3D10.1016%2Fj.bmcl.2011.04.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forcellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shayhidin, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhéaume, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagüe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquin, J.-F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel quinazoline-4-piperidinesulfamide derivatives as inhibitors of NPP1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.01.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29427906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=130-149&author=E.+Forcelliniauthor=S.+Boutinauthor=C.-A.+Lefebvreauthor=E.+E.+Shayhidinauthor=M.-C.+Boulangerauthor=G.+Rh%C3%A9aumeauthor=X.+Barbeauauthor=P.+Lag%C3%BCeauthor=P.+Mathieuauthor=J.-F.+Paquin&title=Synthesis+and+biological+evaluation+of+novel+quinazoline-4-piperidinesulfamide+derivatives+as+inhibitors+of+NPP1&doi=10.1016%2Fj.ejmech.2018.01.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel quinazoline-4-piperidine sulfamide derivatives as inhibitors of NPP1</span></div><div class="casAuthors">Forcellini, Elsa; Boutin, Sophie; Lefebvre, Carole-Anne; Shayhidin, Elnur Elyar; Boulanger, Marie-Chloe; Rheaume, Gabrielle; Barbeau, Xavier; Lague, Patrick; Mathieu, Patrick; Paquin, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">130-149</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The ecto-nucleotide pyrophosphatase/phosphodiesterase-1 (NPP1) was recently shown to promote mineralization of the aortic valve, hence, its inhibition represents a significant target.  A quinazoline-4-piperidine sulfamide compd. (I) has been described as a specific and non-competitive inhibitor of NPP1.  Herein the synthesis and in vitro inhibition studies of novel quinazoline-4-piperidine sulfamide analogs using I as the lead compd. was reported.  Of the 26 derivs. prepd., four compds. were found to have Ki < 105 nM against human NPP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwRlT7Fi3_H7Vg90H21EOLACvtfcHk0lj9F5oQPthIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D&md5=39d4f90e63b612922cfb5a60b47bd688</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.094%26sid%3Dliteratum%253Aachs%26aulast%3DForcellini%26aufirst%3DE.%26aulast%3DBoutin%26aufirst%3DS.%26aulast%3DLefebvre%26aufirst%3DC.-A.%26aulast%3DShayhidin%26aufirst%3DE.%2BE.%26aulast%3DBoulanger%26aufirst%3DM.-C.%26aulast%3DRh%25C3%25A9aume%26aufirst%3DG.%26aulast%3DBarbeau%26aufirst%3DX.%26aulast%3DLag%25C3%25BCe%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DP.%26aulast%3DPaquin%26aufirst%3DJ.-F.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520quinazoline-4-piperidinesulfamide%2520derivatives%2520as%2520inhibitors%2520of%2520NPP1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D130%26epage%3D149%26doi%3D10.1016%2Fj.ejmech.2018.01.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 7-(4-(3-Ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)- N -hydroxyheptanamide (CUDC-101) as a Potent Multi-Acting HDAC, EGFR, and HER2 Inhibitor for the Treatment of Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2009</span>, <span class="refDoi"> DOI: 10.1021/jm901453q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901453q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslSgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2000-2009&author=X.+Caiauthor=H.-X.+Zhaiauthor=J.+Wangauthor=J.+Forresterauthor=H.+Quauthor=L.+Yinauthor=C.-J.+Laiauthor=R.+Baoauthor=C.+Qian&title=Discovery+of+7-%284-%283-Ethynylphenylamino%29-7-methoxyquinazolin-6-yloxy%29-+N+-hydroxyheptanamide+%28CUDC-101%29+as+a+Potent+Multi-Acting+HDAC%2C+EGFR%2C+and+HER2+Inhibitor+for+the+Treatment+of+Cancer&doi=10.1021%2Fjm901453q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 7-(4-(3-Ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)-N-hydroxyheptanamide (CUDC-101) as a Potent Multi-Acting HDAC, EGFR, and HER2 Inhibitor for the Treatment of Cancer</span></div><div class="casAuthors">Cai, Xiong; Zhai, Hai-Xiao; Wang, Jing; Forrester, Jeffrey; Qu, Hui; Yin, Ling; Lai, Cheng-Jung; Bao, Rudi; Qian, Changgeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2000-2009</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By incorporating histone deacetylase (HDAC) inhibitory functionality into the pharmacophore of the epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor 2 (HER2) inhibitors, we synthesized a novel series of compds. with potent, multiacting HDAC, EGFR, and HER2 inhibition and identified 7-(4-(3-ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)-N-hydroxyheptanamide 8 (CUDC-101) as a drug candidate, which is now in clin. development. 8 displays potent in vitro inhibitory activity against HDAC, EGFR, and HER2 with an IC50 of 4.4, 2.4, and 15.7 nM, resp.  In most tumor cell lines tested, 8 exhibits efficient antiproliferative activity with greater potency than vorinostat (SAHA), erlotinib, lapatinib, and combinations of vorinostat/erlotinib and vorinostat/lapatinib.  In vivo, 8 promotes tumor regression or inhibition in various cancer xenograft models including nonsmall cell lung cancer (NSCLC), liver, breast, head and neck, colon, and pancreatic cancers.  These results suggest that a single compd. that simultaneously inhibits HDAC, EGFR, and HER2 may offer greater therapeutic benefits in cancer over single-acting agents through the interference with multiple pathways and potential synergy among HDAC and EGFR/HER2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodBQl1iMXW3rVg90H21EOLACvtfcHk0lj9F5oQPthIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslSgsb0%253D&md5=d4dc339f2fd750d52e511abf3dbf18bd</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm901453q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901453q%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DH.-X.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DForrester%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DH.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.-J.%26aulast%3DBao%26aufirst%3DR.%26aulast%3DQian%26aufirst%3DC.%26atitle%3DDiscovery%2520of%25207-%25284-%25283-Ethynylphenylamino%2529-7-methoxyquinazolin-6-yloxy%2529-%2520N%2520-hydroxyheptanamide%2520%2528CUDC-101%2529%2520as%2520a%2520Potent%2520Multi-Acting%2520HDAC%252C%2520EGFR%252C%2520and%2520HER2%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2000%26epage%3D2009%26doi%3D10.1021%2Fjm901453q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Structure-Activity Relationships of A Novel Class of Dithiocarbamic Acid Esters as Anticancer Agent</span>. <i>Arch. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">320</span>, <span class="refDoi"> DOI: 10.1002/ardp.201000259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fardp.201000259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21509804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2011&pages=320&author=X.+Houauthor=Z.+Geauthor=T.+Wangauthor=W.+Guoauthor=J.+Wuauthor=J.+Cuiauthor=C.+Laiauthor=R.+Li&title=Synthesis+and+Structure-Activity+Relationships+of+A+Novel+Class+of+Dithiocarbamic+Acid+Esters+as+Anticancer+Agent&doi=10.1002%2Fardp.201000259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationships of A Novel Class of Dithiocarbamic Acid Esters as Anticancer Agent</span></div><div class="casAuthors">Hou, Xueling; Ge, Zemei; Wang, Tingmin; Guo, Wei; Wu, Jun; Cui, Jingrong; Lai, Chingsan; Li, Runtao</div><div class="citationInfo"><span class="NLM_cas:title">Archiv der Pharmazie (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">320-332</span>CODEN:
                <span class="NLM_cas:coden">ARPMAS</span>;
        ISSN:<span class="NLM_cas:issn">0365-6233</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Based on a novel lead compd. 4-methylpiperazine-1-carbodithioic acid 3-cyano-3,3-diphenylpropyl ester [I (X = NMe)(II)], the systematic structural modification was carried out.  All the synthesized compds. were evaluated for their in-vitro anticancer activities on four to six different cell lines at three different concns.  Most of the tested compds. could selectively inhibit the growth of HL-60 and Bel-7402 cell lines at a medium concn.  Four compds. {III [R = H (IV), Me (V)], I [X = NH (VI), SO2 (VII)]} were selected for the IC50 test, and the results revealed that three compds. (V, VI and VII) showed almost the same or a slightly weaker activity than compd. against HL-60, and three compds. (IV, V and VI) showed > 2-fold higher potency than compd. II against Bel-7402.  The in-vivo efficacy of VI·HCl was evaluated with transplanted hepatocyte carcinoma 22 as an in-vivo test model.  It was found that VI·HCl could inhibit significantly the growth of tumor, and that this effect was dose-dependent.  Meanwhile, the compd. VI·HCl showed low toxicity compared with compd. II·HCl as evidenced by the little body-wt. loss.  These results confirmed that compd. VI·HCl is more potent than the lead compd. II·HCl.  Preliminary structure-activity relationships indicated that: (a) Both nitrile group and the cyclic amine contg. at least two nitrogens were indispensable moieties to keep the activity; (b) substitution of the piperazine ring is unfavorable for the improvement of activity; (c) the suitable linker joining the piperazinyl dithiocarboxyl and diphenylacetonitril group should be ethylene; (d) a non-coplanar arrangement of the two benzene rings appears to be essential for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOKLHG3n1QpLVg90H21EOLACvtfcHk0lj9F5oQPthIoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFGhsL4%253D&md5=a3547077b21c44e43d9741bb0259a353</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fardp.201000259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.201000259%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DSynthesis%2520and%2520Structure-Activity%2520Relationships%2520of%2520A%2520Novel%2520Class%2520of%2520Dithiocarbamic%2520Acid%2520Esters%2520as%2520Anticancer%2520Agent%26jtitle%3DArch.%2520Pharm.%26date%3D2011%26volume%3D344%26spage%3D320%26doi%3D10.1002%2Fardp.201000259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, R. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou-Seri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldehna, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, N. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgazwi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghabbour, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaweish, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou El Ella, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel series of 6-(2-substitutedacetamido)-4-anilinoquinazolines as EGFR-ERK signal transduction inhibitors in MCF-7 breast cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">782</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30047410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1aksb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=782-796&author=R.+S.+M.+Ismailauthor=S.+M.+Abou-Seriauthor=W.+M.+Eldehnaauthor=N.+S.+M.+Ismailauthor=S.+M.+Elgazwiauthor=H.+A.+Ghabbourauthor=M.+S.+Ahmedauthor=F.+T.+Halaweishauthor=D.+A.+Abou+El+Ella&title=Novel+series+of+6-%282-substitutedacetamido%29-4-anilinoquinazolines+as+EGFR-ERK+signal+transduction+inhibitors+in+MCF-7+breast+cancer+cells&doi=10.1016%2Fj.ejmech.2018.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Novel series of 6-(2-substituted acetamido)-4-anilinoquinazolines as EGFR-ERK signal transduction inhibitors in MCF-7 breast cancer cells</span></div><div class="casAuthors">Ismail, Rania S. M.; Abou-Seri, Sahar M.; Eldehna, Wagdy M.; Ismail, Nasser S. M.; Elgazwi, Sara M.; Ghabbour, Hazem A.; Ahmed, Mahmoud Salama; Halaweish, Fathi T.; Abou El Ella, Dalal A.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">782-796</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The design and synthesis of two series of 6-(2-substituted acetamido)-4-anilinoquinazolines I [R = 3-Cl, 3-CF3, 3-Br, etc.; R1 = pyrrolidino, piperidino, morpholino, 1-methylpiperazin-4-yl] and II [R2 = pyrrolidino, morpholino; R3 = H, F] as EGFR inhibitors was reported.  All the newly synthesized quinazoline derivs. I and II were in-vitro evaluated for their anti-proliferative activity towards MCF-7 (Breast Cancer) and HepG2 (Hepatocellular carcinoma) cell lines.  In particular, compd. I [R = 3-Br, R1 = piperidino] showed significant inhibitory activity against MCF-7 and HepG2 cell lines (IC50 = 3 and 16μM, resp.), compared to that of Erlotinib (IC50 = 20 and 25μM, resp.).  Western blotting of compd. I [R = 3-Br, R1 = piperidino] at MCF-7 cell line revealed the dual inhibitory activity of compd. I [R = 3-Br, R1 = piperidino] towards diminishing the phosphorylated levels for EGFR and ERK.  Also, ELISA assay confirmed the anti-EGFR activity of compd. I [R = 3-Br, R1 = piperidino] (IC50 = 0.037 μM).  Finally, a mol. docking study showed the potential binding mode of compd. I [R = 3-Br, R1 = piperidino] within the ATP catalytic binding site of EGFR, exhibiting similar binding mode to EGFR inhibitor Erlotinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoehX6XD-UKErVg90H21EOLACvtfcHk0lgojORY56BCmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1aksb7M&md5=db4ed7de18503f4d793983db26a3b9d2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DR.%2BS.%2BM.%26aulast%3DAbou-Seri%26aufirst%3DS.%2BM.%26aulast%3DEldehna%26aufirst%3DW.%2BM.%26aulast%3DIsmail%26aufirst%3DN.%2BS.%2BM.%26aulast%3DElgazwi%26aufirst%3DS.%2BM.%26aulast%3DGhabbour%26aufirst%3DH.%2BA.%26aulast%3DAhmed%26aufirst%3DM.%2BS.%26aulast%3DHalaweish%26aufirst%3DF.%2BT.%26aulast%3DAbou%2BEl%2BElla%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520series%2520of%25206-%25282-substitutedacetamido%2529-4-anilinoquinazolines%2520as%2520EGFR-ERK%2520signal%2520transduction%2520inhibitors%2520in%2520MCF-7%2520breast%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D782%26epage%3D796%26doi%3D10.1016%2Fj.ejmech.2018.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noren, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquale, E. B.</span></span> <span> </span><span class="NLM_article-title">The EphB4 receptor suppresses breast cancer cell tumorigenicity through an Abl-Crk pathway</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1038/ncb1438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fncb1438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16862147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFWqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=815-825&author=N.+K.+Norenauthor=G.+Foosauthor=C.+A.+Hauserauthor=E.+B.+Pasquale&title=The+EphB4+receptor+suppresses+breast+cancer+cell+tumorigenicity+through+an+Abl-Crk+pathway&doi=10.1038%2Fncb1438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The EphB4 receptor suppresses breast cancer cell tumorigenicity through an Abl-Crk pathway</span></div><div class="casAuthors">Noren, Nicole K.; Foos, Gabriele; Hauser, Craig A.; Pasquale, Elena B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">815-825</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent evidence supports a role for EphB receptor tyrosine kinases as tumor suppressors in colorectal and prostate cancer.  However, it is unclear how these receptors inhibit cancer cell tumorigenicity - an activity that is highly unusual for a family of receptor tyrosine kinases.  Here, we report that the EphB4 receptor can behave as a tumor suppressor in a mouse xenograft model of breast cancer when stimulated by its ligand, ephrin-B2.  In breast cancer cells, EphB4 activates an antioncogenic pathway involving Abl family tyrosine kinases and the Crk adaptor protein.  This Abl-Crk pathway inhibits breast cancer cell viability and proliferation in addn. to motility and invasion, and also downregulates the pro-invasive matrix metalloprotease, MMP-2.  Consistent with these effects, EphB4 and the Abl-Crk pathway are constitutively active in non-transformed mammary epithelial cells.  These findings identify a novel Eph receptor signaling pathway with tumor-suppressor activity and predict that therapeutic intervention to activate EphB4 signaling will inhibit tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ4SmHZkAiX7Vg90H21EOLACvtfcHk0lgojORY56BCmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFWqtbY%253D&md5=d628bfa7d0ec8ac407eff0b6ab024978</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fncb1438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1438%26sid%3Dliteratum%253Aachs%26aulast%3DNoren%26aufirst%3DN.%2BK.%26aulast%3DFoos%26aufirst%3DG.%26aulast%3DHauser%26aufirst%3DC.%2BA.%26aulast%3DPasquale%26aufirst%3DE.%2BB.%26atitle%3DThe%2520EphB4%2520receptor%2520suppresses%2520breast%2520cancer%2520cell%2520tumorigenicity%2520through%2520an%2520Abl-Crk%2520pathway%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2006%26volume%3D8%26spage%3D815%26epage%3D825%26doi%3D10.1038%2Fncb1438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Henn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patsialou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condeelis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleske, A. J.</span></span> <span> </span><span class="NLM_article-title">Arg/Abl2 promotes invasion and attenuates proliferation of breast cancer in vivo</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2622</span>– <span class="NLM_lpage">2630</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fonc.2012.284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22777352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=2622-2630&author=H.+Gil-Hennauthor=A.+Patsialouauthor=Y.+Wangauthor=M.+S.+Warrenauthor=J.+S.+Condeelisauthor=A.+J.+Koleske&title=Arg%2FAbl2+promotes+invasion+and+attenuates+proliferation+of+breast+cancer+in+vivo&doi=10.1038%2Fonc.2012.284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Arg/Abl2 promotes invasion and attenuates proliferation of breast cancer in vivo</span></div><div class="casAuthors">Gil-Henn, H.; Patsialou, A.; Wang, Y.; Warren, M. S.; Condeelis, J. S.; Koleske, A. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2622-2630</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor progression is a complex, multistep process involving accumulation of genetic aberrations and alterations in gene expression patterns leading to uncontrolled cell division, invasion into surrounding tissue and finally dissemination and metastasis.  We have previously shown that the Arg/Abl2 non-receptor tyrosine kinase acts downstream of the EGF receptor and Src tyrosine kinases to promote invadopodium function in breast cancer cells, thereby promoting their invasiveness.  However, whether and how Arg contributes to tumor development and dissemination in vivo has never been investigated.  Using a mouse xenograft model, we show that knocking down Arg in breast cancer cells leads to increased tumor cell proliferation and significantly enlarged tumor size.  Despite having larger tumors, the Arg-knockdown (Arg KD) tumor-bearing mice exhibit significant redns. in tumor cell invasion, intravasation into blood vessels and spontaneous metastasis to lungs.  Interestingly, we found that proliferation-assocd. genes in the Ras-MAPK (mitogen-activated protein kinase) pathway are upregulated in Arg KD breast cancer cells, as is Ras-MAPK signaling, while invasion-assocd. genes are significantly downregulated.  These data suggest that Arg promotes tumor cell invasion and dissemination, while simultaneously inhibiting tumor growth.  We propose that Arg acts as a switch in metastatic cancer cells that governs the decision to grow or go' (divide or invade).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroI_rJvnATw7Vg90H21EOLACvtfcHk0lgojORY56BCmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGmsLc%253D&md5=e2303167a3df6b3d2ffbbb7a1d1213cc</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.284%26sid%3Dliteratum%253Aachs%26aulast%3DGil-Henn%26aufirst%3DH.%26aulast%3DPatsialou%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DCondeelis%26aufirst%3DJ.%2BS.%26aulast%3DKoleske%26aufirst%3DA.%2BJ.%26atitle%3DArg%252FAbl2%2520promotes%2520invasion%2520and%2520attenuates%2520proliferation%2520of%2520breast%2520cancer%2520in%2520vivo%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D2622%26epage%3D2630%26doi%3D10.1038%2Fonc.2012.284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, L.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Combating trastuzumab resistance by targeting SRC, a common node downstream of multiple resistance pathways</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1038/nm.2309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fnm.2309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21399647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVGntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=461-469&author=S.+Zhangauthor=W.-C.+Huangauthor=P.+Liauthor=H.+Guoauthor=S.-B.+Pohauthor=S.+W.+Bradyauthor=Y.+Xiongauthor=L.-M.+Tsengauthor=S.-H.+Liauthor=Z.+Dingauthor=A.+A.+Sahinauthor=F.+J.+Estevaauthor=G.+N.+Hortobagyiauthor=D.+Yu&title=Combating+trastuzumab+resistance+by+targeting+SRC%2C+a+common+node+downstream+of+multiple+resistance+pathways&doi=10.1038%2Fnm.2309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Combating trastuzumab resistance by targeting SRC, a common node downstream of multiple resistance pathways</span></div><div class="casAuthors">Zhang, Siyuan; Huang, Wen-Chien; Li, Ping; Guo, Hua; Poh, Say-Bee; Brady, Samuel W.; Xiong, Yan; Tseng, Ling-Ming; Li, Shau-Hsuan; Ding, Zhaoxi; Sahin, Aysegul A.; Esteva, Francisco J.; Hortobagyi, Gabriel N.; Yu, Dihua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">461-469</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Trastuzumab is a successful rationally designed ERBB2-targeted therapy.  However, about half of individuals with ERBB2-overexpressing breast cancer do not respond to trastuzumab-based therapies, owing to various resistance mechanisms.  Clin. applicable regimens for overcoming trastuzumab resistance of different mechanisms are not yet available.  We show that the nonreceptor tyrosine kinase c-SRC (SRC) is a key modulator of trastuzumab response and a common node downstream of multiple trastuzumab resistance pathways.  We find that SRC is activated in both acquired and de novo trastuzumab-resistant cells and uncover a novel mechanism of SRC regulation involving dephosphorylation by PTEN.  Increased SRC activation conferred considerable trastuzumab resistance in breast cancer cells and correlated with trastuzumab resistance in patients.  Targeting SRC in combination with trastuzumab sensitized multiple lines of trastuzumab-resistant cells to trastuzumab and eliminated trastuzumab-resistant tumors in vivo, suggesting the potential clin. application of this strategy to overcome trastuzumab resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAglfHbddvf7Vg90H21EOLACvtfcHk0li0QvI_LIC5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVGntb8%253D&md5=e316dee0106724d2c593358dae9c7801</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnm.2309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2309%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.-C.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DPoh%26aufirst%3DS.-B.%26aulast%3DBrady%26aufirst%3DS.%2BW.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DTseng%26aufirst%3DL.-M.%26aulast%3DLi%26aufirst%3DS.-H.%26aulast%3DDing%26aufirst%3DZ.%26aulast%3DSahin%26aufirst%3DA.%2BA.%26aulast%3DEsteva%26aufirst%3DF.%2BJ.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DCombating%2520trastuzumab%2520resistance%2520by%2520targeting%2520SRC%252C%2520a%2520common%2520node%2520downstream%2520of%2520multiple%2520resistance%2520pathways%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D461%26epage%3D469%26doi%3D10.1038%2Fnm.2309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span> <span> </span><span class="NLM_article-title">Targeting FAK in human cancer: from finding to first clinical trials</span>. <i>Front. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.2741/4236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2741%2F4236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24389213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=687-706&author=V.+Golubovskaya&title=Targeting+FAK+in+human+cancer%3A+from+finding+to+first+clinical+trials&doi=10.2741%2F4236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FAK in human cancer: from finding to first clinical trials</span></div><div class="casAuthors">Golubovskaya, Vita M.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-706, 20</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  It is twenty years since Focal Adhesion Kinase (FAK) was found to be overexpressed in many types of human cancer.  FAK plays an important role in adhesion, spreading, motility, invasion, metastasis, survival, angiogenesis, and recently has been found to play an important role as well in epithelial to mesenchymal transition (EMT), cancer stem cells and tumor microenvironment.  FAK has kinase-dependent and kinase independent scaffolding, cytoplasmic and nuclear functions.  Several years ago FAK was proposed as a potential therapeutic target; the first clin. trials were just reported, and they supported further studies of FAK as a promising therapeutic target.  This review discusses the main functions of FAK in cancer, and specifically focuses on recent novel findings on the role of FAK in cancer stem cells, microenvironment, epithelial-to-mesenchymal transition, invasion, metastasis, and also highlight new approaches of targeting FAK and critically discuss challenges that lie ahead for its targeted therapeutics.  The review provides a summary of translational approaches of FAK-targeted and combination therapies and outline perspectives and future directions of FAK research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EGzIEc6UMbVg90H21EOLACvtfcHk0li0QvI_LIC5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI&md5=977f2530b2d4ed073e84ec64e498c3a3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.2741%2F4236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4236%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26atitle%3DTargeting%2520FAK%2520in%2520human%2520cancer%253A%2520from%2520finding%2520to%2520first%2520clinical%2520trials%26jtitle%3DFront.%2520Biosci.%26date%3D2014%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.2741%2F4236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span> <span> </span><span class="NLM_article-title">Structure-based modification of carbonyl-diphenylpyrimidines (Car-DPPYs) as a novel focal adhesion kinase (FAK) inhibitor against various stubborn cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2019.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30978560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSgtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2019&pages=154-162&author=L.+Wangauthor=M.+Aiauthor=J.+Yuauthor=L.+Jinauthor=C.+Wangauthor=Z.+Liuauthor=X.+Shuauthor=Z.+Tangauthor=K.+Liuauthor=H.+Luoauthor=W.+Guanauthor=X.+Sunauthor=X.+Ma&title=Structure-based+modification+of+carbonyl-diphenylpyrimidines+%28Car-DPPYs%29+as+a+novel+focal+adhesion+kinase+%28FAK%29+inhibitor+against+various+stubborn+cancer+cells&doi=10.1016%2Fj.ejmech.2019.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based modification of carbonyl-diphenylpyrimidines (Car-DPPYs) as a novel focal adhesion kinase (FAK) inhibitor against various stubborn cancer cells</span></div><div class="casAuthors">Wang, Luhong; Ai, Min; Yu, Jiawen; Jin, Lingling; Wang, Changyuan; Liu, Zhihao; Shu, Xiaohong; Tang, Zeyao; Liu, Kexin; Luo, Hui; Guan, Wenshun; Sun, Xiuli; Ma, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">154-162</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A family of carbonyl-substituted diphenylpyrimidine derivs. (Car-DPPYs) with strong activity against focal adhesion kinase (FAK), were described in this manuscript.  Among them, compds. 7a(I) (IC50 = 5.17 nM) and 7f(II) (IC50 = 2.58 nM) displayed equal anti-FAK enzymic activity to the lead compd. TAE226 (6.79 nM).  In particular, I also exhibited strong antiproliferative activity against several stubborn cancer cells, including AsPC-1 cells (IC50 = 0.105 μM), BxPC-3 cells (IC50 = 0.090 μM), and MCF-7/ADR cells (IC50 = 0.59 μM).  Addnl., I also showed great antitumor efficacy in vivo via aAsPC-1 cancer Xenograft mouse model.  The preliminary mechanism study by Western blot anal. revealed that I repressed FAK phosphorylation in AsPC cancer cells.  Taken together, the results indicate that I may serve as a promising preclin. candidate for treatment of stubborn cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jTLS3BSpz7Vg90H21EOLACvtfcHk0li0QvI_LIC5TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSgtbg%253D&md5=1e37d0d16bf3d6c86238ff2bcde12259</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAi%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DShu%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DX.%26atitle%3DStructure-based%2520modification%2520of%2520carbonyl-diphenylpyrimidines%2520%2528Car-DPPYs%2529%2520as%2520a%2520novel%2520focal%2520adhesion%2520kinase%2520%2528FAK%2529%2520inhibitor%2520against%2520various%2520stubborn%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D172%26spage%3D154%26epage%3D162%26doi%3D10.1016%2Fj.ejmech.2019.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, K. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor growth and angiogenesis of tamoxifen-resistant breast cancer cells by ruxolitinib, a selective JAK2 inhibitor</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3981</span>– <span class="NLM_lpage">3989</span>, <span class="refDoi"> DOI: 10.3892/ol.2019.10059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3892%2Fol.2019.10059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30930994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlShs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=3981-3989&author=J.+W.+Kimauthor=J.+Gautamauthor=J.+E.+Kimauthor=J.+A.+Kimauthor=K.+W.+Kang&title=Inhibition+of+tumor+growth+and+angiogenesis+of+tamoxifen-resistant+breast+cancer+cells+by+ruxolitinib%2C+a+selective+JAK2+inhibitor&doi=10.3892%2Fol.2019.10059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of tumor growth and angiogenesis of tamoxifen-resistant breast cancer cells by ruxolitinib, a selective JAK2 inhibitor</span></div><div class="casAuthors">Kim, Ji Won; Gautam, Jaya; Kim, Ji Eun; Kim, Jung-Ae; Kang, Keon Wook</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">3981-3989</span>CODEN:
                <span class="NLM_cas:coden">OLNEB5</span>;
        ISSN:<span class="NLM_cas:issn">1792-1082</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Tamoxifen (TAM) is the most widely used treatment for estrogen receptor-pos. breast cancer patients.  Unfortunately, the majority of these patients exhibit TAM resistance following treatment.  We previously reported that proliferation and migration were greater in TAM-resistant MCF-7 (TAMR-MCF-7) cells than in parental MCF-7 cells.  Janus kinases (JAKs) are cytosolic tyrosine kinases that transduce signals from plasma membrane cytokines and growth factor receptors.  JAK2 selectively phosphorylates signal transducer and activator of transcription (STAT)-3, and the JAK2-STAT3 signaling pathway is known as a crucial signaling pathway for the regulation of cancer progression and metastasis.  In the present study, basal phosphorylation of STAT3 was revealed to be greater in TAMR-MCF-7 cells than in control MCF-7 cells.  Ruxolitinib, a potent JAK2 inhibitor, was demonstrated to attenuate STAT3 phosphorylation and the proliferation of TAMR-MCF-7 cells.  Ruxolitinib also suppressed the enhanced cell migration of TAMR-MCF-7 cells through the inhibition of epithelial mesenchymal transition.  Vascular endothelial growth factor (VEGF), a representative target gene of the JAK2-STAT3 pathway, functions as a key regulator of invasion and angiogenesis.  Ruxolitinib significantly inhibited VEGF mRNA expression and transcriptional activity.  The present study also performed a chick embryo chorioallantoic membrane assay to assess tumor growth and angiogenesis in TAMR-MCF-7 cells.  Ruxolitinib reduced tumor wt. and the no. of blood vessels produced by TAMR-MCF-7 cells in a concn.-dependent manner.  These results indicated that JAK2 could be a new therapeutic target for TAM-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6j7dPYlq207Vg90H21EOLACvtfcHk0li0Jgt_XZoZ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlShs77K&md5=88bb39074c18bd6a995d10867cad3801</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.3892%2Fol.2019.10059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2019.10059%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DGautam%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DKim%26aufirst%3DJ.%2BA.%26aulast%3DKang%26aufirst%3DK.%2BW.%26atitle%3DInhibition%2520of%2520tumor%2520growth%2520and%2520angiogenesis%2520of%2520tamoxifen-resistant%2520breast%2520cancer%2520cells%2520by%2520ruxolitinib%252C%2520a%2520selective%2520JAK2%2520inhibitor%26jtitle%3DOncol.%2520Lett.%26date%3D2019%26volume%3D17%26spage%3D3981%26epage%3D3989%26doi%3D10.3892%2Fol.2019.10059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekaran, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, G. C.</span></span> <span> </span><span class="NLM_article-title">Receptor tyrosine kinases (RTKs) in breast cancer: signaling, therapeutic implications and challenges</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0797-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fs12943-018-0797-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29455658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=34-51&author=R.+Buttiauthor=S.+Dasauthor=V.+P.+Gunasekaranauthor=A.+S.+Yadavauthor=D.+Kumarauthor=G.+C.+Kundu&title=Receptor+tyrosine+kinases+%28RTKs%29+in+breast+cancer%3A+signaling%2C+therapeutic+implications+and+challenges&doi=10.1186%2Fs12943-018-0797-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinases (RTKs) in breast cancer: signaling, therapeutic implications and challenges</span></div><div class="casAuthors">Butti, Ramesh; Das, Sumit; Gunasekaran, Vinoth Prasanna; Yadav, Amit Singh; Kumar, Dhiraj; Kundu, Gopal C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/18</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Breast cancer is a multifactorial disease and driven by aberrant regulation of cell signaling pathways due to the acquisition of genetic and epigenetic changes.  An array of growth factors and their receptors is involved in cancer development and metastasis.  Receptor Tyrosine Kinases (RTKs) constitute a class of receptors that play important role in cancer progression.  RTKs are cell surface receptors with specialized structural and biol. features which respond to environmental cues by initiating appropriate signaling cascades in tumor cells.  RTKs are known to regulate various downstream signaling pathways such as MAPK, PI3K/Akt and JAK/STAT.  These pathways have a pivotal role in the regulation of cancer stemness, angiogenesis and metastasis.  These pathways are also imperative for a reciprocal interaction of tumor and stromal cells.  Multi-faceted role of RTKs renders them amenable to therapy in breast cancer.  However, structural mutations, gene amplification and alternate pathway activation pose challenges to anti-RTK therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGyFoVNhOMwLVg90H21EOLACvtfcHk0li0Jgt_XZoZ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7jK&md5=2bfb39c593fb1df3337082e8d93a7005</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0797-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0797-x%26sid%3Dliteratum%253Aachs%26aulast%3DButti%26aufirst%3DR.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DGunasekaran%26aufirst%3DV.%2BP.%26aulast%3DYadav%26aufirst%3DA.%2BS.%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DKundu%26aufirst%3DG.%2BC.%26atitle%3DReceptor%2520tyrosine%2520kinases%2520%2528RTKs%2529%2520in%2520breast%2520cancer%253A%2520signaling%252C%2520therapeutic%2520implications%2520and%2520challenges%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D34%26epage%3D51%26doi%3D10.1186%2Fs12943-018-0797-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjöblom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontén, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heldin, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jirström, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Östman, A.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of stromal platelet-derived growth factor beta-receptor expression in human breast cancer</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2009.081030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2353%2Fajpath.2009.081030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19498003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1MvksVekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2009&pages=334-341&author=J.+Paulssonauthor=T.+Sj%C3%B6blomauthor=P.+Mickeauthor=F.+Pont%C3%A9nauthor=G.+Landbergauthor=C.-H.+Heldinauthor=J.+Berghauthor=D.+J.+Brennanauthor=K.+Jirstr%C3%B6mauthor=A.+%C3%96stman&title=Prognostic+significance+of+stromal+platelet-derived+growth+factor+beta-receptor+expression+in+human+breast+cancer&doi=10.2353%2Fajpath.2009.081030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of stromal platelet-derived growth factor beta-receptor expression in human breast cancer</span></div><div class="casAuthors">Paulsson Janna; Sjoblom Tobias; Micke Patrick; Ponten Fredrik; Landberg Goran; Heldin Carl-Henrik; Bergh Jonas; Brennan Donal J; Jirstrom Karin; Ostman Arne</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">334-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This study systematically analyzes platelet-derived growth factor (PDGF) receptor expression in six types of common tumors as well as examines associations between PDGF beta-receptor status and clinicopathological characteristics in breast cancer.  PDGF receptor expression was determined by immunohistochemistry on tumor tissue microarrays.  Breast tumor data were combined with prognostic factors and related to outcome endpoints.  PDGF alpha- and beta-receptors were independently expressed, at variable frequencies, in the tumor stroma of all tested tumor types.  There was a significant association between PDGF beta-receptor expression on fibroblasts and perivascular cells in individual colon and prostate tumors.  In breast cancer, high stromal PDGF beta-receptor expression was significantly associated with high histopathological grade, estrogen receptor negativity, and high HER2 expression.  High stromal PDGF beta-receptor expression was correlated with significantly shorter recurrence-free and breast cancer-specific survival.  The prognostic significance of stromal PDGF beta-receptor expression was particularly prominent in tumors from premenopausal women.  Stromal PDGF alpha- and beta-receptor expression is a common, but variable and independent, property of solid tumors.  In breast cancer, stromal PDGF beta-receptor expression significantly correlates with less favorable clinicopathological parameters and shorter survival.  These findings highlight the prognostic significance of stromal markers and should be considered in ongoing clinical development of PDGF receptor inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRH5GHwn_GPqRwLZxei1HN8fW6udTcc2eYd2DJXcL8n67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvksVekuw%253D%253D&md5=13b0ee365c15209a154b4d11d54ffe76</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2009.081030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2009.081030%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsson%26aufirst%3DJ.%26aulast%3DSj%25C3%25B6blom%26aufirst%3DT.%26aulast%3DMicke%26aufirst%3DP.%26aulast%3DPont%25C3%25A9n%26aufirst%3DF.%26aulast%3DLandberg%26aufirst%3DG.%26aulast%3DHeldin%26aufirst%3DC.-H.%26aulast%3DBergh%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DD.%2BJ.%26aulast%3DJirstr%25C3%25B6m%26aufirst%3DK.%26aulast%3D%25C3%2596stman%26aufirst%3DA.%26atitle%3DPrognostic%2520significance%2520of%2520stromal%2520platelet-derived%2520growth%2520factor%2520beta-receptor%2520expression%2520in%2520human%2520breast%2520cancer%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2009%26volume%3D175%26spage%3D334%26epage%3D341%26doi%3D10.2353%2Fajpath.2009.081030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carvalho, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanezi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, F.</span></span> <span> </span><span class="NLM_article-title">Overexpression of platelet-derived growth factor receptor alpha in breast cancer is associated with tumour progression</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R788</span>– <span class="NLM_lpage">R795</span>, <span class="refDoi"> DOI: 10.1186/bcr1304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr1304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16168125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=R788-R795&author=I.+Carvalhoauthor=F.+Milaneziauthor=A.+Martinsauthor=R.+M.+Reisauthor=F.+Schmitt&title=Overexpression+of+platelet-derived+growth+factor+receptor+alpha+in+breast+cancer+is+associated+with+tumour+progression&doi=10.1186%2Fbcr1304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of platelet-derived growth factor receptor α in breast cancer is associated with tumour progression</span></div><div class="casAuthors">Carvalho, Ines; Milanezi, Fernanda; Martins, Albino; Reis, Rui M.; Schmitt, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">R788-R795</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Introduction: Receptor tyrosine kinases have been extensively studied owing to their frequently abnormal activation in the development and progression of human cancers.  Platelet-derived growth factor receptors (PDGFRs) are receptors with intrinsic tyrosine kinase activity that regulate several functions in normal cells and are widely expressed in a variety of malignancies.  After the demonstration that gastrointestinal stromal tumors without c-Kit mutations harbor PDGFR-α-activating mutations and that PDGFR-α is also a therapeutic target for imatinib mesylate, the interest for this receptor has increased considerably.  Because breast cancer is one of the most frequent neoplasias in women worldwide, and only one study has reported PDGFR-α expression in breast carcinomas, the aim of this work was to investigate the potential significance of PDGFR-α expression in invasive mammary carcinomas.  Methods: We used immunohistochem. to detect PDGFR-α overexpression on a series of 181 formalin-fixed paraffin-embedded invasive ductal breast carcinomas and in two breast cancer cell lines: MCF-7 and HS578T.  We assocd. its expression with known prognostic factors and we also performed polymerase chain reaction-single-stranded conformational polymorphism and direct sequencing to screen for PDGFR-α mutations.  Results: PDGFR-α expression was obsd. in 39.2% of the breast carcinomas and showed an assocn. with lymph node metastasis (P = 0.0079), HER-2 expression (P = 0.0265) and Bcl2 expression (P = 0.0121).  A correlation was also found with the expression of platelet-derived growth factor A (PDGF-A; P = 0.0194).  The two cell lines tested did not express PDGFR-α.  Screening for mutations revealed alterations in the PDGFR-α gene at the following locations: 2500A→G, 2529T→A and 2472C→T in exon 18 and 1701G→A in exon 12.  We also found an intronic insertion IVS17-50insA at exon 18 in all sequenced cases.  None of these genetic alterations was correlated with PDGFR-α expression.  The cell lines did not reveal any alterations in the PDGFR-α gene sequence.  Conclusion: PDGFR-α is expressed in invasive breast carcinomas and is assocd. with biol. aggressiveness.  The genetic alterations described were not correlated with protein expression, but other mechanisms such as gene amplification or constitutive activation of a signaling pathway inducing this receptor could still sustain PDGFR-α as a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0dHnzJg0e7Vg90H21EOLACvtfcHk0li0Jgt_XZoZ8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOgtbs%253D&md5=1dab08c050341c1b246a9eda112560d7</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1186%2Fbcr1304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr1304%26sid%3Dliteratum%253Aachs%26aulast%3DCarvalho%26aufirst%3DI.%26aulast%3DMilanezi%26aufirst%3DF.%26aulast%3DMartins%26aufirst%3DA.%26aulast%3DReis%26aufirst%3DR.%2BM.%26aulast%3DSchmitt%26aufirst%3DF.%26atitle%3DOverexpression%2520of%2520platelet-derived%2520growth%2520factor%2520receptor%2520alpha%2520in%2520breast%2520cancer%2520is%2520associated%2520with%2520tumour%2520progression%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2005%26volume%3D7%26spage%3DR788%26epage%3DR795%26doi%3D10.1186%2Fbcr1304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klassen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, B.</span></span> <span> </span><span class="NLM_article-title">Localization of platelet-derived growth factor beta receptor expression in the periepithelial stroma of human breast carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">773</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=9816230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK28Xis1Wlurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1996&pages=773&author=B.+Bhardwajauthor=J.+Klassenauthor=N.+Cossetteauthor=E.+Sternsauthor=A.+Tuckauthor=R.+Deeleyauthor=S.+Senguptaauthor=B.+Elliott&title=Localization+of+platelet-derived+growth+factor+beta+receptor+expression+in+the+periepithelial+stroma+of+human+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Localization of platelet-derived growth factor β receptor expression in the periepithelial stroma of human breast carcinoma</span></div><div class="casAuthors">Bhardwaj, Bhavna; Klassen, Jennifer; Cossette, Nadine; Sterns, Ernest; Tuck, Alan; Deeley, Roger; Sengupta, Sandip; Elliott, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-82</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Platelet-derived growth factor (PDGF) BB is secreted by most human breast carcinoma cells; however, only recently have PDGF β receptors been demonstrated in malignant breast tissue.  In the present study, the tissue localization of PDGF β receptor expression was studied in human breast carcinoma and nonmalignant breast tissues stained using both immunofluorescence and immunoperoxidase techniques.  The authors examd. a total of 29 cases of breast carcinomas, which showed both in situ and invasive components.  PDGF β receptor staining was localized in the periepithelial stroma and was particularly intense in regions immediately adjacent to carcinoma in situ components in all tumors examd.  A diffuse low level of PDGF β receptor staining was seen throughout the stroma of eight nonmalignant breast tissues as well as of nonmalignant regions of tumor tissues.  Image anal. was used to assess the coincidence of staining of PDGF β receptor with epithelial or stromal cells in 13 of the 29 tumor tissues studied.  Less than 5% of malignant ductal epithelium or myoepithelium showed PDGF β receptor staining.  Anal. with stromal cell type-specific markers indicated significant localization of PDGF β receptor primarily within α smooth muscle actin-staining cells (32%) and vascular endothelial cells (41%) in the periepithelial stroma.  PDGF β receptor positivity was strongly assocd. with periepithelial stromal cells adjacent to the basement membrane surrounding regions of carcinoma in situ but was less intense in regions of invasive carcinoma where basement membrane was degraded.  The absence of PDGF β receptors on carcinoma cells and their presence in the surrounding stroma suggest a paracrine stimulation of adjacent stromal tissue by malignant epithelial cells in human breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQVfzjQEWBw7Vg90H21EOLACvtfcHk0liZgPtRVWD72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1Wlurc%253D&md5=806364664031dd15ee36f31b84747abd</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DB.%26aulast%3DKlassen%26aufirst%3DJ.%26aulast%3DCossette%26aufirst%3DN.%26aulast%3DSterns%26aufirst%3DE.%26aulast%3DTuck%26aufirst%3DA.%26aulast%3DDeeley%26aufirst%3DR.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DElliott%26aufirst%3DB.%26atitle%3DLocalization%2520of%2520platelet-derived%2520growth%2520factor%2520beta%2520receptor%2520expression%2520in%2520the%2520periepithelial%2520stroma%2520of%2520human%2520breast%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1996%26volume%3D2%26spage%3D773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veeraraghavan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethunath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiser, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renwick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zabransky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamness, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis-Filho, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">HER2 reactivation through acquisition of the HER2 L755S mutation as a mechanism of acquired resistance to HER2-targeted therapy in HER2(+) breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5123</span>– <span class="NLM_lpage">5134</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-2191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-16-2191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28487443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCit7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5123-5134&author=X.+Xuauthor=C.+De+Angelisauthor=K.+A.+Burkeauthor=A.+Nardoneauthor=H.+Huauthor=L.+Qinauthor=J.+Veeraraghavanauthor=V.+Sethunathauthor=L.+M.+Heiserauthor=N.+Wangauthor=C.+K.+Y.+Ngauthor=E.+S.+Chenauthor=A.+Renwickauthor=T.+Wangauthor=S.+Nandaauthor=M.+Sheaauthor=T.+Mitchellauthor=M.+Rajendranauthor=I.+Watersauthor=D.+J.+Zabranskyauthor=K.+L.+Scottauthor=C.+Gutierrezauthor=C.+Nagiauthor=F.+C.+Geyerauthor=G.+C.+Chamnessauthor=B.+H.+Parkauthor=C.+A.+Shawauthor=S.+G.+Hilsenbeckauthor=M.+F.+Rimawiauthor=J.+W.+Grayauthor=B.+Weigeltauthor=J.+S.+Reis-Filhoauthor=C.+K.+Osborneauthor=R.+Schiff&title=HER2+reactivation+through+acquisition+of+the+HER2+L755S+mutation+as+a+mechanism+of+acquired+resistance+to+HER2-targeted+therapy+in+HER2%28%2B%29+breast+cancer&doi=10.1158%2F1078-0432.ccr-16-2191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 Reactivation through Acquisition of the HER2 L755S Mutation as a Mechanism of Acquired Resistance to HER2-targeted Therapy in HER2+ Breast Cancer</span></div><div class="casAuthors">Xu, Xiaowei; De Angelis, Carmine; Burke, Kathleen A.; Nardone, Agostina; Hu, Huizhong; Qin, Lanfang; Veeraraghavan, Jamunarani; Sethunath, Vidyalakshmi; Heiser, Laura M.; Wang, Nicholas; Ng, Charlotte K. Y.; Chen, Edward S.; Renwick, Alexander; Wang, Tao; Nanda, Sarmistha; Shea, Martin; Mitchell, Tamika; Rajendran, Mahitha; Waters, Ian; Zabransky, Daniel J.; Scott, Kenneth L.; Gutierrez, Carolina; Nagi, Chandandeep; Geyer, Felipe C.; Chamness, Gary C.; Park, Ben H.; Shaw, Chad A.; Hilsenbeck, Susan G.; Rimawi, Mothaffar F.; Gray, Joe W.; Weigelt, Britta; Reis-Filho, Jorge S.; Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5123-5134</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to anti-HER2 therapies in HER2+ breast cancer can occur through activation of alternative survival pathways or reactivation of the HER signaling network.  Here we employed BT474 parental and treatment-resistant cell line models to investigate a mechanism by which HER2+ breast cancer can reactivate the HER network under potent HER2-targeted therapies.  Resistant derivs. to lapatinib (L), trastuzumab (T), or the combination (LR/TR/LTR) were developed independently from two independent estrogen receptor ER+/HER2+ BT474 cell lines (AZ/ATCC).  Two derivs. resistant to the lapatinib-contg. regimens (BT474/AZ-LR and BT474/ATCC-LTR lines) that showed HER2 reactivation at the time of resistance were subjected to massive parallel sequencing and compared with parental lines.  Ectopic expression and mutant-specific siRNA interference were applied to analyze the mutation functionally.  In vitro and in vivo expts. were performed to test alternative therapies for mutant HER2 inhibition.  Genomic analyses revealed that the HER2L755S mutation was the only common somatic mutation gained in the BT474/AZ-LR and BT474/ATCC-LTR lines.  Ectopic expression of HER2L755S induced acquired lapatinib resistance in the BT474/AZ, SK-BR-3, and AU565 parental cell lines.  HER2L755S-specific siRNA knockdown reversed the resistance in BT474/AZ-LR and BT474/ATCC-LTR lines.  The HER1/2-irreversible inhibitors afatinib and neratinib substantially inhibited both resistant cell growth and the HER2 and downstream AKT/MAPK signaling driven by HER2L755S in vitro and in vivo.  HER2 reactivation through acquisition of the HER2L755S mutation was identified as a mechanism of acquired resistance to lapatinib-contg. HER2-targeted therapy in preclin. HER2-amplified breast cancer models, which can be overcome by irreversible HER1/2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqr23ozygbtLVg90H21EOLACvtfcHk0liZgPtRVWD72A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCit7jP&md5=2c984948a7b4b223241e320025526fe0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-2191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-2191%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DK.%2BA.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DVeeraraghavan%26aufirst%3DJ.%26aulast%3DSethunath%26aufirst%3DV.%26aulast%3DHeiser%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DNg%26aufirst%3DC.%2BK.%2BY.%26aulast%3DChen%26aufirst%3DE.%2BS.%26aulast%3DRenwick%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DNanda%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DMitchell%26aufirst%3DT.%26aulast%3DRajendran%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DI.%26aulast%3DZabransky%26aufirst%3DD.%2BJ.%26aulast%3DScott%26aufirst%3DK.%2BL.%26aulast%3DGutierrez%26aufirst%3DC.%26aulast%3DNagi%26aufirst%3DC.%26aulast%3DGeyer%26aufirst%3DF.%2BC.%26aulast%3DChamness%26aufirst%3DG.%2BC.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DShaw%26aufirst%3DC.%2BA.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DWeigelt%26aufirst%3DB.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DHER2%2520reactivation%2520through%2520acquisition%2520of%2520the%2520HER2%2520L755S%2520mutation%2520as%2520a%2520mechanism%2520of%2520acquired%2520resistance%2520to%2520HER2-targeted%2520therapy%2520in%2520HER2%2528%252B%2529%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D5123%26epage%3D5134%26doi%3D10.1158%2F1078-0432.ccr-16-2191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazalpour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millis, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatalica, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, M. F.</span></span> <span> </span><span class="NLM_article-title">mutations in the kinase domain of the her2/erbb2 gene identified in a wide variety of human cancers</span>. <i>J. Mol. Diagn.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.jmoldx.2015.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jmoldx.2015.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26320869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOntLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=487-495&author=W.+Wenauthor=W.+Chenauthor=N.+Xiaoauthor=R.+Benderauthor=A.+Ghazalpourauthor=Z.+Tanauthor=J.+Swensenauthor=S.+Z.+Millisauthor=G.+Basuauthor=Z.+Gatalicaauthor=M.+F.+Press&title=mutations+in+the+kinase+domain+of+the+her2%2Ferbb2+gene+identified+in+a+wide+variety+of+human+cancers&doi=10.1016%2Fj.jmoldx.2015.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the Kinase Domain of the HER2/ERBB2 Gene Identified in a Wide Variety of Human Cancers</span></div><div class="casAuthors">Wen, Wenhsiang; Chen, Wangjuh; Xiao, Nick; Bender, Ryan; Ghazalpour, Anatole; Tan, Zheng; Swensen, Jeffrey; Millis, Sherri Z.; Basu, Gargi; Gatalica, Zoran; Press, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Diagnostics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">487-495</span>CODEN:
                <span class="NLM_cas:coden">JMDIFP</span>;
        ISSN:<span class="NLM_cas:issn">1525-1578</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The HER2 (official name ERBB2) gene encodes a membrane receptor in the epidermal growth factor receptor family amplified and overexpressed in adenocarcinoma.  Activating mutations also occur in several cancers.  We report mutation analyses of the HER2 kinase domain in 7497 histol. diverse cancers.  Forty-five genes, including the kinase domain of HER2 with HER2 IHC and dual in situ hybridization, were analyzed in tumors from 7497 patients with cancer, including 850 breast, 770 colorectal, 910 non-small cell lung, 823 uterine or cervical, 1372 ovarian, and 297 pancreatic cancers, as well as 323 melanomas and 2152 other solid tumors.  Sixty-nine HER2 kinase domain mutations were identified in tumors from 68 patients (approx. 1% of all cases, ranging from absent in sarcomas to 4% in urothelial cancers), which included previously published activating mutations and 13 novel mutations.  Fourteen cases with coexisting HER2 mutation and amplification and/or overexpression were identified.  Fifty-two of 68 patients had addnl. mutations in other analyzed genes, whereas 16 patients (23%) had HER2 mutations identified as the sole driver mutation.  HER2 mutations coexisted with HER2 gene amplification and overexpression and with mutations in other functionally important genes.  HER2 mutations were identified as the only driver mutation in a significant proportion of solid cancers.  Evaluation of anti-HER2 therapies in nonamplified, HER2-mutated cancers is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3gPcn7n7fYbVg90H21EOLACvtfcHk0lj9vJGulvI62w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOntLjP&md5=045c22bad0d1d3513ce930c44a62cc30</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.jmoldx.2015.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmoldx.2015.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DN.%26aulast%3DBender%26aufirst%3DR.%26aulast%3DGhazalpour%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DZ.%26aulast%3DSwensen%26aufirst%3DJ.%26aulast%3DMillis%26aufirst%3DS.%2BZ.%26aulast%3DBasu%26aufirst%3DG.%26aulast%3DGatalica%26aufirst%3DZ.%26aulast%3DPress%26aufirst%3DM.%2BF.%26atitle%3Dmutations%2520in%2520the%2520kinase%2520domain%2520of%2520the%2520her2%252Ferbb2%2520gene%2520identified%2520in%2520a%2520wide%2520variety%2520of%2520human%2520cancers%26jtitle%3DJ.%2520Mol.%2520Diagn.%26date%3D2015%26volume%3D17%26spage%3D487%26epage%3D495%26doi%3D10.1016%2Fj.jmoldx.2015.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">AACR
Project GENIE Consortium</span> <span> </span><span class="NLM_article-title">AACR project genie: powering precision medicine through an international
consortium</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-0151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F2159-8290.CD-17-0151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28572459" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=818-831&author=AACR%0AProject+GENIE+Consortium&title=AACR+project+genie%3A+powering+precision+medicine+through+an+international%0Aconsortium&doi=10.1158%2F2159-8290.CD-17-0151"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0151%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAACR%2520project%2520genie%253A%2520powering%2520precision%2520medicine%2520through%2520an%2520international%250Aconsortium%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D818%26epage%3D831%26doi%3D10.1158%2F2159-8290.CD-17-0151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span> </span><span class="NLM_article-title">My Cancer Genome, Genetically Informed Cancer Medicine</span>. <a href="https://www.mycancergenome.org/content/alteration/erbb2-l755s/" class="extLink">https://www.mycancergenome.org/content/alteration/erbb2-l755s/</a> (accessed Feb 14, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=My+Cancer+Genome%2C+Genetically+Informed+Cancer+Medicine.+https%3A%2F%2Fwww.mycancergenome.org%2Fcontent%2Falteration%2Ferbb2-l755s%2F+%28accessed+Feb+14%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DMy%2520Cancer%2520Genome%252C%2520Genetically%2520Informed%2520Cancer%2520Medicine" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zabransky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yankaskas, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croessmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavuri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Red Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimino-Mathews, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravero, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Button, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyker-Snowman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantopoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span> <span> </span><span class="NLM_article-title">HER2 missense mutations have distinct effects on oncogenic signaling and migration</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E6205</span>– <span class="NLM_lpage">E6214</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516853112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1073%2Fpnas.1516853112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26508629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsleksbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E6205-E6214&author=D.+J.+Zabranskyauthor=C.+L.+Yankaskasauthor=R.+L.+Cochranauthor=H.+Y.+Wongauthor=S.+Croessmannauthor=D.+Chuauthor=S.+M.+Kavuriauthor=M.+Red+Brewerauthor=D.+M.+Rosenauthor=W.+B.+Daltonauthor=A.+Cimino-Mathewsauthor=K.+Craveroauthor=B.+Buttonauthor=K.+Kyker-Snowmanauthor=J.+Cidadoauthor=B.+Erlangerauthor=H.+A.+Parsonsauthor=K.+M.+Mantoauthor=R.+Boseauthor=J.+Lauringauthor=C.+L.+Arteagaauthor=K.+Konstantopoulosauthor=B.+H.+Park&title=HER2+missense+mutations+have+distinct+effects+on+oncogenic+signaling+and+migration&doi=10.1073%2Fpnas.1516853112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 missense mutations have distinct effects on oncogenic signaling and migration</span></div><div class="casAuthors">Zabransky, Daniel J.; Yankaskas, Christopher L.; Cochran, Rory L.; Wong, Hong Yuen; Croessmann, Sarah; Chu, David; Kavuri, Shyam M.; Brewer, Monica Red; Rosen, D. Marc; Dalton, W. Brian; Cimino-Mathews, Ashley; Cravero, Karen; Button, Berry; Kyker-Snowman, Kelly; Cidado, Justin; Erlanger, Bracha; Parsons, Heather A.; Manto, Kristen M.; Bose, Ron; Lauring, Josh; Arteaga, Carlos L.; Konstantopoulos, Konstantinos; Park, Ben Ho</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E6205-E6214</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recurrent human epidermal growth factor receptor 2 (HER2) missense mutations have been reported in human cancers.  These mutations occur primarily in the absence of HER2 gene amplification such that most HER2-mutant tumors are classified as "neg." by FISH or immunohistochem. assays.  It remains unclear whether nonamplified HER2 missense mutations are oncogenic and whether they are targets for HER2-directed therapies that are currently approved for the treatment of HER2 gene-amplified breast cancers.  Here we functionally characterize HER2 kinase and extracellular domain mutations through gene editing of the endogenous loci in HER2 nonamplified human breast epithelial cells.  In in vitro and in vivo assays, the majority of HER2 missense mutations do not impart detectable oncogenic changes.  However, the HER2 V777L mutation increased biochem. pathway activation and, in the context of a PIK3CA mutation, enhanced migratory features in vitro.  However, the V777L mutation did not alter in vivo tumorigenicity or sensitivity to HER2-directed therapies in proliferation assays.  Our results suggest the oncogenicity and potential targeting of HER2 missense mutations should be considered in the context of cooperating genetic alterations and provide previously unidentified insights into functional anal. of HER2 mutations and strategies to target them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQmDBjvnG-6LVg90H21EOLACvtfcHk0lj9vJGulvI62w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsleksbnL&md5=b78457c6678b940e28afed1b50554f44</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516853112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516853112%26sid%3Dliteratum%253Aachs%26aulast%3DZabransky%26aufirst%3DD.%2BJ.%26aulast%3DYankaskas%26aufirst%3DC.%2BL.%26aulast%3DCochran%26aufirst%3DR.%2BL.%26aulast%3DWong%26aufirst%3DH.%2BY.%26aulast%3DCroessmann%26aufirst%3DS.%26aulast%3DChu%26aufirst%3DD.%26aulast%3DKavuri%26aufirst%3DS.%2BM.%26aulast%3DRed%2BBrewer%26aufirst%3DM.%26aulast%3DRosen%26aufirst%3DD.%2BM.%26aulast%3DDalton%26aufirst%3DW.%2BB.%26aulast%3DCimino-Mathews%26aufirst%3DA.%26aulast%3DCravero%26aufirst%3DK.%26aulast%3DButton%26aufirst%3DB.%26aulast%3DKyker-Snowman%26aufirst%3DK.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DErlanger%26aufirst%3DB.%26aulast%3DParsons%26aufirst%3DH.%2BA.%26aulast%3DManto%26aufirst%3DK.%2BM.%26aulast%3DBose%26aufirst%3DR.%26aulast%3DLauring%26aufirst%3DJ.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DKonstantopoulos%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DB.%2BH.%26atitle%3DHER2%2520missense%2520mutations%2520have%2520distinct%2520effects%2520on%2520oncogenic%2520signaling%2520and%2520migration%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3DE6205%26epage%3DE6214%26doi%3D10.1073%2Fpnas.1516853112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavuri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searleman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koboldt, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span> <span> </span><span class="NLM_article-title">Activating HER2 mutations in HER2 gene amplification negative breast cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-12-0349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F2159-8290.cd-12-0349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23220880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=224-237&author=R.+Boseauthor=S.+M.+Kavuriauthor=A.+C.+Searlemanauthor=W.+Shenauthor=D.+Shenauthor=D.+C.+Koboldtauthor=J.+Monseyauthor=N.+Goelauthor=A.+B.+Aronsonauthor=S.+Liauthor=C.+X.+Maauthor=L.+Dingauthor=E.+R.+Mardisauthor=M.+J.+Ellis&title=Activating+HER2+mutations+in+HER2+gene+amplification+negative+breast+cancer&doi=10.1158%2F2159-8290.cd-12-0349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Activating HER2 Mutations in HER2 Gene Amplification Negative Breast Cancer</span></div><div class="casAuthors">Bose, Ron; Kavuri, Shyam M.; Searleman, Adam C.; Shen, Wei; Shen, Dong; Koboldt, Daniel C.; Monsey, John; Goel, Nicholas; Aronson, Adam B.; Li, Shunqiang; Ma, Cynthia X.; Ding, Li; Mardis, Elaine R.; Ellis, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-237</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Data from 8 breast cancer genome-sequencing projects identified 25 patients with HER2 somatic mutations in cancers lacking HER2 gene amplification.  To det. the phenotype of these mutations, we functionally characterized 13 HER2 mutations using in vitro kinase assays, protein structure anal., cell culture, and xenograft expts.  Seven of these mutations are activating mutations, including G309A, D769H, D769Y, V777L, P780ins, V842I, and R896C. HER2 in-frame deletion 755-759, which is homologous to EGF receptor (EGFR) exon 19 in-frame deletions, had a neomorphic phenotype with increased phosphorylation of EGFR or HER3.  L755S produced lapatinib resistance, but was not an activating mutation in our exptl. systems.  All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.  These findings show that HER2 somatic mutation is an alternative mechanism to activate HER2 in breast cancer and they validate HER2 somatic mutations as drug targets for breast cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpInHiFiSDrrVg90H21EOLACvtfcHk0lgT7Wl2Y6cFpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wktrg%253D&md5=4b271e961ad4a68862e4a3ba80e88972</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-12-0349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-12-0349%26sid%3Dliteratum%253Aachs%26aulast%3DBose%26aufirst%3DR.%26aulast%3DKavuri%26aufirst%3DS.%2BM.%26aulast%3DSearleman%26aufirst%3DA.%2BC.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DD.%26aulast%3DKoboldt%26aufirst%3DD.%2BC.%26aulast%3DMonsey%26aufirst%3DJ.%26aulast%3DGoel%26aufirst%3DN.%26aulast%3DAronson%26aufirst%3DA.%2BB.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DC.%2BX.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26atitle%3DActivating%2520HER2%2520mutations%2520in%2520HER2%2520gene%2520amplification%2520negative%2520breast%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D224%26epage%3D237%26doi%3D10.1158%2F2159-8290.cd-12-0349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kancha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Bubnoff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartosch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peschel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duyster, J.</span></span> <span> </span><span class="NLM_article-title">Differential sensitivity of ERBB2 kinase domain mutations towards lapatinib</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e26760</span>– <span class="NLM_lpage">e26769</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0026760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0026760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22046346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e26760-e26769&author=R.+K.+Kanchaauthor=N.+von+Bubnoffauthor=N.+Bartoschauthor=C.+Peschelauthor=R.+A.+Enghauthor=J.+Duyster&title=Differential+sensitivity+of+ERBB2+kinase+domain+mutations+towards+lapatinib&doi=10.1371%2Fjournal.pone.0026760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Differential sensitivity of ERBB2 kinase domain mutations towards lapatinib</span></div><div class="casAuthors">Kancha, Rama Krishna; von Bubnoff, Nikolas; Bartosch, Natalie; Peschel, Christian; Engh, Richard A.; Duyster, Justus</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e26760</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Overexpression of the ERBB2 kinase is obsd. in about one-third of breast cancer patients and the dual ERBB1/ERBB2 kinase inhibitor lapatinib was recently approved for the treatment of advanced ERBB2-pos. breast cancer.  Mutations in the ERBB2 receptor have recently been reported in breast cancer at diagnosis and also in gastric, colorectal and lung cancer.  These mutations may have an impact on the clin. responses achieved with lapatinib in breast cancer and may also have a potential impact on the use of lapatinib in other solid cancers.  However, the sensitivity of lapatinib towards clin. obsd. ERBB2 mutations is not known.  Methodol./Principal Findings: We cloned a panel of 8 clin. obsd. ERBB2 mutations, established stable cell lines and characterized their sensitivity towards lapatinib and alternative ERBB2 inhibitors.  Both lapatinib-sensitive and lapatinib-resistant ERBB2 mutations were obsd.  Interestingly, we were able to generate lapatinib resistance mutations in wt-ERBB2 cells incubated with lapatinib for prolonged periods of time.  This indicates that these resistance mutations may also cause secondary resistance in lapatinib-treated patients.  Lapatinib-resistant ERBB2 mutations were found to be highly resistant towards AEE788 treatment but remained sensitive towards the dual irreversible inhibitors CL-387785 and WZ-4002.  Conclusions/Significance: Patients harbouring certain ERBB2 kinase domain mutations at diagnosis may not benefit from lapatinib treatment.  Moreover, secondary lapatinib resistance may develop due to kinase domain mutations.  Irreversible ERBB2 inhibitors may offer alternative treatment options for breast cancer and other solid tumor patients harbouring lapatinib resistance mutations.  In addn., these inhibitors may be of interest in the scenario of secondary lapatinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGXRqTTP0WubVg90H21EOLACvtfcHk0lgT7Wl2Y6cFpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu7fF&md5=84ef463604d86644c79f52af52d093e8</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0026760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0026760%26sid%3Dliteratum%253Aachs%26aulast%3DKancha%26aufirst%3DR.%2BK.%26aulast%3Dvon%2BBubnoff%26aufirst%3DN.%26aulast%3DBartosch%26aufirst%3DN.%26aulast%3DPeschel%26aufirst%3DC.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DDuyster%26aufirst%3DJ.%26atitle%3DDifferential%2520sensitivity%2520of%2520ERBB2%2520kinase%2520domain%2520mutations%2520towards%2520lapatinib%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De26760%26epage%3De26769%26doi%3D10.1371%2Fjournal.pone.0026760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukouvala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, R. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gendreau, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfrey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vysotskaia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, T. S.</span></span> <span> </span><span class="NLM_article-title">EXEL-7647 inhibits mutant forms of ErbB2 associated with lapatinib resistance and neoplastic transformation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-07-4367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-07-4367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18413839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=2465-2475&author=T.+Troweauthor=S.+Boukouvalaauthor=K.+Calkinsauthor=R.+E.+Cutlerauthor=R.+Fongauthor=R.+Funkeauthor=S.+B.+Gendreauauthor=Y.+D.+Kimauthor=N.+Millerauthor=J.+R.+Woolfreyauthor=V.+Vysotskaiaauthor=J.+P.+Yangauthor=M.+E.+Gerritsenauthor=D.+J.+Matthewsauthor=P.+Lambauthor=T.+S.+Heuer&title=EXEL-7647+inhibits+mutant+forms+of+ErbB2+associated+with+lapatinib+resistance+and+neoplastic+transformation&doi=10.1158%2F1078-0432.ccr-07-4367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">EXEL-7647 Inhibits Mutant Forms of ErbB2 Associated with Lapatinib Resistance and Neoplastic Transformation</span></div><div class="casAuthors">Trowe, Torsten; Boukouvala, Sotiria; Calkins, Keith; Cutler, Richard E., Jr.; Fong, Ryan; Funke, Roel; Gendreau, Steven B.; Kim, Yong D.; Miller, Nicole; Woolfrey, John R.; Vysotskaia, Valentina; Yang, Jing Ping; Gerritsen, Mary E.; Matthews, David J.; Lamb, Peter; Heuer, Timothy S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2465-2475</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations assocd. with resistance to kinase inhibition are an important mechanism of intrinsic or acquired loss of clin. efficacy for kinase-targeted therapeutics.  We report the prospective discovery of ErbB2 mutations that confer resistance to the small-mol. inhibitor lapatinib.  We did in vitro screening using a randomly mutagenized ErbB2 expression library in Ba/F3 cells, which were dependent on ErbB2 activity for survival and growth.  Lapatinib resistance screens identified mutations at 16 different ErbB2 amino acid residues, with 12 mutated amino acids mapping to the kinase domain.  Mutations conferring the greatest lapatinib resistance cluster in the NH2-terminal kinase lobe and hinge region.  Structural computer modeling studies suggest that lapatinib resistance is caused by multiple mechanisms; including direct steric interference and restriction of conformational flexibility (the inactive state required for lapatinib binding is energetically unfavorable).  ErbB2 T798I imparts the strongest lapatinib resistance effect and is analogous to the epidermal growth factor receptor T790M, ABL T315I, and cKIT T670I gatekeeper mutations that are assocd. with clin. drug resistance.  ErbB2 mutants assocd. with lapatinib resistance transformed NIH-3T3 cells, including L755S and T733I mutations known to occur in human breast and gastric carcinomas, supporting a direct mechanism for lapatinib resistance in ErbB2-driven human cancers.  The epidermal growth factor receptor/ErbB2/vascular endothelial growth factor receptor inhibitor EXEL-7647 was found to inhibit almost all lapatinib resistance-assocd. mutations.  Furthermore, no ErbB2 mutations were found to be assocd. with EXEL-7647 resistance and lapatinib sensitivity.  Taken together, these data suggest potential target-based mechanisms of resistance to lapatinib and suggest that EXEL-7647 may be able to circumvent these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIufvHezxJo7Vg90H21EOLACvtfcHk0lgT7Wl2Y6cFpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D&md5=c51774479df7be7c06d2cc22e8c58afc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-07-4367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-07-4367%26sid%3Dliteratum%253Aachs%26aulast%3DTrowe%26aufirst%3DT.%26aulast%3DBoukouvala%26aufirst%3DS.%26aulast%3DCalkins%26aufirst%3DK.%26aulast%3DCutler%26aufirst%3DR.%2BE.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGendreau%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BD.%26aulast%3DMiller%26aufirst%3DN.%26aulast%3DWoolfrey%26aufirst%3DJ.%2BR.%26aulast%3DVysotskaia%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DJ.%2BP.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DMatthews%26aufirst%3DD.%2BJ.%26aulast%3DLamb%26aufirst%3DP.%26aulast%3DHeuer%26aufirst%3DT.%2BS.%26atitle%3DEXEL-7647%2520inhibits%2520mutant%2520forms%2520of%2520ErbB2%2520associated%2520with%2520lapatinib%2520resistance%2520and%2520neoplastic%2520transformation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D2465%26epage%3D2475%26doi%3D10.1158%2F1078-0432.ccr-07-4367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Breast cancer cell line classification and its relevance with breast tumor subtyping</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.7150/jca.18457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.7150%2Fjca.18457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29158785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1M3jtlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3131-3141&author=X.+Daiauthor=H.+Chengauthor=Z.+Baiauthor=J.+Li&title=Breast+cancer+cell+line+classification+and+its+relevance+with+breast+tumor+subtyping&doi=10.7150%2Fjca.18457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Breast Cancer Cell Line Classification and Its Relevance with Breast Tumor Subtyping</span></div><div class="casAuthors">Dai Xiaofeng; Cheng Hongye; Bai Zhonghu; Li Jia; Dai Xiaofeng; Cheng Hongye; Bai Zhonghu; Li Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3131-3141</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Breast cancer cell lines have been widely used for breast cancer modelling which encompasses a panel of diseases with distinct phenotypical associations.  Though cell lines provide unlimited homogenous materials for tumor studies and are relatively easy to culture, they are known to accumulate mutations duringthe initial establishment and subsequent series of cultivations.  Thus, whether breast cancer cell line heterogeneity reflects that of carcinoma remains an important issue to resolve before drawing any reliable conclusion at the tumor level using cell lines.  Inconsistent nomenclatures used for breast cancer cell line subtyping and the different number of subtypes grouped for cell lines and tumors make their direct matching elusive.  By analyzing the molecular features of 92 breast cancer cell lines as documented by different literatures, we categorize 84 cell lines into 5 groups to be consistent with breast tumor classification.  After combing through these cell lines, we summarized the molecular features, genetically and epigenetically, of each subtype, and manually documented 10 cell lines lacking explicit information on subtyping.  Nine cell lines, either found inconsistent on their primary molecular features from different studies or being contaminated at the origin, are not suggested as the first choice for experimental use.  We conclude that breast tumor cell lines, though having a high mutational frequency with many uncertainties and could not fully capture breast cancer heterogeneity, are feasible but crude models for tumors of the same subtype.  New cell lines with enriched interferon regulated genes need to be established to enlarge the coverage of cell lines on tumor heterogeneity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSgfx4Cpa1dYL5oR8RchjrrfW6udTcc2ebh5sBYVnnMJ7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3jtlWjtg%253D%253D&md5=b350670e7d74de3157883aa28e950851</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.7150%2Fjca.18457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.18457%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DBreast%2520cancer%2520cell%2520line%2520classification%2520and%2520its%2520relevance%2520with%2520breast%2520tumor%2520subtyping%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D3131%26epage%3D3141%26doi%3D10.7150%2Fjca.18457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neve, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridlyand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baehner, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fevr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppe, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVries, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilwell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albertson, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span> <span> </span><span class="NLM_article-title">A collection of breast cancer cell lines for the study of functionally distinct cancer subtypes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2006.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ccr.2006.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17157791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCnsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=515-527&author=R.+M.+Neveauthor=K.+Chinauthor=J.+Fridlyandauthor=J.+Yehauthor=F.+L.+Baehnerauthor=T.+Fevrauthor=L.+Clarkauthor=N.+Bayaniauthor=J.-P.+Coppeauthor=F.+Tongauthor=T.+Speedauthor=P.+T.+Spellmanauthor=S.+DeVriesauthor=A.+Lapukauthor=N.+J.+Wangauthor=W.-L.+Kuoauthor=J.+L.+Stilwellauthor=D.+Pinkelauthor=D.+G.+Albertsonauthor=F.+M.+Waldmanauthor=F.+McCormickauthor=R.+B.+Dicksonauthor=M.+D.+Johnsonauthor=M.+Lippmanauthor=S.+Ethierauthor=A.+Gazdarauthor=J.+W.+Gray&title=A+collection+of+breast+cancer+cell+lines+for+the+study+of+functionally+distinct+cancer+subtypes&doi=10.1016%2Fj.ccr.2006.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A collection of breast cancer cell lines for the study of functionally distinct cancer subtypes</span></div><div class="casAuthors">Neve, Richard M.; Chin, Koei; Fridlyand, Jane; Yeh, Jennifer; Baehner, Frederick L.; Fevr, Tea; Clark, Laura; Bayani, Nora; Coppe, Jean-Philippe; Tong, Frances; Speed, Terry; Spellman, Paul T.; DeVries, Sandy; Lapuk, Anna; Wang, Nick J.; Kuo, Wen-Lin; Stilwell, Jackie L.; Pinkel, Daniel; Albertson, Donna G.; Waldman, Frederic M.; McCormick, Frank; Dickson, Robert B.; Johnson, Michael D.; Lippman, Marc; Ethier, Stephen; Gazdar, Adi; Gray, Joe W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">515-527</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent studies suggest that thousands of genes may contribute to breast cancer pathophysiologies when deregulated by genomic or epigenomic events.  Here, we describe a model "system" to appraise the functional contributions of these genes to breast cancer subsets.  In general, the recurrent genomic and transcriptional characteristics of 51 breast cancer cell lines mirror those of 145 primary breast tumors, although some significant differences are documented.  The cell lines that comprise the system also exhibit the substantial genomic, transcriptional, and biol. heterogeneity found in primary tumors.  We show, using Trastuzumab (Herceptin) monotherapy as an example, that the system can be used to identify mol. features that predict or indicate response to targeted therapies or other physiol. perturbations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHlT0rEcwBrVg90H21EOLACvtfcHk0lj6Rd1PHPpTXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCnsrbE&md5=218682c6f7ffebff329208d476ab35ad</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2006.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2006.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DNeve%26aufirst%3DR.%2BM.%26aulast%3DChin%26aufirst%3DK.%26aulast%3DFridlyand%26aufirst%3DJ.%26aulast%3DYeh%26aufirst%3DJ.%26aulast%3DBaehner%26aufirst%3DF.%2BL.%26aulast%3DFevr%26aufirst%3DT.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DBayani%26aufirst%3DN.%26aulast%3DCoppe%26aufirst%3DJ.-P.%26aulast%3DTong%26aufirst%3DF.%26aulast%3DSpeed%26aufirst%3DT.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DDeVries%26aufirst%3DS.%26aulast%3DLapuk%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DN.%2BJ.%26aulast%3DKuo%26aufirst%3DW.-L.%26aulast%3DStilwell%26aufirst%3DJ.%2BL.%26aulast%3DPinkel%26aufirst%3DD.%26aulast%3DAlbertson%26aufirst%3DD.%2BG.%26aulast%3DWaldman%26aufirst%3DF.%2BM.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26aulast%3DLippman%26aufirst%3DM.%26aulast%3DEthier%26aufirst%3DS.%26aulast%3DGazdar%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26atitle%3DA%2520collection%2520of%2520breast%2520cancer%2520cell%2520lines%2520for%2520the%2520study%2520of%2520functionally%2520distinct%2520cancer%2520subtypes%26jtitle%3DCancer%2520Cell%26date%3D2006%26volume%3D10%26spage%3D515%26epage%3D527%26doi%3D10.1016%2Fj.ccr.2006.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leclercq, G.</span></span> <span> </span><span class="NLM_article-title">Relevance of breast cancer cell lines as models for breast tumours: an update</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1023/b:brea.0000014042.54925.cc</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1023%2Fb%3Abrea.0000014042.54925.cc" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=14758095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Cisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2004&pages=249-289&author=M.+Lacroixauthor=G.+Leclercq&title=Relevance+of+breast+cancer+cell+lines+as+models+for+breast+tumours%3A+an+update&doi=10.1023%2Fb%3Abrea.0000014042.54925.cc"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Relevance of Breast Cancer Cell Lines as Models for Breast Tumours: An Update</span></div><div class="casAuthors">Lacroix, Marc; Leclercq, Guy</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-289</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  The no. of available breast cancer cell (BCC) lines is small, and only a very few of them were extensively studied.  Whether they are representative of the tumors from which they originated remains a matter of debate.  Whether their diversity mirrors the well-known inter-tumoral heterogeneity is another essential question.  While numerous similarities have long been found between cell lines and tumors, recent tech. advances, including the use of micro-arrays and comparative genetic anal., have brought new data to the discussion.  This paper presents most of the BCC lines that have been described in some detail to date.  It evaluates the accuracy of the few of them widely used (MCF-7, T-47D, BT-474, SK-BR-3, MDA-MB-231, Hs578T) as tumor models.  It is concluded that BCC lines are likely to reflect, to a large extent, the features of cancer cells in vivo.  The importance of estrogen receptor-α (gene ESR1) and Her-2/neu (ERBB2) as classifiers for cell lines and tumors is underlined.  The recourse to a larger set of cell lines is suggested since the exact origin of some of the widely used lines remains ambiguous.  Investigations on addnl. specific lines are expected to improve our knowledge of BCC and of the dialogue that these maintain with their surrounding normal cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof86ncAv_qXrVg90H21EOLACvtfcHk0lj6Rd1PHPpTXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Cisw%253D%253D&md5=d477e4aea1298cd3b2d6fe2ff4efcb08</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1023%2Fb%3Abrea.0000014042.54925.cc&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fb%253Abrea.0000014042.54925.cc%26sid%3Dliteratum%253Aachs%26aulast%3DLacroix%26aufirst%3DM.%26aulast%3DLeclercq%26aufirst%3DG.%26atitle%3DRelevance%2520of%2520breast%2520cancer%2520cell%2520lines%2520as%2520models%2520for%2520breast%2520tumours%253A%2520an%2520update%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2004%26volume%3D83%26spage%3D249%26epage%3D289%26doi%3D10.1023%2Fb%3Abrea.0000014042.54925.cc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bacus, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huberman, E.</span></span> <span> </span><span class="NLM_article-title">Differentiation of cultured human breast cancer cells (AU-565 and MCF-7) associated with loss of cell surface HER-2/neu antigen</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1002/mc.2940030607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fmc.2940030607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1980588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK3MXitVCksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=350-362&author=S.+S.+Bacusauthor=K.+Kiguchiauthor=D.+Chinauthor=C.+R.+Kingauthor=E.+Huberman&title=Differentiation+of+cultured+human+breast+cancer+cells+%28AU-565+and+MCF-7%29+associated+with+loss+of+cell+surface+HER-2%2Fneu+antigen&doi=10.1002%2Fmc.2940030607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation of cultured human breast cancer cells (AU-565 and MCF-7) associated with loss of cell surface HER-2/neu antigen</span></div><div class="casAuthors">Bacus, Sarah S.; Kiguchi, Kaoru; Chin, Dot; King, C. Richter; Huberman, Eliezer</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">350-62</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    </div><div class="casAbstract">The relationship between terminal cell differentiation and changes in the subcellular levels of the HER-2/neu antigen was investigated in cultured human breast cancer cells: AU-565 cells (which overexpress the HER-2/neu gene) and MCF-7 cells (which do not overexpress this gene).  Differentiation was achieved by treating the cells with mycophenolic acid (MPA), phorbol 12-myristate 13-acetate (PMA), retinoic acid (RA), or the TA-1 monoclonal antibody to the extracellular domain of the HER-2/neu protein.  Ten to twenty percent of the cells in untreated, sparsely growing AU-565 cultures exhibited morphol. maturation characterized by large lacy nuclei surrounded by sizable flat cytoplasms.  A fraction of these cells harbored milk factors such as casein and large lipid droplets.  Treatment of the AU-565 cells for 4 d with 9 μM MPA, 1.6 nM PMA, 2.5 μM RA, or 1 μg/mL TA-1 antibody resulted in cell growth inhibition and an increase in the percentage of cells (48-97%) that exhibit a mature phenotype.  MCF-7 cells were also susceptible to differentiation by MPA and RA, but to a lesser degree than the AU-565 cells.  Differentiation in the AU-565 and MCF-7 cells was assocd. with reduced immunostaining for the HER-2/neu protein at the cell surface membrane and with a transient increased diffuse immunostaining for this protein in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI686OvH-udLVg90H21EOLACvtfcHk0liysbRmK1ED0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVCksL4%253D&md5=f917f0c668f7a762e4dcd9063e414f77</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1002%2Fmc.2940030607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.2940030607%26sid%3Dliteratum%253Aachs%26aulast%3DBacus%26aufirst%3DS.%2BS.%26aulast%3DKiguchi%26aufirst%3DK.%26aulast%3DChin%26aufirst%3DD.%26aulast%3DKing%26aufirst%3DC.%2BR.%26aulast%3DHuberman%26aufirst%3DE.%26atitle%3DDifferentiation%2520of%2520cultured%2520human%2520breast%2520cancer%2520cells%2520%2528AU-565%2520and%2520MCF-7%2529%2520associated%2520with%2520loss%2520of%2520cell%2520surface%2520HER-2%252Fneu%2520antigen%26jtitle%3DMol.%2520Carcinog.%26date%3D1990%26volume%3D3%26spage%3D350%26epage%3D362%26doi%3D10.1002%2Fmc.2940030607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speirs, V.</span></span> <span> </span><span class="NLM_article-title">Choosing the right cell line for breast cancer research</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">215</span>, <span class="refDoi"> DOI: 10.1186/bcr2889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr2889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21884641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=215&author=D.+L.+Hollidayauthor=V.+Speirs&title=Choosing+the+right+cell+line+for+breast+cancer+research&doi=10.1186%2Fbcr2889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the right cell line for breast cancer research</span></div><div class="casAuthors">Holliday Deborah L; Speirs Valerie</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research : BCR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is a complex and heterogeneous disease.  Gene expression profiling has contributed significantly to our understanding of this heterogeneity at a molecular level, refining taxonomy based on simple measures such as histological type, tumour grade, lymph node status and the presence of predictive markers like oestrogen receptor and human epidermal growth factor receptor 2 (HER2) to a more sophisticated classification comprising luminal A, luminal B, basal-like, HER2-positive and normal subgroups.  In the laboratory, breast cancer is often modelled using established cell lines.  In the present review we discuss some of the issues surrounding the use of breast cancer cell lines as experimental models, in light of these revised clinical classifications, and put forward suggestions for improving their use in translational breast cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9z00kKYf1zjdblLE_E9PWfW6udTcc2eZO3sMON-Krhbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D&md5=9fbb9c5debbec50a5708666272fff46f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1186%2Fbcr2889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2889%26sid%3Dliteratum%253Aachs%26aulast%3DHolliday%26aufirst%3DD.%2BL.%26aulast%3DSpeirs%26aufirst%3DV.%26atitle%3DChoosing%2520the%2520right%2520cell%2520line%2520for%2520breast%2520cancer%2520research%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2011%26volume%3D13%26spage%3D215%26doi%3D10.1186%2Fbcr2889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Jaarsveld, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollestelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prager-van der Smissen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boersma, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmijr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiemer, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W.</span></span> <span> </span><span class="NLM_article-title">miRNA expression profiling of 51 human breast cancer cell lines reveals subtype and driver mutation-specific miRNAs</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">R33</span>, <span class="refDoi"> DOI: 10.1186/bcr3415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr3415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23601657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGqtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=R33&author=M.+Riazauthor=M.+T.+van+Jaarsveldauthor=A.+Hollestelleauthor=W.+J.+Prager-van+der+Smissenauthor=A.+A.+Heineauthor=A.+W.+Boersmaauthor=J.+Liuauthor=J.+Helmijrauthor=B.+Ozturkauthor=M.+Smidauthor=E.+A.+Wiemerauthor=J.+A.+Foekensauthor=J.+W.+Martens&title=miRNA+expression+profiling+of+51+human+breast+cancer+cell+lines+reveals+subtype+and+driver+mutation-specific+miRNAs&doi=10.1186%2Fbcr3415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">miRNA expression profiling of 51 human breast cancer cell lines reveals subtype and driver mutation-specific miRNAs</span></div><div class="casAuthors">Riaz, Muhammad; van Jaarsveld, Marijn T. M.; Hollestelle, Antoinette; Prager-van der Smissen, Wendy J. C.; Heine, Anouk A. J.; Boersma, Antonius W. M.; Liu, Jingjing; Helmijr, Jean; Ozturk, Bahar; Smid, Marcel; Wiemer, Erik A.; Foekens, John A.; Martens, John W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">R33</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Introduction: Breast cancer is a genetically and phenotypically complex disease.  To understand the role of miRNAs in this mol. complexity, we performed miRNA expression anal. in a cohort of molecularly well-characterized human breast cancer cell lines to identify miRNAs assocd. with the most common mol. subtypes and the most frequent genetic aberrations.  Methods: Using a microarray carrying LNA modified oligonucleotide capture probes, expression levels of 725 human miRNAs were measured in 51 breast cancer cell lines.  Differential miRNA expression was explored by unsupervised cluster anal. and was then assocd. with the mol. subtypes and genetic aberrations commonly present in breast cancer.  Results: Unsupervised cluster anal. using the most variably expressed miRNAs divided the 51 breast cancer cell lines into a major and a minor cluster predominantly mirroring the luminal and basal intrinsic subdivision of breast cancer cell lines.  One hundred and thirteen miRNAs were differentially expressed between these two main clusters.  Forty miRNAs were differentially expressed between basal-like and normal-like/claudin-low cell lines.  Within the luminal-group, 39 miRNAs were assocd. with ERBB2 overexpression and 24 with E-cadherin gene mutations, which are frequent in this subtype of breast cancer cell lines.  In contrast, 31 miRNAs were assocd. with E-cadherin promoter hypermethylation, which, contrary to E-cadherin mutation, is exclusively obsd. in breast cancer cell lines that are not of luminal origin.  Thirty miRNAs were assocd. with p16INK4 status while only a few miRNAs were assocd. with BRCA1, PIK3CA/PTEN and TP53 mutation status.  Twelve miRNAs were assocd. with DNA copy no. variation of the resp. locus.  Conclusion: Luminal-basal and epithelial-mesenchymal assocd. miRNAs det. the subdivision of miRNA transcriptome of breast cancer cell lines.  Specific sets of miRNAs were assocd. with ERBB2 overexpression, p16INK4a or E-cadherin mutation or E-cadherin methylation status, which implies that these miRNAs may contribute to the driver role of these genetic aberrations.  Addnl., miRNAs, which are located in a genomic region showing recurrent genetic aberrations, may themselves play a driver role in breast carcinogenesis or contribute to a driver gene in their vicinity.  In short, our study provides detailed mol. miRNA portraits of breast cancer cell lines, which can be exploited for functional studies of clin. important miRNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWhcz19FuvErVg90H21EOLACvtfcHk0liysbRmK1ED0Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGqtb%252FP&md5=fc80fbb51e25f6a80be909c9bbd4fc0c</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fbcr3415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr3415%26sid%3Dliteratum%253Aachs%26aulast%3DRiaz%26aufirst%3DM.%26aulast%3Dvan%2BJaarsveld%26aufirst%3DM.%2BT.%26aulast%3DHollestelle%26aufirst%3DA.%26aulast%3DPrager-van%2Bder%2BSmissen%26aufirst%3DW.%2BJ.%26aulast%3DHeine%26aufirst%3DA.%2BA.%26aulast%3DBoersma%26aufirst%3DA.%2BW.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHelmijr%26aufirst%3DJ.%26aulast%3DOzturk%26aufirst%3DB.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DWiemer%26aufirst%3DE.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%26atitle%3DmiRNA%2520expression%2520profiling%2520of%252051%2520human%2520breast%2520cancer%2520cell%2520lines%2520reveals%2520subtype%2520and%2520driver%2520mutation-specific%2520miRNAs%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2013%26volume%3D15%26spage%3DR33%26doi%3D10.1186%2Fbcr3415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollestelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elstrodt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasielewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozendaal, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koopman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ten Hagen, T. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leeuw, B. H. C. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwarthoff, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teunisse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spek, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinjens, W. N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochemsen, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Bakker, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutte, M.</span></span> <span> </span><span class="NLM_article-title">Distinct gene mutation profiles among luminal-type and basal-type breast cancer cell lines</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0460-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs10549-009-0460-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19593635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3c3mslemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2010&pages=53-64&author=A.+Hollestelleauthor=J.+H.+A.+Nagelauthor=M.+Smidauthor=S.+Lamauthor=F.+Elstrodtauthor=M.+Wasielewskiauthor=S.+S.+Ngauthor=P.+J.+Frenchauthor=J.+K.+Peetersauthor=M.+J.+Rozendaalauthor=M.+Riazauthor=D.+G.+Koopmanauthor=T.+L.+M.+Ten+Hagenauthor=B.+H.+C.+G.+M.+de+Leeuwauthor=E.+C.+Zwarthoffauthor=A.+Teunisseauthor=P.+J.+van+der+Spekauthor=J.+G.+M.+Klijnauthor=W.+N.+M.+Dinjensauthor=S.+P.+Ethierauthor=H.+Cleversauthor=A.+G.+Jochemsenauthor=M.+A.+den+Bakkerauthor=J.+A.+Foekensauthor=J.+W.+M.+Martensauthor=M.+Schutte&title=Distinct+gene+mutation+profiles+among+luminal-type+and+basal-type+breast+cancer+cell+lines&doi=10.1007%2Fs10549-009-0460-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct gene mutation profiles among luminal-type and basal-type breast cancer cell lines</span></div><div class="casAuthors">Hollestelle Antoinette; Nagel Jord H A; Smid Marcel; Lam Suzanne; Elstrodt Fons; Wasielewski Marijke; Ng Ser Sue; French Pim J; Peeters Justine K; Rozendaal Marieke J; Riaz Muhammad; Koopman Daphne G; Ten Hagen Timo L M; de Leeuw Bertie H C G M; Zwarthoff Ellen C; Teunisse Amina; van der Spek Peter J; Klijn Jan G M; Dinjens Winand N M; Ethier Stephen P; Clevers Hans; Jochemsen Aart G; den Bakker Michael A; Foekens John A; Martens John W M; Schutte Mieke</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer has for long been recognized as a highly diverse tumor group, but the underlying genetic basis has been elusive.  Here, we report an extensive molecular characterization of a collection of 41 human breast cancer cell lines.  Protein and gene expression analyses indicated that the collection of breast cancer cell lines has retained most, if not all, molecular characteristics that are typical for clinical breast cancers.  Gene mutation analyses identified 146 oncogenic mutations among 27 well-known cancer genes, amounting to an average of 3.6 mutations per cell line.  Mutations in genes from the p53, RB and PI3K tumor suppressor pathways were widespread among all breast cancer cell lines.  Most important, we have identified two gene mutation profiles that are specifically associated with luminal-type and basal-type breast cancer cell lines.  The luminal mutation profile involved E-cadherin and MAP2K4 gene mutations and amplifications of Cyclin D1, ERBB2 and HDM2, whereas the basal mutation profile involved BRCA1, RB1, RAS and BRAF gene mutations and deletions of p16 and p14ARF.  These subtype-specific gene mutation profiles constitute a genetic basis for the heterogeneity observed among human breast cancers, providing clues for their underlying biology and providing guidance for targeted pharmacogenetic intervention in breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyyljB3E0iTtenUz0lpNqnfW6udTcc2eY0hdE6kGpGErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3mslemtQ%253D%253D&md5=6b501ebd85d79898ec1fcce1608de5e0</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0460-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0460-8%26sid%3Dliteratum%253Aachs%26aulast%3DHollestelle%26aufirst%3DA.%26aulast%3DNagel%26aufirst%3DJ.%2BH.%2BA.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DLam%26aufirst%3DS.%26aulast%3DElstrodt%26aufirst%3DF.%26aulast%3DWasielewski%26aufirst%3DM.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DFrench%26aufirst%3DP.%2BJ.%26aulast%3DPeeters%26aufirst%3DJ.%2BK.%26aulast%3DRozendaal%26aufirst%3DM.%2BJ.%26aulast%3DRiaz%26aufirst%3DM.%26aulast%3DKoopman%26aufirst%3DD.%2BG.%26aulast%3DTen%2BHagen%26aufirst%3DT.%2BL.%2BM.%26aulast%3Dde%2BLeeuw%26aufirst%3DB.%2BH.%2BC.%2BG.%2BM.%26aulast%3DZwarthoff%26aufirst%3DE.%2BC.%26aulast%3DTeunisse%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BSpek%26aufirst%3DP.%2BJ.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%2BM.%26aulast%3DDinjens%26aufirst%3DW.%2BN.%2BM.%26aulast%3DEthier%26aufirst%3DS.%2BP.%26aulast%3DClevers%26aufirst%3DH.%26aulast%3DJochemsen%26aufirst%3DA.%2BG.%26aulast%3Dden%2BBakker%26aufirst%3DM.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DSchutte%26aufirst%3DM.%26atitle%3DDistinct%2520gene%2520mutation%2520profiles%2520among%2520luminal-type%2520and%2520basal-type%2520breast%2520cancer%2520cell%2520lines%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D121%26spage%3D53%26epage%3D64%26doi%3D10.1007%2Fs10549-009-0460-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocanegra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwei, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Boussard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular profiling of breast cancer cell lines defines relevant tumor models and provides a resource for cancer gene discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">e6146</span>– <span class="NLM_lpage">e6161</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0006146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0006146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19582160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1MvmvV2kug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6146-e6161&author=J.+Kaoauthor=K.+Salariauthor=M.+Bocanegraauthor=Y.-L.+Choiauthor=L.+Girardauthor=J.+Gandhiauthor=K.+A.+Kweiauthor=T.+Hernandez-Boussardauthor=P.+Wangauthor=A.+F.+Gazdarauthor=J.+D.+Minnaauthor=J.+R.+Pollack&title=Molecular+profiling+of+breast+cancer+cell+lines+defines+relevant+tumor+models+and+provides+a+resource+for+cancer+gene+discovery&doi=10.1371%2Fjournal.pone.0006146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular profiling of breast cancer cell lines defines relevant tumor models and provides a resource for cancer gene discovery</span></div><div class="casAuthors">Kao Jessica; Salari Keyan; Bocanegra Melanie; Choi Yoon-La; Girard Luc; Gandhi Jeet; Kwei Kevin A; Hernandez-Boussard Tina; Wang Pei; Gazdar Adi F; Minna John D; Pollack Jonathan R</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e6146</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Breast cancer cell lines have been used widely to investigate breast cancer pathobiology and new therapies.  Breast cancer is a molecularly heterogeneous disease, and it is important to understand how well and which cell lines best model that diversity.  In particular, microarray studies have identified molecular subtypes-luminal A, luminal B, ERBB2-associated, basal-like and normal-like-with characteristic gene-expression patterns and underlying DNA copy number alterations (CNAs).  Here, we studied a collection of breast cancer cell lines to catalog molecular profiles and to assess their relation to breast cancer subtypes.  METHODS:  Whole-genome DNA microarrays were used to profile gene expression and CNAs in a collection of 52 widely-used breast cancer cell lines, and comparisons were made to existing profiles of primary breast tumors.  Hierarchical clustering was used to identify gene-expression subtypes, and Gene Set Enrichment Analysis (GSEA) to discover biological features of those subtypes.  Genomic and transcriptional profiles were integrated to discover within high-amplitude CNAs candidate cancer genes with coordinately altered gene copy number and expression.  FINDINGS:  Transcriptional profiling of breast cancer cell lines identified one luminal and two basal-like (A and B) subtypes.  Luminal lines displayed an estrogen receptor (ER) signature and resembled luminal-A/B tumors, basal-A lines were associated with ETS-pathway and BRCA1 signatures and resembled basal-like tumors, and basal-B lines displayed mesenchymal and stem/progenitor-cell characteristics.  Compared to tumors, cell lines exhibited similar patterns of CNA, but an overall higher complexity of CNA (genetically simple luminal-A tumors were not represented), and only partial conservation of subtype-specific CNAs.  We identified 80 high-level DNA amplifications and 13 multi-copy deletions, and the resident genes with concomitantly altered gene-expression, highlighting known and novel candidate breast cancer genes.  CONCLUSIONS:  Overall, breast cancer cell lines were genetically more complex than tumors, but retained expression patterns with relevance to the luminal-basal subtype distinction.  The compendium of molecular profiles defines cell lines suitable for investigations of subtype-specific pathobiology, cancer stem cell biology, biomarkers and therapies, and provides a resource for discovery of new breast cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ64pI1FRXP5q__Jf7-eMvJfW6udTcc2eY0hdE6kGpGErntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvmvV2kug%253D%253D&md5=4fb2fa78b74a7270fbd98aa491bca206</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006146%26sid%3Dliteratum%253Aachs%26aulast%3DKao%26aufirst%3DJ.%26aulast%3DSalari%26aufirst%3DK.%26aulast%3DBocanegra%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DY.-L.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DGandhi%26aufirst%3DJ.%26aulast%3DKwei%26aufirst%3DK.%2BA.%26aulast%3DHernandez-Boussard%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520profiling%2520of%2520breast%2520cancer%2520cell%2520lines%2520defines%2520relevant%2520tumor%2520models%2520and%2520provides%2520a%2520resource%2520for%2520cancer%2520gene%2520discovery%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De6146%26epage%3De6161%26doi%3D10.1371%2Fjournal.pone.0006146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anido, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzmán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arribas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Averbuch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">ZD1839, a specific epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor, induces the formation of inactive EGFR/HER2 and EGFR/HER3 heterodimers and prevents heregulin signaling in HER2-overexpressing breast cancer cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1274</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12684395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXislertrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=1274&author=J.+Anidoauthor=P.+Matarauthor=J.+Albanellauthor=M.+Guzm%C3%A1nauthor=F.+Rojoauthor=J.+Arribasauthor=S.+Averbuchauthor=J.+Baselga&title=ZD1839%2C+a+specific+epidermal+growth+factor+receptor+%28EGFR%29+tyrosine+kinase+inhibitor%2C+induces+the+formation+of+inactive+EGFR%2FHER2+and+EGFR%2FHER3+heterodimers+and+prevents+heregulin+signaling+in+HER2-overexpressing+breast+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">ZD1839, a Specific Epidermal Growth Factor Receptor (EGFR) Tyrosine Kinase Inhibitor, Induces the Formation of Inactive EGFR/HER2 and EGFR/HER3 Heterodimers and Prevents Heregulin Signaling in HER2-overexpressing Breast Cancer Cells</span></div><div class="casAuthors">Anido, Judit; Matar, Pablo; Albanell, Joan; Guzman, Marta; Rojo, Federico; Arribas, Joaquin; Averbuch, Steve; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1274-1283</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ZD1839 is a tyrosine kinase inhibitor of the epidermal growth factor receptor (EGFR) that has shown clin. activity against EGFR-expressing tumors.  Our aim was to explore the effects of ZD1839 in breast cancer cell lines expressing different levels of EGFR and the closely related HER2 receptor.  We studied the growth-inhibitory effects of ZD1839 in a series of breast carcinoma cell lines.  In HER2-overexpressing BT-474 breast cancer cells, we studied the effects of ZD1839 on cell growth and heterodimerization of receptors under basal and ligand-stimulated conditions.  ZD1839 was an equally potent inhibitor of growth in breast cancer cells expressing high levels of EGFR and HER2.  In BT-474 breast cancer cells, ZD1839 abolished EGF- and heregulin-induced activation of ErbB receptors and downstream signaling mols.  Because ZD1839 does not inhibit the HER2 tyrosine kinase in vitro, and because heregulin is a ligand that activates HER2 by binding to HER3 and HER4 but does not bind to the EGFR, our findings suggested that ZD1839 interfered with HER2 function in intact cells.  Searching for mechanisms, we report that ZD1839 induces the formation of inactive unphosphorylated EGFR/HER2 and EGFR/HER3 heterodimers.  Furthermore, ZD1839 completely abolishes basal and heregulin-induced formation of active phosphorylated HER2/HER3 heterodimers.  ZD1839 inhibits the growth of HER2-overexpressing breast cancer cells, possibly by sequestration of HER2 and HER3 receptors in an inactive heterodimer configuration with the EGFR.  Our findings suggest that there is a strong rationale to conduct clin. trials of ZD1839 in patients with HER2-overexpressing breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA7DnoL5ZfL7Vg90H21EOLACvtfcHk0lhXbuwD9QA6UQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXislertrs%253D&md5=a966a0569f5d9ff89e61b449c9bea1cd</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnido%26aufirst%3DJ.%26aulast%3DMatar%26aufirst%3DP.%26aulast%3DAlbanell%26aufirst%3DJ.%26aulast%3DGuzm%25C3%25A1n%26aufirst%3DM.%26aulast%3DRojo%26aufirst%3DF.%26aulast%3DArribas%26aufirst%3DJ.%26aulast%3DAverbuch%26aufirst%3DS.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DZD1839%252C%2520a%2520specific%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520tyrosine%2520kinase%2520inhibitor%252C%2520induces%2520the%2520formation%2520of%2520inactive%2520EGFR%252FHER2%2520and%2520EGFR%252FHER3%2520heterodimers%2520and%2520prevents%2520heregulin%2520signaling%2520in%2520HER2-overexpressing%2520breast%2520cancer%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Going, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. M.</span></span> <span> </span><span class="NLM_article-title">Expression of the HER1-4 family of receptor tyrosine kinases in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>200</i></span>,  <span class="NLM_fpage">290</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1002/path.1370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fpath.1370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12845624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvF2gu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=200&publication_year=2003&pages=290-297&author=C.+J.+Wittonauthor=J.+R.+Reevesauthor=J.+J.+Goingauthor=T.+G.+Cookeauthor=J.+M.+Bartlett&title=Expression+of+the+HER1-4+family+of+receptor+tyrosine+kinases+in+breast+cancer&doi=10.1002%2Fpath.1370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the HER1-4 family of receptor tyrosine kinases in breast cancer</span></div><div class="casAuthors">Witton, Caroline J.; Reeves, Jonathan R.; Going, James J.; Cooke, Timothy G.; Bartlett, John M. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">200</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">290-297</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">EGFr/HER1 and c-erbB-2/HER2 expression are assocd. with poor prognosis in breast cancer.  The type I receptor tyrosine kinase (RTK) family to which they belong has four members (HER1-4).  In this study, expression of HER1-4 and estrogen receptor (ER) expression were detd. by immunohistochem. in 220 breast carcinomas.  Elevated expression of HER1 was obsd. in 16.4%, HER2 in 22.8%, HER3 in 17.5%, and HER4 in 11.9% of these tumors.  Patients whose tumors overexpressed HER1, 2 or 3 had reduced survival (p =< 0.001), whereas those whose tumors overexpressed HER4 had increased survival (p = 0.013); 38.6% of cases overexpressed one or more of HER1, 2 or 3.  HER4 was rarely overexpressed with other HERs (1.4% of cases).  Cox's multiple regression anal. demonstrated that overexpression of HER1/2/3, HER4, and std. prognostic indicators independently affected survival.  HER1-3 expression was related to ER negativity (p < 0.0001, ηhiX2).  Patients with ER-pos., HER1-3-pos. tumors had a significantly poorer survival (p < 0.001) than those with ER-pos./HER-neg. or HER4-pos. tumors.  Expression of HER RTKs displays complex interactions between different family members.  There is a strong interaction, in terms of survival, between HER expression and ER expression.  The development of HER-targeted agents (eg Herceptin, Iressa), and agents targeted at the downstream signalling pathways, therefore provides new possibilities in the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgp4ubXdx7ibVg90H21EOLACvtfcHk0lgTlFhQTnbz5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvF2gu7c%253D&md5=0b81cfc4bca4857f5380e2ba7635f75d</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1002%2Fpath.1370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.1370%26sid%3Dliteratum%253Aachs%26aulast%3DWitton%26aufirst%3DC.%2BJ.%26aulast%3DReeves%26aufirst%3DJ.%2BR.%26aulast%3DGoing%26aufirst%3DJ.%2BJ.%26aulast%3DCooke%26aufirst%3DT.%2BG.%26aulast%3DBartlett%26aufirst%3DJ.%2BM.%26atitle%3DExpression%2520of%2520the%2520HER1-4%2520family%2520of%2520receptor%2520tyrosine%2520kinases%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2003%26volume%3D200%26spage%3D290%26epage%3D297%26doi%3D10.1002%2Fpath.1370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsutsui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachitanda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, S.</span></span> <span> </span><span class="NLM_article-title">Prognostic value of epidermal growth factor receptor (EGFR) and its relationship to the estrogen receptor status in 1029 patients with breast cancer</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1023/a:1013397232011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1023%2Fa%3A1013397232011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11859875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD387isFCisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2002&pages=67-75&author=S.+Tsutsuiauthor=S.+Ohnoauthor=S.+Murakamiauthor=Y.+Hachitandaauthor=S.+Oda&title=Prognostic+value+of+epidermal+growth+factor+receptor+%28EGFR%29+and+its+relationship+to+the+estrogen+receptor+status+in+1029+patients+with+breast+cancer&doi=10.1023%2Fa%3A1013397232011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic value of epidermal growth factor receptor (EGFR) and its relationship to the estrogen receptor status in 1029 patients with breast cancer</span></div><div class="casAuthors">Tsutsui Shinichi; Ohno Shinji; Murakami Shigeru; Hachitanda Yoichi; Oda Shinya</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    </div><div class="casAbstract">An epidermal growth factor receptor (EGFR) has been reported to be associated with a poor clinical outcome in breast cancer, while its prognostic value remains controversial.  Immunohistochemical staining for EGFR was performed on frozen sections of primary breast cancer from 1029 patients with a mean follow-up duration of 46 months.  EGFR was positive in 277 (26.9%) of 1029 cases which inversely correlated with the estrogen receptor (ER) status.  A univariated analysis indicated that EGFR had a significant prognostic value in both the disease free survival (DFS) and the overall survival (OS), while the same effect was also found in node negative as well as node positive breast cancer.  A multivariate analysis indicated that EGFR was an independently significant prognostic factor for DFS (p = 0.0174) and OS (p = 0.0105) in all patients, but that EGFR demonstrated a prognostic significance only for DFS (p = 0.0241) in node negative and only for OS (p = 0.0333) in node positive breast cancer.  When all patients were stratified for EGFR and ER, a multivariate analysis indicated that the combination of EGFR(+)/ER(-) was an independently significant factor for both DFS and OS in node negative as well as node positive breast cancer.  In conclusion, the prognostic value of EGFR was demonstrated by a multivariate analysis in a large series of breast cancer patients, but the value of EGFR was somewhat insufficient to achieve statistical significance for both DFS and OS in the subgroups divided by nodal status.  On the other hand, the prognostic value of combination of EGFR and ER was sufficient to achieve statistical significance based on a multivariate analysis for both DFS and OS in the subgroups of node negative as well as node positive breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkZRi_NrcOMuIHSmpohpd0fW6udTcc2ebQINzS48iwa7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD387isFCisw%253D%253D&md5=52839385eb2e92b9598baea7c23d3b5b</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1023%2Fa%3A1013397232011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fa%253A1013397232011%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsui%26aufirst%3DS.%26aulast%3DOhno%26aufirst%3DS.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DHachitanda%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DS.%26atitle%3DPrognostic%2520value%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520its%2520relationship%2520to%2520the%2520estrogen%2520receptor%2520status%2520in%25201029%2520patients%2520with%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2002%26volume%3D71%26spage%3D67%26epage%3D75%26doi%3D10.1023%2Fa%3A1013397232011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sayed, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span> <span> </span><span class="NLM_article-title">Prognostic markers in triple-negative breast cancer</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1002/cncr.22381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcncr.22381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17146782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=25-32&author=E.+A.+Rakhaauthor=M.+E.+El-Sayedauthor=A.+R.+Greenauthor=A.+H.+S.+Leeauthor=J.+F.+Robertsonauthor=I.+O.+Ellis&title=Prognostic+markers+in+triple-negative+breast+cancer&doi=10.1002%2Fcncr.22381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic markers in triple-negative breast cancer</span></div><div class="casAuthors">Rakha, Emad A.; El-Sayed, Maysa E.; Green, Andrew R.; Lee, Andrew H. S.; Robertson, John F.; Ellis, Ian O.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">BACKGROUND. Triple-neg. breast cancer (estrogen receptor-neg., progesterone receptor-neg., and HER2-neg.) is a high risk breast cancer that lacks the benefit of specific therapy that targets these proteins.  METHODS. In this study, the authors examd. a large and well characterized series of invasive breast carcinoma (n = 1944) with a long-term clin. follow-up (median, 56 mo) by using tissue microarray.  The series were also stained with concurrent immunohistochem. prognostic panels (estrogen receptor, progesterone receptor, HER-2, androgen receptor, epidermal growth factor receptor (EGFR), P-cadherin, E-cadherin, and basal (CK5/6, CK14), and p53), to characterize this specific subgroup of breast cancer and to identify prognostic markers that can identify tumors with more aggressive behavior.  RESULTS. Of informative cases, 16.3% were of the triple-neg. phenotype.  The majority of these tumors were grade 3, ductal/no-specific-type carcinomas.  There were pos. assocns. with larger size, pushing margins, poorer Nottingham Prognostic Index, development of recurrence and distant metastasis, and poorer outcome.  In addn., assocns. were found with loss of expression of androgen receptor and E-cadherin, and pos. expression of basal cytokeratins (basal phenotype), P-cadherin, p53, and EGFR.  In all tumors, tumor size, lymph node stage, and androgen receptor were the most useful prognostic markers.  In the lymph node-pos. subgroup, both size and androgen receptor retained their prognostic significance.  However, in the lymph node-neg. tumors, basal phenotype was the sole prognostic marker identified in this subgroup.  Other parameters including age, histol. grade, tumor size, vascular invasion or other biomarkers included in the current study were not significant.  CONCLUSIONS. The authors concluded that assessment of androgen receptor and basal phenotype, in addn. to the established pathol. variables, mainly lymph node status and tumor size, can be used to select high-risk and low-risk patients at the time of primary surgery and can provide valuable information on treatment options in these triple-neg. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQnQBrj7p6jLVg90H21EOLACvtfcHk0lgTlFhQTnbz5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKrtLY%253D&md5=fddf1e152783977dd1b7d525b7435ac7</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1002%2Fcncr.22381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.22381%26sid%3Dliteratum%253Aachs%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DEl-Sayed%26aufirst%3DM.%2BE.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DLee%26aufirst%3DA.%2BH.%2BS.%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26atitle%3DPrognostic%2520markers%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%26date%3D2007%26volume%3D109%26spage%3D25%26epage%3D32%26doi%3D10.1002%2Fcncr.22381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarthy, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shyr, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietenpol, J. A.</span></span> <span> </span><span class="NLM_article-title">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">2750</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.1172/jci45014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1172%2Fjci45014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21633166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVKgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2011&pages=2750-2767&author=B.+D.+Lehmannauthor=J.+A.+Bauerauthor=X.+Chenauthor=M.+E.+Sandersauthor=A.+B.+Chakravarthyauthor=Y.+Shyrauthor=J.+A.+Pietenpol&title=Identification+of+human+triple-negative+breast+cancer+subtypes+and+preclinical+models+for+selection+of+targeted+therapies&doi=10.1172%2Fjci45014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies</span></div><div class="casAuthors">Lehmann, Brian D.; Bauer, Joshua A.; Chen, Xi; Sanders, Melinda E.; Chakravarthy, A. Bapsi; Shyr, Yu; Pietenpol, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2750-2767</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a highly diverse group of cancers, and subtyping is necessary to better identify mol.-based therapies.  In this study, we analyzed gene expression (GE) profiles from 21 breast cancer data sets and identified 587 TNBC cases.  Cluster anal. identified 6 TNBC subtypes displaying unique GE and ontologies, including 2 basal-like (BL1 and BL2), an immunomodulatory (IM), a mesenchymal (M), a mesenchymal stem-like (MSL), and a luminal androgen receptor (LAR) subtype.  Further, GE anal. allowed us to identify TNBC cell line models representative of these subtypes.  Predicted "driver" signaling pathways were pharmacol. targeted in these cell line models as proof of concept that anal. of distinct GE signatures can inform therapy selection.  BL1 and BL2 subtypes had higher expression of cell cycle and DNA damage response genes, and representative cell lines preferentially responded to cisplatin.  M and MSL subtypes were enriched in GE for epithelial-mesenchymal transition, and growth factor pathways and cell models responded to NVP-BEZ235 (a PI3K/mTOR inhibitor) and dasatinib (an abl/src inhibitor).  The LAR subtype includes patients with decreased relapse-free survival and was characterized by androgen receptor (AR) signaling.  LAR cell lines were uniquely sensitive to bicalutamide (an AR antagonist).  These data may be useful in biomarker selection, drug discovery, and clin. trial design that will enable alignment of TNBC patients to appropriate targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbhoQQUFogW7Vg90H21EOLACvtfcHk0lgA7Usfdl30yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVKgu78%253D&md5=c3681c79e0b972f16f6d5a5ecdc99fec</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1172%2Fjci45014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci45014%26sid%3Dliteratum%253Aachs%26aulast%3DLehmann%26aufirst%3DB.%2BD.%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSanders%26aufirst%3DM.%2BE.%26aulast%3DChakravarthy%26aufirst%3DA.%2BB.%26aulast%3DShyr%26aufirst%3DY.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26atitle%3DIdentification%2520of%2520human%2520triple-negative%2520breast%2520cancer%2520subtypes%2520and%2520preclinical%2520models%2520for%2520selection%2520of%2520targeted%2520therapies%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2011%26volume%3D121%26spage%3D2750%26epage%3D2767%26doi%3D10.1172%2Fjci45014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeusz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, N. T.</span></span> <span> </span><span class="NLM_article-title">Role of epidermal growth factor receptor in breast cancer</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1007/s10549-012-2289-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs10549-012-2289-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23073759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2qt7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2012&pages=331-345&author=H.+Masudaauthor=D.+Zhangauthor=C.+Bartholomeuszauthor=H.+Doiharaauthor=G.+N.+Hortobagyiauthor=N.+T.+Ueno&title=Role+of+epidermal+growth+factor+receptor+in+breast+cancer&doi=10.1007%2Fs10549-012-2289-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Role of epidermal growth factor receptor in breast cancer</span></div><div class="casAuthors">Masuda, Hiroko; Zhang, Dongwei; Bartholomeusz, Chandra; Doihara, Hiroyoshi; Hortobagyi, Gabriel N.; Ueno, Naoto T.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-345</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Decades of research in mol. oncol. have brought about promising new therapies which are designed to target specific mols. which promote tumor growth and survival.  The epidermal growth factor receptor (EGFR) is one of the first identified important targets of these novel antitumor agents.  Approx. half of cases of triple-neg. breast cancer (TNBC) and inflammatory breast cancer (IBC) overexpress EGFR.  Thus, EGFR inhibitors for treatment of breast cancer have been evaluated in several studies.  However, results so far have been disappointing.  One of the reasons for these unexpected results is the lack of biomarkers for predicting which patients are most likely to respond to EGFR inhibitors.  Recent studies have shown that EGFR and its downstream pathway regulate epithelial-mesenchymal transition, migration, and tumor invasion and that high EGFR expression is an independent predictor of poor prognosis in IBC.  Further, recent studies have shown that targeting EGFR enhances the chemosensitivity of TNBC cells by rewiring apoptotic signaling networks in TNBC.  These studies indicate that EGFR-targeted therapy might have a promising role in TNBC and IBC.  Further studies of the role of EGFR in TNBC and IBC are needed to better understand the best way to use EGFR-targeted therapy-e.g., as a chemosensitizer or to prevent metastases-to treat these aggressive diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG80rBn6V0ubVg90H21EOLACvtfcHk0lgA7Usfdl30yg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2qt7vN&md5=61a75d8b05c3c0717a6959c1d047990f</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10549-012-2289-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-012-2289-9%26sid%3Dliteratum%253Aachs%26aulast%3DMasuda%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DBartholomeusz%26aufirst%3DC.%26aulast%3DDoihara%26aufirst%3DH.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DUeno%26aufirst%3DN.%2BT.%26atitle%3DRole%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D136%26spage%3D331%26epage%3D345%26doi%3D10.1007%2Fs10549-012-2289-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garimella, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipkowitz, S.</span></span> <span> </span><span class="NLM_article-title">Triple negative breast cancer cell lines: one tool in the search for better treatment of triple negative breast cancer</span>. <i>Breast Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.3233/BD-2010-0307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3233%2FBD-2010-0307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21778573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3Mbls1Sjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=35-48&author=K.+J.+Chavezauthor=S.+V.+Garimellaauthor=S.+Lipkowitz&title=Triple+negative+breast+cancer+cell+lines%3A+one+tool+in+the+search+for+better+treatment+of+triple+negative+breast+cancer&doi=10.3233%2FBD-2010-0307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Triple negative breast cancer cell lines: one tool in the search for better treatment of triple negative breast cancer</span></div><div class="casAuthors">Chavez Kathryn J; Garimella Sireesha V; Lipkowitz Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Breast disease</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">35-48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSP84JmqLJuYJVVjGoY74BafW6udTcc2eZ3XWMJJmlJSbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mbls1Sjtg%253D%253D&md5=bb3b4f1ddd945c1bd8e79c41fa44b592</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.3233%2FBD-2010-0307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FBD-2010-0307%26sid%3Dliteratum%253Aachs%26aulast%3DChavez%26aufirst%3DK.%2BJ.%26aulast%3DGarimella%26aufirst%3DS.%2BV.%26aulast%3DLipkowitz%26aufirst%3DS.%26atitle%3DTriple%2520negative%2520breast%2520cancer%2520cell%2520lines%253A%2520one%2520tool%2520in%2520the%2520search%2520for%2520better%2520treatment%2520of%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DBreast%2520Dis.%26date%3D2010%26volume%3D32%26spage%3D35%26epage%3D48%26doi%3D10.3233%2FBD-2010-0307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and in vitro anti-proliferative activity of 4,6-quinazolinediamines as potent EGFR-TK inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2012.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23142066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Kjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=132-145&author=S.+Mowafyauthor=N.+A.+Faragauthor=K.+A.+M.+Abouzid&title=Design%2C+synthesis+and+in+vitro+anti-proliferative+activity+of+4%2C6-quinazolinediamines+as+potent+EGFR-TK+inhibitors&doi=10.1016%2Fj.ejmech.2012.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and in vitro anti-proliferative activity of 4,6-quinazolinediamines as potent EGFR-TK inhibitors</span></div><div class="casAuthors">Mowafy, Samar; Farag, Nahla A.; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">132-145</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">4-Anilino-6-substituted-quinazolines were designed, synthesized and evaluated for EGFR-TK and tumor growth inhibitory activities.  The target compds. were designed with enamine ester or urea moieties appended at the C-6 of quinazoline as addnl. hydrogen bond acceptor functions.  Most of the synthesized compds. displayed potent EGFR-TK inhibitory activity at 10 μM and the 6-ureido-anilinoquinazoline deriv. (I) showed IC50 value of 0.061 μM.  Moreover, six compds. were tested by National Cancer Institute (NCI), USA for their anti-proliferative activity at 10 μM in full NCI 60 cell panel.  Compd. I was further assayed for five dose molar ranges in full NCI 60 cell panel and exhibited remarkable growth inhibitory activity pattern against Non-Small Cell Lung Cancer EKVX (GI50 = 0.37 μM), NCI-H322M (GI50 = 0.36 μM), Renal Cancer A498 (GI50 = 0.46 μM), TK-10 (GI50 = 0.99 μM) and Breast Cancer MDA-MB-468 (GI50 = 1.096 μM) which are of high EGFR expression.  Docking study was performed for the active compds. into ATP binding site of EGFR-TK which showed similar binding mode to gefitinib and addnl. binding with Cys-773 at the gatekeeper of EGFR-TK enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ6Ian6C596LVg90H21EOLACvtfcHk0liVFuxFmJm2tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Kjsb3E&md5=e74abcf0e7ff68497f022ac4e1400ab1</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DFarag%26aufirst%3DN.%2BA.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520in%2520vitro%2520anti-proliferative%2520activity%2520of%25204%252C6-quinazolinediamines%2520as%2520potent%2520EGFR-TK%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D61%26spage%3D132%26epage%3D145%26doi%3D10.1016%2Fj.ejmech.2012.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eustace, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, M. S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shaughnessy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Driscoll, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slamon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J.</span></span> <span> </span><span class="NLM_article-title">Development of acquired resistance to lapatinib may sensitise HER2-positive breast cancer cells to apoptosis induction by obatoclax and TRAIL</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">965</span>, <span class="refDoi"> DOI: 10.1186/s12885-018-4852-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fs12885-018-4852-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30305055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yls7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=965&author=A.+J.+Eustaceauthor=N.+T.+Conlonauthor=M.+S.+J.+McDermottauthor=B.+C.+Browneauthor=P.+O%E2%80%99Learyauthor=F.+A.+Holmesauthor=V.+Espinaauthor=L.+A.+Liottaauthor=J.+O%E2%80%99Shaughnessyauthor=C.+Gallagherauthor=L.+O%E2%80%99Driscollauthor=S.+Raniauthor=S.+F.+Maddenauthor=N.+A.+O%E2%80%99Brienauthor=C.+Gintherauthor=D.+Slamonauthor=N.+Walshauthor=W.+M.+Gallagherauthor=R.+Zagozdzonauthor=W.+R.+Watsonauthor=N.+O%E2%80%99Donovanauthor=J.+Crown&title=Development+of+acquired+resistance+to+lapatinib+may+sensitise+HER2-positive+breast+cancer+cells+to+apoptosis+induction+by+obatoclax+and+TRAIL&doi=10.1186%2Fs12885-018-4852-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Development of acquired resistance to lapatinib may sensitise HER2-positive breast cancer cells to apoptosis induction by obatoclax and TRAIL</span></div><div class="casAuthors">Eustace, Alex J.; Conlon, Neil T.; Mcdermott, Martina S. J.; Browne, Brigid C.; O'leary, Patrick; Holmes, Frankie A.; Espina, Virginia; Liotta, Lance A.; O'shaughnessy, Joyce; Gallagher, Clair; O'driscoll, Lorraine; Rani, Sweta; Madden, Stephen F.; O'brien, Neil A.; Ginther, Charles; Slamon, Dennis; Walsh, Naomi; Gallagher, William M.; Zagozdzon, Radoslaw; Watson, William R.; O'donovan, Norma; Crown, John</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">965</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Lapatinib has clin. efficacy in the treatment of trastuzumab-refractory HER2-pos. breast cancer.  However, a significant proportion of patients develop progressive disease due to acquired resistance to the drug.  Induction of apoptotic cell death is a key mechanism of action of lapatinib in HER2-pos. breast cancer cells.  Methods: We examd. alterations in regulation of the intrinsic and extrinsic apoptosis pathways in cell line models of acquired lapatinib resistance both in vitro and in patient samples from the NCT01485926 clin. trial, and investigated potential strategies to exploit alterations in apoptosis signalling to overcome lapatinib resistance in HER2-pos. breast cancer.  Results: In this study, we examd. two cell lines models of acquired lapatinib resistance (SKBR3-L and HCC1954-L) and showed that lapatinib does not induce apoptosis in these cells.  We identified alterations in members of the BCL-2 family of proteins, in particular MCL-1 and BAX, which may play a role in resistance to lapatinib.  We tested the therapeutic inhibitor obatoclax, which targets MCL-1.  Both SKBR3-L and HCC1954-L cells showed greater sensitivity to obatoclax-induced apoptosis than parental cells.  Interestingly, we also found that the development of acquired resistance to lapatinib resulted in acquired sensitivity to TRAIL in SKBR3-L cells.  Sensitivity to TRAIL in the SKBR3-L cells was assocd. with reduced phosphorylation of AKT, increased expression of FOXO3a and decreased expression of c- FLIP.  In SKBR3-L cells, TRAIL treatment caused activation of caspase 8, caspase 9 and caspase 3/7.  In a second resistant model, HCC1954-L cells, p-AKT levels were not decreased and these cells did not show enhanced sensitivity to TRAIL.  Furthermore, combining obatoclax with TRAIL improved response in SKBR3-L cells but not in HCC1954-L cells.  Conclusions: Our findings highlight the possibility of targeting altered apoptotic signalling to overcome acquired lapatinib resistance, and identify potential novel treatment strategies, with potential biomarkers, for HER2-pos. breast cancer that is resistant to HER2 targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp12PNOAxrnBbVg90H21EOLACvtfcHk0liVFuxFmJm2tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yls7vF&md5=12a66b8484bb13a8db7a58a5b043170f</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1186%2Fs12885-018-4852-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-018-4852-1%26sid%3Dliteratum%253Aachs%26aulast%3DEustace%26aufirst%3DA.%2BJ.%26aulast%3DConlon%26aufirst%3DN.%2BT.%26aulast%3DMcDermott%26aufirst%3DM.%2BS.%2BJ.%26aulast%3DBrowne%26aufirst%3DB.%2BC.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DP.%26aulast%3DHolmes%26aufirst%3DF.%2BA.%26aulast%3DEspina%26aufirst%3DV.%26aulast%3DLiotta%26aufirst%3DL.%2BA.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DJ.%26aulast%3DGallagher%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Driscoll%26aufirst%3DL.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DMadden%26aufirst%3DS.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DN.%2BA.%26aulast%3DGinther%26aufirst%3DC.%26aulast%3DSlamon%26aufirst%3DD.%26aulast%3DWalsh%26aufirst%3DN.%26aulast%3DGallagher%26aufirst%3DW.%2BM.%26aulast%3DZagozdzon%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DW.%2BR.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DN.%26aulast%3DCrown%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520acquired%2520resistance%2520to%2520lapatinib%2520may%2520sensitise%2520HER2-positive%2520breast%2520cancer%2520cells%2520to%2520apoptosis%2520induction%2520by%2520obatoclax%2520and%2520TRAIL%26jtitle%3DBMC%2520Cancer%26date%3D2018%26volume%3D18%26spage%3D965%26doi%3D10.1186%2Fs12885-018-4852-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span> <span> </span><span class="NLM_article-title">Lapatinib induces p27(Kip1)-dependent G(1) arrest through both transcriptional and post-translational mechanisms</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2665</span>– <span class="NLM_lpage">2674</span>, <span class="refDoi"> DOI: 10.4161/cc.25728</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.4161%2Fcc.25728" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23907131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Sgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=2665-2674&author=L.+Tangauthor=Y.+Wangauthor=A.+Stromauthor=J.-%C3%85.+Gustafssonauthor=X.+Guan&title=Lapatinib+induces+p27%28Kip1%29-dependent+G%281%29+arrest+through+both+transcriptional+and+post-translational+mechanisms&doi=10.4161%2Fcc.25728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Lapatinib induces p27Kip1-dependent G1 arrest through both transcriptional and post-translational mechanisms</span></div><div class="casAuthors">Tang, Lin; Wang, Yucai; Strom, Anders; Gustafsson, Jan-Ake; Guan, Xiaoxiang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2665-2674</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Lapatinib, a dual EGFR/HER2 tyrosine kinase inhibitor, has been shown to have potent antitumor effects against human breast cancer.  Recent studies have shown that lapatinib upregulates p27Kip1 (here after referred to as p27) expression and induces G1 cell cycle arrest in various types of cancer cells.  However, the regulation of p27 in lapatinib-induced cell cycle arrest is not well studied.  Here we demonstrate that lapatinib-induced cell growth inhibition and G1 cell cycle arrest in HER2-overexpressing human breast cancer cells were dependent on p27.  We also show that lapatinib-induced upregulation of p27 expression was mediated through both transcriptional and post-translational mechanisms.  On the one hand, lapatinib treatment led to increased FOXO3a expression and enhanced p27 transcription.  On the other hand, lapatinib treatment resulted in increased DYRK1B expression, which correlated with increased p27 phosphorylation at Ser10 and decreased p27 degrdn.  Interestingly, we found that ERβ1 but not ERβ2 expression also upregulated p27 and enhanced lapatinib-induced cell proliferation inhibition and G1 cell cycle arrest in HER2-overexpressing breast cancer cells.  Taken together, our results suggest that lapatinib induces p27 expression via both transcriptional and post-translational upregulations, leading to cell cycle arrest and cell proliferation inhibition, and that its effect on breast cancer cells may be modified by ER expression status.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqFpeD5i8Xl7Vg90H21EOLACvtfcHk0liVFuxFmJm2tA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Sgu7w%253D&md5=938c4430a2cbe7b05f874a7cd49c3e63</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.4161%2Fcc.25728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.25728%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DStrom%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DGuan%26aufirst%3DX.%26atitle%3DLapatinib%2520induces%2520p27%2528Kip1%2529-dependent%2520G%25281%2529%2520arrest%2520through%2520both%2520transcriptional%2520and%2520post-translational%2520mechanisms%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D2665%26epage%3D2674%26doi%3D10.4161%2Fcc.25728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lj5hnxySqWoaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span> <span> </span><span class="NLM_article-title">Promotion of apoptosis does not necessarily mean inhibition of remodeling</span>. <i>Hypertension</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">e3</span>, <span class="refDoi"> DOI: 10.1161/hypertensionaha.112.195784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1161%2Fhypertensionaha.112.195784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22689742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVerur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=e3&author=Q.+Shiauthor=L.+Haoauthor=J.+Peiauthor=W.+Yin&title=Promotion+of+apoptosis+does+not+necessarily+mean+inhibition+of+remodeling&doi=10.1161%2Fhypertensionaha.112.195784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Promotion of apoptosis does not necessarily mean inhibition of remodeling. Comments</span></div><div class="casAuthors">Shi, Quanxing; Hao, Lu; Pei, Jianming; Yin, Wen</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e3</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A polemic in response to Guo et al. [Hypertension. 2012;59:1006-1013] is presented.  Guo et al. concluded that miR-328 is downregulated in pulmonary arteries from hypoxic rats and patients with pulmonary hypertension, affecting hypoxic pulmonary vasoconstriction and pulmonary vascular remodeling by targeting Cav 1.2 and insulin-like growth factor 1 receptor, leading to hypoxic pulmonary hypertension.  However, Shi et al. argue that, from the data presented, the Guo et al. claimed only that miR-328 induced pulmonary artery smooth muscle cell (PASMC) apoptosis through repression of insulin-like growth factor 1 receptor.  Shi et al. reasoned out that promoting PASMC apoptosis does not necessarily mean inhibiting pulmonary vascular remodeling and that the effects of miR-328 on pulmonary arteries remodeling and PASMC apoptosis may be 2 accompanying phenomena.  Shi et al. also raised an issue that MTT assay is not appropriate to analyze apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo--jmoKHwG47Vg90H21EOLACvtfcHk0lj5hnxySqWoaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVerur3E&md5=ab80d47cb67b837fc83e1b2a342df7a0</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1161%2Fhypertensionaha.112.195784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252Fhypertensionaha.112.195784%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DHao%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DW.%26atitle%3DPromotion%2520of%2520apoptosis%2520does%2520not%2520necessarily%2520mean%2520inhibition%2520of%2520remodeling%26jtitle%3DHypertension%26date%3D2012%26volume%3D60%26spage%3De3%26doi%3D10.1161%2Fhypertensionaha.112.195784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Press, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, H.-J.</span></span> <span> </span><span class="NLM_article-title">EGFR, HER2 and VEGF pathways: validated targets for cancer treatment</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.2165/00003495-200767140-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2165%2F00003495-200767140-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17883287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtleqsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2045-2075&author=M.+F.+Pressauthor=H.-J.+Lenz&title=EGFR%2C+HER2+and+VEGF+pathways%3A+validated+targets+for+cancer+treatment&doi=10.2165%2F00003495-200767140-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR, HER2 and VEGF pathways: validated targets for cancer treatment</span></div><div class="casAuthors">Press, Michael F.; Lenz, Heinz-Josef</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2045-2075</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Wolters Kluwer Health</span>)
        </div><div class="casAbstract">A review.  Targeted therapies are rationally designed to interfere with specific mol. events that are important in tumor growth, progression or survival.  Several targeted therapies with anti-tumor activity in human cancer cell lines and xenograft models have now been shown to produce objective responses, delay disease progression and, in some cases, improve survival of patients with advanced malignancies.  These targeted therapies include cetuximab, an anti-epidermal growth factor receptor (EGFR) monoclonal antibody; gefitinib and erlotinib, EGFR-specific tyrosine kinase inhibitors; trastuzumab, an anti-human EGFR type 2 (HER2)-related monoclonal antibody; lapatinib, a dual inhibitor of both EGFR- and HER2-assocd. tyrosine kinases; and bevacizumab, an anti-vascular endothelial growth factor (VEGF) monoclonal antibody.  On the basis of preclin. and clin. evidence, EGFR, HER2 and VEGF represent validated targets for cancer therapy and remain the subject of intensive investigation.  Both EGFR and HER2 are targets found on cancer cells, whereas VEGF is a target that acts in the tumor microenvironment.  Clin. studies are focusing on how to best incorporate targeted therapy into current treatment regimens and other studies are exploring whether different strategies for inhibiting these targets will offer greater benefit.  It is clear that optimal use of targeted therapy will depend on understanding how these drugs work mechanistically, and recognising that their activities may differ across patient populations, tumor types and disease stages, as well as when and how they are used in cancer treatment.  The results achieved with targeted therapies to date are promising, although they illustrate the need for addnl. preclin. and clin. study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruB8rVI305xrVg90H21EOLACvtfcHk0lj5hnxySqWoaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtleqsrrF&md5=d4e6b3997f2b0a252753cdc6a5137a15</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.2165%2F00003495-200767140-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00003495-200767140-00006%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DM.%2BF.%26aulast%3DLenz%26aufirst%3DH.-J.%26atitle%3DEGFR%252C%2520HER2%2520and%2520VEGF%2520pathways%253A%2520validated%2520targets%2520for%2520cancer%2520treatment%26jtitle%3DDrugs%26date%3D2007%26volume%3D67%26spage%3D2045%26epage%3D2075%26doi%3D10.2165%2F00003495-200767140-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lung, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sholl, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gad, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deep, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abd Elmageed, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Dysregulated gene expression predicts tumor aggressiveness in African-American prostate cancer patients</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">16335</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-34637-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-34637-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30397274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3cvmvFWruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=16335&author=H.+E.+A.+Aliauthor=P.-Y.+Lungauthor=A.+B.+Shollauthor=S.+A.+Gadauthor=J.+J.+Bustamanteauthor=H.+I.+Aliauthor=J.+S.+Rhimauthor=G.+Deepauthor=J.+Zhangauthor=Z.+Y.+Abd+Elmageed&title=Dysregulated+gene+expression+predicts+tumor+aggressiveness+in+African-American+prostate+cancer+patients&doi=10.1038%2Fs41598-018-34637-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulated gene expression predicts tumor aggressiveness in African-American prostate cancer patients</span></div><div class="casAuthors">Ali Hamdy E A; Gad Shaimaa A; Bustamante Juan J; Ali Hamed I; Abd Elmageed Zakaria Y; Ali Hamdy E A; Lung Pei-Yau; Zhang Jinfeng; Sholl Andrew B; Rhim Johng S; Deep Gagan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular mechanisms underlying the health disparity of prostate cancer (PCa) have not been fully determined.  In this study, we applied bioinformatic approach to identify and validate dysregulated genes associated with tumor aggressiveness in African American (AA) compared to Caucasian American (CA) men with PCa.  We retrieved and analyzed microarray data from 619 PCa patients, 412 AA and 207 CA, and we validated these genes in tumor tissues and cell lines by Real-Time PCR, Western blot, immunocytochemistry (ICC) and immunohistochemistry (IHC) analyses.  We identified 362 differentially expressed genes in AA men and involved in regulating signaling pathways associated with tumor aggressiveness.  In PCa tissues and cells, NKX3.1, APPL2, TPD52, LTC4S, ALDH1A3 and AMD1 transcripts were significantly upregulated (p < 0.05) compared to normal cells.  IHC confirmed the overexpression of TPD52 (p = 0.0098) and LTC4S (p < 0.0005) in AA compared to CA men.  ICC and Western blot analyses additionally corroborated this observation in PCa cells.  These findings suggest that dysregulation of transcripts in PCa may drive the disparity of PCa outcomes and provide new insights into development of new therapeutic agents against aggressive tumors.  More studies are warranted to investigate the clinical significance of these dysregulated genes in promoting the oncogenic pathways in AA men.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCHmWAdFRpT1w2ofyr07jXfW6udTcc2ea1CabifSmgQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvmvFWruw%253D%253D&md5=663c820ec472b059e2b707ba96465c3b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34637-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34637-8%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DH.%2BE.%2BA.%26aulast%3DLung%26aufirst%3DP.-Y.%26aulast%3DSholl%26aufirst%3DA.%2BB.%26aulast%3DGad%26aufirst%3DS.%2BA.%26aulast%3DBustamante%26aufirst%3DJ.%2BJ.%26aulast%3DAli%26aufirst%3DH.%2BI.%26aulast%3DRhim%26aufirst%3DJ.%2BS.%26aulast%3DDeep%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAbd%2BElmageed%26aufirst%3DZ.%2BY.%26atitle%3DDysregulated%2520gene%2520expression%2520predicts%2520tumor%2520aggressiveness%2520in%2520African-American%2520prostate%2520cancer%2520patients%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D16335%26doi%3D10.1038%2Fs41598-018-34637-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enot, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacchelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquelot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zitvogel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves</span>. <i>Oncoimmunology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1462431</span>– <span class="NLM_lpage">e1462433</span>, <span class="refDoi"> DOI: 10.1080/2162402x.2018.1462431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1080%2F2162402x.2018.1462431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30228932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGjsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e1462431-e1462433&author=D.+P.+Enotauthor=E.+Vacchelliauthor=N.+Jacquelotauthor=L.+Zitvogelauthor=G.+Kroemer&title=TumGrowth%3A+An+open-access+web+tool+for+the+statistical+analysis+of+tumor+growth+curves&doi=10.1080%2F2162402x.2018.1462431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves</span></div><div class="casAuthors">Enot, David P.; Vacchelli, Erika; Jacquelot, Nicolas; Zitvogel, Laurence; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">OncoImmunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1462431/1-e1462431/3</span>CODEN:
                <span class="NLM_cas:coden">ONCOGX</span>;
        ISSN:<span class="NLM_cas:issn">2162-402X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">The anal. of tumor growth curves is std. practice in exptl. oncol. including tumor immunol.  In exptl. oncol., cancer cells are inoculated into rodents (mostly mice) and their growth is monitored by measuring tumor diam., surface or vol. over time as a function of distinct treatments.  Then, different groups of tumors/treatments are compared among each other for their evolution and possible responses to treatment.  The R package TumGrowth has been created as a software tool allowing to carry out a series of statistical comparisons across or between groups of tumor growth curves obtained in a std. lab., for experimenters with limited knowledge in statistics.  TumGrowth is freely available online at https://kroemerlab.shinyapps.io/TumGrowth/ and can be downloaded into any computer.  It offers an exhaustive panoply of tools to visualize and analyze complex data sets including longitudinal, cross-sectional and time-to-endpoint measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7NNJlWdRsMbVg90H21EOLACvtfcHk0ljKmROTObFqPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGjsrfI&md5=82153bdf269c5c1d43e8e896dcf98d9e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1080%2F2162402x.2018.1462431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F2162402x.2018.1462431%26sid%3Dliteratum%253Aachs%26aulast%3DEnot%26aufirst%3DD.%2BP.%26aulast%3DVacchelli%26aufirst%3DE.%26aulast%3DJacquelot%26aufirst%3DN.%26aulast%3DZitvogel%26aufirst%3DL.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DTumGrowth%253A%2520An%2520open-access%2520web%2520tool%2520for%2520the%2520statistical%2520analysis%2520of%2520tumor%2520growth%2520curves%26jtitle%3DOncoimmunology%26date%3D2018%26volume%3D7%26spage%3De1462431%26epage%3De1462433%26doi%3D10.1080%2F2162402x.2018.1462431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ducray, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlaam, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickinson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trigwell, C. B.</span></span> <span> </span><span class="NLM_article-title">Novel 3-alkoxy-1H-pyrazolo[3,4-d]pyrimidines as EGFR and erbB2 receptor tyrosine kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2007.12.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18182285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=959-962&author=R.+Ducrayauthor=P.+Ballardauthor=B.+C.+Barlaamauthor=M.+D.+Hickinsonauthor=J.+G.+Kettleauthor=D.+J.+Ogilvieauthor=C.+B.+Trigwell&title=Novel+3-alkoxy-1H-pyrazolo%5B3%2C4-d%5Dpyrimidines+as+EGFR+and+erbB2+receptor+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2007.12.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3-alkoxy-1H-pyrazolo[3,4-d]pyrimidines as EGFR and erbB2 receptor tyrosine kinase inhibitors</span></div><div class="casAuthors">Ducray, Richard; Ballard, Peter; Barlaam, Bernard C.; Hickinson, Mark D.; Kettle, Jason G.; Ogilvie, Donald J.; Trigwell, Catherine B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">959-962</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of 4-anilino-1H-pyrazolo[3,4-d]pyrimidines was synthesized and evaluated in vitro for their erbB2 and EGFR kinase inhibition.  Compd. I displaying oral bioavailability in rat and dog illustrates the potential of this series to provide orally active erbB2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq7N8uYVaWT7Vg90H21EOLACvtfcHk0lhPPITKzeEjxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWktL0%253D&md5=c40865c049e4f650af70e4370fccb1f4</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.035%26sid%3Dliteratum%253Aachs%26aulast%3DDucray%26aufirst%3DR.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBarlaam%26aufirst%3DB.%2BC.%26aulast%3DHickinson%26aufirst%3DM.%2BD.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DTrigwell%26aufirst%3DC.%2BB.%26atitle%3DNovel%25203-alkoxy-1H-pyrazolo%255B3%252C4-d%255Dpyrimidines%2520as%2520EGFR%2520and%2520erbB2%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D959%26epage%3D962%26doi%3D10.1016%2Fj.bmcl.2007.12.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singla, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banipal, R. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span> <span> </span><span class="NLM_article-title">Recent updates on the therapeutic potential of HER2 tyrosine kinase inhibitors for the treatment of breast cancer</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.2174/1568009617666170623122213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2174%2F1568009617666170623122213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28669349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFaqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=306-327&author=H.+Singlaauthor=A.+Munshiauthor=R.+P.+S.+Banipalauthor=V.+Kumar&title=Recent+updates+on+the+therapeutic+potential+of+HER2+tyrosine+kinase+inhibitors+for+the+treatment+of+breast+cancer&doi=10.2174%2F1568009617666170623122213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Updates on the Therapeutic Potential of HER2 Tyrosine Kinase Inhibitors for the Treatment of Breast Cancer</span></div><div class="casAuthors">Singla, Heena; Munshi, Anjana; Singh Banipal, Raja Paramjit; Kumar, Vinod</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">306-327</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HER2 pos. breast cancer is characterized by the low survival rate in the metastatic patients.  Development of resistance and disease-relapse are the major problems assocd. with the currently available therapies for HER2 pos. breast cancer.  There are two major targeted therapies for HER2 pos. breast cancer viz. monoclonal antibodies and tyrosine-kinase inhibitors, and both of these therapies have their advantages and limitations.  To address the limitations assocd. with the existing therapies, use of antibodies and TKIs as combination therapy proved to be more effective.  Various chem. modifications can be performed on tyrosine-kinase inhibitors to develop novel ligands with increased selectivity for HER2 kinase.  A no. of tyrosine-kinase inhibitors are in various phases of clin. trials for the treatment of HER2 pos. breast cancer.  In the current review article, recent developments on various HER2 tyrosine-kinase inhibitors have been reported.  Various structurally different scaffolds bind to the HER2 receptor and exhibit potent anti-cancer activities.  The structural and pharmacophoric requirements of the scaffolds are discussed in detail so as to discover effective drug candidates for the treatment of HER2 pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8LTwxEd80KrVg90H21EOLACvtfcHk0lhPPITKzeEjxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFaqurw%253D&md5=93ec4f7943db373d853d0810c2964155</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.2174%2F1568009617666170623122213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009617666170623122213%26sid%3Dliteratum%253Aachs%26aulast%3DSingla%26aufirst%3DH.%26aulast%3DMunshi%26aufirst%3DA.%26aulast%3DBanipal%26aufirst%3DR.%2BP.%2BS.%26aulast%3DKumar%26aufirst%3DV.%26atitle%3DRecent%2520updates%2520on%2520the%2520therapeutic%2520potential%2520of%2520HER2%2520tyrosine%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D306%26epage%3D327%26doi%3D10.2174%2F1568009617666170623122213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidibe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bararpour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desmeules, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausburger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daali, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span> <span> </span><span class="NLM_article-title">Detection and identification of reactive drug metabolites leading to idiosyncratic toxicity: lapatinib as a case example</span>. <i>J. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1000242</span>, <span class="refDoi"> DOI: 10.4172/2157-7609.1000242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.4172%2F2157-7609.1000242" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1000242&author=R.+Mohamedauthor=F.+Storelliauthor=J.+Sidibeauthor=N.+Bararpourauthor=J.+Desmeulesauthor=M.+Ausburgerauthor=Y.+Daaliauthor=A.+Thomas&title=Detection+and+identification+of+reactive+drug+metabolites+leading+to+idiosyncratic+toxicity%3A+lapatinib+as+a+case+example&doi=10.4172%2F2157-7609.1000242"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.4172%2F2157-7609.1000242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2157-7609.1000242%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DR.%26aulast%3DStorelli%26aufirst%3DF.%26aulast%3DSidibe%26aufirst%3DJ.%26aulast%3DBararpour%26aufirst%3DN.%26aulast%3DDesmeules%26aufirst%3DJ.%26aulast%3DAusburger%26aufirst%3DM.%26aulast%3DDaali%26aufirst%3DY.%26aulast%3DThomas%26aufirst%3DA.%26atitle%3DDetection%2520and%2520identification%2520of%2520reactive%2520drug%2520metabolites%2520leading%2520to%2520idiosyncratic%2520toxicity%253A%2520lapatinib%2520as%2520a%2520case%2520example%26jtitle%3DJ.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2018%26volume%3D9%26spage%3D1000242%26doi%3D10.4172%2F2157-7609.1000242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of pyrotinib: A novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor with favorable safety profiles for the treatment of breast cancer</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2017.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejps.2017.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28115222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1Wqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2017&pages=51-61&author=X.+Liauthor=C.+Yangauthor=H.+Wanauthor=G.+Zhangauthor=J.+Fengauthor=L.+Zhangauthor=X.+Chenauthor=D.+Zhongauthor=L.+Louauthor=W.+Taoauthor=L.+Zhang&title=Discovery+and+development+of+pyrotinib%3A+A+novel+irreversible+EGFR%2FHER2+dual+tyrosine+kinase+inhibitor+with+favorable+safety+profiles+for+the+treatment+of+breast+cancer&doi=10.1016%2Fj.ejps.2017.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of pyrotinib: A novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor with favorable safety profiles for the treatment of breast cancer</span></div><div class="casAuthors">Li, Xin; Yang, Changyong; Wan, Hong; Zhang, Ge; Feng, Jun; Zhang, Lei; Chen, Xiaoyan; Zhong, Dafang; Lou, Liguang; Tao, Weikang; Zhang, Lianshan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery and development of a novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor SHR1258 (pyrotinib) for the treatment of HER2-postive breast cancer is presented.  The structure-activity relationship of lead series and their pharmacokinetic properties were evaluated to identify the potential candidates for further in vivo efficacy studies and preclin. safety assessments.  Metabolic pathway and drug-drug interaction were also investigated in preclin. settings.  In particular, major metabolites in human and animal species were assessed with regard to potential toxicity or off-target side effects.  Overall, the potent and selective EGFR/HER2 dual inhibitor, pyrotinib, displayed robust anti-tumor effects on HER2-overexpressing xenograft models and sufficiently safety windows in animals as well as favorable pharmacokinetic properties in human, which substantially ensures current clin. development.  Finally, recent advances of pyrotinib in clin. studies are highlighted with very encouraging outcomes in patients with HER2-postive advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdbJnY0jwOjLVg90H21EOLACvtfcHk0liYD2YRt5_28Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1Wqsrw%253D&md5=12f13882a32cfb56959f58deaadadce8</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2017.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2017.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DLou%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DDiscovery%2520and%2520development%2520of%2520pyrotinib%253A%2520A%2520novel%2520irreversible%2520EGFR%252FHER2%2520dual%2520tyrosine%2520kinase%2520inhibitor%2520with%2520favorable%2520safety%2520profiles%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D110%26spage%3D51%26epage%3D61%26doi%3D10.1016%2Fj.ejps.2017.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Comparative evaluation of 11 scoring functions for molecular docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">2287</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1021/jm0203783</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0203783" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVOgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2287-2303&author=R.+Wangauthor=Y.+Luauthor=S.+Wang&title=Comparative+evaluation+of+11+scoring+functions+for+molecular+docking&doi=10.1021%2Fjm0203783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Evaluation of 11 Scoring Functions for Molecular Docking</span></div><div class="casAuthors">Wang, Renxiao; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2287-2303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Eleven popular scoring functions have been tested on 100 protein-ligand complexes to evaluate their abilities to reproduce exptl. detd. structures and binding affinities.  They include four scoring functions implemented in the LigFit module in Cerius2 (LigScore, PLP, PMF, and LUDI), four scoring functions implemented in the CScore module in SYBYL (F-Score, G-Score, D-Score, and ChemScore), the scoring function implemented in the AutoDock program, and two stand-alone scoring functions (DrugScore and X-Score).  These scoring functions are not tested in the context of a particular docking program.  Instead, conformational sampling and scoring are sepd. into two consecutive steps.  First, an exhaustive conformational sampling is performed by using the AutoDock program to generate an ensemble of docked conformations for each ligand mol.  This conformational ensemble is required to cover the entire conformational space as much as possible rather than to focus on a few energy min. Then, each scoring function is applied to score this conformational ensemble to see if it can identify the exptl. obsd. conformation from all of the other decoys.  Among all of the scoring functions under test, six of them, i.e., PLP, F-Score, LigScore, DrugScore, LUDI, and X-Score, yield success rates higher than the AutoDock scoring function.  The success rates of these six scoring functions range from 66% to 76% if using root-mean-square deviation ≤2.0 Å as the criterion.  Combining any two or three of these six scoring functions into a consensus scoring scheme further improves the success rate to nearly 80% or even higher.  However, when applied to reproduce the exptl. detd. binding affinities of the 100 protein-ligand complexes, only X-Score, PLP, DrugScore, and G-Score are able to give correlation coeffs. over 0.50.  All of the 11 scoring functions are further inspected by their abilities to construct a descriptive, funnel-shaped energy surface for protein-ligand complexation.  The results indicate that X-Score and DrugScore perform better than the other ones at this aspect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPH6dgkF_TeLVg90H21EOLACvtfcHk0liYD2YRt5_28Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVOgtr0%253D&md5=2f6e7425f8d612124a06f33e2e6f0400</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Fjm0203783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0203783%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DComparative%2520evaluation%2520of%252011%2520scoring%2520functions%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2287%26epage%3D2303%26doi%3D10.1021%2Fjm0203783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0liYD2YRt5_28Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, F. O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinase inhibitors. 20. optimization of substituted quinazoline and pyrido[3,4-d]pyrimidine derivatives as orally active, irreversible inhibitors of the epidermal growth factor receptor family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8103</span>– <span class="NLM_lpage">8124</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00883</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00883" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8103-8124&author=J.+B.+Smaillauthor=A.+J.+Gonzalesauthor=J.+A.+Spicerauthor=H.+Leeauthor=J.+E.+Reedauthor=K.+Sextonauthor=I.+W.+Althausauthor=T.+Zhuauthor=S.+L.+Blackauthor=A.+Blaserauthor=W.+A.+Dennyauthor=P.+A.+Ellisauthor=S.+Fakhouryauthor=P.+J.+Harveyauthor=K.+Hookauthor=F.+O.+J.+McCarthyauthor=B.+D.+Palmerauthor=F.+Rivaultauthor=K.+Schlosserauthor=T.+Ellisauthor=A.+M.+Thompsonauthor=E.+Trachetauthor=R.+T.+Wintersauthor=H.+Tecleauthor=A.+Bridges&title=Tyrosine+kinase+inhibitors.+20.+optimization+of+substituted+quinazoline+and+pyrido%5B3%2C4-d%5Dpyrimidine+derivatives+as+orally+active%2C+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+family&doi=10.1021%2Facs.jmedchem.6b00883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine Kinase Inhibitors. 20. Optimization of Substituted Quinazoline and Pyrido[3,4-d]pyrimidine Derivatives as Orally Active, Irreversible Inhibitors of the Epidermal Growth Factor Receptor Family</span></div><div class="casAuthors">Smaill, Jeff B.; Gonzales, Andrea J.; Spicer, Julie A.; Lee, Helen; Reed, Jessica E.; Sexton, Karen; Althaus, Irene W.; Zhu, Tong; Black, Shannon L.; Blaser, Adrian; Denny, William A.; Ellis, Paul A.; Fakhoury, Stephen; Harvey, Patricia J.; Hook, Ken; McCarthy, Florence O. J.; Palmer, Brian D.; Rivault, Freddy; Schlosser, Kevin; Ellis, Teresa; Thompson, Andrew M.; Trachet, Erin; Winters, R. Thomas; Tecle, Haile; Bridges, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8103-8124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships for inhibition of erbB1, erbB2 and erbB4 were detd. for a series of quinazoline- and pyrido[3,4-d]pyrimidine-based analogs of the irreversible pan-erbB inhibitor, canertinib.  Cyclic amine bearing crotonamides were detd. to provide rapid inhibition of cellular erbB1 autophosphorylation and good metabolic stability in liver microsome and hepatocyte assays.  The influence of 4-anilino substitution on pan-erbB inhibitory potency was investigated.  Several anilines were identified as providing potent, reversible pan-erbB inhibition.  Optimum 4- and 6-substituents with known 7-substituents provided preferred irreversible inhibitors for pharmacodynamic testing in vivo.  Quinazoline I and pyrido[3,4-d]pyrimidine II were identified as clearly superior to canertinib.  Both compds. possess a piperidinyl crotonamide Michael acceptor and a 3-chloro-4-fluoroaniline, indicating these as optimized 6- and 4-substituents resp.  Pharmacokinetic comparison of compds. I and II across three species selected compd. I as the preferred candidate.  Compd. I (PF-00299804) has been assigned the nomenclature of dacomitinib and is currently under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0-NpOmVq10bVg90H21EOLACvtfcHk0lhuqrVS7YhjZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM&md5=442afda669a5c1b7a190cd82b77bee33</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00883%26sid%3Dliteratum%253Aachs%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DSpicer%26aufirst%3DJ.%2BA.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DSexton%26aufirst%3DK.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBlack%26aufirst%3DS.%2BL.%26aulast%3DBlaser%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DFakhoury%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DHook%26aufirst%3DK.%26aulast%3DMcCarthy%26aufirst%3DF.%2BO.%2BJ.%26aulast%3DPalmer%26aufirst%3DB.%2BD.%26aulast%3DRivault%26aufirst%3DF.%26aulast%3DSchlosser%26aufirst%3DK.%26aulast%3DEllis%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DA.%2BM.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DBridges%26aufirst%3DA.%26atitle%3DTyrosine%2520kinase%2520inhibitors.%252020.%2520optimization%2520of%2520substituted%2520quinazoline%2520and%2520pyrido%255B3%252C4-d%255Dpyrimidine%2520derivatives%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8103%26epage%3D8124%26doi%3D10.1021%2Facs.jmedchem.6b00883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkamhawy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanath, A. N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedair, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leem, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pae, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, E. J.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5147</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26475520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5147-5154&author=A.+Elkamhawyauthor=A.+K.+Faragauthor=A.+N.+I.+Viswanathauthor=T.+M.+Bedairauthor=D.+G.+Leemauthor=K.-T.+Leeauthor=A.+N.+Paeauthor=E.+J.+Roh&title=Targeting+EGFR%2FHER2+tyrosine+kinases+with+a+new+potent+series+of+6-substituted+4-anilinoquinazoline+hybrids%3A+Design%2C+synthesis%2C+kinase+assay%2C+cell-based+assay%2C+and+molecular+docking&doi=10.1016%2Fj.bmcl.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span></div><div class="casAuthors">Elkamhawy, Ahmed; Farag, Ahmed Karam; Viswanath, Ambily Nath Indu; Bedair, Tarek M.; Leem, Dong Gyu; Lee, Kyung-Tae; Pae, Ae Nim; Roh, Eun Joo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5147-5154</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coexpression of EGFR and HER2 has been found in many tumors such as breast, ovarian, colon and prostate cancers, with poor prognosis of the patients.  Herein, our team has designed and synthesized new eighteen compds. with 6-substituted 4-anilinoquinazoline core to selectively inhibit EGFR/HER2 tyrosine kinases.  Twelve compds. showed nanomolar range of IC50 values on EGFR and/or HER2 kinases.  Accordingly, a detailed structure activity relationship (SAR) was established.  A mol. docking study demonstrated the favorable binding modes of I (X = Me, NHEt) at the ATP active site of both kinases.  A kinase selectivity profile performed for compd. 8d showed great selectivity for EGFR and HER2.  In addn., compd. I (X = Me, NHEt) and II exerted selective promising cytotoxic activity over BT-474 cell line with IC50 values of 2.70, 1.82 and 1.95 μM, resp.  From these results, we report analogs I (X = Me, NHEt) and II as promising candidates for the discovery of well-balanced compds. in terms of the kinase inhibitory potency and antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_Ln0rUhX7bVg90H21EOLACvtfcHk0lhuqrVS7YhjZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP&md5=0d7c9e4896863934f0190537c10c42ae</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DElkamhawy%26aufirst%3DA.%26aulast%3DFarag%26aufirst%3DA.%2BK.%26aulast%3DViswanath%26aufirst%3DA.%2BN.%2BI.%26aulast%3DBedair%26aufirst%3DT.%2BM.%26aulast%3DLeem%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DK.-T.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRoh%26aufirst%3DE.%2BJ.%26atitle%3DTargeting%2520EGFR%252FHER2%2520tyrosine%2520kinases%2520with%2520a%2520new%2520potent%2520series%2520of%25206-substituted%25204-anilinoquinazoline%2520hybrids%253A%2520Design%252C%2520synthesis%252C%2520kinase%2520assay%252C%2520cell-based%2520assay%252C%2520and%2520molecular%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5147%26epage%3D5154%26doi%3D10.1016%2Fj.bmcl.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0li7YuF6J1H1Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 4-anilinoquinazoline derivatives as potent inhibitors of epidermal growth factor receptor with antitumor activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6084</span>– <span class="NLM_lpage">6091</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.06.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmc.2013.06.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23962660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12itb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6084-6091&author=Y.-Y.+Xuauthor=S.-N.+Liauthor=G.-J.+Yuauthor=Q.-H.+Huauthor=H.-Q.+Li&title=Discovery+of+novel+4-anilinoquinazoline+derivatives+as+potent+inhibitors+of+epidermal+growth+factor+receptor+with+antitumor+activity&doi=10.1016%2Fj.bmc.2013.06.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel 4-anilinoquinazoline derivatives as potent inhibitors of epidermal growth factor receptor with antitumor activity</span></div><div class="casAuthors">Xu, Yun-Yun; Li, Si-Ning; Yu, Gao-Jian; Hu, Qing-Hua; Li, Huan-Qiu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6084-6091</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two new series of new compds. contg. a 6-amino-substituted group or 6-acrylamide-substituted group linked to a 4-anilinoquinazoline nucleus have been discovered as potential EGFR inhibitors.  These compds. proved efficient effects on antiproliferative activity and EGFR-TK inhibitory activity.  Esp., N6-((5-bromothiophen-2-yl)methyl)-N4-(3-chlorophenyl)quinazoline-4,6-diamine (5e), showed the most potent inhibitory activity (IC50 = 3.11 μM for Hep G2, IC50 = 0.82 μM for A549).  The EGFR mol. docking model suggested that the new compd. is nicely bound to the region of EGFR, and cell morphol. by Hoechst stain expt. suggested that these compds. efficiently induced apoptosis of A549 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3fZ-t5f7RLVg90H21EOLACvtfcHk0li7YuF6J1H1Fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12itb%252FK&md5=e97214f67937ab2757a68f3c6474e98b</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.06.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.06.070%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.-Y.%26aulast%3DLi%26aufirst%3DS.-N.%26aulast%3DYu%26aufirst%3DG.-J.%26aulast%3DHu%26aufirst%3DQ.-H.%26aulast%3DLi%26aufirst%3DH.-Q.%26atitle%3DDiscovery%2520of%2520novel%25204-anilinoquinazoline%2520derivatives%2520as%2520potent%2520inhibitors%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520with%2520antitumor%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6084%26epage%3D6091%26doi%3D10.1016%2Fj.bmc.2013.06.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of quinazoline-phosphoramidate mustard conjugates as anticancer drugs</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.12.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2016.12.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28092860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=442-458&author=S.+Linauthor=Y.+Liauthor=Y.+Zhengauthor=L.+Luoauthor=Q.+Sunauthor=Z.+Geauthor=T.+Chengauthor=R.+Li&title=Design%2C+synthesis+and+biological+evaluation+of+quinazoline-phosphoramidate+mustard+conjugates+as+anticancer+drugs&doi=10.1016%2Fj.ejmech.2016.12.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of quinazoline-phosphoramidate mustard conjugates as anticancer drugs</span></div><div class="casAuthors">Lin, Songwen; Li, Yingbo; Zheng, Yufen; Luo, Laichun; Sun, Qi; Ge, Zemei; Cheng, Tieming; Li, Runtao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">442-458</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Novel compds. with phosphoramide mustard functionality incorporated into the quinazoline scaffold of EGFR/HER2 inhibitors were designed and synthesized as multi-target-directed ligands against tumor cells.  In vitro assays showed that tumor cell lines with high HER2 level were more sensitive to the compds. than tumor cells with low HER2 level.  Compd. 10d (EMB-3) was one of the most potent inhibitors with IC50 of 7.4 nM and 82 nM against EGFR and HER2, resp.  The mechanism studies were also supported by the effect of 10d-induced DNA damage in MDA-MB-468 cells.  In vivo efficacy study showed that 10d could significantly inhibit H522 tumor xenograft model with a TGI of 68% at dose of 100 mg/kg (QDx28, p.o.) and no significant body wt. loss was obsd.  MTD study indicated that compd. 10d had no acute toxicity to mice at doses up to 900 mg/kg (single dose).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUfoD6MEDej7Vg90H21EOLACvtfcHk0liNwhLGgez1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ahsr4%253D&md5=20306ebaa26e1259a951f2eef0db5b55</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.12.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.12.055%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520quinazoline-phosphoramidate%2520mustard%2520conjugates%2520as%2520anticancer%2520drugs%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D127%26spage%3D442%26epage%3D458%26doi%3D10.1016%2Fj.ejmech.2016.12.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamed, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou El Ella, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeton, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piazza, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">6-Aryl and heterocycle quinazoline derivatives as potent EGFR inhibitors with improved activity toward gefitinib-sensitive and -resistant tumor cell lines</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1495</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcmdc.201300147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23847159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVykurzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1495-1504&author=M.+M.+Hamedauthor=D.+A.+Abou+El+Ellaauthor=A.+B.+Keetonauthor=G.+A.+Piazzaauthor=A.+H.+Abadiauthor=R.+W.+Hartmannauthor=M.+Engel&title=6-Aryl+and+heterocycle+quinazoline+derivatives+as+potent+EGFR+inhibitors+with+improved+activity+toward+gefitinib-sensitive+and+-resistant+tumor+cell+lines&doi=10.1002%2Fcmdc.201300147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">6-Aryl and Heterocycle Quinazoline Derivatives as Potent EGFR Inhibitors with Improved Activity toward Gefitinib-Sensitive and -Resistant Tumor Cell Lines</span></div><div class="casAuthors">Hamed, Mostafa M.; Abou El Ella, Dalal A.; Keeton, Adam B.; Piazza, Gary A.; Abadi, Ashraf H.; Hartmann, Rolf W.; Engel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1495-1504</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Anilinoquinazolinyl benzaldimines I [R = R1CH:N; R1 = 4-HOC6H4, 2,4-(HO)2C6H3, 3-O2NC6H4, 4-O2NC6H4, 2-MeOC6H4], thioureas I [R = R1NHC(:S)NH; R1 = PhCH2, 2-(4-morpholinyl)ethyl], aminoacetamides and aminopropanamides II [R = R1NHCH2CONH, R1NHCH2CH2CONH; R1 = PhCH2, 4-(2-thiazolylaminosulfonyl)phenyl], and amides [R = R1CONH; R1 = 4-H2NC6H4, 4-NCC6H4, 4-MeCONHC6H4, 4-O2NC6H4, 3,5-(O2N)2C6H3, 4-MeOC6H4, 3,4-(MeO)2C6H3, 4-morpholinyl, 2-furyl, 3-pyridinyl, 4-pyridinyl] were prepd. as inhibitors of wild-type and mutant epidermal growth factor receptor (EGFR) for potential use as antitumor agents.  Most of the compds. inhibited wild-type EGFR with IC50 values of 4-62 nM; I [R = BnNHC(:S)NH, 4-morpholinylethylaminothiocarbonylamino, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] also retained significant activity toward the gefitinib-insensitive EGFRT790M/L858R mutant, displaying up to 24-fold greater potency than gefitinib.  The inhibition of the growth of cancer cells with wild-type and mutant EGFR was detd.; I [R = BnNHC(:S)NH, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] inhibited cancer cell growth more effectively than the std. gefitinib.  Mol. docking calcns. of I [R = BnNHC(:S)NH, 4-morpholinylethylaminothiocarbonylamino, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] in double mutant EGFR and of I (R = PhCH2NHCH2CONH, 2-furylcarbonylamino) in wild-type EGFR were performed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9J8DhyAlHb7Vg90H21EOLACvtfcHk0liNwhLGgez1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVykurzF&md5=c951d913a38a5062783da74c4cf6afef</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300147%26sid%3Dliteratum%253Aachs%26aulast%3DHamed%26aufirst%3DM.%2BM.%26aulast%3DAbou%2BEl%2BElla%26aufirst%3DD.%2BA.%26aulast%3DKeeton%26aufirst%3DA.%2BB.%26aulast%3DPiazza%26aufirst%3DG.%2BA.%26aulast%3DAbadi%26aufirst%3DA.%2BH.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3D6-Aryl%2520and%2520heterocycle%2520quinazoline%2520derivatives%2520as%2520potent%2520EGFR%2520inhibitors%2520with%2520improved%2520activity%2520toward%2520gefitinib-sensitive%2520and%2520-resistant%2520tumor%2520cell%2520lines%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1495%26epage%3D1504%26doi%3D10.1002%2Fcmdc.201300147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forcellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shayhidin, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhéaume, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagüe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquin, J.-F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel quinazoline-4-piperidinesulfamide derivatives as inhibitors of NPP1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.01.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29427906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=130-149&author=E.+Forcelliniauthor=S.+Boutinauthor=C.-A.+Lefebvreauthor=E.+E.+Shayhidinauthor=M.-C.+Boulangerauthor=G.+Rh%C3%A9aumeauthor=X.+Barbeauauthor=P.+Lag%C3%BCeauthor=P.+Mathieuauthor=J.-F.+Paquin&title=Synthesis+and+biological+evaluation+of+novel+quinazoline-4-piperidinesulfamide+derivatives+as+inhibitors+of+NPP1&doi=10.1016%2Fj.ejmech.2018.01.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel quinazoline-4-piperidine sulfamide derivatives as inhibitors of NPP1</span></div><div class="casAuthors">Forcellini, Elsa; Boutin, Sophie; Lefebvre, Carole-Anne; Shayhidin, Elnur Elyar; Boulanger, Marie-Chloe; Rheaume, Gabrielle; Barbeau, Xavier; Lague, Patrick; Mathieu, Patrick; Paquin, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">130-149</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The ecto-nucleotide pyrophosphatase/phosphodiesterase-1 (NPP1) was recently shown to promote mineralization of the aortic valve, hence, its inhibition represents a significant target.  A quinazoline-4-piperidine sulfamide compd. (I) has been described as a specific and non-competitive inhibitor of NPP1.  Herein the synthesis and in vitro inhibition studies of novel quinazoline-4-piperidine sulfamide analogs using I as the lead compd. was reported.  Of the 26 derivs. prepd., four compds. were found to have Ki < 105 nM against human NPP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwRlT7Fi3_H7Vg90H21EOLACvtfcHk0liNwhLGgez1rA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D&md5=39d4f90e63b612922cfb5a60b47bd688</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.094%26sid%3Dliteratum%253Aachs%26aulast%3DForcellini%26aufirst%3DE.%26aulast%3DBoutin%26aufirst%3DS.%26aulast%3DLefebvre%26aufirst%3DC.-A.%26aulast%3DShayhidin%26aufirst%3DE.%2BE.%26aulast%3DBoulanger%26aufirst%3DM.-C.%26aulast%3DRh%25C3%25A9aume%26aufirst%3DG.%26aulast%3DBarbeau%26aufirst%3DX.%26aulast%3DLag%25C3%25BCe%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DP.%26aulast%3DPaquin%26aufirst%3DJ.-F.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520quinazoline-4-piperidinesulfamide%2520derivatives%2520as%2520inhibitors%2520of%2520NPP1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D130%26epage%3D149%26doi%3D10.1016%2Fj.ejmech.2018.01.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group">National Cancer
Institute,
DCTD Division of Cancer Treatment & Diagnosis. Discovery &
Development Services</span>. <span> </span><span class="NLM_article-title">NCI-60
Human Cancer Cell Line Screen NCI-60 Screening Methodology</span>. <a href="https://dtp.cancer.gov/discovery_development/nci-60/" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/</a> (accessed
Dec 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=National+Cancer%0AInstitute%2C%0ADCTD+Division+of+Cancer+Treatment+%26+Diagnosis.+Discovery+%26%0ADevelopment+Services.+NCI-60%0AHuman+Cancer+Cell+Line+Screen+NCI-60+Screening+Methodology.+https%3A%2F%2Fdtp.cancer.gov%2Fdiscovery_development%2Fnci-60%2F+%28accessed%0ADec+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DNCI-60%250AHuman%2520Cancer%2520Cell%2520Line%2520Screen%2520NCI-60%2520Screening%2520Methodology" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darzynkiewicz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Bino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorczyca, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassota, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traganos, F.</span></span> <span> </span><span class="NLM_article-title">Features of apoptotic cells measured by flow cytometry</span>. <i>Cytometry</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1002/cyto.990130802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcyto.990130802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1333943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADyaK3s%252Fpt1OrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=795-808&author=Z.+Darzynkiewiczauthor=S.+Brunoauthor=G.+Del+Binoauthor=W.+Gorczycaauthor=M.+A.+Hotzauthor=P.+Lassotaauthor=F.+Traganos&title=Features+of+apoptotic+cells+measured+by+flow+cytometry&doi=10.1002%2Fcyto.990130802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Features of apoptotic cells measured by flow cytometry</span></div><div class="casAuthors">Darzynkiewicz Z; Bruno S; Del Bino G; Gorczyca W; Hotz M A; Lassota P; Traganos F</div><div class="citationInfo"><span class="NLM_cas:title">Cytometry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">795-808</span>
        ISSN:<span class="NLM_cas:issn">0196-4763</span>.
    </div><div class="casAbstract">The present review describes several methods to characterize and differentiate between two different mechanisms of cell death, apoptosis and necrosis.  Most of these methods were applied to studies of apoptosis triggered in the human leukemic HL-60 cell line by DNA topoisomerase I or II inhibitors, and in rat thymocytes by either topoisomerase inhibitors or prednisolone.  In most cases, apoptosis was selective to cells in a particular phase of the cell cycle: only S-phase HL-60 cells and G0 thymocytes were mainly affected.  Necrosis was induced by excessively high concentrations of these drugs.  The following cell features were found useful to characterize the mode of cell death: a) Activation of an endonuclease in apoptocic cells resulted in extraction of the low molecular weight DNA following cell permeabilization, which, in turn, led to their decreased stainability with DNA-specific fluorochromes.  Measurements of DNA content made it possible to identify apoptotic cells and to recognize the cell cycle phase specificity of the apoptotic process. b) Plasma membrane integrity, which is lost in necrotic but not apoptotic cells, was probed by the exclusion of propidium iodide (PI).  The combination of PI followed by Hoechst 33342 proved to be an excellent probe to distinguish live, necrotic, early- and late-apoptotic cells. c) Mitochondrial transmembrane potential, assayed by retention of rhodamine 123 was preserved in apoptotic but not necrotic cells. d) The ATP-dependent lysosomal proton pump, tested by the supravital uptake of acridine orange (AO) was also preserved in apoptotic but not necrotic cells. e) Bivariate analysis of cells stained for DNA and protein revealed markedly diminished protein content in apoptotic cells, most likely due to activation of endogenous proteases.  Necrotic cells, having leaky membranes, had minimal protein content. f) Staining of RNA allowed for the discrimination of G0 from G1 cells and thus made it possible to reveal that apoptosis was selective to G0 thymocytes. g) The decrease in forward light scatter, paralleled either by no change (HL-60 cells) or an increase (thymocytes) of right angle scatter, were early changes during apoptosis. h) The sensitivity of DNA in situ to denaturation, was increased in apoptotic and necrotic cells.  This feature, probed by staining with AO at low pH, provided a sensitive and early assay to discriminate between live, apoptotic and necrotic cells, and to evaluate the cell cycle phase specificity of these processes. i) The in situ nick translation assay employing labeled triphosphonucleotides can be used to reveal DNA strand breaks, to detect the very early stages of apoptosis.(ABSTRACT TRUNCATED AT 400 WORDS)</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5cujAkovwXEbxh97K6Vt6fW6udTcc2ea6GWHmaZkafbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s%252Fpt1OrtQ%253D%253D&md5=03a90fef61f38d88683f68707cd24f30</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2Fcyto.990130802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcyto.990130802%26sid%3Dliteratum%253Aachs%26aulast%3DDarzynkiewicz%26aufirst%3DZ.%26aulast%3DBruno%26aufirst%3DS.%26aulast%3DDel%2BBino%26aufirst%3DG.%26aulast%3DGorczyca%26aufirst%3DW.%26aulast%3DHotz%26aufirst%3DM.%2BA.%26aulast%3DLassota%26aufirst%3DP.%26aulast%3DTraganos%26aufirst%3DF.%26atitle%3DFeatures%2520of%2520apoptotic%2520cells%2520measured%2520by%2520flow%2520cytometry%26jtitle%3DCytometry%26date%3D1992%26volume%3D13%26spage%3D795%26epage%3D808%26doi%3D10.1002%2Fcyto.990130802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akl, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busnena, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foudah, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayed, K. A. E.</span></span> <span> </span><span class="NLM_article-title">Olive phenolics as c-Met inhibitors: (-)-Oleocanthal attenuates cell proliferation, invasiveness, and tumor growth in breast cancer models</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e97622</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0097622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0097622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24849787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKhtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e97622&author=M.+R.+Aklauthor=N.+M.+Ayoubauthor=M.+M.+Mohyeldinauthor=B.+A.+Busnenaauthor=A.+I.+Foudahauthor=Y.-Y.+Liuauthor=K.+A.+E.+Sayed&title=Olive+phenolics+as+c-Met+inhibitors%3A+%28-%29-Oleocanthal+attenuates+cell+proliferation%2C+invasiveness%2C+and+tumor+growth+in+breast+cancer+models&doi=10.1371%2Fjournal.pone.0097622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Olive phenolics as c-Met inhibitors: (-)-Oleocanthal attenuates cell proliferation, invasiveness, and tumor growth in breast cancer models</span></div><div class="casAuthors">Akl, Mohamed R.; Ayoub, Nehad M.; Mohyeldin, Mohamed M.; Busnena, Belnaser A.; Foudah, Ahmed I.; Liu, Yong-Yu; El Sayed, Khalid A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e97622/1-e97622/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Dysregulation of the Hepatocyte growth factor (HGF)/c-Met signaling axis upregulates diverse tumor cell functions, including cell proliferation, survival, scattering and motility, epithelial-to-mesenchymal transition (EMT), angiogenesis, invasion, and metastasis.  (-)-Oleocanthal is a naturally occurring secoiridoid from extra-virgin olive oil, which showed antiproliferative and antimigratory activity against different cancer cell lines.  The aim of this study was to characterize the intracellular mechanisms involved in mediating the anticancer effects of (-)-oleocanthal treatment and the potential involvement of c-Met receptor signaling components in breast cancer.  Results showed that (-)-oleocanthal inhibits the growth of human breast cancer cell lines MDA-MB-231, MCF-7 and BT-474 while similar treatment doses were found to have no effect on normal human MCF10A cell growth.  In addn., (-)-oleocanthal treatment caused a dose-dependent inhibition of HGF-induced cell migration, invasion and G1/S cell cycle progression in breast cancer cell lines.  Moreover, (-)-oleocanthal treatment effects were found to be mediated via inhibition of HGF-induced c-Met activation and its downstream mitogenic signaling pathways.  This growth inhibitory effect is assocd. with blockade of EMT and redn. in cellular motility.  Further results from in vivo studies showed that (-)-oleocanthal treatment suppressed tumor cell growth in an orthotopic model of breast cancer in athymic nude mice.  Collectively, the findings of this study suggest that (-)-oleocanthal is a promising dietary supplement lead with potential for therapeutic use to control malignancies with aberrant c-Met activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEdF-y6UaVqbVg90H21EOLACvtfcHk0lg8QP23zzFuKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKhtbbI&md5=c58947fd248d835a77f916acd69da948</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0097622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0097622%26sid%3Dliteratum%253Aachs%26aulast%3DAkl%26aufirst%3DM.%2BR.%26aulast%3DAyoub%26aufirst%3DN.%2BM.%26aulast%3DMohyeldin%26aufirst%3DM.%2BM.%26aulast%3DBusnena%26aufirst%3DB.%2BA.%26aulast%3DFoudah%26aufirst%3DA.%2BI.%26aulast%3DLiu%26aufirst%3DY.-Y.%26aulast%3DSayed%26aufirst%3DK.%2BA.%2BE.%26atitle%3DOlive%2520phenolics%2520as%2520c-Met%2520inhibitors%253A%2520%2528-%2529-Oleocanthal%2520attenuates%2520cell%2520proliferation%252C%2520invasiveness%252C%2520and%2520tumor%2520growth%2520in%2520breast%2520cancer%2520models%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De97622%26doi%3D10.1371%2Fjournal.pone.0097622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="note"><p class="first last">Annexin V-FITC Apoptosis Staining/Detection Kit (ab14085).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abd Elmageed, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsumagari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sholl, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedlander, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Qurayshi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulares, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandil, E.</span></span> <span> </span><span class="NLM_article-title">Prognostic role of BRAF(V600E) cellular localization in melanoma</span>. <i>J. Am. Coll. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">526</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1016/j.jamcollsurg.2017.12.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jamcollsurg.2017.12.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29369798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1Mvkt12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=2018&pages=526-537&author=Z.+Y.+Abd+Elmageedauthor=R.+F.+Mooreauthor=K.+Tsumagariauthor=M.+M.+Leeauthor=A.+B.+Shollauthor=P.+Friedlanderauthor=Z.+Al-Qurayshiauthor=M.+Hassanauthor=A.+R.+Wangauthor=H.+A.+Boularesauthor=E.+Kandil&title=Prognostic+role+of+BRAF%28V600E%29+cellular+localization+in+melanoma&doi=10.1016%2Fj.jamcollsurg.2017.12.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic Role of BRAF(V600E) Cellular Localization in Melanoma</span></div><div class="casAuthors">Abd Elmageed Zakaria Y; Moore Robert F; Tsumagari Koji; Hassan Mohamed; Lee Michael M; Sholl Andrew B; Wang Alun R; Friedlander Paul; Al-Qurayshi Zaid; Boulares Hamid A; Kandil Emad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Surgeons</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">526-537</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Approximately half of cutaneous melanoma tissues harbor BRAF(V600E) mutations, resulting in a constitutive activation of the mitogen-activated protein kinase (MAPK) pathway.  Nuclear-cytoplasmic transport machinery is dysregulated in neoplastic cells and alters the key regulatory proteins that can lead to tumor progression and drug resistance.  The significance of nuclear localization of BRAF(V600E) has not been fully understood.  We examined the clinical significance of intracellular localization of BRAF(V600E) in cutaneous melanoma.  STUDY DESIGN:  Immunohistochemical analysis of BRAF(V600E) was performed on formalin-fixed, paraffin-embedded specimens of cutaneous melanoma (n = 91).  Staining intensity was graded in a blinded manner.  Correlations to clinical factors were analyzed by Fisher's exact test and 2-tailed t-test.  Localization of BRAF(V600E) was determined in melanoma cells, and we investigated their resistance to BRAF(V600E)-specific inhibitor according to nuclear localization in both in vitro and in vivo models.  RESULTS:  We included 91 patients, of whom 32% (29 of 91) had cytoplasmic BRAF(V600E).  Nuclear BRAF(V600E) was observed in 30% (27 of 91).  Overall, BRAF(V600E) expression correlated with TNM stage (p = 0.011), mitotic activity (p = 0.010), and ulceration (p = 0.045).  Nuclear BRAF(V600E) expression correlated with overall clinical stage (p < 0.001), tumor size (p < 0.001), regional lymph node (p < 0.017), depth of invasion (p = 0.005), Clark level (p < 0.001), mitotic activity (p < 0.001), ulceration (p < 0.001), and margin status (p = 0.017).  On a cellular level, BRAF(V600E) was identified in the nucleus, and its translocation was serum dependent.  Our in vitro and in vivo data revealed sequestration of BRAF(V600E) in the cytosol-sensitized resistant cells to vemurafenib; nuclear retention of BRAF(V600E) was associated with aggressiveness and drug resistance.  CONCLUSIONS:  Nuclear localization of BRAF(V600E) is associated with melanoma aggressiveness.  Further multi-institutional studies are warranted to confirm the clinical relevance of nuclear localization of BRAF(V600E).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOTtfrC41t5LGbPKh4T9eAfW6udTcc2ea6GWHmaZkafbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mvkt12itA%253D%253D&md5=4137b2119d30f486607f790a08134d02</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.jamcollsurg.2017.12.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jamcollsurg.2017.12.040%26sid%3Dliteratum%253Aachs%26aulast%3DAbd%2BElmageed%26aufirst%3DZ.%2BY.%26aulast%3DMoore%26aufirst%3DR.%2BF.%26aulast%3DTsumagari%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DSholl%26aufirst%3DA.%2BB.%26aulast%3DFriedlander%26aufirst%3DP.%26aulast%3DAl-Qurayshi%26aufirst%3DZ.%26aulast%3DHassan%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DA.%2BR.%26aulast%3DBoulares%26aufirst%3DH.%2BA.%26aulast%3DKandil%26aufirst%3DE.%26atitle%3DPrognostic%2520role%2520of%2520BRAF%2528V600E%2529%2520cellular%2520localization%2520in%2520melanoma%26jtitle%3DJ.%2520Am.%2520Coll.%2520Surg.%26date%3D2018%26volume%3D226%26spage%3D526%26epage%3D537%26doi%3D10.1016%2Fj.jamcollsurg.2017.12.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':[],'ref120':['cit120']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao-Jie  Wang</span>, <span class="hlFld-ContribAuthor ">Xinxin  Guo</span>, <span class="hlFld-ContribAuthor ">Rui-Qin  Zhai</span>, <span class="hlFld-ContribAuthor ">Changning  Sun</span>, <span class="hlFld-ContribAuthor ">Guokai  Xiao</span>, <span class="hlFld-ContribAuthor ">Jin  Chen</span>, <span class="hlFld-ContribAuthor ">Mei-Yan  Wei</span>, <span class="hlFld-ContribAuthor ">Chang-Lun  Shao</span>, <span class="hlFld-ContribAuthor ">Yuchao  Gu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of penipanoid C-inspired 2-(3,4,5-trimethoxybenzoyl)quinazolin-4(3H)-one derivatives as potential anticancer agents by inhibiting cell proliferation and inducing apoptosis in hepatocellular carcinoma cells. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>224 </em>, 113671. <a href="https://doi.org/10.1016/j.ejmech.2021.113671" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113671%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bpenipanoid%252BC-inspired%252B2-%2525283%25252C4%25252C5-trimethoxybenzoyl%252529quinazolin-4%2525283H%252529-one%252Bderivatives%252Bas%252Bpotential%252Banticancer%252Bagents%252Bby%252Binhibiting%252Bcell%252Bproliferation%252Band%252Binducing%252Bapoptosis%252Bin%252Bhepatocellular%252Bcarcinoma%252Bcells%26aulast%3DWang%26aufirst%3DChao-Jie%26date%3D2021%26volume%3D224%26spage%3D113671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaitlyn A.  Maffuid</span>, <span class="hlFld-ContribAuthor ">Maria  Koyioni</span>, <span class="hlFld-ContribAuthor ">Chad D.  Torrice</span>, <span class="hlFld-ContribAuthor ">William A.  Murphy</span>, <span class="hlFld-ContribAuthor ">Heemaja K.  Mewada</span>, <span class="hlFld-ContribAuthor ">Panayiotis A.  Koutentis</span>, <span class="hlFld-ContribAuthor ">Daniel J.  Crona</span>, <span class="hlFld-ContribAuthor ">Christopher R.M.  Asquith</span>. </span><span class="cited-content_cbyCitation_article-title">Design and evaluation of 1,2,3-dithiazoles and fused 1,2,4-dithiazines as anti-cancer agents. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>43 </em>, 128078. <a href="https://doi.org/10.1016/j.bmcl.2021.128078" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.128078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.128078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.128078%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDesign%252Band%252Bevaluation%252Bof%252B1%25252C2%25252C3-dithiazoles%252Band%252Bfused%252B1%25252C2%25252C4-dithiazines%252Bas%252Banti-cancer%252Bagents%26aulast%3DMaffuid%26aufirst%3DKaitlyn%2BA.%26date%3D2021%26volume%3D43%26spage%3D128078" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0025.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0025.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of EGFR/HER2 inhibitors into their corresponding kinases and the fragments used for their design.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Chemical structure for the FDA-approved EGFR and dual EGFR/HER2 kinase inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Docking mode of lapatinib (ball and sticks) and TAK285 (orange lines) into the HER2 kinase domain (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). Hydrophobic region 1 (L726, A751, L800, G804, L852, and M801); hydrophobic selectivity pocket (M774, L785, L796, T798, D863, and F864); solvent-accessible region for TAK285 (G727, S728, and V734); solvent-accessible region for lapatinib (C805 and D808); α-C helix (P761-V777), and the gatekeeper amino acid (T798).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Rational design enhanced using CADD of the newly synthesized compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Proposed mechanism of Dimroth rearrangement for the preparation of 6-nitroquinazoline derivative (<b>7</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0019.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0019.jpeg" id="rightTab-gr18" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 2-Chloro-4-nitro-1-(3-(substituted)phenoxy/benzyloxy)benzene (<b>3a–d</b>) and 3-Chloro-4-(3-(substituted)phenoxy/benzyloxy)aniline (<b>4a–d</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0019.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF, K<sub>2</sub>CO<sub>3</sub>, 85 °C, 3 h; (b) Fe, CaCl<sub>2</sub>, EtOH, H<sub>2</sub>O, 80 °C, 8 h; (c) acetonitrile, K<sub>2</sub>CO<sub>3</sub>, 85 °C, 5 h; (d) SnCl<sub>2</sub>, MeOH, reflux, under N<sub>2</sub>, 2.5 h.</p></p></figure><figure data-id="sch2" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0020.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0020.jpeg" id="rightTab-gr19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(4-((3-Chloro-4-((3-fluorobenzyl)oxy)phenyl)amino)quinazolin-6-yl)-2-Substituted Acetamide (<b>10a–c</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0020.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) DMF-DMA, reflux 2 h; (b) glacial HOAc, reflux, 3 h; (c) Fe, CaCl<sub>2</sub>, EtOH, H<sub>2</sub>O, 90 °C, 12 h; (d) ClCH<sub>2</sub>COCl, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 1 h; (e) ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h or appropriate secondary amine (50 equiv), reflux, 2 h.</p></p></figure><figure data-id="sch3" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0021.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0021.jpeg" id="rightTab-gr20" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Target Compounds <b>15a–l</b> and <b>17a–d</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0021.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) triethylorthoformate, (Ac)<sub>2</sub>O, reflux, 30 h; (b) glacial HOAc, reflux, 7 h; (c) SnCl<sub>2</sub>, MeOH, reflux, under N<sub>2</sub>, 1.5 h; (d) ClCH<sub>2</sub>COCl, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 1 h; (e) appropriate secondary amine (50 equiv), reflux, 2 h or ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h; (f) using <b>13a</b>, 4-(chloromethyl)benzoyl chloride, NaHCO<sub>3</sub>, acetone, 0 °C, under N<sub>2</sub>, 0.5 h; (g) excess appropriate secondary amine (50 equiv), reflux, 2 h or ethanolamine (5 equiv), MeOH, TEA, reflux, 4 h.</p></p></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Two pathways conducted for the preparation of 6-nitroquinazoline derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0022.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0022.jpeg" id="rightTab-gr21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of the Target Compounds <b>25a–h</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NH<sub>2</sub>CHO, 190 °C, 1 h; (b) AC<sub>2</sub>O, pyridine, 100 °C, 2 h; (c) SOCl<sub>2</sub>, DMF, 95 °C, 5 h; (d) acetonitrile, K<sub>2</sub>CO<sub>3</sub>, rt, 8 h (with <b>4a</b>); acetonitrile, K<sub>2</sub>CO<sub>3</sub>, reflux, 12 h (with <b>4b</b> or <b>4c</b>); (e) NH<sub>4</sub>OH, MeOH, rt, 2 h, (f) Cl(CH<sub>2</sub>)<sub>3</sub>Br, K<sub>2</sub>CO<sub>3</sub>, TBAB, acetonitrile, reflux, 2 h; (g) appropriate amine (100 equiv), reflux, 2 h or ethanolamine (10 equiv), MeOH, TEA, reflux, 8 h.</p></p></figure><figure data-id="fig7" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Protein kinase profiling for compounds (<b>10a</b>, <b>10b</b>, <b>15a</b>, <b>15e</b>, <b>15l</b>, <b>17d</b>, and <b>25f</b>) at 10 μM single dose concentration, represented by % activity or % inhibition, against a panel of 20 kinases in comparison with two reference compounds lapatinib and sorafenib using the [γ-<sup>32</sup>P]ATP radiometric assay method. (−) values: inhibition of target activity; (+) values: activation of target activity; >−25% changes in activity compared to the control were considered to be significant inhibition.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Docking of compound <b>17d</b> (ball and stick, colored by element) into human HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>) within an rmsd of 3.1 Å from the native cocrystallized ligand (03Q). It exhibited two hydrogen bonds (green-dotted lines) with M801 and S728 in addition to multiple hydrophobic interactions. The color codes are hydrophobic (π–alkyl and alkyl–alkyl): ---; hydrophobic (π–π): ---; hydrophobic (π–σ): ---; halogen acceptor bonding: --- <i>m</i>-chloro atom interacts hydrophobically by a halogen acceptor with L796 and K753 residues.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. % IC<sub>50</sub> graph and calculations for the five compounds (<b>10a</b>, <b>10c</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b>) and lapatinib tested against HER2 wild-type kinase using the ADP-Glo assay method at (A) concentrations 0.01–25.0 μM and (B) concentrations 0.5–100 nM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Top disease cases with HER2 L755S (data retrieved from refs <a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> and <a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Dose–response curves for cell viability by WST-8 assay of AU565 and BT474 cells and their corresponding IC<sub>50</sub> values (standard error are given). Cells were treated with compounds <b>10c</b>, <b>15d</b>, <b>15i</b>, <b>17d</b>, and <b>25b</b> at concentrations 0.01–100 μM for 72 h. Compounds were tested in three independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Docking of compound <b>25b</b> (ball and stick, colored by element) into human HER2 kinase (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). It exhibited one hydrogen bond (green dotted lines) with M801. It was docked within an rmsd of 1.10 Å from the native cocrystallized 03Q ligand (shown as yellow sticks).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. Effect of optimized compounds on the HER2 signaling pathway in BC cells. AU565 (A) and BT474 (B) cells were treated with compounds <b>15d</b>, <b>15i</b>, and <b>25b</b> at concentrations of their IC<sub>50</sub>, 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> for each compound at 24 and 72 h. Lapatinib was used as an authenticated control for the test compounds. Western blot analysis was performed and GAPDH was used as a reference loading control. Data represented results collected from at least two independent experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig14" data-index="18" class="article__inlineFigure"><h2 class="fig-label">Figure 14</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0015.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 14. Western blot analysis for compound <b>17d</b> to evaluate the HER2 signaling pathway in BC cells (BT-474). The cells were treated with different concentrations at IC<sub>50</sub> (0.204 μM), 0.5 IC<sub>50</sub>, and 0.1 IC<sub>50</sub> at 24 and 48 h. GAPDH was used as a loading control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig14"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig15" data-index="19" class="article__inlineFigure"><h2 class="fig-label">Figure 15</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0016.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0016.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 15. Effect of compounds <b>17d</b> and <b>25b</b> on tumor growth of BT474 cells in the preclinical mouse model. (A) Represented images of mice implanted with 2 × 10<sup>6</sup> BT474 cells on the dorsal flank. After growing of the nodules, mice were randomized 5 mice per group and received the vehicle and compound <b>17d</b> and <b>25b</b>, respectively; then, all mice were sacrificed 6 weeks after the initial treatment. The remarkable metastases in the vehicle group are pointed out by red arrows. At the end of the experiment, tumor tissues were resected and preserved. (B) Tumor growth was measured twice a week for 6 weeks after treatments. (C,D) Whole-body weight per each mouse was recorded twice a week and the resected tumors were weighed per each animal. (E) Formalin-fixed paraffin-embedded xenograft tissues were stained with hematoxylin (H) and eosin (E), and proliferative index Ki67 for the three groups as indicated. Significance considered at <i>p</i> < 0.05 regarding mice that received vehicle. The magnification for H&E and Ki67 staining was 200× and for inserts was 400×.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0016.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig15"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig16" data-index="20" class="article__inlineFigure"><h2 class="fig-label">Figure 16</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0017.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0017.jpeg" id="rightTab-gr16" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 16. Optimization of the docking protocol of the cocrystallized ligand (03Q) into HER2 kinase (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a>). The five docked trials are labeled (1–5) in comparison with the native bound ligand (03Q, shown as yellow sticks) in which trial 4 (shown as ball and sticks) utilizing M801 as a flexible residue with 50,000 operations exhibited the best-optimized protocol with an rmsd of 1.54 Å and four hydrogen bonds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig16"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig17" data-index="21" class="article__inlineFigure"><h2 class="fig-label">Figure 17</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/medium/jm0c01647_0018.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0018.jpeg" id="rightTab-gr17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 17. SAR studies of the newly synthesized 4,6 disubstituted quinazolines.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2020/jmcmar.2020.63.issue-24/acs.jmedchem.0c01647/20201218/images/large/jm0c01647_0018.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c01647&amp;id=fig17"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i135">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_11722" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_11722" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 120 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fitzmaurice, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dicker, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pain, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamavid, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moradi-Lakeh, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacIntyre, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodbrook, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfe, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamadeh, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werdecker, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gessner, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Te Ao, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karimkhani, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schwebel, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carpenter, D. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nash, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kazi, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Leo, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plass, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ukwaja, K. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thurston, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yun Jin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, E.-K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catalá-López, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">deVeber, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gotay, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hosgood, H. D.,  3rd</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, I. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leasher, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leigh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jonas, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanabria, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beardsley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobsen, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takahashi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franklin, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ronfani, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montico, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naldi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geleijnse, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petzold, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrime, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Younis, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yonemoto, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitborde, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yip, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pourmalek, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lotufo, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteghamati, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hankey, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lunevicius, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malekzadeh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dellavalle, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weintraub, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lucas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojas-Rueda, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westerman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sepanlou, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nolte, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patten, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weichenthal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abera, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fereshtehnejad, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiue, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Driscoll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasankari, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsharif, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahimi-Movaghar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vlassov, V. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcenes, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mekonnen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melaku, Y. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Artaman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campos, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLachlan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mueller, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trillini, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eshrati, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shibuya, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandona, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murthy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowie, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amare, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antonio, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castañeda-Orjuela, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Gool, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Violante, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, I.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deribe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soreide, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knibbs, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kereselidze, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cardenas, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tillmann, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monasta, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheikhbahaei, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hafezi-Nejad, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreeramareddy, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandona, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vollset, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mokdad, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salomon, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lozano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vos, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forouzanfar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naghavi, M.</span></span> <span> </span><span class="NLM_article-title">The global burden of cancer 2013</span>. <i>JAMA Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">505</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1001/jamaoncol.2015.0735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1001%2Fjamaoncol.2015.0735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26181261" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC28%252Fkt1Wksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=505-527&author=C.+Fitzmauriceauthor=D.+Dickerauthor=A.+Painauthor=H.+Hamavidauthor=M.+Moradi-Lakehauthor=M.+F.+MacIntyreauthor=C.+Allenauthor=G.+Hansenauthor=R.+Woodbrookauthor=C.+Wolfeauthor=R.+R.+Hamadehauthor=A.+Mooreauthor=A.+Werdeckerauthor=B.+D.+Gessnerauthor=B.+Te+Aoauthor=B.+McMahonauthor=C.+Karimkhaniauthor=C.+Yuauthor=G.+S.+Cookeauthor=D.+C.+Schwebelauthor=D.+O.+Carpenterauthor=D.+M.+Pereiraauthor=D.+Nashauthor=D.+S.+Kaziauthor=D.+De+Leoauthor=D.+Plassauthor=K.+N.+Ukwajaauthor=G.+D.+Thurstonauthor=K.+Yun+Jinauthor=E.+P.+Simardauthor=E.+Millsauthor=E.-K.+Parkauthor=F.+Catal%C3%A1-L%C3%B3pezauthor=G.+deVeberauthor=C.+Gotayauthor=G.+Khanauthor=H.+D.+Hosgoodauthor=I.+S.+Santosauthor=J.+L.+Leasherauthor=J.+Singhauthor=J.+Leighauthor=J.+B.+Jonasauthor=J.+Sanabriaauthor=J.+Beardsleyauthor=K.+H.+Jacobsenauthor=K.+Takahashiauthor=R.+C.+Franklinauthor=L.+Ronfaniauthor=M.+Monticoauthor=L.+Naldiauthor=M.+Tonelliauthor=J.+Geleijnseauthor=M.+Petzoldauthor=M.+G.+Shrimeauthor=M.+Younisauthor=N.+Yonemotoauthor=N.+Breitbordeauthor=P.+Yipauthor=F.+Pourmalekauthor=P.+A.+Lotufoauthor=A.+Esteghamatiauthor=G.+J.+Hankeyauthor=R.+Aliauthor=R.+Luneviciusauthor=R.+Malekzadehauthor=R.+Dellavalleauthor=R.+Weintraubauthor=R.+Lucasauthor=R.+Hayauthor=D.+Rojas-Ruedaauthor=R.+Westermanauthor=S.+G.+Sepanlouauthor=S.+Nolteauthor=S.+Pattenauthor=S.+Weichenthalauthor=S.+F.+Aberaauthor=S.-M.+Fereshtehnejadauthor=I.+Shiueauthor=T.+Driscollauthor=T.+Vasankariauthor=U.+Alsharifauthor=V.+Rahimi-Movagharauthor=V.+V.+Vlassovauthor=W.+S.+Marcenesauthor=W.+Mekonnenauthor=Y.+A.+Melakuauthor=Y.+Yanoauthor=A.+Artamanauthor=I.+Camposauthor=J.+MacLachlanauthor=U.+Muellerauthor=D.+Kimauthor=M.+Trilliniauthor=B.+Eshratiauthor=H.+C.+Williamsauthor=K.+Shibuyaauthor=R.+Dandonaauthor=K.+Murthyauthor=B.+Cowieauthor=A.+T.+Amareauthor=C.+A.+Antonioauthor=C.+Casta%C3%B1eda-Orjuelaauthor=C.+H.+van+Goolauthor=F.+Violanteauthor=I.-H.+Ohauthor=K.+Deribeauthor=K.+Soreideauthor=L.+Knibbsauthor=M.+Kereselidzeauthor=M.+Greenauthor=R.+Cardenasauthor=N.+Royauthor=T.+Tillmannauthor=Y.+Liauthor=H.+Kruegerauthor=L.+Monastaauthor=S.+Deyauthor=S.+Sheikhbahaeiauthor=N.+Hafezi-Nejadauthor=G.+A.+Kumarauthor=C.+T.+Sreeramareddyauthor=L.+Dandonaauthor=H.+Wangauthor=S.+E.+Vollsetauthor=A.+Mokdadauthor=J.+A.+Salomonauthor=R.+Lozanoauthor=T.+Vosauthor=M.+Forouzanfarauthor=A.+Lopezauthor=C.+Murrayauthor=M.+Naghavi&title=The+global+burden+of+cancer+2013&doi=10.1001%2Fjamaoncol.2015.0735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The Global Burden of Cancer 2013</span></div><div class="casAuthors">Jonas Jost; Tillman Taavi; Fitzmaurice Christina; Dicker Daniel; Pain Amanda; Hamavid Hannah; MacIntyre Michael F; Allen Christine; Hansen Gillian; Woodbrook Rachel; Wang Haidong; Mokdad Ali; Vos Theo; Forouzanfar Mohammad; Murray Christopher; Naghavi Mohsen; Moradi-Lakeh Maziar; Wolfe Charles; Hamadeh Randah R; Moore Ami; Werdecker Andrea; Gessner Bradford D; Te Ao Braden; McMahon Brian; Karimkhani Chante; Yu Chuanhua; Cooke Graham S; Schwebel David C; Carpenter David O; Pereira David M; Nash Denis; Kazi Dhruv S; De Leo Diego; Plass Dietrich; Ukwaja Kingsley N; Thurston George D; Yun Jin Kim; Simard Edgar P; Mills Edward; Park Eun-Kee; Catala-Lopez Ferran; deVeber Gabrielle; Gotay Carolyn; Krueger Hans; Khan Gulfaraz; Hosgood H Dean 3rd; Santos Itamar S; Lotufo Paulo A; Leasher Janet L; Singh Jasvinder; Leigh James; Jonas Jost B; Sanabria Juan; Beardsley Justin; Jacobsen Kathryn H; Takahashi Ken; Franklin Richard C; Ronfani Luca; Montico Marcella; Monasta Lorenzo; Naldi Luigi; Tonelli Marcello; Geleijnse Johanna; Petzold Max; Shrime Mark G; Campos Ismael; Salomon Joshua A; Younis Mustafa; Yonemoto Naohiro; Breitborde Nicholas; Yip Paul; Pourmalek Farshad; Esteghamati Alireza; Sheikhbahaei Sara; Hafezi-Nejad Nima; Hankey Graeme J; Ali Raghib; Lunevicius Raimundas; Malekzadeh Reza; Dellavalle Robert; Weintraub Robert; Lucas Robyn; Hay Roderick; Rojas-Rueda David; Westerman Ronny; Sepanlou Sadaf G; Nolte Sandra; Patten Scott; Weichenthal Scott; Abera Semaw Ferede; Fereshtehnejad Seyed-Mohammad; Shiue Ivy; Driscoll Tim; Vasankari Tommi; Alsharif Ubai; Rahimi-Movaghar Vafa; Vlassov Vasiliy V; Marcenes W S; Mekonnen Wubegzier; Melaku Yohannes Adama; Yano Yuichiro; Artaman Al; MacLachlan Jennifer; Cowie Benjamin; Mueller Ulrich; Kim Daniel; Trillini Matias; Eshrati Babak; Williams Hywel C; Shibuya Kenji; Dandona Rakhi; Murthy Kinnari; Kumar G Anil; Amare Azmeraw T; Antonio Carl Abelardo; Castaneda-Orjuela Carlos; van Gool Coen H; Violante Francesco; Oh In-Hwan; Deribe Kedede; Soreide Kjetil; Knibbs Luke; Kereselidze Maia; Green Mark; Cardenas Rosario; Roy Nobhojit; Tillmann Taavi; Li Yongmei; Dey Subhojit; Sreeramareddy Chandrashekhar T; Dandona Lalit; Vollset Stein Emil; Lozano Rafael; Lopez Alan</div><div class="citationInfo"><span class="NLM_cas:title">JAMA oncology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">505-27</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">IMPORTANCE:  Cancer is among the leading causes of death worldwide.  Current estimates of cancer burden in individual countries and regions are necessary to inform local cancer control strategies.  OBJECTIVE:  To estimate mortality, incidence, years lived with disability (YLDs), years of life lost (YLLs), and disability-adjusted life-years (DALYs) for 28 cancers in 188 countries by sex from 1990 to 2013.  EVIDENCE REVIEW:  The general methodology of the Global Burden of Disease (GBD) 2013 study was used.  Cancer registries were the source for cancer incidence data as well as mortality incidence (MI) ratios.  Sources for cause of death data include vital registration system data, verbal autopsy studies, and other sources.  The MI ratios were used to transform incidence data to mortality estimates and cause of death estimates to incidence estimates.  Cancer prevalence was estimated using MI ratios as surrogates for survival data; YLDs were calculated by multiplying prevalence estimates with disability weights, which were derived from population-based surveys; YLLs were computed by multiplying the number of estimated cancer deaths at each age with a reference life expectancy; and DALYs were calculated as the sum of YLDs and YLLs.  FINDINGS:  In 2013 there were 14.9 million incident cancer cases, 8.2 million deaths, and 196.3 million DALYs.  Prostate cancer was the leading cause for cancer incidence (1.4 million) for men and breast cancer for women (1.8 million).  Tracheal, bronchus, and lung (TBL) cancer was the leading cause for cancer death in men and women, with 1.6 million deaths.  For men, TBL cancer was the leading cause of DALYs (24.9 million).  For women, breast cancer was the leading cause of DALYs (13.1 million).  Age-standardized incidence rates (ASIRs) per 100 000 and age-standardized death rates (ASDRs) per 100 000 for both sexes in 2013 were higher in developing vs developed countries for stomach cancer (ASIR, 17 vs 14; ASDR, 15 vs 11), liver cancer (ASIR, 15 vs 7; ASDR, 16 vs 7), esophageal cancer (ASIR, 9 vs 4; ASDR, 9 vs 4), cervical cancer (ASIR, 8 vs 5; ASDR, 4 vs 2), lip and oral cavity cancer (ASIR, 7 vs 6; ASDR, 2 vs 2), and nasopharyngeal cancer (ASIR, 1.5 vs 0.4; ASDR, 1.2 vs 0.3).  Between 1990 and 2013, ASIRs for all cancers combined (except nonmelanoma skin cancer and Kaposi sarcoma) increased by more than 10% in 113 countries and decreased by more than 10% in 12 of 188 countries.  CONCLUSIONS AND RELEVANCE:  Cancer poses a major threat to public health worldwide, and incidence rates have increased in most countries since 1990.  The trend is a particular threat to developing nations with health systems that are ill-equipped to deal with complex and expensive cancer treatments.  The annual update on the Global Burden of Cancer will provide all stakeholders with timely estimates to guide policy efforts in cancer prevention, screening, treatment, and palliation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQIAyANjE4GzBfKb6ELK8t0fW6udTcc2ea9yIaicQgyPbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28%252Fkt1Wksw%253D%253D&md5=7466761111f16c35c0953901258d0587</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1001%2Fjamaoncol.2015.0735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1001%252Fjamaoncol.2015.0735%26sid%3Dliteratum%253Aachs%26aulast%3DFitzmaurice%26aufirst%3DC.%26aulast%3DDicker%26aufirst%3DD.%26aulast%3DPain%26aufirst%3DA.%26aulast%3DHamavid%26aufirst%3DH.%26aulast%3DMoradi-Lakeh%26aufirst%3DM.%26aulast%3DMacIntyre%26aufirst%3DM.%2BF.%26aulast%3DAllen%26aufirst%3DC.%26aulast%3DHansen%26aufirst%3DG.%26aulast%3DWoodbrook%26aufirst%3DR.%26aulast%3DWolfe%26aufirst%3DC.%26aulast%3DHamadeh%26aufirst%3DR.%2BR.%26aulast%3DMoore%26aufirst%3DA.%26aulast%3DWerdecker%26aufirst%3DA.%26aulast%3DGessner%26aufirst%3DB.%2BD.%26aulast%3DTe%2BAo%26aufirst%3DB.%26aulast%3DMcMahon%26aufirst%3DB.%26aulast%3DKarimkhani%26aufirst%3DC.%26aulast%3DYu%26aufirst%3DC.%26aulast%3DCooke%26aufirst%3DG.%2BS.%26aulast%3DSchwebel%26aufirst%3DD.%2BC.%26aulast%3DCarpenter%26aufirst%3DD.%2BO.%26aulast%3DPereira%26aufirst%3DD.%2BM.%26aulast%3DNash%26aufirst%3DD.%26aulast%3DKazi%26aufirst%3DD.%2BS.%26aulast%3DDe%2BLeo%26aufirst%3DD.%26aulast%3DPlass%26aufirst%3DD.%26aulast%3DUkwaja%26aufirst%3DK.%2BN.%26aulast%3DThurston%26aufirst%3DG.%2BD.%26aulast%3DYun%2BJin%26aufirst%3DK.%26aulast%3DSimard%26aufirst%3DE.%2BP.%26aulast%3DMills%26aufirst%3DE.%26aulast%3DPark%26aufirst%3DE.-K.%26aulast%3DCatal%25C3%25A1-L%25C3%25B3pez%26aufirst%3DF.%26aulast%3DdeVeber%26aufirst%3DG.%26aulast%3DGotay%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DG.%26aulast%3DHosgood%26aufirst%3DH.%2BD.%26aulast%3DSantos%26aufirst%3DI.%2BS.%26aulast%3DLeasher%26aufirst%3DJ.%2BL.%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DLeigh%26aufirst%3DJ.%26aulast%3DJonas%26aufirst%3DJ.%2BB.%26aulast%3DSanabria%26aufirst%3DJ.%26aulast%3DBeardsley%26aufirst%3DJ.%26aulast%3DJacobsen%26aufirst%3DK.%2BH.%26aulast%3DTakahashi%26aufirst%3DK.%26aulast%3DFranklin%26aufirst%3DR.%2BC.%26aulast%3DRonfani%26aufirst%3DL.%26aulast%3DMontico%26aufirst%3DM.%26aulast%3DNaldi%26aufirst%3DL.%26aulast%3DTonelli%26aufirst%3DM.%26aulast%3DGeleijnse%26aufirst%3DJ.%26aulast%3DPetzold%26aufirst%3DM.%26aulast%3DShrime%26aufirst%3DM.%2BG.%26aulast%3DYounis%26aufirst%3DM.%26aulast%3DYonemoto%26aufirst%3DN.%26aulast%3DBreitborde%26aufirst%3DN.%26aulast%3DYip%26aufirst%3DP.%26aulast%3DPourmalek%26aufirst%3DF.%26aulast%3DLotufo%26aufirst%3DP.%2BA.%26aulast%3DEsteghamati%26aufirst%3DA.%26aulast%3DHankey%26aufirst%3DG.%2BJ.%26aulast%3DAli%26aufirst%3DR.%26aulast%3DLunevicius%26aufirst%3DR.%26aulast%3DMalekzadeh%26aufirst%3DR.%26aulast%3DDellavalle%26aufirst%3DR.%26aulast%3DWeintraub%26aufirst%3DR.%26aulast%3DLucas%26aufirst%3DR.%26aulast%3DHay%26aufirst%3DR.%26aulast%3DRojas-Rueda%26aufirst%3DD.%26aulast%3DWesterman%26aufirst%3DR.%26aulast%3DSepanlou%26aufirst%3DS.%2BG.%26aulast%3DNolte%26aufirst%3DS.%26aulast%3DPatten%26aufirst%3DS.%26aulast%3DWeichenthal%26aufirst%3DS.%26aulast%3DAbera%26aufirst%3DS.%2BF.%26aulast%3DFereshtehnejad%26aufirst%3DS.-M.%26aulast%3DShiue%26aufirst%3DI.%26aulast%3DDriscoll%26aufirst%3DT.%26aulast%3DVasankari%26aufirst%3DT.%26aulast%3DAlsharif%26aufirst%3DU.%26aulast%3DRahimi-Movaghar%26aufirst%3DV.%26aulast%3DVlassov%26aufirst%3DV.%2BV.%26aulast%3DMarcenes%26aufirst%3DW.%2BS.%26aulast%3DMekonnen%26aufirst%3DW.%26aulast%3DMelaku%26aufirst%3DY.%2BA.%26aulast%3DYano%26aufirst%3DY.%26aulast%3DArtaman%26aufirst%3DA.%26aulast%3DCampos%26aufirst%3DI.%26aulast%3DMacLachlan%26aufirst%3DJ.%26aulast%3DMueller%26aufirst%3DU.%26aulast%3DKim%26aufirst%3DD.%26aulast%3DTrillini%26aufirst%3DM.%26aulast%3DEshrati%26aufirst%3DB.%26aulast%3DWilliams%26aufirst%3DH.%2BC.%26aulast%3DShibuya%26aufirst%3DK.%26aulast%3DDandona%26aufirst%3DR.%26aulast%3DMurthy%26aufirst%3DK.%26aulast%3DCowie%26aufirst%3DB.%26aulast%3DAmare%26aufirst%3DA.%2BT.%26aulast%3DAntonio%26aufirst%3DC.%2BA.%26aulast%3DCasta%25C3%25B1eda-Orjuela%26aufirst%3DC.%26aulast%3Dvan%2BGool%26aufirst%3DC.%2BH.%26aulast%3DViolante%26aufirst%3DF.%26aulast%3DOh%26aufirst%3DI.-H.%26aulast%3DDeribe%26aufirst%3DK.%26aulast%3DSoreide%26aufirst%3DK.%26aulast%3DKnibbs%26aufirst%3DL.%26aulast%3DKereselidze%26aufirst%3DM.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DCardenas%26aufirst%3DR.%26aulast%3DRoy%26aufirst%3DN.%26aulast%3DTillmann%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DKrueger%26aufirst%3DH.%26aulast%3DMonasta%26aufirst%3DL.%26aulast%3DDey%26aufirst%3DS.%26aulast%3DSheikhbahaei%26aufirst%3DS.%26aulast%3DHafezi-Nejad%26aufirst%3DN.%26aulast%3DKumar%26aufirst%3DG.%2BA.%26aulast%3DSreeramareddy%26aufirst%3DC.%2BT.%26aulast%3DDandona%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DVollset%26aufirst%3DS.%2BE.%26aulast%3DMokdad%26aufirst%3DA.%26aulast%3DSalomon%26aufirst%3DJ.%2BA.%26aulast%3DLozano%26aufirst%3DR.%26aulast%3DVos%26aufirst%3DT.%26aulast%3DForouzanfar%26aufirst%3DM.%26aulast%3DLopez%26aufirst%3DA.%26aulast%3DMurray%26aufirst%3DC.%26aulast%3DNaghavi%26aufirst%3DM.%26atitle%3DThe%2520global%2520burden%2520of%2520cancer%25202013%26jtitle%3DJAMA%2520Oncol.%26date%3D2015%26volume%3D1%26spage%3D505%26epage%3D527%26doi%3D10.1001%2Fjamaoncol.2015.0735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Heron, M.</span></span> <span> </span><span class="NLM_article-title">Deaths : leading causes for 2008</span>. <i>Natl. Vital Stat. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">94</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22827019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC38fisFOjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=1-94&author=M.+Heron&title=Deaths+%3A+leading+causes+for+2008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Deaths: leading causes for 2008</span></div><div class="casAuthors">Heron Melonie</div><div class="citationInfo"><span class="NLM_cas:title">National vital statistics reports : from the Centers for Disease Control and Prevention, National Center for Health Statistics, National Vital Statistics System</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1-94</span>
        ISSN:<span class="NLM_cas:issn">1551-8922</span>.
    </div><div class="casAbstract">OBJECTIVES:  This report presents final 2008 data on the 10 leading causes of death in the United States by age, sex, race, and Hispanic origin.  Leading causes of infant, neonatal, and postneonatal death are also presented.  This report supplements the Division of Vital Statistics' annual report of final mortality statistics.  METHODS:  Data in this report are based on information from all death certificates filed in the 50 states and the District of Columbia in 2008.  Causes of death classified by the International Classification of Diseases, Tenth Revision (ICD-10) are ranked according to the number of deaths assigned to rankable causes.  Cause-of-death statistics are based on the underlying cause of death.  RESULTS:  in 2008, the 10 leading causes of death were, in rank order: Diseases of heart; Malignant neoplasms; Chronic lower respiratory diseases; Cerebrovascular diseases; Accidents (unintentional injuries); Alzheimer's disease; Diabetes mellitus; Influenza and pneumonia; Nephritis, nephrotic syndrome and nephrosis; and Intentional self-harm (suicide).  They accounted for approximately 76 percent of all deaths occurring in the United States.  Differences in the rankings are evident by age, sex, race, and Hispanic origin.  Leading causes of infant death for 2008 were, in rank order: Congenital malformations, deformations and chromosomal abnormalities; Disorders related to short gestation and low birth weight, not elsewhere classified; Sudden infant death syndrome; Newborn affected by maternal complications of pregnancy; Accidents (unintentional injuries); Newborn affected by complications of placenta, cord and membranes; Bacterial sepsis of newborn; Respiratory distress of newborn; Diseases of the circulatory system; and Neonatal hemorrhage.  Important variations in the leading causes of infant death are noted for the neonatal and postneonatal periods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRRrmI3hG59C2Zyxfu64m4ffW6udTcc2eY_n8kI2LUwn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38fisFOjsQ%253D%253D&md5=024b685faf95755d47dc8f80ffce28a3</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHeron%26aufirst%3DM.%26atitle%3DDeaths%2520%253A%2520leading%2520causes%2520for%25202008%26jtitle%3DNatl.%2520Vital%2520Stat.%2520Rep.%26date%3D2012%26volume%3D60%26spage%3D1%26epage%3D94" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fekry, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ezzat, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salama, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alshehri, O. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span> <span> </span><span class="NLM_article-title">Bioactive glycoalkaloides isolated from Solanum melongena fruit peels with potential anticancer properties against hepatocellular carcinoma cells</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1746</span>– <span class="NLM_lpage">1756</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-36089-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-36089-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30741973" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3cfht1GmtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=1746-1756&author=M.+I.+Fekryauthor=S.+M.+Ezzatauthor=M.+M.+Salamaauthor=O.+Y.+Alshehriauthor=A.+M.+Al-Abd&title=Bioactive+glycoalkaloides+isolated+from+Solanum+melongena+fruit+peels+with+potential+anticancer+properties+against+hepatocellular+carcinoma+cells&doi=10.1038%2Fs41598-018-36089-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Bioactive glycoalkaloides isolated from Solanum melongena fruit peels with potential anticancer properties against hepatocellular carcinoma cells</span></div><div class="casAuthors">Fekry Mostafa I; Ezzat Shahira M; Salama Maha M; Fekry Mostafa I; Ezzat Shahira M; Salama Maha M; Alshehri Ohoud Y; Al-Abd Ahmed M; Al-Abd Ahmed M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1746</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Hepatocellular carcinoma (HCC) is progressively increasing tumor with lack of accurate prognosis and inadequate systemic treatment approaches.  Solanum sp. (such as Solanum melongena) is a folk herb which is reported to possess anticancer properties.  In a continuity for our interest in pursuing the anticancer activity of compounds isolated from the fruit peels of Solanum melongena, the HPLC profiling and ESI-MS assessment for the methanolic extract evidenced the presence of bioactive glycoalkaloids (solasonine, solasodine and solamargine).  These glycoalkaloids were isolated, purified and proved to possess in vitro cytotoxicity against human liver cancer cell lines (Huh7 and HepG2).  Herein, we investigated the potential mechanism of action of these compounds using DNA content flow-cytometry and apoptosis/necrosis differential anaylsis using annexin-V/FITC staining.  Solasonine, solasodine and solamargine inducd significant antiproliferative effect against liver cancer cells (Huh7 and HepG2) which was attributed to cell cycle arrest at S-phase.  Solamargine, solasodine and solasonine induced significant apoptosis in Huh7 cells.  Only solamargine-induced cell cycle arrest, was reflected as apoptotic cell killing effect against HepG2 cells.  In conclusion, glycoalkaloids derived from Solanum melongena and particularly, solamargine are promising antiproliferative agents with potential anticancer effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRVXWE2RUz1qENoizSOkG_FfW6udTcc2eY_n8kI2LUwn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cfht1GmtA%253D%253D&md5=caa0c90a3d84a3a920915973a4736e58</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-36089-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-36089-6%26sid%3Dliteratum%253Aachs%26aulast%3DFekry%26aufirst%3DM.%2BI.%26aulast%3DEzzat%26aufirst%3DS.%2BM.%26aulast%3DSalama%26aufirst%3DM.%2BM.%26aulast%3DAlshehri%26aufirst%3DO.%2BY.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26atitle%3DBioactive%2520glycoalkaloides%2520isolated%2520from%2520Solanum%2520melongena%2520fruit%2520peels%2520with%2520potential%2520anticancer%2520properties%2520against%2520hepatocellular%2520carcinoma%2520cells%26jtitle%3DSci.%2520Rep.%26date%3D2019%26volume%3D9%26spage%3D1746%26epage%3D1756%26doi%3D10.1038%2Fs41598-018-36089-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bashmail, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alamoudi, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noorwali, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegazy, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choudhry, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span> <span> </span><span class="NLM_article-title">Thymoquinone synergizes gemcitabine anti-breast cancer activity via modulating its apoptotic and autophagic activities</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">11674</span>– <span class="NLM_lpage">11684</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-30046-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-30046-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30076320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3c7kvFOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=11674-11684&author=H.+A.+Bashmailauthor=A.+A.+Alamoudiauthor=A.+Noorwaliauthor=G.+A.+Hegazyauthor=H.+Choudhryauthor=A.+M.+Al-Abd&title=Thymoquinone+synergizes+gemcitabine+anti-breast+cancer+activity+via+modulating+its+apoptotic+and+autophagic+activities&doi=10.1038%2Fs41598-018-30046-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Thymoquinone synergizes gemcitabine anti-breast cancer activity via modulating its apoptotic and autophagic activities</span></div><div class="casAuthors">Bashmail Hanan A; Alamoudi Aliaa A; Noorwali Abdulwahab; Hegazy Gehan A; AJabnoor Ghada; Alamoudi Aliaa A; Noorwali Abdulwahab; Hegazy Gehan A; Choudhry Hani; Al-Abd Ahmed M; Al-Abd Ahmed M; Al-Abd Ahmed M</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">11674</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of anti-cancer adjuvant therapy is rationalized by potentiating the efficacy, and/or protecting from major side effects of chemotherapeutics.  Thymoquinone (TQ) is a naturally occurring compound with cumulative evidence of anti-cancer properties.  In this study, we assessed the chemomodulatory potential of TQ to gemcitabine (GCB) against human breast adenocarcinoma (MCF-7), and ductal carcinoma (T47D) cells.  TQ showed cytotoxic effects against MCF-7 and T47D with IC50's of 64.9 ± 14 μM and 165 ± 2 μM, respectively.  The IC50's of GCB against MCF-7 and T47D were 0.9 ± 0.18 μM and 14.3 ± 2.8 μM and were significantly reduced after combination with TQ to 0.058 ± 12 μM and 2.3 ± 0.2 μM, respectively.  The CI- values were indicative of synergism in MCF-7 and T47D cells (0.15 and 0.30, respectively).  Further investigation showed that GCB caused significant anti-proliferative effect reflected by increasing cell population in S-phase in both cell lines.  TQ potentiated GCB-induced anti-proliferative activity in both cell lines.  GCB induced considerable apoptosis in T47D cell line, and TQ significantly increased GCB-induced apoptotic effects by 1.5 to 3.6 folds.  Interestingly, GCB, TQ and their combination induced significant autophagic cell death in the apoptosis defected MCF-7 cells.  In addition, TQ, GCB and their combination depleted breast cancer associated stem cell (CD44((+))/CD24((-)/(low))) clone within MCF-7 and T47D cells by 3.8% to 27.5%.  In conclusion, TQ showed promising chemomodulatory effects to GCB against breast cancer cells via inducing apoptosis, necrosis and autophagy, in addition to depleting tumor associated resistant stem cell fraction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTe4YmZ7rOGXN0Q2ypr7jHkfW6udTcc2eY_n8kI2LUwn7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c7kvFOlug%253D%253D&md5=5a30945cde5e3c49b7051e6b85577752</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-30046-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-30046-z%26sid%3Dliteratum%253Aachs%26aulast%3DBashmail%26aufirst%3DH.%2BA.%26aulast%3DAlamoudi%26aufirst%3DA.%2BA.%26aulast%3DNoorwali%26aufirst%3DA.%26aulast%3DHegazy%26aufirst%3DG.%2BA.%26aulast%3DChoudhry%26aufirst%3DH.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26atitle%3DThymoquinone%2520synergizes%2520gemcitabine%2520anti-breast%2520cancer%2520activity%2520via%2520modulating%2520its%2520apoptotic%2520and%2520autophagic%2520activities%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D11674%26epage%3D11684%26doi%3D10.1038%2Fs41598-018-30046-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abd, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Halawany, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdallah, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, S. R. M.</span></span> <span> </span><span class="NLM_article-title">New xanthones and cytotoxic constituents from Garcinia mangostana fruit hulls against human hepatocellular, breast, and colorectal cancer cell lines</span>. <i>J. Ethnopharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>198</i></span>,  <span class="NLM_fpage">302</span>– <span class="NLM_lpage">312</span>, <span class="refDoi"> DOI: 10.1016/j.jep.2017.01.030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jep.2017.01.030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28108382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=198&publication_year=2017&pages=302-312&author=G.+A.+Mohamedauthor=A.+M.+Al-Abdauthor=A.+M.+El-Halawanyauthor=H.+M.+Abdallahauthor=S.+R.+M.+Ibrahim&title=New+xanthones+and+cytotoxic+constituents+from+Garcinia+mangostana+fruit+hulls+against+human+hepatocellular%2C+breast%2C+and+colorectal+cancer+cell+lines&doi=10.1016%2Fj.jep.2017.01.030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">New xanthones and cytotoxic constituents from Garcinia mangostana fruit hulls against human hepatocellular, breast, and colorectal cancer cell lines</span></div><div class="casAuthors">Mohamed, Gamal A.; Al-Abd, Ahmed M.; El-Halawany, Ali M.; Abdallah, Hossam M.; Ibrahim, Sabrin R. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Ethnopharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">198</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">302-312</span>CODEN:
                <span class="NLM_cas:coden">JOETD7</span>;
        ISSN:<span class="NLM_cas:issn">0378-8741</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Cancer has proceeded to surpass one of the most chronic illnesses to be the major cause of mortality in both the developing and developed world.  Garcinia mangostana L. (mangosteen, family Guttiferae) known as the queen of fruits, is one of the most popular tropical fruits.  It is cultivated in Southeast Asian countries: Malaysia, Indonesia, Sri Lanka, Burma, Thailand, and Philippines.  Traditionally, numerous parts of G. mangostana have been utilized to treat various ailments such as abdominal pain, haemorrhoids, food allergies, arthritis, leucorrhoea, gonorrhea, diarrhea, dysentery, wound infection, suppuration, and chronic ulcer.  Although anticancer activity has been reported for the plant, the goal of the study was designed to isolate and characterize the active metabolites from G. mangostana and measure their cytotoxic properties.  In this research, the mechanism of antiproliferative/cytotoxic effects of the tested compds. was investigated.  The CHCl3 fraction of the air-dried fruit hulls was repeatedly chromatographed on SiO2, RP18, Diaion HP-20, and polyamide columns to furnish fourteen compds.  The structures of these metabolites were proven by UV, IR, 1D, and 2D NMR measurements and HRESIMS.  Addnl., the cytotoxic potential of all compds. was assessed against MCF-7, HCT-116, and HepG2 cell lines using SRB-U assay.  Antiproliferative and cell cycle interference effects of potentially potent compds. were tested using DNA content flow cytometry.  The mechanism of cell death induction was also studied using annexin-V/PI differential staining coupled with flow cytometry.  The CHCl3 sol. fraction afforded two new xanthones: mangostanaxanthones V (1) and VI (2), along with twelve known compds.: mangostanaxanthone IV (3), β-mangostin (4), garcinone E (5), α-mangostin (6), nor-mangostin (7), garcimangosone D (8), aromadendrin-8-C-β-D-glucopyranoside (9), 1,2,4,5-tetrahydroxybenzene (10), 2,4,3'-trihydroxybenzophenone-6-O-β-glucopyranoside (11), maclurin-6-O-β-D-glucopyranoside (rhodanthenone) (12), epicatechin (13), and 2,4,6,3',5'-pentahydroxybenzophenone (14).  Only compd. 5 showed considerable antiproliferative/cytotoxic effects with IC50's ranging from 15.8 to 16.7 μM.  Compds. 3, 4, and 6 showed moderate to weak cytotoxic effects (IC50's ranged from 45.7 to 116.4 μM).  Using DNA content flow cytometry, it was found that only 5 induced significant cell cycle arrest at G0/G1-phase which is indicative of its antiproliferative properties.  Addnl., by using annexin V-FITC/PI differential staining, 5 induced cells killing effect via the induction of apoptosis and necrosis in both HepG2 and HCT116 cells.  Compd. 3 produce necrosis and apoptosis only in HCT116 cells.  On contrary, 6 induced apoptosis and necrosis in HepG2 cells and moderate necrosis in HCT116 cells.  Fourteen compds. were isolated from chloroform fraction of G. mangostana fruit hulls.  Cytotoxic properties exhibited by the isolated xanthones from G. mangostana reinforce the avail of it as a natural cytotoxic agent against various cancers.  These evidences could provide relevant bases for the scientific rationale of using G. mangostana in anti-cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbDuWzpsHds7Vg90H21EOLACvtfcHk0liU89EQhjJhaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GktL0%253D&md5=30b3e0075ece49cdeb641a341bb29ffd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.jep.2017.01.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jep.2017.01.030%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DG.%2BA.%26aulast%3DAl-Abd%26aufirst%3DA.%2BM.%26aulast%3DEl-Halawany%26aufirst%3DA.%2BM.%26aulast%3DAbdallah%26aufirst%3DH.%2BM.%26aulast%3DIbrahim%26aufirst%3DS.%2BR.%2BM.%26atitle%3DNew%2520xanthones%2520and%2520cytotoxic%2520constituents%2520from%2520Garcinia%2520mangostana%2520fruit%2520hulls%2520against%2520human%2520hepatocellular%252C%2520breast%252C%2520and%2520colorectal%2520cancer%2520cell%2520lines%26jtitle%3DJ.%2520Ethnopharmacol.%26date%3D2017%26volume%3D198%26spage%3D302%26epage%3D312%26doi%3D10.1016%2Fj.jep.2017.01.030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapierre, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heys, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsit, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malone, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wintringer, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirillo, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Comas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farhat, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fowler, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kohl, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathys, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niemann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omar, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sintchenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soolingen, D. v.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Supply, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tahseen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilcox, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arandjelovic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peto, T. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crook, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iqbal, Z.</span></span> <span> </span><span class="NLM_article-title">Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe</span>. <i>Wellcome Open Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">191</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.12688/wellcomeopenres.15603.1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.12688%2Fwellcomeopenres.15603.1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=32055708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB387hvVOntA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2019&pages=191-222&author=M.+Huntauthor=P.+Bradleyauthor=S.+G.+Lapierreauthor=S.+Heysauthor=M.+Thomsitauthor=M.+B.+Hallauthor=K.+M.+Maloneauthor=P.+Wintringerauthor=T.+M.+Walkerauthor=D.+M.+Cirilloauthor=I.+Comasauthor=M.+R.+Farhatauthor=P.+Fowlerauthor=J.+Gardyauthor=N.+Ismailauthor=T.+A.+Kohlauthor=V.+Mathysauthor=M.+Merkerauthor=S.+Niemannauthor=S.+V.+Omarauthor=V.+Sintchenkoauthor=G.+Smithauthor=D.+v.+Soolingenauthor=P.+Supplyauthor=S.+Tahseenauthor=M.+Wilcoxauthor=I.+Arandjelovicauthor=T.+E.+A.+Petoauthor=D.+W.+Crookauthor=Z.+Iqbal&title=Antibiotic+resistance+prediction+for+Mycobacterium+tuberculosis+from+genome+sequence+data+with+Mykrobe&doi=10.12688%2Fwellcomeopenres.15603.1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antibiotic resistance prediction for Mycobacterium tuberculosis from genome sequence data with Mykrobe</span></div><div class="casAuthors">Hunt Martin; Bradley Phelim; Heys Simon; Thomsit Mark; Hall Michael B; Malone Kerri M; Wintringer Penelope; Iqbal Zamin; Hunt Martin; Walker Timothy M; Fowler Phillip; Peto Tim E A; Crook Derrick W; Lapierre Simon Grandjean; Lapierre Simon Grandjean; Walker Timothy M; Cirillo Daniela M; Comas Inaki; Comas Inaki; Comas Inaki; Farhat Maha R; Gardy Jennifer; Gardy Jennifer; Ismail Nazir; Omar Shaheed Vally; Kohl Thomas A; Merker Matthias; Niemann Stefan; Mathys Vanessa; Niemann Stefan; Sintchenko Vitali; Smith Grace; van Soolingen Dick; Supply Philip; Tahseen Sabira; Wilcox Mark; Wilcox Mark; Arandjelovic Irena; Crook Derrick W</div><div class="citationInfo"><span class="NLM_cas:title">Wellcome open research</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">191</span>
        ISSN:<span class="NLM_cas:issn">2398-502X</span>.
    </div><div class="casAbstract">Two billion people are infected with Mycobacterium tuberculosis, leading to 10 million new cases of active tuberculosis and 1.5 million deaths annually.  Universal access to drug susceptibility testing (DST) has become a World Health Organization priority.  We previously developed a software tool, Mykrobe predictor, which provided offline species identification and drug resistance predictions for M. tuberculosis from whole genome sequencing (WGS) data.  Performance was insufficient to support the use of WGS as an alternative to conventional phenotype-based DST, due to mutation catalogue limitations.  Here we present a new tool, Mykrobe, which provides the same functionality based on a new software implementation.  Improvements include i) an updated mutation catalogue giving greater sensitivity to detect pyrazinamide resistance, ii) support for user-defined resistance catalogues, iii) improved identification of non-tuberculous mycobacterial species, and iv) an updated statistical model for Oxford Nanopore Technologies sequencing data.  Mykrobe is released under MIT license at https://github.com/mykrobe-tools/mykrobe.  We incorporate mutation catalogues from the CRyPTIC consortium et al. (2018) and from Walker et al. (2015), and make improvements based on performance on an initial set of 3206 and an independent set of 5845 M. tuberculosis Illumina sequences.  To give estimates of error rates, we use a prospectively collected dataset of 4362 M. tuberculosis isolates.  Using culture based DST as the reference, we estimate Mykrobe to be 100%, 95%, 82%, 99% sensitive and 99%, 100%, 99%, 99% specific for rifampicin, isoniazid, pyrazinamide and ethambutol resistance prediction respectively.  We benchmark against four other tools on 10207 (=5845+4362) samples, and also show that Mykrobe gives concordant results with nanopore data.  We measure the ability of Mykrobe-based DST to guide personalized therapeutic regimen design in the context of complex drug susceptibility profiles, showing 94% concordance of implied regimen with that driven by phenotypic DST, higher than all other benchmarked tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSD36sZ4K3pbf69nzq6467DfW6udTcc2eYTjIN5Ot9Isbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387hvVOntA%253D%253D&md5=d45c4de8438022a8878aa309865f5be0</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.12688%2Fwellcomeopenres.15603.1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.12688%252Fwellcomeopenres.15603.1%26sid%3Dliteratum%253Aachs%26aulast%3DHunt%26aufirst%3DM.%26aulast%3DBradley%26aufirst%3DP.%26aulast%3DLapierre%26aufirst%3DS.%2BG.%26aulast%3DHeys%26aufirst%3DS.%26aulast%3DThomsit%26aufirst%3DM.%26aulast%3DHall%26aufirst%3DM.%2BB.%26aulast%3DMalone%26aufirst%3DK.%2BM.%26aulast%3DWintringer%26aufirst%3DP.%26aulast%3DWalker%26aufirst%3DT.%2BM.%26aulast%3DCirillo%26aufirst%3DD.%2BM.%26aulast%3DComas%26aufirst%3DI.%26aulast%3DFarhat%26aufirst%3DM.%2BR.%26aulast%3DFowler%26aufirst%3DP.%26aulast%3DGardy%26aufirst%3DJ.%26aulast%3DIsmail%26aufirst%3DN.%26aulast%3DKohl%26aufirst%3DT.%2BA.%26aulast%3DMathys%26aufirst%3DV.%26aulast%3DMerker%26aufirst%3DM.%26aulast%3DNiemann%26aufirst%3DS.%26aulast%3DOmar%26aufirst%3DS.%2BV.%26aulast%3DSintchenko%26aufirst%3DV.%26aulast%3DSmith%26aufirst%3DG.%26aulast%3DSoolingen%26aufirst%3DD.%2Bv.%26aulast%3DSupply%26aufirst%3DP.%26aulast%3DTahseen%26aufirst%3DS.%26aulast%3DWilcox%26aufirst%3DM.%26aulast%3DArandjelovic%26aufirst%3DI.%26aulast%3DPeto%26aufirst%3DT.%2BE.%2BA.%26aulast%3DCrook%26aufirst%3DD.%2BW.%26aulast%3DIqbal%26aufirst%3DZ.%26atitle%3DAntibiotic%2520resistance%2520prediction%2520for%2520Mycobacterium%2520tuberculosis%2520from%2520genome%2520sequence%2520data%2520with%2520Mykrobe%26jtitle%3DWellcome%2520Open%2520Res.%26date%3D2019%26volume%3D4%26spage%3D191%26epage%3D222%26doi%3D10.12688%2Fwellcomeopenres.15603.1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bray, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferlay, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soerjomataram, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Siegel, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torre, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemal, A.</span></span> <span> </span><span class="NLM_article-title">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span>. <i>Ca-Cancer J. Clin.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">394</span>– <span class="NLM_lpage">424</span>, <span class="refDoi"> DOI: 10.3322/caac.21492</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3322%2Fcaac.21492" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30207593" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2018&pages=394-424&author=F.+Brayauthor=J.+Ferlayauthor=I.+Soerjomataramauthor=R.+L.+Siegelauthor=L.+A.+Torreauthor=A.+Jemal&title=Global+cancer+statistics+2018%3A+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+countries&doi=10.3322%2Fcaac.21492"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Global cancer statistics 2018: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 countries</span></div><div class="casAuthors">Bray Freddie; Ferlay Jacques; Soerjomataram Isabelle; Siegel Rebecca L; Torre Lindsey A; Jemal Ahmedin</div><div class="citationInfo"><span class="NLM_cas:title">CA: a cancer journal for clinicians</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">394-424</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This article provides a status report on the global burden of cancer worldwide using the GLOBOCAN 2018 estimates of cancer incidence and mortality produced by the International Agency for Research on Cancer, with a focus on geographic variability across 20 world regions.  There will be an estimated 18.1 million new cancer cases (17.0 million excluding nonmelanoma skin cancer) and 9.6 million cancer deaths (9.5 million excluding nonmelanoma skin cancer) in 2018.  In both sexes combined, lung cancer is the most commonly diagnosed cancer (11.6% of the total cases) and the leading cause of cancer death (18.4% of the total cancer deaths), closely followed by female breast cancer (11.6%), prostate cancer (7.1%), and colorectal cancer (6.1%) for incidence and colorectal cancer (9.2%), stomach cancer (8.2%), and liver cancer (8.2%) for mortality.  Lung cancer is the most frequent cancer and the leading cause of cancer death among males, followed by prostate and colorectal cancer (for incidence) and liver and stomach cancer (for mortality).  Among females, breast cancer is the most commonly diagnosed cancer and the leading cause of cancer death, followed by colorectal and lung cancer (for incidence), and vice versa (for mortality); cervical cancer ranks fourth for both incidence and mortality.  The most frequently diagnosed cancer and the leading cause of cancer death, however, substantially vary across countries and within each country depending on the degree of economic development and associated social and life style factors.  It is noteworthy that high-quality cancer registry data, the basis for planning and implementing evidence-based cancer control programs, are not available in most low- and middle-income countries.  The Global Initiative for Cancer Registry Development is an international partnership that supports better estimation, as well as the collection and use of local data, to prioritize and evaluate national cancer control efforts.  CA: A Cancer Journal for Clinicians 2018;0:1-31. © 2018 American Cancer Society.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQaW5wv8ocuhLxanA7vODO6fW6udTcc2ebYvI4M8ETbXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3c3otVGlsg%253D%253D&md5=48529f8f794092779d70a27eec5c9017</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3322%2Fcaac.21492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3322%252Fcaac.21492%26sid%3Dliteratum%253Aachs%26aulast%3DBray%26aufirst%3DF.%26aulast%3DFerlay%26aufirst%3DJ.%26aulast%3DSoerjomataram%26aufirst%3DI.%26aulast%3DSiegel%26aufirst%3DR.%2BL.%26aulast%3DTorre%26aufirst%3DL.%2BA.%26aulast%3DJemal%26aufirst%3DA.%26atitle%3DGlobal%2520cancer%2520statistics%25202018%253A%2520GLOBOCAN%2520estimates%2520of%2520incidence%2520and%2520mortality%2520worldwide%2520for%252036%2520cancers%2520in%2520185%2520countries%26jtitle%3DCa-Cancer%2520J.%2520Clin.%26date%3D2018%26volume%3D68%26spage%3D394%26epage%3D424%26doi%3D10.3322%2Fcaac.21492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">National
Cancer Institute</span>.  <i>Cancer Stat Facts:
Female Breast Cancer</i>, <span class="NLM_year">2020</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2020&author=National%0ACancer+Institute&title=Cancer+Stat+Facts%3A%0AFemale+Breast+Cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DCancer%2520Stat%2520Facts%253A%250AFemale%2520Breast%2520Cancer%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Olayioye, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neve, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hynes, N. E.</span></span> <span> </span><span class="NLM_article-title">The ErbB signaling network : receptor heterodimerization in development and cancer</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3159</span>– <span class="NLM_lpage">3167</span>, <span class="refDoi"> DOI: 10.1093/emboj/19.13.3159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Femboj%2F19.13.3159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10880430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2000&pages=3159-3167&author=M.+A.+Olayioyeauthor=R.+M.+Neveauthor=H.+A.+Laneauthor=N.+E.+Hynes&title=The+ErbB+signaling+network+%3A+receptor+heterodimerization+in+development+and+cancer&doi=10.1093%2Femboj%2F19.13.3159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">The ErbB signaling network: receptor heterodimerization in development and cancer</span></div><div class="casAuthors">Olayioye, Monilola A.; Neve, Richard M.; Lane, Heidi A.; Hynes, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3159-3167</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review, with ∼126 refs., on individual ErbB receptors and their signaling pot. and examples of specific ErbB heterodimers in signaling and development.  The authors particular emphasis is on their role in mammary gland biol. , an organ in which the ErbB family and its ligands are critically involved in development, differentiation and cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr03qEN815kibVg90H21EOLACvtfcHk0lh1G35p-mrERQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXlt1ajtLY%253D&md5=3e70478dda68f1cfd5ab5bdb910e96a2</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1093%2Femboj%2F19.13.3159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Femboj%252F19.13.3159%26sid%3Dliteratum%253Aachs%26aulast%3DOlayioye%26aufirst%3DM.%2BA.%26aulast%3DNeve%26aufirst%3DR.%2BM.%26aulast%3DLane%26aufirst%3DH.%2BA.%26aulast%3DHynes%26aufirst%3DN.%2BE.%26atitle%3DThe%2520ErbB%2520signaling%2520network%2520%253A%2520receptor%2520heterodimerization%2520in%2520development%2520and%2520cancer%26jtitle%3DEMBO%2520J.%26date%3D2000%26volume%3D19%26spage%3D3159%26epage%3D3167%26doi%3D10.1093%2Femboj%2F19.13.3159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Burgess, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, H.-s.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eigenbrot, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garrett, T. P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leahy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemmon, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ward, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parade, R.</span></span> <span> </span><span class="NLM_article-title">An open-and-shut case ? recent insights into the activation of EGF / ErbB receptors csiro health science and nutrition</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">541</span>– <span class="NLM_lpage">552</span>, <span class="refDoi"> DOI: 10.1016/s1097-2765(03)00350-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs1097-2765%2803%2900350-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=14527402" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvFGrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=541-552&author=A.+W.+Burgessauthor=H.-s.+Choauthor=C.+Eigenbrotauthor=K.+M.+Fergusonauthor=T.+P.+J.+Garrettauthor=D.+J.+Leahyauthor=M.+A.+Lemmonauthor=M.+X.+Sliwkowskiauthor=C.+W.+Wardauthor=S.+Yokoyamaauthor=R.+Paradeauthor=R.+Parade&title=An+open-and-shut+case+%3F+recent+insights+into+the+activation+of+EGF+%2F+ErbB+receptors+csiro+health+science+and+nutrition&doi=10.1016%2Fs1097-2765%2803%2900350-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">An open-and-shut case? Recent insights into the activation of EGF/ErbB receptors</span></div><div class="casAuthors">Burgess, Antony W.; Cho, Hyun-Soo; Eigenbrot, Charles; Ferguson, Kathryn M.; Garrett, Thomas P. J.; Leahy, Daniel J.; Lemmon, Mark A.; Sliwkowski, Mark X.; Ward, Colin W.; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">541-552</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Recent crystallog. studies have provided significant new insight into how receptor tyrosine kinases from the EGF receptor or ErbB family are regulated by their growth factor ligands.  EGF receptor dimerization is mediated by a unique dimerization arm, which becomes exposed only after a dramatic domain rearrangement is promoted by growth factor binding.  ErbB2, a family member that has no ligand, has its dimerization arm constitutively exposed, and this explains several of its unique properties.  The authors outline a mechanistic view of ErbB receptor homo- and hetero-dimerization, which suggests new approaches for interfering with these processes when they are implicated in human cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXA0vpzHMgJrVg90H21EOLACvtfcHk0lhcH84QpM7VXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvFGrsbc%253D&md5=e2e2d07fc67ccdb7eedebeb73e511ae4</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fs1097-2765%2803%2900350-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1097-2765%252803%252900350-2%26sid%3Dliteratum%253Aachs%26aulast%3DBurgess%26aufirst%3DA.%2BW.%26aulast%3DCho%26aufirst%3DH.-s.%26aulast%3DEigenbrot%26aufirst%3DC.%26aulast%3DFerguson%26aufirst%3DK.%2BM.%26aulast%3DGarrett%26aufirst%3DT.%2BP.%2BJ.%26aulast%3DLeahy%26aufirst%3DD.%2BJ.%26aulast%3DLemmon%26aufirst%3DM.%2BA.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DWard%26aufirst%3DC.%2BW.%26aulast%3DYokoyama%26aufirst%3DS.%26aulast%3DParade%26aufirst%3DR.%26aulast%3DParade%26aufirst%3DR.%26atitle%3DAn%2520open-and-shut%2520case%2520%253F%2520recent%2520insights%2520into%2520the%2520activation%2520of%2520EGF%2520%252F%2520ErbB%2520receptors%2520csiro%2520health%2520science%2520and%2520nutrition%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D541%26epage%3D552%26doi%3D10.1016%2Fs1097-2765%2803%2900350-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">ErbB/HER protein-tyrosine kinases: structures and small molecule inhibitors</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">59</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2014.06.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.phrs.2014.06.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24928736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCjsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2014&pages=42-59&author=R.+Roskoski&title=ErbB%2FHER+protein-tyrosine+kinases%3A+structures+and+small+molecule+inhibitors&doi=10.1016%2Fj.phrs.2014.06.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">ErbB/HER protein-tyrosine kinases: Structures and small molecule inhibitors</span></div><div class="casAuthors">Roskoski, Robert Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-59</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The epidermal growth factor receptor (EGFR) family consists of four members that belong to the ErbB lineage of proteins (ErbB1-4).  These receptors consist of an extracellular domain, a single hydrophobic transmembrane segment, and an intracellular portion with a juxtamembrane segment, a protein kinase domain, and a carboxyterminal tail.  The ErbB proteins function as homo and heterodimers.  Growth factor binding to EGFR induces a large conformational change in the extracellular domain.  Two ligand-EGFR complexes unite to form a back-to-back dimer in which the ligands are on opposite sides of the aggregate.  Following ligand binding, EGFR intracellular kinase domains form an asym. dimer.  The carboxyterminal lobe of the activator kinase of the dimer interacts with the amino-terminal lobe of the receiver kinase thereby leading to its allosteric stimulation.  Several malignancies are assocd. with the mutation or increased expression of members of the ErbB family including lung, breast, stomach, colorectal, head and neck, and pancreatic carcinomas.  Gefitinib, erlotinib, and afatinib are orally effective protein-kinase targeted quinazoline derivs. that are used in the treatment of ERBB1-mutant lung cancer and lapatinib is an orally effective quinazoline deriv. used in the treatment of ErbB2-overexpressing breast cancer.  Moreover, monoclonal antibodies that target the extracellular domain of ErbB2 are used for the treatment of ErbB2-pos. breast cancer and monoclonal antibodies that target ErbB1 and are used for the treatment of colorectal cancer.  Cancers treated with these targeted drugs eventually become resistant to them, and a current goal of research is to develop drugs that are effective against drug-resistant tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLydngYGBiMLVg90H21EOLACvtfcHk0liNIseuJJtGAQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCjsLfF&md5=bc670e77f2b34592d168e6f60f56a548</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2014.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2014.06.001%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DErbB%252FHER%2520protein-tyrosine%2520kinases%253A%2520structures%2520and%2520small%2520molecule%2520inhibitors%26jtitle%3DPharmacol.%2520Res.%26date%3D2014%26volume%3D87%26spage%3D42%26epage%3D59%26doi%3D10.1016%2Fj.phrs.2014.06.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yarden, Y.</span></span> <span> </span><span class="NLM_article-title">Biology of HER2 and its importance in breast cancer</span>. <i>Oncology</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1159/000055396</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1159%2F000055396" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11694782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovFWjsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=1-13&author=Y.+Yarden&title=Biology+of+HER2+and+its+importance+in+breast+cancer&doi=10.1159%2F000055396"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Biology of HER2 and Its Importance in Breast Cancer</span></div><div class="casAuthors">Yarden, Yosef</div><div class="citationInfo"><span class="NLM_cas:title">Oncology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">Suppl. 2</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">ONCOBS</span>;
        ISSN:<span class="NLM_cas:issn">0030-2414</span>.
    
            (<span class="NLM_cas:orgname">S. Karger AG</span>)
        </div><div class="casAbstract">A review.  Human epidermal growth factor receptor-2 (HER2/erbB-2) belongs to a family of four transmembrane receptors involved in signal transduction pathways that regulate cell growth and differentiation.  Overexpression/amplification of HER2 is assocd. with malignancy and a poor prognosis in breast cancer.  HER2 acts as a networking receptor that mediates signaling to cancer cells, causing them to proliferate.  HER receptors exist as monomers but dimerize on ligand binding.  HER ligands are bivalent growth factor mols. whose low-affinity site binds to HER2.  No HER2-specific ligand has been identified but HER2 is the preferred heterodimerization partner for other HER receptors.  HER2-contg. heterodimers are relatively long-lived and potent.  HER3 has no inherent activity and is the major and most potent dimerization partner of HER2.  HER2 overexpression biases the formation of HER2-contg. heterodimers, leading to enhanced responsiveness to stromal growth factors and oncogenic transformation.  Removal of HER2 from the cell surface or inhibition of its intrinsic enzymic activity may reduce oncogenicity.  Our research suggests that the antitumor efficacy of HER2-specific antibodies such as Herceptin relates to their ability to direct HER2 to a Cbl- dependent endocytosis and degrdn. pathway.  The reported clin. therapeutic efficacy of anti-HER2 monoclonal antibodies in breast cancer highlights the importance of understanding the biol. of HER2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrtc627aN6-eLVg90H21EOLACvtfcHk0lgiJajdC09Zog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovFWjsLw%253D&md5=fb9f4c11080bb151edc30e62bb8869a8</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1159%2F000055396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1159%252F000055396%26sid%3Dliteratum%253Aachs%26aulast%3DYarden%26aufirst%3DY.%26atitle%3DBiology%2520of%2520HER2%2520and%2520its%2520importance%2520in%2520breast%2520cancer%26jtitle%3DOncology%26date%3D2001%26volume%3D61%26spage%3D1%26epage%3D13%26doi%3D10.1159%2F000055396" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lofgren, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotaling, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendly, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fox, J. A.</span></span> <span> </span><span class="NLM_article-title">Nonclinical studies addressing the mechanism of action of trastuzumab (Herceptin)</span>. <i>Semin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">60</span>– <span class="NLM_lpage">70</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10482195" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK1MXmt1SisbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1999&pages=60-70&author=M.+X.+Sliwkowskiauthor=J.+A.+Lofgrenauthor=G.+D.+Lewisauthor=T.+E.+Hotalingauthor=B.+M.+Fendlyauthor=J.+A.+Fox&title=Nonclinical+studies+addressing+the+mechanism+of+action+of+trastuzumab+%28Herceptin%29"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Nonclinical studies addressing the mechanism of action of trastuzumab (Herceptin)</span></div><div class="casAuthors">Sliwkowski, Mark X.; Lofgren, Julie A.; Lewis, Gail D.; Hotaling, Timothy E.; Fendly, Brian M.; Fox, Judith A.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Oncology</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">4, Suppl. 12</span>),
    <span class="NLM_cas:pages">60-70</span>CODEN:
                <span class="NLM_cas:coden">SOLGAV</span>;
        ISSN:<span class="NLM_cas:issn">0093-7754</span>.
    
            (<span class="NLM_cas:orgname">W. B. Saunders Co.</span>)
        </div><div class="casAbstract">A review with 75 refs.  HER2 is a ligand-less member of the human epidermal growth factor receptor or ErbB family of tyrosine kinases.  In normal biol. systems, HER2 functions as a co-receptor for a multitude of epidermal growth factor-like ligands that bind and activate other HER family members.  HER2 overexpression is obsd. in a no. of human adenocarcinomas and results in constitutive HER2 activation.  Specific targeting of these tumors can be accomplished with antibodies directed against the extracellular domain of the HER2 protein.  One of these antibodies, 4D5, has been fully humanized and is termed trastuzumab (Herceptin; Genentech, San Francisco, CA).  Treatment of HER2-overexpressing breast cancer cell lines with trastuzumab results in induction of p27KIPI and the Rb-related protein, p130, which in turn significantly reduces the no. of cells undergoing S-phase.  A no. of other phenotypic changes are obsd. in vitro as a consequence of trastuzumab binding to HER2-overexpressing cells.  These phenotypic changes include down-modulation of the HER2 receptor, inhibition of tumor cell growth, reversed cytokine resistance, restored E-cadherin expression levels, and reduced vascular endothelial growth factor prodn.  Interaction of trastuzumab with the human immune system via its human IgG1 Fc domain may potentiate its antitumor activities.  In vitro studies demonstrate that trastuzumab is very effective in mediating antibody-dependent cell-mediated cytotoxicity against HER2-overexpressing tumor targets.  Trastuzumab treatment of mouse xenograft models results in marked suppression of tumor growth.  When given in combination with std. cytotoxic chemotherapeutic agents, trastuzumab treatment generally results in statistically superior antitumor efficacy compared with either agent given alone.  Taken together, these studies suggest that the mechanism of action of trastuzumab includes antagonizing the constitutive growth-signaling properties of the HER2 system, enlisting immune cells to attack and kill the tumor target, and augmenting chemotherapy-induced cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp46ajd3v71YbVg90H21EOLACvtfcHk0lgQZre05AGuug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmt1SisbY%253D&md5=daeda0d8548a188580d9b90d15a44175</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DLofgren%26aufirst%3DJ.%2BA.%26aulast%3DLewis%26aufirst%3DG.%2BD.%26aulast%3DHotaling%26aufirst%3DT.%2BE.%26aulast%3DFendly%26aufirst%3DB.%2BM.%26aulast%3DFox%26aufirst%3DJ.%2BA.%26atitle%3DNonclinical%2520studies%2520addressing%2520the%2520mechanism%2520of%2520action%2520of%2520trastuzumab%2520%2528Herceptin%2529%26jtitle%3DSemin.%2520Oncol.%26date%3D1999%26volume%3D26%26spage%3D60%26epage%3D70" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumagai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezov, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murali, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greene, M. I.</span></span> <span> </span><span class="NLM_article-title">Erratum: HER2/Neu: mechanisms of dimerization/oligomerization</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">328</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1205119</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fsj.onc.1205119" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11840330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD38XhtVejtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2002&pages=328&author=P.+J.+Brennanauthor=T.+Kumagaiauthor=A.+Berezovauthor=R.+Muraliauthor=M.+I.+Greene&title=Erratum%3A+HER2%2FNeu%3A+mechanisms+of+dimerization%2Foligomerization&doi=10.1038%2Fsj.onc.1205119"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">HER2/Neu: mechanisms of dimerization/oligomerization. [Erratum to document cited in CA134:205690]</span></div><div class="casAuthors">Brennan, Patrick J.; Kumagai, Toru; Berezov, Alan; Murali, Ramachandran; Greene, Mark I.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">328</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The correct name of the second author is Toru Kumagai, not Kumogai.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrRpC9GeIShsrVg90H21EOLACvtfcHk0lhPPtsrqbppNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhtVejtL8%253D&md5=de1143f8d79b7e239957dca452565172</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1205119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1205119%26sid%3Dliteratum%253Aachs%26aulast%3DBrennan%26aufirst%3DP.%2BJ.%26aulast%3DKumagai%26aufirst%3DT.%26aulast%3DBerezov%26aufirst%3DA.%26aulast%3DMurali%26aufirst%3DR.%26aulast%3DGreene%26aufirst%3DM.%2BI.%26atitle%3DErratum%253A%2520HER2%252FNeu%253A%2520mechanisms%2520of%2520dimerization%252Foligomerization%26jtitle%3DOncogene%26date%3D2002%26volume%3D21%26spage%3D328%26doi%3D10.1038%2Fsj.onc.1205119" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pohlmann, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, I. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mernaugh, R.</span></span> <span> </span><span class="NLM_article-title">Resistance to trastuzumab in breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">7479</span>– <span class="NLM_lpage">7491</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-09-0636</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-09-0636" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=20008848" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGgur3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2009&pages=7479-7491&author=P.+R.+Pohlmannauthor=I.+A.+Mayerauthor=R.+Mernaugh&title=Resistance+to+trastuzumab+in+breast+cancer&doi=10.1158%2F1078-0432.ccr-09-0636"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance to Trastuzumab in Breast Cancer</span></div><div class="casAuthors">Pohlmann, Paula R.; Mayer, Ingrid A.; Mernaugh, Ray</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">7479-7491</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  HER2 is a transmembrane oncoprotein encoded by the HER2/neu gene and is overexpressed in approx. 20 to 25% of invasive breast cancers.  It can be therapeutically targeted by trastuzumab, a humanized IgG1 kappa light chain monoclonal antibody.  Although trastuzumab is currently considered one of the most effective treatments in oncol., a significant no. of patients with HER2-overexpressing breast cancer do not benefit from it.  Understanding the mechanisms of action and resistance to trastuzumab is therefore crucial for the development of new therapeutic strategies.  This review discusses proposed trastuzumab mode of action as well as proposed mechanisms for resistance.  Mechanisms for resistance are grouped into four main categories: (1) obstacles preventing trastuzumab binding to HER2; (2) upregulation of HER2 downstream signaling pathways; (3) signaling through alternate pathways; and (4) failure to trigger an immune-mediated mechanism to destroy tumor cells.  These potential mechanisms through which trastuzumab resistance may arise have been used as a guide to develop drugs, presently in clin. trials, to overcome resistance.  The mechanisms conferring trastuzumab resistance, when completely understood, will provide insight on how best to treat HER2-overexpressing breast cancer.  The understanding of each mechanism of resistance is therefore crit. for the educated development of strategies to overcome it, as well as for the development of tools that would allow definitive and efficient patient selection for each therapy. (Clin Cancer Res 2009;15(24):7479-91).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMog_W0HxkOrVg90H21EOLACvtfcHk0lgOcHran_PtxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGgur3F&md5=704b281777724d433a8d75ac22491d8f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-09-0636&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-09-0636%26sid%3Dliteratum%253Aachs%26aulast%3DPohlmann%26aufirst%3DP.%2BR.%26aulast%3DMayer%26aufirst%3DI.%2BA.%26aulast%3DMernaugh%26aufirst%3DR.%26atitle%3DResistance%2520to%2520trastuzumab%2520in%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2009%26volume%3D15%26spage%3D7479%26epage%3D7491%26doi%3D10.1158%2F1078-0432.ccr-09-0636" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schechter, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chevray, P. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Molecular cloning of the neu gene: Absence of gross structural alteration in oncogenic alleles</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">1986</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">261</span>– <span class="NLM_lpage">264</span>, <span class="refDoi"> DOI: 10.1073/pnas.83.2.261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1073%2Fpnas.83.2.261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=3001730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaL28XotlSnug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=1986&pages=261-264&author=M.+C.+Hungauthor=A.+L.+Schechterauthor=P.+Y.+Chevrayauthor=D.+F.+Sternauthor=R.+A.+Weinberg&title=Molecular+cloning+of+the+neu+gene%3A+Absence+of+gross+structural+alteration+in+oncogenic+alleles&doi=10.1073%2Fpnas.83.2.261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular cloning of the neu gene:  absence of gross structural alteration in oncogenic alleles</span></div><div class="casAuthors">Hung, Mien Chie; Schechter, Alan L.; Chevray, Pierre Yves M.; Stern, David F.; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">1986</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">261-4</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">The rat neu gene is distantly related to the erbB gene and encodes a cell surface protein that appears to function as a growth factor receptor.  To study the mechanisms that caused the conversion of the normal neu gene to an oncogenic allele, mol. clones of the neu oncogene as well as a clone of the corresponding protooncogene were isolated.  The transforming neu oncogene and the proton-neu gene clones exhibit identical restriction enzyme patterns.  Amplification of the proto-neu gene in NIH 3T3 cells by means of cotransfection with a dihydrofolate reductase gene resulted in methotrexate-resistant colonies that produce high levels of normal neu-encoded p185 protein.  In contrast to cells carrying low levels of the oncogene-encoded protein, these cells appeared normal.  The results suggest that the lesion that led to activation of the neu gene is a minor change in DNA sequence and is apparently located in the protein-encoding region of the gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoj3ThZdPzQl7Vg90H21EOLACvtfcHk0lgOcHran_PtxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL28XotlSnug%253D%253D&md5=27357da2ccf1aa2ec3ca6c28ce276a59</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.83.2.261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.83.2.261%26sid%3Dliteratum%253Aachs%26aulast%3DHung%26aufirst%3DM.%2BC.%26aulast%3DSchechter%26aufirst%3DA.%2BL.%26aulast%3DChevray%26aufirst%3DP.%2BY.%26aulast%3DStern%26aufirst%3DD.%2BF.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DMolecular%2520cloning%2520of%2520the%2520neu%2520gene%253A%2520Absence%2520of%2520gross%2520structural%2520alteration%2520in%2520oncogenic%2520alleles%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1986%26volume%3D83%26spage%3D261%26epage%3D264%26doi%3D10.1073%2Fpnas.83.2.261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Funkhouser, W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser-Rogers, K.</span></span> <span> </span><span class="NLM_article-title">Review: Significance of, and optimal screening for, HER-2 gene amplification and protein overexpression in breast carcinoma</span>. <i>Ann. Clin. Lab. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">349</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11688845" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXot1Smt7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2001&pages=349&author=W.+K.+Funkhouserauthor=K.+Kaiser-Rogers&title=Review%3A+Significance+of%2C+and+optimal+screening+for%2C+HER-2+gene+amplification+and+protein+overexpression+in+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Review: Significance of, and optimal screening for, HER-2 gene amplification and protein overexpression in breast carcinoma</span></div><div class="casAuthors">Funkhouser, William K.; Kaiser-Rogers, Kathleen</div><div class="citationInfo"><span class="NLM_cas:title">Annals of Clinical and Laboratory Science</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">349-358</span>CODEN:
                <span class="NLM_cas:coden">ACLSCP</span>;
        ISSN:<span class="NLM_cas:issn">0091-7370</span>.
    
            (<span class="NLM_cas:orgname">Association of Clinical Scientists</span>)
        </div><div class="casAbstract">A review.  Breast carcinoma is a common disease, with an estd. 183,000 new cases expected in the USA during 2000.  Whereas early stage patients have high likelihood of cure, only 20-40% of patients with metastatic breast carcinoma respond to presently available chemotherapy.  A need exists to identify the underlying biol. subsets of morphol. similar carcinomas in order to develop customized therapies for patients who require chemotherapy.  The HER-2 receptor tyrosine kinase is overexpressed in 15-30% of breast carcinomas, and is assocd. with a worse prognosis stage-for-stage.  Humanized monoclonal antibody therapy appears to benefit this subset of patients by improving their response rate and survival following anthracycline- or taxane-based chemotherapeutic regimens.  Both HER-2 gene amplification and protein overexpression correlate with clin. outcomes, and screening for HER-2 gene amplification appears to be the more informative test.  This article reviews data on the HER-2 gene and protein, discusses their assocn. with clin. outcomes, and proposes a strategy for screening for HER-2 excess in formalin-fixed specimens of breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpalKrvVXs2gbVg90H21EOLACvtfcHk0liz8I-zfwrH6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXot1Smt7o%253D&md5=eadfb2d29a99cef461d55b11c26881bc</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFunkhouser%26aufirst%3DW.%2BK.%26aulast%3DKaiser-Rogers%26aufirst%3DK.%26atitle%3DReview%253A%2520Significance%2520of%252C%2520and%2520optimal%2520screening%2520for%252C%2520HER-2%2520gene%2520amplification%2520and%2520protein%2520overexpression%2520in%2520breast%2520carcinoma%26jtitle%3DAnn.%2520Clin.%2520Lab.%2520Sci.%26date%3D2001%26volume%3D31%26spage%3D349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brabender, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salonga, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hölscher, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danenberg, P. V.</span></span> <span> </span><span class="NLM_article-title">Epidermal growth factor receptor and HER2-neu mRNA expression in non-small cell lung cancer is correlated with survival</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">1850</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11448895" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlslaltb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2001&pages=1850&author=J.+Brabenderauthor=K.+D.+Danenbergauthor=R.+Metzgerauthor=P.+M.+Schneiderauthor=J.+Parkauthor=D.+Salongaauthor=A.+H.+H%C3%B6lscherauthor=P.+V.+Danenberg&title=Epidermal+growth+factor+receptor+and+HER2-neu+mRNA+expression+in+non-small+cell+lung+cancer+is+correlated+with+survival"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Epidermal growth factor receptor and HER2-neu mRNA expression in non-small cell lung cancer is correlated with survival</span></div><div class="casAuthors">Brabender, Jan; Danenberg, Kathleen D.; Metzger, Ralf; Schneider, Paul M.; Park, Ji Min; Salonga, Dennis; Holscher, Arnulf H.; Danenberg, Peter V.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1850-1855</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The prognostic role of epidermal growth factor receptor (EGFR) and HER2-neu remains controversial in patients with non-small cell lung cancer (NSCLC).  We studied the assocn. between the mRNA expression of EGFR, HER2-neu, and survival in primary tumor and matching nonmalignant tissues from 83 patients with NSCLC.  Anal. was performed using a quant. real-time PCR system (Taqman).  EGFR and HER2-neu mRNA expression was detectable in all (100%) specimens analyzed.  Twenty-nine (34.9%) patients had high HER2-neu expression, and 28 (33.7%) patients had high EGFR expression.  A high HER2-neu and EGFR coexpression was detectable in 14 (16.9%) patients.  High HER2-neu expression was assocd. with inferior survival (P = 0.004), whereas high EGFR expression showed a trend toward inferior survival (P = 0.176).  The impact of HER2-neu and EGFR coexpression on patients' survival was additive (P = 0.003).  Multivariate anal. detd. high HER2-neu expression (P = 0.041), and high EGFR/HER2-neu coexpression (P = 0.030) as significant and independent unfavorable prognostic factors.  These findings indicate that HER2-neu and EGFR play a crucial role in the biol. behavior of NSCLCs.  Testing of mol. marker coexpression (EGFR and HER2-neu) improves the estn. of prognosis and appears to define low- and high-risk groups for treatment failure in curatively resected NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokEzxS4tTL-7Vg90H21EOLACvtfcHk0ljknqGlqDDqJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlslaltb8%253D&md5=ae41d3db82e5ff88fe10f246429f087d</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBrabender%26aufirst%3DJ.%26aulast%3DDanenberg%26aufirst%3DK.%2BD.%26aulast%3DMetzger%26aufirst%3DR.%26aulast%3DSchneider%26aufirst%3DP.%2BM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DSalonga%26aufirst%3DD.%26aulast%3DH%25C3%25B6lscher%26aufirst%3DA.%2BH.%26aulast%3DDanenberg%26aufirst%3DP.%2BV.%26atitle%3DEpidermal%2520growth%2520factor%2520receptor%2520and%2520HER2-neu%2520mRNA%2520expression%2520in%2520non-small%2520cell%2520lung%2520cancer%2520is%2520correlated%2520with%2520survival%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2001%26volume%3D7%26spage%3D1850" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tu, S.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ling, D.</span></span> <span> </span><span class="NLM_article-title">Oral cancer progression and c-erbB-2/neu proto-oncogene expression</span>. <i>Cancer Lett.</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>65</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">220</span>, <span class="refDoi"> DOI: 10.1016/0304-3835(92)90234-m</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2F0304-3835%2892%2990234-m" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1355405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADyaK38zos1CltQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=65&publication_year=1992&pages=215-220&author=L.+Houauthor=D.+Shiauthor=S.-M.+Tuauthor=H.-Z.+Zhangauthor=M.-C.+Hungauthor=D.+Ling&title=Oral+cancer+progression+and+c-erbB-2%2Fneu+proto-oncogene+expression&doi=10.1016%2F0304-3835%2892%2990234-m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Oral cancer progression and c-erbB-2/neu proto-oncogene expression</span></div><div class="casAuthors">Hou L; Shi D; Tu S M; Zhang H Z; Hung M C; Ling D</div><div class="citationInfo"><span class="NLM_cas:title">Cancer letters</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">65</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">215-20</span>
        ISSN:<span class="NLM_cas:issn">0304-3835</span>.
    </div><div class="casAbstract">Monoclonal antibody PAb3 to c-erbB-2/neu protein was utilized in the immunoperoxidase staining of 86 human specimens from oral mucosa.  These tissue specimens represented a spectrum from 7 normal to 9 simple hyperplasia, 15 mild dysplasia, 14 moderate dysplasia, 20 severe dysplasia and 21 squamous cell carcinoma.  Our study indicated that as the cells acquire a more malignant phenotype, there was a progressive increase in neu expression.  It also suggested that neu may be involved in the development of oral cancers and that its evaluation in the early stages may assist in the diagnosis and management of oral cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8-GGKG5txDokFb3mqPUYDfW6udTcc2eaGQL6b8IfU87ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK38zos1CltQ%253D%253D&md5=fee3cd5310b27f28a0bf471bf91b79b6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2F0304-3835%2892%2990234-m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0304-3835%252892%252990234-m%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DD.%26aulast%3DTu%26aufirst%3DS.-M.%26aulast%3DZhang%26aufirst%3DH.-Z.%26aulast%3DHung%26aufirst%3DM.-C.%26aulast%3DLing%26aufirst%3DD.%26atitle%3DOral%2520cancer%2520progression%2520and%2520c-erbB-2%252Fneu%2520proto-oncogene%2520expression%26jtitle%3DCancer%2520Lett.%26date%3D1992%26volume%3D65%26spage%3D215%26epage%3D220%26doi%3D10.1016%2F0304-3835%2892%2990234-m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, B.</span></span> <span> </span><span class="NLM_article-title">HER2 as a promising target for cytotoxicity T cells in human melanoma therapy</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">e73261</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0073261</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0073261" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24015299" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e73261&author=J.+Maauthor=H.+Hanauthor=D.+Liuauthor=W.+liauthor=H.+Fengauthor=X.+Xueauthor=X.+Wuauthor=G.+Niuauthor=G.+Zhangauthor=Y.+Zhaoauthor=C.+Liuauthor=H.+Taoauthor=B.+Gao&title=HER2+as+a+promising+target+for+cytotoxicity+T+cells+in+human+melanoma+therapy&doi=10.1371%2Fjournal.pone.0073261"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 as a promising target for cytotoxicity T cells in human melanoma therapy</span></div><div class="casAuthors">Ma, Juan; Han, Huamin; Liu, Deruo; Li, Wei; Feng, Hongxiang; Xue, Xin; Wu, Xiaoran; Niu, Ge; Zhang, Ge; Zhao, Yunfeng; Liu, Changzhen; Tao, Hua; Gao, Bin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">e73261</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Anti-HER2/neu antibody therapy has been reported to mediate tumor regression of HER2/neu+ tumors.  Here we demonstrated the expression of HER2 in a wide range of human melanoma cells including a primary culture and seven cell lines, and we further investigated whether HER2 could be served as a target for T cell mediated immunotherapy of human melanoma.  Specific cytolytic activity of activated T cells (ATC) armed with anti-CD3 × anti-HER2 bispecific antibody (HER2Bi-Ab) against Malme-3M-luc cells was evaluated by bioluminescent signal generated by luciferase reporter which did not alter HER2 expression or proliferation ability of Malme-3M cells.  Contrast with unarmed ATC, increased cytotoxic activity of HER2Bi-armed ATC against Malme-3M-luc cells was obsd. at effector/target (E/T) ratios of 1:1, 5:1, and 20:1.  Moreover, HER2Bi-armed ATC expressed higher level of activation marker CD69 and secreted significantly higher level of IFN-γ than unarmed ATC counterpart at the E/T ratio of 20:1.  In addn., compared with anti-HER2 mAb (Herceptin) or unarmed ATC, HER2Bi-armed ATC showed remarkable suppression effect on Malme-3M-luc tumor cells.  Furthermore, in melanoma tumor cell xenograft mice, infusion of HER2Bi-armed ATC successfully inhibited the growth of melanoma tumors.  The anti-tumor effect of HER2Bi-armed ATC may provide a promising immunotherapy for melanoma in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo99ILu7lrNebVg90H21EOLACvtfcHk0liPfv4mFoI4kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVWhtrvI&md5=8d0f508736ee30ac02bb4b69a045c4b4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0073261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0073261%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DJ.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3Dli%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DH.%26aulast%3DXue%26aufirst%3DX.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DNiu%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DTao%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DB.%26atitle%3DHER2%2520as%2520a%2520promising%2520target%2520for%2520cytotoxicity%2520T%2520cells%2520in%2520human%2520melanoma%2520therapy%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De73261%26doi%3D10.1371%2Fjournal.pone.0073261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Musolino, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciccolallo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Panebianco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fontana, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanoni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bozzetti, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Michiara, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silini, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ardizzoni, A.</span></span> <span> </span><span class="NLM_article-title">Multifactorial central nervous system recurrence susceptibility in patients with HER2-positive breast cancer</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>117</i></span>,  <span class="NLM_fpage">1837</span>– <span class="NLM_lpage">1846</span>, <span class="refDoi"> DOI: 10.1002/cncr.25771</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcncr.25771" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21509760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3Mvlt12rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=117&publication_year=2011&pages=1837-1846&author=A.+Musolinoauthor=L.+Ciccolalloauthor=M.+Panebiancoauthor=E.+Fontanaauthor=D.+Zanoniauthor=C.+Bozzettiauthor=M.+Michiaraauthor=E.+M.+Siliniauthor=A.+Ardizzoni&title=Multifactorial+central+nervous+system+recurrence+susceptibility+in+patients+with+HER2-positive+breast+cancer&doi=10.1002%2Fcncr.25771"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Multifactorial central nervous system recurrence susceptibility in patients with HER2-positive breast cancer: epidemiological and clinical data from a population-based cancer registry study</span></div><div class="casAuthors">Musolino Antonino; Ciccolallo Laura; Panebianco Michele; Fontana Elisa; Zanoni Daniele; Bozzetti Cecilia; Michiara Maria; Silini Enrico Maria; Ardizzoni Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">117</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1837-46</span>
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    </div><div class="casAbstract">BACKGROUND:  A series of retrospective studies have reported that patients with human epidermal growth factor receptor 2(HER2)-positive breast cancer are at a greater risk of central nervous system (CNS) metastases.  Trastuzumab, which does not cross the blood-brain barrier, has been associated with this increased risk.  METHODS:  The authors evaluated incidence, survival, and risk factors for CNS metastases in the incident breast cancer population systematically collected by the Parma Province Cancer Registry over the 4-year period between 2004 and 2007.  RESULTS:  A total of 1458 patients with a diagnosis of stage I to III invasive breast cancer were analyzed for study purposes.  At a median follow-up of 4.1 years, CNS events were observed in 1.3% and 5% of HER2-negative patients and HER2-positive patients, respectively (P < .0001).  The administration of trastuzumab either as adjuvant therapy or for metastatic disease was associated with a significantly increased risk of CNS involvement at first disease recurrence and after first extracranial recurrence, respectively.  According to multivariate analysis, HER2-positive status and trastuzumab treatment, high Ki-67 index, and hormone receptor negativity remained independent risk factors for the development of CNS metastasis.  CONCLUSIONS:  To the authors' knowledge, this is the first population-based cancer registry study analyzing factors associated with CNS recurrence in a general population of newly diagnosed breast cancer patients with known HER2 status.  The data from the current study provide evidence that patients with HER2-positive breast cancer have a significantly higher incidence of CNS metastasis after treatment with trastuzumab.  Improvements in systemic control and overall survival associated with trastuzumab-based therapy may lead to an "unmasking" of CNS disease recurrence that would otherwise remain clinically silent before a patient's death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSzAwMcs1oehCM8MEzWGjS9fW6udTcc2ebWB9wkho94l7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mvlt12rsA%253D%253D&md5=8f3833e5f32b9534a231886a90b747d1</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2Fcncr.25771&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.25771%26sid%3Dliteratum%253Aachs%26aulast%3DMusolino%26aufirst%3DA.%26aulast%3DCiccolallo%26aufirst%3DL.%26aulast%3DPanebianco%26aufirst%3DM.%26aulast%3DFontana%26aufirst%3DE.%26aulast%3DZanoni%26aufirst%3DD.%26aulast%3DBozzetti%26aufirst%3DC.%26aulast%3DMichiara%26aufirst%3DM.%26aulast%3DSilini%26aufirst%3DE.%2BM.%26aulast%3DArdizzoni%26aufirst%3DA.%26atitle%3DMultifactorial%2520central%2520nervous%2520system%2520recurrence%2520susceptibility%2520in%2520patients%2520with%2520HER2-positive%2520breast%2520cancer%26jtitle%3DCancer%26date%3D2011%26volume%3D117%26spage%3D1837%26epage%3D1846%26doi%3D10.1002%2Fcncr.25771" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Loibl, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, L.</span></span> <span> </span><span class="NLM_article-title">HER2-positive breast cancer</span>. <i>Lancet</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>389</i></span>,  <span class="NLM_fpage">2415</span>– <span class="NLM_lpage">2429</span>, <span class="refDoi"> DOI: 10.1016/s0140-6736(16)32417-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs0140-6736%2816%2932417-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=27939064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVegurbF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=389&publication_year=2017&pages=2415-2429&author=S.+Loiblauthor=L.+Gianni&title=HER2-positive+breast+cancer&doi=10.1016%2Fs0140-6736%2816%2932417-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-positive breast cancer</span></div><div class="casAuthors">Loibl, Sibylle; Gianni, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Lancet</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">389</span>
        (<span class="NLM_cas:issue">10087</span>),
    <span class="NLM_cas:pages">2415-2429</span>CODEN:
                <span class="NLM_cas:coden">LANCAO</span>;
        ISSN:<span class="NLM_cas:issn">0140-6736</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Anti-HER2 treatment for HER2-pos. breast cancer has changed the natural biol. of this disease.  This Series article reviews the main achievements so far in the treatment of both metastatic and early HER2-pos. breast cancer.  The success of neoadjuvant therapy in HER2-pos. early breast cancer is esp. acknowledged, as pertuzumab has been approved on the basis of a higher proportion of patients achieving a pathol. complete response with pertuzumab and trastuzumab than with trastuzumab alone in a neoadjuvant study.  Event-free survival after the confirmatory adjuvant trial completed recruitment was numerically better with pertuzumab plus trastuzumab than with trastuzumab alone.  With survival rates of almost 5 years in women with metastatic HER2-pos. breast cancer and 75% of patients achieving a pathol. complete response, new treatments in the past decade have clearly improved the prognosis of HER2-pos. breast cancer.  Despite these achievements, however, the persisting high toll of deaths resulting from HER2-pos. breast cancer calls for continued, intensive clin. research of newer therapies and combinations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGogw25S2Ui-abVg90H21EOLACvtfcHk0lii4t7JFA8ZjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVegurbF&md5=65fcd259f84cfb308e78ddacec0eb692</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fs0140-6736%2816%2932417-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0140-6736%252816%252932417-5%26sid%3Dliteratum%253Aachs%26aulast%3DLoibl%26aufirst%3DS.%26aulast%3DGianni%26aufirst%3DL.%26atitle%3DHER2-positive%2520breast%2520cancer%26jtitle%3DLancet%26date%3D2017%26volume%3D389%26spage%3D2415%26epage%3D2429%26doi%3D10.1016%2Fs0140-6736%2816%2932417-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Escrivá-de-Romaní, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arumí, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saura, C.</span></span> <span> </span><span class="NLM_article-title">HER2-positive breast cancer: Current and new therapeutic strategies</span>. <i>Breast</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">80</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/j.breast.2018.03.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.breast.2018.03.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29631097" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1Mjgt1ygsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2018&pages=80-88&author=S.+Escriv%C3%A1-de-Roman%C3%ADauthor=M.+Arum%C3%ADauthor=M.+Belletauthor=C.+Saura&title=HER2-positive+breast+cancer%3A+Current+and+new+therapeutic+strategies&doi=10.1016%2Fj.breast.2018.03.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-positive breast cancer: Current and new therapeutic strategies</span></div><div class="casAuthors">Escriva-de-Romani Santiago; Arumi Miriam; Bellet Meritxell; Saura Cristina</div><div class="citationInfo"><span class="NLM_cas:title">Breast (Edinburgh, Scotland)</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">80-88</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Since the identification of the HER2 receptor amplification as an adverse prognostic factor that defined a special subtype of metastatic breast cancer, there has been a substantial improvement in survival of patients affected with this disease due to the development of anti-HER2 targeted therapies.  The approval of trastuzumab and pertuzumab associated to a taxane in first line and subsequent treatment with the antibody-drug conjugate T-DM1 has certainly contributed to achieve these outcomes.  The Tyrosine Kinase Inhibitor lapatinib was also approved in the basis of an improvement in progression free survival, becoming another commonly used treatment in combination with capecitabine.  Inevitably, despite these therapeutic advances most patients progress on therapy due to primary or acquired resistance or because of an incorrect HER2 positivity assessment.  Hence, it is crucial to correctly categorize HER2 amplified tumors and define mechanisms of resistance to design effective new treatment approaches.  In addition, identifying biomarkers of response or resistance permits to tailor the therapeutic options for each patient sparing them from unnecessary toxicity as well as improving their outcomes.  The aim of this review is to examine new strategies in development to treat HER2-positive metastatic breast cancer referring to the mechanisms of action of new drugs and new combinations including results reported so far.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ3a48Zp0Bo9Nj_Vg2vObihfW6udTcc2eZpDTYx7tMMf7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mjgt1ygsQ%253D%253D&md5=36ea3f9eec7f253d0a0f35ebb812acbc</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.breast.2018.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.breast.2018.03.006%26sid%3Dliteratum%253Aachs%26aulast%3DEscriv%25C3%25A1-de-Roman%25C3%25AD%26aufirst%3DS.%26aulast%3DArum%25C3%25AD%26aufirst%3DM.%26aulast%3DBellet%26aufirst%3DM.%26aulast%3DSaura%26aufirst%3DC.%26atitle%3DHER2-positive%2520breast%2520cancer%253A%2520Current%2520and%2520new%2520therapeutic%2520strategies%26jtitle%3DBreast%26date%3D2018%26volume%3D39%26spage%3D80%26epage%3D88%26doi%3D10.1016%2Fj.breast.2018.03.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Oh, D.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bang, Y.-J.</span></span> <span> </span><span class="NLM_article-title">HER2-targeted therapies - a role beyond breast cancer</span>. <i>Nat. Rev. Clin. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">33</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1038/s41571-019-0268-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41571-019-0268-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=31548601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvVarsLfF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2020&pages=33-48&author=D.-Y.+Ohauthor=Y.-J.+Bang&title=HER2-targeted+therapies+-+a+role+beyond+breast+cancer&doi=10.1038%2Fs41571-019-0268-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">HER2-targeted therapies - a role beyond breast cancer</span></div><div class="casAuthors">Oh, Do-Youn; Bang, Yung-Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-48</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  HER2 is an established therapeutic target in a large subset of women with breast cancer; a variety of agents including trastuzumab, pertuzumab, lapatinib, neratinib and trastuzumab emtansine (T-DM1) have been approved for the treatment of HER2-pos. breast cancer.  HER2 is also overexpressed in subsets of patients with other solid tumors.  Notably, the addn. of trastuzumab to first-line chemotherapy has improved the overall survival of patients with HER2-pos. gastric cancer, and has become the std.-of-care treatment for this group of patients.  However, trials involving pertuzumab, lapatinib and T-DM1 have failed to provide significant improvements in the outcomes of patients with HER2-pos. gastric cancer.  HER2-targeted therapies are also being tested in patients with other solid tumors harbouring HER2 overexpression, and/or amplifications or other mutations of the gene encoding HER2 (ERBB2), including biliary tract, colorectal, non-small-cell lung and bladder cancers.  The experience with gastric cancer suggests that the successes obsd. in HER2-pos. breast cancer might not be replicated in these other tumor types, owing to differences in the level of HER2 overexpression and other aspects of disease biol.  In this Review, we describe the current role of HER2-targeted therapies beyond breast cancer and also highlight the potential of novel HER2-targeted agents that are currently in clin. development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWFUgr5GY13bVg90H21EOLACvtfcHk0lhbJl0rXcMiHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvVarsLfF&md5=8fc3d80ab470b5e01686cae99f944885</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fs41571-019-0268-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41571-019-0268-3%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DD.-Y.%26aulast%3DBang%26aufirst%3DY.-J.%26atitle%3DHER2-targeted%2520therapies%2520-%2520a%2520role%2520beyond%2520breast%2520cancer%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2020%26volume%3D17%26spage%3D33%26epage%3D48%26doi%3D10.1038%2Fs41571-019-0268-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Arribas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedersen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parra-Palau, J. L.</span></span> <span> </span><span class="NLM_article-title">p95HER2 and breast cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">1515</span>– <span class="NLM_lpage">1519</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-10-3795</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-10-3795" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21343397" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXisFCgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=1515-1519&author=J.+Arribasauthor=J.+Baselgaauthor=K.+Pedersenauthor=J.+L.+Parra-Palau&title=p95HER2+and+breast+cancer&doi=10.1158%2F0008-5472.can-10-3795"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">p95HER2 and Breast Cancer</span></div><div class="casAuthors">Arribas, Joaquin; Baselga, Jose; Pedersen, Kim; Parra-Palau, Josep Lluis</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1515-1519</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A review.  A subtype of HER2-pos. tumors with distinct biol. and clin. features expresses a series of carboxy-terminal fragments collectively known as p95HER2.  One of these fragments, named 100- to 115-kDa p95HER2 or 611-CTF, is hyperactive because of its ability to form homodimers maintained by intermol. disulfide bonds.  Despite lacking the majority of the extracellular domain, this HER2 fragment drives breast cancer progression in vivo.  The recent availability of specific anti-p95 antibodies has confirmed previous results indicating that the expression of p95HER2 is predictive of poor prognosis and correlates with resistance to the treatment with trastuzumab, a therapeutic antibody directed against the extracellular domain of HER2.  Cancer Res; 71(5); 1515-9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZSd5rzePjJ7Vg90H21EOLACvtfcHk0lg5W9P6-LyslA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXisFCgu7w%253D&md5=79b01b5f32271d0d833d804828787358</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-10-3795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-10-3795%26sid%3Dliteratum%253Aachs%26aulast%3DArribas%26aufirst%3DJ.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DPedersen%26aufirst%3DK.%26aulast%3DParra-Palau%26aufirst%3DJ.%2BL.%26atitle%3Dp95HER2%2520and%2520breast%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D1515%26epage%3D1519%26doi%3D10.1158%2F0008-5472.can-10-3795" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dokmanovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W. J.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab-mediated cardiotoxicity: current understanding, challenges, and frontiers</span>. <i>Antibody Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">17</span>, <span class="refDoi"> DOI: 10.1093/abt/tby003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Fabt%2Ftby003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30215054" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsVOit7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2018&pages=13-17&author=N.+Mohanauthor=J.+Jiangauthor=M.+Dokmanovicauthor=W.+J.+Wu&title=Trastuzumab-mediated+cardiotoxicity%3A+current+understanding%2C+challenges%2C+and+frontiers&doi=10.1093%2Fabt%2Ftby003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab-mediated cardiotoxicity: current understanding, challenges, and frontiers</span></div><div class="casAuthors">Mohan, Nishant; Jiang, Jiangsong; Dokmanovic, Milos; Wu, Wen Jin</div><div class="citationInfo"><span class="NLM_cas:title">Antibody Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">13-17</span>CODEN:
                <span class="NLM_cas:coden">ATNHAH</span>;
        ISSN:<span class="NLM_cas:issn">2516-4236</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Trastuzumab, an epidermal growth factor receptor 2 (HER2) targeting humanized monoclonal antibody, has been approved for the treatment HER2-pos. breast cancer and HER2-positve metastatic gastric cancer.  However, cardiotoxicity assocd. with its clin. application poses challenges for clinicians and patients, mechanisms of which are still evolving.  This review will summarize the current mechanistic understanding of trastuzumab-mediated cardiotoxicity, discuss the novel role of DNA topoisomerase IIB as a shared target for enhanced cardiotoxicity induced by trastuzumab and anthracyclines-based combination regimens, and speculate the potential impact of trastuzumab intervention in immune checkpoint inhibitors-based therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphm7vff0kdlbVg90H21EOLACvtfcHk0liEYAkf2SqrQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsVOit7Y%253D&md5=2bcca83b7fb6ac29fdc06717626c04f6</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fabt%2Ftby003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fabt%252Ftby003%26sid%3Dliteratum%253Aachs%26aulast%3DMohan%26aufirst%3DN.%26aulast%3DJiang%26aufirst%3DJ.%26aulast%3DDokmanovic%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DW.%2BJ.%26atitle%3DTrastuzumab-mediated%2520cardiotoxicity%253A%2520current%2520understanding%252C%2520challenges%252C%2520and%2520frontiers%26jtitle%3DAntibody%2520Ther.%26date%3D2018%26volume%3D1%26spage%3D13%26epage%3D17%26doi%3D10.1093%2Fabt%2Ftby003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Diéras, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miles, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pegram, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welslau, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krop, I. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoersch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gianni, L.</span></span> <span> </span><span class="NLM_article-title">Trastuzumab emtansine versus capecitabine plus lapatinib in patients with previously treated HER2-positive advanced breast cancer (EMILIA): a descriptive analysis of final overall survival results from a randomised, open-label, phase 3 trial</span>. <i>Lancet Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">732</span>– <span class="NLM_lpage">742</span>, <span class="refDoi"> DOI: 10.1016/s1470-2045(17)30312-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs1470-2045%2817%2930312-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28526536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnvFKiur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=732-742&author=V.+Di%C3%A9rasauthor=D.+Milesauthor=S.+Vermaauthor=M.+Pegramauthor=M.+Welslauauthor=J.+Baselgaauthor=I.+E.+Kropauthor=K.+Blackwellauthor=S.+Hoerschauthor=J.+Xuauthor=M.+Greenauthor=L.+Gianni&title=Trastuzumab+emtansine+versus+capecitabine+plus+lapatinib+in+patients+with+previously+treated+HER2-positive+advanced+breast+cancer+%28EMILIA%29%3A+a+descriptive+analysis+of+final+overall+survival+results+from+a+randomised%2C+open-label%2C+phase+3+trial&doi=10.1016%2Fs1470-2045%2817%2930312-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Trastuzumab emtansine versus capecitabine plus lapatinib in patients with previously treated HER2-positive advanced breast cancer (EMILIA): a descriptive analysis of final overall survival results from a randomised, open-label, phase 3 trial</span></div><div class="casAuthors">Dieras, Veronique; Miles, David; Verma, Sunil; Pegram, Mark; Welslau, Manfred; Baselga, Jose; Krop, Ian E.; Blackwell, Kim; Hoersch, Silke; Xu, Jin; Green, Marjorie; Gianni, Luca</div><div class="citationInfo"><span class="NLM_cas:title">Lancet Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">732-742</span>CODEN:
                <span class="NLM_cas:coden">LOANBN</span>;
        ISSN:<span class="NLM_cas:issn">1470-2045</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The antibody-drug conjugate trastuzumab emtansine is indicated for the treatment of patients with HER2-pos. metastatic breast cancer previously treated with trastuzumab and a taxane.  Approval of this drug was based on progression-free survival and interim overall survival data from the phase 3 EMILIA study.  In this report, we present a descriptive anal. of the final overall survival data from that trial.  EMILIA was a randomized, international, open-label, phase 3 study of men and women aged 18 years or older with HER2-pos. unresectable, locally advanced or metastatic breast cancer previously treated with trastuzumab and a taxane.  Enrolled patients were randomly assigned (1:1) via a hierarchical, dynamic randomization scheme and an interactive voice response system to trastuzumab emtansine (3·6 mg/kg i.v. every 3 wk) or control (capecitabine 1000 mg/m2 self-administered orally twice daily on days 1-14 on each 21-day cycle, plus lapatinib 1250 mg orally once daily on days 1-21).  Randomisation was stratified by world region (USA vs western Europe vs or other), no. of previous chemotherapy regimens for unresectable, locally advanced, or metastatic disease (0 or 1 vs >1), and disease involvement (visceral vs non-visceral).  The coprimary efficacy endpoints were progression-free survival (per independent review committee assessment) and overall survival.  Efficacy was analyzed in the intention-to-treat population; safety was analyzed in all patients who received at least one dose of study treatment, with patients analyzed according to the treatment actually received.  On May 30, 2012, the study protocol was amended to allow crossover from control to trastuzumab emtansine after the second interim overall survival anal. crossed the prespecified overall survival efficacy boundary.  This study is registered with ClinicalTrials.gov, no. NCT00829166.  Between Feb 23, 2009, and Oct 13, 2011, 991 eligible patients were enrolled and randomly assigned to either trastuzumab emtansine (n=495) or capecitabine and lapatinib (control; n=496).  In this final descriptive anal., median overall survival was longer with trastuzumab emtansine than with control (29·9 mo [95% CI 26·3-34·1] vs 25·9 mo [95% CI 22·7-28·3]; hazard ratio 0·75 [95% CI 0·64-0·88]). 136 (27%) of 496 patients crossed over from control to trastuzumab emtansine after the second interim overall survival anal. (median follow-up duration 24·1 mo [IQR 19·5-26·1]).  Of those patients originally randomly assigned to trastuzumab emtansine, 254 (51%) of 495 received capecitabine and 241 [49%] of 495 received lapatinib (sep. or in combination) after study drug discontinuation.  In the safety population (488 patients treated with capecitabine plus lapatinib, 490 patients treated with trastuzumab emtansine), fewer grade 3 or worse adverse events occurred with trastuzumab emtansine (233 [48%] of 490) than with capecitabine plus lapatinib control treatment (291 [60%] of 488).  In the control group, the most frequently reported grade 3 or worse adverse events were diarrhea (103 [21%] of 488 patients) followed by palmar-plantar erythrodysaesthesia syndrome (87 [18%]), and vomiting (24 [5%]).  The safety profile of trastuzumab emtansine was similar to that reported previously; the most frequently reported grade 3 or worse adverse events in the trastuzumab emtansine group were thrombocytopenia (70 [14%] of 490), increased aspartate aminotransferase levels (22 [5%]), and anemia (19 [4%]).  Nine patients died from adverse events; five of these deaths were judged to be related to treatment (two in the control group [coronary artery disease and multiorgan failure] and three in the trastuzumab emtansine group [metabolic encephalopathy, neutropenic sepsis, and acute myeloid leukemia]).  This descriptive anal. of final overall survival in the EMILIA trial shows that trastuzumab emtansine improved overall survival in patients with previously treated HER2-pos. metastatic breast cancer even in the presence of crossover treatment.  The safety profile was similar to that reported in previous analyses, reaffirming trastuzumab emtansine as an efficacious and tolerable treatment in this patient population.F Hoffmann-La Roche/Genentech.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbnbUnIygFBbVg90H21EOLACvtfcHk0lhwDyK7AA2uzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnvFKiur0%253D&md5=5d3663e6a753ddb9a2f23f67a7f3a9a4</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fs1470-2045%2817%2930312-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs1470-2045%252817%252930312-1%26sid%3Dliteratum%253Aachs%26aulast%3DDi%25C3%25A9ras%26aufirst%3DV.%26aulast%3DMiles%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%26aulast%3DPegram%26aufirst%3DM.%26aulast%3DWelslau%26aufirst%3DM.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DKrop%26aufirst%3DI.%2BE.%26aulast%3DBlackwell%26aufirst%3DK.%26aulast%3DHoersch%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DGreen%26aufirst%3DM.%26aulast%3DGianni%26aufirst%3DL.%26atitle%3DTrastuzumab%2520emtansine%2520versus%2520capecitabine%2520plus%2520lapatinib%2520in%2520patients%2520with%2520previously%2520treated%2520HER2-positive%2520advanced%2520breast%2520cancer%2520%2528EMILIA%2529%253A%2520a%2520descriptive%2520analysis%2520of%2520final%2520overall%2520survival%2520results%2520from%2520a%2520randomised%252C%2520open-label%252C%2520phase%25203%2520trial%26jtitle%3DLancet%2520Oncol.%26date%3D2017%26volume%3D18%26spage%3D732%26epage%3D742%26doi%3D10.1016%2Fs1470-2045%2817%2930312-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneeweiss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickish, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eniu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hegg, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tausch, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, Y.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratnayake, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNally, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cortés, J.</span></span> <span> </span><span class="NLM_article-title">Pertuzumab plus trastuzumab in combination with standard neoadjuvant anthracycline-containing and anthracycline-free chemotherapy regimens in patients with HER2-positive early breast cancer: A randomized phase II cardiac safety study (TRYPHAENA)</span>. <i>Ann. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">2278</span>– <span class="NLM_lpage">2284</span>, <span class="refDoi"> DOI: 10.1093/annonc/mdt182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1093%2Fannonc%2Fmdt182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23704196" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3snmsVOqsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2013&pages=2278-2284&author=A.+Schneeweissauthor=S.+Chiaauthor=T.+Hickishauthor=V.+Harveyauthor=A.+Eniuauthor=R.+Heggauthor=C.+Tauschauthor=J.+H.+Seoauthor=Y.-F.+Tsaiauthor=J.+Ratnayakeauthor=V.+McNallyauthor=G.+Rossauthor=J.+Cort%C3%A9s&title=Pertuzumab+plus+trastuzumab+in+combination+with+standard+neoadjuvant+anthracycline-containing+and+anthracycline-free+chemotherapy+regimens+in+patients+with+HER2-positive+early+breast+cancer%3A+A+randomized+phase+II+cardiac+safety+study+%28TRYPHAENA%29&doi=10.1093%2Fannonc%2Fmdt182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Pertuzumab plus trastuzumab in combination with standard neoadjuvant anthracycline-containing and anthracycline-free chemotherapy regimens in patients with HER2-positive early breast cancer: a randomized phase II cardiac safety study (TRYPHAENA)</span></div><div class="casAuthors">Schneeweiss A; Chia S; Hickish T; Harvey V; Eniu A; Hegg R; Tausch C; Seo J H; Tsai Y-F; Ratnayake J; McNally V; Ross G; Cortes J</div><div class="citationInfo"><span class="NLM_cas:title">Annals of oncology : official journal of the European Society for Medical Oncology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2278-84</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Pertuzumab (P) combined with trastuzumab (H)-based chemotherapy improves efficacy in early and advanced HER2-positive breast cancer.  We assessed the tolerability, with particular focus on cardiac safety, of H and P with chemotherapy in the neoadjuvant treatment of HER2-positive early breast cancer.  PATIENTS AND METHODS:  In this multicenter, open-label phase II study, patients with operable, locally advanced, or inflammatory breast cancer were randomized 1 : 1 : 1 to receive six neoadjuvant cycles q3w (Arm A: 5-fluorouracil, epirubicin, cyclophosphamide [FEC] + H + P ×3 → docetaxel [T] + H + P ×3; Arm B: FEC ×3 → T + H + P ×3; Arm C: T + carboplatin + H [TCH]+P ×6). pCR was assessed at surgery and adjuvant therapy given to complete 1 year of H.  RESULTS:  Two hundred twenty-five patients were randomized.  During neoadjuvant treatment, two patients (2.7%; Arm B) experienced symptomatic left ventricular systolic dysfunction (LVSD) and 11 patients (Arm A: 4 [5.6%]; Arm B: 4 [5.3%]; Arm C: 3 [3.9%]) had declines in left ventricular ejection fraction of ≥10% points from baseline to <50%.  Diarrhea was the most common adverse event. pCR (ypT0/is) was reported for 61.6% (Arm A), 57.3% (Arm B), and 66.2% (Arm C) of patients.  CONCLUSION:  The combination of P with H and standard chemotherapy resulted in low rates of symptomatic LVSD.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS_ysMXSzWrcw20zIbnCCI3fW6udTcc2ea-FXXH2eVhcLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3snmsVOqsg%253D%253D&md5=921bf406259fbe2207740b5a19bba16c</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1093%2Fannonc%2Fmdt182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fannonc%252Fmdt182%26sid%3Dliteratum%253Aachs%26aulast%3DSchneeweiss%26aufirst%3DA.%26aulast%3DChia%26aufirst%3DS.%26aulast%3DHickish%26aufirst%3DT.%26aulast%3DHarvey%26aufirst%3DV.%26aulast%3DEniu%26aufirst%3DA.%26aulast%3DHegg%26aufirst%3DR.%26aulast%3DTausch%26aufirst%3DC.%26aulast%3DSeo%26aufirst%3DJ.%2BH.%26aulast%3DTsai%26aufirst%3DY.-F.%26aulast%3DRatnayake%26aufirst%3DJ.%26aulast%3DMcNally%26aufirst%3DV.%26aulast%3DRoss%26aufirst%3DG.%26aulast%3DCort%25C3%25A9s%26aufirst%3DJ.%26atitle%3DPertuzumab%2520plus%2520trastuzumab%2520in%2520combination%2520with%2520standard%2520neoadjuvant%2520anthracycline-containing%2520and%2520anthracycline-free%2520chemotherapy%2520regimens%2520in%2520patients%2520with%2520HER2-positive%2520early%2520breast%2520cancer%253A%2520A%2520randomized%2520phase%2520II%2520cardiac%2520safety%2520study%2520%2528TRYPHAENA%2529%26jtitle%3DAnn.%2520Oncol.%26date%3D2013%26volume%3D24%26spage%3D2278%26epage%3D2284%26doi%3D10.1093%2Fannonc%2Fmdt182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nahta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, L. X. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Du, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, F. J.</span></span> <span> </span><span class="NLM_article-title">Lapatinib induces apoptosis in trastuzumab-resistant breast cancer cells: effects on insulin-like growth factor I signaling</span>. <i>Mol. Cancer Ther.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1158/1535-7163.mct-06-0423</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1535-7163.mct-06-0423" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17308062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhvVWrsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=667-674&author=R.+Nahtaauthor=L.+X.+H.+Yuanauthor=Y.+Duauthor=F.+J.+Esteva&title=Lapatinib+induces+apoptosis+in+trastuzumab-resistant+breast+cancer+cells%3A+effects+on+insulin-like+growth+factor+I+signaling&doi=10.1158%2F1535-7163.mct-06-0423"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Lapatinib induces apoptosis in trastuzumab-resistant breast cancer cells: Effects on insulin-like growth factor I signaling</span></div><div class="casAuthors">Nahta, Rita; Yuan, Linda X. H.; Du, Yi; Esteva, Francisco J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">667-674</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The majority of breast cancer patients who achieve an initial therapeutic response to the HER2-targeted antibody trastuzumab will show disease progression within 1 yr.  Thus, the identification of novel agents that effectively inhibit survival of cancer cells that have progressed on trastuzumab is crit.  In the current study, we show that the dual epidermal growth factor receptor (EGFR)/human EGFR-2 (HER2) kinase inhibitor lapatinib induces apoptosis in trastuzumab-resistant cells derived from the HER2-overexpressing SKBR3 breast cancer line.  Lapatinib inhibited EGFR and HER2 signaling in resistant cells, blocking activation of downstream Akt, mitogen-activated protein kinase, and S6 kinases and inducing expression of p27kip1.  Importantly, lapatinib also inhibited insulin-like growth factor I (IGF-I) signaling and growth-promoting effects in parental and resistant cells, and the cytotoxic effects of lapatinib were further enhanced by the IGF-I receptor-blocking antibody αIR3.  As increased IGF-I receptor signaling has been implicated in trastuzumab resistance, our data strongly support further study of lapatinib as a potential therapeutic in breast cancers that have progressed on trastuzumab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgA2XSBOQGErVg90H21EOLACvtfcHk0liE5VIImgGzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhvVWrsbc%253D&md5=4bee464a1d992a25a4c5f832cb4b0086</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.mct-06-0423&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.mct-06-0423%26sid%3Dliteratum%253Aachs%26aulast%3DNahta%26aufirst%3DR.%26aulast%3DYuan%26aufirst%3DL.%2BX.%2BH.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DEsteva%26aufirst%3DF.%2BJ.%26atitle%3DLapatinib%2520induces%2520apoptosis%2520in%2520trastuzumab-resistant%2520breast%2520cancer%2520cells%253A%2520effects%2520on%2520insulin-like%2520growth%2520factor%2520I%2520signaling%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2007%26volume%3D6%26spage%3D667%26epage%3D674%26doi%3D10.1158%2F1535-7163.mct-06-0423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kannan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eli, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lalani, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J.</span></span> <span> </span><span class="NLM_article-title">Preclinical characteristics of the irreversible pan- her kinase inhibitor neratinib compared with lapatinib: implications for the treatment of HER2- positive and HER2-mutated breast cancer</span>. <i>Cancers</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">737</span>, <span class="refDoi"> DOI: 10.3390/cancers11060737</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3390%2Fcancers11060737" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BB3cXjtlCktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2019&pages=737&author=D.+M.+Collinsauthor=N.+T.+Conlonauthor=S.+Kannanauthor=C.+S.+Vermaauthor=L.+D.+Eliauthor=A.+S.+Lalaniauthor=J.+Crown&title=Preclinical+characteristics+of+the+irreversible+pan-+her+kinase+inhibitor+neratinib+compared+with+lapatinib%3A+implications+for+the+treatment+of+HER2-+positive+and+HER2-mutated+breast+cancer&doi=10.3390%2Fcancers11060737"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical characteristics of the irreversible pan-HER kinase inhibitor neratinib compared with lapatinib: implications for the treatment of HER2-positive and HER2-mutated breast cancer</span></div><div class="casAuthors">Collins, Denis M.; Conlon, Neil T.; Kannan, Srinivasaraghavan; Verma, Chandra S.; Eli, Lisa D.; Lalani, Alshad S.; Crown, John</div><div class="citationInfo"><span class="NLM_cas:title">Cancers</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">737</span>CODEN:
                <span class="NLM_cas:coden">CANCCT</span>;
        ISSN:<span class="NLM_cas:issn">2072-6694</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">An estd. 15-20% of breast cancers overexpress human epidermal growth factor receptor 2 (HER2/ERBB2/neu).  Two small-mol. tyrosine kinase inhibitors (TKIs), lapatinib and neratinib, have been approved for the treatment of HER2-pos. (HER2+) breast cancer.  Lapatinib, a reversible epidermal growth factor receptor (EGFR/ERBB1/HER1) and HER2 TKI, is used for the treatment of advanced HER2+ breast cancer in combination with capecitabine, in combination with trastuzumab in patients with hormone receptor-neg. metastatic breast cancer, and in combination with an aromatase inhibitor for the first-line treatment of HER2+ breast cancer.  Neratinib, a next-generation, irreversible pan-HER TKI, is used in the US for extended adjuvant treatment of adult patients with early-stage HER2+ breast cancer following 1 yr of trastuzumab.  In Europe, neratinib is used in the extended adjuvant treatment of adult patients with early-stage hormone receptor-pos. HER2+ breast cancer who are less than 1 yr from the completion of prior adjuvant trastuzumab-based therapy.  Preclin. studies have shown that these agents have distinct properties that may impact their clin. activity.  This review describes the preclin. characterization of lapatinib and neratinib, with a focus on the differences between these two agents that may have implications for patient management.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqC4tzLmkuc0rVg90H21EOLACvtfcHk0liE5VIImgGzxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXjtlCktrg%253D&md5=c1d2bb10e99b843b9bf028734994868d</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.3390%2Fcancers11060737&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fcancers11060737%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DD.%2BM.%26aulast%3DConlon%26aufirst%3DN.%2BT.%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DVerma%26aufirst%3DC.%2BS.%26aulast%3DEli%26aufirst%3DL.%2BD.%26aulast%3DLalani%26aufirst%3DA.%2BS.%26aulast%3DCrown%26aufirst%3DJ.%26atitle%3DPreclinical%2520characteristics%2520of%2520the%2520irreversible%2520pan-%2520her%2520kinase%2520inhibitor%2520neratinib%2520compared%2520with%2520lapatinib%253A%2520implications%2520for%2520the%2520treatment%2520of%2520HER2-%2520positive%2520and%2520HER2-mutated%2520breast%2520cancer%26jtitle%3DCancers%26date%3D2019%26volume%3D11%26spage%3D737%26doi%3D10.3390%2Fcancers11060737" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><a href="https://www.fda.gov/drugs/new-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products/new-drug-therapy-approvals-2019" class="extLink">https://www.fda.gov/drugs/new-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products/new-drug-therapy-approvals-2019</a>. (accessed Jan 18, 2020).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=https%3A%2F%2Fwww.fda.gov%2Fdrugs%2Fnew-drugs-fda-cders-new-molecular-entities-and-new-therapeutic-biological-products%2Fnew-drug-therapy-approvals-2019.+%28accessed+Jan+18%2C+2020%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">How to train your inhibitor: design strategies to overcome resistance to epidermal growth factor receptor inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>142</i></span>,  <span class="NLM_fpage">131</span>– <span class="NLM_lpage">151</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.07.023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2017.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28754471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1GktLjK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=142&publication_year=2017&pages=131-151&author=S.+N.+Milikauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=K.+A.+M.+Abouzid&title=How+to+train+your+inhibitor%3A+design+strategies+to+overcome+resistance+to+epidermal+growth+factor+receptor+inhibitors&doi=10.1016%2Fj.ejmech.2017.07.023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">How to train your inhibitor: Design strategies to overcome resistance to Epidermal Growth Factor Receptor inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Lasheen, Deena S.; Serya, Rabah A. T.; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">142</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">131-151</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Epidermal Growth Factor Receptor (EGFR) stands out as a key player in the development of many cancers.  Its dysregulation is assocd. with a vast no. of tumors such as nonsmall-cell lung cancer, colon cancer, head-and-neck cancer, breast and ovarian cancer.  Being implicated in the development of a no. of the most lethal cancers worldwide, EGFR has long been considered as a focal target for cancer therapies, ever since the FDA approval of "Gefitinib" in 2003 and up to the last FDA approved small mol. EGFR kinase inhibitor "Osimertinib" in 2015.  Studies are still going on to find more efficient EGFR inhibitors due to the continuous emergence of resistance to the current inhibitors.  Cancerous cells resist EGFR tyrosine kinase inhibitors (TKIs) through various mechanisms, the most commonly reported ones are the T790M mutation and HER2 amplification.  Therefore, tackling EGFR TKIs-resistant tumors through a multitargeting approach comprising a dual EGFR/HER2 inhibitor that is also capable of inhibiting the mutant T790M EGFR is anticipated to overcome drug resistance.  In this review, the authors will survey the structural aspects of EGFR family and the structure-activity relationship of representative dual EGFR/HER2 inhibitors.  To follow, the authors will discuss the structural aspects of the mutation-driven resistance and various design strategies to overcome it.  Finally, the authors will review the SAR of exemplary irreversible dual EGFR/HER2 inhibitors that can overcome the mutation-driven resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY7kCQB9fXZbVg90H21EOLACvtfcHk0ljXm_tNmWzwdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1GktLjK&md5=689e6f4e9a377f847da751187b1875bc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DHow%2520to%2520train%2520your%2520inhibitor%253A%2520design%2520strategies%2520to%2520overcome%2520resistance%2520to%2520epidermal%2520growth%2520factor%2520receptor%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D142%26spage%3D131%26epage%3D151%26doi%3D10.1016%2Fj.ejmech.2017.07.023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: gefitinib (ZD1839) (iressa(R)) tablets</span>. <i>Oncologista</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">303</span>– <span class="NLM_lpage">306</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.8-4-303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.8-4-303" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12897327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFSqtrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2003&pages=303-306&author=M.+H.+Cohenauthor=G.+A.+Williamsauthor=R.+Sridharaauthor=G.+Chenauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+gefitinib+%28ZD1839%29+%28iressa%28R%29%29+tablets&doi=10.1634%2Ftheoncologist.8-4-303"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: gefitinib (ZD1839) (Iressa) tablets</span></div><div class="casAuthors">Cohen, Martin H.; Williams, Grant A.; Sridhara, Rajeshwari; Chen, Gang; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">303-306</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  On May 5, 2003, gefitinib (Iressa, ZD1839) 250-mg tablets received accelerated approval by the U.S. Food and Drug Administration as monotherapy treatment for patients with locally advanced or metastatic non-small cell lung cancer (NSCLC) after failure of both platinum-based and docetaxel chemotherapies.  Information provided in this summary includes efficacy and safety results of relevant clin. trials.  Effectiveness was demonstrated in a randomized, double-blind, phase II, multicenter trial comparing two oral doses of gefitinib (250 mg/day vs. 500 mg/day).  Two hundred sixteen patients were enrolled.  The 142 patients who were refractory to or intolerant of a platinum and docetaxel comprised the evaluable population for the efficacy anal.  A partial tumor response occurred in 14% (9 of 66) of patients receiving gefitinib 250 mg/day and in 8% (6 of 76) of patients receiving gefitinib 500 mg/day.  The overall objective response rate for both doses combined was 10.6% (15 of 142 patients) (95% confidence interval 6.0%-16.8%).  Responses were more frequent in females and in nonsmokers.  The median duration of response was 7.0 mo (range 4.6-18.6+ months).  Other submitted data included the results of two large trials conducted in chemotherapy-naive, stage III and IV NSCLC patients.  Patients were randomized to receive gefitinib (250 mg or 500 mg daily) or placebo, in combination with either gemcitabine plus cisplatin (n = 1,093) or carboplatin plus paclitaxel (n = 1,037).  Results from those studies showed no benefit (response rate, time to progression, or survival) from adding gefitinib to chemotherapy.  Consequently, gefitinib is only recommended for use as monotherapy.  Common adverse events assocd. with gefitinib treatment included diarrhea, rash, acne, dry skin, nausea, and vomiting.  Most toxicities were Common Toxicity Criteria grade 1 or 2.  Interstitial lung disease (ILD) has been obsd. in patients receiving gefitinib.  Worldwide, the incidence of ILD is about 1% (2% in the Japanese postmarketing experience and about 0.3% in a U.S. expanded access program).  Approx. one-third of the cases were fatal.  Physicians should promptly evaluate new or worsening pulmonary symptoms.  If ILD is confirmed, appropriate management includes discontinuation of gefitinib.  Gefitinib was approved under accelerated approval regulations on the basis of a surrogate end point response rate.  No controlled gefitinib trials, to date, demonstrate a clin. benefit, such as improvement in disease-related symptoms or greater survival.  Accelerated approval regulations require the sponsor to conduct further studies to verify that gefitinib therapy produces such a benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqLYgwUCCwY7Vg90H21EOLACvtfcHk0ljXm_tNmWzwdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFSqtrc%253D&md5=b05bb79ab3ac78dbb34f617311339ae0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.8-4-303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.8-4-303%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DWilliams%26aufirst%3DG.%2BA.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520gefitinib%2520%2528ZD1839%2529%2520%2528iressa%2528R%2529%2529%2520tablets%26jtitle%3DOncologista%26date%3D2003%26volume%3D8%26spage%3D303%26epage%3D306%26doi%3D10.1634%2Ftheoncologist.8-4-303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: erlotinib (Tarceva) tablets</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">466</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.10-7-461</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.10-7-461" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16079312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKltrnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2005&pages=461-466&author=M.+H.+Cohenauthor=J.+R.+Johnsonauthor=Y.+F.+Chenauthor=R.+Sridharaauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+erlotinib+%28Tarceva%29+tablets&doi=10.1634%2Ftheoncologist.10-7-461"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: erlotinib (Tarceva) tablets</span></div><div class="casAuthors">Cohen, Martin H.; Johnson, John R.; Chen, Yeh-Fong; Sridhara, Rajeshwari; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">461-466</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">On Nov. 18, 2004, erlotinib (Tarceva; OSI Pharmaceuticals, Inc., Melville, NY, http://www.osip.com, and Genentech, Inc., South San Francisco, CA, http://www.gene.com) received regular approval as monotherapy for the treatment of patients with locally advanced or metastatic non-small cell lung cancer (NSCLC) after failure of at least one prior chemotherapy regimen.  Survival of erlotinib-treated patients was superior to that of placebo-treated patients.  The median survival duration of erlotinib-treated patients was 6.67 mo, compared with 4.70 mo for placebo-treated patients.  Exploratory univariate analyses showed a larger survival prolongation in two subsets of patients: those who never smoked and those with epidermal growth factor receptor (EGFR)-pos. tumors.  Patients who never smoked and were EGFR-pos. had a large erlotinib survival benefit.  Erlotinib was also superior to placebo for progression-free survival and a response rate of 8.9% vs. 0.9%.  Skin rash and diarrhea were the most common erlotinib adverse events.  Severe rash occurred in 8%, and severe diarrhea occurred in 6% of erlotinib-treated patients.  In the first-line treatment of NSCLC, two large, controlled, randomized trials showed no benefit from adding erlotinib to doublet, platinum-based chemotherapy.  Therefore, erlotinib is not indicated for use in this setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqG6dyEamjGgrVg90H21EOLACvtfcHk0lhe7kpr1F0HyA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKltrnJ&md5=e613a64fdb4a24a5e98ade5855477086</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.10-7-461&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.10-7-461%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DJ.%2BR.%26aulast%3DChen%26aufirst%3DY.%2BF.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520erlotinib%2520%2528Tarceva%2529%2520tablets%26jtitle%3DOncologist%26date%3D2005%26volume%3D10%26spage%3D461%26epage%3D466%26doi%3D10.1634%2Ftheoncologist.10-7-461" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ryan, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahim, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ko, C. w.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sridhara, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Justice, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pazdur, R.</span></span> <span> </span><span class="NLM_article-title">FDA drug approval summary: lapatinib in combination with capecitabine for previously treated metastatic breast cancer that overexpresses HER-2</span>. <i>Oncologista</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">1114</span>– <span class="NLM_lpage">1119</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2008-0816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.2008-0816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18849320" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsVGgur7I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=1114-1119&author=Q.+Ryanauthor=A.+Ibrahimauthor=M.+H.+Cohenauthor=J.+Johnsonauthor=C.+w.+Koauthor=R.+Sridharaauthor=R.+Justiceauthor=R.+Pazdur&title=FDA+drug+approval+summary%3A+lapatinib+in+combination+with+capecitabine+for+previously+treated+metastatic+breast+cancer+that+overexpresses+HER-2&doi=10.1634%2Ftheoncologist.2008-0816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">FDA drug approval summary: lapatinib in combination with capecitabine for previously treated metastatic breast cancer that overexpresses HER-2</span></div><div class="casAuthors">Ryan, Qin; Ibrahim, Amna; Cohen, Martin H.; Johnson, John; Ko, Chia-Wen; Sridhara, Rajeshwari; Justice, Robert; Pazdur, Richard</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1114-1119</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">On March 13, 2007, the U.S. Food and Drug Administration approved lapatinib (Tykerb tablets; Glaxo-SmithKline, Philadelphia), an oral, small mol., dual tyrosine kinase inhibitor of ErbB-2 and ErbB-1, for use in combination with capecitabine for the treatment of patients with human epidermal growth factor receptor (HER)-2-overexpressing metastatic breast cancer who had received prior therapy including an anthracycline, a taxane, and trastuzumab.  One multicenter, open-label, randomized trial was submitted.  Eligible patients had stage IIIb or IV breast cancer, ErbB-2 overexpression (immunohistochem. 3+ or 2+ with fluorescence in situ hybridization confirmation), measurable disease, a 0 or 1 Eastern Cooperative Oncol. Group performance status score, a cardiac ejection fraction within the institutional normal range, and adequate lab. function.  Patients received either lapatinib (1,250 mg once daily on days 1-21) plus capecitabine (1,000 mg/m2 every 12 h on days 1-14) every 21 days or capecitabine alone (1,250 mg/m2 every 12 h on days 1-14) every 21 days.  The primary endpoint was time to progression (TTP) detd. by a blinded independent review panel.  After TTP results of a prespecified interim anal. were made available, study enrollment was discontinued (399 patients enrolled).  The median TTP was 27.1 vs. 18.6 wk (hazard ratio, 0.57; p = .00013) favoring the lapatinib plus capecitabine arm.  Response rates were 23.7% (lapatinib plus capecitabine) vs. 13.9% (capecitabine alone).  Survival data were not mature.  Although the toxicities obsd. in the lapatinib and capecitabine combination arm were generally similar to those in the capecitabine alone arm, a higher incidence of diarrhea and rash was noted with the combination.  Grade 3 or 4 adverse reactions that occurred with a frequency of >5% in patients on the combination arm were diarrhea (13%) and palmar-plantar erythrodysesthesia (12%).  There was a 2% incidence of reversible decreased left ventricular function in the combination arm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_M2JQVn8WCbVg90H21EOLACvtfcHk0lgjKuVj_4kVTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsVGgur7I&md5=618f9677ef731dcc593be4a45504470c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2008-0816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2008-0816%26sid%3Dliteratum%253Aachs%26aulast%3DRyan%26aufirst%3DQ.%26aulast%3DIbrahim%26aufirst%3DA.%26aulast%3DCohen%26aufirst%3DM.%2BH.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DKo%26aufirst%3DC.%2Bw.%26aulast%3DSridhara%26aufirst%3DR.%26aulast%3DJustice%26aufirst%3DR.%26aulast%3DPazdur%26aufirst%3DR.%26atitle%3DFDA%2520drug%2520approval%2520summary%253A%2520lapatinib%2520in%2520combination%2520with%2520capecitabine%2520for%2520previously%2520treated%2520metastatic%2520breast%2520cancer%2520that%2520overexpresses%2520HER-2%26jtitle%3DOncologista%26date%3D2008%26volume%3D13%26spage%3D1114%26epage%3D1119%26doi%3D10.1634%2Ftheoncologist.2008-0816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Affleck, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cockerill, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubberfield, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guntrip, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carter, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jowett, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stables, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Topley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brignola, P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kadwell, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reep, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mullin, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keith, B. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crosby, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span> <span> </span><span class="NLM_article-title">The characterization of novel, dual ErbB-2/EGFR, tyrosine kinase inhibitors: Potential therapy for cancer</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2001</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">7196</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11585755" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnsVeht70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2001&pages=7196&author=D.+W.+Rusnakauthor=K.+Affleckauthor=S.+G.+Cockerillauthor=C.+Stubberfieldauthor=R.+Harrisauthor=M.+Pageauthor=K.+J.+Smithauthor=S.+B.+Guntripauthor=M.+C.+Carterauthor=R.+J.+Shawauthor=A.+Jowettauthor=J.+Stablesauthor=P.+Topleyauthor=E.+R.+Woodauthor=P.+S.+Brignolaauthor=S.+H.+Kadwellauthor=B.+R.+Reepauthor=R.+J.+Mullinauthor=K.+J.+Alligoodauthor=B.+R.+Keithauthor=R.+M.+Crosbyauthor=D.+M.+Murrayauthor=W.+B.+Knightauthor=T.+M.+Gilmerauthor=K.+Lackey&title=The+characterization+of+novel%2C+dual+ErbB-2%2FEGFR%2C+tyrosine+kinase+inhibitors%3A+Potential+therapy+for+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">The characterization of novel, dual ErbB-2/EGFR, tyrosine kinase inhibitors: potential therapy for cancer</span></div><div class="casAuthors">Rusnak, David W.; Affleck, Karen; Cockerill, Stuart G.; Stubberfield, Colin; Harris, Robert; Page, Martin; Smith, Kathryn J.; Guntrip, Stephen B.; Carter, Malcolm C.; Shaw, Robert J.; Jowett, Amanda; Stables, Jeremy; Topley, Peter; Wood, Edgar R.; Brignola, Perry S.; Kadwell, Sue H.; Reep, Bryan R.; Mullin, Robert J.; Alligood, Krystal J.; Keith, Barry R.; Crosby, Renae M.; Murray, Doris M.; Knight, W. Blaine; Gilmer, Tona M.; Lackey, Karen</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7196-7203</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">The type 1 receptor tyrosine kinases constitute a family of transmembrane proteins involved in various aspects of cell growth and survival and have been implicated in the initiation and progression of several types of human malignancies.  The best characterized of these proteins are the epidermal growth factor receptor (EGFR) and ErbB-2 (HER-2/neu).  We have developed potent quinazoline and pyrido-[3,4-d]-pyrimidine small mols. that are dual inhibitors of ErbB-2 and EGFR.  The compds. demonstrate potent in vitro inhibition of the ErbB-2 and EGFR kinase domains with IC50s <80 nM.  Growth of ErbB-2- and EGFR-expressing tumor cell lines is inhibited at concns. <0.5 μM.  Selectivity for tumor cell growth inhibition vs. normal human fibroblast growth inhibition ranges from 10- to >75-fold.  Tumor growth in mouse s.c. xenograft models of the BT474 and HN5 cell lines is inhibited in a dose-responsive manner using oral doses of 10 and 30 mg/kg twice per day.  In addn., the tested compds. caused a redn. of ErbB-2 and EGFR autophosphorylation in tumor fragments from these xenograft models.  These data indicate that these compds. have potential use as therapy in the broad population of cancer patients overexpressing ErbB-2 and/or EGFR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoifUxgOrEfMLVg90H21EOLACvtfcHk0lgZKWZjQZB1ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnsVeht70%253D&md5=374cb1b4451356866a5060c5ff39a1d1</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DAffleck%26aufirst%3DK.%26aulast%3DCockerill%26aufirst%3DS.%2BG.%26aulast%3DStubberfield%26aufirst%3DC.%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DPage%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DK.%2BJ.%26aulast%3DGuntrip%26aufirst%3DS.%2BB.%26aulast%3DCarter%26aufirst%3DM.%2BC.%26aulast%3DShaw%26aufirst%3DR.%2BJ.%26aulast%3DJowett%26aufirst%3DA.%26aulast%3DStables%26aufirst%3DJ.%26aulast%3DTopley%26aufirst%3DP.%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DBrignola%26aufirst%3DP.%2BS.%26aulast%3DKadwell%26aufirst%3DS.%2BH.%26aulast%3DReep%26aufirst%3DB.%2BR.%26aulast%3DMullin%26aufirst%3DR.%2BJ.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DKeith%26aufirst%3DB.%2BR.%26aulast%3DCrosby%26aufirst%3DR.%2BM.%26aulast%3DMurray%26aufirst%3DD.%2BM.%26aulast%3DKnight%26aufirst%3DW.%2BB.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26aulast%3DLackey%26aufirst%3DK.%26atitle%3DThe%2520characterization%2520of%2520novel%252C%2520dual%2520ErbB-2%252FEGFR%252C%2520tyrosine%2520kinase%2520inhibitors%253A%2520Potential%2520therapy%2520for%2520cancer%26jtitle%3DCancer%2520Res.%26date%3D2001%26volume%3D61%26spage%3D7196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wood, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truesdale, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDonald, O. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickerson, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennisi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horne, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lackey, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alligood, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rusnak, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilmer, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shewchuk, L.</span></span> <span> </span><span class="NLM_article-title">A unique structure for epidermal growth factor receptor bound to GW572016 (Lapatinib): Relationships among protein conformation, inhibitor off-rate, and receptor activity in tumor cells</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>64</i></span>,  <span class="NLM_fpage">6652</span>– <span class="NLM_lpage">6659</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-04-1168</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-04-1168" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=15374980" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnsFOlu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=64&publication_year=2004&pages=6652-6659&author=E.+R.+Woodauthor=A.+T.+Truesdaleauthor=O.+B.+McDonaldauthor=D.+Yuanauthor=A.+Hassellauthor=S.+H.+Dickersonauthor=B.+Ellisauthor=C.+Pennisiauthor=E.+Horneauthor=K.+Lackeyauthor=K.+J.+Alligoodauthor=D.+W.+Rusnakauthor=T.+M.+Gilmerauthor=L.+Shewchuk&title=A+unique+structure+for+epidermal+growth+factor+receptor+bound+to+GW572016+%28Lapatinib%29%3A+Relationships+among+protein+conformation%2C+inhibitor+off-rate%2C+and+receptor+activity+in+tumor+cells&doi=10.1158%2F0008-5472.can-04-1168"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">A unique structure for epidermal growth factor receptor bound to GW572016 (Lapatinib): relationships among protein conformation, inhibitor off-rate, and receptor activity in tumor cells</span></div><div class="casAuthors">Wood, Edgar R.; Truesdale, Anne T.; McDonald, Octerloney B.; Yuan, Derek; Hassell, Anne; Dickerson, Scott H.; Ellis, Byron; Pennisi, Christopher; Horne, Earnest; Lackey, Karen; Alligood, Krystal J.; Rusnak, David W.; Gilmer, Tona M.; Shewchuk, Lisa</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">64</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6652-6659</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">GW572016 (Lapatinib) is a tyrosine kinase inhibitor in clin. development for cancer that is a potent dual inhibitor of epidermal growth factor receptor (EGFR, ErbB-1) and ErbB-2.  The authors detd. the crystal structure of EGFR bound to GW572016.  The compd. is bound to an inactive-like conformation of EGFR that is very different from the active-like structure bound by the selective EGFR inhibitor OSI-774 (Tarceva) described previously.  Surprisingly, the authors found that GW572016 has a very slow off-rate from the purified intracellular domains of EGFR and ErbB-2 compared with OSI-774 and another EGFR selective inhibitor, ZD-1839 (Iressa).  Treatment of tumor cells with these inhibitors results in down-regulation of receptor tyrosine phosphorylation.  The authors evaluated the duration of the drug effect after washing away free compd. and found that the rate of recovery of receptor phosphorylation in the tumor cells reflected the inhibitor off-rate from the purified intracellular domain.  The slow off-rate of GW572016 correlates with a prolonged down-regulation of receptor tyrosine phosphorylation in tumor cells.  The differences in the off-rates of these drugs and the ability of GW572016 to inhibit ErbB-2 can be explained by the enzyme-inhibitor structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXjJPy6XYk4rVg90H21EOLACvtfcHk0li1Iy_mhalzzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnsFOlu7g%253D&md5=788d92fc5db9f1ecf9d3fbf3142e5c9f</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-04-1168&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-04-1168%26sid%3Dliteratum%253Aachs%26aulast%3DWood%26aufirst%3DE.%2BR.%26aulast%3DTruesdale%26aufirst%3DA.%2BT.%26aulast%3DMcDonald%26aufirst%3DO.%2BB.%26aulast%3DYuan%26aufirst%3DD.%26aulast%3DHassell%26aufirst%3DA.%26aulast%3DDickerson%26aufirst%3DS.%2BH.%26aulast%3DEllis%26aufirst%3DB.%26aulast%3DPennisi%26aufirst%3DC.%26aulast%3DHorne%26aufirst%3DE.%26aulast%3DLackey%26aufirst%3DK.%26aulast%3DAlligood%26aufirst%3DK.%2BJ.%26aulast%3DRusnak%26aufirst%3DD.%2BW.%26aulast%3DGilmer%26aufirst%3DT.%2BM.%26aulast%3DShewchuk%26aufirst%3DL.%26atitle%3DA%2520unique%2520structure%2520for%2520epidermal%2520growth%2520factor%2520receptor%2520bound%2520to%2520GW572016%2520%2528Lapatinib%2529%253A%2520Relationships%2520among%2520protein%2520conformation%252C%2520inhibitor%2520off-rate%252C%2520and%2520receptor%2520activity%2520in%2520tumor%2520cells%26jtitle%3DCancer%2520Res.%26date%3D2004%26volume%3D64%26spage%3D6652%26epage%3D6659%26doi%3D10.1158%2F0008-5472.can-04-1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roskoski, R.</span></span> <span> </span><span class="NLM_article-title">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span>. <i>Pharmacol. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">26</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.1016/j.phrs.2015.10.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.phrs.2015.10.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26529477" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2016&pages=26-48&author=R.+Roskoski&title=Classification+of+small+molecule+protein+kinase+inhibitors+based+upon+the+structures+of+their+drug-enzyme+complexes&doi=10.1016%2Fj.phrs.2015.10.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of small molecule protein kinase inhibitors based upon the structures of their drug-enzyme complexes</span></div><div class="casAuthors">Roskoski, Robert</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">26-48</span>CODEN:
                <span class="NLM_cas:coden">PHMREP</span>;
        ISSN:<span class="NLM_cas:issn">1043-6618</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Because dysregulation and mutations of protein kinases play causal roles in human disease, this family of enzymes has become one of the most important drug targets over the past two decades.  The X-ray crystal structures of 21 of the 27 FDA-approved small mol. inhibitors bound to their target protein kinases are depicted in this paper.  The structure of the enzyme-bound antagonist complex is used in the classification of these inhibitors.  Type I inhibitors bind to the active protein kinase conformation (DFG-Asp in, αC-helix in).  Type I1/2 inhibitors bind to a DFG-Asp in inactive conformation while Type II inhibitors bind to a DFG-Asp out inactive conformation.  Type I, I1/2, and type II inhibitors occupy part of the adenine binding pocket and form hydrogen bonds with the hinge region connecting the small and large lobes of the enzyme.  Type III inhibitors bind next to the ATP-binding pocket and type IV inhibitors do not bind to the ATP or peptide substrate binding sites.  Type III and IV inhibitors are allosteric in nature.  Type V inhibitors bind to two different regions of the protein kinase domain and are therefore bivalent inhibitors.  The type I-V inhibitors are reversible.  In contrast, type VI inhibitors bind covalently to their target enzyme.  Type I, I1/2, and II inhibitors are divided into A and B subtypes.  The type A inhibitors bind in the front cleft, the back cleft, and near the gatekeeper residue, all of which occur within the region sepg. the small and large lobes of the protein kinase.  The type B inhibitors bind in the front cleft and gate area but do not extend into the back cleft.  An anal. of the limited available data indicates that type A inhibitors have a long residence time (minutes to hours) while the type B inhibitors have a short residence time (seconds to minutes).  The catalytic spine includes residues from the small and large lobes and interacts with the adenine ring of ATP.  Nearly all of the approved protein kinase inhibitors occupy the adenine-binding pocket; thus it is not surprising that these inhibitors interact with nearby catalytic spine (CS) residues.  Moreover, a significant no. of approved drugs also interact with regulatory spine (RS) residues.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpH0COwqC0CabVg90H21EOLACvtfcHk0lgG5TjdTKtl6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslKksL%252FN&md5=e6b45c8644e322f30f898e763780c64b</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.phrs.2015.10.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.phrs.2015.10.021%26sid%3Dliteratum%253Aachs%26aulast%3DRoskoski%26aufirst%3DR.%26atitle%3DClassification%2520of%2520small%2520molecule%2520protein%2520kinase%2520inhibitors%2520based%2520upon%2520the%2520structures%2520of%2520their%2520drug-enzyme%2520complexes%26jtitle%3DPharmacol.%2520Res.%26date%3D2016%26volume%3D103%26spage%3D26%26epage%3D48%26doi%3D10.1016%2Fj.phrs.2015.10.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sadek, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serrya, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kafafy, A.-H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Discovery of new HER2/EGFR dual kinase inhibitors based on the anilinoquinazoline scaffold as potential anti-cancer agents</span>. <i>J. Enzyme Inhib. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">222</span>, <span class="refDoi"> DOI: 10.3109/14756366.2013.765417</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3109%2F14756366.2013.765417" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23402383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXktVSmur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=215-222&author=M.+M.+Sadekauthor=R.+A.+Serryaauthor=A.-H.+N.+Kafafyauthor=M.+Ahmedauthor=F.+Wangauthor=K.+A.+M.+Abouzid&title=Discovery+of+new+HER2%2FEGFR+dual+kinase+inhibitors+based+on+the+anilinoquinazoline+scaffold+as+potential+anti-cancer+agents&doi=10.3109%2F14756366.2013.765417"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of new HER2/EGFR dual kinase inhibitors based on the anilinoquinazoline scaffold as potential anti-cancer agents</span></div><div class="casAuthors">Sadek, Maiada M.; Serrya, Rabah A.; Kafafy, Abdel-Hamid N.; Ahmed, Marawan; Wang, Feng; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Enzyme Inhibition and Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">215-222</span>CODEN:
                <span class="NLM_cas:coden">JEIMAZ</span>;
        ISSN:<span class="NLM_cas:issn">1475-6366</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">Herein, we designed and synthesized certain anilinoquinazoline derivs. bearing bulky arylpyridinyl, arylpropenoyl and arylpyrazolyl moieties at the 4' position of the anilinoquinazoline, as potential dual HER2/EGFR kinase inhibitors.  A detailed mol. modeling study was performed by docking the synthesized compds. in the active site of the epidermal growth factor receptor (EGFR).  The synthesized compds. were further tested for their inhibitory activity on EGFR and HER2 tyrosine kinases.  The aryl 2-imino-1,2-dihydropyridine derivs. 5d and 5e displayed the most potent inhibitory activity on EGFR with IC50 equal to 2.09 and 1.94 μM, resp., and with IC50 equal to 3.98 and 1.04 μM on HER2, resp.  Furthermore, the anti-proliferative activity of these most active compds. on MDA-MB-231 breast cancer cell lines, known to overexpress EGFR, showed an IC50 range of 2.4 and 2.5 μM, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa_wzITyW3RbVg90H21EOLACvtfcHk0lgt_YtCPwGwtA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXktVSmur4%253D&md5=47495022661acb01543e95b85e0cf1a2</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.3109%2F14756366.2013.765417&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F14756366.2013.765417%26sid%3Dliteratum%253Aachs%26aulast%3DSadek%26aufirst%3DM.%2BM.%26aulast%3DSerrya%26aufirst%3DR.%2BA.%26aulast%3DKafafy%26aufirst%3DA.-H.%2BN.%26aulast%3DAhmed%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DDiscovery%2520of%2520new%2520HER2%252FEGFR%2520dual%2520kinase%2520inhibitors%2520based%2520on%2520the%2520anilinoquinazoline%2520scaffold%2520as%2520potential%2520anti-cancer%2520agents%26jtitle%3DJ.%2520Enzyme%2520Inhib.%2520Med.%2520Chem.%26date%3D2014%26volume%3D29%26spage%3D215%26epage%3D222%26doi%3D10.3109%2F14756366.2013.765417" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukouvala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gendreau, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfrey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vysotskaia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, T. S.</span></span> <span> </span><span class="NLM_article-title">EXEL-7647 inhibits mutant forms of ErbB2 associated with lapatinib resistance and neoplastic transformation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-07-4367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-07-4367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18413839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=2465-2475&author=T.+Troweauthor=S.+Boukouvalaauthor=K.+Calkinsauthor=R.+E.+Cutlerauthor=R.+Fongauthor=R.+Funkeauthor=S.+B.+Gendreauauthor=Y.+D.+Kimauthor=N.+Millerauthor=J.+R.+Woolfreyauthor=V.+Vysotskaiaauthor=J.+P.+Yangauthor=M.+E.+Gerritsenauthor=D.+J.+Matthewsauthor=P.+Lambauthor=T.+S.+Heuer&title=EXEL-7647+inhibits+mutant+forms+of+ErbB2+associated+with+lapatinib+resistance+and+neoplastic+transformation&doi=10.1158%2F1078-0432.ccr-07-4367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">EXEL-7647 Inhibits Mutant Forms of ErbB2 Associated with Lapatinib Resistance and Neoplastic Transformation</span></div><div class="casAuthors">Trowe, Torsten; Boukouvala, Sotiria; Calkins, Keith; Cutler, Richard E., Jr.; Fong, Ryan; Funke, Roel; Gendreau, Steven B.; Kim, Yong D.; Miller, Nicole; Woolfrey, John R.; Vysotskaia, Valentina; Yang, Jing Ping; Gerritsen, Mary E.; Matthews, David J.; Lamb, Peter; Heuer, Timothy S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2465-2475</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations assocd. with resistance to kinase inhibition are an important mechanism of intrinsic or acquired loss of clin. efficacy for kinase-targeted therapeutics.  We report the prospective discovery of ErbB2 mutations that confer resistance to the small-mol. inhibitor lapatinib.  We did in vitro screening using a randomly mutagenized ErbB2 expression library in Ba/F3 cells, which were dependent on ErbB2 activity for survival and growth.  Lapatinib resistance screens identified mutations at 16 different ErbB2 amino acid residues, with 12 mutated amino acids mapping to the kinase domain.  Mutations conferring the greatest lapatinib resistance cluster in the NH2-terminal kinase lobe and hinge region.  Structural computer modeling studies suggest that lapatinib resistance is caused by multiple mechanisms; including direct steric interference and restriction of conformational flexibility (the inactive state required for lapatinib binding is energetically unfavorable).  ErbB2 T798I imparts the strongest lapatinib resistance effect and is analogous to the epidermal growth factor receptor T790M, ABL T315I, and cKIT T670I gatekeeper mutations that are assocd. with clin. drug resistance.  ErbB2 mutants assocd. with lapatinib resistance transformed NIH-3T3 cells, including L755S and T733I mutations known to occur in human breast and gastric carcinomas, supporting a direct mechanism for lapatinib resistance in ErbB2-driven human cancers.  The epidermal growth factor receptor/ErbB2/vascular endothelial growth factor receptor inhibitor EXEL-7647 was found to inhibit almost all lapatinib resistance-assocd. mutations.  Furthermore, no ErbB2 mutations were found to be assocd. with EXEL-7647 resistance and lapatinib sensitivity.  Taken together, these data suggest potential target-based mechanisms of resistance to lapatinib and suggest that EXEL-7647 may be able to circumvent these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIufvHezxJo7Vg90H21EOLACvtfcHk0lj5mW5D7pFlhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D&md5=c51774479df7be7c06d2cc22e8c58afc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-07-4367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-07-4367%26sid%3Dliteratum%253Aachs%26aulast%3DTrowe%26aufirst%3DT.%26aulast%3DBoukouvala%26aufirst%3DS.%26aulast%3DCalkins%26aufirst%3DK.%26aulast%3DCutler%26aufirst%3DR.%2BE.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGendreau%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BD.%26aulast%3DMiller%26aufirst%3DN.%26aulast%3DWoolfrey%26aufirst%3DJ.%2BR.%26aulast%3DVysotskaia%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DJ.%2BP.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DMatthews%26aufirst%3DD.%2BJ.%26aulast%3DLamb%26aufirst%3DP.%26aulast%3DHeuer%26aufirst%3DT.%2BS.%26atitle%3DEXEL-7647%2520inhibits%2520mutant%2520forms%2520of%2520ErbB2%2520associated%2520with%2520lapatinib%2520resistance%2520and%2520neoplastic%2520transformation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D2465%26epage%3D2475%26doi%3D10.1158%2F1078-0432.ccr-07-4367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Diarrhea With HER2-Targeted Agents in Cancer Patients: A Systematic Review and Meta-Analysis</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">935</span>– <span class="NLM_lpage">946</span>, <span class="refDoi"> DOI: 10.1002/jcph.1382</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fjcph.1382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30707455" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhtVyjs7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=935-946&author=J.+Li&title=Diarrhea+With+HER2-Targeted+Agents+in+Cancer+Patients%3A+A+Systematic+Review+and+Meta-Analysis&doi=10.1002%2Fjcph.1382"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Diarrhea With HER2-Targeted Agents in Cancer Patients: A Systematic Review and Meta-Analysis</span></div><div class="casAuthors">Li, Jing</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">7, VIII International Conference Nanomaterials and Technologies, 2019</span>),
    <span class="NLM_cas:pages">935-946</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">To fully investigate the diarrhea of human epidermal growth factor receptor 2 (HER2)-targeted agents in cancer patients.  The relevant studies of the randomized, controlled trials (RCTs) in cancer patients treated with HER2-targeted agents were retrieved, and the systematic evaluation was conducted.  EMBASE, MEDLINE, and PubMed were searched for articles published until August 2018.  Forty-two RCTs and 21 633 patients were included.  The current meta-anal. suggested that the use of HER2-targeted agents significantly increases the risk of developing all-grade diarrhea (RR, 2.78; 95%CI, 2.37-3.25; P < .00001) and high-grade diarrhea (RR, 4.89; 95%CI, 3.09-7.75; P < .00001).  The RRs of all-grade diarrhea and high-grade diarrhea varied significantly according to drug type, control group, and treatment regimen.  The RR of all-grade diarrhea varied significantly according to tumor type.  Afatinib and neratinib tended to assoc. with the highest risk of all-grade diarrhea and high-grade diarrhea, resp.  Trastuzumab was assocd. with the lowest risk of diarrhea.  Breast cancer patients tended to have a higher risk of all-grade diarrhea than patients with nonbreast cancer when receiving a HER2-targeted agent.  The available data suggested that the use of HER2-targeted agents is assocd. with a significantly increased risk of diarrhea in cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTXA94GBpC27Vg90H21EOLACvtfcHk0lj5mW5D7pFlhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhtVyjs7jL&md5=5052d8511a449cd2581b848fc165f511</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fjcph.1382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.1382%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DDiarrhea%2520With%2520HER2-Targeted%2520Agents%2520in%2520Cancer%2520Patients%253A%2520A%2520Systematic%2520Review%2520and%2520Meta-Analysis%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2019%26volume%3D59%26spage%3D935%26epage%3D946%26doi%3D10.1002%2Fjcph.1382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee-Hoeflich, S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crocker, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pham, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munroe, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sliwkowski, M. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, H. M.</span></span> <span> </span><span class="NLM_article-title">A central role for HER3 in HER2-amplified breast cancer: Implications for targeted therapy</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>68</i></span>,  <span class="NLM_fpage">5878</span>– <span class="NLM_lpage">5887</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-08-0380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-08-0380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18632642" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXovVCnsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2008&pages=5878-5887&author=S.+T.+Lee-Hoeflichauthor=L.+Crockerauthor=E.+Yaoauthor=T.+Phamauthor=X.+Munroeauthor=K.+P.+Hoeflichauthor=M.+X.+Sliwkowskiauthor=H.+M.+Stern&title=A+central+role+for+HER3+in+HER2-amplified+breast+cancer%3A+Implications+for+targeted+therapy&doi=10.1158%2F0008-5472.can-08-0380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">A Central Role for HER3 in HER2-Amplified Breast Cancer: Implications for Targeted Therapy</span></div><div class="casAuthors">Lee-Hoeflich, Si Tuen; Crocker, Lisa; Yao, Evelyn; Pham, Thinh; Munroe, Xander; Hoeflich, Klaus P.; Sliwkowski, Mark X.; Stern, Howard M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5878-5887</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Epidermal growth factor receptor (EGFR) and HER3 each form heterodimers with HER2 and have independently been implicated as key coreceptors that drive HER2-amplified breast cancer.  Some studies suggest a dominant role for EGFR, a notion of renewed interest given the development of dual HER2/EGFR small-mol. inhibitors.  Other studies point to HER3 as the primary coreceptor.  To clarify the relative contributions of EGFR and HER3 to HER2 signaling, the authors studied receptor knockdown via small interfering RNA technol. across a panel of 6 HER2-overexpressing cell lines.  Interestingly, HER3 was as crit. as HER2 for maintaining cell proliferation in most cell lines, whereas EGFR was dispensable.  Induction of HER3 knockdown in the HER2-overexpressing BT474M1 cell line was found to inhibit growth in 3-dimensional culture and induce rapid tumor regression of in vivo xenografts.  Furthermore, preferential phosphorylation of HER3, but not EGFR, was obsd. in HER2-amplified breast cancer tissues.  Given these data suggesting HER3 as an important therapeutic target, the authors examd. the activity of pertuzumab, a HER2 antibody that inhibits HER3 signaling by blocking ligand-induced HER2/HER3 heterodimerization.  Pertuzumab inhibited ligand-dependent morphogenesis in 3-dimensional culture and induced tumor regression in the heregulin-dependent MDA-MB-175 xenograft model.  Importantly, these activities of pertuzumab were distinct from those of trastuzumab, a monoclonal antibody currently used for treatment of HER2-amplified breast cancer patients.  Thus, inhibition of HER3 may be more clin. relevant than inhibition of EGFR in HER2-amplified breast cancer and also adding pertuzumab to trastuzumab may augment therapeutic benefit by blocking HER2/HER3 signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9dxNZ1FF4bVg90H21EOLACvtfcHk0lj5mW5D7pFlhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXovVCnsbs%253D&md5=de1600e07012542e98eea06cc2db60d8</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-08-0380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-08-0380%26sid%3Dliteratum%253Aachs%26aulast%3DLee-Hoeflich%26aufirst%3DS.%2BT.%26aulast%3DCrocker%26aufirst%3DL.%26aulast%3DYao%26aufirst%3DE.%26aulast%3DPham%26aufirst%3DT.%26aulast%3DMunroe%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DSliwkowski%26aufirst%3DM.%2BX.%26aulast%3DStern%26aufirst%3DH.%2BM.%26atitle%3DA%2520central%2520role%2520for%2520HER3%2520in%2520HER2-amplified%2520breast%2520cancer%253A%2520Implications%2520for%2520targeted%2520therapy%26jtitle%3DCancer%2520Res.%26date%3D2008%26volume%3D68%26spage%3D5878%26epage%3D5887%26doi%3D10.1158%2F0008-5472.can-08-0380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jackson-Fisher, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellinger, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shum, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duong, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perkins, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gassmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muller, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kent Lloyd, K. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stern, D. F.</span></span> <span> </span><span class="NLM_article-title">Formation of Neu/ErbB2-induced mammary tumors is unaffected by loss of ErbB4</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5664</span>– <span class="NLM_lpage">5672</span>, <span class="refDoi"> DOI: 10.1038/sj.onc.1209574</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fsj.onc.1209574" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16652155" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XpsVOrsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=5664-5672&author=A.+J.+Jackson-Fisherauthor=G.+Bellingerauthor=E.+Shumauthor=J.+K.+Duongauthor=A.+S.+Perkinsauthor=M.+Gassmannauthor=W.+Mullerauthor=K.+C.+Kent+Lloydauthor=D.+F.+Stern&title=Formation+of+Neu%2FErbB2-induced+mammary+tumors+is+unaffected+by+loss+of+ErbB4&doi=10.1038%2Fsj.onc.1209574"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of Neu/ErbB2-induced mammary tumors is unaffected by loss of ErbB4</span></div><div class="casAuthors">Jackson-Fisher, A. J.; Bellinger, G.; Shum, E.; Duong, J. K.; Perkins, A. S.; Gassmann, M.; Muller, W.; Kent Lloyd, K. C.; Stern, D. F.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">41</span>),
    <span class="NLM_cas:pages">5664-5672</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The four members of the ErbB family of receptor tyrosine kinases are involved in development and tumorigenesis of the mammary gland.  Whereas the epidermal growth factor receptor, ErbB2 and ErbB3 are pos. assocd. with various cancers, clin. studies of ErbB4 in breast cancer are contradictory.  Results from tissue culture analyses and some clin. studies suggested that ErbB4 is either a tumor suppressor or is a neg. regulator of ErbB2-driven tumors.  Neu-Cre-ErbB4flox/null mice in which ErbB4 was inactivated by Cre-lox-mediated recombination in the mammary gland developed MMTV-Neu-driven mammary tumors with a similar latency period to mice with one or two wild-type ErbB4 alleles.  Moreover, there was no difference in the histologies of tumors that developed, nor in the propensity to form lung metastases.  Taken together these results suggest that ErbB4 is not a potent, highly penetrant tumor suppressor, nor is it a factor in Neu-mediated tumorigenesis in this model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr6EvUyXV8xJLVg90H21EOLACvtfcHk0ljPwn5ZdGWJHg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpsVOrsrg%253D&md5=12fc4fac982975e27b5f88be36cd74f6</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1038%2Fsj.onc.1209574&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.onc.1209574%26sid%3Dliteratum%253Aachs%26aulast%3DJackson-Fisher%26aufirst%3DA.%2BJ.%26aulast%3DBellinger%26aufirst%3DG.%26aulast%3DShum%26aufirst%3DE.%26aulast%3DDuong%26aufirst%3DJ.%2BK.%26aulast%3DPerkins%26aufirst%3DA.%2BS.%26aulast%3DGassmann%26aufirst%3DM.%26aulast%3DMuller%26aufirst%3DW.%26aulast%3DKent%2BLloyd%26aufirst%3DK.%2BC.%26aulast%3DStern%26aufirst%3DD.%2BF.%26atitle%3DFormation%2520of%2520Neu%252FErbB2-induced%2520mammary%2520tumors%2520is%2520unaffected%2520by%2520loss%2520of%2520ErbB4%26jtitle%3DOncogene%26date%3D2006%26volume%3D25%26spage%3D5664%26epage%3D5672%26doi%3D10.1038%2Fsj.onc.1209574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez-Soler, R.</span></span> <span> </span><span class="NLM_article-title">Skin toxicities associated with epidermal growth factor receptor inhibitors</span>. <i>Target. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">107</span>– <span class="NLM_lpage">119</span>, <span class="refDoi"> DOI: 10.1007/s11523-009-0114-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs11523-009-0114-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19452131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1Mzos1KhtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=107-119&author=T.+Liauthor=R.+Perez-Soler&title=Skin+toxicities+associated+with+epidermal+growth+factor+receptor+inhibitors&doi=10.1007%2Fs11523-009-0114-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Skin toxicities associated with epidermal growth factor receptor inhibitors</span></div><div class="casAuthors">Li Tianhong; Perez-Soler Roman</div><div class="citationInfo"><span class="NLM_cas:title">Targeted oncology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">107-19</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The use of epidermal growth factor receptor (EGFR) inhibitors in several epithelial tumors has increased considerably in recent years.  Currently, they are approved in non-small cell lung cancer (NSCLC), pancreatic cancer, colorectal cancer and head and neck cancer.  Skin toxicity is a class-specific side effect that is typically manifested as a papulopustular rash in the majority (45-100%) of patients receiving EGFR inhibitors.  The skin toxicity is related to the inhibition of EGFR in the skin, which is crucial for the normal development and physiology of the epidermis.  Although rarely life-threatening, skin toxicity may cause significant physical and psycho-social discomfort.  Nevertheless, the presence and severity of skin rash is associated with improved clinical efficacy in patients receiving EGFR inhibitors.  The goal of managing EGFR inhibitor-associated skin toxicity is to minimize the detrimental effects of the rash on patients' quality of life and treatment course without antagonizing the clinical efficacy of EGFR inhibitors.  There is currently no evidence-based treatment guideline to prevent or treat the EGFR inhibitor-associated skin toxicities.  Expert panels recommend a proactive, multidisciplinary approach that includes patient education and the use of a grade-based treatment algorithm.  Elucidation of the mechanisms of EGFR inhibitor-associated skin toxicity and development of mechanism-based novel therapies are urgently needed.  Preclinical data suggest topical application of a potent phosphatase inhibitor menadione (Vitamin K3) can rescue the inhibition of EGFR and downstream signaling molecules in the skin of mice receiving systemic EGFR inhibitor erlotinib or cetuximab.  A randomized, double-blinded, placebo-controlled study has been initiated to evaluate the clinical efficacy of menadione topical cream, in the treatment or prevention of EGFR inhibitor-induced skin toxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTBjiE5iFNI-Nts4TFFB11gfW6udTcc2eYt-qMqqoDh3bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1Mzos1KhtA%253D%253D&md5=bd52132174e886f4a5d63368881e1bdb</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs11523-009-0114-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11523-009-0114-0%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DT.%26aulast%3DPerez-Soler%26aufirst%3DR.%26atitle%3DSkin%2520toxicities%2520associated%2520with%2520epidermal%2520growth%2520factor%2520receptor%2520inhibitors%26jtitle%3DTarget.%2520Oncol.%26date%3D2009%26volume%3D4%26spage%3D107%26epage%3D119%26doi%3D10.1007%2Fs11523-009-0114-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shuch, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bratslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Linehan, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Srinivasan, R.</span></span> <span> </span><span class="NLM_article-title">Sarcomatoid renal cell carcinoma: A comprehensive review of the biology and current treatment strategies</span>. <i>Oncologist</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">46</span>– <span class="NLM_lpage">54</span>, <span class="refDoi"> DOI: 10.1634/theoncologist.2011-0227</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1634%2Ftheoncologist.2011-0227" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22234634" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XjslGqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=46-54&author=B.+Shuchauthor=G.+Bratslavskyauthor=W.+M.+Linehanauthor=R.+Srinivasan&title=Sarcomatoid+renal+cell+carcinoma%3A+A+comprehensive+review+of+the+biology+and+current+treatment+strategies&doi=10.1634%2Ftheoncologist.2011-0227"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Sarcomatoid renal cell carcinoma: a comprehensive review of the biology and current treatment strategies</span></div><div class="casAuthors">Shuch, Brian; Bratslavsky, Gennady; Linehan, W. Marston; Srinivasan, Ramaprasad</div><div class="citationInfo"><span class="NLM_cas:title">Oncologist</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">46-54</span>CODEN:
                <span class="NLM_cas:coden">OCOLF6</span>;
        ISSN:<span class="NLM_cas:issn">1083-7159</span>.
    
            (<span class="NLM_cas:orgname">AlphaMed Press</span>)
        </div><div class="casAbstract">A review.  Recent advancements in the mol. characterization of renal cell carcinoma altered the classification system and now kidney cancer is divided into several distinct histol. subtypes.  Although once a sep. histol. category, sarcomatoid renal cell carcinoma is no longer considered a sep. tumor type because it can occur with all histol. subtypes.  Limited research on tumors with sarcomatoid change has led to minimal progress in the understanding and treatment of these tumors.  Because the sarcomatoid variant of renal cell carcinoma can account for approx. one in six cases of advanced kidney cancer, we hope to familiarize clinicians with these tumors by describing the historic background, histol. features, mol. characterization, diagnosis, prognosis, treatment strategies, and active clin. trials of this aggressive type of tumor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAAitmhpDnN7Vg90H21EOLACvtfcHk0lg_OmFOWrIEQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjslGqsLw%253D&md5=822e5c9520c4e7ed429fff02c0237f8f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1634%2Ftheoncologist.2011-0227&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1634%252Ftheoncologist.2011-0227%26sid%3Dliteratum%253Aachs%26aulast%3DShuch%26aufirst%3DB.%26aulast%3DBratslavsky%26aufirst%3DG.%26aulast%3DLinehan%26aufirst%3DW.%2BM.%26aulast%3DSrinivasan%26aufirst%3DR.%26atitle%3DSarcomatoid%2520renal%2520cell%2520carcinoma%253A%2520A%2520comprehensive%2520review%2520of%2520the%2520biology%2520and%2520current%2520treatment%2520strategies%26jtitle%3DOncologist%26date%3D2012%26volume%3D17%26spage%3D46%26epage%3D54%26doi%3D10.1634%2Ftheoncologist.2011-0227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0lg35whdEzPSZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jani, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coleman, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Connell, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currier, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emerson, E. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harriman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kath, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pustilnik, L. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rafidi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ralston, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rossi, A. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steyn, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winter, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S. K.</span></span> <span> </span><span class="NLM_article-title">Discovery and pharmacologic characterization of CP-724,714, a selective ErbB2 tyrosine kinase inhibitor</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">9887</span>– <span class="NLM_lpage">9893</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-06-3559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F0008-5472.can-06-3559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17942920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFOnsbvE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=9887-9893&author=J.+P.+Janiauthor=R.+S.+Finnauthor=M.+Campbellauthor=K.+G.+Colemanauthor=R.+D.+Connellauthor=N.+Currierauthor=E.+O.+Emersonauthor=E.+Floydauthor=S.+Harrimanauthor=J.+C.+Kathauthor=J.+Morrisauthor=J.+D.+Moyerauthor=L.+R.+Pustilnikauthor=K.+Rafidiauthor=S.+Ralstonauthor=A.+M.+K.+Rossiauthor=S.+J.+Steynauthor=L.+Wagnerauthor=S.+M.+Winterauthor=S.+K.+Bhattacharya&title=Discovery+and+pharmacologic+characterization+of+CP-724%2C714%2C+a+selective+ErbB2+tyrosine+kinase+inhibitor&doi=10.1158%2F0008-5472.can-06-3559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Pharmacologic Characterization of CP-724,714, a Selective ErbB2 Tyrosine Kinase Inhibitor</span></div><div class="casAuthors">Jani, Jitesh P.; Finn, Richard S.; Campbell, Mary; Coleman, Kevin G.; Connell, Richard D.; Currier, Nicolas; Emerson, Erling O.; Floyd, Eugenia; Harriman, Shawn; Kath, John C.; Morris, Joel; Moyer, James D.; Pustilnik, Leslie R.; Rafidi, Kristina; Ralston, Sherry; Rossi, Ann Marie K.; Steyn, Stefanus J.; Wagner, Larry; Winter, Steven M.; Bhattacharya, Samit K.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9887-9893</span>CODEN:
                <span class="NLM_cas:coden">CNREA8</span>;
        ISSN:<span class="NLM_cas:issn">0008-5472</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Amplification and overexpression of erbB2 (Her-2/neu) proto-oncogene has been linked to human malignancies including tumors of the breast, ovary, and stomach.  It has been implicated in tumor growth, sensitivity to std. chemotherapy, prognosis of patients, and disease-free survival.  Although the clin. use of trastuzumab (Herceptin) has prolonged the survival of breast cancer patients with erbB2-overexpressing tumors, there is an urgent need for more potent and orally bioavailable small-mol. inhibitors.  CP-724,714 is a potent inhibitor of erbB2 receptor autophosphorylation in intact cells and is currently undergoing phase I clin. trials.  Here, we describe the effects of CP-724,714 in vitro and in vivo in human breast cancer models.  CP-724,714 is selective for inhibiting growth of HER2-driven cell lines.  In addn., we show that it induces G1 cell cycle block in erbB2-overexpressing BT-474 human breast carcinoma cells and inhibits erbB2 autophosphorylation in xenografts when administered p.o. to athymic mice.  It induces a marked redn. of extracellular signal-regulated kinase and Akt phosphorylation, tumor cell apoptosis, and release of caspase-3.  P.o. administration (q.d. or b.i.d.) of CP-724,714 inhibits the growth of erbB2-overexpressing tumors in athymic mice without overt adverse effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo_a6qjvEZI-rVg90H21EOLACvtfcHk0lgfgKGFRoU24A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFOnsbvE&md5=21dc688108dd46aba15694d7e3159321</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-06-3559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-06-3559%26sid%3Dliteratum%253Aachs%26aulast%3DJani%26aufirst%3DJ.%2BP.%26aulast%3DFinn%26aufirst%3DR.%2BS.%26aulast%3DCampbell%26aufirst%3DM.%26aulast%3DColeman%26aufirst%3DK.%2BG.%26aulast%3DConnell%26aufirst%3DR.%2BD.%26aulast%3DCurrier%26aufirst%3DN.%26aulast%3DEmerson%26aufirst%3DE.%2BO.%26aulast%3DFloyd%26aufirst%3DE.%26aulast%3DHarriman%26aufirst%3DS.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DMorris%26aufirst%3DJ.%26aulast%3DMoyer%26aufirst%3DJ.%2BD.%26aulast%3DPustilnik%26aufirst%3DL.%2BR.%26aulast%3DRafidi%26aufirst%3DK.%26aulast%3DRalston%26aufirst%3DS.%26aulast%3DRossi%26aufirst%3DA.%2BM.%2BK.%26aulast%3DSteyn%26aufirst%3DS.%2BJ.%26aulast%3DWagner%26aufirst%3DL.%26aulast%3DWinter%26aufirst%3DS.%2BM.%26aulast%3DBhattacharya%26aufirst%3DS.%2BK.%26atitle%3DDiscovery%2520and%2520pharmacologic%2520characterization%2520of%2520CP-724%252C714%252C%2520a%2520selective%2520ErbB2%2520tyrosine%2520kinase%2520inhibitor%26jtitle%3DCancer%2520Res.%26date%3D2007%26volume%3D67%26spage%3D9887%26epage%3D9893%26doi%3D10.1158%2F0008-5472.can-06-3559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skene, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sang, B.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snell, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwamoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirokawa, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span> <span> </span><span class="NLM_article-title">Structural analysis of the mechanism of inhibition and allosteric activation of the kinase domain of HER2 protein</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">18756</span>– <span class="NLM_lpage">18765</span>, <span class="refDoi"> DOI: 10.1074/jbc.m110.206193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1074%2Fjbc.m110.206193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21454582" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2011&pages=18756-18765&author=K.+Aertgeertsauthor=R.+Skeneauthor=J.+Yanoauthor=B.-C.+Sangauthor=H.+Zouauthor=G.+Snellauthor=A.+Jenningsauthor=K.+Iwamotoauthor=N.+Habukaauthor=A.+Hirokawaauthor=T.+Ishikawaauthor=T.+Tanakaauthor=H.+Mikiauthor=Y.+Ohtaauthor=S.+Sogabe&title=Structural+analysis+of+the+mechanism+of+inhibition+and+allosteric+activation+of+the+kinase+domain+of+HER2+protein&doi=10.1074%2Fjbc.m110.206193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Analysis of the Mechanism of Inhibition and Allosteric Activation of the Kinase Domain of HER2 Protein</span></div><div class="casAuthors">Aertgeerts, Kathleen; Skene, Robert; Yano, Jason; Sang, Bi-Ching; Zou, Hua; Snell, Gyorgy; Jennings, Andy; Iwamoto, Keiji; Habuka, Noriyuki; Hirokawa, Aki; Ishikawa, Tomoyasu; Tanaka, Toshimasa; Miki, Hiroshi; Ohta, Yoshikazu; Sogabe, Satoshi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">18756-18765</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Aberrant signaling of ErbB family members human epidermal growth factor 2 (HER2) and epidermal growth factor receptor (EGFR) is implicated in many human cancers, and HER2 expression is predictive of human disease recurrence and prognosis.  Small mol. kinase inhibitors of EGFR and of both HER2 and EGFR have received approval for the treatment of cancer.  We present the first high resoln. crystal structure of the kinase domain of HER2 in complex with a selective inhibitor to understand protein activation, inhibition, and function at the mol. level.  HER2 kinase domain crystallizes as a dimer and suggests evidence for an allosteric mechanism of activation comparable with previously reported activation mechanisms for EGFR and HER4.  A unique Gly-rich region in HER2 following the α-helix C is responsible for increased conformational flexibility within the active site and could explain the low intrinsic catalytic activity previously reported for HER2.  In addn., we solved the crystal structure of the kinase domain of EGFR in complex with a HER2/EGFR dual inhibitor (TAK-285).  Comparison with previously reported inactive and active EGFR kinase domain structures gave insight into the mechanism of HER2 and EGFR inhibition and may help guide the design and development of new cancer drugs with improved potency and selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomtAlJ0FGUX7Vg90H21EOLACvtfcHk0lglMIJdfMZOYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmtlGltL0%253D&md5=c7897f5363d6452f7b60a8ced0f2bc30</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m110.206193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m110.206193%26sid%3Dliteratum%253Aachs%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DSkene%26aufirst%3DR.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DSang%26aufirst%3DB.-C.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSnell%26aufirst%3DG.%26aulast%3DJennings%26aufirst%3DA.%26aulast%3DIwamoto%26aufirst%3DK.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DHirokawa%26aufirst%3DA.%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DSogabe%26aufirst%3DS.%26atitle%3DStructural%2520analysis%2520of%2520the%2520mechanism%2520of%2520inhibition%2520and%2520allosteric%2520activation%2520of%2520the%2520kinase%2520domain%2520of%2520HER2%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2011%26volume%3D286%26spage%3D18756%26epage%3D18765%26doi%3D10.1074%2Fjbc.m110.206193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Verdonk, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartshorn, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, R. D.</span></span> <span> </span><span class="NLM_article-title">Improved protein-ligand docking using GOLD</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">609</span>– <span class="NLM_lpage">623</span>, <span class="refDoi"> DOI: 10.1002/prot.10465</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fprot.10465" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12910460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2003&pages=609-623&author=M.+L.+Verdonkauthor=J.+C.+Coleauthor=M.+J.+Hartshornauthor=C.+W.+Murrayauthor=R.+D.+Taylor&title=Improved+protein-ligand+docking+using+GOLD&doi=10.1002%2Fprot.10465"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Improved protein-ligand docking using GOLD</span></div><div class="casAuthors">Verdonk, Marcel L.; Cole, Jason C.; Hartshorn, Michael J.; Murray, Christopher W.; Taylor, Richard D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Genetics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">609-623</span>CODEN:
                <span class="NLM_cas:coden">PSFGEY</span>;
        ISSN:<span class="NLM_cas:issn">0887-3585</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The Chemscore function was implemented as a scoring function for the protein-ligand docking program GOLD, and its performance compared to the original Goldscore function and two consensus docking protocols, "Goldscore-CS" and "Chemscore-GS," in terms of docking accuracy, prediction of binding affinities, and speed.  In the "Goldscore-CS" protocol, dockings produced with the Goldscore function are scored and ranked with the Chemscore function; in the "Chemscore-GS" protocol, dockings produced with the Chemscore function are scored and ranked with the Goldscore function.  Comparisons were made for a "clean" set of 224 protein-ligand complexes, and for two subsets of this set, one for which the ligands are "drug-like," the other for which they are "fragment-like.".  For "drug-like" and "fragment-like" ligands, the docking accuracies obtained with Chemscore and Goldscore functions are similar.  For larger ligands, Goldscore gives superior results.  Docking with the Chemscore function is up to three times faster than docking with the Goldscore function.  Both combined docking protocols give significant improvements in docking accuracy over the use of the Goldscore or Chemscore function alone.  "Goldscore-CS" gives success rates of up to 81% (top-ranked GOLD soln. within 2.0 Å of the exptl. binding mode) for the "clean list," but at the cost of long search times.  For most virtual screening applications, "Chemscore-GS" seems optimal; search settings that give docking speeds of around 0.25-1.3 min/compd. have success rates of about 78% for "drug-like" compds. and 85% for "fragment-like" compds.  In terms of producing binding energy ests., the Goldscore function appears to perform better than the Chemscore function and the two consensus protocols, particularly for faster search settings.  Even at docking speeds of around 1-2 min/compd., the Goldscore function predicts binding energies with a std. deviation of ∼10.5 kJ/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoKKgOct_CAcbVg90H21EOLACvtfcHk0ljC62HJSstyrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmvFGrsLg%253D&md5=73a627d99dab6de1ae364c8e8e5f0fea</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fprot.10465&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10465%26sid%3Dliteratum%253Aachs%26aulast%3DVerdonk%26aufirst%3DM.%2BL.%26aulast%3DCole%26aufirst%3DJ.%2BC.%26aulast%3DHartshorn%26aufirst%3DM.%2BJ.%26aulast%3DMurray%26aufirst%3DC.%2BW.%26aulast%3DTaylor%26aufirst%3DR.%2BD.%26atitle%3DImproved%2520protein-ligand%2520docking%2520using%2520GOLD%26jtitle%3DProteins%26date%3D2003%26volume%3D52%26spage%3D609%26epage%3D623%26doi%3D10.1002%2Fprot.10465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkamhawy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanath, A. N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedair, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leem, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pae, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, E. J.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5147</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26475520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5147-5154&author=A.+Elkamhawyauthor=A.+K.+Faragauthor=A.+N.+I.+Viswanathauthor=T.+M.+Bedairauthor=D.+G.+Leemauthor=K.-T.+Leeauthor=A.+N.+Paeauthor=E.+J.+Roh&title=Targeting+EGFR%2FHER2+tyrosine+kinases+with+a+new+potent+series+of+6-substituted+4-anilinoquinazoline+hybrids%3A+Design%2C+synthesis%2C+kinase+assay%2C+cell-based+assay%2C+and+molecular+docking&doi=10.1016%2Fj.bmcl.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span></div><div class="casAuthors">Elkamhawy, Ahmed; Farag, Ahmed Karam; Viswanath, Ambily Nath Indu; Bedair, Tarek M.; Leem, Dong Gyu; Lee, Kyung-Tae; Pae, Ae Nim; Roh, Eun Joo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5147-5154</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coexpression of EGFR and HER2 has been found in many tumors such as breast, ovarian, colon and prostate cancers, with poor prognosis of the patients.  Herein, our team has designed and synthesized new eighteen compds. with 6-substituted 4-anilinoquinazoline core to selectively inhibit EGFR/HER2 tyrosine kinases.  Twelve compds. showed nanomolar range of IC50 values on EGFR and/or HER2 kinases.  Accordingly, a detailed structure activity relationship (SAR) was established.  A mol. docking study demonstrated the favorable binding modes of I (X = Me, NHEt) at the ATP active site of both kinases.  A kinase selectivity profile performed for compd. 8d showed great selectivity for EGFR and HER2.  In addn., compd. I (X = Me, NHEt) and II exerted selective promising cytotoxic activity over BT-474 cell line with IC50 values of 2.70, 1.82 and 1.95 μM, resp.  From these results, we report analogs I (X = Me, NHEt) and II as promising candidates for the discovery of well-balanced compds. in terms of the kinase inhibitory potency and antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_Ln0rUhX7bVg90H21EOLACvtfcHk0ljC62HJSstyrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP&md5=0d7c9e4896863934f0190537c10c42ae</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DElkamhawy%26aufirst%3DA.%26aulast%3DFarag%26aufirst%3DA.%2BK.%26aulast%3DViswanath%26aufirst%3DA.%2BN.%2BI.%26aulast%3DBedair%26aufirst%3DT.%2BM.%26aulast%3DLeem%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DK.-T.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRoh%26aufirst%3DE.%2BJ.%26atitle%3DTargeting%2520EGFR%252FHER2%2520tyrosine%2520kinases%2520with%2520a%2520new%2520potent%2520series%2520of%25206-substituted%25204-anilinoquinazoline%2520hybrids%253A%2520Design%252C%2520synthesis%252C%2520kinase%2520assay%252C%2520cell-based%2520assay%252C%2520and%2520molecular%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5147%26epage%3D5154%26doi%3D10.1016%2Fj.bmcl.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0lhAziuAy5QHqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bellamy, F. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ou, K.</span></span> <span> </span><span class="NLM_article-title">Selective reduction of aromatic nitro compounds with stannous chloride in non acidic and non aqueous medium</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">839</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1016/s0040-4039(01)80041-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fs0040-4039%2801%2980041-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaL2cXlslSrsro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1984&pages=839-842&author=F.+D.+Bellamyauthor=K.+Ou&title=Selective+reduction+of+aromatic+nitro+compounds+with+stannous+chloride+in+non+acidic+and+non+aqueous+medium&doi=10.1016%2Fs0040-4039%2801%2980041-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Selective reduction of aromatic nitro compounds with stannous chloride in nonacidic and nonaqueous medium</span></div><div class="casAuthors">Bellamy, F. D.; Ou, K.</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">839-42</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    </div><div class="casAbstract">Arom. nitro compds. are reduced by SnCl2.2H2O in EtOH or EtOAc or by anhyd. SnCl2 in EtOH and other reducible or acid sensitive groups such as aldehyde, ketone, ester, cyano, halogen and O-benzyl remain unaffected.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqxnQ2t7MilbVg90H21EOLACvtfcHk0lg0dhZiCRVhhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2cXlslSrsro%253D&md5=db13aa6ac466d0a9bb0998184183dab2</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1016%2Fs0040-4039%2801%2980041-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0040-4039%252801%252980041-1%26sid%3Dliteratum%253Aachs%26aulast%3DBellamy%26aufirst%3DF.%2BD.%26aulast%3DOu%26aufirst%3DK.%26atitle%3DSelective%2520reduction%2520of%2520aromatic%2520nitro%2520compounds%2520with%2520stannous%2520chloride%2520in%2520non%2520acidic%2520and%2520non%2520aqueous%2520medium%26jtitle%3DTetrahedron%2520Lett.%26date%3D1984%26volume%3D25%26spage%3D839%26epage%3D842%26doi%3D10.1016%2Fs0040-4039%2801%2980041-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0ljnGDRU4nTfmQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Porter, H. K.</span></span> <i>The Zinin Reduction of Nitroarenes</i>; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>, <span class="NLM_year">2011</span>; Vol.  <span class="NLM_volume">20</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2F0471264180.or020.04" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&author=H.+K.+Porter&title=The+Zinin+Reduction+of+Nitroarenes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1002%2F0471264180.or020.04&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F0471264180.or020.04%26sid%3Dliteratum%253Aachs%26aulast%3DPorter%26aufirst%3DH.%2BK.%26btitle%3DThe%2520Zinin%2520Reduction%2520of%2520Nitroarenes%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2011%26volume%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Foucourt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubouilh-Benard, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chosson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbière, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buquet, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iannelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leblond, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marsais, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Besson, T.</span></span> <span> </span><span class="NLM_article-title">Microwave-accelerated Dimroth rearrangement for the synthesis of 4-anilino-6-nitroquinazolines. Application to an efficient synthesis of a microtubule destabilizing agent</span>. <i>Tetrahedron</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">4495</span>– <span class="NLM_lpage">4502</span>, <span class="refDoi"> DOI: 10.1016/j.tet.2010.04.066</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.tet.2010.04.066" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVSksLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=4495-4502&author=A.+Foucourtauthor=C.+Dubouilh-Benardauthor=E.+Chossonauthor=C.+Corbi%C3%A8reauthor=C.+Buquetauthor=M.+Iannelliauthor=B.+Leblondauthor=F.+Marsaisauthor=T.+Besson&title=Microwave-accelerated+Dimroth+rearrangement+for+the+synthesis+of+4-anilino-6-nitroquinazolines.+Application+to+an+efficient+synthesis+of+a+microtubule+destabilizing+agent&doi=10.1016%2Fj.tet.2010.04.066"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Microwave-accelerated Dimroth rearrangement for the synthesis of 4-anilino-6-nitroquinazolines. Application to an efficient synthesis of a microtubule destabilizing agent</span></div><div class="casAuthors">Foucourt, Alicia; Dubouilh-Benard, Carole; Chosson, Elizabeth; Corbiere, Cecile; Buquet, Catherine; Iannelli, Mauro; Leblond, Bertrand; Marsais, Francis; Besson, Thierry</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">4495-4502</span>CODEN:
                <span class="NLM_cas:coden">TETRAB</span>;
        ISSN:<span class="NLM_cas:issn">0040-4020</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Useful and rapid access to 4-anilino-6-nitroquinazolines was investigated on a multi-gram scale via a microwave-accelerated condensation and Dimroth rearrangement of anilines with imines, obtained by the reaction of anthranilonitriles with formamide dimethylacetal.  A novel short and efficient route to Azixa (EPi28495, MPC-6827), I, a microtubule destabilizing agent and apoptosis inducer, was performed successfully and demonstrates that well controlled parameters offer comfortable, safe and environmentally friendly usage of microwave technol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBk2zlyCgBXLVg90H21EOLACvtfcHk0liyXjljDfiMOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVSksLY%253D&md5=68d8109f726976e656002b8dc3aa0bad</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.tet.2010.04.066&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tet.2010.04.066%26sid%3Dliteratum%253Aachs%26aulast%3DFoucourt%26aufirst%3DA.%26aulast%3DDubouilh-Benard%26aufirst%3DC.%26aulast%3DChosson%26aufirst%3DE.%26aulast%3DCorbi%25C3%25A8re%26aufirst%3DC.%26aulast%3DBuquet%26aufirst%3DC.%26aulast%3DIannelli%26aufirst%3DM.%26aulast%3DLeblond%26aufirst%3DB.%26aulast%3DMarsais%26aufirst%3DF.%26aulast%3DBesson%26aufirst%3DT.%26atitle%3DMicrowave-accelerated%2520Dimroth%2520rearrangement%2520for%2520the%2520synthesis%2520of%25204-anilino-6-nitroquinazolines.%2520Application%2520to%2520an%2520efficient%2520synthesis%2520of%2520a%2520microtubule%2520destabilizing%2520agent%26jtitle%3DTetrahedron%26date%3D2010%26volume%3D66%26spage%3D4495%26epage%3D4502%26doi%3D10.1016%2Fj.tet.2010.04.066" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Von
Niementowski, S.</span></span> <span> </span><span class="NLM_article-title">Synthesen von chinazolinverbindungen</span>. <i>J. Prakt. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1894</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">564</span>– <span class="NLM_lpage">572</span>, <span class="refDoi"> DOI: 10.1002/prac.18950510150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fprac.18950510150" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=1894&pages=564-572&author=S.+Von%0ANiementowski&title=Synthesen+von+chinazolinverbindungen&doi=10.1002%2Fprac.18950510150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1002%2Fprac.18950510150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprac.18950510150%26sid%3Dliteratum%253Aachs%26aulast%3DVon%2BNiementowski%26aufirst%3DS.%26atitle%3DSynthesen%2520von%2520chinazolinverbindungen%26jtitle%3DJ.%2520Prakt.%2520Chem.%26date%3D1894%26volume%3D51%26spage%3D564%26epage%3D572%26doi%3D10.1002%2Fprac.18950510150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.-T.</span></span> <span> </span><span class="NLM_article-title">Novel EGFR inhibitors prepared by combination of dithiocarbamic acid esters and 4-anilinoquinazolines</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">3637</span>– <span class="NLM_lpage">3640</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2011.04.096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2011.04.096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21570843" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntFKhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=3637-3640&author=R.-D.+Liauthor=X.+Zhangauthor=Q.-Y.+Liauthor=Z.-M.+Geauthor=R.-T.+Li&title=Novel+EGFR+inhibitors+prepared+by+combination+of+dithiocarbamic+acid+esters+and+4-anilinoquinazolines&doi=10.1016%2Fj.bmcl.2011.04.096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel EGFR inhibitors prepared by combination of dithiocarbamic acid esters and 4-anilinoquinazolines</span></div><div class="casAuthors">Li, Ri-Dong; Zhang, Xin; Li, Qiao-Yan; Ge, Ze-Mei; Li, Run-Tao</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3637-3640</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">On the basis of combination strategy, a novel series of EGFR inhibitors were designed and synthesized by combination of dithiocarbamic acid esters and 4-anilinoquinazolines.  The effect of the synthesized compds. on cell proliferation was evaluated by MTT assay in three human cancer cell lines: MDA-MB-468, SK-BR-3 and HCT-116.  Some compds. were found more potent against all three cell lines and other compds. were found more potent against both MDA-MB-468 and SK-BR-3 than Lapatinib.  SAR studies revealed that the substituents on C6 and C7 positions of quinazoline, the amine component of dithiocarbamate moiety and the linker greatly affected the activity.  This work provides a promising new strategy for the prepn. of potent tyrosine kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo92wKhvesdvrVg90H21EOLACvtfcHk0liyXjljDfiMOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntFKhs7s%253D&md5=e76bffa5c3d8b62e357442dfcf74686c</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2011.04.096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2011.04.096%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DR.-D.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DQ.-Y.%26aulast%3DGe%26aufirst%3DZ.-M.%26aulast%3DLi%26aufirst%3DR.-T.%26atitle%3DNovel%2520EGFR%2520inhibitors%2520prepared%2520by%2520combination%2520of%2520dithiocarbamic%2520acid%2520esters%2520and%25204-anilinoquinazolines%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2011%26volume%3D21%26spage%3D3637%26epage%3D3640%26doi%3D10.1016%2Fj.bmcl.2011.04.096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forcellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shayhidin, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhéaume, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagüe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquin, J.-F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel quinazoline-4-piperidinesulfamide derivatives as inhibitors of NPP1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.01.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29427906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=130-149&author=E.+Forcelliniauthor=S.+Boutinauthor=C.-A.+Lefebvreauthor=E.+E.+Shayhidinauthor=M.-C.+Boulangerauthor=G.+Rh%C3%A9aumeauthor=X.+Barbeauauthor=P.+Lag%C3%BCeauthor=P.+Mathieuauthor=J.-F.+Paquin&title=Synthesis+and+biological+evaluation+of+novel+quinazoline-4-piperidinesulfamide+derivatives+as+inhibitors+of+NPP1&doi=10.1016%2Fj.ejmech.2018.01.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel quinazoline-4-piperidine sulfamide derivatives as inhibitors of NPP1</span></div><div class="casAuthors">Forcellini, Elsa; Boutin, Sophie; Lefebvre, Carole-Anne; Shayhidin, Elnur Elyar; Boulanger, Marie-Chloe; Rheaume, Gabrielle; Barbeau, Xavier; Lague, Patrick; Mathieu, Patrick; Paquin, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">130-149</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The ecto-nucleotide pyrophosphatase/phosphodiesterase-1 (NPP1) was recently shown to promote mineralization of the aortic valve, hence, its inhibition represents a significant target.  A quinazoline-4-piperidine sulfamide compd. (I) has been described as a specific and non-competitive inhibitor of NPP1.  Herein the synthesis and in vitro inhibition studies of novel quinazoline-4-piperidine sulfamide analogs using I as the lead compd. was reported.  Of the 26 derivs. prepd., four compds. were found to have Ki < 105 nM against human NPP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwRlT7Fi3_H7Vg90H21EOLACvtfcHk0lhGKXbfLVUapw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D&md5=39d4f90e63b612922cfb5a60b47bd688</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.094%26sid%3Dliteratum%253Aachs%26aulast%3DForcellini%26aufirst%3DE.%26aulast%3DBoutin%26aufirst%3DS.%26aulast%3DLefebvre%26aufirst%3DC.-A.%26aulast%3DShayhidin%26aufirst%3DE.%2BE.%26aulast%3DBoulanger%26aufirst%3DM.-C.%26aulast%3DRh%25C3%25A9aume%26aufirst%3DG.%26aulast%3DBarbeau%26aufirst%3DX.%26aulast%3DLag%25C3%25BCe%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DP.%26aulast%3DPaquin%26aufirst%3DJ.-F.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520quinazoline-4-piperidinesulfamide%2520derivatives%2520as%2520inhibitors%2520of%2520NPP1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D130%26epage%3D149%26doi%3D10.1016%2Fj.ejmech.2018.01.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, H.-X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bao, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qian, C.</span></span> <span> </span><span class="NLM_article-title">Discovery of 7-(4-(3-Ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)- N -hydroxyheptanamide (CUDC-101) as a Potent Multi-Acting HDAC, EGFR, and HER2 Inhibitor for the Treatment of Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2009</span>, <span class="refDoi"> DOI: 10.1021/jm901453q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901453q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhslSgsb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2000-2009&author=X.+Caiauthor=H.-X.+Zhaiauthor=J.+Wangauthor=J.+Forresterauthor=H.+Quauthor=L.+Yinauthor=C.-J.+Laiauthor=R.+Baoauthor=C.+Qian&title=Discovery+of+7-%284-%283-Ethynylphenylamino%29-7-methoxyquinazolin-6-yloxy%29-+N+-hydroxyheptanamide+%28CUDC-101%29+as+a+Potent+Multi-Acting+HDAC%2C+EGFR%2C+and+HER2+Inhibitor+for+the+Treatment+of+Cancer&doi=10.1021%2Fjm901453q"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of 7-(4-(3-Ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)-N-hydroxyheptanamide (CUDC-101) as a Potent Multi-Acting HDAC, EGFR, and HER2 Inhibitor for the Treatment of Cancer</span></div><div class="casAuthors">Cai, Xiong; Zhai, Hai-Xiao; Wang, Jing; Forrester, Jeffrey; Qu, Hui; Yin, Ling; Lai, Cheng-Jung; Bao, Rudi; Qian, Changgeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2000-2009</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">By incorporating histone deacetylase (HDAC) inhibitory functionality into the pharmacophore of the epidermal growth factor receptor (EGFR) and human epidermal growth factor receptor 2 (HER2) inhibitors, we synthesized a novel series of compds. with potent, multiacting HDAC, EGFR, and HER2 inhibition and identified 7-(4-(3-ethynylphenylamino)-7-methoxyquinazolin-6-yloxy)-N-hydroxyheptanamide 8 (CUDC-101) as a drug candidate, which is now in clin. development. 8 displays potent in vitro inhibitory activity against HDAC, EGFR, and HER2 with an IC50 of 4.4, 2.4, and 15.7 nM, resp.  In most tumor cell lines tested, 8 exhibits efficient antiproliferative activity with greater potency than vorinostat (SAHA), erlotinib, lapatinib, and combinations of vorinostat/erlotinib and vorinostat/lapatinib.  In vivo, 8 promotes tumor regression or inhibition in various cancer xenograft models including nonsmall cell lung cancer (NSCLC), liver, breast, head and neck, colon, and pancreatic cancers.  These results suggest that a single compd. that simultaneously inhibits HDAC, EGFR, and HER2 may offer greater therapeutic benefits in cancer over single-acting agents through the interference with multiple pathways and potential synergy among HDAC and EGFR/HER2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodBQl1iMXW3rVg90H21EOLACvtfcHk0lg3uwkrvp2fIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhslSgsb0%253D&md5=d4dc339f2fd750d52e511abf3dbf18bd</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Fjm901453q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901453q%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DX.%26aulast%3DZhai%26aufirst%3DH.-X.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DForrester%26aufirst%3DJ.%26aulast%3DQu%26aufirst%3DH.%26aulast%3DYin%26aufirst%3DL.%26aulast%3DLai%26aufirst%3DC.-J.%26aulast%3DBao%26aufirst%3DR.%26aulast%3DQian%26aufirst%3DC.%26atitle%3DDiscovery%2520of%25207-%25284-%25283-Ethynylphenylamino%2529-7-methoxyquinazolin-6-yloxy%2529-%2520N%2520-hydroxyheptanamide%2520%2528CUDC-101%2529%2520as%2520a%2520Potent%2520Multi-Acting%2520HDAC%252C%2520EGFR%252C%2520and%2520HER2%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2000%26epage%3D2009%26doi%3D10.1021%2Fjm901453q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hou, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Structure-Activity Relationships of A Novel Class of Dithiocarbamic Acid Esters as Anticancer Agent</span>. <i>Arch. Pharm.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">320</span>, <span class="refDoi"> DOI: 10.1002/ardp.201000259</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fardp.201000259" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21509804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkvFGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2011&pages=320&author=X.+Houauthor=Z.+Geauthor=T.+Wangauthor=W.+Guoauthor=J.+Wuauthor=J.+Cuiauthor=C.+Laiauthor=R.+Li&title=Synthesis+and+Structure-Activity+Relationships+of+A+Novel+Class+of+Dithiocarbamic+Acid+Esters+as+Anticancer+Agent&doi=10.1002%2Fardp.201000259"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and Structure-Activity Relationships of A Novel Class of Dithiocarbamic Acid Esters as Anticancer Agent</span></div><div class="casAuthors">Hou, Xueling; Ge, Zemei; Wang, Tingmin; Guo, Wei; Wu, Jun; Cui, Jingrong; Lai, Chingsan; Li, Runtao</div><div class="citationInfo"><span class="NLM_cas:title">Archiv der Pharmazie (Weinheim, Germany)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">320-332</span>CODEN:
                <span class="NLM_cas:coden">ARPMAS</span>;
        ISSN:<span class="NLM_cas:issn">0365-6233</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Based on a novel lead compd. 4-methylpiperazine-1-carbodithioic acid 3-cyano-3,3-diphenylpropyl ester [I (X = NMe)(II)], the systematic structural modification was carried out.  All the synthesized compds. were evaluated for their in-vitro anticancer activities on four to six different cell lines at three different concns.  Most of the tested compds. could selectively inhibit the growth of HL-60 and Bel-7402 cell lines at a medium concn.  Four compds. {III [R = H (IV), Me (V)], I [X = NH (VI), SO2 (VII)]} were selected for the IC50 test, and the results revealed that three compds. (V, VI and VII) showed almost the same or a slightly weaker activity than compd. against HL-60, and three compds. (IV, V and VI) showed > 2-fold higher potency than compd. II against Bel-7402.  The in-vivo efficacy of VI·HCl was evaluated with transplanted hepatocyte carcinoma 22 as an in-vivo test model.  It was found that VI·HCl could inhibit significantly the growth of tumor, and that this effect was dose-dependent.  Meanwhile, the compd. VI·HCl showed low toxicity compared with compd. II·HCl as evidenced by the little body-wt. loss.  These results confirmed that compd. VI·HCl is more potent than the lead compd. II·HCl.  Preliminary structure-activity relationships indicated that: (a) Both nitrile group and the cyclic amine contg. at least two nitrogens were indispensable moieties to keep the activity; (b) substitution of the piperazine ring is unfavorable for the improvement of activity; (c) the suitable linker joining the piperazinyl dithiocarboxyl and diphenylacetonitril group should be ethylene; (d) a non-coplanar arrangement of the two benzene rings appears to be essential for activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOKLHG3n1QpLVg90H21EOLACvtfcHk0lgd4tMfXwl5Bg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkvFGhsL4%253D&md5=a3547077b21c44e43d9741bb0259a353</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1002%2Fardp.201000259&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fardp.201000259%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DGuo%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DCui%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DSynthesis%2520and%2520Structure-Activity%2520Relationships%2520of%2520A%2520Novel%2520Class%2520of%2520Dithiocarbamic%2520Acid%2520Esters%2520as%2520Anticancer%2520Agent%26jtitle%3DArch.%2520Pharm.%26date%3D2011%26volume%3D344%26spage%3D320%26doi%3D10.1002%2Fardp.201000259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, R. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou-Seri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eldehna, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, N. S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elgazwi, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghabbour, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halaweish, F. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou El Ella, D. A.</span></span> <span> </span><span class="NLM_article-title">Novel series of 6-(2-substitutedacetamido)-4-anilinoquinazolines as EGFR-ERK signal transduction inhibitors in MCF-7 breast cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">782</span>– <span class="NLM_lpage">796</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.024" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30047410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1aksb7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=782-796&author=R.+S.+M.+Ismailauthor=S.+M.+Abou-Seriauthor=W.+M.+Eldehnaauthor=N.+S.+M.+Ismailauthor=S.+M.+Elgazwiauthor=H.+A.+Ghabbourauthor=M.+S.+Ahmedauthor=F.+T.+Halaweishauthor=D.+A.+Abou+El+Ella&title=Novel+series+of+6-%282-substitutedacetamido%29-4-anilinoquinazolines+as+EGFR-ERK+signal+transduction+inhibitors+in+MCF-7+breast+cancer+cells&doi=10.1016%2Fj.ejmech.2018.06.024"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Novel series of 6-(2-substituted acetamido)-4-anilinoquinazolines as EGFR-ERK signal transduction inhibitors in MCF-7 breast cancer cells</span></div><div class="casAuthors">Ismail, Rania S. M.; Abou-Seri, Sahar M.; Eldehna, Wagdy M.; Ismail, Nasser S. M.; Elgazwi, Sara M.; Ghabbour, Hazem A.; Ahmed, Mahmoud Salama; Halaweish, Fathi T.; Abou El Ella, Dalal A.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">782-796</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The design and synthesis of two series of 6-(2-substituted acetamido)-4-anilinoquinazolines I [R = 3-Cl, 3-CF3, 3-Br, etc.; R1 = pyrrolidino, piperidino, morpholino, 1-methylpiperazin-4-yl] and II [R2 = pyrrolidino, morpholino; R3 = H, F] as EGFR inhibitors was reported.  All the newly synthesized quinazoline derivs. I and II were in-vitro evaluated for their anti-proliferative activity towards MCF-7 (Breast Cancer) and HepG2 (Hepatocellular carcinoma) cell lines.  In particular, compd. I [R = 3-Br, R1 = piperidino] showed significant inhibitory activity against MCF-7 and HepG2 cell lines (IC50 = 3 and 16μM, resp.), compared to that of Erlotinib (IC50 = 20 and 25μM, resp.).  Western blotting of compd. I [R = 3-Br, R1 = piperidino] at MCF-7 cell line revealed the dual inhibitory activity of compd. I [R = 3-Br, R1 = piperidino] towards diminishing the phosphorylated levels for EGFR and ERK.  Also, ELISA assay confirmed the anti-EGFR activity of compd. I [R = 3-Br, R1 = piperidino] (IC50 = 0.037 μM).  Finally, a mol. docking study showed the potential binding mode of compd. I [R = 3-Br, R1 = piperidino] within the ATP catalytic binding site of EGFR, exhibiting similar binding mode to EGFR inhibitor Erlotinib.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoehX6XD-UKErVg90H21EOLACvtfcHk0ljEplBxbNDA9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1aksb7M&md5=db4ed7de18503f4d793983db26a3b9d2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.024%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DR.%2BS.%2BM.%26aulast%3DAbou-Seri%26aufirst%3DS.%2BM.%26aulast%3DEldehna%26aufirst%3DW.%2BM.%26aulast%3DIsmail%26aufirst%3DN.%2BS.%2BM.%26aulast%3DElgazwi%26aufirst%3DS.%2BM.%26aulast%3DGhabbour%26aufirst%3DH.%2BA.%26aulast%3DAhmed%26aufirst%3DM.%2BS.%26aulast%3DHalaweish%26aufirst%3DF.%2BT.%26aulast%3DAbou%2BEl%2BElla%26aufirst%3DD.%2BA.%26atitle%3DNovel%2520series%2520of%25206-%25282-substitutedacetamido%2529-4-anilinoquinazolines%2520as%2520EGFR-ERK%2520signal%2520transduction%2520inhibitors%2520in%2520MCF-7%2520breast%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D782%26epage%3D796%26doi%3D10.1016%2Fj.ejmech.2018.06.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Noren, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foos, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasquale, E. B.</span></span> <span> </span><span class="NLM_article-title">The EphB4 receptor suppresses breast cancer cell tumorigenicity through an Abl-Crk pathway</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">815</span>– <span class="NLM_lpage">825</span>, <span class="refDoi"> DOI: 10.1038/ncb1438</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fncb1438" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16862147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XnsFWqtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2006&pages=815-825&author=N.+K.+Norenauthor=G.+Foosauthor=C.+A.+Hauserauthor=E.+B.+Pasquale&title=The+EphB4+receptor+suppresses+breast+cancer+cell+tumorigenicity+through+an+Abl-Crk+pathway&doi=10.1038%2Fncb1438"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">The EphB4 receptor suppresses breast cancer cell tumorigenicity through an Abl-Crk pathway</span></div><div class="casAuthors">Noren, Nicole K.; Foos, Gabriele; Hauser, Craig A.; Pasquale, Elena B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">815-825</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent evidence supports a role for EphB receptor tyrosine kinases as tumor suppressors in colorectal and prostate cancer.  However, it is unclear how these receptors inhibit cancer cell tumorigenicity - an activity that is highly unusual for a family of receptor tyrosine kinases.  Here, we report that the EphB4 receptor can behave as a tumor suppressor in a mouse xenograft model of breast cancer when stimulated by its ligand, ephrin-B2.  In breast cancer cells, EphB4 activates an antioncogenic pathway involving Abl family tyrosine kinases and the Crk adaptor protein.  This Abl-Crk pathway inhibits breast cancer cell viability and proliferation in addn. to motility and invasion, and also downregulates the pro-invasive matrix metalloprotease, MMP-2.  Consistent with these effects, EphB4 and the Abl-Crk pathway are constitutively active in non-transformed mammary epithelial cells.  These findings identify a novel Eph receptor signaling pathway with tumor-suppressor activity and predict that therapeutic intervention to activate EphB4 signaling will inhibit tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQ4SmHZkAiX7Vg90H21EOLACvtfcHk0lhG_5dFHOBJHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XnsFWqtbY%253D&md5=d628bfa7d0ec8ac407eff0b6ab024978</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1038%2Fncb1438&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1438%26sid%3Dliteratum%253Aachs%26aulast%3DNoren%26aufirst%3DN.%2BK.%26aulast%3DFoos%26aufirst%3DG.%26aulast%3DHauser%26aufirst%3DC.%2BA.%26aulast%3DPasquale%26aufirst%3DE.%2BB.%26atitle%3DThe%2520EphB4%2520receptor%2520suppresses%2520breast%2520cancer%2520cell%2520tumorigenicity%2520through%2520an%2520Abl-Crk%2520pathway%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2006%26volume%3D8%26spage%3D815%26epage%3D825%26doi%3D10.1038%2Fncb1438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gil-Henn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patsialou, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warren, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Condeelis, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koleske, A. J.</span></span> <span> </span><span class="NLM_article-title">Arg/Abl2 promotes invasion and attenuates proliferation of breast cancer in vivo</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">2622</span>– <span class="NLM_lpage">2630</span>, <span class="refDoi"> DOI: 10.1038/onc.2012.284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fonc.2012.284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22777352" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFGmsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=2622-2630&author=H.+Gil-Hennauthor=A.+Patsialouauthor=Y.+Wangauthor=M.+S.+Warrenauthor=J.+S.+Condeelisauthor=A.+J.+Koleske&title=Arg%2FAbl2+promotes+invasion+and+attenuates+proliferation+of+breast+cancer+in+vivo&doi=10.1038%2Fonc.2012.284"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Arg/Abl2 promotes invasion and attenuates proliferation of breast cancer in vivo</span></div><div class="casAuthors">Gil-Henn, H.; Patsialou, A.; Wang, Y.; Warren, M. S.; Condeelis, J. S.; Koleske, A. J.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2622-2630</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor progression is a complex, multistep process involving accumulation of genetic aberrations and alterations in gene expression patterns leading to uncontrolled cell division, invasion into surrounding tissue and finally dissemination and metastasis.  We have previously shown that the Arg/Abl2 non-receptor tyrosine kinase acts downstream of the EGF receptor and Src tyrosine kinases to promote invadopodium function in breast cancer cells, thereby promoting their invasiveness.  However, whether and how Arg contributes to tumor development and dissemination in vivo has never been investigated.  Using a mouse xenograft model, we show that knocking down Arg in breast cancer cells leads to increased tumor cell proliferation and significantly enlarged tumor size.  Despite having larger tumors, the Arg-knockdown (Arg KD) tumor-bearing mice exhibit significant redns. in tumor cell invasion, intravasation into blood vessels and spontaneous metastasis to lungs.  Interestingly, we found that proliferation-assocd. genes in the Ras-MAPK (mitogen-activated protein kinase) pathway are upregulated in Arg KD breast cancer cells, as is Ras-MAPK signaling, while invasion-assocd. genes are significantly downregulated.  These data suggest that Arg promotes tumor cell invasion and dissemination, while simultaneously inhibiting tumor growth.  We propose that Arg acts as a switch in metastatic cancer cells that governs the decision to grow or go' (divide or invade).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroI_rJvnATw7Vg90H21EOLACvtfcHk0liGOgvFzDVw7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFGmsLc%253D&md5=e2303167a3df6b3d2ffbbb7a1d1213cc</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.284%26sid%3Dliteratum%253Aachs%26aulast%3DGil-Henn%26aufirst%3DH.%26aulast%3DPatsialou%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DWarren%26aufirst%3DM.%2BS.%26aulast%3DCondeelis%26aufirst%3DJ.%2BS.%26aulast%3DKoleske%26aufirst%3DA.%2BJ.%26atitle%3DArg%252FAbl2%2520promotes%2520invasion%2520and%2520attenuates%2520proliferation%2520of%2520breast%2520cancer%2520in%2520vivo%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D2622%26epage%3D2630%26doi%3D10.1038%2Fonc.2012.284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, S.-B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brady, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, L.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sahin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esteva, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, D.</span></span> <span> </span><span class="NLM_article-title">Combating trastuzumab resistance by targeting SRC, a common node downstream of multiple resistance pathways</span>. <i>Nat. Med.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">461</span>– <span class="NLM_lpage">469</span>, <span class="refDoi"> DOI: 10.1038/nm.2309</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fnm.2309" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21399647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjtVGntb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=461-469&author=S.+Zhangauthor=W.-C.+Huangauthor=P.+Liauthor=H.+Guoauthor=S.-B.+Pohauthor=S.+W.+Bradyauthor=Y.+Xiongauthor=L.-M.+Tsengauthor=S.-H.+Liauthor=Z.+Dingauthor=A.+A.+Sahinauthor=F.+J.+Estevaauthor=G.+N.+Hortobagyiauthor=D.+Yu&title=Combating+trastuzumab+resistance+by+targeting+SRC%2C+a+common+node+downstream+of+multiple+resistance+pathways&doi=10.1038%2Fnm.2309"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Combating trastuzumab resistance by targeting SRC, a common node downstream of multiple resistance pathways</span></div><div class="casAuthors">Zhang, Siyuan; Huang, Wen-Chien; Li, Ping; Guo, Hua; Poh, Say-Bee; Brady, Samuel W.; Xiong, Yan; Tseng, Ling-Ming; Li, Shau-Hsuan; Ding, Zhaoxi; Sahin, Aysegul A.; Esteva, Francisco J.; Hortobagyi, Gabriel N.; Yu, Dihua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">461-469</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Trastuzumab is a successful rationally designed ERBB2-targeted therapy.  However, about half of individuals with ERBB2-overexpressing breast cancer do not respond to trastuzumab-based therapies, owing to various resistance mechanisms.  Clin. applicable regimens for overcoming trastuzumab resistance of different mechanisms are not yet available.  We show that the nonreceptor tyrosine kinase c-SRC (SRC) is a key modulator of trastuzumab response and a common node downstream of multiple trastuzumab resistance pathways.  We find that SRC is activated in both acquired and de novo trastuzumab-resistant cells and uncover a novel mechanism of SRC regulation involving dephosphorylation by PTEN.  Increased SRC activation conferred considerable trastuzumab resistance in breast cancer cells and correlated with trastuzumab resistance in patients.  Targeting SRC in combination with trastuzumab sensitized multiple lines of trastuzumab-resistant cells to trastuzumab and eliminated trastuzumab-resistant tumors in vivo, suggesting the potential clin. application of this strategy to overcome trastuzumab resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrAglfHbddvf7Vg90H21EOLACvtfcHk0ljZQDJYd7T3lQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjtVGntb8%253D&md5=e316dee0106724d2c593358dae9c7801</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnm.2309&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.2309%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.-C.%26aulast%3DLi%26aufirst%3DP.%26aulast%3DGuo%26aufirst%3DH.%26aulast%3DPoh%26aufirst%3DS.-B.%26aulast%3DBrady%26aufirst%3DS.%2BW.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DTseng%26aufirst%3DL.-M.%26aulast%3DLi%26aufirst%3DS.-H.%26aulast%3DDing%26aufirst%3DZ.%26aulast%3DSahin%26aufirst%3DA.%2BA.%26aulast%3DEsteva%26aufirst%3DF.%2BJ.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DYu%26aufirst%3DD.%26atitle%3DCombating%2520trastuzumab%2520resistance%2520by%2520targeting%2520SRC%252C%2520a%2520common%2520node%2520downstream%2520of%2520multiple%2520resistance%2520pathways%26jtitle%3DNat.%2520Med.%26date%3D2011%26volume%3D17%26spage%3D461%26epage%3D469%26doi%3D10.1038%2Fnm.2309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Golubovskaya, V.</span></span> <span> </span><span class="NLM_article-title">Targeting FAK in human cancer: from finding to first clinical trials</span>. <i>Front. Biosci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">687</span>– <span class="NLM_lpage">706</span>, <span class="refDoi"> DOI: 10.2741/4236</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2741%2F4236" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24389213" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=687-706&author=V.+Golubovskaya&title=Targeting+FAK+in+human+cancer%3A+from+finding+to+first+clinical+trials&doi=10.2741%2F4236"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting FAK in human cancer: from finding to first clinical trials</span></div><div class="casAuthors">Golubovskaya, Vita M.</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in Bioscience, Landmark Edition</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">687-706, 20</span>CODEN:
                <span class="NLM_cas:coden">FRBIF6</span>;
        ISSN:<span class="NLM_cas:issn">1093-4715</span>.
    
            (<span class="NLM_cas:orgname">Frontiers in Bioscience</span>)
        </div><div class="casAbstract">A review.  It is twenty years since Focal Adhesion Kinase (FAK) was found to be overexpressed in many types of human cancer.  FAK plays an important role in adhesion, spreading, motility, invasion, metastasis, survival, angiogenesis, and recently has been found to play an important role as well in epithelial to mesenchymal transition (EMT), cancer stem cells and tumor microenvironment.  FAK has kinase-dependent and kinase independent scaffolding, cytoplasmic and nuclear functions.  Several years ago FAK was proposed as a potential therapeutic target; the first clin. trials were just reported, and they supported further studies of FAK as a promising therapeutic target.  This review discusses the main functions of FAK in cancer, and specifically focuses on recent novel findings on the role of FAK in cancer stem cells, microenvironment, epithelial-to-mesenchymal transition, invasion, metastasis, and also highlight new approaches of targeting FAK and critically discuss challenges that lie ahead for its targeted therapeutics.  The review provides a summary of translational approaches of FAK-targeted and combination therapies and outline perspectives and future directions of FAK research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5EGzIEc6UMbVg90H21EOLACvtfcHk0lgDiguMXAiIfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtl2ktb%252FI&md5=977f2530b2d4ed073e84ec64e498c3a3</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.2741%2F4236&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2741%252F4236%26sid%3Dliteratum%253Aachs%26aulast%3DGolubovskaya%26aufirst%3DV.%26atitle%3DTargeting%2520FAK%2520in%2520human%2520cancer%253A%2520from%2520finding%2520to%2520first%2520clinical%2520trials%26jtitle%3DFront.%2520Biosci.%26date%3D2014%26volume%3D19%26spage%3D687%26epage%3D706%26doi%3D10.2741%2F4236" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span> <span> </span><span class="NLM_article-title">Structure-based modification of carbonyl-diphenylpyrimidines (Car-DPPYs) as a novel focal adhesion kinase (FAK) inhibitor against various stubborn cancer cells</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>172</i></span>,  <span class="NLM_fpage">154</span>– <span class="NLM_lpage">162</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.04.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2019.04.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30978560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsVSgtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=172&publication_year=2019&pages=154-162&author=L.+Wangauthor=M.+Aiauthor=J.+Yuauthor=L.+Jinauthor=C.+Wangauthor=Z.+Liuauthor=X.+Shuauthor=Z.+Tangauthor=K.+Liuauthor=H.+Luoauthor=W.+Guanauthor=X.+Sunauthor=X.+Ma&title=Structure-based+modification+of+carbonyl-diphenylpyrimidines+%28Car-DPPYs%29+as+a+novel+focal+adhesion+kinase+%28FAK%29+inhibitor+against+various+stubborn+cancer+cells&doi=10.1016%2Fj.ejmech.2019.04.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based modification of carbonyl-diphenylpyrimidines (Car-DPPYs) as a novel focal adhesion kinase (FAK) inhibitor against various stubborn cancer cells</span></div><div class="casAuthors">Wang, Luhong; Ai, Min; Yu, Jiawen; Jin, Lingling; Wang, Changyuan; Liu, Zhihao; Shu, Xiaohong; Tang, Zeyao; Liu, Kexin; Luo, Hui; Guan, Wenshun; Sun, Xiuli; Ma, Xiaodong</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">172</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">154-162</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A family of carbonyl-substituted diphenylpyrimidine derivs. (Car-DPPYs) with strong activity against focal adhesion kinase (FAK), were described in this manuscript.  Among them, compds. 7a(I) (IC50 = 5.17 nM) and 7f(II) (IC50 = 2.58 nM) displayed equal anti-FAK enzymic activity to the lead compd. TAE226 (6.79 nM).  In particular, I also exhibited strong antiproliferative activity against several stubborn cancer cells, including AsPC-1 cells (IC50 = 0.105 μM), BxPC-3 cells (IC50 = 0.090 μM), and MCF-7/ADR cells (IC50 = 0.59 μM).  Addnl., I also showed great antitumor efficacy in vivo via aAsPC-1 cancer Xenograft mouse model.  The preliminary mechanism study by Western blot anal. revealed that I repressed FAK phosphorylation in AsPC cancer cells.  Taken together, the results indicate that I may serve as a promising preclin. candidate for treatment of stubborn cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3jTLS3BSpz7Vg90H21EOLACvtfcHk0liZN7al77uHCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsVSgtbg%253D&md5=1e37d0d16bf3d6c86238ff2bcde12259</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.04.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.04.004%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAi%26aufirst%3DM.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DShu%26aufirst%3DX.%26aulast%3DTang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DGuan%26aufirst%3DW.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMa%26aufirst%3DX.%26atitle%3DStructure-based%2520modification%2520of%2520carbonyl-diphenylpyrimidines%2520%2528Car-DPPYs%2529%2520as%2520a%2520novel%2520focal%2520adhesion%2520kinase%2520%2528FAK%2529%2520inhibitor%2520against%2520various%2520stubborn%2520cancer%2520cells%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D172%26spage%3D154%26epage%3D162%26doi%3D10.1016%2Fj.ejmech.2019.04.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gautam, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, K. W.</span></span> <span> </span><span class="NLM_article-title">Inhibition of tumor growth and angiogenesis of tamoxifen-resistant breast cancer cells by ruxolitinib, a selective JAK2 inhibitor</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">3981</span>– <span class="NLM_lpage">3989</span>, <span class="refDoi"> DOI: 10.3892/ol.2019.10059</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3892%2Fol.2019.10059" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30930994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlShs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2019&pages=3981-3989&author=J.+W.+Kimauthor=J.+Gautamauthor=J.+E.+Kimauthor=J.+A.+Kimauthor=K.+W.+Kang&title=Inhibition+of+tumor+growth+and+angiogenesis+of+tamoxifen-resistant+breast+cancer+cells+by+ruxolitinib%2C+a+selective+JAK2+inhibitor&doi=10.3892%2Fol.2019.10059"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of tumor growth and angiogenesis of tamoxifen-resistant breast cancer cells by ruxolitinib, a selective JAK2 inhibitor</span></div><div class="casAuthors">Kim, Ji Won; Gautam, Jaya; Kim, Ji Eun; Kim, Jung-Ae; Kang, Keon Wook</div><div class="citationInfo"><span class="NLM_cas:title">Oncology Letters</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">3981-3989</span>CODEN:
                <span class="NLM_cas:coden">OLNEB5</span>;
        ISSN:<span class="NLM_cas:issn">1792-1082</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Tamoxifen (TAM) is the most widely used treatment for estrogen receptor-pos. breast cancer patients.  Unfortunately, the majority of these patients exhibit TAM resistance following treatment.  We previously reported that proliferation and migration were greater in TAM-resistant MCF-7 (TAMR-MCF-7) cells than in parental MCF-7 cells.  Janus kinases (JAKs) are cytosolic tyrosine kinases that transduce signals from plasma membrane cytokines and growth factor receptors.  JAK2 selectively phosphorylates signal transducer and activator of transcription (STAT)-3, and the JAK2-STAT3 signaling pathway is known as a crucial signaling pathway for the regulation of cancer progression and metastasis.  In the present study, basal phosphorylation of STAT3 was revealed to be greater in TAMR-MCF-7 cells than in control MCF-7 cells.  Ruxolitinib, a potent JAK2 inhibitor, was demonstrated to attenuate STAT3 phosphorylation and the proliferation of TAMR-MCF-7 cells.  Ruxolitinib also suppressed the enhanced cell migration of TAMR-MCF-7 cells through the inhibition of epithelial mesenchymal transition.  Vascular endothelial growth factor (VEGF), a representative target gene of the JAK2-STAT3 pathway, functions as a key regulator of invasion and angiogenesis.  Ruxolitinib significantly inhibited VEGF mRNA expression and transcriptional activity.  The present study also performed a chick embryo chorioallantoic membrane assay to assess tumor growth and angiogenesis in TAMR-MCF-7 cells.  Ruxolitinib reduced tumor wt. and the no. of blood vessels produced by TAMR-MCF-7 cells in a concn.-dependent manner.  These results indicated that JAK2 could be a new therapeutic target for TAM-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6j7dPYlq207Vg90H21EOLACvtfcHk0liiHSfquskqfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlShs77K&md5=88bb39074c18bd6a995d10867cad3801</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.3892%2Fol.2019.10059&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2019.10059%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DGautam%26aufirst%3DJ.%26aulast%3DKim%26aufirst%3DJ.%2BE.%26aulast%3DKim%26aufirst%3DJ.%2BA.%26aulast%3DKang%26aufirst%3DK.%2BW.%26atitle%3DInhibition%2520of%2520tumor%2520growth%2520and%2520angiogenesis%2520of%2520tamoxifen-resistant%2520breast%2520cancer%2520cells%2520by%2520ruxolitinib%252C%2520a%2520selective%2520JAK2%2520inhibitor%26jtitle%3DOncol.%2520Lett.%26date%3D2019%26volume%3D17%26spage%3D3981%26epage%3D3989%26doi%3D10.3892%2Fol.2019.10059" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Butti, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Das, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gunasekaran, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yadav, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kundu, G. C.</span></span> <span> </span><span class="NLM_article-title">Receptor tyrosine kinases (RTKs) in breast cancer: signaling, therapeutic implications and challenges</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">34</span>– <span class="NLM_lpage">51</span>, <span class="refDoi"> DOI: 10.1186/s12943-018-0797-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fs12943-018-0797-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29455658" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=34-51&author=R.+Buttiauthor=S.+Dasauthor=V.+P.+Gunasekaranauthor=A.+S.+Yadavauthor=D.+Kumarauthor=G.+C.+Kundu&title=Receptor+tyrosine+kinases+%28RTKs%29+in+breast+cancer%3A+signaling%2C+therapeutic+implications+and+challenges&doi=10.1186%2Fs12943-018-0797-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Receptor tyrosine kinases (RTKs) in breast cancer: signaling, therapeutic implications and challenges</span></div><div class="casAuthors">Butti, Ramesh; Das, Sumit; Gunasekaran, Vinoth Prasanna; Yadav, Amit Singh; Kumar, Dhiraj; Kundu, Gopal C.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">34/1-34/18</span>CODEN:
                <span class="NLM_cas:coden">MCOACG</span>;
        ISSN:<span class="NLM_cas:issn">1476-4598</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Breast cancer is a multifactorial disease and driven by aberrant regulation of cell signaling pathways due to the acquisition of genetic and epigenetic changes.  An array of growth factors and their receptors is involved in cancer development and metastasis.  Receptor Tyrosine Kinases (RTKs) constitute a class of receptors that play important role in cancer progression.  RTKs are cell surface receptors with specialized structural and biol. features which respond to environmental cues by initiating appropriate signaling cascades in tumor cells.  RTKs are known to regulate various downstream signaling pathways such as MAPK, PI3K/Akt and JAK/STAT.  These pathways have a pivotal role in the regulation of cancer stemness, angiogenesis and metastasis.  These pathways are also imperative for a reciprocal interaction of tumor and stromal cells.  Multi-faceted role of RTKs renders them amenable to therapy in breast cancer.  However, structural mutations, gene amplification and alternate pathway activation pose challenges to anti-RTK therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGyFoVNhOMwLVg90H21EOLACvtfcHk0ljSLID5PdTF-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitF2nt7jK&md5=2bfb39c593fb1df3337082e8d93a7005</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1186%2Fs12943-018-0797-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12943-018-0797-x%26sid%3Dliteratum%253Aachs%26aulast%3DButti%26aufirst%3DR.%26aulast%3DDas%26aufirst%3DS.%26aulast%3DGunasekaran%26aufirst%3DV.%2BP.%26aulast%3DYadav%26aufirst%3DA.%2BS.%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DKundu%26aufirst%3DG.%2BC.%26atitle%3DReceptor%2520tyrosine%2520kinases%2520%2528RTKs%2529%2520in%2520breast%2520cancer%253A%2520signaling%252C%2520therapeutic%2520implications%2520and%2520challenges%26jtitle%3DMol.%2520Cancer%26date%3D2018%26volume%3D17%26spage%3D34%26epage%3D51%26doi%3D10.1186%2Fs12943-018-0797-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Paulsson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sjöblom, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Micke, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pontén, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Landberg, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heldin, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bergh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brennan, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jirström, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Östman, A.</span></span> <span> </span><span class="NLM_article-title">Prognostic significance of stromal platelet-derived growth factor beta-receptor expression in human breast cancer</span>. <i>Am. J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>175</i></span>,  <span class="NLM_fpage">334</span>– <span class="NLM_lpage">341</span>, <span class="refDoi"> DOI: 10.2353/ajpath.2009.081030</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2353%2Fajpath.2009.081030" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19498003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1MvksVekuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=175&publication_year=2009&pages=334-341&author=J.+Paulssonauthor=T.+Sj%C3%B6blomauthor=P.+Mickeauthor=F.+Pont%C3%A9nauthor=G.+Landbergauthor=C.-H.+Heldinauthor=J.+Berghauthor=D.+J.+Brennanauthor=K.+Jirstr%C3%B6mauthor=A.+%C3%96stman&title=Prognostic+significance+of+stromal+platelet-derived+growth+factor+beta-receptor+expression+in+human+breast+cancer&doi=10.2353%2Fajpath.2009.081030"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic significance of stromal platelet-derived growth factor beta-receptor expression in human breast cancer</span></div><div class="casAuthors">Paulsson Janna; Sjoblom Tobias; Micke Patrick; Ponten Fredrik; Landberg Goran; Heldin Carl-Henrik; Bergh Jonas; Brennan Donal J; Jirstrom Karin; Ostman Arne</div><div class="citationInfo"><span class="NLM_cas:title">The American journal of pathology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">175</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">334-41</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">This study systematically analyzes platelet-derived growth factor (PDGF) receptor expression in six types of common tumors as well as examines associations between PDGF beta-receptor status and clinicopathological characteristics in breast cancer.  PDGF receptor expression was determined by immunohistochemistry on tumor tissue microarrays.  Breast tumor data were combined with prognostic factors and related to outcome endpoints.  PDGF alpha- and beta-receptors were independently expressed, at variable frequencies, in the tumor stroma of all tested tumor types.  There was a significant association between PDGF beta-receptor expression on fibroblasts and perivascular cells in individual colon and prostate tumors.  In breast cancer, high stromal PDGF beta-receptor expression was significantly associated with high histopathological grade, estrogen receptor negativity, and high HER2 expression.  High stromal PDGF beta-receptor expression was correlated with significantly shorter recurrence-free and breast cancer-specific survival.  The prognostic significance of stromal PDGF beta-receptor expression was particularly prominent in tumors from premenopausal women.  Stromal PDGF alpha- and beta-receptor expression is a common, but variable and independent, property of solid tumors.  In breast cancer, stromal PDGF beta-receptor expression significantly correlates with less favorable clinicopathological parameters and shorter survival.  These findings highlight the prognostic significance of stromal markers and should be considered in ongoing clinical development of PDGF receptor inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRH5GHwn_GPqRwLZxei1HN8fW6udTcc2ebwdZICduJNXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvksVekuw%253D%253D&md5=13b0ee365c15209a154b4d11d54ffe76</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.2353%2Fajpath.2009.081030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2353%252Fajpath.2009.081030%26sid%3Dliteratum%253Aachs%26aulast%3DPaulsson%26aufirst%3DJ.%26aulast%3DSj%25C3%25B6blom%26aufirst%3DT.%26aulast%3DMicke%26aufirst%3DP.%26aulast%3DPont%25C3%25A9n%26aufirst%3DF.%26aulast%3DLandberg%26aufirst%3DG.%26aulast%3DHeldin%26aufirst%3DC.-H.%26aulast%3DBergh%26aufirst%3DJ.%26aulast%3DBrennan%26aufirst%3DD.%2BJ.%26aulast%3DJirstr%25C3%25B6m%26aufirst%3DK.%26aulast%3D%25C3%2596stman%26aufirst%3DA.%26atitle%3DPrognostic%2520significance%2520of%2520stromal%2520platelet-derived%2520growth%2520factor%2520beta-receptor%2520expression%2520in%2520human%2520breast%2520cancer%26jtitle%3DAm.%2520J.%2520Pathol.%26date%3D2009%26volume%3D175%26spage%3D334%26epage%3D341%26doi%3D10.2353%2Fajpath.2009.081030" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Carvalho, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanezi, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martins, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmitt, F.</span></span> <span> </span><span class="NLM_article-title">Overexpression of platelet-derived growth factor receptor alpha in breast cancer is associated with tumour progression</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">R788</span>– <span class="NLM_lpage">R795</span>, <span class="refDoi"> DOI: 10.1186/bcr1304</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr1304" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=16168125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2MXpvFOgtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=R788-R795&author=I.+Carvalhoauthor=F.+Milaneziauthor=A.+Martinsauthor=R.+M.+Reisauthor=F.+Schmitt&title=Overexpression+of+platelet-derived+growth+factor+receptor+alpha+in+breast+cancer+is+associated+with+tumour+progression&doi=10.1186%2Fbcr1304"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Overexpression of platelet-derived growth factor receptor α in breast cancer is associated with tumour progression</span></div><div class="casAuthors">Carvalho, Ines; Milanezi, Fernanda; Martins, Albino; Reis, Rui M.; Schmitt, Fernando</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">R788-R795</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Introduction: Receptor tyrosine kinases have been extensively studied owing to their frequently abnormal activation in the development and progression of human cancers.  Platelet-derived growth factor receptors (PDGFRs) are receptors with intrinsic tyrosine kinase activity that regulate several functions in normal cells and are widely expressed in a variety of malignancies.  After the demonstration that gastrointestinal stromal tumors without c-Kit mutations harbor PDGFR-α-activating mutations and that PDGFR-α is also a therapeutic target for imatinib mesylate, the interest for this receptor has increased considerably.  Because breast cancer is one of the most frequent neoplasias in women worldwide, and only one study has reported PDGFR-α expression in breast carcinomas, the aim of this work was to investigate the potential significance of PDGFR-α expression in invasive mammary carcinomas.  Methods: We used immunohistochem. to detect PDGFR-α overexpression on a series of 181 formalin-fixed paraffin-embedded invasive ductal breast carcinomas and in two breast cancer cell lines: MCF-7 and HS578T.  We assocd. its expression with known prognostic factors and we also performed polymerase chain reaction-single-stranded conformational polymorphism and direct sequencing to screen for PDGFR-α mutations.  Results: PDGFR-α expression was obsd. in 39.2% of the breast carcinomas and showed an assocn. with lymph node metastasis (P = 0.0079), HER-2 expression (P = 0.0265) and Bcl2 expression (P = 0.0121).  A correlation was also found with the expression of platelet-derived growth factor A (PDGF-A; P = 0.0194).  The two cell lines tested did not express PDGFR-α.  Screening for mutations revealed alterations in the PDGFR-α gene at the following locations: 2500A→G, 2529T→A and 2472C→T in exon 18 and 1701G→A in exon 12.  We also found an intronic insertion IVS17-50insA at exon 18 in all sequenced cases.  None of these genetic alterations was correlated with PDGFR-α expression.  The cell lines did not reveal any alterations in the PDGFR-α gene sequence.  Conclusion: PDGFR-α is expressed in invasive breast carcinomas and is assocd. with biol. aggressiveness.  The genetic alterations described were not correlated with protein expression, but other mechanisms such as gene amplification or constitutive activation of a signaling pathway inducing this receptor could still sustain PDGFR-α as a potential therapeutic target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0dHnzJg0e7Vg90H21EOLACvtfcHk0ljVRY-QzUjynQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXpvFOgtbs%253D&md5=1dab08c050341c1b246a9eda112560d7</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1186%2Fbcr1304&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr1304%26sid%3Dliteratum%253Aachs%26aulast%3DCarvalho%26aufirst%3DI.%26aulast%3DMilanezi%26aufirst%3DF.%26aulast%3DMartins%26aufirst%3DA.%26aulast%3DReis%26aufirst%3DR.%2BM.%26aulast%3DSchmitt%26aufirst%3DF.%26atitle%3DOverexpression%2520of%2520platelet-derived%2520growth%2520factor%2520receptor%2520alpha%2520in%2520breast%2520cancer%2520is%2520associated%2520with%2520tumour%2520progression%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2005%26volume%3D7%26spage%3DR788%26epage%3DR795%26doi%3D10.1186%2Fbcr1304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bhardwaj, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klassen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cossette, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterns, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuck, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deeley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elliott, B.</span></span> <span> </span><span class="NLM_article-title">Localization of platelet-derived growth factor beta receptor expression in the periepithelial stroma of human breast carcinoma</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">773</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=9816230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK28Xis1Wlurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1996&pages=773&author=B.+Bhardwajauthor=J.+Klassenauthor=N.+Cossetteauthor=E.+Sternsauthor=A.+Tuckauthor=R.+Deeleyauthor=S.+Senguptaauthor=B.+Elliott&title=Localization+of+platelet-derived+growth+factor+beta+receptor+expression+in+the+periepithelial+stroma+of+human+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Localization of platelet-derived growth factor β receptor expression in the periepithelial stroma of human breast carcinoma</span></div><div class="casAuthors">Bhardwaj, Bhavna; Klassen, Jennifer; Cossette, Nadine; Sterns, Ernest; Tuck, Alan; Deeley, Roger; Sengupta, Sandip; Elliott, Bruce</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">773-82</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Platelet-derived growth factor (PDGF) BB is secreted by most human breast carcinoma cells; however, only recently have PDGF β receptors been demonstrated in malignant breast tissue.  In the present study, the tissue localization of PDGF β receptor expression was studied in human breast carcinoma and nonmalignant breast tissues stained using both immunofluorescence and immunoperoxidase techniques.  The authors examd. a total of 29 cases of breast carcinomas, which showed both in situ and invasive components.  PDGF β receptor staining was localized in the periepithelial stroma and was particularly intense in regions immediately adjacent to carcinoma in situ components in all tumors examd.  A diffuse low level of PDGF β receptor staining was seen throughout the stroma of eight nonmalignant breast tissues as well as of nonmalignant regions of tumor tissues.  Image anal. was used to assess the coincidence of staining of PDGF β receptor with epithelial or stromal cells in 13 of the 29 tumor tissues studied.  Less than 5% of malignant ductal epithelium or myoepithelium showed PDGF β receptor staining.  Anal. with stromal cell type-specific markers indicated significant localization of PDGF β receptor primarily within α smooth muscle actin-staining cells (32%) and vascular endothelial cells (41%) in the periepithelial stroma.  PDGF β receptor positivity was strongly assocd. with periepithelial stromal cells adjacent to the basement membrane surrounding regions of carcinoma in situ but was less intense in regions of invasive carcinoma where basement membrane was degraded.  The absence of PDGF β receptors on carcinoma cells and their presence in the surrounding stroma suggest a paracrine stimulation of adjacent stromal tissue by malignant epithelial cells in human breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQVfzjQEWBw7Vg90H21EOLACvtfcHk0liEuWxaYLMuDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis1Wlurc%253D&md5=806364664031dd15ee36f31b84747abd</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBhardwaj%26aufirst%3DB.%26aulast%3DKlassen%26aufirst%3DJ.%26aulast%3DCossette%26aufirst%3DN.%26aulast%3DSterns%26aufirst%3DE.%26aulast%3DTuck%26aufirst%3DA.%26aulast%3DDeeley%26aufirst%3DR.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DElliott%26aufirst%3DB.%26atitle%3DLocalization%2520of%2520platelet-derived%2520growth%2520factor%2520beta%2520receptor%2520expression%2520in%2520the%2520periepithelial%2520stroma%2520of%2520human%2520breast%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1996%26volume%3D2%26spage%3D773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Angelis, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nardone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veeraraghavan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethunath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heiser, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, C. K. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, E. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Renwick, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nanda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shea, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rajendran, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waters, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zabransky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutierrez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geyer, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chamness, G. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilsenbeck, S. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rimawi, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weigelt, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis-Filho, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osborne, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiff, R.</span></span> <span> </span><span class="NLM_article-title">HER2 reactivation through acquisition of the HER2 L755S mutation as a mechanism of acquired resistance to HER2-targeted therapy in HER2(+) breast cancer</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">5123</span>– <span class="NLM_lpage">5134</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-16-2191</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-16-2191" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28487443" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVCit7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=5123-5134&author=X.+Xuauthor=C.+De+Angelisauthor=K.+A.+Burkeauthor=A.+Nardoneauthor=H.+Huauthor=L.+Qinauthor=J.+Veeraraghavanauthor=V.+Sethunathauthor=L.+M.+Heiserauthor=N.+Wangauthor=C.+K.+Y.+Ngauthor=E.+S.+Chenauthor=A.+Renwickauthor=T.+Wangauthor=S.+Nandaauthor=M.+Sheaauthor=T.+Mitchellauthor=M.+Rajendranauthor=I.+Watersauthor=D.+J.+Zabranskyauthor=K.+L.+Scottauthor=C.+Gutierrezauthor=C.+Nagiauthor=F.+C.+Geyerauthor=G.+C.+Chamnessauthor=B.+H.+Parkauthor=C.+A.+Shawauthor=S.+G.+Hilsenbeckauthor=M.+F.+Rimawiauthor=J.+W.+Grayauthor=B.+Weigeltauthor=J.+S.+Reis-Filhoauthor=C.+K.+Osborneauthor=R.+Schiff&title=HER2+reactivation+through+acquisition+of+the+HER2+L755S+mutation+as+a+mechanism+of+acquired+resistance+to+HER2-targeted+therapy+in+HER2%28%2B%29+breast+cancer&doi=10.1158%2F1078-0432.ccr-16-2191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 Reactivation through Acquisition of the HER2 L755S Mutation as a Mechanism of Acquired Resistance to HER2-targeted Therapy in HER2+ Breast Cancer</span></div><div class="casAuthors">Xu, Xiaowei; De Angelis, Carmine; Burke, Kathleen A.; Nardone, Agostina; Hu, Huizhong; Qin, Lanfang; Veeraraghavan, Jamunarani; Sethunath, Vidyalakshmi; Heiser, Laura M.; Wang, Nicholas; Ng, Charlotte K. Y.; Chen, Edward S.; Renwick, Alexander; Wang, Tao; Nanda, Sarmistha; Shea, Martin; Mitchell, Tamika; Rajendran, Mahitha; Waters, Ian; Zabransky, Daniel J.; Scott, Kenneth L.; Gutierrez, Carolina; Nagi, Chandandeep; Geyer, Felipe C.; Chamness, Gary C.; Park, Ben H.; Shaw, Chad A.; Hilsenbeck, Susan G.; Rimawi, Mothaffar F.; Gray, Joe W.; Weigelt, Britta; Reis-Filho, Jorge S.; Osborne, C. Kent; Schiff, Rachel</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">5123-5134</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Resistance to anti-HER2 therapies in HER2+ breast cancer can occur through activation of alternative survival pathways or reactivation of the HER signaling network.  Here we employed BT474 parental and treatment-resistant cell line models to investigate a mechanism by which HER2+ breast cancer can reactivate the HER network under potent HER2-targeted therapies.  Resistant derivs. to lapatinib (L), trastuzumab (T), or the combination (LR/TR/LTR) were developed independently from two independent estrogen receptor ER+/HER2+ BT474 cell lines (AZ/ATCC).  Two derivs. resistant to the lapatinib-contg. regimens (BT474/AZ-LR and BT474/ATCC-LTR lines) that showed HER2 reactivation at the time of resistance were subjected to massive parallel sequencing and compared with parental lines.  Ectopic expression and mutant-specific siRNA interference were applied to analyze the mutation functionally.  In vitro and in vivo expts. were performed to test alternative therapies for mutant HER2 inhibition.  Genomic analyses revealed that the HER2L755S mutation was the only common somatic mutation gained in the BT474/AZ-LR and BT474/ATCC-LTR lines.  Ectopic expression of HER2L755S induced acquired lapatinib resistance in the BT474/AZ, SK-BR-3, and AU565 parental cell lines.  HER2L755S-specific siRNA knockdown reversed the resistance in BT474/AZ-LR and BT474/ATCC-LTR lines.  The HER1/2-irreversible inhibitors afatinib and neratinib substantially inhibited both resistant cell growth and the HER2 and downstream AKT/MAPK signaling driven by HER2L755S in vitro and in vivo.  HER2 reactivation through acquisition of the HER2L755S mutation was identified as a mechanism of acquired resistance to lapatinib-contg. HER2-targeted therapy in preclin. HER2-amplified breast cancer models, which can be overcome by irreversible HER1/2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqr23ozygbtLVg90H21EOLACvtfcHk0lhe8XzGNEtwzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVCit7jP&md5=2c984948a7b4b223241e320025526fe0</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-16-2191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-16-2191%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DDe%2BAngelis%26aufirst%3DC.%26aulast%3DBurke%26aufirst%3DK.%2BA.%26aulast%3DNardone%26aufirst%3DA.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DQin%26aufirst%3DL.%26aulast%3DVeeraraghavan%26aufirst%3DJ.%26aulast%3DSethunath%26aufirst%3DV.%26aulast%3DHeiser%26aufirst%3DL.%2BM.%26aulast%3DWang%26aufirst%3DN.%26aulast%3DNg%26aufirst%3DC.%2BK.%2BY.%26aulast%3DChen%26aufirst%3DE.%2BS.%26aulast%3DRenwick%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DNanda%26aufirst%3DS.%26aulast%3DShea%26aufirst%3DM.%26aulast%3DMitchell%26aufirst%3DT.%26aulast%3DRajendran%26aufirst%3DM.%26aulast%3DWaters%26aufirst%3DI.%26aulast%3DZabransky%26aufirst%3DD.%2BJ.%26aulast%3DScott%26aufirst%3DK.%2BL.%26aulast%3DGutierrez%26aufirst%3DC.%26aulast%3DNagi%26aufirst%3DC.%26aulast%3DGeyer%26aufirst%3DF.%2BC.%26aulast%3DChamness%26aufirst%3DG.%2BC.%26aulast%3DPark%26aufirst%3DB.%2BH.%26aulast%3DShaw%26aufirst%3DC.%2BA.%26aulast%3DHilsenbeck%26aufirst%3DS.%2BG.%26aulast%3DRimawi%26aufirst%3DM.%2BF.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26aulast%3DWeigelt%26aufirst%3DB.%26aulast%3DReis-Filho%26aufirst%3DJ.%2BS.%26aulast%3DOsborne%26aufirst%3DC.%2BK.%26aulast%3DSchiff%26aufirst%3DR.%26atitle%3DHER2%2520reactivation%2520through%2520acquisition%2520of%2520the%2520HER2%2520L755S%2520mutation%2520as%2520a%2520mechanism%2520of%2520acquired%2520resistance%2520to%2520HER2-targeted%2520therapy%2520in%2520HER2%2528%252B%2529%2520breast%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D5123%26epage%3D5134%26doi%3D10.1158%2F1078-0432.ccr-16-2191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bender, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ghazalpour, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swensen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Millis, S. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gatalica, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, M. F.</span></span> <span> </span><span class="NLM_article-title">mutations in the kinase domain of the her2/erbb2 gene identified in a wide variety of human cancers</span>. <i>J. Mol. Diagn.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">487</span>– <span class="NLM_lpage">495</span>, <span class="refDoi"> DOI: 10.1016/j.jmoldx.2015.04.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jmoldx.2015.04.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26320869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOntLjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2015&pages=487-495&author=W.+Wenauthor=W.+Chenauthor=N.+Xiaoauthor=R.+Benderauthor=A.+Ghazalpourauthor=Z.+Tanauthor=J.+Swensenauthor=S.+Z.+Millisauthor=G.+Basuauthor=Z.+Gatalicaauthor=M.+F.+Press&title=mutations+in+the+kinase+domain+of+the+her2%2Ferbb2+gene+identified+in+a+wide+variety+of+human+cancers&doi=10.1016%2Fj.jmoldx.2015.04.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Mutations in the Kinase Domain of the HER2/ERBB2 Gene Identified in a Wide Variety of Human Cancers</span></div><div class="casAuthors">Wen, Wenhsiang; Chen, Wangjuh; Xiao, Nick; Bender, Ryan; Ghazalpour, Anatole; Tan, Zheng; Swensen, Jeffrey; Millis, Sherri Z.; Basu, Gargi; Gatalica, Zoran; Press, Michael F.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Diagnostics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">487-495</span>CODEN:
                <span class="NLM_cas:coden">JMDIFP</span>;
        ISSN:<span class="NLM_cas:issn">1525-1578</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">The HER2 (official name ERBB2) gene encodes a membrane receptor in the epidermal growth factor receptor family amplified and overexpressed in adenocarcinoma.  Activating mutations also occur in several cancers.  We report mutation analyses of the HER2 kinase domain in 7497 histol. diverse cancers.  Forty-five genes, including the kinase domain of HER2 with HER2 IHC and dual in situ hybridization, were analyzed in tumors from 7497 patients with cancer, including 850 breast, 770 colorectal, 910 non-small cell lung, 823 uterine or cervical, 1372 ovarian, and 297 pancreatic cancers, as well as 323 melanomas and 2152 other solid tumors.  Sixty-nine HER2 kinase domain mutations were identified in tumors from 68 patients (approx. 1% of all cases, ranging from absent in sarcomas to 4% in urothelial cancers), which included previously published activating mutations and 13 novel mutations.  Fourteen cases with coexisting HER2 mutation and amplification and/or overexpression were identified.  Fifty-two of 68 patients had addnl. mutations in other analyzed genes, whereas 16 patients (23%) had HER2 mutations identified as the sole driver mutation.  HER2 mutations coexisted with HER2 gene amplification and overexpression and with mutations in other functionally important genes.  HER2 mutations were identified as the only driver mutation in a significant proportion of solid cancers.  Evaluation of anti-HER2 therapies in nonamplified, HER2-mutated cancers is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo3gPcn7n7fYbVg90H21EOLACvtfcHk0lgVGfxKN2_4Kg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOntLjP&md5=045c22bad0d1d3513ce930c44a62cc30</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.jmoldx.2015.04.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmoldx.2015.04.003%26sid%3Dliteratum%253Aachs%26aulast%3DWen%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DXiao%26aufirst%3DN.%26aulast%3DBender%26aufirst%3DR.%26aulast%3DGhazalpour%26aufirst%3DA.%26aulast%3DTan%26aufirst%3DZ.%26aulast%3DSwensen%26aufirst%3DJ.%26aulast%3DMillis%26aufirst%3DS.%2BZ.%26aulast%3DBasu%26aufirst%3DG.%26aulast%3DGatalica%26aufirst%3DZ.%26aulast%3DPress%26aufirst%3DM.%2BF.%26atitle%3Dmutations%2520in%2520the%2520kinase%2520domain%2520of%2520the%2520her2%252Ferbb2%2520gene%2520identified%2520in%2520a%2520wide%2520variety%2520of%2520human%2520cancers%26jtitle%3DJ.%2520Mol.%2520Diagn.%26date%3D2015%26volume%3D17%26spage%3D487%26epage%3D495%26doi%3D10.1016%2Fj.jmoldx.2015.04.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">AACR
Project GENIE Consortium</span> <span> </span><span class="NLM_article-title">AACR project genie: powering precision medicine through an international
consortium</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">818</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.CD-17-0151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F2159-8290.CD-17-0151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28572459" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2017&pages=818-831&author=AACR%0AProject+GENIE+Consortium&title=AACR+project+genie%3A+powering+precision+medicine+through+an+international%0Aconsortium&doi=10.1158%2F2159-8290.CD-17-0151"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-17-0151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-17-0151%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DAACR%2520project%2520genie%253A%2520powering%2520precision%2520medicine%2520through%2520an%2520international%250Aconsortium%26jtitle%3DCancer%2520Discovery%26date%3D2017%26volume%3D7%26spage%3D818%26epage%3D831%26doi%3D10.1158%2F2159-8290.CD-17-0151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span> </span><span class="NLM_article-title">My Cancer Genome, Genetically Informed Cancer Medicine</span>. <a href="https://www.mycancergenome.org/content/alteration/erbb2-l755s/" class="extLink">https://www.mycancergenome.org/content/alteration/erbb2-l755s/</a> (accessed Feb 14, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=My+Cancer+Genome%2C+Genetically+Informed+Cancer+Medicine.+https%3A%2F%2Fwww.mycancergenome.org%2Fcontent%2Falteration%2Ferbb2-l755s%2F+%28accessed+Feb+14%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26atitle%3DMy%2520Cancer%2520Genome%252C%2520Genetically%2520Informed%2520Cancer%2520Medicine" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zabransky, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yankaskas, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cochran, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, H. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croessmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavuri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Red Brewer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalton, W. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cimino-Mathews, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cravero, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Button, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyker-Snowman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cidado, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Erlanger, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manto, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lauring, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arteaga, C. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konstantopoulos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, B. H.</span></span> <span> </span><span class="NLM_article-title">HER2 missense mutations have distinct effects on oncogenic signaling and migration</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">E6205</span>– <span class="NLM_lpage">E6214</span>, <span class="refDoi"> DOI: 10.1073/pnas.1516853112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1073%2Fpnas.1516853112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26508629" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsleksbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=E6205-E6214&author=D.+J.+Zabranskyauthor=C.+L.+Yankaskasauthor=R.+L.+Cochranauthor=H.+Y.+Wongauthor=S.+Croessmannauthor=D.+Chuauthor=S.+M.+Kavuriauthor=M.+Red+Brewerauthor=D.+M.+Rosenauthor=W.+B.+Daltonauthor=A.+Cimino-Mathewsauthor=K.+Craveroauthor=B.+Buttonauthor=K.+Kyker-Snowmanauthor=J.+Cidadoauthor=B.+Erlangerauthor=H.+A.+Parsonsauthor=K.+M.+Mantoauthor=R.+Boseauthor=J.+Lauringauthor=C.+L.+Arteagaauthor=K.+Konstantopoulosauthor=B.+H.+Park&title=HER2+missense+mutations+have+distinct+effects+on+oncogenic+signaling+and+migration&doi=10.1073%2Fpnas.1516853112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">HER2 missense mutations have distinct effects on oncogenic signaling and migration</span></div><div class="casAuthors">Zabransky, Daniel J.; Yankaskas, Christopher L.; Cochran, Rory L.; Wong, Hong Yuen; Croessmann, Sarah; Chu, David; Kavuri, Shyam M.; Brewer, Monica Red; Rosen, D. Marc; Dalton, W. Brian; Cimino-Mathews, Ashley; Cravero, Karen; Button, Berry; Kyker-Snowman, Kelly; Cidado, Justin; Erlanger, Bracha; Parsons, Heather A.; Manto, Kristen M.; Bose, Ron; Lauring, Josh; Arteaga, Carlos L.; Konstantopoulos, Konstantinos; Park, Ben Ho</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E6205-E6214</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Recurrent human epidermal growth factor receptor 2 (HER2) missense mutations have been reported in human cancers.  These mutations occur primarily in the absence of HER2 gene amplification such that most HER2-mutant tumors are classified as "neg." by FISH or immunohistochem. assays.  It remains unclear whether nonamplified HER2 missense mutations are oncogenic and whether they are targets for HER2-directed therapies that are currently approved for the treatment of HER2 gene-amplified breast cancers.  Here we functionally characterize HER2 kinase and extracellular domain mutations through gene editing of the endogenous loci in HER2 nonamplified human breast epithelial cells.  In in vitro and in vivo assays, the majority of HER2 missense mutations do not impart detectable oncogenic changes.  However, the HER2 V777L mutation increased biochem. pathway activation and, in the context of a PIK3CA mutation, enhanced migratory features in vitro.  However, the V777L mutation did not alter in vivo tumorigenicity or sensitivity to HER2-directed therapies in proliferation assays.  Our results suggest the oncogenicity and potential targeting of HER2 missense mutations should be considered in the context of cooperating genetic alterations and provide previously unidentified insights into functional anal. of HER2 mutations and strategies to target them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQmDBjvnG-6LVg90H21EOLACvtfcHk0lim9AoFHcPhqw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsleksbnL&md5=b78457c6678b940e28afed1b50554f44</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1516853112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1516853112%26sid%3Dliteratum%253Aachs%26aulast%3DZabransky%26aufirst%3DD.%2BJ.%26aulast%3DYankaskas%26aufirst%3DC.%2BL.%26aulast%3DCochran%26aufirst%3DR.%2BL.%26aulast%3DWong%26aufirst%3DH.%2BY.%26aulast%3DCroessmann%26aufirst%3DS.%26aulast%3DChu%26aufirst%3DD.%26aulast%3DKavuri%26aufirst%3DS.%2BM.%26aulast%3DRed%2BBrewer%26aufirst%3DM.%26aulast%3DRosen%26aufirst%3DD.%2BM.%26aulast%3DDalton%26aufirst%3DW.%2BB.%26aulast%3DCimino-Mathews%26aufirst%3DA.%26aulast%3DCravero%26aufirst%3DK.%26aulast%3DButton%26aufirst%3DB.%26aulast%3DKyker-Snowman%26aufirst%3DK.%26aulast%3DCidado%26aufirst%3DJ.%26aulast%3DErlanger%26aufirst%3DB.%26aulast%3DParsons%26aufirst%3DH.%2BA.%26aulast%3DManto%26aufirst%3DK.%2BM.%26aulast%3DBose%26aufirst%3DR.%26aulast%3DLauring%26aufirst%3DJ.%26aulast%3DArteaga%26aufirst%3DC.%2BL.%26aulast%3DKonstantopoulos%26aufirst%3DK.%26aulast%3DPark%26aufirst%3DB.%2BH.%26atitle%3DHER2%2520missense%2520mutations%2520have%2520distinct%2520effects%2520on%2520oncogenic%2520signaling%2520and%2520migration%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2015%26volume%3D112%26spage%3DE6205%26epage%3DE6214%26doi%3D10.1073%2Fpnas.1516853112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bose, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kavuri, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Searleman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koboldt, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monsey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronson, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, C. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mardis, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, M. J.</span></span> <span> </span><span class="NLM_article-title">Activating HER2 mutations in HER2 gene amplification negative breast cancer</span>. <i>Cancer Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">224</span>– <span class="NLM_lpage">237</span>, <span class="refDoi"> DOI: 10.1158/2159-8290.cd-12-0349</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F2159-8290.cd-12-0349" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23220880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXis1Wktrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2013&pages=224-237&author=R.+Boseauthor=S.+M.+Kavuriauthor=A.+C.+Searlemanauthor=W.+Shenauthor=D.+Shenauthor=D.+C.+Koboldtauthor=J.+Monseyauthor=N.+Goelauthor=A.+B.+Aronsonauthor=S.+Liauthor=C.+X.+Maauthor=L.+Dingauthor=E.+R.+Mardisauthor=M.+J.+Ellis&title=Activating+HER2+mutations+in+HER2+gene+amplification+negative+breast+cancer&doi=10.1158%2F2159-8290.cd-12-0349"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Activating HER2 Mutations in HER2 Gene Amplification Negative Breast Cancer</span></div><div class="casAuthors">Bose, Ron; Kavuri, Shyam M.; Searleman, Adam C.; Shen, Wei; Shen, Dong; Koboldt, Daniel C.; Monsey, John; Goel, Nicholas; Aronson, Adam B.; Li, Shunqiang; Ma, Cynthia X.; Ding, Li; Mardis, Elaine R.; Ellis, Matthew J.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">224-237</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Data from 8 breast cancer genome-sequencing projects identified 25 patients with HER2 somatic mutations in cancers lacking HER2 gene amplification.  To det. the phenotype of these mutations, we functionally characterized 13 HER2 mutations using in vitro kinase assays, protein structure anal., cell culture, and xenograft expts.  Seven of these mutations are activating mutations, including G309A, D769H, D769Y, V777L, P780ins, V842I, and R896C. HER2 in-frame deletion 755-759, which is homologous to EGF receptor (EGFR) exon 19 in-frame deletions, had a neomorphic phenotype with increased phosphorylation of EGFR or HER3.  L755S produced lapatinib resistance, but was not an activating mutation in our exptl. systems.  All of these mutations were sensitive to the irreversible kinase inhibitor, neratinib.  These findings show that HER2 somatic mutation is an alternative mechanism to activate HER2 in breast cancer and they validate HER2 somatic mutations as drug targets for breast cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqpInHiFiSDrrVg90H21EOLACvtfcHk0lgMO1n9zacK-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXis1Wktrg%253D&md5=4b271e961ad4a68862e4a3ba80e88972</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.cd-12-0349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.cd-12-0349%26sid%3Dliteratum%253Aachs%26aulast%3DBose%26aufirst%3DR.%26aulast%3DKavuri%26aufirst%3DS.%2BM.%26aulast%3DSearleman%26aufirst%3DA.%2BC.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DD.%26aulast%3DKoboldt%26aufirst%3DD.%2BC.%26aulast%3DMonsey%26aufirst%3DJ.%26aulast%3DGoel%26aufirst%3DN.%26aulast%3DAronson%26aufirst%3DA.%2BB.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DC.%2BX.%26aulast%3DDing%26aufirst%3DL.%26aulast%3DMardis%26aufirst%3DE.%2BR.%26aulast%3DEllis%26aufirst%3DM.%2BJ.%26atitle%3DActivating%2520HER2%2520mutations%2520in%2520HER2%2520gene%2520amplification%2520negative%2520breast%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2013%26volume%3D3%26spage%3D224%26epage%3D237%26doi%3D10.1158%2F2159-8290.cd-12-0349" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kancha, R. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">von Bubnoff, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartosch, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peschel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engh, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duyster, J.</span></span> <span> </span><span class="NLM_article-title">Differential sensitivity of ERBB2 kinase domain mutations towards lapatinib</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">e26760</span>– <span class="NLM_lpage">e26769</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0026760</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0026760" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22046346" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu7fF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e26760-e26769&author=R.+K.+Kanchaauthor=N.+von+Bubnoffauthor=N.+Bartoschauthor=C.+Peschelauthor=R.+A.+Enghauthor=J.+Duyster&title=Differential+sensitivity+of+ERBB2+kinase+domain+mutations+towards+lapatinib&doi=10.1371%2Fjournal.pone.0026760"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Differential sensitivity of ERBB2 kinase domain mutations towards lapatinib</span></div><div class="casAuthors">Kancha, Rama Krishna; von Bubnoff, Nikolas; Bartosch, Natalie; Peschel, Christian; Engh, Richard A.; Duyster, Justus</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">e26760</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Background: Overexpression of the ERBB2 kinase is obsd. in about one-third of breast cancer patients and the dual ERBB1/ERBB2 kinase inhibitor lapatinib was recently approved for the treatment of advanced ERBB2-pos. breast cancer.  Mutations in the ERBB2 receptor have recently been reported in breast cancer at diagnosis and also in gastric, colorectal and lung cancer.  These mutations may have an impact on the clin. responses achieved with lapatinib in breast cancer and may also have a potential impact on the use of lapatinib in other solid cancers.  However, the sensitivity of lapatinib towards clin. obsd. ERBB2 mutations is not known.  Methodol./Principal Findings: We cloned a panel of 8 clin. obsd. ERBB2 mutations, established stable cell lines and characterized their sensitivity towards lapatinib and alternative ERBB2 inhibitors.  Both lapatinib-sensitive and lapatinib-resistant ERBB2 mutations were obsd.  Interestingly, we were able to generate lapatinib resistance mutations in wt-ERBB2 cells incubated with lapatinib for prolonged periods of time.  This indicates that these resistance mutations may also cause secondary resistance in lapatinib-treated patients.  Lapatinib-resistant ERBB2 mutations were found to be highly resistant towards AEE788 treatment but remained sensitive towards the dual irreversible inhibitors CL-387785 and WZ-4002.  Conclusions/Significance: Patients harbouring certain ERBB2 kinase domain mutations at diagnosis may not benefit from lapatinib treatment.  Moreover, secondary lapatinib resistance may develop due to kinase domain mutations.  Irreversible ERBB2 inhibitors may offer alternative treatment options for breast cancer and other solid tumor patients harbouring lapatinib resistance mutations.  In addn., these inhibitors may be of interest in the scenario of secondary lapatinib resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGXRqTTP0WubVg90H21EOLACvtfcHk0lhbmYCgxH5adQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVyqu7fF&md5=84ef463604d86644c79f52af52d093e8</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0026760&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0026760%26sid%3Dliteratum%253Aachs%26aulast%3DKancha%26aufirst%3DR.%2BK.%26aulast%3Dvon%2BBubnoff%26aufirst%3DN.%26aulast%3DBartosch%26aufirst%3DN.%26aulast%3DPeschel%26aufirst%3DC.%26aulast%3DEngh%26aufirst%3DR.%2BA.%26aulast%3DDuyster%26aufirst%3DJ.%26atitle%3DDifferential%2520sensitivity%2520of%2520ERBB2%2520kinase%2520domain%2520mutations%2520towards%2520lapatinib%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De26760%26epage%3De26769%26doi%3D10.1371%2Fjournal.pone.0026760" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trowe, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boukouvala, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calkins, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cutler, R. E.,  Jr.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Funke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gendreau, S. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woolfrey, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vysotskaia, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerritsen, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthews, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lamb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heuer, T. S.</span></span> <span> </span><span class="NLM_article-title">EXEL-7647 inhibits mutant forms of ErbB2 associated with lapatinib resistance and neoplastic transformation</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2465</span>– <span class="NLM_lpage">2475</span>, <span class="refDoi"> DOI: 10.1158/1078-0432.ccr-07-4367</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1158%2F1078-0432.ccr-07-4367" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18413839" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2008&pages=2465-2475&author=T.+Troweauthor=S.+Boukouvalaauthor=K.+Calkinsauthor=R.+E.+Cutlerauthor=R.+Fongauthor=R.+Funkeauthor=S.+B.+Gendreauauthor=Y.+D.+Kimauthor=N.+Millerauthor=J.+R.+Woolfreyauthor=V.+Vysotskaiaauthor=J.+P.+Yangauthor=M.+E.+Gerritsenauthor=D.+J.+Matthewsauthor=P.+Lambauthor=T.+S.+Heuer&title=EXEL-7647+inhibits+mutant+forms+of+ErbB2+associated+with+lapatinib+resistance+and+neoplastic+transformation&doi=10.1158%2F1078-0432.ccr-07-4367"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">EXEL-7647 Inhibits Mutant Forms of ErbB2 Associated with Lapatinib Resistance and Neoplastic Transformation</span></div><div class="casAuthors">Trowe, Torsten; Boukouvala, Sotiria; Calkins, Keith; Cutler, Richard E., Jr.; Fong, Ryan; Funke, Roel; Gendreau, Steven B.; Kim, Yong D.; Miller, Nicole; Woolfrey, John R.; Vysotskaia, Valentina; Yang, Jing Ping; Gerritsen, Mary E.; Matthews, David J.; Lamb, Peter; Heuer, Timothy S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2465-2475</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations assocd. with resistance to kinase inhibition are an important mechanism of intrinsic or acquired loss of clin. efficacy for kinase-targeted therapeutics.  We report the prospective discovery of ErbB2 mutations that confer resistance to the small-mol. inhibitor lapatinib.  We did in vitro screening using a randomly mutagenized ErbB2 expression library in Ba/F3 cells, which were dependent on ErbB2 activity for survival and growth.  Lapatinib resistance screens identified mutations at 16 different ErbB2 amino acid residues, with 12 mutated amino acids mapping to the kinase domain.  Mutations conferring the greatest lapatinib resistance cluster in the NH2-terminal kinase lobe and hinge region.  Structural computer modeling studies suggest that lapatinib resistance is caused by multiple mechanisms; including direct steric interference and restriction of conformational flexibility (the inactive state required for lapatinib binding is energetically unfavorable).  ErbB2 T798I imparts the strongest lapatinib resistance effect and is analogous to the epidermal growth factor receptor T790M, ABL T315I, and cKIT T670I gatekeeper mutations that are assocd. with clin. drug resistance.  ErbB2 mutants assocd. with lapatinib resistance transformed NIH-3T3 cells, including L755S and T733I mutations known to occur in human breast and gastric carcinomas, supporting a direct mechanism for lapatinib resistance in ErbB2-driven human cancers.  The epidermal growth factor receptor/ErbB2/vascular endothelial growth factor receptor inhibitor EXEL-7647 was found to inhibit almost all lapatinib resistance-assocd. mutations.  Furthermore, no ErbB2 mutations were found to be assocd. with EXEL-7647 resistance and lapatinib sensitivity.  Taken together, these data suggest potential target-based mechanisms of resistance to lapatinib and suggest that EXEL-7647 may be able to circumvent these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIufvHezxJo7Vg90H21EOLACvtfcHk0lgqrSFs7Hd1zw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkvFGmur8%253D&md5=c51774479df7be7c06d2cc22e8c58afc</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.ccr-07-4367&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.ccr-07-4367%26sid%3Dliteratum%253Aachs%26aulast%3DTrowe%26aufirst%3DT.%26aulast%3DBoukouvala%26aufirst%3DS.%26aulast%3DCalkins%26aufirst%3DK.%26aulast%3DCutler%26aufirst%3DR.%2BE.%26aulast%3DFong%26aufirst%3DR.%26aulast%3DFunke%26aufirst%3DR.%26aulast%3DGendreau%26aufirst%3DS.%2BB.%26aulast%3DKim%26aufirst%3DY.%2BD.%26aulast%3DMiller%26aufirst%3DN.%26aulast%3DWoolfrey%26aufirst%3DJ.%2BR.%26aulast%3DVysotskaia%26aufirst%3DV.%26aulast%3DYang%26aufirst%3DJ.%2BP.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DMatthews%26aufirst%3DD.%2BJ.%26aulast%3DLamb%26aufirst%3DP.%26aulast%3DHeuer%26aufirst%3DT.%2BS.%26atitle%3DEXEL-7647%2520inhibits%2520mutant%2520forms%2520of%2520ErbB2%2520associated%2520with%2520lapatinib%2520resistance%2520and%2520neoplastic%2520transformation%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2008%26volume%3D14%26spage%3D2465%26epage%3D2475%26doi%3D10.1158%2F1078-0432.ccr-07-4367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bai, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span> <span> </span><span class="NLM_article-title">Breast cancer cell line classification and its relevance with breast tumor subtyping</span>. <i>J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">3131</span>– <span class="NLM_lpage">3141</span>, <span class="refDoi"> DOI: 10.7150/jca.18457</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.7150%2Fjca.18457" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29158785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1M3jtlWjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=3131-3141&author=X.+Daiauthor=H.+Chengauthor=Z.+Baiauthor=J.+Li&title=Breast+cancer+cell+line+classification+and+its+relevance+with+breast+tumor+subtyping&doi=10.7150%2Fjca.18457"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Breast Cancer Cell Line Classification and Its Relevance with Breast Tumor Subtyping</span></div><div class="casAuthors">Dai Xiaofeng; Cheng Hongye; Bai Zhonghu; Li Jia; Dai Xiaofeng; Cheng Hongye; Bai Zhonghu; Li Jia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3131-3141</span>
        ISSN:<span class="NLM_cas:issn">1837-9664</span>.
    </div><div class="casAbstract">Breast cancer cell lines have been widely used for breast cancer modelling which encompasses a panel of diseases with distinct phenotypical associations.  Though cell lines provide unlimited homogenous materials for tumor studies and are relatively easy to culture, they are known to accumulate mutations duringthe initial establishment and subsequent series of cultivations.  Thus, whether breast cancer cell line heterogeneity reflects that of carcinoma remains an important issue to resolve before drawing any reliable conclusion at the tumor level using cell lines.  Inconsistent nomenclatures used for breast cancer cell line subtyping and the different number of subtypes grouped for cell lines and tumors make their direct matching elusive.  By analyzing the molecular features of 92 breast cancer cell lines as documented by different literatures, we categorize 84 cell lines into 5 groups to be consistent with breast tumor classification.  After combing through these cell lines, we summarized the molecular features, genetically and epigenetically, of each subtype, and manually documented 10 cell lines lacking explicit information on subtyping.  Nine cell lines, either found inconsistent on their primary molecular features from different studies or being contaminated at the origin, are not suggested as the first choice for experimental use.  We conclude that breast tumor cell lines, though having a high mutational frequency with many uncertainties and could not fully capture breast cancer heterogeneity, are feasible but crude models for tumors of the same subtype.  New cell lines with enriched interferon regulated genes need to be established to enlarge the coverage of cell lines on tumor heterogeneity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSgfx4Cpa1dYL5oR8RchjrrfW6udTcc2ebAZ4vFWgktKrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M3jtlWjtg%253D%253D&md5=b350670e7d74de3157883aa28e950851</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.7150%2Fjca.18457&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.7150%252Fjca.18457%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DBai%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26atitle%3DBreast%2520cancer%2520cell%2520line%2520classification%2520and%2520its%2520relevance%2520with%2520breast%2520tumor%2520subtyping%26jtitle%3DJ.%2520Cancer%26date%3D2017%26volume%3D8%26spage%3D3131%26epage%3D3141%26doi%3D10.7150%2Fjca.18457" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neve, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fridlyand, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeh, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baehner, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fevr, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clark, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bayani, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coppe, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tong, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speed, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spellman, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeVries, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lapuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuo, W.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stilwell, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pinkel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albertson, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldman, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCormick, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickson, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lippman, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, J. W.</span></span> <span> </span><span class="NLM_article-title">A collection of breast cancer cell lines for the study of functionally distinct cancer subtypes</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">515</span>– <span class="NLM_lpage">527</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2006.10.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ccr.2006.10.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17157791" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD28XhtlCnsrbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2006&pages=515-527&author=R.+M.+Neveauthor=K.+Chinauthor=J.+Fridlyandauthor=J.+Yehauthor=F.+L.+Baehnerauthor=T.+Fevrauthor=L.+Clarkauthor=N.+Bayaniauthor=J.-P.+Coppeauthor=F.+Tongauthor=T.+Speedauthor=P.+T.+Spellmanauthor=S.+DeVriesauthor=A.+Lapukauthor=N.+J.+Wangauthor=W.-L.+Kuoauthor=J.+L.+Stilwellauthor=D.+Pinkelauthor=D.+G.+Albertsonauthor=F.+M.+Waldmanauthor=F.+McCormickauthor=R.+B.+Dicksonauthor=M.+D.+Johnsonauthor=M.+Lippmanauthor=S.+Ethierauthor=A.+Gazdarauthor=J.+W.+Gray&title=A+collection+of+breast+cancer+cell+lines+for+the+study+of+functionally+distinct+cancer+subtypes&doi=10.1016%2Fj.ccr.2006.10.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A collection of breast cancer cell lines for the study of functionally distinct cancer subtypes</span></div><div class="casAuthors">Neve, Richard M.; Chin, Koei; Fridlyand, Jane; Yeh, Jennifer; Baehner, Frederick L.; Fevr, Tea; Clark, Laura; Bayani, Nora; Coppe, Jean-Philippe; Tong, Frances; Speed, Terry; Spellman, Paul T.; DeVries, Sandy; Lapuk, Anna; Wang, Nick J.; Kuo, Wen-Lin; Stilwell, Jackie L.; Pinkel, Daniel; Albertson, Donna G.; Waldman, Frederic M.; McCormick, Frank; Dickson, Robert B.; Johnson, Michael D.; Lippman, Marc; Ethier, Stephen; Gazdar, Adi; Gray, Joe W.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">515-527</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent studies suggest that thousands of genes may contribute to breast cancer pathophysiologies when deregulated by genomic or epigenomic events.  Here, we describe a model "system" to appraise the functional contributions of these genes to breast cancer subsets.  In general, the recurrent genomic and transcriptional characteristics of 51 breast cancer cell lines mirror those of 145 primary breast tumors, although some significant differences are documented.  The cell lines that comprise the system also exhibit the substantial genomic, transcriptional, and biol. heterogeneity found in primary tumors.  We show, using Trastuzumab (Herceptin) monotherapy as an example, that the system can be used to identify mol. features that predict or indicate response to targeted therapies or other physiol. perturbations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprHlT0rEcwBrVg90H21EOLACvtfcHk0lh5_tgz2rtymQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XhtlCnsrbE&md5=218682c6f7ffebff329208d476ab35ad</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2006.10.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2006.10.008%26sid%3Dliteratum%253Aachs%26aulast%3DNeve%26aufirst%3DR.%2BM.%26aulast%3DChin%26aufirst%3DK.%26aulast%3DFridlyand%26aufirst%3DJ.%26aulast%3DYeh%26aufirst%3DJ.%26aulast%3DBaehner%26aufirst%3DF.%2BL.%26aulast%3DFevr%26aufirst%3DT.%26aulast%3DClark%26aufirst%3DL.%26aulast%3DBayani%26aufirst%3DN.%26aulast%3DCoppe%26aufirst%3DJ.-P.%26aulast%3DTong%26aufirst%3DF.%26aulast%3DSpeed%26aufirst%3DT.%26aulast%3DSpellman%26aufirst%3DP.%2BT.%26aulast%3DDeVries%26aufirst%3DS.%26aulast%3DLapuk%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DN.%2BJ.%26aulast%3DKuo%26aufirst%3DW.-L.%26aulast%3DStilwell%26aufirst%3DJ.%2BL.%26aulast%3DPinkel%26aufirst%3DD.%26aulast%3DAlbertson%26aufirst%3DD.%2BG.%26aulast%3DWaldman%26aufirst%3DF.%2BM.%26aulast%3DMcCormick%26aufirst%3DF.%26aulast%3DDickson%26aufirst%3DR.%2BB.%26aulast%3DJohnson%26aufirst%3DM.%2BD.%26aulast%3DLippman%26aufirst%3DM.%26aulast%3DEthier%26aufirst%3DS.%26aulast%3DGazdar%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DJ.%2BW.%26atitle%3DA%2520collection%2520of%2520breast%2520cancer%2520cell%2520lines%2520for%2520the%2520study%2520of%2520functionally%2520distinct%2520cancer%2520subtypes%26jtitle%3DCancer%2520Cell%26date%3D2006%26volume%3D10%26spage%3D515%26epage%3D527%26doi%3D10.1016%2Fj.ccr.2006.10.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leclercq, G.</span></span> <span> </span><span class="NLM_article-title">Relevance of breast cancer cell lines as models for breast tumours: an update</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>83</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">289</span>, <span class="refDoi"> DOI: 10.1023/b:brea.0000014042.54925.cc</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1023%2Fb%3Abrea.0000014042.54925.cc" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=14758095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2cXos1Cisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=83&publication_year=2004&pages=249-289&author=M.+Lacroixauthor=G.+Leclercq&title=Relevance+of+breast+cancer+cell+lines+as+models+for+breast+tumours%3A+an+update&doi=10.1023%2Fb%3Abrea.0000014042.54925.cc"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Relevance of Breast Cancer Cell Lines as Models for Breast Tumours: An Update</span></div><div class="casAuthors">Lacroix, Marc; Leclercq, Guy</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">83</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">249-289</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic Publishers</span>)
        </div><div class="casAbstract">A review.  The no. of available breast cancer cell (BCC) lines is small, and only a very few of them were extensively studied.  Whether they are representative of the tumors from which they originated remains a matter of debate.  Whether their diversity mirrors the well-known inter-tumoral heterogeneity is another essential question.  While numerous similarities have long been found between cell lines and tumors, recent tech. advances, including the use of micro-arrays and comparative genetic anal., have brought new data to the discussion.  This paper presents most of the BCC lines that have been described in some detail to date.  It evaluates the accuracy of the few of them widely used (MCF-7, T-47D, BT-474, SK-BR-3, MDA-MB-231, Hs578T) as tumor models.  It is concluded that BCC lines are likely to reflect, to a large extent, the features of cancer cells in vivo.  The importance of estrogen receptor-α (gene ESR1) and Her-2/neu (ERBB2) as classifiers for cell lines and tumors is underlined.  The recourse to a larger set of cell lines is suggested since the exact origin of some of the widely used lines remains ambiguous.  Investigations on addnl. specific lines are expected to improve our knowledge of BCC and of the dialogue that these maintain with their surrounding normal cells in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGof86ncAv_qXrVg90H21EOLACvtfcHk0lhkGyvRDI2_YQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXos1Cisw%253D%253D&md5=d477e4aea1298cd3b2d6fe2ff4efcb08</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1023%2Fb%3Abrea.0000014042.54925.cc&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fb%253Abrea.0000014042.54925.cc%26sid%3Dliteratum%253Aachs%26aulast%3DLacroix%26aufirst%3DM.%26aulast%3DLeclercq%26aufirst%3DG.%26atitle%3DRelevance%2520of%2520breast%2520cancer%2520cell%2520lines%2520as%2520models%2520for%2520breast%2520tumours%253A%2520an%2520update%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2004%26volume%3D83%26spage%3D249%26epage%3D289%26doi%3D10.1023%2Fb%3Abrea.0000014042.54925.cc" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bacus, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiguchi, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">King, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huberman, E.</span></span> <span> </span><span class="NLM_article-title">Differentiation of cultured human breast cancer cells (AU-565 and MCF-7) associated with loss of cell surface HER-2/neu antigen</span>. <i>Mol. Carcinog.</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">350</span>– <span class="NLM_lpage">362</span>, <span class="refDoi"> DOI: 10.1002/mc.2940030607</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fmc.2940030607" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1980588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADyaK3MXitVCksL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1990&pages=350-362&author=S.+S.+Bacusauthor=K.+Kiguchiauthor=D.+Chinauthor=C.+R.+Kingauthor=E.+Huberman&title=Differentiation+of+cultured+human+breast+cancer+cells+%28AU-565+and+MCF-7%29+associated+with+loss+of+cell+surface+HER-2%2Fneu+antigen&doi=10.1002%2Fmc.2940030607"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Differentiation of cultured human breast cancer cells (AU-565 and MCF-7) associated with loss of cell surface HER-2/neu antigen</span></div><div class="casAuthors">Bacus, Sarah S.; Kiguchi, Kaoru; Chin, Dot; King, C. Richter; Huberman, Eliezer</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Carcinogenesis</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">350-62</span>CODEN:
                <span class="NLM_cas:coden">MOCAE8</span>;
        ISSN:<span class="NLM_cas:issn">0899-1987</span>.
    </div><div class="casAbstract">The relationship between terminal cell differentiation and changes in the subcellular levels of the HER-2/neu antigen was investigated in cultured human breast cancer cells: AU-565 cells (which overexpress the HER-2/neu gene) and MCF-7 cells (which do not overexpress this gene).  Differentiation was achieved by treating the cells with mycophenolic acid (MPA), phorbol 12-myristate 13-acetate (PMA), retinoic acid (RA), or the TA-1 monoclonal antibody to the extracellular domain of the HER-2/neu protein.  Ten to twenty percent of the cells in untreated, sparsely growing AU-565 cultures exhibited morphol. maturation characterized by large lacy nuclei surrounded by sizable flat cytoplasms.  A fraction of these cells harbored milk factors such as casein and large lipid droplets.  Treatment of the AU-565 cells for 4 d with 9 μM MPA, 1.6 nM PMA, 2.5 μM RA, or 1 μg/mL TA-1 antibody resulted in cell growth inhibition and an increase in the percentage of cells (48-97%) that exhibit a mature phenotype.  MCF-7 cells were also susceptible to differentiation by MPA and RA, but to a lesser degree than the AU-565 cells.  Differentiation in the AU-565 and MCF-7 cells was assocd. with reduced immunostaining for the HER-2/neu protein at the cell surface membrane and with a transient increased diffuse immunostaining for this protein in the cytoplasm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpI686OvH-udLVg90H21EOLACvtfcHk0ljmJuJDXtitEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXitVCksL4%253D&md5=f917f0c668f7a762e4dcd9063e414f77</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1002%2Fmc.2940030607&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmc.2940030607%26sid%3Dliteratum%253Aachs%26aulast%3DBacus%26aufirst%3DS.%2BS.%26aulast%3DKiguchi%26aufirst%3DK.%26aulast%3DChin%26aufirst%3DD.%26aulast%3DKing%26aufirst%3DC.%2BR.%26aulast%3DHuberman%26aufirst%3DE.%26atitle%3DDifferentiation%2520of%2520cultured%2520human%2520breast%2520cancer%2520cells%2520%2528AU-565%2520and%2520MCF-7%2529%2520associated%2520with%2520loss%2520of%2520cell%2520surface%2520HER-2%252Fneu%2520antigen%26jtitle%3DMol.%2520Carcinog.%26date%3D1990%26volume%3D3%26spage%3D350%26epage%3D362%26doi%3D10.1002%2Fmc.2940030607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Holliday, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Speirs, V.</span></span> <span> </span><span class="NLM_article-title">Choosing the right cell line for breast cancer research</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">215</span>, <span class="refDoi"> DOI: 10.1186/bcr2889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr2889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21884641" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2011&pages=215&author=D.+L.+Hollidayauthor=V.+Speirs&title=Choosing+the+right+cell+line+for+breast+cancer+research&doi=10.1186%2Fbcr2889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Choosing the right cell line for breast cancer research</span></div><div class="casAuthors">Holliday Deborah L; Speirs Valerie</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research : BCR</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">215</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer is a complex and heterogeneous disease.  Gene expression profiling has contributed significantly to our understanding of this heterogeneity at a molecular level, refining taxonomy based on simple measures such as histological type, tumour grade, lymph node status and the presence of predictive markers like oestrogen receptor and human epidermal growth factor receptor 2 (HER2) to a more sophisticated classification comprising luminal A, luminal B, basal-like, HER2-positive and normal subgroups.  In the laboratory, breast cancer is often modelled using established cell lines.  In the present review we discuss some of the issues surrounding the use of breast cancer cell lines as experimental models, in light of these revised clinical classifications, and put forward suggestions for improving their use in translational breast cancer research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR9z00kKYf1zjdblLE_E9PWfW6udTcc2eadnMQ5EEFeNLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC38%252FkvFCnuw%253D%253D&md5=9fbb9c5debbec50a5708666272fff46f</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1186%2Fbcr2889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr2889%26sid%3Dliteratum%253Aachs%26aulast%3DHolliday%26aufirst%3DD.%2BL.%26aulast%3DSpeirs%26aufirst%3DV.%26atitle%3DChoosing%2520the%2520right%2520cell%2520line%2520for%2520breast%2520cancer%2520research%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2011%26volume%3D13%26spage%3D215%26doi%3D10.1186%2Fbcr2889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Riaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Jaarsveld, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hollestelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prager-van der Smissen, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heine, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boersma, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helmijr, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ozturk, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiemer, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W.</span></span> <span> </span><span class="NLM_article-title">miRNA expression profiling of 51 human breast cancer cell lines reveals subtype and driver mutation-specific miRNAs</span>. <i>Breast Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">R33</span>, <span class="refDoi"> DOI: 10.1186/bcr3415</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fbcr3415" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23601657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVGqtb%252FP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2013&pages=R33&author=M.+Riazauthor=M.+T.+van+Jaarsveldauthor=A.+Hollestelleauthor=W.+J.+Prager-van+der+Smissenauthor=A.+A.+Heineauthor=A.+W.+Boersmaauthor=J.+Liuauthor=J.+Helmijrauthor=B.+Ozturkauthor=M.+Smidauthor=E.+A.+Wiemerauthor=J.+A.+Foekensauthor=J.+W.+Martens&title=miRNA+expression+profiling+of+51+human+breast+cancer+cell+lines+reveals+subtype+and+driver+mutation-specific+miRNAs&doi=10.1186%2Fbcr3415"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">miRNA expression profiling of 51 human breast cancer cell lines reveals subtype and driver mutation-specific miRNAs</span></div><div class="casAuthors">Riaz, Muhammad; van Jaarsveld, Marijn T. M.; Hollestelle, Antoinette; Prager-van der Smissen, Wendy J. C.; Heine, Anouk A. J.; Boersma, Antonius W. M.; Liu, Jingjing; Helmijr, Jean; Ozturk, Bahar; Smid, Marcel; Wiemer, Erik A.; Foekens, John A.; Martens, John W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">R33</span>CODEN:
                <span class="NLM_cas:coden">BRCRFS</span>;
        ISSN:<span class="NLM_cas:issn">1465-542X</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Introduction: Breast cancer is a genetically and phenotypically complex disease.  To understand the role of miRNAs in this mol. complexity, we performed miRNA expression anal. in a cohort of molecularly well-characterized human breast cancer cell lines to identify miRNAs assocd. with the most common mol. subtypes and the most frequent genetic aberrations.  Methods: Using a microarray carrying LNA modified oligonucleotide capture probes, expression levels of 725 human miRNAs were measured in 51 breast cancer cell lines.  Differential miRNA expression was explored by unsupervised cluster anal. and was then assocd. with the mol. subtypes and genetic aberrations commonly present in breast cancer.  Results: Unsupervised cluster anal. using the most variably expressed miRNAs divided the 51 breast cancer cell lines into a major and a minor cluster predominantly mirroring the luminal and basal intrinsic subdivision of breast cancer cell lines.  One hundred and thirteen miRNAs were differentially expressed between these two main clusters.  Forty miRNAs were differentially expressed between basal-like and normal-like/claudin-low cell lines.  Within the luminal-group, 39 miRNAs were assocd. with ERBB2 overexpression and 24 with E-cadherin gene mutations, which are frequent in this subtype of breast cancer cell lines.  In contrast, 31 miRNAs were assocd. with E-cadherin promoter hypermethylation, which, contrary to E-cadherin mutation, is exclusively obsd. in breast cancer cell lines that are not of luminal origin.  Thirty miRNAs were assocd. with p16INK4 status while only a few miRNAs were assocd. with BRCA1, PIK3CA/PTEN and TP53 mutation status.  Twelve miRNAs were assocd. with DNA copy no. variation of the resp. locus.  Conclusion: Luminal-basal and epithelial-mesenchymal assocd. miRNAs det. the subdivision of miRNA transcriptome of breast cancer cell lines.  Specific sets of miRNAs were assocd. with ERBB2 overexpression, p16INK4a or E-cadherin mutation or E-cadherin methylation status, which implies that these miRNAs may contribute to the driver role of these genetic aberrations.  Addnl., miRNAs, which are located in a genomic region showing recurrent genetic aberrations, may themselves play a driver role in breast carcinogenesis or contribute to a driver gene in their vicinity.  In short, our study provides detailed mol. miRNA portraits of breast cancer cell lines, which can be exploited for functional studies of clin. important miRNAs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWhcz19FuvErVg90H21EOLACvtfcHk0lhdyLUrJH4lNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVGqtb%252FP&md5=fc80fbb51e25f6a80be909c9bbd4fc0c</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1186%2Fbcr3415&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fbcr3415%26sid%3Dliteratum%253Aachs%26aulast%3DRiaz%26aufirst%3DM.%26aulast%3Dvan%2BJaarsveld%26aufirst%3DM.%2BT.%26aulast%3DHollestelle%26aufirst%3DA.%26aulast%3DPrager-van%2Bder%2BSmissen%26aufirst%3DW.%2BJ.%26aulast%3DHeine%26aufirst%3DA.%2BA.%26aulast%3DBoersma%26aufirst%3DA.%2BW.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DHelmijr%26aufirst%3DJ.%26aulast%3DOzturk%26aufirst%3DB.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DWiemer%26aufirst%3DE.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%26atitle%3DmiRNA%2520expression%2520profiling%2520of%252051%2520human%2520breast%2520cancer%2520cell%2520lines%2520reveals%2520subtype%2520and%2520driver%2520mutation-specific%2520miRNAs%26jtitle%3DBreast%2520Cancer%2520Res.%26date%3D2013%26volume%3D15%26spage%3DR33%26doi%3D10.1186%2Fbcr3415" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hollestelle, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, J. H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smid, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elstrodt, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wasielewski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">French, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peeters, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozendaal, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riaz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koopman, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ten Hagen, T. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Leeuw, B. H. C. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwarthoff, E. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teunisse, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Spek, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klijn, J. G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dinjens, W. N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ethier, S. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clevers, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jochemsen, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">den Bakker, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foekens, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martens, J. W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schutte, M.</span></span> <span> </span><span class="NLM_article-title">Distinct gene mutation profiles among luminal-type and basal-type breast cancer cell lines</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1007/s10549-009-0460-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs10549-009-0460-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19593635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3c3mslemtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2010&pages=53-64&author=A.+Hollestelleauthor=J.+H.+A.+Nagelauthor=M.+Smidauthor=S.+Lamauthor=F.+Elstrodtauthor=M.+Wasielewskiauthor=S.+S.+Ngauthor=P.+J.+Frenchauthor=J.+K.+Peetersauthor=M.+J.+Rozendaalauthor=M.+Riazauthor=D.+G.+Koopmanauthor=T.+L.+M.+Ten+Hagenauthor=B.+H.+C.+G.+M.+de+Leeuwauthor=E.+C.+Zwarthoffauthor=A.+Teunisseauthor=P.+J.+van+der+Spekauthor=J.+G.+M.+Klijnauthor=W.+N.+M.+Dinjensauthor=S.+P.+Ethierauthor=H.+Cleversauthor=A.+G.+Jochemsenauthor=M.+A.+den+Bakkerauthor=J.+A.+Foekensauthor=J.+W.+M.+Martensauthor=M.+Schutte&title=Distinct+gene+mutation+profiles+among+luminal-type+and+basal-type+breast+cancer+cell+lines&doi=10.1007%2Fs10549-009-0460-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Distinct gene mutation profiles among luminal-type and basal-type breast cancer cell lines</span></div><div class="casAuthors">Hollestelle Antoinette; Nagel Jord H A; Smid Marcel; Lam Suzanne; Elstrodt Fons; Wasielewski Marijke; Ng Ser Sue; French Pim J; Peeters Justine K; Rozendaal Marieke J; Riaz Muhammad; Koopman Daphne G; Ten Hagen Timo L M; de Leeuw Bertie H C G M; Zwarthoff Ellen C; Teunisse Amina; van der Spek Peter J; Klijn Jan G M; Dinjens Winand N M; Ethier Stephen P; Clevers Hans; Jochemsen Aart G; den Bakker Michael A; Foekens John A; Martens John W M; Schutte Mieke</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-64</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Breast cancer has for long been recognized as a highly diverse tumor group, but the underlying genetic basis has been elusive.  Here, we report an extensive molecular characterization of a collection of 41 human breast cancer cell lines.  Protein and gene expression analyses indicated that the collection of breast cancer cell lines has retained most, if not all, molecular characteristics that are typical for clinical breast cancers.  Gene mutation analyses identified 146 oncogenic mutations among 27 well-known cancer genes, amounting to an average of 3.6 mutations per cell line.  Mutations in genes from the p53, RB and PI3K tumor suppressor pathways were widespread among all breast cancer cell lines.  Most important, we have identified two gene mutation profiles that are specifically associated with luminal-type and basal-type breast cancer cell lines.  The luminal mutation profile involved E-cadherin and MAP2K4 gene mutations and amplifications of Cyclin D1, ERBB2 and HDM2, whereas the basal mutation profile involved BRCA1, RB1, RAS and BRAF gene mutations and deletions of p16 and p14ARF.  These subtype-specific gene mutation profiles constitute a genetic basis for the heterogeneity observed among human breast cancers, providing clues for their underlying biology and providing guidance for targeted pharmacogenetic intervention in breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRyyljB3E0iTtenUz0lpNqnfW6udTcc2eZ7UxD1ge5Em7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c3mslemtQ%253D%253D&md5=6b501ebd85d79898ec1fcce1608de5e0</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1007%2Fs10549-009-0460-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-009-0460-8%26sid%3Dliteratum%253Aachs%26aulast%3DHollestelle%26aufirst%3DA.%26aulast%3DNagel%26aufirst%3DJ.%2BH.%2BA.%26aulast%3DSmid%26aufirst%3DM.%26aulast%3DLam%26aufirst%3DS.%26aulast%3DElstrodt%26aufirst%3DF.%26aulast%3DWasielewski%26aufirst%3DM.%26aulast%3DNg%26aufirst%3DS.%2BS.%26aulast%3DFrench%26aufirst%3DP.%2BJ.%26aulast%3DPeeters%26aufirst%3DJ.%2BK.%26aulast%3DRozendaal%26aufirst%3DM.%2BJ.%26aulast%3DRiaz%26aufirst%3DM.%26aulast%3DKoopman%26aufirst%3DD.%2BG.%26aulast%3DTen%2BHagen%26aufirst%3DT.%2BL.%2BM.%26aulast%3Dde%2BLeeuw%26aufirst%3DB.%2BH.%2BC.%2BG.%2BM.%26aulast%3DZwarthoff%26aufirst%3DE.%2BC.%26aulast%3DTeunisse%26aufirst%3DA.%26aulast%3Dvan%2Bder%2BSpek%26aufirst%3DP.%2BJ.%26aulast%3DKlijn%26aufirst%3DJ.%2BG.%2BM.%26aulast%3DDinjens%26aufirst%3DW.%2BN.%2BM.%26aulast%3DEthier%26aufirst%3DS.%2BP.%26aulast%3DClevers%26aufirst%3DH.%26aulast%3DJochemsen%26aufirst%3DA.%2BG.%26aulast%3Dden%2BBakker%26aufirst%3DM.%2BA.%26aulast%3DFoekens%26aufirst%3DJ.%2BA.%26aulast%3DMartens%26aufirst%3DJ.%2BW.%2BM.%26aulast%3DSchutte%26aufirst%3DM.%26atitle%3DDistinct%2520gene%2520mutation%2520profiles%2520among%2520luminal-type%2520and%2520basal-type%2520breast%2520cancer%2520cell%2520lines%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2010%26volume%3D121%26spage%3D53%26epage%3D64%26doi%3D10.1007%2Fs10549-009-0460-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bocanegra, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandhi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwei, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez-Boussard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazdar, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minna, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollack, J. R.</span></span> <span> </span><span class="NLM_article-title">Molecular profiling of breast cancer cell lines defines relevant tumor models and provides a resource for cancer gene discovery</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">e6146</span>– <span class="NLM_lpage">e6161</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0006146</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0006146" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=19582160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD1MvmvV2kug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=e6146-e6161&author=J.+Kaoauthor=K.+Salariauthor=M.+Bocanegraauthor=Y.-L.+Choiauthor=L.+Girardauthor=J.+Gandhiauthor=K.+A.+Kweiauthor=T.+Hernandez-Boussardauthor=P.+Wangauthor=A.+F.+Gazdarauthor=J.+D.+Minnaauthor=J.+R.+Pollack&title=Molecular+profiling+of+breast+cancer+cell+lines+defines+relevant+tumor+models+and+provides+a+resource+for+cancer+gene+discovery&doi=10.1371%2Fjournal.pone.0006146"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular profiling of breast cancer cell lines defines relevant tumor models and provides a resource for cancer gene discovery</span></div><div class="casAuthors">Kao Jessica; Salari Keyan; Bocanegra Melanie; Choi Yoon-La; Girard Luc; Gandhi Jeet; Kwei Kevin A; Hernandez-Boussard Tina; Wang Pei; Gazdar Adi F; Minna John D; Pollack Jonathan R</div><div class="citationInfo"><span class="NLM_cas:title">PloS one</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e6146</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Breast cancer cell lines have been used widely to investigate breast cancer pathobiology and new therapies.  Breast cancer is a molecularly heterogeneous disease, and it is important to understand how well and which cell lines best model that diversity.  In particular, microarray studies have identified molecular subtypes-luminal A, luminal B, ERBB2-associated, basal-like and normal-like-with characteristic gene-expression patterns and underlying DNA copy number alterations (CNAs).  Here, we studied a collection of breast cancer cell lines to catalog molecular profiles and to assess their relation to breast cancer subtypes.  METHODS:  Whole-genome DNA microarrays were used to profile gene expression and CNAs in a collection of 52 widely-used breast cancer cell lines, and comparisons were made to existing profiles of primary breast tumors.  Hierarchical clustering was used to identify gene-expression subtypes, and Gene Set Enrichment Analysis (GSEA) to discover biological features of those subtypes.  Genomic and transcriptional profiles were integrated to discover within high-amplitude CNAs candidate cancer genes with coordinately altered gene copy number and expression.  FINDINGS:  Transcriptional profiling of breast cancer cell lines identified one luminal and two basal-like (A and B) subtypes.  Luminal lines displayed an estrogen receptor (ER) signature and resembled luminal-A/B tumors, basal-A lines were associated with ETS-pathway and BRCA1 signatures and resembled basal-like tumors, and basal-B lines displayed mesenchymal and stem/progenitor-cell characteristics.  Compared to tumors, cell lines exhibited similar patterns of CNA, but an overall higher complexity of CNA (genetically simple luminal-A tumors were not represented), and only partial conservation of subtype-specific CNAs.  We identified 80 high-level DNA amplifications and 13 multi-copy deletions, and the resident genes with concomitantly altered gene-expression, highlighting known and novel candidate breast cancer genes.  CONCLUSIONS:  Overall, breast cancer cell lines were genetically more complex than tumors, but retained expression patterns with relevance to the luminal-basal subtype distinction.  The compendium of molecular profiles defines cell lines suitable for investigations of subtype-specific pathobiology, cancer stem cell biology, biomarkers and therapies, and provides a resource for discovery of new breast cancer genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ64pI1FRXP5q__Jf7-eMvJfW6udTcc2ebrItUmLwjYpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD1MvmvV2kug%253D%253D&md5=4fb2fa78b74a7270fbd98aa491bca206</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0006146&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0006146%26sid%3Dliteratum%253Aachs%26aulast%3DKao%26aufirst%3DJ.%26aulast%3DSalari%26aufirst%3DK.%26aulast%3DBocanegra%26aufirst%3DM.%26aulast%3DChoi%26aufirst%3DY.-L.%26aulast%3DGirard%26aufirst%3DL.%26aulast%3DGandhi%26aufirst%3DJ.%26aulast%3DKwei%26aufirst%3DK.%2BA.%26aulast%3DHernandez-Boussard%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DP.%26aulast%3DGazdar%26aufirst%3DA.%2BF.%26aulast%3DMinna%26aufirst%3DJ.%2BD.%26aulast%3DPollack%26aufirst%3DJ.%2BR.%26atitle%3DMolecular%2520profiling%2520of%2520breast%2520cancer%2520cell%2520lines%2520defines%2520relevant%2520tumor%2520models%2520and%2520provides%2520a%2520resource%2520for%2520cancer%2520gene%2520discovery%26jtitle%3DPLoS%2520One%26date%3D2009%26volume%3D4%26spage%3De6146%26epage%3De6161%26doi%3D10.1371%2Fjournal.pone.0006146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anido, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matar, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albanell, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzmán, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rojo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arribas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Averbuch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baselga, J.</span></span> <span> </span><span class="NLM_article-title">ZD1839, a specific epidermal growth factor receptor (EGFR) tyrosine kinase inhibitor, induces the formation of inactive EGFR/HER2 and EGFR/HER3 heterodimers and prevents heregulin signaling in HER2-overexpressing breast cancer cells</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1274</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12684395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXislertrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2003&pages=1274&author=J.+Anidoauthor=P.+Matarauthor=J.+Albanellauthor=M.+Guzm%C3%A1nauthor=F.+Rojoauthor=J.+Arribasauthor=S.+Averbuchauthor=J.+Baselga&title=ZD1839%2C+a+specific+epidermal+growth+factor+receptor+%28EGFR%29+tyrosine+kinase+inhibitor%2C+induces+the+formation+of+inactive+EGFR%2FHER2+and+EGFR%2FHER3+heterodimers+and+prevents+heregulin+signaling+in+HER2-overexpressing+breast+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">ZD1839, a Specific Epidermal Growth Factor Receptor (EGFR) Tyrosine Kinase Inhibitor, Induces the Formation of Inactive EGFR/HER2 and EGFR/HER3 Heterodimers and Prevents Heregulin Signaling in HER2-overexpressing Breast Cancer Cells</span></div><div class="casAuthors">Anido, Judit; Matar, Pablo; Albanell, Joan; Guzman, Marta; Rojo, Federico; Arribas, Joaquin; Averbuch, Steve; Baselga, Jose</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1274-1283</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">ZD1839 is a tyrosine kinase inhibitor of the epidermal growth factor receptor (EGFR) that has shown clin. activity against EGFR-expressing tumors.  Our aim was to explore the effects of ZD1839 in breast cancer cell lines expressing different levels of EGFR and the closely related HER2 receptor.  We studied the growth-inhibitory effects of ZD1839 in a series of breast carcinoma cell lines.  In HER2-overexpressing BT-474 breast cancer cells, we studied the effects of ZD1839 on cell growth and heterodimerization of receptors under basal and ligand-stimulated conditions.  ZD1839 was an equally potent inhibitor of growth in breast cancer cells expressing high levels of EGFR and HER2.  In BT-474 breast cancer cells, ZD1839 abolished EGF- and heregulin-induced activation of ErbB receptors and downstream signaling mols.  Because ZD1839 does not inhibit the HER2 tyrosine kinase in vitro, and because heregulin is a ligand that activates HER2 by binding to HER3 and HER4 but does not bind to the EGFR, our findings suggested that ZD1839 interfered with HER2 function in intact cells.  Searching for mechanisms, we report that ZD1839 induces the formation of inactive unphosphorylated EGFR/HER2 and EGFR/HER3 heterodimers.  Furthermore, ZD1839 completely abolishes basal and heregulin-induced formation of active phosphorylated HER2/HER3 heterodimers.  ZD1839 inhibits the growth of HER2-overexpressing breast cancer cells, possibly by sequestration of HER2 and HER3 receptors in an inactive heterodimer configuration with the EGFR.  Our findings suggest that there is a strong rationale to conduct clin. trials of ZD1839 in patients with HER2-overexpressing breast tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoA7DnoL5ZfL7Vg90H21EOLACvtfcHk0lgtd8OnCPwN6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXislertrs%253D&md5=a966a0569f5d9ff89e61b449c9bea1cd</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAnido%26aufirst%3DJ.%26aulast%3DMatar%26aufirst%3DP.%26aulast%3DAlbanell%26aufirst%3DJ.%26aulast%3DGuzm%25C3%25A1n%26aufirst%3DM.%26aulast%3DRojo%26aufirst%3DF.%26aulast%3DArribas%26aufirst%3DJ.%26aulast%3DAverbuch%26aufirst%3DS.%26aulast%3DBaselga%26aufirst%3DJ.%26atitle%3DZD1839%252C%2520a%2520specific%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520tyrosine%2520kinase%2520inhibitor%252C%2520induces%2520the%2520formation%2520of%2520inactive%2520EGFR%252FHER2%2520and%2520EGFR%252FHER3%2520heterodimers%2520and%2520prevents%2520heregulin%2520signaling%2520in%2520HER2-overexpressing%2520breast%2520cancer%2520cells%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2003%26volume%3D9%26spage%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Witton, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reeves, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Going, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooke, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartlett, J. M.</span></span> <span> </span><span class="NLM_article-title">Expression of the HER1-4 family of receptor tyrosine kinases in breast cancer</span>. <i>J. Pathol.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>200</i></span>,  <span class="NLM_fpage">290</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1002/path.1370</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fpath.1370" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=12845624" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnvF2gu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=200&publication_year=2003&pages=290-297&author=C.+J.+Wittonauthor=J.+R.+Reevesauthor=J.+J.+Goingauthor=T.+G.+Cookeauthor=J.+M.+Bartlett&title=Expression+of+the+HER1-4+family+of+receptor+tyrosine+kinases+in+breast+cancer&doi=10.1002%2Fpath.1370"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Expression of the HER1-4 family of receptor tyrosine kinases in breast cancer</span></div><div class="casAuthors">Witton, Caroline J.; Reeves, Jonathan R.; Going, James J.; Cooke, Timothy G.; Bartlett, John M. S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pathology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">200</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">290-297</span>CODEN:
                <span class="NLM_cas:coden">JPTLAS</span>;
        ISSN:<span class="NLM_cas:issn">0022-3417</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons Ltd.</span>)
        </div><div class="casAbstract">EGFr/HER1 and c-erbB-2/HER2 expression are assocd. with poor prognosis in breast cancer.  The type I receptor tyrosine kinase (RTK) family to which they belong has four members (HER1-4).  In this study, expression of HER1-4 and estrogen receptor (ER) expression were detd. by immunohistochem. in 220 breast carcinomas.  Elevated expression of HER1 was obsd. in 16.4%, HER2 in 22.8%, HER3 in 17.5%, and HER4 in 11.9% of these tumors.  Patients whose tumors overexpressed HER1, 2 or 3 had reduced survival (p =< 0.001), whereas those whose tumors overexpressed HER4 had increased survival (p = 0.013); 38.6% of cases overexpressed one or more of HER1, 2 or 3.  HER4 was rarely overexpressed with other HERs (1.4% of cases).  Cox's multiple regression anal. demonstrated that overexpression of HER1/2/3, HER4, and std. prognostic indicators independently affected survival.  HER1-3 expression was related to ER negativity (p < 0.0001, ηhiX2).  Patients with ER-pos., HER1-3-pos. tumors had a significantly poorer survival (p < 0.001) than those with ER-pos./HER-neg. or HER4-pos. tumors.  Expression of HER RTKs displays complex interactions between different family members.  There is a strong interaction, in terms of survival, between HER expression and ER expression.  The development of HER-targeted agents (eg Herceptin, Iressa), and agents targeted at the downstream signalling pathways, therefore provides new possibilities in the treatment of breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgp4ubXdx7ibVg90H21EOLACvtfcHk0lgtd8OnCPwN6A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnvF2gu7c%253D&md5=0b81cfc4bca4857f5380e2ba7635f75d</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1002%2Fpath.1370&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpath.1370%26sid%3Dliteratum%253Aachs%26aulast%3DWitton%26aufirst%3DC.%2BJ.%26aulast%3DReeves%26aufirst%3DJ.%2BR.%26aulast%3DGoing%26aufirst%3DJ.%2BJ.%26aulast%3DCooke%26aufirst%3DT.%2BG.%26aulast%3DBartlett%26aufirst%3DJ.%2BM.%26atitle%3DExpression%2520of%2520the%2520HER1-4%2520family%2520of%2520receptor%2520tyrosine%2520kinases%2520in%2520breast%2520cancer%26jtitle%3DJ.%2520Pathol.%26date%3D2003%26volume%3D200%26spage%3D290%26epage%3D297%26doi%3D10.1002%2Fpath.1370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsutsui, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murakami, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hachitanda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oda, S.</span></span> <span> </span><span class="NLM_article-title">Prognostic value of epidermal growth factor receptor (EGFR) and its relationship to the estrogen receptor status in 1029 patients with breast cancer</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>71</i></span>,  <span class="NLM_fpage">67</span>– <span class="NLM_lpage">75</span>, <span class="refDoi"> DOI: 10.1023/a:1013397232011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1023%2Fa%3A1013397232011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=11859875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BD387isFCisw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2002&pages=67-75&author=S.+Tsutsuiauthor=S.+Ohnoauthor=S.+Murakamiauthor=Y.+Hachitandaauthor=S.+Oda&title=Prognostic+value+of+epidermal+growth+factor+receptor+%28EGFR%29+and+its+relationship+to+the+estrogen+receptor+status+in+1029+patients+with+breast+cancer&doi=10.1023%2Fa%3A1013397232011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic value of epidermal growth factor receptor (EGFR) and its relationship to the estrogen receptor status in 1029 patients with breast cancer</span></div><div class="casAuthors">Tsutsui Shinichi; Ohno Shinji; Murakami Shigeru; Hachitanda Yoichi; Oda Shinya</div><div class="citationInfo"><span class="NLM_cas:title">Breast cancer research and treatment</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">71</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">67-75</span>
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    </div><div class="casAbstract">An epidermal growth factor receptor (EGFR) has been reported to be associated with a poor clinical outcome in breast cancer, while its prognostic value remains controversial.  Immunohistochemical staining for EGFR was performed on frozen sections of primary breast cancer from 1029 patients with a mean follow-up duration of 46 months.  EGFR was positive in 277 (26.9%) of 1029 cases which inversely correlated with the estrogen receptor (ER) status.  A univariated analysis indicated that EGFR had a significant prognostic value in both the disease free survival (DFS) and the overall survival (OS), while the same effect was also found in node negative as well as node positive breast cancer.  A multivariate analysis indicated that EGFR was an independently significant prognostic factor for DFS (p = 0.0174) and OS (p = 0.0105) in all patients, but that EGFR demonstrated a prognostic significance only for DFS (p = 0.0241) in node negative and only for OS (p = 0.0333) in node positive breast cancer.  When all patients were stratified for EGFR and ER, a multivariate analysis indicated that the combination of EGFR(+)/ER(-) was an independently significant factor for both DFS and OS in node negative as well as node positive breast cancer.  In conclusion, the prognostic value of EGFR was demonstrated by a multivariate analysis in a large series of breast cancer patients, but the value of EGFR was somewhat insufficient to achieve statistical significance for both DFS and OS in the subgroups divided by nodal status.  On the other hand, the prognostic value of combination of EGFR and ER was sufficient to achieve statistical significance based on a multivariate analysis for both DFS and OS in the subgroups of node negative as well as node positive breast cancer patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTkZRi_NrcOMuIHSmpohpd0fW6udTcc2eaHP9J2gWBLTLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD387isFCisw%253D%253D&md5=52839385eb2e92b9598baea7c23d3b5b</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1023%2Fa%3A1013397232011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252Fa%253A1013397232011%26sid%3Dliteratum%253Aachs%26aulast%3DTsutsui%26aufirst%3DS.%26aulast%3DOhno%26aufirst%3DS.%26aulast%3DMurakami%26aufirst%3DS.%26aulast%3DHachitanda%26aufirst%3DY.%26aulast%3DOda%26aufirst%3DS.%26atitle%3DPrognostic%2520value%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520and%2520its%2520relationship%2520to%2520the%2520estrogen%2520receptor%2520status%2520in%25201029%2520patients%2520with%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2002%26volume%3D71%26spage%3D67%26epage%3D75%26doi%3D10.1023%2Fa%3A1013397232011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rakha, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Sayed, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, I. O.</span></span> <span> </span><span class="NLM_article-title">Prognostic markers in triple-negative breast cancer</span>. <i>Cancer</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">32</span>, <span class="refDoi"> DOI: 10.1002/cncr.22381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcncr.22381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17146782" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVKrtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=25-32&author=E.+A.+Rakhaauthor=M.+E.+El-Sayedauthor=A.+R.+Greenauthor=A.+H.+S.+Leeauthor=J.+F.+Robertsonauthor=I.+O.+Ellis&title=Prognostic+markers+in+triple-negative+breast+cancer&doi=10.1002%2Fcncr.22381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic markers in triple-negative breast cancer</span></div><div class="casAuthors">Rakha, Emad A.; El-Sayed, Maysa E.; Green, Andrew R.; Lee, Andrew H. S.; Robertson, John F.; Ellis, Ian O.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer (Hoboken, NJ, United States)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25-32</span>CODEN:
                <span class="NLM_cas:coden">CANCAR</span>;
        ISSN:<span class="NLM_cas:issn">0008-543X</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">BACKGROUND. Triple-neg. breast cancer (estrogen receptor-neg., progesterone receptor-neg., and HER2-neg.) is a high risk breast cancer that lacks the benefit of specific therapy that targets these proteins.  METHODS. In this study, the authors examd. a large and well characterized series of invasive breast carcinoma (n = 1944) with a long-term clin. follow-up (median, 56 mo) by using tissue microarray.  The series were also stained with concurrent immunohistochem. prognostic panels (estrogen receptor, progesterone receptor, HER-2, androgen receptor, epidermal growth factor receptor (EGFR), P-cadherin, E-cadherin, and basal (CK5/6, CK14), and p53), to characterize this specific subgroup of breast cancer and to identify prognostic markers that can identify tumors with more aggressive behavior.  RESULTS. Of informative cases, 16.3% were of the triple-neg. phenotype.  The majority of these tumors were grade 3, ductal/no-specific-type carcinomas.  There were pos. assocns. with larger size, pushing margins, poorer Nottingham Prognostic Index, development of recurrence and distant metastasis, and poorer outcome.  In addn., assocns. were found with loss of expression of androgen receptor and E-cadherin, and pos. expression of basal cytokeratins (basal phenotype), P-cadherin, p53, and EGFR.  In all tumors, tumor size, lymph node stage, and androgen receptor were the most useful prognostic markers.  In the lymph node-pos. subgroup, both size and androgen receptor retained their prognostic significance.  However, in the lymph node-neg. tumors, basal phenotype was the sole prognostic marker identified in this subgroup.  Other parameters including age, histol. grade, tumor size, vascular invasion or other biomarkers included in the current study were not significant.  CONCLUSIONS. The authors concluded that assessment of androgen receptor and basal phenotype, in addn. to the established pathol. variables, mainly lymph node status and tumor size, can be used to select high-risk and low-risk patients at the time of primary surgery and can provide valuable information on treatment options in these triple-neg. tumors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQnQBrj7p6jLVg90H21EOLACvtfcHk0lhagYdTmbMz7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVKrtLY%253D&md5=fddf1e152783977dd1b7d525b7435ac7</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1002%2Fcncr.22381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcncr.22381%26sid%3Dliteratum%253Aachs%26aulast%3DRakha%26aufirst%3DE.%2BA.%26aulast%3DEl-Sayed%26aufirst%3DM.%2BE.%26aulast%3DGreen%26aufirst%3DA.%2BR.%26aulast%3DLee%26aufirst%3DA.%2BH.%2BS.%26aulast%3DRobertson%26aufirst%3DJ.%2BF.%26aulast%3DEllis%26aufirst%3DI.%2BO.%26atitle%3DPrognostic%2520markers%2520in%2520triple-negative%2520breast%2520cancer%26jtitle%3DCancer%26date%3D2007%26volume%3D109%26spage%3D25%26epage%3D32%26doi%3D10.1002%2Fcncr.22381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bauer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarthy, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shyr, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pietenpol, J. A.</span></span> <span> </span><span class="NLM_article-title">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>121</i></span>,  <span class="NLM_fpage">2750</span>– <span class="NLM_lpage">2767</span>, <span class="refDoi"> DOI: 10.1172/jci45014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1172%2Fjci45014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21633166" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXovVKgu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=121&publication_year=2011&pages=2750-2767&author=B.+D.+Lehmannauthor=J.+A.+Bauerauthor=X.+Chenauthor=M.+E.+Sandersauthor=A.+B.+Chakravarthyauthor=Y.+Shyrauthor=J.+A.+Pietenpol&title=Identification+of+human+triple-negative+breast+cancer+subtypes+and+preclinical+models+for+selection+of+targeted+therapies&doi=10.1172%2Fjci45014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of human triple-negative breast cancer subtypes and preclinical models for selection of targeted therapies</span></div><div class="casAuthors">Lehmann, Brian D.; Bauer, Joshua A.; Chen, Xi; Sanders, Melinda E.; Chakravarthy, A. Bapsi; Shyr, Yu; Pietenpol, Jennifer A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">121</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2750-2767</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    
            (<span class="NLM_cas:orgname">American Society for Clinical Investigation</span>)
        </div><div class="casAbstract">Triple-neg. breast cancer (TNBC) is a highly diverse group of cancers, and subtyping is necessary to better identify mol.-based therapies.  In this study, we analyzed gene expression (GE) profiles from 21 breast cancer data sets and identified 587 TNBC cases.  Cluster anal. identified 6 TNBC subtypes displaying unique GE and ontologies, including 2 basal-like (BL1 and BL2), an immunomodulatory (IM), a mesenchymal (M), a mesenchymal stem-like (MSL), and a luminal androgen receptor (LAR) subtype.  Further, GE anal. allowed us to identify TNBC cell line models representative of these subtypes.  Predicted "driver" signaling pathways were pharmacol. targeted in these cell line models as proof of concept that anal. of distinct GE signatures can inform therapy selection.  BL1 and BL2 subtypes had higher expression of cell cycle and DNA damage response genes, and representative cell lines preferentially responded to cisplatin.  M and MSL subtypes were enriched in GE for epithelial-mesenchymal transition, and growth factor pathways and cell models responded to NVP-BEZ235 (a PI3K/mTOR inhibitor) and dasatinib (an abl/src inhibitor).  The LAR subtype includes patients with decreased relapse-free survival and was characterized by androgen receptor (AR) signaling.  LAR cell lines were uniquely sensitive to bicalutamide (an AR antagonist).  These data may be useful in biomarker selection, drug discovery, and clin. trial design that will enable alignment of TNBC patients to appropriate targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbhoQQUFogW7Vg90H21EOLACvtfcHk0lgMhXVb5gvKhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXovVKgu78%253D&md5=c3681c79e0b972f16f6d5a5ecdc99fec</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1172%2Fjci45014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci45014%26sid%3Dliteratum%253Aachs%26aulast%3DLehmann%26aufirst%3DB.%2BD.%26aulast%3DBauer%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DSanders%26aufirst%3DM.%2BE.%26aulast%3DChakravarthy%26aufirst%3DA.%2BB.%26aulast%3DShyr%26aufirst%3DY.%26aulast%3DPietenpol%26aufirst%3DJ.%2BA.%26atitle%3DIdentification%2520of%2520human%2520triple-negative%2520breast%2520cancer%2520subtypes%2520and%2520preclinical%2520models%2520for%2520selection%2520of%2520targeted%2520therapies%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2011%26volume%3D121%26spage%3D2750%26epage%3D2767%26doi%3D10.1172%2Fjci45014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeusz, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doihara, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hortobagyi, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ueno, N. T.</span></span> <span> </span><span class="NLM_article-title">Role of epidermal growth factor receptor in breast cancer</span>. <i>Breast Cancer Res. Treat.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">345</span>, <span class="refDoi"> DOI: 10.1007/s10549-012-2289-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1007%2Fs10549-012-2289-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23073759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsF2qt7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2012&pages=331-345&author=H.+Masudaauthor=D.+Zhangauthor=C.+Bartholomeuszauthor=H.+Doiharaauthor=G.+N.+Hortobagyiauthor=N.+T.+Ueno&title=Role+of+epidermal+growth+factor+receptor+in+breast+cancer&doi=10.1007%2Fs10549-012-2289-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Role of epidermal growth factor receptor in breast cancer</span></div><div class="casAuthors">Masuda, Hiroko; Zhang, Dongwei; Bartholomeusz, Chandra; Doihara, Hiroyoshi; Hortobagyi, Gabriel N.; Ueno, Naoto T.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">331-345</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  Decades of research in mol. oncol. have brought about promising new therapies which are designed to target specific mols. which promote tumor growth and survival.  The epidermal growth factor receptor (EGFR) is one of the first identified important targets of these novel antitumor agents.  Approx. half of cases of triple-neg. breast cancer (TNBC) and inflammatory breast cancer (IBC) overexpress EGFR.  Thus, EGFR inhibitors for treatment of breast cancer have been evaluated in several studies.  However, results so far have been disappointing.  One of the reasons for these unexpected results is the lack of biomarkers for predicting which patients are most likely to respond to EGFR inhibitors.  Recent studies have shown that EGFR and its downstream pathway regulate epithelial-mesenchymal transition, migration, and tumor invasion and that high EGFR expression is an independent predictor of poor prognosis in IBC.  Further, recent studies have shown that targeting EGFR enhances the chemosensitivity of TNBC cells by rewiring apoptotic signaling networks in TNBC.  These studies indicate that EGFR-targeted therapy might have a promising role in TNBC and IBC.  Further studies of the role of EGFR in TNBC and IBC are needed to better understand the best way to use EGFR-targeted therapy-e.g., as a chemosensitizer or to prevent metastases-to treat these aggressive diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpG80rBn6V0ubVg90H21EOLACvtfcHk0lh7nUWND_v0bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsF2qt7vN&md5=61a75d8b05c3c0717a6959c1d047990f</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10549-012-2289-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-012-2289-9%26sid%3Dliteratum%253Aachs%26aulast%3DMasuda%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DD.%26aulast%3DBartholomeusz%26aufirst%3DC.%26aulast%3DDoihara%26aufirst%3DH.%26aulast%3DHortobagyi%26aufirst%3DG.%2BN.%26aulast%3DUeno%26aufirst%3DN.%2BT.%26atitle%3DRole%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520in%2520breast%2520cancer%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2012%26volume%3D136%26spage%3D331%26epage%3D345%26doi%3D10.1007%2Fs10549-012-2289-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chavez, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garimella, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipkowitz, S.</span></span> <span> </span><span class="NLM_article-title">Triple negative breast cancer cell lines: one tool in the search for better treatment of triple negative breast cancer</span>. <i>Breast Dis.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">35</span>– <span class="NLM_lpage">48</span>, <span class="refDoi"> DOI: 10.3233/BD-2010-0307</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.3233%2FBD-2010-0307" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21778573" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC3Mbls1Sjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2010&pages=35-48&author=K.+J.+Chavezauthor=S.+V.+Garimellaauthor=S.+Lipkowitz&title=Triple+negative+breast+cancer+cell+lines%3A+one+tool+in+the+search+for+better+treatment+of+triple+negative+breast+cancer&doi=10.3233%2FBD-2010-0307"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">Triple negative breast cancer cell lines: one tool in the search for better treatment of triple negative breast cancer</span></div><div class="casAuthors">Chavez Kathryn J; Garimella Sireesha V; Lipkowitz Stanley</div><div class="citationInfo"><span class="NLM_cas:title">Breast disease</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">35-48</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSP84JmqLJuYJVVjGoY74BafW6udTcc2eZLqOIihzFBZrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3Mbls1Sjtg%253D%253D&md5=bb3b4f1ddd945c1bd8e79c41fa44b592</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.3233%2FBD-2010-0307&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3233%252FBD-2010-0307%26sid%3Dliteratum%253Aachs%26aulast%3DChavez%26aufirst%3DK.%2BJ.%26aulast%3DGarimella%26aufirst%3DS.%2BV.%26aulast%3DLipkowitz%26aufirst%3DS.%26atitle%3DTriple%2520negative%2520breast%2520cancer%2520cell%2520lines%253A%2520one%2520tool%2520in%2520the%2520search%2520for%2520better%2520treatment%2520of%2520triple%2520negative%2520breast%2520cancer%26jtitle%3DBreast%2520Dis.%26date%3D2010%26volume%3D32%26spage%3D35%26epage%3D48%26doi%3D10.3233%2FBD-2010-0307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mowafy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and in vitro anti-proliferative activity of 4,6-quinazolinediamines as potent EGFR-TK inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">132</span>– <span class="NLM_lpage">145</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2012.10.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2012.10.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23142066" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Kjsb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2013&pages=132-145&author=S.+Mowafyauthor=N.+A.+Faragauthor=K.+A.+M.+Abouzid&title=Design%2C+synthesis+and+in+vitro+anti-proliferative+activity+of+4%2C6-quinazolinediamines+as+potent+EGFR-TK+inhibitors&doi=10.1016%2Fj.ejmech.2012.10.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and in vitro anti-proliferative activity of 4,6-quinazolinediamines as potent EGFR-TK inhibitors</span></div><div class="casAuthors">Mowafy, Samar; Farag, Nahla A.; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">132-145</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">4-Anilino-6-substituted-quinazolines were designed, synthesized and evaluated for EGFR-TK and tumor growth inhibitory activities.  The target compds. were designed with enamine ester or urea moieties appended at the C-6 of quinazoline as addnl. hydrogen bond acceptor functions.  Most of the synthesized compds. displayed potent EGFR-TK inhibitory activity at 10 μM and the 6-ureido-anilinoquinazoline deriv. (I) showed IC50 value of 0.061 μM.  Moreover, six compds. were tested by National Cancer Institute (NCI), USA for their anti-proliferative activity at 10 μM in full NCI 60 cell panel.  Compd. I was further assayed for five dose molar ranges in full NCI 60 cell panel and exhibited remarkable growth inhibitory activity pattern against Non-Small Cell Lung Cancer EKVX (GI50 = 0.37 μM), NCI-H322M (GI50 = 0.36 μM), Renal Cancer A498 (GI50 = 0.46 μM), TK-10 (GI50 = 0.99 μM) and Breast Cancer MDA-MB-468 (GI50 = 1.096 μM) which are of high EGFR expression.  Docking study was performed for the active compds. into ATP binding site of EGFR-TK which showed similar binding mode to gefitinib and addnl. binding with Cys-773 at the gatekeeper of EGFR-TK enzyme.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJ6Ian6C596LVg90H21EOLACvtfcHk0ljs9LEvQqPe3g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Kjsb3E&md5=e74abcf0e7ff68497f022ac4e1400ab1</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2012.10.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2012.10.017%26sid%3Dliteratum%253Aachs%26aulast%3DMowafy%26aufirst%3DS.%26aulast%3DFarag%26aufirst%3DN.%2BA.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DDesign%252C%2520synthesis%2520and%2520in%2520vitro%2520anti-proliferative%2520activity%2520of%25204%252C6-quinazolinediamines%2520as%2520potent%2520EGFR-TK%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2013%26volume%3D61%26spage%3D132%26epage%3D145%26doi%3D10.1016%2Fj.ejmech.2012.10.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Eustace, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conlon, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McDermott, M. S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Browne, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Holmes, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Espina, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liotta, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Shaughnessy, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Driscoll, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rani, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madden, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ginther, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slamon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walsh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallagher, W. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zagozdzon, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watson, W. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Donovan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crown, J.</span></span> <span> </span><span class="NLM_article-title">Development of acquired resistance to lapatinib may sensitise HER2-positive breast cancer cells to apoptosis induction by obatoclax and TRAIL</span>. <i>BMC Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">965</span>, <span class="refDoi"> DOI: 10.1186/s12885-018-4852-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1186%2Fs12885-018-4852-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30305055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1MXht1yls7vF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=965&author=A.+J.+Eustaceauthor=N.+T.+Conlonauthor=M.+S.+J.+McDermottauthor=B.+C.+Browneauthor=P.+O%E2%80%99Learyauthor=F.+A.+Holmesauthor=V.+Espinaauthor=L.+A.+Liottaauthor=J.+O%E2%80%99Shaughnessyauthor=C.+Gallagherauthor=L.+O%E2%80%99Driscollauthor=S.+Raniauthor=S.+F.+Maddenauthor=N.+A.+O%E2%80%99Brienauthor=C.+Gintherauthor=D.+Slamonauthor=N.+Walshauthor=W.+M.+Gallagherauthor=R.+Zagozdzonauthor=W.+R.+Watsonauthor=N.+O%E2%80%99Donovanauthor=J.+Crown&title=Development+of+acquired+resistance+to+lapatinib+may+sensitise+HER2-positive+breast+cancer+cells+to+apoptosis+induction+by+obatoclax+and+TRAIL&doi=10.1186%2Fs12885-018-4852-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Development of acquired resistance to lapatinib may sensitise HER2-positive breast cancer cells to apoptosis induction by obatoclax and TRAIL</span></div><div class="casAuthors">Eustace, Alex J.; Conlon, Neil T.; Mcdermott, Martina S. J.; Browne, Brigid C.; O'leary, Patrick; Holmes, Frankie A.; Espina, Virginia; Liotta, Lance A.; O'shaughnessy, Joyce; Gallagher, Clair; O'driscoll, Lorraine; Rani, Sweta; Madden, Stephen F.; O'brien, Neil A.; Ginther, Charles; Slamon, Dennis; Walsh, Naomi; Gallagher, William M.; Zagozdzon, Radoslaw; Watson, William R.; O'donovan, Norma; Crown, John</div><div class="citationInfo"><span class="NLM_cas:title">BMC Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">965</span>CODEN:
                <span class="NLM_cas:coden">BCMACL</span>;
        ISSN:<span class="NLM_cas:issn">1471-2407</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Lapatinib has clin. efficacy in the treatment of trastuzumab-refractory HER2-pos. breast cancer.  However, a significant proportion of patients develop progressive disease due to acquired resistance to the drug.  Induction of apoptotic cell death is a key mechanism of action of lapatinib in HER2-pos. breast cancer cells.  Methods: We examd. alterations in regulation of the intrinsic and extrinsic apoptosis pathways in cell line models of acquired lapatinib resistance both in vitro and in patient samples from the NCT01485926 clin. trial, and investigated potential strategies to exploit alterations in apoptosis signalling to overcome lapatinib resistance in HER2-pos. breast cancer.  Results: In this study, we examd. two cell lines models of acquired lapatinib resistance (SKBR3-L and HCC1954-L) and showed that lapatinib does not induce apoptosis in these cells.  We identified alterations in members of the BCL-2 family of proteins, in particular MCL-1 and BAX, which may play a role in resistance to lapatinib.  We tested the therapeutic inhibitor obatoclax, which targets MCL-1.  Both SKBR3-L and HCC1954-L cells showed greater sensitivity to obatoclax-induced apoptosis than parental cells.  Interestingly, we also found that the development of acquired resistance to lapatinib resulted in acquired sensitivity to TRAIL in SKBR3-L cells.  Sensitivity to TRAIL in the SKBR3-L cells was assocd. with reduced phosphorylation of AKT, increased expression of FOXO3a and decreased expression of c- FLIP.  In SKBR3-L cells, TRAIL treatment caused activation of caspase 8, caspase 9 and caspase 3/7.  In a second resistant model, HCC1954-L cells, p-AKT levels were not decreased and these cells did not show enhanced sensitivity to TRAIL.  Furthermore, combining obatoclax with TRAIL improved response in SKBR3-L cells but not in HCC1954-L cells.  Conclusions: Our findings highlight the possibility of targeting altered apoptotic signalling to overcome acquired lapatinib resistance, and identify potential novel treatment strategies, with potential biomarkers, for HER2-pos. breast cancer that is resistant to HER2 targeted therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp12PNOAxrnBbVg90H21EOLACvtfcHk0liNtdLgQ81OzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXht1yls7vF&md5=12a66b8484bb13a8db7a58a5b043170f</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1186%2Fs12885-018-4852-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12885-018-4852-1%26sid%3Dliteratum%253Aachs%26aulast%3DEustace%26aufirst%3DA.%2BJ.%26aulast%3DConlon%26aufirst%3DN.%2BT.%26aulast%3DMcDermott%26aufirst%3DM.%2BS.%2BJ.%26aulast%3DBrowne%26aufirst%3DB.%2BC.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DP.%26aulast%3DHolmes%26aufirst%3DF.%2BA.%26aulast%3DEspina%26aufirst%3DV.%26aulast%3DLiotta%26aufirst%3DL.%2BA.%26aulast%3DO%25E2%2580%2599Shaughnessy%26aufirst%3DJ.%26aulast%3DGallagher%26aufirst%3DC.%26aulast%3DO%25E2%2580%2599Driscoll%26aufirst%3DL.%26aulast%3DRani%26aufirst%3DS.%26aulast%3DMadden%26aufirst%3DS.%2BF.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DN.%2BA.%26aulast%3DGinther%26aufirst%3DC.%26aulast%3DSlamon%26aufirst%3DD.%26aulast%3DWalsh%26aufirst%3DN.%26aulast%3DGallagher%26aufirst%3DW.%2BM.%26aulast%3DZagozdzon%26aufirst%3DR.%26aulast%3DWatson%26aufirst%3DW.%2BR.%26aulast%3DO%25E2%2580%2599Donovan%26aufirst%3DN.%26aulast%3DCrown%26aufirst%3DJ.%26atitle%3DDevelopment%2520of%2520acquired%2520resistance%2520to%2520lapatinib%2520may%2520sensitise%2520HER2-positive%2520breast%2520cancer%2520cells%2520to%2520apoptosis%2520induction%2520by%2520obatoclax%2520and%2520TRAIL%26jtitle%3DBMC%2520Cancer%26date%3D2018%26volume%3D18%26spage%3D965%26doi%3D10.1186%2Fs12885-018-4852-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strom, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gustafsson, J.-Å.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guan, X.</span></span> <span> </span><span class="NLM_article-title">Lapatinib induces p27(Kip1)-dependent G(1) arrest through both transcriptional and post-translational mechanisms</span>. <i>Cell Cycle</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">2665</span>– <span class="NLM_lpage">2674</span>, <span class="refDoi"> DOI: 10.4161/cc.25728</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.4161%2Fcc.25728" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23907131" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1Sgu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=2665-2674&author=L.+Tangauthor=Y.+Wangauthor=A.+Stromauthor=J.-%C3%85.+Gustafssonauthor=X.+Guan&title=Lapatinib+induces+p27%28Kip1%29-dependent+G%281%29+arrest+through+both+transcriptional+and+post-translational+mechanisms&doi=10.4161%2Fcc.25728"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Lapatinib induces p27Kip1-dependent G1 arrest through both transcriptional and post-translational mechanisms</span></div><div class="casAuthors">Tang, Lin; Wang, Yucai; Strom, Anders; Gustafsson, Jan-Ake; Guan, Xiaoxiang</div><div class="citationInfo"><span class="NLM_cas:title">Cell Cycle</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">2665-2674</span>CODEN:
                <span class="NLM_cas:coden">CCEYAS</span>;
        ISSN:<span class="NLM_cas:issn">1538-4101</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Lapatinib, a dual EGFR/HER2 tyrosine kinase inhibitor, has been shown to have potent antitumor effects against human breast cancer.  Recent studies have shown that lapatinib upregulates p27Kip1 (here after referred to as p27) expression and induces G1 cell cycle arrest in various types of cancer cells.  However, the regulation of p27 in lapatinib-induced cell cycle arrest is not well studied.  Here we demonstrate that lapatinib-induced cell growth inhibition and G1 cell cycle arrest in HER2-overexpressing human breast cancer cells were dependent on p27.  We also show that lapatinib-induced upregulation of p27 expression was mediated through both transcriptional and post-translational mechanisms.  On the one hand, lapatinib treatment led to increased FOXO3a expression and enhanced p27 transcription.  On the other hand, lapatinib treatment resulted in increased DYRK1B expression, which correlated with increased p27 phosphorylation at Ser10 and decreased p27 degrdn.  Interestingly, we found that ERβ1 but not ERβ2 expression also upregulated p27 and enhanced lapatinib-induced cell proliferation inhibition and G1 cell cycle arrest in HER2-overexpressing breast cancer cells.  Taken together, our results suggest that lapatinib induces p27 expression via both transcriptional and post-translational upregulations, leading to cell cycle arrest and cell proliferation inhibition, and that its effect on breast cancer cells may be modified by ER expression status.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqFpeD5i8Xl7Vg90H21EOLACvtfcHk0lgutleKXZkmig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1Sgu7w%253D&md5=938c4430a2cbe7b05f874a7cd49c3e63</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.4161%2Fcc.25728&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.25728%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DStrom%26aufirst%3DA.%26aulast%3DGustafsson%26aufirst%3DJ.-%25C3%2585.%26aulast%3DGuan%26aufirst%3DX.%26atitle%3DLapatinib%2520induces%2520p27%2528Kip1%2529-dependent%2520G%25281%2529%2520arrest%2520through%2520both%2520transcriptional%2520and%2520post-translational%2520mechanisms%26jtitle%3DCell%2520Cycle%26date%3D2013%26volume%3D12%26spage%3D2665%26epage%3D2674%26doi%3D10.4161%2Fcc.25728" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanahan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinberg, R. A.</span></span> <span> </span><span class="NLM_article-title">Hallmarks of cancer: the next generation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">646</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.cell.2011.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=21376230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=2011&pages=646-674&author=D.+Hanahanauthor=R.+A.+Weinberg&title=Hallmarks+of+cancer%3A+the+next+generation&doi=10.1016%2Fj.cell.2011.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Hallmarks of cancer: the next generation</span></div><div class="casAuthors">Hanahan, Douglas; Weinberg, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">144</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">646-674</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The hallmarks of cancer comprise six biol. capabilities acquired during the multistep development of human tumors.  The hallmarks constitute an organizing principle for rationalizing the complexities of neoplastic disease.  They include sustaining proliferative signaling, evading growth suppressors, resisting cell death, enabling replicative immortality, inducing angiogenesis, and activating invasion and metastasis.  Underlying these hallmarks are genome instability, which generates the genetic diversity that expedites their acquisition, and inflammation, which fosters multiple hallmark functions.  Conceptual progress in the last decade has added two emerging hallmarks of potential generality to this list-reprogramming of energy metab. and evading immune destruction.  In addn. to cancer cells, tumors exhibit another dimension of complexity: they contain a repertoire of recruited, ostensibly normal cells that contribute to the acquisition of hallmark traits by creating the "tumor microenvironment.".  Recognition of the widespread applicability of these concepts will increasingly affect the development of new means to treat human cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTcpqXALcb6LVg90H21EOLACvtfcHk0lh8AxKA1-iqDw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXjsFeqtrk%253D&md5=b36f160c417a712e83cb83c577f0018e</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DHanahan%26aufirst%3DD.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26atitle%3DHallmarks%2520of%2520cancer%253A%2520the%2520next%2520generation%26jtitle%3DCell%26date%3D2011%26volume%3D144%26spage%3D646%26epage%3D674%26doi%3D10.1016%2Fj.cell.2011.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hao, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, W.</span></span> <span> </span><span class="NLM_article-title">Promotion of apoptosis does not necessarily mean inhibition of remodeling</span>. <i>Hypertension</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">e3</span>, <span class="refDoi"> DOI: 10.1161/hypertensionaha.112.195784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1161%2Fhypertensionaha.112.195784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=22689742" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVerur3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2012&pages=e3&author=Q.+Shiauthor=L.+Haoauthor=J.+Peiauthor=W.+Yin&title=Promotion+of+apoptosis+does+not+necessarily+mean+inhibition+of+remodeling&doi=10.1161%2Fhypertensionaha.112.195784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Promotion of apoptosis does not necessarily mean inhibition of remodeling. Comments</span></div><div class="casAuthors">Shi, Quanxing; Hao, Lu; Pei, Jianming; Yin, Wen</div><div class="citationInfo"><span class="NLM_cas:title">Hypertension</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e3</span>CODEN:
                <span class="NLM_cas:coden">HPRTDN</span>;
        ISSN:<span class="NLM_cas:issn">0194-911X</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">A polemic in response to Guo et al. [Hypertension. 2012;59:1006-1013] is presented.  Guo et al. concluded that miR-328 is downregulated in pulmonary arteries from hypoxic rats and patients with pulmonary hypertension, affecting hypoxic pulmonary vasoconstriction and pulmonary vascular remodeling by targeting Cav 1.2 and insulin-like growth factor 1 receptor, leading to hypoxic pulmonary hypertension.  However, Shi et al. argue that, from the data presented, the Guo et al. claimed only that miR-328 induced pulmonary artery smooth muscle cell (PASMC) apoptosis through repression of insulin-like growth factor 1 receptor.  Shi et al. reasoned out that promoting PASMC apoptosis does not necessarily mean inhibiting pulmonary vascular remodeling and that the effects of miR-328 on pulmonary arteries remodeling and PASMC apoptosis may be 2 accompanying phenomena.  Shi et al. also raised an issue that MTT assay is not appropriate to analyze apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo--jmoKHwG47Vg90H21EOLACvtfcHk0lg0JyiMNUITDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVerur3E&md5=ab80d47cb67b837fc83e1b2a342df7a0</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1161%2Fhypertensionaha.112.195784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252Fhypertensionaha.112.195784%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DQ.%26aulast%3DHao%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DW.%26atitle%3DPromotion%2520of%2520apoptosis%2520does%2520not%2520necessarily%2520mean%2520inhibition%2520of%2520remodeling%26jtitle%3DHypertension%26date%3D2012%26volume%3D60%26spage%3De3%26doi%3D10.1161%2Fhypertensionaha.112.195784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Press, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenz, H.-J.</span></span> <span> </span><span class="NLM_article-title">EGFR, HER2 and VEGF pathways: validated targets for cancer treatment</span>. <i>Drugs</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">2045</span>– <span class="NLM_lpage">2075</span>, <span class="refDoi"> DOI: 10.2165/00003495-200767140-00006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2165%2F00003495-200767140-00006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=17883287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtleqsrrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2007&pages=2045-2075&author=M.+F.+Pressauthor=H.-J.+Lenz&title=EGFR%2C+HER2+and+VEGF+pathways%3A+validated+targets+for+cancer+treatment&doi=10.2165%2F00003495-200767140-00006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">EGFR, HER2 and VEGF pathways: validated targets for cancer treatment</span></div><div class="casAuthors">Press, Michael F.; Lenz, Heinz-Josef</div><div class="citationInfo"><span class="NLM_cas:title">Drugs</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">2045-2075</span>CODEN:
                <span class="NLM_cas:coden">DRUGAY</span>;
        ISSN:<span class="NLM_cas:issn">0012-6667</span>.
    
            (<span class="NLM_cas:orgname">Wolters Kluwer Health</span>)
        </div><div class="casAbstract">A review.  Targeted therapies are rationally designed to interfere with specific mol. events that are important in tumor growth, progression or survival.  Several targeted therapies with anti-tumor activity in human cancer cell lines and xenograft models have now been shown to produce objective responses, delay disease progression and, in some cases, improve survival of patients with advanced malignancies.  These targeted therapies include cetuximab, an anti-epidermal growth factor receptor (EGFR) monoclonal antibody; gefitinib and erlotinib, EGFR-specific tyrosine kinase inhibitors; trastuzumab, an anti-human EGFR type 2 (HER2)-related monoclonal antibody; lapatinib, a dual inhibitor of both EGFR- and HER2-assocd. tyrosine kinases; and bevacizumab, an anti-vascular endothelial growth factor (VEGF) monoclonal antibody.  On the basis of preclin. and clin. evidence, EGFR, HER2 and VEGF represent validated targets for cancer therapy and remain the subject of intensive investigation.  Both EGFR and HER2 are targets found on cancer cells, whereas VEGF is a target that acts in the tumor microenvironment.  Clin. studies are focusing on how to best incorporate targeted therapy into current treatment regimens and other studies are exploring whether different strategies for inhibiting these targets will offer greater benefit.  It is clear that optimal use of targeted therapy will depend on understanding how these drugs work mechanistically, and recognising that their activities may differ across patient populations, tumor types and disease stages, as well as when and how they are used in cancer treatment.  The results achieved with targeted therapies to date are promising, although they illustrate the need for addnl. preclin. and clin. study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruB8rVI305xrVg90H21EOLACvtfcHk0lhYaFpMD_LYGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtleqsrrF&md5=d4e6b3997f2b0a252753cdc6a5137a15</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.2165%2F00003495-200767140-00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2165%252F00003495-200767140-00006%26sid%3Dliteratum%253Aachs%26aulast%3DPress%26aufirst%3DM.%2BF.%26aulast%3DLenz%26aufirst%3DH.-J.%26atitle%3DEGFR%252C%2520HER2%2520and%2520VEGF%2520pathways%253A%2520validated%2520targets%2520for%2520cancer%2520treatment%26jtitle%3DDrugs%26date%3D2007%26volume%3D67%26spage%3D2045%26epage%3D2075%26doi%3D10.2165%2F00003495-200767140-00006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lung, P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sholl, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gad, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bustamante, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, H. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deep, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abd Elmageed, Z. Y.</span></span> <span> </span><span class="NLM_article-title">Dysregulated gene expression predicts tumor aggressiveness in African-American prostate cancer patients</span>. <i>Sci. Rep.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">16335</span>, <span class="refDoi"> DOI: 10.1038/s41598-018-34637-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1038%2Fs41598-018-34637-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30397274" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BB3cvmvFWruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2018&pages=16335&author=H.+E.+A.+Aliauthor=P.-Y.+Lungauthor=A.+B.+Shollauthor=S.+A.+Gadauthor=J.+J.+Bustamanteauthor=H.+I.+Aliauthor=J.+S.+Rhimauthor=G.+Deepauthor=J.+Zhangauthor=Z.+Y.+Abd+Elmageed&title=Dysregulated+gene+expression+predicts+tumor+aggressiveness+in+African-American+prostate+cancer+patients&doi=10.1038%2Fs41598-018-34637-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Dysregulated gene expression predicts tumor aggressiveness in African-American prostate cancer patients</span></div><div class="casAuthors">Ali Hamdy E A; Gad Shaimaa A; Bustamante Juan J; Ali Hamed I; Abd Elmageed Zakaria Y; Ali Hamdy E A; Lung Pei-Yau; Zhang Jinfeng; Sholl Andrew B; Rhim Johng S; Deep Gagan</div><div class="citationInfo"><span class="NLM_cas:title">Scientific reports</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">16335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Molecular mechanisms underlying the health disparity of prostate cancer (PCa) have not been fully determined.  In this study, we applied bioinformatic approach to identify and validate dysregulated genes associated with tumor aggressiveness in African American (AA) compared to Caucasian American (CA) men with PCa.  We retrieved and analyzed microarray data from 619 PCa patients, 412 AA and 207 CA, and we validated these genes in tumor tissues and cell lines by Real-Time PCR, Western blot, immunocytochemistry (ICC) and immunohistochemistry (IHC) analyses.  We identified 362 differentially expressed genes in AA men and involved in regulating signaling pathways associated with tumor aggressiveness.  In PCa tissues and cells, NKX3.1, APPL2, TPD52, LTC4S, ALDH1A3 and AMD1 transcripts were significantly upregulated (p < 0.05) compared to normal cells.  IHC confirmed the overexpression of TPD52 (p = 0.0098) and LTC4S (p < 0.0005) in AA compared to CA men.  ICC and Western blot analyses additionally corroborated this observation in PCa cells.  These findings suggest that dysregulation of transcripts in PCa may drive the disparity of PCa outcomes and provide new insights into development of new therapeutic agents against aggressive tumors.  More studies are warranted to investigate the clinical significance of these dysregulated genes in promoting the oncogenic pathways in AA men.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRCHmWAdFRpT1w2ofyr07jXfW6udTcc2ebgYKB3TqnLXLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cvmvFWruw%253D%253D&md5=663c820ec472b059e2b707ba96465c3b</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fs41598-018-34637-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41598-018-34637-8%26sid%3Dliteratum%253Aachs%26aulast%3DAli%26aufirst%3DH.%2BE.%2BA.%26aulast%3DLung%26aufirst%3DP.-Y.%26aulast%3DSholl%26aufirst%3DA.%2BB.%26aulast%3DGad%26aufirst%3DS.%2BA.%26aulast%3DBustamante%26aufirst%3DJ.%2BJ.%26aulast%3DAli%26aufirst%3DH.%2BI.%26aulast%3DRhim%26aufirst%3DJ.%2BS.%26aulast%3DDeep%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAbd%2BElmageed%26aufirst%3DZ.%2BY.%26atitle%3DDysregulated%2520gene%2520expression%2520predicts%2520tumor%2520aggressiveness%2520in%2520African-American%2520prostate%2520cancer%2520patients%26jtitle%3DSci.%2520Rep.%26date%3D2018%26volume%3D8%26spage%3D16335%26doi%3D10.1038%2Fs41598-018-34637-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Enot, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vacchelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquelot, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zitvogel, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kroemer, G.</span></span> <span> </span><span class="NLM_article-title">TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves</span>. <i>Oncoimmunology</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">e1462431</span>– <span class="NLM_lpage">e1462433</span>, <span class="refDoi"> DOI: 10.1080/2162402x.2018.1462431</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1080%2F2162402x.2018.1462431" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=30228932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVGjsrfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=e1462431-e1462433&author=D.+P.+Enotauthor=E.+Vacchelliauthor=N.+Jacquelotauthor=L.+Zitvogelauthor=G.+Kroemer&title=TumGrowth%3A+An+open-access+web+tool+for+the+statistical+analysis+of+tumor+growth+curves&doi=10.1080%2F2162402x.2018.1462431"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">TumGrowth: An open-access web tool for the statistical analysis of tumor growth curves</span></div><div class="casAuthors">Enot, David P.; Vacchelli, Erika; Jacquelot, Nicolas; Zitvogel, Laurence; Kroemer, Guido</div><div class="citationInfo"><span class="NLM_cas:title">OncoImmunology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1462431/1-e1462431/3</span>CODEN:
                <span class="NLM_cas:coden">ONCOGX</span>;
        ISSN:<span class="NLM_cas:issn">2162-402X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis, Inc.</span>)
        </div><div class="casAbstract">The anal. of tumor growth curves is std. practice in exptl. oncol. including tumor immunol.  In exptl. oncol., cancer cells are inoculated into rodents (mostly mice) and their growth is monitored by measuring tumor diam., surface or vol. over time as a function of distinct treatments.  Then, different groups of tumors/treatments are compared among each other for their evolution and possible responses to treatment.  The R package TumGrowth has been created as a software tool allowing to carry out a series of statistical comparisons across or between groups of tumor growth curves obtained in a std. lab., for experimenters with limited knowledge in statistics.  TumGrowth is freely available online at https://kroemerlab.shinyapps.io/TumGrowth/ and can be downloaded into any computer.  It offers an exhaustive panoply of tools to visualize and analyze complex data sets including longitudinal, cross-sectional and time-to-endpoint measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7NNJlWdRsMbVg90H21EOLACvtfcHk0lhow8b0lzAkHw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVGjsrfI&md5=82153bdf269c5c1d43e8e896dcf98d9e</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1080%2F2162402x.2018.1462431&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F2162402x.2018.1462431%26sid%3Dliteratum%253Aachs%26aulast%3DEnot%26aufirst%3DD.%2BP.%26aulast%3DVacchelli%26aufirst%3DE.%26aulast%3DJacquelot%26aufirst%3DN.%26aulast%3DZitvogel%26aufirst%3DL.%26aulast%3DKroemer%26aufirst%3DG.%26atitle%3DTumGrowth%253A%2520An%2520open-access%2520web%2520tool%2520for%2520the%2520statistical%2520analysis%2520of%2520tumor%2520growth%2520curves%26jtitle%3DOncoimmunology%26date%3D2018%26volume%3D7%26spage%3De1462431%26epage%3De1462433%26doi%3D10.1080%2F2162402x.2018.1462431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ducray, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barlaam, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hickinson, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kettle, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ogilvie, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trigwell, C. B.</span></span> <span> </span><span class="NLM_article-title">Novel 3-alkoxy-1H-pyrazolo[3,4-d]pyrimidines as EGFR and erbB2 receptor tyrosine kinase inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">959</span>– <span class="NLM_lpage">962</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2007.12.035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2007.12.035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=18182285" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFWktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2008&pages=959-962&author=R.+Ducrayauthor=P.+Ballardauthor=B.+C.+Barlaamauthor=M.+D.+Hickinsonauthor=J.+G.+Kettleauthor=D.+J.+Ogilvieauthor=C.+B.+Trigwell&title=Novel+3-alkoxy-1H-pyrazolo%5B3%2C4-d%5Dpyrimidines+as+EGFR+and+erbB2+receptor+tyrosine+kinase+inhibitors&doi=10.1016%2Fj.bmcl.2007.12.035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3-alkoxy-1H-pyrazolo[3,4-d]pyrimidines as EGFR and erbB2 receptor tyrosine kinase inhibitors</span></div><div class="casAuthors">Ducray, Richard; Ballard, Peter; Barlaam, Bernard C.; Hickinson, Mark D.; Kettle, Jason G.; Ogilvie, Donald J.; Trigwell, Catherine B.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">959-962</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of 4-anilino-1H-pyrazolo[3,4-d]pyrimidines was synthesized and evaluated in vitro for their erbB2 and EGFR kinase inhibition.  Compd. I displaying oral bioavailability in rat and dog illustrates the potential of this series to provide orally active erbB2 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoq7N8uYVaWT7Vg90H21EOLACvtfcHk0lhGFD3f4G8JWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFWktL0%253D&md5=c40865c049e4f650af70e4370fccb1f4</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2007.12.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2007.12.035%26sid%3Dliteratum%253Aachs%26aulast%3DDucray%26aufirst%3DR.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DBarlaam%26aufirst%3DB.%2BC.%26aulast%3DHickinson%26aufirst%3DM.%2BD.%26aulast%3DKettle%26aufirst%3DJ.%2BG.%26aulast%3DOgilvie%26aufirst%3DD.%2BJ.%26aulast%3DTrigwell%26aufirst%3DC.%2BB.%26atitle%3DNovel%25203-alkoxy-1H-pyrazolo%255B3%252C4-d%255Dpyrimidines%2520as%2520EGFR%2520and%2520erbB2%2520receptor%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2008%26volume%3D18%26spage%3D959%26epage%3D962%26doi%3D10.1016%2Fj.bmcl.2007.12.035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Singla, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munshi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banipal, R. P. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, V.</span></span> <span> </span><span class="NLM_article-title">Recent updates on the therapeutic potential of HER2 tyrosine kinase inhibitors for the treatment of breast cancer</span>. <i>Curr. Cancer Drug Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">306</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.2174/1568009617666170623122213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.2174%2F1568009617666170623122213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28669349" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnvFaqurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=306-327&author=H.+Singlaauthor=A.+Munshiauthor=R.+P.+S.+Banipalauthor=V.+Kumar&title=Recent+updates+on+the+therapeutic+potential+of+HER2+tyrosine+kinase+inhibitors+for+the+treatment+of+breast+cancer&doi=10.2174%2F1568009617666170623122213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Updates on the Therapeutic Potential of HER2 Tyrosine Kinase Inhibitors for the Treatment of Breast Cancer</span></div><div class="casAuthors">Singla, Heena; Munshi, Anjana; Singh Banipal, Raja Paramjit; Kumar, Vinod</div><div class="citationInfo"><span class="NLM_cas:title">Current Cancer Drug Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">306-327</span>CODEN:
                <span class="NLM_cas:coden">CCDTB9</span>;
        ISSN:<span class="NLM_cas:issn">1568-0096</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  HER2 pos. breast cancer is characterized by the low survival rate in the metastatic patients.  Development of resistance and disease-relapse are the major problems assocd. with the currently available therapies for HER2 pos. breast cancer.  There are two major targeted therapies for HER2 pos. breast cancer viz. monoclonal antibodies and tyrosine-kinase inhibitors, and both of these therapies have their advantages and limitations.  To address the limitations assocd. with the existing therapies, use of antibodies and TKIs as combination therapy proved to be more effective.  Various chem. modifications can be performed on tyrosine-kinase inhibitors to develop novel ligands with increased selectivity for HER2 kinase.  A no. of tyrosine-kinase inhibitors are in various phases of clin. trials for the treatment of HER2 pos. breast cancer.  In the current review article, recent developments on various HER2 tyrosine-kinase inhibitors have been reported.  Various structurally different scaffolds bind to the HER2 receptor and exhibit potent anti-cancer activities.  The structural and pharmacophoric requirements of the scaffolds are discussed in detail so as to discover effective drug candidates for the treatment of HER2 pos. breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8LTwxEd80KrVg90H21EOLACvtfcHk0lgzGR_3oNOmpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnvFaqurw%253D&md5=93ec4f7943db373d853d0810c2964155</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.2174%2F1568009617666170623122213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568009617666170623122213%26sid%3Dliteratum%253Aachs%26aulast%3DSingla%26aufirst%3DH.%26aulast%3DMunshi%26aufirst%3DA.%26aulast%3DBanipal%26aufirst%3DR.%2BP.%2BS.%26aulast%3DKumar%26aufirst%3DV.%26atitle%3DRecent%2520updates%2520on%2520the%2520therapeutic%2520potential%2520of%2520HER2%2520tyrosine%2520kinase%2520inhibitors%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DCurr.%2520Cancer%2520Drug%2520Targets%26date%3D2018%26volume%3D18%26spage%3D306%26epage%3D327%26doi%3D10.2174%2F1568009617666170623122213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mohamed, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storelli, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sidibe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bararpour, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Desmeules, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausburger, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daali, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, A.</span></span> <span> </span><span class="NLM_article-title">Detection and identification of reactive drug metabolites leading to idiosyncratic toxicity: lapatinib as a case example</span>. <i>J. Drug Metab. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1000242</span>, <span class="refDoi"> DOI: 10.4172/2157-7609.1000242</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.4172%2F2157-7609.1000242" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1000242&author=R.+Mohamedauthor=F.+Storelliauthor=J.+Sidibeauthor=N.+Bararpourauthor=J.+Desmeulesauthor=M.+Ausburgerauthor=Y.+Daaliauthor=A.+Thomas&title=Detection+and+identification+of+reactive+drug+metabolites+leading+to+idiosyncratic+toxicity%3A+lapatinib+as+a+case+example&doi=10.4172%2F2157-7609.1000242"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.4172%2F2157-7609.1000242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4172%252F2157-7609.1000242%26sid%3Dliteratum%253Aachs%26aulast%3DMohamed%26aufirst%3DR.%26aulast%3DStorelli%26aufirst%3DF.%26aulast%3DSidibe%26aufirst%3DJ.%26aulast%3DBararpour%26aufirst%3DN.%26aulast%3DDesmeules%26aufirst%3DJ.%26aulast%3DAusburger%26aufirst%3DM.%26aulast%3DDaali%26aufirst%3DY.%26aulast%3DThomas%26aufirst%3DA.%26atitle%3DDetection%2520and%2520identification%2520of%2520reactive%2520drug%2520metabolites%2520leading%2520to%2520idiosyncratic%2520toxicity%253A%2520lapatinib%2520as%2520a%2520case%2520example%26jtitle%3DJ.%2520Drug%2520Metab.%2520Toxicol.%26date%3D2018%26volume%3D9%26spage%3D1000242%26doi%3D10.4172%2F2157-7609.1000242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">Discovery and development of pyrotinib: A novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor with favorable safety profiles for the treatment of breast cancer</span>. <i>Eur. J. Pharm. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">51</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.ejps.2017.01.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejps.2017.01.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28115222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXis1Wqsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2017&pages=51-61&author=X.+Liauthor=C.+Yangauthor=H.+Wanauthor=G.+Zhangauthor=J.+Fengauthor=L.+Zhangauthor=X.+Chenauthor=D.+Zhongauthor=L.+Louauthor=W.+Taoauthor=L.+Zhang&title=Discovery+and+development+of+pyrotinib%3A+A+novel+irreversible+EGFR%2FHER2+dual+tyrosine+kinase+inhibitor+with+favorable+safety+profiles+for+the+treatment+of+breast+cancer&doi=10.1016%2Fj.ejps.2017.01.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and development of pyrotinib: A novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor with favorable safety profiles for the treatment of breast cancer</span></div><div class="casAuthors">Li, Xin; Yang, Changyong; Wan, Hong; Zhang, Ge; Feng, Jun; Zhang, Lei; Chen, Xiaoyan; Zhong, Dafang; Lou, Liguang; Tao, Weikang; Zhang, Lianshan</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">51-61</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The discovery and development of a novel irreversible EGFR/HER2 dual tyrosine kinase inhibitor SHR1258 (pyrotinib) for the treatment of HER2-postive breast cancer is presented.  The structure-activity relationship of lead series and their pharmacokinetic properties were evaluated to identify the potential candidates for further in vivo efficacy studies and preclin. safety assessments.  Metabolic pathway and drug-drug interaction were also investigated in preclin. settings.  In particular, major metabolites in human and animal species were assessed with regard to potential toxicity or off-target side effects.  Overall, the potent and selective EGFR/HER2 dual inhibitor, pyrotinib, displayed robust anti-tumor effects on HER2-overexpressing xenograft models and sufficiently safety windows in animals as well as favorable pharmacokinetic properties in human, which substantially ensures current clin. development.  Finally, recent advances of pyrotinib in clin. studies are highlighted with very encouraging outcomes in patients with HER2-postive advanced breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqdbJnY0jwOjLVg90H21EOLACvtfcHk0lioUAnR1xxwNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXis1Wqsrw%253D&md5=12f13882a32cfb56959f58deaadadce8</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1016%2Fj.ejps.2017.01.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejps.2017.01.021%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DWan%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DLou%26aufirst%3DL.%26aulast%3DTao%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DDiscovery%2520and%2520development%2520of%2520pyrotinib%253A%2520A%2520novel%2520irreversible%2520EGFR%252FHER2%2520dual%2520tyrosine%2520kinase%2520inhibitor%2520with%2520favorable%2520safety%2520profiles%2520for%2520the%2520treatment%2520of%2520breast%2520cancer%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2017%26volume%3D110%26spage%3D51%26epage%3D61%26doi%3D10.1016%2Fj.ejps.2017.01.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Comparative evaluation of 11 scoring functions for molecular docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">2287</span>– <span class="NLM_lpage">2303</span>, <span class="refDoi"> DOI: 10.1021/jm0203783</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0203783" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BD3sXjsVOgtr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=2287-2303&author=R.+Wangauthor=Y.+Luauthor=S.+Wang&title=Comparative+evaluation+of+11+scoring+functions+for+molecular+docking&doi=10.1021%2Fjm0203783"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">Comparative Evaluation of 11 Scoring Functions for Molecular Docking</span></div><div class="casAuthors">Wang, Renxiao; Lu, Yipin; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2287-2303</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Eleven popular scoring functions have been tested on 100 protein-ligand complexes to evaluate their abilities to reproduce exptl. detd. structures and binding affinities.  They include four scoring functions implemented in the LigFit module in Cerius2 (LigScore, PLP, PMF, and LUDI), four scoring functions implemented in the CScore module in SYBYL (F-Score, G-Score, D-Score, and ChemScore), the scoring function implemented in the AutoDock program, and two stand-alone scoring functions (DrugScore and X-Score).  These scoring functions are not tested in the context of a particular docking program.  Instead, conformational sampling and scoring are sepd. into two consecutive steps.  First, an exhaustive conformational sampling is performed by using the AutoDock program to generate an ensemble of docked conformations for each ligand mol.  This conformational ensemble is required to cover the entire conformational space as much as possible rather than to focus on a few energy min. Then, each scoring function is applied to score this conformational ensemble to see if it can identify the exptl. obsd. conformation from all of the other decoys.  Among all of the scoring functions under test, six of them, i.e., PLP, F-Score, LigScore, DrugScore, LUDI, and X-Score, yield success rates higher than the AutoDock scoring function.  The success rates of these six scoring functions range from 66% to 76% if using root-mean-square deviation ≤2.0 Å as the criterion.  Combining any two or three of these six scoring functions into a consensus scoring scheme further improves the success rate to nearly 80% or even higher.  However, when applied to reproduce the exptl. detd. binding affinities of the 100 protein-ligand complexes, only X-Score, PLP, DrugScore, and G-Score are able to give correlation coeffs. over 0.50.  All of the 11 scoring functions are further inspected by their abilities to construct a descriptive, funnel-shaped energy surface for protein-ligand complexation.  The results indicate that X-Score and DrugScore perform better than the other ones at this aspect.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPH6dgkF_TeLVg90H21EOLACvtfcHk0ljpwIvkkqVvug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXjsVOgtr0%253D&md5=2f6e7425f8d612124a06f33e2e6f0400</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1021%2Fjm0203783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0203783%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DComparative%2520evaluation%2520of%252011%2520scoring%2520functions%2520for%2520molecular%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D2287%26epage%3D2303%26doi%3D10.1021%2Fjm0203783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Milik, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdel-Aziz, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lasheen, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Serya, R. A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minucci, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abouzid, K. A. M.</span></span> <span> </span><span class="NLM_article-title">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">316</span>– <span class="NLM_lpage">336</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.06.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29902719" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=316-336&author=S.+N.+Milikauthor=A.+K.+Abdel-Azizauthor=D.+S.+Lasheenauthor=R.+A.+T.+Seryaauthor=S.+Minucciauthor=K.+A.+M.+Abouzid&title=Surmounting+the+resistance+against+EGFR+inhibitors+through+the+development+of+thieno%5B2%2C3-d%5Dpyrimidine-based+dual+EGFR%2FHER2+inhibitors&doi=10.1016%2Fj.ejmech.2018.06.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Surmounting the resistance against EGFR inhibitors through the development of thieno[2,3-d]pyrimidine-based dual EGFR/HER2 inhibitors</span></div><div class="casAuthors">Milik, Sandra N.; Abdel-Aziz, Amal Kamal; Lasheen, Deena S.; Serya, Rabah A. T.; Minucci, Saverio; Abouzid, Khaled A. M.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">316-336</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">In light of the emergence of resistance against the currently available EGFR inhibitors, our study focuses on tackling this problem through the development of dual EGFR/HER2 inhibitors with improved enzymic affinities.  Guided by the binding mode of the marketed dual EGFR/HER2 inhibitor, Lapatinib, we proposed the design of dual EGFR/HER2 inhibitors based on the 6-phenylthieno[2,3-d]pyrimidine as a core scaffold and hinge binder.  After two cycles of screening aiming to identify the optimum aniline headgroup and solubilizing group, we eventually identified 27b as a dual EGFR/HER2 inhibitor with IC50 values of 91.7 nM and 1.2 μM, resp.  Notably, 27b dramatically reduced the viability of various patient-derived cancer cells preferentially overexpressing EGFR/HER2 (A431, MDA-MBA-361 and SKBr3 with IC50 values of 1.45, 3.5 and 4.83 μM, resp.).  Addnl., 27b efficiently thwarted the proliferation of lapatinib-resistant human non-small lung carcinoma (NCI-H1975) cells, harboring T790 M mutation, with IC50 of 4.2 μM.  Consistently, 27b significantly blocked EGF-induced EGFR activation and inactivated its downstream AKT/mTOR/S6 signalling pathway triggering apoptotic cell death in NCI-H1975 cells.  The present study presents a promising candidate for further design and development of novel EGFR/HER2 inhibitors capable of overcoming EGFR TKIs resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoR7DEfVDfa_rVg90H21EOLACvtfcHk0lhMF8BxdB3LQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFemsbzK&md5=d4c6b29330f43b8ca3f7f3f6e2e85f8c</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.011%26sid%3Dliteratum%253Aachs%26aulast%3DMilik%26aufirst%3DS.%2BN.%26aulast%3DAbdel-Aziz%26aufirst%3DA.%2BK.%26aulast%3DLasheen%26aufirst%3DD.%2BS.%26aulast%3DSerya%26aufirst%3DR.%2BA.%2BT.%26aulast%3DMinucci%26aufirst%3DS.%26aulast%3DAbouzid%26aufirst%3DK.%2BA.%2BM.%26atitle%3DSurmounting%2520the%2520resistance%2520against%2520EGFR%2520inhibitors%2520through%2520the%2520development%2520of%2520thieno%255B2%252C3-d%255Dpyrimidine-based%2520dual%2520EGFR%252FHER2%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D316%26epage%3D336%26doi%3D10.1016%2Fj.ejmech.2018.06.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smaill, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonzales, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spicer, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reed, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Althaus, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Black, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blaser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denny, W. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fakhoury, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harvey, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hook, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCarthy, F. O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schlosser, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ellis, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trachet, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winters, R. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tecle, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bridges, A.</span></span> <span> </span><span class="NLM_article-title">Tyrosine kinase inhibitors. 20. optimization of substituted quinazoline and pyrido[3,4-d]pyrimidine derivatives as orally active, irreversible inhibitors of the epidermal growth factor receptor family</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8103</span>– <span class="NLM_lpage">8124</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00883</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00883" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8103-8124&author=J.+B.+Smaillauthor=A.+J.+Gonzalesauthor=J.+A.+Spicerauthor=H.+Leeauthor=J.+E.+Reedauthor=K.+Sextonauthor=I.+W.+Althausauthor=T.+Zhuauthor=S.+L.+Blackauthor=A.+Blaserauthor=W.+A.+Dennyauthor=P.+A.+Ellisauthor=S.+Fakhouryauthor=P.+J.+Harveyauthor=K.+Hookauthor=F.+O.+J.+McCarthyauthor=B.+D.+Palmerauthor=F.+Rivaultauthor=K.+Schlosserauthor=T.+Ellisauthor=A.+M.+Thompsonauthor=E.+Trachetauthor=R.+T.+Wintersauthor=H.+Tecleauthor=A.+Bridges&title=Tyrosine+kinase+inhibitors.+20.+optimization+of+substituted+quinazoline+and+pyrido%5B3%2C4-d%5Dpyrimidine+derivatives+as+orally+active%2C+irreversible+inhibitors+of+the+epidermal+growth+factor+receptor+family&doi=10.1021%2Facs.jmedchem.6b00883"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Tyrosine Kinase Inhibitors. 20. Optimization of Substituted Quinazoline and Pyrido[3,4-d]pyrimidine Derivatives as Orally Active, Irreversible Inhibitors of the Epidermal Growth Factor Receptor Family</span></div><div class="casAuthors">Smaill, Jeff B.; Gonzales, Andrea J.; Spicer, Julie A.; Lee, Helen; Reed, Jessica E.; Sexton, Karen; Althaus, Irene W.; Zhu, Tong; Black, Shannon L.; Blaser, Adrian; Denny, William A.; Ellis, Paul A.; Fakhoury, Stephen; Harvey, Patricia J.; Hook, Ken; McCarthy, Florence O. J.; Palmer, Brian D.; Rivault, Freddy; Schlosser, Kevin; Ellis, Teresa; Thompson, Andrew M.; Trachet, Erin; Winters, R. Thomas; Tecle, Haile; Bridges, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">8103-8124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-activity relationships for inhibition of erbB1, erbB2 and erbB4 were detd. for a series of quinazoline- and pyrido[3,4-d]pyrimidine-based analogs of the irreversible pan-erbB inhibitor, canertinib.  Cyclic amine bearing crotonamides were detd. to provide rapid inhibition of cellular erbB1 autophosphorylation and good metabolic stability in liver microsome and hepatocyte assays.  The influence of 4-anilino substitution on pan-erbB inhibitory potency was investigated.  Several anilines were identified as providing potent, reversible pan-erbB inhibition.  Optimum 4- and 6-substituents with known 7-substituents provided preferred irreversible inhibitors for pharmacodynamic testing in vivo.  Quinazoline I and pyrido[3,4-d]pyrimidine II were identified as clearly superior to canertinib.  Both compds. possess a piperidinyl crotonamide Michael acceptor and a 3-chloro-4-fluoroaniline, indicating these as optimized 6- and 4-substituents resp.  Pharmacokinetic comparison of compds. I and II across three species selected compd. I as the preferred candidate.  Compd. I (PF-00299804) has been assigned the nomenclature of dacomitinib and is currently under clin. evaluation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0-NpOmVq10bVg90H21EOLACvtfcHk0liMW35_WMIxxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht12gtbvM&md5=442afda669a5c1b7a190cd82b77bee33</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00883%26sid%3Dliteratum%253Aachs%26aulast%3DSmaill%26aufirst%3DJ.%2BB.%26aulast%3DGonzales%26aufirst%3DA.%2BJ.%26aulast%3DSpicer%26aufirst%3DJ.%2BA.%26aulast%3DLee%26aufirst%3DH.%26aulast%3DReed%26aufirst%3DJ.%2BE.%26aulast%3DSexton%26aufirst%3DK.%26aulast%3DAlthaus%26aufirst%3DI.%2BW.%26aulast%3DZhu%26aufirst%3DT.%26aulast%3DBlack%26aufirst%3DS.%2BL.%26aulast%3DBlaser%26aufirst%3DA.%26aulast%3DDenny%26aufirst%3DW.%2BA.%26aulast%3DEllis%26aufirst%3DP.%2BA.%26aulast%3DFakhoury%26aufirst%3DS.%26aulast%3DHarvey%26aufirst%3DP.%2BJ.%26aulast%3DHook%26aufirst%3DK.%26aulast%3DMcCarthy%26aufirst%3DF.%2BO.%2BJ.%26aulast%3DPalmer%26aufirst%3DB.%2BD.%26aulast%3DRivault%26aufirst%3DF.%26aulast%3DSchlosser%26aufirst%3DK.%26aulast%3DEllis%26aufirst%3DT.%26aulast%3DThompson%26aufirst%3DA.%2BM.%26aulast%3DTrachet%26aufirst%3DE.%26aulast%3DWinters%26aufirst%3DR.%2BT.%26aulast%3DTecle%26aufirst%3DH.%26aulast%3DBridges%26aufirst%3DA.%26atitle%3DTyrosine%2520kinase%2520inhibitors.%252020.%2520optimization%2520of%2520substituted%2520quinazoline%2520and%2520pyrido%255B3%252C4-d%255Dpyrimidine%2520derivatives%2520as%2520orally%2520active%252C%2520irreversible%2520inhibitors%2520of%2520the%2520epidermal%2520growth%2520factor%2520receptor%2520family%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8103%26epage%3D8124%26doi%3D10.1021%2Facs.jmedchem.6b00883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elkamhawy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farag, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanath, A. N. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bedair, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leem, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, K.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pae, A. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roh, E. J.</span></span> <span> </span><span class="NLM_article-title">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">5147</span>– <span class="NLM_lpage">5154</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2015.10.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmcl.2015.10.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=26475520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=5147-5154&author=A.+Elkamhawyauthor=A.+K.+Faragauthor=A.+N.+I.+Viswanathauthor=T.+M.+Bedairauthor=D.+G.+Leemauthor=K.-T.+Leeauthor=A.+N.+Paeauthor=E.+J.+Roh&title=Targeting+EGFR%2FHER2+tyrosine+kinases+with+a+new+potent+series+of+6-substituted+4-anilinoquinazoline+hybrids%3A+Design%2C+synthesis%2C+kinase+assay%2C+cell-based+assay%2C+and+molecular+docking&doi=10.1016%2Fj.bmcl.2015.10.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EGFR/HER2 tyrosine kinases with a new potent series of 6-substituted 4-anilinoquinazoline hybrids: Design, synthesis, kinase assay, cell-based assay, and molecular docking</span></div><div class="casAuthors">Elkamhawy, Ahmed; Farag, Ahmed Karam; Viswanath, Ambily Nath Indu; Bedair, Tarek M.; Leem, Dong Gyu; Lee, Kyung-Tae; Pae, Ae Nim; Roh, Eun Joo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">5147-5154</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Coexpression of EGFR and HER2 has been found in many tumors such as breast, ovarian, colon and prostate cancers, with poor prognosis of the patients.  Herein, our team has designed and synthesized new eighteen compds. with 6-substituted 4-anilinoquinazoline core to selectively inhibit EGFR/HER2 tyrosine kinases.  Twelve compds. showed nanomolar range of IC50 values on EGFR and/or HER2 kinases.  Accordingly, a detailed structure activity relationship (SAR) was established.  A mol. docking study demonstrated the favorable binding modes of I (X = Me, NHEt) at the ATP active site of both kinases.  A kinase selectivity profile performed for compd. 8d showed great selectivity for EGFR and HER2.  In addn., compd. I (X = Me, NHEt) and II exerted selective promising cytotoxic activity over BT-474 cell line with IC50 values of 2.70, 1.82 and 1.95 μM, resp.  From these results, we report analogs I (X = Me, NHEt) and II as promising candidates for the discovery of well-balanced compds. in terms of the kinase inhibitory potency and antiproliferative activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq_Ln0rUhX7bVg90H21EOLACvtfcHk0lidp7uG_DSWXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Kru7zP&md5=0d7c9e4896863934f0190537c10c42ae</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.10.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.10.003%26sid%3Dliteratum%253Aachs%26aulast%3DElkamhawy%26aufirst%3DA.%26aulast%3DFarag%26aufirst%3DA.%2BK.%26aulast%3DViswanath%26aufirst%3DA.%2BN.%2BI.%26aulast%3DBedair%26aufirst%3DT.%2BM.%26aulast%3DLeem%26aufirst%3DD.%2BG.%26aulast%3DLee%26aufirst%3DK.-T.%26aulast%3DPae%26aufirst%3DA.%2BN.%26aulast%3DRoh%26aufirst%3DE.%2BJ.%26atitle%3DTargeting%2520EGFR%252FHER2%2520tyrosine%2520kinases%2520with%2520a%2520new%2520potent%2520series%2520of%25206-substituted%25204-anilinoquinazoline%2520hybrids%253A%2520Design%252C%2520synthesis%252C%2520kinase%2520assay%252C%2520cell-based%2520assay%252C%2520and%2520molecular%2520docking%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D5147%26epage%3D5154%26doi%3D10.1016%2Fj.bmcl.2015.10.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banno, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakita, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oorui, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohta, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakayama, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miki, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiguchi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Habuka, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sogabe, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aertgeerts, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamiyama, K.</span></span> <span> </span><span class="NLM_article-title">Design and synthesis of novel human epidermal growth factor receptor 2 (HER2)/epidermal growth factor receptor (EGFR) dual inhibitors bearing a pyrrolo[3,2-d]pyrimidine scaffold</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">8030</span>– <span class="NLM_lpage">8050</span>, <span class="refDoi"> DOI: 10.1021/jm2008634</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm2008634" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8030-8050&author=T.+Ishikawaauthor=M.+Setoauthor=H.+Bannoauthor=Y.+Kawakitaauthor=M.+Ooruiauthor=T.+Taniguchiauthor=Y.+Ohtaauthor=T.+Tamuraauthor=A.+Nakayamaauthor=H.+Mikiauthor=H.+Kamiguchiauthor=T.+Tanakaauthor=N.+Habukaauthor=S.+Sogabeauthor=J.+Yanoauthor=K.+Aertgeertsauthor=K.+Kamiyama&title=Design+and+synthesis+of+novel+human+epidermal+growth+factor+receptor+2+%28HER2%29%2Fepidermal+growth+factor+receptor+%28EGFR%29+dual+inhibitors+bearing+a+pyrrolo%5B3%2C2-d%5Dpyrimidine+scaffold&doi=10.1021%2Fjm2008634"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of Novel Human Epidermal Growth Factor Receptor 2 (HER2)/Epidermal Growth Factor Receptor (EGFR) Dual Inhibitors Bearing a Pyrrolo[3,2-d]pyrimidine Scaffold</span></div><div class="casAuthors">Ishikawa, Tomoyasu; Seto, Masaki; Banno, Hiroshi; Kawakita, Youichi; Oorui, Mami; Taniguchi, Takahiko; Ohta, Yoshikazu; Tamura, Toshiya; Nakayama, Akiko; Miki, Hiroshi; Kamiguchi, Hidenori; Tanaka, Toshimasa; Habuka, Noriyuki; Sogabe, Satoshi; Yano, Jason; Aertgeerts, Kathleen; Kamiyama, Keiji</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8030-8050</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibitors of human epidermal growth factor receptor 2 (HER2) and epidermal growth factor receptor (EGFR) have been investigated for breast, lung, gastric, prostate, and other cancers; one, lapatinib, is currently approved for breast cancer.  To develop novel HER2/EGFR dual kinase inhibitors, we designed and synthesized pyrrolo[3,2-d]pyrimidine derivs., e.g. I, capable of fitting into the receptors' ATP binding site.  Among the prepd. compds., I showed potent HER2 and EGFR (HER1) inhibitory activities as well as tumor growth inhibitory activity.  The X-ray cocrystal structures of I with both HER2 and EGFR demonstrated that I interacts with the expected residues in their resp. ATP pockets.  Furthermore, reflecting its good oral bioavailability, I exhibited potent in vivo efficacy in HER2-overexpressing tumor xenograft models.  On the basis of these findings, we report I (TAK-285) as a promising candidate for clin. development as a novel HER2/EGFR dual kinase inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9QtqAj6idYrVg90H21EOLACvtfcHk0lidp7uG_DSWXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlajur7M&md5=ce030ac133a1c0ff7cb78b269ff85987</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1021%2Fjm2008634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm2008634%26sid%3Dliteratum%253Aachs%26aulast%3DIshikawa%26aufirst%3DT.%26aulast%3DSeto%26aufirst%3DM.%26aulast%3DBanno%26aufirst%3DH.%26aulast%3DKawakita%26aufirst%3DY.%26aulast%3DOorui%26aufirst%3DM.%26aulast%3DTaniguchi%26aufirst%3DT.%26aulast%3DOhta%26aufirst%3DY.%26aulast%3DTamura%26aufirst%3DT.%26aulast%3DNakayama%26aufirst%3DA.%26aulast%3DMiki%26aufirst%3DH.%26aulast%3DKamiguchi%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DT.%26aulast%3DHabuka%26aufirst%3DN.%26aulast%3DSogabe%26aufirst%3DS.%26aulast%3DYano%26aufirst%3DJ.%26aulast%3DAertgeerts%26aufirst%3DK.%26aulast%3DKamiyama%26aufirst%3DK.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520novel%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520%2528HER2%2529%252Fepidermal%2520growth%2520factor%2520receptor%2520%2528EGFR%2529%2520dual%2520inhibitors%2520bearing%2520a%2520pyrrolo%255B3%252C2-d%255Dpyrimidine%2520scaffold%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8030%26epage%3D8050%26doi%3D10.1021%2Fjm2008634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.-N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, G.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.-Q.</span></span> <span> </span><span class="NLM_article-title">Discovery of novel 4-anilinoquinazoline derivatives as potent inhibitors of epidermal growth factor receptor with antitumor activity</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">6084</span>– <span class="NLM_lpage">6091</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2013.06.070</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.bmc.2013.06.070" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23962660" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12itb%252FK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=6084-6091&author=Y.-Y.+Xuauthor=S.-N.+Liauthor=G.-J.+Yuauthor=Q.-H.+Huauthor=H.-Q.+Li&title=Discovery+of+novel+4-anilinoquinazoline+derivatives+as+potent+inhibitors+of+epidermal+growth+factor+receptor+with+antitumor+activity&doi=10.1016%2Fj.bmc.2013.06.070"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of novel 4-anilinoquinazoline derivatives as potent inhibitors of epidermal growth factor receptor with antitumor activity</span></div><div class="casAuthors">Xu, Yun-Yun; Li, Si-Ning; Yu, Gao-Jian; Hu, Qing-Hua; Li, Huan-Qiu</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">6084-6091</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Two new series of new compds. contg. a 6-amino-substituted group or 6-acrylamide-substituted group linked to a 4-anilinoquinazoline nucleus have been discovered as potential EGFR inhibitors.  These compds. proved efficient effects on antiproliferative activity and EGFR-TK inhibitory activity.  Esp., N6-((5-bromothiophen-2-yl)methyl)-N4-(3-chlorophenyl)quinazoline-4,6-diamine (5e), showed the most potent inhibitory activity (IC50 = 3.11 μM for Hep G2, IC50 = 0.82 μM for A549).  The EGFR mol. docking model suggested that the new compd. is nicely bound to the region of EGFR, and cell morphol. by Hoechst stain expt. suggested that these compds. efficiently induced apoptosis of A549 cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3fZ-t5f7RLVg90H21EOLACvtfcHk0liihOPmxs-M_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12itb%252FK&md5=e97214f67937ab2757a68f3c6474e98b</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2013.06.070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2013.06.070%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.-Y.%26aulast%3DLi%26aufirst%3DS.-N.%26aulast%3DYu%26aufirst%3DG.-J.%26aulast%3DHu%26aufirst%3DQ.-H.%26aulast%3DLi%26aufirst%3DH.-Q.%26atitle%3DDiscovery%2520of%2520novel%25204-anilinoquinazoline%2520derivatives%2520as%2520potent%2520inhibitors%2520of%2520epidermal%2520growth%2520factor%2520receptor%2520with%2520antitumor%2520activity%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2013%26volume%3D21%26spage%3D6084%26epage%3D6091%26doi%3D10.1016%2Fj.bmc.2013.06.070" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, R.</span></span> <span> </span><span class="NLM_article-title">Design, synthesis and biological evaluation of quinazoline-phosphoramidate mustard conjugates as anticancer drugs</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">442</span>– <span class="NLM_lpage">458</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2016.12.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2016.12.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=28092860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1ahsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=442-458&author=S.+Linauthor=Y.+Liauthor=Y.+Zhengauthor=L.+Luoauthor=Q.+Sunauthor=Z.+Geauthor=T.+Chengauthor=R.+Li&title=Design%2C+synthesis+and+biological+evaluation+of+quinazoline-phosphoramidate+mustard+conjugates+as+anticancer+drugs&doi=10.1016%2Fj.ejmech.2016.12.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and biological evaluation of quinazoline-phosphoramidate mustard conjugates as anticancer drugs</span></div><div class="casAuthors">Lin, Songwen; Li, Yingbo; Zheng, Yufen; Luo, Laichun; Sun, Qi; Ge, Zemei; Cheng, Tieming; Li, Runtao</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">442-458</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Novel compds. with phosphoramide mustard functionality incorporated into the quinazoline scaffold of EGFR/HER2 inhibitors were designed and synthesized as multi-target-directed ligands against tumor cells.  In vitro assays showed that tumor cell lines with high HER2 level were more sensitive to the compds. than tumor cells with low HER2 level.  Compd. 10d (EMB-3) was one of the most potent inhibitors with IC50 of 7.4 nM and 82 nM against EGFR and HER2, resp.  The mechanism studies were also supported by the effect of 10d-induced DNA damage in MDA-MB-468 cells.  In vivo efficacy study showed that 10d could significantly inhibit H522 tumor xenograft model with a TGI of 68% at dose of 100 mg/kg (QDx28, p.o.) and no significant body wt. loss was obsd.  MTD study indicated that compd. 10d had no acute toxicity to mice at doses up to 900 mg/kg (single dose).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUfoD6MEDej7Vg90H21EOLACvtfcHk0lgnUpX7HZ38nA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1ahsr4%253D&md5=20306ebaa26e1259a951f2eef0db5b55</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2016.12.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2016.12.055%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DQ.%26aulast%3DGe%26aufirst%3DZ.%26aulast%3DCheng%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DR.%26atitle%3DDesign%252C%2520synthesis%2520and%2520biological%2520evaluation%2520of%2520quinazoline-phosphoramidate%2520mustard%2520conjugates%2520as%2520anticancer%2520drugs%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D127%26spage%3D442%26epage%3D458%26doi%3D10.1016%2Fj.ejmech.2016.12.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hamed, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abou El Ella, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keeton, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Piazza, G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abadi, A. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartmann, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engel, M.</span></span> <span> </span><span class="NLM_article-title">6-Aryl and heterocycle quinazoline derivatives as potent EGFR inhibitors with improved activity toward gefitinib-sensitive and -resistant tumor cell lines</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1495</span>– <span class="NLM_lpage">1504</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201300147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcmdc.201300147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=23847159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtVykurzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1495-1504&author=M.+M.+Hamedauthor=D.+A.+Abou+El+Ellaauthor=A.+B.+Keetonauthor=G.+A.+Piazzaauthor=A.+H.+Abadiauthor=R.+W.+Hartmannauthor=M.+Engel&title=6-Aryl+and+heterocycle+quinazoline+derivatives+as+potent+EGFR+inhibitors+with+improved+activity+toward+gefitinib-sensitive+and+-resistant+tumor+cell+lines&doi=10.1002%2Fcmdc.201300147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">6-Aryl and Heterocycle Quinazoline Derivatives as Potent EGFR Inhibitors with Improved Activity toward Gefitinib-Sensitive and -Resistant Tumor Cell Lines</span></div><div class="casAuthors">Hamed, Mostafa M.; Abou El Ella, Dalal A.; Keeton, Adam B.; Piazza, Gary A.; Abadi, Ashraf H.; Hartmann, Rolf W.; Engel, Matthias</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1495-1504</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Anilinoquinazolinyl benzaldimines I [R = R1CH:N; R1 = 4-HOC6H4, 2,4-(HO)2C6H3, 3-O2NC6H4, 4-O2NC6H4, 2-MeOC6H4], thioureas I [R = R1NHC(:S)NH; R1 = PhCH2, 2-(4-morpholinyl)ethyl], aminoacetamides and aminopropanamides II [R = R1NHCH2CONH, R1NHCH2CH2CONH; R1 = PhCH2, 4-(2-thiazolylaminosulfonyl)phenyl], and amides [R = R1CONH; R1 = 4-H2NC6H4, 4-NCC6H4, 4-MeCONHC6H4, 4-O2NC6H4, 3,5-(O2N)2C6H3, 4-MeOC6H4, 3,4-(MeO)2C6H3, 4-morpholinyl, 2-furyl, 3-pyridinyl, 4-pyridinyl] were prepd. as inhibitors of wild-type and mutant epidermal growth factor receptor (EGFR) for potential use as antitumor agents.  Most of the compds. inhibited wild-type EGFR with IC50 values of 4-62 nM; I [R = BnNHC(:S)NH, 4-morpholinylethylaminothiocarbonylamino, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] also retained significant activity toward the gefitinib-insensitive EGFRT790M/L858R mutant, displaying up to 24-fold greater potency than gefitinib.  The inhibition of the growth of cancer cells with wild-type and mutant EGFR was detd.; I [R = BnNHC(:S)NH, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] inhibited cancer cell growth more effectively than the std. gefitinib.  Mol. docking calcns. of I [R = BnNHC(:S)NH, 4-morpholinylethylaminothiocarbonylamino, R1NHCH2CH2CONH; R1 = 4-(2-thiazolylaminosulfonyl)phenyl] in double mutant EGFR and of I (R = PhCH2NHCH2CONH, 2-furylcarbonylamino) in wild-type EGFR were performed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9J8DhyAlHb7Vg90H21EOLACvtfcHk0liQpNCfydzryQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtVykurzF&md5=c951d913a38a5062783da74c4cf6afef</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201300147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201300147%26sid%3Dliteratum%253Aachs%26aulast%3DHamed%26aufirst%3DM.%2BM.%26aulast%3DAbou%2BEl%2BElla%26aufirst%3DD.%2BA.%26aulast%3DKeeton%26aufirst%3DA.%2BB.%26aulast%3DPiazza%26aufirst%3DG.%2BA.%26aulast%3DAbadi%26aufirst%3DA.%2BH.%26aulast%3DHartmann%26aufirst%3DR.%2BW.%26aulast%3DEngel%26aufirst%3DM.%26atitle%3D6-Aryl%2520and%2520heterocycle%2520quinazoline%2520derivatives%2520as%2520potent%2520EGFR%2520inhibitors%2520with%2520improved%2520activity%2520toward%2520gefitinib-sensitive%2520and%2520-resistant%2520tumor%2520cell%2520lines%26jtitle%3DChemMedChem%26date%3D2013%26volume%3D8%26spage%3D1495%26epage%3D1504%26doi%3D10.1002%2Fcmdc.201300147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Forcellini, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boutin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lefebvre, C.-A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shayhidin, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulanger, M.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rhéaume, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbeau, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lagüe, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathieu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paquin, J.-F.</span></span> <span> </span><span class="NLM_article-title">Synthesis and biological evaluation of novel quinazoline-4-piperidinesulfamide derivatives as inhibitors of NPP1</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">130</span>– <span class="NLM_lpage">149</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.01.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.ejmech.2018.01.094" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29427906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2018&pages=130-149&author=E.+Forcelliniauthor=S.+Boutinauthor=C.-A.+Lefebvreauthor=E.+E.+Shayhidinauthor=M.-C.+Boulangerauthor=G.+Rh%C3%A9aumeauthor=X.+Barbeauauthor=P.+Lag%C3%BCeauthor=P.+Mathieuauthor=J.-F.+Paquin&title=Synthesis+and+biological+evaluation+of+novel+quinazoline-4-piperidinesulfamide+derivatives+as+inhibitors+of+NPP1&doi=10.1016%2Fj.ejmech.2018.01.094"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological evaluation of novel quinazoline-4-piperidine sulfamide derivatives as inhibitors of NPP1</span></div><div class="casAuthors">Forcellini, Elsa; Boutin, Sophie; Lefebvre, Carole-Anne; Shayhidin, Elnur Elyar; Boulanger, Marie-Chloe; Rheaume, Gabrielle; Barbeau, Xavier; Lague, Patrick; Mathieu, Patrick; Paquin, Jean-Francois</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">130-149</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The ecto-nucleotide pyrophosphatase/phosphodiesterase-1 (NPP1) was recently shown to promote mineralization of the aortic valve, hence, its inhibition represents a significant target.  A quinazoline-4-piperidine sulfamide compd. (I) has been described as a specific and non-competitive inhibitor of NPP1.  Herein the synthesis and in vitro inhibition studies of novel quinazoline-4-piperidine sulfamide analogs using I as the lead compd. was reported.  Of the 26 derivs. prepd., four compds. were found to have Ki < 105 nM against human NPP1.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwRlT7Fi3_H7Vg90H21EOLACvtfcHk0lgElq4t6bF-Ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFCisrs%253D&md5=39d4f90e63b612922cfb5a60b47bd688</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.01.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.01.094%26sid%3Dliteratum%253Aachs%26aulast%3DForcellini%26aufirst%3DE.%26aulast%3DBoutin%26aufirst%3DS.%26aulast%3DLefebvre%26aufirst%3DC.-A.%26aulast%3DShayhidin%26aufirst%3DE.%2BE.%26aulast%3DBoulanger%26aufirst%3DM.-C.%26aulast%3DRh%25C3%25A9aume%26aufirst%3DG.%26aulast%3DBarbeau%26aufirst%3DX.%26aulast%3DLag%25C3%25BCe%26aufirst%3DP.%26aulast%3DMathieu%26aufirst%3DP.%26aulast%3DPaquin%26aufirst%3DJ.-F.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520novel%2520quinazoline-4-piperidinesulfamide%2520derivatives%2520as%2520inhibitors%2520of%2520NPP1%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D147%26spage%3D130%26epage%3D149%26doi%3D10.1016%2Fj.ejmech.2018.01.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group">National Cancer
Institute,
DCTD Division of Cancer Treatment & Diagnosis. Discovery &
Development Services</span>. <span> </span><span class="NLM_article-title">NCI-60
Human Cancer Cell Line Screen NCI-60 Screening Methodology</span>. <a href="https://dtp.cancer.gov/discovery_development/nci-60/" class="extLink">https://dtp.cancer.gov/discovery_development/nci-60/</a> (accessed
Dec 22, 2019).</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=National+Cancer%0AInstitute%2C%0ADCTD+Division+of+Cancer+Treatment+%26+Diagnosis.+Discovery+%26%0ADevelopment+Services.+NCI-60%0AHuman+Cancer+Cell+Line+Screen+NCI-60+Screening+Methodology.+https%3A%2F%2Fdtp.cancer.gov%2Fdiscovery_development%2Fnci-60%2F+%28accessed%0ADec+22%2C+2019%29."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweblink%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DNCI-60%250AHuman%2520Cancer%2520Cell%2520Line%2520Screen%2520NCI-60%2520Screening%2520Methodology" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Darzynkiewicz, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruno, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Del Bino, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorczyca, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hotz, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lassota, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Traganos, F.</span></span> <span> </span><span class="NLM_article-title">Features of apoptotic cells measured by flow cytometry</span>. <i>Cytometry</i> <span class="NLM_year" style="font-weight: bold;">1992</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">795</span>– <span class="NLM_lpage">808</span>, <span class="refDoi"> DOI: 10.1002/cyto.990130802</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1002%2Fcyto.990130802" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1333943" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADyaK3s%252Fpt1OrtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=795-808&author=Z.+Darzynkiewiczauthor=S.+Brunoauthor=G.+Del+Binoauthor=W.+Gorczycaauthor=M.+A.+Hotzauthor=P.+Lassotaauthor=F.+Traganos&title=Features+of+apoptotic+cells+measured+by+flow+cytometry&doi=10.1002%2Fcyto.990130802"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Features of apoptotic cells measured by flow cytometry</span></div><div class="casAuthors">Darzynkiewicz Z; Bruno S; Del Bino G; Gorczyca W; Hotz M A; Lassota P; Traganos F</div><div class="citationInfo"><span class="NLM_cas:title">Cytometry</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">795-808</span>
        ISSN:<span class="NLM_cas:issn">0196-4763</span>.
    </div><div class="casAbstract">The present review describes several methods to characterize and differentiate between two different mechanisms of cell death, apoptosis and necrosis.  Most of these methods were applied to studies of apoptosis triggered in the human leukemic HL-60 cell line by DNA topoisomerase I or II inhibitors, and in rat thymocytes by either topoisomerase inhibitors or prednisolone.  In most cases, apoptosis was selective to cells in a particular phase of the cell cycle: only S-phase HL-60 cells and G0 thymocytes were mainly affected.  Necrosis was induced by excessively high concentrations of these drugs.  The following cell features were found useful to characterize the mode of cell death: a) Activation of an endonuclease in apoptocic cells resulted in extraction of the low molecular weight DNA following cell permeabilization, which, in turn, led to their decreased stainability with DNA-specific fluorochromes.  Measurements of DNA content made it possible to identify apoptotic cells and to recognize the cell cycle phase specificity of the apoptotic process. b) Plasma membrane integrity, which is lost in necrotic but not apoptotic cells, was probed by the exclusion of propidium iodide (PI).  The combination of PI followed by Hoechst 33342 proved to be an excellent probe to distinguish live, necrotic, early- and late-apoptotic cells. c) Mitochondrial transmembrane potential, assayed by retention of rhodamine 123 was preserved in apoptotic but not necrotic cells. d) The ATP-dependent lysosomal proton pump, tested by the supravital uptake of acridine orange (AO) was also preserved in apoptotic but not necrotic cells. e) Bivariate analysis of cells stained for DNA and protein revealed markedly diminished protein content in apoptotic cells, most likely due to activation of endogenous proteases.  Necrotic cells, having leaky membranes, had minimal protein content. f) Staining of RNA allowed for the discrimination of G0 from G1 cells and thus made it possible to reveal that apoptosis was selective to G0 thymocytes. g) The decrease in forward light scatter, paralleled either by no change (HL-60 cells) or an increase (thymocytes) of right angle scatter, were early changes during apoptosis. h) The sensitivity of DNA in situ to denaturation, was increased in apoptotic and necrotic cells.  This feature, probed by staining with AO at low pH, provided a sensitive and early assay to discriminate between live, apoptotic and necrotic cells, and to evaluate the cell cycle phase specificity of these processes. i) The in situ nick translation assay employing labeled triphosphonucleotides can be used to reveal DNA strand breaks, to detect the very early stages of apoptosis.(ABSTRACT TRUNCATED AT 400 WORDS)</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT5cujAkovwXEbxh97K6Vt6fW6udTcc2eZLZuZAJpKB07ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaK3s%252Fpt1OrtQ%253D%253D&md5=03a90fef61f38d88683f68707cd24f30</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1002%2Fcyto.990130802&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcyto.990130802%26sid%3Dliteratum%253Aachs%26aulast%3DDarzynkiewicz%26aufirst%3DZ.%26aulast%3DBruno%26aufirst%3DS.%26aulast%3DDel%2BBino%26aufirst%3DG.%26aulast%3DGorczyca%26aufirst%3DW.%26aulast%3DHotz%26aufirst%3DM.%2BA.%26aulast%3DLassota%26aufirst%3DP.%26aulast%3DTraganos%26aufirst%3DF.%26atitle%3DFeatures%2520of%2520apoptotic%2520cells%2520measured%2520by%2520flow%2520cytometry%26jtitle%3DCytometry%26date%3D1992%26volume%3D13%26spage%3D795%26epage%3D808%26doi%3D10.1002%2Fcyto.990130802" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akl, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ayoub, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohyeldin, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Busnena, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foudah, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sayed, K. A. E.</span></span> <span> </span><span class="NLM_article-title">Olive phenolics as c-Met inhibitors: (-)-Oleocanthal attenuates cell proliferation, invasiveness, and tumor growth in breast cancer models</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">e97622</span>, <span class="refDoi"> DOI: 10.1371/journal.pone.0097622</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1371%2Fjournal.pone.0097622" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=24849787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKhtbbI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e97622&author=M.+R.+Aklauthor=N.+M.+Ayoubauthor=M.+M.+Mohyeldinauthor=B.+A.+Busnenaauthor=A.+I.+Foudahauthor=Y.-Y.+Liuauthor=K.+A.+E.+Sayed&title=Olive+phenolics+as+c-Met+inhibitors%3A+%28-%29-Oleocanthal+attenuates+cell+proliferation%2C+invasiveness%2C+and+tumor+growth+in+breast+cancer+models&doi=10.1371%2Fjournal.pone.0097622"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">Olive phenolics as c-Met inhibitors: (-)-Oleocanthal attenuates cell proliferation, invasiveness, and tumor growth in breast cancer models</span></div><div class="casAuthors">Akl, Mohamed R.; Ayoub, Nehad M.; Mohyeldin, Mohamed M.; Busnena, Belnaser A.; Foudah, Ahmed I.; Liu, Yong-Yu; El Sayed, Khalid A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">e97622/1-e97622/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Dysregulation of the Hepatocyte growth factor (HGF)/c-Met signaling axis upregulates diverse tumor cell functions, including cell proliferation, survival, scattering and motility, epithelial-to-mesenchymal transition (EMT), angiogenesis, invasion, and metastasis.  (-)-Oleocanthal is a naturally occurring secoiridoid from extra-virgin olive oil, which showed antiproliferative and antimigratory activity against different cancer cell lines.  The aim of this study was to characterize the intracellular mechanisms involved in mediating the anticancer effects of (-)-oleocanthal treatment and the potential involvement of c-Met receptor signaling components in breast cancer.  Results showed that (-)-oleocanthal inhibits the growth of human breast cancer cell lines MDA-MB-231, MCF-7 and BT-474 while similar treatment doses were found to have no effect on normal human MCF10A cell growth.  In addn., (-)-oleocanthal treatment caused a dose-dependent inhibition of HGF-induced cell migration, invasion and G1/S cell cycle progression in breast cancer cell lines.  Moreover, (-)-oleocanthal treatment effects were found to be mediated via inhibition of HGF-induced c-Met activation and its downstream mitogenic signaling pathways.  This growth inhibitory effect is assocd. with blockade of EMT and redn. in cellular motility.  Further results from in vivo studies showed that (-)-oleocanthal treatment suppressed tumor cell growth in an orthotopic model of breast cancer in athymic nude mice.  Collectively, the findings of this study suggest that (-)-oleocanthal is a promising dietary supplement lead with potential for therapeutic use to control malignancies with aberrant c-Met activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEdF-y6UaVqbVg90H21EOLACvtfcHk0lglM1s30-kCdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKhtbbI&md5=c58947fd248d835a77f916acd69da948</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0097622&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0097622%26sid%3Dliteratum%253Aachs%26aulast%3DAkl%26aufirst%3DM.%2BR.%26aulast%3DAyoub%26aufirst%3DN.%2BM.%26aulast%3DMohyeldin%26aufirst%3DM.%2BM.%26aulast%3DBusnena%26aufirst%3DB.%2BA.%26aulast%3DFoudah%26aufirst%3DA.%2BI.%26aulast%3DLiu%26aufirst%3DY.-Y.%26aulast%3DSayed%26aufirst%3DK.%2BA.%2BE.%26atitle%3DOlive%2520phenolics%2520as%2520c-Met%2520inhibitors%253A%2520%2528-%2529-Oleocanthal%2520attenuates%2520cell%2520proliferation%252C%2520invasiveness%252C%2520and%2520tumor%2520growth%2520in%2520breast%2520cancer%2520models%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De97622%26doi%3D10.1371%2Fjournal.pone.0097622" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="note"><p class="first last">Annexin V-FITC Apoptosis Staining/Detection Kit (ab14085).</p></div><div class="citationLinks"><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abd Elmageed, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsumagari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sholl, A. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedlander, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Qurayshi, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boulares, H. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kandil, E.</span></span> <span> </span><span class="NLM_article-title">Prognostic role of BRAF(V600E) cellular localization in melanoma</span>. <i>J. Am. Coll. Surg.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>226</i></span>,  <span class="NLM_fpage">526</span>– <span class="NLM_lpage">537</span>, <span class="refDoi"> DOI: 10.1016/j.jamcollsurg.2017.12.040</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=10.1016%2Fj.jamcollsurg.2017.12.040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=29369798" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;key=1%3ACAS%3A280%3ADC%252BC1Mvkt12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=226&publication_year=2018&pages=526-537&author=Z.+Y.+Abd+Elmageedauthor=R.+F.+Mooreauthor=K.+Tsumagariauthor=M.+M.+Leeauthor=A.+B.+Shollauthor=P.+Friedlanderauthor=Z.+Al-Qurayshiauthor=M.+Hassanauthor=A.+R.+Wangauthor=H.+A.+Boularesauthor=E.+Kandil&title=Prognostic+role+of+BRAF%28V600E%29+cellular+localization+in+melanoma&doi=10.1016%2Fj.jamcollsurg.2017.12.040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">Prognostic Role of BRAF(V600E) Cellular Localization in Melanoma</span></div><div class="casAuthors">Abd Elmageed Zakaria Y; Moore Robert F; Tsumagari Koji; Hassan Mohamed; Lee Michael M; Sholl Andrew B; Wang Alun R; Friedlander Paul; Al-Qurayshi Zaid; Boulares Hamid A; Kandil Emad</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American College of Surgeons</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">226</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">526-537</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Approximately half of cutaneous melanoma tissues harbor BRAF(V600E) mutations, resulting in a constitutive activation of the mitogen-activated protein kinase (MAPK) pathway.  Nuclear-cytoplasmic transport machinery is dysregulated in neoplastic cells and alters the key regulatory proteins that can lead to tumor progression and drug resistance.  The significance of nuclear localization of BRAF(V600E) has not been fully understood.  We examined the clinical significance of intracellular localization of BRAF(V600E) in cutaneous melanoma.  STUDY DESIGN:  Immunohistochemical analysis of BRAF(V600E) was performed on formalin-fixed, paraffin-embedded specimens of cutaneous melanoma (n = 91).  Staining intensity was graded in a blinded manner.  Correlations to clinical factors were analyzed by Fisher's exact test and 2-tailed t-test.  Localization of BRAF(V600E) was determined in melanoma cells, and we investigated their resistance to BRAF(V600E)-specific inhibitor according to nuclear localization in both in vitro and in vivo models.  RESULTS:  We included 91 patients, of whom 32% (29 of 91) had cytoplasmic BRAF(V600E).  Nuclear BRAF(V600E) was observed in 30% (27 of 91).  Overall, BRAF(V600E) expression correlated with TNM stage (p = 0.011), mitotic activity (p = 0.010), and ulceration (p = 0.045).  Nuclear BRAF(V600E) expression correlated with overall clinical stage (p < 0.001), tumor size (p < 0.001), regional lymph node (p < 0.017), depth of invasion (p = 0.005), Clark level (p < 0.001), mitotic activity (p < 0.001), ulceration (p < 0.001), and margin status (p = 0.017).  On a cellular level, BRAF(V600E) was identified in the nucleus, and its translocation was serum dependent.  Our in vitro and in vivo data revealed sequestration of BRAF(V600E) in the cytosol-sensitized resistant cells to vemurafenib; nuclear retention of BRAF(V600E) was associated with aggressiveness and drug resistance.  CONCLUSIONS:  Nuclear localization of BRAF(V600E) is associated with melanoma aggressiveness.  Further multi-institutional studies are warranted to confirm the clinical relevance of nuclear localization of BRAF(V600E).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSOTtfrC41t5LGbPKh4T9eAfW6udTcc2eadJDBDFvk3m7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1Mvkt12itA%253D%253D&md5=4137b2119d30f486607f790a08134d02</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1016%2Fj.jamcollsurg.2017.12.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jamcollsurg.2017.12.040%26sid%3Dliteratum%253Aachs%26aulast%3DAbd%2BElmageed%26aufirst%3DZ.%2BY.%26aulast%3DMoore%26aufirst%3DR.%2BF.%26aulast%3DTsumagari%26aufirst%3DK.%26aulast%3DLee%26aufirst%3DM.%2BM.%26aulast%3DSholl%26aufirst%3DA.%2BB.%26aulast%3DFriedlander%26aufirst%3DP.%26aulast%3DAl-Qurayshi%26aufirst%3DZ.%26aulast%3DHassan%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DA.%2BR.%26aulast%3DBoulares%26aufirst%3DH.%2BA.%26aulast%3DKandil%26aufirst%3DE.%26atitle%3DPrognostic%2520role%2520of%2520BRAF%2528V600E%2529%2520cellular%2520localization%2520in%2520melanoma%26jtitle%3DJ.%2520Am.%2520Coll.%2520Surg.%26date%3D2018%26volume%3D226%26spage%3D526%26epage%3D537%26doi%3D10.1016%2Fj.jamcollsurg.2017.12.040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0','PDB','3PP0'); return false;">PDB: 3PP0</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i131"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_32651"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c01647?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c01647</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings for the target compounds: <b>10a–25h</b>, additional discussion of <i>in vitro</i> single-dose and five-dose screening against the NCI 60 cell panel and the selectivity ratio, Annexin V FITC/PI apoptosis assays, average tumor growth rate for mice groups that received the vehicle, compound <b>17d</b>, and <b>25b</b>, the structural optimization using CADD and the docking results into HER2 kinase, and the spectra of <sup>1</sup>H NMR, <sup>13</sup>C DEPT, <sup>13</sup>C NMR, and HRMS for compounds <b>4c–25h</b> and UPLC/MS chromatograms and final analytical reports of compounds (1<b>0a–c</b>,<b>15a</b>,<b>b</b>,<b>c</b>,<b>e</b>,<b>f</b>,<b>i</b>,<b>k</b>,<b>l</b>, <b>17d</b>, <b>and 25a</b>,<b>b</b>,<b>c</b>,<b>e</b>,<b>f</b>,<b>g</b>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Antiproliferative activity of compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_002.csv">CSV</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a> dimer (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_003.pdb">PDB</a>)</p></li><li><p class="inline"><a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PP0">3PP0</a> monomer (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_004.pdb">PDB</a>)</p></li><li><p class="inline">Optimization of docking validation (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_005.pdb">PDB</a>)</p></li><li><p class="inline">Analysis of 27 compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_006.pdb">PDB</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_001.pdf">jm0c01647_si_001.pdf (37.13 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_002.csv">jm0c01647_si_002.csv (2.75 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_003.pdb">jm0c01647_si_003.pdb (391.33 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_004.pdb">jm0c01647_si_004.pdb (191.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_005.pdb">jm0c01647_si_005.pdb (406.61 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c01647/suppl_file/jm0c01647_si_006.pdb">jm0c01647_si_006.pdb (526.02 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c01647&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2020.63.issue-24%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c01647%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c01647" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0fac57921258d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
