
<html lang="en"     class="pb-page"  data-request-id="3ec83157-12d2-4ca7-8120-11e89e414e77"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.8b01448;wgroup:string:ACHS website Group;issue:issue:10.1021/jmcmar.2019.62.issue-3;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model" /></meta><meta name="dc.Creator" content="Sabine  Depauw" /></meta><meta name="dc.Creator" content="Mélanie  Lambert" /></meta><meta name="dc.Creator" content="Samy  Jambon" /></meta><meta name="dc.Creator" content="Ananya  Paul" /></meta><meta name="dc.Creator" content="Paul  Peixoto" /></meta><meta name="dc.Creator" content="Raja  Nhili" /></meta><meta name="dc.Creator" content="Laura  Marongiu" /></meta><meta name="dc.Creator" content="Martin  Figeac" /></meta><meta name="dc.Creator" content="Christelle  Dassi" /></meta><meta name="dc.Creator" content="Charles  Paul-Constant" /></meta><meta name="dc.Creator" content="Benjamin  Billoré" /></meta><meta name="dc.Creator" content="Arvind  Kumar" /></meta><meta name="dc.Creator" content="Abdelbasset A.  Farahat" /></meta><meta name="dc.Creator" content="Mohamed A.  Ismail" /></meta><meta name="dc.Creator" content="Ekaterina  Mineva" /></meta><meta name="dc.Creator" content="Daniel P.  Sweat" /></meta><meta name="dc.Creator" content="Chad E.  Stephens" /></meta><meta name="dc.Creator" content="David W.  Boykin" /></meta><meta name="dc.Creator" content="W. David  Wilson" /></meta><meta name="dc.Creator" content="Marie-Hélène  David-Cordonnier" /></meta><meta name="dc.Description" content="Most transcription factors were for a long time considered as undruggable targets because of the absence of binding pockets for direct targeting. HOXA9, implicated in acute myeloid leukemia, is one..." /></meta><meta name="Description" content="Most transcription factors were for a long time considered as undruggable targets because of the absence of binding pockets for direct targeting. HOXA9, implicated in acute myeloid leukemia, is one..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="January 15, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01448" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01448" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01448" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01448" /></link>
        
    
    

<title>Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01448" /></meta><meta property="og:title" content="Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0017.jpeg" /></meta><meta property="og:description" content="Most transcription factors were for a long time considered as undruggable targets because of the absence of binding pockets for direct targeting. HOXA9, implicated in acute myeloid leukemia, is one of them. To date, only indirect targeting of HOXA9 expression or multitarget HOX/PBX protein/protein interaction inhibitors has been developed. As an attractive alternative by inhibiting the DNA binding, we selected a series of heterocyclic diamidines as efficient competitors for the HOXA9/DNA interaction through binding as minor groove DNA ligands on the HOXA9 cognate sequence. Selected DB818 and DB1055 compounds altered HOXA9-mediated transcription in luciferase assays, cell survival, and cell cycle, but increased cell death and granulocyte/monocyte differentiation, two main HOXA9 functions also highlighted using transcriptomic analysis of DB818-treated murine Hoxa9-transformed hematopoietic cells. Altogether, these data demonstrate for the first time the propensity of sequence-selective DNA ligands to inhibit HOXA9/DNA binding both in vitro and in a murine Hoxa9-dependent leukemic cell model." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01448"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01448">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01448&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01448&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01448&amp;href=/doi/10.1021/acs.jmedchem.8b01448" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1306-1329</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.8b01402" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.8b01464" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Sabine Depauw</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Sabine Depauw</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Sabine++Depauw">Sabine Depauw</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8148-7900" title="Orcid link">http://orcid.org/0000-0002-8148-7900</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mélanie Lambert</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mélanie Lambert</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=M%C3%A9lanie++Lambert">Mélanie Lambert</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7472-4146" title="Orcid link">http://orcid.org/0000-0002-7472-4146</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Samy Jambon</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Samy Jambon</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Samy++Jambon">Samy Jambon</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7236-3239" title="Orcid link">http://orcid.org/0000-0001-7236-3239</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ananya Paul</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ananya Paul</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ananya++Paul">Ananya Paul</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Paul Peixoto</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Paul Peixoto</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Paul++Peixoto">Paul Peixoto</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Raja Nhili</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Raja Nhili</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Raja++Nhili">Raja Nhili</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Laura Marongiu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Laura Marongiu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Laura++Marongiu">Laura Marongiu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Martin Figeac</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Martin Figeac</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Functional and Structural Genomic Platform, Lille University, F-59000 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Martin++Figeac">Martin Figeac</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christelle Dassi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christelle Dassi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christelle++Dassi">Christelle Dassi</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Charles Paul-Constant</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Charles Paul-Constant</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Charles++Paul-Constant">Charles Paul-Constant</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Benjamin Billoré</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benjamin Billoré</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benjamin++Billor%C3%A9">Benjamin Billoré</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Arvind Kumar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Arvind Kumar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Arvind++Kumar">Arvind Kumar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Abdelbasset A. Farahat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Abdelbasset A. Farahat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Pharmaceutical Organic Chemistry, Faculty of Pharmacy, Mansoura University, Mansoura 35516, Egypt</div><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Abdelbasset+A.++Farahat">Abdelbasset A. Farahat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Mohamed A. Ismail</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mohamed A. Ismail</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Faculty of Science, Mansoura University, Mansoura 35516, Egypt</div><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mohamed+A.++Ismail">Mohamed A. Ismail</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ekaterina Mineva</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ekaterina Mineva</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ekaterina++Mineva">Ekaterina Mineva</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Daniel P. Sweat</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Daniel P. Sweat</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Physics, Augusta University, Augusta, Georgia 30904, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Daniel+P.++Sweat">Daniel P. Sweat</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Chad E. Stephens</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Chad E. Stephens</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry and Physics, Augusta University, Augusta, Georgia 30904, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Chad+E.++Stephens">Chad E. Stephens</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David W. Boykin</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David W. Boykin</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+W.++Boykin">David W. Boykin</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">W. David Wilson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">W. David Wilson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Chemistry, Georgia State University, Atlanta, Georgia 30303, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=W.+David++Wilson">W. David Wilson</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5225-5089" title="Orcid link">http://orcid.org/0000-0001-5225-5089</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Marie-Hélène David-Cordonnier</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marie-Hélène David-Cordonnier</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">UMR-S1172-JPARC (Jean-Pierre Aubert Research Center), INSERM, University of Lille, Centre Hospitalier Universitaire de Lille, Institut pour la Recherche sur le Cancer de Lille (IRCL), F-59045 Lille, France</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#d8b5b9aab1bdf5b0bdb4bdb6bdf6bcb9aeb1bc98b1b6abbdaab5f6beaa"><span class="__cf_email__" data-cfemail="b5d8d4c7dcd098ddd0d9d0dbd09bd1d4c3dcd1f5dcdbc6d0c7d89bd3c7">[email protected]</span></a>. Phone: +33 320 169 223. Fax: +33 320 538 562.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marie-H%C3%A9l%C3%A8ne++David-Cordonnier">Marie-Hélène David-Cordonnier</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9831-5577" title="Orcid link">http://orcid.org/0000-0001-9831-5577</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01448&amp;href=/doi/10.1021%2Facs.jmedchem.8b01448" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1306–1329</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 15, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 September 2018</li><li><span class="item_label"><b>Published</b> online</span>15 January 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 February 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01448" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01448</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1306%26pageCount%3D24%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSabine%2BDepauw%252C%2BM%25C3%25A9lanie%2BLambert%252C%2BSamy%2BJambon%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D3%26contentID%3Dacs.jmedchem.8b01448%26title%3DHeterocyclic%2BDiamidine%2BDNA%2BLigands%2Bas%2BHOXA9%2BTranscription%2BFactor%2BInhibitors%253A%2BDesign%252C%2BMolecular%2BEvaluation%252C%2Band%2BCellular%2BConsequences%2Bin%2Ba%2BHOXA9-Dependant%2BLeukemia%2BCell%2BModel%26numPages%3D24%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1329%26publicationDate%3DFebruary%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01448"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">960</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01448" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Heterocyclic Diamidine DNA Ligands as HOXA9 Transcription Factor Inhibitors: Design, Molecular Evaluation, and Cellular Consequences in a HOXA9-Dependant Leukemia Cell Model&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Sabine&quot;,&quot;last_name&quot;:&quot;Depauw&quot;},{&quot;first_name&quot;:&quot;Mélanie&quot;,&quot;last_name&quot;:&quot;Lambert&quot;},{&quot;first_name&quot;:&quot;Samy&quot;,&quot;last_name&quot;:&quot;Jambon&quot;},{&quot;first_name&quot;:&quot;Ananya&quot;,&quot;last_name&quot;:&quot;Paul&quot;},{&quot;first_name&quot;:&quot;Paul&quot;,&quot;last_name&quot;:&quot;Peixoto&quot;},{&quot;first_name&quot;:&quot;Raja&quot;,&quot;last_name&quot;:&quot;Nhili&quot;},{&quot;first_name&quot;:&quot;Laura&quot;,&quot;last_name&quot;:&quot;Marongiu&quot;},{&quot;first_name&quot;:&quot;Martin&quot;,&quot;last_name&quot;:&quot;Figeac&quot;},{&quot;first_name&quot;:&quot;Christelle&quot;,&quot;last_name&quot;:&quot;Dassi&quot;},{&quot;first_name&quot;:&quot;Charles&quot;,&quot;last_name&quot;:&quot;Paul-Constant&quot;},{&quot;first_name&quot;:&quot;Benjamin&quot;,&quot;last_name&quot;:&quot;Billoré&quot;},{&quot;first_name&quot;:&quot;Arvind&quot;,&quot;last_name&quot;:&quot;Kumar&quot;},{&quot;first_name&quot;:&quot;Abdelbasset&quot;,&quot;last_name&quot;:&quot;A. Farahat&quot;},{&quot;first_name&quot;:&quot;Mohamed&quot;,&quot;last_name&quot;:&quot;A. Ismail&quot;},{&quot;first_name&quot;:&quot;Ekaterina&quot;,&quot;last_name&quot;:&quot;Mineva&quot;},{&quot;first_name&quot;:&quot;Daniel&quot;,&quot;last_name&quot;:&quot;P. Sweat&quot;},{&quot;first_name&quot;:&quot;Chad&quot;,&quot;last_name&quot;:&quot;E. Stephens&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;W. Boykin&quot;},{&quot;first_name&quot;:&quot;W.&quot;,&quot;last_name&quot;:&quot;David Wilson&quot;},{&quot;first_name&quot;:&quot;Marie-Hélène&quot;,&quot;last_name&quot;:&quot;David-Cordonnier&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;01&quot;,&quot;day&quot;:&quot;15&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;1306-1329&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01448&quot;},&quot;abstract&quot;:&quot;Most transcription factors were for a long time considered as undruggable targets because of the absence of binding pockets for direct targeting. HOXA9, implicated in acute myeloid leukemia, is one of them. To date, only indirect targeting of HOXA9 expression or multitarget HOX/PBX protein/protein interaction inhibitors has been developed. As an attractive alternative by inhibiting the DNA binding, we selected a series of heterocyclic diamidines as efficient competitors for the HOXA9/DNA interaction through binding as minor groove DNA ligands on the HOXA9 cognate sequence. Selected DB818 and DB1055 compounds altered HOXA9-mediated transcription in luciferase assays, cell survival, and cell cycle, but increased cell death and granulocyte/monocyte differentiation, two main HOXA9 functions also highlighted using transcriptomic analysis of DB818-treated murine Hoxa9-transformed hematopoietic cells. Altogether, these data demonstrate for the first time the propensity of sequence-selective DNA ligands to inhibit HO&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01448&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01448" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01448&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01448" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01448&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01448" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01448&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01448&amp;href=/doi/10.1021/acs.jmedchem.8b01448" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01448" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01448" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (11 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01448%26sid%3Dliteratum%253Aachs%26pmid%3D30645099%26genre%3Darticle%26aulast%3DDepauw%26date%3D2019%26atitle%3DHeterocyclic%2BDiamidine%2BDNA%2BLigands%2Bas%2BHOXA9%2BTranscription%2BFactor%2BInhibitors%253A%2BDesign%252C%2BMolecular%2BEvaluation%252C%2Band%2BCellular%2BConsequences%2Bin%2Ba%2BHOXA9-Dependant%2BLeukemia%2BCell%2BModel%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D3%26spage%3D1306%26epage%3D1329%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291881" title="Differentiation">Differentiation</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291993" title="Nucleic acid structure">Nucleic acid structure</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/jmcmar.2019.62.issue-3/20190214/jmcmar.2019.62.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Most transcription factors were for a long time considered as undruggable targets because of the absence of binding pockets for direct targeting. HOXA9, implicated in acute myeloid leukemia, is one of them. To date, only indirect targeting of HOXA9 expression or multitarget HOX/PBX protein/protein interaction inhibitors has been developed. As an attractive alternative by inhibiting the DNA binding, we selected a series of heterocyclic diamidines as efficient competitors for the HOXA9/DNA interaction through binding as minor groove DNA ligands on the HOXA9 cognate sequence. Selected <b>DB818</b> and <b>DB1055</b> compounds altered HOXA9-mediated transcription in luciferase assays, cell survival, and cell cycle, but increased cell death and granulocyte/monocyte differentiation, two main HOXA9 functions also highlighted using transcriptomic analysis of <b>DB818</b>-treated murine Hoxa9-transformed hematopoietic cells. Altogether, these data demonstrate for the first time the propensity of sequence-selective DNA ligands to inhibit HOXA9/DNA binding both in vitro and in a murine Hoxa9-dependent leukemic cell model.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16542" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16542" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Transcription factors are often associated with different phases of the oncogenic process (initiation, development, invasion, metastasis, etc.) and represent ∼20% of the established targets in cancers. However, only a few of them are targeted in cancer therapies. The different approaches for targeting transcription factors are (i) to directly target the protein via a functional pocket, (ii) to target protein/protein interactions, and (iii) to interfere with the transcription factor’s ability to interact with its cognate binding site on a targeted DNA sequence.<a onclick="showRef(event, 'ref1 ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref4">(1−4)</a> Only a few of them can be targeted through their ligand binding pocket. Hormone/retinoid receptors are a notable example, and more precisely the retinoid receptor RARα, which can be targeted by all-trans retinoic acid (ATRA). This compound has been used in human therapy for 3 decades in PML-RARα-rearranged acute promyelocytic leukemia (APL) through the induction of differentiation.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Other transcription factors families have been considered as undruggable targets.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Alternatives are protein/protein inhibitors, that can for example target the p53 and mdm2/mdmx interface,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> BCL6<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> and STAT3 homodimers,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> YAP/TAZ<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> or CBP/βcatenin<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> interactions, or the development of drugs from phenotypic screening as against NRF2<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> or HSF1.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Only a limited number of strategies are dedicated to direct targeting of transcription factor–DNA binding domains, and a recent example is the PAX2 transcription factor DNA binding domain.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Alternatively, another manner to target a transcription factor at the DNA binding level is to develop molecules that target the DNA promoter sequence and compete with transcription factor binding to its cognate sequence. This is the case for a variety of minor groove DNA binding structures. Many different series of DNA minor groove binders are also in preclinical development such as mithramycines, polyamides, and diamidine heterocyclic derivatives.<a onclick="showRef(event, 'ref1 ref4 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref1 ref4 ref16 ref17">(1,4,16,17)</a> Among this group, mithramycin A and some of its derivatives inhibit SP1/DNA,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> which is associated with the deregulation of many SP1-driven genes such as MYC, FOXM1, p21, Ki-67, CRABP1, BAK, or KLF5.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Mithramycin A and derivatives also block the EWS-Fli-1/DNA interaction<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> and have entered clinical trials against refractory Ewing sarcoma.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Interestingly, two other minor groove DNA binders recently showed interesting anti-leukemic activities by targeting transcription factors associated with acute myeloid leukemia (AML): the alkylating agent-conjugated pyrrole-imidazole polyamide Chb-M against RUNX1<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and <b>DB2313</b> against PU.1.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a></div><div class="NLM_p">HOXA9 is a well-known transcription factor implicated in oncogenic and leukemic processes. Besides its implication in some solid tumors,<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27">(23−27)</a> HOXA9 is clearly associated with different hematologic malignancies.<a onclick="showRef(event, 'ref28 ref29 ref30 ref31 ref32'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30 ref31 ref32">(28−32)</a> Indeed, HOXA9 is over-expressed in ∼70% of AML or fused to NUP98 in the t(7; 11) (p15; p15) translocation leading to the hybrid protein NUP98-HOXA9 responsible for 2–5% of AML<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> and 5% of myelodysplasic syndromes (MDS).<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Functionally, HOXA9 is involved in the differentiation blockade, resulting in the accumulation of leukemic blasts in patients.<a onclick="showRef(event, 'ref28 ref29 ref32 ref36'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref32 ref36">(28,29,32,36)</a> Such leukemic activity is directly attributable to the HOXA9/DNA binding activity. Indeed, deletion of its homeodomain abolishes the propensity of HOXA9 to induce leukemia in mice.<a onclick="showRef(event, 'ref37 ref38 ref39'); return false;" href="javascript:void(0);" class="ref ref37 ref38 ref39">(37−39)</a> Furthermore, HOXA9 was found quite recently as one of the genes associated with leukemic stem cell signature and resulting in poor prognosis.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> The HOXA9 transcription factor is therefore a good candidate for the development of new targeted therapies for differentiation that would be intended for a large number of patients suffering from AML.</div><div class="NLM_p last">HOXA9 binds to its regulatory DNA sequence at a 5′-tgATTTAt site (JASPAR MA0594.1), as defined by ChIP-seq in a murine model expressing a tagged-Hoxa9 protein.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> DNA recognition occurs through the interaction between the third α-helix of its homeodomain and the DNA major groove, and between the amino-terminal unfolded module of its homeodomain which makes deep contacts in the minor groove of the AT-rich segment of the consensus binding site.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> In the present study, we evaluated a focused library of heterocyclic diamidine DNA ligands as functional inhibitors of human HOXA9/DNA binding activity. We evaluated their mode of binding to DNA in vitro and the cellular consequences of the protein–DNA complex inhibition. We further selected <b>DB818</b> and <b>DB1055</b> for additional investigation and evidenced their inhibitory effect on HOXA9-induced transcription control by using HOXA9-driven luciferase gene expression. The compounds effects on cell survival and clonogenic activity of Hoxa9-transformed murine bone marrow cells argue for the cell differentiation process, as highlighted by transcriptomic analyses of <b>DB818</b>-treated leukemic cells. These results clearly show for the first time the selection of direct functional inhibitors of HOXA9/DNA binding through the targeting of the HOXA9 consensus binding site by DNA ligands.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51184" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51184" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4">2.1.  Design of the Evaluated Compounds</h3><div class="NLM_p">In order to develop inhibitors for the HOXA9 transcription factor at its DNA binding level, we took advantage of the HOXA9 DNA binding domain particularities. Indeed, HOXA9 binds through a direct interaction of the third α-helix and the N-terminal portion of its homeodomain with the major and minor grooves, respectively. The key interaction is with the 5′-ATT(A/T) sequence of the 5′-tgATTTAt defined consensus site.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a> We were specifically interested in heterocyclic diamidine compounds that have been previously reported to be potent in vitro transcription factor DNA binding inhibitors: <b>DB293</b> against Pit-1 and Brn-3<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and <b>DB1255</b> against the ERG transcription factor.<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> The AT specific minor groove binding diamidines evaluated as HOXA9 inhibitors were obtained from our large library of heterocyclic diamidines synthesized as potential anti-parasitic agents. The selections were made from a number of structurally diverse diamidine types with many selected for their binding to AT-rich, ATT(A/T), or ATGA binding to DNA,<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46,47)</a> and new structures were designed on the basis of three central aromatic groups R<sub>1</sub>, R<sub>2</sub>, and R<sub>3</sub> flanked by an amidine moiety at each extremity (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). A series of 39 compounds is classified based on the presence of R<sub>2</sub> as a 5- or 6-membered ring, on the nature or R<sub>3</sub> rings (general with a phenyl or a benzimidazole). Modifications of the R<sub>1</sub> ring and the amidine extremities (R<sub>4</sub> and R<sub>5</sub>) offer a large diversity of molecules to expand structure/function relationships (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structures of the 39 Evaluated Diamidine Derivatives<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0016.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Numbers in brackets refer to numbering used in the synthesis section and schemes presented in chemistry section.</p></div></div><div></div></div><div class="NLM_p">Approximately two-thirds of those selected have been previously reported and references to their synthesis are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Supporting Information</a>. The syntheses of those previously unreported are described below.</div><div class="NLM_p">The synthesis of three furan benzimidazole analogues (<b>4a–c</b>; <b>DB915</b>, <b>DB928</b>, and <b>DB995</b>) is outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The 2-cyanoaryl-5-formyl furans were prepared employing standard Stille cross-coupling conditions allowing 2-(tri-<i>n</i>-butylstannyl)-5-(dimethoxymethyl)furan to react with the various cyanoaryl bromides in the presence of tetrakis(triphenylphosphine)palladium. The required bis-nitriles were obtained by oxidative coupling of 3,4-diaminobenzonitrile with the various 2-cyanoaryl-5-formyl furans using benzoquinone as the oxidant. The diamidines were prepared by first converting the bis-nitrile into the corresponding amidoximes, which were acetylated, and subsequently subjected to hydrogenolysis to yield the desired diamidines. In the case of <b>DB928</b>, the hydrogenolysis in addition to cleaving the N–OAc bond also cleaved the benzyl group.</div><figure id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and Conditions: (a) (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-Dioxane, (ii) 2.0 M HCl; (b) 3,4-Diaminobenzonitrile, 1,4-Benzoquinone, EtOH; (c) NH<sub>2</sub>OH–HCl/KO-<i>t</i>-Bu, DMSO; (d) (i) AcOH/Ac<sub>2</sub>O, (ii) H<sub>2</sub>/Pd–C, AcOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The route employed for synthesis of the thiazole benzimidazole (<b>7</b>; <b>DB2651</b>) is shown in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. In this case, Suzuki cross-coupling between 5-bromothiazole-2-carbaldehyde and 4-cyanophenylboronic acid was utilized to obtain 5-(4-cyanophenyl)thiazole-2-carbaldehyde. In this example, the bis-nitrile was achieved by oxidative coupling of 3,4-diaminobenzonitrile and the thiazole aldehyde using sodium metabisulfite as the oxidant. The diamidine was obtained from the bis-nitrile using classical Pinner methodology.</div><figure id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Reagents and Conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>/H<sub>2</sub>O, Dioxane, Reflux; (b) 3,4-Diaminobenzonitrile, Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>/DMSO, 140 °C; (c) (i) HCl (g), EtOH, (ii) NH<sub>3</sub> (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In <a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>, the synthesis approach to five (<b>11a–e</b>; <b>DB1478</b>, <b>DB1481</b>, <b>DB1879</b>, <b>DB2005</b>, and <b>DB2529</b>) heteroaryl indole diamidines is presented. The 2-arylcyanofurans and thiophenes were obtained using Stille coupling between 2-(tri-<i>n</i>-butylstannyl)furan and 2-(tri-<i>n</i>-butylstannyl)thiophene with the indicated cyanoaryl bromides. NBS bromination of the 2-aryl furans and thiophenes at room temperature yielded the corresponding 5-bromo-2-aryl furans and thiophenes. Reaction of either the Boc-protected 5 or 6-cyano-2-(trimethlystannyl)indole with the 5-bromo-2-aryl furans or thiophenes yields the Boc-protected bis-nitriles. The bis-nitriles were converted into the diamidines by reaction with lithium bis(trimethylsilyl) amide.</div><figure id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Reagents and Conditions; (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Dioxane; (b) NBS, DMF, rt; (c) (i) LiN(TMS)<sub>2</sub>/THF, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthesis of the bis-<i>N</i>-(4-nitrophenylethyl) analogue (<b>12a</b>; <b>DB421</b>) of furamidine and the tellurophene analogue (<b>12b</b>; <b>DB1751</b>) of furamidine is presented in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a>. The former was obtained from the bis-nitrile by conversion into the corresponding imidate ester, which on reaction with 4-nitrophenethyl amine yielded <b>DB421</b>. Reaction of 2,5-bis-(4-cynophenyl)tellurophene with lithium bis(trimethylsilyl)amide provided <b>DB1751</b>.</div><figure id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Reagents and Conditions; (a) (i) HCl (g), EtOH (ii) 4-Nitrophenethylamine, EtOH; (b) (i) LiN(TMS)<sub>2</sub>, THF, rt, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The synthesis of the <i>N</i>-methylbenzimidazole analogue (<b>14</b>; <b>DB1314</b>) is shown in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The required bis-nitrile was obtained by oxidative coupling of 3-(methylamino)-4-aminobenzonitrile with 4-cyano-3′-formyl-1,1′-biphenyl using sodium metabisulfite as the oxidant. The mentioned lithium bis(trimethylsilyl)amide method was used to convert the bis-nitrile into the diamidine.</div><figure id="sch5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Reagents and Conditions: (a) 4-Amino-3-(methylamino)benzonitrile, Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>, DMF, Reflux. (b) (i) LiN(TMS)<sub>2</sub>, THF, rt (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The route utilized for synthesis of the quinoline analog (<b>18</b>; <b>DB1504</b>) is outlined in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. The Pfitzinger reaction between 2-acetylthiophene and 6-bromoisatin provided 6-bromo-2-(thiophen-2-yl)quinoline-4-carboxylic acid which on heating in the presence of copper(I) cyanide results in both decarboxylation and replacement of the bromo group with a cyano group. Once again, NBS bromination of the 2-aryl thiophene at room temperature yielded the desired 5-bromo-2-aryl thiophene analogue. Reaction of 4-cyanphenylboronic acid with the 5-bromo-2-aryl thiophene derivative under standard Suzuki coupling conditions provided the needed bis-nitrile. Again, the lithium bis(trimethylsilyl)amide methodology was employed to obtain the diamidine <b>DB1504</b>.</div><figure id="sch6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0014.jpeg" id="gr14" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Reagents and Conditions: (a) NaOH, H<sub>2</sub>O, 100 °C, Overnight; (b) CuCN, DMF, 140 °C (c) NBS, DMF, rt; (d) 4-Cyanophenylboronic Acid, 1,4-Dioxane, Pd(PPh<sub>3</sub>)<sub>4</sub>, 2 M K<sub>2</sub>CO<sub>3</sub>, 70 °C, Overnight; (e) (i) LiN(TMS)<sub>2</sub>, THF, rt, 48 h, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a> shows the approach used for the synthesis of the bis-<i>N</i>-methylbenzimidazole analogue (<b>19</b>; <b>DB2572</b>). In this case, the bis-aldehyde was oxidatively coupled, mediated by benzoquinone, with 3-amino-4-(methylamino)benzamidine to directly yield <b>DB2572</b>.</div><figure id="sch7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0015.jpeg" id="gr15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Reagents and Conditions: (a) 3-Amino-4-(methylamino)benzamidine, Benzoquinone, EtOH, Reflux</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">2.2.  Selection of HOXA9/DNA Binding Inhibitors</h3><div class="NLM_p">An ELISA-derived DNA binding inhibition assay was first used to evaluate this series of compounds for the inhibition of the HOXA9 protein interaction with its cognate DNA binding sequence 5′-aTGATTTA (HBS for HOXA9-binding site) which is 5′-biotinylated in order to be immobilized on streptavidin-coated 96-well plates. Specific (HBS) and nonspecific (NS) oligonucleotides (sequences in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) were used as competitor DNAs to validate the experimental conditions for selective protein/DNA inhibition experiments. The previously characterized AT-rich DNA ligand <b>DB75</b> and the ATGA-site DNA binding compound <b>DB293</b> were used as controls. Among those evaluated, 10 compounds showed significant activity on HOXA9/DNA binding (gray and white box for at least 50 and 75% inhibition, respectively) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) and were further evaluated using electrophoretic mobility shift assay (EMSA) (representative gels presented in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">S1</a>). HOXA9 protein expressed using a reticulocyte lysate system was incubated with radiolabeled HBS-double stranded oligonucleotide (sequence in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) and increasing concentrations of the tested compounds. The retarded band on the gel shows the formation of the HOXA9/HBS complex (bound, b), separated from the free HBS DNA (f). This conclusion was further validated by the addition of HOXA9-directed antibody (Ab) and by the absence of the retarded band using reticulocyte lysate prepared in the absence of the HOXA9-expressing vector (first and last lanes “0” from the upper left panel in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B). Quantification of the HOXA9/DNA inhibition from EMSA is presented in <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>C. This result demonstrated that <b>DB818</b>, <b>DB1055</b>, <b>DB884</b>, <b>DB1879</b>, and <b>DB2529</b> are highly potent inhibitors of HOXA9/DNA binding with more than 90% inhibition at 2.5 μM (“high” panel). <b>DB1177</b>, <b>DB883</b>, and <b>DB2223</b>, on the other hand, are less efficient (“medium” panel) with 90% inhibition reached using 7.5 μM. By contrast, <b>DB1620</b>, <b>DB1878</b>, and <b>DB911</b> are poor inhibitors (“low” panel), while others are considered to be inactive (right panel, “inactive”).</div><figure id="fig1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. In vitro selection of <b>DB818</b> and <b>DB1055</b> as HOXA9/DNA binding inhibitors. (A) ELISA-derived protein/DNA binding inhibition assay for selection of HOXA9/DNA binding inhibitors. HOXA9 protein was expressed from reticulocyte lysate and incubated with immobilized HBS-containing sequence (CTR). Free HBS or nonspecific (NS) DNA at 50× or 100× the quantity of immobilized HBS oligonucleotide or the various indicated DB compounds (5 or 10 μM) were then added. Black, dark gray, light gray, and white bars correspond to control, >50%, between 25 and 50% or <25% of HOXA9/DNA remaining complexes. Graph represents mean ± s.e.m. of up to 15 independent measurements. (B) EMSA gel. 5′-end <sup>32</sup>P-radiolabeled HBS DNA sequence (“f”, free DNA) was incubated with reticulocyte lysate expressing HOXA9 protein (“HOXA9”) to form a complex (bound “b”) and in the presence HOXA9-directed antibody (“Ab”) to generate a super-shifted (“Ss”) DNA band, or in the presence of increasing concentrations of the indicated DB compounds (μM). (C). Quantitative analysis of the inhibition of HOXA9/DNA complex formation from EMSA defines subgroups based on their HOXA9/DNA binding inhibition activity. The percentage of the bound HOXA9/DNA complex was quantified from EMSA in panel (B), relatively to the mean of the three control samples (100%) and plotted over drug concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In terms of the structure–activity relationship, <b>DB818</b>, <b>DB1879</b>, and <b>DB2529</b> are structurally related compounds that share a common central thiophene ring but differ by the presence or absence of a nitrogen atom that converts (i) the indole ring of <b>DB1879</b> to a benzimidazole ring in the <b>DB818</b> for R<sub>3</sub> position or (ii) a phenyl ring in R<sub>1</sub> position of <b>DB1879</b> and to a pyridine ring in <b>DB2529</b>. Neither modification alters their propensity to inhibit the HOXA9/DNA interaction. By contrast, changing the thiophene ring of <b>DB818</b>, <b>DB1879</b>, or <b>DB2529</b> to a furan (<b>DB293</b>, <b>DB1878</b>, or <b>DB915</b>, respectively) or a thiazole ring (<b>DB2651</b>) in the benzimidazole series strongly reduces HOXA9 inhibition efficacy, whereas a pyrazole ring modification still confers some weaker inhibitory activity to <b>DB2223</b>. Surprisingly, such modification of the thiophene ring to a thiazole ring in the di-phenyl series changes an inactive compound (<b>DB351</b>) to an effective (but low) inhibitor of HOXA9/DNA binding (<b>DB1620</b>). The addition of a methyl group on the benzimidazole ring (<b>DB818</b>) at the R<sub>3</sub> position or changing the <b>DB818</b> benzimidazole ring to a quinoline ring or to a <i>N</i>-isopropyl-benzimidazole-phenyl moiety strongly alters its propensity to inhibit the HOXA9/DNA interaction (see <b>DB2429</b>, <b>DB1504</b>, or <b>DB2708</b>) relative to <b>DB818</b>. A methyl group on the phenyl ring in R<sub>1</sub> also alters the inhibitory effect of <b>DB1878</b> (unmethylated) as shown using <b>DB2005</b> (methylated) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A). Moreover, the addition of a methyl group to the benzimidazole ring in the R<sub>3</sub> position alters the HOXA9 inhibition efficiency as observed with the highly active unmethylated compounds <b>DB818</b> and <b>DB2529</b>, relative to their inactive methylated counterparts <b>DB2429</b> and <b>DB2430</b> (of note, the latter also differs by the presence of a furan 5-membered ring in the R<sub>2</sub> position instead of a thiophene). Finally, the two pyrrole derivatives bearing an additional ring (phenyl for <b>DB884</b> or pyridine for <b>DB883</b>) at both the R<sub>1</sub>/R<sub>4</sub> positions are identified as high (<b>DB884</b>) and medium (<b>DB883</b>) inhibitors.</div><div class="NLM_p last">The results obtained from ELISA-derived DNA binding inhibition assay and EMSA are in good agreement and highlight <b>DB818</b>, <b>DB1055</b>, <b>DB883</b>, <b>DB1879</b>, and <b>DB2529</b> as interesting inhibitors of HOXA9/DNA binding. By contrast, <b>DB828</b>, that failed to inhibit HOXA9/DNA binding, was selected as a negative control for further analyses. It is also worth noting that neither <b>DB818</b> nor <b>DB1055</b> inhibits the DNA binding of the HMG-box protein, a protein known to interact with DNA on AT-rich sites (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S1B,C</a>).</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14">2.3.  Direct DNA Binding of the Selected HOXA9/DNA Inhibitors into the Groove of DNA</h3><div class="NLM_p">Thermal melting experiments allow a robust screening for the relative binding affinity of the ligands with DNA sequences. In order to understand ligand–DNA complex stability, we conducted thermal melting studies of different diamidine compounds with cognate HBS sequences. Ligand and HBS duplex DNA sequences were prepared at a ratio of 1:1 and 2:1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). At <i>R</i> = 1:1, <b>DB818</b>, <b>DB1055</b>, and <b>DB2529</b> showed Δ<i>T</i><sub>m</sub> values of 7–10 °C. The thermal stability of the complex increase at the <i>R</i> = 2:1 ligand–DNA ratio (Δ<i>T</i><sub>m</sub> > 10 °C) for <b>DB818</b>, <b>DB1055</b>, <b>DB2529</b>, <b>DB1879</b>, and <b>DB1878</b>. On the other hand, <b>DB884</b> and <b>DB1177</b> showed moderate binding with the HBS duplex (Δ<i>T</i><sub>m</sub> = 8–9 °C). However, <b>DB883</b>, <b>DB884</b>, <b>DB2223</b>, and <b>DB1620</b> bind very weakly with the cognate HBS sequence.</div><figure id="fig2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison and mode of binding to an oligonucleotide containing a HOXA9/DNA binding site. (A). Melting temperature studies on the HBS-containing DNA. Double-stranded HBS-containing oligonucleotides (0.5 μM) were incubated with 0.5 μM (<i>R</i> = 1:1) or 1 μM (<i>R</i> = 2:1) of the indicated compounds in TNE buffer prior to be subjected to the measurement of the DNA hyperchromic variation upon step by step heating using absorbance measurement at 260 nm. The Δ<i>T</i><sub>m</sub> values are calculated from <i>T</i><sub>m(DNA+DB)</sub> – <i>T</i><sub>m(DNA)</sub>. White and dashed bars highlight an increase of the <i>T</i><sub>m</sub> of more or less than 5 °C, respectively, whereas black bars are used for <b>DB828</b>, chosen as a control drug for further cellular analyses. (B) CD spectra is showing minor groove binding of duplex DNA. Double-stranded HBS-containing oligonucleotides (5 μM) were incubated in TNE buffer with increasing concentrations of the indicated compounds at increasing drug/oligonucleotide ratios <i>R</i>, evidencing a strong positively induced CD at the drug wavelength (0, 1, 2, 3, 4, 5, 6, 8, and 10 μM of drug, as indicated by arrows from min to max concentrations). Dashed lanes are CD spectra for oligonucleotides in the absence of compounds. Thick lanes highlight CD spectra at <i>R</i> = 1:1 and <i>R</i> = 2:1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To evaluate the binding mode of diamidine compounds with the same HOXA9-binding site, the circular dichroism (CD) spectral experiment was performed (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B). Neither the duplex DNA nor the free diamidines compound exhibits CD signals in the compound absorption region above 300 nm. Adding an increasing amount of diamidine compounds that evidenced in vitro positive results leads to substantial positively induced CD signals (ICD) for the complex formation with DNA at the wavelengths between 300 and 400 nm. This observation indicates that the diamidine compounds bind within the groove of the HBS DNA sequence, more likely within the minor groove as strongly evidenced in the crystal and/or NMR structure for <b>DB818</b>, <b>DB1055</b>, <b>DB884</b>, or <b>DB293</b> to 5′-AATT-containing DNA (PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VZK">1VZK</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5A">2I5A</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I2I">2I2I</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GYX">2GYX</a>).<a onclick="showRef(event, 'ref47 ref48 ref49'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49">(47−49)</a> On the other hand, <b>DB828</b> induces only weak CD changes at the highest drug concentrations in agreement with the low Δ<i>T</i><sub>m</sub>. All tested compounds except <b>DB1177</b> and <b>DB828</b> changed the positive ellipticity band of the DNA helix (positive CD band near 275 nm), suggesting that they induce modification of the stacking of the successive bases of the DNA.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">2.4.  HOXA9/DNA Inhibitors Directly Bind to the HOXA9-Cognate Sequence</h3><div class="NLM_p">To define the specific drug/DNA binding sequence, DNaseI footprinting experiments were used for selected compounds from the “high” (<b>DB818</b>, <b>DB1055</b>, <b>DB1879</b>, <b>DB2529</b>), “medium” (<b>DB1177</b>, <b>DB883</b>), “low” (<b>DB1878</b>), and “inactive” (<b>DB828</b>) subgroups for HOXA9/DNA binding inhibition. As can be seen from <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A, DNase I-generated DNA fragments were separated upon electrophoretic migration from top to bottom. Some of the cleavage sites were protected from DNaseI digestion around the 50 bp position with the addition of increasing concentrations of selected diamidine compounds. The protected site was clearly evidenced on the corresponding densitometric analysis located at the HOXA9 binding site 5′-ATTTA for compounds from the “high” subgroup <b>DB818</b>, <b>DB1055</b>, <b>DB1879</b>, and <b>DB2529</b> (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). By contrast, <b>DB828</b> and <b>DB883</b> failed to recognize the 5′-ATTTA sequence, whereas <b>DB1878</b> and <b>DB1177</b> have a much weaker binding activity to the 5′-ATTTA site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). These results are in agreement with the DNA melting temperature studies on the HBS sequence (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). We used the same UV/vis DNA melting approach to evaluate the binding of <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> on the mutated HBS-oligonucleotide sequences. We have mutated the sequence at the 5′-ATGA (HBS-ATGAm) or the AT-rich (HBS-ATm) or containing both mutations (HBS-2mut) (oligonucleotide sequences are presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>). For the HBS-ATGAm sequence, the Δ<i>T</i><sub>m</sub> values obtained with <b>DB818</b> and <b>DB1055</b> were similar to those obtained with wild-type HBS oligonucleotide, which suggests that the diamidine compounds do not bind at the 5′-ATG binding site of HBS (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). By contrast, when the 5′-ATTTA site was mutated alone for HBS-ATm or together with the ATGA portion for HBS-2mut, Δ<i>T</i><sub>m</sub> values decreased by 4 °C, which is similar to the observed value for the <b>DB828</b>. From the previous experiment (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A), it has been observed that <b>DB828</b> binds with HBS as a weak nonspecific binder. This result is in agreement with the DNaseI footprinting experiments which suggest that, among others, <b>DB818</b> and <b>DB1055</b> interact at the 5′-ATTTA-sequence of the HOXA9 binding site (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) but did not bind to an HBS-mutated DNA as shown in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S2</a>. This sequence specific binding evidenced <b>DB818</b> and <b>DB1055</b> as competitors for HOXA9 binding to its cognate sequence on the DNA helix.</div><figure id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Evaluation of the drug interaction to the HOXA9 consensus DNA binding site. (A) DNaseI footprinting gels. Increasing concentrations of the indicated compounds (μM) were incubated with a 3′-end <sup>32</sup>P-radiolabeled HBS-containing DNA fragment prior to DNaseI mild-digestion to then be subjected to denaturing gel electrophoresis. “G” lanes, G-track localizes guanines within the radiolabeled DNA sequence. “0” lanes, DNA fragment without the drug. The gray line locates the HOXA9-binding site (HBS). (B) Densitometric analyses from DNaseI footprinting gels. Each band from lanes generated in the presence of various concentrations of the indicated compounds was quantified regarding each identical base from the untreated lanes (“0”) to visualize the differential cleavage at the HBS sequence (gray lines) or not depending on the evaluated compounds from gels in A. (C) Melting temperature studies on ATGA- and/or AT-mutated sites from the HBS-containing DNA. Experiments were performed at an equimolar drug/DNA ratio (<i>R</i> = 1:1) in similar conditions as in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A but with oligonucleotides mutated on the ATGA sequence (ATGAm), the AT-rich site (ATm), or both (2Mut) (sequences presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>). Statistic, Student <i>t</i>-test: ns, <i>p</i> > 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Surface plasmon resonance (SPR) experiments were applied to measure the dissociation constants <i>K</i><sub>D</sub> for <b>DB818</b> or <b>DB1055</b> on HBS or HBS-2mut sequences (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). <b>DB818</b> and <b>DB1055</b> have similar strong <i>K</i><sub>D</sub> values on the HBS sequence (4.6 and 5.5 nM, respectively). The dissociation constant is much greater on the mutated sequence (0.35 and 1.7 μM, respectively), highlighting that <b>DB818</b> is ∼75-fold more selective for HBS than for the HBS-mutated sequence, whereas <b>DB1055</b> is even more selective with >300-fold difference in <i>K</i><sub>D</sub> values (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic and affinity analysis of <b>DB818</b> and <b>DB1055</b> binding to HBS and HBS-mut DNA sequences using BIAcore technology. (A) SPR sensorgrams for the evaluation of the interaction kinetics of <b>DB818</b> and <b>DB1055</b> with the HBS (“ATGATTTA”) or HBS-mut (“ATGACCTA”) DNA sequences in P20-TNE binding buffer. The black lines are best fit curves determined by kinetic fitting. The ligand concentrations are indicated in the graphs from bottom to top. The surface interaction of the diamidines at high concentrations causes the drift in the sensorgrams. (B) Steady-state fitting curves of <b>DB818</b> and <b>DB1055</b> on the HBS or HBS-mut DNA hairpin sequences. A stoichiometric value of one was used for data fitting based on the maximum RU values for the different sequences that were similar to those for netropsin binding to the AATT sequence used as chip control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To gain structural insight into the mechanism by which the <b>DB818</b> and related diamidines compounds inhibit HOXA9 binding to the cognate HBS, we performed a 100 ns molecular dynamics (MD) simulation (MD) of the <b>DB818</b> binding to the ATGATTTA sequence (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">S3</a>, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S2</a>). The HBS-bound complex showed that <b>DB818</b> is selectively bound to the minor groove of the AT-rich 5′-A<sub>1</sub>T<sub>2</sub>T<sub>3</sub>T<sub>4</sub>A<sub>5</sub>/5′-T<sub>5′</sub>A<sub>4′</sub>A<sub>3′</sub>A<sub>2′</sub>T<sub>1′</sub> binding site and makes excellent van der Waals interactions with the walls of the minor groove (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>A,B). Two amidine groups, −N–Hs, form H-bond either with T═O of dT<sub>1′</sub> base or A–N3 of dA<sub>5</sub> base (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>C–D) that are an average of 2.3 Å in length. The third strong H-bond is formed with the T═O of dT<sub>4</sub> base to the H–<i>N</i>-benzimidazole of <b>DB818</b> with 2.0 Å bonding length, to account for much of the binding selectivity of <b>DB818</b> (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). Additional selectivity in binding is provided by the −CH group of the 6-member ring of benzimidazole that points deeply into the minor groove and forms a dynamic close interaction with A–N3 of the dA<sub>4′</sub> base of the A·T bp. Interaction of the phenyl ring with the dT base further stabilizes the <b>DB818</b>/DNA interaction and increases sequence selectivity. The sum of these numerous relatively stabilizing interactions marks <b>DB818</b> a strong ATTTA sequence binder within the minor groove which may explain how <b>DB818</b> inhibits the binding of the HOXA9 protein at the major groove and minor groove of its cognate binding site (see the <a class="ref internalNav" href="#sec3" aria-label="Conclusions">Conclusions</a> section).</div><figure id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MD model of <b>DB818</b> bound to ds[(5′-CCATGATTTACG-3′) (5′-CGTAAATCATGG-3′)]. (A,B) Ball-stick (DNA) and space-filling (ligand) model viewed into the minor groove (A) and major groove (B) of the ATTTA binding site with bound <b>DB818</b>. The DNA bases are represented in the tan-white-red-blue-yellow (C–H–O–N–P) color scheme and <b>DB818</b> is represented in the light purple-white-blue-yellow (C–H–N–S) color scheme. (C,D) Important interactions between different sections of the <b>DB818</b>/DNA complex are presented for the phenyl-amidine (C) and benzimidazole-amidine (D) moieties. Interactions with the indicated bases of the HBS sequence are schematized by yellow dashed lines and the corresponding distances are indicated above (in angstrom).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">2.5.  HOXA9/DNA Binding Inhibitors Are Effective at Entering Cells and Localizing to the Nucleus</h3><div class="NLM_p">In order to address the cellular consequences of HOXA9 inhibitors, we used MigA9 cells, resulting from retroviral transduction of the murine Hoxa9 gene in murine bone marrow cells that were injected in sublethally irradiated mice and then collected from the bone marrow after leukemia burden and cultured in appropriate medium as described.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> We first evaluated the cytotoxic effect of a series of diamidine compounds. Cell survival activity (IC<sub>50</sub>) on MigA9 cells was measured at 72 h of treatment by using a colorimetric assay based on the reduction of a MTS tetrazolium dye. The IC<sub>50</sub> values of diamidine compounds are at the micromolar range, except that <b>DB828</b> which has very high IC<sub>50</sub> and supports previous in vitro observations. Surprisingly, <b>DB2223</b>, as an effective in vitro inhibitor of the HOXA9/DNA binding complex, failed to alter MigA9 cell survival (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). We also evaluated their susceptibilities to enter cells and localize in the nucleus by using fluorescence microscopy as previously described for <b>DB75</b> and its derivatives.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> Most of the diamidines have intrinsic fluorescence properties in blue wavelengths. Compounds were evaluated either on the MigA9 murine leukemia cell line (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">S4</a>) and/or HT-29 adherent cell line (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S5</a>). All evaluated compounds entered the nucleus except <b>DB911</b> which could not be detected because of the lack of intrinsic fluorescence properties. <b>DB828</b>, which also localized to the nucleus but failed to inhibit HOXA9 DNA binding, was used as a negative control in the following cellular studies.</div><figure id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cellular analysis of HOXA9 transcription factor inhibition by <b>DB818</b> and <b>DB1055</b>. (A) Subcellular distribution of <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> using cell fluorescence microscopy. GFP-positive MigA9 cells (in green) were treated with 20 μM of the indicated compounds (in blue) for 2 h. The nucleus was counterstained using TO-PRO-3 or propidium iodide (PI) staining of DNA (in red). Merged images are presented on the right. Scale bars correspond to 5 μm. <sup>1</sup>: Images acquired with a confocal Zeiss LSM880 microscope. GFP (Ex 488 nm, Em 495–551 nm), TO-PRO-3 (Ex 633 nm, Em 640–735 nm), <b>DB818</b> (Ex 405 nm, Em 445–485 nm), <b>DB828</b> (Ex 405 nm, Em 425–465 nm). <sup>2</sup>: Images acquired with a widefield Leica DMi8 microscope. GFP (Ex BP480/40, Em BP527/30), PI (Ex BP546/10, Em BP585/40), DB (Ex BP350/50, Em BP460/50). (B) Inhibition of HOXA9 transcriptional control by <b>DB818</b> and <b>DB1055</b> evidenced by luciferase assays. K562 cells were co-transfected with pGL3-prom reporter vector containing 6-repeated HBS sequences as WT sequence (“HBS”) or mutated sequence (“HBSmut”) ± pCMV-AD-HOXA9 (“HOXA9”) prior to addition of the indicated DB compounds or vehicle. Graphs represent mean ± s.e.m. of 3 to 8 independent points. Statistic, Student <i>t</i>-test: *, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Cell Survival and Cell Death<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center" char="±">% PI-positive cells</th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compound</th><th class="colsep0 rowsep0" align="center">MTS IC<sub>50</sub> 72 h (μM)</th><th class="colsep0 rowsep0" align="center">conc. (μM)</th><th class="colsep0 rowsep0" align="center" char="±">24 h</th><th class="colsep0 rowsep0" align="center" char="±">48 h</th><th class="colsep0 rowsep0" align="center">% subG1 48 h</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB818</b></td><td class="colsep0 rowsep0" align="left">9.97 ± 2.40</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="±">6.95 ± 0.48</td><td class="colsep0 rowsep0" align="char" char="±"><b>29.40</b><b>±</b><b>3.29</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>9.46 ± 1.91</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±"><b>13.50</b><b>±</b><b>1.37</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>64.03</b><b>±</b><b>3.25</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>14.58 ± 1.30</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char="±"><b>21.07</b><b>±</b><b>1.25</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>75.53</b><b>±</b><b>0.60</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>12.00 ± 0.90</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1055</b></td><td class="colsep0 rowsep0" align="left">14.09 ± 1.69</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±">3.87 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">5.72 ± 0.06</td><td class="colsep0 rowsep0" align="left">1.27 ± 0.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char="±">5.17 ± 0.35</td><td class="colsep0 rowsep0" align="char" char="±">9.45 ± 0.52</td><td class="colsep0 rowsep0" align="left">1.65 ± 0.37</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1878</b></td><td class="colsep0 rowsep0" align="left">6.34 ± 1.41</td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="±">8.72 ± 0.35</td><td class="colsep0 rowsep0" align="char" char="±"><b>44.93</b><b>±</b><b>1.03</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>15.94 ± 1.80</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±">11.76 ± 1.88</td><td class="colsep0 rowsep0" align="char" char="±"><b>52.73</b><b>±</b><b>17.16</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="left"><b>27.88 ± 16.14*</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1879</b></td><td class="colsep0 rowsep0" align="left">3.58 ± 0.92</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char="±">7.42 ± 0.19</td><td class="colsep0 rowsep0" align="char" char="±"><b>77.87</b><b>±</b><b>0.32</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>33.64 ± 15.95</b><sup><b>#</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="±"><b>23.83</b><b>±</b><b>0.58</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>98.17</b><b>±</b><b>0.20</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB2529</b></td><td class="colsep0 rowsep0" align="left">2.24 ± 0.22</td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="char" char="±">6.21 ± 0.34</td><td class="colsep0 rowsep0" align="char" char="±"><b>39.40</b><b>±</b><b>12.96</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="left"><b>4.48 ± 0.95</b><sup><b>†</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="char" char="±"><b>13.57</b><b>±</b><b>1.19</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>96.03</b><b>±</b><b>0.77</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB883</b></td><td class="colsep0 rowsep0" align="left">3.87 ± 0.17</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="char" char="±">5.19 ± 0.10</td><td class="colsep0 rowsep0" align="char" char="±">8.01 ± 0.12</td><td class="colsep0 rowsep0" align="left">1.23 ± 0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±"><b>14.43</b><b>±</b><b>0.83</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>17.60</b><b>±</b><b>1.06</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">1.04 ± 0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB884</b></td><td class="colsep0 rowsep0" align="left">2.14 ± 0.26</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="char" char="±"><b>12.90</b><b>±</b><b>0.40</b><sup><b>#</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>18.33</b><b>±</b><b>0.77</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>2.47 ± 1.02*</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="char" char="±"><b>68.80</b><b>±</b><b>1.06</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="char" char="±"><b>99.33</b><b>±</b><b>0.15</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1620</b></td><td class="colsep0 rowsep0" align="left">23.93 ± 0.86</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±">5.69 ± 0.21</td><td class="colsep0 rowsep0" align="char" char="±">10.45 ± 1.50</td><td class="colsep0 rowsep0" align="left">0.88 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char="±">5.91 ± 0.15</td><td class="colsep0 rowsep0" align="char" char="±"><b>22.90</b><b>±</b><b>2.36</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>1.71 ± 0.06</b><sup><b>†</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB2223</b></td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±">8.64 ± 0.42</td><td class="colsep0 rowsep0" align="char" char="±">6.48 ± 0.11</td><td class="colsep0 rowsep0" align="left">0.65 ± 0.10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char="±">10.43 ± 0.69</td><td class="colsep0 rowsep0" align="char" char="±">6.72 ± 0.18</td><td class="colsep0 rowsep0" align="left">0.67 ± 0.17</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB828</b></td><td class="colsep0 rowsep0" align="left">>50</td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="char" char="±">8.89 ± 0.32</td><td class="colsep0 rowsep0" align="char" char="±">7.32 ± 0.03</td><td class="colsep0 rowsep0" align="left">0.95 ± 0.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="char" char="±">8.38 ± 0.45</td><td class="colsep0 rowsep0" align="char" char="±">7.77 ± 0.10</td><td class="colsep0 rowsep0" align="left">0.92 ± 0.19</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">untreated</td><td class="colsep0 rowsep0" align="char" char="±">9.06 ± 0.53</td><td class="colsep0 rowsep0" align="char" char="±">8.75 ± 0.23</td><td class="colsep0 rowsep0" align="left">0.76 ± 0.16</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">Cell survival is determined by using the MTS measurements (<i>n</i> > 3) on the Hoxa9-expressing murine MigA9 cell line. IC<sub>50</sub> corresponds to the concentration of the compound associated with the presence of 50% cell survival relatively to untreated controls, measured by using MTS reagent. “>50” means that IC<sub>50</sub> was not reached, even at the maximum amount concentration of 50 μM used in the experiments. Cell death is established from PI-labeling of MigA9 dead cells (PI-positive) or as sub-G1 population from PI-labeling of fixed MigA9 cells treated for 24 (PI-positive cells) or 48 h (PI-positive cells and subG1 quantification) with the indicated concentrations of evaluated compounds. Results are expressed as mean values or mean percentage ± s.e.m. (<i>n</i> = 3–7). Statistic, Student <i>t</i>-test: *, <i>p</i> < 0.05; <sup>#</sup>, <i>p</i> < 0.01; <sup>†</sup>, <i>p</i> < 0.001; <sup>$</sup>, <i>p</i> < 0.0001; na, not applicable due to high proportion of dead cells.</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">2.6.  <b>DB818</b> and <b>DB1055</b>, but Not <b>DB828</b>, Inhibit HOXA9-Dependant Transcription Control</h3><div class="NLM_p last">In order to address the inhibitory effect of <b>DB818</b> and <b>DB1055</b> on the HOXA9-mediated transcriptional control, we performed luciferase assays. We used a pGL3-promoter vector in which the minimal promoter is under the control of 6-repeated HBS sequences, either the wild-type or mutated sequence at the 5′-ATTTA binding site (HBS and HBS-mut oligonucleotides, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>). Cell transfection was performed in the K562 cell line as a leukemic cell model that is known for its lack of internal HOXA9 expression.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> The absence of HOXA9 expression has been validated by using quantitative reverse transcription-polymerase chain reaction (qRT-PCR) and western-blotting, personal information and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S6A</a>, respectively. In this cell line, <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> compounds have no effect on cell survival and clonogenic propensities (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S5B</a>, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>, top lanes). Extrinsic HOXA9 protein was expressed by transfecting the pCMV-AD-HOXA9 vector in the K562 HOXA9-negative cells (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S6A</a>). HOXA9 protein has only low propensity to transactivate by itself in the absence of co-factors. Therefore, the region encoding the transactivation domain (AD) of NF-κB was fused in frame to the HOXA9 full length open-reading frame in order to express the HOXA9-AD fused protein that presents increased transactivation properties relatively to HOXA9 alone (personal information, see also <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>). Luciferase assays were performed in the presence versus absence of evaluated compounds <b>DB818</b> and <b>DB1055</b> as HOXA9/DNA binding inhibitors, or <b>DB828</b> as a negative control. The results showed that both <b>DB818</b> and <b>DB1055</b>, but not <b>DB828</b>, were able to inhibit HOXA9-mediated transactivation in the presence of wild-type HBS sequences by controlling luciferase expression but not in the presence of HBS-mut sequences (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). These results also suggested that the HBS binding site is required for selective inhibition of HOXA9 transcriptional activity.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">2.7.  Gene Expression Analysis of MigA9 Cells Treated with <b>DB818</b> Highlights Induced Cell Death Processes</h3><div class="NLM_p">In order to investigate the cellular mechanistic action of selected diamidine HOXA9/DNA inhibitors, we initially used a global gene expression analysis of MigA9 cells which were treated with <b>DB818</b> (2.5 μM) for 6, 12, and 24 h. Co-deregulated genes with |mean fold changes| ≥ 1.5 are presented as a heatmap in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A (|mean fold changes| ≥ 1.5). GSEA analyses on MSigDB C2-chemical and genetic perturbation gene sets (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B) evidenced that the significant genes upregulated in <b>DB818</b>-treated MigA9 cells at the different times correspond to genes that were downregulated in human CD34+ cells transduced with NUP98-HOXA9 fusion (TAKEDA TARGETS OF NUP98 HOXA9 FUSION 8D DN), <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B. Similarly, the list of significant commonly downregulated genes in <b>DB818</b>-treated MigA9 cells are highlighted in geneset enrichments corresponding to genes upregulated in murine Hoxa9 + Meis1 transformed cells (WANG IMMORTALIZED BY HOXA9 AND MEIS1 UP). Both results of CGP genesets enrichments suggest that the gene deregulation is in an opposite way between the transformation of hematopoietic progenitors with Hoxa9 + Meis1 or NUP98-HOXA9 and the inhibition of HOXA9 activity by <b>DB818</b> in MigA9. Other genesets associated with HOXA9 expression control (NUYTTEN EZH2 TARGETS UP),<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> genes controlled by HOXA9 (GABRIELY MIR21 TARGETS),<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> leukemic stem cell inhibition (GAL LEUKEMIC STEM CELL DN), and AML-induced differentiation (MARTENS TRETINOIN RESPONSE DN) were also found as significantly enriched in our dataset (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>B). Ingenuity pathway analysis (IPA) of deregulated genes highlights the activation of macrophage differentiation, a reduced quantity of hematopoietic cells, and an increase in apoptosis (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>C). Analysis of the BloodSpot database (<a href="http://www.bloodspot.eu" class="extLink">www.bloodspot.eu</a>) found multiple commonly deregulated genes (identified as arrows in <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>A) which are significantly associated with the differentiation of granulocyte/monocyte progenitors (GMP) to granulocytes and monocytes (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S7</a>). Among them, the well-known monocyte/macrophage expressing the integrin Itgam (CD11b) is expressed upon differentiation of AML blasts<a onclick="showRef(event, 'ref54 ref55'); return false;" href="javascript:void(0);" class="ref ref54 ref55">(54,55)</a> and the Oasl2 enzyme that is also associated with the differentiation of monocytes and dendritic cells.<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> Such overexpression of Itgam and Oasl2 was validated by qRT-PCR after 48 h of treatment with <b>DB818</b> (5 or 10 μM), as was validated the overexpression of Akap13 and Arfgef1 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D) as examples of genes associated with granulocyte and/or monocyte differentiation (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S7</a>). Similar results of overexpression of these genes were also observed for <b>DB1055</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D), but <b>DB828</b> failed to achieve the similar results. Recently, Sun et al. (2018) identified murine Hoxa9 DNA binding sites by ChIP-seq analysis of a murine leukemia model obtained by transformation of murine progenitor cells by co-expression of hemagglutinin (HA)-tagged HOXA9 and MEIS1 using an anti-HA antibody.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> Among Hoxa9 ChIP-peaks, the sequence identified as “DE-651 peak” located at position chr7:82861417–82863417 (from NCBI37/mm9 mouse reference genome) is positioned within the Akap13 gene. Interestingly, we identified a cognate HOXA9 binding sequence within the DE-651 peak sequence (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>A). An oligonucleotide overlapping this HBS site (Akap13-HBS, <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) was synthesized and radiolabeled to then be evaluated using EMSA for HOXA9 binding and <b>DB818</b> or <b>DB1055</b> inhibitory effect. As evidenced in <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>B, HOXA9 interacts with this sequence and such HOXA9/DNA binding is inhibited by the addition of increasing concentrations of <b>DB818</b> or <b>DB1055</b>. Such an interaction and its inhibition are a possible option to explain HOXA9-mediated control of Akap13 expression, assuming that HOXA9 binding is associated with an inhibition of Akap13 expression. <b>DB818</b> and <b>DB1055</b> would abolish such negative control of Akap13 expression by HOXA9, leading to an overexpression of Akap13 as evidenced by qRT-PCR (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>D) for this gene which overexpression is associated with granulocyte and monocyte differentiation (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S7</a>).</div><figure id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Transcriptomic analysis of MigA9 treated with 2.5 μM of <b>DB818</b> for 6, 12, or 24 h. (A) Commonly deregulated genes with |mean fold changes| ≥ 1.5 are presented as heatmap for upregulated genes (top panel) or downregulated genes (bottom panel). Arrows identified genes up- or downregulated during hematopoietic differentiation from progenitors to granulocyte/monocyte differentiated cells as presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S7</a>. (B) GSEA analyses of upregulated or downregulated gene expression data on MSigDB C2-chemical and genetic perturbation genesets. (C) Cellular networks (“diseases and functions” analysis in hematological systems) deduced from IPA of genes that were significantly deregulated (adjusted <i>p</i>-values ≤ 0.05) by <b>DB818</b> treatment of MigA9 cells with |fold-changes| ≥ 1.5. <i>Z</i>-score, the number of genes (“# genes”), their names, and the corresponding up (north east arrow) or down (south east arrow) expression changes are indicated. (D) Quantification of deregulated genes using qRT-PCR on total mRNA from MigA9 cells treated with <b>DB818</b>, <b>DB1055</b>, or <b>DB828</b> (5 or 10 μM) for 48 h, as 6 independent replicates. Each gene expression was normalized relatively to that of Gapdh (see primers in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) and given as mean ± s.e.m. Statistics (Student <i>t</i>-test): *, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001; ****, <i>p</i> < 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Evaluation of <b>DB818</b> and <b>DB1055</b> inhibition of HOXA9 binding to murine Akap13 gene regulatory region. (A) DNA sequence of the DE-651 peak, located at position chr7:82861417–82863417 from NCBI37/mm9 mouse reference genome (within Akap13 gene) as identified by Sun et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> from ChIP-seq analysis using a hemagglutinin antibody to trap the HOXA9 binding site from the murine leukemia model obtained by transformation of murine progenitor cells by co-expression of hemagglutinin (HA)-tagged HOXA9 and MEIS1. (B) EMSA gel. 5′-end The <sup>32</sup>P-radiolabeled AKAP13-HBS DNA sequence (“f”, free DNA) was incubated with reticulocyte lysate expressing HOXA9 protein (“HOXA9”) or not (“0”) to form a complex (bound “b”) and in the presence of increasing concentrations of <b>DB818</b> or <b>DB1055</b> compounds (μM). ns, non-specific band.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As highlighted above using the global list of deregulated genes, IPA verified two main biological processes that were altered following <b>DB818</b> treatment on MigA9 cells: cell differentiation (upregulated) and cell proliferation (downregulated). Based on this in silico transcriptomic analysis, cellular analyses were then performed to address the impact of <b>DB818</b> and its derivatives on the cell proliferation and differentiation in MigA9 cells.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20">2.8.  Inhibition of MigA9 Cell Proliferation</h3><div class="NLM_p">MTS measurements (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) showed that most of the diamidines (except <b>DB828</b> and <b>DB2223</b>) reduced the cell survival of the Hoxa9-transformed cell line MigA9. Comprehensive changes in cell survival, determined by using MTS may be due to multiple cellular processes such as cell death, cell cycle arrest, and cell differentiation processes among others. Consequently, we then further addressed the impact of diamidine compound treatment on each of those cellular processes to have a better insight into the mechanism of action of those molecules. First, we validated the induction of cell death by <b>DB818</b>, as highlighted in transcriptomic analysis, using both the quantification of the proportion of propidium iodide-positive cells (PI<sup>+</sup>) and of subG1 population, two markers of cell death addressed from cell cycle analysis. The results in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> show an increase in the PI-positive population for both 24 and 48 h treatment and for the subG1 proportion at 48 h treatment with <b>DB818</b>. These results suggest that <b>DB818</b> induces apoptotic cell death, in agreement with transcriptomic analyses that highlighted expression of gene sets associated with a decrease of the quantity of blood cells upon treatment with <b>DB818</b> (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Using the same experimental approaches, we showed that <b>DB1878</b>, <b>DB1879</b>, and <b>DB2529</b>, as close derivatives of <b>DB818</b>, also strongly induce cell death (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>). Interestingly, the five ring containing compounds <b>DB883</b> and <b>DB884</b> were the most active in terms of cell death, suggesting that cell death is the main cellular process that is responsible for the decrease of cell survival by <b>DB883</b> and <b>DB884</b>. By contrast, <b>DB1055</b> does not induce cell death after 24 or 48 h treatment to explain its effect on global cell survival. The impact of drug treatment on the cell cycle resulted in a strong reduction in the proportion of MigA9 cells in S-phase upon treatment with <b>DB818</b> and derivatives (<b>DB1878</b>/1879/2529), as well as with <b>DB1055</b> and <b>DB1620</b> (<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>). This reduction of cycling cells is associated with a strong increase in the G0/G1 phase (<b>DB1055</b>, <b>DB1878</b>, and <b>DB2529</b>), in the G2/M phase (<b>DB818</b>) or in both phases (<b>DB1879</b>, <b>DB1620</b>) at the tested concentrations and at 48 h treatment.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Cell Cycle<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">treatment</th><th class="rowsep1 colsep0" colspan="3" align="center">cell cycle</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">Cpd</th><th class="colsep0 rowsep0" align="center">conc. (μM)</th><th class="colsep0 rowsep0" align="center">G0/G1</th><th class="colsep0 rowsep0" align="center"><i>S</i></th><th class="colsep0 rowsep0" align="center">G2/M</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">none</td><td class="colsep0 rowsep0" align="left">untreated</td><td class="colsep0 rowsep0" align="left">48.77 ± 0.19</td><td class="colsep0 rowsep0" align="left">46.68 ± 0.21</td><td class="colsep0 rowsep0" align="left">4.55 ± 0.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB818</b></td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">50.38 ± 0.77*</td><td class="colsep0 rowsep0" align="left">35.42 ± 1.57<sup>$</sup></td><td class="colsep0 rowsep0" align="left"><b>14.19 ± 1.50</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">58.30 ± 3.71<sup>#</sup></td><td class="colsep0 rowsep0" align="left"><b>14.80 ± 3.59</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>26.90 ± 1.96</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">56.02 ± 2.41<sup>#</sup></td><td class="colsep0 rowsep0" align="left"><b>9.25 ± 2.09</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>34.73 ± 0.93</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1055</b></td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left"><b>61.49 ± 0.19</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>34.13 ± 0.93</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">4.38 ± 0.20</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>70.98 ± 0.90</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>23.91 ± 1.29</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">5.11 ± 0.38</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1878</b></td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left"><b>87.28 ± 0.44</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>8.02 ± 0.24</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">4.70 ± 0.58</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left"><b>85.95 ± 5.39</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>10.06 ± 5.92</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">3.99 ± 1.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1879</b></td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><b>77.92 ± 1.89</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>9.15 ± 2.81</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>12.93 ± 4.68*</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB2529</b></td><td class="colsep0 rowsep0" align="left">5</td><td class="colsep0 rowsep0" align="left"><b>67.95 ± 1.78</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>22.55 ± 1.80</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>9.50 ± 0.09</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">10</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB883</b></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">48.44 ± 0.35</td><td class="colsep0 rowsep0" align="left">46.06 ± 0.58</td><td class="colsep0 rowsep0" align="left">5.50 ± 0.27*</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">50.15 ± 0.19 *</td><td class="colsep0 rowsep0" align="left">43.51 ± 0.95<sup>#</sup></td><td class="colsep0 rowsep0" align="left">6.34 ± 0.23<sup>†</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB884</b></td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">53.83 ± 1.07<sup>$</sup></td><td class="colsep0 rowsep0" align="left">38.84 ± 1.18<sup>$</sup></td><td class="colsep0 rowsep0" align="left"><b>7.32 ± 0.30</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td><td class="colsep0 rowsep0" align="left">na</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB1620</b></td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">54.87 ± 2.31 <sup>#</sup></td><td class="colsep0 rowsep0" align="left">38.65 ± 2.94<sup>#</sup></td><td class="colsep0 rowsep0" align="left">6.47 ± 0.65<sup>#</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>64.53 ± 1.69</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>27.33 ± 1.99</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left"><b>8.14 ± 0.30</b><sup><b>$</b></sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB2223</b></td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left">52.29 ± 0.63<sup>$</sup></td><td class="colsep0 rowsep0" align="left">43.65 ± 0.65<sup>†</sup></td><td class="colsep0 rowsep0" align="left">4.06 ± 0.33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left">53.56 ± 0.53<sup>$</sup></td><td class="colsep0 rowsep0" align="left">43.02 ± 0.80<sup>†</sup></td><td class="colsep0 rowsep0" align="left">3.42 ± 0.27<sup>#</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>DB828</b></td><td class="colsep0 rowsep0" align="left">15</td><td class="colsep0 rowsep0" align="left"><b>58.66 ± 0.38</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">37.22 ± 0.54<sup>$</sup></td><td class="colsep0 rowsep0" align="left">4.12 ± 0.22</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">20</td><td class="colsep0 rowsep0" align="left"><b>62.21 ± 0.20</b><sup><b>$</b></sup></td><td class="colsep0 rowsep0" align="left">33.86 ± 0.22<sup>$</sup></td><td class="colsep0 rowsep0" align="left">3.93 ± 0.11*</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">Hoxa9-expressing murine MigA9 cells were collected after 48 h treatment with the indicated concentrations of evaluated compounds or control cells (<i>n</i> = 3–6) and further analyzed by cytometry for the cell cycle using EdU and PI staining on fixed cells. All results are expressed as mean percentage ± s.e.m. Statistic, Student <i>t</i>-test: *, <i>p</i> < 0.05; <sup>#</sup>, <i>p</i> < 0.01; <sup>†</sup>, <i>p</i> < 0.001; <sup>$</sup>, <i>p</i> < 0.0001; na, not applicable due to high proportion of dead cells (PI-positive cells, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>).</p></div></div></div><div class="NLM_p">Anti-clonogenic activities were then performed by using <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> to address cell death and cell differentiation processes. This experiment first highlights the impact of both <b>DB818</b> and <b>DB1055</b> on global clonogenic propensity of the HOXA9-positive MigA9 cell line, but not on the HOXA9-negative K562 cell line (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S6C,D</a>). Surprisingly, the intrinsic fluorescence of <b>DB818</b> and <b>DB828</b> were still visible even after 10 days of culture in Methocult using fluorescence microscopy with DAPI-filters, which indicates very high stability of the diamidine compounds within the cells (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S6D</a>). Results for clonogenic assays clearly show reduction in the number of colony subtypes obtained in the presence of <b>DB818</b> or <b>DB1055</b>, but not <b>DB828</b> (<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a>).</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Anti-clonogenic Activities<a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">colony assay</th><th class="colsep0 rowsep0" align="center">type of colonies</th><th class="colsep0 rowsep0" align="center">CTR</th><th class="colsep0 rowsep0" align="center"><b>DB818</b></th><th class="colsep0 rowsep0" align="center"><b>DB1055</b></th><th class="colsep0 rowsep0" align="center"><b>DB828</b></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>K562</b> number</td><td class="colsep0 rowsep0" align="left">total</td><td class="colsep0 rowsep0" align="left">445 ± 28</td><td class="colsep0 rowsep0" align="left">448 ± 17</td><td class="colsep0 rowsep0" align="left">467 ± 12</td><td class="colsep0 rowsep0" align="left">nd</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">total</td><td class="colsep0 rowsep0" align="left">148 (100%)</td><td class="colsep0 rowsep0" align="left">101 (100%)</td><td class="colsep0 rowsep0" align="left">115 (100%)</td><td class="colsep0 rowsep0" align="left">170 (100%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>MigA9</b> number (%)</td><td class="colsep0 rowsep0" align="left">cfu-GM</td><td class="colsep0 rowsep0" align="left">64 (43.2%)</td><td class="colsep0 rowsep0" align="left"><b>23 (22.8%)</b></td><td class="colsep0 rowsep0" align="left"><b>0 (0%)</b></td><td class="colsep0 rowsep0" align="left">90 (52.9%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cfu-G</td><td class="colsep0 rowsep0" align="left">49 (33.1%)</td><td class="colsep0 rowsep0" align="left"><b>55 (54.5%)</b></td><td class="colsep0 rowsep0" align="left"><b>58 (50.4%)</b></td><td class="colsep0 rowsep0" align="left">62 (36.5%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">cfu-M</td><td class="colsep0 rowsep0" align="left">28 (18.9%)</td><td class="colsep0 rowsep0" align="left">20 (19.8%)</td><td class="colsep0 rowsep0" align="left"><b>55 (47.8%)</b></td><td class="colsep0 rowsep0" align="left">16 (9.4%)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">BFU-E</td><td class="colsep0 rowsep0" align="left">7 (4.7%)</td><td class="colsep0 rowsep0" align="left">3 (3%)</td><td class="colsep0 rowsep0" align="left">2 (1.7%)</td><td class="colsep0 rowsep0" align="left">2 (1.2%)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">MigA9 cells (1000 cells/well) were seeded in duplicates in complete M3434 Mouse Colony-Forming Cell Assays MethoCult in the presence or absence (CTR) of 2.5 μM of the indicated compounds and plated in 6-well plates for 10 days prior to quantification of clones and further identification of the individual cfu-GM, cfu-M and cfu-G colonies on morphological criteria. na: not applicable; nd: not determined. K562 cells (1000 cells/well) were treated in the same conditions, plated on H4535 (human) Colony-Forming Cell Assays MethoCult and counted (mean ± s.e.m.). Data that are statistically different from the untreated control are presented in bold letters.</p></div></div></div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23">2.9.  Induction of Granulocyte/Monocyte Differentiation</h3><div class="NLM_p last">Finally, using the same anti-clonogenic experiment, we proved that the global decrease in the colony number is associated with a reduced proportion of cfu-GM (granulocyte and monocyte) in favor of an increase of cfu-M (monocyte only) and cfu-G (granulocyte only). These results suggest that both <b>DB818</b> and <b>DB1055</b> induced a decrease in the number of the most immature hematopoietic progenitors cells (cfu-GM) and favored the development of colonies that are more engaged in granulocytic or monocytic differentiation (cfu-G, cfu-M). This result is in agreement with gene expression analyses with <b>DB818</b> (GSEA, IPA analyses), confirmed by qRT-PCR for both <b>DB818</b> and <b>DB1055</b> (but not for <b>DB828</b>) on Itgam (CD11b) as well as on the CD14 expression as two markers of monocytic to macrophagic differentiation. Altogether, these molecular and cellular experiments highlight the leukemic cell differentiation process associated with treatment with the HOXA9/DNA binding inhibitors <b>DB818</b> and <b>DB1055</b>.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24">3.  Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_63856" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_63856" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In the present work, we selected a series of heterocyclic diamidines and derivatives as sequence-selective DNA ligands that directly inhibit HOXA9/DNA binding (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A) through groove binding (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) on the HOXA9 cognate DNA sequence (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>). Particularly, we identified <b>DB818</b>, <b>DB1055</b>, <b>DB884</b>, <b>DB1879</b>, and <b>DB2529</b> as strong inhibitors of HOXA9/DNA binding using both an ELISA-derived DNA binding inhibition assay (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>A) and EMSA with more than 90% inhibition at 2.5 μM (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>B–C). Those compounds also strongly stabilize the DNA helix based on DNA melting temperature results (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A). <b>DB818</b>, <b>DB1055</b>, and <b>DB2529</b> were the most efficient compounds based on results obtained at the lower drug/DNA ratio (<i>R</i> = 1:1). The recognition occurs at the HOXA9 cognate DNA sequence as demonstrated using DNaseI footprinting experiments (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>), through groove binding, as shown using CD (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B), pointing deeply in the minor groove and making close contacts with AT bases of the HBS sequence as evidenced using molecular modeling studies (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). DNase I footprinting experiments also evidenced that <b>DB818</b> and <b>DB1055</b> appeared to be very active and selective compounds on binding to the HOXA9-binding site (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) but not to the mutated HBS sequence (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S2</a>). Moreover, <b>DB818</b> and <b>DB1055</b> failed to inhibit DNA binding of the HMG-box protein used as a specificity control (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S1B,C</a>). We therefore selected <b>DB818</b> and <b>DB1055</b> as highly potent inhibitors that compete with HOXA9 for binding to its cognate sequence. <b>DB818</b> and <b>DB1055</b> were previously shown to interact with AT-rich DNA sequences<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> with binding <i>K</i><sub>D</sub> values of 9 and 17 nM, respectively. Our <i>K</i><sub>D</sub> measurements obtained with the HBS-containing DNA sequence highlight a stronger interaction with values of 4.6 and 5.5 nM, respectively (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>). The binding of <b>DB818</b> and <b>DB1055</b> to HBS requires the presence of the 5′-ATTTA portion of HBS because mutation of the HBS site on this AT-rich site abolished binding of <b>DB818</b> and <b>DB1055</b> to the DNA. More precisely, the interaction of <b>DB818</b> or <b>DB1055</b> in the minor groove of DNA (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>) is an interesting point regarding the mode of DNA binding of the HOXA9 transcription factor that implies the correct orientation of three alpha-helices of the HOXA9 homeodomain. The HOXA9 helix-3 binds in the major groove of the DNA helix, whereas the amino-terminal part of its homeodomain deeply bound in the minor groove of the AT-rich portion of the HOXA9-binding site.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This interaction with the minor groove is associated with direct contact of Arg-5 with A<sub>2′</sub> on the lower strand and T<sub>3</sub> on upper strand of the HOXA9/DNA binding site (upper strand/lower strand: A<sub>1</sub>T<sub>2</sub>T<sub>3</sub>T<sub>4</sub>A<sub>5</sub>/5′-T<sub>5′</sub>A<sub>4′</sub>A<sub>3′</sub>A<sub>2′</sub>T<sub>1′</sub>). Water-mediated additional contacts of Arg-5 with A<sub>3′</sub> of the lower strand, and water-mediated contacts of Arg-2 with both A<sub>4′</sub> and T<sub>5′</sub> on lower strand reinforce the DNA binding.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> This AT-rich portion is the exact position where <b>DB818</b> and <b>DB1055</b> are lying in the minor groove, which allows them to block the correct positioning of the HOXA9 homeodomain to its cognate site. HOXA9 binding to the DNA is associated with a DNA bending of ∼20° resulting from minor groove compression in order to allow deep contacts of helix-3 to an enlarged major groove on the opposite side. Such DNA distortion toward the minor groove results from interaction of amino acids of the amino-terminal arm (Pro-7 and Tyr-8) with helix-1 of the homeodomain of HOXA9 (Thr-13). Binding of <b>DB818</b> and <b>DB1055</b> to the HOXA9 binding site may also not be compatible with minor groove compression by helix-3 binding in the major groove and may enhance HOXA9/DNA binding inhibition by <b>DB818</b>/<b>1055</b>. At the cellular level, both <b>DB818</b> and <b>DB1055</b> control HOXA9-directed transcription based on luciferase gene expression assays (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B) and induce, among other derivatives, cell death, cell cycle arrest, and cell differentiation in a model of the Hoxa9-transformed murine MigA9 leukemia cell line (<a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>–<a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>, and <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">S6</a>). Interestingly, <b>DB828</b>, used here as a control as an unselective minor groove DNA binding drug that failed to interact with the HOXA9 cognate sequence (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>) and to inhibit the HOXA9/DNA interaction in vitro (based on ELISA and EMSA experiments, <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) and in vivo (based on luciferase assays, <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). <b>DB828</b> also failed to alter cell survival (MTS, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) to induce cell death (PI-positive or sub-G1 populations analysis, <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a>) and MigA9 cell differentiation (anti-clonogenic activities and CD11b/CD14 differentiation markers expression, <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> and <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>).</div><div class="NLM_p">Up to now, no direct HOXA9/DNA binding inhibitors have been discovered. Only indirect inhibition strategies have been developed. The first approach is by targeting proteins of the MLL complex that controls HOXA9 gene expression. This is the case with inhibitors of DOT1L such as EPZ-5676 (pinometostat, currently in a phase I clinical trial) that reduce HOXA9 gene expression and induce cell differentiation<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58,59)</a> or of inhibitors of the menin/MLL interaction.<a onclick="showRef(event, 'ref60 ref61'); return false;" href="javascript:void(0);" class="ref ref60 ref61">(60,61)</a> The second approach is the targeting of the HOX/PBX interface of multiple HOX proteins with their co-factors of the PBX family by the HXR9 peptidomimetic developed against a variety of solid tumors and hematopoietic malignancies.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a></div><div class="NLM_p">Some other heterocyclic diamidines were previously identified as transcription factors inhibitors, <b>DB293</b> against Pit-1 and Brn-3,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a><b>DB1255</b> against ERG,<a onclick="showRef(event, 'ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref44 ref45">(44,45)</a> and <b>DB2113</b> against PU.1,<a onclick="showRef(event, 'ref22 ref63'); return false;" href="javascript:void(0);" class="ref ref22 ref63">(22,63)</a> but none were previously evaluated against the HOXA9 transcription factor.</div><div class="NLM_p last">Identification of <b>DB818</b> and <b>DB1055</b> as HOXA9/DNA binding competitors and inducers of granulocyte/monocyte differentiation offers an exciting new opportunity to target the HOXA9 transcription factor. HOXA9 is one of the master regulators of the differentiation blockade in AML,<a onclick="showRef(event, 'ref28 ref29 ref32 ref36'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref32 ref36">(28,29,32,36)</a> being at the crossroads of many cell signaling perturbations resulting from well-characterized mutations or translocations defining AML subgroups such as for instance MLL translocations, MYST3-CREBBP or NUP98-NSD1 fusion proteins, and NPM1 or EZH2 mutations.<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30,31)</a> Therefore, treatment of such AML subtypes with <b>DB818</b>/<b>DB1055</b> needs to be further evaluated in the future to offer new potential differentiation therapies. HOXA9 is also associated with different solid tumor models such as glioblastoma and gastric and prostate cancers<a onclick="showRef(event, 'ref23 ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26">(23−26)</a> against which <b>DB818</b>, <b>DB1055</b>, and/or derivatives might also be evaluated to propose future therapeutic strategies. This DNA targeting approach to inhibit HOXA9/DNA binding and function is a very promising strategy with the potential to develop new anti-cancer treatments and particularly new differentiation therapies in leukemia.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">4.  Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87771" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87771" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26">4.1.  Chemistry</h3><div class="NLM_p">References for previously reported diamidines are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Supporting Information</a>. The synthesis of the previously unreported diamidines follows below and in the <a class="ref internalNav" href="#sec2" aria-label="Results and Discussion">Results and Discussion</a> section. Representatives <sup>1</sup>H NMR spectra for the diamidines are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Supporting Information</a>. The purity (≥95%) of all compounds evaluated in this study was established by elemental analysis and <sup>1</sup>H high resolution NMR spectroscopy. Elemental Analysis was performed by Atlantic Microlab Inc., Norcross, GA.</div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> General Procedure for the Synthesis of Compounds <b>1a–c</b></h4><div class="NLM_p">To a stirred solution of the bromo compounds<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> (10 mmol), 5-(dimethoxymethyl)furan-2-yl-tri-<i>n</i>-butylstannane<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> (10 mmol) in 30 mL anhydrous 1,4-dioxane was added tetrakis-triphenylphosphine palladium (300 mg). The stirred mixture was warmed to 90–100 °C for 16 h. The solvent was removed under reduced pressure; the residue was extracted using 200 mL of dichloromethane (DCM) from aqueous solution containing 3 mL of concentrated ammonia, then washed with water, and passed through celite. The organic layer was evaporated and the oily residue was mixed with 15 mL 2.0 M HCl, and stirred for 2 h, the separated precipitate was filtered off and washed with ethanol. The product was crystallized from ethanol.</div><div id="sec4_1_1_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i28" class="anchor-spacer"></div><h5 class="article-section__title" id="_i28">4.1.1.1.  3-Benzyloxy-4-(5-formylfuran-2-yl)-benzonitrile (<b>1a</b>)</h5><div class="NLM_p last">73% yield, mp 164–165 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 5.38 (s, 2H), 7.16 (d, <i>J</i> = 3.6 Hz, 1H), 7.40–7.46 (m, 3H), 7.51–7.56 (m, 3H), 7.61 (d, <i>J</i> = 3.6 Hz, 1H), 7.80 (s, 1H), 8.00 (d, <i>J</i> = 8.1 Hz, 1H), 9.63 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 303 (M<sup>+</sup>, 40), 275 (25), 198 (10), 91 (100).</div></div><div id="sec4_1_1_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i29" class="anchor-spacer"></div><h5 class="article-section__title" id="_i29">4.1.1.2.  6-(5-Formylfuran-2-yl)nicotinonitrile (<b>1b</b>)</h5><div class="NLM_p last">Yield 60%, mp 204–205 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 7.56 (d, <i>J</i> = 3.6 Hz, 1H), 7.71 (d, <i>J</i> = 3.6 Hz, 1H), 8.07 (d, <i>J</i> = 8.4 Hz, 1H), 8.44 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 9.09 (d, <i>J</i> = 2.1 Hz, 1H), 9.72 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 199 (M<sup>+</sup> + 1, 100).</div></div><div id="sec4_1_1_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i30" class="anchor-spacer"></div><h5 class="article-section__title" id="_i30">4.1.1.3.  5-(5-Formylfuran-2-yl)picolinonitrile (<b>1c</b>)</h5><div class="NLM_p last">Yield 62%, mp 191–192 °C (EtOH). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 7.66 (d, <i>J</i> = 3.6 Hz, 1H), 7.74 (d, <i>J</i> = 3.6 Hz, 1H), 8.17 (d, <i>J</i> = 8.1 Hz, 1H), 8.46 (dd, <i>J</i> = 8.1, 2.1 Hz, 1H), 9.25 (d, <i>J</i> = 2.1 Hz, 1H), 9.71 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 198 (M<sup>+</sup>, 100), 197 (70), 141 (30).</div></div><div id="sec4_1_1_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i31" class="anchor-spacer"></div><h5 class="article-section__title" id="_i31">4.1.1.4.  2-{5-[(2-Benzyloxy)-4-cyanophenyl]-furan-2-yl}-1<i>H</i>-benzimidazole-6-carbonitrile (<b>2a</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 81%, mp 247–248.5 °C (DMF). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 5.39 (s, 2H), 7.18 (d, <i>J</i> = 3.6 Hz, 1H), 7.40–7.49 (m, 4H), 7.55–7.67 (m, 4H), 7.74–7.79 (m, 2H), 8.14 (s, 1H), 8.30 (d, <i>J</i> = 8.0 Hz, 1H), 13.0 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 417 (M<sup>+</sup> + 1, 100), 397 (10), 323 (60).</div></div><div id="sec4_1_1_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32">4.1.1.5.  2-[5-(5-Cyanopyridin-2-yl)furan-2-yl]-1<i>H</i>-benzimidazole-6-carbonitrile (<b>2b</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 74%, mp > 300 °C (DMF). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 7.51 (d, <i>J</i> = 3.6 Hz, 1H), 7.56 (d, <i>J</i> = 3.6 Hz, 1H), 7.64 (d, <i>J</i> = 8.1 Hz, 1H), 7.78 (d, <i>J</i> = 8.1 Hz, 1H), 8.16 (d, <i>J</i> = 8.1 Hz, 2H), 8.49 (dd, <i>J</i> = 8.1, 2.1 Hz, 1H), 9.06 (d, <i>J</i> = 2.1 Hz, 1H), 13.70 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 312 (M<sup>+</sup> + 1, 100).</div></div><div id="sec4_1_1_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33">4.1.1.6.  2-[5-(6-Cyanopyridin-3-yl)furan-2-yl]-1<i>H</i>-benzimidazole-6-carbonitrile (<b>2c</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 75%, mp 279–281 °C (DMF). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 7.50 (d, <i>J</i> = 3.6 Hz, 1H), 7.64 (d, <i>J</i> = 3.6 Hz, 1H), 7.65–7.80 (m, 2H), 8.10–8.21 (m, 2H), 8.50 (dd, <i>J</i> = 8.1, 1.8 Hz, 1H), 9.32 (d, <i>J</i> = 1.8 Hz, 1H), 13.72 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 312 (M<sup>+</sup> + 1, 100).</div></div><div id="sec4_1_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34">4.1.1.7.  <i>N</i>-Hydroxy-2-{5-[(2-benzyloxy)-4-(<i>N</i>-hydroxyamidino)phenyl]-furan-2-yl}-1<i>H</i>-benzimidazole-6-amidine (<b>3a</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> Yield 94%, mp 253–255 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 5.35 (s, 2H), 5.86 (s, 2H), 5.97 (s, 2H), 7.05 (d, <i>J</i> = 3.6 Hz, 1H), 7.29 (d, <i>J</i> = 3.6 Hz, 1H), 7.40–7.81 (m, 10H), 8.16 (d, <i>J</i> = 8.1 Hz, 1H), 9.61 (s, 1H), 9.80 (s, 1H), 13.11 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 483 (M<sup>+</sup> + 1, 10), 465 (5), 432 (5), 392 (55), 375 (100).</div></div><div id="sec4_1_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35">4.1.1.8.  <i>N</i>-Hydroxy-2-{5-[5-(<i>N</i>-hydroxyamidino)pyridin-2-yl]furan-2-yl}-1<i>H</i>-benzimidazole-6-amidine (<b>3b</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 91%, mp > 300 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 5.89 (s, 2H), 6.08 (s, 2H), 7.38 (d, <i>J</i> = 3.6 Hz, 1H), 7.40 (d, <i>J</i> = 3.6 Hz, 1H), 7.53–7.66 (m, 2H), 7.99–8.06 (m, 2H), 8.22 (dd, <i>J</i> = 8.4, 2.4 Hz, 1H), 8.95 (d, <i>J</i> = 2.4 Hz, 1H), 9.64 (s, 1H), 9.97 (s, 1H), 13.22 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 378 (M<sup>+</sup> + 1, 80), 286 (10), 256 (18), 240 (100).</div></div><div id="sec4_1_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36">4.1.1.9.  <i>N</i>-Hydroxy-2-{[5-[6-(<i>N</i>-hydroxyamidino)pyridin-3-yl]furan-2-yl]}-1<i>H</i>-benzimidazole-6-amidine (<b>3c</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 90%, mp 285–287 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 5.88 (s, 2H), 5.92 (s, 2H), 7.38 (d, <i>J</i> = 3.6 Hz, 1H), 7.41 (d, <i>J</i> = 3.6 Hz, 1H), 7.55–7.63 (m, 2H), 7.83–7.99 (m, 2H), 8.29 (dd, <i>J</i> = 8.4, 2.1 Hz, 1H), 9.16 (d, <i>J</i> = 2.1 Hz, 1H), 9.63 (s, 1H), 10.05 (s, 1H), 13.17 (s, 1H). MS (<i>m</i>/<i>z</i>, rel. int.): 378 (M<sup>+</sup> + 1, 40), 362 (8), 101 (100).</div></div><div id="sec4_1_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37">4.1.1.10.  2-[5-(4-Amidino-2-hydroxyphenyl)furan-2-yl]-1<i>H</i>-benzimidazole-6-amidine acetate (<b>4a</b>, <b>DB928</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 78%, mp 277–279 °C. <sup>1</sup>H NMR (D<sub>2</sub>O/DMSO-<i>d</i><sub>6</sub>): δ 1.78 (s, 3 × CH<sub>3</sub>), 7.18–7.43 (m, 4H), 7.52–7.75 (m, 2H), 8.04 (s, 2H). ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>6</sub>O<sub>2</sub>, 361.1413; found, 361.1410 (M<sup>+</sup> + 1). Anal. Calcd for C<sub>19</sub>H<sub>16</sub>N<sub>6</sub>O<sub>2</sub>–3.0AcOH–1.9H<sub>2</sub>O: C, 52.24; H, 5.57; N, 14.63. Found: C, 52.04; H, 5.29; N, 14.77.</div></div><div id="sec4_1_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38">4.1.1.11.  2-[5-(5-Amidinopyridin-2-yl)furan-2-yl]-1<i>H</i>-benzimidazole-6-amidine Acetate (<b>4b</b>, <b>DB915</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 84%, mp 221–223 °C. <sup>1</sup>H NMR (D<sub>2</sub>O/DMSO-<i>d</i><sub>6</sub>): δ 1.79 (s, 2.3 × CH<sub>3</sub>), 7.33 (s, 1H), 7.43 (s, 1H), 7.50–7.70 (m, 2H), 8.00–8.10 (m, 2H), 8.29 (s, 1H), 8.97 (s, 1H). ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>7</sub>O, 346.1416; found, 346.1425 (M<sup>+</sup> + 1). Anal. Calcd for C<sub>18</sub>H<sub>15</sub>N<sub>7</sub>O–2.3AcOH–3.1H<sub>2</sub>O: C, 50.32; H, 5.68; N, 18.17. Found: C, 50.02; H, 5.74; N, 18.36.</div></div><div id="sec4_1_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39">4.1.1.12.  2-[5-(6-Amidinopyridin-3-yl)furan-2-yl]-1<i>H</i>-benzimidazole-6-amidine Acetate (<b>4c</b>, <b>DB995</b>)</h5><div class="NLM_p last">It is prepared adopting reported methodology.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Yield 70%, mp 222–224 °C. <sup>1</sup>H NMR (D<sub>2</sub>O/DMSO-<i>d</i><sub>6</sub>): δ 1.87 (s, 3 × CH<sub>3</sub>), 7.38 (s, 1H), 7.50–7.69 (m, 3H), 8.08 (s, 1H), 8.30–8.45 (m, 2H), 9.23 (s, 1H). ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>7</sub>O, 346.1416; found, 346.1426 (M<sup>+</sup> + 1). Anal. Calcd for C<sub>18</sub>H<sub>15</sub>N<sub>7</sub>O–3.0AcOH–2.0H<sub>2</sub>O–0.25EtOH: C, 51.34, H, 5.70, N, 17.10. Found: C, 51.53, H, 5.54, N, 16.70.</div></div><div id="sec4_1_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40">4.1.1.13.  4-(2-Formylthiazol-5-yl)benzonitrile (<b>5</b>)</h5><div class="NLM_p last">Na<sub>2</sub>CO<sub>3</sub> (20 mmol) in water (5 mL) and 4-cyanophenylboronic acid (11 mmol) methanol (10 mL) were added to a stirred solution of 5-bromothiazole-2-carbaldehyde (10 mmol) in dioxane (30 mL), and the mixture was deaerated under nitrogen for 20 min. Tetrakistriphenylphosphine palladium (0.46 g, 0.4 mmol) was added and the reaction mixture was vigorously stirred at 100 °C for 24 h. The solvent was evaporated under reduced pressure, and the solid was partitioned between ethyl acetate (200 mL) and 2 M aqueous Na<sub>2</sub>CO<sub>3</sub> (25 mL) containing 5 mL of concentrated ammonia, to remove palladium residues, then washed with water, passed through Celite to remove the catalyst, dried (sodium sulfate), and evaporated. The product was purified using column chromatography on silica gel with hexanes/ethyl acetate as an eluent. Brown solid (1.09 g, 51%), mp 155–156 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 10.02 (s, 1H), 8.93 (s, 1H), 8.23 (d, <i>J</i> = 7.8 Hz, 2H), 7.98 (d, <i>J</i> = 7.8 Hz, 2H); ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>11</sub>H<sub>7</sub>N<sub>2</sub>OS, 215.0274; found, 215.0272 (M<sup>+</sup> + 1).</div></div><div id="sec4_1_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i41" class="anchor-spacer"></div><h5 class="article-section__title" id="_i41">4.1.1.14.  2-(5-(4-Cyanophenyl)thiazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carbonitrile (<b>6</b>)</h5><div class="NLM_p last">Sodium metabisulphite (6 mmol) was added to a solution of 3,4-diaminobenzonitrile (3 mmol) and 4-(2-formylthiazol-5-yl)benzonitrile (3 mmol) in DMSO (10 mL), and the mixture was heated at 140 °C for 30 min. The reaction mixture was poured into water, filtered, and dried. Purification was by crystallization from acetone. Yellow solid (0.52 g, 54%), mp > 300 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 8.66 (br s, 1H), 8.29 (m, 3H), 8.00 (d, <i>J</i> = 6.4 Hz, 2H), 7.69 (m, 2H); ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>10</sub>N<sub>5</sub>S, 328.0651; found, 328.0641 (M<sup>+</sup> + 1).</div></div><div id="sec4_1_1_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42">4.1.1.15.  2-(5-(4-Carbamimidoylphenyl)thiazol-2-yl)-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carboximidamide Trihydrochloride (<b>7</b>, <b>DB2651</b>)</h5><div class="NLM_p last">The bis-nitrile (1 mmol) was suspended in dry ethanol (20 mL) and cooled in an ice bath, and HCl gas was bubbled through the reaction mixture for 30 min and the reaction flask was tightly sealed and stirred at room temperature for 14 days. The yellow precipitate of imidate ester hydrochloride was filtered and washed with dry ethanol and anhydrous ether and dried under vacuum at room temperature for 3 h. The imidate ester was suspended in dry ethanol and cooled in an ice bath, and ammonia gas was bubbled through the solution for 30 min and the reaction flask was tightly sealed and stirred at room temperature for 2 days. The formed yellow precipitate was filtered and washed with dry ethanol and acetone and dried under vacuum at 100 °C for 12 h. Yellow solid (0.18 g, 37%), mp > 300 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 9.58 (s, 2H), 9.48 (s, 2H), 9.33 (s, 2H), 9.24 (s, 2H), 8.73 (br s, 1H), 8.37 (d, <i>J</i> = 8 Hz, 2H), 8.24 (br s, 1H), 8.06 (d, <i>J</i> = 7.6 Hz, 2H), 7.85 (d, <i>J</i> = 8.4 Hz, 1H), 7.76 (d, <i>J</i> = 8.4 Hz, 1H); ESI-HRMS: <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>7</sub>S, 362.1182; found, 362.1177 (amidine base M<sup>+</sup> + 1). Anal. Calcd for C<sub>18</sub>H<sub>15</sub>N<sub>7</sub>S·3HCl·2H<sub>2</sub>O·0.1C<sub>3</sub>H<sub>6</sub>O: C, 42.87; H, 4.44; N, 19.12. Found: C, 43.07; H, 4.26; N, 18.82.</div></div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43">4.1.2.  Synthesis of 2-(4-Cyanoaryl)furans and Thiophenes (<b>8a–e</b>)</h4><div class="NLM_p">Tetrakistriphenylphosphine palladium (0.576 g, 0.5 mmol) was added to a stirred mixture of the 2(tributylstannyl)furan or thiophene, (10 mmol) and the bromoarylbenzonitriles (10 mmol) in deaerated dioxane (25 mL) under nitrogen. The reaction flask was heated at 100 °C for 12 h. The solvent was concentrated under reduced pressure, and the resulting paste was stirred with ether and filtered. The solid obtained was purified by column chromatography on silica gel, using hexanes/ethyl acetate as eluent.</div><div class="NLM_p">The following compounds were previously reported: 4-(furan-2-yl)benzonitrile <b>8a</b>,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> 6-(furan-2-yl)nicotinonitrile <b>8b</b>,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> 4-(thiophen-2-yl)benzonitrile <b>8c</b>,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> 6-(thiophen-2-yl)nicotinonitrile <b>8d</b>.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div><div id="sec4_1_2_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44">4.1.2.1.  2-(2-Methoxy-4-cyanophenyl)furan (<b>8e</b>)</h5><div class="NLM_p last">White solid, yield (1.61 g, 81%). mp 54–54.5 °C; <sup>1</sup>H NMR (CDCl<sub>3</sub>): δ 7.95 (dd, 1H, <i>J</i> = 8, 2 Hz), 7.54 (br s, 1H), 7.46 (d, 1H, <i>J</i> = 8 Hz), 7.19 (br s, 1H), 7.13 (br s, 1H), 6.56 (dd, 1H, <i>J</i> = 3.2, 2 Hz), 4 (s, 3H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): δ 149.2, 143.1, 137, 134.6, 130.8, 130.1, 129.6, 127.7, 118.6, 113.6, 111.2, 53.2; ESI-MS: <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>9</sub>NO<sub>2</sub>, 199.2; found, 201.1 (M<sup>+</sup> + 2); Anal. Calcd for C<sub>12</sub>H<sub>9</sub>NO<sub>2</sub>: C, 72.35; H, 4.55; N, 7.03. Found: C, 72.41; H, 4.71; N, 6.97.</div></div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45">4.1.3.  Synthesis of 2-Bromo-5-(4-cyanoaryl)furans and Thiophenes (<b>9a–e</b>)</h4><div class="NLM_p"><i>N</i>-Bromosuccinimide (2.13 g, 12 mmol) was added portionwise to a stirred solution of the nitriles <b>8a–e</b> (10 mmol) in dimethylformamide (20 mL) in an ice bath. The reaction mixture was stirred overnight at room temperature and then poured onto ice water, and the obtained precipitate was collected and dried. Purification was done by column chromatography on silica gel, using hexanes/ethyl acetate as eluent.</div><div class="NLM_p">The following compounds were previously reported: 4-(5-bromofuran-2-yl)benzonitrile <b>9a</b>,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> 6-(5-bromofuran-2-yl)nicotinonitrile <b>9b</b>,<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> 4-(5-bromothiophen-2-yl)benzonitrile <b>9c</b>,<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> 6-(5-bromothiophen-2-yl)nicotinonitrile <b>9d</b>.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a></div><div id="sec4_1_3_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46">4.1.3.1.  2-Bromo-5-(2-methoxy-4-cyanophenyl)furan (<b>9e</b>)</h5><div class="NLM_p last">Yellow solid, yield (2.41 g, 87%). mp 71–71.5 °C; <sup>1</sup>H NMR (CDCl<sub>3</sub>): δ 7.89 (br s, 1H), 7.33–7.19 (m, 3H), 6.47 (br s, 1H), 3.99 (s, 3H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): δ 155.1, 150.9, 126, 125.3, 122.7, 119.1, 116, 115.8, 115.7, 114.9, 111, 56.8; ESI-MS: <i>m</i>/<i>z</i> calcd for C<sub>12</sub>H<sub>8</sub>BrNO<sub>2</sub>, 278.1; found, 279.2 (M<sup>+</sup> + 1); Anal. Calcd for C<sub>12</sub>H<sub>8</sub>BrNO<sub>2</sub>: C, 51.83; H, 2.90; N, 5.04. Found: C, 51.59; H, 2.76; N, 5.14.</div></div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47">4.1.4.  General Procedure for the Synthesis of the Dinitriles (<b>10a–e</b>)</h4><div class="NLM_p">A mixture of the heteroaryl bromides <b>9a–e</b> (0.01 mol) and <i>tert</i>-butyl 5 or 6-cyano-2-(trimethylstannyl)-1<i>H</i>-indole-1-carboxylate<a onclick="showRef(event, 'ref20 ref28'); return false;" href="javascript:void(0);" class="ref ref20 ref28">(20,28)</a> 4.06 g (0.01 mol) in 75 mL anhydrous dioxane with Pd(PPh<sub>3</sub>)<sub>4</sub> 0.23 g (0.0002 mol) (under nitrogen) was heated under reflux for 12–24 h [thin layer chromatography (TLC) monitored]. The solvent was removed under reduced pressure, and the resultant solid was filtered, washed with hexane, and dried in air. The solid was often a mixture of N-Boc and non-Boced indoles. The solid was suspended in DCM (100 mL); di-<i>t</i>-butyl dicarbonate 2.18 g (0.01 mol) and 10 mg DMAP were stirred for 12 h at rt and diluted with 50 mL water; organic layer was separated, dried over MgSO<sub>4</sub>, filtered, concentrated, and triturated with hexane; and the yellow solid was filtered, yields 61–82%.</div><div id="sec4_1_4_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48">4.1.4.1.  <i>t</i>-Butyl 5-Cyano-2-(5-(4-cyanophenyl)furan-2-yl)-1<i>H</i>-indole-1-carboxylate (<b>10a</b>)</h5><div class="NLM_p last">Yield of 3.27 g (80%), mp > 300 °C dec; <sup>1</sup>H NMR (CDCl<sub>3</sub>): 8.33 (d, 1H, <i>J</i> = 8.4 Hz), 7.94 (d, 1H, <i>J</i> = 1.6 Hz), 7.80 (d, 2H, <i>J</i> = 8.0 Hz), 7.70 (d, 2H, <i>J</i> = 8.0 Hz), 7.63 (dd, 1H, <i>J</i> = 1.6, 8.4 Hz), 6.96 (d, 1H, <i>J</i> = 3.6 Hz), 6.90 (s, 1H), 6.78 (d, 1H, <i>J</i> = 3.6 Hz), 1.47 (s, 9H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): 152.4, 149.2, 147.3, 139.3, 134.2, 132.9, 131.4, 128.7, 128.3, 125.9, 124.0, 119.7, 118.9, 116.4, 113.0, 111.2, 110.9, 109.7, 106.8, 85.3, 27.9; HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>19</sub>N<sub>3</sub>O<sub>3</sub>Na, 432.1324 (M<sup>+</sup> + 1); found, 432.1340.</div></div><div id="sec4_1_4_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49">4.1.4.2.  <i>t</i>-Butyl-5-cyano-2-(5-(5-cyanopyridin-2-yl)furan-2-yl)-1<i>H</i>-indole-1-carboxylate (<b>10b</b>)</h5><div class="NLM_p last">Yield of the yellow solid 2.9 g (72%), mp > 300 °C dec; <sup>1</sup>H NMR (CDCl<sub>3</sub>): 8.60 (d, 1H, <i>J</i> = 1.2 Hz), 8.32 (d, 1H, <i>J</i> = 8.4 Hz), 7.99 (dd, 1H, <i>J</i> = 2.4, 8.4 Hz), 7.94 (br s, 1H), 7.79 (d, 1H, <i>J</i> = 8.4 Hz), 7.63 (dd, 1H, <i>J</i> = 1.2, 8.4 Hz), 7.37 (d, 1H, 3.6 Hz), 6.94 (br s, 1H), 6.84 (d, 1H, <i>J</i> = 3.6 Hz), 1.47 (s, 9H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): 152.9, 152.5, 151.4, 149.1, 148.7, 140.1, 139.3, 131.1, 128.6, 128.5, 126.0, 119.6, 117.9, 117.0, 116.5, 113.9, 113.5, 111.8, 107.5, 106.9, 85.5, 27.9; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>18</sub>N<sub>4</sub>O<sub>3</sub>Na, 433.1277 (M<sup>+</sup> + Na); found, 433.1311.</div></div><div id="sec4_1_4_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50">4.1.4.3.  <i>t</i>-Butyl 6-Cyano-2-(5-(4-cyanophenyl)thiophen-2-yl)-1<i>H</i>-indole-1-carboxylate (<b>10c</b>)</h5><div class="NLM_p last">Yield of the yellow solid 3.37 g (74%), mp > 300 °C dec; <sup>1</sup>H NMR (CDCl<sub>3</sub>): 8.55 (d, 1H, <i>J</i> = 1.2 Hz), 7.74–7.68 (m, 5H), 7.64 (d, 1H, <i>J</i> = 8.0 Hz), 7.53 (d, 1H, <i>J</i> = 1.2, 8.0 Hz), 7.44 (d, 1H, <i>J</i> = 3.6 Hz), 7.21 (d, 1H, <i>J</i> = 3.6 Hz), 1.51 (s, 9H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): 149.2, 142.3, 138.1, 136.5, 135.7, 135.6, 132.9, 132.0, 129.8, 126.4, 126.0, 124.9, 121.5, 120.2, 120.0, 118.8, 112.0, 111.1, 107.7, 85.6, 27.8; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>25</sub>H<sub>19</sub>N<sub>3</sub>O<sub>2</sub>SNa, 448.1096 (M<sup>+</sup> + 1); found, 448.1081.</div></div><div id="sec4_1_4_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51">4.1.4.4.  <i>t</i>-Butyl-6-cyano-2-(5-(5-cyanopyridin-2-yl)thiophen-2-yl)-1<i>H</i>-indole-1-carboxylate (<b>10d</b>)</h5><div class="NLM_p last">Yield 3.35 g (81%), mp > 280 °C dec; <sup>1</sup>H NMR (CDCl<sub>3</sub>): 8.82 (d, 1H, <i>J</i> = 1.6 Hz), 8.54 (s, 1H), 7.97 (dd, 1H, <i>J</i> = 2.4 Hz, 8.4 Hz), 7.77 (dd, 1H, <i>J</i> = 0.8, 8.4 Hz), 7.69 (d, <i>J</i> = 4.0 Hz), 7.65 (d, 1H, <i>J</i> = 8.0 Hz), 7.52 (dd, 1H, <i>J</i> = 1.6, <i>J</i> = 8.4 Hz), 7.26 (d, 1H, <i>J</i> = 4.0 Hz), 6.83 (s, 1H), 1.51 (s, 9H); <sup>13</sup>C NMR (CDCl<sub>3</sub>): 155.1, 152.7, 149.2, 143.9, 139.9, 138.9, 136.7, 135.8, 132.0, 129.8, 126.8, 126.4, 121.7, 120.3, 120.1, 118.2, 117.0, 112.1, 107.9, 107.5, 85.7, 27.9; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>24</sub>H<sub>18</sub>N<sub>4</sub>O<sub>2</sub>SNa, 449.1045 (M<sup>+</sup> + Na); found, 449.1057.</div></div><div id="sec4_1_4_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52">4.1.4.5.  <i>t</i>-Butyl-6-cyano-2-(5-(4-cyano-2-methoxyphenyl)furan-2-yl)-1<i>H</i>-indole-1-carboxylate (<b>10e</b>)</h5><div class="NLM_p last">Yield of the yellow solid, yield 1.33 g (61%). mp > 300 °C; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 8.44 (s, 1H), 7.97 (d, 1H, <i>J</i> = 8 Hz), 7.89 (d, 1H, <i>J</i> = 8 Hz), 7.71 (d, 1H, <i>J</i> = 8 Hz), 7.65 (br s, 1H), 7.53 (d, 1H, <i>J</i> = 8 Hz), 7.31–7.30 (m, 1H), 7.24 (s, 1H), 7.06–7.05 (m, 1H), 4.04 (s, 3H), 1.36 (s, 9H); ESI-MS: <i>m</i>/<i>z</i> calcd for C<sub>26</sub>H<sub>21</sub>N<sub>3</sub>O<sub>4</sub>: 439.46, found, 440.4 (M<sup>+</sup> + 1); Anal. Calcd for C<sub>26</sub>H<sub>21</sub>N<sub>3</sub>O<sub>4</sub>: C, 71.06; H, 4.82; N, 9.56. Found: C, 71.22; H, 4.83; N, 9.49.</div></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53">4.1.5.  General Procedure for the Synthesis of the Heteroaryl Diamidines Hydrochloride (<b>11a–e</b>)</h4><div class="NLM_p">To a cold and stirred suspension of the dinitriles (0.001 mol) in 15 mL dry THF was added 6.0 mL (0.006 mol) LiN(TMS)<sub>2</sub> (1 M in THF), stirred for 24 h, cooled, acidified carefully with sat. ethanolic-HCl, and the precipitated white solid was stirred for 2 h, the solvent was removed under reduced pressure, ether was added, and the solid was filtered. The dark solid was collected and 10 mL ice water was added, basified with 2 M NaOH to pH of ca. 10, and the precipitate was filtered, washed with water, and air dried. The solid was suspended in dry ethanol (15 mL) and 5 mL sat. ethanolic HCl and stirred for 6 h; the ethanol was distilled off and the resultant solid was triturated with dry ether and filtered. The solid was dried under reduced pressure at 80 °C for 12 h to yield (33–75%) diamidine hydrochlorides.</div><div id="sec4_1_5_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54">4.1.5.1.  2-(5-(4-Carbamimidoylphenyl)furan-2-yl)-1<i>H</i>-indole-5-carboximidamide Dihydrochloride (<b>11a</b>, <b>DB1478</b>)</h5><div class="NLM_p last">A yellow brown solid, mp > 300 °C; yield 2.5 g (80%), mp > 300 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 12.59 (s, 1H), 9.50 (br s, 2H), 9.29 (br s, 2H), 9.26 (br s, 2H), 9.04 (br s, 2H), 8.12 (s, 1H), 8.13 (d, 2H, <i>J</i> = 8.4 Hz), 8.0 (d, 2H, <i>J</i> = 8.4 Hz), 7.63 (br s, 2H), 7.46 (d, 1H, <i>J</i> = 3.6 Hz), 7.28 (d, 1H, <i>J</i> = 3.6 Hz), 7.13 (d, 1H, <i>J</i> = 1.6 Hz); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 166.4, 164.8, 151.3, 147.7, 139.7, 134.4, 131.0, 129.0, 127.8, 125.0, 123.9, 123.6, 121.5, 118.9, 111.7, 111.4, 110.1, 99.6; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>5</sub>O, 344.1511 (M<sup>+</sup> + 1); found, 344.1513; Anal. Calcd for C<sub>20</sub>H<sub>17</sub>N<sub>5</sub>O·2HCl·1.25H<sub>2</sub>O: C, 54.74; H, 4.93; N, 15.96. Found: C, 54.67; H, 5.03; N, 15.79.</div></div><div id="sec4_1_5_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55">4.1.5.2.  2-(5-(5-Carbamimidoylpyridin-2-yl)furan-2-yl)-1<i>H</i>-indole-5-carboximidamide Trihydrochloride (<b>11b</b>, <b>DB1481</b>)</h5><div class="NLM_p last">A red brown solid, 0.35 g (72%); mp > 300 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 12.70 (s, 1H), 9.70 (br, 2H), 9.39 (br s, 2H), 9.31 (br s, 2H), 9.06 s, 2H), 9.05 (d, 1H, <i>J</i> = 2.4 Hz), 8.41 (dd, 1H, <i>J</i> = 2.4, 8.4 Hz), 8.22 (d, 1H, <i>J</i> = 8.4 Hz), 8.20 (s, 1H), 7.54–7.52 (br s, 2H), 7.53 (d, 1H, <i>J</i> = 3.6 Hz), 7.33 (d, 1H, <i>J</i> = 3.6 Hz), 7.18 (d, 1H, <i>J</i> = 2.0 Hz); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 166.4, 163.6, 151.6 (signal is big accounts 2 carbons), 149.3, 149.1, 139.9, 137.5, 130.8, 127.8, 122.1, 121.83, 121.77, 119.2, 118.0, 114.5, 112.0, 110.4, 100.5; Anal. Calcd for C<sub>19</sub>H<sub>16</sub>N<sub>6</sub>O·3HCl·1.75H<sub>2</sub>O: C, 47.02; H, 4.67; N, 13.31. Found: C, 47.83; H, 4.73; N, 13.49.</div></div><div id="sec4_1_5_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56">4.1.5.3.  2-(5-(4-Carbamimidoylphenyl)thiophen-2-yl)-1<i>H</i>-indole-6-carboximidamide Diydrochloride (<b>11c</b>, <b>DB1879</b>)</h5><div class="NLM_p last">A brown solid, yield 0.32 g (70%), mp > 300 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 12.72 (d, 1H, <i>J</i> = 1.2 Hz), 9.57 (br s, 2H), 9.39 (br s, 2H), 9.33 (brs 2H), 9.13 (br s, 2H), 7.98–7.95 (m, 3H), 7.93 (d, 2H, <i>J</i> = 7.6 Hz), 7.87 (d, 1H, <i>J</i> = 3.6 Hz), 7.84 (d, 1H, <i>J</i> = 3.6 Hz), 7.72 (d, 1H, <i>J</i> = 8.4 Hz), 7.48 (dd, 1H, <i>J</i> = 1.6, 8.4 Hz), 6.88 (br s, 1H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 166.5, 164.9, 141.2, 138.2, 136.1, 135.9, 135.6, 132.5, 129.2, 127.3, 127.0, 126.5, 125.3, 120.6, 120.3, 119.1, 112.0, 100.0; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>18</sub>N<sub>5</sub>S, 360.1283 (M<sup>+</sup> + 1); found, 360.1271; Anal. Calcd for C<sub>20</sub>H<sub>17</sub>N<sub>5</sub>S·2HCl·1.25H<sub>2</sub>O: C, 45.51; H, 4.48; N, 13.26. Found: C, 45.63; H, 4.62; N, 13.56.</div></div><div id="sec4_1_5_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57">4.1.5.4.  2-(5-(5-Carbamimidoylpyridin-2-yl)thiophen-2-yl)-1<i>H</i>-indole-6-carboximidamide Trihydrochloride (<b>11d</b>, <b>DB2529</b>)</h5><div class="NLM_p last">A reddish brown solid, yield 0.35 g (74%), mp > 280 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 12.73 (d, 1H, <i>J</i> = 1.2 Hz), 9.74 (s, 2H), 9.44 (s, 2H), 9.39 (s, 2H), 9.14 (s, 2H), 9.02 (s, 1H), 8.34 (dd, 1H, <i>J</i> = 1.6, 8.4 Hz), 8.22 (d, 1H, <i>J</i> = 8.4 Hz), 8.10 (d, 1H, <i>J</i> = 3.6 Hz), 7.98 (br s, 1H), 7.73 (d, 1H, <i>J</i> = 3.6 Hz), 7.47 (dd, 1H, <i>J</i> = 1.6, 8.4 Hz), 6.99 (d, 1H, <i>J</i> = 1.6 Hz); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 166.48, 163.5, 155.3, 149.2, 142.6, 138.3, 137.2, 136.2, 136.0, 132.5, 129.0, 126.9, 122.1, 120.8, 120.4, 119.1, 118.1, 112.1, 100.4; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>17</sub>N<sub>6</sub>S, 361.1235 (M<sup>+</sup> + 1); found, 361.0285; Anal. Calcd for C<sub>19</sub>H<sub>16</sub>N<sub>6</sub>S·3HCl: C, 48.71; H, 4.09; N, 17.95. Found: C, 48.86; H, 4.12; N, 17.84.</div></div><div id="sec4_1_5_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58">4.1.5.5.  2-(5-(4-Carbamimidoyl-2-methoxyphenyl)furan-2-yl)-1<i>H</i>-indole-6-carboximidamide (<b>11e</b>, <b>DB2005</b>)</h5><div class="NLM_p last">Yellow solid, yield (0.184 g, 33%), mp > 300 °C; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 12.7 (s, 1H), 9.61 (s, 2H), 9.51 (s, 2H), 9.47 (s, 2H), 9.36 (s, 2H), 8.42 (d, 1H, <i>J</i> = 8 Hz), 8.02 (s, 1H), 7.76 (d, 1H, <i>J</i> = 8 Hz), 7.69–7.65 (m, 2H), 7.49 (d, 1H, <i>J</i> = 8 Hz), 7.34–7.32 (m, 2H), 7.1 (s, 1H), 4.09 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): δ 166.9, 165.1, 155.2, 148.8, 147.1, 136.3, 133.1, 132.8, 127.4, 126.3, 125.9, 123.3, 121.0, 120.9, 119.5, 115.8, 112.5, 112.0, 111.2, 99.7, 56.7; ESI-MS: <i>m</i>/<i>z</i> calcd for C<sub>21</sub>H<sub>19</sub>N<sub>5</sub>O<sub>2</sub>, 373.41; found, 374.40 (amidine base M<sup>+</sup> + 1); Anal. Calcd for C<sub>21</sub>H<sub>19</sub>N<sub>5</sub>O<sub>2</sub>·2HCl·1.75H<sub>2</sub>O·0.15EtOH: C, 52.77; H, 5.28; N, 14.44. Found: C, 52.42; H, 5.34; N, 14.39.</div></div><div id="sec4_1_5_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i59" class="anchor-spacer"></div><h5 class="article-section__title" id="_i59">4.1.5.6.  4,4′-(Furan-2,5-diyl)bis(<i>N</i>-(4-nitrophenethyl)benzimidamide)dihydrochloride (<b>12a</b>, <b>DB421</b>)</h5><div class="NLM_p last">4-(Nitrophenyl)ethylamine free base [prepared from HCl salt 0.405 g (0.002) mol and freshly prepared NaOMe)] in 10 mL dry ethanol was added to a suspension of imidate ester dihydrochloride prepared from 2,5 bis(4-cyanophenyl) furan<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> 0.435 g (0.001 mol in 35 mL dry ethanol, and the reaction mixture was stirred at rt 12 h (TLC monitored). The solvent was removed in vacuo. Diluted with 20 mL ice water and basified to pH 10 with 2 M NaOH (aq), the yellow solid was filtered, washed with water, and dried under reduced pressure at rt. The free base was converted to its dihydrochloride by treating with sat. ethanolic HCl, yellow solid, yield 0.52 g (77%), mp > 310 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 10.15 (br s, 2H), 9.69 (br s, 2H), 9.44 (br s, 2H), 8.22 (d, 4H, <i>J</i> = 7.6 Hz), 8.09 (d, 4H, <i>J</i> = 7.6 Hz), 7.85 (d, 4H, <i>J</i> = 7.6 Hz), 7.69 (d, 4H, <i>J</i> = 7.6 Hz), 7.41 (s, 2H), 3.81 (t, 4H, <i>J</i> = 6.8 Hz), 3.18 (t, 4H, <i>J</i> = 6.8 Hz); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 162.2, 152.4, 146.7, 146.4, 133.9, 130.4, 129.0, 127.4, 123.7, 123.5, 11.5, 43.2, 33.1; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>34</sub>H<sub>31</sub>N<sub>6</sub>O5, 603.2356 (M<sup>+</sup> + 1); found, 603.2291; Anal. Calcd for C<sub>34</sub>H<sub>30</sub>N<sub>6</sub>O<sub>5</sub>·2HCl: C, 60.45; H, 4.77; N, 12.44. Found: C, 60.33; H, 4.82; N, 12.56.</div></div><div id="sec4_1_5_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60">4.1.5.7.  2,5-Bis(4-amidinophenyl)tellurophene Dihydrochloride (<b>12b</b>, <b>DB1751</b>)</h5><div class="NLM_p last">In a 50 mL flame-dried round bottom flask, 2,5-bis(4-cyanophenyl)tellurophene<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a> (0.191 g, 0.5 mmol) was dissolved in dry tetrahydrofuran (10 mL, freshly distilled from Na). Lithium hexamethyldisilazane (LiHMDS, 1.0 M in THF, 2.5 mL) was added slowly by a syringe, and the darkened mixture was stirred overnight under nitrogen. The solution was then treated with HCl-saturated EtOH (3 mL) to yield a yellow precipitate. After stirring the suspension for another 6 h, it was diluted with ether (∼25 mL) and vacuum-filtered to yield an orange solid. The solid was then triturated in water (∼10 mL) to dissolve any ammonium chloride, then refiltered, and dried overnight in vacuo under low heat to yield the title compound as a reddish solid (0.164 g, 63%). <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 7.85 (s, 8H), 8.34 (s, 2H), 9.10 (br s, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 126.7, 126.7, 129.0, 137.2, 144.1, 148.1, 164.8. HRMS (ESI): calcd for C<sub>18</sub>H<sub>16</sub>N<sub>4</sub><sup>130</sup>Te, 419.0516; found, 419.0531 [M + H]<sup>+</sup>. Anal. Calcd for C<sub>18</sub>H<sub>16</sub>N<sub>4</sub>Te–2HCl–2H<sub>2</sub>O (524.89): C, 41.19; H, 4.22; N, 10.67. Found: C, 41.36; H, 4.13; N, 11.05.</div></div><div id="sec4_1_5_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61">4.1.5.8.  2-(4′-Carbamimidoyl-[1,1′-biphenyl]-3-yl)-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carboximidamide Trihydrochloride (<b>14</b>, <b>DB1314</b>)</h5><div class="NLM_p last">To a stirred solution of 3-(4-cyanophenyl)benzaldehyde<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> 2.07 g (0.01 mol) and 4-amino-3-(methylamino)benzonitrile 1.47 g (0.01 mol) in 20 mL dry DMF under N<sub>2</sub> was added sodium bisulfite 2.08 (0.02 mol), and the mixture was heated under reflux for 12 h (TLC monitored). The solvent was removed, the residue was triturated with cold water, and the separate<u class="uu">d</u> solid was filtered, washed with water, and air dried. A light brown solid was stirred with a 1:1 mixture of DCM–ether, filtered, and dried under reduced pressure at 60 °C for 4 h, which gave 2-(4′-cyano-[1,1′-biphenyl]-3-yl)-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-6-carbonitrile (<b>13</b>), yield 2.3 g (69%); mp > 300 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 8.36 (d, 1H, <i>J</i> = 1.2 Hz), 8.24 (t, 1H, <i>J</i> = 1.6 Hz), 8.03–7.97 (m, 6H), 7.88 (d, 1H, <i>J</i> = 8.4 Hz), 7.76 (dd, 1H, <i>J</i> = 8.0 Hz), 7.67 (dd, 1H, <i>J</i> = 1.2, 8.4 Hz), 4.01 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 155.7, 145.1, 143.5, 138.6, 136.1, 132.5, 129.9, 129.3, 129.2, 128.5, 127.6, 127.5, 125.0, 119.8, 119.4, 118.2, 115.6, 110.3, 104.0, 31.7; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>15</sub>N<sub>4</sub>, 335.1297 (M<sup>+</sup> + 1); found, 335.1307. Using 0.334 g (0.001 mol) of the previous bisnitrile <b>13</b> yielded the final diamidine <b>DB1314</b> following the LiN(TMS)<sub>2</sub> method described above; yield 0.32 g (70%), mp > 300 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 9.51 (s, 2H), 9.50 (s, 2H), 9.26 (s, 2H), 9.20 (s, 2H), 8.35 (br s, 1H), 9.31 (br s, 1H), 8.09 (d, 2H, <i>J</i> = 8.4 Hz), 8.06–8.0 (m, 4H), 7.89 (br d, 1H, <i>J</i> = 8.8 Hz), 7.82 (t, 1H, <i>J</i> = 8.0 Hz), 4.06 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 165.5, 165.2, 153.7, 144.0, 139.2, 138.2, 135.6, 130.48, 130.12, 130.07, 129.07, 128.9, 127.5, 127.4, 126.4, 124.2, 123.8, 117.3, 112.9, 33.0; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>22</sub>H<sub>21</sub>N<sub>6</sub>, 369.1827 (M<sup>+</sup> + 1); found, <i>m</i>/<i>z</i> 369.1827; Anal. Calcd for C<sub>22</sub>H<sub>20</sub>N<sub>6</sub>.3.0HCl·1.5H<sub>2</sub>O: C, 52.34; H, 5.19; N, 16.69. Found: C, 52.43; H, 5.32; N, 16.50.</div></div><div id="sec4_1_5_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62">4.1.5.9.  2-(Thiophen-2-yl)quinoline-6-carbonitrile (<b>15</b>)</h5><div class="NLM_p last">It was synthesized from 6-bromo-2-thiophene-quinoline-4-carboxylic acid,<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> a product of Pfitzinger reaction of 6-bromoisatin and 2-acetylthiophene according to the published procedure.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> 6-Bromo-2-thiophene-quinoline-4-carboxylic acid (0.15 g, 0.45 mmol) was heated with CuCN (80 mg, 0.9 mmol) in dry DMF at 140–150 °C for 3 days. The mixture is quenched with aq KCN and extracted with EtOAc to give 33 mg of 6-cyano-2-thiophene-quinoline in 31% yield after chromatographic purification (30% Et<sub>2</sub>O/hexanes); mp 141.5–142.5 °C; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 8.50 (d, <i>J</i> = 1.8 Hz, 1H), 8.47 (d, <i>J</i> = 9.0 Hz, 1H), 8.20 (d, <i>J</i> = 8.7 Hz, 1H), 8.07 (d, <i>J</i> = 4.2 Hz, 1H), 8.05 (s, 1H), 7.97 (dd, <i>J</i><sub>1</sub> = 2.1, 6.0 Hz, 1H), 7.80 (d, <i>J</i> = 5.1 Hz, 1H), 7.25 (q, <i>J</i><sub>1</sub> = 3.6, 1.2 Hz, 1H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): δ 152.9, 146.9, 142.4, 135.6, 132.4, 128.8, 128.7, 128.2, 126.8, 126.2, 124.7, 117.4, 116.8, 106.9. MS (ESI): 237.08 (M<sup>+</sup> + 1, 100). IR <i>h</i>ν (cm<sup>–1</sup>): 2219 (CN).</div></div><div id="sec4_1_5_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63">4.1.5.10.  2-(5-Bromothiophen-2-yl)quinoline-6-carbonitrile (<b>16</b>)</h5><div class="NLM_p last">It was synthesized from <b>15</b> (0.33 g, 1.6 mmol) and NBS (0.3 g, 1.8 mmol) in dry DMF according to a general procedure<a onclick="showRef(event, 'ref68 ref69'); return false;" href="javascript:void(0);" class="ref ref68 ref69">(68,69)</a> to give 0.47 g of <b>3</b> in 93% yield, mp 216–218 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 8.57 (s, 1H), 8.50 (d, <i>J</i> = 9 Hz, 1H), 8.20 (d, <i>J</i> = 9 Hz, 1H), 8.03 (m, 2H), 7.90 (d, <i>J</i> = 3.9 Hz, 1H), 7.36 (d, <i>J</i> = 3.9 Hz, 1H). HRMS (ESI) <i>m</i>/<i>z</i>: calcd for C<sub>14</sub>H<sub>8</sub>N<sub>2</sub>BrS, 314.9513; found, 314.9592.</div></div><div id="sec4_1_5_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64">4.1.5.11.  2-(5-(4-Cyanophenyl)thiophen-2-yl)quinoline-6-carbonitrile (<b>17</b>)</h5><div class="NLM_p last">It was synthesized under general Suzuki coupling conditions from <b>16</b> (0.37 g, 1.04 mmol), Pd(Ph<sub>3</sub>P)<sub>4</sub> (60 mg, 0.05 mmol), 4-cyanophenylboronicacid (183.7 mg, 1.25 mmol), and 2 M K<sub>2</sub>CO<sub>3</sub> (1 mL) in 1,4-dioxane (5 mL) to give <b>4</b> (280 mg), 80% yield; mp > 300 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 8.59 (s, 1H), 8.53 (d, <i>J</i> = 8.4 Hz, 1H), 8.28 (d, <i>J</i> = 8.8 Hz, 1H), 8.13 (d, <i>J</i> = 6.4 Hz, 1H), 8.10 (s, 1H), 8.00 (m, 3H), 7.89 (d, <i>J</i> = 8.0 Hz, 2H), 7.83 (s, 1H). Anal. Calcd for C<sub>21</sub>H<sub>11</sub>N<sub>3</sub>S: C, 74.76; H, 3.29. Found: C, 74.48; H, 3.09.</div></div><div id="sec4_1_5_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65">4.1.5.12.  2-(5-(4-Carbamimidoylphenyl)thiophen-2-yl)quinoline-6-carboximidamide Trihydrochloride (<b>18</b>, <b>DB1504</b>)</h5><div class="NLM_p last">It was synthesized from <b>4</b> (95 mg, 0.27 mmol) and LiN(TMS)<sub>2</sub> (1 M, 1.6 mL, 1.6 mmol) in dry THF according to the procedure described above to give after work-up 79 mg (79%) of <b>DB1504</b>, mp > 300 °C. <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): δ 9.46 (s, 2H, exchangeable with D<sub>2</sub>O) 9.50 (s, 2H, exchangeable with D<sub>2</sub>O), 9.42 (s, 2H, exchangeable with D<sub>2</sub>O), 9.29 (s, 2H, exchangeable with D<sub>2</sub>O), 8.58 (m, 2H), 8.34 (d, <i>J</i> = 8.4 Hz, 1H), 8.20 (d, <i>J</i> = 3.6 Hz, 1H), 8.10 (m, 4H), 7.94 (m, 3H). <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): δ 165.2, 164.9, 154.1, 149.3, 145.5, 145.1, 138.6, 138.4, 130.1, 129.9, 129.3, 129.2, 128.4, 127.6, 127.0, 126.2, 125.9, 125.4, 119.2. MS (ESI) <i>m</i>/<i>z</i>: 372.1 (M<sup>+</sup> + 1). Anal. Calcd for C<sub>21</sub>H<sub>17</sub>N<sub>5</sub>S–3HCl–0.5H<sub>2</sub>O–0.2EtOH: C, 51.50; H, 4.48; N, 14.03. Found: C, 51.78; H, 4.27; N, 13.70.</div></div><div id="sec4_1_5_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66">4.1.5.13.  2-(5-(4-(5-Carbamimidoyl-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)phenyl)thiophen-2-yl)-1-methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-5-carboximidamide Tetrahydrochloride (<b>19</b>, <b>DB2572</b>)</h5><div class="NLM_p last">5-(4-formylphenyl)thiophene-2-carboxaldehyde<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> (0.108 g, 0.0005 mol), 3-amino-4-(methylamino) benzamidine hydrochloride<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> (0.228 g, 0.001 mol), and 1, 4-benzoquinone (0.108 g, 0.001 mol) in dry ethanol (50 mL) (under nitrogen) were heated at reflux for 12 h. The reaction mixture following standard work-up and drying at 50 °C (12 h in vac.) yielded di hydrochloride salt, which was converted to tetra hydrochloride salt by dissolving in 15 mL ethanol and treating with 2 mL sat. ethanolic HCl; greenish solid, yield 0.25 g (72%), mp > 280 °C dec; <sup>1</sup>H NMR (DMSO-<i>d</i><sub>6</sub>): 9.60 (br s, 2H), 9.47 (br s, 2H), 9.34 (br s, 2H), 9.23 (br s, 2H), 8.37 (d, 1H, <i>J</i> = 1.2 Hz), 8.27 (d, 1H, <i>J</i> = 1.2 Hz), 8.12–8.55 (m, 5H), 8.03 (d, 1H, <i>J</i> = 3.6 Hz), 7.97–7.92 (m, 3H), 7.82 (dd, 1H, <i>J</i> = 1.2, 8.8 Hz), 4.17 (s, 3H), 4.08 (s, 3H); <sup>13</sup>C NMR (DMSO-<i>d</i><sub>6</sub>): 165.8, 165.5, 153.3, 148.8, 145.5, 140.4, 139.7, 138.4, 136.5, 138.4, 130.93, 130.91, 130.6, 126.4, 125.8, 125.5, 123.5, 123.1, 122.4, 121.7, 118.7, 117.4, 112.2, 111.1, 32.6, 32.2; MS: HRMS-ESI-POS: <i>m</i>/<i>z</i> calcd for C<sub>28</sub>H<sub>25</sub>NS, 505.1923 (M<sup>+</sup> + 1); found, 505.1876; Anal. Calcd for C<sub>28</sub>H<sub>24</sub>N<sub>8</sub>S·4HCl·3.0H<sub>2</sub>O: C, 47.73; H, 4.86; N, 15.91. Found: C, 47.93; H, 4.73; N, 15.79.</div></div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i67" class="anchor-spacer"></div><h3 class="article-section__title" id="_i67">4.2.  Biological Assays</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68">4.2.1.  DNA and Proteins</h4><div class="NLM_p last">All oligonucleotides (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) were purchased from Eurogentec (Belgium). The pGL3-prom-HBS-1R, pGL3-prom-HBS-6R, and pGL3-prom-HBSmut-6R reporter vectors were obtained by double digestion of the pGL3-promoter plasmid (Promega, France) at MluI and NheI (NEB, France) restriction sites localized 5′ upstream to the SV40 minimal promoter, followed by ligation with phosphorylated double-stranded oligonucleotides obtained from hybridization of FP-HBS-1Ra and FP-HBS-1Rb, Luc-HBS-6Ra and Luc-HBS-6Rb, or Luc-HBSmut-6Ra and Luc-HBSmut-6Rb oligonucleotides. The pcDNA<sub>3</sub>-HOXA9 expression vector was a kind gift from Dr Nabeel B. Yaseen (Northwestern University, Chicago, USA).<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> A TY-tag was added in 5′ to HOXA9 cDNA to be expressed in the reticulocyte lysate system for ELISA-derived protein/DNA binding inhibition assay. The pCMV-AD-HOXA9 expression vector used for luciferase assays was constructed by subcloning HOXA9 cDNA from pRc/CMV-HOXA9 (a gift from Prof. Corey Largman<sup>†</sup>, University of California, San Francisco, USA) into the pCMV-AD vector (Stratagene, France) to fuse the NF-κB activation domain to HOXA9 protein in order to improve the transactivation capacity of HOXA9 in the absence of co-activators for luciferase assays.</div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69">4.2.2.  ELISA-Derived Protein/DNA Binding Inhibition Assay (EPDBi)</h4><div class="NLM_p last">The 96-well screening procedure was performed essentially as described<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> with the following modifications. Streptavidin-coated 96-well plates (Thermo Fisher, France) were blocked for 1 h at room temperature in 200 μL of blocking solution of TBS-T (Tris-HCl 10 mM pH 8.0, NaCl 150 mM, Tween 0.5%) supplemented with 10% skimmed milk powder. After three 5 min washes at room temperature with TBS-T, 20 nM of double-stranded biotinylated HOXA9-binding site containing oligonucleotides (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) were immobilized per well for 20 min at room temperature in 100 μL of binding buffer (Tris-HCl 10 mM pH 7.5, EDTA 1 mM pH 7.5, NaCl 50 mM, DTT 1 mM, BSA 10 μg/μL, glycerol 6%) with or without 5 or 10 μM of the tested diamidine compounds. After two 5 min washes at room temperature with 200 μL of binding buffer, 40 μL of binding buffer containing HOXA9 proteins expressed from the reticulocyte lysate system (Promega, Madison, Wisconsin, USA) (2 μL of lysate) was incubated for 1 h at room temperature with or without 50- or 100-fold excess of non-biotinylated oligonucleotides of the same HBS-containing sequence or a nonspecific sequence. The unbound proteins were removed by three successive washes with the binding buffer. The HOXA9/DNA complex was revealed by the addition of 85 ng of a mouse anti-TY1-tag antibody (C1520054, Diagenode, Belgium) in TBS-T for 1 h at room temperature followed by three washing steps and addition of a 1/2000 dilution of a secondary sheep HRP-conjugated anti-mouse antibody (NXA931, GE Healthcare, France). After three 5 min washes with 200 μL of TBS-T, 50 μL of TMB (Promega, Madison, Wisconsin, USA) was added for 7 min at room temperature. The reaction was stopped by the addition of 50 μL of 0.5 M H<sub>2</sub>SO<sub>4</sub> and the absorbance was measured at 450 nm on a microplate spectrometer (Spectramax-I3, Molecular Devices). All collected data were analyzed using SoftMax Pro software.</div></div><div id="sec4_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70">4.2.3.  Electrophoretic Mobility Shift Assay</h4><div class="NLM_p last">The HBS or the Akap13-HBS double-strand oligonucleotides (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) were radiolabeled at 5′-end as described previously.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The binding experiments were performed as previously reported<a onclick="showRef(event, 'ref43 ref44'); return false;" href="javascript:void(0);" class="ref ref43 ref44">(43,44)</a> with some modifications. Briefly, 2 μL of HOXA9 protein-expressing reticulocyte lysates (TNT coupled reticulocyte lysate system, Promega, Madison, Wisconsin, USA) were incubated for 20 min at 4 °C in binding buffer with the radiolabeled HBS-containing DNA and diamidine compounds at the indicated concentrations. Free DNA was separated from protein–DNA complexes on a 6% native polyacrylamide gel under electrophoresis for 3 h at 300 V in 0.5× TBE buffer. Gel revelations were performed using a Pharos-PMI equipment (BioRad) and analyzed with the ImageQuant-3.3 software.</div></div><div id="sec4_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71">4.2.4.  CD and DNA Thermal Melting Assay</h4><div class="NLM_p last">CD spectra and DNA melting temperature studies were performed using double-stranded oligonucleotides of various DNA sequences in the absence or presence of diamidine compounds essentially as previously reported,<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> with the following modifications. Double-stranded oligonucleotides were obtained by equimolar hybridization of the corresponding -a and -b strands (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>). For CD analysis, 5 μM of double-stranded HBS oligonucleotides was incubated in TNE buffer (50 mM Tris-HCl pH 7.8 + 100 mM NaCl + 1 mM EDTA), and CD spectra were collected in a quartz cell of 10 mm path length as an average of three accumulations from 480 to 230 nm using a J-810 Jasco spectropolarimeter at a controlled temperature of 25 °C fixed by a PTC-424S/L Peltier type cell changer (Jasco) as described previously.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a><b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> (1, 2, 3, 4, 5, 6, 8, and 10 μM) were gradually added to the DNA sample separately, and CD was measured at each drug/DNA ratio. For <i>T</i><sub>m</sub> measurements, indicated compounds and oligonucleotides were mixed in TNE buffer at a drug/DNA ratio <i>R</i> of 1:1 or 2:1 as indicated. The DNA absorbance at 260 nm was measured in quartz cells using an Uvikon XL spectrophotometer thermostated with a Peltier cryostat every min over a range of 15–100 °C with an increment of 1 °C/min. The <i>T</i><sub>m</sub> values were deduced from the midpoint of the hyperchromic transition obtained from first-derivative plots. The variation of melting temperature (Δ<i>T</i><sub>m</sub>) was obtained by subtracting the melting temperature measurement of CT-DNA alone (control <i>T</i><sub>m</sub>) to that obtained with DNA incubated with the compounds (Δ<i>T</i><sub>m</sub> values = <i>T</i><sub>m[compound + DNA]</sub> – <i>T</i><sub>m[DNA alone]</sub>).</div></div><div id="sec4_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i72" class="anchor-spacer"></div><h4 class="article-section__title" id="_i72">4.2.5.  DNase I Footprint Assay</h4><div class="NLM_p last">The pGL3-prom-HBS-1R or pGL3-prom-HBSmut-1R plasmid was digested by ClaI and BglII restriction enzymes and radiolabeled at 3′ termini with [α-<sup>32</sup>P]-dATP (PerkinElmer, France). DNase I digestion and electrophoretic separation were performed as reported.<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> The DNA sequence was deduced from guanine positions cleaved upon DMS and piperidine treatment (G-track). The gels were exposed to storage screen for the appropriated delay at room temperature. The results were collected using Pharos-PMI equipment (BioRad).</div></div><div id="sec4_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i73" class="anchor-spacer"></div><h4 class="article-section__title" id="_i73">4.2.6.  Surface Plasmon Resonance</h4><div class="NLM_p">SPR measurements were performed with four-channel Biacore T200 optical biosensor systems (GE Healthcare, Inc., Piscataway, NJ). A streptavidin-immobilized (SA) sensor chip was prepared for use by conditioning with a series of 60 s injections of 1 M NaCl in 50 mM NaOH (activation buffer) followed by extensive washing with HBS buffer [10 mM HEPES, 150 mM NaCl, 3 mM EDTA, and 0.05% P20 (pH 7.4)]. Biotinylated HP-HBS and HP-HBS-ATm (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) DNA hairpins (25–50 nM) were prepared in HBS buffer and immobilized on the flow cell surface by noncovalent capture as previously described.<a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(79,80)</a> Flow cell 1 was left blank as a reference, while flow cells 2–4 were immobilized with DNA by manual injection of DNA stock solutions (flow rate of 1 μL/min) until the desired amount of DNA response units (RU) was obtained (320–330 RU). Ligand solutions were prepared with degassed and filtered 50 mM Tris-HCl buffer pH 7.42 by serial dilutions from a concentrated stock solution. Typically, a series of different ligand concentrations (from 1 nM to 1 μM) were injected over the DNA sensor chip at a flow rate of 100 μL/min until a constant steady-state response was obtained (3 min), and this was followed by buffer flow for ligand dissociation (10–20 min). After each cycle, the sensor chip surface was regenerated with a 10 mM glycine solution at pH 2.5 for 30 s followed by multiple buffer injections to yield a stable baseline for the following cycles. RUobs was plotted as a function of free ligand concentration (<i>C</i><sub>free</sub>), and the equilibrium binding constants (<i>K</i><sub>A</sub>) were determined either with a one-site binding model (<i>K</i><sub>2</sub> = 0) for <b>DB818</b> or with a two-site model for <b>DB1055</b> (due to some nonspecific interactions), where r represents the moles of bound compound per mol of DNA hairpin duplex and <i>K</i><sub>1</sub> and <i>K</i><sub>2</sub> are macroscopic binding constants.<span class="NLM_disp-formula" id="eq1"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_m001.gif" alt="" id="_i74" /></img><span class="labelSpan">(1)</span></span></div><div class="NLM_p last">RUmax in the equation was used as a fitting parameter, and the obtained value was compared to the predicted maximal response per bound ligand to independently evaluate the stoichiometry.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> One-site binding models have been applied for <b>DB818</b> with HP-HBS and HP-HBS-ATm DNA and also for <b>DB1055</b> with HP-HBS-ATm DNA. Because of some nonspecific interactions, a two-site interaction model has been applied for <b>DB1055</b> with HP-HBS DNA. Kinetic analyses were performed by globally fitting the binding results for the entire concentration series using a standard 1:1 kinetic model with integrated mass transport-limited binding parameters as described previously.<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a></div></div><div id="sec4_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75">4.2.7.  Ab-Initio Calculations and MD</h4><div class="NLM_p">Simulation optimization and electrostatic potential calculations were performed for the <b>DB818</b> molecule using the DFT/B3LYP theory with the 6-31+G* basis set in Gaussian 09 (Gaussian, Inc., 2009, Wallingford, CT) with Gauss-view 5.09. Partial charges were derived using the RESP fitting method (restrained electrostatic potential).<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> AMBER 14 (assisted model building with energy refinement)<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> software suite was used to perform MD simulations. Canonical B-form ds[(5′-CCATGATTTACG-3′)(5′-CGTAAATCATGG-3′)] DNA was built in the Nucleic Acid Builder (NAB) tool in AMBER. AMBER preparation and force field parameter files required to run MD simulations for the <b>DB818</b> molecule were produced using ANTECHAMBER.<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> Specific atom types assigned for the <b>DB818</b> molecule (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S3</a>) were adapted from the ff99 force field. Most of the force field parameters for <b>DB818</b> molecules were derived from the existing set of bonds, angles, and dihedrals for the similar atom types in parm99 and GAFF force fields.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> Some dihedral angle parameters were obtained from previously reported parametrized data.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> Parameters of <b>DB818</b> in the frcmod file are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S2</a>.</div><div class="NLM_p last">AutoDock Vina program<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> was used to dock the <b>DB818</b> in the minor groove of DNA to obtain the initial structure for the <b>DB818</b>–DNA complex. MD simulations were performed in explicit solvation conditions where the DNA–<b>DB818</b> complex was placed in a truncated octahedron box filled with TIP3P water using xleap program in AMBER. Sodium ions were used to neutralize the system. A 10 Å cutoff was applied on all van der Waals interactions. The MD simulation was carried out using the Sander module with the SHAKE algorithm applied to constrain all bonds. Initially, the system was relaxed with 500 steps of steepest-descent energy minimization. The temperature of the system was then increased from 0 to 310 K for over 10 ps under constant-volume conditions. In the final step, the production run on the system was subsequently performed for 100 ns under <i>NPT</i> (constant-pressure) conditions.</div></div><div id="sec4_2_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i76" class="anchor-spacer"></div><h4 class="article-section__title" id="_i76">4.2.8.  Cellular Distribution of the Compounds Using Fluorescence Microscopy</h4><div class="NLM_p last">GFP-expressing MigA9 suspension cells (2 × 10<sup>5</sup>) were treated with the indicated compounds at 20 μM during 2 h and allowed to attach onto poly-<span class="smallcaps smallerCapital">l</span>-lysine-coated coverslips overnight prior to fixation with formaldehyde 4%. The nuclear compartment was stained through DNA binding of TO-PRO-3 iodide (1 mM, Invitrogen, Thermo Fisher Scientific, USA) or propidium iodide (10 μg/mL, Sigma-Aldrich, USA) supplemented with RNase A (300 μg/mL, Sigma-Aldrich, USA) for 30 min in the dark at room temperature. Coverslips were mounted with a drop of anti-fade medium. Adherent human colon carcinoma HT-29 cells were treated with 5 μM of the indicated compounds for 16 h and fixed with 2% paraformaldehyde prior to the addition of 75 nM of DiOC<sub>6</sub> and 1 μg/mL of PI supplemented with RNaseA. Based on the intrinsic fluorescence of evaluated compounds, images were acquired either on a confocal inverted LSM880 microscope (Carl Zeiss microscopy, Zeiss GmbH, Germany) or on a wide field fluorescence inverted DMi8 (Leica Microsystems, Germany) microscope, both with a ×63 objective. Excitation and emission wavelengths are indicated in the legends.</div></div><div id="sec4_2_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i77" class="anchor-spacer"></div><h4 class="article-section__title" id="_i77">4.2.9.  Transient Transfections</h4><div class="NLM_p last">The K562 cell line was used as an HOXA9-negative leukemia cell model that has strong potency for transfection (AMAXA V-kit system, Lonza). K562 cells (2 × 10<sup>6</sup>) were precultured overnight in RPMI-1640 supplemented with GlutaMAX-I (Gibco) and mixed with 2 μg of the expression vector (pCMVAD-HOXA9 or pCMVAD as control), reporter vector (pGL3-Prom-HBS-6R, pGL3-Prom-HBSmut-6R or pGL3-Prom as control), and vectors for validation/normalizing of transfection efficiency pmaxGFP (0.5 μg) and pCDNA4-βgal (0.5 μg) in 100 μL of BufferV to then be electroporated using a Amaxa Nucleofactor as recommended by the provider (Nucleofector kit V, LONZA, France). Each transfected cell sample was recovered in 4 mL of complete RPMI-media to be split in two equal amounts (1 × 10<sup>6</sup> cells) in 12-well plates. After 6 h, compounds were added at 20 μM in one of them, whereas the other one received vehicle (deionized H<sub>2</sub>O). After 48 h, cells were rinsed with PBS and lysed under nondenaturing conditions for 20 min at 4 °C with 0.5 mL of lysis passive buffer 5× (Promega, Madison, Wisconsin, USA), and the supernatant was recovered after a 15 min centrifugation at 13 000<i>g</i>. For β-galactosidase activity, 200 μL of luminescent β-galactosidase detection kit-II (Clontech, France) was added to 25 μL of the supernatant. The fluorescence of GFP protein was detected using a 96-well fluorimeter at λ<sub>excitation</sub> = 485 nm and λ<sub>emission</sub> = 535 nm. The luciferase activity was finally measured using the luciferase assay system (Promega, Madison, Wisconsin, USA) according to manual recommendations. All data were collected using Mithras LB940 (Berthold).</div></div><div id="sec4_2_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i78" class="anchor-spacer"></div><h4 class="article-section__title" id="_i78">4.2.10.  DNA Microarray Analysis</h4><div class="NLM_p last">MigA9 cells (2 × 10<sup>6</sup>) were treated in the presence and absence of 5 μM of <b>DB818</b> in duplicates for 6, 12, or 24 h treatments. Total RNA was then purified with the RNAeasy kit from Qiagen including DNAseI treatment, and the duplicates were pooled. Mouse Whole Genome Agilent 44K 60-mer oligonucleotide microarrays were performed according to the two-color microarray-based gene expression protocol (Agilent Technologies). Microarrays were scanned using the Agilent scanner G2505B. The expression data were extracted by feature extraction and processed with Limma R package,<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> combining the three data points (6, 12, and 24 h) for normalization, filtering, and statistical analysis. The genes commonly upregulated or downregulated with adjusted <i>p</i>-values <0.05 and with |fold change| ≥ 1.5 at 6, 12, and 24 h were further analyzed with GSEA and IPA software.</div></div><div id="sec4_2_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i79" class="anchor-spacer"></div><h4 class="article-section__title" id="_i79">4.2.11.  qRT-PCR</h4><div class="NLM_p last">MigA9 cells were treated in the presence and absence of 5 and 10 μM of <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> at <i>n</i> = 6 per condition for 48 h treatments prior to the purification of total RNA. RNAs were reverse-transcribed with High-Capacity cDNA Reverse Transcription Kit (Applied Biosystems, ThermoFisher, USA) to generate the corresponding cDNAs and amplified using qPCR primer sequences as presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>. Specificity and efficacy of the primers were addressed. PCR amplification was performed in duplicate using 200 nM of oligonucleotide primers and SYBR Green PCR Master Mix reagent (ThermoFisher Scientific, USA) following the manufacturer’s instructions on a StepOnePlus apparatus using StepOneTM Real-Time PCR software (Applied Biosystems, ThermoFisher, USA). The mRNA levels of each target gene were normalized relatively to the mouse housekeeping gene Gapdh quantified for each sample. The relative expression ratio was calculated using the ΔΔ<i>C</i><sub>t</sub> method.</div></div><div id="sec4_2_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i80" class="anchor-spacer"></div><h4 class="article-section__title" id="_i80">4.2.12.  Cell Survival</h4><div class="NLM_p last">For the MTS assay, MigA9 cells were seeded into 96-well plates at a density of 1.5 × 10<sup>4</sup>/well in 100 μL of complete medium and treated with <b>DB818</b>, <b>DB1055</b>, or <b>DB828</b> compounds at 0.1–50 μM or the corresponding control vehicle for 72 h before analysis using the CellTiter 96 AQueous One Solution Cell Proliferation Assay kit following the manufacturer’s instruction. The absorbance was detected at 490 nm with a Microplate Reader (Spectramax, Molecular Devices). Measurements were performed in duplicates or triplicates and repeated at least three times.</div></div><div id="sec4_2_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i81" class="anchor-spacer"></div><h4 class="article-section__title" id="_i81">4.2.13.  Cell Death and Cell Cycle Analysis Using Flow Cytometry</h4><div class="NLM_p last">MigA9 cells (10<sup>5</sup> cells/mL) were treated for 24 and/or 48 h with various diamidine compounds at the mentioned concentrations, in triplicate. For propidium iodide-positive cell labeling, 2.5 × 10<sup>4</sup> treated cells were stained by propidium iodide (P4170, Sigma-Aldrich, USA, 5 μg/mL). Cells were analyzed by flow cytometry using LSR-Fortessa X20 (BD Biosciences), and data were analyzed using FlowJo software (BD Biosciences). For cell cycle analysis, 10<sup>6</sup> cells were first treated with 10 μM EdU for 1 h using Click-iT EdU Flow Cytometry Assay Kit (Molecular probes) prior to fixation using cold 70% ethanol followed by addition of propidium iodide at 50 μg/mL in the presence of RNaseA 100 μg/mL for 30 min. The cell cycle was acquired by flow cytometry using CyAn-ADP (Beckman Coulter, USA), and data were analyzed using Summit software (Beckman Coulter, USA).</div></div><div id="sec4_2_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i82" class="anchor-spacer"></div><h4 class="article-section__title" id="_i82">4.2.14.  Colony-Forming Assays</h4><div class="NLM_p last">The Murine MigA9 or human K562 cell line (1000 cells/well of 6-wells plates) was mixed with 1 mL of complete M3434 (murine) or H4535 (human) Colony-Forming Cell Assays MethoCult (Stemcell Technologies, France) in the presence or absence of 2.5 μM of the mentioned compounds in duplicates and plated in 6-well plates for 10 days. Full wells were then photographed under visible light and clones were quantified as individual clones for K562 and murine cell lines. For the MigA9 cell line, representative clones were photographed under visible light or for intrinsic compound fluorescence using an inverted fluorescence microscope with an excitation wavelength/bandpath of 350/50 nm and an emission wavelength/bandpath of 460/50 nm with a dry ×5 objective, and the individual clones were further identified as cfu-GM, cfu-M cfu-G or BFU-E based on morphological criteria as presented by the supplier.</div></div><div id="sec4_2_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83">4.2.15.  Statistics</h4><div class="NLM_p last">Student unpaired <i>t</i>-tests were performed using Prism Software 7.0 (GraphPad, La Jolla, CA, USA).</div></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i84"><a href="/doi/suppl/10.1021/acs.jmedchem.8b01448" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_25544" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_25544" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.8b01448" class="ext-link">10.1021/acs.jmedchem.8b01448</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Supplementary Tables S1 and S2. Supplementary Figures S1–S7. Supplementary chemistry section. Supplementary references. Representatives <sup>1</sup>H NMR spectra (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf">jm8b01448_si_001.pdf (2.94 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_002.csv">jm8b01448_si_002.csv (5.03 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01448" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74732" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74732" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marie-Hélène David-Cordonnier</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-9831-5577" title="Orcid link">http://orcid.org/0000-0001-9831-5577</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#7d101c0f14185015181118131853191c0b14193d14130e180f10531b0f"><span class="__cf_email__" data-cfemail="4a272b38232f67222f262f242f642e2b3c232e0a2324392f3827642c38">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sabine Depauw</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-8148-7900" title="Orcid link">http://orcid.org/0000-0002-8148-7900</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mélanie Lambert</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7472-4146" title="Orcid link">http://orcid.org/0000-0002-7472-4146</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Samy Jambon</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-7236-3239" title="Orcid link">http://orcid.org/0000-0001-7236-3239</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ananya Paul</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Paul Peixoto</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span>; 
    <span>Present Address:
                        EA3922, University of Bourgogne Franche-Comté, Besançon, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raja Nhili</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Laura Marongiu</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Martin Figeac</span> - <span class="hlFld-Affiliation affiliation">Functional and Structural Genomic Platform, Lille University, F-59000 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christelle Dassi</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Charles Paul-Constant</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benjamin Billoré</span> - <span class="hlFld-Affiliation affiliation">UMR-S1172-JPARC
(Jean-Pierre Aubert Research Center), INSERM, University of Lille,
Centre Hospitalier Universitaire de Lille, Institut pour la Recherche
sur le Cancer de Lille (IRCL), F-59045 Lille, France</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Arvind Kumar</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Abdelbasset A. Farahat</span> - <span class="hlFld-Affiliation affiliation">Department of Pharmaceutical Organic Chemistry, Faculty
of Pharmacy, 
    , and , Mansoura University, Mansoura 35516, Egypt</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mohamed A. Ismail</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Faculty of Science, 
    , and , Mansoura University, Mansoura 35516, Egypt</span>; 
    <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ekaterina Mineva</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Daniel P. Sweat</span> - <span class="hlFld-Affiliation affiliation">Department of
Chemistry and Physics, Augusta University, Augusta, Georgia 30904, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chad E. Stephens</span> - <span class="hlFld-Affiliation affiliation">Department of
Chemistry and Physics, Augusta University, Augusta, Georgia 30904, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David W. Boykin</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">W. David Wilson</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, Georgia State
University, Atlanta, Georgia 30303, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5225-5089" title="Orcid link">http://orcid.org/0000-0001-5225-5089</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.D., M.L., and S.J. contributed equally, and are considered by all the authors, to be joint first authors. The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i87">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75251" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75251" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work is supported by the Ligue contre le Cancer-Septentrion (Comité du Nord, Comité du Pas-de-Calais), the Association Laurette Fugain [ALF-2009/06, ALF-2016/05], the Association pour la Recherche sur le Cancer [ARC-SFI20111203669], the FEDER-Région Nord-Pas-de-Calais and the Institut pour la Recherche sur le Cancer de Lille (IRCL) to M.-H.D.-C., and by the National Institutes of Health [GM111749 to W.D.W. and D.W.B.]. M.L. thanks the University of Lille, the French Research Ministry and the French Hematological Society (SFH), S.J. thanks the ARC Foundation, P.P. and R.N. thanks the IRCL, for their respective PhD fellowships. M.-H.D.-C. thanks the Institut pour la Recherche sur le Cancer de Lille (IRCL) for technical expertise, Dr Nabeel Yaseen (Northwestern University, Chicago, USA) for the kind gift of pcDNA3-HOXA9 expressing vector, Dr Dominique Payet-Bornet and Dr Andrew Travers (Cambridge University, UK) for the gift of purified HMG-D100 protein, Dr Samuel Meignan (Centre Oscar Lambret, Lille) for access to the confocal inverted LSM880 microscope, the University of Lille, for access to the Pharos-PMI phosphor imaging equipment and the flow cytometry platform of the BioImaging Center of Lille Nord de France (BiCeL) at IRCL for access to equipment and technical formation.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">Akap13</td><td class="NLM_def"><p class="first last">A-kinase anchoring protein 13</p></td></tr><tr><td class="NLM_term">AML</td><td class="NLM_def"><p class="first last">acute myeloid leukemia</p></td></tr><tr><td class="NLM_term">Arfgef1</td><td class="NLM_def"><p class="first last">ADP ribosylation factor guanine nucleotide exchange factor 1</p></td></tr><tr><td class="NLM_term">dATP</td><td class="NLM_def"><p class="first last">deoxyadenosine triphosphate</p></td></tr><tr><td class="NLM_term">ATRA</td><td class="NLM_def"><p class="first last">all-trans retinoic acid</p></td></tr><tr><td class="NLM_term">APL</td><td class="NLM_def"><p class="first last">acute promyelocytic leukemia</p></td></tr><tr><td class="NLM_term">BCL6</td><td class="NLM_def"><p class="first last">B-cell lymphoma 6 protein</p></td></tr><tr><td class="NLM_term">CBP</td><td class="NLM_def"><p class="first last">CREB-binding protein</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichroism</p></td></tr><tr><td class="NLM_term">cfu</td><td class="NLM_def"><p class="first last">colony forming unit</p></td></tr><tr><td class="NLM_term">CGP</td><td class="NLM_def"><p class="first last">chemical and genetic perturbations</p></td></tr><tr><td class="NLM_term">CRABP1</td><td class="NLM_def"><p class="first last">cellular retinoic acid binding protein 1</p></td></tr><tr><td class="NLM_term">CTR</td><td class="NLM_def"><p class="first last">control</p></td></tr><tr><td class="NLM_term">DAPI</td><td class="NLM_def"><p class="first last">4′,6-diamidino-2-phenylindole</p></td></tr><tr><td class="NLM_term">DiOC<sub>6</sub></td><td class="NLM_def"><p class="first last">3,3′-dihexyloxacarbocyanine iodide</p></td></tr><tr><td class="NLM_term">DMS</td><td class="NLM_def"><p class="first last">dimethyl sulfide</p></td></tr><tr><td class="NLM_term">ELISA</td><td class="NLM_def"><p class="first last">enzyme-linked immunosorbent assay</p></td></tr><tr><td class="NLM_term">EMSA</td><td class="NLM_def"><p class="first last">electrophoretic mobility shift assay</p></td></tr><tr><td class="NLM_term">ERG</td><td class="NLM_def"><p class="first last">ETS-related gene</p></td></tr><tr><td class="NLM_term">EWS</td><td class="NLM_def"><p class="first last">Ewing’s sarcoma</p></td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste homolog 2</p></td></tr><tr><td class="NLM_term">FOXM1</td><td class="NLM_def"><p class="first last">forkhead box M1 transcription factor</p></td></tr><tr><td class="NLM_term">GFP</td><td class="NLM_def"><p class="first last">green fluorescent protein</p></td></tr><tr><td class="NLM_term">GMP</td><td class="NLM_def"><p class="first last">granulocyte/monocyte progenitors</p></td></tr><tr><td class="NLM_term">GSEA</td><td class="NLM_def"><p class="first last">gene set enrichment analysis</p></td></tr><tr><td class="NLM_term">HBS</td><td class="NLM_def"><p class="first last">HOXA9-binding site</p></td></tr><tr><td class="NLM_term">HMG</td><td class="NLM_def"><p class="first last">high mobility group</p></td></tr><tr><td class="NLM_term">HSF1</td><td class="NLM_def"><p class="first last">heat shock factor 1</p></td></tr><tr><td class="NLM_term">HP</td><td class="NLM_def"><p class="first last">hairpin</p></td></tr><tr><td class="NLM_term">HRP</td><td class="NLM_def"><p class="first last">horseradish peroxidase</p></td></tr><tr><td class="NLM_term">IPA</td><td class="NLM_def"><p class="first last">ingenuity pathway analysis</p></td></tr><tr><td class="NLM_term">Itgam</td><td class="NLM_def"><p class="first last">integrin alpha M</p></td></tr><tr><td class="NLM_term">KLF5</td><td class="NLM_def"><p class="first last">Kruppel like factor 5</p></td></tr><tr><td class="NLM_term">mdm</td><td class="NLM_def"><p class="first last">murine double minute</p></td></tr><tr><td class="NLM_term">MLL</td><td class="NLM_def"><p class="first last">mixed-lineage leukemia</p></td></tr><tr><td class="NLM_term">NPM1</td><td class="NLM_def"><p class="first last">nucleophosmin1</p></td></tr><tr><td class="NLM_term">NRF2</td><td class="NLM_def"><p class="first last">nuclear factor erythroid-2-related factor 2</p></td></tr><tr><td class="NLM_term">NS</td><td class="NLM_def"><p class="first last">nonspecific</p></td></tr><tr><td class="NLM_term">NUP98</td><td class="NLM_def"><p class="first last">nuclear pore complex protein 98</p></td></tr><tr><td class="NLM_term">Oasl2</td><td class="NLM_def"><p class="first last">2′-5′-oligoadenylate synthetase like 2</p></td></tr><tr><td class="NLM_term">PAX</td><td class="NLM_def"><p class="first last">paired box</p></td></tr><tr><td class="NLM_term">PBS</td><td class="NLM_def"><p class="first last">phosphate buffer saline</p></td></tr><tr><td class="NLM_term">PBX</td><td class="NLM_def"><p class="first last">pre-B-cell leukemia homeobox</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">protein data bank</p></td></tr><tr><td class="NLM_term">PI</td><td class="NLM_def"><p class="first last">propidium iodide</p></td></tr><tr><td class="NLM_term">RUNX1</td><td class="NLM_def"><p class="first last">runt related transcription factor 1</p></td></tr><tr><td class="NLM_term">SPR</td><td class="NLM_def"><p class="first last">surface plasmon resonance</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">TBS</td><td class="NLM_def"><p class="first last">tris buffer saline</p></td></tr><tr><td class="NLM_term">YAP</td><td class="NLM_def"><p class="first last">yes-associated protein</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i90">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21512" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21512" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 87 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factor Binding to DNA by Competing with DNA Binders as an Approach for Controlling Gene Expression</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150413154713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.2174%2F1568026615666150413154713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25866275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVGnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=1323-1358&author=M.+Bouhlelauthor=M.+Lambertauthor=M.-H.+David-Cordonnier&title=Targeting+Transcription+Factor+Binding+to+DNA+by+Competing+with+DNA+Binders+as+an+Approach+for+Controlling+Gene+Expression&doi=10.2174%2F1568026615666150413154713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Transcription Factor Binding to DNA by Competing with DNA Binders as an Approach for Controlling Gene Expression</span></div><div class="casAuthors">Bouhlel, Mohamed Amine; Lambert, Melanie; David-Cordonnier, Marie-Helene</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1323-1358</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Transcription factors are recognized as the master regulators of gene expression.  Interestingly, about 10% of the transcription factors described in mammals are up to date directly implicated in a very large no. of human diseases.  With the exception of ligand-inducible nuclear receptors, transcription factors have longtime been considered as 'undruggable' targets for therapeutics.  However, the significant breakthroughs in their protein biochem. and interactions with DNA at the structural level, together with increasing needs for new targeted-approaches particularly in cancers, has changed this postulate and opened the way for targeting transcription factors.  Along with a better knowledge of their specific DNA binding sequences by genome wide and high throughput sequencing assay, these informations make possible the potent targeting of the transcription factors by three approaches dependently of their mechanism of action.  In this review, we discuss the different physicochem. interactions between the transcription factors and the DNA helix, and the protein/protein interactions within a transcription factor complex and their impacts on the DNA structure.  In order to impair transcription factor activities, small mols. compds. can either act by direct interaction on the transcription factor, or by blocking the protein/protein interactions in a transcription complex, or by competing with the transcription factor itself and specifically targeting its cognate binding sequence.  For this latter mode of transcription targeting, we pay special attention to the DNA intercalating, alkylating or groove binders for transcription factor/DNA binding modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQKweR5JILLLVg90H21EOLACvtfcHk0lgkwbd8X2JDhw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVGnurY%253D&md5=14a54bbf9c2ffda02d7893f58a80f1b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150413154713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150413154713%26sid%3Dliteratum%253Aachs%26aulast%3DBouhlel%26aufirst%3DM.%26aulast%3DLambert%26aufirst%3DM.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520Transcription%2520Factor%2520Binding%2520to%2520DNA%2520by%2520Competing%2520with%2520DNA%2520Binders%2520as%2520an%2520Approach%2520for%2520Controlling%2520Gene%2520Expression%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D1323%26epage%3D1358%26doi%3D10.2174%2F1568026615666150413154713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting transcription factors by small compounds-Current strategies and future implications</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2015.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bcp.2015.12.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1-13&author=J.+Hagenbuchnerauthor=M.+J.+Ausserlechner&title=Targeting+transcription+factors+by+small+compounds-Current+strategies+and+future+implications&doi=10.1016%2Fj.bcp.2015.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuchner%26aufirst%3DJ.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520transcription%2520factors%2520by%2520small%2520compounds-Current%2520strategies%2520and%2520future%2520implications%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D107%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2015.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Portugal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barceló, F.</span></span> <span> </span><span class="NLM_article-title">Noncovalent Binding to DNA: Still a Target in Developing Anticancer Agents</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4108</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160902153511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.2174%2F0929867323666160902153511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27593961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2rtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=4108-4134&author=J.+Portugalauthor=F.+Barcel%C3%B3&title=Noncovalent+Binding+to+DNA%3A+Still+a+Target+in+Developing+Anticancer+Agents&doi=10.2174%2F0929867323666160902153511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Noncovalent Binding to DNA: Still a Target in Developing Anticancer Agents</span></div><div class="casAuthors">Portugal, Jose; Barcelo, Francisca</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">4108-4134</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">DNA-binding compds. are of extraordinary importance in medicine, accounting for a substantial portion of antitumor drugs in clin. usage.  However, their mechanisms of action remain sometimes incompletely understood.  This review critically examines two broad classes of mols. that bind noncovalently to DNA: intercalators and groove binders.  Intercalators bind to DNA by inserting their chromophore moiety between two consecutive base pairs, whereas groove binders fit into the grooves of DNA.  Noncovalent DNAinteractive drugs can recognize certain supramol. DNA structures such as the Gquadruplexes found in telomeres and in numerous gene promoters, and they can act as topoisomerase I and II poisons.  We discuss how DNA-binding compds. affect transcription and compete with protein factors for binding to consensus binding sites in gene promoters both in vitro and in cultured cancer cells.  Moreover, we comment on the design of mols. that can tightly and specifically bind to any desired target DNA, such as various hairpin polyamides which efficacy as chemotherapeutic agents is being evaluated.  At present, genome-wide studies, which provide details of events that may influence both cancer progression and therapeutic outcome, are a common way used to analyze the effects of DNA-binding compds.  A conclusive feature that emerges from reviewing the information on DNA-binding compds. is that both natural sources and chem. approaches can be productively used to obtain drugs to manipulate gene expression in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5f3wK3WVyv7Vg90H21EOLACvtfcHk0liPUz60bEEC7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2rtrrO&md5=0cf7e9633ef98408209a5a7ab486da31</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160902153511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160902153511%26sid%3Dliteratum%253Aachs%26aulast%3DPortugal%26aufirst%3DJ.%26aulast%3DBarcel%25C3%25B3%26aufirst%3DF.%26atitle%3DNoncovalent%2520Binding%2520to%2520DNA%253A%2520Still%2520a%2520Target%2520in%2520Developing%2520Anticancer%2520Agents%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D4108%26epage%3D4134%26doi%3D10.2174%2F0929867323666160902153511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jambon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factors for Cancer Treatment</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1479</span>, <span class="refDoi"> DOI: 10.3390/molecules23061479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.3390%2Fmolecules23061479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKjs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1479&author=M.+Lambertauthor=S.+Jambonauthor=S.+Depauwauthor=M.-H.+David-Cordonnier&title=Targeting+Transcription+Factors+for+Cancer+Treatment&doi=10.3390%2Fmolecules23061479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription factors for cancer treatment</span></div><div class="casAuthors">Lambert, Melanie; Jambon, Samy; Depauw, Sabine; David-Cordonnier, Marie-Helene</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1479/1-1479/51</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcription factors are involved in a large no. of human diseases such as cancers for which they account for about 20% of all oncogenes identified so far.  For long time, with the exception of ligand-inducible nuclear receptors, transcription factors were considered as "undruggable" targets.  Advances knowledge of these transcription factors, in terms of structure, function (expression, degrdn., interaction with co-factors and other proteins) and the dynamics of their mode of binding to DNA has changed this postulate and paved the way for new therapies targeted against transcription factors.  Here, we discuss various ways to target transcription factors in cancer models: by modulating their expression or degrdn., by blocking protein/protein interactions, by targeting the transcription factor itself to prevent its DNA binding either through a binding pocket or at the DNA-interacting site, some of these inhibitors being currently used or evaluated for cancer treatment.  Such different targeting of transcription factors by small mols. is facilitated by modern chem. developing a wide variety of original mols. designed to specifically abort transcription factor and by an increased knowledge of their pathol. implication through the use of new technologies in order to make it possible to improve therapeutic control of transcription factor oncogenic functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0VibdksQ77Vg90H21EOLACvtfcHk0liPUz60bEEC7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKjs7zK&md5=035d1312de71ad3c2dabec495edb498d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23061479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23061479%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DM.%26aulast%3DJambon%26aufirst%3DS.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520Transcription%2520Factors%2520for%2520Cancer%2520Treatment%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1479%26doi%3D10.3390%2Fmolecules23061479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo-Coco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicconi, L.</span></span> <span> </span><span class="NLM_article-title">History of Acute Promyelocytic Leukemia: A Tale of Endless Revolution</span>. <i>Mediterr. J. Hematol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e2011067</span> <span class="refDoi"> DOI: 10.4084/mjhid.2011.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.4084%2Fmjhid.2011.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22220264" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&author=F.+Lo-Cocoauthor=L.+Cicconi&title=History+of+Acute+Promyelocytic+Leukemia%3A+A+Tale+of+Endless+Revolution&doi=10.4084%2Fmjhid.2011.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4084%2Fmjhid.2011.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4084%252Fmjhid.2011.067%26sid%3Dliteratum%253Aachs%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DCicconi%26aufirst%3DL.%26atitle%3DHistory%2520of%2520Acute%2520Promyelocytic%2520Leukemia%253A%2520A%2520Tale%2520of%2520Endless%2520Revolution%26jtitle%3DMediterr.%2520J.%2520Hematol.%2520Infect.%2520Dis.%26date%3D2011%26volume%3D3%26doi%3D10.4084%2Fmjhid.2011.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ’undruggable’ cancer targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%99undruggable%E2%80%99+cancer+targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0ljr9lqaIe4hzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2599undruggable%25E2%2580%2599%2520cancer%2520targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tisato, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voltan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Secchiero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zauli, G.</span></span> <span> </span><span class="NLM_article-title">MDM2/X Inhibitors under Clinical Evaluation: Perspectives for the Management of Hematological Malignancies and Pediatric Cancer</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">133</span>, <span class="refDoi"> DOI: 10.1186/s13045-017-0500-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs13045-017-0500-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28673313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOru7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=133&author=V.+Tisatoauthor=R.+Voltanauthor=A.+Gonelliauthor=P.+Secchieroauthor=G.+Zauli&title=MDM2%2FX+Inhibitors+under+Clinical+Evaluation%3A+Perspectives+for+the+Management+of+Hematological+Malignancies+and+Pediatric+Cancer&doi=10.1186%2Fs13045-017-0500-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MDM2/X inhibitors under clinical evaluation: perspectives for the management of hematological malignancies and pediatric cancer</span></div><div class="casAuthors">Tisato, Veronica; Voltan, Rebecca; Gonelli, Arianna; Secchiero, Paola; Zauli, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">133/1-133/17</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The two murine double minute (MDM) family members MDM2 and MDMX are at the center of an intense clin. assessment as mol. target for the management of cancer.  Indeed, the two proteins act as regulators of P53, a well-known key controller of the cell cycle regulation and cell proliferation that, when altered, plays a direct role on cancer development and progression.  Several evidence demonstrated that functional aberrations of P53 in tumors are in most cases the consequence of alterations on the MDM2 and MDMX regulatory proteins, in particular in patients with hematol. malignancies where TP53 shows a relatively low frequency of mutation while MDM2 and MDMX are frequently found amplified/overexpressed.  The pharmacol. targeting of these two P53- regulators in order to restore or increase P53 expression and activity represents therefore a strategy for cancer therapy.  From the discovery of the Nutlins in 2004, several compds. have been developed and reported with the ability of targeting the P53-MDM2/X axis by inhibiting MDM2 and/or MDMX.  From natural compds. up to small mols. and stapled peptides, these MDM2/X pharmacol. inhibitors have been extensively studied, revealing different biol. features and different rate of efficacy when tested in in vitro and in vivo exptl. tumor models.  The data/evidence coming from the preclin. experimentation have allowed the identification of the most promising mols. and the setting of clin. studies for their evaluation as monotherapy or in therapeutic combination with conventional chemotherapy or with innovative therapeutic protocols in different tumor settings.  Preliminary results have been recently published reporting data about safety, tolerability, potential side effects, and efficacy of such therapeutic approaches.  In this light, the aim of this review is to give an updated overview about the state of the art of the clin. evaluation of MDM2/X inhibitor compds. with a special attention to hematol. malignancies and to the potential for the management of pediatric cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ1TexF8T_crVg90H21EOLACvtfcHk0ljr9lqaIe4hzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOru7jL&md5=70e587889b574f3909f7b11b36bf3933</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fs13045-017-0500-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-017-0500-5%26sid%3Dliteratum%253Aachs%26aulast%3DTisato%26aufirst%3DV.%26aulast%3DVoltan%26aufirst%3DR.%26aulast%3DGonelli%26aufirst%3DA.%26aulast%3DSecchiero%26aufirst%3DP.%26aulast%3DZauli%26aufirst%3DG.%26atitle%3DMDM2%252FX%2520Inhibitors%2520under%2520Clinical%2520Evaluation%253A%2520Perspectives%2520for%2520the%2520Management%2520of%2520Hematological%2520Malignancies%2520and%2520Pediatric%2520Cancer%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2017%26volume%3D10%26spage%3D133%26doi%3D10.1186%2Fs13045-017-0500-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeman-Neill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagat, G.</span></span> <span> </span><span class="NLM_article-title">BCL6 as a Therapeutic Target for Lymphoma</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1420782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1080%2F14728222.2018.1420782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29262721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVChu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=143-152&author=R.+J.+Leeman-Neillauthor=G.+Bhagat&title=BCL6+as+a+Therapeutic+Target+for+Lymphoma&doi=10.1080%2F14728222.2018.1420782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">BCL6 as a therapeutic target for lymphoma</span></div><div class="casAuthors">Leeman-Neill, Rebecca J.; Bhagat, Govind</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-152</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Introduction: B cell lymphoma 6 (BCL6) is a transcriptional repressor crit. for the development and maintenance of germinal centers (GCs), which are required for generation of an effective humoral immune response.  Genomic aberrations of BCL6, including mutations and translocations that occur during the GC reaction, as well as alterations of genes that regulate BCL6 expression, lead to sustained activity of BCL6, which promotes the development of GC-derived lymphomas.  Since many types of B cell non-Hodgkin lymphomas (B-NHL) arise from neoplastic transformation of GC B cells and a high proportion harbor genetic lesions that deregulate BCL6 expression, inhibition of BCL6 has emerged as an attractive therapeutic strategy for lymphomas.  Areas covered: This review examines the rationale for and challenges in therapeutic targeting of BCL6 in lymphomas.  We describe approaches that have been used and are currently being considered for inhibition of BCL6.  Expert opinion: Several BCL6 inhibiting agents, including peptidomimetics, small mols., and natural compds., most of which target the BTB domain of the protein at the corepressor binding site, have been developed with demonstration of anti-lymphoma activity in preclin. models.  Future clin. trials will be important to investigate the efficacy of targeting BCL6 in B-NHL (and other neoplasms), particularly in combination with other therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_UsynF__UrVg90H21EOLACvtfcHk0ljr9lqaIe4hzA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVChu7Y%253D&md5=dbbd71e6ce8d84829c366bccab0205b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1420782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1420782%26sid%3Dliteratum%253Aachs%26aulast%3DLeeman-Neill%26aufirst%3DR.%2BJ.%26aulast%3DBhagat%26aufirst%3DG.%26atitle%3DBCL6%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Lymphoma%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D143%26epage%3D152%26doi%3D10.1080%2F14728222.2018.1420782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furtek, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reigan, P.</span></span> <span> </span><span class="NLM_article-title">Strategies and Approaches of Targeting STAT3 for Cancer Treatment</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">308</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=308-318&author=S.+L.+Furtekauthor=D.+S.+Backosauthor=C.+J.+Mathesonauthor=P.+Reigan&title=Strategies+and+Approaches+of+Targeting+STAT3+for+Cancer+Treatment&doi=10.1021%2Facschembio.5b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and Approaches of Targeting STAT3 for Cancer Treatment</span></div><div class="casAuthors">Furtek, Steffanie L.; Backos, Donald S.; Matheson, Christopher J.; Reigan, Philip</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">308-318</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor that regulates the expression of genes related to cell cycle, cell survival, and immune response assocd. with cancer progression and malignancy in a no. of cancer types.  Once activated, STAT3 forms a homodimer and translocates to the nucleus where it binds DNA promoting the translation of target genes assocd. with antiapoptosis, angiogenesis, and invasion/migration.  In normal cells, levels of activated STAT3 remain transient; however, STAT3 remains constitutively active in approx. 70% of human solid tumors.  The pivotal role of STAT3 in tumor progression has promoted a campaign in drug discovery to identify small mols. that disrupt the function of STAT3.  A range of approaches have been used to identify novel small mol. inhibitors of STAT3, including high-throughput screening of chem. libraries, computational-based virtual screening, and fragment-based design strategies.  The most common approaches in targeting STAT3 activity are either via the inhibition of tyrosine kinases capable of phosphorylating and thereby activating STAT3 or by preventing the formation of functional STAT3 dimers through disruption of the SH2 domains.  However, the targeting of the STAT3 DNA-binding domain and disruption of binding of STAT3 to its DNA promoter have not been thoroughly examd., mainly due to the lack of adequate assay systems.  This review summarizes the development of STAT3 inhibitors organized by the approach used to inhibit STAT3, the current inhibitors of each class, and the assay systems used to evaluate STAT3 inhibition and offers an insight into future approaches for small mol. STAT3 inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokCygAKrPD6rVg90H21EOLACvtfcHk0lgSsGyCyRu7zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOqsA%253D%253D&md5=379321e035a5dbce720e1c787da55aad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00945%26sid%3Dliteratum%253Aachs%26aulast%3DFurtek%26aufirst%3DS.%2BL.%26aulast%3DBackos%26aufirst%3DD.%2BS.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DReigan%26aufirst%3DP.%26atitle%3DStrategies%2520and%2520Approaches%2520of%2520Targeting%2520STAT3%2520for%2520Cancer%2520Treatment%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D308%26epage%3D318%26doi%3D10.1021%2Facschembio.5b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu-Chittenden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span> <span> </span><span class="NLM_article-title">Genetic and Pharmacological Disruption of the TEAD-YAP Complex Suppresses the Oncogenic Activity of YAP</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1101/gad.192856.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1101%2Fgad.192856.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22677547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVegs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1300-1305&author=Y.+Liu-Chittendenauthor=B.+Huangauthor=J.+S.+Shimauthor=Q.+Chenauthor=S.-J.+Leeauthor=R.+A.+Andersauthor=J.+O.+Liuauthor=D.+Pan&title=Genetic+and+Pharmacological+Disruption+of+the+TEAD-YAP+Complex+Suppresses+the+Oncogenic+Activity+of+YAP&doi=10.1101%2Fgad.192856.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and pharmacological disruption of the TEAD-YAP complex suppresses the oncogenic activity of YAP</span></div><div class="casAuthors">Liu-Chittenden, Yi; Huang, Bo; Shim, Joong Sup; Chen, Qian; Lee, Se-Jin; Anders, Robert A.; Liu, Jun O.; Pan, Duojia</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1300-1305</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The Drosophila TEAD ortholog Scalloped is required for Yki-mediated overgrowth but is largely dispensable for normal tissue growth, suggesting that its mammalian counterpart may be exploited for selective inhibition of oncogenic growth driven by YAP hyperactivation.  Here we test this hypothesis genetically and pharmacol.  We show that a dominant-neg. TEAD mol. does not perturb normal liver growth but potently suppresses hepatomegaly/tumorigenesis resulting from YAP overexpression or Neurofibromin 2 (NF2)/Merlin inactivation.  We further identify verteporfin as a small mol. that inhibits TEAD-YAP assocn. and YAP-induced liver overgrowth.  These findings provide proof of principle that inhibiting TEAD-YAP interactions is a pharmacol. viable strategy against the YAP oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiYAxCKwHiDLVg90H21EOLACvtfcHk0lijRikWUHXUGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVegs7%252FL&md5=dcd42db76d9b45bd15a276e101864917</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.192856.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.192856.112%26sid%3Dliteratum%253Aachs%26aulast%3DLiu-Chittenden%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DShim%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26aulast%3DPan%26aufirst%3DD.%26atitle%3DGenetic%2520and%2520Pharmacological%2520Disruption%2520of%2520the%2520TEAD-YAP%2520Complex%2520Suppresses%2520the%2520Oncogenic%2520Activity%2520of%2520YAP%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D1300%26epage%3D1305%26doi%3D10.1101%2Fgad.192856.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaisier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coevoet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Molecular Features of the YAP Inhibitor Verteporfin: Synthesis of Hexasubstituted Dipyrrins as Potential Inhibitors of YAP/TAZ, the Downstream Effectors of the Hippo Pathway</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">954</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fcmdc.201700063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28334506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlSgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=954-961&author=F.+Gibaultauthor=F.+Baillyauthor=M.+Corvaisierauthor=M.+Coevoetauthor=G.+Huetauthor=P.+Melnykauthor=P.+Cotelle&title=Molecular+Features+of+the+YAP+Inhibitor+Verteporfin%3A+Synthesis+of+Hexasubstituted+Dipyrrins+as+Potential+Inhibitors+of+YAP%2FTAZ%2C+the+Downstream+Effectors+of+the+Hippo+Pathway&doi=10.1002%2Fcmdc.201700063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Features of the YAP Inhibitor Verteporfin: Synthesis of Hexasubstituted Dipyrrins as Potential Inhibitors of YAP/TAZ, the Downstream Effectors of the Hippo Pathway</span></div><div class="casAuthors">Gibault, Floriane; Bailly, Fabrice; Corvaisier, Matthieu; Coevoet, Mathilde; Huet, Guillemette; Melnyk, Patricia; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">954-961</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Porphyrin derivs., in particular verteporfin (VP), a photosensitizer initially designed for cancer therapy, have been identified as inhibitors of the YAP-TEAD interaction and transcriptional activity.  Herein the authors report the efficient convergent synthesis of the dipyrrin half of protoporphyrin IX di-Me ester (PPIX-DME), in which the sensitive vinyl group was created at the final stage by a dehydroiodination reaction.  Two other dipyrrin derivs. were synthesized, including dipyrrin 19 [(Z)-2-((3,5-dimethyl-4-vinyl-2H-pyrrol-2-ylidene)methyl)-3,5-dimethyl-4-vinyl-1H-pyrrole], contg. two vinyl groups.  The authors found that VP and dipyrrin 19 showed significant inhibitory effects on TEAD transcriptional activity in MDA-MB-231 human breast cancer cells, whereas other compds. did not show significant changes.  In addn., the authors obsd. a marked decrease in both YAP and TAZ levels following VP treatment, whereas dipyrrin 19 treatment primarily decreased the levels of YAP and receptor kinase AXL, a downstream target of YAP.  Together, the authors' data suggest that, due to their chem. structures, porphyrin- and dipyrrin-related derivs. can directly target YAP and/or TAZ proteins and inhibit TEAD transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppSdVREJKEorVg90H21EOLACvtfcHk0lijRikWUHXUGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlSgsL8%253D&md5=d75845ed2c4a1ddc1f318a03555c12f8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700063%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DCorvaisier%26aufirst%3DM.%26aulast%3DCoevoet%26aufirst%3DM.%26aulast%3DHuet%26aufirst%3DG.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DMolecular%2520Features%2520of%2520the%2520YAP%2520Inhibitor%2520Verteporfin%253A%2520Synthesis%2520of%2520Hexasubstituted%2520Dipyrrins%2520as%2520Potential%2520Inhibitors%2520of%2520YAP%252FTAZ%252C%2520the%2520Downstream%2520Effectors%2520of%2520the%2520Hippo%2520Pathway%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D954%26epage%3D961%26doi%3D10.1002%2Fcmdc.201700063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gang, E.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, M.</span></span> <span> </span><span class="NLM_article-title">CBP/Catenin antagonists: Targeting LSCs’ Achilles heel</span>. <i>Exp. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.exphem.2017.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.exphem.2017.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28479420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2017&pages=1-11&author=Y.-M.+Kimauthor=E.-J.+Gangauthor=M.+Kahn&title=CBP%2FCatenin+antagonists%3A+Targeting+LSCs%E2%80%99+Achilles+heel&doi=10.1016%2Fj.exphem.2017.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/Catenin antagonists: Targeting LSCs' Achilles heel</span></div><div class="casAuthors">Kim, Yong-Mi; Gang, Eun-Ji; Kahn, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Hematology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">EXHMA6</span>;
        ISSN:<span class="NLM_cas:issn">0301-472X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs), including leukemia stem cells (LSCs), exhibit self-renewal capacity and differentiation potential and have the capacity to maintain or renew and propagate a tumor/leukemia.  The initial isolation of CSCs/LSCs was in adult myelogenous leukemia, although more recently, the existence of CSCs in a wide variety of other cancers has been reported.  CSCs, in general, and LSCs, specifically with respect to this review, are responsible for initiation of disease, therapeutic resistance and ultimately disease relapse.  One key focus in cancer research over the past decade has been the development of therapies that safely eliminate the LSC/CSC population.  One major obstacle to this goal is the identification of key mechanisms that distinguish LSCs from normal endogenous hematopoietic stem cells.  An addnl. daunting feature that has recently come to light with advances in next-generation sequencing and single-cell sequencing is the heterogeneity within leukemias/tumors, with multiple combinations of mutations, gain and loss of function of genes, and so on being capable of driving disease, even within the CSC/LSC population.  The focus of this review/perspective is on our work in identifying and validating, in both chronic myelogenous leukemia and acute lymphoblastic leukemia, a safe and efficacious mechanism to target an evolutionarily conserved signaling nexus, which constitutes a common "Achilles heel" for LSCs/CSCs, using small mol.-specific CBP/catenin antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI5h5QcvGrdbVg90H21EOLACvtfcHk0lg1hpQU1VTKeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSqsr4%253D&md5=848d6692beeb60072a7726ed055b8b46</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.exphem.2017.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.exphem.2017.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.-M.%26aulast%3DGang%26aufirst%3DE.-J.%26aulast%3DKahn%26aufirst%3DM.%26atitle%3DCBP%252FCatenin%2520antagonists%253A%2520Targeting%2520LSCs%25E2%2580%2599%2520Achilles%2520heel%26jtitle%3DExp.%2520Hematol.%26date%3D2017%26volume%3D52%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.exphem.2017.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pariollaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chitnis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fornwald, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of PERK Activators by Phenotypic Screening and Their Effects on NRF2 Activation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0119738</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0119738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1371%2Fjournal.pone.0119738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25780921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWgtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+Xieauthor=M.+Pariollaudauthor=W.+E.+Wixtedauthor=N.+Chitnisauthor=J.+Fornwaldauthor=M.+Truongauthor=C.+Paoauthor=Y.+Liuauthor=R.+S.+Amesauthor=J.+Callahanauthor=R.+Solariauthor=Y.+Sanchezauthor=A.+Diehlauthor=H.+Li&title=Identification+and+Characterization+of+PERK+Activators+by+Phenotypic+Screening+and+Their+Effects+on+NRF2+Activation&doi=10.1371%2Fjournal.pone.0119738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of PERK activators by phenotypic screening and their effects on NRF2 activation</span></div><div class="casAuthors">Xie, Wensheng; Pariollaud, Marie; Wixted, William E.; Chitnis, Nilesh; Fornwald, James; Truong, Maggie; Pao, Christina; Liu, Yan; Ames, Robert S.; Callahan, James; Solari, Roberto; Sanchez, Yolanda; Diehl, Alan; Li, Hu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0119738/1-e0119738/21</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Endoplasmic reticulum stress plays a crit. role to restore the homeostasis of protein prodn. in eukaryotic cells.  This vital process is hence involved in many types of diseases including COPD.  PERK, one branch in the ER stress signaling pathways, has been reported to activate NRF2 signaling pathway, a known protective response to COPD.  Based on this scientific rationale, we aimed to identify PERK activators as a mechanism to achieve NRF2 activation.  In this report, we describe a phenotypic screening assay to identify PERK activators.  This assay measures phosphorylation of GFP-tagged eIF2α upon PERK activation via a cell-based LanthaScreen technol.  To obtain a robust assay with sufficient signal to background and low variation, multiple parameters were optimized including GFP-tagged eIF2α BacMam concn., cell d. and serum concn.  The assay was validated by a tool compd., Thapsigargin, which induces phosphorylation of eIF2α.  In our assay, this compd. showed maximal signal window of approx. 2.5-fold with a pEC50 of 8.0, consistent with literature reports.  To identify novel PERK activators through phosphorylation of eIF2α, a focused set of 8,400 compds. was screened in this assay at 10 μM.  A no. of hits were identified and validated.  The mol. mechanisms for several selected hits were further characterized in terms of PERK activation and effects on PERK downstream components.  Specificity of these compds. in activating PERK was demonstrated with a PERK specific inhibitor and in PERK knockout mouse embryonic fibroblast (MEF) cells.  In addn., these hits showed NRF2-dependent anti-oxidant gene induction.  In summary, our phenotypic screening assay is demonstrated to be able to identify PERK specific activators.  The identified PERK activators could potentially be used as chem. probes to further investigate this pathway as well as the link between PERK activation and NRF2 pathway activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4FS7McfUsXrVg90H21EOLACvtfcHk0lg1hpQU1VTKeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWgtbrP&md5=037fab9d359f34f3f48cd84b9783baf6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0119738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0119738%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DW.%26aulast%3DPariollaud%26aufirst%3DM.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DChitnis%26aufirst%3DN.%26aulast%3DFornwald%26aufirst%3DJ.%26aulast%3DTruong%26aufirst%3DM.%26aulast%3DPao%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DAmes%26aufirst%3DR.%2BS.%26aulast%3DCallahan%26aufirst%3DJ.%26aulast%3DSolari%26aufirst%3DR.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DDiehl%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520PERK%2520Activators%2520by%2520Phenotypic%2520Screening%2520and%2520Their%2520Effects%2520on%2520NRF2%2520Activation%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0119738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessum, N. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rye, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqua, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Fee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henley, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">te Poele, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">201</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=180-201&author=M.+D.+Cheesemanauthor=N.+E.+A.+Chessumauthor=C.+S.+Ryeauthor=A.+E.+Pasquaauthor=M.+J.+Tuckerauthor=B.+Wildingauthor=L.+E.+Evansauthor=S.+Lepriauthor=M.+Richardsauthor=S.+Y.+Sharpauthor=S.+Aliauthor=M.+Rowlandsauthor=L.+O%E2%80%99Feeauthor=A.+Miahauthor=A.+Hayesauthor=A.+T.+Henleyauthor=M.+Powersauthor=R.+te+Poeleauthor=E.+De+Billyauthor=L.+Pellegrinoauthor=F.+Raynaudauthor=R.+Burkeauthor=R.+L.+M.+van+Montfortauthor=S.+A.+Ecclesauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+a+Chemical+Probe+Bisamide+%28CCT251236%29%3A+An+Orally+Bioavailable+Efficacious+Pirin+Ligand+from+a+Heat+Shock+Transcription+Factor+1+%28HSF1%29+Phenotypic+Screen&doi=10.1021%2Facs.jmedchem.6b01055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01055%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DTucker%26aufirst%3DM.%2BJ.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Fee%26aufirst%3DL.%26aulast%3DMiah%26aufirst%3DA.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%2BT.%26aulast%3DPowers%26aufirst%3DM.%26aulast%3Dte%2BPoele%26aufirst%3DR.%26aulast%3DDe%2BBilly%26aufirst%3DE.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520Chemical%2520Probe%2520Bisamide%2520%2528CCT251236%2529%253A%2520An%2520Orally%2520Bioavailable%2520Efficacious%2520Pirin%2520Ligand%2520from%2520a%2520Heat%2520Shock%2520Transcription%2520Factor%25201%2520%2528HSF1%2529%2520Phenotypic%2520Screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D60%26spage%3D180%26epage%3D201%26doi%3D10.1021%2Facs.jmedchem.6b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranghini, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressler, G. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Pax2 Transcription Activation with a Small Molecule That Targets the DNA Binding Domain</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">734</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVarsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=724-734&author=E.+Grimleyauthor=C.+Liaoauthor=E.+J.+Ranghiniauthor=Z.+Nikolovska-Coleskaauthor=G.+R.+Dressler&title=Inhibition+of+Pax2+Transcription+Activation+with+a+Small+Molecule+That+Targets+the+DNA+Binding+Domain&doi=10.1021%2Facschembio.6b00782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Pax2 Transcription Activation with a Small Molecule that Targets the DNA Binding Domain</span></div><div class="casAuthors">Grimley, Edward; Liao, Chenzhong; Ranghini, Egon J.; Nikolovska-Coleska, Zaneta; Dressler, Gregory R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">724-734</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pax gene family encodes DNA binding transcription factors that control crit. steps in embryonic development and differentiation of specific cell lineages.  Often, Pax proteins are re-expressed or ectopically expressed in cancer and other diseases of abnormal proliferation, making them attractive targets for tissue specific inhibition by small mols.  The authors used a homol. model of the Pax2 paired domain and a virtual screen to identify small mols. that can inhibit binding of the paired domain to DNA and Pax2 mediated transcription activation.  Candidates from the virtual screen were then confirmed in a cell based Pax2 transactivation assay.  Subsequently, the authors tested analogs of these hits to identify a single compd. that effectively blocked Pax2 activity and DNA binding with a Kd of 1.35-1.5 μM.  The compd., termed EG1, was used to inhibit embryonic kidney development, a process directly dependent on Pax2 activity.  Furthermore, the authors show that EG1 can inhibit proliferation of Pax2 pos. renal and ovarian cancer cell lines but has little effect on Pax2 neg. cancer cells.  These data confirm that small mols. targeting the DNA binding paired domain can be identified and may be good lead compds. for developing tissue and cell-type specific anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8YEmyueW-0LVg90H21EOLACvtfcHk0lirjcXRy470gw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVarsLc%253D&md5=9a5790d3a402a1a66932ae97f4958879</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00782%26sid%3Dliteratum%253Aachs%26aulast%3DGrimley%26aufirst%3DE.%26aulast%3DLiao%26aufirst%3DC.%26aulast%3DRanghini%26aufirst%3DE.%2BJ.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DDressler%26aufirst%3DG.%2BR.%26atitle%3DInhibition%2520of%2520Pax2%2520Transcription%2520Activation%2520with%2520a%2520Small%2520Molecule%2520That%2520Targets%2520the%2520DNA%2520Binding%2520Domain%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D724%26epage%3D734%26doi%3D10.1021%2Facschembio.6b00782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V. P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">DNA-Binding Small Molecules as Inhibitors of Transcription Factors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1002/med.21266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fmed.21266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22549740" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=823-846&author=C.-H.+Leungauthor=D.+S.-H.+Chanauthor=V.+P.-Y.+Maauthor=D.-L.+Ma&title=DNA-Binding+Small+Molecules+as+Inhibitors+of+Transcription+Factors&doi=10.1002%2Fmed.21266"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fmed.21266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21266%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DMa%26aufirst%3DV.%2BP.-Y.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DDNA-Binding%2520Small%2520Molecules%2520as%2520Inhibitors%2520of%2520Transcription%2520Factors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2012%26volume%3D33%26spage%3D823%26epage%3D846%26doi%3D10.1002%2Fmed.21266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fontaine, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">François, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Manipulation of Transcription Factor Protein-Protein Interactions: Opportunities and Obstacles</span>. <i>Cell Regen.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi"> DOI: 10.1186/s13619-015-0015-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs13619-015-0015-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC2MjhslyjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=2&author=F.+Fontaineauthor=J.+Overmanauthor=M.+Fran%C3%A7ois&title=Pharmacological+Manipulation+of+Transcription+Factor+Protein-Protein+Interactions%3A+Opportunities+and+Obstacles&doi=10.1186%2Fs13619-015-0015-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological manipulation of transcription factor protein-protein interactions: opportunities and obstacles</span></div><div class="casAuthors">Fontaine Frank; Overman Jeroen; Francois Mathias</div><div class="citationInfo"><span class="NLM_cas:title">Cell regeneration (London, England)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2</span>
        ISSN:<span class="NLM_cas:issn">2045-9769</span>.
    </div><div class="casAbstract">Much research on transcription factor biology and their genetic pathways has been undertaken over the last 30 years, especially in the field of developmental biology and cancer.  Yet, very little is known about the molecular modalities of highly dynamic interactions between transcription factors, genomic DNA, and protein partners.  Methodological breakthroughs such as RNA-seq (RNA-sequencing), ChIP-seq (chromatin immunoprecipitation sequencing), RIME (rapid immunoprecipitation mass spectrometry of endogenous proteins), and single-molecule imaging will dramatically accelerate the discovery rate of their molecular mode of action in the next few years.  From a pharmacological viewpoint, conventional methods used to target transcription factor activity with molecules mimicking endogenous ligands fail to achieve high specificity and are limited by a lack of identification of new molecular targets.  Protein-protein interactions are likely to represent one of the next major classes of therapeutic targets.  Transcription factors, known to act mostly via protein-protein interaction, may well be at the forefront of this type of drug development.  One hurdle in this field remains the difficulty to collate structural data into meaningful information for rational drug design.  Another hurdle is the lack of chemical libraries meeting the structural requirements of protein-protein interaction disruption.  As more attempts at modulating transcription factor activity are undertaken, valuable knowledge will be accumulated on the modality of action required to modulate transcription and how these findings can be applied to developing transcription factor drugs.  Key discoveries will spawn into new therapeutic approaches not only as anticancer targets but also for other indications, such as those with an inflammatory component including neurodegenerative disorders, diabetes, and chronic liver and kidney diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvMXrDUnx-8JSWdIdmEou5fW6udTcc2eaDRDeEvthWp7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjhslyjsQ%253D%253D&md5=e092b82911757d58c596da8bceea04e0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs13619-015-0015-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13619-015-0015-x%26sid%3Dliteratum%253Aachs%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DOverman%26aufirst%3DJ.%26aulast%3DFran%25C3%25A7ois%26aufirst%3DM.%26atitle%3DPharmacological%2520Manipulation%2520of%2520Transcription%2520Factor%2520Protein-Protein%2520Interactions%253A%2520Opportunities%2520and%2520Obstacles%26jtitle%3DCell%2520Regen.%26date%3D2015%26volume%3D4%26spage%3D2%26doi%3D10.1186%2Fs13619-015-0015-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blume, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. M.</span></span> <span> </span><span class="NLM_article-title">Mithramycin Inhibits SP1 Binding and Selectively Inhibits Transcriptional Activity of the Dihydrofolate Reductase Gene in Vitro and in Vivo</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">1613</span>– <span class="NLM_lpage">1621</span>, <span class="refDoi"> DOI: 10.1172/jci115474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci115474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1834700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK38XhsFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=1613-1621&author=S.+W.+Blumeauthor=R.+C.+Snyderauthor=R.+Rayauthor=S.+Thomasauthor=C.+A.+Kollerauthor=D.+M.+Miller&title=Mithramycin+Inhibits+SP1+Binding+and+Selectively+Inhibits+Transcriptional+Activity+of+the+Dihydrofolate+Reductase+Gene+in+Vitro+and+in+Vivo&doi=10.1172%2Fjci115474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mithramycin inhibits SP1 binding and selectively inhibits transcriptional activity of the dihydrofolate reductase gene in vitro and in vivo</span></div><div class="casAuthors">Blume, Scott W.; Snyder, Richard C.; Ray, Ratna; Thomas, Shelia; Koller, Charles A.; Miller, Donald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1613-21</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">The promoter of the human dihydrofolate reductase (DHFR) gene contains two consensus binding sites for the DNA binding protein Sp1.  DNAse protection and gel mobility shift assays demonstrate binding of recombinant Sp1 to both decanucleotide Sp1 binding sequences which are located 49 and 14 base pairs upstream of the transcription start site.  The more distal of the two binding sites exhibits a somewhat higher affinity for Sp1.  The G-C specific DNA binding drug mithramycin binds to both consensus sequences and prevents subsequent Sp1 binding.  Promoter-dependent in vitro transcription of a DHER template is selectively inhibited by mithramycin when compared to the human H2b histone gene.  A similar effect is also noted in vivo.  Mithramycin treatment of MCF-7 human breast carcinoma cells contg. an amplified DHFR gene induces selective inhibition of DHFR transcription initiation, resulting in a decline in DHFR mRNA level and enzyme activity.  This selective inhibition of DHFR expression suggests that it is possible to modulate the overexpression of the DHFR gene in methotrexate resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDg0_h53y-hrVg90H21EOLACvtfcHk0liwU2a05D_eIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsFCmtg%253D%253D&md5=4972e7fd70c21156e6f6db63a1301358</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1172%2Fjci115474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci115474%26sid%3Dliteratum%253Aachs%26aulast%3DBlume%26aufirst%3DS.%2BW.%26aulast%3DSnyder%26aufirst%3DR.%2BC.%26aulast%3DRay%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DKoller%26aufirst%3DC.%2BA.%26aulast%3DMiller%26aufirst%3DD.%2BM.%26atitle%3DMithramycin%2520Inhibits%2520SP1%2520Binding%2520and%2520Selectively%2520Inhibits%2520Transcriptional%2520Activity%2520of%2520the%2520Dihydrofolate%2520Reductase%2520Gene%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1991%26volume%3D88%26spage%3D1613%26epage%3D1621%26doi%3D10.1172%2Fjci115474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grohar, P.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woldemichael, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currier, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span> <span> </span><span class="NLM_article-title">Identification of an Inhibitor of the EWS-FLI1 Oncogenic Transcription Factor by High-Throughput Screening</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">978</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fjnci%2Fdjr156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=21653923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1Cltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=962-978&author=P.%0AJ.+Groharauthor=G.+M.+Woldemichaelauthor=L.+B.+Griffinauthor=A.+Mendozaauthor=Q.-R.+Chenauthor=C.+Yeungauthor=D.+G.+Currierauthor=S.+Davisauthor=C.+Khannaauthor=J.+Khanauthor=J.+B.+McMahonauthor=L.+J.+Helman&title=Identification+of+an+Inhibitor+of+the+EWS-FLI1+Oncogenic+Transcription+Factor+by+High-Throughput+Screening&doi=10.1093%2Fjnci%2Fdjr156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an Inhibitor of the EWS-FLI1 Oncogenic Transcription Factor by High-Throughput Screening</span></div><div class="casAuthors">Grohar, Patrick J.; Woldemichael, Girma M.; Griffin, Laurie B.; Mendoza, Arnulfo; Chen, Qing-Rong; Yeung, Choh; Currier, Duane G.; Davis, Sean; Khanna, Chand; Khan, Javed; McMahon, James B.; Helman, Lee J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">962-978</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background Chromosomal translocations generating oncogenic transcription factors are the hallmark of a variety of tumors, including many sarcomas.  Ewing sarcoma family of tumors (ESFTs) are characterized by the t(11;22)(q24;q12) translocation that generates the Ewing sarcoma breakpoint region 1 and Friend leukemia virus integration 1 (EWS-FLI1) fusion transcription factor responsible for the highly malignant phenotype of this tumor.  Although continued expression of EWS-FLI1 is believed to be crit. for ESFT cell survival, a clin. effective small-mol. inhibitor remains elusive likely because EWS-FLI1 is a transcription factor and therefore widely felt to be "undruggable.  " Methods We developed a high-throughput screen to evaluate more than 50 000 compds. for inhibition of EWS-FLI1 activity in TC32 ESFT cells.  We used a TC32 cell-based luciferase reporter screen using the EWS-FLI1 downstream target NR0B1 promoter and a gene signature secondary screen to sort and prioritize the compds.  We characterized the lead compd., mithramycin, based on its ability to inhibit EWS-FLI1 activity in vitro using microarray expression profiling, quant. reverse transcription-polymerase chain reaction, and immunoblot anal., and in vivo using immunohistochem.  We studied the impact of this inhibition on cell viability in vitro and on tumor growth in ESFT xenograft models in vivo (n = 15-20 mice per group).  All statistical tests were two-sided.  Results Mithramycin inhibited expression of EWS-FLI1 downstream targets at the mRNA and protein levels and decreased the growth of ESFT cells at half maximal inhibitory concns. between 10 (95% confidence interval [CI] = 8 to 13 nM) and 15 nM (95% CI = 13 to 19 nM).  Mithramycin suppressed the growth of two different ESFT xenograft tumors and prolonged the survival of ESFT xenograft-bearing mice by causing a decrease in mean tumor vol.  For example, in the TC32 xenograft model, on day 15 of treatment, the mean tumor vol. for the mithramycin-treated mice was approx. 3% of the tumor vol. obsd. in the control mice (mithramycin vs control: 69 vs 2388 mm3, difference = 2319 mm3, 95% CI = 1766 to 2872 mm3, P < .001).  Conclusion Mithramycin inhibits EWS-FLI1 activity and demonstrates ESFT antitumor activity both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-zpZMmMy4bLVg90H21EOLACvtfcHk0lhVFj7Iaw4PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1Cltro%253D&md5=814cc95d2b54de9b275cc1df8a1b19ee</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr156%26sid%3Dliteratum%253Aachs%26aulast%3DGrohar%26aufirst%3DP.%2BJ.%26aulast%3DWoldemichael%26aufirst%3DG.%2BM.%26aulast%3DGriffin%26aufirst%3DL.%2BB.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DQ.-R.%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DCurrier%26aufirst%3DD.%2BG.%26aulast%3DDavis%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DMcMahon%26aufirst%3DJ.%2BB.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26atitle%3DIdentification%2520of%2520an%2520Inhibitor%2520of%2520the%2520EWS-FLI1%2520Oncogenic%2520Transcription%2520Factor%2520by%2520High-Throughput%2520Screening%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26spage%3D962%26epage%3D978%26doi%3D10.1093%2Fjnci%2Fdjr156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grohar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sissung, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnaldez, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitcomb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widemann, B. C.</span></span> <span> </span><span class="NLM_article-title">A phase I/II trial and pharmacokinetic study of mithramycin in children and adults with refractory Ewing sarcoma and EWS-FLI1 fusion transcript</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1007/s00280-017-3382-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1007%2Fs00280-017-3382-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28735378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Wru77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2017&pages=645-652&author=P.+J.+Groharauthor=J.+Glodauthor=C.+J.+Peerauthor=T.+M.+Sissungauthor=F.+I.+Arnaldezauthor=L.+Longauthor=W.+D.+Figgauthor=P.+Whitcombauthor=L.+J.+Helmanauthor=B.+C.+Widemann&title=A+phase+I%2FII+trial+and+pharmacokinetic+study+of+mithramycin+in+children+and+adults+with+refractory+Ewing+sarcoma+and+EWS-FLI1+fusion+transcript&doi=10.1007%2Fs00280-017-3382-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A phase I/II trial and pharmacokinetic study of mithramycin in children and adults with refractory Ewing sarcoma and EWS-FLI1 fusion transcript</span></div><div class="casAuthors">Grohar, Patrick J.; Glod, John; Peer, Cody J.; Sissung, Tristan M.; Arnaldez, Fernanda I.; Long, Lauren; Figg, William D.; Whitcomb, Patricia; Helman, Lee J.; Widemann, Brigitte C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-652</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: In a preclin. drug screen, mithramycin was identified as a potent inhibitor of the Ewing sarcoma EWS-FLI1 transcription factor.  We conducted a phase I/II trial to det. the dose-limiting toxicities (DLT), max. tolerated dose (MTD), and pharmacokinetics (PK) of mithramycin in children with refractory solid tumors, and the activity in children and adults with refractory Ewing sarcoma.  Patients and methods: Mithramycin was administered i.v. over 6 h once daily for 7 days for 28 day cycles.  Adult patients (phase II) initially received mithramycin at the previously detd. recommended dose of 25 μg/kg/dose.  The planned starting dose for children (phase I) was 17.5 μg/kg/dose.  Plasma samples were obtained for mithramycin PK anal.  Results: The first two adult patients experienced reversible grade 4 alanine aminotransferase (ALT)/aspartate aminotransferase (AST) elevation exceeding the MTD.  Subsequent adult patients received mithramycin at 17.5 μg/kg/dose, and children at 13 μg/kg/dose with dexamethasone pretreatment.  None of the four subsequent adult and two pediatric patients experienced cycle 1 DLT.  No clin. responses were obsd.  The av. maximal mithramycin plasma concn. in four patients was 17.8 ± 4.6 ng/mL.  This is substantially below the sustained mithramycin concns. ≥50 nmol/L required to suppress EWS-FLI1 transcriptional activity in preclin. studies.  Due to inability to safely achieve the desired mithramycin exposure, the trial was closed to enrollment.  Conclusions: Hepatotoxicity precluded the administration of a mithramycin at a dose required to inhibit EWS-FLI1.  Evaluation of mithramycin in patients selected for decreased susceptibility to elevated transaminases may allow for improved drug exposure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtCG7I9CUUwLVg90H21EOLACvtfcHk0lhVFj7Iaw4PCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Wru77F&md5=c83b447fc26734e8bb71aa61420ada13</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00280-017-3382-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-017-3382-x%26sid%3Dliteratum%253Aachs%26aulast%3DGrohar%26aufirst%3DP.%2BJ.%26aulast%3DGlod%26aufirst%3DJ.%26aulast%3DPeer%26aufirst%3DC.%2BJ.%26aulast%3DSissung%26aufirst%3DT.%2BM.%26aulast%3DArnaldez%26aufirst%3DF.%2BI.%26aulast%3DLong%26aufirst%3DL.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DWhitcomb%26aufirst%3DP.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DWidemann%26aufirst%3DB.%2BC.%26atitle%3DA%2520phase%2520I%252FII%2520trial%2520and%2520pharmacokinetic%2520study%2520of%2520mithramycin%2520in%2520children%2520and%2520adults%2520with%2520refractory%2520Ewing%2520sarcoma%2520and%2520EWS-FLI1%2520fusion%2520transcript%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2017%26volume%3D80%26spage%3D645%26epage%3D652%26doi%3D10.1007%2Fs00280-017-3382-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokushige, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwazaki, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kataoka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyose, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokumasu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamikubo, Y.</span></span> <span> </span><span class="NLM_article-title">Genetic Regulation of the RUNX Transcription Factor Family Has Antitumor Effects</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2815</span>– <span class="NLM_lpage">2828</span>, <span class="refDoi"> DOI: 10.1172/jci91788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci91788" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=2815-2828&author=K.+Moritaauthor=K.+Suzukiauthor=S.+Maedaauthor=A.+Matsuoauthor=Y.+Mitsudaauthor=C.+Tokushigeauthor=G.+Kashiwazakiauthor=J.+Taniguchiauthor=R.+Maedaauthor=M.+Nouraauthor=M.+Hirataauthor=T.+Kataokaauthor=A.+Yanoauthor=Y.+Yamadaauthor=H.+Kiyoseauthor=M.+Tokumasuauthor=H.+Matsuoauthor=S.+Tanakaauthor=Y.+Okunoauthor=M.+Mutoauthor=K.+Nakaauthor=K.+Itoauthor=T.+Kitamuraauthor=Y.+Kanedaauthor=P.+P.+Liuauthor=T.+Bandoauthor=S.+Adachiauthor=H.+Sugiyamaauthor=Y.+Kamikubo&title=Genetic+Regulation+of+the+RUNX+Transcription+Factor+Family+Has+Antitumor+Effects&doi=10.1172%2Fjci91788"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1172%2Fjci91788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci91788%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DMatsuo%26aufirst%3DA.%26aulast%3DMitsuda%26aufirst%3DY.%26aulast%3DTokushige%26aufirst%3DC.%26aulast%3DKashiwazaki%26aufirst%3DG.%26aulast%3DTaniguchi%26aufirst%3DJ.%26aulast%3DMaeda%26aufirst%3DR.%26aulast%3DNoura%26aufirst%3DM.%26aulast%3DHirata%26aufirst%3DM.%26aulast%3DKataoka%26aufirst%3DT.%26aulast%3DYano%26aufirst%3DA.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DKiyose%26aufirst%3DH.%26aulast%3DTokumasu%26aufirst%3DM.%26aulast%3DMatsuo%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DMuto%26aufirst%3DM.%26aulast%3DNaka%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DKaneda%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%2BP.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DAdachi%26aufirst%3DS.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DKamikubo%26aufirst%3DY.%26atitle%3DGenetic%2520Regulation%2520of%2520the%2520RUNX%2520Transcription%2520Factor%2520Family%2520Has%2520Antitumor%2520Effects%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D2815%26epage%3D2828%26doi%3D10.1172%2Fjci91788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antony-Debré, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leite, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvajal, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todorova, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholdy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanagari, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambesi-Impiombato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantzaris, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, G. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steidl, U.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Inhibition of the Transcription Factor PU.1 in Leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4313</span>, <span class="refDoi"> DOI: 10.1172/jci92504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci92504" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=4297-4313&author=I.+Antony-Debr%C3%A9author=A.+Paulauthor=J.+Leiteauthor=K.+Mitchellauthor=H.+M.+Kimauthor=L.+A.+Carvajalauthor=T.+I.+Todorovaauthor=K.+Huangauthor=A.+Kumarauthor=A.+A.+Farahatauthor=B.+Bartholdyauthor=S.-R.+Narayanagariauthor=J.+Chenauthor=A.+Ambesi-Impiombatoauthor=A.+A.+Ferrandoauthor=I.+Mantzarisauthor=E.+Gavathiotisauthor=A.+Vermaauthor=B.+Willauthor=D.+W.+Boykinauthor=W.+D.+Wilsonauthor=G.+M.+K.+Poonauthor=U.+Steidl&title=Pharmacological+Inhibition+of+the+Transcription+Factor+PU.1+in+Leukemia&doi=10.1172%2Fjci92504"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1172%2Fjci92504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci92504%26sid%3Dliteratum%253Aachs%26aulast%3DAntony-Debr%25C3%25A9%26aufirst%3DI.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DLeite%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%2BM.%26aulast%3DCarvajal%26aufirst%3DL.%2BA.%26aulast%3DTodorova%26aufirst%3DT.%2BI.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBartholdy%26aufirst%3DB.%26aulast%3DNarayanagari%26aufirst%3DS.-R.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DAmbesi-Impiombato%26aufirst%3DA.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DMantzaris%26aufirst%3DI.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DWill%26aufirst%3DB.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DPoon%26aufirst%3DG.%2BM.%2BK.%26aulast%3DSteidl%26aufirst%3DU.%26atitle%3DPharmacological%2520Inhibition%2520of%2520the%2520Transcription%2520Factor%2520PU.1%2520in%2520Leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D4297%26epage%3D4313%26doi%3D10.1172%2Fjci92504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span> <span> </span><span class="NLM_article-title">The Hox Genes and Their Roles in Oncogenesis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nrc2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnrc2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20357775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVGju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=361-371&author=N.+Shahauthor=S.+Sukumar&title=The+Hox+Genes+and+Their+Roles+in+Oncogenesis&doi=10.1038%2Fnrc2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Hox genes and their roles in oncogenesis</span></div><div class="casAuthors">Shah, Nilay; Sukumar, Saraswati</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-371</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Hox genes, a highly conserved subgroup of the homeobox superfamily, have crucial roles in development, regulating numerous processes including apoptosis, receptor signaling, differentiation, motility and angiogenesis.  Aberrations in Hox gene expression have been reported in abnormal development and malignancy, indicating that altered expression of Hox genes could be important for both oncogenesis and tumor suppression, depending on context.  Therefore, Hox gene expression could be important in diagnosis and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzXDSElPxjirVg90H21EOLACvtfcHk0lj-vUB40c49SA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVGju7g%253D&md5=d51cad3c6625422525b8e9e6dff4a6cf</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2826%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%26aulast%3DSukumar%26aufirst%3DS.%26atitle%3DThe%2520Hox%2520Genes%2520and%2520Their%2520Roles%2520in%2520Oncogenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D361%26epage%3D371%26doi%3D10.1038%2Fnrc2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier-Magalhães, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pojo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, B. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Profiling of HOXA9-Regulated Genes in Human Glioblastoma Cell Models</span>. <i>Genomics Data</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.gdata.2015.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.gdata.2015.05.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=54-58&author=C.+S.+Gon%C3%A7alvesauthor=A.+Xavier-Magalh%C3%A3esauthor=M.+Pojoauthor=A.+I.+Oliveiraauthor=S.+Correiaauthor=R.+M.+Reisauthor=N.+Sousaauthor=M.+Rochaauthor=B.+M.+Costa&title=Transcriptional+Profiling+of+HOXA9-Regulated+Genes+in+Human+Glioblastoma+Cell+Models&doi=10.1016%2Fj.gdata.2015.05.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.gdata.2015.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gdata.2015.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DGon%25C3%25A7alves%26aufirst%3DC.%2BS.%26aulast%3DXavier-Magalh%25C3%25A3es%26aufirst%3DA.%26aulast%3DPojo%26aufirst%3DM.%26aulast%3DOliveira%26aufirst%3DA.%2BI.%26aulast%3DCorreia%26aufirst%3DS.%26aulast%3DReis%26aufirst%3DR.%2BM.%26aulast%3DSousa%26aufirst%3DN.%26aulast%3DRocha%26aufirst%3DM.%26aulast%3DCosta%26aufirst%3DB.%2BM.%26atitle%3DTranscriptional%2520Profiling%2520of%2520HOXA9-Regulated%2520Genes%2520in%2520Human%2520Glioblastoma%2520Cell%2520Models%26jtitle%3DGenomics%2520Data%26date%3D2015%26volume%3D5%26spage%3D54%26epage%3D58%26doi%3D10.1016%2Fj.gdata.2015.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires-Luís, A.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Pinheiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerónimo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of Clear Cell Renal Cell Carcinoma and Oncocytoma Using a Three-Gene Promoter Methylation Panel</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1186/s12967-017-1248-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs12967-017-1248-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28662726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=149-157&author=A.%0AS.+Pires-Lu%C3%ADsauthor=P.+Costa-Pinheiroauthor=M.+J.+Ferreiraauthor=L.+Antunesauthor=F.+Loboauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jer%C3%B3nimo&title=Identification+of+Clear+Cell+Renal+Cell+Carcinoma+and+Oncocytoma+Using+a+Three-Gene+Promoter+Methylation+Panel&doi=10.1186%2Fs12967-017-1248-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identifcation of clear cell renal cell carcinoma and oncocytoma using a three-gene promoter methylation panel</span></div><div class="casAuthors">Pires-Luis, Ana Silvia; Costa-Pinheiro, Pedro; Ferreira, Maria Joao; Antunes, Luis; Lobo, Francisco; Oliveira, Jorge; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Translational Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149/1-149/9</span>CODEN:
                <span class="NLM_cas:coden">JTMOBV</span>;
        ISSN:<span class="NLM_cas:issn">1479-5876</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Promoter methylation has emerged as a promising class of epigenetic biomarkers for diagnosis and prognosis of renal cell tumors (RCTs).  Although differential gene promoter methylation patterns have been reported for the major subtypes (clear cell, papillary and chromophobe renal cell carcinoma, and oncocytoma), validation of diagnostic performance in independent series have been seldom performed.  Herein, we aimed at assessing the diagnostic performance of genes previously shown to be hypermethylated in RCTs in different clin. settings.  Methods: Promoter methylation levels of HOXA9 and OXR1 were assessed by quant. methylation specific PCR.  ROC curves were generated for OXR1, OXR1 combined with MST1R and HOXA9.  Sensitivity, specificity, pos. predictive value, neg. predictive value and accuracy were computed, maximizing specificity.  Methylation levels were also correlated with clin. and pathol. relevant parameters.  Results: HOXA9 and OXR1 promoter methylation was disclosed in 73 and 87% of RCTs, resp.  A two-gene methylation panel comprising OXR1 and MST1R identified malignancy with 98% sensitivity and 100% specificity, and clear cell renal cell carcinoma with 90% sensitivity and 98% specificity.  HOXA9 promoter methylation allowed for discrimination between oncocytoma and both papillary and chromophobe renal cell carcinoma but only with 77% sensitivity and 73% specificity.  Significantly higher OXR1 promoter methylation levels (p = 0.005) were assocd. with high nuclear grade in ccRCC.  Conclusions: A panel including OXR1 and MST1R promoter methylation allows specific and sensitive identification of renal cell tumors, and, esp., of clear cell renal cell carcinoma.  Moreover, higher OXR1 promoter methylation levels assoc. with clear cell renal cell carcinoma nuclear grade, a surrogate for tumor aggressiveness.  Thus, gene promoter methylation anal. might a useful ancillary tool in diagnostic management of renal masses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbEGBik2z4GbVg90H21EOLACvtfcHk0lhufLFDB6NFwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOqtrvP&md5=262ecf7824f683600340a54b614af553</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2Fs12967-017-1248-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12967-017-1248-y%26sid%3Dliteratum%253Aachs%26aulast%3DPires-Lu%25C3%25ADs%26aufirst%3DA.%2BS.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DM.%2BJ.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DLobo%26aufirst%3DF.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJer%25C3%25B3nimo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Clear%2520Cell%2520Renal%2520Cell%2520Carcinoma%2520and%2520Oncocytoma%2520Using%2520a%2520Three-Gene%2520Promoter%2520Methylation%2520Panel%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2017%26volume%3D15%26spage%3D149%26epage%3D157%26doi%3D10.1186%2Fs12967-017-1248-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajula, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nghiem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nugent, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taparra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemtiri-Chlieh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">True, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWeese, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span> <span> </span><span class="NLM_article-title">TWIST1-WDR5- Hottip Regulates Hoxa9 Chromatin to Facilitate Prostate Cancer Metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">3181</span>– <span class="NLM_lpage">3193</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-2797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1158%2F0008-5472.can-16-2797" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=3181-3193&author=R.+Malekauthor=R.+P.+Gajulaauthor=R.+D.+Williamsauthor=B.+Nghiemauthor=B.+W.+Simonsauthor=K.+Nugentauthor=H.+Wangauthor=K.+Taparraauthor=G.+Lemtiri-Chliehauthor=A.+R.+Yoonauthor=L.+Trueauthor=S.+S.+Anauthor=T.+L.+DeWeeseauthor=A.+E.+Rossauthor=E.+M.+Schaefferauthor=K.+J.+Pientaauthor=P.+J.+Hurleyauthor=C.+Morrisseyauthor=P.+T.+Tran&title=TWIST1-WDR5-+Hottip+Regulates+Hoxa9+Chromatin+to+Facilitate+Prostate+Cancer+Metastasis&doi=10.1158%2F0008-5472.can-16-2797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-16-2797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-16-2797%26sid%3Dliteratum%253Aachs%26aulast%3DMalek%26aufirst%3DR.%26aulast%3DGajula%26aufirst%3DR.%2BP.%26aulast%3DWilliams%26aufirst%3DR.%2BD.%26aulast%3DNghiem%26aufirst%3DB.%26aulast%3DSimons%26aufirst%3DB.%2BW.%26aulast%3DNugent%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTaparra%26aufirst%3DK.%26aulast%3DLemtiri-Chlieh%26aufirst%3DG.%26aulast%3DYoon%26aufirst%3DA.%2BR.%26aulast%3DTrue%26aufirst%3DL.%26aulast%3DAn%26aufirst%3DS.%2BS.%26aulast%3DDeWeese%26aufirst%3DT.%2BL.%26aulast%3DRoss%26aufirst%3DA.%2BE.%26aulast%3DSchaeffer%26aufirst%3DE.%2BM.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DHurley%26aufirst%3DP.%2BJ.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DTran%26aufirst%3DP.%2BT.%26atitle%3DTWIST1-WDR5-%2520Hottip%2520Regulates%2520Hoxa9%2520Chromatin%2520to%2520Facilitate%2520Prostate%2520Cancer%2520Metastasis%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D3181%26epage%3D3193%26doi%3D10.1158%2F0008-5472.can-16-2797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ru, G.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">High level of homeobox A9 and PBX homeobox 3 expression in gastric cancer correlates with poor prognosis</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5883</span>– <span class="NLM_lpage">5889</span>, <span class="refDoi"> DOI: 10.3892/ol.2017.6937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.3892%2Fol.2017.6937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29113222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC1M7oslOjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=5883-5889&author=Y.-Y.+Maauthor=Y.+Zhangauthor=X.-Z.+Mouauthor=Z.-C.+Liuauthor=G.-Q.+Ruauthor=E.+Li&title=High+level+of+homeobox+A9+and+PBX+homeobox+3+expression+in+gastric+cancer+correlates+with+poor+prognosis&doi=10.3892%2Fol.2017.6937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High level of homeobox A9 and PBX homeobox 3 expression in gastric cancer correlates with poor prognosis</span></div><div class="casAuthors">Ma Ying-Yu; Zhang Yuancheng; Li Erguang; Ma Ying-Yu; Mou Xiao-Zhou; Liu Zheng-Chuang; Ru Guo-Qing</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5883-5889</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">The homeobox protein homeobox (HOXA9) is a transcriptional factor that regulates patterning during embryogenesis and controls cell differentiation.  HOXA9 dysfunction has been implicated in certain cancers.  However, the role of HOXA9 in gastric cancer is poorly understood.  The present study investigated HOXA9 and its cofactor PBX homeobox 3 (PBX3) expression in patients with gastric cancer.  Paired tissue samples from 24 patients and paraffin embedded tissues of gastric cancer patients (104 males and 24 females) were included.  HOXA9 and PBX3 expression levels were determined by reverse transcription quantitative polymerase chain reaction in fresh tissues, and by immunohistochemical staining in paraffin embedded tissues.  The association between HOXA9/PBX3 expression and clinicopathological features was established.  The results demonstrated that HOXA9 and PBX3 mRNA levels were significantly upregulated (P=0.032 for HOXA9 and P=0.031 for PBX3) in gastric cancer tissue.  Immunohistochemical staining revealed that HOXA9 expression was associated with differentiation, lymph node metastasis and tumor-node-metastasis (TNM) stage, and PBX3 expression was associated with lymph node metastasis and TNM stage.  Correlation analysis revealed a high coincidental expression of HOXA9 and PBX3 levels in gastric cancer (r=0.391; P<0.001).  Survival analysis showed that high expression of HOXA9 or PBX3 was associated with poor survival of gastric cancer, and multivariate analysis using Cox's regression model showed that PBX3 expression was an independent prognostic factor in gastric cancer.  There was elevated expression of HOXA9 and PBX3 in gastric cancer patients, and high-level expression of those proteins was associated with poor prognosis of gastric cancer.  The present study underlines the significance of HOXA9/PBX3 in the development of gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSUPVh83kF2SNK0qsdY6Nc1fW6udTcc2eY75e3LhVvhaLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7oslOjtg%253D%253D&md5=668a323a7311cc1520e1906d58c8a94d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6937%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.-Y.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMou%26aufirst%3DX.-Z.%26aulast%3DLiu%26aufirst%3DZ.-C.%26aulast%3DRu%26aufirst%3DG.-Q.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DHigh%2520level%2520of%2520homeobox%2520A9%2520and%2520PBX%2520homeobox%25203%2520expression%2520in%2520gastric%2520cancer%2520correlates%2520with%2520poor%2520prognosis%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D5883%26epage%3D5889%26doi%3D10.3892%2Fol.2017.6937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijapurkar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largman, C.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C-Mediated Phosphorylation of the Leukemia-Associated HOXA9 Protein Impairs Its DNA Binding Ability and Induces Myeloid Differentiation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3837</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.9.3827-3837.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1128%2Fmcb.24.9.3827-3837.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15082777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVyrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=3827-3837&author=U.+Vijapurkarauthor=N.+Fischbachauthor=W.+Shenauthor=C.+Brandtsauthor=D.+Stokoeauthor=H.+J.+Lawrenceauthor=C.+Largman&title=Protein+Kinase+C-Mediated+Phosphorylation+of+the+Leukemia-Associated+HOXA9+Protein+Impairs+Its+DNA+Binding+Ability+and+Induces+Myeloid+Differentiation&doi=10.1128%2Fmcb.24.9.3827-3837.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C-mediated phosphorylation of the leukemia-associated HOXA9 protein impairs its DNA binding ability and induces myeloid differentiation</span></div><div class="casAuthors">Vijapurkar, Ulka; Fischbach, Neal; Shen, Weifang; Brandts, Christian; Stokoe, David; Lawrence, H. Jeffrey; Largman, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3827-3837</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HOXA9 expression is a common feature of acute myeloid leukemia, and high-level expression is correlated with poor prognosis.  Moreover, HOXA9 overexpression immortalizes murine marrow progenitors that are arrested at a promyelocytic stage of differentiation when cultured and causes leukemia in recipient mice following transplantation of HOXA9 expressing bone marrow.  The mol. mechanisms underlying the physiol. functions and transforming properties of HOXA9 are poorly understood.  This study demonstrates that HOXA9 is phosphorylated by protein kinase C (PKC) and casein kinase II and that PKC mediates phosphorylation of purified HOXA9 on S204 as well as on T205, within a highly conserved consensus sequence, in the N-terminal region of the homeodomain.  S204 in the endogenous HOXA9 protein was phosphorylated in PLB985 myeloid cells, as well as in HOXA9-immortalized murine marrow cells.  This phosphorylation was enhanced by phorbol ester, a known inducer of PKC, and was inhibited by a specific PKC inhibitor.  PKC-mediated phosphorylation of S204 decreased HOXA9 DNA binding affinity in vitro and the ability of the endogenous HOXA9 to form cooperative DNA binding complexes with PBX.  PKC inhibition significantly reduced the phorbol-ester induced differentiation of the PLB985 hematopoietic cell line as well as HOXA9-immortalized murine bone marrow cells.  These data suggest that phorbol ester-induced myeloid differentiation is in part due to PKC-mediated phosphorylation of HOXA9, which decreases the DNA binding of the homeoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJCVepTtX_hLVg90H21EOLACvtfcHk0lir-ndWHjcsQg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVyrs7k%253D&md5=93bc97c26d1c5a154965ef5acfa05fb4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2Fmcb.24.9.3827-3837.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.24.9.3827-3837.2004%26sid%3Dliteratum%253Aachs%26aulast%3DVijapurkar%26aufirst%3DU.%26aulast%3DFischbach%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DBrandts%26aufirst%3DC.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DH.%2BJ.%26aulast%3DLargman%26aufirst%3DC.%26atitle%3DProtein%2520Kinase%2520C-Mediated%2520Phosphorylation%2520of%2520the%2520Leukemia-Associated%2520HOXA9%2520Protein%2520Impairs%2520Its%2520DNA%2520Binding%2520Ability%2520and%2520Induces%2520Myeloid%2520Differentiation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D3827%26epage%3D3837%26doi%3D10.1128%2Fmcb.24.9.3827-3837.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Heuvel-Eibrink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Is Required for Survival in Human MLL-Rearranged Acute Leukemias</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2375</span>– <span class="NLM_lpage">2385</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-09-113597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2007-09-113597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=19056693" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2008&pages=2375-2385&author=J.+Faberauthor=A.+V.+Krivtsovauthor=M.+C.+Stubbsauthor=R.+Wrightauthor=T.+N.+Davisauthor=M.+van+den+Heuvel-Eibrinkauthor=C.+M.+Zwaanauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=HOXA9+Is+Required+for+Survival+in+Human+MLL-Rearranged+Acute+Leukemias&doi=10.1182%2Fblood-2007-09-113597"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-09-113597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-09-113597%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DWright%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3Dvan%2Bden%2BHeuvel-Eibrink%26aufirst%3DM.%26aulast%3DZwaan%26aufirst%3DC.%2BM.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DHOXA9%2520Is%2520Required%2520for%2520Survival%2520in%2520Human%2520MLL-Rearranged%2520Acute%2520Leukemias%26jtitle%3DBlood%26date%3D2008%26volume%3D113%26spage%3D2375%26epage%3D2385%26doi%3D10.1182%2Fblood-2007-09-113597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Role of HOXA9 in Leukemia: Dysregulation, Cofactors and Essential Targets</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1090</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fonc.2015.174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=26028034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1090-1098&author=C.+T.+Collinsauthor=J.+L.+Hess&title=Role+of+HOXA9+in+Leukemia%3A+Dysregulation%2C+Cofactors+and+Essential+Targets&doi=10.1038%2Fonc.2015.174"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.174%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DC.%2BT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DRole%2520of%2520HOXA9%2520in%2520Leukemia%253A%2520Dysregulation%252C%2520Cofactors%2520and%2520Essential%2520Targets%26jtitle%3DOncogene%26date%3D2015%26volume%3D35%26spage%3D1090%26epage%3D1098%26doi%3D10.1038%2Fonc.2015.174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Deregulation of the HOXA9/MEIS1 Axis in Acute Leukemia</span>. <i>Curr. Opin. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">354</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1097/moh.0000000000000245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1097%2Fmoh.0000000000000245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27258906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFGmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=354-361&author=C.+T.+Collinsauthor=J.+L.+Hess&title=Deregulation+of+the+HOXA9%2FMEIS1+Axis+in+Acute+Leukemia&doi=10.1097%2Fmoh.0000000000000245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of the HOXA9/MEIS1 axis in acute leukemia</span></div><div class="casAuthors">Collins, Cailin T.; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Hematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">354-361</span>CODEN:
                <span class="NLM_cas:coden">COHEF4</span>;
        ISSN:<span class="NLM_cas:issn">1065-6251</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: HOXA9 is a homeodomain transcription factor that plays an essential role in normal hematopoiesis and acute leukemia, in which its overexpression is strongly correlated with poor prognosis.  The present review highlights recent advances in the understanding of genetic alterations leading to deregulation of HOXA9 and the downstream mechanisms of HOXA9-mediated transformation.  Recent findings: A variety of genetic alterations including MLL translocations, NUP98-fusions, NPM1 mutations, CDX deregulation, and MOZ-fusions lead to high-level HOXA9 expression in acute leukemias.  The mechanisms resulting in HOXA9 overexpression are beginning to be defined and represent attractive therapeutic targets.  Small mols. targeting MLL-fusion protein complex members, such as DOT1L and menin, have shown promising results in animal models, and a DOT1L inhibitor is currently being tested in clin. trials.  Essential HOXA9 cofactors and collaborators are also being identified, including transcription factors PU.1 and C/EBPα, which are required for HOXA9-driven leukemia.  HOXA9 targets including IGF1, CDX4, INK4A/INK4B/ARF, mir-21, and mir-196b and many others provide another avenue for potential drug development.  Summary: HOXA9 deregulation underlies a large subset of aggressive acute leukemias.  Understanding the mechanisms regulating the expression and activity of HOXA9, along with its crit. downstream targets, shows promise for the development of more selective and effective leukemia therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUxf970ShOmrVg90H21EOLACvtfcHk0ljvWlwDfz-yXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFGmtLc%253D&md5=02931b1f9cdee0f39e6ad8b71d97bf54</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1097%2Fmoh.0000000000000245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fmoh.0000000000000245%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DC.%2BT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DDeregulation%2520of%2520the%2520HOXA9%252FMEIS1%2520Axis%2520in%2520Acute%2520Leukemia%26jtitle%3DCurr.%2520Opin.%2520Hematol.%26date%3D2016%26volume%3D23%26spage%3D354%26epage%3D361%26doi%3D10.1097%2Fmoh.0000000000000245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Is Critical in the Proliferation, Differentiation, and Malignancy of Leukaemia Cells Both in Vitro and in Vivo</span>. <i>Cell Biochem. Funct.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1002/cbf.3293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fcbf.3293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28961318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOnt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=433-440&author=S.+Chenauthor=J.+Yuauthor=X.+Lvauthor=L.+Zhang&title=HOXA9+Is+Critical+in+the+Proliferation%2C+Differentiation%2C+and+Malignancy+of+Leukaemia+Cells+Both+in+Vitro+and+in+Vivo&doi=10.1002%2Fcbf.3293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HOXA9 is critical in the proliferation, differentiation, and malignancy of leukaemia cells both in vitro and in vivo</span></div><div class="casAuthors">Chen, Shibing; Yu, Juan; Lv, Xin; Zhang, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Function</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">433-440</span>CODEN:
                <span class="NLM_cas:coden">CBFUDH</span>;
        ISSN:<span class="NLM_cas:issn">1099-0844</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Progress in the understanding of the mol. mechanism for acute myeloid leukemia is of great significance to generate new potential targets for treatment.  Recent studies showed that HOXA9, a homeodomain-contg. transcription factor, is commonly deregulated in acute leukemia.  In this study, we elucidated the direct correlation between HoxA9 expression and progression of leukemia using 2 different types of leukemia cells HL-60 and MOLT-3.  The HoxA9 expression level was decreased in leukemia cells with the treatment of all-trans retinoic acid or arsenic trioxide (As2O3).  Downregulation of HoxA9 could impair the proliferation and promote the leukemia cell death.  HoxA9 silencing also potentiated the differentiation of leukemia cells, and in vivo studies demonstrated that HoxA9 downregulation could interfere the tumor growth.  Interestingly, HoxA9 silencing also led to the alteration in miRNA expression, mediating the promoting effect on the leukemia cell differentiation.  Therefore, this work provided a promising and potentially efficient target to leukemia treatment, indicating that HoxA9 is likely to be an ideal candidate in the gene therapy against acute myeloid leukemia.  In this study, we elucidated the crit. role of HoxA9 in the proliferation and differentiation of leukemia cells both in vitro and in vivo.  The effect of HoxA9 modulation was correlated with the clin. effect of all-trans retinoic acid and As2O3.  Furthermore, HoxA9 also regulated the miRNA expression, controlling the leukemia cell differentiation.  Therefore, this work provided new insights into mol. mechanism underlying the leukemia treatment, potentially putting forward a brand new target to the gene therapy against leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTeka4BsTWQ7Vg90H21EOLACvtfcHk0lgE-43MchFEuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOnt7fM&md5=9bc59f50d3a5dfbe1a31c527e0ef30b5</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcbf.3293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbf.3293%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DHOXA9%2520Is%2520Critical%2520in%2520the%2520Proliferation%252C%2520Differentiation%252C%2520and%2520Malignancy%2520of%2520Leukaemia%2520Cells%2520Both%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DCell%2520Biochem.%2520Funct.%26date%3D2017%26volume%3D35%26spage%3D433%26epage%3D440%26doi%3D10.1002%2Fcbf.3293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valerio, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koche, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">NUP98 Fusion Proteins Interact with the NSL and MLL1 Complexes to Drive Leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccell.2016.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27889185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=863-878&author=H.+Xuauthor=D.+G.+Valerioauthor=M.+E.+Eisoldauthor=A.+Sinhaauthor=R.+P.+Kocheauthor=W.+Huauthor=C.-W.+Chenauthor=S.+H.+Chuauthor=G.+L.+Brienauthor=C.+Y.+Parkauthor=J.+J.+Hsiehauthor=P.+Ernstauthor=S.+A.+Armstrong&title=NUP98+Fusion+Proteins+Interact+with+the+NSL+and+MLL1+Complexes+to+Drive+Leukemogenesis&doi=10.1016%2Fj.ccell.2016.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NUP98 F1usion Proteins Interact with the NSL and MLL1 Complexes to Drive Leukemogenesis</span></div><div class="casAuthors">Xu, Haiming; Valerio, Daria G.; Eisold, Meghan E.; Sinha, Amit; Koche, Richard P.; Hu, Wenhuo; Chen, Chun-Wei; Chu, S. Haihua; Brien, Gerard L.; Park, Christopher Y.; Hsieh, James J.; Ernst, Patricia; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">863-878</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The nucleoporin 98 gene (NUP98) is fused to a variety of partner genes in multiple hematopoietic malignancies.  Here, we demonstrate that NUP98 fusion proteins, including NUP98-HOXA9 (NHA9), NUP98-HOXD13 (NHD13), NUP98-NSD1, NUP98-PHF23, and NUP98-TOP1 phys. interact with mixed lineage leukemia 1 (MLL1) and the non-specific lethal (NSL) histone-modifying complexes.  Chromatin immunopptn. sequencing illustrates that NHA9 and MLL1 co-localize on chromatin and are found assocd. with Hox gene promoter regions.  Furthermore, MLL1 is required for the proliferation of NHA9 cells in vitro and in vivo.  Inactivation of MLL1 leads to decreased expression of genes bound by NHA9 and MLL1 and reverses a gene expression signature found in NUP98-rearranged human leukemias.  Our data reveal a mol. dependency on MLL1 function in NUP98-fusion-driven leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4BUrzPjB5bVg90H21EOLACvtfcHk0lgE-43MchFEuw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgtL%252FO&md5=c1baef1125dc36f2d288de70d83640f9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DValerio%26aufirst%3DD.%2BG.%26aulast%3DEisold%26aufirst%3DM.%2BE.%26aulast%3DSinha%26aufirst%3DA.%26aulast%3DKoche%26aufirst%3DR.%2BP.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.-W.%26aulast%3DChu%26aufirst%3DS.%2BH.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BY.%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DNUP98%2520Fusion%2520Proteins%2520Interact%2520with%2520the%2520NSL%2520and%2520MLL1%2520Complexes%2520to%2520Drive%2520Leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D863%26epage%3D878%26doi%3D10.1016%2Fj.ccell.2016.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rio-Machin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-López, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Martinez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgado, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrestha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres-Ruiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larráyoz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calasanz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgibbon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cigudosa, J. C.</span></span> <span> </span><span class="NLM_article-title">The Molecular Pathogenesis of the NUP98-HOXA9 Fusion Protein in Acute Myeloid Leukemia</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2005</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fleu.2017.194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28630438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=2000-2005&author=A.+Rio-Machinauthor=G.+G%C3%B3mez-L%C3%B3pezauthor=J.+Mu%C3%B1ozauthor=F.+Garcia-Martinezauthor=A.+Maiques-Diazauthor=S.+Alvarezauthor=R.+N.+Salgadoauthor=M.+Shresthaauthor=R.+Torres-Ruizauthor=C.+Haferlachauthor=M.+J.+Larr%C3%A1yozauthor=M.+J.+Calasanzauthor=J.+Fitzgibbonauthor=J.+C.+Cigudosa&title=The+Molecular+Pathogenesis+of+the+NUP98-HOXA9+Fusion+Protein+in+Acute+Myeloid+Leukemia&doi=10.1038%2Fleu.2017.194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular pathogenesis of the NUP98-HOXA9 fusion protein in acute myeloid leukemia</span></div><div class="casAuthors">Rio-Machin, A.; Gomez-Lopez, G.; Munoz, J.; Garcia-Martinez, F.; Maiques-Diaz, A.; Alvarez, S.; Salgado, R. N.; Shrestha, M.; Torres-Ruiz, R.; Haferlach, C.; Larrayoz, M. J.; Calasanz, M. J.; Fitzgibbon, J.; Cigudosa, J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000-2005</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations are common initiation events and have provided important insights into the pathogenesis of AML, paving the way for the introduction of novel targeted therapies.  However, clin. outcomes, in particular for patients with adverse cytogenetic features remain suboptimal.  The chromosomal translocation t(7;11)(p15, p15), encoding the fusion protein NUP98-HOXA9 (NHA9), is a rare poor risk cytogenetic event in AML assocd. with a particularly poor prognosis.  NHA9 brings the FG repeat-rich portion of the nucleoporin NUP98 upstream of the homeodomain and PBX heterodimerization domains of HOXA9, and acts as oncogenic transcription factor.1.  The pathogenic events underlying NHA9 remain poorly understood and herein, we aim to characterize the downstream mediators of this oncoprotein by detg. the effects of the fusion using human cellular models.  We set out initially to compare the DNA binding sites of NHA9, HOXA9 and NUP98, by forced expression of these genes alone or the corresponding fusion gene by retroviral transduction of HEK93FT cell line and cord blood-isolated human hematopoietic progenitors (hHP).  ChIP-seq anal. in the HEK293FT cellular model identified 4471 significant genomic regions (false discovery rate (FDR) <0.05) as target sites of the fusion protein, all located within -5/+ kb from the annotated transcription start site (TSS) (Supplementary Figure S1A).  They correspond to 1368 genes and 17 miRNAs (Supplementary Table S1) of which 399 genes were also shown to be common targets of HOXA9 and 4 of NUP98 (Figure 1a, Supplementary Table S2, Supplementary Figures S1C-D) (Supplementary methods) (Data deposited in GEO http://www.ncbi.nlm.nih.gov/geo/, accession no.: GSE62587).  Ingenuity pathway anal. of the NHA9 target series demonstrated a significant enrichment of pathways assocd. with tumorigenesis and leukemic differentiation (Supplementary Figure S1B).  We next performed a detailed sequence anal. of the NHA9 binding sites using the MEME-ChIP algorithm and detected a significant overlap with binding of several HOX genes, including HOXA9, supporting a role for this homeodomain in the DNA binding of NHA9.  Strikingly, NHA9 sites were enriched for a novel binding motif, CA/gTTT, that was present in one-third (n=1421) of all NHA9 ChIP-seq regions (Supplementary Table S3).  This motif had not been previously assocd. with any known transcription factor and was not obsd. in wild type HOXA9 or NUP98 binding site expts., suggesting that it is specific to NHA9 DNA binding.  MEME-ChIP (SpaMO) was used to identify significant co-occurrences of other known DNA binding motifs with this novel NHA9 DNA binding motif.  Binding motifs corresponding to 12 transcription factors, including other HOX family proteins such as HOXB7 or HOXD11, were found to be overrepresented within the region adjacent to CA/gTTT (Supplementary Table S4), suggesting a possible functional cooperation with the fusion oncoprotein.  As the NHA9 target motifs are preferentially located more than 1 kb upstream/downstream of the TSS (Supplementary Figure S1A), we reasoned that NHA9 binding may coincide with particular enhancer elements.  A similar distribution was also found for the identified HOXA9 target regions whereas NUP98 binding sites were mostly located within promoters, both in agreement with previous studies.2, 3 We selected eight leukemia-related genes (MEIS1, HOXA9, PBX3, MET, BRAF, AF9, PTEN and NF1) identified as part of our NHA9 ChIP-seq expts., for locus specific qChIP studies.  A significant enrichment of H3K4me1, a chromatin mark that predicts poised and active enhancers, and RNA Polymerase II (PolII), which is consistent with the presence of the active form of the enhancers,4, 5 was shown within the NHA9 binding sites upstream of the eight genes (Figure 1b and Supplementary Figure S1E).  NHA9 expression levels were demonstrated to be comparable in our two cellular models (HEK293FT and hHP) (Supplementary Figure S1G).  Accordingly, we validated the ChIP-seq results in the HEK293FT model (Supplementary Figure S1F) using the same set of eight NHA9 target genes and also demonstrated binding of NHA9 to the eight enhancers in our second model system of NHA9-expressing hHP cells (Figure 1c), allowing us to confirm these findings in primary human hematopoiesis.  We next focused attention on the transcription factors MEIS1, HOXA9 and PBX3, as their overexpression is significantly related to adverse prognosis in AML (The Cancer Genome Atlas;6Supplementary Figure S1H) and were previously reported to drive leukemogenesis through the formation of a transcriptional activator complex.7.  To test the importance of these three transcription factors in NHA9 pathogenesis, we completed reporter assays in HEK293FT cells by cloning the identified enhancers of MEIS1, HOXA9 or PBX3 into a luciferase reporter vector.  A significant 1.6-2.8 fold induction in luciferase activity was obsd. when NHA9 was co-expressed for all three enhancers, indicating a direct induction of MEIS1, HOXA9 and PBX3 expression through the NHA9 interaction with their corresponding regulatory regions (Figure 1d) (Supplementary Methods).  This observation was accompanied by upregulation of all three transcription factors and of three of their known target genes (MYB, MEF2C and FLT3)7 in NHA9-expressing hHP cells (Figure 1e and Supplementary Figure S1I).  Gene Expression Profiling performed in three independent NHA9-expressing hHP clones and AMLs from five patients with t(7;11)(p15,p15), confirmed MEIS1-HOXA9-PBX3 overexpression and it was further validated by RT-qPCR anal. in three addnl. NHA9 primary samples (Supplementary Figure S2A).  These observations suggested that the NHA9-expressing hHP cells can be sensitive to HXR9, a specific peptide inhibitor of HOXA9 and PBX3 interaction that leads to disruption of the MEIS1-HOXA9-PBX3 complex.8.  We tested this hypothesis by treating these cells with HXR9 that resulted in a selective decrease in their viability (Figure 1f and Supplementary Figure S2B-D) (Supplementary Methods) without affecting cell differentiation (data not shown), therefore confirming the relevance of these downstream mediators in driving the oncogenic activity of NHA9.  In order to explore other mechanisms driving NHA9 pathogenesis and to better understand its role in transcriptional regulation, we interrogated our ChIP-seq and gene expression profiling data, which revealed both activation and repression of gene expression induced by this fusion oncoprotein (Figure 2a).  The cooperation of MLL1 and CRM1 with NHA9 in the upregulation of some target genes has been shown recently by Xu et al.,9, 10 which was also supported by comparing NHA9 target genes identified in our ChIP-seq expts. with MLL1 and CRM1 targets.  We found that 25% and 35% of NHA9 target genes were also in common with MLL1 and CRM1 target genes, resp. (Supplementary Figure S2E).  Notably, 151 target genes, including MEIS1 and HOXA9, were shared by all three proteins (NHA9, MLL1 and CRM1), suggesting a possible cooperation among these transcription factors in NHA9-driven leukemias.  It has also been reported that NUP98, through its FG repeat domain, may interact with transcriptional activator p300 and repressor HDACs,11 allowing us to postulate that transcriptional effects of NHA9 in enhancers could be mediated by these regulators.  We first demonstrated NHA9 binding to both p300 and HDAC1 by co-immunopptn. expts. (Figure 2b) (Supplementary Methods) and went on to examine their binding potential in a panel of eight regulatory regions of NHA9 target genes (four upregulated and four downregulated target genes) in the presence of the fusion protein by qChIP.  These expts. demonstrated selective binding of p300 to the regulatory regions of the upregulated genes MEIS1, HOXA9, PBX3 and AFF3 (Figure 2c), and of HDAC1 to the downregulated genes BIRC3, SMAD1, FILIP1L and PTEN (Figure 2d).  Altogether this data suggests that p300 and HDAC1 are selectively recruited by NHA9 at enhancer regions to modulate the expression of genes involved in leukemogenesis.  As the interaction of NHA9 with HDAC1/2 was validated by mass spectrometry anal. using the NHA9-expressing HEK293FT model (Proteomics data have been deposited on the ProteomeXchange Consortium via the PRIDE partner repository, data set identifier PXD001828) (Supplementary Methods), we had a mol. rationale for testing HDAC inhibitors (HDACi) in NHA9 AML.  It has to be noted that LBH589 did not induce differentiation in NHA9-expressing cells as no significant changes in the no. of CD11b pos. cells were obsd. by flow cytometry anal. post treatment (data not shown).  These observations are in accordance with a recent report suggesting the combination of COX or DNMT inhibitors with HDACi for treatment of NHA9 AML patients,16 however in this study we identified the mol. rationale for HDACi therapy as well as a panel of target genes downstream of NHA9 that can be used as biomarkers for response to this treatment.  However, the biol. consequences of this therapy, as well as the best dosage-time relation for the translation into clinics need to be further investigated.  In summary, NHA9 deregulates the expression of key leukemic genes, including MEIS1-HOXA9-PBX3 complex, through the enhancer binding and the direct interaction of the fusion protein with HDAC and p300 transcriptional regulators.  The oncogenic effects of NHA9 can be overcome by HDACi treatment, demonstrating a significant inhibitory effects against NHA9-driven leukemic cells and suggesting a novel approach to treatment of this high-risk group of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjzGKKJ18_YLVg90H21EOLACvtfcHk0ligEHQzXuPZhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrs73F&md5=57d51492b5bbe6d50c47f97825835030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.194%26sid%3Dliteratum%253Aachs%26aulast%3DRio-Machin%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-L%25C3%25B3pez%26aufirst%3DG.%26aulast%3DMu%25C3%25B1oz%26aufirst%3DJ.%26aulast%3DGarcia-Martinez%26aufirst%3DF.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DS.%26aulast%3DSalgado%26aufirst%3DR.%2BN.%26aulast%3DShrestha%26aufirst%3DM.%26aulast%3DTorres-Ruiz%26aufirst%3DR.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DLarr%25C3%25A1yoz%26aufirst%3DM.%2BJ.%26aulast%3DCalasanz%26aufirst%3DM.%2BJ.%26aulast%3DFitzgibbon%26aufirst%3DJ.%26aulast%3DCigudosa%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Molecular%2520Pathogenesis%2520of%2520the%2520NUP98-HOXA9%2520Fusion%2520Protein%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D2000%26epage%3D2005%26doi%3D10.1038%2Fleu.2017.194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubé, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohyashiki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D. E.</span></span> <span> </span><span class="NLM_article-title">The t(7;11)(p15;p15) translocation in acute myeloid leukaemia fuses the genes for nucleoporin NUP96 and class I homeoprotein HOXA9</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1038/ng0296-159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fng0296-159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=8563754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK28XosFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1996&pages=159-167&author=J.+Borrowauthor=A.+M.+Shearmanauthor=V.+P.+Stantonauthor=R.+Becherauthor=T.+Collinsauthor=A.+J.+Williamsauthor=I.+Dub%C3%A9author=F.+Katzauthor=Y.+L.+Kwongauthor=C.+Morrisauthor=K.+Ohyashikiauthor=K.+Toyamaauthor=J.+Rowleyauthor=D.+E.+Housman&title=The+t%287%3B11%29%28p15%3Bp15%29+translocation+in+acute+myeloid+leukaemia+fuses+the+genes+for+nucleoporin+NUP96+and+class+I+homeoprotein+HOXA9&doi=10.1038%2Fng0296-159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The t(7;11)(p15;p15) translocation in acute myeloid leukemia fuses the genes for nucleoporin NUP98 and class I homeoprotein HOXA9</span></div><div class="casAuthors">Borrow, Julian; Shearman, Amanda M.; Stanton, Vincent P., Jr.; Becher, Reinhard; Collins, Tucker; Williams, Amy J.; Dube, Ian; Katz, Fay; Kwong, Yok L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-67</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">The t(7,11)(p15;p15) translocation is a recurrent chromosomal abnormality assocd. primarily with acute myeloid leukemia (FAB M2 and M4).  The authors present here the mol. definition of this translocation.  On chromosome 7 positional cloning revealed the consistent rearrangement of the HOXA9 gene, which encodes a class I homeodomain protein potentially involved in myeloid differentiation.  On chromosome 11 the translocation targets the human homolog of NUP98, a member of the GLFG nucleoporin family.  Chimeric messages spliced over the breakpoint fuse the GLFG repeat domains of NUP98 in-frame to the HOXA9 homeobox.  The predicted NUP98-HOXA9 fusion protein may promote leukemogenesis through inhibition of HOXA9-mediated terminal differentiation and/or aberrant nucleocytoplasmic transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7t1VxF9trU7Vg90H21EOLACvtfcHk0ligEHQzXuPZhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XosFCmtg%253D%253D&md5=15ca1aeb405fc5cc0ad416227a68d826</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fng0296-159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0296-159%26sid%3Dliteratum%253Aachs%26aulast%3DBorrow%26aufirst%3DJ.%26aulast%3DShearman%26aufirst%3DA.%2BM.%26aulast%3DStanton%26aufirst%3DV.%2BP.%26aulast%3DBecher%26aufirst%3DR.%26aulast%3DCollins%26aufirst%3DT.%26aulast%3DWilliams%26aufirst%3DA.%2BJ.%26aulast%3DDub%25C3%25A9%26aufirst%3DI.%26aulast%3DKatz%26aufirst%3DF.%26aulast%3DKwong%26aufirst%3DY.%2BL.%26aulast%3DMorris%26aufirst%3DC.%26aulast%3DOhyashiki%26aufirst%3DK.%26aulast%3DToyama%26aufirst%3DK.%26aulast%3DRowley%26aufirst%3DJ.%26aulast%3DHousman%26aufirst%3DD.%2BE.%26atitle%3DThe%2520t%25287%253B11%2529%2528p15%253Bp15%2529%2520translocation%2520in%2520acute%2520myeloid%2520leukaemia%2520fuses%2520the%2520genes%2520for%2520nucleoporin%2520NUP96%2520and%2520class%2520I%2520homeoprotein%2520HOXA9%26jtitle%3DNat.%2520Genet.%26date%3D1996%26volume%3D12%26spage%3D159%26epage%3D167%26doi%3D10.1038%2Fng0296-159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span> <span> </span><span class="NLM_article-title">Silencing of HMGA2 Reverses Retardance of Cell Differentiation in Human Myeloid Leukaemia</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1038/bjc.2017.403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fbjc.2017.403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29384529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=405-415&author=L.+Tanauthor=H.+Xuauthor=G.+Chenauthor=X.+Weiauthor=B.+Yuauthor=J.+Yeauthor=L.+Xuauthor=H.+Tan&title=Silencing+of+HMGA2+Reverses+Retardance+of+Cell+Differentiation+in+Human+Myeloid+Leukaemia&doi=10.1038%2Fbjc.2017.403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing of HMGA2 reverses retardance of cell differentiation in human myeloid leukaemia</span></div><div class="casAuthors">Tan, Li; Xu, Hongfa; Chen, Guoshu; Wei, Xiaoping; Yu, Baodan; Ye, Jingmei; Xu, Lihua; Tan, Huo</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-415</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Background: High-mobility group AT-hook 2 (HMGA2) may serve as an architectural transcription factor, and it can regulate a range of normal biol. processes including proliferation and differentiation.  Upregulation of HMGA2 expression is correlated to the undifferentiated phenotype of immature leukemic cells.  However, the underlying mechanism of HMGA2-dependent myeloid differentiation blockage in leukemia is unknown.  Methods: To reveal the role and mechanism of HMGA2 in differentiation arrest of myeloid leukemia cells, the quant. expression of HMGA2 and homeobox A9 (HOXA9) was analyzed by real-time PCR (qRT-PCR).  The regulatory function of HMGA2 in blockage of differentiation in human myeloid leukemia was investigated through in vitro assays (XTT assay, May-Grunwald-Giemsa, flow cytometry anal. and western blot).  Results: We found that the expression of HMGA2 and HOXA9 was reduced during the process of granulo-monocytic maturation of acute myeloid leukemia (AML) cells, knockdown of HMGA2 promotes terminal (granulocytic and monocytic) differentiation of myeloid leukemia primary blasts and cell lines, and HOXA9 was significantly downregulated in leukemic cells with knockdown of HMGA2.  Downregulation of HOXA9 in myeloid leukemia cells led to increased differentiation capacity in vitro.  Conclusions: Our data suggest that increased expression of HMGA2 represents a possible new mechanism of myeloid differentiation blockage of leukemia.  Aberrant expression of HMGA2 may enhance HOXA9-dependent leukaemogenesis and myeloid leukemia phenotype.  Disturbance of the HMGA2-HOXA9 pathway is probably a therapeutic strategy in myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzgIDafLy8n7Vg90H21EOLACvtfcHk0liOzv13_OFRig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGmsA%253D%253D&md5=87c0d65a8a8f1e9cc3228f8657a95628</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2017.403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2017.403%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DH.%26atitle%3DSilencing%2520of%2520HMGA2%2520Reverses%2520Retardance%2520of%2520Cell%2520Differentiation%2520in%2520Human%2520Myeloid%2520Leukaemia%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2018%26volume%3D118%26spage%3D405%26epage%3D415%26doi%3D10.1038%2Fbjc.2017.403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasillas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamps, M. P.</span></span> <span> </span><span class="NLM_article-title">Hoxa9 Immortalizes a Granulocyte-Macrophage Colony-Stimulating Factor-Dependent Promyelocyte Capable of Biphenotypic Differentiation to Neutrophils or Macrophages, Independent of Enforced Meis Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3274</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1128/mcb.20.9.3274-3285.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1128%2Fmcb.20.9.3274-3285.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10757811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivFSrsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=3274-3285&author=K.+R.+Calvoauthor=D.+B.+Sykesauthor=M.+Pasillasauthor=M.+P.+Kamps&title=Hoxa9+Immortalizes+a+Granulocyte-Macrophage+Colony-Stimulating+Factor-Dependent+Promyelocyte+Capable+of+Biphenotypic+Differentiation+to+Neutrophils+or+Macrophages%2C+Independent+of+Enforced+Meis+Expression&doi=10.1128%2Fmcb.20.9.3274-3285.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Hoxa9 immortalizes a granulocyte-macrophage colony-stimulating factor-dependent promyelocyte capable of biphenotypic differentiation to neutrophils or macrophages, independent of enforced Meis expression</span></div><div class="casAuthors">Calvo, Katherine R.; Sykes, David B.; Pasillas, Martina; Kamps, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3274-3285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The genes encoding Hoxa9 and Meis1 are transcriptionally coactivated in a subset of acute myeloid leukemia (AML) in mice.  In marrow reconstitution expts., coexpression of both genes produces rapid AML, while neither gene alone generates overt leukemia.  Although Hoxa9 and Meis1 can bind DNA as heterodimers, both can also heterodimerize with Pbx proteins.  Thus, while their coactivation may result from the necessity to bind promoters as heterodimers, it may also result from the necessity of altering independent biochem. pathways that cooperate to generate AML, either as monomers or as heterodimers with Pbx proteins.  Here we demonstrate that constitutive expression of Hoxa9 in primary murine marrow immortalizes a late myelomonocytic progenitor, preventing it from executing terminal differentiation to granulocytes or monocytes in the presence of granulocyte-macrophage colony-stimulating factor (GM-CSF) or interleukin-3.  This immortalized pheno-type is achieved in the absence of endogenous or exogenous Meis gene expression.  The Hoxa9-immortalized progenitor exhibited a promyelocytic transcriptional profile, expressing PU.1, AML1, c-Myb, C/EBP alpha, and C/EBP epsilon as well as their target genes, the receptors for GM-CSF, G-CSF, and M-CSF and the primary granule proteins myeloperoxidase and neutrophil elastase.  G-CSF obviated the differentiation block of Hoxa9, inducing neutrophilic differentiation with accompanying expression of neutrophil gelatinase B and upregulation of gp91phox.  M-CSF also obviated the differentiation block, inducing monocytic differentiation with accompanying expression of the macrophage acetyl-low-d. lipoprotein scavenger receptor and F4/80 antigen.  Versions of Hoxa9 lacking the ANWL Pbx interaction motif (PIM) also immortalized a promyelocytic progenitor with intrinsic biphenotypic differentiation potential.  Therefore, Hoxa9 evokes a cytokine-selective block in differentiation by a mechanism that does not require Meis gene expression or interaction with Pbx through the PIM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXqXOPNl4HbLVg90H21EOLACvtfcHk0liOzv13_OFRig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivFSrsLs%253D&md5=4d2cbadfcb20141851ac4ab45bdd7eef</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2Fmcb.20.9.3274-3285.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.20.9.3274-3285.2000%26sid%3Dliteratum%253Aachs%26aulast%3DCalvo%26aufirst%3DK.%2BR.%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DPasillas%26aufirst%3DM.%26aulast%3DKamps%26aufirst%3DM.%2BP.%26atitle%3DHoxa9%2520Immortalizes%2520a%2520Granulocyte-Macrophage%2520Colony-Stimulating%2520Factor-Dependent%2520Promyelocyte%2520Capable%2520of%2520Biphenotypic%2520Differentiation%2520to%2520Neutrophils%2520or%2520Macrophages%252C%2520Independent%2520of%2520Enforced%2520Meis%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D3274%26epage%3D3285%26doi%3D10.1128%2Fmcb.20.9.3274-3285.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passegue, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. J.</span></span> <span> </span><span class="NLM_article-title">Evidence That the Pim1 Kinase Gene Is a Direct Target of HOXA9</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">4732</span>– <span class="NLM_lpage">4738</span>, <span class="refDoi"> DOI: 10.1182/blood-2006-08-043356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2006-08-043356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=17327400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Ogtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=4732-4738&author=Y.-L.+Huauthor=E.+Passegueauthor=S.+Fongauthor=C.+Largmanauthor=H.+J.+Lawrence&title=Evidence+That+the+Pim1+Kinase+Gene+Is+a+Direct+Target+of+HOXA9&doi=10.1182%2Fblood-2006-08-043356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence that the Pim1 kinase gene is a direct target of HOXA9</span></div><div class="casAuthors">Hu, Yu-Long; Passegue, Emmanuelle; Fong, Stephen; Largman, Corey; Lawrence, Hugh Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-4738</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The HOXA9 homeoprotein exerts dramatic effects in hematopoiesis.  Enforced expression of HOXA9 enhances proliferation of primitive blood cells, expands hematopoietic stem cells (HSCs), and leads to myeloid leukemia.  Conversely, loss of HOXA9 inhibits proliferation and impairs HSC function.  The pathways by which HOXA9 acts are largely unknown, and although HOXA9 is a transcription factor, few direct target genes have been identified.  Our previous study suggested that HOXA9 pos. regulates Pim1, an oncogenic kinase.  The hematol. phenotypes of Hoxa9- and Pim1-deficient animals are strikingly similar.  Here we show that HOXA9 protein binds to the Pim1 promoter and induces Pim1 mRNA and protein in hematopoietic cells.  Pim1 protein is diminished in Hoxa9-/- cells, and Hoxa9 and Pim1 mRNA levels track together in early hematopoietic compartments.  Induction of Pim1 protein by HOXA9 increases the phosphorylation and inactivation of the proapoptotic BAD protein, a target of Pim1.  Hoxa9-/- cells show increased apoptosis and decreased proliferation, defects that are ameliorated by reintroduction of Pim1.  Thus Pim1 appears to be a direct transcriptional target of HOXA9 and a mediator of its antiapoptotic and proproliferative effects in early cells.  Since HOXA9 is frequently up-regulated in acute myeloid leukemia, Pim1 may be a therapeutic target in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo1_SUxER0HLVg90H21EOLACvtfcHk0liWJGMikDnNPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Ogtbw%253D&md5=c2d36102fe3085e133c10dabf206bc50</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1182%2Fblood-2006-08-043356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2006-08-043356%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.-L.%26aulast%3DPassegue%26aufirst%3DE.%26aulast%3DFong%26aufirst%3DS.%26aulast%3DLargman%26aufirst%3DC.%26aulast%3DLawrence%26aufirst%3DH.%2BJ.%26atitle%3DEvidence%2520That%2520the%2520Pim1%2520Kinase%2520Gene%2520Is%2520a%2520Direct%2520Target%2520of%2520HOXA9%26jtitle%3DBlood%26date%3D2007%26volume%3D109%26spage%3D4732%26epage%3D4738%26doi%3D10.1182%2Fblood-2006-08-043356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breitinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maethner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Cuellar, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">The Homeodomain Region Controls the Phenotype of HOX-Induced Murine Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4018</span>– <span class="NLM_lpage">4027</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-10-384685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2011-10-384685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22990017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsleqsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4018-4027&author=C.+Breitingerauthor=E.+Maethnerauthor=M.-P.+Garcia-Cuellarauthor=R.+K.+Slany&title=The+Homeodomain+Region+Controls+the+Phenotype+of+HOX-Induced+Murine+Leukemia&doi=10.1182%2Fblood-2011-10-384685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The homeodomain region controls the phenotype of HOX-induced murine leukemia</span></div><div class="casAuthors">Breitinger, Constanze; Maethner, Emanuel; Garcia-Cuellar, Maria-Paz; Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4018-4027</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">HOX proteins are widely involved in hematopoietic development.  These transcription factors combine a conserved DNA-binding homeobox with a divergent N-terminus that mediates interaction with variable cofactors.  The resulting combinatorial diversity is thought to be responsible for mammalian HOX specificity.  Contrasting this proposed mechanism for normal HOX function, here we demonstrate that, in the context of hematopoietic immortalization and leukemogenesis, individual HOX properties are governed almost exclusively by the homeodomain.  Swap expts. between HOXA1 and HOXA9, 2 members of nonrelated paralog groups, revealed that gene expression patterns of HOX transformed cells in vitro are detd. by the nature of the homeodomain.  Similar results were seen in vivo during HOX-mediated leukemogenesis.  An exchange of the homeodomains was sufficient to convert the slow, low-penetrance phenotype of HOXA1-induced leukemia to the aggressive fast-acting disease elicited by HOXA9 and vice versa.  Mutation and deletion studies identified several subregions within the DNA binding domain responsible for paralog specificity.  Previously defined binding sites for PBX cofactors within the exchangeable, nonhomeobox segment were dispensable for in vitro oncogenic HOX activity but affected in vivo disease development.  The transcriptional activator domain shared by HOXA1 and HOXA9 at the very N-terminus proved essential for all transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv6PpgbfWHhLVg90H21EOLACvtfcHk0liWJGMikDnNPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsleqsL%252FE&md5=8fb1ed7283c0082113b5627536deb5d7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-10-384685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-10-384685%26sid%3Dliteratum%253Aachs%26aulast%3DBreitinger%26aufirst%3DC.%26aulast%3DMaethner%26aufirst%3DE.%26aulast%3DGarcia-Cuellar%26aufirst%3DM.-P.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520Homeodomain%2520Region%2520Controls%2520the%2520Phenotype%2520of%2520HOX-Induced%2520Murine%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4018%26epage%3D4027%26doi%3D10.1182%2Fblood-2011-10-384685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahimova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arruda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popescu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuh, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Görlich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büchner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiddemann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wörmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheok, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preudhomme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzeler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buske, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valk, P. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandstra, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. C. Y.</span></span> <span> </span><span class="NLM_article-title">A 17-Gene Stemness Score for Rapid Determination of Risk in Acute Leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nature20598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnature20598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27926740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=433-437&author=S.+W.+K.+Ngauthor=A.+Mitchellauthor=J.+A.+Kennedyauthor=W.+C.+Chenauthor=J.+McLeodauthor=N.+Ibrahimovaauthor=A.+Arrudaauthor=A.+Popescuauthor=V.+Guptaauthor=A.+D.+Schimmerauthor=A.+C.+Schuhauthor=K.+W.+Yeeauthor=L.+Bullingerauthor=T.+Heroldauthor=D.+G%C3%B6rlichauthor=T.+B%C3%BCchnerauthor=W.+Hiddemannauthor=W.+E.+Berdelauthor=B.+W%C3%B6rmannauthor=M.+Cheokauthor=C.+Preudhommeauthor=H.+Dombretauthor=K.+Metzelerauthor=C.+Buskeauthor=B.+L%C3%B6wenbergauthor=P.+J.+M.+Valkauthor=P.+W.+Zandstraauthor=M.+D.+Mindenauthor=J.+E.+Dickauthor=J.+C.+Y.+Wang&title=A+17-Gene+Stemness+Score+for+Rapid+Determination+of+Risk+in+Acute+Leukaemia&doi=10.1038%2Fnature20598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A 17-gene stemness score for rapid determination of risk in acute leukaemia</span></div><div class="casAuthors">Ng, Stanley W. K.; Mitchell, Amanda; Kennedy, James A.; Chen, Weihsu C.; McLeod, Jessica; Ibrahimova, Narmin; Arruda, Andrea; Popescu, Andreea; Gupta, Vikas; Schimmer, Aaron D.; Schuh, Andre C.; Yee, Karen W.; Bullinger, Lars; Herold, Tobias; Gorlich, Dennis; Buchner, Thomas; Hiddemann, Wolfgang; Berdel, Wolfgang E.; Wormann, Bernhard; Cheok, Meyling; Preudhomme, Claude; Dombret, Herve; Metzeler, Klaus; Buske, Christian; Lowenberg, Bob; Valk, Peter J. M.; Zandstra, Peter W.; Minden, Mark D.; Dick, John E.; Wang, Jean C. Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">433-437</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Refractoriness to induction chemotherapy and relapse after achievement of remission are the main obstacles to cure in acute myeloid leukemia (AML).  After std. induction chemotherapy, patients are assigned to different post-remission strategies on the basis of cytogenetic and mol. abnormalities that broadly define adverse, intermediate and favorable risk categories.  However, some patients do not respond to induction therapy and another subset will eventually relapse despite the lack of adverse risk factors.  There is an urgent need for better biomarkers to identify these high-risk patients before starting induction chemotherapy, to enable testing of alternative induction strategies in clin. trials.  The high rate of relapse in AML has been attributed to the persistence of leukemia stem cells (LSCs), which possess a no. of stem cell properties, including quiescence, that are linked to therapy resistance.  Here, to develop predictive and/or prognostic biomarkers related to stemness, we generated a list of genes that are differentially expressed between 138 LSC+ and 89 LSC- cell fractions from 78 AML patients validated by xenotransplantation.  To ext. the core transcriptional components of stemness relevant to clin. outcomes, we performed sparse regression anal. of LSC gene expression against survival in a large training cohort, generating a 17-gene LSC score (LSC17).  The LSC17 score was highly prognostic in five independent cohorts comprising patients of diverse AML subtypes (n = 908) and contributed greatly to accurate prediction of initial therapy resistance.  Patients with high LSC17 scores had poor outcomes with current treatments including allogeneic stem cell transplantation.  The LSC17 score provides clinicians with a rapid and powerful tool to identify AML patients who do not benefit from std. therapy and who should be enrolled in trials evaluating novel upfront or post-remission strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_GpeG8ZvuZ7Vg90H21EOLACvtfcHk0liWJGMikDnNPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurfO&md5=71fee194f9c91f9f24b84cfeefd0b601</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature20598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20598%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BW.%2BK.%26aulast%3DMitchell%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DMcLeod%26aufirst%3DJ.%26aulast%3DIbrahimova%26aufirst%3DN.%26aulast%3DArruda%26aufirst%3DA.%26aulast%3DPopescu%26aufirst%3DA.%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26aulast%3DSchuh%26aufirst%3DA.%2BC.%26aulast%3DYee%26aufirst%3DK.%2BW.%26aulast%3DBullinger%26aufirst%3DL.%26aulast%3DHerold%26aufirst%3DT.%26aulast%3DG%25C3%25B6rlich%26aufirst%3DD.%26aulast%3DB%25C3%25BCchner%26aufirst%3DT.%26aulast%3DHiddemann%26aufirst%3DW.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DW%25C3%25B6rmann%26aufirst%3DB.%26aulast%3DCheok%26aufirst%3DM.%26aulast%3DPreudhomme%26aufirst%3DC.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DMetzeler%26aufirst%3DK.%26aulast%3DBuske%26aufirst%3DC.%26aulast%3DL%25C3%25B6wenberg%26aufirst%3DB.%26aulast%3DValk%26aufirst%3DP.%2BJ.%2BM.%26aulast%3DZandstra%26aufirst%3DP.%2BW.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DWang%26aufirst%3DJ.%2BC.%2BY.%26atitle%3DA%252017-Gene%2520Stemness%2520Score%2520for%2520Rapid%2520Determination%2520of%2520Risk%2520in%2520Acute%2520Leukaemia%26jtitle%3DNature%26date%3D2016%26volume%3D540%26spage%3D433%26epage%3D437%26doi%3D10.1038%2Fnature20598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitwala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandekar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cezard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilenky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Hoxa9 Binding Sites in Hematopoietic Cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-03-341081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2011-03-341081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22072553" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2011&pages=388-398&author=Y.+Huangauthor=K.+Sitwalaauthor=J.+Bronsteinauthor=D.+Sandersauthor=M.+Dandekarauthor=C.+Collinsauthor=G.+Robertsonauthor=J.+MacDonaldauthor=T.+Cezardauthor=M.+Bilenkyauthor=N.+Thiessenauthor=Y.+Zhaoauthor=T.+Zengauthor=M.+Hirstauthor=A.+Heroauthor=S.+Jonesauthor=J.+L.+Hess&title=Identification+and+Characterization+of+Hoxa9+Binding+Sites+in+Hematopoietic+Cells&doi=10.1182%2Fblood-2011-03-341081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-03-341081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-03-341081%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DSitwala%26aufirst%3DK.%26aulast%3DBronstein%26aufirst%3DJ.%26aulast%3DSanders%26aufirst%3DD.%26aulast%3DDandekar%26aufirst%3DM.%26aulast%3DCollins%26aufirst%3DC.%26aulast%3DRobertson%26aufirst%3DG.%26aulast%3DMacDonald%26aufirst%3DJ.%26aulast%3DCezard%26aufirst%3DT.%26aulast%3DBilenky%26aufirst%3DM.%26aulast%3DThiessen%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DT.%26aulast%3DHirst%26aufirst%3DM.%26aulast%3DHero%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Hoxa9%2520Binding%2520Sites%2520in%2520Hematopoietic%2520Cells%26jtitle%3DBlood%26date%3D2011%26volume%3D119%26spage%3D388%26epage%3D398%26doi%3D10.1182%2Fblood-2011-03-341081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaRonde-LeBlanc, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span> <span> </span><span class="NLM_article-title">Structure of HoxA9 and Pbx1 Bound to DNA: Hox Hexapeptide and DNA Recognition Anterior to Posterior</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.1101/gad.1103303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1101%2Fgad.1103303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=2060-2072&author=N.+A.+LaRonde-LeBlancauthor=C.+Wolberger&title=Structure+of+HoxA9+and+Pbx1+Bound+to+DNA%3A+Hox+Hexapeptide+and+DNA+Recognition+Anterior+to+Posterior&doi=10.1101%2Fgad.1103303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1101%2Fgad.1103303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1103303%26sid%3Dliteratum%253Aachs%26aulast%3DLaRonde-LeBlanc%26aufirst%3DN.%2BA.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructure%2520of%2520HoxA9%2520and%2520Pbx1%2520Bound%2520to%2520DNA%253A%2520Hox%2520Hexapeptide%2520and%2520DNA%2520Recognition%2520Anterior%2520to%2520Posterior%26jtitle%3DGenes%2520Dev.%26date%3D2003%26volume%3D17%26spage%3D2060%26epage%3D2072%26doi%3D10.1101%2Fgad.1103303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Direct Inhibition of the DNA-Binding Activity of POU Transcription Factors Pit-1 and Brn-3 by Selective Binding of a Phenyl-Furan-Benzimidazole Dication</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3341</span>– <span class="NLM_lpage">3353</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkn208" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3341-3353&author=P.+Peixotoauthor=Y.+Liuauthor=S.+Depauwauthor=M.-P.+Hildebrandauthor=D.+W.+Boykinauthor=C.+Baillyauthor=W.+D.+Wilsonauthor=M.-H.+David-Cordonnier&title=Direct+Inhibition+of+the+DNA-Binding+Activity+of+POU+Transcription+Factors+Pit-1+and+Brn-3+by+Selective+Binding+of+a+Phenyl-Furan-Benzimidazole+Dication&doi=10.1093%2Fnar%2Fgkn208"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn208%26sid%3Dliteratum%253Aachs%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DHildebrand%26aufirst%3DM.-P.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DBailly%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DDirect%2520Inhibition%2520of%2520the%2520DNA-Binding%2520Activity%2520of%2520POU%2520Transcription%2520Factors%2520Pit-1%2520and%2520Brn-3%2520by%2520Selective%2520Binding%2520of%2520a%2520Phenyl-Furan-Benzimidazole%2520Dication%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3341%26epage%3D3353%26doi%3D10.1093%2Fnar%2Fgkn208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flajollet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dezitter, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duterque-Coquillaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Wilson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA-Binding Activity of the Human ERG Transcription Factor Using New Heterocyclic Dithiophene Diamidines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1093/nar/gks971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgks971" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=125-138&author=R.+Nhiliauthor=P.+Peixotoauthor=S.+Depauwauthor=S.+Flajolletauthor=X.+Dezitterauthor=M.+M.+Mundeauthor=M.+A.+Ismailauthor=A.+Kumarauthor=A.+A.+Farahatauthor=C.+E.+Stephensauthor=M.+Duterque-Coquillaudauthor=W.+David+Wilsonauthor=D.+W.+Boykinauthor=M.-H.+David-Cordonnier&title=Targeting+the+DNA-Binding+Activity+of+the+Human+ERG+Transcription+Factor+Using+New+Heterocyclic+Dithiophene+Diamidines&doi=10.1093%2Fnar%2Fgks971"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks971%26sid%3Dliteratum%253Aachs%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DFlajollet%26aufirst%3DS.%26aulast%3DDezitter%26aufirst%3DX.%26aulast%3DMunde%26aufirst%3DM.%2BM.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DDuterque-Coquillaud%26aufirst%3DM.%26aulast%3DDavid%2BWilson%26aufirst%3DW.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520the%2520DNA-Binding%2520Activity%2520of%2520the%2520Human%2520ERG%2520Transcription%2520Factor%2520Using%2520New%2520Heterocyclic%2520Dithiophene%2520Diamidines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D41%26spage%3D125%26epage%3D138%26doi%3D10.1093%2Fnar%2Fgks971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Say, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">The Unusual Monomer Recognition of Guanine-Containing Mixed Sequence DNA by a Dithiophene Heterocyclic Diamidine</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1227</span>, <span class="refDoi"> DOI: 10.1021/bi401582t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi401582t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslygsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=1218-1227&author=M.+Mundeauthor=A.+Kumarauthor=P.+Peixotoauthor=S.+Depauwauthor=M.+A.+Ismailauthor=A.+A.+Farahatauthor=A.+Paulauthor=M.+V.+Sayauthor=M.-H.+David-Cordonnierauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=The+Unusual+Monomer+Recognition+of+Guanine-Containing+Mixed+Sequence+DNA+by+a+Dithiophene+Heterocyclic+Diamidine&doi=10.1021%2Fbi401582t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The Unusual Monomer Recognition of Guanine-Containing Mixed Sequence DNA by a Dithiophene Heterocyclic Diamidine</span></div><div class="casAuthors">Munde, Manoj; Kumar, Arvind; Peixoto, Paul; Depauw, Sabine; Ismail, Mohamed A.; Farahat, Abdelbasset A.; Paul, Ananya; Say, Martial V.; David-Cordonnier, Marie-Helene; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1218-1227</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DB1255 is a sym. diamidinophenyl-dithiophene that exhibits cellular activity by binding to DNA and inhibiting binding of ERG, an ETS family transcription factor that is commonly overexpressed or translocated in leukemia and prostate cancer.  Because transcription factor inhibition is complex but is an attractive area for anticancer and antiparasitic drug development, the authors have evaluated the DNA interactions of addnl. derivs. of DB1255 to gain an improved understanding of the biophys. chem. of complex function and inhibition.  DNase I footprinting, biosensor surface plasmon resonance, and CD expts. show that DB1255 has an unusual and strong monomer binding mode in minor groove sites that contain a single GC base pair flanked by AT base pairs, for example, 5'-ATGAT-3'.  Closely related derivs., such as compds. with the thiophene replaced with furan or selenophane, bind very weakly to GC-contg. sequences and do not have biol. activity.  DB1255 is selective for the ATGAT site; however, a similar sequence, 5'-ATGAC-3', binds DB1255 more weakly and does not produce a footprint.  Mol. docking studies show that the two thiophene sulfur atoms form strong, bifurcated hydrogen bond-type interactions with the G-N-H sequence that extends into the minor groove while the amidines form hydrogen bonds to the flanking AT base pairs.  The central dithiophene unit of DB1255 thus forms an excellent, but unexpected, single-GC base pair recognition module in a monomer minor groove complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rJFOeeJwP7Vg90H21EOLACvtfcHk0ljn1TTnEmPX4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslygsr8%253D&md5=8db5faa095f9114d368aa45108db0424</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi401582t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi401582t%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DSay%26aufirst%3DM.%2BV.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThe%2520Unusual%2520Monomer%2520Recognition%2520of%2520Guanine-Containing%2520Mixed%2520Sequence%2520DNA%2520by%2520a%2520Dithiophene%2520Heterocyclic%2520Diamidine%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D1218%26epage%3D1227%26doi%3D10.1021%2Fbi401582t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista-Parra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">DNA Minor Groove Induced Dimerization of Heterocyclic Cations: Compound Structure, Binding Affinity, and Specificity for a TTAA Site</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>402</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.jmb.2010.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20713062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=2010&pages=847-864&author=M.+Mundeauthor=A.+Kumarauthor=R.+Nhiliauthor=S.+Depauwauthor=M.-H.+David-Cordonnierauthor=M.+A.+Ismailauthor=C.+E.+Stephensauthor=A.+A.+Farahatauthor=A.+Batista-Parraauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=DNA+Minor+Groove+Induced+Dimerization+of+Heterocyclic+Cations%3A+Compound+Structure%2C+Binding+Affinity%2C+and+Specificity+for+a+TTAA+Site&doi=10.1016%2Fj.jmb.2010.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Minor Groove Induced Dimerization of Heterocyclic Cations: Compound Structure, Binding Affinity, and Specificity for a TTAA Site</span></div><div class="casAuthors">Munde, Manoj; Kumar, Arvind; Nhili, Raja; Depauw, Sabine; David-Cordonnier, Marie-Helene; Ismail, Mohamed A.; Stephens, Chad E.; Farahat, Abdelbasset A.; Batista-Parra, Adalgisa; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">402</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">847-864</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With the increasing no. and variations of genome sequences available, control of gene expression with synthetic, cell-permeable mols. is within reach.  The variety of sequence-specific binding agents is, however, still quite limited.  Many minor groove binding agents selectivity recognize AT over GC sequences but have less ability to distinguish among different AT sequences.  The goal with this article is to develop compds. that can bind selectively to different AT sequences.  A no. of studies indicate that AATT and TTAA sequences have significantly different phys. and interaction properties and different requirements for minor groove recognition.  Although it has been difficult to get minor groove binding at TTAA, DB293, a phenyl-furan-benzimidazole diamidine, was found to bind as a strong, cooperative dimer at TTAA but with no selectivity over AATT.  In order to improve selectivity, we made modifications to each unit of DB293.  Binding affinities and stoichiometries obtained from biosensor-surface plasmon resonance expts. show that DB1003, a furan-furan-benzimidazole diamidine, binds strongly to TTAA as a dimer and has selectivity (KTTAA/KAATT = 6).  CD and DNase I footprinting studies confirmed the preference of this compd. for TTAA.  In summary, (i) a favorable stacking surface provided by the pi system, (ii) H-bond donors to interact with TA base pairs at the floor of the groove provided by a benzimidazole (or indole) -NH and amidines, and (iii) appropriate curvature of the dimer complex to match the curvature of the minor groove play important roles in differentiating the TTAA and AATT minor grooves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnCOf12xu4WbVg90H21EOLACvtfcHk0ljn1TTnEmPX4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7jP&md5=ad50b8908a3ec39796cd1fadc79345e8</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBatista-Parra%26aufirst%3DA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DDNA%2520Minor%2520Groove%2520Induced%2520Dimerization%2520of%2520Heterocyclic%2520Cations%253A%2520Compound%2520Structure%252C%2520Binding%2520Affinity%252C%2520and%2520Specificity%2520for%2520a%2520TTAA%2520Site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D402%26spage%3D847%26epage%3D864%26doi%3D10.1016%2Fj.jmb.2010.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Say, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chackal-Catoen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista-Parra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Water-Mediated Binding of Agents That Target the DNA Minor Groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">10171</span>– <span class="NLM_lpage">10183</span>, <span class="refDoi"> DOI: 10.1021/ja202006u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202006u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1ent74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=10171-10183&author=Y.+Liuauthor=A.+Kumarauthor=S.+Depauwauthor=R.+Nhiliauthor=M.-H.+David-Cordonnierauthor=M.+P.+Leeauthor=M.+A.+Ismailauthor=A.+A.+Farahatauthor=M.+Sayauthor=S.+Chackal-Catoenauthor=A.+Batista-Parraauthor=S.+Neidleauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Water-Mediated+Binding+of+Agents+That+Target+the+DNA+Minor+Groove&doi=10.1021%2Fja202006u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Mediated Binding of Agents that Target the DNA Minor Groove</span></div><div class="casAuthors">Liu, Yang; Kumar, Arvind; Depauw, Sabine; Nhili, Raja; David-Cordonnier, Marie-Helene; Lee, Michael P.; Ismail, Mohamed A.; Farahat, Abdelbasset A.; Say, Martial; Chackal-Catoen, Sarah; Batista-Parra, Adalgisa; Neidle, Stephen; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10171-10183</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. complexes with DNA that incorporate linking water mols. are rare, and the DB921-DNA complex has provided a unique and well-defined system for anal. of water-mediated binding in the context of a DNA complex.  DB921 has a benzimidazole-biphenyl system with terminal amidines that results in a linear conformation that does not possess the appropriate radius of curvature to match the minor groove shape and represents a new paradigm that does not fit the classical model of minor groove interactions.  To better understand the role of the bound water mol. obsd. in the x-ray crystal structure of the DB921 complex, synthetic modifications have been made in the DB921 structure, and the interactions of the new compds. with DNA AT sites have been evaluated with an array of methods, including DNase I footprinting, biosensor-surface plasmon resonance, isothermal titrn. microcalorimetry, and CD.  The interaction of a key compd., which has the amidine at the Ph shifted from the para position in DB921 to the meta position, has also been examd. by x-ray crystallog.  The detailed structural, thermodn., and kinetic results provide valuable new information for incorporation of water mols. in the design of new lead scaffolds for targeting DNA in chem. biol. and therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSvdCZXTKwj7Vg90H21EOLACvtfcHk0liDMFy87jzXvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1ent74%253D&md5=01200ceedc05328d90050ac02cf236ad</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja202006u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202006u%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DLee%26aufirst%3DM.%2BP.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DSay%26aufirst%3DM.%26aulast%3DChackal-Catoen%26aufirst%3DS.%26aulast%3DBatista-Parra%26aufirst%3DA.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DWater-Mediated%2520Binding%2520of%2520Agents%2520That%2520Target%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D10171%26epage%3D10183%26doi%3D10.1021%2Fja202006u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Thiophene-Based Diamidine Forms a ″Super″ AT Binding Minor Groove Agent</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">13659</span>– <span class="NLM_lpage">13669</span>, <span class="refDoi"> DOI: 10.1021/ja048175m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048175m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=13659-13669&author=S.+Mallenaauthor=M.+P.+H.+Leeauthor=C.+Baillyauthor=S.+Neidleauthor=A.+Kumarauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Thiophene-Based+Diamidine+Forms+a+%E2%80%B3Super%E2%80%B3+AT+Binding+Minor+Groove+Agent&doi=10.1021%2Fja048175m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja048175m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048175m%26sid%3Dliteratum%253Aachs%26aulast%3DMallena%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BP.%2BH.%26aulast%3DBailly%26aufirst%3DC.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThiophene-Based%2520Diamidine%2520Forms%2520a%2520%25E2%2580%25B3Super%25E2%2580%25B3%2520AT%2520Binding%2520Minor%2520Groove%2520Agent%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D13659%26epage%3D13669%26doi%3D10.1021%2Fja048175m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-Based DNA-Targeting Strategies with Small Molecule Ligands for Drug Discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1002/med.21278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fmed.21278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=23633219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ChtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1119-1173&author=J.+Shengauthor=J.+Ganauthor=Z.+Huang&title=Structure-Based+DNA-Targeting+Strategies+with+Small+Molecule+Ligands+for+Drug+Discovery&doi=10.1002%2Fmed.21278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based DNA-Targeting Strategies with Small Molecule Ligands for Drug Discovery</span></div><div class="casAuthors">Sheng, Jia; Gan, Jianhua; Huang, Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1119-1173</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleic acids are the mol. targets of many clin. anticancer drugs.  However, compared with proteins, nucleic acids have traditionally attracted much less attention as drug targets in structure-based drug design, partially because limited structural information of nucleic acids complexed with potential drugs is available.  Over the past several years, enormous progresses in nucleic acid crystn., heavy-atom derivatization, phasing, and structural biol. have been made.  Many complicated nucleic acid structures have been detd., providing new insights into the mol. functions and interactions of nucleic acids, esp. DNAs complexed with small mol. ligands.  Thus, opportunities have been created to further discover nucleic acid-targeting drugs for disease treatments.  This review focuses on the structure studies of DNAs complexed with small mol. ligands for discovering lead compds., drug candidates, and/or therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozFxbOusdeU7Vg90H21EOLACvtfcHk0ljzlIAz_xi8uQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ChtbvO&md5=876fcb98aa17acf928d9d6bce0619d40</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fmed.21278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21278%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26atitle%3DStructure-Based%2520DNA-Targeting%2520Strategies%2520with%2520Small%2520Molecule%2520Ligands%2520for%2520Drug%2520Discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D1119%26epage%3D1173%26doi%3D10.1002%2Fmed.21278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansiaux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span> <span> </span><span class="NLM_article-title">Distribution of furamidine analogues in tumor cells: targeting of the nucleus or mitochondria depending on the amidine substitution</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7219</span>– <span class="NLM_lpage">7229</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=7219-7229&author=A.+Lansiauxauthor=F.+Taniousauthor=Z.+Mishalauthor=L.+Dassonnevilleauthor=A.+Kumarauthor=C.+E.+Stephensauthor=Q.+Huauthor=W.+D.+Wilsonauthor=D.+W.+Boykinauthor=C.+Bailly&title=Distribution+of+furamidine+analogues+in+tumor+cells%3A+targeting+of+the+nucleus+or+mitochondria+depending+on+the+amidine+substitution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLansiaux%26aufirst%3DA.%26aulast%3DTanious%26aufirst%3DF.%26aulast%3DMishal%26aufirst%3DZ.%26aulast%3DDassonneville%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DBailly%26aufirst%3DC.%26atitle%3DDistribution%2520of%2520furamidine%2520analogues%2520in%2520tumor%2520cells%253A%2520targeting%2520of%2520the%2520nucleus%2520or%2520mitochondria%2520depending%2520on%2520the%2520amidine%2520substitution%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D7219%26epage%3D7229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghannam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaseen, N. R.</span></span> <span> </span><span class="NLM_article-title">The OncogeneNup98-HOXA9Induces Gene Transcription in Myeloid Cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1074/jbc.m307280200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1074%2Fjbc.m307280200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2003&pages=866-875&author=G.+Ghannamauthor=A.+Takedaauthor=T.+Camarataauthor=M.+A.+Mooreauthor=A.+Vialeauthor=N.+R.+Yaseen&title=The+OncogeneNup98-HOXA9Induces+Gene+Transcription+in+Myeloid+Cells&doi=10.1074%2Fjbc.m307280200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m307280200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m307280200%26sid%3Dliteratum%253Aachs%26aulast%3DGhannam%26aufirst%3DG.%26aulast%3DTakeda%26aufirst%3DA.%26aulast%3DCamarata%26aufirst%3DT.%26aulast%3DMoore%26aufirst%3DM.%2BA.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DYaseen%26aufirst%3DN.%2BR.%26atitle%3DThe%2520OncogeneNup98-HOXA9Induces%2520Gene%2520Transcription%2520in%2520Myeloid%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D279%26spage%3D866%26epage%3D875%26doi%3D10.1074%2Fjbc.m307280200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venturini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquez, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span> <span> </span><span class="NLM_article-title">Polycomb Repressor Complex 2 Regulates HOXA9 and HOXA10, Activating ID2 in NK/T-Cell Lines</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">151</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-9-151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2F1476-4598-9-151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20565746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC3cnjvVGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=151&author=S.+Nagelauthor=L.+Venturiniauthor=V.+E.+Marquezauthor=C.+Meyerauthor=M.+Kaufmannauthor=M.+Scherrauthor=R.+A.+MacLeodauthor=H.+G.+Drexler&title=Polycomb+Repressor+Complex+2+Regulates+HOXA9+and+HOXA10%2C+Activating+ID2+in+NK%2FT-Cell+Lines&doi=10.1186%2F1476-4598-9-151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb repressor complex 2 regulates HOXA9 and HOXA10, activating ID2 in NK/T-cell lines</span></div><div class="casAuthors">Nagel Stefan; Venturini Letizia; Marquez Victor E; Meyer Corinna; Kaufmann Maren; Scherr Michaela; MacLeod Roderick Af; Drexler Hans G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  NK- and T-cells are closely related lymphocytes, originating from the same early progenitor cells during hematopoiesis.  In these differentiation processes deregulation of developmental genes may contribute to leukemogenesis.  Here, we compared expression profiles of NK- and T-cell lines for identification of aberrantly expressed genes in T-cell acute lymphoblastic leukemia (T-ALL) which physiologically regulate the differentiation program of the NK-cell lineage.  RESULTS:  This analysis showed high expression levels of HOXA9, HOXA10 and ID2 in NK-cell lines in addition to T-cell line LOUCY, suggesting leukemic deregulation therein.  Overexpression experiments, chromatin immuno-precipitation and promoter analysis demonstrated that HOXA9 and HOXA10 directly activated expression of ID2.  Concomitantly elevated expression levels of HOXA9 and HOXA10 together with ID2 in cell lines containing MLL translocations confirmed this form of regulation in both ALL and acute myeloid leukemia.  Overexpression of HOXA9, HOXA10 or ID2 resulted in repressed expression of apoptosis factor BIM.  Furthermore, profiling data of genes coding for chromatin regulators of homeobox genes, including components of polycomb repressor complex 2 (PRC2), indicated lacking expression of EZH2 in LOUCY and exclusive expression of HOP in NK-cell lines.  Subsequent treatment of T-cell lines JURKAT and LOUCY with DZNep, an inhibitor of EZH2/PRC2, resulted in elevated and unchanged HOXA9/10 expression levels, respectively.  Moreover, siRNA-mediated knockdown of EZH2 in JURKAT enhanced HOXA10 expression, confirming HOXA10-repression by EZH2.  Additionally, profiling data and overexpression analysis indicated that reduced expression of E2F cofactor TFDP1 contributed to the lack of EZH2 in LOUCY.  Forced expression of HOP in JURKAT cells resulted in reduced HOXA10 and ID2 expression levels, suggesting enhancement of PRC2 repression.  CONCLUSIONS:  Our results show that major differentiation factors of the NK-cell lineage, including HOXA9, HOXA10 and ID2, were (de)regulated via PRC2 which therefore contributes to T-cell leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWv8RzrWVfouQ4Pl9hzjQhfW6udTcc2eZqg0AidNg29bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnjvVGjsg%253D%253D&md5=485d85bee87872e3c643b01dae469e86</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-9-151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-9-151%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DS.%26aulast%3DVenturini%26aufirst%3DL.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DScherr%26aufirst%3DM.%26aulast%3DMacLeod%26aufirst%3DR.%2BA.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DPolycomb%2520Repressor%2520Complex%25202%2520Regulates%2520HOXA9%2520and%2520HOXA10%252C%2520Activating%2520ID2%2520in%2520NK%252FT-Cell%2520Lines%26jtitle%3DMol.%2520Cancer%26date%3D2010%26volume%3D9%26spage%3D151%26doi%3D10.1186%2F1476-4598-9-151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velu, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaubey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jegga, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancelas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebelein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, H. L.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Antagonists of MicroRNAs Deplete Leukemia-Initiating Cell Activity</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1172/jci66005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci66005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=222-236&author=C.+S.+Veluauthor=A.+Chaubeyauthor=J.+D.+Phelanauthor=S.+R.+Hormanauthor=M.+Wunderlichauthor=M.+L.+Guzmanauthor=A.+G.+Jeggaauthor=N.+J.+Zeleznik-Leauthor=J.+Chenauthor=J.+C.+Mulloyauthor=J.+A.+Cancelasauthor=C.+T.+Jordanauthor=B.+J.+Aronowauthor=G.+Marcucciauthor=B.+Bhatauthor=B.+Gebeleinauthor=H.+L.+Grimes&title=Therapeutic+Antagonists+of+MicroRNAs+Deplete+Leukemia-Initiating+Cell+Activity&doi=10.1172%2Fjci66005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1172%2Fjci66005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci66005%26sid%3Dliteratum%253Aachs%26aulast%3DVelu%26aufirst%3DC.%2BS.%26aulast%3DChaubey%26aufirst%3DA.%26aulast%3DPhelan%26aufirst%3DJ.%2BD.%26aulast%3DHorman%26aufirst%3DS.%2BR.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DJegga%26aufirst%3DA.%2BG.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DCancelas%26aufirst%3DJ.%2BA.%26aulast%3DJordan%26aufirst%3DC.%2BT.%26aulast%3DAronow%26aufirst%3DB.%2BJ.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DB.%26aulast%3DGebelein%26aufirst%3DB.%26aulast%3DGrimes%26aufirst%3DH.%2BL.%26atitle%3DTherapeutic%2520Antagonists%2520of%2520MicroRNAs%2520Deplete%2520Leukemia-Initiating%2520Cell%2520Activity%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D124%26spage%3D222%26epage%3D236%26doi%3D10.1172%2Fjci66005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscinski, L. C.</span></span> <span> </span><span class="NLM_article-title">β2 Integrins are characteristically absent in acute promyelocytic leukemia and rapidly upregulated in vivo upon differentiation with all-trans retinoic acid</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2006.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.leukres.2006.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=16764927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BD28nmvFKksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2007&pages=49-57&author=J.+J.+Wuauthor=A.+Cantorauthor=L.+C.+Moscinski&title=%CE%B22+Integrins+are+characteristically+absent+in+acute+promyelocytic+leukemia+and+rapidly+upregulated+in+vivo+upon+differentiation+with+all-trans+retinoic+acid&doi=10.1016%2Fj.leukres.2006.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">beta2 Integrins are characteristically absent in acute promyelocytic leukemia and rapidly upregulated in vivo upon differentiation with all-trans retinoic acid</span></div><div class="casAuthors">Wu Jamie J; Cantor Alan; Moscinski Lynn C</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-57</span>
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    </div><div class="casAbstract">Although little is known about migration of hematopoietic stem cells and their neoplastic counterparts into tissues and peripheral blood, adhesion proteins likely play an important role.  We studied 339 patients with acute myelogenous leukemia (AML) to discern the relationship between adhesion protein expression, circulating blasts, and white blood cell (WBC) count.  Expression levels of CD11b and CD11c strongly correlated with increased WBC count, independent of FAB subtype (p<0.0001).  However, 93% (25/27) of cases of AML-M3 completely lacked beta2 integrin expression, compared to 11% (35/312) of the non-M3 cases (p<0.0001).  Seven of the 27 patients with AML-M3 were followed during standard induction therapy with ATRA.  Within 3 days, weak CD11c became detectable, followed by CD11b and CD11a.  Our data suggest an important link between beta2 integrin expression and the level of circulating leukemic cells in AML.  We demonstrate the clinical usefulness of a panel of beta2 integrins (CD11a, CD11b and CD11c) in accurate prediction of AML-M3, and recommend inclusion of this immunophenotypic analysis to identify patients who require ATRA therapy.  Finally, we illustrate the rapidity at which AML-M3 blasts up-regulate beta2 integrins, and suggest a possible association between this finding and the tissue infiltration that characterizes the "ATRA syndrome".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3NtIGKSCkv2Zyxfu64m4ffW6udTcc2eZqg0AidNg29bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28nmvFKksw%253D%253D&md5=1030910c1f46798fa388c019486650c6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2006.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2006.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%2BJ.%26aulast%3DCantor%26aufirst%3DA.%26aulast%3DMoscinski%26aufirst%3DL.%2BC.%26atitle%3D%25CE%25B22%2520Integrins%2520are%2520characteristically%2520absent%2520in%2520acute%2520promyelocytic%2520leukemia%2520and%2520rapidly%2520upregulated%2520in%2520vivo%2520upon%2520differentiation%2520with%2520all-trans%2520retinoic%2520acid%26jtitle%3DLeuk.%2520Res.%26date%3D2007%26volume%3D31%26spage%3D49%26epage%3D57%26doi%3D10.1016%2Fj.leukres.2006.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span> <span> </span><span class="NLM_article-title">Upregulation of CD11b and CD86 through LSD1 Inhibition Promotes Myeloid Differentiation and Suppresses Cell Proliferation in Human Monocytic Leukemia Cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">85085</span>– <span class="NLM_lpage">85101</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.18632%2Foncotarget.18564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=85085-85101&author=J.+Fangauthor=H.+Yingauthor=T.+Maoauthor=Y.+Fangauthor=Y.+Luauthor=H.+Wangauthor=I.+Zangauthor=Z.+Wangauthor=Y.+Linauthor=M.+Zhaoauthor=X.+Luoauthor=Z.+Wangauthor=Y.+Zhangauthor=C.+Zhangauthor=W.+Xiaoauthor=Y.+Wangauthor=W.+Tanauthor=Z.+Chenauthor=C.+Luauthor=P.+Atadjaauthor=E.+Liauthor=K.+Zhaoauthor=J.+Liuauthor=J.+Gu&title=Upregulation+of+CD11b+and+CD86+through+LSD1+Inhibition+Promotes+Myeloid+Differentiation+and+Suppresses+Cell+Proliferation+in+Human+Monocytic+Leukemia+Cells&doi=10.18632%2Foncotarget.18564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18564%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DJ.%26atitle%3DUpregulation%2520of%2520CD11b%2520and%2520CD86%2520through%2520LSD1%2520Inhibition%2520Promotes%2520Myeloid%2520Differentiation%2520and%2520Suppresses%2520Cell%2520Proliferation%2520in%2520Human%2520Monocytic%2520Leukemia%2520Cells%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D85085%26epage%3D85101%26doi%3D10.18632%2Foncotarget.18564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marincola, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroncek, D. F.</span></span> <span> </span><span class="NLM_article-title">Molecular Signatures of Maturing Dendritic Cells: Implications for Testing the Quality of Dendritic Cell Therapies</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1186/1479-5876-8-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2F1479-5876-8-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20078880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ptlWntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=4&author=P.+Jinauthor=T.+Hanauthor=J.+Renauthor=S.+Saundersauthor=E.+Wangauthor=F.+M.+Marincolaauthor=D.+F.+Stroncek&title=Molecular+Signatures+of+Maturing+Dendritic+Cells%3A+Implications+for+Testing+the+Quality+of+Dendritic+Cell+Therapies&doi=10.1186%2F1479-5876-8-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular signatures of maturing dendritic cells: implications for testing the quality of dendritic cell therapies</span></div><div class="casAuthors">Jin Ping; Han Tae Hee; Ren Jiaqiang; Saunders Stefanie; Wang Ena; Marincola Francesco M; Stroncek David F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of translational medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Dendritic cells (DCs) are often produced by granulocyte-macrophage colony-stimulating factor (GM-CSF) and interleukin-4 (IL-4) stimulation of monocytes.  To improve the effectiveness of DC adoptive immune cancer therapy, many different agents have been used to mature DCs.  We analyzed the kinetics of DC maturation by lipopolysaccharide (LPS) and interferon-gamma (IFN-gamma) induction in order to characterize the usefulness of mature DCs (mDCs) for immune therapy and to identify biomarkers for assessing the quality of mDCs.  METHODS:  Peripheral blood mononuclear cells were collected from 6 healthy subjects by apheresis, monocytes were isolated by elutriation, and immature DCs (iDCs) were produced by 3 days of culture with GM-CSF and IL-4.  The iDCs were sampled after 4, 8 and 24 hours in culture with LPS and IFN-gamma and were then assessed by flow cytometry, ELISA, and global gene and microRNA (miRNA) expression analysis.  RESULTS:  After 24 hours of LPS and IFN-gamma stimulation, DC surface expression of CD80, CD83, CD86, and HLA Class II antigens were up-regulated.  Th1 attractant genes such as CXCL9, CXCL10, CXCL11 and CCL5 were up-regulated during maturation but not Treg attractants such as CCL22 and CXCL12.  The expression of classical mDC biomarker genes CD83, CCR7, CCL5, CCL8, SOD2, MT2A, OASL, GBP1 and HES4 were up-regulated throughout maturation while MTIB, MTIE, MTIG, MTIH, GADD45A and LAMP3 were only up-regulated late in maturation.  The expression of miR-155 was up-regulated 8-fold in mDCs.  CONCLUSION:  DCs, matured with LPS and IFN-gamma, were characterized by increased levels of Th1 attractants as opposed to Treg attractants and may be particularly effective for adoptive immune cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8ValubdQ-Q9K0qsdY6Nc1fW6udTcc2ebSF8jNgX-8j7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ptlWntg%253D%253D&md5=f66c11cb98efbe921a26de3b7ade2a27</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1186%2F1479-5876-8-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-5876-8-4%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DSaunders%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DMarincola%26aufirst%3DF.%2BM.%26aulast%3DStroncek%26aufirst%3DD.%2BF.%26atitle%3DMolecular%2520Signatures%2520of%2520Maturing%2520Dendritic%2520Cells%253A%2520Implications%2520for%2520Testing%2520the%2520Quality%2520of%2520Dendritic%2520Cell%2520Therapies%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2010%26volume%3D8%26spage%3D4%26doi%3D10.1186%2F1479-5876-8-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phuc Khoa, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koche, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressler, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Reprograms the Enhancer Landscape to Promote Leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccell.2018.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=30270123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVahsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=643-658&author=Y.+Sunauthor=B.+Zhouauthor=F.+Maoauthor=J.+Xuauthor=H.+Miaoauthor=Z.+Zouauthor=L.+T.+Phuc+Khoaauthor=Y.+Jangauthor=S.+Caiauthor=M.+Witkinauthor=R.+Kocheauthor=K.+Geauthor=G.+R.+Dresslerauthor=R.+L.+Levineauthor=S.+A.+Armstrongauthor=Y.+Douauthor=J.+L.+Hess&title=HOXA9+Reprograms+the+Enhancer+Landscape+to+Promote+Leukemogenesis&doi=10.1016%2Fj.ccell.2018.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">HOXA9 Reprograms the Enhancer Landscape to Promote Leukemogenesis</span></div><div class="casAuthors">Sun, Yuqing; Zhou, Bo; Mao, Fengbiao; Xu, Jing; Miao, Hongzhi; Zou, Zhenhua; Phuc Khoa, Le Tran; Jang, Younghoon; Cai, Sheng; Witkin, Matthew; Koche, Richard; Ge, Kai; Dressler, Gregory R.; Levine, Ross L.; Armstrong, Scott A.; Dou, Yali; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">643-658.e5</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Aberrant expression of HOXA9 is a prominent feature of acute leukemia driven by diverse oncogenes.  Here we show that HOXA9 overexpression in myeloid and B progenitor cells leads to significant enhancer reorganizations with prominent emergence of leukemia-specific de novo enhancers.  Alterations in the enhancer landscape lead to activation of an ectopic embryonic gene program.  We show that HOXA9 functions as a pioneer factor at de novo enhancers and recruits CEBPα and the MLL3/MLL4 complex.  Genetic deletion of MLL3/MLL4 blocks histone H3K4 methylation at de novo enhancers and inhibits HOXA9/MEIS1-mediated leukemogenesis in vivo.  These results suggest that therapeutic targeting of HOXA9-dependent enhancer reorganization can be an effective therapeutic strategy in acute leukemia with HOXA9 overexpression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnd105uBPOFbVg90H21EOLACvtfcHk0lg9RND537KZjw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVahsbzE&md5=88f4ad9726cc4dd8f038e77d6653c7eb</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DPhuc%2BKhoa%26aufirst%3DL.%2BT.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DS.%26aulast%3DWitkin%26aufirst%3DM.%26aulast%3DKoche%26aufirst%3DR.%26aulast%3DGe%26aufirst%3DK.%26aulast%3DDressler%26aufirst%3DG.%2BR.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DHOXA9%2520Reprograms%2520the%2520Enhancer%2520Landscape%2520to%2520Promote%2520Leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D34%26spage%3D643%26epage%3D658%26doi%3D10.1016%2Fj.ccell.2018.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therkelsen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernt, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span> <span> </span><span class="NLM_article-title">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.-C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+Killing+of+Mixed+Lineage+Leukemia+Cells+by+a+Potent+Small-Molecule+DOT1L+Inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lhpalI4jwoVKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.-C.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520Killing%2520of%2520Mixed%2520Lineage%2520Leukemia%2520Cells%2520by%2520a%2520Potent%2520Small-Molecule%2520DOT1L%2520Inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span> <span> </span><span class="NLM_article-title">Preclinical Pharmacokinetics and Pharmacodynamics of Pinometostat (EPZ-5676), a First-in-Class, Small Molecule S-Adenosyl Methionine Competitive Inhibitor of DOT1L</span>. <i>Eur. J. Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1007/s13318-017-0404-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1007%2Fs13318-017-0404-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28229434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtFKls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=891-901&author=N.+J.+Waters&title=Preclinical+Pharmacokinetics+and+Pharmacodynamics+of+Pinometostat+%28EPZ-5676%29%2C+a+First-in-Class%2C+Small+Molecule+S-Adenosyl+Methionine+Competitive+Inhibitor+of+DOT1L&doi=10.1007%2Fs13318-017-0404-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Pharmacokinetics and Pharmacodynamics of Pinometostat (EPZ-5676), a First-in-Class, Small Molecule S-Adenosyl Methionine Competitive Inhibitor of DOT1L</span></div><div class="casAuthors">Waters, Nigel J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">891-901</span>CODEN:
                <span class="NLM_cas:coden">EJDPD2</span>;
        ISSN:<span class="NLM_cas:issn">0378-7966</span>.
    
            (<span class="NLM_cas:orgname">Springer France</span>)
        </div><div class="casAbstract">Acute leukemias bearing mixed lineage leukemia (MLL) rearrangements are aggressive diseases characterized by a poor overall prognosis despite multi-agent chemotherapy.  Aberrant fusion proteins involving the MLL histone methyltransferase (HMT) lead to recruitment of DOT1L, to a multi-protein complex resulting in aberrant methylation of histone H3 lysine 79 at MLL target genes, and ultimately enhanced expression of crit. genes for hematopoietic differentiation, including HOXA9 and MEIS1, and as such defines the established mechanism for leukemogenesis in MLL-rearrangement (MLL-r) leukemias.  Pinometostat is a first-in-class, small mol. inhibitor of DOT1L with sub-nanomolar affinity and >37,000 fold selectivity against non-MLL HMTs, and was the first member of the novel HMT inhibitor class to enter Phase 1 clin. trials in both adult and pediatric MLL-r leukemia patients.  In this article, the preclin. pharmacokinetics/pharmacodynamics and drug disposition of pinometostat are reviewed including discussion of how these data were used to inform early clin. studies, and how they translated to the clin. experience.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxU8juCNJnELVg90H21EOLACvtfcHk0liQWI08H1ZUqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtFKls7c%253D&md5=edacbc0f25f056f736705ae7c6f184cf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs13318-017-0404-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13318-017-0404-3%26sid%3Dliteratum%253Aachs%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26atitle%3DPreclinical%2520Pharmacokinetics%2520and%2520Pharmacodynamics%2520of%2520Pinometostat%2520%2528EPZ-5676%2529%252C%2520a%2520First-in-Class%252C%2520Small%2520Molecule%2520S-Adenosyl%2520Methionine%2520Competitive%2520Inhibitor%2520of%2520DOT1L%26jtitle%3DEur.%2520J.%2520Drug%2520Metab.%2520Pharmacokinet.%26date%3D2017%26volume%3D42%26spage%3D891%26epage%3D901%26doi%3D10.1007%2Fs13318-017-0404-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into Inhibition of the Bivalent Menin-MLL Interaction by Small Molecules in Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+Insights+into+Inhibition+of+the+Bivalent+Menin-MLL+Interaction+by+Small+Molecules+in+Leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0liQWI08H1ZUqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520Insights%2520into%2520Inhibition%2520of%2520the%2520Bivalent%2520Menin-MLL%2520Interaction%2520by%2520Small%2520Molecules%2520in%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fern%C3%A1ndez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+First-in-Class%2C+Highly+Potent+Irreversible+Inhibitor+Targeting+the+Menin-MLL+Protein-Protein+Interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0lie_fD-Jwq9Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFern%25C3%25A1ndez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520First-in-Class%252C%2520Highly%2520Potent%2520Irreversible%2520Inhibitor%2520Targeting%2520the%2520Menin-MLL%2520Protein-Protein%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Tanani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandha, H. S.</span></span> <span> </span><span class="NLM_article-title">Targeting HOX/PBX dimers in cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">32322</span>– <span class="NLM_lpage">32331</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.15971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.18632%2Foncotarget.15971" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=32322-32331&author=R.+Morganauthor=M.+El-Tananiauthor=K.+D.+Hunterauthor=K.+J.+Harringtonauthor=H.+S.+Pandha&title=Targeting+HOX%2FPBX+dimers+in+cancer&doi=10.18632%2Foncotarget.15971"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.15971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.15971%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DR.%26aulast%3DEl-Tanani%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DK.%2BD.%26aulast%3DHarrington%26aufirst%3DK.%2BJ.%26aulast%3DPandha%26aufirst%3DH.%2BS.%26atitle%3DTargeting%2520HOX%252FPBX%2520dimers%2520in%2520cancer%26jtitle%3DOncotarget%26date%3D2015%26volume%3D8%26spage%3D32322%26epage%3D32331%26doi%3D10.18632%2Foncotarget.15971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, G. M. K.</span></span> <span> </span><span class="NLM_article-title">Structure-Dependent Inhibition of the ETS-Family Transcription Factor PU.1 by Novel Heterocyclic Diamidines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkt955" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=1379-1390&author=M.+Mundeauthor=S.+Wangauthor=A.+Kumarauthor=C.+E.+Stephensauthor=A.+A.+Farahatauthor=D.+W.+Boykinauthor=W.+D.+Wilsonauthor=G.+M.+K.+Poon&title=Structure-Dependent+Inhibition+of+the+ETS-Family+Transcription+Factor+PU.1+by+Novel+Heterocyclic+Diamidines&doi=10.1093%2Fnar%2Fgkt955"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt955%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DPoon%26aufirst%3DG.%2BM.%2BK.%26atitle%3DStructure-Dependent%2520Inhibition%2520of%2520the%2520ETS-Family%2520Transcription%2520Factor%2520PU.1%2520by%2520Novel%2520Heterocyclic%2520Diamidines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D42%26spage%3D1379%26epage%3D1390%26doi%3D10.1093%2Fnar%2Fgkt955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anbazhagan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis of benzyloxybromobenzonitriles</span>. <i>Synth. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1081/scc-120028347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1081%2Fscc-120028347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFylsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=751-758&author=M.+A.+Ismailauthor=M.+Anbazhaganauthor=C.+E.+Stephensauthor=D.+W.+Boykin&title=Synthesis+of+benzyloxybromobenzonitriles&doi=10.1081%2Fscc-120028347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of benzyloxybromobenzonitriles</span></div><div class="casAuthors">Ismail, Mohamed A.; Anbazhagan, Mariappan; Stephens, Chad E.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">751-758</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">The synthesis of four new benzyloxybromobenzonitriles including: 2-benzyloxy-4-bromobenzonitrile, 3-benzyloxy-4-bromobenzonitrile, 5-benzyloxy-2-bromobenzonitrile, and 3-benzyloxy-2-bromobenzonitrile is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYxuYazkd4MrVg90H21EOLACvtfcHk0lie_fD-Jwq9Ug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFylsL0%253D&md5=d61f8813faeb4fa875212c21199a1823</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1081%2Fscc-120028347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252Fscc-120028347%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DAnbazhagan%26aufirst%3DM.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520of%2520benzyloxybromobenzonitriles%26jtitle%3DSynth.%2520Commun.%26date%3D2004%26volume%3D34%26spage%3D751%26epage%3D758%26doi%3D10.1081%2Fscc-120028347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barghash, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisa, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Bichalcophenes: A Concise Synthesis of Formyl Ester- and Cyano Ester-Substituted Bithiophenes, Bifurans, and Furanothiophenes</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1002/jhet.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fjhet.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=24634543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2010&pages=167-170&author=A.+A.+Farahatauthor=A.+Kumarauthor=A.+M.+Barghashauthor=F.+E.+Godaauthor=H.+M.+Eisaauthor=D.+W.+Boykin&title=Bichalcophenes%3A+A+Concise+Synthesis+of+Formyl+Ester-+and+Cyano+Ester-Substituted+Bithiophenes%2C+Bifurans%2C+and+Furanothiophenes&doi=10.1002%2Fjhet.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Bichalcophenes: A concise synthesis of formyl ester- and cyano ester-substituted bithiophenes, bifurans, and furanothiophenes</span></div><div class="casAuthors">Farahat, Abdelbasset A.; Kumar, Arvind; Barghash, Alaa El-Din M.; Goda, Fatma E.; Eisa, Hassan M.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-170</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Syntheses of formyl- and cyano methoxycarbonyl-substituted 2,2'-bithiophenes, 2,2'-bifurans, and 2-furan-2-ylthiophenes in good yield were described.  The key synthetic step utilized Stille coupling of appropriate brominated 5-ring heterocycles with stannylated 5-ring heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOxzOisylM_7Vg90H21EOLACvtfcHk0lhWbAD2Vy9xtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ktrk%253D&md5=32661d2dff161ef9b295f29ce18f2daf</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjhet.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.295%26sid%3Dliteratum%253Aachs%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBarghash%26aufirst%3DA.%2BM.%26aulast%3DGoda%26aufirst%3DF.%2BE.%26aulast%3DEisa%26aufirst%3DH.%2BM.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DBichalcophenes%253A%2520A%2520Concise%2520Synthesis%2520of%2520Formyl%2520Ester-%2520and%2520Cyano%2520Ester-Substituted%2520Bithiophenes%252C%2520Bifurans%252C%2520and%2520Furanothiophenes%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2010%26volume%3D47%26spage%3D167%26epage%3D170%26doi%3D10.1002%2Fjhet.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Dicationic biphenyl benzimidazole derivatives as antiprotozoal agents</span>. <i>Bioorg. & Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5405</span>– <span class="NLM_lpage">5413</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2004.07.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bmc.2004.07.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15388167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFWksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5405-5413&author=M.+A.+Ismailauthor=R.+Brunauthor=T.+Wenzlerauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Dicationic+biphenyl+benzimidazole+derivatives+as+antiprotozoal+agents&doi=10.1016%2Fj.bmc.2004.07.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Dicationic biphenyl benzimidazole derivatives as antiprotozoal agents</span></div><div class="casAuthors">Ismail, Mohamed A.; Brun, Reto; Wenzler, Tanja; Tanious, Farial A.; Wilson, W. David; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5405-5413</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of biphenyl benzimidazoles diamidines were synthesized from their resp. diamidoximes, through the bis-O-acetoxyamidoxime followed by hydrogenation in glacial acetic acid/ethanol in the presence of Pd-C.  The target compds. contain hydroxy and/or methoxy substituted 1,3-Ph groups as the central spacer between the two amidino bearing aryl groups.  All of the diamidines showed strong DNA affinities as judged by high ΔTm values with poly(dA·dT)2, which varied with structure and is discussed.  Seven of the nine new diamidines gave in vitro IC50 values of approx. 30 nM or less vs. Trypanosoma brucei rhodesiense (T.b.r.).  Generally the diamidines were less active vs. Plasmodium falciparum (P.f.), however one compd. exhibited excellent activity with an IC50 value of 2.1 nM.  Five of the nine diamidines exhibited excellent in vivo activity in the trypanosomal STIB900 mouse model giving 3/4 or 4/4 cures at dosage of 20 mg/kg i.p. and three showed similar efficacy at dosage of 10 mg/kg or lower.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-c9VNmcRZx7Vg90H21EOLACvtfcHk0lhWbAD2Vy9xtg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFWksrc%253D&md5=acfad2e365ee89fc0d8986a43b171db5</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.07.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.07.056%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DWenzler%26aufirst%3DT.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DDicationic%2520biphenyl%2520benzimidazole%2520derivatives%2520as%2520antiprotozoal%2520agents%26jtitle%3DBioorg.%2520%2526%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5405%26epage%3D5413%26doi%3D10.1016%2Fj.bmc.2004.07.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Novel Dicationic Imidazo[1,2-a]pyridines and 5,6,7,8-Tetrahydro-imidazo[1,2-a]pyridines as Antiprotozoal Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3658</span>– <span class="NLM_lpage">3664</span>, <span class="refDoi"> DOI: 10.1021/jm0400092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0400092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3658-3664&author=M.+A.+Ismailauthor=R.+Brunauthor=T.+Wenzlerauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Novel+Dicationic+Imidazo%5B1%2C2-a%5Dpyridines+and+5%2C6%2C7%2C8-Tetrahydro-imidazo%5B1%2C2-a%5Dpyridines+as+Antiprotozoal+Agents&doi=10.1021%2Fjm0400092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Dicationic Imidazo[1,2-a]pyridines and 5,6,7,8-Tetrahydro-imidazo[1,2-a]pyridines as Antiprotozoal Agents</span></div><div class="casAuthors">Ismail, Mohamed A.; Brun, Reto; Wenzler, Tanja; Tanious, Farial A.; Wilson, W. David; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3658-3664</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetate or hydrochloride salts of 2-[5-(4-amidinophenyl)-furan-2-yl]-imidazo[1,2-a]pyridine-6-carboxamidines I (R1 = H, MeO; R2 = H, Me) and of the 5,6,7,8-tetrahydro analog (R1 = R2 = H) were synthesized from imidazo[1,2-a]pyridine-6-carbonitrile II via its bis-O-acetoxyamidoxime.  II were obtained in four steps starting with two successive brominations of 2-acetylfuran first with N-bromosuccinimide, and second with bromine to form α-bromo-2-acetyl-5-bromofuran in a moderate yield.  The condensation reaction of the latter with 6-aminonicotinonitrile or 6-amino-5-methylnicotinonitrile followed by Suzuki coupling with 4-cyanophenylboronic acid gave II in good yields.  The analogs with 1,4-phenylene and 1,3-phenylene bridges instead of 2,5-furanyl bridge were prepd. similarly.  By these approaches eight new diamidines and four potential prodrugs were prepd.  All of the diamidines showed strong DNA affinities as judged by high ΔTm values.  Six of the eight diamidines gave in vitro IC50 values of 63 nM or less vs T. b. rhodesiense with two exhibiting values of 6 nM and 1 nM.  Also, six of the eight diamidines gave in vitro IC50 values of 88 nM or less vs P. falciparum with two exhibiting values of 14 nM.  Excellent in vivo activity in the trypanosomal STIB900 mouse model was found for five of the diamidines on i.p. dosage; these compds. gave 4/4 cures in this model.  The oral activity of the prodrugs was modest with only one showing 2/4 cures in the same mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL3SdrHukHxbVg90H21EOLACvtfcHk0lhtWFiiH4SWrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejurs%253D&md5=eb506ce99fd11e93837b1d5155422973</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm0400092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0400092%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DWenzler%26aufirst%3DT.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DNovel%2520Dicationic%2520Imidazo%255B1%252C2-a%255Dpyridines%2520and%25205%252C6%252C7%252C8-Tetrahydro-imidazo%255B1%252C2-a%255Dpyridines%2520as%2520Antiprotozoal%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3658%26epage%3D3664%26doi%3D10.1021%2Fjm0400092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Structural Selectivity of Aromatic Diamidines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5729</span>– <span class="NLM_lpage">5742</span>, <span class="refDoi"> DOI: 10.1021/jm049491e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049491e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5729-5742&author=J.%0AB.+Chairesauthor=J.+Renauthor=D.+Hamelbergauthor=A.+Kumarauthor=V.+Pandyaauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Structural+Selectivity+of+Aromatic+Diamidines&doi=10.1021%2Fjm049491e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Selectivity of Aromatic Diamidines</span></div><div class="casAuthors">Chaires, Jonathan B.; Ren, Jinsong; Hamelberg, Donald; Kumar, Arvind; Pandya, Vandna; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5729-5742</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition dialysis was used to study the interactions of 13 substituted arom. diamidine compds. with 13 nucleic acid structures and sequences.  The results show a striking selectivity of these compds. for the triplex structure poly dA:(poly dT)2, a novel aspect of their interaction with nucleic acids not previously described.  The triplex selectivity of selected compds. was confirmed by thermal denaturation studies.  Triplex selectivity was found to be modulated by the location of amidine substituents on the core phenyl-furan-Ph ring scaffold.  Mol. models were constructed to rationalize the triplex selectivity of DB359, the most selective compd. in the series.  Its triplex selectivity was found to arise from optimal ring stacking on base triplets, along with proper positioning of its amidine substituents to occupy the minor and the major-minor grooves of the triplex.  New insights into the mol. recognition of nucleic acid structures emerged from these studies, adding to the list of available design principles for selectively targeting DNA and RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-gzBJdMjo7Vg90H21EOLACvtfcHk0lhtWFiiH4SWrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D&md5=303d146fba1b5e650230142c955ad530</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm049491e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049491e%26sid%3Dliteratum%253Aachs%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPandya%26aufirst%3DV.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DStructural%2520Selectivity%2520of%2520Aromatic%2520Diamidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5729%26epage%3D5742%26doi%3D10.1021%2Fjm049491e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Antiprotozoal Activity of Aza-Analogues of Furamidine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4761</span>– <span class="NLM_lpage">4769</span>, <span class="refDoi"> DOI: 10.1021/jm0302602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0302602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4761-4769&author=M.%0AA.+Ismailauthor=R.+Brunauthor=J.+D.+Easterbrookauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Synthesis+and+Antiprotozoal+Activity+of+Aza-Analogues+of+Furamidine&doi=10.1021%2Fjm0302602"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm0302602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0302602%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DEasterbrook%26aufirst%3DJ.%2BD.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520and%2520Antiprotozoal%2520Activity%2520of%2520Aza-Analogues%2520of%2520Furamidine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4761%26epage%3D4769%26doi%3D10.1021%2Fjm0302602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Palladium catalyzed reactions for the synthesis of 2,5-disubstituted furans</span>. <i>Heterocycl. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1515/hc.1999.5.4.301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1515%2Fhc.1999.5.4.301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK1MXmtl2ku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=301-304&author=A.+Kumarauthor=C.+E.+Stephensauthor=D.+W.+Boykin&title=Palladium+catalyzed+reactions+for+the+synthesis+of+2%2C5-disubstituted+furans&doi=10.1515%2Fhc.1999.5.4.301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium catalyzed cross-coupling reactions for the synthesis of 2,5-disubstituted furans</span></div><div class="casAuthors">Kumar, Arvind; Stephens, Chad E.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Heterocyclic Communications</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-304</span>CODEN:
                <span class="NLM_cas:coden">HCOMEX</span>;
        ISSN:<span class="NLM_cas:issn">0793-0283</span>.
    
            (<span class="NLM_cas:orgname">Freund Publishing House Ltd.</span>)
        </div><div class="casAbstract">A convenient synthesis of 2,5-bisarylfurans, 2,5-bisbenzylfurans and 2,5-bisbenzoylfurans employing 2,5-bis(tri-n-butylstannyl)furan (I) and palladium catalyzed cross-coupling reactions is described.  Thus, reaction of I with RC6H4X (R = 4-cyano, 3-cyano, 4-O2N, 3-O2N, 4-EtO2C; X = Br, iodo) in dioxane contg. Pd(PPh34)4 gave 70-86% diphenylfurans II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXNaaVdNF6ObVg90H21EOLACvtfcHk0lhtWFiiH4SWrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtl2ku7w%253D&md5=43aa5c35f5cbd48c882c23c47ec32125</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1515%2Fhc.1999.5.4.301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhc.1999.5.4.301%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DPalladium%2520catalyzed%2520reactions%2520for%2520the%2520synthesis%2520of%25202%252C5-disubstituted%2520furans%26jtitle%3DHeterocycl.%2520Commun.%26date%3D1999%26volume%3D5%26spage%3D301%26epage%3D304%26doi%3D10.1515%2Fhc.1999.5.4.301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweat, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Stille cross-coupling reactions of 2-(tributylstannyl)- and 2,5-bis(tributylstannyl)tellurophene</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3214</span>– <span class="NLM_lpage">3218</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=3214-3218&author=D.+P.+Sweatauthor=C.+E.+Stephens&title=Synthesis+and+Stille+cross-coupling+reactions+of+2-%28tributylstannyl%29-+and+2%2C5-bis%28tributylstannyl%29tellurophene"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSweat%26aufirst%3DD.%2BP.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26atitle%3DSynthesis%2520and%2520Stille%2520cross-coupling%2520reactions%2520of%25202-%2528tributylstannyl%2529-%2520and%25202%252C5-bis%2528tributylstannyl%2529tellurophene%26jtitle%3DSynthesis%26date%3D2009%26volume%3D5%26spage%3D3214%26epage%3D3218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buu-Hoi, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuong, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquignon, P.</span></span> <span> </span><span class="NLM_article-title">The Pfitzinger reaction in the synthesis of quinoline derivatives</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1953</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1209</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jo50015a019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo50015a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaG2MXhtlCq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1953&pages=1209-1224&author=N.+P.+Buu-Hoiauthor=R.+Royerauthor=N.+D.+Xuongauthor=P.+Jacquignon&title=The+Pfitzinger+reaction+in+the+synthesis+of+quinoline+derivatives&doi=10.1021%2Fjo50015a019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfitzinger reaction in the synthesis of quinoline derivatives</span></div><div class="casAuthors">Buu-Hoi, Ng. Ph.; Royer, R.; Xuong, Ng. D.; Jacquignon, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1953</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1209-24</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">A no. of substituted cinchoninic acids and quinolines are prepd. by the Pfitzinger reaction by condensation of isatin (I) and substituted I with ketones, RCOCH2R', in an alk. medium.  Refluxing 6 g. PrCOC6H13, 5.9 g. I, and 6.9 g. KOH in 25 cc.  EtOH 24 h., distg. off the EtOH, taking up the residue in H2O, and acidifying the washed (ether) aq. soln. with AcOH give 6 g. of a mixt. of cinchoninic acids from which, on repeated crystn., is isolated a product that seems to be 3-ethyl-2-hexylcinchoninic acid (II), needles, m. 108-9°.  5-Bromoisatin (III) in lieu of I gives the 6-Br analog of II, needles, m. 162-3°.  Refluxing 2 g. III, 2 g. 2-methylcyclohexanone, and 2 g. KOH in EtOH gives 80% 7-bromo-4-methyl-1,2,3,4-tetrahydroacridine-9-carboxylic acid (IV), yellowish needles, m. 272-3° (decompn.), decarboxylated to the corresponding acridine (V) (picrate, yellow prisms, m. 196-7°).  7-Cl analog of IV, silky needles, m. 224-5°; 4-Et homolog of IV, yellowish needles, m. 298-9°; 4-Et homolog of V, slender pale yellow prisms, m. 92°.  Refluxing 2 g. β-naphthisatin (3H-benz[e]indole-1,2-dione), 2 g. MeCOC6H13 and 1.7 g. KOH in 10 cc. EtOH 24 h. gives 2.4 g. 2-hexyl-5,6-benzocinchoninic acid (VI), pale yellow needles, m. 245-6° (decompn.), decarboxylated to 2-hexyl-5,6-benzoquinoline (picrate, silky yellow needles, m. 147°).  VI is also obtained from 2-C10H7NH2 (VII), C6H13CHO, and MeCOCO2H (VIII).  2-C9H19 homolog of VI, m. 245-7°, is also obtained from VII, C9H19CHO, and VIII.  6-Chloro-2-phenylcinchoninic acid (IX), prepd. in 90% yield from 5-chloroisatin, PhAc, and KOH, prisms, m. 282°; 6-chloro-2-phenylquinoline, silky needles, m. 110-11° (picrate, m. 189-90°); 2-p-tolyl homolog of IX, needles, m. 270-1°; 6-methyl-2-(p-chlorophenyl)cinchoninic acid, silky prisms, m. 219-20°; 6-methyl-2-(p-chlorophenyl)quinoline, silky needles, m. 164° (picrate, m. 185-6°); 3,6-dimethyl-2-(p-chlorophenyl)cinchoninic acid, needles, does not m. below 320°. p-PhCH2COC6H4SPh, prepd. in 75% yield from BzCH2Cl and AlCl3 in CS2, shiny leaflets, m. 105-6%; 4-iso-PrCOC6H4SPh, mobile oil, b12 224-6°, nD23 1.6234; 4-caproyl homolog, pale yellow oil, b0.9 204-5°, nD23.5 1.6062, leaflets, m. 29-30°; 4-octanoyl homolog, pale yellow oil, b2 219-20°, nD23 1.5934, m. 31-2°; 4-Bz analog, slender prisms, m. 70°; 4-(p-MeOC6H4CO) analog, silky needles, m. 81°.  Friedel-Crafts acylations of PhSEt according to Auwers and Arndt (C.A. 3, 2577) gives 4-EtCOC6H4SEt, b15 178-80°, large prisms, m. 45°; 4-PrCO homolog, b15 189-90°, large plates, m. 43°.  The substituted cinchoninic acids (X) and quinolines (XI) prepd. by these methods are listed [substituent, m.p. (°C.) of X, b.p. (°C.) of XI (picrate m.p.) given]: 2-Undecyl, 129-30, 233/14 (83-4); 2-Undecyl-4-hydroxy, -, 270-80/14 (-); 8-Methyl-2-undecyl, 83-4, 235-8/14 (76); 2-Amyl, 135, - (-); 2-Amyl-4-hydroxy, -, m. 128 (-); 6-Methyl-2-amyl, 174, 300-2 (188-9); 6-Bromo-2-amyl, 180, 207-8/14 (192); 6-Methyl-2-hexyl, 135, - (-); 6-Chloro-2-hexyl, 143, - (-); 3-Methyl-2-hexyl, 197, - (-); 3-Methyl-2-pentadecyl, -, m. 55 (105-6); 3-Ethyl-2-iso-Bu, 239, 298-300 (160-1); 3-Propyl-2-iso-Bu, m. 175, resolidifies, m. 205, - (-); 6-Bromo-3-methyl-2-pentadecyl, 189, m. 54-5 (114); 2-Cyclopropyl, 206, 145-8/17 (175); 8-Methyl-2-cyclopropyl, 212b, - (-); 6-Chloro-2-cyclopropyl, 230b, - (-); 6-Bromo-2-cyclopropyl, 268-70b, - (-); 6-Methyl-2,3-pentamethylene, 308b, m. 105 (189-90); 6-Bromo-2,3-pentamethylene, 309, m. 128 (200-1); 6-Chloro-2,3-pentamethylene, 251,m. 109 (191-2); b = decompn.  The 2-arylcinchoninic acids (XII) (XIIa, R = CO2H, R'' = H), their corresponding quinolines (XIII) (XIIa, R = R'' = H), the 2-aryl-6-bromo-cinchoninic acids (XIV) (XIIa, R = CO2H, R'' = Br), and their corresponding quinolines (XV) (XIIa, R = H, R'' = Br) are listed [Ar, R', m.p. XII, m.p. XIII (picrate m.p.), m.p. XIV, m.p. XV (picrate m.p.) given]: 4-PhC6H4, H, -, - (-), 306, 222 (224-5); 4-PhC6H4, Me, 303-4, 121 (230), 316-17, 196 (-); 4-PhC6H4, Et, > 310, 100 (211), 292, 172 (215); 2-C10H11c, Et, -, - (-), 310, - (-); 6-C10H11c, H, -, - (-), 235, 133 (212); 6-C10H11c Me, 254-5, 70 (-), 316, 123 (216); 6-C10H11c, Et, > 300a b16 275-80 (196-7), -, 97 (217-18); 2-Ph, Me, -, - (-), 328, 129 (-); 4-Tolyl, Me, > 305, 59 (220), -, - (-); 4-Tolyl, Et, > 310, - (-), -, - (-); 3,4-Me2C6H3, H, 209-10, 90-1 (-), 227, 135 (206-7); 2,5-Me2C6H3, H, 200-1, 60 (179), 239, 124 (205); 2,5-Me2C6H3, Me, 301, - (-), 345, 109 (192); 2,4-Me2C6H3, H, -, - (-), 235, 98 (194); 2,4-Me2C6H3, Me, > 305, - (-), > 321, 122 (209);  4-EtC6H4, H, 190, 50 (196-7), -, - (-); 4-EtC6H4, Me, 290-2a b16 220-5 (191-2), 301, 115 (-); 4-EtC6H4, Et, 296, b16 228-30 (182-3), -, - (-); 4-Iso-PrC6H4, H, 195, 50 (190), 218, 143 (194); 4-Iso-PrC6H4, Me, 297, 61 (193), 298-9, 130 (185); 4-Iso-PrC6H4, Et, 267, - (-), 275-6, 93 (204); 4-Me3CC6H4, Me, 298, 66b (-), >311, 132 (154); 2,4,5-Me3C6H2, Me, 313, b13 230 (212-13), -, (-); 4,1-MeC10H6, Me, >305, b13 270-2 (224), >307, 181 (228); 5-Acenaphthenyl, Me, -, - (-), >310, - (-); 3-Pyrenyl, H, -, - (-), >310, 205 (240-2); 3-Pyrenyl, Me, >310, - (-), >310, - (-); 3-Pyrenyl, Et, >310, - (-), >310, - (-); 4-ClC6H4, H, 241-2, 112 (192-3), 271, 177 (-); 4-ClC6H4, Me, 310-11, 92 (-), 307, 160 (206-7); 4-ClC6H4, Et, 310, 42 (181), >310, 147 (196); 4-ClC6H4, Ph, -, - (-), >328, 161 (194); 3,4-Cl2C6H3, H, 258, 112 (204), 252, 169 (192); 4-BrC6H4, H, 250, 123-4 (181-2), 259, 188 (207); 4-BrC6H4, Me, >320, 108 (195-6), 325, 189-90 (163);  4-BrC6H4, Et, 329-30, 62 (192), >310, 110 (-); 4-BrC6H4, CH2CO2H, -, - (-), 307, - (-); 4-BrC6H4, Ph, -, - (-), >310, 171-2 (192-3); 4-IC6H4, H, -, - (-), 277-8, 185 (197-8); 2,4-(MeO)2C6H3, Ph, -, - (-), 297-8, 89 (202-3); 3,4-(MeO)2C6H3, Et, -, - (-), 240, - (-); a.  Decompn. b. B. 250-60°/15 mm. c.  Tetrahydronaphthyl. Ar, R, m.p. XII, m.p. XIII (picrate m.p.) given: 4-EtOC6H4, H, 222, 127 (218); 4-EtOC6H4, Me, 302, 92 (177); 4-EtOC6H4, Et, 305, oil (190); 4-EtOC6H4, Ph, 295, 100 (192); 4-MeSC6H4, Me, 305, 86 (180); 4-Me-SC6H4, CH2CO2H, 325, - (-); 4-EtSC6H4, H, 195, - (-); 4-EtSC6H4, Me, 278, 70 (186); 4-EtSC6H4, Et, 269, oil (184); 4-PhOC6H4, H, 203, 121 (-); 4-PhOC6H4, Me, 303, 91 (-); 4-PhOC6H4, Et, 282, 75 (188); 4-PhOC6H4, Ph, 283, 109a (201); 4-PhSC6H4, H, 215, 119 (-); 4-PhSC6H4, Me, 275, 76 (175); 4-PhSC6H4, Ph, 289, 62 (192); 4-(4-MeOC6H4)C6H4, H, 237, 219 (202); 4-(4-MeOC6H4)-C6H4, Me, 295, 149 (180); a. HBr salt, m. 215°.  The following 6-bromo-2-thienylcinchoninic acids substituted in the thiophene ring (XVI) (R'' = CO2H) and 6-bromo-2-(2-thienyl)quinolines (XVI, R'' = H) (a) are prepd. (R, R',m.p. given): H, H, 271-2°, a, yellowish needles, m. 124°; H, Me, 290-1°, a, m. 90°; Cl, H, 270-1°; Cl, Me, 278-9°; Cl, Et, 281-2°; Br, H, 247-8°, a, m. 175-6°; Br, Me, 284-5°, a, yellow needles, m. 147-8°; Br, Et, 260-1°; Br, Ph, 274-5°.  6-Bromo-2,5-dimethyl-3(2-thienyl)cinchoninic acid m. 234-5°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCnySqY_eXsrVg90H21EOLACvtfcHk0lhwtMNXD2dWNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2MXhtlCq&md5=c75a824431b320cac72314e20847fa2a</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjo50015a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo50015a019%26sid%3Dliteratum%253Aachs%26aulast%3DBuu-Hoi%26aufirst%3DN.%2BP.%26aulast%3DRoyer%26aufirst%3DR.%26aulast%3DXuong%26aufirst%3DN.%2BD.%26aulast%3DJacquignon%26aufirst%3DP.%26atitle%3DThe%2520Pfitzinger%2520reaction%2520in%2520the%2520synthesis%2520of%2520quinoline%2520derivatives%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1953%26volume%3D18%26spage%3D1209%26epage%3D1224%26doi%3D10.1021%2Fjo50015a019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalluraya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreenivasa, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological properties of some quinoline derivatives</span>. <i>Il Farmaco</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1016/s0014-827x(98)00037-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fs0014-827x%2898%2900037-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1998&pages=399-404&author=B.+Kallurayaauthor=S.+Sreenivasa&title=Synthesis+and+Pharmacological+properties+of+some+quinoline+derivatives&doi=10.1016%2Fs0014-827x%2898%2900037-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fs0014-827x%2898%2900037-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0014-827x%252898%252900037-8%26sid%3Dliteratum%253Aachs%26aulast%3DKalluraya%26aufirst%3DB.%26aulast%3DSreenivasa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520properties%2520of%2520some%2520quinoline%2520derivatives%26jtitle%3DIl%2520Farmaco%26date%3D1998%26volume%3D53%26spage%3D399%26epage%3D404%26doi%3D10.1016%2Fs0014-827x%2898%2900037-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajdačić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senerovic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vranić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pekmezovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arsic-Arsnijevic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselinovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselinovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šolaja, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikodinovic-Runic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opsenica, I. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of thiophene-based quanylhydrazones(iminoguanidines) efficient against panel of voriconazole-resistant fungal isolates</span>. <i>Bioorg. & Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1291</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bmc.2016.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=26867487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1antrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1277-1291&author=V.+Ajda%C4%8Di%C4%87author=L.+Senerovicauthor=M.+Vrani%C4%87author=M.+Pekmezovicauthor=V.+Arsic-Arsnijevicauthor=A.+Veselinovicauthor=J.+Veselinovicauthor=B.+A.+%C5%A0olajaauthor=J.+Nikodinovic-Runicauthor=I.+M.+Opsenica&title=Synthesis+and+evaluation+of+thiophene-based+quanylhydrazones%28iminoguanidines%29+efficient+against+panel+of+voriconazole-resistant+fungal+isolates&doi=10.1016%2Fj.bmc.2016.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of thiophene-based guanylhydrazones (iminoguanidines) efficient against panel of voriconazole-resistant fungal isolates</span></div><div class="casAuthors">Ajdacic, Vladimir; Senerovic, Lidija; Vranic, Marija; Pekmezovic, Marina; Arsic-Arsnijevic, Valentina; Veselinovic, Aleksandar; Veselinovic, Jovana; Solaja, Bogdan A.; Nikodinovic-Runic, Jasmina; Opsenica, Igor M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1277-1291</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of new thiophene-based guanylhydrazones (iminoguanidines) was synthesized in high yields using a straightforward two-step procedure.  The antifungal activity of the compds. was evaluated against a wide range of medically important fungal strains, including yeasts, molds, and dermatophytes, in comparison to the clin. used drug voriconazole.  Cytotoxic properties of the compds. were also detd. using a human lung fibroblast cell line and a hemolysis assay.  All guanylhydrazones showed significant activity against a broad spectrum of clin. important species of Candida spp., Aspergillus fumigatus, Fusarium oxysporum, Microsporum canis and Trichophyton mentagrophytes that was in some cases comparable or better than the activity of voriconazole.  More importantly, compds. 10, 11, 13, 14, 18 and 21 exhibited excellent activity against voriconazole-resistant Candida albicans CA5 with very low minimal inhibitory concn. (MIC) values <2 μg mL-1.  Deriv. 14, bearing bromine on the Ph ring, was the most effective compd. with MICs ranging from 0.25 to 6.25 μg mL-1.  However, bis-guanylhydrazone 18 showed better selectivity in terms of therapeutic index values.  In vivo embryotoxicity on zebrafish (Danio rerio) showed an improved toxicity profile of 11, 14 and 18 in comparison to that of voriconazole.  Most guanylhydrazones also inhibited C. albicans yeast to hyphal transition, essential for its biofilm formation, while 11 and 18 were able to disperse preformed Candida biofilms.  All guanylhydrazones showed the equal potential to interact with genomic DNA of C. albicans in vitro, thus indicating a possible mechanism of their action, as well as possible mechanism of obsd. cytotoxic effects.  Tested compds. did not have significant hemolytic effect and caused low liposome leakage, which excluded the cell membrane as a primary target.  On the basis of computational docking expts. using both human and cytochrome P 450 from Candida, it was concluded that the most active guanylhydrazones had minimal structural prerequisites to interact with cytochrome P 450 14α-demethylase (CYP51).  Promising guanylhydrazone derivs. also showed satisfactory pharmacokinetic profiles based on mol. calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbvwK5BWnR3rVg90H21EOLACvtfcHk0lhwtMNXD2dWNQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1antrc%253D&md5=e0605f4cc2f1e07fe613b733b10c3af5</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DAjda%25C4%258Di%25C4%2587%26aufirst%3DV.%26aulast%3DSenerovic%26aufirst%3DL.%26aulast%3DVrani%25C4%2587%26aufirst%3DM.%26aulast%3DPekmezovic%26aufirst%3DM.%26aulast%3DArsic-Arsnijevic%26aufirst%3DV.%26aulast%3DVeselinovic%26aufirst%3DA.%26aulast%3DVeselinovic%26aufirst%3DJ.%26aulast%3D%25C5%25A0olaja%26aufirst%3DB.%2BA.%26aulast%3DNikodinovic-Runic%26aufirst%3DJ.%26aulast%3DOpsenica%26aufirst%3DI.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520thiophene-based%2520quanylhydrazones%2528iminoguanidines%2529%2520efficient%2520against%2520panel%2520of%2520voriconazole-resistant%2520fungal%2520isolates%26jtitle%3DBioorg.%2520%2526%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1277%26epage%3D1291%26doi%3D10.1016%2Fj.bmc.2016.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">The Thiophene ″Sigma-Hole″ as a Concept for Preorganized, Specific Recognition of G·C Base Pairs in the DNA Minor Groove</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">15404</span>– <span class="NLM_lpage">15412</span>, <span class="refDoi"> DOI: 10.1002/chem.201603422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fchem.201603422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27624927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2rur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=15404-15412&author=P.+Guoauthor=A.+Paulauthor=A.+Kumarauthor=A.+A.+Farahatauthor=D.+Kumarauthor=S.+Wangauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=The+Thiophene+%E2%80%B3Sigma-Hole%E2%80%B3+as+a+Concept+for+Preorganized%2C+Specific+Recognition+of+G%C2%B7C+Base+Pairs+in+the+DNA+Minor+Groove&doi=10.1002%2Fchem.201603422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The Thiophene "Sigma-Hole" as a Concept for Preorganized, Specific Recognition of G-C Base Pairs in the DNA Minor Groove</span></div><div class="casAuthors">Guo, Pu; Paul, Ananya; Kumar, Arvind; Farahat, Abdelbasset A.; Kumar, Dhiraj; Wang, Siming; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15404-15412</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In spite of its importance in cell function, targeting DNA is under-represented in the design of small mols.  A barrier to progress in this area is the lack of a variety of modules that recognize G-C base pairs (bp) in DNA sequences.  To overcome this barrier, an entirely new design concept for modules that can bind to mixed G-C and A-T sequences of DNA is reported herein.  Because of their successes in biol. applications, minor-groove-binding heterocyclic cations were selected as the platform for design.  Binding to A-T sequences requires hydrogen-bond donors whereas recognition of the G-NH2 requires an acceptor.  The concept that we report herein uses pre-organized N-methylbenzimidazole (N-MeBI) thiophene modules for selective binding with mixed bp DNA sequences.  The interaction between the thiophene sigma hole (pos. electrostatic potential) and the electron-donor nitrogen of N-MeBI preorganizes the conformation for accepting an hydrogen bond from G-NH2.  The compd.-DNA interactions were evaluated with a powerful array of biophys. methods and the results show that N-MeBI-thiophene monomer compds. can strongly and selectively recognize single G-C bp sequences.  Replacing the thiophene with other moieties significantly reduces binding affinity and specificity, as predicted by the design concept.  These results show that the use of mol. features, such as sigma-holes, can lead to new approaches for small mols. in biomol. interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoqg6sKBdEcLVg90H21EOLACvtfcHk0lgNFGNE1vaTYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2rur%252FE&md5=4563d0e90f84c69fff74ae9922dba2a6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fchem.201603422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201603422%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DP.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThe%2520Thiophene%2520%25E2%2580%25B3Sigma-Hole%25E2%2580%25B3%2520as%2520a%2520Concept%2520for%2520Preorganized%252C%2520Specific%2520Recognition%2520of%2520G%25C2%25B7C%2520Base%2520Pairs%2520in%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D15404%26epage%3D15412%26doi%3D10.1002%2Fchem.201603422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Racané, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tralić-Kulenović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavelić, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratkaj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavelić, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karminski-Zamola, G.</span></span> <span> </span><span class="NLM_article-title">Novel Diamidino-Substituted Derivatives of Phenyl Benzothiazolyl and Dibenzothiazolyl Furans and Thiophenes: Synthesis, Antiproliferative and DNA Binding Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2418</span>– <span class="NLM_lpage">2432</span>, <span class="refDoi"> DOI: 10.1021/jm901441b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901441b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFyhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2418-2432&author=L.+Racan%C3%A9author=V.+Trali%C4%87-Kulenovi%C4%87author=S.+K.+Paveli%C4%87author=I.+Ratkajauthor=P.+Peixotoauthor=R.+Nhiliauthor=S.+Depauwauthor=M.+P.+Hildebrandauthor=M.+H.+David-Cordonnierauthor=K.+Paveli%C4%87author=G.+Karminski-Zamola&title=Novel+Diamidino-Substituted+Derivatives+of+Phenyl+Benzothiazolyl+and+Dibenzothiazolyl+Furans+and+Thiophenes%3A+Synthesis%2C+Antiproliferative+and+DNA+Binding+Properties&doi=10.1021%2Fjm901441b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Diamidino-Substituted Derivatives of Phenyl Benzothiazolyl and Dibenzothiazolyl Furans and Thiophenes: Synthesis, Antiproliferative and DNA Binding Properties</span></div><div class="casAuthors">Racane, Livio; Tralic-Kulenovic, Vesna; Kraljevic Pavelic, Sandra; Ratkaj, Ivana; Peixoto, Paul; Nhili, Raja; Depauw, Sabine; Hildebrand, Marie-Paule; David-Cordonnier, Marie-Helene; Pavelic, Kresimir; Karminski-Zamola, Grace</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2418-2432</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new diamidino-, diisopropylamidino-, and diimidazolinyl-substituted derivs. of Ph benzothiazolyl and dibenzothiazolyl furans and thiophenes were successfully prepd. and evaluated for their antiproliferative activity on tumor cell lines in vitro, DNA binding propensity, and sequence selectivity as well as cellular distribution.  A strong antiproliferative effect of the tested compds. was obsd. on all tested cell lines in a concn.-dependent response pattern.  In general, imidazolinyl-substituted derivs. and/or the thiophene core were in correlation with increased antiproliferative activity.  Two compds. I (Am = iso-Pr carboxyimidamide or 4,5-dihydro-1H-imidazol-2-yl) were chosen for biol. studies due to their differential antiproliferative properties.  The DNA binding properties of this new series of compds. were assessed and evidenced their efficient minor groove binding properties with preferential interaction at AT-rich sites.  Both compds. also present nuclear subcellular localization, suggesting that their cellular mode of action implies localization in the DNA compartment and direct inhibition of DNA replication and induction of apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RPFnF5AXKLVg90H21EOLACvtfcHk0lgNFGNE1vaTYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFyhtb8%253D&md5=cb3660325372379f0479e9d70865e78b</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm901441b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901441b%26sid%3Dliteratum%253Aachs%26aulast%3DRacan%25C3%25A9%26aufirst%3DL.%26aulast%3DTrali%25C4%2587-Kulenovi%25C4%2587%26aufirst%3DV.%26aulast%3DPaveli%25C4%2587%26aufirst%3DS.%2BK.%26aulast%3DRatkaj%26aufirst%3DI.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DHildebrand%26aufirst%3DM.%2BP.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DPaveli%25C4%2587%26aufirst%3DK.%26aulast%3DKarminski-Zamola%26aufirst%3DG.%26atitle%3DNovel%2520Diamidino-Substituted%2520Derivatives%2520of%2520Phenyl%2520Benzothiazolyl%2520and%2520Dibenzothiazolyl%2520Furans%2520and%2520Thiophenes%253A%2520Synthesis%252C%2520Antiproliferative%2520and%2520DNA%2520Binding%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2418%26epage%3D2432%26doi%3D10.1021%2Fjm901441b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Messaoudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léonce, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierré, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudhomme, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Cytotoxicities of 7-Aza Rebeccamycin Analogues Bearing Various Substituents on the Sugar Moiety, on the Imide Nitrogen and on the Carbazole Framework</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2005.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ejmech.2005.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15907347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVynsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2005&pages=961-971&author=S.+Messaoudiauthor=F.+Anizonauthor=S.+L%C3%A9onceauthor=A.+Pierr%C3%A9author=B.+Pfeifferauthor=M.+Prudhomme&title=Synthesis+and+Cytotoxicities+of+7-Aza+Rebeccamycin+Analogues+Bearing+Various+Substituents+on+the+Sugar+Moiety%2C+on+the+Imide+Nitrogen+and+on+the+Carbazole+Framework&doi=10.1016%2Fj.ejmech.2005.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and cytotoxicities of 7-aza rebeccamycin analogs bearing various substituents on the sugar moiety, on the imide nitrogen and on the carbazole framework</span></div><div class="casAuthors">Messaoudi, Samir; Anizon, Fabrice; Leonce, Stephane; Pierre, Alain; Pfeiffer, Bruno; Prudhomme, Michelle</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">961-971</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The synthesis of a family of rebeccamycin analogs in which one indole unit has been replaced by a 7-azaindole moiety is described.  Substitutions have been carried out on the imide nitrogen, on the carbazole framework and on the sugar part.  Compds. with a lactam upper heterocycle have also been prepd.  The cytotoxicities of the newly synthesized compds. toward four tumor cell lines, one murine leukemia (L1210) and three human tumor cell lines (prostate carcinoma DU145, colon carcinoma HT29, and non-small cell lung carcinoma A549) have been evaluated and compared to those of rebeccamycin and parent non-aza and aza compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwEv9BskZ5WrVg90H21EOLACvtfcHk0ljrRbjMqXV_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVynsbnI&md5=1a950fea7b127ffca6da496203b57fec</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2005.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2005.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DMessaoudi%26aufirst%3DS.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DL%25C3%25A9once%26aufirst%3DS.%26aulast%3DPierr%25C3%25A9%26aufirst%3DA.%26aulast%3DPfeiffer%26aufirst%3DB.%26aulast%3DPrudhomme%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520Cytotoxicities%2520of%25207-Aza%2520Rebeccamycin%2520Analogues%2520Bearing%2520Various%2520Substituents%2520on%2520the%2520Sugar%2520Moiety%252C%2520on%2520the%2520Imide%2520Nitrogen%2520and%2520on%2520the%2520Carbazole%2520Framework%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2005%26volume%3D40%26spage%3D961%26epage%3D971%26doi%3D10.1016%2Fj.ejmech.2005.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahoum, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantagrel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rilova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">New Insights on the Mechanism of Quinoline-Based DNA Methyltransferase Inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">6293</span>– <span class="NLM_lpage">6302</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.594671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1074%2Fjbc.m114.594671" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=6293-6302&author=C.+Grosauthor=L.+Fleuryauthor=V.+Nahoumauthor=C.+Fauxauthor=S.+Valenteauthor=D.+Labellaauthor=F.+Cantagrelauthor=E.+Rilovaauthor=M.+A.+Bouhlelauthor=M.-H.+David-Cordonnierauthor=I.+Dufauauthor=F.+Ausseilauthor=A.+Maiauthor=L.+Moureyauthor=L.+Lacroixauthor=P.+B.+Arimondo&title=New+Insights+on+the+Mechanism+of+Quinoline-Based+DNA+Methyltransferase+Inhibitors&doi=10.1074%2Fjbc.m114.594671"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.594671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.594671%26sid%3Dliteratum%253Aachs%26aulast%3DGros%26aufirst%3DC.%26aulast%3DFleury%26aufirst%3DL.%26aulast%3DNahoum%26aufirst%3DV.%26aulast%3DFaux%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DCantagrel%26aufirst%3DF.%26aulast%3DRilova%26aufirst%3DE.%26aulast%3DBouhlel%26aufirst%3DM.%2BA.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DDufau%26aufirst%3DI.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMourey%26aufirst%3DL.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DNew%2520Insights%2520on%2520the%2520Mechanism%2520of%2520Quinoline-Based%2520DNA%2520Methyltransferase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D6293%26epage%3D6302%26doi%3D10.1074%2Fjbc.m114.594671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Biosensor-Surface Plasmon Resonance: Quantitative Analysis of Small Molecule-Nucleic Acid Interactions</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ymeth.2006.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=17472897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=150-161&author=B.+Nguyenauthor=F.+A.+Taniousauthor=W.+D.+Wilson&title=Biosensor-Surface+Plasmon+Resonance%3A+Quantitative+Analysis+of+Small+Molecule-Nucleic+Acid+Interactions&doi=10.1016%2Fj.ymeth.2006.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-surface plasmon resonance: quantitative analysis of small molecule-nucleic acid interactions</span></div><div class="casAuthors">Nguyen, Binh; Tanious, Farial A.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Surface plasmon resonance (SPR)-biosensor techniques directly provide essential information for the study and characterization of small mol.-nucleic acid interactions, and the use of these methods is steadily increasing.  The method is label-free and monitors the interactions in real time.  Both dynamic and steady-state information can be obtained for a wide range of reaction rates and binding affinities.  This article presents the basics of the SPR technique, provides suggestions for exptl. design, and illustrates data processing and anal. of results.  A specific example of the interaction of a well-known minor groove binding agent, netropsin, with DNA is evaluated by both kinetic and steady-state SPR methods.  Three different expts. are used to illustrate different approaches and anal. methods.  The three sets of results show the reproducibility of the binding consts. and agreement from both steady-state and kinetic analyses.  These expts. also show that reliable kinetic information can be obtained, even with difficult systems, if the exptl. conditions are optimized to minimize mass transport effects.  Limitations of the biosensor-SPR technique are also discussed to provide an awareness of the care needed to conduct a successful expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIMfX4ZwyTSLVg90H21EOLACvtfcHk0ljrRbjMqXV_eQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D&md5=9455fa47e2eefef19166b544873c9102</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DBiosensor-Surface%2520Plasmon%2520Resonance%253A%2520Quantitative%2520Analysis%2520of%2520Small%2520Molecule-Nucleic%2520Acid%2520Interactions%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.ymeth.2006.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanjunda, R.</span>; <span class="NLM_string-name">Munde, M.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Real-Time Monitoring of Nucleic Acid Interactions with Biosensor-Surface Plasmon Resonance</span>. In  <i>Methods for Studying Nucleic Acid/Drug Interactions</i>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Yitzhak, T.</span>, <span class="NLM_string-name">Meni, W.</span></span>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, Florida, USA</span>, <span class="NLM_year">2011</span>; Chapter 4, pp  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">122</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1201%2Fb11691-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=91-122&author=R.+Nanjunda&author=M.+Munde&author=Y.+Liu&author=W.+D.+Wilsonauthor=T.+Yitzhak&author=W.+Meni&title=Methods+for+Studying+Nucleic+Acid%2FDrug+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1201%2Fb11691-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252Fb11691-7%26sid%3Dliteratum%253Aachs%26aulast%3DNanjunda%26aufirst%3DR.%26atitle%3DReal-Time%2520Monitoring%2520of%2520Nucleic%2520Acid%2520Interactions%2520with%2520Biosensor-Surface%2520Plasmon%2520Resonance%26btitle%3DMethods%2520for%2520Studying%2520Nucleic%2520Acid%252FDrug%2520Interactions%26aulast%3DYitzhak%26aufirst%3DT.%26pub%3DCRC%2520Press%26date%3D2011%26spage%3D91%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: the RESP model</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">10269</span>– <span class="NLM_lpage">10280</span>, <span class="refDoi"> DOI: 10.1021/j100142a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100142a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK3sXlvVyqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1993&pages=10269-10280&author=C.+I.+Baylyauthor=P.+Cieplakauthor=W.+Cornellauthor=P.+A.+Kollman&title=A+well-behaved+electrostatic+potential+based+method+using+charge+restraints+for+deriving+atomic+charges%3A+the+RESP+model&doi=10.1021%2Fj100142a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: the RESP model</span></div><div class="casAuthors">Bayly, Christopher I.; Cieplak, Piotr; Cornell, Wendy; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">10269-80</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">The authors present a new approach to generating electrostatic potential (ESP) derived charges for mols.  The major strength of electrostatic potential derived charges is that they optimally reproduce the intermol. interaction properties of mols. with a simple two-body additive potential, provided, of course, that a suitably accurate level of quantum mech. calcn. is used to derive the ESP around the mol.  Previously, the major weaknesses of these charges have been that they were not easily transferably between common functional groups in related mols., they have often been conformationally dependent, and the large charges that frequently occur can be problematic for simulating intramol. interactions.  Introducing restraints in the form of a penalty function into the fitting process considerably reduces the above problems, with only a minor decrease in the quality of the fit to the quantum mech. ESP.  Several other refinements in addn. to the restrained electrostatic potential (RESP) fit yield a general and algorithmic charge fitting procedure for generating atom-centered point charges.  This approach can thus be recommended for general use in mol. mechanics, mol. dynamics, and free energy calcns. for any org. or bioorg. system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRAz3o6S1gOrVg90H21EOLACvtfcHk0lh0AQONT3-ECw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvVyqsLs%253D&md5=e65c6a556ffc174df4f327687912a0bd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fj100142a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100142a004%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DCornell%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DA%2520well-behaved%2520electrostatic%2520potential%2520based%2520method%2520using%2520charge%2520restraints%2520for%2520deriving%2520atomic%2520charges%253A%2520the%2520RESP%2520model%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1993%26volume%3D97%26spage%3D10269%26epage%3D10280%26doi%3D10.1021%2Fj100142a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.,  III</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Madej, B.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, W.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+A.+Case&author=V.+Babin&author=J.+Berryman&author=R.+M.+Betz&author=Q.+Cai&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=H.+Gohlke&author=A.+W.+Goetz&author=S.+Gusarov&author=N.+Homeyer&author=P.+Janowski&author=J.+Kaus&author=I.+Kolossv%C3%A1ry&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=F.+Paesani&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=R.+Salomon-Ferrer&author=G.+Seabra&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=P.+A.+Kollman&title=AMBER+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graph. Model</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0lh0AQONT3-ECw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harika, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germann, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">First Structure of a Designed Minor Groove Binding Heterocyclic Cation that Specifically Recognizes Mixed DNA Base Pair Sequences</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">17612</span>– <span class="NLM_lpage">17620</span>, <span class="refDoi"> DOI: 10.1002/chem.201704563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fchem.201704563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29044822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVartrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=17612-17620&author=N.+K.+Harikaauthor=M.+W.+Germannauthor=W.+D.+Wilson&title=First+Structure+of+a+Designed+Minor+Groove+Binding+Heterocyclic+Cation+that+Specifically+Recognizes+Mixed+DNA+Base+Pair+Sequences&doi=10.1002%2Fchem.201704563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">First structure of a designed minor groove binding heterocyclic cation that specifically recognizes mixed DNA base pair sequences</span></div><div class="casAuthors">Harika, Narinder K.; Germann, Markus W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">69</span>),
    <span class="NLM_cas:pages">17612-17620</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The high-resoln. NMR structure of the 1st heterocyclic, non-amide, org. cation that strongly and selectively recognizes mixed AT/GC bp sequences of DNA in a 1:1 complex is described.  Compd. designs of this type provide essential methods for control of functional, non-genomic DNA sequences and have broad cell uptake capability, based on studies from animals to humans.  The high-resoln. structural studies described in this report are essential for understanding the mol. basis for the sequence-specific binding as well as for new ideas for addnl. compd. designs for sequence-specific recognition.  The mol. features, in this report, explain the mechanism of recognition of both A·T and G·C bps and are an interesting mol. recognition story.  Examn. of the exptl. structure and the NMR restrained mol. dynamics model suggested that recognition of the G·C base pair involves 2 specific H-bonds.  The structure illustrated a wealth of information on different DNA interactions and illustrated an interfacial water mol. that is a key component of the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMTEBch-mXLVg90H21EOLACvtfcHk0lhotomMVKCwVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVartrzI&md5=bf4a3094e2932ba703e4f0ce0de590fc</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fchem.201704563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201704563%26sid%3Dliteratum%253Aachs%26aulast%3DHarika%26aufirst%3DN.%2BK.%26aulast%3DGermann%26aufirst%3DM.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DFirst%2520Structure%2520of%2520a%2520Designed%2520Minor%2520Groove%2520Binding%2520Heterocyclic%2520Cation%2520that%2520Specifically%2520Recognizes%2520Mixed%2520DNA%2520Base%2520Pair%2520Sequences%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D17612%26epage%3D17620%26doi%3D10.1002%2Fchem.201704563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Athri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics of Water-Mediated Interactions of a Linear Benzimidazole–Biphenyl Diamidine with the DNA Minor Groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">7618</span>– <span class="NLM_lpage">7625</span>, <span class="refDoi"> DOI: 10.1021/ja809249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=7618-7625&author=P.+Athriauthor=W.+D.+Wilson&title=Molecular+Dynamics+of+Water-Mediated+Interactions+of+a+Linear+Benzimidazole%E2%80%93Biphenyl+Diamidine+with+the+DNA+Minor+Groove&doi=10.1021%2Fja809249h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics of Water-Mediated Interactions of a Linear Benzimidazole-Biphenyl Diamidine with the DNA Minor Groove</span></div><div class="casAuthors">Athri, Prashanth; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7618-7625</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DB921 has a benzimidazole-biphenyl system with terminal amidines that gives the compd. a linear conformation with a radius of curvature that does not match the DNA minor groove shape.  Surprisingly, the compd. binds in the groove with an unusually high equil. const.  X-ray crystallog. anal. of DB921 bound to -AATT- in d(CGCGAATTCGCG)2 showed that the benzimidazole is in position to directly interact with bases at the floor of the groove, while the phenylamidine of DB921 forms indirect contacts with the bases through an interfacial water.  The DB921-water pair forms a curved, flexible module with a high Ka (or a low Kd) value of binding.  To better understand the dynamics of the DB921-DNA complex and how water can be used in the design of compds. to recognize DNA, a 100 ns mol. dynamics simulation of the complex was conducted.  In addn. to the x-ray conformation, some significantly variant, dynamic conformations, which had addnl. interfacial water mols. between DB921 and DNA, appeared in the MD simulation.  The benzimidazole contacts remained relatively const. through the entire simulation.  The biphenylamidine end of the bound mol., however, undergoes much larger changes in orientation relative to the floor of the groove as well as variations in the type of water interactions.  The results provide an understanding of how water couples the linear DB921 compd. to the minor groove for tight binding, without a large unfavorable contribution to the entropy of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHDydOY5wGOrVg90H21EOLACvtfcHk0lhotomMVKCwVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWntLo%253D&md5=55099792edbdaf5e3626b311ed97615a</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fja809249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809249h%26sid%3Dliteratum%253Aachs%26aulast%3DAthri%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DMolecular%2520Dynamics%2520of%2520Water-Mediated%2520Interactions%2520of%2520a%2520Linear%2520Benzimidazole%25E2%2580%2593Biphenyl%2520Diamidine%2520with%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D7618%26epage%3D7625%26doi%3D10.1021%2Fja809249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization and multithreading</span>. <i>Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhotomMVKCwVg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%2520and%2520multithreading%26jtitle%3DComput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span> <span> </span><span class="NLM_article-title">limma powers differential expression analyses for RNA-sequencing and microarray studies</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>, <span class="NLM_elocation-id">e47</span> <span class="refDoi"> DOI: 10.1093/nar/gkv007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkv007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25605792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&author=M.+E.+Ritchieauthor=B.+Phipsonauthor=D.+Wuauthor=Y.+Huauthor=C.+W.+Lawauthor=W.+Shiauthor=G.+K.+Smyth&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies&doi=10.1093%2Fnar%2Fgkv007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">limma powers differential expression analyses for RNA-sequencing and microarray studies</span></div><div class="casAuthors">Ritchie, Matthew E.; Phipson, Belinda; Wu, Di; Hu, Yifang; Law, Charity W.; Shi, Wei; Smyth, Gordon K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e47/1-e47/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Limma is an R/Bioconductor software package that provides an integrated soln. for analyzing data from gene expression expts.  It contains rich features for handling complex exptl. designs and for information borrowing to overcome the problem of small sample sizes.  Over the past decade, limma has been a popular choice for gene discovery through differential expression analyses of microarray and high-throughput PCR data.  The package contains particularly strong facilities for reading, normalizing and exploring such data.  Recently, the capabilities of limma have been significantly expanded in two important directions.  First, the package can now perform both differential expression and differential splicing analyses of RNA sequencing (RNA-seq) data.  All the downstream anal. tools previously restricted to microarray data are now available for RNA-seq as well.  These capabilities allow users to analyze both RNA-seq and microarray data with very similar pipelines.  Second, the package is now able to go past the traditional gene-wise expression analyses in a variety of ways, analyzing expression profiles in terms of co-regulated sets of genes or in terms of higher-order expression signatures.  This provides enhanced possibilities for biol. interpretation of gene expression differences.  This article reviews the philosophy and design of the limma package, summarizing both new and historical features, with an emphasis on recent enhancements and features that have not been previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJykfsLP7n9bVg90H21EOLACvtfcHk0lgN3EEp3kg2Vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN&md5=e8a5f44160c650ad89adc48a9912bd96</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv007%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DM.%2BE.%26aulast%3DPhipson%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLaw%26aufirst%3DC.%2BW.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26atitle%3Dlimma%2520powers%2520differential%2520expression%2520analyses%2520for%2520RNA-sequencing%2520and%2520microarray%2520studies%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26doi%3D10.1093%2Fnar%2Fgkv007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Pu Guo, Abdelbasset A. Farahat, Ananya Paul, Arvind Kumar, David W. Boykin, <span class="NLM_string-name hlFld-ContribAuthor">W. David Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">Extending the σ-Hole Motif for Sequence-Specific Recognition of the DNA Minor Groove. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2020,</strong> <em>59 </em>
                                    (18)
                                     , 1756-1768. <a href="https://doi.org/10.1021/acs.biochem.0c00090" title="DOI URL">https://doi.org/10.1021/acs.biochem.0c00090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.0c00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.0c00090%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DExtending%252Bthe%252B%2525CF%252583-Hole%252BMotif%252Bfor%252BSequence-Specific%252BRecognition%252Bof%252Bthe%252BDNA%252BMinor%252BGroove%26aulast%3DGuo%26aufirst%3DPu%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31012020%26date%3D08042020%26date%3D20042020%26date%3D15042020%26volume%3D59%26issue%3D18%26spage%3D1756%26epage%3D1768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Reem K.  Arafa</span>, <span class="hlFld-ContribAuthor ">Mohamed A.  Ismail</span>, <span class="hlFld-ContribAuthor ">Tanja  Wenzler</span>, <span class="hlFld-ContribAuthor ">Reto  Brun</span>, <span class="hlFld-ContribAuthor ">Ananya  Paul</span>, <span class="hlFld-ContribAuthor ">W. David  Wilson</span>, <span class="hlFld-ContribAuthor ">Amira A.  Alakhdar</span>, <span class="hlFld-ContribAuthor ">David W.  Boykin</span>. </span><span class="cited-content_cbyCitation_article-title">New antiparasitic flexible triaryl diamidines, their prodrugs and aza analogues: Synthesis, in vitro and in vivo biological evaluation, and molecular modelling studies. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>222 </em>, 113625. <a href="https://doi.org/10.1016/j.ejmech.2021.113625" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113625</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113625%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNew%252Bantiparasitic%252Bflexible%252Btriaryl%252Bdiamidines%25252C%252Btheir%252Bprodrugs%252Band%252Baza%252Banalogues%25253A%252BSynthesis%25252C%252Bin%2525C2%2525A0vitro%252Band%252Bin%2525C2%2525A0vivo%252Bbiological%252Bevaluation%25252C%252Band%252Bmolecular%252Bmodelling%252Bstudies%26aulast%3DArafa%26aufirst%3DReem%2BK.%26date%3D2021%26volume%3D222%26spage%3D113625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fares Hezam  Al-Ostoot</span>, <span class="hlFld-ContribAuthor ">Zabiulla</span>, <span class="hlFld-ContribAuthor ">Salma  Salah</span>, <span class="hlFld-ContribAuthor ">Shaukath Ara  Khanum</span>. </span><span class="cited-content_cbyCitation_article-title">Recent investigations into synthesis and pharmacological activities of phenoxy acetamide and its derivatives (chalcone, indole and quinoline) as possible therapeutic candidates. </span><span class="cited-content_cbyCitation_journal-name">Journal of the Iranian Chemical Society</span><span> <strong>2021,</strong> <em>18 </em>
                                    (8)
                                     , 1839-1875. <a href="https://doi.org/10.1007/s13738-021-02172-5" title="DOI URL">https://doi.org/10.1007/s13738-021-02172-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s13738-021-02172-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs13738-021-02172-5%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520Iranian%2520Chemical%2520Society%26atitle%3DRecent%252Binvestigations%252Binto%252Bsynthesis%252Band%252Bpharmacological%252Bactivities%252Bof%252Bphenoxy%252Bacetamide%252Band%252Bits%252Bderivatives%252B%252528chalcone%25252C%252Bindole%252Band%252Bquinoline%252529%252Bas%252Bpossible%252Btherapeutic%252Bcandidates%26aulast%3DAl-Ostoot%26aufirst%3DFares%2BHezam%26date%3D2021%26date%3D2021%26volume%3D18%26issue%3D8%26spage%3D1839%26epage%3D1875" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Raed M.  Al‐Zoubi</span>, <span class="hlFld-ContribAuthor ">Walid K.  Al‐Jammal</span>, <span class="hlFld-ContribAuthor ">Mazhar S.  Al‐Zoubi</span>, <span class="hlFld-ContribAuthor ">Michael J.  Ferguson</span>, <span class="hlFld-ContribAuthor ">Ahmad  Zarour</span>, <span class="hlFld-ContribAuthor ">Aksam  Yassin</span>, <span class="hlFld-ContribAuthor ">Abdulla  Al‐Ansari</span>. </span><span class="cited-content_cbyCitation_article-title">Palladium‐Catalyzed Regioselective Coupling of Amidines and 1,2,3‐Triiodobenzenes: Facile Synthesis of 2,3‐Diiodinated
              N
              ‐Arylbenzimidamides as Potential MDM
              2
              and MDM
              4
              Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemistrySelect</span><span> <strong>2021,</strong> <em>6 </em>
                                    (14)
                                     , 3417-3423. <a href="https://doi.org/10.1002/slct.202100671" title="DOI URL">https://doi.org/10.1002/slct.202100671</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/slct.202100671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fslct.202100671%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistrySelect%26atitle%3DPalladium%2525E2%252580%252590Catalyzed%252BRegioselective%252BCoupling%252Bof%252BAmidines%252Band%252B1%25252C2%25252C3%2525E2%252580%252590Triiodobenzenes%25253A%252BFacile%252BSynthesis%252Bof%252B2%25252C3%2525E2%252580%252590Diiodinated%252BN%252B%2525E2%252580%252590Arylbenzimidamides%252Bas%252BPotential%252BMDM%252B2%252Band%252BMDM%252B4%252BInhibitors%26aulast%3DAl%25E2%2580%2590Zoubi%26aufirst%3DRaed%2BM.%26date%3D2021%26date%3D2021%26volume%3D6%26issue%3D14%26spage%3D3417%26epage%3D3423" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara A.  Al-Shun</span>, <span class="hlFld-ContribAuthor ">Fardous F.  El-Senduny</span>, <span class="hlFld-ContribAuthor ">Mohamed A.  Ismail</span>, <span class="hlFld-ContribAuthor ">Wael M.  El-Sayed</span>, <span class="hlFld-ContribAuthor ">Farid A.  Badria</span>, <span class="hlFld-ContribAuthor ">Magdy M.  Youssef</span>. </span><span class="cited-content_cbyCitation_article-title">Anticancer activity of new cationic arylthiophenes against hepatocellular carcinoma. </span><span class="cited-content_cbyCitation_journal-name">Life Sciences</span><span> <strong>2021,</strong> <em>269 </em>, 119028. <a href="https://doi.org/10.1016/j.lfs.2021.119028" title="DOI URL">https://doi.org/10.1016/j.lfs.2021.119028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.lfs.2021.119028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.lfs.2021.119028%26sid%3Dliteratum%253Aachs%26jtitle%3DLife%2520Sciences%26atitle%3DAnticancer%252Bactivity%252Bof%252Bnew%252Bcationic%252Barylthiophenes%252Bagainst%252Bhepatocellular%252Bcarcinoma%26aulast%3DAl-Shun%26aufirst%3DSara%2BA.%26date%3D2021%26volume%3D269%26spage%3D119028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sally  Elmogy</span>, <span class="hlFld-ContribAuthor ">Mohamed A.  Ismail</span>, <span class="hlFld-ContribAuthor ">Rabeay Y. A.  Hassan</span>, <span class="hlFld-ContribAuthor ">Ahmed  Noureldeen</span>, <span class="hlFld-ContribAuthor ">Hadeer  Darwish</span>, <span class="hlFld-ContribAuthor ">Eman  Fayad</span>, <span class="hlFld-ContribAuthor ">Fahmy  Elsaid</span>, <span class="hlFld-ContribAuthor ">Ashraf  Elsayed</span>. </span><span class="cited-content_cbyCitation_article-title">Biological Insights of Fluoroaryl-2,2′-Bichalcophene Compounds on Multi-Drug Resistant Staphylococcus aureus. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (1)
                                     , 139. <a href="https://doi.org/10.3390/molecules26010139" title="DOI URL">https://doi.org/10.3390/molecules26010139</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26010139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26010139%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DBiological%252BInsights%252Bof%252BFluoroaryl-2%25252C2%2525E2%252580%2525B2-Bichalcophene%252BCompounds%252Bon%252BMulti-Drug%252BResistant%252BStaphylococcus%252Baureus%26aulast%3DElmogy%26aufirst%3DSally%26date%3D2021%26date%3D2020%26volume%3D26%26issue%3D1%26spage%3D139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed A.  Ismail</span>, <span class="hlFld-ContribAuthor ">Mohamed H.  Abdel-Rhman</span>, <span class="hlFld-ContribAuthor ">Ghada A.  Abdelwahab</span>, <span class="hlFld-ContribAuthor ">Wafaa S.  Hamama</span>, <span class="hlFld-ContribAuthor ">Heba M.  El-Shafeai</span>, <span class="hlFld-ContribAuthor ">Wael M.  El-Sayed</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of new thienylpicolinamidine derivatives and possible mechanisms of antiproliferative activity. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2020,</strong> <em>10 </em>
                                    (67)
                                     , 41165-41176. <a href="https://doi.org/10.1039/D0RA08796C" title="DOI URL">https://doi.org/10.1039/D0RA08796C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0RA08796C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0RA08796C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSynthesis%252Bof%252Bnew%252Bthienylpicolinamidine%252Bderivatives%252Band%252Bpossible%252Bmechanisms%252Bof%252Bantiproliferative%252Bactivity%26aulast%3DIsmail%26aufirst%3DMohamed%2BA.%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D67%26spage%3D41165%26epage%3D41176" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdelbasset A.  Farahat</span>, <span class="hlFld-ContribAuthor ">Pu  Guo</span>, <span class="hlFld-ContribAuthor ">Hadir  Shoeib</span>, <span class="hlFld-ContribAuthor ">Ananya  Paul</span>, <span class="hlFld-ContribAuthor ">David W.  Boykin</span>, <span class="hlFld-ContribAuthor ">W. David  Wilson</span>. </span><span class="cited-content_cbyCitation_article-title">Small Sequence‐Sensitive Compounds for Specific Recognition of the G⋅C Base Pair in DNA Minor Groove. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2020,</strong> <em>26 </em>
                                    (20)
                                     , 4539-4551. <a href="https://doi.org/10.1002/chem.201904396" title="DOI URL">https://doi.org/10.1002/chem.201904396</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201904396&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201904396%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DSmall%252BSequence%2525E2%252580%252590Sensitive%252BCompounds%252Bfor%252BSpecific%252BRecognition%252Bof%252Bthe%252BG%2525E2%25258B%252585C%252BBase%252BPair%252Bin%252BDNA%252BMinor%252BGroove%26aulast%3DFarahat%26aufirst%3DAbdelbasset%2BA.%26date%3D2020%26date%3D2020%26volume%3D26%26issue%3D20%26spage%3D4539%26epage%3D4551" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Emily A.  Dickie</span>, <span class="hlFld-ContribAuthor ">Federica  Giordani</span>, <span class="hlFld-ContribAuthor ">Matthew K.  Gould</span>, <span class="hlFld-ContribAuthor ">Pascal  Mäser</span>, <span class="hlFld-ContribAuthor ">Christian  Burri</span>, <span class="hlFld-ContribAuthor ">Jeremy C.  Mottram</span>, <span class="hlFld-ContribAuthor ">Srinivasa P. S.  Rao</span>, <span class="hlFld-ContribAuthor ">Michael P.  Barrett</span>. </span><span class="cited-content_cbyCitation_article-title">New Drugs for Human African Trypanosomiasis: A Twenty First Century Success Story. </span><span class="cited-content_cbyCitation_journal-name">Tropical Medicine and Infectious Disease</span><span> <strong>2020,</strong> <em>5 </em>
                                    (1)
                                     , 29. <a href="https://doi.org/10.3390/tropicalmed5010029" title="DOI URL">https://doi.org/10.3390/tropicalmed5010029</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/tropicalmed5010029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Ftropicalmed5010029%26sid%3Dliteratum%253Aachs%26jtitle%3DTropical%2520Medicine%2520and%2520Infectious%2520Disease%26atitle%3DNew%252BDrugs%252Bfor%252BHuman%252BAfrican%252BTrypanosomiasis%25253A%252BA%252BTwenty%252BFirst%252BCentury%252BSuccess%252BStory%26aulast%3DDickie%26aufirst%3DEmily%2BA.%26date%3D2020%26date%3D2020%26volume%3D5%26issue%3D1%26spage%3D29" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mohamed  A. Ismail</span>, <span class="hlFld-ContribAuthor ">Wael  M. El-Sayed</span>, <span class="hlFld-ContribAuthor ">Saad  Shaaban</span>, <span class="hlFld-ContribAuthor ">Ghada  A. Abdelwahab</span>, <span class="hlFld-ContribAuthor ">Wafaa  S. Hamama</span>. </span><span class="cited-content_cbyCitation_article-title">A Review of Cationic Arylfurans and Their Isosteres: Synthesis and Biological Importance. </span><span class="cited-content_cbyCitation_journal-name">Current Organic Chemistry</span><span> <strong>2020,</strong> <em>23 </em>
                                    (24)
                                     , 2751-2782. <a href="https://doi.org/10.2174/1385272823666191029114830" title="DOI URL">https://doi.org/10.2174/1385272823666191029114830</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1385272823666191029114830&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1385272823666191029114830%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Organic%2520Chemistry%26atitle%3DA%252BReview%252Bof%252BCationic%252BArylfurans%252Band%252BTheir%252BIsosteres%25253A%252BSynthesis%252Band%252BBiological%252BImportance%26aulast%3DA.%2BIsmail%26aufirst%3DMohamed%26date%3D2020%26volume%3D23%26issue%3D24%26spage%3D2751%26epage%3D2782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lambert</span>, <span class="hlFld-ContribAuthor ">Alioui</span>, <span class="hlFld-ContribAuthor ">Jambon</span>, <span class="hlFld-ContribAuthor ">Depauw</span>, <span class="hlFld-ContribAuthor ">Van Seuningen</span>, <span class="hlFld-ContribAuthor ">David-Cordonnier</span>. </span><span class="cited-content_cbyCitation_article-title">Direct and Indirect Targeting of HOXA9 Transcription Factor in Acute Myeloid Leukemia.. </span><span class="cited-content_cbyCitation_journal-name">Cancers</span><span> <strong>2019,</strong> <em>11 </em>
                                    (6)
                                     , 837. <a href="https://doi.org/10.3390/cancers11060837" title="DOI URL">https://doi.org/10.3390/cancers11060837</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/cancers11060837&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fcancers11060837%26sid%3Dliteratum%253Aachs%26jtitle%3DCancers%26atitle%3DDirect%252Band%252BIndirect%252BTargeting%252Bof%252BHOXA9%252BTranscription%252BFactor%252Bin%252BAcute%252BMyeloid%252BLeukemia.%26aulast%3DLambert%26date%3D2019%26date%3D2019%26volume%3D11%26issue%3D6%26spage%3D837" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0017.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0017.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Reagents and Conditions: (a) (i) Pd(PPh<sub>3</sub>)<sub>4</sub>, 1,4-Dioxane, (ii) 2.0 M HCl; (b) 3,4-Diaminobenzonitrile, 1,4-Benzoquinone, EtOH; (c) NH<sub>2</sub>OH–HCl/KO-<i>t</i>-Bu, DMSO; (d) (i) AcOH/Ac<sub>2</sub>O, (ii) H<sub>2</sub>/Pd–C, AcOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Reagents and Conditions: (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Na<sub>2</sub>CO<sub>3</sub>/H<sub>2</sub>O, Dioxane, Reflux; (b) 3,4-Diaminobenzonitrile, Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>/DMSO, 140 °C; (c) (i) HCl (g), EtOH, (ii) NH<sub>3</sub> (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Reagents and Conditions; (a) Pd(PPh<sub>3</sub>)<sub>4</sub>, Dioxane; (b) NBS, DMF, rt; (c) (i) LiN(TMS)<sub>2</sub>/THF, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Reagents and Conditions; (a) (i) HCl (g), EtOH (ii) 4-Nitrophenethylamine, EtOH; (b) (i) LiN(TMS)<sub>2</sub>, THF, rt, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Reagents and Conditions: (a) 4-Amino-3-(methylamino)benzonitrile, Na<sub>2</sub>S<sub>2</sub>O<sub>3</sub>, DMF, Reflux. (b) (i) LiN(TMS)<sub>2</sub>, THF, rt (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0014.jpeg" id="rightTab-gr14" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Reagents and Conditions: (a) NaOH, H<sub>2</sub>O, 100 °C, Overnight; (b) CuCN, DMF, 140 °C (c) NBS, DMF, rt; (d) 4-Cyanophenylboronic Acid, 1,4-Dioxane, Pd(PPh<sub>3</sub>)<sub>4</sub>, 2 M K<sub>2</sub>CO<sub>3</sub>, 70 °C, Overnight; (e) (i) LiN(TMS)<sub>2</sub>, THF, rt, 48 h, (ii) HCl (g), EtOH</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0015.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0015.jpeg" id="rightTab-gr15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Reagents and Conditions: (a) 3-Amino-4-(methylamino)benzamidine, Benzoquinone, EtOH, Reflux</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0015.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. In vitro selection of <b>DB818</b> and <b>DB1055</b> as HOXA9/DNA binding inhibitors. (A) ELISA-derived protein/DNA binding inhibition assay for selection of HOXA9/DNA binding inhibitors. HOXA9 protein was expressed from reticulocyte lysate and incubated with immobilized HBS-containing sequence (CTR). Free HBS or nonspecific (NS) DNA at 50× or 100× the quantity of immobilized HBS oligonucleotide or the various indicated DB compounds (5 or 10 μM) were then added. Black, dark gray, light gray, and white bars correspond to control, >50%, between 25 and 50% or <25% of HOXA9/DNA remaining complexes. Graph represents mean ± s.e.m. of up to 15 independent measurements. (B) EMSA gel. 5′-end <sup>32</sup>P-radiolabeled HBS DNA sequence (“f”, free DNA) was incubated with reticulocyte lysate expressing HOXA9 protein (“HOXA9”) to form a complex (bound “b”) and in the presence HOXA9-directed antibody (“Ab”) to generate a super-shifted (“Ss”) DNA band, or in the presence of increasing concentrations of the indicated DB compounds (μM). (C). Quantitative analysis of the inhibition of HOXA9/DNA complex formation from EMSA defines subgroups based on their HOXA9/DNA binding inhibition activity. The percentage of the bound HOXA9/DNA complex was quantified from EMSA in panel (B), relatively to the mean of the three control samples (100%) and plotted over drug concentration.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Comparison and mode of binding to an oligonucleotide containing a HOXA9/DNA binding site. (A). Melting temperature studies on the HBS-containing DNA. Double-stranded HBS-containing oligonucleotides (0.5 μM) were incubated with 0.5 μM (<i>R</i> = 1:1) or 1 μM (<i>R</i> = 2:1) of the indicated compounds in TNE buffer prior to be subjected to the measurement of the DNA hyperchromic variation upon step by step heating using absorbance measurement at 260 nm. The Δ<i>T</i><sub>m</sub> values are calculated from <i>T</i><sub>m(DNA+DB)</sub> – <i>T</i><sub>m(DNA)</sub>. White and dashed bars highlight an increase of the <i>T</i><sub>m</sub> of more or less than 5 °C, respectively, whereas black bars are used for <b>DB828</b>, chosen as a control drug for further cellular analyses. (B) CD spectra is showing minor groove binding of duplex DNA. Double-stranded HBS-containing oligonucleotides (5 μM) were incubated in TNE buffer with increasing concentrations of the indicated compounds at increasing drug/oligonucleotide ratios <i>R</i>, evidencing a strong positively induced CD at the drug wavelength (0, 1, 2, 3, 4, 5, 6, 8, and 10 μM of drug, as indicated by arrows from min to max concentrations). Dashed lanes are CD spectra for oligonucleotides in the absence of compounds. Thick lanes highlight CD spectra at <i>R</i> = 1:1 and <i>R</i> = 2:1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Evaluation of the drug interaction to the HOXA9 consensus DNA binding site. (A) DNaseI footprinting gels. Increasing concentrations of the indicated compounds (μM) were incubated with a 3′-end <sup>32</sup>P-radiolabeled HBS-containing DNA fragment prior to DNaseI mild-digestion to then be subjected to denaturing gel electrophoresis. “G” lanes, G-track localizes guanines within the radiolabeled DNA sequence. “0” lanes, DNA fragment without the drug. The gray line locates the HOXA9-binding site (HBS). (B) Densitometric analyses from DNaseI footprinting gels. Each band from lanes generated in the presence of various concentrations of the indicated compounds was quantified regarding each identical base from the untreated lanes (“0”) to visualize the differential cleavage at the HBS sequence (gray lines) or not depending on the evaluated compounds from gels in A. (C) Melting temperature studies on ATGA- and/or AT-mutated sites from the HBS-containing DNA. Experiments were performed at an equimolar drug/DNA ratio (<i>R</i> = 1:1) in similar conditions as in <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A but with oligonucleotides mutated on the ATGA sequence (ATGAm), the AT-rich site (ATm), or both (2Mut) (sequences presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>). Statistic, Student <i>t</i>-test: ns, <i>p</i> > 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Kinetic and affinity analysis of <b>DB818</b> and <b>DB1055</b> binding to HBS and HBS-mut DNA sequences using BIAcore technology. (A) SPR sensorgrams for the evaluation of the interaction kinetics of <b>DB818</b> and <b>DB1055</b> with the HBS (“ATGATTTA”) or HBS-mut (“ATGACCTA”) DNA sequences in P20-TNE binding buffer. The black lines are best fit curves determined by kinetic fitting. The ligand concentrations are indicated in the graphs from bottom to top. The surface interaction of the diamidines at high concentrations causes the drift in the sensorgrams. (B) Steady-state fitting curves of <b>DB818</b> and <b>DB1055</b> on the HBS or HBS-mut DNA hairpin sequences. A stoichiometric value of one was used for data fitting based on the maximum RU values for the different sequences that were similar to those for netropsin binding to the AATT sequence used as chip control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. MD model of <b>DB818</b> bound to ds[(5′-CCATGATTTACG-3′) (5′-CGTAAATCATGG-3′)]. (A,B) Ball-stick (DNA) and space-filling (ligand) model viewed into the minor groove (A) and major groove (B) of the ATTTA binding site with bound <b>DB818</b>. The DNA bases are represented in the tan-white-red-blue-yellow (C–H–O–N–P) color scheme and <b>DB818</b> is represented in the light purple-white-blue-yellow (C–H–N–S) color scheme. (C,D) Important interactions between different sections of the <b>DB818</b>/DNA complex are presented for the phenyl-amidine (C) and benzimidazole-amidine (D) moieties. Interactions with the indicated bases of the HBS sequence are schematized by yellow dashed lines and the corresponding distances are indicated above (in angstrom).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Cellular analysis of HOXA9 transcription factor inhibition by <b>DB818</b> and <b>DB1055</b>. (A) Subcellular distribution of <b>DB818</b>, <b>DB1055</b>, and <b>DB828</b> using cell fluorescence microscopy. GFP-positive MigA9 cells (in green) were treated with 20 μM of the indicated compounds (in blue) for 2 h. The nucleus was counterstained using TO-PRO-3 or propidium iodide (PI) staining of DNA (in red). Merged images are presented on the right. Scale bars correspond to 5 μm. <sup>1</sup>: Images acquired with a confocal Zeiss LSM880 microscope. GFP (Ex 488 nm, Em 495–551 nm), TO-PRO-3 (Ex 633 nm, Em 640–735 nm), <b>DB818</b> (Ex 405 nm, Em 445–485 nm), <b>DB828</b> (Ex 405 nm, Em 425–465 nm). <sup>2</sup>: Images acquired with a widefield Leica DMi8 microscope. GFP (Ex BP480/40, Em BP527/30), PI (Ex BP546/10, Em BP585/40), DB (Ex BP350/50, Em BP460/50). (B) Inhibition of HOXA9 transcriptional control by <b>DB818</b> and <b>DB1055</b> evidenced by luciferase assays. K562 cells were co-transfected with pGL3-prom reporter vector containing 6-repeated HBS sequences as WT sequence (“HBS”) or mutated sequence (“HBSmut”) ± pCMV-AD-HOXA9 (“HOXA9”) prior to addition of the indicated DB compounds or vehicle. Graphs represent mean ± s.e.m. of 3 to 8 independent points. Statistic, Student <i>t</i>-test: *, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Transcriptomic analysis of MigA9 treated with 2.5 μM of <b>DB818</b> for 6, 12, or 24 h. (A) Commonly deregulated genes with |mean fold changes| ≥ 1.5 are presented as heatmap for upregulated genes (top panel) or downregulated genes (bottom panel). Arrows identified genes up- or downregulated during hematopoietic differentiation from progenitors to granulocyte/monocyte differentiated cells as presented in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Figure S7</a>. (B) GSEA analyses of upregulated or downregulated gene expression data on MSigDB C2-chemical and genetic perturbation genesets. (C) Cellular networks (“diseases and functions” analysis in hematological systems) deduced from IPA of genes that were significantly deregulated (adjusted <i>p</i>-values ≤ 0.05) by <b>DB818</b> treatment of MigA9 cells with |fold-changes| ≥ 1.5. <i>Z</i>-score, the number of genes (“# genes”), their names, and the corresponding up (north east arrow) or down (south east arrow) expression changes are indicated. (D) Quantification of deregulated genes using qRT-PCR on total mRNA from MigA9 cells treated with <b>DB818</b>, <b>DB1055</b>, or <b>DB828</b> (5 or 10 μM) for 48 h, as 6 independent replicates. Each gene expression was normalized relatively to that of Gapdh (see primers in <a href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf" class="ext-link">Table S1</a>) and given as mean ± s.e.m. Statistics (Student <i>t</i>-test): *, <i>p</i> < 0.05; **, <i>p</i> < 0.01; ***, <i>p</i> < 0.001; ****, <i>p</i> < 0.0001.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/medium/jm-2018-01448v_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Evaluation of <b>DB818</b> and <b>DB1055</b> inhibition of HOXA9 binding to murine Akap13 gene regulatory region. (A) DNA sequence of the DE-651 peak, located at position chr7:82861417–82863417 from NCBI37/mm9 mouse reference genome (within Akap13 gene) as identified by Sun et al.<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> from ChIP-seq analysis using a hemagglutinin antibody to trap the HOXA9 binding site from the murine leukemia model obtained by transformation of murine progenitor cells by co-expression of hemagglutinin (HA)-tagged HOXA9 and MEIS1. (B) EMSA gel. 5′-end The <sup>32</sup>P-radiolabeled AKAP13-HBS DNA sequence (“f”, free DNA) was incubated with reticulocyte lysate expressing HOXA9 protein (“HOXA9”) or not (“0”) to form a complex (bound “b”) and in the presence of increasing concentrations of <b>DB818</b> or <b>DB1055</b> compounds (μM). ns, non-specific band.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-3/acs.jmedchem.8b01448/20190211/images/large/jm-2018-01448v_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01448&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i90">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40760" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40760" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 87 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factor Binding to DNA by Competing with DNA Binders as an Approach for Controlling Gene Expression</span>. <i>Curr. Top. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1358</span>, <span class="refDoi"> DOI: 10.2174/1568026615666150413154713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.2174%2F1568026615666150413154713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25866275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVGnurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2015&pages=1323-1358&author=M.+Bouhlelauthor=M.+Lambertauthor=M.-H.+David-Cordonnier&title=Targeting+Transcription+Factor+Binding+to+DNA+by+Competing+with+DNA+Binders+as+an+Approach+for+Controlling+Gene+Expression&doi=10.2174%2F1568026615666150413154713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Transcription Factor Binding to DNA by Competing with DNA Binders as an Approach for Controlling Gene Expression</span></div><div class="casAuthors">Bouhlel, Mohamed Amine; Lambert, Melanie; David-Cordonnier, Marie-Helene</div><div class="citationInfo"><span class="NLM_cas:title">Current Topics in Medicinal Chemistry (Sharjah, United Arab Emirates)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">1323-1358</span>CODEN:
                <span class="NLM_cas:coden">CTMCCL</span>;
        ISSN:<span class="NLM_cas:issn">1568-0266</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">Transcription factors are recognized as the master regulators of gene expression.  Interestingly, about 10% of the transcription factors described in mammals are up to date directly implicated in a very large no. of human diseases.  With the exception of ligand-inducible nuclear receptors, transcription factors have longtime been considered as 'undruggable' targets for therapeutics.  However, the significant breakthroughs in their protein biochem. and interactions with DNA at the structural level, together with increasing needs for new targeted-approaches particularly in cancers, has changed this postulate and opened the way for targeting transcription factors.  Along with a better knowledge of their specific DNA binding sequences by genome wide and high throughput sequencing assay, these informations make possible the potent targeting of the transcription factors by three approaches dependently of their mechanism of action.  In this review, we discuss the different physicochem. interactions between the transcription factors and the DNA helix, and the protein/protein interactions within a transcription factor complex and their impacts on the DNA structure.  In order to impair transcription factor activities, small mols. compds. can either act by direct interaction on the transcription factor, or by blocking the protein/protein interactions in a transcription complex, or by competing with the transcription factor itself and specifically targeting its cognate binding sequence.  For this latter mode of transcription targeting, we pay special attention to the DNA intercalating, alkylating or groove binders for transcription factor/DNA binding modulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoQKweR5JILLLVg90H21EOLACvtfcHk0lhfLOUGu72TWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVGnurY%253D&md5=14a54bbf9c2ffda02d7893f58a80f1b5</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.2174%2F1568026615666150413154713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1568026615666150413154713%26sid%3Dliteratum%253Aachs%26aulast%3DBouhlel%26aufirst%3DM.%26aulast%3DLambert%26aufirst%3DM.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520Transcription%2520Factor%2520Binding%2520to%2520DNA%2520by%2520Competing%2520with%2520DNA%2520Binders%2520as%2520an%2520Approach%2520for%2520Controlling%2520Gene%2520Expression%26jtitle%3DCurr.%2520Top.%2520Med.%2520Chem.%26date%3D2015%26volume%3D15%26spage%3D1323%26epage%3D1358%26doi%3D10.2174%2F1568026615666150413154713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hagenbuchner, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausserlechner, M. J.</span></span> <span> </span><span class="NLM_article-title">Targeting transcription factors by small compounds-Current strategies and future implications</span>. <i>Biochem. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>107</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">13</span>, <span class="refDoi"> DOI: 10.1016/j.bcp.2015.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bcp.2015.12.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2016&pages=1-13&author=J.+Hagenbuchnerauthor=M.+J.+Ausserlechner&title=Targeting+transcription+factors+by+small+compounds-Current+strategies+and+future+implications&doi=10.1016%2Fj.bcp.2015.12.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2015.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2015.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DHagenbuchner%26aufirst%3DJ.%26aulast%3DAusserlechner%26aufirst%3DM.%2BJ.%26atitle%3DTargeting%2520transcription%2520factors%2520by%2520small%2520compounds-Current%2520strategies%2520and%2520future%2520implications%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2016%26volume%3D107%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.bcp.2015.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Portugal, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barceló, F.</span></span> <span> </span><span class="NLM_article-title">Noncovalent Binding to DNA: Still a Target in Developing Anticancer Agents</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">4108</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.2174/0929867323666160902153511</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.2174%2F0929867323666160902153511" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27593961" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XitV2rtrrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=4108-4134&author=J.+Portugalauthor=F.+Barcel%C3%B3&title=Noncovalent+Binding+to+DNA%3A+Still+a+Target+in+Developing+Anticancer+Agents&doi=10.2174%2F0929867323666160902153511"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Noncovalent Binding to DNA: Still a Target in Developing Anticancer Agents</span></div><div class="casAuthors">Portugal, Jose; Barcelo, Francisca</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">4108-4134</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">DNA-binding compds. are of extraordinary importance in medicine, accounting for a substantial portion of antitumor drugs in clin. usage.  However, their mechanisms of action remain sometimes incompletely understood.  This review critically examines two broad classes of mols. that bind noncovalently to DNA: intercalators and groove binders.  Intercalators bind to DNA by inserting their chromophore moiety between two consecutive base pairs, whereas groove binders fit into the grooves of DNA.  Noncovalent DNAinteractive drugs can recognize certain supramol. DNA structures such as the Gquadruplexes found in telomeres and in numerous gene promoters, and they can act as topoisomerase I and II poisons.  We discuss how DNA-binding compds. affect transcription and compete with protein factors for binding to consensus binding sites in gene promoters both in vitro and in cultured cancer cells.  Moreover, we comment on the design of mols. that can tightly and specifically bind to any desired target DNA, such as various hairpin polyamides which efficacy as chemotherapeutic agents is being evaluated.  At present, genome-wide studies, which provide details of events that may influence both cancer progression and therapeutic outcome, are a common way used to analyze the effects of DNA-binding compds.  A conclusive feature that emerges from reviewing the information on DNA-binding compds. is that both natural sources and chem. approaches can be productively used to obtain drugs to manipulate gene expression in cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5f3wK3WVyv7Vg90H21EOLACvtfcHk0lhfLOUGu72TWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitV2rtrrO&md5=0cf7e9633ef98408209a5a7ab486da31</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.2174%2F0929867323666160902153511&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F0929867323666160902153511%26sid%3Dliteratum%253Aachs%26aulast%3DPortugal%26aufirst%3DJ.%26aulast%3DBarcel%25C3%25B3%26aufirst%3DF.%26atitle%3DNoncovalent%2520Binding%2520to%2520DNA%253A%2520Still%2520a%2520Target%2520in%2520Developing%2520Anticancer%2520Agents%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2016%26volume%3D23%26spage%3D4108%26epage%3D4134%26doi%3D10.2174%2F0929867323666160902153511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lambert, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jambon, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting Transcription Factors for Cancer Treatment</span>. <i>Molecules</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1479</span>, <span class="refDoi"> DOI: 10.3390/molecules23061479</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.3390%2Fmolecules23061479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVKjs7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=1479&author=M.+Lambertauthor=S.+Jambonauthor=S.+Depauwauthor=M.-H.+David-Cordonnier&title=Targeting+Transcription+Factors+for+Cancer+Treatment&doi=10.3390%2Fmolecules23061479"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription factors for cancer treatment</span></div><div class="casAuthors">Lambert, Melanie; Jambon, Samy; Depauw, Sabine; David-Cordonnier, Marie-Helene</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1479/1-1479/51</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcription factors are involved in a large no. of human diseases such as cancers for which they account for about 20% of all oncogenes identified so far.  For long time, with the exception of ligand-inducible nuclear receptors, transcription factors were considered as "undruggable" targets.  Advances knowledge of these transcription factors, in terms of structure, function (expression, degrdn., interaction with co-factors and other proteins) and the dynamics of their mode of binding to DNA has changed this postulate and paved the way for new therapies targeted against transcription factors.  Here, we discuss various ways to target transcription factors in cancer models: by modulating their expression or degrdn., by blocking protein/protein interactions, by targeting the transcription factor itself to prevent its DNA binding either through a binding pocket or at the DNA-interacting site, some of these inhibitors being currently used or evaluated for cancer treatment.  Such different targeting of transcription factors by small mols. is facilitated by modern chem. developing a wide variety of original mols. designed to specifically abort transcription factor and by an increased knowledge of their pathol. implication through the use of new technologies in order to make it possible to improve therapeutic control of transcription factor oncogenic functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH0VibdksQ77Vg90H21EOLACvtfcHk0lhfLOUGu72TWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVKjs7zK&md5=035d1312de71ad3c2dabec495edb498d</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3390%2Fmolecules23061479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules23061479%26sid%3Dliteratum%253Aachs%26aulast%3DLambert%26aufirst%3DM.%26aulast%3DJambon%26aufirst%3DS.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520Transcription%2520Factors%2520for%2520Cancer%2520Treatment%26jtitle%3DMolecules%26date%3D2018%26volume%3D23%26spage%3D1479%26doi%3D10.3390%2Fmolecules23061479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lo-Coco, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cicconi, L.</span></span> <span> </span><span class="NLM_article-title">History of Acute Promyelocytic Leukemia: A Tale of Endless Revolution</span>. <i>Mediterr. J. Hematol. Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>3</i></span>, <span class="NLM_elocation-id">e2011067</span> <span class="refDoi"> DOI: 10.4084/mjhid.2011.067</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.4084%2Fmjhid.2011.067" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22220264" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&author=F.+Lo-Cocoauthor=L.+Cicconi&title=History+of+Acute+Promyelocytic+Leukemia%3A+A+Tale+of+Endless+Revolution&doi=10.4084%2Fmjhid.2011.067"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.4084%2Fmjhid.2011.067&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4084%252Fmjhid.2011.067%26sid%3Dliteratum%253Aachs%26aulast%3DLo-Coco%26aufirst%3DF.%26aulast%3DCicconi%26aufirst%3DL.%26atitle%3DHistory%2520of%2520Acute%2520Promyelocytic%2520Leukemia%253A%2520A%2520Tale%2520of%2520Endless%2520Revolution%26jtitle%3DMediterr.%2520J.%2520Hematol.%2520Infect.%2520Dis.%26date%3D2011%26volume%3D3%26doi%3D10.4084%2Fmjhid.2011.067" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the ’undruggable’ cancer targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%99undruggable%E2%80%99+cancer+targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0liGWOQ22RlO5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%2599undruggable%25E2%2580%2599%2520cancer%2520targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tisato, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voltan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gonelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Secchiero, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zauli, G.</span></span> <span> </span><span class="NLM_article-title">MDM2/X Inhibitors under Clinical Evaluation: Perspectives for the Management of Hematological Malignancies and Pediatric Cancer</span>. <i>J. Hematol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">133</span>, <span class="refDoi"> DOI: 10.1186/s13045-017-0500-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs13045-017-0500-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28673313" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOru7jL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2017&pages=133&author=V.+Tisatoauthor=R.+Voltanauthor=A.+Gonelliauthor=P.+Secchieroauthor=G.+Zauli&title=MDM2%2FX+Inhibitors+under+Clinical+Evaluation%3A+Perspectives+for+the+Management+of+Hematological+Malignancies+and+Pediatric+Cancer&doi=10.1186%2Fs13045-017-0500-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">MDM2/X inhibitors under clinical evaluation: perspectives for the management of hematological malignancies and pediatric cancer</span></div><div class="casAuthors">Tisato, Veronica; Voltan, Rebecca; Gonelli, Arianna; Secchiero, Paola; Zauli, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Hematology & Oncology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">133/1-133/17</span>CODEN:
                <span class="NLM_cas:coden">JHOOAO</span>;
        ISSN:<span class="NLM_cas:issn">1756-8722</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  The two murine double minute (MDM) family members MDM2 and MDMX are at the center of an intense clin. assessment as mol. target for the management of cancer.  Indeed, the two proteins act as regulators of P53, a well-known key controller of the cell cycle regulation and cell proliferation that, when altered, plays a direct role on cancer development and progression.  Several evidence demonstrated that functional aberrations of P53 in tumors are in most cases the consequence of alterations on the MDM2 and MDMX regulatory proteins, in particular in patients with hematol. malignancies where TP53 shows a relatively low frequency of mutation while MDM2 and MDMX are frequently found amplified/overexpressed.  The pharmacol. targeting of these two P53- regulators in order to restore or increase P53 expression and activity represents therefore a strategy for cancer therapy.  From the discovery of the Nutlins in 2004, several compds. have been developed and reported with the ability of targeting the P53-MDM2/X axis by inhibiting MDM2 and/or MDMX.  From natural compds. up to small mols. and stapled peptides, these MDM2/X pharmacol. inhibitors have been extensively studied, revealing different biol. features and different rate of efficacy when tested in in vitro and in vivo exptl. tumor models.  The data/evidence coming from the preclin. experimentation have allowed the identification of the most promising mols. and the setting of clin. studies for their evaluation as monotherapy or in therapeutic combination with conventional chemotherapy or with innovative therapeutic protocols in different tumor settings.  Preliminary results have been recently published reporting data about safety, tolerability, potential side effects, and efficacy of such therapeutic approaches.  In this light, the aim of this review is to give an updated overview about the state of the art of the clin. evaluation of MDM2/X inhibitor compds. with a special attention to hematol. malignancies and to the potential for the management of pediatric cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQ1TexF8T_crVg90H21EOLACvtfcHk0liGWOQ22RlO5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOru7jL&md5=70e587889b574f3909f7b11b36bf3933</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1186%2Fs13045-017-0500-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13045-017-0500-5%26sid%3Dliteratum%253Aachs%26aulast%3DTisato%26aufirst%3DV.%26aulast%3DVoltan%26aufirst%3DR.%26aulast%3DGonelli%26aufirst%3DA.%26aulast%3DSecchiero%26aufirst%3DP.%26aulast%3DZauli%26aufirst%3DG.%26atitle%3DMDM2%252FX%2520Inhibitors%2520under%2520Clinical%2520Evaluation%253A%2520Perspectives%2520for%2520the%2520Management%2520of%2520Hematological%2520Malignancies%2520and%2520Pediatric%2520Cancer%26jtitle%3DJ.%2520Hematol.%2520Oncol.%26date%3D2017%26volume%3D10%26spage%3D133%26doi%3D10.1186%2Fs13045-017-0500-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leeman-Neill, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhagat, G.</span></span> <span> </span><span class="NLM_article-title">BCL6 as a Therapeutic Target for Lymphoma</span>. <i>Expert Opin. Ther. Targets</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">143</span>– <span class="NLM_lpage">152</span>, <span class="refDoi"> DOI: 10.1080/14728222.2018.1420782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1080%2F14728222.2018.1420782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29262721" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVChu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2018&pages=143-152&author=R.+J.+Leeman-Neillauthor=G.+Bhagat&title=BCL6+as+a+Therapeutic+Target+for+Lymphoma&doi=10.1080%2F14728222.2018.1420782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">BCL6 as a therapeutic target for lymphoma</span></div><div class="casAuthors">Leeman-Neill, Rebecca J.; Bhagat, Govind</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Targets</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">143-152</span>CODEN:
                <span class="NLM_cas:coden">EOTTAO</span>;
        ISSN:<span class="NLM_cas:issn">1472-8222</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  Introduction: B cell lymphoma 6 (BCL6) is a transcriptional repressor crit. for the development and maintenance of germinal centers (GCs), which are required for generation of an effective humoral immune response.  Genomic aberrations of BCL6, including mutations and translocations that occur during the GC reaction, as well as alterations of genes that regulate BCL6 expression, lead to sustained activity of BCL6, which promotes the development of GC-derived lymphomas.  Since many types of B cell non-Hodgkin lymphomas (B-NHL) arise from neoplastic transformation of GC B cells and a high proportion harbor genetic lesions that deregulate BCL6 expression, inhibition of BCL6 has emerged as an attractive therapeutic strategy for lymphomas.  Areas covered: This review examines the rationale for and challenges in therapeutic targeting of BCL6 in lymphomas.  We describe approaches that have been used and are currently being considered for inhibition of BCL6.  Expert opinion: Several BCL6 inhibiting agents, including peptidomimetics, small mols., and natural compds., most of which target the BTB domain of the protein at the corepressor binding site, have been developed with demonstration of anti-lymphoma activity in preclin. models.  Future clin. trials will be important to investigate the efficacy of targeting BCL6 in B-NHL (and other neoplasms), particularly in combination with other therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqf_UsynF__UrVg90H21EOLACvtfcHk0lh3KnMrKRckbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVChu7Y%253D&md5=dbbd71e6ce8d84829c366bccab0205b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1080%2F14728222.2018.1420782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F14728222.2018.1420782%26sid%3Dliteratum%253Aachs%26aulast%3DLeeman-Neill%26aufirst%3DR.%2BJ.%26aulast%3DBhagat%26aufirst%3DG.%26atitle%3DBCL6%2520as%2520a%2520Therapeutic%2520Target%2520for%2520Lymphoma%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Targets%26date%3D2018%26volume%3D22%26spage%3D143%26epage%3D152%26doi%3D10.1080%2F14728222.2018.1420782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Furtek, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Backos, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matheson, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reigan, P.</span></span> <span> </span><span class="NLM_article-title">Strategies and Approaches of Targeting STAT3 for Cancer Treatment</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">308</span>– <span class="NLM_lpage">318</span>, <span class="refDoi"> DOI: 10.1021/acschembio.5b00945</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.5b00945" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvVOqsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=308-318&author=S.+L.+Furtekauthor=D.+S.+Backosauthor=C.+J.+Mathesonauthor=P.+Reigan&title=Strategies+and+Approaches+of+Targeting+STAT3+for+Cancer+Treatment&doi=10.1021%2Facschembio.5b00945"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Strategies and Approaches of Targeting STAT3 for Cancer Treatment</span></div><div class="casAuthors">Furtek, Steffanie L.; Backos, Donald S.; Matheson, Christopher J.; Reigan, Philip</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">308-318</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signal transducer and activator of transcription 3 (STAT3) is a transcription factor that regulates the expression of genes related to cell cycle, cell survival, and immune response assocd. with cancer progression and malignancy in a no. of cancer types.  Once activated, STAT3 forms a homodimer and translocates to the nucleus where it binds DNA promoting the translation of target genes assocd. with antiapoptosis, angiogenesis, and invasion/migration.  In normal cells, levels of activated STAT3 remain transient; however, STAT3 remains constitutively active in approx. 70% of human solid tumors.  The pivotal role of STAT3 in tumor progression has promoted a campaign in drug discovery to identify small mols. that disrupt the function of STAT3.  A range of approaches have been used to identify novel small mol. inhibitors of STAT3, including high-throughput screening of chem. libraries, computational-based virtual screening, and fragment-based design strategies.  The most common approaches in targeting STAT3 activity are either via the inhibition of tyrosine kinases capable of phosphorylating and thereby activating STAT3 or by preventing the formation of functional STAT3 dimers through disruption of the SH2 domains.  However, the targeting of the STAT3 DNA-binding domain and disruption of binding of STAT3 to its DNA promoter have not been thoroughly examd., mainly due to the lack of adequate assay systems.  This review summarizes the development of STAT3 inhibitors organized by the approach used to inhibit STAT3, the current inhibitors of each class, and the assay systems used to evaluate STAT3 inhibition and offers an insight into future approaches for small mol. STAT3 inhibitor development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokCygAKrPD6rVg90H21EOLACvtfcHk0lh3KnMrKRckbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvVOqsA%253D%253D&md5=379321e035a5dbce720e1c787da55aad</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facschembio.5b00945&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.5b00945%26sid%3Dliteratum%253Aachs%26aulast%3DFurtek%26aufirst%3DS.%2BL.%26aulast%3DBackos%26aufirst%3DD.%2BS.%26aulast%3DMatheson%26aufirst%3DC.%2BJ.%26aulast%3DReigan%26aufirst%3DP.%26atitle%3DStrategies%2520and%2520Approaches%2520of%2520Targeting%2520STAT3%2520for%2520Cancer%2520Treatment%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2016%26volume%3D11%26spage%3D308%26epage%3D318%26doi%3D10.1021%2Facschembio.5b00945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu-Chittenden, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shim, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anders, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, D.</span></span> <span> </span><span class="NLM_article-title">Genetic and Pharmacological Disruption of the TEAD-YAP Complex Suppresses the Oncogenic Activity of YAP</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">1300</span>– <span class="NLM_lpage">1305</span>, <span class="refDoi"> DOI: 10.1101/gad.192856.112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1101%2Fgad.192856.112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22677547" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVegs7%252FL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2012&pages=1300-1305&author=Y.+Liu-Chittendenauthor=B.+Huangauthor=J.+S.+Shimauthor=Q.+Chenauthor=S.-J.+Leeauthor=R.+A.+Andersauthor=J.+O.+Liuauthor=D.+Pan&title=Genetic+and+Pharmacological+Disruption+of+the+TEAD-YAP+Complex+Suppresses+the+Oncogenic+Activity+of+YAP&doi=10.1101%2Fgad.192856.112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and pharmacological disruption of the TEAD-YAP complex suppresses the oncogenic activity of YAP</span></div><div class="casAuthors">Liu-Chittenden, Yi; Huang, Bo; Shim, Joong Sup; Chen, Qian; Lee, Se-Jin; Anders, Robert A.; Liu, Jun O.; Pan, Duojia</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1300-1305</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">The Drosophila TEAD ortholog Scalloped is required for Yki-mediated overgrowth but is largely dispensable for normal tissue growth, suggesting that its mammalian counterpart may be exploited for selective inhibition of oncogenic growth driven by YAP hyperactivation.  Here we test this hypothesis genetically and pharmacol.  We show that a dominant-neg. TEAD mol. does not perturb normal liver growth but potently suppresses hepatomegaly/tumorigenesis resulting from YAP overexpression or Neurofibromin 2 (NF2)/Merlin inactivation.  We further identify verteporfin as a small mol. that inhibits TEAD-YAP assocn. and YAP-induced liver overgrowth.  These findings provide proof of principle that inhibiting TEAD-YAP interactions is a pharmacol. viable strategy against the YAP oncoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiYAxCKwHiDLVg90H21EOLACvtfcHk0lh3KnMrKRckbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVegs7%252FL&md5=dcd42db76d9b45bd15a276e101864917</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1101%2Fgad.192856.112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.192856.112%26sid%3Dliteratum%253Aachs%26aulast%3DLiu-Chittenden%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DShim%26aufirst%3DJ.%2BS.%26aulast%3DChen%26aufirst%3DQ.%26aulast%3DLee%26aufirst%3DS.-J.%26aulast%3DAnders%26aufirst%3DR.%2BA.%26aulast%3DLiu%26aufirst%3DJ.%2BO.%26aulast%3DPan%26aufirst%3DD.%26atitle%3DGenetic%2520and%2520Pharmacological%2520Disruption%2520of%2520the%2520TEAD-YAP%2520Complex%2520Suppresses%2520the%2520Oncogenic%2520Activity%2520of%2520YAP%26jtitle%3DGenes%2520Dev.%26date%3D2012%26volume%3D26%26spage%3D1300%26epage%3D1305%26doi%3D10.1101%2Fgad.192856.112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gibault, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corvaisier, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coevoet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huet, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melnyk, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotelle, P.</span></span> <span> </span><span class="NLM_article-title">Molecular Features of the YAP Inhibitor Verteporfin: Synthesis of Hexasubstituted Dipyrrins as Potential Inhibitors of YAP/TAZ, the Downstream Effectors of the Hippo Pathway</span>. <i>ChemMedChem</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">954</span>– <span class="NLM_lpage">961</span>, <span class="refDoi"> DOI: 10.1002/cmdc.201700063</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fcmdc.201700063" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28334506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmtlSgsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=954-961&author=F.+Gibaultauthor=F.+Baillyauthor=M.+Corvaisierauthor=M.+Coevoetauthor=G.+Huetauthor=P.+Melnykauthor=P.+Cotelle&title=Molecular+Features+of+the+YAP+Inhibitor+Verteporfin%3A+Synthesis+of+Hexasubstituted+Dipyrrins+as+Potential+Inhibitors+of+YAP%2FTAZ%2C+the+Downstream+Effectors+of+the+Hippo+Pathway&doi=10.1002%2Fcmdc.201700063"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Features of the YAP Inhibitor Verteporfin: Synthesis of Hexasubstituted Dipyrrins as Potential Inhibitors of YAP/TAZ, the Downstream Effectors of the Hippo Pathway</span></div><div class="casAuthors">Gibault, Floriane; Bailly, Fabrice; Corvaisier, Matthieu; Coevoet, Mathilde; Huet, Guillemette; Melnyk, Patricia; Cotelle, Philippe</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">954-961</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Porphyrin derivs., in particular verteporfin (VP), a photosensitizer initially designed for cancer therapy, have been identified as inhibitors of the YAP-TEAD interaction and transcriptional activity.  Herein the authors report the efficient convergent synthesis of the dipyrrin half of protoporphyrin IX di-Me ester (PPIX-DME), in which the sensitive vinyl group was created at the final stage by a dehydroiodination reaction.  Two other dipyrrin derivs. were synthesized, including dipyrrin 19 [(Z)-2-((3,5-dimethyl-4-vinyl-2H-pyrrol-2-ylidene)methyl)-3,5-dimethyl-4-vinyl-1H-pyrrole], contg. two vinyl groups.  The authors found that VP and dipyrrin 19 showed significant inhibitory effects on TEAD transcriptional activity in MDA-MB-231 human breast cancer cells, whereas other compds. did not show significant changes.  In addn., the authors obsd. a marked decrease in both YAP and TAZ levels following VP treatment, whereas dipyrrin 19 treatment primarily decreased the levels of YAP and receptor kinase AXL, a downstream target of YAP.  Together, the authors' data suggest that, due to their chem. structures, porphyrin- and dipyrrin-related derivs. can directly target YAP and/or TAZ proteins and inhibit TEAD transcriptional activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppSdVREJKEorVg90H21EOLACvtfcHk0liq-UlRBU8Tfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmtlSgsL8%253D&md5=d75845ed2c4a1ddc1f318a03555c12f8</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.201700063&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.201700063%26sid%3Dliteratum%253Aachs%26aulast%3DGibault%26aufirst%3DF.%26aulast%3DBailly%26aufirst%3DF.%26aulast%3DCorvaisier%26aufirst%3DM.%26aulast%3DCoevoet%26aufirst%3DM.%26aulast%3DHuet%26aufirst%3DG.%26aulast%3DMelnyk%26aufirst%3DP.%26aulast%3DCotelle%26aufirst%3DP.%26atitle%3DMolecular%2520Features%2520of%2520the%2520YAP%2520Inhibitor%2520Verteporfin%253A%2520Synthesis%2520of%2520Hexasubstituted%2520Dipyrrins%2520as%2520Potential%2520Inhibitors%2520of%2520YAP%252FTAZ%252C%2520the%2520Downstream%2520Effectors%2520of%2520the%2520Hippo%2520Pathway%26jtitle%3DChemMedChem%26date%3D2017%26volume%3D12%26spage%3D954%26epage%3D961%26doi%3D10.1002%2Fcmdc.201700063" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gang, E.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahn, M.</span></span> <span> </span><span class="NLM_article-title">CBP/Catenin antagonists: Targeting LSCs’ Achilles heel</span>. <i>Exp. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">11</span>, <span class="refDoi"> DOI: 10.1016/j.exphem.2017.04.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.exphem.2017.04.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28479420" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSqsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2017&pages=1-11&author=Y.-M.+Kimauthor=E.-J.+Gangauthor=M.+Kahn&title=CBP%2FCatenin+antagonists%3A+Targeting+LSCs%E2%80%99+Achilles+heel&doi=10.1016%2Fj.exphem.2017.04.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">CBP/Catenin antagonists: Targeting LSCs' Achilles heel</span></div><div class="casAuthors">Kim, Yong-Mi; Gang, Eun-Ji; Kahn, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Hematology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-11</span>CODEN:
                <span class="NLM_cas:coden">EXHMA6</span>;
        ISSN:<span class="NLM_cas:issn">0301-472X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Cancer stem cells (CSCs), including leukemia stem cells (LSCs), exhibit self-renewal capacity and differentiation potential and have the capacity to maintain or renew and propagate a tumor/leukemia.  The initial isolation of CSCs/LSCs was in adult myelogenous leukemia, although more recently, the existence of CSCs in a wide variety of other cancers has been reported.  CSCs, in general, and LSCs, specifically with respect to this review, are responsible for initiation of disease, therapeutic resistance and ultimately disease relapse.  One key focus in cancer research over the past decade has been the development of therapies that safely eliminate the LSC/CSC population.  One major obstacle to this goal is the identification of key mechanisms that distinguish LSCs from normal endogenous hematopoietic stem cells.  An addnl. daunting feature that has recently come to light with advances in next-generation sequencing and single-cell sequencing is the heterogeneity within leukemias/tumors, with multiple combinations of mutations, gain and loss of function of genes, and so on being capable of driving disease, even within the CSC/LSC population.  The focus of this review/perspective is on our work in identifying and validating, in both chronic myelogenous leukemia and acute lymphoblastic leukemia, a safe and efficacious mechanism to target an evolutionarily conserved signaling nexus, which constitutes a common "Achilles heel" for LSCs/CSCs, using small mol.-specific CBP/catenin antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqI5h5QcvGrdbVg90H21EOLACvtfcHk0liq-UlRBU8Tfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSqsr4%253D&md5=848d6692beeb60072a7726ed055b8b46</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.exphem.2017.04.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.exphem.2017.04.010%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DY.-M.%26aulast%3DGang%26aufirst%3DE.-J.%26aulast%3DKahn%26aufirst%3DM.%26atitle%3DCBP%252FCatenin%2520antagonists%253A%2520Targeting%2520LSCs%25E2%2580%2599%2520Achilles%2520heel%26jtitle%3DExp.%2520Hematol.%26date%3D2017%26volume%3D52%26spage%3D1%26epage%3D11%26doi%3D10.1016%2Fj.exphem.2017.04.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pariollaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wixted, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chitnis, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fornwald, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ames, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Callahan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Solari, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diehl, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of PERK Activators by Phenotypic Screening and Their Effects on NRF2 Activation</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e0119738</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0119738</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1371%2Fjournal.pone.0119738" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25780921" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslWgtbrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2015&author=W.+Xieauthor=M.+Pariollaudauthor=W.+E.+Wixtedauthor=N.+Chitnisauthor=J.+Fornwaldauthor=M.+Truongauthor=C.+Paoauthor=Y.+Liuauthor=R.+S.+Amesauthor=J.+Callahanauthor=R.+Solariauthor=Y.+Sanchezauthor=A.+Diehlauthor=H.+Li&title=Identification+and+Characterization+of+PERK+Activators+by+Phenotypic+Screening+and+Their+Effects+on+NRF2+Activation&doi=10.1371%2Fjournal.pone.0119738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and characterization of PERK activators by phenotypic screening and their effects on NRF2 activation</span></div><div class="casAuthors">Xie, Wensheng; Pariollaud, Marie; Wixted, William E.; Chitnis, Nilesh; Fornwald, James; Truong, Maggie; Pao, Christina; Liu, Yan; Ames, Robert S.; Callahan, James; Solari, Roberto; Sanchez, Yolanda; Diehl, Alan; Li, Hu</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e0119738/1-e0119738/21</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Endoplasmic reticulum stress plays a crit. role to restore the homeostasis of protein prodn. in eukaryotic cells.  This vital process is hence involved in many types of diseases including COPD.  PERK, one branch in the ER stress signaling pathways, has been reported to activate NRF2 signaling pathway, a known protective response to COPD.  Based on this scientific rationale, we aimed to identify PERK activators as a mechanism to achieve NRF2 activation.  In this report, we describe a phenotypic screening assay to identify PERK activators.  This assay measures phosphorylation of GFP-tagged eIF2α upon PERK activation via a cell-based LanthaScreen technol.  To obtain a robust assay with sufficient signal to background and low variation, multiple parameters were optimized including GFP-tagged eIF2α BacMam concn., cell d. and serum concn.  The assay was validated by a tool compd., Thapsigargin, which induces phosphorylation of eIF2α.  In our assay, this compd. showed maximal signal window of approx. 2.5-fold with a pEC50 of 8.0, consistent with literature reports.  To identify novel PERK activators through phosphorylation of eIF2α, a focused set of 8,400 compds. was screened in this assay at 10 μM.  A no. of hits were identified and validated.  The mol. mechanisms for several selected hits were further characterized in terms of PERK activation and effects on PERK downstream components.  Specificity of these compds. in activating PERK was demonstrated with a PERK specific inhibitor and in PERK knockout mouse embryonic fibroblast (MEF) cells.  In addn., these hits showed NRF2-dependent anti-oxidant gene induction.  In summary, our phenotypic screening assay is demonstrated to be able to identify PERK specific activators.  The identified PERK activators could potentially be used as chem. probes to further investigate this pathway as well as the link between PERK activation and NRF2 pathway activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4FS7McfUsXrVg90H21EOLACvtfcHk0liq-UlRBU8Tfg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslWgtbrP&md5=037fab9d359f34f3f48cd84b9783baf6</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0119738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0119738%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DW.%26aulast%3DPariollaud%26aufirst%3DM.%26aulast%3DWixted%26aufirst%3DW.%2BE.%26aulast%3DChitnis%26aufirst%3DN.%26aulast%3DFornwald%26aufirst%3DJ.%26aulast%3DTruong%26aufirst%3DM.%26aulast%3DPao%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DAmes%26aufirst%3DR.%2BS.%26aulast%3DCallahan%26aufirst%3DJ.%26aulast%3DSolari%26aufirst%3DR.%26aulast%3DSanchez%26aufirst%3DY.%26aulast%3DDiehl%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520PERK%2520Activators%2520by%2520Phenotypic%2520Screening%2520and%2520Their%2520Effects%2520on%2520NRF2%2520Activation%26jtitle%3DPLoS%2520One%26date%3D2015%26volume%3D10%26doi%3D10.1371%2Fjournal.pone.0119738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheeseman, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chessum, N. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rye, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasqua, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tucker, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilding, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, L. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lepri, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richards, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ali, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowlands, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Fee, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miah, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hayes, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henley, A. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">te Poele, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Billy, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellegrino, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raynaud, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burke, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Montfort, R. L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eccles, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Workman, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, K.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Chemical Probe Bisamide (CCT251236): An Orally Bioavailable Efficacious Pirin Ligand from a Heat Shock Transcription Factor 1 (HSF1) Phenotypic Screen</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">180</span>– <span class="NLM_lpage">201</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01055</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01055" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2016&pages=180-201&author=M.+D.+Cheesemanauthor=N.+E.+A.+Chessumauthor=C.+S.+Ryeauthor=A.+E.+Pasquaauthor=M.+J.+Tuckerauthor=B.+Wildingauthor=L.+E.+Evansauthor=S.+Lepriauthor=M.+Richardsauthor=S.+Y.+Sharpauthor=S.+Aliauthor=M.+Rowlandsauthor=L.+O%E2%80%99Feeauthor=A.+Miahauthor=A.+Hayesauthor=A.+T.+Henleyauthor=M.+Powersauthor=R.+te+Poeleauthor=E.+De+Billyauthor=L.+Pellegrinoauthor=F.+Raynaudauthor=R.+Burkeauthor=R.+L.+M.+van+Montfortauthor=S.+A.+Ecclesauthor=P.+Workmanauthor=K.+Jones&title=Discovery+of+a+Chemical+Probe+Bisamide+%28CCT251236%29%3A+An+Orally+Bioavailable+Efficacious+Pirin+Ligand+from+a+Heat+Shock+Transcription+Factor+1+%28HSF1%29+Phenotypic+Screen&doi=10.1021%2Facs.jmedchem.6b01055"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01055%26sid%3Dliteratum%253Aachs%26aulast%3DCheeseman%26aufirst%3DM.%2BD.%26aulast%3DChessum%26aufirst%3DN.%2BE.%2BA.%26aulast%3DRye%26aufirst%3DC.%2BS.%26aulast%3DPasqua%26aufirst%3DA.%2BE.%26aulast%3DTucker%26aufirst%3DM.%2BJ.%26aulast%3DWilding%26aufirst%3DB.%26aulast%3DEvans%26aufirst%3DL.%2BE.%26aulast%3DLepri%26aufirst%3DS.%26aulast%3DRichards%26aufirst%3DM.%26aulast%3DSharp%26aufirst%3DS.%2BY.%26aulast%3DAli%26aufirst%3DS.%26aulast%3DRowlands%26aufirst%3DM.%26aulast%3DO%25E2%2580%2599Fee%26aufirst%3DL.%26aulast%3DMiah%26aufirst%3DA.%26aulast%3DHayes%26aufirst%3DA.%26aulast%3DHenley%26aufirst%3DA.%2BT.%26aulast%3DPowers%26aufirst%3DM.%26aulast%3Dte%2BPoele%26aufirst%3DR.%26aulast%3DDe%2BBilly%26aufirst%3DE.%26aulast%3DPellegrino%26aufirst%3DL.%26aulast%3DRaynaud%26aufirst%3DF.%26aulast%3DBurke%26aufirst%3DR.%26aulast%3Dvan%2BMontfort%26aufirst%3DR.%2BL.%2BM.%26aulast%3DEccles%26aufirst%3DS.%2BA.%26aulast%3DWorkman%26aufirst%3DP.%26aulast%3DJones%26aufirst%3DK.%26atitle%3DDiscovery%2520of%2520a%2520Chemical%2520Probe%2520Bisamide%2520%2528CCT251236%2529%253A%2520An%2520Orally%2520Bioavailable%2520Efficacious%2520Pirin%2520Ligand%2520from%2520a%2520Heat%2520Shock%2520Transcription%2520Factor%25201%2520%2528HSF1%2529%2520Phenotypic%2520Screen%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D60%26spage%3D180%26epage%3D201%26doi%3D10.1021%2Facs.jmedchem.6b01055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranghini, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikolovska-Coleska, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressler, G. R.</span></span> <span> </span><span class="NLM_article-title">Inhibition of Pax2 Transcription Activation with a Small Molecule That Targets the DNA Binding Domain</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">724</span>– <span class="NLM_lpage">734</span>, <span class="refDoi"> DOI: 10.1021/acschembio.6b00782</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00782" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVarsLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=724-734&author=E.+Grimleyauthor=C.+Liaoauthor=E.+J.+Ranghiniauthor=Z.+Nikolovska-Coleskaauthor=G.+R.+Dressler&title=Inhibition+of+Pax2+Transcription+Activation+with+a+Small+Molecule+That+Targets+the+DNA+Binding+Domain&doi=10.1021%2Facschembio.6b00782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of Pax2 Transcription Activation with a Small Molecule that Targets the DNA Binding Domain</span></div><div class="casAuthors">Grimley, Edward; Liao, Chenzhong; Ranghini, Egon J.; Nikolovska-Coleska, Zaneta; Dressler, Gregory R.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">724-734</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Pax gene family encodes DNA binding transcription factors that control crit. steps in embryonic development and differentiation of specific cell lineages.  Often, Pax proteins are re-expressed or ectopically expressed in cancer and other diseases of abnormal proliferation, making them attractive targets for tissue specific inhibition by small mols.  The authors used a homol. model of the Pax2 paired domain and a virtual screen to identify small mols. that can inhibit binding of the paired domain to DNA and Pax2 mediated transcription activation.  Candidates from the virtual screen were then confirmed in a cell based Pax2 transactivation assay.  Subsequently, the authors tested analogs of these hits to identify a single compd. that effectively blocked Pax2 activity and DNA binding with a Kd of 1.35-1.5 μM.  The compd., termed EG1, was used to inhibit embryonic kidney development, a process directly dependent on Pax2 activity.  Furthermore, the authors show that EG1 can inhibit proliferation of Pax2 pos. renal and ovarian cancer cell lines but has little effect on Pax2 neg. cancer cells.  These data confirm that small mols. targeting the DNA binding paired domain can be identified and may be good lead compds. for developing tissue and cell-type specific anti-cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8YEmyueW-0LVg90H21EOLACvtfcHk0ljkvqv1Gfi1NA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVarsLc%253D&md5=9a5790d3a402a1a66932ae97f4958879</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00782%26sid%3Dliteratum%253Aachs%26aulast%3DGrimley%26aufirst%3DE.%26aulast%3DLiao%26aufirst%3DC.%26aulast%3DRanghini%26aufirst%3DE.%2BJ.%26aulast%3DNikolovska-Coleska%26aufirst%3DZ.%26aulast%3DDressler%26aufirst%3DG.%2BR.%26atitle%3DInhibition%2520of%2520Pax2%2520Transcription%2520Activation%2520with%2520a%2520Small%2520Molecule%2520That%2520Targets%2520the%2520DNA%2520Binding%2520Domain%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2017%26volume%3D12%26spage%3D724%26epage%3D734%26doi%3D10.1021%2Facschembio.6b00782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Leung, C.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, D. S.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, V. P.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, D.-L.</span></span> <span> </span><span class="NLM_article-title">DNA-Binding Small Molecules as Inhibitors of Transcription Factors</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">846</span>, <span class="refDoi"> DOI: 10.1002/med.21266</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fmed.21266" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22549740" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2012&pages=823-846&author=C.-H.+Leungauthor=D.+S.-H.+Chanauthor=V.+P.-Y.+Maauthor=D.-L.+Ma&title=DNA-Binding+Small+Molecules+as+Inhibitors+of+Transcription+Factors&doi=10.1002%2Fmed.21266"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fmed.21266&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21266%26sid%3Dliteratum%253Aachs%26aulast%3DLeung%26aufirst%3DC.-H.%26aulast%3DChan%26aufirst%3DD.%2BS.-H.%26aulast%3DMa%26aufirst%3DV.%2BP.-Y.%26aulast%3DMa%26aufirst%3DD.-L.%26atitle%3DDNA-Binding%2520Small%2520Molecules%2520as%2520Inhibitors%2520of%2520Transcription%2520Factors%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2012%26volume%3D33%26spage%3D823%26epage%3D846%26doi%3D10.1002%2Fmed.21266" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fontaine, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">François, M.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Manipulation of Transcription Factor Protein-Protein Interactions: Opportunities and Obstacles</span>. <i>Cell Regen.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">2</span>, <span class="refDoi"> DOI: 10.1186/s13619-015-0015-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs13619-015-0015-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC2MjhslyjsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2015&pages=2&author=F.+Fontaineauthor=J.+Overmanauthor=M.+Fran%C3%A7ois&title=Pharmacological+Manipulation+of+Transcription+Factor+Protein-Protein+Interactions%3A+Opportunities+and+Obstacles&doi=10.1186%2Fs13619-015-0015-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological manipulation of transcription factor protein-protein interactions: opportunities and obstacles</span></div><div class="casAuthors">Fontaine Frank; Overman Jeroen; Francois Mathias</div><div class="citationInfo"><span class="NLM_cas:title">Cell regeneration (London, England)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">2</span>
        ISSN:<span class="NLM_cas:issn">2045-9769</span>.
    </div><div class="casAbstract">Much research on transcription factor biology and their genetic pathways has been undertaken over the last 30 years, especially in the field of developmental biology and cancer.  Yet, very little is known about the molecular modalities of highly dynamic interactions between transcription factors, genomic DNA, and protein partners.  Methodological breakthroughs such as RNA-seq (RNA-sequencing), ChIP-seq (chromatin immunoprecipitation sequencing), RIME (rapid immunoprecipitation mass spectrometry of endogenous proteins), and single-molecule imaging will dramatically accelerate the discovery rate of their molecular mode of action in the next few years.  From a pharmacological viewpoint, conventional methods used to target transcription factor activity with molecules mimicking endogenous ligands fail to achieve high specificity and are limited by a lack of identification of new molecular targets.  Protein-protein interactions are likely to represent one of the next major classes of therapeutic targets.  Transcription factors, known to act mostly via protein-protein interaction, may well be at the forefront of this type of drug development.  One hurdle in this field remains the difficulty to collate structural data into meaningful information for rational drug design.  Another hurdle is the lack of chemical libraries meeting the structural requirements of protein-protein interaction disruption.  As more attempts at modulating transcription factor activity are undertaken, valuable knowledge will be accumulated on the modality of action required to modulate transcription and how these findings can be applied to developing transcription factor drugs.  Key discoveries will spawn into new therapeutic approaches not only as anticancer targets but also for other indications, such as those with an inflammatory component including neurodegenerative disorders, diabetes, and chronic liver and kidney diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRvMXrDUnx-8JSWdIdmEou5fW6udTcc2ebxJPTAaRp-gLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjhslyjsQ%253D%253D&md5=e092b82911757d58c596da8bceea04e0</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1186%2Fs13619-015-0015-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs13619-015-0015-x%26sid%3Dliteratum%253Aachs%26aulast%3DFontaine%26aufirst%3DF.%26aulast%3DOverman%26aufirst%3DJ.%26aulast%3DFran%25C3%25A7ois%26aufirst%3DM.%26atitle%3DPharmacological%2520Manipulation%2520of%2520Transcription%2520Factor%2520Protein-Protein%2520Interactions%253A%2520Opportunities%2520and%2520Obstacles%26jtitle%3DCell%2520Regen.%26date%3D2015%26volume%3D4%26spage%3D2%26doi%3D10.1186%2Fs13619-015-0015-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Blume, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snyder, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ray, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koller, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. M.</span></span> <span> </span><span class="NLM_article-title">Mithramycin Inhibits SP1 Binding and Selectively Inhibits Transcriptional Activity of the Dihydrofolate Reductase Gene in Vitro and in Vivo</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>88</i></span>,  <span class="NLM_fpage">1613</span>– <span class="NLM_lpage">1621</span>, <span class="refDoi"> DOI: 10.1172/jci115474</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci115474" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1834700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK38XhsFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=88&publication_year=1991&pages=1613-1621&author=S.+W.+Blumeauthor=R.+C.+Snyderauthor=R.+Rayauthor=S.+Thomasauthor=C.+A.+Kollerauthor=D.+M.+Miller&title=Mithramycin+Inhibits+SP1+Binding+and+Selectively+Inhibits+Transcriptional+Activity+of+the+Dihydrofolate+Reductase+Gene+in+Vitro+and+in+Vivo&doi=10.1172%2Fjci115474"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Mithramycin inhibits SP1 binding and selectively inhibits transcriptional activity of the dihydrofolate reductase gene in vitro and in vivo</span></div><div class="casAuthors">Blume, Scott W.; Snyder, Richard C.; Ray, Ratna; Thomas, Shelia; Koller, Charles A.; Miller, Donald M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Investigation</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">88</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1613-21</span>CODEN:
                <span class="NLM_cas:coden">JCINAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-9738</span>.
    </div><div class="casAbstract">The promoter of the human dihydrofolate reductase (DHFR) gene contains two consensus binding sites for the DNA binding protein Sp1.  DNAse protection and gel mobility shift assays demonstrate binding of recombinant Sp1 to both decanucleotide Sp1 binding sequences which are located 49 and 14 base pairs upstream of the transcription start site.  The more distal of the two binding sites exhibits a somewhat higher affinity for Sp1.  The G-C specific DNA binding drug mithramycin binds to both consensus sequences and prevents subsequent Sp1 binding.  Promoter-dependent in vitro transcription of a DHER template is selectively inhibited by mithramycin when compared to the human H2b histone gene.  A similar effect is also noted in vivo.  Mithramycin treatment of MCF-7 human breast carcinoma cells contg. an amplified DHFR gene induces selective inhibition of DHFR transcription initiation, resulting in a decline in DHFR mRNA level and enzyme activity.  This selective inhibition of DHFR expression suggests that it is possible to modulate the overexpression of the DHFR gene in methotrexate resistant cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDg0_h53y-hrVg90H21EOLACvtfcHk0liWaYXk2DfOjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsFCmtg%253D%253D&md5=4972e7fd70c21156e6f6db63a1301358</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1172%2Fjci115474&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci115474%26sid%3Dliteratum%253Aachs%26aulast%3DBlume%26aufirst%3DS.%2BW.%26aulast%3DSnyder%26aufirst%3DR.%2BC.%26aulast%3DRay%26aufirst%3DR.%26aulast%3DThomas%26aufirst%3DS.%26aulast%3DKoller%26aufirst%3DC.%2BA.%26aulast%3DMiller%26aufirst%3DD.%2BM.%26atitle%3DMithramycin%2520Inhibits%2520SP1%2520Binding%2520and%2520Selectively%2520Inhibits%2520Transcriptional%2520Activity%2520of%2520the%2520Dihydrofolate%2520Reductase%2520Gene%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1991%26volume%3D88%26spage%3D1613%26epage%3D1621%26doi%3D10.1172%2Fjci115474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grohar, P.
J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woldemichael, G. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, L. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendoza, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Q.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yeung, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Currier, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khanna, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McMahon, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span> <span> </span><span class="NLM_article-title">Identification of an Inhibitor of the EWS-FLI1 Oncogenic Transcription Factor by High-Throughput Screening</span>. <i>J. Natl. Cancer Inst.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>103</i></span>,  <span class="NLM_fpage">962</span>– <span class="NLM_lpage">978</span>, <span class="refDoi"> DOI: 10.1093/jnci/djr156</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fjnci%2Fdjr156" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=21653923" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXot1Cltro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2011&pages=962-978&author=P.%0AJ.+Groharauthor=G.+M.+Woldemichaelauthor=L.+B.+Griffinauthor=A.+Mendozaauthor=Q.-R.+Chenauthor=C.+Yeungauthor=D.+G.+Currierauthor=S.+Davisauthor=C.+Khannaauthor=J.+Khanauthor=J.+B.+McMahonauthor=L.+J.+Helman&title=Identification+of+an+Inhibitor+of+the+EWS-FLI1+Oncogenic+Transcription+Factor+by+High-Throughput+Screening&doi=10.1093%2Fjnci%2Fdjr156"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of an Inhibitor of the EWS-FLI1 Oncogenic Transcription Factor by High-Throughput Screening</span></div><div class="casAuthors">Grohar, Patrick J.; Woldemichael, Girma M.; Griffin, Laurie B.; Mendoza, Arnulfo; Chen, Qing-Rong; Yeung, Choh; Currier, Duane G.; Davis, Sean; Khanna, Chand; Khan, Javed; McMahon, James B.; Helman, Lee J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">962-978</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background Chromosomal translocations generating oncogenic transcription factors are the hallmark of a variety of tumors, including many sarcomas.  Ewing sarcoma family of tumors (ESFTs) are characterized by the t(11;22)(q24;q12) translocation that generates the Ewing sarcoma breakpoint region 1 and Friend leukemia virus integration 1 (EWS-FLI1) fusion transcription factor responsible for the highly malignant phenotype of this tumor.  Although continued expression of EWS-FLI1 is believed to be crit. for ESFT cell survival, a clin. effective small-mol. inhibitor remains elusive likely because EWS-FLI1 is a transcription factor and therefore widely felt to be "undruggable.  " Methods We developed a high-throughput screen to evaluate more than 50 000 compds. for inhibition of EWS-FLI1 activity in TC32 ESFT cells.  We used a TC32 cell-based luciferase reporter screen using the EWS-FLI1 downstream target NR0B1 promoter and a gene signature secondary screen to sort and prioritize the compds.  We characterized the lead compd., mithramycin, based on its ability to inhibit EWS-FLI1 activity in vitro using microarray expression profiling, quant. reverse transcription-polymerase chain reaction, and immunoblot anal., and in vivo using immunohistochem.  We studied the impact of this inhibition on cell viability in vitro and on tumor growth in ESFT xenograft models in vivo (n = 15-20 mice per group).  All statistical tests were two-sided.  Results Mithramycin inhibited expression of EWS-FLI1 downstream targets at the mRNA and protein levels and decreased the growth of ESFT cells at half maximal inhibitory concns. between 10 (95% confidence interval [CI] = 8 to 13 nM) and 15 nM (95% CI = 13 to 19 nM).  Mithramycin suppressed the growth of two different ESFT xenograft tumors and prolonged the survival of ESFT xenograft-bearing mice by causing a decrease in mean tumor vol.  For example, in the TC32 xenograft model, on day 15 of treatment, the mean tumor vol. for the mithramycin-treated mice was approx. 3% of the tumor vol. obsd. in the control mice (mithramycin vs control: 69 vs 2388 mm3, difference = 2319 mm3, 95% CI = 1766 to 2872 mm3, P < .001).  Conclusion Mithramycin inhibits EWS-FLI1 activity and demonstrates ESFT antitumor activity both in vitro and in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-zpZMmMy4bLVg90H21EOLACvtfcHk0liWaYXk2DfOjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXot1Cltro%253D&md5=814cc95d2b54de9b275cc1df8a1b19ee</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjr156&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjr156%26sid%3Dliteratum%253Aachs%26aulast%3DGrohar%26aufirst%3DP.%2BJ.%26aulast%3DWoldemichael%26aufirst%3DG.%2BM.%26aulast%3DGriffin%26aufirst%3DL.%2BB.%26aulast%3DMendoza%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DQ.-R.%26aulast%3DYeung%26aufirst%3DC.%26aulast%3DCurrier%26aufirst%3DD.%2BG.%26aulast%3DDavis%26aufirst%3DS.%26aulast%3DKhanna%26aufirst%3DC.%26aulast%3DKhan%26aufirst%3DJ.%26aulast%3DMcMahon%26aufirst%3DJ.%2BB.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26atitle%3DIdentification%2520of%2520an%2520Inhibitor%2520of%2520the%2520EWS-FLI1%2520Oncogenic%2520Transcription%2520Factor%2520by%2520High-Throughput%2520Screening%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2011%26volume%3D103%26spage%3D962%26epage%3D978%26doi%3D10.1093%2Fjnci%2Fdjr156" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grohar, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sissung, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arnaldez, F. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Figg, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitcomb, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helman, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Widemann, B. C.</span></span> <span> </span><span class="NLM_article-title">A phase I/II trial and pharmacokinetic study of mithramycin in children and adults with refractory Ewing sarcoma and EWS-FLI1 fusion transcript</span>. <i>Cancer Chemother. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>80</i></span>,  <span class="NLM_fpage">645</span>– <span class="NLM_lpage">652</span>, <span class="refDoi"> DOI: 10.1007/s00280-017-3382-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1007%2Fs00280-017-3382-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28735378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1Wru77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2017&pages=645-652&author=P.+J.+Groharauthor=J.+Glodauthor=C.+J.+Peerauthor=T.+M.+Sissungauthor=F.+I.+Arnaldezauthor=L.+Longauthor=W.+D.+Figgauthor=P.+Whitcombauthor=L.+J.+Helmanauthor=B.+C.+Widemann&title=A+phase+I%2FII+trial+and+pharmacokinetic+study+of+mithramycin+in+children+and+adults+with+refractory+Ewing+sarcoma+and+EWS-FLI1+fusion+transcript&doi=10.1007%2Fs00280-017-3382-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">A phase I/II trial and pharmacokinetic study of mithramycin in children and adults with refractory Ewing sarcoma and EWS-FLI1 fusion transcript</span></div><div class="casAuthors">Grohar, Patrick J.; Glod, John; Peer, Cody J.; Sissung, Tristan M.; Arnaldez, Fernanda I.; Long, Lauren; Figg, William D.; Whitcomb, Patricia; Helman, Lee J.; Widemann, Brigitte C.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Chemotherapy and Pharmacology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">80</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">645-652</span>CODEN:
                <span class="NLM_cas:coden">CCPHDZ</span>;
        ISSN:<span class="NLM_cas:issn">0344-5704</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Purpose: In a preclin. drug screen, mithramycin was identified as a potent inhibitor of the Ewing sarcoma EWS-FLI1 transcription factor.  We conducted a phase I/II trial to det. the dose-limiting toxicities (DLT), max. tolerated dose (MTD), and pharmacokinetics (PK) of mithramycin in children with refractory solid tumors, and the activity in children and adults with refractory Ewing sarcoma.  Patients and methods: Mithramycin was administered i.v. over 6 h once daily for 7 days for 28 day cycles.  Adult patients (phase II) initially received mithramycin at the previously detd. recommended dose of 25 μg/kg/dose.  The planned starting dose for children (phase I) was 17.5 μg/kg/dose.  Plasma samples were obtained for mithramycin PK anal.  Results: The first two adult patients experienced reversible grade 4 alanine aminotransferase (ALT)/aspartate aminotransferase (AST) elevation exceeding the MTD.  Subsequent adult patients received mithramycin at 17.5 μg/kg/dose, and children at 13 μg/kg/dose with dexamethasone pretreatment.  None of the four subsequent adult and two pediatric patients experienced cycle 1 DLT.  No clin. responses were obsd.  The av. maximal mithramycin plasma concn. in four patients was 17.8 ± 4.6 ng/mL.  This is substantially below the sustained mithramycin concns. ≥50 nmol/L required to suppress EWS-FLI1 transcriptional activity in preclin. studies.  Due to inability to safely achieve the desired mithramycin exposure, the trial was closed to enrollment.  Conclusions: Hepatotoxicity precluded the administration of a mithramycin at a dose required to inhibit EWS-FLI1.  Evaluation of mithramycin in patients selected for decreased susceptibility to elevated transaminases may allow for improved drug exposure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtCG7I9CUUwLVg90H21EOLACvtfcHk0liWaYXk2DfOjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1Wru77F&md5=c83b447fc26734e8bb71aa61420ada13</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1007%2Fs00280-017-3382-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00280-017-3382-x%26sid%3Dliteratum%253Aachs%26aulast%3DGrohar%26aufirst%3DP.%2BJ.%26aulast%3DGlod%26aufirst%3DJ.%26aulast%3DPeer%26aufirst%3DC.%2BJ.%26aulast%3DSissung%26aufirst%3DT.%2BM.%26aulast%3DArnaldez%26aufirst%3DF.%2BI.%26aulast%3DLong%26aufirst%3DL.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DWhitcomb%26aufirst%3DP.%26aulast%3DHelman%26aufirst%3DL.%2BJ.%26aulast%3DWidemann%26aufirst%3DB.%2BC.%26atitle%3DA%2520phase%2520I%252FII%2520trial%2520and%2520pharmacokinetic%2520study%2520of%2520mithramycin%2520in%2520children%2520and%2520adults%2520with%2520refractory%2520Ewing%2520sarcoma%2520and%2520EWS-FLI1%2520fusion%2520transcript%26jtitle%3DCancer%2520Chemother.%2520Pharmacol.%26date%3D2017%26volume%3D80%26spage%3D645%26epage%3D652%26doi%3D10.1007%2Fs00280-017-3382-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morita, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitsuda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokushige, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kashiwazaki, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taniguchi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maeda, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Noura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirata, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kataoka, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiyose, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tokumasu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matsuo, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okuno, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naka, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ito, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitamura, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaneda, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bando, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adachi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugiyama, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamikubo, Y.</span></span> <span> </span><span class="NLM_article-title">Genetic Regulation of the RUNX Transcription Factor Family Has Antitumor Effects</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">2815</span>– <span class="NLM_lpage">2828</span>, <span class="refDoi"> DOI: 10.1172/jci91788</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci91788" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=2815-2828&author=K.+Moritaauthor=K.+Suzukiauthor=S.+Maedaauthor=A.+Matsuoauthor=Y.+Mitsudaauthor=C.+Tokushigeauthor=G.+Kashiwazakiauthor=J.+Taniguchiauthor=R.+Maedaauthor=M.+Nouraauthor=M.+Hirataauthor=T.+Kataokaauthor=A.+Yanoauthor=Y.+Yamadaauthor=H.+Kiyoseauthor=M.+Tokumasuauthor=H.+Matsuoauthor=S.+Tanakaauthor=Y.+Okunoauthor=M.+Mutoauthor=K.+Nakaauthor=K.+Itoauthor=T.+Kitamuraauthor=Y.+Kanedaauthor=P.+P.+Liuauthor=T.+Bandoauthor=S.+Adachiauthor=H.+Sugiyamaauthor=Y.+Kamikubo&title=Genetic+Regulation+of+the+RUNX+Transcription+Factor+Family+Has+Antitumor+Effects&doi=10.1172%2Fjci91788"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1172%2Fjci91788&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci91788%26sid%3Dliteratum%253Aachs%26aulast%3DMorita%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DK.%26aulast%3DMaeda%26aufirst%3DS.%26aulast%3DMatsuo%26aufirst%3DA.%26aulast%3DMitsuda%26aufirst%3DY.%26aulast%3DTokushige%26aufirst%3DC.%26aulast%3DKashiwazaki%26aufirst%3DG.%26aulast%3DTaniguchi%26aufirst%3DJ.%26aulast%3DMaeda%26aufirst%3DR.%26aulast%3DNoura%26aufirst%3DM.%26aulast%3DHirata%26aufirst%3DM.%26aulast%3DKataoka%26aufirst%3DT.%26aulast%3DYano%26aufirst%3DA.%26aulast%3DYamada%26aufirst%3DY.%26aulast%3DKiyose%26aufirst%3DH.%26aulast%3DTokumasu%26aufirst%3DM.%26aulast%3DMatsuo%26aufirst%3DH.%26aulast%3DTanaka%26aufirst%3DS.%26aulast%3DOkuno%26aufirst%3DY.%26aulast%3DMuto%26aufirst%3DM.%26aulast%3DNaka%26aufirst%3DK.%26aulast%3DIto%26aufirst%3DK.%26aulast%3DKitamura%26aufirst%3DT.%26aulast%3DKaneda%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DP.%2BP.%26aulast%3DBando%26aufirst%3DT.%26aulast%3DAdachi%26aufirst%3DS.%26aulast%3DSugiyama%26aufirst%3DH.%26aulast%3DKamikubo%26aufirst%3DY.%26atitle%3DGenetic%2520Regulation%2520of%2520the%2520RUNX%2520Transcription%2520Factor%2520Family%2520Has%2520Antitumor%2520Effects%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D2815%26epage%3D2828%26doi%3D10.1172%2Fjci91788" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Antony-Debré, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leite, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carvajal, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Todorova, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholdy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayanagari, S.-R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ambesi-Impiombato, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferrando, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mantzaris, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gavathiotis, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verma, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Will, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, G. M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steidl, U.</span></span> <span> </span><span class="NLM_article-title">Pharmacological Inhibition of the Transcription Factor PU.1 in Leukemia</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>127</i></span>,  <span class="NLM_fpage">4297</span>– <span class="NLM_lpage">4313</span>, <span class="refDoi"> DOI: 10.1172/jci92504</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci92504" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2017&pages=4297-4313&author=I.+Antony-Debr%C3%A9author=A.+Paulauthor=J.+Leiteauthor=K.+Mitchellauthor=H.+M.+Kimauthor=L.+A.+Carvajalauthor=T.+I.+Todorovaauthor=K.+Huangauthor=A.+Kumarauthor=A.+A.+Farahatauthor=B.+Bartholdyauthor=S.-R.+Narayanagariauthor=J.+Chenauthor=A.+Ambesi-Impiombatoauthor=A.+A.+Ferrandoauthor=I.+Mantzarisauthor=E.+Gavathiotisauthor=A.+Vermaauthor=B.+Willauthor=D.+W.+Boykinauthor=W.+D.+Wilsonauthor=G.+M.+K.+Poonauthor=U.+Steidl&title=Pharmacological+Inhibition+of+the+Transcription+Factor+PU.1+in+Leukemia&doi=10.1172%2Fjci92504"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1172%2Fjci92504&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci92504%26sid%3Dliteratum%253Aachs%26aulast%3DAntony-Debr%25C3%25A9%26aufirst%3DI.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DLeite%26aufirst%3DJ.%26aulast%3DMitchell%26aufirst%3DK.%26aulast%3DKim%26aufirst%3DH.%2BM.%26aulast%3DCarvajal%26aufirst%3DL.%2BA.%26aulast%3DTodorova%26aufirst%3DT.%2BI.%26aulast%3DHuang%26aufirst%3DK.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBartholdy%26aufirst%3DB.%26aulast%3DNarayanagari%26aufirst%3DS.-R.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DAmbesi-Impiombato%26aufirst%3DA.%26aulast%3DFerrando%26aufirst%3DA.%2BA.%26aulast%3DMantzaris%26aufirst%3DI.%26aulast%3DGavathiotis%26aufirst%3DE.%26aulast%3DVerma%26aufirst%3DA.%26aulast%3DWill%26aufirst%3DB.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DPoon%26aufirst%3DG.%2BM.%2BK.%26aulast%3DSteidl%26aufirst%3DU.%26atitle%3DPharmacological%2520Inhibition%2520of%2520the%2520Transcription%2520Factor%2520PU.1%2520in%2520Leukemia%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2017%26volume%3D127%26spage%3D4297%26epage%3D4313%26doi%3D10.1172%2Fjci92504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shah, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sukumar, S.</span></span> <span> </span><span class="NLM_article-title">The Hox Genes and Their Roles in Oncogenesis</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">361</span>– <span class="NLM_lpage">371</span>, <span class="refDoi"> DOI: 10.1038/nrc2826</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnrc2826" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20357775" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktVGju7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2010&pages=361-371&author=N.+Shahauthor=S.+Sukumar&title=The+Hox+Genes+and+Their+Roles+in+Oncogenesis&doi=10.1038%2Fnrc2826"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The Hox genes and their roles in oncogenesis</span></div><div class="casAuthors">Shah, Nilay; Sukumar, Saraswati</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">361-371</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Hox genes, a highly conserved subgroup of the homeobox superfamily, have crucial roles in development, regulating numerous processes including apoptosis, receptor signaling, differentiation, motility and angiogenesis.  Aberrations in Hox gene expression have been reported in abnormal development and malignancy, indicating that altered expression of Hox genes could be important for both oncogenesis and tumor suppression, depending on context.  Therefore, Hox gene expression could be important in diagnosis and therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzXDSElPxjirVg90H21EOLACvtfcHk0li-ZRGAWa3ocg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktVGju7g%253D&md5=d51cad3c6625422525b8e9e6dff4a6cf</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fnrc2826&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2826%26sid%3Dliteratum%253Aachs%26aulast%3DShah%26aufirst%3DN.%26aulast%3DSukumar%26aufirst%3DS.%26atitle%3DThe%2520Hox%2520Genes%2520and%2520Their%2520Roles%2520in%2520Oncogenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2010%26volume%3D10%26spage%3D361%26epage%3D371%26doi%3D10.1038%2Fnrc2826" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gonçalves, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier-Magalhães, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pojo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Correia, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reis, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sousa, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa, B. M.</span></span> <span> </span><span class="NLM_article-title">Transcriptional Profiling of HOXA9-Regulated Genes in Human Glioblastoma Cell Models</span>. <i>Genomics Data</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">54</span>– <span class="NLM_lpage">58</span>, <span class="refDoi"> DOI: 10.1016/j.gdata.2015.05.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.gdata.2015.05.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=54-58&author=C.+S.+Gon%C3%A7alvesauthor=A.+Xavier-Magalh%C3%A3esauthor=M.+Pojoauthor=A.+I.+Oliveiraauthor=S.+Correiaauthor=R.+M.+Reisauthor=N.+Sousaauthor=M.+Rochaauthor=B.+M.+Costa&title=Transcriptional+Profiling+of+HOXA9-Regulated+Genes+in+Human+Glioblastoma+Cell+Models&doi=10.1016%2Fj.gdata.2015.05.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.gdata.2015.05.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gdata.2015.05.010%26sid%3Dliteratum%253Aachs%26aulast%3DGon%25C3%25A7alves%26aufirst%3DC.%2BS.%26aulast%3DXavier-Magalh%25C3%25A3es%26aufirst%3DA.%26aulast%3DPojo%26aufirst%3DM.%26aulast%3DOliveira%26aufirst%3DA.%2BI.%26aulast%3DCorreia%26aufirst%3DS.%26aulast%3DReis%26aufirst%3DR.%2BM.%26aulast%3DSousa%26aufirst%3DN.%26aulast%3DRocha%26aufirst%3DM.%26aulast%3DCosta%26aufirst%3DB.%2BM.%26atitle%3DTranscriptional%2520Profiling%2520of%2520HOXA9-Regulated%2520Genes%2520in%2520Human%2520Glioblastoma%2520Cell%2520Models%26jtitle%3DGenomics%2520Data%26date%3D2015%26volume%3D5%26spage%3D54%26epage%3D58%26doi%3D10.1016%2Fj.gdata.2015.05.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pires-Luís, A.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Costa-Pinheiro, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ferreira, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Antunes, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lobo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oliveira, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henrique, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jerónimo, C.</span></span> <span> </span><span class="NLM_article-title">Identification of Clear Cell Renal Cell Carcinoma and Oncocytoma Using a Three-Gene Promoter Methylation Panel</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">149</span>– <span class="NLM_lpage">157</span>, <span class="refDoi"> DOI: 10.1186/s12967-017-1248-y</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2Fs12967-017-1248-y" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28662726" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=149-157&author=A.%0AS.+Pires-Lu%C3%ADsauthor=P.+Costa-Pinheiroauthor=M.+J.+Ferreiraauthor=L.+Antunesauthor=F.+Loboauthor=J.+Oliveiraauthor=R.+Henriqueauthor=C.+Jer%C3%B3nimo&title=Identification+of+Clear+Cell+Renal+Cell+Carcinoma+and+Oncocytoma+Using+a+Three-Gene+Promoter+Methylation+Panel&doi=10.1186%2Fs12967-017-1248-y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identifcation of clear cell renal cell carcinoma and oncocytoma using a three-gene promoter methylation panel</span></div><div class="casAuthors">Pires-Luis, Ana Silvia; Costa-Pinheiro, Pedro; Ferreira, Maria Joao; Antunes, Luis; Lobo, Francisco; Oliveira, Jorge; Henrique, Rui; Jeronimo, Carmen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Translational Medicine</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">149/1-149/9</span>CODEN:
                <span class="NLM_cas:coden">JTMOBV</span>;
        ISSN:<span class="NLM_cas:issn">1479-5876</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: Promoter methylation has emerged as a promising class of epigenetic biomarkers for diagnosis and prognosis of renal cell tumors (RCTs).  Although differential gene promoter methylation patterns have been reported for the major subtypes (clear cell, papillary and chromophobe renal cell carcinoma, and oncocytoma), validation of diagnostic performance in independent series have been seldom performed.  Herein, we aimed at assessing the diagnostic performance of genes previously shown to be hypermethylated in RCTs in different clin. settings.  Methods: Promoter methylation levels of HOXA9 and OXR1 were assessed by quant. methylation specific PCR.  ROC curves were generated for OXR1, OXR1 combined with MST1R and HOXA9.  Sensitivity, specificity, pos. predictive value, neg. predictive value and accuracy were computed, maximizing specificity.  Methylation levels were also correlated with clin. and pathol. relevant parameters.  Results: HOXA9 and OXR1 promoter methylation was disclosed in 73 and 87% of RCTs, resp.  A two-gene methylation panel comprising OXR1 and MST1R identified malignancy with 98% sensitivity and 100% specificity, and clear cell renal cell carcinoma with 90% sensitivity and 98% specificity.  HOXA9 promoter methylation allowed for discrimination between oncocytoma and both papillary and chromophobe renal cell carcinoma but only with 77% sensitivity and 73% specificity.  Significantly higher OXR1 promoter methylation levels (p = 0.005) were assocd. with high nuclear grade in ccRCC.  Conclusions: A panel including OXR1 and MST1R promoter methylation allows specific and sensitive identification of renal cell tumors, and, esp., of clear cell renal cell carcinoma.  Moreover, higher OXR1 promoter methylation levels assoc. with clear cell renal cell carcinoma nuclear grade, a surrogate for tumor aggressiveness.  Thus, gene promoter methylation anal. might a useful ancillary tool in diagnostic management of renal masses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbEGBik2z4GbVg90H21EOLACvtfcHk0lgXbUhogH6puA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOqtrvP&md5=262ecf7824f683600340a54b614af553</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1186%2Fs12967-017-1248-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12967-017-1248-y%26sid%3Dliteratum%253Aachs%26aulast%3DPires-Lu%25C3%25ADs%26aufirst%3DA.%2BS.%26aulast%3DCosta-Pinheiro%26aufirst%3DP.%26aulast%3DFerreira%26aufirst%3DM.%2BJ.%26aulast%3DAntunes%26aufirst%3DL.%26aulast%3DLobo%26aufirst%3DF.%26aulast%3DOliveira%26aufirst%3DJ.%26aulast%3DHenrique%26aufirst%3DR.%26aulast%3DJer%25C3%25B3nimo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520Clear%2520Cell%2520Renal%2520Cell%2520Carcinoma%2520and%2520Oncocytoma%2520Using%2520a%2520Three-Gene%2520Promoter%2520Methylation%2520Panel%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2017%26volume%3D15%26spage%3D149%26epage%3D157%26doi%3D10.1186%2Fs12967-017-1248-y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Malek, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajula, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nghiem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simons, B. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nugent, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taparra, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lemtiri-Chlieh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoon, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">True, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DeWeese, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, A. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaeffer, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pienta, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hurley, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrissey, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran, P. T.</span></span> <span> </span><span class="NLM_article-title">TWIST1-WDR5- Hottip Regulates Hoxa9 Chromatin to Facilitate Prostate Cancer Metastasis</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">3181</span>– <span class="NLM_lpage">3193</span>, <span class="refDoi"> DOI: 10.1158/0008-5472.can-16-2797</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1158%2F0008-5472.can-16-2797" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2017&pages=3181-3193&author=R.+Malekauthor=R.+P.+Gajulaauthor=R.+D.+Williamsauthor=B.+Nghiemauthor=B.+W.+Simonsauthor=K.+Nugentauthor=H.+Wangauthor=K.+Taparraauthor=G.+Lemtiri-Chliehauthor=A.+R.+Yoonauthor=L.+Trueauthor=S.+S.+Anauthor=T.+L.+DeWeeseauthor=A.+E.+Rossauthor=E.+M.+Schaefferauthor=K.+J.+Pientaauthor=P.+J.+Hurleyauthor=C.+Morrisseyauthor=P.+T.+Tran&title=TWIST1-WDR5-+Hottip+Regulates+Hoxa9+Chromatin+to+Facilitate+Prostate+Cancer+Metastasis&doi=10.1158%2F0008-5472.can-16-2797"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.can-16-2797&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.can-16-2797%26sid%3Dliteratum%253Aachs%26aulast%3DMalek%26aufirst%3DR.%26aulast%3DGajula%26aufirst%3DR.%2BP.%26aulast%3DWilliams%26aufirst%3DR.%2BD.%26aulast%3DNghiem%26aufirst%3DB.%26aulast%3DSimons%26aufirst%3DB.%2BW.%26aulast%3DNugent%26aufirst%3DK.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DTaparra%26aufirst%3DK.%26aulast%3DLemtiri-Chlieh%26aufirst%3DG.%26aulast%3DYoon%26aufirst%3DA.%2BR.%26aulast%3DTrue%26aufirst%3DL.%26aulast%3DAn%26aufirst%3DS.%2BS.%26aulast%3DDeWeese%26aufirst%3DT.%2BL.%26aulast%3DRoss%26aufirst%3DA.%2BE.%26aulast%3DSchaeffer%26aufirst%3DE.%2BM.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DHurley%26aufirst%3DP.%2BJ.%26aulast%3DMorrissey%26aufirst%3DC.%26aulast%3DTran%26aufirst%3DP.%2BT.%26atitle%3DTWIST1-WDR5-%2520Hottip%2520Regulates%2520Hoxa9%2520Chromatin%2520to%2520Facilitate%2520Prostate%2520Cancer%2520Metastasis%26jtitle%3DCancer%2520Res.%26date%3D2017%26volume%3D77%26spage%3D3181%26epage%3D3193%26doi%3D10.1158%2F0008-5472.can-16-2797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, Y.-Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ru, G.-Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span> <span> </span><span class="NLM_article-title">High level of homeobox A9 and PBX homeobox 3 expression in gastric cancer correlates with poor prognosis</span>. <i>Oncol. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">5883</span>– <span class="NLM_lpage">5889</span>, <span class="refDoi"> DOI: 10.3892/ol.2017.6937</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.3892%2Fol.2017.6937" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29113222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC1M7oslOjtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2017&pages=5883-5889&author=Y.-Y.+Maauthor=Y.+Zhangauthor=X.-Z.+Mouauthor=Z.-C.+Liuauthor=G.-Q.+Ruauthor=E.+Li&title=High+level+of+homeobox+A9+and+PBX+homeobox+3+expression+in+gastric+cancer+correlates+with+poor+prognosis&doi=10.3892%2Fol.2017.6937"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">High level of homeobox A9 and PBX homeobox 3 expression in gastric cancer correlates with poor prognosis</span></div><div class="casAuthors">Ma Ying-Yu; Zhang Yuancheng; Li Erguang; Ma Ying-Yu; Mou Xiao-Zhou; Liu Zheng-Chuang; Ru Guo-Qing</div><div class="citationInfo"><span class="NLM_cas:title">Oncology letters</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">5883-5889</span>
        ISSN:<span class="NLM_cas:issn">1792-1074</span>.
    </div><div class="casAbstract">The homeobox protein homeobox (HOXA9) is a transcriptional factor that regulates patterning during embryogenesis and controls cell differentiation.  HOXA9 dysfunction has been implicated in certain cancers.  However, the role of HOXA9 in gastric cancer is poorly understood.  The present study investigated HOXA9 and its cofactor PBX homeobox 3 (PBX3) expression in patients with gastric cancer.  Paired tissue samples from 24 patients and paraffin embedded tissues of gastric cancer patients (104 males and 24 females) were included.  HOXA9 and PBX3 expression levels were determined by reverse transcription quantitative polymerase chain reaction in fresh tissues, and by immunohistochemical staining in paraffin embedded tissues.  The association between HOXA9/PBX3 expression and clinicopathological features was established.  The results demonstrated that HOXA9 and PBX3 mRNA levels were significantly upregulated (P=0.032 for HOXA9 and P=0.031 for PBX3) in gastric cancer tissue.  Immunohistochemical staining revealed that HOXA9 expression was associated with differentiation, lymph node metastasis and tumor-node-metastasis (TNM) stage, and PBX3 expression was associated with lymph node metastasis and TNM stage.  Correlation analysis revealed a high coincidental expression of HOXA9 and PBX3 levels in gastric cancer (r=0.391; P<0.001).  Survival analysis showed that high expression of HOXA9 or PBX3 was associated with poor survival of gastric cancer, and multivariate analysis using Cox's regression model showed that PBX3 expression was an independent prognostic factor in gastric cancer.  There was elevated expression of HOXA9 and PBX3 in gastric cancer patients, and high-level expression of those proteins was associated with poor prognosis of gastric cancer.  The present study underlines the significance of HOXA9/PBX3 in the development of gastric cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSUPVh83kF2SNK0qsdY6Nc1fW6udTcc2ebd1-01oHg_Bbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1M7oslOjtg%253D%253D&md5=668a323a7311cc1520e1906d58c8a94d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.3892%2Fol.2017.6937&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fol.2017.6937%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DY.-Y.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DMou%26aufirst%3DX.-Z.%26aulast%3DLiu%26aufirst%3DZ.-C.%26aulast%3DRu%26aufirst%3DG.-Q.%26aulast%3DLi%26aufirst%3DE.%26atitle%3DHigh%2520level%2520of%2520homeobox%2520A9%2520and%2520PBX%2520homeobox%25203%2520expression%2520in%2520gastric%2520cancer%2520correlates%2520with%2520poor%2520prognosis%26jtitle%3DOncol.%2520Lett.%26date%3D2017%26volume%3D14%26spage%3D5883%26epage%3D5889%26doi%3D10.3892%2Fol.2017.6937" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vijapurkar, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischbach, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brandts, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokoe, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largman, C.</span></span> <span> </span><span class="NLM_article-title">Protein Kinase C-Mediated Phosphorylation of the Leukemia-Associated HOXA9 Protein Impairs Its DNA Binding Ability and Induces Myeloid Differentiation</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3827</span>– <span class="NLM_lpage">3837</span>, <span class="refDoi"> DOI: 10.1128/mcb.24.9.3827-3837.2004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1128%2Fmcb.24.9.3827-3837.2004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15082777" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVyrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2004&pages=3827-3837&author=U.+Vijapurkarauthor=N.+Fischbachauthor=W.+Shenauthor=C.+Brandtsauthor=D.+Stokoeauthor=H.+J.+Lawrenceauthor=C.+Largman&title=Protein+Kinase+C-Mediated+Phosphorylation+of+the+Leukemia-Associated+HOXA9+Protein+Impairs+Its+DNA+Binding+Ability+and+Induces+Myeloid+Differentiation&doi=10.1128%2Fmcb.24.9.3827-3837.2004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinase C-mediated phosphorylation of the leukemia-associated HOXA9 protein impairs its DNA binding ability and induces myeloid differentiation</span></div><div class="casAuthors">Vijapurkar, Ulka; Fischbach, Neal; Shen, Weifang; Brandts, Christian; Stokoe, David; Lawrence, H. Jeffrey; Largman, Corey</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3827-3837</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">HOXA9 expression is a common feature of acute myeloid leukemia, and high-level expression is correlated with poor prognosis.  Moreover, HOXA9 overexpression immortalizes murine marrow progenitors that are arrested at a promyelocytic stage of differentiation when cultured and causes leukemia in recipient mice following transplantation of HOXA9 expressing bone marrow.  The mol. mechanisms underlying the physiol. functions and transforming properties of HOXA9 are poorly understood.  This study demonstrates that HOXA9 is phosphorylated by protein kinase C (PKC) and casein kinase II and that PKC mediates phosphorylation of purified HOXA9 on S204 as well as on T205, within a highly conserved consensus sequence, in the N-terminal region of the homeodomain.  S204 in the endogenous HOXA9 protein was phosphorylated in PLB985 myeloid cells, as well as in HOXA9-immortalized murine marrow cells.  This phosphorylation was enhanced by phorbol ester, a known inducer of PKC, and was inhibited by a specific PKC inhibitor.  PKC-mediated phosphorylation of S204 decreased HOXA9 DNA binding affinity in vitro and the ability of the endogenous HOXA9 to form cooperative DNA binding complexes with PBX.  PKC inhibition significantly reduced the phorbol-ester induced differentiation of the PLB985 hematopoietic cell line as well as HOXA9-immortalized murine bone marrow cells.  These data suggest that phorbol ester-induced myeloid differentiation is in part due to PKC-mediated phosphorylation of HOXA9, which decreases the DNA binding of the homeoprotein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJCVepTtX_hLVg90H21EOLACvtfcHk0lhOnp1tTmXrVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVyrs7k%253D&md5=93bc97c26d1c5a154965ef5acfa05fb4</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1128%2Fmcb.24.9.3827-3837.2004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.24.9.3827-3837.2004%26sid%3Dliteratum%253Aachs%26aulast%3DVijapurkar%26aufirst%3DU.%26aulast%3DFischbach%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DW.%26aulast%3DBrandts%26aufirst%3DC.%26aulast%3DStokoe%26aufirst%3DD.%26aulast%3DLawrence%26aufirst%3DH.%2BJ.%26aulast%3DLargman%26aufirst%3DC.%26atitle%3DProtein%2520Kinase%2520C-Mediated%2520Phosphorylation%2520of%2520the%2520Leukemia-Associated%2520HOXA9%2520Protein%2520Impairs%2520Its%2520DNA%2520Binding%2520Ability%2520and%2520Induces%2520Myeloid%2520Differentiation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2004%26volume%3D24%26spage%3D3827%26epage%3D3837%26doi%3D10.1128%2Fmcb.24.9.3827-3837.2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Faber, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krivtsov, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stubbs, M. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wright, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, T. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van den Heuvel-Eibrink, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaan, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Is Required for Survival in Human MLL-Rearranged Acute Leukemias</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">2375</span>– <span class="NLM_lpage">2385</span>, <span class="refDoi"> DOI: 10.1182/blood-2007-09-113597</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2007-09-113597" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=19056693" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2008&pages=2375-2385&author=J.+Faberauthor=A.+V.+Krivtsovauthor=M.+C.+Stubbsauthor=R.+Wrightauthor=T.+N.+Davisauthor=M.+van+den+Heuvel-Eibrinkauthor=C.+M.+Zwaanauthor=A.+L.+Kungauthor=S.+A.+Armstrong&title=HOXA9+Is+Required+for+Survival+in+Human+MLL-Rearranged+Acute+Leukemias&doi=10.1182%2Fblood-2007-09-113597"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1182%2Fblood-2007-09-113597&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2007-09-113597%26sid%3Dliteratum%253Aachs%26aulast%3DFaber%26aufirst%3DJ.%26aulast%3DKrivtsov%26aufirst%3DA.%2BV.%26aulast%3DStubbs%26aufirst%3DM.%2BC.%26aulast%3DWright%26aufirst%3DR.%26aulast%3DDavis%26aufirst%3DT.%2BN.%26aulast%3Dvan%2Bden%2BHeuvel-Eibrink%26aufirst%3DM.%26aulast%3DZwaan%26aufirst%3DC.%2BM.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DHOXA9%2520Is%2520Required%2520for%2520Survival%2520in%2520Human%2520MLL-Rearranged%2520Acute%2520Leukemias%26jtitle%3DBlood%26date%3D2008%26volume%3D113%26spage%3D2375%26epage%3D2385%26doi%3D10.1182%2Fblood-2007-09-113597" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Role of HOXA9 in Leukemia: Dysregulation, Cofactors and Essential Targets</span>. <i>Oncogene</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">1090</span>– <span class="NLM_lpage">1098</span>, <span class="refDoi"> DOI: 10.1038/onc.2015.174</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fonc.2015.174" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=26028034" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2015&pages=1090-1098&author=C.+T.+Collinsauthor=J.+L.+Hess&title=Role+of+HOXA9+in+Leukemia%3A+Dysregulation%2C+Cofactors+and+Essential+Targets&doi=10.1038%2Fonc.2015.174"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fonc.2015.174&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2015.174%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DC.%2BT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DRole%2520of%2520HOXA9%2520in%2520Leukemia%253A%2520Dysregulation%252C%2520Cofactors%2520and%2520Essential%2520Targets%26jtitle%3DOncogene%26date%3D2015%26volume%3D35%26spage%3D1090%26epage%3D1098%26doi%3D10.1038%2Fonc.2015.174" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Deregulation of the HOXA9/MEIS1 Axis in Acute Leukemia</span>. <i>Curr. Opin. Hematol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">354</span>– <span class="NLM_lpage">361</span>, <span class="refDoi"> DOI: 10.1097/moh.0000000000000245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1097%2Fmoh.0000000000000245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27258906" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XptFGmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2016&pages=354-361&author=C.+T.+Collinsauthor=J.+L.+Hess&title=Deregulation+of+the+HOXA9%2FMEIS1+Axis+in+Acute+Leukemia&doi=10.1097%2Fmoh.0000000000000245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Deregulation of the HOXA9/MEIS1 axis in acute leukemia</span></div><div class="casAuthors">Collins, Cailin T.; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Hematology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">354-361</span>CODEN:
                <span class="NLM_cas:coden">COHEF4</span>;
        ISSN:<span class="NLM_cas:issn">1065-6251</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">Purpose of review: HOXA9 is a homeodomain transcription factor that plays an essential role in normal hematopoiesis and acute leukemia, in which its overexpression is strongly correlated with poor prognosis.  The present review highlights recent advances in the understanding of genetic alterations leading to deregulation of HOXA9 and the downstream mechanisms of HOXA9-mediated transformation.  Recent findings: A variety of genetic alterations including MLL translocations, NUP98-fusions, NPM1 mutations, CDX deregulation, and MOZ-fusions lead to high-level HOXA9 expression in acute leukemias.  The mechanisms resulting in HOXA9 overexpression are beginning to be defined and represent attractive therapeutic targets.  Small mols. targeting MLL-fusion protein complex members, such as DOT1L and menin, have shown promising results in animal models, and a DOT1L inhibitor is currently being tested in clin. trials.  Essential HOXA9 cofactors and collaborators are also being identified, including transcription factors PU.1 and C/EBPα, which are required for HOXA9-driven leukemia.  HOXA9 targets including IGF1, CDX4, INK4A/INK4B/ARF, mir-21, and mir-196b and many others provide another avenue for potential drug development.  Summary: HOXA9 deregulation underlies a large subset of aggressive acute leukemias.  Understanding the mechanisms regulating the expression and activity of HOXA9, along with its crit. downstream targets, shows promise for the development of more selective and effective leukemia therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUxf970ShOmrVg90H21EOLACvtfcHk0lhOnp1tTmXrVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XptFGmtLc%253D&md5=02931b1f9cdee0f39e6ad8b71d97bf54</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1097%2Fmoh.0000000000000245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1097%252Fmoh.0000000000000245%26sid%3Dliteratum%253Aachs%26aulast%3DCollins%26aufirst%3DC.%2BT.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DDeregulation%2520of%2520the%2520HOXA9%252FMEIS1%2520Axis%2520in%2520Acute%2520Leukemia%26jtitle%3DCurr.%2520Opin.%2520Hematol.%26date%3D2016%26volume%3D23%26spage%3D354%26epage%3D361%26doi%3D10.1097%2Fmoh.0000000000000245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Is Critical in the Proliferation, Differentiation, and Malignancy of Leukaemia Cells Both in Vitro and in Vivo</span>. <i>Cell Biochem. Funct.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">440</span>, <span class="refDoi"> DOI: 10.1002/cbf.3293</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fcbf.3293" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28961318" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsFOnt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2017&pages=433-440&author=S.+Chenauthor=J.+Yuauthor=X.+Lvauthor=L.+Zhang&title=HOXA9+Is+Critical+in+the+Proliferation%2C+Differentiation%2C+and+Malignancy+of+Leukaemia+Cells+Both+in+Vitro+and+in+Vivo&doi=10.1002%2Fcbf.3293"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">HOXA9 is critical in the proliferation, differentiation, and malignancy of leukaemia cells both in vitro and in vivo</span></div><div class="casAuthors">Chen, Shibing; Yu, Juan; Lv, Xin; Zhang, Lijuan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Function</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">433-440</span>CODEN:
                <span class="NLM_cas:coden">CBFUDH</span>;
        ISSN:<span class="NLM_cas:issn">1099-0844</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Progress in the understanding of the mol. mechanism for acute myeloid leukemia is of great significance to generate new potential targets for treatment.  Recent studies showed that HOXA9, a homeodomain-contg. transcription factor, is commonly deregulated in acute leukemia.  In this study, we elucidated the direct correlation between HoxA9 expression and progression of leukemia using 2 different types of leukemia cells HL-60 and MOLT-3.  The HoxA9 expression level was decreased in leukemia cells with the treatment of all-trans retinoic acid or arsenic trioxide (As2O3).  Downregulation of HoxA9 could impair the proliferation and promote the leukemia cell death.  HoxA9 silencing also potentiated the differentiation of leukemia cells, and in vivo studies demonstrated that HoxA9 downregulation could interfere the tumor growth.  Interestingly, HoxA9 silencing also led to the alteration in miRNA expression, mediating the promoting effect on the leukemia cell differentiation.  Therefore, this work provided a promising and potentially efficient target to leukemia treatment, indicating that HoxA9 is likely to be an ideal candidate in the gene therapy against acute myeloid leukemia.  In this study, we elucidated the crit. role of HoxA9 in the proliferation and differentiation of leukemia cells both in vitro and in vivo.  The effect of HoxA9 modulation was correlated with the clin. effect of all-trans retinoic acid and As2O3.  Furthermore, HoxA9 also regulated the miRNA expression, controlling the leukemia cell differentiation.  Therefore, this work provided new insights into mol. mechanism underlying the leukemia treatment, potentially putting forward a brand new target to the gene therapy against leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTeka4BsTWQ7Vg90H21EOLACvtfcHk0lhfGraZnatfjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsFOnt7fM&md5=9bc59f50d3a5dfbe1a31c527e0ef30b5</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1002%2Fcbf.3293&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbf.3293%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DLv%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DL.%26atitle%3DHOXA9%2520Is%2520Critical%2520in%2520the%2520Proliferation%252C%2520Differentiation%252C%2520and%2520Malignancy%2520of%2520Leukaemia%2520Cells%2520Both%2520in%2520Vitro%2520and%2520in%2520Vivo%26jtitle%3DCell%2520Biochem.%2520Funct.%26date%3D2017%26volume%3D35%26spage%3D433%26epage%3D440%26doi%3D10.1002%2Fcbf.3293" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valerio, D. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisold, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sinha, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koche, R. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brien, G. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, C. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsieh, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ernst, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span> <span> </span><span class="NLM_article-title">NUP98 Fusion Proteins Interact with the NSL and MLL1 Complexes to Drive Leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">863</span>– <span class="NLM_lpage">878</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2016.10.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccell.2016.10.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27889185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFGgtL%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=863-878&author=H.+Xuauthor=D.+G.+Valerioauthor=M.+E.+Eisoldauthor=A.+Sinhaauthor=R.+P.+Kocheauthor=W.+Huauthor=C.-W.+Chenauthor=S.+H.+Chuauthor=G.+L.+Brienauthor=C.+Y.+Parkauthor=J.+J.+Hsiehauthor=P.+Ernstauthor=S.+A.+Armstrong&title=NUP98+Fusion+Proteins+Interact+with+the+NSL+and+MLL1+Complexes+to+Drive+Leukemogenesis&doi=10.1016%2Fj.ccell.2016.10.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">NUP98 F1usion Proteins Interact with the NSL and MLL1 Complexes to Drive Leukemogenesis</span></div><div class="casAuthors">Xu, Haiming; Valerio, Daria G.; Eisold, Meghan E.; Sinha, Amit; Koche, Richard P.; Hu, Wenhuo; Chen, Chun-Wei; Chu, S. Haihua; Brien, Gerard L.; Park, Christopher Y.; Hsieh, James J.; Ernst, Patricia; Armstrong, Scott A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">863-878</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The nucleoporin 98 gene (NUP98) is fused to a variety of partner genes in multiple hematopoietic malignancies.  Here, we demonstrate that NUP98 fusion proteins, including NUP98-HOXA9 (NHA9), NUP98-HOXD13 (NHD13), NUP98-NSD1, NUP98-PHF23, and NUP98-TOP1 phys. interact with mixed lineage leukemia 1 (MLL1) and the non-specific lethal (NSL) histone-modifying complexes.  Chromatin immunopptn. sequencing illustrates that NHA9 and MLL1 co-localize on chromatin and are found assocd. with Hox gene promoter regions.  Furthermore, MLL1 is required for the proliferation of NHA9 cells in vitro and in vivo.  Inactivation of MLL1 leads to decreased expression of genes bound by NHA9 and MLL1 and reverses a gene expression signature found in NUP98-rearranged human leukemias.  Our data reveal a mol. dependency on MLL1 function in NUP98-fusion-driven leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrX4BUrzPjB5bVg90H21EOLACvtfcHk0lhfGraZnatfjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFGgtL%252FO&md5=c1baef1125dc36f2d288de70d83640f9</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.10.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.10.019%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DH.%26aulast%3DValerio%26aufirst%3DD.%2BG.%26aulast%3DEisold%26aufirst%3DM.%2BE.%26aulast%3DSinha%26aufirst%3DA.%26aulast%3DKoche%26aufirst%3DR.%2BP.%26aulast%3DHu%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DC.-W.%26aulast%3DChu%26aufirst%3DS.%2BH.%26aulast%3DBrien%26aufirst%3DG.%2BL.%26aulast%3DPark%26aufirst%3DC.%2BY.%26aulast%3DHsieh%26aufirst%3DJ.%2BJ.%26aulast%3DErnst%26aufirst%3DP.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26atitle%3DNUP98%2520Fusion%2520Proteins%2520Interact%2520with%2520the%2520NSL%2520and%2520MLL1%2520Complexes%2520to%2520Drive%2520Leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D863%26epage%3D878%26doi%3D10.1016%2Fj.ccell.2016.10.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rio-Machin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gómez-López, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muñoz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Martinez, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maiques-Diaz, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alvarez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salgado, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shrestha, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Torres-Ruiz, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haferlach, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larráyoz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calasanz, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fitzgibbon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cigudosa, J. C.</span></span> <span> </span><span class="NLM_article-title">The Molecular Pathogenesis of the NUP98-HOXA9 Fusion Protein in Acute Myeloid Leukemia</span>. <i>Leukemia</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2000</span>– <span class="NLM_lpage">2005</span>, <span class="refDoi"> DOI: 10.1038/leu.2017.194</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fleu.2017.194" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28630438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFOrs73F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2017&pages=2000-2005&author=A.+Rio-Machinauthor=G.+G%C3%B3mez-L%C3%B3pezauthor=J.+Mu%C3%B1ozauthor=F.+Garcia-Martinezauthor=A.+Maiques-Diazauthor=S.+Alvarezauthor=R.+N.+Salgadoauthor=M.+Shresthaauthor=R.+Torres-Ruizauthor=C.+Haferlachauthor=M.+J.+Larr%C3%A1yozauthor=M.+J.+Calasanzauthor=J.+Fitzgibbonauthor=J.+C.+Cigudosa&title=The+Molecular+Pathogenesis+of+the+NUP98-HOXA9+Fusion+Protein+in+Acute+Myeloid+Leukemia&doi=10.1038%2Fleu.2017.194"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The molecular pathogenesis of the NUP98-HOXA9 fusion protein in acute myeloid leukemia</span></div><div class="casAuthors">Rio-Machin, A.; Gomez-Lopez, G.; Munoz, J.; Garcia-Martinez, F.; Maiques-Diaz, A.; Alvarez, S.; Salgado, R. N.; Shrestha, M.; Torres-Ruiz, R.; Haferlach, C.; Larrayoz, M. J.; Calasanz, M. J.; Fitzgibbon, J.; Cigudosa, J. C.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2000-2005</span>CODEN:
                <span class="NLM_cas:coden">LEUKED</span>;
        ISSN:<span class="NLM_cas:issn">0887-6924</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recurrent chromosomal translocations are common initiation events and have provided important insights into the pathogenesis of AML, paving the way for the introduction of novel targeted therapies.  However, clin. outcomes, in particular for patients with adverse cytogenetic features remain suboptimal.  The chromosomal translocation t(7;11)(p15, p15), encoding the fusion protein NUP98-HOXA9 (NHA9), is a rare poor risk cytogenetic event in AML assocd. with a particularly poor prognosis.  NHA9 brings the FG repeat-rich portion of the nucleoporin NUP98 upstream of the homeodomain and PBX heterodimerization domains of HOXA9, and acts as oncogenic transcription factor.1.  The pathogenic events underlying NHA9 remain poorly understood and herein, we aim to characterize the downstream mediators of this oncoprotein by detg. the effects of the fusion using human cellular models.  We set out initially to compare the DNA binding sites of NHA9, HOXA9 and NUP98, by forced expression of these genes alone or the corresponding fusion gene by retroviral transduction of HEK93FT cell line and cord blood-isolated human hematopoietic progenitors (hHP).  ChIP-seq anal. in the HEK293FT cellular model identified 4471 significant genomic regions (false discovery rate (FDR) <0.05) as target sites of the fusion protein, all located within -5/+ kb from the annotated transcription start site (TSS) (Supplementary Figure S1A).  They correspond to 1368 genes and 17 miRNAs (Supplementary Table S1) of which 399 genes were also shown to be common targets of HOXA9 and 4 of NUP98 (Figure 1a, Supplementary Table S2, Supplementary Figures S1C-D) (Supplementary methods) (Data deposited in GEO http://www.ncbi.nlm.nih.gov/geo/, accession no.: GSE62587).  Ingenuity pathway anal. of the NHA9 target series demonstrated a significant enrichment of pathways assocd. with tumorigenesis and leukemic differentiation (Supplementary Figure S1B).  We next performed a detailed sequence anal. of the NHA9 binding sites using the MEME-ChIP algorithm and detected a significant overlap with binding of several HOX genes, including HOXA9, supporting a role for this homeodomain in the DNA binding of NHA9.  Strikingly, NHA9 sites were enriched for a novel binding motif, CA/gTTT, that was present in one-third (n=1421) of all NHA9 ChIP-seq regions (Supplementary Table S3).  This motif had not been previously assocd. with any known transcription factor and was not obsd. in wild type HOXA9 or NUP98 binding site expts., suggesting that it is specific to NHA9 DNA binding.  MEME-ChIP (SpaMO) was used to identify significant co-occurrences of other known DNA binding motifs with this novel NHA9 DNA binding motif.  Binding motifs corresponding to 12 transcription factors, including other HOX family proteins such as HOXB7 or HOXD11, were found to be overrepresented within the region adjacent to CA/gTTT (Supplementary Table S4), suggesting a possible functional cooperation with the fusion oncoprotein.  As the NHA9 target motifs are preferentially located more than 1 kb upstream/downstream of the TSS (Supplementary Figure S1A), we reasoned that NHA9 binding may coincide with particular enhancer elements.  A similar distribution was also found for the identified HOXA9 target regions whereas NUP98 binding sites were mostly located within promoters, both in agreement with previous studies.2, 3 We selected eight leukemia-related genes (MEIS1, HOXA9, PBX3, MET, BRAF, AF9, PTEN and NF1) identified as part of our NHA9 ChIP-seq expts., for locus specific qChIP studies.  A significant enrichment of H3K4me1, a chromatin mark that predicts poised and active enhancers, and RNA Polymerase II (PolII), which is consistent with the presence of the active form of the enhancers,4, 5 was shown within the NHA9 binding sites upstream of the eight genes (Figure 1b and Supplementary Figure S1E).  NHA9 expression levels were demonstrated to be comparable in our two cellular models (HEK293FT and hHP) (Supplementary Figure S1G).  Accordingly, we validated the ChIP-seq results in the HEK293FT model (Supplementary Figure S1F) using the same set of eight NHA9 target genes and also demonstrated binding of NHA9 to the eight enhancers in our second model system of NHA9-expressing hHP cells (Figure 1c), allowing us to confirm these findings in primary human hematopoiesis.  We next focused attention on the transcription factors MEIS1, HOXA9 and PBX3, as their overexpression is significantly related to adverse prognosis in AML (The Cancer Genome Atlas;6Supplementary Figure S1H) and were previously reported to drive leukemogenesis through the formation of a transcriptional activator complex.7.  To test the importance of these three transcription factors in NHA9 pathogenesis, we completed reporter assays in HEK293FT cells by cloning the identified enhancers of MEIS1, HOXA9 or PBX3 into a luciferase reporter vector.  A significant 1.6-2.8 fold induction in luciferase activity was obsd. when NHA9 was co-expressed for all three enhancers, indicating a direct induction of MEIS1, HOXA9 and PBX3 expression through the NHA9 interaction with their corresponding regulatory regions (Figure 1d) (Supplementary Methods).  This observation was accompanied by upregulation of all three transcription factors and of three of their known target genes (MYB, MEF2C and FLT3)7 in NHA9-expressing hHP cells (Figure 1e and Supplementary Figure S1I).  Gene Expression Profiling performed in three independent NHA9-expressing hHP clones and AMLs from five patients with t(7;11)(p15,p15), confirmed MEIS1-HOXA9-PBX3 overexpression and it was further validated by RT-qPCR anal. in three addnl. NHA9 primary samples (Supplementary Figure S2A).  These observations suggested that the NHA9-expressing hHP cells can be sensitive to HXR9, a specific peptide inhibitor of HOXA9 and PBX3 interaction that leads to disruption of the MEIS1-HOXA9-PBX3 complex.8.  We tested this hypothesis by treating these cells with HXR9 that resulted in a selective decrease in their viability (Figure 1f and Supplementary Figure S2B-D) (Supplementary Methods) without affecting cell differentiation (data not shown), therefore confirming the relevance of these downstream mediators in driving the oncogenic activity of NHA9.  In order to explore other mechanisms driving NHA9 pathogenesis and to better understand its role in transcriptional regulation, we interrogated our ChIP-seq and gene expression profiling data, which revealed both activation and repression of gene expression induced by this fusion oncoprotein (Figure 2a).  The cooperation of MLL1 and CRM1 with NHA9 in the upregulation of some target genes has been shown recently by Xu et al.,9, 10 which was also supported by comparing NHA9 target genes identified in our ChIP-seq expts. with MLL1 and CRM1 targets.  We found that 25% and 35% of NHA9 target genes were also in common with MLL1 and CRM1 target genes, resp. (Supplementary Figure S2E).  Notably, 151 target genes, including MEIS1 and HOXA9, were shared by all three proteins (NHA9, MLL1 and CRM1), suggesting a possible cooperation among these transcription factors in NHA9-driven leukemias.  It has also been reported that NUP98, through its FG repeat domain, may interact with transcriptional activator p300 and repressor HDACs,11 allowing us to postulate that transcriptional effects of NHA9 in enhancers could be mediated by these regulators.  We first demonstrated NHA9 binding to both p300 and HDAC1 by co-immunopptn. expts. (Figure 2b) (Supplementary Methods) and went on to examine their binding potential in a panel of eight regulatory regions of NHA9 target genes (four upregulated and four downregulated target genes) in the presence of the fusion protein by qChIP.  These expts. demonstrated selective binding of p300 to the regulatory regions of the upregulated genes MEIS1, HOXA9, PBX3 and AFF3 (Figure 2c), and of HDAC1 to the downregulated genes BIRC3, SMAD1, FILIP1L and PTEN (Figure 2d).  Altogether this data suggests that p300 and HDAC1 are selectively recruited by NHA9 at enhancer regions to modulate the expression of genes involved in leukemogenesis.  As the interaction of NHA9 with HDAC1/2 was validated by mass spectrometry anal. using the NHA9-expressing HEK293FT model (Proteomics data have been deposited on the ProteomeXchange Consortium via the PRIDE partner repository, data set identifier PXD001828) (Supplementary Methods), we had a mol. rationale for testing HDAC inhibitors (HDACi) in NHA9 AML.  It has to be noted that LBH589 did not induce differentiation in NHA9-expressing cells as no significant changes in the no. of CD11b pos. cells were obsd. by flow cytometry anal. post treatment (data not shown).  These observations are in accordance with a recent report suggesting the combination of COX or DNMT inhibitors with HDACi for treatment of NHA9 AML patients,16 however in this study we identified the mol. rationale for HDACi therapy as well as a panel of target genes downstream of NHA9 that can be used as biomarkers for response to this treatment.  However, the biol. consequences of this therapy, as well as the best dosage-time relation for the translation into clinics need to be further investigated.  In summary, NHA9 deregulates the expression of key leukemic genes, including MEIS1-HOXA9-PBX3 complex, through the enhancer binding and the direct interaction of the fusion protein with HDAC and p300 transcriptional regulators.  The oncogenic effects of NHA9 can be overcome by HDACi treatment, demonstrating a significant inhibitory effects against NHA9-driven leukemic cells and suggesting a novel approach to treatment of this high-risk group of patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjzGKKJ18_YLVg90H21EOLACvtfcHk0lhfGraZnatfjg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFOrs73F&md5=57d51492b5bbe6d50c47f97825835030</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fleu.2017.194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fleu.2017.194%26sid%3Dliteratum%253Aachs%26aulast%3DRio-Machin%26aufirst%3DA.%26aulast%3DG%25C3%25B3mez-L%25C3%25B3pez%26aufirst%3DG.%26aulast%3DMu%25C3%25B1oz%26aufirst%3DJ.%26aulast%3DGarcia-Martinez%26aufirst%3DF.%26aulast%3DMaiques-Diaz%26aufirst%3DA.%26aulast%3DAlvarez%26aufirst%3DS.%26aulast%3DSalgado%26aufirst%3DR.%2BN.%26aulast%3DShrestha%26aufirst%3DM.%26aulast%3DTorres-Ruiz%26aufirst%3DR.%26aulast%3DHaferlach%26aufirst%3DC.%26aulast%3DLarr%25C3%25A1yoz%26aufirst%3DM.%2BJ.%26aulast%3DCalasanz%26aufirst%3DM.%2BJ.%26aulast%3DFitzgibbon%26aufirst%3DJ.%26aulast%3DCigudosa%26aufirst%3DJ.%2BC.%26atitle%3DThe%2520Molecular%2520Pathogenesis%2520of%2520the%2520NUP98-HOXA9%2520Fusion%2520Protein%2520in%2520Acute%2520Myeloid%2520Leukemia%26jtitle%3DLeukemia%26date%3D2017%26volume%3D31%26spage%3D2000%26epage%3D2005%26doi%3D10.1038%2Fleu.2017.194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Borrow, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shearman, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanton, V. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becher, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dubé, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katz, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwong, Y. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morris, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohyashiki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toyama, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rowley, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Housman, D. E.</span></span> <span> </span><span class="NLM_article-title">The t(7;11)(p15;p15) translocation in acute myeloid leukaemia fuses the genes for nucleoporin NUP96 and class I homeoprotein HOXA9</span>. <i>Nat. Genet.</i> <span class="NLM_year" style="font-weight: bold;">1996</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">167</span>, <span class="refDoi"> DOI: 10.1038/ng0296-159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fng0296-159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=8563754" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK28XosFCmtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1996&pages=159-167&author=J.+Borrowauthor=A.+M.+Shearmanauthor=V.+P.+Stantonauthor=R.+Becherauthor=T.+Collinsauthor=A.+J.+Williamsauthor=I.+Dub%C3%A9author=F.+Katzauthor=Y.+L.+Kwongauthor=C.+Morrisauthor=K.+Ohyashikiauthor=K.+Toyamaauthor=J.+Rowleyauthor=D.+E.+Housman&title=The+t%287%3B11%29%28p15%3Bp15%29+translocation+in+acute+myeloid+leukaemia+fuses+the+genes+for+nucleoporin+NUP96+and+class+I+homeoprotein+HOXA9&doi=10.1038%2Fng0296-159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">The t(7;11)(p15;p15) translocation in acute myeloid leukemia fuses the genes for nucleoporin NUP98 and class I homeoprotein HOXA9</span></div><div class="casAuthors">Borrow, Julian; Shearman, Amanda M.; Stanton, Vincent P., Jr.; Becher, Reinhard; Collins, Tucker; Williams, Amy J.; Dube, Ian; Katz, Fay; Kwong, Yok L.; et al.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">159-67</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Co.</span>)
        </div><div class="casAbstract">The t(7,11)(p15;p15) translocation is a recurrent chromosomal abnormality assocd. primarily with acute myeloid leukemia (FAB M2 and M4).  The authors present here the mol. definition of this translocation.  On chromosome 7 positional cloning revealed the consistent rearrangement of the HOXA9 gene, which encodes a class I homeodomain protein potentially involved in myeloid differentiation.  On chromosome 11 the translocation targets the human homolog of NUP98, a member of the GLFG nucleoporin family.  Chimeric messages spliced over the breakpoint fuse the GLFG repeat domains of NUP98 in-frame to the HOXA9 homeobox.  The predicted NUP98-HOXA9 fusion protein may promote leukemogenesis through inhibition of HOXA9-mediated terminal differentiation and/or aberrant nucleocytoplasmic transport.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7t1VxF9trU7Vg90H21EOLACvtfcHk0lhRUIiS9HW4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XosFCmtg%253D%253D&md5=15ca1aeb405fc5cc0ad416227a68d826</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1038%2Fng0296-159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng0296-159%26sid%3Dliteratum%253Aachs%26aulast%3DBorrow%26aufirst%3DJ.%26aulast%3DShearman%26aufirst%3DA.%2BM.%26aulast%3DStanton%26aufirst%3DV.%2BP.%26aulast%3DBecher%26aufirst%3DR.%26aulast%3DCollins%26aufirst%3DT.%26aulast%3DWilliams%26aufirst%3DA.%2BJ.%26aulast%3DDub%25C3%25A9%26aufirst%3DI.%26aulast%3DKatz%26aufirst%3DF.%26aulast%3DKwong%26aufirst%3DY.%2BL.%26aulast%3DMorris%26aufirst%3DC.%26aulast%3DOhyashiki%26aufirst%3DK.%26aulast%3DToyama%26aufirst%3DK.%26aulast%3DRowley%26aufirst%3DJ.%26aulast%3DHousman%26aufirst%3DD.%2BE.%26atitle%3DThe%2520t%25287%253B11%2529%2528p15%253Bp15%2529%2520translocation%2520in%2520acute%2520myeloid%2520leukaemia%2520fuses%2520the%2520genes%2520for%2520nucleoporin%2520NUP96%2520and%2520class%2520I%2520homeoprotein%2520HOXA9%26jtitle%3DNat.%2520Genet.%26date%3D1996%26volume%3D12%26spage%3D159%26epage%3D167%26doi%3D10.1038%2Fng0296-159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ye, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, H.</span></span> <span> </span><span class="NLM_article-title">Silencing of HMGA2 Reverses Retardance of Cell Differentiation in Human Myeloid Leukaemia</span>. <i>Br. J. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>118</i></span>,  <span class="NLM_fpage">405</span>– <span class="NLM_lpage">415</span>, <span class="refDoi"> DOI: 10.1038/bjc.2017.403</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fbjc.2017.403" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29384529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitlGmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2018&pages=405-415&author=L.+Tanauthor=H.+Xuauthor=G.+Chenauthor=X.+Weiauthor=B.+Yuauthor=J.+Yeauthor=L.+Xuauthor=H.+Tan&title=Silencing+of+HMGA2+Reverses+Retardance+of+Cell+Differentiation+in+Human+Myeloid+Leukaemia&doi=10.1038%2Fbjc.2017.403"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing of HMGA2 reverses retardance of cell differentiation in human myeloid leukaemia</span></div><div class="casAuthors">Tan, Li; Xu, Hongfa; Chen, Guoshu; Wei, Xiaoping; Yu, Baodan; Ye, Jingmei; Xu, Lihua; Tan, Huo</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Cancer</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">405-415</span>CODEN:
                <span class="NLM_cas:coden">BJCAAI</span>;
        ISSN:<span class="NLM_cas:issn">0007-0920</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Background: High-mobility group AT-hook 2 (HMGA2) may serve as an architectural transcription factor, and it can regulate a range of normal biol. processes including proliferation and differentiation.  Upregulation of HMGA2 expression is correlated to the undifferentiated phenotype of immature leukemic cells.  However, the underlying mechanism of HMGA2-dependent myeloid differentiation blockage in leukemia is unknown.  Methods: To reveal the role and mechanism of HMGA2 in differentiation arrest of myeloid leukemia cells, the quant. expression of HMGA2 and homeobox A9 (HOXA9) was analyzed by real-time PCR (qRT-PCR).  The regulatory function of HMGA2 in blockage of differentiation in human myeloid leukemia was investigated through in vitro assays (XTT assay, May-Grunwald-Giemsa, flow cytometry anal. and western blot).  Results: We found that the expression of HMGA2 and HOXA9 was reduced during the process of granulo-monocytic maturation of acute myeloid leukemia (AML) cells, knockdown of HMGA2 promotes terminal (granulocytic and monocytic) differentiation of myeloid leukemia primary blasts and cell lines, and HOXA9 was significantly downregulated in leukemic cells with knockdown of HMGA2.  Downregulation of HOXA9 in myeloid leukemia cells led to increased differentiation capacity in vitro.  Conclusions: Our data suggest that increased expression of HMGA2 represents a possible new mechanism of myeloid differentiation blockage of leukemia.  Aberrant expression of HMGA2 may enhance HOXA9-dependent leukaemogenesis and myeloid leukemia phenotype.  Disturbance of the HMGA2-HOXA9 pathway is probably a therapeutic strategy in myeloid leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzgIDafLy8n7Vg90H21EOLACvtfcHk0lhRUIiS9HW4aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitlGmsA%253D%253D&md5=87c0d65a8a8f1e9cc3228f8657a95628</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fbjc.2017.403&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fbjc.2017.403%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DWei%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DL.%26aulast%3DTan%26aufirst%3DH.%26atitle%3DSilencing%2520of%2520HMGA2%2520Reverses%2520Retardance%2520of%2520Cell%2520Differentiation%2520in%2520Human%2520Myeloid%2520Leukaemia%26jtitle%3DBr.%2520J.%2520Cancer%26date%3D2018%26volume%3D118%26spage%3D405%26epage%3D415%26doi%3D10.1038%2Fbjc.2017.403" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Calvo, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sykes, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pasillas, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamps, M. P.</span></span> <span> </span><span class="NLM_article-title">Hoxa9 Immortalizes a Granulocyte-Macrophage Colony-Stimulating Factor-Dependent Promyelocyte Capable of Biphenotypic Differentiation to Neutrophils or Macrophages, Independent of Enforced Meis Expression</span>. <i>Mol. Cell. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3274</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1128/mcb.20.9.3274-3285.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1128%2Fmcb.20.9.3274-3285.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10757811" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD3cXivFSrsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=3274-3285&author=K.+R.+Calvoauthor=D.+B.+Sykesauthor=M.+Pasillasauthor=M.+P.+Kamps&title=Hoxa9+Immortalizes+a+Granulocyte-Macrophage+Colony-Stimulating+Factor-Dependent+Promyelocyte+Capable+of+Biphenotypic+Differentiation+to+Neutrophils+or+Macrophages%2C+Independent+of+Enforced+Meis+Expression&doi=10.1128%2Fmcb.20.9.3274-3285.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Hoxa9 immortalizes a granulocyte-macrophage colony-stimulating factor-dependent promyelocyte capable of biphenotypic differentiation to neutrophils or macrophages, independent of enforced Meis expression</span></div><div class="casAuthors">Calvo, Katherine R.; Sykes, David B.; Pasillas, Martina; Kamps, Mark P.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3274-3285</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The genes encoding Hoxa9 and Meis1 are transcriptionally coactivated in a subset of acute myeloid leukemia (AML) in mice.  In marrow reconstitution expts., coexpression of both genes produces rapid AML, while neither gene alone generates overt leukemia.  Although Hoxa9 and Meis1 can bind DNA as heterodimers, both can also heterodimerize with Pbx proteins.  Thus, while their coactivation may result from the necessity to bind promoters as heterodimers, it may also result from the necessity of altering independent biochem. pathways that cooperate to generate AML, either as monomers or as heterodimers with Pbx proteins.  Here we demonstrate that constitutive expression of Hoxa9 in primary murine marrow immortalizes a late myelomonocytic progenitor, preventing it from executing terminal differentiation to granulocytes or monocytes in the presence of granulocyte-macrophage colony-stimulating factor (GM-CSF) or interleukin-3.  This immortalized pheno-type is achieved in the absence of endogenous or exogenous Meis gene expression.  The Hoxa9-immortalized progenitor exhibited a promyelocytic transcriptional profile, expressing PU.1, AML1, c-Myb, C/EBP alpha, and C/EBP epsilon as well as their target genes, the receptors for GM-CSF, G-CSF, and M-CSF and the primary granule proteins myeloperoxidase and neutrophil elastase.  G-CSF obviated the differentiation block of Hoxa9, inducing neutrophilic differentiation with accompanying expression of neutrophil gelatinase B and upregulation of gp91phox.  M-CSF also obviated the differentiation block, inducing monocytic differentiation with accompanying expression of the macrophage acetyl-low-d. lipoprotein scavenger receptor and F4/80 antigen.  Versions of Hoxa9 lacking the ANWL Pbx interaction motif (PIM) also immortalized a promyelocytic progenitor with intrinsic biphenotypic differentiation potential.  Therefore, Hoxa9 evokes a cytokine-selective block in differentiation by a mechanism that does not require Meis gene expression or interaction with Pbx through the PIM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXqXOPNl4HbLVg90H21EOLACvtfcHk0ljRzv1WFO_BeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXivFSrsLs%253D&md5=4d2cbadfcb20141851ac4ab45bdd7eef</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1128%2Fmcb.20.9.3274-3285.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252Fmcb.20.9.3274-3285.2000%26sid%3Dliteratum%253Aachs%26aulast%3DCalvo%26aufirst%3DK.%2BR.%26aulast%3DSykes%26aufirst%3DD.%2BB.%26aulast%3DPasillas%26aufirst%3DM.%26aulast%3DKamps%26aufirst%3DM.%2BP.%26atitle%3DHoxa9%2520Immortalizes%2520a%2520Granulocyte-Macrophage%2520Colony-Stimulating%2520Factor-Dependent%2520Promyelocyte%2520Capable%2520of%2520Biphenotypic%2520Differentiation%2520to%2520Neutrophils%2520or%2520Macrophages%252C%2520Independent%2520of%2520Enforced%2520Meis%2520Expression%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D3274%26epage%3D3285%26doi%3D10.1128%2Fmcb.20.9.3274-3285.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Passegue, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fong, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Largman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawrence, H. J.</span></span> <span> </span><span class="NLM_article-title">Evidence That the Pim1 Kinase Gene Is a Direct Target of HOXA9</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">4732</span>– <span class="NLM_lpage">4738</span>, <span class="refDoi"> DOI: 10.1182/blood-2006-08-043356</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2006-08-043356" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=17327400" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmt1Ogtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2007&pages=4732-4738&author=Y.-L.+Huauthor=E.+Passegueauthor=S.+Fongauthor=C.+Largmanauthor=H.+J.+Lawrence&title=Evidence+That+the+Pim1+Kinase+Gene+Is+a+Direct+Target+of+HOXA9&doi=10.1182%2Fblood-2006-08-043356"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Evidence that the Pim1 kinase gene is a direct target of HOXA9</span></div><div class="casAuthors">Hu, Yu-Long; Passegue, Emmanuelle; Fong, Stephen; Largman, Corey; Lawrence, Hugh Jeffrey</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4732-4738</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">The HOXA9 homeoprotein exerts dramatic effects in hematopoiesis.  Enforced expression of HOXA9 enhances proliferation of primitive blood cells, expands hematopoietic stem cells (HSCs), and leads to myeloid leukemia.  Conversely, loss of HOXA9 inhibits proliferation and impairs HSC function.  The pathways by which HOXA9 acts are largely unknown, and although HOXA9 is a transcription factor, few direct target genes have been identified.  Our previous study suggested that HOXA9 pos. regulates Pim1, an oncogenic kinase.  The hematol. phenotypes of Hoxa9- and Pim1-deficient animals are strikingly similar.  Here we show that HOXA9 protein binds to the Pim1 promoter and induces Pim1 mRNA and protein in hematopoietic cells.  Pim1 protein is diminished in Hoxa9-/- cells, and Hoxa9 and Pim1 mRNA levels track together in early hematopoietic compartments.  Induction of Pim1 protein by HOXA9 increases the phosphorylation and inactivation of the proapoptotic BAD protein, a target of Pim1.  Hoxa9-/- cells show increased apoptosis and decreased proliferation, defects that are ameliorated by reintroduction of Pim1.  Thus Pim1 appears to be a direct transcriptional target of HOXA9 and a mediator of its antiapoptotic and proproliferative effects in early cells.  Since HOXA9 is frequently up-regulated in acute myeloid leukemia, Pim1 may be a therapeutic target in human disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo1_SUxER0HLVg90H21EOLACvtfcHk0ljRzv1WFO_BeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmt1Ogtbw%253D&md5=c2d36102fe3085e133c10dabf206bc50</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1182%2Fblood-2006-08-043356&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2006-08-043356%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.-L.%26aulast%3DPassegue%26aufirst%3DE.%26aulast%3DFong%26aufirst%3DS.%26aulast%3DLargman%26aufirst%3DC.%26aulast%3DLawrence%26aufirst%3DH.%2BJ.%26atitle%3DEvidence%2520That%2520the%2520Pim1%2520Kinase%2520Gene%2520Is%2520a%2520Direct%2520Target%2520of%2520HOXA9%26jtitle%3DBlood%26date%3D2007%26volume%3D109%26spage%3D4732%26epage%3D4738%26doi%3D10.1182%2Fblood-2006-08-043356" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Breitinger, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maethner, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia-Cuellar, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slany, R. K.</span></span> <span> </span><span class="NLM_article-title">The Homeodomain Region Controls the Phenotype of HOX-Induced Murine Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4018</span>– <span class="NLM_lpage">4027</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-10-384685</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2011-10-384685" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22990017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsleqsL%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4018-4027&author=C.+Breitingerauthor=E.+Maethnerauthor=M.-P.+Garcia-Cuellarauthor=R.+K.+Slany&title=The+Homeodomain+Region+Controls+the+Phenotype+of+HOX-Induced+Murine+Leukemia&doi=10.1182%2Fblood-2011-10-384685"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">The homeodomain region controls the phenotype of HOX-induced murine leukemia</span></div><div class="casAuthors">Breitinger, Constanze; Maethner, Emanuel; Garcia-Cuellar, Maria-Paz; Slany, Robert K.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4018-4027</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">HOX proteins are widely involved in hematopoietic development.  These transcription factors combine a conserved DNA-binding homeobox with a divergent N-terminus that mediates interaction with variable cofactors.  The resulting combinatorial diversity is thought to be responsible for mammalian HOX specificity.  Contrasting this proposed mechanism for normal HOX function, here we demonstrate that, in the context of hematopoietic immortalization and leukemogenesis, individual HOX properties are governed almost exclusively by the homeodomain.  Swap expts. between HOXA1 and HOXA9, 2 members of nonrelated paralog groups, revealed that gene expression patterns of HOX transformed cells in vitro are detd. by the nature of the homeodomain.  Similar results were seen in vivo during HOX-mediated leukemogenesis.  An exchange of the homeodomains was sufficient to convert the slow, low-penetrance phenotype of HOXA1-induced leukemia to the aggressive fast-acting disease elicited by HOXA9 and vice versa.  Mutation and deletion studies identified several subregions within the DNA binding domain responsible for paralog specificity.  Previously defined binding sites for PBX cofactors within the exchangeable, nonhomeobox segment were dispensable for in vitro oncogenic HOX activity but affected in vivo disease development.  The transcriptional activator domain shared by HOXA1 and HOXA9 at the very N-terminus proved essential for all transformation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv6PpgbfWHhLVg90H21EOLACvtfcHk0ljRzv1WFO_BeA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsleqsL%252FE&md5=8fb1ed7283c0082113b5627536deb5d7</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-10-384685&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-10-384685%26sid%3Dliteratum%253Aachs%26aulast%3DBreitinger%26aufirst%3DC.%26aulast%3DMaethner%26aufirst%3DE.%26aulast%3DGarcia-Cuellar%26aufirst%3DM.-P.%26aulast%3DSlany%26aufirst%3DR.%2BK.%26atitle%3DThe%2520Homeodomain%2520Region%2520Controls%2520the%2520Phenotype%2520of%2520HOX-Induced%2520Murine%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4018%26epage%3D4027%26doi%3D10.1182%2Fblood-2011-10-384685" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ng, S. W. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLeod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibrahimova, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arruda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popescu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gupta, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmer, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuh, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yee, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bullinger, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herold, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Görlich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Büchner, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hiddemann, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berdel, W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wörmann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheok, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Preudhomme, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dombret, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Metzeler, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buske, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Löwenberg, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valk, P. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zandstra, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minden, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dick, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J. C. Y.</span></span> <span> </span><span class="NLM_article-title">A 17-Gene Stemness Score for Rapid Determination of Risk in Acute Leukaemia</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>540</i></span>,  <span class="NLM_fpage">433</span>– <span class="NLM_lpage">437</span>, <span class="refDoi"> DOI: 10.1038/nature20598</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1038%2Fnature20598" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27926740" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVWmurfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=540&publication_year=2016&pages=433-437&author=S.+W.+K.+Ngauthor=A.+Mitchellauthor=J.+A.+Kennedyauthor=W.+C.+Chenauthor=J.+McLeodauthor=N.+Ibrahimovaauthor=A.+Arrudaauthor=A.+Popescuauthor=V.+Guptaauthor=A.+D.+Schimmerauthor=A.+C.+Schuhauthor=K.+W.+Yeeauthor=L.+Bullingerauthor=T.+Heroldauthor=D.+G%C3%B6rlichauthor=T.+B%C3%BCchnerauthor=W.+Hiddemannauthor=W.+E.+Berdelauthor=B.+W%C3%B6rmannauthor=M.+Cheokauthor=C.+Preudhommeauthor=H.+Dombretauthor=K.+Metzelerauthor=C.+Buskeauthor=B.+L%C3%B6wenbergauthor=P.+J.+M.+Valkauthor=P.+W.+Zandstraauthor=M.+D.+Mindenauthor=J.+E.+Dickauthor=J.+C.+Y.+Wang&title=A+17-Gene+Stemness+Score+for+Rapid+Determination+of+Risk+in+Acute+Leukaemia&doi=10.1038%2Fnature20598"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">A 17-gene stemness score for rapid determination of risk in acute leukaemia</span></div><div class="casAuthors">Ng, Stanley W. K.; Mitchell, Amanda; Kennedy, James A.; Chen, Weihsu C.; McLeod, Jessica; Ibrahimova, Narmin; Arruda, Andrea; Popescu, Andreea; Gupta, Vikas; Schimmer, Aaron D.; Schuh, Andre C.; Yee, Karen W.; Bullinger, Lars; Herold, Tobias; Gorlich, Dennis; Buchner, Thomas; Hiddemann, Wolfgang; Berdel, Wolfgang E.; Wormann, Bernhard; Cheok, Meyling; Preudhomme, Claude; Dombret, Herve; Metzeler, Klaus; Buske, Christian; Lowenberg, Bob; Valk, Peter J. M.; Zandstra, Peter W.; Minden, Mark D.; Dick, John E.; Wang, Jean C. Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">540</span>
        (<span class="NLM_cas:issue">7633</span>),
    <span class="NLM_cas:pages">433-437</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Refractoriness to induction chemotherapy and relapse after achievement of remission are the main obstacles to cure in acute myeloid leukemia (AML).  After std. induction chemotherapy, patients are assigned to different post-remission strategies on the basis of cytogenetic and mol. abnormalities that broadly define adverse, intermediate and favorable risk categories.  However, some patients do not respond to induction therapy and another subset will eventually relapse despite the lack of adverse risk factors.  There is an urgent need for better biomarkers to identify these high-risk patients before starting induction chemotherapy, to enable testing of alternative induction strategies in clin. trials.  The high rate of relapse in AML has been attributed to the persistence of leukemia stem cells (LSCs), which possess a no. of stem cell properties, including quiescence, that are linked to therapy resistance.  Here, to develop predictive and/or prognostic biomarkers related to stemness, we generated a list of genes that are differentially expressed between 138 LSC+ and 89 LSC- cell fractions from 78 AML patients validated by xenotransplantation.  To ext. the core transcriptional components of stemness relevant to clin. outcomes, we performed sparse regression anal. of LSC gene expression against survival in a large training cohort, generating a 17-gene LSC score (LSC17).  The LSC17 score was highly prognostic in five independent cohorts comprising patients of diverse AML subtypes (n = 908) and contributed greatly to accurate prediction of initial therapy resistance.  Patients with high LSC17 scores had poor outcomes with current treatments including allogeneic stem cell transplantation.  The LSC17 score provides clinicians with a rapid and powerful tool to identify AML patients who do not benefit from std. therapy and who should be enrolled in trials evaluating novel upfront or post-remission strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_GpeG8ZvuZ7Vg90H21EOLACvtfcHk0lhHcX8hq-LRXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVWmurfO&md5=71fee194f9c91f9f24b84cfeefd0b601</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature20598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature20598%26sid%3Dliteratum%253Aachs%26aulast%3DNg%26aufirst%3DS.%2BW.%2BK.%26aulast%3DMitchell%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DJ.%2BA.%26aulast%3DChen%26aufirst%3DW.%2BC.%26aulast%3DMcLeod%26aufirst%3DJ.%26aulast%3DIbrahimova%26aufirst%3DN.%26aulast%3DArruda%26aufirst%3DA.%26aulast%3DPopescu%26aufirst%3DA.%26aulast%3DGupta%26aufirst%3DV.%26aulast%3DSchimmer%26aufirst%3DA.%2BD.%26aulast%3DSchuh%26aufirst%3DA.%2BC.%26aulast%3DYee%26aufirst%3DK.%2BW.%26aulast%3DBullinger%26aufirst%3DL.%26aulast%3DHerold%26aufirst%3DT.%26aulast%3DG%25C3%25B6rlich%26aufirst%3DD.%26aulast%3DB%25C3%25BCchner%26aufirst%3DT.%26aulast%3DHiddemann%26aufirst%3DW.%26aulast%3DBerdel%26aufirst%3DW.%2BE.%26aulast%3DW%25C3%25B6rmann%26aufirst%3DB.%26aulast%3DCheok%26aufirst%3DM.%26aulast%3DPreudhomme%26aufirst%3DC.%26aulast%3DDombret%26aufirst%3DH.%26aulast%3DMetzeler%26aufirst%3DK.%26aulast%3DBuske%26aufirst%3DC.%26aulast%3DL%25C3%25B6wenberg%26aufirst%3DB.%26aulast%3DValk%26aufirst%3DP.%2BJ.%2BM.%26aulast%3DZandstra%26aufirst%3DP.%2BW.%26aulast%3DMinden%26aufirst%3DM.%2BD.%26aulast%3DDick%26aufirst%3DJ.%2BE.%26aulast%3DWang%26aufirst%3DJ.%2BC.%2BY.%26atitle%3DA%252017-Gene%2520Stemness%2520Score%2520for%2520Rapid%2520Determination%2520of%2520Risk%2520in%2520Acute%2520Leukaemia%26jtitle%3DNature%26date%3D2016%26volume%3D540%26spage%3D433%26epage%3D437%26doi%3D10.1038%2Fnature20598" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sitwala, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronstein, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanders, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dandekar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Collins, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robertson, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cezard, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilenky, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thiessen, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeng, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirst, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hero, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">Identification and Characterization of Hoxa9 Binding Sites in Hematopoietic Cells</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>119</i></span>,  <span class="NLM_fpage">388</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1182/blood-2011-03-341081</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2011-03-341081" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22072553" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=2011&pages=388-398&author=Y.+Huangauthor=K.+Sitwalaauthor=J.+Bronsteinauthor=D.+Sandersauthor=M.+Dandekarauthor=C.+Collinsauthor=G.+Robertsonauthor=J.+MacDonaldauthor=T.+Cezardauthor=M.+Bilenkyauthor=N.+Thiessenauthor=Y.+Zhaoauthor=T.+Zengauthor=M.+Hirstauthor=A.+Heroauthor=S.+Jonesauthor=J.+L.+Hess&title=Identification+and+Characterization+of+Hoxa9+Binding+Sites+in+Hematopoietic+Cells&doi=10.1182%2Fblood-2011-03-341081"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1182%2Fblood-2011-03-341081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2011-03-341081%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DSitwala%26aufirst%3DK.%26aulast%3DBronstein%26aufirst%3DJ.%26aulast%3DSanders%26aufirst%3DD.%26aulast%3DDandekar%26aufirst%3DM.%26aulast%3DCollins%26aufirst%3DC.%26aulast%3DRobertson%26aufirst%3DG.%26aulast%3DMacDonald%26aufirst%3DJ.%26aulast%3DCezard%26aufirst%3DT.%26aulast%3DBilenky%26aufirst%3DM.%26aulast%3DThiessen%26aufirst%3DN.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DZeng%26aufirst%3DT.%26aulast%3DHirst%26aufirst%3DM.%26aulast%3DHero%26aufirst%3DA.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DIdentification%2520and%2520Characterization%2520of%2520Hoxa9%2520Binding%2520Sites%2520in%2520Hematopoietic%2520Cells%26jtitle%3DBlood%26date%3D2011%26volume%3D119%26spage%3D388%26epage%3D398%26doi%3D10.1182%2Fblood-2011-03-341081" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">LaRonde-LeBlanc, N. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolberger, C.</span></span> <span> </span><span class="NLM_article-title">Structure of HoxA9 and Pbx1 Bound to DNA: Hox Hexapeptide and DNA Recognition Anterior to Posterior</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">2060</span>– <span class="NLM_lpage">2072</span>, <span class="refDoi"> DOI: 10.1101/gad.1103303</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1101%2Fgad.1103303" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=2060-2072&author=N.+A.+LaRonde-LeBlancauthor=C.+Wolberger&title=Structure+of+HoxA9+and+Pbx1+Bound+to+DNA%3A+Hox+Hexapeptide+and+DNA+Recognition+Anterior+to+Posterior&doi=10.1101%2Fgad.1103303"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1101%2Fgad.1103303&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.1103303%26sid%3Dliteratum%253Aachs%26aulast%3DLaRonde-LeBlanc%26aufirst%3DN.%2BA.%26aulast%3DWolberger%26aufirst%3DC.%26atitle%3DStructure%2520of%2520HoxA9%2520and%2520Pbx1%2520Bound%2520to%2520DNA%253A%2520Hox%2520Hexapeptide%2520and%2520DNA%2520Recognition%2520Anterior%2520to%2520Posterior%26jtitle%3DGenes%2520Dev.%26date%3D2003%26volume%3D17%26spage%3D2060%26epage%3D2072%26doi%3D10.1101%2Fgad.1103303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Direct Inhibition of the DNA-Binding Activity of POU Transcription Factors Pit-1 and Brn-3 by Selective Binding of a Phenyl-Furan-Benzimidazole Dication</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">3341</span>– <span class="NLM_lpage">3353</span>, <span class="refDoi"> DOI: 10.1093/nar/gkn208</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkn208" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=3341-3353&author=P.+Peixotoauthor=Y.+Liuauthor=S.+Depauwauthor=M.-P.+Hildebrandauthor=D.+W.+Boykinauthor=C.+Baillyauthor=W.+D.+Wilsonauthor=M.-H.+David-Cordonnier&title=Direct+Inhibition+of+the+DNA-Binding+Activity+of+POU+Transcription+Factors+Pit-1+and+Brn-3+by+Selective+Binding+of+a+Phenyl-Furan-Benzimidazole+Dication&doi=10.1093%2Fnar%2Fgkn208"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkn208&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkn208%26sid%3Dliteratum%253Aachs%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DHildebrand%26aufirst%3DM.-P.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DBailly%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DDirect%2520Inhibition%2520of%2520the%2520DNA-Binding%2520Activity%2520of%2520POU%2520Transcription%2520Factors%2520Pit-1%2520and%2520Brn-3%2520by%2520Selective%2520Binding%2520of%2520a%2520Phenyl-Furan-Benzimidazole%2520Dication%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3D3341%26epage%3D3353%26doi%3D10.1093%2Fnar%2Fgkn208" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flajollet, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dezitter, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duterque-Coquillaud, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David Wilson, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span> <span> </span><span class="NLM_article-title">Targeting the DNA-Binding Activity of the Human ERG Transcription Factor Using New Heterocyclic Dithiophene Diamidines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">138</span>, <span class="refDoi"> DOI: 10.1093/nar/gks971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgks971" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=125-138&author=R.+Nhiliauthor=P.+Peixotoauthor=S.+Depauwauthor=S.+Flajolletauthor=X.+Dezitterauthor=M.+M.+Mundeauthor=M.+A.+Ismailauthor=A.+Kumarauthor=A.+A.+Farahatauthor=C.+E.+Stephensauthor=M.+Duterque-Coquillaudauthor=W.+David+Wilsonauthor=D.+W.+Boykinauthor=M.-H.+David-Cordonnier&title=Targeting+the+DNA-Binding+Activity+of+the+Human+ERG+Transcription+Factor+Using+New+Heterocyclic+Dithiophene+Diamidines&doi=10.1093%2Fnar%2Fgks971"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks971%26sid%3Dliteratum%253Aachs%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DFlajollet%26aufirst%3DS.%26aulast%3DDezitter%26aufirst%3DX.%26aulast%3DMunde%26aufirst%3DM.%2BM.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DDuterque-Coquillaud%26aufirst%3DM.%26aulast%3DDavid%2BWilson%26aufirst%3DW.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26atitle%3DTargeting%2520the%2520DNA-Binding%2520Activity%2520of%2520the%2520Human%2520ERG%2520Transcription%2520Factor%2520Using%2520New%2520Heterocyclic%2520Dithiophene%2520Diamidines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D41%26spage%3D125%26epage%3D138%26doi%3D10.1093%2Fnar%2Fgks971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Say, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">The Unusual Monomer Recognition of Guanine-Containing Mixed Sequence DNA by a Dithiophene Heterocyclic Diamidine</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">1218</span>– <span class="NLM_lpage">1227</span>, <span class="refDoi"> DOI: 10.1021/bi401582t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi401582t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslygsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=1218-1227&author=M.+Mundeauthor=A.+Kumarauthor=P.+Peixotoauthor=S.+Depauwauthor=M.+A.+Ismailauthor=A.+A.+Farahatauthor=A.+Paulauthor=M.+V.+Sayauthor=M.-H.+David-Cordonnierauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=The+Unusual+Monomer+Recognition+of+Guanine-Containing+Mixed+Sequence+DNA+by+a+Dithiophene+Heterocyclic+Diamidine&doi=10.1021%2Fbi401582t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The Unusual Monomer Recognition of Guanine-Containing Mixed Sequence DNA by a Dithiophene Heterocyclic Diamidine</span></div><div class="casAuthors">Munde, Manoj; Kumar, Arvind; Peixoto, Paul; Depauw, Sabine; Ismail, Mohamed A.; Farahat, Abdelbasset A.; Paul, Ananya; Say, Martial V.; David-Cordonnier, Marie-Helene; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1218-1227</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DB1255 is a sym. diamidinophenyl-dithiophene that exhibits cellular activity by binding to DNA and inhibiting binding of ERG, an ETS family transcription factor that is commonly overexpressed or translocated in leukemia and prostate cancer.  Because transcription factor inhibition is complex but is an attractive area for anticancer and antiparasitic drug development, the authors have evaluated the DNA interactions of addnl. derivs. of DB1255 to gain an improved understanding of the biophys. chem. of complex function and inhibition.  DNase I footprinting, biosensor surface plasmon resonance, and CD expts. show that DB1255 has an unusual and strong monomer binding mode in minor groove sites that contain a single GC base pair flanked by AT base pairs, for example, 5'-ATGAT-3'.  Closely related derivs., such as compds. with the thiophene replaced with furan or selenophane, bind very weakly to GC-contg. sequences and do not have biol. activity.  DB1255 is selective for the ATGAT site; however, a similar sequence, 5'-ATGAC-3', binds DB1255 more weakly and does not produce a footprint.  Mol. docking studies show that the two thiophene sulfur atoms form strong, bifurcated hydrogen bond-type interactions with the G-N-H sequence that extends into the minor groove while the amidines form hydrogen bonds to the flanking AT base pairs.  The central dithiophene unit of DB1255 thus forms an excellent, but unexpected, single-GC base pair recognition module in a monomer minor groove complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7rJFOeeJwP7Vg90H21EOLACvtfcHk0ljWtZ0EgHSZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslygsr8%253D&md5=8db5faa095f9114d368aa45108db0424</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fbi401582t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi401582t%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DSay%26aufirst%3DM.%2BV.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThe%2520Unusual%2520Monomer%2520Recognition%2520of%2520Guanine-Containing%2520Mixed%2520Sequence%2520DNA%2520by%2520a%2520Dithiophene%2520Heterocyclic%2520Diamidine%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D1218%26epage%3D1227%26doi%3D10.1021%2Fbi401582t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista-Parra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">DNA Minor Groove Induced Dimerization of Heterocyclic Cations: Compound Structure, Binding Affinity, and Specificity for a TTAA Site</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>402</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">864</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2010.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.jmb.2010.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20713062" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=402&publication_year=2010&pages=847-864&author=M.+Mundeauthor=A.+Kumarauthor=R.+Nhiliauthor=S.+Depauwauthor=M.-H.+David-Cordonnierauthor=M.+A.+Ismailauthor=C.+E.+Stephensauthor=A.+A.+Farahatauthor=A.+Batista-Parraauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=DNA+Minor+Groove+Induced+Dimerization+of+Heterocyclic+Cations%3A+Compound+Structure%2C+Binding+Affinity%2C+and+Specificity+for+a+TTAA+Site&doi=10.1016%2Fj.jmb.2010.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">DNA Minor Groove Induced Dimerization of Heterocyclic Cations: Compound Structure, Binding Affinity, and Specificity for a TTAA Site</span></div><div class="casAuthors">Munde, Manoj; Kumar, Arvind; Nhili, Raja; Depauw, Sabine; David-Cordonnier, Marie-Helene; Ismail, Mohamed A.; Stephens, Chad E.; Farahat, Abdelbasset A.; Batista-Parra, Adalgisa; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">402</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">847-864</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">With the increasing no. and variations of genome sequences available, control of gene expression with synthetic, cell-permeable mols. is within reach.  The variety of sequence-specific binding agents is, however, still quite limited.  Many minor groove binding agents selectivity recognize AT over GC sequences but have less ability to distinguish among different AT sequences.  The goal with this article is to develop compds. that can bind selectively to different AT sequences.  A no. of studies indicate that AATT and TTAA sequences have significantly different phys. and interaction properties and different requirements for minor groove recognition.  Although it has been difficult to get minor groove binding at TTAA, DB293, a phenyl-furan-benzimidazole diamidine, was found to bind as a strong, cooperative dimer at TTAA but with no selectivity over AATT.  In order to improve selectivity, we made modifications to each unit of DB293.  Binding affinities and stoichiometries obtained from biosensor-surface plasmon resonance expts. show that DB1003, a furan-furan-benzimidazole diamidine, binds strongly to TTAA as a dimer and has selectivity (KTTAA/KAATT = 6).  CD and DNase I footprinting studies confirmed the preference of this compd. for TTAA.  In summary, (i) a favorable stacking surface provided by the pi system, (ii) H-bond donors to interact with TA base pairs at the floor of the groove provided by a benzimidazole (or indole) -NH and amidines, and (iii) appropriate curvature of the dimer complex to match the curvature of the minor groove play important roles in differentiating the TTAA and AATT minor grooves.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnCOf12xu4WbVg90H21EOLACvtfcHk0ljWtZ0EgHSZLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1Cgu7jP&md5=ad50b8908a3ec39796cd1fadc79345e8</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2010.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2010.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBatista-Parra%26aufirst%3DA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DDNA%2520Minor%2520Groove%2520Induced%2520Dimerization%2520of%2520Heterocyclic%2520Cations%253A%2520Compound%2520Structure%252C%2520Binding%2520Affinity%252C%2520and%2520Specificity%2520for%2520a%2520TTAA%2520Site%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2010%26volume%3D402%26spage%3D847%26epage%3D864%26doi%3D10.1016%2Fj.jmb.2010.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Say, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chackal-Catoen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Batista-Parra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Water-Mediated Binding of Agents That Target the DNA Minor Groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>133</i></span>,  <span class="NLM_fpage">10171</span>– <span class="NLM_lpage">10183</span>, <span class="refDoi"> DOI: 10.1021/ja202006u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja202006u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXns1ent74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=133&publication_year=2011&pages=10171-10183&author=Y.+Liuauthor=A.+Kumarauthor=S.+Depauwauthor=R.+Nhiliauthor=M.-H.+David-Cordonnierauthor=M.+P.+Leeauthor=M.+A.+Ismailauthor=A.+A.+Farahatauthor=M.+Sayauthor=S.+Chackal-Catoenauthor=A.+Batista-Parraauthor=S.+Neidleauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Water-Mediated+Binding+of+Agents+That+Target+the+DNA+Minor+Groove&doi=10.1021%2Fja202006u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Water-Mediated Binding of Agents that Target the DNA Minor Groove</span></div><div class="casAuthors">Liu, Yang; Kumar, Arvind; Depauw, Sabine; Nhili, Raja; David-Cordonnier, Marie-Helene; Lee, Michael P.; Ismail, Mohamed A.; Farahat, Abdelbasset A.; Say, Martial; Chackal-Catoen, Sarah; Batista-Parra, Adalgisa; Neidle, Stephen; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">133</span>
        (<span class="NLM_cas:issue">26</span>),
    <span class="NLM_cas:pages">10171-10183</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Small mol. complexes with DNA that incorporate linking water mols. are rare, and the DB921-DNA complex has provided a unique and well-defined system for anal. of water-mediated binding in the context of a DNA complex.  DB921 has a benzimidazole-biphenyl system with terminal amidines that results in a linear conformation that does not possess the appropriate radius of curvature to match the minor groove shape and represents a new paradigm that does not fit the classical model of minor groove interactions.  To better understand the role of the bound water mol. obsd. in the x-ray crystal structure of the DB921 complex, synthetic modifications have been made in the DB921 structure, and the interactions of the new compds. with DNA AT sites have been evaluated with an array of methods, including DNase I footprinting, biosensor-surface plasmon resonance, isothermal titrn. microcalorimetry, and CD.  The interaction of a key compd., which has the amidine at the Ph shifted from the para position in DB921 to the meta position, has also been examd. by x-ray crystallog.  The detailed structural, thermodn., and kinetic results provide valuable new information for incorporation of water mols. in the design of new lead scaffolds for targeting DNA in chem. biol. and therapeutic applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSvdCZXTKwj7Vg90H21EOLACvtfcHk0lhqZlmNr1YiAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXns1ent74%253D&md5=01200ceedc05328d90050ac02cf236ad</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1021%2Fja202006u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja202006u%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DLee%26aufirst%3DM.%2BP.%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DSay%26aufirst%3DM.%26aulast%3DChackal-Catoen%26aufirst%3DS.%26aulast%3DBatista-Parra%26aufirst%3DA.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DWater-Mediated%2520Binding%2520of%2520Agents%2520That%2520Target%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2011%26volume%3D133%26spage%3D10171%26epage%3D10183%26doi%3D10.1021%2Fja202006u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mallena, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, M. P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neidle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Thiophene-Based Diamidine Forms a ″Super″ AT Binding Minor Groove Agent</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>126</i></span>,  <span class="NLM_fpage">13659</span>– <span class="NLM_lpage">13669</span>, <span class="refDoi"> DOI: 10.1021/ja048175m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja048175m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=126&publication_year=2004&pages=13659-13669&author=S.+Mallenaauthor=M.+P.+H.+Leeauthor=C.+Baillyauthor=S.+Neidleauthor=A.+Kumarauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Thiophene-Based+Diamidine+Forms+a+%E2%80%B3Super%E2%80%B3+AT+Binding+Minor+Groove+Agent&doi=10.1021%2Fja048175m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1021%2Fja048175m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja048175m%26sid%3Dliteratum%253Aachs%26aulast%3DMallena%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BP.%2BH.%26aulast%3DBailly%26aufirst%3DC.%26aulast%3DNeidle%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThiophene-Based%2520Diamidine%2520Forms%2520a%2520%25E2%2580%25B3Super%25E2%2580%25B3%2520AT%2520Binding%2520Minor%2520Groove%2520Agent%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2004%26volume%3D126%26spage%3D13659%26epage%3D13669%26doi%3D10.1021%2Fja048175m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span> <span> </span><span class="NLM_article-title">Structure-Based DNA-Targeting Strategies with Small Molecule Ligands for Drug Discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">1119</span>– <span class="NLM_lpage">1173</span>, <span class="refDoi"> DOI: 10.1002/med.21278</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fmed.21278" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=23633219" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1ChtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2013&pages=1119-1173&author=J.+Shengauthor=J.+Ganauthor=Z.+Huang&title=Structure-Based+DNA-Targeting+Strategies+with+Small+Molecule+Ligands+for+Drug+Discovery&doi=10.1002%2Fmed.21278"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Based DNA-Targeting Strategies with Small Molecule Ligands for Drug Discovery</span></div><div class="casAuthors">Sheng, Jia; Gan, Jianhua; Huang, Zhen</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1119-1173</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Nucleic acids are the mol. targets of many clin. anticancer drugs.  However, compared with proteins, nucleic acids have traditionally attracted much less attention as drug targets in structure-based drug design, partially because limited structural information of nucleic acids complexed with potential drugs is available.  Over the past several years, enormous progresses in nucleic acid crystn., heavy-atom derivatization, phasing, and structural biol. have been made.  Many complicated nucleic acid structures have been detd., providing new insights into the mol. functions and interactions of nucleic acids, esp. DNAs complexed with small mol. ligands.  Thus, opportunities have been created to further discover nucleic acid-targeting drugs for disease treatments.  This review focuses on the structure studies of DNAs complexed with small mol. ligands for discovering lead compds., drug candidates, and/or therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozFxbOusdeU7Vg90H21EOLACvtfcHk0lhgKt6UTuyr2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1ChtbvO&md5=876fcb98aa17acf928d9d6bce0619d40</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1002%2Fmed.21278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21278%26sid%3Dliteratum%253Aachs%26aulast%3DSheng%26aufirst%3DJ.%26aulast%3DGan%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26atitle%3DStructure-Based%2520DNA-Targeting%2520Strategies%2520with%2520Small%2520Molecule%2520Ligands%2520for%2520Drug%2520Discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2013%26volume%3D33%26spage%3D1119%26epage%3D1173%26doi%3D10.1002%2Fmed.21278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lansiaux, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishal, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dassonneville, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bailly, C.</span></span> <span> </span><span class="NLM_article-title">Distribution of furamidine analogues in tumor cells: targeting of the nucleus or mitochondria depending on the amidine substitution</span>. <i>Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">7219</span>– <span class="NLM_lpage">7229</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2002&pages=7219-7229&author=A.+Lansiauxauthor=F.+Taniousauthor=Z.+Mishalauthor=L.+Dassonnevilleauthor=A.+Kumarauthor=C.+E.+Stephensauthor=Q.+Huauthor=W.+D.+Wilsonauthor=D.+W.+Boykinauthor=C.+Bailly&title=Distribution+of+furamidine+analogues+in+tumor+cells%3A+targeting+of+the+nucleus+or+mitochondria+depending+on+the+amidine+substitution"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLansiaux%26aufirst%3DA.%26aulast%3DTanious%26aufirst%3DF.%26aulast%3DMishal%26aufirst%3DZ.%26aulast%3DDassonneville%26aufirst%3DL.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DBailly%26aufirst%3DC.%26atitle%3DDistribution%2520of%2520furamidine%2520analogues%2520in%2520tumor%2520cells%253A%2520targeting%2520of%2520the%2520nucleus%2520or%2520mitochondria%2520depending%2520on%2520the%2520amidine%2520substitution%26jtitle%3DCancer%2520Res.%26date%3D2002%26volume%3D62%26spage%3D7219%26epage%3D7229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ghannam, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takeda, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Camarata, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moore, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viale, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaseen, N. R.</span></span> <span> </span><span class="NLM_article-title">The OncogeneNup98-HOXA9Induces Gene Transcription in Myeloid Cells</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">866</span>– <span class="NLM_lpage">875</span>, <span class="refDoi"> DOI: 10.1074/jbc.m307280200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1074%2Fjbc.m307280200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2003&pages=866-875&author=G.+Ghannamauthor=A.+Takedaauthor=T.+Camarataauthor=M.+A.+Mooreauthor=A.+Vialeauthor=N.+R.+Yaseen&title=The+OncogeneNup98-HOXA9Induces+Gene+Transcription+in+Myeloid+Cells&doi=10.1074%2Fjbc.m307280200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m307280200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m307280200%26sid%3Dliteratum%253Aachs%26aulast%3DGhannam%26aufirst%3DG.%26aulast%3DTakeda%26aufirst%3DA.%26aulast%3DCamarata%26aufirst%3DT.%26aulast%3DMoore%26aufirst%3DM.%2BA.%26aulast%3DViale%26aufirst%3DA.%26aulast%3DYaseen%26aufirst%3DN.%2BR.%26atitle%3DThe%2520OncogeneNup98-HOXA9Induces%2520Gene%2520Transcription%2520in%2520Myeloid%2520Cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D279%26spage%3D866%26epage%3D875%26doi%3D10.1074%2Fjbc.m307280200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Venturini, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marquez, V. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scherr, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacLeod, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span> <span> </span><span class="NLM_article-title">Polycomb Repressor Complex 2 Regulates HOXA9 and HOXA10, Activating ID2 in NK/T-Cell Lines</span>. <i>Mol. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">151</span>, <span class="refDoi"> DOI: 10.1186/1476-4598-9-151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2F1476-4598-9-151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20565746" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC3cnjvVGjsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=151&author=S.+Nagelauthor=L.+Venturiniauthor=V.+E.+Marquezauthor=C.+Meyerauthor=M.+Kaufmannauthor=M.+Scherrauthor=R.+A.+MacLeodauthor=H.+G.+Drexler&title=Polycomb+Repressor+Complex+2+Regulates+HOXA9+and+HOXA10%2C+Activating+ID2+in+NK%2FT-Cell+Lines&doi=10.1186%2F1476-4598-9-151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb repressor complex 2 regulates HOXA9 and HOXA10, activating ID2 in NK/T-cell lines</span></div><div class="casAuthors">Nagel Stefan; Venturini Letizia; Marquez Victor E; Meyer Corinna; Kaufmann Maren; Scherr Michaela; MacLeod Roderick Af; Drexler Hans G</div><div class="citationInfo"><span class="NLM_cas:title">Molecular cancer</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">151</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  NK- and T-cells are closely related lymphocytes, originating from the same early progenitor cells during hematopoiesis.  In these differentiation processes deregulation of developmental genes may contribute to leukemogenesis.  Here, we compared expression profiles of NK- and T-cell lines for identification of aberrantly expressed genes in T-cell acute lymphoblastic leukemia (T-ALL) which physiologically regulate the differentiation program of the NK-cell lineage.  RESULTS:  This analysis showed high expression levels of HOXA9, HOXA10 and ID2 in NK-cell lines in addition to T-cell line LOUCY, suggesting leukemic deregulation therein.  Overexpression experiments, chromatin immuno-precipitation and promoter analysis demonstrated that HOXA9 and HOXA10 directly activated expression of ID2.  Concomitantly elevated expression levels of HOXA9 and HOXA10 together with ID2 in cell lines containing MLL translocations confirmed this form of regulation in both ALL and acute myeloid leukemia.  Overexpression of HOXA9, HOXA10 or ID2 resulted in repressed expression of apoptosis factor BIM.  Furthermore, profiling data of genes coding for chromatin regulators of homeobox genes, including components of polycomb repressor complex 2 (PRC2), indicated lacking expression of EZH2 in LOUCY and exclusive expression of HOP in NK-cell lines.  Subsequent treatment of T-cell lines JURKAT and LOUCY with DZNep, an inhibitor of EZH2/PRC2, resulted in elevated and unchanged HOXA9/10 expression levels, respectively.  Moreover, siRNA-mediated knockdown of EZH2 in JURKAT enhanced HOXA10 expression, confirming HOXA10-repression by EZH2.  Additionally, profiling data and overexpression analysis indicated that reduced expression of E2F cofactor TFDP1 contributed to the lack of EZH2 in LOUCY.  Forced expression of HOP in JURKAT cells resulted in reduced HOXA10 and ID2 expression levels, suggesting enhancement of PRC2 repression.  CONCLUSIONS:  Our results show that major differentiation factors of the NK-cell lineage, including HOXA9, HOXA10 and ID2, were (de)regulated via PRC2 which therefore contributes to T-cell leukemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSWv8RzrWVfouQ4Pl9hzjQhfW6udTcc2eb26d0eLJ6_C7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnjvVGjsg%253D%253D&md5=485d85bee87872e3c643b01dae469e86</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1186%2F1476-4598-9-151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1476-4598-9-151%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DS.%26aulast%3DVenturini%26aufirst%3DL.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DScherr%26aufirst%3DM.%26aulast%3DMacLeod%26aufirst%3DR.%2BA.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26atitle%3DPolycomb%2520Repressor%2520Complex%25202%2520Regulates%2520HOXA9%2520and%2520HOXA10%252C%2520Activating%2520ID2%2520in%2520NK%252FT-Cell%2520Lines%26jtitle%3DMol.%2520Cancer%26date%3D2010%26volume%3D9%26spage%3D151%26doi%3D10.1186%2F1476-4598-9-151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velu, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaubey, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phelan, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Horman, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jegga, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zeleznik-Le, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulloy, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cancelas, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jordan, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aronow, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcucci, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gebelein, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimes, H. L.</span></span> <span> </span><span class="NLM_article-title">Therapeutic Antagonists of MicroRNAs Deplete Leukemia-Initiating Cell Activity</span>. <i>J. Clin. Invest.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">236</span>, <span class="refDoi"> DOI: 10.1172/jci66005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1172%2Fjci66005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2013&pages=222-236&author=C.+S.+Veluauthor=A.+Chaubeyauthor=J.+D.+Phelanauthor=S.+R.+Hormanauthor=M.+Wunderlichauthor=M.+L.+Guzmanauthor=A.+G.+Jeggaauthor=N.+J.+Zeleznik-Leauthor=J.+Chenauthor=J.+C.+Mulloyauthor=J.+A.+Cancelasauthor=C.+T.+Jordanauthor=B.+J.+Aronowauthor=G.+Marcucciauthor=B.+Bhatauthor=B.+Gebeleinauthor=H.+L.+Grimes&title=Therapeutic+Antagonists+of+MicroRNAs+Deplete+Leukemia-Initiating+Cell+Activity&doi=10.1172%2Fjci66005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1172%2Fjci66005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1172%252Fjci66005%26sid%3Dliteratum%253Aachs%26aulast%3DVelu%26aufirst%3DC.%2BS.%26aulast%3DChaubey%26aufirst%3DA.%26aulast%3DPhelan%26aufirst%3DJ.%2BD.%26aulast%3DHorman%26aufirst%3DS.%2BR.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DGuzman%26aufirst%3DM.%2BL.%26aulast%3DJegga%26aufirst%3DA.%2BG.%26aulast%3DZeleznik-Le%26aufirst%3DN.%2BJ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DMulloy%26aufirst%3DJ.%2BC.%26aulast%3DCancelas%26aufirst%3DJ.%2BA.%26aulast%3DJordan%26aufirst%3DC.%2BT.%26aulast%3DAronow%26aufirst%3DB.%2BJ.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DB.%26aulast%3DGebelein%26aufirst%3DB.%26aulast%3DGrimes%26aufirst%3DH.%2BL.%26atitle%3DTherapeutic%2520Antagonists%2520of%2520MicroRNAs%2520Deplete%2520Leukemia-Initiating%2520Cell%2520Activity%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D2013%26volume%3D124%26spage%3D222%26epage%3D236%26doi%3D10.1172%2Fjci66005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantor, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moscinski, L. C.</span></span> <span> </span><span class="NLM_article-title">β2 Integrins are characteristically absent in acute promyelocytic leukemia and rapidly upregulated in vivo upon differentiation with all-trans retinoic acid</span>. <i>Leuk. Res.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">49</span>– <span class="NLM_lpage">57</span>, <span class="refDoi"> DOI: 10.1016/j.leukres.2006.04.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.leukres.2006.04.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=16764927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BD28nmvFKksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2007&pages=49-57&author=J.+J.+Wuauthor=A.+Cantorauthor=L.+C.+Moscinski&title=%CE%B22+Integrins+are+characteristically+absent+in+acute+promyelocytic+leukemia+and+rapidly+upregulated+in+vivo+upon+differentiation+with+all-trans+retinoic+acid&doi=10.1016%2Fj.leukres.2006.04.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">beta2 Integrins are characteristically absent in acute promyelocytic leukemia and rapidly upregulated in vivo upon differentiation with all-trans retinoic acid</span></div><div class="casAuthors">Wu Jamie J; Cantor Alan; Moscinski Lynn C</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">49-57</span>
        ISSN:<span class="NLM_cas:issn">0145-2126</span>.
    </div><div class="casAbstract">Although little is known about migration of hematopoietic stem cells and their neoplastic counterparts into tissues and peripheral blood, adhesion proteins likely play an important role.  We studied 339 patients with acute myelogenous leukemia (AML) to discern the relationship between adhesion protein expression, circulating blasts, and white blood cell (WBC) count.  Expression levels of CD11b and CD11c strongly correlated with increased WBC count, independent of FAB subtype (p<0.0001).  However, 93% (25/27) of cases of AML-M3 completely lacked beta2 integrin expression, compared to 11% (35/312) of the non-M3 cases (p<0.0001).  Seven of the 27 patients with AML-M3 were followed during standard induction therapy with ATRA.  Within 3 days, weak CD11c became detectable, followed by CD11b and CD11a.  Our data suggest an important link between beta2 integrin expression and the level of circulating leukemic cells in AML.  We demonstrate the clinical usefulness of a panel of beta2 integrins (CD11a, CD11b and CD11c) in accurate prediction of AML-M3, and recommend inclusion of this immunophenotypic analysis to identify patients who require ATRA therapy.  Finally, we illustrate the rapidity at which AML-M3 blasts up-regulate beta2 integrins, and suggest a possible association between this finding and the tissue infiltration that characterizes the "ATRA syndrome".</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT3NtIGKSCkv2Zyxfu64m4ffW6udTcc2eYYQIn23VOE2Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28nmvFKksw%253D%253D&md5=1030910c1f46798fa388c019486650c6</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.leukres.2006.04.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.leukres.2006.04.012%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DJ.%2BJ.%26aulast%3DCantor%26aufirst%3DA.%26aulast%3DMoscinski%26aufirst%3DL.%2BC.%26atitle%3D%25CE%25B22%2520Integrins%2520are%2520characteristically%2520absent%2520in%2520acute%2520promyelocytic%2520leukemia%2520and%2520rapidly%2520upregulated%2520in%2520vivo%2520upon%2520differentiation%2520with%2520all-trans%2520retinoic%2520acid%26jtitle%3DLeuk.%2520Res.%26date%3D2007%26volume%3D31%26spage%3D49%26epage%3D57%26doi%3D10.1016%2Fj.leukres.2006.04.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, J.</span></span> <span> </span><span class="NLM_article-title">Upregulation of CD11b and CD86 through LSD1 Inhibition Promotes Myeloid Differentiation and Suppresses Cell Proliferation in Human Monocytic Leukemia Cells</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">85085</span>– <span class="NLM_lpage">85101</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.18564</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.18632%2Foncotarget.18564" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=85085-85101&author=J.+Fangauthor=H.+Yingauthor=T.+Maoauthor=Y.+Fangauthor=Y.+Luauthor=H.+Wangauthor=I.+Zangauthor=Z.+Wangauthor=Y.+Linauthor=M.+Zhaoauthor=X.+Luoauthor=Z.+Wangauthor=Y.+Zhangauthor=C.+Zhangauthor=W.+Xiaoauthor=Y.+Wangauthor=W.+Tanauthor=Z.+Chenauthor=C.+Luauthor=P.+Atadjaauthor=E.+Liauthor=K.+Zhaoauthor=J.+Liuauthor=J.+Gu&title=Upregulation+of+CD11b+and+CD86+through+LSD1+Inhibition+Promotes+Myeloid+Differentiation+and+Suppresses+Cell+Proliferation+in+Human+Monocytic+Leukemia+Cells&doi=10.18632%2Foncotarget.18564"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.18564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.18564%26sid%3Dliteratum%253Aachs%26aulast%3DFang%26aufirst%3DJ.%26aulast%3DYing%26aufirst%3DH.%26aulast%3DMao%26aufirst%3DT.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DC.%26aulast%3DXiao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DTan%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DGu%26aufirst%3DJ.%26atitle%3DUpregulation%2520of%2520CD11b%2520and%2520CD86%2520through%2520LSD1%2520Inhibition%2520Promotes%2520Myeloid%2520Differentiation%2520and%2520Suppresses%2520Cell%2520Proliferation%2520in%2520Human%2520Monocytic%2520Leukemia%2520Cells%26jtitle%3DOncotarget%26date%3D2017%26volume%3D8%26spage%3D85085%26epage%3D85101%26doi%3D10.18632%2Foncotarget.18564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saunders, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marincola, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stroncek, D. F.</span></span> <span> </span><span class="NLM_article-title">Molecular Signatures of Maturing Dendritic Cells: Implications for Testing the Quality of Dendritic Cell Therapies</span>. <i>J. Transl. Med.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">4</span>, <span class="refDoi"> DOI: 10.1186/1479-5876-8-4</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1186%2F1479-5876-8-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=20078880" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A280%3ADC%252BC3c7ptlWntg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2010&pages=4&author=P.+Jinauthor=T.+Hanauthor=J.+Renauthor=S.+Saundersauthor=E.+Wangauthor=F.+M.+Marincolaauthor=D.+F.+Stroncek&title=Molecular+Signatures+of+Maturing+Dendritic+Cells%3A+Implications+for+Testing+the+Quality+of+Dendritic+Cell+Therapies&doi=10.1186%2F1479-5876-8-4"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular signatures of maturing dendritic cells: implications for testing the quality of dendritic cell therapies</span></div><div class="casAuthors">Jin Ping; Han Tae Hee; Ren Jiaqiang; Saunders Stefanie; Wang Ena; Marincola Francesco M; Stroncek David F</div><div class="citationInfo"><span class="NLM_cas:title">Journal of translational medicine</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">4</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Dendritic cells (DCs) are often produced by granulocyte-macrophage colony-stimulating factor (GM-CSF) and interleukin-4 (IL-4) stimulation of monocytes.  To improve the effectiveness of DC adoptive immune cancer therapy, many different agents have been used to mature DCs.  We analyzed the kinetics of DC maturation by lipopolysaccharide (LPS) and interferon-gamma (IFN-gamma) induction in order to characterize the usefulness of mature DCs (mDCs) for immune therapy and to identify biomarkers for assessing the quality of mDCs.  METHODS:  Peripheral blood mononuclear cells were collected from 6 healthy subjects by apheresis, monocytes were isolated by elutriation, and immature DCs (iDCs) were produced by 3 days of culture with GM-CSF and IL-4.  The iDCs were sampled after 4, 8 and 24 hours in culture with LPS and IFN-gamma and were then assessed by flow cytometry, ELISA, and global gene and microRNA (miRNA) expression analysis.  RESULTS:  After 24 hours of LPS and IFN-gamma stimulation, DC surface expression of CD80, CD83, CD86, and HLA Class II antigens were up-regulated.  Th1 attractant genes such as CXCL9, CXCL10, CXCL11 and CCL5 were up-regulated during maturation but not Treg attractants such as CCL22 and CXCL12.  The expression of classical mDC biomarker genes CD83, CCR7, CCL5, CCL8, SOD2, MT2A, OASL, GBP1 and HES4 were up-regulated throughout maturation while MTIB, MTIE, MTIG, MTIH, GADD45A and LAMP3 were only up-regulated late in maturation.  The expression of miR-155 was up-regulated 8-fold in mDCs.  CONCLUSION:  DCs, matured with LPS and IFN-gamma, were characterized by increased levels of Th1 attractants as opposed to Treg attractants and may be particularly effective for adoptive immune cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcR8ValubdQ-Q9K0qsdY6Nc1fW6udTcc2ea4MblGs09lrrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3c7ptlWntg%253D%253D&md5=f66c11cb98efbe921a26de3b7ade2a27</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1186%2F1479-5876-8-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-5876-8-4%26sid%3Dliteratum%253Aachs%26aulast%3DJin%26aufirst%3DP.%26aulast%3DHan%26aufirst%3DT.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DSaunders%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DMarincola%26aufirst%3DF.%2BM.%26aulast%3DStroncek%26aufirst%3DD.%2BF.%26atitle%3DMolecular%2520Signatures%2520of%2520Maturing%2520Dendritic%2520Cells%253A%2520Implications%2520for%2520Testing%2520the%2520Quality%2520of%2520Dendritic%2520Cell%2520Therapies%26jtitle%3DJ.%2520Transl.%2520Med.%26date%3D2010%26volume%3D8%26spage%3D4%26doi%3D10.1186%2F1479-5876-8-4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phuc Khoa, L. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witkin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koche, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ge, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dressler, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hess, J. L.</span></span> <span> </span><span class="NLM_article-title">HOXA9 Reprograms the Enhancer Landscape to Promote Leukemogenesis</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">643</span>– <span class="NLM_lpage">658</span>, <span class="refDoi"> DOI: 10.1016/j.ccell.2018.08.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccell.2018.08.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=30270123" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVahsbzE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2018&pages=643-658&author=Y.+Sunauthor=B.+Zhouauthor=F.+Maoauthor=J.+Xuauthor=H.+Miaoauthor=Z.+Zouauthor=L.+T.+Phuc+Khoaauthor=Y.+Jangauthor=S.+Caiauthor=M.+Witkinauthor=R.+Kocheauthor=K.+Geauthor=G.+R.+Dresslerauthor=R.+L.+Levineauthor=S.+A.+Armstrongauthor=Y.+Douauthor=J.+L.+Hess&title=HOXA9+Reprograms+the+Enhancer+Landscape+to+Promote+Leukemogenesis&doi=10.1016%2Fj.ccell.2018.08.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">HOXA9 Reprograms the Enhancer Landscape to Promote Leukemogenesis</span></div><div class="casAuthors">Sun, Yuqing; Zhou, Bo; Mao, Fengbiao; Xu, Jing; Miao, Hongzhi; Zou, Zhenhua; Phuc Khoa, Le Tran; Jang, Younghoon; Cai, Sheng; Witkin, Matthew; Koche, Richard; Ge, Kai; Dressler, Gregory R.; Levine, Ross L.; Armstrong, Scott A.; Dou, Yali; Hess, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">643-658.e5</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">Aberrant expression of HOXA9 is a prominent feature of acute leukemia driven by diverse oncogenes.  Here we show that HOXA9 overexpression in myeloid and B progenitor cells leads to significant enhancer reorganizations with prominent emergence of leukemia-specific de novo enhancers.  Alterations in the enhancer landscape lead to activation of an ectopic embryonic gene program.  We show that HOXA9 functions as a pioneer factor at de novo enhancers and recruits CEBPα and the MLL3/MLL4 complex.  Genetic deletion of MLL3/MLL4 blocks histone H3K4 methylation at de novo enhancers and inhibits HOXA9/MEIS1-mediated leukemogenesis in vivo.  These results suggest that therapeutic targeting of HOXA9-dependent enhancer reorganization can be an effective therapeutic strategy in acute leukemia with HOXA9 overexpression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnd105uBPOFbVg90H21EOLACvtfcHk0lhEFyzqObJeIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVahsbzE&md5=88f4ad9726cc4dd8f038e77d6653c7eb</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2018.08.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2018.08.018%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DMao%26aufirst%3DF.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DMiao%26aufirst%3DH.%26aulast%3DZou%26aufirst%3DZ.%26aulast%3DPhuc%2BKhoa%26aufirst%3DL.%2BT.%26aulast%3DJang%26aufirst%3DY.%26aulast%3DCai%26aufirst%3DS.%26aulast%3DWitkin%26aufirst%3DM.%26aulast%3DKoche%26aufirst%3DR.%26aulast%3DGe%26aufirst%3DK.%26aulast%3DDressler%26aufirst%3DG.%2BR.%26aulast%3DLevine%26aufirst%3DR.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DDou%26aufirst%3DY.%26aulast%3DHess%26aufirst%3DJ.%2BL.%26atitle%3DHOXA9%2520Reprograms%2520the%2520Enhancer%2520Landscape%2520to%2520Promote%2520Leukemogenesis%26jtitle%3DCancer%2520Cell%26date%3D2018%26volume%3D34%26spage%3D643%26epage%3D658%26doi%3D10.1016%2Fj.ccell.2018.08.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Daigle, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olhava, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Therkelsen, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majer, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sneeringer, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnston, L. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuntz, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chesworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyer, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernt, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tseng, J.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kung, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armstrong, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Copeland, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richon, V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollock, R. M.</span></span> <span> </span><span class="NLM_article-title">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">53</span>– <span class="NLM_lpage">65</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2011.06.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ccr.2011.06.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=21741596" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=53-65&author=S.+R.+Daigleauthor=E.+J.+Olhavaauthor=C.+A.+Therkelsenauthor=C.+R.+Majerauthor=C.+J.+Sneeringerauthor=J.+Songauthor=L.+D.+Johnstonauthor=M.+P.+Scottauthor=J.+J.+Smithauthor=Y.+Xiaoauthor=L.+Jinauthor=K.+W.+Kuntzauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=K.+M.+Berntauthor=J.-C.+Tsengauthor=A.+L.+Kungauthor=S.+A.+Armstrongauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollock&title=Selective+Killing+of+Mixed+Lineage+Leukemia+Cells+by+a+Potent+Small-Molecule+DOT1L+Inhibitor&doi=10.1016%2Fj.ccr.2011.06.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Killing of Mixed Lineage Leukemia Cells by a Potent Small-Molecule DOT1L Inhibitor</span></div><div class="casAuthors">Daigle, Scott R.; Olhava, Edward J.; Therkelsen, Carly A.; Majer, Christina R.; Sneeringer, Christopher J.; Song, Jeffrey; Johnston, L. Danielle; Scott, Margaret Porter; Smith, Jesse J.; Xiao, Yonghong; Jin, Lei; Kuntz, Kevin W.; Chesworth, Richard; Moyer, Mikel P.; Bernt, Kathrin M.; Tseng, Jen-Chieh; Kung, Andrew L.; Armstrong, Scott A.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">53-65</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Mislocated enzymic activity of DOT1L has been proposed as a driver of leukemogenesis in mixed lineage leukemia (MLL).  The characterization of EPZ004777, a potent, selective inhibitor of DOT1L is reported.  Treatment of MLL cells with the compd. selectively inhibits H3K79 methylation and blocks expression of leukemogenic genes.  Exposure of leukemic cells to EPZ004777 results in selective killing of those cells bearing the MLL gene translocation, with little effect on non-MLL-translocated cells.  Finally, in vivo administration of EPZ004777 leads to extension of survival in a mouse MLL xenograft model.  These results provide compelling support for DOT1L inhibition as a basis for targeted therapeutics against MLL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodbBbxHFmyybVg90H21EOLACvtfcHk0lhEFyzqObJeIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXoslCms7c%253D&md5=1d39f99c2983b6b5fccc079110092c3d</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2011.06.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2011.06.009%26sid%3Dliteratum%253Aachs%26aulast%3DDaigle%26aufirst%3DS.%2BR.%26aulast%3DOlhava%26aufirst%3DE.%2BJ.%26aulast%3DTherkelsen%26aufirst%3DC.%2BA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DBernt%26aufirst%3DK.%2BM.%26aulast%3DTseng%26aufirst%3DJ.-C.%26aulast%3DKung%26aufirst%3DA.%2BL.%26aulast%3DArmstrong%26aufirst%3DS.%2BA.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26atitle%3DSelective%2520Killing%2520of%2520Mixed%2520Lineage%2520Leukemia%2520Cells%2520by%2520a%2520Potent%2520Small-Molecule%2520DOT1L%2520Inhibitor%26jtitle%3DCancer%2520Cell%26date%3D2011%26volume%3D20%26spage%3D53%26epage%3D65%26doi%3D10.1016%2Fj.ccr.2011.06.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Waters, N. J.</span></span> <span> </span><span class="NLM_article-title">Preclinical Pharmacokinetics and Pharmacodynamics of Pinometostat (EPZ-5676), a First-in-Class, Small Molecule S-Adenosyl Methionine Competitive Inhibitor of DOT1L</span>. <i>Eur. J. Drug Metab. Pharmacokinet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">891</span>– <span class="NLM_lpage">901</span>, <span class="refDoi"> DOI: 10.1007/s13318-017-0404-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1007%2Fs13318-017-0404-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=28229434" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjtFKls7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2017&pages=891-901&author=N.+J.+Waters&title=Preclinical+Pharmacokinetics+and+Pharmacodynamics+of+Pinometostat+%28EPZ-5676%29%2C+a+First-in-Class%2C+Small+Molecule+S-Adenosyl+Methionine+Competitive+Inhibitor+of+DOT1L&doi=10.1007%2Fs13318-017-0404-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Pharmacokinetics and Pharmacodynamics of Pinometostat (EPZ-5676), a First-in-Class, Small Molecule S-Adenosyl Methionine Competitive Inhibitor of DOT1L</span></div><div class="casAuthors">Waters, Nigel J.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Drug Metabolism and Pharmacokinetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">891-901</span>CODEN:
                <span class="NLM_cas:coden">EJDPD2</span>;
        ISSN:<span class="NLM_cas:issn">0378-7966</span>.
    
            (<span class="NLM_cas:orgname">Springer France</span>)
        </div><div class="casAbstract">Acute leukemias bearing mixed lineage leukemia (MLL) rearrangements are aggressive diseases characterized by a poor overall prognosis despite multi-agent chemotherapy.  Aberrant fusion proteins involving the MLL histone methyltransferase (HMT) lead to recruitment of DOT1L, to a multi-protein complex resulting in aberrant methylation of histone H3 lysine 79 at MLL target genes, and ultimately enhanced expression of crit. genes for hematopoietic differentiation, including HOXA9 and MEIS1, and as such defines the established mechanism for leukemogenesis in MLL-rearrangement (MLL-r) leukemias.  Pinometostat is a first-in-class, small mol. inhibitor of DOT1L with sub-nanomolar affinity and >37,000 fold selectivity against non-MLL HMTs, and was the first member of the novel HMT inhibitor class to enter Phase 1 clin. trials in both adult and pediatric MLL-r leukemia patients.  In this article, the preclin. pharmacokinetics/pharmacodynamics and drug disposition of pinometostat are reviewed including discussion of how these data were used to inform early clin. studies, and how they translated to the clin. experience.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpxU8juCNJnELVg90H21EOLACvtfcHk0likljvr4LBFvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjtFKls7c%253D&md5=edacbc0f25f056f736705ae7c6f184cf</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1007%2Fs13318-017-0404-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs13318-017-0404-3%26sid%3Dliteratum%253Aachs%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26atitle%3DPreclinical%2520Pharmacokinetics%2520and%2520Pharmacodynamics%2520of%2520Pinometostat%2520%2528EPZ-5676%2529%252C%2520a%2520First-in-Class%252C%2520Small%2520Molecule%2520S-Adenosyl%2520Methionine%2520Competitive%2520Inhibitor%2520of%2520DOT1L%26jtitle%3DEur.%2520J.%2520Drug%2520Metab.%2520Pharmacokinet.%26date%3D2017%26volume%3D42%26spage%3D891%26epage%3D901%26doi%3D10.1007%2Fs13318-017-0404-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murai, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lund, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartley, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Purohit, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chruszcz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grembecka, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cierpicki, T.</span></span> <span> </span><span class="NLM_article-title">Structural Insights into Inhibition of the Bivalent Menin-MLL Interaction by Small Molecules in Leukemia</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>120</i></span>,  <span class="NLM_fpage">4461</span>– <span class="NLM_lpage">4469</span>, <span class="refDoi"> DOI: 10.1182/blood-2012-05-429274</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1182%2Fblood-2012-05-429274" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=22936661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2012&pages=4461-4469&author=A.+Shiauthor=M.+J.+Muraiauthor=S.+Heauthor=G.+Lundauthor=T.+Hartleyauthor=T.+Purohitauthor=G.+Reddyauthor=M.+Chruszczauthor=J.+Grembeckaauthor=T.+Cierpicki&title=Structural+Insights+into+Inhibition+of+the+Bivalent+Menin-MLL+Interaction+by+Small+Molecules+in+Leukemia&doi=10.1182%2Fblood-2012-05-429274"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insights into inhibition of the bivalent menin-MLL interaction by small molecules in leukemia</span></div><div class="casAuthors">Shi, Aibin; Murai, Marcelo J.; He, Shihan; Lund, George; Hartley, Thomas; Purohit, Trupta; Reddy, Gireesh; Chruszcz, Maksymilian; Grembecka, Jolanta; Cierpicki, Tomasz</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">4461-4469</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Menin functions as a crit. oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in the development of acute leukemias, and inhibition of the menin interaction with MLL fusion proteins represents a very promising strategy to reverse their oncogenic activity.  MLL interacts with menin in a bivalent mode involving 2 N-terminal fragments of MLL.  In the present study, we reveal the first high-resoln. crystal structure of human menin in complex with a small-mol. inhibitor of the menin-MLL interaction, MI-2.  The structure shows that the compd. binds to the MLL pocket in menin and mimics the key interactions of MLL with menin.  Based on the menin-MI-2 structure, we developed MI-2-2, a compd. that binds to menin with low nanomolar affinity (Kd = 22nM) and very effectively disrupts the bivalent protein-protein interaction between menin and MLL.  MI-2-2 demonstrated specific and very pronounced activity in MLL leukemia cells, including inhibition of cell proliferation, down-regulation of Hoxa9 expression, and differentiation.  Our results provide the rational and essential structural basis to design next generation of inhibitors for effective targeting of the menin-MLL interaction in leukemia and demonstrate a proof of concept that inhibition of complex multivalent protein-protein interactions can be achieved by a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyxFC9mQS_4LVg90H21EOLACvtfcHk0liqKi1K_5vPkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVWgtb7J&md5=c1ab14816cf5ea3590fe03b0ed405ea5</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1182%2Fblood-2012-05-429274&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2012-05-429274%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DA.%26aulast%3DMurai%26aufirst%3DM.%2BJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DLund%26aufirst%3DG.%26aulast%3DHartley%26aufirst%3DT.%26aulast%3DPurohit%26aufirst%3DT.%26aulast%3DReddy%26aufirst%3DG.%26aulast%3DChruszcz%26aufirst%3DM.%26aulast%3DGrembecka%26aufirst%3DJ.%26aulast%3DCierpicki%26aufirst%3DT.%26atitle%3DStructural%2520Insights%2520into%2520Inhibition%2520of%2520the%2520Bivalent%2520Menin-MLL%2520Interaction%2520by%2520Small%2520Molecules%2520in%2520Leukemia%26jtitle%3DBlood%26date%3D2012%26volume%3D120%26spage%3D4461%26epage%3D4469%26doi%3D10.1182%2Fblood-2012-05-429274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aguilar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stuckey, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chinnaswamy, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernard, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernández-Salas, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McEachern, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Przybranowski, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foster, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span> <span> </span><span class="NLM_article-title">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1601</span>– <span class="NLM_lpage">1605</span>, <span class="refDoi"> DOI: 10.1002/anie.201711828</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fanie.201711828" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2018&pages=1601-1605&author=S.+Xuauthor=A.+Aguilarauthor=T.+Xuauthor=K.+Zhengauthor=L.+Huangauthor=J.+Stuckeyauthor=K.+Chinnaswamyauthor=D.+Bernardauthor=E.+Fern%C3%A1ndez-Salasauthor=L.+Liuauthor=M.+Wangauthor=D.+McEachernauthor=S.+Przybranowskiauthor=C.+Fosterauthor=S.+Wang&title=Design+of+the+First-in-Class%2C+Highly+Potent+Irreversible+Inhibitor+Targeting+the+Menin-MLL+Protein-Protein+Interaction&doi=10.1002%2Fanie.201711828"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design of the First-in-Class, Highly Potent Irreversible Inhibitor Targeting the Menin-MLL Protein-Protein Interaction</span></div><div class="casAuthors">Xu, Shilin; Aguilar, Angelo; Xu, Tianfeng; Zheng, Ke; Huang, Liyue; Stuckey, Jeanne; Chinnaswamy, Krishnapriya; Bernard, Denzil; Fernandez-Salas, Ester; Liu, Liu; Wang, Mi; McEachern, Donna; Przybranowski, Sally; Foster, Caroline; Wang, Shaomeng</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1605</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The structure-based design of M-525 (I) as the first-in-class, highly potent, irreversible small-mol. inhibitor of the menin-MLL interaction is presented.  M-525 targets cellular menin protein at sub-nanomolar concns. and achieves low nanomolar potencies in cell growth inhibition and in the suppression of MLL-regulated gene expression in MLL leukemia cells.  M-525 demonstrates high cellular specificity over non-MLL leukemia cells and is more than 30 times more potent than its corresponding reversible inhibitors.  Mass spectrometric anal. and co-crystal structure of M-525 in complex with menin firmly establish its mode of action.  A single administration of M-525 effectively suppresses MLL-regulated gene expression in tumor tissue.  An efficient procedure was developed to synthesize M-525.  This study demonstrates that irreversible inhibition of menin may be a promising therapeutic strategy for MLL leukemia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9fbcCaeUFsLVg90H21EOLACvtfcHk0liqKi1K_5vPkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXovVWgtA%253D%253D&md5=f7d189e2138b88a4d8c21c72586b7502</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1002%2Fanie.201711828&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201711828%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DS.%26aulast%3DAguilar%26aufirst%3DA.%26aulast%3DXu%26aufirst%3DT.%26aulast%3DZheng%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DStuckey%26aufirst%3DJ.%26aulast%3DChinnaswamy%26aufirst%3DK.%26aulast%3DBernard%26aufirst%3DD.%26aulast%3DFern%25C3%25A1ndez-Salas%26aufirst%3DE.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DMcEachern%26aufirst%3DD.%26aulast%3DPrzybranowski%26aufirst%3DS.%26aulast%3DFoster%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DS.%26atitle%3DDesign%2520of%2520the%2520First-in-Class%252C%2520Highly%2520Potent%2520Irreversible%2520Inhibitor%2520Targeting%2520the%2520Menin-MLL%2520Protein-Protein%2520Interaction%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2018%26volume%3D57%26spage%3D1601%26epage%3D1605%26doi%3D10.1002%2Fanie.201711828" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Morgan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El-Tanani, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunter, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harrington, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandha, H. S.</span></span> <span> </span><span class="NLM_article-title">Targeting HOX/PBX dimers in cancer</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">32322</span>– <span class="NLM_lpage">32331</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.15971</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.18632%2Foncotarget.15971" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=32322-32331&author=R.+Morganauthor=M.+El-Tananiauthor=K.+D.+Hunterauthor=K.+J.+Harringtonauthor=H.+S.+Pandha&title=Targeting+HOX%2FPBX+dimers+in+cancer&doi=10.18632%2Foncotarget.15971"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.15971&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.15971%26sid%3Dliteratum%253Aachs%26aulast%3DMorgan%26aufirst%3DR.%26aulast%3DEl-Tanani%26aufirst%3DM.%26aulast%3DHunter%26aufirst%3DK.%2BD.%26aulast%3DHarrington%26aufirst%3DK.%2BJ.%26aulast%3DPandha%26aufirst%3DH.%2BS.%26atitle%3DTargeting%2520HOX%252FPBX%2520dimers%2520in%2520cancer%26jtitle%3DOncotarget%26date%3D2015%26volume%3D8%26spage%3D32322%26epage%3D32331%26doi%3D10.18632%2Foncotarget.15971" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Munde, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, G. M. K.</span></span> <span> </span><span class="NLM_article-title">Structure-Dependent Inhibition of the ETS-Family Transcription Factor PU.1 by Novel Heterocyclic Diamidines</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">1379</span>– <span class="NLM_lpage">1390</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt955</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkt955" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=1379-1390&author=M.+Mundeauthor=S.+Wangauthor=A.+Kumarauthor=C.+E.+Stephensauthor=A.+A.+Farahatauthor=D.+W.+Boykinauthor=W.+D.+Wilsonauthor=G.+M.+K.+Poon&title=Structure-Dependent+Inhibition+of+the+ETS-Family+Transcription+Factor+PU.1+by+Novel+Heterocyclic+Diamidines&doi=10.1093%2Fnar%2Fgkt955"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt955&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt955%26sid%3Dliteratum%253Aachs%26aulast%3DMunde%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DPoon%26aufirst%3DG.%2BM.%2BK.%26atitle%3DStructure-Dependent%2520Inhibition%2520of%2520the%2520ETS-Family%2520Transcription%2520Factor%2520PU.1%2520by%2520Novel%2520Heterocyclic%2520Diamidines%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D42%26spage%3D1379%26epage%3D1390%26doi%3D10.1093%2Fnar%2Fgkt955" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anbazhagan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis of benzyloxybromobenzonitriles</span>. <i>Synth. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">751</span>– <span class="NLM_lpage">758</span>, <span class="refDoi"> DOI: 10.1081/scc-120028347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1081%2Fscc-120028347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFylsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2004&pages=751-758&author=M.+A.+Ismailauthor=M.+Anbazhaganauthor=C.+E.+Stephensauthor=D.+W.+Boykin&title=Synthesis+of+benzyloxybromobenzonitriles&doi=10.1081%2Fscc-120028347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis of benzyloxybromobenzonitriles</span></div><div class="casAuthors">Ismail, Mohamed A.; Anbazhagan, Mariappan; Stephens, Chad E.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Synthetic Communications</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">751-758</span>CODEN:
                <span class="NLM_cas:coden">SYNCAV</span>;
        ISSN:<span class="NLM_cas:issn">0039-7911</span>.
    
            (<span class="NLM_cas:orgname">Marcel Dekker, Inc.</span>)
        </div><div class="casAbstract">The synthesis of four new benzyloxybromobenzonitriles including: 2-benzyloxy-4-bromobenzonitrile, 3-benzyloxy-4-bromobenzonitrile, 5-benzyloxy-2-bromobenzonitrile, and 3-benzyloxy-2-bromobenzonitrile is reported.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYxuYazkd4MrVg90H21EOLACvtfcHk0lgDnZjUJdpumA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFylsL0%253D&md5=d61f8813faeb4fa875212c21199a1823</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1081%2Fscc-120028347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1081%252Fscc-120028347%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DAnbazhagan%26aufirst%3DM.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520of%2520benzyloxybromobenzonitriles%26jtitle%3DSynth.%2520Commun.%26date%3D2004%26volume%3D34%26spage%3D751%26epage%3D758%26doi%3D10.1081%2Fscc-120028347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barghash, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goda, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisa, H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Bichalcophenes: A Concise Synthesis of Formyl Ester- and Cyano Ester-Substituted Bithiophenes, Bifurans, and Furanothiophenes</span>. <i>J. Heterocyclic Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">170</span>, <span class="refDoi"> DOI: 10.1002/jhet.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fjhet.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=24634543" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht12ktrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2010&pages=167-170&author=A.+A.+Farahatauthor=A.+Kumarauthor=A.+M.+Barghashauthor=F.+E.+Godaauthor=H.+M.+Eisaauthor=D.+W.+Boykin&title=Bichalcophenes%3A+A+Concise+Synthesis+of+Formyl+Ester-+and+Cyano+Ester-Substituted+Bithiophenes%2C+Bifurans%2C+and+Furanothiophenes&doi=10.1002%2Fjhet.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Bichalcophenes: A concise synthesis of formyl ester- and cyano ester-substituted bithiophenes, bifurans, and furanothiophenes</span></div><div class="casAuthors">Farahat, Abdelbasset A.; Kumar, Arvind; Barghash, Alaa El-Din M.; Goda, Fatma E.; Eisa, Hassan M.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Heterocyclic Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">167-170</span>CODEN:
                <span class="NLM_cas:coden">JHTCAD</span>;
        ISSN:<span class="NLM_cas:issn">1943-5193</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Syntheses of formyl- and cyano methoxycarbonyl-substituted 2,2'-bithiophenes, 2,2'-bifurans, and 2-furan-2-ylthiophenes in good yield were described.  The key synthetic step utilized Stille coupling of appropriate brominated 5-ring heterocycles with stannylated 5-ring heterocycles.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpOxzOisylM_7Vg90H21EOLACvtfcHk0lj8B4ZgCYPkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht12ktrk%253D&md5=32661d2dff161ef9b295f29ce18f2daf</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1002%2Fjhet.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjhet.295%26sid%3Dliteratum%253Aachs%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DBarghash%26aufirst%3DA.%2BM.%26aulast%3DGoda%26aufirst%3DF.%2BE.%26aulast%3DEisa%26aufirst%3DH.%2BM.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DBichalcophenes%253A%2520A%2520Concise%2520Synthesis%2520of%2520Formyl%2520Ester-%2520and%2520Cyano%2520Ester-Substituted%2520Bithiophenes%252C%2520Bifurans%252C%2520and%2520Furanothiophenes%26jtitle%3DJ.%2520Heterocyclic%2520Chem.%26date%3D2010%26volume%3D47%26spage%3D167%26epage%3D170%26doi%3D10.1002%2Fjhet.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Dicationic biphenyl benzimidazole derivatives as antiprotozoal agents</span>. <i>Bioorg. & Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">5405</span>– <span class="NLM_lpage">5413</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2004.07.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bmc.2004.07.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15388167" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXnvFWksrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2004&pages=5405-5413&author=M.+A.+Ismailauthor=R.+Brunauthor=T.+Wenzlerauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Dicationic+biphenyl+benzimidazole+derivatives+as+antiprotozoal+agents&doi=10.1016%2Fj.bmc.2004.07.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Dicationic biphenyl benzimidazole derivatives as antiprotozoal agents</span></div><div class="casAuthors">Ismail, Mohamed A.; Brun, Reto; Wenzler, Tanja; Tanious, Farial A.; Wilson, W. David; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">5405-5413</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A series of biphenyl benzimidazoles diamidines were synthesized from their resp. diamidoximes, through the bis-O-acetoxyamidoxime followed by hydrogenation in glacial acetic acid/ethanol in the presence of Pd-C.  The target compds. contain hydroxy and/or methoxy substituted 1,3-Ph groups as the central spacer between the two amidino bearing aryl groups.  All of the diamidines showed strong DNA affinities as judged by high ΔTm values with poly(dA·dT)2, which varied with structure and is discussed.  Seven of the nine new diamidines gave in vitro IC50 values of approx. 30 nM or less vs. Trypanosoma brucei rhodesiense (T.b.r.).  Generally the diamidines were less active vs. Plasmodium falciparum (P.f.), however one compd. exhibited excellent activity with an IC50 value of 2.1 nM.  Five of the nine diamidines exhibited excellent in vivo activity in the trypanosomal STIB900 mouse model giving 3/4 or 4/4 cures at dosage of 20 mg/kg i.p. and three showed similar efficacy at dosage of 10 mg/kg or lower.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-c9VNmcRZx7Vg90H21EOLACvtfcHk0lj8B4ZgCYPkIg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXnvFWksrc%253D&md5=acfad2e365ee89fc0d8986a43b171db5</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2004.07.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2004.07.056%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DWenzler%26aufirst%3DT.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DDicationic%2520biphenyl%2520benzimidazole%2520derivatives%2520as%2520antiprotozoal%2520agents%26jtitle%3DBioorg.%2520%2526%2520Med.%2520Chem.%26date%3D2004%26volume%3D12%26spage%3D5405%26epage%3D5413%26doi%3D10.1016%2Fj.bmc.2004.07.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wenzler, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Novel Dicationic Imidazo[1,2-a]pyridines and 5,6,7,8-Tetrahydro-imidazo[1,2-a]pyridines as Antiprotozoal Agents</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">3658</span>– <span class="NLM_lpage">3664</span>, <span class="refDoi"> DOI: 10.1021/jm0400092</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0400092" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksVejurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=3658-3664&author=M.+A.+Ismailauthor=R.+Brunauthor=T.+Wenzlerauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Novel+Dicationic+Imidazo%5B1%2C2-a%5Dpyridines+and+5%2C6%2C7%2C8-Tetrahydro-imidazo%5B1%2C2-a%5Dpyridines+as+Antiprotozoal+Agents&doi=10.1021%2Fjm0400092"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Dicationic Imidazo[1,2-a]pyridines and 5,6,7,8-Tetrahydro-imidazo[1,2-a]pyridines as Antiprotozoal Agents</span></div><div class="casAuthors">Ismail, Mohamed A.; Brun, Reto; Wenzler, Tanja; Tanious, Farial A.; Wilson, W. David; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3658-3664</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Acetate or hydrochloride salts of 2-[5-(4-amidinophenyl)-furan-2-yl]-imidazo[1,2-a]pyridine-6-carboxamidines I (R1 = H, MeO; R2 = H, Me) and of the 5,6,7,8-tetrahydro analog (R1 = R2 = H) were synthesized from imidazo[1,2-a]pyridine-6-carbonitrile II via its bis-O-acetoxyamidoxime.  II were obtained in four steps starting with two successive brominations of 2-acetylfuran first with N-bromosuccinimide, and second with bromine to form α-bromo-2-acetyl-5-bromofuran in a moderate yield.  The condensation reaction of the latter with 6-aminonicotinonitrile or 6-amino-5-methylnicotinonitrile followed by Suzuki coupling with 4-cyanophenylboronic acid gave II in good yields.  The analogs with 1,4-phenylene and 1,3-phenylene bridges instead of 2,5-furanyl bridge were prepd. similarly.  By these approaches eight new diamidines and four potential prodrugs were prepd.  All of the diamidines showed strong DNA affinities as judged by high ΔTm values.  Six of the eight diamidines gave in vitro IC50 values of 63 nM or less vs T. b. rhodesiense with two exhibiting values of 6 nM and 1 nM.  Also, six of the eight diamidines gave in vitro IC50 values of 88 nM or less vs P. falciparum with two exhibiting values of 14 nM.  Excellent in vivo activity in the trypanosomal STIB900 mouse model was found for five of the diamidines on i.p. dosage; these compds. gave 4/4 cures in this model.  The oral activity of the prodrugs was modest with only one showing 2/4 cures in the same mouse model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoL3SdrHukHxbVg90H21EOLACvtfcHk0li8eu60cIi3pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksVejurs%253D&md5=eb506ce99fd11e93837b1d5155422973</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm0400092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0400092%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DWenzler%26aufirst%3DT.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DNovel%2520Dicationic%2520Imidazo%255B1%252C2-a%255Dpyridines%2520and%25205%252C6%252C7%252C8-Tetrahydro-imidazo%255B1%252C2-a%255Dpyridines%2520as%2520Antiprotozoal%2520Agents%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D3658%26epage%3D3664%26doi%3D10.1021%2Fjm0400092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chaires, J.
B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hamelberg, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandya, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Structural Selectivity of Aromatic Diamidines</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">5729</span>– <span class="NLM_lpage">5742</span>, <span class="refDoi"> DOI: 10.1021/jm049491e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049491e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5729-5742&author=J.%0AB.+Chairesauthor=J.+Renauthor=D.+Hamelbergauthor=A.+Kumarauthor=V.+Pandyaauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=Structural+Selectivity+of+Aromatic+Diamidines&doi=10.1021%2Fjm049491e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Selectivity of Aromatic Diamidines</span></div><div class="casAuthors">Chaires, Jonathan B.; Ren, Jinsong; Hamelberg, Donald; Kumar, Arvind; Pandya, Vandna; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">5729-5742</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Competition dialysis was used to study the interactions of 13 substituted arom. diamidine compds. with 13 nucleic acid structures and sequences.  The results show a striking selectivity of these compds. for the triplex structure poly dA:(poly dT)2, a novel aspect of their interaction with nucleic acids not previously described.  The triplex selectivity of selected compds. was confirmed by thermal denaturation studies.  Triplex selectivity was found to be modulated by the location of amidine substituents on the core phenyl-furan-Ph ring scaffold.  Mol. models were constructed to rationalize the triplex selectivity of DB359, the most selective compd. in the series.  Its triplex selectivity was found to arise from optimal ring stacking on base triplets, along with proper positioning of its amidine substituents to occupy the minor and the major-minor grooves of the triplex.  New insights into the mol. recognition of nucleic acid structures emerged from these studies, adding to the list of available design principles for selectively targeting DNA and RNA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8-gzBJdMjo7Vg90H21EOLACvtfcHk0li8eu60cIi3pQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXotVCisL0%253D&md5=303d146fba1b5e650230142c955ad530</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1021%2Fjm049491e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049491e%26sid%3Dliteratum%253Aachs%26aulast%3DChaires%26aufirst%3DJ.%2BB.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DHamelberg%26aufirst%3DD.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DPandya%26aufirst%3DV.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DStructural%2520Selectivity%2520of%2520Aromatic%2520Diamidines%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5729%26epage%3D5742%26doi%3D10.1021%2Fjm049491e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ismail, M.
A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brun, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Easterbrook, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Antiprotozoal Activity of Aza-Analogues of Furamidine</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">4761</span>– <span class="NLM_lpage">4769</span>, <span class="refDoi"> DOI: 10.1021/jm0302602</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0302602" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=4761-4769&author=M.%0AA.+Ismailauthor=R.+Brunauthor=J.+D.+Easterbrookauthor=F.+A.+Taniousauthor=W.+D.+Wilsonauthor=D.+W.+Boykin&title=Synthesis+and+Antiprotozoal+Activity+of+Aza-Analogues+of+Furamidine&doi=10.1021%2Fjm0302602"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm0302602&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0302602%26sid%3Dliteratum%253Aachs%26aulast%3DIsmail%26aufirst%3DM.%2BA.%26aulast%3DBrun%26aufirst%3DR.%26aulast%3DEasterbrook%26aufirst%3DJ.%2BD.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DSynthesis%2520and%2520Antiprotozoal%2520Activity%2520of%2520Aza-Analogues%2520of%2520Furamidine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D4761%26epage%3D4769%26doi%3D10.1021%2Fjm0302602" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span> <span> </span><span class="NLM_article-title">Palladium catalyzed reactions for the synthesis of 2,5-disubstituted furans</span>. <i>Heterocycl. Commun.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">301</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1515/hc.1999.5.4.301</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1515%2Fhc.1999.5.4.301" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK1MXmtl2ku7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1999&pages=301-304&author=A.+Kumarauthor=C.+E.+Stephensauthor=D.+W.+Boykin&title=Palladium+catalyzed+reactions+for+the+synthesis+of+2%2C5-disubstituted+furans&doi=10.1515%2Fhc.1999.5.4.301"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Palladium catalyzed cross-coupling reactions for the synthesis of 2,5-disubstituted furans</span></div><div class="casAuthors">Kumar, Arvind; Stephens, Chad E.; Boykin, David W.</div><div class="citationInfo"><span class="NLM_cas:title">Heterocyclic Communications</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">301-304</span>CODEN:
                <span class="NLM_cas:coden">HCOMEX</span>;
        ISSN:<span class="NLM_cas:issn">0793-0283</span>.
    
            (<span class="NLM_cas:orgname">Freund Publishing House Ltd.</span>)
        </div><div class="casAbstract">A convenient synthesis of 2,5-bisarylfurans, 2,5-bisbenzylfurans and 2,5-bisbenzoylfurans employing 2,5-bis(tri-n-butylstannyl)furan (I) and palladium catalyzed cross-coupling reactions is described.  Thus, reaction of I with RC6H4X (R = 4-cyano, 3-cyano, 4-O2N, 3-O2N, 4-EtO2C; X = Br, iodo) in dioxane contg. Pd(PPh34)4 gave 70-86% diphenylfurans II.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXNaaVdNF6ObVg90H21EOLACvtfcHk0lh2zSS1k77kSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmtl2ku7w%253D&md5=43aa5c35f5cbd48c882c23c47ec32125</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1515%2Fhc.1999.5.4.301&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252Fhc.1999.5.4.301%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26atitle%3DPalladium%2520catalyzed%2520reactions%2520for%2520the%2520synthesis%2520of%25202%252C5-disubstituted%2520furans%26jtitle%3DHeterocycl.%2520Commun.%26date%3D1999%26volume%3D5%26spage%3D301%26epage%3D304%26doi%3D10.1515%2Fhc.1999.5.4.301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sweat, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stephens, C. E.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Stille cross-coupling reactions of 2-(tributylstannyl)- and 2,5-bis(tributylstannyl)tellurophene</span>. <i>Synthesis</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3214</span>– <span class="NLM_lpage">3218</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2009&pages=3214-3218&author=D.+P.+Sweatauthor=C.+E.+Stephens&title=Synthesis+and+Stille+cross-coupling+reactions+of+2-%28tributylstannyl%29-+and+2%2C5-bis%28tributylstannyl%29tellurophene"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSweat%26aufirst%3DD.%2BP.%26aulast%3DStephens%26aufirst%3DC.%2BE.%26atitle%3DSynthesis%2520and%2520Stille%2520cross-coupling%2520reactions%2520of%25202-%2528tributylstannyl%2529-%2520and%25202%252C5-bis%2528tributylstannyl%2529tellurophene%26jtitle%3DSynthesis%26date%3D2009%26volume%3D5%26spage%3D3214%26epage%3D3218" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Buu-Hoi, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Royer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xuong, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacquignon, P.</span></span> <span> </span><span class="NLM_article-title">The Pfitzinger reaction in the synthesis of quinoline derivatives</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1953</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">1209</span>– <span class="NLM_lpage">1224</span>, <span class="refDoi"> DOI: 10.1021/jo50015a019</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo50015a019" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaG2MXhtlCq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=1953&pages=1209-1224&author=N.+P.+Buu-Hoiauthor=R.+Royerauthor=N.+D.+Xuongauthor=P.+Jacquignon&title=The+Pfitzinger+reaction+in+the+synthesis+of+quinoline+derivatives&doi=10.1021%2Fjo50015a019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">The Pfitzinger reaction in the synthesis of quinoline derivatives</span></div><div class="casAuthors">Buu-Hoi, Ng. Ph.; Royer, R.; Xuong, Ng. D.; Jacquignon, P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1953</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1209-24</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    </div><div class="casAbstract">A no. of substituted cinchoninic acids and quinolines are prepd. by the Pfitzinger reaction by condensation of isatin (I) and substituted I with ketones, RCOCH2R', in an alk. medium.  Refluxing 6 g. PrCOC6H13, 5.9 g. I, and 6.9 g. KOH in 25 cc.  EtOH 24 h., distg. off the EtOH, taking up the residue in H2O, and acidifying the washed (ether) aq. soln. with AcOH give 6 g. of a mixt. of cinchoninic acids from which, on repeated crystn., is isolated a product that seems to be 3-ethyl-2-hexylcinchoninic acid (II), needles, m. 108-9°.  5-Bromoisatin (III) in lieu of I gives the 6-Br analog of II, needles, m. 162-3°.  Refluxing 2 g. III, 2 g. 2-methylcyclohexanone, and 2 g. KOH in EtOH gives 80% 7-bromo-4-methyl-1,2,3,4-tetrahydroacridine-9-carboxylic acid (IV), yellowish needles, m. 272-3° (decompn.), decarboxylated to the corresponding acridine (V) (picrate, yellow prisms, m. 196-7°).  7-Cl analog of IV, silky needles, m. 224-5°; 4-Et homolog of IV, yellowish needles, m. 298-9°; 4-Et homolog of V, slender pale yellow prisms, m. 92°.  Refluxing 2 g. β-naphthisatin (3H-benz[e]indole-1,2-dione), 2 g. MeCOC6H13 and 1.7 g. KOH in 10 cc. EtOH 24 h. gives 2.4 g. 2-hexyl-5,6-benzocinchoninic acid (VI), pale yellow needles, m. 245-6° (decompn.), decarboxylated to 2-hexyl-5,6-benzoquinoline (picrate, silky yellow needles, m. 147°).  VI is also obtained from 2-C10H7NH2 (VII), C6H13CHO, and MeCOCO2H (VIII).  2-C9H19 homolog of VI, m. 245-7°, is also obtained from VII, C9H19CHO, and VIII.  6-Chloro-2-phenylcinchoninic acid (IX), prepd. in 90% yield from 5-chloroisatin, PhAc, and KOH, prisms, m. 282°; 6-chloro-2-phenylquinoline, silky needles, m. 110-11° (picrate, m. 189-90°); 2-p-tolyl homolog of IX, needles, m. 270-1°; 6-methyl-2-(p-chlorophenyl)cinchoninic acid, silky prisms, m. 219-20°; 6-methyl-2-(p-chlorophenyl)quinoline, silky needles, m. 164° (picrate, m. 185-6°); 3,6-dimethyl-2-(p-chlorophenyl)cinchoninic acid, needles, does not m. below 320°. p-PhCH2COC6H4SPh, prepd. in 75% yield from BzCH2Cl and AlCl3 in CS2, shiny leaflets, m. 105-6%; 4-iso-PrCOC6H4SPh, mobile oil, b12 224-6°, nD23 1.6234; 4-caproyl homolog, pale yellow oil, b0.9 204-5°, nD23.5 1.6062, leaflets, m. 29-30°; 4-octanoyl homolog, pale yellow oil, b2 219-20°, nD23 1.5934, m. 31-2°; 4-Bz analog, slender prisms, m. 70°; 4-(p-MeOC6H4CO) analog, silky needles, m. 81°.  Friedel-Crafts acylations of PhSEt according to Auwers and Arndt (C.A. 3, 2577) gives 4-EtCOC6H4SEt, b15 178-80°, large prisms, m. 45°; 4-PrCO homolog, b15 189-90°, large plates, m. 43°.  The substituted cinchoninic acids (X) and quinolines (XI) prepd. by these methods are listed [substituent, m.p. (°C.) of X, b.p. (°C.) of XI (picrate m.p.) given]: 2-Undecyl, 129-30, 233/14 (83-4); 2-Undecyl-4-hydroxy, -, 270-80/14 (-); 8-Methyl-2-undecyl, 83-4, 235-8/14 (76); 2-Amyl, 135, - (-); 2-Amyl-4-hydroxy, -, m. 128 (-); 6-Methyl-2-amyl, 174, 300-2 (188-9); 6-Bromo-2-amyl, 180, 207-8/14 (192); 6-Methyl-2-hexyl, 135, - (-); 6-Chloro-2-hexyl, 143, - (-); 3-Methyl-2-hexyl, 197, - (-); 3-Methyl-2-pentadecyl, -, m. 55 (105-6); 3-Ethyl-2-iso-Bu, 239, 298-300 (160-1); 3-Propyl-2-iso-Bu, m. 175, resolidifies, m. 205, - (-); 6-Bromo-3-methyl-2-pentadecyl, 189, m. 54-5 (114); 2-Cyclopropyl, 206, 145-8/17 (175); 8-Methyl-2-cyclopropyl, 212b, - (-); 6-Chloro-2-cyclopropyl, 230b, - (-); 6-Bromo-2-cyclopropyl, 268-70b, - (-); 6-Methyl-2,3-pentamethylene, 308b, m. 105 (189-90); 6-Bromo-2,3-pentamethylene, 309, m. 128 (200-1); 6-Chloro-2,3-pentamethylene, 251,m. 109 (191-2); b = decompn.  The 2-arylcinchoninic acids (XII) (XIIa, R = CO2H, R'' = H), their corresponding quinolines (XIII) (XIIa, R = R'' = H), the 2-aryl-6-bromo-cinchoninic acids (XIV) (XIIa, R = CO2H, R'' = Br), and their corresponding quinolines (XV) (XIIa, R = H, R'' = Br) are listed [Ar, R', m.p. XII, m.p. XIII (picrate m.p.), m.p. XIV, m.p. XV (picrate m.p.) given]: 4-PhC6H4, H, -, - (-), 306, 222 (224-5); 4-PhC6H4, Me, 303-4, 121 (230), 316-17, 196 (-); 4-PhC6H4, Et, > 310, 100 (211), 292, 172 (215); 2-C10H11c, Et, -, - (-), 310, - (-); 6-C10H11c, H, -, - (-), 235, 133 (212); 6-C10H11c Me, 254-5, 70 (-), 316, 123 (216); 6-C10H11c, Et, > 300a b16 275-80 (196-7), -, 97 (217-18); 2-Ph, Me, -, - (-), 328, 129 (-); 4-Tolyl, Me, > 305, 59 (220), -, - (-); 4-Tolyl, Et, > 310, - (-), -, - (-); 3,4-Me2C6H3, H, 209-10, 90-1 (-), 227, 135 (206-7); 2,5-Me2C6H3, H, 200-1, 60 (179), 239, 124 (205); 2,5-Me2C6H3, Me, 301, - (-), 345, 109 (192); 2,4-Me2C6H3, H, -, - (-), 235, 98 (194); 2,4-Me2C6H3, Me, > 305, - (-), > 321, 122 (209);  4-EtC6H4, H, 190, 50 (196-7), -, - (-); 4-EtC6H4, Me, 290-2a b16 220-5 (191-2), 301, 115 (-); 4-EtC6H4, Et, 296, b16 228-30 (182-3), -, - (-); 4-Iso-PrC6H4, H, 195, 50 (190), 218, 143 (194); 4-Iso-PrC6H4, Me, 297, 61 (193), 298-9, 130 (185); 4-Iso-PrC6H4, Et, 267, - (-), 275-6, 93 (204); 4-Me3CC6H4, Me, 298, 66b (-), >311, 132 (154); 2,4,5-Me3C6H2, Me, 313, b13 230 (212-13), -, (-); 4,1-MeC10H6, Me, >305, b13 270-2 (224), >307, 181 (228); 5-Acenaphthenyl, Me, -, - (-), >310, - (-); 3-Pyrenyl, H, -, - (-), >310, 205 (240-2); 3-Pyrenyl, Me, >310, - (-), >310, - (-); 3-Pyrenyl, Et, >310, - (-), >310, - (-); 4-ClC6H4, H, 241-2, 112 (192-3), 271, 177 (-); 4-ClC6H4, Me, 310-11, 92 (-), 307, 160 (206-7); 4-ClC6H4, Et, 310, 42 (181), >310, 147 (196); 4-ClC6H4, Ph, -, - (-), >328, 161 (194); 3,4-Cl2C6H3, H, 258, 112 (204), 252, 169 (192); 4-BrC6H4, H, 250, 123-4 (181-2), 259, 188 (207); 4-BrC6H4, Me, >320, 108 (195-6), 325, 189-90 (163);  4-BrC6H4, Et, 329-30, 62 (192), >310, 110 (-); 4-BrC6H4, CH2CO2H, -, - (-), 307, - (-); 4-BrC6H4, Ph, -, - (-), >310, 171-2 (192-3); 4-IC6H4, H, -, - (-), 277-8, 185 (197-8); 2,4-(MeO)2C6H3, Ph, -, - (-), 297-8, 89 (202-3); 3,4-(MeO)2C6H3, Et, -, - (-), 240, - (-); a.  Decompn. b. B. 250-60°/15 mm. c.  Tetrahydronaphthyl. Ar, R, m.p. XII, m.p. XIII (picrate m.p.) given: 4-EtOC6H4, H, 222, 127 (218); 4-EtOC6H4, Me, 302, 92 (177); 4-EtOC6H4, Et, 305, oil (190); 4-EtOC6H4, Ph, 295, 100 (192); 4-MeSC6H4, Me, 305, 86 (180); 4-Me-SC6H4, CH2CO2H, 325, - (-); 4-EtSC6H4, H, 195, - (-); 4-EtSC6H4, Me, 278, 70 (186); 4-EtSC6H4, Et, 269, oil (184); 4-PhOC6H4, H, 203, 121 (-); 4-PhOC6H4, Me, 303, 91 (-); 4-PhOC6H4, Et, 282, 75 (188); 4-PhOC6H4, Ph, 283, 109a (201); 4-PhSC6H4, H, 215, 119 (-); 4-PhSC6H4, Me, 275, 76 (175); 4-PhSC6H4, Ph, 289, 62 (192); 4-(4-MeOC6H4)C6H4, H, 237, 219 (202); 4-(4-MeOC6H4)-C6H4, Me, 295, 149 (180); a. HBr salt, m. 215°.  The following 6-bromo-2-thienylcinchoninic acids substituted in the thiophene ring (XVI) (R'' = CO2H) and 6-bromo-2-(2-thienyl)quinolines (XVI, R'' = H) (a) are prepd. (R, R',m.p. given): H, H, 271-2°, a, yellowish needles, m. 124°; H, Me, 290-1°, a, m. 90°; Cl, H, 270-1°; Cl, Me, 278-9°; Cl, Et, 281-2°; Br, H, 247-8°, a, m. 175-6°; Br, Me, 284-5°, a, yellow needles, m. 147-8°; Br, Et, 260-1°; Br, Ph, 274-5°.  6-Bromo-2,5-dimethyl-3(2-thienyl)cinchoninic acid m. 234-5°.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCnySqY_eXsrVg90H21EOLACvtfcHk0lgkPHM4ZiEOug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaG2MXhtlCq&md5=c75a824431b320cac72314e20847fa2a</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fjo50015a019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo50015a019%26sid%3Dliteratum%253Aachs%26aulast%3DBuu-Hoi%26aufirst%3DN.%2BP.%26aulast%3DRoyer%26aufirst%3DR.%26aulast%3DXuong%26aufirst%3DN.%2BD.%26aulast%3DJacquignon%26aufirst%3DP.%26atitle%3DThe%2520Pfitzinger%2520reaction%2520in%2520the%2520synthesis%2520of%2520quinoline%2520derivatives%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1953%26volume%3D18%26spage%3D1209%26epage%3D1224%26doi%3D10.1021%2Fjo50015a019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kalluraya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sreenivasa, S.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Pharmacological properties of some quinoline derivatives</span>. <i>Il Farmaco</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">399</span>– <span class="NLM_lpage">404</span>, <span class="refDoi"> DOI: 10.1016/s0014-827x(98)00037-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fs0014-827x%2898%2900037-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1998&pages=399-404&author=B.+Kallurayaauthor=S.+Sreenivasa&title=Synthesis+and+Pharmacological+properties+of+some+quinoline+derivatives&doi=10.1016%2Fs0014-827x%2898%2900037-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fs0014-827x%2898%2900037-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fs0014-827x%252898%252900037-8%26sid%3Dliteratum%253Aachs%26aulast%3DKalluraya%26aufirst%3DB.%26aulast%3DSreenivasa%26aufirst%3DS.%26atitle%3DSynthesis%2520and%2520Pharmacological%2520properties%2520of%2520some%2520quinoline%2520derivatives%26jtitle%3DIl%2520Farmaco%26date%3D1998%26volume%3D53%26spage%3D399%26epage%3D404%26doi%3D10.1016%2Fs0014-827x%2898%2900037-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ajdačić, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senerovic, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vranić, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pekmezovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arsic-Arsnijevic, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselinovic, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselinovic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Šolaja, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikodinovic-Runic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Opsenica, I. M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and evaluation of thiophene-based quanylhydrazones(iminoguanidines) efficient against panel of voriconazole-resistant fungal isolates</span>. <i>Bioorg. & Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">1277</span>– <span class="NLM_lpage">1291</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2016.01.058</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.bmc.2016.01.058" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=26867487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1antrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=1277-1291&author=V.+Ajda%C4%8Di%C4%87author=L.+Senerovicauthor=M.+Vrani%C4%87author=M.+Pekmezovicauthor=V.+Arsic-Arsnijevicauthor=A.+Veselinovicauthor=J.+Veselinovicauthor=B.+A.+%C5%A0olajaauthor=J.+Nikodinovic-Runicauthor=I.+M.+Opsenica&title=Synthesis+and+evaluation+of+thiophene-based+quanylhydrazones%28iminoguanidines%29+efficient+against+panel+of+voriconazole-resistant+fungal+isolates&doi=10.1016%2Fj.bmc.2016.01.058"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of thiophene-based guanylhydrazones (iminoguanidines) efficient against panel of voriconazole-resistant fungal isolates</span></div><div class="casAuthors">Ajdacic, Vladimir; Senerovic, Lidija; Vranic, Marija; Pekmezovic, Marina; Arsic-Arsnijevic, Valentina; Veselinovic, Aleksandar; Veselinovic, Jovana; Solaja, Bogdan A.; Nikodinovic-Runic, Jasmina; Opsenica, Igor M.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1277-1291</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A series of new thiophene-based guanylhydrazones (iminoguanidines) was synthesized in high yields using a straightforward two-step procedure.  The antifungal activity of the compds. was evaluated against a wide range of medically important fungal strains, including yeasts, molds, and dermatophytes, in comparison to the clin. used drug voriconazole.  Cytotoxic properties of the compds. were also detd. using a human lung fibroblast cell line and a hemolysis assay.  All guanylhydrazones showed significant activity against a broad spectrum of clin. important species of Candida spp., Aspergillus fumigatus, Fusarium oxysporum, Microsporum canis and Trichophyton mentagrophytes that was in some cases comparable or better than the activity of voriconazole.  More importantly, compds. 10, 11, 13, 14, 18 and 21 exhibited excellent activity against voriconazole-resistant Candida albicans CA5 with very low minimal inhibitory concn. (MIC) values <2 μg mL-1.  Deriv. 14, bearing bromine on the Ph ring, was the most effective compd. with MICs ranging from 0.25 to 6.25 μg mL-1.  However, bis-guanylhydrazone 18 showed better selectivity in terms of therapeutic index values.  In vivo embryotoxicity on zebrafish (Danio rerio) showed an improved toxicity profile of 11, 14 and 18 in comparison to that of voriconazole.  Most guanylhydrazones also inhibited C. albicans yeast to hyphal transition, essential for its biofilm formation, while 11 and 18 were able to disperse preformed Candida biofilms.  All guanylhydrazones showed the equal potential to interact with genomic DNA of C. albicans in vitro, thus indicating a possible mechanism of their action, as well as possible mechanism of obsd. cytotoxic effects.  Tested compds. did not have significant hemolytic effect and caused low liposome leakage, which excluded the cell membrane as a primary target.  On the basis of computational docking expts. using both human and cytochrome P 450 from Candida, it was concluded that the most active guanylhydrazones had minimal structural prerequisites to interact with cytochrome P 450 14α-demethylase (CYP51).  Promising guanylhydrazone derivs. also showed satisfactory pharmacokinetic profiles based on mol. calcns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbvwK5BWnR3rVg90H21EOLACvtfcHk0lhq9Lq1cc2qpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1antrc%253D&md5=e0605f4cc2f1e07fe613b733b10c3af5</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2016.01.058&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2016.01.058%26sid%3Dliteratum%253Aachs%26aulast%3DAjda%25C4%258Di%25C4%2587%26aufirst%3DV.%26aulast%3DSenerovic%26aufirst%3DL.%26aulast%3DVrani%25C4%2587%26aufirst%3DM.%26aulast%3DPekmezovic%26aufirst%3DM.%26aulast%3DArsic-Arsnijevic%26aufirst%3DV.%26aulast%3DVeselinovic%26aufirst%3DA.%26aulast%3DVeselinovic%26aufirst%3DJ.%26aulast%3D%25C5%25A0olaja%26aufirst%3DB.%2BA.%26aulast%3DNikodinovic-Runic%26aufirst%3DJ.%26aulast%3DOpsenica%26aufirst%3DI.%2BM.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520thiophene-based%2520quanylhydrazones%2528iminoguanidines%2529%2520efficient%2520against%2520panel%2520of%2520voriconazole-resistant%2520fungal%2520isolates%26jtitle%3DBioorg.%2520%2526%2520Med.%2520Chem.%26date%3D2016%26volume%3D24%26spage%3D1277%26epage%3D1291%26doi%3D10.1016%2Fj.bmc.2016.01.058" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farahat, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boykin, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">The Thiophene ″Sigma-Hole″ as a Concept for Preorganized, Specific Recognition of G·C Base Pairs in the DNA Minor Groove</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">15404</span>– <span class="NLM_lpage">15412</span>, <span class="refDoi"> DOI: 10.1002/chem.201603422</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fchem.201603422" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=27624927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsV2rur%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=15404-15412&author=P.+Guoauthor=A.+Paulauthor=A.+Kumarauthor=A.+A.+Farahatauthor=D.+Kumarauthor=S.+Wangauthor=D.+W.+Boykinauthor=W.+D.+Wilson&title=The+Thiophene+%E2%80%B3Sigma-Hole%E2%80%B3+as+a+Concept+for+Preorganized%2C+Specific+Recognition+of+G%C2%B7C+Base+Pairs+in+the+DNA+Minor+Groove&doi=10.1002%2Fchem.201603422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">The Thiophene "Sigma-Hole" as a Concept for Preorganized, Specific Recognition of G-C Base Pairs in the DNA Minor Groove</span></div><div class="casAuthors">Guo, Pu; Paul, Ananya; Kumar, Arvind; Farahat, Abdelbasset A.; Kumar, Dhiraj; Wang, Siming; Boykin, David W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">15404-15412</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">In spite of its importance in cell function, targeting DNA is under-represented in the design of small mols.  A barrier to progress in this area is the lack of a variety of modules that recognize G-C base pairs (bp) in DNA sequences.  To overcome this barrier, an entirely new design concept for modules that can bind to mixed G-C and A-T sequences of DNA is reported herein.  Because of their successes in biol. applications, minor-groove-binding heterocyclic cations were selected as the platform for design.  Binding to A-T sequences requires hydrogen-bond donors whereas recognition of the G-NH2 requires an acceptor.  The concept that we report herein uses pre-organized N-methylbenzimidazole (N-MeBI) thiophene modules for selective binding with mixed bp DNA sequences.  The interaction between the thiophene sigma hole (pos. electrostatic potential) and the electron-donor nitrogen of N-MeBI preorganizes the conformation for accepting an hydrogen bond from G-NH2.  The compd.-DNA interactions were evaluated with a powerful array of biophys. methods and the results show that N-MeBI-thiophene monomer compds. can strongly and selectively recognize single G-C bp sequences.  Replacing the thiophene with other moieties significantly reduces binding affinity and specificity, as predicted by the design concept.  These results show that the use of mol. features, such as sigma-holes, can lead to new approaches for small mols. in biomol. interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpoqg6sKBdEcLVg90H21EOLACvtfcHk0lhq9Lq1cc2qpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsV2rur%252FE&md5=4563d0e90f84c69fff74ae9922dba2a6</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1002%2Fchem.201603422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201603422%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DP.%26aulast%3DPaul%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DFarahat%26aufirst%3DA.%2BA.%26aulast%3DKumar%26aufirst%3DD.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DBoykin%26aufirst%3DD.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DThe%2520Thiophene%2520%25E2%2580%25B3Sigma-Hole%25E2%2580%25B3%2520as%2520a%2520Concept%2520for%2520Preorganized%252C%2520Specific%2520Recognition%2520of%2520G%25C2%25B7C%2520Base%2520Pairs%2520in%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2016%26volume%3D22%26spage%3D15404%26epage%3D15412%26doi%3D10.1002%2Fchem.201603422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Racané, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tralić-Kulenović, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavelić, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratkaj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peixoto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nhili, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Depauw, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hildebrand, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pavelić, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karminski-Zamola, G.</span></span> <span> </span><span class="NLM_article-title">Novel Diamidino-Substituted Derivatives of Phenyl Benzothiazolyl and Dibenzothiazolyl Furans and Thiophenes: Synthesis, Antiproliferative and DNA Binding Properties</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">2418</span>– <span class="NLM_lpage">2432</span>, <span class="refDoi"> DOI: 10.1021/jm901441b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901441b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFyhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2418-2432&author=L.+Racan%C3%A9author=V.+Trali%C4%87-Kulenovi%C4%87author=S.+K.+Paveli%C4%87author=I.+Ratkajauthor=P.+Peixotoauthor=R.+Nhiliauthor=S.+Depauwauthor=M.+P.+Hildebrandauthor=M.+H.+David-Cordonnierauthor=K.+Paveli%C4%87author=G.+Karminski-Zamola&title=Novel+Diamidino-Substituted+Derivatives+of+Phenyl+Benzothiazolyl+and+Dibenzothiazolyl+Furans+and+Thiophenes%3A+Synthesis%2C+Antiproliferative+and+DNA+Binding+Properties&doi=10.1021%2Fjm901441b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Diamidino-Substituted Derivatives of Phenyl Benzothiazolyl and Dibenzothiazolyl Furans and Thiophenes: Synthesis, Antiproliferative and DNA Binding Properties</span></div><div class="casAuthors">Racane, Livio; Tralic-Kulenovic, Vesna; Kraljevic Pavelic, Sandra; Ratkaj, Ivana; Peixoto, Paul; Nhili, Raja; Depauw, Sabine; Hildebrand, Marie-Paule; David-Cordonnier, Marie-Helene; Pavelic, Kresimir; Karminski-Zamola, Grace</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2418-2432</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of new diamidino-, diisopropylamidino-, and diimidazolinyl-substituted derivs. of Ph benzothiazolyl and dibenzothiazolyl furans and thiophenes were successfully prepd. and evaluated for their antiproliferative activity on tumor cell lines in vitro, DNA binding propensity, and sequence selectivity as well as cellular distribution.  A strong antiproliferative effect of the tested compds. was obsd. on all tested cell lines in a concn.-dependent response pattern.  In general, imidazolinyl-substituted derivs. and/or the thiophene core were in correlation with increased antiproliferative activity.  Two compds. I (Am = iso-Pr carboxyimidamide or 4,5-dihydro-1H-imidazol-2-yl) were chosen for biol. studies due to their differential antiproliferative properties.  The DNA binding properties of this new series of compds. were assessed and evidenced their efficient minor groove binding properties with preferential interaction at AT-rich sites.  Both compds. also present nuclear subcellular localization, suggesting that their cellular mode of action implies localization in the DNA compartment and direct inhibition of DNA replication and induction of apoptosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1RPFnF5AXKLVg90H21EOLACvtfcHk0lgVoR93t66RnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFyhtb8%253D&md5=cb3660325372379f0479e9d70865e78b</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1021%2Fjm901441b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901441b%26sid%3Dliteratum%253Aachs%26aulast%3DRacan%25C3%25A9%26aufirst%3DL.%26aulast%3DTrali%25C4%2587-Kulenovi%25C4%2587%26aufirst%3DV.%26aulast%3DPaveli%25C4%2587%26aufirst%3DS.%2BK.%26aulast%3DRatkaj%26aufirst%3DI.%26aulast%3DPeixoto%26aufirst%3DP.%26aulast%3DNhili%26aufirst%3DR.%26aulast%3DDepauw%26aufirst%3DS.%26aulast%3DHildebrand%26aufirst%3DM.%2BP.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.%2BH.%26aulast%3DPaveli%25C4%2587%26aufirst%3DK.%26aulast%3DKarminski-Zamola%26aufirst%3DG.%26atitle%3DNovel%2520Diamidino-Substituted%2520Derivatives%2520of%2520Phenyl%2520Benzothiazolyl%2520and%2520Dibenzothiazolyl%2520Furans%2520and%2520Thiophenes%253A%2520Synthesis%252C%2520Antiproliferative%2520and%2520DNA%2520Binding%2520Properties%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2418%26epage%3D2432%26doi%3D10.1021%2Fjm901441b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Messaoudi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anizon, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Léonce, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pierré, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeiffer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prudhomme, M.</span></span> <span> </span><span class="NLM_article-title">Synthesis and Cytotoxicities of 7-Aza Rebeccamycin Analogues Bearing Various Substituents on the Sugar Moiety, on the Imide Nitrogen and on the Carbazole Framework</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">961</span>– <span class="NLM_lpage">971</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2005.04.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ejmech.2005.04.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=15907347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVynsbnI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2005&pages=961-971&author=S.+Messaoudiauthor=F.+Anizonauthor=S.+L%C3%A9onceauthor=A.+Pierr%C3%A9author=B.+Pfeifferauthor=M.+Prudhomme&title=Synthesis+and+Cytotoxicities+of+7-Aza+Rebeccamycin+Analogues+Bearing+Various+Substituents+on+the+Sugar+Moiety%2C+on+the+Imide+Nitrogen+and+on+the+Carbazole+Framework&doi=10.1016%2Fj.ejmech.2005.04.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and cytotoxicities of 7-aza rebeccamycin analogs bearing various substituents on the sugar moiety, on the imide nitrogen and on the carbazole framework</span></div><div class="casAuthors">Messaoudi, Samir; Anizon, Fabrice; Leonce, Stephane; Pierre, Alain; Pfeiffer, Bruno; Prudhomme, Michelle</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">961-971</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The synthesis of a family of rebeccamycin analogs in which one indole unit has been replaced by a 7-azaindole moiety is described.  Substitutions have been carried out on the imide nitrogen, on the carbazole framework and on the sugar part.  Compds. with a lactam upper heterocycle have also been prepd.  The cytotoxicities of the newly synthesized compds. toward four tumor cell lines, one murine leukemia (L1210) and three human tumor cell lines (prostate carcinoma DU145, colon carcinoma HT29, and non-small cell lung carcinoma A549) have been evaluated and compared to those of rebeccamycin and parent non-aza and aza compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwEv9BskZ5WrVg90H21EOLACvtfcHk0lgVoR93t66RnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVynsbnI&md5=1a950fea7b127ffca6da496203b57fec</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2005.04.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2005.04.002%26sid%3Dliteratum%253Aachs%26aulast%3DMessaoudi%26aufirst%3DS.%26aulast%3DAnizon%26aufirst%3DF.%26aulast%3DL%25C3%25A9once%26aufirst%3DS.%26aulast%3DPierr%25C3%25A9%26aufirst%3DA.%26aulast%3DPfeiffer%26aufirst%3DB.%26aulast%3DPrudhomme%26aufirst%3DM.%26atitle%3DSynthesis%2520and%2520Cytotoxicities%2520of%25207-Aza%2520Rebeccamycin%2520Analogues%2520Bearing%2520Various%2520Substituents%2520on%2520the%2520Sugar%2520Moiety%252C%2520on%2520the%2520Imide%2520Nitrogen%2520and%2520on%2520the%2520Carbazole%2520Framework%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2005%26volume%3D40%26spage%3D961%26epage%3D971%26doi%3D10.1016%2Fj.ejmech.2005.04.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gros, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fleury, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahoum, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faux, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valente, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labella, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cantagrel, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rilova, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bouhlel, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">David-Cordonnier, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dufau, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ausseil, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mourey, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lacroix, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arimondo, P. B.</span></span> <span> </span><span class="NLM_article-title">New Insights on the Mechanism of Quinoline-Based DNA Methyltransferase Inhibitors</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">6293</span>– <span class="NLM_lpage">6302</span>, <span class="refDoi"> DOI: 10.1074/jbc.m114.594671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1074%2Fjbc.m114.594671" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=6293-6302&author=C.+Grosauthor=L.+Fleuryauthor=V.+Nahoumauthor=C.+Fauxauthor=S.+Valenteauthor=D.+Labellaauthor=F.+Cantagrelauthor=E.+Rilovaauthor=M.+A.+Bouhlelauthor=M.-H.+David-Cordonnierauthor=I.+Dufauauthor=F.+Ausseilauthor=A.+Maiauthor=L.+Moureyauthor=L.+Lacroixauthor=P.+B.+Arimondo&title=New+Insights+on+the+Mechanism+of+Quinoline-Based+DNA+Methyltransferase+Inhibitors&doi=10.1074%2Fjbc.m114.594671"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1074%2Fjbc.m114.594671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.m114.594671%26sid%3Dliteratum%253Aachs%26aulast%3DGros%26aufirst%3DC.%26aulast%3DFleury%26aufirst%3DL.%26aulast%3DNahoum%26aufirst%3DV.%26aulast%3DFaux%26aufirst%3DC.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DLabella%26aufirst%3DD.%26aulast%3DCantagrel%26aufirst%3DF.%26aulast%3DRilova%26aufirst%3DE.%26aulast%3DBouhlel%26aufirst%3DM.%2BA.%26aulast%3DDavid-Cordonnier%26aufirst%3DM.-H.%26aulast%3DDufau%26aufirst%3DI.%26aulast%3DAusseil%26aufirst%3DF.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DMourey%26aufirst%3DL.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DArimondo%26aufirst%3DP.%2BB.%26atitle%3DNew%2520Insights%2520on%2520the%2520Mechanism%2520of%2520Quinoline-Based%2520DNA%2520Methyltransferase%2520Inhibitors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D6293%26epage%3D6302%26doi%3D10.1074%2Fjbc.m114.594671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanious, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Biosensor-Surface Plasmon Resonance: Quantitative Analysis of Small Molecule-Nucleic Acid Interactions</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">150</span>– <span class="NLM_lpage">161</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2006.09.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.ymeth.2006.09.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=17472897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2007&pages=150-161&author=B.+Nguyenauthor=F.+A.+Taniousauthor=W.+D.+Wilson&title=Biosensor-Surface+Plasmon+Resonance%3A+Quantitative+Analysis+of+Small+Molecule-Nucleic+Acid+Interactions&doi=10.1016%2Fj.ymeth.2006.09.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Biosensor-surface plasmon resonance: quantitative analysis of small molecule-nucleic acid interactions</span></div><div class="casAuthors">Nguyen, Binh; Tanious, Farial A.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">150-161</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Surface plasmon resonance (SPR)-biosensor techniques directly provide essential information for the study and characterization of small mol.-nucleic acid interactions, and the use of these methods is steadily increasing.  The method is label-free and monitors the interactions in real time.  Both dynamic and steady-state information can be obtained for a wide range of reaction rates and binding affinities.  This article presents the basics of the SPR technique, provides suggestions for exptl. design, and illustrates data processing and anal. of results.  A specific example of the interaction of a well-known minor groove binding agent, netropsin, with DNA is evaluated by both kinetic and steady-state SPR methods.  Three different expts. are used to illustrate different approaches and anal. methods.  The three sets of results show the reproducibility of the binding consts. and agreement from both steady-state and kinetic analyses.  These expts. also show that reliable kinetic information can be obtained, even with difficult systems, if the exptl. conditions are optimized to minimize mass transport effects.  Limitations of the biosensor-SPR technique are also discussed to provide an awareness of the care needed to conduct a successful expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoIMfX4ZwyTSLVg90H21EOLACvtfcHk0ljMzSSgm7srWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslCntrs%253D&md5=9455fa47e2eefef19166b544873c9102</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2006.09.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2006.09.009%26sid%3Dliteratum%253Aachs%26aulast%3DNguyen%26aufirst%3DB.%26aulast%3DTanious%26aufirst%3DF.%2BA.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DBiosensor-Surface%2520Plasmon%2520Resonance%253A%2520Quantitative%2520Analysis%2520of%2520Small%2520Molecule-Nucleic%2520Acid%2520Interactions%26jtitle%3DMethods%26date%3D2007%26volume%3D42%26spage%3D150%26epage%3D161%26doi%3D10.1016%2Fj.ymeth.2006.09.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nanjunda, R.</span>; <span class="NLM_string-name">Munde, M.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Real-Time Monitoring of Nucleic Acid Interactions with Biosensor-Surface Plasmon Resonance</span>. In  <i>Methods for Studying Nucleic Acid/Drug Interactions</i>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_contrib-group"><span class="NLM_string-name">Yitzhak, T.</span>, <span class="NLM_string-name">Meni, W.</span></span>; <span class="NLM_publisher-name">CRC Press</span>: <span class="NLM_publisher-loc">Boca Raton, Florida, USA</span>, <span class="NLM_year">2011</span>; Chapter 4, pp  <span class="NLM_fpage">91</span>– <span class="NLM_lpage">122</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1201%2Fb11691-7" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2011&pages=91-122&author=R.+Nanjunda&author=M.+Munde&author=Y.+Liu&author=W.+D.+Wilsonauthor=T.+Yitzhak&author=W.+Meni&title=Methods+for+Studying+Nucleic+Acid%2FDrug+Interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1201%2Fb11691-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1201%252Fb11691-7%26sid%3Dliteratum%253Aachs%26aulast%3DNanjunda%26aufirst%3DR.%26atitle%3DReal-Time%2520Monitoring%2520of%2520Nucleic%2520Acid%2520Interactions%2520with%2520Biosensor-Surface%2520Plasmon%2520Resonance%26btitle%3DMethods%2520for%2520Studying%2520Nucleic%2520Acid%252FDrug%2520Interactions%26aulast%3DYitzhak%26aufirst%3DT.%26pub%3DCRC%2520Press%26date%3D2011%26spage%3D91%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bayly, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cieplak, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornell, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span> <span> </span><span class="NLM_article-title">A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: the RESP model</span>. <i>J. Phys. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>97</i></span>,  <span class="NLM_fpage">10269</span>– <span class="NLM_lpage">10280</span>, <span class="refDoi"> DOI: 10.1021/j100142a004</span> </span><div class="citationLinks">[<a href="/doi/10.1021/j100142a004" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADyaK3sXlvVyqsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=1993&pages=10269-10280&author=C.+I.+Baylyauthor=P.+Cieplakauthor=W.+Cornellauthor=P.+A.+Kollman&title=A+well-behaved+electrostatic+potential+based+method+using+charge+restraints+for+deriving+atomic+charges%3A+the+RESP+model&doi=10.1021%2Fj100142a004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">A well-behaved electrostatic potential based method using charge restraints for deriving atomic charges: the RESP model</span></div><div class="casAuthors">Bayly, Christopher I.; Cieplak, Piotr; Cornell, Wendy; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">97</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">10269-80</span>CODEN:
                <span class="NLM_cas:coden">JPCHAX</span>;
        ISSN:<span class="NLM_cas:issn">0022-3654</span>.
    </div><div class="casAbstract">The authors present a new approach to generating electrostatic potential (ESP) derived charges for mols.  The major strength of electrostatic potential derived charges is that they optimally reproduce the intermol. interaction properties of mols. with a simple two-body additive potential, provided, of course, that a suitably accurate level of quantum mech. calcn. is used to derive the ESP around the mol.  Previously, the major weaknesses of these charges have been that they were not easily transferably between common functional groups in related mols., they have often been conformationally dependent, and the large charges that frequently occur can be problematic for simulating intramol. interactions.  Introducing restraints in the form of a penalty function into the fitting process considerably reduces the above problems, with only a minor decrease in the quality of the fit to the quantum mech. ESP.  Several other refinements in addn. to the restrained electrostatic potential (RESP) fit yield a general and algorithmic charge fitting procedure for generating atom-centered point charges.  This approach can thus be recommended for general use in mol. mechanics, mol. dynamics, and free energy calcns. for any org. or bioorg. system.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRAz3o6S1gOrVg90H21EOLACvtfcHk0liA1ru6zPtlpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXlvVyqsLs%253D&md5=e65c6a556ffc174df4f327687912a0bd</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fj100142a004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fj100142a004%26sid%3Dliteratum%253Aachs%26aulast%3DBayly%26aufirst%3DC.%2BI.%26aulast%3DCieplak%26aufirst%3DP.%26aulast%3DCornell%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DA%2520well-behaved%2520electrostatic%2520potential%2520based%2520method%2520using%2520charge%2520restraints%2520for%2520deriving%2520atomic%2520charges%253A%2520the%2520RESP%2520model%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1993%26volume%3D97%26spage%3D10269%26epage%3D10280%26doi%3D10.1021%2Fj100142a004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span>; <span class="NLM_string-name">Babin, V.</span>; <span class="NLM_string-name">Berryman, J.</span>; <span class="NLM_string-name">Betz, R. M.</span>; <span class="NLM_string-name">Cai, Q.</span>; <span class="NLM_string-name">Cerutti, D. S.</span>; <span class="NLM_string-name">Cheatham, T. E.,  III</span>; <span class="NLM_string-name">Darden, T. A.</span>; <span class="NLM_string-name">Duke, R. E.</span>; <span class="NLM_string-name">Gohlke, H.</span>; <span class="NLM_string-name">Goetz, A. W.</span>; <span class="NLM_string-name">Gusarov, S.</span>; <span class="NLM_string-name">Homeyer, N.</span>; <span class="NLM_string-name">Janowski, P.</span>; <span class="NLM_string-name">Kaus, J.</span>; <span class="NLM_string-name">Kolossváry, I.</span>; <span class="NLM_string-name">Kovalenko, A.</span>; <span class="NLM_string-name">Lee, T. S.</span>; <span class="NLM_string-name">LeGrand, S.</span>; <span class="NLM_string-name">Luchko, T.</span>; <span class="NLM_string-name">Luo, R.</span>; <span class="NLM_string-name">Madej, B.</span>; <span class="NLM_string-name">Merz, K. M.</span>; <span class="NLM_string-name">Paesani, F.</span>; <span class="NLM_string-name">Roe, D. R.</span>; <span class="NLM_string-name">Roitberg, A.</span>; <span class="NLM_string-name">Sagui, C.</span>; <span class="NLM_string-name">Salomon-Ferrer, R.</span>; <span class="NLM_string-name">Seabra, G.</span>; <span class="NLM_string-name">Simmerling, C. L.</span>; <span class="NLM_string-name">Smith, W.</span>; <span class="NLM_string-name">Swails, J.</span>; <span class="NLM_string-name">Walker, R. C.</span>; <span class="NLM_string-name">Wang, J.</span>; <span class="NLM_string-name">Wolf, R. M.</span>; <span class="NLM_string-name">Wu, X.</span>; <span class="NLM_string-name">Kollman, P. A.</span></span> <i>AMBER 14</i>; <span class="NLM_publisher-name">University of California</span>: <span class="NLM_publisher-loc">San Francisco</span>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=D.+A.+Case&author=V.+Babin&author=J.+Berryman&author=R.+M.+Betz&author=Q.+Cai&author=D.+S.+Cerutti&author=T.+E.+Cheatham&author=T.+A.+Darden&author=R.+E.+Duke&author=H.+Gohlke&author=A.+W.+Goetz&author=S.+Gusarov&author=N.+Homeyer&author=P.+Janowski&author=J.+Kaus&author=I.+Kolossv%C3%A1ry&author=A.+Kovalenko&author=T.+S.+Lee&author=S.+LeGrand&author=T.+Luchko&author=R.+Luo&author=B.+Madej&author=K.+M.+Merz&author=F.+Paesani&author=D.+R.+Roe&author=A.+Roitberg&author=C.+Sagui&author=R.+Salomon-Ferrer&author=G.+Seabra&author=C.+L.+Simmerling&author=W.+Smith&author=J.+Swails&author=R.+C.+Walker&author=J.+Wang&author=R.+M.+Wolf&author=X.+Wu&author=P.+A.+Kollman&title=AMBER+14"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DCase%26aufirst%3DD.%2BA.%26jtitle%3DAMBER%252014%26pub%3DUniversity%2520of%2520California%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kollman, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Case, D. A.</span></span> <span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span>. <i>J. Mol. Graph. Model</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">247</span>– <span class="NLM_lpage">260</span>, <span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0liA1ru6zPtlpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graph.%2520Model%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harika, N. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Germann, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">First Structure of a Designed Minor Groove Binding Heterocyclic Cation that Specifically Recognizes Mixed DNA Base Pair Sequences</span>. <i>Chem.—Eur. J.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">17612</span>– <span class="NLM_lpage">17620</span>, <span class="refDoi"> DOI: 10.1002/chem.201704563</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fchem.201704563" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=29044822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVartrzI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=17612-17620&author=N.+K.+Harikaauthor=M.+W.+Germannauthor=W.+D.+Wilson&title=First+Structure+of+a+Designed+Minor+Groove+Binding+Heterocyclic+Cation+that+Specifically+Recognizes+Mixed+DNA+Base+Pair+Sequences&doi=10.1002%2Fchem.201704563"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">First structure of a designed minor groove binding heterocyclic cation that specifically recognizes mixed DNA base pair sequences</span></div><div class="casAuthors">Harika, Narinder K.; Germann, Markus W.; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - A European Journal</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">69</span>),
    <span class="NLM_cas:pages">17612-17620</span>CODEN:
                <span class="NLM_cas:coden">CEUJED</span>;
        ISSN:<span class="NLM_cas:issn">0947-6539</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The high-resoln. NMR structure of the 1st heterocyclic, non-amide, org. cation that strongly and selectively recognizes mixed AT/GC bp sequences of DNA in a 1:1 complex is described.  Compd. designs of this type provide essential methods for control of functional, non-genomic DNA sequences and have broad cell uptake capability, based on studies from animals to humans.  The high-resoln. structural studies described in this report are essential for understanding the mol. basis for the sequence-specific binding as well as for new ideas for addnl. compd. designs for sequence-specific recognition.  The mol. features, in this report, explain the mechanism of recognition of both A·T and G·C bps and are an interesting mol. recognition story.  Examn. of the exptl. structure and the NMR restrained mol. dynamics model suggested that recognition of the G·C base pair involves 2 specific H-bonds.  The structure illustrated a wealth of information on different DNA interactions and illustrated an interfacial water mol. that is a key component of the complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppMTEBch-mXLVg90H21EOLACvtfcHk0lhoRNi7zzlCPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVartrzI&md5=bf4a3094e2932ba703e4f0ce0de590fc</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1002%2Fchem.201704563&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fchem.201704563%26sid%3Dliteratum%253Aachs%26aulast%3DHarika%26aufirst%3DN.%2BK.%26aulast%3DGermann%26aufirst%3DM.%2BW.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DFirst%2520Structure%2520of%2520a%2520Designed%2520Minor%2520Groove%2520Binding%2520Heterocyclic%2520Cation%2520that%2520Specifically%2520Recognizes%2520Mixed%2520DNA%2520Base%2520Pair%2520Sequences%26jtitle%3DChem.%25E2%2580%2594Eur.%2520J.%26date%3D2017%26volume%3D23%26spage%3D17612%26epage%3D17620%26doi%3D10.1002%2Fchem.201704563" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Athri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, W. D.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics of Water-Mediated Interactions of a Linear Benzimidazole–Biphenyl Diamidine with the DNA Minor Groove</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>131</i></span>,  <span class="NLM_fpage">7618</span>– <span class="NLM_lpage">7625</span>, <span class="refDoi"> DOI: 10.1021/ja809249h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja809249h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD1MXlvFWntLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=7618-7625&author=P.+Athriauthor=W.+D.+Wilson&title=Molecular+Dynamics+of+Water-Mediated+Interactions+of+a+Linear+Benzimidazole%E2%80%93Biphenyl+Diamidine+with+the+DNA+Minor+Groove&doi=10.1021%2Fja809249h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics of Water-Mediated Interactions of a Linear Benzimidazole-Biphenyl Diamidine with the DNA Minor Groove</span></div><div class="casAuthors">Athri, Prashanth; Wilson, W. David</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7618-7625</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">DB921 has a benzimidazole-biphenyl system with terminal amidines that gives the compd. a linear conformation with a radius of curvature that does not match the DNA minor groove shape.  Surprisingly, the compd. binds in the groove with an unusually high equil. const.  X-ray crystallog. anal. of DB921 bound to -AATT- in d(CGCGAATTCGCG)2 showed that the benzimidazole is in position to directly interact with bases at the floor of the groove, while the phenylamidine of DB921 forms indirect contacts with the bases through an interfacial water.  The DB921-water pair forms a curved, flexible module with a high Ka (or a low Kd) value of binding.  To better understand the dynamics of the DB921-DNA complex and how water can be used in the design of compds. to recognize DNA, a 100 ns mol. dynamics simulation of the complex was conducted.  In addn. to the x-ray conformation, some significantly variant, dynamic conformations, which had addnl. interfacial water mols. between DB921 and DNA, appeared in the MD simulation.  The benzimidazole contacts remained relatively const. through the entire simulation.  The biphenylamidine end of the bound mol., however, undergoes much larger changes in orientation relative to the floor of the groove as well as variations in the type of water interactions.  The results provide an understanding of how water couples the linear DB921 compd. to the minor groove for tight binding, without a large unfavorable contribution to the entropy of binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHDydOY5wGOrVg90H21EOLACvtfcHk0lhoRNi7zzlCPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXlvFWntLo%253D&md5=55099792edbdaf5e3626b311ed97615a</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1021%2Fja809249h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja809249h%26sid%3Dliteratum%253Aachs%26aulast%3DAthri%26aufirst%3DP.%26aulast%3DWilson%26aufirst%3DW.%2BD.%26atitle%3DMolecular%2520Dynamics%2520of%2520Water-Mediated%2520Interactions%2520of%2520a%2520Linear%2520Benzimidazole%25E2%2580%2593Biphenyl%2520Diamidine%2520with%2520the%2520DNA%2520Minor%2520Groove%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D7618%26epage%3D7625%26doi%3D10.1021%2Fja809249h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization and multithreading</span>. <i>Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhp_3rjKK6y_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%2520and%2520multithreading%26jtitle%3DComput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ritchie, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phipson, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Law, C. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smyth, G. K.</span></span> <span> </span><span class="NLM_article-title">limma powers differential expression analyses for RNA-sequencing and microarray studies</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>43</i></span>, <span class="NLM_elocation-id">e47</span> <span class="refDoi"> DOI: 10.1093/nar/gkv007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=10.1093%2Fnar%2Fgkv007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=25605792" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&author=M.+E.+Ritchieauthor=B.+Phipsonauthor=D.+Wuauthor=Y.+Huauthor=C.+W.+Lawauthor=W.+Shiauthor=G.+K.+Smyth&title=limma+powers+differential+expression+analyses+for+RNA-sequencing+and+microarray+studies&doi=10.1093%2Fnar%2Fgkv007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">limma powers differential expression analyses for RNA-sequencing and microarray studies</span></div><div class="casAuthors">Ritchie, Matthew E.; Phipson, Belinda; Wu, Di; Hu, Yifang; Law, Charity W.; Shi, Wei; Smyth, Gordon K.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e47/1-e47/13</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Limma is an R/Bioconductor software package that provides an integrated soln. for analyzing data from gene expression expts.  It contains rich features for handling complex exptl. designs and for information borrowing to overcome the problem of small sample sizes.  Over the past decade, limma has been a popular choice for gene discovery through differential expression analyses of microarray and high-throughput PCR data.  The package contains particularly strong facilities for reading, normalizing and exploring such data.  Recently, the capabilities of limma have been significantly expanded in two important directions.  First, the package can now perform both differential expression and differential splicing analyses of RNA sequencing (RNA-seq) data.  All the downstream anal. tools previously restricted to microarray data are now available for RNA-seq as well.  These capabilities allow users to analyze both RNA-seq and microarray data with very similar pipelines.  Second, the package is now able to go past the traditional gene-wise expression analyses in a variety of ways, analyzing expression profiles in terms of co-regulated sets of genes or in terms of higher-order expression signatures.  This provides enhanced possibilities for biol. interpretation of gene expression differences.  This article reviews the philosophy and design of the limma package, summarizing both new and historical features, with an emphasis on recent enhancements and features that have not been previously described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJykfsLP7n9bVg90H21EOLACvtfcHk0lhp_3rjKK6y_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsFaiu7%252FN&md5=e8a5f44160c650ad89adc48a9912bd96</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv007%26sid%3Dliteratum%253Aachs%26aulast%3DRitchie%26aufirst%3DM.%2BE.%26aulast%3DPhipson%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DHu%26aufirst%3DY.%26aulast%3DLaw%26aufirst%3DC.%2BW.%26aulast%3DShi%26aufirst%3DW.%26aulast%3DSmyth%26aufirst%3DG.%2BK.%26atitle%3Dlimma%2520powers%2520differential%2520expression%2520analyses%2520for%2520RNA-sequencing%2520and%2520microarray%2520studies%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26doi%3D10.1093%2Fnar%2Fgkv007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VZK" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1VZK','PDB','1VZK'); return false;">PDB: 1VZK</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I5A','PDB','2I5A'); return false;">PDB: 2I5A</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I2I" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2I2I','PDB','2I2I'); return false;">PDB: 2I2I</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GYX" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2GYX','PDB','2GYX'); return false;">PDB: 2GYX</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i84"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01448">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_27607"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.8b01448">10.1021/acs.jmedchem.8b01448</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Supplementary Tables S1 and S2. Supplementary Figures S1–S7. Supplementary chemistry section. Supplementary references. Representatives <sup>1</sup>H NMR spectra (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_001.pdf">jm8b01448_si_001.pdf (2.94 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.8b01448/suppl_file/jm8b01448_si_002.csv">jm8b01448_si_002.csv (5.03 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01448&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01448%3Bwgroup%3Astring%3AACHS+website+Group%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-3%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01448" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                11MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e11d95fa39247f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
