
<html lang="en"     class="pb-page"  data-request-id="fcfedede-3c16-4361-92ce-25d7f03b1991"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;article:article:10.1021/jm5018187;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2015.58.issue-3"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of Small Molecule Inhibitors to Krüppel-like Factor 10 (KLF10): Implications for Modulation of T Regulatory Cell Differentiation" /></meta><meta name="dc.Creator" content="Santosh A.  Khedkar" /></meta><meta name="dc.Creator" content="Xinghui  Sun" /></meta><meta name="dc.Creator" content="Alan C.  Rigby" /></meta><meta name="dc.Creator" content="Mark W.  Feinberg" /></meta><meta name="dc.Description" content="The Krüppel-like family of transcription factors (KLFs) constitute a subfamily of C2H2-type zinc finger proteins with distinct cell-type expression patterns and regulate functional aspects of cell ..." /></meta><meta name="Description" content="The Krüppel-like family of transcription factors (KLFs) constitute a subfamily of C2H2-type zinc finger proteins with distinct cell-type expression patterns and regulate functional aspects of cell ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 4, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm5018187" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm5018187" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm5018187" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm5018187" /></link>
        
    
    

<title>Discovery of Small Molecule Inhibitors to Krüppel-like Factor 10 (KLF10): Implications for Modulation of T Regulatory Cell Differentiation | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm5018187" /></meta><meta property="og:title" content="Discovery of Small Molecule Inhibitors to Krüppel-like Factor 10 (KLF10): Implications for Modulation of T Regulatory Cell Differentiation" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0010.jpeg" /></meta><meta property="og:description" content="The Krüppel-like family of transcription factors (KLFs) constitute a subfamily of C2H2-type zinc finger proteins with distinct cell-type expression patterns and regulate functional aspects of cell growth and differentiation, activation, or development. KLF10 has been previously shown to critically regulate the acquisition of CD4+CD25+ T regulatory cell differentiation and function, an effect important to the maintenance of self-tolerance, immune suppression, and tumor immunosurveillance. To date, there are no selective pharmacological inhibitors to KLF10. Herein, we report on the discovery of first-in-class small molecule compounds that inhibit the KLF10-DNA interaction interface using computer-aided drug design (CADD) screens of chemical libraries. Interrogation of a “druggable” pocket in the second zinc finger of KLF10 revealed three small molecules, #48, #48-15, and #15-09, with similar scaffolds and binding patterns. Each of these small molecules inhibited KLF10-DNA binding and transcriptional activity, conversion of CD4+CD25– T cells to CD4+CD25+ T regulatory cells, and KLF10 target gene expression. Taken together, these findings support the feasibility of using CADD with functional assays to identify small molecules that target members of the KLF subfamily of transcription factors to regulate biological functions in health and disease. We hope these novel compounds will serve as useful mechanistic probes for KLF10-mediated effects and T regulatory cell biology." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm5018187"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm5018187">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm5018187&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm5018187&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm5018187&amp;href=/doi/10.1021/jm5018187" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 3</span><span class="cit-fg-pageRange">, 1466-1478</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/3" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm501772w" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm501787c" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Small Molecule Inhibitors to Krüppel-like Factor 10 (KLF10): Implications for Modulation of T Regulatory Cell Differentiation</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Santosh+A.++Khedkar">Santosh A. Khedkar</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xinghui++Sun">Xinghui Sun</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Alan+C.++Rigby">Alan C. Rigby</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mark+W.++Feinberg">Mark W. Feinberg</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Medicine, Division of Molecular and Vascular Medicine, Center for Vascular Biology Research, Beth Israel Deaconess Medical Center, 99 Brookline Avenue, RN-227, Boston, Massachusetts 02215, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Cardiovascular Division, Brigham and Women’s Hospital, Harvard Medical School, 77 Avenue Louis Pasteur, NRB-742F, Boston, Massachusetts 02115, United States</span></div><div class="corresp-info"><strong>*</strong>A.C.R.: e-mail, <a href="/cdn-cgi/l/email-protection#3c7d505d52126e555b5e457c55515f50535259125f5351"><span class="__cf_email__" data-cfemail="6f2e030e01413d06080d162f06020c0300010a410c0002">[email protected]</span></a>; phone, +1-646-742-8401.</div><div class="corresp-info"><strong>*</strong>M.W.F.: e-mail, <a href="/cdn-cgi/l/email-protection#b8d5deddd1d6daddcadff8c8d9caccd6ddcacb96d7cadf"><span class="__cf_email__" data-cfemail="49242f2c20272b2c3b2e0939283b3d272c3b3a67263b2e">[email protected]</span></a>; phone, +1-617-525-4381; fax, +1-617-525-4380.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm5018187&amp;href=/doi/10.1021%2Fjm5018187" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 3</span><span class="cit-pageRange">, 1466–1478</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 12, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>16 November 2014</li><li><span class="item_label"><b>Published</b> online</span>4 February 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 12 February 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm5018187" title="DOI URL">https://doi.org/10.1021/jm5018187</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D1466%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSantosh%2BA.%2BKhedkar%252C%2BXinghui%2BSun%252C%2BAlan%2BC.%2BRigby%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D3%26contentID%3Djm5018187%26title%3DDiscovery%2Bof%2BSmall%2BMolecule%2BInhibitors%2Bto%2BKr%25C3%25BCppel-like%2BFactor%2B10%2B%2528KLF10%2529%253A%2BImplications%2Bfor%2BModulation%2Bof%2BT%2BRegulatory%2BCell%2BDifferentiation%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D1478%26publicationDate%3DFebruary%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm5018187"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1202</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">10</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm5018187" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Small Molecule Inhibitors to Krüppel-like Factor 10 (KLF10): Implications for Modulation of T Regulatory Cell Differentiation&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Santosh&quot;,&quot;last_name&quot;:&quot;A. Khedkar&quot;},{&quot;first_name&quot;:&quot;Xinghui&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Alan&quot;,&quot;last_name&quot;:&quot;C. Rigby&quot;},{&quot;first_name&quot;:&quot;Mark&quot;,&quot;last_name&quot;:&quot;W. Feinberg&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;3&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;1466-1478&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm5018187&quot;},&quot;abstract&quot;:&quot;The Krüppel-like family of transcription factors (KLFs) constitute a subfamily of C2H2-type zinc finger proteins with distinct cell-type expression patterns and regulate functional aspects of cell growth and differentiation, activation, or development. KLF10 has been previously shown to critically regulate the acquisition of CD4+CD25+ T regulatory cell differentiation and function, an effect important to the maintenance of self-tolerance, immune suppression, and tumor immunosurveillance. To date, there are no selective pharmacological inhibitors to KLF10. Herein, we report on the discovery of first-in-class small molecule compounds that inhibit the KLF10-DNA interaction interface using computer-aided drug design (CADD) screens of chemical libraries. Interrogation of a “druggable” pocket in the second zinc finger of KLF10 revealed three small molecules, #48, #48-15, and #15-09, with similar scaffolds and binding patterns. Each of these small molecules inhibited KLF10-DNA binding and transcriptional activity, c&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5018187&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5018187" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5018187&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5018187" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm5018187&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5018187" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm5018187&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm5018187&amp;href=/doi/10.1021/jm5018187" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm5018187" /></input><a href="/doi/pdf/10.1021/jm5018187" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (2 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm5018187&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm5018187%26sid%3Dliteratum%253Aachs%26pmid%3D25581017%26genre%3Darticle%26aulast%3DKhedkar%26date%3D2015%26atitle%3DDiscovery%2Bof%2BSmall%2BMolecule%2BInhibitors%2Bto%2BKr%25C3%25BCppel-like%2BFactor%2B10%2B%2528KLF10%2529%253A%2BImplications%2Bfor%2BModulation%2Bof%2BT%2BRegulatory%2BCell%2BDifferentiation%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D3%26spage%3D1466%26epage%3D1478%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291399" title="Zinc">Zinc</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=292276" title="Small molecules">Small molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/3" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jmcmar.2015.58.issue-3/20150212/jmcmar.2015.58.issue-3.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">The Krüppel-like family of transcription factors (KLFs) constitute a subfamily of C<sub>2</sub>H<sub>2</sub>-type zinc finger proteins with distinct cell-type expression patterns and regulate functional aspects of cell growth and differentiation, activation, or development. KLF10 has been previously shown to critically regulate the acquisition of CD4+CD25+ T regulatory cell differentiation and function, an effect important to the maintenance of self-tolerance, immune suppression, and tumor immunosurveillance. To date, there are no selective pharmacological inhibitors to KLF10. Herein, we report on the discovery of first-in-class small molecule compounds that inhibit the KLF10-DNA interaction interface using computer-aided drug design (CADD) screens of chemical libraries. Interrogation of a “druggable” pocket in the second zinc finger of KLF10 revealed three small molecules, #48, #48-15, and #15-09, with similar scaffolds and binding patterns. Each of these small molecules inhibited KLF10-DNA binding and transcriptional activity, conversion of CD4+CD25– T cells to CD4+CD25+ T regulatory cells, and KLF10 target gene expression. Taken together, these findings support the feasibility of using CADD with functional assays to identify small molecules that target members of the KLF subfamily of transcription factors to regulate biological functions in health and disease. We hope these novel compounds will serve as useful mechanistic probes for KLF10-mediated effects and T regulatory cell biology.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div id="ac_i1" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i1"> </h2><div class="NLM_p">Zinc fingers (ZFs) are the most ubiquitous family of transcription factors (TFs) with more than 1000 members, comprising ∼1% of the human genome.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> The Krüppel-like family of transcription factors (KLFs), so named for their homology to the <i>Drosophila melanogaster</i> Krüppel protein, is a C<sub>2</sub>H<sub>2</sub>-type zinc finger (containing two cysteine and two histidine residues) with a simple structure consisting of 25–30 amino acid residues that includes two β-pleated sheets in the amino terminal half and an α-helix in the carboxyl terminal half, held together at the base by a zinc (Zn<sup>2+</sup>) ion.<a onclick="showRef(event, 'ref2 ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref2 ref3 ref4">(2-4)</a> The ZFTFs are responsible for interacting with either a CACCC-element or GC-rich sites in the promoter region of target genes, thereby regulating transcriptional activity and gene expression.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> These DNA binding proteins are requisite components of the transcriptional machinery involved in mediating cellular phenotypic responses to extracellular signals. The KLF family currently includes 18 members, namely, KLFs 1–18 with distinct cell-type expression patterns.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Furthermore, KLFs participate in various functional aspects of cell growth and differentiation, activation, or development. As such, therapeutic targeting of select KLF family members may be desirable for achieving distinct biological effects.</div><div class="NLM_p">CD4+CD25+ T regulatory cells (T regs) actively participate in the maintenance of self-tolerance and immune suppression and constitute a significant obstacle for effective tumor immunosurveillance, vaccine-induced antitumor immune responses, and clearance of bacterial pathogens.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11">(7-11)</a> We and others have previously demonstrated that KLF10, a transforming growth factor β1 (TGF-β1) responsive transcription factor, is a critical regulator of T reg cell differentiation and function in vitro and in vivo.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> KLF10–/– mice exhibit reduced peripheral T reg cells with reduced expression of Foxp3, a hallmark of T reg cell differentiation. Indeed, KLF10 mediates this effect, in part, by binding to the proximal CACCC sites of the Foxp3 and TGF-β1 promoters. These observations raise the possibility that KLF10 may be an excellent therapeutic target for a range of disease states in which reduced T reg cell numbers or function would be advantageous such as enhanced antitumor immunity.<a onclick="showRef(event, 'ref7 ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref7 ref8 ref9 ref10 ref11">(7-11)</a></div><div class="NLM_p last">Currently, there is no clinically available treatment that specifically targets T reg cells using a small druglike molecule. Herein, we report computational discovery of small molecule inhibitors of the KLF10-DNA interaction interface, an effect that reduced KLF10-DNA binding and transcriptional activity. Furthermore, we also report an investigation into the effects of our inhibitor compounds on the conversion of CD4+CD25– T cells to CD4+CD25+ T regs and FoxP3 expression. Overall, our findings support the feasibility of using computer-aided drug design (CADD) with functional assays to identify small molecules that target members of the KLF subfamily of transcription factors to regulate biological functions.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_70872" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_70872" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Homology Model of KLF10 Reveals a Large Binding Pocket within the DNA-Binding Region</h3><div class="NLM_p">A homology model of KLF10 protein was generated using template coordinates of a DNA-binding protein having three consensus-sequence-based zinc finger domains. The overall topology of the KLF10 structure matches with the expected fold of classical C<sub>2</sub>H<sub>2</sub> zinc finger domains in which two short strands of antiparallel β strands linked by a rubredoxin-like turn are followed by an α-helix that contains two Zn<sup>2+</sup>-coordinating histidine residues. In our current efforts to identify modulators of TF-dependent transcription, we hypothesized that small organic molecules capable of binding to the KLF10 zinc finger region that interfaces the DNA would be able to inhibit the binding of KLF10 with DNA and, subsequently, reduce transcriptional activity. An analysis of the zinc finger-DNA binding region using multiple binding pocket identification programs such as SiteMap<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> (Schrodinger), MolCad<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (Tripos), and SiteFinder<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> (Chemical Computing Group) uniquely identified a relatively large pocket in the middle zinc finger which has maximum surface contacts with the DNA (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A,B). We subclassified this binding pocket into α- and β-pockets depending on the proximity with α-helix or β-sheet residues encompassing it.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Homology model of KLF10 protein revealing the proposed pocket of small molecule binding and its relative location in the DNA-bound complex (A, B). Molecular surface of DNA-binding domain of KLF10 protein (blue) along with DNA strands (plum) shown to represent the relative orientation/location of proposed binding pockets along with compound #48-15 (A; green, mesh representation). Proposed binding pocket within second zinc finger of KLF10 (electrostatic surface; blue color for positive potential and red color for negative potential), with α and β subpockets separated by Glu415 side chain (B). Inhibition of KLF10 transcriptional activity by small molecule compounds #48, #48-15, and #15-09 (C–E): HeLa cells were transiently transfected with human KLF10 and a CACCC-luciferase reporter. After transfection, cells were allowed to grow for 24 h and subsequently treated with compounds #48 (C), #48-15 (D), and #15-09 (E) for another 24 h, at which time they were harvested for luciferase activity assays. In (F), HeLa cells were transfected in a similar manner as above, except they were treated with small molecule compounds #3, #23, #37, #40, #65, #79, #83, #96, #104, #106, #114, #118, #121, #126, and #146. Data show the mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.01; Con, empty-vector control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> In Silico Screens Identified Small Molecule Inhibitors of KLF10</h3><div class="NLM_p last">High-throughput docking of chemical libraries using Glide docking<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> with extra precision (XP)<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> scoring into grids encompassing the α- and β-pockets resulted in ∼1400 unique hits. This subset was further evaluated with another two independently developed dockers: Surflex<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and FlexX.<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> The highest scored 700 molecules were then subjected to consensus scoring and ADME/Tox profiling analysis to select 40 molecules, which were subsequently tested in KLF10 reporter assays as detailed below.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Identified Compounds Inhibit KLF10 Transcriptional Activity</h3><div class="NLM_p">We developed a cell-based reporter assay to test the transcriptional activity of in silico identified compounds. A KLF10 expression plasmid and luciferase reporter driven by a CACCC-responsive promoter were transfected into HeLa cells. Exogenous KLF10-induced luciferase activity was approximately 2.5-fold compared with transfection of empty vector (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C,E). In vitro screening using this luciferase reporter assay resulted in the discovery of 11 inhibitors of KLF10 reporter activity (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Several structural analogs of two scaffolds, #48 and #15 (Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>), were computationally investigated to explore the chemical space governing the activity; many of these analogs showed equal or better potency than the parent compound in luciferase reporter assays. For example, the induction of luciferase activity by exogenous KLF10 expression was almost completely blocked by 100 μM compound #48 (96.6%, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>C). In the presence of 100 μM compound #48-15, reporter activity was reduced to even lower than basal conditions (−20% vs control, Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>D). Similarly, compound #15-09 also potently inhibited KLF10 induced luciferase activity to −31% of basal conditions (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>E). In contrast, small molecule compounds #3, #23, #37, #40, #65, #79, #83, #96, #104, #106, #114, #118, #121, #126, and #146 had no effect on KLF10-induced luciferase activity (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>F). We also examined dose-dependent responses of compounds #48, #48-15, and #15-09 on KLF10 transcriptional activity. IC<sub>50</sub> values, calculated based on nonlinear regression analysis, for compounds #48, #48-15, and #15-09 were 112, 40, and 43 μM, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). Collectively, these findings highlight the identification of three small molecule compounds, #48, #48-15, and #15-09, which could effectively inhibit KLF10 transcriptional activity.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Dose–response curves of KLF10 transcriptional activity of HeLa cells treated with small molecule compounds #48, #48-15, and #15-09. HeLa cells were transiently transfected with human KLF10 and a CACCC-luciferase reporter. After transfection, cells were allowed to grow for 24 h and subsequently treated with different amounts of #48 (A), #48-15 (B), and #15-09 (C) for another 24 h, at which time they were harvested for luciferase activity assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. First-Generation Inhibitors of KLF10 Reporter Activity<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0013.gif" alt="" id="GRAPHIC-d3383e353-autogenerated" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">Footnotes: <sup>§</sup>Percent inhibition at 100 μM (sd, standard deviation). <sup>¶</sup>Compound concentration in μM that inhibited 50% of KLF10 reporter activity; n.d., not determined. <sup>€</sup>Tolcapone (Tasmar, catechol-<i>O</i>-methyl transferase inhibitor) and atomoxetine (Strattera, norepinephrine reuptake inhibitor) are approved drugs. <sup>≠</sup>Represents respective rank order within SBVS-150 set by Glide XP, Surflex, and FlexX dockers’ scores. N.A. indicates hits that were not part of the original consensus docking protocol but were included later to test a similar molecule (#71-S) or repurposing of approved drugs<sup>€</sup>.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Chemical Expansion (Structure–Activity Relationship) of Hit Compound #48 Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0008.gif" alt="" id="GRAPHIC-d3383e385-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">% inhibition at 100 μM (±sd)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">R<sub>2</sub></th><th class="colsep0 rowsep0" align="center">R<sub>4</sub></th><th class="colsep0 rowsep0" align="center">R<sub>5</sub></th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center">R<sub>2</sub>′</th><th class="colsep0 rowsep0" align="center">R<sub>3</sub>′</th><th class="colsep0 rowsep0" align="center">R<sub>4</sub>′</th><th class="colsep0 rowsep0" align="center">R<sub>5</sub>′</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48</td><td class="colsep0 rowsep0" align="left">54.5 ± 8.0</td><td class="colsep0 rowsep0" align="left">112</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-02</td><td class="colsep0 rowsep0" align="left">62.1 ± 0.5</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-03</td><td class="colsep0 rowsep0" align="left">41.1 ± 2.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-04</td><td class="colsep0 rowsep0" align="left">75.65 ± 1.9</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">CH═CH–CH═CH</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-05</td><td class="colsep0 rowsep0" align="left">91.4 ± 0.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH(CH<sub>3</sub>)<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-06</td><td class="colsep0 rowsep0" align="left">91.6 ± 4.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Ph</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-07</td><td class="colsep0 rowsep0" align="left">49.3 ± 1.0</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-08</td><td class="colsep0 rowsep0" align="left">39.0 ± 3.5</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCO</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-09</td><td class="colsep0 rowsep0" align="left">43.5 ± 0.7</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCO</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-10</td><td class="colsep0 rowsep0" align="left">39.6 ± 2.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCO</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-11</td><td class="colsep0 rowsep0" align="left">81.5 ± 3.3</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-12</td><td class="colsep0 rowsep0" align="left">83.5 ± 0.4</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-13</td><td class="colsep0 rowsep0" align="left">41.4 ± 5.0</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-14</td><td class="colsep0 rowsep0" align="left">79.1 ± 1.4</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">C<sub>2</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-15</td><td class="colsep0 rowsep0" align="left">78.0 ± 1.3</td><td class="colsep0 rowsep0" align="left">40</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCO</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-20</td><td class="colsep0 rowsep0" align="left">35.49 ± 3.4</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#48-19</td><td class="colsep0 rowsep0" align="left">22.3 ± 2.4</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OCO</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" colspan="2" align="left">CH═CH–CH═CH</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">Percent inhibition at 100 μM (sd, standard deviation).</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">Compound concentration in μM that inhibited 50% of KLF10 reporter activity; n.d., not determined.</p></div></div></div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Chemical Expansion (Structure–Activity Relationship) o Hit Compound #15 Scaffold</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0009.gif" alt="" id="fx3" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">% inhibition at 100 μM (±sd)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">R2</th><th class="colsep0 rowsep0" align="center">R3</th><th class="colsep0 rowsep0" align="center">R4</th><th class="colsep0 rowsep0" align="center">R<sub>3</sub>′</th><th class="colsep0 rowsep0" align="center">R<sub>4</sub>′</th><th class="colsep0 rowsep0" align="center">R<sub>5</sub>′</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15</td><td class="colsep0 rowsep0" align="left">45.5 ± 4.1</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-07</td><td class="colsep0 rowsep0" align="left">39.1 ± 0.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH3</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-09</td><td class="colsep0 rowsep0" align="left">85.9 ± 1.3</td><td class="colsep0 rowsep0" align="left">43</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" colspan="2" align="left">S–C(−CH<sub>3</sub>)═C(−CH<sub>3</sub>)</td><td class="colsep0 rowsep0" align="left">OH</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-11</td><td class="colsep0 rowsep0" align="left">39.3 ± 1.6</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">CH3</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-13</td><td class="colsep0 rowsep0" align="left">32.6 ± 4.1</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">OCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH═CH–CH═CH</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">H</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-17</td><td class="colsep0 rowsep0" align="left">32.3 ± 5.8</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">COOCH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-OH</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">#15-21</td><td class="colsep0 rowsep0" align="left">37.4 ± 2.8</td><td class="colsep0 rowsep0" align="left">n.d.</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">-CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">CH<sub>3</sub></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Percent inhibition at 100 μM (sd, standard deviation).</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">Compound concentration in μM that inhibited 50% of KLF10 reporter activity; n.d., not determined.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Lead Compounds Inhibit Binding of KLF10 to DNA</h3><div class="NLM_p">To determine whether identified compounds inhibit the binding of KLF10 to DNA, KLF10 was incubated with double-stranded DNA containing a KLF10 consensus binding site in the presence of identified compounds #48, #48-15, or #15-09 or DMSO control. We first validated that purified KLF10 protein bound to the KLF10 consensus DNA-binding site in a specific manner, as the binding was completely blocked by unlabeled oligonucleotide DNA probe, and there was no binding using KLF10 protein lacking the DNA-binding domain (GST-KLF10 amino acid 111–210); in addition, the KLF10 DNA–protein complex was supershifted by a KLF10 antibody (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>A). The identified small molecule compounds were next examined for effects on KLF10 DNA-protein binding. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>B, compound #48 inhibited KLF10 DNA-binding activity by 25% and 11% at 40 μM and 8 μM, respectively. Compound #48-15 inhibited KLF10 DNA-binding activity by 45% and 12% at 40 μM and 8 μM, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>C). In addition, compound #15-09 inhibited KLF10 DNA-binding activity by 83% and 25% at 50 μM and 8 μM concentrations, respectively (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>D). In contrast, no dose-dependent inhibition on DNA-binding was observed for another member of the KLF family, KLF1 (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>E,F), showing specificity. Taken together, these data indicate that small molecule compounds #48, #48-15, and #15-09 could each inhibit binding of KLF10 protein to DNA.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of identified small molecule compounds on KLF10 DNA-binding activity by electromobility gel shift assays. (A) GST-KLF10 specifically bound to a KLF10 consensus DNA-binding sequence. Small molecule compounds #48 (B), #48-15 (C), and #15-09 (D) inhibited KLF10 DNA-binding activity. (E, F) In contrast, there was no inhibition of DNA-protein binding of the related KLF, KLF1, by small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Lead Compounds Inhibit Conversion of CD4+CD25– T Cells to CD4+CD25+ T Regulatory Cells</h3><div class="NLM_p">Our previous work and that of others have identified a critical role for KLF10 in its ability to convert CD4+CD25– T cells into CD4+CD25+ T regs.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12, 13)</a> To test if these lead compounds had any effect on T reg cell differentiation, CD4+CD25– T cells were cultured for the conversion of CD4+CD25– T cells to CD4+CD25+ T regulatory cells in the presence of the indicated small molecule compounds #48, #48-15, and #15-09. As shown in Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>, <a class="ref internalNav" href="#notes-3" aria-label="supplemental Figure 1">supplemental Figure 1</a>, and <a class="ref internalNav" href="#notes-3" aria-label="supplemental Table 1">supplemental Table 1</a>, compounds #48, #48-15, and #15-09 inhibited the percentage of CD4+CD25+ T regs by 57.4%, 59.1%, and 29.2%, respectively. Collectively, these findings suggest that small molecule compounds #48, #48-15, and #15-09 have the ability to alter T reg cell differentiation.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibition of conversion of CD4+CD25- T cells to CD4+CD25+ T regs by small molecule compounds #48, #48-15, and #15-09. CD4+CD25– T cells were isolated and cultured for the conversion to CD4+CD25+ T regs in the presence or absence of small molecule compounds #48 (A), #48-15 (B), and #15-09 (C) for 3 days and harvested for analysis by flow cytometry. Data show mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Identified Small Molecule Compounds Inhibited FoxP3 Expression during the Conversion of CD4+CD25– T Cells to CD4+CD25+ T Regulatory Cells</h3><div class="NLM_p">Foxp3, a forkhead transcription factor, is a critical protein involved in the differentiation of T regs.<a onclick="showRef(event, 'ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref23 ref24 ref25 ref26 ref27">(23-27)</a> Indeed, our previous report demonstrated that KLF10 can potently induce Foxp3 expression by binding to an evolutionarily conserved CACCC DNA-binding site in the proximal Foxp3 promoter.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> To examine the effects of the identified tool compounds #48, #48-15, and #15-09 on Foxp3 expression, CD4+CD25– T cells were cultured for the conversion to CD4+CD25+ T regs in the presence of each of the identified compounds. Compound #48, #48-15, and #15-09 inhibited Foxp3 expression by 84.7%, 82.5%, and 90.8%, respectively, at 50 μM (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> and <a class="ref internalNav" href="#notes-3" aria-label="supplemental Table 2">supplemental Table 2</a>). Taken together, these observations indicate that these compounds effectively suppress the KLF10 target gene Foxp3, a critical regulator of T reg cell differentiation.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition of FoxP3 expression by small molecule compounds #48, #48-15, and #15-09. CD4+CD25– T cells were isolated and cultured for the conversion to CD4+CD25+ T regs in the presence or absence of small molecule compounds #48 (50 μM), #48-15 (50 μM), and #15-09 (100 μM) and harvested for analysis by qPCR for Foxp3 expression. Data show the mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_14772" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_14772" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In recent years, in silico CADD and lead optimization strategies including structure-based and ligand-based virtual screens have become popular and successfully complement or, in certain cases, replace HTS approaches for early stage identification of potential lead compounds with improved chemical diversity and enhanced pharmacodynamic and pharmacokinetic properties. KLFs constitute an important class of transcriptional regulators in diverse aspects of cell growth and differentiation. Furthermore, specific KLFs have been implicated in disease states such as tumorigenesis, inflammation, and cardiovascular disease, among others. However, given the lack of enzymatic activity, transcription factors in general have largely been considered “nondruggable” to small molecule inhibition.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a> This provides an exciting and challenging opportunity for those in the field of drug discovery. To date, examples of small molecules targeting transcription factors have largely focused on protein–protein interactions of “dimeric” transcription factors, so-called because of their need to form heterodimers or homodimers for efficient transcriptional activation and DNA binding including c-Myc/Max, HiF1-α/HIF-1β, and STAT3 dimers.<a onclick="showRef(event, 'ref29 ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref29 ref30 ref31">(29-31)</a></div><div class="NLM_p">There are few reported attempts of targeting other ZFTFs by small druglike molecules. Lin et al.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> screened a 250-membered heterocycle library containing diverse scaffolds for activation of the reporter gene after transfection with the zinc finger C7 protein, a structural variant of the murine zinc finger zif268. As a result, a potent 2-(4′-quinoline)benzimidazole compound was identified that induced luciferase activity by about 18-fold at 100 μM concentration (EC<sub>50</sub> = 35 μM); however, at concentrations above 100 μM it exhibited significant cytotoxicity. Lauth et al.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> used a cell-based screen to identify small-molecule antagonists of GLI-mediated transcription, thereby inhibiting Hedgehog signaling implicated in several malignancies. They discovered two small molecules, GANT61 and GANT58 (NSC 136476 and NSC 75503, respectively, where GANT stands for GLI antagonist), which inhibited the GLI-induced transcription in transfected HEK293 cells at 30 μM concentration (<a class="ref internalNav" href="#notes-3" aria-label="supplemental Figure 2">supplemental Figure 2</a>).</div><div class="NLM_p">We report the first-in class discovery of inhibitors of the KLF10-DNA interaction interface using small “druglike” molecules. For structure-based screening, we generated a working homology model of KLF10 in association with DNA with attention to modeling of side chains of amino acids at the DNA interface (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>A). In the absence of prior knowledge of binding pocket or point-mutation data critical in DNA binding, we carefully studied the protein–DNA interface of KLF10 using independently developed pocket identification algorithms to identify, with consensus, a potentially druggable pocket for high throughput docking (HTD) of small molecule databases. A large, shallow pocket was observed within the middle zinc finger region, and we subclassified this binding pocket as α- and β-pocket depending on the proximity with α-helix or β-sheet residues surrounding it (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B). We hypothesized that tighter binding of small molecules in this pocket would provide an opportunity to interfere with KLF10-DNA binding and subsequent inhibition of transcriptional activity.</div><div class="NLM_p">A fast in silico HTD screen of publically available databases using the α- and β-pockets separately was performed with constrained Glide docking in SP settings. To offer both pockets an equal opportunity for binding, we selected ∼1800 molecules that exhibited binding in both pockets. In the next level of filtering we provided every molecule in this set to dock without constraints and used a more accurate scoring function available through Glide XP docking. The outcome of this analysis showed that a good portion of these molecules (∼400 molecules) were thrown away from this interface and ∼1400 molecules (GlideXP-1400) were able to virtually bind in one of the proposed pockets.</div><div class="NLM_p">While the current state of the art in docking simulations and scoring technologies are very good at generating the correct binding pose of a given ligand, they are less facile in identifying the bioactive conformation as a top scoring pose (scoring issue) or rank-ordering the bioactive poses of different ligands (rank-ordering issue), effects that determine the success of a docker in high throughput virtual screening. Because none of the presently available dockers are capable of performing well in overcoming the scoring and rank-ordering issues, researchers apply consensus docking and scoring strategies to improve the enrichment of in silico screens.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> In the present study, we performed docking of the GlideXP-1400 molecule set using two additional well-known dockers commercially available to the scientific community: Surflex and FlexX. We selected 50% of the molecules top-ranked by all three dockers (Glide XP, Surflex, and FlexX) and chose a common set of 150 molecules (SBVS-150) after performing ADME/Tox filtering analysis (<a class="ref internalNav" href="#notes-3" aria-label="supplemental Table 3">supplemental Table 3</a>). The KLF10 luciferase reporter screen of 40 molecules from the SBVS-150 set validated the activity of 11 structurally different inhibitor scaffolds (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). We selected compounds #48 and #15 for further chemical space expansion leading to more than 30 additional inhibitors. Each of the three compounds #48, #48-15, and #15-09 inhibited KLF10 transcriptional activity (Figures <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a> and <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>), KLF10-DNA binding (gel mobility shift assay) (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), conversion of CD4+CD25– T cells to CD4+CD25+ T regs (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>), and Foxp3 gene expression (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>).</div><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Binding Mode and Molecular Interactions Analysis of Compounds #48, #48-15, and #15-09</h3><div class="NLM_p">Owing to the shallow pocket, there was variation in top binding poses from different docking runs. In order to explore this variability problem in multiple scales of speed and accuracy, we performed two types of optimizations on the top 50 poses of #15 and #48 series of inhibitors obtained from the Glide XP docking run as below: (1) Glide “refine”ment, in which only the ligand was allowed to relax in the protein binding pocket followed by rescoring of each pose with Glide XP function; (2) eMBrAcE minimization, which allowed both ligand and its surrounding 5 Å residues to relax, followed by in-place scoring and ranking of eMBrAcE minimized complexes using Glide. An analysis of output of these two optimization procedures revealed that rank ordering or enrichment of eMBrAcE-optimized complexes correlated better to luciferase activity of these hits, justifying the need for protein and ligand relaxation to obtain a more stable binding pose (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>). However, none of these attempts categorically separated actives from false positives (decoys).</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0007.jpeg" id="GRAPHIC-d3383e1621-autogenerated" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding modes of compounds #48 (A), #48-15 (B), and #15-09 (C) as predicted by Glide docking simulation. Protein side chains in green refer to the original position of receptor coordinates used in docking. The compound in gray stick refers to the rescored pose. The compound in torquise (sky blue) stick refers to the refined ligand pose. The compound in plum/pink stick refers to the eMBrAcE minimized (and xGlide rescored top) pose. Yellow dotted lines indicate H-bonds between ligand and receptor. The eMBrAcE minimization considers flexibility of both ligand and residues within 5 Å pocket surrounding the ligand and is shown superimposed (plum colored side chain) to outline the changes after minimization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0006.jpeg jm-2014-018187_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Docking with eMBrAcE minimization of compound #48 within the KLF10 pocket shows the possibility of various binding modes without substantial loss of binding energy (docking score, DS: −5.4 to −4.5), with its resorcinol ring occupying either the α- or β-pocket. The top pose of compound #48 stably docked in α-pocket (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A); however, other poses can also fit into β-pocket. In the α-pocket, the resorcinol ring of compound #48 participates in π-interactions with the phenyl ring of Phe410 and two hydroxyl groups form hydrogen bond interactions with side chains of His419 (through <i>p</i>-OH) and Glu415 (through <i>o</i>-OH). Among the Glide “refine” poses (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A, green sticks), the top pose occupied only the α-pocket with two -OH forming hydrogen bonds with His419 (through <i>o</i>-OH) and Arg418 (through <i>p</i>-OH) and toluene ring resting in the extreme of the α-pocket, whereas the rest of the poses occupied both pockets with either of two possible sets of hydrogen bonds between hydroxyl groups of compound #48 and His419 and Glu415 or Glu415 and Lys397 (C═O). Compound #48-15 represents a binding mode with two rings occupying two subpockets and a hydroxyl group forming a hydrogen bond with either His419 or Glu415, an effect that may be due to its symmetrical linker (-CO-CH<sub>2</sub>-CO-) connecting two rings (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). An analysis of Glide “refine” poses (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, green sticks) also revealed a unique binding mode occupying both pockets, with a hydroxyl group making hydrogen bonding interactions with the side chain carboxyl of Glu415 and the backbone amine of Ala411 (bifurcated H-bond), whereas the carbonyl oxygen of ester in compound #48-15 forms a hydrogen bond with Lys397. In the representative binding modes of the eMBrAcE minimized poses (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B, plum sticks), the phenolic ring occupied β-pocket (in some cases it forms H-bond with carboxylic oxygen of Glu415), while oxygen of methyl ether in the α-pocket forms a hydrogen bond with side chain nitrogen of His419 and carbonyl oxygen of linker ester forms a hydrogen bond with Lys397.</div><div class="NLM_p">The binding mode analysis of #15-09 revealed that it occupies both pockets (Figures <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C; gray, green, and plum sticks), with the pyrimidothiophene ring residing in the β-pocket and <i>m</i>-OH group of the other ring in the α-pocket, forming hydrogen bonds with the guanidinium side chain of Arg418, backbone oxygen of Glu415, and/or the side chain imidazole of His419. The Glide “refine” as well as eMBrAcE minimized poses also revealed the similar pattern of binding for compound #15-09 but more equivocally and with least variation in poses due to relaxation of ligand and/or receptor pocket after optimization.</div><div class="NLM_p last">Thus, the general binding patterns of compound #15 series scaffolds has an overall similarity with the binding pattern of #48 compound series scaffold such that two aromatic rings occupy the two pockets. This may be, in part, due to the similarity in the linker [−X–CH2–C(═O)−], separating the aromatic rings placed in two respective pockets and forming a bridge to connect them. This similarity, in part, may also underlie the mechanism of binding of these compounds, as has been reflected in their luciferase potencies (IC<sub>50</sub> values of #48-15 and #15-09 are ∼40 μM), ability to inhibit DNA binding, and T reg cell differentiation. The length and nature of the linker [X = O versus X = OC(═O)] in these series (Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) may play a serious role in proper placement of aromatic rings in respective pockets which may afford further stability through aromatic and hydrophobic interactions in addition to physical anchoring for tighter binding. As a result, modifying the linker could provide future scope for additional series of inhibitors, which may be even more potent than the ones reported here.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_15378" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_15378" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we demonstrate proof-of-principle for the discovery of small molecule inhibitors of the KLF10-DNA interaction interface using in silico screens of chemical libraries. Interrogation of a “druggable” pocket in the second zinc-finger of KLF10 revealed three molecules, #48, #48-15, and #15-09, with similar scaffolds and binding patterns. Indeed, each of these small molecules inhibited KLF10-DNA binding and transcriptional activity, conversion of CD4+CD25– T cells to CD4+CD25+ T regs, and KLF10 target gene expression. Taken together, these findings support the feasibility of using CADD with functional assays to identify small molecules that target members of the KLF subfamily of transcription factors to regulate biological functions in health and disease.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65144" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65144" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Computational Methods</h3><div class="NLM_p last">All molecular modeling operations were carried out in the Maestro (version 9.0; Schrödinger Inc., USA), Sybyl (version 7.2; Certara Inc. USA), and Molecular Operating Environment (version 2009.10; Chemical Computing Group Inc., Canada) modeling packages running on Dell Precision 690 workstation with 8 CPUs, 10 GB memory, and Red Hat Enterprise 5 Linux operating system.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Homology Modeling of KLF10</h3><div class="NLM_p">A truncated model of three zinc fingers of KLF10 (amino acids 361–459) was built using F-chain of X-ray crystal structure<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MEY">1MEY</a>) of a designed zinc finger protein in complex with DNA. Sequences of KLF10 and template proteins exhibited 51% identity, leaving weaker coordinate information for two small loops: amino acids 431–436 and 449–451. The two loops were refined using Extended High (accuracy) loop sampling method in Prime (version 2.1).<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35, 36)</a> The coordinates for Zn<sup>2+</sup> ions were extracted from the template structure and verified for coordination distances with corresponding cystines and histidines. The (coordinate bonded) Cys-Zn<sup>2+</sup>-His pattern was preserved from the X-ray structure positions during side chain refinements by applying distance constraints. This was followed by a protein structure analysis and refinement of disallowed backbone (θ, ψ, and Ω) and side chain dihedral angles, peptide planarity, bond length and angle deviations, and steric clashes between side chains. Finally, DNA-bound KLF10 structure was built by extracting DNA coordinates from X-ray crystal structure and resolving steric clashes by rotamer search of side chains of interface residues having bumps with DNA nucleotides, followed by optimization of overall interaction energy of complex using OPLS-2005 (optimized potential for liquid simulations)<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> molecular mechanics energy minimizations (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). The optimized structure (without DNA coordinates) was then subjected to binding pocket analysis using pocket identification algorithms employing different methods. For example, SiteMap<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14, 15)</a> which is a combination of geometry-based and energy-based methods<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> along with molecular surface calculations in Maestro was applied in addition to MolCad<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> (Sybyl/Tripos), which is a grid-based flood-filled solvation technique.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Another tool that was used in pocket identification, SiteFinder,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> is based on the calculation of appropriately exposed α spheres that are clustered to produce a collection of sites. Each site consists of several α spheres, at least one of which is hydrophobic. The sites are then ranked according to the number of hydrophobic contacts made with the receptor, i.e., the number of hydrophobic atoms within a contact distance of any of the retained α spheres to uniquely identify a “druggable” pocket at the interface.</div><div class="NLM_p last">These programs identified, with consensus, a relatively big pocket in the second ZF (aa 385–425) which spans from β-sheet to α-helix (near Zn<sup>2+</sup>), subdivided with a small cleft into, correspondingly termed, α- and β-subpockets, which were subsequently used for high throughput docking (HTD) of chemical libraries (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>B).</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Structure-Based Virtual Screening (SBVS)</h3><div class="NLM_p last">The chemical libraries used in the present screening included structures from Chembridge (<a href="http://www.chembridge.com" class="extLink">http://www.chembridge.com</a>; ∼400 000 molecules), Asinex (<a href="http://www.asinex.com" class="extLink">http://www.asinex.com</a>; ∼200 000 molecules), National Cancer Institute (<a href="http://dtp.nci.nih.gov/docs/3d_database/dis3d.html" class="extLink">http://dtp.nci.nih.gov/docs/3d_database/dis3d.html</a>; ∼250 000 molecules), and ZINC<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> databases (downloaded as of June 2008 and accessed online later for similarity and substructure searches). The structures were prepared using Ligprep (version 2.2), which performed hydrogen addition and 3D structure optimization. Epik (through Ligprep) was used to identify possible ionization and tautomer states at physiological pH (7.0 ± 2.0), along with generating ring conformers and stereoisomers using default settings. The ionization state penalties for each protomer state were also calculated and stored. Two separate receptor grids for docking calculations were prepared, one for each of α- and β-subpocket, by assigning individual residues forming the respective pocket, with hydrophobic and H-bond constraints. A high-throughput docking (HTD) was performed to filter out weak binders in α- and β-subpockets separately using Glide (version 5.0)<a onclick="showRef(event, 'ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref18 ref19">(18, 19)</a> in standard precision (SP) scoring mode. An analysis of hits that satisfied the hydrogen bond and hydrophobic constraints revealed that approximately 4 times more hits preferred α-pocket over β-pocket, and ∼1800 hits were common to both pockets. A third receptor grid encompassing residues from both α- and β-subpockets was used for Glide (version 5.5) docking without constraints, and using more accurate extra precision (XP) scoring.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Postdocking analysis identified that 1400 hits occupied α- and/or β-subpocket, leaving out remaining molecules docked off-site of the TF-DNA interface. This GlideXP-1400 set was then further docked using additional two independently developed dockers, Surflex<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> and FlexX<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> in Sybyl (version 7.2). The hits common in top 700 rank-order of each docking algorithm were compared to short list of 150 molecules (SBVS-150 set) after ADMET evaluation using QikProp (version 3.1; Schrodinger) and Sarchitect (Strand Life Science). A subset of 40 compounds was evaluated in luciferase reporter assays. Compounds were acquired from different vendors and dissolved in DMSO. The 2D structures of active compounds are shown in Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>. The purity of compounds #48, #48-15, and #15-09 was verified to be ≥95% using HPLC equipped with (a) column, Onyx C18 50 mm × 4.6 mm (flow rate of 3.75 mL/min), (b) UV detector operating at dual wavelengths, and (c) evaporative light-scattering detector (<a class="ref internalNav" href="#notes-3" aria-label="supplemental Figure 3">supplemental Figure 3</a>). Additionally, small molecules were also screened for substructural features that appear in frequent hitters (promiscuous compounds) in several biochemical HTS. Such compounds are also known as pan assay interference compounds (PAINS), and researchers have created systematic filters to remove such compounds from virtual screens.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> We employed PAINS1, PAINS2, and PAINS3 filter definitions as implemented in Schrodinger’s Canvas program that correspond to filter C, filter B, and filter A, respectively, of a previous study.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> None of our screened compounds (Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>–<a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) showed PAINS attributes as per the structure definitions of PAINS1 (filter C) and PAINS3 (filter A). However, compounds #57 (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), #15-07, and #15-09 (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) were marked as possible PAINS according to PAINS2 (filter B) substructure definitions. Since these compounds were passed in other two PAINS (1 and 3) filters, we assume that the promiscuous liability is relatively less, making worth investigating their effects in this novel (nonenzymatic) and relatively undruggable target space.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Chemical Structure Exploration of Lead Compound Scaffolds</h3><div class="NLM_p last">Potent and promising hits were subjected to chemical space exploration by performing substructure search of their scaffolds to find their close analogs (second-generation compounds) in databases. The structural information on inhibitors identified in SBVS screens was employed to identify a working pharmacophore query using Phase<a onclick="showRef(event, 'ref42 ref43'); return false;" href="javascript:void(0);" class="ref ref42 ref43">(42, 43)</a> (version 3.1; Schrodinger, Inc.). This query was used to bias the selection of second-generation compounds toward chemical features of known inhibitors to improve enrichment. The structural analogs were subsequently evaluated using Glide XP docking to rank-order a small subset for activity testing (Tables <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a> and <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Constructs, Cell Culture and Transfection, and Luciferase Reporter Assays</h3><div class="NLM_p last">CACC-TK-luc plasmid contains three tandem copies of CACCC sites upstream to a luciferase reporter in pGL3-basic as previously described.<a onclick="showRef(event, 'ref12 ref44'); return false;" href="javascript:void(0);" class="ref ref12 ref44">(12, 44)</a> Empty vector pCMV-SPORT6 and human KLF10 expressing plasmid pCMV-SPORT6-KLF10 are from Open Biosystems (Huntsville, AL). HeLa cells from ATCC were cultured in DMEM medium with 10% fetal bovine serum, 1% antibiotics. Cells were transfected by using Fugene 6 transfection reagent from Roche. In brief, cells were plated (40 000/well) in triplicate in a 12-well plate. After growing to 70–80% confluency, cells were transfected with 500 ng of CACC-TK-luc reporter construct, 200 ng of β-galactosidase expression plasmids, and 100 ng of pcDNA3.1 empty vector or pcDNA3.1-KLF10. After a 24 h incubation, cells were treated with 100 μM or indicated concentration of small molecule compounds for 24 h. Treated cells were collected in 200 μL of 1× reporter lysis buffer (Promega). The activity of luciferase were measured and normalized to total protein read for the same lysate performed in triplicate replicates.</div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Real Time qPCR</h3><div class="NLM_p last">T cells were suspended in TRIzol reagent (Invitrogen), and total RNA was isolated according to the manufacturer’s instructions. Reverse transcriptions were performed by using the reverse transcription kit and QuantiTect SYBR Green reverse transcription-PCR kit from Qiagen for quantitative real-time quantitative PCR analysis on a Mx3000P real-time PCR system (Stratagene) following the manufacturer’s instructions. Primers were used to detect mouse FoxP3 as previously described.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Gel Mobility Shift Assays</h3><div class="NLM_p last">KLF10 recombinant protein was obtained from Novus Biologicals. Oligonucleotides 5′-GCAGTGAAAAGGGGGTGTGTCAGGATGC and 5′-GCATCCTGACACACCCCCTTTTCACTGC were generated as previously described<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> and labeled with biotin using biotin 3′ end DNA labeling kit (Pierce). End-labeled oligos (1 μmol/L) were annealed in a buffer system with 10 mM Tris, 50 mM NaCl, pH 8.0, by heating at 95 °C for 5 min, then slowly cooled to 65 °C for 30 min incubation, and cooled to room temperature. Lightshift chemiluminescent EMSA kit (20148) was from Thermo Scientific. Binding reactions contain 1× binding buffer, 2.5% glycerol, 5 mM MgCl<sub>2</sub>, 50 ng/μL poly(dI·dC), 0.05% NP-40, and 5 fmol of biotin-labeled probes in the presence of different concentrations of compounds, and 100 ng of purified recombinant KLF10. Compounds were incubated with KLF10 in a reaction system for 4 h at room temperature, and then biotin-labeled probes were added for a 1 h incubation. Binding reactions were electrophoresed for 80 min at 100 V on 6% polyacrylamide gel in 0.5× TBE. The binding reactions were transferred to nylon membranes at 100 V for 45 min, and transferred DNA was cross-linked to membrane using an HL-2000 Hybrilinker instrument. Biotin-labeled DNAs were detected by chemiluminescence.</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> T Regulatory Cell Differentiation Assay and Flow Cytometry</h3><div class="NLM_p last">Mouse spleens were meshed through a cell strainer into phosphate buffered saline (PBS) with 2% fetal bovine serum and 1% penicillin/streptomycin, and cells were collected by spinning at 1000 rpm for 10 min. CD4+CD25– T cells were purified by using CD4+CD25+ T cell isolation kit (Miltenyi Biotec) according to the manufacturer’s instructions. Briefly, cells were incubated with biotin-conjugated antibodies, followed by anti-biotin microbeads and PE-conjugated anti-CD25 antibody for 30 min on ice with gentle shaking. Mixtures were loaded into prewashed LD columns in the magnetic field, and CD4+ cells were collected and then incubated with anti-PE microbeads. Mixtures were loaded into prewashed MS columns in the magnetic field to collect CD4+CD25– T cells (nonbound cell fraction). Cells were plated into 12-well plates precoated with 1 μg/mL anti-CD3e (eBioscience) overnight in RPMI1640 medium with 10% FBS, 1% penicillin/streptomycin, 1% nonessential amino acids, and 2-mercaptoethanol, 1 μg/mL anti-CD28 (BD Biosciences), and 1 ng/mL TGF-β1 (Peprotech). Cells were cultured at 37 °C up to 5 days for the conversion of CD4+CD25– T cells to CD4+CD25+ T regs. For flow cytometry, cells were stained with anti-mouse CD25 Alexa Fluo488 (eBioscience) and analyzed by using multicolor FACS (ARIA; BD Biosciences) as previously described.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i24"><a href="/doi/suppl/10.1021/jm5018187">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Supplemental Figures 1–3 and supplemental Tables 1–3 that include the raw data (flow cytometry and RT-PCR) used to generate Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a> and <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a> (inhibition of conversion of CD4+CD25– T cells to CD4+CD25+ T regs by small molecule compounds), structures of previously reported inhibitors of another transcription factor GLI, the analytical data that support purity of compounds #48, #48-15, and #15-09, and the list of molecules from the SBVS-150 set. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5018187/suppl_file/jm5018187_si_001.pdf">jm5018187_si_001.pdf (20.13 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm5018187" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24065" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24065" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Alan C. Rigby</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Molecular and Vascular Medicine, Center for
Vascular Biology Research, Beth Israel Deaconess
Medical Center, 99 Brookline
Avenue, RN-227, Boston, Massachusetts 02215, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#62230e030c4c300b05001b220b0f010e0d0c074c010d0f"><span class="__cf_email__" data-cfemail="a6e7cac7c888f4cfc1c4dfe6cfcbc5cac9c8c388c5c9cb">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mark W. Feinberg</span> - <span class="hlFld-Affiliation affiliation">Cardiovascular
Division, Brigham and Women’s Hospital, Harvard Medical School, 77 Avenue Louis Pasteur, NRB-742F, Boston, Massachusetts 02115, United States</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ed808b8884838f889f8aad9d8c9f9983889f9ec3829f8a"><span class="__cf_email__" data-cfemail="e18c8784888f83849386a1918093958f849392cf8e9386">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Santosh A. Khedkar</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicine, Division of Molecular and Vascular Medicine, Center for
Vascular Biology Research, Beth Israel Deaconess
Medical Center, 99 Brookline
Avenue, RN-227, Boston, Massachusetts 02215, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xinghui Sun</span> - <span class="hlFld-Affiliation affiliation">Cardiovascular
Division, Brigham and Women’s Hospital, Harvard Medical School, 77 Avenue Louis Pasteur, NRB-742F, Boston, Massachusetts 02115, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.A.K. and X.S. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i27" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i27"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i28" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i28"> Abbreviations Used</h2><tr><td class="NLM_term">CD4</td><td class="NLM_def"><p class="first last">cluster of differentiation 4</p></td></tr><tr><td class="NLM_term">CD25</td><td class="NLM_def"><p class="first last">cluster of differentiation 25</p></td></tr><tr><td class="NLM_term">eMBrAcE</td><td class="NLM_def"><p class="first last">multiligand bimolecular association with energetics</p></td></tr><tr><td class="NLM_term">Foxp3</td><td class="NLM_def"><p class="first last">forkhead box P3</p></td></tr><tr><td class="NLM_term">HTD</td><td class="NLM_def"><p class="first last">high throughput docking</p></td></tr><tr><td class="NLM_term">KLF10</td><td class="NLM_def"><p class="first last">Krüppel-like factor 10</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan assay interference compounds</p></td></tr><tr><td class="NLM_term">TF</td><td class="NLM_def"><p class="first last">transcription factor</p></td></tr><tr><td class="NLM_term">T regs</td><td class="NLM_def"><p class="first last">T regulatory cells</p></td></tr><tr><td class="NLM_term">TGF-β1</td><td class="NLM_def"><p class="first last">transforming growth factor β1</p></td></tr><tr><td class="NLM_term">SP</td><td class="NLM_def"><p class="first last">standard precision</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">standard deviation</p></td></tr><tr><td class="NLM_term">XP</td><td class="NLM_def"><p class="first last">extra precision</p></td></tr><tr><td class="NLM_term">ZF</td><td class="NLM_def"><p class="first last">zinc finger</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_75478" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_75478" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 45 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Venter, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mural, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yandell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holt, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocayne, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amanatides, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballew, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodira, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skupski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabor Miklos, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadeau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKusick, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinder, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slayman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunkapiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delcher, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dew, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasulo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanigan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florea, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halpern, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannenhalli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kravitz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobarry, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinert, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remington, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Threideh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beasley, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biddick, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonazzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cargill, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandramouliswaran, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlab, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Francesco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eilbeck, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelista, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabrielian, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketchum, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkulov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milshina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naik, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nusskern, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shue, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wides, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumhueter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravchik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodage, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldwin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baden, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnstead, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrow, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beeson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Center, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danaher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desilets, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doup, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferriera, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garg, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gluecksmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hladun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howland, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibegwam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalush, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kline, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koduru, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Love, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCawley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntosh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMullen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfannkoch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratts, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puri, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qureshi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reardon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romblad, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruhfel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smallwood, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tint, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vech, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn-Deen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaveri, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaveri, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abril, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guigo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjolander, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlak, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kejariwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazareva, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narechania, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diemer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muruganujan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bafna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Istrail, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walenz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yooseph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxendale, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blick, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caminha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carnes-Stine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caulk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mays, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombroski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esparham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fosler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gire, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glanowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glodek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorokhov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gropman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoover, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasha, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraft, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levitsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majoros, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDaniel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nodell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprague, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venter, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zandieh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span> </span><span class="NLM_article-title">The sequence of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">1351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1126%2Fscience.1058040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=11181995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtlSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2001&pages=1304-1351&author=J.+C.+Venterauthor=M.+D.+Adamsauthor=E.+W.+Myersauthor=P.+W.+Liauthor=R.+J.+Muralauthor=G.+G.+Suttonauthor=H.+O.+Smithauthor=M.+Yandellauthor=C.+A.+Evansauthor=R.+A.+Holtauthor=J.+D.+Gocayneauthor=P.+Amanatidesauthor=R.+M.+Ballewauthor=D.+H.+Husonauthor=J.+R.+Wortmanauthor=Q.+Zhangauthor=C.+D.+Kodiraauthor=X.+H.+Zhengauthor=L.+Chenauthor=M.+Skupskiauthor=G.+Subramanianauthor=P.+D.+Thomasauthor=J.+Zhangauthor=G.+L.+Gabor+Miklosauthor=C.+Nelsonauthor=S.+Broderauthor=A.+G.+Clarkauthor=J.+Nadeauauthor=V.+A.+McKusickauthor=N.+Zinderauthor=A.+J.+Levineauthor=R.+J.+Robertsauthor=M.+Simonauthor=C.+Slaymanauthor=M.+Hunkapillerauthor=R.+Bolanosauthor=A.+Delcherauthor=I.+Dewauthor=D.+Fasuloauthor=M.+Flaniganauthor=L.+Floreaauthor=A.+Halpernauthor=S.+Hannenhalliauthor=S.+Kravitzauthor=S.+Levyauthor=C.+Mobarryauthor=K.+Reinertauthor=K.+Remingtonauthor=J.+Abu-Threidehauthor=E.+Beasleyauthor=K.+Biddickauthor=V.+Bonazziauthor=R.+Brandonauthor=M.+Cargillauthor=I.+Chandramouliswaranauthor=R.+Charlabauthor=K.+Chaturvediauthor=Z.+Dengauthor=V.+Di+Francescoauthor=P.+Dunnauthor=K.+Eilbeckauthor=C.+Evangelistaauthor=A.+E.+Gabrielianauthor=W.+Ganauthor=W.+Geauthor=F.+Gongauthor=Z.+Guauthor=P.+Guanauthor=T.+J.+Heimanauthor=M.+E.+Higginsauthor=R.+R.+Jiauthor=Z.+Keauthor=K.+A.+Ketchumauthor=Z.+Laiauthor=Y.+Leiauthor=Z.+Liauthor=J.+Liauthor=Y.+Liangauthor=X.+Linauthor=F.+Luauthor=G.+V.+Merkulovauthor=N.+Milshinaauthor=H.+M.+Mooreauthor=A.+K.+Naikauthor=V.+A.+Narayanauthor=B.+Neelamauthor=D.+Nusskernauthor=D.+B.+Ruschauthor=S.+Salzbergauthor=W.+Shaoauthor=B.+Shueauthor=J.+Sunauthor=Z.+Wangauthor=A.+Wangauthor=X.+Wangauthor=J.+Wangauthor=M.+Weiauthor=R.+Widesauthor=C.+Xiaoauthor=C.+Yanauthor=A.+Yaoauthor=J.+Yeauthor=M.+Zhanauthor=W.+Zhangauthor=H.+Zhangauthor=Q.+Zhaoauthor=L.+Zhengauthor=F.+Zhongauthor=W.+Zhongauthor=S.+Zhuauthor=S.+Zhaoauthor=D.+Gilbertauthor=S.+Baumhueterauthor=G.+Spierauthor=C.+Carterauthor=A.+Cravchikauthor=T.+Woodageauthor=F.+Aliauthor=H.+Anauthor=A.+Aweauthor=D.+Baldwinauthor=H.+Badenauthor=M.+Barnsteadauthor=I.+Barrowauthor=K.+Beesonauthor=D.+Busamauthor=A.+Carverauthor=A.+Centerauthor=M.+L.+Chengauthor=L.+Curryauthor=S.+Danaherauthor=L.+Davenportauthor=R.+Desiletsauthor=S.+Dietzauthor=K.+Dodsonauthor=L.+Doupauthor=S.+Ferrieraauthor=N.+Gargauthor=A.+Gluecksmannauthor=B.+Hartauthor=J.+Haynesauthor=C.+Haynesauthor=C.+Heinerauthor=S.+Hladunauthor=D.+Hostinauthor=J.+Houckauthor=T.+Howlandauthor=C.+Ibegwamauthor=J.+Johnsonauthor=F.+Kalushauthor=L.+Klineauthor=S.+Koduruauthor=A.+Loveauthor=F.+Mannauthor=D.+Mayauthor=S.+McCawleyauthor=T.+McIntoshauthor=I.+McMullenauthor=M.+Moyauthor=L.+Moyauthor=B.+Murphyauthor=K.+Nelsonauthor=C.+Pfannkochauthor=E.+Prattsauthor=V.+Puriauthor=H.+Qureshiauthor=M.+Reardonauthor=R.+Rodriguezauthor=Y.+H.+Rogersauthor=D.+Rombladauthor=B.+Ruhfelauthor=R.+Scottauthor=C.+Sitterauthor=M.+Smallwoodauthor=E.+Stewartauthor=R.+Strongauthor=E.+Suhauthor=R.+Thomasauthor=N.+N.+Tintauthor=S.+Tseauthor=C.+Vechauthor=G.+Wangauthor=J.+Wetterauthor=S.+Williamsauthor=M.+Williamsauthor=S.+Windsorauthor=E.+Winn-Deenauthor=K.+Wolfeauthor=J.+Zaveriauthor=K.+Zaveriauthor=J.+F.+Abrilauthor=R.+Guigoauthor=M.+J.+Campbellauthor=K.+V.+Sjolanderauthor=B.+Karlakauthor=A.+Kejariwalauthor=H.+Miauthor=B.+Lazarevaauthor=T.+Hattonauthor=A.+Narechaniaauthor=K.+Diemerauthor=A.+Muruganujanauthor=N.+Guoauthor=S.+Satoauthor=V.+Bafnaauthor=S.+Istrailauthor=R.+Lippertauthor=R.+Schwartzauthor=B.+Walenzauthor=S.+Yoosephauthor=D.+Allenauthor=A.+Basuauthor=J.+Baxendaleauthor=L.+Blickauthor=M.+Caminhaauthor=J.+Carnes-Stineauthor=P.+Caulkauthor=Y.+H.+Chiangauthor=M.+Coyneauthor=C.+Dahlkeauthor=A.+Maysauthor=M.+Dombroskiauthor=M.+Donnellyauthor=D.+Elyauthor=S.+Esparhamauthor=C.+Foslerauthor=H.+Gireauthor=S.+Glanowskiauthor=K.+Glasserauthor=A.+Glodekauthor=M.+Gorokhovauthor=K.+Grahamauthor=B.+Gropmanauthor=M.+Harrisauthor=J.+Heilauthor=S.+Hendersonauthor=J.+Hooverauthor=D.+Jenningsauthor=C.+Jordanauthor=J.+Jordanauthor=J.+Kashaauthor=L.+Kaganauthor=C.+Kraftauthor=A.+Levitskyauthor=M.+Lewisauthor=X.+Liuauthor=J.+Lopezauthor=D.+Maauthor=W.+Majorosauthor=J.+McDanielauthor=S.+Murphyauthor=M.+Newmanauthor=T.+Nguyenauthor=N.+Nguyenauthor=M.+Nodellauthor=S.+Panauthor=J.+Peckauthor=M.+Petersonauthor=W.+Roweauthor=R.+Sandersauthor=J.+Scottauthor=M.+Simpsonauthor=T.+Smithauthor=A.+Spragueauthor=T.+Stockwellauthor=R.+Turnerauthor=E.+Venterauthor=M.+Wangauthor=M.+Wenauthor=D.+Wuauthor=M.+Wuauthor=A.+Xiaauthor=A.+Zandiehauthor=X.+Zhu&title=The+sequence+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The sequence of the human genome</span></div><div class="casAuthors">Venter, J. Craig; Adams, Mark D.; Myers, Eugene W.; Li, Peter W.; Mural, Richard J.; Sutton, Granger G.; Smith, Hamilton O.; Yandell, Mark; Evans, Cheryl A.; Holt, Robert A.; Gocayne, Jeannine D.; Amanatides, Peter; Ballew, Richard M.; Huson, Daniel H.; Wortman, Jennifer Russo; Zhang, Qing; Kodira, Chinnappa D.; Zheng, Xiangqun H.; Chen, Lin; Skupski, Marian; Subramanian, Gangadharan; Thomas, Paul D.; Zhang, Jinghui; Miklos, George L. Gabor; Nelson, Catherine; Broder, Samuel; Clark, Andrew G.; Nadeau, Joe; McKusick, Victor A.; Zinder, Norton; Levine, Arnold J.; Roberts, Richard J.; Simon, Mel; Slayman, Carolyn; Hunkapiller, Michael; Bolanos, Randall; Delcher, Arthur; Dew, Ian; Fasulo, Daniel; Flanigan, Michael; Florea, Liliana; Halpern, Aaron; Hannenhalli, Sridhar; Kravitz, Saul; Levy, Samuel; Mobarry, Clark; Reiner, Knut; Remington, Karin; Abu-Threide, Jane; Beasle, Ellen; Biddick, Kendra; Bonazzi, Vivien; Brandon, Rhonda; Cargill, Michele; Chandramouliswaran, Ishwar; Charlab, Rosane; Chaturved, Kabir; Deng, Zuoming; Di Francesco, Valentina; Dunn, Patrick; Eilbeck, Karen; Evangelista, Carlos; Gabrielian, Andrei E.; Gan, Weiniu; Ge, Wangmao; Gon, Fangcheng; Gu, Zhiping; Guan, Ping; Heiman, Thomas J.; Higgins, Maureen E.; Ji, Rui-Ru; Ke, Zhaoxi; Ketchum, Karen A.; Lai, Zhongwu; Lei, Yiding; Li, Zhenya; Li, Jiayin; Liang, Yong; Lin, Xiaoying; Lu, Fu; Merkulov, Gennady V.; Milshina, Natalia; Moore, Helen M.; Naik, Ashwinikumar K.; Narayan, Vaibhav A.; Neelam, Beena; Nusskern, Deborah; Rusch, Douglas B.; Salzberg, Steven; Shao, Wei; Shue, Bixiong; Sun, Jingtao; Wang, Zhen Yuan; Wang, Aihui; Wang, Xin; Wang, Jian; Wei, Ming-Hui; Wides, Ron; Xiao, Chunlin; Yan, Chunhua; Yao, Alison; Ye, Jane; Zhan, Ming; Zhan, Weiqing; Zhang, Hongyu; Zhao, Qi; Zheng, Liansheng; Zhong, Fei; Zhong, Wenyan; Zhu, Shiaoping C.; Zhao, Shaying; Gilbert, Dennis; Baumhueter, Suzanna; Spier, Gene; Carter, Christine; Cravchik, Anibal; Woodage, Trevor; Ali, Feroze; An, Huijin; Awe, Aderonke; Baldwin, Danita; Baden, Holly; Barnstead, Mary; Barrow, Lan; Beeson, Karen; Busam, Dana; Carver, Amy; Center, Angela; Cheng, Ming Lai; Curry, Liz; Danaher, Steve; Davenport, Lionel; Desilets, Raymond; Dietz, Susanne; Dodson, Kristina; Doup, Lisa; Ferriera, Steven; Garg, Neha; Hostin, Damon; Houck, Jarrett; Howland, Timothy; Ibegwam, Chinyerer; Johnson, Jeffery; Kalush, Francis; Kline, Lesley; Koduru, Shashi; Love, Amy; Mann, Felecia; May, David; Nelson, Keith; Pfannkoch, Cynthia; Pratts, Eric; Puri, Vinita; Qureshi, Hina; Reardon, Matthew; Rodriguez, Robert; Rogers, Yu-Hui; Romblad, Deanna; Ruhfel, Bob; Scott, Richard; Sitter, Cynthia; Smallwood, Michelle; Stewart, Erin; Strong, Renee; Suh, Ellen; Thomas, Reginald; Tint, Ni Ni; Tse, Sukyee; Vech, Claire; Wang, Gary; Wetter, Jeremy; Williams, Sherita; Williams, Monica; Windsor, Sandra; Winn-Deen, Emily; Wolfe, Keriellen; Zaveri, Jayshree; Zaveri, Karena; Abril, Josep F.; Guigo, Roderic; Campbell, Michael J.; sjolander, Kimmen V.; Karlak, Brian; Kejariwal, Anish; Mi, Huaiyu; Lazareva, Betty; Hatton, Thomas; Narechania, Apurva; Diemer, Karen; Muruganujan, Anushya; Guo, Nan; Sato, Shiji; Bafna, Vineet; Istrail, Sorin; Lippert, Ross; Schwartz, Russell; Walenz, Brian; Yooseph, Shibu; Allen, David; Basu, Anand; Baxendale, James; Block, Louis; Caminha, Marcelo; Carnes-Stine, John; Caulk, Parris; Chiang, Yen-hhui; Coyne, My; Dahike, Carl; Mays, Anne Deslattes; Dombroski, Maria; Donnelly, Michael; Ely, Dale; Esparham, Shiva; Foster, Carl; Gire, Harold; Glanowski, Stephen; Glasser, Kenneth; Glodek, Anna; Gorokhov, Mark; Graham, Ken; Gropman, Barry; Harris, Michael; Heil, Jeremy; Henderson, Scott; Hoover, Jeffrey; Jennings, Donald; Jordan, Catherine; Jordan, James; Kasha, John; Kagan, Leonid; Kraft, Cheryl; Levitsky, Alexander; Lewis, Mark; Liu, Xiangjun; Lopez, John; Ma, Daniel; Majoros, William; McDaniel, Joe; Murphy, Sean; Newman, Matthew; Nguyen, Trung; Nguyen, Ngoc; Nodell, Marc; Pan, Sue; Peck, Jim; Peterson, Marshall; Rowe, William; Sanders, Robert; Scott, John; Simpson, Michael; Smith, Thomas; Sprague, Arlan; Stockwell, Timothy; Turner, Russell; Venter, Eli; Wang, Mei; Wen, Meiyuan; Wu, David; Xia, Ashley; Zandieh, Ali; Zhu, Xiaohong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">5507</span>),
    <span class="NLM_cas:pages">1304-1351</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A 2.91-billion base pair (bp) consensus sequence of the euchromatic portion of the human genome was generated by the whole-genome shotgun sequencing method.  The 14.8-billion bp DNA sequence was generated over 9 mo from 27,271,853 high-quality sequence reads (5.11-fold coverage of the genome) from both ends of plasmid clones made from the DNA of 5 individuals.  Two assembly strategies - a whole-genome assembly and a regional chromosome assembly - were used, each combining sequence data from Celera and the publicly funded genome effort.  The public data, were shredded into 550-bp segments to create a 2.9-fold coverage of those genome regions that had been sequenced, without including biases inherent in the cloning and assembly procedure used by the publicly funded group.  This brought the effective coverage in the assemblies to 8-fold, reducing the no. and size of gaps in. the final assembly over what would be obtained with 5.11-fold coverage.  The 2 assembly strategies yielded very similar results that largely agree with independent mapping data.  The assemblies effectively cover the euchromatic regions of the human chromosomes.  More than 90% of the genome is in scaffold assemblies of ≥100,000 bp, and 25% of the genome is in scaffolds of ≥10 million bp.  Anal. of the genome sequence revealed 26,588 protein-encoding transcripts for which there was strong corroborating evidence and an addnl. ∼12,000 computationally derived genes with mouse matches or other weak supporting evidence.  Although gene-dense clusters are obvious, almost half the genes are dispersed in low G+C sequence sepd. by large tracts of apparently noncoding sequence.  Only 1.1% of the genome is spanned by exons, whereas 24% is in introns, with 75% of the genome being intergenic DNA.  Duplications of segmental blocks, ranging in size up to chromosomal lengths, are abundant throughout the genome and reveal a complex evolutionary history.  Comparative genomic anal. indicates vertebrate expansions of genes assocd. with neuronal function, with tissue-specific developmental regulation, and with the hemostasis and immune systems.  DNA sequence comparisons between the consensus sequence and publicly funded genome data provided locations of 2.1 million single-nucleotide polymorphisms (SNPs).  A random pair of human haploid genomes differed at a rate of 1 bp per 1250 on av., but there was marked heterogeneity in the level of polymorphism across the genome.  Less than 1% of all SNPs resulted in variation in proteins, but the task of detg. which SNPs have functional consequences remains an open challenge.  The genome sequence and addnl. supporting information are available to academic scientists at the Web site www.celera.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYYCo_5N1a7bVg90H21EOLACvtfcHk0ljHPTiydZGeIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtlSgsbo%253D&md5=5ec083b5a91372535a562f7aa12ffb28</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1058040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1058040%26sid%3Dliteratum%253Aachs%26aulast%3DVenter%26aufirst%3DJ.%2BC.%26aulast%3DAdams%26aufirst%3DM.%2BD.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLi%26aufirst%3DP.%2BW.%26aulast%3DMural%26aufirst%3DR.%2BJ.%26aulast%3DSutton%26aufirst%3DG.%2BG.%26aulast%3DSmith%26aufirst%3DH.%2BO.%26aulast%3DYandell%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DC.%2BA.%26aulast%3DHolt%26aufirst%3DR.%2BA.%26aulast%3DGocayne%26aufirst%3DJ.%2BD.%26aulast%3DAmanatides%26aufirst%3DP.%26aulast%3DBallew%26aufirst%3DR.%2BM.%26aulast%3DHuson%26aufirst%3DD.%2BH.%26aulast%3DWortman%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKodira%26aufirst%3DC.%2BD.%26aulast%3DZheng%26aufirst%3DX.%2BH.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSkupski%26aufirst%3DM.%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DThomas%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGabor%2BMiklos%26aufirst%3DG.%2BL.%26aulast%3DNelson%26aufirst%3DC.%26aulast%3DBroder%26aufirst%3DS.%26aulast%3DClark%26aufirst%3DA.%2BG.%26aulast%3DNadeau%26aufirst%3DJ.%26aulast%3DMcKusick%26aufirst%3DV.%2BA.%26aulast%3DZinder%26aufirst%3DN.%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26aulast%3DRoberts%26aufirst%3DR.%2BJ.%26aulast%3DSimon%26aufirst%3DM.%26aulast%3DSlayman%26aufirst%3DC.%26aulast%3DHunkapiller%26aufirst%3DM.%26aulast%3DBolanos%26aufirst%3DR.%26aulast%3DDelcher%26aufirst%3DA.%26aulast%3DDew%26aufirst%3DI.%26aulast%3DFasulo%26aufirst%3DD.%26aulast%3DFlanigan%26aufirst%3DM.%26aulast%3DFlorea%26aufirst%3DL.%26aulast%3DHalpern%26aufirst%3DA.%26aulast%3DHannenhalli%26aufirst%3DS.%26aulast%3DKravitz%26aufirst%3DS.%26aulast%3DLevy%26aufirst%3DS.%26aulast%3DMobarry%26aufirst%3DC.%26aulast%3DReinert%26aufirst%3DK.%26aulast%3DRemington%26aufirst%3DK.%26aulast%3DAbu-Threideh%26aufirst%3DJ.%26aulast%3DBeasley%26aufirst%3DE.%26aulast%3DBiddick%26aufirst%3DK.%26aulast%3DBonazzi%26aufirst%3DV.%26aulast%3DBrandon%26aufirst%3DR.%26aulast%3DCargill%26aufirst%3DM.%26aulast%3DChandramouliswaran%26aufirst%3DI.%26aulast%3DCharlab%26aufirst%3DR.%26aulast%3DChaturvedi%26aufirst%3DK.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDi%2BFrancesco%26aufirst%3DV.%26aulast%3DDunn%26aufirst%3DP.%26aulast%3DEilbeck%26aufirst%3DK.%26aulast%3DEvangelista%26aufirst%3DC.%26aulast%3DGabrielian%26aufirst%3DA.%2BE.%26aulast%3DGan%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DW.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DHeiman%26aufirst%3DT.%2BJ.%26aulast%3DHiggins%26aufirst%3DM.%2BE.%26aulast%3DJi%26aufirst%3DR.%2BR.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DKetchum%26aufirst%3DK.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DMerkulov%26aufirst%3DG.%2BV.%26aulast%3DMilshina%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DH.%2BM.%26aulast%3DNaik%26aufirst%3DA.%2BK.%26aulast%3DNarayan%26aufirst%3DV.%2BA.%26aulast%3DNeelam%26aufirst%3DB.%26aulast%3DNusskern%26aufirst%3DD.%26aulast%3DRusch%26aufirst%3DD.%2BB.%26aulast%3DSalzberg%26aufirst%3DS.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DShue%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DWides%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DZhong%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DBaumhueter%26aufirst%3DS.%26aulast%3DSpier%26aufirst%3DG.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DCravchik%26aufirst%3DA.%26aulast%3DWoodage%26aufirst%3DT.%26aulast%3DAli%26aufirst%3DF.%26aulast%3DAn%26aufirst%3DH.%26aulast%3DAwe%26aufirst%3DA.%26aulast%3DBaldwin%26aufirst%3DD.%26aulast%3DBaden%26aufirst%3DH.%26aulast%3DBarnstead%26aufirst%3DM.%26aulast%3DBarrow%26aufirst%3DI.%26aulast%3DBeeson%26aufirst%3DK.%26aulast%3DBusam%26aufirst%3DD.%26aulast%3DCarver%26aufirst%3DA.%26aulast%3DCenter%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DM.%2BL.%26aulast%3DCurry%26aufirst%3DL.%26aulast%3DDanaher%26aufirst%3DS.%26aulast%3DDavenport%26aufirst%3DL.%26aulast%3DDesilets%26aufirst%3DR.%26aulast%3DDietz%26aufirst%3DS.%26aulast%3DDodson%26aufirst%3DK.%26aulast%3DDoup%26aufirst%3DL.%26aulast%3DFerriera%26aufirst%3DS.%26aulast%3DGarg%26aufirst%3DN.%26aulast%3DGluecksmann%26aufirst%3DA.%26aulast%3DHart%26aufirst%3DB.%26aulast%3DHaynes%26aufirst%3DJ.%26aulast%3DHaynes%26aufirst%3DC.%26aulast%3DHeiner%26aufirst%3DC.%26aulast%3DHladun%26aufirst%3DS.%26aulast%3DHostin%26aufirst%3DD.%26aulast%3DHouck%26aufirst%3DJ.%26aulast%3DHowland%26aufirst%3DT.%26aulast%3DIbegwam%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DKalush%26aufirst%3DF.%26aulast%3DKline%26aufirst%3DL.%26aulast%3DKoduru%26aufirst%3DS.%26aulast%3DLove%26aufirst%3DA.%26aulast%3DMann%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DD.%26aulast%3DMcCawley%26aufirst%3DS.%26aulast%3DMcIntosh%26aufirst%3DT.%26aulast%3DMcMullen%26aufirst%3DI.%26aulast%3DMoy%26aufirst%3DM.%26aulast%3DMoy%26aufirst%3DL.%26aulast%3DMurphy%26aufirst%3DB.%26aulast%3DNelson%26aufirst%3DK.%26aulast%3DPfannkoch%26aufirst%3DC.%26aulast%3DPratts%26aufirst%3DE.%26aulast%3DPuri%26aufirst%3DV.%26aulast%3DQureshi%26aufirst%3DH.%26aulast%3DReardon%26aufirst%3DM.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DRogers%26aufirst%3DY.%2BH.%26aulast%3DRomblad%26aufirst%3DD.%26aulast%3DRuhfel%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DR.%26aulast%3DSitter%26aufirst%3DC.%26aulast%3DSmallwood%26aufirst%3DM.%26aulast%3DStewart%26aufirst%3DE.%26aulast%3DStrong%26aufirst%3DR.%26aulast%3DSuh%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DR.%26aulast%3DTint%26aufirst%3DN.%2BN.%26aulast%3DTse%26aufirst%3DS.%26aulast%3DVech%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWetter%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DS.%26aulast%3DWinn-Deen%26aufirst%3DE.%26aulast%3DWolfe%26aufirst%3DK.%26aulast%3DZaveri%26aufirst%3DJ.%26aulast%3DZaveri%26aufirst%3DK.%26aulast%3DAbril%26aufirst%3DJ.%2BF.%26aulast%3DGuigo%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DM.%2BJ.%26aulast%3DSjolander%26aufirst%3DK.%2BV.%26aulast%3DKarlak%26aufirst%3DB.%26aulast%3DKejariwal%26aufirst%3DA.%26aulast%3DMi%26aufirst%3DH.%26aulast%3DLazareva%26aufirst%3DB.%26aulast%3DHatton%26aufirst%3DT.%26aulast%3DNarechania%26aufirst%3DA.%26aulast%3DDiemer%26aufirst%3DK.%26aulast%3DMuruganujan%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DBafna%26aufirst%3DV.%26aulast%3DIstrail%26aufirst%3DS.%26aulast%3DLippert%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DR.%26aulast%3DWalenz%26aufirst%3DB.%26aulast%3DYooseph%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DD.%26aulast%3DBasu%26aufirst%3DA.%26aulast%3DBaxendale%26aufirst%3DJ.%26aulast%3DBlick%26aufirst%3DL.%26aulast%3DCaminha%26aufirst%3DM.%26aulast%3DCarnes-Stine%26aufirst%3DJ.%26aulast%3DCaulk%26aufirst%3DP.%26aulast%3DChiang%26aufirst%3DY.%2BH.%26aulast%3DCoyne%26aufirst%3DM.%26aulast%3DDahlke%26aufirst%3DC.%26aulast%3DMays%26aufirst%3DA.%26aulast%3DDombroski%26aufirst%3DM.%26aulast%3DDonnelly%26aufirst%3DM.%26aulast%3DEly%26aufirst%3DD.%26aulast%3DEsparham%26aufirst%3DS.%26aulast%3DFosler%26aufirst%3DC.%26aulast%3DGire%26aufirst%3DH.%26aulast%3DGlanowski%26aufirst%3DS.%26aulast%3DGlasser%26aufirst%3DK.%26aulast%3DGlodek%26aufirst%3DA.%26aulast%3DGorokhov%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DK.%26aulast%3DGropman%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DM.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DHenderson%26aufirst%3DS.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DJennings%26aufirst%3DD.%26aulast%3DJordan%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DJ.%26aulast%3DKasha%26aufirst%3DJ.%26aulast%3DKagan%26aufirst%3DL.%26aulast%3DKraft%26aufirst%3DC.%26aulast%3DLevitsky%26aufirst%3DA.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLopez%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DMajoros%26aufirst%3DW.%26aulast%3DMcDaniel%26aufirst%3DJ.%26aulast%3DMurphy%26aufirst%3DS.%26aulast%3DNewman%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DNodell%26aufirst%3DM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DPeck%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DM.%26aulast%3DRowe%26aufirst%3DW.%26aulast%3DSanders%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DJ.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DSprague%26aufirst%3DA.%26aulast%3DStockwell%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DVenter%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWen%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DXia%26aufirst%3DA.%26aulast%3DZandieh%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DThe%2520sequence%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2001%26volume%3D291%26spage%3D1304%26epage%3D1351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Kim, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, J. M.</span><span> </span><span class="NLM_article-title">A 2.2 Å resolution crystal structure of a designed zinc finger protein bound to DNA</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">940</span><span class="NLM_x">–</span> <span class="NLM_lpage">945</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=940-945&author=C.+A.+Kimauthor=J.+M.+Berg&title=A+2.2+%C3%85+resolution+crystal+structure+of+a+designed+zinc+finger+protein+bound+to+DNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DC.%2BA.%26aulast%3DBerg%26aufirst%3DJ.%2BM.%26atitle%3DA%25202.2%2520%25C3%2585%2520resolution%2520crystal%2520structure%2520of%2520a%2520designed%2520zinc%2520finger%2520protein%2520bound%2520to%2520DNA%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D940%26epage%3D945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Simpson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cram, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czolij, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crossley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, J. P.</span><span> </span><span class="NLM_article-title">CCHX zinc finger derivatives retain the ability to bind Zn(II) and mediate protein-DNA interactions</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">28011</span><span class="NLM_x">–</span> <span class="NLM_lpage">28018</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=28011-28018&author=R.+J.+Simpsonauthor=E.+D.+Cramauthor=R.+Czolijauthor=J.+M.+Matthewsauthor=M.+Crossleyauthor=J.+P.+Mackay&title=CCHX+zinc+finger+derivatives+retain+the+ability+to+bind+Zn%28II%29+and+mediate+protein-DNA+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSimpson%26aufirst%3DR.%2BJ.%26aulast%3DCram%26aufirst%3DE.%2BD.%26aulast%3DCzolij%26aufirst%3DR.%26aulast%3DMatthews%26aufirst%3DJ.%2BM.%26aulast%3DCrossley%26aufirst%3DM.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26atitle%3DCCHX%2520zinc%2520finger%2520derivatives%2520retain%2520the%2520ability%2520to%2520bind%2520Zn%2528II%2529%2520and%2520mediate%2520protein-DNA%2520interactions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D28011%26epage%3D28018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Pabo, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, R. A.</span><span> </span><span class="NLM_article-title">Design and selection of novel Cys2His2 zinc finger proteins</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1146%2Fannurev.biochem.70.1.313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=11395410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2001&pages=313-340&author=C.+O.+Paboauthor=E.+Peisachauthor=R.+A.+Grant&title=Design+and+selection+of+novel+Cys2His2+zinc+finger+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Design and selection of novel Cys2His2 zinc finger proteins</span></div><div class="casAuthors">Pabo, Carl O.; Peisach, Ezra; Grant, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">313-340</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with 141 refs.  Cys2His2 zinc finger proteins offer a stable and versatile framework for the design of proteins that recognize desired target sites on double-stranded DNA.  Individual fingers from these proteins have a simple ββα structure that folds around a central zinc ion, and tandem sets of fingers can contact neighboring subsites of 3-4 base pairs along the major groove of the DNA.  Although there is no simple, general code for zinc finger-DNA recognition, selection strategies have been developed that allow these proteins to be targeted to almost any desired site on double-stranded DNA.  The affinity and specificity of these new proteins can also be improved by linking more fingers together or by designing proteins that bind as dimers and thus recognize an extended site.  These new proteins can then be modified by adding other domains-for activation or repression of transcription, for DNA cleavage, or for other activities.  Such designer transcription factors and other new proteins will have important applications in biomedical research and in gene therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreBI78my_xQbVg90H21EOLACvtfcHk0lj_JuWKuzTckg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7g%253D&md5=2dac8b200d0c0710e8c23d44bf6dd9c3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.70.1.313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.70.1.313%26sid%3Dliteratum%253Aachs%26aulast%3DPabo%26aufirst%3DC.%2BO.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DGrant%26aufirst%3DR.%2BA.%26atitle%3DDesign%2520and%2520selection%2520of%2520novel%2520Cys2His2%2520zinc%2520finger%2520proteins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2001%26volume%3D70%26spage%3D313%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">van Vliet, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crossley, M.</span><span> </span><span class="NLM_article-title">Human Kruppel-like factor 8: a CACCC-box binding protein that associates with CtBP and represses transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1955</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1955-1962&author=J.+van+Vlietauthor=J.+Turnerauthor=M.+Crossley&title=Human+Kruppel-like+factor+8%3A+a+CACCC-box+binding+protein+that+associates+with+CtBP+and+represses+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BVliet%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DCrossley%26aufirst%3DM.%26atitle%3DHuman%2520Kruppel-like%2520factor%25208%253A%2520a%2520CACCC-box%2520binding%2520protein%2520that%2520associates%2520with%2520CtBP%2520and%2520represses%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D1955%26epage%3D1962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Role of Kruppel-like factors in leukocyte development, function, and disease</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">4404</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=4404-4414&author=Z.+Caoauthor=X.+Sunauthor=B.+Icliauthor=A.+K.+Waraauthor=M.+W.+Feinberg&title=Role+of+Kruppel-like+factors+in+leukocyte+development%2C+function%2C+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DRole%2520of%2520Kruppel-like%2520factors%2520in%2520leukocyte%2520development%252C%2520function%252C%2520and%2520disease%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D4404%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Gallimore, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godkin, A.</span><span> </span><span class="NLM_article-title">Regulatory T cells and tumour immunity—observations in mice and men</span> <span class="citation_source-journal">Immunology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">163</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2008&pages=157-163&author=A.+Gallimoreauthor=A.+Godkin&title=Regulatory+T+cells+and+tumour+immunity%E2%80%94observations+in+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGallimore%26aufirst%3DA.%26aulast%3DGodkin%26aufirst%3DA.%26atitle%3DRegulatory%2520T%2520cells%2520and%2520tumour%2520immunity%25E2%2580%2594observations%2520in%2520mice%2520and%2520men%26jtitle%3DImmunology%26date%3D2008%26volume%3D123%26spage%3D157%26epage%3D163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Clarke, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betts, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plant, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Shanawany, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrop, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torkington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, B. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallimore, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godkin, A. J.</span><span> </span><span class="NLM_article-title">CD4+CD25+FOXP3+ regulatory T cells suppress anti-tumor immune responses in patients with colorectal cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">e129</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=e129&author=S.+L.+Clarkeauthor=G.+J.+Bettsauthor=A.+Plantauthor=K.+L.+Wrightauthor=T.+M.+El-Shanawanyauthor=R.+Harropauthor=J.+Torkingtonauthor=B.+I.+Reesauthor=G.+T.+Williamsauthor=A.+M.+Gallimoreauthor=A.+J.+Godkin&title=CD4%2BCD25%2BFOXP3%2B+regulatory+T+cells+suppress+anti-tumor+immune+responses+in+patients+with+colorectal+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BL.%26aulast%3DBetts%26aufirst%3DG.%2BJ.%26aulast%3DPlant%26aufirst%3DA.%26aulast%3DWright%26aufirst%3DK.%2BL.%26aulast%3DEl-Shanawany%26aufirst%3DT.%2BM.%26aulast%3DHarrop%26aufirst%3DR.%26aulast%3DTorkington%26aufirst%3DJ.%26aulast%3DRees%26aufirst%3DB.%2BI.%26aulast%3DWilliams%26aufirst%3DG.%2BT.%26aulast%3DGallimore%26aufirst%3DA.%2BM.%26aulast%3DGodkin%26aufirst%3DA.%2BJ.%26atitle%3DCD4%252BCD25%252BFOXP3%252B%2520regulatory%2520T%2520cells%2520suppress%2520anti-tumor%2520immune%2520responses%2520in%2520patients%2520with%2520colorectal%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2006%26volume%3D1%26spage%3De129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3(+) regulatory T cells, immune stimulation and host defence against infection</span> <span class="citation_source-journal">Immunology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2012&pages=1-10&author=J.+H.+Roweauthor=J.+M.+Erteltauthor=S.+S.+Way&title=Foxp3%28%2B%29+regulatory+T+cells%2C+immune+stimulation+and+host+defence+against+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%2528%252B%2529%2520regulatory%2520T%2520cells%252C%2520immune%2520stimulation%2520and%2520host%2520defence%2520against%2520infection%26jtitle%3DImmunology%26date%3D2012%26volume%3D136%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrar, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3(+) regulatory T cell expansion required for sustaining pregnancy compromises host defense against prenatal bacterial pathogens</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.chom.2011.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=21767812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVynu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=54-64&author=J.+H.+Roweauthor=J.+M.+Erteltauthor=M.+N.+Aguileraauthor=M.+A.+Farrarauthor=S.+S.+Way&title=Foxp3%28%2B%29+regulatory+T+cell+expansion+required+for+sustaining+pregnancy+compromises+host+defense+against+prenatal+bacterial+pathogens"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Foxp3+ Regulatory T Cell Expansion Required for Sustaining Pregnancy Compromises Host Defense against Prenatal Bacterial Pathogens</span></div><div class="casAuthors">Rowe, Jared H.; Ertelt, James M.; Aguilera, Marijo N.; Farrar, Michael A.; Way, Sing Sing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-64</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Although pregnancy confers unique susceptibility to infection, the pregnancy-assocd. immune defects that erode host defense remain largely undefined.  Herein, we demonstrate that expansion of immune-suppressive Foxp3+ regulatory T cells (Tregs) which occurs physiol. during pregnancy or when exptl. induced in transgenic mice caused enhanced susceptibility to prenatal pathogens including Listeria and Salmonella species.  Reciprocally, infection susceptibility was uniformly reduced with Treg ablation.  Importantly however, the sustained expansion of maternal Tregs was essential for maintaining immune tolerance to the developing fetus because even partial transient ablation of Foxp3-expressing cells fractured maternal tolerance to fetal antigen and triggered fetal resorption.  Interestingly, Foxp3 cell-intrinsic defects in the immune-suppressive cytokine IL-10 alone were sufficient to override Treg-mediated infection susceptibility, while IL-10 was nonessential for sustaining pregnancy.  Thus, maternal Treg expansion required for sustaining pregnancy creates naturally occurring holes in host defense that confer prenatal infection susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3KvmgCB0InrVg90H21EOLACvtfcHk0lgGlHt77Wnfsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVynu7k%253D&md5=d0897d97f51d0308961334587b9b3040</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2011.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2011.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DAguilera%26aufirst%3DM.%2BN.%26aulast%3DFarrar%26aufirst%3DM.%2BA.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%2528%252B%2529%2520regulatory%2520T%2520cell%2520expansion%2520required%2520for%2520sustaining%2520pregnancy%2520compromises%2520host%2520defense%2520against%2520prenatal%2520bacterial%2520pathogens%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2011%26volume%3D10%26spage%3D54%26epage%3D64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roychowdhury, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3+ regulatory T cells impede the priming of protective CD8+ T cells</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">2569</span><span class="NLM_x">–</span> <span class="NLM_lpage">2577</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2011&pages=2569-2577&author=J.+M.+Erteltauthor=J.+H.+Roweauthor=M.+A.+Myszauthor=C.+Singhauthor=M.+Roychowdhuryauthor=M.+N.+Aguileraauthor=S.+S.+Way&title=Foxp3%2B+regulatory+T+cells+impede+the+priming+of+protective+CD8%2B+T+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DMysz%26aufirst%3DM.%2BA.%26aulast%3DSingh%26aufirst%3DC.%26aulast%3DRoychowdhury%26aufirst%3DM.%26aulast%3DAguilera%26aufirst%3DM.%2BN.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%252B%2520regulatory%2520T%2520cells%2520impede%2520the%2520priming%2520of%2520protective%2520CD8%252B%2520T%2520cells%26jtitle%3DJ.%2520Immunol.%26date%3D2011%26volume%3D187%26spage%3D2569%26epage%3D2577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stapleton, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kretschmer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolou, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Boehmer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansson, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spelsberg, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Libby, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Kruppel-like factor KLF10 targets transforming growth factor-beta1 to regulate CD4(+)CD25(−) T cells and T regulatory cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24914</span><span class="NLM_x">–</span> <span class="NLM_lpage">24924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24914-24924&author=Z.+Caoauthor=A.+K.+Waraauthor=B.+Icliauthor=X.+Sunauthor=R.+R.+Packardauthor=F.+Esenauthor=C.+J.+Stapletonauthor=M.+Subramaniamauthor=K.+Kretschmerauthor=I.+Apostolouauthor=H.+von+Boehmerauthor=G.+K.+Hanssonauthor=T.+C.+Spelsbergauthor=P.+Libbyauthor=M.+W.+Feinberg&title=Kruppel-like+factor+KLF10+targets+transforming+growth+factor-beta1+to+regulate+CD4%28%2B%29CD25%28%E2%88%92%29+T+cells+and+T+regulatory+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DPackard%26aufirst%3DR.%2BR.%26aulast%3DEsen%26aufirst%3DF.%26aulast%3DStapleton%26aufirst%3DC.%2BJ.%26aulast%3DSubramaniam%26aufirst%3DM.%26aulast%3DKretschmer%26aufirst%3DK.%26aulast%3DApostolou%26aufirst%3DI.%26aulast%3Dvon%2BBoehmer%26aufirst%3DH.%26aulast%3DHansson%26aufirst%3DG.%2BK.%26aulast%3DSpelsberg%26aufirst%3DT.%2BC.%26aulast%3DLibby%26aufirst%3DP.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DKruppel-like%2520factor%2520KLF10%2520targets%2520transforming%2520growth%2520factor-beta1%2520to%2520regulate%2520CD4%2528%252B%2529CD25%2528%25E2%2588%2592%2529%2520T%2520cells%2520and%2520T%2520regulatory%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24914%26epage%3D24924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Venuprasad, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elly, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spelsberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span> </span><span class="NLM_article-title">The E3 ubiquitin ligase Itch regulates expression of transcription factor Foxp3 and airway inflammation by enhancing the function of transcription factor TIEG1</span> <span class="citation_source-journal">Nat. Immunol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni1564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=18278048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVOjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=245-253&author=K.+Venuprasadauthor=H.+Huangauthor=Y.+Haradaauthor=C.+Ellyauthor=M.+Subramaniamauthor=T.+Spelsbergauthor=J.+Suauthor=Y.+C.+Liu&title=The+E3+ubiquitin+ligase+Itch+regulates+expression+of+transcription+factor+Foxp3+and+airway+inflammation+by+enhancing+the+function+of+transcription+factor+TIEG1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The E3 ubiquitin ligase Itch regulates expression of transcription factor Foxp3 and airway inflammation by enhancing the function of transcription factor TIEG1</span></div><div class="casAuthors">Venuprasad, K.; Huang, Haining; Harada, Yousuke; Elly, Chris; Subramaniam, Malayannan; Spelsberg, Thomas; Su, Jin; Liu, Yun-Cai</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-253</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transforming growth factor-β (TGF-β) signaling in naive T cells induces expression of the transcription factor Foxp3, a master regulator of regulatory T cells (Treg).  However, the mol. mechanisms leading to Foxp3 induction remain unclear.  Here the authors show that Itch-/- T cells were resistant to TGF-β treatment and had less Foxp3 expression.  The E3 ubiquitin ligase Itch assocd. with and promoted conjugation of ubiquitin to the transcription factor TIEG1.  Itch cooperated with TIEG1 to induce Foxp3 expression, which was reversed by TIEG1 deficiency.  Functionally, "TGF-β-converted" Treg cells generated from TIEG1-deficient mice were unable to suppress airway inflammation in vivo.  Apparently, TIEG and Itch contribute to a ubiquitin-dependent non-proteolytic pathway that regulates inducible Foxp3 expression and the control of allergic responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0XoQ1p1xnz7Vg90H21EOLACvtfcHk0lh3JmjHic4btQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVOjtrs%253D&md5=d31a7410efd14d29a4934426e1d06480</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fni1564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni1564%26sid%3Dliteratum%253Aachs%26aulast%3DVenuprasad%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHarada%26aufirst%3DY.%26aulast%3DElly%26aufirst%3DC.%26aulast%3DSubramaniam%26aufirst%3DM.%26aulast%3DSpelsberg%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26atitle%3DThe%2520E3%2520ubiquitin%2520ligase%2520Itch%2520regulates%2520expression%2520of%2520transcription%2520factor%2520Foxp3%2520and%2520airway%2520inflammation%2520by%2520enhancing%2520the%2520function%2520of%2520transcription%2520factor%2520TIEG1%26jtitle%3DNat.%2520Immunol%26date%3D2008%26volume%3D9%26spage%3D245%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Halgren, T.</span><span> </span><span class="NLM_article-title">New method for fast and accurate binding-site identification and analysis</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+method+for+fast+and+accurate+binding-site+identification+and+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0lh3JmjHic4btQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520method%2520for%2520fast%2520and%2520accurate%2520binding-site%2520identification%2520and%2520analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">389</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lgqSsrVUtrKMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Brickmann, J.; Goetze, T.; Heiden, W.; Moeckel, G.; Reiling, S.; Vollhardt, H.; Zachmann, C.-D.</span><span> </span><span class="NLM_article-title">Interactive Visualization of Molecular Scenarios with the MOLCAD/SYBYL</span>. In  <span class="citation_source-book">Data Visualization in Molecular Science: Tools for Insight and Innovation</span>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_publisher-name">Addison-Wesley</span>: <span class="NLM_publisher-loc">Reading, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">1994</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=J.+Brickmann&author=T.+Goetze&author=W.+Heiden&author=G.+Moeckel&author=S.+Reiling&author=H.+Vollhardt&author=C.-D.+Zachmann&title=Data+Visualization+in+Molecular+Science%3A+Tools+for+Insight+and+Innovation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBrickmann%26aufirst%3DJ.%26atitle%3DInteractive%2520Visualization%2520of%2520Molecular%2520Scenarios%2520with%2520the%2520MOLCAD%252FSYBYL%26btitle%3DData%2520Visualization%2520in%2520Molecular%2520Science%253A%2520Tools%2520for%2520Insight%2520and%2520Innovation%26pub%3DAddison-Wesley%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Soga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirayama, N.</span><span> </span><span class="NLM_article-title">Use of amino acid composition to predict ligand-binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6002202" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=400-406&author=S.+Sogaauthor=H.+Shiraiauthor=M.+Koboriauthor=N.+Hirayama&title=Use+of+amino+acid+composition+to+predict+ligand-binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Amino Acid Composition to Predict Ligand-Binding Sites</span></div><div class="casAuthors">Soga, Shinji; Shirai, Hiroki; Kobori, Masato; Hirayama, Noriaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">400-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for predicting the binding sites for druglike compds. on the surface of proteins was developed on the basis of the specific amino acid compn. obsd. at the ligand-binding sites of ligand-protein complexes detd. by x-ray anal.  A profile representing the preference of each of the 20 std. amino acids at the binding sites of druglike mols. was obtained for a small set of high-quality complex structures.  An index termed propensity for ligand binding (PLB) was created from these profiles.  The PLB index was used to predict the propensity of binding for 804 ligands at all potential binding sites on the proteins whose structures were detd. by x-ray anal.  If the sites with the first 2 highest PLB indexes are taken into consideration, the successfully predicted sites reached a high percentage of 86.  The PLB prediction is relatively simple, but the validation study showed that it is both fast and accurate to detect ligand-binding sites, esp. the binding sites of druglike mols.  Therefore, the PLB index can be used to predict the ligand-binding sites of uncharacterized protein structures and also to identify novel drug-binding sites of known drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptxKvbII_oZbVg90H21EOLACvtfcHk0lgqSsrVUtrKMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D&md5=53bddb67710c84f45c99173b0d08d9ce</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci6002202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002202%26sid%3Dliteratum%253Aachs%26aulast%3DSoga%26aufirst%3DS.%26aulast%3DShirai%26aufirst%3DH.%26aulast%3DKobori%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DN.%26atitle%3DUse%2520of%2520amino%2520acid%2520composition%2520to%2520predict%2520ligand-binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D400%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhB4jaPyA5WYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhB4jaPyA5WYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision Glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+Glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lhB4jaPyA5WYw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520Glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020406h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=499-511&author=A.+N.+Jain&title=Surflex%3A+fully+automatic+flexible+molecular+docking+using+a+molecular+similarity-based+search+engine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Surflex is a fully automatic flexible mol. docking algorithm that combines the scoring function from the Hammerhead docking system with a search engine that relies on a surface-based mol. similarity method as a means to rapidly generate suitable putative poses for mol. fragments.  Results are presented evaluating reliability and accuracy of dockings compared with crystallog. exptl. results on 81 protein/ligand pairs of substantial structural diversity.  In over 80% of the complexes, Surflex's highest scoring docked pose was within 2.5 Å root-mean-square deviation (rmsd), with over 90% of the complexes having one of the top ranked poses within 2.5 Å rmsd.  Results are also presented assessing Surflex's utility as a screening tool on two protein targets (thymidine kinase and estrogen receptor) using data sets on which competing methods were run.  Performance of Surflex was significantly better, with true pos. rates of greater than 80% at false pos. rates of less than 1%.  Docking time was roughly linear in no. of rotatable bonds, beginning with a few seconds for rigid mols. and adding approx. 10 s per rotatable bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawn5WHbx5p7Vg90H21EOLACvtfcHk0lgIruAjkTqrhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D&md5=67a08aafeadf69101d56b2f60a922359</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm020406h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020406h%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex%253A%2520fully%2520automatic%2520flexible%2520molecular%2520docking%2520using%2520a%2520molecular%2520similarity-based%2520search%2520engine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D499%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A fast flexible docking method using an incremental construction algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1006%2Fjmbi.1996.0477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=8780787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=470-489&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauerauthor=G.+Klebe&title=A+fast+flexible+docking+method+using+an+incremental+construction+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A fast flexible docking method using an incremental construction algorithm</span></div><div class="casAuthors">Rarey, Matthias; Kramer, Bernd; Lengauer, Thomas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-489</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We present an automatic method for docking org. ligands into protein binding sites.  The method can be used in the design process of specific protein ligands.  It combines an appropriate model of the physico-chem. properties of the docked mols. with efficient methods for sampling the conformational space of the ligand.  If the ligand is flexible, it can adopt a large variety of different conformations.  Each such min. in conformational space presents a potential candidate for the conformation of the ligand in the complexed state.  Our docking method samples the conformation space of the ligand on the basis of a discrete model and uses a tree-search technique for placing the ligand incrementally into the active site.  For placing the first fragment of the ligand into the protein, we use hashing techniques adapted from computer vision.  The incremental construction algorithm is based on a greedy strategy combined with efficient methods for overlap detection and for the search of new interactions.  We present results on 19 complexes of which the binding geometry has been crystallog. detd.  All considered ligands are docked in at most three minutes on a current workstation.  The exptl. obsd. binding mode of the ligand is reproduced with 0.5 to 1.2 Å rms deviation.  It is almost always found among the highest-ranking conformations computed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx9N2sFXkOfLVg90H21EOLACvtfcHk0lgIruAjkTqrhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D&md5=8cd8f384906ed90e894d118399e4dc4a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0477%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520fast%2520flexible%2520docking%2520method%2520using%2520an%2520incremental%2520construction%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D470%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Sakaguchi, S.</span><span> </span><span class="NLM_article-title">Naturally arising Foxp3-expressing CD25+CD4+ regulatory T cells in immunological tolerance to self and non-self</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni1178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15785760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVyitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=345-352&author=S.+Sakaguchi&title=Naturally+arising+Foxp3-expressing+CD25%2BCD4%2B+regulatory+T+cells+in+immunological+tolerance+to+self+and+non-self"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Naturally arising Foxp3-expressing CD25+CD4+ regulatory T cells in immunological tolerance to self and non-self</span></div><div class="casAuthors">Sakaguchi, Shimon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">345-352</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naturally arising CD25+CD4+ regulatory T cells actively maintain immunol. self-tolerance.  Deficiency in or dysfunction of these cells can be a cause of autoimmune disease.  A redn. in their no. or function can also elicit tumor immunity, whereas their antigen-specific population expansion can establish transplantation tolerance.  They are therefore a good target for designing ways to induce or abrogate immunol. tolerance to self and non-self antigens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo2HWNwsl-erVg90H21EOLACvtfcHk0lgJPlGRsyBdgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVyitLY%253D&md5=d873618a9666d7d03e457ce828e85f37</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fni1178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni1178%26sid%3Dliteratum%253Aachs%26aulast%3DSakaguchi%26aufirst%3DS.%26atitle%3DNaturally%2520arising%2520Foxp3-expressing%2520CD25%252BCD4%252B%2520regulatory%2520T%2520cells%2520in%2520immunological%2520tolerance%2520to%2520self%2520and%2520non-self%26jtitle%3DNat.%2520Immunol.%26date%3D2005%26volume%3D6%26spage%3D345%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Bluestone, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Q.</span><span> </span><span class="NLM_article-title">How do CD4+CD25+ regulatory T cells control autoimmunity?</span> <span class="citation_source-journal">Curr. Opin Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.coi.2005.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16209918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=638-642&author=J.+A.+Bluestoneauthor=Q.+Tang&title=How+do+CD4%2BCD25%2B+regulatory+T+cells+control+autoimmunity%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">How do CD4+CD25+ regulatory T cells control autoimmunity?</span></div><div class="casAuthors">Bluestone, Jeffrey A.; Tang, Qizhi</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">638-642</span>CODEN:
                <span class="NLM_cas:coden">COPIEL</span>;
        ISSN:<span class="NLM_cas:issn">0952-7915</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Any scientist opening up an immunol. journal today will observe immediately that suppressor T cells, renamed regulatory T cells' (Tregs) have become a central concept in the immunol. lexicon.  Hundreds of Treg publications over the past few years have validated the existence of this unique T cell lineage armed with an ability to regulate autoimmunity.  The CD4+CD25+Foxp3+ Treg subset develops in the thymus, can be induced in the periphery during the course of normal immune responses and utilizes a T cell repertoire skewed towards autoantigens.  Despite these advances, however, there is still controversy over their mechanism of action.  This confusion stems from the differences obsd. in in vitro vs. in vivo studies.  Recent in vivo analyses support a model in which Tregs directly or indirectly alter the activation and differentiation of pathogenic T cells through an effect on antigen presenting cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddm9vwk36N7Vg90H21EOLACvtfcHk0lgJPlGRsyBdgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsb3F&md5=0dd48e94a65efcc18508e0a51e9c1127</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.coi.2005.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coi.2005.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DBluestone%26aufirst%3DJ.%2BA.%26aulast%3DTang%26aufirst%3DQ.%26atitle%3DHow%2520do%2520CD4%252BCD25%252B%2520regulatory%2520T%2520cells%2520control%2520autoimmunity%253F%26jtitle%3DCurr.%2520Opin%2520Immunol.%26date%3D2005%26volume%3D17%26spage%3D638%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Fontenot, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, A. Y.</span><span> </span><span class="NLM_article-title">Foxp3 programs the development and function of CD4+CD25+ regulatory T cells</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=12612578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlyqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=330-336&author=J.+D.+Fontenotauthor=M.+A.+Gavinauthor=A.+Y.+Rudensky&title=Foxp3+programs+the+development+and+function+of+CD4%2BCD25%2B+regulatory+T+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Foxp3 programs the development and function of CD4+CD25+ regulatory T cells</span></div><div class="casAuthors">Fontenot, Jason D.; Gavin, Marc A.; Rudensky, Alexander Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">330-336</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD4+CD25+ regulatory T cells are essential for the active suppression of autoimmunity.  Here the authors report that the forkhead transcription factor Foxp3 is specifically expressed in CD4+CD25+ regulatory T cells and is required for their development.  The lethal autoimmune syndrome obsd. in Foxp3-mutant scurfy mice and Foxp3-null mice results from a CD4+CD25+ regulatory T cell deficiency and not from a cell-intrinsic defect of CD4+CD25- T cells.  CD4+CD25+ regulatory T cells rescue disease development and preferentially expand when transferred into neonatal Foxp3-deficient mice.  Furthermore, ectopic expression of Foxp3 confers suppressor function on peripheral CD4+CD25- T cells.  Thus, Foxp3 is a crit. regulator of CD4+CD25+ regulatory T cell development and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriHBlgR28euLVg90H21EOLACvtfcHk0lgJPlGRsyBdgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlyqs74%253D&md5=e923b4054210b2902140bd155e8a7daf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fni904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni904%26sid%3Dliteratum%253Aachs%26aulast%3DFontenot%26aufirst%3DJ.%2BD.%26aulast%3DGavin%26aufirst%3DM.%2BA.%26aulast%3DRudensky%26aufirst%3DA.%2BY.%26atitle%3DFoxp3%2520programs%2520the%2520development%2520and%2520function%2520of%2520CD4%252BCD25%252B%2520regulatory%2520T%2520cells%26jtitle%3DNat.%2520Immunol.%26date%3D2003%26volume%3D4%26spage%3D330%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Piccirillo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shevach, E. M.</span><span> </span><span class="NLM_article-title">Naturally-occurring CD4+CD25+ immunoregulatory T cells: central players in the arena of peripheral tolerance</span> <span class="citation_source-journal">Semin. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.smim.2003.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15036231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht1Khtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=81-88&author=C.+A.+Piccirilloauthor=E.+M.+Shevach&title=Naturally-occurring+CD4%2BCD25%2B+immunoregulatory+T+cells%3A+central+players+in+the+arena+of+peripheral+tolerance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Naturally-occurring CD4+CD25+ immunoregulatory T cells: central players in the arena of peripheral tolerance</span></div><div class="casAuthors">Piccirillo, Ciriaco A.; Shevach, Ethan M.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-88</span>CODEN:
                <span class="NLM_cas:coden">SEIME2</span>;
        ISSN:<span class="NLM_cas:issn">1044-5323</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Self/non-self discrimination is a complex process that involves maintaining tolerance to autoantigens while preserving the potential to generate an effective humoral and cellular immune responses against invading pathogens.  In the last decade, there has been a remarkable resurgence in research on suppressor or regulatory T cells.  Few areas have captivated the attention of immunologists so vividly and entertained so many passionate debates.  Indeed, CD4+CD25+ Treg, which are at the very center of this fascination, have emerged as a dominant T cell population capable of mediating peripheral tolerance to autoantigens, but whose functions have now been extended to regulation of T cell responses directed to foreign antigens.  However, a no. of fundamental questions concerning the origin, phenotypic nature and mechanism of action of CD4+CD25+ Treg cells remain elusive and misunderstood.  We propose the existence of two general subsets of CD4+CD25+ Treg cells, naturally-occurring and induced, that differ in their origin, developmental and activation requirements, and mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJNDe4bgB0tLVg90H21EOLACvtfcHk0lhFt3ovHcWPzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht1Khtb0%253D&md5=00b723fccb0ca5fd7ba341366e27fe67</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.smim.2003.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.smim.2003.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DPiccirillo%26aufirst%3DC.%2BA.%26aulast%3DShevach%26aufirst%3DE.%2BM.%26atitle%3DNaturally-occurring%2520CD4%252BCD25%252B%2520immunoregulatory%2520T%2520cells%253A%2520central%2520players%2520in%2520the%2520arena%2520of%2520peripheral%2520tolerance%26jtitle%3DSemin.%2520Immunol.%26date%3D2004%26volume%3D16%26spage%3D81%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Ziegler, S. F.</span><span> </span><span class="NLM_article-title">FOXP3: of mice and men</span> <span class="citation_source-journal">Annu. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1146%2Fannurev.immunol.24.021605.090547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16551248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFSqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=209-226&author=S.+F.+Ziegler&title=FOXP3%3A+of+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">FOXP3: of mice and men</span></div><div class="casAuthors">Ziegler, Steven F.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">209-226</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The immune system has evolved mechanisms to recognize and eliminate threats, as well as to protect against self-destruction.  Tolerance to self-antigens is generated through two fundamental mechanisms: (a) elimination of self-reactive cells in the thymus during selection and (b) generation of a variety of peripheral regulatory cells to control self-reactive cells that escape the thymus.  It is becoming increasing apparent that a population of thymically derived CD4+ regulatory T cells, exemplified by the expression of the IL-2Rα chain, is essential for the maintenance of peripheral tolerance.  Recent work has shown that the forkhead family transcription factor Foxp3 is critically important for the development and function of the regulatory T cells.  Lack of Foxp3 leads to development of fatal autoimmune lymphoproliferative disease; furthermore, ectopic Foxp3 expression can phenotypically convert effector T cells to regulatory T cells.  This review focuses on Foxp3 expression and function and highlights differences between humans and mice regarding Foxp3 regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1xFg4P8Ss7Vg90H21EOLACvtfcHk0lhFt3ovHcWPzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFSqt7s%253D&md5=76ece472511a29ba04d71173ef09ddee</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1146%2Fannurev.immunol.24.021605.090547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.immunol.24.021605.090547%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%2BF.%26atitle%3DFOXP3%253A%2520of%2520mice%2520and%2520men%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2006%26volume%3D24%26spage%3D209%26epage%3D226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Role of Kruppel-like factors in leukocyte development, function, and disease</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">4404</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=4404-4414&author=Z.+Caoauthor=X.+Sunauthor=B.+Icliauthor=A.+K.+Waraauthor=M.+W.+Feinberg&title=Role+of+Kruppel-like+factors+in+leukocyte+development%2C+function%2C+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DRole%2520of%2520Kruppel-like%2520factors%2520in%2520leukocyte%2520development%252C%2520function%252C%2520and%2520disease%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D4404%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Majmudar, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapp, A. K.</span><span> </span><span class="NLM_article-title">Chemical approaches to transcriptional regulation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2005.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16122970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=467-474&author=C.+Y.+Majmudarauthor=A.+K.+Mapp&title=Chemical+approaches+to+transcriptional+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to transcriptional regulation</span></div><div class="casAuthors">Majmudar, Chinmay Y.; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Given the correlation between many human diseases and mis-regulated transcription, there is a growing need for mols. that can inhibit or mimic key interactions between transcriptional activators and their binding partners.  Because transcriptional activators typically participate in many different protein-protein binding events, the identification of small mols. or peptides that specifically target individual interactions represents a significant challenge.  In spite of this, several small mols. that preferentially inhibit particular complexes of transcriptional activators or mimic the function of activators have recently been reported.  These mols. serve as excellent mechanistic tools for studying transcription and, further, have outstanding therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeAxupgngKD7Vg90H21EOLACvtfcHk0liWUj-EJjwYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7rO&md5=2f2d06fa71d25f30b57c632c99e888ff</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DMajmudar%26aufirst%3DC.%2BY.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DChemical%2520approaches%2520to%2520transcriptional%2520regulation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D467%26epage%3D474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Berg, T.</span><span> </span><span class="NLM_article-title">Inhibition of transcription factors with small organic molecules</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2008.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=18706517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=464-471&author=T.+Berg&title=Inhibition+of+transcription+factors+with+small+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of transcription factors with small organic molecules</span></div><div class="casAuthors">Berg, Thorsten</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">464-471</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Gene-specific transcription factors are DNA binding proteins that function at the center of the transcriptional machinery to mediate the cellular phenotype in response to extracellular signals.  Owing to the lack of enzymic activities of these intracellular proteins, manipulating the activity of transcription factors by small org. mols. is widely considered to be a difficult task.  In this review, the author discusses general approaches designed to inhibit a subset of transcription factors, referred to as dimeric transcription factors, by cell-permeable small org. mols. that inhibit protein-protein interactions or protein-DNA interactions.  These dimeric transcription factors include hypoxia-inducible factor (HIF)-1, c-Myc, and signal transducer and activator of transcription (STAT)-3.  The data presented here indicate that dimeric transcription factors can be potently and selectively inhibited by drug-like small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfB82cxqtqjLVg90H21EOLACvtfcHk0liWUj-EJjwYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsrjO&md5=1b0e3890a0091f2ba8e5cac978e9c6ba</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520transcription%2520factors%2520with%2520small%2520organic%2520molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D464%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Koehler, A. N.</span><span> </span><span class="NLM_article-title">A complex task? Direct modulation of transcription factors with small molecules</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2010.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=20395165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=331-340&author=A.+N.+Koehler&title=A+complex+task%3F+Direct+modulation+of+transcription+factors+with+small+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A complex task? Direct modulation of transcription factors with small molecules</span></div><div class="casAuthors">Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-340</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Transcription factors with aberrant activity in disease are promising yet untested targets for therapeutic development, particularly in oncol.  Directly inhibiting or activating the function of a transcription factor requires specific disruption or recruitment of protein-protein or protein-DNA interactions.  The discovery or design of small mols. that specifically modulate these interactions has thus far proven to be a significant challenge and the protein class is often perceived to be undruggable.  ' This review will summarize recent progress in the development of small-mol. probes of transcription factors and provide evidence to challenge the notion that this important protein class is chem. intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr83_m66fDJsrVg90H21EOLACvtfcHk0liWUj-EJjwYDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D&md5=a3c2dfa1aa4edc7e387b5048c87bf17e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DA%2520complex%2520task%253F%2520Direct%2520modulation%2520of%2520transcription%2520factors%2520with%2520small%2520molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D331%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Lin, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbas, C. F.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, P. G.</span><span> </span><span class="NLM_article-title">Small-molecule switches for zinc finger transcription factors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">613</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja028408e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=612-613&author=Q.+Linauthor=C.+F.+Barbasauthor=P.+G.+Schultz&title=Small-molecule+switches+for+zinc+finger+transcription+factors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja028408e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja028408e%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DSmall-molecule%2520switches%2520for%2520zinc%2520finger%2520transcription%2520factors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D612%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Lauth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergstrom, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toftgard, R.</span><span> </span><span class="NLM_article-title">Inhibition of GLI-mediated transcription and tumor cell growth by small-molecule antagonists</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8455</span><span class="NLM_x">–</span> <span class="NLM_lpage">8460</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1073%2Fpnas.0609699104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17494766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVChtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8455-8460&author=M.+Lauthauthor=A.+Bergstromauthor=T.+Shimokawaauthor=R.+Toftgard&title=Inhibition+of+GLI-mediated+transcription+and+tumor+cell+growth+by+small-molecule+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of GLI-mediated transcription and tumor cell growth by small-molecule antagonists</span></div><div class="casAuthors">Lauth, Matthias; Bergstroem, Asa; Shimokawa, Takashi; Toftgard, Rune</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8455-8460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The developmentally important Hedgehog (Hh) signaling pathway has recently been implicated in several forms of solid cancer.  Current drug development programs focus on targeting the protooncogene Smoothened, a key transmembrane pathway member.  These drug candidates, albeit promising, do not address the scenario in which pathway activation occurs downstream of Smoothened, as obsd. in cases of medulloblastoma, glioma, pericytoma, breast cancer, and prostate cancer.  A cellular screen for small-mol. antagonists of GLI-mediated transcription, which constitutes the final step in the Hh pathway, revealed two mols. that are able to selectively inhibit GLI-mediated gene transactivation.  We provide genetic evidence of downstream pathway blockade by these compds. and demonstrate the ineffectiveness of upstream antagonists such as cyclopamine in such situations.  Mechanistically, both inhibitors act in the nucleus to block GLI function, and one of them interferes with GLI1 DNA binding in living cells.  Importantly, the discovered compds. efficiently inhibited in vitro tumor cell proliferation in a GLI-dependent manner and successfully blocked cell growth in an in vivo xenograft model using human prostate cancer cells harboring downstream activation of the Hh pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnlBrXj0X1ELVg90H21EOLACvtfcHk0lhmXmHtqcz3wg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVChtbw%253D&md5=c579b952f8c878fa3058b21d41e16f07</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609699104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609699104%26sid%3Dliteratum%253Aachs%26aulast%3DLauth%26aufirst%3DM.%26aulast%3DBergstrom%26aufirst%3DA.%26aulast%3DShimokawa%26aufirst%3DT.%26aulast%3DToftgard%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520GLI-mediated%2520transcription%2520and%2520tumor%2520cell%2520growth%2520by%2520small-molecule%2520antagonists%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D8455%26epage%3D8460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kroemer, R. T.</span><span> </span><span class="NLM_article-title">Structure-based drug design: docking and scoring</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+docking+and+scoring"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0lghzwQ2nnIeTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D312%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">597</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0lghzwQ2nnIeTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lghzwQ2nnIeTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp003919d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=6474-6487&author=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+Reparametrization+of+the+OPLS-AA+Force+Field+for+Proteins+via+Comparison+with+Accurate+Quantum+Chemical+Calculations+on+Peptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span></div><div class="casAuthors">Kaminski, George A.; Friesner, Richard A.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6474-6487</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present results of improving the OPLS-AA force field for peptides by means of refitting the key Fourier torsional coeffs.  The fitting technique combines using accurate ab initio data as the target, choosing an efficient fitting subspace of the whole potential-energy surface, and detg. wts. for each of the fitting points based on magnitudes of the potential-energy gradient.  The av. energy RMS deviation from the LMP2/cc-pVTZ(-f)//HF/6-31G** data is reduced by ∼40% from 0.81 to 0.47 kcal/mol as a result of the fitting for the electrostatically uncharged dipeptides.  Transferability of the parameters is demonstrated by using the same alanine dipeptide-fitted backbone torsional parameters for all of the other dipeptides (with the appropriate side-chain refitting) and the alanine tetrapeptide.  Parameters of nonbonded interactions have also been refitted for the sulfur-contg. dipeptides (cysteine and methionine), and the validity of the new Coulombic charges and the van der Waals σ's and ε's is proved through reproducing gas-phase energies of complex formation heats of vaporization and densities of pure model liqs.  Moreover, a novel approach to fitting torsional parameters for electrostatically charged mol. systems has been presented and successfully tested on five dipeptides with charged side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSHwmffIQePbVg90H21EOLACvtfcHk0lhWaHmBzg-txg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D&md5=3ff059626977ee7f6342466f5820f5b7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjp003919d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp003919d%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520Reparametrization%2520of%2520the%2520OPLS-AA%2520Force%2520Field%2520for%2520Proteins%2520via%2520Comparison%2520with%2520Accurate%2520Quantum%2520Chemical%2520Calculations%2520on%2520Peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D105%26spage%3D6474%26epage%3D6487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Kontoyianni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosnick, C. B.</span><span> </span><span class="NLM_article-title">Functional prediction of binding pockets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2005912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVyit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=824-833&author=M.+Kontoyianniauthor=C.+B.+Rosnick&title=Functional+prediction+of+binding+pockets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Prediction of Binding Pockets</span></div><div class="casAuthors">Kontoyianni, Maria; Rosnick, Christopher B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-833</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Putative function for targets with no known ligands has typically been detd. from liganded homologous proteins using sequence and structure comparisons.  However, it is debatable what percentage of sequence identity implies similar function, whereas structural similarity is focused on global folds and could miss divergent structures and novel global folds.  The present study describes an approach to classify a diverse set of proteins and predict their function.  Descriptors corresponding to structural, physicochem., and geometric properties of the ligand-binding cavities of a collection of 434 complexes (17 protein families) were calcd. and analyzed by statistical methods.  The best model using discriminant function anal. (DFA) consisted of 371 proteins (15 families) and had correct classification rates of 90% and cross-validation 86%.  DFA with one protein and a random sample of the remaining proteins led to 100% correct prediction of putative protein function for 10 of the 15 protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFlWhIcocHrVg90H21EOLACvtfcHk0lhWaHmBzg-txg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVyit7g%253D&md5=b1d043af08019a8c911941d3bd069ffe</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci2005912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2005912%26sid%3Dliteratum%253Aachs%26aulast%3DKontoyianni%26aufirst%3DM.%26aulast%3DRosnick%26aufirst%3DC.%2BB.%26atitle%3DFunctional%2520prediction%2520of%2520binding%2520pockets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D824%26epage%3D833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Ho, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Cavity search: an algorithm for the isolation and display of cavity-like binding regions</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">354</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1990&pages=337-354&author=C.+M.+Hoauthor=G.+R.+Marshall&title=Cavity+search%3A+an+algorithm+for+the+isolation+and+display+of+cavity-like+binding+regions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DC.%2BM.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DCavity%2520search%253A%2520an%2520algorithm%2520for%2520the%2520isolation%2520and%2520display%2520of%2520cavity-like%2520binding%2520regions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1990%26volume%3D4%26spage%3D337%26epage%3D354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: a free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+free+tool+to+discover+chemistry+for+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0liTFKmYlF-dFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liTFKmYlF-dFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smondyrev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">PHASE: a new engine for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">671</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=647-671&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=E.+H.+Knollauthor=S.+N.+Raoauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=PHASE%3A+a+new+engine+for+pharmacophore+perception%2C+3D+QSAR+model+development%2C+and+3D+database+screening%3A+1.+Methodology+and+preliminary+results"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DRao%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DPHASE%253A%2520a%2520new%2520engine%2520for%2520pharmacophore%2520perception%252C%25203D%2520QSAR%2520model%2520development%252C%2520and%25203D%2520database%2520screening%253A%25201.%2520Methodology%2520and%2520preliminary%2520results%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D647%26epage%3D671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smondyrev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, S. N.</span><span> </span><span class="NLM_article-title">PHASE: a novel approach to pharmacophore modeling and 3D database searching</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">370</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1111%2Fj.1747-0285.2006.00384.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16784462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=370-372&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=S.+N.+Rao&title=PHASE%3A+a+novel+approach+to+pharmacophore+modeling+and+3D+database+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Novel approach to pharmacophore modeling and 3D database searching</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Rao, Shashidhar N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">370-372</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmacophore modeling and 3D database searching are now recognized as integral components of lead discovery and lead optimization, and the continuing need for improved pharmacophore-based tools has driven the development of PHASE.  PHASE pharmacophore and 3D QSAR models provide a means for rationalizing structure-activity data, predicting the activity of new compds., and efficiently extg. actives from a prototypical database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo50KpRHTGgAbVg90H21EOLACvtfcHk0liB0aLmfwMn7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D&md5=30d06fe99776e203eed35b0f774bbe64</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00384.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00384.x%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DRao%26aufirst%3DS.%2BN.%26atitle%3DPHASE%253A%2520a%2520novel%2520approach%2520to%2520pharmacophore%2520modeling%2520and%25203D%2520database%2520searching%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D370%26epage%3D372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Yet, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McA’Nulty, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folta, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Layne, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrella, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. E.</span><span> </span><span class="NLM_article-title">Human EZF, a Kruppel-like zinc finger protein, is expressed in vascular endothelial cells and contains transcriptional activation and repression domains</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1031</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=1026-1031&author=S.+F.+Yetauthor=M.+M.+McA%E2%80%99Nultyauthor=S.+C.+Foltaauthor=H.+W.+Yenauthor=M.+Yoshizumiauthor=C.+M.+Hsiehauthor=M.+D.+Layneauthor=M.+T.+Chinauthor=H.+Wangauthor=M.+A.+Perrellaauthor=M.+K.+Jainauthor=M.+E.+Lee&title=Human+EZF%2C+a+Kruppel-like+zinc+finger+protein%2C+is+expressed+in+vascular+endothelial+cells+and+contains+transcriptional+activation+and+repression+domains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYet%26aufirst%3DS.%2BF.%26aulast%3DMcA%25E2%2580%2599Nulty%26aufirst%3DM.%2BM.%26aulast%3DFolta%26aufirst%3DS.%2BC.%26aulast%3DYen%26aufirst%3DH.%2BW.%26aulast%3DYoshizumi%26aufirst%3DM.%26aulast%3DHsieh%26aufirst%3DC.%2BM.%26aulast%3DLayne%26aufirst%3DM.%2BD.%26aulast%3DChin%26aufirst%3DM.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPerrella%26aufirst%3DM.%2BA.%26aulast%3DJain%26aufirst%3DM.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BE.%26atitle%3DHuman%2520EZF%252C%2520a%2520Kruppel-like%2520zinc%2520finger%2520protein%252C%2520is%2520expressed%2520in%2520vascular%2520endothelial%2520cells%2520and%2520contains%2520transcriptional%2520activation%2520and%2520repression%2520domains%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D1026%26epage%3D1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Chrisman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tindall, D. J.</span><span> </span><span class="NLM_article-title">Identification and characterization of a consensus DNA binding element for the zinc finger transcription factor TIEG/EGRalpha</span> <span class="citation_source-journal">DNA Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=187-199&author=H.+R.+Chrismanauthor=D.+J.+Tindall&title=Identification+and+characterization+of+a+consensus+DNA+binding+element+for+the+zinc+finger+transcription+factor+TIEG%2FEGRalpha"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChrisman%26aufirst%3DH.%2BR.%26aulast%3DTindall%26aufirst%3DD.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520a%2520consensus%2520DNA%2520binding%2520element%2520for%2520the%2520zinc%2520finger%2520transcription%2520factor%2520TIEG%252FEGRalpha%26jtitle%3DDNA%2520Cell%2520Biol.%26date%3D2003%26volume%3D22%26spage%3D187%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 10 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Walter  Arancio</span>. </span><span class="cited-content_cbyCitation_article-title">ceRNA analysis of SARS-CoV-2. </span><span class="cited-content_cbyCitation_journal-name">Archives of Virology</span><span> <strong>2021,</strong> <em>166 </em>
                                    (1)
                                     , 271-274. <a href="https://doi.org/10.1007/s00705-020-04856-4" title="DOI URL">https://doi.org/10.1007/s00705-020-04856-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00705-020-04856-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00705-020-04856-4%26sid%3Dliteratum%253Aachs%26jtitle%3DArchives%2520of%2520Virology%26atitle%3DceRNA%252Banalysis%252Bof%252BSARS-CoV-2%26aulast%3DArancio%26aufirst%3DWalter%26date%3D2021%26date%3D2020%26volume%3D166%26issue%3D1%26spage%3D271%26epage%3D274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Akm Khyrul  Wara</span>, <span class="hlFld-ContribAuthor ">Shijia  Wang</span>, <span class="hlFld-ContribAuthor ">Chun  Wu</span>, <span class="hlFld-ContribAuthor ">Fang  Fang</span>, <span class="hlFld-ContribAuthor ">Stefan  Haemmig</span>, <span class="hlFld-ContribAuthor ">Brittany N.  Weber</span>, <span class="hlFld-ContribAuthor ">Ceren O.  Aydogan</span>, <span class="hlFld-ContribAuthor ">Yevgenia  Tesmenitsky</span>, <span class="hlFld-ContribAuthor ">Hassan  Aliakbarian</span>, <span class="hlFld-ContribAuthor ">John R.  Hawse</span>, <span class="hlFld-ContribAuthor ">Malayannan  Subramaniam</span>, <span class="hlFld-ContribAuthor ">Lei  Zhao</span>, <span class="hlFld-ContribAuthor ">Peter T.  Sage</span>, <span class="hlFld-ContribAuthor ">Ali  Tavakkoli</span>, <span class="hlFld-ContribAuthor ">Amanda  Garza</span>, <span class="hlFld-ContribAuthor ">Lydia  Lynch</span>, <span class="hlFld-ContribAuthor ">Alexander S.  Banks</span>, <span class="hlFld-ContribAuthor ">Mark W.  Feinberg</span>. </span><span class="cited-content_cbyCitation_article-title">KLF10 Deficiency in CD4+ T Cells Triggers Obesity, Insulin Resistance, and Fatty Liver. </span><span class="cited-content_cbyCitation_journal-name">Cell Reports</span><span> <strong>2020,</strong> <em>33 </em>
                                    (13)
                                     , 108550. <a href="https://doi.org/10.1016/j.celrep.2020.108550" title="DOI URL">https://doi.org/10.1016/j.celrep.2020.108550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.celrep.2020.108550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.celrep.2020.108550%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Reports%26atitle%3DKLF10%252BDeficiency%252Bin%252BCD4%25252B%252BT%252BCells%252BTriggers%252BObesity%25252C%252BInsulin%252BResistance%25252C%252Band%252BFatty%252BLiver%26aulast%3DWara%26aufirst%3DAkm%2BKhyrul%26date%3D2020%26volume%3D33%26issue%3D13%26spage%3D108550" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyan  Chen</span>, <span class="hlFld-ContribAuthor ">Wenjie  Shi</span>, <span class="hlFld-ContribAuthor ">Heng  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">The role of KLF14 in multiple disease processes. </span><span class="cited-content_cbyCitation_journal-name">BioFactors</span><span> <strong>2020,</strong> <em>46 </em>
                                    (2)
                                     , 276-282. <a href="https://doi.org/10.1002/biof.1612" title="DOI URL">https://doi.org/10.1002/biof.1612</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/biof.1612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbiof.1612%26sid%3Dliteratum%253Aachs%26jtitle%3DBioFactors%26atitle%3DThe%252Brole%252Bof%252BKLF14%252Bin%252Bmultiple%252Bdisease%252Bprocesses%26aulast%3DChen%26aufirst%3DXiaoyan%26date%3D2020%26volume%3D46%26issue%3D2%26spage%3D276%26epage%3D282" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaoyan  Chen</span>, <span class="hlFld-ContribAuthor ">Qinghai  Tan</span>, <span class="hlFld-ContribAuthor ">Yunwu  Wang</span>, <span class="hlFld-ContribAuthor ">Huijie  Lv</span>, <span class="hlFld-ContribAuthor ">Zhihui  Wang</span>, <span class="hlFld-ContribAuthor ">Zhuoying  Lin</span>, <span class="hlFld-ContribAuthor ">Zhipeng  Du</span>, <span class="hlFld-ContribAuthor ">Si  Xiong</span>, <span class="hlFld-ContribAuthor ">Jian  Han</span>, <span class="hlFld-ContribAuthor ">Dean  Tian</span>, <span class="hlFld-ContribAuthor ">Bo  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Overexpression of KLF14 protects against immune-mediated hepatic injury in mice. </span><span class="cited-content_cbyCitation_journal-name">Laboratory Investigation</span><span> <strong>2019,</strong> <em>99 </em>
                                    (1)
                                     , 37-47. <a href="https://doi.org/10.1038/s41374-018-0134-4" title="DOI URL">https://doi.org/10.1038/s41374-018-0134-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41374-018-0134-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41374-018-0134-4%26sid%3Dliteratum%253Aachs%26jtitle%3DLaboratory%2520Investigation%26atitle%3DOverexpression%252Bof%252BKLF14%252Bprotects%252Bagainst%252Bimmune-mediated%252Bhepatic%252Binjury%252Bin%252Bmice%26aulast%3DChen%26aufirst%3DXiaoyan%26date%3D2019%26date%3D2018%26volume%3D99%26issue%3D1%26spage%3D37%26epage%3D47" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Marrero-Rodríguez</span>, <span class="hlFld-ContribAuthor ">Keiko  Taniguchi-Ponciano</span>, <span class="hlFld-ContribAuthor ">Malayannan  Subramaniam</span>, <span class="hlFld-ContribAuthor ">John R.  Hawse</span>, <span class="hlFld-ContribAuthor ">Kevin S.  Pitel</span>, <span class="hlFld-ContribAuthor ">Hugo  Arreola-De la Cruz</span>, <span class="hlFld-ContribAuthor ">Victor  Huerta-Padilla</span>, <span class="hlFld-ContribAuthor ">Gustavo  Ponce-Navarrete</span>, <span class="hlFld-ContribAuthor ">Ma. del Pilar  Figueroa-Corona</span>, <span class="hlFld-ContribAuthor ">Laura  Gomez-Virgilio</span>, <span class="hlFld-ContribAuthor ">Teresa I.  Martinez-Cuevas</span>, <span class="hlFld-ContribAuthor ">Monica  Mendoza-Rodriguez</span>, <span class="hlFld-ContribAuthor ">Miriam  Rodriguez-Esquivel</span>, <span class="hlFld-ContribAuthor ">Pablo  Romero-Morelos</span>, <span class="hlFld-ContribAuthor ">Jorge  Ramirez-Salcedo</span>, <span class="hlFld-ContribAuthor ">Michael  Baudis</span>, <span class="hlFld-ContribAuthor ">Marco  Meraz-Rios</span>, <span class="hlFld-ContribAuthor ">Florinda  Jimenez-Vega</span>, <span class="hlFld-ContribAuthor ">Mauricio  Salcedo</span>. </span><span class="cited-content_cbyCitation_article-title">Krüppel-Like Factor 10 participates in cervical cancer immunoediting through transcriptional regulation of Pregnancy-Specific Beta-1 Glycoproteins. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-27711-8" title="DOI URL">https://doi.org/10.1038/s41598-018-27711-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-27711-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-27711-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DKr%2525C3%2525BCppel-Like%252BFactor%252B10%252Bparticipates%252Bin%252Bcervical%252Bcancer%252Bimmunoediting%252Bthrough%252Btranscriptional%252Bregulation%252Bof%252BPregnancy-Specific%252BBeta-1%252BGlycoproteins%26aulast%3DMarrero-Rodr%25C3%25ADguez%26aufirst%3DDaniel%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David R.  Sweet</span>, <span class="hlFld-ContribAuthor ">Liyan  Fan</span>, <span class="hlFld-ContribAuthor ">Paishiun N.  Hsieh</span>, <span class="hlFld-ContribAuthor ">Mukesh K.  Jain</span>. </span><span class="cited-content_cbyCitation_article-title">Krüppel-Like Factors in Vascular Inflammation: Mechanistic Insights and Therapeutic Potential. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Cardiovascular Medicine</span><span> <strong>2018,</strong> <em>5 </em><a href="https://doi.org/10.3389/fcvm.2018.00006" title="DOI URL">https://doi.org/10.3389/fcvm.2018.00006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fcvm.2018.00006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffcvm.2018.00006%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Cardiovascular%2520Medicine%26atitle%3DKr%2525C3%2525BCppel-Like%252BFactors%252Bin%252BVascular%252BInflammation%25253A%252BMechanistic%252BInsights%252Band%252BTherapeutic%252BPotential%26aulast%3DSweet%26aufirst%3DDavid%2BR.%26date%3D2018%26date%3D2018%26volume%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yanbo  Fan</span>, <span class="hlFld-ContribAuthor ">Haocheng  Lu</span>, <span class="hlFld-ContribAuthor ">Wenying  Liang</span>, <span class="hlFld-ContribAuthor ">Wenting  Hu</span>, <span class="hlFld-ContribAuthor ">Jifeng  Zhang</span>, <span class="hlFld-ContribAuthor ">Y Eugene  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Krüppel-like factors and vascular wall homeostasis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Cell Biology</span><span> <strong>2017,</strong> <em>9 </em>
                                    (5)
                                     , 352-363. <a href="https://doi.org/10.1093/jmcb/mjx037" title="DOI URL">https://doi.org/10.1093/jmcb/mjx037</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/jmcb/mjx037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fjmcb%2Fmjx037%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Cell%2520Biology%26atitle%3DKr%2525C3%2525BCppel-like%252Bfactors%252Band%252Bvascular%252Bwall%252Bhomeostasis%26aulast%3DFan%26aufirst%3DYanbo%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D5%26spage%3D352%26epage%3D363" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Miguel F.  Segura</span>, <span class="hlFld-ContribAuthor ">Luz  Jubierre</span>, <span class="hlFld-ContribAuthor ">SiDe  Li</span>, <span class="hlFld-ContribAuthor ">Aroa  Soriano</span>, <span class="hlFld-ContribAuthor ">Lisa  Koetz</span>, <span class="hlFld-ContribAuthor ">Avital  Gaziel-Sovran</span>, <span class="hlFld-ContribAuthor ">Marc  Masanas</span>, <span class="hlFld-ContribAuthor ">Kevin  Kleffman</span>, <span class="hlFld-ContribAuthor ">John F.  Dankert</span>, <span class="hlFld-ContribAuthor ">Martin J  Walsh</span>, <span class="hlFld-ContribAuthor ">Eva  Hernando</span>. </span><span class="cited-content_cbyCitation_article-title">Krüppel-like factor 4 (KLF4) regulates the miR-183~96~182 cluster under physiologic and pathologic conditions. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (16)
                                     , 26298-26311. <a href="https://doi.org/10.18632/oncotarget.15459" title="DOI URL">https://doi.org/10.18632/oncotarget.15459</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.15459&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.15459%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DKr%2525C3%2525BCppel-like%252Bfactor%252B4%252B%252528KLF4%252529%252Bregulates%252Bthe%252BmiR-183%25257E96%25257E182%252Bcluster%252Bunder%252Bphysiologic%252Band%252Bpathologic%252Bconditions%26aulast%3DSegura%26aufirst%3DMiguel%2BF.%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D16%26spage%3D26298%26epage%3D26311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Judith  Hagenbuchner</span>, <span class="hlFld-ContribAuthor ">Michael J.  Ausserlechner</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting transcription factors by small compounds—Current strategies and future implications. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2016,</strong> <em>107 </em>, 1-13. <a href="https://doi.org/10.1016/j.bcp.2015.12.006" title="DOI URL">https://doi.org/10.1016/j.bcp.2015.12.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2015.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2015.12.006%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DTargeting%252Btranscription%252Bfactors%252Bby%252Bsmall%252Bcompounds%2525E2%252580%252594Current%252Bstrategies%252Band%252Bfuture%252Bimplications%26aulast%3DHagenbuchner%26aufirst%3DJudith%26date%3D2016%26volume%3D107%26spage%3D1%26epage%3D13" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Archibald L.  Svogie</span>, <span class="hlFld-ContribAuthor ">Michelle  Isaacs</span>, <span class="hlFld-ContribAuthor ">Heinrich C.  Hoppe</span>, <span class="hlFld-ContribAuthor ">Setshaba D.  Khanye</span>, <span class="hlFld-ContribAuthor ">Clinton G.L.  Veale</span>. </span><span class="cited-content_cbyCitation_article-title">Indolyl-3-ethanone-α-thioethers: A promising new class of non-toxic antimalarial agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>114 </em>, 79-88. <a href="https://doi.org/10.1016/j.ejmech.2016.02.056" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.02.056</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.02.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.02.056%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIndolyl-3-ethanone-%2525CE%2525B1-thioethers%25253A%252BA%252Bpromising%252Bnew%252Bclass%252Bof%252Bnon-toxic%252Bantimalarial%252Bagents%26aulast%3DSvogie%26aufirst%3DArchibald%2BL.%26date%3D2016%26volume%3D114%26spage%3D79%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Homology model of KLF10 protein revealing the proposed pocket of small molecule binding and its relative location in the DNA-bound complex (A, B). Molecular surface of DNA-binding domain of KLF10 protein (blue) along with DNA strands (plum) shown to represent the relative orientation/location of proposed binding pockets along with compound #48-15 (A; green, mesh representation). Proposed binding pocket within second zinc finger of KLF10 (electrostatic surface; blue color for positive potential and red color for negative potential), with α and β subpockets separated by Glu415 side chain (B). Inhibition of KLF10 transcriptional activity by small molecule compounds #48, #48-15, and #15-09 (C–E): HeLa cells were transiently transfected with human KLF10 and a CACCC-luciferase reporter. After transfection, cells were allowed to grow for 24 h and subsequently treated with compounds #48 (C), #48-15 (D), and #15-09 (E) for another 24 h, at which time they were harvested for luciferase activity assays. In (F), HeLa cells were transfected in a similar manner as above, except they were treated with small molecule compounds #3, #23, #37, #40, #65, #79, #83, #96, #104, #106, #114, #118, #121, #126, and #146. Data show the mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.01; Con, empty-vector control.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Dose–response curves of KLF10 transcriptional activity of HeLa cells treated with small molecule compounds #48, #48-15, and #15-09. HeLa cells were transiently transfected with human KLF10 and a CACCC-luciferase reporter. After transfection, cells were allowed to grow for 24 h and subsequently treated with different amounts of #48 (A), #48-15 (B), and #15-09 (C) for another 24 h, at which time they were harvested for luciferase activity assays.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Effect of identified small molecule compounds on KLF10 DNA-binding activity by electromobility gel shift assays. (A) GST-KLF10 specifically bound to a KLF10 consensus DNA-binding sequence. Small molecule compounds #48 (B), #48-15 (C), and #15-09 (D) inhibited KLF10 DNA-binding activity. (E, F) In contrast, there was no inhibition of DNA-protein binding of the related KLF, KLF1, by small molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Inhibition of conversion of CD4+CD25- T cells to CD4+CD25+ T regs by small molecule compounds #48, #48-15, and #15-09. CD4+CD25– T cells were isolated and cultured for the conversion to CD4+CD25+ T regs in the presence or absence of small molecule compounds #48 (A), #48-15 (B), and #15-09 (C) for 3 days and harvested for analysis by flow cytometry. Data show mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.05.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition of FoxP3 expression by small molecule compounds #48, #48-15, and #15-09. CD4+CD25– T cells were isolated and cultured for the conversion to CD4+CD25+ T regs in the presence or absence of small molecule compounds #48 (50 μM), #48-15 (50 μM), and #15-09 (100 μM) and harvested for analysis by qPCR for Foxp3 expression. Data show the mean ± SD, <i>n</i> = 3: ∗, <i>P</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/medium/jm-2014-018187_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0007.jpeg" id="rightTab-GRAPHIC-d3383e1621-autogenerated" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Binding modes of compounds #48 (A), #48-15 (B), and #15-09 (C) as predicted by Glide docking simulation. Protein side chains in green refer to the original position of receptor coordinates used in docking. The compound in gray stick refers to the rescored pose. The compound in torquise (sky blue) stick refers to the refined ligand pose. The compound in plum/pink stick refers to the eMBrAcE minimized (and xGlide rescored top) pose. Yellow dotted lines indicate H-bonds between ligand and receptor. The eMBrAcE minimization considers flexibility of both ligand and residues within 5 Å pocket surrounding the ligand and is shown superimposed (plum colored side chain) to outline the changes after minimization.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-3/jm5018187/20150206/images/large/jm-2014-018187_0006.jpeg jm-2014-018187_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm5018187&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i29">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05030" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05030" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 45 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Venter, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Adams, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myers, E. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mural, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutton, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, H. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yandell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holt, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gocayne, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amanatides, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballew, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huson, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wortman, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kodira, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, X. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skupski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramanian, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, P. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabor Miklos, G. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clark, A. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nadeau, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKusick, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zinder, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levine, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Slayman, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunkapiller, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delcher, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dew, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fasulo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flanigan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Florea, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halpern, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannenhalli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kravitz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobarry, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinert, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Remington, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abu-Threideh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beasley, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Biddick, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonazzi, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandon, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cargill, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chandramouliswaran, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlab, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Francesco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dunn, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eilbeck, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evangelista, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabrielian, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ge, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiman, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Higgins, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ji, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ketchum, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lai, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkulov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milshina, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Naik, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narayan, V. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neelam, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nusskern, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusch, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salzberg, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shue, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wides, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baumhueter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spier, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravchik, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodage, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ali, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Awe, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldwin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baden, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnstead, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrow, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beeson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Busam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Center, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Curry, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Danaher, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desilets, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dietz, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doup, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferriera, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garg, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gluecksmann, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hart, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haynes, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heiner, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hladun, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hostin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Houck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howland, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibegwam, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalush, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kline, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koduru, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Love, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">May, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCawley, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McIntosh, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMullen, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moy, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nelson, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfannkoch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pratts, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Puri, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qureshi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reardon, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rodriguez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogers, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romblad, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ruhfel, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sitter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smallwood, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stewart, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Strong, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suh, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thomas, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tint, N. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tse, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vech, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wetter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Windsor, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winn-Deen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolfe, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaveri, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaveri, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abril, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guigo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sjolander, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karlak, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kejariwal, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lazareva, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Narechania, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diemer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muruganujan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sato, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bafna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Istrail, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lippert, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walenz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yooseph, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Basu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baxendale, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blick, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caminha, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carnes-Stine, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caulk, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coyne, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahlke, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mays, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dombroski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donnelly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ely, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esparham, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fosler, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gire, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glanowski, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glasser, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glodek, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gorokhov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gropman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Henderson, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoover, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jordan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kasha, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraft, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Levitsky, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lewis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majoros, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDaniel, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nodell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanders, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simpson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprague, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stockwell, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venter, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xia, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zandieh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, X.</span><span> </span><span class="NLM_article-title">The sequence of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">291</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">1351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1126%2Fscience.1058040" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=11181995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXhtlSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=291&publication_year=2001&pages=1304-1351&author=J.+C.+Venterauthor=M.+D.+Adamsauthor=E.+W.+Myersauthor=P.+W.+Liauthor=R.+J.+Muralauthor=G.+G.+Suttonauthor=H.+O.+Smithauthor=M.+Yandellauthor=C.+A.+Evansauthor=R.+A.+Holtauthor=J.+D.+Gocayneauthor=P.+Amanatidesauthor=R.+M.+Ballewauthor=D.+H.+Husonauthor=J.+R.+Wortmanauthor=Q.+Zhangauthor=C.+D.+Kodiraauthor=X.+H.+Zhengauthor=L.+Chenauthor=M.+Skupskiauthor=G.+Subramanianauthor=P.+D.+Thomasauthor=J.+Zhangauthor=G.+L.+Gabor+Miklosauthor=C.+Nelsonauthor=S.+Broderauthor=A.+G.+Clarkauthor=J.+Nadeauauthor=V.+A.+McKusickauthor=N.+Zinderauthor=A.+J.+Levineauthor=R.+J.+Robertsauthor=M.+Simonauthor=C.+Slaymanauthor=M.+Hunkapillerauthor=R.+Bolanosauthor=A.+Delcherauthor=I.+Dewauthor=D.+Fasuloauthor=M.+Flaniganauthor=L.+Floreaauthor=A.+Halpernauthor=S.+Hannenhalliauthor=S.+Kravitzauthor=S.+Levyauthor=C.+Mobarryauthor=K.+Reinertauthor=K.+Remingtonauthor=J.+Abu-Threidehauthor=E.+Beasleyauthor=K.+Biddickauthor=V.+Bonazziauthor=R.+Brandonauthor=M.+Cargillauthor=I.+Chandramouliswaranauthor=R.+Charlabauthor=K.+Chaturvediauthor=Z.+Dengauthor=V.+Di+Francescoauthor=P.+Dunnauthor=K.+Eilbeckauthor=C.+Evangelistaauthor=A.+E.+Gabrielianauthor=W.+Ganauthor=W.+Geauthor=F.+Gongauthor=Z.+Guauthor=P.+Guanauthor=T.+J.+Heimanauthor=M.+E.+Higginsauthor=R.+R.+Jiauthor=Z.+Keauthor=K.+A.+Ketchumauthor=Z.+Laiauthor=Y.+Leiauthor=Z.+Liauthor=J.+Liauthor=Y.+Liangauthor=X.+Linauthor=F.+Luauthor=G.+V.+Merkulovauthor=N.+Milshinaauthor=H.+M.+Mooreauthor=A.+K.+Naikauthor=V.+A.+Narayanauthor=B.+Neelamauthor=D.+Nusskernauthor=D.+B.+Ruschauthor=S.+Salzbergauthor=W.+Shaoauthor=B.+Shueauthor=J.+Sunauthor=Z.+Wangauthor=A.+Wangauthor=X.+Wangauthor=J.+Wangauthor=M.+Weiauthor=R.+Widesauthor=C.+Xiaoauthor=C.+Yanauthor=A.+Yaoauthor=J.+Yeauthor=M.+Zhanauthor=W.+Zhangauthor=H.+Zhangauthor=Q.+Zhaoauthor=L.+Zhengauthor=F.+Zhongauthor=W.+Zhongauthor=S.+Zhuauthor=S.+Zhaoauthor=D.+Gilbertauthor=S.+Baumhueterauthor=G.+Spierauthor=C.+Carterauthor=A.+Cravchikauthor=T.+Woodageauthor=F.+Aliauthor=H.+Anauthor=A.+Aweauthor=D.+Baldwinauthor=H.+Badenauthor=M.+Barnsteadauthor=I.+Barrowauthor=K.+Beesonauthor=D.+Busamauthor=A.+Carverauthor=A.+Centerauthor=M.+L.+Chengauthor=L.+Curryauthor=S.+Danaherauthor=L.+Davenportauthor=R.+Desiletsauthor=S.+Dietzauthor=K.+Dodsonauthor=L.+Doupauthor=S.+Ferrieraauthor=N.+Gargauthor=A.+Gluecksmannauthor=B.+Hartauthor=J.+Haynesauthor=C.+Haynesauthor=C.+Heinerauthor=S.+Hladunauthor=D.+Hostinauthor=J.+Houckauthor=T.+Howlandauthor=C.+Ibegwamauthor=J.+Johnsonauthor=F.+Kalushauthor=L.+Klineauthor=S.+Koduruauthor=A.+Loveauthor=F.+Mannauthor=D.+Mayauthor=S.+McCawleyauthor=T.+McIntoshauthor=I.+McMullenauthor=M.+Moyauthor=L.+Moyauthor=B.+Murphyauthor=K.+Nelsonauthor=C.+Pfannkochauthor=E.+Prattsauthor=V.+Puriauthor=H.+Qureshiauthor=M.+Reardonauthor=R.+Rodriguezauthor=Y.+H.+Rogersauthor=D.+Rombladauthor=B.+Ruhfelauthor=R.+Scottauthor=C.+Sitterauthor=M.+Smallwoodauthor=E.+Stewartauthor=R.+Strongauthor=E.+Suhauthor=R.+Thomasauthor=N.+N.+Tintauthor=S.+Tseauthor=C.+Vechauthor=G.+Wangauthor=J.+Wetterauthor=S.+Williamsauthor=M.+Williamsauthor=S.+Windsorauthor=E.+Winn-Deenauthor=K.+Wolfeauthor=J.+Zaveriauthor=K.+Zaveriauthor=J.+F.+Abrilauthor=R.+Guigoauthor=M.+J.+Campbellauthor=K.+V.+Sjolanderauthor=B.+Karlakauthor=A.+Kejariwalauthor=H.+Miauthor=B.+Lazarevaauthor=T.+Hattonauthor=A.+Narechaniaauthor=K.+Diemerauthor=A.+Muruganujanauthor=N.+Guoauthor=S.+Satoauthor=V.+Bafnaauthor=S.+Istrailauthor=R.+Lippertauthor=R.+Schwartzauthor=B.+Walenzauthor=S.+Yoosephauthor=D.+Allenauthor=A.+Basuauthor=J.+Baxendaleauthor=L.+Blickauthor=M.+Caminhaauthor=J.+Carnes-Stineauthor=P.+Caulkauthor=Y.+H.+Chiangauthor=M.+Coyneauthor=C.+Dahlkeauthor=A.+Maysauthor=M.+Dombroskiauthor=M.+Donnellyauthor=D.+Elyauthor=S.+Esparhamauthor=C.+Foslerauthor=H.+Gireauthor=S.+Glanowskiauthor=K.+Glasserauthor=A.+Glodekauthor=M.+Gorokhovauthor=K.+Grahamauthor=B.+Gropmanauthor=M.+Harrisauthor=J.+Heilauthor=S.+Hendersonauthor=J.+Hooverauthor=D.+Jenningsauthor=C.+Jordanauthor=J.+Jordanauthor=J.+Kashaauthor=L.+Kaganauthor=C.+Kraftauthor=A.+Levitskyauthor=M.+Lewisauthor=X.+Liuauthor=J.+Lopezauthor=D.+Maauthor=W.+Majorosauthor=J.+McDanielauthor=S.+Murphyauthor=M.+Newmanauthor=T.+Nguyenauthor=N.+Nguyenauthor=M.+Nodellauthor=S.+Panauthor=J.+Peckauthor=M.+Petersonauthor=W.+Roweauthor=R.+Sandersauthor=J.+Scottauthor=M.+Simpsonauthor=T.+Smithauthor=A.+Spragueauthor=T.+Stockwellauthor=R.+Turnerauthor=E.+Venterauthor=M.+Wangauthor=M.+Wenauthor=D.+Wuauthor=M.+Wuauthor=A.+Xiaauthor=A.+Zandiehauthor=X.+Zhu&title=The+sequence+of+the+human+genome"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The sequence of the human genome</span></div><div class="casAuthors">Venter, J. Craig; Adams, Mark D.; Myers, Eugene W.; Li, Peter W.; Mural, Richard J.; Sutton, Granger G.; Smith, Hamilton O.; Yandell, Mark; Evans, Cheryl A.; Holt, Robert A.; Gocayne, Jeannine D.; Amanatides, Peter; Ballew, Richard M.; Huson, Daniel H.; Wortman, Jennifer Russo; Zhang, Qing; Kodira, Chinnappa D.; Zheng, Xiangqun H.; Chen, Lin; Skupski, Marian; Subramanian, Gangadharan; Thomas, Paul D.; Zhang, Jinghui; Miklos, George L. Gabor; Nelson, Catherine; Broder, Samuel; Clark, Andrew G.; Nadeau, Joe; McKusick, Victor A.; Zinder, Norton; Levine, Arnold J.; Roberts, Richard J.; Simon, Mel; Slayman, Carolyn; Hunkapiller, Michael; Bolanos, Randall; Delcher, Arthur; Dew, Ian; Fasulo, Daniel; Flanigan, Michael; Florea, Liliana; Halpern, Aaron; Hannenhalli, Sridhar; Kravitz, Saul; Levy, Samuel; Mobarry, Clark; Reiner, Knut; Remington, Karin; Abu-Threide, Jane; Beasle, Ellen; Biddick, Kendra; Bonazzi, Vivien; Brandon, Rhonda; Cargill, Michele; Chandramouliswaran, Ishwar; Charlab, Rosane; Chaturved, Kabir; Deng, Zuoming; Di Francesco, Valentina; Dunn, Patrick; Eilbeck, Karen; Evangelista, Carlos; Gabrielian, Andrei E.; Gan, Weiniu; Ge, Wangmao; Gon, Fangcheng; Gu, Zhiping; Guan, Ping; Heiman, Thomas J.; Higgins, Maureen E.; Ji, Rui-Ru; Ke, Zhaoxi; Ketchum, Karen A.; Lai, Zhongwu; Lei, Yiding; Li, Zhenya; Li, Jiayin; Liang, Yong; Lin, Xiaoying; Lu, Fu; Merkulov, Gennady V.; Milshina, Natalia; Moore, Helen M.; Naik, Ashwinikumar K.; Narayan, Vaibhav A.; Neelam, Beena; Nusskern, Deborah; Rusch, Douglas B.; Salzberg, Steven; Shao, Wei; Shue, Bixiong; Sun, Jingtao; Wang, Zhen Yuan; Wang, Aihui; Wang, Xin; Wang, Jian; Wei, Ming-Hui; Wides, Ron; Xiao, Chunlin; Yan, Chunhua; Yao, Alison; Ye, Jane; Zhan, Ming; Zhan, Weiqing; Zhang, Hongyu; Zhao, Qi; Zheng, Liansheng; Zhong, Fei; Zhong, Wenyan; Zhu, Shiaoping C.; Zhao, Shaying; Gilbert, Dennis; Baumhueter, Suzanna; Spier, Gene; Carter, Christine; Cravchik, Anibal; Woodage, Trevor; Ali, Feroze; An, Huijin; Awe, Aderonke; Baldwin, Danita; Baden, Holly; Barnstead, Mary; Barrow, Lan; Beeson, Karen; Busam, Dana; Carver, Amy; Center, Angela; Cheng, Ming Lai; Curry, Liz; Danaher, Steve; Davenport, Lionel; Desilets, Raymond; Dietz, Susanne; Dodson, Kristina; Doup, Lisa; Ferriera, Steven; Garg, Neha; Hostin, Damon; Houck, Jarrett; Howland, Timothy; Ibegwam, Chinyerer; Johnson, Jeffery; Kalush, Francis; Kline, Lesley; Koduru, Shashi; Love, Amy; Mann, Felecia; May, David; Nelson, Keith; Pfannkoch, Cynthia; Pratts, Eric; Puri, Vinita; Qureshi, Hina; Reardon, Matthew; Rodriguez, Robert; Rogers, Yu-Hui; Romblad, Deanna; Ruhfel, Bob; Scott, Richard; Sitter, Cynthia; Smallwood, Michelle; Stewart, Erin; Strong, Renee; Suh, Ellen; Thomas, Reginald; Tint, Ni Ni; Tse, Sukyee; Vech, Claire; Wang, Gary; Wetter, Jeremy; Williams, Sherita; Williams, Monica; Windsor, Sandra; Winn-Deen, Emily; Wolfe, Keriellen; Zaveri, Jayshree; Zaveri, Karena; Abril, Josep F.; Guigo, Roderic; Campbell, Michael J.; sjolander, Kimmen V.; Karlak, Brian; Kejariwal, Anish; Mi, Huaiyu; Lazareva, Betty; Hatton, Thomas; Narechania, Apurva; Diemer, Karen; Muruganujan, Anushya; Guo, Nan; Sato, Shiji; Bafna, Vineet; Istrail, Sorin; Lippert, Ross; Schwartz, Russell; Walenz, Brian; Yooseph, Shibu; Allen, David; Basu, Anand; Baxendale, James; Block, Louis; Caminha, Marcelo; Carnes-Stine, John; Caulk, Parris; Chiang, Yen-hhui; Coyne, My; Dahike, Carl; Mays, Anne Deslattes; Dombroski, Maria; Donnelly, Michael; Ely, Dale; Esparham, Shiva; Foster, Carl; Gire, Harold; Glanowski, Stephen; Glasser, Kenneth; Glodek, Anna; Gorokhov, Mark; Graham, Ken; Gropman, Barry; Harris, Michael; Heil, Jeremy; Henderson, Scott; Hoover, Jeffrey; Jennings, Donald; Jordan, Catherine; Jordan, James; Kasha, John; Kagan, Leonid; Kraft, Cheryl; Levitsky, Alexander; Lewis, Mark; Liu, Xiangjun; Lopez, John; Ma, Daniel; Majoros, William; McDaniel, Joe; Murphy, Sean; Newman, Matthew; Nguyen, Trung; Nguyen, Ngoc; Nodell, Marc; Pan, Sue; Peck, Jim; Peterson, Marshall; Rowe, William; Sanders, Robert; Scott, John; Simpson, Michael; Smith, Thomas; Sprague, Arlan; Stockwell, Timothy; Turner, Russell; Venter, Eli; Wang, Mei; Wen, Meiyuan; Wu, David; Xia, Ashley; Zandieh, Ali; Zhu, Xiaohong</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">291</span>
        (<span class="NLM_cas:issue">5507</span>),
    <span class="NLM_cas:pages">1304-1351</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A 2.91-billion base pair (bp) consensus sequence of the euchromatic portion of the human genome was generated by the whole-genome shotgun sequencing method.  The 14.8-billion bp DNA sequence was generated over 9 mo from 27,271,853 high-quality sequence reads (5.11-fold coverage of the genome) from both ends of plasmid clones made from the DNA of 5 individuals.  Two assembly strategies - a whole-genome assembly and a regional chromosome assembly - were used, each combining sequence data from Celera and the publicly funded genome effort.  The public data, were shredded into 550-bp segments to create a 2.9-fold coverage of those genome regions that had been sequenced, without including biases inherent in the cloning and assembly procedure used by the publicly funded group.  This brought the effective coverage in the assemblies to 8-fold, reducing the no. and size of gaps in. the final assembly over what would be obtained with 5.11-fold coverage.  The 2 assembly strategies yielded very similar results that largely agree with independent mapping data.  The assemblies effectively cover the euchromatic regions of the human chromosomes.  More than 90% of the genome is in scaffold assemblies of ≥100,000 bp, and 25% of the genome is in scaffolds of ≥10 million bp.  Anal. of the genome sequence revealed 26,588 protein-encoding transcripts for which there was strong corroborating evidence and an addnl. ∼12,000 computationally derived genes with mouse matches or other weak supporting evidence.  Although gene-dense clusters are obvious, almost half the genes are dispersed in low G+C sequence sepd. by large tracts of apparently noncoding sequence.  Only 1.1% of the genome is spanned by exons, whereas 24% is in introns, with 75% of the genome being intergenic DNA.  Duplications of segmental blocks, ranging in size up to chromosomal lengths, are abundant throughout the genome and reveal a complex evolutionary history.  Comparative genomic anal. indicates vertebrate expansions of genes assocd. with neuronal function, with tissue-specific developmental regulation, and with the hemostasis and immune systems.  DNA sequence comparisons between the consensus sequence and publicly funded genome data provided locations of 2.1 million single-nucleotide polymorphisms (SNPs).  A random pair of human haploid genomes differed at a rate of 1 bp per 1250 on av., but there was marked heterogeneity in the level of polymorphism across the genome.  Less than 1% of all SNPs resulted in variation in proteins, but the task of detg. which SNPs have functional consequences remains an open challenge.  The genome sequence and addnl. supporting information are available to academic scientists at the Web site www.celera.com.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYYCo_5N1a7bVg90H21EOLACvtfcHk0lgR0PcIQHqb-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXhtlSgsbo%253D&md5=5ec083b5a91372535a562f7aa12ffb28</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1126%2Fscience.1058040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1058040%26sid%3Dliteratum%253Aachs%26aulast%3DVenter%26aufirst%3DJ.%2BC.%26aulast%3DAdams%26aufirst%3DM.%2BD.%26aulast%3DMyers%26aufirst%3DE.%2BW.%26aulast%3DLi%26aufirst%3DP.%2BW.%26aulast%3DMural%26aufirst%3DR.%2BJ.%26aulast%3DSutton%26aufirst%3DG.%2BG.%26aulast%3DSmith%26aufirst%3DH.%2BO.%26aulast%3DYandell%26aufirst%3DM.%26aulast%3DEvans%26aufirst%3DC.%2BA.%26aulast%3DHolt%26aufirst%3DR.%2BA.%26aulast%3DGocayne%26aufirst%3DJ.%2BD.%26aulast%3DAmanatides%26aufirst%3DP.%26aulast%3DBallew%26aufirst%3DR.%2BM.%26aulast%3DHuson%26aufirst%3DD.%2BH.%26aulast%3DWortman%26aufirst%3DJ.%2BR.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DKodira%26aufirst%3DC.%2BD.%26aulast%3DZheng%26aufirst%3DX.%2BH.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSkupski%26aufirst%3DM.%26aulast%3DSubramanian%26aufirst%3DG.%26aulast%3DThomas%26aufirst%3DP.%2BD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGabor%2BMiklos%26aufirst%3DG.%2BL.%26aulast%3DNelson%26aufirst%3DC.%26aulast%3DBroder%26aufirst%3DS.%26aulast%3DClark%26aufirst%3DA.%2BG.%26aulast%3DNadeau%26aufirst%3DJ.%26aulast%3DMcKusick%26aufirst%3DV.%2BA.%26aulast%3DZinder%26aufirst%3DN.%26aulast%3DLevine%26aufirst%3DA.%2BJ.%26aulast%3DRoberts%26aufirst%3DR.%2BJ.%26aulast%3DSimon%26aufirst%3DM.%26aulast%3DSlayman%26aufirst%3DC.%26aulast%3DHunkapiller%26aufirst%3DM.%26aulast%3DBolanos%26aufirst%3DR.%26aulast%3DDelcher%26aufirst%3DA.%26aulast%3DDew%26aufirst%3DI.%26aulast%3DFasulo%26aufirst%3DD.%26aulast%3DFlanigan%26aufirst%3DM.%26aulast%3DFlorea%26aufirst%3DL.%26aulast%3DHalpern%26aufirst%3DA.%26aulast%3DHannenhalli%26aufirst%3DS.%26aulast%3DKravitz%26aufirst%3DS.%26aulast%3DLevy%26aufirst%3DS.%26aulast%3DMobarry%26aufirst%3DC.%26aulast%3DReinert%26aufirst%3DK.%26aulast%3DRemington%26aufirst%3DK.%26aulast%3DAbu-Threideh%26aufirst%3DJ.%26aulast%3DBeasley%26aufirst%3DE.%26aulast%3DBiddick%26aufirst%3DK.%26aulast%3DBonazzi%26aufirst%3DV.%26aulast%3DBrandon%26aufirst%3DR.%26aulast%3DCargill%26aufirst%3DM.%26aulast%3DChandramouliswaran%26aufirst%3DI.%26aulast%3DCharlab%26aufirst%3DR.%26aulast%3DChaturvedi%26aufirst%3DK.%26aulast%3DDeng%26aufirst%3DZ.%26aulast%3DDi%2BFrancesco%26aufirst%3DV.%26aulast%3DDunn%26aufirst%3DP.%26aulast%3DEilbeck%26aufirst%3DK.%26aulast%3DEvangelista%26aufirst%3DC.%26aulast%3DGabrielian%26aufirst%3DA.%2BE.%26aulast%3DGan%26aufirst%3DW.%26aulast%3DGe%26aufirst%3DW.%26aulast%3DGong%26aufirst%3DF.%26aulast%3DGu%26aufirst%3DZ.%26aulast%3DGuan%26aufirst%3DP.%26aulast%3DHeiman%26aufirst%3DT.%2BJ.%26aulast%3DHiggins%26aufirst%3DM.%2BE.%26aulast%3DJi%26aufirst%3DR.%2BR.%26aulast%3DKe%26aufirst%3DZ.%26aulast%3DKetchum%26aufirst%3DK.%2BA.%26aulast%3DLai%26aufirst%3DZ.%26aulast%3DLei%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DF.%26aulast%3DMerkulov%26aufirst%3DG.%2BV.%26aulast%3DMilshina%26aufirst%3DN.%26aulast%3DMoore%26aufirst%3DH.%2BM.%26aulast%3DNaik%26aufirst%3DA.%2BK.%26aulast%3DNarayan%26aufirst%3DV.%2BA.%26aulast%3DNeelam%26aufirst%3DB.%26aulast%3DNusskern%26aufirst%3DD.%26aulast%3DRusch%26aufirst%3DD.%2BB.%26aulast%3DSalzberg%26aufirst%3DS.%26aulast%3DShao%26aufirst%3DW.%26aulast%3DShue%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DA.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWei%26aufirst%3DM.%26aulast%3DWides%26aufirst%3DR.%26aulast%3DXiao%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DYao%26aufirst%3DA.%26aulast%3DYe%26aufirst%3DJ.%26aulast%3DZhan%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DQ.%26aulast%3DZheng%26aufirst%3DL.%26aulast%3DZhong%26aufirst%3DF.%26aulast%3DZhong%26aufirst%3DW.%26aulast%3DZhu%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DS.%26aulast%3DGilbert%26aufirst%3DD.%26aulast%3DBaumhueter%26aufirst%3DS.%26aulast%3DSpier%26aufirst%3DG.%26aulast%3DCarter%26aufirst%3DC.%26aulast%3DCravchik%26aufirst%3DA.%26aulast%3DWoodage%26aufirst%3DT.%26aulast%3DAli%26aufirst%3DF.%26aulast%3DAn%26aufirst%3DH.%26aulast%3DAwe%26aufirst%3DA.%26aulast%3DBaldwin%26aufirst%3DD.%26aulast%3DBaden%26aufirst%3DH.%26aulast%3DBarnstead%26aufirst%3DM.%26aulast%3DBarrow%26aufirst%3DI.%26aulast%3DBeeson%26aufirst%3DK.%26aulast%3DBusam%26aufirst%3DD.%26aulast%3DCarver%26aufirst%3DA.%26aulast%3DCenter%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DM.%2BL.%26aulast%3DCurry%26aufirst%3DL.%26aulast%3DDanaher%26aufirst%3DS.%26aulast%3DDavenport%26aufirst%3DL.%26aulast%3DDesilets%26aufirst%3DR.%26aulast%3DDietz%26aufirst%3DS.%26aulast%3DDodson%26aufirst%3DK.%26aulast%3DDoup%26aufirst%3DL.%26aulast%3DFerriera%26aufirst%3DS.%26aulast%3DGarg%26aufirst%3DN.%26aulast%3DGluecksmann%26aufirst%3DA.%26aulast%3DHart%26aufirst%3DB.%26aulast%3DHaynes%26aufirst%3DJ.%26aulast%3DHaynes%26aufirst%3DC.%26aulast%3DHeiner%26aufirst%3DC.%26aulast%3DHladun%26aufirst%3DS.%26aulast%3DHostin%26aufirst%3DD.%26aulast%3DHouck%26aufirst%3DJ.%26aulast%3DHowland%26aufirst%3DT.%26aulast%3DIbegwam%26aufirst%3DC.%26aulast%3DJohnson%26aufirst%3DJ.%26aulast%3DKalush%26aufirst%3DF.%26aulast%3DKline%26aufirst%3DL.%26aulast%3DKoduru%26aufirst%3DS.%26aulast%3DLove%26aufirst%3DA.%26aulast%3DMann%26aufirst%3DF.%26aulast%3DMay%26aufirst%3DD.%26aulast%3DMcCawley%26aufirst%3DS.%26aulast%3DMcIntosh%26aufirst%3DT.%26aulast%3DMcMullen%26aufirst%3DI.%26aulast%3DMoy%26aufirst%3DM.%26aulast%3DMoy%26aufirst%3DL.%26aulast%3DMurphy%26aufirst%3DB.%26aulast%3DNelson%26aufirst%3DK.%26aulast%3DPfannkoch%26aufirst%3DC.%26aulast%3DPratts%26aufirst%3DE.%26aulast%3DPuri%26aufirst%3DV.%26aulast%3DQureshi%26aufirst%3DH.%26aulast%3DReardon%26aufirst%3DM.%26aulast%3DRodriguez%26aufirst%3DR.%26aulast%3DRogers%26aufirst%3DY.%2BH.%26aulast%3DRomblad%26aufirst%3DD.%26aulast%3DRuhfel%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DR.%26aulast%3DSitter%26aufirst%3DC.%26aulast%3DSmallwood%26aufirst%3DM.%26aulast%3DStewart%26aufirst%3DE.%26aulast%3DStrong%26aufirst%3DR.%26aulast%3DSuh%26aufirst%3DE.%26aulast%3DThomas%26aufirst%3DR.%26aulast%3DTint%26aufirst%3DN.%2BN.%26aulast%3DTse%26aufirst%3DS.%26aulast%3DVech%26aufirst%3DC.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DWetter%26aufirst%3DJ.%26aulast%3DWilliams%26aufirst%3DS.%26aulast%3DWilliams%26aufirst%3DM.%26aulast%3DWindsor%26aufirst%3DS.%26aulast%3DWinn-Deen%26aufirst%3DE.%26aulast%3DWolfe%26aufirst%3DK.%26aulast%3DZaveri%26aufirst%3DJ.%26aulast%3DZaveri%26aufirst%3DK.%26aulast%3DAbril%26aufirst%3DJ.%2BF.%26aulast%3DGuigo%26aufirst%3DR.%26aulast%3DCampbell%26aufirst%3DM.%2BJ.%26aulast%3DSjolander%26aufirst%3DK.%2BV.%26aulast%3DKarlak%26aufirst%3DB.%26aulast%3DKejariwal%26aufirst%3DA.%26aulast%3DMi%26aufirst%3DH.%26aulast%3DLazareva%26aufirst%3DB.%26aulast%3DHatton%26aufirst%3DT.%26aulast%3DNarechania%26aufirst%3DA.%26aulast%3DDiemer%26aufirst%3DK.%26aulast%3DMuruganujan%26aufirst%3DA.%26aulast%3DGuo%26aufirst%3DN.%26aulast%3DSato%26aufirst%3DS.%26aulast%3DBafna%26aufirst%3DV.%26aulast%3DIstrail%26aufirst%3DS.%26aulast%3DLippert%26aufirst%3DR.%26aulast%3DSchwartz%26aufirst%3DR.%26aulast%3DWalenz%26aufirst%3DB.%26aulast%3DYooseph%26aufirst%3DS.%26aulast%3DAllen%26aufirst%3DD.%26aulast%3DBasu%26aufirst%3DA.%26aulast%3DBaxendale%26aufirst%3DJ.%26aulast%3DBlick%26aufirst%3DL.%26aulast%3DCaminha%26aufirst%3DM.%26aulast%3DCarnes-Stine%26aufirst%3DJ.%26aulast%3DCaulk%26aufirst%3DP.%26aulast%3DChiang%26aufirst%3DY.%2BH.%26aulast%3DCoyne%26aufirst%3DM.%26aulast%3DDahlke%26aufirst%3DC.%26aulast%3DMays%26aufirst%3DA.%26aulast%3DDombroski%26aufirst%3DM.%26aulast%3DDonnelly%26aufirst%3DM.%26aulast%3DEly%26aufirst%3DD.%26aulast%3DEsparham%26aufirst%3DS.%26aulast%3DFosler%26aufirst%3DC.%26aulast%3DGire%26aufirst%3DH.%26aulast%3DGlanowski%26aufirst%3DS.%26aulast%3DGlasser%26aufirst%3DK.%26aulast%3DGlodek%26aufirst%3DA.%26aulast%3DGorokhov%26aufirst%3DM.%26aulast%3DGraham%26aufirst%3DK.%26aulast%3DGropman%26aufirst%3DB.%26aulast%3DHarris%26aufirst%3DM.%26aulast%3DHeil%26aufirst%3DJ.%26aulast%3DHenderson%26aufirst%3DS.%26aulast%3DHoover%26aufirst%3DJ.%26aulast%3DJennings%26aufirst%3DD.%26aulast%3DJordan%26aufirst%3DC.%26aulast%3DJordan%26aufirst%3DJ.%26aulast%3DKasha%26aufirst%3DJ.%26aulast%3DKagan%26aufirst%3DL.%26aulast%3DKraft%26aufirst%3DC.%26aulast%3DLevitsky%26aufirst%3DA.%26aulast%3DLewis%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLopez%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DD.%26aulast%3DMajoros%26aufirst%3DW.%26aulast%3DMcDaniel%26aufirst%3DJ.%26aulast%3DMurphy%26aufirst%3DS.%26aulast%3DNewman%26aufirst%3DM.%26aulast%3DNguyen%26aufirst%3DT.%26aulast%3DNguyen%26aufirst%3DN.%26aulast%3DNodell%26aufirst%3DM.%26aulast%3DPan%26aufirst%3DS.%26aulast%3DPeck%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DM.%26aulast%3DRowe%26aufirst%3DW.%26aulast%3DSanders%26aufirst%3DR.%26aulast%3DScott%26aufirst%3DJ.%26aulast%3DSimpson%26aufirst%3DM.%26aulast%3DSmith%26aufirst%3DT.%26aulast%3DSprague%26aufirst%3DA.%26aulast%3DStockwell%26aufirst%3DT.%26aulast%3DTurner%26aufirst%3DR.%26aulast%3DVenter%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DWen%26aufirst%3DM.%26aulast%3DWu%26aufirst%3DD.%26aulast%3DWu%26aufirst%3DM.%26aulast%3DXia%26aufirst%3DA.%26aulast%3DZandieh%26aufirst%3DA.%26aulast%3DZhu%26aufirst%3DX.%26atitle%3DThe%2520sequence%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2001%26volume%3D291%26spage%3D1304%26epage%3D1351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Kim, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berg, J. M.</span><span> </span><span class="NLM_article-title">A 2.2 Å resolution crystal structure of a designed zinc finger protein bound to DNA</span> <span class="citation_source-journal">Nat. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">940</span><span class="NLM_x">–</span> <span class="NLM_lpage">945</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1996&pages=940-945&author=C.+A.+Kimauthor=J.+M.+Berg&title=A+2.2+%C3%85+resolution+crystal+structure+of+a+designed+zinc+finger+protein+bound+to+DNA"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DC.%2BA.%26aulast%3DBerg%26aufirst%3DJ.%2BM.%26atitle%3DA%25202.2%2520%25C3%2585%2520resolution%2520crystal%2520structure%2520of%2520a%2520designed%2520zinc%2520finger%2520protein%2520bound%2520to%2520DNA%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1996%26volume%3D3%26spage%3D940%26epage%3D945" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Simpson, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cram, E. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Czolij, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matthews, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crossley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mackay, J. P.</span><span> </span><span class="NLM_article-title">CCHX zinc finger derivatives retain the ability to bind Zn(II) and mediate protein-DNA interactions</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">28011</span><span class="NLM_x">–</span> <span class="NLM_lpage">28018</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=28011-28018&author=R.+J.+Simpsonauthor=E.+D.+Cramauthor=R.+Czolijauthor=J.+M.+Matthewsauthor=M.+Crossleyauthor=J.+P.+Mackay&title=CCHX+zinc+finger+derivatives+retain+the+ability+to+bind+Zn%28II%29+and+mediate+protein-DNA+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSimpson%26aufirst%3DR.%2BJ.%26aulast%3DCram%26aufirst%3DE.%2BD.%26aulast%3DCzolij%26aufirst%3DR.%26aulast%3DMatthews%26aufirst%3DJ.%2BM.%26aulast%3DCrossley%26aufirst%3DM.%26aulast%3DMackay%26aufirst%3DJ.%2BP.%26atitle%3DCCHX%2520zinc%2520finger%2520derivatives%2520retain%2520the%2520ability%2520to%2520bind%2520Zn%2528II%2529%2520and%2520mediate%2520protein-DNA%2520interactions%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D28011%26epage%3D28018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Pabo, C. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peisach, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, R. A.</span><span> </span><span class="NLM_article-title">Design and selection of novel Cys2His2 zinc finger proteins</span> <span class="citation_source-journal">Annu. Rev. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">70</span><span class="NLM_x">, </span> <span class="NLM_fpage">313</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1146%2Fannurev.biochem.70.1.313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=11395410" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2001&pages=313-340&author=C.+O.+Paboauthor=E.+Peisachauthor=R.+A.+Grant&title=Design+and+selection+of+novel+Cys2His2+zinc+finger+proteins"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Design and selection of novel Cys2His2 zinc finger proteins</span></div><div class="casAuthors">Pabo, Carl O.; Peisach, Ezra; Grant, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">313-340</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review, with 141 refs.  Cys2His2 zinc finger proteins offer a stable and versatile framework for the design of proteins that recognize desired target sites on double-stranded DNA.  Individual fingers from these proteins have a simple ββα structure that folds around a central zinc ion, and tandem sets of fingers can contact neighboring subsites of 3-4 base pairs along the major groove of the DNA.  Although there is no simple, general code for zinc finger-DNA recognition, selection strategies have been developed that allow these proteins to be targeted to almost any desired site on double-stranded DNA.  The affinity and specificity of these new proteins can also be improved by linking more fingers together or by designing proteins that bind as dimers and thus recognize an extended site.  These new proteins can then be modified by adding other domains-for activation or repression of transcription, for DNA cleavage, or for other activities.  Such designer transcription factors and other new proteins will have important applications in biomedical research and in gene therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreBI78my_xQbVg90H21EOLACvtfcHk0liYxA26hCk8mw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXlsVeht7g%253D&md5=2dac8b200d0c0710e8c23d44bf6dd9c3</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1146%2Fannurev.biochem.70.1.313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.biochem.70.1.313%26sid%3Dliteratum%253Aachs%26aulast%3DPabo%26aufirst%3DC.%2BO.%26aulast%3DPeisach%26aufirst%3DE.%26aulast%3DGrant%26aufirst%3DR.%2BA.%26atitle%3DDesign%2520and%2520selection%2520of%2520novel%2520Cys2His2%2520zinc%2520finger%2520proteins%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2001%26volume%3D70%26spage%3D313%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">van Vliet, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Turner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crossley, M.</span><span> </span><span class="NLM_article-title">Human Kruppel-like factor 8: a CACCC-box binding protein that associates with CtBP and represses transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1955</span><span class="NLM_x">–</span> <span class="NLM_lpage">1962</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=1955-1962&author=J.+van+Vlietauthor=J.+Turnerauthor=M.+Crossley&title=Human+Kruppel-like+factor+8%3A+a+CACCC-box+binding+protein+that+associates+with+CtBP+and+represses+transcription"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BVliet%26aufirst%3DJ.%26aulast%3DTurner%26aufirst%3DJ.%26aulast%3DCrossley%26aufirst%3DM.%26atitle%3DHuman%2520Kruppel-like%2520factor%25208%253A%2520a%2520CACCC-box%2520binding%2520protein%2520that%2520associates%2520with%2520CtBP%2520and%2520represses%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D1955%26epage%3D1962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Role of Kruppel-like factors in leukocyte development, function, and disease</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">4404</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=4404-4414&author=Z.+Caoauthor=X.+Sunauthor=B.+Icliauthor=A.+K.+Waraauthor=M.+W.+Feinberg&title=Role+of+Kruppel-like+factors+in+leukocyte+development%2C+function%2C+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DRole%2520of%2520Kruppel-like%2520factors%2520in%2520leukocyte%2520development%252C%2520function%252C%2520and%2520disease%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D4404%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Gallimore, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godkin, A.</span><span> </span><span class="NLM_article-title">Regulatory T cells and tumour immunity—observations in mice and men</span> <span class="citation_source-journal">Immunology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">123</span><span class="NLM_x">, </span> <span class="NLM_fpage">157</span><span class="NLM_x">–</span> <span class="NLM_lpage">163</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=123&publication_year=2008&pages=157-163&author=A.+Gallimoreauthor=A.+Godkin&title=Regulatory+T+cells+and+tumour+immunity%E2%80%94observations+in+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGallimore%26aufirst%3DA.%26aulast%3DGodkin%26aufirst%3DA.%26atitle%3DRegulatory%2520T%2520cells%2520and%2520tumour%2520immunity%25E2%2580%2594observations%2520in%2520mice%2520and%2520men%26jtitle%3DImmunology%26date%3D2008%26volume%3D123%26spage%3D157%26epage%3D163" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Clarke, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Betts, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Plant, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Shanawany, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harrop, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torkington, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rees, B. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Williams, G. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallimore, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Godkin, A. J.</span><span> </span><span class="NLM_article-title">CD4+CD25+FOXP3+ regulatory T cells suppress anti-tumor immune responses in patients with colorectal cancer</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">1</span><span class="NLM_x">, </span> <span class="NLM_fpage">e129</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=e129&author=S.+L.+Clarkeauthor=G.+J.+Bettsauthor=A.+Plantauthor=K.+L.+Wrightauthor=T.+M.+El-Shanawanyauthor=R.+Harropauthor=J.+Torkingtonauthor=B.+I.+Reesauthor=G.+T.+Williamsauthor=A.+M.+Gallimoreauthor=A.+J.+Godkin&title=CD4%2BCD25%2BFOXP3%2B+regulatory+T+cells+suppress+anti-tumor+immune+responses+in+patients+with+colorectal+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DS.%2BL.%26aulast%3DBetts%26aufirst%3DG.%2BJ.%26aulast%3DPlant%26aufirst%3DA.%26aulast%3DWright%26aufirst%3DK.%2BL.%26aulast%3DEl-Shanawany%26aufirst%3DT.%2BM.%26aulast%3DHarrop%26aufirst%3DR.%26aulast%3DTorkington%26aufirst%3DJ.%26aulast%3DRees%26aufirst%3DB.%2BI.%26aulast%3DWilliams%26aufirst%3DG.%2BT.%26aulast%3DGallimore%26aufirst%3DA.%2BM.%26aulast%3DGodkin%26aufirst%3DA.%2BJ.%26atitle%3DCD4%252BCD25%252BFOXP3%252B%2520regulatory%2520T%2520cells%2520suppress%2520anti-tumor%2520immune%2520responses%2520in%2520patients%2520with%2520colorectal%2520cancer%26jtitle%3DPLoS%2520One%26date%3D2006%26volume%3D1%26spage%3De129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3(+) regulatory T cells, immune stimulation and host defence against infection</span> <span class="citation_source-journal">Immunology</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">136</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2012&pages=1-10&author=J.+H.+Roweauthor=J.+M.+Erteltauthor=S.+S.+Way&title=Foxp3%28%2B%29+regulatory+T+cells%2C+immune+stimulation+and+host+defence+against+infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%2528%252B%2529%2520regulatory%2520T%2520cells%252C%2520immune%2520stimulation%2520and%2520host%2520defence%2520against%2520infection%26jtitle%3DImmunology%26date%3D2012%26volume%3D136%26spage%3D1%26epage%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Farrar, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3(+) regulatory T cell expansion required for sustaining pregnancy compromises host defense against prenatal bacterial pathogens</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">54</span><span class="NLM_x">–</span> <span class="NLM_lpage">64</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.chom.2011.06.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=21767812" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptVynu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=54-64&author=J.+H.+Roweauthor=J.+M.+Erteltauthor=M.+N.+Aguileraauthor=M.+A.+Farrarauthor=S.+S.+Way&title=Foxp3%28%2B%29+regulatory+T+cell+expansion+required+for+sustaining+pregnancy+compromises+host+defense+against+prenatal+bacterial+pathogens"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Foxp3+ Regulatory T Cell Expansion Required for Sustaining Pregnancy Compromises Host Defense against Prenatal Bacterial Pathogens</span></div><div class="casAuthors">Rowe, Jared H.; Ertelt, James M.; Aguilera, Marijo N.; Farrar, Michael A.; Way, Sing Sing</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">54-64</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: Although pregnancy confers unique susceptibility to infection, the pregnancy-assocd. immune defects that erode host defense remain largely undefined.  Herein, we demonstrate that expansion of immune-suppressive Foxp3+ regulatory T cells (Tregs) which occurs physiol. during pregnancy or when exptl. induced in transgenic mice caused enhanced susceptibility to prenatal pathogens including Listeria and Salmonella species.  Reciprocally, infection susceptibility was uniformly reduced with Treg ablation.  Importantly however, the sustained expansion of maternal Tregs was essential for maintaining immune tolerance to the developing fetus because even partial transient ablation of Foxp3-expressing cells fractured maternal tolerance to fetal antigen and triggered fetal resorption.  Interestingly, Foxp3 cell-intrinsic defects in the immune-suppressive cytokine IL-10 alone were sufficient to override Treg-mediated infection susceptibility, while IL-10 was nonessential for sustaining pregnancy.  Thus, maternal Treg expansion required for sustaining pregnancy creates naturally occurring holes in host defense that confer prenatal infection susceptibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3KvmgCB0InrVg90H21EOLACvtfcHk0lj-uhG5dXINvw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptVynu7k%253D&md5=d0897d97f51d0308961334587b9b3040</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2011.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2011.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DAguilera%26aufirst%3DM.%2BN.%26aulast%3DFarrar%26aufirst%3DM.%2BA.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%2528%252B%2529%2520regulatory%2520T%2520cell%2520expansion%2520required%2520for%2520sustaining%2520pregnancy%2520compromises%2520host%2520defense%2520against%2520prenatal%2520bacterial%2520pathogens%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2011%26volume%3D10%26spage%3D54%26epage%3D64" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Ertelt, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowe, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysz, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roychowdhury, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aguilera, M. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Way, S. S.</span><span> </span><span class="NLM_article-title">Foxp3+ regulatory T cells impede the priming of protective CD8+ T cells</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">187</span><span class="NLM_x">, </span> <span class="NLM_fpage">2569</span><span class="NLM_x">–</span> <span class="NLM_lpage">2577</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=187&publication_year=2011&pages=2569-2577&author=J.+M.+Erteltauthor=J.+H.+Roweauthor=M.+A.+Myszauthor=C.+Singhauthor=M.+Roychowdhuryauthor=M.+N.+Aguileraauthor=S.+S.+Way&title=Foxp3%2B+regulatory+T+cells+impede+the+priming+of+protective+CD8%2B+T+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DErtelt%26aufirst%3DJ.%2BM.%26aulast%3DRowe%26aufirst%3DJ.%2BH.%26aulast%3DMysz%26aufirst%3DM.%2BA.%26aulast%3DSingh%26aufirst%3DC.%26aulast%3DRoychowdhury%26aufirst%3DM.%26aulast%3DAguilera%26aufirst%3DM.%2BN.%26aulast%3DWay%26aufirst%3DS.%2BS.%26atitle%3DFoxp3%252B%2520regulatory%2520T%2520cells%2520impede%2520the%2520priming%2520of%2520protective%2520CD8%252B%2520T%2520cells%26jtitle%3DJ.%2520Immunol.%26date%3D2011%26volume%3D187%26spage%3D2569%26epage%3D2577" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packard, R. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Esen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stapleton, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kretschmer, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Apostolou, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Boehmer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hansson, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spelsberg, T. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Libby, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Kruppel-like factor KLF10 targets transforming growth factor-beta1 to regulate CD4(+)CD25(−) T cells and T regulatory cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">24914</span><span class="NLM_x">–</span> <span class="NLM_lpage">24924</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=24914-24924&author=Z.+Caoauthor=A.+K.+Waraauthor=B.+Icliauthor=X.+Sunauthor=R.+R.+Packardauthor=F.+Esenauthor=C.+J.+Stapletonauthor=M.+Subramaniamauthor=K.+Kretschmerauthor=I.+Apostolouauthor=H.+von+Boehmerauthor=G.+K.+Hanssonauthor=T.+C.+Spelsbergauthor=P.+Libbyauthor=M.+W.+Feinberg&title=Kruppel-like+factor+KLF10+targets+transforming+growth+factor-beta1+to+regulate+CD4%28%2B%29CD25%28%E2%88%92%29+T+cells+and+T+regulatory+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DPackard%26aufirst%3DR.%2BR.%26aulast%3DEsen%26aufirst%3DF.%26aulast%3DStapleton%26aufirst%3DC.%2BJ.%26aulast%3DSubramaniam%26aufirst%3DM.%26aulast%3DKretschmer%26aufirst%3DK.%26aulast%3DApostolou%26aufirst%3DI.%26aulast%3Dvon%2BBoehmer%26aufirst%3DH.%26aulast%3DHansson%26aufirst%3DG.%2BK.%26aulast%3DSpelsberg%26aufirst%3DT.%2BC.%26aulast%3DLibby%26aufirst%3DP.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DKruppel-like%2520factor%2520KLF10%2520targets%2520transforming%2520growth%2520factor-beta1%2520to%2520regulate%2520CD4%2528%252B%2529CD25%2528%25E2%2588%2592%2529%2520T%2520cells%2520and%2520T%2520regulatory%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D24914%26epage%3D24924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Venuprasad, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harada, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elly, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Subramaniam, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spelsberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span> </span><span class="NLM_article-title">The E3 ubiquitin ligase Itch regulates expression of transcription factor Foxp3 and airway inflammation by enhancing the function of transcription factor TIEG1</span> <span class="citation_source-journal">Nat. Immunol</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">245</span><span class="NLM_x">–</span> <span class="NLM_lpage">253</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni1564" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=18278048" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1cXitVOjtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2008&pages=245-253&author=K.+Venuprasadauthor=H.+Huangauthor=Y.+Haradaauthor=C.+Ellyauthor=M.+Subramaniamauthor=T.+Spelsbergauthor=J.+Suauthor=Y.+C.+Liu&title=The+E3+ubiquitin+ligase+Itch+regulates+expression+of+transcription+factor+Foxp3+and+airway+inflammation+by+enhancing+the+function+of+transcription+factor+TIEG1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">The E3 ubiquitin ligase Itch regulates expression of transcription factor Foxp3 and airway inflammation by enhancing the function of transcription factor TIEG1</span></div><div class="casAuthors">Venuprasad, K.; Huang, Haining; Harada, Yousuke; Elly, Chris; Subramaniam, Malayannan; Spelsberg, Thomas; Su, Jin; Liu, Yun-Cai</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">245-253</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Transforming growth factor-β (TGF-β) signaling in naive T cells induces expression of the transcription factor Foxp3, a master regulator of regulatory T cells (Treg).  However, the mol. mechanisms leading to Foxp3 induction remain unclear.  Here the authors show that Itch-/- T cells were resistant to TGF-β treatment and had less Foxp3 expression.  The E3 ubiquitin ligase Itch assocd. with and promoted conjugation of ubiquitin to the transcription factor TIEG1.  Itch cooperated with TIEG1 to induce Foxp3 expression, which was reversed by TIEG1 deficiency.  Functionally, "TGF-β-converted" Treg cells generated from TIEG1-deficient mice were unable to suppress airway inflammation in vivo.  Apparently, TIEG and Itch contribute to a ubiquitin-dependent non-proteolytic pathway that regulates inducible Foxp3 expression and the control of allergic responses.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0XoQ1p1xnz7Vg90H21EOLACvtfcHk0lgub_R1PLhmuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXitVOjtrs%253D&md5=d31a7410efd14d29a4934426e1d06480</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fni1564&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni1564%26sid%3Dliteratum%253Aachs%26aulast%3DVenuprasad%26aufirst%3DK.%26aulast%3DHuang%26aufirst%3DH.%26aulast%3DHarada%26aufirst%3DY.%26aulast%3DElly%26aufirst%3DC.%26aulast%3DSubramaniam%26aufirst%3DM.%26aulast%3DSpelsberg%26aufirst%3DT.%26aulast%3DSu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26atitle%3DThe%2520E3%2520ubiquitin%2520ligase%2520Itch%2520regulates%2520expression%2520of%2520transcription%2520factor%2520Foxp3%2520and%2520airway%2520inflammation%2520by%2520enhancing%2520the%2520function%2520of%2520transcription%2520factor%2520TIEG1%26jtitle%3DNat.%2520Immunol%26date%3D2008%26volume%3D9%26spage%3D245%26epage%3D253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Halgren, T.</span><span> </span><span class="NLM_article-title">New method for fast and accurate binding-site identification and analysis</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">–</span> <span class="NLM_lpage">148</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1111%2Fj.1747-0285.2007.00483.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17381729" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2007&pages=146-148&author=T.+Halgren&title=New+method+for+fast+and+accurate+binding-site+identification+and+analysis"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">New method for fast and accurate binding-site identification and analysis</span></div><div class="casAuthors">Halgren, Tom</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-148</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">Structure-based drug design seeks to exploit the structure of protein-ligand or protein-protein binding sites, but the site is not always known at the outset.  Even when the site is known, the researcher may wish to identify alternative prospective binding sites that may result in different biol. effects or new class of compds.  It is also vital in lead optimization to clearly understand the degree to which known binders or docking hits satisfy or violate complementarity to the receptor.  SiteMap is a new technique for identifying potential binding sites and for predicting their druggability in lead-discovery applications and for characterizing binding sites and critically assessing prospective ligands in lead-optimization applications.  In large-scale validation tests, SiteMap correctly identifies the known binding site in > 96% of the cases, with best results (> 98%) coming for sites that bind ligands tightly.  It also accurately distinguishes between sites that bind ligands and sites that don't.  In binding-site anal., SiteMap provides a wealth of quant. and graphical information that can help guide efforts to modify ligand structure to enhance potency or improve phys. properties.  These attributes allow SiteMap to nicely complement techniques such as docking and computational lead optimization in structure-base drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr245hmZWXilrVg90H21EOLACvtfcHk0lgub_R1PLhmuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkt1OnsLg%253D&md5=c233276ba9580393b282f255781f9ce6</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2007.00483.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2007.00483.x%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%26atitle%3DNew%2520method%2520for%2520fast%2520and%2520accurate%2520binding-site%2520identification%2520and%2520analysis%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2007%26volume%3D69%26spage%3D146%26epage%3D148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span> </span><span class="NLM_article-title">Identifying and characterizing binding sites and assessing druggability</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">389</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci800324m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=377-389&author=T.+A.+Halgren&title=Identifying+and+characterizing+binding+sites+and+assessing+druggability"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying and Characterizing Binding Sites and Assessing Druggability</span></div><div class="casAuthors">Halgren, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">377-389</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Identification and characterization of binding sites is key in the process of structure-based drug design.  In some cases there may not be any information about the binding site for a target of interest.  In other cases, a putative binding site has been identified by computational or exptl. means, but the druggability of the target is not known.  Even when a site for a given target is known, it may be desirable to find addnl. sites whose targeting could produce a desired biol. response.  A new program, called SiteMap, is presented for identifying and analyzing binding sites and for predicting target druggability.  In a large-scale validation, SiteMap correctly identifies the known binding site as the top-ranked site in 86% of the cases, with best results (>98%) coming for sites that bind ligands with subnanomolar affinity.  In addn., a modified version of the score employed for binding-site identification allows SiteMap to accurately classify the druggability of proteins as measured by their ability to bind passively absorbed small mols. tightly.  In characterizing binding sites, SiteMap provides quant. and graphical information that can help guide efforts to critically assess virtual hits in a lead-discovery application or to modify ligand structure to enhance potency or improve phys. properties in a lead-optimization context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfAt0-gvcXDLVg90H21EOLACvtfcHk0lgPVQSH0BHzqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXns1OhsA%253D%253D&md5=f03c7b27e30ae03d10e4127ed1f1ed3c</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Fci800324m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800324m%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26atitle%3DIdentifying%2520and%2520characterizing%2520binding%2520sites%2520and%2520assessing%2520druggability%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D377%26epage%3D389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Brickmann, J.; Goetze, T.; Heiden, W.; Moeckel, G.; Reiling, S.; Vollhardt, H.; Zachmann, C.-D.</span><span> </span><span class="NLM_article-title">Interactive Visualization of Molecular Scenarios with the MOLCAD/SYBYL</span>. In  <span class="citation_source-book">Data Visualization in Molecular Science: Tools for Insight and Innovation</span>, <span class="NLM_edition">1</span>st ed.; <span class="NLM_publisher-name">Addison-Wesley</span>: <span class="NLM_publisher-loc">Reading, MA</span>,<span class="NLM_x"> </span><span class="NLM_year">1994</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1994&author=J.+Brickmann&author=T.+Goetze&author=W.+Heiden&author=G.+Moeckel&author=S.+Reiling&author=H.+Vollhardt&author=C.-D.+Zachmann&title=Data+Visualization+in+Molecular+Science%3A+Tools+for+Insight+and+Innovation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBrickmann%26aufirst%3DJ.%26atitle%3DInteractive%2520Visualization%2520of%2520Molecular%2520Scenarios%2520with%2520the%2520MOLCAD%252FSYBYL%26btitle%3DData%2520Visualization%2520in%2520Molecular%2520Science%253A%2520Tools%2520for%2520Insight%2520and%2520Innovation%26pub%3DAddison-Wesley%26date%3D1994" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Soga, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobori, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirayama, N.</span><span> </span><span class="NLM_article-title">Use of amino acid composition to predict ligand-binding sites</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">400</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci6002202" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=400-406&author=S.+Sogaauthor=H.+Shiraiauthor=M.+Koboriauthor=N.+Hirayama&title=Use+of+amino+acid+composition+to+predict+ligand-binding+sites"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Use of Amino Acid Composition to Predict Ligand-Binding Sites</span></div><div class="casAuthors">Soga, Shinji; Shirai, Hiroki; Kobori, Masato; Hirayama, Noriaki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">400-406</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel method for predicting the binding sites for druglike compds. on the surface of proteins was developed on the basis of the specific amino acid compn. obsd. at the ligand-binding sites of ligand-protein complexes detd. by x-ray anal.  A profile representing the preference of each of the 20 std. amino acids at the binding sites of druglike mols. was obtained for a small set of high-quality complex structures.  An index termed propensity for ligand binding (PLB) was created from these profiles.  The PLB index was used to predict the propensity of binding for 804 ligands at all potential binding sites on the proteins whose structures were detd. by x-ray anal.  If the sites with the first 2 highest PLB indexes are taken into consideration, the successfully predicted sites reached a high percentage of 86.  The PLB prediction is relatively simple, but the validation study showed that it is both fast and accurate to detect ligand-binding sites, esp. the binding sites of druglike mols.  Therefore, the PLB index can be used to predict the ligand-binding sites of uncharacterized protein structures and also to identify novel drug-binding sites of known drug targets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGptxKvbII_oZbVg90H21EOLACvtfcHk0lgPVQSH0BHzqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXns1OmsQ%253D%253D&md5=53bddb67710c84f45c99173b0d08d9ce</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1021%2Fci6002202&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci6002202%26sid%3Dliteratum%253Aachs%26aulast%3DSoga%26aufirst%3DS.%26aulast%3DShirai%26aufirst%3DH.%26aulast%3DKobori%26aufirst%3DM.%26aulast%3DHirayama%26aufirst%3DN.%26atitle%3DUse%2520of%2520amino%2520acid%2520composition%2520to%2520predict%2520ligand-binding%2520sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D400%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lhT9jny8NEHYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhT9jny8NEHYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra precision Glide: docking and scoring incorporating a model of hydrophobic enclosure for protein-ligand complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+precision+Glide%3A+docking+and+scoring+incorporating+a+model+of+hydrophobic+enclosure+for+protein-ligand+complexes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0ljJF8QFPd04XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520precision%2520Glide%253A%2520docking%2520and%2520scoring%2520incorporating%2520a%2520model%2520of%2520hydrophobic%2520enclosure%2520for%2520protein-ligand%2520complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Jain, A. N.</span><span> </span><span class="NLM_article-title">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">499</span><span class="NLM_x">–</span> <span class="NLM_lpage">511</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020406h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2003&pages=499-511&author=A.+N.+Jain&title=Surflex%3A+fully+automatic+flexible+molecular+docking+using+a+molecular+similarity-based+search+engine"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Surflex: fully automatic flexible molecular docking using a molecular similarity-based search engine</span></div><div class="casAuthors">Jain, Ajay N.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">499-511</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Surflex is a fully automatic flexible mol. docking algorithm that combines the scoring function from the Hammerhead docking system with a search engine that relies on a surface-based mol. similarity method as a means to rapidly generate suitable putative poses for mol. fragments.  Results are presented evaluating reliability and accuracy of dockings compared with crystallog. exptl. results on 81 protein/ligand pairs of substantial structural diversity.  In over 80% of the complexes, Surflex's highest scoring docked pose was within 2.5 Å root-mean-square deviation (rmsd), with over 90% of the complexes having one of the top ranked poses within 2.5 Å rmsd.  Results are also presented assessing Surflex's utility as a screening tool on two protein targets (thymidine kinase and estrogen receptor) using data sets on which competing methods were run.  Performance of Surflex was significantly better, with true pos. rates of greater than 80% at false pos. rates of less than 1%.  Docking time was roughly linear in no. of rotatable bonds, beginning with a few seconds for rigid mols. and adding approx. 10 s per rotatable bond.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrawn5WHbx5p7Vg90H21EOLACvtfcHk0ljJF8QFPd04XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXkvFKjsA%253D%253D&md5=67a08aafeadf69101d56b2f60a922359</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Fjm020406h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020406h%26sid%3Dliteratum%253Aachs%26aulast%3DJain%26aufirst%3DA.%2BN.%26atitle%3DSurflex%253A%2520fully%2520automatic%2520flexible%2520molecular%2520docking%2520using%2520a%2520molecular%2520similarity-based%2520search%2520engine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2003%26volume%3D46%26spage%3D499%26epage%3D511" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Rarey, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kramer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klebe, G.</span><span> </span><span class="NLM_article-title">A fast flexible docking method using an incremental construction algorithm</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">261</span><span class="NLM_x">, </span> <span class="NLM_fpage">470</span><span class="NLM_x">–</span> <span class="NLM_lpage">489</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1006%2Fjmbi.1996.0477" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=8780787" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=261&publication_year=1996&pages=470-489&author=M.+Rareyauthor=B.+Kramerauthor=T.+Lengauerauthor=G.+Klebe&title=A+fast+flexible+docking+method+using+an+incremental+construction+algorithm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A fast flexible docking method using an incremental construction algorithm</span></div><div class="casAuthors">Rarey, Matthias; Kramer, Bernd; Lengauer, Thomas; Klebe, Gerhard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">261</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">470-489</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Academic</span>)
        </div><div class="casAbstract">We present an automatic method for docking org. ligands into protein binding sites.  The method can be used in the design process of specific protein ligands.  It combines an appropriate model of the physico-chem. properties of the docked mols. with efficient methods for sampling the conformational space of the ligand.  If the ligand is flexible, it can adopt a large variety of different conformations.  Each such min. in conformational space presents a potential candidate for the conformation of the ligand in the complexed state.  Our docking method samples the conformation space of the ligand on the basis of a discrete model and uses a tree-search technique for placing the ligand incrementally into the active site.  For placing the first fragment of the ligand into the protein, we use hashing techniques adapted from computer vision.  The incremental construction algorithm is based on a greedy strategy combined with efficient methods for overlap detection and for the search of new interactions.  We present results on 19 complexes of which the binding geometry has been crystallog. detd.  All considered ligands are docked in at most three minutes on a current workstation.  The exptl. obsd. binding mode of the ligand is reproduced with 0.5 to 1.2 Å rms deviation.  It is almost always found among the highest-ranking conformations computed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrx9N2sFXkOfLVg90H21EOLACvtfcHk0ljJF8QFPd04XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28XltlKisLo%253D&md5=8cd8f384906ed90e894d118399e4dc4a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1996.0477&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1996.0477%26sid%3Dliteratum%253Aachs%26aulast%3DRarey%26aufirst%3DM.%26aulast%3DKramer%26aufirst%3DB.%26aulast%3DLengauer%26aufirst%3DT.%26aulast%3DKlebe%26aufirst%3DG.%26atitle%3DA%2520fast%2520flexible%2520docking%2520method%2520using%2520an%2520incremental%2520construction%2520algorithm%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1996%26volume%3D261%26spage%3D470%26epage%3D489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Sakaguchi, S.</span><span> </span><span class="NLM_article-title">Naturally arising Foxp3-expressing CD25+CD4+ regulatory T cells in immunological tolerance to self and non-self</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">345</span><span class="NLM_x">–</span> <span class="NLM_lpage">352</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni1178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15785760" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXisVyitLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=345-352&author=S.+Sakaguchi&title=Naturally+arising+Foxp3-expressing+CD25%2BCD4%2B+regulatory+T+cells+in+immunological+tolerance+to+self+and+non-self"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Naturally arising Foxp3-expressing CD25+CD4+ regulatory T cells in immunological tolerance to self and non-self</span></div><div class="casAuthors">Sakaguchi, Shimon</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">345-352</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naturally arising CD25+CD4+ regulatory T cells actively maintain immunol. self-tolerance.  Deficiency in or dysfunction of these cells can be a cause of autoimmune disease.  A redn. in their no. or function can also elicit tumor immunity, whereas their antigen-specific population expansion can establish transplantation tolerance.  They are therefore a good target for designing ways to induce or abrogate immunol. tolerance to self and non-self antigens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpo2HWNwsl-erVg90H21EOLACvtfcHk0liwicqczG5yTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXisVyitLY%253D&md5=d873618a9666d7d03e457ce828e85f37</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1038%2Fni1178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni1178%26sid%3Dliteratum%253Aachs%26aulast%3DSakaguchi%26aufirst%3DS.%26atitle%3DNaturally%2520arising%2520Foxp3-expressing%2520CD25%252BCD4%252B%2520regulatory%2520T%2520cells%2520in%2520immunological%2520tolerance%2520to%2520self%2520and%2520non-self%26jtitle%3DNat.%2520Immunol.%26date%3D2005%26volume%3D6%26spage%3D345%26epage%3D352" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Bluestone, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, Q.</span><span> </span><span class="NLM_article-title">How do CD4+CD25+ regulatory T cells control autoimmunity?</span> <span class="citation_source-journal">Curr. Opin Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">638</span><span class="NLM_x">–</span> <span class="NLM_lpage">642</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.coi.2005.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16209918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsb3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2005&pages=638-642&author=J.+A.+Bluestoneauthor=Q.+Tang&title=How+do+CD4%2BCD25%2B+regulatory+T+cells+control+autoimmunity%3F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">How do CD4+CD25+ regulatory T cells control autoimmunity?</span></div><div class="casAuthors">Bluestone, Jeffrey A.; Tang, Qizhi</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">638-642</span>CODEN:
                <span class="NLM_cas:coden">COPIEL</span>;
        ISSN:<span class="NLM_cas:issn">0952-7915</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Any scientist opening up an immunol. journal today will observe immediately that suppressor T cells, renamed regulatory T cells' (Tregs) have become a central concept in the immunol. lexicon.  Hundreds of Treg publications over the past few years have validated the existence of this unique T cell lineage armed with an ability to regulate autoimmunity.  The CD4+CD25+Foxp3+ Treg subset develops in the thymus, can be induced in the periphery during the course of normal immune responses and utilizes a T cell repertoire skewed towards autoantigens.  Despite these advances, however, there is still controversy over their mechanism of action.  This confusion stems from the differences obsd. in in vitro vs. in vivo studies.  Recent in vivo analyses support a model in which Tregs directly or indirectly alter the activation and differentiation of pathogenic T cells through an effect on antigen presenting cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrddm9vwk36N7Vg90H21EOLACvtfcHk0liwicqczG5yTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKhsb3F&md5=0dd48e94a65efcc18508e0a51e9c1127</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.coi.2005.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.coi.2005.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DBluestone%26aufirst%3DJ.%2BA.%26aulast%3DTang%26aufirst%3DQ.%26atitle%3DHow%2520do%2520CD4%252BCD25%252B%2520regulatory%2520T%2520cells%2520control%2520autoimmunity%253F%26jtitle%3DCurr.%2520Opin%2520Immunol.%26date%3D2005%26volume%3D17%26spage%3D638%26epage%3D642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Fontenot, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gavin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rudensky, A. Y.</span><span> </span><span class="NLM_article-title">Foxp3 programs the development and function of CD4+CD25+ regulatory T cells</span> <span class="citation_source-journal">Nat. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">330</span><span class="NLM_x">–</span> <span class="NLM_lpage">336</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1038%2Fni904" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=12612578" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3sXitlyqs74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2003&pages=330-336&author=J.+D.+Fontenotauthor=M.+A.+Gavinauthor=A.+Y.+Rudensky&title=Foxp3+programs+the+development+and+function+of+CD4%2BCD25%2B+regulatory+T+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Foxp3 programs the development and function of CD4+CD25+ regulatory T cells</span></div><div class="casAuthors">Fontenot, Jason D.; Gavin, Marc A.; Rudensky, Alexander Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Immunology</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">330-336</span>CODEN:
                <span class="NLM_cas:coden">NIAMCZ</span>;
        ISSN:<span class="NLM_cas:issn">1529-2908</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">CD4+CD25+ regulatory T cells are essential for the active suppression of autoimmunity.  Here the authors report that the forkhead transcription factor Foxp3 is specifically expressed in CD4+CD25+ regulatory T cells and is required for their development.  The lethal autoimmune syndrome obsd. in Foxp3-mutant scurfy mice and Foxp3-null mice results from a CD4+CD25+ regulatory T cell deficiency and not from a cell-intrinsic defect of CD4+CD25- T cells.  CD4+CD25+ regulatory T cells rescue disease development and preferentially expand when transferred into neonatal Foxp3-deficient mice.  Furthermore, ectopic expression of Foxp3 confers suppressor function on peripheral CD4+CD25- T cells.  Thus, Foxp3 is a crit. regulator of CD4+CD25+ regulatory T cell development and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriHBlgR28euLVg90H21EOLACvtfcHk0lgju9qjQJGNhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXitlyqs74%253D&md5=e923b4054210b2902140bd155e8a7daf</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fni904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fni904%26sid%3Dliteratum%253Aachs%26aulast%3DFontenot%26aufirst%3DJ.%2BD.%26aulast%3DGavin%26aufirst%3DM.%2BA.%26aulast%3DRudensky%26aufirst%3DA.%2BY.%26atitle%3DFoxp3%2520programs%2520the%2520development%2520and%2520function%2520of%2520CD4%252BCD25%252B%2520regulatory%2520T%2520cells%26jtitle%3DNat.%2520Immunol.%26date%3D2003%26volume%3D4%26spage%3D330%26epage%3D336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Piccirillo, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shevach, E. M.</span><span> </span><span class="NLM_article-title">Naturally-occurring CD4+CD25+ immunoregulatory T cells: central players in the arena of peripheral tolerance</span> <span class="citation_source-journal">Semin. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">81</span><span class="NLM_x">–</span> <span class="NLM_lpage">88</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.smim.2003.12.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15036231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXht1Khtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2004&pages=81-88&author=C.+A.+Piccirilloauthor=E.+M.+Shevach&title=Naturally-occurring+CD4%2BCD25%2B+immunoregulatory+T+cells%3A+central+players+in+the+arena+of+peripheral+tolerance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Naturally-occurring CD4+CD25+ immunoregulatory T cells: central players in the arena of peripheral tolerance</span></div><div class="casAuthors">Piccirillo, Ciriaco A.; Shevach, Ethan M.</div><div class="citationInfo"><span class="NLM_cas:title">Seminars in Immunology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">81-88</span>CODEN:
                <span class="NLM_cas:coden">SEIME2</span>;
        ISSN:<span class="NLM_cas:issn">1044-5323</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Self/non-self discrimination is a complex process that involves maintaining tolerance to autoantigens while preserving the potential to generate an effective humoral and cellular immune responses against invading pathogens.  In the last decade, there has been a remarkable resurgence in research on suppressor or regulatory T cells.  Few areas have captivated the attention of immunologists so vividly and entertained so many passionate debates.  Indeed, CD4+CD25+ Treg, which are at the very center of this fascination, have emerged as a dominant T cell population capable of mediating peripheral tolerance to autoantigens, but whose functions have now been extended to regulation of T cell responses directed to foreign antigens.  However, a no. of fundamental questions concerning the origin, phenotypic nature and mechanism of action of CD4+CD25+ Treg cells remain elusive and misunderstood.  We propose the existence of two general subsets of CD4+CD25+ Treg cells, naturally-occurring and induced, that differ in their origin, developmental and activation requirements, and mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJNDe4bgB0tLVg90H21EOLACvtfcHk0lgju9qjQJGNhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXht1Khtb0%253D&md5=00b723fccb0ca5fd7ba341366e27fe67</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2Fj.smim.2003.12.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.smim.2003.12.003%26sid%3Dliteratum%253Aachs%26aulast%3DPiccirillo%26aufirst%3DC.%2BA.%26aulast%3DShevach%26aufirst%3DE.%2BM.%26atitle%3DNaturally-occurring%2520CD4%252BCD25%252B%2520immunoregulatory%2520T%2520cells%253A%2520central%2520players%2520in%2520the%2520arena%2520of%2520peripheral%2520tolerance%26jtitle%3DSemin.%2520Immunol.%26date%3D2004%26volume%3D16%26spage%3D81%26epage%3D88" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Ziegler, S. F.</span><span> </span><span class="NLM_article-title">FOXP3: of mice and men</span> <span class="citation_source-journal">Annu. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">–</span> <span class="NLM_lpage">226</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1146%2Fannurev.immunol.24.021605.090547" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16551248" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XkvFSqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2006&pages=209-226&author=S.+F.+Ziegler&title=FOXP3%3A+of+mice+and+men"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">FOXP3: of mice and men</span></div><div class="casAuthors">Ziegler, Steven F.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Immunology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">209-226</span>CODEN:
                <span class="NLM_cas:coden">ARIMDU</span>;
        ISSN:<span class="NLM_cas:issn">0732-0582</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  The immune system has evolved mechanisms to recognize and eliminate threats, as well as to protect against self-destruction.  Tolerance to self-antigens is generated through two fundamental mechanisms: (a) elimination of self-reactive cells in the thymus during selection and (b) generation of a variety of peripheral regulatory cells to control self-reactive cells that escape the thymus.  It is becoming increasing apparent that a population of thymically derived CD4+ regulatory T cells, exemplified by the expression of the IL-2Rα chain, is essential for the maintenance of peripheral tolerance.  Recent work has shown that the forkhead family transcription factor Foxp3 is critically important for the development and function of the regulatory T cells.  Lack of Foxp3 leads to development of fatal autoimmune lymphoproliferative disease; furthermore, ectopic Foxp3 expression can phenotypically convert effector T cells to regulatory T cells.  This review focuses on Foxp3 expression and function and highlights differences between humans and mice regarding Foxp3 regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqM1xFg4P8Ss7Vg90H21EOLACvtfcHk0lgju9qjQJGNhQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XkvFSqt7s%253D&md5=76ece472511a29ba04d71173ef09ddee</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1146%2Fannurev.immunol.24.021605.090547&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.immunol.24.021605.090547%26sid%3Dliteratum%253Aachs%26aulast%3DZiegler%26aufirst%3DS.%2BF.%26atitle%3DFOXP3%253A%2520of%2520mice%2520and%2520men%26jtitle%3DAnnu.%2520Rev.%2520Immunol.%26date%3D2006%26volume%3D24%26spage%3D209%26epage%3D226" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Cao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Icli, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wara, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feinberg, M. W.</span><span> </span><span class="NLM_article-title">Role of Kruppel-like factors in leukocyte development, function, and disease</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">116</span><span class="NLM_x">, </span> <span class="NLM_fpage">4404</span><span class="NLM_x">–</span> <span class="NLM_lpage">4414</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=4404-4414&author=Z.+Caoauthor=X.+Sunauthor=B.+Icliauthor=A.+K.+Waraauthor=M.+W.+Feinberg&title=Role+of+Kruppel-like+factors+in+leukocyte+development%2C+function%2C+and+disease"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DZ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DIcli%26aufirst%3DB.%26aulast%3DWara%26aufirst%3DA.%2BK.%26aulast%3DFeinberg%26aufirst%3DM.%2BW.%26atitle%3DRole%2520of%2520Kruppel-like%2520factors%2520in%2520leukocyte%2520development%252C%2520function%252C%2520and%2520disease%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D4404%26epage%3D4414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Majmudar, C. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapp, A. K.</span><span> </span><span class="NLM_article-title">Chemical approaches to transcriptional regulation</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">467</span><span class="NLM_x">–</span> <span class="NLM_lpage">474</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2005.08.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16122970" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2005&pages=467-474&author=C.+Y.+Majmudarauthor=A.+K.+Mapp&title=Chemical+approaches+to+transcriptional+regulation"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical approaches to transcriptional regulation</span></div><div class="casAuthors">Majmudar, Chinmay Y.; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">467-474</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Given the correlation between many human diseases and mis-regulated transcription, there is a growing need for mols. that can inhibit or mimic key interactions between transcriptional activators and their binding partners.  Because transcriptional activators typically participate in many different protein-protein binding events, the identification of small mols. or peptides that specifically target individual interactions represents a significant challenge.  In spite of this, several small mols. that preferentially inhibit particular complexes of transcriptional activators or mimic the function of activators have recently been reported.  These mols. serve as excellent mechanistic tools for studying transcription and, further, have outstanding therapeutic potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpeAxupgngKD7Vg90H21EOLACvtfcHk0liY-AB2AintZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVehs7rO&md5=2f2d06fa71d25f30b57c632c99e888ff</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2005.08.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2005.08.012%26sid%3Dliteratum%253Aachs%26aulast%3DMajmudar%26aufirst%3DC.%2BY.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DChemical%2520approaches%2520to%2520transcriptional%2520regulation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2005%26volume%3D9%26spage%3D467%26epage%3D474" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Berg, T.</span><span> </span><span class="NLM_article-title">Inhibition of transcription factors with small organic molecules</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">464</span><span class="NLM_x">–</span> <span class="NLM_lpage">471</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2008.07.023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=18706517" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=464-471&author=T.+Berg&title=Inhibition+of+transcription+factors+with+small+organic+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of transcription factors with small organic molecules</span></div><div class="casAuthors">Berg, Thorsten</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">464-471</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Gene-specific transcription factors are DNA binding proteins that function at the center of the transcriptional machinery to mediate the cellular phenotype in response to extracellular signals.  Owing to the lack of enzymic activities of these intracellular proteins, manipulating the activity of transcription factors by small org. mols. is widely considered to be a difficult task.  In this review, the author discusses general approaches designed to inhibit a subset of transcription factors, referred to as dimeric transcription factors, by cell-permeable small org. mols. that inhibit protein-protein interactions or protein-DNA interactions.  These dimeric transcription factors include hypoxia-inducible factor (HIF)-1, c-Myc, and signal transducer and activator of transcription (STAT)-3.  The data presented here indicate that dimeric transcription factors can be potently and selectively inhibited by drug-like small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqfB82cxqtqjLVg90H21EOLACvtfcHk0liY-AB2AintZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtV2jsrjO&md5=1b0e3890a0091f2ba8e5cac978e9c6ba</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.07.023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.07.023%26sid%3Dliteratum%253Aachs%26aulast%3DBerg%26aufirst%3DT.%26atitle%3DInhibition%2520of%2520transcription%2520factors%2520with%2520small%2520organic%2520molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D464%26epage%3D471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Koehler, A. N.</span><span> </span><span class="NLM_article-title">A complex task? Direct modulation of transcription factors with small molecules</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">331</span><span class="NLM_x">–</span> <span class="NLM_lpage">340</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2Fj.cbpa.2010.03.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=20395165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2010&pages=331-340&author=A.+N.+Koehler&title=A+complex+task%3F+Direct+modulation+of+transcription+factors+with+small+molecules"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A complex task? Direct modulation of transcription factors with small molecules</span></div><div class="casAuthors">Koehler, Angela N.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">331-340</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Transcription factors with aberrant activity in disease are promising yet untested targets for therapeutic development, particularly in oncol.  Directly inhibiting or activating the function of a transcription factor requires specific disruption or recruitment of protein-protein or protein-DNA interactions.  The discovery or design of small mols. that specifically modulate these interactions has thus far proven to be a significant challenge and the protein class is often perceived to be undruggable.  ' This review will summarize recent progress in the development of small-mol. probes of transcription factors and provide evidence to challenge the notion that this important protein class is chem. intractable.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr83_m66fDJsrVg90H21EOLACvtfcHk0liY-AB2AintZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmvVWqsL0%253D&md5=a3c2dfa1aa4edc7e387b5048c87bf17e</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2010.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2010.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DKoehler%26aufirst%3DA.%2BN.%26atitle%3DA%2520complex%2520task%253F%2520Direct%2520modulation%2520of%2520transcription%2520factors%2520with%2520small%2520molecules%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D14%26spage%3D331%26epage%3D340" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Lin, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbas, C. F.,  3rd</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, P. G.</span><span> </span><span class="NLM_article-title">Small-molecule switches for zinc finger transcription factors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">613</span></span><div class="citationLinks">[<a href="/doi/10.1021/ja028408e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2003&pages=612-613&author=Q.+Linauthor=C.+F.+Barbasauthor=P.+G.+Schultz&title=Small-molecule+switches+for+zinc+finger+transcription+factors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fja028408e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja028408e%26sid%3Dliteratum%253Aachs%26aulast%3DLin%26aufirst%3DQ.%26aulast%3DBarbas%26aufirst%3DC.%2BF.%26aulast%3DSchultz%26aufirst%3DP.%2BG.%26atitle%3DSmall-molecule%2520switches%2520for%2520zinc%2520finger%2520transcription%2520factors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2003%26volume%3D125%26spage%3D612%26epage%3D613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Lauth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergstrom, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimokawa, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toftgard, R.</span><span> </span><span class="NLM_article-title">Inhibition of GLI-mediated transcription and tumor cell growth by small-molecule antagonists</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U.S.A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">104</span><span class="NLM_x">, </span> <span class="NLM_fpage">8455</span><span class="NLM_x">–</span> <span class="NLM_lpage">8460</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1073%2Fpnas.0609699104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17494766" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXmtVChtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=8455-8460&author=M.+Lauthauthor=A.+Bergstromauthor=T.+Shimokawaauthor=R.+Toftgard&title=Inhibition+of+GLI-mediated+transcription+and+tumor+cell+growth+by+small-molecule+antagonists"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of GLI-mediated transcription and tumor cell growth by small-molecule antagonists</span></div><div class="casAuthors">Lauth, Matthias; Bergstroem, Asa; Shimokawa, Takashi; Toftgard, Rune</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8455-8460</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The developmentally important Hedgehog (Hh) signaling pathway has recently been implicated in several forms of solid cancer.  Current drug development programs focus on targeting the protooncogene Smoothened, a key transmembrane pathway member.  These drug candidates, albeit promising, do not address the scenario in which pathway activation occurs downstream of Smoothened, as obsd. in cases of medulloblastoma, glioma, pericytoma, breast cancer, and prostate cancer.  A cellular screen for small-mol. antagonists of GLI-mediated transcription, which constitutes the final step in the Hh pathway, revealed two mols. that are able to selectively inhibit GLI-mediated gene transactivation.  We provide genetic evidence of downstream pathway blockade by these compds. and demonstrate the ineffectiveness of upstream antagonists such as cyclopamine in such situations.  Mechanistically, both inhibitors act in the nucleus to block GLI function, and one of them interferes with GLI1 DNA binding in living cells.  Importantly, the discovered compds. efficiently inhibited in vitro tumor cell proliferation in a GLI-dependent manner and successfully blocked cell growth in an in vivo xenograft model using human prostate cancer cells harboring downstream activation of the Hh pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnlBrXj0X1ELVg90H21EOLACvtfcHk0lgELXDOzR-0Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXmtVChtbw%253D&md5=c579b952f8c878fa3058b21d41e16f07</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609699104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609699104%26sid%3Dliteratum%253Aachs%26aulast%3DLauth%26aufirst%3DM.%26aulast%3DBergstrom%26aufirst%3DA.%26aulast%3DShimokawa%26aufirst%3DT.%26aulast%3DToftgard%26aufirst%3DR.%26atitle%3DInhibition%2520of%2520GLI-mediated%2520transcription%2520and%2520tumor%2520cell%2520growth%2520by%2520small-molecule%2520antagonists%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2007%26volume%3D104%26spage%3D8455%26epage%3D8460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kroemer, R. T.</span><span> </span><span class="NLM_article-title">Structure-based drug design: docking and scoring</span> <span class="citation_source-journal">Curr. Protein Pept. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">312</span><span class="NLM_x">–</span> <span class="NLM_lpage">328</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.2174%2F138920307781369382" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=17696866" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=312-328&author=R.+T.+Kroemer&title=Structure-based+drug+design%3A+docking+and+scoring"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based drug design: docking and scoring</span></div><div class="casAuthors">Kroemer, Romano T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Protein and Peptide Science</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">312-328</span>CODEN:
                <span class="NLM_cas:coden">CPPSCM</span>;
        ISSN:<span class="NLM_cas:issn">1389-2037</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">This review gives an introduction into ligand - receptor docking and illustrates the basic underlying concepts.  An overview of different approaches and algorithms is provided.  Although the application of docking and scoring has led to some remarkable successes, there are still some major challenges ahead, which are outlined here as well.  Approaches to address some of these challenges and the latest developments in the area are presented.  Some aspects of the assessment of docking program performance are discussed.  A no. of successful applications of structure-based virtual screening are described.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbXWKN0_d9g7Vg90H21EOLACvtfcHk0lgELXDOzR-0Ow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpslKrs7w%253D&md5=5bfb61ca9c4b405a98f50ddc2bb00ed7</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.2174%2F138920307781369382&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F138920307781369382%26sid%3Dliteratum%253Aachs%26aulast%3DKroemer%26aufirst%3DR.%2BT.%26atitle%3DStructure-based%2520drug%2520design%253A%2520docking%2520and%2520scoring%26jtitle%3DCurr.%2520Protein%2520Pept.%2520Sci.%26date%3D2007%26volume%3D8%26spage%3D312%26epage%3D328" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span> </span><span class="NLM_article-title">On the role of the crystal environment in determining protein side-chain conformations</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">320</span><span class="NLM_x">, </span> <span class="NLM_fpage">597</span><span class="NLM_x">–</span> <span class="NLM_lpage">608</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1016%2FS0022-2836%2802%2900470-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=12096912" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=320&publication_year=2002&pages=597-608&author=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=Z.+Xiangauthor=B.+Honig&title=On+the+role+of+the+crystal+environment+in+determining+protein+side-chain+conformations"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">On the Role of the Crystal Environment in Determining Protein Side-chain Conformations</span></div><div class="casAuthors">Jacobson, Matthew P.; Friesner, Richard A.; Xiang, Zhexin; Honig, Barry</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">320</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">597-608</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ltd.</span>)
        </div><div class="casAbstract">The role of crystal packing in detg. the obsd. conformations of amino acid side-chains in protein crystals is investigated by (1) anal. of a database of proteins that have been crystd. in different unit cells (space group or unit cell dimensions) and (2) theor. predictions of side-chain conformations with the crystal environment explicitly represented.  Both of these approaches indicate that the crystal environment plays an important role in detg. the conformations of polar side-chains on the surfaces of proteins.  Inclusion of the crystal environment permits a more sensitive measurement of the achievable accuracy of side-chain prediction programs, when validating against structures obtained by x-ray crystallog.  Our side-chain prediction program uses an all-atom force field and a Generalized Born model of solvation and is thus capable of modeling simple packing effects (i.e. van der Waals interactions), electrostatic effects, and desolvation, which are all important mechanisms by which the crystal environment impacts obsd. side-chain conformations.  Our results are also relevant to the understanding of changes in side-chain conformation that may result from ligand docking and protein-protein assocn., insofar as the results reveal how side-chain conformations change in response to their local environment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgHEOVpIEiErVg90H21EOLACvtfcHk0ljl1E2dDi3EYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XltVKmu70%253D&md5=006de6bd2d0f233ab32d6798dc1a3fbc</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2802%2900470-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252802%252900470-9%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DXiang%26aufirst%3DZ.%26aulast%3DHonig%26aufirst%3DB.%26atitle%3DOn%2520the%2520role%2520of%2520the%2520crystal%2520environment%2520in%2520determining%2520protein%2520side-chain%2520conformations%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D320%26spage%3D597%26epage%3D608" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Jacobson, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pincus, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rapp, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Day, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honig, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span> <span class="citation_source-journal">Proteins</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">351</span><span class="NLM_x">–</span> <span class="NLM_lpage">367</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Å for 5 residue loops, 1.00/0.44 Å for 8 residue loops, and 2.47/1.83 Å for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Å for 5 residue loops, 0.84 Å for 8 residue loops, and 1.63 Å for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0liJzH23a3OSwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Kaminski, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tirado-Rives, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jorgensen, W. L.</span><span> </span><span class="NLM_article-title">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span> <span class="citation_source-journal">J. Phys. Chem. B</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">6474</span><span class="NLM_x">–</span> <span class="NLM_lpage">6487</span></span><div class="citationLinks">[<a href="/doi/10.1021/jp003919d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2001&pages=6474-6487&author=G.+A.+Kaminskiauthor=R.+A.+Friesnerauthor=J.+Tirado-Rivesauthor=W.+L.+Jorgensen&title=Evaluation+and+Reparametrization+of+the+OPLS-AA+Force+Field+for+Proteins+via+Comparison+with+Accurate+Quantum+Chemical+Calculations+on+Peptides"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation and Reparametrization of the OPLS-AA Force Field for Proteins via Comparison with Accurate Quantum Chemical Calculations on Peptides</span></div><div class="casAuthors">Kaminski, George A.; Friesner, Richard A.; Tirado-Rives, Julian; Jorgensen, William L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Physical Chemistry B</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">6474-6487</span>CODEN:
                <span class="NLM_cas:coden">JPCBFK</span>;
        ISSN:<span class="NLM_cas:issn">1089-5647</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present results of improving the OPLS-AA force field for peptides by means of refitting the key Fourier torsional coeffs.  The fitting technique combines using accurate ab initio data as the target, choosing an efficient fitting subspace of the whole potential-energy surface, and detg. wts. for each of the fitting points based on magnitudes of the potential-energy gradient.  The av. energy RMS deviation from the LMP2/cc-pVTZ(-f)//HF/6-31G** data is reduced by ∼40% from 0.81 to 0.47 kcal/mol as a result of the fitting for the electrostatically uncharged dipeptides.  Transferability of the parameters is demonstrated by using the same alanine dipeptide-fitted backbone torsional parameters for all of the other dipeptides (with the appropriate side-chain refitting) and the alanine tetrapeptide.  Parameters of nonbonded interactions have also been refitted for the sulfur-contg. dipeptides (cysteine and methionine), and the validity of the new Coulombic charges and the van der Waals σ's and ε's is proved through reproducing gas-phase energies of complex formation heats of vaporization and densities of pure model liqs.  Moreover, a novel approach to fitting torsional parameters for electrostatically charged mol. systems has been presented and successfully tested on five dipeptides with charged side chains.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSHwmffIQePbVg90H21EOLACvtfcHk0liJzH23a3OSwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXislKhsLk%253D&md5=3ff059626977ee7f6342466f5820f5b7</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjp003919d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp003919d%26sid%3Dliteratum%253Aachs%26aulast%3DKaminski%26aufirst%3DG.%2BA.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DTirado-Rives%26aufirst%3DJ.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26atitle%3DEvaluation%2520and%2520Reparametrization%2520of%2520the%2520OPLS-AA%2520Force%2520Field%2520for%2520Proteins%2520via%2520Comparison%2520with%2520Accurate%2520Quantum%2520Chemical%2520Calculations%2520on%2520Peptides%26jtitle%3DJ.%2520Phys.%2520Chem.%2520B%26date%3D2001%26volume%3D105%26spage%3D6474%26epage%3D6487" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Kontoyianni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosnick, C. B.</span><span> </span><span class="NLM_article-title">Functional prediction of binding pockets</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">824</span><span class="NLM_x">–</span> <span class="NLM_lpage">833</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2005912" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC38XisVyit7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=824-833&author=M.+Kontoyianniauthor=C.+B.+Rosnick&title=Functional+prediction+of+binding+pockets"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Functional Prediction of Binding Pockets</span></div><div class="casAuthors">Kontoyianni, Maria; Rosnick, Christopher B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">824-833</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Putative function for targets with no known ligands has typically been detd. from liganded homologous proteins using sequence and structure comparisons.  However, it is debatable what percentage of sequence identity implies similar function, whereas structural similarity is focused on global folds and could miss divergent structures and novel global folds.  The present study describes an approach to classify a diverse set of proteins and predict their function.  Descriptors corresponding to structural, physicochem., and geometric properties of the ligand-binding cavities of a collection of 434 complexes (17 protein families) were calcd. and analyzed by statistical methods.  The best model using discriminant function anal. (DFA) consisted of 371 proteins (15 families) and had correct classification rates of 90% and cross-validation 86%.  DFA with one protein and a random sample of the remaining proteins led to 100% correct prediction of putative protein function for 10 of the 15 protein families.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLFlWhIcocHrVg90H21EOLACvtfcHk0ljruQeavEK6kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XisVyit7g%253D&md5=b1d043af08019a8c911941d3bd069ffe</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fci2005912&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2005912%26sid%3Dliteratum%253Aachs%26aulast%3DKontoyianni%26aufirst%3DM.%26aulast%3DRosnick%26aufirst%3DC.%2BB.%26atitle%3DFunctional%2520prediction%2520of%2520binding%2520pockets%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D824%26epage%3D833" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Ho, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marshall, G. R.</span><span> </span><span class="NLM_article-title">Cavity search: an algorithm for the isolation and display of cavity-like binding regions</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">337</span><span class="NLM_x">–</span> <span class="NLM_lpage">354</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1990&pages=337-354&author=C.+M.+Hoauthor=G.+R.+Marshall&title=Cavity+search%3A+an+algorithm+for+the+isolation+and+display+of+cavity-like+binding+regions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DC.%2BM.%26aulast%3DMarshall%26aufirst%3DG.%2BR.%26atitle%3DCavity%2520search%253A%2520an%2520algorithm%2520for%2520the%2520isolation%2520and%2520display%2520of%2520cavity-like%2520binding%2520regions%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1990%26volume%3D4%26spage%3D337%26epage%3D354" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Irwin, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sterling, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mysinger, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolstad, E. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coleman, R. G.</span><span> </span><span class="NLM_article-title">ZINC: a free tool to discover chemistry for biology</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">1757</span><span class="NLM_x">–</span> <span class="NLM_lpage">1768</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci3001277" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=1757-1768&author=J.+J.+Irwinauthor=T.+Sterlingauthor=M.+M.+Mysingerauthor=E.+S.+Bolstadauthor=R.+G.+Coleman&title=ZINC%3A+a+free+tool+to+discover+chemistry+for+biology"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">ZINC: A Free Tool to Discover Chemistry for Biology</span></div><div class="casAuthors">Irwin, John J.; Sterling, Teague; Mysinger, Michael M.; Bolstad, Erin S.; Coleman, Ryan G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1757-1768</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">ZINC is a free public resource for ligand discovery.  The database contains over twenty million com. available mols. in biol. relevant representations that may be downloaded in popular ready-to-dock formats and subsets.  The Web site also enables searches by structure, biol. activity, phys. property, vendor, catalog no., name, and CAS no.  Small custom subsets may be created, edited, shared, docked, downloaded, and conveyed to a vendor for purchase.  The database is maintained and curated for a high purchasing success rate and is freely available at zinc.docking.org.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotyhntxgvRpbVg90H21EOLACvtfcHk0ljruQeavEK6kw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvFGnsrg%253D&md5=97f2ede64afc6b5e3ea2f279e38e32a0</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fci3001277&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3001277%26sid%3Dliteratum%253Aachs%26aulast%3DIrwin%26aufirst%3DJ.%2BJ.%26aulast%3DSterling%26aufirst%3DT.%26aulast%3DMysinger%26aufirst%3DM.%2BM.%26aulast%3DBolstad%26aufirst%3DE.%2BS.%26aulast%3DColeman%26aufirst%3DR.%2BG.%26atitle%3DZINC%253A%2520a%2520free%2520tool%2520to%2520discover%2520chemistry%2520for%2520biology%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D1757%26epage%3D1768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lhiXZcKhFI9Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smondyrev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span> </span><span class="NLM_article-title">PHASE: a new engine for pharmacophore perception, 3D QSAR model development, and 3D database screening: 1. Methodology and preliminary results</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">647</span><span class="NLM_x">–</span> <span class="NLM_lpage">671</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2006&pages=647-671&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=E.+H.+Knollauthor=S.+N.+Raoauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=PHASE%3A+a+new+engine+for+pharmacophore+perception%2C+3D+QSAR+model+development%2C+and+3D+database+screening%3A+1.+Methodology+and+preliminary+results"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DRao%26aufirst%3DS.%2BN.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DPHASE%253A%2520a%2520new%2520engine%2520for%2520pharmacophore%2520perception%252C%25203D%2520QSAR%2520model%2520development%252C%2520and%25203D%2520database%2520screening%253A%25201.%2520Methodology%2520and%2520preliminary%2520results%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2006%26volume%3D20%26spage%3D647%26epage%3D671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Dixon, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smondyrev, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, S. N.</span><span> </span><span class="NLM_article-title">PHASE: a novel approach to pharmacophore modeling and 3D database searching</span> <span class="citation_source-journal">Chem. Biol. Drug Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">67</span><span class="NLM_x">, </span> <span class="NLM_fpage">370</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Fjm5018187&amp;key=10.1111%2Fj.1747-0285.2006.00384.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Fjm5018187&amp;key=16784462" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Fjm5018187&amp;key=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2006&pages=370-372&author=S.+L.+Dixonauthor=A.+M.+Smondyrevauthor=S.+N.+Rao&title=PHASE%3A+a+novel+approach+to+pharmacophore+modeling+and+3D+database+searching"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Novel approach to pharmacophore modeling and 3D database searching</span></div><div class="casAuthors">Dixon, Steven L.; Smondyrev, Alexander M.; Rao, Shashidhar N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">370-372</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">A review.  Pharmacophore modeling and 3D database searching are now recognized as integral components of lead discovery and lead optimization, and the continuing need for improved pharmacophore-based tools has driven the development of PHASE.  PHASE pharmacophore and 3D QSAR models provide a means for rationalizing structure-activity data, predicting the activity of new compds., and efficiently extg. actives from a prototypical database.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo50KpRHTGgAbVg90H21EOLACvtfcHk0lhiXZcKhFI9Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotVeqsbg%253D&md5=30d06fe99776e203eed35b0f774bbe64</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1111%2Fj.1747-0285.2006.00384.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1747-0285.2006.00384.x%26sid%3Dliteratum%253Aachs%26aulast%3DDixon%26aufirst%3DS.%2BL.%26aulast%3DSmondyrev%26aufirst%3DA.%2BM.%26aulast%3DRao%26aufirst%3DS.%2BN.%26atitle%3DPHASE%253A%2520a%2520novel%2520approach%2520to%2520pharmacophore%2520modeling%2520and%25203D%2520database%2520searching%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2006%26volume%3D67%26spage%3D370%26epage%3D372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Yet, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McA’Nulty, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Folta, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yen, H. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoshizumi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsieh, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Layne, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrella, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jain, M. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. E.</span><span> </span><span class="NLM_article-title">Human EZF, a Kruppel-like zinc finger protein, is expressed in vascular endothelial cells and contains transcriptional activation and repression domains</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">1026</span><span class="NLM_x">–</span> <span class="NLM_lpage">1031</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=1026-1031&author=S.+F.+Yetauthor=M.+M.+McA%E2%80%99Nultyauthor=S.+C.+Foltaauthor=H.+W.+Yenauthor=M.+Yoshizumiauthor=C.+M.+Hsiehauthor=M.+D.+Layneauthor=M.+T.+Chinauthor=H.+Wangauthor=M.+A.+Perrellaauthor=M.+K.+Jainauthor=M.+E.+Lee&title=Human+EZF%2C+a+Kruppel-like+zinc+finger+protein%2C+is+expressed+in+vascular+endothelial+cells+and+contains+transcriptional+activation+and+repression+domains"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYet%26aufirst%3DS.%2BF.%26aulast%3DMcA%25E2%2580%2599Nulty%26aufirst%3DM.%2BM.%26aulast%3DFolta%26aufirst%3DS.%2BC.%26aulast%3DYen%26aufirst%3DH.%2BW.%26aulast%3DYoshizumi%26aufirst%3DM.%26aulast%3DHsieh%26aufirst%3DC.%2BM.%26aulast%3DLayne%26aufirst%3DM.%2BD.%26aulast%3DChin%26aufirst%3DM.%2BT.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DPerrella%26aufirst%3DM.%2BA.%26aulast%3DJain%26aufirst%3DM.%2BK.%26aulast%3DLee%26aufirst%3DM.%2BE.%26atitle%3DHuman%2520EZF%252C%2520a%2520Kruppel-like%2520zinc%2520finger%2520protein%252C%2520is%2520expressed%2520in%2520vascular%2520endothelial%2520cells%2520and%2520contains%2520transcriptional%2520activation%2520and%2520repression%2520domains%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D1026%26epage%3D1031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Chrisman, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tindall, D. J.</span><span> </span><span class="NLM_article-title">Identification and characterization of a consensus DNA binding element for the zinc finger transcription factor TIEG/EGRalpha</span> <span class="citation_source-journal">DNA Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">187</span><span class="NLM_x">–</span> <span class="NLM_lpage">199</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2003&pages=187-199&author=H.+R.+Chrismanauthor=D.+J.+Tindall&title=Identification+and+characterization+of+a+consensus+DNA+binding+element+for+the+zinc+finger+transcription+factor+TIEG%2FEGRalpha"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChrisman%26aufirst%3DH.%2BR.%26aulast%3DTindall%26aufirst%3DD.%2BJ.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520a%2520consensus%2520DNA%2520binding%2520element%2520for%2520the%2520zinc%2520finger%2520transcription%2520factor%2520TIEG%252FEGRalpha%26jtitle%3DDNA%2520Cell%2520Biol.%26date%3D2003%26volume%3D22%26spage%3D187%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MEY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1MEY','PDB','1MEY'); return false;">PDB: 1MEY</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i24"><a href="/doi/suppl/10.1021/jm5018187">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_37581"></div></div></div></div></div><hr /></hr><p class="last">Supplemental Figures 1–3 and supplemental Tables 1–3 that include the raw data (flow cytometry and RT-PCR) used to generate Figures <a id="rightTab-" class=" internalNav" href="#fig4">4</a> and <a id="rightTab-" class=" internalNav" href="#fig5">5</a> (inhibition of conversion of CD4+CD25– T cells to CD4+CD25+ T regs by small molecule compounds), structures of previously reported inhibitors of another transcription factor GLI, the analytical data that support purity of compounds #48, #48-15, and #15-09, and the list of molecules from the SBVS-150 set. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm5018187/suppl_file/jm5018187_si_001.pdf">jm5018187_si_001.pdf (20.13 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm5018187&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Barticle%3Aarticle%3A10.1021%2Fjm5018187%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-3" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm5018187" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                2MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0f697cec52388","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
