
<html lang="en"     class="pb-page"  data-request-id="0e6b85ea-d9f4-4e93-802e-5239faec21aa"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.8b01749;issue:issue:10.1021/jmcmar.2019.62.issue-14;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Allosteric Modulator Discovery: From Serendipity to Structure-Based Design" /></meta><meta name="dc.Creator" content="Shaoyong  Lu" /></meta><meta name="dc.Creator" content="Xinheng  He" /></meta><meta name="dc.Creator" content="Duan  Ni" /></meta><meta name="dc.Creator" content="Jian  Zhang" /></meta><meta name="dc.Description" content="Allosteric modulators bound to structurally diverse allosteric sites can achieve better pharmacological advantages than orthosteric ligands. The discovery of allosteric modulators, however, has bee..." /></meta><meta name="Description" content="Allosteric modulators bound to structurally diverse allosteric sites can achieve better pharmacological advantages than orthosteric ligands. The discovery of allosteric modulators, however, has bee..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 28, 2019" /></meta><meta name="dc.Type" content="review-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.8b01749" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01749" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.8b01749" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.8b01749" /></link>
        
    
    

<title>Allosteric Modulator Discovery: From Serendipity to Structure-Based Design | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.8b01749" /></meta><meta property="og:title" content="Allosteric Modulator Discovery: From Serendipity to Structure-Based Design" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0014.jpeg" /></meta><meta property="og:description" content="Allosteric modulators bound to structurally diverse allosteric sites can achieve better pharmacological advantages than orthosteric ligands. The discovery of allosteric modulators, however, has been traditionally serendipitous, owing to the complex nature of allosteric modulation. Recent advances in the understanding of allosteric regulatory mechanisms and remarkable availability of structural data of allosteric proteins and modulators, as well as their complexes, have greatly contributed to the development of various computational approaches to guide the structure-based discovery of allosteric modulators. This Perspective first outlines the evolution of the allosteric concept and describes the advantages and hurdles facing allosteric modulator discovery. The current available computational approaches, together with experimental approaches aiming to highlight allosteric studies, are then highlighted, emphasizing successful examples with their combined applications. We aimed to increase the awareness of the feasibility of the structure-based discovery of allosteric modulators using an integrated computational and experimental paradigm." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.8b01749"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.8b01749">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.8b01749&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.8b01749&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.8b01749&amp;href=/doi/10.1021/acs.jmedchem.8b01749" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6405-6421</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><span class="content-navigation__contentType">Perspective</span><a href="/doi/10.1021/acs.jmedchem.8b01760" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Allosteric Modulator Discovery: From Serendipity to Structure-Based Design</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Shaoyong Lu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shaoyong Lu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div><div class="loa-info-affiliations-info">Medicinal Bioinformatics Center, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shaoyong++Lu">Shaoyong Lu</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath1">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xinheng He</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xinheng He</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xinheng++He">Xinheng He</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath2">View Biography</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Duan Ni</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Duan Ni</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Duan++Ni">Duan Ni</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath3">View Biography</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Jian Zhang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Jian Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Key Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div><div class="loa-info-affiliations-info">Medicinal Bioinformatics Center, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: +86-21-63846590-776922. Fax: +86-21-64154900. E-mail: <a href="/cdn-cgi/l/email-protection#1f75767e713165777e71785f6c756b6a317a7b6a317c71"><span class="__cf_email__" data-cfemail="274d4e4649095d4f46494067544d535209424352094449">[email protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Jian++Zhang">Jian Zhang</a></span></div><div class="loa-info-bio"><span><a class="internalNav" href="#ath4">View Biography</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6558-791X" title="Orcid link">http://orcid.org/0000-0002-6558-791X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01749&amp;href=/doi/10.1021%2Facs.jmedchem.8b01749" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6405–6421</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">February 28, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 November 2018</li><li><span class="item_label"><b>Published</b> online</span>28 February 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 July 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.8b01749" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01749</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6405%26pageCount%3D17%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DShaoyong%2BLu%252C%2BXinheng%2BHe%252C%2BDuan%2BNi%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D14%26contentID%3Dacs.jmedchem.8b01749%26title%3DAllosteric%2BModulator%2BDiscovery%253A%2BFrom%2BSerendipity%2Bto%2BStructure-Based%2BDesign%26numPages%3D17%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6421%26publicationDate%3DJuly%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.8b01749"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">7384</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">48</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.8b01749" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Allosteric Modulator Discovery: From Serendipity to Structure-Based Design&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Shaoyong&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Xinheng&quot;,&quot;last_name&quot;:&quot;He&quot;},{&quot;first_name&quot;:&quot;Duan&quot;,&quot;last_name&quot;:&quot;Ni&quot;},{&quot;first_name&quot;:&quot;Jian&quot;,&quot;last_name&quot;:&quot;Zhang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;28&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;6405-6421&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.8b01749&quot;},&quot;abstract&quot;:&quot;Allosteric modulators bound to structurally diverse allosteric sites can achieve better pharmacological advantages than orthosteric ligands. The discovery of allosteric modulators, however, has been traditionally serendipitous, owing to the complex nature of allosteric modulation. Recent advances in the understanding of allosteric regulatory mechanisms and remarkable availability of structural data of allosteric proteins and modulators, as well as their complexes, have greatly contributed to the development of various computational approaches to guide the structure-based discovery of allosteric modulators. This Perspective first outlines the evolution of the allosteric concept and describes the advantages and hurdles facing allosteric modulator discovery. The current available computational approaches, together with experimental approaches aiming to highlight allosteric studies, are then highlighted, emphasizing successful examples with their combined applications. We aimed to increase the awareness of the fe&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01749&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01749" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01749&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01749" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.8b01749&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01749" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.8b01749&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.8b01749&amp;href=/doi/10.1021/acs.jmedchem.8b01749" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.8b01749" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.8b01749" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.8b01749%26sid%3Dliteratum%253Aachs%26pmid%3D30817889%26genre%3Darticle%26aulast%3DLu%26date%3D2019%26atitle%3DAllosteric%2BModulator%2BDiscovery%253A%2BFrom%2BSerendipity%2Bto%2BStructure-Based%2BDesign%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D14%26spage%3D6405%26epage%3D6421%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291631" title="Modulators">Modulators</a>,</li><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/jmcmar.2019.62.issue-14/20190725/jmcmar.2019.62.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Allosteric modulators bound to structurally diverse allosteric sites can achieve better pharmacological advantages than orthosteric ligands. The discovery of allosteric modulators, however, has been traditionally serendipitous, owing to the complex nature of allosteric modulation. Recent advances in the understanding of allosteric regulatory mechanisms and remarkable availability of structural data of allosteric proteins and modulators, as well as their complexes, have greatly contributed to the development of various computational approaches to guide the structure-based discovery of allosteric modulators. This Perspective first outlines the evolution of the allosteric concept and describes the advantages and hurdles facing allosteric modulator discovery. The current available computational approaches, together with experimental approaches aiming to highlight allosteric studies, are then highlighted, emphasizing successful examples with their combined applications. We aimed to increase the awareness of the feasibility of the structure-based discovery of allosteric modulators using an integrated computational and experimental paradigm.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">1.  Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_83731" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_83731" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Allostery is an inherent property of biomacromolecules in which topographically distinct binding sites within a protein are functionally coupled and can communicate over a distance.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a> Typically, the allosteric signal propagation from allosteric to functional, orthosteric sites is elicited by perturbations at the allosteric site such as effector (ions, small molecules, proteins, DNA, and RNA) binding, point mutations, and posttranslational modifications.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> The existence of allosteric modulation in the control of biomacromolecule function allows for exquisite control of innumerable biological processes, ranging from enzyme catalysis, gene expression, and cellular differentiation to metabolism and homeostasis.<a onclick="showRef(event, 'ref6 ref7 ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8 ref9 ref10">(6−10)</a> Because of the omnipresence of allosteric control over normal cellular processes, allosteric modulation has thus been regarded as the “second secret of life”.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> This notion has been well established as a fundamental biological phenomenon for insights into cellular functions and diseases.<a onclick="showRef(event, 'ref5 ref12'); return false;" href="javascript:void(0);" class="ref ref5 ref12">(5,12)</a></div><div class="NLM_p">From a pharmaceutical standpoint, protein allostery provides a promising, novel opportunity for the development of innovative therapeutics. Owing to the nonoverlap of allosteric and orthosteric sites, allosteric modulators, by attaching to allosteric sites, did not compete with endogenous or exogenous ligands bound to orthosteric sites. This feature guarantees the cooperative regulation of protein function by both allosteric modulators and orthosteric ligands in individual proteins.<a onclick="showRef(event, 'ref13 ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref13 ref14 ref15">(13−15)</a> Moreover, the much more diversified allosteric sites relative to the highly conserved orthosteric sites endow allosteric modulators with potentially enhanced selectivity and reduced toxicity than orthosteric ligands.<a onclick="showRef(event, 'ref16 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref16 ref17 ref18">(16−18)</a></div><div class="NLM_p">Despite the potential advantages of allosteric therapeutics, allosteric modulator discovery has encountered a key challenge. Over the years, most allosteric modulators have been discovered serendipitously by high-throughput screening experiments.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Such unexpected findings manifest a lack of a comprehensive understanding of allosteric interactions and their effects on the molecular mechanism of proteins. Fortunately, structural data pertaining to allosteric proteins and their allosteric sites and modulators have become more available in the past few years.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Such data are beneficial to develop computational approaches to predict allosteric interactions and mechanisms and subsequently screen allosteric modulators for the intended allosteric sites. Mounting evidence suggests that allosteric modulators can be discovered in conjunction with computational and experimental methods, indicative of the advent of structure-based allosteric modulator discovery.<a onclick="showRef(event, 'ref21 ref22 ref23 ref24 ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref23 ref24 ref25 ref26 ref27">(21−27)</a></div><div class="NLM_p last">In this Perspective, we briefly overview the evolution of the allosteric concept and describe the advantages and hurdles facing allosteric modulator discovery. Importantly, we elucidate the currently available computational methods that complement experimental methods and provide powerful tools to facilitate allosteric modulator discovery. We mainly focus on the identification of allosteric modulators guided by computational methods, followed by experimental validations. We outline some guidelines to increase the awareness of the structure-based discovery of allosteric modulators using an integrated computational and experimental paradigm. Finally, we discuss the future directions that can contribute to allosteric modulator discovery.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">2.  Past and Present of the Allosteric Concept</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10713" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10713" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The pioneering work by Bohr in 1904 uncovered an allosteric phenomenon where the binding of carbon dioxide influences the binding affinity of oxygen to hemoglobin. This well-recognized phenomenon describing the cooperative binding of molecules to two distinct sites is known as the “Bohr effect”, which is currently named the “allosteric effect”.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a></div><div class="NLM_p">In 1961, Monod and Jacob first proposed the concept of allostery to account for the mechanism of feedback inhibition of enzyme activity in which “the inhibitor is not a steric analogue of the substrate”.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Later, in the 1960s, two classic models of allosteric interactions were created to explain the cooperative binding of regulatory molecules; one is the “concerted” or MWC (Monod–Wyman–Changeux) (1965)<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> model, and the other is the “sequential” or KNF (Koshland–Nemethy–Filmer) (1966)<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> model. Both models underscore the role of conformational changes in the allosteric regulation of two end-state structures.</div><div class="NLM_p">In 1984, Cooper and Dryden proposed a theoretical model named “dynamics-driven allostery” by which allostery occurs without significant conformational changes in the average structure, revealing the importance of the entropic contribution to allostery.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Advances in solution nuclear magnetic resonance (NMR) spectroscopy<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> have substantially contributed to probe “dynamics/entropy-driven allostery” through the characterization of protein internal motions such as in the catabolite activator protein (CAP),<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> PDZ domains,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> calmodulin,<a onclick="showRef(event, 'ref36 ref37 ref38'); return false;" href="javascript:void(0);" class="ref ref36 ref37 ref38">(36−38)</a> and protein kinase A.<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> Dramatically, Frederick et al. found a linear relationship between the protein conformational entropy and change in the overall binding entropy,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> revealing the important role of conformational entropy in the recognition of proteins.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> They further developed an empirical, quantitative method to describe the relationship between changes in conformational dynamics characterized by the motion of side chains and changes in protein conformational entropy.<a onclick="showRef(event, 'ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref41 ref42">(41,42)</a></div><div class="NLM_p">In 1999, Nussinov and co-workers proposed a “conformational selection and population shift” model that explains how allostery works from the viewpoint of free energy landscape theory.<a onclick="showRef(event, 'ref43 ref44 ref45'); return false;" href="javascript:void(0);" class="ref ref43 ref44 ref45">(43−45)</a> This model broadens the concept of allostery from two structural states to conformational ensembles of multiple states. In the same year, Ranganathan and co-workers proposed an “allosteric networks” model by which signal transmission between distinct sites of proteins is achieved by a network of physically interconnected residues.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> This structural view of allostery has been demonstrated by numerous studies using high-resolution structures.<a onclick="showRef(event, 'ref47 ref48 ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref47 ref48 ref49 ref50">(47−50)</a></div><div class="NLM_p">Recently, Hilser and co-workers have proposed an “ensemble allosteric” model to explain the origin of allostery.<a onclick="showRef(event, 'ref51 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53">(51−53)</a> The Hilser’s model is similar to the Nussinov’s model in the explanation of the allosteric origin. Both models assume that all possible conformations of a protein ensemble are populated in terms of their respective energies, and the binding of allosteric effectors to the protein reshapes the free energy landscape. However, Hilser’s model further extends Nussinov’s model to account for allostery in intrinsically disordered proteins; the former provides a framework that unifies the illustrations of allostery in structured, dynamic, and disordered systems.</div><div class="NLM_p last">As mentioned above, allosteric concepts and models have advanced over 50 years since the word “allostery” was coined by Monod and Jacob.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> It is obvious that the concept of allostery has evolved from the two-state model, the static structural model to the dynamic ensemble model.<a onclick="showRef(event, 'ref54 ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref54 ref55 ref56">(54−56)</a> Importantly, some models have been applied to develop computational methods to predict allosteric sites and mutations, discover allosteric modulators, assess allosteric interactions, and investigate allosteric mechanisms, all of which will be elaborated in the following sections.</div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">3.  Advantages and Challenges Facing Allosteric Modulator Discovery</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5">3.1.  Advantages</h3><div class="NLM_p">Allosteric modulators harbor several distinct advantages over orthosteric ligands that occupy a protein’s active site. The advantageous characteristics of allosteric modulators can be witnessed by their high specificity and low adverse effects.<a onclick="showRef(event, 'ref57 ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref57 ref58 ref59">(57−59)</a> G-protein-coupled receptors (GPCRs) and protein kinases are the two most important drug targets in therapeutic pharmacology. The highly evolutionarily conserved endogenous orthosteric binding sites represent a key challenge to develop selective orthosteric drugs for GPCRs and protein kinases. It is recognized that orthosteric ligands of an intended GPCR or protein kinase frequently suffer from cross-reactivities with its homologous proteins, leading to unwanted side effects and off-target toxicity.<a onclick="showRef(event, 'ref16 ref60'); return false;" href="javascript:void(0);" class="ref ref16 ref60">(16,60)</a> As an alternative, targeting the structural diversity and topological difference of allosteric sites enables one to surmount two major obstacles encountered by orthosteric ligands.<a onclick="showRef(event, 'ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref61 ref62">(61,62)</a> For instance, pyrazolopyridone-based compound <b>10</b> (<b>1</b>) bound to the allosteric site at the C-lobe of the kinase exhibits excellent selectivity for p38α (IC<sub>50</sub> = 1.2 μM) against three closely related structures, p38β, p38γ, and p38δ (IC<sub>50</sub> > 40 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>), due to the residue differences within this site.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Surface representation of the X-ray crystal structure of p38α complexed to an allosteric compound <b>10</b> (<b>1</b>) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW">3NEW</a>). The orthosteric ATP binding site and allosteric site at the C-lobe of the kinase are highlighted in blue and red, respectively. Multiple sequence alignments of allosteric site residues in p38α, p38β, p38γ, and p38δ are shown, with identical and similarity residues marked by black and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Allosteric modulators have a spatiotemporal basis in specificity. They, together with endogenous ligands, exert their cooperative actions by the fine-tuned modulation of the protein function rather than by shutting off or turning on endogenous physiological signaling.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> This feature can result in increased safety in the event of an overdose of allosteric drugs.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a></div><div class="NLM_p">Importantly, allosteric modulators have the potential to combat drug-resistant mutations situated at orthosteric sites in patients. For example, the “gatekeeper mutation” T315I of BCR-ABL1 causes resistance to a set of clinically approved orthosteric drugs such as imatinib, bosutinib, nilotinib, and dasatinib.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> Treatment of the T315I mutant with an allosteric inhibitor ABL001 (asciminib) bound to the myristoyl pocket at the C-lobe of the kinase can overcome drug resistance (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>).<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65,66)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Surface representation of the X-ray crystal structure of BCR-ABL1 complexed to both orthosteric drug imatinib and the allosteric inhibitor ABL001 (asciminib) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MO4">5MO4</a>), with orthosteric and allosteric sites highlighted in blue and red, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A protein exists in an ensemble consisting of numerous, distinct conformational states that may be involved in multiple signaling pathways.<a onclick="showRef(event, 'ref9 ref67'); return false;" href="javascript:void(0);" class="ref ref9 ref67">(9,67)</a> An allosteric modulator binding to a unique conformation of the protein can achieve biased signaling.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> For example, G-protein- or β-arrestin-mediated downstream signaling cascades of GPCRs can be activated by different conformational states of receptor–ligand complexes.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a></div><div class="NLM_p">Remarkably, allosteric regulation also provides a new avenue for drugging “undruggable” targets that are difficult to target pharmacologically.<a onclick="showRef(event, 'ref69 ref70'); return false;" href="javascript:void(0);" class="ref ref69 ref70">(69,70)</a> This category of proteins is intractable to drugs because of strong substrate binding, the lack of deep binding sites, or large and flat protein–protein interaction (PPI) interfaces.<a onclick="showRef(event, 'ref71 ref72'); return false;" href="javascript:void(0);" class="ref ref71 ref72">(71,72)</a> Under these circumstances, allosteric modulators can bypass direct competition with orthosteric sites or PPI interfaces and bind alternative sites to fine-tune protein activities. Furthermore, cryptic (or hidden) allosteric sites can be captured from the minor or intermediate conformations of the protein and exploited for drug design, if the evident binding sites are invisible in the solved crystal structures.<a onclick="showRef(event, 'ref73 ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref73 ref74 ref75">(73−75)</a> Currently, many previously considered “undruggable” targets have proven to be treatable through allosteric regulatory modules, including K-Ras,<a onclick="showRef(event, 'ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref76 ref77">(76,77)</a> transcription factors (MYC<a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> and nuclear factor-κB<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a>), the SH2-kinase domain PPI interface of BCR-ABL,<a onclick="showRef(event, 'ref80'); return false;" href="javascript:void(0);" class="ref ref80">(80)</a> phosphatases (SHP2,<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> PTP1B,<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> and PP2A<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a>), hypoxia inducible factor-2,<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> and STAT3.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></div><div class="NLM_p last">The explosive growth in the number of allosteric modulators in recent years has demonstrated advantages to the exploration of allostery in drug discovery.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> However, allosteric modulator discovery also poses significant challenges, which may be relieved by the deeper understanding of the molecular details of allosteric mechanisms and structural and biochemical properties of allosteric proteins and modulators.</div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6">3.2.  Challenges</h3><div class="NLM_p">A key step for allosteric modulator discovery is the identification of bona fide allosteric sites in a protein. Unlike orthosteric ligands that occupy a conserved, known orthosteric site, allosteric sites are less evolutionally conserved than orthosteric sites (vide infra).<a onclick="showRef(event, 'ref87 ref88'); return false;" href="javascript:void(0);" class="ref ref87 ref88">(87,88)</a> Theoretically, any binding pocket in a protein spatially distinct from the orthosteric site can be regarded as a latent allosteric site. Moreover, the regulatory effect of the latent allosteric site on the orthosteric site is difficult to determine. These disadvantages have severely impeded the allosteric modulator discovery process over a long period. Fortunately, the ample availability of structural complexes of allosteric protein–modulator interaction provided by X-ray crystallography and NMR spectroscopy in recent years has partially addressed this quandary.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Many computational methods (see below) have been developed to predict allosteric sites based on the characteristics of known allosteric sites and the underlying mechanisms of allosteric regulation,<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> facilitating the structure-based discovery of allosteric modulators.</div><div class="NLM_p">Challenges can also stem from the emergence of drug-resistant mutations at allosteric sites, the same situation faced by orthosteric drugs. For example, in the case of Ba/F3 cell lines expressing BCR-ABL1 variants, single-point mutations (A337V, P465S, V468F, and I502L) at the allosteric myristoyl site in the C-lobe of the kinase confer resistance to the allosteric inhibitor ABL001 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>).<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Furthermore, a combination of ABL001 and an orthosteric drug nilotinib can inhibit the mutant more effectively than each agent alone.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> This additive effect suggests the possibility of a synergistic effect of allosteric and orthosteric drugs in the treatment of human diseases. A large body of evidence suggests that allosteric sites have evolved under lower evolutionary pressure compared with orthosteric sites, implying that allosteric mutations occur more frequently than orthosteric mutations.<a onclick="showRef(event, 'ref87 ref88'); return false;" href="javascript:void(0);" class="ref ref87 ref88">(87,88)</a> Indeed, more than 20 disease-associated mutations at the allosteric regulator binding domain of pyruvate kinase have been observed owing to the evolutionarily less conserved allosteric site.<a onclick="showRef(event, 'ref91'); return false;" href="javascript:void(0);" class="ref ref91">(91)</a> Additionally, mutations can occur at the allosteric communication pathways beyond allosteric sites, altering the protein dynamics and causing indirect resistance to allosteric modulators.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Such mutations are very difficult to identify experimentally because of the complexity of allosteric networks in proteins.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cartoon representation of the X-ray crystal structure of BCR-ABL1. The drug-resistant mutation residues at the orthosteric site (Thr315) and allosteric site (Ala337, Pro465, Val468, and Ile502) are depicted by sphere representations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Additionally, allosteric modulators show significant species differences owing to the low evolutionary conservation of allosteric sites.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> For instance, the actions of allosteric candidates are effective in recombinant human receptors in vitro. However, when using rodent receptors or models to examine drug effects, it is possible to obtain the opposite result.<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a></div><div class="NLM_p last">In general, allosteric modulators have lower binding affinity than orthosteric ligands<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> and often encounter “flat,” nontractable structure–activity relationships.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Furthermore, allosteric modulators often have lower aqueous solubility than orthosteric ligands.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> These properties can lead not only to the difficulty of advancing allosteric modulators as drug candidates for clinical testing but also to the crystallization of allosteric protein–modulator complexes.</div></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i7">4.  Methods To Investigate Allostery</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18151" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18151" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8">4.1.  Experimental Approaches</h3><div class="NLM_p">X-ray crystallography, the most frequently used experimental approach in structural biology, has provided insights into structural and mechanistic investigations of biologically important macromolecules.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> Advances in X-ray crystallographic technology in the past decade have substantially contributed to determine the structures of challenging macromolecules such as GPCRs,<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> the proteasome,<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> the maltose transporter,<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> and eukaryotic exosomes.<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> Remarkably, recent breakthroughs in the field of single-particle cryoelectron microscopy (cryo-EM) have enabled the characterization of the conformation of large macromolecular assemblies such as viruses,<a onclick="showRef(event, 'ref99'); return false;" href="javascript:void(0);" class="ref ref99">(99)</a> cytoskeletal proteins,<a onclick="showRef(event, 'ref100'); return false;" href="javascript:void(0);" class="ref ref100">(100)</a> and spliceosomes<a onclick="showRef(event, 'ref101'); return false;" href="javascript:void(0);" class="ref ref101">(101)</a> at near-atomic resolution, providing potential applications in drug discovery.<a onclick="showRef(event, 'ref102'); return false;" href="javascript:void(0);" class="ref ref102">(102)</a> Both X-ray crystallography and cryo-EM can provide detailed essential information about the three-dimensional (3D) structures of ligand-bound (<i>holo</i>) and unbound (<i>apo</i>) forms. These methods have enabled a structural view of allostery. In an allosterically regulated protein, the conformational change at the orthosteric site that occurs upon modulator binding can be generally uncovered by structural comparison of the <i>holo</i> and <i>apo</i> forms of the protein. By contrast, in the case of dynamics-driven allosteric proteins such as CAP<a onclick="showRef(event, 'ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref33 ref34">(33,34)</a> and PDZ domains,<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> the orthosteric site and whole protein exhibit no structural change between the <i>holo</i> and <i>apo</i> forms. In fact, X-ray and cryo-EM crystal structures are static average snapshots of macromolecules. However, these molecules can interconvert along many distinct conformations under physiological conditions to mediate different biological functions. The lack of conformational change based on the static crystal structure occasionally limits the usefulness of X-ray crystallography and cryo-EM in studies of allostery at the molecular level.</div><div class="NLM_p">In nature, allostery is a dynamic process. NMR spectroscopy, on the other hand, can probe the dynamic processes of biomolecules on a wide range of ps–ms time scales and observe conformational states in solution that are sparsely populated, yielding detailed dynamical information that is very suitable for the characterization of their allosteric mechanisms.<a onclick="showRef(event, 'ref103 ref104 ref105 ref106'); return false;" href="javascript:void(0);" class="ref ref103 ref104 ref105 ref106">(103−106)</a> It can be used to study allosterically regulated proteins with significant conformational changes when bound to allosteric modulators. Most notably, it is particularly suited to explore protein allostery when conformational change of the protein is rarely detected in response to allosteric modulator binding, thus providing direct experimental evidence to establish the dynamics or ensemble view of allostery. As an example, using chemical shift and relaxation dispersion NMR analyses, Tzeng and Kalodimos revealed that one cAMP molecule binding to the first homodimeric CAP has a minor effect on the fast dynamics of CAP, while the binding of another cAMP molecule to the second homodimeric CAP quenches such dynamics, further highlighting the role of conformational entropy in the allosteric regulation of CAP, which was validated by ITC measurements.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Traditionally, solution NMR spectroscopy has focused on relatively small proteins. Recent advances in the amide-/methyl-TROSY approaches, isotope labeling, and pulse sequence techniques<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> have made it possible to characterize large-molecular-weight proteins such as the 20S proteasome core particle<a onclick="showRef(event, 'ref108'); return false;" href="javascript:void(0);" class="ref ref108">(108)</a> and molecular chaperones such as HSP90<a onclick="showRef(event, 'ref109'); return false;" href="javascript:void(0);" class="ref ref109">(109)</a> and GroEL.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a></div><div class="NLM_p last">X-ray crystallography, cryo-EM, and NMR spectroscopy can provide direct experimental proof for allosteric modulation of biological macromolecule function. In addition to these direct approaches, a few indirect experimental approaches have been developed to explore allostery in a particular protein. These include site-directed mutagenesis such as disulfide trapping (or tethering)<a onclick="showRef(event, 'ref111'); return false;" href="javascript:void(0);" class="ref ref111">(111)</a> and alanine scanning,<a onclick="showRef(event, 'ref112'); return false;" href="javascript:void(0);" class="ref ref112">(112)</a> fluorescent<a onclick="showRef(event, 'ref113 ref114'); return false;" href="javascript:void(0);" class="ref ref113 ref114">(113,114)</a> and photoaffinity<a onclick="showRef(event, 'ref115'); return false;" href="javascript:void(0);" class="ref ref115">(115)</a> labeling, and hydrogen/deuterium exchange (HDX) mass spectrometry.<a onclick="showRef(event, 'ref116'); return false;" href="javascript:void(0);" class="ref ref116">(116)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9">4.2.  Computational Approaches</h3><div class="NLM_p">It is typically time-consuming and frequently unsatisfactory to determine allosteric sites in proteins and the mechanism of allostery using experimental approaches such as X-ray crystallography and NMR because of the complex nature of allostery in biological processes. As a viable alternative to experimental methods, computational approaches for the study of allostery provide robust tools to aid in drug design, which has been an area of intensive research.<a onclick="showRef(event, 'ref21 ref89 ref90 ref117'); return false;" href="javascript:void(0);" class="ref ref21 ref89 ref90 ref117">(21,89,90,117)</a> For example, AlloSteric Database (ASD)<a onclick="showRef(event, 'ref20 ref118 ref119'); return false;" href="javascript:void(0);" class="ref ref20 ref118 ref119">(20,118,119)</a> and ASBench<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> benchmark are two major developments that have facilitated the development of computational methods to predict allosteric sites. A large amount of the state-of-the-art computational methods have emerged, facilitated by ASD and ASBench data in the past few years.<a onclick="showRef(event, 'ref21 ref89 ref90 ref117'); return false;" href="javascript:void(0);" class="ref ref21 ref89 ref90 ref117">(21,89,90,117)</a> Most critically, many of the theoretical predictions have been confirmed by experimental observations, which will be described in the following section, suggesting the potential usefulness of computational methods in structure-based drug discovery. As such, a combined computational and experimental approach should be become a new strategy in allosteric modulator discovery.</div><div class="NLM_p"><a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> summarizes the current availability of representative computational allosteric prediction methods. Such methods include the prediction of allosteric sites, allosteric mutations, allosteric communication in proteins, assessment of allosteric protein–modulator interactions, and virtual screening of allosteric modulators. Remarkably, most computational methods focus on the identification of allosteric sites because the determination of the location of allosteric sites is the first step for structure-based allosteric modulator discovery. A wealth of predictive models for allosteric sites have been created based on different methodologies, including structure-based methods (e.g., Allosite,<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> AllositePro,<a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a> ALLO,<a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a> and a random forest model<a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a>), evolution-based methods (e.g., statistical coupling analysis (SCA)<a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a>), normal model analysis-based methods (e.g., AlloPred<a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a> and PARS<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a>), dynamics-based methods (a two-state Go̅ model,<a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a> ExProSE,<a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a> molecular dynamics (MD) simulations with enhanced conformational sampling,<a onclick="showRef(event, 'ref130'); return false;" href="javascript:void(0);" class="ref ref130">(130)</a> and AllosMod<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a>), energy-based perturbation methods (e.g., reverse perturbation),<a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a> and correlation-based methods (e.g., CorrSite).<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> Moreover, some of these programs, such as Allosite,<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> PARS,<a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a> AllosMod,<a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a> and CorrSite,<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> have been deployed as Web servers, facilitating medicinal chemists to find allosteric modulators. In addition to allosteric site prediction, Web servers or tools such as MCPath,<a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a> SPACER,<a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a> DynOmics ENM,<a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a> AlloSigMA,<a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a> AlloMAPS,<a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a> and AlloDriver<a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a> for allosteric communication and mutation analysis, STRESS<a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a> for allosteric hotspot residues prediction, Alloscore<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> for allosteric interaction evaluation, and CovCys<a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a> for covalent allosteric modulator design have also been developed.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Databases of Structures Containing Allosteric Sites and Software To Detect and Predict Them</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">name</th><th class="colsep0 rowsep0" align="center">ref</th><th class="colsep0 rowsep0" align="center">Web server available</th><th class="colsep0 rowsep0" align="center">content, methods, and applications</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>database</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ASD</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>, <a onclick="showRef(event, 'ref118'); return false;" href="javascript:void(0);" class="ref ref118">(118)</a>, <a onclick="showRef(event, 'ref119'); return false;" href="javascript:void(0);" class="ref ref119">(119)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://mdl.shsmu.edu.cn/ASD" class="extLink">http://mdl.shsmu.edu.cn/ASD</a></td><td class="colsep0 rowsep0" align="left">collection of experimentally determined allosteric proteins, modulators, and pathways</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>benchmark</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ASBench</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://mdl.shsmu.edu.cn/asbench" class="extLink">http://mdl.shsmu.edu.cn/asbench</a></td><td class="colsep0 rowsep0" align="left">two benchmark sets of allosteric sites: “core set” and “core-diversity set”</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>allosteric site identification</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Allosite</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://mdl.shsmu.edu.cn/AST" class="extLink">http://mdl.shsmu.edu.cn/AST</a></td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on a structure-based machine learning method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AllositePro</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref122'); return false;" href="javascript:void(0);" class="ref ref122">(122)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on a combined structure- and normal-mode analysis-based method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ALLO</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref123'); return false;" href="javascript:void(0);" class="ref ref123">(123)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on a structure-based machine learning method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">random forest model</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref124'); return false;" href="javascript:void(0);" class="ref ref124">(124)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on a structure-based machine learning method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SCA</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref125'); return false;" href="javascript:void(0);" class="ref ref125">(125)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on an evolutionary method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AlloPred</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref126'); return false;" href="javascript:void(0);" class="ref ref126">(126)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on the normal-mode analysis method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PARS</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref127'); return false;" href="javascript:void(0);" class="ref ref127">(127)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://bioinf.uab.cat/pars" class="extLink">http://bioinf.uab.cat/pars</a></td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on the normal-mode analysis method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">two-state Go̅ model</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref128'); return false;" href="javascript:void(0);" class="ref ref128">(128)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on an ensemble generated by coarse-grained simulations</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">ExProSE</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref129'); return false;" href="javascript:void(0);" class="ref ref129">(129)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on an ensemble generated by a distance geometry-based method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AllosMod</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref131'); return false;" href="javascript:void(0);" class="ref ref131">(131)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://modbase.compbio.ucsf.edu/allosmod" class="extLink">http://modbase.compbio.ucsf.edu/allosmod</a></td><td class="colsep0 rowsep0" align="left">detection of allosteric sites and allosteric mechanism research from MD simulations based on a modeled energy landscape</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">reverse perturbation approach</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref132'); return false;" href="javascript:void(0);" class="ref ref132">(132)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on analysis of energetics of reverse allosteric communication from orthosteric to allosteric sites</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CorrSite</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a>, <a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://www.pkumdl.cn/cavityplus" class="extLink">http://www.pkumdl.cn/cavityplus</a></td><td class="colsep0 rowsep0" align="left">detection of allosteric sites based on motion correlation analysis of allosteric and orthosteric sites</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>allosteric hotspot residues prediction</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">STRESS</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref140'); return false;" href="javascript:void(0);" class="ref ref140">(140)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://stress.molmovdb.org" class="extLink">http://stress.molmovdb.org</a></td><td class="colsep0 rowsep0" align="left">Prediction of allosteric hotspot residues based on models of conformational change generated by Monte Carlo simulations</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>allosteric communication and mutation analysis</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MCPath</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref134'); return false;" href="javascript:void(0);" class="ref ref134">(134)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://safir.prc.boun.edu.tr/clbet_server" class="extLink">http://safir.prc.boun.edu.tr/clbet_server</a></td><td class="colsep0 rowsep0" align="left">prediction of allosteric communication and allosteric residues based on an ensemble generated by Monte Carlo path simulation</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SPACER</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref135'); return false;" href="javascript:void(0);" class="ref ref135">(135)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://allostery.bii.a-star.edu.sg" class="extLink">http://allostery.bii.a-star.edu.sg</a></td><td class="colsep0 rowsep0" align="left">prediction of allosteric communication and allosteric sites based on the normal-mode analysis method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DynOmics ENM</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref136'); return false;" href="javascript:void(0);" class="ref ref136">(136)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://dyn.life.nthu.edu.tw/oENM" class="extLink">http://dyn.life.nthu.edu.tw/oENM</a></td><td class="colsep0 rowsep0" align="left">prediction of allosteric communication and functional residues based on elastic network model</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AlloSigMA</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref137'); return false;" href="javascript:void(0);" class="ref ref137">(137)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://allosigma.bii.a-star.edu.sg/home" class="extLink">http://allosigma.bii.a-star.edu.sg/home</a></td><td class="colsep0 rowsep0" align="left">assessment of the energetics of allosteric communication induced by ligand binding and mutation</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AlloMAPS</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref138'); return false;" href="javascript:void(0);" class="ref ref138">(138)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://allomaps.bii.a-star.edu.sg" class="extLink">http://allomaps.bii.a-star.edu.sg</a></td><td class="colsep0 rowsep0" align="left">evaluation of allosteric effects of mutations and prediction of latent regulatory exosites</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AlloDriver</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref139'); return false;" href="javascript:void(0);" class="ref ref139">(139)</a></td><td class="colsep0 rowsep0" align="left">no</td><td class="colsep0 rowsep0" align="left">analysis of clinic high-throughput data on allosteric mutations</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>allosteric interaction scoring</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Alloscore</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://mdl.shsmu.edu.cn/alloscore" class="extLink">http://mdl.shsmu.edu.cn/alloscore</a></td><td class="colsep0 rowsep0" align="left">prediction of binding affinities of allosteric protein–modulator interactions based on a machine learning method</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="left"><b>allosteric modulator screening and design</b></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">AlloFinder</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://mdl.shsmu.edu.cn/ALF" class="extLink">http://mdl.shsmu.edu.cn/ALF</a></td><td class="colsep0 rowsep0" align="left">automatic discovery of allosteric modulators and allosteric mechanism research</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">CovCys</td><td class="colsep0 rowsep0" align="left"><a onclick="showRef(event, 'ref142'); return false;" href="javascript:void(0);" class="ref ref142">(142)</a></td><td class="colsep0 rowsep0" align="left"><a href="http://www.pkumdl.cn/cavityplus" class="extLink">http://www.pkumdl.cn/cavityplus</a></td><td class="colsep0 rowsep0" align="left">automatic detection of druggable cysteine residues for allosteric covalent drug design</td></tr></tbody></table></div></div></div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">5.  Principles for Allosteric Modulator Discovery</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05676" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05676" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The latest release of ASD (Nov 2018) contains ∼1800 allosteric proteins regardless of species (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>), which are mainly distributed among eight protein families: kinases, GPCRs, transcription factors, ion channels, oxidoreductases, lyases, hydrolases, and transferases (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>). At the nascent stage of allosteric research, the allosteric regulatory phenomena were observed in multimeric proteins with hemoglobin representing the prototype of this class. Owing to the progress in chemical and structural biology techniques, allosteric binding has been observed in monomeric proteins such as GPCRs, kinases, and enzymes, greatly expanding the repertoire of available allosteric drug targets.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Total number of X-ray or NMR structures of allosteric proteins and yearly growth of total structures in the AlloSteric Database (ASD). Representative structures of allosteric proteins are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Major class distribution of the structures of allosteric proteins in ASD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The number of allosteric modulators has been rapidly increasing in recent years due to the growing investment of the scientific community in allosteric modulator discovery (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>). ASD now contains ∼80 000 allosteric modulators that target more than 1300 proteins. Among the allosteric modulators, five chemical entities have been approved by the U.S. FDA as marketed drugs, including three GPCR (Cinacalcet, Maraviroc, and Plerixafor) and two kinase (Gleevec and Mekinist) allosteric drugs. Additionally, many promising candidates are now entering clinical trials.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Yearly growth of allosteric modulators in ASD. The chemical structures of the marketed allosteric drugs are indicated in the chart.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12">5.1.  Characteristics of Allosteric Sites</h3><div class="NLM_p">The molecular evolution of allosteric and orthosteric sites is different. Kim and co-workers analyzed the sequence conservation of allosteric and orthosteric sites in 56 allosterically regulated enzymes.<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> The results showed that allosteric site residues (average conservation score = 0.58) are significantly less conserved than orthosteric site residues (average conservation score = 0.94; <i>P</i> = 1.3 × 10<sup>–67</sup>). Similarly, comparing the sequence conservation of ATP binding sites in the allosteric and orthosteric data sets, we found that residues in the orthosteric ATP binding site (average conservation score = 0.63) are less conserved than residues in the orthosteric ATP binding site (average conservation score = 0.83; <i>P</i> = 1.2 × 10<sup>–3</sup>).<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> Furthermore, the shapes calculated by the pocket similarity score (PS-score) between the allosteric and orthosteric ATP binding sites are distinct. The global structural similarity of the allosteric ATP binding site is lower (average PS-score = 0.3) than that of the orthosteric ATP binding site (average PS-score = 0.6).<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> Collectively, these data are consistent with the higher target selectivity observed for allosteric modulators relative to orthosteric ligands.</div><div class="NLM_p">The physicochemical properties of allosteric and orthosteric sites in an individual protein were further explored by comparison of the amino acid compositions of the two sets of binding sites curated from ASD version 3.0.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The results showed that hydrophobic residues such as isoleucine and tyrosine are highly enriched in allosteric sites (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). By contrast, polar residues such as aspartic acid, glutamine, histidine, and glycine are highly enriched in orthosteric sites. These data suggest that allosteric sites are more hydrophobic than orthosteric sites.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Residue compositions in the orthosteric (light blue) and allosteric (blue) sites calculated from ASD version 3.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13">5.2.  Characteristics of Allosteric Modulators</h3><div class="NLM_p last">On the basis of the allosteric modulators from ASD version 3.0 and orthosteric ligands from ChEMBL, Carlson and co-workers compared the physicochemical properties of allosteric modulators and orthosteric ligands.<a onclick="showRef(event, 'ref143'); return false;" href="javascript:void(0);" class="ref ref143">(143)</a> The results showed that allosteric modulators are more rigid and aromatic than orthosteric ligands, in accordance with previous studies by Overington and co-workers.<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> This principle can be applied to help select potential allosteric modulators for a protein after virtual screening; that is, in cases where it is known there are differences in polarity between orthosteric and allosteric sites, choosing compounds that are rigid and more aromatic can increase the likelihood of occupying an allosteric site.</div></div></div><div id="sec6" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">6.  Examples of the Structure-Based Design of Allosteric Modulators</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42080" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42080" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Due to the recent progress in the molecular understanding of the allosteric process in recent years, many computational approaches have been developed for various aspects of allosteric applications. Such approaches proved successful in finding putative allosteric modulators, confirmed by subsequent crystallographic/NMR or biological experiments. In this section, we will present some representative examples of using combined computational and experimental tactics to identify allosteric modulators.</div><div id="sec6_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15">6.1.  SIRT6 Allosteric Activators</h3><div class="NLM_p">Sirtuin 6 (SIRT6), a member of the human sirtuin family (SIRT1–7), is a histone deacetylase that transfers the acetyl group from the lysine side chain of a protein or peptide substrate to nicotinamide adenine dinucleotide (NAD<sup>+</sup>).<a onclick="showRef(event, 'ref144'); return false;" href="javascript:void(0);" class="ref ref144">(144)</a> The regulation of SIRT6 has been closely associated with multiple biological processes, encompassing DNA damage repair, organ metabolism, aging, and tumorigenesis.<a onclick="showRef(event, 'ref145'); return false;" href="javascript:void(0);" class="ref ref145">(145)</a> Accumulating evidence strongly indicates that SIRT6 knockout mice are accompanied by a severe premature aging syndrome, while mice containing SIRT6 overexpression display a dramatically longer lifespan. They also show massive apoptosis in various cancer cells but not in normal cells.<a onclick="showRef(event, 'ref146 ref147 ref148'); return false;" href="javascript:void(0);" class="ref ref146 ref147 ref148">(146−148)</a> Pharmacological activation of SIRT6 by small-molecule compounds may, therefore, provide a potential new avenue in age- and cancer-related treatment.<a onclick="showRef(event, 'ref149 ref150'); return false;" href="javascript:void(0);" class="ref ref149 ref150">(149,150)</a></div><div class="NLM_p">To increase SIRT6 catalytic activity, the design of compounds to directly compete with endogenous NAD<sup>+</sup> is infeasible because NAD<sup>+</sup> is the SIRT6 cofactor involved in the catalytic reaction. As such, it is advisable and effective to target allosteric sites outside the NAD<sup>+</sup> orthosteric site. However, the allosteric sites of SIRT6 were unavailable at the onset of our studies on SIRT6 allosteric activators in 2013.</div><div class="NLM_p">To address this issue, we employed our previously developed structure-based Allosite<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> approach to computationally predict potential allosteric sites (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>). The prediction informed us of a potential allosteric site formed by residues Phe82 and Phe86 adjacent to the N-terminus of SIRT6. On the basis of this predicted allosteric site, virtual screening of seven commercial chemical libraries (SPECS, ChemBridge, ChemDiv, Enamine, InterBioScreen, Life Chemicals, and Maybridge) consisting of more than 5 000 000 compounds was performed. According to the principle that allosteric modulators are rigid and more aromatic than orthosteric ligands, we chose and purchased 20 rigid and aromatic compounds from the top-ranked compounds calculated by our Alloscore<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> approach to assess allosteric interactions. After experimental testing, two initial hits from the SPECS chemical library, AN-988/40889624 (<b>2</b>) and AH-487/41802661 (<b>3</b>) (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>), were found to dose-dependently activate SIRT6 deacetylation, with half-maximal effective concentration values of 173.8 ± 1.3 μM and 217.6 ± 1.1 μM, respectively.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> To improve the activation potency of the hits, medicinal chemistry optimizations of <b>3</b> led to two activators MDL-800 (<b>4</b>) and MDL-801 (<b>5</b>) (<a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a></a>), both of which significantly enhanced the deacetylation activity of SIRT6, with EC<sub>50</sub> values of 10.3 ± 0.3 μM and 5.7 ± 0.3 μM, respectively. Crystallographic studies further confirmed that <b>5</b> binds to the allosteric site located at the surface exit of the long SIRT6 channel pocket but not to the substrate site of SIRT6. This allosteric site is formed by the N-terminal residues Val3-Leu9, Val70, Glu74, Phe82, Phe86, Val153, and Met157 (<a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a></a>),<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> in agreement with the computationally predicted allosteric site by Allosite<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> and supporting the feasibility of Allosite in the identification of as-yet-unknown allosteric sites. Cellular and animal model studies on the growth inhibitory effects of human hepatocellular carcinoma (HCC) further demonstrated that allosteric SIRT6 activators can suppress the proliferation of HCC.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> These findings should provide a starting point to facilitate the exploration of allosteric SIRT6 activators as therapeutic agents or as probe compounds to understand SIRT6 biology.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Workflow of the rational discovery of SIRT6 allosteric activators. The potential allosteric site of SIRT6 was predicted using Allosite (<a href="http://mdl.shsmu.edu.cn/AST" class="extLink">http://mdl.shsmu.edu.cn/AST</a>). SIRT6 is shown as surface colored in gray, with the orthosteric and predicted allosteric sites highlighted in blue and red, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Detailed interactions between MDL-801 and SIRT6 at the allosteric site determined by the cocrystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F">5Y2F</a>). SIRT6 is shown in gray, and MDL-801 is depicted by sticks with carbon atoms colored in yellow. The key residues within the allosteric site of MDL-801 are represented by sticks with carbon atoms colored in cyan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16">6.2.  STAT3 Allosteric Inhibitors</h3><div class="NLM_p">Signal transducer and activator of transcription 3 (STAT3) play an important role in many biological processes, including differentiation, survival, proliferation, and angiogenesis.<a onclick="showRef(event, 'ref151'); return false;" href="javascript:void(0);" class="ref ref151">(151)</a> However, constitutive or aberrant activation of STAT3 has been linked to malignant transformation and tumorigenesis.<a onclick="showRef(event, 'ref152'); return false;" href="javascript:void(0);" class="ref ref152">(152)</a> As such, inhibition of STAT3 activity represents an attractive strategy for anticancer drug discovery.<a onclick="showRef(event, 'ref153'); return false;" href="javascript:void(0);" class="ref ref153">(153)</a></div><div class="NLM_p">STAT3 is a multidomain protein consisting of six distinct functional domains (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). Many STAT3 inhibitors were previously developed that targeted to the Src homology 2 (SH2) domain and, to a lesser extent, the DNA-binding domain (DBD).<a onclick="showRef(event, 'ref154 ref155 ref156 ref157'); return false;" href="javascript:void(0);" class="ref ref154 ref155 ref156 ref157">(154−157)</a> Such inhibitors often encounter poor pharmacokinetic properties and cytotoxicity during clinical trials, making them ineffective drug candidates.<a onclick="showRef(event, 'ref158'); return false;" href="javascript:void(0);" class="ref ref158">(158)</a> Unlike the SH2 domain and DBD, inhibitors targeted to other domains such as the coiled-coil domain (CCD) are scarce. The CCD can be regarded as an allosteric domain to regulate STAT3 functional activity.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0010.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Workflow of the rational discovery of a STAT3 allosteric inhibitor. An X-ray structure of STAT3 (residues 130–722) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1">1BG1</a>) is colored by the domain. The predicted allosteric site at the coiled-coil domain (CCD), large-scale molecular docking of compound libraries at the CCD allosteric site, and allosteric interaction evaluation after docking were executed automatically by AlloFinder (<a href="http://mdl.shsmu.edu.cn/ALF" class="extLink">http://mdl.shsmu.edu.cn/ALF</a>). Enlarged portion shows the potential interactions between the allosteric inhibitor K116 (<b>6</b>) and STAT3, consistent with mutagenesis experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">On the basis of the STAT3 CCD, we used our developed AlloFinder<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> approach to screen in silico for potential allosteric inhibitors. This approach can automatically predict allosteric sites by Allosite,<a onclick="showRef(event, 'ref121'); return false;" href="javascript:void(0);" class="ref ref121">(121)</a> perform molecular docking by AutoDock Vina,<a onclick="showRef(event, 'ref159'); return false;" href="javascript:void(0);" class="ref ref159">(159)</a> and rank the docked poses by Alloscore.<a onclick="showRef(event, 'ref141'); return false;" href="javascript:void(0);" class="ref ref141">(141)</a> We chose and purchased the top 15 hits for bioassays; these hits are bound to the predicted allosteric site formed by residues Asp171, Asn175, Gln202, and Met213 (<a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig10" id="" class=" internalNav">10</a></a>). After biological testing, we observed that the small-molecule compound K116 (AH-034/11936955) (<b>6</b>) from the SPECS chemical library can bind to STAT3 CCD, with a <i>K</i><sub>d</sub> value of 3.22 ± 0.85 μM.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Site-directed mutagenesis experiments showed that both double mutants (N175G/K177G and Q212A/M213G) decreased the binding affinity of <b>6</b> to the mutants by ∼10-fold compared with wild-type STAT3.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> These data support the possibility of the discovery of STAT3 allosteric inhibitors by the AlloFinder approach. However, a further crystallographic structural complex is required to confirm the allosteric inhibition of STAT3 by <b>6</b> directly. Currently, medicinal chemistry optimization of <b>6</b> is in progress, and structural determination of STAT3 in complex with <b>6</b> derivatives will be performed soon.</div></div><div id="sec6_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17">6.3.  GPX4 Allosteric Activators</h3><div class="NLM_p">By analysis of the correlations between allosteric sites and corresponding orthosteric sites in monomer or oligomeric allosteric proteins using a Gaussian network model, Lai and co-workers found that the motions of allosteric and orthosteric sites in proteins are highly correlated.<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> According to this principle, they developed a correlation analysis method named CorrSite<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> to predict potential allosteric sites in proteins.</div><div class="NLM_p">Glutathione peroxidase 4 (GPX4) controls many biological processes, including cell membrane repair, inflammation suppression, and ferroptosis inhibition.<a onclick="showRef(event, 'ref160'); return false;" href="javascript:void(0);" class="ref ref160">(160)</a> Upregulating GPX4 enzyme activity has a clinical benefit in the treatment of inflammation- and ferroptosis-associated diseases.<a onclick="showRef(event, 'ref161 ref162'); return false;" href="javascript:void(0);" class="ref ref161 ref162">(161,162)</a> As such, the development of GPX4 activators can provide a pharmacological benefit.</div><div class="NLM_p">To discover GPX4 activators, Lai and co-workers recently have combined the computational methods CAVITY<a onclick="showRef(event, 'ref163'); return false;" href="javascript:void(0);" class="ref ref163">(163)</a> and CorrSite<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> to predict a potential allosteric site in GPX4.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> The predicted allosteric site, with a CorreSite score of 1.49 (cavities with CorreSite scores of >0.5 are considered potential allosteric sites by the CorrSite method), is located at the opposite side of the protein from the substrate binding site (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>). This allosteric site is formed by three acidic residues (Asp21, Asp23, and Asp101), two basic residues (Lys31 and Lys90), and seven hydrophobic residues (Ile22, Ala93, Ala94, Val98, Phe100, Met102, and Phe103). Virtual screening of the SPECS chemical library was then performed based on the predicted allosteric site, and 251 compounds were selected for biological testing with ≥1 protein–ligand hydrogen bond and good hydrophobic interactions. Among these compounds, compound <b>1</b> (<b>7</b>) (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>) was observed to dose-dependently increase GPX4 activity to 260% at an activator concentration of 500 μM in the cell-free assay, with a <i>K</i><sub>d</sub> value of 63 ± 5 μM. On the basis of the modeling of CPX4-<b>7</b> protein–ligand interactions, SAR studies of <b>7</b> were further performed to generate the strongest compound, <b>1d4</b> (<b>8</b>) (<a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig11" id="" class=" internalNav">11</a></a>), which significantly increased GPX4 activity to 150% at 20 μM in the cell-free assay and 61 μM in cell extracts. A double mutant (D21A/D23A) and two single mutants (D21A and D23A) showed reduced activation of GPX4 in response to both <b>7</b> and <b>8</b>, confirming the predicted allosteric site by the CorrSite<a onclick="showRef(event, 'ref133'); return false;" href="javascript:void(0);" class="ref ref133">(133)</a> approach.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0011.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Workflow of the rational discovery of GPX4 allosteric activators. The allosteric site of GPX4 was predicted using CAVITY and CorrSite (<a href="http://www.pkumdl.cn/cavityplus" class="extLink">http://www.pkumdl.cn/cavityplus</a>) based on the crystal structure of GPX4 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBI">2OBI</a>). The predicted allosteric site (red) is on the opposite side of the orthosteric site (blue). The predicted binding mode of <b>1d4</b> (<b>8</b>) represented by blue sticks to GPX4 is shown, with key residues within the binding site highlighted in orange by stick representations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18">6.4.  Cathepsin K Allosteric Inhibitors</h3><div class="NLM_p">Evolution-based prediction of allosteric sites exploit sequence alignments and other genetics analysis methods to identify conserved residues throughout evolution.<a onclick="showRef(event, 'ref164'); return false;" href="javascript:void(0);" class="ref ref164">(164)</a> SCA is a prototype of an evolution-based prediction method, which utilizes multiple sequence alignment (MSA) to uncover evolutionarily conserved networks of residues within a protein family.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Such networks, referred to as protein sectors, are proposed to mediate allosteric communications between functional orthosteric sites and regulatory allosteric sites.<a onclick="showRef(event, 'ref46 ref165'); return false;" href="javascript:void(0);" class="ref ref46 ref165">(46,165)</a> Nevertheless, the practicality of SCA has limitations.<a onclick="showRef(event, 'ref166 ref167 ref168 ref169 ref170'); return false;" href="javascript:void(0);" class="ref ref166 ref167 ref168 ref169 ref170">(166−170)</a> Major concerns include that this method is only applicable to residues in close physical contacts that are immediately subject to evolution selection<a onclick="showRef(event, 'ref167 ref169 ref170'); return false;" href="javascript:void(0);" class="ref ref167 ref169 ref170">(167,169,170)</a> and whether its prediction of the evolutionarily constrained positions can completely reflect the thermodynamic or allosteric couplings within protein structures.<a onclick="showRef(event, 'ref166'); return false;" href="javascript:void(0);" class="ref ref166">(166)</a></div><div class="NLM_p">Despite its advantages and disadvantages, the potential application of SCA has been exemplified in the discovery of the first low-molecular-weight allosteric inhibitor of cathepsin K, NSC13345 (<b>9</b>) (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>).<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a> Novinec and co-workers first constructed an MSA of 1239 catalytic domains from the papain-like cysteine peptidase family.<a onclick="showRef(event, 'ref171'); return false;" href="javascript:void(0);" class="ref ref171">(171)</a> The pairwise correlations between all position pairs within the alignment were revealed through the positional correlation matrix calculations, which were then clustered using hierarchical clustering. A continuous network surrounding the orthosteric site and stretching around cathepsin K (i.e., the protein sector) was identified through automated sector identification. This sector comprises the coevolving residues in the protein family with functional importance. A few potential allosteric sites were subsequently detected using the AutoLigand<a onclick="showRef(event, 'ref172'); return false;" href="javascript:void(0);" class="ref ref172">(172)</a> cavity prediction method. <b>9</b> from the NCI Diversity Set III compound library was identified as a potential hit. A crystal structure of cathepsin K in complex with <b>9</b>, solved to determine the allosteric regulatory mechanism for its binding, showed that it was bound to a remote surface-exposed allosteric site (<a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig12" id="" class=" internalNav">12</a></a>), site 6. The latter mainly comprises helix α10 and adjacent β sheets and loops. Within this site, <b>9</b> makes hydrogen bonds with Gly113, Lys236, Arg237, and Gln313. Further biochemical assays revealed that <b>9</b> abolished the collagenolytic activity of cathepsin K, with an IC<sub>50</sub> value of 80 ± 30 μM. Together, these results suggest the possibility of applying an evolution-based allosteric method in the discovery of allosteric modulators. In fact, this method has been applied in other cases, such as the PDZ domain,<a onclick="showRef(event, 'ref173'); return false;" href="javascript:void(0);" class="ref ref173">(173)</a> G proteins,<a onclick="showRef(event, 'ref174'); return false;" href="javascript:void(0);" class="ref ref174">(174)</a> and TonB-dependent transporters.<a onclick="showRef(event, 'ref175'); return false;" href="javascript:void(0);" class="ref ref175">(175)</a></div><figure id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0012.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structural overview of cathepsin K in complex with its allosteric inhibitor NSC13345 (<b>9</b>) and substrate TCO (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94">5J94</a>). The backbone structure of cathepsin K is shown in gray. The allosteric site (site 6) and orthosteric site are highlighted in red and blue, respectively. The upper panel shows the chemical structure of <b>9</b>, and the lower panel shows the detailed interactions between <b>9</b> and cathepsin K. Hydrogen bonds are displayed in red dashed lines, and residues within the allosteric site are represented by sticks. Substrate TCO is manually docked into the orthosteric site based on the superposition of PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94">5J94</a> with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q6K">1Q6K</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec6_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19">6.5.  c-Src Allosteric Inhibitors</h3><div class="NLM_p">Protein allostery is rooted in protein dynamics.<a onclick="showRef(event, 'ref176'); return false;" href="javascript:void(0);" class="ref ref176">(176)</a> The proper understanding of allosteric effects and accurate prediction of allostery cannot be achieved unless the conformational dynamics and energetic landscape of protein structures are thoroughly dissected. MD simulations can characterize both large-scale conformational changes in overall structures and subtle alterations in residue orientations, as well as provide energetic insights into conformational dynamics.<a onclick="showRef(event, 'ref177 ref178 ref179 ref180'); return false;" href="javascript:void(0);" class="ref ref177 ref178 ref179 ref180">(177−180)</a> Thus, MD-based approaches, coupled with subsequent correlated motions and community analyses, can uncover allosteric sites coupled to orthosteric sites with a high accuracy.<a onclick="showRef(event, 'ref181 ref182 ref183 ref184'); return false;" href="javascript:void(0);" class="ref ref181 ref182 ref183 ref184">(181−184)</a> Furthermore, MD simulations enable the production of complete dynamic biological processes, facilitating the detection of the highly flexible or even transient, so-called “hidden” allosteric sites.<a onclick="showRef(event, 'ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref73 ref74">(73,74)</a></div><div class="NLM_p">Pande and co-workers performed 550 μs of massively distributed MD simulations and Markov state model (MSM) analyses to identify potential allosteric sites on the anticancer drug target c-Src.<a onclick="showRef(event, 'ref185'); return false;" href="javascript:void(0);" class="ref ref185">(185)</a> Snapshot structures from simulated trajectories were clustered using MSM analysis, which separated the dynamic process into a series of discrete states and calculated the population of each state and transitions between them. Two intermediate states (I<sub>1</sub> and I<sub>2</sub>) were thereby identified along the activation pathway of c-Src. A high similarity was observed between the I<sub>2</sub> conformational state and previously reported structure of an allosteric inhibitor (ANS) (<b>10</b>)-bound cyclin-dependent kinase 2 (CDK2).<a onclick="showRef(event, 'ref186'); return false;" href="javascript:void(0);" class="ref ref186">(186)</a> In these two complexes, the activation loop (A-loop) adopted an unfolded conformation, forming a potential allosteric site with the C-helix and adjacent β-sheets (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A), which cannot be found in both the active (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>B and <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>C) and inactive (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>D) states of c-Src. The binding of <b>10</b> to the c-Src was further confirmed by simulations of a <b>10</b>-bound I<sub>2</sub> structure. <b>10</b> was bound to the pocket formed by the A-loop, C-helix, and β3, 5, 6 of c-Src, similar to that in the crystal structure of the CDK2-<b>10</b> complex (<a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig13" id="" class=" internalNav">13</a></a>A). Through blocking the displacement of the C-helix, <b>10</b> allosterically inhibited c-Src by trapping it in a partially active intermediate state. Collectively, this study highlighted that large-scale MD simulations can detect a novel, cryptic allosteric site in the intermediate state structure of c-Src. Due to their ability to characterize the kinetic, thermodynamic, and structural features of the protein ensembles, MD simulations, coupled with subsequent MSM analysis, have become a popular, robust tool for allosteric site and modulator discovery.<a onclick="showRef(event, 'ref187 ref188 ref189 ref190'); return false;" href="javascript:void(0);" class="ref ref187 ref188 ref189 ref190">(187−190)</a></div><figure id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0013.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. (A) Chemical structure of ANS (<b>10</b>) and structural overview of CDK2 in complex with <b>10</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF">3PXF</a>). (B) The binding site of <b>10</b> is in a closed conformation in the active state of c-Src (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Y57">1Y57</a>). (C) Structural comparison between the <b>10</b>-bound CDK2 (yellow) and active state of c-Src (blue) shows that the allosteric site is in a closed conformation mainly due to the displacements of the C-helix and A-loop. (D) The binding site of <b>10</b> is also in a closed conformation in the inactive state of c-Src (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC">2SRC</a>). The C-helix, A-loop, and β-sheets involved in the binding site of <b>10</b> are colored in orange, pink, and cyan, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec7" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">7.  Conclusions and Perspectives</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Allostery underlies a myriad of biological processes that regulate protein functions, where binding of an effector at the allosteric site affects catalytic processing at an orthosteric site. From a medicinal chemistry perspective, allostery represents an effective tool for the medicinal chemist to drug “undruggable” proteins or alternatively to expand the repertoire of the available drug targets. Allosteric drugs differ from orthosteric drugs in the mode of drug action, with the possibility of achieving higher selectivity, fewer adverse effects, and lower toxicity than orthosteric drugs. Allosterism in drug discovery has, therefore, emerged as a promising strategy to develop efficient and safe therapeutic agents. Because the nature of allosteric modulation is complicated, allosteric modulators were traditionally discovered serendipitously by high-throughput screening.</div><div class="NLM_p">Over the past decade, understanding of the basic mechanisms of allosteric modulation, technology advances in the experimental approaches to study allostery, and remarkable availability of structural data have contributed to the development of various computational approaches to predict allosteric phenomena.<a onclick="showRef(event, 'ref191'); return false;" href="javascript:void(0);" class="ref ref191">(191)</a> Such predictive models were created based on the structure-, evolution-, or dynamics-based methods to detect and discover allosteric sites and modulators.<a onclick="showRef(event, 'ref89 ref90'); return false;" href="javascript:void(0);" class="ref ref89 ref90">(89,90)</a> Some of these computational predictions have been confirmed by experimental observations, highlighting the possibility of computational approaches in the discovery of allosteric modulators.</div><div class="NLM_p">Despite decades of allosteric research and the development of many models for allosteric applications, allosteric modulator discovery continues to face challenges. One of the biggest challenges originates from identifying putative allosteric sites. Particularly, in the case of cryptic allosteric sites that are invisible in <i>apo</i> crystal structures but can emerge in the conformational transition of proteins such as in <i>holo</i> crystal structures, current structure- and evolution-based allosteric computational methods cannot detect such sites in the <i>apo</i> structures.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Large-scale MD simulations, coupled with MSM analyses, can provide a potential opportunity to identify cryptic allosteric sites, but such simulations with an effective conformational sampling are extremely time-consuming.<a onclick="showRef(event, 'ref192 ref193'); return false;" href="javascript:void(0);" class="ref ref192 ref193">(192,193)</a> Alternatively, using normal-mode analysis to rapidly capture the conformational ensemble of a protein and then employing computational approaches to identify cryptic allosteric sites can offer guidelines to help to circumvent this dilemma.</div><div class="NLM_p">Despite the current understanding of the general characteristics of known allosteric sites and modulators, the guidelines for the structural modifications of initial allosteric hits to increase affinities (hit-to-lead optimization) are lacking. Because the properties of allosteric sites and modulators differ from the corresponding orthosteric sites and ligands, software traditionally used for orthosteric drug design faces challenges in guiding allosteric modulator design. There is a strong demand for insights into the unique protein–modulator interactions underlying allosteric binding. Our previously created ASBench<a onclick="showRef(event, 'ref120'); return false;" href="javascript:void(0);" class="ref ref120">(120)</a> benchmark of high-quality data of allosteric protein–modulator complexes can assist in uncovering interactions useful for designing allosteric-active compounds.</div><div class="NLM_p">Many reports of analyses of allostery on the single-molecule level have emerged, leaving the role of allostery at the cellular level unexplored.<a onclick="showRef(event, 'ref194'); return false;" href="javascript:void(0);" class="ref ref194">(194)</a> Allostery is a fundamental cellular phenomenon. Thus, it is required to develop multiscale approaches that unite computational and experimental methods to explore allosteric effects on cellular networks. The system-centric method will identify new targets to broaden the allosteric modulator discovery scope.<a onclick="showRef(event, 'ref195 ref196 ref197'); return false;" href="javascript:void(0);" class="ref ref195 ref196 ref197">(195−197)</a> For example, the allo-network drug concept has recently been proposed to design allosteric modulators in system-based drug design.<a onclick="showRef(event, 'ref198'); return false;" href="javascript:void(0);" class="ref ref198">(198)</a></div><div class="NLM_p last">Overall, recently developed computational allosteric prediction approaches have shown some success in designing allosteric modulators for known protein targets. These techniques, coupled with advances in experimental approaches to understand allosteric activation or inhibition, indicate that allosteric modulator discovery has emerged as a new strategy for drug design.</div></div><div class="NLM_back"><div class="article_supporting-info" id=""></div><div class="testing" data-doi="10.1021/acs.jmedchem.8b01749" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33610" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33610" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jian Zhang</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry
of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Bioinformatics Center, Shanghai Jiao-Tong
University School of Medicine, Shanghai 200025, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6558-791X" title="Orcid link">http://orcid.org/0000-0002-6558-791X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#503a39313e7e2a38313e3710233a24257e3534257e333e"><span class="__cf_email__" data-cfemail="94fefdf5fabaeefcf5faf3d4e7fee0e1baf1f0e1baf7fa">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shaoyong Lu</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry
of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</span>; 
    <span class="hlFld-Affiliation affiliation">Medicinal
Bioinformatics Center, Shanghai Jiao-Tong
University School of Medicine, Shanghai 200025, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xinheng He</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry
of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Duan Ni</span> - <span class="hlFld-Affiliation affiliation">Key
Laboratory of Cell Differentiation and Apoptosis of Chinese Ministry
of Education, Clinical and Fundamental Research Center, Renji Hospital, Shanghai Jiao-Tong University School of Medicine, Shanghai 200025, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>S.L., X.H., and D.N. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="bio" rid="ath1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="biographies">Biographies</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31047" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31047" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><figure id="ath1" data-index="14" class="article__inlineFigure"><h2 class="fig-label">Shaoyong Lu</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=BIO-d7e1663-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Shaoyong Lu</b> received his undergraduate Applied Chemistry degree from Hangzhou Normal University in 2007. Thereafter, he obtained a Ph.D. degree in Chemistry from Zhejiang University, China, in 2012. He then continued with postdoctoral research at Shanghai Jiao Tong University, School of Medicine, in Professor Jian Zhang’s laboratory and is now an Associate Professor. During 2014–2015, he studied as a Visiting Scholar in Professor Ruth Nussinov’s group at the National Cancer Institute—Frederick. His main interests include the study of allosteric protein–ligand interactions and the development of allosteric methods and their applications.</p></figure></div><div class="bio" rid="ath2"><figure id="ath2" data-index="15" class="article__inlineFigure"><h2 class="fig-label">Xinheng He</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=BIO-d7e1668-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Xinheng He</b> is an undergraduate student from the Discipline of Biomedical Science, Shanghai Jiao Tong University, School of Medicine, in Professor Jian Zhang’s group. He mainly focuses on allosteric mechanism research and allosteric drug design.</p></figure></div><div class="bio" rid="ath3"><figure id="ath3" data-index="16" class="article__inlineFigure"><h2 class="fig-label">Duan Ni</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=BIO-d7e1673-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Duan Ni</b> is an undergraduate student from the Discipline of Biomedical Science, Shanghai Jiao Tong University, School of Medicine, in Professor Jian Zhang’s group. He mainly focuses on using computational methods to investigate the allosteric mechanisms and predict allosteric sites.</p></figure></div><div class="bio" rid="ath4"><figure id="ath4" data-index="17" class="article__inlineFigure"><h2 class="fig-label">Jian Zhang</h2><div class="figure-bottom-links"><div class="download-hi-res-img"></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=BIO-d7e1678-autogenerated"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><b>Jian Zhang</b> received a B.M. degree in Pharmacology in 2002 from Peking University and a Ph.D. in 2007 from Shanghai Institute of Materia Medica, Chinese Academy of Sciences. After receiving his Ph.D., he moved to the University of Michigan to conduct postdoctoral research in Professor Shaomeng Wang’s laboratory. In 2009, he joined Shanghai Jiao Tong University, School of Medicine, where he is a full-time researcher and doctoral supervisor. Now, he is also the director of the Medicinal Bioinformatics Center. His fields of research include first-in-class drug discovery and chemical biology that mainly pertains to the repertoire of allostery. His Web site provides further details: <a href="http://mdl.shsmu.edu.cn/" class="extLink">http://mdl.shsmu.edu.cn/</a>.</p></figure></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08460" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08460" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the National Natural Science Foundation of China (Grants 21778037, 81603023, 81322046, 91753117, 81473137), the National Basic Research Program of China (973 Program) (Grant 2015CB910403), and the Innovation Team “Diagnosis and Treatment of Major Cardiovascular Diseases” of High-Level Universities in Shanghai.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">A-loop</td><td class="NLM_def"><p class="first last">activation loop</p></td></tr><tr><td class="NLM_term">ASD</td><td class="NLM_def"><p class="first last">AlloSteric Database</p></td></tr><tr><td class="NLM_term">CAP</td><td class="NLM_def"><p class="first last">catabolite activator protein</p></td></tr><tr><td class="NLM_term">cryo-EM</td><td class="NLM_def"><p class="first last">cryoelectron microscopy</p></td></tr><tr><td class="NLM_term">CCD</td><td class="NLM_def"><p class="first last">coiled-coil domain</p></td></tr><tr><td class="NLM_term">CDK2</td><td class="NLM_def"><p class="first last">cyclin-dependent kinase 2</p></td></tr><tr><td class="NLM_term">3D</td><td class="NLM_def"><p class="first last">three-dimensional</p></td></tr><tr><td class="NLM_term">DBD</td><td class="NLM_def"><p class="first last">DNA-binding domain</p></td></tr><tr><td class="NLM_term">GPX4</td><td class="NLM_def"><p class="first last">glutathione peroxidase 4</p></td></tr><tr><td class="NLM_term">GPCR</td><td class="NLM_def"><p class="first last">G-protein-coupled receptor</p></td></tr><tr><td class="NLM_term">HCC</td><td class="NLM_def"><p class="first last">human hepatocellular carcinoma</p></td></tr><tr><td class="NLM_term">HDX</td><td class="NLM_def"><p class="first last">hydrogen/deuterium exchange</p></td></tr><tr><td class="NLM_term">KNF</td><td class="NLM_def"><p class="first last">Koshland–Nemethy–Filmer</p></td></tr><tr><td class="NLM_term">MSM</td><td class="NLM_def"><p class="first last">Markov state model</p></td></tr><tr><td class="NLM_term">MD</td><td class="NLM_def"><p class="first last">molecular dynamics</p></td></tr><tr><td class="NLM_term">MWC</td><td class="NLM_def"><p class="first last">Monod–Wyman–Changeux</p></td></tr><tr><td class="NLM_term">MSA</td><td class="NLM_def"><p class="first last">multiple sequence alignment</p></td></tr><tr><td class="NLM_term">NAD<sup>+</sup></td><td class="NLM_def"><p class="first last">nicotinamide adenine dinucleotide</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">PPI</td><td class="NLM_def"><p class="first last">protein–protein interaction</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">SH2</td><td class="NLM_def"><p class="first last">Src homology 2</p></td></tr><tr><td class="NLM_term">SCA</td><td class="NLM_def"><p class="first last">statistical coupling analysis</p></td></tr><tr><td class="NLM_term">SIRT6</td><td class="NLM_def"><p class="first last">sirtuin 6</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_52040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_52040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 198 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. A. S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, V.</span></span> <span> </span><span class="NLM_article-title">A Chemical Perspective on Allostery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6488</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00543</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00543" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6488-6502&author=A.+A.+S.+T.+Ribeiroauthor=V.+Ortiz&title=A+Chemical+Perspective+on+Allostery&doi=10.1021%2Facs.chemrev.5b00543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Perspective on Allostery</span></div><div class="casAuthors">Ribeiro, Andre A. S. T.; Ortiz, Vanessa</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6488-6502</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Much work has been done in the past decade to quantify the phenomenon of allosteric communication in proteins.  Every new study unveils an extra piece of the puzzle in our search for an understanding of allostery that allows us to make predictions on the response of a protein to medically relevant stimuli such as pathol. mutations or drug binding.  This review summarizes recent advances in the anal. of mechanisms of allosteric communication in proteins, and combines this new knowledge to offer a perspective of allostery which is consistent with chem. views of mol. processes.  First, we review recent work, particularly computational, on the characterization of signal propagation and conformational changes in allosteric proteins.  We then compare different models of allostery, and discuss the significance of the concept of an allosteric pathway.  We argue that allostery can be rationalized in terms of pathways of residues that efficiently transmit energy between different binding sites.  We then provide examples that show how this picture could account for most of the obsd. data, since energy flow may be manifested as changes in both structure and dynamics.  We conclude by acknowledging that the proposed view is still a simplification and should not be taken as a rigorous model of allosteric communication in proteins.  Nevertheless, simple pictures like this can go a long way in improving our understanding of many complex phenomena obsd. in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwFr3gYH5v97Vg90H21EOLACvtfcHk0lhbCvMpfLu6cQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGhsrY%253D&md5=13482cdbc6b7669e5a735b5ff4b98efd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00543%26sid%3Dliteratum%253Aachs%26aulast%3DRibeiro%26aufirst%3DA.%2BA.%2BS.%2BT.%26aulast%3DOrtiz%26aufirst%3DV.%26atitle%3DA%2520Chemical%2520Perspective%2520on%2520Allostery%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6488%26epage%3D6502%26doi%3D10.1021%2Facs.chemrev.5b00543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">The Concept of Allosteric Modulation: An Overview</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e223</span>– <span class="NLM_lpage">e228</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ddtec.2012.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24050272" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e223-e228&author=J.-P.+Changeux&title=The+Concept+of+Allosteric+Modulation%3A+An+Overview&doi=10.1016%2Fj.ddtec.2012.07.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DThe%2520Concept%2520of%2520Allosteric%2520Modulation%253A%2520An%2520Overview%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De223%26epage%3De228%26doi%3D10.1016%2Fj.ddtec.2012.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dokholyan, N. V.</span></span> <span> </span><span class="NLM_article-title">Controlling Allosteric Networks in Proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6463</span>– <span class="NLM_lpage">6487</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6463-6487&author=N.+V.+Dokholyan&title=Controlling+Allosteric+Networks+in+Proteins&doi=10.1021%2Facs.chemrev.5b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Controlling Allosteric Networks in Proteins</span></div><div class="casAuthors">Dokholyan, Nikolay V.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6463-6487</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the phys. and evolutionary origin of protein allostery defined as conformational changes induced by ligand binding, as well as its importance to protein regulation, drug discovery, and biol. processes in living systems; including exptl. and computational protein engineering approaches for control of protein function by modulation of allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmC_LdB7LS-bVg90H21EOLACvtfcHk0lj9CC8XbB4bFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVeltL0%253D&md5=9294593fdeb42a59d015bd84a10091b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00544%26sid%3Dliteratum%253Aachs%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26atitle%3DControlling%2520Allosteric%2520Networks%2520in%2520Proteins%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6463%26epage%3D6487%26doi%3D10.1021%2Facs.chemrev.5b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radivojac, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Post-Translational Modification Codes</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">455</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22884395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKisbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2012&pages=447-455&author=R.+Nussinovauthor=C.+J.+Tsaiauthor=F.+Xinauthor=P.+Radivojac&title=Allosteric+Post-Translational+Modification+Codes&doi=10.1016%2Fj.tibs.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric post-translational modification codes</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Xin, Fuxiao; Radivojac, Predrag</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">447-455</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) have been recognized to impact protein function in two ways: (i) orthosterically, via direct recognition by protein domains or through interference with binding; and (ii) allosterically, via conformational changes induced at the functional sites.  Because different chem. types of PTMs elicit different structural alterations, the effects of combinatorial codes of PTMs are vastly larger than previously believed.  Combined with orthosteric PTMs, the impact of PTMs on cellular regulation is immense.  From an evolutionary standpoint, harnessing this immense, yet highly specific, PTM code is an extremely efficient vehicle that can save a cell several-fold in gene no. and speed up its response to environmental change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzvEY3AcxUjrVg90H21EOLACvtfcHk0lj9CC8XbB4bFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKisbfJ&md5=93f2ef54896d461366150e9da3b7450f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DXin%26aufirst%3DF.%26aulast%3DRadivojac%26aufirst%3DP.%26atitle%3DAllosteric%2520Post-Translational%2520Modification%2520Codes%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2012%26volume%3D37%26spage%3D447%26epage%3D455%26doi%3D10.1016%2Fj.tibs.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">Allostery in Disease and in Drug Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2013.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23582321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=293-305&author=R.+Nussinovauthor=C.+J.+Tsai&title=Allostery+in+Disease+and+in+Drug+Discovery&doi=10.1016%2Fj.cell.2013.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in Disease and in Drug Discovery</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-305</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Allostery is largely assocd. with conformational and functional transitions in individual proteins.  This concept can be extended to consider the impact of conformational perturbations on cellular function and disease states.  Here, we clarify the concept of allostery and how it controls physiol. activities.  We focus on the challenging questions of how allostery can both cause disease and contribute to development of new therapeutics.  We aim to increase the awareness of the linkage between disease symptoms on the cellular level and specific aberrant allosteric actions on the mol. level and to emphasize the potential of allosteric drugs in innovative therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5bXPgUjfFLVg90H21EOLACvtfcHk0lj9CC8XbB4bFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D&md5=14a91c0d364187044e5ad3e048868514</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DAllostery%2520in%2520Disease%2520and%2520in%2520Drug%2520Discovery%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D293%26epage%3D305%26doi%3D10.1016%2Fj.cell.2013.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodey, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Regulation and Catalysis Emerge via a Common Route</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">482</span>, <span class="refDoi"> DOI: 10.1038/nchembio.98</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.98" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18641628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslClsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=474-482&author=N.+M.+Goodeyauthor=S.+J.+Benkovic&title=Allosteric+Regulation+and+Catalysis+Emerge+via+a+Common+Route&doi=10.1038%2Fnchembio.98"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric regulation and catalysis emerge via a common route</span></div><div class="casAuthors">Goodey, Nina M.; Benkovic, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">474-482</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric regulation of protein function is a mechanism by which an event in one place of a protein structure causes an effect at another site, much like the behavior of a telecommunications network in which a collection of transmitters, receivers and transceivers communicate with each other across long distances.  For example, ligand binding or an amino acid mutation at an allosteric site can alter enzymic activity or binding affinity in a distal region such as the active site or a second binding site.  The mechanism of this site-to-site communication is of great interest, esp. since allosteric effects must be considered in drug design and protein engineering.  In this review, conformational mobility as the common route between allosteric regulation and catalysis is discussed.  We summarize recent exptl. data and the resulting insights into allostery within proteins, and we discuss the nature of future studies and the new applications that may result from increased understanding of this regulatory mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5blf5uB1i9bVg90H21EOLACvtfcHk0lj9CC8XbB4bFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslClsb8%253D&md5=74d502f64c00fcef21c623bc84db3159</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.98&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.98%26sid%3Dliteratum%253Aachs%26aulast%3DGoodey%26aufirst%3DN.%2BM.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26atitle%3DAllosteric%2520Regulation%2520and%2520Catalysis%2520Emerge%2520via%2520a%2520Common%2520Route%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D474%26epage%3D482%26doi%3D10.1038%2Fnchembio.98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Conformational Barcodes Direct Signaling in the Cell</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2013.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24010710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSht77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1509-1521&author=R.+Nussinovauthor=B.+Maauthor=C.+J.+Tsaiauthor=P.+Csermely&title=Allosteric+Conformational+Barcodes+Direct+Signaling+in+the+Cell&doi=10.1016%2Fj.str.2013.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Conformational Barcodes Direct Signaling in the Cell</span></div><div class="casAuthors">Nussinov, Ruth; Ma, Buyong; Tsai, Chung-Jung; Csermely, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1509-1521</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The cellular network is highly interconnected.  Pathways merge and diverge.  They proceed through shared proteins and may change directions.  How are cellular pathways controlled and their directions decided, coded, and read.  These questions become particularly acute when we consider that a small no. of pathways, such as signaling pathways that regulate cell fates, cell proliferation, and cell death in development, are extensively exploited.  This review focuses on these signaling questions from the structural standpoint and discusses the literature in this light.  All co-occurring allosteric events (including posttranslational modifications, pathogen binding, and gain-of-function mutations) collectively tag the protein functional site with a unique barcode.  The barcode shape is read by an interacting mol., which transmits the signal.  A conformational barcode provides an intracellular address label, which selectively favors binding to one partner and quenches binding to others, and, in this way, dets. the pathway direction, and, eventually, the cell's response and fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGlw4jCU8JqLVg90H21EOLACvtfcHk0lin0Twh6LsqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSht77L&md5=02198e089623b6d6f09b6c53a3965935</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DCsermely%26aufirst%3DP.%26atitle%3DAllosteric%2520Conformational%2520Barcodes%2520Direct%2520Signaling%2520in%2520the%2520Cell%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1509%26epage%3D1521%26doi%3D10.1016%2Fj.str.2013.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Receptors: From Electric Organ to Cognition</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.010909.105741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev.pharmtox.010909.105741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20055696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisVeltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1-38&author=J.-P.+Changeux&title=Allosteric+Receptors%3A+From+Electric+Organ+to+Cognition&doi=10.1146%2Fannurev.pharmtox.010909.105741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric receptors: From electric organ to cognition</span></div><div class="casAuthors">Changeux, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-38</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  This autobiog. account by Jean-Pierre Chageux covers the past 50 yr of allosteric protein research beginning with the development of the concept of allosteric proteins and its application to pharmacol. receptors.  It continues with the identification of the nicotinic acetylcholine receptor, the discovery of its mol. organization, the structure of the acetylcholine-binding site and of the ion channel, and the demonstration of its allosteric transitions.  The article then traces the origins of the concept of allosteric modulator and its consequences in pharmacol.  It proceeds with the theory of selective stabilization of synapses and with exptl. studies carried out on the contribution of the nicotinic receptor in the morphogenesis of the neuromuscular junction.  The knowledge acquired with the nicotinic receptor is further exploited to reach higher levels of brain organization, and the contribution of nicotinic receptors to the action of nicotine on reward and cognition is explored, in particular, using a novel exptl. strategy that combines nicotinic receptor genes knock-out and stereotaxic gene re-expression.  Theor. models of cognitive functions are proposed that link the mol. to the cognitive level.  The report ends with a discussion on nicotinic receptors and the pharmacol. of the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJFMHTsb-Mx7Vg90H21EOLACvtfcHk0lin0Twh6LsqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisVeltbs%253D&md5=60318b638eacddd3e43fce0f86bc3869</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.010909.105741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.010909.105741%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DAllosteric%2520Receptors%253A%2520From%2520Electric%2520Organ%2520to%2520Cognition%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D1%26epage%3D38%26doi%3D10.1146%2Fannurev.pharmtox.010909.105741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratcioglu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Ras Conformational Ensembles, Allostery, and Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6607</span>– <span class="NLM_lpage">6665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6607-6665&author=S.+Luauthor=H.+Jangauthor=S.+Muratciogluauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinovauthor=J.+Zhang&title=Ras+Conformational+Ensembles%2C+Allostery%2C+and+Signaling&doi=10.1021%2Facs.chemrev.5b00542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ras Conformational Ensembles, Allostery, and Signaling</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Muratcioglu, Serena; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6607-6665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are classical members of small GTPases that function as mol. switches by alternating between inactive GDP-bound and active GTP-bound states.  Ras activation is regulated by guanine nucleotide exchange factors that catalyze the exchange of GDP by GTP, and inactivation is terminated by GTPase-activating proteins that accelerate the intrinsic GTP hydrolysis rate by orders of magnitude.  In this review, we focus on data that have accumulated over the past few years pertaining to the conformational ensembles and the allosteric regulation of Ras proteins and their interpretation from our conformational landscape standpoint.  The Ras ensemble embodies all states, including the ligand-bound conformations, the activated (or inactivated) allosteric modulated states, post-translationally modified states, mutational states, transition states, and nonfunctional states serving as a reservoir for emerging functions.  The ensemble is shifted by distinct mutational events, cofactors, post-translational modifications, and different membrane compns.  A better understanding of Ras biol. can contribute to therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2oezVaRCvbVg90H21EOLACvtfcHk0lin0Twh6LsqSQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D&md5=7600b971015bd3cf635ca1b4299dc7d5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00542%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DMuratcioglu%26aufirst%3DS.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRas%2520Conformational%2520Ensembles%252C%2520Allostery%252C%2520and%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6607%26epage%3D6665%26doi%3D10.1021%2Facs.chemrev.5b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lisi, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, J. P.</span></span> <span> </span><span class="NLM_article-title">Allostery in Enzyme Catalysis</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2017.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2017.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28865247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2017&pages=123-130&author=G.+P.+Lisiauthor=J.+P.+Loria&title=Allostery+in+Enzyme+Catalysis&doi=10.1016%2Fj.sbi.2017.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in enzyme catalysis</span></div><div class="casAuthors">Lisi, George P.; Loria, J. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-130</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Modern interpretations of allostery typically rely on conformational ensembles to describe enzyme function.  Conformational motions controlling these ensembles are often stimulated or quenched by allosteric effectors, and are crit. to optimizing ligand binding pockets and enzyme architectures.  Thus, enzymes rely on dynamic allosteric pathways that transmit long-range binding information to control catalysis.  In this review, we provide a brief discussion of the ever-expanding principles of allosteric regulation in enzyme catalysis and highlight in-depth studies of three enzymes that have contributed to the paradigms of dynamic allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaQWntFm9s0rVg90H21EOLACvtfcHk0lgN0fkmgy7_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnO&md5=6f017e27e75b6e139ec75058d2b5326e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2017.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2017.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DLisi%26aufirst%3DG.%2BP.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26atitle%3DAllostery%2520in%2520Enzyme%2520Catalysis%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D47%26spage%3D123%26epage%3D130%26doi%3D10.1016%2Fj.sbi.2017.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, A. W.</span></span> <span> </span><span class="NLM_article-title">Allostery: An Illustrated Definition for the “Second Secret of Life”</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">425</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2008.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2008.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18706817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVyktLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=420-425&author=A.+W.+Fenton&title=Allostery%3A+An+Illustrated+Definition+for+the+%E2%80%9CSecond+Secret+of+Life%E2%80%9D&doi=10.1016%2Fj.tibs.2008.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery: an illustrated definition for the 'second secret of life'</span></div><div class="casAuthors">Fenton, Aron W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">420-425</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although allosteric regulation is the 'second secret of life', the mol. mechanisms that give rise to allostery currently elude understanding.  In my opinion, exptl. progress is hampered by a commonly used but misleading definition of allostery as protein structural changes that are elicited by the binding of a single ligand.  Allostery is more strictly defined in functional terms as a comparison of how one ligand binds in the absence, vs. the presence, of a second ligand.  Therefore, as each of the two binding events involves two protein complexes, a study of allostery must consider four complexes and not just two.  Such a comparison can distinguish allosteric from non-allosteric protein changes, the importance of which is frequently overlooked.  When a study of all four complexes is not feasible, an alternative, albeit limited, strategy can identify subsets of allosteric-specific changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbEo7wm-kW9LVg90H21EOLACvtfcHk0lgN0fkmgy7_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVyktLfO&md5=34040a91726dd4fe1174bf00e4a4e59c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2008.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2008.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DA.%2BW.%26atitle%3DAllostery%253A%2520An%2520Illustrated%2520Definition%2520for%2520the%2520%25E2%2580%259CSecond%2520Secret%2520of%2520Life%25E2%2580%259D%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D420%26epage%3D425%26doi%3D10.1016%2Fj.tibs.2008.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation as a Unifying Mechanism for Receptor Function and Regulation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">1084</span>– <span class="NLM_lpage">1102</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2016.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27565340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVehu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=1084-1102&author=J.+P.+Changeuxauthor=A.+Christopoulos&title=Allosteric+Modulation+as+a+Unifying+Mechanism+for+Receptor+Function+and+Regulation&doi=10.1016%2Fj.cell.2016.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modulation as a Unifying Mechanism for Receptor Function and Regulation</span></div><div class="casAuthors">Changeux, Jean-Pierre; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1084-1102</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Four major receptor families enable cells to respond to chem. and phys. signals from their proximal environment.  The ligand- and voltage-gated ion channels, G-protein-coupled receptors, nuclear hormone receptors, and receptor tyrosine kinases are all allosteric proteins that carry multiple, spatially distinct, yet conformationally linked ligand-binding sites.  Recent studies point to common mechanisms governing the allosteric transitions of these receptors, including the impact of oligomerization, pre-existing and functionally distinct conformational ensembles, intrinsically disordered regions, and the occurrence of allosteric modulatory sites.  Importantly, synthetic allosteric modulators are being discovered for these receptors, providing an enriched, yet challenging, landscape for novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCsajEiGEo6rVg90H21EOLACvtfcHk0lgN0fkmgy7_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVehu7zK&md5=7507e4abbd4c08319e8d80ded942c425</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520Modulation%2520as%2520a%2520Unifying%2520Mechanism%2520for%2520Receptor%2520Function%2520and%2520Regulation%26jtitle%3DCell%26date%3D2016%26volume%3D166%26spage%3D1084%26epage%3D1102%26doi%3D10.1016%2Fj.cell.2016.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span> <span> </span><span class="NLM_article-title">Emerging Paradigms in GPCR Allostery: Implications for Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">630</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd4052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd4052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23903222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=630-644&author=D.+Woottenauthor=A.+Christopoulosauthor=P.+M.+Sexton&title=Emerging+Paradigms+in+GPCR+Allostery%3A+Implications+for+Drug+Discovery&doi=10.1038%2Fnrd4052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging paradigms in GPCR allostery: implications for drug discovery</span></div><div class="casAuthors">Wootten, Denise; Christopoulos, Arthur; Sexton, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">630-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric ligands bind to G protein-coupled receptors (GPCRs; also known as seven-transmembrane receptors) at sites that are distinct from the sites to which endogenous ligands bind.  The existence of allosteric ligands has enriched the ways in which the functions of GPCRs can be manipulated for potential therapeutic benefit, yet the complexity of their actions provides both challenges and opportunities for drug screening and development.  Converging avenues of research in areas such as biased signalling by allosteric ligands and the mechanisms by which allosteric ligands modulate the effects of diverse endogenous ligands have provided new insights into how interactions between allosteric ligands and GPCRs could be exploited for drug discovery.  These new findings have the potential to alter how screening for allosteric drugs is performed and may increase the chances of success in the development of allosteric modulators as clin. lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxRlmQUCUlsLVg90H21EOLACvtfcHk0lgN0fkmgy7_SQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO&md5=59b74e5fbd5eeb832785dd879daa7c77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd4052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4052%26sid%3Dliteratum%253Aachs%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26atitle%3DEmerging%2520Paradigms%2520in%2520GPCR%2520Allostery%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D630%26epage%3D644%26doi%3D10.1038%2Fnrd4052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rettenmaier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Mimic of a Peptide Docking Motif Inhibits the Protein Kinase PDK1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18590</span>– <span class="NLM_lpage">18595</span>, <span class="refDoi"> DOI: 10.1073/pnas.1415365112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1415365112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25518860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18590-18595&author=T.+J.+Rettenmaierauthor=J.+D.+Sadowskyauthor=N.+D.+Thomsenauthor=S.+C.+Chenauthor=A.+K.+Doakauthor=M.+R.+Arkinauthor=J.+A.+Wells&title=A+Small-Molecule+Mimic+of+a+Peptide+Docking+Motif+Inhibits+the+Protein+Kinase+PDK1&doi=10.1073%2Fpnas.1415365112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span></div><div class="casAuthors">Rettenmaier, T. Justin; Sadowsky, Jack D.; Thomsen, Nathan D.; Chen, Steven C.; Doak, Allison K.; Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18590-18595</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is great interest in developing selective protein kinase inhibitors by targeting allosteric sites, but these sites often involve protein-protein or protein-peptide interfaces that are very challenging to target with small mols.  Here we present a systematic approach to targeting a functionally conserved allosteric site on the protein kinase PDK1 called the PDK1-interacting fragment (PIF)tide-binding site, or PIF pocket.  More than two dozen prosurvival and progrowth kinases dock a conserved peptide tail into this binding site, which recruits them to PDK1 to become activated.  Using a site-directed chem. screen, we identified and chem. optimized ligand-efficient, selective, and cell-penetrant small mols. (mol. wt. ∼380 Da) that compete with the peptide docking motif for binding to PDK1.  We solved the first high-resoln. structure of a peptide docking motif (PIFtide) bound to PDK1 and mapped binding energy hot spots using mutational anal.  We then solved structures of PDK1 bound to the allosteric small mols., which revealed a binding mode that remarkably mimics three of five hot-spot residues in PIFtide.  These allosteric small mols. are substrate-selective PDK1 inhibitors when used as single agents, but when combined with an ATP-competitive inhibitor, they completely suppress the activation of the downstream kinases.  This work provides a promising new scaffold for the development of high-affinity PIF pocket ligands, which may be used to enhance the anticancer activity of existing PDK1 inhibitors.  Moreover, our results provide further impetus for exploring the helix αC patches of other protein kinases as potential therapeutic targets even though they involve protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeMoENY8xXbVg90H21EOLACvtfcHk0lheDA0Xnw5Nnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ&md5=4a1d88395865677d5a9c8d3dee58eca2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1415365112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1415365112%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DThomsen%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520Small-Molecule%2520Mimic%2520of%2520a%2520Peptide%2520Docking%2520Motif%2520Inhibits%2520the%2520Protein%2520Kinase%2520PDK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18590%26epage%3D18595%26doi%3D10.1073%2Fpnas.1415365112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierks, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojciechowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendrich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bcr-Abl by Combining Allosteric with ATP-Binding-Site Inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1038/nature08675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature08675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=501-506&author=J.+Zhangauthor=F.+J.+Adri%C3%A1nauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+G.+Liauthor=R.+E.+Iacobauthor=T.+Simauthor=J.+Powersauthor=C.+Dierksauthor=F.+Sunauthor=G.+R.+Guoauthor=Q.+Dingauthor=B.+Okramauthor=Y.+Choiauthor=A.+Wojciechowskiauthor=X.+Dengauthor=G.+Liuauthor=G.+Fendrichauthor=A.+Straussauthor=N.+Vajpaiauthor=S.+Grzesiekauthor=T.+Tuntlandauthor=Y.+Liuauthor=B.+Bursulayaauthor=M.+Azamauthor=P.+W.+Manleyauthor=J.+R.+Engenauthor=G.+Q.+Daleyauthor=M.+Warmuthauthor=N.+S.+Gray&title=Targeting+Bcr-Abl+by+Combining+Allosteric+with+ATP-Binding-Site+Inhibitors&doi=10.1038%2Fnature08675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Bcr-Abl by combining allosteric with ATP-binding-site inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Adrian, Francisco J.; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Li, Allen G.; Iacob, Roxana E.; Sim, Taebo; Powers, John; Dierks, Christine; Sun, Fangxian; Guo, Gui-Rong; Ding, Qiang; Okram, Barun; Choi, Yongmun; Wojciechowski, Amy; Deng, Xianming; Liu, Guoxun; Fendrich, Gabriele; Strauss, Andre; Vajpai, Navratna; Grzesiek, Stephan; Tuntland, Tove; Liu, Yi; Bursulaya, Badry; Azam, Mohammad; Manley, Paul W.; Engen, John R.; Daley, George Q.; Warmuth, Markus; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">501-506</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In an effort to find new pharmacol. modalities to overcome resistance to ATP-binding-site inhibitors of Bcr-Abl, we recently reported the discovery of GNF-2, a selective allosteric Bcr-Abl inhibitor.  Here, using soln. NMR, X-ray crystallog., mutagenesis and hydrogen exchange mass spectrometry, we show that GNF-2 binds to the myristate-binding site of Abl, leading to changes in the structural dynamics of the ATP-binding site.  GNF-5, an analog of GNF-2 with improved pharmacokinetic properties, when used in combination with the ATP-competitive inhibitors imatinib or nilotinib, suppressed the emergence of resistance mutations in vitro, displayed additive inhibitory activity in biochem. and cellular assays against T315I mutant human Bcr-Abl and displayed in vivo efficacy against this recalcitrant mutant in a murine bone-marrow transplantation model.  These results show that therapeutically relevant inhibition of Bcr-Abl activity can be achieved with inhibitors that bind to the myristate-binding site and that combining allosteric and ATP-competitive inhibitors can overcome resistance to either agent alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUtkKFuTQx1LVg90H21EOLACvtfcHk0lheDA0Xnw5Nnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D&md5=46c122384fa004d27865c77f4a9165b7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature08675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLi%26aufirst%3DA.%2BG.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DDierks%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DG.%2BR.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DOkram%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DWojciechowski%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DStrauss%26aufirst%3DA.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DDaley%26aufirst%3DG.%2BQ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Bcr-Abl%2520by%2520Combining%2520Allosteric%2520with%2520ATP-Binding-Site%2520Inhibitors%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D501%26epage%3D506%26doi%3D10.1038%2Fnature08675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wenthur, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Drugs for Allosteric Sites on Receptors</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-pharmtox-010611-134525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24111540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVSqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=165-184&author=C.+J.+Wenthurauthor=P.+R.+Gentryauthor=T.+P.+Mathewsauthor=C.+W.+Lindsley&title=Drugs+for+Allosteric+Sites+on+Receptors&doi=10.1146%2Fannurev-pharmtox-010611-134525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for allosteric sites on receptors</span></div><div class="casAuthors">Wenthur, Cody J.; Gentry, Patrick R.; Mathews, Thomas P.; Lindlsye, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">165-184</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The presence of druggable, topog. distinct allosteric sites on a wide range of receptor families has offered new paradigms for small mols. to modulate receptor function.  Moreover, ligands that target allosteric sites offer significant advantages over the corresponding orthosteric ligands in terms of selectivity, including subtype selectivity within receptor families, and can also impart improved physicochem. properties.  However, allosteric ligands are not a panacea.  Many chem. issues (e.g., flat structure-activity relationships) and pharmacol. issues (e.g., ligand-biased signaling) that are allosteric centric have emerged.  Notably, the fact that allosteric sites are less evolutionarily conserved leads to improved selectivity; however, this can also lead to species differences that can hinder safety assessment.  Many allosteric ligands possess mol. switches, wherein a small structural change (chem. or metabolic) can modulate the mode of pharmacol. or receptor subtype selectivity.  As the field has matured, as described here, key principles and strategies have emerged for the design of ligands/drugs for allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx6zv9s5vbLLVg90H21EOLACvtfcHk0lheDA0Xnw5Nnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVSqtrs%253D&md5=56aff98cecab954cbabac96c94f25f47</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134525%26sid%3Dliteratum%253Aachs%26aulast%3DWenthur%26aufirst%3DC.%2BJ.%26aulast%3DGentry%26aufirst%3DP.%2BR.%26aulast%3DMathews%26aufirst%3DT.%2BP.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DDrugs%2520for%2520Allosteric%2520Sites%2520on%2520Receptors%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2014%26volume%3D54%26spage%3D165%26epage%3D184%26doi%3D10.1146%2Fannurev-pharmtox-010611-134525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Smet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Targeting of Receptor Tyrosine Kinases</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1120</span>, <span class="refDoi"> DOI: 10.1038/nbt.3028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnbt.3028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25380447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Shu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1113-1120&author=F.+De%0ASmetauthor=A.+Christopoulosauthor=P.+Carmeliet&title=Allosteric+Targeting+of+Receptor+Tyrosine+Kinases&doi=10.1038%2Fnbt.3028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric targeting of receptor tyrosine kinases</span></div><div class="casAuthors">De Smet, Frederik; Christopoulos, Arthur; Carmeliet, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1113-1120</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug discovery landscape has been transformed over the past decade by the discovery of allosteric modulators of all major mammalian receptor superfamilies.  Allosteric ligands are a rich potential source of drugs and drug targets with clear therapeutic advantages.  G protein-coupled receptors, ligand-gated ion channels and intracellular nuclear hormone receptors have all been targeted by allosteric modulators.  More recently, a receptor tyrosine kinase (RTK) has been targeted by an extracellular small-mol. allosteric modulator.  Allosteric mechanisms of structurally distinct mols. that target the various receptor families are more alike than originally anticipated and include selectivity, orthosteric probe dependence and pathway-biased signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhUEiFrNdlxrVg90H21EOLACvtfcHk0lgLt45qyOpdzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Shu7c%253D&md5=5dd8e5bf1d6578a418dce00686cd2334</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3028%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSmet%26aufirst%3DF.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DCarmeliet%26aufirst%3DP.%26atitle%3DAllosteric%2520Targeting%2520of%2520Receptor%2520Tyrosine%2520Kinases%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D1113%26epage%3D1120%26doi%3D10.1038%2Fnbt.3028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwood, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask-Andersen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiöth, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Trends in GPCR Drug Discovery: New Agents, Targets and Indications</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd.2017.178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29075003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslansbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=829-842&author=A.%0AS.+Hauserauthor=M.+M.+Attwoodauthor=M.+Rask-Andersenauthor=H.+B.+Schi%C3%B6thauthor=D.+E.+Gloriam&title=Trends+in+GPCR+Drug+Discovery%3A+New+Agents%2C+Targets+and+Indications&doi=10.1038%2Fnrd.2017.178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in GPCR drug discovery: new agents, targets and indications</span></div><div class="casAuthors">Hauser, Alexander S.; Attwood, Misty M.; Rask-Andersen, Mathias; Schioth, Helgi B.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">829-842</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are the most intensively studied drug targets, mostly due to their substantial involvement in human pathophysiol. and their pharmacol. tractability.  Here, we report an up-to-date anal. of all GPCR drugs and agents in clin. trials, which reveals current trends across mol. types, drug targets and therapeutic indications, including showing that 475 drugs (∼34% of all drugs approved by the US Food and Drug Administration (FDA)) act at 108 unique GPCRs.  Approx. 321 agents are currently in clin. trials, of which ∼20% target 66 potentially novel GPCR targets without an approved drug, and the no. of biol. drugs, allosteric modulators and biased agonists has increased.  The major disease indications for GPCR modulators show a shift towards diabetes, obesity and Alzheimer disease, although several central nervous system disorders are also highly represented.  The 224 (56%) non-olfactory GPCRs that have not yet been explored in clin. trials have broad untapped therapeutic potential, particularly in genetic and immune system disorders.  Finally, we provide an interactive online resource to analyze and infer trends in GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9iw1QqR9GKrVg90H21EOLACvtfcHk0lgLt45qyOpdzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslansbzF&md5=560893258493bd96c0af24d3bef08b6f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.178%26sid%3Dliteratum%253Aachs%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DAttwood%26aufirst%3DM.%2BM.%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DSchi%25C3%25B6th%26aufirst%3DH.%2BB.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DTrends%2520in%2520GPCR%2520Drug%2520Discovery%253A%2520New%2520Agents%252C%2520Targets%2520and%2520Indications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D829%26epage%3D842%26doi%3D10.1038%2Fnrd.2017.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Searching for New Allosteric Sites in Enzymes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">715</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15582395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVClurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=706-715&author=J.+A.+Hardyauthor=J.+A.+Wells&title=Searching+for+New+Allosteric+Sites+in+Enzymes&doi=10.1016%2Fj.sbi.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for new allosteric sites in enzymes</span></div><div class="casAuthors">Hardy, Jeanne A.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">706-715</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  The discovery of new allosteric sites generates opportunities for the identification of novel pharmaceuticals and increases the understanding of basic biol. processes.  Increasingly, allosteric sites are being discovered in various families of proteins by several methods, paving the way for the development of entirely new classes of drugs with a wide range of chemotypes.  New allosteric sites in enzymes have been discovered both incidentally and by directed means, and the mechanisms by which allosteric activation and inhibition occur at these sites have been investigated.  By exploring recent structurally well-characterized examples, trends have begun to emerge for both the modes of binding and mechanisms of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatMfcNeAgPrVg90H21EOLACvtfcHk0lgLt45qyOpdzg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVClurnO&md5=27470dff57a3b3756656c23664dd0da7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DJ.%2BA.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSearching%2520for%2520New%2520Allosteric%2520Sites%2520in%2520Enzymes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D706%26epage%3D715%26doi%3D10.1016%2Fj.sbi.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD v3.0: Unraveling Allosteric Regulation with Structural Mechanisms and Biological Networks</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D527</span>– <span class="NLM_lpage">D535</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkv902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26365237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D527-D535&author=Q.+Shenauthor=G.+Wangauthor=S.+Liauthor=X.+Liuauthor=S.+Luauthor=Z.+Chenauthor=K.+Songauthor=J.+Yanauthor=L.+Gengauthor=Z.+Huangauthor=W.+Huangauthor=G.+Chenauthor=J.+Zhang&title=ASD+v3.0%3A+Unraveling+Allosteric+Regulation+with+Structural+Mechanisms+and+Biological+Networks&doi=10.1093%2Fnar%2Fgkv902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ASD v3.0: unraveling allosteric regulation with structural mechanisms and biological networks</span></div><div class="casAuthors">Shen, Qiancheng; Wang, Guanqiao; Li, Shuai; Liu, Xinyi; Lu, Shaoyong; Chen, Zhongjie; Song, Kun; Yan, Junhao; Geng, Lv; Huang, Zhimin; Huang, Wenkang; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D527-D535</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allosteric regulation, the most direct and efficient way of regulating protein function, is induced by the binding of a ligand at one site that is topog. distinct from an orthosteric site.  Allosteric Database (ASD, available online at http://mdl.shsmu.edu.cn/ ASD) has been developed to provide comprehensive information featuring allosteric regulation.  With increasing data, fundamental questions pertaining to allostery are currently receiving more attention from the mechanism of allosteric changes in an individual protein to the entire effect of the changes in the interconnected network in the cell.  Thus, the following novel features were added to this updated version: (i) structural mechanisms of more than 1600 allosteric actions were elucidated by a comparison of site structures before and after the binding of an modulator; (ii) 261 allosteric networks were identified to unveil how the allosteric action in a single protein would propagate to affect downstream proteins; (iii) two of the largest human allosteromes, protein kinases and GPCRs, were thoroughly constructed; and (iv) web interface and data organization were completely redesigned for efficient access.  In addn., allosteric data have largely expanded in this update.  These updates are useful for facilitating the investigation of allosteric mechanisms, dynamic networks and drug discoveries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuMFm_lHedLVg90H21EOLACvtfcHk0lizeBErQZhB8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfO&md5=16abbb6763bc125f68c67fe8f2f812dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv902%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%2520v3.0%253A%2520Unraveling%2520Allosteric%2520Regulation%2520with%2520Structural%2520Mechanisms%2520and%2520Biological%2520Networks%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD527%26epage%3DD535%26doi%3D10.1093%2Fnar%2Fgkv902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durrant, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">Emerging Computational Methods for the Rational Discovery of Allosteric Drugs</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6370</span>– <span class="NLM_lpage">6390</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVWis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6370-6390&author=J.+R.+Wagnerauthor=C.+T.+Leeauthor=J.+D.+Durrantauthor=R.+D.+Malmstromauthor=V.+A.+Feherauthor=R.+E.+Amaro&title=Emerging+Computational+Methods+for+the+Rational+Discovery+of+Allosteric+Drugs&doi=10.1021%2Facs.chemrev.5b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Computational Methods for the Rational Discovery of Allosteric Drugs</span></div><div class="casAuthors">Wagner, Jeffrey R.; Lee, Christopher T.; Durrant, Jacob D.; Malmstrom, Robert D.; Feher, Victoria A.; Amaro, Rommie E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6370-6390</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric drug development holds promise for delivering medicines that are more selective and less toxic than those that target orthosteric sites.  To date, the discovery of allosteric binding sites and lead compds. has been mostly serendipitous, achieved through high-throughput screening.  Over the past decade, structural data has become more readily available for larger protein systems and more membrane protein classes (e.g., GPCRs and ion channels), which are common allosteric drug targets.  In parallel, improved simulation methods now provide better atomistic understanding of the protein dynamics and cooperative motions that are crit. to allosteric mechanisms.  As a result of these advances, the field of predictive allosteric drug development is now on the cusp of a new era of rational structure-based computational methods.  Here, we review algorithms that predict allosteric sites based on sequence data and mol. dynamics simulations, describe tools that assess the druggability of these pockets, and discuss how Markov state models and topol. analyses provide insight into the relationship between protein dynamics and allosteric drug binding.  In each section, we first provide an overview of the various method classes before describing relevant algorithms and software packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQHNOd1yzZrVg90H21EOLACvtfcHk0lizeBErQZhB8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVWis7Y%253D&md5=f5e4bdbcbbec577e20edb2ab6ec6e96f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00631%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DJ.%2BR.%26aulast%3DLee%26aufirst%3DC.%2BT.%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DEmerging%2520Computational%2520Methods%2520for%2520the%2520Rational%2520Discovery%2520of%2520Allosteric%2520Drugs%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6370%26epage%3D6390%26doi%3D10.1021%2Facs.chemrev.5b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Designed Covalent Allosteric Modulators: An Emerging Paradigm in Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27888140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=447-453&author=S.+Luauthor=J.+Zhang&title=Designed+Covalent+Allosteric+Modulators%3A+An+Emerging+Paradigm+in+Drug+Discovery&doi=10.1016%2Fj.drudis.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent allosteric modulators possess the pharmacol. advantages (high potency, extended duration of action and low drug resistance) of covalent ligands and the addnl. benefit of the higher specificity and lower toxicity of allosteric modulators.  This approach is gaining increasing recognition as a valuable tool in drug discovery.  Here, we review the recent advances in the design of covalent allosteric modulators with an emphasis on successful examples.  A broad spectrum of protein targets capable of being modulated by them reflects the prevalence of this strategy.  We also discuss the challenges and future directions in the development of covalent allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnMtSOrLWxdrVg90H21EOLACvtfcHk0lizeBErQZhB8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ&md5=bc656114f3f956ce44ba06c1a7669da4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDesigned%2520Covalent%2520Allosteric%2520Modulators%253A%2520An%2520Emerging%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.drudis.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perna, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geppert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simister, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feller, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Helicobacter Pylori HtrA Protease by Addressing a Computationally Predicted Allosteric Ligand Binding Site</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3583</span>– <span class="NLM_lpage">3590</span>, <span class="refDoi"> DOI: 10.1039/C4SC01443J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2FC4SC01443J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26819700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3583-3590&author=A.+M.+Pernaauthor=F.+Reisenauthor=T.+P.+Schmidtauthor=T.+Geppertauthor=M.+Pillongauthor=M.+Weiselauthor=B.+Hoyauthor=P.+C.+Simisterauthor=S.+M.+Fellerauthor=S.+Wesslerauthor=G.+Schneider&title=Inhibiting+Helicobacter+Pylori+HtrA+Protease+by+Addressing+a+Computationally+Predicted+Allosteric+Ligand+Binding+Site&doi=10.1039%2FC4SC01443J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Helicobacter pylori HtrA protease by addressing a computationally predicted allosteric ligand binding site</span></div><div class="casAuthors">Perna, Anna Maria; Reisen, Felix; Schmidt, Thomas P.; Geppert, Tim; Pillong, Max; Weisel, Martin; Hoy, Benjamin; Simister, Philip C.; Feller, Stephan M.; Wessler, Silja; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3583-3590</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Helicobacter pylori is assocd. with inflammatory diseases and can cause gastric cancer and mucosa-assocd. lymphoma.  One of the bacterium's key proteins is high temp. requirement A (HpHtrA) protein, an extracellular serine protease that cleaves E-cadherin of gastric epithelial cells, which leads to loss of cell-cell adhesion.  Inhibition of HpHtrA may constitute an intervention strategy against H. pylori infection.  Guided by the computational prediction of hypothetical ligand binding sites on the surface of HpHtrA, we performed residue mutation expts. that confirmed the functional relevance of an allosteric region.  We virtually screened for potential ligands addressing this surface cleft located between the catalytic and PDZ1 domains.  Our receptor-based computational method represents protein surface pockets in terms of graph frameworks and retrieves small mols. that satisfy the constraints given by the pocket framework.  A new chem. entity was identified that blocked E-cadherin cleavage in vitro by direct binding to HpHtrA, and efficiently blocked pathogen transmigration across the gastric epithelial barrier.  A preliminary crystal structure of HpHtrA confirms the validity of a comparative "homol." model of the enzyme, which we used for the computational study.  The results of this study demonstrate that addressing orphan protein surface cavities of target macromols. can lead to new bioactive ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq83JdNlgAcrVg90H21EOLACvtfcHk0lizeBErQZhB8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKrsbnF&md5=6c5842c5694fa43745ccfecaf1851e19</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC4SC01443J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC01443J%26sid%3Dliteratum%253Aachs%26aulast%3DPerna%26aufirst%3DA.%2BM.%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DT.%2BP.%26aulast%3DGeppert%26aufirst%3DT.%26aulast%3DPillong%26aufirst%3DM.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DHoy%26aufirst%3DB.%26aulast%3DSimister%26aufirst%3DP.%2BC.%26aulast%3DFeller%26aufirst%3DS.%2BM.%26aulast%3DWessler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DInhibiting%2520Helicobacter%2520Pylori%2520HtrA%2520Protease%2520by%2520Addressing%2520a%2520Computationally%2520Predicted%2520Allosteric%2520Ligand%2520Binding%2520Site%26jtitle%3DChem.%2520Sci.%26date%3D2014%26volume%3D5%26spage%3D3583%26epage%3D3590%26doi%3D10.1039%2FC4SC01443J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rambahal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagineedu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanslas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorfe, A. A.</span></span> <span> </span><span class="NLM_article-title">Andrographolide Derivatives Inhibit Guanine Nucleotide Exchange and Abrogate Oncogenic Ras Function</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">10201</span>– <span class="NLM_lpage">10206</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300016110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1300016110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23737504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOmsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10201-10206&author=H.+J.+Hockerauthor=K.-J.+Choauthor=C.-Y.+K.+Chenauthor=N.+Rambahalauthor=S.+R.+Sagineeduauthor=K.+Shaariauthor=J.+Stanslasauthor=J.+F.+Hancockauthor=A.+A.+Gorfe&title=Andrographolide+Derivatives+Inhibit+Guanine+Nucleotide+Exchange+and+Abrogate+Oncogenic+Ras+Function&doi=10.1073%2Fpnas.1300016110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Andrographolide derivatives inhibit guanine nucleotide exchange and abrogate oncogenic Ras function</span></div><div class="casAuthors">Hocker, Harrison J.; Cho, Kwang-Jin; Chen, Chung-Ying K.; Rambahal, Nandini; Sagineedu, Sreenivasa Rao; Shaari, Khozirah; Stanslas, Johnson; Hancock, John F.; Gorfe, Alemayehu A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10201-10206, S10201/1-S10201/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aberrant signaling by oncogenic mutant rat sarcoma (Ras) proteins occurs in ∼15% of all human tumors, yet direct inhibition of Ras by small mols. has remained elusive.  Recently, several small-mol. ligands have been discovered that directly bind Ras and inhibit its function by interfering with exchange factor binding.  However, it is unclear whether, or how, these ligands could lead to drugs that act against constitutively active oncogenic mutant Ras.  Using a dynamics-based pocket identification scheme, ensemble docking, and innovative cell-based assays, here we show that andrographolide (AGP) - a bicyclic diterpenoid lactone isolated from Andrographis paniculata - and its benzylidene derivs. bind to transient pockets on Kirsten-Ras (K-Ras) and inhibit GDP-GTP exchange.  As expected for inhibitors of exchange factor binding, AGP derivs. reduced GTP loading of wild-type K-Ras in response to acute EGF stimulation with a concomitant redn. in MAPK activation.  Remarkably, however, prolonged treatment with AGP derivs. also reduced GTP loading of, and signal transmission by, oncogenic mutant K-RasG12V.  In sum, the combined anal. of our computational and cell biol. results show that AGP derivs. directly bind Ras, block GDP-GTP exchange, and inhibit both wild-type and oncogenic K-Ras signaling.  Importantly, our findings not only show that nucleotide exchange factors are required for oncogenic Ras signaling but also demonstrate that inhibiting nucleotide exchange is a valid approach to abrogating the function of oncogenic mutant Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCdMqypiyfGrVg90H21EOLACvtfcHk0ljHJ48YRV8TzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOmsLzP&md5=7d39d426f5b107b87ad76cb8eb9a7f26</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300016110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300016110%26sid%3Dliteratum%253Aachs%26aulast%3DHocker%26aufirst%3DH.%2BJ.%26aulast%3DCho%26aufirst%3DK.-J.%26aulast%3DChen%26aufirst%3DC.-Y.%2BK.%26aulast%3DRambahal%26aufirst%3DN.%26aulast%3DSagineedu%26aufirst%3DS.%2BR.%26aulast%3DShaari%26aufirst%3DK.%26aulast%3DStanslas%26aufirst%3DJ.%26aulast%3DHancock%26aufirst%3DJ.%2BF.%26aulast%3DGorfe%26aufirst%3DA.%2BA.%26atitle%3DAndrographolide%2520Derivatives%2520Inhibit%2520Guanine%2520Nucleotide%2520Exchange%2520and%2520Abrogate%2520Oncogenic%2520Ras%2520Function%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D10201%26epage%3D10206%26doi%3D10.1073%2Fpnas.1300016110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Identification of a Cellularly Active SIRT6 Allosteric Activator</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0150-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fs41589-018-0150-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30374165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1118-1126&author=Z.+Huangauthor=J.+Zhaoauthor=W.+Dengauthor=Y.+Chenauthor=J.+Shangauthor=K.+Songauthor=L.+Zhangauthor=C.+Wangauthor=S.+Luauthor=X.+Yangauthor=B.+Heauthor=J.+Minauthor=H.+Huauthor=M.+Tanauthor=J.+Xuauthor=Q.+Zhangauthor=J.+Zhongauthor=X.+Sunauthor=Z.+Maoauthor=H.+Linauthor=M.+Xiaoauthor=Y.+E.+Chinauthor=H.+Jiangauthor=Y.+Xuauthor=G.+Chenauthor=J.+Zhang&title=Identification+of+a+Cellularly+Active+SIRT6+Allosteric+Activator&doi=10.1038%2Fs41589-018-0150-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a cellularly active SIRT6 allosteric activator</span></div><div class="casAuthors">Huang, Zhimin; Zhao, Junxing; Deng, Wei; Chen, Yingyi; Shang, Jialin; Song, Kun; Zhang, Lu; Wang, Chengxiang; Lu, Shaoyong; Yang, Xiuyan; He, Bin; Min, Jinrong; Hu, Hao; Tan, Minjia; Xu, Jianrong; Zhang, Qiufen; Zhong, Jie; Sun, Xiaoxiang; Mao, Zhiyong; Lin, Houwen; Xiao, Mingzhe; Chin, Y. Eugene; Jiang, Hualiang; Xu, Ying; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1126</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6, a member of the SIRT deacetylase family, is responsible for deacetylation of histone H3 Nε-acetyl-lysines 9 (H3K9ac) and 56 (H3K56ac).  As a tumor suppressor, SIRT6 has frequently been found to have low expression in various cancers.  Here, we report the identification of MDL-800, a selective SIRT6 activator.  MDL-800 increased the deacetylase activity of SIRT6 by up to 22-fold via binding to an allosteric site; this interaction led to a global decrease in H3K9ac and H3K56ac levels in human hepatocellular carcinoma (HCC) cells.  Consequently, MDL-800 inhibited the proliferation of HCC cells via SIRT6-driven cell-cycle arrest and was effective in a tumor xenograft model.  Together, these data demonstrate that pharmacol. activation of SIRT6 is a potential therapeutic approach for the treatment of HCC.  MDL-800 is a first-in-class small-mol. cellular SIRT6 activator that can be used to physiol. and pathol. investigate the roles of SIRT6 deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFPmr-laqlLVg90H21EOLACvtfcHk0ljHJ48YRV8TzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O&md5=5fb5c1de5b6be0291ffa45debf72516e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0150-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0150-0%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DChin%26aufirst%3DY.%2BE.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520a%2520Cellularly%2520Active%2520SIRT6%2520Allosteric%2520Activator%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1118%26epage%3D1126%26doi%3D10.1038%2Fs41589-018-0150-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">AlloFinder: A Strategy for Allosteric Modulator Discovery and Allosterome Analyses</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W451</span>– <span class="NLM_lpage">W458</span>, <span class="refDoi"> DOI: 10.1093/nar/gky374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29757429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W451-W458&author=M.+Huangauthor=K.+Songauthor=X.+Liuauthor=S.+Luauthor=Q.+Shenauthor=R.+Wangauthor=J.+Gaoauthor=Y.+Hongauthor=Q.+Liauthor=D.+Niauthor=J.+Xuauthor=G.+Chenauthor=J.+Zhang&title=AlloFinder%3A+A+Strategy+for+Allosteric+Modulator+Discovery+and+Allosterome+Analyses&doi=10.1093%2Fnar%2Fgky374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">AlloFinder: a strategy for allosteric modulator discovery and allosterome analyses</span></div><div class="casAuthors">Huang, Min; Song, Kun; Liu, Xinyi; Lu, Shaoyong; Shen, Qiancheng; Wang, Renxiao; Gao, Jingze; Hong, Yuanyuan; Li, Qian; Ni, Duan; Xu, Jianrong; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W451-W458</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery tweaks innumerable biol. processes and plays a fundamental role in human disease and drug discovery.  Exploration of allostery has thus been regarded as a crucial requirement for research on biol. mechanisms and the development of novel therapeutics.  Here, based on our previously developed allosteric data and methods, we present an interactive platform called AlloFinder that identifies potential endogenous or exogenous allosteric modulators and their involvement in human allosterome.  AlloFinder automatically amalgamates allosteric site identification, allosteric screening and allosteric scoring evaluation of modulator-protein complexes to identify allosteric modulators, followed by allosterome mapping analyses of predicted allosteric sites and modulators in human proteome.  This web server exhibits prominent performance in the reemergence of allosteric metabolites and exogenous allosteric modulators in known allosteric proteins.  Specifically, AlloFinder enables identification of allosteric metabolites for metabolic enzymes and screening of potential allosteric compds. for disease-related targets.  Significantly, the feasibility of AlloFinder to discover allosteric modulators was tested in a real case of signal transduction and activation of transcription 3 (STAT3) and validated by mutagenesis and functional expts.  Collectively, AlloFinder is expected to contribute to exploration of the mechanisms of allosteric regulation between metabolites and metabolic enzymes, and to accelerate allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolD0xX0ogJuLVg90H21EOLACvtfcHk0ljCWWxrqrsIPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyru7Y%253D&md5=0144562803e465e28d9a7237f0e97a47</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky374%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DNi%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAlloFinder%253A%2520A%2520Strategy%2520for%2520Allosteric%2520Modulator%2520Discovery%2520and%2520Allosterome%2520Analyses%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW451%26epage%3DW458%26doi%3D10.1093%2Fnar%2Fgky374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeli, J. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Novel Allosteric Activators for Ferroptosis Regulator Glutathione Peroxidase 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXot1Cjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=266-275&author=C.+Liauthor=X.+Dengauthor=W.+Zhangauthor=X.+Xieauthor=M.+Conradauthor=Y.+Liuauthor=J.+P.+F.+Angeliauthor=L.+Lai&title=Novel+Allosteric+Activators+for+Ferroptosis+Regulator+Glutathione+Peroxidase+4&doi=10.1021%2Facs.jmedchem.8b00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Allosteric Activators for Ferroptosis Regulator Glutathione Peroxidase 4</span></div><div class="casAuthors">Li, Cong; Deng, Xiaobing; Zhang, Weilin; Xie, Xiaowen; Conrad, Marcus; Liu, Ying; Angeli, Jose Pedro Friedmann; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">266-275</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutathione peroxidase 4 (GPX4) is essential for cell membrane repair, inflammation suppression, and ferroptosis inhibition.  GPX4 upregulation provides unique drug discovery opportunities for inflammation and ferroptosis-related diseases.  However, rational design of protein activators is challenging.  Until now, no compd. has been reported to activate the enzyme activity of GPX4.  Here, we identified a potential allosteric site in GPX4 and successfully found eight GPX4 activators using a novel computational strategy and exptl. studies.  Compd. 1 from the virtual screen increased GPX4 activity, suppressed ferroptosis, reduced pro-inflammatory lipid mediator prodn., and inhibited NF-κB pathway activation.  Further chem. synthesis and structure-activity relationship studies revealed seven more activators.  The strongest compd., 1d4, increased GPX4 activity to 150% at 20 μM in the cell-free assay and 61 μM in cell exts.  Therefore, we demonstrated that GPX4 can be directly activated using chem. compds. to suppress ferroptosis and inflammation.  Meanwhile, the discovery of GPX4 activators verified the possibility of rational design of allosteric activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1yzEJLJGkorVg90H21EOLACvtfcHk0ljCWWxrqrsIPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXot1Cjsrs%253D&md5=f4d07515f6cbf30298ca4a11974fb50b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00315%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DConrad%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DAngeli%26aufirst%3DJ.%2BP.%2BF.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DNovel%2520Allosteric%2520Activators%2520for%2520Ferroptosis%2520Regulator%2520Glutathione%2520Peroxidase%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D266%26epage%3D275%26doi%3D10.1021%2Facs.jmedchem.8b00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, F.</span></span> <span> </span><span class="NLM_article-title">Telenomic Mechanisms in Cellular Metabolism, Growth, and Differentiation</span>. <i>Cold Spring Harbor Symp. Quant. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1961</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1101/SQB.1961.026.01.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1101%2FSQB.1961.026.01.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14475415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADyaF38%252Fns1yquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1961&pages=389-401&author=J.+Monodauthor=F.+Jacob&title=Telenomic+Mechanisms+in+Cellular+Metabolism%2C+Growth%2C+and+Differentiation&doi=10.1101%2FSQB.1961.026.01.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Teleonomic mechanisms in cellular metabolism, growth, and differentiation</span></div><div class="casAuthors">MONOD J; JACOB F</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor symposia on quantitative biology</span>
        (<span class="NLM_cas:date">1961</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389-401</span>
        ISSN:<span class="NLM_cas:issn">0091-7451</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBZpoAYqt75JPq5LRczm8ffW6udTcc2eaybCJq9aRnHbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF38%252Fns1yquw%253D%253D&md5=a09cda000efcd8f0be0baeae7fbea61a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2FSQB.1961.026.01.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252FSQB.1961.026.01.048%26sid%3Dliteratum%253Aachs%26aulast%3DMonod%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DF.%26atitle%3DTelenomic%2520Mechanisms%2520in%2520Cellular%2520Metabolism%252C%2520Growth%252C%2520and%2520Differentiation%26jtitle%3DCold%2520Spring%2520Harbor%2520Symp.%2520Quant.%2520Biol.%26date%3D1961%26volume%3D26%26spage%3D389%26epage%3D401%26doi%3D10.1101%2FSQB.1961.026.01.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">On the Nature of Allosteric Transitions: A Plausible Model</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(65)80285-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2FS0022-2836%2865%2980285-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14343300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaF2MXkt1Ghsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1965&pages=88-188&author=J.+Monodauthor=J.+Wymanauthor=J.-P.+Changeux&title=On+the+Nature+of+Allosteric+Transitions%3A+A+Plausible+Model&doi=10.1016%2FS0022-2836%2865%2980285-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">On the nature of allosteric transitions: a plausible model</span></div><div class="casAuthors">Monod, Jacques; Wyman, Jeffries; Changeux, Jean Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1965</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-118</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A math. model is proposed for those regulatory proteins which are oligomeric in nature and the model applied with considerable success to published results on several enzymes or metabolically active proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9xh239rJ4sbVg90H21EOLACvtfcHk0lhzMl8ZoQ8XNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2MXkt1Ghsb4%253D&md5=daf94bee21838498c35c0cce65869a77</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2865%2980285-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252865%252980285-6%26sid%3Dliteratum%253Aachs%26aulast%3DMonod%26aufirst%3DJ.%26aulast%3DWyman%26aufirst%3DJ.%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DOn%2520the%2520Nature%2520of%2520Allosteric%2520Transitions%253A%2520A%2520Plausible%2520Model%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1965%26volume%3D12%26spage%3D88%26epage%3D188%26doi%3D10.1016%2FS0022-2836%2865%2980285-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koshland, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemethy, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filmer, D.</span></span> <span> </span><span class="NLM_article-title">Comparison of Experimental Binding Data and Theoretical Models in Proteins Containing Subunits</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1966</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1021/bi00865a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00865a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaF28XivVGnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1966&pages=365-385&author=D.+E.+Koshlandauthor=J.+G.+Nemethyauthor=D.+Filmer&title=Comparison+of+Experimental+Binding+Data+and+Theoretical+Models+in+Proteins+Containing+Subunits&doi=10.1021%2Fbi00865a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of experimental binding data and theoretical models in proteins containing subunits</span></div><div class="casAuthors">Koshland, D. E., Jr.; Nemethy, G.; Filmer, D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1966</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">365-85</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Models for subunit interactions are examd. by means of interpreting ligand saturation curves.  Equations are derived to indicate the effect of variables such as the strength of binding of the ligand, the geometrical arrangement of subunits, the strength of interaction between subunits, the energy of the conformation change, and the effect of nonidentical subunits.  Rapid methods, i.e., equations and nomograms, are developed to fit theoretical curves to exptl. data with a min. of parameters.  Applying these procedures to the binding of O by hemoglobin as an illustrative example, it is seen that a no. of simple models can represent the published data accurately.  In general, it appears that unique mechanisms cannot be established from ligand saturation curves by themselves, but the math. analysis of the curves indicates possible sources of addnl. information to make such distinctions possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBsA5hKxWFubVg90H21EOLACvtfcHk0lhzMl8ZoQ8XNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF28XivVGnsw%253D%253D&md5=e72f08b584282e1bfda351e41c94f28c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi00865a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00865a047%26sid%3Dliteratum%253Aachs%26aulast%3DKoshland%26aufirst%3DD.%2BE.%26aulast%3DNemethy%26aufirst%3DJ.%2BG.%26aulast%3DFilmer%26aufirst%3DD.%26atitle%3DComparison%2520of%2520Experimental%2520Binding%2520Data%2520and%2520Theoretical%2520Models%2520in%2520Proteins%2520Containing%2520Subunits%26jtitle%3DBiochemistry%26date%3D1966%26volume%3D5%26spage%3D365%26epage%3D385%26doi%3D10.1021%2Fbi00865a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryden, D. T.</span></span> <span> </span><span class="NLM_article-title">Allostery without Conformational Change. A Plausible Model</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1007/BF00276625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2FBF00276625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=6544679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaL2MXktFOrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1984&pages=103-109&author=A.+Cooperauthor=D.+T.+Dryden&title=Allostery+without+Conformational+Change.+A+Plausible+Model&doi=10.1007%2FBF00276625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery without conformational change.  A plausible model</span></div><div class="casAuthors">Cooper, A.; Dryden, D. T. F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-9</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    </div><div class="casAbstract">A general model is presented whereby ligand-induced changes in protein dynamics could produce allosteric communication between distinct binding sites, even in the absence of a macromol. conformational change.  Theor. anal., based on the statistical thermodn. of ligand binding, shows that cooperative interaction free energies amounting to several kilojoules per mol may be generated by this means.  The effect arises out of the possible changes in frequencies and amplitudes of macromol. thermal fluctuations in response to ligand attachment and can involve all forms of dynamic behavior, ranging from highly correlated, low-frequency normal mode vibrations to random local anharmonic motions of individual atoms or groups.  Dynamic allostery of this form is primarily an entropy effect, and approx. expressions are derived which might allow the magnitude of the interaction in real systems to be calcd. directly from exptl. observations such as changes in normal mode frequencies and mean-square at. displacements.  Long-range influence of kinetic processes at different sites might also be mediated by a similar mechanism.  Proteins and other biol. macromols. may have evolved to take functional advantage not only of mean conformational states but also of the inevitable thermal fluctuations about the mean.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUH5GNkFF82LVg90H21EOLACvtfcHk0lhzMl8ZoQ8XNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktFOrtA%253D%253D&md5=5981f3a82ce85d17068a8f7e8b7b09bc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2FBF00276625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00276625%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DDryden%26aufirst%3DD.%2BT.%26atitle%3DAllostery%2520without%2520Conformational%2520Change.%2520A%2520Plausible%2520Model%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D1984%26volume%3D11%26spage%3D103%26epage%3D109%26doi%3D10.1007%2FBF00276625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Igumenova, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Fast Dynamics of Protein Amino Acid Side Chains Using NMR Relaxation in Solution</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1672</span>– <span class="NLM_lpage">1699</span>, <span class="refDoi"> DOI: 10.1021/cr040422h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040422h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFyjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=1672-1699&author=T.+I.+Igumenovaauthor=K.+K.+Frederickauthor=A.+J.+Wand&title=Characterization+of+the+Fast+Dynamics+of+Protein+Amino+Acid+Side+Chains+Using+NMR+Relaxation+in+Solution&doi=10.1021%2Fcr040422h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the fast dynamics of protein amino acid side chains using NMR relaxation in solution</span></div><div class="casAuthors">Igumenova, Tatyana I.; King Frederick, Kendra; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1672-1699</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the current state-of-the-art of soln. NMR-based characterization of the fast internal dynamics of protein side chains.  A survey of basic observations about side chain dynamics derived from NMR relaxation studies is presented along with several analyses meant to dispel commonly held but apparently inaccurate correlations between dynamics, structure, and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwe-gODQ8P_LVg90H21EOLACvtfcHk0lhLcyxmawMCCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFyjtLY%253D&md5=85b6c44b477f995e26c23ea2df972770</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcr040422h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040422h%26sid%3Dliteratum%253Aachs%26aulast%3DIgumenova%26aufirst%3DT.%2BI.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DCharacterization%2520of%2520the%2520Fast%2520Dynamics%2520of%2520Protein%2520Amino%2520Acid%2520Side%2520Chains%2520Using%2520NMR%2520Relaxation%2520in%2520Solution%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D1672%26epage%3D1699%26doi%3D10.1021%2Fcr040422h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Dynamic Activation of an Allosteric Regulatory Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nature08560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature08560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19924217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVeqtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=368-372&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Dynamic+Activation+of+an+Allosteric+Regulatory+Protein&doi=10.1038%2Fnature08560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic activation of an allosteric regulatory protein</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7271</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Allosteric regulation is used as a very efficient mechanism to control protein activity in most biol. processes, including signal transduction, metab., catalysis and gene regulation.  Allosteric proteins can exist in several conformational states with distinct binding or enzymic activity.  Effectors are considered to function in a purely structural manner by selectively stabilizing a specific conformational state, thereby regulating protein activity.  Here we show that allosteric proteins can be regulated predominantly by changes in their structural dynamics.  We have used NMR spectroscopy and isothermal titrn. calorimetry to characterize cAMP binding to the catabolite activator protein (CAP), a transcriptional activator that has been a prototype for understanding effector-mediated allosteric control of protein activity.  CAMP switches CAP from the 'off' state (inactive), which binds DNA weakly and non-specifically, to the 'on' state (active), which binds DNA strongly and specifically.  In contrast, cAMP binding to a single CAP mutant, CAP-S62F, fails to elicit the active conformation; yet cAMP binding to CAP-S62F strongly activates the protein for DNA binding.  NMR and thermodn. analyses show that despite the fact that CAP-S62F-cAMP2 adopts the inactive conformation, its strong binding to DNA is driven by a large conformational entropy originating in enhanced protein motions induced by DNA binding.  The results provide strong evidence that changes in protein motions may activate allosteric proteins that are otherwise structurally inactive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohyohMBfeojrVg90H21EOLACvtfcHk0lhLcyxmawMCCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVeqtrfM&md5=66ae42c3522f19ada2ae21fb808ee9db</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature08560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08560%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DDynamic%2520Activation%2520of%2520an%2520Allosteric%2520Regulatory%2520Protein%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnature08560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Protein Activity Regulation by Conformational Entropy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">236</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/nature11271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature11271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22801505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=236-240&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Protein+Activity+Regulation+by+Conformational+Entropy&doi=10.1038%2Fnature11271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein activity regulation by conformational entropy</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7410</span>),
    <span class="NLM_cas:pages">236-240</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">How the interplay between protein structure and internal dynamics regulates protein function is poorly understood.  Often, ligand binding, post-translational modifications and mutations modify protein activity in a manner that is not possible to rationalize solely on the basis of structural data.  It is likely that changes in the internal motions of proteins have a major role in regulating protein activity, but the nature of their contributions remains elusive, esp. in quant. terms.  Here we show that changes in conformational entropy can det. whether protein-ligand interactions will occur, even among protein complexes with identical binding interfaces.  We have used NMR spectroscopy to det. the changes in structure and internal dynamics that are elicited by the binding of DNA to several variants of the catabolite activator protein (CAP) that differentially populate the inactive and active DNA-binding domain states.  We found that the CAP variants have markedly different affinities for DNA, despite the CAP-DNA-binding interfaces being essentially identical in the various complexes.  Combined with thermodn. data, the results show that conformational entropy changes can inhibit the binding of CAP variants that are structurally poised for optimal DNA binding or can stimulate the binding activity of CAP variants that only transiently populate the DNA-binding-domain active state.  Collectively, the data show how changes in fast internal dynamics (conformational entropy) and slow internal dynamics (energetically excited conformational states) can regulate binding activity in a way that cannot be predicted on the basis of the protein's ground-state structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUawV2tVuAuLVg90H21EOLACvtfcHk0lhLcyxmawMCCg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbzF&md5=5cc4c2d27af99e6efd2888372a751ca5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11271%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DProtein%2520Activity%2520Regulation%2520by%2520Conformational%2520Entropy%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D236%26epage%3D240%26doi%3D10.1038%2Fnature11271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapienza, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. L.</span></span> <span> </span><span class="NLM_article-title">Hidden Dynamic Allostery in a PDZ Domain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">18249</span>– <span class="NLM_lpage">18254</span>, <span class="refDoi"> DOI: 10.1073/pnas.0904492106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0904492106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19828436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18249-18254&author=C.+M.+Petitauthor=J.+Zhangauthor=P.+J.+Sapienzaauthor=E.+J.+Fuentesauthor=A.+L.+Lee&title=Hidden+Dynamic+Allostery+in+a+PDZ+Domain&doi=10.1073%2Fpnas.0904492106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hidden dynamic allostery in a PDZ domain</span></div><div class="casAuthors">Petit, Chad M.; Zhang, Jun; Sapienza, Paul J.; Fuentes, Ernesto J.; Lee, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18249-18254</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structure-function relationships in proteins are predicated on the spatial proximity of noncovalently interacting groups of atoms.  Thus, structural elements located away from a protein's active site are typically presumed to serve a stabilizing or scaffolding role for the larger structure.  Here we report a functional role for a distal structural element in a PDZ domain, even though it is not required to maintain PDZ structure.  The third PDZ domain from PSD-95/SAP90 (PDZ3) has an unusual addnl. third alpha helix (α3) that packs in contiguous fashion against the globular domain.  Although α3 lies outside the active site and does not make direct contact with C-terminal peptide ligand, removal of α3 reduces ligand affinity by 21-fold.  Further investigation revealed that the difference in binding free energies between the full-length and truncated constructs is predominantly entropic in nature and that without α3, picosecond-nanosecond side-chain dynamics are enhanced throughout the domain, as detd. by 2H Me NMR relaxation.  Thus, the distal modulation of binding function appears to occur via a delocalized conformational entropy mechanism.  Without removal of α3 and characterization of side-chain dynamics, this dynamic allostery would have gone unnoticed.  Moreover, what appeared at first to be an artificial modification of PDZ3 has been corroborated by exptl. verified phosphorylation of α3, revealing a tangible biol. mechanism for this novel regulatory scheme.  This hidden dynamic allostery raises the possibility of as-yet unidentified or untapped allosteric regulation in this PDZ domain and is a very clear example of function arising from dynamics rather than from structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXqoxKwIs4nbVg90H21EOLACvtfcHk0lgyLHQYLKaLhg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWrsrjO&md5=9ae15474e5320584dbe68dee381b2f5c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0904492106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0904492106%26sid%3Dliteratum%253Aachs%26aulast%3DPetit%26aufirst%3DC.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSapienza%26aufirst%3DP.%2BJ.%26aulast%3DFuentes%26aufirst%3DE.%2BJ.%26aulast%3DLee%26aufirst%3DA.%2BL.%26atitle%3DHidden%2520Dynamic%2520Allostery%2520in%2520a%2520PDZ%2520Domain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D18249%26epage%3D18254%26doi%3D10.1073%2Fpnas.0904492106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundström, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akke, M.</span></span> <span> </span><span class="NLM_article-title">Correlated Dynamics of Consecutive Residues Reveal Transient and Cooperative Unfolding of Secondary Structure in Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">16984</span>– <span class="NLM_lpage">16989</span>, <span class="refDoi"> DOI: 10.1073/pnas.0504361102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0504361102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16278300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BD2Mnit12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=16984-16989&author=P.+Lundstr%C3%B6mauthor=F.+A.+Mulderauthor=M.+Akke&title=Correlated+Dynamics+of+Consecutive+Residues+Reveal+Transient+and+Cooperative+Unfolding+of+Secondary+Structure+in+Proteins&doi=10.1073%2Fpnas.0504361102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Correlated dynamics of consecutive residues reveal transient and cooperative unfolding of secondary structure in proteins</span></div><div class="casAuthors">Lundstrom Patrik; Mulder Frans A A; Akke Mikael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">16984-9</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Nuclear spin relaxation is a powerful method for studying molecular dynamics at atomic resolution.  Recent methods development in biomolecular NMR spectroscopy has enabled detailed investigations of molecular dynamics that are critical for biological function, with prominent examples addressing allostery, enzyme catalysis, and protein folding.  Dynamic processes with similar correlation times are often detected in multiple locations of the molecule, raising the question of whether the underlying motions are correlated (corresponding to concerted fluctuations involving many atoms distributed across extended regions of the molecule) or uncorrelated (corresponding to independent fluctuations involving few atoms in localized regions).  Here, we have used (13)C(alpha)(i - 1)/(13)C(alpha)(i) differential multiple-quantum spin relaxation to provide direct evidence for correlated dynamics of consecutive amino acid residues in the protein sequence.  By monitoring overlapping pairs of residues (i - 1 and i, i and i + 1, etc.), we identified correlated motions that extend through continuous segments of the sequence.  We detected significant correlated conformational transitions in the native state of the E140Q mutant of the calmodulin C-terminal domain.  Previous work has shown that this domain exchanges between two major conformational states that resemble the functionally relevant open and closed states of the WT protein, with a mean correlation time of approximately 20 micros.  The present results reveal that an entire alpha-helix undergoes partial unraveling in a transient and cooperative manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6GqfmEoQr_mlSCSgBhwJFfW6udTcc2eazZ6QKYtdH67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2Mnit12itA%253D%253D&md5=e268cc06a712f9721c3492b1699884ee</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504361102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504361102%26sid%3Dliteratum%253Aachs%26aulast%3DLundstr%25C3%25B6m%26aufirst%3DP.%26aulast%3DMulder%26aufirst%3DF.%2BA.%26aulast%3DAkke%26aufirst%3DM.%26atitle%3DCorrelated%2520Dynamics%2520of%2520Consecutive%2520Residues%2520Reveal%2520Transient%2520and%2520Cooperative%2520Unfolding%2520of%2520Secondary%2520Structure%2520in%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D16984%26epage%3D16989%26doi%3D10.1073%2Fpnas.0504361102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Conformational Entropy in Molecular Recognition by Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>448</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1038/nature05959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature05959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17637663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=325-329&author=K.+K.+Frederickauthor=M.+S.+Marlowauthor=K.+G.+Valentineauthor=A.+J.+Wand&title=Conformational+Entropy+in+Molecular+Recognition+by+Proteins&doi=10.1038%2Fnature05959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational entropy in molecular recognition by proteins</span></div><div class="casAuthors">Frederick, Kendra King; Marlow, Michael S.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7151</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. recognition by proteins is fundamental to almost every biol. process, particularly the protein assocns. underlying cellular signal transduction.  Understanding the basis for protein-protein interactions requires the full characterization of the thermodn. of their assocn.  Historically, it has been virtually impossible to exptl. est. changes in protein conformational entropy, a potentially important component of the free energy of protein assocn.  However, NMR spectroscopy has emerged as a powerful tool for characterizing the dynamics of proteins.  Here, the authors employed changes in conformational dynamics as a proxy for corresponding changes in conformational entropy.  It was found that the change in internal dynamics of the protein, calmodulin, varied significantly on binding a variety of target domains.  Surprisingly, the apparent change in the corresponding conformational entropy was linearly related to the change in the overall binding entropy.  This indicated that changes in protein conformational entropy can contribute significantly to the free energy of protein-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohP3eEb0Q6srVg90H21EOLACvtfcHk0li7wFJiINFDvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D&md5=99cb195dbee8c4e67e348d353451dc1b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature05959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05959%26sid%3Dliteratum%253Aachs%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DConformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fnature05959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinnear, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Redistribution and Loss of Side Chain Entropy upon Formation of a Calmodulin-Peptide Complex</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1038/71280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2F71280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10625431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD3cXls1SnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=72-77&author=A.+L.+Leeauthor=S.+A.+Kinnearauthor=A.+J.+Wand&title=Redistribution+and+Loss+of+Side+Chain+Entropy+upon+Formation+of+a+Calmodulin-Peptide+Complex&doi=10.1038%2F71280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Redistribution and loss of side chain entropy upon formation of a calmodulin-peptide complex</span></div><div class="casAuthors">Lee, Andrew L.; Kinnear, Sandra A.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-77</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">The response of the internal dynamics of calcium-satd. calmodulin to the formation of a complex with a peptide model of the calmodulin-binding domain of the smooth muscle myosin light chain kinase has been studied using NMR relaxation methods.  The backbone of calmodulin is found to be unaffected by the binding of the domain, whereas the dynamics of side chains are significantly perturbed.  The changes in dynamics are interpreted in terms of a heterogeneous partitioning between structure (enthalpy) and dynamics (entropy).  These data provide a microscopic view of the residual entropy of a protein in two functional states and suggest extensive enthalpy/entropy exchange during the formation of a protein-protein interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGBIxp9SqOLVg90H21EOLACvtfcHk0li7wFJiINFDvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXls1SnsQ%253D%253D&md5=53ddeaa98a9c769b84ec531ef5400439</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2F71280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F71280%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BL.%26aulast%3DKinnear%26aufirst%3DS.%2BA.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DRedistribution%2520and%2520Loss%2520of%2520Side%2520Chain%2520Entropy%2520upon%2520Formation%2520of%2520a%2520Calmodulin-Peptide%2520Complex%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D72%26epage%3D77%26doi%3D10.1038%2F71280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kornev, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span> <span> </span><span class="NLM_article-title">Dynamics-Driven Allostery in Protein Kinases</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2015.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26481499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Orsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=628-647&author=A.+P.+Kornevauthor=S.+S.+Taylor&title=Dynamics-Driven+Allostery+in+Protein+Kinases&doi=10.1016%2Fj.tibs.2015.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics-Driven Allostery in Protein Kinases</span></div><div class="casAuthors">Kornev, Alexandr P.; Taylor, Susan S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">628-647</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases have very dynamic structures and their functionality strongly depends on their dynamic state.  Active kinases reveal a dynamic pattern with residues clustering into semirigid communities that move in μs-ms timescale.  Previously detected hydrophobic spines serve as connectors between communities.  Communities do not follow the traditional subdomain structure of the kinase core or its secondary structure elements.  Instead they are organized around main functional units.  Integration of the communities depends on the assembly of the hydrophobic spine and phosphorylation of the activation loop.  Single mutations can significantly disrupt the dynamic infrastructure and thereby interfere with long-distance allosteric signaling that propagates throughout the whole mol.  Dynamics is proposed to be the underlying mechanism for allosteric regulation in protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdvYRkTg5UTLVg90H21EOLACvtfcHk0li7wFJiINFDvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Orsb3L&md5=3c4cefa885c773608ed279b000195917</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DDynamics-Driven%2520Allostery%2520in%2520Protein%2520Kinases%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D628%26epage%3D647%26doi%3D10.1016%2Fj.tibs.2015.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Bringing Disorder and Dynamics in Protein Allostery into Focus</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">4278</span>– <span class="NLM_lpage">4280</span>, <span class="refDoi"> DOI: 10.1073/pnas.1703796114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1703796114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28420785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsFGmu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=4278-4280&author=A.+J.+Wand&title=Bringing+Disorder+and+Dynamics+in+Protein+Allostery+into+Focus&doi=10.1073%2Fpnas.1703796114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Bringing disorder and dynamics in protein allostery into focus</span></div><div class="casAuthors">Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4278-4280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprheK2WctkR7Vg90H21EOLACvtfcHk0lgG-1RFuVHkAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsFGmu7k%253D&md5=b52cb644bc8607821c7fdbd50b6bfaf2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1703796114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1703796114%26sid%3Dliteratum%253Aachs%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DBringing%2520Disorder%2520and%2520Dynamics%2520in%2520Protein%2520Allostery%2520into%2520Focus%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D4278%26epage%3D4280%26doi%3D10.1073%2Fpnas.1703796114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, J. A.</span></span> <span> </span><span class="NLM_article-title">The Role of Conformational Entropy in Molecular Recognition by Calmodulin</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1038/nchembio.347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20383153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=352-358&author=M.+S.+Marlowauthor=J.+Doganauthor=K.+K.+Frederickauthor=K.+G.+Valentineauthor=J.+A.+Wand&title=The+Role+of+Conformational+Entropy+in+Molecular+Recognition+by+Calmodulin&doi=10.1038%2Fnchembio.347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The role of conformational entropy in molecular recognition by calmodulin</span></div><div class="casAuthors">Marlow, Michael S.; Dogan, Jakob; Frederick, Kendra K.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The phys. basis for high-affinity interactions involving proteins is complex and potentially involves a range of energetic contributions.  Among these are changes in protein conformational entropy, which cannot yet be reliably computed from mol. structures.  We have recently used changes in conformational dynamics as a proxy for changes in conformational entropy of calmodulin upon assocn. with domains from regulated proteins.  The apparent change in conformational entropy was linearly related to the overall binding entropy.  This view warrants a more quant. foundation.  Here we calibrate an 'entropy meter' using an exptl. dynamical proxy based on NMR relaxation and show that changes in the conformational entropy of calmodulin are a significant component of the energetics of binding.  Furthermore, the distribution of motion at the interface between the target domain and calmodulin is surprisingly noncomplementary.  These observations promote modification of our understanding of the energetics of protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmK5VBOAHwT7Vg90H21EOLACvtfcHk0lgG-1RFuVHkAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D&md5=e9ee34fffa91f612441a77a7385015ff</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.347%26sid%3Dliteratum%253Aachs%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Role%2520of%2520Conformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Calmodulin%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D352%26epage%3D358%26doi%3D10.1038%2Fnchembio.347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasinath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Microscopic Insights into the NMR Relaxation-Based Protein Conformational Entropy Meter</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">15092</span>– <span class="NLM_lpage">15100</span>, <span class="refDoi"> DOI: 10.1021/ja405200u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja405200u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSitLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=15092-15100&author=V.+Kasinathauthor=K.+A.+Sharpauthor=A.+J.+Wand&title=Microscopic+Insights+into+the+NMR+Relaxation-Based+Protein+Conformational+Entropy+Meter&doi=10.1021%2Fja405200u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Microscopic Insights into the NMR Relaxation-Based Protein Conformational Entropy Meter</span></div><div class="casAuthors">Kasinath, Vignesh; Sharp, Kim A.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">15092-15100</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational entropy is a potentially important thermodn. parameter contributing to protein function.  Quant. measures of conformational entropy are necessary for an understanding of its role but have been difficult to obtain.  An empirical method that uses changes in conformational dynamics as a proxy for changes in conformational entropy has recently been introduced.  Here the authors probe the microscopic origins of the link between conformational dynamics and conformational entropy using mol. dynamics simulations.  Simulation of seven proteins gave an excellent correlation with measures of side-chain motion derived from NMR relaxation.  The simulations show that the motion of methyl-bearing side chains are sufficiently coupled to that of other side chains to serve as excellent reporters of the overall side-chain conformational entropy.  These results tend to validate the use of exptl. accessible measures of Me motion-the NMR-derived generalized order parameters-as a proxy from which to derive changes in protein conformational entropy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL2g-WdJGoFrVg90H21EOLACvtfcHk0lgG-1RFuVHkAg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSitLrK&md5=a3d96078d1a414e75dda8202c3d12216</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja405200u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja405200u%26sid%3Dliteratum%253Aachs%26aulast%3DKasinath%26aufirst%3DV.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DMicroscopic%2520Insights%2520into%2520the%2520NMR%2520Relaxation-Based%2520Protein%2520Conformational%2520Entropy%2520Meter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D15092%26epage%3D15100%26doi%3D10.1021%2Fja405200u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels and Conformational Transitions via Hinge-Bending Motions</span>. <i>Cell Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/BF02738169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2FBF02738169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10593256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXnslKqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1999&pages=141-164&author=S.+Kumarauthor=B.+Maauthor=C.+J.+Tsaiauthor=H.+Wolfsonauthor=R.+Nussinov&title=Folding+Funnels+and+Conformational+Transitions+via+Hinge-Bending+Motions&doi=10.1007%2FBF02738169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels and conformational transitions via hinge-bending motions</span></div><div class="casAuthors">Kumar, Sandeep; Ma, Buyong; Tsai, Chung-Jung; Wolfson, Haim; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-164</span>CODEN:
                <span class="NLM_cas:coden">CBBIFV</span>;
        ISSN:<span class="NLM_cas:issn">1085-9195</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review, with 72 refs.  In this article we focus on presenting a broad range of examples illustrating low-energy transitions via hinge-bending motions.  The examples are divided according to the type of hinge-bending involved; namely, motions involving fragments of the protein chains, hinge-bending motions involving protein domains, and hinge-bending motions between the covalently unconnected subunits.  We further make a distinction between allosterically and nonallosterically regulated proteins.  These transitions are discussed within the general framework of folding and binding funnels.  We propose that the conformers manifesting such swiveling motions are not the outcome of "induced fit" binding mechanism; instead, mols. exist in an ensemble of conformations that are in equil. in soln.  These ensembles, which populate the bottoms of the funnels, a priori contain both the "open" and the "closed" conformational isomers.  Furthermore, we argue that there are no fundamental differences among the phys. principles behind the folding and binding funnels.  Hence, there is no basic difference between funnels depicting ensembles of conformers of single mols. with fragment, or domain motions, as compared to subunits in multimeric quaternary structures, also showing such conformational transitions.  The difference relates only to the size and complexity of the system.  The larger the system, the more complex its corresponding fused funnel(s).  In particular, funnels assocd. with allosterically regulated proteins are expected to be more complicated, because allostery is frequently involved with movements between subunits, and consequently is often obsd. in multichain and multimol. complexes.  This review centers on the crit. role played by flexibility and conformational fluctuations in enzyme activity.  Internal motions that extend over different time scales and with different amplitudes are known to be essential for the catalytic cycle.  The conformational change obsd. in enzyme-substrate complexes as compared to the unbound enzyme state, and in particular the hinge-bending motions obsd. in enzymes with two domains, have a substantial effect on the enzymic catalytic activity.  The examples we review span the lipolytic enzymes that are particularly interesting, owing to their activation at the water-oil interface; an allosterically controlled dehydrogenase (lactate dehydrogenase); a DNA methyltransferase, with a covalently-bound intermediate; large-scale flexible loop motions in a glycolytic enzyme (TIM); domain motion in PGK, an enzyme which is essential in most cells, both for ATP generation in aerobes and for fermn. in anaerobes; adenylate kinase, showing large conformational changes, owing to their need to shield their catalytic centers from water; a calcium-binding protein (calmodulin), involved in a wide range of cellular calcium-dependent signaling; diphtheria toxin, whose large domain motion has been shown to yield "domain swapping;" the hexameric glutamate dehydrogenase, which has been studied both in a thermophile and in a mesophile; an allosteric enzyme, showing subunit motion between the R and the T states (aspartate transcarbamoylase), and the historically well-studied lac repressor.  Nonallosteric subunit transitions are also addressed, with some examples (aspartate receptor and BamHI endonuclease).  Hence, using this enzyme-catalysis-centered discussion, we address energy funnel landscapes of large-scale conformational transitions, rather than the faster, quasi-harmonic, thermal fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq56G0PLHUlALVg90H21EOLACvtfcHk0liWObtUje_o7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnslKqtrw%253D&md5=8457d4d795f2226fd8115ed83e3ccceb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF02738169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02738169%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DWolfson%26aufirst%3DH.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%2520and%2520Conformational%2520Transitions%2520via%2520Hinge-Bending%2520Motions%26jtitle%3DCell%2520Biochem.%2520Biophys.%26date%3D1999%26volume%3D31%26spage%3D141%26epage%3D164%26doi%3D10.1007%2FBF02738169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels and Binding Mechanisms</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1093/protein/12.9.713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fprotein%2F12.9.713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVagsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1999&pages=713-720&author=B.+Maauthor=S.+Kumarauthor=C.+J.+Tsaiauthor=R.+Nussinov&title=Folding+Funnels+and+Binding+Mechanisms&doi=10.1093%2Fprotein%2F12.9.713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels and binding mechanisms</span></div><div class="casAuthors">Ma, Buyong; Kumar, Sandeep; Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">713-720</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The long-held views on lock-and-key vs. induced fit in binding arose from the notion that a protein exists in a single, most stable conformation, dictated by its sequence.  However, in soln., proteins exist in a range of conformations, which may be described by statistical mech. laws, and their populations follow statistical distributions.  Upon binding, the equil. will shift in favor of the bound conformation, from the ensemble of conformations around the bottom of the folding funnel.  Hence, here we extend the implications and the usefulness of the folding funnel concept to explain fundamental binding mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_0UdAhInii7Vg90H21EOLACvtfcHk0liWObtUje_o7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVagsbY%253D&md5=8c229c85bad0fa20e90c459f90b4a760</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F12.9.713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F12.9.713%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%2520and%2520Binding%2520Mechanisms%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1999%26volume%3D12%26spage%3D713%26epage%3D720%26doi%3D10.1093%2Fprotein%2F12.9.713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels, Binding Funnels, and Protein Function</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1110/ps.8.6.1181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1110%2Fps.8.6.1181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10386868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXjvF2hurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=1181-1190&author=C.+J.+Tsaiauthor=S.+Kumarauthor=B.+Maauthor=R.+Nussinov&title=Folding+Funnels%2C+Binding+Funnels%2C+and+Protein+Function&doi=10.1110%2Fps.8.6.1181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels, binding funnels, and protein function</span></div><div class="casAuthors">Tsai, Chung-Jung; Kumar, Sandeep; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review with 66 refs.  Folding funnels have been the focus of considerable attention during the last few years.  These have mostly been discussed in the general context of the theory of protein folding.  Here we extend the utility of the concept of folding funnels, relating them to biol. mechanisms and function.  In particular, here we describe the shape of the funnels in light of protein synthesis and folding; flexibility, conformational diversity, and binding mechanisms; and the assocd. binding funnels, illustrating the multiple routes and the range of complexed conformers.  Specifically, the walls of the folding funnels, their crevices, and bumps are related to the complexity of protein folding, and hence to sequential vs. non-sequential folding.  Whereas the former is more frequently obsd. in eukaryotic proteins, where the rate of protein synthesis is slower, the latter is more frequent in prokaryotes, with faster translation rates.  The bottoms of the funnels reflect the extent of the flexibility of the proteins.  Rugged floors imply a range of conformational isomers, which may be close on the energy landscape.  Rather than undergoing an induced fit binding mechanism, the conformational ensembles around the rugged bottoms argue that the conformers, which are most complementary to the ligand, will bind to it with the equil. shifting in their favor.  Furthermore, depending on the extent of the ruggedness, or of the smoothness with only a few min., we may infer nonspecific, broad range vs. specific binding.  In particular, folding and binding are similar processes, with similar underlying principles.  Hence, the shape of the folding funnel of the monomer enables making reasonable guesses regarding the shape of the corresponding binding funnel.  Proteins having a broad range of binding, such as proteolytic enzymes or relatively nonspecific endonucleases, may be expected to have not only rugged floors in their folding funnels, but their binding funnels will also behave similarly, with a range of complexed conformations.  Hence, knowledge of the shape of the folding funnels is biol. very useful.  The converse also holds: if kinetic and thermodn. data are available, hints regarding the role of the protein and its binding selectivity may be obtained.  Thus, the utility of the concept of the funnel carries over to the origin of the protein and to its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Lm9WfsGisrVg90H21EOLACvtfcHk0liWObtUje_o7Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvF2hurg%253D&md5=9f33ddad55f16afb66f7544d68bf9b48</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1110%2Fps.8.6.1181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.6.1181%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%252C%2520Binding%2520Funnels%252C%2520and%2520Protein%2520Function%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26spage%3D1181%26epage%3D1190%26doi%3D10.1110%2Fps.8.6.1181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lockless, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Evolutionarily Conserved Pathways of Energetic Connectivity in Protein Families</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1126/science.286.5438.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.286.5438.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10514373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVCgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1999&pages=295-299&author=S.+W.+Locklessauthor=R.+Ranganathan&title=Evolutionarily+Conserved+Pathways+of+Energetic+Connectivity+in+Protein+Families&doi=10.1126%2Fscience.286.5438.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionarily conserved pathways of energetic connectivity in protein families</span></div><div class="casAuthors">Lockless, Steve W.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">5438</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">For mapping energetic interactions in proteins, a technique was developed that uses evolutionary data for a protein family to measure statistical interactions between amino acid positions.  For the PDZ domain family, this anal. predicted a set of energetically coupled positions for a binding site residue that includes unexpected long-range interactions.  Mutational studies confirm these predictions, demonstrating that the statistical energy function is a good indicator of thermodn. coupling in proteins.  Sets of interacting residues form connected pathways through the protein fold that may be the basis for efficient energy conduction within proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-fqjxe7280bVg90H21EOLACvtfcHk0lhqPwOEgY5GjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVCgurs%253D&md5=94bd674ba82f21086db0cbbad98f9679</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1126%2Fscience.286.5438.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.286.5438.295%26sid%3Dliteratum%253Aachs%26aulast%3DLockless%26aufirst%3DS.%2BW.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DEvolutionarily%2520Conserved%2520Pathways%2520of%2520Energetic%2520Connectivity%2520in%2520Protein%2520Families%26jtitle%3DScience%26date%3D1999%26volume%3D286%26spage%3D295%26epage%3D299%26doi%3D10.1126%2Fscience.286.5438.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">del
Sol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">The Origin of Allosteric Functional Modulation: Multiple Pre-Existing Pathways</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1050</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2009.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19679084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Clt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1042-1050&author=A.+del%0ASolauthor=C.+J.+Tsaiauthor=B.+Maauthor=R.+Nussinov&title=The+Origin+of+Allosteric+Functional+Modulation%3A+Multiple+Pre-Existing+Pathways&doi=10.1016%2Fj.str.2009.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Origin of Allosteric Functional Modulation: Multiple Pre-existing Pathways</span></div><div class="casAuthors">del Sol, Antonio; Tsai, Chung-Jung; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1042-1050</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Although allostery draws increasing attention, not much is known about allosteric mechanisms.  Here we argue that in all proteins, allosteric signals transmit through multiple, pre-existing pathways; which pathways dominate depend on protein topologies, specific binding events, covalent modifications, and cellular (environmental) conditions.  Further, perturbation events at any site on the protein surface (or in the interior) will not create new pathways but only shift the pre-existing ensemble of pathways.  Drugs binding at different sites or mutational events in disease shift the ensemble toward the same conformations; however, the relative populations of the different states will change.  Consequently the obsd. functional, conformational, and dynamic effects will be different.  This is the origin of allosteric functional modulation in dynamic proteins: allostery does not necessarily need to invoke conformational rearrangements to control protein activity and pre-existing pathways are always defaulted to during allostery regardless of the stimulant and perturbation site in the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbjZObe-bsobVg90H21EOLACvtfcHk0lhqPwOEgY5GjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Clt7s%253D&md5=a5c26c07e4a4fab4e85d0b1fae65a4ac</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.06.008%26sid%3Dliteratum%253Aachs%26aulast%3Ddel%2BSol%26aufirst%3DA.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DThe%2520Origin%2520of%2520Allosteric%2520Functional%2520Modulation%253A%2520Multiple%2520Pre-Existing%2520Pathways%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D1042%26epage%3D1050%26doi%3D10.1016%2Fj.str.2009.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of Molecular Networks: A Novel Paradigm of Drug Discovery: A Comprehensive Review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2013.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2013.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23384594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2013&pages=333-408&author=P.+Csermelyauthor=T.+Korcsm%C3%A1rosauthor=H.+J.+M.+Kissauthor=G.+Londonauthor=R.+Nussinov&title=Structure+and+Dynamics+of+Molecular+Networks%3A+A+Novel+Paradigm+of+Drug+Discovery%3A+A+Comprehensive+Review&doi=10.1016%2Fj.pharmthera.2013.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of molecular networks: A novel paradigm of drug discovery</span></div><div class="casAuthors">Csermely, Peter; Korcsmaros, Tamas; Kiss, Huba J. M.; London, Gabor; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-408</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and in rational drug design, the increase in approved drugs in the past decade did not match the increase of drug development costs.  Network description and anal. not only give a systems-level understanding of drug action and disease complexity, but can also help to improve the efficiency of drug design.  We give a comprehensive assessment of the anal. tools of network topol. and dynamics.  The state-of-the-art use of chem. similarity, protein structure, protein-protein interaction, signaling, genetic interaction and metabolic networks in the discovery of drug targets is summarized.  We propose that network targeting follows two basic strategies.  The "central hit strategy" selectively targets central nodes/edges of the flexible networks of infectious agents or cancer cells to kill them.  The "network influence strategy" works against other diseases, where an efficient reconfiguration of rigid networks needs to be achieved by targeting the neighbors of central nodes/edges.  It is shown how network techniques can help in the identification of single-target, edgetic, multi-target and allo-network drug target candidates.  We review the recent boom in network methods helping hit identification, lead selection optimizing drug efficacy, as well as minimizing side-effects and drug toxicity.  Successful network-based drug development strategies are shown through the examples of infections, cancer, metabolic diseases, neurodegenerative diseases and aging.  Summarizing >1200 refs. we suggest an optimized protocol of network-aided drug development, and provide a list of systems-level hallmarks of drug quality.  Finally, we highlight network-related drug development trends helping to achieve these hallmarks by a cohesive, global approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFqNviuDk7kbVg90H21EOLACvtfcHk0lhqPwOEgY5GjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D&md5=d70c48a1572815c1b4905cc6e8cdfaa6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2013.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2013.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26aulast%3DKiss%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DLondon%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520Molecular%2520Networks%253A%2520A%2520Novel%2520Paradigm%2520of%2520Drug%2520Discovery%253A%2520A%2520Comprehensive%2520Review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2013%26volume%3D138%26spage%3D333%26epage%3D408%26doi%3D10.1016%2Fj.pharmthera.2013.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The Mechanism of ATP-Dependent Allosteric Protection of Akt Kinase Phosphorylation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2015.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26256536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSju7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1725-1734&author=S.+Luauthor=R.+Dengauthor=H.+Jiangauthor=H.+Songauthor=S.+Liauthor=Q.+Shenauthor=W.+Huangauthor=R.+Nussinovauthor=J.+Yuauthor=J.+Zhang&title=The+Mechanism+of+ATP-Dependent+Allosteric+Protection+of+Akt+Kinase+Phosphorylation&doi=10.1016%2Fj.str.2015.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Mechanism of ATP-Dependent Allosteric Protection of Akt Kinase Phosphorylation</span></div><div class="casAuthors">Lu, Shaoyong; Deng, Rong; Jiang, Haiming; Song, Huili; Li, Shuai; Shen, Qiancheng; Huang, Wenkang; Nussinov, Ruth; Yu, Jianxiu; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1725-1734</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kinases use ATP to phosphorylate substrates; recent findings underscore the addnl. regulatory roles of ATP.  Here, we propose a mechanism for allosteric regulation of Akt1 kinase phosphorylation by ATP.  Our 4.7-μs mol. dynamics simulations of Akt1 and its mutants in the ATP/ADP bound/unbound states revealed that ATP occupancy of the ATP-binding site stabilizes the closed conformation, allosterically protecting pT308 by restraining phosphatase access and key interconnected residues on the ATP→pT308 allosteric pathway.  Following ATP→ADP hydrolysis, pT308 is exposed and readily dephosphorylated.  Site-directed mutagenesis validated these predictions and indicated that the mutations do not impair PDK1 and PP2A phosphatase recruitment.  We further probed the function of residues around pT308 at the at. level, and predicted and exptl. confirmed that Akt1H194R/R273H double mutant rescues pathol.-related Akt1R273H.  Anal. of classical Akt homologs suggests that this mechanism can provide a general model of allosteric kinase regulation by ATP; as such, it offers a potential avenue for allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP6VKa16r6C7Vg90H21EOLACvtfcHk0lhxb2bmZkPyAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSju7nM&md5=147d9333779db1bc1eb6b05f36524174</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520Mechanism%2520of%2520ATP-Dependent%2520Allosteric%2520Protection%2520of%2520Akt%2520Kinase%2520Phosphorylation%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D1725%26epage%3D1734%26doi%3D10.1016%2Fj.str.2015.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelstein, S. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Mechanisms of Signal Transduction</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>308</i></span>,  <span class="NLM_fpage">1424</span>– <span class="NLM_lpage">1428</span>, <span class="refDoi"> DOI: 10.1126/science.1108595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1108595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15933191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1eqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=308&publication_year=2005&pages=1424-1428&author=J.-P.+Changeuxauthor=S.+J.+Edelstein&title=Allosteric+Mechanisms+of+Signal+Transduction&doi=10.1126%2Fscience.1108595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Mechanisms of Signal Transduction</span></div><div class="casAuthors">Changeux, Jean-Pierre; Edelstein, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">308</span>
        (<span class="NLM_cas:issue">5727</span>),
    <span class="NLM_cas:pages">1424-1428</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Forty years ago, a simple model of allosteric mechanisms (indirect interactions between distinct sites), used initially to explain feedback-inhibited enzymes, was presented by Monod, Wyman, and Changeux.  We review the MWC theory and its applications for the understanding of signal transduction in biol., and also identify remaining issues that deserve theor. and exptl. substantiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8FOn7QVpjAbVg90H21EOLACvtfcHk0lhxb2bmZkPyAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1eqtrs%253D&md5=ec3750bf12645a3e6c4d2b13f221fc03</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.1108595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1108595%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26aulast%3DEdelstein%26aufirst%3DS.%2BJ.%26atitle%3DAllosteric%2520Mechanisms%2520of%2520Signal%2520Transduction%26jtitle%3DScience%26date%3D2005%26volume%3D308%26spage%3D1424%26epage%3D1428%26doi%3D10.1126%2Fscience.1108595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span> <span> </span><span class="NLM_article-title">An Ensemble View of Allostery</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1126/science.1186121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1186121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20133562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFamtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=653-654&author=V.+J.+Hilser&title=An+Ensemble+View+of+Allostery&doi=10.1126%2Fscience.1186121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">An ensemble view of allostery</span></div><div class="casAuthors">Hilser, Vincent J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5966</span>),
    <span class="NLM_cas:pages">653-654</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_hUdt32OorVg90H21EOLACvtfcHk0lhxb2bmZkPyAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFamtbg%253D&md5=c52626431046cd4e42f85b55404de86b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1126%2Fscience.1186121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1186121%26sid%3Dliteratum%253Aachs%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26atitle%3DAn%2520Ensemble%2520View%2520of%2520Allostery%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D653%26epage%3D654%26doi%3D10.1126%2Fscience.1186121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motlagh, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrabl, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span> <span> </span><span class="NLM_article-title">The Ensemble Nature of Allostery</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1038/nature13001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature13001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24740064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=331-339&author=H.+N.+Motlaghauthor=J.+O.+Wrablauthor=J.+Liauthor=V.+J.+Hilser&title=The+Ensemble+Nature+of+Allostery&doi=10.1038%2Fnature13001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The ensemble nature of allostery</span></div><div class="casAuthors">Motlagh, Hesam N.; Wrabl, James O.; Li, Jing; Hilser, Vincent J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7496</span>),
    <span class="NLM_cas:pages">331-339</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allostery is the process by which biol. macromols. (mostly proteins) transmit the effect of binding at one site to another, often distal, functional site, allowing for regulation of activity.  Recent exptl. observations demonstrating that allostery can be facilitated by dynamic and intrinsically disordered proteins have resulted in a new paradigm for understanding allosteric mechanisms, which focuses on the conformational ensemble and the statistical nature of the interactions responsible for the transmission of information.  Anal. of allosteric ensembles reveals a rich spectrum of regulatory strategies, as well as a framework to unify the description of allosteric mechanisms from different systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAlOmcHvsANLVg90H21EOLACvtfcHk0lhxb2bmZkPyAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWnurc%253D&md5=cf832c576e409cab75d1031a98f418c3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnature13001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13001%26sid%3Dliteratum%253Aachs%26aulast%3DMotlagh%26aufirst%3DH.%2BN.%26aulast%3DWrabl%26aufirst%3DJ.%2BO.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26atitle%3DThe%2520Ensemble%2520Nature%2520of%2520Allostery%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D331%26epage%3D339%26doi%3D10.1038%2Fnature13001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrabl, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlagh, H. N.</span></span> <span> </span><span class="NLM_article-title">Structural and Energetic Basis of Allostery</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-050511-102319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-050511-102319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22577828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yhsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=585-609&author=V.+J.+Hilserauthor=J.+O.+Wrablauthor=H.+N.+Motlagh&title=Structural+and+Energetic+Basis+of+Allostery&doi=10.1146%2Fannurev-biophys-050511-102319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and energetic basis of allostery</span></div><div class="casAuthors">Hilser, Vincent J.; Wrabl, James O.; Motlagh, Hesam N.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">585-609</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Allostery is a biol. phenomenon of fundamental importance in regulation and signaling, and efforts to understand this process have led to the development of numerous models.  In spite of individual successes in understanding the structural determinants of allostery in well-documented systems, much less success has been achieved in identifying a set of quant. and transferable ground rules that provide an understanding of how allostery works.  Are there organizing principles that allow us to relate structurally different proteins, or are the determinants of allostery unique to each system.  Using an ensemble-based model, we show that allosteric phenomena can be formulated in terms of conformational free energies of the cooperative elements in a protein and the coupling interactions between them.  Interestingly, the resulting allosteric ground rules provide a framework to reconcile observations that challenge purely structural models of site-to-site coupling, including (a) allostery in the absence of pathways of structural distortions, (b) allostery in the absence of any structural change, and (c) the ability of allosteric ligands to act as agonists under some circumstances and antagonists under others.  The ensemble view of allostery that emerges provides insights into the energetic prerequisites of site-to-site coupling and thus into how allostery works.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxNCm1otnOSrVg90H21EOLACvtfcHk0lif-hYqhDtchg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yhsbo%253D&md5=b8e81b54704ea6d3eb776b8cac81985f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-050511-102319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-050511-102319%26sid%3Dliteratum%253Aachs%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26aulast%3DWrabl%26aufirst%3DJ.%2BO.%26aulast%3DMotlagh%26aufirst%3DH.%2BN.%26atitle%3DStructural%2520and%2520Energetic%2520Basis%2520of%2520Allostery%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D585%26epage%3D609%26doi%3D10.1146%2Fannurev-biophys-050511-102319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Allostery: An Overview of Its History, Concepts, Methods, and Applications</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1004966</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27253437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWntbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1004966&author=J.+Liuauthor=R.+Nussinov&title=Allostery%3A+An+Overview+of+Its+History%2C+Concepts%2C+Methods%2C+and+Applications&doi=10.1371%2Fjournal.pcbi.1004966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery: an overview of its history, concepts, methods, and applications</span></div><div class="casAuthors">Liu, Jin; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1004966/1-e1004966/5</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The concept of allostery has evolved in the past century.  In this Editorial, we briefly overview the history of allostery, from the pre-allostery nomenclature era starting with the Bohr effect (1904) to the birth of allostery by Monod and Jacob (1961).  We describe the evolution of the allostery concept, from a conformational change in a two-state model (1965, 1966) to dynamic allostery in the ensemble model (1999); from multi-subunit (1965) proteins to all proteins (2004).  We highlight the current available methods to study allostery and their applications in studies of conformational mechanisms, disease, and allosteric drug discovery.  We outline the challenges and future directions that we foresee.  Altogether, this Editorial narrates the history of this fundamental concept in the life sciences, its significance, methodologies to detect and predict it, and its application in a broad range of living systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUNj0xAGD3qrVg90H21EOLACvtfcHk0lif-hYqhDtchg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWntbbJ&md5=1a753ef03a696bd2591eb1d7206f50d7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004966%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DAllostery%253A%2520An%2520Overview%2520of%2520Its%2520History%252C%2520Concepts%252C%2520Methods%252C%2520and%2520Applications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3De1004966%26doi%3D10.1371%2Fjournal.pcbi.1004966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">A Unified View of “How Allostery Works”</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003394</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24516370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=C.-J.+Tsaiauthor=R.+Nussinov&title=A+Unified+View+of+%E2%80%9CHow+Allostery+Works%E2%80%9D&doi=10.1371%2Fjournal.pcbi.1003394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A unified view of "how allostery works"</span></div><div class="casAuthors">Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1003394/1-e1003394/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The question of how allostery works was posed almost 50 years ago.  Since then it has been the focus of much effort.  This is for two reasons: first, the intellectual curiosity of basic science and the desire to understand fundamental phenomena, and second, its vast practical importance.  Allostery is at play in all processes in the living cell, and increasingly in drug discovery.  Many models have been successfully formulated, and are able to describe allostery even in the absence of a detailed structural mechanism.  However, conceptual schemes designed to qual. explain allosteric mechanisms usually lack a quant. math. model, and are unable to link its thermodn. and structural foundations.  This hampers insight into oncogenic mutations in cancer progression and biased agonists' actions.  Here, we describe how allostery works from three different standpoints: thermodn., free energy landscape of population shift, and structure; all with exactly the same allosteric descriptors.  This results in a unified view which not only clarifies the elusive allosteric mechanism but also provides structural grasp of agonist-mediated signaling pathways, and guides allosteric drug discovery.  Of note, the unified view reasons that allosteric coupling (or communication) does not det. the allosteric efficacy; however, a communication channel is what makes potential binding sites allosteric.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEgA5AKySrFbVg90H21EOLACvtfcHk0lif-hYqhDtchg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D&md5=3912bf89813eeed254b3e3e4e1937f99</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003394%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.-J.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DA%2520Unified%2520View%2520of%2520%25E2%2580%259CHow%2520Allostery%2520Works%25E2%2580%259D%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Allostery and Cooperativity Revisited</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1295</span>– <span class="NLM_lpage">1307</span>, <span class="refDoi"> DOI: 10.1110/ps.03259908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1110%2Fps.03259908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18560010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVyjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1295-1307&author=Q.+Cuiauthor=M.+Karplus&title=Allostery+and+Cooperativity+Revisited&doi=10.1110%2Fps.03259908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery and cooperativity revisited</span></div><div class="casAuthors">Cui, Qiang; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1295-1307</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Although phenomenol. models that account for cooperativity in allosteric systems date back to the early and mid-60's [e.g., the D. E. Koshland, G. Nemethy, and D. Filmer, 1966 (KNF) and J. Momod, J. Wyman, and J. P. Changeux, 1965 (MWC) models], there is resurgent interest in the topic due to the recent exptl. and computational studies that attempted to reveal, at an atomistic level, how allostery actually works.  Here, using systems for which atomistic simulations have been carried out in their groups as examples, the authors describe the current understanding of allostery and how the mechanisms go beyond the classical MWC/Pauling-KNF descriptions, and point out that the "new view" of allostery, emphasizing "population shifts," is, in fact, an "old view.".  The presentation offers not only an up-to-date description of allostery from a theor./computational perspective, but also helps to resolve several outstanding issues concerning allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot34ax4lJjPLVg90H21EOLACvtfcHk0lif-hYqhDtchg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVyjurY%253D&md5=40d789fc47dcb9f01e797aa724861f77</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1110%2Fps.03259908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.03259908%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAllostery%2520and%2520Cooperativity%2520Revisited%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26spage%3D1295%26epage%3D1307%26doi%3D10.1110%2Fps.03259908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bechle, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilsland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braganza, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bungay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbett, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greasley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitching, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninkovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scales, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skerratt, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran-Dubé, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent, Selective, and Peripherally Restricted Pan-Trk Kinase Inhibitors for the Treatment of Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6779</span>– <span class="NLM_lpage">6800</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00633</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00633" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SnurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6779-6800&author=S.+K.+Bagalauthor=M.+Andrewsauthor=B.+M.+Bechleauthor=J.+Bianauthor=J.+Bilslandauthor=D.+C.+Blakemoreauthor=J.+F.+Braganzaauthor=P.+J.+Bungayauthor=M.+S.+Corbettauthor=C.+N.+Croninauthor=J.+J.+Cuiauthor=R.+Diasauthor=N.+J.+Flanaganauthor=S.+E.+Greasleyauthor=R.+Grimleyauthor=K.+Jamesauthor=E.+Johnsonauthor=L.+Kitchingauthor=M.+L.+Krausauthor=I.+McAlpineauthor=A.+Nagataauthor=S.+Ninkovicauthor=K.+Omotoauthor=S.+Scalesauthor=S.+E.+Skerrattauthor=J.+Sunauthor=M.+Tran-Dub%C3%A9author=G.+J.+Waldronauthor=F.+Wangauthor=J.+S.+Warmus&title=Discovery+of+Potent%2C+Selective%2C+and+Peripherally+Restricted+Pan-Trk+Kinase+Inhibitors+for+the+Treatment+of+Pain&doi=10.1021%2Facs.jmedchem.8b00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent, Selective, and Peripherally Restricted Pan-Trk Kinase Inhibitors for the Treatment of Pain</span></div><div class="casAuthors">Bagal, Sharan K.; Andrews, Mark; Bechle, Bruce M.; Bian, Jianwei; Bilsland, James; Blakemore, David C.; Braganza, John F.; Bungay, Peter J.; Corbett, Matthew S.; Cronin, Ciaran N.; Cui, Jingrong Jean; Dias, Rebecca; Flanagan, Neil J.; Greasley, Samantha E.; Grimley, Rachel; James, Kim; Johnson, Eric; Kitching, Linda; Kraus, Michelle L.; McAlpine, Indrawan; Nagata, Asako; Ninkovic, Sacha; Omoto, Kiyoyuki; Scales, Stephanie; Skerratt, Sarah E.; Sun, Jianmin; Tran-Dube, Michelle; Waldron, Gareth J.; Wang, Fen; Warmus, Joseph S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6779-6800</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hormones of the neurotrophin family: nerve growth factor (NGF), brain derived neurotrophic factor (BDNF), neurotrophin 3 (NT3) and neurotrophin 4 (NT4) are known to activate the family of Tropomyosin receptor kinases (TrkA, TrkB, TrkC).  Moreover, inhibition of the TrkA kinase pathway in pain has been clin. validated by the NGF antibody tanezumab leading to significant interest in the development of small mol. inhibitors of TrkA.  Furthermore, Trk inhibitors having an acceptable safety profile will require minimal brain availability.  Herein we discuss the discovery of two potent, selective, peripherally restricted, efficacious and well-tolerated series of pan-Trk inhibitors which successfully delivered three candidate quality compds. I, II and III.  All three compds. are predicted to possess low metabolic clearance in human that does not proceed via aldehyde oxidase-catalyzed reactions, thus addressing the potential clearance prediction liability assocd. with our current pan-Trk development candidate PF-06273340.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhKF1Su9NvabVg90H21EOLACvtfcHk0lg5kzzv7q2sNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SnurfN&md5=65b55ef6cdd82c44680c6c49310df7b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00633%26sid%3Dliteratum%253Aachs%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DBechle%26aufirst%3DB.%2BM.%26aulast%3DBian%26aufirst%3DJ.%26aulast%3DBilsland%26aufirst%3DJ.%26aulast%3DBlakemore%26aufirst%3DD.%2BC.%26aulast%3DBraganza%26aufirst%3DJ.%2BF.%26aulast%3DBungay%26aufirst%3DP.%2BJ.%26aulast%3DCorbett%26aufirst%3DM.%2BS.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DDias%26aufirst%3DR.%26aulast%3DFlanagan%26aufirst%3DN.%2BJ.%26aulast%3DGreasley%26aufirst%3DS.%2BE.%26aulast%3DGrimley%26aufirst%3DR.%26aulast%3DJames%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DKraus%26aufirst%3DM.%2BL.%26aulast%3DMcAlpine%26aufirst%3DI.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DNinkovic%26aufirst%3DS.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DScales%26aufirst%3DS.%26aulast%3DSkerratt%26aufirst%3DS.%2BE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DTran-Dub%25C3%25A9%26aufirst%3DM.%26aulast%3DWaldron%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520of%2520Potent%252C%2520Selective%252C%2520and%2520Peripherally%2520Restricted%2520Pan-Trk%2520Kinase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6779%26epage%3D6800%26doi%3D10.1021%2Facs.jmedchem.8b00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1204-1217&author=H.+%C3%9C.+Kaniskanauthor=M.+S.+Eramauthor=K.+Zhaoauthor=M.+M.+Szewczykauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=S.+Xiaoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=F.+Liauthor=E.+Dobrovetskyauthor=D.+Smilauthor=S.+J.+Minauthor=J.+Lin-Jonesauthor=M.+Schapiraauthor=P.+Atadjaauthor=E.+Liauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=F.+Liuauthor=Z.+Yuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+Potent+and+Selective+Allosteric+Inhibitors+of+Protein+Arginine+Methyltransferase+3+%28PRMT3%29&doi=10.1021%2Facs.jmedchem.7b01674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Umit; Eram, Mohammad S.; Zhao, Kehao; Szewczyk, Magdalena M.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Xiao, Sean; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Li, Fengling; Dobrovetsky, Elena; Smil, David; Min, Sun-Joon; Lin-Jones, Jennifer; Schapira, Matthieu; Atadja, Peter; Li, En; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Feng; Yu, Zhengtian; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1204-1217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is crucial for maturation of ribosomes, and has been implicated in several diseases.  We recently disclosed a highly potent, selective and cell-active allosteric inhibitor of PRMT3, compd. 4.  Here, we report comprehensive structure-activity relationship studies that target the allosteric binding site of PRMT3.  We conducted design, synthesis and evaluation of novel compds. in biochem., selectivity and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10-36 nM), selective and cell-active allosteric inhibitors of PRMT3 (compds. 29, 30, 36 and 37).  In addn., we generated compds. that are very close analogs of these potent inhibitors, but displayed drastically reduced potency as neg. controls (compds. 49-51).  These inhibitors and neg. controls are valuable chem. tools for the biomedical community to further investigate biol. functions and disease assocns. of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgUnzNZOB2rVg90H21EOLACvtfcHk0lg5kzzv7q2sNw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO&md5=a784663c26234c3a99386ea4fe3f8abc</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Potent%2520and%2520Selective%2520Allosteric%2520Inhibitors%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1204%26epage%3D1217%26doi%3D10.1021%2Facs.jmedchem.7b01674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Veldhoven, J. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Spronsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Velden, W. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oenema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9146</span>– <span class="NLM_lpage">9161</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9146-9161&author=N.+V.+Ortiz%0AZacar%C3%ADasauthor=J.+P.+D.+van+Veldhovenauthor=L.+Portnerauthor=E.+van+Spronsenauthor=S.+Ulloauthor=M.+Veenhuizenauthor=W.+J.+C.+van+der+Veldenauthor=A.+J.+M.+Zweemerauthor=R.+M.+Kreekelauthor=K.+Oenemaauthor=E.+B.+Lenselinkauthor=L.+H.+Heitmanauthor=A.+P.+Ijzerman&title=Pyrrolone+Derivatives+as+Intracellular+Allosteric+Modulators+for+Chemokine+Receptors%3A+Selective+and+Dual-Targeting+Inhibitors+of+CC+Chemokine+Receptors+1+and+2&doi=10.1021%2Facs.jmedchem.8b00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; van Veldhoven, Jacobus P. D.; Portner, Laura; van Spronsen, Eric; Ullo, Salviana; Veenhuizen, Margo; van der Velden, Wijnand J. C.; Zweemer, Annelien J. M.; Kreekel, Roy M.; Oenema, Kenny; Lenselink, Eelke B.; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9146-9161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structures of CC chemokine receptors 2 and 9 (CCR2 and CCR9) have provided structural evidence for an allosteric, intracellular binding site.  The high conservation of residues involved in this site suggests its presence in most chemokine receptors, including the close homolog CCR1.  By using [3H]CCR2-RA-[R], a high-affinity, CCR2 intracellular ligand, we report an intracellular binding site in CCR1, where this radioligand also binds with high affinity.  In addn., we report the synthesis and biol. characterization of a series of pyrrolone derivs. for CCR1 and CCR2, which allowed us to identify several high-affinity intracellular ligands, including selective and potential multitarget antagonists.  Evaluation of selected compds. in a functional [35S]GTPγS assay revealed that they act as inverse agonists in CCR1, providing a new manner of pharmacol. modulation.  Thus, this intracellular binding site enables the design of selective and multitarget inhibitors as a novel therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HeUOWen2U7Vg90H21EOLACvtfcHk0ljEMBfFySiaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM&md5=bf05c560b983659de579074242d299d1</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00605%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3DPortner%26aufirst%3DL.%26aulast%3Dvan%2BSpronsen%26aufirst%3DE.%26aulast%3DUllo%26aufirst%3DS.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BVelden%26aufirst%3DW.%2BJ.%2BC.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DKreekel%26aufirst%3DR.%2BM.%26aulast%3DOenema%26aufirst%3DK.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DPyrrolone%2520Derivatives%2520as%2520Intracellular%2520Allosteric%2520Modulators%2520for%2520Chemokine%2520Receptors%253A%2520Selective%2520and%2520Dual-Targeting%2520Inhibitors%2520of%2520CC%2520Chemokine%2520Receptors%25201%2520and%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9146%26epage%3D9161%26doi%3D10.1021%2Facs.jmedchem.8b00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljEMBfFySiaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span> <span> </span><span class="NLM_article-title">Allosteric Small-Molecule Kinase Inhibitors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2015.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26478442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2015&pages=59-68&author=P.+Wuauthor=M.+H.+Clausenauthor=T.+E.+Nielsen&title=Allosteric+Small-Molecule+Kinase+Inhibitors&doi=10.1016%2Fj.pharmthera.2015.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Clausen, Mads H.; Nielsen, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors are invaluable targeted therapeutics for the treatment of various human diseases, esp. cancers.  While the majority of approved and developed preclin. small-mol. inhibitors are characterized as type I or type II inhibitors that target the ATP-binding pocket of kinases, the remarkable sequential and structural similarity among ATP pockets renders the selective inhibition of kinases a daunting challenge.  Therefore, targeting allosteric pockets of kinases outside the highly conversed ATP pocket has been proposed as a promising alternative to overcome current barriers of kinase inhibitors, including poor selectivity and emergence of drug resistance.  In spite of the small no. of identified allosteric inhibitors in comparison with that of inhibitors targeting the ATP pocket, encouraging results, such as the FDA-approval of the first small-mol. allosteric inhibitor trametinib in 2013, the progress of more than 10 other allosteric inhibitors in clin. trials, and the emergence of a pipeline of highly selective and potent preclin. mols., have been reported in the past decade.  In this article, we present the current knowledge on allosteric inhibition in terms of conception, classification, potential advantages, and summarized debatable topics in the field.  Recent progress and allosteric inhibitors that were identified in the past three years are highlighted in this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK12iPyaK2ObVg90H21EOLACvtfcHk0ljEMBfFySiaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qsb3N&md5=26e4658d325b8f016be29df27bf69ec2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26atitle%3DAllosteric%2520Small-Molecule%2520Kinase%2520Inhibitors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D156%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.pharmthera.2015.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebois, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of Kinases and GPCRs: Design Principles and Structural Diversity</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cbpa.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18342020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ersr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=269-280&author=J.+A.+Lewisauthor=E.+P.+Leboisauthor=C.+W.+Lindsley&title=Allosteric+Modulation+of+Kinases+and+GPCRs%3A+Design+Principles+and+Structural+Diversity&doi=10.1016%2Fj.cbpa.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of kinases and GPCRs: design principles and structural diversity</span></div><div class="casAuthors">Lewis, Jana A.; Lebois, Evan P.; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-280</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Allosteric binding sites, as opposed to traditional orthosteric binding sites, offer unparalleled opportunities for drug discovery by providing high levels of selectivity, mimicking physiol. conditions, affording fewer side effects because of desensitization/downregulation, and engendering ligands with chemotypes divergent from orthosteric ligands.  For kinases, allosteric mechanisms described to date include alteration of protein kinase conformation blocking productive ATP binding which appear ATP competitive' or blocking kinase activation by conformational changes that are ATP non-competitive'.  For GPCRs, allosteric mechanisms impart multiple modes of target modulation (pos. allosteric modulation (PAM), neg. allosteric modulation (NAM), neutral cooperativity, partial antagonism (PA), allosteric agonism and allosteric antagonism).  Here, we review recent developments in the design principles and structural diversity of allosteric ligands for kinases and GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiz5X-sQXXv7Vg90H21EOLACvtfcHk0lj3TC-3FeuUeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ersr4%253D&md5=79533b9634a920154e08e4bad0f6275e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BA.%26aulast%3DLebois%26aufirst%3DE.%2BP.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DAllosteric%2520Modulation%2520of%2520Kinases%2520and%2520GPCRs%253A%2520Design%2520Principles%2520and%2520Structural%2520Diversity%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D269%26epage%3D280%26doi%3D10.1016%2Fj.cbpa.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gum, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groebe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clampit, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haasch, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoll, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderwood, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/cb1002619</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1002619" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=234-244&author=K.+M.+Comessauthor=C.+Sunauthor=C.+Abad-Zapateroauthor=E.+R.+Goedkenauthor=R.+J.+Gumauthor=D.+W.+Borhaniauthor=M.+Argiriadiauthor=D.+R.+Groebeauthor=Y.+Jiaauthor=J.+E.+Clampitauthor=D.+L.+Haaschauthor=H.+T.+Smithauthor=S.+Wangauthor=D.+Songauthor=M.+L.+Coenauthor=T.+E.+Cloutierauthor=H.+Tangauthor=X.+Chengauthor=C.+Quinnauthor=B.+Liuauthor=Z.+Xinauthor=G.+Liuauthor=E.+H.+Fryauthor=V.+Stollauthor=T.+I.+Ngauthor=D.+Banachauthor=D.+Marcotteauthor=D.+J.+Burnsauthor=D.+J.+Calderwoodauthor=P.+J.+Hajduk&title=Discovery+and+Characterization+of+Non-ATP+Site+Inhibitors+of+the+Mitogen+Activated+Protein+%28MAP%29+Kinases&doi=10.1021%2Fcb1002619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span></div><div class="casAuthors">Comess, Kenneth M.; Sun, Chaohong; Abad-Zapatero, Cele; Goedken, Eric R.; Gum, Rebecca J.; Borhani, David W.; Argiriadi, Maria; Groebe, Duncan R.; Jia, Yong; Clampit, Jill E.; Haasch, Deanna L.; Smith, Harriet T.; Wang, Sanyi; Song, Danying; Coen, Michael L.; Cloutier, Timothy E.; Tang, Hua; Cheng, Xueheng; Quinn, Christopher; Liu, Bo; Xin, Zhili; Liu, Gang; Fry, Elizabeth H.; Stoll, Vincent; Ng, Teresa I.; Banach, David; Marcotte, Doug; Burns, David J.; Calderwood, David J.; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of protein kinases has validated therapeutic utility for cancer, with at least seven kinase inhibitor drugs on the market.  Protein kinase inhibition also has significant potential for a variety of other diseases, including diabetes, pain, cognition, and chronic inflammatory and immunol. diseases.  However, as the vast majority of current approaches to kinase inhibition target the highly conserved ATP-binding site, the use of kinase inhibitors in treating nononcol. diseases may require great selectivity for the target kinase.  As protein kinases are signal transducers that are involved in binding to a variety of other proteins, targeting alternative, less conserved sites on the protein may provide an avenue for greater selectivity.  Here we report an affinity-based, high-throughput screening technique that allows nonbiased interrogation of small mol. libraries for binding to all exposed sites on a protein surface.  This approach was used to screen both the c-Jun N-terminal protein kinase Jnk-1 (involved in insulin signaling) and p38α (involved in the formation of TNFα and other cytokines).  In addn. to canonical ATP-site ligands, compds. were identified that bind to novel allosteric sites.  The nature, biol. relevance, and mode of binding of these ligands were extensively characterized using two-dimensional 1H/13C NMR spectroscopy, protein X-ray crystallog., surface plasmon resonance, and direct enzymic activity and activation cascade assays.  Jnk-1 and p38α both belong to the MAP kinase family, and the allosteric ligands for both targets bind similarly on a ledge of the protein surface exposed by the MAP insertion present in the CMGC family of protein kinases and distant from the active site.  Medicinal chem. studies resulted in an improved Jnk-1 ligand able to increase adiponectin secretion in human adipocytes and increase insulin-induced protein kinase PKB phosphorylation in human hepatocytes, in similar fashion to Jnk-1 siRNA and to rosiglitazone treatment.  Together, the data suggest that these new ligand series bind to a novel, allosteric, and physiol. relevant site and therefore represent a unique approach to identify kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyEwNo6QURxLVg90H21EOLACvtfcHk0lj3TC-3FeuUeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D&md5=857035713020ac66110b7ef6114232b9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fcb1002619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1002619%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DGum%26aufirst%3DR.%2BJ.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DGroebe%26aufirst%3DD.%2BR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DClampit%26aufirst%3DJ.%2BE.%26aulast%3DHaasch%26aufirst%3DD.%2BL.%26aulast%3DSmith%26aufirst%3DH.%2BT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DCloutier%26aufirst%3DT.%2BE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFry%26aufirst%3DE.%2BH.%26aulast%3DStoll%26aufirst%3DV.%26aulast%3DNg%26aufirst%3DT.%2BI.%26aulast%3DBanach%26aufirst%3DD.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCalderwood%26aufirst%3DD.%2BJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Non-ATP%2520Site%2520Inhibitors%2520of%2520the%2520Mitogen%2520Activated%2520Protein%2520%2528MAP%2529%2520Kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D234%26epage%3D244%26doi%3D10.1021%2Fcb1002619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherbenou, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bumm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turaga, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span> <span> </span><span class="NLM_article-title">BCR-ABL SH3-SH2 Domain Mutations in Chronic Myeloid Leukemia Patients on Imatinib</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">3278</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-10-183665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1182%2Fblood-2008-10-183665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20519627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVeisLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=3278-3285&author=D.+W.+Sherbenouauthor=O.+Hantschelauthor=I.+Kaupeauthor=S.+Willisauthor=T.+Bummauthor=L.+P.+Turagaauthor=T.+Langeauthor=K.+H.+Daoauthor=R.+D.+Pressauthor=B.+J.+Drukerauthor=G.+Superti-Furgaauthor=M.+W.+Deininger&title=BCR-ABL+SH3-SH2+Domain+Mutations+in+Chronic+Myeloid+Leukemia+Patients+on+Imatinib&doi=10.1182%2Fblood-2008-10-183665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">BCR-ABL SH3-SH2 domain mutations in chronic myeloid leukemia patients on imatinib</span></div><div class="casAuthors">Sherbenou, Daniel W.; Hantschel, Oliver; Kaupe, Ines; Willis, Stephanie; Bumm, Thomas; Turaga, Lalita P.; Lange, Thoralf; Dao, Kim-Hien; Press, Richard D.; Druker, Brian J.; Superti-Furga, Giulio; Deininger, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3278-3285</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Point mutations in the kinase domain of BCR-ABL are the most common mechanism of drug resistance in chronic myeloid leukemia (CML) patients treated with ABL kinase inhibitors, including imatinib.  It has also been shown in vitro that mutations outside the kinase domain in the neighboring linker, SH2, SH3, and Cap domains can confer imatinib resistance.  In the context of ABL, these domains have an autoinhibitory effect on kinase activity, and mutations in this region can activate the enzyme.  To det. the frequency and relevance to resistance of regulatory domain mutations in CML patients on imatinib, we screened for such mutations in a cohort of consecutive CML patients with various levels of response.  Regulatory domain mutations were detected in 7 of 98 patients, whereas kinase domain mutations were detected in 29.  One mutation (T212R) conferred in vitro tyrosine kinase inhibitor resistance and was assocd. with relapse, whereas most other mutations did not affect drug sensitivity.  Mechanistic studies showed that T212R increased the activity of ABL and BCR-ABL and that T212R-induced resistance may be partially the result of stabilization of an active kinase conformation.  Regulatory domain mutations are uncommon but may explain resistance in some patients without mutations in the kinase domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWYplM3K_TmLVg90H21EOLACvtfcHk0lj3TC-3FeuUeQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVeisLjI&md5=a4cdbf97c4af950d257b38d8ee276fac</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-10-183665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-10-183665%26sid%3Dliteratum%253Aachs%26aulast%3DSherbenou%26aufirst%3DD.%2BW.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DKaupe%26aufirst%3DI.%26aulast%3DWillis%26aufirst%3DS.%26aulast%3DBumm%26aufirst%3DT.%26aulast%3DTuraga%26aufirst%3DL.%2BP.%26aulast%3DLange%26aufirst%3DT.%26aulast%3DDao%26aufirst%3DK.%2BH.%26aulast%3DPress%26aufirst%3DR.%2BD.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26atitle%3DBCR-ABL%2520SH3-SH2%2520Domain%2520Mutations%2520in%2520Chronic%2520Myeloid%2520Leukemia%2520Patients%2520on%2520Imatinib%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D3278%26epage%3D3285%26doi%3D10.1182%2Fblood-2008-10-183665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotesta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groell, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellé, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8120</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01040</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01040" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8120-8135&author=J.+Schoepferauthor=W.+Jahnkeauthor=G.+Berelliniauthor=S.+Buonamiciauthor=S.+Cotestaauthor=S.+W.+Cowan-Jacobauthor=S.+Doddauthor=P.+Drueckesauthor=D.+Fabbroauthor=T.+Gabrielauthor=J.+M.+Groellauthor=R.+M.+Grotzfeldauthor=A.+Q.+Hassanauthor=C.+Henryauthor=V.+Iyerauthor=D.+Jonesauthor=F.+Lombardoauthor=A.+Looauthor=P.+W.+Manleyauthor=X.+Pell%C3%A9author=G.+Rummelauthor=B.+Salemauthor=M.+Warmuthauthor=A.+A.+Wylieauthor=T.+Zollerauthor=A.+L.+Marzinzikauthor=P.+Furet&title=Discovery+of+Asciminib+%28ABL001%29%2C+an+Allosteric+Inhibitor+of+the+Tyrosine+Kinase+Activity+of+BCR-ABL1&doi=10.1021%2Facs.jmedchem.8b01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span></div><div class="casAuthors">Schoepfer, Joseph; Jahnke, Wolfgang; Berellini, Giuliano; Buonamici, Silvia; Cotesta, Simona; Cowan-Jacob, Sandra W.; Dodd, Stephanie; Drueckes, Peter; Fabbro, Doriano; Gabriel, Tobias; Groell, Jean-Marc; Grotzfeld, Robert M.; Hassan, A. Quamrul; Henry, Chrystele; Iyer, Varsha; Jones, Darryl; Lombardo, Franco; Loo, Alice; Manley, Paul W.; Pelle, Xavier; Rummel, Gabriele; Salem, Bahaa; Warmuth, Markus; Wylie, Andrew A.; Zoller, Thomas; Marzinzik, Andreas L.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8120-8135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of asciminib I, an allosteric inhibitor of the BCR-ABL1 kinase for use in treating chronic myelogenous leukemia (CML), is delineated.  I binds to the myristate pocket of BCR-ABL1 and maintains activity against cancer cells possessing ATP-site mutations in BCR-ABL1 that render them resistant to tyrosine kinase inhibitors; cancer cells resistant to myristate-site mutations are likely to be susceptible to ATP-competitive inhibitors so that combinations of I and tyrosine kinase inhibitors should suppress the emergence of resistance.  Fragment-based screening using NMR and X-ray yielded ligands for the myristate pocket, while an NMR-based conformational assay guided the transformation of these inactive ligands into ABL1 inhibitors.  Further structure-based optimization for potency, physicochem., pharmacokinetic, and drug-like properties, culminated in I which is currently undergoing clin. studies in CML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIQbEIXXgRgbVg90H21EOLACvtfcHk0lhSLqQ4RhE_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN&md5=023a566cdf4faba6dfc05501b8b6a6ed</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01040%26sid%3Dliteratum%253Aachs%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DCotesta%26aufirst%3DS.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGroell%26aufirst%3DJ.%2BM.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DHenry%26aufirst%3DC.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DPell%25C3%25A9%26aufirst%3DX.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DSalem%26aufirst%3DB.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Asciminib%2520%2528ABL001%2529%252C%2520an%2520Allosteric%2520Inhibitor%2520of%2520the%2520Tyrosine%2520Kinase%2520Activity%2520of%2520BCR-ABL1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8120%26epage%3D8135%26doi%3D10.1021%2Facs.jmedchem.8b01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The Allosteric Inhibitor ABL001 Enables Dual Targeting of BCR-ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+Allosteric+Inhibitor+ABL001+Enables+Dual+Targeting+of+BCR-ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0lhSLqQ4RhE_Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520Allosteric%2520Inhibitor%2520ABL001%2520Enables%2520Dual%2520Targeting%2520of%2520BCR-ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span> <span> </span><span class="NLM_article-title">Protein Ensembles: How Does Nature Harness Thermodynamic Fluctuations for Life? The Diverse Functional Roles of Conformational Ensembles in the Cell</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6551</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6516-6551&author=G.+Weiauthor=W.+Xiauthor=R.+Nussinovauthor=B.+Ma&title=Protein+Ensembles%3A+How+Does+Nature+Harness+Thermodynamic+Fluctuations+for+Life%3F+The+Diverse+Functional+Roles+of+Conformational+Ensembles+in+the+Cell&doi=10.1021%2Facs.chemrev.5b00562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Ensembles: How Does Nature Harness Thermodynamic Fluctuations for Life? The Diverse Functional Roles of Conformational Ensembles in the Cell</span></div><div class="casAuthors">Wei, Guanghong; Xi, Wenhui; Nussinov, Ruth; Ma, Buyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6516-6551</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">All sol. proteins populate conformational ensembles that together constitute the native state.  Their fluctuations in water are intrinsic thermodn. phenomena, and the distributions of the states on the energy landscape are detd. by statistical thermodn.; however, they are optimized to perform their biol. functions.  In this review we briefly describe advances in free energy landscape studies of protein conformational ensembles.  Exptl. (NMR, small-angle X-ray scattering, single-mol. spectroscopy, and cryo-electron microscopy) and computational (replica-exchange mol. dynamics, metadynamics, and Markov state models) approaches have made great progress in recent years.  These address the challenging characterization of the highly flexible and heterogeneous protein ensembles.  We focus on structural aspects of protein conformational distributions, from collective motions of single- and multi-domain proteins, intrinsically disordered proteins, to multiprotein complexes.  Importantly, we highlight recent studies that illustrate functional adjustment of protein conformational ensembles in the crowded cellular environment.  We center on the role of the ensemble in recognition of small- and macro-mols. (protein and RNA/DNA) and emphasize emerging concepts of protein dynamics in enzyme catalysis.  Overall, protein ensembles link fundamental physicochem. principles and protein behavior and the cellular network and its regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Kcwy8a6LHrVg90H21EOLACvtfcHk0ljaAM4XDwj-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsLo%253D&md5=fac9ab64e11aa3a4f2f0988bf1db1209</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00562%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DG.%26aulast%3DXi%26aufirst%3DW.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DB.%26atitle%3DProtein%2520Ensembles%253A%2520How%2520Does%2520Nature%2520Harness%2520Thermodynamic%2520Fluctuations%2520for%2520Life%253F%2520The%2520Diverse%2520Functional%2520Roles%2520of%2520Conformational%2520Ensembles%2520in%2520the%2520Cell%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6516%26epage%3D6551%26doi%3D10.1021%2Facs.chemrev.5b00562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">The Molecular Basis of G Protein–Coupled Receptor Activation</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060614-033910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biochem-060614-033910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29925258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=897-919&author=W.+I.+Weisauthor=B.+K.+Kobilka&title=The+Molecular+Basis+of+G+Protein%E2%80%93Coupled+Receptor+Activation&doi=10.1146%2Fannurev-biochem-060614-033910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Basis of G Protein-Coupled Receptor Activation</span></div><div class="casAuthors">Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">897-919</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) mediate the majority of cellular responses to external stimuli.  Upon activation by a ligand, the receptor binds to a partner heterotrimeric G protein and promotes exchange of GTP for GDP, leading to dissocn. of the G protein into α and βγ subunits that mediate downstream signals.  GPCRs can also activate distinct signaling pathways through arrestins.  Active states of GPCRs form by small rearrangements of the ligand-binding, or orthosteric, site that are amplified into larger conformational changes.  Mol. understanding of the allosteric coupling between ligand binding and G protein or arrestin interaction is emerging from structures of several GPCRs crystd. in inactive and active states, spectroscopic data, and computer simulations.  The coupling is loose, rather than concerted, and agonist binding does not fully stabilize the receptor in an active conformation.  Distinct intermediates whose populations are shifted by ligands of different efficacies underlie the complex pharmacol. of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfSIylPxq7yLVg90H21EOLACvtfcHk0ljaAM4XDwj-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsrvM&md5=71f1f2d3c1d9be27da70dd472d7d44f2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060614-033910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060614-033910%26sid%3Dliteratum%253Aachs%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DThe%2520Molecular%2520Basis%2520of%2520G%2520Protein%25E2%2580%2593Coupled%2520Receptor%2520Activation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D897%26epage%3D919%26doi%3D10.1146%2Fannurev-biochem-060614-033910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharlow, E. R.</span></span> <span> </span><span class="NLM_article-title">Drugging Undruggable Molecular Cancer Targets</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-pharmtox-010715-103440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26527069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=23-40&author=J.+S.+Lazoauthor=E.+R.+Sharlow&title=Drugging+Undruggable+Molecular+Cancer+Targets&doi=10.1146%2Fannurev-pharmtox-010715-103440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Undruggable Molecular Cancer Targets</span></div><div class="casAuthors">Lazo, John S.; Sharlow, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-40</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Cancer, more than any other human disease, now has a surfeit of potential mol. targets poised for therapeutic exploitation.  Currently, a no. of attractive and validated cancer targets remain outside of the reach of pharmacol. regulation.  Some have been described as undruggable, at least by traditional strategies.  In this article, we outline the basis for the undruggable moniker, propose a reclassification of these targets as undrugged, and highlight three general classes of this imposing group as exemplars with some attendant strategies currently being explored to reclassify them.  Expanding the spectrum of disease-relevant targets to pharmacol. manipulation is central to reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpny-ev1lJKqbVg90H21EOLACvtfcHk0ljaAM4XDwj-9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O&md5=5ab0518d4155a00e2a898a6f68d551d8</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103440%26sid%3Dliteratum%253Aachs%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DSharlow%26aufirst%3DE.%2BR.%26atitle%3DDrugging%2520Undruggable%2520Molecular%2520Cancer%2520Targets%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D23%26epage%3D40%26doi%3D10.1146%2Fannurev-pharmtox-010715-103440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the “Undruggable” Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%9CUndruggable%E2%80%9D+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0lgFdEnz5tDmCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%259CUndruggable%25E2%2580%259D%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyanaraman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Journey, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Exploration of Benzothiazole Rhodacyanines as Allosteric Inhibitors of Protein-Protein Interactions with Heat Shock Protein 70 (Hsp70)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6163</span>– <span class="NLM_lpage">6177</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1altb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6163-6177&author=H.+Shaoauthor=X.+Liauthor=M.+A.+Mosesauthor=L.+A.+Gilbertauthor=C.+Kalyanaramanauthor=Z.+T.+Youngauthor=M.+Chernovaauthor=S.+N.+Journeyauthor=J.+S.+Weissmanauthor=B.+Hannauthor=M.+P.+Jacobsonauthor=L.+Neckersauthor=J.+E.+Gestwicki&title=Exploration+of+Benzothiazole+Rhodacyanines+as+Allosteric+Inhibitors+of+Protein-Protein+Interactions+with+Heat+Shock+Protein+70+%28Hsp70%29&doi=10.1021%2Facs.jmedchem.8b00583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Benzothiazole Rhodacyanines as Allosteric Inhibitors of Protein-Protein Interactions with Heat Shock Protein 70 (Hsp70)</span></div><div class="casAuthors">Shao, Hao; Li, Xiaokai; Moses, Michael A.; Gilbert, Luke A.; Kalyanaraman, Chakrapani; Young, Zapporah T.; Chernova, Margarita; Journey, Sara N.; Weissman, Jonathan S.; Hann, Byron; Jacobson, Matthew P.; Neckers, Len; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6163-6177</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer cells rely on the chaperone heat shock protein 70 (Hsp70) for survival and proliferation.  Recently, benzothiazole rhodacyanines have been shown to bind an allosteric site on Hsp70, interrupting its binding to nucleotide-exchange factors (NEFs) and promoting cell death in breast cancer cell lines.  However, proof-of-concept mols., such as JG-98, have relatively modest potency (EC50 ≈ 0.7-0.4 μM) and are rapidly metabolized in animals.  Here, the authors explored this chem. series through structure- and property-based design of ∼300 analogs, showing that the most potent had >10-fold improved EC50 values (∼0.05 to 0.03 μM) against two breast cancer cells.  Biomarkers and whole genome CRISPRi screens confirmed members of the Hsp70 family as cellular targets.  On the basis of these results, JG-231 was found to reduce tumor burden in an MDA-MB-231 xenograft model (4 mg/kg, i.p.).  Together, these studies support the hypothesis that Hsp70 may be a promising target for anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZKt2SFyYM6rVg90H21EOLACvtfcHk0lgFdEnz5tDmCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1altb3O&md5=16effcd3cd1b4772583fefcc5967a75e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00583%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMoses%26aufirst%3DM.%2BA.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DYoung%26aufirst%3DZ.%2BT.%26aulast%3DChernova%26aufirst%3DM.%26aulast%3DJourney%26aufirst%3DS.%2BN.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DNeckers%26aufirst%3DL.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DExploration%2520of%2520Benzothiazole%2520Rhodacyanines%2520as%2520Allosteric%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%2520with%2520Heat%2520Shock%2520Protein%252070%2520%2528Hsp70%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6163%26epage%3D6177%26doi%3D10.1021%2Facs.jmedchem.8b00583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pricer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, A. K.</span></span> <span> </span><span class="NLM_article-title">From Fuzzy to Function: The New Frontier of Protein-Protein Interactions</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.6b00565</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.6b00565" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1WktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=584-589&author=R.+Pricerauthor=J.+E.+Gestwickiauthor=A.+K.+Mapp&title=From+Fuzzy+to+Function%3A+The+New+Frontier+of+Protein-Protein+Interactions&doi=10.1021%2Facs.accounts.6b00565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">From Fuzzy to Function: The New Frontier of Protein-Protein Interactions</span></div><div class="casAuthors">Pricer, Rachel; Gestwicki, Jason E.; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">584-589</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The notion that many proteins have a high degree of disorder has been in the literature for decades.  Such sequences have been commonly regarded as a nuisance; they caused aggregation and pptn. and often eluded structural and mechanistic studies.  These motifs have also been regarded as largely useless from a functional standpoint, perhaps because no few believed that floppy motifs could encode information that facilitated specific function.  Here, the role of these intrinsically disordered motifs in protein-protein interactions is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkBKJUlxPLkLVg90H21EOLACvtfcHk0lgFdEnz5tDmCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1WktL4%253D&md5=70b5132b30c231747534ce42962401dd</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.6b00565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.6b00565%26sid%3Dliteratum%253Aachs%26aulast%3DPricer%26aufirst%3DR.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DFrom%2520Fuzzy%2520to%2520Function%253A%2520The%2520New%2520Frontier%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D584%26epage%3D589%26doi%3D10.1021%2Facs.accounts.6b00565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Hidden Allosteric Sites as Novel Targets for Allosteric Drug Design</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2017.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29030241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=359-365&author=S.+Luauthor=M.+Jiauthor=D.+Niauthor=J.+Zhang&title=Discovery+of+Hidden+Allosteric+Sites+as+Novel+Targets+for+Allosteric+Drug+Design&doi=10.1016%2Fj.drudis.2017.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of hidden allosteric sites as novel targets for allosteric drug design</span></div><div class="casAuthors">Lu, Shaoyong; Ji, Mingfei; Ni, Duan; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-365</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hidden allosteric sites, as a novel type of allosteric site, are invisible in ligand-unbound (apo) crystal structures, but can emerge in ligand-bound (holo) crystal structures when a specific ligand binds to, and stabilizes, a unique conformation favored by the ligand.  However, the identification of these sites is a significant challenge.  Several computational and exptl. approaches have been developed to identify such sites in proteins.  Here, we outline these approaches, with a focus on examples of the successful use of such techniques.  The discovery of hidden allosteric sites offers a new avenue for facilitating drug design by greatly expanding the repertoire of available drug targets, contributing to the search for allosteric drugs for the treatment of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpObJvqLArGy7Vg90H21EOLACvtfcHk0lhsRogVtGZs8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kqs77F&md5=da29385caf2393feb520aa402dbf4134</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Hidden%2520Allosteric%2520Sites%2520as%2520Novel%2520Targets%2520for%2520Allosteric%2520Drug%2520Design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D359%26epage%3D365%26doi%3D10.1016%2Fj.drudis.2017.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brogi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramunno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecorelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pambianchi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galatello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compagnoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Focher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biamonti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valacchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campiani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span> <span> </span><span class="NLM_article-title">First Dual AK/GSK-3β Inhibitors Endowed with Antioxidant Properties as Multifunctional, Potential Neuroprotective Agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ejmech.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28689095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=438-457&author=S.+Brogiauthor=A.+Ramunnoauthor=L.+Saviauthor=G.+Chemiauthor=G.+Alfanoauthor=A.+Pecorelliauthor=E.+Pambianchiauthor=P.+Galatelloauthor=G.+Compagnoniauthor=F.+Focherauthor=G.+Biamontiauthor=G.+Valacchiauthor=S.+Butiniauthor=S.+Gemmaauthor=G.+Campianiauthor=M.+Brindisi&title=First+Dual+AK%2FGSK-3%CE%B2+Inhibitors+Endowed+with+Antioxidant+Properties+as+Multifunctional%2C+Potential+Neuroprotective+Agents&doi=10.1016%2Fj.ejmech.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">First dual AK/GSK-3β inhibitors endowed with antioxidant properties as multifunctional, potential neuroprotective agents</span></div><div class="casAuthors">Brogi, Simone; Ramunno, Anna; Savi, Lida; Chemi, Giulia; Alfano, Gloria; Pecorelli, Alessandra; Pambianchi, Erika; Galatello, Paola; Compagnoni, Giulia; Focher, Federico; Biamonti, Giuseppe; Valacchi, Giuseppe; Butini, Stefania; Gemma, Sandra; Campiani, Giuseppe; Brindisi, Margherita</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">438-457</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The manuscript deals with the design, synthesis and biol. evaluation of novel benzoxazinone- and indole-based compds. as multifunctional neuroprotective agents.  These compds. inhibit human adenosine kinase (hAK) and human glycogen synthase kinase 3 beta (hGSK-3β) enzymes.  In silico hints drove the synthesis of appropriately decorated benzoxazinones I (R1 = Ph, 1-naphthyl, 2-naphthyl; R2 = CH2C6H5, 4-ClC6H4CH2, 4-O2NC6H4CH2, etc.; X = H, F, morpholin-1-yl; Y = H; XY = OCH2O; Z = H, O) and indoles II (R1 = 4-H3CC6H4, 1-naphthyl; R2 = CH3, N(Et)2) and biochem. anal. revealed their behavior as allosteric inhibitors of hGSK-3β.  For hit III and compds. I (R1 = 1-naphthyl; R2 = 4-FC6H4CH2; X = Y = H; Z = O), I (R1 = 1-naphthyl; R2 = CH2C6H5; XY = OCH2O; Z = O) and II (R1 = 1-naphthyl; R2 = CH3) the potential antioxidant profile was assessed in human neuroblastoma cell lines (IMR 32), by evaluating the protective effect of selected compds. against H2O2 cytotoxicity and reactive oxygen species (ROS) prodn.  Results showed a strong efficacy of the tested compds., even at the lower doses, in counteracting the induced oxidative stress (50 μM of H2O2) and in preventing ROS formation.  In addn., the tested compds. did not show any cytotoxic effect detd. by the LDH release, at the concn. range analyzed (from 0.1 to 50 μM).  This study allowed the identification of compd. I (R1 = 1-naphthyl; R2 = CH2C6H5; XY = OCH2O; Z = O), as the first dual hAK/hGSK-3β inhibitor reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrno6lxql7c_7Vg90H21EOLACvtfcHk0lhsRogVtGZs8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvO&md5=1ea9d5251ac6d10f173c170e7c749b4b</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DBrogi%26aufirst%3DS.%26aulast%3DRamunno%26aufirst%3DA.%26aulast%3DSavi%26aufirst%3DL.%26aulast%3DChemi%26aufirst%3DG.%26aulast%3DAlfano%26aufirst%3DG.%26aulast%3DPecorelli%26aufirst%3DA.%26aulast%3DPambianchi%26aufirst%3DE.%26aulast%3DGalatello%26aufirst%3DP.%26aulast%3DCompagnoni%26aufirst%3DG.%26aulast%3DFocher%26aufirst%3DF.%26aulast%3DBiamonti%26aufirst%3DG.%26aulast%3DValacchi%26aufirst%3DG.%26aulast%3DButini%26aufirst%3DS.%26aulast%3DGemma%26aufirst%3DS.%26aulast%3DCampiani%26aufirst%3DG.%26aulast%3DBrindisi%26aufirst%3DM.%26atitle%3DFirst%2520Dual%2520AK%252FGSK-3%25CE%25B2%2520Inhibitors%2520Endowed%2520with%2520Antioxidant%2520Properties%2520as%2520Multifunctional%252C%2520Potential%2520Neuroprotective%2520Agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D438%26epage%3D457%26doi%3D10.1016%2Fj.ejmech.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Are the Apo Proteins Suitable for the Rational Discovery of Allosteric Drugs?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00735</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00735" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCisb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=597-604&author=X.+Anauthor=S.+Luauthor=K.+Songauthor=Q.+Shenauthor=M.+Huangauthor=X.+Yaoauthor=H.+Liuauthor=J.+Zhang&title=Are+the+Apo+Proteins+Suitable+for+the+Rational+Discovery+of+Allosteric+Drugs%3F&doi=10.1021%2Facs.jcim.8b00735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Are the Apo Proteins Suitable for the Rational Discovery of Allosteric Drugs?</span></div><div class="casAuthors">An, Xiaoli; Lu, Shaoyong; Song, Kun; Shen, Qiancheng; Huang, Meilan; Yao, Xiaojun; Liu, Huanxiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">597-604</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric modulators, by targeting the less-conserved allosteric sites, represent an innovative strategy in drug discovery.  These modulators have a distinctive advantage over orthosteric ligands that attach to the conserved, functional orthosteric sites.  However, in structure-based drug design, it remains unclear whether allosteric protein structures detd. without orthosteric ligand binding are suitable for allosteric drug screening.  In this study, we performed large-scale conformational samplings of six representative allosteric proteins uncomplexed (apo) and complexed (holo) with orthosteric ligands to explore the effect of orthosteric site binding on the conformational dynamics of allosteric sites.  The results, coupled with the redocking evaluation of allosteric modulators to their apo and holo proteins using their MD trajectories, indicated that orthosteric site binding had an effect on the dynamics of the allosteric sites and allosteric modulators preferentially bound to their holo proteins.  According to the anal. data, we constructed a new correlation model for quantifying the allosteric site change driven by substrate binding to the orthosteric site.  These results highlight the strong demand to select holo allosteric proteins as initial inputs in structure-based allosteric drug screening when the distance between orthosteric and allosteric sites in the protein is below 5 Å, which is expected to contribute to allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2oyvmsSoYj7Vg90H21EOLACvtfcHk0lgmBEGqN-WvqQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCisb3J&md5=b98fc6933d1075664cf53d3d45520339</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00735%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAre%2520the%2520Apo%2520Proteins%2520Suitable%2520for%2520the%2520Rational%2520Discovery%2520of%2520Allosteric%2520Drugs%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D597%26epage%3D604%26doi%3D10.1021%2Facs.jcim.8b00735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Drugging Ras GTPase: A Comprehensive Mechanistic and Signaling Structural View</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">4929</span>– <span class="NLM_lpage">4952</span>, <span class="refDoi"> DOI: 10.1039/C5CS00911A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2FC5CS00911A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27396271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCiurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2016&pages=4929-4952&author=S.+Luauthor=H.+Jangauthor=S.+Guauthor=J.+Zhangauthor=R.+Nussinov&title=Drugging+Ras+GTPase%3A+A+Comprehensive+Mechanistic+and+Signaling+Structural+View&doi=10.1039%2FC5CS00911A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Ras GTPase: a comprehensive mechanistic and signaling structural view</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Gu, Shuo; Zhang, Jian; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4929-4952</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Ras proteins are small GTPases, cycling between inactive GDP-bound and active GTP-bound states.  Through these switches they regulate signaling that controls cell growth and proliferation.  Activating Ras mutations are assocd. with approx. 30% of human cancers, which are frequently resistant to std. therapies.  Over the past few years, structural biol. and in silico drug design, coupled with improved screening technol., led to a handful of promising inhibitors, raising the possibility of drugging Ras proteins.  At the same time, the invariable emergence of drug resistance argues for the crit. importance of addnl. honing in on signaling pathways which are likely to be involved.  Here we overview current advances in Ras structural knowledge, including the conformational dynamic of full-length Ras in soln. and at the membrane, therapeutic inhibition of Ras activity by targeting its active site, allosteric sites, and Ras-effector protein-protein interfaces, Ras dimers, the K-Ras4B/calmodulin/PI3Kα trimer, and targeting Ras with siRNA.  To mitigate drug resistance, we propose signaling pathways that can be co-targeted along with Ras and explain why.  These include pathways leading to the expression (or activation) of YAP1 and c-Myc.  We postulate that these and Ras signaling pathways, MAPK/ERK and PI3K/Akt/mTOR, act independently and in corresponding ways in cell cycle control.  The structural data are instrumental in the discovery and development of Ras inhibitors for treating RAS-driven cancers.  Together with the signaling blueprints through which drug resistance can evolve, this review provides a comprehensive and innovative master plan for tackling mutant Ras proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBc0hxnrzfRrVg90H21EOLACvtfcHk0liM2sZ7h8Hpiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCiurrK&md5=f98483885d066c0ec809535b1caefc8a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1039%2FC5CS00911A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CS00911A%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DDrugging%2520Ras%2520GTPase%253A%2520A%2520Comprehensive%2520Mechanistic%2520and%2520Signaling%2520Structural%2520View%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2016%26volume%3D45%26spage%3D4929%26epage%3D4952%26doi%3D10.1039%2FC5CS00911A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging the Undruggable RAS: Mission Possible?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1038/nrd4389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd4389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25323927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=828-851&author=A.+D.+Coxauthor=S.+W.+Fesikauthor=A.+C.+Kimmelmanauthor=J.+Luoauthor=C.+J.+Der&title=Drugging+the+Undruggable+RAS%3A+Mission+Possible%3F&doi=10.1038%2Fnrd4389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggable RAS: Mission Possible?</span></div><div class="casAuthors">Cox, Adrienne D.; Fesik, Stephen W.; Kimmelman, Alec C.; Luo, Ji; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">828-851</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite more than three decades of intensive effort, no effective pharmacol. inhibitors of the RAS oncoproteins have reached the clinic, prompting the widely held perception that RAS proteins are 'undruggable'.  However, recent data from the lab. and the clinic have renewed our hope for the development of RAS-inhibitory mols.  In this Review, we summarize the progress and the promise of five key approaches.  Firstly, we focus on the prospects of using direct inhibitors of RAS.  Secondly, we address the issue of whether blocking RAS membrane assocn. is a viable approach.  Thirdly, we assess the status of targeting RAS downstream effector signalling, which is arguably the most favorable current approach.  Fourthly, we address whether the search for synthetic lethal interactors of mutant RAS still holds promise.  Finally, RAS-mediated changes in cell metab. have recently been described and we discuss whether these changes could be exploited for new therapeutic directions.  We conclude with perspectives on how addnl. complexities, which are not yet fully understood, may affect each of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvWzW1JqxN4LVg90H21EOLACvtfcHk0liM2sZ7h8Hpiw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE&md5=6c8645293b079023de7d31011ae1a192</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnrd4389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4389%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520the%2520Undruggable%2520RAS%253A%2520Mission%2520Possible%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D828%26epage%3D851%26doi%3D10.1038%2Fnrd4389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerowitz, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekritz, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonds, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthay, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span> <span> </span><span class="NLM_article-title">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ccr.2014.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25175806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=414-427&author=W.+C.+Gustafsonauthor=J.+G.+Meyerowitzauthor=E.+A.+Nekritzauthor=J.+Chenauthor=C.+Benesauthor=E.+Charronauthor=E.+F.+Simondsauthor=R.+Seegerauthor=K.+K.+Matthayauthor=N.+T.+Hertzauthor=M.+Eilersauthor=K.+M.+Shokatauthor=W.+A.+Weiss&title=Drugging+MYCN+through+an+Allosteric+Transition+in+Aurora+Kinase+A&doi=10.1016%2Fj.ccr.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span></div><div class="casAuthors">Gustafson, William Clay; Meyerowitz, Justin Gabriel; Nekritz, Erin A.; Chen, Justin; Benes, Cyril; Charron, Elise; Simonds, Erin F.; Seeger, Robert; Matthay, Katherine K.; Hertz, Nicholas T.; Eilers, Martin; Shokat, Kevan M.; Weiss, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">414-427</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC proteins are major drivers of cancer, yet are considered undruggable because their DNA binding domains are composed of two extended alpha helixes with no apparent surfaces for small-mol. binding.  Proteolytic degrdn. of MYCN protein is regulated in part by a kinase-independent function of Aurora A.  We describe a class of inhibitors that disrupts the native conformation of Aurora A and drives the degrdn. of MYCN protein across MYCN-driven cancers.  Comparison of cocrystal structures with structure-activity relationships across multiple inhibitors and chemotypes, coupled with mechanistic studies and biochem. assays, delineates an Aurora A conformation-specific effect on proteolytic degrdn. of MYCN, rather than simple nanomolar-level inhibition of Aurora A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8At2LNPE-ubVg90H21EOLACvtfcHk0lhg187PAqwtvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ&md5=20944894851fe5466422c149a0a0b266</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DW.%2BC.%26aulast%3DMeyerowitz%26aufirst%3DJ.%2BG.%26aulast%3DNekritz%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%26aulast%3DCharron%26aufirst%3DE.%26aulast%3DSimonds%26aufirst%3DE.%2BF.%26aulast%3DSeeger%26aufirst%3DR.%26aulast%3DMatthay%26aufirst%3DK.%2BK.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26atitle%3DDrugging%2520MYCN%2520through%2520an%2520Allosteric%2520Transition%2520in%2520Aurora%2520Kinase%2520A%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D26%26spage%3D414%26epage%3D427%26doi%3D10.1016%2Fj.ccr.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehmani, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Serrano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span> <span> </span><span class="NLM_article-title">Pirin Is an Iron-Dependent Redox Regulator of NF- B</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">9722</span>– <span class="NLM_lpage">9727</span>, <span class="refDoi"> DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+Is+an+Iron-Dependent+Redox+Regulator+of+NF-+B&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-κB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor κB (NF-κB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-κB proteins to target κB genes, a finding that suggests that pirin performs a redox-sensing role in NF-κB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an β-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-κB binding that is responsible for modulation of NF-κB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-κB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lhg187PAqwtvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520Is%2520an%2520Iron-Dependent%2520Redox%2520Regulator%2520of%2520NF-%2520B%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konidaris, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonks, N. K.</span></span> <span> </span><span class="NLM_article-title">A Potent, Selective, and Orally Bioavailable Inhibitor of the Protein-Tyrosine Phosphatase PTP1B Improves Insulin and Leptin Signaling in Animal Models</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1517</span>– <span class="NLM_lpage">1525</span>, <span class="refDoi"> DOI: 10.1074/jbc.C117.819110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.C117.819110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29217773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVygurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1517-1525&author=N.+Krishnanauthor=K.+F.+Konidarisauthor=G.+Gasserauthor=N.+K.+Tonks&title=A+Potent%2C+Selective%2C+and+Orally+Bioavailable+Inhibitor+of+the+Protein-Tyrosine+Phosphatase+PTP1B+Improves+Insulin+and+Leptin+Signaling+in+Animal+Models&doi=10.1074%2Fjbc.C117.819110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">A potent, selective, and orally bioavailable inhibitor of the protein-tyrosine phosphatase PTP1B improves insulin and leptin signaling in animal models</span></div><div class="casAuthors">Krishnan, Navasona; Konidaris, Konstantis F.; Gasser, Gilles; Tonks, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1517-1525</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein tyrosine phosphatase PTP1B is a neg. regulator of insulin and leptin signaling and a highly validated therapeutic target for diabetes and obesity.  Conventional approaches to drug development have produced potent and specific PTP1B inhibitors, but these inhibitors lack oral bioavailability, which limits their potential for drug development.  Here, the authors report that DPM-1001, an analog of the specific PTP1B inhibitor trodusquemine (MSI-1436), is a potent, specific, and orally bioavailable inhibitor of PTP1B.  DPM-1001 also chelates copper, which enhanced its potency as a PTP1B inhibitor.  DPM-1001 displayed anti-diabetic properties that were assocd. with enhanced signaling through insulin and leptin receptors in animal models of diet-induced obesity.  Therefore, DPM-1001 represents a proof of concept for a new approach to therapeutic intervention in diabetes and obesity.  Although the PTPs have been considered undruggable, the findings of this study suggest that allosteric PTP inhibitors may help reinvigorate drug development efforts that focus on this important family of signal-transducing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBBIeqnX9-xLVg90H21EOLACvtfcHk0liV5LFY9h4JWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVygurY%253D&md5=0c020dfd2dbb33d80e925f0755c9b0f0</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C117.819110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C117.819110%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DN.%26aulast%3DKonidaris%26aufirst%3DK.%2BF.%26aulast%3DGasser%26aufirst%3DG.%26aulast%3DTonks%26aufirst%3DN.%2BK.%26atitle%3DA%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520the%2520Protein-Tyrosine%2520Phosphatase%2520PTP1B%2520Improves%2520Insulin%2520and%2520Leptin%2520Signaling%2520in%2520Animal%2520Models%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1517%26epage%3D1525%26doi%3D10.1074%2Fjbc.C117.819110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10205</span>– <span class="NLM_lpage">10219</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01520</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01520" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10205-10219&author=J.+Xieauthor=X.+Siauthor=S.+Guauthor=M.+Wangauthor=J.+Shenauthor=H.+Liauthor=D.+Liauthor=Y.+Fangauthor=C.+Liuauthor=J.+Zhu&title=Allosteric+Inhibitors+of+SHP2+with+Therapeutic+Potential+for+Cancer+Treatment&doi=10.1021%2Facs.jmedchem.7b01520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span></div><div class="casAuthors">Xie, Jingjing; Si, Xiaojia; Gu, Shoulai; Wang, Mingliang; Shen, Jian; Li, Haoyan; Li, Dan; Fang, Yanjia; Liu, Cong; Zhu, Jidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10205-10219</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2, a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene, is involved in multiple cell signaling processes including Ras/MAPK and Hippo/YAP pathways.  SHP2 has been shown to contribute to the progression of a no. of cancer types including leukemia, gastric, and breast cancers.  It also regulates T-cell activation by interacting with inhibitory immune checkpoint receptors such as the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA).  Thus, SHP2 inhibitors have drawn great attention by both inhibiting tumor cell proliferation and activating T cell immune responses toward cancer cells.  In this study, the authors report the identification of an allosteric SHP2 inhibitor I that locks SHP2 in a closed conformation by binding to the interface of the N-terminal SH2, C-terminal SH2, and phosphatase domains.  Compd. I suppresses MAPK signaling pathway and YAP transcriptional activity and shows antitumor activity in vivo.  The results indicate that allosteric inhibition of SHP2 could be a feasible approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWHA25LKpH7Vg90H21EOLACvtfcHk0liV5LFY9h4JWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM&md5=4aaabafae3250476b7bf3687d83cffd3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01520%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSi%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DAllosteric%2520Inhibitors%2520of%2520SHP2%2520with%2520Therapeutic%2520Potential%2520for%2520Cancer%2520Treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10205%26epage%3D10219%26doi%3D10.1021%2Facs.jmedchem.7b01520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koveal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akshinthala, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kragelj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauss, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peti, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonks, N. K.</span></span> <span> </span><span class="NLM_article-title">Targeting the Disordered C Terminus of PTP1B with an Allosteric Inhibitor</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24845231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=558-566&author=N.+Krishnanauthor=D.+Kovealauthor=D.+H.+Millerauthor=B.+Xueauthor=S.+D.+Akshinthalaauthor=J.+Krageljauthor=M.+R.+Jensenauthor=C.+M.+Gaussauthor=R.+Pageauthor=M.+Blackledgeauthor=S.+K.+Muthuswamyauthor=W.+Petiauthor=N.+K.+Tonks&title=Targeting+the+Disordered+C+Terminus+of+PTP1B+with+an+Allosteric+Inhibitor&doi=10.1038%2Fnchembio.1528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the disordered C terminus of PTP1B with an allosteric inhibitor</span></div><div class="casAuthors">Krishnan, Navasona; Koveal, Dorothy; Miller, Daniel H.; Xue, Bin; Akshinthala, Sai Dipikaa; Kragelj, Jaka; Jensen, Malene Ringkjoebing; Gauss, Carla-Maria; Page, Rebecca; Blackledge, Martin; Muthuswamy, Senthil K.; Peti, Wolfgang; Tonks, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">558-566</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PTP1B, a validated therapeutic target for diabetes and obesity, has a crit. pos. role in HER2 signaling in breast tumorigenesis.  Efforts to develop therapeutic inhibitors of PTP1B have been frustrated by the chem. properties of the active site.  We define a new mechanism of allosteric inhibition that targets the C-terminal, noncatalytic segment of PTP1B.  We present what is to our knowledge the first ensemble structure of PTP1B contg. this intrinsically disordered segment, within which we identified a binding site for the small-mol. inhibitor MSI-1436.  We demonstrate binding to a second site close to the catalytic domain, with cooperative effects between the two sites locking PTP1B in an inactive state.  MSI-1436 antagonized HER2 signaling, inhibited tumorigenesis in xenografts and abrogated metastasis in the NDL2 mouse model of breast cancer, validating inhibition of PTP1B as a therapeutic strategy in breast cancer.  This new approach to inhibition of PTP1B emphasizes the potential of disordered segments of proteins as specific binding sites for therapeutic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo526if7ja9XbVg90H21EOLACvtfcHk0ljvfsl1yQK7Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ylsL8%253D&md5=10a460f3a5deafb9f7c206246d67a29b</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1528%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DN.%26aulast%3DKoveal%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DD.%2BH.%26aulast%3DXue%26aufirst%3DB.%26aulast%3DAkshinthala%26aufirst%3DS.%2BD.%26aulast%3DKragelj%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DGauss%26aufirst%3DC.%2BM.%26aulast%3DPage%26aufirst%3DR.%26aulast%3DBlackledge%26aufirst%3DM.%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26aulast%3DPeti%26aufirst%3DW.%26aulast%3DTonks%26aufirst%3DN.%2BK.%26atitle%3DTargeting%2520the%2520Disordered%2520C%2520Terminus%2520of%2520PTP1B%2520with%2520an%2520Allosteric%2520Inhibitor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D558%26epage%3D566%26doi%3D10.1038%2Fnchembio.1528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanevich, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satyshur, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadzinski, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Protein Phosphatase 2A Stable Latency</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1611</span>– <span class="NLM_lpage">1699</span>, <span class="refDoi"> DOI: 10.1038/ncomms2663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms2663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23511475" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1611-1699&author=L.+Jiangauthor=V.+Stanevichauthor=K.+A.+Satyshurauthor=M.+Kongauthor=G.+R.+Watkinsauthor=B.+E.+Wadzinskiauthor=R.+Senguptaauthor=Y.+Xing&title=Structural+Basis+of+Protein+Phosphatase+2A+Stable+Latency&doi=10.1038%2Fncomms2663"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1038%2Fncomms2663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2663%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DStanevich%26aufirst%3DV.%26aulast%3DSatyshur%26aufirst%3DK.%2BA.%26aulast%3DKong%26aufirst%3DM.%26aulast%3DWatkins%26aufirst%3DG.%2BR.%26aulast%3DWadzinski%26aufirst%3DB.%2BE.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DXing%26aufirst%3DY.%26atitle%3DStructural%2520Basis%2520of%2520Protein%2520Phosphatase%25202A%2520Stable%2520Latency%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D1611%26epage%3D1699%26doi%3D10.1038%2Fncomms2663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longgood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tambar, U. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibition of Hypoxia Inducible Factor-2 with Small Molecules</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23434853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFygtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=271-276&author=T.+H.+Scheuermannauthor=Q.+Liauthor=H.+W.+Maauthor=J.+Keyauthor=L.+Zhangauthor=R.+Chenauthor=J.+A.+Garciaauthor=J.+Naidooauthor=J.+Longgoodauthor=D.+E.+Frantzauthor=U.+K.+Tambarauthor=K.+H.+Gardnerauthor=R.+K.+Bruick&title=Allosteric+Inhibition+of+Hypoxia+Inducible+Factor-2+with+Small+Molecules&doi=10.1038%2Fnchembio.1185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibition of hypoxia inducible factor-2 with small molecules</span></div><div class="casAuthors">Scheuermann, Thomas H.; Li, Qiming; Ma, He-Wen; Key, Jason; Zhang, Lei; Chen, Rui; Garcia, Joseph A.; Naidoo, Jacinth; Longgood, Jamie; Frantz, Doug E.; Tambar, Uttam K.; Gardner, Kevin H.; Bruick, Richard K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-276</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hypoxia inducible factors (HIFs) are heterodimeric transcription factors induced in many cancers where they frequently promote the expression of protumorigenic pathways.  Though transcription factors are typically considered 'undruggable', the PAS-B domain of the HIF-2α subunit contains a large cavity within its hydrophobic core that offers a unique foothold for small-mol. regulation.  Here we identify artificial ligands that bind within this pocket and characterize the resulting structural and functional changes caused by binding.  Notably, these ligands antagonize HIF-2 heterodimerization and DNA-binding activity in vitro and in cultured cells, reducing HIF-2 target gene expression.  Despite the high sequence identity between HIF-2α and HIF-1α, these ligands are highly selective and do not affect HIF-1 function.  These chem. tools establish the mol. basis for selective regulation of HIF-2, providing potential therapeutic opportunities to intervene in HIF-2-driven tumors, such as renal cell carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBimby6zSuR7Vg90H21EOLACvtfcHk0ljvfsl1yQK7Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFygtLc%253D&md5=e2e58842b3fe6472248c10012de365fb</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1185%26sid%3Dliteratum%253Aachs%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DH.%2BW.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DJ.%2BA.%26aulast%3DNaidoo%26aufirst%3DJ.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DTambar%26aufirst%3DU.%2BK.%26aulast%3DGardner%26aufirst%3DK.%2BH.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26atitle%3DAllosteric%2520Inhibition%2520of%2520Hypoxia%2520Inducible%2520Factor-2%2520with%2520Small%2520Molecules%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D271%26epage%3D276%26doi%3D10.1038%2Fnchembio.1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minus, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vohidov, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasembeli, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolosov, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tweardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, Z. T.</span></span> <span> </span><span class="NLM_article-title">Rhodium(II) Proximity-Labeling Identifies a Novel Target Site on STAT3 for Inhibitors with Potent Anti-Leukemia Activity</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">13085</span>– <span class="NLM_lpage">13089</span>, <span class="refDoi"> DOI: 10.1002/anie.201506889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fanie.201506889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOmsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=13085-13089&author=M.+B.+Minusauthor=W.+Liuauthor=F.+Vohidovauthor=M.+M.+Kasembeliauthor=X.+Longauthor=M.+J.+Kruegerauthor=A.+Stevensauthor=M.+I.+Kolosovauthor=D.+J.+Tweardyauthor=E.+A.+R.+Sisonauthor=M.+S.+Redellauthor=Z.+T.+Ball&title=Rhodium%28II%29+Proximity-Labeling+Identifies+a+Novel+Target+Site+on+STAT3+for+Inhibitors+with+Potent+Anti-Leukemia+Activity&doi=10.1002%2Fanie.201506889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodium(II) Proximity-Labeling Identifies a Novel Target Site on STAT3 for Inhibitors with Potent Anti-Leukemia Activity</span></div><div class="casAuthors">Minus, Matthew B.; Liu, Wei; Vohidov, Farrukh; Kasembeli, Moses M.; Long, Xin; Krueger, Michael J.; Stevens, Alexandra; Kolosov, Mikhail I.; Tweardy, David J.; Sison, Edward Allan R.; Redell, Michele S.; Ball, Zachary T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">13085-13089</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Nearly 40 % of children with acute myeloid leukemia (AML) suffer relapse arising from chemoresistance, often involving upregulation of the oncoprotein STAT3 (signal transducer and activator of transcription 3).  Herein, rhodium(II)-catalyzed, proximity-driven modification identifies the STAT3 coiled-coil domain (CCD) as a novel ligand-binding site, and we describe a new naphthalene sulfonamide inhibitor that targets the CCD, blocks STAT3 function, and halts its disease-promoting effects in vitro, in tumor growth models, and in a leukemia mouse model, validating this new therapeutic target for resistant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaEI9lh5LaebVg90H21EOLACvtfcHk0ljvfsl1yQK7Gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOmsr3F&md5=8d1c509437ac74ff8382ff7a43898de9</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506889%26sid%3Dliteratum%253Aachs%26aulast%3DMinus%26aufirst%3DM.%2BB.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DVohidov%26aufirst%3DF.%26aulast%3DKasembeli%26aufirst%3DM.%2BM.%26aulast%3DLong%26aufirst%3DX.%26aulast%3DKrueger%26aufirst%3DM.%2BJ.%26aulast%3DStevens%26aufirst%3DA.%26aulast%3DKolosov%26aufirst%3DM.%2BI.%26aulast%3DTweardy%26aufirst%3DD.%2BJ.%26aulast%3DSison%26aufirst%3DE.%2BA.%2BR.%26aulast%3DRedell%26aufirst%3DM.%2BS.%26aulast%3DBall%26aufirst%3DZ.%2BT.%26atitle%3DRhodium%2528II%2529%2520Proximity-Labeling%2520Identifies%2520a%2520Novel%2520Target%2520Site%2520on%2520STAT3%2520for%2520Inhibitors%2520with%2520Potent%2520Anti-Leukemia%2520Activity%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D13085%26epage%3D13089%26doi%3D10.1002%2Fanie.201506889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulators</span>. <i>Compr. Med. Chem. III</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-409547-2.12324-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2FB978-0-12-409547-2.12324-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=276-296&author=S.+Luauthor=J.+Zhang&title=Allosteric+Modulators&doi=10.1016%2FB978-0-12-409547-2.12324-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-409547-2.12324-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-409547-2.12324-8%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosteric%2520Modulators%26jtitle%3DCompr.%2520Med.%2520Chem.%2520III%26date%3D2017%26volume%3D2%26spage%3D276%26epage%3D296%26doi%3D10.1016%2FB978-0-12-409547-2.12324-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis of ATP as an Allosteric Modulator</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003831</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25211773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=S.+Luauthor=W.+Huangauthor=Q.+Wangauthor=Q.+Shenauthor=S.+Liauthor=R.+Nussinovauthor=J.+Zhang&title=The+Structural+Basis+of+ATP+as+an+Allosteric+Modulator&doi=10.1371%2Fjournal.pcbi.1003831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of ATP as an allosteric modulator</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Wang, Qi; Shen, Qiancheng; Li, Shuai; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003831/1-e1003831/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Adenosine-5'-triphosphate (ATP) is generally regarded as a substrate for energy currency and protein modification.  Recent findings uncovered the allosteric function of ATP in cellular signal transduction but little is understood about this crit. behavior of ATP.  Through extensive anal. of ATP in soln. and proteins, we found that the free ATP can exist in the compact and extended conformations in soln., and the two different conformational characteristics may be responsible for ATP to exert distinct biol. functions: ATP mols. adopt both compact and extended conformations in the allosteric binding sites but conserve extended conformations in the substrate binding sites.  Nudged elastic band simulations unveiled the distinct dynamic processes of ATP binding to the corresponding allosteric and substrate binding sites of uridine monophosphate kinase, and suggested that in soln. ATP preferentially binds to the substrate binding sites of proteins.  When the ATP mols. occupy the allosteric binding sites, the allosteric trigger from ATP to fuel allosteric communication between allosteric and functional sites is stemmed mainly from the triphosphate part of ATP, with a small no. from the adenine part of ATP.  Taken together, our results provide overall understanding of ATP allosteric functions responsible for regulation in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5lsXXAkI3trVg90H21EOLACvtfcHk0lhg6gbElOPKow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamu7vN&md5=03e1c40183d194cb069cb55fe421982a</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003831%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520Structural%2520Basis%2520of%2520ATP%2520as%2520an%2520Allosteric%2520Modulator%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Rational Engineering of Enzyme Allosteric Regulation through Sequence Evolution Analysis</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002612</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1002612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1002612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22807670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOnsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=J.-S.+Yangauthor=S.+W.+Seoauthor=S.+Jangauthor=G.+Y.+Jungauthor=S.+Kim&title=Rational+Engineering+of+Enzyme+Allosteric+Regulation+through+Sequence+Evolution+Analysis&doi=10.1371%2Fjournal.pcbi.1002612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Rational engineering of enzyme allosteric regulation through sequence evolution analysis</span></div><div class="casAuthors">Yang, Jae-Seong; Seo, Sang Woo; Jang, Sungho; Jung, Gyoo Yeol; Kim, Sanguk</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1002612</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Control of enzyme allosteric regulation is required to drive metabolic flux toward desired levels.  Although the three-dimensional (3D) structures of many enzyme-ligand complexes are available, it is still difficult to rationally engineer an allosterically regulatable enzyme without decreasing its catalytic activity.  Here, we describe an effective strategy to deregulate the allosteric inhibition of enzymes based on the mol. evolution and physicochem. characteristics of allosteric ligand-binding sites.  We found that allosteric sites are evolutionarily variable and comprised of more hydrophobic residues than catalytic sites.  We applied our findings to design mutations in selected target residues that deregulate the allosteric activity of fructose-1,6-bisphosphatase (FBPase).  Specifically, charged amino acids at less conserved positions were substituted with hydrophobic or neutral amino acids with similar sizes.  The engineered proteins successfully diminished the allosteric inhibition of E. coli FBPase without affecting its catalytic efficiency.  We expect that our method will aid the rational design of enzyme allosteric regulation strategies and facilitate the control of metabolic flux.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8XyCGaD871LVg90H21EOLACvtfcHk0lhg6gbElOPKow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOnsLvI&md5=7f600c82cb9ee88a8905a0a4a0b7a19e</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002612%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.-S.%26aulast%3DSeo%26aufirst%3DS.%2BW.%26aulast%3DJang%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DG.%2BY.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DRational%2520Engineering%2520of%2520Enzyme%2520Allosteric%2520Regulation%2520through%2520Sequence%2520Evolution%2520Analysis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Recent Computational Advances in the Identification of Allosteric Sites in Proteins</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1595</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25107670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1595-1600&author=S.+Luauthor=W.+Huangauthor=J.+Zhang&title=Recent+Computational+Advances+in+the+Identification+of+Allosteric+Sites+in+Proteins&doi=10.1016%2Fj.drudis.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Recent computational advances in the identification of allosteric sites in proteins</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1595-1600</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric modulators have the potential to fine-tune protein functional activity.  Therefore, the targeting of allosteric sites, as a strategy in drug design, is gaining increasing attention.  Currently, it is not trivial to find and characterize new allosteric sites by exptl. approaches.  Alternatively, computational approaches are useful in helping researchers analyze and select potential allosteric sites for drug discovery.  Here, we review state-of-the-art computational approaches directed at predicting putative allosteric sites in proteins, along with examples of successes in identifying allosteric sites utilizing these methods.  We also discuss the challenges in developing reliable methods for predicting allosteric sites and tactics to resolve demanding tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaspE0Qbe47Vg90H21EOLACvtfcHk0lhg6gbElOPKow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F&md5=fbaefacf51bee99d30fb882f0f04a47d</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRecent%2520Computational%2520Advances%2520in%2520the%2520Identification%2520of%2520Allosteric%2520Sites%2520in%2520Proteins%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1595%26epage%3D1600%26doi%3D10.1016%2Fj.drudis.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schueler-Furman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodak, S. J.</span></span> <span> </span><span class="NLM_article-title">Computational Approaches to Investigating Allostery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2016.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2016.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27607077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2016&pages=159-171&author=O.+Schueler-Furmanauthor=S.+J.+Wodak&title=Computational+Approaches+to+Investigating+Allostery&doi=10.1016%2Fj.sbi.2016.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to investigating allostery</span></div><div class="casAuthors">Schueler-Furman, Ora; Wodak, Shoshana J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">159-171</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Allosteric regulation plays a key role in many biol. processes, such as signal transduction, transcriptional regulation, and many more.  It is rooted in fundamental thermodn. and dynamic properties of macromol. systems that are still poorly understood and are moreover modulated by the cellular context.  Here we review the computational approaches used in the investigation of allosteric processes in protein systems.  We outline how the models of allostery have evolved from their initial formulation in the sixties to the current views, which more fully account for the roles of the thermodn. and dynamic properties of the system.  We then describe the major classes of computational approaches employed to elucidate the mechanisms of allostery, the insights they have provided, as well as their limitations.  We complement this anal. by highlighting the role of computational approaches in promising practical applications, such as the engineering of regulatory modules and identifying allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6oCnqZfK_x7Vg90H21EOLACvtfcHk0lgSpbLSGXQ0Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7jF&md5=7c28ee359b0701f757e7d9e042078522</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2016.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2016.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DSchueler-Furman%26aufirst%3DO.%26aulast%3DWodak%26aufirst%3DS.%2BJ.%26atitle%3DComputational%2520Approaches%2520to%2520Investigating%2520Allostery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D41%26spage%3D159%26epage%3D171%26doi%3D10.1016%2Fj.sbi.2016.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendergrass, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, A. W.</span></span> <span> </span><span class="NLM_article-title">Mining for Allosteric Information: Natural Mutations and Positional Sequence Conservation in Pyruvate Kinase</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1080/15216540500531705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1080%2F15216540500531705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16540430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=31-38&author=D.+C.+Pendergrassauthor=R.+Williamsauthor=J.+B.+Blairauthor=A.+W.+Fenton&title=Mining+for+Allosteric+Information%3A+Natural+Mutations+and+Positional+Sequence+Conservation+in+Pyruvate+Kinase&doi=10.1080%2F15216540500531705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Mining for allosteric information: Natural mutations and positional sequence conservation in pyruvate kinase</span></div><div class="casAuthors">Pendergrass, David; Williams, Rachel; Blair, James; Fenton, Aron</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-38</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Although the amino acid sequences and the structures of pyruvate kinase (PYK) isoenzymes are highly conserved, allosteric regulations differ.  This suggests that amino acids with low conservation play important roles in the allosteric mechanism.  The current work exploits a natural screen, the 122 point mutations identified in the human gene encoding the erythrocyte PYK isoenzyme and assocd. with non-spherocytic hemolytic anemia, to learn what amino acid positions in PYK may be important for allosteric regulations.  In addn. to the mutations, we consider the conservation of each amino acid position across 241 PYK sequences.  Three groups of residue positions have been created, those with: (1) no disease causing mutation identified; (2) a disease causing mutation identified and high conservation across isoenzymes; and (3) a disease causing mutation identified and low conservation.  Mutations at positions not identified in the natural screen are likely to be tolerated with minimal loss of function.  Mutations at highly conserved positions are more likely to disrupt properties common to all PYK isoenzymes (e.g., structure, catalysis).  Residues in the third group are likely to be involved in roles that are necessary for function but not common to all isoenzymes (e.g., allostery).  Many of the Group 3 residues are located in the C-domain and to a lesser extent the A domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY6rCK-AqkrLVg90H21EOLACvtfcHk0lgSpbLSGXQ0Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVaqtbk%253D&md5=8a2d6ae344a59ee8282945ac72a53185</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1080%2F15216540500531705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15216540500531705%26sid%3Dliteratum%253Aachs%26aulast%3DPendergrass%26aufirst%3DD.%2BC.%26aulast%3DWilliams%26aufirst%3DR.%26aulast%3DBlair%26aufirst%3DJ.%2BB.%26aulast%3DFenton%26aufirst%3DA.%2BW.%26atitle%3DMining%2520for%2520Allosteric%2520Information%253A%2520Natural%2520Mutations%2520and%2520Positional%2520Sequence%2520Conservation%2520in%2520Pyruvate%2520Kinase%26jtitle%3DIUBMB%2520Life%26date%3D2006%26volume%3D58%26spage%3D31%26epage%3D38%26doi%3D10.1080%2F15216540500531705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">Allostery at G Protein-Coupled Receptor Homo- and Heteromers: Uncharted Pharmacological Landscapes</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1124/pr.110.002667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1124%2Fpr.110.002667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21079041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktlCi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=701-725&author=N.+J.+Smithauthor=G.+Milligan&title=Allostery+at+G+Protein-Coupled+Receptor+Homo-+and+Heteromers%3A+Uncharted+Pharmacological+Landscapes&doi=10.1124%2Fpr.110.002667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery at G protein-coupled receptor homo- and heteromers: uncharted pharmacological landscapes</span></div><div class="casAuthors">Smith, Nicola J.; Milligan, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-725</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  For many years seven transmembrane domain G protein-coupled receptors (GPCRs) were thought to exist and function exclusively as monomeric units.  However, evidence both from native cells and heterologous expression systems has demonstrated that GPCRs can both traffic and signal within higher-order complexes.  As for other protein-protein interactions, conformational changes in one polypeptide, including those resulting from binding of pharmacol. ligands, have the capacity to alter the conformation and therefore the response of the interacting protein(s), a process known as allosterism.  For GPCRs, allosterism across homo- or heteromers, whether dimers or higher-order oligomers, represents an addnl. topog. landscape that must now be considered pharmacol.  Such effects may offer the opportunity for novel therapeutic approaches.  Allosterism at GPCR heteromers is particularly exciting in that it offers addnl. scope to provide receptor subtype selectivity and tissue specificity as well as fine-tuning of receptor signal strength.  Herein, we introduce the concept of allosterism at both GPCR homomers and heteromers and discuss the various questions that must be addressed before significant advances can be made in drug discovery at these GPCR complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2q5UArpDHnLVg90H21EOLACvtfcHk0lgSpbLSGXQ0Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktlCi&md5=0fef78a27d0b81a586b4bb58e6374515</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1124%2Fpr.110.002667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.110.002667%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DN.%2BJ.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DAllostery%2520at%2520G%2520Protein-Coupled%2520Receptor%2520Homo-%2520and%2520Heteromers%253A%2520Uncharted%2520Pharmacological%2520Landscapes%26jtitle%3DPharmacol.%2520Rev.%26date%3D2010%26volume%3D62%26spage%3D701%26epage%3D725%26doi%3D10.1124%2Fpr.110.002667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Chemical, Target, and Bioactive Properties of Allosteric Modulation</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003559</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24699297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGlsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=G.+J.+P.+van%0AWestenauthor=A.+Gaultonauthor=J.+P.+Overington&title=Chemical%2C+Target%2C+and+Bioactive+Properties+of+Allosteric+Modulation&doi=10.1371%2Fjournal.pcbi.1003559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical, target, and bioactive properties of allosteric modulation</span></div><div class="casAuthors">van Westen, Gerard J. P.; Gaulton, Anna; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003559/1-e1003559/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Allosteric modulators are ligands for proteins that exert their effects via a different binding site than the natural (orthosteric) ligand site and hence form a conceptually distinct class of ligands for a target of interest.  Here, the physicochem. and structural features of a large set of allosteric and non-allosteric ligands from the ChEMBL database of bioactive mols. are analyzed.  In general allosteric modulators are relatively smaller, more lipophilic and more rigid compds., though large differences exist between different targets and target classes.  Furthermore, there are differences in the distribution of targets that bind these allosteric modulators.  Allosteric modulators are over-represented in membrane receptors, ligand-gated ion channels and nuclear receptor targets, but are underrepresented in enzymes (primarily proteases and kinases).  Moreover, allosteric modulators tend to bind to their targets with a slightly lower potency (5.96 log units vs. 6.66 log units, p<0.01).  However, this lower abs. affinity is compensated by their lower mol. wt. and more lipophilic nature, leading to similar binding efficiency and surface efficiency indexes.  Subsequently a series of classifier models are trained, initially target class independent models followed by finer-grained target (architecture/functional class) based models using the target hierarchy of the ChEMBL database.  Applications of these insights include the selection of likely allosteric modulators from existing compd. collections, the design of novel chem. libraries biased towards allosteric regulators and the selection of targets potentially likely to yield allosteric modulators on screening.  All data sets used in the paper are available for download.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmlEn-nvE_H7Vg90H21EOLACvtfcHk0ljZX7RzsiHGeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGlsL7J&md5=bc2e7cca0c27b94f684224668f3d1f76</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003559%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChemical%252C%2520Target%252C%2520and%2520Bioactive%2520Properties%2520of%2520Allosteric%2520Modulation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">A Glimpse of Structural Biology through X-Ray Crystallography</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">995</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2014.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25416941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOgsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=995-1014&author=Y.+Shi&title=A+Glimpse+of+Structural+Biology+through+X-Ray+Crystallography&doi=10.1016%2Fj.cell.2014.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">A glimpse of structural biology through x-ray crystallography</span></div><div class="casAuthors">Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">995-1014</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Since the detn. of myoglobin structure in 1957, x-ray crystallog., as the anchoring tool of structural biol., has played an instrumental role in deciphering the secrets of life.  Knowledge gained through x-ray crystallog. has fundamentally advanced views on cellular processes and greatly facilitated the development of modern medicine.  Here, the author describes his personal understanding of the evolution of structural biol. through x-ray crystallog., using as examples mechanistic understanding of protein kinases and integral membrane proteins, and comments on the impact of technol. developments and the outlook for x-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrISBaUSogzUrVg90H21EOLACvtfcHk0ljZX7RzsiHGeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOgsrjL&md5=7c2837928184f9071ee618593fd7ceeb</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DA%2520Glimpse%2520of%2520Structural%2520Biology%2520through%2520X-Ray%2520Crystallography%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D995%26epage%3D1014%26doi%3D10.1016%2Fj.cell.2014.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug–Target Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=24-45&author=S.+Luauthor=J.+Zhang&title=Small+Molecule+Allosteric+Modulators+of+G-Protein-Coupled+Receptors%3A+Drug%E2%80%93Target+Interactions&doi=10.1021%2Facs.jmedchem.7b01844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-45</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are the largest class of signaling receptors that are most frequently targeted by therapeutic drugs.  Allosteric modulators bound to GPCRs at allosteric sites provide the potential for differential selectivity and improved safety compared with traditional orthosteric ligands.  The recent breakthroughs in GPCR structural biol. have made structures of GPCRs from classes A, B, C, and F complexed with small-mol. allosteric modulators available.  Knowledge of the detailed receptor-modulator interactions at the allosteric sites is useful for structure-based GPCR drug design of novel therapeutics.  This Perspective comprehensively summarizes the current status of structural complexes between GPCRs and their small-mol. allosteric modulators, particularly the key receptor-modulator interactions at the allosteric sites.  Then, the structural diversity of allosteric sites across four GPCR subfamilies is compared.  This study is expected to contribute to the design of GPCR allosteric drugs with an improved therapeutic action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVDf1DtcU6ZLVg90H21EOLACvtfcHk0ljZX7RzsiHGeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D&md5=7a41f4c00dadecd0661d135811644ad8</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01844%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DSmall%2520Molecule%2520Allosteric%2520Modulators%2520of%2520G-Protein-Coupled%2520Receptors%253A%2520Drug%25E2%2580%2593Target%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D24%26epage%3D45%26doi%3D10.1021%2Facs.jmedchem.7b01844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unverdorben, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweitzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, F.</span></span> <span> </span><span class="NLM_article-title">Near-Atomic Resolution Structural Model of the Yeast 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14870</span>– <span class="NLM_lpage">14875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1213333109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1213333109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22927375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVelsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14870-14875&author=F.+Beckauthor=P.+Unverdorbenauthor=S.+Bohnauthor=A.+Schweitzerauthor=G.+Pfeiferauthor=E.+Sakataauthor=S.+Nickellauthor=J.+M.+Plitzkoauthor=E.+Villaauthor=W.+Baumeisterauthor=F.+Forster&title=Near-Atomic+Resolution+Structural+Model+of+the+Yeast+26S+Proteasome&doi=10.1073%2Fpnas.1213333109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Near-atomic resolution structural model of the yeast 26S proteasome</span></div><div class="casAuthors">Beck, Florian; Unverdorben, Pia; Bohn, Stefan; Schweitzer, Andreas; Pfeifer, Guenter; Sakata, Eri; Nickell, Stephan; Plitzko, Jurgen M.; Villa, Elizabeth; Baumeister, Wolfgang; Foerster, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">14870-14875, S14870/1-S14870/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 26S proteasome operates at the executive end of the ubiquitin-proteasome pathway.  Here, we present a cryo-EM structure of the Saccharomyces cerevisiae 26S proteasome at a resoln. of 7.4 Å or 6.7 Å (Fourier-Shell Correlation of 0.5 or 0.3, resp.).  We used this map in conjunction with mol. dynamics-based flexible fitting to build a near-at. resoln. model of the holocomplex.  The quality of the map allowed us to assign α-helixes, the predominant secondary structure element of the regulatory particle subunits, throughout the entire map.  We were able to det. the architecture of the Rpn8/Rpn11 heterodimer, which had hitherto remained elusive.  The MPN domain of Rpn11 is positioned directly above the AAA-ATPase N-ring suggesting that Rpn11 deubiquitylates substrates immediately following commitment and prior to their unfolding by the AAA-ATPase module.  The MPN domain of Rpn11 dimerizes with that of Rpn8 and the C-termini of both subunits form long helixes, which are integral parts of a coiled-coil module.  Together with the C-terminal helixes of the 6 PCI-domain subunits they form a very large coiled-coil bundle, which appears to serve as a flexible anchoring device for all the lid subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWCbYbuDRJULVg90H21EOLACvtfcHk0lhuLRHIPEn02g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVelsrjL&md5=3b359d054944bb4dc14992bc72e5fb20</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1213333109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1213333109%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DF.%26aulast%3DUnverdorben%26aufirst%3DP.%26aulast%3DBohn%26aufirst%3DS.%26aulast%3DSchweitzer%26aufirst%3DA.%26aulast%3DPfeifer%26aufirst%3DG.%26aulast%3DSakata%26aufirst%3DE.%26aulast%3DNickell%26aufirst%3DS.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DForster%26aufirst%3DF.%26atitle%3DNear-Atomic%2520Resolution%2520Structural%2520Model%2520of%2520the%2520Yeast%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14870%26epage%3D14875%26doi%3D10.1073%2Fpnas.1213333109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Carbon Catabolite Repression of the Maltose Transporter Revealed by X-Ray Crystallography</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>499</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1038/nature12232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23770568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsV2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=364-368&author=S.+Chenauthor=M.+L.+Oldhamauthor=A.+L.+Davidsonauthor=J.+Chen&title=Carbon+Catabolite+Repression+of+the+Maltose+Transporter+Revealed+by+X-Ray+Crystallography&doi=10.1038%2Fnature12232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon catabolite repression of the maltose transporter revealed by X-ray crystallography</span></div><div class="casAuthors">Chen, Shanshuang; Oldham, Michael L.; Davidson, Amy L.; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">499</span>
        (<span class="NLM_cas:issue">7458</span>),
    <span class="NLM_cas:pages">364-368</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Efficient carbon utilization is crit. to the survival of microorganisms in competitive environments.  To optimize energy usage, bacteria have developed an integrated control system to preferentially uptake carbohydrates that support rapid growth.  The availability of a preferred carbon source, such as glucose, represses the synthesis and activities of proteins necessary for the transport and metab. of secondary carbon sources.  This regulatory phenomenon is defined as carbon catabolite repression.  In enteric bacteria, the key player of carbon catabolite repression is a component of the glucose-specific phosphotransferase system, enzyme IIA (EIIAGlc).  It is known that unphosphorylated EIIAGlc binds to and inhibits a variety of transporters when glucose is available.  However, understanding the underlying mol. mechanism has been hindered by the complete absence of structures for any EIIAGlc-transporter complexes.  Here we present the 3.9Å crystal structure of Escherichia coli EIIAGlc in complex with the maltose transporter, an ATP-binding cassette (ABC) transporter.  The structure shows that two EIIAGlc mols. bind to the cytoplasmic ATPase subunits, stabilizing the transporter in an inward-facing conformation and preventing the structural rearrangements necessary for ATP hydrolysis.  We also show that the half-maximal inhibitory concns. of the full-length EIIAGlc and an amino-terminal truncation mutant differ by 60-fold, consistent with the hypothesis that the amino-terminal region, disordered in the crystal structure, functions as a membrane anchor to increase the effective EIIAGlc concn. at the membrane.  Together these data suggest a model of how the central regulatory protein EIIAGlc allosterically inhibits maltose uptake in E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9xchOtuWIu7Vg90H21EOLACvtfcHk0lhuLRHIPEn02g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsV2it70%253D&md5=e0d4089ce27f3879e7454c0a3060b00a</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnature12232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12232%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DOldham%26aufirst%3DM.%2BL.%26aulast%3DDavidson%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DCarbon%2520Catabolite%2520Repression%2520of%2520the%2520Maltose%2520Transporter%2520Revealed%2520by%2520X-Ray%2520Crystallography%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D364%26epage%3D368%26doi%3D10.1038%2Fnature12232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of the Lsm Complex Bound to the 3′ End Sequence of U6 Small Nuclear RNA</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>506</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/nature12803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24240276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2msb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=506&publication_year=2014&pages=116-120&author=L.+Zhouauthor=J.+Hangauthor=Y.+Zhouauthor=R.+Wanauthor=G.+Luauthor=P.+Yinauthor=C.+Yanauthor=Y.+Shi&title=Crystal+Structures+of+the+Lsm+Complex+Bound+to+the+3%E2%80%B2+End+Sequence+of+U6+Small+Nuclear+RNA&doi=10.1038%2Fnature12803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the Lsm complex bound to the 3' end sequence of U6 small nuclear RNA</span></div><div class="casAuthors">Zhou, Lijun; Hang, Jing; Zhou, Yulin; Wan, Ruixue; Lu, Guifeng; Yin, Ping; Yan, Chuangye; Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">506</span>
        (<span class="NLM_cas:issue">7486</span>),
    <span class="NLM_cas:pages">116-120</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Splicing of precursor mRNA (pre-mRNA) in eukaryotic cells is carried out by the spliceosome, which consists of five small nuclear ribonucleoproteins (snRNPs) and a no. of accessory factors and enzymes.  Each snRNP contains a ring-shaped subcomplex of seven proteins and a specific RNA mol.  The U6 snRNP contains a unique heptameric Lsm protein complex, which specifically recognizes the U6 small nuclear RNA at its 3' end.  Here we report the crystal structures of the heptameric Lsm complex, both by itself and in complex with a 3' fragment of U6 snRNA, at 2.8 Å resoln.  Each of the seven Lsm proteins interacts with two neighboring Lsm components to form a doughnut-shaped assembly, with the order Lsm3-2-8-4-7-5-6.  The four uridine nucleotides at the 3' end of U6 snRNA are modularly recognized by Lsm3, Lsm2, Lsm8 and Lsm4, with the uracil base specificity conferred by a highly conserved asparagine residue.  The uracil base at the extreme 3' end is sandwiched by His 36 and Arg 69 from Lsm3, through π-π and cation-π interactions, resp.  The distinctive end-recognition of U6 snRNA by the Lsm complex contrasts with RNA binding by the Sm complex in the other snRNPs.  The structural features and assocd. biochem. analyses deepen mechanistic understanding of the U6 snRNP function in pre-mRNA splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5TH6BTgbiBLVg90H21EOLACvtfcHk0lhuLRHIPEn02g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2msb4%253D&md5=5115e59fc9d93d3b6f8ecf6804fd7ca6</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1038%2Fnature12803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12803%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DHang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWan%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DYin%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520Lsm%2520Complex%2520Bound%2520to%2520the%25203%25E2%2580%25B2%2520End%2520Sequence%2520of%2520U6%2520Small%2520Nuclear%2520RNA%26jtitle%3DNature%26date%3D2014%26volume%3D506%26spage%3D116%26epage%3D120%26doi%3D10.1038%2Fnature12803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stass, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilca, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huiskonen, J. T.</span></span> <span> </span><span class="NLM_article-title">Beyond Structures of Highly Symmetric Purified Viral Capsids by Cryo-EM</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2018.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30096461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVers77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=25-31&author=R.+Stassauthor=S.+L.+Ilcaauthor=J.+T.+Huiskonen&title=Beyond+Structures+of+Highly+Symmetric+Purified+Viral+Capsids+by+Cryo-EM&doi=10.1016%2Fj.sbi.2018.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond structures of highly symmetric purified viral capsids by cryo-EM</span></div><div class="casAuthors">Stass, Robert; Ilca, Serban L.; Huiskonen, Juha T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25-31</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cryogenic transmission electron microscopy (cryo-EM) is widely used to det. high-resoln. structures of sym. virus capsids.  The method holds promise for extending studies beyond purified capsids and their sym. protein shells.  The non-sym. genome component has been addressed in dsRNA cypoviruses and ssRNA bacteriophages Qβ and MS2.  The structure of human herpes simplex virus type 1 capsids has been detd. within intact virions to resolve capsid-tegument interactions.  Electron tomog. under cryogenic conditions (cryo-ET), has allowed resolving an early membrane fusion intermediate of Rift Valley fever virus.  Antibody-affinity based sample grids allow capturing of virions directly from cell cultures or even clin. samples.  These and other emerging methods will support studies to address viral entry, assembly and neutralization processes at increasingly high resolns. and native conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnUKUCNhkCbVg90H21EOLACvtfcHk0lij1P8mNqoljw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVers77F&md5=f63b9c7ce6fb2ea47cc902a61a0226b7</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DStass%26aufirst%3DR.%26aulast%3DIlca%26aufirst%3DS.%2BL.%26aulast%3DHuiskonen%26aufirst%3DJ.%2BT.%26atitle%3DBeyond%2520Structures%2520of%2520Highly%2520Symmetric%2520Purified%2520Viral%2520Capsids%2520by%2520Cryo-EM%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D25%26epage%3D31%26doi%3D10.1016%2Fj.sbi.2018.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pospich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raunser, S.</span></span> <span> </span><span class="NLM_article-title">Single Particle Cryo-EM — an Optimal Tool to Study Cytoskeletal Proteins</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2018.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30056307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKmtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=16-24&author=S.+Pospichauthor=S.+Raunser&title=Single+Particle+Cryo-EM+%E2%80%94+an+Optimal+Tool+to+Study+Cytoskeletal+Proteins&doi=10.1016%2Fj.sbi.2018.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Single particle cryo-EM - an optimal tool to study cytoskeletal proteins</span></div><div class="casAuthors">Pospich, Sabrina; Raunser, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16-24</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cytoskeletal proteins play essential roles in many cellular processes.  Knowledge of their structures is important to understand their function and regulation.  Since cytoskeletal polymers are difficult to crystallize, cryo-EM has been the predominant method of choice to study their structures.  Recent advances in the methodol. have enabled reconstructions at near-at. resoln.  In this review, we focus on novel insights gained from high-resoln. cryo-EM structures of cytoskeletal polymers.  These include eukaryotic proteins such as F-actin and microtubules as well as their prokaryotic homologues.  The unprecedented high-resolns. allow identifying small mols., including nucleotides and drugs, as well as subtle changes at interfaces that are key to complex processes, such as nucleotide hydrolysis in microtubules and actin filaments.  While major methodol. advances have already promoted the structural anal. of cytoskeletal polymers, there are still specific methodol. challenges to overcome and many scientific questions remain to be answered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCyCym1d9lO7Vg90H21EOLACvtfcHk0lij1P8mNqoljw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKmtLvF&md5=e2738a2df6e5cdfcba5215b61fb786e7</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPospich%26aufirst%3DS.%26aulast%3DRaunser%26aufirst%3DS.%26atitle%3DSingle%2520Particle%2520Cryo-EM%2520%25E2%2580%2594%2520an%2520Optimal%2520Tool%2520to%2520Study%2520Cytoskeletal%2520Proteins%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D16%26epage%3D24%26doi%3D10.1016%2Fj.sbi.2018.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fica, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span> <span> </span><span class="NLM_article-title">Cryo-Electron Microscopy Snapshots of the Spliceosome: Structural Insights into a Dynamic Ribonucleoprotein Machine</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnsmb.3463" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28981077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=791-799&author=S.+M.+Ficaauthor=K.+Nagai&title=Cryo-Electron+Microscopy+Snapshots+of+the+Spliceosome%3A+Structural+Insights+into+a+Dynamic+Ribonucleoprotein+Machine&doi=10.1038%2Fnsmb.3463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-electron microscopy snapshots of the spliceosome: structural insights into a dynamic ribonucleoprotein machine</span></div><div class="casAuthors">Fica, Sebastian M.; Nagai, Kiyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">791-799</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The spliceosome excises introns from pre-mRNAs using an RNA-based active site that is cradled by a dynamic protein scaffold.  A recent revolution in cryo-electron microscopy (cryo-EM) has led to near-at.-resoln. structures of key spliceosome complexes that provide insight into the mechanism of activation, splice site positioning, catalysis, protein rearrangements and ATPase-mediated dynamics of the active site.  The cryo-EM structures rationalize decades of observations from genetic and biochem. studies and provide a mol. framework for future functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphCYa1XIdT5LVg90H21EOLACvtfcHk0lij1P8mNqoljw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqs7jO&md5=8ba1a0da86dd368aedfc0336dadcfae9</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3463%26sid%3Dliteratum%253Aachs%26aulast%3DFica%26aufirst%3DS.%2BM.%26aulast%3DNagai%26aufirst%3DK.%26atitle%3DCryo-Electron%2520Microscopy%2520Snapshots%2520of%2520the%2520Spliceosome%253A%2520Structural%2520Insights%2520into%2520a%2520Dynamic%2520Ribonucleoprotein%2520Machine%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D791%26epage%3D799%26doi%3D10.1038%2Fnsmb.3463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciferri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rémigy, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM in Drug Discovery: Achievements, Limitations and Prospects</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd.2018.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29880918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyqurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=471-492&author=J.+P.+Renaudauthor=A.+Chariauthor=C.+Ciferriauthor=W.+T.+Liuauthor=H.+W.+R%C3%A9migyauthor=H.+Starkauthor=C.+Wiesmann&title=Cryo-EM+in+Drug+Discovery%3A+Achievements%2C+Limitations+and+Prospects&doi=10.1038%2Fnrd.2018.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM in drug discovery: achievements, limitations and prospects</span></div><div class="casAuthors">Renaud, Jean-Paul; Chari, Ashwin; Ciferri, Claudio; Liu, Wen-ti; Remigy, Herve-William; Stark, Holger; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">471-492</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Cryo-electron microscopy (cryo-EM) of non-cryst. single particles is a biophys. technique that can be used to det. the structure of biol. macromols. and assemblies.  Historically, its potential for application in drug discovery has been heavily limited by two issues: the min. size of the structures it can be used to study and the resoln. of the images.  However, recent technol. advances - including the development of direct electron detectors and more effective computational image anal. techniques - are revolutionizing the utility of cryo-EM, leading to a burst of high-resoln. structures of large macromol. assemblies.  These advances have raised hopes that single-particle cryo-EM might soon become an important tool for drug discovery, particularly if they could enable structural detn. for 'intractable' targets that are still not accessible to X-ray crystallog. anal.  This article describes the recent advances in the field and critically assesses their relevance for drug discovery as well as discussing at what stages of the drug discovery pipeline cryo-EM can be useful today and what to expect in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfGKiqaXhRQ7Vg90H21EOLACvtfcHk0ljuOG3lU4GZsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyqurnJ&md5=131daaa12f4c51c1e6478b55eb709157</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.77%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChari%26aufirst%3DA.%26aulast%3DCiferri%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DW.%2BT.%26aulast%3DR%25C3%25A9migy%26aufirst%3DH.%2BW.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DCryo-EM%2520in%2520Drug%2520Discovery%253A%2520Achievements%252C%2520Limitations%2520and%2520Prospects%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D471%26epage%3D492%26doi%3D10.1038%2Fnrd.2018.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grutsch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüschweiler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tollinger, M.</span></span> <span> </span><span class="NLM_article-title">NMR Methods to Study Dynamic Allostery</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1004620</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26964042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1WjurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1004620&author=S.+Grutschauthor=S.+Br%C3%BCschweilerauthor=M.+Tollinger&title=NMR+Methods+to+Study+Dynamic+Allostery&doi=10.1371%2Fjournal.pcbi.1004620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">NMR methods to study dynamic allostery</span></div><div class="casAuthors">Grutsch, Sarina; Brueschweiler, Sven; Tollinger, Martin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1004620/1-e1004620/20</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">NMR (NMR) spectroscopy provides a unique toolbox of exptl. probes for studying dynamic processes on a wide range of timescales, ranging from picoseconds to milliseconds and beyond.  Along with NMR hardware developments, recent methodol. advancements have enabled the characterization of allosteric proteins at unprecedented detail, revealing intriguing aspects of allosteric mechanisms and increasing the proportion of the conformational ensemble that can be obsd. by expt.  Here, we present an overview of NMR spectroscopic methods for characterizing equil. fluctuations in free and bound states of allosteric proteins that have been most influential in the field.  By combining NMR exptl. approaches with mol. simulations, atomistic level descriptions of the mechanisms by which allosteric phenomena take place are now within reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtDC4ZmjidbVg90H21EOLACvtfcHk0ljuOG3lU4GZsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1WjurrP&md5=3c3bb15547707024281794fe0fcd1928</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004620%26sid%3Dliteratum%253Aachs%26aulast%3DGrutsch%26aufirst%3DS.%26aulast%3DBr%25C3%25BCschweiler%26aufirst%3DS.%26aulast%3DTollinger%26aufirst%3DM.%26atitle%3DNMR%2520Methods%2520to%2520Study%2520Dynamic%2520Allostery%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3De1004620%26doi%3D10.1371%2Fjournal.pcbi.1004620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics and Allostery: An NMR View</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2010.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21109422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKhuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=62-67&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Protein+Dynamics+and+Allostery%3A+An+NMR+View&doi=10.1016%2Fj.sbi.2010.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and allostery: An NMR view</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allostery, the process by which distant sites within a protein system are energetically coupled, is an efficient and ubiquitous mechanism for activity regulation.  A purely mech. view of allostery invoking only structural changes has developed over the decades as the classical view of the phenomenon.  However, a fast growing list of examples illustrate the intimate link between internal motions over a wide range of time scales and function in protein-ligand interactions.  Proteins respond to perturbations by redistributing their motions and they use fluctuating conformational states for binding and conformational entropy as a carrier of allosteric energy to modulate assocn. with ligands.  In several cases allosteric interactions proceed with minimal or no structural changes.  We discuss emerging paradigms for the central role of protein dynamics in allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5G7kpkHz2M7Vg90H21EOLACvtfcHk0lgAJaLh1YRIug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKhuro%253D&md5=7ee2b2686e0c6a8ca4d337a33fd322ce</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DProtein%2520Dynamics%2520and%2520Allostery%253A%2520An%2520NMR%2520View%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.sbi.2010.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boulton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melacini, G.</span></span> <span> </span><span class="NLM_article-title">Advances in NMR Methods to Map Allosteric Sites: From Models to Translation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6267</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00718</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00718" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsFCrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6267-6304&author=S.+Boultonauthor=G.+Melacini&title=Advances+in+NMR+Methods+to+Map+Allosteric+Sites%3A+From+Models+to+Translation&doi=10.1021%2Facs.chemrev.5b00718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in NMR Methods To Map Allosteric Sites: From Models to Translation</span></div><div class="casAuthors">Boulton, Stephen; Melacini, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6267-6304</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The last five years have witnessed major developments in the understanding of the allosteric phenomenon, broadly defined as coupling between remote mol. sites.  Such advances have been driven not only by new theor. models and pharmacol. applications of allostery, but also by progress in the exptl. approaches designed to map allosteric sites and transitions.  Among these techniques, NMR spectroscopy has played a major role given its unique near-at. resoln. and sensitivity to the dynamics that underlie allosteric couplings.  Here, we highlight recent progress in the NMR methods tailored to investigate allostery with the goal of offering an overview of which NMR approaches are best suited for which allosterically relevant questions.  The picture of the allosteric "NMR toolbox" is provided starting from one of the simplest models of allostery (i.e., the four-state thermodn. cycle) and continuing to more complex multistate mechanisms.  We also review how such an "NMR toolbox" has assisted the elucidation of the allosteric mol. basis for disease-related mutations and the discovery of novel leads for allosteric drugs.  From this overview, it is clear that NMR plays a central role not only in exptl. validating transformative theories of allostery, but also in tapping the full translational potential of allosteric systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS33N8DJcfE7Vg90H21EOLACvtfcHk0lgAJaLh1YRIug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsFCrsr0%253D&md5=02fb528cd9cc3ec3ce82a6928f9170f5</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00718%26sid%3Dliteratum%253Aachs%26aulast%3DBoulton%26aufirst%3DS.%26aulast%3DMelacini%26aufirst%3DG.%26atitle%3DAdvances%2520in%2520NMR%2520Methods%2520to%2520Map%2520Allosteric%2520Sites%253A%2520From%2520Models%2520to%2520Translation%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6267%26epage%3D6304%26doi%3D10.1021%2Facs.chemrev.5b00718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lisi, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, J. P.</span></span> <span> </span><span class="NLM_article-title">Solution NMR Spectroscopy for the Study of Enzyme Allostery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6323</span>– <span class="NLM_lpage">6369</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00541</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00541" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6323-6369&author=G.+P.+Lisiauthor=J.+P.+Loria&title=Solution+NMR+Spectroscopy+for+the+Study+of+Enzyme+Allostery&doi=10.1021%2Facs.chemrev.5b00541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Solution NMR Spectroscopy for the Study of Enzyme Allostery</span></div><div class="casAuthors">Lisi, George P.; Loria, J. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6323-6369</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a ubiquitous biol. regulatory process in which distant binding sites within a protein or enzyme are functionally and thermodynamically coupled.  Allosteric interactions play essential roles in many enzymol. mechanisms, often facilitating formation of enzyme-substrate complexes and/or product release.  Thus, elucidating the forces that drive allostery is crit. to understanding the complex transformations of biomols.  Currently, a no. of models exist to describe allosteric behavior, taking into account energetics as well as conformational rearrangements and fluctuations.  In the following Review, we discuss the use of soln. NMR techniques designed to probe allosteric mechanisms in enzymes.  NMR spectroscopy is unequaled in its ability to detect structural and dynamical changes in biomols., and the case studies presented herein demonstrate the range of insights to be gained from this valuable method.  We also provide a detailed tech. discussion of several specialized NMR expts. that are ideally suited for the study of enzymic allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgJUg3gOrsHrVg90H21EOLACvtfcHk0lgAJaLh1YRIug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGqsrs%253D&md5=db1fd116e1222158a8e63e5246812680</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00541%26sid%3Dliteratum%253Aachs%26aulast%3DLisi%26aufirst%3DG.%2BP.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26atitle%3DSolution%2520NMR%2520Spectroscopy%2520for%2520the%2520Study%2520of%2520Enzyme%2520Allostery%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6323%26epage%3D6369%26doi%3D10.1021%2Facs.chemrev.5b00541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">NMR Studies of Large Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2667</span>– <span class="NLM_lpage">2676</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.jmb.2017.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28728982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2667-2676&author=Y.+Jiangauthor=C.+G.+Kalodimos&title=NMR+Studies+of+Large+Proteins&doi=10.1016%2Fj.jmb.2017.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">NMR Studies of Large Proteins</span></div><div class="casAuthors">Jiang, Yajun; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2667-2676</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent breakthroughs in isotope-labeling and pulse sequence techniques have enabled the NMR characterization of large protein systems with mol. masses of hundreds of kDa.  NMR studies of a great variety of large proteins provided unique insights into the binding, dynamic, and allosteric mechanisms.  Here the authors present a brief summary of these developments by highlighting a few cases that exemplify the uniqueness of NMR in providing at. resoln. information into key dynamic processes and structures of protein complexes with high degree of flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJF6WXjHLqtLVg90H21EOLACvtfcHk0lgAJaLh1YRIug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurfN&md5=7aa8e2e6f171902030910b0d54750ec4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DNMR%2520Studies%2520of%2520Large%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2667%26epage%3D2676%26doi%3D10.1016%2Fj.jmb.2017.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velyvis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span> <span> </span><span class="NLM_article-title">Measurement of Active Site Ionization Equilibria in the 670 kDa Proteasome Core Particle Using Methyl-TROSY NMR</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">9259</span>– <span class="NLM_lpage">9262</span>, <span class="refDoi"> DOI: 10.1021/ja403091c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja403091c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFKqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=9259-9262&author=A.+Velyvisauthor=L.+E.+Kay&title=Measurement+of+Active+Site+Ionization+Equilibria+in+the+670+kDa+Proteasome+Core+Particle+Using+Methyl-TROSY+NMR&doi=10.1021%2Fja403091c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of active site ionization equilibria in the 670 kDa proteasome core particle using methyl-TROSY NMR</span></div><div class="casAuthors">Velyvis, Algirdas; Kay, Lewis E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">9259-9262</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome core particle is a mol. machine that plays a central role in the regulation of cellular function through proteolysis, and it has emerged as a valuable drug target for certain classes of cancers.  Central to the development of new and potent pharmaceuticals is an understanding of the mechanism by which the proteasome cleaves substrates.  A no. of high-resoln. structures of the 20S proteasome with and without inhibitors have emerged that provide insight into the chem. of peptide bond cleavage and establish the role of Thr-1 Oγ-1 atom as the catalytic nucleophile.  The source of the base that accepts the Thr-1 Hγ-1 atom is less clear.  Here, using a highly deuterated sample of the proteasome labeled with 13CH3 at the Thr-γ positions, the pKa of the Thr-1 amino group was measured as 6.3 and hence deprotonated in the range of maximal enzyme activity.  This provided strong evidence that the terminal amino group of Thr-1 serves as the base in the 1st step of the peptide bond cleavage reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj_d1-nMFHEbVg90H21EOLACvtfcHk0lhSFYFJv4nIHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFKqurc%253D&md5=31f90f50707d0ed55da43a677083af52</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fja403091c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja403091c%26sid%3Dliteratum%253Aachs%26aulast%3DVelyvis%26aufirst%3DA.%26aulast%3DKay%26aufirst%3DL.%2BE.%26atitle%3DMeasurement%2520of%2520Active%2520Site%2520Ionization%2520Equilibria%2520in%2520the%2520670%2520kDa%2520Proteasome%2520Core%2520Particle%2520Using%2520Methyl-TROSY%2520NMR%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D9259%26epage%3D9262%26doi%3D10.1021%2Fja403091c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karagöz, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, A. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akoury, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biernat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morán Luengo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordhues, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veprintsev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandelkow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, S. G. D.</span></span> <span> </span><span class="NLM_article-title">Hsp90-Tau Complex Reveals Molecular Basis for Specificity in Chaperone Action</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">963</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.01.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2014.01.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24581495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvpsFagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=963-974&author=G.+E.+Karag%C3%B6zauthor=A.+M.+S.+Duarteauthor=E.+Akouryauthor=H.+Ippelauthor=J.+Biernatauthor=T.+Mor%C3%A1n+Luengoauthor=M.+Radliauthor=T.+Didenkoauthor=B.+A.+Nordhuesauthor=D.+B.+Veprintsevauthor=C.+A.+Dickeyauthor=E.+Mandelkowauthor=M.+Zweckstetterauthor=R.+Boelensauthor=T.+Madlauthor=S.+G.+D.+R%C3%BCdiger&title=Hsp90-Tau+Complex+Reveals+Molecular+Basis+for+Specificity+in+Chaperone+Action&doi=10.1016%2Fj.cell.2014.01.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Hsp90-Tau complex reveals molecular basis for specificity in chaperone action</span></div><div class="casAuthors">Karagoz G Elif; Duarte Afonso M S; Akoury Elias; Ippel Hans; Biernat Jacek; Moran Luengo Tania; Radli Martina; Didenko Tatiana; Nordhues Bryce A; Dickey Chad A; Veprintsev Dmitry B; Mandelkow Eckhard; Zweckstetter Markus; Boelens Rolf; Madl Tobias; Rudiger Stefan G D</div><div class="citationInfo"><span class="NLM_cas:title">Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">963-74</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein folding in the cell relies on the orchestrated action of conserved families of molecular chaperones, the Hsp70 and Hsp90 systems.  Hsp70 acts early and Hsp90 late in the folding path, yet the molecular basis of this timing is enigmatic, mainly because the substrate specificity of Hsp90 is poorly understood.  Here, we obtained a structural model of Hsp90 in complex with its natural disease-associated substrate, the intrinsically disordered Tau protein.  Hsp90 binds to a broad region in Tau that includes the aggregation-prone repeats.  Complementarily, a 106-ÅA-long substrate-binding interface in Hsp90 enables many low-affinity contacts.  This allows recognition of scattered hydrophobic residues in late folding intermediates that remain after early burial of the Hsp70 sites.  Our model resolves the paradox of how Hsp90 specifically selects for late folding intermediates but also for some intrinsically disordered proteins-through the eyes of Hsp90 they look the same.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJ1lLYwX_fzXl8icjszYl0fW6udTcc2eYVckFAZG0wILntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvpsFagtg%253D%253D&md5=ac8a4ccf143cd572c3e52517a9d4d2fc</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.01.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.01.037%26sid%3Dliteratum%253Aachs%26aulast%3DKarag%25C3%25B6z%26aufirst%3DG.%2BE.%26aulast%3DDuarte%26aufirst%3DA.%2BM.%2BS.%26aulast%3DAkoury%26aufirst%3DE.%26aulast%3DIppel%26aufirst%3DH.%26aulast%3DBiernat%26aufirst%3DJ.%26aulast%3DMor%25C3%25A1n%2BLuengo%26aufirst%3DT.%26aulast%3DRadli%26aufirst%3DM.%26aulast%3DDidenko%26aufirst%3DT.%26aulast%3DNordhues%26aufirst%3DB.%2BA.%26aulast%3DVeprintsev%26aufirst%3DD.%2BB.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26aulast%3DMandelkow%26aufirst%3DE.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DMadl%26aufirst%3DT.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DS.%2BG.%2BD.%26atitle%3DHsp90-Tau%2520Complex%2520Reveals%2520Molecular%2520Basis%2520for%2520Specificity%2520in%2520Chaperone%2520Action%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D963%26epage%3D974%26doi%3D10.1016%2Fj.cell.2014.01.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Libich, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawzi, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span> <span> </span><span class="NLM_article-title">Probing the Transient Dark State of Substrate Binding to GroEL by Relaxation-Based Solution NMR</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">11361</span>– <span class="NLM_lpage">11366</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305715110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1305715110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23798407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1GlsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=11361-11366&author=D.+S.+Libichauthor=N.+L.+Fawziauthor=J.+Yingauthor=G.+M.+Clore&title=Probing+the+Transient+Dark+State+of+Substrate+Binding+to+GroEL+by+Relaxation-Based+Solution+NMR&doi=10.1073%2Fpnas.1305715110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the transient dark state of substrate binding to GroEL by relaxation-based solution NMR</span></div><div class="casAuthors">Libich, David S.; Fawzi, Nicolas L.; Ying, Jinfa; Clore, G. Marius</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11361-11366, S11361/1-S11361/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The mechanism whereby the prototypical chaperonin GroEL performs work on substrate proteins has not yet been fully elucidated, hindered by lack of detailed structural and dynamic information on the bound substrate.  Previous studies produced conflicting reports on the state of GroEL-bound polypeptides, largely due to the transient and dynamic nature of these complexes.  Here, the authors present a unique approach, based on combined anal. of four complementary relaxation-based NMR expts., to probe directly the "dark" NMR-invisible state of the model, intrinsically disordered, polypeptide amyloid β (Aβ40) bound to GroEL.  The four NMR expts., lifetime line-broadening, dark-state exchange satn. transfer, relaxation dispersion, and small exchange-induced chem. shifts, are dependent in different ways on the overall exchange rates and populations of the free and bound states of the substrate, as well as on residue-specific dynamics and structure within the bound state as reported by transverse magnetization relaxation rates and backbone chem. shifts, resp.  Global fitting of all the NMR data shows that the complex is transient with a lifetime of <1 ms, that binding involves two predominantly hydrophobic segments corresponding to predicted GroEL consensus binding sequences, and that the structure of the bound polypeptide remains intrinsically and dynamically disordered with minimal changes in secondary structure propensity relative to the free state.  The authors' results establish a unique method to observe NMR-invisible dynamic states of GroEL-bound substrates and to describe at at. resoln. the events between substrate binding and encapsulation that are crucial for understanding the normal and stress-related metabolic function of chaperonins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhROaLvWZ3jLVg90H21EOLACvtfcHk0lhSFYFJv4nIHQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1GlsrfJ&md5=457756cba725fd0dc3d1edfe5a0d30af</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305715110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305715110%26sid%3Dliteratum%253Aachs%26aulast%3DLibich%26aufirst%3DD.%2BS.%26aulast%3DFawzi%26aufirst%3DN.%2BL.%26aulast%3DYing%26aufirst%3DJ.%26aulast%3DClore%26aufirst%3DG.%2BM.%26atitle%3DProbing%2520the%2520Transient%2520Dark%2520State%2520of%2520Substrate%2520Binding%2520to%2520GroEL%2520by%2520Relaxation-Based%2520Solution%2520NMR%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D11361%26epage%3D11366%26doi%3D10.1073%2Fpnas.1305715110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref111"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref111'); return false;" data-citation="" class="refNumLink">111</a></strong><div class="NLM_citation" id="cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) Inhibitors Allosterically Control GTP Affinity and Effector Interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+Inhibitors+Allosterically+Control+GTP+Affinity+and+Effector+Interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0lj2Cu9Kk0E_Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520Inhibitors%2520Allosterically%2520Control%2520GTP%2520Affinity%2520and%2520Effector%2520Interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref112"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref112'); return false;" data-citation="" class="refNumLink">112</a></strong><div class="NLM_citation" id="cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehlein, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannah, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. D.</span></span> <span> </span><span class="NLM_article-title">Probing Allosteric Binding Sites of the Maize Endosperm ADP-Glucose Pyrophosphorylase</span>. <i>Plant Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1104/pp.109.146928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1104%2Fpp.109.146928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19889875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2mur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2010&pages=85-95&author=S.+K.+Boehleinauthor=J.+R.+Shawauthor=L.+C.+Hannahauthor=J.+D.+Stewart&title=Probing+Allosteric+Binding+Sites+of+the+Maize+Endosperm+ADP-Glucose+Pyrophosphorylase&doi=10.1104%2Fpp.109.146928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Probing allosteric binding sites of the maize endosperm ADP-glucose pyrophosphorylase</span></div><div class="casAuthors">Boehlein, Susan K.; Shaw, Janine R.; Hannah, L. Curtis; Stewart, Jon D.</div><div class="citationInfo"><span class="NLM_cas:title">Plant Physiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-95</span>CODEN:
                <span class="NLM_cas:coden">PLPHAY</span>;
        ISSN:<span class="NLM_cas:issn">0032-0889</span>.
    
            (<span class="NLM_cas:orgname">American Society of Plant Biologists</span>)
        </div><div class="casAbstract">Maize (Zea mays) endosperm ADP-glucose pyrophosphorylase (AGPase) is a highly regulated enzyme that catalyzes the rate-limiting step in starch biosynthesis.  Although the structure of the heterotetrameric maize endosperm AGPase remains unsolved, structures of a nonnative, low-activity form of the potato tuber (Solanum tuberosum) AGPase (small subunit homotetramer) reported previously by others revealed that several sulfate ions bind to each enzyme.  These sites are also believed to interact with allosteric regulators such as inorg. phosphate and 3-phosphoglycerate (3-PGA).  Several arginine (Arg) side chains contact the bound sulfate ions in the potato structure and likely play important roles in allosteric effector binding.  Alanine-scanning mutagenesis was applied to the corresponding Arg residues in both the small and large subunits of maize endosperm AGPase to det. their roles in allosteric regulation and thermal stability.  Steady-state kinetic and regulatory parameters were measured for each mutant.  All of the Arg mutants examd.-in both the small and large subunits-bound 3-PGA more weakly than the wild type (A50 increased by 3.5- to 20-fold).  By contrast, the binding of two other maize AGPase allosteric activators (fructose-6-phosphate and glucose-6-phosphate) did not always mimic the changes obsd. for 3-PGA.  In fact, compared to 3-PGA, fructose-6-phosphate is a more efficient activator in two of the Arg mutants.  Phosphate binding was also affected by Arg substitutions.  The combined data support a model for the binding interactions assocd. with 3-PGA in which allosteric activators and inorg. phosphate compete directly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7x7HX27hNsLVg90H21EOLACvtfcHk0lj2Cu9Kk0E_Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2mur0%253D&md5=df848e68fbe6a173a7064b626f0e2f20</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1104%2Fpp.109.146928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1104%252Fpp.109.146928%26sid%3Dliteratum%253Aachs%26aulast%3DBoehlein%26aufirst%3DS.%2BK.%26aulast%3DShaw%26aufirst%3DJ.%2BR.%26aulast%3DHannah%26aufirst%3DL.%2BC.%26aulast%3DStewart%26aufirst%3DJ.%2BD.%26atitle%3DProbing%2520Allosteric%2520Binding%2520Sites%2520of%2520the%2520Maize%2520Endosperm%2520ADP-Glucose%2520Pyrophosphorylase%26jtitle%3DPlant%2520Physiol.%26date%3D2010%26volume%3D152%26spage%3D85%26epage%3D95%26doi%3D10.1104%2Fpp.109.146928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref113"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref113'); return false;" data-citation="" class="refNumLink">113</a></strong><div class="NLM_citation" id="cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Direct Binding Assay for the Detection of Type IV Allosteric Inhibitors of Abl</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">9138</span>– <span class="NLM_lpage">9141</span>, <span class="refDoi"> DOI: 10.1021/ja303858w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303858w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=9138-9141&author=R.+Schneiderauthor=C.+Beckerauthor=J.+R.+Simardauthor=M.+Getlikauthor=N.+Bohlkeauthor=P.+Janningauthor=D.+Rauh&title=Direct+Binding+Assay+for+the+Detection+of+Type+IV+Allosteric+Inhibitors+of+Abl&doi=10.1021%2Fja303858w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Binding Assay for the Detection of Type IV Allosteric Inhibitors of Abl</span></div><div class="casAuthors">Schneider, Ralf; Becker, Christian; Simard, Jeffrey R.; Getlik, Matthaeus; Bohlke, Nina; Janning, Petra; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9138-9141</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abelson (Abl) tyrosine kinase is an important cellular enzyme that is rendered constitutively active in the breakpoint cluster region (BCR)-Abl fusion protein, contributing to several forms of leukemia.  Although inhibiting BCR-Abl activity with imatinib shows great clin. success, many patients acquire secondary mutations that result in resistance to imatinib.  Second-generation inhibitors such as dasatinib and nilotinib can overcome the majority of these mutations but fail to treat patients with an esp. prevalent T315I mutation at the gatekeeper position of the kinase domain.  However, a combination of nilotinib with an allosteric type IV inhibitor was recently shown to overcome this clin. relevant point mutation.  In this study, we present the development of a direct binding assay that enables the straightforward detection of allosteric inhibitors which bind within the myristate pocket of Abl.  The assay is amenable to high-throughput screening and exclusively detects the binding of ligands to this unique allosteric site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHtVe0LFdqurVg90H21EOLACvtfcHk0lj2Cu9Kk0E_Dg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt74%253D&md5=95aa204ff4d761f2a2155112abe35799</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Fja303858w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303858w%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DBohlke%26aufirst%3DN.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DDirect%2520Binding%2520Assay%2520for%2520the%2520Detection%2520of%2520Type%2520IV%2520Allosteric%2520Inhibitors%2520of%2520Abl%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D9138%26epage%3D9141%26doi%3D10.1021%2Fja303858w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref114"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref114'); return false;" data-citation="" class="refNumLink">114</a></strong><div class="NLM_citation" id="cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beumer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Selective Detection of Allosteric Phosphatase Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6838</span>– <span class="NLM_lpage">6841</span>, <span class="refDoi"> DOI: 10.1021/ja4030484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4030484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlWhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6838-6841&author=R.+Schneiderauthor=C.+Beumerauthor=J.+R.+Simardauthor=C.+Gr%C3%BCtterauthor=D.+Rauh&title=Selective+Detection+of+Allosteric+Phosphatase+Inhibitors&doi=10.1021%2Fja4030484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Detection of Allosteric Phosphatase Inhibitors</span></div><div class="casAuthors">Schneider, Ralf; Beumer, Claudia; Simard, Jeffrey R.; Gruetter, Christian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6838-6841</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Normal cellular function, such as signal transduction, is largely controlled by the reversible phosphorylation of cellular proteins catalyzed by two major classes of enzymes, kinases and phosphatases.  A misbalance in this complex and dynamic interplay leads to a variety of severe diseases, such as cancer, inflammation, or autoimmune diseases.  This makes kinases as well as phosphatases equally attractive targets for therapeutic manipulation by small mols.  While the development of kinase inhibitors has resulted in several blockbuster drugs, such as imatinib, with remarkable success in the clinic and sales of many billions of U.S. dollars per yr, not a single phosphatase inhibitor has yet been approved for clin. use.  Similar to the kinase world, substrate-competitive phosphatase inhibitors have been developed but were not suitable for further development into clin. candidates due to their charge and limited selectivity.  Research efforts, therefore, have shifted to the exploitation of allosteric sites that can regulate phosphatase activity and may enable the discovery of novel modulators of phosphatase activity with much improved pharmacol. properties.  However, assay systems, which enable the straightforward discovery of these inhibitor types, are missing.  Here, we present a novel binding assay capable of detecting ligands of an allosteric pocket of the protein tyrosine phosphatase 1B.  This assay is suitable for high-throughput screening and selectively detects ligands which bind to this unique site with a clear discrimination from substrate-competitive ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMMpcm8RfGWrVg90H21EOLACvtfcHk0ljszaKiaaG01w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlWhtbk%253D&md5=49a980ad7f749718803f1afe1d8d1a04</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fja4030484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4030484%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBeumer%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DSelective%2520Detection%2520of%2520Allosteric%2520Phosphatase%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6838%26epage%3D6841%26doi%3D10.1021%2Fja4030484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref115"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref115'); return false;" data-citation="" class="refNumLink">115</a></strong><div class="NLM_citation" id="cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magnaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valsasina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avanzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasparri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cucchi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orrenius, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polucci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span> <span> </span><span class="NLM_article-title">Covalent and Allosteric Inhibitors of the ATPase VCP/p97 Induce Cancer Cell Death</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23892893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=548-559&author=P.+Magnaghiauthor=R.+D%E2%80%99Alessioauthor=B.+Valsasinaauthor=N.+Avanziauthor=S.+Rizziauthor=D.+Asaauthor=F.+Gasparriauthor=L.+Cozziauthor=U.+Cucchiauthor=C.+Orreniusauthor=P.+Polucciauthor=D.+Ballinariauthor=C.+Perreraauthor=A.+Leoneauthor=G.+Cerviauthor=E.+Casaleauthor=Y.+Xiaoauthor=C.+Wongauthor=D.+J.+Andersonauthor=A.+Galvaniauthor=D.+Donatiauthor=T.+O%E2%80%99Brienauthor=P.+K.+Jacksonauthor=A.+Isacchi&title=Covalent+and+Allosteric+Inhibitors+of+the+ATPase+VCP%2Fp97+Induce+Cancer+Cell+Death&doi=10.1038%2Fnchembio.1313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent and allosteric inhibitors of the ATPase VCP/p97 induce cancer cell death</span></div><div class="casAuthors">Magnaghi, Paola; D'Alessio, Roberto; Valsasina, Barbara; Avanzi, Nilla; Rizzi, Simona; Asa, Daniela; Gasparri, Fabio; Cozzi, Liviana; Cucchi, Ulisse; Orrenius, Christian; Polucci, Paolo; Ballinari, Dario; Perrera, Claudia; Leone, Antonella; Cervi, Giovanni; Casale, Elena; Xiao, Yang; Wong, Chihunt; Anderson, Daniel J.; Galvani, Arturo; Donati, Daniele; O'Brien, Tom; Jackson, Peter K.; Isacchi, Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">548-556</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">VCP (also known as p97 or Cdc48p in yeast) is an AAA+ ATPase regulating endoplasmic reticulum-assocd. degrdn.  After high-throughput screening, we developed compds. that inhibit VCP via different mechanisms, including covalent modification of an active site cysteine and a new allosteric mechanism.  Using photoaffinity labeling, structural anal. and mutagenesis, we mapped the binding site of allosteric inhibitors to a region spanning the D1 and D2 domains of adjacent protomers encompassing elements important for nucleotide-state sensing and ATP hydrolysis.  These compds. induced an increased affinity for nucleotides.  Interference with nucleotide turnover in individual subunits and distortion of interprotomer communication cooperated to impair VCP enzymic activity.  Chem. expansion of this allosteric class identified NMS-873, the most potent and specific VCP inhibitor described to date, which activated the unfolded protein response, interfered with autophagy and induced cancer cell death.  The consistent pattern of cancer cell killing by covalent and allosteric inhibitors provided crit. validation of VCP as a cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUmfk1JLyHdrVg90H21EOLACvtfcHk0ljszaKiaaG01w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOht7jK&md5=76bbca9d88073e2208fef5677a9223ba</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1313%26sid%3Dliteratum%253Aachs%26aulast%3DMagnaghi%26aufirst%3DP.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DR.%26aulast%3DValsasina%26aufirst%3DB.%26aulast%3DAvanzi%26aufirst%3DN.%26aulast%3DRizzi%26aufirst%3DS.%26aulast%3DAsa%26aufirst%3DD.%26aulast%3DGasparri%26aufirst%3DF.%26aulast%3DCozzi%26aufirst%3DL.%26aulast%3DCucchi%26aufirst%3DU.%26aulast%3DOrrenius%26aufirst%3DC.%26aulast%3DPolucci%26aufirst%3DP.%26aulast%3DBallinari%26aufirst%3DD.%26aulast%3DPerrera%26aufirst%3DC.%26aulast%3DLeone%26aufirst%3DA.%26aulast%3DCervi%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DIsacchi%26aufirst%3DA.%26atitle%3DCovalent%2520and%2520Allosteric%2520Inhibitors%2520of%2520the%2520ATPase%2520VCP%252Fp97%2520Induce%2520Cancer%2520Cell%2520Death%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D548%26epage%3D559%26doi%3D10.1038%2Fnchembio.1313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref116"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref116'); return false;" data-citation="" class="refNumLink">116</a></strong><div class="NLM_citation" id="cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Vera, I. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brust, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Tello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojetin, D. J.</span></span> <span> </span><span class="NLM_article-title">Synergistic Regulation of Coregulator/Nuclear Receptor Interaction by Ligand and DNA</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2017.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28890360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKqu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1506-1518&author=I.+M.+S.+de+Veraauthor=J.+Zhengauthor=S.+Novickauthor=J.+Shangauthor=T.+S.+Hughesauthor=R.+Brustauthor=P.+Munoz-Telloauthor=W.+J.+Gardnerauthor=D.+P.+Marcianoauthor=X.+Kongauthor=P.+R.+Griffinauthor=D.+J.+Kojetin&title=Synergistic+Regulation+of+Coregulator%2FNuclear+Receptor+Interaction+by+Ligand+and+DNA&doi=10.1016%2Fj.str.2017.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic Regulation of Coregulator/Nuclear Receptor Interaction by Ligand and DNA</span></div><div class="casAuthors">de Vera, Ian Mitchelle S.; Zheng, Jie; Novick, Scott; Shang, Jinsai; Hughes, Travis S.; Brust, Richard; Munoz-Tello, Paola; Gardner, William J., Jr.; Marciano, David P.; Kong, Xiangming; Griffin, Patrick R.; Kojetin, Douglas J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1506-1518.e4</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Nuclear receptor (NR) transcription factors bind various coreceptors, small-mol. ligands, DNA response element sequences, and transcriptional coregulator proteins to affect gene transcription.  Small-mol. ligands and DNA are known to influence receptor structure, coregulator protein interaction, and function; however, little is known on the mechanism of synergy between ligand and DNA.  Using quant. biochem., biophys., and soln. structural methods, including 13C-detected NMR and hydrogen/deuterium exchange (HDX) mass spectrometry, we show that ligand and DNA cooperatively recruit the intrinsically disordered steroid receptor coactivator-2 (SRC-2/TIF2/GRIP1/NCoA-2) receptor interaction domain to peroxisome proliferator-activated receptor gamma-retinoid X receptor alpha (PPARγ-RXRα) heterodimer and reveal the binding determinants of the complex.  Our data reveal a thermodn. mechanism by which DNA binding propagates a conformational change in PPARγ-RXRα, stabilizes the receptor ligand binding domain dimer interface, and impacts ligand potency and cooperativity in NR coactivator recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEHAepTsadQ7Vg90H21EOLACvtfcHk0lgGSv6NF27Jpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKqu7rI&md5=12580b50c1e459c4c67ca59312ed3bb9</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.07.019%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVera%26aufirst%3DI.%2BM.%2BS.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DNovick%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DHughes%26aufirst%3DT.%2BS.%26aulast%3DBrust%26aufirst%3DR.%26aulast%3DMunoz-Tello%26aufirst%3DP.%26aulast%3DGardner%26aufirst%3DW.%2BJ.%26aulast%3DMarciano%26aufirst%3DD.%2BP.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DKojetin%26aufirst%3DD.%2BJ.%26atitle%3DSynergistic%2520Regulation%2520of%2520Coregulator%252FNuclear%2520Receptor%2520Interaction%2520by%2520Ligand%2520and%2520DNA%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D1506%26epage%3D1518%26doi%3D10.1016%2Fj.str.2017.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref117"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref117'); return false;" data-citation="" class="refNumLink">117</a></strong><div class="NLM_citation" id="cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Prediction of Protein Allostery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2017.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2017.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29080471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GjsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=1-8&author=J.+G.+Greenerauthor=M.+J.+Sternberg&title=Structure-Based+Prediction+of+Protein+Allostery&doi=10.1016%2Fj.sbi.2017.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of protein allostery</span></div><div class="casAuthors">Greener, Joe G.; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Allostery is the functional change at one site on a protein caused by a change at a distant site.  In order for the benefits of allostery to be taken advantage of, both for basic understanding of proteins and to develop new classes of drugs, the structure-based prediction of allosteric binding sites, modulators and communication pathways is necessary.  Here we review the recently emerging field of allosteric prediction, focusing mainly on computational methods.  We also describe the search for cryptic binding pockets and attempts to design allostery into proteins.  The development and adoption of such methods is essential or the long-preached potential of allostery will remain elusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq4PyWF3NhFrVg90H21EOLACvtfcHk0lgGSv6NF27Jpw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GjsrrJ&md5=3dc8bf712c04354650571ca1b51849aa</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2017.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2017.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DStructure-Based%2520Prediction%2520of%2520Protein%2520Allostery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D50%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.sbi.2017.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref118"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref118'); return false;" data-citation="" class="refNumLink">118</a></strong><div class="NLM_citation" id="cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD: A Comprehensive Database of Allosteric Proteins and Modulators</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D663</span>– <span class="NLM_lpage">D669</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq1022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkq1022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21051350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D663-D669&author=Z.+Huangauthor=L.+Zhuauthor=Y.+Caoauthor=G.+Wuauthor=X.+Liuauthor=Y.+Chenauthor=Q.+Wangauthor=T.+Shiauthor=Y.+Zhaoauthor=Y.+Wangauthor=W.+Liauthor=Y.+Liauthor=H.+Chenauthor=G.+Chenauthor=J.+Zhang&title=ASD%3A+A+Comprehensive+Database+of+Allosteric+Proteins+and+Modulators&doi=10.1093%2Fnar%2Fgkq1022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">ASD: a comprehensive database of allosteric proteins and modulators</span></div><div class="casAuthors">Huang, Zhimin; Zhu, Liang; Cao, Yan; Wu, Geng; Liu, Xinyi; Chen, Yingyi; Wang, Qi; Shi, Ting; Zhao, Yaxue; Wang, Yuefei; Li, Weihua; Li, Yixue; Chen, Haifeng; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D663-D669</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct, rapid and efficient way of regulating protein function, ranging from the control of metabolic mechanisms to signal-transduction pathways.  However, an enormous amt. of unsystematic allosteric information has deterred scientists who could benefit from this field.  Here, we present the AlloSteric Database (ASD), the first online database that provides a central resource for the display, search and anal. of structure, function and related annotation for allosteric mols.  Currently, ASD contains 336 allosteric proteins from 101 species and 8095 modulators in three categories (activators, inhibitors and regulators).  Proteins are annotated with a detailed description of allostery, biol. process and related diseases, and modulators with binding affinity, physicochem. properties and therapeutic area.  Integrating the information of allosteric proteins in ASD should allow for the identification of specific allosteric sites of a given subtype among proteins of the same family that can potentially serve as ideal targets for exptl. validation.  In addn., modulators curated in ASD can be used to investigate potent allosteric targets for the query compd., and also help chemists to implement structure modifications for novel allosteric drug design.  Therefore, ASD could be a platform and a starting point for biologists and medicinal chemists for furthering allosteric research.  ASD is freely available at http://mdl.shsmu.edu.cn/ASD/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8WPj2tR4JbVg90H21EOLACvtfcHk0ljxX8_Q3bgJZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D&md5=28a63caabbc2942d98ca03ca1b4256ee</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq1022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq1022%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%253A%2520A%2520Comprehensive%2520Database%2520of%2520Allosteric%2520Proteins%2520and%2520Modulators%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD663%26epage%3DD669%26doi%3D10.1093%2Fnar%2Fgkq1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref119"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref119'); return false;" data-citation="" class="refNumLink">119</a></strong><div class="NLM_citation" id="cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD v2.0: Updated Content and Novel Features Focusing on Allosteric Regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D510</span>– <span class="NLM_lpage">D516</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt1247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24293647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D510-D516&author=Z.+Huangauthor=L.+Mouauthor=Q.+Shenauthor=S.+Luauthor=C.+Liauthor=X.+Liuauthor=G.+Wangauthor=S.+Liauthor=L.+Gengauthor=Y.+Liuauthor=J.+Wuauthor=G.+Chenauthor=J.+Zhang&title=ASD+v2.0%3A+Updated+Content+and+Novel+Features+Focusing+on+Allosteric+Regulation&doi=10.1093%2Fnar%2Fgkt1247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">ASD v2.0: updated content and novel features focusing on allosteric regulation</span></div><div class="casAuthors">Huang, Zhimin; Mou, Linkai; Shen, Qiancheng; Lu, Shaoyong; Li, Chuangang; Liu, Xinyi; Wang, Guanqiao; Li, Shuai; Geng, Lv; Liu, Yaqin; Wu, Jiawei; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D510-D516</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct and efficient way for regulation of biol. macromol. function and is induced by the binding of a ligand at an allosteric site topog. distinct from the orthosteric site.  AlloSteric Database (ASD, http://mdl.shsmu.edu.cn/ASD) has been developed to provide comprehensive information on allostery.  Owing to the inherent high receptor selectivity and lower target-based toxicity, allosteric regulation is expected to assume a more prominent role in drug discovery and bioengineering, leading to the rapid growth of allosteric findings.  In this updated version, ASD v2.0 has expanded to 1286 allosteric proteins, 565 allosteric diseases and 22 008 allosteric modulators.  A total of 907 allosteric site-modulator structural complexes and >200 structural pairs of orthosteric/allosteric sites in the allosteric proteins were constructed for researchers to develop allosteric site and pathway tools in response to community demands.  Up-to-date allosteric pathways were manually curated in the updated version.  In addn., both the front-end and the back-end of ASD have been redesigned and enhanced to allow more efficient access.  Taken together, these updates are useful for facilitating the investigation of allosteric mechanisms, allosteric target identification and allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMY2SmS8wLI7Vg90H21EOLACvtfcHk0ljxX8_Q3bgJZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gh&md5=2cdd4c0b36e00381b2050890f5b5295e</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1247%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DMou%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%2520v2.0%253A%2520Updated%2520Content%2520and%2520Novel%2520Features%2520Focusing%2520on%2520Allosteric%2520Regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD510%26epage%3DD516%26doi%3D10.1093%2Fnar%2Fgkt1247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref120"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref120'); return false;" data-citation="" class="refNumLink">120</a></strong><div class="NLM_citation" id="cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASBench: Benchmarking Sets for Allosteric Discovery</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2598</span>– <span class="NLM_lpage">2600</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtv169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25810427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Gju7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=2598-2600&author=W.+Huangauthor=G.+Wangauthor=Q.+Shenauthor=X.+Liuauthor=S.+Luauthor=L.+Gengauthor=Z.+Huangauthor=J.+Zhang&title=ASBench%3A+Benchmarking+Sets+for+Allosteric+Discovery&doi=10.1093%2Fbioinformatics%2Fbtv169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">ASBench: benchmarking sets for allosteric discovery</span></div><div class="casAuthors">Huang, Wenkang; Wang, Guanqiao; Shen, Qiancheng; Liu, Xinyi; Lu, Shaoyong; Geng, Lv; Huang, Zhimin; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2598-2600</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: Allostery allows for the fine-tuning of protein function.  Targeting allosteric sites is gaining increasing recognition as a novel strategy in drug design.  The key challenge in the discovery of allosteric sites has strongly motivated the development of computational methods and thus high-quality, publicly accessible std. data have become indispensable.  Here, we report benchmarking data for exptl. detd. allosteric sites through a complex process, including a 'Core set' with 235 unique allosteric sites and a 'Core-Diversity set' with 147 structurally diverse allosteric sites.  These benchmarking sets can be exploited to develop efficient computational methods to predict unknown allosteric sites in proteins and reveal unique allosteric ligand-protein interactions to guide allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw-vf0LmMbprVg90H21EOLACvtfcHk0lhEZ_VDueTHkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Gju7zK&md5=12dce810b56a48c987dfb8743be95548</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv169%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASBench%253A%2520Benchmarking%2520Sets%2520for%2520Allosteric%2520Discovery%26jtitle%3DBioinformatics%26date%3D2015%26volume%3D31%26spage%3D2598%26epage%3D2600%26doi%3D10.1093%2Fbioinformatics%2Fbtv169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref121"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref121'); return false;" data-citation="" class="refNumLink">121</a></strong><div class="NLM_citation" id="cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosite: A Method for Predicting Allosteric Sites</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2359</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtt399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23842804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmsLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2357-2359&author=W.+Huangauthor=S.+Luauthor=Z.+Huangauthor=X.+Liuauthor=L.+Mouauthor=Y.+Luoauthor=Y.+Zhaoauthor=Y.+Liuauthor=Z.+Chenauthor=T.+Houauthor=J.+Zhang&title=Allosite%3A+A+Method+for+Predicting+Allosteric+Sites&doi=10.1093%2Fbioinformatics%2Fbtt399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Allosite: a method for predicting allosteric sites</span></div><div class="casAuthors">Huang, Wenkang; Lu, Shaoyong; Huang, Zhimin; Liu, Xinyi; Mou, Linkai; Luo, Yu; Zhao, Yanlong; Liu, Yaqin; Chen, Zhongjie; Hou, Tingjun; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2357-2359</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The use of allosteric modulators as preferred therapeutic agents against classic orthosteric ligands has colossal advantages, including higher specificity, fewer side effects and lower toxicity.  Therefore, the computational prediction of allosteric sites in proteins is receiving increased attention in the field of drug discovery.  Allosite is a newly developed automatic tool for the prediction of allosteric sites in proteins of interest and is now available through a web server.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBwW1VByREtbVg90H21EOLACvtfcHk0lhEZ_VDueTHkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmsLfL&md5=24b4661f3e4cee3d5cab6cc8c86bc607</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt399%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMou%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosite%253A%2520A%2520Method%2520for%2520Predicting%2520Allosteric%2520Sites%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D2357%26epage%3D2359%26doi%3D10.1093%2Fbioinformatics%2Fbtt399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref122"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref122'); return false;" data-citation="" class="refNumLink">122</a></strong><div class="NLM_citation" id="cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Improved Method for the Identification and Validation of Allosteric Sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2358</span>– <span class="NLM_lpage">2363</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGlu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2358-2363&author=K.+Songauthor=X.+Liuauthor=W.+Huangauthor=S.+Luauthor=Q.+Shenauthor=L.+Zhangauthor=J.+Zhang&title=Improved+Method+for+the+Identification+and+Validation+of+Allosteric+Sites&doi=10.1021%2Facs.jcim.7b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Method for the Identification and Validation of Allosteric Sites</span></div><div class="casAuthors">Song, Kun; Liu, Xinyi; Huang, Wenkang; Lu, Shaoyong; Shen, Qiancheng; Zhang, Lu; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2358-2363</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric regulation induced by modulators binding to different, often distant, allosteric sites allows for exquisite control of protein functional activity.  The structural diversity of allosteric sites endows allosteric modulators with high selectivity and low toxicity.  Targeting allosteric sites, a novel tactic in drug discovery, has garnered much attention in the scientific community, and the identification of allosteric sites has become an important component of the development of allosteric drugs.  Here we present AllositePro, a method which predicts allosteric sites in proteins by combining pocket features with perturbation anal.  The performance of AllositePro is superior to that of the other currently available methods.  Using AllositePro, we predicted a novel allosteric site in cyclin-dependent kinase 2 (CDK2) and validated it by site-directed mutagenesis assay.  Thus, the AllositePro method provides an effective way to identify allosteric sites and could be a useful strategy for allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq23d-Zg0_4G7Vg90H21EOLACvtfcHk0lhEZ_VDueTHkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGlu7vL&md5=fbc362245d0bfb34686e0bc2fb38203b</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00014%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DImproved%2520Method%2520for%2520the%2520Identification%2520and%2520Validation%2520of%2520Allosteric%2520Sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2358%26epage%3D2363%26doi%3D10.1021%2Facs.jcim.7b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref123"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref123'); return false;" data-citation="" class="refNumLink">123</a></strong><div class="NLM_citation" id="cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akbar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helms, V.</span></span> <span> </span><span class="NLM_article-title">ALLO: A Tool to Discriminate and Prioritize Allosteric Pockets</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">853</span>, <span class="refDoi"> DOI: 10.1111/cbdd.13161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1111%2Fcbdd.13161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29250934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2018&pages=845-853&author=R.+Akbarauthor=V.+Helms&title=ALLO%3A+A+Tool+to+Discriminate+and+Prioritize+Allosteric+Pockets&doi=10.1111%2Fcbdd.13161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">ALLO: A tool to discriminate and prioritize allosteric pockets</span></div><div class="casAuthors">Akbar, Rahmad; Helms, Volkhard</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">845-853</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Allosteric proteins make up a substantial proportion of human drug targets.  Thus, rational design of small mol. binders that target these proteins requires the identification of putative allosteric pockets and an understanding of their potential activity.  Here, we characterized allosteric pockets using a set of physicochem. descriptors and compared them to pockets that are found on the surface of a protein.  Further, we trained predictive models capable of discriminating allosteric pockets from orthosteric pockets and models capable of prioritizing allosteric pockets in a set of pockets found on a given protein.  Such models might be useful for identifying novel allosteric sites and in turn, potentially new allosteric drug targets.  Datasets along with a Python program encapsulating the predictive models are available at .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPa9JRzKq5VbVg90H21EOLACvtfcHk0lhEZ_VDueTHkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOntrg%253D&md5=a06e7d563aa27493e1de845acf6c2581</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.13161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.13161%26sid%3Dliteratum%253Aachs%26aulast%3DAkbar%26aufirst%3DR.%26aulast%3DHelms%26aufirst%3DV.%26atitle%3DALLO%253A%2520A%2520Tool%2520to%2520Discriminate%2520and%2520Prioritize%2520Allosteric%2520Pockets%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2018%26volume%3D91%26spage%3D845%26epage%3D853%26doi%3D10.1111%2Fcbdd.13161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref124"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref124'); return false;" data-citation="" class="refNumLink">124</a></strong><div class="NLM_citation" id="cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavridis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B. O.</span></span> <span> </span><span class="NLM_article-title">A Random Forest Model for Predicting Allosteric and Functional Sites on Proteins</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1002/minf.201500108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fminf.201500108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27491922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=125-135&author=A.+S.+Y.+Chenauthor=N.+J.+Westwoodauthor=P.+Brearauthor=G.+W.+Rogersauthor=L.+Mavridisauthor=J.+B.+O.+Mitchell&title=A+Random+Forest+Model+for+Predicting+Allosteric+and+Functional+Sites+on+Proteins&doi=10.1002%2Fminf.201500108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">A Random Forest Model for Predicting Allosteric and Functional Sites on Proteins</span></div><div class="casAuthors">Chen, Ava S.-Y.; Westwood, Nicholas J.; Brear, Paul; Rogers, Graeme W.; Mavridis, Lazaros; Mitchell, John B. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">125-135</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We created a computational method to identify allosteric sites using a machine learning method trained and tested on protein structures contg. bound ligand mols.  The Random Forest machine learning approach was adopted to build our three-way predictive model.  Based on descriptors collated for each ligand and binding site, the classification model allows us to assign protein cavities as allosteric, regular or orthosteric, and hence to identify allosteric sites. 43 structural descriptors per complex were derived and were used to characterize individual protein-ligand binding sites belonging to the three classes, allosteric, regular and orthosteric.  We carried out a sep. validation on a further unseen set of protein structures contg. the ligand 2-(N-cyclohexylamino) ethane sulfonic acid (CHES).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryYcl0PpE-r7Vg90H21EOLACvtfcHk0liOecW-hRzvhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSqsbs%253D&md5=aa74e9ba8d0bbe06a2b61fbd31b0854c</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1002%2Fminf.201500108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201500108%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DA.%2BS.%2BY.%26aulast%3DWestwood%26aufirst%3DN.%2BJ.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DG.%2BW.%26aulast%3DMavridis%26aufirst%3DL.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%2BO.%26atitle%3DA%2520Random%2520Forest%2520Model%2520for%2520Predicting%2520Allosteric%2520and%2520Functional%2520Sites%2520on%2520Proteins%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D125%26epage%3D135%26doi%3D10.1002%2Fminf.201500108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref125"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref125'); return false;" data-citation="" class="refNumLink">125</a></strong><div class="NLM_citation" id="cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smock, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivoire, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swain, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierasch, L. M.</span></span> <span> </span><span class="NLM_article-title">An Interdomain Sector Mediating Allostery in Hsp70 Molecular Chaperones</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/msb.2010.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fmsb.2010.65" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1-10&author=R.+G.+Smockauthor=O.+Rivoireauthor=W.+P.+Russauthor=J.+F.+Swainauthor=S.+Leiblerauthor=R.+Ranganathanauthor=L.+M.+Gierasch&title=An+Interdomain+Sector+Mediating+Allostery+in+Hsp70+Molecular+Chaperones&doi=10.1038%2Fmsb.2010.65"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2010.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2010.65%26sid%3Dliteratum%253Aachs%26aulast%3DSmock%26aufirst%3DR.%2BG.%26aulast%3DRivoire%26aufirst%3DO.%26aulast%3DRuss%26aufirst%3DW.%2BP.%26aulast%3DSwain%26aufirst%3DJ.%2BF.%26aulast%3DLeibler%26aufirst%3DS.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DGierasch%26aufirst%3DL.%2BM.%26atitle%3DAn%2520Interdomain%2520Sector%2520Mediating%2520Allostery%2520in%2520Hsp70%2520Molecular%2520Chaperones%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fmsb.2010.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref126"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref126'); return false;" data-citation="" class="refNumLink">126</a></strong><div class="NLM_citation" id="cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">AlloPred: Prediction of Allosteric Pockets on Proteins Using Normal Mode Perturbation Analysis</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">335</span>, <span class="refDoi"> DOI: 10.1186/s12859-015-0771-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1186%2Fs12859-015-0771-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26493317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC28zlt12ksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=335&author=J.+G.+Greenerauthor=M.+J.+Sternberg&title=AlloPred%3A+Prediction+of+Allosteric+Pockets+on+Proteins+Using+Normal+Mode+Perturbation+Analysis&doi=10.1186%2Fs12859-015-0771-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">AlloPred: prediction of allosteric pockets on proteins using normal mode perturbation analysis</span></div><div class="casAuthors">Greener Joe G; Sternberg Michael J E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Despite being hugely important in biological processes, allostery is poorly understood and no universal mechanism has been discovered.  Allosteric drugs are a largely unexplored prospect with many potential advantages over orthosteric drugs.  Computational methods to predict allosteric sites on proteins are needed to aid the discovery of allosteric drugs, as well as to advance our fundamental understanding of allostery.  RESULTS:  AlloPred, a novel method to predict allosteric pockets on proteins, was developed.  AlloPred uses perturbation of normal modes alongside pocket descriptors in a machine learning approach that ranks the pockets on a protein.  AlloPred ranked an allosteric pocket top for 23 out of 40 known allosteric proteins, showing comparable and complementary performance to two existing methods.  In 28 of 40 cases an allosteric pocket was ranked first or second.  The AlloPred web server, freely available at http://www.sbg.bio.ic.ac.uk/allopred/home, allows visualisation and analysis of predictions.  The source code and dataset information are also available from this site.  CONCLUSIONS:  Perturbation of normal modes can enhance our ability to predict allosteric sites on proteins.  Computational methods such as AlloPred assist drug discovery efforts by suggesting sites on proteins for further experimental study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQpnbMrYjzf52LFM_lbR1VmfW6udTcc2eaVxTQzdKbnW7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zlt12ksw%253D%253D&md5=2de7386cee597dc9300f1fd25c3b8631</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1186%2Fs12859-015-0771-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-015-0771-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DAlloPred%253A%2520Prediction%2520of%2520Allosteric%2520Pockets%2520on%2520Proteins%2520Using%2520Normal%2520Mode%2520Perturbation%2520Analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2015%26volume%3D16%26spage%3D335%26doi%3D10.1186%2Fs12859-015-0771-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref127"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref127'); return false;" data-citation="" class="refNumLink">127</a></strong><div class="NLM_citation" id="cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjkovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daura, X.</span></span> <span> </span><span class="NLM_article-title">PARS: A Web Server for the Prediction of Protein Allosteric and Regulatory Sites</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1314</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btu002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtu002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24413526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvFequr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=1314-1315&author=A.+Panjkovichauthor=X.+Daura&title=PARS%3A+A+Web+Server+for+the+Prediction+of+Protein+Allosteric+and+Regulatory+Sites&doi=10.1093%2Fbioinformatics%2Fbtu002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">PARS: a web server for the prediction of Protein Allosteric and Regulatory Sites</span></div><div class="casAuthors">Panjkovich, Alejandro; Daura, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1314-1315</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The regulation of protein activity is a key aspect of life at the mol. level.  Unveiling its details is thus crucial to understanding signalling and metabolic pathways.  The most common and powerful mechanism of protein-function regulation is allostery, which has been increasingly calling the attention of medicinal chemists due to its potential for the discovery of novel therapeutics.  In this context, PARS is a simple and fast method that queries protein dynamics and structural conservation to identify pockets on a protein structure that may exert a regulatory effect on the binding of a small-mol. ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNQj4nH_68ybVg90H21EOLACvtfcHk0liOecW-hRzvhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvFequr8%253D&md5=ae1c218abc25988f911af6eace2d38a1</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtu002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtu002%26sid%3Dliteratum%253Aachs%26aulast%3DPanjkovich%26aufirst%3DA.%26aulast%3DDaura%26aufirst%3DX.%26atitle%3DPARS%253A%2520A%2520Web%2520Server%2520for%2520the%2520Prediction%2520of%2520Protein%2520Allosteric%2520and%2520Regulatory%2520Sites%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26spage%3D1314%26epage%3D1315%26doi%3D10.1093%2Fbioinformatics%2Fbtu002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref128"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref128'); return false;" data-citation="" class="refNumLink">128</a></strong><div class="NLM_citation" id="cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Identifying Allosteric Binding Sites in Proteins with a Two-State Go? Model for Novel Allosteric Effector Discovery</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2962</span>– <span class="NLM_lpage">2971</span>, <span class="refDoi"> DOI: 10.1021/ct300395h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300395h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFemtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2962-2971&author=Y.+Qiauthor=Q.+Wangauthor=B.+Tangauthor=L.+Lai&title=Identifying+Allosteric+Binding+Sites+in+Proteins+with+a+Two-State+Go%3F+Model+for+Novel+Allosteric+Effector+Discovery&doi=10.1021%2Fct300395h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Allosteric Binding Sites in Proteins with a Two-State G‾o Model for Novel Allosteric Effector Discovery</span></div><div class="casAuthors">Qi, Yifei; Wang, Qian; Tang, Bo; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2962-2971</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a common mechanism of controlling many biol. processes such as enzyme catalysis, signal transduction, and metabolic regulation.  The use of allostery to regulate protein activity is an important and promising strategy in drug discovery and biol. network regulation.  In order to modulate protein activity by allostery, predictive methods need to be developed to discover allosteric binding sites.  In the present study, we developed a new approach to identify allosteric sites in proteins based on the coarse-grained two-state G‾o model.  Starting from the concept that allostery is a conformation population shift process, we first constructed an ensemble of two functional states of a protein and tuned the energy landscape to bias one state.  We then added perturbations to a binding site and monitored the population distribution of the new ensemble.  If population redistribution occurred, then the binding perturbed site was predicted as a potential allosteric site.  Our approach successfully identified all the known allosteric sites in a set of test proteins.  Several new allosteric sites in the test proteins were also predicted.  By use of one of the new allosteric sites predicted from Escherichia coli phosphoglycerate dehydrogenase (PGDH), novel allosteric regulating mols. were screened by mol. docking and enzymic assay.  Three novel allosteric inhibitors were discovered and their binding modes were confirmed by mutation expts. and competitive assay.  The IC50 of the strongest inhibitor discovered was 21 μM, which is comparable to that of the native allosteric inhibitor L-serine.  The novel allosteric site discovered in PGDH is L-serine-independent, and inhibitors targeting this site can be used as novel regulators of the E. coli serine synthesis pathway.  Our approach for allosteric site prediction is generally applicable and the predicted sites can be used in discovering novel allosteric regulating mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZLFKLyehCX7Vg90H21EOLACvtfcHk0ljjG7MCdpGNEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFemtrY%253D&md5=6c95574165b32eba99bd8432118e5fa9</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fct300395h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300395h%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DTang%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DIdentifying%2520Allosteric%2520Binding%2520Sites%2520in%2520Proteins%2520with%2520a%2520Two-State%2520Go%253F%2520Model%2520for%2520Novel%2520Allosteric%2520Effector%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2962%26epage%3D2971%26doi%3D10.1021%2Fct300395h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref129"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref129'); return false;" data-citation="" class="refNumLink">129</a></strong><div class="NLM_citation" id="cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2017.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28190781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=546-558&author=J.+G.+Greenerauthor=I.+Filippisauthor=M.+J.+E.+Sternberg&title=Predicting+Protein+Dynamics+and+Allostery+Using+Multi-Protein+Atomic+Distance+Constraints&doi=10.1016%2Fj.str.2017.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span></div><div class="casAuthors">Greener, Joe G.; Filippis, Ioannis; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">546-558</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The related concepts of protein dynamics, conformational ensembles and allostery are often difficult to study with mol. dynamics (MD) due to the timescales involved.  We present ExProSE (Exploration of Protein Structural Ensembles), a distance geometry-based method that generates an ensemble of protein structures from two input structures.  ExProSE provides a unified framework for the exploration of protein structure and dynamics in a fast and accessible way.  Using a dataset of apo/holo pairs it is shown that existing coarse-grained methods often cannot span large conformational changes.  For T4-lysozyme, ExProSE is able to generate ensembles that are more native-like than tCONCOORD and NMSim, and comparable with targeted MD.  By adding addnl. constraints representing potential modulators, ExProSE can predict allosteric sites.  ExProSE ranks an allosteric pocket first or second for 27 out of 58 allosteric proteins, which is similar and complementary to existing methods.  The ExProSE source code is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF34-q1BCDrVg90H21EOLACvtfcHk0ljjG7MCdpGNEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D&md5=2d2ea675b3b29ff52f9915ddecabe032</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DFilippis%26aufirst%3DI.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DPredicting%2520Protein%2520Dynamics%2520and%2520Allostery%2520Using%2520Multi-Protein%2520Atomic%2520Distance%2520Constraints%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D546%26epage%3D558%26doi%3D10.1016%2Fj.str.2017.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref130"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref130'); return false;" data-citation="" class="refNumLink">130</a></strong><div class="NLM_citation" id="cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of Molecular Dynamics and Related Methods in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+Molecular+Dynamics+and+Related+Methods+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0ljjG7MCdpGNEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520Molecular%2520Dynamics%2520and%2520Related%2520Methods%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref131"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref131'); return false;" data-citation="" class="refNumLink">131</a></strong><div class="NLM_citation" id="cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinkam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Model of Allostery Predicts Coupling between Distant Sites</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">4875</span>– <span class="NLM_lpage">4880</span>, <span class="refDoi"> DOI: 10.1073/pnas.1116274109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1116274109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22403063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1agtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=4875-4880&author=P.+Weinkamauthor=J.+Ponsauthor=A.+Sali&title=Structure-Based+Model+of+Allostery+Predicts+Coupling+between+Distant+Sites&doi=10.1073%2Fpnas.1116274109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based model of allostery predicts coupling between distant sites</span></div><div class="casAuthors">Weinkam, Patrick; Pons, Jaume; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4875-4880, S4875/1-S4875/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Allostery is a phenomenon that couples effector ligand binding at an allosteric site to a structural and/or dynamic change at a distant regulated site.  To study an allosteric transition, we vary the size of the allosteric site and its interactions to construct a series of energy landscapes with pronounced min. corresponding to both the effector bound and unbound crystal structures.  We use mol. dynamics to sample these landscapes.  The degree of perturbation by the effector, modeled by the size of the allosteric site, provides an order parameter for allostery that allows us to det. how microscopic motions give rise to commonly discussed macroscopic mechanisms: (i) induced fit, (ii) population shift, and (iii) driven.  These mechanisms involve decreasing structural differences between the effector bound and unbound populations.  A metric (ligand-induced cooperativity) can measure how cooperatively a given regulated site responds to effector binding and therefore what kind of allosteric mechanism is involved.  We apply the model to 3 proteins with exptl. characterized transitions: (i) calmodulin-GFP Ca2+ sensor protein, (ii) maltose binding protein, and (iii) CSL transcription factor.  Remarkably, the model is able to reproduce allosteric motion and predict coupling in a manner consistent with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom_5ajotvZS7Vg90H21EOLACvtfcHk0li39Ee_unvjPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1agtL8%253D&md5=da9e811dffb0d3492f9f2ac8c2f7281b</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116274109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116274109%26sid%3Dliteratum%253Aachs%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-Based%2520Model%2520of%2520Allostery%2520Predicts%2520Coupling%2520between%2520Distant%2520Sites%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D4875%26epage%3D4880%26doi%3D10.1073%2Fpnas.1116274109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref132"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref132'); return false;" data-citation="" class="refNumLink">132</a></strong><div class="NLM_citation" id="cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">Reversing Allosteric Communication: From Detecting Allosteric Sites to Inducing and Tuning Targeted Allosteric Response</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">e1006228</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1006228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1006228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29912863" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&author=W.-V.+Teeauthor=E.+Guarneraauthor=I.+N.+Berezovsky&title=Reversing+Allosteric+Communication%3A+From+Detecting+Allosteric+Sites+to+Inducing+and+Tuning+Targeted+Allosteric+Response&doi=10.1371%2Fjournal.pcbi.1006228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1006228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1006228%26sid%3Dliteratum%253Aachs%26aulast%3DTee%26aufirst%3DW.-V.%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DReversing%2520Allosteric%2520Communication%253A%2520From%2520Detecting%2520Allosteric%2520Sites%2520to%2520Inducing%2520and%2520Tuning%2520Targeted%2520Allosteric%2520Response%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2018%26volume%3D14%26doi%3D10.1371%2Fjournal.pcbi.1006228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref133"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref133'); return false;" data-citation="" class="refNumLink">133</a></strong><div class="NLM_citation" id="cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Motions of Allosteric and Orthosteric Ligand-Binding Sites in Proteins Are Highly Correlated</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1733</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVemtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1725-1733&author=X.+Maauthor=H.+Mengauthor=L.+Lai&title=Motions+of+Allosteric+and+Orthosteric+Ligand-Binding+Sites+in+Proteins+Are+Highly+Correlated&doi=10.1021%2Facs.jcim.6b00039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Motions of Allosteric and Orthosteric Ligand-Binding Sites in Proteins are Highly Correlated</span></div><div class="casAuthors">Ma, Xiaomin; Meng, Hu; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1725-1733</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is the phenomenon in which a ligand binding at one site affects other sites in the same macromol.  Allostery has important roles in many biol. processes.  Theor., all non-fibrous proteins are potentially allosteric.  However, few allosteric proteins have been validated, and the identification of novel allosteric sites remains a challenge.  The motion of residues and subunits underlies protein function; therefore, the authors hypothesized that the motions of allosteric and orthosteric sites are correlated.  The authors used a dataset of 24 known allosteric sites from 23 monomer proteins to calc. the correlations between potential ligand-binding sites and corresponding orthosteric sites using a Gaussian network model (GNM).  Most of the known allosteric site motions showed high correlations with corresponding orthosteric site motions, whereas other surface cavities did not.  These high correlations were robust when using different structural data for the same protein, such as structures for the apo state and the orthosteric effector-binding state, whereas the contributions of different frequency modes to motion correlations depend on the given protein.  The high correlations between allosteric and orthosteric site motions were also obsd. in oligomeric allosteric proteins.  The authors applied motion correlation anal. to predict potential allosteric sites in the 23 monomer proteins, and some of these predictions were in good agreement with published exptl. data.  The authors also performed motion correlation anal. to identify a novel allosteric site in 15-lipoxygenase (an enzyme in the arachidonic acid metabolic network) using recently reported activating compds.  The authors' anal. correctly identified this novel allosteric site along with two other sites that are currently under exptl. study.  The authors' study demonstrates that the motions of allosteric sites are highly correlated with the motions of orthosteric sites.  The authors' correlation anal. method provides new tools for predicting potential allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbJEt1Kw-KErVg90H21EOLACvtfcHk0li39Ee_unvjPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVemtLrM&md5=c868c49d9026b2573554bb19f219a93d</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00039%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DMotions%2520of%2520Allosteric%2520and%2520Orthosteric%2520Ligand-Binding%2520Sites%2520in%2520Proteins%2520Are%2520Highly%2520Correlated%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1725%26epage%3D1733%26doi%3D10.1021%2Facs.jcim.6b00039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref134"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref134'); return false;" data-citation="" class="refNumLink">134</a></strong><div class="NLM_citation" id="cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armutlulu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haliloglu, T.</span></span> <span> </span><span class="NLM_article-title">MCPath: Monte Carlo Path Generation Approach to Predict Likely Allosteric Pathways and Functional Residues</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">W249</span>– <span class="NLM_lpage">W255</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23742907" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W249-W255&author=C.+Kayaauthor=A.+Armutluluauthor=S.+Ekesanauthor=T.+Haliloglu&title=MCPath%3A+Monte+Carlo+Path+Generation+Approach+to+Predict+Likely+Allosteric+Pathways+and+Functional+Residues&doi=10.1093%2Fnar%2Fgkt284"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt284%26sid%3Dliteratum%253Aachs%26aulast%3DKaya%26aufirst%3DC.%26aulast%3DArmutlulu%26aufirst%3DA.%26aulast%3DEkesan%26aufirst%3DS.%26aulast%3DHaliloglu%26aufirst%3DT.%26atitle%3DMCPath%253A%2520Monte%2520Carlo%2520Path%2520Generation%2520Approach%2520to%2520Predict%2520Likely%2520Allosteric%2520Pathways%2520and%2520Functional%2520Residues%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW249%26epage%3DW255%26doi%3D10.1093%2Fnar%2Fgkt284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref135"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref135'); return false;" data-citation="" class="refNumLink">135</a></strong><div class="NLM_citation" id="cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goncearenco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitternacht, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">SPACER: Server for Predicting Allosteric Communication and Effects of Regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23737445" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=266-272&author=A.+Goncearencoauthor=S.+Mitternachtauthor=T.+Yongauthor=B.+Eisenhaberauthor=F.+Eisenhaberauthor=I.+N.+Berezovsky&title=SPACER%3A+Server+for+Predicting+Allosteric+Communication+and+Effects+of+Regulation&doi=10.1093%2Fnar%2Fgkt460"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt460%26sid%3Dliteratum%253Aachs%26aulast%3DGoncearenco%26aufirst%3DA.%26aulast%3DMitternacht%26aufirst%3DS.%26aulast%3DYong%26aufirst%3DT.%26aulast%3DEisenhaber%26aufirst%3DB.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DSPACER%253A%2520Server%2520for%2520Predicting%2520Allosteric%2520Communication%2520and%2520Effects%2520of%2520Regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D266%26epage%3D272%26doi%3D10.1093%2Fnar%2Fgkt460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref136"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref136'); return false;" data-citation="" class="refNumLink">136</a></strong><div class="NLM_citation" id="cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. W.</span></span> <span> </span><span class="NLM_article-title">DynOmics: Dynamics of Structural Proteome and Beyond</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">W374</span>– <span class="NLM_lpage">W380</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkx385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28472330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1amt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W374-W380&author=H.+Liauthor=Y.+Y.+Changauthor=J.+Y.+Leeauthor=I.+Baharauthor=L.+W.+Yang&title=DynOmics%3A+Dynamics+of+Structural+Proteome+and+Beyond&doi=10.1093%2Fnar%2Fgkx385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">DynOmics: dynamics of structural proteome and beyond</span></div><div class="casAuthors">Li, Hongchun; Chang, Yuan-Yu; Lee, Ji Young; Bahar, Ivet; Yang, Lee-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W374-W380</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  DynOmics is a portal developed to leverage rapidly growing structural proteomics data by efficiently and accurately evaluating the dynamics of structurally resolved systems, from individual mols. to large complexes and assemblies, in the context of their physiol. environment.  At the core of the portal is a newly developed server, ENM 1.0, which permits users to efficiently generate information on the collective dynamics of any structure in PDB format, user-uploaded or database-retrieved.  ENM 1.0 integrates two widely used elastic network models (ENMs) -the Gaussian Network Model (GNM) and the Anisotropic Network Model (ANM), extended to take account of mol. environment.  It enables users to assess potentially functional sites, signal transduction or allosteric communication mechanisms, and protein-protein and protein-DNA interaction poses, in addn. to delivering ensembles of accessible conformers reconstructed at. details based on the global modes of motions predicted by the ANM.  The 'environment' is defined in a flexible manner, from lipid bilayer and crystal contacts, to substrate or ligands bound to a protein, or surrounding subunits in a multimeric structure or assembly.  User-friendly interactive features permit users to easily visualize how the environment alter the intrinsic dynamics of the query systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGAbiwacCgkbVg90H21EOLACvtfcHk0lgdk4ncifjgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1amt7s%253D&md5=6b4171f6201ed5ff69175a32f0daf69a</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx385%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%2BY.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DBahar%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DL.%2BW.%26atitle%3DDynOmics%253A%2520Dynamics%2520of%2520Structural%2520Proteome%2520and%2520Beyond%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DW374%26epage%3DW380%26doi%3D10.1093%2Fnar%2Fgkx385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref137"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref137'); return false;" data-citation="" class="refNumLink">137</a></strong><div class="NLM_citation" id="cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">AlloSigMA: Allosteric Signaling and Mutation Analysis Server</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3996</span>– <span class="NLM_lpage">3998</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btx430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtx430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29106449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3996-3998&author=E.+Guarneraauthor=Z.+W.+Tanauthor=Z.+Zhengauthor=I.+N.+Berezovsky&title=AlloSigMA%3A+Allosteric+Signaling+and+Mutation+Analysis+Server&doi=10.1093%2Fbioinformatics%2Fbtx430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">AlloSigMA: Allosteric signaling and mutation analysis server</span></div><div class="casAuthors">Guarnera, Enrico; Tan, Zhen Wah; Zheng, Zejun; Berezovsky, Igor N.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3996-3998</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4811</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Allostery is an omnipresent mechanism of the function modulation in proteins via either effector binding or mutations in the exosites.  Despite the growing no. of online servers and databases devoted to prediction/classification of allosteric sites and their characteristics, there is a lack of resources for an efficient and quick estn. of the causality and energetics of allosteric communication.  Results: The AlloSigMA server implements a unique approach on the basis of the recently introduced structure-based statistical mech. models of allosteric signaling.  It provides an interactive framework for estg. the allosteric free energy as a result of the ligand(s) binding, mutation(s) and their combinations.  Latent regulatory exosites and allosteric effect of mutations can be detected and explored, facilitating the research efforts in protein engineering and allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMXne1_sWhrLVg90H21EOLACvtfcHk0lgdk4ncifjgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsbnO&md5=2a6a59793713dfd13708bab94ca51d4b</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtx430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtx430%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DTan%26aufirst%3DZ.%2BW.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DAlloSigMA%253A%2520Allosteric%2520Signaling%2520and%2520Mutation%2520Analysis%2520Server%26jtitle%3DBioinformatics%26date%3D2017%26volume%3D33%26spage%3D3996%26epage%3D3998%26doi%3D10.1093%2Fbioinformatics%2Fbtx430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref138"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref138'); return false;" data-citation="" class="refNumLink">138</a></strong><div class="NLM_citation" id="cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">AlloMAPS: Allosteric Mutation Analysis and Polymorphism of Signaling Database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D265</span>– <span class="NLM_lpage">D270</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky1028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30365033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1CgtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D265-D270&author=Z.+W.+Tanauthor=W.-V.+Teeauthor=E.+Guarneraauthor=L.+Boothauthor=I.+N.+Berezovsky&title=AlloMAPS%3A+Allosteric+Mutation+Analysis+and+Polymorphism+of+Signaling+Database&doi=10.1093%2Fnar%2Fgky1028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        cas_cit138R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1028%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DZ.%2BW.%26aulast%3DTee%26aufirst%3DW.-V.%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DBooth%26aufirst%3DL.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DAlloMAPS%253A%2520Allosteric%2520Mutation%2520Analysis%2520and%2520Polymorphism%2520of%2520Signaling%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD265%26epage%3DD270%26doi%3D10.1093%2Fnar%2Fgky1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref139"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref139'); return false;" data-citation="" class="refNumLink">139</a></strong><div class="NLM_citation" id="cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Proteome-Scale Investigation of Protein Allosteric Regulation Perturbed by Somatic Mutations in 7,000 Cancer Genomes</span>. <i>Am. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.ajhg.2016.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ajhg.2016.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27939638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOjsr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2017&pages=5-20&author=Q.+Shenauthor=F.+Chengauthor=H.+Songauthor=W.+Luauthor=J.+Zhaoauthor=X.+Anauthor=M.+Liuauthor=G.+Chenauthor=Z.+Zhaoauthor=J.+Zhang&title=Proteome-Scale+Investigation+of+Protein+Allosteric+Regulation+Perturbed+by+Somatic+Mutations+in+7%2C000+Cancer+Genomes&doi=10.1016%2Fj.ajhg.2016.09.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Scale Investigation of Protein Allosteric Regulation Perturbed by Somatic Mutations in 7,000 Cancer Genomes</span></div><div class="casAuthors">Shen, Qiancheng; Cheng, Feixiong; Song, Huili; Lu, Weiqiang; Zhao, Junfei; An, Xiaoli; Liu, Mingyao; Chen, Guoqiang; Zhao, Zhongming; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-20</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The allosteric regulation triggering the protein's functional activity via conformational changes is an intrinsic function of protein under many physiol. and pathol. conditions, including cancer.  Identification of the biol. effects of specific somatic variants on allosteric proteins and the phenotypes that they alter during tumor initiation and progression is a central challenge for cancer genomes in the post-genomic era.  Here, we mapped more than 47,000 somatic missense mutations obsd. in approx. 7,000 tumor-normal matched samples across 33 cancer types into protein allosteric sites to prioritize the mutated allosteric proteins and we tested our prediction in cancer cell lines.  We found that the deleterious mutations identified in cancer genomes were more significantly enriched at protein allosteric sites than tolerated mutations, suggesting a crit. role for protein allosteric variants in cancer.  Next, we developed a statistical approach, namely AlloDriver, and further identified 15 potential mutated allosteric proteins during pan-cancer and individual cancer-type analyses.  More importantly, we exptl. confirmed that p.Pro360Ala on PDE10A played a potential oncogenic role in mediating tumorigenesis in non-small cell lung cancer (NSCLC).  In summary, these findings shed light on the role of allosteric regulation during tumorigenesis and provide a useful tool for the timely development of targeted cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDSiE8UeSCcbVg90H21EOLACvtfcHk0lgdk4ncifjgYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOjsr3I&md5=8124a5ae3b2da1cfff8d7e90170666f2</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2016.09.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2016.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DProteome-Scale%2520Investigation%2520of%2520Protein%2520Allosteric%2520Regulation%2520Perturbed%2520by%2520Somatic%2520Mutations%2520in%25207%252C000%2520Cancer%2520Genomes%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2017%26volume%3D100%26spage%3D5%26epage%3D20%26doi%3D10.1016%2Fj.ajhg.2016.09.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref140"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref140'); return false;" data-citation="" class="refNumLink">140</a></strong><div class="NLM_citation" id="cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, R. W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span> <span> </span><span class="NLM_article-title">Identifying Allosteric Hotspots with Dynamics: Application to Inter- and Intra-Species Conservation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">826</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27066750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslels7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=826-837&author=D.+Clarkeauthor=A.+Sethiauthor=S.+Liauthor=S.+Kumarauthor=R.+W.+F.+Changauthor=J.+Chenauthor=M.+Gerstein&title=Identifying+Allosteric+Hotspots+with+Dynamics%3A+Application+to+Inter-+and+Intra-Species+Conservation&doi=10.1016%2Fj.str.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Allosteric Hotspots with Dynamics: Application to Inter- and Intra-species Conservation</span></div><div class="casAuthors">Clarke, Declan; Sethi, Anurag; Li, Shantao; Kumar, Sushant; Chang, Richard W. F.; Chen, Jieming; Gerstein, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">826-837</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The rapidly growing vol. of data being produced by next-generation sequencing initiatives is enabling more in-depth analyses of conservation than previously possible.  Deep sequencing is uncovering disease loci and regions under selective constraint, despite the fact that intuitive biophys. reasons for such constraint are sometimes absent.  Allostery may often provide the missing explanatory link.  We use models of protein conformational change to identify allosteric residues by finding essential surface pockets and information-flow bottlenecks, and we develop a software tool that enables users to perform this anal. on their own proteins of interest.  Though fundamentally 3D-structural in nature, our anal. is computationally fast, thereby allowing us to run it across the PDB and to evaluate general properties of predicted allosteric residues.  We find that these tend to be conserved over diverse evolutionary time scales.  Finally, we highlight examples of allosteric residues that help explain poorly understood disease-assocd. variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRj593ixDCd7Vg90H21EOLACvtfcHk0lhRkQL_DZ4-cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslels7o%253D&md5=a630e9fdd6d6ee889ffef8fe54d8ec02</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DSethi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DR.%2BW.%2BF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26atitle%3DIdentifying%2520Allosteric%2520Hotspots%2520with%2520Dynamics%253A%2520Application%2520to%2520Inter-%2520and%2520Intra-Species%2520Conservation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D826%26epage%3D837%26doi%3D10.1016%2Fj.str.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref141"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref141'); return false;" data-citation="" class="refNumLink">141</a></strong><div class="NLM_citation" id="cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Alloscore: A Method for Predicting Allosteric Ligand–protein Interactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1574</span>– <span class="NLM_lpage">1576</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btw036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtw036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26803160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWku7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1574-1576&author=S.+Liauthor=Q.+Shenauthor=M.+Suauthor=X.+Liuauthor=S.+Luauthor=Z.+Chenauthor=R.+Wangauthor=J.+Zhang&title=Alloscore%3A+A+Method+for+Predicting+Allosteric+Ligand%E2%80%93protein+Interactions&doi=10.1093%2Fbioinformatics%2Fbtw036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Alloscore: a method for predicting allosteric ligand-protein interactions</span></div><div class="casAuthors">Li, Shuai; Shen, Qiancheng; Su, Minyi; Liu, Xinyi; Lu, Shaoyong; Chen, Zhongjie; Wang, Renxiao; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1574-1576</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allosteric ligands have increasingly gained attention as potential therapeutic agents due to their higher target selectivity and lower toxicity compared with classic orthosteric ligands.  Despite the great interest in the development of allosteric drugs as a new tactic in drug discovery, the understanding of the ligand-protein interactions underlying allosteric binding represents a key challenge.  Herein, we introduce Alloscore, a web server that predicts the binding affinities of allosteric ligand-protein interactions.  This method exhibits prominent performance in describing allosteric binding and could be useful in allosteric virtual screening and the structural optimization of allosteric agonists/antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvCDDNX-XiybVg90H21EOLACvtfcHk0lhRkQL_DZ4-cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWku7rE&md5=9f5fef0132358b98e1a4ce13673dd15a</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtw036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtw036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAlloscore%253A%2520A%2520Method%2520for%2520Predicting%2520Allosteric%2520Ligand%25E2%2580%2593protein%2520Interactions%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D1574%26epage%3D1576%26doi%3D10.1093%2Fbioinformatics%2Fbtw036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref142"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref142'); return false;" data-citation="" class="refNumLink">142</a></strong><div class="NLM_citation" id="cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span> <span> </span><span class="NLM_article-title">CavityPlus: A Web Server for Protein Cavity Detection with Pharmacophore Modelling, Allosteric Site Identification and Covalent Ligand Binding Ability Prediction</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W374</span>– <span class="NLM_lpage">W379</span>, <span class="refDoi"> DOI: 10.1093/nar/gky380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29750256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W374-W379&author=Y.+Xuauthor=S.+Wangauthor=Q.+Huauthor=S.+Gaoauthor=X.+Maauthor=W.+Zhangauthor=Y.+Shenauthor=F.+Chenauthor=L.+Laiauthor=J.+Pei&title=CavityPlus%3A+A+Web+Server+for+Protein+Cavity+Detection+with+Pharmacophore+Modelling%2C+Allosteric+Site+Identification+and+Covalent+Ligand+Binding+Ability+Prediction&doi=10.1093%2Fnar%2Fgky380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">CavityPlus: a web server for protein cavity detection with pharmacophore modelling, allosteric site identification and covalent ligand binding ability prediction</span></div><div class="casAuthors">Xu, Youjun; Wang, Shiwei; Hu, Qiwan; Gao, Shuaishi; Ma, Xiaomin; Zhang, Weilin; Shen, Yihang; Chen, Fangjin; Lai, Luhua; Pei, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W374-W379</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">CavityPlus is a web server that offers protein cavity detection and various functional analyses.  Using protein three-dimensional structural information as the input, CavityPlus applies CAVITY to detect potential binding sites on the surface of a given protein structure and rank them based on ligandability and druggability scores.  These potential binding sites can be further analyzed using three submodules, CavPharmer, CorrSite, and CovCys.  CavPharmer uses a receptor-based pharmacophore modeling program, Pocket, to automatically ext. pharmacophore features within cavities.  CorrSite identifies potential allosteric ligand-binding sites based on motion correlation analyses between cavities.  CovCys automatically detects druggable cysteine residues, which is esp. useful to identify novel binding sites for designing covalent allosteric ligands.  Overall, CavityPlus provides an integrated platform for analyzing comprehensive properties of protein binding cavities.  Such analyses are useful for many aspects of drug design and discovery, including target selection and identification, virtual screening, de novo drug design, and allosteric and covalent-binding drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcK4njo3ri1LVg90H21EOLACvtfcHk0lhRkQL_DZ4-cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyrs7k%253D&md5=9825bc2fb3ae8a462e9b31889338b770</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky380%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26atitle%3DCavityPlus%253A%2520A%2520Web%2520Server%2520for%2520Protein%2520Cavity%2520Detection%2520with%2520Pharmacophore%2520Modelling%252C%2520Allosteric%2520Site%2520Identification%2520and%2520Covalent%2520Ligand%2520Binding%2520Ability%2520Prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW374%26epage%3DW379%26doi%3D10.1093%2Fnar%2Fgky380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref143"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref143'); return false;" data-citation="" class="refNumLink">143</a></strong><div class="NLM_citation" id="cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Are There Physicochemical Differences between Allosteric and Competitive Ligands?</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1005813</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1005813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29125840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Wjs7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=R.+D.+Smithauthor=J.+Luauthor=H.+A.+Carlson&title=Are+There+Physicochemical+Differences+between+Allosteric+and+Competitive+Ligands%3F&doi=10.1371%2Fjournal.pcbi.1005813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Are there physicochemical differences between allosteric and competitive ligands?</span></div><div class="casAuthors">Smith, Richard D.; Lu, Jing; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e1005813/1-e1005813/18</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Previous studies have compared the physicochem. properties of allosteric compds. to non-allosteric compds.  Those studies have found that allosteric compds. tend to be smaller, more rigid, more hydrophobic, and more drug-like than non-allosteric compds.  However, previous studies have not properly cor. for the fact that some protein targets have much more data than other systems.  This generates concern regarding the possible skew that can be introduced by the inherent bias in the available data.  Hence, this study aims to det. how robust the previous findings are to the addn. of newer data.  This study utilizes the Allosteric Database (ASD v3.0) and ChEMBL v20 to systematically obtain large datasets of both allosteric and competitive ligands.  This dataset contains 70,219 and 9,511 unique ligands for the allosteric and competitive sets, resp.  Phys. relevant compd. descriptors were computed to examine the differences in their chem. properties.  Particular attention was given to removing redundancy in the data and normalizing across ligand diversity and varied protein targets.  The resulting distributions only show that allosteric ligands tend to be more arom. and rigid and do not confirm the increase in hydrophobicity or difference in drug-likeness.  These results are robust across different normalization schemes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdXKH3FrE5ObVg90H21EOLACvtfcHk0litRpw1X8dB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Wjs7%252FN&md5=629689f223c65b9fa351b7dfbdf45d98</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005813%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DAre%2520There%2520Physicochemical%2520Differences%2520between%2520Allosteric%2520and%2520Competitive%2520Ligands%253F%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.pcbi.1005813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref144"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref144'); return false;" data-citation="" class="refNumLink">144</a></strong><div class="NLM_citation" id="cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beauharnois, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolívar, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, J. T.</span></span> <span> </span><span class="NLM_article-title">Sirtuin 6: A Review of Biological Effects and Potential Therapeutic Properties</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1806</span>, <span class="refDoi"> DOI: 10.1039/c3mb00001j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2Fc3mb00001j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23592245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosl2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1789-1806&author=J.+M.+Beauharnoisauthor=B.+E.+Bol%C3%ADvarauthor=J.+T.+Welch&title=Sirtuin+6%3A+A+Review+of+Biological+Effects+and+Potential+Therapeutic+Properties&doi=10.1039%2Fc3mb00001j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin 6: a review of biological effects and potential therapeutic properties</span></div><div class="casAuthors">Beauharnois, Jade M.; Bolivar, Beatriz E.; Welch, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1789-1806</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Sirtuins, possessing either histone deacetylase or mono-ribosyltransferase activity, regulate important pathways in bacteria, archaea and eukaryotes.  SIRT6, an enzyme highly expressed in skeletal muscles, brain, heart, liver, and thymus, affects transcriptional regulation in a tissue-specific manner.  This enzyme has a two-domain structure that consists of a large Rossmann fold and a smaller and structurally more varied sequence contg. a Zn2+-binding motif.  The C-terminus is required for proper nuclear localization, while the N-terminus is important for chromatin assocn. and for intrinsic catalytic activity.  SIRT6 promotes resistance to DNA damage and oxidative stress, the principal defects assocd. with age-related diseases.  The modulation of aging and other metabolic functions by SIRT6 may be indicative of previously unrecognized regulatory systems in the cell.  The propensity of individual SIRT6 mols. to undergo intramol. mono-ADP-ribosylation, suggests this auto-ribosylation may contribute to the self-regulation of SIRT6 function.  Until recently, SIRT6 was an orphan enzyme whose catalytic activity and substrates were unclear.  It was known that, similar to the yeast Sir2 protein, human SIRT6 deacetylates histones and regulates DNA stability and repair; however, new mechanistic insights can be derived from the discovery of the highly substrate-specific histone deacetylase activity of SIRT6.  This deacetylase activity promotes proper chromatin function in several physiol. contexts, to include telomere and genome stabilization, gene expression and DNA repair.  By maintaining both the integrity and the expression of the mammalian genome, SIRT6 may help prevent cellular senescence.  Moreover, successful mol. modulation of SIRT6 activity may lead to the development of new chemotherapeutic modalities.  The action of SIRT6 is described in this review, with an emphasis on the cellular roles of the enzyme and the relation of those enzymic functions to human biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2U7MB2fgid7Vg90H21EOLACvtfcHk0litRpw1X8dB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosl2qsLY%253D&md5=d10bd8d38712a80aff92447fdfdd8587</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1039%2Fc3mb00001j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb00001j%26sid%3Dliteratum%253Aachs%26aulast%3DBeauharnois%26aufirst%3DJ.%2BM.%26aulast%3DBol%25C3%25ADvar%26aufirst%3DB.%2BE.%26aulast%3DWelch%26aufirst%3DJ.%2BT.%26atitle%3DSirtuin%25206%253A%2520A%2520Review%2520of%2520Biological%2520Effects%2520and%2520Potential%2520Therapeutic%2520Properties%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D1789%26epage%3D1806%26doi%3D10.1039%2Fc3mb00001j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref145"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref145'); return false;" data-citation="" class="refNumLink">145</a></strong><div class="NLM_citation" id="cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tennen, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">Chromatin Regulation and Genome Maintenance by Mammalian SIRT6</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2010.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20729089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=39-46&author=R.+I.+Tennenauthor=K.+F.+Chua&title=Chromatin+Regulation+and+Genome+Maintenance+by+Mammalian+SIRT6&doi=10.1016%2Fj.tibs.2010.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin regulation and genome maintenance by mammalian SIRT6</span></div><div class="casAuthors">Tennen, Ruth I.; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-46</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Saccharomyces cerevisiae Sir2 is an NAD+-dependent histone deacetylase that links chromatin silencing to genomic stability, cellular metab. and lifespan regulation.  In mice, deficiency for the Sir2 family member SIRT6 leads to genomic instability, metabolic defects and degenerative pathologies assocd. with aging.  Until recently, SIRT6 was an orphan enzyme whose catalytic activity and substrates were unclear.  However, new mechanistic insights have come from the discovery that SIRT6 is a highly substrate-specific histone deacetylase that promotes proper chromatin function in several physiol. contexts, including telomere and genome stabilization, gene expression and DNA repair.  By maintaining both the integrity and the expression of the mammalian genome, SIRT6 thus serves several roles that parallel Sir2 function.  In this article, we review recent advances in understanding the mechanisms of SIRT6 action and their implications for human biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkssgUQhRPUrVg90H21EOLACvtfcHk0litRpw1X8dB5A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgtA%253D%253D&md5=ef3dd7d19bce27f143e83bd4432cbb90</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DTennen%26aufirst%3DR.%2BI.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DChromatin%2520Regulation%2520and%2520Genome%2520Maintenance%2520by%2520Mammalian%2520SIRT6%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D39%26epage%3D46%26doi%3D10.1016%2Fj.tibs.2010.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref146"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref146'); return false;" data-citation="" class="refNumLink">146</a></strong><div class="NLM_citation" id="cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frendewey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenzuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hursting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yancopoulos, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span> <span> </span><span class="NLM_article-title">Genomic Instability and Aging-like Phenotype in the Absence of Mammalian SIRT6</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2005.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16439206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=315-329&author=R.+Mostoslavskyauthor=K.+F.+Chuaauthor=D.+B.+Lombardauthor=W.+W.+Pangauthor=M.+R.+Fischerauthor=L.+Gellonauthor=P.+Liuauthor=G.+Mostoslavskyauthor=S.+Francoauthor=M.+M.+Murphyauthor=K.+D.+Millsauthor=P.+Patelauthor=J.+T.+Hsuauthor=A.+L.+Hongauthor=E.+Fordauthor=H.+L.+Chengauthor=C.+Kennedyauthor=N.+Nunezauthor=R.+Bronsonauthor=D.+Frendeweyauthor=W.+Auerbachauthor=D.+Valenzuelaauthor=M.+Karowauthor=M.+O.+Hottigerauthor=S.+Hurstingauthor=J.+C.+Barrettauthor=L.+Guarenteauthor=R.+Mulliganauthor=B.+Dempleauthor=G.+D.+Yancopoulosauthor=F.+W.+Alt&title=Genomic+Instability+and+Aging-like+Phenotype+in+the+Absence+of+Mammalian+SIRT6&doi=10.1016%2Fj.cell.2005.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span></div><div class="casAuthors">Mostoslavsky, Raul; Chua, Katrin F.; Lombard, David B.; Pang, Wendy W.; Fischer, Miriam R.; Gellon, Lionel; Liu, Pingfang; Mostoslavsky, Gustavo; Franco, Sonia; Murphy, Michael M.; Mills, Kevin D.; Patel, Parin; Hsu, Joyce T.; Hong, Andrew L.; Ford, Ethan; Cheng, Hwei-Ling; Kennedy, Caitlin; Nunez, Nomeli; Bronson, Roderick; Frendewey, David; Auerbach, Wojtek; Valenzuela, David; Karow, Margaret; Hottiger, Michael O.; Hursting, Stephen; Barrett, J. Carl; Guarente, Leonard; Mulligan, Richard; Demple, Bruce; Yancopoulos, George D.; Alt, Frederick W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">315-329</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Sir2 histone deacetylase functions as a chromatin silencer to regulate recombination, genomic stability, and aging in budding yeast.  Seven mammalian Sir2 homologs have been identified (SIRT1-SIRT7), and it has been speculated that some may have similar functions to Sir2.  Here, we demonstrate that SIRT6 is a nuclear, chromatin-assocd. protein that promotes resistance to DNA damage and suppresses genomic instability in mouse cells, in assocn. with a role in base excision repair (BER).  SIRT6-deficient mice are small and at 2-3 wk of age develop abnormalities that include profound lymphopenia, loss of s.c. fat, lordokyphosis, and severe metabolic defects, eventually dying at about 4 wk.  We conclude that one function of SIRT6 is to promote normal DNA repair, and that SIRT6 loss leads to abnormalities in mice that overlap with aging-assocd. degenerative processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iTboO5w_ObVg90H21EOLACvtfcHk0lhG1aj4Oj1BRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D&md5=14971074ed55f5382960b500370ecff2</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DPang%26aufirst%3DW.%2BW.%26aulast%3DFischer%26aufirst%3DM.%2BR.%26aulast%3DGellon%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DM.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DHong%26aufirst%3DA.%2BL.%26aulast%3DFord%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DH.%2BL.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DNunez%26aufirst%3DN.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DFrendewey%26aufirst%3DD.%26aulast%3DAuerbach%26aufirst%3DW.%26aulast%3DValenzuela%26aufirst%3DD.%26aulast%3DKarow%26aufirst%3DM.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DHursting%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DGuarente%26aufirst%3DL.%26aulast%3DMulligan%26aufirst%3DR.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DYancopoulos%26aufirst%3DG.%2BD.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26atitle%3DGenomic%2520Instability%2520and%2520Aging-like%2520Phenotype%2520in%2520the%2520Absence%2520of%2520Mammalian%2520SIRT6%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.cell.2005.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref147"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref147'); return false;" data-citation="" class="refNumLink">147</a></strong><div class="NLM_citation" id="cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanfi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naiman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peshti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Joseph, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H. Y.</span></span> <span> </span><span class="NLM_article-title">The Sirtuin SIRT6 Regulates Lifespan in Male Mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature10815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature10815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22367546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=218-221&author=Y.+Kanfiauthor=S.+Naimanauthor=G.+Amirauthor=V.+Peshtiauthor=G.+Zinmanauthor=L.+Nahumauthor=Z.+Bar-Josephauthor=H.+Y.+Cohen&title=The+Sirtuin+SIRT6+Regulates+Lifespan+in+Male+Mice&doi=10.1038%2Fnature10815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 regulates lifespan in male mice</span></div><div class="casAuthors">Kanfi, Yariv; Naiman, Shoshana; Amir, Gail; Peshti, Victoria; Zinman, Guy; Nahum, Liat; Bar-Joseph, Ziv; Cohen, Haim Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7388</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The significant increase in human lifespan during the past century confronts us with great medical challenges.  To meet these challenges, the mechanisms that det. healthy ageing must be understood and controlled.  Sirtuins are highly conserved deacetylases that have been shown to regulate lifespan in yeast, nematodes and fruitflies.  However, the role of sirtuins in regulating worm and fly lifespan has recently become controversial.  Moreover, the role of the seven mammalian sirtuins, SIRT1 to SIRT7 (homologues of the yeast sirtuin Sir2), in regulating lifespan is unclear.  Here we show that male, but not female, transgenic mice overexpressing Sirt6 (ref. 4) have a significantly longer lifespan than wild-type mice.  Gene expression anal. revealed significant differences between male Sirt6-transgenic mice and male wild-type mice: transgenic males displayed lower serum levels of insulin-like growth factor 1 (IGF1), higher levels of IGF-binding protein 1 and altered phosphorylation levels of major components of IGF1 signalling, a key pathway in the regulation of lifespan.  This study shows the regulation of mammalian lifespan by a sirtuin family member and has important therapeutic implications for age-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo08gRkOlbJDrVg90H21EOLACvtfcHk0lhG1aj4Oj1BRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D&md5=7f87716fd9e336558e1137081146f351</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1038%2Fnature10815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10815%26sid%3Dliteratum%253Aachs%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DNaiman%26aufirst%3DS.%26aulast%3DAmir%26aufirst%3DG.%26aulast%3DPeshti%26aufirst%3DV.%26aulast%3DZinman%26aufirst%3DG.%26aulast%3DNahum%26aufirst%3DL.%26aulast%3DBar-Joseph%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26atitle%3DThe%2520Sirtuin%2520SIRT6%2520Regulates%2520Lifespan%2520in%2520Male%2520Mice%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnature10815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref148"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref148'); return false;" data-citation="" class="refNumLink">148</a></strong><div class="NLM_citation" id="cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaans, B. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gymrek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truelove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenson, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlynn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiels, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The Histone Deacetylase SIRT6 Is a Tumor Suppressor That Controls Cancer Metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1199</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.10.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2012.10.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23217706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=1185-1199&author=C.+Sebasti%C3%A1nauthor=B.+M.+M.+Zwaansauthor=D.+M.+Silbermanauthor=M.+Gymrekauthor=A.+Gorenauthor=L.+Zhongauthor=O.+Ramauthor=J.+Trueloveauthor=A.+R.+Guimaraesauthor=D.+Toiberauthor=C.+Cosentinoauthor=J.+K.+Greensonauthor=A.+I.+MacDonaldauthor=L.+McGlynnauthor=F.+Maxwellauthor=J.+Edwardsauthor=S.+Giacosaauthor=E.+Guccioneauthor=R.+Weisslederauthor=B.+E.+Bernsteinauthor=A.+Regevauthor=P.+G.+Shielsauthor=D.+B.+Lombardauthor=R.+Mostoslavsky&title=The+Histone+Deacetylase+SIRT6+Is+a+Tumor+Suppressor+That+Controls+Cancer+Metabolism&doi=10.1016%2Fj.cell.2012.10.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span></div><div class="casAuthors">Sebastian, Carlos; Zwaans, Bernadette M. M.; Silberman, Dafne M.; Gymrek, Melissa; Goren, Alon; Zhong, Lei; Ram, Oren; Truelove, Jessica; Guimaraes, Alexander R.; Toiber, Debra; Cosentino, Claudia; Greenson, Joel K.; MacDonald, Alasdair I.; McGlynn, Liane; Maxwell, Fraser; Edwards, Joanne; Giacosa, Sofia; Guccione, Ernesto; Weissleder, Ralph; Bernstein, Bradley E.; Regev, Aviv; Shiels, Paul G.; Lombard, David B.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1185-1199</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reprogramming of cellular metab. is a key event during tumorigenesis.  Despite being known for decades (Warburg effect), the mol. mechanisms regulating this switch remained unexplored.  Here, we identify SIRT6 as a tumor suppressor that regulates aerobic glycolysis in cancer cells.  Importantly, loss of SIRT6 leads to tumor formation without activation of known oncogenes, whereas transformed SIRT6-deficient cells display increased glycolysis and tumor growth, suggesting that SIRT6 plays a role in both establishment and maintenance of cancer.  By using a conditional SIRT6 allele, we show that SIRT6 deletion in vivo increases the no., size, and aggressiveness of tumors.  SIRT6 also functions as a regulator of ribosome metab. by corepressing MYC transcriptional activity.  Lastly, Sirt6 is selectively downregulated in several human cancers, and expression levels of SIRT6 predict prognosis and tumor-free survival rates, highlighting SIRT6 as a crit. modulator of cancer metab.  Our studies reveal SIRT6 to be a potent tumor suppressor acting to suppress cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6UVcycnyph7Vg90H21EOLACvtfcHk0lhG1aj4Oj1BRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J&md5=bb6df5c1d96df3ee5809773f91bb7ef5</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.047%26sid%3Dliteratum%253Aachs%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DZwaans%26aufirst%3DB.%2BM.%2BM.%26aulast%3DSilberman%26aufirst%3DD.%2BM.%26aulast%3DGymrek%26aufirst%3DM.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DRam%26aufirst%3DO.%26aulast%3DTruelove%26aufirst%3DJ.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DCosentino%26aufirst%3DC.%26aulast%3DGreenson%26aufirst%3DJ.%2BK.%26aulast%3DMacDonald%26aufirst%3DA.%2BI.%26aulast%3DMcGlynn%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DF.%26aulast%3DEdwards%26aufirst%3DJ.%26aulast%3DGiacosa%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DShiels%26aufirst%3DP.%2BG.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520Histone%2520Deacetylase%2520SIRT6%2520Is%2520a%2520Tumor%2520Suppressor%2520That%2520Controls%2520Cancer%2520Metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D1185%26epage%3D1199%26doi%3D10.1016%2Fj.cell.2012.10.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref149"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref149'); return false;" data-citation="" class="refNumLink">149</a></strong><div class="NLM_citation" id="cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">The Current State of NAD+ -Dependent Histone Deacetylases (Sirtuins) as Novel Therapeutic Targets</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1002/med.21436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fmed.21436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28094444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=147-200&author=M.+Schiedelauthor=D.+Robaaauthor=T.+Rumpfauthor=W.+Sipplauthor=M.+Jung&title=The+Current+State+of+NAD%2B+-Dependent+Histone+Deacetylases+%28Sirtuins%29+as+Novel+Therapeutic+Targets&doi=10.1002%2Fmed.21436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">The Current State of NAD+-Dependent Histone Deacetylases (Sirtuins) as Novel Therapeutic Targets</span></div><div class="casAuthors">Schiedel, Matthias; Robaa, Dina; Rumpf, Tobias; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">147-200</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are NAD+-dependent protein deacylases that cleave off acetyl, as well as other acyl groups, from the e-amino group of lysines in histones and other substrate proteins.  Seven sirtuin isotypes (Sirt1-7) have been identified in mammalian cells.  As sirtuins are involved in the regulation of various physiol. processes such as cell survival, cell cycle progression, apoptosis, DNA repair, cell metab., and caloric restriction, a dysregulation of their enzymic activity has been assocd. with the pathogenesis of neoplastic, metabolic, infectious, and neurodegenerative diseases.  Thus, sirtuins are promising targets for pharmaceutical intervention.  Growing interest in a modulation of sirtuin activity has prompted the discovery of several small mols., able to inhibit or activate certain sirtuin isotypes.  Herein, this paper gives an update to previous review on the topic in this journal (Schemies, 2010), focusing on recent developments in sirtuin biol., sirtuin modulators, and their potential as novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqt9mrksHom2rVg90H21EOLACvtfcHk0li8H1T_u3PLPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitbrO&md5=eecf3180206a18d21ba42aa79710c89d</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1002%2Fmed.21436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21436%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DRumpf%26aufirst%3DT.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520Current%2520State%2520of%2520NAD%252B%2520-Dependent%2520Histone%2520Deacetylases%2520%2528Sirtuins%2529%2520as%2520Novel%2520Therapeutic%2520Targets%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26volume%3D38%26spage%3D147%26epage%3D200%26doi%3D10.1002%2Fmed.21436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref150"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref150'); return false;" data-citation="" class="refNumLink">150</a></strong><div class="NLM_citation" id="cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Fidalgo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Hidalgo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alarcon de la Lastra, C.</span></span> <span> </span><span class="NLM_article-title">Sirtuin Modulators: Mechanisms and Potential Clinical Implications</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2414</span>– <span class="NLM_lpage">2441</span>, <span class="refDoi"> DOI: 10.2174/092986712800269272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.2174%2F092986712800269272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22455589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1egs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=2414-2441&author=S.+Sanchez-Fidalgoauthor=I.+Villegasauthor=M.+Sanchez-Hidalgoauthor=C.+Alarcon+de+la+Lastra&title=Sirtuin+Modulators%3A+Mechanisms+and+Potential+Clinical+Implications&doi=10.2174%2F092986712800269272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin modulators: mechanisms and potential clinical implications</span></div><div class="casAuthors">Sanchez-Fidalgo, S.; Villegas, I.; Sanchez-Hidalgo, M.; Alarcon de la Lastra, C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2414-2441</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  In the last years, studies about longevity have highlighted that caloric restriction can be linked with a less normal aging-assocd. damage, and in the same way, with the activity of the Silent Information Regulator 2 (SIR2) gene.  Sir2-like genes, known as sirtuins (SIRTs), have been found in organisms ranging from bacteria to mammals promoting health and survival.  At the moment, it has been identified seven classes of SIRTs in mammalian and the understanding of many of them remains still rudimentary.  However, they are in the spotlight by their potential protection against aging-assocd. diseases and have emerged as key mediators of longevity in evolutionarily distant organisms models.  SIRTs are proteins found in numerous compartments within the cell, which are NAD+-dependent protein deacetylases and ADP (ADP)-ribosyltransferases.  They catalyze a reaction in which NAD+ and an acetylated substrate are converted into a deacetylated substrate, nicotinamide and a novel metabolite O-acetyl ADP ribose.  Therefore, its enzymic activity requires NAD+, which is a crucial mol. intermediary of many metabolic reactions in cells.  Basically, SIRTs are mediators of aging process, they have the potential of ameliorating and taking part in important cellular processes assocd., such as metabolic homeostasis, tumorigenesis and cancer cell proliferation, inflammatory disorders, cardiovascular diseases and neurodegeneration.  This background opens up new lines of investigation into the modulation of SIRTs activity in order to develop novel therapeutic targets to these age-related diseases.  Current expts. using mol. activators or inhibitors and genetically engineered animals have facilitated new insights into the role of these enzymes and contributed to highlight some of the potentially relevant targets.  This review is intended to provide an appreciation of the possible protection against aging-assocd. diseases by these enzymes, summarize novel underlying mechanisms and evaluate potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprcTt8kyEd_7Vg90H21EOLACvtfcHk0li8H1T_u3PLPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1egs7o%253D&md5=de1ca82bc810faa6ac84a5f54a10aeaa</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.2174%2F092986712800269272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712800269272%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez-Fidalgo%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DI.%26aulast%3DSanchez-Hidalgo%26aufirst%3DM.%26aulast%3DAlarcon%2Bde%2Bla%2BLastra%26aufirst%3DC.%26atitle%3DSirtuin%2520Modulators%253A%2520Mechanisms%2520and%2520Potential%2520Clinical%2520Implications%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D2414%26epage%3D2441%26doi%3D10.2174%2F092986712800269272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref151"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref151'); return false;" data-citation="" class="refNumLink">151</a></strong><div class="NLM_citation" id="cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span> <span> </span><span class="NLM_article-title">STAT3 Inhibition, a Novel Approach to Enhancing Targeted Therapy in Human Cancers (Review)</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1191</span>, <span class="refDoi"> DOI: 10.3892/ijo.2012.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3892%2Fijo.2012.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22842992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyntLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=1181-1191&author=X.+Wangauthor=P.+J.+Croweauthor=D.+Goldsteinauthor=J.+L.+Yang&title=STAT3+Inhibition%2C+a+Novel+Approach+to+Enhancing+Targeted+Therapy+in+Human+Cancers+%28Review%29&doi=10.3892%2Fijo.2012.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 inhibition, a novel approach to enhancing targeted therapy in human cancers (review)</span></div><div class="casAuthors">Wang, Xiaochun; Crowe, Philip J.; Goldstein, David; Yang, Jia-Lin</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1181-1191</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) regulates many crit. functions in human normal and malignant tissues, such as differentiation, proliferation, survival, angiogenesis and immune function.  Constitutive activation of STAT3 is implicated in a wide range of human cancers.  As such, STAT3 has been studied as a tumor therapeutic target.  This review aimed principally to summarize the updated research on STAT3 inhibition studies and their therapeutic potential in solid tumors.  Recent literature assocd. with STAT3 inhibition was reviewed through PubMed and Medline database, followed by crit. comparison and anal.  Constitutive activation of STAT3 has been identified as abnormal and oncogenic.  The pathway of STAT3 activation and signal transduction identifies 3 approaches for inhibition: modulating upstream pos. or neg. regulators, regulating RNA (DN-STAT3, anti-sense RNA, siRNA and microRNA) or targeting STAT3 protein at different domains.  The last approach using small mol. STAT3 inhibitors has been the most examd. so far with both preclin. and clin. studies.  Targeting STAT3 using a specific inhibitor may be a useful cancer treatment approach, with the potential for a broad clin. impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnx1B_eI0cs7Vg90H21EOLACvtfcHk0li8H1T_u3PLPg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyntLjF&md5=91d64864c492ebe00e5d6a1cc6072284</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1568%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCrowe%26aufirst%3DP.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26atitle%3DSTAT3%2520Inhibition%252C%2520a%2520Novel%2520Approach%2520to%2520Enhancing%2520Targeted%2520Therapy%2520in%2520Human%2520Cancers%2520%2528Review%2529%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26spage%3D1181%26epage%3D1191%26doi%3D10.3892%2Fijo.2012.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref152"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref152'); return false;" data-citation="" class="refNumLink">152</a></strong><div class="NLM_citation" id="cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span> <span> </span><span class="NLM_article-title">Feedback Activation of STAT3 as a Cancer Drug-Resistance Mechanism</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tips.2015.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26576830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslektr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=47-61&author=C.+Zhaoauthor=H.+Liauthor=H.+J.+Linauthor=S.+Yangauthor=J.+Linauthor=G.+Liang&title=Feedback+Activation+of+STAT3+as+a+Cancer+Drug-Resistance+Mechanism&doi=10.1016%2Fj.tips.2015.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of STAT3 as a Cancer Drug-Resistance Mechanism</span></div><div class="casAuthors">Zhao, Chengguang; Li, Huameng; Lin, Huey-Jen; Yang, Shulin; Lin, Jiayuh; Liang, Guang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-61</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) plays crucial roles in several cellular processes such as cell proliferation and survival, and has been found to be aberrantly activated in many cancers.  Much research has explored the leading mechanisms for regulating the STAT3 pathway and its role in promoting tumorigenesis.  We focus here on recent evidence suggesting that feedback activation of STAT3 plays a prominent role in mediating drug resistance to a broad spectrum of targeted cancer therapies and chemotherapies.  We highlight the potential of co-targeting STAT3 and its primary target to overcome drug resistance, and provide perspective on repurposing clin. approved drugs as STAT3 pathway inhibitors, in combination with the FDA-approved receptor tyrosine kinase (RTK) inhibitors, to improve clin. outcome of cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV-bWS3i24YLVg90H21EOLACvtfcHk0ljzBTmnLL8Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslektr3J&md5=0fe954b8173ecbf8d855785ffd14bc0f</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DH.%2BJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DG.%26atitle%3DFeedback%2520Activation%2520of%2520STAT3%2520as%2520a%2520Cancer%2520Drug-Resistance%2520Mechanism%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26spage%3D47%26epage%3D61%26doi%3D10.1016%2Fj.tips.2015.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref153"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref153'); return false;" data-citation="" class="refNumLink">153</a></strong><div class="NLM_citation" id="cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sgrignani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garofalo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merulla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of STAT3 and Its Implications for Anticancer Therapies Development</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1591</span>, <span class="refDoi"> DOI: 10.3390/ijms19061591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms19061591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSrt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1591&author=J.+Sgrignaniauthor=M.+Garofaloauthor=M.+Matkovicauthor=J.+Merullaauthor=C.+V.+Catapanoauthor=A.+Cavalli&title=Structural+Biology+of+STAT3+and+Its+Implications+for+Anticancer+Therapies+Development&doi=10.3390%2Fijms19061591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of STAT3 and its implications for anticancer therapies development</span></div><div class="casAuthors">Sgrignani, Jacopo; Garofalo, Maura; Matkovic, Milos; Merulla, Jessica; Catapano, Carlo V.; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1591/1-1591/14</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcription factors are proteins able to bind DNA and induce the transcription of specific genes.  Consequently, they play a pivotal role in multiple cellular pathways and are frequently over-expressed or dysregulated in cancer.  Here, we will focus on a specific "signal transducer and activator of transcription" (STAT3) factor that is involved in several pathologies, including cancer.  For long time, the mechanism by which STAT3 exerts its cellular functions has been summarized by a three steps process: (1) Protein phosphorylation by specific kinases, (2) dimerization promoted by phosphorylation, (3) activation of gene expression by the phosphorylated dimer.  Consequently, most of the inhibitors reported in literature aimed at blocking phosphorylation and dimerization.  However, recent observations reopened the debate and the entire functional mechanism has been revisited stimulating the scientific community to pursue new inhibition strategies.  In particular, the dimerization of the unphosphorylated species has been exptl. demonstrated and specific roles proposed also for these dimers.  Despite difficulties in the expression and purifn. of the full length STAT3, structural biol. investigations allowed the detn. of atomistic structures of STAT3 dimers and several protein domains.  Starting from this information, computational methods have been used both to improve the understanding of the STAT3 functional mechanism and to design new inhibitors to be used as anticancer drugs.  In this review, we will focus on the contribution of structural biol. to understand the roles of STAT3, to design new inhibitors and to suggest new strategies of pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwOe7is1W8ZLVg90H21EOLACvtfcHk0ljzBTmnLL8Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSrt73O&md5=5144a31d327b9fd68e0956f8f2dffeeb</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.3390%2Fijms19061591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19061591%26sid%3Dliteratum%253Aachs%26aulast%3DSgrignani%26aufirst%3DJ.%26aulast%3DGarofalo%26aufirst%3DM.%26aulast%3DMatkovic%26aufirst%3DM.%26aulast%3DMerulla%26aufirst%3DJ.%26aulast%3DCatapano%26aufirst%3DC.%2BV.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DStructural%2520Biology%2520of%2520STAT3%2520and%2520Its%2520Implications%2520for%2520Anticancer%2520Therapies%2520Development%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D1591%26doi%3D10.3390%2Fijms19061591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref154"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref154'); return false;" data-citation="" class="refNumLink">154</a></strong><div class="NLM_citation" id="cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (stat3) Protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6645</span>– <span class="NLM_lpage">6648</span>, <span class="refDoi"> DOI: 10.1021/jm300207s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300207s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6645-6648&author=B.+Debnathauthor=S.+Xuauthor=N.+Neamati&title=Small+Molecule+Inhibitors+of+Signal+Transducer+and+Activator+of+Transcription+3+%28stat3%29+Protein&doi=10.1021%2Fjm300207s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (Stat3) Protein</span></div><div class="casAuthors">Debnath, Bikash; Xu, Shili; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6645-6668</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (Stat3) is a transcription factor belonging to the STAT protein family, members of which regulate a wide variety of biol. processes, including cell growth, proliferation, survival, differentiation, development, movement and inflammation in response to extracellular signaling mols. such as cytokines and growth factors.  Receptor assocd. tyrosine kinases phosphorylate Stat3 protein at Tyr705 and enable it to dimerize and subsequently enter the nucleus where it activates the transcription of its target genes.  Accumulating evidence has established Stat3's role in oncogenesis, cardiovascular diseases, rheumatoid arthritis, fibrotic, and Alzheimer's diseases.  Thus, Stat3 inhibition is a promising therapeutic approach not only for cancer, but also for other diseases.  Although Stat3 has been considered as an important therapeutic target, progress in discovery and development of effective and specific Stat3 inhibitors remains slow.  In this article we summarize Stat3 inhibitors published between 2006 and 2011.  New strategies along with novel technologies are required to achieve breakthrough in the development of Stat3 inhibitors that can be translated into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWWQl9gV1-ZLVg90H21EOLACvtfcHk0ljzBTmnLL8Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D&md5=f5f3ac9f8fc2326c75edd2a9c136275b</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1021%2Fjm300207s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300207s%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Signal%2520Transducer%2520and%2520Activator%2520of%2520Transcription%25203%2520%2528stat3%2529%2520Protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6645%26epage%3D6648%26doi%3D10.1021%2Fjm300207s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref155"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref155'); return false;" data-citation="" class="refNumLink">155</a></strong><div class="NLM_citation" id="cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miklossy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youn, U. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkaria, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongwiwatthananukit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Hirsutinolide Series Inhibit Stat3 Activity, Alter GCN1, MAP1B, Hsp105, G6PD, Vimentin, TrxR1, and Importin α-2 Expression, and Induce Antitumor Effects against Human Glioma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7734</span>– <span class="NLM_lpage">7748</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCgu7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7734-7748&author=G.+Miklossyauthor=U.+J.+Younauthor=P.+Yueauthor=M.+Zhangauthor=C.+H.+Chenauthor=T.+S.+Hilliardauthor=D.+Paladinoauthor=Y.+Liauthor=J.+Choiauthor=J.+N.+Sarkariaauthor=J.+K.+Kawakamiauthor=S.+Wongwiwatthananukitauthor=Y.+Chenauthor=D.+Sunauthor=L.+C.+Changauthor=J.+Turkson&title=Hirsutinolide+Series+Inhibit+Stat3+Activity%2C+Alter+GCN1%2C+MAP1B%2C+Hsp105%2C+G6PD%2C+Vimentin%2C+TrxR1%2C+and+Importin+%CE%B1-2+Expression%2C+and+Induce+Antitumor+Effects+against+Human+Glioma&doi=10.1021%2Facs.jmedchem.5b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Hirsutinolide Series Inhibit Stat3 Activity, Alter GCN1, MAP1B, Hsp105, G6PD, Vimentin, TrxR1, and Importin α-2 Expression, and Induce Antitumor Effects against Human Glioma</span></div><div class="casAuthors">Miklossy, Gabriella; Youn, Ui Joung; Yue, Peibin; Zhang, Mingming; Chen, Chih-Hong; Hilliard, Tyvette S.; Paladino, David; Li, Yifei; Choi, Justin; Sarkaria, Jann N.; Kawakami, Joel K.; Wongwiwatthananukit, Supakit; Chen, Yuan; Sun, Dianqing; Chang, Leng Chee; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7734-7748</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report that hirsutinolide series and the semisynthetic analogs inhibit constitutively active signal transducer and activator of transcription (Stat)3 and malignant glioma phenotype.  A position 13 lipophilic ester group is required for activity.  Mol. modeling and NMR structural analyses reveal direct hirsutinolide:Stat3 binding.  One-hour treatment of cells with I and II also upregulated importin subunit α-2 levels and repressed translational activator GCN1, microtubule-assocd. protein (MAP)1B, thioredoxin reductase (TrxR)1 cytoplasmic isoform 3, glucose-6-phosphate 1-dehydrogenase isoform a, Hsp105, vimentin, and tumor necrosis factor α-induced protein (TNAP)2 expression.  Active hirsutinolides inhibited anchorage-dependent and three-dimensional spheroid growth, survival, and migration of human glioma lines and glioma patients' tumor-derived xenograft cells harboring constitutively active Stat3.  Oral gavage delivery of I or II inhibited human glioma tumor growth in s.c. mouse xenografts.  The inhibition of Stat3 signaling represents part of the hirsutinolide-mediated mechanisms to induce antitumor effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-5VLBoRoafbVg90H21EOLACvtfcHk0lgsOfAEs8DfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCgu7jN&md5=65cfae17ba81a38ca5dd00e0c07265b4</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00686%26sid%3Dliteratum%253Aachs%26aulast%3DMiklossy%26aufirst%3DG.%26aulast%3DYoun%26aufirst%3DU.%2BJ.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DHilliard%26aufirst%3DT.%2BS.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DSarkaria%26aufirst%3DJ.%2BN.%26aulast%3DKawakami%26aufirst%3DJ.%2BK.%26aulast%3DWongwiwatthananukit%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DL.%2BC.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DHirsutinolide%2520Series%2520Inhibit%2520Stat3%2520Activity%252C%2520Alter%2520GCN1%252C%2520MAP1B%252C%2520Hsp105%252C%2520G6PD%252C%2520Vimentin%252C%2520TrxR1%252C%2520and%2520Importin%2520%25CE%25B1-2%2520Expression%252C%2520and%2520Induce%2520Antitumor%2520Effects%2520against%2520Human%2520Glioma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7734%26epage%3D7748%26doi%3D10.1021%2Facs.jmedchem.5b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref156"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref156'); return false;" data-citation="" class="refNumLink">156</a></strong><div class="NLM_citation" id="cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Escobar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjartell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canesin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans-Axelsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellsten, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, M. H.</span></span> <span> </span><span class="NLM_article-title">Preclinical Characterization of 3β-(N-Acetyl L -Cysteine Methyl Ester)-2aβ,3-Dihydrogaliellalactone (GPA512), a Prodrug of a Direct STAT3 Inhibitor for the Treatment of Prostate Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4551</span>– <span class="NLM_lpage">4562</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01814</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01814" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Gqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4551-4562&author=Z.+Escobarauthor=A.+Bjartellauthor=G.+Canesinauthor=S.+Evans-Axelssonauthor=O.+Sternerauthor=R.+Hellstenauthor=M.+H.+Johansson&title=Preclinical+Characterization+of+3%CE%B2-%28N-Acetyl+L+-Cysteine+Methyl+Ester%29-2a%CE%B2%2C3-Dihydrogaliellalactone+%28GPA512%29%2C+a+Prodrug+of+a+Direct+STAT3+Inhibitor+for+the+Treatment+of+Prostate+Cancer&doi=10.1021%2Facs.jmedchem.5b01814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of 3β-(N-Acetyl L-cysteine methyl ester)-2aβ,3-dihydrogaliellalactone (GPA512), a Prodrug of a Direct STAT3 Inhibitor for the Treatment of Prostate Cancer</span></div><div class="casAuthors">Escobar, Zilma; Bjartell, Anders; Canesin, Giacomo; Evans-Axelsson, Susan; Sterner, Olov; Hellsten, Rebecka; Johansson, Martin H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4551-4562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor STAT3 is a potential target for the treatment of castration-resistant prostate cancer.  Galiellalactone (1), a direct inhibitor of STAT3, prevents the transcription of STAT3 regulated genes.  In this study we characterized 6, a prodrug of 1.  In vitro studies showed that 6 is rapidly converted to 1 in plasma and is stable in a buffer soln.  The pharmacokinetics of 6 following a single oral dose indicated that the prodrug was rapidly absorbed and converted to 1 with a tmax of 15 min.  Oral administration of 6 in mice increased the plasma exposure of the active parent compd. 20-fold compared to when 1 was dosed orally. 6 treated mice bearing DU145 xenograft tumors had significantly reduced tumor growth compared to untreated mice.  The favorable druglike properties and safety profile of 6 warrant further studies of 6 for the treatment of castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ-05xP9iOM7Vg90H21EOLACvtfcHk0lgsOfAEs8DfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Gqu7g%253D&md5=1ed1bfdda22e882cbcdad28162d9c719</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01814%26sid%3Dliteratum%253Aachs%26aulast%3DEscobar%26aufirst%3DZ.%26aulast%3DBjartell%26aufirst%3DA.%26aulast%3DCanesin%26aufirst%3DG.%26aulast%3DEvans-Axelsson%26aufirst%3DS.%26aulast%3DSterner%26aufirst%3DO.%26aulast%3DHellsten%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DM.%2BH.%26atitle%3DPreclinical%2520Characterization%2520of%25203%25CE%25B2-%2528N-Acetyl%2520L%2520-Cysteine%2520Methyl%2520Ester%2529-2a%25CE%25B2%252C3-Dihydrogaliellalactone%2520%2528GPA512%2529%252C%2520a%2520Prodrug%2520of%2520a%2520Direct%2520STAT3%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4551%26epage%3D4562%26doi%3D10.1021%2Facs.jmedchem.5b01814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref157"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref157'); return false;" data-citation="" class="refNumLink">157</a></strong><div class="NLM_citation" id="cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2731</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2718-2731&author=W.+Yuauthor=C.+Liauthor=W.+Zhangauthor=Y.+Xiaauthor=S.+Liauthor=J.-y.+Linauthor=K.+Yuauthor=M.+Liuauthor=L.+Yangauthor=J.+Luoauthor=Y.+Chenauthor=H.+Sunauthor=L.+Kong&title=Discovery+of+an+Orally+Selective+Inhibitor+of+Signal+Transducer+and+Activator+of+Transcription+3+Using+Advanced+Multiple+Ligand+Simultaneous+Docking&doi=10.1021%2Facs.jmedchem.6b01489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span></div><div class="casAuthors">Yu, Wenying; Li, Chenglong; Zhang, Wenda; Xia, Yuanzheng; Li, Shanshan; Lin, Jia-yuh; Yu, Keqin; Liu, Mu; Yang, Lei; Luo, Jianguang; Chen, Yijun; Sun, Hongbin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting signal transducer and activator of transcription 3 (STAT3) is a potential anticancer strategy.  However, STAT3 inhibitors with good selectivity and bioavailability are rare.  The aim of this study was to discover selective direct STAT3 inhibitors with good drug-likeness.  By advanced multiple ligand simultaneous docking (AMLSD) method, compd. 9 (5,8-dioxo-6-((2-(piperazin-1-yl)phenyl)amino)-5,8-dihydronaphthalene-1-sulfonamide) was designed as an orally-bioavailable STAT3 inhibitor, which presented superior druggability and selectivity to other representative STAT3 inhibitors.  Compd. 9 directly and selectively inhibited the pY705 site of STAT3 with an affinity (Ki) of 440nM.  The IC50 of 9 for breast cancer cell MDA-MB-231 was 184-fold lower than its IC50 for normal breast epithelial cell MCF-10A.  Compd. 9 in vivo induced significant antitumor responses better than Gefitinib and its therapeutic index should be over 100, indicating good safety of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT8F_tCrky7Vg90H21EOLACvtfcHk0lgsOfAEs8DfoA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D&md5=d9554e49fd17a559e8c7171f36c53607</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01489%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.-y.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520an%2520Orally%2520Selective%2520Inhibitor%2520of%2520Signal%2520Transducer%2520and%2520Activator%2520of%2520Transcription%25203%2520Using%2520Advanced%2520Multiple%2520Ligand%2520Simultaneous%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2718%26epage%3D2731%26doi%3D10.1021%2Facs.jmedchem.6b01489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref158"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref158'); return false;" data-citation="" class="refNumLink">158</a></strong><div class="NLM_citation" id="cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span> <span> </span><span class="NLM_article-title">Two Decades of Research in Discovery of Anticancer Drugs Targeting STAT3, How Close Are We?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2018.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29933035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=74-91&author=J.+D.+Beebeauthor=J.+Y.+Liuauthor=J.+T.+Zhang&title=Two+Decades+of+Research+in+Discovery+of+Anticancer+Drugs+Targeting+STAT3%2C+How+Close+Are+We%3F&doi=10.1016%2Fj.pharmthera.2018.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span></div><div class="casAuthors">Beebe, Jenny D.; Liu, Jing-Yuan; Zhang, Jian-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-91</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) controls many biol. processes including differentiation, survival, proliferation, and angiogenesis.  In normal healthy cells, STAT3 is tightly regulated to maintain a momentary active state.  However, aberrant or constitutively activated STAT3 has been obsd. in many different cancers and constitutively activated STAT3 has been shown to assoc. with poor prognosis and tumor progression.  For this reason, STAT3 has been studied as a possible target in the treatment of many different types of cancers.  However, despite decades of research, a FDA-approved STAT3 inhibitor has yet to emerge.  In this review, we will analyze past studies targeting STAT3 for drug discovery, understand possible causes of failure in these studies, and provide potential insights for future efforts to overcome these roadblocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWDPJOIaRKrVg90H21EOLACvtfcHk0li-nVYju8KKtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO&md5=048e7d95608bdab18335c55d084d62ab</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26atitle%3DTwo%2520Decades%2520of%2520Research%2520in%2520Discovery%2520of%2520Anticancer%2520Drugs%2520Targeting%2520STAT3%252C%2520How%2520Close%2520Are%2520We%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D74%26epage%3D91%26doi%3D10.1016%2Fj.pharmthera.2018.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref159"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref159'); return false;" data-citation="" class="refNumLink">159</a></strong><div class="NLM_citation" id="cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+Improving+the+Speed+and+Accuracy+of+Docking+with+a+New+Scoring+Function%2C+Efficient+Optimization%2C+and+Multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0li-nVYju8KKtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520Improving%2520the%2520Speed%2520and%2520Accuracy%2520of%2520Docking%2520with%2520a%2520New%2520Scoring%2520Function%252C%2520Efficient%2520Optimization%252C%2520and%2520Multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref160"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref160'); return false;" data-citation="" class="refNumLink">160</a></strong><div class="NLM_citation" id="cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savaskan, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ufer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kühn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchert, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Biology of Glutathione Peroxidase 4: From Genomic Structure to Developmental Expression and Neural Function</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>388</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1515/BC.2007.126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1515%2FBC.2007.126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17937614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFCntrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2007&pages=1007-1017&author=N.+E.+Savaskanauthor=C.+Uferauthor=H.+K%C3%BChnauthor=A.+Borchert&title=Molecular+Biology+of+Glutathione+Peroxidase+4%3A+From+Genomic+Structure+to+Developmental+Expression+and+Neural+Function&doi=10.1515%2FBC.2007.126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular biology of glutathione peroxidase 4: from genomic structure to developmental expression and neural function</span></div><div class="casAuthors">Savaskan, Nicolai E.; Ufer, Christoph; Kuehn, Hartmut; Borchert, Astrid</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Selenoproteins have been recognized as modulators of brain function and signaling.  Phospholipid hydroperoxide glutathione peroxidase (GPx4/PHGPx) is a unique member of the selenium-dependent glutathione peroxidases in mammals with a pivotal role in brain development and function.  GPx4 exists as a cytosolic, mitochondrial, and nuclear isoform derived from a single gene.  In mice, the GPx4 gene is located on chromosome 10 in close proximity to a functional retrotransposome that is expressed under the control of captured regulatory elements.  Elucidation of crystallog. data uncovered structural peculiarities of GPx4 that provide the mol. basis for its unique enzymic properties and substrate specificity.  Monomeric GPx4 is multifunctional: it acts as a reducing enzyme of peroxidized phospholipids and thiols and as a structural protein.  Transcriptional regulation of the different GPx4 isoforms requires several isoform-specific cis-regulatory sequences and trans-activating factors.  Cytosolic and mitochondrial GPx4 are the major isoforms exclusively expressed by neurons in the developing brain.  In stark contrast, following brain trauma, GPx4 is specifically upregulated in non-neuronal cells, i.e., reactive astrocytes.  Mol. approaches to genetic modification in mice have revealed an essential and isoform-specific function for GPx4 in development and disease.  Here we review recent findings on GPx4 with emphasis on its mol. structure and function and consider potential mechanisms that underlie neural development and neuropathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov7VCeQGwYIbVg90H21EOLACvtfcHk0li-nVYju8KKtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFCntrfK&md5=dbdfa4e56338459b614042500022a661</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1515%2FBC.2007.126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2007.126%26sid%3Dliteratum%253Aachs%26aulast%3DSavaskan%26aufirst%3DN.%2BE.%26aulast%3DUfer%26aufirst%3DC.%26aulast%3DK%25C3%25BChn%26aufirst%3DH.%26aulast%3DBorchert%26aufirst%3DA.%26atitle%3DMolecular%2520Biology%2520of%2520Glutathione%2520Peroxidase%25204%253A%2520From%2520Genomic%2520Structure%2520to%2520Developmental%2520Expression%2520and%2520Neural%2520Function%26jtitle%3DBiol.%2520Chem.%26date%3D2007%26volume%3D388%26spage%3D1007%26epage%3D1017%26doi%3D10.1515%2FBC.2007.126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref161"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref161'); return false;" data-citation="" class="refNumLink">161</a></strong><div class="NLM_citation" id="cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriramaratnam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skouta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotti, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornish, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Regulation of Ferroptotic Cancer Cell Death by GPX4</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2013.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24439385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtF2is70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=317-331&author=W.+S.+Yangauthor=R.+Sriramaratnamauthor=M.+E.+Welschauthor=K.+Shimadaauthor=R.+Skoutaauthor=V.+S.+Viswanathanauthor=J.+H.+Cheahauthor=P.+A.+Clemonsauthor=A.+F.+Shamjiauthor=C.+B.+Clishauthor=L.+M.+Brownauthor=A.+W.+Girottiauthor=V.+W.+Cornishauthor=S.+L.+Schreiberauthor=B.+R.+Stockwell&title=Regulation+of+Ferroptotic+Cancer+Cell+Death+by+GPX4&doi=10.1016%2Fj.cell.2013.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Ferroptotic Cancer Cell Death by GPX4</span></div><div class="casAuthors">Yang, Wan Seok; SriRamaratnam, Rohitha; Welsch, Matthew E.; Shimada, Kenichi; Skouta, Rachid; Viswanathan, Vasanthi S.; Cheah, Jaime H.; Clemons, Paul A.; Shamji, Alykhan F.; Clish, Clary B.; Brown, Lewis M.; Girotti, Albert W.; Cornish, Virginia W.; Schreiber, Stuart L.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">317-331</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ferroptosis is a form of nonapoptotic cell death for which key regulators remain unknown.  We sought a common mediator for the lethality of 12 ferroptosis-inducing small mols.  We used targeted metabolomic profiling to discover that depletion of glutathione causes inactivation of glutathione peroxidases (GPXs) in response to one class of compds. and a chemoproteomics strategy to discover that GPX4 is directly inhibited by a second class of compds.  GPX4 overexpression and knockdown modulated the lethality of 12 ferroptosis inducers, but not of 11 compds. with other lethal mechanisms.  In addn., two representative ferroptosis inducers prevented tumor growth in xenograft mouse tumor models.  Sensitivity profiling in 177 cancer cell lines revealed that diffuse large B cell lymphomas and renal cell carcinomas are particularly susceptible to GPX4-regulated ferroptosis.  Thus, GPX4 is an essential regulator of ferroptotic cancer cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdxNz56kpRuLVg90H21EOLACvtfcHk0li-nVYju8KKtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtF2is70%253D&md5=f5f607e12540847d9a25d8c205ba2e3c</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%2BS.%26aulast%3DSriramaratnam%26aufirst%3DR.%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DShimada%26aufirst%3DK.%26aulast%3DSkouta%26aufirst%3DR.%26aulast%3DViswanathan%26aufirst%3DV.%2BS.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBrown%26aufirst%3DL.%2BM.%26aulast%3DGirotti%26aufirst%3DA.%2BW.%26aulast%3DCornish%26aufirst%3DV.%2BW.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DRegulation%2520of%2520Ferroptotic%2520Cancer%2520Cell%2520Death%2520by%2520GPX4%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D317%26epage%3D331%26doi%3D10.1016%2Fj.cell.2013.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref162"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref162'); return false;" data-citation="" class="refNumLink">162</a></strong><div class="NLM_citation" id="cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedmann
Angeli, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Activation of Glutathione Peroxidase 4 as a Novel Anti-Inflammatory Strategy</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3389%2Ffphar.2018.01120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29387012" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-12&author=C.+Liauthor=X.+Dengauthor=X.+Xieauthor=Y.+Liuauthor=J.+P.+Friedmann%0AAngeliauthor=L.+Lai&title=Activation+of+Glutathione+Peroxidase+4+as+a+Novel+Anti-Inflammatory+Strategy&doi=10.3389%2Ffphar.2018.01120"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01120%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFriedmann%2BAngeli%26aufirst%3DJ.%2BP.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DActivation%2520of%2520Glutathione%2520Peroxidase%25204%2520as%2520a%2520Novel%2520Anti-Inflammatory%2520Strategy%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D12%26doi%3D10.3389%2Ffphar.2018.01120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref163"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref163'); return false;" data-citation="" class="refNumLink">163</a></strong><div class="NLM_citation" id="cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">LigBuilder 2: A Practical de Novo Drug Design Approach</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1083</span>– <span class="NLM_lpage">1091</span>, <span class="refDoi"> DOI: 10.1021/ci100350u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100350u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVKqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1083-1091&author=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=LigBuilder+2%3A+A+Practical+de+Novo+Drug+Design+Approach&doi=10.1021%2Fci100350u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">LigBuilder 2: A Practical de Novo Drug Design Approach</span></div><div class="casAuthors">Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1083-1091</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have developed a new version (2.0) of the de novo drug design program LigBuilder.  With LigBuilder 2.0, the synthesis accessibility of designed compds. can be analyzed, and a cavity detection procedure is implemented to detect the positions and shapes of the binding sites on the surface of a given protein structure and to quant. est. drugability.  Ligands are designed to best fit the detected cavities using a set of rules for evaluation.  Drug-like and privileged fragments are used to construct the ligands with the aid of internal and external absorption, distribution, metab., excretion, and toxicity (ADME/T) and drug-like filters.  The authors have tested LigBuilder 2.0 in designing and optimizing Cyclophilin A inhibitors.  By using only a one-round design, a novel inhibitor with greater potency than the pos. control, cyclosporin A, was discovered.  This successful de novo drug design example verified the effectiveness of LigBuilder 2.0.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqb53kWl8fbVg90H21EOLACvtfcHk0lgaoTfGNxz26g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVKqsbc%253D&md5=e33a958dcc48f86e4225dbfe10c2fd51</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fci100350u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100350u%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DLigBuilder%25202%253A%2520A%2520Practical%2520de%2520Novo%2520Drug%2520Design%2520Approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1083%26epage%3D1091%26doi%3D10.1021%2Fci100350u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref164"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref164'); return false;" data-citation="" class="refNumLink">164</a></strong><div class="NLM_citation" id="cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nashine, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Socolich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Surface Sites for Engineering Allosteric Control in Proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>322</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">442</span>, <span class="refDoi"> DOI: 10.1126/science.1159052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1159052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18927392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Citr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2008&pages=438-442&author=J.+Leeauthor=M.+Natarajanauthor=V.+C.+Nashineauthor=M.+Socolichauthor=T.+Voauthor=W.+P.+Russauthor=S.+J.+Benkovicauthor=R.+Ranganathan&title=Surface+Sites+for+Engineering+Allosteric+Control+in+Proteins&doi=10.1126%2Fscience.1159052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Sites for Engineering Allosteric Control in Proteins</span></div><div class="casAuthors">Lee, Jeeyeon; Natarajan, Madhusudan; Nashine, Vishal C.; Socolich, Michael; Vo, Tina; Russ, William P.; Benkovic, Stephen J.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">5900</span>),
    <span class="NLM_cas:pages">438-442</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Statistical analyses of protein families reveal networks of coevolving amino acids that functionally link distantly positioned functional surfaces.  Such linkages suggest a concept for engineering allosteric control into proteins: the intramol. networks of two proteins could be joined across their surface sites such that the activity of one protein might control the activity of the other.  We tested this idea by creating PAS-DHFR, a designed chimeric protein that connects a light-sensing signaling domain from a plant member of the Per/Arnt/Sim (PAS) family of proteins with Escherichia coli dihydrofolate reductase (DHFR).  With no optimization, PAS-DHFR exhibited light-dependent catalytic activity that depended on the site of connection and on known signaling mechanisms in both proteins.  PAS-DHFR serves as a proof of concept for engineering regulatory activities into proteins through interface design at conserved allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYSUy8SM8YL7Vg90H21EOLACvtfcHk0lgaoTfGNxz26g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Citr3J&md5=0349bddb15613bd551271721a780789c</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159052%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNatarajan%26aufirst%3DM.%26aulast%3DNashine%26aufirst%3DV.%2BC.%26aulast%3DSocolich%26aufirst%3DM.%26aulast%3DVo%26aufirst%3DT.%26aulast%3DRuss%26aufirst%3DW.%2BP.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DSurface%2520Sites%2520for%2520Engineering%2520Allosteric%2520Control%2520in%2520Proteins%26jtitle%3DScience%26date%3D2008%26volume%3D322%26spage%3D438%26epage%3D442%26doi%3D10.1126%2Fscience.1159052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref165"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref165'); return false;" data-citation="" class="refNumLink">165</a></strong><div class="NLM_citation" id="cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Hot Spots for Allosteric Regulation on Protein Surfaces</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1564</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2011.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22196731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ort73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=1564-1575&author=K.+A.+Reynoldsauthor=R.+N.+McLaughlinauthor=R.+Ranganathan&title=Hot+Spots+for+Allosteric+Regulation+on+Protein+Surfaces&doi=10.1016%2Fj.cell.2011.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spots for Allosteric Regulation on Protein Surfaces</span></div><div class="casAuthors">Reynolds, Kimberly A.; McLaughlin, Richard N.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1564-1575</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent work indicates a general architecture for proteins in which sparse networks of phys. contiguous and coevolving amino acids underlie basic aspects of structure and function.  These networks, termed sectors, are spatially organized such that active sites are linked to many surface sites distributed throughout the structure.  Using the metabolic enzyme dihydrofolate reductase (DHFR) as a model system, we show that: (1) the sector is strongly correlated to a network of residues undergoing millisecond conformational fluctuations assocd. with enzyme catalysis, and (2) sector-connected surface sites are statistically preferred locations for the emergence of allosteric control in vivo.  Thus, sectors represent an evolutionarily conserved "wiring" mechanism that can enable perturbations at sp. surface positions to rapidly initiate conformational control over protein function.  These findings suggest that sectors enable the evolution of intermol. communication and regulation.  Protein sectors, networks of phys. contiguous and coevolving amino acids, facilitate the emergence of allosteric regulation, as demonstrated with chimeric proteins.  The findings suggest that sectors also naturally enable the evolution of intermol. communication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp62r-KReV7z7Vg90H21EOLACvtfcHk0liXxh5LjH_eQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ort73J&md5=e86fcb4f822f2408d96756b07afdb389</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26aulast%3DMcLaughlin%26aufirst%3DR.%2BN.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DHot%2520Spots%2520for%2520Allosteric%2520Regulation%2520on%2520Protein%2520Surfaces%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D1564%26epage%3D1575%26doi%3D10.1016%2Fj.cell.2011.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref166"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref166'); return false;" data-citation="" class="refNumLink">166</a></strong><div class="NLM_citation" id="cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, R. W.</span></span> <span> </span><span class="NLM_article-title">On Evolutionary Conservation of Thermodynamic Coupling in Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">19046</span>– <span class="NLM_lpage">19050</span>, <span class="refDoi"> DOI: 10.1074/jbc.M402560200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.M402560200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15023994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVKmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=19046-19050&author=A.+A.+Fodorauthor=R.+W.+Aldrich&title=On+Evolutionary+Conservation+of+Thermodynamic+Coupling+in+Proteins&doi=10.1074%2Fjbc.M402560200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">On Evolutionary Conservation of Thermodynamic Coupling in Proteins</span></div><div class="casAuthors">Fodor, Anthony A.; Aldrich, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">19046-19050</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The inherent complexity of thermodn. coupling in proteins presents a major challenge in understanding and engineering protein function.  Recent work has argued that the study of proteins can be simplified by the use of correlated mutations in the evolutionary record to locate a small subset of thermodynamically coupled residues that participate in functionally important, evolutionarily conserved energetic pathways.  To test this hypothesis, we examd. the predictions of correlated mutation algorithms for a no. of proteins for which coupling between residues has been detd. by anal. of double mutant cycles.  We find that correlated mutation algorithms can find residue pairs that are phys. close and that phys. close residue pairs tend to be thermodynamically coupled.  We find little evidence, however, for the hypothesis that thermodn. coupling is limited to the subset of evolutionarily constrained residue positions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVUz7Ul6I1ZbVg90H21EOLACvtfcHk0liXxh5LjH_eQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVKmtL8%253D&md5=74e23b9782a2c024b7e2df6d56d58426</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402560200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402560200%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DA.%2BA.%26aulast%3DAldrich%26aufirst%3DR.%2BW.%26atitle%3DOn%2520Evolutionary%2520Conservation%2520of%2520Thermodynamic%2520Coupling%2520in%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D19046%26epage%3D19050%26doi%3D10.1074%2Fjbc.M402560200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref167"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref167'); return false;" data-citation="" class="refNumLink">167</a></strong><div class="NLM_citation" id="cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elfström, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snäll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">Reassessing a Sparse Energetic Network within a Single Protein Domain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">4679</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711732105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0711732105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18339805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt12qs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=4679-4684&author=C.+N.+Chiauthor=L.+Elfstr%C3%B6mauthor=Y.+Shiauthor=T.+Sn%C3%A4llauthor=A.+Engstr%C3%B6mauthor=P.+Jemth&title=Reassessing+a+Sparse+Energetic+Network+within+a+Single+Protein+Domain&doi=10.1073%2Fpnas.0711732105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Reassessing a sparse energetic network within a single protein domain</span></div><div class="casAuthors">Chi, Celestine N.; Eelfstroem, Lisa; Shi, Yao; Snaell, Tord; Engstroem, Aake; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4679-4684</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding the mol. principles that govern allosteric communication is an important goal in protein science.  One way allostery could be transmitted is via sparse energetic networks of residues, and one such evolutionary conserved network was identified in the PDZ domain family of proteins by multiple sequence alignment.  We have reassessed the energetic coupling of these residues by double mutant cycles together with ligand binding and stability expts. and found that coupling is not a special property of the coevolved network of residues in PDZ domains.  The obsd. coupling for ligand binding is better explained by a distance relationship, where residues close in space are more likely to couple than distal residues.  Our study demonstrates that statistical coupling from sequence anal. is not necessarily a reporter of energetic coupling and allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFuXABqS1debVg90H21EOLACvtfcHk0liXxh5LjH_eQA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt12qs7k%253D&md5=93aa623be2915a0b00ea7ea872bab99b</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711732105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711732105%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DC.%2BN.%26aulast%3DElfstr%25C3%25B6m%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSn%25C3%25A4ll%26aufirst%3DT.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DReassessing%2520a%2520Sparse%2520Energetic%2520Network%2520within%2520a%2520Single%2520Protein%2520Domain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D4679%26epage%3D4684%26doi%3D10.1073%2Fpnas.0711732105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref168"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref168'); return false;" data-citation="" class="refNumLink">168</a></strong><div class="NLM_citation" id="cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span> <span> </span><span class="NLM_article-title">On the Accuracy of Inferring Energetic Coupling between Distant Sites in Protein Families from Evolutionary Imprints: Illustrations Using Lattice Model</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1002/prot.22498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fprot.22498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19639636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2009&pages=823-831&author=Z.+Liuauthor=J.+Chenauthor=D.+Thirumalai&title=On+the+Accuracy+of+Inferring+Energetic+Coupling+between+Distant+Sites+in+Protein+Families+from+Evolutionary+Imprints%3A+Illustrations+Using+Lattice+Model&doi=10.1002%2Fprot.22498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">On the accuracy of inferring energetic coupling between distant sites in protein families from evolutionary imprints: Illustrations using lattice model</span></div><div class="casAuthors">Liu, Zhenxing; Chen, Jie; Thirumalai, D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-831</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">It is suspected that correlated motions among a subset of spatially sepd. residues drive conformational dynamics not only in multidomain but also in single domain proteins.  Sequence and structure-based methods have been proposed to det. covariation between two sites on a protein.  The statistical coupling anal. (SCA) that compares the changes in probability at two sites in a multiple sequence alignment (MSA) and a subset of the MSA has been used to infer the network of residues that encodes allosteric signals in protein families.  The structural perturbation method (SPM), that probes the response of a local perturbation at all other sites, has been used to probe the allostery wiring diagram in biol. machines and enzymes.  To assess the efficacy of the SCA, we used an exactly sol. two dimensional lattice model and performed double-mutant cycle (DMC) calcns. to predict the extent of phys. coupling between two sites.  The predictions of the SCA and the DMC results show that only residues that are in contact in the native state are accurately identified.  In addn., covariations among strongly interacting residues are most easily identified by the SCA.  These conclusions are consistent with the DMC expts. on the PDZ family.  Good correlation between the SCA and the DMC is only obtained by performing multiple expts. that vary the nature of amino acids at a given site.  In contrast, the energetic coupling found in expts. for the PDZ domain are recovered using the SPM.  We also predict, using the SPM, several residues that are coupled energetically.  Proteins 2009. © 2009 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLo4K1cqhX3bVg90H21EOLACvtfcHk0lgftoIF2W9_RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSru7nI&md5=27cfadf669f2d63961d2a8e9ad4bc8b8</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1002%2Fprot.22498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22498%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DThirumalai%26aufirst%3DD.%26atitle%3DOn%2520the%2520Accuracy%2520of%2520Inferring%2520Energetic%2520Coupling%2520between%2520Distant%2520Sites%2520in%2520Protein%2520Families%2520from%2520Evolutionary%2520Imprints%253A%2520Illustrations%2520Using%2520Lattice%2520Model%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D77%26spage%3D823%26epage%3D831%26doi%3D10.1002%2Fprot.22498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref169"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref169'); return false;" data-citation="" class="refNumLink">169</a></strong><div class="NLM_citation" id="cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livesay, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreth, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, A. A.</span></span> <span> </span><span class="NLM_article-title">A Critical Evaluation of Correlated Mutation Algorithms and Coevolution within Allosteric Mechanisms</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>796</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-334-9_21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2F978-1-61779-334-9_21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22052502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OgurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=796&publication_year=2012&pages=385-398&author=D.+R.+Livesayauthor=K.+E.+Krethauthor=A.+A.+Fodor&title=A+Critical+Evaluation+of+Correlated+Mutation+Algorithms+and+Coevolution+within+Allosteric+Mechanisms&doi=10.1007%2F978-1-61779-334-9_21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">A critical evaluation of correlated mutation algorithms and coevolution within allosteric mechanisms</span></div><div class="casAuthors">Livesay, Dennis R.; Kreth, Kyle E.; Fodor, Anthony A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">796</span>
        (<span class="NLM_cas:issue">Allostery</span>),
    <span class="NLM_cas:pages">385-398</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The notion of using the evolutionary history encoded within multiple sequence alignments to predict allosteric mechanisms is appealing.  In this approach, correlated mutations are expected to reflect coordinated changes that maintain intramol. coupling between residue pairs.  Despite much early fanfare, the general suitability of correlated mutations to predict allosteric couplings has not yet been established.  Lack of progress along these lines has been hindered by several algorithmic limitations including phylogenetic artifacts within alignments masking true covariance and the computational intractability of consideration of more than two correlated residues at a time.  Recent progress in algorithm development, however, has been substantial with a new generation of correlated mutation algorithms that have made fundamental progress toward solving these difficult problems.  Despite these encouraging results, there remains little evidence to suggest that the evolutionary constraints acting on allosteric couplings are sufficient to be recovered from multiple sequence alignments.  In this review, we argue that due to the exquisite sensitivity of protein dynamics, and hence that of allosteric mechanisms, the latter vary widely within protein families.  If it turns out to be generally true that even very similar homologs display a wide divergence of allosteric mechanisms, then even a perfect correlated mutation algorithm could not be reliably used as a general mechanism for discovery of allosteric pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqVeVwUhH4i7Vg90H21EOLACvtfcHk0lgftoIF2W9_RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OgurrL&md5=243db85ca72fac574b8c49493aee7320</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-334-9_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-334-9_21%26sid%3Dliteratum%253Aachs%26aulast%3DLivesay%26aufirst%3DD.%2BR.%26aulast%3DKreth%26aufirst%3DK.%2BE.%26aulast%3DFodor%26aufirst%3DA.%2BA.%26atitle%3DA%2520Critical%2520Evaluation%2520of%2520Correlated%2520Mutation%2520Algorithms%2520and%2520Coevolution%2520within%2520Allosteric%2520Mechanisms%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D796%26spage%3D385%26epage%3D398%26doi%3D10.1007%2F978-1-61779-334-9_21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref170"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref170'); return false;" data-citation="" class="refNumLink">170</a></strong><div class="NLM_citation" id="cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anishchenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovchinnikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamisetty, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Origins of Coevolution between Residues Distant in Protein 3D Structures</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">9122</span>– <span class="NLM_lpage">9127</span>, <span class="refDoi"> DOI: 10.1073/pnas.1702664114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1702664114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28784799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=9122-9127&author=I.+Anishchenkoauthor=S.+Ovchinnikovauthor=H.+Kamisettyauthor=D.+Baker&title=Origins+of+Coevolution+between+Residues+Distant+in+Protein+3D+Structures&doi=10.1073%2Fpnas.1702664114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Origins of coevolution between residues distant in protein 3D structures</span></div><div class="casAuthors">Anishchenko, Ivan; Ovchinnikov, Sergey; Kamisetty, Hetunandan; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">9122-9127</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Residue pairs that directly coevolve in protein families are generally close in protein 3-dimensional (3D) structures.  Here, we studied the exceptions to this general trend, directly coevolving residue pairs that are distant in protein structures, to det. the origins of evolutionary pressure on spatially distant residues and to understand the sources of error in contact-based structure prediction.  Over a set of 4000 protein families, we found that 25% of directly coevolving residue pairs were sepd. by >5 Å in protein structures and 3% by >15 Å.  The majority (91%) of directly coevolving residue pairs in the 5-15 Å range were found to be in contact in at least one homologous structure; these exceptions arose from structural variation in the family in the region contg. the residues.  Thirty-five percent of the exceptions at >15 Å were at homo-oligomeric interfaces, 19% arose from family structural variation, and 27% were in repeat proteins likely reflecting alignment errors.  Of the remaining long-range exceptions (<1% of the total no. of coupled pairs), many could be attributed to close interactions in an oligomeric state.  Overall, the results suggest that directly coevolving residue pairs not in repeat proteins are spatially proximal in at least one biol. relevant protein conformation within the family; we found little evidence for direct coupling between residues at spatially sepd. allosteric and functional sites or for increased direct coupling between residue pairs on putative allosteric pathways connecting them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHshwNC4Ovu7Vg90H21EOLACvtfcHk0lgftoIF2W9_RQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yku7fI&md5=dd465e6c3065f443ea86406a4f150be6</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1702664114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1702664114%26sid%3Dliteratum%253Aachs%26aulast%3DAnishchenko%26aufirst%3DI.%26aulast%3DOvchinnikov%26aufirst%3DS.%26aulast%3DKamisetty%26aufirst%3DH.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DOrigins%2520of%2520Coevolution%2520between%2520Residues%2520Distant%2520in%2520Protein%25203D%2520Structures%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D9122%26epage%3D9127%26doi%3D10.1073%2Fpnas.1702664114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref171"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref171'); return false;" data-citation="" class="refNumLink">171</a></strong><div class="NLM_citation" id="cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novinec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenč, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenarčič, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baici, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Allosteric Mechanism in the Cysteine Peptidase Cathepsin K Discovered by Computational Methods</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3287</span>, <span class="refDoi"> DOI: 10.1038/ncomms4287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms4287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24518821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvksVaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3287&author=M.+Novinecauthor=M.+Koren%C4%8Dauthor=A.+Caflischauthor=R.+Ranganathanauthor=B.+Lenar%C4%8Di%C4%8Dauthor=A.+Baici&title=A+Novel+Allosteric+Mechanism+in+the+Cysteine+Peptidase+Cathepsin+K+Discovered+by+Computational+Methods&doi=10.1038%2Fncomms4287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">A novel allosteric mechanism in the cysteine peptidase cathepsin K discovered by computational methods</span></div><div class="casAuthors">Novinec Marko; Korenc Matevz; Caflisch Amedeo; Baici Antonio; Ranganathan Rama; Lenarcic Brigita</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3287</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Allosteric modifiers have the potential to fine-tune enzyme activity.  Therefore, targeting allosteric sites is gaining increasing recognition as a strategy in drug design.  Here we report the use of computational methods for the discovery of the first small-molecule allosteric inhibitor of the collagenolytic cysteine peptidase cathepsin K, a major target for the treatment of osteoporosis.  The molecule NSC13345 is identified by high-throughput docking of compound libraries to surface sites on the peptidase that are connected to the active site by an evolutionarily conserved network of residues (protein sector).  The crystal structure of the complex shows that NSC13345 binds to a novel allosteric site on cathepsin K.  The compound acts as a hyperbolic mixed modifier in the presence of a synthetic substrate, it completely inhibits collagen degradation and has good selectivity for cathepsin K over related enzymes.  Altogether, these properties qualify our methodology and NSC13345 as promising candidates for allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJl-p6b4YuipHZzFRCXxVKfW6udTcc2eaobmw12UAzQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvksVaitA%253D%253D&md5=522c75841220139748cdf2b0f6cede07</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1038%2Fncomms4287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4287%26sid%3Dliteratum%253Aachs%26aulast%3DNovinec%26aufirst%3DM.%26aulast%3DKoren%25C4%258D%26aufirst%3DM.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DLenar%25C4%258Di%25C4%258D%26aufirst%3DB.%26aulast%3DBaici%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Allosteric%2520Mechanism%2520in%2520the%2520Cysteine%2520Peptidase%2520Cathepsin%2520K%2520Discovered%2520by%2520Computational%2520Methods%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3287%26doi%3D10.1038%2Fncomms4287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref172"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref172'); return false;" data-citation="" class="refNumLink">172</a></strong><div class="NLM_citation" id="cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">Automated Prediction of Ligand-Binding Sites in Proteins</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1002/prot.21645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fprot.21645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17910060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFyrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1506-1517&author=R.+Harrisauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+Prediction+of+Ligand-Binding+Sites+in+Proteins&doi=10.1002%2Fprot.21645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Automated prediction of ligand-binding sites in proteins</span></div><div class="casAuthors">Harris, Rodney; Olson, Arthur J.; Goodsell, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1506-1517</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The authors present a method, termed AutoLigand, for the prediction of ligand-binding sites in proteins of known structure.  The method searches the space surrounding the protein and finds the contiguous envelope with the specified vol. of atoms, which has the largest possible interaction energy with the protein.  It uses a full at. representation, with atom types for carbon, hydrogen, oxygen, nitrogen and sulfur (and others, if desired), and is designed to minimize the need for artificial geometry.  Testing on a set of 187 diverse protein-ligand complexes has shown that the method is successful in predicting the location and approx. vol. of the binding site in 73% of cases.  Addnl. testing was performed on a set of 96 protein-ligand complexes with crystallog. structures of apo and holo forms, and AutoLigand was able to predict the binding site in 80% of the apo structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwD6aCbA5coLVg90H21EOLACvtfcHk0ljKICAuP56qwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFyrtbk%253D&md5=a8fcf5e1d18f9315b4336573f71b1629</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1002%2Fprot.21645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21645%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520Prediction%2520of%2520Ligand-Binding%2520Sites%2520in%2520Proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D70%26spage%3D1506%26epage%3D1517%26doi%3D10.1002%2Fprot.21645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref173"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref173'); return false;" data-citation="" class="refNumLink">173</a></strong><div class="NLM_citation" id="cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelwijk, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosal, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">The Spatial Architecture of Protein Function and Adaptation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>491</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1038/nature11500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature11500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23041932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKisbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=491&publication_year=2012&pages=138-142&author=R.+N.+McLaughlinauthor=F.+J.+Poelwijkauthor=A.+Ramanauthor=W.+S.+Gosalauthor=R.+Ranganathan&title=The+Spatial+Architecture+of+Protein+Function+and+Adaptation&doi=10.1038%2Fnature11500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">The spatial architecture of protein function and adaptation</span></div><div class="casAuthors">McLaughlin, Richard N., Jr.; Poelwijk, Frank J.; Raman, Arjun; Gosal, Walraj S.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">491</span>
        (<span class="NLM_cas:issue">7422</span>),
    <span class="NLM_cas:pages">138-142</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Statistical anal. of protein evolution suggests a design for natural proteins in which sparse networks of coevolving amino acids (termed sectors) comprise the essence of three-dimensional structure and function.  However, proteins are also subject to pressures deriving from the dynamics of the evolutionary process itself-the ability to tolerate mutation and to be adaptive to changing selection pressures.  To understand the relationship of the sector architecture to these properties, we developed a high-throughput quant. method for a comprehensive single-mutation study in which every position is substituted individually to every other amino acid.  Using a PDZ domain (PSD95pdz3) model system, we show that sector positions are functionally sensitive to mutation, whereas non-sector positions are more tolerant to substitution.  In addn., we find that adaptation to a new binding specificity initiates exclusively through variation within sector residues.  A combination of just two sector mutations located near and away from the ligand-binding site suffices to switch the binding specificity of PSD95pdz3 quant. towards a class-switching ligand.  The localization of functional constraint and adaptive variation within the sector has important implications for understanding and engineering proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUbG0JzDZ9arVg90H21EOLACvtfcHk0ljKICAuP56qwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKisbbE&md5=c276f35ff757d6a1e1d286cfa9444cc2</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1038%2Fnature11500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11500%26sid%3Dliteratum%253Aachs%26aulast%3DMcLaughlin%26aufirst%3DR.%2BN.%26aulast%3DPoelwijk%26aufirst%3DF.%2BJ.%26aulast%3DRaman%26aufirst%3DA.%26aulast%3DGosal%26aufirst%3DW.%2BS.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DThe%2520Spatial%2520Architecture%2520of%2520Protein%2520Function%2520and%2520Adaptation%26jtitle%3DNature%26date%3D2012%26volume%3D491%26spage%3D138%26epage%3D142%26doi%3D10.1038%2Fnature11500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref174"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref174'); return false;" data-citation="" class="refNumLink">174</a></strong><div class="NLM_citation" id="cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockless, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Allosteric Determinants in Guanine Nucleotide-Binding Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">14445</span>– <span class="NLM_lpage">14450</span>, <span class="refDoi"> DOI: 10.1073/pnas.1835919100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1835919100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14623969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsFGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=14445-14450&author=M.+E.+Hatleyauthor=S.+W.+Locklessauthor=S.+K.+Gibsonauthor=A.+G.+Gilmanauthor=R.+Ranganathan&title=Allosteric+Determinants+in+Guanine+Nucleotide-Binding+Proteins&doi=10.1073%2Fpnas.1835919100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric determinants in guanine nucleotide-binding proteins</span></div><div class="casAuthors">Hatley, Mark E.; Lockless, Steve W.; Gibson, Scott K.; Gilman, Alfred G.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">14445-14450</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Members of the G protein superfamily contain nucleotide-dependent switches that dictate the specificity of their interactions with binding partners.  Using a sequence-based method termed statistical coupling anal. (SCA), we have attempted to identify the allosteric core of these proteins, the network of amino acid residues that couples the domains responsible for nucleotide binding and protein-protein interactions.  One-third of the 38 residues identified by SCA were mutated in the G protein Gsα, and the interactions of guanosine 5'-3-O-(thio)triphosphate- and GDP-bound mutant proteins were tested with both adenylyl cyclase (preferential binding to GTP- Gsα) and the G protein βγ subunit complex (preferential binding to GDP-Gsα).  A two-state allosteric model predicts that mutation of residues that control the equil. between GDP- and GTP-bound conformations of the protein will cause the ratio of affinities of these species for adenylyl cyclase and Gβγ to vary in a reciprocal fashion.  Obsd. results were consistent with this prediction.  The network of residues identified by the SCA appears to comprise a core allosteric mechanism conferring nucleotide-dependent switching; the specific features of different G protein family members are built on this core.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrusowIjSLwArVg90H21EOLACvtfcHk0ljKICAuP56qwA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsFGhsL4%253D&md5=1535d111d43c094c4e4c9c5cc074ab4f</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1835919100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1835919100%26sid%3Dliteratum%253Aachs%26aulast%3DHatley%26aufirst%3DM.%2BE.%26aulast%3DLockless%26aufirst%3DS.%2BW.%26aulast%3DGibson%26aufirst%3DS.%2BK.%26aulast%3DGilman%26aufirst%3DA.%2BG.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DAllosteric%2520Determinants%2520in%2520Guanine%2520Nucleotide-Binding%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D14445%26epage%3D14450%26doi%3D10.1073%2Fpnas.1835919100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref175"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref175'); return false;" data-citation="" class="refNumLink">175</a></strong><div class="NLM_citation" id="cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amezcua, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabi, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelliah, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Signal Transduction Pathway of TonB-Dependent Transporters</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1073/pnas.0609887104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0609887104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17197416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=513-518&author=A.+D.+Fergusonauthor=C.+A.+Amezcuaauthor=N.+M.+Halabiauthor=Y.+Chelliahauthor=M.+K.+Rosenauthor=R.+Ranganathanauthor=J.+Deisenhofer&title=Signal+Transduction+Pathway+of+TonB-Dependent+Transporters&doi=10.1073%2Fpnas.0609887104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Signal transduction pathway of TonB-dependent transporters</span></div><div class="casAuthors">Ferguson, Andrew D.; Amezcua, Carlos A.; Halabi, Najeeb M.; Chelliah, Yogarany; Rosen, Michael K.; Ranganathan, Rama; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">513-518</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transcription of the ferric citrate import system is regulated by ferric citrate binding to the outer membrane transporter FecA.  A signal indicating transporter occupancy is relayed across the outer membrane to energy-transducing and regulatory proteins embedded in the cytoplasmic membrane.  Because transcriptional activation is not coupled to ferric citrate import, an allosteric mechanism underlies this complex signaling mechanism.  Using evolution-based statistical anal. we have identified a sparse but structurally connected network of residues that links distant functional sites in FecA.  Functional analyses of these positions confirm their involvement in the mechanism that regulates transcriptional activation in response to ferric citrate binding at the cell surface.  This mechanism appears to be conserved and provides the structural basis for the allosteric signaling of TonB-dependent transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8WUEQZ3n0xrVg90H21EOLACvtfcHk0ljh_dSaZNB1Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVCnuw%253D%253D&md5=8e833d52d0b582bd84097b7c2465ad4b</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609887104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609887104%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DAmezcua%26aufirst%3DC.%2BA.%26aulast%3DHalabi%26aufirst%3DN.%2BM.%26aulast%3DChelliah%26aufirst%3DY.%26aulast%3DRosen%26aufirst%3DM.%2BK.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DSignal%2520Transduction%2520Pathway%2520of%2520TonB-Dependent%2520Transporters%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D513%26epage%3D518%26doi%3D10.1073%2Fpnas.0609887104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref176"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref176'); return false;" data-citation="" class="refNumLink">176</a></strong><div class="NLM_citation" id="cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. X.</span></span> <span> </span><span class="NLM_article-title">Protein Allostery and Conformational Dynamics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6503</span>– <span class="NLM_lpage">6515</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00590</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00590" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6503-6515&author=J.+Guoauthor=H.+X.+Zhou&title=Protein+Allostery+and+Conformational+Dynamics&doi=10.1021%2Facs.chemrev.5b00590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Allostery and Conformational Dynamics</span></div><div class="casAuthors">Guo, Jingjing; Zhou, Huan-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6503-6515</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The functions of many proteins are regulated through allostery, whereby effector binding at a distal site changes the functional activity (e.g., substrate binding affinity or catalytic efficiency) at the active site.  Most allosteric studies have focused on thermodn. properties, in particular, substrate binding affinity.  Changes in substrate binding affinity by allosteric effectors have generally been thought to be mediated by conformational transitions of the proteins or, alternatively, by changes in the broadness of the free energy basin of the protein conformational state without shifting the basin min. position.  When effector binding changes the free energy landscape of a protein in conformational space, the change affects not only thermodn. properties but also dynamic properties, including the amplitudes of motions on different time scales and rates of conformational transitions.  Here we assess the roles of conformational dynamics in allosteric regulation.  Two cases are highlighted where NMR spectroscopy and mol. dynamics simulation have been used as complementary approaches to identify residues possibly involved in allosteric communication.  Perspectives on contentious issues, for example, the relationship between picosecond-nanosecond local and microsecond-millisecond conformational exchange dynamics, are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVLHVu6pyerrVg90H21EOLACvtfcHk0ljh_dSaZNB1Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVKjsr8%253D&md5=8a148ea143d0e31927cfc56745fd13bd</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00590%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DProtein%2520Allostery%2520and%2520Conformational%2520Dynamics%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6503%26epage%3D6515%26doi%3D10.1021%2Facs.chemrev.5b00590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref177"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref177'); return false;" data-citation="" class="refNumLink">177</a></strong><div class="NLM_citation" id="cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perilla, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations of Large Macromolecular Complexes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25845770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFemsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=64-74&author=J.+R.+Perillaauthor=B.+C.+Gohauthor=C.+K.+Cassidyauthor=B.+Liuauthor=R.+C.+Bernardiauthor=T.+Rudackauthor=H.+Yuauthor=Z.+Wuauthor=K.+Schulten&title=Molecular+Dynamics+Simulations+of+Large+Macromolecular+Complexes&doi=10.1016%2Fj.sbi.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of large macromolecular complexes</span></div><div class="casAuthors">Perilla, Juan R.; Goh, Boon Chong; Cassidy, C. Keith; Liu, Bo; Bernardi, Rafael C.; Rudack, Till; Yu, Hang; Wu, Zhe; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64-74</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Connecting dynamics to structural data from diverse exptl. sources, mol. dynamics simulations permit the exploration of biol. phenomena in unparalleled detail.  Advances in simulations are moving the at. resoln. descriptions of biol. systems into the million-to-billion atom regime, in which numerous cell functions reside.  In this opinion, the authors review the progress, driven by large-scale mol. dynamics simulations, in the study of viruses, ribosomes, bioenergetic systems, and other diverse applications.  These examples highlight the utility of mol. dynamics simulations in the crit. task of relating at. detail to the function of supramol. complexes, a task that cannot be achieved by smaller-scale simulations or existing exptl. approaches alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIz83au-c0jrVg90H21EOLACvtfcHk0livbKczW0o5yQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFemsb4%253D&md5=8fb981e606aabacf4b75ede88434ec6a</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DPerilla%26aufirst%3DJ.%2BR.%26aulast%3DGoh%26aufirst%3DB.%2BC.%26aulast%3DCassidy%26aufirst%3DC.%2BK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBernardi%26aufirst%3DR.%2BC.%26aulast%3DRudack%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520of%2520Large%2520Macromolecular%2520Complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D31%26spage%3D64%26epage%3D74%26doi%3D10.1016%2Fj.sbi.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref178"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref178'); return false;" data-citation="" class="refNumLink">178</a></strong><div class="NLM_citation" id="cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elber, R.</span></span> <span> </span><span class="NLM_article-title">Simulations of Allosteric Transitions</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2011.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21333527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=167-172&author=R.+Elber&title=Simulations+of+Allosteric+Transitions&doi=10.1016%2Fj.sbi.2011.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Simulations of allosteric transitions</span></div><div class="casAuthors">Elber, Ron</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-172</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric transitions are one of the subtler mechanisms used by nature to fine tune protein activity.  Effector binding to a specific site on the protein surface induces significant activity change, and initiates a conformational transition that frequently includes domain motions and is very large.  From a theor. and biophys. perspective, 2 problems are particularly intriguing.  The 1st is the way in which a launching signal, which is spatially confined and includes only a few interacting atoms, is propagated to a large-scale conformational transition frequently seen in allosteric transitions.  Hence, there is the question of how a small perturbation is magnified to yield motions of thousands of atoms.  The 2nd puzzle is of focus, coherence, and efficiency.  The impact of the binding of the effector is sometimes extended over tens of angstroms.  How the signal is transmitted and kept significant over such large distances in the 'noisy' mol. environment is another major direction of investigation.  Here, the author has examd. different theor. and computational attempts to solve the questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvNcZY9GuJsLVg90H21EOLACvtfcHk0livkWsHiUmNWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrc%253D&md5=1aed7f77805946b84d66e7e35f82976a</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DElber%26aufirst%3DR.%26atitle%3DSimulations%2520of%2520Allosteric%2520Transitions%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D167%26epage%3D172%26doi%3D10.1016%2Fj.sbi.2011.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref179"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref179'); return false;" data-citation="" class="refNumLink">179</a></strong><div class="NLM_citation" id="cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations and Dynamic Network Analysis Reveal the Allosteric Unbinding of Monobody to H-Ras Triggered by R135K Mutation</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2249</span>, <span class="refDoi"> DOI: 10.3390/ijms18112249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms18112249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOgtLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=2249&author=D.+Niauthor=K.+Songauthor=J.+Zhangauthor=S.+Lu&title=Molecular+Dynamics+Simulations+and+Dynamic+Network+Analysis+Reveal+the+Allosteric+Unbinding+of+Monobody+to+H-Ras+Triggered+by+R135K+Mutation&doi=10.3390%2Fijms18112249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and dynamic network analysis reveal the allosteric unbinding of monobody to H-Ras triggered by R135K mutation</span></div><div class="casAuthors">Ni, Duan; Song, Kun; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2249/1-2249/16</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Ras proteins, as small GTPases, mediate cell proliferation, survival and differentiation.  Ras mutations have been assocd. with a broad spectrum of human cancers and thus targeting Ras represents a potential way forward for cancer therapy.  A recently reported monobody NS1 allosterically disrupts the Ras-mediated signaling pathway, but its efficacy is reduced by R135K mutation in H-Ras.  However, the detailed mechanism is unresolved.  Here, using mol. dynamics (MD) simulations and dynamic network anal., we explored the mol. mechanism for the unbinding of NS1 to H-Ras and shed light on the underlying allosteric network in H-Ras.  MD simulations revealed that the overall structures of the two complexes did not change significantly, but the H-Ras-NS1 interface underwent significant conformational alteration in the mutant Binding free energy anal. showed that NS1 binding was unfavored after R135K mutation, which resulted in the unfavorable binding of NS1.  Furthermore, the crit. residues on H-Ras responsible for the loss of binding of NS1 were identified.  Importantly, the allosteric networks for these important residues were revealed, which yielded a novel insight into the allosteric regulatory mechanism of H-Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhfQu8bG_CGrVg90H21EOLACvtfcHk0livkWsHiUmNWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOgtLzN&md5=61ae175456844dedba08536f410e343d</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.3390%2Fijms18112249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18112249%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520and%2520Dynamic%2520Network%2520Analysis%2520Reveal%2520the%2520Allosteric%2520Unbinding%2520of%2520Monobody%2520to%2520H-Ras%2520Triggered%2520by%2520R135K%2520Mutation%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D2249%26doi%3D10.3390%2Fijms18112249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref180"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref180'); return false;" data-citation="" class="refNumLink">180</a></strong><div class="NLM_citation" id="cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Computational Insights into the Interactions between Calmodulin and the c/nSH2 Domains of p85α Regulatory Subunit of PI3Kα: Implication for PI3Kα Activation by Calmodulin</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">151</span>, <span class="refDoi"> DOI: 10.3390/ijms19010151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms19010151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1ahtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=151&author=D.+Niauthor=D.+Liuauthor=J.+Zhangauthor=S.+Lu&title=Computational+Insights+into+the+Interactions+between+Calmodulin+and+the+c%2FnSH2+Domains+of+p85%CE%B1+Regulatory+Subunit+of+PI3K%CE%B1%3A+Implication+for+PI3K%CE%B1+Activation+by+Calmodulin&doi=10.3390%2Fijms19010151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Computational insights into the interactions between calmodulin and the c/nsh2 domains of p85α regulatory subunit of PI3K : implication for PI3Kα activation by Calmodulin</span></div><div class="casAuthors">Ni, Duan; Liu, Dingyu; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151/1-151/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Calmodulin (CaM) and phosphatidylinositide-3 kinase (PI3Kα) are well known for their multiple roles in a series of intracellular signaling pathways and in the progression of several human cancers.  Crosstalk between CaM and PI3Kα has been an area of intensive research.  Recent expts. have shown that in adenocarcinoma, K-Ras4B is involved in the CaM-PI3Kα crosstalk.  Based on exptl. results, we have recently put forward a hypothesis that the coordination of CaM and PI3Kα with K-Ras4B forms a CaM-PI3Kα-K-Ras4B ternary complex, which leads to the formation of pancreatic ductal adenocarcinoma.  However, the mechanism for the CaM-PI3Kα crosstalk is unresolved.  Based on mol. modeling and mol. dynamics simulations, here we explored the potential interactions between CaM and the c/nSH2 domains of p85α subunit of PI3Kα.  We demonstrated that CaM can interact with the c/nSH2 domains and the interaction details were unraveled.  Moreover, the possible modes for the CaM-cSH2 and CaM-nSH2 interactions were uncovered and we used them to construct a complete CaM-PI3Kα complex model.  The structural model of CaM-PI3Kα interaction not only offers a support for our previous ternary complex hypothesis, but also is useful for drug design targeted at CaM-PI3Kα protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUEQ7a-94mybVg90H21EOLACvtfcHk0livkWsHiUmNWg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1ahtLvM&md5=bf7d6c3c24a130a8cf596aa4bcee7fd4</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.3390%2Fijms19010151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19010151%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DComputational%2520Insights%2520into%2520the%2520Interactions%2520between%2520Calmodulin%2520and%2520the%2520c%252FnSH2%2520Domains%2520of%2520p85%25CE%25B1%2520Regulatory%2520Subunit%2520of%2520PI3K%25CE%25B1%253A%2520Implication%2520for%2520PI3K%25CE%25B1%2520Activation%2520by%2520Calmodulin%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D151%26doi%3D10.3390%2Fijms19010151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref181"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref181'); return false;" data-citation="" class="refNumLink">181</a></strong><div class="NLM_citation" id="cit181"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span> <span> </span><span class="NLM_article-title">Protein Binding Pocket Dynamics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1ansrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=809-815&author=A.+Stankauthor=D.+B.+Kokhauthor=J.+C.+Fullerauthor=R.+C.+Wade&title=Protein+Binding+Pocket+Dynamics&doi=10.1021%2Facs.accounts.5b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Binding Pocket Dynamics</span></div><div class="casAuthors">Stank, Antonia; Kokh, Daria B.; Fuller, Jonathan C.; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">809-815</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamics of protein binding pockets are crucial for their interaction specificity.  Structural flexibility allows proteins to adapt to their individual mol. binding partners and facilitates the binding process.  This implies the necessity to consider protein internal motion in detg. and predicting binding properties and in designing new binders.  Although accounting for protein dynamics presents a challenge for computational approaches, it expands the structural and physicochem. space for compd. design and thus offers the prospect of improved binding specificity and selectivity.A cavity on the surface or in the interior of a protein that possesses suitable properties for binding a ligand is usually referred to as a binding pocket.  The set of amino acid residues around a binding pocket dets. its physicochem. characteristics and, together with its shape and location in a protein, defines its functionality.  Residues outside the binding site can also have a long-range effect on the properties of the binding pocket.  Cavities with similar functionalities are often conserved across protein families.  For example, enzyme active sites are usually concave surfaces that present amino acid residues in a suitable configuration for binding low mol. wt. compds.  Macromol. binding pockets, on the other hand, are located on the protein surface and are often shallower.  The mobility of proteins allows the opening, closing, and adaptation of binding pockets to regulate binding processes and specific protein functionalities.  For example, channels and tunnels can exist permanently or transiently to transport compds. to and from a binding site.  The influence of protein flexibility on binding pockets can vary from small changes to an already existent pocket to the formation of a completely new pocket.  Here, we review recent developments in computational methods to detect and define binding pockets and to study pocket dynamics.  We introduce five different classes of protein pocket dynamics: (1) appearance/disappearance of a subpocket in an existing pocket; (2) appearance/disappearance of an adjacent pocket on the protein surface in the direct vicinity of an already existing pocket; (3) pocket breathing, which may be caused by side-chain fluctuations or backbone or interdomain vibrational motion; (4) opening/closing of a channel or tunnel, connecting a pocket inside the protein with solvent, including lid motion; and (5) the appearance/disappearance of an allosteric pocket at a site on a protein distinct from an already existing pocket with binding of a ligand to the allosteric binding site affecting the original pocket.We suggest that the class of pocket dynamics, as well as the type and extent of protein motion affecting the binding pocket, should be factors considered in choosing the most appropriate computational approach to study a given binding pocket.  Furthermore, we examine the relationship between pocket dynamics classes and induced fit, conformational selection, and gating models of ligand binding on binding kinetics and thermodn.  We discuss the implications of protein binding pocket dynamics for drug design and conclude with potential future directions for computational anal. of protein binding pocket dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ6TUcwjXcebVg90H21EOLACvtfcHk0ljHiXDG7H7hTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1ansrs%253D&md5=5c16ef089e87f13bf004f0935d1e4f35</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00516%26sid%3Dliteratum%253Aachs%26aulast%3DStank%26aufirst%3DA.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DFuller%26aufirst%3DJ.%2BC.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DProtein%2520Binding%2520Pocket%2520Dynamics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D809%26epage%3D815%26doi%3D10.1021%2Facs.accounts.5b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref182"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref182'); return false;" data-citation="" class="refNumLink">182</a></strong><div class="NLM_citation" id="cit182"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcourt, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Modulation of a G-Protein-Coupled Receptor by Allosteric Drugs</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+Basis+for+Modulation+of+a+G-Protein-Coupled+Receptor+by+Allosteric+Drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0ljHiXDG7H7hTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520Basis%2520for%2520Modulation%2520of%2520a%2520G-Protein-Coupled%2520Receptor%2520by%2520Allosteric%2520Drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref183"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref183'); return false;" data-citation="" class="refNumLink">183</a></strong><div class="NLM_citation" id="cit183"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hertig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorraca, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span> <span> </span><span class="NLM_article-title">Revealing Atomic-Level Mechanisms of Protein Allostery with Molecular Dynamics Simulations</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1004746</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27285999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyit7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=S.+Hertigauthor=N.+R.+Latorracaauthor=R.+O.+Dror&title=Revealing+Atomic-Level+Mechanisms+of+Protein+Allostery+with+Molecular+Dynamics+Simulations&doi=10.1371%2Fjournal.pcbi.1004746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">Revealing atomic-level mechanisms of protein allostery with molecular dynamics simulations</span></div><div class="casAuthors">Hertig, Samuel; Latorraca, Naomi R.; Dror, Ron O.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1004746/1-e1004746/16</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations have become a powerful and popular method for the study of protein allostery, the widespread phenomenon in which a stimulus at one site on a protein influences the properties of another site on the protein.  By capturing the motions of a protein's constituent atoms, simulations can enable the discovery of allosteric binding sites and the detn. of the mechanistic basis for allostery.  These results can provide a foundation for applications including rational drug design and protein engineering.  Here, we provide an introduction to the investigation of protein allostery using mol. dynamics simulation.  We emphasize the importance of designing simulations that include appropriate perturbations to the mol. system, such as the addn. or removal of ligands or the application of mech. force.  We also demonstrate how the bidirectional nature of allostery- the fact that the two sites involved influence one another in a sym. manner- can facilitate such investigations.  Through a series of case studies, we illustrate how these concepts have been used to reveal the structural basis for allostery in several proteins and protein complexes of biol. and pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobTyr-sWnXWLVg90H21EOLACvtfcHk0ljHiXDG7H7hTQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyit7nI&md5=40178de113d54ac2c453d372e2dd5bfb</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004746%26sid%3Dliteratum%253Aachs%26aulast%3DHertig%26aufirst%3DS.%26aulast%3DLatorraca%26aufirst%3DN.%2BR.%26aulast%3DDror%26aufirst%3DR.%2BO.%26atitle%3DRevealing%2520Atomic-Level%2520Mechanisms%2520of%2520Protein%2520Allostery%2520with%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.pcbi.1004746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref184"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref184'); return false;" data-citation="" class="refNumLink">184</a></strong><div class="NLM_citation" id="cit184"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaponenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">GTP Binding and Oncogenic Mutations May Attenuate Hypervariable Region (HVR)-Catalytic Domain Interactions in Small GTPase K-Ras4B, Exposing the Effector Binding Site</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28887</span>– <span class="NLM_lpage">28900</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.664755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.M115.664755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26453300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWqtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28887-28900&author=S.+Luauthor=A.+Banerjeeauthor=H.+Jangauthor=J.+Zhangauthor=V.+Gaponenkoauthor=R.+Nussinov&title=GTP+Binding+and+Oncogenic+Mutations+May+Attenuate+Hypervariable+Region+%28HVR%29-Catalytic+Domain+Interactions+in+Small+GTPase+K-Ras4B%2C+Exposing+the+Effector+Binding+Site&doi=10.1074%2Fjbc.M115.664755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">GTP Binding and Oncogenic Mutations May Attenuate Hypervariable Region (HVR)-Catalytic Domain Interactions in Small GTPase K-Ras4B, Exposing the Effector Binding Site</span></div><div class="casAuthors">Lu, Shaoyong; Banerjee, Avik; Jang, Hyunbum; Zhang, Jian; Gaponenko, Vadim; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">28887-28900</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">K-Ras4B, a frequently mutated oncogene in cancer, plays an essential role in cell growth, differentiation, and survival.  Its C-terminal membrane-assocd. hypervariable region (HVR) is required for full biol. activity.  In the active GTP-bound state, the HVR interacts with acidic plasma membrane (PM) headgroups, whereas the farnesyl anchors in the membrane; in the inactive GDP-bound state, the HVR may interact with both the PM and the catalytic domain at the effector binding region, obstructing signaling and nucleotide exchange.  Here, using mol. dynamics simulations and NMR, we aim to figure out the effects of nucleotides (GTP and GDP) and frequent (G12C, G12D, G12V, G13D, and Q61H) and infrequent (E37K and R164Q) oncogenic mutations on full-length K-Ras4B.  The mutations are away from or directly at the HVR switch I/effector binding site.  Our results suggest that full-length wild-type GDP-bound K-Ras4B (K-Ras4BWT-GDP) is in an intrinsically autoinhibited state via tight HVR-catalytic domain interactions.  The looser assocn. in K-Ras4BWT-GTP may release the HVR.  Some of the oncogenic mutations weaken the HVR-catalytic domain assocn. in the K-Ras4B-GDP/-GTP bound states, which may facilitate the HVR dissocn. in a nucleotide-independent manner, thereby up-regulating oncogenic Ras signaling.  Thus, our results suggest that mutations can exert their effects in more than one way, abolishing GTP hydrolysis and facilitating effector binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPP0csIe-DcLVg90H21EOLACvtfcHk0li-wXbU3AQuOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWqtbvK&md5=4046eed0caa6a965930b6c293d90ed0e</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.664755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.664755%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGaponenko%26aufirst%3DV.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DGTP%2520Binding%2520and%2520Oncogenic%2520Mutations%2520May%2520Attenuate%2520Hypervariable%2520Region%2520%2528HVR%2529-Catalytic%2520Domain%2520Interactions%2520in%2520Small%2520GTPase%2520K-Ras4B%252C%2520Exposing%2520the%2520Effector%2520Binding%2520Site%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28887%26epage%3D28900%26doi%3D10.1074%2Fjbc.M115.664755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref185"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref185'); return false;" data-citation="" class="refNumLink">185</a></strong><div class="NLM_citation" id="cit185"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Activation Pathway of Src Kinase Reveals Intermediate States as Targets for Drug Design</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3397</span>, <span class="refDoi"> DOI: 10.1038/ncomms4397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms4397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24584478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvpsF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3397&author=D.+Shuklaauthor=Y.+Mengauthor=B.+Rouxauthor=V.+S.+Pande&title=Activation+Pathway+of+Src+Kinase+Reveals+Intermediate+States+as+Targets+for+Drug+Design&doi=10.1038%2Fncomms4397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Activation pathway of Src kinase reveals intermediate states as targets for drug design</span></div><div class="casAuthors">Shukla Diwakar; Pande Vijay S; Meng Yilin; Roux Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3397</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Unregulated activation of Src kinases leads to aberrant signalling, uncontrolled growth and differentiation of cancerous cells.  Reaching a complete mechanistic understanding of large-scale conformational transformations underlying the activation of kinases could greatly help in the development of therapeutic drugs for the treatment of these pathologies.  In principle, the nature of conformational transition could be modelled in silico via atomistic molecular dynamics simulations, although this is very challenging because of the long activation timescales.  Here we employ a computational paradigm that couples transition pathway techniques and Markov state model-based massively distributed simulations for mapping the conformational landscape of c-src tyrosine kinase.  The computations provide the thermodynamics and kinetics of kinase activation for the first time, and help identify key structural intermediates.  Furthermore, the presence of a novel allosteric site in an intermediate state of c-src that could be potentially used for drug design is predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2z_2F1JvAAYvKgEt20y1XfW6udTcc2eaHkHyNvSJPUbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvpsF2huw%253D%253D&md5=f37ef7a345fdf37b69a83c677ef3d851</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1038%2Fncomms4397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4397%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DActivation%2520Pathway%2520of%2520Src%2520Kinase%2520Reveals%2520Intermediate%2520States%2520as%2520Targets%2520for%2520Drug%2520Design%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3397%26doi%3D10.1038%2Fncomms4397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref186"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref186'); return false;" data-citation="" class="refNumLink">186</a></strong><div class="NLM_citation" id="cit186"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubbers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkaraj, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potential Allosteric Ligand Binding Site in CDK2</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1021/cb100410m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100410m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=492-501&author=S.+Betziauthor=R.+Alamauthor=M.+Martinauthor=D.+J.+Lubbersauthor=H.+Hanauthor=S.+R.+Jakkarajauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Discovery+of+a+Potential+Allosteric+Ligand+Binding+Site+in+CDK2&doi=10.1021%2Fcb100410m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potential Allosteric Ligand Binding Site in CDK2</span></div><div class="casAuthors">Betzi, Stephane; Alam, Riazul; Martin, Mathew; Lubbers, Donna J.; Han, Huijong; Jakkaraj, Sudhakar R.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-501</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are key regulatory enzymes in cell cycle progression and transcription.  Aberrant activity of CDKs has been implicated in a no. of medical conditions, and numerous small mol. CDK inhibitors have been reported as potential drug leads.  However, these inhibitors exclusively bind to the ATP site, which is largely conserved among protein kinases, and clin. trials have not resulted in viable drug candidates, attributed in part to the lack of target selectivity.  CDKs are unique among protein kinases, as their functionality strictly depends on assocn. with their partner proteins, the cyclins.  In an effort to identify potential target sites for disruption of the CDK-cyclin interaction, we probed the extrinsic fluorophore 8-anilino-1-naphthalene sulfonate (ANS) with human CDK2 and cyclin A using fluorescence spectroscopy and protein crystallog.  ANS interacts with free CDK2 in a satn.-dependent manner with an apparent Kd of 37 μM, and cyclin A displaced ANS from CDK2 with an EC50 value of 0.6 μM.  Co-crystal structures with ANS alone and in ternary complex with ATP site-directed inhibitors revealed two ANS mols. bound adjacent to one another, away from the ATP site, in a large pocket that extends from the DFG region above the C-helix.  Binding of ANS is accompanied by substantial structural changes in CDK2, resulting in a C-helix conformation that is incompatible for cyclin A assocn.  These findings indicate the potential of the ANS binding pocket as a new target site for allosteric inhibitors disrupting the interaction of CDKs and cyclins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGlhg8YO4Yl7Vg90H21EOLACvtfcHk0li-wXbU3AQuOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gisb4%253D&md5=893aed13b1638f7626fbb4851014fe6e</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1021%2Fcb100410m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100410m%26sid%3Dliteratum%253Aachs%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DM.%26aulast%3DLubbers%26aufirst%3DD.%2BJ.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DJakkaraj%26aufirst%3DS.%2BR.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520a%2520Potential%2520Allosteric%2520Ligand%2520Binding%2520Site%2520in%2520CDK2%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D492%26epage%3D501%26doi%3D10.1021%2Fcb100410m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref187"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref187'); return false;" data-citation="" class="refNumLink">187</a></strong><div class="NLM_citation" id="cit187"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sultan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Towards Simple Kinetic Models of Functional Dynamics for a Kinase Subfamily</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0077-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fs41557-018-0077-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29988151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=903-909&author=M.+M.+Sultanauthor=G.+Kissauthor=V.+S.+Pande&title=Towards+Simple+Kinetic+Models+of+Functional+Dynamics+for+a+Kinase+Subfamily&doi=10.1038%2Fs41557-018-0077-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Towards simple kinetic models of functional dynamics for a kinase subfamily</span></div><div class="casAuthors">Sultan, Mohammad M.; Kiss, Gert; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">903-909</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Kinases are ubiquitous enzymes involved in the regulation of crit. cellular pathways.  However, in silico modeling of the conformational ensembles of these enzymes is difficult due to inherent limitations and the cost of computational approaches.  Recent algorithmic advances combined with homol. modeling and parallel simulations have enabled researchers to address this computational sampling bottleneck.  Here, we present the results of mol. dynamics studies for seven Src family kinase (SFK) members: Fyn, Lyn, Lck, Hck, Fgr, Yes and Blk.  We present a sequence invariant extension to Markov state models, which allows us to quant. compare the structural ensembles of the seven kinases.  Our findings indicate that in the absence of their regulatory partners, SFK members have similar in silico dynamics with active state populations ranging from 4 to 40% and activation timescales in the hundreds of microseconds.  Furthermore, we observe several potentially druggable intermediate states, including a pocket next to the ATP-binding site that could potentially be targeted via a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMN4rFMADt6bVg90H21EOLACvtfcHk0liqE4kmpsPEIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWnsrjM&md5=2f5e46697674a52b248e6d09202312f3</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0077-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0077-9%26sid%3Dliteratum%253Aachs%26aulast%3DSultan%26aufirst%3DM.%2BM.%26aulast%3DKiss%26aufirst%3DG.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DTowards%2520Simple%2520Kinetic%2520Models%2520of%2520Functional%2520Dynamics%2520for%2520a%2520Kinase%2520Subfamily%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D903%26epage%3D909%26doi%3D10.1038%2Fs41557-018-0077-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref188"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref188'); return false;" data-citation="" class="refNumLink">188</a></strong><div class="NLM_citation" id="cit188"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Husic, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Markov State Models: From an Art to a Science</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2386</span>– <span class="NLM_lpage">2396</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b12191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b12191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2386-2396&author=B.+E.+Husicauthor=V.+S.+Pande&title=Markov+State+Models%3A+From+an+Art+to+a+Science&doi=10.1021%2Fjacs.7b12191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models: From an Art to a Science</span></div><div class="casAuthors">Husic, Brooke E.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2386-2396</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov state models (MSMs) are a powerful framework for analyzing dynamical systems, such as mol. dynamics (MD) simulations, that have gained widespread use over the past several decades.  This perspective offers an overview of the MSM field to date, presented for a general audience as a timeline of key developments in the field.  We sequentially address early studies that motivated the method, canonical papers that established the use of MSMs for MD anal., and subsequent advances in software and anal. protocols.  The derivation of a variational principle for MSMs in 2013 signified a turning point from expertise-driving MSM building to a systematic, objective protocol.  The variational approach, combined with best practices for model selection and open-source software, enabled a wide range of MSM anal. for applications such as protein folding and allostery, ligand binding, and protein-protein assocn.  To conclude, the current frontiers of methods development are highlighted, as well as exciting applications in exptl. design and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo43eE2dHXoHrVg90H21EOLACvtfcHk0liqE4kmpsPEIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D&md5=f00943acedec985c4d21abd86ecfce4a</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b12191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b12191%26sid%3Dliteratum%253Aachs%26aulast%3DHusic%26aufirst%3DB.%2BE.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520State%2520Models%253A%2520From%2520an%2520Art%2520to%2520a%2520Science%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2386%26epage%3D2396%26doi%3D10.1021%2Fjacs.7b12191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref189"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref189'); return false;" data-citation="" class="refNumLink">189</a></strong><div class="NLM_citation" id="cit189"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fronik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaiser, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejer
Pedersen, D.</span></span> <span> </span><span class="NLM_article-title">Bitopic Ligands and Metastable Binding Sites: Opportunities for G Protein-Coupled Receptor (GPCR) Medicinal Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4126</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01601</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01601" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVyktbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4126-4134&author=P.+Fronikauthor=B.+I.+Gaiserauthor=D.+Sejer%0APedersen&title=Bitopic+Ligands+and+Metastable+Binding+Sites%3A+Opportunities+for+G+Protein-Coupled+Receptor+%28GPCR%29+Medicinal+Chemistry&doi=10.1021%2Facs.jmedchem.6b01601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Bitopic Ligands and Metastable Binding Sites: Opportunities for G Protein-Coupled Receptor (GPCR) Medicinal Chemistry</span></div><div class="casAuthors">Fronik, Philipp; Gaiser, Birgit I.; Sejer Pedersen, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4126-4134</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) belong to a large superfamily of membrane receptors mediating a variety of physiol. functions.  As such they are attractive targets for drug therapy.  However, it remains a challenge to develop subtype selective GPCR ligands, due to the high conservation of orthosteric binding sites.  Bitopic ligands have been employed to address the selectivity problem by combining (linking) an orthosteric ligand with an allosteric modulator, theor. leading to high-affinity subtype selective ligands.  However, it remains a challenge to identify suitable allosteric binding sites.  Computational studies on ligand binding to GPCRs have revealed transient, low-affinity binding sites, termed metastable binding sites.  Metastable binding sites may provide a new source of allosteric binding sites that could be exploited in the design of bitopic ligands.  Unlike the bitopic ligands that have been reported to date this type of bitopic ligands would be composed of two identical pharmacophores.  Herein, the authors outline the concept of bitopic ligands, review metastable binding sites and discuss their potential as a new source of allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpihAXt959S37Vg90H21EOLACvtfcHk0liqE4kmpsPEIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVyktbw%253D&md5=a20875145b65a948c772e988ff1217bc</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01601%26sid%3Dliteratum%253Aachs%26aulast%3DFronik%26aufirst%3DP.%26aulast%3DGaiser%26aufirst%3DB.%2BI.%26aulast%3DSejer%2BPedersen%26aufirst%3DD.%26atitle%3DBitopic%2520Ligands%2520and%2520Metastable%2520Binding%2520Sites%253A%2520Opportunities%2520for%2520G%2520Protein-Coupled%2520Receptor%2520%2528GPCR%2529%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4126%26epage%3D4134%26doi%3D10.1021%2Facs.jmedchem.6b01601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref190"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref190'); return false;" data-citation="" class="refNumLink">190</a></strong><div class="NLM_citation" id="cit190"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Multiple Hidden Allosteric Sites by Combining Markov State Models and Experiments</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">2734</span>– <span class="NLM_lpage">2739</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417811112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1417811112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25730859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=2734-2739&author=G.+R.+Bowmanauthor=E.+R.+Bolinauthor=K.+M.+Hartauthor=B.+C.+Maguireauthor=S.+Marqusee&title=Discovery+of+Multiple+Hidden+Allosteric+Sites+by+Combining+Markov+State+Models+and+Experiments&doi=10.1073%2Fpnas.1417811112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of multiple hidden allosteric sites by combining Markov state models and experiments</span></div><div class="casAuthors">Bowman, Gregory R.; Bolin, Eric R.; Hart, Kathryn M.; Maguire, Brendan C.; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2734-2739</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The discovery of drug-like mols. that bind pockets in proteins that are not present in crystallog. structures yet exert allosteric control over activity has generated great interest in designing pharmaceuticals that exploit allosteric effects.  However, there have only been a small no. of successes, so the therapeutic potential of these pockets--called hidden allosteric sites--remains unclear.  One challenge for assessing their utility is that rational drug design approaches require foreknowledge of the target site, but most hidden allosteric sites are only discovered when a small mol. is found to stabilize them.  The authors present a means of decoupling the identification of hidden allosteric sites from the discovery of drugs that bind them by drawing on new developments in Markov state modeling that provide unprecedented access to microsecond- to millisecond-timescale fluctuations of a protein's structure.  Visualizing these fluctuations allows the authors to identify potential hidden allosteric sites, which the authors then test via thiol labeling expts.  Application of these methods reveals multiple hidden allosteric sites in an important antibiotic target-TEM-1 β-lactamase.  This result supports the hypothesis that there are many as yet undiscovered hidden allosteric sites and suggests the authors' methodol. can identify such sites, providing a starting point for future drug design efforts.  More generally, the authors' results demonstrate the power of using Markov state models to guide expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY4EqjsensDLVg90H21EOLACvtfcHk0liqE4kmpsPEIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D&md5=f20db19a2f0f13679bc64f4616c856e7</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417811112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417811112%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DBolin%26aufirst%3DE.%2BR.%26aulast%3DHart%26aufirst%3DK.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%2BC.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Multiple%2520Hidden%2520Allosteric%2520Sites%2520by%2520Combining%2520Markov%2520State%2520Models%2520and%2520Experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D2734%26epage%3D2739%26doi%3D10.1073%2Fpnas.1417811112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref191"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref191'); return false;" data-citation="" class="refNumLink">191</a></strong><div class="NLM_citation" id="cit191"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Harnessing Allostery: A Novel Approach to Drug Discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1285</span>, <span class="refDoi"> DOI: 10.1002/med.21317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fmed.21317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24827416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslalu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=1242-1285&author=S.+Luauthor=S.+Liauthor=J.+Zhang&title=Harnessing+Allostery%3A+A+Novel+Approach+to+Drug+Discovery&doi=10.1002%2Fmed.21317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit191R"><div class="casContent"><span class="casTitleNuber">191</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Allostery: A Novel Approach to Drug Discovery</span></div><div class="casAuthors">Lu, Shaoyong; Li, Shuai; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1242-1285</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Allostery is the most direct and efficient way for regulation of biol. macromol. function, ranging from the control of metabolic mechanisms to signal transduction pathways.  Allosteric modulators target to allosteric sites, offering distinct advantages compared to orthosteric ligands that target to active sites, such as greater specificity, reduced side effects, and lower toxicity.  Allosteric modulators have therefore drawn increasing attention as potential therapeutic drugs in the design and development of new drugs.  In recent years, advancements in our understanding of the fundamental principles underlying allostery, coupled with the exploitation of powerful techniques and methods in the field of allostery, provide unprecedented opportunities to discover allosteric proteins, detect and characterize allosteric sites, design and develop novel efficient allosteric drugs, and recapitulate the universal features of allosteric proteins and allosteric modulators.  In the present review, we summarize the recent advances in the repertoire of allostery, with a particular focus on the aforementioned allosteric compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa1YgTPNMI5rVg90H21EOLACvtfcHk0lgn9kKNylovyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslalu7rO&md5=50ce678b5018e53915817e95ef670e88</span></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2Fmed.21317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21317%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DHarnessing%2520Allostery%253A%2520A%2520Novel%2520Approach%2520to%2520Drug%2520Discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2014%26volume%3D34%26spage%3D1242%26epage%3D1285%26doi%3D10.1002%2Fmed.21317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref192"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref192'); return false;" data-citation="" class="refNumLink">192</a></strong><div class="NLM_citation" id="cit192"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirks, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Biomolecular Simulation: A Computational Microscope for Molecular Biology</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-042910-155245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-042910-155245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22577825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=429-452&author=R.+O.+Drorauthor=R.+M.+Dirksauthor=J.+P.+Grossmanauthor=H.+Xuauthor=D.+E.+Shaw&title=Biomolecular+Simulation%3A+A+Computational+Microscope+for+Molecular+Biology&doi=10.1146%2Fannurev-biophys-042910-155245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">Biomolecular simulation: a computational microscope for molecular biology</span></div><div class="casAuthors">Dror, Ron O.; Dirks, Robert M.; Grossman, J. P.; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">429-452</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics simulations capture the behavior of biol. macromols. in full at. detail, but their computational demands, combined with the challenge of appropriately modeling the relevant physics, have historically restricted their length and accuracy.  Dramatic recent improvements in achievable simulation speed and the underlying phys. models have enabled at.-level simulations on timescales as long as milliseconds that capture key biochem. processes such as protein folding, drug binding, membrane transport, and the conformational changes crit. to protein function.  Such simulation may serve as a computational microscope, revealing biomol. mechanisms at spatial and temporal scales that are difficult to observe exptl.  We describe the rapidly evolving state of the art for at.-level biomol. simulation, illustrate the types of biol. discoveries that can now be made through simulation, and discuss challenges motivating continued innovation in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIwvqxfn3SjLVg90H21EOLACvtfcHk0lgn9kKNylovyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yhs7s%253D&md5=3f872bcd93c1c2141ef3f020c5c6d45d</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-042910-155245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-042910-155245%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DDirks%26aufirst%3DR.%2BM.%26aulast%3DGrossman%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DBiomolecular%2520Simulation%253A%2520A%2520Computational%2520Microscope%2520for%2520Molecular%2520Biology%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D429%26epage%3D452%26doi%3D10.1146%2Fannurev-biophys-042910-155245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref193"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref193'); return false;" data-citation="" class="refNumLink">193</a></strong><div class="NLM_citation" id="cit193"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Structure-Based Drug Design Targeting Class A G Protein-Coupled Receptors Utilizing Crystal Structures and Computational Simulations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01453</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01453" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyktLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1-46&author=Y.+Leeauthor=S.+Basithauthor=S.+Choi&title=Recent+Advances+in+Structure-Based+Drug+Design+Targeting+Class+A+G+Protein-Coupled+Receptors+Utilizing+Crystal+Structures+and+Computational+Simulations&doi=10.1021%2Facs.jmedchem.6b01453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit193R"><div class="casContent"><span class="casTitleNuber">193</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in Structure-Based Drug Design Targeting Class A G Protein-Coupled Receptors Utilizing Crystal Structures and Computational Simulations</span></div><div class="casAuthors">Lee, Yoonji; Basith, Shaherin; Choi, Sun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-46</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent the largest and most physiol. important integral membrane protein family, and these receptors respond to a wide variety of physiol. and environmental stimuli.  GPCRs are among the most crit. therapeutic targets for numerous human diseases, and approx. one-third of the currently marketed drugs target this receptor family.  The recent breakthroughs in GPCR structural biol. have significantly contributed to our understanding of GPCR function, ligand binding, and pharmacol. action as well as to the design of new drugs.  This perspective highlights the latest advances in GPCR structures with a focus on the receptor-ligand interactions of each receptor family in class A nonrhodopsin GPCRs as well as the structural features for their activation, biased signaling, and allosteric mechanisms.  The current state-of-the-art methodologies of structure-based drug design (SBDD) approaches in the GPCR research field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp90SKZ9osqkbVg90H21EOLACvtfcHk0lgn9kKNylovyg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyktLjI&md5=0706fa6680cfe73076a0bdd8d1267240</span></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01453%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DBasith%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DS.%26atitle%3DRecent%2520Advances%2520in%2520Structure-Based%2520Drug%2520Design%2520Targeting%2520Class%2520A%2520G%2520Protein-Coupled%2520Receptors%2520Utilizing%2520Crystal%2520Structures%2520and%2520Computational%2520Simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1%26epage%3D46%26doi%3D10.1021%2Facs.jmedchem.6b01453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref194"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref194'); return false;" data-citation="" class="refNumLink">194</a></strong><div class="NLM_citation" id="cit194"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span> <span> </span><span class="NLM_article-title">The Underappreciated Role of Allostery in the Cellular Network</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-083012-130257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-083012-130257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23451894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFGrs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=169-189&author=R.+Nussinovauthor=C.-J.+Tsaiauthor=B.+Ma&title=The+Underappreciated+Role+of+Allostery+in+the+Cellular+Network&doi=10.1146%2Fannurev-biophys-083012-130257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">The underappreciated role of allostery in the cellular network</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Ma, Buyong</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">169-189</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Allosteric propagation results in communication between distinct sites in the protein structure; it also encodes specific effects on cellular pathways, and in this way it shapes cellular response.  One example of long-range effects is binding of morphogens to cell surface receptors, which initiates a cascade of protein interactions that leads to genome activation and specific cellular action.  Allosteric propagation results from combinations of multiple factors, takes place through dynamic shifts of conformational ensembles, and affects the equil. of macromol. interactions.  Here, we (a) emphasize the well-known yet still underappreciated role of allostery in conveying explicit signals across large multimol. assemblies and distances to specify cellular action; (b) stress the need for quantitation of the allosteric effects; and finally, (c) propose that each specific combination of allosteric effectors along the pathway spells a distinct function.  The challenges are colossal; the inspiring reward will be predicting function, misfunction, and outcomes of drug regimes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqMSQUbLZg-bVg90H21EOLACvtfcHk0liup3WTHjLdwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFGrs7bJ&md5=9d239bd6685c4c158ef706bd6081a183</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-083012-130257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-083012-130257%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.-J.%26aulast%3DMa%26aufirst%3DB.%26atitle%3DThe%2520Underappreciated%2520Role%2520of%2520Allostery%2520in%2520the%2520Cellular%2520Network%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2013%26volume%3D42%26spage%3D169%26epage%3D189%26doi%3D10.1146%2Fannurev-biophys-083012-130257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref195"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref195'); return false;" data-citation="" class="refNumLink">195</a></strong><div class="NLM_citation" id="cit195"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Systems Biology Brings New Dimensions for Structure-Based Drug Design</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">11556</span>– <span class="NLM_lpage">11565</span>, <span class="refDoi"> DOI: 10.1021/ja504810z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja504810z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=11556-11565&author=J.+Peiauthor=N.+Yinauthor=X.+Maauthor=L.+Lai&title=Systems+Biology+Brings+New+Dimensions+for+Structure-Based+Drug+Design&doi=10.1021%2Fja504810z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Systems Biology Brings New Dimensions for Structure-Based Drug Design</span></div><div class="casAuthors">Pei, Jianfeng; Yin, Ning; Ma, Xiaomin; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11556-11565</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In this Perspective, we focus on new, systems-centric views of structure-based drug design (SBDD) that we believe will impact future drug discovery research and development.  We will first discuss new ways to identify drug targets based on systems intervention anal., and then we will introduce emerging SBDD methods driven by advancements in systems biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreR4j6y808PrVg90H21EOLACvtfcHk0liup3WTHjLdwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3E&md5=7834c0474a316442c136bb601ee3882c</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1021%2Fja504810z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja504810z%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DSystems%2520Biology%2520Brings%2520New%2520Dimensions%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D11556%26epage%3D11565%26doi%3D10.1021%2Fja504810z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref196"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref196'); return false;" data-citation="" class="refNumLink">196</a></strong><div class="NLM_citation" id="cit196"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span> <span> </span><span class="NLM_article-title">Principles of Allosteric Interactions in Cell Signaling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">17692</span>– <span class="NLM_lpage">17701</span>, <span class="refDoi"> DOI: 10.1021/ja510028c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja510028c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVegurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=17692-17701&author=R.+Nussinovauthor=C.+Tsaiauthor=J.+Liu&title=Principles+of+Allosteric+Interactions+in+Cell+Signaling&doi=10.1021%2Fja510028c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of allosteric interactions in cell signaling</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Liu, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">17692-17701</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Linking cell signaling events to the fundamental physicochem. basis of the conformational behavior of single mols. and ultimately to cellular function is a key challenge facing the life sciences.  Here, the authors outline the emerging principles of allosteric interactions in cell signaling, with emphasis on the following points.  (1) Allosteric efficacy is not a function of the chem. compn. of the allosteric pocket but reflects the extent of the population shift between the inactive and active states; i.e., the allosteric effect is detd. by the extent of preferred binding, not by the overall binding affinity.  (2) Coupling between the allosteric and active sites does not decide the allosteric effect; however, it does define the propagation pathways, the allosteric binding sites, and key on-path residues.  (3) Atoms of allosteric effectors can act as "driver" or "anchor" and create attractive "pulling" or repulsive "pushing" interactions.  Deciphering, quantifying, and integrating the multiple co-occurring events present daunting challenges to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpehHLG3w1sSrVg90H21EOLACvtfcHk0liup3WTHjLdwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVegurnL&md5=1ff293ab2498b4946cba72ec5f67abef</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Fja510028c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja510028c%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26atitle%3DPrinciples%2520of%2520Allosteric%2520Interactions%2520in%2520Cell%2520Signaling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D17692%26epage%3D17701%26doi%3D10.1021%2Fja510028c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref197"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref197'); return false;" data-citation="" class="refNumLink">197</a></strong><div class="NLM_citation" id="cit197"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csizmok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follis, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriwacki, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Dynamic Protein Interaction Networks and New Structural Paradigms in Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6424</span>– <span class="NLM_lpage">6462</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00548</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00548" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Cqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6424-6462&author=V.+Csizmokauthor=A.+V.+Follisauthor=R.+W.+Kriwackiauthor=J.+D.+Forman-Kay&title=Dynamic+Protein+Interaction+Networks+and+New+Structural+Paradigms+in+Signaling&doi=10.1021%2Facs.chemrev.5b00548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Protein Interaction Networks and New Structural Paradigms in Signaling</span></div><div class="casAuthors">Csizmok, Veronika; Follis, Ariele Viacava; Kriwacki, Richard W.; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6424-6462</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding signaling and other complex biol. processes requires elucidating the crit. roles of intrinsically disordered proteins (IDPs) and regions (IDRs), which represent ∼30% of the proteome and enable unique regulatory mechanisms.  In this review, we describe the structural heterogeneity of disordered proteins that underpins these mechanisms and the latest progress in obtaining structural descriptions of conformational ensembles of disordered proteins that are needed for linking structure and dynamics to function.  We describe the diverse interactions of IDPs that can have unusual characteristics such as "ultrasensitivity" and "regulated folding and unfolding".  We also summarize the mounting data showing that large-scale assembly and protein phase sepn. occurs within a variety of signaling complexes and cellular structures.  In addn., we discuss efforts to therapeutically target disordered proteins with small mols.  Overall, we interpret the remodeling of disordered state ensembles due to binding and post-translational modifications within an expanded framework for allostery that provides significant insights into how disordered proteins transmit biol. information.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYLVEaEWv217Vg90H21EOLACvtfcHk0liup3WTHjLdwQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Cqsr0%253D&md5=7a4c31eb254a215673a3304e1d3e0c0e</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00548%26sid%3Dliteratum%253Aachs%26aulast%3DCsizmok%26aufirst%3DV.%26aulast%3DFollis%26aufirst%3DA.%2BV.%26aulast%3DKriwacki%26aufirst%3DR.%2BW.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DDynamic%2520Protein%2520Interaction%2520Networks%2520and%2520New%2520Structural%2520Paradigms%2520in%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6424%26epage%3D6462%26doi%3D10.1021%2Facs.chemrev.5b00548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref198"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref198'); return false;" data-citation="" class="refNumLink">198</a></strong><div class="NLM_citation" id="cit198"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span> <span> </span><span class="NLM_article-title">Allo-Network Drugs: Harnessing Allostery in Cellular Networks</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tips.2011.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21925743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2mu7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=686-693&author=R.+Nussinovauthor=C.+J.+Tsaiauthor=P.+Csermely&title=Allo-Network+Drugs%3A+Harnessing+Allostery+in+Cellular+Networks&doi=10.1016%2Fj.tips.2011.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Allo-network drugs: harnessing allostery in cellular networks</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Csermely, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Allosteric drugs are increasingly used because they produce fewer side effects.  Allosteric signal propagation does not stop at the end' of a protein, but may be dynamically transmitted across the cell.  We propose here that the concept of allosteric drugs can be broadened to allo-network drugs' - whose effects can propagate either within a protein, or across several proteins, to enhance or inhibit specific interactions along a pathway.  We posit that current allosteric drugs are a special case of allo-network drugs, and suggest that allo-network drugs can achieve specific, limited changes at the systems level, and in this way can achieve fewer side effects and lower toxicity.  Finally, we propose steps and methods to identify allo-network drug targets and sites that outline a new paradigm in systems-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomhg6HNPWpZLVg90H21EOLACvtfcHk0ljOYE3EgkBy1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2mu7%252FE&md5=1d1e2834e293b18e9d3aaeecdf9ba828</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DCsermely%26aufirst%3DP.%26atitle%3DAllo-Network%2520Drugs%253A%2520Harnessing%2520Allostery%2520in%2520Cellular%2520Networks%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2011%26volume%3D32%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.tips.2011.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110'],'ref111':['cit111'],'ref112':['cit112'],'ref113':['cit113'],'ref114':['cit114'],'ref115':['cit115'],'ref116':['cit116'],'ref117':['cit117'],'ref118':['cit118'],'ref119':['cit119'],'ref120':['cit120'],'ref121':['cit121'],'ref122':['cit122'],'ref123':['cit123'],'ref124':['cit124'],'ref125':['cit125'],'ref126':['cit126'],'ref127':['cit127'],'ref128':['cit128'],'ref129':['cit129'],'ref130':['cit130'],'ref131':['cit131'],'ref132':['cit132'],'ref133':['cit133'],'ref134':['cit134'],'ref135':['cit135'],'ref136':['cit136'],'ref137':['cit137'],'ref138':['cit138'],'ref139':['cit139'],'ref140':['cit140'],'ref141':['cit141'],'ref142':['cit142'],'ref143':['cit143'],'ref144':['cit144'],'ref145':['cit145'],'ref146':['cit146'],'ref147':['cit147'],'ref148':['cit148'],'ref149':['cit149'],'ref150':['cit150'],'ref151':['cit151'],'ref152':['cit152'],'ref153':['cit153'],'ref154':['cit154'],'ref155':['cit155'],'ref156':['cit156'],'ref157':['cit157'],'ref158':['cit158'],'ref159':['cit159'],'ref160':['cit160'],'ref161':['cit161'],'ref162':['cit162'],'ref163':['cit163'],'ref164':['cit164'],'ref165':['cit165'],'ref166':['cit166'],'ref167':['cit167'],'ref168':['cit168'],'ref169':['cit169'],'ref170':['cit170'],'ref171':['cit171'],'ref172':['cit172'],'ref173':['cit173'],'ref174':['cit174'],'ref175':['cit175'],'ref176':['cit176'],'ref177':['cit177'],'ref178':['cit178'],'ref179':['cit179'],'ref180':['cit180'],'ref181':['cit181'],'ref182':['cit182'],'ref183':['cit183'],'ref184':['cit184'],'ref185':['cit185'],'ref186':['cit186'],'ref187':['cit187'],'ref188':['cit188'],'ref189':['cit189'],'ref190':['cit190'],'ref191':['cit191'],'ref192':['cit192'],'ref193':['cit193'],'ref194':['cit194'],'ref195':['cit195'],'ref196':['cit196'],'ref197':['cit197'],'ref198':['cit198']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 48 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Mingyu Wu, Yu Sun, Meiru Zhu, Laiyu Zhu, Junhong Lü, <span class="NLM_string-name hlFld-ContribAuthor">Feng Geng</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Dynamics-Based Allosteric Prediction Method to Design Key Residues in Threonine Dehydrogenase for Amino-Acid Production. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2021,</strong> <em>6 </em>
                                    (16)
                                     , 10975-10983. <a href="https://doi.org/10.1021/acsomega.1c00798" title="DOI URL">https://doi.org/10.1021/acsomega.1c00798</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.1c00798&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.1c00798%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DMolecular%252BDynamics-Based%252BAllosteric%252BPrediction%252BMethod%252Bto%252BDesign%252BKey%252BResidues%252Bin%252BThreonine%252BDehydrogenase%252Bfor%252BAmino-Acid%252BProduction%26aulast%3DWu%26aufirst%3DMingyu%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D12022021%26date%3D01042021%26date%3D15042021%26volume%3D6%26issue%3D16%26spage%3D10975%26epage%3D10983" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Bingsong Han, Francesco G. Salituro, <span class="NLM_string-name hlFld-ContribAuthor">Maria-Jesus Blanco</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Allosteric Modulation in Drug Discovery: Innovation in Emerging Chemical Modalities. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>11 </em>
                                    (10)
                                     , 1810-1819. <a href="https://doi.org/10.1021/acsmedchemlett.9b00655" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00655</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00655%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DImpact%252Bof%252BAllosteric%252BModulation%252Bin%252BDrug%252BDiscovery%25253A%252BInnovation%252Bin%252BEmerging%252BChemical%252BModalities%26aulast%3DHan%26aufirst%3DBingsong%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D26122019%26date%3D10032020%26date%3D20032020%26date%3D10032020%26volume%3D11%26issue%3D10%26spage%3D1810%26epage%3D1819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ting Ran, Rongquan Xiao, Qixuan Huang, Haoliang Yuan, Tao Lu, <span class="NLM_string-name hlFld-ContribAuthor">Wen Liu</span>. </span><span class="cited-content_cbyCitation_article-title">In Silico Discovery of JMJD6 Inhibitors for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>10 </em>
                                    (12)
                                     , 1609-1613. <a href="https://doi.org/10.1021/acsmedchemlett.9b00264" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.9b00264</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.9b00264&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.9b00264%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIn%252BSilico%252BDiscovery%252Bof%252BJMJD6%252BInhibitors%252Bfor%252BCancer%252BTreatment%26aulast%3DRan%26aufirst%3DTing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D12062019%26date%3D19112019%26date%3D22112019%26date%3D19112019%26volume%3D10%26issue%3D12%26spage%3D1609%26epage%3D1613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sophia F  Mersmann</span>, <span class="hlFld-ContribAuthor ">Léonie  Strömich</span>, <span class="hlFld-ContribAuthor ">Florian J  Song</span>, <span class="hlFld-ContribAuthor ">Nan  Wu</span>, <span class="hlFld-ContribAuthor ">Francesca  Vianello</span>, <span class="hlFld-ContribAuthor ">Mauricio  Barahona</span>, <span class="hlFld-ContribAuthor ">Sophia N  Yaliraki</span>. </span><span class="cited-content_cbyCitation_article-title">ProteinLens: a web-based application for the analysis of allosteric signalling on atomistic graphs of biomolecules. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (W1)
                                     , W551-W558. <a href="https://doi.org/10.1093/nar/gkab350" title="DOI URL">https://doi.org/10.1093/nar/gkab350</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab350&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab350%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DProteinLens%25253A%252Ba%252Bweb-based%252Bapplication%252Bfor%252Bthe%252Banalysis%252Bof%252Ballosteric%252Bsignalling%252Bon%252Batomistic%252Bgraphs%252Bof%252Bbiomolecules%26aulast%3DMersmann%26aufirst%3DSophia%25C2%25A0F%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3DW1%26spage%3DW551%26epage%3DW558" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Senfeng  Xiang</span>, <span class="hlFld-ContribAuthor ">Xiaoxian  Shi</span>, <span class="hlFld-ContribAuthor ">Pengfei  Chen</span>, <span class="hlFld-ContribAuthor ">Yifan  Chen</span>, <span class="hlFld-ContribAuthor ">Shaowei  Bing</span>, <span class="hlFld-ContribAuthor ">Xizhi  Jin</span>, <span class="hlFld-ContribAuthor ">Ji  Cao</span>, <span class="hlFld-ContribAuthor ">Jinhu  Wang</span>, <span class="hlFld-ContribAuthor ">Bo  Yang</span>, <span class="hlFld-ContribAuthor ">Xuejing  Shao</span>, <span class="hlFld-ContribAuthor ">Qiaojun  He</span>, <span class="hlFld-ContribAuthor ">Meidan  Ying</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Cul3-scaffold E3 ligase complex via KLHL substrate adaptors for cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Pharmacological Research</span><span> <strong>2021,</strong> <em>169 </em>, 105616. <a href="https://doi.org/10.1016/j.phrs.2021.105616" title="DOI URL">https://doi.org/10.1016/j.phrs.2021.105616</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.phrs.2021.105616&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.phrs.2021.105616%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacological%2520Research%26atitle%3DTargeting%252BCul3-scaffold%252BE3%252Bligase%252Bcomplex%252Bvia%252BKLHL%252Bsubstrate%252Badaptors%252Bfor%252Bcancer%252Btherapy%26aulast%3DXiang%26aufirst%3DSenfeng%26date%3D2021%26volume%3D169%26spage%3D105616" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiufen  Zhang</span>, <span class="hlFld-ContribAuthor ">Yingyi  Chen</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Zhimin  Huang</span>, <span class="hlFld-ContribAuthor ">Jiacheng  Wei</span>, <span class="hlFld-ContribAuthor ">Li  Feng</span>, <span class="hlFld-ContribAuthor ">Jun-Cheng  Su</span>, <span class="hlFld-ContribAuthor ">Yingqing  Wei</span>, <span class="hlFld-ContribAuthor ">Shaobo  Ning</span>, <span class="hlFld-ContribAuthor ">Xiuyan  Yang</span>, <span class="hlFld-ContribAuthor ">Mingzhu  Zhao</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Kun  Song</span>, <span class="hlFld-ContribAuthor ">Zhengtian  Yu</span>, <span class="hlFld-ContribAuthor ">Jianrong  Xu</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Houwen  Lin</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting a cryptic allosteric site of SIRT6 with small-molecule inhibitors that inhibit the migration of pancreatic cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>100 </em><a href="https://doi.org/10.1016/j.apsb.2021.06.015" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.06.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.06.015%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DTargeting%252Ba%252Bcryptic%252Ballosteric%252Bsite%252Bof%252BSIRT6%252Bwith%252Bsmall-molecule%252Binhibitors%252Bthat%252Binhibit%252Bthe%252Bmigration%252Bof%252Bpancreatic%252Bcancer%252Bcells%26aulast%3DZhang%26aufirst%3DQiufen%26date%3D2021%26volume%3D100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Monica  Civera</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Moroni</span>, <span class="hlFld-ContribAuthor ">Luca  Sorrentino</span>, <span class="hlFld-ContribAuthor ">Francesca  Vasile</span>, <span class="hlFld-ContribAuthor ">Sara  Sattin</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical and Biophysical Approaches to Allosteric Modulation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>48 </em><a href="https://doi.org/10.1002/ejoc.202100506" title="DOI URL">https://doi.org/10.1002/ejoc.202100506</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202100506&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202100506%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DChemical%252Band%252BBiophysical%252BApproaches%252Bto%252BAllosteric%252BModulation%26aulast%3DCivera%26aufirst%3DMonica%26date%3D2021%26date%3D2021%26volume%3D48" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexios  Chatzigoulas</span>, <span class="hlFld-ContribAuthor ">Zoe  Cournia</span>. </span><span class="cited-content_cbyCitation_article-title">Rational design of allosteric modulators: Challenges and successes. </span><span class="cited-content_cbyCitation_journal-name">WIREs Computational Molecular Science</span><span> <strong>2021,</strong> <em>17 </em><a href="https://doi.org/10.1002/wcms.1529" title="DOI URL">https://doi.org/10.1002/wcms.1529</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1529%26sid%3Dliteratum%253Aachs%26jtitle%3DWIREs%2520Computational%2520Molecular%2520Science%26atitle%3DRational%252Bdesign%252Bof%252Ballosteric%252Bmodulators%25253A%252BChallenges%252Band%252Bsuccesses%26aulast%3DChatzigoulas%26aufirst%3DAlexios%26date%3D2021%26date%3D2021%26volume%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Yingyi  Chen</span>, <span class="hlFld-ContribAuthor ">Jiacheng  Wei</span>, <span class="hlFld-ContribAuthor ">Mingzhu  Zhao</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanism of allosteric activation of SIRT6 revealed by the action of rationally designed activators. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>11 </em>
                                    (5)
                                     , 1355-1361. <a href="https://doi.org/10.1016/j.apsb.2020.09.010" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.09.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.09.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.09.010%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DMechanism%252Bof%252Ballosteric%252Bactivation%252Bof%252BSIRT6%252Brevealed%252Bby%252Bthe%252Baction%252Bof%252Brationally%252Bdesigned%252Bactivators%26aulast%3DLu%26aufirst%3DShaoyong%26date%3D2021%26volume%3D11%26issue%3D5%26spage%3D1355%26epage%3D1361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chen  Davidovich</span>, <span class="hlFld-ContribAuthor ">Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric regulation of histone lysine methyltransferases: from context-specific regulation to selective drugs. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2021,</strong> <em>49 </em>
                                    (2)
                                     , 591-607. <a href="https://doi.org/10.1042/BST20200238" title="DOI URL">https://doi.org/10.1042/BST20200238</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20200238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20200238%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DAllosteric%252Bregulation%252Bof%252Bhistone%252Blysine%252Bmethyltransferases%25253A%252Bfrom%252Bcontext-specific%252Bregulation%252Bto%252Bselective%252Bdrugs%26aulast%3DDavidovich%26aufirst%3DChen%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D2%26spage%3D591%26epage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Seyedeh Maryam  Salehi</span>, <span class="hlFld-ContribAuthor ">Markus  Meuwly</span>. </span><span class="cited-content_cbyCitation_article-title">Site-selective dynamics of azidolysozyme. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Chemical Physics</span><span> <strong>2021,</strong> <em>154 </em>
                                    (16)
                                     , 165101. <a href="https://doi.org/10.1063/5.0047330" title="DOI URL">https://doi.org/10.1063/5.0047330</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1063/5.0047330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1063%2F5.0047330%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Chemical%2520Physics%26atitle%3DSite-selective%252Bdynamics%252Bof%252Bazidolysozyme%26aulast%3DSalehi%26aufirst%3DSeyedeh%2BMaryam%26date%3D2021%26volume%3D154%26issue%3D16%26spage%3D165101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lauren M.  Slosky</span>, <span class="hlFld-ContribAuthor ">Marc G.  Caron</span>, <span class="hlFld-ContribAuthor ">Lawrence S.  Barak</span>. </span><span class="cited-content_cbyCitation_article-title">Biased Allosteric Modulators: New Frontiers in GPCR Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2021,</strong> <em>42 </em>
                                    (4)
                                     , 283-299. <a href="https://doi.org/10.1016/j.tips.2020.12.005" title="DOI URL">https://doi.org/10.1016/j.tips.2020.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.12.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DBiased%252BAllosteric%252BModulators%25253A%252BNew%252BFrontiers%252Bin%252BGPCR%252BDrug%252BDiscovery%26aulast%3DSlosky%26aufirst%3DLauren%2BM.%26date%3D2021%26volume%3D42%26issue%3D4%26spage%3D283%26epage%3D299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rati Kailash Prasad  Tripathi</span>, <span class="hlFld-ContribAuthor ">Senthil Raja  Ayyannan</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging chemical scaffolds with potential SHP2 phosphatase inhibitory capabilities – A comprehensive review. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2021,</strong> <em>97 </em>
                                    (3)
                                     , 721-773. <a href="https://doi.org/10.1111/cbdd.13807" title="DOI URL">https://doi.org/10.1111/cbdd.13807</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13807&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13807%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DEmerging%252Bchemical%252Bscaffolds%252Bwith%252Bpotential%252BSHP2%252Bphosphatase%252Binhibitory%252Bcapabilities%252B%2525E2%252580%252593%252BA%252Bcomprehensive%252Breview%26aulast%3DTripathi%26aufirst%3DRati%2BKailash%2BPrasad%26date%3D2021%26date%3D2020%26volume%3D97%26issue%3D3%26spage%3D721%26epage%3D773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">István  Ledneczki</span>, <span class="hlFld-ContribAuthor ">Pál  Tapolcsányi</span>, <span class="hlFld-ContribAuthor ">Eszter  Gábor</span>, <span class="hlFld-ContribAuthor ">András  Visegrády</span>, <span class="hlFld-ContribAuthor ">Márton  Vass</span>, <span class="hlFld-ContribAuthor ">János  Éles</span>, <span class="hlFld-ContribAuthor ">Patrik  Holm</span>, <span class="hlFld-ContribAuthor ">Anita  Horváth</span>, <span class="hlFld-ContribAuthor ">Anikó  Pocsai</span>, <span class="hlFld-ContribAuthor ">Sándor  Mahó</span>, <span class="hlFld-ContribAuthor ">István  Greiner</span>, <span class="hlFld-ContribAuthor ">Balázs  Krámos</span>, <span class="hlFld-ContribAuthor ">Zoltán  Béni</span>, <span class="hlFld-ContribAuthor ">János  Kóti</span>, <span class="hlFld-ContribAuthor ">Anna E.  Káncz</span>, <span class="hlFld-ContribAuthor ">Márta  Thán</span>, <span class="hlFld-ContribAuthor ">Sándor  Kolok</span>, <span class="hlFld-ContribAuthor ">Judit  Laszy</span>, <span class="hlFld-ContribAuthor ">Ottilia  Balázs</span>, <span class="hlFld-ContribAuthor ">Gyula  Bugovits</span>, <span class="hlFld-ContribAuthor ">József  Nagy</span>, <span class="hlFld-ContribAuthor ">Mónika  Vastag</span>, <span class="hlFld-ContribAuthor ">Ágota  Szájli</span>, <span class="hlFld-ContribAuthor ">Éva  Bozó</span>, <span class="hlFld-ContribAuthor ">György  Lévay</span>, <span class="hlFld-ContribAuthor ">Balázs  Lendvai</span>, <span class="hlFld-ContribAuthor ">Zsolt  Némethy</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of novel positive allosteric modulators of the α7 nicotinic acetylcholine receptor: Scaffold hopping approach. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>214 </em>, 113189. <a href="https://doi.org/10.1016/j.ejmech.2021.113189" title="DOI URL">https://doi.org/10.1016/j.ejmech.2021.113189</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2021.113189&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2021.113189%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252Bnovel%252Bpositive%252Ballosteric%252Bmodulators%252Bof%252Bthe%252B%2525CE%2525B17%252Bnicotinic%252Bacetylcholine%252Breceptor%25253A%252BScaffold%252Bhopping%252Bapproach%26aulast%3DLedneczki%26aufirst%3DIstv%25C3%25A1n%26date%3D2021%26volume%3D214%26spage%3D113189" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Yuanhao  Wang</span>, <span class="hlFld-ContribAuthor ">Zongtao  Chai</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Jie  Chen</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Chuan  Lv</span>, <span class="hlFld-ContribAuthor ">Mingfei  Ji</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting RAS phosphorylation in cancer therapy: Mechanisms and modulators. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2021,</strong> <em>116 </em><a href="https://doi.org/10.1016/j.apsb.2021.02.014" title="DOI URL">https://doi.org/10.1016/j.apsb.2021.02.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2021.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2021.02.014%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DTargeting%252BRAS%252Bphosphorylation%252Bin%252Bcancer%252Btherapy%25253A%252BMechanisms%252Band%252Bmodulators%26aulast%3DQiu%26aufirst%3DYuran%26date%3D2021%26volume%3D116" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Ning  Huang</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Xiao-Lan  Yin</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Conformational Selection Mechanism Provides Structural Insights into the Optimization of APC-Asef Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (4)
                                     , 962. <a href="https://doi.org/10.3390/molecules26040962" title="DOI URL">https://doi.org/10.3390/molecules26040962</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26040962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26040962%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DConformational%252BSelection%252BMechanism%252BProvides%252BStructural%252BInsights%252Binto%252Bthe%252BOptimization%252Bof%252BAPC-Asef%252BInhibitors%26aulast%3DHe%26aufirst%3DXinheng%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D4%26spage%3D962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Jiacheng  Wei</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Ashfaq Ur  Rehman</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of cryptic allosteric sites using reversed allosteric communication by a combined computational and experimental strategy. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2021,</strong> <em>12 </em>
                                    (1)
                                     , 464-476. <a href="https://doi.org/10.1039/D0SC05131D" title="DOI URL">https://doi.org/10.1039/D0SC05131D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0SC05131D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0SC05131D%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DDiscovery%252Bof%252Bcryptic%252Ballosteric%252Bsites%252Busing%252Breversed%252Ballosteric%252Bcommunication%252Bby%252Ba%252Bcombined%252Bcomputational%252Band%252Bexperimental%252Bstrategy%26aulast%3DNi%26aufirst%3DDuan%26date%3D2021%26date%3D2021%26volume%3D12%26issue%3D1%26spage%3D464%26epage%3D476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mingyu  Li</span>, <span class="hlFld-ContribAuthor ">Ashfaq Ur  Rehman</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Kai  Chen</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Dual roles of ATP-binding site in protein kinases: Orthosteric inhibition and allosteric regulation. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 87-119. <a href="https://doi.org/10.1016/bs.apcsb.2020.09.005" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2020.09.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2020.09.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2020.09.005%26sid%3Dliteratum%253Aachs%26atitle%3DDual%252Broles%252Bof%252BATP-binding%252Bsite%252Bin%252Bprotein%252Bkinases%25253A%252BOrthosteric%252Binhibition%252Band%252Ballosteric%252Bregulation%26aulast%3DLi%26aufirst%3DMingyu%26date%3D2021%26spage%3D87%26epage%3D119%26pub%3DElsevier%26atitle%3DProtein%252BKinases%252Bin%252BDrug%252BDiscovery%26date%3D2021%26volume%3D124" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuanhao  Wang</span>, <span class="hlFld-ContribAuthor ">Dong  Ji</span>, <span class="hlFld-ContribAuthor ">Chaoyu  Lei</span>, <span class="hlFld-ContribAuthor ">Yingfei  Chen</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Mingyu  Li</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Fu</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanistic insights into the effect of phosphorylation on Ras conformational dynamics and its interactions with cell signaling proteins. </span><span class="cited-content_cbyCitation_journal-name">Computational and Structural Biotechnology Journal</span><span> <strong>2021,</strong> <em>19 </em>, 1184-1199. <a href="https://doi.org/10.1016/j.csbj.2021.01.044" title="DOI URL">https://doi.org/10.1016/j.csbj.2021.01.044</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.csbj.2021.01.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.csbj.2021.01.044%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520and%2520Structural%2520Biotechnology%2520Journal%26atitle%3DMechanistic%252Binsights%252Binto%252Bthe%252Beffect%252Bof%252Bphosphorylation%252Bon%252BRas%252Bconformational%252Bdynamics%252Band%252Bits%252Binteractions%252Bwith%252Bcell%252Bsignaling%252Bproteins%26aulast%3DWang%26aufirst%3DYuanhao%26date%3D2021%26volume%3D19%26spage%3D1184%26epage%3D1199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arfaxad  Reyes-Alcaraz</span>, <span class="hlFld-ContribAuthor ">Emilio  Y. Lucero Garcia-Rojas</span>, <span class="hlFld-ContribAuthor ">Richard  A. Bond</span>, <span class="hlFld-ContribAuthor ">Bradley  K. McConnell</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Modulators for GPCRs as a Therapeutic Alternative with High Potential in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,<a href="https://doi.org/10.5772/intechopen.91838" title="DOI URL">https://doi.org/10.5772/intechopen.91838</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5772/intechopen.91838&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5772%2Fintechopen.91838%26sid%3Dliteratum%253Aachs%26atitle%3DAllosteric%252BModulators%252Bfor%252BGPCRs%252Bas%252Ba%252BTherapeutic%252BAlternative%252Bwith%252BHigh%252BPotential%252Bin%252BDrug%252BDiscovery%26aulast%3DReyes-Alcaraz%26aufirst%3DArfaxad%26date%3D2020%26date%3D2020%26pub%3DIntechOpen%26atitle%3DMolecular%252BPharmacology%26aulast%3DCatala%26aufirst%3DAngel%26date%3D2020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Computational methods-guided design of modulators targeting protein-protein interactions (PPIs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112764. <a href="https://doi.org/10.1016/j.ejmech.2020.112764" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112764</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112764%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252Bmethods-guided%252Bdesign%252Bof%252Bmodulators%252Btargeting%252Bprotein-protein%252Binteractions%252B%252528PPIs%252529%26aulast%3DQiu%26aufirst%3DYuran%26date%3D2020%26volume%3D207%26spage%3D112764" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Fu</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Zinc-mediated conformational preselection mechanism in the allosteric control of DNA binding to the zinc transcriptional regulator (ZitR). </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-70381-8" title="DOI URL">https://doi.org/10.1038/s41598-020-70381-8</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-70381-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-70381-8%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DZinc-mediated%252Bconformational%252Bpreselection%252Bmechanism%252Bin%252Bthe%252Ballosteric%252Bcontrol%252Bof%252BDNA%252Bbinding%252Bto%252Bthe%252Bzinc%252Btranscriptional%252Bregulator%252B%252528ZitR%252529%26aulast%3DHe%26aufirst%3DXinheng%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eileen K.  Jaffe</span>. </span><span class="cited-content_cbyCitation_article-title">Wrangling Shape-Shifting Morpheeins to Tackle Disease and Approach Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fmolb.2020.582966" title="DOI URL">https://doi.org/10.3389/fmolb.2020.582966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2020.582966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2020.582966%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DWrangling%252BShape-Shifting%252BMorpheeins%252Bto%252BTackle%252BDisease%252Band%252BApproach%252BDrug%252BDiscovery%26aulast%3DJaffe%26aufirst%3DEileen%2BK.%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuemin  Bian</span>, <span class="hlFld-ContribAuthor ">Jaden Jungho  Jun</span>, <span class="hlFld-ContribAuthor ">Jacob  Cuyler</span>, <span class="hlFld-ContribAuthor ">Xiang-Qun  Xie</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent allosteric modulation: An emerging strategy for GPCRs drug discovery. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>206 </em>, 112690. <a href="https://doi.org/10.1016/j.ejmech.2020.112690" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112690</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112690&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112690%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCovalent%252Ballosteric%252Bmodulation%25253A%252BAn%252Bemerging%252Bstrategy%252Bfor%252BGPCRs%252Bdrug%252Bdiscovery%26aulast%3DBian%26aufirst%3DYuemin%26date%3D2020%26volume%3D206%26spage%3D112690" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Mingyu  Ye</span>, <span class="hlFld-ContribAuthor ">Yue  Wang</span>, <span class="hlFld-ContribAuthor ">Ming  Qiu</span>, <span class="hlFld-ContribAuthor ">Tingting  Fu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Zhou</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">How Parkinson's disease-related mutations disrupt the dimerization of WD40 domain in LRRK2: a comparative molecular dynamics simulation study. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2020,</strong> <em>22 </em>
                                    (36)
                                     , 20421-20433. <a href="https://doi.org/10.1039/D0CP03171B" title="DOI URL">https://doi.org/10.1039/D0CP03171B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CP03171B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CP03171B%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DHow%252BParkinson%252527s%252Bdisease-related%252Bmutations%252Bdisrupt%252Bthe%252Bdimerization%252Bof%252BWD40%252Bdomain%252Bin%252BLRRK2%25253A%252Ba%252Bcomparative%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudy%26aulast%3DLi%26aufirst%3DXinyi%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D36%26spage%3D20421%26epage%3D20433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francesca  Carofiglio</span>, <span class="hlFld-ContribAuthor ">Daniela  Trisciuzzi</span>, <span class="hlFld-ContribAuthor ">Nicola  Gambacorta</span>, <span class="hlFld-ContribAuthor ">Francesco  Leonetti</span>, <span class="hlFld-ContribAuthor ">Angela  Stefanachi</span>, <span class="hlFld-ContribAuthor ">Orazio  Nicolotti</span>. </span><span class="cited-content_cbyCitation_article-title">Bcr-Abl Allosteric Inhibitors: Where We Are and Where We Are Going to. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (18)
                                     , 4210. <a href="https://doi.org/10.3390/molecules25184210" title="DOI URL">https://doi.org/10.3390/molecules25184210</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25184210&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25184210%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DBcr-Abl%252BAllosteric%252BInhibitors%25253A%252BWhere%252BWe%252BAre%252Band%252BWhere%252BWe%252BAre%252BGoing%252Bto%26aulast%3DCarofiglio%26aufirst%3DFrancesca%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D18%26spage%3D4210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qiaohong  Geng</span>, <span class="hlFld-ContribAuthor ">Sagar O.  Rohondia</span>, <span class="hlFld-ContribAuthor ">Harras J.  Khan</span>, <span class="hlFld-ContribAuthor ">Peifu  Jiao</span>, <span class="hlFld-ContribAuthor ">Q. Ping  Dou</span>. </span><span class="cited-content_cbyCitation_article-title">Small molecules as antagonists of co-inhibitory pathways for cancer immunotherapy: a patent review (2018-2019). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2020,</strong> <em>30 </em>
                                    (9)
                                     , 677-694. <a href="https://doi.org/10.1080/13543776.2020.1801640" title="DOI URL">https://doi.org/10.1080/13543776.2020.1801640</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2020.1801640&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2020.1801640%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DSmall%252Bmolecules%252Bas%252Bantagonists%252Bof%252Bco-inhibitory%252Bpathways%252Bfor%252Bcancer%252Bimmunotherapy%25253A%252Ba%252Bpatent%252Breview%252B%2525282018-2019%252529%26aulast%3DGeng%26aufirst%3DQiaohong%26date%3D2020%26date%3D2020%26volume%3D30%26issue%3D9%26spage%3D677%26epage%3D694" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher R. M.  Asquith</span>, <span class="hlFld-ContribAuthor ">Graham J.  Tizzard</span>, <span class="hlFld-ContribAuthor ">James M.  Bennett</span>, <span class="hlFld-ContribAuthor ">Carrow I.  Wells</span>, <span class="hlFld-ContribAuthor ">Jonathan M.  Elkins</span>, <span class="hlFld-ContribAuthor ">Timothy M.  Willson</span>, <span class="hlFld-ContribAuthor ">Antti  Poso</span>, <span class="hlFld-ContribAuthor ">Tuomo  Laitinen</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting the Water Network in Cyclin G‐Associated Kinase (GAK) with 4‐Anilino‐quin(az)oline Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2020,</strong> <em>15 </em>
                                    (13)
                                     , 1200-1215. <a href="https://doi.org/10.1002/cmdc.202000150" title="DOI URL">https://doi.org/10.1002/cmdc.202000150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.202000150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.202000150%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DTargeting%252Bthe%252BWater%252BNetwork%252Bin%252BCyclin%252BG%2525E2%252580%252590Associated%252BKinase%252B%252528GAK%252529%252Bwith%252B4%2525E2%252580%252590Anilino%2525E2%252580%252590quin%252528az%252529oline%252BInhibitors%26aulast%3DAsquith%26aufirst%3DChristopher%2BR.%2BM.%26date%3D2020%26date%3D2020%26volume%3D15%26issue%3D13%26spage%3D1200%26epage%3D1215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enrico  Guarnera</span>, <span class="hlFld-ContribAuthor ">Igor N  Berezovsky</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric drugs and mutations: chances, challenges, and necessity. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>62 </em>, 149-157. <a href="https://doi.org/10.1016/j.sbi.2020.01.010" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.01.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.01.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DAllosteric%252Bdrugs%252Band%252Bmutations%25253A%252Bchances%25252C%252Bchallenges%25252C%252Band%252Bnecessity%26aulast%3DGuarnera%26aufirst%3DEnrico%26date%3D2020%26volume%3D62%26spage%3D149%26epage%3D157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Archana S  Bhat</span>, <span class="hlFld-ContribAuthor ">Richard  Dustin Schaeffer</span>, <span class="hlFld-ContribAuthor ">Lisa  Kinch</span>, <span class="hlFld-ContribAuthor ">Kirill E  Medvedev</span>, <span class="hlFld-ContribAuthor ">Nick V  Grishin</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances suggest increased influence of selective pressure in allostery. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Structural Biology</span><span> <strong>2020,</strong> <em>62 </em>, 183-188. <a href="https://doi.org/10.1016/j.sbi.2020.02.004" title="DOI URL">https://doi.org/10.1016/j.sbi.2020.02.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.sbi.2020.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.sbi.2020.02.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Structural%2520Biology%26atitle%3DRecent%252Badvances%252Bsuggest%252Bincreased%252Binfluence%252Bof%252Bselective%252Bpressure%252Bin%252Ballostery%26aulast%3DBhat%26aufirst%3DArchana%2BS%26date%3D2020%26volume%3D62%26spage%3D183%26epage%3D188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Yun  Li</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Combining Allosteric and Orthosteric Drugs to Overcome Drug Resistance. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (5)
                                     , 336-348. <a href="https://doi.org/10.1016/j.tips.2020.02.001" title="DOI URL">https://doi.org/10.1016/j.tips.2020.02.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.02.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DCombining%252BAllosteric%252Band%252BOrthosteric%252BDrugs%252Bto%252BOvercome%252BDrug%252BResistance%26aulast%3DNi%26aufirst%3DDuan%26date%3D2020%26volume%3D41%26issue%3D5%26spage%3D336%26epage%3D348" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marjorie  Bruder</span>, <span class="hlFld-ContribAuthor ">Gina  Polo</span>, <span class="hlFld-ContribAuthor ">Daniela B. B.  Trivella</span>. </span><span class="cited-content_cbyCitation_article-title">Natural allosteric modulators and their biological targets: molecular signatures and mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2020,</strong> <em>37 </em>
                                    (4)
                                     , 488-514. <a href="https://doi.org/10.1039/C9NP00064J" title="DOI URL">https://doi.org/10.1039/C9NP00064J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9NP00064J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9NP00064J%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DNatural%252Ballosteric%252Bmodulators%252Band%252Btheir%252Bbiological%252Btargets%25253A%252Bmolecular%252Bsignatures%252Band%252Bmechanisms%26aulast%3DBruder%26aufirst%3DMarjorie%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D4%26spage%3D488%26epage%3D514" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Linkai  Mou</span>, <span class="hlFld-ContribAuthor ">Zilong  Ma</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Meng</span>, <span class="hlFld-ContribAuthor ">Wenxing  Li</span>, <span class="hlFld-ContribAuthor ">Shuai  Liang</span>, <span class="hlFld-ContribAuthor ">Xiangyu  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Exploration of the selective binding mechanism of GSK3β via molecular modeling and molecular dynamics simulation studies. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2020,</strong> <em>29 </em>
                                    (4)
                                     , 690-698. <a href="https://doi.org/10.1007/s00044-020-02514-7" title="DOI URL">https://doi.org/10.1007/s00044-020-02514-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02514-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02514-7%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DExploration%252Bof%252Bthe%252Bselective%252Bbinding%252Bmechanism%252Bof%252BGSK3%2525CE%2525B2%252Bvia%252Bmolecular%252Bmodeling%252Band%252Bmolecular%252Bdynamics%252Bsimulation%252Bstudies%26aulast%3DMou%26aufirst%3DLinkai%26date%3D2020%26date%3D2020%26volume%3D29%26issue%3D4%26spage%3D690%26epage%3D698" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shuang  Zhao</span>, <span class="hlFld-ContribAuthor ">Yan-Yan  Zhu</span>, <span class="hlFld-ContribAuthor ">Xiao-Yu  Wang</span>, <span class="hlFld-ContribAuthor ">Yong-Sheng  Liu</span>, <span class="hlFld-ContribAuthor ">Yun-Xiang  Sun</span>, <span class="hlFld-ContribAuthor ">Qing-Jie  Zhao</span>, <span class="hlFld-ContribAuthor ">Hui-Yu  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insight into the Interactions between Structurally Similar Inhibitors and SIRT6. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (7)
                                     , 2601. <a href="https://doi.org/10.3390/ijms21072601" title="DOI URL">https://doi.org/10.3390/ijms21072601</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21072601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21072601%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DStructural%252BInsight%252Binto%252Bthe%252BInteractions%252Bbetween%252BStructurally%252BSimilar%252BInhibitors%252Band%252BSIRT6%26aulast%3DZhao%26aufirst%3DShuang%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D7%26spage%3D2601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Junhong  Qin</span>, <span class="hlFld-ContribAuthor ">Xiaofei  Shen</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Da  Jia</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric inhibitors of the STAT3 signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>190 </em>, 112122. <a href="https://doi.org/10.1016/j.ejmech.2020.112122" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112122</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112122%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252Binhibitors%252Bof%252Bthe%252BSTAT3%252Bsignaling%252Bpathway%26aulast%3DQin%26aufirst%3DJunhong%26date%3D2020%26volume%3D190%26spage%3D112122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xianxian  Liu</span>, <span class="hlFld-ContribAuthor ">Wenchao  Tian</span>, <span class="hlFld-ContribAuthor ">Jinying  Cheng</span>, <span class="hlFld-ContribAuthor ">Dongmei  Li</span>, <span class="hlFld-ContribAuthor ">Tonggang  Liu</span>, <span class="hlFld-ContribAuthor ">Liguo  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Microsecond molecular dynamics simulations reveal the allosteric regulatory mechanism of p53 R249S mutation in p53-associated liver cancer. </span><span class="cited-content_cbyCitation_journal-name">Computational Biology and Chemistry</span><span> <strong>2020,</strong> <em>84 </em>, 107194. <a href="https://doi.org/10.1016/j.compbiolchem.2019.107194" title="DOI URL">https://doi.org/10.1016/j.compbiolchem.2019.107194</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.compbiolchem.2019.107194&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.compbiolchem.2019.107194%26sid%3Dliteratum%253Aachs%26jtitle%3DComputational%2520Biology%2520and%2520Chemistry%26atitle%3DMicrosecond%252Bmolecular%252Bdynamics%252Bsimulations%252Breveal%252Bthe%252Ballosteric%252Bregulatory%252Bmechanism%252Bof%252Bp53%252BR249S%252Bmutation%252Bin%252Bp53-associated%252Bliver%252Bcancer%26aulast%3DLiu%26aufirst%3DXianxian%26date%3D2020%26volume%3D84%26spage%3D107194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Jinyuan  Dai</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Qiang  Fu</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Insight into the mechanism of allosteric activation of PI3Kα by oncoprotein K-Ras4B. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2020,</strong> <em>144 </em>, 643-655. <a href="https://doi.org/10.1016/j.ijbiomac.2019.12.020" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2019.12.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2019.12.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2019.12.020%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DInsight%252Binto%252Bthe%252Bmechanism%252Bof%252Ballosteric%252Bactivation%252Bof%252BPI3K%2525CE%2525B1%252Bby%252Boncoprotein%252BK-Ras4B%26aulast%3DLi%26aufirst%3DXinyi%26date%3D2020%26volume%3D144%26spage%3D643%26epage%3D655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivier  Sheik Amamuddy</span>, <span class="hlFld-ContribAuthor ">Wayde  Veldman</span>, <span class="hlFld-ContribAuthor ">Colleen  Manyumwa</span>, <span class="hlFld-ContribAuthor ">Afrah  Khairallah</span>, <span class="hlFld-ContribAuthor ">Steve  Agajanian</span>, <span class="hlFld-ContribAuthor ">Odeyemi  Oluyemi</span>, <span class="hlFld-ContribAuthor ">Gennady M.  Verkhivker</span>, <span class="hlFld-ContribAuthor ">Özlem  Tastan Bishop</span>. </span><span class="cited-content_cbyCitation_article-title">Integrated Computational Approaches and Tools for Allosteric Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (3)
                                     , 847. <a href="https://doi.org/10.3390/ijms21030847" title="DOI URL">https://doi.org/10.3390/ijms21030847</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21030847&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21030847%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DIntegrated%252BComputational%252BApproaches%252Band%252BTools%252Bfor%252BAllosteric%252BDrug%252BDiscovery%26aulast%3DSheik%2BAmamuddy%26aufirst%3DOlivier%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D3%26spage%3D847" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaolan  Chen</span>, <span class="hlFld-ContribAuthor ">Chunqiong  Li</span>, <span class="hlFld-ContribAuthor ">Dada  Wang</span>, <span class="hlFld-ContribAuthor ">Yu  Chen</span>, <span class="hlFld-ContribAuthor ">Na  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in the Discovery of CK2 Allosteric Inhibitors: From Traditional Screening to Structure-Based Design. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (4)
                                     , 870. <a href="https://doi.org/10.3390/molecules25040870" title="DOI URL">https://doi.org/10.3390/molecules25040870</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25040870&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25040870%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DRecent%252BAdvances%252Bin%252Bthe%252BDiscovery%252Bof%252BCK2%252BAllosteric%252BInhibitors%25253A%252BFrom%252BTraditional%252BScreening%252Bto%252BStructure-Based%252BDesign%26aulast%3DChen%26aufirst%3DXiaolan%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D4%26spage%3D870" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zarko  Gagic</span>, <span class="hlFld-ContribAuthor ">Dusan  Ruzic</span>, <span class="hlFld-ContribAuthor ">Nemanja  Djokovic</span>, <span class="hlFld-ContribAuthor ">Teodora  Djikic</span>, <span class="hlFld-ContribAuthor ">Katarina  Nikolic</span>. </span><span class="cited-content_cbyCitation_article-title">In silico Methods for Design of Kinase Inhibitors as Anticancer Drugs. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00873" title="DOI URL">https://doi.org/10.3389/fchem.2019.00873</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00873&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00873%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DIn%252Bsilico%252BMethods%252Bfor%252BDesign%252Bof%252BKinase%252BInhibitors%252Bas%252BAnticancer%252BDrugs%26aulast%3DGagic%26aufirst%3DZarko%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhongjie  Liang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhu</span>, <span class="hlFld-ContribAuthor ">Xingyi  Liu</span>, <span class="hlFld-ContribAuthor ">Guang  Hu</span>. </span><span class="cited-content_cbyCitation_article-title">Role of protein-protein interactions in allosteric drug design for DNA methyltransferases. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 49-84. <a href="https://doi.org/10.1016/bs.apcsb.2019.12.005" title="DOI URL">https://doi.org/10.1016/bs.apcsb.2019.12.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.apcsb.2019.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.apcsb.2019.12.005%26sid%3Dliteratum%253Aachs%26atitle%3DRole%252Bof%252Bprotein-protein%252Binteractions%252Bin%252Ballosteric%252Bdrug%252Bdesign%252Bfor%252BDNA%252Bmethyltransferases%26aulast%3DLiang%26aufirst%3DZhongjie%26date%3D2020%26spage%3D49%26epage%3D84%26pub%3DElsevier%26date%3D2020%26volume%3D121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Yuran  Qiu</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Jun  Pu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Emergence of allosteric drug-resistance mutations: new challenges for allosteric drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (1)
                                     , 177-184. <a href="https://doi.org/10.1016/j.drudis.2019.10.006" title="DOI URL">https://doi.org/10.1016/j.drudis.2019.10.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2019.10.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2019.10.006%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DEmergence%252Bof%252Ballosteric%252Bdrug-resistance%252Bmutations%25253A%252Bnew%252Bchallenges%252Bfor%252Ballosteric%252Bdrug%252Bdiscovery%26aulast%3DLu%26aufirst%3DShaoyong%26date%3D2020%26volume%3D25%26issue%3D1%26spage%3D177%26epage%3D184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chunqiong  Li</span>, <span class="hlFld-ContribAuthor ">Xuewen  Zhang</span>, <span class="hlFld-ContribAuthor ">Na  Zhang</span>, <span class="hlFld-ContribAuthor ">Yue  Zhou</span>, <span class="hlFld-ContribAuthor ">Guohui  Sun</span>, <span class="hlFld-ContribAuthor ">Lijiao  Zhao</span>, <span class="hlFld-ContribAuthor ">Rugang  Zhong</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Biological Evaluation of CK2 Allosteric Fragments through Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (1)
                                     , 237. <a href="https://doi.org/10.3390/molecules25010237" title="DOI URL">https://doi.org/10.3390/molecules25010237</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25010237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25010237%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Band%252BBiological%252BEvaluation%252Bof%252BCK2%252BAllosteric%252BFragments%252Bthrough%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DLi%26aufirst%3DChunqiong%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D1%26spage%3D237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fei  Li</span>, <span class="hlFld-ContribAuthor ">Xi  Jiang</span>, <span class="hlFld-ContribAuthor ">Ling-Ling  Luo</span>, <span class="hlFld-ContribAuthor ">Yue-Ming  Xu</span>, <span class="hlFld-ContribAuthor ">Xing-Xu  Huang</span>, <span class="hlFld-ContribAuthor ">Cheng  Huang</span>, <span class="hlFld-ContribAuthor ">Yu  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">A piggyBac-based TANGO GFP assay for high throughput screening of GPCR ligands in live cells. </span><span class="cited-content_cbyCitation_journal-name">Cell Communication and Signaling</span><span> <strong>2019,</strong> <em>17 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12964-019-0359-x" title="DOI URL">https://doi.org/10.1186/s12964-019-0359-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12964-019-0359-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12964-019-0359-x%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Communication%2520and%2520Signaling%26atitle%3DA%252BpiggyBac-based%252BTANGO%252BGFP%252Bassay%252Bfor%252Bhigh%252Bthroughput%252Bscreening%252Bof%252BGPCR%252Bligands%252Bin%252Blive%252Bcells%26aulast%3DLi%26aufirst%3DFei%26date%3D2019%26date%3D2019%26volume%3D17%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging roles of allosteric modulators in the regulation of protein‐protein interactions (PPIs): A new paradigm for PPI drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2019,</strong> <em>39 </em>
                                    (6)
                                     , 2314-2342. <a href="https://doi.org/10.1002/med.21585" title="DOI URL">https://doi.org/10.1002/med.21585</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21585&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21585%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DEmerging%252Broles%252Bof%252Ballosteric%252Bmodulators%252Bin%252Bthe%252Bregulation%252Bof%252Bprotein%2525E2%252580%252590protein%252Binteractions%252B%252528PPIs%252529%25253A%252BA%252Bnew%252Bparadigm%252Bfor%252BPPI%252Bdrug%252Bdiscovery%26aulast%3DNi%26aufirst%3DDuan%26date%3D2019%26date%3D2019%26volume%3D39%26issue%3D6%26spage%3D2314%26epage%3D2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinyi  Liu</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>, <span class="hlFld-ContribAuthor ">Kun  Song</span>, <span class="hlFld-ContribAuthor ">Qiancheng  Shen</span>, <span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Qian  Li</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Qi  Wang</span>, <span class="hlFld-ContribAuthor ">Yingyi  Chen</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Jing  Wu</span>, <span class="hlFld-ContribAuthor ">Chunquan  Sheng</span>, <span class="hlFld-ContribAuthor ">Guoqiang  Chen</span>, <span class="hlFld-ContribAuthor ">Yaqin  Liu</span>, <span class="hlFld-ContribAuthor ">Xuefeng  Lu</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Unraveling allosteric landscapes of allosterome with ASD. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2019,</strong> <em>10 </em><a href="https://doi.org/10.1093/nar/gkz958" title="DOI URL">https://doi.org/10.1093/nar/gkz958</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkz958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkz958%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DUnraveling%252Ballosteric%252Blandscapes%252Bof%252Ballosterome%252Bwith%252BASD%26aulast%3DLiu%26aufirst%3DXinyi%26date%3D2019%26date%3D2019%26volume%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Duan  Ni</span>, <span class="hlFld-ContribAuthor ">Xinyi  Li</span>, <span class="hlFld-ContribAuthor ">Xinheng  He</span>, <span class="hlFld-ContribAuthor ">Hao  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Shaoyong  Lu</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging K-RasG12C through covalent inhibitors: Mission possible?. </span><span class="cited-content_cbyCitation_journal-name">Pharmacology & Therapeutics</span><span> <strong>2019,</strong> <em>202 </em>, 1-17. <a href="https://doi.org/10.1016/j.pharmthera.2019.06.007" title="DOI URL">https://doi.org/10.1016/j.pharmthera.2019.06.007</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pharmthera.2019.06.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pharmthera.2019.06.007%26sid%3Dliteratum%253Aachs%26jtitle%3DPharmacology%2520%2526%2520Therapeutics%26atitle%3DDrugging%252BK-RasG12C%252Bthrough%252Bcovalent%252Binhibitors%25253A%252BMission%252Bpossible%25253F%26aulast%3DNi%26aufirst%3DDuan%26date%3D2019%26volume%3D202%26spage%3D1%26epage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Likun  Zhao</span>, <span class="hlFld-ContribAuthor ">Luhua  Lai</span>, <span class="hlFld-ContribAuthor ">Zhuqing  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">How calcium ion binding induces the conformational transition of the calmodulin N-terminal domain—an atomic level characterization. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2019,</strong> <em>21 </em>
                                    (36)
                                     , 19795-19804. <a href="https://doi.org/10.1039/C9CP03917A" title="DOI URL">https://doi.org/10.1039/C9CP03917A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CP03917A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CP03917A%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DHow%252Bcalcium%252Bion%252Bbinding%252Binduces%252Bthe%252Bconformational%252Btransition%252Bof%252Bthe%252Bcalmodulin%252BN-terminal%252Bdomain%2525E2%252580%252594an%252Batomic%252Blevel%252Bcharacterization%26aulast%3DZhao%26aufirst%3DLikun%26date%3D2019%26date%3D2019%26volume%3D21%26issue%3D36%26spage%3D19795%26epage%3D19804" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Surface representation of the X-ray crystal structure of p38α complexed to an allosteric compound <b>10</b> (<b>1</b>) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW">3NEW</a>). The orthosteric ATP binding site and allosteric site at the C-lobe of the kinase are highlighted in blue and red, respectively. Multiple sequence alignments of allosteric site residues in p38α, p38β, p38γ, and p38δ are shown, with identical and similarity residues marked by black and gray, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Surface representation of the X-ray crystal structure of BCR-ABL1 complexed to both orthosteric drug imatinib and the allosteric inhibitor ABL001 (asciminib) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MO4">5MO4</a>), with orthosteric and allosteric sites highlighted in blue and red, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Cartoon representation of the X-ray crystal structure of BCR-ABL1. The drug-resistant mutation residues at the orthosteric site (Thr315) and allosteric site (Ala337, Pro465, Val468, and Ile502) are depicted by sphere representations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Total number of X-ray or NMR structures of allosteric proteins and yearly growth of total structures in the AlloSteric Database (ASD). Representative structures of allosteric proteins are shown.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Major class distribution of the structures of allosteric proteins in ASD.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Yearly growth of allosteric modulators in ASD. The chemical structures of the marketed allosteric drugs are indicated in the chart.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Residue compositions in the orthosteric (light blue) and allosteric (blue) sites calculated from ASD version 3.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Workflow of the rational discovery of SIRT6 allosteric activators. The potential allosteric site of SIRT6 was predicted using Allosite (<a href="http://mdl.shsmu.edu.cn/AST" class="extLink">http://mdl.shsmu.edu.cn/AST</a>). SIRT6 is shown as surface colored in gray, with the orthosteric and predicted allosteric sites highlighted in blue and red, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Detailed interactions between MDL-801 and SIRT6 at the allosteric site determined by the cocrystal structure (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F">5Y2F</a>). SIRT6 is shown in gray, and MDL-801 is depicted by sticks with carbon atoms colored in yellow. The key residues within the allosteric site of MDL-801 are represented by sticks with carbon atoms colored in cyan.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0010.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. Workflow of the rational discovery of a STAT3 allosteric inhibitor. An X-ray structure of STAT3 (residues 130–722) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1">1BG1</a>) is colored by the domain. The predicted allosteric site at the coiled-coil domain (CCD), large-scale molecular docking of compound libraries at the CCD allosteric site, and allosteric interaction evaluation after docking were executed automatically by AlloFinder (<a href="http://mdl.shsmu.edu.cn/ALF" class="extLink">http://mdl.shsmu.edu.cn/ALF</a>). Enlarged portion shows the potential interactions between the allosteric inhibitor K116 (<b>6</b>) and STAT3, consistent with mutagenesis experiments.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0011.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Workflow of the rational discovery of GPX4 allosteric activators. The allosteric site of GPX4 was predicted using CAVITY and CorrSite (<a href="http://www.pkumdl.cn/cavityplus" class="extLink">http://www.pkumdl.cn/cavityplus</a>) based on the crystal structure of GPX4 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBI">2OBI</a>). The predicted allosteric site (red) is on the opposite side of the orthosteric site (blue). The predicted binding mode of <b>1d4</b> (<b>8</b>) represented by blue sticks to GPX4 is shown, with key residues within the binding site highlighted in orange by stick representations.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig12" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 12</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0012.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 12. Structural overview of cathepsin K in complex with its allosteric inhibitor NSC13345 (<b>9</b>) and substrate TCO (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94">5J94</a>). The backbone structure of cathepsin K is shown in gray. The allosteric site (site 6) and orthosteric site are highlighted in red and blue, respectively. The upper panel shows the chemical structure of <b>9</b>, and the lower panel shows the detailed interactions between <b>9</b> and cathepsin K. Hydrogen bonds are displayed in red dashed lines, and residues within the allosteric site are represented by sticks. Substrate TCO is manually docked into the orthosteric site based on the superposition of PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94">5J94</a> with PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q6K">1Q6K</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig12"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig13" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 13</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/medium/jm-2018-01749k_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0013.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 13. (A) Chemical structure of ANS (<b>10</b>) and structural overview of CDK2 in complex with <b>10</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF">3PXF</a>). (B) The binding site of <b>10</b> is in a closed conformation in the active state of c-Src (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Y57">1Y57</a>). (C) Structural comparison between the <b>10</b>-bound CDK2 (yellow) and active state of c-Src (blue) shows that the allosteric site is in a closed conformation mainly due to the displacements of the C-helix and A-loop. (D) The binding site of <b>10</b> is also in a closed conformation in the inactive state of c-Src (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC">2SRC</a>). The C-helix, A-loop, and β-sheets involved in the binding site of <b>10</b> are colored in orange, pink, and cyan, respectively.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-14/acs.jmedchem.8b01749/20190719/images/large/jm-2018-01749k_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.8b01749&amp;id=fig13"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i28">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64710" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64710" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 198 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ribeiro, A. A. S. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz, V.</span></span> <span> </span><span class="NLM_article-title">A Chemical Perspective on Allostery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6488</span>– <span class="NLM_lpage">6502</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00543</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00543" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGhsrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6488-6502&author=A.+A.+S.+T.+Ribeiroauthor=V.+Ortiz&title=A+Chemical+Perspective+on+Allostery&doi=10.1021%2Facs.chemrev.5b00543"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">A Chemical Perspective on Allostery</span></div><div class="casAuthors">Ribeiro, Andre A. S. T.; Ortiz, Vanessa</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6488-6502</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Much work has been done in the past decade to quantify the phenomenon of allosteric communication in proteins.  Every new study unveils an extra piece of the puzzle in our search for an understanding of allostery that allows us to make predictions on the response of a protein to medically relevant stimuli such as pathol. mutations or drug binding.  This review summarizes recent advances in the anal. of mechanisms of allosteric communication in proteins, and combines this new knowledge to offer a perspective of allostery which is consistent with chem. views of mol. processes.  First, we review recent work, particularly computational, on the characterization of signal propagation and conformational changes in allosteric proteins.  We then compare different models of allostery, and discuss the significance of the concept of an allosteric pathway.  We argue that allostery can be rationalized in terms of pathways of residues that efficiently transmit energy between different binding sites.  We then provide examples that show how this picture could account for most of the obsd. data, since energy flow may be manifested as changes in both structure and dynamics.  We conclude by acknowledging that the proposed view is still a simplification and should not be taken as a rigorous model of allosteric communication in proteins.  Nevertheless, simple pictures like this can go a long way in improving our understanding of many complex phenomena obsd. in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwFr3gYH5v97Vg90H21EOLACvtfcHk0lhAzGlc5sfFOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGhsrY%253D&md5=13482cdbc6b7669e5a735b5ff4b98efd</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00543&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00543%26sid%3Dliteratum%253Aachs%26aulast%3DRibeiro%26aufirst%3DA.%2BA.%2BS.%2BT.%26aulast%3DOrtiz%26aufirst%3DV.%26atitle%3DA%2520Chemical%2520Perspective%2520on%2520Allostery%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6488%26epage%3D6502%26doi%3D10.1021%2Facs.chemrev.5b00543" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">The Concept of Allosteric Modulation: An Overview</span>. <i>Drug Discovery Today: Technol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">e223</span>– <span class="NLM_lpage">e228</span>, <span class="refDoi"> DOI: 10.1016/j.ddtec.2012.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ddtec.2012.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24050272" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=e223-e228&author=J.-P.+Changeux&title=The+Concept+of+Allosteric+Modulation%3A+An+Overview&doi=10.1016%2Fj.ddtec.2012.07.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.ddtec.2012.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ddtec.2012.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DThe%2520Concept%2520of%2520Allosteric%2520Modulation%253A%2520An%2520Overview%26jtitle%3DDrug%2520Discovery%2520Today%253A%2520Technol.%26date%3D2013%26volume%3D10%26spage%3De223%26epage%3De228%26doi%3D10.1016%2Fj.ddtec.2012.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dokholyan, N. V.</span></span> <span> </span><span class="NLM_article-title">Controlling Allosteric Networks in Proteins</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6463</span>– <span class="NLM_lpage">6487</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00544</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00544" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XivVeltL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6463-6487&author=N.+V.+Dokholyan&title=Controlling+Allosteric+Networks+in+Proteins&doi=10.1021%2Facs.chemrev.5b00544"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Controlling Allosteric Networks in Proteins</span></div><div class="casAuthors">Dokholyan, Nikolay V.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6463-6487</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the phys. and evolutionary origin of protein allostery defined as conformational changes induced by ligand binding, as well as its importance to protein regulation, drug discovery, and biol. processes in living systems; including exptl. and computational protein engineering approaches for control of protein function by modulation of allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmC_LdB7LS-bVg90H21EOLACvtfcHk0lhAzGlc5sfFOQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XivVeltL0%253D&md5=9294593fdeb42a59d015bd84a10091b9</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00544%26sid%3Dliteratum%253Aachs%26aulast%3DDokholyan%26aufirst%3DN.%2BV.%26atitle%3DControlling%2520Allosteric%2520Networks%2520in%2520Proteins%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6463%26epage%3D6487%26doi%3D10.1021%2Facs.chemrev.5b00544" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radivojac, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Post-Translational Modification Codes</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">455</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2012.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2012.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22884395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFKisbfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2012&pages=447-455&author=R.+Nussinovauthor=C.+J.+Tsaiauthor=F.+Xinauthor=P.+Radivojac&title=Allosteric+Post-Translational+Modification+Codes&doi=10.1016%2Fj.tibs.2012.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric post-translational modification codes</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Xin, Fuxiao; Radivojac, Predrag</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">447-455</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Post-translational modifications (PTMs) have been recognized to impact protein function in two ways: (i) orthosterically, via direct recognition by protein domains or through interference with binding; and (ii) allosterically, via conformational changes induced at the functional sites.  Because different chem. types of PTMs elicit different structural alterations, the effects of combinatorial codes of PTMs are vastly larger than previously believed.  Combined with orthosteric PTMs, the impact of PTMs on cellular regulation is immense.  From an evolutionary standpoint, harnessing this immense, yet highly specific, PTM code is an extremely efficient vehicle that can save a cell several-fold in gene no. and speed up its response to environmental change.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzvEY3AcxUjrVg90H21EOLACvtfcHk0lgF3cyddoYlYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFKisbfJ&md5=93f2ef54896d461366150e9da3b7450f</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2012.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2012.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DXin%26aufirst%3DF.%26aulast%3DRadivojac%26aufirst%3DP.%26atitle%3DAllosteric%2520Post-Translational%2520Modification%2520Codes%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2012%26volume%3D37%26spage%3D447%26epage%3D455%26doi%3D10.1016%2Fj.tibs.2012.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span> <span> </span><span class="NLM_article-title">Allostery in Disease and in Drug Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>153</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">305</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.03.034</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2013.03.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23582321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=153&publication_year=2013&pages=293-305&author=R.+Nussinovauthor=C.+J.+Tsai&title=Allostery+in+Disease+and+in+Drug+Discovery&doi=10.1016%2Fj.cell.2013.03.034"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in Disease and in Drug Discovery</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">153</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">293-305</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Allostery is largely assocd. with conformational and functional transitions in individual proteins.  This concept can be extended to consider the impact of conformational perturbations on cellular function and disease states.  Here, we clarify the concept of allostery and how it controls physiol. activities.  We focus on the challenging questions of how allostery can both cause disease and contribute to development of new therapeutics.  We aim to increase the awareness of the linkage between disease symptoms on the cellular level and specific aberrant allosteric actions on the mol. level and to emphasize the potential of allosteric drugs in innovative therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz5bXPgUjfFLVg90H21EOLACvtfcHk0lgF3cyddoYlYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVChu70%253D&md5=14a91c0d364187044e5ad3e048868514</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.03.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.03.034%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26atitle%3DAllostery%2520in%2520Disease%2520and%2520in%2520Drug%2520Discovery%26jtitle%3DCell%26date%3D2013%26volume%3D153%26spage%3D293%26epage%3D305%26doi%3D10.1016%2Fj.cell.2013.03.034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goodey, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Regulation and Catalysis Emerge via a Common Route</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">474</span>– <span class="NLM_lpage">482</span>, <span class="refDoi"> DOI: 10.1038/nchembio.98</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.98" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18641628" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXoslClsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2008&pages=474-482&author=N.+M.+Goodeyauthor=S.+J.+Benkovic&title=Allosteric+Regulation+and+Catalysis+Emerge+via+a+Common+Route&doi=10.1038%2Fnchembio.98"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric regulation and catalysis emerge via a common route</span></div><div class="casAuthors">Goodey, Nina M.; Benkovic, Stephen J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">474-482</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric regulation of protein function is a mechanism by which an event in one place of a protein structure causes an effect at another site, much like the behavior of a telecommunications network in which a collection of transmitters, receivers and transceivers communicate with each other across long distances.  For example, ligand binding or an amino acid mutation at an allosteric site can alter enzymic activity or binding affinity in a distal region such as the active site or a second binding site.  The mechanism of this site-to-site communication is of great interest, esp. since allosteric effects must be considered in drug design and protein engineering.  In this review, conformational mobility as the common route between allosteric regulation and catalysis is discussed.  We summarize recent exptl. data and the resulting insights into allostery within proteins, and we discuss the nature of future studies and the new applications that may result from increased understanding of this regulatory mechanism.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5blf5uB1i9bVg90H21EOLACvtfcHk0lgF3cyddoYlYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXoslClsb8%253D&md5=74d502f64c00fcef21c623bc84db3159</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.98&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.98%26sid%3Dliteratum%253Aachs%26aulast%3DGoodey%26aufirst%3DN.%2BM.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26atitle%3DAllosteric%2520Regulation%2520and%2520Catalysis%2520Emerge%2520via%2520a%2520Common%2520Route%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D4%26spage%3D474%26epage%3D482%26doi%3D10.1038%2Fnchembio.98" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Conformational Barcodes Direct Signaling in the Cell</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">1509</span>– <span class="NLM_lpage">1521</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.06.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2013.06.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24010710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSht77L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=1509-1521&author=R.+Nussinovauthor=B.+Maauthor=C.+J.+Tsaiauthor=P.+Csermely&title=Allosteric+Conformational+Barcodes+Direct+Signaling+in+the+Cell&doi=10.1016%2Fj.str.2013.06.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Conformational Barcodes Direct Signaling in the Cell</span></div><div class="casAuthors">Nussinov, Ruth; Ma, Buyong; Tsai, Chung-Jung; Csermely, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1509-1521</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The cellular network is highly interconnected.  Pathways merge and diverge.  They proceed through shared proteins and may change directions.  How are cellular pathways controlled and their directions decided, coded, and read.  These questions become particularly acute when we consider that a small no. of pathways, such as signaling pathways that regulate cell fates, cell proliferation, and cell death in development, are extensively exploited.  This review focuses on these signaling questions from the structural standpoint and discusses the literature in this light.  All co-occurring allosteric events (including posttranslational modifications, pathogen binding, and gain-of-function mutations) collectively tag the protein functional site with a unique barcode.  The barcode shape is read by an interacting mol., which transmits the signal.  A conformational barcode provides an intracellular address label, which selectively favors binding to one partner and quenches binding to others, and, in this way, dets. the pathway direction, and, eventually, the cell's response and fate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGlw4jCU8JqLVg90H21EOLACvtfcHk0lgF3cyddoYlYQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSht77L&md5=02198e089623b6d6f09b6c53a3965935</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.06.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.06.002%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DCsermely%26aufirst%3DP.%26atitle%3DAllosteric%2520Conformational%2520Barcodes%2520Direct%2520Signaling%2520in%2520the%2520Cell%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D1509%26epage%3D1521%26doi%3D10.1016%2Fj.str.2013.06.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Receptors: From Electric Organ to Cognition</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1146/annurev.pharmtox.010909.105741</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev.pharmtox.010909.105741" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20055696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXisVeltbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=1-38&author=J.-P.+Changeux&title=Allosteric+Receptors%3A+From+Electric+Organ+to+Cognition&doi=10.1146%2Fannurev.pharmtox.010909.105741"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric receptors: From electric organ to cognition</span></div><div class="casAuthors">Changeux, Jean-Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-38</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  This autobiog. account by Jean-Pierre Chageux covers the past 50 yr of allosteric protein research beginning with the development of the concept of allosteric proteins and its application to pharmacol. receptors.  It continues with the identification of the nicotinic acetylcholine receptor, the discovery of its mol. organization, the structure of the acetylcholine-binding site and of the ion channel, and the demonstration of its allosteric transitions.  The article then traces the origins of the concept of allosteric modulator and its consequences in pharmacol.  It proceeds with the theory of selective stabilization of synapses and with exptl. studies carried out on the contribution of the nicotinic receptor in the morphogenesis of the neuromuscular junction.  The knowledge acquired with the nicotinic receptor is further exploited to reach higher levels of brain organization, and the contribution of nicotinic receptors to the action of nicotine on reward and cognition is explored, in particular, using a novel exptl. strategy that combines nicotinic receptor genes knock-out and stereotaxic gene re-expression.  Theor. models of cognitive functions are proposed that link the mol. to the cognitive level.  The report ends with a discussion on nicotinic receptors and the pharmacol. of the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJFMHTsb-Mx7Vg90H21EOLACvtfcHk0ljtJSZ4zHfIvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXisVeltbs%253D&md5=60318b638eacddd3e43fce0f86bc3869</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.010909.105741&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.010909.105741%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DAllosteric%2520Receptors%253A%2520From%2520Electric%2520Organ%2520to%2520Cognition%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D1%26epage%3D38%26doi%3D10.1146%2Fannurev.pharmtox.010909.105741" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratcioglu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gursoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keskin, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Ras Conformational Ensembles, Allostery, and Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6607</span>– <span class="NLM_lpage">6665</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00542</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00542" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6607-6665&author=S.+Luauthor=H.+Jangauthor=S.+Muratciogluauthor=A.+Gursoyauthor=O.+Keskinauthor=R.+Nussinovauthor=J.+Zhang&title=Ras+Conformational+Ensembles%2C+Allostery%2C+and+Signaling&doi=10.1021%2Facs.chemrev.5b00542"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Ras Conformational Ensembles, Allostery, and Signaling</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Muratcioglu, Serena; Gursoy, Attila; Keskin, Ozlem; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6607-6665</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Ras proteins are classical members of small GTPases that function as mol. switches by alternating between inactive GDP-bound and active GTP-bound states.  Ras activation is regulated by guanine nucleotide exchange factors that catalyze the exchange of GDP by GTP, and inactivation is terminated by GTPase-activating proteins that accelerate the intrinsic GTP hydrolysis rate by orders of magnitude.  In this review, we focus on data that have accumulated over the past few years pertaining to the conformational ensembles and the allosteric regulation of Ras proteins and their interpretation from our conformational landscape standpoint.  The Ras ensemble embodies all states, including the ligand-bound conformations, the activated (or inactivated) allosteric modulated states, post-translationally modified states, mutational states, transition states, and nonfunctional states serving as a reservoir for emerging functions.  The ensemble is shifted by distinct mutational events, cofactors, post-translational modifications, and different membrane compns.  A better understanding of Ras biol. can contribute to therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY2oezVaRCvbVg90H21EOLACvtfcHk0ljtJSZ4zHfIvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhu74%253D&md5=7600b971015bd3cf635ca1b4299dc7d5</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00542&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00542%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DMuratcioglu%26aufirst%3DS.%26aulast%3DGursoy%26aufirst%3DA.%26aulast%3DKeskin%26aufirst%3DO.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRas%2520Conformational%2520Ensembles%252C%2520Allostery%252C%2520and%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6607%26epage%3D6665%26doi%3D10.1021%2Facs.chemrev.5b00542" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lisi, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, J. P.</span></span> <span> </span><span class="NLM_article-title">Allostery in Enzyme Catalysis</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">123</span>– <span class="NLM_lpage">130</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2017.08.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2017.08.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28865247" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2017&pages=123-130&author=G.+P.+Lisiauthor=J.+P.+Loria&title=Allostery+in+Enzyme+Catalysis&doi=10.1016%2Fj.sbi.2017.08.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery in enzyme catalysis</span></div><div class="casAuthors">Lisi, George P.; Loria, J. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">123-130</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Modern interpretations of allostery typically rely on conformational ensembles to describe enzyme function.  Conformational motions controlling these ensembles are often stimulated or quenched by allosteric effectors, and are crit. to optimizing ligand binding pockets and enzyme architectures.  Thus, enzymes rely on dynamic allosteric pathways that transmit long-range binding information to control catalysis.  In this review, we provide a brief discussion of the ever-expanding principles of allosteric regulation in enzyme catalysis and highlight in-depth studies of three enzymes that have contributed to the paradigms of dynamic allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqaQWntFm9s0rVg90H21EOLACvtfcHk0ljtJSZ4zHfIvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlOrtrnO&md5=6f017e27e75b6e139ec75058d2b5326e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2017.08.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2017.08.002%26sid%3Dliteratum%253Aachs%26aulast%3DLisi%26aufirst%3DG.%2BP.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26atitle%3DAllostery%2520in%2520Enzyme%2520Catalysis%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2017%26volume%3D47%26spage%3D123%26epage%3D130%26doi%3D10.1016%2Fj.sbi.2017.08.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, A. W.</span></span> <span> </span><span class="NLM_article-title">Allostery: An Illustrated Definition for the “Second Secret of Life”</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">420</span>– <span class="NLM_lpage">425</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2008.05.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2008.05.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18706817" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtVyktLfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2008&pages=420-425&author=A.+W.+Fenton&title=Allostery%3A+An+Illustrated+Definition+for+the+%E2%80%9CSecond+Secret+of+Life%E2%80%9D&doi=10.1016%2Fj.tibs.2008.05.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery: an illustrated definition for the 'second secret of life'</span></div><div class="casAuthors">Fenton, Aron W.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">420-425</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Although allosteric regulation is the 'second secret of life', the mol. mechanisms that give rise to allostery currently elude understanding.  In my opinion, exptl. progress is hampered by a commonly used but misleading definition of allostery as protein structural changes that are elicited by the binding of a single ligand.  Allostery is more strictly defined in functional terms as a comparison of how one ligand binds in the absence, vs. the presence, of a second ligand.  Therefore, as each of the two binding events involves two protein complexes, a study of allostery must consider four complexes and not just two.  Such a comparison can distinguish allosteric from non-allosteric protein changes, the importance of which is frequently overlooked.  When a study of all four complexes is not feasible, an alternative, albeit limited, strategy can identify subsets of allosteric-specific changes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbEo7wm-kW9LVg90H21EOLACvtfcHk0ljtJSZ4zHfIvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtVyktLfO&md5=34040a91726dd4fe1174bf00e4a4e59c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2008.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2008.05.009%26sid%3Dliteratum%253Aachs%26aulast%3DFenton%26aufirst%3DA.%2BW.%26atitle%3DAllostery%253A%2520An%2520Illustrated%2520Definition%2520for%2520the%2520%25E2%2580%259CSecond%2520Secret%2520of%2520Life%25E2%2580%259D%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2008%26volume%3D33%26spage%3D420%26epage%3D425%26doi%3D10.1016%2Fj.tibs.2008.05.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation as a Unifying Mechanism for Receptor Function and Regulation</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>166</i></span>,  <span class="NLM_fpage">1084</span>– <span class="NLM_lpage">1102</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2016.08.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2016.08.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27565340" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVehu7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2016&pages=1084-1102&author=J.+P.+Changeuxauthor=A.+Christopoulos&title=Allosteric+Modulation+as+a+Unifying+Mechanism+for+Receptor+Function+and+Regulation&doi=10.1016%2Fj.cell.2016.08.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Modulation as a Unifying Mechanism for Receptor Function and Regulation</span></div><div class="casAuthors">Changeux, Jean-Pierre; Christopoulos, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1084-1102</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Four major receptor families enable cells to respond to chem. and phys. signals from their proximal environment.  The ligand- and voltage-gated ion channels, G-protein-coupled receptors, nuclear hormone receptors, and receptor tyrosine kinases are all allosteric proteins that carry multiple, spatially distinct, yet conformationally linked ligand-binding sites.  Recent studies point to common mechanisms governing the allosteric transitions of these receptors, including the impact of oligomerization, pre-existing and functionally distinct conformational ensembles, intrinsically disordered regions, and the occurrence of allosteric modulatory sites.  Importantly, synthetic allosteric modulators are being discovered for these receptors, providing an enriched, yet challenging, landscape for novel therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCsajEiGEo6rVg90H21EOLACvtfcHk0ljr5ktGKNb9HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVehu7zK&md5=7507e4abbd4c08319e8d80ded942c425</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2016.08.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2016.08.015%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.%2BP.%26aulast%3DChristopoulos%26aufirst%3DA.%26atitle%3DAllosteric%2520Modulation%2520as%2520a%2520Unifying%2520Mechanism%2520for%2520Receptor%2520Function%2520and%2520Regulation%26jtitle%3DCell%26date%3D2016%26volume%3D166%26spage%3D1084%26epage%3D1102%26doi%3D10.1016%2Fj.cell.2016.08.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wootten, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span> <span> </span><span class="NLM_article-title">Emerging Paradigms in GPCR Allostery: Implications for Drug Discovery</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">630</span>– <span class="NLM_lpage">644</span>, <span class="refDoi"> DOI: 10.1038/nrd4052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd4052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23903222" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=630-644&author=D.+Woottenauthor=A.+Christopoulosauthor=P.+M.+Sexton&title=Emerging+Paradigms+in+GPCR+Allostery%3A+Implications+for+Drug+Discovery&doi=10.1038%2Fnrd4052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging paradigms in GPCR allostery: implications for drug discovery</span></div><div class="casAuthors">Wootten, Denise; Christopoulos, Arthur; Sexton, Patrick M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">630-644</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allosteric ligands bind to G protein-coupled receptors (GPCRs; also known as seven-transmembrane receptors) at sites that are distinct from the sites to which endogenous ligands bind.  The existence of allosteric ligands has enriched the ways in which the functions of GPCRs can be manipulated for potential therapeutic benefit, yet the complexity of their actions provides both challenges and opportunities for drug screening and development.  Converging avenues of research in areas such as biased signalling by allosteric ligands and the mechanisms by which allosteric ligands modulate the effects of diverse endogenous ligands have provided new insights into how interactions between allosteric ligands and GPCRs could be exploited for drug discovery.  These new findings have the potential to alter how screening for allosteric drugs is performed and may increase the chances of success in the development of allosteric modulators as clin. lead compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrxRlmQUCUlsLVg90H21EOLACvtfcHk0ljr5ktGKNb9HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1SjtbvO&md5=59b74e5fbd5eeb832785dd879daa7c77</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1038%2Fnrd4052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4052%26sid%3Dliteratum%253Aachs%26aulast%3DWootten%26aufirst%3DD.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26atitle%3DEmerging%2520Paradigms%2520in%2520GPCR%2520Allostery%253A%2520Implications%2520for%2520Drug%2520Discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2013%26volume%3D12%26spage%3D630%26epage%3D644%26doi%3D10.1038%2Fnrd4052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rettenmaier, T. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sadowsky, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doak, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arkin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">A Small-Molecule Mimic of a Peptide Docking Motif Inhibits the Protein Kinase PDK1</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">18590</span>– <span class="NLM_lpage">18595</span>, <span class="refDoi"> DOI: 10.1073/pnas.1415365112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1415365112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25518860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=18590-18595&author=T.+J.+Rettenmaierauthor=J.+D.+Sadowskyauthor=N.+D.+Thomsenauthor=S.+C.+Chenauthor=A.+K.+Doakauthor=M.+R.+Arkinauthor=J.+A.+Wells&title=A+Small-Molecule+Mimic+of+a+Peptide+Docking+Motif+Inhibits+the+Protein+Kinase+PDK1&doi=10.1073%2Fpnas.1415365112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A small-molecule mimic of a peptide docking motif inhibits the protein kinase PDK1</span></div><div class="casAuthors">Rettenmaier, T. Justin; Sadowsky, Jack D.; Thomsen, Nathan D.; Chen, Steven C.; Doak, Allison K.; Arkin, Michelle R.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">52</span>),
    <span class="NLM_cas:pages">18590-18595</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">There is great interest in developing selective protein kinase inhibitors by targeting allosteric sites, but these sites often involve protein-protein or protein-peptide interfaces that are very challenging to target with small mols.  Here we present a systematic approach to targeting a functionally conserved allosteric site on the protein kinase PDK1 called the PDK1-interacting fragment (PIF)tide-binding site, or PIF pocket.  More than two dozen prosurvival and progrowth kinases dock a conserved peptide tail into this binding site, which recruits them to PDK1 to become activated.  Using a site-directed chem. screen, we identified and chem. optimized ligand-efficient, selective, and cell-penetrant small mols. (mol. wt. ∼380 Da) that compete with the peptide docking motif for binding to PDK1.  We solved the first high-resoln. structure of a peptide docking motif (PIFtide) bound to PDK1 and mapped binding energy hot spots using mutational anal.  We then solved structures of PDK1 bound to the allosteric small mols., which revealed a binding mode that remarkably mimics three of five hot-spot residues in PIFtide.  These allosteric small mols. are substrate-selective PDK1 inhibitors when used as single agents, but when combined with an ATP-competitive inhibitor, they completely suppress the activation of the downstream kinases.  This work provides a promising new scaffold for the development of high-affinity PIF pocket ligands, which may be used to enhance the anticancer activity of existing PDK1 inhibitors.  Moreover, our results provide further impetus for exploring the helix αC patches of other protein kinases as potential therapeutic targets even though they involve protein-protein interfaces.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPeMoENY8xXbVg90H21EOLACvtfcHk0ljr5ktGKNb9HQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitFCrt7jJ&md5=4a1d88395865677d5a9c8d3dee58eca2</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1415365112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1415365112%26sid%3Dliteratum%253Aachs%26aulast%3DRettenmaier%26aufirst%3DT.%2BJ.%26aulast%3DSadowsky%26aufirst%3DJ.%2BD.%26aulast%3DThomsen%26aufirst%3DN.%2BD.%26aulast%3DChen%26aufirst%3DS.%2BC.%26aulast%3DDoak%26aufirst%3DA.%2BK.%26aulast%3DArkin%26aufirst%3DM.%2BR.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DA%2520Small-Molecule%2520Mimic%2520of%2520a%2520Peptide%2520Docking%2520Motif%2520Inhibits%2520the%2520Protein%2520Kinase%2520PDK1%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D18590%26epage%3D18595%26doi%3D10.1073%2Fpnas.1415365112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adrián, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iacob, R. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powers, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dierks, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okram, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wojciechowski, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fendrich, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strauss, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vajpai, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grzesiek, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tuntland, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bursulaya, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Azam, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engen, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daley, G. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span> <span> </span><span class="NLM_article-title">Targeting Bcr-Abl by Combining Allosteric with ATP-Binding-Site Inhibitors</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>463</i></span>,  <span class="NLM_fpage">501</span>– <span class="NLM_lpage">506</span>, <span class="refDoi"> DOI: 10.1038/nature08675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature08675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20072125" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=463&publication_year=2010&pages=501-506&author=J.+Zhangauthor=F.+J.+Adri%C3%A1nauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+G.+Liauthor=R.+E.+Iacobauthor=T.+Simauthor=J.+Powersauthor=C.+Dierksauthor=F.+Sunauthor=G.+R.+Guoauthor=Q.+Dingauthor=B.+Okramauthor=Y.+Choiauthor=A.+Wojciechowskiauthor=X.+Dengauthor=G.+Liuauthor=G.+Fendrichauthor=A.+Straussauthor=N.+Vajpaiauthor=S.+Grzesiekauthor=T.+Tuntlandauthor=Y.+Liuauthor=B.+Bursulayaauthor=M.+Azamauthor=P.+W.+Manleyauthor=J.+R.+Engenauthor=G.+Q.+Daleyauthor=M.+Warmuthauthor=N.+S.+Gray&title=Targeting+Bcr-Abl+by+Combining+Allosteric+with+ATP-Binding-Site+Inhibitors&doi=10.1038%2Fnature08675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting Bcr-Abl by combining allosteric with ATP-binding-site inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Adrian, Francisco J.; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Li, Allen G.; Iacob, Roxana E.; Sim, Taebo; Powers, John; Dierks, Christine; Sun, Fangxian; Guo, Gui-Rong; Ding, Qiang; Okram, Barun; Choi, Yongmun; Wojciechowski, Amy; Deng, Xianming; Liu, Guoxun; Fendrich, Gabriele; Strauss, Andre; Vajpai, Navratna; Grzesiek, Stephan; Tuntland, Tove; Liu, Yi; Bursulaya, Badry; Azam, Mohammad; Manley, Paul W.; Engen, John R.; Daley, George Q.; Warmuth, Markus; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">463</span>
        (<span class="NLM_cas:issue">7280</span>),
    <span class="NLM_cas:pages">501-506</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In an effort to find new pharmacol. modalities to overcome resistance to ATP-binding-site inhibitors of Bcr-Abl, we recently reported the discovery of GNF-2, a selective allosteric Bcr-Abl inhibitor.  Here, using soln. NMR, X-ray crystallog., mutagenesis and hydrogen exchange mass spectrometry, we show that GNF-2 binds to the myristate-binding site of Abl, leading to changes in the structural dynamics of the ATP-binding site.  GNF-5, an analog of GNF-2 with improved pharmacokinetic properties, when used in combination with the ATP-competitive inhibitors imatinib or nilotinib, suppressed the emergence of resistance mutations in vitro, displayed additive inhibitory activity in biochem. and cellular assays against T315I mutant human Bcr-Abl and displayed in vivo efficacy against this recalcitrant mutant in a murine bone-marrow transplantation model.  These results show that therapeutically relevant inhibition of Bcr-Abl activity can be achieved with inhibitors that bind to the myristate-binding site and that combining allosteric and ATP-competitive inhibitors can overcome resistance to either agent alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoUtkKFuTQx1LVg90H21EOLACvtfcHk0liCE4gzqEVMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlCnuw%253D%253D&md5=46c122384fa004d27865c77f4a9165b7</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fnature08675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08675%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DAdri%25C3%25A1n%26aufirst%3DF.%2BJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLi%26aufirst%3DA.%2BG.%26aulast%3DIacob%26aufirst%3DR.%2BE.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DPowers%26aufirst%3DJ.%26aulast%3DDierks%26aufirst%3DC.%26aulast%3DSun%26aufirst%3DF.%26aulast%3DGuo%26aufirst%3DG.%2BR.%26aulast%3DDing%26aufirst%3DQ.%26aulast%3DOkram%26aufirst%3DB.%26aulast%3DChoi%26aufirst%3DY.%26aulast%3DWojciechowski%26aufirst%3DA.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFendrich%26aufirst%3DG.%26aulast%3DStrauss%26aufirst%3DA.%26aulast%3DVajpai%26aufirst%3DN.%26aulast%3DGrzesiek%26aufirst%3DS.%26aulast%3DTuntland%26aufirst%3DT.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DAzam%26aufirst%3DM.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DEngen%26aufirst%3DJ.%2BR.%26aulast%3DDaley%26aufirst%3DG.%2BQ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520Bcr-Abl%2520by%2520Combining%2520Allosteric%2520with%2520ATP-Binding-Site%2520Inhibitors%26jtitle%3DNature%26date%3D2010%26volume%3D463%26spage%3D501%26epage%3D506%26doi%3D10.1038%2Fnature08675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wenthur, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gentry, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mathews, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Drugs for Allosteric Sites on Receptors</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">165</span>– <span class="NLM_lpage">184</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010611-134525</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-pharmtox-010611-134525" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24111540" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXjsVSqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=165-184&author=C.+J.+Wenthurauthor=P.+R.+Gentryauthor=T.+P.+Mathewsauthor=C.+W.+Lindsley&title=Drugs+for+Allosteric+Sites+on+Receptors&doi=10.1146%2Fannurev-pharmtox-010611-134525"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Drugs for allosteric sites on receptors</span></div><div class="casAuthors">Wenthur, Cody J.; Gentry, Patrick R.; Mathews, Thomas P.; Lindlsye, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">165-184</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  The presence of druggable, topog. distinct allosteric sites on a wide range of receptor families has offered new paradigms for small mols. to modulate receptor function.  Moreover, ligands that target allosteric sites offer significant advantages over the corresponding orthosteric ligands in terms of selectivity, including subtype selectivity within receptor families, and can also impart improved physicochem. properties.  However, allosteric ligands are not a panacea.  Many chem. issues (e.g., flat structure-activity relationships) and pharmacol. issues (e.g., ligand-biased signaling) that are allosteric centric have emerged.  Notably, the fact that allosteric sites are less evolutionarily conserved leads to improved selectivity; however, this can also lead to species differences that can hinder safety assessment.  Many allosteric ligands possess mol. switches, wherein a small structural change (chem. or metabolic) can modulate the mode of pharmacol. or receptor subtype selectivity.  As the field has matured, as described here, key principles and strategies have emerged for the design of ligands/drugs for allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqx6zv9s5vbLLVg90H21EOLACvtfcHk0liCE4gzqEVMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXjsVSqtrs%253D&md5=56aff98cecab954cbabac96c94f25f47</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010611-134525&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010611-134525%26sid%3Dliteratum%253Aachs%26aulast%3DWenthur%26aufirst%3DC.%2BJ.%26aulast%3DGentry%26aufirst%3DP.%2BR.%26aulast%3DMathews%26aufirst%3DT.%2BP.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DDrugs%2520for%2520Allosteric%2520Sites%2520on%2520Receptors%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2014%26volume%3D54%26spage%3D165%26epage%3D184%26doi%3D10.1146%2Fannurev-pharmtox-010611-134525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De
Smet, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeliet, P.</span></span> <span> </span><span class="NLM_article-title">Allosteric Targeting of Receptor Tyrosine Kinases</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1113</span>– <span class="NLM_lpage">1120</span>, <span class="refDoi"> DOI: 10.1038/nbt.3028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnbt.3028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25380447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Shu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2014&pages=1113-1120&author=F.+De%0ASmetauthor=A.+Christopoulosauthor=P.+Carmeliet&title=Allosteric+Targeting+of+Receptor+Tyrosine+Kinases&doi=10.1038%2Fnbt.3028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric targeting of receptor tyrosine kinases</span></div><div class="casAuthors">De Smet, Frederik; Christopoulos, Arthur; Carmeliet, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1113-1120</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The drug discovery landscape has been transformed over the past decade by the discovery of allosteric modulators of all major mammalian receptor superfamilies.  Allosteric ligands are a rich potential source of drugs and drug targets with clear therapeutic advantages.  G protein-coupled receptors, ligand-gated ion channels and intracellular nuclear hormone receptors have all been targeted by allosteric modulators.  More recently, a receptor tyrosine kinase (RTK) has been targeted by an extracellular small-mol. allosteric modulator.  Allosteric mechanisms of structurally distinct mols. that target the various receptor families are more alike than originally anticipated and include selectivity, orthosteric probe dependence and pathway-biased signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhUEiFrNdlxrVg90H21EOLACvtfcHk0liCE4gzqEVMxQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Shu7c%253D&md5=5dd8e5bf1d6578a418dce00686cd2334</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Fnbt.3028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.3028%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BSmet%26aufirst%3DF.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DCarmeliet%26aufirst%3DP.%26atitle%3DAllosteric%2520Targeting%2520of%2520Receptor%2520Tyrosine%2520Kinases%26jtitle%3DNat.%2520Biotechnol.%26date%3D2014%26volume%3D32%26spage%3D1113%26epage%3D1120%26doi%3D10.1038%2Fnbt.3028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hauser, A.
S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Attwood, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rask-Andersen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schiöth, H. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gloriam, D. E.</span></span> <span> </span><span class="NLM_article-title">Trends in GPCR Drug Discovery: New Agents, Targets and Indications</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">829</span>– <span class="NLM_lpage">842</span>, <span class="refDoi"> DOI: 10.1038/nrd.2017.178</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd.2017.178" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29075003" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhslansbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2017&pages=829-842&author=A.%0AS.+Hauserauthor=M.+M.+Attwoodauthor=M.+Rask-Andersenauthor=H.+B.+Schi%C3%B6thauthor=D.+E.+Gloriam&title=Trends+in+GPCR+Drug+Discovery%3A+New+Agents%2C+Targets+and+Indications&doi=10.1038%2Fnrd.2017.178"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Trends in GPCR drug discovery: new agents, targets and indications</span></div><div class="casAuthors">Hauser, Alexander S.; Attwood, Misty M.; Rask-Andersen, Mathias; Schioth, Helgi B.; Gloriam, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">829-842</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) are the most intensively studied drug targets, mostly due to their substantial involvement in human pathophysiol. and their pharmacol. tractability.  Here, we report an up-to-date anal. of all GPCR drugs and agents in clin. trials, which reveals current trends across mol. types, drug targets and therapeutic indications, including showing that 475 drugs (∼34% of all drugs approved by the US Food and Drug Administration (FDA)) act at 108 unique GPCRs.  Approx. 321 agents are currently in clin. trials, of which ∼20% target 66 potentially novel GPCR targets without an approved drug, and the no. of biol. drugs, allosteric modulators and biased agonists has increased.  The major disease indications for GPCR modulators show a shift towards diabetes, obesity and Alzheimer disease, although several central nervous system disorders are also highly represented.  The 224 (56%) non-olfactory GPCRs that have not yet been explored in clin. trials have broad untapped therapeutic potential, particularly in genetic and immune system disorders.  Finally, we provide an interactive online resource to analyze and infer trends in GPCR drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9iw1QqR9GKrVg90H21EOLACvtfcHk0lhZYkF4COHHkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhslansbzF&md5=560893258493bd96c0af24d3bef08b6f</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2017.178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2017.178%26sid%3Dliteratum%253Aachs%26aulast%3DHauser%26aufirst%3DA.%2BS.%26aulast%3DAttwood%26aufirst%3DM.%2BM.%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DSchi%25C3%25B6th%26aufirst%3DH.%2BB.%26aulast%3DGloriam%26aufirst%3DD.%2BE.%26atitle%3DTrends%2520in%2520GPCR%2520Drug%2520Discovery%253A%2520New%2520Agents%252C%2520Targets%2520and%2520Indications%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2017%26volume%3D16%26spage%3D829%26epage%3D842%26doi%3D10.1038%2Fnrd.2017.178" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hardy, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span> <span> </span><span class="NLM_article-title">Searching for New Allosteric Sites in Enzymes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">706</span>– <span class="NLM_lpage">715</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2004.10.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2004.10.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15582395" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVClurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=706-715&author=J.+A.+Hardyauthor=J.+A.+Wells&title=Searching+for+New+Allosteric+Sites+in+Enzymes&doi=10.1016%2Fj.sbi.2004.10.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Searching for new allosteric sites in enzymes</span></div><div class="casAuthors">Hardy, Jeanne A.; Wells, James A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">706-715</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review and discussion.  The discovery of new allosteric sites generates opportunities for the identification of novel pharmaceuticals and increases the understanding of basic biol. processes.  Increasingly, allosteric sites are being discovered in various families of proteins by several methods, paving the way for the development of entirely new classes of drugs with a wide range of chemotypes.  New allosteric sites in enzymes have been discovered both incidentally and by directed means, and the mechanisms by which allosteric activation and inhibition occur at these sites have been investigated.  By exploring recent structurally well-characterized examples, trends have begun to emerge for both the modes of binding and mechanisms of inhibition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpatMfcNeAgPrVg90H21EOLACvtfcHk0lhZYkF4COHHkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVClurnO&md5=27470dff57a3b3756656c23664dd0da7</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2004.10.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2004.10.009%26sid%3Dliteratum%253Aachs%26aulast%3DHardy%26aufirst%3DJ.%2BA.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26atitle%3DSearching%2520for%2520New%2520Allosteric%2520Sites%2520in%2520Enzymes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2004%26volume%3D14%26spage%3D706%26epage%3D715%26doi%3D10.1016%2Fj.sbi.2004.10.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD v3.0: Unraveling Allosteric Regulation with Structural Mechanisms and Biological Networks</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>44</i></span>,  <span class="NLM_fpage">D527</span>– <span class="NLM_lpage">D535</span>, <span class="refDoi"> DOI: 10.1093/nar/gkv902</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkv902" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26365237" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D527-D535&author=Q.+Shenauthor=G.+Wangauthor=S.+Liauthor=X.+Liuauthor=S.+Luauthor=Z.+Chenauthor=K.+Songauthor=J.+Yanauthor=L.+Gengauthor=Z.+Huangauthor=W.+Huangauthor=G.+Chenauthor=J.+Zhang&title=ASD+v3.0%3A+Unraveling+Allosteric+Regulation+with+Structural+Mechanisms+and+Biological+Networks&doi=10.1093%2Fnar%2Fgkv902"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">ASD v3.0: unraveling allosteric regulation with structural mechanisms and biological networks</span></div><div class="casAuthors">Shen, Qiancheng; Wang, Guanqiao; Li, Shuai; Liu, Xinyi; Lu, Shaoyong; Chen, Zhongjie; Song, Kun; Yan, Junhao; Geng, Lv; Huang, Zhimin; Huang, Wenkang; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D527-D535</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allosteric regulation, the most direct and efficient way of regulating protein function, is induced by the binding of a ligand at one site that is topog. distinct from an orthosteric site.  Allosteric Database (ASD, available online at http://mdl.shsmu.edu.cn/ ASD) has been developed to provide comprehensive information featuring allosteric regulation.  With increasing data, fundamental questions pertaining to allostery are currently receiving more attention from the mechanism of allosteric changes in an individual protein to the entire effect of the changes in the interconnected network in the cell.  Thus, the following novel features were added to this updated version: (i) structural mechanisms of more than 1600 allosteric actions were elucidated by a comparison of site structures before and after the binding of an modulator; (ii) 261 allosteric networks were identified to unveil how the allosteric action in a single protein would propagate to affect downstream proteins; (iii) two of the largest human allosteromes, protein kinases and GPCRs, were thoroughly constructed; and (iv) web interface and data organization were completely redesigned for efficient access.  In addn., allosteric data have largely expanded in this update.  These updates are useful for facilitating the investigation of allosteric mechanisms, dynamic networks and drug discoveries.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPuMFm_lHedLVg90H21EOLACvtfcHk0lhZYkF4COHHkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfO&md5=16abbb6763bc125f68c67fe8f2f812dd</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv902&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv902%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DYan%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%2520v3.0%253A%2520Unraveling%2520Allosteric%2520Regulation%2520with%2520Structural%2520Mechanisms%2520and%2520Biological%2520Networks%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD527%26epage%3DD535%26doi%3D10.1093%2Fnar%2Fgkv902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wagner, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durrant, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malmstrom, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feher, V. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amaro, R. E.</span></span> <span> </span><span class="NLM_article-title">Emerging Computational Methods for the Rational Discovery of Allosteric Drugs</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6370</span>– <span class="NLM_lpage">6390</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00631</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00631" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XmtVWis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6370-6390&author=J.+R.+Wagnerauthor=C.+T.+Leeauthor=J.+D.+Durrantauthor=R.+D.+Malmstromauthor=V.+A.+Feherauthor=R.+E.+Amaro&title=Emerging+Computational+Methods+for+the+Rational+Discovery+of+Allosteric+Drugs&doi=10.1021%2Facs.chemrev.5b00631"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Emerging Computational Methods for the Rational Discovery of Allosteric Drugs</span></div><div class="casAuthors">Wagner, Jeffrey R.; Lee, Christopher T.; Durrant, Jacob D.; Malmstrom, Robert D.; Feher, Victoria A.; Amaro, Rommie E.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6370-6390</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric drug development holds promise for delivering medicines that are more selective and less toxic than those that target orthosteric sites.  To date, the discovery of allosteric binding sites and lead compds. has been mostly serendipitous, achieved through high-throughput screening.  Over the past decade, structural data has become more readily available for larger protein systems and more membrane protein classes (e.g., GPCRs and ion channels), which are common allosteric drug targets.  In parallel, improved simulation methods now provide better atomistic understanding of the protein dynamics and cooperative motions that are crit. to allosteric mechanisms.  As a result of these advances, the field of predictive allosteric drug development is now on the cusp of a new era of rational structure-based computational methods.  Here, we review algorithms that predict allosteric sites based on sequence data and mol. dynamics simulations, describe tools that assess the druggability of these pockets, and discuss how Markov state models and topol. analyses provide insight into the relationship between protein dynamics and allosteric drug binding.  In each section, we first provide an overview of the various method classes before describing relevant algorithms and software packages.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzQHNOd1yzZrVg90H21EOLACvtfcHk0lhZYkF4COHHkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmtVWis7Y%253D&md5=f5e4bdbcbbec577e20edb2ab6ec6e96f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00631%26sid%3Dliteratum%253Aachs%26aulast%3DWagner%26aufirst%3DJ.%2BR.%26aulast%3DLee%26aufirst%3DC.%2BT.%26aulast%3DDurrant%26aufirst%3DJ.%2BD.%26aulast%3DMalmstrom%26aufirst%3DR.%2BD.%26aulast%3DFeher%26aufirst%3DV.%2BA.%26aulast%3DAmaro%26aufirst%3DR.%2BE.%26atitle%3DEmerging%2520Computational%2520Methods%2520for%2520the%2520Rational%2520Discovery%2520of%2520Allosteric%2520Drugs%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6370%26epage%3D6390%26doi%3D10.1021%2Facs.chemrev.5b00631" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Designed Covalent Allosteric Modulators: An Emerging Paradigm in Drug Discovery</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>22</i></span>,  <span class="NLM_fpage">447</span>– <span class="NLM_lpage">453</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2016.11.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2016.11.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27888140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2017&pages=447-453&author=S.+Luauthor=J.+Zhang&title=Designed+Covalent+Allosteric+Modulators%3A+An+Emerging+Paradigm+in+Drug+Discovery&doi=10.1016%2Fj.drudis.2016.11.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Designed covalent allosteric modulators: an emerging paradigm in drug discovery</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">447-453</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Covalent allosteric modulators possess the pharmacol. advantages (high potency, extended duration of action and low drug resistance) of covalent ligands and the addnl. benefit of the higher specificity and lower toxicity of allosteric modulators.  This approach is gaining increasing recognition as a valuable tool in drug discovery.  Here, we review the recent advances in the design of covalent allosteric modulators with an emphasis on successful examples.  A broad spectrum of protein targets capable of being modulated by them reflects the prevalence of this strategy.  We also discuss the challenges and future directions in the development of covalent allosteric modulators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqnMtSOrLWxdrVg90H21EOLACvtfcHk0lg74bdb113qdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhvFOju7zJ&md5=bc656114f3f956ce44ba06c1a7669da4</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2016.11.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2016.11.013%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDesigned%2520Covalent%2520Allosteric%2520Modulators%253A%2520An%2520Emerging%2520Paradigm%2520in%2520Drug%2520Discovery%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2017%26volume%3D22%26spage%3D447%26epage%3D453%26doi%3D10.1016%2Fj.drudis.2016.11.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perna, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reisen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geppert, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weisel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simister, P. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feller, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wessler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span> <span> </span><span class="NLM_article-title">Inhibiting Helicobacter Pylori HtrA Protease by Addressing a Computationally Predicted Allosteric Ligand Binding Site</span>. <i>Chem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3583</span>– <span class="NLM_lpage">3590</span>, <span class="refDoi"> DOI: 10.1039/C4SC01443J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2FC4SC01443J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26819700" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVKrsbnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3583-3590&author=A.+M.+Pernaauthor=F.+Reisenauthor=T.+P.+Schmidtauthor=T.+Geppertauthor=M.+Pillongauthor=M.+Weiselauthor=B.+Hoyauthor=P.+C.+Simisterauthor=S.+M.+Fellerauthor=S.+Wesslerauthor=G.+Schneider&title=Inhibiting+Helicobacter+Pylori+HtrA+Protease+by+Addressing+a+Computationally+Predicted+Allosteric+Ligand+Binding+Site&doi=10.1039%2FC4SC01443J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibiting Helicobacter pylori HtrA protease by addressing a computationally predicted allosteric ligand binding site</span></div><div class="casAuthors">Perna, Anna Maria; Reisen, Felix; Schmidt, Thomas P.; Geppert, Tim; Pillong, Max; Weisel, Martin; Hoy, Benjamin; Simister, Philip C.; Feller, Stephan M.; Wessler, Silja; Schneider, Gisbert</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Science</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3583-3590</span>CODEN:
                <span class="NLM_cas:coden">CSHCCN</span>;
        ISSN:<span class="NLM_cas:issn">2041-6520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Helicobacter pylori is assocd. with inflammatory diseases and can cause gastric cancer and mucosa-assocd. lymphoma.  One of the bacterium's key proteins is high temp. requirement A (HpHtrA) protein, an extracellular serine protease that cleaves E-cadherin of gastric epithelial cells, which leads to loss of cell-cell adhesion.  Inhibition of HpHtrA may constitute an intervention strategy against H. pylori infection.  Guided by the computational prediction of hypothetical ligand binding sites on the surface of HpHtrA, we performed residue mutation expts. that confirmed the functional relevance of an allosteric region.  We virtually screened for potential ligands addressing this surface cleft located between the catalytic and PDZ1 domains.  Our receptor-based computational method represents protein surface pockets in terms of graph frameworks and retrieves small mols. that satisfy the constraints given by the pocket framework.  A new chem. entity was identified that blocked E-cadherin cleavage in vitro by direct binding to HpHtrA, and efficiently blocked pathogen transmigration across the gastric epithelial barrier.  A preliminary crystal structure of HpHtrA confirms the validity of a comparative "homol." model of the enzyme, which we used for the computational study.  The results of this study demonstrate that addressing orphan protein surface cavities of target macromols. can lead to new bioactive ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq83JdNlgAcrVg90H21EOLACvtfcHk0lg74bdb113qdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVKrsbnF&md5=6c5842c5694fa43745ccfecaf1851e19</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1039%2FC4SC01443J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4SC01443J%26sid%3Dliteratum%253Aachs%26aulast%3DPerna%26aufirst%3DA.%2BM.%26aulast%3DReisen%26aufirst%3DF.%26aulast%3DSchmidt%26aufirst%3DT.%2BP.%26aulast%3DGeppert%26aufirst%3DT.%26aulast%3DPillong%26aufirst%3DM.%26aulast%3DWeisel%26aufirst%3DM.%26aulast%3DHoy%26aufirst%3DB.%26aulast%3DSimister%26aufirst%3DP.%2BC.%26aulast%3DFeller%26aufirst%3DS.%2BM.%26aulast%3DWessler%26aufirst%3DS.%26aulast%3DSchneider%26aufirst%3DG.%26atitle%3DInhibiting%2520Helicobacter%2520Pylori%2520HtrA%2520Protease%2520by%2520Addressing%2520a%2520Computationally%2520Predicted%2520Allosteric%2520Ligand%2520Binding%2520Site%26jtitle%3DChem.%2520Sci.%26date%3D2014%26volume%3D5%26spage%3D3583%26epage%3D3590%26doi%3D10.1039%2FC4SC01443J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cho, K.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.-Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rambahal, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sagineedu, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaari, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanslas, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hancock, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gorfe, A. A.</span></span> <span> </span><span class="NLM_article-title">Andrographolide Derivatives Inhibit Guanine Nucleotide Exchange and Abrogate Oncogenic Ras Function</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">10201</span>– <span class="NLM_lpage">10206</span>, <span class="refDoi"> DOI: 10.1073/pnas.1300016110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1300016110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23737504" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOmsLzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=10201-10206&author=H.+J.+Hockerauthor=K.-J.+Choauthor=C.-Y.+K.+Chenauthor=N.+Rambahalauthor=S.+R.+Sagineeduauthor=K.+Shaariauthor=J.+Stanslasauthor=J.+F.+Hancockauthor=A.+A.+Gorfe&title=Andrographolide+Derivatives+Inhibit+Guanine+Nucleotide+Exchange+and+Abrogate+Oncogenic+Ras+Function&doi=10.1073%2Fpnas.1300016110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Andrographolide derivatives inhibit guanine nucleotide exchange and abrogate oncogenic Ras function</span></div><div class="casAuthors">Hocker, Harrison J.; Cho, Kwang-Jin; Chen, Chung-Ying K.; Rambahal, Nandini; Sagineedu, Sreenivasa Rao; Shaari, Khozirah; Stanslas, Johnson; Hancock, John F.; Gorfe, Alemayehu A.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">10201-10206, S10201/1-S10201/12</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Aberrant signaling by oncogenic mutant rat sarcoma (Ras) proteins occurs in ∼15% of all human tumors, yet direct inhibition of Ras by small mols. has remained elusive.  Recently, several small-mol. ligands have been discovered that directly bind Ras and inhibit its function by interfering with exchange factor binding.  However, it is unclear whether, or how, these ligands could lead to drugs that act against constitutively active oncogenic mutant Ras.  Using a dynamics-based pocket identification scheme, ensemble docking, and innovative cell-based assays, here we show that andrographolide (AGP) - a bicyclic diterpenoid lactone isolated from Andrographis paniculata - and its benzylidene derivs. bind to transient pockets on Kirsten-Ras (K-Ras) and inhibit GDP-GTP exchange.  As expected for inhibitors of exchange factor binding, AGP derivs. reduced GTP loading of wild-type K-Ras in response to acute EGF stimulation with a concomitant redn. in MAPK activation.  Remarkably, however, prolonged treatment with AGP derivs. also reduced GTP loading of, and signal transmission by, oncogenic mutant K-RasG12V.  In sum, the combined anal. of our computational and cell biol. results show that AGP derivs. directly bind Ras, block GDP-GTP exchange, and inhibit both wild-type and oncogenic K-Ras signaling.  Importantly, our findings not only show that nucleotide exchange factors are required for oncogenic Ras signaling but also demonstrate that inhibiting nucleotide exchange is a valid approach to abrogating the function of oncogenic mutant Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCdMqypiyfGrVg90H21EOLACvtfcHk0ljZAoOS1_nrAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOmsLzP&md5=7d39d426f5b107b87ad76cb8eb9a7f26</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1300016110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1300016110%26sid%3Dliteratum%253Aachs%26aulast%3DHocker%26aufirst%3DH.%2BJ.%26aulast%3DCho%26aufirst%3DK.-J.%26aulast%3DChen%26aufirst%3DC.-Y.%2BK.%26aulast%3DRambahal%26aufirst%3DN.%26aulast%3DSagineedu%26aufirst%3DS.%2BR.%26aulast%3DShaari%26aufirst%3DK.%26aulast%3DStanslas%26aufirst%3DJ.%26aulast%3DHancock%26aufirst%3DJ.%2BF.%26aulast%3DGorfe%26aufirst%3DA.%2BA.%26atitle%3DAndrographolide%2520Derivatives%2520Inhibit%2520Guanine%2520Nucleotide%2520Exchange%2520and%2520Abrogate%2520Oncogenic%2520Ras%2520Function%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D10201%26epage%3D10206%26doi%3D10.1073%2Fpnas.1300016110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chin, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Identification of a Cellularly Active SIRT6 Allosteric Activator</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">1118</span>– <span class="NLM_lpage">1126</span>, <span class="refDoi"> DOI: 10.1038/s41589-018-0150-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fs41589-018-0150-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30374165" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&pages=1118-1126&author=Z.+Huangauthor=J.+Zhaoauthor=W.+Dengauthor=Y.+Chenauthor=J.+Shangauthor=K.+Songauthor=L.+Zhangauthor=C.+Wangauthor=S.+Luauthor=X.+Yangauthor=B.+Heauthor=J.+Minauthor=H.+Huauthor=M.+Tanauthor=J.+Xuauthor=Q.+Zhangauthor=J.+Zhongauthor=X.+Sunauthor=Z.+Maoauthor=H.+Linauthor=M.+Xiaoauthor=Y.+E.+Chinauthor=H.+Jiangauthor=Y.+Xuauthor=G.+Chenauthor=J.+Zhang&title=Identification+of+a+Cellularly+Active+SIRT6+Allosteric+Activator&doi=10.1038%2Fs41589-018-0150-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of a cellularly active SIRT6 allosteric activator</span></div><div class="casAuthors">Huang, Zhimin; Zhao, Junxing; Deng, Wei; Chen, Yingyi; Shang, Jialin; Song, Kun; Zhang, Lu; Wang, Chengxiang; Lu, Shaoyong; Yang, Xiuyan; He, Bin; Min, Jinrong; Hu, Hao; Tan, Minjia; Xu, Jianrong; Zhang, Qiufen; Zhong, Jie; Sun, Xiaoxiang; Mao, Zhiyong; Lin, Houwen; Xiao, Mingzhe; Chin, Y. Eugene; Jiang, Hualiang; Xu, Ying; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1118-1126</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">SIRT6, a member of the SIRT deacetylase family, is responsible for deacetylation of histone H3 Nε-acetyl-lysines 9 (H3K9ac) and 56 (H3K56ac).  As a tumor suppressor, SIRT6 has frequently been found to have low expression in various cancers.  Here, we report the identification of MDL-800, a selective SIRT6 activator.  MDL-800 increased the deacetylase activity of SIRT6 by up to 22-fold via binding to an allosteric site; this interaction led to a global decrease in H3K9ac and H3K56ac levels in human hepatocellular carcinoma (HCC) cells.  Consequently, MDL-800 inhibited the proliferation of HCC cells via SIRT6-driven cell-cycle arrest and was effective in a tumor xenograft model.  Together, these data demonstrate that pharmacol. activation of SIRT6 is a potential therapeutic approach for the treatment of HCC.  MDL-800 is a first-in-class small-mol. cellular SIRT6 activator that can be used to physiol. and pathol. investigate the roles of SIRT6 deacetylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrzFPmr-laqlLVg90H21EOLACvtfcHk0ljZAoOS1_nrAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVCitL7O&md5=5fb5c1de5b6be0291ffa45debf72516e</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fs41589-018-0150-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41589-018-0150-0%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DDeng%26aufirst%3DW.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DC.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DHe%26aufirst%3DB.%26aulast%3DMin%26aufirst%3DJ.%26aulast%3DHu%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DSun%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DZ.%26aulast%3DLin%26aufirst%3DH.%26aulast%3DXiao%26aufirst%3DM.%26aulast%3DChin%26aufirst%3DY.%2BE.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520a%2520Cellularly%2520Active%2520SIRT6%2520Allosteric%2520Activator%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2018%26volume%3D14%26spage%3D1118%26epage%3D1126%26doi%3D10.1038%2Fs41589-018-0150-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">AlloFinder: A Strategy for Allosteric Modulator Discovery and Allosterome Analyses</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W451</span>– <span class="NLM_lpage">W458</span>, <span class="refDoi"> DOI: 10.1093/nar/gky374</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky374" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29757429" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyru7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W451-W458&author=M.+Huangauthor=K.+Songauthor=X.+Liuauthor=S.+Luauthor=Q.+Shenauthor=R.+Wangauthor=J.+Gaoauthor=Y.+Hongauthor=Q.+Liauthor=D.+Niauthor=J.+Xuauthor=G.+Chenauthor=J.+Zhang&title=AlloFinder%3A+A+Strategy+for+Allosteric+Modulator+Discovery+and+Allosterome+Analyses&doi=10.1093%2Fnar%2Fgky374"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">AlloFinder: a strategy for allosteric modulator discovery and allosterome analyses</span></div><div class="casAuthors">Huang, Min; Song, Kun; Liu, Xinyi; Lu, Shaoyong; Shen, Qiancheng; Wang, Renxiao; Gao, Jingze; Hong, Yuanyuan; Li, Qian; Ni, Duan; Xu, Jianrong; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W451-W458</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery tweaks innumerable biol. processes and plays a fundamental role in human disease and drug discovery.  Exploration of allostery has thus been regarded as a crucial requirement for research on biol. mechanisms and the development of novel therapeutics.  Here, based on our previously developed allosteric data and methods, we present an interactive platform called AlloFinder that identifies potential endogenous or exogenous allosteric modulators and their involvement in human allosterome.  AlloFinder automatically amalgamates allosteric site identification, allosteric screening and allosteric scoring evaluation of modulator-protein complexes to identify allosteric modulators, followed by allosterome mapping analyses of predicted allosteric sites and modulators in human proteome.  This web server exhibits prominent performance in the reemergence of allosteric metabolites and exogenous allosteric modulators in known allosteric proteins.  Specifically, AlloFinder enables identification of allosteric metabolites for metabolic enzymes and screening of potential allosteric compds. for disease-related targets.  Significantly, the feasibility of AlloFinder to discover allosteric modulators was tested in a real case of signal transduction and activation of transcription 3 (STAT3) and validated by mutagenesis and functional expts.  Collectively, AlloFinder is expected to contribute to exploration of the mechanisms of allosteric regulation between metabolites and metabolic enzymes, and to accelerate allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolD0xX0ogJuLVg90H21EOLACvtfcHk0ljZAoOS1_nrAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyru7Y%253D&md5=0144562803e465e28d9a7237f0e97a47</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky374%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGao%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DNi%26aufirst%3DD.%26aulast%3DXu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAlloFinder%253A%2520A%2520Strategy%2520for%2520Allosteric%2520Modulator%2520Discovery%2520and%2520Allosterome%2520Analyses%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW451%26epage%3DW458%26doi%3D10.1093%2Fnar%2Fgky374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Conrad, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Angeli, J. P. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Novel Allosteric Activators for Ferroptosis Regulator Glutathione Peroxidase 4</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">275</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXot1Cjsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=266-275&author=C.+Liauthor=X.+Dengauthor=W.+Zhangauthor=X.+Xieauthor=M.+Conradauthor=Y.+Liuauthor=J.+P.+F.+Angeliauthor=L.+Lai&title=Novel+Allosteric+Activators+for+Ferroptosis+Regulator+Glutathione+Peroxidase+4&doi=10.1021%2Facs.jmedchem.8b00315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Allosteric Activators for Ferroptosis Regulator Glutathione Peroxidase 4</span></div><div class="casAuthors">Li, Cong; Deng, Xiaobing; Zhang, Weilin; Xie, Xiaowen; Conrad, Marcus; Liu, Ying; Angeli, Jose Pedro Friedmann; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">266-275</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Glutathione peroxidase 4 (GPX4) is essential for cell membrane repair, inflammation suppression, and ferroptosis inhibition.  GPX4 upregulation provides unique drug discovery opportunities for inflammation and ferroptosis-related diseases.  However, rational design of protein activators is challenging.  Until now, no compd. has been reported to activate the enzyme activity of GPX4.  Here, we identified a potential allosteric site in GPX4 and successfully found eight GPX4 activators using a novel computational strategy and exptl. studies.  Compd. 1 from the virtual screen increased GPX4 activity, suppressed ferroptosis, reduced pro-inflammatory lipid mediator prodn., and inhibited NF-κB pathway activation.  Further chem. synthesis and structure-activity relationship studies revealed seven more activators.  The strongest compd., 1d4, increased GPX4 activity to 150% at 20 μM in the cell-free assay and 61 μM in cell exts.  Therefore, we demonstrated that GPX4 can be directly activated using chem. compds. to suppress ferroptosis and inflammation.  Meanwhile, the discovery of GPX4 activators verified the possibility of rational design of allosteric activators.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1yzEJLJGkorVg90H21EOLACvtfcHk0ljNwGVorpmI5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXot1Cjsrs%253D&md5=f4d07515f6cbf30298ca4a11974fb50b</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00315%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DConrad%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DAngeli%26aufirst%3DJ.%2BP.%2BF.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DNovel%2520Allosteric%2520Activators%2520for%2520Ferroptosis%2520Regulator%2520Glutathione%2520Peroxidase%25204%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D266%26epage%3D275%26doi%3D10.1021%2Facs.jmedchem.8b00315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacob, F.</span></span> <span> </span><span class="NLM_article-title">Telenomic Mechanisms in Cellular Metabolism, Growth, and Differentiation</span>. <i>Cold Spring Harbor Symp. Quant. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1961</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">389</span>– <span class="NLM_lpage">401</span>, <span class="refDoi"> DOI: 10.1101/SQB.1961.026.01.048</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1101%2FSQB.1961.026.01.048" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14475415" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADyaF38%252Fns1yquw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1961&pages=389-401&author=J.+Monodauthor=F.+Jacob&title=Telenomic+Mechanisms+in+Cellular+Metabolism%2C+Growth%2C+and+Differentiation&doi=10.1101%2FSQB.1961.026.01.048"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Teleonomic mechanisms in cellular metabolism, growth, and differentiation</span></div><div class="casAuthors">MONOD J; JACOB F</div><div class="citationInfo"><span class="NLM_cas:title">Cold Spring Harbor symposia on quantitative biology</span>
        (<span class="NLM_cas:date">1961</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">389-401</span>
        ISSN:<span class="NLM_cas:issn">0091-7451</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRBZpoAYqt75JPq5LRczm8ffW6udTcc2ebOR5LV5llDKrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADyaF38%252Fns1yquw%253D%253D&md5=a09cda000efcd8f0be0baeae7fbea61a</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1101%2FSQB.1961.026.01.048&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252FSQB.1961.026.01.048%26sid%3Dliteratum%253Aachs%26aulast%3DMonod%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DF.%26atitle%3DTelenomic%2520Mechanisms%2520in%2520Cellular%2520Metabolism%252C%2520Growth%252C%2520and%2520Differentiation%26jtitle%3DCold%2520Spring%2520Harbor%2520Symp.%2520Quant.%2520Biol.%26date%3D1961%26volume%3D26%26spage%3D389%26epage%3D401%26doi%3D10.1101%2FSQB.1961.026.01.048" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Monod, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wyman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span> <span> </span><span class="NLM_article-title">On the Nature of Allosteric Transitions: A Plausible Model</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1965</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">188</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(65)80285-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2FS0022-2836%2865%2980285-6" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14343300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaF2MXkt1Ghsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1965&pages=88-188&author=J.+Monodauthor=J.+Wymanauthor=J.-P.+Changeux&title=On+the+Nature+of+Allosteric+Transitions%3A+A+Plausible+Model&doi=10.1016%2FS0022-2836%2865%2980285-6"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">On the nature of allosteric transitions: a plausible model</span></div><div class="casAuthors">Monod, Jacques; Wyman, Jeffries; Changeux, Jean Pierre</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1965</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-118</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">A math. model is proposed for those regulatory proteins which are oligomeric in nature and the model applied with considerable success to published results on several enzymes or metabolically active proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9xh239rJ4sbVg90H21EOLACvtfcHk0ljNwGVorpmI5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF2MXkt1Ghsb4%253D&md5=daf94bee21838498c35c0cce65869a77</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2865%2980285-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252865%252980285-6%26sid%3Dliteratum%253Aachs%26aulast%3DMonod%26aufirst%3DJ.%26aulast%3DWyman%26aufirst%3DJ.%26aulast%3DChangeux%26aufirst%3DJ.-P.%26atitle%3DOn%2520the%2520Nature%2520of%2520Allosteric%2520Transitions%253A%2520A%2520Plausible%2520Model%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1965%26volume%3D12%26spage%3D88%26epage%3D188%26doi%3D10.1016%2FS0022-2836%2865%2980285-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Koshland, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nemethy, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filmer, D.</span></span> <span> </span><span class="NLM_article-title">Comparison of Experimental Binding Data and Theoretical Models in Proteins Containing Subunits</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">1966</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">365</span>– <span class="NLM_lpage">385</span>, <span class="refDoi"> DOI: 10.1021/bi00865a047</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi00865a047" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaF28XivVGnsw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1966&pages=365-385&author=D.+E.+Koshlandauthor=J.+G.+Nemethyauthor=D.+Filmer&title=Comparison+of+Experimental+Binding+Data+and+Theoretical+Models+in+Proteins+Containing+Subunits&doi=10.1021%2Fbi00865a047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of experimental binding data and theoretical models in proteins containing subunits</span></div><div class="casAuthors">Koshland, D. E., Jr.; Nemethy, G.; Filmer, D.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">1966</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">365-85</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    </div><div class="casAbstract">Models for subunit interactions are examd. by means of interpreting ligand saturation curves.  Equations are derived to indicate the effect of variables such as the strength of binding of the ligand, the geometrical arrangement of subunits, the strength of interaction between subunits, the energy of the conformation change, and the effect of nonidentical subunits.  Rapid methods, i.e., equations and nomograms, are developed to fit theoretical curves to exptl. data with a min. of parameters.  Applying these procedures to the binding of O by hemoglobin as an illustrative example, it is seen that a no. of simple models can represent the published data accurately.  In general, it appears that unique mechanisms cannot be established from ligand saturation curves by themselves, but the math. analysis of the curves indicates possible sources of addnl. information to make such distinctions possible.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBsA5hKxWFubVg90H21EOLACvtfcHk0ljNwGVorpmI5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaF28XivVGnsw%253D%253D&md5=e72f08b584282e1bfda351e41c94f28c</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fbi00865a047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi00865a047%26sid%3Dliteratum%253Aachs%26aulast%3DKoshland%26aufirst%3DD.%2BE.%26aulast%3DNemethy%26aufirst%3DJ.%2BG.%26aulast%3DFilmer%26aufirst%3DD.%26atitle%3DComparison%2520of%2520Experimental%2520Binding%2520Data%2520and%2520Theoretical%2520Models%2520in%2520Proteins%2520Containing%2520Subunits%26jtitle%3DBiochemistry%26date%3D1966%26volume%3D5%26spage%3D365%26epage%3D385%26doi%3D10.1021%2Fbi00865a047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dryden, D. T.</span></span> <span> </span><span class="NLM_article-title">Allostery without Conformational Change. A Plausible Model</span>. <i>Eur. Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">1984</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">103</span>– <span class="NLM_lpage">109</span>, <span class="refDoi"> DOI: 10.1007/BF00276625</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2FBF00276625" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=6544679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaL2MXktFOrtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=1984&pages=103-109&author=A.+Cooperauthor=D.+T.+Dryden&title=Allostery+without+Conformational+Change.+A+Plausible+Model&doi=10.1007%2FBF00276625"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery without conformational change.  A plausible model</span></div><div class="casAuthors">Cooper, A.; Dryden, D. T. F.</div><div class="citationInfo"><span class="NLM_cas:title">European Biophysics Journal</span>
        (<span class="NLM_cas:date">1984</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">103-9</span>CODEN:
                <span class="NLM_cas:coden">EBJOE8</span>;
        ISSN:<span class="NLM_cas:issn">0175-7571</span>.
    </div><div class="casAbstract">A general model is presented whereby ligand-induced changes in protein dynamics could produce allosteric communication between distinct binding sites, even in the absence of a macromol. conformational change.  Theor. anal., based on the statistical thermodn. of ligand binding, shows that cooperative interaction free energies amounting to several kilojoules per mol may be generated by this means.  The effect arises out of the possible changes in frequencies and amplitudes of macromol. thermal fluctuations in response to ligand attachment and can involve all forms of dynamic behavior, ranging from highly correlated, low-frequency normal mode vibrations to random local anharmonic motions of individual atoms or groups.  Dynamic allostery of this form is primarily an entropy effect, and approx. expressions are derived which might allow the magnitude of the interaction in real systems to be calcd. directly from exptl. observations such as changes in normal mode frequencies and mean-square at. displacements.  Long-range influence of kinetic processes at different sites might also be mediated by a similar mechanism.  Proteins and other biol. macromols. may have evolved to take functional advantage not only of mean conformational states but also of the inevitable thermal fluctuations about the mean.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUH5GNkFF82LVg90H21EOLACvtfcHk0liZPRMt1OB0ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL2MXktFOrtA%253D%253D&md5=5981f3a82ce85d17068a8f7e8b7b09bc</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1007%2FBF00276625&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF00276625%26sid%3Dliteratum%253Aachs%26aulast%3DCooper%26aufirst%3DA.%26aulast%3DDryden%26aufirst%3DD.%2BT.%26atitle%3DAllostery%2520without%2520Conformational%2520Change.%2520A%2520Plausible%2520Model%26jtitle%3DEur.%2520Biophys.%2520J.%26date%3D1984%26volume%3D11%26spage%3D103%26epage%3D109%26doi%3D10.1007%2FBF00276625" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Igumenova, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Characterization of the Fast Dynamics of Protein Amino Acid Side Chains Using NMR Relaxation in Solution</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">1672</span>– <span class="NLM_lpage">1699</span>, <span class="refDoi"> DOI: 10.1021/cr040422h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr040422h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28XjvFyjtLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2006&pages=1672-1699&author=T.+I.+Igumenovaauthor=K.+K.+Frederickauthor=A.+J.+Wand&title=Characterization+of+the+Fast+Dynamics+of+Protein+Amino+Acid+Side+Chains+Using+NMR+Relaxation+in+Solution&doi=10.1021%2Fcr040422h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Characterization of the fast dynamics of protein amino acid side chains using NMR relaxation in solution</span></div><div class="casAuthors">Igumenova, Tatyana I.; King Frederick, Kendra; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1672-1699</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review on the current state-of-the-art of soln. NMR-based characterization of the fast internal dynamics of protein side chains.  A survey of basic observations about side chain dynamics derived from NMR relaxation studies is presented along with several analyses meant to dispel commonly held but apparently inaccurate correlations between dynamics, structure, and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrwe-gODQ8P_LVg90H21EOLACvtfcHk0liZPRMt1OB0ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XjvFyjtLY%253D&md5=85b6c44b477f995e26c23ea2df972770</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fcr040422h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr040422h%26sid%3Dliteratum%253Aachs%26aulast%3DIgumenova%26aufirst%3DT.%2BI.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DCharacterization%2520of%2520the%2520Fast%2520Dynamics%2520of%2520Protein%2520Amino%2520Acid%2520Side%2520Chains%2520Using%2520NMR%2520Relaxation%2520in%2520Solution%26jtitle%3DChem.%2520Rev.%26date%3D2006%26volume%3D106%26spage%3D1672%26epage%3D1699%26doi%3D10.1021%2Fcr040422h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Dynamic Activation of an Allosteric Regulatory Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">368</span>– <span class="NLM_lpage">372</span>, <span class="refDoi"> DOI: 10.1038/nature08560</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature08560" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19924217" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVeqtrfM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=368-372&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Dynamic+Activation+of+an+Allosteric+Regulatory+Protein&doi=10.1038%2Fnature08560"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic activation of an allosteric regulatory protein</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7271</span>),
    <span class="NLM_cas:pages">368-372</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Allosteric regulation is used as a very efficient mechanism to control protein activity in most biol. processes, including signal transduction, metab., catalysis and gene regulation.  Allosteric proteins can exist in several conformational states with distinct binding or enzymic activity.  Effectors are considered to function in a purely structural manner by selectively stabilizing a specific conformational state, thereby regulating protein activity.  Here we show that allosteric proteins can be regulated predominantly by changes in their structural dynamics.  We have used NMR spectroscopy and isothermal titrn. calorimetry to characterize cAMP binding to the catabolite activator protein (CAP), a transcriptional activator that has been a prototype for understanding effector-mediated allosteric control of protein activity.  CAMP switches CAP from the 'off' state (inactive), which binds DNA weakly and non-specifically, to the 'on' state (active), which binds DNA strongly and specifically.  In contrast, cAMP binding to a single CAP mutant, CAP-S62F, fails to elicit the active conformation; yet cAMP binding to CAP-S62F strongly activates the protein for DNA binding.  NMR and thermodn. analyses show that despite the fact that CAP-S62F-cAMP2 adopts the inactive conformation, its strong binding to DNA is driven by a large conformational entropy originating in enhanced protein motions induced by DNA binding.  The results provide strong evidence that changes in protein motions may activate allosteric proteins that are otherwise structurally inactive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohyohMBfeojrVg90H21EOLACvtfcHk0liZPRMt1OB0ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVeqtrfM&md5=66ae42c3522f19ada2ae21fb808ee9db</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature08560&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08560%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DDynamic%2520Activation%2520of%2520an%2520Allosteric%2520Regulatory%2520Protein%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D368%26epage%3D372%26doi%3D10.1038%2Fnature08560" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Protein Activity Regulation by Conformational Entropy</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>488</i></span>,  <span class="NLM_fpage">236</span>– <span class="NLM_lpage">240</span>, <span class="refDoi"> DOI: 10.1038/nature11271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature11271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22801505" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbzF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=488&publication_year=2012&pages=236-240&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Protein+Activity+Regulation+by+Conformational+Entropy&doi=10.1038%2Fnature11271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Protein activity regulation by conformational entropy</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">488</span>
        (<span class="NLM_cas:issue">7410</span>),
    <span class="NLM_cas:pages">236-240</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">How the interplay between protein structure and internal dynamics regulates protein function is poorly understood.  Often, ligand binding, post-translational modifications and mutations modify protein activity in a manner that is not possible to rationalize solely on the basis of structural data.  It is likely that changes in the internal motions of proteins have a major role in regulating protein activity, but the nature of their contributions remains elusive, esp. in quant. terms.  Here we show that changes in conformational entropy can det. whether protein-ligand interactions will occur, even among protein complexes with identical binding interfaces.  We have used NMR spectroscopy to det. the changes in structure and internal dynamics that are elicited by the binding of DNA to several variants of the catabolite activator protein (CAP) that differentially populate the inactive and active DNA-binding domain states.  We found that the CAP variants have markedly different affinities for DNA, despite the CAP-DNA-binding interfaces being essentially identical in the various complexes.  Combined with thermodn. data, the results show that conformational entropy changes can inhibit the binding of CAP variants that are structurally poised for optimal DNA binding or can stimulate the binding activity of CAP variants that only transiently populate the DNA-binding-domain active state.  Collectively, the data show how changes in fast internal dynamics (conformational entropy) and slow internal dynamics (energetically excited conformational states) can regulate binding activity in a way that cannot be predicted on the basis of the protein's ground-state structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUawV2tVuAuLVg90H21EOLACvtfcHk0ljMz-LeIjGrkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtFWjtbzF&md5=5cc4c2d27af99e6efd2888372a751ca5</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnature11271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11271%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DProtein%2520Activity%2520Regulation%2520by%2520Conformational%2520Entropy%26jtitle%3DNature%26date%3D2012%26volume%3D488%26spage%3D236%26epage%3D240%26doi%3D10.1038%2Fnature11271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Petit, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapienza, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuentes, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. L.</span></span> <span> </span><span class="NLM_article-title">Hidden Dynamic Allostery in a PDZ Domain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>106</i></span>,  <span class="NLM_fpage">18249</span>– <span class="NLM_lpage">18254</span>, <span class="refDoi"> DOI: 10.1073/pnas.0904492106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0904492106" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19828436" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVWrsrjO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18249-18254&author=C.+M.+Petitauthor=J.+Zhangauthor=P.+J.+Sapienzaauthor=E.+J.+Fuentesauthor=A.+L.+Lee&title=Hidden+Dynamic+Allostery+in+a+PDZ+Domain&doi=10.1073%2Fpnas.0904492106"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Hidden dynamic allostery in a PDZ domain</span></div><div class="casAuthors">Petit, Chad M.; Zhang, Jun; Sapienza, Paul J.; Fuentes, Ernesto J.; Lee, Andrew L.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">106</span>
        (<span class="NLM_cas:issue">43</span>),
    <span class="NLM_cas:pages">18249-18254</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Structure-function relationships in proteins are predicated on the spatial proximity of noncovalently interacting groups of atoms.  Thus, structural elements located away from a protein's active site are typically presumed to serve a stabilizing or scaffolding role for the larger structure.  Here we report a functional role for a distal structural element in a PDZ domain, even though it is not required to maintain PDZ structure.  The third PDZ domain from PSD-95/SAP90 (PDZ3) has an unusual addnl. third alpha helix (α3) that packs in contiguous fashion against the globular domain.  Although α3 lies outside the active site and does not make direct contact with C-terminal peptide ligand, removal of α3 reduces ligand affinity by 21-fold.  Further investigation revealed that the difference in binding free energies between the full-length and truncated constructs is predominantly entropic in nature and that without α3, picosecond-nanosecond side-chain dynamics are enhanced throughout the domain, as detd. by 2H Me NMR relaxation.  Thus, the distal modulation of binding function appears to occur via a delocalized conformational entropy mechanism.  Without removal of α3 and characterization of side-chain dynamics, this dynamic allostery would have gone unnoticed.  Moreover, what appeared at first to be an artificial modification of PDZ3 has been corroborated by exptl. verified phosphorylation of α3, revealing a tangible biol. mechanism for this novel regulatory scheme.  This hidden dynamic allostery raises the possibility of as-yet unidentified or untapped allosteric regulation in this PDZ domain and is a very clear example of function arising from dynamics rather than from structure.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoXqoxKwIs4nbVg90H21EOLACvtfcHk0ljMz-LeIjGrkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVWrsrjO&md5=9ae15474e5320584dbe68dee381b2f5c</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0904492106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0904492106%26sid%3Dliteratum%253Aachs%26aulast%3DPetit%26aufirst%3DC.%2BM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DSapienza%26aufirst%3DP.%2BJ.%26aulast%3DFuentes%26aufirst%3DE.%2BJ.%26aulast%3DLee%26aufirst%3DA.%2BL.%26atitle%3DHidden%2520Dynamic%2520Allostery%2520in%2520a%2520PDZ%2520Domain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D18249%26epage%3D18254%26doi%3D10.1073%2Fpnas.0904492106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lundström, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulder, F. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akke, M.</span></span> <span> </span><span class="NLM_article-title">Correlated Dynamics of Consecutive Residues Reveal Transient and Cooperative Unfolding of Secondary Structure in Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>102</i></span>,  <span class="NLM_fpage">16984</span>– <span class="NLM_lpage">16989</span>, <span class="refDoi"> DOI: 10.1073/pnas.0504361102</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0504361102" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16278300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BD2Mnit12itA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2005&pages=16984-16989&author=P.+Lundstr%C3%B6mauthor=F.+A.+Mulderauthor=M.+Akke&title=Correlated+Dynamics+of+Consecutive+Residues+Reveal+Transient+and+Cooperative+Unfolding+of+Secondary+Structure+in+Proteins&doi=10.1073%2Fpnas.0504361102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Correlated dynamics of consecutive residues reveal transient and cooperative unfolding of secondary structure in proteins</span></div><div class="casAuthors">Lundstrom Patrik; Mulder Frans A A; Akke Mikael</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">16984-9</span>
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    </div><div class="casAbstract">Nuclear spin relaxation is a powerful method for studying molecular dynamics at atomic resolution.  Recent methods development in biomolecular NMR spectroscopy has enabled detailed investigations of molecular dynamics that are critical for biological function, with prominent examples addressing allostery, enzyme catalysis, and protein folding.  Dynamic processes with similar correlation times are often detected in multiple locations of the molecule, raising the question of whether the underlying motions are correlated (corresponding to concerted fluctuations involving many atoms distributed across extended regions of the molecule) or uncorrelated (corresponding to independent fluctuations involving few atoms in localized regions).  Here, we have used (13)C(alpha)(i - 1)/(13)C(alpha)(i) differential multiple-quantum spin relaxation to provide direct evidence for correlated dynamics of consecutive amino acid residues in the protein sequence.  By monitoring overlapping pairs of residues (i - 1 and i, i and i + 1, etc.), we identified correlated motions that extend through continuous segments of the sequence.  We detected significant correlated conformational transitions in the native state of the E140Q mutant of the calmodulin C-terminal domain.  Previous work has shown that this domain exchanges between two major conformational states that resemble the functionally relevant open and closed states of the WT protein, with a mean correlation time of approximately 20 micros.  The present results reveal that an entire alpha-helix undergoes partial unraveling in a transient and cooperative manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT6GqfmEoQr_mlSCSgBhwJFfW6udTcc2ea9IouYxgzF3rntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2Mnit12itA%253D%253D&md5=e268cc06a712f9721c3492b1699884ee</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0504361102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0504361102%26sid%3Dliteratum%253Aachs%26aulast%3DLundstr%25C3%25B6m%26aufirst%3DP.%26aulast%3DMulder%26aufirst%3DF.%2BA.%26aulast%3DAkke%26aufirst%3DM.%26atitle%3DCorrelated%2520Dynamics%2520of%2520Consecutive%2520Residues%2520Reveal%2520Transient%2520and%2520Cooperative%2520Unfolding%2520of%2520Secondary%2520Structure%2520in%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2005%26volume%3D102%26spage%3D16984%26epage%3D16989%26doi%3D10.1073%2Fpnas.0504361102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Conformational Entropy in Molecular Recognition by Proteins</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>448</i></span>,  <span class="NLM_fpage">325</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1038/nature05959</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature05959" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17637663" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=448&publication_year=2007&pages=325-329&author=K.+K.+Frederickauthor=M.+S.+Marlowauthor=K.+G.+Valentineauthor=A.+J.+Wand&title=Conformational+Entropy+in+Molecular+Recognition+by+Proteins&doi=10.1038%2Fnature05959"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Conformational entropy in molecular recognition by proteins</span></div><div class="casAuthors">Frederick, Kendra King; Marlow, Michael S.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">448</span>
        (<span class="NLM_cas:issue">7151</span>),
    <span class="NLM_cas:pages">325-329</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mol. recognition by proteins is fundamental to almost every biol. process, particularly the protein assocns. underlying cellular signal transduction.  Understanding the basis for protein-protein interactions requires the full characterization of the thermodn. of their assocn.  Historically, it has been virtually impossible to exptl. est. changes in protein conformational entropy, a potentially important component of the free energy of protein assocn.  However, NMR spectroscopy has emerged as a powerful tool for characterizing the dynamics of proteins.  Here, the authors employed changes in conformational dynamics as a proxy for corresponding changes in conformational entropy.  It was found that the change in internal dynamics of the protein, calmodulin, varied significantly on binding a variety of target domains.  Surprisingly, the apparent change in the corresponding conformational entropy was linearly related to the change in the overall binding entropy.  This indicated that changes in protein conformational entropy can contribute significantly to the free energy of protein-ligand assocn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohP3eEb0Q6srVg90H21EOLACvtfcHk0ljMz-LeIjGrkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnvVeqsbk%253D&md5=99cb195dbee8c4e67e348d353451dc1b</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Fnature05959&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature05959%26sid%3Dliteratum%253Aachs%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DConformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Proteins%26jtitle%3DNature%26date%3D2007%26volume%3D448%26spage%3D325%26epage%3D329%26doi%3D10.1038%2Fnature05959" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kinnear, S. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Redistribution and Loss of Side Chain Entropy upon Formation of a Calmodulin-Peptide Complex</span>. <i>Nat. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">72</span>– <span class="NLM_lpage">77</span>, <span class="refDoi"> DOI: 10.1038/71280</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2F71280" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10625431" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD3cXls1SnsQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2000&pages=72-77&author=A.+L.+Leeauthor=S.+A.+Kinnearauthor=A.+J.+Wand&title=Redistribution+and+Loss+of+Side+Chain+Entropy+upon+Formation+of+a+Calmodulin-Peptide+Complex&doi=10.1038%2F71280"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Redistribution and loss of side chain entropy upon formation of a calmodulin-peptide complex</span></div><div class="casAuthors">Lee, Andrew L.; Kinnear, Sandra A.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">72-77</span>CODEN:
                <span class="NLM_cas:coden">NSBIEW</span>;
        ISSN:<span class="NLM_cas:issn">1072-8368</span>.
    
            (<span class="NLM_cas:orgname">Nature America</span>)
        </div><div class="casAbstract">The response of the internal dynamics of calcium-satd. calmodulin to the formation of a complex with a peptide model of the calmodulin-binding domain of the smooth muscle myosin light chain kinase has been studied using NMR relaxation methods.  The backbone of calmodulin is found to be unaffected by the binding of the domain, whereas the dynamics of side chains are significantly perturbed.  The changes in dynamics are interpreted in terms of a heterogeneous partitioning between structure (enthalpy) and dynamics (entropy).  These data provide a microscopic view of the residual entropy of a protein in two functional states and suggest extensive enthalpy/entropy exchange during the formation of a protein-protein interface.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEGBIxp9SqOLVg90H21EOLACvtfcHk0li51eVbIuh9aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXls1SnsQ%253D%253D&md5=53ddeaa98a9c769b84ec531ef5400439</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1038%2F71280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F71280%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DA.%2BL.%26aulast%3DKinnear%26aufirst%3DS.%2BA.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DRedistribution%2520and%2520Loss%2520of%2520Side%2520Chain%2520Entropy%2520upon%2520Formation%2520of%2520a%2520Calmodulin-Peptide%2520Complex%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D2000%26volume%3D7%26spage%3D72%26epage%3D77%26doi%3D10.1038%2F71280" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kornev, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, S. S.</span></span> <span> </span><span class="NLM_article-title">Dynamics-Driven Allostery in Protein Kinases</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">628</span>– <span class="NLM_lpage">647</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2015.09.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2015.09.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26481499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1Orsb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2015&pages=628-647&author=A.+P.+Kornevauthor=S.+S.+Taylor&title=Dynamics-Driven+Allostery+in+Protein+Kinases&doi=10.1016%2Fj.tibs.2015.09.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamics-Driven Allostery in Protein Kinases</span></div><div class="casAuthors">Kornev, Alexandr P.; Taylor, Susan S.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">628-647</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases have very dynamic structures and their functionality strongly depends on their dynamic state.  Active kinases reveal a dynamic pattern with residues clustering into semirigid communities that move in μs-ms timescale.  Previously detected hydrophobic spines serve as connectors between communities.  Communities do not follow the traditional subdomain structure of the kinase core or its secondary structure elements.  Instead they are organized around main functional units.  Integration of the communities depends on the assembly of the hydrophobic spine and phosphorylation of the activation loop.  Single mutations can significantly disrupt the dynamic infrastructure and thereby interfere with long-distance allosteric signaling that propagates throughout the whole mol.  Dynamics is proposed to be the underlying mechanism for allosteric regulation in protein kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdvYRkTg5UTLVg90H21EOLACvtfcHk0li51eVbIuh9aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1Orsb3L&md5=3c4cefa885c773608ed279b000195917</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2015.09.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2015.09.002%26sid%3Dliteratum%253Aachs%26aulast%3DKornev%26aufirst%3DA.%2BP.%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26atitle%3DDynamics-Driven%2520Allostery%2520in%2520Protein%2520Kinases%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2015%26volume%3D40%26spage%3D628%26epage%3D647%26doi%3D10.1016%2Fj.tibs.2015.09.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Bringing Disorder and Dynamics in Protein Allostery into Focus</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">4278</span>– <span class="NLM_lpage">4280</span>, <span class="refDoi"> DOI: 10.1073/pnas.1703796114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1703796114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28420785" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXmsFGmu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=4278-4280&author=A.+J.+Wand&title=Bringing+Disorder+and+Dynamics+in+Protein+Allostery+into+Focus&doi=10.1073%2Fpnas.1703796114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Bringing disorder and dynamics in protein allostery into focus</span></div><div class="casAuthors">Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">4278-4280</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprheK2WctkR7Vg90H21EOLACvtfcHk0li51eVbIuh9aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXmsFGmu7k%253D&md5=b52cb644bc8607821c7fdbd50b6bfaf2</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1703796114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1703796114%26sid%3Dliteratum%253Aachs%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DBringing%2520Disorder%2520and%2520Dynamics%2520in%2520Protein%2520Allostery%2520into%2520Focus%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D4278%26epage%3D4280%26doi%3D10.1073%2Fpnas.1703796114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Marlow, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dogan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frederick, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valentine, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, J. A.</span></span> <span> </span><span class="NLM_article-title">The Role of Conformational Entropy in Molecular Recognition by Calmodulin</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">352</span>– <span class="NLM_lpage">358</span>, <span class="refDoi"> DOI: 10.1038/nchembio.347</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.347" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20383153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=352-358&author=M.+S.+Marlowauthor=J.+Doganauthor=K.+K.+Frederickauthor=K.+G.+Valentineauthor=J.+A.+Wand&title=The+Role+of+Conformational+Entropy+in+Molecular+Recognition+by+Calmodulin&doi=10.1038%2Fnchembio.347"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">The role of conformational entropy in molecular recognition by calmodulin</span></div><div class="casAuthors">Marlow, Michael S.; Dogan, Jakob; Frederick, Kendra K.; Valentine, Kathleen G.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">352-358</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The phys. basis for high-affinity interactions involving proteins is complex and potentially involves a range of energetic contributions.  Among these are changes in protein conformational entropy, which cannot yet be reliably computed from mol. structures.  We have recently used changes in conformational dynamics as a proxy for changes in conformational entropy of calmodulin upon assocn. with domains from regulated proteins.  The apparent change in conformational entropy was linearly related to the overall binding entropy.  This view warrants a more quant. foundation.  Here we calibrate an 'entropy meter' using an exptl. dynamical proxy based on NMR relaxation and show that changes in the conformational entropy of calmodulin are a significant component of the energetics of binding.  Furthermore, the distribution of motion at the interface between the target domain and calmodulin is surprisingly noncomplementary.  These observations promote modification of our understanding of the energetics of protein-ligand interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmK5VBOAHwT7Vg90H21EOLACvtfcHk0li51eVbIuh9aQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFakt78%253D&md5=e9ee34fffa91f612441a77a7385015ff</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.347&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.347%26sid%3Dliteratum%253Aachs%26aulast%3DMarlow%26aufirst%3DM.%2BS.%26aulast%3DDogan%26aufirst%3DJ.%26aulast%3DFrederick%26aufirst%3DK.%2BK.%26aulast%3DValentine%26aufirst%3DK.%2BG.%26aulast%3DWand%26aufirst%3DJ.%2BA.%26atitle%3DThe%2520Role%2520of%2520Conformational%2520Entropy%2520in%2520Molecular%2520Recognition%2520by%2520Calmodulin%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D352%26epage%3D358%26doi%3D10.1038%2Fnchembio.347" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kasinath, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharp, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wand, A. J.</span></span> <span> </span><span class="NLM_article-title">Microscopic Insights into the NMR Relaxation-Based Protein Conformational Entropy Meter</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">15092</span>– <span class="NLM_lpage">15100</span>, <span class="refDoi"> DOI: 10.1021/ja405200u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja405200u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsVSitLrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=15092-15100&author=V.+Kasinathauthor=K.+A.+Sharpauthor=A.+J.+Wand&title=Microscopic+Insights+into+the+NMR+Relaxation-Based+Protein+Conformational+Entropy+Meter&doi=10.1021%2Fja405200u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Microscopic Insights into the NMR Relaxation-Based Protein Conformational Entropy Meter</span></div><div class="casAuthors">Kasinath, Vignesh; Sharp, Kim A.; Wand, A. Joshua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">40</span>),
    <span class="NLM_cas:pages">15092-15100</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Conformational entropy is a potentially important thermodn. parameter contributing to protein function.  Quant. measures of conformational entropy are necessary for an understanding of its role but have been difficult to obtain.  An empirical method that uses changes in conformational dynamics as a proxy for changes in conformational entropy has recently been introduced.  Here the authors probe the microscopic origins of the link between conformational dynamics and conformational entropy using mol. dynamics simulations.  Simulation of seven proteins gave an excellent correlation with measures of side-chain motion derived from NMR relaxation.  The simulations show that the motion of methyl-bearing side chains are sufficiently coupled to that of other side chains to serve as excellent reporters of the overall side-chain conformational entropy.  These results tend to validate the use of exptl. accessible measures of Me motion-the NMR-derived generalized order parameters-as a proxy from which to derive changes in protein conformational entropy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrL2g-WdJGoFrVg90H21EOLACvtfcHk0ljcBwfjTv9DFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsVSitLrK&md5=a3d96078d1a414e75dda8202c3d12216</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1021%2Fja405200u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja405200u%26sid%3Dliteratum%253Aachs%26aulast%3DKasinath%26aufirst%3DV.%26aulast%3DSharp%26aufirst%3DK.%2BA.%26aulast%3DWand%26aufirst%3DA.%2BJ.%26atitle%3DMicroscopic%2520Insights%2520into%2520the%2520NMR%2520Relaxation-Based%2520Protein%2520Conformational%2520Entropy%2520Meter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D15092%26epage%3D15100%26doi%3D10.1021%2Fja405200u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wolfson, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels and Conformational Transitions via Hinge-Bending Motions</span>. <i>Cell Biochem. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">141</span>– <span class="NLM_lpage">164</span>, <span class="refDoi"> DOI: 10.1007/BF02738169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2FBF02738169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10593256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXnslKqtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1999&pages=141-164&author=S.+Kumarauthor=B.+Maauthor=C.+J.+Tsaiauthor=H.+Wolfsonauthor=R.+Nussinov&title=Folding+Funnels+and+Conformational+Transitions+via+Hinge-Bending+Motions&doi=10.1007%2FBF02738169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels and conformational transitions via hinge-bending motions</span></div><div class="casAuthors">Kumar, Sandeep; Ma, Buyong; Tsai, Chung-Jung; Wolfson, Haim; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Cell Biochemistry and Biophysics</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">141-164</span>CODEN:
                <span class="NLM_cas:coden">CBBIFV</span>;
        ISSN:<span class="NLM_cas:issn">1085-9195</span>.
    
            (<span class="NLM_cas:orgname">Humana Press Inc.</span>)
        </div><div class="casAbstract">A review, with 72 refs.  In this article we focus on presenting a broad range of examples illustrating low-energy transitions via hinge-bending motions.  The examples are divided according to the type of hinge-bending involved; namely, motions involving fragments of the protein chains, hinge-bending motions involving protein domains, and hinge-bending motions between the covalently unconnected subunits.  We further make a distinction between allosterically and nonallosterically regulated proteins.  These transitions are discussed within the general framework of folding and binding funnels.  We propose that the conformers manifesting such swiveling motions are not the outcome of "induced fit" binding mechanism; instead, mols. exist in an ensemble of conformations that are in equil. in soln.  These ensembles, which populate the bottoms of the funnels, a priori contain both the "open" and the "closed" conformational isomers.  Furthermore, we argue that there are no fundamental differences among the phys. principles behind the folding and binding funnels.  Hence, there is no basic difference between funnels depicting ensembles of conformers of single mols. with fragment, or domain motions, as compared to subunits in multimeric quaternary structures, also showing such conformational transitions.  The difference relates only to the size and complexity of the system.  The larger the system, the more complex its corresponding fused funnel(s).  In particular, funnels assocd. with allosterically regulated proteins are expected to be more complicated, because allostery is frequently involved with movements between subunits, and consequently is often obsd. in multichain and multimol. complexes.  This review centers on the crit. role played by flexibility and conformational fluctuations in enzyme activity.  Internal motions that extend over different time scales and with different amplitudes are known to be essential for the catalytic cycle.  The conformational change obsd. in enzyme-substrate complexes as compared to the unbound enzyme state, and in particular the hinge-bending motions obsd. in enzymes with two domains, have a substantial effect on the enzymic catalytic activity.  The examples we review span the lipolytic enzymes that are particularly interesting, owing to their activation at the water-oil interface; an allosterically controlled dehydrogenase (lactate dehydrogenase); a DNA methyltransferase, with a covalently-bound intermediate; large-scale flexible loop motions in a glycolytic enzyme (TIM); domain motion in PGK, an enzyme which is essential in most cells, both for ATP generation in aerobes and for fermn. in anaerobes; adenylate kinase, showing large conformational changes, owing to their need to shield their catalytic centers from water; a calcium-binding protein (calmodulin), involved in a wide range of cellular calcium-dependent signaling; diphtheria toxin, whose large domain motion has been shown to yield "domain swapping;" the hexameric glutamate dehydrogenase, which has been studied both in a thermophile and in a mesophile; an allosteric enzyme, showing subunit motion between the R and the T states (aspartate transcarbamoylase), and the historically well-studied lac repressor.  Nonallosteric subunit transitions are also addressed, with some examples (aspartate receptor and BamHI endonuclease).  Hence, using this enzyme-catalysis-centered discussion, we address energy funnel landscapes of large-scale conformational transitions, rather than the faster, quasi-harmonic, thermal fluctuations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq56G0PLHUlALVg90H21EOLACvtfcHk0ljcBwfjTv9DFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXnslKqtrw%253D&md5=8457d4d795f2226fd8115ed83e3ccceb</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1007%2FBF02738169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252FBF02738169%26sid%3Dliteratum%253Aachs%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DWolfson%26aufirst%3DH.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%2520and%2520Conformational%2520Transitions%2520via%2520Hinge-Bending%2520Motions%26jtitle%3DCell%2520Biochem.%2520Biophys.%26date%3D1999%26volume%3D31%26spage%3D141%26epage%3D164%26doi%3D10.1007%2FBF02738169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels and Binding Mechanisms</span>. <i>Protein Eng., Des. Sel.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">713</span>– <span class="NLM_lpage">720</span>, <span class="refDoi"> DOI: 10.1093/protein/12.9.713</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fprotein%2F12.9.713" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVagsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1999&pages=713-720&author=B.+Maauthor=S.+Kumarauthor=C.+J.+Tsaiauthor=R.+Nussinov&title=Folding+Funnels+and+Binding+Mechanisms&doi=10.1093%2Fprotein%2F12.9.713"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels and binding mechanisms</span></div><div class="casAuthors">Ma, Buyong; Kumar, Sandeep; Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Protein Engineering</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">713-720</span>CODEN:
                <span class="NLM_cas:coden">PRENE9</span>;
        ISSN:<span class="NLM_cas:issn">0269-2139</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The long-held views on lock-and-key vs. induced fit in binding arose from the notion that a protein exists in a single, most stable conformation, dictated by its sequence.  However, in soln., proteins exist in a range of conformations, which may be described by statistical mech. laws, and their populations follow statistical distributions.  Upon binding, the equil. will shift in favor of the bound conformation, from the ensemble of conformations around the bottom of the folding funnel.  Hence, here we extend the implications and the usefulness of the folding funnel concept to explain fundamental binding mechanisms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq_0UdAhInii7Vg90H21EOLACvtfcHk0ljcBwfjTv9DFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVagsbY%253D&md5=8c229c85bad0fa20e90c459f90b4a760</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F12.9.713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F12.9.713%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%2520and%2520Binding%2520Mechanisms%26jtitle%3DProtein%2520Eng.%252C%2520Des.%2520Sel.%26date%3D1999%26volume%3D12%26spage%3D713%26epage%3D720%26doi%3D10.1093%2Fprotein%2F12.9.713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Folding Funnels, Binding Funnels, and Protein Function</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1190</span>, <span class="refDoi"> DOI: 10.1110/ps.8.6.1181</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1110%2Fps.8.6.1181" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10386868" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXjvF2hurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1999&pages=1181-1190&author=C.+J.+Tsaiauthor=S.+Kumarauthor=B.+Maauthor=R.+Nussinov&title=Folding+Funnels%2C+Binding+Funnels%2C+and+Protein+Function&doi=10.1110%2Fps.8.6.1181"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Folding funnels, binding funnels, and protein function</span></div><div class="casAuthors">Tsai, Chung-Jung; Kumar, Sandeep; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1181-1190</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cambridge University Press</span>)
        </div><div class="casAbstract">A review with 66 refs.  Folding funnels have been the focus of considerable attention during the last few years.  These have mostly been discussed in the general context of the theory of protein folding.  Here we extend the utility of the concept of folding funnels, relating them to biol. mechanisms and function.  In particular, here we describe the shape of the funnels in light of protein synthesis and folding; flexibility, conformational diversity, and binding mechanisms; and the assocd. binding funnels, illustrating the multiple routes and the range of complexed conformers.  Specifically, the walls of the folding funnels, their crevices, and bumps are related to the complexity of protein folding, and hence to sequential vs. non-sequential folding.  Whereas the former is more frequently obsd. in eukaryotic proteins, where the rate of protein synthesis is slower, the latter is more frequent in prokaryotes, with faster translation rates.  The bottoms of the funnels reflect the extent of the flexibility of the proteins.  Rugged floors imply a range of conformational isomers, which may be close on the energy landscape.  Rather than undergoing an induced fit binding mechanism, the conformational ensembles around the rugged bottoms argue that the conformers, which are most complementary to the ligand, will bind to it with the equil. shifting in their favor.  Furthermore, depending on the extent of the ruggedness, or of the smoothness with only a few min., we may infer nonspecific, broad range vs. specific binding.  In particular, folding and binding are similar processes, with similar underlying principles.  Hence, the shape of the folding funnel of the monomer enables making reasonable guesses regarding the shape of the corresponding binding funnel.  Proteins having a broad range of binding, such as proteolytic enzymes or relatively nonspecific endonucleases, may be expected to have not only rugged floors in their folding funnels, but their binding funnels will also behave similarly, with a range of complexed conformations.  Hence, knowledge of the shape of the folding funnels is biol. very useful.  The converse also holds: if kinetic and thermodn. data are available, hints regarding the role of the protein and its binding selectivity may be obtained.  Thus, the utility of the concept of the funnel carries over to the origin of the protein and to its function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Lm9WfsGisrVg90H21EOLACvtfcHk0lhcQKI04tUYSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXjvF2hurg%253D&md5=9f33ddad55f16afb66f7544d68bf9b48</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1110%2Fps.8.6.1181&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.8.6.1181%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DFolding%2520Funnels%252C%2520Binding%2520Funnels%252C%2520and%2520Protein%2520Function%26jtitle%3DProtein%2520Sci.%26date%3D1999%26volume%3D8%26spage%3D1181%26epage%3D1190%26doi%3D10.1110%2Fps.8.6.1181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lockless, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Evolutionarily Conserved Pathways of Energetic Connectivity in Protein Families</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1126/science.286.5438.295</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.286.5438.295" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10514373" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADyaK1MXmvVCgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=1999&pages=295-299&author=S.+W.+Locklessauthor=R.+Ranganathan&title=Evolutionarily+Conserved+Pathways+of+Energetic+Connectivity+in+Protein+Families&doi=10.1126%2Fscience.286.5438.295"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Evolutionarily conserved pathways of energetic connectivity in protein families</span></div><div class="casAuthors">Lockless, Steve W.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">5438</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">For mapping energetic interactions in proteins, a technique was developed that uses evolutionary data for a protein family to measure statistical interactions between amino acid positions.  For the PDZ domain family, this anal. predicted a set of energetically coupled positions for a binding site residue that includes unexpected long-range interactions.  Mutational studies confirm these predictions, demonstrating that the statistical energy function is a good indicator of thermodn. coupling in proteins.  Sets of interacting residues form connected pathways through the protein fold that may be the basis for efficient energy conduction within proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-fqjxe7280bVg90H21EOLACvtfcHk0lhcQKI04tUYSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXmvVCgurs%253D&md5=94bd674ba82f21086db0cbbad98f9679</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1126%2Fscience.286.5438.295&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.286.5438.295%26sid%3Dliteratum%253Aachs%26aulast%3DLockless%26aufirst%3DS.%2BW.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DEvolutionarily%2520Conserved%2520Pathways%2520of%2520Energetic%2520Connectivity%2520in%2520Protein%2520Families%26jtitle%3DScience%26date%3D1999%26volume%3D286%26spage%3D295%26epage%3D299%26doi%3D10.1126%2Fscience.286.5438.295" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">del
Sol, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">The Origin of Allosteric Functional Modulation: Multiple Pre-Existing Pathways</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1042</span>– <span class="NLM_lpage">1050</span>, <span class="refDoi"> DOI: 10.1016/j.str.2009.06.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2009.06.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19679084" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXps1Clt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=1042-1050&author=A.+del%0ASolauthor=C.+J.+Tsaiauthor=B.+Maauthor=R.+Nussinov&title=The+Origin+of+Allosteric+Functional+Modulation%3A+Multiple+Pre-Existing+Pathways&doi=10.1016%2Fj.str.2009.06.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">The Origin of Allosteric Functional Modulation: Multiple Pre-existing Pathways</span></div><div class="casAuthors">del Sol, Antonio; Tsai, Chung-Jung; Ma, Buyong; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1042-1050</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Although allostery draws increasing attention, not much is known about allosteric mechanisms.  Here we argue that in all proteins, allosteric signals transmit through multiple, pre-existing pathways; which pathways dominate depend on protein topologies, specific binding events, covalent modifications, and cellular (environmental) conditions.  Further, perturbation events at any site on the protein surface (or in the interior) will not create new pathways but only shift the pre-existing ensemble of pathways.  Drugs binding at different sites or mutational events in disease shift the ensemble toward the same conformations; however, the relative populations of the different states will change.  Consequently the obsd. functional, conformational, and dynamic effects will be different.  This is the origin of allosteric functional modulation in dynamic proteins: allostery does not necessarily need to invoke conformational rearrangements to control protein activity and pre-existing pathways are always defaulted to during allostery regardless of the stimulant and perturbation site in the protein.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbjZObe-bsobVg90H21EOLACvtfcHk0lhcQKI04tUYSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXps1Clt7s%253D&md5=a5c26c07e4a4fab4e85d0b1fae65a4ac</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2009.06.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2009.06.008%26sid%3Dliteratum%253Aachs%26aulast%3Ddel%2BSol%26aufirst%3DA.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DMa%26aufirst%3DB.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DThe%2520Origin%2520of%2520Allosteric%2520Functional%2520Modulation%253A%2520Multiple%2520Pre-Existing%2520Pathways%26jtitle%3DStructure%26date%3D2009%26volume%3D17%26spage%3D1042%26epage%3D1050%26doi%3D10.1016%2Fj.str.2009.06.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korcsmáros, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, H. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">London, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Structure and Dynamics of Molecular Networks: A Novel Paradigm of Drug Discovery: A Comprehensive Review</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">333</span>– <span class="NLM_lpage">408</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2013.01.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2013.01.016" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23384594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2013&pages=333-408&author=P.+Csermelyauthor=T.+Korcsm%C3%A1rosauthor=H.+J.+M.+Kissauthor=G.+Londonauthor=R.+Nussinov&title=Structure+and+Dynamics+of+Molecular+Networks%3A+A+Novel+Paradigm+of+Drug+Discovery%3A+A+Comprehensive+Review&doi=10.1016%2Fj.pharmthera.2013.01.016"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and dynamics of molecular networks: A novel paradigm of drug discovery</span></div><div class="casAuthors">Csermely, Peter; Korcsmaros, Tamas; Kiss, Huba J. M.; London, Gabor; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">333-408</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Despite considerable progress in genome- and proteome-based high-throughput screening methods and in rational drug design, the increase in approved drugs in the past decade did not match the increase of drug development costs.  Network description and anal. not only give a systems-level understanding of drug action and disease complexity, but can also help to improve the efficiency of drug design.  We give a comprehensive assessment of the anal. tools of network topol. and dynamics.  The state-of-the-art use of chem. similarity, protein structure, protein-protein interaction, signaling, genetic interaction and metabolic networks in the discovery of drug targets is summarized.  We propose that network targeting follows two basic strategies.  The "central hit strategy" selectively targets central nodes/edges of the flexible networks of infectious agents or cancer cells to kill them.  The "network influence strategy" works against other diseases, where an efficient reconfiguration of rigid networks needs to be achieved by targeting the neighbors of central nodes/edges.  It is shown how network techniques can help in the identification of single-target, edgetic, multi-target and allo-network drug target candidates.  We review the recent boom in network methods helping hit identification, lead selection optimizing drug efficacy, as well as minimizing side-effects and drug toxicity.  Successful network-based drug development strategies are shown through the examples of infections, cancer, metabolic diseases, neurodegenerative diseases and aging.  Summarizing >1200 refs. we suggest an optimized protocol of network-aided drug development, and provide a list of systems-level hallmarks of drug quality.  Finally, we highlight network-related drug development trends helping to achieve these hallmarks by a cohesive, global approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFqNviuDk7kbVg90H21EOLACvtfcHk0lhcQKI04tUYSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXls1CgtLk%253D&md5=d70c48a1572815c1b4905cc6e8cdfaa6</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2013.01.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2013.01.016%26sid%3Dliteratum%253Aachs%26aulast%3DCsermely%26aufirst%3DP.%26aulast%3DKorcsm%25C3%25A1ros%26aufirst%3DT.%26aulast%3DKiss%26aufirst%3DH.%2BJ.%2BM.%26aulast%3DLondon%26aufirst%3DG.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DStructure%2520and%2520Dynamics%2520of%2520Molecular%2520Networks%253A%2520A%2520Novel%2520Paradigm%2520of%2520Drug%2520Discovery%253A%2520A%2520Comprehensive%2520Review%26jtitle%3DPharmacol.%2520Ther.%26date%3D2013%26volume%3D138%26spage%3D333%26epage%3D408%26doi%3D10.1016%2Fj.pharmthera.2013.01.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The Mechanism of ATP-Dependent Allosteric Protection of Akt Kinase Phosphorylation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1734</span>, <span class="refDoi"> DOI: 10.1016/j.str.2015.06.027</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2015.06.027" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26256536" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlSju7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2015&pages=1725-1734&author=S.+Luauthor=R.+Dengauthor=H.+Jiangauthor=H.+Songauthor=S.+Liauthor=Q.+Shenauthor=W.+Huangauthor=R.+Nussinovauthor=J.+Yuauthor=J.+Zhang&title=The+Mechanism+of+ATP-Dependent+Allosteric+Protection+of+Akt+Kinase+Phosphorylation&doi=10.1016%2Fj.str.2015.06.027"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">The Mechanism of ATP-Dependent Allosteric Protection of Akt Kinase Phosphorylation</span></div><div class="casAuthors">Lu, Shaoyong; Deng, Rong; Jiang, Haiming; Song, Huili; Li, Shuai; Shen, Qiancheng; Huang, Wenkang; Nussinov, Ruth; Yu, Jianxiu; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1725-1734</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Kinases use ATP to phosphorylate substrates; recent findings underscore the addnl. regulatory roles of ATP.  Here, we propose a mechanism for allosteric regulation of Akt1 kinase phosphorylation by ATP.  Our 4.7-μs mol. dynamics simulations of Akt1 and its mutants in the ATP/ADP bound/unbound states revealed that ATP occupancy of the ATP-binding site stabilizes the closed conformation, allosterically protecting pT308 by restraining phosphatase access and key interconnected residues on the ATP→pT308 allosteric pathway.  Following ATP→ADP hydrolysis, pT308 is exposed and readily dephosphorylated.  Site-directed mutagenesis validated these predictions and indicated that the mutations do not impair PDK1 and PP2A phosphatase recruitment.  We further probed the function of residues around pT308 at the at. level, and predicted and exptl. confirmed that Akt1H194R/R273H double mutant rescues pathol.-related Akt1R273H.  Anal. of classical Akt homologs suggests that this mechanism can provide a general model of allosteric kinase regulation by ATP; as such, it offers a potential avenue for allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP6VKa16r6C7Vg90H21EOLACvtfcHk0ljoNjELp5pgtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlSju7nM&md5=147d9333779db1bc1eb6b05f36524174</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2015.06.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2015.06.027%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DDeng%26aufirst%3DR.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520Mechanism%2520of%2520ATP-Dependent%2520Allosteric%2520Protection%2520of%2520Akt%2520Kinase%2520Phosphorylation%26jtitle%3DStructure%26date%3D2015%26volume%3D23%26spage%3D1725%26epage%3D1734%26doi%3D10.1016%2Fj.str.2015.06.027" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Changeux, J.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edelstein, S. J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Mechanisms of Signal Transduction</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>308</i></span>,  <span class="NLM_fpage">1424</span>– <span class="NLM_lpage">1428</span>, <span class="refDoi"> DOI: 10.1126/science.1108595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1108595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15933191" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1eqtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=308&publication_year=2005&pages=1424-1428&author=J.-P.+Changeuxauthor=S.+J.+Edelstein&title=Allosteric+Mechanisms+of+Signal+Transduction&doi=10.1126%2Fscience.1108595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Mechanisms of Signal Transduction</span></div><div class="casAuthors">Changeux, Jean-Pierre; Edelstein, Stuart J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">308</span>
        (<span class="NLM_cas:issue">5727</span>),
    <span class="NLM_cas:pages">1424-1428</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Forty years ago, a simple model of allosteric mechanisms (indirect interactions between distinct sites), used initially to explain feedback-inhibited enzymes, was presented by Monod, Wyman, and Changeux.  We review the MWC theory and its applications for the understanding of signal transduction in biol., and also identify remaining issues that deserve theor. and exptl. substantiation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8FOn7QVpjAbVg90H21EOLACvtfcHk0ljoNjELp5pgtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1eqtrs%253D&md5=ec3750bf12645a3e6c4d2b13f221fc03</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1126%2Fscience.1108595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1108595%26sid%3Dliteratum%253Aachs%26aulast%3DChangeux%26aufirst%3DJ.-P.%26aulast%3DEdelstein%26aufirst%3DS.%2BJ.%26atitle%3DAllosteric%2520Mechanisms%2520of%2520Signal%2520Transduction%26jtitle%3DScience%26date%3D2005%26volume%3D308%26spage%3D1424%26epage%3D1428%26doi%3D10.1126%2Fscience.1108595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span> <span> </span><span class="NLM_article-title">An Ensemble View of Allostery</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>327</i></span>,  <span class="NLM_fpage">653</span>– <span class="NLM_lpage">654</span>, <span class="refDoi"> DOI: 10.1126/science.1186121</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1186121" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20133562" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXitFamtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=327&publication_year=2010&pages=653-654&author=V.+J.+Hilser&title=An+Ensemble+View+of+Allostery&doi=10.1126%2Fscience.1186121"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">An ensemble view of allostery</span></div><div class="casAuthors">Hilser, Vincent J.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">327</span>
        (<span class="NLM_cas:issue">5966</span>),
    <span class="NLM_cas:pages">653-654</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ_hUdt32OorVg90H21EOLACvtfcHk0ljoNjELp5pgtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXitFamtbg%253D&md5=c52626431046cd4e42f85b55404de86b</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1126%2Fscience.1186121&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1186121%26sid%3Dliteratum%253Aachs%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26atitle%3DAn%2520Ensemble%2520View%2520of%2520Allostery%26jtitle%3DScience%26date%3D2010%26volume%3D327%26spage%3D653%26epage%3D654%26doi%3D10.1126%2Fscience.1186121" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Motlagh, H. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrabl, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span> <span> </span><span class="NLM_article-title">The Ensemble Nature of Allostery</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>508</i></span>,  <span class="NLM_fpage">331</span>– <span class="NLM_lpage">339</span>, <span class="refDoi"> DOI: 10.1038/nature13001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature13001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24740064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmtlWnurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=508&publication_year=2014&pages=331-339&author=H.+N.+Motlaghauthor=J.+O.+Wrablauthor=J.+Liauthor=V.+J.+Hilser&title=The+Ensemble+Nature+of+Allostery&doi=10.1038%2Fnature13001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">The ensemble nature of allostery</span></div><div class="casAuthors">Motlagh, Hesam N.; Wrabl, James O.; Li, Jing; Hilser, Vincent J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">508</span>
        (<span class="NLM_cas:issue">7496</span>),
    <span class="NLM_cas:pages">331-339</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Allostery is the process by which biol. macromols. (mostly proteins) transmit the effect of binding at one site to another, often distal, functional site, allowing for regulation of activity.  Recent exptl. observations demonstrating that allostery can be facilitated by dynamic and intrinsically disordered proteins have resulted in a new paradigm for understanding allosteric mechanisms, which focuses on the conformational ensemble and the statistical nature of the interactions responsible for the transmission of information.  Anal. of allosteric ensembles reveals a rich spectrum of regulatory strategies, as well as a framework to unify the description of allosteric mechanisms from different systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpAlOmcHvsANLVg90H21EOLACvtfcHk0ljbs8tm2WZg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmtlWnurc%253D&md5=cf832c576e409cab75d1031a98f418c3</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1038%2Fnature13001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13001%26sid%3Dliteratum%253Aachs%26aulast%3DMotlagh%26aufirst%3DH.%2BN.%26aulast%3DWrabl%26aufirst%3DJ.%2BO.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26atitle%3DThe%2520Ensemble%2520Nature%2520of%2520Allostery%26jtitle%3DNature%26date%3D2014%26volume%3D508%26spage%3D331%26epage%3D339%26doi%3D10.1038%2Fnature13001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hilser, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wrabl, J. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motlagh, H. N.</span></span> <span> </span><span class="NLM_article-title">Structural and Energetic Basis of Allostery</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">585</span>– <span class="NLM_lpage">609</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-050511-102319</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-050511-102319" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22577828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yhsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=585-609&author=V.+J.+Hilserauthor=J.+O.+Wrablauthor=H.+N.+Motlagh&title=Structural+and+Energetic+Basis+of+Allostery&doi=10.1146%2Fannurev-biophys-050511-102319"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Structural and energetic basis of allostery</span></div><div class="casAuthors">Hilser, Vincent J.; Wrabl, James O.; Motlagh, Hesam N.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">585-609</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Allostery is a biol. phenomenon of fundamental importance in regulation and signaling, and efforts to understand this process have led to the development of numerous models.  In spite of individual successes in understanding the structural determinants of allostery in well-documented systems, much less success has been achieved in identifying a set of quant. and transferable ground rules that provide an understanding of how allostery works.  Are there organizing principles that allow us to relate structurally different proteins, or are the determinants of allostery unique to each system.  Using an ensemble-based model, we show that allosteric phenomena can be formulated in terms of conformational free energies of the cooperative elements in a protein and the coupling interactions between them.  Interestingly, the resulting allosteric ground rules provide a framework to reconcile observations that challenge purely structural models of site-to-site coupling, including (a) allostery in the absence of pathways of structural distortions, (b) allostery in the absence of any structural change, and (c) the ability of allosteric ligands to act as agonists under some circumstances and antagonists under others.  The ensemble view of allostery that emerges provides insights into the energetic prerequisites of site-to-site coupling and thus into how allostery works.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxNCm1otnOSrVg90H21EOLACvtfcHk0ljbs8tm2WZg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yhsbo%253D&md5=b8e81b54704ea6d3eb776b8cac81985f</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-050511-102319&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-050511-102319%26sid%3Dliteratum%253Aachs%26aulast%3DHilser%26aufirst%3DV.%2BJ.%26aulast%3DWrabl%26aufirst%3DJ.%2BO.%26aulast%3DMotlagh%26aufirst%3DH.%2BN.%26atitle%3DStructural%2520and%2520Energetic%2520Basis%2520of%2520Allostery%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D585%26epage%3D609%26doi%3D10.1146%2Fannurev-biophys-050511-102319" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Allostery: An Overview of Its History, Concepts, Methods, and Applications</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1004966</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004966</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004966" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27253437" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWntbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1004966&author=J.+Liuauthor=R.+Nussinov&title=Allostery%3A+An+Overview+of+Its+History%2C+Concepts%2C+Methods%2C+and+Applications&doi=10.1371%2Fjournal.pcbi.1004966"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery: an overview of its history, concepts, methods, and applications</span></div><div class="casAuthors">Liu, Jin; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1004966/1-e1004966/5</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The concept of allostery has evolved in the past century.  In this Editorial, we briefly overview the history of allostery, from the pre-allostery nomenclature era starting with the Bohr effect (1904) to the birth of allostery by Monod and Jacob (1961).  We describe the evolution of the allostery concept, from a conformational change in a two-state model (1965, 1966) to dynamic allostery in the ensemble model (1999); from multi-subunit (1965) proteins to all proteins (2004).  We highlight the current available methods to study allostery and their applications in studies of conformational mechanisms, disease, and allosteric drug discovery.  We outline the challenges and future directions that we foresee.  Altogether, this Editorial narrates the history of this fundamental concept in the life sciences, its significance, methodologies to detect and predict it, and its application in a broad range of living systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUNj0xAGD3qrVg90H21EOLACvtfcHk0ljbs8tm2WZg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWntbbJ&md5=1a753ef03a696bd2591eb1d7206f50d7</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004966%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DAllostery%253A%2520An%2520Overview%2520of%2520Its%2520History%252C%2520Concepts%252C%2520Methods%252C%2520and%2520Applications%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3De1004966%26doi%3D10.1371%2Fjournal.pcbi.1004966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">A Unified View of “How Allostery Works”</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003394</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003394</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003394" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24516370" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=C.-J.+Tsaiauthor=R.+Nussinov&title=A+Unified+View+of+%E2%80%9CHow+Allostery+Works%E2%80%9D&doi=10.1371%2Fjournal.pcbi.1003394"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">A unified view of "how allostery works"</span></div><div class="casAuthors">Tsai, Chung-Jung; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e1003394/1-e1003394/12, 12 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">The question of how allostery works was posed almost 50 years ago.  Since then it has been the focus of much effort.  This is for two reasons: first, the intellectual curiosity of basic science and the desire to understand fundamental phenomena, and second, its vast practical importance.  Allostery is at play in all processes in the living cell, and increasingly in drug discovery.  Many models have been successfully formulated, and are able to describe allostery even in the absence of a detailed structural mechanism.  However, conceptual schemes designed to qual. explain allosteric mechanisms usually lack a quant. math. model, and are unable to link its thermodn. and structural foundations.  This hampers insight into oncogenic mutations in cancer progression and biased agonists' actions.  Here, we describe how allostery works from three different standpoints: thermodn., free energy landscape of population shift, and structure; all with exactly the same allosteric descriptors.  This results in a unified view which not only clarifies the elusive allosteric mechanism but also provides structural grasp of agonist-mediated signaling pathways, and guides allosteric drug discovery.  Of note, the unified view reasons that allosteric coupling (or communication) does not det. the allosteric efficacy; however, a communication channel is what makes potential binding sites allosteric.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEgA5AKySrFbVg90H21EOLACvtfcHk0ljbs8tm2WZg9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXkvVCmtLo%253D&md5=3912bf89813eeed254b3e3e4e1937f99</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003394&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003394%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DC.-J.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DA%2520Unified%2520View%2520of%2520%25E2%2580%259CHow%2520Allostery%2520Works%25E2%2580%259D%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cui, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karplus, M.</span></span> <span> </span><span class="NLM_article-title">Allostery and Cooperativity Revisited</span>. <i>Protein Sci.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">1295</span>– <span class="NLM_lpage">1307</span>, <span class="refDoi"> DOI: 10.1110/ps.03259908</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1110%2Fps.03259908" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18560010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXpsVyjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2008&pages=1295-1307&author=Q.+Cuiauthor=M.+Karplus&title=Allostery+and+Cooperativity+Revisited&doi=10.1110%2Fps.03259908"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery and cooperativity revisited</span></div><div class="casAuthors">Cui, Qiang; Karplus, Martin</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1295-1307</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">0961-8368</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">A review.  Although phenomenol. models that account for cooperativity in allosteric systems date back to the early and mid-60's [e.g., the D. E. Koshland, G. Nemethy, and D. Filmer, 1966 (KNF) and J. Momod, J. Wyman, and J. P. Changeux, 1965 (MWC) models], there is resurgent interest in the topic due to the recent exptl. and computational studies that attempted to reveal, at an atomistic level, how allostery actually works.  Here, using systems for which atomistic simulations have been carried out in their groups as examples, the authors describe the current understanding of allostery and how the mechanisms go beyond the classical MWC/Pauling-KNF descriptions, and point out that the "new view" of allostery, emphasizing "population shifts," is, in fact, an "old view.".  The presentation offers not only an up-to-date description of allostery from a theor./computational perspective, but also helps to resolve several outstanding issues concerning allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot34ax4lJjPLVg90H21EOLACvtfcHk0lgSltoEQZNE1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXpsVyjurY%253D&md5=40d789fc47dcb9f01e797aa724861f77</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1110%2Fps.03259908&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1110%252Fps.03259908%26sid%3Dliteratum%253Aachs%26aulast%3DCui%26aufirst%3DQ.%26aulast%3DKarplus%26aufirst%3DM.%26atitle%3DAllostery%2520and%2520Cooperativity%2520Revisited%26jtitle%3DProtein%2520Sci.%26date%3D2008%26volume%3D17%26spage%3D1295%26epage%3D1307%26doi%3D10.1110%2Fps.03259908" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bagal, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andrews, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bechle, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bian, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bilsland, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blakemore, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Braganza, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bungay, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corbett, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cronin, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cui, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dias, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flanagan, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greasley, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grimley, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">James, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kitching, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kraus, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAlpine, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagata, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ninkovic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Omoto, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scales, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skerratt, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tran-Dubé, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Waldron, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmus, J. S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent, Selective, and Peripherally Restricted Pan-Trk Kinase Inhibitors for the Treatment of Pain</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6779</span>– <span class="NLM_lpage">6800</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00633</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00633" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1SnurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6779-6800&author=S.+K.+Bagalauthor=M.+Andrewsauthor=B.+M.+Bechleauthor=J.+Bianauthor=J.+Bilslandauthor=D.+C.+Blakemoreauthor=J.+F.+Braganzaauthor=P.+J.+Bungayauthor=M.+S.+Corbettauthor=C.+N.+Croninauthor=J.+J.+Cuiauthor=R.+Diasauthor=N.+J.+Flanaganauthor=S.+E.+Greasleyauthor=R.+Grimleyauthor=K.+Jamesauthor=E.+Johnsonauthor=L.+Kitchingauthor=M.+L.+Krausauthor=I.+McAlpineauthor=A.+Nagataauthor=S.+Ninkovicauthor=K.+Omotoauthor=S.+Scalesauthor=S.+E.+Skerrattauthor=J.+Sunauthor=M.+Tran-Dub%C3%A9author=G.+J.+Waldronauthor=F.+Wangauthor=J.+S.+Warmus&title=Discovery+of+Potent%2C+Selective%2C+and+Peripherally+Restricted+Pan-Trk+Kinase+Inhibitors+for+the+Treatment+of+Pain&doi=10.1021%2Facs.jmedchem.8b00633"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent, Selective, and Peripherally Restricted Pan-Trk Kinase Inhibitors for the Treatment of Pain</span></div><div class="casAuthors">Bagal, Sharan K.; Andrews, Mark; Bechle, Bruce M.; Bian, Jianwei; Bilsland, James; Blakemore, David C.; Braganza, John F.; Bungay, Peter J.; Corbett, Matthew S.; Cronin, Ciaran N.; Cui, Jingrong Jean; Dias, Rebecca; Flanagan, Neil J.; Greasley, Samantha E.; Grimley, Rachel; James, Kim; Johnson, Eric; Kitching, Linda; Kraus, Michelle L.; McAlpine, Indrawan; Nagata, Asako; Ninkovic, Sacha; Omoto, Kiyoyuki; Scales, Stephanie; Skerratt, Sarah E.; Sun, Jianmin; Tran-Dube, Michelle; Waldron, Gareth J.; Wang, Fen; Warmus, Joseph S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6779-6800</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Hormones of the neurotrophin family: nerve growth factor (NGF), brain derived neurotrophic factor (BDNF), neurotrophin 3 (NT3) and neurotrophin 4 (NT4) are known to activate the family of Tropomyosin receptor kinases (TrkA, TrkB, TrkC).  Moreover, inhibition of the TrkA kinase pathway in pain has been clin. validated by the NGF antibody tanezumab leading to significant interest in the development of small mol. inhibitors of TrkA.  Furthermore, Trk inhibitors having an acceptable safety profile will require minimal brain availability.  Herein we discuss the discovery of two potent, selective, peripherally restricted, efficacious and well-tolerated series of pan-Trk inhibitors which successfully delivered three candidate quality compds. I, II and III.  All three compds. are predicted to possess low metabolic clearance in human that does not proceed via aldehyde oxidase-catalyzed reactions, thus addressing the potential clearance prediction liability assocd. with our current pan-Trk development candidate PF-06273340.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhKF1Su9NvabVg90H21EOLACvtfcHk0lgSltoEQZNE1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1SnurfN&md5=65b55ef6cdd82c44680c6c49310df7b2</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00633&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00633%26sid%3Dliteratum%253Aachs%26aulast%3DBagal%26aufirst%3DS.%2BK.%26aulast%3DAndrews%26aufirst%3DM.%26aulast%3DBechle%26aufirst%3DB.%2BM.%26aulast%3DBian%26aufirst%3DJ.%26aulast%3DBilsland%26aufirst%3DJ.%26aulast%3DBlakemore%26aufirst%3DD.%2BC.%26aulast%3DBraganza%26aufirst%3DJ.%2BF.%26aulast%3DBungay%26aufirst%3DP.%2BJ.%26aulast%3DCorbett%26aufirst%3DM.%2BS.%26aulast%3DCronin%26aufirst%3DC.%2BN.%26aulast%3DCui%26aufirst%3DJ.%2BJ.%26aulast%3DDias%26aufirst%3DR.%26aulast%3DFlanagan%26aufirst%3DN.%2BJ.%26aulast%3DGreasley%26aufirst%3DS.%2BE.%26aulast%3DGrimley%26aufirst%3DR.%26aulast%3DJames%26aufirst%3DK.%26aulast%3DJohnson%26aufirst%3DE.%26aulast%3DKitching%26aufirst%3DL.%26aulast%3DKraus%26aufirst%3DM.%2BL.%26aulast%3DMcAlpine%26aufirst%3DI.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DNinkovic%26aufirst%3DS.%26aulast%3DOmoto%26aufirst%3DK.%26aulast%3DScales%26aufirst%3DS.%26aulast%3DSkerratt%26aufirst%3DS.%2BE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DTran-Dub%25C3%25A9%26aufirst%3DM.%26aulast%3DWaldron%26aufirst%3DG.%2BJ.%26aulast%3DWang%26aufirst%3DF.%26aulast%3DWarmus%26aufirst%3DJ.%2BS.%26atitle%3DDiscovery%2520of%2520Potent%252C%2520Selective%252C%2520and%2520Peripherally%2520Restricted%2520Pan-Trk%2520Kinase%2520Inhibitors%2520for%2520the%2520Treatment%2520of%2520Pain%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6779%26epage%3D6800%26doi%3D10.1021%2Facs.jmedchem.8b00633" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaniskan, H. Ü.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eram, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Szewczyk, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schmidt, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">He, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zang, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dobrovetsky, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smil, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin-Jones, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schapira, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atadja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barsyte-Lovejoy, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arrowsmith, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1204</span>– <span class="NLM_lpage">1217</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01674</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01674" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1204-1217&author=H.+%C3%9C.+Kaniskanauthor=M.+S.+Eramauthor=K.+Zhaoauthor=M.+M.+Szewczykauthor=X.+Yangauthor=K.+Schmidtauthor=X.+Luoauthor=S.+Xiaoauthor=M.+Daiauthor=F.+Heauthor=I.+Zangauthor=Y.+Linauthor=F.+Liauthor=E.+Dobrovetskyauthor=D.+Smilauthor=S.+J.+Minauthor=J.+Lin-Jonesauthor=M.+Schapiraauthor=P.+Atadjaauthor=E.+Liauthor=D.+Barsyte-Lovejoyauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=F.+Liuauthor=Z.+Yuauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+Potent+and+Selective+Allosteric+Inhibitors+of+Protein+Arginine+Methyltransferase+3+%28PRMT3%29&doi=10.1021%2Facs.jmedchem.7b01674"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Potent and Selective Allosteric Inhibitors of Protein Arginine Methyltransferase 3 (PRMT3)</span></div><div class="casAuthors">Kaniskan, H. Umit; Eram, Mohammad S.; Zhao, Kehao; Szewczyk, Magdalena M.; Yang, Xiaobao; Schmidt, Keith; Luo, Xiao; Xiao, Sean; Dai, Miao; He, Feng; Zang, Irene; Lin, Ying; Li, Fengling; Dobrovetsky, Elena; Smil, David; Min, Sun-Joon; Lin-Jones, Jennifer; Schapira, Matthieu; Atadja, Peter; Li, En; Barsyte-Lovejoy, Dalia; Arrowsmith, Cheryl H.; Brown, Peter J.; Liu, Feng; Yu, Zhengtian; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1204-1217</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PRMT3 catalyzes the asym. dimethylation of arginine residues of various proteins.  It is crucial for maturation of ribosomes, and has been implicated in several diseases.  We recently disclosed a highly potent, selective and cell-active allosteric inhibitor of PRMT3, compd. 4.  Here, we report comprehensive structure-activity relationship studies that target the allosteric binding site of PRMT3.  We conducted design, synthesis and evaluation of novel compds. in biochem., selectivity and cellular assays that culminated in the discovery of 4 and other highly potent (IC50 values: ∼10-36 nM), selective and cell-active allosteric inhibitors of PRMT3 (compds. 29, 30, 36 and 37).  In addn., we generated compds. that are very close analogs of these potent inhibitors, but displayed drastically reduced potency as neg. controls (compds. 49-51).  These inhibitors and neg. controls are valuable chem. tools for the biomedical community to further investigate biol. functions and disease assocns. of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGodgUnzNZOB2rVg90H21EOLACvtfcHk0lhvWXe_8JplVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvF2gsbzO&md5=a784663c26234c3a99386ea4fe3f8abc</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01674%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259C.%26aulast%3DEram%26aufirst%3DM.%2BS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DSzewczyk%26aufirst%3DM.%2BM.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DSchmidt%26aufirst%3DK.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DXiao%26aufirst%3DS.%26aulast%3DDai%26aufirst%3DM.%26aulast%3DHe%26aufirst%3DF.%26aulast%3DZang%26aufirst%3DI.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DMin%26aufirst%3DS.%2BJ.%26aulast%3DLin-Jones%26aufirst%3DJ.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Potent%2520and%2520Selective%2520Allosteric%2520Inhibitors%2520of%2520Protein%2520Arginine%2520Methyltransferase%25203%2520%2528PRMT3%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1204%26epage%3D1217%26doi%3D10.1021%2Facs.jmedchem.7b01674" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ortiz
Zacarías, N. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Veldhoven, J. P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Portner, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van Spronsen, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ullo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veenhuizen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Velden, W. J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweemer, A. J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreekel, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oenema, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenselink, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heitman, L. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ijzerman, A. P.</span></span> <span> </span><span class="NLM_article-title">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">9146</span>– <span class="NLM_lpage">9161</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00605</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00605" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=9146-9161&author=N.+V.+Ortiz%0AZacar%C3%ADasauthor=J.+P.+D.+van+Veldhovenauthor=L.+Portnerauthor=E.+van+Spronsenauthor=S.+Ulloauthor=M.+Veenhuizenauthor=W.+J.+C.+van+der+Veldenauthor=A.+J.+M.+Zweemerauthor=R.+M.+Kreekelauthor=K.+Oenemaauthor=E.+B.+Lenselinkauthor=L.+H.+Heitmanauthor=A.+P.+Ijzerman&title=Pyrrolone+Derivatives+as+Intracellular+Allosteric+Modulators+for+Chemokine+Receptors%3A+Selective+and+Dual-Targeting+Inhibitors+of+CC+Chemokine+Receptors+1+and+2&doi=10.1021%2Facs.jmedchem.8b00605"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Pyrrolone Derivatives as Intracellular Allosteric Modulators for Chemokine Receptors: Selective and Dual-Targeting Inhibitors of CC Chemokine Receptors 1 and 2</span></div><div class="casAuthors">Ortiz Zacarias, Natalia V.; van Veldhoven, Jacobus P. D.; Portner, Laura; van Spronsen, Eric; Ullo, Salviana; Veenhuizen, Margo; van der Velden, Wijnand J. C.; Zweemer, Annelien J. M.; Kreekel, Roy M.; Oenema, Kenny; Lenselink, Eelke B.; Heitman, Laura H.; IJzerman, Adriaan P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">9146-9161</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The recent crystal structures of CC chemokine receptors 2 and 9 (CCR2 and CCR9) have provided structural evidence for an allosteric, intracellular binding site.  The high conservation of residues involved in this site suggests its presence in most chemokine receptors, including the close homolog CCR1.  By using [3H]CCR2-RA-[R], a high-affinity, CCR2 intracellular ligand, we report an intracellular binding site in CCR1, where this radioligand also binds with high affinity.  In addn., we report the synthesis and biol. characterization of a series of pyrrolone derivs. for CCR1 and CCR2, which allowed us to identify several high-affinity intracellular ligands, including selective and potential multitarget antagonists.  Evaluation of selected compds. in a functional [35S]GTPγS assay revealed that they act as inverse agonists in CCR1, providing a new manner of pharmacol. modulation.  Thus, this intracellular binding site enables the design of selective and multitarget inhibitors as a novel therapeutic approach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr3HeUOWen2U7Vg90H21EOLACvtfcHk0lhvWXe_8JplVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSis7rM&md5=bf05c560b983659de579074242d299d1</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00605&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00605%26sid%3Dliteratum%253Aachs%26aulast%3DOrtiz%2BZacar%25C3%25ADas%26aufirst%3DN.%2BV.%26aulast%3Dvan%2BVeldhoven%26aufirst%3DJ.%2BP.%2BD.%26aulast%3DPortner%26aufirst%3DL.%26aulast%3Dvan%2BSpronsen%26aufirst%3DE.%26aulast%3DUllo%26aufirst%3DS.%26aulast%3DVeenhuizen%26aufirst%3DM.%26aulast%3Dvan%2Bder%2BVelden%26aufirst%3DW.%2BJ.%2BC.%26aulast%3DZweemer%26aufirst%3DA.%2BJ.%2BM.%26aulast%3DKreekel%26aufirst%3DR.%2BM.%26aulast%3DOenema%26aufirst%3DK.%26aulast%3DLenselink%26aufirst%3DE.%2BB.%26aulast%3DHeitman%26aufirst%3DL.%2BH.%26aulast%3DIjzerman%26aufirst%3DA.%2BP.%26atitle%3DPyrrolone%2520Derivatives%2520as%2520Intracellular%2520Allosteric%2520Modulators%2520for%2520Chemokine%2520Receptors%253A%2520Selective%2520and%2520Dual-Targeting%2520Inhibitors%2520of%2520CC%2520Chemokine%2520Receptors%25201%2520and%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D9146%26epage%3D9161%26doi%3D10.1021%2Facs.jmedchem.8b00605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive Analysis of Kinase Inhibitor Selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>– <span class="NLM_lpage">1051</span>, <span class="refDoi"> DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+Analysis+of+Kinase+Inhibitor+Selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lhvWXe_8JplVw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520Analysis%2520of%2520Kinase%2520Inhibitor%2520Selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clausen, M. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nielsen, T. E.</span></span> <span> </span><span class="NLM_article-title">Allosteric Small-Molecule Kinase Inhibitors</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">68</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2015.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2015.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26478442" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs12qsb3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2015&pages=59-68&author=P.+Wuauthor=M.+H.+Clausenauthor=T.+E.+Nielsen&title=Allosteric+Small-Molecule+Kinase+Inhibitors&doi=10.1016%2Fj.pharmthera.2015.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric small-molecule kinase inhibitors</span></div><div class="casAuthors">Wu, Peng; Clausen, Mads H.; Nielsen, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-68</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Small-mol. kinase inhibitors are invaluable targeted therapeutics for the treatment of various human diseases, esp. cancers.  While the majority of approved and developed preclin. small-mol. inhibitors are characterized as type I or type II inhibitors that target the ATP-binding pocket of kinases, the remarkable sequential and structural similarity among ATP pockets renders the selective inhibition of kinases a daunting challenge.  Therefore, targeting allosteric pockets of kinases outside the highly conversed ATP pocket has been proposed as a promising alternative to overcome current barriers of kinase inhibitors, including poor selectivity and emergence of drug resistance.  In spite of the small no. of identified allosteric inhibitors in comparison with that of inhibitors targeting the ATP pocket, encouraging results, such as the FDA-approval of the first small-mol. allosteric inhibitor trametinib in 2013, the progress of more than 10 other allosteric inhibitors in clin. trials, and the emergence of a pipeline of highly selective and potent preclin. mols., have been reported in the past decade.  In this article, we present the current knowledge on allosteric inhibition in terms of conception, classification, potential advantages, and summarized debatable topics in the field.  Recent progress and allosteric inhibitors that were identified in the past three years are highlighted in this paper.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrK12iPyaK2ObVg90H21EOLACvtfcHk0liDMQJ1hxI9Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs12qsb3N&md5=26e4658d325b8f016be29df27bf69ec2</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2015.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2015.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DClausen%26aufirst%3DM.%2BH.%26aulast%3DNielsen%26aufirst%3DT.%2BE.%26atitle%3DAllosteric%2520Small-Molecule%2520Kinase%2520Inhibitors%26jtitle%3DPharmacol.%2520Ther.%26date%3D2015%26volume%3D156%26spage%3D59%26epage%3D68%26doi%3D10.1016%2Fj.pharmthera.2015.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lebois, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsley, C. W.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulation of Kinases and GPCRs: Design Principles and Structural Diversity</span>. <i>Curr. Opin. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">280</span>, <span class="refDoi"> DOI: 10.1016/j.cbpa.2008.02.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cbpa.2008.02.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18342020" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXnt1ersr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2008&pages=269-280&author=J.+A.+Lewisauthor=E.+P.+Leboisauthor=C.+W.+Lindsley&title=Allosteric+Modulation+of+Kinases+and+GPCRs%3A+Design+Principles+and+Structural+Diversity&doi=10.1016%2Fj.cbpa.2008.02.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric modulation of kinases and GPCRs: design principles and structural diversity</span></div><div class="casAuthors">Lewis, Jana A.; Lebois, Evan P.; Lindsley, Craig W.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">269-280</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Allosteric binding sites, as opposed to traditional orthosteric binding sites, offer unparalleled opportunities for drug discovery by providing high levels of selectivity, mimicking physiol. conditions, affording fewer side effects because of desensitization/downregulation, and engendering ligands with chemotypes divergent from orthosteric ligands.  For kinases, allosteric mechanisms described to date include alteration of protein kinase conformation blocking productive ATP binding which appear ATP competitive' or blocking kinase activation by conformational changes that are ATP non-competitive'.  For GPCRs, allosteric mechanisms impart multiple modes of target modulation (pos. allosteric modulation (PAM), neg. allosteric modulation (NAM), neutral cooperativity, partial antagonism (PA), allosteric agonism and allosteric antagonism).  Here, we review recent developments in the design principles and structural diversity of allosteric ligands for kinases and GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoiz5X-sQXXv7Vg90H21EOLACvtfcHk0liDMQJ1hxI9Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXnt1ersr4%253D&md5=79533b9634a920154e08e4bad0f6275e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2008.02.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2008.02.014%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DJ.%2BA.%26aulast%3DLebois%26aufirst%3DE.%2BP.%26aulast%3DLindsley%26aufirst%3DC.%2BW.%26atitle%3DAllosteric%2520Modulation%2520of%2520Kinases%2520and%2520GPCRs%253A%2520Design%2520Principles%2520and%2520Structural%2520Diversity%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2008%26volume%3D12%26spage%3D269%26epage%3D280%26doi%3D10.1016%2Fj.cbpa.2008.02.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Comess, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abad-Zapatero, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goedken, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gum, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Argiriadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groebe, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clampit, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haasch, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, H. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Coen, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cloutier, T. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quinn, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xin, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fry, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stoll, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banach, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcotte, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burns, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Calderwood, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hajduk, P. J.</span></span> <span> </span><span class="NLM_article-title">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">234</span>– <span class="NLM_lpage">244</span>, <span class="refDoi"> DOI: 10.1021/cb1002619</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb1002619" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=234-244&author=K.+M.+Comessauthor=C.+Sunauthor=C.+Abad-Zapateroauthor=E.+R.+Goedkenauthor=R.+J.+Gumauthor=D.+W.+Borhaniauthor=M.+Argiriadiauthor=D.+R.+Groebeauthor=Y.+Jiaauthor=J.+E.+Clampitauthor=D.+L.+Haaschauthor=H.+T.+Smithauthor=S.+Wangauthor=D.+Songauthor=M.+L.+Coenauthor=T.+E.+Cloutierauthor=H.+Tangauthor=X.+Chengauthor=C.+Quinnauthor=B.+Liuauthor=Z.+Xinauthor=G.+Liuauthor=E.+H.+Fryauthor=V.+Stollauthor=T.+I.+Ngauthor=D.+Banachauthor=D.+Marcotteauthor=D.+J.+Burnsauthor=D.+J.+Calderwoodauthor=P.+J.+Hajduk&title=Discovery+and+Characterization+of+Non-ATP+Site+Inhibitors+of+the+Mitogen+Activated+Protein+%28MAP%29+Kinases&doi=10.1021%2Fcb1002619"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and Characterization of Non-ATP Site Inhibitors of the Mitogen Activated Protein (MAP) Kinases</span></div><div class="casAuthors">Comess, Kenneth M.; Sun, Chaohong; Abad-Zapatero, Cele; Goedken, Eric R.; Gum, Rebecca J.; Borhani, David W.; Argiriadi, Maria; Groebe, Duncan R.; Jia, Yong; Clampit, Jill E.; Haasch, Deanna L.; Smith, Harriet T.; Wang, Sanyi; Song, Danying; Coen, Michael L.; Cloutier, Timothy E.; Tang, Hua; Cheng, Xueheng; Quinn, Christopher; Liu, Bo; Xin, Zhili; Liu, Gang; Fry, Elizabeth H.; Stoll, Vincent; Ng, Teresa I.; Banach, David; Marcotte, Doug; Burns, David J.; Calderwood, David J.; Hajduk, Philip J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibition of protein kinases has validated therapeutic utility for cancer, with at least seven kinase inhibitor drugs on the market.  Protein kinase inhibition also has significant potential for a variety of other diseases, including diabetes, pain, cognition, and chronic inflammatory and immunol. diseases.  However, as the vast majority of current approaches to kinase inhibition target the highly conserved ATP-binding site, the use of kinase inhibitors in treating nononcol. diseases may require great selectivity for the target kinase.  As protein kinases are signal transducers that are involved in binding to a variety of other proteins, targeting alternative, less conserved sites on the protein may provide an avenue for greater selectivity.  Here we report an affinity-based, high-throughput screening technique that allows nonbiased interrogation of small mol. libraries for binding to all exposed sites on a protein surface.  This approach was used to screen both the c-Jun N-terminal protein kinase Jnk-1 (involved in insulin signaling) and p38α (involved in the formation of TNFα and other cytokines).  In addn. to canonical ATP-site ligands, compds. were identified that bind to novel allosteric sites.  The nature, biol. relevance, and mode of binding of these ligands were extensively characterized using two-dimensional 1H/13C NMR spectroscopy, protein X-ray crystallog., surface plasmon resonance, and direct enzymic activity and activation cascade assays.  Jnk-1 and p38α both belong to the MAP kinase family, and the allosteric ligands for both targets bind similarly on a ledge of the protein surface exposed by the MAP insertion present in the CMGC family of protein kinases and distant from the active site.  Medicinal chem. studies resulted in an improved Jnk-1 ligand able to increase adiponectin secretion in human adipocytes and increase insulin-induced protein kinase PKB phosphorylation in human hepatocytes, in similar fashion to Jnk-1 siRNA and to rosiglitazone treatment.  Together, the data suggest that these new ligand series bind to a novel, allosteric, and physiol. relevant site and therefore represent a unique approach to identify kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyEwNo6QURxLVg90H21EOLACvtfcHk0liDMQJ1hxI9Fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVSltA%253D%253D&md5=857035713020ac66110b7ef6114232b9</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fcb1002619&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb1002619%26sid%3Dliteratum%253Aachs%26aulast%3DComess%26aufirst%3DK.%2BM.%26aulast%3DSun%26aufirst%3DC.%26aulast%3DAbad-Zapatero%26aufirst%3DC.%26aulast%3DGoedken%26aufirst%3DE.%2BR.%26aulast%3DGum%26aufirst%3DR.%2BJ.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DArgiriadi%26aufirst%3DM.%26aulast%3DGroebe%26aufirst%3DD.%2BR.%26aulast%3DJia%26aufirst%3DY.%26aulast%3DClampit%26aufirst%3DJ.%2BE.%26aulast%3DHaasch%26aufirst%3DD.%2BL.%26aulast%3DSmith%26aufirst%3DH.%2BT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DD.%26aulast%3DCoen%26aufirst%3DM.%2BL.%26aulast%3DCloutier%26aufirst%3DT.%2BE.%26aulast%3DTang%26aufirst%3DH.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DQuinn%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DXin%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DFry%26aufirst%3DE.%2BH.%26aulast%3DStoll%26aufirst%3DV.%26aulast%3DNg%26aufirst%3DT.%2BI.%26aulast%3DBanach%26aufirst%3DD.%26aulast%3DMarcotte%26aufirst%3DD.%26aulast%3DBurns%26aufirst%3DD.%2BJ.%26aulast%3DCalderwood%26aufirst%3DD.%2BJ.%26aulast%3DHajduk%26aufirst%3DP.%2BJ.%26atitle%3DDiscovery%2520and%2520Characterization%2520of%2520Non-ATP%2520Site%2520Inhibitors%2520of%2520the%2520Mitogen%2520Activated%2520Protein%2520%2528MAP%2529%2520Kinases%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D234%26epage%3D244%26doi%3D10.1021%2Fcb1002619" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherbenou, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hantschel, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaupe, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willis, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bumm, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turaga, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lange, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dao, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Press, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Druker, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Superti-Furga, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deininger, M. W.</span></span> <span> </span><span class="NLM_article-title">BCR-ABL SH3-SH2 Domain Mutations in Chronic Myeloid Leukemia Patients on Imatinib</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">3278</span>– <span class="NLM_lpage">3285</span>, <span class="refDoi"> DOI: 10.1182/blood-2008-10-183665</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1182%2Fblood-2008-10-183665" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20519627" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVeisLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2010&pages=3278-3285&author=D.+W.+Sherbenouauthor=O.+Hantschelauthor=I.+Kaupeauthor=S.+Willisauthor=T.+Bummauthor=L.+P.+Turagaauthor=T.+Langeauthor=K.+H.+Daoauthor=R.+D.+Pressauthor=B.+J.+Drukerauthor=G.+Superti-Furgaauthor=M.+W.+Deininger&title=BCR-ABL+SH3-SH2+Domain+Mutations+in+Chronic+Myeloid+Leukemia+Patients+on+Imatinib&doi=10.1182%2Fblood-2008-10-183665"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">BCR-ABL SH3-SH2 domain mutations in chronic myeloid leukemia patients on imatinib</span></div><div class="casAuthors">Sherbenou, Daniel W.; Hantschel, Oliver; Kaupe, Ines; Willis, Stephanie; Bumm, Thomas; Turaga, Lalita P.; Lange, Thoralf; Dao, Kim-Hien; Press, Richard D.; Druker, Brian J.; Superti-Furga, Giulio; Deininger, Michael W.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3278-3285</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-4971</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Point mutations in the kinase domain of BCR-ABL are the most common mechanism of drug resistance in chronic myeloid leukemia (CML) patients treated with ABL kinase inhibitors, including imatinib.  It has also been shown in vitro that mutations outside the kinase domain in the neighboring linker, SH2, SH3, and Cap domains can confer imatinib resistance.  In the context of ABL, these domains have an autoinhibitory effect on kinase activity, and mutations in this region can activate the enzyme.  To det. the frequency and relevance to resistance of regulatory domain mutations in CML patients on imatinib, we screened for such mutations in a cohort of consecutive CML patients with various levels of response.  Regulatory domain mutations were detected in 7 of 98 patients, whereas kinase domain mutations were detected in 29.  One mutation (T212R) conferred in vitro tyrosine kinase inhibitor resistance and was assocd. with relapse, whereas most other mutations did not affect drug sensitivity.  Mechanistic studies showed that T212R increased the activity of ABL and BCR-ABL and that T212R-induced resistance may be partially the result of stabilization of an active kinase conformation.  Regulatory domain mutations are uncommon but may explain resistance in some patients without mutations in the kinase domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWYplM3K_TmLVg90H21EOLACvtfcHk0lid7zMV7nbQsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVeisLjI&md5=a4cdbf97c4af950d257b38d8ee276fac</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1182%2Fblood-2008-10-183665&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2008-10-183665%26sid%3Dliteratum%253Aachs%26aulast%3DSherbenou%26aufirst%3DD.%2BW.%26aulast%3DHantschel%26aufirst%3DO.%26aulast%3DKaupe%26aufirst%3DI.%26aulast%3DWillis%26aufirst%3DS.%26aulast%3DBumm%26aufirst%3DT.%26aulast%3DTuraga%26aufirst%3DL.%2BP.%26aulast%3DLange%26aufirst%3DT.%26aulast%3DDao%26aufirst%3DK.%2BH.%26aulast%3DPress%26aufirst%3DR.%2BD.%26aulast%3DDruker%26aufirst%3DB.%2BJ.%26aulast%3DSuperti-Furga%26aufirst%3DG.%26aulast%3DDeininger%26aufirst%3DM.%2BW.%26atitle%3DBCR-ABL%2520SH3-SH2%2520Domain%2520Mutations%2520in%2520Chronic%2520Myeloid%2520Leukemia%2520Patients%2520on%2520Imatinib%26jtitle%3DBlood%26date%3D2010%26volume%3D116%26spage%3D3278%26epage%3D3285%26doi%3D10.1182%2Fblood-2008-10-183665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cotesta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drueckes, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabriel, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Groell, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotzfeld, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Henry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jones, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manley, P. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pellé, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salem, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zoller, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span> <span> </span><span class="NLM_article-title">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">8120</span>– <span class="NLM_lpage">8135</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b01040</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b01040" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=8120-8135&author=J.+Schoepferauthor=W.+Jahnkeauthor=G.+Berelliniauthor=S.+Buonamiciauthor=S.+Cotestaauthor=S.+W.+Cowan-Jacobauthor=S.+Doddauthor=P.+Drueckesauthor=D.+Fabbroauthor=T.+Gabrielauthor=J.+M.+Groellauthor=R.+M.+Grotzfeldauthor=A.+Q.+Hassanauthor=C.+Henryauthor=V.+Iyerauthor=D.+Jonesauthor=F.+Lombardoauthor=A.+Looauthor=P.+W.+Manleyauthor=X.+Pell%C3%A9author=G.+Rummelauthor=B.+Salemauthor=M.+Warmuthauthor=A.+A.+Wylieauthor=T.+Zollerauthor=A.+L.+Marzinzikauthor=P.+Furet&title=Discovery+of+Asciminib+%28ABL001%29%2C+an+Allosteric+Inhibitor+of+the+Tyrosine+Kinase+Activity+of+BCR-ABL1&doi=10.1021%2Facs.jmedchem.8b01040"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Asciminib (ABL001), an Allosteric Inhibitor of the Tyrosine Kinase Activity of BCR-ABL1</span></div><div class="casAuthors">Schoepfer, Joseph; Jahnke, Wolfgang; Berellini, Giuliano; Buonamici, Silvia; Cotesta, Simona; Cowan-Jacob, Sandra W.; Dodd, Stephanie; Drueckes, Peter; Fabbro, Doriano; Gabriel, Tobias; Groell, Jean-Marc; Grotzfeld, Robert M.; Hassan, A. Quamrul; Henry, Chrystele; Iyer, Varsha; Jones, Darryl; Lombardo, Franco; Loo, Alice; Manley, Paul W.; Pelle, Xavier; Rummel, Gabriele; Salem, Bahaa; Warmuth, Markus; Wylie, Andrew A.; Zoller, Thomas; Marzinzik, Andreas L.; Furet, Pascal</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">8120-8135</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The development of asciminib I, an allosteric inhibitor of the BCR-ABL1 kinase for use in treating chronic myelogenous leukemia (CML), is delineated.  I binds to the myristate pocket of BCR-ABL1 and maintains activity against cancer cells possessing ATP-site mutations in BCR-ABL1 that render them resistant to tyrosine kinase inhibitors; cancer cells resistant to myristate-site mutations are likely to be susceptible to ATP-competitive inhibitors so that combinations of I and tyrosine kinase inhibitors should suppress the emergence of resistance.  Fragment-based screening using NMR and X-ray yielded ligands for the myristate pocket, while an NMR-based conformational assay guided the transformation of these inactive ligands into ABL1 inhibitors.  Further structure-based optimization for potency, physicochem., pharmacokinetic, and drug-like properties, culminated in I which is currently undergoing clin. studies in CML patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrIQbEIXXgRgbVg90H21EOLACvtfcHk0lid7zMV7nbQsA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFKru7bN&md5=023a566cdf4faba6dfc05501b8b6a6ed</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b01040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b01040%26sid%3Dliteratum%253Aachs%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DCotesta%26aufirst%3DS.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DDrueckes%26aufirst%3DP.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DGabriel%26aufirst%3DT.%26aulast%3DGroell%26aufirst%3DJ.%2BM.%26aulast%3DGrotzfeld%26aufirst%3DR.%2BM.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DHenry%26aufirst%3DC.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DJones%26aufirst%3DD.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DManley%26aufirst%3DP.%2BW.%26aulast%3DPell%25C3%25A9%26aufirst%3DX.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DSalem%26aufirst%3DB.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DZoller%26aufirst%3DT.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DFuret%26aufirst%3DP.%26atitle%3DDiscovery%2520of%2520Asciminib%2520%2528ABL001%2529%252C%2520an%2520Allosteric%2520Inhibitor%2520of%2520the%2520Tyrosine%2520Kinase%2520Activity%2520of%2520BCR-ABL1%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D8120%26epage%3D8135%26doi%3D10.1021%2Facs.jmedchem.8b01040" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wylie, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schoepfer, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loo, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Furet, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pelle, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donovan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Buonamici, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hassan, A. Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombardo, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iyer, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palmer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berellini, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodd, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thohan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bitter, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Branford, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ross, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petruzzelli, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vanasse, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warmuth, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hofmann, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keen, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sellers, W. R.</span></span> <span> </span><span class="NLM_article-title">The Allosteric Inhibitor ABL001 Enables Dual Targeting of BCR-ABL1</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>543</i></span>,  <span class="NLM_fpage">733</span>– <span class="NLM_lpage">737</span>, <span class="refDoi"> DOI: 10.1038/nature21702</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature21702" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28329763" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=543&publication_year=2017&pages=733-737&author=A.+A.+Wylieauthor=J.+Schoepferauthor=W.+Jahnkeauthor=S.+W.+Cowan-Jacobauthor=A.+Looauthor=P.+Furetauthor=A.+L.+Marzinzikauthor=X.+Pelleauthor=J.+Donovanauthor=W.+Zhuauthor=S.+Buonamiciauthor=A.+Q.+Hassanauthor=F.+Lombardoauthor=V.+Iyerauthor=M.+Palmerauthor=G.+Berelliniauthor=S.+Doddauthor=S.+Thohanauthor=H.+Bitterauthor=S.+Branfordauthor=D.+M.+Rossauthor=T.+P.+Hughesauthor=L.+Petruzzelliauthor=K.+G.+Vanasseauthor=M.+Warmuthauthor=F.+Hofmannauthor=N.+J.+Keenauthor=W.+R.+Sellers&title=The+Allosteric+Inhibitor+ABL001+Enables+Dual+Targeting+of+BCR-ABL1&doi=10.1038%2Fnature21702"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">The allosteric inhibitor ABL001 enables dual targeting of BCR-ABL1</span></div><div class="casAuthors">Wylie, Andrew A.; Schoepfer, Joseph; Jahnke, Wolfgang; Cowan-Jacob, Sandra W.; Loo, Alice; Furet, Pascal; Marzinzik, Andreas L.; Pelle, Xavier; Donovan, Jerry; Zhu, Wenjing; Buonamici, Silvia; Hassan, A. Quamrul; Lombardo, Franco; Iyer, Varsha; Palmer, Michael; Berellini, Giuliano; Dodd, Stephanie; Thohan, Sanjeev; Bitter, Hans; Branford, Susan; Ross, David M.; Hughes, Timothy P.; Petruzzelli, Lilli; Vanasse, K. Gary; Warmuth, Markus; Hofmann, Francesco; Keen, Nicholas J.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">543</span>
        (<span class="NLM_cas:issue">7647</span>),
    <span class="NLM_cas:pages">733-737</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Chronic myeloid leukemia (CML) is driven by the activity of the BCR-ABL1 fusion oncoprotein.  ABL1 kinase inhibitors have improved the clin. outcomes for patients with CML, with over 80% of patients treated with imatinib surviving for more than 10 years.  Second-generation ABL1 kinase inhibitors induce more potent mol. responses in both previously untreated and imatinib-resistant patients with CML.  Studies in patients with chronic-phase CML have shown that around 50% of patients who achieve and maintain undetectable BCR-ABL1 transcript levels for at least 2 years remain disease-free after the withdrawal of treatment.  Here we characterize ABL001 (asciminib), a potent and selective allosteric ABL1 inhibitor that is undergoing clin. development testing in patients with CML and Philadelphia chromosome-pos. (Ph+) acute lymphoblastic leukemia.  In contrast to catalytic-site ABL1 kinase inhibitors, ABL001 binds to the myristoyl pocket of ABL1 and induces the formation of an inactive kinase conformation.  ABL001 and second-generation catalytic inhibitors have similar cellular potencies but distinct patterns of resistance mutations, with genetic barcoding studies revealing pre-existing clonal populations with no shared resistance between ABL001 and the catalytic inhibitor nilotinib.  Consistent with this profile, acquired resistance was obsd. with single-agent therapy in mice; however, the combination of ABL001 and nilotinib led to complete disease control and eradicated CML xenograft tumors without recurrence after the cessation of treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreJ5YSS_wiS7Vg90H21EOLACvtfcHk0lj8_mHXnQxTZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXltVWntbc%253D&md5=20d00f222ab8d9387cb340a9531b8ba5</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1038%2Fnature21702&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature21702%26sid%3Dliteratum%253Aachs%26aulast%3DWylie%26aufirst%3DA.%2BA.%26aulast%3DSchoepfer%26aufirst%3DJ.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DLoo%26aufirst%3DA.%26aulast%3DFuret%26aufirst%3DP.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26aulast%3DPelle%26aufirst%3DX.%26aulast%3DDonovan%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DBuonamici%26aufirst%3DS.%26aulast%3DHassan%26aufirst%3DA.%2BQ.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DIyer%26aufirst%3DV.%26aulast%3DPalmer%26aufirst%3DM.%26aulast%3DBerellini%26aufirst%3DG.%26aulast%3DDodd%26aufirst%3DS.%26aulast%3DThohan%26aufirst%3DS.%26aulast%3DBitter%26aufirst%3DH.%26aulast%3DBranford%26aufirst%3DS.%26aulast%3DRoss%26aufirst%3DD.%2BM.%26aulast%3DHughes%26aufirst%3DT.%2BP.%26aulast%3DPetruzzelli%26aufirst%3DL.%26aulast%3DVanasse%26aufirst%3DK.%2BG.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DHofmann%26aufirst%3DF.%26aulast%3DKeen%26aufirst%3DN.%2BJ.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DThe%2520Allosteric%2520Inhibitor%2520ABL001%2520Enables%2520Dual%2520Targeting%2520of%2520BCR-ABL1%26jtitle%3DNature%26date%3D2017%26volume%3D543%26spage%3D733%26epage%3D737%26doi%3D10.1038%2Fnature21702" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wei, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xi, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span> <span> </span><span class="NLM_article-title">Protein Ensembles: How Does Nature Harness Thermodynamic Fluctuations for Life? The Diverse Functional Roles of Conformational Ensembles in the Cell</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6516</span>– <span class="NLM_lpage">6551</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00562</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00562" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVyhsLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6516-6551&author=G.+Weiauthor=W.+Xiauthor=R.+Nussinovauthor=B.+Ma&title=Protein+Ensembles%3A+How+Does+Nature+Harness+Thermodynamic+Fluctuations+for+Life%3F+The+Diverse+Functional+Roles+of+Conformational+Ensembles+in+the+Cell&doi=10.1021%2Facs.chemrev.5b00562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Ensembles: How Does Nature Harness Thermodynamic Fluctuations for Life? The Diverse Functional Roles of Conformational Ensembles in the Cell</span></div><div class="casAuthors">Wei, Guanghong; Xi, Wenhui; Nussinov, Ruth; Ma, Buyong</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6516-6551</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">All sol. proteins populate conformational ensembles that together constitute the native state.  Their fluctuations in water are intrinsic thermodn. phenomena, and the distributions of the states on the energy landscape are detd. by statistical thermodn.; however, they are optimized to perform their biol. functions.  In this review we briefly describe advances in free energy landscape studies of protein conformational ensembles.  Exptl. (NMR, small-angle X-ray scattering, single-mol. spectroscopy, and cryo-electron microscopy) and computational (replica-exchange mol. dynamics, metadynamics, and Markov state models) approaches have made great progress in recent years.  These address the challenging characterization of the highly flexible and heterogeneous protein ensembles.  We focus on structural aspects of protein conformational distributions, from collective motions of single- and multi-domain proteins, intrinsically disordered proteins, to multiprotein complexes.  Importantly, we highlight recent studies that illustrate functional adjustment of protein conformational ensembles in the crowded cellular environment.  We center on the role of the ensemble in recognition of small- and macro-mols. (protein and RNA/DNA) and emphasize emerging concepts of protein dynamics in enzyme catalysis.  Overall, protein ensembles link fundamental physicochem. principles and protein behavior and the cellular network and its regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2Kcwy8a6LHrVg90H21EOLACvtfcHk0lj8_mHXnQxTZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVyhsLo%253D&md5=fac9ab64e11aa3a4f2f0988bf1db1209</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00562%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DG.%26aulast%3DXi%26aufirst%3DW.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DMa%26aufirst%3DB.%26atitle%3DProtein%2520Ensembles%253A%2520How%2520Does%2520Nature%2520Harness%2520Thermodynamic%2520Fluctuations%2520for%2520Life%253F%2520The%2520Diverse%2520Functional%2520Roles%2520of%2520Conformational%2520Ensembles%2520in%2520the%2520Cell%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6516%26epage%3D6551%26doi%3D10.1021%2Facs.chemrev.5b00562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weis, W. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kobilka, B. K.</span></span> <span> </span><span class="NLM_article-title">The Molecular Basis of G Protein–Coupled Receptor Activation</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>87</i></span>,  <span class="NLM_fpage">897</span>– <span class="NLM_lpage">919</span>, <span class="refDoi"> DOI: 10.1146/annurev-biochem-060614-033910</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biochem-060614-033910" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29925258" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsrvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=87&publication_year=2018&pages=897-919&author=W.+I.+Weisauthor=B.+K.+Kobilka&title=The+Molecular+Basis+of+G+Protein%E2%80%93Coupled+Receptor+Activation&doi=10.1146%2Fannurev-biochem-060614-033910"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">The Molecular Basis of G Protein-Coupled Receptor Activation</span></div><div class="casAuthors">Weis, William I.; Kobilka, Brian K.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">87</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">897-919</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) mediate the majority of cellular responses to external stimuli.  Upon activation by a ligand, the receptor binds to a partner heterotrimeric G protein and promotes exchange of GTP for GDP, leading to dissocn. of the G protein into α and βγ subunits that mediate downstream signals.  GPCRs can also activate distinct signaling pathways through arrestins.  Active states of GPCRs form by small rearrangements of the ligand-binding, or orthosteric, site that are amplified into larger conformational changes.  Mol. understanding of the allosteric coupling between ligand binding and G protein or arrestin interaction is emerging from structures of several GPCRs crystd. in inactive and active states, spectroscopic data, and computer simulations.  The coupling is loose, rather than concerted, and agonist binding does not fully stabilize the receptor in an active conformation.  Distinct intermediates whose populations are shifted by ligands of different efficacies underlie the complex pharmacol. of GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfSIylPxq7yLVg90H21EOLACvtfcHk0lj8_mHXnQxTZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtFOrsrvM&md5=71f1f2d3c1d9be27da70dd472d7d44f2</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-060614-033910&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-060614-033910%26sid%3Dliteratum%253Aachs%26aulast%3DWeis%26aufirst%3DW.%2BI.%26aulast%3DKobilka%26aufirst%3DB.%2BK.%26atitle%3DThe%2520Molecular%2520Basis%2520of%2520G%2520Protein%25E2%2580%2593Coupled%2520Receptor%2520Activation%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2018%26volume%3D87%26spage%3D897%26epage%3D919%26doi%3D10.1146%2Fannurev-biochem-060614-033910" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lazo, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharlow, E. R.</span></span> <span> </span><span class="NLM_article-title">Drugging Undruggable Molecular Cancer Targets</span>. <i>Annu. Rev. Pharmacol. Toxicol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">23</span>– <span class="NLM_lpage">40</span>, <span class="refDoi"> DOI: 10.1146/annurev-pharmtox-010715-103440</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-pharmtox-010715-103440" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26527069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=23-40&author=J.+S.+Lazoauthor=E.+R.+Sharlow&title=Drugging+Undruggable+Molecular+Cancer+Targets&doi=10.1146%2Fannurev-pharmtox-010715-103440"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Undruggable Molecular Cancer Targets</span></div><div class="casAuthors">Lazo, John S.; Sharlow, Elizabeth R.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Pharmacology and Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">23-40</span>CODEN:
                <span class="NLM_cas:coden">ARPTDI</span>;
        ISSN:<span class="NLM_cas:issn">0362-1642</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">Cancer, more than any other human disease, now has a surfeit of potential mol. targets poised for therapeutic exploitation.  Currently, a no. of attractive and validated cancer targets remain outside of the reach of pharmacol. regulation.  Some have been described as undruggable, at least by traditional strategies.  In this article, we outline the basis for the undruggable moniker, propose a reclassification of these targets as undrugged, and highlight three general classes of this imposing group as exemplars with some attendant strategies currently being explored to reclassify them.  Expanding the spectrum of disease-relevant targets to pharmacol. manipulation is central to reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpny-ev1lJKqbVg90H21EOLACvtfcHk0lgSaZu-mbfQkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGlsr7O&md5=5ab0518d4155a00e2a898a6f68d551d8</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1146%2Fannurev-pharmtox-010715-103440&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-pharmtox-010715-103440%26sid%3Dliteratum%253Aachs%26aulast%3DLazo%26aufirst%3DJ.%2BS.%26aulast%3DSharlow%26aufirst%3DE.%2BR.%26atitle%3DDrugging%2520Undruggable%2520Molecular%2520Cancer%2520Targets%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2016%26volume%3D56%26spage%3D23%26epage%3D40%26doi%3D10.1146%2Fannurev-pharmtox-010715-103440" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dang, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soucek, L.</span></span> <span> </span><span class="NLM_article-title">Drugging the “Undruggable” Cancer Targets</span>. <i>Nat. Rev. Cancer</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">508</span>, <span class="refDoi"> DOI: 10.1038/nrc.2017.36</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrc.2017.36" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28643779" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2017&pages=502-508&author=C.+V.+Dangauthor=E.+P.+Reddyauthor=K.+M.+Shokatauthor=L.+Soucek&title=Drugging+the+%E2%80%9CUndruggable%E2%80%9D+Cancer+Targets&doi=10.1038%2Fnrc.2017.36"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the 'undruggable' cancer targets</span></div><div class="casAuthors">Dang, Chi V.; Reddy, E. Premkumar; Shokat, Kevan M.; Soucek, Laura</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">502-508</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The term 'undruggable' was coined to describe proteins that could not be targeted pharmacol.  However, progress is being made to 'drug' many of these targets, and therefore more appropriate terms might be 'difficult to drug' or 'yet to be drugged'.  Many desirable targets in cancer fall into this category, including the RAS and MYC oncogenes, and pharmacol. targeting these intractable proteins is now a key challenge in cancer research that requires innovation and the development of new technologies.  In this Viewpoint article, we asked four scientists working in this field for their opinions on the most crucial advances, as well as the challenges and what the future holds for this important area of research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoElcMCmqGx_rVg90H21EOLACvtfcHk0lgSaZu-mbfQkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVChu7jP&md5=774da7ed04931a3fd3689eb9acea594a</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2017.36&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2017.36%26sid%3Dliteratum%253Aachs%26aulast%3DDang%26aufirst%3DC.%2BV.%26aulast%3DReddy%26aufirst%3DE.%2BP.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DSoucek%26aufirst%3DL.%26atitle%3DDrugging%2520the%2520%25E2%2580%259CUndruggable%25E2%2580%259D%2520Cancer%2520Targets%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2017%26volume%3D17%26spage%3D502%26epage%3D508%26doi%3D10.1038%2Fnrc.2017.36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shao, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moses, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilbert, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalyanaraman, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, Z. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chernova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Journey, S. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissman, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hann, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neckers, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span> <span> </span><span class="NLM_article-title">Exploration of Benzothiazole Rhodacyanines as Allosteric Inhibitors of Protein-Protein Interactions with Heat Shock Protein 70 (Hsp70)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">6163</span>– <span class="NLM_lpage">6177</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.8b00583</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.8b00583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1altb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=6163-6177&author=H.+Shaoauthor=X.+Liauthor=M.+A.+Mosesauthor=L.+A.+Gilbertauthor=C.+Kalyanaramanauthor=Z.+T.+Youngauthor=M.+Chernovaauthor=S.+N.+Journeyauthor=J.+S.+Weissmanauthor=B.+Hannauthor=M.+P.+Jacobsonauthor=L.+Neckersauthor=J.+E.+Gestwicki&title=Exploration+of+Benzothiazole+Rhodacyanines+as+Allosteric+Inhibitors+of+Protein-Protein+Interactions+with+Heat+Shock+Protein+70+%28Hsp70%29&doi=10.1021%2Facs.jmedchem.8b00583"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Exploration of Benzothiazole Rhodacyanines as Allosteric Inhibitors of Protein-Protein Interactions with Heat Shock Protein 70 (Hsp70)</span></div><div class="casAuthors">Shao, Hao; Li, Xiaokai; Moses, Michael A.; Gilbert, Luke A.; Kalyanaraman, Chakrapani; Young, Zapporah T.; Chernova, Margarita; Journey, Sara N.; Weissman, Jonathan S.; Hann, Byron; Jacobson, Matthew P.; Neckers, Len; Gestwicki, Jason E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6163-6177</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cancer cells rely on the chaperone heat shock protein 70 (Hsp70) for survival and proliferation.  Recently, benzothiazole rhodacyanines have been shown to bind an allosteric site on Hsp70, interrupting its binding to nucleotide-exchange factors (NEFs) and promoting cell death in breast cancer cell lines.  However, proof-of-concept mols., such as JG-98, have relatively modest potency (EC50 ≈ 0.7-0.4 μM) and are rapidly metabolized in animals.  Here, the authors explored this chem. series through structure- and property-based design of ∼300 analogs, showing that the most potent had >10-fold improved EC50 values (∼0.05 to 0.03 μM) against two breast cancer cells.  Biomarkers and whole genome CRISPRi screens confirmed members of the Hsp70 family as cellular targets.  On the basis of these results, JG-231 was found to reduce tumor burden in an MDA-MB-231 xenograft model (4 mg/kg, i.p.).  Together, these studies support the hypothesis that Hsp70 may be a promising target for anticancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZKt2SFyYM6rVg90H21EOLACvtfcHk0lgSaZu-mbfQkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1altb3O&md5=16effcd3cd1b4772583fefcc5967a75e</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.8b00583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.8b00583%26sid%3Dliteratum%253Aachs%26aulast%3DShao%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DMoses%26aufirst%3DM.%2BA.%26aulast%3DGilbert%26aufirst%3DL.%2BA.%26aulast%3DKalyanaraman%26aufirst%3DC.%26aulast%3DYoung%26aufirst%3DZ.%2BT.%26aulast%3DChernova%26aufirst%3DM.%26aulast%3DJourney%26aufirst%3DS.%2BN.%26aulast%3DWeissman%26aufirst%3DJ.%2BS.%26aulast%3DHann%26aufirst%3DB.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DNeckers%26aufirst%3DL.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26atitle%3DExploration%2520of%2520Benzothiazole%2520Rhodacyanines%2520as%2520Allosteric%2520Inhibitors%2520of%2520Protein-Protein%2520Interactions%2520with%2520Heat%2520Shock%2520Protein%252070%2520%2528Hsp70%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D6163%26epage%3D6177%26doi%3D10.1021%2Facs.jmedchem.8b00583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pricer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gestwicki, J. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mapp, A. K.</span></span> <span> </span><span class="NLM_article-title">From Fuzzy to Function: The New Frontier of Protein-Protein Interactions</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">584</span>– <span class="NLM_lpage">589</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.6b00565</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.6b00565" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXkt1WktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2017&pages=584-589&author=R.+Pricerauthor=J.+E.+Gestwickiauthor=A.+K.+Mapp&title=From+Fuzzy+to+Function%3A+The+New+Frontier+of+Protein-Protein+Interactions&doi=10.1021%2Facs.accounts.6b00565"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">From Fuzzy to Function: The New Frontier of Protein-Protein Interactions</span></div><div class="casAuthors">Pricer, Rachel; Gestwicki, Jason E.; Mapp, Anna K.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">584-589</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The notion that many proteins have a high degree of disorder has been in the literature for decades.  Such sequences have been commonly regarded as a nuisance; they caused aggregation and pptn. and often eluded structural and mechanistic studies.  These motifs have also been regarded as largely useless from a functional standpoint, perhaps because no few believed that floppy motifs could encode information that facilitated specific function.  Here, the role of these intrinsically disordered motifs in protein-protein interactions is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkBKJUlxPLkLVg90H21EOLACvtfcHk0ljB3um2Q3nYYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXkt1WktL4%253D&md5=70b5132b30c231747534ce42962401dd</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.6b00565&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.6b00565%26sid%3Dliteratum%253Aachs%26aulast%3DPricer%26aufirst%3DR.%26aulast%3DGestwicki%26aufirst%3DJ.%2BE.%26aulast%3DMapp%26aufirst%3DA.%2BK.%26atitle%3DFrom%2520Fuzzy%2520to%2520Function%253A%2520The%2520New%2520Frontier%2520of%2520Protein-Protein%2520Interactions%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2017%26volume%3D50%26spage%3D584%26epage%3D589%26doi%3D10.1021%2Facs.accounts.6b00565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Hidden Allosteric Sites as Novel Targets for Allosteric Drug Design</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">359</span>– <span class="NLM_lpage">365</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2017.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2017.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29030241" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Kqs77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2018&pages=359-365&author=S.+Luauthor=M.+Jiauthor=D.+Niauthor=J.+Zhang&title=Discovery+of+Hidden+Allosteric+Sites+as+Novel+Targets+for+Allosteric+Drug+Design&doi=10.1016%2Fj.drudis.2017.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of hidden allosteric sites as novel targets for allosteric drug design</span></div><div class="casAuthors">Lu, Shaoyong; Ji, Mingfei; Ni, Duan; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">359-365</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Hidden allosteric sites, as a novel type of allosteric site, are invisible in ligand-unbound (apo) crystal structures, but can emerge in ligand-bound (holo) crystal structures when a specific ligand binds to, and stabilizes, a unique conformation favored by the ligand.  However, the identification of these sites is a significant challenge.  Several computational and exptl. approaches have been developed to identify such sites in proteins.  Here, we outline these approaches, with a focus on examples of the successful use of such techniques.  The discovery of hidden allosteric sites offers a new avenue for facilitating drug design by greatly expanding the repertoire of available drug targets, contributing to the search for allosteric drugs for the treatment of human diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpObJvqLArGy7Vg90H21EOLACvtfcHk0ljB3um2Q3nYYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Kqs77F&md5=da29385caf2393feb520aa402dbf4134</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2017.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2017.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJi%26aufirst%3DM.%26aulast%3DNi%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Hidden%2520Allosteric%2520Sites%2520as%2520Novel%2520Targets%2520for%2520Allosteric%2520Drug%2520Design%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2018%26volume%3D23%26spage%3D359%26epage%3D365%26doi%3D10.1016%2Fj.drudis.2017.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brogi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramunno, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chemi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfano, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecorelli, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pambianchi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galatello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Compagnoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Focher, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biamonti, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valacchi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butini, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gemma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campiani, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brindisi, M.</span></span> <span> </span><span class="NLM_article-title">First Dual AK/GSK-3β Inhibitors Endowed with Antioxidant Properties as Multifunctional, Potential Neuroprotective Agents</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>138</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">457</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2017.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ejmech.2017.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28689095" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2017&pages=438-457&author=S.+Brogiauthor=A.+Ramunnoauthor=L.+Saviauthor=G.+Chemiauthor=G.+Alfanoauthor=A.+Pecorelliauthor=E.+Pambianchiauthor=P.+Galatelloauthor=G.+Compagnoniauthor=F.+Focherauthor=G.+Biamontiauthor=G.+Valacchiauthor=S.+Butiniauthor=S.+Gemmaauthor=G.+Campianiauthor=M.+Brindisi&title=First+Dual+AK%2FGSK-3%CE%B2+Inhibitors+Endowed+with+Antioxidant+Properties+as+Multifunctional%2C+Potential+Neuroprotective+Agents&doi=10.1016%2Fj.ejmech.2017.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">First dual AK/GSK-3β inhibitors endowed with antioxidant properties as multifunctional, potential neuroprotective agents</span></div><div class="casAuthors">Brogi, Simone; Ramunno, Anna; Savi, Lida; Chemi, Giulia; Alfano, Gloria; Pecorelli, Alessandra; Pambianchi, Erika; Galatello, Paola; Compagnoni, Giulia; Focher, Federico; Biamonti, Giuseppe; Valacchi, Giuseppe; Butini, Stefania; Gemma, Sandra; Campiani, Giuseppe; Brindisi, Margherita</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">438-457</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The manuscript deals with the design, synthesis and biol. evaluation of novel benzoxazinone- and indole-based compds. as multifunctional neuroprotective agents.  These compds. inhibit human adenosine kinase (hAK) and human glycogen synthase kinase 3 beta (hGSK-3β) enzymes.  In silico hints drove the synthesis of appropriately decorated benzoxazinones I (R1 = Ph, 1-naphthyl, 2-naphthyl; R2 = CH2C6H5, 4-ClC6H4CH2, 4-O2NC6H4CH2, etc.; X = H, F, morpholin-1-yl; Y = H; XY = OCH2O; Z = H, O) and indoles II (R1 = 4-H3CC6H4, 1-naphthyl; R2 = CH3, N(Et)2) and biochem. anal. revealed their behavior as allosteric inhibitors of hGSK-3β.  For hit III and compds. I (R1 = 1-naphthyl; R2 = 4-FC6H4CH2; X = Y = H; Z = O), I (R1 = 1-naphthyl; R2 = CH2C6H5; XY = OCH2O; Z = O) and II (R1 = 1-naphthyl; R2 = CH3) the potential antioxidant profile was assessed in human neuroblastoma cell lines (IMR 32), by evaluating the protective effect of selected compds. against H2O2 cytotoxicity and reactive oxygen species (ROS) prodn.  Results showed a strong efficacy of the tested compds., even at the lower doses, in counteracting the induced oxidative stress (50 μM of H2O2) and in preventing ROS formation.  In addn., the tested compds. did not show any cytotoxic effect detd. by the LDH release, at the concn. range analyzed (from 0.1 to 50 μM).  This study allowed the identification of compd. I (R1 = 1-naphthyl; R2 = CH2C6H5; XY = OCH2O; Z = O), as the first dual hAK/hGSK-3β inhibitor reported to date.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrno6lxql7c_7Vg90H21EOLACvtfcHk0ljB3um2Q3nYYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtFChurvO&md5=1ea9d5251ac6d10f173c170e7c749b4b</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2017.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2017.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DBrogi%26aufirst%3DS.%26aulast%3DRamunno%26aufirst%3DA.%26aulast%3DSavi%26aufirst%3DL.%26aulast%3DChemi%26aufirst%3DG.%26aulast%3DAlfano%26aufirst%3DG.%26aulast%3DPecorelli%26aufirst%3DA.%26aulast%3DPambianchi%26aufirst%3DE.%26aulast%3DGalatello%26aufirst%3DP.%26aulast%3DCompagnoni%26aufirst%3DG.%26aulast%3DFocher%26aufirst%3DF.%26aulast%3DBiamonti%26aufirst%3DG.%26aulast%3DValacchi%26aufirst%3DG.%26aulast%3DButini%26aufirst%3DS.%26aulast%3DGemma%26aufirst%3DS.%26aulast%3DCampiani%26aufirst%3DG.%26aulast%3DBrindisi%26aufirst%3DM.%26atitle%3DFirst%2520Dual%2520AK%252FGSK-3%25CE%25B2%2520Inhibitors%2520Endowed%2520with%2520Antioxidant%2520Properties%2520as%2520Multifunctional%252C%2520Potential%2520Neuroprotective%2520Agents%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2017%26volume%3D138%26spage%3D438%26epage%3D457%26doi%3D10.1016%2Fj.ejmech.2017.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group"><span class="NLM_string-name">An, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Are the Apo Proteins Suitable for the Rational Discovery of Allosteric Drugs?</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">597</span>– <span class="NLM_lpage">604</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.8b00735</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.8b00735" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisVCisb3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2019&pages=597-604&author=X.+Anauthor=S.+Luauthor=K.+Songauthor=Q.+Shenauthor=M.+Huangauthor=X.+Yaoauthor=H.+Liuauthor=J.+Zhang&title=Are+the+Apo+Proteins+Suitable+for+the+Rational+Discovery+of+Allosteric+Drugs%3F&doi=10.1021%2Facs.jcim.8b00735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Are the Apo Proteins Suitable for the Rational Discovery of Allosteric Drugs?</span></div><div class="casAuthors">An, Xiaoli; Lu, Shaoyong; Song, Kun; Shen, Qiancheng; Huang, Meilan; Yao, Xiaojun; Liu, Huanxiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">597-604</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric modulators, by targeting the less-conserved allosteric sites, represent an innovative strategy in drug discovery.  These modulators have a distinctive advantage over orthosteric ligands that attach to the conserved, functional orthosteric sites.  However, in structure-based drug design, it remains unclear whether allosteric protein structures detd. without orthosteric ligand binding are suitable for allosteric drug screening.  In this study, we performed large-scale conformational samplings of six representative allosteric proteins uncomplexed (apo) and complexed (holo) with orthosteric ligands to explore the effect of orthosteric site binding on the conformational dynamics of allosteric sites.  The results, coupled with the redocking evaluation of allosteric modulators to their apo and holo proteins using their MD trajectories, indicated that orthosteric site binding had an effect on the dynamics of the allosteric sites and allosteric modulators preferentially bound to their holo proteins.  According to the anal. data, we constructed a new correlation model for quantifying the allosteric site change driven by substrate binding to the orthosteric site.  These results highlight the strong demand to select holo allosteric proteins as initial inputs in structure-based allosteric drug screening when the distance between orthosteric and allosteric sites in the protein is below 5 Å, which is expected to contribute to allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2oyvmsSoYj7Vg90H21EOLACvtfcHk0ljB3um2Q3nYYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisVCisb3J&md5=b98fc6933d1075664cf53d3d45520339</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.8b00735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.8b00735%26sid%3Dliteratum%253Aachs%26aulast%3DAn%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DHuang%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAre%2520the%2520Apo%2520Proteins%2520Suitable%2520for%2520the%2520Rational%2520Discovery%2520of%2520Allosteric%2520Drugs%253F%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2019%26volume%3D59%26spage%3D597%26epage%3D604%26doi%3D10.1021%2Facs.jcim.8b00735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">Drugging Ras GTPase: A Comprehensive Mechanistic and Signaling Structural View</span>. <i>Chem. Soc. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">4929</span>– <span class="NLM_lpage">4952</span>, <span class="refDoi"> DOI: 10.1039/C5CS00911A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2FC5CS00911A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27396271" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFCiurrK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2016&pages=4929-4952&author=S.+Luauthor=H.+Jangauthor=S.+Guauthor=J.+Zhangauthor=R.+Nussinov&title=Drugging+Ras+GTPase%3A+A+Comprehensive+Mechanistic+and+Signaling+Structural+View&doi=10.1039%2FC5CS00911A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging Ras GTPase: a comprehensive mechanistic and signaling structural view</span></div><div class="casAuthors">Lu, Shaoyong; Jang, Hyunbum; Gu, Shuo; Zhang, Jian; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Society Reviews</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4929-4952</span>CODEN:
                <span class="NLM_cas:coden">CSRVBR</span>;
        ISSN:<span class="NLM_cas:issn">0306-0012</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Ras proteins are small GTPases, cycling between inactive GDP-bound and active GTP-bound states.  Through these switches they regulate signaling that controls cell growth and proliferation.  Activating Ras mutations are assocd. with approx. 30% of human cancers, which are frequently resistant to std. therapies.  Over the past few years, structural biol. and in silico drug design, coupled with improved screening technol., led to a handful of promising inhibitors, raising the possibility of drugging Ras proteins.  At the same time, the invariable emergence of drug resistance argues for the crit. importance of addnl. honing in on signaling pathways which are likely to be involved.  Here we overview current advances in Ras structural knowledge, including the conformational dynamic of full-length Ras in soln. and at the membrane, therapeutic inhibition of Ras activity by targeting its active site, allosteric sites, and Ras-effector protein-protein interfaces, Ras dimers, the K-Ras4B/calmodulin/PI3Kα trimer, and targeting Ras with siRNA.  To mitigate drug resistance, we propose signaling pathways that can be co-targeted along with Ras and explain why.  These include pathways leading to the expression (or activation) of YAP1 and c-Myc.  We postulate that these and Ras signaling pathways, MAPK/ERK and PI3K/Akt/mTOR, act independently and in corresponding ways in cell cycle control.  The structural data are instrumental in the discovery and development of Ras inhibitors for treating RAS-driven cancers.  Together with the signaling blueprints through which drug resistance can evolve, this review provides a comprehensive and innovative master plan for tackling mutant Ras proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBc0hxnrzfRrVg90H21EOLACvtfcHk0liILRaQQY3rxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFCiurrK&md5=f98483885d066c0ec809535b1caefc8a</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1039%2FC5CS00911A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5CS00911A%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DDrugging%2520Ras%2520GTPase%253A%2520A%2520Comprehensive%2520Mechanistic%2520and%2520Signaling%2520Structural%2520View%26jtitle%3DChem.%2520Soc.%2520Rev.%26date%3D2016%26volume%3D45%26spage%3D4929%26epage%3D4952%26doi%3D10.1039%2FC5CS00911A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cox, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fesik, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kimmelman, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Der, C. J.</span></span> <span> </span><span class="NLM_article-title">Drugging the Undruggable RAS: Mission Possible?</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">828</span>– <span class="NLM_lpage">851</span>, <span class="refDoi"> DOI: 10.1038/nrd4389</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd4389" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25323927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=828-851&author=A.+D.+Coxauthor=S.+W.+Fesikauthor=A.+C.+Kimmelmanauthor=J.+Luoauthor=C.+J.+Der&title=Drugging+the+Undruggable+RAS%3A+Mission+Possible%3F&doi=10.1038%2Fnrd4389"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging the undruggable RAS: Mission Possible?</span></div><div class="casAuthors">Cox, Adrienne D.; Fesik, Stephen W.; Kimmelman, Alec C.; Luo, Ji; Der, Channing J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">828-851</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Despite more than three decades of intensive effort, no effective pharmacol. inhibitors of the RAS oncoproteins have reached the clinic, prompting the widely held perception that RAS proteins are 'undruggable'.  However, recent data from the lab. and the clinic have renewed our hope for the development of RAS-inhibitory mols.  In this Review, we summarize the progress and the promise of five key approaches.  Firstly, we focus on the prospects of using direct inhibitors of RAS.  Secondly, we address the issue of whether blocking RAS membrane assocn. is a viable approach.  Thirdly, we assess the status of targeting RAS downstream effector signalling, which is arguably the most favorable current approach.  Fourthly, we address whether the search for synthetic lethal interactors of mutant RAS still holds promise.  Finally, RAS-mediated changes in cell metab. have recently been described and we discuss whether these changes could be exploited for new therapeutic directions.  We conclude with perspectives on how addnl. complexities, which are not yet fully understood, may affect each of these approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvWzW1JqxN4LVg90H21EOLACvtfcHk0liILRaQQY3rxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslKmu7zE&md5=6c8645293b079023de7d31011ae1a192</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1038%2Fnrd4389&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd4389%26sid%3Dliteratum%253Aachs%26aulast%3DCox%26aufirst%3DA.%2BD.%26aulast%3DFesik%26aufirst%3DS.%2BW.%26aulast%3DKimmelman%26aufirst%3DA.%2BC.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DDer%26aufirst%3DC.%2BJ.%26atitle%3DDrugging%2520the%2520Undruggable%2520RAS%253A%2520Mission%2520Possible%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2014%26volume%3D13%26spage%3D828%26epage%3D851%26doi%3D10.1038%2Fnrd4389" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gustafson, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyerowitz, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nekritz, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benes, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charron, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simonds, E. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seeger, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matthay, K. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hertz, N. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eilers, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weiss, W. A.</span></span> <span> </span><span class="NLM_article-title">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span>. <i>Cancer Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">414</span>– <span class="NLM_lpage">427</span>, <span class="refDoi"> DOI: 10.1016/j.ccr.2014.07.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ccr.2014.07.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25175806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=414-427&author=W.+C.+Gustafsonauthor=J.+G.+Meyerowitzauthor=E.+A.+Nekritzauthor=J.+Chenauthor=C.+Benesauthor=E.+Charronauthor=E.+F.+Simondsauthor=R.+Seegerauthor=K.+K.+Matthayauthor=N.+T.+Hertzauthor=M.+Eilersauthor=K.+M.+Shokatauthor=W.+A.+Weiss&title=Drugging+MYCN+through+an+Allosteric+Transition+in+Aurora+Kinase+A&doi=10.1016%2Fj.ccr.2014.07.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging MYCN through an Allosteric Transition in Aurora Kinase A</span></div><div class="casAuthors">Gustafson, William Clay; Meyerowitz, Justin Gabriel; Nekritz, Erin A.; Chen, Justin; Benes, Cyril; Charron, Elise; Simonds, Erin F.; Seeger, Robert; Matthay, Katherine K.; Hertz, Nicholas T.; Eilers, Martin; Shokat, Kevan M.; Weiss, William A.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">414-427</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">MYC proteins are major drivers of cancer, yet are considered undruggable because their DNA binding domains are composed of two extended alpha helixes with no apparent surfaces for small-mol. binding.  Proteolytic degrdn. of MYCN protein is regulated in part by a kinase-independent function of Aurora A.  We describe a class of inhibitors that disrupts the native conformation of Aurora A and drives the degrdn. of MYCN protein across MYCN-driven cancers.  Comparison of cocrystal structures with structure-activity relationships across multiple inhibitors and chemotypes, coupled with mechanistic studies and biochem. assays, delineates an Aurora A conformation-specific effect on proteolytic degrdn. of MYCN, rather than simple nanomolar-level inhibition of Aurora A kinase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8At2LNPE-ubVg90H21EOLACvtfcHk0liILRaQQY3rxw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKmsLnJ&md5=20944894851fe5466422c149a0a0b266</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2014.07.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2014.07.015%26sid%3Dliteratum%253Aachs%26aulast%3DGustafson%26aufirst%3DW.%2BC.%26aulast%3DMeyerowitz%26aufirst%3DJ.%2BG.%26aulast%3DNekritz%26aufirst%3DE.%2BA.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DBenes%26aufirst%3DC.%26aulast%3DCharron%26aufirst%3DE.%26aulast%3DSimonds%26aufirst%3DE.%2BF.%26aulast%3DSeeger%26aufirst%3DR.%26aulast%3DMatthay%26aufirst%3DK.%2BK.%26aulast%3DHertz%26aufirst%3DN.%2BT.%26aulast%3DEilers%26aufirst%3DM.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26aulast%3DWeiss%26aufirst%3DW.%2BA.%26atitle%3DDrugging%2520MYCN%2520through%2520an%2520Allosteric%2520Transition%2520in%2520Aurora%2520Kinase%2520A%26jtitle%3DCancer%2520Cell%26date%3D2014%26volume%3D26%26spage%3D414%26epage%3D427%26doi%3D10.1016%2Fj.ccr.2014.07.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rehmani, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Esaki, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">de Serrano, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, A.</span></span> <span> </span><span class="NLM_article-title">Pirin Is an Iron-Dependent Redox Regulator of NF- B</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">9722</span>– <span class="NLM_lpage">9727</span>, <span class="refDoi"> DOI: 10.1073/pnas.1221743110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1221743110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23716661" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=9722-9727&author=F.+Liuauthor=I.+Rehmaniauthor=S.+Esakiauthor=R.+Fuauthor=L.+Chenauthor=V.+de+Serranoauthor=A.+Liu&title=Pirin+Is+an+Iron-Dependent+Redox+Regulator+of+NF-+B&doi=10.1073%2Fpnas.1221743110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Pirin is an iron-dependent redox regulator of NF-κB</span></div><div class="casAuthors">Liu, Fange; Rehmani, Imran; Esaki, Shingo; Fu, Rong; Chen, Lirong; de Serrano, Vesna; Liu, Aimin</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">9722-9727, S9722/1-S9722/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Pirin is a nuclear nonheme Fe protein of unknown function present in all human tissues.  Here the authors describe that pirin may act as a redox sensor for the nuclear factor κB (NF-κB) transcription factor, a crit. mediator of intracellular signaling that has been linked to cellular responses to proinflammatory signals and controls the expression of a vast array of genes involved in immune and stress responses.  Pirin's regulatory effect was tested with several metals and at different oxidns. states, and the authors' spectroscopic results show that only the ferric form of pirin substantially facilitates binding of NF-κB proteins to target κB genes, a finding that suggests that pirin performs a redox-sensing role in NF-κB regulation.  The mol. mechanism of such a metal identity- and redox state-dependent regulation is revealed by the authors' structural studies of pirin.  The ferrous and ferric pirin proteins differ only by one electron, yet they have distinct conformations.  The Fe center is shown to play an allosteric role on an β-shaped surface area that has two distinct conformations based on the identity and the formal redox state of the metal.  The authors show that the R-shaped area composes the interface for pirin-NF-κB binding that is responsible for modulation of NF-κB's DNA-binding properties.  The nonheme Fe protein pirin is proposed to serve as a reversible functional switch that enables NF-κB to respond to changes in the redox levels of the cell nucleus.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7Tm-I0Q4mD7Vg90H21EOLACvtfcHk0lgxvTMm6Feo8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOrt7bF&md5=d81627e3a8aa09ee6a46a8557719394d</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1221743110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1221743110%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DRehmani%26aufirst%3DI.%26aulast%3DEsaki%26aufirst%3DS.%26aulast%3DFu%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DL.%26aulast%3Dde%2BSerrano%26aufirst%3DV.%26aulast%3DLiu%26aufirst%3DA.%26atitle%3DPirin%2520Is%2520an%2520Iron-Dependent%2520Redox%2520Regulator%2520of%2520NF-%2520B%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D9722%26epage%3D9727%26doi%3D10.1073%2Fpnas.1221743110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Konidaris, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasser, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonks, N. K.</span></span> <span> </span><span class="NLM_article-title">A Potent, Selective, and Orally Bioavailable Inhibitor of the Protein-Tyrosine Phosphatase PTP1B Improves Insulin and Leptin Signaling in Animal Models</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>293</i></span>,  <span class="NLM_fpage">1517</span>– <span class="NLM_lpage">1525</span>, <span class="refDoi"> DOI: 10.1074/jbc.C117.819110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.C117.819110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29217773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXnsVygurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=293&publication_year=2018&pages=1517-1525&author=N.+Krishnanauthor=K.+F.+Konidarisauthor=G.+Gasserauthor=N.+K.+Tonks&title=A+Potent%2C+Selective%2C+and+Orally+Bioavailable+Inhibitor+of+the+Protein-Tyrosine+Phosphatase+PTP1B+Improves+Insulin+and+Leptin+Signaling+in+Animal+Models&doi=10.1074%2Fjbc.C117.819110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">A potent, selective, and orally bioavailable inhibitor of the protein-tyrosine phosphatase PTP1B improves insulin and leptin signaling in animal models</span></div><div class="casAuthors">Krishnan, Navasona; Konidaris, Konstantis F.; Gasser, Gilles; Tonks, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">293</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1517-1525</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein tyrosine phosphatase PTP1B is a neg. regulator of insulin and leptin signaling and a highly validated therapeutic target for diabetes and obesity.  Conventional approaches to drug development have produced potent and specific PTP1B inhibitors, but these inhibitors lack oral bioavailability, which limits their potential for drug development.  Here, the authors report that DPM-1001, an analog of the specific PTP1B inhibitor trodusquemine (MSI-1436), is a potent, specific, and orally bioavailable inhibitor of PTP1B.  DPM-1001 also chelates copper, which enhanced its potency as a PTP1B inhibitor.  DPM-1001 displayed anti-diabetic properties that were assocd. with enhanced signaling through insulin and leptin receptors in animal models of diet-induced obesity.  Therefore, DPM-1001 represents a proof of concept for a new approach to therapeutic intervention in diabetes and obesity.  Although the PTPs have been considered undruggable, the findings of this study suggest that allosteric PTP inhibitors may help reinvigorate drug development efforts that focus on this important family of signal-transducing enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBBIeqnX9-xLVg90H21EOLACvtfcHk0lgxvTMm6Feo8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXnsVygurY%253D&md5=0c020dfd2dbb33d80e925f0755c9b0f0</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C117.819110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C117.819110%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DN.%26aulast%3DKonidaris%26aufirst%3DK.%2BF.%26aulast%3DGasser%26aufirst%3DG.%26aulast%3DTonks%26aufirst%3DN.%2BK.%26atitle%3DA%2520Potent%252C%2520Selective%252C%2520and%2520Orally%2520Bioavailable%2520Inhibitor%2520of%2520the%2520Protein-Tyrosine%2520Phosphatase%2520PTP1B%2520Improves%2520Insulin%2520and%2520Leptin%2520Signaling%2520in%2520Animal%2520Models%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2018%26volume%3D293%26spage%3D1517%26epage%3D1525%26doi%3D10.1074%2Fjbc.C117.819110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">10205</span>– <span class="NLM_lpage">10219</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01520</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01520" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=10205-10219&author=J.+Xieauthor=X.+Siauthor=S.+Guauthor=M.+Wangauthor=J.+Shenauthor=H.+Liauthor=D.+Liauthor=Y.+Fangauthor=C.+Liuauthor=J.+Zhu&title=Allosteric+Inhibitors+of+SHP2+with+Therapeutic+Potential+for+Cancer+Treatment&doi=10.1021%2Facs.jmedchem.7b01520"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric Inhibitors of SHP2 with Therapeutic Potential for Cancer Treatment</span></div><div class="casAuthors">Xie, Jingjing; Si, Xiaojia; Gu, Shoulai; Wang, Mingliang; Shen, Jian; Li, Haoyan; Li, Dan; Fang, Yanjia; Liu, Cong; Zhu, Jidong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">10205-10219</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SHP2, a cytoplasmic protein tyrosine phosphatase encoded by the PTPN11 gene, is involved in multiple cell signaling processes including Ras/MAPK and Hippo/YAP pathways.  SHP2 has been shown to contribute to the progression of a no. of cancer types including leukemia, gastric, and breast cancers.  It also regulates T-cell activation by interacting with inhibitory immune checkpoint receptors such as the programmed cell death 1 (PD-1) and B- and T-lymphocyte attenuator (BTLA).  Thus, SHP2 inhibitors have drawn great attention by both inhibiting tumor cell proliferation and activating T cell immune responses toward cancer cells.  In this study, the authors report the identification of an allosteric SHP2 inhibitor I that locks SHP2 in a closed conformation by binding to the interface of the N-terminal SH2, C-terminal SH2, and phosphatase domains.  Compd. I suppresses MAPK signaling pathway and YAP transcriptional activity and shows antitumor activity in vivo.  The results indicate that allosteric inhibition of SHP2 could be a feasible approach for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriWHA25LKpH7Vg90H21EOLACvtfcHk0lgxvTMm6Feo8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVCks7%252FM&md5=4aaabafae3250476b7bf3687d83cffd3</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01520&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01520%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DJ.%26aulast%3DSi%26aufirst%3DX.%26aulast%3DGu%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DD.%26aulast%3DFang%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DC.%26aulast%3DZhu%26aufirst%3DJ.%26atitle%3DAllosteric%2520Inhibitors%2520of%2520SHP2%2520with%2520Therapeutic%2520Potential%2520for%2520Cancer%2520Treatment%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D10205%26epage%3D10219%26doi%3D10.1021%2Facs.jmedchem.7b01520" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Krishnan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Koveal, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xue, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akshinthala, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kragelj, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jensen, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gauss, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Page, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackledge, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muthuswamy, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peti, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tonks, N. K.</span></span> <span> </span><span class="NLM_article-title">Targeting the Disordered C Terminus of PTP1B with an Allosteric Inhibitor</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">558</span>– <span class="NLM_lpage">566</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1528</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1528" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24845231" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXot1ylsL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=558-566&author=N.+Krishnanauthor=D.+Kovealauthor=D.+H.+Millerauthor=B.+Xueauthor=S.+D.+Akshinthalaauthor=J.+Krageljauthor=M.+R.+Jensenauthor=C.+M.+Gaussauthor=R.+Pageauthor=M.+Blackledgeauthor=S.+K.+Muthuswamyauthor=W.+Petiauthor=N.+K.+Tonks&title=Targeting+the+Disordered+C+Terminus+of+PTP1B+with+an+Allosteric+Inhibitor&doi=10.1038%2Fnchembio.1528"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting the disordered C terminus of PTP1B with an allosteric inhibitor</span></div><div class="casAuthors">Krishnan, Navasona; Koveal, Dorothy; Miller, Daniel H.; Xue, Bin; Akshinthala, Sai Dipikaa; Kragelj, Jaka; Jensen, Malene Ringkjoebing; Gauss, Carla-Maria; Page, Rebecca; Blackledge, Martin; Muthuswamy, Senthil K.; Peti, Wolfgang; Tonks, Nicholas K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">558-566</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">PTP1B, a validated therapeutic target for diabetes and obesity, has a crit. pos. role in HER2 signaling in breast tumorigenesis.  Efforts to develop therapeutic inhibitors of PTP1B have been frustrated by the chem. properties of the active site.  We define a new mechanism of allosteric inhibition that targets the C-terminal, noncatalytic segment of PTP1B.  We present what is to our knowledge the first ensemble structure of PTP1B contg. this intrinsically disordered segment, within which we identified a binding site for the small-mol. inhibitor MSI-1436.  We demonstrate binding to a second site close to the catalytic domain, with cooperative effects between the two sites locking PTP1B in an inactive state.  MSI-1436 antagonized HER2 signaling, inhibited tumorigenesis in xenografts and abrogated metastasis in the NDL2 mouse model of breast cancer, validating inhibition of PTP1B as a therapeutic strategy in breast cancer.  This new approach to inhibition of PTP1B emphasizes the potential of disordered segments of proteins as specific binding sites for therapeutic small mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo526if7ja9XbVg90H21EOLACvtfcHk0liItMp0ihoslw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXot1ylsL8%253D&md5=10a460f3a5deafb9f7c206246d67a29b</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1528&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1528%26sid%3Dliteratum%253Aachs%26aulast%3DKrishnan%26aufirst%3DN.%26aulast%3DKoveal%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DD.%2BH.%26aulast%3DXue%26aufirst%3DB.%26aulast%3DAkshinthala%26aufirst%3DS.%2BD.%26aulast%3DKragelj%26aufirst%3DJ.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DGauss%26aufirst%3DC.%2BM.%26aulast%3DPage%26aufirst%3DR.%26aulast%3DBlackledge%26aufirst%3DM.%26aulast%3DMuthuswamy%26aufirst%3DS.%2BK.%26aulast%3DPeti%26aufirst%3DW.%26aulast%3DTonks%26aufirst%3DN.%2BK.%26atitle%3DTargeting%2520the%2520Disordered%2520C%2520Terminus%2520of%2520PTP1B%2520with%2520an%2520Allosteric%2520Inhibitor%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D558%26epage%3D566%26doi%3D10.1038%2Fnchembio.1528" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stanevich, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satyshur, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wadzinski, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, Y.</span></span> <span> </span><span class="NLM_article-title">Structural Basis of Protein Phosphatase 2A Stable Latency</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>4</i></span>,  <span class="NLM_fpage">1611</span>– <span class="NLM_lpage">1699</span>, <span class="refDoi"> DOI: 10.1038/ncomms2663</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms2663" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23511475" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2013&pages=1611-1699&author=L.+Jiangauthor=V.+Stanevichauthor=K.+A.+Satyshurauthor=M.+Kongauthor=G.+R.+Watkinsauthor=B.+E.+Wadzinskiauthor=R.+Senguptaauthor=Y.+Xing&title=Structural+Basis+of+Protein+Phosphatase+2A+Stable+Latency&doi=10.1038%2Fncomms2663"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1038%2Fncomms2663&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms2663%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DL.%26aulast%3DStanevich%26aufirst%3DV.%26aulast%3DSatyshur%26aufirst%3DK.%2BA.%26aulast%3DKong%26aufirst%3DM.%26aulast%3DWatkins%26aufirst%3DG.%2BR.%26aulast%3DWadzinski%26aufirst%3DB.%2BE.%26aulast%3DSengupta%26aufirst%3DR.%26aulast%3DXing%26aufirst%3DY.%26atitle%3DStructural%2520Basis%2520of%2520Protein%2520Phosphatase%25202A%2520Stable%2520Latency%26jtitle%3DNat.%2520Commun.%26date%3D2013%26volume%3D4%26spage%3D1611%26epage%3D1699%26doi%3D10.1038%2Fncomms2663" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scheuermann, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Key, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naidoo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Longgood, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frantz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tambar, U. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, K. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bruick, R. K.</span></span> <span> </span><span class="NLM_article-title">Allosteric Inhibition of Hypoxia Inducible Factor-2 with Small Molecules</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">276</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1185</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1185" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23434853" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFygtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=271-276&author=T.+H.+Scheuermannauthor=Q.+Liauthor=H.+W.+Maauthor=J.+Keyauthor=L.+Zhangauthor=R.+Chenauthor=J.+A.+Garciaauthor=J.+Naidooauthor=J.+Longgoodauthor=D.+E.+Frantzauthor=U.+K.+Tambarauthor=K.+H.+Gardnerauthor=R.+K.+Bruick&title=Allosteric+Inhibition+of+Hypoxia+Inducible+Factor-2+with+Small+Molecules&doi=10.1038%2Fnchembio.1185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric inhibition of hypoxia inducible factor-2 with small molecules</span></div><div class="casAuthors">Scheuermann, Thomas H.; Li, Qiming; Ma, He-Wen; Key, Jason; Zhang, Lei; Chen, Rui; Garcia, Joseph A.; Naidoo, Jacinth; Longgood, Jamie; Frantz, Doug E.; Tambar, Uttam K.; Gardner, Kevin H.; Bruick, Richard K.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-276</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Hypoxia inducible factors (HIFs) are heterodimeric transcription factors induced in many cancers where they frequently promote the expression of protumorigenic pathways.  Though transcription factors are typically considered 'undruggable', the PAS-B domain of the HIF-2α subunit contains a large cavity within its hydrophobic core that offers a unique foothold for small-mol. regulation.  Here we identify artificial ligands that bind within this pocket and characterize the resulting structural and functional changes caused by binding.  Notably, these ligands antagonize HIF-2 heterodimerization and DNA-binding activity in vitro and in cultured cells, reducing HIF-2 target gene expression.  Despite the high sequence identity between HIF-2α and HIF-1α, these ligands are highly selective and do not affect HIF-1 function.  These chem. tools establish the mol. basis for selective regulation of HIF-2, providing potential therapeutic opportunities to intervene in HIF-2-driven tumors, such as renal cell carcinomas.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBimby6zSuR7Vg90H21EOLACvtfcHk0liItMp0ihoslw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFygtLc%253D&md5=e2e58842b3fe6472248c10012de365fb</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1185%26sid%3Dliteratum%253Aachs%26aulast%3DScheuermann%26aufirst%3DT.%2BH.%26aulast%3DLi%26aufirst%3DQ.%26aulast%3DMa%26aufirst%3DH.%2BW.%26aulast%3DKey%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DJ.%2BA.%26aulast%3DNaidoo%26aufirst%3DJ.%26aulast%3DLonggood%26aufirst%3DJ.%26aulast%3DFrantz%26aufirst%3DD.%2BE.%26aulast%3DTambar%26aufirst%3DU.%2BK.%26aulast%3DGardner%26aufirst%3DK.%2BH.%26aulast%3DBruick%26aufirst%3DR.%2BK.%26atitle%3DAllosteric%2520Inhibition%2520of%2520Hypoxia%2520Inducible%2520Factor-2%2520with%2520Small%2520Molecules%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D271%26epage%3D276%26doi%3D10.1038%2Fnchembio.1185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Minus, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vohidov, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kasembeli, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Long, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krueger, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevens, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kolosov, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tweardy, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sison, E. A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redell, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ball, Z. T.</span></span> <span> </span><span class="NLM_article-title">Rhodium(II) Proximity-Labeling Identifies a Novel Target Site on STAT3 for Inhibitors with Potent Anti-Leukemia Activity</span>. <i>Angew. Chem., Int. Ed.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>54</i></span>,  <span class="NLM_fpage">13085</span>– <span class="NLM_lpage">13089</span>, <span class="refDoi"> DOI: 10.1002/anie.201506889</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fanie.201506889" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVOmsr3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2015&pages=13085-13089&author=M.+B.+Minusauthor=W.+Liuauthor=F.+Vohidovauthor=M.+M.+Kasembeliauthor=X.+Longauthor=M.+J.+Kruegerauthor=A.+Stevensauthor=M.+I.+Kolosovauthor=D.+J.+Tweardyauthor=E.+A.+R.+Sisonauthor=M.+S.+Redellauthor=Z.+T.+Ball&title=Rhodium%28II%29+Proximity-Labeling+Identifies+a+Novel+Target+Site+on+STAT3+for+Inhibitors+with+Potent+Anti-Leukemia+Activity&doi=10.1002%2Fanie.201506889"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Rhodium(II) Proximity-Labeling Identifies a Novel Target Site on STAT3 for Inhibitors with Potent Anti-Leukemia Activity</span></div><div class="casAuthors">Minus, Matthew B.; Liu, Wei; Vohidov, Farrukh; Kasembeli, Moses M.; Long, Xin; Krueger, Michael J.; Stevens, Alexandra; Kolosov, Mikhail I.; Tweardy, David J.; Sison, Edward Allan R.; Redell, Michele S.; Ball, Zachary T.</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">13085-13089</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Nearly 40 % of children with acute myeloid leukemia (AML) suffer relapse arising from chemoresistance, often involving upregulation of the oncoprotein STAT3 (signal transducer and activator of transcription 3).  Herein, rhodium(II)-catalyzed, proximity-driven modification identifies the STAT3 coiled-coil domain (CCD) as a novel ligand-binding site, and we describe a new naphthalene sulfonamide inhibitor that targets the CCD, blocks STAT3 function, and halts its disease-promoting effects in vitro, in tumor growth models, and in a leukemia mouse model, validating this new therapeutic target for resistant AML.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaEI9lh5LaebVg90H21EOLACvtfcHk0lhVxp2oGrZdDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVOmsr3F&md5=8d1c509437ac74ff8382ff7a43898de9</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fanie.201506889&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201506889%26sid%3Dliteratum%253Aachs%26aulast%3DMinus%26aufirst%3DM.%2BB.%26aulast%3DLiu%26aufirst%3DW.%26aulast%3DVohidov%26aufirst%3DF.%26aulast%3DKasembeli%26aufirst%3DM.%2BM.%26aulast%3DLong%26aufirst%3DX.%26aulast%3DKrueger%26aufirst%3DM.%2BJ.%26aulast%3DStevens%26aufirst%3DA.%26aulast%3DKolosov%26aufirst%3DM.%2BI.%26aulast%3DTweardy%26aufirst%3DD.%2BJ.%26aulast%3DSison%26aufirst%3DE.%2BA.%2BR.%26aulast%3DRedell%26aufirst%3DM.%2BS.%26aulast%3DBall%26aufirst%3DZ.%2BT.%26atitle%3DRhodium%2528II%2529%2520Proximity-Labeling%2520Identifies%2520a%2520Novel%2520Target%2520Site%2520on%2520STAT3%2520for%2520Inhibitors%2520with%2520Potent%2520Anti-Leukemia%2520Activity%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2015%26volume%3D54%26spage%3D13085%26epage%3D13089%26doi%3D10.1002%2Fanie.201506889" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosteric Modulators</span>. <i>Compr. Med. Chem. III</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">276</span>– <span class="NLM_lpage">296</span>, <span class="refDoi"> DOI: 10.1016/B978-0-12-409547-2.12324-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2FB978-0-12-409547-2.12324-8" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2017&pages=276-296&author=S.+Luauthor=J.+Zhang&title=Allosteric+Modulators&doi=10.1016%2FB978-0-12-409547-2.12324-8"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2FB978-0-12-409547-2.12324-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FB978-0-12-409547-2.12324-8%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosteric%2520Modulators%26jtitle%3DCompr.%2520Med.%2520Chem.%2520III%26date%3D2017%26volume%3D2%26spage%3D276%26epage%3D296%26doi%3D10.1016%2FB978-0-12-409547-2.12324-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">The Structural Basis of ATP as an Allosteric Modulator</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003831</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25211773" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslamu7vN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=S.+Luauthor=W.+Huangauthor=Q.+Wangauthor=Q.+Shenauthor=S.+Liauthor=R.+Nussinovauthor=J.+Zhang&title=The+Structural+Basis+of+ATP+as+an+Allosteric+Modulator&doi=10.1371%2Fjournal.pcbi.1003831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">The structural basis of ATP as an allosteric modulator</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Wang, Qi; Shen, Qiancheng; Li, Shuai; Nussinov, Ruth; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e1003831/1-e1003831/15, 15 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Adenosine-5'-triphosphate (ATP) is generally regarded as a substrate for energy currency and protein modification.  Recent findings uncovered the allosteric function of ATP in cellular signal transduction but little is understood about this crit. behavior of ATP.  Through extensive anal. of ATP in soln. and proteins, we found that the free ATP can exist in the compact and extended conformations in soln., and the two different conformational characteristics may be responsible for ATP to exert distinct biol. functions: ATP mols. adopt both compact and extended conformations in the allosteric binding sites but conserve extended conformations in the substrate binding sites.  Nudged elastic band simulations unveiled the distinct dynamic processes of ATP binding to the corresponding allosteric and substrate binding sites of uridine monophosphate kinase, and suggested that in soln. ATP preferentially binds to the substrate binding sites of proteins.  When the ATP mols. occupy the allosteric binding sites, the allosteric trigger from ATP to fuel allosteric communication between allosteric and functional sites is stemmed mainly from the triphosphate part of ATP, with a small no. from the adenine part of ATP.  Taken together, our results provide overall understanding of ATP allosteric functions responsible for regulation in biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5lsXXAkI3trVg90H21EOLACvtfcHk0lhVxp2oGrZdDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslamu7vN&md5=03e1c40183d194cb069cb55fe421982a</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003831%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DThe%2520Structural%2520Basis%2520of%2520ATP%2520as%2520an%2520Allosteric%2520Modulator%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Seo, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, G. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span> <span> </span><span class="NLM_article-title">Rational Engineering of Enzyme Allosteric Regulation through Sequence Evolution Analysis</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e1002612</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1002612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1002612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22807670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtVOnsLvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&author=J.-S.+Yangauthor=S.+W.+Seoauthor=S.+Jangauthor=G.+Y.+Jungauthor=S.+Kim&title=Rational+Engineering+of+Enzyme+Allosteric+Regulation+through+Sequence+Evolution+Analysis&doi=10.1371%2Fjournal.pcbi.1002612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Rational engineering of enzyme allosteric regulation through sequence evolution analysis</span></div><div class="casAuthors">Yang, Jae-Seong; Seo, Sang Woo; Jang, Sungho; Jung, Gyoo Yeol; Kim, Sanguk</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">e1002612</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Control of enzyme allosteric regulation is required to drive metabolic flux toward desired levels.  Although the three-dimensional (3D) structures of many enzyme-ligand complexes are available, it is still difficult to rationally engineer an allosterically regulatable enzyme without decreasing its catalytic activity.  Here, we describe an effective strategy to deregulate the allosteric inhibition of enzymes based on the mol. evolution and physicochem. characteristics of allosteric ligand-binding sites.  We found that allosteric sites are evolutionarily variable and comprised of more hydrophobic residues than catalytic sites.  We applied our findings to design mutations in selected target residues that deregulate the allosteric activity of fructose-1,6-bisphosphatase (FBPase).  Specifically, charged amino acids at less conserved positions were substituted with hydrophobic or neutral amino acids with similar sizes.  The engineered proteins successfully diminished the allosteric inhibition of E. coli FBPase without affecting its catalytic efficiency.  We expect that our method will aid the rational design of enzyme allosteric regulation strategies and facilitate the control of metabolic flux.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp8XyCGaD871LVg90H21EOLACvtfcHk0lhVxp2oGrZdDQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtVOnsLvI&md5=7f600c82cb9ee88a8905a0a4a0b7a19e</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1002612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1002612%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.-S.%26aulast%3DSeo%26aufirst%3DS.%2BW.%26aulast%3DJang%26aufirst%3DS.%26aulast%3DJung%26aufirst%3DG.%2BY.%26aulast%3DKim%26aufirst%3DS.%26atitle%3DRational%2520Engineering%2520of%2520Enzyme%2520Allosteric%2520Regulation%2520through%2520Sequence%2520Evolution%2520Analysis%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2012%26volume%3D8%26doi%3D10.1371%2Fjournal.pcbi.1002612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Recent Computational Advances in the Identification of Allosteric Sites in Proteins</span>. <i>Drug Discovery Today</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1595</span>– <span class="NLM_lpage">1600</span>, <span class="refDoi"> DOI: 10.1016/j.drudis.2014.07.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.drudis.2014.07.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25107670" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2014&pages=1595-1600&author=S.+Luauthor=W.+Huangauthor=J.+Zhang&title=Recent+Computational+Advances+in+the+Identification+of+Allosteric+Sites+in+Proteins&doi=10.1016%2Fj.drudis.2014.07.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">Recent computational advances in the identification of allosteric sites in proteins</span></div><div class="casAuthors">Lu, Shaoyong; Huang, Wenkang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1595-1600</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric modulators have the potential to fine-tune protein functional activity.  Therefore, the targeting of allosteric sites, as a strategy in drug design, is gaining increasing attention.  Currently, it is not trivial to find and characterize new allosteric sites by exptl. approaches.  Alternatively, computational approaches are useful in helping researchers analyze and select potential allosteric sites for drug discovery.  Here, we review state-of-the-art computational approaches directed at predicting putative allosteric sites in proteins, along with examples of successes in identifying allosteric sites utilizing these methods.  We also discuss the challenges in developing reliable methods for predicting allosteric sites and tactics to resolve demanding tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeaspE0Qbe47Vg90H21EOLACvtfcHk0lhE5S8DDem6tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCis77F&md5=fbaefacf51bee99d30fb882f0f04a47d</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.07.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.07.012%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DRecent%2520Computational%2520Advances%2520in%2520the%2520Identification%2520of%2520Allosteric%2520Sites%2520in%2520Proteins%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2014%26volume%3D19%26spage%3D1595%26epage%3D1600%26doi%3D10.1016%2Fj.drudis.2014.07.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schueler-Furman, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodak, S. J.</span></span> <span> </span><span class="NLM_article-title">Computational Approaches to Investigating Allostery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">159</span>– <span class="NLM_lpage">171</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2016.06.017</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2016.06.017" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27607077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7jF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2016&pages=159-171&author=O.+Schueler-Furmanauthor=S.+J.+Wodak&title=Computational+Approaches+to+Investigating+Allostery&doi=10.1016%2Fj.sbi.2016.06.017"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Computational approaches to investigating allostery</span></div><div class="casAuthors">Schueler-Furman, Ora; Wodak, Shoshana J.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">159-171</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Allosteric regulation plays a key role in many biol. processes, such as signal transduction, transcriptional regulation, and many more.  It is rooted in fundamental thermodn. and dynamic properties of macromol. systems that are still poorly understood and are moreover modulated by the cellular context.  Here we review the computational approaches used in the investigation of allosteric processes in protein systems.  We outline how the models of allostery have evolved from their initial formulation in the sixties to the current views, which more fully account for the roles of the thermodn. and dynamic properties of the system.  We then describe the major classes of computational approaches employed to elucidate the mechanisms of allostery, the insights they have provided, as well as their limitations.  We complement this anal. by highlighting the role of computational approaches in promising practical applications, such as the engineering of regulatory modules and identifying allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6oCnqZfK_x7Vg90H21EOLACvtfcHk0lhE5S8DDem6tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFWjs7jF&md5=7c28ee359b0701f757e7d9e042078522</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2016.06.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2016.06.017%26sid%3Dliteratum%253Aachs%26aulast%3DSchueler-Furman%26aufirst%3DO.%26aulast%3DWodak%26aufirst%3DS.%2BJ.%26atitle%3DComputational%2520Approaches%2520to%2520Investigating%2520Allostery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2016%26volume%3D41%26spage%3D159%26epage%3D171%26doi%3D10.1016%2Fj.sbi.2016.06.017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pendergrass, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blair, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fenton, A. W.</span></span> <span> </span><span class="NLM_article-title">Mining for Allosteric Information: Natural Mutations and Positional Sequence Conservation in Pyruvate Kinase</span>. <i>IUBMB Life</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">31</span>– <span class="NLM_lpage">38</span>, <span class="refDoi"> DOI: 10.1080/15216540500531705</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1080%2F15216540500531705" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16540430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28XisVaqtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2006&pages=31-38&author=D.+C.+Pendergrassauthor=R.+Williamsauthor=J.+B.+Blairauthor=A.+W.+Fenton&title=Mining+for+Allosteric+Information%3A+Natural+Mutations+and+Positional+Sequence+Conservation+in+Pyruvate+Kinase&doi=10.1080%2F15216540500531705"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Mining for allosteric information: Natural mutations and positional sequence conservation in pyruvate kinase</span></div><div class="casAuthors">Pendergrass, David; Williams, Rachel; Blair, James; Fenton, Aron</div><div class="citationInfo"><span class="NLM_cas:title">IUBMB Life</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-38</span>CODEN:
                <span class="NLM_cas:coden">IULIF8</span>;
        ISSN:<span class="NLM_cas:issn">1521-6543</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Although the amino acid sequences and the structures of pyruvate kinase (PYK) isoenzymes are highly conserved, allosteric regulations differ.  This suggests that amino acids with low conservation play important roles in the allosteric mechanism.  The current work exploits a natural screen, the 122 point mutations identified in the human gene encoding the erythrocyte PYK isoenzyme and assocd. with non-spherocytic hemolytic anemia, to learn what amino acid positions in PYK may be important for allosteric regulations.  In addn. to the mutations, we consider the conservation of each amino acid position across 241 PYK sequences.  Three groups of residue positions have been created, those with: (1) no disease causing mutation identified; (2) a disease causing mutation identified and high conservation across isoenzymes; and (3) a disease causing mutation identified and low conservation.  Mutations at positions not identified in the natural screen are likely to be tolerated with minimal loss of function.  Mutations at highly conserved positions are more likely to disrupt properties common to all PYK isoenzymes (e.g., structure, catalysis).  Residues in the third group are likely to be involved in roles that are necessary for function but not common to all isoenzymes (e.g., allostery).  Many of the Group 3 residues are located in the C-domain and to a lesser extent the A domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpY6rCK-AqkrLVg90H21EOLACvtfcHk0lhE5S8DDem6tQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XisVaqtbk%253D&md5=8a2d6ae344a59ee8282945ac72a53185</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1080%2F15216540500531705&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15216540500531705%26sid%3Dliteratum%253Aachs%26aulast%3DPendergrass%26aufirst%3DD.%2BC.%26aulast%3DWilliams%26aufirst%3DR.%26aulast%3DBlair%26aufirst%3DJ.%2BB.%26aulast%3DFenton%26aufirst%3DA.%2BW.%26atitle%3DMining%2520for%2520Allosteric%2520Information%253A%2520Natural%2520Mutations%2520and%2520Positional%2520Sequence%2520Conservation%2520in%2520Pyruvate%2520Kinase%26jtitle%3DIUBMB%2520Life%26date%3D2006%26volume%3D58%26spage%3D31%26epage%3D38%26doi%3D10.1080%2F15216540500531705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milligan, G.</span></span> <span> </span><span class="NLM_article-title">Allostery at G Protein-Coupled Receptor Homo- and Heteromers: Uncharted Pharmacological Landscapes</span>. <i>Pharmacol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">701</span>– <span class="NLM_lpage">725</span>, <span class="refDoi"> DOI: 10.1124/pr.110.002667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1124%2Fpr.110.002667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21079041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref92/cit92&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktlCi" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2010&pages=701-725&author=N.+J.+Smithauthor=G.+Milligan&title=Allostery+at+G+Protein-Coupled+Receptor+Homo-+and+Heteromers%3A+Uncharted+Pharmacological+Landscapes&doi=10.1124%2Fpr.110.002667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit92R"><div class="casContent"><span class="casTitleNuber">92</span><div class="casTitle"><span class="NLM_cas:atitle">Allostery at G protein-coupled receptor homo- and heteromers: uncharted pharmacological landscapes</span></div><div class="casAuthors">Smith, Nicola J.; Milligan, Graeme</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacological Reviews</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">701-725</span>CODEN:
                <span class="NLM_cas:coden">PAREAQ</span>;
        ISSN:<span class="NLM_cas:issn">0031-6997</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">A review.  For many years seven transmembrane domain G protein-coupled receptors (GPCRs) were thought to exist and function exclusively as monomeric units.  However, evidence both from native cells and heterologous expression systems has demonstrated that GPCRs can both traffic and signal within higher-order complexes.  As for other protein-protein interactions, conformational changes in one polypeptide, including those resulting from binding of pharmacol. ligands, have the capacity to alter the conformation and therefore the response of the interacting protein(s), a process known as allosterism.  For GPCRs, allosterism across homo- or heteromers, whether dimers or higher-order oligomers, represents an addnl. topog. landscape that must now be considered pharmacol.  Such effects may offer the opportunity for novel therapeutic approaches.  Allosterism at GPCR heteromers is particularly exciting in that it offers addnl. scope to provide receptor subtype selectivity and tissue specificity as well as fine-tuning of receptor signal strength.  Herein, we introduce the concept of allosterism at both GPCR homomers and heteromers and discuss the various questions that must be addressed before significant advances can be made in drug discovery at these GPCR complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2q5UArpDHnLVg90H21EOLACvtfcHk0lgEdH2IVet_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktlCi&md5=0fef78a27d0b81a586b4bb58e6374515</span></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=10.1124%2Fpr.110.002667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fpr.110.002667%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DN.%2BJ.%26aulast%3DMilligan%26aufirst%3DG.%26atitle%3DAllostery%2520at%2520G%2520Protein-Coupled%2520Receptor%2520Homo-%2520and%2520Heteromers%253A%2520Uncharted%2520Pharmacological%2520Landscapes%26jtitle%3DPharmacol.%2520Rev.%26date%3D2010%26volume%3D62%26spage%3D701%26epage%3D725%26doi%3D10.1124%2Fpr.110.002667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group"><span class="NLM_string-name">van
Westen, G. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaulton, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Overington, J. P.</span></span> <span> </span><span class="NLM_article-title">Chemical, Target, and Bioactive Properties of Allosteric Modulation</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>10</i></span>, <span class="NLM_elocation-id">e1003559</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1003559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1003559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24699297" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGlsL7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&author=G.+J.+P.+van%0AWestenauthor=A.+Gaultonauthor=J.+P.+Overington&title=Chemical%2C+Target%2C+and+Bioactive+Properties+of+Allosteric+Modulation&doi=10.1371%2Fjournal.pcbi.1003559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical, target, and bioactive properties of allosteric modulation</span></div><div class="casAuthors">van Westen, Gerard J. P.; Gaulton, Anna; Overington, John P.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003559/1-e1003559/17, 17 pp.</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Allosteric modulators are ligands for proteins that exert their effects via a different binding site than the natural (orthosteric) ligand site and hence form a conceptually distinct class of ligands for a target of interest.  Here, the physicochem. and structural features of a large set of allosteric and non-allosteric ligands from the ChEMBL database of bioactive mols. are analyzed.  In general allosteric modulators are relatively smaller, more lipophilic and more rigid compds., though large differences exist between different targets and target classes.  Furthermore, there are differences in the distribution of targets that bind these allosteric modulators.  Allosteric modulators are over-represented in membrane receptors, ligand-gated ion channels and nuclear receptor targets, but are underrepresented in enzymes (primarily proteases and kinases).  Moreover, allosteric modulators tend to bind to their targets with a slightly lower potency (5.96 log units vs. 6.66 log units, p<0.01).  However, this lower abs. affinity is compensated by their lower mol. wt. and more lipophilic nature, leading to similar binding efficiency and surface efficiency indexes.  Subsequently a series of classifier models are trained, initially target class independent models followed by finer-grained target (architecture/functional class) based models using the target hierarchy of the ChEMBL database.  Applications of these insights include the selection of likely allosteric modulators from existing compd. collections, the design of novel chem. libraries biased towards allosteric regulators and the selection of targets potentially likely to yield allosteric modulators on screening.  All data sets used in the paper are available for download.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrmlEn-nvE_H7Vg90H21EOLACvtfcHk0lgEdH2IVet_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGlsL7J&md5=bc2e7cca0c27b94f684224668f3d1f76</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1003559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1003559%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BWesten%26aufirst%3DG.%2BJ.%2BP.%26aulast%3DGaulton%26aufirst%3DA.%26aulast%3DOverington%26aufirst%3DJ.%2BP.%26atitle%3DChemical%252C%2520Target%252C%2520and%2520Bioactive%2520Properties%2520of%2520Allosteric%2520Modulation%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2014%26volume%3D10%26doi%3D10.1371%2Fjournal.pcbi.1003559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">A Glimpse of Structural Biology through X-Ray Crystallography</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>159</i></span>,  <span class="NLM_fpage">995</span>– <span class="NLM_lpage">1014</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.10.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2014.10.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25416941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref94/cit94&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFOgsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=159&publication_year=2014&pages=995-1014&author=Y.+Shi&title=A+Glimpse+of+Structural+Biology+through+X-Ray+Crystallography&doi=10.1016%2Fj.cell.2014.10.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit94R"><div class="casContent"><span class="casTitleNuber">94</span><div class="casTitle"><span class="NLM_cas:atitle">A glimpse of structural biology through x-ray crystallography</span></div><div class="casAuthors">Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">159</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">995-1014</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  Since the detn. of myoglobin structure in 1957, x-ray crystallog., as the anchoring tool of structural biol., has played an instrumental role in deciphering the secrets of life.  Knowledge gained through x-ray crystallog. has fundamentally advanced views on cellular processes and greatly facilitated the development of modern medicine.  Here, the author describes his personal understanding of the evolution of structural biol. through x-ray crystallog., using as examples mechanistic understanding of protein kinases and integral membrane proteins, and comments on the impact of technol. developments and the outlook for x-ray crystallog.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrISBaUSogzUrVg90H21EOLACvtfcHk0lgEdH2IVet_ew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFOgsrjL&md5=7c2837928184f9071ee618593fd7ceeb</span></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.10.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.10.051%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DY.%26atitle%3DA%2520Glimpse%2520of%2520Structural%2520Biology%2520through%2520X-Ray%2520Crystallography%26jtitle%3DCell%26date%3D2014%26volume%3D159%26spage%3D995%26epage%3D1014%26doi%3D10.1016%2Fj.cell.2014.10.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug–Target Interactions</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>62</i></span>,  <span class="NLM_fpage">24</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01844</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01844" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2019&pages=24-45&author=S.+Luauthor=J.+Zhang&title=Small+Molecule+Allosteric+Modulators+of+G-Protein-Coupled+Receptors%3A+Drug%E2%80%93Target+Interactions&doi=10.1021%2Facs.jmedchem.7b01844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Allosteric Modulators of G-Protein-Coupled Receptors: Drug-Target Interactions</span></div><div class="casAuthors">Lu, Shaoyong; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">62</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">24-45</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G-protein-coupled receptors (GPCRs) are the largest class of signaling receptors that are most frequently targeted by therapeutic drugs.  Allosteric modulators bound to GPCRs at allosteric sites provide the potential for differential selectivity and improved safety compared with traditional orthosteric ligands.  The recent breakthroughs in GPCR structural biol. have made structures of GPCRs from classes A, B, C, and F complexed with small-mol. allosteric modulators available.  Knowledge of the detailed receptor-modulator interactions at the allosteric sites is useful for structure-based GPCR drug design of novel therapeutics.  This Perspective comprehensively summarizes the current status of structural complexes between GPCRs and their small-mol. allosteric modulators, particularly the key receptor-modulator interactions at the allosteric sites.  Then, the structural diversity of allosteric sites across four GPCR subfamilies is compared.  This study is expected to contribute to the design of GPCR allosteric drugs with an improved therapeutic action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVDf1DtcU6ZLVg90H21EOLACvtfcHk0liFwkG8uisS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXivFCgtb0%253D&md5=7a41f4c00dadecd0661d135811644ad8</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01844%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DSmall%2520Molecule%2520Allosteric%2520Modulators%2520of%2520G-Protein-Coupled%2520Receptors%253A%2520Drug%25E2%2580%2593Target%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2019%26volume%3D62%26spage%3D24%26epage%3D45%26doi%3D10.1021%2Facs.jmedchem.7b01844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beck, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Unverdorben, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohn, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schweitzer, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pfeifer, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sakata, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nickell, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plitzko, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villa, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baumeister, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forster, F.</span></span> <span> </span><span class="NLM_article-title">Near-Atomic Resolution Structural Model of the Yeast 26S Proteasome</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">14870</span>– <span class="NLM_lpage">14875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1213333109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1213333109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22927375" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVelsrjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=14870-14875&author=F.+Beckauthor=P.+Unverdorbenauthor=S.+Bohnauthor=A.+Schweitzerauthor=G.+Pfeiferauthor=E.+Sakataauthor=S.+Nickellauthor=J.+M.+Plitzkoauthor=E.+Villaauthor=W.+Baumeisterauthor=F.+Forster&title=Near-Atomic+Resolution+Structural+Model+of+the+Yeast+26S+Proteasome&doi=10.1073%2Fpnas.1213333109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Near-atomic resolution structural model of the yeast 26S proteasome</span></div><div class="casAuthors">Beck, Florian; Unverdorben, Pia; Bohn, Stefan; Schweitzer, Andreas; Pfeifer, Guenter; Sakata, Eri; Nickell, Stephan; Plitzko, Jurgen M.; Villa, Elizabeth; Baumeister, Wolfgang; Foerster, Friedrich</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">14870-14875, S14870/1-S14870/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The 26S proteasome operates at the executive end of the ubiquitin-proteasome pathway.  Here, we present a cryo-EM structure of the Saccharomyces cerevisiae 26S proteasome at a resoln. of 7.4 Å or 6.7 Å (Fourier-Shell Correlation of 0.5 or 0.3, resp.).  We used this map in conjunction with mol. dynamics-based flexible fitting to build a near-at. resoln. model of the holocomplex.  The quality of the map allowed us to assign α-helixes, the predominant secondary structure element of the regulatory particle subunits, throughout the entire map.  We were able to det. the architecture of the Rpn8/Rpn11 heterodimer, which had hitherto remained elusive.  The MPN domain of Rpn11 is positioned directly above the AAA-ATPase N-ring suggesting that Rpn11 deubiquitylates substrates immediately following commitment and prior to their unfolding by the AAA-ATPase module.  The MPN domain of Rpn11 dimerizes with that of Rpn8 and the C-termini of both subunits form long helixes, which are integral parts of a coiled-coil module.  Together with the C-terminal helixes of the 6 PCI-domain subunits they form a very large coiled-coil bundle, which appears to serve as a flexible anchoring device for all the lid subunits.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqWCbYbuDRJULVg90H21EOLACvtfcHk0liFwkG8uisS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVelsrjL&md5=3b359d054944bb4dc14992bc72e5fb20</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1213333109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1213333109%26sid%3Dliteratum%253Aachs%26aulast%3DBeck%26aufirst%3DF.%26aulast%3DUnverdorben%26aufirst%3DP.%26aulast%3DBohn%26aufirst%3DS.%26aulast%3DSchweitzer%26aufirst%3DA.%26aulast%3DPfeifer%26aufirst%3DG.%26aulast%3DSakata%26aufirst%3DE.%26aulast%3DNickell%26aufirst%3DS.%26aulast%3DPlitzko%26aufirst%3DJ.%2BM.%26aulast%3DVilla%26aufirst%3DE.%26aulast%3DBaumeister%26aufirst%3DW.%26aulast%3DForster%26aufirst%3DF.%26atitle%3DNear-Atomic%2520Resolution%2520Structural%2520Model%2520of%2520the%2520Yeast%252026S%2520Proteasome%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D14870%26epage%3D14875%26doi%3D10.1073%2Fpnas.1213333109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oldham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davidson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span> <span> </span><span class="NLM_article-title">Carbon Catabolite Repression of the Maltose Transporter Revealed by X-Ray Crystallography</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>499</i></span>,  <span class="NLM_fpage">364</span>– <span class="NLM_lpage">368</span>, <span class="refDoi"> DOI: 10.1038/nature12232</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12232" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23770568" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpsV2it70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=499&publication_year=2013&pages=364-368&author=S.+Chenauthor=M.+L.+Oldhamauthor=A.+L.+Davidsonauthor=J.+Chen&title=Carbon+Catabolite+Repression+of+the+Maltose+Transporter+Revealed+by+X-Ray+Crystallography&doi=10.1038%2Fnature12232"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Carbon catabolite repression of the maltose transporter revealed by X-ray crystallography</span></div><div class="casAuthors">Chen, Shanshuang; Oldham, Michael L.; Davidson, Amy L.; Chen, Jue</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">499</span>
        (<span class="NLM_cas:issue">7458</span>),
    <span class="NLM_cas:pages">364-368</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Efficient carbon utilization is crit. to the survival of microorganisms in competitive environments.  To optimize energy usage, bacteria have developed an integrated control system to preferentially uptake carbohydrates that support rapid growth.  The availability of a preferred carbon source, such as glucose, represses the synthesis and activities of proteins necessary for the transport and metab. of secondary carbon sources.  This regulatory phenomenon is defined as carbon catabolite repression.  In enteric bacteria, the key player of carbon catabolite repression is a component of the glucose-specific phosphotransferase system, enzyme IIA (EIIAGlc).  It is known that unphosphorylated EIIAGlc binds to and inhibits a variety of transporters when glucose is available.  However, understanding the underlying mol. mechanism has been hindered by the complete absence of structures for any EIIAGlc-transporter complexes.  Here we present the 3.9Å crystal structure of Escherichia coli EIIAGlc in complex with the maltose transporter, an ATP-binding cassette (ABC) transporter.  The structure shows that two EIIAGlc mols. bind to the cytoplasmic ATPase subunits, stabilizing the transporter in an inward-facing conformation and preventing the structural rearrangements necessary for ATP hydrolysis.  We also show that the half-maximal inhibitory concns. of the full-length EIIAGlc and an amino-terminal truncation mutant differ by 60-fold, consistent with the hypothesis that the amino-terminal region, disordered in the crystal structure, functions as a membrane anchor to increase the effective EIIAGlc concn. at the membrane.  Together these data suggest a model of how the central regulatory protein EIIAGlc allosterically inhibits maltose uptake in E. coli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9xchOtuWIu7Vg90H21EOLACvtfcHk0liFwkG8uisS-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpsV2it70%253D&md5=e0d4089ce27f3879e7454c0a3060b00a</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1038%2Fnature12232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12232%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DOldham%26aufirst%3DM.%2BL.%26aulast%3DDavidson%26aufirst%3DA.%2BL.%26aulast%3DChen%26aufirst%3DJ.%26atitle%3DCarbon%2520Catabolite%2520Repression%2520of%2520the%2520Maltose%2520Transporter%2520Revealed%2520by%2520X-Ray%2520Crystallography%26jtitle%3DNature%26date%3D2013%26volume%3D499%26spage%3D364%26epage%3D368%26doi%3D10.1038%2Fnature12232" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures of the Lsm Complex Bound to the 3′ End Sequence of U6 Small Nuclear RNA</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>506</i></span>,  <span class="NLM_fpage">116</span>– <span class="NLM_lpage">120</span>, <span class="refDoi"> DOI: 10.1038/nature12803</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12803" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24240276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref98/cit98&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsl2msb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=506&publication_year=2014&pages=116-120&author=L.+Zhouauthor=J.+Hangauthor=Y.+Zhouauthor=R.+Wanauthor=G.+Luauthor=P.+Yinauthor=C.+Yanauthor=Y.+Shi&title=Crystal+Structures+of+the+Lsm+Complex+Bound+to+the+3%E2%80%B2+End+Sequence+of+U6+Small+Nuclear+RNA&doi=10.1038%2Fnature12803"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit98R"><div class="casContent"><span class="casTitleNuber">98</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures of the Lsm complex bound to the 3' end sequence of U6 small nuclear RNA</span></div><div class="casAuthors">Zhou, Lijun; Hang, Jing; Zhou, Yulin; Wan, Ruixue; Lu, Guifeng; Yin, Ping; Yan, Chuangye; Shi, Yigong</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">506</span>
        (<span class="NLM_cas:issue">7486</span>),
    <span class="NLM_cas:pages">116-120</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Splicing of precursor mRNA (pre-mRNA) in eukaryotic cells is carried out by the spliceosome, which consists of five small nuclear ribonucleoproteins (snRNPs) and a no. of accessory factors and enzymes.  Each snRNP contains a ring-shaped subcomplex of seven proteins and a specific RNA mol.  The U6 snRNP contains a unique heptameric Lsm protein complex, which specifically recognizes the U6 small nuclear RNA at its 3' end.  Here we report the crystal structures of the heptameric Lsm complex, both by itself and in complex with a 3' fragment of U6 snRNA, at 2.8 Å resoln.  Each of the seven Lsm proteins interacts with two neighboring Lsm components to form a doughnut-shaped assembly, with the order Lsm3-2-8-4-7-5-6.  The four uridine nucleotides at the 3' end of U6 snRNA are modularly recognized by Lsm3, Lsm2, Lsm8 and Lsm4, with the uracil base specificity conferred by a highly conserved asparagine residue.  The uracil base at the extreme 3' end is sandwiched by His 36 and Arg 69 from Lsm3, through π-π and cation-π interactions, resp.  The distinctive end-recognition of U6 snRNA by the Lsm complex contrasts with RNA binding by the Sm complex in the other snRNPs.  The structural features and assocd. biochem. analyses deepen mechanistic understanding of the U6 snRNP function in pre-mRNA splicing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5TH6BTgbiBLVg90H21EOLACvtfcHk0lhzb3-w1-lw4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsl2msb4%253D&md5=5115e59fc9d93d3b6f8ecf6804fd7ca6</span></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=10.1038%2Fnature12803&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12803%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DHang%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWan%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DG.%26aulast%3DYin%26aufirst%3DP.%26aulast%3DYan%26aufirst%3DC.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DCrystal%2520Structures%2520of%2520the%2520Lsm%2520Complex%2520Bound%2520to%2520the%25203%25E2%2580%25B2%2520End%2520Sequence%2520of%2520U6%2520Small%2520Nuclear%2520RNA%26jtitle%3DNature%26date%3D2014%26volume%3D506%26spage%3D116%26epage%3D120%26doi%3D10.1038%2Fnature12803" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stass, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilca, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huiskonen, J. T.</span></span> <span> </span><span class="NLM_article-title">Beyond Structures of Highly Symmetric Purified Viral Capsids by Cryo-EM</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">25</span>– <span class="NLM_lpage">31</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.07.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2018.07.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30096461" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsVers77F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=25-31&author=R.+Stassauthor=S.+L.+Ilcaauthor=J.+T.+Huiskonen&title=Beyond+Structures+of+Highly+Symmetric+Purified+Viral+Capsids+by+Cryo-EM&doi=10.1016%2Fj.sbi.2018.07.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit99R"><div class="casContent"><span class="casTitleNuber">99</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond structures of highly symmetric purified viral capsids by cryo-EM</span></div><div class="casAuthors">Stass, Robert; Ilca, Serban L.; Huiskonen, Juha T.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">25-31</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cryogenic transmission electron microscopy (cryo-EM) is widely used to det. high-resoln. structures of sym. virus capsids.  The method holds promise for extending studies beyond purified capsids and their sym. protein shells.  The non-sym. genome component has been addressed in dsRNA cypoviruses and ssRNA bacteriophages Qβ and MS2.  The structure of human herpes simplex virus type 1 capsids has been detd. within intact virions to resolve capsid-tegument interactions.  Electron tomog. under cryogenic conditions (cryo-ET), has allowed resolving an early membrane fusion intermediate of Rift Valley fever virus.  Antibody-affinity based sample grids allow capturing of virions directly from cell cultures or even clin. samples.  These and other emerging methods will support studies to address viral entry, assembly and neutralization processes at increasingly high resolns. and native conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVnUKUCNhkCbVg90H21EOLACvtfcHk0lhzb3-w1-lw4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsVers77F&md5=f63b9c7ce6fb2ea47cc902a61a0226b7</span></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.07.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.07.011%26sid%3Dliteratum%253Aachs%26aulast%3DStass%26aufirst%3DR.%26aulast%3DIlca%26aufirst%3DS.%2BL.%26aulast%3DHuiskonen%26aufirst%3DJ.%2BT.%26atitle%3DBeyond%2520Structures%2520of%2520Highly%2520Symmetric%2520Purified%2520Viral%2520Capsids%2520by%2520Cryo-EM%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D25%26epage%3D31%26doi%3D10.1016%2Fj.sbi.2018.07.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pospich, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raunser, S.</span></span> <span> </span><span class="NLM_article-title">Single Particle Cryo-EM — an Optimal Tool to Study Cytoskeletal Proteins</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>52</i></span>,  <span class="NLM_fpage">16</span>– <span class="NLM_lpage">24</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2018.07.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2018.07.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30056307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlKmtLvF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2018&pages=16-24&author=S.+Pospichauthor=S.+Raunser&title=Single+Particle+Cryo-EM+%E2%80%94+an+Optimal+Tool+to+Study+Cytoskeletal+Proteins&doi=10.1016%2Fj.sbi.2018.07.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit100R"><div class="casContent"><span class="casTitleNuber">100</span><div class="casTitle"><span class="NLM_cas:atitle">Single particle cryo-EM - an optimal tool to study cytoskeletal proteins</span></div><div class="casAuthors">Pospich, Sabrina; Raunser, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">16-24</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Cytoskeletal proteins play essential roles in many cellular processes.  Knowledge of their structures is important to understand their function and regulation.  Since cytoskeletal polymers are difficult to crystallize, cryo-EM has been the predominant method of choice to study their structures.  Recent advances in the methodol. have enabled reconstructions at near-at. resoln.  In this review, we focus on novel insights gained from high-resoln. cryo-EM structures of cytoskeletal polymers.  These include eukaryotic proteins such as F-actin and microtubules as well as their prokaryotic homologues.  The unprecedented high-resolns. allow identifying small mols., including nucleotides and drugs, as well as subtle changes at interfaces that are key to complex processes, such as nucleotide hydrolysis in microtubules and actin filaments.  While major methodol. advances have already promoted the structural anal. of cytoskeletal polymers, there are still specific methodol. challenges to overcome and many scientific questions remain to be answered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqCyCym1d9lO7Vg90H21EOLACvtfcHk0lhzb3-w1-lw4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlKmtLvF&md5=e2738a2df6e5cdfcba5215b61fb786e7</span></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2018.07.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2018.07.006%26sid%3Dliteratum%253Aachs%26aulast%3DPospich%26aufirst%3DS.%26aulast%3DRaunser%26aufirst%3DS.%26atitle%3DSingle%2520Particle%2520Cryo-EM%2520%25E2%2580%2594%2520an%2520Optimal%2520Tool%2520to%2520Study%2520Cytoskeletal%2520Proteins%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D52%26spage%3D16%26epage%3D24%26doi%3D10.1016%2Fj.sbi.2018.07.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fica, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagai, K.</span></span> <span> </span><span class="NLM_article-title">Cryo-Electron Microscopy Snapshots of the Spliceosome: Structural Insights into a Dynamic Ribonucleoprotein Machine</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">791</span>– <span class="NLM_lpage">799</span>, <span class="refDoi"> DOI: 10.1038/nsmb.3463</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnsmb.3463" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28981077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2017&pages=791-799&author=S.+M.+Ficaauthor=K.+Nagai&title=Cryo-Electron+Microscopy+Snapshots+of+the+Spliceosome%3A+Structural+Insights+into+a+Dynamic+Ribonucleoprotein+Machine&doi=10.1038%2Fnsmb.3463"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-electron microscopy snapshots of the spliceosome: structural insights into a dynamic ribonucleoprotein machine</span></div><div class="casAuthors">Fica, Sebastian M.; Nagai, Kiyoshi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">791-799</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">The spliceosome excises introns from pre-mRNAs using an RNA-based active site that is cradled by a dynamic protein scaffold.  A recent revolution in cryo-electron microscopy (cryo-EM) has led to near-at.-resoln. structures of key spliceosome complexes that provide insight into the mechanism of activation, splice site positioning, catalysis, protein rearrangements and ATPase-mediated dynamics of the active site.  The cryo-EM structures rationalize decades of observations from genetic and biochem. studies and provide a mol. framework for future functional studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphCYa1XIdT5LVg90H21EOLACvtfcHk0lhzb3-w1-lw4g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1Sqs7jO&md5=8ba1a0da86dd368aedfc0336dadcfae9</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.3463&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.3463%26sid%3Dliteratum%253Aachs%26aulast%3DFica%26aufirst%3DS.%2BM.%26aulast%3DNagai%26aufirst%3DK.%26atitle%3DCryo-Electron%2520Microscopy%2520Snapshots%2520of%2520the%2520Spliceosome%253A%2520Structural%2520Insights%2520into%2520a%2520Dynamic%2520Ribonucleoprotein%2520Machine%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D24%26spage%3D791%26epage%3D799%26doi%3D10.1038%2Fnsmb.3463" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Renaud, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciferri, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rémigy, H. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stark, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wiesmann, C.</span></span> <span> </span><span class="NLM_article-title">Cryo-EM in Drug Discovery: Achievements, Limitations and Prospects</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">471</span>– <span class="NLM_lpage">492</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnrd.2018.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29880918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtVyqurnJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=471-492&author=J.+P.+Renaudauthor=A.+Chariauthor=C.+Ciferriauthor=W.+T.+Liuauthor=H.+W.+R%C3%A9migyauthor=H.+Starkauthor=C.+Wiesmann&title=Cryo-EM+in+Drug+Discovery%3A+Achievements%2C+Limitations+and+Prospects&doi=10.1038%2Fnrd.2018.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit102R"><div class="casContent"><span class="casTitleNuber">102</span><div class="casTitle"><span class="NLM_cas:atitle">Cryo-EM in drug discovery: achievements, limitations and prospects</span></div><div class="casAuthors">Renaud, Jean-Paul; Chari, Ashwin; Ciferri, Claudio; Liu, Wen-ti; Remigy, Herve-William; Stark, Holger; Wiesmann, Christian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">471-492</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Cryo-electron microscopy (cryo-EM) of non-cryst. single particles is a biophys. technique that can be used to det. the structure of biol. macromols. and assemblies.  Historically, its potential for application in drug discovery has been heavily limited by two issues: the min. size of the structures it can be used to study and the resoln. of the images.  However, recent technol. advances - including the development of direct electron detectors and more effective computational image anal. techniques - are revolutionizing the utility of cryo-EM, leading to a burst of high-resoln. structures of large macromol. assemblies.  These advances have raised hopes that single-particle cryo-EM might soon become an important tool for drug discovery, particularly if they could enable structural detn. for 'intractable' targets that are still not accessible to X-ray crystallog. anal.  This article describes the recent advances in the field and critically assesses their relevance for drug discovery as well as discussing at what stages of the drug discovery pipeline cryo-EM can be useful today and what to expect in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfGKiqaXhRQ7Vg90H21EOLACvtfcHk0ljif0RdzVnISA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtVyqurnJ&md5=131daaa12f4c51c1e6478b55eb709157</span></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.77%26sid%3Dliteratum%253Aachs%26aulast%3DRenaud%26aufirst%3DJ.%2BP.%26aulast%3DChari%26aufirst%3DA.%26aulast%3DCiferri%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DW.%2BT.%26aulast%3DR%25C3%25A9migy%26aufirst%3DH.%2BW.%26aulast%3DStark%26aufirst%3DH.%26aulast%3DWiesmann%26aufirst%3DC.%26atitle%3DCryo-EM%2520in%2520Drug%2520Discovery%253A%2520Achievements%252C%2520Limitations%2520and%2520Prospects%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D471%26epage%3D492%26doi%3D10.1038%2Fnrd.2018.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grutsch, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brüschweiler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tollinger, M.</span></span> <span> </span><span class="NLM_article-title">NMR Methods to Study Dynamic Allostery</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">e1004620</span>, <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004620</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004620" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26964042" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1WjurrP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=e1004620&author=S.+Grutschauthor=S.+Br%C3%BCschweilerauthor=M.+Tollinger&title=NMR+Methods+to+Study+Dynamic+Allostery&doi=10.1371%2Fjournal.pcbi.1004620"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">NMR methods to study dynamic allostery</span></div><div class="casAuthors">Grutsch, Sarina; Brueschweiler, Sven; Tollinger, Martin</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">e1004620/1-e1004620/20</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">NMR (NMR) spectroscopy provides a unique toolbox of exptl. probes for studying dynamic processes on a wide range of timescales, ranging from picoseconds to milliseconds and beyond.  Along with NMR hardware developments, recent methodol. advancements have enabled the characterization of allosteric proteins at unprecedented detail, revealing intriguing aspects of allosteric mechanisms and increasing the proportion of the conformational ensemble that can be obsd. by expt.  Here, we present an overview of NMR spectroscopic methods for characterizing equil. fluctuations in free and bound states of allosteric proteins that have been most influential in the field.  By combining NMR exptl. approaches with mol. simulations, atomistic level descriptions of the mechanisms by which allosteric phenomena take place are now within reach.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqtDC4ZmjidbVg90H21EOLACvtfcHk0ljif0RdzVnISA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1WjurrP&md5=3c3bb15547707024281794fe0fcd1928</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004620&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004620%26sid%3Dliteratum%253Aachs%26aulast%3DGrutsch%26aufirst%3DS.%26aulast%3DBr%25C3%25BCschweiler%26aufirst%3DS.%26aulast%3DTollinger%26aufirst%3DM.%26atitle%3DNMR%2520Methods%2520to%2520Study%2520Dynamic%2520Allostery%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3De1004620%26doi%3D10.1371%2Fjournal.pcbi.1004620" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tzeng, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">Protein Dynamics and Allostery: An NMR View</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">62</span>– <span class="NLM_lpage">67</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2010.10.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2010.10.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21109422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhvFKhuro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=62-67&author=S.+R.+Tzengauthor=C.+G.+Kalodimos&title=Protein+Dynamics+and+Allostery%3A+An+NMR+View&doi=10.1016%2Fj.sbi.2010.10.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Protein dynamics and allostery: An NMR view</span></div><div class="casAuthors">Tzeng, Shiou-Ru; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">62-67</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allostery, the process by which distant sites within a protein system are energetically coupled, is an efficient and ubiquitous mechanism for activity regulation.  A purely mech. view of allostery invoking only structural changes has developed over the decades as the classical view of the phenomenon.  However, a fast growing list of examples illustrate the intimate link between internal motions over a wide range of time scales and function in protein-ligand interactions.  Proteins respond to perturbations by redistributing their motions and they use fluctuating conformational states for binding and conformational entropy as a carrier of allosteric energy to modulate assocn. with ligands.  In several cases allosteric interactions proceed with minimal or no structural changes.  We discuss emerging paradigms for the central role of protein dynamics in allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5G7kpkHz2M7Vg90H21EOLACvtfcHk0ljif0RdzVnISA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhvFKhuro%253D&md5=7ee2b2686e0c6a8ca4d337a33fd322ce</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2010.10.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2010.10.007%26sid%3Dliteratum%253Aachs%26aulast%3DTzeng%26aufirst%3DS.%2BR.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DProtein%2520Dynamics%2520and%2520Allostery%253A%2520An%2520NMR%2520View%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D62%26epage%3D67%26doi%3D10.1016%2Fj.sbi.2010.10.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boulton, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Melacini, G.</span></span> <span> </span><span class="NLM_article-title">Advances in NMR Methods to Map Allosteric Sites: From Models to Translation</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6267</span>– <span class="NLM_lpage">6304</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00718</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00718" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref105/cit105&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsFCrsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6267-6304&author=S.+Boultonauthor=G.+Melacini&title=Advances+in+NMR+Methods+to+Map+Allosteric+Sites%3A+From+Models+to+Translation&doi=10.1021%2Facs.chemrev.5b00718"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit105R"><div class="casContent"><span class="casTitleNuber">105</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in NMR Methods To Map Allosteric Sites: From Models to Translation</span></div><div class="casAuthors">Boulton, Stephen; Melacini, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6267-6304</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The last five years have witnessed major developments in the understanding of the allosteric phenomenon, broadly defined as coupling between remote mol. sites.  Such advances have been driven not only by new theor. models and pharmacol. applications of allostery, but also by progress in the exptl. approaches designed to map allosteric sites and transitions.  Among these techniques, NMR spectroscopy has played a major role given its unique near-at. resoln. and sensitivity to the dynamics that underlie allosteric couplings.  Here, we highlight recent progress in the NMR methods tailored to investigate allostery with the goal of offering an overview of which NMR approaches are best suited for which allosterically relevant questions.  The picture of the allosteric "NMR toolbox" is provided starting from one of the simplest models of allostery (i.e., the four-state thermodn. cycle) and continuing to more complex multistate mechanisms.  We also review how such an "NMR toolbox" has assisted the elucidation of the allosteric mol. basis for disease-related mutations and the discovery of novel leads for allosteric drugs.  From this overview, it is clear that NMR plays a central role not only in exptl. validating transformative theories of allostery, but also in tapping the full translational potential of allosteric systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqS33N8DJcfE7Vg90H21EOLACvtfcHk0lgl47vhxAQnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsFCrsr0%253D&md5=02fb528cd9cc3ec3ce82a6928f9170f5</span></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00718&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00718%26sid%3Dliteratum%253Aachs%26aulast%3DBoulton%26aufirst%3DS.%26aulast%3DMelacini%26aufirst%3DG.%26atitle%3DAdvances%2520in%2520NMR%2520Methods%2520to%2520Map%2520Allosteric%2520Sites%253A%2520From%2520Models%2520to%2520Translation%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6267%26epage%3D6304%26doi%3D10.1021%2Facs.chemrev.5b00718" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lisi, G. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loria, J. P.</span></span> <span> </span><span class="NLM_article-title">Solution NMR Spectroscopy for the Study of Enzyme Allostery</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6323</span>– <span class="NLM_lpage">6369</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00541</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00541" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVGqsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6323-6369&author=G.+P.+Lisiauthor=J.+P.+Loria&title=Solution+NMR+Spectroscopy+for+the+Study+of+Enzyme+Allostery&doi=10.1021%2Facs.chemrev.5b00541"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">Solution NMR Spectroscopy for the Study of Enzyme Allostery</span></div><div class="casAuthors">Lisi, George P.; Loria, J. Patrick</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6323-6369</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a ubiquitous biol. regulatory process in which distant binding sites within a protein or enzyme are functionally and thermodynamically coupled.  Allosteric interactions play essential roles in many enzymol. mechanisms, often facilitating formation of enzyme-substrate complexes and/or product release.  Thus, elucidating the forces that drive allostery is crit. to understanding the complex transformations of biomols.  Currently, a no. of models exist to describe allosteric behavior, taking into account energetics as well as conformational rearrangements and fluctuations.  In the following Review, we discuss the use of soln. NMR techniques designed to probe allosteric mechanisms in enzymes.  NMR spectroscopy is unequaled in its ability to detect structural and dynamical changes in biomols., and the case studies presented herein demonstrate the range of insights to be gained from this valuable method.  We also provide a detailed tech. discussion of several specialized NMR expts. that are ideally suited for the study of enzymic allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpgJUg3gOrsHrVg90H21EOLACvtfcHk0lgl47vhxAQnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVGqsrs%253D&md5=db1fd116e1222158a8e63e5246812680</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00541&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00541%26sid%3Dliteratum%253Aachs%26aulast%3DLisi%26aufirst%3DG.%2BP.%26aulast%3DLoria%26aufirst%3DJ.%2BP.%26atitle%3DSolution%2520NMR%2520Spectroscopy%2520for%2520the%2520Study%2520of%2520Enzyme%2520Allostery%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6323%26epage%3D6369%26doi%3D10.1021%2Facs.chemrev.5b00541" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kalodimos, C. G.</span></span> <span> </span><span class="NLM_article-title">NMR Studies of Large Proteins</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>429</i></span>,  <span class="NLM_fpage">2667</span>– <span class="NLM_lpage">2676</span>, <span class="refDoi"> DOI: 10.1016/j.jmb.2017.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.jmb.2017.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28728982" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=429&publication_year=2017&pages=2667-2676&author=Y.+Jiangauthor=C.+G.+Kalodimos&title=NMR+Studies+of+Large+Proteins&doi=10.1016%2Fj.jmb.2017.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">NMR Studies of Large Proteins</span></div><div class="casAuthors">Jiang, Yajun; Kalodimos, Charalampos G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">429</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2667-2676</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Recent breakthroughs in isotope-labeling and pulse sequence techniques have enabled the NMR characterization of large protein systems with mol. masses of hundreds of kDa.  NMR studies of a great variety of large proteins provided unique insights into the binding, dynamic, and allosteric mechanisms.  Here the authors present a brief summary of these developments by highlighting a few cases that exemplify the uniqueness of NMR in providing at. resoln. information into key dynamic processes and structures of protein complexes with high degree of flexibility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJF6WXjHLqtLVg90H21EOLACvtfcHk0lgl47vhxAQnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtF2qurfN&md5=7aa8e2e6f171902030910b0d54750ec4</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1016%2Fj.jmb.2017.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmb.2017.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DKalodimos%26aufirst%3DC.%2BG.%26atitle%3DNMR%2520Studies%2520of%2520Large%2520Proteins%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2017%26volume%3D429%26spage%3D2667%26epage%3D2676%26doi%3D10.1016%2Fj.jmb.2017.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Velyvis, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kay, L. E.</span></span> <span> </span><span class="NLM_article-title">Measurement of Active Site Ionization Equilibria in the 670 kDa Proteasome Core Particle Using Methyl-TROSY NMR</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">9259</span>– <span class="NLM_lpage">9262</span>, <span class="refDoi"> DOI: 10.1021/ja403091c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja403091c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmvFKqurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=9259-9262&author=A.+Velyvisauthor=L.+E.+Kay&title=Measurement+of+Active+Site+Ionization+Equilibria+in+the+670+kDa+Proteasome+Core+Particle+Using+Methyl-TROSY+NMR&doi=10.1021%2Fja403091c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit108R"><div class="casContent"><span class="casTitleNuber">108</span><div class="casTitle"><span class="NLM_cas:atitle">Measurement of active site ionization equilibria in the 670 kDa proteasome core particle using methyl-TROSY NMR</span></div><div class="casAuthors">Velyvis, Algirdas; Kay, Lewis E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">9259-9262</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The 20S proteasome core particle is a mol. machine that plays a central role in the regulation of cellular function through proteolysis, and it has emerged as a valuable drug target for certain classes of cancers.  Central to the development of new and potent pharmaceuticals is an understanding of the mechanism by which the proteasome cleaves substrates.  A no. of high-resoln. structures of the 20S proteasome with and without inhibitors have emerged that provide insight into the chem. of peptide bond cleavage and establish the role of Thr-1 Oγ-1 atom as the catalytic nucleophile.  The source of the base that accepts the Thr-1 Hγ-1 atom is less clear.  Here, using a highly deuterated sample of the proteasome labeled with 13CH3 at the Thr-γ positions, the pKa of the Thr-1 amino group was measured as 6.3 and hence deprotonated in the range of maximal enzyme activity.  This provided strong evidence that the terminal amino group of Thr-1 serves as the base in the 1st step of the peptide bond cleavage reaction.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqj_d1-nMFHEbVg90H21EOLACvtfcHk0lgl47vhxAQnlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmvFKqurc%253D&md5=31f90f50707d0ed55da43a677083af52</span></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1021%2Fja403091c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja403091c%26sid%3Dliteratum%253Aachs%26aulast%3DVelyvis%26aufirst%3DA.%26aulast%3DKay%26aufirst%3DL.%2BE.%26atitle%3DMeasurement%2520of%2520Active%2520Site%2520Ionization%2520Equilibria%2520in%2520the%2520670%2520kDa%2520Proteasome%2520Core%2520Particle%2520Using%2520Methyl-TROSY%2520NMR%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D9259%26epage%3D9262%26doi%3D10.1021%2Fja403091c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karagöz, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duarte, A. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akoury, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ippel, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Biernat, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morán Luengo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Radli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Didenko, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nordhues, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veprintsev, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dickey, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mandelkow, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zweckstetter, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Boelens, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madl, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rüdiger, S. G. D.</span></span> <span> </span><span class="NLM_article-title">Hsp90-Tau Complex Reveals Molecular Basis for Specificity in Chaperone Action</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">963</span>– <span class="NLM_lpage">974</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2014.01.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2014.01.037" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24581495" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvpsFagtg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=963-974&author=G.+E.+Karag%C3%B6zauthor=A.+M.+S.+Duarteauthor=E.+Akouryauthor=H.+Ippelauthor=J.+Biernatauthor=T.+Mor%C3%A1n+Luengoauthor=M.+Radliauthor=T.+Didenkoauthor=B.+A.+Nordhuesauthor=D.+B.+Veprintsevauthor=C.+A.+Dickeyauthor=E.+Mandelkowauthor=M.+Zweckstetterauthor=R.+Boelensauthor=T.+Madlauthor=S.+G.+D.+R%C3%BCdiger&title=Hsp90-Tau+Complex+Reveals+Molecular+Basis+for+Specificity+in+Chaperone+Action&doi=10.1016%2Fj.cell.2014.01.037"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">Hsp90-Tau complex reveals molecular basis for specificity in chaperone action</span></div><div class="casAuthors">Karagoz G Elif; Duarte Afonso M S; Akoury Elias; Ippel Hans; Biernat Jacek; Moran Luengo Tania; Radli Martina; Didenko Tatiana; Nordhues Bryce A; Dickey Chad A; Veprintsev Dmitry B; Mandelkow Eckhard; Zweckstetter Markus; Boelens Rolf; Madl Tobias; Rudiger Stefan G D</div><div class="citationInfo"><span class="NLM_cas:title">Cell</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">963-74</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Protein folding in the cell relies on the orchestrated action of conserved families of molecular chaperones, the Hsp70 and Hsp90 systems.  Hsp70 acts early and Hsp90 late in the folding path, yet the molecular basis of this timing is enigmatic, mainly because the substrate specificity of Hsp90 is poorly understood.  Here, we obtained a structural model of Hsp90 in complex with its natural disease-associated substrate, the intrinsically disordered Tau protein.  Hsp90 binds to a broad region in Tau that includes the aggregation-prone repeats.  Complementarily, a 106-ÅA-long substrate-binding interface in Hsp90 enables many low-affinity contacts.  This allows recognition of scattered hydrophobic residues in late folding intermediates that remain after early burial of the Hsp70 sites.  Our model resolves the paradox of how Hsp90 specifically selects for late folding intermediates but also for some intrinsically disordered proteins-through the eyes of Hsp90 they look the same.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQJ1lLYwX_fzXl8icjszYl0fW6udTcc2eaMwYyHvmw7prntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvpsFagtg%253D%253D&md5=ac8a4ccf143cd572c3e52517a9d4d2fc</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2014.01.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2014.01.037%26sid%3Dliteratum%253Aachs%26aulast%3DKarag%25C3%25B6z%26aufirst%3DG.%2BE.%26aulast%3DDuarte%26aufirst%3DA.%2BM.%2BS.%26aulast%3DAkoury%26aufirst%3DE.%26aulast%3DIppel%26aufirst%3DH.%26aulast%3DBiernat%26aufirst%3DJ.%26aulast%3DMor%25C3%25A1n%2BLuengo%26aufirst%3DT.%26aulast%3DRadli%26aufirst%3DM.%26aulast%3DDidenko%26aufirst%3DT.%26aulast%3DNordhues%26aufirst%3DB.%2BA.%26aulast%3DVeprintsev%26aufirst%3DD.%2BB.%26aulast%3DDickey%26aufirst%3DC.%2BA.%26aulast%3DMandelkow%26aufirst%3DE.%26aulast%3DZweckstetter%26aufirst%3DM.%26aulast%3DBoelens%26aufirst%3DR.%26aulast%3DMadl%26aufirst%3DT.%26aulast%3DR%25C3%25BCdiger%26aufirst%3DS.%2BG.%2BD.%26atitle%3DHsp90-Tau%2520Complex%2520Reveals%2520Molecular%2520Basis%2520for%2520Specificity%2520in%2520Chaperone%2520Action%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D963%26epage%3D974%26doi%3D10.1016%2Fj.cell.2014.01.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Libich, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fawzi, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ying, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clore, G. M.</span></span> <span> </span><span class="NLM_article-title">Probing the Transient Dark State of Substrate Binding to GroEL by Relaxation-Based Solution NMR</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">11361</span>– <span class="NLM_lpage">11366</span>, <span class="refDoi"> DOI: 10.1073/pnas.1305715110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1305715110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23798407" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref110/cit110&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1GlsrfJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=11361-11366&author=D.+S.+Libichauthor=N.+L.+Fawziauthor=J.+Yingauthor=G.+M.+Clore&title=Probing+the+Transient+Dark+State+of+Substrate+Binding+to+GroEL+by+Relaxation-Based+Solution+NMR&doi=10.1073%2Fpnas.1305715110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit110R"><div class="casContent"><span class="casTitleNuber">110</span><div class="casTitle"><span class="NLM_cas:atitle">Probing the transient dark state of substrate binding to GroEL by relaxation-based solution NMR</span></div><div class="casAuthors">Libich, David S.; Fawzi, Nicolas L.; Ying, Jinfa; Clore, G. Marius</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">28</span>),
    <span class="NLM_cas:pages">11361-11366, S11361/1-S11361/14</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The mechanism whereby the prototypical chaperonin GroEL performs work on substrate proteins has not yet been fully elucidated, hindered by lack of detailed structural and dynamic information on the bound substrate.  Previous studies produced conflicting reports on the state of GroEL-bound polypeptides, largely due to the transient and dynamic nature of these complexes.  Here, the authors present a unique approach, based on combined anal. of four complementary relaxation-based NMR expts., to probe directly the "dark" NMR-invisible state of the model, intrinsically disordered, polypeptide amyloid β (Aβ40) bound to GroEL.  The four NMR expts., lifetime line-broadening, dark-state exchange satn. transfer, relaxation dispersion, and small exchange-induced chem. shifts, are dependent in different ways on the overall exchange rates and populations of the free and bound states of the substrate, as well as on residue-specific dynamics and structure within the bound state as reported by transverse magnetization relaxation rates and backbone chem. shifts, resp.  Global fitting of all the NMR data shows that the complex is transient with a lifetime of <1 ms, that binding involves two predominantly hydrophobic segments corresponding to predicted GroEL consensus binding sequences, and that the structure of the bound polypeptide remains intrinsically and dynamically disordered with minimal changes in secondary structure propensity relative to the free state.  The authors' results establish a unique method to observe NMR-invisible dynamic states of GroEL-bound substrates and to describe at at. resoln. the events between substrate binding and encapsulation that are crucial for understanding the normal and stress-related metabolic function of chaperonins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhROaLvWZ3jLVg90H21EOLACvtfcHk0lhmgIzniscCBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1GlsrfJ&md5=457756cba725fd0dc3d1edfe5a0d30af</span></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1305715110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1305715110%26sid%3Dliteratum%253Aachs%26aulast%3DLibich%26aufirst%3DD.%2BS.%26aulast%3DFawzi%26aufirst%3DN.%2BL.%26aulast%3DYing%26aufirst%3DJ.%26aulast%3DClore%26aufirst%3DG.%2BM.%26atitle%3DProbing%2520the%2520Transient%2520Dark%2520State%2520of%2520Substrate%2520Binding%2520to%2520GroEL%2520by%2520Relaxation-Based%2520Solution%2520NMR%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D11361%26epage%3D11366%26doi%3D10.1073%2Fpnas.1305715110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref111"><div class="reference"><strong class="refLabel"><a href="#ref111" class="rightTabRefNumLink">111</a></strong><div class="NLM_citation" id="rightTab-cit111"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ostrem, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peters, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sos, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shokat, K. M.</span></span> <span> </span><span class="NLM_article-title">K-Ras(G12C) Inhibitors Allosterically Control GTP Affinity and Effector Interactions</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">551</span>, <span class="refDoi"> DOI: 10.1038/nature12796</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12796" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24256730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref111/cit111&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=548-551&author=J.+M.+Ostremauthor=U.+Petersauthor=M.+L.+Sosauthor=J.+A.+Wellsauthor=K.+M.+Shokat&title=K-Ras%28G12C%29+Inhibitors+Allosterically+Control+GTP+Affinity+and+Effector+Interactions&doi=10.1038%2Fnature12796"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit111R"><div class="casContent"><span class="casTitleNuber">111</span><div class="casTitle"><span class="NLM_cas:atitle">K-Ras(G12C) inhibitors allosterically control GTP affinity and effector interactions</span></div><div class="casAuthors">Ostrem, Jonathan M.; Peters, Ulf; Sos, Martin L.; Wells, James A.; Shokat, Kevan M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7477</span>),
    <span class="NLM_cas:pages">548-551</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Somatic mutations in the small GTPase K-Ras are the most common activating lesions found in human cancer, and are generally assocd. with poor response to std. therapies.  Efforts to target this oncogene directly have faced difficulties owing to its picomolar affinity for GTP/GDP and the absence of known allosteric regulatory sites.  Oncogenic mutations result in functional activation of Ras family proteins by impairing GTP hydrolysis.  With diminished regulation by GTPase activity, the nucleotide state of Ras becomes more dependent on relative nucleotide affinity and concn.  This gives GTP an advantage over GDP and increases the proportion of active GTP-bound Ras.  Here we report the development of small mols. that irreversibly bind to a common oncogenic mutant, K-Ras(G12C).  These compds. rely on the mutant cysteine for binding and therefore do not affect the wild-type protein.  Crystallog. studies reveal the formation of a new pocket that is not apparent in previous structures of Ras, beneath the effector binding switch-II region.  Binding of these inhibitors to K-Ras(G12C) disrupts both switch-I and switch-II, subverting the native nucleotide preference to favor GDP over GTP and impairing binding to Raf.  Our data provide structure-based validation of a new allosteric regulatory site on Ras that is targetable in a mutant-specific manner.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokkgcVXuBn_rVg90H21EOLACvtfcHk0lhmgIzniscCBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVGqs7jO&md5=27c1e7d8ef6b7e4b2919e9f00a073923</span></div><a href="/servlet/linkout?suffix=cit111&amp;dbid=16384&amp;doi=10.1038%2Fnature12796&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12796%26sid%3Dliteratum%253Aachs%26aulast%3DOstrem%26aufirst%3DJ.%2BM.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DSos%26aufirst%3DM.%2BL.%26aulast%3DWells%26aufirst%3DJ.%2BA.%26aulast%3DShokat%26aufirst%3DK.%2BM.%26atitle%3DK-Ras%2528G12C%2529%2520Inhibitors%2520Allosterically%2520Control%2520GTP%2520Affinity%2520and%2520Effector%2520Interactions%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D548%26epage%3D551%26doi%3D10.1038%2Fnature12796" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref112"><div class="reference"><strong class="refLabel"><a href="#ref112" class="rightTabRefNumLink">112</a></strong><div class="NLM_citation" id="rightTab-cit112"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Boehlein, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hannah, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stewart, J. D.</span></span> <span> </span><span class="NLM_article-title">Probing Allosteric Binding Sites of the Maize Endosperm ADP-Glucose Pyrophosphorylase</span>. <i>Plant Physiol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>152</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">95</span>, <span class="refDoi"> DOI: 10.1104/pp.109.146928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1104%2Fpp.109.146928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19889875" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref112/cit112&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3cXmsF2mur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=152&publication_year=2010&pages=85-95&author=S.+K.+Boehleinauthor=J.+R.+Shawauthor=L.+C.+Hannahauthor=J.+D.+Stewart&title=Probing+Allosteric+Binding+Sites+of+the+Maize+Endosperm+ADP-Glucose+Pyrophosphorylase&doi=10.1104%2Fpp.109.146928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit112R"><div class="casContent"><span class="casTitleNuber">112</span><div class="casTitle"><span class="NLM_cas:atitle">Probing allosteric binding sites of the maize endosperm ADP-glucose pyrophosphorylase</span></div><div class="casAuthors">Boehlein, Susan K.; Shaw, Janine R.; Hannah, L. Curtis; Stewart, Jon D.</div><div class="citationInfo"><span class="NLM_cas:title">Plant Physiology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">152</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">85-95</span>CODEN:
                <span class="NLM_cas:coden">PLPHAY</span>;
        ISSN:<span class="NLM_cas:issn">0032-0889</span>.
    
            (<span class="NLM_cas:orgname">American Society of Plant Biologists</span>)
        </div><div class="casAbstract">Maize (Zea mays) endosperm ADP-glucose pyrophosphorylase (AGPase) is a highly regulated enzyme that catalyzes the rate-limiting step in starch biosynthesis.  Although the structure of the heterotetrameric maize endosperm AGPase remains unsolved, structures of a nonnative, low-activity form of the potato tuber (Solanum tuberosum) AGPase (small subunit homotetramer) reported previously by others revealed that several sulfate ions bind to each enzyme.  These sites are also believed to interact with allosteric regulators such as inorg. phosphate and 3-phosphoglycerate (3-PGA).  Several arginine (Arg) side chains contact the bound sulfate ions in the potato structure and likely play important roles in allosteric effector binding.  Alanine-scanning mutagenesis was applied to the corresponding Arg residues in both the small and large subunits of maize endosperm AGPase to det. their roles in allosteric regulation and thermal stability.  Steady-state kinetic and regulatory parameters were measured for each mutant.  All of the Arg mutants examd.-in both the small and large subunits-bound 3-PGA more weakly than the wild type (A50 increased by 3.5- to 20-fold).  By contrast, the binding of two other maize AGPase allosteric activators (fructose-6-phosphate and glucose-6-phosphate) did not always mimic the changes obsd. for 3-PGA.  In fact, compared to 3-PGA, fructose-6-phosphate is a more efficient activator in two of the Arg mutants.  Phosphate binding was also affected by Arg substitutions.  The combined data support a model for the binding interactions assocd. with 3-PGA in which allosteric activators and inorg. phosphate compete directly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7x7HX27hNsLVg90H21EOLACvtfcHk0ljIvUKani222g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXmsF2mur0%253D&md5=df848e68fbe6a173a7064b626f0e2f20</span></div><a href="/servlet/linkout?suffix=cit112&amp;dbid=16384&amp;doi=10.1104%2Fpp.109.146928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1104%252Fpp.109.146928%26sid%3Dliteratum%253Aachs%26aulast%3DBoehlein%26aufirst%3DS.%2BK.%26aulast%3DShaw%26aufirst%3DJ.%2BR.%26aulast%3DHannah%26aufirst%3DL.%2BC.%26aulast%3DStewart%26aufirst%3DJ.%2BD.%26atitle%3DProbing%2520Allosteric%2520Binding%2520Sites%2520of%2520the%2520Maize%2520Endosperm%2520ADP-Glucose%2520Pyrophosphorylase%26jtitle%3DPlant%2520Physiol.%26date%3D2010%26volume%3D152%26spage%3D85%26epage%3D95%26doi%3D10.1104%2Fpp.109.146928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref113"><div class="reference"><strong class="refLabel"><a href="#ref113" class="rightTabRefNumLink">113</a></strong><div class="NLM_citation" id="rightTab-cit113"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Getlik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bohlke, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janning, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Direct Binding Assay for the Detection of Type IV Allosteric Inhibitors of Abl</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>134</i></span>,  <span class="NLM_fpage">9138</span>– <span class="NLM_lpage">9141</span>, <span class="refDoi"> DOI: 10.1021/ja303858w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja303858w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref113/cit113&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=134&publication_year=2012&pages=9138-9141&author=R.+Schneiderauthor=C.+Beckerauthor=J.+R.+Simardauthor=M.+Getlikauthor=N.+Bohlkeauthor=P.+Janningauthor=D.+Rauh&title=Direct+Binding+Assay+for+the+Detection+of+Type+IV+Allosteric+Inhibitors+of+Abl&doi=10.1021%2Fja303858w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit113R"><div class="casContent"><span class="casTitleNuber">113</span><div class="casTitle"><span class="NLM_cas:atitle">Direct Binding Assay for the Detection of Type IV Allosteric Inhibitors of Abl</span></div><div class="casAuthors">Schneider, Ralf; Becker, Christian; Simard, Jeffrey R.; Getlik, Matthaeus; Bohlke, Nina; Janning, Petra; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">134</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9138-9141</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Abelson (Abl) tyrosine kinase is an important cellular enzyme that is rendered constitutively active in the breakpoint cluster region (BCR)-Abl fusion protein, contributing to several forms of leukemia.  Although inhibiting BCR-Abl activity with imatinib shows great clin. success, many patients acquire secondary mutations that result in resistance to imatinib.  Second-generation inhibitors such as dasatinib and nilotinib can overcome the majority of these mutations but fail to treat patients with an esp. prevalent T315I mutation at the gatekeeper position of the kinase domain.  However, a combination of nilotinib with an allosteric type IV inhibitor was recently shown to overcome this clin. relevant point mutation.  In this study, we present the development of a direct binding assay that enables the straightforward detection of allosteric inhibitors which bind within the myristate pocket of Abl.  The assay is amenable to high-throughput screening and exclusively detects the binding of ligands to this unique allosteric site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHtVe0LFdqurVg90H21EOLACvtfcHk0ljIvUKani222g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnt1Wrt74%253D&md5=95aa204ff4d761f2a2155112abe35799</span></div><a href="/servlet/linkout?suffix=cit113&amp;dbid=16384&amp;doi=10.1021%2Fja303858w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja303858w%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGetlik%26aufirst%3DM.%26aulast%3DBohlke%26aufirst%3DN.%26aulast%3DJanning%26aufirst%3DP.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DDirect%2520Binding%2520Assay%2520for%2520the%2520Detection%2520of%2520Type%2520IV%2520Allosteric%2520Inhibitors%2520of%2520Abl%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2012%26volume%3D134%26spage%3D9138%26epage%3D9141%26doi%3D10.1021%2Fja303858w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref114"><div class="reference"><strong class="refLabel"><a href="#ref114" class="rightTabRefNumLink">114</a></strong><div class="NLM_citation" id="rightTab-cit114"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beumer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simard, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grütter, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rauh, D.</span></span> <span> </span><span class="NLM_article-title">Selective Detection of Allosteric Phosphatase Inhibitors</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">6838</span>– <span class="NLM_lpage">6841</span>, <span class="refDoi"> DOI: 10.1021/ja4030484</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja4030484" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref114/cit114&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtlWhtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=6838-6841&author=R.+Schneiderauthor=C.+Beumerauthor=J.+R.+Simardauthor=C.+Gr%C3%BCtterauthor=D.+Rauh&title=Selective+Detection+of+Allosteric+Phosphatase+Inhibitors&doi=10.1021%2Fja4030484"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit114R"><div class="casContent"><span class="casTitleNuber">114</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Detection of Allosteric Phosphatase Inhibitors</span></div><div class="casAuthors">Schneider, Ralf; Beumer, Claudia; Simard, Jeffrey R.; Gruetter, Christian; Rauh, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">6838-6841</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Normal cellular function, such as signal transduction, is largely controlled by the reversible phosphorylation of cellular proteins catalyzed by two major classes of enzymes, kinases and phosphatases.  A misbalance in this complex and dynamic interplay leads to a variety of severe diseases, such as cancer, inflammation, or autoimmune diseases.  This makes kinases as well as phosphatases equally attractive targets for therapeutic manipulation by small mols.  While the development of kinase inhibitors has resulted in several blockbuster drugs, such as imatinib, with remarkable success in the clinic and sales of many billions of U.S. dollars per yr, not a single phosphatase inhibitor has yet been approved for clin. use.  Similar to the kinase world, substrate-competitive phosphatase inhibitors have been developed but were not suitable for further development into clin. candidates due to their charge and limited selectivity.  Research efforts, therefore, have shifted to the exploitation of allosteric sites that can regulate phosphatase activity and may enable the discovery of novel modulators of phosphatase activity with much improved pharmacol. properties.  However, assay systems, which enable the straightforward discovery of these inhibitor types, are missing.  Here, we present a novel binding assay capable of detecting ligands of an allosteric pocket of the protein tyrosine phosphatase 1B.  This assay is suitable for high-throughput screening and selectively detects ligands which bind to this unique site with a clear discrimination from substrate-competitive ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMMpcm8RfGWrVg90H21EOLACvtfcHk0ljIvUKani222g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtlWhtbk%253D&md5=49a980ad7f749718803f1afe1d8d1a04</span></div><a href="/servlet/linkout?suffix=cit114&amp;dbid=16384&amp;doi=10.1021%2Fja4030484&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja4030484%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DR.%26aulast%3DBeumer%26aufirst%3DC.%26aulast%3DSimard%26aufirst%3DJ.%2BR.%26aulast%3DGr%25C3%25BCtter%26aufirst%3DC.%26aulast%3DRauh%26aufirst%3DD.%26atitle%3DSelective%2520Detection%2520of%2520Allosteric%2520Phosphatase%2520Inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D6838%26epage%3D6841%26doi%3D10.1021%2Fja4030484" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref115"><div class="reference"><strong class="refLabel"><a href="#ref115" class="rightTabRefNumLink">115</a></strong><div class="NLM_citation" id="rightTab-cit115"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Magnaghi, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">D’Alessio, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valsasina, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Avanzi, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rizzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Asa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gasparri, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cozzi, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cucchi, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orrenius, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polucci, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballinari, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perrera, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cervi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casale, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderson, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galvani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Donati, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Brien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jackson, P. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Isacchi, A.</span></span> <span> </span><span class="NLM_article-title">Covalent and Allosteric Inhibitors of the ATPase VCP/p97 Induce Cancer Cell Death</span>. <i>Nat. Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">548</span>– <span class="NLM_lpage">559</span>, <span class="refDoi"> DOI: 10.1038/nchembio.1313</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnchembio.1313" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23892893" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref115/cit115&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFOht7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=548-559&author=P.+Magnaghiauthor=R.+D%E2%80%99Alessioauthor=B.+Valsasinaauthor=N.+Avanziauthor=S.+Rizziauthor=D.+Asaauthor=F.+Gasparriauthor=L.+Cozziauthor=U.+Cucchiauthor=C.+Orreniusauthor=P.+Polucciauthor=D.+Ballinariauthor=C.+Perreraauthor=A.+Leoneauthor=G.+Cerviauthor=E.+Casaleauthor=Y.+Xiaoauthor=C.+Wongauthor=D.+J.+Andersonauthor=A.+Galvaniauthor=D.+Donatiauthor=T.+O%E2%80%99Brienauthor=P.+K.+Jacksonauthor=A.+Isacchi&title=Covalent+and+Allosteric+Inhibitors+of+the+ATPase+VCP%2Fp97+Induce+Cancer+Cell+Death&doi=10.1038%2Fnchembio.1313"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit115R"><div class="casContent"><span class="casTitleNuber">115</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent and allosteric inhibitors of the ATPase VCP/p97 induce cancer cell death</span></div><div class="casAuthors">Magnaghi, Paola; D'Alessio, Roberto; Valsasina, Barbara; Avanzi, Nilla; Rizzi, Simona; Asa, Daniela; Gasparri, Fabio; Cozzi, Liviana; Cucchi, Ulisse; Orrenius, Christian; Polucci, Paolo; Ballinari, Dario; Perrera, Claudia; Leone, Antonella; Cervi, Giovanni; Casale, Elena; Xiao, Yang; Wong, Chihunt; Anderson, Daniel J.; Galvani, Arturo; Donati, Daniele; O'Brien, Tom; Jackson, Peter K.; Isacchi, Antonella</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">548-556</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">VCP (also known as p97 or Cdc48p in yeast) is an AAA+ ATPase regulating endoplasmic reticulum-assocd. degrdn.  After high-throughput screening, we developed compds. that inhibit VCP via different mechanisms, including covalent modification of an active site cysteine and a new allosteric mechanism.  Using photoaffinity labeling, structural anal. and mutagenesis, we mapped the binding site of allosteric inhibitors to a region spanning the D1 and D2 domains of adjacent protomers encompassing elements important for nucleotide-state sensing and ATP hydrolysis.  These compds. induced an increased affinity for nucleotides.  Interference with nucleotide turnover in individual subunits and distortion of interprotomer communication cooperated to impair VCP enzymic activity.  Chem. expansion of this allosteric class identified NMS-873, the most potent and specific VCP inhibitor described to date, which activated the unfolded protein response, interfered with autophagy and induced cancer cell death.  The consistent pattern of cancer cell killing by covalent and allosteric inhibitors provided crit. validation of VCP as a cancer target.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUmfk1JLyHdrVg90H21EOLACvtfcHk0lgsy_W_budNgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFOht7jK&md5=76bbca9d88073e2208fef5677a9223ba</span></div><a href="/servlet/linkout?suffix=cit115&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1313%26sid%3Dliteratum%253Aachs%26aulast%3DMagnaghi%26aufirst%3DP.%26aulast%3DD%25E2%2580%2599Alessio%26aufirst%3DR.%26aulast%3DValsasina%26aufirst%3DB.%26aulast%3DAvanzi%26aufirst%3DN.%26aulast%3DRizzi%26aufirst%3DS.%26aulast%3DAsa%26aufirst%3DD.%26aulast%3DGasparri%26aufirst%3DF.%26aulast%3DCozzi%26aufirst%3DL.%26aulast%3DCucchi%26aufirst%3DU.%26aulast%3DOrrenius%26aufirst%3DC.%26aulast%3DPolucci%26aufirst%3DP.%26aulast%3DBallinari%26aufirst%3DD.%26aulast%3DPerrera%26aufirst%3DC.%26aulast%3DLeone%26aufirst%3DA.%26aulast%3DCervi%26aufirst%3DG.%26aulast%3DCasale%26aufirst%3DE.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DWong%26aufirst%3DC.%26aulast%3DAnderson%26aufirst%3DD.%2BJ.%26aulast%3DGalvani%26aufirst%3DA.%26aulast%3DDonati%26aufirst%3DD.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DJackson%26aufirst%3DP.%2BK.%26aulast%3DIsacchi%26aufirst%3DA.%26atitle%3DCovalent%2520and%2520Allosteric%2520Inhibitors%2520of%2520the%2520ATPase%2520VCP%252Fp97%2520Induce%2520Cancer%2520Cell%2520Death%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2013%26volume%3D9%26spage%3D548%26epage%3D559%26doi%3D10.1038%2Fnchembio.1313" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref116"><div class="reference"><strong class="refLabel"><a href="#ref116" class="rightTabRefNumLink">116</a></strong><div class="NLM_citation" id="rightTab-cit116"><span><span class="NLM_contrib-group"><span class="NLM_string-name">de Vera, I. M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Novick, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hughes, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brust, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Munoz-Tello, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gardner, W. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marciano, D. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Griffin, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kojetin, D. J.</span></span> <span> </span><span class="NLM_article-title">Synergistic Regulation of Coregulator/Nuclear Receptor Interaction by Ligand and DNA</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1518</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.07.019</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2017.07.019" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28890360" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref116/cit116&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhsVKqu7rI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=1506-1518&author=I.+M.+S.+de+Veraauthor=J.+Zhengauthor=S.+Novickauthor=J.+Shangauthor=T.+S.+Hughesauthor=R.+Brustauthor=P.+Munoz-Telloauthor=W.+J.+Gardnerauthor=D.+P.+Marcianoauthor=X.+Kongauthor=P.+R.+Griffinauthor=D.+J.+Kojetin&title=Synergistic+Regulation+of+Coregulator%2FNuclear+Receptor+Interaction+by+Ligand+and+DNA&doi=10.1016%2Fj.str.2017.07.019"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit116R"><div class="casContent"><span class="casTitleNuber">116</span><div class="casTitle"><span class="NLM_cas:atitle">Synergistic Regulation of Coregulator/Nuclear Receptor Interaction by Ligand and DNA</span></div><div class="casAuthors">de Vera, Ian Mitchelle S.; Zheng, Jie; Novick, Scott; Shang, Jinsai; Hughes, Travis S.; Brust, Richard; Munoz-Tello, Paola; Gardner, William J., Jr.; Marciano, David P.; Kong, Xiangming; Griffin, Patrick R.; Kojetin, Douglas J.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1506-1518.e4</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Nuclear receptor (NR) transcription factors bind various coreceptors, small-mol. ligands, DNA response element sequences, and transcriptional coregulator proteins to affect gene transcription.  Small-mol. ligands and DNA are known to influence receptor structure, coregulator protein interaction, and function; however, little is known on the mechanism of synergy between ligand and DNA.  Using quant. biochem., biophys., and soln. structural methods, including 13C-detected NMR and hydrogen/deuterium exchange (HDX) mass spectrometry, we show that ligand and DNA cooperatively recruit the intrinsically disordered steroid receptor coactivator-2 (SRC-2/TIF2/GRIP1/NCoA-2) receptor interaction domain to peroxisome proliferator-activated receptor gamma-retinoid X receptor alpha (PPARγ-RXRα) heterodimer and reveal the binding determinants of the complex.  Our data reveal a thermodn. mechanism by which DNA binding propagates a conformational change in PPARγ-RXRα, stabilizes the receptor ligand binding domain dimer interface, and impacts ligand potency and cooperativity in NR coactivator recruitment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEHAepTsadQ7Vg90H21EOLACvtfcHk0lgsy_W_budNgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhsVKqu7rI&md5=12580b50c1e459c4c67ca59312ed3bb9</span></div><a href="/servlet/linkout?suffix=cit116&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.07.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.07.019%26sid%3Dliteratum%253Aachs%26aulast%3Dde%2BVera%26aufirst%3DI.%2BM.%2BS.%26aulast%3DZheng%26aufirst%3DJ.%26aulast%3DNovick%26aufirst%3DS.%26aulast%3DShang%26aufirst%3DJ.%26aulast%3DHughes%26aufirst%3DT.%2BS.%26aulast%3DBrust%26aufirst%3DR.%26aulast%3DMunoz-Tello%26aufirst%3DP.%26aulast%3DGardner%26aufirst%3DW.%2BJ.%26aulast%3DMarciano%26aufirst%3DD.%2BP.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DGriffin%26aufirst%3DP.%2BR.%26aulast%3DKojetin%26aufirst%3DD.%2BJ.%26atitle%3DSynergistic%2520Regulation%2520of%2520Coregulator%252FNuclear%2520Receptor%2520Interaction%2520by%2520Ligand%2520and%2520DNA%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D1506%26epage%3D1518%26doi%3D10.1016%2Fj.str.2017.07.019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref117"><div class="reference"><strong class="refLabel"><a href="#ref117" class="rightTabRefNumLink">117</a></strong><div class="NLM_citation" id="rightTab-cit117"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Prediction of Protein Allostery</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>50</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">8</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2017.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2017.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29080471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref117/cit117&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs1GjsrrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2018&pages=1-8&author=J.+G.+Greenerauthor=M.+J.+Sternberg&title=Structure-Based+Prediction+of+Protein+Allostery&doi=10.1016%2Fj.sbi.2017.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit117R"><div class="casContent"><span class="casTitleNuber">117</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based prediction of protein allostery</span></div><div class="casAuthors">Greener, Joe G.; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1-8</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Allostery is the functional change at one site on a protein caused by a change at a distant site.  In order for the benefits of allostery to be taken advantage of, both for basic understanding of proteins and to develop new classes of drugs, the structure-based prediction of allosteric binding sites, modulators and communication pathways is necessary.  Here we review the recently emerging field of allosteric prediction, focusing mainly on computational methods.  We also describe the search for cryptic binding pockets and attempts to design allostery into proteins.  The development and adoption of such methods is essential or the long-preached potential of allostery will remain elusive.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq4PyWF3NhFrVg90H21EOLACvtfcHk0lgsy_W_budNgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs1GjsrrJ&md5=3dc8bf712c04354650571ca1b51849aa</span></div><a href="/servlet/linkout?suffix=cit117&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2017.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2017.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DStructure-Based%2520Prediction%2520of%2520Protein%2520Allostery%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2018%26volume%3D50%26spage%3D1%26epage%3D8%26doi%3D10.1016%2Fj.sbi.2017.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref118"><div class="reference"><strong class="refLabel"><a href="#ref118" class="rightTabRefNumLink">118</a></strong><div class="NLM_citation" id="rightTab-cit118"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD: A Comprehensive Database of Allosteric Proteins and Modulators</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">D663</span>– <span class="NLM_lpage">D669</span>, <span class="refDoi"> DOI: 10.1093/nar/gkq1022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkq1022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21051350" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref118/cit118&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2011&pages=D663-D669&author=Z.+Huangauthor=L.+Zhuauthor=Y.+Caoauthor=G.+Wuauthor=X.+Liuauthor=Y.+Chenauthor=Q.+Wangauthor=T.+Shiauthor=Y.+Zhaoauthor=Y.+Wangauthor=W.+Liauthor=Y.+Liauthor=H.+Chenauthor=G.+Chenauthor=J.+Zhang&title=ASD%3A+A+Comprehensive+Database+of+Allosteric+Proteins+and+Modulators&doi=10.1093%2Fnar%2Fgkq1022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit118R"><div class="casContent"><span class="casTitleNuber">118</span><div class="casTitle"><span class="NLM_cas:atitle">ASD: a comprehensive database of allosteric proteins and modulators</span></div><div class="casAuthors">Huang, Zhimin; Zhu, Liang; Cao, Yan; Wu, Geng; Liu, Xinyi; Chen, Yingyi; Wang, Qi; Shi, Ting; Zhao, Yaxue; Wang, Yuefei; Li, Weihua; Li, Yixue; Chen, Haifeng; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">Suppl. 1</span>),
    <span class="NLM_cas:pages">D663-D669</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct, rapid and efficient way of regulating protein function, ranging from the control of metabolic mechanisms to signal-transduction pathways.  However, an enormous amt. of unsystematic allosteric information has deterred scientists who could benefit from this field.  Here, we present the AlloSteric Database (ASD), the first online database that provides a central resource for the display, search and anal. of structure, function and related annotation for allosteric mols.  Currently, ASD contains 336 allosteric proteins from 101 species and 8095 modulators in three categories (activators, inhibitors and regulators).  Proteins are annotated with a detailed description of allostery, biol. process and related diseases, and modulators with binding affinity, physicochem. properties and therapeutic area.  Integrating the information of allosteric proteins in ASD should allow for the identification of specific allosteric sites of a given subtype among proteins of the same family that can potentially serve as ideal targets for exptl. validation.  In addn., modulators curated in ASD can be used to investigate potent allosteric targets for the query compd., and also help chemists to implement structure modifications for novel allosteric drug design.  Therefore, ASD could be a platform and a starting point for biologists and medicinal chemists for furthering allosteric research.  ASD is freely available at http://mdl.shsmu.edu.cn/ASD/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpg8WPj2tR4JbVg90H21EOLACvtfcHk0lhDrwD9mrgv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivF2ks74%253D&md5=28a63caabbc2942d98ca03ca1b4256ee</span></div><a href="/servlet/linkout?suffix=cit118&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkq1022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkq1022%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhu%26aufirst%3DL.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DShi%26aufirst%3DT.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%253A%2520A%2520Comprehensive%2520Database%2520of%2520Allosteric%2520Proteins%2520and%2520Modulators%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2011%26volume%3D39%26spage%3DD663%26epage%3DD669%26doi%3D10.1093%2Fnar%2Fgkq1022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref119"><div class="reference"><strong class="refLabel"><a href="#ref119" class="rightTabRefNumLink">119</a></strong><div class="NLM_citation" id="rightTab-cit119"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASD v2.0: Updated Content and Novel Features Focusing on Allosteric Regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">D510</span>– <span class="NLM_lpage">D516</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt1247</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt1247" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24293647" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref119/cit119&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXos1Gh" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=D510-D516&author=Z.+Huangauthor=L.+Mouauthor=Q.+Shenauthor=S.+Luauthor=C.+Liauthor=X.+Liuauthor=G.+Wangauthor=S.+Liauthor=L.+Gengauthor=Y.+Liuauthor=J.+Wuauthor=G.+Chenauthor=J.+Zhang&title=ASD+v2.0%3A+Updated+Content+and+Novel+Features+Focusing+on+Allosteric+Regulation&doi=10.1093%2Fnar%2Fgkt1247"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit119R"><div class="casContent"><span class="casTitleNuber">119</span><div class="casTitle"><span class="NLM_cas:atitle">ASD v2.0: updated content and novel features focusing on allosteric regulation</span></div><div class="casAuthors">Huang, Zhimin; Mou, Linkai; Shen, Qiancheng; Lu, Shaoyong; Li, Chuangang; Liu, Xinyi; Wang, Guanqiao; Li, Shuai; Geng, Lv; Liu, Yaqin; Wu, Jiawei; Chen, Guoqiang; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D510-D516</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allostery is the most direct and efficient way for regulation of biol. macromol. function and is induced by the binding of a ligand at an allosteric site topog. distinct from the orthosteric site.  AlloSteric Database (ASD, http://mdl.shsmu.edu.cn/ASD) has been developed to provide comprehensive information on allostery.  Owing to the inherent high receptor selectivity and lower target-based toxicity, allosteric regulation is expected to assume a more prominent role in drug discovery and bioengineering, leading to the rapid growth of allosteric findings.  In this updated version, ASD v2.0 has expanded to 1286 allosteric proteins, 565 allosteric diseases and 22 008 allosteric modulators.  A total of 907 allosteric site-modulator structural complexes and >200 structural pairs of orthosteric/allosteric sites in the allosteric proteins were constructed for researchers to develop allosteric site and pathway tools in response to community demands.  Up-to-date allosteric pathways were manually curated in the updated version.  In addn., both the front-end and the back-end of ASD have been redesigned and enhanced to allow more efficient access.  Taken together, these updates are useful for facilitating the investigation of allosteric mechanisms, allosteric target identification and allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMY2SmS8wLI7Vg90H21EOLACvtfcHk0lhDrwD9mrgv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXos1Gh&md5=2cdd4c0b36e00381b2050890f5b5295e</span></div><a href="/servlet/linkout?suffix=cit119&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt1247&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt1247%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DMou%26aufirst%3DL.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASD%2520v2.0%253A%2520Updated%2520Content%2520and%2520Novel%2520Features%2520Focusing%2520on%2520Allosteric%2520Regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3DD510%26epage%3DD516%26doi%3D10.1093%2Fnar%2Fgkt1247" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref120"><div class="reference"><strong class="refLabel"><a href="#ref120" class="rightTabRefNumLink">120</a></strong><div class="NLM_citation" id="rightTab-cit120"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Geng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">ASBench: Benchmarking Sets for Allosteric Discovery</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">2598</span>– <span class="NLM_lpage">2600</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btv169</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtv169" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25810427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref120/cit120&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1Gju7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=2598-2600&author=W.+Huangauthor=G.+Wangauthor=Q.+Shenauthor=X.+Liuauthor=S.+Luauthor=L.+Gengauthor=Z.+Huangauthor=J.+Zhang&title=ASBench%3A+Benchmarking+Sets+for+Allosteric+Discovery&doi=10.1093%2Fbioinformatics%2Fbtv169"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit120R"><div class="casContent"><span class="casTitleNuber">120</span><div class="casTitle"><span class="NLM_cas:atitle">ASBench: benchmarking sets for allosteric discovery</span></div><div class="casAuthors">Huang, Wenkang; Wang, Guanqiao; Shen, Qiancheng; Liu, Xinyi; Lu, Shaoyong; Geng, Lv; Huang, Zhimin; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2598-2600</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: Allostery allows for the fine-tuning of protein function.  Targeting allosteric sites is gaining increasing recognition as a novel strategy in drug design.  The key challenge in the discovery of allosteric sites has strongly motivated the development of computational methods and thus high-quality, publicly accessible std. data have become indispensable.  Here, we report benchmarking data for exptl. detd. allosteric sites through a complex process, including a 'Core set' with 235 unique allosteric sites and a 'Core-Diversity set' with 147 structurally diverse allosteric sites.  These benchmarking sets can be exploited to develop efficient computational methods to predict unknown allosteric sites in proteins and reveal unique allosteric ligand-protein interactions to guide allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpw-vf0LmMbprVg90H21EOLACvtfcHk0lhDrwD9mrgv2Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1Gju7zK&md5=12dce810b56a48c987dfb8743be95548</span></div><a href="/servlet/linkout?suffix=cit120&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtv169&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtv169%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DG.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DGeng%26aufirst%3DL.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DASBench%253A%2520Benchmarking%2520Sets%2520for%2520Allosteric%2520Discovery%26jtitle%3DBioinformatics%26date%3D2015%26volume%3D31%26spage%3D2598%26epage%3D2600%26doi%3D10.1093%2Fbioinformatics%2Fbtv169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref121"><div class="reference"><strong class="refLabel"><a href="#ref121" class="rightTabRefNumLink">121</a></strong><div class="NLM_citation" id="rightTab-cit121"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Allosite: A Method for Predicting Allosteric Sites</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2357</span>– <span class="NLM_lpage">2359</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btt399</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtt399" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23842804" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref121/cit121&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGmsLfL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=2357-2359&author=W.+Huangauthor=S.+Luauthor=Z.+Huangauthor=X.+Liuauthor=L.+Mouauthor=Y.+Luoauthor=Y.+Zhaoauthor=Y.+Liuauthor=Z.+Chenauthor=T.+Houauthor=J.+Zhang&title=Allosite%3A+A+Method+for+Predicting+Allosteric+Sites&doi=10.1093%2Fbioinformatics%2Fbtt399"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit121R"><div class="casContent"><span class="casTitleNuber">121</span><div class="casTitle"><span class="NLM_cas:atitle">Allosite: a method for predicting allosteric sites</span></div><div class="casAuthors">Huang, Wenkang; Lu, Shaoyong; Huang, Zhimin; Liu, Xinyi; Mou, Linkai; Luo, Yu; Zhao, Yanlong; Liu, Yaqin; Chen, Zhongjie; Hou, Tingjun; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">2357-2359</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: The use of allosteric modulators as preferred therapeutic agents against classic orthosteric ligands has colossal advantages, including higher specificity, fewer side effects and lower toxicity.  Therefore, the computational prediction of allosteric sites in proteins is receiving increased attention in the field of drug discovery.  Allosite is a newly developed automatic tool for the prediction of allosteric sites in proteins of interest and is now available through a web server.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBwW1VByREtbVg90H21EOLACvtfcHk0liGhuyHTIVFHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGmsLfL&md5=24b4661f3e4cee3d5cab6cc8c86bc607</span></div><a href="/servlet/linkout?suffix=cit121&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtt399&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtt399%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DMou%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DHou%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAllosite%253A%2520A%2520Method%2520for%2520Predicting%2520Allosteric%2520Sites%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D2357%26epage%3D2359%26doi%3D10.1093%2Fbioinformatics%2Fbtt399" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref122"><div class="reference"><strong class="refLabel"><a href="#ref122" class="rightTabRefNumLink">122</a></strong><div class="NLM_citation" id="rightTab-cit122"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Improved Method for the Identification and Validation of Allosteric Sites</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">2358</span>– <span class="NLM_lpage">2363</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.7b00014</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.7b00014" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref122/cit122&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtlGlu7vL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2017&pages=2358-2363&author=K.+Songauthor=X.+Liuauthor=W.+Huangauthor=S.+Luauthor=Q.+Shenauthor=L.+Zhangauthor=J.+Zhang&title=Improved+Method+for+the+Identification+and+Validation+of+Allosteric+Sites&doi=10.1021%2Facs.jcim.7b00014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit122R"><div class="casContent"><span class="casTitleNuber">122</span><div class="casTitle"><span class="NLM_cas:atitle">Improved Method for the Identification and Validation of Allosteric Sites</span></div><div class="casAuthors">Song, Kun; Liu, Xinyi; Huang, Wenkang; Lu, Shaoyong; Shen, Qiancheng; Zhang, Lu; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2358-2363</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allosteric regulation induced by modulators binding to different, often distant, allosteric sites allows for exquisite control of protein functional activity.  The structural diversity of allosteric sites endows allosteric modulators with high selectivity and low toxicity.  Targeting allosteric sites, a novel tactic in drug discovery, has garnered much attention in the scientific community, and the identification of allosteric sites has become an important component of the development of allosteric drugs.  Here we present AllositePro, a method which predicts allosteric sites in proteins by combining pocket features with perturbation anal.  The performance of AllositePro is superior to that of the other currently available methods.  Using AllositePro, we predicted a novel allosteric site in cyclin-dependent kinase 2 (CDK2) and validated it by site-directed mutagenesis assay.  Thus, the AllositePro method provides an effective way to identify allosteric sites and could be a useful strategy for allosteric drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq23d-Zg0_4G7Vg90H21EOLACvtfcHk0liGhuyHTIVFHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtlGlu7vL&md5=fbc362245d0bfb34686e0bc2fb38203b</span></div><a href="/servlet/linkout?suffix=cit122&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.7b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.7b00014%26sid%3Dliteratum%253Aachs%26aulast%3DSong%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DHuang%26aufirst%3DW.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DImproved%2520Method%2520for%2520the%2520Identification%2520and%2520Validation%2520of%2520Allosteric%2520Sites%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2017%26volume%3D57%26spage%3D2358%26epage%3D2363%26doi%3D10.1021%2Facs.jcim.7b00014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref123"><div class="reference"><strong class="refLabel"><a href="#ref123" class="rightTabRefNumLink">123</a></strong><div class="NLM_citation" id="rightTab-cit123"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Akbar, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Helms, V.</span></span> <span> </span><span class="NLM_article-title">ALLO: A Tool to Discriminate and Prioritize Allosteric Pockets</span>. <i>Chem. Biol. Drug Des.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>91</i></span>,  <span class="NLM_fpage">845</span>– <span class="NLM_lpage">853</span>, <span class="refDoi"> DOI: 10.1111/cbdd.13161</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1111%2Fcbdd.13161" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29250934" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref123/cit123&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlOntrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=91&publication_year=2018&pages=845-853&author=R.+Akbarauthor=V.+Helms&title=ALLO%3A+A+Tool+to+Discriminate+and+Prioritize+Allosteric+Pockets&doi=10.1111%2Fcbdd.13161"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit123R"><div class="casContent"><span class="casTitleNuber">123</span><div class="casTitle"><span class="NLM_cas:atitle">ALLO: A tool to discriminate and prioritize allosteric pockets</span></div><div class="casAuthors">Akbar, Rahmad; Helms, Volkhard</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Biology & Drug Design</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">91</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">845-853</span>CODEN:
                <span class="NLM_cas:coden">CBDDAL</span>;
        ISSN:<span class="NLM_cas:issn">1747-0277</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Allosteric proteins make up a substantial proportion of human drug targets.  Thus, rational design of small mol. binders that target these proteins requires the identification of putative allosteric pockets and an understanding of their potential activity.  Here, we characterized allosteric pockets using a set of physicochem. descriptors and compared them to pockets that are found on the surface of a protein.  Further, we trained predictive models capable of discriminating allosteric pockets from orthosteric pockets and models capable of prioritizing allosteric pockets in a set of pockets found on a given protein.  Such models might be useful for identifying novel allosteric sites and in turn, potentially new allosteric drug targets.  Datasets along with a Python program encapsulating the predictive models are available at .</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPa9JRzKq5VbVg90H21EOLACvtfcHk0ljsiIbCaIKGpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlOntrg%253D&md5=a06e7d563aa27493e1de845acf6c2581</span></div><a href="/servlet/linkout?suffix=cit123&amp;dbid=16384&amp;doi=10.1111%2Fcbdd.13161&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fcbdd.13161%26sid%3Dliteratum%253Aachs%26aulast%3DAkbar%26aufirst%3DR.%26aulast%3DHelms%26aufirst%3DV.%26atitle%3DALLO%253A%2520A%2520Tool%2520to%2520Discriminate%2520and%2520Prioritize%2520Allosteric%2520Pockets%26jtitle%3DChem.%2520Biol.%2520Drug%2520Des.%26date%3D2018%26volume%3D91%26spage%3D845%26epage%3D853%26doi%3D10.1111%2Fcbdd.13161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref124"><div class="reference"><strong class="refLabel"><a href="#ref124" class="rightTabRefNumLink">124</a></strong><div class="NLM_citation" id="rightTab-cit124"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, A. S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Westwood, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brear, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rogers, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mavridis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, J. B. O.</span></span> <span> </span><span class="NLM_article-title">A Random Forest Model for Predicting Allosteric and Functional Sites on Proteins</span>. <i>Mol. Inf.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">125</span>– <span class="NLM_lpage">135</span>, <span class="refDoi"> DOI: 10.1002/minf.201500108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fminf.201500108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27491922" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref124/cit124&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSqsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2016&pages=125-135&author=A.+S.+Y.+Chenauthor=N.+J.+Westwoodauthor=P.+Brearauthor=G.+W.+Rogersauthor=L.+Mavridisauthor=J.+B.+O.+Mitchell&title=A+Random+Forest+Model+for+Predicting+Allosteric+and+Functional+Sites+on+Proteins&doi=10.1002%2Fminf.201500108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit124R"><div class="casContent"><span class="casTitleNuber">124</span><div class="casTitle"><span class="NLM_cas:atitle">A Random Forest Model for Predicting Allosteric and Functional Sites on Proteins</span></div><div class="casAuthors">Chen, Ava S.-Y.; Westwood, Nicholas J.; Brear, Paul; Rogers, Graeme W.; Mavridis, Lazaros; Mitchell, John B. O.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Informatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">3-4</span>),
    <span class="NLM_cas:pages">125-135</span>CODEN:
                <span class="NLM_cas:coden">MIONBS</span>;
        ISSN:<span class="NLM_cas:issn">1868-1743</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">We created a computational method to identify allosteric sites using a machine learning method trained and tested on protein structures contg. bound ligand mols.  The Random Forest machine learning approach was adopted to build our three-way predictive model.  Based on descriptors collated for each ligand and binding site, the classification model allows us to assign protein cavities as allosteric, regular or orthosteric, and hence to identify allosteric sites. 43 structural descriptors per complex were derived and were used to characterize individual protein-ligand binding sites belonging to the three classes, allosteric, regular and orthosteric.  We carried out a sep. validation on a further unseen set of protein structures contg. the ligand 2-(N-cyclohexylamino) ethane sulfonic acid (CHES).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGryYcl0PpE-r7Vg90H21EOLACvtfcHk0ljsiIbCaIKGpg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSqsbs%253D&md5=aa74e9ba8d0bbe06a2b61fbd31b0854c</span></div><a href="/servlet/linkout?suffix=cit124&amp;dbid=16384&amp;doi=10.1002%2Fminf.201500108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fminf.201500108%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DA.%2BS.%2BY.%26aulast%3DWestwood%26aufirst%3DN.%2BJ.%26aulast%3DBrear%26aufirst%3DP.%26aulast%3DRogers%26aufirst%3DG.%2BW.%26aulast%3DMavridis%26aufirst%3DL.%26aulast%3DMitchell%26aufirst%3DJ.%2BB.%2BO.%26atitle%3DA%2520Random%2520Forest%2520Model%2520for%2520Predicting%2520Allosteric%2520and%2520Functional%2520Sites%2520on%2520Proteins%26jtitle%3DMol.%2520Inf.%26date%3D2016%26volume%3D35%26spage%3D125%26epage%3D135%26doi%3D10.1002%2Fminf.201500108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref125"><div class="reference"><strong class="refLabel"><a href="#ref125" class="rightTabRefNumLink">125</a></strong><div class="NLM_citation" id="rightTab-cit125"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smock, R. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rivoire, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Swain, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibler, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gierasch, L. M.</span></span> <span> </span><span class="NLM_article-title">An Interdomain Sector Mediating Allostery in Hsp70 Molecular Chaperones</span>. <i>Mol. Syst. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">10</span>, <span class="refDoi"> DOI: 10.1038/msb.2010.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref125/cit125&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fmsb.2010.65" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&pages=1-10&author=R.+G.+Smockauthor=O.+Rivoireauthor=W.+P.+Russauthor=J.+F.+Swainauthor=S.+Leiblerauthor=R.+Ranganathanauthor=L.+M.+Gierasch&title=An+Interdomain+Sector+Mediating+Allostery+in+Hsp70+Molecular+Chaperones&doi=10.1038%2Fmsb.2010.65"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit125&amp;dbid=16384&amp;doi=10.1038%2Fmsb.2010.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fmsb.2010.65%26sid%3Dliteratum%253Aachs%26aulast%3DSmock%26aufirst%3DR.%2BG.%26aulast%3DRivoire%26aufirst%3DO.%26aulast%3DRuss%26aufirst%3DW.%2BP.%26aulast%3DSwain%26aufirst%3DJ.%2BF.%26aulast%3DLeibler%26aufirst%3DS.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DGierasch%26aufirst%3DL.%2BM.%26atitle%3DAn%2520Interdomain%2520Sector%2520Mediating%2520Allostery%2520in%2520Hsp70%2520Molecular%2520Chaperones%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2010%26volume%3D6%26spage%3D1%26epage%3D10%26doi%3D10.1038%2Fmsb.2010.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref126"><div class="reference"><strong class="refLabel"><a href="#ref126" class="rightTabRefNumLink">126</a></strong><div class="NLM_citation" id="rightTab-cit126"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J.</span></span> <span> </span><span class="NLM_article-title">AlloPred: Prediction of Allosteric Pockets on Proteins Using Normal Mode Perturbation Analysis</span>. <i>BMC Bioinf.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">335</span>, <span class="refDoi"> DOI: 10.1186/s12859-015-0771-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1186%2Fs12859-015-0771-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26493317" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref126/cit126&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC28zlt12ksw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2015&pages=335&author=J.+G.+Greenerauthor=M.+J.+Sternberg&title=AlloPred%3A+Prediction+of+Allosteric+Pockets+on+Proteins+Using+Normal+Mode+Perturbation+Analysis&doi=10.1186%2Fs12859-015-0771-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit126R"><div class="casContent"><span class="casTitleNuber">126</span><div class="casTitle"><span class="NLM_cas:atitle">AlloPred: prediction of allosteric pockets on proteins using normal mode perturbation analysis</span></div><div class="casAuthors">Greener Joe G; Sternberg Michael J E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">335</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Despite being hugely important in biological processes, allostery is poorly understood and no universal mechanism has been discovered.  Allosteric drugs are a largely unexplored prospect with many potential advantages over orthosteric drugs.  Computational methods to predict allosteric sites on proteins are needed to aid the discovery of allosteric drugs, as well as to advance our fundamental understanding of allostery.  RESULTS:  AlloPred, a novel method to predict allosteric pockets on proteins, was developed.  AlloPred uses perturbation of normal modes alongside pocket descriptors in a machine learning approach that ranks the pockets on a protein.  AlloPred ranked an allosteric pocket top for 23 out of 40 known allosteric proteins, showing comparable and complementary performance to two existing methods.  In 28 of 40 cases an allosteric pocket was ranked first or second.  The AlloPred web server, freely available at http://www.sbg.bio.ic.ac.uk/allopred/home, allows visualisation and analysis of predictions.  The source code and dataset information are also available from this site.  CONCLUSIONS:  Perturbation of normal modes can enhance our ability to predict allosteric sites on proteins.  Computational methods such as AlloPred assist drug discovery efforts by suggesting sites on proteins for further experimental study.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQpnbMrYjzf52LFM_lbR1VmfW6udTcc2eauzMlgCanetrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC28zlt12ksw%253D%253D&md5=2de7386cee597dc9300f1fd25c3b8631</span></div><a href="/servlet/linkout?suffix=cit126&amp;dbid=16384&amp;doi=10.1186%2Fs12859-015-0771-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-015-0771-1%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%26atitle%3DAlloPred%253A%2520Prediction%2520of%2520Allosteric%2520Pockets%2520on%2520Proteins%2520Using%2520Normal%2520Mode%2520Perturbation%2520Analysis%26jtitle%3DBMC%2520Bioinf.%26date%3D2015%26volume%3D16%26spage%3D335%26doi%3D10.1186%2Fs12859-015-0771-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref127"><div class="reference"><strong class="refLabel"><a href="#ref127" class="rightTabRefNumLink">127</a></strong><div class="NLM_citation" id="rightTab-cit127"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Panjkovich, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daura, X.</span></span> <span> </span><span class="NLM_article-title">PARS: A Web Server for the Prediction of Protein Allosteric and Regulatory Sites</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">1314</span>– <span class="NLM_lpage">1315</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btu002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtu002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24413526" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref127/cit127&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXmvFequr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2014&pages=1314-1315&author=A.+Panjkovichauthor=X.+Daura&title=PARS%3A+A+Web+Server+for+the+Prediction+of+Protein+Allosteric+and+Regulatory+Sites&doi=10.1093%2Fbioinformatics%2Fbtu002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit127R"><div class="casContent"><span class="casTitleNuber">127</span><div class="casTitle"><span class="NLM_cas:atitle">PARS: a web server for the prediction of Protein Allosteric and Regulatory Sites</span></div><div class="casAuthors">Panjkovich, Alejandro; Daura, Xavier</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1314-1315</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Summary: The regulation of protein activity is a key aspect of life at the mol. level.  Unveiling its details is thus crucial to understanding signalling and metabolic pathways.  The most common and powerful mechanism of protein-function regulation is allostery, which has been increasingly calling the attention of medicinal chemists due to its potential for the discovery of novel therapeutics.  In this context, PARS is a simple and fast method that queries protein dynamics and structural conservation to identify pockets on a protein structure that may exert a regulatory effect on the binding of a small-mol. ligand.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrNQj4nH_68ybVg90H21EOLACvtfcHk0lhzSS4z-tTgdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXmvFequr8%253D&md5=ae1c218abc25988f911af6eace2d38a1</span></div><a href="/servlet/linkout?suffix=cit127&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtu002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtu002%26sid%3Dliteratum%253Aachs%26aulast%3DPanjkovich%26aufirst%3DA.%26aulast%3DDaura%26aufirst%3DX.%26atitle%3DPARS%253A%2520A%2520Web%2520Server%2520for%2520the%2520Prediction%2520of%2520Protein%2520Allosteric%2520and%2520Regulatory%2520Sites%26jtitle%3DBioinformatics%26date%3D2014%26volume%3D30%26spage%3D1314%26epage%3D1315%26doi%3D10.1093%2Fbioinformatics%2Fbtu002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref128"><div class="reference"><strong class="refLabel"><a href="#ref128" class="rightTabRefNumLink">128</a></strong><div class="NLM_citation" id="rightTab-cit128"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Qi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Identifying Allosteric Binding Sites in Proteins with a Two-State Go? Model for Novel Allosteric Effector Discovery</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2962</span>– <span class="NLM_lpage">2971</span>, <span class="refDoi"> DOI: 10.1021/ct300395h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct300395h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref128/cit128&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XpvFemtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=2962-2971&author=Y.+Qiauthor=Q.+Wangauthor=B.+Tangauthor=L.+Lai&title=Identifying+Allosteric+Binding+Sites+in+Proteins+with+a+Two-State+Go%3F+Model+for+Novel+Allosteric+Effector+Discovery&doi=10.1021%2Fct300395h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit128R"><div class="casContent"><span class="casTitleNuber">128</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Allosteric Binding Sites in Proteins with a Two-State G‾o Model for Novel Allosteric Effector Discovery</span></div><div class="casAuthors">Qi, Yifei; Wang, Qian; Tang, Bo; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2962-2971</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is a common mechanism of controlling many biol. processes such as enzyme catalysis, signal transduction, and metabolic regulation.  The use of allostery to regulate protein activity is an important and promising strategy in drug discovery and biol. network regulation.  In order to modulate protein activity by allostery, predictive methods need to be developed to discover allosteric binding sites.  In the present study, we developed a new approach to identify allosteric sites in proteins based on the coarse-grained two-state G‾o model.  Starting from the concept that allostery is a conformation population shift process, we first constructed an ensemble of two functional states of a protein and tuned the energy landscape to bias one state.  We then added perturbations to a binding site and monitored the population distribution of the new ensemble.  If population redistribution occurred, then the binding perturbed site was predicted as a potential allosteric site.  Our approach successfully identified all the known allosteric sites in a set of test proteins.  Several new allosteric sites in the test proteins were also predicted.  By use of one of the new allosteric sites predicted from Escherichia coli phosphoglycerate dehydrogenase (PGDH), novel allosteric regulating mols. were screened by mol. docking and enzymic assay.  Three novel allosteric inhibitors were discovered and their binding modes were confirmed by mutation expts. and competitive assay.  The IC50 of the strongest inhibitor discovered was 21 μM, which is comparable to that of the native allosteric inhibitor L-serine.  The novel allosteric site discovered in PGDH is L-serine-independent, and inhibitors targeting this site can be used as novel regulators of the E. coli serine synthesis pathway.  Our approach for allosteric site prediction is generally applicable and the predicted sites can be used in discovering novel allosteric regulating mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZLFKLyehCX7Vg90H21EOLACvtfcHk0lignMSSkrjJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XpvFemtrY%253D&md5=6c95574165b32eba99bd8432118e5fa9</span></div><a href="/servlet/linkout?suffix=cit128&amp;dbid=16384&amp;doi=10.1021%2Fct300395h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct300395h%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DTang%26aufirst%3DB.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DIdentifying%2520Allosteric%2520Binding%2520Sites%2520in%2520Proteins%2520with%2520a%2520Two-State%2520Go%253F%2520Model%2520for%2520Novel%2520Allosteric%2520Effector%2520Discovery%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2012%26volume%3D8%26spage%3D2962%26epage%3D2971%26doi%3D10.1021%2Fct300395h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref129"><div class="reference"><strong class="refLabel"><a href="#ref129" class="rightTabRefNumLink">129</a></strong><div class="NLM_citation" id="rightTab-cit129"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Greener, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Filippis, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sternberg, M. J. E.</span></span> <span> </span><span class="NLM_article-title">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">546</span>– <span class="NLM_lpage">558</span>, <span class="refDoi"> DOI: 10.1016/j.str.2017.01.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2017.01.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28190781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref129/cit129&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2017&pages=546-558&author=J.+G.+Greenerauthor=I.+Filippisauthor=M.+J.+E.+Sternberg&title=Predicting+Protein+Dynamics+and+Allostery+Using+Multi-Protein+Atomic+Distance+Constraints&doi=10.1016%2Fj.str.2017.01.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit129R"><div class="casContent"><span class="casTitleNuber">129</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Protein Dynamics and Allostery Using Multi-Protein Atomic Distance Constraints</span></div><div class="casAuthors">Greener, Joe G.; Filippis, Ioannis; Sternberg, Michael J. E.</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">546-558</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The related concepts of protein dynamics, conformational ensembles and allostery are often difficult to study with mol. dynamics (MD) due to the timescales involved.  We present ExProSE (Exploration of Protein Structural Ensembles), a distance geometry-based method that generates an ensemble of protein structures from two input structures.  ExProSE provides a unified framework for the exploration of protein structure and dynamics in a fast and accessible way.  Using a dataset of apo/holo pairs it is shown that existing coarse-grained methods often cannot span large conformational changes.  For T4-lysozyme, ExProSE is able to generate ensembles that are more native-like than tCONCOORD and NMSim, and comparable with targeted MD.  By adding addnl. constraints representing potential modulators, ExProSE can predict allosteric sites.  ExProSE ranks an allosteric pocket first or second for 27 out of 58 allosteric proteins, which is similar and complementary to existing methods.  The ExProSE source code is freely available.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSF34-q1BCDrVg90H21EOLACvtfcHk0lignMSSkrjJRg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisF2isr8%253D&md5=2d2ea675b3b29ff52f9915ddecabe032</span></div><a href="/servlet/linkout?suffix=cit129&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2017.01.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2017.01.008%26sid%3Dliteratum%253Aachs%26aulast%3DGreener%26aufirst%3DJ.%2BG.%26aulast%3DFilippis%26aufirst%3DI.%26aulast%3DSternberg%26aufirst%3DM.%2BJ.%2BE.%26atitle%3DPredicting%2520Protein%2520Dynamics%2520and%2520Allostery%2520Using%2520Multi-Protein%2520Atomic%2520Distance%2520Constraints%26jtitle%3DStructure%26date%3D2017%26volume%3D25%26spage%3D546%26epage%3D558%26doi%3D10.1016%2Fj.str.2017.01.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref130"><div class="reference"><strong class="refLabel"><a href="#ref130" class="rightTabRefNumLink">130</a></strong><div class="NLM_citation" id="rightTab-cit130"><span><span class="NLM_contrib-group"><span class="NLM_string-name">De Vivo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bottegoni, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Role of Molecular Dynamics and Related Methods in Drug Discovery</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4035</span>– <span class="NLM_lpage">4061</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01684</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01684" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref130/cit130&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4035-4061&author=M.+De+Vivoauthor=M.+Masettiauthor=G.+Bottegoniauthor=A.+Cavalli&title=Role+of+Molecular+Dynamics+and+Related+Methods+in+Drug+Discovery&doi=10.1021%2Facs.jmedchem.5b01684"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit130R"><div class="casContent"><span class="casTitleNuber">130</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Molecular Dynamics and Related Methods in Drug Discovery</span></div><div class="casAuthors">De Vivo, Marco; Masetti, Matteo; Bottegoni, Giovanni; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4035-4061</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) and related methods are close to becoming routine computational tools for drug discovery.  Their main advantage is in explicitly treating structural flexibility and entropic effects.  This allows a more accurate est. of the thermodn. and kinetics assocd. with drug-target recognition and binding, as better algorithms and hardware architectures increase their use.  Here, we review the theor. background of MD and enhanced sampling methods, focusing on free-energy perturbation, metadynamics, steered MD, and other methods most consistently used to study drug-target binding.  We discuss unbiased MD simulations that nowadays allow the observation of unsupervised ligand-target binding, assessing how these approaches help optimizing target affinity and drug residence time toward improved drug efficacy.  Further issues discussed include allosteric modulation and the role of water mols. in ligand binding and optimization.  We conclude by calling for more prospective studies to attest to these methods' utility in discovering novel drug candidates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQb_02a7Nmh7Vg90H21EOLACvtfcHk0ljPaBkrpRM33g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlWksLs%253D&md5=b3d8a4fb5705a3555c2bb2a962420396</span></div><a href="/servlet/linkout?suffix=cit130&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BVivo%26aufirst%3DM.%26aulast%3DMasetti%26aufirst%3DM.%26aulast%3DBottegoni%26aufirst%3DG.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DRole%2520of%2520Molecular%2520Dynamics%2520and%2520Related%2520Methods%2520in%2520Drug%2520Discovery%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4035%26epage%3D4061%26doi%3D10.1021%2Facs.jmedchem.5b01684" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref131"><div class="reference"><strong class="refLabel"><a href="#ref131" class="rightTabRefNumLink">131</a></strong><div class="NLM_citation" id="rightTab-cit131"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Weinkam, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pons, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sali, A.</span></span> <span> </span><span class="NLM_article-title">Structure-Based Model of Allostery Predicts Coupling between Distant Sites</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">4875</span>– <span class="NLM_lpage">4880</span>, <span class="refDoi"> DOI: 10.1073/pnas.1116274109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1116274109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22403063" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref131/cit131&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xlt1agtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=4875-4880&author=P.+Weinkamauthor=J.+Ponsauthor=A.+Sali&title=Structure-Based+Model+of+Allostery+Predicts+Coupling+between+Distant+Sites&doi=10.1073%2Fpnas.1116274109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit131R"><div class="casContent"><span class="casTitleNuber">131</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-based model of allostery predicts coupling between distant sites</span></div><div class="casAuthors">Weinkam, Patrick; Pons, Jaume; Sali, Andrej</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4875-4880, S4875/1-S4875/7</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Allostery is a phenomenon that couples effector ligand binding at an allosteric site to a structural and/or dynamic change at a distant regulated site.  To study an allosteric transition, we vary the size of the allosteric site and its interactions to construct a series of energy landscapes with pronounced min. corresponding to both the effector bound and unbound crystal structures.  We use mol. dynamics to sample these landscapes.  The degree of perturbation by the effector, modeled by the size of the allosteric site, provides an order parameter for allostery that allows us to det. how microscopic motions give rise to commonly discussed macroscopic mechanisms: (i) induced fit, (ii) population shift, and (iii) driven.  These mechanisms involve decreasing structural differences between the effector bound and unbound populations.  A metric (ligand-induced cooperativity) can measure how cooperatively a given regulated site responds to effector binding and therefore what kind of allosteric mechanism is involved.  We apply the model to 3 proteins with exptl. characterized transitions: (i) calmodulin-GFP Ca2+ sensor protein, (ii) maltose binding protein, and (iii) CSL transcription factor.  Remarkably, the model is able to reproduce allosteric motion and predict coupling in a manner consistent with expt.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGom_5ajotvZS7Vg90H21EOLACvtfcHk0ljPaBkrpRM33g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xlt1agtL8%253D&md5=da9e811dffb0d3492f9f2ac8c2f7281b</span></div><a href="/servlet/linkout?suffix=cit131&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1116274109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1116274109%26sid%3Dliteratum%253Aachs%26aulast%3DWeinkam%26aufirst%3DP.%26aulast%3DPons%26aufirst%3DJ.%26aulast%3DSali%26aufirst%3DA.%26atitle%3DStructure-Based%2520Model%2520of%2520Allostery%2520Predicts%2520Coupling%2520between%2520Distant%2520Sites%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D4875%26epage%3D4880%26doi%3D10.1073%2Fpnas.1116274109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref132"><div class="reference"><strong class="refLabel"><a href="#ref132" class="rightTabRefNumLink">132</a></strong><div class="NLM_citation" id="rightTab-cit132"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">Reversing Allosteric Communication: From Detecting Allosteric Sites to Inducing and Tuning Targeted Allosteric Response</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>14</i></span>, <span class="NLM_elocation-id">e1006228</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1006228</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1006228" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref132/cit132&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29912863" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2018&author=W.-V.+Teeauthor=E.+Guarneraauthor=I.+N.+Berezovsky&title=Reversing+Allosteric+Communication%3A+From+Detecting+Allosteric+Sites+to+Inducing+and+Tuning+Targeted+Allosteric+Response&doi=10.1371%2Fjournal.pcbi.1006228"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit132&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1006228&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1006228%26sid%3Dliteratum%253Aachs%26aulast%3DTee%26aufirst%3DW.-V.%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DReversing%2520Allosteric%2520Communication%253A%2520From%2520Detecting%2520Allosteric%2520Sites%2520to%2520Inducing%2520and%2520Tuning%2520Targeted%2520Allosteric%2520Response%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2018%26volume%3D14%26doi%3D10.1371%2Fjournal.pcbi.1006228" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref133"><div class="reference"><strong class="refLabel"><a href="#ref133" class="rightTabRefNumLink">133</a></strong><div class="NLM_citation" id="rightTab-cit133"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Motions of Allosteric and Orthosteric Ligand-Binding Sites in Proteins Are Highly Correlated</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1725</span>– <span class="NLM_lpage">1733</span>, <span class="refDoi"> DOI: 10.1021/acs.jcim.6b00039</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00039" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref133/cit133&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVemtLrM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1725-1733&author=X.+Maauthor=H.+Mengauthor=L.+Lai&title=Motions+of+Allosteric+and+Orthosteric+Ligand-Binding+Sites+in+Proteins+Are+Highly+Correlated&doi=10.1021%2Facs.jcim.6b00039"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit133R"><div class="casContent"><span class="casTitleNuber">133</span><div class="casTitle"><span class="NLM_cas:atitle">Motions of Allosteric and Orthosteric Ligand-Binding Sites in Proteins are Highly Correlated</span></div><div class="casAuthors">Ma, Xiaomin; Meng, Hu; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1725-1733</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Allostery is the phenomenon in which a ligand binding at one site affects other sites in the same macromol.  Allostery has important roles in many biol. processes.  Theor., all non-fibrous proteins are potentially allosteric.  However, few allosteric proteins have been validated, and the identification of novel allosteric sites remains a challenge.  The motion of residues and subunits underlies protein function; therefore, the authors hypothesized that the motions of allosteric and orthosteric sites are correlated.  The authors used a dataset of 24 known allosteric sites from 23 monomer proteins to calc. the correlations between potential ligand-binding sites and corresponding orthosteric sites using a Gaussian network model (GNM).  Most of the known allosteric site motions showed high correlations with corresponding orthosteric site motions, whereas other surface cavities did not.  These high correlations were robust when using different structural data for the same protein, such as structures for the apo state and the orthosteric effector-binding state, whereas the contributions of different frequency modes to motion correlations depend on the given protein.  The high correlations between allosteric and orthosteric site motions were also obsd. in oligomeric allosteric proteins.  The authors applied motion correlation anal. to predict potential allosteric sites in the 23 monomer proteins, and some of these predictions were in good agreement with published exptl. data.  The authors also performed motion correlation anal. to identify a novel allosteric site in 15-lipoxygenase (an enzyme in the arachidonic acid metabolic network) using recently reported activating compds.  The authors' anal. correctly identified this novel allosteric site along with two other sites that are currently under exptl. study.  The authors' study demonstrates that the motions of allosteric sites are highly correlated with the motions of orthosteric sites.  The authors' correlation anal. method provides new tools for predicting potential allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbJEt1Kw-KErVg90H21EOLACvtfcHk0ljPaBkrpRM33g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVemtLrM&md5=c868c49d9026b2573554bb19f219a93d</span></div><a href="/servlet/linkout?suffix=cit133&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00039&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00039%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DX.%26aulast%3DMeng%26aufirst%3DH.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DMotions%2520of%2520Allosteric%2520and%2520Orthosteric%2520Ligand-Binding%2520Sites%2520in%2520Proteins%2520Are%2520Highly%2520Correlated%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1725%26epage%3D1733%26doi%3D10.1021%2Facs.jcim.6b00039" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref134"><div class="reference"><strong class="refLabel"><a href="#ref134" class="rightTabRefNumLink">134</a></strong><div class="NLM_citation" id="rightTab-cit134"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kaya, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Armutlulu, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ekesan, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haliloglu, T.</span></span> <span> </span><span class="NLM_article-title">MCPath: Monte Carlo Path Generation Approach to Predict Likely Allosteric Pathways and Functional Residues</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">W249</span>– <span class="NLM_lpage">W255</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt284</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt284" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref134/cit134&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23742907" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=W249-W255&author=C.+Kayaauthor=A.+Armutluluauthor=S.+Ekesanauthor=T.+Haliloglu&title=MCPath%3A+Monte+Carlo+Path+Generation+Approach+to+Predict+Likely+Allosteric+Pathways+and+Functional+Residues&doi=10.1093%2Fnar%2Fgkt284"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit134&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt284&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt284%26sid%3Dliteratum%253Aachs%26aulast%3DKaya%26aufirst%3DC.%26aulast%3DArmutlulu%26aufirst%3DA.%26aulast%3DEkesan%26aufirst%3DS.%26aulast%3DHaliloglu%26aufirst%3DT.%26atitle%3DMCPath%253A%2520Monte%2520Carlo%2520Path%2520Generation%2520Approach%2520to%2520Predict%2520Likely%2520Allosteric%2520Pathways%2520and%2520Functional%2520Residues%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3DW249%26epage%3DW255%26doi%3D10.1093%2Fnar%2Fgkt284" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref135"><div class="reference"><strong class="refLabel"><a href="#ref135" class="rightTabRefNumLink">135</a></strong><div class="NLM_citation" id="rightTab-cit135"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Goncearenco, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitternacht, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yong, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eisenhaber, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">SPACER: Server for Predicting Allosteric Communication and Effects of Regulation</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">266</span>– <span class="NLM_lpage">272</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt460</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkt460" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref135/cit135&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23737445" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=266-272&author=A.+Goncearencoauthor=S.+Mitternachtauthor=T.+Yongauthor=B.+Eisenhaberauthor=F.+Eisenhaberauthor=I.+N.+Berezovsky&title=SPACER%3A+Server+for+Predicting+Allosteric+Communication+and+Effects+of+Regulation&doi=10.1093%2Fnar%2Fgkt460"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit135&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt460&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt460%26sid%3Dliteratum%253Aachs%26aulast%3DGoncearenco%26aufirst%3DA.%26aulast%3DMitternacht%26aufirst%3DS.%26aulast%3DYong%26aufirst%3DT.%26aulast%3DEisenhaber%26aufirst%3DB.%26aulast%3DEisenhaber%26aufirst%3DF.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DSPACER%253A%2520Server%2520for%2520Predicting%2520Allosteric%2520Communication%2520and%2520Effects%2520of%2520Regulation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D266%26epage%3D272%26doi%3D10.1093%2Fnar%2Fgkt460" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref136"><div class="reference"><strong class="refLabel"><a href="#ref136" class="rightTabRefNumLink">136</a></strong><div class="NLM_citation" id="rightTab-cit136"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bahar, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L. W.</span></span> <span> </span><span class="NLM_article-title">DynOmics: Dynamics of Structural Proteome and Beyond</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>45</i></span>,  <span class="NLM_fpage">W374</span>– <span class="NLM_lpage">W380</span>, <span class="refDoi"> DOI: 10.1093/nar/gkx385</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgkx385" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28472330" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref136/cit136&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhs1amt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=W374-W380&author=H.+Liauthor=Y.+Y.+Changauthor=J.+Y.+Leeauthor=I.+Baharauthor=L.+W.+Yang&title=DynOmics%3A+Dynamics+of+Structural+Proteome+and+Beyond&doi=10.1093%2Fnar%2Fgkx385"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit136R"><div class="casContent"><span class="casTitleNuber">136</span><div class="casTitle"><span class="NLM_cas:atitle">DynOmics: dynamics of structural proteome and beyond</span></div><div class="casAuthors">Li, Hongchun; Chang, Yuan-Yu; Lee, Ji Young; Bahar, Ivet; Yang, Lee-Wei</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W374-W380</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  DynOmics is a portal developed to leverage rapidly growing structural proteomics data by efficiently and accurately evaluating the dynamics of structurally resolved systems, from individual mols. to large complexes and assemblies, in the context of their physiol. environment.  At the core of the portal is a newly developed server, ENM 1.0, which permits users to efficiently generate information on the collective dynamics of any structure in PDB format, user-uploaded or database-retrieved.  ENM 1.0 integrates two widely used elastic network models (ENMs) -the Gaussian Network Model (GNM) and the Anisotropic Network Model (ANM), extended to take account of mol. environment.  It enables users to assess potentially functional sites, signal transduction or allosteric communication mechanisms, and protein-protein and protein-DNA interaction poses, in addn. to delivering ensembles of accessible conformers reconstructed at. details based on the global modes of motions predicted by the ANM.  The 'environment' is defined in a flexible manner, from lipid bilayer and crystal contacts, to substrate or ligands bound to a protein, or surrounding subunits in a multimeric structure or assembly.  User-friendly interactive features permit users to easily visualize how the environment alter the intrinsic dynamics of the query systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrGAbiwacCgkbVg90H21EOLACvtfcHk0lioYFwFGuy64A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhs1amt7s%253D&md5=6b4171f6201ed5ff69175a32f0daf69a</span></div><a href="/servlet/linkout?suffix=cit136&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkx385&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkx385%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DH.%26aulast%3DChang%26aufirst%3DY.%2BY.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DBahar%26aufirst%3DI.%26aulast%3DYang%26aufirst%3DL.%2BW.%26atitle%3DDynOmics%253A%2520Dynamics%2520of%2520Structural%2520Proteome%2520and%2520Beyond%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DW374%26epage%3DW380%26doi%3D10.1093%2Fnar%2Fgkx385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref137"><div class="reference"><strong class="refLabel"><a href="#ref137" class="rightTabRefNumLink">137</a></strong><div class="NLM_citation" id="rightTab-cit137"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">AlloSigMA: Allosteric Signaling and Mutation Analysis Server</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>33</i></span>,  <span class="NLM_fpage">3996</span>– <span class="NLM_lpage">3998</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btx430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtx430" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29106449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref137/cit137&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsbnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2017&pages=3996-3998&author=E.+Guarneraauthor=Z.+W.+Tanauthor=Z.+Zhengauthor=I.+N.+Berezovsky&title=AlloSigMA%3A+Allosteric+Signaling+and+Mutation+Analysis+Server&doi=10.1093%2Fbioinformatics%2Fbtx430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit137R"><div class="casContent"><span class="casTitleNuber">137</span><div class="casTitle"><span class="NLM_cas:atitle">AlloSigMA: Allosteric signaling and mutation analysis server</span></div><div class="casAuthors">Guarnera, Enrico; Tan, Zhen Wah; Zheng, Zejun; Berezovsky, Igor N.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">3996-3998</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4811</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Motivation: Allostery is an omnipresent mechanism of the function modulation in proteins via either effector binding or mutations in the exosites.  Despite the growing no. of online servers and databases devoted to prediction/classification of allosteric sites and their characteristics, there is a lack of resources for an efficient and quick estn. of the causality and energetics of allosteric communication.  Results: The AlloSigMA server implements a unique approach on the basis of the recently introduced structure-based statistical mech. models of allosteric signaling.  It provides an interactive framework for estg. the allosteric free energy as a result of the ligand(s) binding, mutation(s) and their combinations.  Latent regulatory exosites and allosteric effect of mutations can be detected and explored, facilitating the research efforts in protein engineering and allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpMXne1_sWhrLVg90H21EOLACvtfcHk0lioYFwFGuy64A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCnsbnO&md5=2a6a59793713dfd13708bab94ca51d4b</span></div><a href="/servlet/linkout?suffix=cit137&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtx430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtx430%26sid%3Dliteratum%253Aachs%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DTan%26aufirst%3DZ.%2BW.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DAlloSigMA%253A%2520Allosteric%2520Signaling%2520and%2520Mutation%2520Analysis%2520Server%26jtitle%3DBioinformatics%26date%3D2017%26volume%3D33%26spage%3D3996%26epage%3D3998%26doi%3D10.1093%2Fbioinformatics%2Fbtx430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref138"><div class="reference"><strong class="refLabel"><a href="#ref138" class="rightTabRefNumLink">138</a></strong><div class="NLM_citation" id="rightTab-cit138"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Z. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tee, W.-V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarnera, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Booth, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berezovsky, I. N.</span></span> <span> </span><span class="NLM_article-title">AlloMAPS: Allosteric Mutation Analysis and Polymorphism of Signaling Database</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">D265</span>– <span class="NLM_lpage">D270</span>, <span class="refDoi"> DOI: 10.1093/nar/gky1028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky1028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=30365033" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref138/cit138&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhs1CgtL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2019&pages=D265-D270&author=Z.+W.+Tanauthor=W.-V.+Teeauthor=E.+Guarneraauthor=L.+Boothauthor=I.+N.+Berezovsky&title=AlloMAPS%3A+Allosteric+Mutation+Analysis+and+Polymorphism+of+Signaling+Database&doi=10.1093%2Fnar%2Fgky1028"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><div class="casRecord" id="&#xA;                        rightTab-cas_cit138R"><div class="casContent"></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="#"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=&md5=9874b665cc7a056b8e2f928dd3112440</span></div><a href="/servlet/linkout?suffix=cit138&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky1028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky1028%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DZ.%2BW.%26aulast%3DTee%26aufirst%3DW.-V.%26aulast%3DGuarnera%26aufirst%3DE.%26aulast%3DBooth%26aufirst%3DL.%26aulast%3DBerezovsky%26aufirst%3DI.%2BN.%26atitle%3DAlloMAPS%253A%2520Allosteric%2520Mutation%2520Analysis%2520and%2520Polymorphism%2520of%2520Signaling%2520Database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2019%26volume%3D47%26spage%3DD265%26epage%3DD270%26doi%3D10.1093%2Fnar%2Fgky1028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref139"><div class="reference"><strong class="refLabel"><a href="#ref139" class="rightTabRefNumLink">139</a></strong><div class="NLM_citation" id="rightTab-cit139"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">An, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Proteome-Scale Investigation of Protein Allosteric Regulation Perturbed by Somatic Mutations in 7,000 Cancer Genomes</span>. <i>Am. J. Hum. Genet.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">5</span>– <span class="NLM_lpage">20</span>, <span class="refDoi"> DOI: 10.1016/j.ajhg.2016.09.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.ajhg.2016.09.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27939638" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref139/cit139&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XitVOjsr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2017&pages=5-20&author=Q.+Shenauthor=F.+Chengauthor=H.+Songauthor=W.+Luauthor=J.+Zhaoauthor=X.+Anauthor=M.+Liuauthor=G.+Chenauthor=Z.+Zhaoauthor=J.+Zhang&title=Proteome-Scale+Investigation+of+Protein+Allosteric+Regulation+Perturbed+by+Somatic+Mutations+in+7%2C000+Cancer+Genomes&doi=10.1016%2Fj.ajhg.2016.09.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit139R"><div class="casContent"><span class="casTitleNuber">139</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-Scale Investigation of Protein Allosteric Regulation Perturbed by Somatic Mutations in 7,000 Cancer Genomes</span></div><div class="casAuthors">Shen, Qiancheng; Cheng, Feixiong; Song, Huili; Lu, Weiqiang; Zhao, Junfei; An, Xiaoli; Liu, Mingyao; Chen, Guoqiang; Zhao, Zhongming; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">American Journal of Human Genetics</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">5-20</span>CODEN:
                <span class="NLM_cas:coden">AJHGAG</span>;
        ISSN:<span class="NLM_cas:issn">0002-9297</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The allosteric regulation triggering the protein's functional activity via conformational changes is an intrinsic function of protein under many physiol. and pathol. conditions, including cancer.  Identification of the biol. effects of specific somatic variants on allosteric proteins and the phenotypes that they alter during tumor initiation and progression is a central challenge for cancer genomes in the post-genomic era.  Here, we mapped more than 47,000 somatic missense mutations obsd. in approx. 7,000 tumor-normal matched samples across 33 cancer types into protein allosteric sites to prioritize the mutated allosteric proteins and we tested our prediction in cancer cell lines.  We found that the deleterious mutations identified in cancer genomes were more significantly enriched at protein allosteric sites than tolerated mutations, suggesting a crit. role for protein allosteric variants in cancer.  Next, we developed a statistical approach, namely AlloDriver, and further identified 15 potential mutated allosteric proteins during pan-cancer and individual cancer-type analyses.  More importantly, we exptl. confirmed that p.Pro360Ala on PDE10A played a potential oncogenic role in mediating tumorigenesis in non-small cell lung cancer (NSCLC).  In summary, these findings shed light on the role of allosteric regulation during tumorigenesis and provide a useful tool for the timely development of targeted cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDSiE8UeSCcbVg90H21EOLACvtfcHk0lhpFegeSq8Gig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XitVOjsr3I&md5=8124a5ae3b2da1cfff8d7e90170666f2</span></div><a href="/servlet/linkout?suffix=cit139&amp;dbid=16384&amp;doi=10.1016%2Fj.ajhg.2016.09.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ajhg.2016.09.020%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DCheng%26aufirst%3DF.%26aulast%3DSong%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DAn%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DG.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DProteome-Scale%2520Investigation%2520of%2520Protein%2520Allosteric%2520Regulation%2520Perturbed%2520by%2520Somatic%2520Mutations%2520in%25207%252C000%2520Cancer%2520Genomes%26jtitle%3DAm.%2520J.%2520Hum.%2520Genet.%26date%3D2017%26volume%3D100%26spage%3D5%26epage%3D20%26doi%3D10.1016%2Fj.ajhg.2016.09.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref140"><div class="reference"><strong class="refLabel"><a href="#ref140" class="rightTabRefNumLink">140</a></strong><div class="NLM_citation" id="rightTab-cit140"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sethi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, R. W. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gerstein, M.</span></span> <span> </span><span class="NLM_article-title">Identifying Allosteric Hotspots with Dynamics: Application to Inter- and Intra-Species Conservation</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">826</span>– <span class="NLM_lpage">837</span>, <span class="refDoi"> DOI: 10.1016/j.str.2016.03.008</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.str.2016.03.008" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27066750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref140/cit140&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlslels7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2016&pages=826-837&author=D.+Clarkeauthor=A.+Sethiauthor=S.+Liauthor=S.+Kumarauthor=R.+W.+F.+Changauthor=J.+Chenauthor=M.+Gerstein&title=Identifying+Allosteric+Hotspots+with+Dynamics%3A+Application+to+Inter-+and+Intra-Species+Conservation&doi=10.1016%2Fj.str.2016.03.008"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit140R"><div class="casContent"><span class="casTitleNuber">140</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Allosteric Hotspots with Dynamics: Application to Inter- and Intra-species Conservation</span></div><div class="casAuthors">Clarke, Declan; Sethi, Anurag; Li, Shantao; Kumar, Sushant; Chang, Richard W. F.; Chen, Jieming; Gerstein, Mark</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">826-837</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">The rapidly growing vol. of data being produced by next-generation sequencing initiatives is enabling more in-depth analyses of conservation than previously possible.  Deep sequencing is uncovering disease loci and regions under selective constraint, despite the fact that intuitive biophys. reasons for such constraint are sometimes absent.  Allostery may often provide the missing explanatory link.  We use models of protein conformational change to identify allosteric residues by finding essential surface pockets and information-flow bottlenecks, and we develop a software tool that enables users to perform this anal. on their own proteins of interest.  Though fundamentally 3D-structural in nature, our anal. is computationally fast, thereby allowing us to run it across the PDB and to evaluate general properties of predicted allosteric residues.  We find that these tend to be conserved over diverse evolutionary time scales.  Finally, we highlight examples of allosteric residues that help explain poorly understood disease-assocd. variants.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRj593ixDCd7Vg90H21EOLACvtfcHk0lhpFegeSq8Gig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlslels7o%253D&md5=a630e9fdd6d6ee889ffef8fe54d8ec02</span></div><a href="/servlet/linkout?suffix=cit140&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2016.03.008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2016.03.008%26sid%3Dliteratum%253Aachs%26aulast%3DClarke%26aufirst%3DD.%26aulast%3DSethi%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DKumar%26aufirst%3DS.%26aulast%3DChang%26aufirst%3DR.%2BW.%2BF.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DGerstein%26aufirst%3DM.%26atitle%3DIdentifying%2520Allosteric%2520Hotspots%2520with%2520Dynamics%253A%2520Application%2520to%2520Inter-%2520and%2520Intra-Species%2520Conservation%26jtitle%3DStructure%26date%3D2016%26volume%3D24%26spage%3D826%26epage%3D837%26doi%3D10.1016%2Fj.str.2016.03.008" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref141"><div class="reference"><strong class="refLabel"><a href="#ref141" class="rightTabRefNumLink">141</a></strong><div class="NLM_citation" id="rightTab-cit141"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Su, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Alloscore: A Method for Predicting Allosteric Ligand–protein Interactions</span>. <i>Bioinformatics</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">1574</span>– <span class="NLM_lpage">1576</span>, <span class="refDoi"> DOI: 10.1093/bioinformatics/btw036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fbioinformatics%2Fbtw036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26803160" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref141/cit141&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XhslWku7rE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2016&pages=1574-1576&author=S.+Liauthor=Q.+Shenauthor=M.+Suauthor=X.+Liuauthor=S.+Luauthor=Z.+Chenauthor=R.+Wangauthor=J.+Zhang&title=Alloscore%3A+A+Method+for+Predicting+Allosteric+Ligand%E2%80%93protein+Interactions&doi=10.1093%2Fbioinformatics%2Fbtw036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit141R"><div class="casContent"><span class="casTitleNuber">141</span><div class="casTitle"><span class="NLM_cas:atitle">Alloscore: a method for predicting allosteric ligand-protein interactions</span></div><div class="casAuthors">Li, Shuai; Shen, Qiancheng; Su, Minyi; Liu, Xinyi; Lu, Shaoyong; Chen, Zhongjie; Wang, Renxiao; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1574-1576</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Allosteric ligands have increasingly gained attention as potential therapeutic agents due to their higher target selectivity and lower toxicity compared with classic orthosteric ligands.  Despite the great interest in the development of allosteric drugs as a new tactic in drug discovery, the understanding of the ligand-protein interactions underlying allosteric binding represents a key challenge.  Herein, we introduce Alloscore, a web server that predicts the binding affinities of allosteric ligand-protein interactions.  This method exhibits prominent performance in describing allosteric binding and could be useful in allosteric virtual screening and the structural optimization of allosteric agonists/antagonists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvCDDNX-XiybVg90H21EOLACvtfcHk0lhpFegeSq8Gig"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhslWku7rE&md5=9f5fef0132358b98e1a4ce13673dd15a</span></div><a href="/servlet/linkout?suffix=cit141&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtw036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtw036%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DSu%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DAlloscore%253A%2520A%2520Method%2520for%2520Predicting%2520Allosteric%2520Ligand%25E2%2580%2593protein%2520Interactions%26jtitle%3DBioinformatics%26date%3D2016%26volume%3D32%26spage%3D1574%26epage%3D1576%26doi%3D10.1093%2Fbioinformatics%2Fbtw036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref142"><div class="reference"><strong class="refLabel"><a href="#ref142" class="rightTabRefNumLink">142</a></strong><div class="NLM_citation" id="rightTab-cit142"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span> <span> </span><span class="NLM_article-title">CavityPlus: A Web Server for Protein Cavity Detection with Pharmacophore Modelling, Allosteric Site Identification and Covalent Ligand Binding Ability Prediction</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>46</i></span>,  <span class="NLM_fpage">W374</span>– <span class="NLM_lpage">W379</span>, <span class="refDoi"> DOI: 10.1093/nar/gky380</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1093%2Fnar%2Fgky380" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29750256" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref142/cit142&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1MXosVyrs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2018&pages=W374-W379&author=Y.+Xuauthor=S.+Wangauthor=Q.+Huauthor=S.+Gaoauthor=X.+Maauthor=W.+Zhangauthor=Y.+Shenauthor=F.+Chenauthor=L.+Laiauthor=J.+Pei&title=CavityPlus%3A+A+Web+Server+for+Protein+Cavity+Detection+with+Pharmacophore+Modelling%2C+Allosteric+Site+Identification+and+Covalent+Ligand+Binding+Ability+Prediction&doi=10.1093%2Fnar%2Fgky380"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit142R"><div class="casContent"><span class="casTitleNuber">142</span><div class="casTitle"><span class="NLM_cas:atitle">CavityPlus: a web server for protein cavity detection with pharmacophore modelling, allosteric site identification and covalent ligand binding ability prediction</span></div><div class="casAuthors">Xu, Youjun; Wang, Shiwei; Hu, Qiwan; Gao, Shuaishi; Ma, Xiaomin; Zhang, Weilin; Shen, Yihang; Chen, Fangjin; Lai, Luhua; Pei, Jianfeng</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">W1</span>),
    <span class="NLM_cas:pages">W374-W379</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">1362-4962</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">CavityPlus is a web server that offers protein cavity detection and various functional analyses.  Using protein three-dimensional structural information as the input, CavityPlus applies CAVITY to detect potential binding sites on the surface of a given protein structure and rank them based on ligandability and druggability scores.  These potential binding sites can be further analyzed using three submodules, CavPharmer, CorrSite, and CovCys.  CavPharmer uses a receptor-based pharmacophore modeling program, Pocket, to automatically ext. pharmacophore features within cavities.  CorrSite identifies potential allosteric ligand-binding sites based on motion correlation analyses between cavities.  CovCys automatically detects druggable cysteine residues, which is esp. useful to identify novel binding sites for designing covalent allosteric ligands.  Overall, CavityPlus provides an integrated platform for analyzing comprehensive properties of protein binding cavities.  Such analyses are useful for many aspects of drug design and discovery, including target selection and identification, virtual screening, de novo drug design, and allosteric and covalent-binding drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcK4njo3ri1LVg90H21EOLACvtfcHk0liKSm2pfpgEkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXosVyrs7k%253D&md5=9825bc2fb3ae8a462e9b31889338b770</span></div><a href="/servlet/linkout?suffix=cit142&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgky380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgky380%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DGao%26aufirst%3DS.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DShen%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DLai%26aufirst%3DL.%26aulast%3DPei%26aufirst%3DJ.%26atitle%3DCavityPlus%253A%2520A%2520Web%2520Server%2520for%2520Protein%2520Cavity%2520Detection%2520with%2520Pharmacophore%2520Modelling%252C%2520Allosteric%2520Site%2520Identification%2520and%2520Covalent%2520Ligand%2520Binding%2520Ability%2520Prediction%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2018%26volume%3D46%26spage%3DW374%26epage%3DW379%26doi%3D10.1093%2Fnar%2Fgky380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref143"><div class="reference"><strong class="refLabel"><a href="#ref143" class="rightTabRefNumLink">143</a></strong><div class="NLM_citation" id="rightTab-cit143"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Smith, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carlson, H. A.</span></span> <span> </span><span class="NLM_article-title">Are There Physicochemical Differences between Allosteric and Competitive Ligands?</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>13</i></span>, <span class="NLM_elocation-id">e1005813</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1005813</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1005813" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29125840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref143/cit143&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Wjs7%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2017&author=R.+D.+Smithauthor=J.+Luauthor=H.+A.+Carlson&title=Are+There+Physicochemical+Differences+between+Allosteric+and+Competitive+Ligands%3F&doi=10.1371%2Fjournal.pcbi.1005813"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit143R"><div class="casContent"><span class="casTitleNuber">143</span><div class="casTitle"><span class="NLM_cas:atitle">Are there physicochemical differences between allosteric and competitive ligands?</span></div><div class="casAuthors">Smith, Richard D.; Lu, Jing; Carlson, Heather A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">e1005813/1-e1005813/18</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Previous studies have compared the physicochem. properties of allosteric compds. to non-allosteric compds.  Those studies have found that allosteric compds. tend to be smaller, more rigid, more hydrophobic, and more drug-like than non-allosteric compds.  However, previous studies have not properly cor. for the fact that some protein targets have much more data than other systems.  This generates concern regarding the possible skew that can be introduced by the inherent bias in the available data.  Hence, this study aims to det. how robust the previous findings are to the addn. of newer data.  This study utilizes the Allosteric Database (ASD v3.0) and ChEMBL v20 to systematically obtain large datasets of both allosteric and competitive ligands.  This dataset contains 70,219 and 9,511 unique ligands for the allosteric and competitive sets, resp.  Phys. relevant compd. descriptors were computed to examine the differences in their chem. properties.  Particular attention was given to removing redundancy in the data and normalizing across ligand diversity and varied protein targets.  The resulting distributions only show that allosteric ligands tend to be more arom. and rigid and do not confirm the increase in hydrophobicity or difference in drug-likeness.  These results are robust across different normalization schemes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdXKH3FrE5ObVg90H21EOLACvtfcHk0liKSm2pfpgEkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Wjs7%252FN&md5=629689f223c65b9fa351b7dfbdf45d98</span></div><a href="/servlet/linkout?suffix=cit143&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1005813&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1005813%26sid%3Dliteratum%253Aachs%26aulast%3DSmith%26aufirst%3DR.%2BD.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DCarlson%26aufirst%3DH.%2BA.%26atitle%3DAre%2520There%2520Physicochemical%2520Differences%2520between%2520Allosteric%2520and%2520Competitive%2520Ligands%253F%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2017%26volume%3D13%26doi%3D10.1371%2Fjournal.pcbi.1005813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref144"><div class="reference"><strong class="refLabel"><a href="#ref144" class="rightTabRefNumLink">144</a></strong><div class="NLM_citation" id="rightTab-cit144"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beauharnois, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolívar, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welch, J. T.</span></span> <span> </span><span class="NLM_article-title">Sirtuin 6: A Review of Biological Effects and Potential Therapeutic Properties</span>. <i>Mol. BioSyst.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1789</span>– <span class="NLM_lpage">1806</span>, <span class="refDoi"> DOI: 10.1039/c3mb00001j</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1039%2Fc3mb00001j" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23592245" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref144/cit144&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXosl2qsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=1789-1806&author=J.+M.+Beauharnoisauthor=B.+E.+Bol%C3%ADvarauthor=J.+T.+Welch&title=Sirtuin+6%3A+A+Review+of+Biological+Effects+and+Potential+Therapeutic+Properties&doi=10.1039%2Fc3mb00001j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit144R"><div class="casContent"><span class="casTitleNuber">144</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin 6: a review of biological effects and potential therapeutic properties</span></div><div class="casAuthors">Beauharnois, Jade M.; Bolivar, Beatriz E.; Welch, John T.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1789-1806</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">A review.  Sirtuins, possessing either histone deacetylase or mono-ribosyltransferase activity, regulate important pathways in bacteria, archaea and eukaryotes.  SIRT6, an enzyme highly expressed in skeletal muscles, brain, heart, liver, and thymus, affects transcriptional regulation in a tissue-specific manner.  This enzyme has a two-domain structure that consists of a large Rossmann fold and a smaller and structurally more varied sequence contg. a Zn2+-binding motif.  The C-terminus is required for proper nuclear localization, while the N-terminus is important for chromatin assocn. and for intrinsic catalytic activity.  SIRT6 promotes resistance to DNA damage and oxidative stress, the principal defects assocd. with age-related diseases.  The modulation of aging and other metabolic functions by SIRT6 may be indicative of previously unrecognized regulatory systems in the cell.  The propensity of individual SIRT6 mols. to undergo intramol. mono-ADP-ribosylation, suggests this auto-ribosylation may contribute to the self-regulation of SIRT6 function.  Until recently, SIRT6 was an orphan enzyme whose catalytic activity and substrates were unclear.  It was known that, similar to the yeast Sir2 protein, human SIRT6 deacetylates histones and regulates DNA stability and repair; however, new mechanistic insights can be derived from the discovery of the highly substrate-specific histone deacetylase activity of SIRT6.  This deacetylase activity promotes proper chromatin function in several physiol. contexts, to include telomere and genome stabilization, gene expression and DNA repair.  By maintaining both the integrity and the expression of the mammalian genome, SIRT6 may help prevent cellular senescence.  Moreover, successful mol. modulation of SIRT6 activity may lead to the development of new chemotherapeutic modalities.  The action of SIRT6 is described in this review, with an emphasis on the cellular roles of the enzyme and the relation of those enzymic functions to human biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo2U7MB2fgid7Vg90H21EOLACvtfcHk0liKSm2pfpgEkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXosl2qsLY%253D&md5=d10bd8d38712a80aff92447fdfdd8587</span></div><a href="/servlet/linkout?suffix=cit144&amp;dbid=16384&amp;doi=10.1039%2Fc3mb00001j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc3mb00001j%26sid%3Dliteratum%253Aachs%26aulast%3DBeauharnois%26aufirst%3DJ.%2BM.%26aulast%3DBol%25C3%25ADvar%26aufirst%3DB.%2BE.%26aulast%3DWelch%26aufirst%3DJ.%2BT.%26atitle%3DSirtuin%25206%253A%2520A%2520Review%2520of%2520Biological%2520Effects%2520and%2520Potential%2520Therapeutic%2520Properties%26jtitle%3DMol.%2520BioSyst.%26date%3D2013%26volume%3D9%26spage%3D1789%26epage%3D1806%26doi%3D10.1039%2Fc3mb00001j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref145"><div class="reference"><strong class="refLabel"><a href="#ref145" class="rightTabRefNumLink">145</a></strong><div class="NLM_citation" id="rightTab-cit145"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tennen, R. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span> <span> </span><span class="NLM_article-title">Chromatin Regulation and Genome Maintenance by Mammalian SIRT6</span>. <i>Trends Biochem. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">39</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1016/j.tibs.2010.07.009</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tibs.2010.07.009" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=20729089" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref145/cit145&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFKgtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=39-46&author=R.+I.+Tennenauthor=K.+F.+Chua&title=Chromatin+Regulation+and+Genome+Maintenance+by+Mammalian+SIRT6&doi=10.1016%2Fj.tibs.2010.07.009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit145R"><div class="casContent"><span class="casTitleNuber">145</span><div class="casTitle"><span class="NLM_cas:atitle">Chromatin regulation and genome maintenance by mammalian SIRT6</span></div><div class="casAuthors">Tennen, Ruth I.; Chua, Katrin F.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">39-46</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Saccharomyces cerevisiae Sir2 is an NAD+-dependent histone deacetylase that links chromatin silencing to genomic stability, cellular metab. and lifespan regulation.  In mice, deficiency for the Sir2 family member SIRT6 leads to genomic instability, metabolic defects and degenerative pathologies assocd. with aging.  Until recently, SIRT6 was an orphan enzyme whose catalytic activity and substrates were unclear.  However, new mechanistic insights have come from the discovery that SIRT6 is a highly substrate-specific histone deacetylase that promotes proper chromatin function in several physiol. contexts, including telomere and genome stabilization, gene expression and DNA repair.  By maintaining both the integrity and the expression of the mammalian genome, SIRT6 thus serves several roles that parallel Sir2 function.  In this article, we review recent advances in understanding the mechanisms of SIRT6 action and their implications for human biol. and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkssgUQhRPUrVg90H21EOLACvtfcHk0lgDEyIHFJPRkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFKgtA%253D%253D&md5=ef3dd7d19bce27f143e83bd4432cbb90</span></div><a href="/servlet/linkout?suffix=cit145&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.07.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.07.009%26sid%3Dliteratum%253Aachs%26aulast%3DTennen%26aufirst%3DR.%2BI.%26aulast%3DChua%26aufirst%3DK.%2BF.%26atitle%3DChromatin%2520Regulation%2520and%2520Genome%2520Maintenance%2520by%2520Mammalian%2520SIRT6%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D39%26epage%3D46%26doi%3D10.1016%2Fj.tibs.2010.07.009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref146"><div class="reference"><strong class="refLabel"><a href="#ref146" class="rightTabRefNumLink">146</a></strong><div class="NLM_citation" id="rightTab-cit146"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chua, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, W. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gellon, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Franco, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mills, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hsu, J. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hong, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ford, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kennedy, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nunez, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bronson, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frendewey, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Auerbach, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valenzuela, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karow, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hottiger, M. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hursting, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barrett, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guarente, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mulligan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Demple, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yancopoulos, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alt, F. W.</span></span> <span> </span><span class="NLM_article-title">Genomic Instability and Aging-like Phenotype in the Absence of Mammalian SIRT6</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>124</i></span>,  <span class="NLM_fpage">315</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2005.11.044</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2005.11.044" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=16439206" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref146/cit146&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=124&publication_year=2006&pages=315-329&author=R.+Mostoslavskyauthor=K.+F.+Chuaauthor=D.+B.+Lombardauthor=W.+W.+Pangauthor=M.+R.+Fischerauthor=L.+Gellonauthor=P.+Liuauthor=G.+Mostoslavskyauthor=S.+Francoauthor=M.+M.+Murphyauthor=K.+D.+Millsauthor=P.+Patelauthor=J.+T.+Hsuauthor=A.+L.+Hongauthor=E.+Fordauthor=H.+L.+Chengauthor=C.+Kennedyauthor=N.+Nunezauthor=R.+Bronsonauthor=D.+Frendeweyauthor=W.+Auerbachauthor=D.+Valenzuelaauthor=M.+Karowauthor=M.+O.+Hottigerauthor=S.+Hurstingauthor=J.+C.+Barrettauthor=L.+Guarenteauthor=R.+Mulliganauthor=B.+Dempleauthor=G.+D.+Yancopoulosauthor=F.+W.+Alt&title=Genomic+Instability+and+Aging-like+Phenotype+in+the+Absence+of+Mammalian+SIRT6&doi=10.1016%2Fj.cell.2005.11.044"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit146R"><div class="casContent"><span class="casTitleNuber">146</span><div class="casTitle"><span class="NLM_cas:atitle">Genomic instability and aging-like phenotype in the absence of mammalian SIRT6</span></div><div class="casAuthors">Mostoslavsky, Raul; Chua, Katrin F.; Lombard, David B.; Pang, Wendy W.; Fischer, Miriam R.; Gellon, Lionel; Liu, Pingfang; Mostoslavsky, Gustavo; Franco, Sonia; Murphy, Michael M.; Mills, Kevin D.; Patel, Parin; Hsu, Joyce T.; Hong, Andrew L.; Ford, Ethan; Cheng, Hwei-Ling; Kennedy, Caitlin; Nunez, Nomeli; Bronson, Roderick; Frendewey, David; Auerbach, Wojtek; Valenzuela, David; Karow, Margaret; Hottiger, Michael O.; Hursting, Stephen; Barrett, J. Carl; Guarente, Leonard; Mulligan, Richard; Demple, Bruce; Yancopoulos, George D.; Alt, Frederick W.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">124</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">315-329</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Sir2 histone deacetylase functions as a chromatin silencer to regulate recombination, genomic stability, and aging in budding yeast.  Seven mammalian Sir2 homologs have been identified (SIRT1-SIRT7), and it has been speculated that some may have similar functions to Sir2.  Here, we demonstrate that SIRT6 is a nuclear, chromatin-assocd. protein that promotes resistance to DNA damage and suppresses genomic instability in mouse cells, in assocn. with a role in base excision repair (BER).  SIRT6-deficient mice are small and at 2-3 wk of age develop abnormalities that include profound lymphopenia, loss of s.c. fat, lordokyphosis, and severe metabolic defects, eventually dying at about 4 wk.  We conclude that one function of SIRT6 is to promote normal DNA repair, and that SIRT6 loss leads to abnormalities in mice that overlap with aging-assocd. degenerative processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr-iTboO5w_ObVg90H21EOLACvtfcHk0lgDEyIHFJPRkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1KktLc%253D&md5=14971074ed55f5382960b500370ecff2</span></div><a href="/servlet/linkout?suffix=cit146&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2005.11.044&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2005.11.044%26sid%3Dliteratum%253Aachs%26aulast%3DMostoslavsky%26aufirst%3DR.%26aulast%3DChua%26aufirst%3DK.%2BF.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DPang%26aufirst%3DW.%2BW.%26aulast%3DFischer%26aufirst%3DM.%2BR.%26aulast%3DGellon%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DMostoslavsky%26aufirst%3DG.%26aulast%3DFranco%26aufirst%3DS.%26aulast%3DMurphy%26aufirst%3DM.%2BM.%26aulast%3DMills%26aufirst%3DK.%2BD.%26aulast%3DPatel%26aufirst%3DP.%26aulast%3DHsu%26aufirst%3DJ.%2BT.%26aulast%3DHong%26aufirst%3DA.%2BL.%26aulast%3DFord%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DH.%2BL.%26aulast%3DKennedy%26aufirst%3DC.%26aulast%3DNunez%26aufirst%3DN.%26aulast%3DBronson%26aufirst%3DR.%26aulast%3DFrendewey%26aufirst%3DD.%26aulast%3DAuerbach%26aufirst%3DW.%26aulast%3DValenzuela%26aufirst%3DD.%26aulast%3DKarow%26aufirst%3DM.%26aulast%3DHottiger%26aufirst%3DM.%2BO.%26aulast%3DHursting%26aufirst%3DS.%26aulast%3DBarrett%26aufirst%3DJ.%2BC.%26aulast%3DGuarente%26aufirst%3DL.%26aulast%3DMulligan%26aufirst%3DR.%26aulast%3DDemple%26aufirst%3DB.%26aulast%3DYancopoulos%26aufirst%3DG.%2BD.%26aulast%3DAlt%26aufirst%3DF.%2BW.%26atitle%3DGenomic%2520Instability%2520and%2520Aging-like%2520Phenotype%2520in%2520the%2520Absence%2520of%2520Mammalian%2520SIRT6%26jtitle%3DCell%26date%3D2006%26volume%3D124%26spage%3D315%26epage%3D329%26doi%3D10.1016%2Fj.cell.2005.11.044" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref147"><div class="reference"><strong class="refLabel"><a href="#ref147" class="rightTabRefNumLink">147</a></strong><div class="NLM_citation" id="rightTab-cit147"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kanfi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Naiman, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amir, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peshti, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zinman, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nahum, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bar-Joseph, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, H. Y.</span></span> <span> </span><span class="NLM_article-title">The Sirtuin SIRT6 Regulates Lifespan in Male Mice</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>483</i></span>,  <span class="NLM_fpage">218</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1038/nature10815</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature10815" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22367546" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref147/cit147&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=218-221&author=Y.+Kanfiauthor=S.+Naimanauthor=G.+Amirauthor=V.+Peshtiauthor=G.+Zinmanauthor=L.+Nahumauthor=Z.+Bar-Josephauthor=H.+Y.+Cohen&title=The+Sirtuin+SIRT6+Regulates+Lifespan+in+Male+Mice&doi=10.1038%2Fnature10815"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit147R"><div class="casContent"><span class="casTitleNuber">147</span><div class="casTitle"><span class="NLM_cas:atitle">The sirtuin SIRT6 regulates lifespan in male mice</span></div><div class="casAuthors">Kanfi, Yariv; Naiman, Shoshana; Amir, Gail; Peshti, Victoria; Zinman, Guy; Nahum, Liat; Bar-Joseph, Ziv; Cohen, Haim Y.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7388</span>),
    <span class="NLM_cas:pages">218-221</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The significant increase in human lifespan during the past century confronts us with great medical challenges.  To meet these challenges, the mechanisms that det. healthy ageing must be understood and controlled.  Sirtuins are highly conserved deacetylases that have been shown to regulate lifespan in yeast, nematodes and fruitflies.  However, the role of sirtuins in regulating worm and fly lifespan has recently become controversial.  Moreover, the role of the seven mammalian sirtuins, SIRT1 to SIRT7 (homologues of the yeast sirtuin Sir2), in regulating lifespan is unclear.  Here we show that male, but not female, transgenic mice overexpressing Sirt6 (ref. 4) have a significantly longer lifespan than wild-type mice.  Gene expression anal. revealed significant differences between male Sirt6-transgenic mice and male wild-type mice: transgenic males displayed lower serum levels of insulin-like growth factor 1 (IGF1), higher levels of IGF-binding protein 1 and altered phosphorylation levels of major components of IGF1 signalling, a key pathway in the regulation of lifespan.  This study shows the regulation of mammalian lifespan by a sirtuin family member and has important therapeutic implications for age-related diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo08gRkOlbJDrVg90H21EOLACvtfcHk0lgDEyIHFJPRkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XislyntbY%253D&md5=7f87716fd9e336558e1137081146f351</span></div><a href="/servlet/linkout?suffix=cit147&amp;dbid=16384&amp;doi=10.1038%2Fnature10815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature10815%26sid%3Dliteratum%253Aachs%26aulast%3DKanfi%26aufirst%3DY.%26aulast%3DNaiman%26aufirst%3DS.%26aulast%3DAmir%26aufirst%3DG.%26aulast%3DPeshti%26aufirst%3DV.%26aulast%3DZinman%26aufirst%3DG.%26aulast%3DNahum%26aufirst%3DL.%26aulast%3DBar-Joseph%26aufirst%3DZ.%26aulast%3DCohen%26aufirst%3DH.%2BY.%26atitle%3DThe%2520Sirtuin%2520SIRT6%2520Regulates%2520Lifespan%2520in%2520Male%2520Mice%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D218%26epage%3D221%26doi%3D10.1038%2Fnature10815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref148"><div class="reference"><strong class="refLabel"><a href="#ref148" class="rightTabRefNumLink">148</a></strong><div class="NLM_citation" id="rightTab-cit148"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sebastián, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwaans, B. M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silberman, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gymrek, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goren, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ram, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Truelove, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guimaraes, A. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Toiber, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cosentino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenson, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacDonald, A. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McGlynn, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edwards, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giacosa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weissleder, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernstein, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Regev, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shiels, P. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lombard, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mostoslavsky, R.</span></span> <span> </span><span class="NLM_article-title">The Histone Deacetylase SIRT6 Is a Tumor Suppressor That Controls Cancer Metabolism</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>151</i></span>,  <span class="NLM_fpage">1185</span>– <span class="NLM_lpage">1199</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2012.10.047</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2012.10.047" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23217706" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref148/cit148&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=2012&pages=1185-1199&author=C.+Sebasti%C3%A1nauthor=B.+M.+M.+Zwaansauthor=D.+M.+Silbermanauthor=M.+Gymrekauthor=A.+Gorenauthor=L.+Zhongauthor=O.+Ramauthor=J.+Trueloveauthor=A.+R.+Guimaraesauthor=D.+Toiberauthor=C.+Cosentinoauthor=J.+K.+Greensonauthor=A.+I.+MacDonaldauthor=L.+McGlynnauthor=F.+Maxwellauthor=J.+Edwardsauthor=S.+Giacosaauthor=E.+Guccioneauthor=R.+Weisslederauthor=B.+E.+Bernsteinauthor=A.+Regevauthor=P.+G.+Shielsauthor=D.+B.+Lombardauthor=R.+Mostoslavsky&title=The+Histone+Deacetylase+SIRT6+Is+a+Tumor+Suppressor+That+Controls+Cancer+Metabolism&doi=10.1016%2Fj.cell.2012.10.047"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit148R"><div class="casContent"><span class="casTitleNuber">148</span><div class="casTitle"><span class="NLM_cas:atitle">The histone deacetylase SIRT6 is a tumor suppressor that controls cancer metabolism</span></div><div class="casAuthors">Sebastian, Carlos; Zwaans, Bernadette M. M.; Silberman, Dafne M.; Gymrek, Melissa; Goren, Alon; Zhong, Lei; Ram, Oren; Truelove, Jessica; Guimaraes, Alexander R.; Toiber, Debra; Cosentino, Claudia; Greenson, Joel K.; MacDonald, Alasdair I.; McGlynn, Liane; Maxwell, Fraser; Edwards, Joanne; Giacosa, Sofia; Guccione, Ernesto; Weissleder, Ralph; Bernstein, Bradley E.; Regev, Aviv; Shiels, Paul G.; Lombard, David B.; Mostoslavsky, Raul</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">151</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1185-1199</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Reprogramming of cellular metab. is a key event during tumorigenesis.  Despite being known for decades (Warburg effect), the mol. mechanisms regulating this switch remained unexplored.  Here, we identify SIRT6 as a tumor suppressor that regulates aerobic glycolysis in cancer cells.  Importantly, loss of SIRT6 leads to tumor formation without activation of known oncogenes, whereas transformed SIRT6-deficient cells display increased glycolysis and tumor growth, suggesting that SIRT6 plays a role in both establishment and maintenance of cancer.  By using a conditional SIRT6 allele, we show that SIRT6 deletion in vivo increases the no., size, and aggressiveness of tumors.  SIRT6 also functions as a regulator of ribosome metab. by corepressing MYC transcriptional activity.  Lastly, Sirt6 is selectively downregulated in several human cancers, and expression levels of SIRT6 predict prognosis and tumor-free survival rates, highlighting SIRT6 as a crit. modulator of cancer metab.  Our studies reveal SIRT6 to be a potent tumor suppressor acting to suppress cancer metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6UVcycnyph7Vg90H21EOLACvtfcHk0li7tTWOuEzB-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvVaisr3J&md5=bb6df5c1d96df3ee5809773f91bb7ef5</span></div><a href="/servlet/linkout?suffix=cit148&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2012.10.047&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2012.10.047%26sid%3Dliteratum%253Aachs%26aulast%3DSebasti%25C3%25A1n%26aufirst%3DC.%26aulast%3DZwaans%26aufirst%3DB.%2BM.%2BM.%26aulast%3DSilberman%26aufirst%3DD.%2BM.%26aulast%3DGymrek%26aufirst%3DM.%26aulast%3DGoren%26aufirst%3DA.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DRam%26aufirst%3DO.%26aulast%3DTruelove%26aufirst%3DJ.%26aulast%3DGuimaraes%26aufirst%3DA.%2BR.%26aulast%3DToiber%26aufirst%3DD.%26aulast%3DCosentino%26aufirst%3DC.%26aulast%3DGreenson%26aufirst%3DJ.%2BK.%26aulast%3DMacDonald%26aufirst%3DA.%2BI.%26aulast%3DMcGlynn%26aufirst%3DL.%26aulast%3DMaxwell%26aufirst%3DF.%26aulast%3DEdwards%26aufirst%3DJ.%26aulast%3DGiacosa%26aufirst%3DS.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DWeissleder%26aufirst%3DR.%26aulast%3DBernstein%26aufirst%3DB.%2BE.%26aulast%3DRegev%26aufirst%3DA.%26aulast%3DShiels%26aufirst%3DP.%2BG.%26aulast%3DLombard%26aufirst%3DD.%2BB.%26aulast%3DMostoslavsky%26aufirst%3DR.%26atitle%3DThe%2520Histone%2520Deacetylase%2520SIRT6%2520Is%2520a%2520Tumor%2520Suppressor%2520That%2520Controls%2520Cancer%2520Metabolism%26jtitle%3DCell%26date%3D2012%26volume%3D151%26spage%3D1185%26epage%3D1199%26doi%3D10.1016%2Fj.cell.2012.10.047" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref149"><div class="reference"><strong class="refLabel"><a href="#ref149" class="rightTabRefNumLink">149</a></strong><div class="NLM_citation" id="rightTab-cit149"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schiedel, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robaa, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rumpf, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sippl, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jung, M.</span></span> <span> </span><span class="NLM_article-title">The Current State of NAD+ -Dependent Histone Deacetylases (Sirtuins) as Novel Therapeutic Targets</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">147</span>– <span class="NLM_lpage">200</span>, <span class="refDoi"> DOI: 10.1002/med.21436</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fmed.21436" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28094444" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref149/cit149&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvFKitbrO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2018&pages=147-200&author=M.+Schiedelauthor=D.+Robaaauthor=T.+Rumpfauthor=W.+Sipplauthor=M.+Jung&title=The+Current+State+of+NAD%2B+-Dependent+Histone+Deacetylases+%28Sirtuins%29+as+Novel+Therapeutic+Targets&doi=10.1002%2Fmed.21436"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit149R"><div class="casContent"><span class="casTitleNuber">149</span><div class="casTitle"><span class="NLM_cas:atitle">The Current State of NAD+-Dependent Histone Deacetylases (Sirtuins) as Novel Therapeutic Targets</span></div><div class="casAuthors">Schiedel, Matthias; Robaa, Dina; Rumpf, Tobias; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">147-200</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">A review.  Sirtuins are NAD+-dependent protein deacylases that cleave off acetyl, as well as other acyl groups, from the e-amino group of lysines in histones and other substrate proteins.  Seven sirtuin isotypes (Sirt1-7) have been identified in mammalian cells.  As sirtuins are involved in the regulation of various physiol. processes such as cell survival, cell cycle progression, apoptosis, DNA repair, cell metab., and caloric restriction, a dysregulation of their enzymic activity has been assocd. with the pathogenesis of neoplastic, metabolic, infectious, and neurodegenerative diseases.  Thus, sirtuins are promising targets for pharmaceutical intervention.  Growing interest in a modulation of sirtuin activity has prompted the discovery of several small mols., able to inhibit or activate certain sirtuin isotypes.  Herein, this paper gives an update to previous review on the topic in this journal (Schemies, 2010), focusing on recent developments in sirtuin biol., sirtuin modulators, and their potential as novel therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqt9mrksHom2rVg90H21EOLACvtfcHk0li7tTWOuEzB-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvFKitbrO&md5=eecf3180206a18d21ba42aa79710c89d</span></div><a href="/servlet/linkout?suffix=cit149&amp;dbid=16384&amp;doi=10.1002%2Fmed.21436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21436%26sid%3Dliteratum%253Aachs%26aulast%3DSchiedel%26aufirst%3DM.%26aulast%3DRobaa%26aufirst%3DD.%26aulast%3DRumpf%26aufirst%3DT.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DThe%2520Current%2520State%2520of%2520NAD%252B%2520-Dependent%2520Histone%2520Deacetylases%2520%2528Sirtuins%2529%2520as%2520Novel%2520Therapeutic%2520Targets%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2018%26volume%3D38%26spage%3D147%26epage%3D200%26doi%3D10.1002%2Fmed.21436" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref150"><div class="reference"><strong class="refLabel"><a href="#ref150" class="rightTabRefNumLink">150</a></strong><div class="NLM_citation" id="rightTab-cit150"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Fidalgo, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanchez-Hidalgo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alarcon de la Lastra, C.</span></span> <span> </span><span class="NLM_article-title">Sirtuin Modulators: Mechanisms and Potential Clinical Implications</span>. <i>Curr. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">2414</span>– <span class="NLM_lpage">2441</span>, <span class="refDoi"> DOI: 10.2174/092986712800269272</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.2174%2F092986712800269272" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22455589" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref150/cit150&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xms1egs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=2414-2441&author=S.+Sanchez-Fidalgoauthor=I.+Villegasauthor=M.+Sanchez-Hidalgoauthor=C.+Alarcon+de+la+Lastra&title=Sirtuin+Modulators%3A+Mechanisms+and+Potential+Clinical+Implications&doi=10.2174%2F092986712800269272"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit150R"><div class="casContent"><span class="casTitleNuber">150</span><div class="casTitle"><span class="NLM_cas:atitle">Sirtuin modulators: mechanisms and potential clinical implications</span></div><div class="casAuthors">Sanchez-Fidalgo, S.; Villegas, I.; Sanchez-Hidalgo, M.; Alarcon de la Lastra, C.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">2414-2441</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  In the last years, studies about longevity have highlighted that caloric restriction can be linked with a less normal aging-assocd. damage, and in the same way, with the activity of the Silent Information Regulator 2 (SIR2) gene.  Sir2-like genes, known as sirtuins (SIRTs), have been found in organisms ranging from bacteria to mammals promoting health and survival.  At the moment, it has been identified seven classes of SIRTs in mammalian and the understanding of many of them remains still rudimentary.  However, they are in the spotlight by their potential protection against aging-assocd. diseases and have emerged as key mediators of longevity in evolutionarily distant organisms models.  SIRTs are proteins found in numerous compartments within the cell, which are NAD+-dependent protein deacetylases and ADP (ADP)-ribosyltransferases.  They catalyze a reaction in which NAD+ and an acetylated substrate are converted into a deacetylated substrate, nicotinamide and a novel metabolite O-acetyl ADP ribose.  Therefore, its enzymic activity requires NAD+, which is a crucial mol. intermediary of many metabolic reactions in cells.  Basically, SIRTs are mediators of aging process, they have the potential of ameliorating and taking part in important cellular processes assocd., such as metabolic homeostasis, tumorigenesis and cancer cell proliferation, inflammatory disorders, cardiovascular diseases and neurodegeneration.  This background opens up new lines of investigation into the modulation of SIRTs activity in order to develop novel therapeutic targets to these age-related diseases.  Current expts. using mol. activators or inhibitors and genetically engineered animals have facilitated new insights into the role of these enzymes and contributed to highlight some of the potentially relevant targets.  This review is intended to provide an appreciation of the possible protection against aging-assocd. diseases by these enzymes, summarize novel underlying mechanisms and evaluate potential clin. applications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprcTt8kyEd_7Vg90H21EOLACvtfcHk0li7tTWOuEzB-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xms1egs7o%253D&md5=de1ca82bc810faa6ac84a5f54a10aeaa</span></div><a href="/servlet/linkout?suffix=cit150&amp;dbid=16384&amp;doi=10.2174%2F092986712800269272&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712800269272%26sid%3Dliteratum%253Aachs%26aulast%3DSanchez-Fidalgo%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DI.%26aulast%3DSanchez-Hidalgo%26aufirst%3DM.%26aulast%3DAlarcon%2Bde%2Bla%2BLastra%26aufirst%3DC.%26atitle%3DSirtuin%2520Modulators%253A%2520Mechanisms%2520and%2520Potential%2520Clinical%2520Implications%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D2414%26epage%3D2441%26doi%3D10.2174%2F092986712800269272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref151"><div class="reference"><strong class="refLabel"><a href="#ref151" class="rightTabRefNumLink">151</a></strong><div class="NLM_citation" id="rightTab-cit151"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crowe, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goldstein, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, J. L.</span></span> <span> </span><span class="NLM_article-title">STAT3 Inhibition, a Novel Approach to Enhancing Targeted Therapy in Human Cancers (Review)</span>. <i>Int. J. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">1181</span>– <span class="NLM_lpage">1191</span>, <span class="refDoi"> DOI: 10.3892/ijo.2012.1568</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3892%2Fijo.2012.1568" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22842992" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref151/cit151&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFyntLjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=1181-1191&author=X.+Wangauthor=P.+J.+Croweauthor=D.+Goldsteinauthor=J.+L.+Yang&title=STAT3+Inhibition%2C+a+Novel+Approach+to+Enhancing+Targeted+Therapy+in+Human+Cancers+%28Review%29&doi=10.3892%2Fijo.2012.1568"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit151R"><div class="casContent"><span class="casTitleNuber">151</span><div class="casTitle"><span class="NLM_cas:atitle">STAT3 inhibition, a novel approach to enhancing targeted therapy in human cancers (review)</span></div><div class="casAuthors">Wang, Xiaochun; Crowe, Philip J.; Goldstein, David; Yang, Jia-Lin</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Oncology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1181-1191</span>CODEN:
                <span class="NLM_cas:coden">IJONES</span>;
        ISSN:<span class="NLM_cas:issn">1019-6439</span>.
    
            (<span class="NLM_cas:orgname">International Journal of Oncology</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) regulates many crit. functions in human normal and malignant tissues, such as differentiation, proliferation, survival, angiogenesis and immune function.  Constitutive activation of STAT3 is implicated in a wide range of human cancers.  As such, STAT3 has been studied as a tumor therapeutic target.  This review aimed principally to summarize the updated research on STAT3 inhibition studies and their therapeutic potential in solid tumors.  Recent literature assocd. with STAT3 inhibition was reviewed through PubMed and Medline database, followed by crit. comparison and anal.  Constitutive activation of STAT3 has been identified as abnormal and oncogenic.  The pathway of STAT3 activation and signal transduction identifies 3 approaches for inhibition: modulating upstream pos. or neg. regulators, regulating RNA (DN-STAT3, anti-sense RNA, siRNA and microRNA) or targeting STAT3 protein at different domains.  The last approach using small mol. STAT3 inhibitors has been the most examd. so far with both preclin. and clin. studies.  Targeting STAT3 using a specific inhibitor may be a useful cancer treatment approach, with the potential for a broad clin. impact.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnx1B_eI0cs7Vg90H21EOLACvtfcHk0lil57gvyq5J8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFyntLjF&md5=91d64864c492ebe00e5d6a1cc6072284</span></div><a href="/servlet/linkout?suffix=cit151&amp;dbid=16384&amp;doi=10.3892%2Fijo.2012.1568&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijo.2012.1568%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DX.%26aulast%3DCrowe%26aufirst%3DP.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DJ.%2BL.%26atitle%3DSTAT3%2520Inhibition%252C%2520a%2520Novel%2520Approach%2520to%2520Enhancing%2520Targeted%2520Therapy%2520in%2520Human%2520Cancers%2520%2528Review%2529%26jtitle%3DInt.%2520J.%2520Oncol.%26date%3D2012%26volume%3D41%26spage%3D1181%26epage%3D1191%26doi%3D10.3892%2Fijo.2012.1568" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref152"><div class="reference"><strong class="refLabel"><a href="#ref152" class="rightTabRefNumLink">152</a></strong><div class="NLM_citation" id="rightTab-cit152"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, H. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, G.</span></span> <span> </span><span class="NLM_article-title">Feedback Activation of STAT3 as a Cancer Drug-Resistance Mechanism</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">61</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2015.10.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tips.2015.10.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26576830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref152/cit152&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslektr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=47-61&author=C.+Zhaoauthor=H.+Liauthor=H.+J.+Linauthor=S.+Yangauthor=J.+Linauthor=G.+Liang&title=Feedback+Activation+of+STAT3+as+a+Cancer+Drug-Resistance+Mechanism&doi=10.1016%2Fj.tips.2015.10.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit152R"><div class="casContent"><span class="casTitleNuber">152</span><div class="casTitle"><span class="NLM_cas:atitle">Feedback Activation of STAT3 as a Cancer Drug-Resistance Mechanism</span></div><div class="casAuthors">Zhao, Chengguang; Li, Huameng; Lin, Huey-Jen; Yang, Shulin; Lin, Jiayuh; Liang, Guang</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">47-61</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) plays crucial roles in several cellular processes such as cell proliferation and survival, and has been found to be aberrantly activated in many cancers.  Much research has explored the leading mechanisms for regulating the STAT3 pathway and its role in promoting tumorigenesis.  We focus here on recent evidence suggesting that feedback activation of STAT3 plays a prominent role in mediating drug resistance to a broad spectrum of targeted cancer therapies and chemotherapies.  We highlight the potential of co-targeting STAT3 and its primary target to overcome drug resistance, and provide perspective on repurposing clin. approved drugs as STAT3 pathway inhibitors, in combination with the FDA-approved receptor tyrosine kinase (RTK) inhibitors, to improve clin. outcome of cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrV-bWS3i24YLVg90H21EOLACvtfcHk0lil57gvyq5J8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslektr3J&md5=0fe954b8173ecbf8d855785ffd14bc0f</span></div><a href="/servlet/linkout?suffix=cit152&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2015.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2015.10.001%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DH.%2BJ.%26aulast%3DYang%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.%26aulast%3DLiang%26aufirst%3DG.%26atitle%3DFeedback%2520Activation%2520of%2520STAT3%2520as%2520a%2520Cancer%2520Drug-Resistance%2520Mechanism%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2016%26volume%3D37%26spage%3D47%26epage%3D61%26doi%3D10.1016%2Fj.tips.2015.10.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref153"><div class="reference"><strong class="refLabel"><a href="#ref153" class="rightTabRefNumLink">153</a></strong><div class="NLM_citation" id="rightTab-cit153"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sgrignani, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garofalo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkovic, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Merulla, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Catapano, C. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavalli, A.</span></span> <span> </span><span class="NLM_article-title">Structural Biology of STAT3 and Its Implications for Anticancer Therapies Development</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1591</span>, <span class="refDoi"> DOI: 10.3390/ijms19061591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms19061591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref153/cit153&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXisFSrt73O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=1591&author=J.+Sgrignaniauthor=M.+Garofaloauthor=M.+Matkovicauthor=J.+Merullaauthor=C.+V.+Catapanoauthor=A.+Cavalli&title=Structural+Biology+of+STAT3+and+Its+Implications+for+Anticancer+Therapies+Development&doi=10.3390%2Fijms19061591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit153R"><div class="casContent"><span class="casTitleNuber">153</span><div class="casTitle"><span class="NLM_cas:atitle">Structural biology of STAT3 and its implications for anticancer therapies development</span></div><div class="casAuthors">Sgrignani, Jacopo; Garofalo, Maura; Matkovic, Milos; Merulla, Jessica; Catapano, Carlo V.; Cavalli, Andrea</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1591/1-1591/14</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Transcription factors are proteins able to bind DNA and induce the transcription of specific genes.  Consequently, they play a pivotal role in multiple cellular pathways and are frequently over-expressed or dysregulated in cancer.  Here, we will focus on a specific "signal transducer and activator of transcription" (STAT3) factor that is involved in several pathologies, including cancer.  For long time, the mechanism by which STAT3 exerts its cellular functions has been summarized by a three steps process: (1) Protein phosphorylation by specific kinases, (2) dimerization promoted by phosphorylation, (3) activation of gene expression by the phosphorylated dimer.  Consequently, most of the inhibitors reported in literature aimed at blocking phosphorylation and dimerization.  However, recent observations reopened the debate and the entire functional mechanism has been revisited stimulating the scientific community to pursue new inhibition strategies.  In particular, the dimerization of the unphosphorylated species has been exptl. demonstrated and specific roles proposed also for these dimers.  Despite difficulties in the expression and purifn. of the full length STAT3, structural biol. investigations allowed the detn. of atomistic structures of STAT3 dimers and several protein domains.  Starting from this information, computational methods have been used both to improve the understanding of the STAT3 functional mechanism and to design new inhibitors to be used as anticancer drugs.  In this review, we will focus on the contribution of structural biol. to understand the roles of STAT3, to design new inhibitors and to suggest new strategies of pharmacol. intervention.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwOe7is1W8ZLVg90H21EOLACvtfcHk0lil57gvyq5J8w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXisFSrt73O&md5=5144a31d327b9fd68e0956f8f2dffeeb</span></div><a href="/servlet/linkout?suffix=cit153&amp;dbid=16384&amp;doi=10.3390%2Fijms19061591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19061591%26sid%3Dliteratum%253Aachs%26aulast%3DSgrignani%26aufirst%3DJ.%26aulast%3DGarofalo%26aufirst%3DM.%26aulast%3DMatkovic%26aufirst%3DM.%26aulast%3DMerulla%26aufirst%3DJ.%26aulast%3DCatapano%26aufirst%3DC.%2BV.%26aulast%3DCavalli%26aufirst%3DA.%26atitle%3DStructural%2520Biology%2520of%2520STAT3%2520and%2520Its%2520Implications%2520for%2520Anticancer%2520Therapies%2520Development%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D1591%26doi%3D10.3390%2Fijms19061591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref154"><div class="reference"><strong class="refLabel"><a href="#ref154" class="rightTabRefNumLink">154</a></strong><div class="NLM_citation" id="rightTab-cit154"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Debnath, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Neamati, N.</span></span> <span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (stat3) Protein</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">6645</span>– <span class="NLM_lpage">6648</span>, <span class="refDoi"> DOI: 10.1021/jm300207s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300207s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref154/cit154&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6645-6648&author=B.+Debnathauthor=S.+Xuauthor=N.+Neamati&title=Small+Molecule+Inhibitors+of+Signal+Transducer+and+Activator+of+Transcription+3+%28stat3%29+Protein&doi=10.1021%2Fjm300207s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit154R"><div class="casContent"><span class="casTitleNuber">154</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecule Inhibitors of Signal Transducer and Activator of Transcription 3 (Stat3) Protein</span></div><div class="casAuthors">Debnath, Bikash; Xu, Shili; Neamati, Nouri</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6645-6668</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (Stat3) is a transcription factor belonging to the STAT protein family, members of which regulate a wide variety of biol. processes, including cell growth, proliferation, survival, differentiation, development, movement and inflammation in response to extracellular signaling mols. such as cytokines and growth factors.  Receptor assocd. tyrosine kinases phosphorylate Stat3 protein at Tyr705 and enable it to dimerize and subsequently enter the nucleus where it activates the transcription of its target genes.  Accumulating evidence has established Stat3's role in oncogenesis, cardiovascular diseases, rheumatoid arthritis, fibrotic, and Alzheimer's diseases.  Thus, Stat3 inhibition is a promising therapeutic approach not only for cancer, but also for other diseases.  Although Stat3 has been considered as an important therapeutic target, progress in discovery and development of effective and specific Stat3 inhibitors remains slow.  In this article we summarize Stat3 inhibitors published between 2006 and 2011.  New strategies along with novel technologies are required to achieve breakthrough in the development of Stat3 inhibitors that can be translated into clin. use.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWWQl9gV1-ZLVg90H21EOLACvtfcHk0lgHwnmz0i3VFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xnsl2msrc%253D&md5=f5f3ac9f8fc2326c75edd2a9c136275b</span></div><a href="/servlet/linkout?suffix=cit154&amp;dbid=16384&amp;doi=10.1021%2Fjm300207s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300207s%26sid%3Dliteratum%253Aachs%26aulast%3DDebnath%26aufirst%3DB.%26aulast%3DXu%26aufirst%3DS.%26aulast%3DNeamati%26aufirst%3DN.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Signal%2520Transducer%2520and%2520Activator%2520of%2520Transcription%25203%2520%2528stat3%2529%2520Protein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6645%26epage%3D6648%26doi%3D10.1021%2Fjm300207s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref155"><div class="reference"><strong class="refLabel"><a href="#ref155" class="rightTabRefNumLink">155</a></strong><div class="NLM_citation" id="rightTab-cit155"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miklossy, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Youn, U. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilliard, T. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paladino, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sarkaria, J. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kawakami, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wongwiwatthananukit, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chang, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Turkson, J.</span></span> <span> </span><span class="NLM_article-title">Hirsutinolide Series Inhibit Stat3 Activity, Alter GCN1, MAP1B, Hsp105, G6PD, Vimentin, TrxR1, and Importin α-2 Expression, and Induce Antitumor Effects against Human Glioma</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">7734</span>– <span class="NLM_lpage">7748</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b00686</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00686" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref155/cit155&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVCgu7jN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=7734-7748&author=G.+Miklossyauthor=U.+J.+Younauthor=P.+Yueauthor=M.+Zhangauthor=C.+H.+Chenauthor=T.+S.+Hilliardauthor=D.+Paladinoauthor=Y.+Liauthor=J.+Choiauthor=J.+N.+Sarkariaauthor=J.+K.+Kawakamiauthor=S.+Wongwiwatthananukitauthor=Y.+Chenauthor=D.+Sunauthor=L.+C.+Changauthor=J.+Turkson&title=Hirsutinolide+Series+Inhibit+Stat3+Activity%2C+Alter+GCN1%2C+MAP1B%2C+Hsp105%2C+G6PD%2C+Vimentin%2C+TrxR1%2C+and+Importin+%CE%B1-2+Expression%2C+and+Induce+Antitumor+Effects+against+Human+Glioma&doi=10.1021%2Facs.jmedchem.5b00686"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit155R"><div class="casContent"><span class="casTitleNuber">155</span><div class="casTitle"><span class="NLM_cas:atitle">Hirsutinolide Series Inhibit Stat3 Activity, Alter GCN1, MAP1B, Hsp105, G6PD, Vimentin, TrxR1, and Importin α-2 Expression, and Induce Antitumor Effects against Human Glioma</span></div><div class="casAuthors">Miklossy, Gabriella; Youn, Ui Joung; Yue, Peibin; Zhang, Mingming; Chen, Chih-Hong; Hilliard, Tyvette S.; Paladino, David; Li, Yifei; Choi, Justin; Sarkaria, Jann N.; Kawakami, Joel K.; Wongwiwatthananukit, Supakit; Chen, Yuan; Sun, Dianqing; Chang, Leng Chee; Turkson, James</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7734-7748</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors report that hirsutinolide series and the semisynthetic analogs inhibit constitutively active signal transducer and activator of transcription (Stat)3 and malignant glioma phenotype.  A position 13 lipophilic ester group is required for activity.  Mol. modeling and NMR structural analyses reveal direct hirsutinolide:Stat3 binding.  One-hour treatment of cells with I and II also upregulated importin subunit α-2 levels and repressed translational activator GCN1, microtubule-assocd. protein (MAP)1B, thioredoxin reductase (TrxR)1 cytoplasmic isoform 3, glucose-6-phosphate 1-dehydrogenase isoform a, Hsp105, vimentin, and tumor necrosis factor α-induced protein (TNAP)2 expression.  Active hirsutinolides inhibited anchorage-dependent and three-dimensional spheroid growth, survival, and migration of human glioma lines and glioma patients' tumor-derived xenograft cells harboring constitutively active Stat3.  Oral gavage delivery of I or II inhibited human glioma tumor growth in s.c. mouse xenografts.  The inhibition of Stat3 signaling represents part of the hirsutinolide-mediated mechanisms to induce antitumor effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq-5VLBoRoafbVg90H21EOLACvtfcHk0lgHwnmz0i3VFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVCgu7jN&md5=65cfae17ba81a38ca5dd00e0c07265b4</span></div><a href="/servlet/linkout?suffix=cit155&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00686&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00686%26sid%3Dliteratum%253Aachs%26aulast%3DMiklossy%26aufirst%3DG.%26aulast%3DYoun%26aufirst%3DU.%2BJ.%26aulast%3DYue%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DC.%2BH.%26aulast%3DHilliard%26aufirst%3DT.%2BS.%26aulast%3DPaladino%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChoi%26aufirst%3DJ.%26aulast%3DSarkaria%26aufirst%3DJ.%2BN.%26aulast%3DKawakami%26aufirst%3DJ.%2BK.%26aulast%3DWongwiwatthananukit%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DL.%2BC.%26aulast%3DTurkson%26aufirst%3DJ.%26atitle%3DHirsutinolide%2520Series%2520Inhibit%2520Stat3%2520Activity%252C%2520Alter%2520GCN1%252C%2520MAP1B%252C%2520Hsp105%252C%2520G6PD%252C%2520Vimentin%252C%2520TrxR1%252C%2520and%2520Importin%2520%25CE%25B1-2%2520Expression%252C%2520and%2520Induce%2520Antitumor%2520Effects%2520against%2520Human%2520Glioma%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D7734%26epage%3D7748%26doi%3D10.1021%2Facs.jmedchem.5b00686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref156"><div class="reference"><strong class="refLabel"><a href="#ref156" class="rightTabRefNumLink">156</a></strong><div class="NLM_citation" id="rightTab-cit156"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Escobar, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bjartell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canesin, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans-Axelsson, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sterner, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hellsten, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johansson, M. H.</span></span> <span> </span><span class="NLM_article-title">Preclinical Characterization of 3β-(N-Acetyl L -Cysteine Methyl Ester)-2aβ,3-Dihydrogaliellalactone (GPA512), a Prodrug of a Direct STAT3 Inhibitor for the Treatment of Prostate Cancer</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">4551</span>– <span class="NLM_lpage">4562</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01814</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01814" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref156/cit156&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1Gqu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4551-4562&author=Z.+Escobarauthor=A.+Bjartellauthor=G.+Canesinauthor=S.+Evans-Axelssonauthor=O.+Sternerauthor=R.+Hellstenauthor=M.+H.+Johansson&title=Preclinical+Characterization+of+3%CE%B2-%28N-Acetyl+L+-Cysteine+Methyl+Ester%29-2a%CE%B2%2C3-Dihydrogaliellalactone+%28GPA512%29%2C+a+Prodrug+of+a+Direct+STAT3+Inhibitor+for+the+Treatment+of+Prostate+Cancer&doi=10.1021%2Facs.jmedchem.5b01814"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit156R"><div class="casContent"><span class="casTitleNuber">156</span><div class="casTitle"><span class="NLM_cas:atitle">Preclinical Characterization of 3β-(N-Acetyl L-cysteine methyl ester)-2aβ,3-dihydrogaliellalactone (GPA512), a Prodrug of a Direct STAT3 Inhibitor for the Treatment of Prostate Cancer</span></div><div class="casAuthors">Escobar, Zilma; Bjartell, Anders; Canesin, Giacomo; Evans-Axelsson, Susan; Sterner, Olov; Hellsten, Rebecka; Johansson, Martin H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4551-4562</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The transcription factor STAT3 is a potential target for the treatment of castration-resistant prostate cancer.  Galiellalactone (1), a direct inhibitor of STAT3, prevents the transcription of STAT3 regulated genes.  In this study we characterized 6, a prodrug of 1.  In vitro studies showed that 6 is rapidly converted to 1 in plasma and is stable in a buffer soln.  The pharmacokinetics of 6 following a single oral dose indicated that the prodrug was rapidly absorbed and converted to 1 with a tmax of 15 min.  Oral administration of 6 in mice increased the plasma exposure of the active parent compd. 20-fold compared to when 1 was dosed orally. 6 treated mice bearing DU145 xenograft tumors had significantly reduced tumor growth compared to untreated mice.  The favorable druglike properties and safety profile of 6 warrant further studies of 6 for the treatment of castration-resistant prostate cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZ-05xP9iOM7Vg90H21EOLACvtfcHk0lgHwnmz0i3VFA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1Gqu7g%253D&md5=1ed1bfdda22e882cbcdad28162d9c719</span></div><a href="/servlet/linkout?suffix=cit156&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01814&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01814%26sid%3Dliteratum%253Aachs%26aulast%3DEscobar%26aufirst%3DZ.%26aulast%3DBjartell%26aufirst%3DA.%26aulast%3DCanesin%26aufirst%3DG.%26aulast%3DEvans-Axelsson%26aufirst%3DS.%26aulast%3DSterner%26aufirst%3DO.%26aulast%3DHellsten%26aufirst%3DR.%26aulast%3DJohansson%26aufirst%3DM.%2BH.%26atitle%3DPreclinical%2520Characterization%2520of%25203%25CE%25B2-%2528N-Acetyl%2520L%2520-Cysteine%2520Methyl%2520Ester%2529-2a%25CE%25B2%252C3-Dihydrogaliellalactone%2520%2528GPA512%2529%252C%2520a%2520Prodrug%2520of%2520a%2520Direct%2520STAT3%2520Inhibitor%2520for%2520the%2520Treatment%2520of%2520Prostate%2520Cancer%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4551%26epage%3D4562%26doi%3D10.1021%2Facs.jmedchem.5b01814" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref157"><div class="reference"><strong class="refLabel"><a href="#ref157" class="rightTabRefNumLink">157</a></strong><div class="NLM_citation" id="rightTab-cit157"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xia, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, J.-y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Luo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, L.</span></span> <span> </span><span class="NLM_article-title">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2718</span>– <span class="NLM_lpage">2731</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01489</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01489" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref157/cit157&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=2718-2731&author=W.+Yuauthor=C.+Liauthor=W.+Zhangauthor=Y.+Xiaauthor=S.+Liauthor=J.-y.+Linauthor=K.+Yuauthor=M.+Liuauthor=L.+Yangauthor=J.+Luoauthor=Y.+Chenauthor=H.+Sunauthor=L.+Kong&title=Discovery+of+an+Orally+Selective+Inhibitor+of+Signal+Transducer+and+Activator+of+Transcription+3+Using+Advanced+Multiple+Ligand+Simultaneous+Docking&doi=10.1021%2Facs.jmedchem.6b01489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit157R"><div class="casContent"><span class="casTitleNuber">157</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an Orally Selective Inhibitor of Signal Transducer and Activator of Transcription 3 Using Advanced Multiple Ligand Simultaneous Docking</span></div><div class="casAuthors">Yu, Wenying; Li, Chenglong; Zhang, Wenda; Xia, Yuanzheng; Li, Shanshan; Lin, Jia-yuh; Yu, Keqin; Liu, Mu; Yang, Lei; Luo, Jianguang; Chen, Yijun; Sun, Hongbin; Kong, Lingyi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2718-2731</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeting signal transducer and activator of transcription 3 (STAT3) is a potential anticancer strategy.  However, STAT3 inhibitors with good selectivity and bioavailability are rare.  The aim of this study was to discover selective direct STAT3 inhibitors with good drug-likeness.  By advanced multiple ligand simultaneous docking (AMLSD) method, compd. 9 (5,8-dioxo-6-((2-(piperazin-1-yl)phenyl)amino)-5,8-dihydronaphthalene-1-sulfonamide) was designed as an orally-bioavailable STAT3 inhibitor, which presented superior druggability and selectivity to other representative STAT3 inhibitors.  Compd. 9 directly and selectively inhibited the pY705 site of STAT3 with an affinity (Ki) of 440nM.  The IC50 of 9 for breast cancer cell MDA-MB-231 was 184-fold lower than its IC50 for normal breast epithelial cell MCF-10A.  Compd. 9 in vivo induced significant antitumor responses better than Gefitinib and its therapeutic index should be over 100, indicating good safety of 9.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiT8F_tCrky7Vg90H21EOLACvtfcHk0lhNf2n7xucw8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXjsFalurw%253D&md5=d9554e49fd17a559e8c7171f36c53607</span></div><a href="/servlet/linkout?suffix=cit157&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01489%26sid%3Dliteratum%253Aachs%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DXia%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DLin%26aufirst%3DJ.-y.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DSun%26aufirst%3DH.%26aulast%3DKong%26aufirst%3DL.%26atitle%3DDiscovery%2520of%2520an%2520Orally%2520Selective%2520Inhibitor%2520of%2520Signal%2520Transducer%2520and%2520Activator%2520of%2520Transcription%25203%2520Using%2520Advanced%2520Multiple%2520Ligand%2520Simultaneous%2520Docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D2718%26epage%3D2731%26doi%3D10.1021%2Facs.jmedchem.6b01489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref158"><div class="reference"><strong class="refLabel"><a href="#ref158" class="rightTabRefNumLink">158</a></strong><div class="NLM_citation" id="rightTab-cit158"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J. T.</span></span> <span> </span><span class="NLM_article-title">Two Decades of Research in Discovery of Anticancer Drugs Targeting STAT3, How Close Are We?</span>. <i>Pharmacol. Ther.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>191</i></span>,  <span class="NLM_fpage">74</span>– <span class="NLM_lpage">91</span>, <span class="refDoi"> DOI: 10.1016/j.pharmthera.2018.06.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.pharmthera.2018.06.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29933035" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref158/cit158&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=191&publication_year=2018&pages=74-91&author=J.+D.+Beebeauthor=J.+Y.+Liuauthor=J.+T.+Zhang&title=Two+Decades+of+Research+in+Discovery+of+Anticancer+Drugs+Targeting+STAT3%2C+How+Close+Are+We%3F&doi=10.1016%2Fj.pharmthera.2018.06.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit158R"><div class="casContent"><span class="casTitleNuber">158</span><div class="casTitle"><span class="NLM_cas:atitle">Two decades of research in discovery of anticancer drugs targeting STAT3, how close are we?</span></div><div class="casAuthors">Beebe, Jenny D.; Liu, Jing-Yuan; Zhang, Jian-Ting</div><div class="citationInfo"><span class="NLM_cas:title">Pharmacology & Therapeutics</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">191</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">74-91</span>CODEN:
                <span class="NLM_cas:coden">PHTHDT</span>;
        ISSN:<span class="NLM_cas:issn">0163-7258</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">A review.  Signal transducer and activator of transcription 3 (STAT3) controls many biol. processes including differentiation, survival, proliferation, and angiogenesis.  In normal healthy cells, STAT3 is tightly regulated to maintain a momentary active state.  However, aberrant or constitutively activated STAT3 has been obsd. in many different cancers and constitutively activated STAT3 has been shown to assoc. with poor prognosis and tumor progression.  For this reason, STAT3 has been studied as a possible target in the treatment of many different types of cancers.  However, despite decades of research, a FDA-approved STAT3 inhibitor has yet to emerge.  In this review, we will analyze past studies targeting STAT3 for drug discovery, understand possible causes of failure in these studies, and provide potential insights for future efforts to overcome these roadblocks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqSWDPJOIaRKrVg90H21EOLACvtfcHk0lhNf2n7xucw8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtF2ltrfO&md5=048e7d95608bdab18335c55d084d62ab</span></div><a href="/servlet/linkout?suffix=cit158&amp;dbid=16384&amp;doi=10.1016%2Fj.pharmthera.2018.06.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.pharmthera.2018.06.006%26sid%3Dliteratum%253Aachs%26aulast%3DBeebe%26aufirst%3DJ.%2BD.%26aulast%3DLiu%26aufirst%3DJ.%2BY.%26aulast%3DZhang%26aufirst%3DJ.%2BT.%26atitle%3DTwo%2520Decades%2520of%2520Research%2520in%2520Discovery%2520of%2520Anticancer%2520Drugs%2520Targeting%2520STAT3%252C%2520How%2520Close%2520Are%2520We%253F%26jtitle%3DPharmacol.%2520Ther.%26date%3D2018%26volume%3D191%26spage%3D74%26epage%3D91%26doi%3D10.1016%2Fj.pharmthera.2018.06.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref159"><div class="reference"><strong class="refLabel"><a href="#ref159" class="rightTabRefNumLink">159</a></strong><div class="NLM_citation" id="rightTab-cit159"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Trott, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span> <span> </span><span class="NLM_article-title">AutoDock Vina: Improving the Speed and Accuracy of Docking with a New Scoring Function, Efficient Optimization, and Multithreading</span>. <i>J. Comput. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">455</span>– <span class="NLM_lpage">461</span>, <span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref159/cit159&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-461&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+Improving+the+Speed+and+Accuracy+of+Docking+with+a+New+Scoring+Function%2C+Efficient+Optimization%2C+and+Multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit159R"><div class="casContent"><span class="casTitleNuber">159</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhNf2n7xucw8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit159&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520Improving%2520the%2520Speed%2520and%2520Accuracy%2520of%2520Docking%2520with%2520a%2520New%2520Scoring%2520Function%252C%2520Efficient%2520Optimization%252C%2520and%2520Multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D461%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref160"><div class="reference"><strong class="refLabel"><a href="#ref160" class="rightTabRefNumLink">160</a></strong><div class="NLM_citation" id="rightTab-cit160"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Savaskan, N. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ufer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kühn, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchert, A.</span></span> <span> </span><span class="NLM_article-title">Molecular Biology of Glutathione Peroxidase 4: From Genomic Structure to Developmental Expression and Neural Function</span>. <i>Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>388</i></span>,  <span class="NLM_fpage">1007</span>– <span class="NLM_lpage">1017</span>, <span class="refDoi"> DOI: 10.1515/BC.2007.126</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1515%2FBC.2007.126" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17937614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref160/cit160&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtFCntrfK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=388&publication_year=2007&pages=1007-1017&author=N.+E.+Savaskanauthor=C.+Uferauthor=H.+K%C3%BChnauthor=A.+Borchert&title=Molecular+Biology+of+Glutathione+Peroxidase+4%3A+From+Genomic+Structure+to+Developmental+Expression+and+Neural+Function&doi=10.1515%2FBC.2007.126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit160R"><div class="casContent"><span class="casTitleNuber">160</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular biology of glutathione peroxidase 4: from genomic structure to developmental expression and neural function</span></div><div class="casAuthors">Savaskan, Nicolai E.; Ufer, Christoph; Kuehn, Hartmut; Borchert, Astrid</div><div class="citationInfo"><span class="NLM_cas:title">Biological Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">388</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1007-1017</span>CODEN:
                <span class="NLM_cas:coden">BICHF3</span>;
        ISSN:<span class="NLM_cas:issn">1431-6730</span>.
    
            (<span class="NLM_cas:orgname">Walter de Gruyter GmbH & Co. KG</span>)
        </div><div class="casAbstract">A review.  Selenoproteins have been recognized as modulators of brain function and signaling.  Phospholipid hydroperoxide glutathione peroxidase (GPx4/PHGPx) is a unique member of the selenium-dependent glutathione peroxidases in mammals with a pivotal role in brain development and function.  GPx4 exists as a cytosolic, mitochondrial, and nuclear isoform derived from a single gene.  In mice, the GPx4 gene is located on chromosome 10 in close proximity to a functional retrotransposome that is expressed under the control of captured regulatory elements.  Elucidation of crystallog. data uncovered structural peculiarities of GPx4 that provide the mol. basis for its unique enzymic properties and substrate specificity.  Monomeric GPx4 is multifunctional: it acts as a reducing enzyme of peroxidized phospholipids and thiols and as a structural protein.  Transcriptional regulation of the different GPx4 isoforms requires several isoform-specific cis-regulatory sequences and trans-activating factors.  Cytosolic and mitochondrial GPx4 are the major isoforms exclusively expressed by neurons in the developing brain.  In stark contrast, following brain trauma, GPx4 is specifically upregulated in non-neuronal cells, i.e., reactive astrocytes.  Mol. approaches to genetic modification in mice have revealed an essential and isoform-specific function for GPx4 in development and disease.  Here we review recent findings on GPx4 with emphasis on its mol. structure and function and consider potential mechanisms that underlie neural development and neuropathol. conditions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov7VCeQGwYIbVg90H21EOLACvtfcHk0liN3FzKYua0fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtFCntrfK&md5=dbdfa4e56338459b614042500022a661</span></div><a href="/servlet/linkout?suffix=cit160&amp;dbid=16384&amp;doi=10.1515%2FBC.2007.126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1515%252FBC.2007.126%26sid%3Dliteratum%253Aachs%26aulast%3DSavaskan%26aufirst%3DN.%2BE.%26aulast%3DUfer%26aufirst%3DC.%26aulast%3DK%25C3%25BChn%26aufirst%3DH.%26aulast%3DBorchert%26aufirst%3DA.%26atitle%3DMolecular%2520Biology%2520of%2520Glutathione%2520Peroxidase%25204%253A%2520From%2520Genomic%2520Structure%2520to%2520Developmental%2520Expression%2520and%2520Neural%2520Function%26jtitle%3DBiol.%2520Chem.%26date%3D2007%26volume%3D388%26spage%3D1007%26epage%3D1017%26doi%3D10.1515%2FBC.2007.126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref161"><div class="reference"><strong class="refLabel"><a href="#ref161" class="rightTabRefNumLink">161</a></strong><div class="NLM_citation" id="rightTab-cit161"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yang, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sriramaratnam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Welsch, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shimada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skouta, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswanathan, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheah, J. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clemons, P. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clish, C. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Girotti, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornish, V. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, B. R.</span></span> <span> </span><span class="NLM_article-title">Regulation of Ferroptotic Cancer Cell Death by GPX4</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>156</i></span>,  <span class="NLM_fpage">317</span>– <span class="NLM_lpage">331</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2013.12.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2013.12.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24439385" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref161/cit161&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtF2is70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=156&publication_year=2014&pages=317-331&author=W.+S.+Yangauthor=R.+Sriramaratnamauthor=M.+E.+Welschauthor=K.+Shimadaauthor=R.+Skoutaauthor=V.+S.+Viswanathanauthor=J.+H.+Cheahauthor=P.+A.+Clemonsauthor=A.+F.+Shamjiauthor=C.+B.+Clishauthor=L.+M.+Brownauthor=A.+W.+Girottiauthor=V.+W.+Cornishauthor=S.+L.+Schreiberauthor=B.+R.+Stockwell&title=Regulation+of+Ferroptotic+Cancer+Cell+Death+by+GPX4&doi=10.1016%2Fj.cell.2013.12.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit161R"><div class="casContent"><span class="casTitleNuber">161</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of Ferroptotic Cancer Cell Death by GPX4</span></div><div class="casAuthors">Yang, Wan Seok; SriRamaratnam, Rohitha; Welsch, Matthew E.; Shimada, Kenichi; Skouta, Rachid; Viswanathan, Vasanthi S.; Cheah, Jaime H.; Clemons, Paul A.; Shamji, Alykhan F.; Clish, Clary B.; Brown, Lewis M.; Girotti, Albert W.; Cornish, Virginia W.; Schreiber, Stuart L.; Stockwell, Brent R.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">156</span>
        (<span class="NLM_cas:issue">1-2</span>),
    <span class="NLM_cas:pages">317-331</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Ferroptosis is a form of nonapoptotic cell death for which key regulators remain unknown.  We sought a common mediator for the lethality of 12 ferroptosis-inducing small mols.  We used targeted metabolomic profiling to discover that depletion of glutathione causes inactivation of glutathione peroxidases (GPXs) in response to one class of compds. and a chemoproteomics strategy to discover that GPX4 is directly inhibited by a second class of compds.  GPX4 overexpression and knockdown modulated the lethality of 12 ferroptosis inducers, but not of 11 compds. with other lethal mechanisms.  In addn., two representative ferroptosis inducers prevented tumor growth in xenograft mouse tumor models.  Sensitivity profiling in 177 cancer cell lines revealed that diffuse large B cell lymphomas and renal cell carcinomas are particularly susceptible to GPX4-regulated ferroptosis.  Thus, GPX4 is an essential regulator of ferroptotic cancer cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdxNz56kpRuLVg90H21EOLACvtfcHk0liN3FzKYua0fg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtF2is70%253D&md5=f5f607e12540847d9a25d8c205ba2e3c</span></div><a href="/servlet/linkout?suffix=cit161&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2013.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2013.12.010%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DW.%2BS.%26aulast%3DSriramaratnam%26aufirst%3DR.%26aulast%3DWelsch%26aufirst%3DM.%2BE.%26aulast%3DShimada%26aufirst%3DK.%26aulast%3DSkouta%26aufirst%3DR.%26aulast%3DViswanathan%26aufirst%3DV.%2BS.%26aulast%3DCheah%26aufirst%3DJ.%2BH.%26aulast%3DClemons%26aufirst%3DP.%2BA.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DClish%26aufirst%3DC.%2BB.%26aulast%3DBrown%26aufirst%3DL.%2BM.%26aulast%3DGirotti%26aufirst%3DA.%2BW.%26aulast%3DCornish%26aufirst%3DV.%2BW.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DStockwell%26aufirst%3DB.%2BR.%26atitle%3DRegulation%2520of%2520Ferroptotic%2520Cancer%2520Cell%2520Death%2520by%2520GPX4%26jtitle%3DCell%26date%3D2014%26volume%3D156%26spage%3D317%26epage%3D331%26doi%3D10.1016%2Fj.cell.2013.12.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref162"><div class="reference"><strong class="refLabel"><a href="#ref162" class="rightTabRefNumLink">162</a></strong><div class="NLM_citation" id="rightTab-cit162"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xie, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedmann
Angeli, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Activation of Glutathione Peroxidase 4 as a Novel Anti-Inflammatory Strategy</span>. <i>Front. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">12</span>, <span class="refDoi"> DOI: 10.3389/fphar.2018.01120</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3389%2Ffphar.2018.01120" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref162/cit162&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29387012" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2018&pages=1-12&author=C.+Liauthor=X.+Dengauthor=X.+Xieauthor=Y.+Liuauthor=J.+P.+Friedmann%0AAngeliauthor=L.+Lai&title=Activation+of+Glutathione+Peroxidase+4+as+a+Novel+Anti-Inflammatory+Strategy&doi=10.3389%2Ffphar.2018.01120"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit162&amp;dbid=16384&amp;doi=10.3389%2Ffphar.2018.01120&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffphar.2018.01120%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DC.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DXie%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DFriedmann%2BAngeli%26aufirst%3DJ.%2BP.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DActivation%2520of%2520Glutathione%2520Peroxidase%25204%2520as%2520a%2520Novel%2520Anti-Inflammatory%2520Strategy%26jtitle%3DFront.%2520Pharmacol.%26date%3D2018%26volume%3D9%26spage%3D1%26epage%3D12%26doi%3D10.3389%2Ffphar.2018.01120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref163"><div class="reference"><strong class="refLabel"><a href="#ref163" class="rightTabRefNumLink">163</a></strong><div class="NLM_citation" id="rightTab-cit163"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">LigBuilder 2: A Practical de Novo Drug Design Approach</span>. <i>J. Chem. Inf. Model.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">1083</span>– <span class="NLM_lpage">1091</span>, <span class="refDoi"> DOI: 10.1021/ci100350u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100350u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref163/cit163&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlsVKqsbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1083-1091&author=Y.+Yuanauthor=J.+Peiauthor=L.+Lai&title=LigBuilder+2%3A+A+Practical+de+Novo+Drug+Design+Approach&doi=10.1021%2Fci100350u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit163R"><div class="casContent"><span class="casTitleNuber">163</span><div class="casTitle"><span class="NLM_cas:atitle">LigBuilder 2: A Practical de Novo Drug Design Approach</span></div><div class="casAuthors">Yuan, Yaxia; Pei, Jianfeng; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1083-1091</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The authors have developed a new version (2.0) of the de novo drug design program LigBuilder.  With LigBuilder 2.0, the synthesis accessibility of designed compds. can be analyzed, and a cavity detection procedure is implemented to detect the positions and shapes of the binding sites on the surface of a given protein structure and to quant. est. drugability.  Ligands are designed to best fit the detected cavities using a set of rules for evaluation.  Drug-like and privileged fragments are used to construct the ligands with the aid of internal and external absorption, distribution, metab., excretion, and toxicity (ADME/T) and drug-like filters.  The authors have tested LigBuilder 2.0 in designing and optimizing Cyclophilin A inhibitors.  By using only a one-round design, a novel inhibitor with greater potency than the pos. control, cyclosporin A, was discovered.  This successful de novo drug design example verified the effectiveness of LigBuilder 2.0.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqb53kWl8fbVg90H21EOLACvtfcHk0lj1rSBTX86M9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlsVKqsbc%253D&md5=e33a958dcc48f86e4225dbfe10c2fd51</span></div><a href="/servlet/linkout?suffix=cit163&amp;dbid=16384&amp;doi=10.1021%2Fci100350u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100350u%26sid%3Dliteratum%253Aachs%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DLigBuilder%25202%253A%2520A%2520Practical%2520de%2520Novo%2520Drug%2520Design%2520Approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1083%26epage%3D1091%26doi%3D10.1021%2Fci100350u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref164"><div class="reference"><strong class="refLabel"><a href="#ref164" class="rightTabRefNumLink">164</a></strong><div class="NLM_citation" id="rightTab-cit164"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Natarajan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nashine, V. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Socolich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vo, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russ, W. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benkovic, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Surface Sites for Engineering Allosteric Control in Proteins</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>322</i></span>,  <span class="NLM_fpage">438</span>– <span class="NLM_lpage">442</span>, <span class="refDoi"> DOI: 10.1126/science.1159052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1126%2Fscience.1159052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18927392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref164/cit164&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Citr3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2008&pages=438-442&author=J.+Leeauthor=M.+Natarajanauthor=V.+C.+Nashineauthor=M.+Socolichauthor=T.+Voauthor=W.+P.+Russauthor=S.+J.+Benkovicauthor=R.+Ranganathan&title=Surface+Sites+for+Engineering+Allosteric+Control+in+Proteins&doi=10.1126%2Fscience.1159052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit164R"><div class="casContent"><span class="casTitleNuber">164</span><div class="casTitle"><span class="NLM_cas:atitle">Surface Sites for Engineering Allosteric Control in Proteins</span></div><div class="casAuthors">Lee, Jeeyeon; Natarajan, Madhusudan; Nashine, Vishal C.; Socolich, Michael; Vo, Tina; Russ, William P.; Benkovic, Stephen J.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">322</span>
        (<span class="NLM_cas:issue">5900</span>),
    <span class="NLM_cas:pages">438-442</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Statistical analyses of protein families reveal networks of coevolving amino acids that functionally link distantly positioned functional surfaces.  Such linkages suggest a concept for engineering allosteric control into proteins: the intramol. networks of two proteins could be joined across their surface sites such that the activity of one protein might control the activity of the other.  We tested this idea by creating PAS-DHFR, a designed chimeric protein that connects a light-sensing signaling domain from a plant member of the Per/Arnt/Sim (PAS) family of proteins with Escherichia coli dihydrofolate reductase (DHFR).  With no optimization, PAS-DHFR exhibited light-dependent catalytic activity that depended on the site of connection and on known signaling mechanisms in both proteins.  PAS-DHFR serves as a proof of concept for engineering regulatory activities into proteins through interface design at conserved allosteric sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYSUy8SM8YL7Vg90H21EOLACvtfcHk0ljLx4wGEIds4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Citr3J&md5=0349bddb15613bd551271721a780789c</span></div><a href="/servlet/linkout?suffix=cit164&amp;dbid=16384&amp;doi=10.1126%2Fscience.1159052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1159052%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%26aulast%3DNatarajan%26aufirst%3DM.%26aulast%3DNashine%26aufirst%3DV.%2BC.%26aulast%3DSocolich%26aufirst%3DM.%26aulast%3DVo%26aufirst%3DT.%26aulast%3DRuss%26aufirst%3DW.%2BP.%26aulast%3DBenkovic%26aufirst%3DS.%2BJ.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DSurface%2520Sites%2520for%2520Engineering%2520Allosteric%2520Control%2520in%2520Proteins%26jtitle%3DScience%26date%3D2008%26volume%3D322%26spage%3D438%26epage%3D442%26doi%3D10.1126%2Fscience.1159052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref165"><div class="reference"><strong class="refLabel"><a href="#ref165" class="rightTabRefNumLink">165</a></strong><div class="NLM_citation" id="rightTab-cit165"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Reynolds, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Hot Spots for Allosteric Regulation on Protein Surfaces</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>147</i></span>,  <span class="NLM_fpage">1564</span>– <span class="NLM_lpage">1575</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2011.10.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.cell.2011.10.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22196731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref165/cit165&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1Ort73J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=147&publication_year=2011&pages=1564-1575&author=K.+A.+Reynoldsauthor=R.+N.+McLaughlinauthor=R.+Ranganathan&title=Hot+Spots+for+Allosteric+Regulation+on+Protein+Surfaces&doi=10.1016%2Fj.cell.2011.10.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit165R"><div class="casContent"><span class="casTitleNuber">165</span><div class="casTitle"><span class="NLM_cas:atitle">Hot Spots for Allosteric Regulation on Protein Surfaces</span></div><div class="casAuthors">Reynolds, Kimberly A.; McLaughlin, Richard N.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">147</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1564-1575</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Recent work indicates a general architecture for proteins in which sparse networks of phys. contiguous and coevolving amino acids underlie basic aspects of structure and function.  These networks, termed sectors, are spatially organized such that active sites are linked to many surface sites distributed throughout the structure.  Using the metabolic enzyme dihydrofolate reductase (DHFR) as a model system, we show that: (1) the sector is strongly correlated to a network of residues undergoing millisecond conformational fluctuations assocd. with enzyme catalysis, and (2) sector-connected surface sites are statistically preferred locations for the emergence of allosteric control in vivo.  Thus, sectors represent an evolutionarily conserved "wiring" mechanism that can enable perturbations at sp. surface positions to rapidly initiate conformational control over protein function.  These findings suggest that sectors enable the evolution of intermol. communication and regulation.  Protein sectors, networks of phys. contiguous and coevolving amino acids, facilitate the emergence of allosteric regulation, as demonstrated with chimeric proteins.  The findings suggest that sectors also naturally enable the evolution of intermol. communication.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp62r-KReV7z7Vg90H21EOLACvtfcHk0ljLx4wGEIds4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1Ort73J&md5=e86fcb4f822f2408d96756b07afdb389</span></div><a href="/servlet/linkout?suffix=cit165&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2011.10.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2011.10.049%26sid%3Dliteratum%253Aachs%26aulast%3DReynolds%26aufirst%3DK.%2BA.%26aulast%3DMcLaughlin%26aufirst%3DR.%2BN.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DHot%2520Spots%2520for%2520Allosteric%2520Regulation%2520on%2520Protein%2520Surfaces%26jtitle%3DCell%26date%3D2011%26volume%3D147%26spage%3D1564%26epage%3D1575%26doi%3D10.1016%2Fj.cell.2011.10.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref166"><div class="reference"><strong class="refLabel"><a href="#ref166" class="rightTabRefNumLink">166</a></strong><div class="NLM_citation" id="rightTab-cit166"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aldrich, R. W.</span></span> <span> </span><span class="NLM_article-title">On Evolutionary Conservation of Thermodynamic Coupling in Proteins</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>279</i></span>,  <span class="NLM_fpage">19046</span>– <span class="NLM_lpage">19050</span>, <span class="refDoi"> DOI: 10.1074/jbc.M402560200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.M402560200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=15023994" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref166/cit166&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjsVKmtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=19046-19050&author=A.+A.+Fodorauthor=R.+W.+Aldrich&title=On+Evolutionary+Conservation+of+Thermodynamic+Coupling+in+Proteins&doi=10.1074%2Fjbc.M402560200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit166R"><div class="casContent"><span class="casTitleNuber">166</span><div class="casTitle"><span class="NLM_cas:atitle">On Evolutionary Conservation of Thermodynamic Coupling in Proteins</span></div><div class="casAuthors">Fodor, Anthony A.; Aldrich, Richard W.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">19046-19050</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The inherent complexity of thermodn. coupling in proteins presents a major challenge in understanding and engineering protein function.  Recent work has argued that the study of proteins can be simplified by the use of correlated mutations in the evolutionary record to locate a small subset of thermodynamically coupled residues that participate in functionally important, evolutionarily conserved energetic pathways.  To test this hypothesis, we examd. the predictions of correlated mutation algorithms for a no. of proteins for which coupling between residues has been detd. by anal. of double mutant cycles.  We find that correlated mutation algorithms can find residue pairs that are phys. close and that phys. close residue pairs tend to be thermodynamically coupled.  We find little evidence, however, for the hypothesis that thermodn. coupling is limited to the subset of evolutionarily constrained residue positions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpVUz7Ul6I1ZbVg90H21EOLACvtfcHk0ljLx4wGEIds4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjsVKmtL8%253D&md5=74e23b9782a2c024b7e2df6d56d58426</span></div><a href="/servlet/linkout?suffix=cit166&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M402560200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M402560200%26sid%3Dliteratum%253Aachs%26aulast%3DFodor%26aufirst%3DA.%2BA.%26aulast%3DAldrich%26aufirst%3DR.%2BW.%26atitle%3DOn%2520Evolutionary%2520Conservation%2520of%2520Thermodynamic%2520Coupling%2520in%2520Proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D19046%26epage%3D19050%26doi%3D10.1074%2Fjbc.M402560200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref167"><div class="reference"><strong class="refLabel"><a href="#ref167" class="rightTabRefNumLink">167</a></strong><div class="NLM_citation" id="rightTab-cit167"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chi, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elfström, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Snäll, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Engström, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jemth, P.</span></span> <span> </span><span class="NLM_article-title">Reassessing a Sparse Energetic Network within a Single Protein Domain</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>105</i></span>,  <span class="NLM_fpage">4679</span>– <span class="NLM_lpage">4684</span>, <span class="refDoi"> DOI: 10.1073/pnas.0711732105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0711732105" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=18339805" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref167/cit167&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXkt12qs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=4679-4684&author=C.+N.+Chiauthor=L.+Elfstr%C3%B6mauthor=Y.+Shiauthor=T.+Sn%C3%A4llauthor=A.+Engstr%C3%B6mauthor=P.+Jemth&title=Reassessing+a+Sparse+Energetic+Network+within+a+Single+Protein+Domain&doi=10.1073%2Fpnas.0711732105"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit167R"><div class="casContent"><span class="casTitleNuber">167</span><div class="casTitle"><span class="NLM_cas:atitle">Reassessing a sparse energetic network within a single protein domain</span></div><div class="casAuthors">Chi, Celestine N.; Eelfstroem, Lisa; Shi, Yao; Snaell, Tord; Engstroem, Aake; Jemth, Per</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4679-4684</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Understanding the mol. principles that govern allosteric communication is an important goal in protein science.  One way allostery could be transmitted is via sparse energetic networks of residues, and one such evolutionary conserved network was identified in the PDZ domain family of proteins by multiple sequence alignment.  We have reassessed the energetic coupling of these residues by double mutant cycles together with ligand binding and stability expts. and found that coupling is not a special property of the coevolved network of residues in PDZ domains.  The obsd. coupling for ligand binding is better explained by a distance relationship, where residues close in space are more likely to couple than distal residues.  Our study demonstrates that statistical coupling from sequence anal. is not necessarily a reporter of energetic coupling and allostery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFuXABqS1debVg90H21EOLACvtfcHk0lgm7XRSILa0qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXkt12qs7k%253D&md5=93aa623be2915a0b00ea7ea872bab99b</span></div><a href="/servlet/linkout?suffix=cit167&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0711732105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0711732105%26sid%3Dliteratum%253Aachs%26aulast%3DChi%26aufirst%3DC.%2BN.%26aulast%3DElfstr%25C3%25B6m%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DSn%25C3%25A4ll%26aufirst%3DT.%26aulast%3DEngstr%25C3%25B6m%26aufirst%3DA.%26aulast%3DJemth%26aufirst%3DP.%26atitle%3DReassessing%2520a%2520Sparse%2520Energetic%2520Network%2520within%2520a%2520Single%2520Protein%2520Domain%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D4679%26epage%3D4684%26doi%3D10.1073%2Fpnas.0711732105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref168"><div class="reference"><strong class="refLabel"><a href="#ref168" class="rightTabRefNumLink">168</a></strong><div class="NLM_citation" id="rightTab-cit168"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thirumalai, D.</span></span> <span> </span><span class="NLM_article-title">On the Accuracy of Inferring Energetic Coupling between Distant Sites in Protein Families from Evolutionary Imprints: Illustrations Using Lattice Model</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>77</i></span>,  <span class="NLM_fpage">823</span>– <span class="NLM_lpage">831</span>, <span class="refDoi"> DOI: 10.1002/prot.22498</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fprot.22498" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=19639636" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref168/cit168&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtlSru7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=2009&pages=823-831&author=Z.+Liuauthor=J.+Chenauthor=D.+Thirumalai&title=On+the+Accuracy+of+Inferring+Energetic+Coupling+between+Distant+Sites+in+Protein+Families+from+Evolutionary+Imprints%3A+Illustrations+Using+Lattice+Model&doi=10.1002%2Fprot.22498"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit168R"><div class="casContent"><span class="casTitleNuber">168</span><div class="casTitle"><span class="NLM_cas:atitle">On the accuracy of inferring energetic coupling between distant sites in protein families from evolutionary imprints: Illustrations using lattice model</span></div><div class="casAuthors">Liu, Zhenxing; Chen, Jie; Thirumalai, D.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">77</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">823-831</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">It is suspected that correlated motions among a subset of spatially sepd. residues drive conformational dynamics not only in multidomain but also in single domain proteins.  Sequence and structure-based methods have been proposed to det. covariation between two sites on a protein.  The statistical coupling anal. (SCA) that compares the changes in probability at two sites in a multiple sequence alignment (MSA) and a subset of the MSA has been used to infer the network of residues that encodes allosteric signals in protein families.  The structural perturbation method (SPM), that probes the response of a local perturbation at all other sites, has been used to probe the allostery wiring diagram in biol. machines and enzymes.  To assess the efficacy of the SCA, we used an exactly sol. two dimensional lattice model and performed double-mutant cycle (DMC) calcns. to predict the extent of phys. coupling between two sites.  The predictions of the SCA and the DMC results show that only residues that are in contact in the native state are accurately identified.  In addn., covariations among strongly interacting residues are most easily identified by the SCA.  These conclusions are consistent with the DMC expts. on the PDZ family.  Good correlation between the SCA and the DMC is only obtained by performing multiple expts. that vary the nature of amino acids at a given site.  In contrast, the energetic coupling found in expts. for the PDZ domain are recovered using the SPM.  We also predict, using the SPM, several residues that are coupled energetically.  Proteins 2009. © 2009 Wiley-Liss, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLo4K1cqhX3bVg90H21EOLACvtfcHk0lgm7XRSILa0qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtlSru7nI&md5=27cfadf669f2d63961d2a8e9ad4bc8b8</span></div><a href="/servlet/linkout?suffix=cit168&amp;dbid=16384&amp;doi=10.1002%2Fprot.22498&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.22498%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DThirumalai%26aufirst%3DD.%26atitle%3DOn%2520the%2520Accuracy%2520of%2520Inferring%2520Energetic%2520Coupling%2520between%2520Distant%2520Sites%2520in%2520Protein%2520Families%2520from%2520Evolutionary%2520Imprints%253A%2520Illustrations%2520Using%2520Lattice%2520Model%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2009%26volume%3D77%26spage%3D823%26epage%3D831%26doi%3D10.1002%2Fprot.22498" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref169"><div class="reference"><strong class="refLabel"><a href="#ref169" class="rightTabRefNumLink">169</a></strong><div class="NLM_citation" id="rightTab-cit169"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Livesay, D. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kreth, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fodor, A. A.</span></span> <span> </span><span class="NLM_article-title">A Critical Evaluation of Correlated Mutation Algorithms and Coevolution within Allosteric Mechanisms</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>796</i></span>,  <span class="NLM_fpage">385</span>– <span class="NLM_lpage">398</span>, <span class="refDoi"> DOI: 10.1007/978-1-61779-334-9_21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1007%2F978-1-61779-334-9_21" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22052502" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref169/cit169&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1OgurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=796&publication_year=2012&pages=385-398&author=D.+R.+Livesayauthor=K.+E.+Krethauthor=A.+A.+Fodor&title=A+Critical+Evaluation+of+Correlated+Mutation+Algorithms+and+Coevolution+within+Allosteric+Mechanisms&doi=10.1007%2F978-1-61779-334-9_21"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit169R"><div class="casContent"><span class="casTitleNuber">169</span><div class="casTitle"><span class="NLM_cas:atitle">A critical evaluation of correlated mutation algorithms and coevolution within allosteric mechanisms</span></div><div class="casAuthors">Livesay, Dennis R.; Kreth, Kyle E.; Fodor, Anthony A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">796</span>
        (<span class="NLM_cas:issue">Allostery</span>),
    <span class="NLM_cas:pages">385-398</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">A review.  The notion of using the evolutionary history encoded within multiple sequence alignments to predict allosteric mechanisms is appealing.  In this approach, correlated mutations are expected to reflect coordinated changes that maintain intramol. coupling between residue pairs.  Despite much early fanfare, the general suitability of correlated mutations to predict allosteric couplings has not yet been established.  Lack of progress along these lines has been hindered by several algorithmic limitations including phylogenetic artifacts within alignments masking true covariance and the computational intractability of consideration of more than two correlated residues at a time.  Recent progress in algorithm development, however, has been substantial with a new generation of correlated mutation algorithms that have made fundamental progress toward solving these difficult problems.  Despite these encouraging results, there remains little evidence to suggest that the evolutionary constraints acting on allosteric couplings are sufficient to be recovered from multiple sequence alignments.  In this review, we argue that due to the exquisite sensitivity of protein dynamics, and hence that of allosteric mechanisms, the latter vary widely within protein families.  If it turns out to be generally true that even very similar homologs display a wide divergence of allosteric mechanisms, then even a perfect correlated mutation algorithm could not be reliably used as a general mechanism for discovery of allosteric pathways.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqVeVwUhH4i7Vg90H21EOLACvtfcHk0lgm7XRSILa0qw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1OgurrL&md5=243db85ca72fac574b8c49493aee7320</span></div><a href="/servlet/linkout?suffix=cit169&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-334-9_21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-334-9_21%26sid%3Dliteratum%253Aachs%26aulast%3DLivesay%26aufirst%3DD.%2BR.%26aulast%3DKreth%26aufirst%3DK.%2BE.%26aulast%3DFodor%26aufirst%3DA.%2BA.%26atitle%3DA%2520Critical%2520Evaluation%2520of%2520Correlated%2520Mutation%2520Algorithms%2520and%2520Coevolution%2520within%2520Allosteric%2520Mechanisms%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D796%26spage%3D385%26epage%3D398%26doi%3D10.1007%2F978-1-61779-334-9_21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref170"><div class="reference"><strong class="refLabel"><a href="#ref170" class="rightTabRefNumLink">170</a></strong><div class="NLM_citation" id="rightTab-cit170"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Anishchenko, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ovchinnikov, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kamisetty, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, D.</span></span> <span> </span><span class="NLM_article-title">Origins of Coevolution between Residues Distant in Protein 3D Structures</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">9122</span>– <span class="NLM_lpage">9127</span>, <span class="refDoi"> DOI: 10.1073/pnas.1702664114</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1702664114" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=28784799" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref170/cit170&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1yku7fI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2017&pages=9122-9127&author=I.+Anishchenkoauthor=S.+Ovchinnikovauthor=H.+Kamisettyauthor=D.+Baker&title=Origins+of+Coevolution+between+Residues+Distant+in+Protein+3D+Structures&doi=10.1073%2Fpnas.1702664114"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit170R"><div class="casContent"><span class="casTitleNuber">170</span><div class="casTitle"><span class="NLM_cas:atitle">Origins of coevolution between residues distant in protein 3D structures</span></div><div class="casAuthors">Anishchenko, Ivan; Ovchinnikov, Sergey; Kamisetty, Hetunandan; Baker, David</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">9122-9127</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Residue pairs that directly coevolve in protein families are generally close in protein 3-dimensional (3D) structures.  Here, we studied the exceptions to this general trend, directly coevolving residue pairs that are distant in protein structures, to det. the origins of evolutionary pressure on spatially distant residues and to understand the sources of error in contact-based structure prediction.  Over a set of 4000 protein families, we found that 25% of directly coevolving residue pairs were sepd. by >5 Å in protein structures and 3% by >15 Å.  The majority (91%) of directly coevolving residue pairs in the 5-15 Å range were found to be in contact in at least one homologous structure; these exceptions arose from structural variation in the family in the region contg. the residues.  Thirty-five percent of the exceptions at >15 Å were at homo-oligomeric interfaces, 19% arose from family structural variation, and 27% were in repeat proteins likely reflecting alignment errors.  Of the remaining long-range exceptions (<1% of the total no. of coupled pairs), many could be attributed to close interactions in an oligomeric state.  Overall, the results suggest that directly coevolving residue pairs not in repeat proteins are spatially proximal in at least one biol. relevant protein conformation within the family; we found little evidence for direct coupling between residues at spatially sepd. allosteric and functional sites or for increased direct coupling between residue pairs on putative allosteric pathways connecting them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHshwNC4Ovu7Vg90H21EOLACvtfcHk0lhCRBELbSn7og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1yku7fI&md5=dd465e6c3065f443ea86406a4f150be6</span></div><a href="/servlet/linkout?suffix=cit170&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1702664114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1702664114%26sid%3Dliteratum%253Aachs%26aulast%3DAnishchenko%26aufirst%3DI.%26aulast%3DOvchinnikov%26aufirst%3DS.%26aulast%3DKamisetty%26aufirst%3DH.%26aulast%3DBaker%26aufirst%3DD.%26atitle%3DOrigins%2520of%2520Coevolution%2520between%2520Residues%2520Distant%2520in%2520Protein%25203D%2520Structures%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2017%26volume%3D114%26spage%3D9122%26epage%3D9127%26doi%3D10.1073%2Fpnas.1702664114" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref171"><div class="reference"><strong class="refLabel"><a href="#ref171" class="rightTabRefNumLink">171</a></strong><div class="NLM_citation" id="rightTab-cit171"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Novinec, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Korenč, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caflisch, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lenarčič, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baici, A.</span></span> <span> </span><span class="NLM_article-title">A Novel Allosteric Mechanism in the Cysteine Peptidase Cathepsin K Discovered by Computational Methods</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3287</span>, <span class="refDoi"> DOI: 10.1038/ncomms4287</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms4287" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24518821" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref171/cit171&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvksVaitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3287&author=M.+Novinecauthor=M.+Koren%C4%8Dauthor=A.+Caflischauthor=R.+Ranganathanauthor=B.+Lenar%C4%8Di%C4%8Dauthor=A.+Baici&title=A+Novel+Allosteric+Mechanism+in+the+Cysteine+Peptidase+Cathepsin+K+Discovered+by+Computational+Methods&doi=10.1038%2Fncomms4287"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit171R"><div class="casContent"><span class="casTitleNuber">171</span><div class="casTitle"><span class="NLM_cas:atitle">A novel allosteric mechanism in the cysteine peptidase cathepsin K discovered by computational methods</span></div><div class="casAuthors">Novinec Marko; Korenc Matevz; Caflisch Amedeo; Baici Antonio; Ranganathan Rama; Lenarcic Brigita</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3287</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Allosteric modifiers have the potential to fine-tune enzyme activity.  Therefore, targeting allosteric sites is gaining increasing recognition as a strategy in drug design.  Here we report the use of computational methods for the discovery of the first small-molecule allosteric inhibitor of the collagenolytic cysteine peptidase cathepsin K, a major target for the treatment of osteoporosis.  The molecule NSC13345 is identified by high-throughput docking of compound libraries to surface sites on the peptidase that are connected to the active site by an evolutionarily conserved network of residues (protein sector).  The crystal structure of the complex shows that NSC13345 binds to a novel allosteric site on cathepsin K.  The compound acts as a hyperbolic mixed modifier in the presence of a synthetic substrate, it completely inhibits collagen degradation and has good selectivity for cathepsin K over related enzymes.  Altogether, these properties qualify our methodology and NSC13345 as promising candidates for allosteric drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTJl-p6b4YuipHZzFRCXxVKfW6udTcc2eZWAG3uBUz45bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvksVaitA%253D%253D&md5=522c75841220139748cdf2b0f6cede07</span></div><a href="/servlet/linkout?suffix=cit171&amp;dbid=16384&amp;doi=10.1038%2Fncomms4287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4287%26sid%3Dliteratum%253Aachs%26aulast%3DNovinec%26aufirst%3DM.%26aulast%3DKoren%25C4%258D%26aufirst%3DM.%26aulast%3DCaflisch%26aufirst%3DA.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DLenar%25C4%258Di%25C4%258D%26aufirst%3DB.%26aulast%3DBaici%26aufirst%3DA.%26atitle%3DA%2520Novel%2520Allosteric%2520Mechanism%2520in%2520the%2520Cysteine%2520Peptidase%2520Cathepsin%2520K%2520Discovered%2520by%2520Computational%2520Methods%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3287%26doi%3D10.1038%2Fncomms4287" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref172"><div class="reference"><strong class="refLabel"><a href="#ref172" class="rightTabRefNumLink">172</a></strong><div class="NLM_citation" id="rightTab-cit172"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harris, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Olson, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goodsell, D. S.</span></span> <span> </span><span class="NLM_article-title">Automated Prediction of Ligand-Binding Sites in Proteins</span>. <i>Proteins: Struct., Funct., Genet.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>70</i></span>,  <span class="NLM_fpage">1506</span>– <span class="NLM_lpage">1517</span>, <span class="refDoi"> DOI: 10.1002/prot.21645</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fprot.21645" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17910060" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref172/cit172&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFyrtbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=70&publication_year=2008&pages=1506-1517&author=R.+Harrisauthor=A.+J.+Olsonauthor=D.+S.+Goodsell&title=Automated+Prediction+of+Ligand-Binding+Sites+in+Proteins&doi=10.1002%2Fprot.21645"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit172R"><div class="casContent"><span class="casTitleNuber">172</span><div class="casTitle"><span class="NLM_cas:atitle">Automated prediction of ligand-binding sites in proteins</span></div><div class="casAuthors">Harris, Rodney; Olson, Arthur J.; Goodsell, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">70</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1506-1517</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The authors present a method, termed AutoLigand, for the prediction of ligand-binding sites in proteins of known structure.  The method searches the space surrounding the protein and finds the contiguous envelope with the specified vol. of atoms, which has the largest possible interaction energy with the protein.  It uses a full at. representation, with atom types for carbon, hydrogen, oxygen, nitrogen and sulfur (and others, if desired), and is designed to minimize the need for artificial geometry.  Testing on a set of 187 diverse protein-ligand complexes has shown that the method is successful in predicting the location and approx. vol. of the binding site in 73% of cases.  Addnl. testing was performed on a set of 96 protein-ligand complexes with crystallog. structures of apo and holo forms, and AutoLigand was able to predict the binding site in 80% of the apo structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpwD6aCbA5coLVg90H21EOLACvtfcHk0lhCRBELbSn7og"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFyrtbk%253D&md5=a8fcf5e1d18f9315b4336573f71b1629</span></div><a href="/servlet/linkout?suffix=cit172&amp;dbid=16384&amp;doi=10.1002%2Fprot.21645&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.21645%26sid%3Dliteratum%253Aachs%26aulast%3DHarris%26aufirst%3DR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DGoodsell%26aufirst%3DD.%2BS.%26atitle%3DAutomated%2520Prediction%2520of%2520Ligand-Binding%2520Sites%2520in%2520Proteins%26jtitle%3DProteins%253A%2520Struct.%252C%2520Funct.%252C%2520Genet.%26date%3D2008%26volume%3D70%26spage%3D1506%26epage%3D1517%26doi%3D10.1002%2Fprot.21645" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref173"><div class="reference"><strong class="refLabel"><a href="#ref173" class="rightTabRefNumLink">173</a></strong><div class="NLM_citation" id="rightTab-cit173"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McLaughlin, R. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poelwijk, F. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raman, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gosal, W. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">The Spatial Architecture of Protein Function and Adaptation</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>491</i></span>,  <span class="NLM_fpage">138</span>– <span class="NLM_lpage">142</span>, <span class="refDoi"> DOI: 10.1038/nature11500</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature11500" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23041932" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref173/cit173&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVKisbbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=491&publication_year=2012&pages=138-142&author=R.+N.+McLaughlinauthor=F.+J.+Poelwijkauthor=A.+Ramanauthor=W.+S.+Gosalauthor=R.+Ranganathan&title=The+Spatial+Architecture+of+Protein+Function+and+Adaptation&doi=10.1038%2Fnature11500"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit173R"><div class="casContent"><span class="casTitleNuber">173</span><div class="casTitle"><span class="NLM_cas:atitle">The spatial architecture of protein function and adaptation</span></div><div class="casAuthors">McLaughlin, Richard N., Jr.; Poelwijk, Frank J.; Raman, Arjun; Gosal, Walraj S.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">491</span>
        (<span class="NLM_cas:issue">7422</span>),
    <span class="NLM_cas:pages">138-142</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Statistical anal. of protein evolution suggests a design for natural proteins in which sparse networks of coevolving amino acids (termed sectors) comprise the essence of three-dimensional structure and function.  However, proteins are also subject to pressures deriving from the dynamics of the evolutionary process itself-the ability to tolerate mutation and to be adaptive to changing selection pressures.  To understand the relationship of the sector architecture to these properties, we developed a high-throughput quant. method for a comprehensive single-mutation study in which every position is substituted individually to every other amino acid.  Using a PDZ domain (PSD95pdz3) model system, we show that sector positions are functionally sensitive to mutation, whereas non-sector positions are more tolerant to substitution.  In addn., we find that adaptation to a new binding specificity initiates exclusively through variation within sector residues.  A combination of just two sector mutations located near and away from the ligand-binding site suffices to switch the binding specificity of PSD95pdz3 quant. towards a class-switching ligand.  The localization of functional constraint and adaptive variation within the sector has important implications for understanding and engineering proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUbG0JzDZ9arVg90H21EOLACvtfcHk0lhz_EEh9RWSfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVKisbbE&md5=c276f35ff757d6a1e1d286cfa9444cc2</span></div><a href="/servlet/linkout?suffix=cit173&amp;dbid=16384&amp;doi=10.1038%2Fnature11500&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11500%26sid%3Dliteratum%253Aachs%26aulast%3DMcLaughlin%26aufirst%3DR.%2BN.%26aulast%3DPoelwijk%26aufirst%3DF.%2BJ.%26aulast%3DRaman%26aufirst%3DA.%26aulast%3DGosal%26aufirst%3DW.%2BS.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DThe%2520Spatial%2520Architecture%2520of%2520Protein%2520Function%2520and%2520Adaptation%26jtitle%3DNature%26date%3D2012%26volume%3D491%26spage%3D138%26epage%3D142%26doi%3D10.1038%2Fnature11500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref174"><div class="reference"><strong class="refLabel"><a href="#ref174" class="rightTabRefNumLink">174</a></strong><div class="NLM_citation" id="rightTab-cit174"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hatley, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockless, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gibson, S. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gilman, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span> <span> </span><span class="NLM_article-title">Allosteric Determinants in Guanine Nucleotide-Binding Proteins</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>100</i></span>,  <span class="NLM_fpage">14445</span>– <span class="NLM_lpage">14450</span>, <span class="refDoi"> DOI: 10.1073/pnas.1835919100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1835919100" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=14623969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref174/cit174&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD3sXpsFGhsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=14445-14450&author=M.+E.+Hatleyauthor=S.+W.+Locklessauthor=S.+K.+Gibsonauthor=A.+G.+Gilmanauthor=R.+Ranganathan&title=Allosteric+Determinants+in+Guanine+Nucleotide-Binding+Proteins&doi=10.1073%2Fpnas.1835919100"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit174R"><div class="casContent"><span class="casTitleNuber">174</span><div class="casTitle"><span class="NLM_cas:atitle">Allosteric determinants in guanine nucleotide-binding proteins</span></div><div class="casAuthors">Hatley, Mark E.; Lockless, Steve W.; Gibson, Scott K.; Gilman, Alfred G.; Ranganathan, Rama</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">100</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">14445-14450</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Members of the G protein superfamily contain nucleotide-dependent switches that dictate the specificity of their interactions with binding partners.  Using a sequence-based method termed statistical coupling anal. (SCA), we have attempted to identify the allosteric core of these proteins, the network of amino acid residues that couples the domains responsible for nucleotide binding and protein-protein interactions.  One-third of the 38 residues identified by SCA were mutated in the G protein Gsα, and the interactions of guanosine 5'-3-O-(thio)triphosphate- and GDP-bound mutant proteins were tested with both adenylyl cyclase (preferential binding to GTP- Gsα) and the G protein βγ subunit complex (preferential binding to GDP-Gsα).  A two-state allosteric model predicts that mutation of residues that control the equil. between GDP- and GTP-bound conformations of the protein will cause the ratio of affinities of these species for adenylyl cyclase and Gβγ to vary in a reciprocal fashion.  Obsd. results were consistent with this prediction.  The network of residues identified by the SCA appears to comprise a core allosteric mechanism conferring nucleotide-dependent switching; the specific features of different G protein family members are built on this core.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrusowIjSLwArVg90H21EOLACvtfcHk0lhz_EEh9RWSfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXpsFGhsL4%253D&md5=1535d111d43c094c4e4c9c5cc074ab4f</span></div><a href="/servlet/linkout?suffix=cit174&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1835919100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1835919100%26sid%3Dliteratum%253Aachs%26aulast%3DHatley%26aufirst%3DM.%2BE.%26aulast%3DLockless%26aufirst%3DS.%2BW.%26aulast%3DGibson%26aufirst%3DS.%2BK.%26aulast%3DGilman%26aufirst%3DA.%2BG.%26aulast%3DRanganathan%26aufirst%3DR.%26atitle%3DAllosteric%2520Determinants%2520in%2520Guanine%2520Nucleotide-Binding%2520Proteins%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D14445%26epage%3D14450%26doi%3D10.1073%2Fpnas.1835919100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref175"><div class="reference"><strong class="refLabel"><a href="#ref175" class="rightTabRefNumLink">175</a></strong><div class="NLM_citation" id="rightTab-cit175"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ferguson, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amezcua, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halabi, N. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chelliah, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosen, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ranganathan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deisenhofer, J.</span></span> <span> </span><span class="NLM_article-title">Signal Transduction Pathway of TonB-Dependent Transporters</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>104</i></span>,  <span class="NLM_fpage">513</span>– <span class="NLM_lpage">518</span>, <span class="refDoi"> DOI: 10.1073/pnas.0609887104</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.0609887104" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=17197416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref175/cit175&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BD2sXptVCnuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=104&publication_year=2007&pages=513-518&author=A.+D.+Fergusonauthor=C.+A.+Amezcuaauthor=N.+M.+Halabiauthor=Y.+Chelliahauthor=M.+K.+Rosenauthor=R.+Ranganathanauthor=J.+Deisenhofer&title=Signal+Transduction+Pathway+of+TonB-Dependent+Transporters&doi=10.1073%2Fpnas.0609887104"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit175R"><div class="casContent"><span class="casTitleNuber">175</span><div class="casTitle"><span class="NLM_cas:atitle">Signal transduction pathway of TonB-dependent transporters</span></div><div class="casAuthors">Ferguson, Andrew D.; Amezcua, Carlos A.; Halabi, Najeeb M.; Chelliah, Yogarany; Rosen, Michael K.; Ranganathan, Rama; Deisenhofer, Johann</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">104</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">513-518</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Transcription of the ferric citrate import system is regulated by ferric citrate binding to the outer membrane transporter FecA.  A signal indicating transporter occupancy is relayed across the outer membrane to energy-transducing and regulatory proteins embedded in the cytoplasmic membrane.  Because transcriptional activation is not coupled to ferric citrate import, an allosteric mechanism underlies this complex signaling mechanism.  Using evolution-based statistical anal. we have identified a sparse but structurally connected network of residues that links distant functional sites in FecA.  Functional analyses of these positions confirm their involvement in the mechanism that regulates transcriptional activation in response to ferric citrate binding at the cell surface.  This mechanism appears to be conserved and provides the structural basis for the allosteric signaling of TonB-dependent transporters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8WUEQZ3n0xrVg90H21EOLACvtfcHk0lhz_EEh9RWSfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXptVCnuw%253D%253D&md5=8e833d52d0b582bd84097b7c2465ad4b</span></div><a href="/servlet/linkout?suffix=cit175&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0609887104&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0609887104%26sid%3Dliteratum%253Aachs%26aulast%3DFerguson%26aufirst%3DA.%2BD.%26aulast%3DAmezcua%26aufirst%3DC.%2BA.%26aulast%3DHalabi%26aufirst%3DN.%2BM.%26aulast%3DChelliah%26aufirst%3DY.%26aulast%3DRosen%26aufirst%3DM.%2BK.%26aulast%3DRanganathan%26aufirst%3DR.%26aulast%3DDeisenhofer%26aufirst%3DJ.%26atitle%3DSignal%2520Transduction%2520Pathway%2520of%2520TonB-Dependent%2520Transporters%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2007%26volume%3D104%26spage%3D513%26epage%3D518%26doi%3D10.1073%2Fpnas.0609887104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref176"><div class="reference"><strong class="refLabel"><a href="#ref176" class="rightTabRefNumLink">176</a></strong><div class="NLM_citation" id="rightTab-cit176"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, H. X.</span></span> <span> </span><span class="NLM_article-title">Protein Allostery and Conformational Dynamics</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6503</span>– <span class="NLM_lpage">6515</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00590</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00590" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref176/cit176&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28XisVKjsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6503-6515&author=J.+Guoauthor=H.+X.+Zhou&title=Protein+Allostery+and+Conformational+Dynamics&doi=10.1021%2Facs.chemrev.5b00590"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit176R"><div class="casContent"><span class="casTitleNuber">176</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Allostery and Conformational Dynamics</span></div><div class="casAuthors">Guo, Jingjing; Zhou, Huan-Xiang</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6503-6515</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The functions of many proteins are regulated through allostery, whereby effector binding at a distal site changes the functional activity (e.g., substrate binding affinity or catalytic efficiency) at the active site.  Most allosteric studies have focused on thermodn. properties, in particular, substrate binding affinity.  Changes in substrate binding affinity by allosteric effectors have generally been thought to be mediated by conformational transitions of the proteins or, alternatively, by changes in the broadness of the free energy basin of the protein conformational state without shifting the basin min. position.  When effector binding changes the free energy landscape of a protein in conformational space, the change affects not only thermodn. properties but also dynamic properties, including the amplitudes of motions on different time scales and rates of conformational transitions.  Here we assess the roles of conformational dynamics in allosteric regulation.  Two cases are highlighted where NMR spectroscopy and mol. dynamics simulation have been used as complementary approaches to identify residues possibly involved in allosteric communication.  Perspectives on contentious issues, for example, the relationship between picosecond-nanosecond local and microsecond-millisecond conformational exchange dynamics, are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqVLHVu6pyerrVg90H21EOLACvtfcHk0lhz_EEh9RWSfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XisVKjsr8%253D&md5=8a148ea143d0e31927cfc56745fd13bd</span></div><a href="/servlet/linkout?suffix=cit176&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00590&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00590%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DJ.%26aulast%3DZhou%26aufirst%3DH.%2BX.%26atitle%3DProtein%2520Allostery%2520and%2520Conformational%2520Dynamics%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6503%26epage%3D6515%26doi%3D10.1021%2Facs.chemrev.5b00590" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref177"><div class="reference"><strong class="refLabel"><a href="#ref177" class="rightTabRefNumLink">177</a></strong><div class="NLM_citation" id="rightTab-cit177"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perilla, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goh, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cassidy, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bernardi, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schulten, K.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations of Large Macromolecular Complexes</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>31</i></span>,  <span class="NLM_fpage">64</span>– <span class="NLM_lpage">74</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25845770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref177/cit177&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlsFemsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2015&pages=64-74&author=J.+R.+Perillaauthor=B.+C.+Gohauthor=C.+K.+Cassidyauthor=B.+Liuauthor=R.+C.+Bernardiauthor=T.+Rudackauthor=H.+Yuauthor=Z.+Wuauthor=K.+Schulten&title=Molecular+Dynamics+Simulations+of+Large+Macromolecular+Complexes&doi=10.1016%2Fj.sbi.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit177R"><div class="casContent"><span class="casTitleNuber">177</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations of large macromolecular complexes</span></div><div class="casAuthors">Perilla, Juan R.; Goh, Boon Chong; Cassidy, C. Keith; Liu, Bo; Bernardi, Rafael C.; Rudack, Till; Yu, Hang; Wu, Zhe; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">64-74</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Connecting dynamics to structural data from diverse exptl. sources, mol. dynamics simulations permit the exploration of biol. phenomena in unparalleled detail.  Advances in simulations are moving the at. resoln. descriptions of biol. systems into the million-to-billion atom regime, in which numerous cell functions reside.  In this opinion, the authors review the progress, driven by large-scale mol. dynamics simulations, in the study of viruses, ribosomes, bioenergetic systems, and other diverse applications.  These examples highlight the utility of mol. dynamics simulations in the crit. task of relating at. detail to the function of supramol. complexes, a task that cannot be achieved by smaller-scale simulations or existing exptl. approaches alone.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIz83au-c0jrVg90H21EOLACvtfcHk0lgqaN3u-sgI9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlsFemsb4%253D&md5=8fb981e606aabacf4b75ede88434ec6a</span></div><a href="/servlet/linkout?suffix=cit177&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DPerilla%26aufirst%3DJ.%2BR.%26aulast%3DGoh%26aufirst%3DB.%2BC.%26aulast%3DCassidy%26aufirst%3DC.%2BK.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DBernardi%26aufirst%3DR.%2BC.%26aulast%3DRudack%26aufirst%3DT.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DZ.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520of%2520Large%2520Macromolecular%2520Complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2015%26volume%3D31%26spage%3D64%26epage%3D74%26doi%3D10.1016%2Fj.sbi.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref178"><div class="reference"><strong class="refLabel"><a href="#ref178" class="rightTabRefNumLink">178</a></strong><div class="NLM_citation" id="rightTab-cit178"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Elber, R.</span></span> <span> </span><span class="NLM_article-title">Simulations of Allosteric Transitions</span>. <i>Curr. Opin. Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">167</span>– <span class="NLM_lpage">172</span>, <span class="refDoi"> DOI: 10.1016/j.sbi.2011.01.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.sbi.2011.01.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21333527" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref178/cit178&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2011&pages=167-172&author=R.+Elber&title=Simulations+of+Allosteric+Transitions&doi=10.1016%2Fj.sbi.2011.01.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit178R"><div class="casContent"><span class="casTitleNuber">178</span><div class="casTitle"><span class="NLM_cas:atitle">Simulations of allosteric transitions</span></div><div class="casAuthors">Elber, Ron</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-172</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Allosteric transitions are one of the subtler mechanisms used by nature to fine tune protein activity.  Effector binding to a specific site on the protein surface induces significant activity change, and initiates a conformational transition that frequently includes domain motions and is very large.  From a theor. and biophys. perspective, 2 problems are particularly intriguing.  The 1st is the way in which a launching signal, which is spatially confined and includes only a few interacting atoms, is propagated to a large-scale conformational transition frequently seen in allosteric transitions.  Hence, there is the question of how a small perturbation is magnified to yield motions of thousands of atoms.  The 2nd puzzle is of focus, coherence, and efficiency.  The impact of the binding of the effector is sometimes extended over tens of angstroms.  How the signal is transmitted and kept significant over such large distances in the 'noisy' mol. environment is another major direction of investigation.  Here, the author has examd. different theor. and computational attempts to solve the questions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvNcZY9GuJsLVg90H21EOLACvtfcHk0lgqaN3u-sgI9A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXkt1Kisrc%253D&md5=1aed7f77805946b84d66e7e35f82976a</span></div><a href="/servlet/linkout?suffix=cit178&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2011.01.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2011.01.012%26sid%3Dliteratum%253Aachs%26aulast%3DElber%26aufirst%3DR.%26atitle%3DSimulations%2520of%2520Allosteric%2520Transitions%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2011%26volume%3D21%26spage%3D167%26epage%3D172%26doi%3D10.1016%2Fj.sbi.2011.01.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref179"><div class="reference"><strong class="refLabel"><a href="#ref179" class="rightTabRefNumLink">179</a></strong><div class="NLM_citation" id="rightTab-cit179"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Molecular Dynamics Simulations and Dynamic Network Analysis Reveal the Allosteric Unbinding of Monobody to H-Ras Triggered by R135K Mutation</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">2249</span>, <span class="refDoi"> DOI: 10.3390/ijms18112249</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms18112249" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref179/cit179&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitVOgtLzN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2017&pages=2249&author=D.+Niauthor=K.+Songauthor=J.+Zhangauthor=S.+Lu&title=Molecular+Dynamics+Simulations+and+Dynamic+Network+Analysis+Reveal+the+Allosteric+Unbinding+of+Monobody+to+H-Ras+Triggered+by+R135K+Mutation&doi=10.3390%2Fijms18112249"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit179R"><div class="casContent"><span class="casTitleNuber">179</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular dynamics simulations and dynamic network analysis reveal the allosteric unbinding of monobody to H-Ras triggered by R135K mutation</span></div><div class="casAuthors">Ni, Duan; Song, Kun; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2249/1-2249/16</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Ras proteins, as small GTPases, mediate cell proliferation, survival and differentiation.  Ras mutations have been assocd. with a broad spectrum of human cancers and thus targeting Ras represents a potential way forward for cancer therapy.  A recently reported monobody NS1 allosterically disrupts the Ras-mediated signaling pathway, but its efficacy is reduced by R135K mutation in H-Ras.  However, the detailed mechanism is unresolved.  Here, using mol. dynamics (MD) simulations and dynamic network anal., we explored the mol. mechanism for the unbinding of NS1 to H-Ras and shed light on the underlying allosteric network in H-Ras.  MD simulations revealed that the overall structures of the two complexes did not change significantly, but the H-Ras-NS1 interface underwent significant conformational alteration in the mutant Binding free energy anal. showed that NS1 binding was unfavored after R135K mutation, which resulted in the unfavorable binding of NS1.  Furthermore, the crit. residues on H-Ras responsible for the loss of binding of NS1 were identified.  Importantly, the allosteric networks for these important residues were revealed, which yielded a novel insight into the allosteric regulatory mechanism of H-Ras.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhfQu8bG_CGrVg90H21EOLACvtfcHk0ljV_K9CsZdN8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitVOgtLzN&md5=61ae175456844dedba08536f410e343d</span></div><a href="/servlet/linkout?suffix=cit179&amp;dbid=16384&amp;doi=10.3390%2Fijms18112249&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms18112249%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DSong%26aufirst%3DK.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DMolecular%2520Dynamics%2520Simulations%2520and%2520Dynamic%2520Network%2520Analysis%2520Reveal%2520the%2520Allosteric%2520Unbinding%2520of%2520Monobody%2520to%2520H-Ras%2520Triggered%2520by%2520R135K%2520Mutation%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2017%26volume%3D18%26spage%3D2249%26doi%3D10.3390%2Fijms18112249" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref180"><div class="reference"><strong class="refLabel"><a href="#ref180" class="rightTabRefNumLink">180</a></strong><div class="NLM_citation" id="rightTab-cit180"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ni, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span> <span> </span><span class="NLM_article-title">Computational Insights into the Interactions between Calmodulin and the c/nSH2 Domains of p85α Regulatory Subunit of PI3Kα: Implication for PI3Kα Activation by Calmodulin</span>. <i>Int. J. Mol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">151</span>, <span class="refDoi"> DOI: 10.3390/ijms19010151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.3390%2Fijms19010151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref180/cit180&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXit1ahtLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2018&pages=151&author=D.+Niauthor=D.+Liuauthor=J.+Zhangauthor=S.+Lu&title=Computational+Insights+into+the+Interactions+between+Calmodulin+and+the+c%2FnSH2+Domains+of+p85%CE%B1+Regulatory+Subunit+of+PI3K%CE%B1%3A+Implication+for+PI3K%CE%B1+Activation+by+Calmodulin&doi=10.3390%2Fijms19010151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit180R"><div class="casContent"><span class="casTitleNuber">180</span><div class="casTitle"><span class="NLM_cas:atitle">Computational insights into the interactions between calmodulin and the c/nsh2 domains of p85α regulatory subunit of PI3K : implication for PI3Kα activation by Calmodulin</span></div><div class="casAuthors">Ni, Duan; Liu, Dingyu; Zhang, Jian; Lu, Shaoyong</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Sciences</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">151/1-151/15</span>CODEN:
                <span class="NLM_cas:coden">IJMCFK</span>;
        ISSN:<span class="NLM_cas:issn">1422-0067</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">Calmodulin (CaM) and phosphatidylinositide-3 kinase (PI3Kα) are well known for their multiple roles in a series of intracellular signaling pathways and in the progression of several human cancers.  Crosstalk between CaM and PI3Kα has been an area of intensive research.  Recent expts. have shown that in adenocarcinoma, K-Ras4B is involved in the CaM-PI3Kα crosstalk.  Based on exptl. results, we have recently put forward a hypothesis that the coordination of CaM and PI3Kα with K-Ras4B forms a CaM-PI3Kα-K-Ras4B ternary complex, which leads to the formation of pancreatic ductal adenocarcinoma.  However, the mechanism for the CaM-PI3Kα crosstalk is unresolved.  Based on mol. modeling and mol. dynamics simulations, here we explored the potential interactions between CaM and the c/nSH2 domains of p85α subunit of PI3Kα.  We demonstrated that CaM can interact with the c/nSH2 domains and the interaction details were unraveled.  Moreover, the possible modes for the CaM-cSH2 and CaM-nSH2 interactions were uncovered and we used them to construct a complete CaM-PI3Kα complex model.  The structural model of CaM-PI3Kα interaction not only offers a support for our previous ternary complex hypothesis, but also is useful for drug design targeted at CaM-PI3Kα protein-protein interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpUEQ7a-94mybVg90H21EOLACvtfcHk0ljV_K9CsZdN8g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXit1ahtLvM&md5=bf7d6c3c24a130a8cf596aa4bcee7fd4</span></div><a href="/servlet/linkout?suffix=cit180&amp;dbid=16384&amp;doi=10.3390%2Fijms19010151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fijms19010151%26sid%3Dliteratum%253Aachs%26aulast%3DNi%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DD.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DS.%26atitle%3DComputational%2520Insights%2520into%2520the%2520Interactions%2520between%2520Calmodulin%2520and%2520the%2520c%252FnSH2%2520Domains%2520of%2520p85%25CE%25B1%2520Regulatory%2520Subunit%2520of%2520PI3K%25CE%25B1%253A%2520Implication%2520for%2520PI3K%25CE%25B1%2520Activation%2520by%2520Calmodulin%26jtitle%3DInt.%2520J.%2520Mol.%2520Sci.%26date%3D2018%26volume%3D19%26spage%3D151%26doi%3D10.3390%2Fijms19010151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref181"><div class="reference"><strong class="refLabel"><a href="#ref181" class="rightTabRefNumLink">181</a></strong><div class="NLM_citation" id="rightTab-cit181"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stank, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kokh, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fuller, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wade, R. C.</span></span> <span> </span><span class="NLM_article-title">Protein Binding Pocket Dynamics</span>. <i>Acc. Chem. Res.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">809</span>– <span class="NLM_lpage">815</span>, <span class="refDoi"> DOI: 10.1021/acs.accounts.5b00516</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.accounts.5b00516" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref181/cit181&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xms1ansrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2016&pages=809-815&author=A.+Stankauthor=D.+B.+Kokhauthor=J.+C.+Fullerauthor=R.+C.+Wade&title=Protein+Binding+Pocket+Dynamics&doi=10.1021%2Facs.accounts.5b00516"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit181R"><div class="casContent"><span class="casTitleNuber">181</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Binding Pocket Dynamics</span></div><div class="casAuthors">Stank, Antonia; Kokh, Daria B.; Fuller, Jonathan C.; Wade, Rebecca C.</div><div class="citationInfo"><span class="NLM_cas:title">Accounts of Chemical Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">809-815</span>CODEN:
                <span class="NLM_cas:coden">ACHRE4</span>;
        ISSN:<span class="NLM_cas:issn">0001-4842</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The dynamics of protein binding pockets are crucial for their interaction specificity.  Structural flexibility allows proteins to adapt to their individual mol. binding partners and facilitates the binding process.  This implies the necessity to consider protein internal motion in detg. and predicting binding properties and in designing new binders.  Although accounting for protein dynamics presents a challenge for computational approaches, it expands the structural and physicochem. space for compd. design and thus offers the prospect of improved binding specificity and selectivity.A cavity on the surface or in the interior of a protein that possesses suitable properties for binding a ligand is usually referred to as a binding pocket.  The set of amino acid residues around a binding pocket dets. its physicochem. characteristics and, together with its shape and location in a protein, defines its functionality.  Residues outside the binding site can also have a long-range effect on the properties of the binding pocket.  Cavities with similar functionalities are often conserved across protein families.  For example, enzyme active sites are usually concave surfaces that present amino acid residues in a suitable configuration for binding low mol. wt. compds.  Macromol. binding pockets, on the other hand, are located on the protein surface and are often shallower.  The mobility of proteins allows the opening, closing, and adaptation of binding pockets to regulate binding processes and specific protein functionalities.  For example, channels and tunnels can exist permanently or transiently to transport compds. to and from a binding site.  The influence of protein flexibility on binding pockets can vary from small changes to an already existent pocket to the formation of a completely new pocket.  Here, we review recent developments in computational methods to detect and define binding pockets and to study pocket dynamics.  We introduce five different classes of protein pocket dynamics: (1) appearance/disappearance of a subpocket in an existing pocket; (2) appearance/disappearance of an adjacent pocket on the protein surface in the direct vicinity of an already existing pocket; (3) pocket breathing, which may be caused by side-chain fluctuations or backbone or interdomain vibrational motion; (4) opening/closing of a channel or tunnel, connecting a pocket inside the protein with solvent, including lid motion; and (5) the appearance/disappearance of an allosteric pocket at a site on a protein distinct from an already existing pocket with binding of a ligand to the allosteric binding site affecting the original pocket.We suggest that the class of pocket dynamics, as well as the type and extent of protein motion affecting the binding pocket, should be factors considered in choosing the most appropriate computational approach to study a given binding pocket.  Furthermore, we examine the relationship between pocket dynamics classes and induced fit, conformational selection, and gating models of ligand binding on binding kinetics and thermodn.  We discuss the implications of protein binding pocket dynamics for drug design and conclude with potential future directions for computational anal. of protein binding pocket dynamics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJ6TUcwjXcebVg90H21EOLACvtfcHk0ljbY9hS0vtiTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xms1ansrs%253D&md5=5c16ef089e87f13bf004f0935d1e4f35</span></div><a href="/servlet/linkout?suffix=cit181&amp;dbid=16384&amp;doi=10.1021%2Facs.accounts.5b00516&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.accounts.5b00516%26sid%3Dliteratum%253Aachs%26aulast%3DStank%26aufirst%3DA.%26aulast%3DKokh%26aufirst%3DD.%2BB.%26aulast%3DFuller%26aufirst%3DJ.%2BC.%26aulast%3DWade%26aufirst%3DR.%2BC.%26atitle%3DProtein%2520Binding%2520Pocket%2520Dynamics%26jtitle%3DAcc.%2520Chem.%2520Res.%26date%3D2016%26volume%3D49%26spage%3D809%26epage%3D815%26doi%3D10.1021%2Facs.accounts.5b00516" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref182"><div class="reference"><strong class="refLabel"><a href="#ref182" class="rightTabRefNumLink">182</a></strong><div class="NLM_citation" id="rightTab-cit182"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, H. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valant, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borhani, D. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Valcourt, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arlow, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canals, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lane, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rahmani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baell, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Christopoulos, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Modulation of a G-Protein-Coupled Receptor by Allosteric Drugs</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>503</i></span>,  <span class="NLM_fpage">295</span>– <span class="NLM_lpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature12595</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fnature12595" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24121438" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref182/cit182&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=503&publication_year=2013&pages=295-299&author=R.+O.+Drorauthor=H.+F.+Greenauthor=C.+Valantauthor=D.+W.+Borhaniauthor=J.+R.+Valcourtauthor=A.+C.+Panauthor=D.+H.+Arlowauthor=M.+Canalsauthor=J.+R.+Laneauthor=R.+Rahmaniauthor=J.+B.+Baellauthor=P.+M.+Sextonauthor=A.+Christopoulosauthor=D.+E.+Shaw&title=Structural+Basis+for+Modulation+of+a+G-Protein-Coupled+Receptor+by+Allosteric+Drugs&doi=10.1038%2Fnature12595"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit182R"><div class="casContent"><span class="casTitleNuber">182</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for modulation of a G-protein-coupled receptor by allosteric drugs</span></div><div class="casAuthors">Dror, Ron O.; Green, Hillary F.; Valant, Celine; Borhani, David W.; Valcourt, James R.; Pan, Albert C.; Arlow, Daniel H.; Canals, Meritxell; Lane, J. Robert; Rahmani, Raphael; Baell, Jonathan B.; Sexton, Patrick M.; Christopoulos, Arthur; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">503</span>
        (<span class="NLM_cas:issue">7475</span>),
    <span class="NLM_cas:pages">295-299</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The design of G-protein-coupled receptor (GPCR) allosteric modulators, an active area of modern pharmaceutical research, has proved challenging because neither the binding modes nor the mol. mechanisms of such drugs are known.  Here we det. binding sites, bound conformations and specific drug-receptor interactions for several allosteric modulators of the M2 muscarinic acetylcholine receptor (M2 receptor), a prototypical family A GPCR, using at.-level simulations in which the modulators spontaneously assoc. with the receptor.  Despite substantial structural diversity, all modulators form cation-π interactions with clusters of arom. residues in the receptor extracellular vestibule, approx. 15 Å from the classical, 'orthosteric' ligand-binding site.  We validate the obsd. modulator binding modes through radioligand binding expts. on receptor mutants designed, on the basis of our simulations, either to increase or to decrease modulator affinity.  Simulations also revealed mechanisms that contribute to pos. and neg. allosteric modulation of classical ligand binding, including coupled conformational changes of the two binding sites and electrostatic interactions between ligands in these sites.  These observations enabled the design of chem. modifications that substantially alter a modulator's allosteric effects.  Our findings thus provide a structural basis for the rational design of allosteric modulators targeting muscarinic and possibly other GPCRs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTlvAcaBf-1rVg90H21EOLACvtfcHk0ljbY9hS0vtiTg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSns7nN&md5=2fa40ecd46ce62125ce4a12c18270b1f</span></div><a href="/servlet/linkout?suffix=cit182&amp;dbid=16384&amp;doi=10.1038%2Fnature12595&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature12595%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DGreen%26aufirst%3DH.%2BF.%26aulast%3DValant%26aufirst%3DC.%26aulast%3DBorhani%26aufirst%3DD.%2BW.%26aulast%3DValcourt%26aufirst%3DJ.%2BR.%26aulast%3DPan%26aufirst%3DA.%2BC.%26aulast%3DArlow%26aufirst%3DD.%2BH.%26aulast%3DCanals%26aufirst%3DM.%26aulast%3DLane%26aufirst%3DJ.%2BR.%26aulast%3DRahmani%26aufirst%3DR.%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DSexton%26aufirst%3DP.%2BM.%26aulast%3DChristopoulos%26aufirst%3DA.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DStructural%2520Basis%2520for%2520Modulation%2520of%2520a%2520G-Protein-Coupled%2520Receptor%2520by%2520Allosteric%2520Drugs%26jtitle%3DNature%26date%3D2013%26volume%3D503%26spage%3D295%26epage%3D299%26doi%3D10.1038%2Fnature12595" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref183"><div class="reference"><strong class="refLabel"><a href="#ref183" class="rightTabRefNumLink">183</a></strong><div class="NLM_citation" id="rightTab-cit183"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hertig, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorraca, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span> <span> </span><span class="NLM_article-title">Revealing Atomic-Level Mechanisms of Protein Allostery with Molecular Dynamics Simulations</span>. <i>PLoS Comput. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e1004746</span> <span class="refDoi"> DOI: 10.1371/journal.pcbi.1004746</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1371%2Fjournal.pcbi.1004746" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=27285999" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref183/cit183&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhslyit7nI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&author=S.+Hertigauthor=N.+R.+Latorracaauthor=R.+O.+Dror&title=Revealing+Atomic-Level+Mechanisms+of+Protein+Allostery+with+Molecular+Dynamics+Simulations&doi=10.1371%2Fjournal.pcbi.1004746"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit183R"><div class="casContent"><span class="casTitleNuber">183</span><div class="casTitle"><span class="NLM_cas:atitle">Revealing atomic-level mechanisms of protein allostery with molecular dynamics simulations</span></div><div class="casAuthors">Hertig, Samuel; Latorraca, Naomi R.; Dror, Ron O.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Computational Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e1004746/1-e1004746/16</span>CODEN:
                <span class="NLM_cas:coden">PCBLBG</span>;
        ISSN:<span class="NLM_cas:issn">1553-7358</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Mol. dynamics (MD) simulations have become a powerful and popular method for the study of protein allostery, the widespread phenomenon in which a stimulus at one site on a protein influences the properties of another site on the protein.  By capturing the motions of a protein's constituent atoms, simulations can enable the discovery of allosteric binding sites and the detn. of the mechanistic basis for allostery.  These results can provide a foundation for applications including rational drug design and protein engineering.  Here, we provide an introduction to the investigation of protein allostery using mol. dynamics simulation.  We emphasize the importance of designing simulations that include appropriate perturbations to the mol. system, such as the addn. or removal of ligands or the application of mech. force.  We also demonstrate how the bidirectional nature of allostery- the fact that the two sites involved influence one another in a sym. manner- can facilitate such investigations.  Through a series of case studies, we illustrate how these concepts have been used to reveal the structural basis for allostery in several proteins and protein complexes of biol. and pharmaceutical interest.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGobTyr-sWnXWLVg90H21EOLACvtfcHk0liYhv75h15zLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhslyit7nI&md5=40178de113d54ac2c453d372e2dd5bfb</span></div><a href="/servlet/linkout?suffix=cit183&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pcbi.1004746&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pcbi.1004746%26sid%3Dliteratum%253Aachs%26aulast%3DHertig%26aufirst%3DS.%26aulast%3DLatorraca%26aufirst%3DN.%2BR.%26aulast%3DDror%26aufirst%3DR.%2BO.%26atitle%3DRevealing%2520Atomic-Level%2520Mechanisms%2520of%2520Protein%2520Allostery%2520with%2520Molecular%2520Dynamics%2520Simulations%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2016%26volume%3D12%26doi%3D10.1371%2Fjournal.pcbi.1004746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref184"><div class="reference"><strong class="refLabel"><a href="#ref184" class="rightTabRefNumLink">184</a></strong><div class="NLM_citation" id="rightTab-cit184"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banerjee, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaponenko, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span> <span> </span><span class="NLM_article-title">GTP Binding and Oncogenic Mutations May Attenuate Hypervariable Region (HVR)-Catalytic Domain Interactions in Small GTPase K-Ras4B, Exposing the Effector Binding Site</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">28887</span>– <span class="NLM_lpage">28900</span>, <span class="refDoi"> DOI: 10.1074/jbc.M115.664755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1074%2Fjbc.M115.664755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=26453300" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref184/cit184&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvFWqtbvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=28887-28900&author=S.+Luauthor=A.+Banerjeeauthor=H.+Jangauthor=J.+Zhangauthor=V.+Gaponenkoauthor=R.+Nussinov&title=GTP+Binding+and+Oncogenic+Mutations+May+Attenuate+Hypervariable+Region+%28HVR%29-Catalytic+Domain+Interactions+in+Small+GTPase+K-Ras4B%2C+Exposing+the+Effector+Binding+Site&doi=10.1074%2Fjbc.M115.664755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit184R"><div class="casContent"><span class="casTitleNuber">184</span><div class="casTitle"><span class="NLM_cas:atitle">GTP Binding and Oncogenic Mutations May Attenuate Hypervariable Region (HVR)-Catalytic Domain Interactions in Small GTPase K-Ras4B, Exposing the Effector Binding Site</span></div><div class="casAuthors">Lu, Shaoyong; Banerjee, Avik; Jang, Hyunbum; Zhang, Jian; Gaponenko, Vadim; Nussinov, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">48</span>),
    <span class="NLM_cas:pages">28887-28900</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">K-Ras4B, a frequently mutated oncogene in cancer, plays an essential role in cell growth, differentiation, and survival.  Its C-terminal membrane-assocd. hypervariable region (HVR) is required for full biol. activity.  In the active GTP-bound state, the HVR interacts with acidic plasma membrane (PM) headgroups, whereas the farnesyl anchors in the membrane; in the inactive GDP-bound state, the HVR may interact with both the PM and the catalytic domain at the effector binding region, obstructing signaling and nucleotide exchange.  Here, using mol. dynamics simulations and NMR, we aim to figure out the effects of nucleotides (GTP and GDP) and frequent (G12C, G12D, G12V, G13D, and Q61H) and infrequent (E37K and R164Q) oncogenic mutations on full-length K-Ras4B.  The mutations are away from or directly at the HVR switch I/effector binding site.  Our results suggest that full-length wild-type GDP-bound K-Ras4B (K-Ras4BWT-GDP) is in an intrinsically autoinhibited state via tight HVR-catalytic domain interactions.  The looser assocn. in K-Ras4BWT-GTP may release the HVR.  Some of the oncogenic mutations weaken the HVR-catalytic domain assocn. in the K-Ras4B-GDP/-GTP bound states, which may facilitate the HVR dissocn. in a nucleotide-independent manner, thereby up-regulating oncogenic Ras signaling.  Thus, our results suggest that mutations can exert their effects in more than one way, abolishing GTP hydrolysis and facilitating effector binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPP0csIe-DcLVg90H21EOLACvtfcHk0liYhv75h15zLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvFWqtbvK&md5=4046eed0caa6a965930b6c293d90ed0e</span></div><a href="/servlet/linkout?suffix=cit184&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M115.664755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M115.664755%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DBanerjee%26aufirst%3DA.%26aulast%3DJang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DGaponenko%26aufirst%3DV.%26aulast%3DNussinov%26aufirst%3DR.%26atitle%3DGTP%2520Binding%2520and%2520Oncogenic%2520Mutations%2520May%2520Attenuate%2520Hypervariable%2520Region%2520%2528HVR%2529-Catalytic%2520Domain%2520Interactions%2520in%2520Small%2520GTPase%2520K-Ras4B%252C%2520Exposing%2520the%2520Effector%2520Binding%2520Site%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D28887%26epage%3D28900%26doi%3D10.1074%2Fjbc.M115.664755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref185"><div class="reference"><strong class="refLabel"><a href="#ref185" class="rightTabRefNumLink">185</a></strong><div class="NLM_citation" id="rightTab-cit185"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shukla, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roux, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Activation Pathway of Src Kinase Reveals Intermediate States as Targets for Drug Design</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">3397</span>, <span class="refDoi"> DOI: 10.1038/ncomms4397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fncomms4397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24584478" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref185/cit185&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A280%3ADC%252BC2cvpsF2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3397&author=D.+Shuklaauthor=Y.+Mengauthor=B.+Rouxauthor=V.+S.+Pande&title=Activation+Pathway+of+Src+Kinase+Reveals+Intermediate+States+as+Targets+for+Drug+Design&doi=10.1038%2Fncomms4397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit185R"><div class="casContent"><span class="casTitleNuber">185</span><div class="casTitle"><span class="NLM_cas:atitle">Activation pathway of Src kinase reveals intermediate states as targets for drug design</span></div><div class="casAuthors">Shukla Diwakar; Pande Vijay S; Meng Yilin; Roux Benoit</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3397</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Unregulated activation of Src kinases leads to aberrant signalling, uncontrolled growth and differentiation of cancerous cells.  Reaching a complete mechanistic understanding of large-scale conformational transformations underlying the activation of kinases could greatly help in the development of therapeutic drugs for the treatment of these pathologies.  In principle, the nature of conformational transition could be modelled in silico via atomistic molecular dynamics simulations, although this is very challenging because of the long activation timescales.  Here we employ a computational paradigm that couples transition pathway techniques and Markov state model-based massively distributed simulations for mapping the conformational landscape of c-src tyrosine kinase.  The computations provide the thermodynamics and kinetics of kinase activation for the first time, and help identify key structural intermediates.  Furthermore, the presence of a novel allosteric site in an intermediate state of c-src that could be potentially used for drug design is predicted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ2z_2F1JvAAYvKgEt20y1XfW6udTcc2ebOq_dQvyMFQbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2cvpsF2huw%253D%253D&md5=f37ef7a345fdf37b69a83c677ef3d851</span></div><a href="/servlet/linkout?suffix=cit185&amp;dbid=16384&amp;doi=10.1038%2Fncomms4397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4397%26sid%3Dliteratum%253Aachs%26aulast%3DShukla%26aufirst%3DD.%26aulast%3DMeng%26aufirst%3DY.%26aulast%3DRoux%26aufirst%3DB.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DActivation%2520Pathway%2520of%2520Src%2520Kinase%2520Reveals%2520Intermediate%2520States%2520as%2520Targets%2520for%2520Drug%2520Design%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3397%26doi%3D10.1038%2Fncomms4397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref186"><div class="reference"><strong class="refLabel"><a href="#ref186" class="rightTabRefNumLink">186</a></strong><div class="NLM_citation" id="rightTab-cit186"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Betzi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alam, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lubbers, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Han, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakkaraj, S. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Georg, G. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schönbrunn, E.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Potential Allosteric Ligand Binding Site in CDK2</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">492</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1021/cb100410m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb100410m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref186/cit186&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXit1Gisb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=492-501&author=S.+Betziauthor=R.+Alamauthor=M.+Martinauthor=D.+J.+Lubbersauthor=H.+Hanauthor=S.+R.+Jakkarajauthor=G.+I.+Georgauthor=E.+Sch%C3%B6nbrunn&title=Discovery+of+a+Potential+Allosteric+Ligand+Binding+Site+in+CDK2&doi=10.1021%2Fcb100410m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit186R"><div class="casContent"><span class="casTitleNuber">186</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potential Allosteric Ligand Binding Site in CDK2</span></div><div class="casAuthors">Betzi, Stephane; Alam, Riazul; Martin, Mathew; Lubbers, Donna J.; Han, Huijong; Jakkaraj, Sudhakar R.; Georg, Gunda I.; Schonbrunn, Ernst</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">492-501</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases (CDKs) are key regulatory enzymes in cell cycle progression and transcription.  Aberrant activity of CDKs has been implicated in a no. of medical conditions, and numerous small mol. CDK inhibitors have been reported as potential drug leads.  However, these inhibitors exclusively bind to the ATP site, which is largely conserved among protein kinases, and clin. trials have not resulted in viable drug candidates, attributed in part to the lack of target selectivity.  CDKs are unique among protein kinases, as their functionality strictly depends on assocn. with their partner proteins, the cyclins.  In an effort to identify potential target sites for disruption of the CDK-cyclin interaction, we probed the extrinsic fluorophore 8-anilino-1-naphthalene sulfonate (ANS) with human CDK2 and cyclin A using fluorescence spectroscopy and protein crystallog.  ANS interacts with free CDK2 in a satn.-dependent manner with an apparent Kd of 37 μM, and cyclin A displaced ANS from CDK2 with an EC50 value of 0.6 μM.  Co-crystal structures with ANS alone and in ternary complex with ATP site-directed inhibitors revealed two ANS mols. bound adjacent to one another, away from the ATP site, in a large pocket that extends from the DFG region above the C-helix.  Binding of ANS is accompanied by substantial structural changes in CDK2, resulting in a C-helix conformation that is incompatible for cyclin A assocn.  These findings indicate the potential of the ANS binding pocket as a new target site for allosteric inhibitors disrupting the interaction of CDKs and cyclins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoGlhg8YO4Yl7Vg90H21EOLACvtfcHk0ljhXLLUhCpeCw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXit1Gisb4%253D&md5=893aed13b1638f7626fbb4851014fe6e</span></div><a href="/servlet/linkout?suffix=cit186&amp;dbid=16384&amp;doi=10.1021%2Fcb100410m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb100410m%26sid%3Dliteratum%253Aachs%26aulast%3DBetzi%26aufirst%3DS.%26aulast%3DAlam%26aufirst%3DR.%26aulast%3DMartin%26aufirst%3DM.%26aulast%3DLubbers%26aufirst%3DD.%2BJ.%26aulast%3DHan%26aufirst%3DH.%26aulast%3DJakkaraj%26aufirst%3DS.%2BR.%26aulast%3DGeorg%26aufirst%3DG.%2BI.%26aulast%3DSch%25C3%25B6nbrunn%26aufirst%3DE.%26atitle%3DDiscovery%2520of%2520a%2520Potential%2520Allosteric%2520Ligand%2520Binding%2520Site%2520in%2520CDK2%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2011%26volume%3D6%26spage%3D492%26epage%3D501%26doi%3D10.1021%2Fcb100410m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref187"><div class="reference"><strong class="refLabel"><a href="#ref187" class="rightTabRefNumLink">187</a></strong><div class="NLM_citation" id="rightTab-cit187"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sultan, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kiss, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Towards Simple Kinetic Models of Functional Dynamics for a Kinase Subfamily</span>. <i>Nat. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>10</i></span>,  <span class="NLM_fpage">903</span>– <span class="NLM_lpage">909</span>, <span class="refDoi"> DOI: 10.1038/s41557-018-0077-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1038%2Fs41557-018-0077-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=29988151" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref187/cit187&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhtlWnsrjM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2018&pages=903-909&author=M.+M.+Sultanauthor=G.+Kissauthor=V.+S.+Pande&title=Towards+Simple+Kinetic+Models+of+Functional+Dynamics+for+a+Kinase+Subfamily&doi=10.1038%2Fs41557-018-0077-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit187R"><div class="casContent"><span class="casTitleNuber">187</span><div class="casTitle"><span class="NLM_cas:atitle">Towards simple kinetic models of functional dynamics for a kinase subfamily</span></div><div class="casAuthors">Sultan, Mohammad M.; Kiss, Gert; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">903-909</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Kinases are ubiquitous enzymes involved in the regulation of crit. cellular pathways.  However, in silico modeling of the conformational ensembles of these enzymes is difficult due to inherent limitations and the cost of computational approaches.  Recent algorithmic advances combined with homol. modeling and parallel simulations have enabled researchers to address this computational sampling bottleneck.  Here, we present the results of mol. dynamics studies for seven Src family kinase (SFK) members: Fyn, Lyn, Lck, Hck, Fgr, Yes and Blk.  We present a sequence invariant extension to Markov state models, which allows us to quant. compare the structural ensembles of the seven kinases.  Our findings indicate that in the absence of their regulatory partners, SFK members have similar in silico dynamics with active state populations ranging from 4 to 40% and activation timescales in the hundreds of microseconds.  Furthermore, we observe several potentially druggable intermediate states, including a pocket next to the ATP-binding site that could potentially be targeted via a small-mol. inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMN4rFMADt6bVg90H21EOLACvtfcHk0lgwiuMO0BevfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhtlWnsrjM&md5=2f5e46697674a52b248e6d09202312f3</span></div><a href="/servlet/linkout?suffix=cit187&amp;dbid=16384&amp;doi=10.1038%2Fs41557-018-0077-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41557-018-0077-9%26sid%3Dliteratum%253Aachs%26aulast%3DSultan%26aufirst%3DM.%2BM.%26aulast%3DKiss%26aufirst%3DG.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DTowards%2520Simple%2520Kinetic%2520Models%2520of%2520Functional%2520Dynamics%2520for%2520a%2520Kinase%2520Subfamily%26jtitle%3DNat.%2520Chem.%26date%3D2018%26volume%3D10%26spage%3D903%26epage%3D909%26doi%3D10.1038%2Fs41557-018-0077-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref188"><div class="reference"><strong class="refLabel"><a href="#ref188" class="rightTabRefNumLink">188</a></strong><div class="NLM_citation" id="rightTab-cit188"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Husic, B. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pande, V. S.</span></span> <span> </span><span class="NLM_article-title">Markov State Models: From an Art to a Science</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>140</i></span>,  <span class="NLM_fpage">2386</span>– <span class="NLM_lpage">2396</span>, <span class="refDoi"> DOI: 10.1021/jacs.7b12191</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jacs.7b12191" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref188/cit188&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=140&publication_year=2018&pages=2386-2396&author=B.+E.+Husicauthor=V.+S.+Pande&title=Markov+State+Models%3A+From+an+Art+to+a+Science&doi=10.1021%2Fjacs.7b12191"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit188R"><div class="casContent"><span class="casTitleNuber">188</span><div class="casTitle"><span class="NLM_cas:atitle">Markov State Models: From an Art to a Science</span></div><div class="casAuthors">Husic, Brooke E.; Pande, Vijay S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">140</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2386-2396</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Markov state models (MSMs) are a powerful framework for analyzing dynamical systems, such as mol. dynamics (MD) simulations, that have gained widespread use over the past several decades.  This perspective offers an overview of the MSM field to date, presented for a general audience as a timeline of key developments in the field.  We sequentially address early studies that motivated the method, canonical papers that established the use of MSMs for MD anal., and subsequent advances in software and anal. protocols.  The derivation of a variational principle for MSMs in 2013 signified a turning point from expertise-driving MSM building to a systematic, objective protocol.  The variational approach, combined with best practices for model selection and open-source software, enabled a wide range of MSM anal. for applications such as protein folding and allostery, ligand binding, and protein-protein assocn.  To conclude, the current frontiers of methods development are highlighted, as well as exciting applications in exptl. design and drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo43eE2dHXoHrVg90H21EOLACvtfcHk0lgwiuMO0BevfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotV2huw%253D%253D&md5=f00943acedec985c4d21abd86ecfce4a</span></div><a href="/servlet/linkout?suffix=cit188&amp;dbid=16384&amp;doi=10.1021%2Fjacs.7b12191&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjacs.7b12191%26sid%3Dliteratum%253Aachs%26aulast%3DHusic%26aufirst%3DB.%2BE.%26aulast%3DPande%26aufirst%3DV.%2BS.%26atitle%3DMarkov%2520State%2520Models%253A%2520From%2520an%2520Art%2520to%2520a%2520Science%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2018%26volume%3D140%26spage%3D2386%26epage%3D2396%26doi%3D10.1021%2Fjacs.7b12191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref189"><div class="reference"><strong class="refLabel"><a href="#ref189" class="rightTabRefNumLink">189</a></strong><div class="NLM_citation" id="rightTab-cit189"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fronik, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gaiser, B. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sejer
Pedersen, D.</span></span> <span> </span><span class="NLM_article-title">Bitopic Ligands and Metastable Binding Sites: Opportunities for G Protein-Coupled Receptor (GPCR) Medicinal Chemistry</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">4126</span>– <span class="NLM_lpage">4134</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01601</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01601" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref189/cit189&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVyktbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=4126-4134&author=P.+Fronikauthor=B.+I.+Gaiserauthor=D.+Sejer%0APedersen&title=Bitopic+Ligands+and+Metastable+Binding+Sites%3A+Opportunities+for+G+Protein-Coupled+Receptor+%28GPCR%29+Medicinal+Chemistry&doi=10.1021%2Facs.jmedchem.6b01601"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit189R"><div class="casContent"><span class="casTitleNuber">189</span><div class="casTitle"><span class="NLM_cas:atitle">Bitopic Ligands and Metastable Binding Sites: Opportunities for G Protein-Coupled Receptor (GPCR) Medicinal Chemistry</span></div><div class="casAuthors">Fronik, Philipp; Gaiser, Birgit I.; Sejer Pedersen, Daniel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">4126-4134</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  G protein-coupled receptors (GPCRs) belong to a large superfamily of membrane receptors mediating a variety of physiol. functions.  As such they are attractive targets for drug therapy.  However, it remains a challenge to develop subtype selective GPCR ligands, due to the high conservation of orthosteric binding sites.  Bitopic ligands have been employed to address the selectivity problem by combining (linking) an orthosteric ligand with an allosteric modulator, theor. leading to high-affinity subtype selective ligands.  However, it remains a challenge to identify suitable allosteric binding sites.  Computational studies on ligand binding to GPCRs have revealed transient, low-affinity binding sites, termed metastable binding sites.  Metastable binding sites may provide a new source of allosteric binding sites that could be exploited in the design of bitopic ligands.  Unlike the bitopic ligands that have been reported to date this type of bitopic ligands would be composed of two identical pharmacophores.  Herein, the authors outline the concept of bitopic ligands, review metastable binding sites and discuss their potential as a new source of allosteric binding sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpihAXt959S37Vg90H21EOLACvtfcHk0lgwiuMO0BevfA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVyktbw%253D&md5=a20875145b65a948c772e988ff1217bc</span></div><a href="/servlet/linkout?suffix=cit189&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01601&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01601%26sid%3Dliteratum%253Aachs%26aulast%3DFronik%26aufirst%3DP.%26aulast%3DGaiser%26aufirst%3DB.%2BI.%26aulast%3DSejer%2BPedersen%26aufirst%3DD.%26atitle%3DBitopic%2520Ligands%2520and%2520Metastable%2520Binding%2520Sites%253A%2520Opportunities%2520for%2520G%2520Protein-Coupled%2520Receptor%2520%2528GPCR%2529%2520Medicinal%2520Chemistry%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D4126%26epage%3D4134%26doi%3D10.1021%2Facs.jmedchem.6b01601" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref190"><div class="reference"><strong class="refLabel"><a href="#ref190" class="rightTabRefNumLink">190</a></strong><div class="NLM_citation" id="rightTab-cit190"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowman, G. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bolin, E. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hart, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maguire, B. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marqusee, S.</span></span> <span> </span><span class="NLM_article-title">Discovery of Multiple Hidden Allosteric Sites by Combining Markov State Models and Experiments</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>112</i></span>,  <span class="NLM_fpage">2734</span>– <span class="NLM_lpage">2739</span>, <span class="refDoi"> DOI: 10.1073/pnas.1417811112</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1073%2Fpnas.1417811112" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=25730859" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref190/cit190&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2015&pages=2734-2739&author=G.+R.+Bowmanauthor=E.+R.+Bolinauthor=K.+M.+Hartauthor=B.+C.+Maguireauthor=S.+Marqusee&title=Discovery+of+Multiple+Hidden+Allosteric+Sites+by+Combining+Markov+State+Models+and+Experiments&doi=10.1073%2Fpnas.1417811112"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit190R"><div class="casContent"><span class="casTitleNuber">190</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of multiple hidden allosteric sites by combining Markov state models and experiments</span></div><div class="casAuthors">Bowman, Gregory R.; Bolin, Eric R.; Hart, Kathryn M.; Maguire, Brendan C.; Marqusee, Susan</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2734-2739</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The discovery of drug-like mols. that bind pockets in proteins that are not present in crystallog. structures yet exert allosteric control over activity has generated great interest in designing pharmaceuticals that exploit allosteric effects.  However, there have only been a small no. of successes, so the therapeutic potential of these pockets--called hidden allosteric sites--remains unclear.  One challenge for assessing their utility is that rational drug design approaches require foreknowledge of the target site, but most hidden allosteric sites are only discovered when a small mol. is found to stabilize them.  The authors present a means of decoupling the identification of hidden allosteric sites from the discovery of drugs that bind them by drawing on new developments in Markov state modeling that provide unprecedented access to microsecond- to millisecond-timescale fluctuations of a protein's structure.  Visualizing these fluctuations allows the authors to identify potential hidden allosteric sites, which the authors then test via thiol labeling expts.  Application of these methods reveals multiple hidden allosteric sites in an important antibiotic target-TEM-1 β-lactamase.  This result supports the hypothesis that there are many as yet undiscovered hidden allosteric sites and suggests the authors' methodol. can identify such sites, providing a starting point for future drug design efforts.  More generally, the authors' results demonstrate the power of using Markov state models to guide expts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY4EqjsensDLVg90H21EOLACvtfcHk0ljeuvz-_yWlsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXivFajsbk%253D&md5=f20db19a2f0f13679bc64f4616c856e7</span></div><a href="/servlet/linkout?suffix=cit190&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1417811112&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1417811112%26sid%3Dliteratum%253Aachs%26aulast%3DBowman%26aufirst%3DG.%2BR.%26aulast%3DBolin%26aufirst%3DE.%2BR.%26aulast%3DHart%26aufirst%3DK.%2BM.%26aulast%3DMaguire%26aufirst%3DB.%2BC.%26aulast%3DMarqusee%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Multiple%2520Hidden%2520Allosteric%2520Sites%2520by%2520Combining%2520Markov%2520State%2520Models%2520and%2520Experiments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2015%26volume%3D112%26spage%3D2734%26epage%3D2739%26doi%3D10.1073%2Fpnas.1417811112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref191"><div class="reference"><strong class="refLabel"><a href="#ref191" class="rightTabRefNumLink">191</a></strong><div class="NLM_citation" id="rightTab-cit191"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Harnessing Allostery: A Novel Approach to Drug Discovery</span>. <i>Med. Res. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>34</i></span>,  <span class="NLM_fpage">1242</span>– <span class="NLM_lpage">1285</span>, <span class="refDoi"> DOI: 10.1002/med.21317</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1002%2Fmed.21317" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=24827416" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref191/cit191&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslalu7rO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2014&pages=1242-1285&author=S.+Luauthor=S.+Liauthor=J.+Zhang&title=Harnessing+Allostery%3A+A+Novel+Approach+to+Drug+Discovery&doi=10.1002%2Fmed.21317"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit191R"><div class="casContent"><span class="casTitleNuber">191</span><div class="casTitle"><span class="NLM_cas:atitle">Harnessing Allostery: A Novel Approach to Drug Discovery</span></div><div class="casAuthors">Lu, Shaoyong; Li, Shuai; Zhang, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Research Reviews</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1242-1285</span>CODEN:
                <span class="NLM_cas:coden">MRREDD</span>;
        ISSN:<span class="NLM_cas:issn">0198-6325</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">Allostery is the most direct and efficient way for regulation of biol. macromol. function, ranging from the control of metabolic mechanisms to signal transduction pathways.  Allosteric modulators target to allosteric sites, offering distinct advantages compared to orthosteric ligands that target to active sites, such as greater specificity, reduced side effects, and lower toxicity.  Allosteric modulators have therefore drawn increasing attention as potential therapeutic drugs in the design and development of new drugs.  In recent years, advancements in our understanding of the fundamental principles underlying allostery, coupled with the exploitation of powerful techniques and methods in the field of allostery, provide unprecedented opportunities to discover allosteric proteins, detect and characterize allosteric sites, design and develop novel efficient allosteric drugs, and recapitulate the universal features of allosteric proteins and allosteric modulators.  In the present review, we summarize the recent advances in the repertoire of allostery, with a particular focus on the aforementioned allosteric compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpa1YgTPNMI5rVg90H21EOLACvtfcHk0ljeuvz-_yWlsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslalu7rO&md5=50ce678b5018e53915817e95ef670e88</span></div><a href="/servlet/linkout?suffix=cit191&amp;dbid=16384&amp;doi=10.1002%2Fmed.21317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fmed.21317%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DJ.%26atitle%3DHarnessing%2520Allostery%253A%2520A%2520Novel%2520Approach%2520to%2520Drug%2520Discovery%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2014%26volume%3D34%26spage%3D1242%26epage%3D1285%26doi%3D10.1002%2Fmed.21317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref192"><div class="reference"><strong class="refLabel"><a href="#ref192" class="rightTabRefNumLink">192</a></strong><div class="NLM_citation" id="rightTab-cit192"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dror, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dirks, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grossman, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Biomolecular Simulation: A Computational Microscope for Molecular Biology</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">429</span>– <span class="NLM_lpage">452</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-042910-155245</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-042910-155245" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=22577825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref192/cit192&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC38Xpt1yhs7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2012&pages=429-452&author=R.+O.+Drorauthor=R.+M.+Dirksauthor=J.+P.+Grossmanauthor=H.+Xuauthor=D.+E.+Shaw&title=Biomolecular+Simulation%3A+A+Computational+Microscope+for+Molecular+Biology&doi=10.1146%2Fannurev-biophys-042910-155245"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit192R"><div class="casContent"><span class="casTitleNuber">192</span><div class="casTitle"><span class="NLM_cas:atitle">Biomolecular simulation: a computational microscope for molecular biology</span></div><div class="casAuthors">Dror, Ron O.; Dirks, Robert M.; Grossman, J. P.; Xu, Huafeng; Shaw, David E.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">429-452</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews Inc.</span>)
        </div><div class="casAbstract">A review.  Mol. dynamics simulations capture the behavior of biol. macromols. in full at. detail, but their computational demands, combined with the challenge of appropriately modeling the relevant physics, have historically restricted their length and accuracy.  Dramatic recent improvements in achievable simulation speed and the underlying phys. models have enabled at.-level simulations on timescales as long as milliseconds that capture key biochem. processes such as protein folding, drug binding, membrane transport, and the conformational changes crit. to protein function.  Such simulation may serve as a computational microscope, revealing biomol. mechanisms at spatial and temporal scales that are difficult to observe exptl.  We describe the rapidly evolving state of the art for at.-level biomol. simulation, illustrate the types of biol. discoveries that can now be made through simulation, and discuss challenges motivating continued innovation in this field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIwvqxfn3SjLVg90H21EOLACvtfcHk0ljeuvz-_yWlsg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xpt1yhs7s%253D&md5=3f872bcd93c1c2141ef3f020c5c6d45d</span></div><a href="/servlet/linkout?suffix=cit192&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-042910-155245&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-042910-155245%26sid%3Dliteratum%253Aachs%26aulast%3DDror%26aufirst%3DR.%2BO.%26aulast%3DDirks%26aufirst%3DR.%2BM.%26aulast%3DGrossman%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DH.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26atitle%3DBiomolecular%2520Simulation%253A%2520A%2520Computational%2520Microscope%2520for%2520Molecular%2520Biology%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2012%26volume%3D41%26spage%3D429%26epage%3D452%26doi%3D10.1146%2Fannurev-biophys-042910-155245" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref193"><div class="reference"><strong class="refLabel"><a href="#ref193" class="rightTabRefNumLink">193</a></strong><div class="NLM_citation" id="rightTab-cit193"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basith, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, S.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Structure-Based Drug Design Targeting Class A G Protein-Coupled Receptors Utilizing Crystal Structures and Computational Simulations</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">46</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01453</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01453" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref193/cit193&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVyktLjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1-46&author=Y.+Leeauthor=S.+Basithauthor=S.+Choi&title=Recent+Advances+in+Structure-Based+Drug+Design+Targeting+Class+A+G+Protein-Coupled+Receptors+Utilizing+Crystal+Structures+and+Computational+Simulations&doi=10.1021%2Facs.jmedchem.6b01453"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit193R"><div class="casContent"><span class="casTitleNuber">193</span><div class="casTitle"><span class="NLM_cas:atitle">Recent Advances in Structure-Based Drug Design Targeting Class A G Protein-Coupled Receptors Utilizing Crystal Structures and Computational Simulations</span></div><div class="casAuthors">Lee, Yoonji; Basith, Shaherin; Choi, Sun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-46</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">G protein-coupled receptors (GPCRs) represent the largest and most physiol. important integral membrane protein family, and these receptors respond to a wide variety of physiol. and environmental stimuli.  GPCRs are among the most crit. therapeutic targets for numerous human diseases, and approx. one-third of the currently marketed drugs target this receptor family.  The recent breakthroughs in GPCR structural biol. have significantly contributed to our understanding of GPCR function, ligand binding, and pharmacol. action as well as to the design of new drugs.  This perspective highlights the latest advances in GPCR structures with a focus on the receptor-ligand interactions of each receptor family in class A nonrhodopsin GPCRs as well as the structural features for their activation, biased signaling, and allosteric mechanisms.  The current state-of-the-art methodologies of structure-based drug design (SBDD) approaches in the GPCR research field are also discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp90SKZ9osqkbVg90H21EOLACvtfcHk0libb_JzjhvWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVyktLjI&md5=0706fa6680cfe73076a0bdd8d1267240</span></div><a href="/servlet/linkout?suffix=cit193&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01453%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DY.%26aulast%3DBasith%26aufirst%3DS.%26aulast%3DChoi%26aufirst%3DS.%26atitle%3DRecent%2520Advances%2520in%2520Structure-Based%2520Drug%2520Design%2520Targeting%2520Class%2520A%2520G%2520Protein-Coupled%2520Receptors%2520Utilizing%2520Crystal%2520Structures%2520and%2520Computational%2520Simulations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1%26epage%3D46%26doi%3D10.1021%2Facs.jmedchem.6b01453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref194"><div class="reference"><strong class="refLabel"><a href="#ref194" class="rightTabRefNumLink">194</a></strong><div class="NLM_citation" id="rightTab-cit194"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.-J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, B.</span></span> <span> </span><span class="NLM_article-title">The Underappreciated Role of Allostery in the Cellular Network</span>. <i>Annu. Rev. Biophys.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>42</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">189</span>, <span class="refDoi"> DOI: 10.1146/annurev-biophys-083012-130257</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1146%2Fannurev-biophys-083012-130257" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=23451894" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref194/cit194&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFGrs7bJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2013&pages=169-189&author=R.+Nussinovauthor=C.-J.+Tsaiauthor=B.+Ma&title=The+Underappreciated+Role+of+Allostery+in+the+Cellular+Network&doi=10.1146%2Fannurev-biophys-083012-130257"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit194R"><div class="casContent"><span class="casTitleNuber">194</span><div class="casTitle"><span class="NLM_cas:atitle">The underappreciated role of allostery in the cellular network</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Ma, Buyong</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biophysics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">169-189</span>CODEN:
                <span class="NLM_cas:coden">ARBNCV</span>;
        ISSN:<span class="NLM_cas:issn">1936-122X</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review.  Allosteric propagation results in communication between distinct sites in the protein structure; it also encodes specific effects on cellular pathways, and in this way it shapes cellular response.  One example of long-range effects is binding of morphogens to cell surface receptors, which initiates a cascade of protein interactions that leads to genome activation and specific cellular action.  Allosteric propagation results from combinations of multiple factors, takes place through dynamic shifts of conformational ensembles, and affects the equil. of macromol. interactions.  Here, we (a) emphasize the well-known yet still underappreciated role of allostery in conveying explicit signals across large multimol. assemblies and distances to specify cellular action; (b) stress the need for quantitation of the allosteric effects; and finally, (c) propose that each specific combination of allosteric effectors along the pathway spells a distinct function.  The challenges are colossal; the inspiring reward will be predicting function, misfunction, and outcomes of drug regimes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqMSQUbLZg-bVg90H21EOLACvtfcHk0libb_JzjhvWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFGrs7bJ&md5=9d239bd6685c4c158ef706bd6081a183</span></div><a href="/servlet/linkout?suffix=cit194&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biophys-083012-130257&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biophys-083012-130257%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.-J.%26aulast%3DMa%26aufirst%3DB.%26atitle%3DThe%2520Underappreciated%2520Role%2520of%2520Allostery%2520in%2520the%2520Cellular%2520Network%26jtitle%3DAnnu.%2520Rev.%2520Biophys.%26date%3D2013%26volume%3D42%26spage%3D169%26epage%3D189%26doi%3D10.1146%2Fannurev-biophys-083012-130257" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref195"><div class="reference"><strong class="refLabel"><a href="#ref195" class="rightTabRefNumLink">195</a></strong><div class="NLM_citation" id="rightTab-cit195"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Pei, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yin, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, L.</span></span> <span> </span><span class="NLM_article-title">Systems Biology Brings New Dimensions for Structure-Based Drug Design</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">11556</span>– <span class="NLM_lpage">11565</span>, <span class="refDoi"> DOI: 10.1021/ja504810z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja504810z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref195/cit195&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=11556-11565&author=J.+Peiauthor=N.+Yinauthor=X.+Maauthor=L.+Lai&title=Systems+Biology+Brings+New+Dimensions+for+Structure-Based+Drug+Design&doi=10.1021%2Fja504810z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit195R"><div class="casContent"><span class="casTitleNuber">195</span><div class="casTitle"><span class="NLM_cas:atitle">Systems Biology Brings New Dimensions for Structure-Based Drug Design</span></div><div class="casAuthors">Pei, Jianfeng; Yin, Ning; Ma, Xiaomin; Lai, Luhua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">33</span>),
    <span class="NLM_cas:pages">11556-11565</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  In this Perspective, we focus on new, systems-centric views of structure-based drug design (SBDD) that we believe will impact future drug discovery research and development.  We will first discuss new ways to identify drug targets based on systems intervention anal., and then we will introduce emerging SBDD methods driven by advancements in systems biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreR4j6y808PrVg90H21EOLACvtfcHk0libb_JzjhvWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3E&md5=7834c0474a316442c136bb601ee3882c</span></div><a href="/servlet/linkout?suffix=cit195&amp;dbid=16384&amp;doi=10.1021%2Fja504810z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja504810z%26sid%3Dliteratum%253Aachs%26aulast%3DPei%26aufirst%3DJ.%26aulast%3DYin%26aufirst%3DN.%26aulast%3DMa%26aufirst%3DX.%26aulast%3DLai%26aufirst%3DL.%26atitle%3DSystems%2520Biology%2520Brings%2520New%2520Dimensions%2520for%2520Structure-Based%2520Drug%2520Design%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D11556%26epage%3D11565%26doi%3D10.1021%2Fja504810z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref196"><div class="reference"><strong class="refLabel"><a href="#ref196" class="rightTabRefNumLink">196</a></strong><div class="NLM_citation" id="rightTab-cit196"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span> <span> </span><span class="NLM_article-title">Principles of Allosteric Interactions in Cell Signaling</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>136</i></span>,  <span class="NLM_fpage">17692</span>– <span class="NLM_lpage">17701</span>, <span class="refDoi"> DOI: 10.1021/ja510028c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja510028c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref196/cit196&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVegurnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=136&publication_year=2014&pages=17692-17701&author=R.+Nussinovauthor=C.+Tsaiauthor=J.+Liu&title=Principles+of+Allosteric+Interactions+in+Cell+Signaling&doi=10.1021%2Fja510028c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit196R"><div class="casContent"><span class="casTitleNuber">196</span><div class="casTitle"><span class="NLM_cas:atitle">Principles of allosteric interactions in cell signaling</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Liu, Jin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">136</span>
        (<span class="NLM_cas:issue">51</span>),
    <span class="NLM_cas:pages">17692-17701</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Linking cell signaling events to the fundamental physicochem. basis of the conformational behavior of single mols. and ultimately to cellular function is a key challenge facing the life sciences.  Here, the authors outline the emerging principles of allosteric interactions in cell signaling, with emphasis on the following points.  (1) Allosteric efficacy is not a function of the chem. compn. of the allosteric pocket but reflects the extent of the population shift between the inactive and active states; i.e., the allosteric effect is detd. by the extent of preferred binding, not by the overall binding affinity.  (2) Coupling between the allosteric and active sites does not decide the allosteric effect; however, it does define the propagation pathways, the allosteric binding sites, and key on-path residues.  (3) Atoms of allosteric effectors can act as "driver" or "anchor" and create attractive "pulling" or repulsive "pushing" interactions.  Deciphering, quantifying, and integrating the multiple co-occurring events present daunting challenges to the scientific community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpehHLG3w1sSrVg90H21EOLACvtfcHk0libb_JzjhvWMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVegurnL&md5=1ff293ab2498b4946cba72ec5f67abef</span></div><a href="/servlet/linkout?suffix=cit196&amp;dbid=16384&amp;doi=10.1021%2Fja510028c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja510028c%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DJ.%26atitle%3DPrinciples%2520of%2520Allosteric%2520Interactions%2520in%2520Cell%2520Signaling%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2014%26volume%3D136%26spage%3D17692%26epage%3D17701%26doi%3D10.1021%2Fja510028c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref197"><div class="reference"><strong class="refLabel"><a href="#ref197" class="rightTabRefNumLink">197</a></strong><div class="NLM_citation" id="rightTab-cit197"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Csizmok, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Follis, A. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kriwacki, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forman-Kay, J. D.</span></span> <span> </span><span class="NLM_article-title">Dynamic Protein Interaction Networks and New Structural Paradigms in Signaling</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>116</i></span>,  <span class="NLM_fpage">6424</span>– <span class="NLM_lpage">6462</span>, <span class="refDoi"> DOI: 10.1021/acs.chemrev.5b00548</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrev.5b00548" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref197/cit197&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC28Xjt1Cqsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=116&publication_year=2016&pages=6424-6462&author=V.+Csizmokauthor=A.+V.+Follisauthor=R.+W.+Kriwackiauthor=J.+D.+Forman-Kay&title=Dynamic+Protein+Interaction+Networks+and+New+Structural+Paradigms+in+Signaling&doi=10.1021%2Facs.chemrev.5b00548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit197R"><div class="casContent"><span class="casTitleNuber">197</span><div class="casTitle"><span class="NLM_cas:atitle">Dynamic Protein Interaction Networks and New Structural Paradigms in Signaling</span></div><div class="casAuthors">Csizmok, Veronika; Follis, Ariele Viacava; Kriwacki, Richard W.; Forman-Kay, Julie D.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">116</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">6424-6462</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding signaling and other complex biol. processes requires elucidating the crit. roles of intrinsically disordered proteins (IDPs) and regions (IDRs), which represent ∼30% of the proteome and enable unique regulatory mechanisms.  In this review, we describe the structural heterogeneity of disordered proteins that underpins these mechanisms and the latest progress in obtaining structural descriptions of conformational ensembles of disordered proteins that are needed for linking structure and dynamics to function.  We describe the diverse interactions of IDPs that can have unusual characteristics such as "ultrasensitivity" and "regulated folding and unfolding".  We also summarize the mounting data showing that large-scale assembly and protein phase sepn. occurs within a variety of signaling complexes and cellular structures.  In addn., we discuss efforts to therapeutically target disordered proteins with small mols.  Overall, we interpret the remodeling of disordered state ensembles due to binding and post-translational modifications within an expanded framework for allostery that provides significant insights into how disordered proteins transmit biol. information.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYLVEaEWv217Vg90H21EOLACvtfcHk0ljVku34t8IgUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xjt1Cqsr0%253D&md5=7a4c31eb254a215673a3304e1d3e0c0e</span></div><a href="/servlet/linkout?suffix=cit197&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrev.5b00548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrev.5b00548%26sid%3Dliteratum%253Aachs%26aulast%3DCsizmok%26aufirst%3DV.%26aulast%3DFollis%26aufirst%3DA.%2BV.%26aulast%3DKriwacki%26aufirst%3DR.%2BW.%26aulast%3DForman-Kay%26aufirst%3DJ.%2BD.%26atitle%3DDynamic%2520Protein%2520Interaction%2520Networks%2520and%2520New%2520Structural%2520Paradigms%2520in%2520Signaling%26jtitle%3DChem.%2520Rev.%26date%3D2016%26volume%3D116%26spage%3D6424%26epage%3D6462%26doi%3D10.1021%2Facs.chemrev.5b00548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref198"><div class="reference"><strong class="refLabel"><a href="#ref198" class="rightTabRefNumLink">198</a></strong><div class="NLM_citation" id="rightTab-cit198"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nussinov, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Csermely, P.</span></span> <span> </span><span class="NLM_article-title">Allo-Network Drugs: Harnessing Allostery in Cellular Networks</span>. <i>Trends Pharmacol. Sci.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>32</i></span>,  <span class="NLM_fpage">686</span>– <span class="NLM_lpage">693</span>, <span class="refDoi"> DOI: 10.1016/j.tips.2011.08.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=10.1016%2Fj.tips.2011.08.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=21925743" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref198/cit198&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsV2mu7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2011&pages=686-693&author=R.+Nussinovauthor=C.+J.+Tsaiauthor=P.+Csermely&title=Allo-Network+Drugs%3A+Harnessing+Allostery+in+Cellular+Networks&doi=10.1016%2Fj.tips.2011.08.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit198R"><div class="casContent"><span class="casTitleNuber">198</span><div class="casTitle"><span class="NLM_cas:atitle">Allo-network drugs: harnessing allostery in cellular networks</span></div><div class="casAuthors">Nussinov, Ruth; Tsai, Chung-Jung; Csermely, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">32</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">686-693</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Allosteric drugs are increasingly used because they produce fewer side effects.  Allosteric signal propagation does not stop at the end' of a protein, but may be dynamically transmitted across the cell.  We propose here that the concept of allosteric drugs can be broadened to allo-network drugs' - whose effects can propagate either within a protein, or across several proteins, to enhance or inhibit specific interactions along a pathway.  We posit that current allosteric drugs are a special case of allo-network drugs, and suggest that allo-network drugs can achieve specific, limited changes at the systems level, and in this way can achieve fewer side effects and lower toxicity.  Finally, we propose steps and methods to identify allo-network drug targets and sites that outline a new paradigm in systems-based drug design.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomhg6HNPWpZLVg90H21EOLACvtfcHk0ljVku34t8IgUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsV2mu7%252FE&md5=1d1e2834e293b18e9d3aaeecdf9ba828</span></div><a href="/servlet/linkout?suffix=cit198&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2011.08.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2011.08.004%26sid%3Dliteratum%253Aachs%26aulast%3DNussinov%26aufirst%3DR.%26aulast%3DTsai%26aufirst%3DC.%2BJ.%26aulast%3DCsermely%26aufirst%3DP.%26atitle%3DAllo-Network%2520Drugs%253A%2520Harnessing%2520Allostery%2520in%2520Cellular%2520Networks%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2011%26volume%3D32%26spage%3D686%26epage%3D693%26doi%3D10.1016%2Fj.tips.2011.08.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3NEW','PDB','3NEW'); return false;">PDB: 3NEW</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MO4" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5MO4','PDB','5MO4'); return false;">PDB: 5MO4</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5Y2F','PDB','5Y2F'); return false;">PDB: 5Y2F</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1BG1','PDB','1BG1'); return false;">PDB: 1BG1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBI" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2OBI','PDB','2OBI'); return false;">PDB: 2OBI</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5J94','PDB','5J94'); return false;">PDB: 5J94</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q6K" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Q6K','PDB','1Q6K'); return false;">PDB: 1Q6K</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3PXF','PDB','3PXF'); return false;">PDB: 3PXF</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Y57" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1Y57','PDB','1Y57'); return false;">PDB: 1Y57</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2SRC','PDB','2SRC'); return false;">PDB: 2SRC</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.8b01749&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.8b01749%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-14%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.8b01749" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0b643d8a825a5","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
