
<html lang="en"     class="pb-page"  data-request-id="8a7809f8-57d9-45f9-ba1d-70413ce39abb"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2015.58.issue-20;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.5b01154;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening" /></meta><meta name="dc.Creator" content="Fanwang  Meng" /></meta><meta name="dc.Creator" content="Sufang  Cheng" /></meta><meta name="dc.Creator" content="Hong  Ding" /></meta><meta name="dc.Creator" content="Shien  Liu" /></meta><meta name="dc.Creator" content="Yan  Liu" /></meta><meta name="dc.Creator" content="Kongkai  Zhu" /></meta><meta name="dc.Creator" content="Shijie  Chen" /></meta><meta name="dc.Creator" content="Junyan  Lu" /></meta><meta name="dc.Creator" content="Yiqian  Xie" /></meta><meta name="dc.Creator" content="Linjuan  Li" /></meta><meta name="dc.Creator" content="Rongfeng  Liu" /></meta><meta name="dc.Creator" content="Zhe  Shi" /></meta><meta name="dc.Creator" content="Yu  Zhou" /></meta><meta name="dc.Creator" content="Yu-Chih  Liu" /></meta><meta name="dc.Creator" content="Mingyue  Zheng" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Wencong  Lu" /></meta><meta name="dc.Creator" content="Hong  Liu" /></meta><meta name="dc.Creator" content="Cheng  Luo" /></meta><meta name="dc.Description" content="Histone methyltransferases are involved in various biological functions, and these methylation regulating enzymes’ abnormal expression or activity has been noted in several human cancers. Within th..." /></meta><meta name="Description" content="Histone methyltransferases are involved in various biological functions, and these methylation regulating enzymes’ abnormal expression or activity has been noted in several human cancers. Within th..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="September 30, 2015" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.5b01154" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2015 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01154" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.5b01154" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.5b01154" /></link>
        
    
    

<title>Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.5b01154" /></meta><meta property="og:title" content="Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0009.jpeg" /></meta><meta property="og:description" content="Histone methyltransferases are involved in various biological functions, and these methylation regulating enzymes’ abnormal expression or activity has been noted in several human cancers. Within this context, SET domain-containing (lysine methyltransferase) 7 (SET7, also called KMT7, SETD7, SET9) is of increasing significance due to its diverse roles in biological functions and diseases, such as diabetes, cancers, alopecia areata, atherosclerotic vascular disease, HIV, and HCV. In this study, DC-S100, which was discovered by pharmacophore- and docking-based virtual screening, was identified as the hit compound of SET7 inhibitor. Structure–activity relationship (SAR) analysis was performed on analogs of DC-S100 and according to the putative binding mode of DC-S100, structure modifications were made to improve its activity. Of note, compounds DC-S238 and DC-S239, with IC50 values of 4.88 and 4.59 μM, respectively, displayed selectivity for DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a. Taken together, DC-S238 and DC-S239 can serve as leads for further investigation as SET7 inhibitors and the chemical toolkits for functional biology studies of SET7." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.5b01154"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.5b01154">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-13/achre4.2021.54.issue-13/20210706/achre4.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-6/amrcda.2021.2.issue-6/20210625/amrcda.2021.2.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-6/aabmcb.2021.4.issue-6/20210621/aabmcb.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-6/aaembp.2021.3.issue-6/20210622/aaembp.2021.3.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-6/aaemcq.2021.4.issue-6/20210628/aaemcq.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-26/aamick.2021.13.issue-26/20210707/aamick.2021.13.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-6/aanmf6.2021.4.issue-6/20210625/aanmf6.2021.4.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-13/accacs.2021.11.issue-13/20210702/accacs.2021.11.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-6/acscii.2021.7.issue-6/20210623/acscii.2021.7.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-6/acbcct.2021.16.issue-6/20210618/acbcct.2021.16.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-3/achsc5.2021.28.issue-3/20210524/achsc5.2021.28.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-13/acncdm.2021.12.issue-13/20210707/acncdm.2021.12.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-6/aesccq.2021.5.issue-6/20210617/aesccq.2021.5.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-5/afsthl.2021.1.issue-5/20210618/afsthl.2021.1.issue-5.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-6/amlccd.2021.10.issue-6/20210615/amlccd.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-7/amlcef.2021.3.issue-7/20210705/amlcef.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-6/ancac3.2021.15.issue-6/20210622/ancac3.2021.15.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-27/acsodf.2021.6.issue-27/20210713/acsodf.2021.6.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-6/apchd5.2021.8.issue-6/20210616/apchd5.2021.8.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-6/ascefj.2021.6.issue-6/20210625/ascefj.2021.6.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-27/ascecg.2021.9.issue-27/20210712/ascecg.2021.9.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-6/asbcd6.2021.10.issue-6/20210618/asbcd6.2021.10.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-27/ancham.2021.93.issue-27/20210713/ancham.2021.93.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-27/bichaw.2021.60.issue-27/20210713/bichaw.2021.60.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-6/bcches.2021.32.issue-6/20210616/bcches.2021.32.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-24/cgeabj.2021.99.issue-24/20210628-01/cgeabj.2021.99.issue-24.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-6/crtoec.2021.34.issue-6/20210621/crtoec.2021.34.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-12/chreay.2021.121.issue-12/20210623/chreay.2021.121.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-13/cmatex.2021.33.issue-13/20210713/cmatex.2021.33.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-7/cgdefu.2021.21.issue-7/20210707/cgdefu.2021.21.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-13/enfuem.2021.35.issue-13/20210701/enfuem.2021.35.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-13/esthag.2021.55.issue-13/20210706/esthag.2021.55.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-26/iecred.2021.60.issue-26/20210707/iecred.2021.60.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-13/inocaj.2021.60.issue-13/20210705/inocaj.2021.60.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-6/jaaucr.2021.1.issue-6/20210628/jaaucr.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-26/jafcau.2021.69.issue-26/20210707/jafcau.2021.69.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-6/jcisd8.2021.61.issue-6/20210628/jcisd8.2021.61.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-13/jmcmar.2021.64.issue-13/20210708/jmcmar.2021.64.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-6/jnprdf.2021.84.issue-6/20210625/jnprdf.2021.84.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-13/joceah.2021.86.issue-13/20210702/joceah.2021.86.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-26/jpcafh.2021.125.issue-26/20210708/jpcafh.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-26/jpcbfk.2021.125.issue-26/20210708/jpcbfk.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-26/jpccck.2021.125.issue-26/20210708/jpccck.2021.125.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-26/jpclcd.2021.12.issue-26/20210708/jpclcd.2021.12.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-26/jacsat.2021.143.issue-26/20210707/jacsat.2021.143.issue-26.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-7/jamsef.2021.32.issue-7/20210707/jamsef.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-27/langd5.2021.37.issue-27/20210713/langd5.2021.37.issue-27.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-13/mamobx.2021.54.issue-13/20210713/mamobx.2021.54.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-7/mpohbp.2021.18.issue-7/20210705/mpohbp.2021.18.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-12/nalefd.2021.21.issue-12/20210623/nalefd.2021.21.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-13/orlef7.2021.23.issue-13/20210702/orlef7.2021.23.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-6/oprdfk.2021.25.issue-6/20210618/oprdfk.2021.25.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-13/orgnd7.2021.40.issue-13/20210712/orgnd7.2021.40.issue-13.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.5b01154&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.5b01154&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.5b01154&amp;href=/doi/10.1021/acs.jmedchem.5b01154" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2015</span><span class="cit-fg-volume">, 58</span><span class="cit-fg-issue">, 20</span><span class="cit-fg-pageRange">, 8166-8181</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/58/20" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.5b01037" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.5b01072" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fanwang++Meng">Fanwang Meng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Sufang++Cheng">Sufang Cheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Ding">Hong Ding</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shien++Liu">Shien Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yan++Liu">Yan Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kongkai++Zhu">Kongkai Zhu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Shijie++Chen">Shijie Chen</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Junyan++Lu">Junyan Lu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yiqian++Xie">Yiqian Xie</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Linjuan++Li">Linjuan Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Rongfeng++Liu">Rongfeng Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhe++Shi">Zhe Shi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu++Zhou">Yu Zhou</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yu-Chih++Liu">Yu-Chih Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mingyue++Zheng">Mingyue Zheng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wencong++Lu">Wencong Lu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Liu">Hong Liu</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Luo">Cheng Luo</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Department of Chemistry, College of Sciences, Shanghai University, Shanghai 200444, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Chinese Academy of Sciences Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">School of Life Science and Technology, ShanghaiTech University, Shanghai 200031, China</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">Shanghai ChemPartner Co., Ltd., Zhangjiang Hi-Tech Park, Shanghai 201203, China</span></div><div class="corresp-info"><strong>*</strong>W.L.: phone, 86-21-66132406; e-mail, <a href="/cdn-cgi/l/email-protection#592e3a352c192a312c773c3d2c773a37"><span class="__cf_email__" data-cfemail="e691858a93a6958e93c8838293c88588">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>H.L.: phone, 86-21-50806600; e-mail, <a href="/cdn-cgi/l/email-protection#4c242025390c3f252121622d2f622f22"><span class="__cf_email__" data-cfemail="f59d999c80b5869c9898db9496db969b">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>C.L.: phone, 86-21-50806600; e-mail, <a href="/cdn-cgi/l/email-protection#b1d2ddc4def1c2d8dcdc9fd0d29fd2df"><span class="__cf_email__" data-cfemail="aecdc2dbc1eeddc7c3c380cfcd80cdc0">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01154&amp;href=/doi/10.1021%2Facs.jmedchem.5b01154" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2015</span></span><span class="cit-volume">, 58</span><span class="cit-issue">, 20</span><span class="cit-pageRange">, 8166–8181</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">September 21, 2015</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>6 July 2015</li><li><span class="item_label"><b>Published</b> online</span>30 September 2015</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 22 October 2015</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.5b01154" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01154</a></div><div class="article_header-article-copyright"><strong>Copyright © 2015 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8166%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DFanwang%2BMeng%252C%2BSufang%2BCheng%252C%2BHong%2BDing%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D58%26issueNum%3D20%26contentID%3Dacs.jmedchem.5b01154%26title%3DDiscovery%2Band%2BOptimization%2Bof%2BNovel%252C%2BSelective%2BHistone%2BMethyltransferase%2BSET7%2BInhibitors%2Bby%2BPharmacophore-%2Band%2BDocking-Based%2BVirtual%2BScreening%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8181%26publicationDate%3DOctober%2B2015">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.5b01154"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3992</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">43</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.5b01154" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Optimization of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Pharmacophore- and Docking-Based Virtual Screening&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Fanwang&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Sufang&quot;,&quot;last_name&quot;:&quot;Cheng&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Shien&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Yan&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Kongkai&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Shijie&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Junyan&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Yiqian&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Linjuan&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Rongfeng&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Zhe&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Yu&quot;,&quot;last_name&quot;:&quot;Zhou&quot;},{&quot;first_name&quot;:&quot;Yu-Chih&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Mingyue&quot;,&quot;last_name&quot;:&quot;Zheng&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Wencong&quot;,&quot;last_name&quot;:&quot;Lu&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Luo&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2015&quot;,&quot;month&quot;:&quot;09&quot;,&quot;day&quot;:&quot;30&quot;,&quot;issue&quot;:&quot;20&quot;,&quot;volume&quot;:&quot;58&quot;,&quot;pages&quot;:&quot;8166-8181&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.5b01154&quot;},&quot;abstract&quot;:&quot;Histone methyltransferases are involved in various biological functions, and these methylation regulating enzymes’ abnormal expression or activity has been noted in several human cancers. Within this context, SET domain-containing (lysine methyltransferase) 7 (SET7, also called KMT7, SETD7, SET9) is of increasing significance due to its diverse roles in biological functions and diseases, such as diabetes, cancers, alopecia areata, atherosclerotic vascular disease, HIV, and HCV. In this study, DC-S100, which was discovered by pharmacophore- and docking-based virtual screening, was identified as the hit compound of SET7 inhibitor. Structure–activity relationship (SAR) analysis was performed on analogs of DC-S100 and according to the putative binding mode of DC-S100, structure modifications were made to improve its activity. Of note, compounds DC-S238 and DC-S239, with IC50 values of 4.88 and 4.59 μM, respectively, displayed selectivity for DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a. Taken together, DC-S238 and DC&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01154" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01154&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01154" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.5b01154&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01154" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.5b01154&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.5b01154&amp;href=/doi/10.1021/acs.jmedchem.5b01154" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.5b01154" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.5b01154" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26pmid%3D26390175%26genre%3Darticle%26aulast%3DMeng%26date%3D2015%26atitle%3DDiscovery%2Band%2BOptimization%2Bof%2BNovel%252C%2BSelective%2BHistone%2BMethyltransferase%2BSET7%2BInhibitors%2Bby%2BPharmacophore-%2Band%2BDocking-Based%2BVirtual%2BScreening%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D58%26issue%3D20%26spage%3D8166%26epage%3D8181%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292580" title="Binding modes">Binding modes</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/58/20" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/jmcmar.2015.58.issue-20/20151022/jmcmar.2015.58.issue-20.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_null" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0009.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Histone methyltransferases are involved in various biological functions, and these methylation regulating enzymes’ abnormal expression or activity has been noted in several human cancers. Within this context, SET domain-containing (lysine methyltransferase) 7 (SET7, also called KMT7, SETD7, SET9) is of increasing significance due to its diverse roles in biological functions and diseases, such as diabetes, cancers, alopecia areata, atherosclerotic vascular disease, HIV, and HCV. In this study, DC-S100, which was discovered by pharmacophore- and docking-based virtual screening, was identified as the hit compound of SET7 inhibitor. Structure–activity relationship (SAR) analysis was performed on analogs of DC-S100 and according to the putative binding mode of DC-S100, structure modifications were made to improve its activity. Of note, compounds DC-S238 and DC-S239, with IC<sub>50</sub> values of 4.88 and 4.59 μM, respectively, displayed selectivity for DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a. Taken together, DC-S238 and DC-S239 can serve as leads for further investigation as SET7 inhibitors and the chemical toolkits for functional biology studies of SET7.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_38583" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_38583" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Histone lysine methyltransferases (HKMTs) are histone covalent modifying enzymes that transfer methyl groups to lysine or arginine on histone H3 via an <i>S</i>-adenosylmethionine (SAM)-dependent S<sub>N</sub>2 mechanism. HKMTs are involved in heterochromatin formation, gene expression regulation, DNA repair, cell cycle control,<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> and cancer development.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a>Thus, these epigenetic targets are vital in cancer biology. HKMTs can be divided into two categories: (i) SET domain-containing enzymes, such as SET7 (SET domain-containing lysine methyltransferase 7, also called SETD7, SET9, KMT7) and EZH2, and (ii) non-SET domain enzymes, such as DOT1L. SET7 was first characterized as a monomethyltransferase of H3K4.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> A series of studies later revealed that SET7 modulates various transcriptional regulators, including ARTD1,<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> COL2A1,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> ERα,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> FoxO3,<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> pRb,<a onclick="showRef(event, 'ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref9 ref10">(9, 10)</a> RAR,<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> STAT3,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> SUV39H1,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13, 14)</a> TAF10,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> p65,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> Pdx1,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> HIF-α.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In addition to its role in transcriptional regulation,<a onclick="showRef(event, 'ref16 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref19 ref20">(16, 19, 20)</a> SET7 also functions in cell cycle control,<a onclick="showRef(event, 'ref9 ref10 ref21'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref21">(9, 10, 21)</a> differentiation,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> DNA repair,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and DNMT1 stability,<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> and increasing evidence suggests that SET7 is closely associated with various diseases. SET7 regulates NF-κB activity through lysine monomethylation of p65 in response to TNF-α in human kidney cells,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> and knockdown of SET7 inhibits key NF-κB-dependent gene expression.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Li et al. observed increased inflammatory gene expression and SET7 recruitment in macrophages from diabetic mice, demonstrating the role of SET7 in diabetes.<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> Recently, it was found that epigenetic changes induced by SET7 contribute to vascular dysfunction in patients with type 2 diabetes,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> indicating the targeting of SET7 as a promising option to prevent atherosclerotic vascular disease. Increased SET7 expression is also observed in peripheral blood mononuclear cells from patients with alopecia areata.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In addition, downregulation of SET7 in breast cancer cells attenuates estrogen-induced gene activation, and SET7 knockdown results in a concomitant decrease in the steady-state levels of estrogen receptor α (ERα) protein,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> which is involved in the pathology of breast cancer, endometrial cancer, and osteoporosis. Other studies also support that SET7 is linked to the pathological processes of cancers.<a onclick="showRef(event, 'ref13 ref29'); return false;" href="javascript:void(0);" class="ref ref13 ref29">(13, 29)</a> Recently, the role of SET7 in virus infection was identified; for example, knockdown of SET7 suppresses Tat transcription of the HIV promoter,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and SET7 facilitates HCV replication by attenuating IFN signaling pathways and IFN-related effectors.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> Notably, there is a controversial debate about whether the methylation of p53 by SET7 meditates p53 activity.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> Some studies revealed that the SET7-mediated methylation of Lys-372 at the p53 C-terminus can regulate p53 acetylation, enhancing its antitumor activity.<a onclick="showRef(event, 'ref33 ref34 ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref33 ref34 ref35 ref36">(33-36)</a> However, Lehnertz et al. reported that knockdown of SET7 in mice does not affect p53-depedent transcription and tumor suppression,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> results that are fully consistent with the findings of Campaner et al., who reported that p53-depedent cell cycle arrest or apoptosis is not affected by SET7 deletion.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> This controversial debate calls for the urgent search for a potent SET7 inhibitor that can be used as a chemical probe.</div><div class="NLM_p">As SET7 is a promising target in several diseases, including diabetes, alopecia areata, cancers, and virus infection, several attempts have been made to identify SET7 inhibitors (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Chart S1</a>). Compounds <b>1</b>–<b>6</b> are analogs of SAM, with resulting selectivity issues.<a onclick="showRef(event, 'ref39 ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref39 ref40 ref41">(39-41)</a> Bissinger et al. identified acyl derivatives of <i>p</i>-aminosulfonamides and dapsone as potent hPRMT1 inhibitors, including compound <b>7</b>, yet they showed weak activity toward SET7.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Using high-throughput screening, Verma and co-workers reported that compounds <b>8</b>, <b>9</b>, and <b>10</b> are potent, cell-active inhibitors of EZH2, and these compounds also showed inhibitory activity toward SET7.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> “Biased-privileged” scaffolds were used to design SET7 inhibitors, and SETin-1 (compound <b>11</b>) showed inhibitory activity against both G9a and SET7.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> Compound <b>12</b>, which showed weak activity toward SET7, is a coactivator-associated arginine methyltransferase 1 inhibitor with an IC<sub>50</sub> value of 14.4 μM.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> Similarly, compounds <b>13</b>–<b>16</b> are not selective SET7 inhibitors.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> Following a drug repository strategy, it was found that cyproheptadine (compound <b>17</b>), an antihistaminic and antiserotonergic agent, showed inhibitory activity toward SET7 (IC<sub>50</sub> = 20 μM).<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> Recently, (<i>R</i>)-PFI-2 (compound <b>18</b>) was reported to be a potent and selective inhibitor targeting SET7 in MCF7 cells by occupying the histone lysine binding site, as revealed by X-ray complex structure analysis.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> However, most SET7 inhibitors are multitarget inhibitors with low potencies. On the basis of the diversity of SAM binding modes and receptor-based interactions among HKMTs, which enable the design of selective inhibitors, our research focuses on the development of potent, selective small-molecule inhibitors of SET7 targeting the cofactor binding site.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49, 50)</a></div><div class="NLM_p last">With the rapid development of computational methods, many studies successfully identifying epigenetic inhibitors using this approach have been reported.<a onclick="showRef(event, 'ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67 ref68 ref69 ref70 ref71 ref72 ref73 ref74 ref75 ref76 ref77 ref78 ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref51 ref52 ref53 ref54 ref55 ref56 ref57 ref58 ref59 ref60 ref61 ref62 ref63 ref64 ref65 ref66 ref67 ref68 ref69 ref70 ref71 ref72 ref73 ref74 ref75 ref76 ref77 ref78 ref79 ref80">(51-80)</a> Our group has applied virtual screening methods to identify inhibitors of PRMT1 and DNMT1, as well as inhibitors targeting the interface of EZH2-EED and the menin-mixed lineage leukemia interface;<a onclick="showRef(event, 'ref68 ref81 ref82 ref83 ref84 ref85'); return false;" href="javascript:void(0);" class="ref ref68 ref81 ref82 ref83 ref84 ref85">(68, 81-85)</a> these successes and our previous experience encouraged us to try to identify SET7 inhibitors employing virtual screening. In this study, novel, potent SAM competitive and selective inhibitors of SET7, DC-S238 (IC<sub>50</sub> = 4.88 μM) and DC-S239 (IC<sub>50</sub> = 4.59 μM), were identified through an integrated procedure combining pharmacophore- and docking-based virtual screening, similarity searching, and hit optimization. The virtual screening identified compound DC-S100 as a hit compound possessing moderate activity toward SET7 (IC<sub>50</sub> = 30.04 μM). Similarity-based analog searching enabled the analysis of the structure–activity relationship (SAR) of DC-S100. On the basis of the putative binding mode of DC-S100 and its SAR, chemical optimization was performed to improve its potency. Finally, compounds DC-S238 (IC<sub>50</sub> = 4.88 μM) and DC-S239 (IC<sub>50</sub> = 4.59 μM) were obtained, displaying selectivity for DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a. In summary, this study identified DC-S238 and DC-S239 as potent and selective inhibitors against SET7, and they may serve as the lead compounds for further development and the chemical toolkits for functional biology studies of SET7.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65154" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65154" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Virtual Screening</h3><div class="NLM_p">To exploit all available structural and chemical information, an integrated pharmacophore- and docking-based virtual screening procedure was designed to counterbalance their limitations.<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> Pharmacophore-based virtual screening can be used to preprocess virtual molecule libraries to filter compounds lacking features essential for binding. The 3D pharmacophore model which yields the lead compound DC-S100 was shown in <a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Figure S1</a>. An assessment of the SET domain in different states suggested that the SAM binding pocket has conserved architecture (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Figure S2A</a>), and the crystal structure of SET7 in complex with the bioactive cofactor SAM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>) was selected for the subsequent pharmacophore generation and docking studies. The Specs database (<a href="http://www.specs.net" class="extLink">http://www.specs.net</a>), containing more than 200 000 molecules, was filtered using Lipinski’s rule of five,<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> and “pan-assay interference compounds” (PAINS)<a onclick="showRef(event, 'ref88 ref89 ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref88 ref89 ref90 ref91">(88-91)</a> were removed with Pipeline Pilot, version 7.5 (Pipeline Pilot; Accelrys Software Inc., San Diego, CA). The remaining 182 014 molecules were prepared with LigPrep, version 2.3<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> (Schrödinger, LLC: New York, NY, 2009) to generate all stereoisomers and different protonation states by Epik, version 2.0<a onclick="showRef(event, 'ref93'); return false;" href="javascript:void(0);" class="ref ref93">(93)</a> (Schrödinger, LLC: New York, NY, 2009). The prepared Specs library was used to generate an indexed multiconformer 3D database in Accelrys Discovery Studio 3.0<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> with the conformation method CAESAR.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a> Receptor–ligand based pharmacophores, which was used to screen the 3D database, were generated with Accelrys Discovery Studio 3.0.<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> A total of 15 541 molecules matched the pharmacophore features and were subjected to two rounds of docking-based virtual screening.</div><div class="NLM_p last">Several docking programs were preliminarily evaluated, and Glide<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> was chosen due to its better performance with regard to enrichment factors (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). The redocking studies suggested that Glide is capable of reproducing the bioactive binding conformation of SAM with an rmsd value of 0.30 Å (Glide SP mode<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a>) and 0.22 Å (Glide XP mode<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a>), respectively, which agreed with our docking procedures (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>). The Glide SP and XP modes, as integrated in Maestro 9.2,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> were subsequently used to dock the compound library into the SAM binding site. As lysine-294 may be involved in the specific recognition of SAM by SET7,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> we chose this residue as a hydrogen bond constraint during the Glide XP docking procedure. The top-ranked 649 molecules were clustered into 108 groups with Pipeline Pilot, version 7.5 (Pipeline Pilot; Accelrys Software Inc., San Diego, CA) using its protocol component Cluster Molecules for visual inspection. The candidate molecules were selected on the basis of the following considerations. (1) The binding poses were reasonable. Molecules with high strain energy or not occupying the SAM binding pocket were not selected for further validation. (2) Among a cluster of similar molecules, smaller ones are preferred. (3) Molecules that can form distinguishable hydrogen bonds with Lys-294, Ala-226, Glu-228, His-297, Glu-356 and π–π stacking interactions with Trp-352 are good candidates because these residues can form key interactions with SAM, and the disruption of the key interactions is believed to contribute to potency. (4) In each clustered group, at least one molecule was retained to cover more chemical space, which in turn improves hit rates. Consequently, 127 compounds were purchased from Specs Company for biochemical assays.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> SET7 Enzymatic Assays</h3><div class="NLM_p">The 127 candidate molecules selected were tested for SET7 inhibition to determine their biochemical activity in vitro. We used SAH as reference compounds in the AlphaLISA assay at 100 μM. The enzyme solution was transferred to assay plates, and 5 μL of 1× assay buffer (modified Tris buffer) for low control was incubated at room temperature for 15 min. Acceptor and donor beads were prepared in 1× AlphaLISA buffer and incubated for 60 min at room temperature, shielded from light. The end point was read using EnSpire in Alpha mode. In the assay, seven compounds demonstrated significant inhibition of SET7 activity at 100 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). Among those seven molecules, compound DC-S100 was characterized as a structurally novel SET7 inhibitor (IC<sub>50</sub> = 30.04 μM).</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Virtual screening procedures and activity assays for SET7 in vitro. (A) An integrated virtual screening procedure combining pharmacophore- and docking-based virtual screening. Numerals denote the number of molecules in each stage. (B) Inhibitory activity of the 127 candidate molecules based on virtual screening at 100 μM. The red columnar bar represents the activity of DC-S100, and the blue bar denotes the reference compound SAH. (C) Inhibitory activity of the 109 analogous compounds of DC-S100 selected from the similarity search, in which the inhibition rate was determined at 100 μM. The blue columnar bar represents the activity of the reference compound SAH.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Similarity-Based Analog Searching and SAR Analysis</h3><div class="NLM_p">To explore the structure–activity relationship (SAR) of compound DC-S100 and to optimize the compound, a similarity-based analog search was conducted using Pipeline Pilot, version 7.5 (Accelrys Inc., San Diego, CA, USA). A total of 109 derivatives of compound DC-S100 were purchased for the same AlphaLISA assay described above. Seven compounds displayed greater than 50% inhibition of SET7 activity at 100 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>C), and the IC<sub>50</sub> values against SET7 were measured (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Table S3</a>).</div><div class="NLM_p">We analyzed the activity data of the analogs (see <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) carefully and explored the SAR of DC-S100 to find clues for hit optimization. In scaffolding (I), the R<sub>2</sub> group is expected to be aromatic rings, as opposed to other side chains, because substitutions of aromatic ring resulted in a significant decrease in potency (DC-S133 and DC-S134), with inhibition rates of 10.64% and 22.06%, respectively. In contrast, when R<sub>2</sub> is a derivative of furan (DC-S128, DC-S129) or benzene (DC-S130 to DC-S132), inhibition activity increased. The length of the amide group and R<sub>1</sub> group connecting <i>m</i>-nitrobenzene and the aromatic ring (R<sub>2</sub>) has strong effects on the activity. A significant decrease in activity was also observed when the linker was longer than that of the amide group, such as for DC-S135 and DC-S136, with inhibition rates 12.14% and 15.78%, respectively; thus, an amide group is the most appropriate linker. Moreover, the introduction of a bulky group into the aromatic rings (R<sub>2</sub>) is not tolerated because large groups often lead to a decrease in activity (DC-S138 to DC-S143). For example, when an iodine atom was substituted in the benzene ring (DC-S137), the inhibition rate decreased to 1.62%, with almost no inhibitory activity against SET7. Regarding scaffold (II), we can conclude the same results. First, a linker group that is too long is not tolerated, such as in compound DC-S144. Second, a larger R<sub>2</sub> group also led to a dramatic decrease in potency (DC-S145 and DC-S146), supporting the previous hypothesis. In summary, a <i>m</i>-nitrobenzene group and an aromatic ring linked by an amide group was used as the basic scaffold for the next stage of chemical optimization.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Structure-Activity Relationship (SAR) of DC-S100</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0006.gif" alt="" id="fx3" /></img><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemical Modification of DC-S100</h3><div class="NLM_p">To gain a better understanding of the molecular basis of the inhibitory activity, we analyzed the putative binding mode of DC-S100. As depicted in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A, the nitro group is capable of forming hydrogen bonds with Glu-228 and Lys-294. The hydrogen bond lengths of Glu-228 and Lys-294 are 2.0 and 2.3 Å, respectively, indicating that the hydrogen bonds are strong and essential for activity. The amide group, which is the most appropriate linker according to the SAR analysis, forms a strong hydrogen bond with Asn-296, with a bond length of 1.9 Å. The π–π stacking interactions between indole ring of Trp-352 and the benzene ring of DC-S100 are present, indicating that an aromatic ring is indispensable for activity. When the amide group is replaced by longer linkers, the position of the benzene is not favorable for π–π stacking interactions, and thus, the inhibition rate decreased. His-297 is the key residue forming two hydrogen bonds with SAM, but no polar interactions between DC-S100 and His-297 exist. If the compound could form polar interactions with Glu-356, disrupting polar interactions between Glu-356 and SAM, its binding affinity with SET7 will be enhanced.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Putative binding modes and activities of DC-S100 and DC-S237. (A) Binding mode of DC-S100 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). Carbon atoms of DC-S100 are displayed in yellow; nitrogen atoms are in blue and oxygen atoms in red. Key residues are shown as sticks, and carbon is in gray, nitrogen in blue, and oxygen in red. Key hydrogen bonds are shown as a red dashed line. All of the residues and compounds are labeled. (B) Binding mode of DC-S237 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). DC-S237 is shown as yellow sticks, and key interacting residues are shown as gray sticks. All of the oxygen atoms are in red, and nitrogen atoms are in blue. Hydrogen bonds between DC-S237 and SET7 are represented as a red dashed line. (C) Superimposition of the binding modes of DC-S100 and DC-S237 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). Both DC-S100 and DC-S237 share a similar binding conformation anchored in the cofactor binding pocket. DC-S237 is shown as blue sticks, and DC-S100 is depicted in cyan, with the surface of SET7 depicted in vacuum electrostatics. Key residue Lys-294 is highlighted. All of the binding mode figures were generated with PyMOL, version 1.3r1.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> (D) Inhibitory activities of DC-S100 and DC-S237 against SET7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We presumed that if we introduced an amino group to the aromatic ring, the amino group may form hydrogen bonds with His-297 and Glu-356, which in turn will enhance its activity against SET7. This later proved to be a successful strategy.</div><div class="NLM_p">Taking feasibility and accessibility into consideration, we synthesized DC-S237 via the Gewald reaction and tested its activity toward SET7. DC-S237 displayed activity that was similar to DC-S100, with an IC<sub>50</sub> value of 37.67 μM (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D). In pursuit of molecular details, a docking simulation was conducted, and the putative binding mode is shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B. The binding mode suggested that although the hydrogen bond with Glu-228 disappeared, DC-S237 formed new polar interactions with His-297 and Glu-356. A predicted conformation alignment revealed that both DC-S100 and DC-S237 share similar binding modes, though the direction of the amide group in DC-S237 is altered (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C). The binding mode of DC-S237 also suggested that the terminal isobutyl may be slightly too large to completely fit into the SAM binding pocket, which prompted us to substitute the isobutyl with smaller alkyl groups. Compounds DC-S238 and DC-S239 were synthesized, both of which displayed better inhibitory activity, as expected. The IC<sub>50</sub> values of DC-S238 and DC-S239 were determined to be 4.88 and 4.59 μM, respectively (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Activities of synthesized compounds. (A) Inhibitory activity of synthesized compounds. Activity data of reference compound SAH are represented as a blue columnar bar. (B) IC<sub>50</sub> curves of DC-S238 and DC-S239. The green sigmoidal curve graph depicts the activity profile of DC-S238 (IC<sub>50</sub> = 4.88 μM), and the pink sigmoidal curve graph displays that of DC-S239 (IC<sub>50</sub> = 4.59 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Selectivity</h3><div class="NLM_p">We determined the selectivity of DC-S238 and DC-S239 for SET7 compared to six other enzymes (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). The results showed that DC-S238 and DC-S239 exhibited a low inhibition rate, below 45%, for DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Moreover, DC-S239 possesses a good selectivity profile over DNMT3A/3L at 100 μM, with an inhibition rate of −4.25%. Therefore, DC-S238 and DC-S239 are selective inhibitors of SET7.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Selectivity of DC-S238 and DC-S239 over Other Epigenetic Proteins</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">target</th><th class="colsep0 rowsep0" align="center" char=".">inhibition ratio at 100 μM, %</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DC-S238</td><td class="colsep0 rowsep0" align="left">SET7</td><td class="colsep0 rowsep0" align="char" char=".">94.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DNMT1</td><td class="colsep0 rowsep0" align="char" char=".">9.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DOT1L</td><td class="colsep0 rowsep0" align="char" char=".">–28.88</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EZH2</td><td class="colsep0 rowsep0" align="char" char=".">19.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NSD1</td><td class="colsep0 rowsep0" align="char" char=".">25.05</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SETD8</td><td class="colsep0 rowsep0" align="char" char=".">45.45</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">G9a</td><td class="colsep0 rowsep0" align="char" char=".">29.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DC-S239</td><td class="colsep0 rowsep0" align="left">SET7</td><td class="colsep0 rowsep0" align="char" char=".">90.51</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DNMT1</td><td class="colsep0 rowsep0" align="char" char=".">16.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DNMT3A/3L</td><td class="colsep0 rowsep0" align="char" char=".">–4.25</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">DOT1L</td><td class="colsep0 rowsep0" align="char" char=".">–5.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">EZH2</td><td class="colsep0 rowsep0" align="char" char=".">19.61</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">NSD1</td><td class="colsep0 rowsep0" align="char" char=".">27.16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">SETD8</td><td class="colsep0 rowsep0" align="char" char=".">40.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">G9a</td><td class="colsep0 rowsep0" align="char" char=".">–0.71</td></tr></tbody></table></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Cell Activity</h3><div class="NLM_p">Because DC-S239 is a selective inhibitor of SET7, we assessed whether SET7 can show selective profiles in different cell lines. Therefore, the antiproliferation activities of compound DC-S239 against the MCF7, HL60, DHL4, MV4-11, and HCT116 cell lines were determined by the MTT or alamarBlue assay. DC-S239 inhibited the proliferation of MCF7, HL60, and MV4-11 cells in a dose-dependent manner while showing no cellular activity against HCT116 and DHL4 cells (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). DC-S239 displayed IC<sub>50</sub> values of 10.93 μM in MCF-7 cells and 16.43 μM in HL60 cells. Moreover, DC-S239 also displayed dose-dependent inhibition activity toward MV4-11 cells. These inhibition profiles suggested that DC-S239 is more preferential toward breast cancer and leukemia cell lines.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cellular activities of DC-S239 against SET7 in different cell lines. (A) IC<sub>50</sub> curve of DC-S239 toward the MCF7 cell line. (B) IC<sub>50</sub> curve of DC-S239 toward the HL60 cell line. (C) Dose–response relationship of DC-S239 toward the MV4-11 cell line. (D) Activity of DC<b>-</b>S239 toward the HCT116 cell line. (E) Activity of DC-S239 toward the DHL4 cell. The antiproliferation activity of DC-S239 is determined by the MTT (MCF7, MV4-11) or alamarBlue assay (HL60, MV4-11, DHL4). The experimental data were analyzed with GraphPad Prism 5.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> SAR Analysis of DC-S239 and Binding Mode Prediction of DC-S239</h3><div class="NLM_p">To explore the SAR of DC-S239, an analog series was synthesized, and the activities were tested against SET7 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). Comparing DC-S197, DC-S198, and DC-S199 with DC-S238, it can be concluded that an amino group (R<sub>2</sub>) is essential for activity (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). When the amino group is replaced with another group, such as in DC-S197, DC-S198, and DC-S199, which are analogs from the similarity search, no inhibitory activity was observed against SET7. Another finding is that a bulkier terminal R<sub>3</sub> group resulted in reduced activity (DC-S238 to DC-S244) from 94.71% to 1.53%. Activity gradually decreased when the R<sub>3</sub> group was larger than the ethyl group; in particular, DC-S242, with the largest alkyl group <i>tert</i>-butyl, showed no inhibitory activity toward SET7. This may be due to steric effects hindering the molecules from fitting into the SAM binding site. Inhibitory activities also revealed that the substitution of the nitro group leads to a significant loss in potency. For example, when the nitro group was substituted with methoxy (DC-S249) or chlorine (DC-S257), a dramatic decrease in activity was observed. Other substitutions did not contribute to activity. Thus, a nitro group at the meta-position is indispensable. In summary, a nitro group at the meta-position of the benzene ring, a thiophene ring with an amino group at the α position, and an amide group are essential building blocks for active SET7 inhibitors. In addition, a methyl or ethyl group linked by an ester group is the most appropriate alkyl group.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structure-Activity Relationship (SAR) of DC-S239</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0007.gif" alt="" id="fx4" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0008.gif" alt="" id="fx5" /></img><div></div></div><div class="NLM_p">To explore the molecular basis of inhibitory activity, we conducted docking studies, and the results revealed that DC-S239 displays a similar binding mode as SAM, fitting closely into the open and solvent-accessible binding pocket (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>B). The nitro group is thought to form a hydrogen bond with Lys-294, which may be involved in selectivity.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Thus, substitution of the nitro group disrupted hydrogen bonding with Lys-294, and a loss in activity was observed. The binding mode of DC-S239 (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>A) suggested that the amino group is capable of forming hydrogen bonds with His-297 and Glu-356, enhancing the binding affinity for SET7. When the amino group was replaced with other groups, such as in compounds DC-S197, DC-S198, and DC-S199, the disruption of the hydrogen bonds resulted in a significant decrease in activity. The binding mode also suggested that π–π stacking interactions between the thiophene ring and Trp-352 contributed to the activity against SET7. If the linker is longer than the amide group, the positions of the aromatic rings are altered, preventing π–π stacking interactions. Therefore, the amide group is the most proper linker, and the aromatic ring is essential to activity. Additionally, the terminal alkyl group cannot be too large because large groups can result in steric effects, making the SAM pocket inaccessible. A larger terminal alkyl group leading to potency loss is in accordance with the SAR of DC-S239.</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of DC-S239 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). (A) Key interactions involved in stabilizing DC-S239 at the cofactor binding site. DC-S239 is depicted as yellow sticks, and key residues are labeled. Hydrogen bonds are shown as red lines, and the π–π stacking interaction involving residue Trp-352 is also highlighted. (B) Superimposition of the binding modes of DC-S239 and SAM. The structure of SET7 is displayed in vacuum electrostatics. DC-S239 is shown as yellow sticks, and SAM is displayed as gray sticks. Key residue Lys-294 is highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65567" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65567" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">SET7 plays a significant role in transcriptional regulation,<a onclick="showRef(event, 'ref16 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref16 ref19 ref20">(16, 19, 20)</a> cell cycle control,<a onclick="showRef(event, 'ref9 ref10 ref21'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref21">(9, 10, 21)</a> differentiation,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> DNA repair,<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> and DNMT1 stability.<a onclick="showRef(event, 'ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref23 ref24">(23, 24)</a> SET7-mediated methylation is also associated with diabetes,<a onclick="showRef(event, 'ref25 ref26 ref99 ref100'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref99 ref100">(25, 26, 99, 100)</a> cancers,<a onclick="showRef(event, 'ref7 ref13 ref29'); return false;" href="javascript:void(0);" class="ref ref7 ref13 ref29">(7, 13, 29)</a> alopecia areata,<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> atherosclerotic vascular disease,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> transcription of the HIV promoter,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> and HCV replication.<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> The controversial debate regarding whether SET7 can regulate p53 highlights the need for specific and potent inhibitors. Notably, studies reporting the role of SET7 in the regulation of ERα,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> STAT3,<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> and NF-κB,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> including p53, have all employed knockdown or overexpression methods, and this may obscure or exaggerate the delicate regulatory mechanism of SET7. To date, only (<i>R</i>)-PFI-2 has been reported to be a potent selective SET7 inhibitor targeting the substrate binding site.<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Other inhibitors lack either potency or selectivity, hindering an understanding of the functional biology of SET7 as well as the exploration of its potential as a therapeutic target.</div><div class="NLM_p">Nonetheless, efforts have been made to discover potent and selective SET7 inhibitors through structure-based virtual screening and chemical optimization. First, we employed pharmacophore-based and docking-based virtual screening to mine potential candidate molecules. DC-S100 was identified as a hit, with an IC<sub>50</sub> value of 30.04 μM. Subsequently, similarity-based analog searching was performed, and their activities were tested to explore the SAR of DC-S100. The SAR of DC-S100 suggested that the nitro group at the meta position, a linker amide group, and an aromatic ring that does not cause steric effects are essential elements for activity. Docking studies revealed that there are no polar interactions between His-297, Glu-356, and DC-S100, providing a direction for hit optimization. Considering the issues discussed above and feasibility, DC-S237 was synthesized via the Gewald reaction. With respect to the binding mode, DC-S237 may be too large to fit well into the SAM binding pocket, which prompted us to synthesize DC-S238 and DC-S239 as well as their analogs. DC-S239 proved to be a potent SET7 inhibitor, with an IC<sub>50</sub> value of 4.59 μM. Binding mode prediction revealed that DC-S239 shares a similar binding moiety with SAM, and the key interactions are in accordance with the SAR analysis of DC-S239. Biological testing showed that DC-S239 displays selectivity for SET7 over DNMT1, DNMT3A/3L, DOT1L, EZH2, NSD1, SETD8, and G9a, with inhibition ratio below 45.45%. DC-S238 is also a potent SET7 inhibitor with an IC<sub>50</sub> value of 4.88 μM and possesses a good selectivity profile over DNMT1, DOT1L, EZH2, NSD1, SETD8, and G9a.</div><div class="NLM_p">According to the GeneCards platform,<a onclick="showRef(event, 'ref101 ref102'); return false;" href="javascript:void(0);" class="ref ref101 ref102">(101, 102)</a> SET7 is highly expressed in the breast cancer MCF7 cell line, and SET7 can modulate the stability of ERα via the methylation of K302.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> ERα is overexpressed in approximately 70% of breast cancer cases, and the blockade of ERα action has proved to be a successful strategy in these patients.<a onclick="showRef(event, 'ref103'); return false;" href="javascript:void(0);" class="ref ref103">(103)</a> Thus, our small molecules have the potential to function as modulators of ERα.</div><div class="NLM_p last">The study presented here details the discovery and development process of hit optimization from structure-based drug virtual screenings to potent, selective SET7 inhibitors with novel scaffolds using a structure-guided drug design approach. It also highlights the significance of selecting a proper hit compound with low molecular weight and a structure-guided drug design approach. The encouraging findings obtained in our study provide lead compounds for the further development of SET7 inhibitors and will shed light on chemical biology studies of SET7.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_40631" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_40631" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> Virtual Screening. Ligand Database Preparation</h3><div class="NLM_p last">The ligand database was extracted from the Specs library, which contains approximately 200 000 molecules. The ligands were filtered by Lipinski’s rule of five,<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> and “pan-assay interference compounds” (PAINS)<a onclick="showRef(event, 'ref88 ref89 ref90 ref91'); return false;" href="javascript:void(0);" class="ref ref88 ref89 ref90 ref91">(88-91)</a> were removed using Pipeline Pilot, version 7.5 (Pipeline Pilot; Accelrys Software Inc., San Diego, CA). The remaining 182 014 molecules were prepared with LigPrep<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> to generate all stereoisomers and different protonation states with Epik. The prepared Specs database was used to generate an indexed multiconformer 3D database in Accelrys Discovery Studio 3.0<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> with the conformation method CAESAR.<a onclick="showRef(event, 'ref95'); return false;" href="javascript:void(0);" class="ref ref95">(95)</a></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Protein Preparation</h3><div class="NLM_p last">Twenty-nine X-ray crystal structures of SET7 are in the PDB database, and we aligned the SET domain of these structures for comparison, which is conserved in the SET domain-containing family. We obtained a maximum root mean square derivation (rmsd) value of 0.37, suggesting that the three-dimensional structure of the SET domain in SET7 is conserved. Considering the structure resolution and integrality, the crystal structure of SET7 in a complex with SAM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>) was chosen for pharmacophore- and docking-based virtual screening. The protein was prepared with Protein Preparation Wizard Workflow, as provided in Maestro,<a onclick="showRef(event, 'ref98'); return false;" href="javascript:void(0);" class="ref ref98">(98)</a> with a pH of 7.0 ± 2.0. Other parameters were set as the default.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Pharmacophore-Based Virtual Screening</h3><div class="NLM_p last">The receptor–ligand based pharmacophore model was generated with Accelrys Discovery Studio 3.0<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> using the X-ray structure in complexes with SAM (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). The pharmacophore features were edited in combinations in Discovery Studio 3.0<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> in order to cover a border chemical space when matching the prepared multiconformer 3D Specs database. We obtained 15 541 compounds after searching the 3D database with those pharmacophore models, and the 15 541 molecules were prepared using LigPrep<a onclick="showRef(event, 'ref92'); return false;" href="javascript:void(0);" class="ref ref92">(92)</a> for further docking validation.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Docking-Based Virtual Screening</h3><div class="NLM_p last">To rationalize our docking procedures, the enrichment factor (EF)<a onclick="showRef(event, 'ref104'); return false;" href="javascript:void(0);" class="ref ref104">(104)</a> was used to evaluate the performance of several docking programs, including Glide 5.5,<a onclick="showRef(event, 'ref105'); return false;" href="javascript:void(0);" class="ref ref105">(105)</a> AutoDock Vina,<a onclick="showRef(event, 'ref106'); return false;" href="javascript:void(0);" class="ref ref106">(106)</a> LibDock in Discovery Studio<a onclick="showRef(event, 'ref94'); return false;" href="javascript:void(0);" class="ref ref94">(94)</a> and Dock 6.<a onclick="showRef(event, 'ref107'); return false;" href="javascript:void(0);" class="ref ref107">(107)</a> The results suggested that Glide 5.5 performs better than the others (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Table S1</a>). A re-docking evaluation illustrated that Glide was capable of reproducing the active conformation, with rmsd values of 0.300 Å (Glide SP mode) and 0.218 Å (Glide XP mode), respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Supporting Information Figure S2</a>). The grid box was set to 25 Å × 25 Å × 25 Å centered at the bioactive ligand SAM. Because lysine-294 may be involved in specific recognition of SAM by SET7,<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> we set lysine-294 as a hydrogen bond constraint during grid generation and docking. Glide<a onclick="showRef(event, 'ref96'); return false;" href="javascript:void(0);" class="ref ref96">(96)</a> SP mode and Glide XP mode<a onclick="showRef(event, 'ref97'); return false;" href="javascript:void(0);" class="ref ref97">(97)</a> were subsequently used to dock the compound library into the SAM binding site. The top-ranked 649 molecules were clustered for visual inspection, and 127 compounds were selected and purchased for biological testing.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Similarity-Based Analog Searching</h3><div class="NLM_p last">According to the results of the biological tests, we searched the prepared Specs library for derivatives of DC-S100 using Similarity Filter from File in Pipeline Pilot, version 7.5. (Pipeline Pilot; Accelrys Software Inc., San Diego, CA). We purchased 109 compounds and tested their biological activity toward SET7 to investigate the SAR of DC-S100.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> SET7 Inhibition Assays</h3><div class="NLM_p last">AlphaLISA was applied to determine the inhibition activity of the compounds over SET7. The purified SET7 protein was incubated in modified Tris buffer in 384-well plates (PerkinElmer, catalog NO. 6007299) at room temperature for 15 min. The compounds were transferred to the assay plate using Echo in 100% DMSO, and 5 μL of substrate solution was added to each well to start the reaction. The substrate solution was incubated in each well for 60 min. Acceptor and donor beads (15 μL) were added and incubated for 60 min at room temperature, shielded from light. The end point was evaluated with EnSpire in Alpha mode. The experimental data were fitted in GraphPad Prism 5 to obtain inhibition values using<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/jm-2015-011546_m001.gif" alt="" /></img></span></div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Enzymatic Selectivity Assay</h3><div class="NLM_p">A radioactive methylation inhibition assay of DNMT1 was performed in modified Tris buffer using poly(dI-DC) (Sigma, USA, product NO. P4929) as a substrate. The enzyme solution was incubated at room temperature for 15 min before 10 μL of substrate and 10 μL of [<sup>3</sup>H]SAM (PerkinElmer Inc., USA, lot. NO. 1790854) were added to start the reaction. After incubation at 37 °C for 120 min, the reactions were transferred to filter plates; the plates were preincubated for 15 min with 0.5% PEI, and residual solution was removed by vacuum. The plates were washed 3 times with ddH<sub>2</sub>O under vacuum, and the counts were measured using a MicroBeta (PerkinElmer). The maximum signal was obtained with the addition of the enzyme and substrate, and the minimum signal was obtained with the substrate only. SAH (Sigma, product NO. A9384) was used as the reference compound.</div><div class="NLM_p">The radioactive methylation inhibition assay of DNMT3A/3L utilized IDT-01 sequence 1 and IDT-01 sequence 2 as substrates in modified Tris buffer. The enzyme solution was incubated at room temperature for 15 min before 10 μL of substrate solution was added to each well to start the reaction. After incubation at room temperature for 4 h, cold SAM solution (10 μL) was added to each well to stop the reaction. Subsequently, 25 μL of the reaction was transferred to a FlashPlate microplate (PerkinElmer Inc., USA, catalog NO. SMP410A001PK), and the radioactivity was determined by liquid scintillation counting (MicroBeta, PerkinElmer). Sinefungin (Sigma, product NO. S8559) was used as the reference compound.</div><div class="NLM_p">The radioactive methylation inhibition assay of EZH2 was performed in modified Tris buffer. H3K27me peptide and [<sup>3</sup>H]SAM (PerkinElmer Inc., USA, lot. NO. 1731619) were added in 1× buffer as the substrate solution. The enzyme solution was incubated at room temperature for 15 min before 10 μL of substrate solution was added to each well to start the reaction. Cold SAM (Sigma, catalog NO. 7007) was added in 1× buffer to prepare the stop mix (final concentration 0.5 mM), and 10 μL was added per well; 25 μL was transferred to a FlashPlate (PerkinElmer, catalog NO. SMP410A001PK) and incubated at room temperature for a minimum of 1 h. The plate was washed three times with dH<sub>2</sub>O and 0.1% Tween-20, and the radioactivity was determined by liquid scintillation counting (MicroBeta, PerkinElmer). GSK-126 was used as the reference compound.</div><div class="NLM_p">A radioactive methylation inhibition assay of NSD1 was performed in modified Tris buffer. The enzyme solution of NSD1 (10 μL; Biogenie, catalog NO. M1057) was transferred to an assay plate and incubated at room temperature for 15 min. Subsequently, 10 μL of substrate solution was added to each well, as was 10 μL of [<sup>3</sup>H]SAM solution to start the reaction. After incubation for 4 h, 15 μL of cold SAM solution was added to each well to stop the reaction. The reaction mixture (40 μL) was transferred to GF/B plate (Millipore, catalog NO. MSFBN6B50) (pretreated with 0.5% PEI for 15 min) using Platemate, and the plate was washed 3 times with ddH<sub>2</sub>O under vacuum. The radioactivity was determined by liquid scintillation counting (MicroBeta, PerkinElmer). Chaetocin was used as the reference compound.</div><div class="NLM_p">The inhibition activities of DC-S238 and DC-S239 against DOT1L were determined using AlphaLISA protocols. In each plate well, an amount of 5 μL of compound or assay buffer was added and so was 2.5 μL of DOT1L enzyme. After incubation for 15 min at room temperature, 2.5 μL of oligonucleosomes/SAM mix was added to start the reaction; the plate was covered with TopSeal-A film (PerkinElmer, NO. 6050195) and incubated at room temperature for 30 min. To stop the reaction, 5 μL of high-salt buffer was added. A 5× mixture of anti-histone H3 acceptor beads (PerkinElmer, NO. AL147) and biotinylated anti-H3K79me2 antibody (PerkinElmer, NO. AL148) was prepared at 50 μg/mL and 0.5 nM in a 25 μL total assay volume. Subsequently, 5 μL of 5× acceptor beads/biotinylated antibody mix was added; the samples were covered with TopSeal-A film and incubated for 60 min at room temperature. A 5× streptavidin donor beads (PerkinElmer, NO. 6760002) sample at 50 μg/mL was prepared in 1× detection buffer and shielded from light (final concentration of 10 μg/mL in 25 μL total assay volume). Donor beads (5 μL) were added, shielded from light, and the sample was covered with TopSeal-A film and incubated shielded from light for 30 min at room temperature. The signal was measured in Alpha mode with Envision. EPZ004777 (Selleckchem, NO. S7353) was used as the reference compound.</div><div class="NLM_p">The radioactive enzymatic assay of SETD8 was performed in modified Tris buffer. The compounds were first transferred to assay plate by Echo in 100% DMSO. Both peptide and [<sup>3</sup>H]-SAM in 1× assay buffer (modified Tris buffer) were added to each well to start the reaction before the compounds, and an amount of 10 μL of enzyme solution was incubated at room temperature for 15 min. And 10 μL of cold SAM in 1× assay buffer was added to each well to stop the reaction after 120 min. 25 μL of volume per well was transferred to streptavidin coated Flashplate microplate (PerkinElmer Inc., USA, catalog NO. SMP410A001PK) from assay plate, and each was incubated for 60 min minimum at room temperature. The Flashplate was washed with dH<sub>2</sub>O + 0.1% Tween-20 three times and read on Microbeta. SAH (Sigma, catalog NO. A9384-25MG) was used as the reference compound.</div><div class="NLM_p last">The inhibition activities of DC-S238 and DC-S239 against G9a were determined using AlphaLISA protocols. Both DC-S238 and DC-S239 were transferred to assay plate by Echo in 100% DMSO, and the enzyme solution was prepared in 1× assay buffer (modified Tris buffer). The compounds and 5 μL of the enzyme solution were incubated in assay plate in each well at room temperature for 15 min. The 5 μL of susbstrate mix solution which was prepared in 1× assay buffer was added to each well to start the reation and was incubated at room temperature for 60 min. Then 15 μL of acceptor and donor beads mix solution was added and incubated for 60 min at room temperature, in subdued light.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cell Culture and Cell Viability Assay</h3><div class="NLM_p last">MCF7, HL60, MV4-11, HCT116, and DHL4 cell lines were purchased from American Type Culture Collection. Fetal bovine serum was purchased from Life Technologies. MCF7 cells were cultured in DMEM medium supplemented with 10% fetal bovine serum at 37 °C in an incubator with 5% CO<sub>2</sub> atmosphere. HL60, MV4-11, and DHL4 cells were cultured in 1640 medium supplemented with 10% fetal bovine serum. HCT116 cells were cultured in 5A medium. DC-S239 was dissolved in DMSO (Sigma) and then stored at 4 °C. Approximately 2 × 10<sup>4</sup> cells were seeded in 100 μL of medium in each well of a 96-well flat-bottom plate and then grown in a CO<sub>2</sub> incubator overnight. The cells were incubated with DC-S239 at different concentrations ranging from 0 to 100 μM for approximately 120 h. MCF7 and HCT116 cells were treated with 10 μL MTT solution and incubated for 2 h at 37 °C. The absorbance was measured at 490 nm using a microplate reader (BMG Labtech) after adding 100 μL of DMSO to each well to dissolve the crystals; 630 nm was used as the reference wavelength. The activity of DC-S239 against HL60, MV4-11, and DHL4 cells was measured by the alamarBlue assay.<a onclick="showRef(event, 'ref108 ref109'); return false;" href="javascript:void(0);" class="ref ref108 ref109">(108, 109)</a> Fluorescence was measured using a fluorescence excitation wavelength of 544 nm and fluorescence emission at 590 nm. To express the percent of proliferation, the absorbance and fluorescence were normalized with control wells.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry</h3><div class="NLM_p last">Commercially available chemicals were used without further purification. All products were characterized by their NMR and MS spectra. <sup>1</sup>H NMR spectra were recorded using a 400 MHz Varian Mercury Plus 400 instrument, and <sup>13</sup>C NMR spectra were recorded using a 400 MHz Varian Mercury Plus 400 instrument or a 500 MHz Bruker AVANCE III 500 instrument. Chemical shifts are reported in parts per million (ppm, δ) downfield from tetramethylsilane. Proton coupling patterns are described as singlet (s), doublet (d), triplet (t), quartet (q), multiplet (m). Low-resolution mass spectra (LRMS) were measured using a Finnigan LCQ/DECA spectrometer. High-resolution mass spectra (HRMS) were measured using a Micromass Ultra Q-TOF spectrometer. Melting points (uncorrected) were determined using a SGWX-4B micro melting point apparatus. All of the microwave-assisted reactions were performed using an Anton Paar Monowave 300 monomode microwave reactor.</div></div><div id="sec4_9_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> General Procedure for Preparation of Acetoacetanilide Derivatives <b>B01</b>–<b>B15</b></h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0010.gif" alt="" id="fx1" /></img><div class="NLM_p last">A mixture of ethyl acetoacetate (0.06 mol), an appropriate amine (0.03 mol), and a catalytic amount of potassium <i>tert</i>-butoxide was heated in a microwave reactor at 120 °C for 2 h. The crude product was purified by flash chromatography on silica gel using dichloromethane as the eluent to generate the product as a solid in 40–70% yield.</div></div><div id="sec4_9_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> <i>N</i>-(3-Nitrophenyl)-3-oxobutanamide (<b>B01</b>)</h3><div class="NLM_p last"><b>B01</b> was prepared from 3-nitroaniline (<b>A01</b>). LRMS (ESI) <i>m</i>/<i>z</i> 223 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.57 (s, 1H), 8.61 (t, <i>J</i> = 2.1 Hz, 1H), 8.01–7.78 (m, 2H), 7.61 (t, <i>J</i> = 8.2 Hz, 1H), 3.61 (s, 2H), 2.22 (s, 3H).</div></div><div id="sec4_9_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> 3-Oxo-<i>N</i>-phenylbutanamide (<b>B02</b>)</h3><div class="NLM_p last"><b>B02</b> was prepared from aniline (<b>A02</b>). LRMS (ESI) <i>m</i>/<i>z</i> 178 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.08 (s, 1H), 7.57 (dd, <i>J</i> = 8.6, 1.1 Hz, 2H), 7.36–7.24 (m, 2H), 7.10–6.97 (m, 1H), 3.54 (s, 2H), 2.20 (s, 3H).</div></div><div id="sec4_9_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(4-Methoxyphenyl)-3-oxobutanamide (<b>B03</b>)</h3><div class="NLM_p last"><b>B03</b> was prepared from 4-methoxyaniline (<b>A03</b>). LRMS (ESI) <i>m</i>/<i>z</i> 208 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.94 (s, 1H), 7.55–7.39 (m, 2H), 6.97–6.81 (m, 2H), 3.71 (s, 3H), 3.50 (s, 2H), 2.20 (s, 3H).</div></div><div id="sec4_9_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> <i>N</i>-(4-Bromophenyl)-3-oxobutanamide (<b>B04</b>)</h3><div class="NLM_p last"><b>B04</b> was prepared from 4-bromoaniline (<b>A04</b>). LRMS (ESI) <i>m</i>/<i>z</i> 256 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.21 (s, 1H), 7.57–7.52 (m, 2H), 7.51–7.46 (m, 2H), 3.55 (s, 2H), 2.20 (s, 3H).</div></div><div id="sec4_9_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> <i>N</i>-(2-Fluorophenyl)-3-oxobutanamide (<b>B05</b>)</h3><div class="NLM_p last"><b>B05</b> was prepared from 2-fluoroaniline (<b>A05</b>). LRMS (ESI) <i>m</i>/<i>z</i> 196 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.91 (s, 1H), 8.00–7.92 (m, 1H), 7.30–7.22 (m, 1H), 7.20–7.09 (m, 2H), 3.64 (s, 2H), 2.20 (s, 3H).</div></div><div id="sec4_9_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> <i>N</i>-(3-Methoxyphenyl)-3-oxobutanamide (<b>B06</b>)</h3><div class="NLM_p last"><b>B06</b> was prepared from 3-methoxyaniline (<b>A06</b>). LRMS (ESI) <i>m</i>/<i>z</i> 208 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.05 (s, 1H), 7.25 (t, <i>J</i> = 2.2 Hz, 1H), 7.19 (t, <i>J</i> = 8.1 Hz, 1H), 7.11–7.02 (m, 1H), 6.62 (ddd, <i>J</i> = 8.2, 2.5, 0.8 Hz, 1H), 3.71 (s, 3H), 3.52 (s, 2H), 2.19 (s, 3H).</div></div><div id="sec4_9_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> 4-(3-Oxobutanamido)benzamide (<b>B07</b>)</h3><div class="NLM_p last"><b>B07</b> was prepared from 4-aminobenzamide (<b>A07</b>). LRMS (ESI) <i>m</i>/<i>z</i> 221 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.29 (s, 1H), 7.87 (s, 1H), 7.83 (d, <i>J</i> = 8.7 Hz, 2H), 7.62 (d, <i>J</i> = 8.7 Hz, 2H), 7.26 (s, 1H), 3.58 (s, 2H), 2.21 (s, 3H).</div></div><div id="sec4_9_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> <i>N</i>-(4-Chlorophenyl)-3-oxobutanamide (<b>B08</b>)</h3><div class="NLM_p last"><b>B08</b> was prepared from 4-chloroaniline (<b>A08</b>). LRMS (ESI) <i>m</i>/<i>z</i> 212 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.21 (s, 1H), 7.65–7.52 (m, 2H), 7.42–7.27 (m, 2H), 3.55 (s, 2H), 2.20 (s, 3H).</div></div><div id="sec4_9_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> <i>N</i>-(2-Chlorophenyl)-3-oxobutanamide (<b>B09</b>)</h3><div class="NLM_p last"><b>B09</b> was prepared from 2-chloroaniline (<b>A09</b>). LRMS (ESI) <i>m</i>/<i>z</i> 212 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO) δ 9.78 (s, 1H), 7.88–7.75 (m, 1H), 7.50 (dd, <i>J</i> = 8.0, 1.3 Hz, 1H), 7.39–7.28 (m, 1H), 7.18 (td, <i>J</i> = 7.9, 1.4 Hz, 1H), 3.66 (s, 2H), 2.21 (s, 3H).</div></div><div id="sec4_9_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> <i>N</i>-(2,5-Dichlorophenyl)-3-oxobutanamide (<b>B10</b>)</h3><div class="NLM_p last"><b>B10</b> was prepared from 2,5-dichloroaniline (<b>A10</b>). LRMS (ESI) <i>m</i>/<i>z</i> 246 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.89 (s, 1H), 8.01 (d, <i>J</i> = 2.4 Hz, 1H), 7.54 (d, <i>J</i> = 8.6 Hz, 1H), 7.25 (dd, <i>J</i> = 8.6, 2.5 Hz, 1H), 3.71 (s, 2H), 2.21 (s, 3H).</div></div><div id="sec4_9_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> <i>N</i>-(4-Chloro-2,5-dimethoxyphenyl)-3-oxobutanamide (<b>B11</b>)</h3><div class="NLM_p last"><b>B11</b> was prepared from 4-chloro-2,5-dimethoxyaniline (<b>A11</b>). LRMS (ESI) <i>m</i>/<i>z</i> 272 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.58 (s, 1H), 8.03 (s, 1H), 7.14 (s, 1H), 3.81 (s, 3H), 3.75 (s, 3H), 3.69 (s, 2H), 2.18 (s, 3H).</div></div><div id="sec4_9_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> <i>N</i>-(2-Methoxyphenyl)-3-oxobutanamide (<b>B12</b>)</h3><div class="NLM_p last"><b>B12</b> was prepared from 2-methoxyaniline (<b>A12</b>). LRMS (ESI) <i>m</i>/<i>z</i> 208 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.47 (s, 1H), 8.06–7.95 (m, 1H), 7.11–6.99 (m, 2H), 6.94–6.83 (m, 1H), 3.83 (s, 3H), 3.66 (s, 2H), 2.18 (s, 3H).</div></div><div id="sec4_9_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> <i>N</i>-(4-Fluorophenyl)-3-oxobutanamide (<b>B13</b>)</h3><div class="NLM_p last"><b>B13</b> was prepared from 4-fluoroaniline (<b>A13</b>). LRMS (ESI) <i>m</i>/<i>z</i> 196 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 7.64–7.49 (m, 2H), 7.13 (t, <i>J</i> = 8.9 Hz, 2H), 3.52 (s, 2H), 2.19 (s, 3H).</div></div><div id="sec4_9_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> <i>N</i>-(3-Chlorophenyl)-3-oxobutanamide (<b>B14</b>)</h3><div class="NLM_p last"><b>B14</b> was prepared from 3-chloroaniline (<b>A14</b>). LRMS (ESI) <i>m</i>/<i>z</i> 212 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.27 (s, 1H), 7.81 (t, <i>J</i> = 2.0 Hz, 1H), 7.46–7.24 (m, 2H), 7.12 (ddd, <i>J</i> = 7.8, 2.1, 1.2 Hz, 1H), 3.56 (s, 2H), 2.21 (s, 3H).</div></div><div id="sec4_9_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> <i>N</i>-(3-Cyanophenyl)-3-oxobutanamide (<b>B15</b>)</h3><div class="NLM_p last"><b>B15</b> was prepared from 3-aminobenzonitrile (<b>A15</b>). LRMS (ESI) <i>m</i>/<i>z</i> 203 [M + H]<sup>+</sup>; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.43 (s, 1H), 8.08 (d, <i>J</i> = 1.0 Hz, 1H), 7.79–7.72 (m, 1H), 7.57–7.49 (m, 2H), 3.59 (s, 2H), 2.21 (s, 3H).</div></div><div id="sec4_9_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> General Procedure for Preparation of 2-Amino-4-methylthiophene Derivatives DC-S237 to DC-S258</h3><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0011.gif" alt="" id="fx2" /></img><div class="NLM_p last">These compounds were prepared via the Gewald reaction. To a stirred equimolecular suspension of acetoacetanilide derivative (0.005 mol), ethyl cyanoacetate (isobutyl cyanoacetate for DC-S237, methyl cyanoacetate for DC-S238, 2-cyanoacetamide for DC-S240, isopropyl cyanoacetate for DC-S241, <i>tert</i>-butyl cyanoacetate for DC-S242, cyclohexyl cyanoacetate for DC-S243, and benzyl cyanoacetate for DC-S244) (0.005 mol) and elemental sulfur (0.005 mol) in ethanol (isobutanol for DC-S237, methanol for DC-S238, isopropanol for DC-S241, cyclohexanol for DC-S243, and benzyl alcohol for DC-S244) (25 mL) were added simultaneously in quintuple, with an excess amount of morpholine. The mixture was stirred at 80 °C for 1–2 h. The precipitate formed after cooling was filtered off and washed with ethanol to afford the desired compound. As DC-S249 has good solubility in ethanol, no precipitate was formed after cooling, and it was obtained via purification of the crude product by flash chromatography on silica gel. For preparation of DC-S243 and DC-S244, the reaction mixture was diluted with ethanol after cooling to room temperature and stirred, and the precipitate then formed. The purity of the synthesized compounds was determined to be greater than 95% except for DC-S243 (purity 93.2%) by HPLC analysis [HPLC: column, Agilent XDB-C18, 150 mm × 4.6 mm, 5 μm; solvent system, MeOH–1 ‰ DEA and water with isocratic elution (68% organic phase for DC-S243, DC-S253, and DC-S254; 80% for the rest compounds); flow rate, 1.0 mL/min; UV detection, 254 nm; oven, 20 °C. Agilent 1260, Agilent Technologies, California, USA]. Detailed HPLC analysis data of the synthesized compounds are displayed in (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">Table S4 Supporting Information</a>).</div></div><div id="sec4_9_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Isobutyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S237)</h3><div class="NLM_p last">DC-S237 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and isobutyl cyanoacetate following the general procedure indicated above. Yield, 62%; mp 190–191 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.15 (s, 1H), 8.62 (t, <i>J</i> = 2.1 Hz, 1H), 8.00 (dd, <i>J</i> = 8.2, 1.2 Hz, 1H), 7.95–7.83 (m, 3H), 7.60 (t, <i>J</i> = 8.2 Hz, 1H), 4.00 (d, <i>J</i> = 6.4 Hz, 2H), 2.54 (s, 3H), 2.05–1.95 (m, 1H), 0.96 (d, <i>J</i> = 6.7 Hz, 6H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.75, 165.22, 161.81, 147.89, 141.63, 140.36, 129.98, 126.01, 117.75, 114.10, 111.78, 105.62, 69.54, 27.35, 19.17, 16.80; LRMS (ESI) <i>m</i>/<i>z</i> 378 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>17</sub>H<sub>20</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 378.1118, found 378.1126.</div></div><div id="sec4_9_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Methyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S238)</h3><div class="NLM_p last">DC-S238 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and methyl cyanoacetate following the general procedure indicated above. Yield, 59%; mp 238–239 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 8.62 (t, <i>J</i> = 2.1 Hz, 1H), 8.00 (ddd, <i>J</i> = 8.2, 2.0, 0.8 Hz, 1H), 7.91 (ddd, <i>J</i> = 8.2, 2.3, 0.8 Hz, 1H), 7.84 (s, 2H), 7.60 (t, <i>J</i> = 8.2 Hz, 1H), 3.75 (s, 3H), 2.51 (d, <i>J</i> = 4.5 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.35, 165.12, 161.77, 147.89, 142.32, 140.38, 129.95, 126.04, 117.73, 114.13, 111.58, 105.61, 50.81, 16.48; LRMS (ESI) <i>m</i>/<i>z</i> 336 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>14</sub>H<sub>14</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 336.0649, found 336.0648.</div></div><div id="sec4_9_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Ethyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S239)</h3><div class="NLM_p last">DC-S239 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 78%; mp 202–204 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.13 (s, 1H), 8.63 (t, <i>J</i> = 2.1 Hz, 1H), 8.00 (ddd, <i>J</i> = 8.2, 1.9, 0.7 Hz, 1H), 7.91 (ddd, <i>J</i> = 8.2, 2.3, 0.8 Hz, 1H), 7.84 (s, 2H), 7.60 (t, <i>J</i> = 8.2 Hz, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.52 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.42, 164.87, 161.80, 147.89, 142.12, 140.38, 129.97, 126.01, 117.73, 114.10, 111.66, 105.73, 59.41, 16.62, 14.31; LRMS (ESI) <i>m</i>/<i>z</i> 350 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 350.0805, found 350.0811.</div></div><div id="sec4_9_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> 5-Amino-3-methyl-<i>N</i><sup>2</sup>-(3-nitrophenyl)thiophene-2,4-dicarboxamide (DC-S240)</h3><div class="NLM_p last">DC-S240 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and 2-cyanoacetamide following the general procedure indicated above. Yield, 63%; mp 244–245 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.96 (s, 1H), 8.63 (t, <i>J</i> = 2.1 Hz, 1H), 8.04–7.97 (m, 1H), 7.89 (ddd, <i>J</i> = 8.2, 2.3, 0.8 Hz, 1H), 7.59 (t, <i>J</i> = 8.2 Hz, 1H), 7.37 (s, 2H), 7.05 (s, 2H), 2.51 (s, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.79, 162.41, 161.77, 148.32, 142.12, 141.08, 130.31, 126.58, 117.95, 114.64, 112.94, 110.60, 16.38; LRMS (ESI) <i>m</i>/<i>z</i> 321 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>13</sub>H<sub>13</sub>O<sub>4</sub>N<sub>4</sub>S [M + H]<sup>+</sup> 321.0652, found 321.0650.</div></div><div id="sec4_9_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Isopropyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S241)</h3><div class="NLM_p last">DC-S241 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and isopropyl cyanoacetate following the general procedure indicated above. Yield, 81%; mp 197–199 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.14 (s, 1H), 8.63 (t, <i>J</i> = 2.1 Hz, 1H), 8.05–7.97 (m, 1H), 7.91 (ddd, <i>J</i> = 8.2, 2.2, 0.7 Hz, 1H), 7.84 (s, 2H), 7.60 (t, <i>J</i> = 8.2 Hz, 1H), 5.13–5.04 (m, 1H), 2.51 (s, 3H), 1.29 (d, <i>J</i> = 6.3 Hz, 6H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.47, 164.58, 161.82, 147.89, 141.86, 140.39, 129.97, 125.96, 117.72, 114.06, 111.79, 105.93, 66.83, 21.83, 16.79; LRMS (ESI) <i>m</i>/<i>z</i> 364 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>18</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 364.0962, found 364.0968.</div></div><div id="sec4_9_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> <i>tert</i>-Butyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S242)</h3><div class="NLM_p last">DC-S242 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and <i>tert</i>-butyl cyanoacetate following the general procedure indicated above. Yield, 63%; mp 196–197 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 8.63 (t, <i>J</i> = 2.0 Hz, 1H), 8.00 (dd, <i>J</i> = 8.2, 1.1 Hz, 1H), 7.90 (dd, <i>J</i> = 8.2, 1.5 Hz, 1H), 7.81 (s, 2H), 7.59 (t, <i>J</i> = 8.2 Hz, 1H), 2.49 (s, 3H), 1.52 (s, 9H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.30, 164.71, 161.87, 147.89, 141.77, 140.41, 129.97, 125.93, 117.69, 114.02, 111.57, 106.89, 80.20, 28.13, 16.98; LRMS (ESI) <i>m</i>/<i>z</i> 378 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>17</sub>H<sub>20</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 378.1118, found 378.1128.</div></div><div id="sec4_9_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Cyclohexyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S243)</h3><div class="NLM_p last">DC-S243 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and cyclohexyl cyanoacetate following the general procedure indicated above. After cooling to room temperature, the reaction mixture was diluted with ethanol and stirred, and the precipitate then formed. Yield, 52%; mp 189–191 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 8.61 (s, 1H), 7.99 (d, <i>J</i> = 8.3 Hz, 1H), 7.89 (dd, <i>J</i> = 8.1, 1.4 Hz, 1H), 7.84 (s, 2H), 7.59 (t, <i>J</i> = 8.2 Hz, 1H), 4.97–4.79 (m, 1H), 2.53 (s, 3H), 1.83 (d, <i>J</i> = 3.7 Hz, 2H), 1.66 (s, 2H), 1.53 (dd, <i>J</i> = 19.3, 10.0 Hz, 3H), 1.44–1.28 (m, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.62, 164.57, 161.82, 147.90, 141.65, 140.38, 129.98, 125.97, 117.73, 114.07, 111.85, 105.93, 71.25, 65.00, 44.23, 31.13, 24.97, 23.12, 16.93; LRMS (ESI) <i>m</i>/<i>z</i> 404 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>19</sub>H<sub>22</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 404.1275, found 404.1270.</div></div><div id="sec4_9_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Benzyl 2-Amino-4-methyl-5-((3-nitrophenyl)carbamoyl)thiophene-3-carboxylate (DC-S244)</h3><div class="NLM_p last">DC-S244 was prepared from <i>N</i>-(3-nitrophenyl)-3-oxobutanamide (<b>B01</b>) and benzyl cyanoacetate following the general procedure indicated above. After cooling to room temperature, the reaction mixture was diluted with ethanol and stirred, and the precipitate then formed. Yield, 33%; mp 205–207 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 10.12 (s, 1H), 9.11–6.78 (m, 11H), 5.29 (s, 2H), 2.50 (s, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.86, 164.64, 161.76, 147.90, 141.76, 140.34, 136.67, 129.98, 128.51, 127.96, 126.01, 117.76, 114.10, 111.95, 105.37, 64.96, 16.78; LRMS (ESI) <i>m</i>/<i>z</i> 412 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>20</sub>H<sub>18</sub>O<sub>5</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 412.0962, found 412.0953.</div></div><div id="sec4_9_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Ethyl 2-Amino-4-methyl-5-(phenylcarbamoyl)thiophene-3-carboxylate (DC-S245)</h3><div class="NLM_p last">DC-S245 was prepared from 3-oxo-<i>N</i>-phenylbutanamide (<b>B02</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 65%; mp 178–179 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.69 (s, 1H), 7.75 (s, 2H), 7.60 (d, <i>J</i> = 7.6 Hz, 2H), 7.37–7.23 (m, 2H), 7.05 (t, <i>J</i> = 7.4 Hz, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.49 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.11, 164.95, 161.42, 140.29, 139.14, 128.59, 123.32, 120.05, 112.96, 105.40, 59.31, 16.58, 14.33; LRMS (ESI) <i>m</i>/<i>z</i> 305 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>17</sub>O<sub>3</sub>N<sub>2</sub>S [M + H]<sup>+</sup> 305.0954, found 305.0948.</div></div><div id="sec4_9_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Ethyl 2-Amino-5-((4-methoxyphenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S246)</h3><div class="NLM_p last">DC-S246 was prepared from <i>N</i>-(4-methoxyphenyl)-3-oxobutanamide (<b>B03</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 49%; mp 160–161 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.55 (s, 1H), 7.72 (s, 2H), 7.56–7.46 (m, 2H), 6.91–6.84 (m, 2H), 4.22 (q, <i>J</i> = 7.1 Hz, 2H), 3.72 (s, 3H), 2.48 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 164.97, 161.11, 155.37, 139.82, 132.19, 121.74, 113.71, 113.08, 105.33, 59.29, 55.17, 16.55, 14.33; LRMS (ESI) <i>m</i>/<i>z</i> 335 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>19</sub>O<sub>4</sub>N<sub>2</sub>S [M + H]<sup>+</sup> 335.1060, found 335.1057.</div></div><div id="sec4_9_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Ethyl 2-Amino-5-((4-bromophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S247)</h3><div class="NLM_p last">DC-S247 was prepared from <i>N</i>-(4-bromophenyl)-3-oxobutanamide (<b>B04</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 73%; mp 170–171 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.81 (s, 1H), 7.77 (s, 2H), 7.63–7.54 (m, 2H), 7.52–7.44 (m, 2H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.49 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.21, 164.91, 161.45, 140.94, 138.55, 131.39, 121.93, 114.92, 112.46, 105.51, 59.35, 16.59, 14.32; LRMS (ESI) <i>m</i>/<i>z</i> 383 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>BrS [M + H]<sup>+</sup> 383.0060, found 383.0063.</div></div><div id="sec4_9_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Ethyl 2-Amino-5-((2-fluorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S248)</h3><div class="NLM_p last">DC-S248 was prepared from <i>N</i>-(2-fluorophenyl)-3-oxobutanamide (<b>B05</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 55%; mp 140–141 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.35 (s, 1H), 7.78 (s, 2H), 7.59 (td, <i>J</i> = 7.8, 1.9 Hz, 1H), 7.29–7.11 (m, 3H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.54 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.32, 164.93, 161.39, 155.30 (d, <i>J</i> = 2476.4 Hz), 141.55, 126.28 (d, <i>J</i> = 7.5 Hz), 126.11, 126.03, 124.26 (d, <i>J</i> = 3.3 Hz), 115.66 (d, <i>J</i> = 19.8 Hz), 111.99, 105.68, 59.36, 16.42, 14.31; LRMS (ESI) <i>m</i>/<i>z</i> 323 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>FS [M + H]<sup>+</sup> 323.0860, found 323.0865.</div></div><div id="sec4_9_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Ethyl 2-Amino-5-((3-methoxyphenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S249)</h3><div class="NLM_p last">DC-S249 was prepared from <i>N</i>-(3-methoxyphenyl)-3-oxobutanamide (<b>B06</b>) and ethyl cyanoacetate following the general procedure indicated above. After cooling to room temperature, the reaction mixture was diluted with EtOAc and washed with H<sub>2</sub>O and then brine. The organic layer was dried over anhydrous Na<sub>2</sub>SO<sub>4</sub>, filtered, and concentrated under reduced pressure. The crude product was purified by flash chromatography on silica gel. Chromatographic eluent: 20% EtOAc in petroleum ether. Yield, 38%; mp 124–125 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.67 (s, 1H), 7.76 (s, 2H), 7.34–7.27 (m, 1H), 7.25–7.08 (m, 2H), 6.63 (dt, <i>J</i> = 7.2, 2.4 Hz, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 3.72 (s, 3H), 2.48 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.14, 164.95, 161.42, 159.43, 140.35, 140.34, 129.35, 112.96, 112.26, 108.80, 105.70, 105.42, 59.32, 54.98, 16.59, 14.32; LRMS (ESI) <i>m</i>/<i>z</i> 335 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>19</sub>O<sub>4</sub>N<sub>2</sub>S [M + H]<sup>+</sup> 335.1060, found 335.1057.</div></div><div id="sec4_9_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> Ethyl 2-Amino-5-((4-carbamoylphenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S250)</h3><div class="NLM_p last">DC-S250 was prepared from 4-(3-oxobutanamido)benzamide (<b>B07</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 47%; mp 270–272 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.90 (s, 1H), 7.90–7.76 (m, 5H), 7.67 (d, <i>J</i> = 8.8 Hz, 2H), 7.26 (s, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.50 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.42, 165.29, 164.92, 161.56, 141.85, 141.09, 128.75, 128.23, 118.98, 112.55, 105.56, 59.37, 16.62, 14.32; LRMS (ESI) <i>m</i>/<i>z</i> 348 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>18</sub>O<sub>4</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 348.1013, found 348.1013.</div></div><div id="sec4_9_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Ethyl 2-Amino-5-((4-chlorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S251)</h3><div class="NLM_p last">DC-S251 was prepared from <i>N</i>-(4-chlorophenyl)-3-oxobutanamide (<b>B08</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 53%; mp 168–169 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.82 (s, 1H), 7.78 (s, 2H), 7.69–7.56 (m, 2H), 7.46–7.23 (m, 2H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.49 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.65, 165.37, 161.90, 141.35, 138.58, 128.92, 127.34, 122.02, 112.92, 105.97, 59.79, 17.02, 14.76; LRMS (ESI) <i>m</i>/<i>z</i> 339 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>ClS [M + H]<sup>+</sup> 339.0565, found 339.0573.</div></div><div id="sec4_9_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Ethyl 2-Amino-5-((2-chlorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S252)</h3><div class="NLM_p last">DC-S252 was prepared from <i>N</i>-(2-chlorophenyl)-3-oxobutanamide (<b>B09</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 44%; mp 154–156 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.18 (s, 1H), 7.80 (s, 2H), 7.70 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 7.51 (dd, <i>J</i> = 8.0, 1.4 Hz, 1H), 7.34 (td, <i>J</i> = 7.8, 1.4 Hz, 1H), 7.22 (td, <i>J</i> = 7.8, 1.6 Hz, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.59 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.84, 165.36, 161.70, 141.93, 135.73, 129.89, 128.32, 127.92, 127.05, 127.01, 112.63, 106.24, 59.83, 16.85, 14.76; LRMS (ESI) <i>m</i>/<i>z</i> 339 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>ClS [M + H]<sup>+</sup> 339.0565, found 339.0563.</div></div><div id="sec4_9_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Ethyl 2-Amino-5-((2,5-dichlorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S253)</h3><div class="NLM_p last">DC-S253 was prepared from <i>N</i>-(2,5-dichlorophenyl)-3-oxobutanamide (<b>B10</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 64%; mp 173–175 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.24 (s, 1H), 7.88 (d, <i>J</i> = 2.6 Hz, 1H), 7.85 (s, 2H), 7.56 (d, <i>J</i> = 8.6 Hz, 1H), 7.29 (dd, <i>J</i> = 8.6, 2.6 Hz, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 2.60 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>) δ 166.06, 165.30, 161.67, 142.70, 137.02, 131.97, 131.21, 126.40, 125.75, 112.19, 106.39, 59.88, 16.87, 14.75; LRMS (ESI) <i>m</i>/<i>z</i> 373 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>15</sub>O<sub>3</sub>N<sub>2</sub>Cl<sub>2</sub>S [M + H]<sup>+</sup> 373.0175, found 373.0177.</div></div><div id="sec4_9_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> Ethyl 2-Amino-5-((4-chloro-2,5-dimethoxyphenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S254)</h3><div class="NLM_p last">DC-S254 was prepared from <i>N</i>-(4-chloro-2,5-dimethoxyphenyl)-3-oxobutanamide (<b>B11</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 67%; mp 190–191 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.59 (s, 1H), 7.92 (s, 1H), 7.83 (s, 2H), 7.18 (s, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 3.83 (s, 3H), 3.78 (s, 3H), 2.59 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.50, 164.86, 160.70, 148.18, 143.56, 141.10, 127.13, 115.06, 113.11, 112.70, 106.29, 105.97, 59.44, 56.88, 56.36, 16.07, 14.31; LRMS (ESI) <i>m</i>/<i>z</i> 399 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>17</sub>H<sub>20</sub>O<sub>5</sub>N<sub>2</sub>ClS [M + H]<sup>+</sup> 399.0776, found 399.0780.</div></div><div id="sec4_9_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Ethyl 2-Amino-5-((2-methoxyphenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S255)</h3><div class="NLM_p last">DC-S255 was prepared from <i>N</i>-(2-methoxyphenyl)-3-oxobutanamide (<b>B12</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 48%; mp 142–143 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 8.54 (s, 1H), 7.95 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.79 (s, 2H), 7.20–7.00 (m, 2H), 6.99–6.82 (m, 1H), 4.23 (q, <i>J</i> = 7.1 Hz, 2H), 3.85 (s, 3H), 2.59 (s, 3H), 1.29 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.33, 164.91, 160.65, 149.55, 140.46, 127.37, 124.44, 121.37, 120.41, 113.10, 111.08, 105.84, 59.40, 55.98, 16.06, 14.32; LRMS (ESI) <i>m</i>/<i>z</i> 335 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>19</sub>O<sub>4</sub>N<sub>2</sub>S [M + H]<sup>+</sup> 335.1060, found 335.1058.</div></div><div id="sec4_9_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Ethyl 2-Amino-5-((4-fluorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S256)</h3><div class="NLM_p last">DC-S256 was prepared from <i>N</i>-(4-fluorophenyl)-3-oxobutanamide (<b>B13</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 51%; mp 151–153 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.73 (s, 1H), 7.75 (s, 2H), 7.67–7.54 (m, 2H), 7.14 (t, <i>J</i> = 8.9 Hz, 2H), 4.22 (q, <i>J</i> = 7.1 Hz, 2H), 2.48 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.11, 164.94, 161.37, 158.12 (d, <i>J</i> = 239.8 Hz), 140.53, 135.48, 121.91 (d, <i>J</i> = 7.7 Hz), 115.14 (d, <i>J</i> = 22.1 Hz), 112.61, 105.44, 59.32, 16.56, 14.32; LRMS (ESI) <i>m</i>/<i>z</i> 323 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>FS [M + H]<sup>+</sup> 323.0860, found 323.0856.</div></div><div id="sec4_9_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Ethyl 2-Amino-5-((3-chlorophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S257)</h3><div class="NLM_p last">DC-S257 was prepared from <i>N</i>-(3-chlorophenyl)-3-oxobutanamide (<b>B14</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 58%; mp 154–156 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.85 (s, 1H), 7.79 (s, 2H), 7.78 (t, <i>J</i> = 2.0 Hz, 1H), 7.53 (ddd, <i>J</i> = 8.3, 1.9, 0.9 Hz, 1H), 7.32 (t, <i>J</i> = 8.1 Hz, 1H), 7.10 (ddd, <i>J</i> = 8.0, 2.1, 0.9 Hz, 1H), 4.22 (q, <i>J</i> = 7.1 Hz, 2H), 2.49 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.29, 164.90, 161.58, 141.26, 140.66, 132.92, 130.27, 122.95, 119.41, 118.35, 112.27, 105.58, 59.37, 16.59, 14.31; LRMS (ESI) <i>m</i>/<i>z</i> 339 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>15</sub>H<sub>16</sub>O<sub>3</sub>N<sub>2</sub>ClS [M + H]<sup>+</sup> 339.0565, found 339.0555.</div></div><div id="sec4_9_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Ethyl 2-Amino-5-((3-cyanophenyl)carbamoyl)-4-methylthiophene-3-carboxylate (DC-S258)</h3><div class="NLM_p last">DC-S258 was prepared from <i>N</i>-(3-cyanophenyl)-3-oxobutanamide (<b>B15</b>) and ethyl cyanoacetate following the general procedure indicated above. Yield, 66%; mp 209–210 °C; <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ 9.98 (s, 1H), 8.19–8.00 (m, 1H), 7.91–7.84 (m, 1H), 7.82 (s, 2H), 7.60–7.41 (m, 2H), 4.22 (q, <i>J</i> = 7.1 Hz, 2H), 2.50 (s, 3H), 1.28 (t, <i>J</i> = 7.1 Hz, 3H); <sup>13</sup>C NMR (125 MHz, DMSO-<i>d</i><sub>6</sub>) δ 165.38, 164.88, 161.72, 141.87, 140.01, 130.06, 126.74, 124.61, 122.76, 118.77, 111.81, 111.42, 105.70, 59.40, 16.60, 14.31; LRMS (ESI) <i>m</i>/<i>z</i> 330 [M + H]<sup>+</sup>; HRMS (ESI) <i>m</i>/<i>z</i> calcd C<sub>16</sub>H<sub>16</sub>O<sub>3</sub>N<sub>3</sub>S [M + H]<sup>+</sup> 330.0907, found 330.0905.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01154">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_73228" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_73228" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.5b01154" class="ext-link">10.1021/acs.jmedchem.5b01154</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Additional figures and tables illustrating the inhibition data, assessment of docking programs, validation of docking procedure, and 2D interaction analysis of DC-S100, DC-S237, and DC-S239 (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf">jm5b01154_si_001.pdf (957.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_002.csv">jm5b01154_si_002.csv (16.11 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.5b01154" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_50121" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_50121" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wencong Lu</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, College of Sciences, Shanghai University, Shanghai 200444, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#2651454a5366554e5308434253084548"><span class="__cf_email__" data-cfemail="f780949b82b7849f82d9929382d99499">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Liu</span> - <span class="hlFld-Affiliation affiliation">Chinese Academy of Sciences Key Laboratory of Receptor Research,
Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#3c545055497c4f555151125d5f125f52"><span class="__cf_email__" data-cfemail="aec6c2c7dbeeddc7c3c380cfcd80cdc0">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Luo</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#f2919e879db2819b9f9fdc9391dc919c"><span class="__cf_email__" data-cfemail="b9dad5ccd6f9cad0d4d497d8da97dad7">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fanwang Meng</span> - <span class="hlFld-Affiliation affiliation">Department of Chemistry, College of Sciences, Shanghai University, Shanghai 200444, China</span>; 
    <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Sufang Cheng</span> - <span class="hlFld-Affiliation affiliation">Chinese Academy of Sciences Key Laboratory of Receptor Research,
Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Ding</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shien Liu</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yan Liu</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kongkai Zhu</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shijie Chen</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Junyan Lu</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yiqian Xie</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Linjuan Li</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Science and Technology, ShanghaiTech University, Shanghai 200031, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rongfeng Liu</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner
Co., Ltd., Zhangjiang Hi-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhe Shi</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner
Co., Ltd., Zhangjiang Hi-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu Zhou</span> - <span class="hlFld-Affiliation affiliation">Chinese Academy of Sciences Key Laboratory of Receptor Research,
Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yu-Chih Liu</span> - <span class="hlFld-Affiliation affiliation">Shanghai ChemPartner
Co., Ltd., Zhangjiang Hi-Tech Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mingyue Zheng</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">Drug Discovery
and Design Center, State Key Laboratory of Drug Research, Shanghai
Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School of Life Science and Technology, ShanghaiTech University, Shanghai 200031, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>F.M., S.C., and H.D. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d2553e3534-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i66">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was supported by the Hi-Tech Research and Development Program of China (Grant 2012AA020302 to C.L. and Grant 2012AA01A305 to H.J.), the Ministry of Science and Technology of China (Grant 2015CB910304 to H.L.), the National Natural Science Foundation of China (Grants 21210003 and 81230076 to H.J., Grant 91229204 to H.L., Grants 21472208 and 8143000629 to C.L.), the National Science and Technology Major Project “Key New Drug Creation and Manufacturing Program” (Grant 2014ZX09507002 to M.Z.) and China Postdoctoral Science Foundation (Grant 2014M551474 to S.C.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i67" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i67"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i68" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i68"> Abbreviations Used</h2><tr><td class="NLM_term">HIV</td><td class="NLM_def"><p class="first last">human immunodeficiency virus</p></td></tr><tr><td class="NLM_term">HCV</td><td class="NLM_def"><p class="first last">hepatitis C virus</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">IC<sub>50</sub></td><td class="NLM_def"><p class="first last">half-maximum inhibitory concentration</p></td></tr><tr><td class="NLM_term">HKMT</td><td class="NLM_def"><p class="first last">histone lysine methyltransferase</p></td></tr><tr><td class="NLM_term">SET7</td><td class="NLM_def"><p class="first last">SET domain containing lysine methyltransferase 7</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">PAINS</td><td class="NLM_def"><p class="first last">pan-assay interference compounds</p></td></tr><tr><td class="NLM_term">rmsd</td><td class="NLM_def"><p class="first last">root-mean square deviation</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr><tr><td class="NLM_term">XP</td><td class="NLM_def"><p class="first last">extra precision</p></td></tr><tr><td class="NLM_term">SP</td><td class="NLM_def"><p class="first last">standard precision</p></td></tr><tr><td class="NLM_term">ARTD1</td><td class="NLM_def"><p class="first last">ADP-ribosyltransferase diphtheria toxin-like 1</p></td></tr><tr><td class="NLM_term">COL2A1</td><td class="NLM_def"><p class="first last">collagen α-1(II) chain</p></td></tr><tr><td class="NLM_term">ERα</td><td class="NLM_def"><p class="first last">estrogen receptor α</p></td></tr><tr><td class="NLM_term">FoxO3</td><td class="NLM_def"><p class="first last">forkhead box protein O3</p></td></tr><tr><td class="NLM_term">pRb</td><td class="NLM_def"><p class="first last">retinoblastoma protein</p></td></tr><tr><td class="NLM_term">STAT3</td><td class="NLM_def"><p class="first last">signal transducer and activator of transcription 3</p></td></tr><tr><td class="NLM_term">SUV39H1</td><td class="NLM_def"><p class="first last">suppressor of variegation 3-9 homolog 1</p></td></tr><tr><td class="NLM_term">TAF10</td><td class="NLM_def"><p class="first last">transcription initiation factor TFIID subunit 10</p></td></tr><tr><td class="NLM_term">p65</td><td class="NLM_def"><p class="first last">transcription factor p65</p></td></tr><tr><td class="NLM_term">p53</td><td class="NLM_def"><p class="first last">cellular tumor antigen p53</p></td></tr><tr><td class="NLM_term">Pdx1</td><td class="NLM_def"><p class="first last">pancreatic duodenal homeobox 1</p></td></tr><tr><td class="NLM_term">HIFα</td><td class="NLM_def"><p class="first last">hypoxia-inducible factor</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA (cytosine-5)-methyltransferase 1</p></td></tr><tr><td class="NLM_term">DNMT3A/3L</td><td class="NLM_def"><p class="first last">DNA (cytosine-5)-methyltransferase 3A</p></td></tr><tr><td class="NLM_term">DOT1L</td><td class="NLM_def"><p class="first last">DOT1-like histone H3K79 methyltransferase</p></td></tr><tr><td class="NLM_term">EZH2</td><td class="NLM_def"><p class="first last">enhancer of zeste homolog 2</p></td></tr><tr><td class="NLM_term">NSD1</td><td class="NLM_def"><p class="first last">nuclear receptor binding SET domain protein 1</p></td></tr><tr><td class="NLM_term">SETD8</td><td class="NLM_def"><p class="first last">SET domain containing lysine methyltransferase 8</p></td></tr><tr><td class="NLM_term">G9a</td><td class="NLM_def"><p class="first last">euchromatic histone lysine <i>N</i>-methyltransferase 2 (EHMT2)</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81587" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81587" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 110 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-49&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lic-kz8Kwbhvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D49%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1815</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.bbcan.2010.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bbcan.2010.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20951770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer&doi=10.1016%2Fj.bbcan.2010.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine methylation and demethylation pathways in cancer</span></div><div class="casAuthors">Varier, Radhika A.; Timmers, H. T. Marc</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1815</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-89</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The genetic changes leading to the development of human cancer are accompanied by alterations in the structure and modification status of chromatin, which represent powerful regulatory mechanisms for gene expression and genome stability.  These epigenetic alterations have sparked interest into deciphering the regulatory pathways and function of post-translational modifications of histones during the initiation and progression of cancer.  In this review we describe and summarize the current knowledge of several histone lysine methyltransferase and demethylase pathways relevant to cancer.  Mechanistic insight into histone modifications will pave the way for the development and therapeutic application of "epidrugs" in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MyfXQs2YR7Vg90H21EOLACvtfcHk0lic-kz8Kwbhvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D&md5=3331b133c4f1b15beab600420bb859f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89%26doi%3D10.1016%2Fj.bbcan.2010.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasisht, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Structure and catalyticmechanism of the human histonemethyltransferase SET7/9</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">421</span><span class="NLM_x">, </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">656</span><span class="refDoi"> DOI: 10.1038/nature01378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnature01378" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=652-656&author=B.+Xiaoauthor=C.+Jingauthor=J.+R.+Wilsonauthor=P.+A.+Walkerauthor=N.+Vasishtauthor=G.+Kellyauthor=S.+Howellauthor=I.+A.+Taylorauthor=G.+M.+Blackburnauthor=S.+J.+Gamblin&title=Structure+and+catalyticmechanism+of+the+human+histonemethyltransferase+SET7%2F9&doi=10.1038%2Fnature01378"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature01378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01378%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJing%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DVasisht%26aufirst%3DN.%26aulast%3DKelly%26aufirst%3DG.%26aulast%3DHowell%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DI.%2BA.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructure%2520and%2520catalyticmechanism%2520of%2520the%2520human%2520histonemethyltransferase%2520SET7%252F9%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D652%26epage%3D656%26doi%3D10.1038%2Fnature01378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, B.</span><span> </span><span class="NLM_article-title">Crystal Structure and Functional Analysis of the Histone Methyltransferase SET7/9</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">115</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00964-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS0092-8674%2802%2900964-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2002&pages=105-115&author=J.+R.+Wilsonauthor=C.+Jingauthor=P.+A.+Walkerauthor=S.+R.+Martinauthor=S.+A.+Howellauthor=G.+M.+Blackburnauthor=S.+J.+Gamblinauthor=B.+Xiao&title=Crystal+Structure+and+Functional+Analysis+of+the+Histone+Methyltransferase+SET7%2F9&doi=10.1016%2FS0092-8674%2802%2900964-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900964-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900964-9%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DJing%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DHowell%26aufirst%3DS.%2BA.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DXiao%26aufirst%3DB.%26atitle%3DCrystal%2520Structure%2520and%2520Functional%2520Analysis%2520of%2520the%2520Histone%2520Methyltransferase%2520SET7%252F9%26jtitle%3DCell%26date%3D2002%26volume%3D111%26spage%3D105%26epage%3D115%26doi%3D10.1016%2FS0092-8674%2802%2900964-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Kassner, I.; Andersson, A.; Fey, M.; Tomas, M.; Ferrando-May, E.; Hottiger, M. O.</span><span> </span><span class="NLM_article-title">SET7/9-dependent methylation of ARTD1 at K508 stimulates poly-ADP-ribose formation after oxidative stress</span>.  <span class="citation_source-journal">Open Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span><span class="refDoi"> DOI: 10.1098/rsob.120173</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1098%2Frsob.120173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24088713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=I.+Kassner&author=A.+Andersson&author=M.+Fey&author=M.+Tomas&author=E.+Ferrando-May&author=M.+O.+Hottiger&title=SET7%2F9-dependent+methylation+of+ARTD1+at+K508+stimulates+poly-ADP-ribose+formation+after+oxidative+stress&doi=10.1098%2Frsob.120173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1098%2Frsob.120173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frsob.120173%26sid%3Dliteratum%253Aachs%26aulast%3DKassner%26aufirst%3DI.%26atitle%3DSET7%252F9-dependent%2520methylation%2520of%2520ARTD1%2520at%2520K508%2520stimulates%2520poly-ADP-ribose%2520formation%2520after%2520oxidative%2520stress%26jtitle%3DOpen%2520Biol.%26date%3D2013%26doi%3D10.1098%2Frsob.120173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Oppenheimer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haze, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liebergall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvir-Ginzberg, M.</span><span> </span><span class="NLM_article-title">Set7/9 Impacts COL2A1 Expression Through Binding and Repression of SirT1 Histone Deacetylation</span> <span class="citation_source-journal">J. Bone Miner. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">348</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span><span class="refDoi"> DOI: 10.1002/jbmr.2052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjbmr.2052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=23873758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ehsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=348-360&author=H.+Oppenheimerauthor=A.+Kumarauthor=H.+Meirauthor=I.+Schwartzauthor=A.+Ziniauthor=A.+Hazeauthor=L.+Kandelauthor=Y.+Mattanauthor=M.+Liebergallauthor=M.+Dvir-Ginzberg&title=Set7%2F9+Impacts+COL2A1+Expression+Through+Binding+and+Repression+of+SirT1+Histone+Deacetylation&doi=10.1002%2Fjbmr.2052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Set7/9 impacts COL2A1 expression through binding and repression of SirT1 histone deacetylation</span></div><div class="casAuthors">Oppenheimer, Hanna; Kumar, Ashok; Meir, Hadar; Schwartz, Israel; Zini, Avi; Haze, Amir; Kandel, Leonid; Mattan, Yoav; Liebergall, Meir; Dvir-Ginzberg, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bone and Mineral Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-360</span>CODEN:
                <span class="NLM_cas:coden">JBMREJ</span>;
        ISSN:<span class="NLM_cas:issn">0884-0431</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Type II collagen is a key cartilaginous extracellular protein required for normal endochondral development and cartilage homeostasis.  COL2A1 gene expression is pos. regulated by the NAD-dependent protein deacetylase Sirtuin 1 (SirT1), through its ability to bind chromatin regions of the COL2A1 promoter and enhancer.  Although SirT1/Sox9 binding on the enhancer site of COL2A1 was previously demonstrated, little is known about its functional role on the gene promoter site.  Here, we examd. the mechanism by which promoter-assocd. SirT1 governs COL2A1 expression.  Human chondrocytes were encapsulated in three-dimensional (3D) alginate beads where they exhibited upregulated COL2A1 mRNA expression and increased levels of SirT1 occupancy on the promoter and enhancer regions, when compared to monolayer controls.  Chromatin immunopptn. (ChIP) analyses of 3D cultures showed augmented levels of the DNA-binding transcription factor SP1, and the histone methyltransferase Set7/9, on the COL2A1 promoter site.  ChIP reChIP assays revealed that SirT1 and Set7/9 form a protein complex on the COL2A1 promoter region of 3D-cultured chondrocytes, which also demonstrated elevated trimethylated lysine 4 on histone 3 (3MeH3K4), a hallmark of Set7/9 methyltransferase activity.  Advanced passaging of chondrocytes yielded a decrease in 3MeH3K4 and Set7/9 levels on the COL2A1 promoter and reduced COL2A1 expression, suggesting that the SirT1/Set7/9 complex is preferentially formed on the COL2A1 promoter and required for gene activation.  Interestingly, despite SirT1 occupancy, its deacetylation targets (ie, H3K9/14 and H4K16) were found acetylated on the COL2A1 promoter of 3D-cultured chondrocytes.  A possible explanation for this phenotype is the enrichment of the histone acetyltransferases P300 and GCN5 on the COL2A1 promoter of 3D-cultured chondrocytes.  Our study indicates that Set7/9 prevents the histone deacetylase activity of SirT1, potentiating euchromatin formation on the promoter site of COL2A1 and resulting in morphol.-dependent COL2A1 gene transactivation. © 2014 American Society for Bone and Mineral Research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUcjHmY7WRrVg90H21EOLACvtfcHk0ljoeaK9vX6tBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ehsLs%253D&md5=06cc70f1955098565d188ae475ce9cea</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fjbmr.2052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjbmr.2052%26sid%3Dliteratum%253Aachs%26aulast%3DOppenheimer%26aufirst%3DH.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMeir%26aufirst%3DH.%26aulast%3DSchwartz%26aufirst%3DI.%26aulast%3DZini%26aufirst%3DA.%26aulast%3DHaze%26aufirst%3DA.%26aulast%3DKandel%26aufirst%3DL.%26aulast%3DMattan%26aufirst%3DY.%26aulast%3DLiebergall%26aufirst%3DM.%26aulast%3DDvir-Ginzberg%26aufirst%3DM.%26atitle%3DSet7%252F9%2520Impacts%2520COL2A1%2520Expression%2520Through%2520Binding%2520and%2520Repression%2520of%2520SirT1%2520Histone%2520Deacetylation%26jtitle%3DJ.%2520Bone%2520Miner.%2520Res.%26date%3D2014%26volume%3D29%26spage%3D348%26epage%3D360%26doi%3D10.1002%2Fjbmr.2052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Subramanian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor-Vazirani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vertino, P. M.</span><span> </span><span class="NLM_article-title">Regulation of Estrogen Receptor α by the SET7 Lysine Methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">336</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2008.03.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=336-347&author=K.+Subramanianauthor=D.+Jiaauthor=P.+Kapoor-Vaziraniauthor=D.+R.+Powellauthor=R.+E.+Collinsauthor=D.+Sharmaauthor=J.+Pengauthor=X.+Chengauthor=P.+M.+Vertino&title=Regulation+of+Estrogen+Receptor+%CE%B1+by+the+SET7+Lysine+Methyltransferase&doi=10.1016%2Fj.molcel.2008.03.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DK.%26aulast%3DJia%26aufirst%3DD.%26aulast%3DKapoor-Vazirani%26aufirst%3DP.%26aulast%3DPowell%26aufirst%3DD.%2BR.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DSharma%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26atitle%3DRegulation%2520of%2520Estrogen%2520Receptor%2520%25CE%25B1%2520by%2520the%2520SET7%2520Lysine%2520Methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D336%26epage%3D347%26doi%3D10.1016%2Fj.molcel.2008.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Calnan, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stowe, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goswami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunet, A.</span><span> </span><span class="NLM_article-title">Methylation by Set9 modulates FoxO3 stability and transcriptional activity</span> <span class="citation_source-journal">Aging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.18632%2Faging.100471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22820736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1anu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=462-479&author=D.+R.+Calnanauthor=A.+E.+Webbauthor=J.+L.+Whiteauthor=T.+R.+Stoweauthor=T.+Goswamiauthor=X.+Shiauthor=A.+Espejoauthor=M.+T.+Bedfordauthor=O.+Gozaniauthor=S.+P.+Gygiauthor=A.+Brunet&title=Methylation+by+Set9+modulates+FoxO3+stability+and+transcriptional+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation by Set9 modulates FoxO3 stability and transcriptional activity</span></div><div class="casAuthors">Calnan, Daniel R.; Webb, Ashley E.; White, Jamie L.; Stowe, Timothy R.; Goswami, Tapasree; Shi, Xiaobing; Espejo, Alexsandra; Bedford, Mark T.; Gozani, Or; Gygi, Steven P.; Brunet, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Aging</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">462-479</span>CODEN:
                <span class="NLM_cas:coden">AGINCN</span>;
        ISSN:<span class="NLM_cas:issn">1945-4589</span>.
    
            (<span class="NLM_cas:orgname">Impact Journals LLC</span>)
        </div><div class="casAbstract">The FoxO family of transcription factors plays an important role in longevity and tumor suppression by regulating the expression of a wide range of target genes.  FoxO3 has recently been found to be assocd. with extreme longevity in humans and to regulate the homeostasis of adult stem cell pools in mammals, which may contribute to longevity.  The activity of FoxO3 is controlled by a variety of post-translational modifications that have been proposed to form a 'code' affecting FoxO3 subcellular localization, DNA binding ability, protein-protein interactions and protein stability.  Lysine methylation is a crucial post-translational modification on histones that regulates chromatin accessibility and is a key part of the 'histone code'.  However, whether lysine methylation plays a role in modulating FoxO3 activity has never been examd.  Here we show that the methyltransferase Set9 directly methylates FoxO3 in vitro and in cells.  Using a combination of tandem mass spectrometry and methyl-specific antibodies, we find that Set9 methylates FoxO3 at a single residue, lysine 271, a site previously known to be deacetylated by Sirt1.  Methylation of FoxO3 by Set9 decreases FoxO3 protein stability, while moderately increasing FoxO3 transcriptional activity.  The modulation of FoxO3 stability and activity by methylation may be crit. for fine-tuning cellular responses to stress stimuli, which may in turn affect FoxO3's ability to promote tumor suppression and longevity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCIuFLql3YlLVg90H21EOLACvtfcHk0liZFmMYUO6Zdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1anu7zE&md5=a21545a042463e556b937198cbc51752</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Faging.100471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Faging.100471%26sid%3Dliteratum%253Aachs%26aulast%3DCalnan%26aufirst%3DD.%2BR.%26aulast%3DWebb%26aufirst%3DA.%2BE.%26aulast%3DWhite%26aufirst%3DJ.%2BL.%26aulast%3DStowe%26aufirst%3DT.%2BR.%26aulast%3DGoswami%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DEspejo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DBrunet%26aufirst%3DA.%26atitle%3DMethylation%2520by%2520Set9%2520modulates%2520FoxO3%2520stability%2520and%2520transcriptional%2520activity%26jtitle%3DAging%26date%3D2012%26volume%3D4%26spage%3D462%26epage%3D479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Cell cycle control by a methylation-phosphorylation switch</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">734</span><span class="refDoi"> DOI: 10.4161/cc.10.5.14958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4161%2Fcc.10.5.14958" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=733-734&author=S.+M.+Carrauthor=N.+B.+La+Thangue&title=Cell+cycle+control+by+a+methylation-phosphorylation+switch&doi=10.4161%2Fcc.10.5.14958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.5.14958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.5.14958%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DCell%2520cycle%2520control%2520by%2520a%2520methylation-phosphorylation%2520switch%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D733%26epage%3D734%26doi%3D10.4161%2Fcc.10.5.14958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Interplay between lysine methylation and Cdk phosphorylation in growth control by the retinoblastoma protein</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.1038/emboj.2010.311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Femboj.2010.311" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=317-327&author=S.+M.+Carrauthor=S.+Munroauthor=B.+Kesslerauthor=U.+Oppermannauthor=N.+B.+La+Thangue&title=Interplay+between+lysine+methylation+and+Cdk+phosphorylation+in+growth+control+by+the+retinoblastoma+protein&doi=10.1038%2Femboj.2010.311"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.311%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DInterplay%2520between%2520lysine%2520methylation%2520and%2520Cdk%2520phosphorylation%2520in%2520growth%2520control%2520by%2520the%2520retinoblastoma%2520protein%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D317%26epage%3D327%26doi%3D10.1038%2Femboj.2010.311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Huq, M. D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, N.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.-N.</span><span> </span><span class="NLM_article-title">Lysine trimethylation of retinoic acid receptor-alpha - A novel means to regulate receptor function</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">688</span><span class="refDoi"> DOI: 10.1074/mcp.M600223-MCP200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fmcp.M600223-MCP200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17205979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=677-688&author=M.+D.+M.+Huqauthor=N.-P.+Tsaiauthor=S.+A.+Khanauthor=L.-N.+Wei&title=Lysine+trimethylation+of+retinoic+acid+receptor-alpha+-+A+novel+means+to+regulate+receptor+function&doi=10.1074%2Fmcp.M600223-MCP200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine trimethylation of retinoic acid receptor-α: a novel means to regulate receptor function</span></div><div class="casAuthors">Huq, M. D. Mostaqul; Tsai, Nien-Pei; Khan, Shaukat Ali; Wei, Li-Na</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">677-688</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Retinoic acid receptors (RARs) belong to the nuclear receptor superfamily.  The mechanism of ligand-dependent activation of RARs is well known.  The effect of protein phosphorylation on the activity of RARs has also been demonstrated.  However, it is unclear whether other types of modifications exist and if so whether they can affect the activity of RARs.  In a mass spectrometric anal. of mouse RARα expressed in insect cells, we identified a trimethylation site on Lys347 in the ligand binding domain.  The modification site was verified in mammalian cells, and site-directed mutagenesis studies revealed the functionality of Lys347 methylation in vivo.  Constitutive neg. mutants, mimicking hypomethylated RARα, were prepd. by replacing methylated Lys347 with either alanine or glutamine.  A constitutive pos. mutant partially mimicking the hypermethylated RARα was generated by replacing the methylated lysine residue with phenylalanine, a bulky hydrophobic amino acid, to introduce a site-specific hydrophobicity similar to that contributed by lysine methylation.  Studies of these mutants revealed that trimethylation of Lys347 of RARα facilitated its interactions with cofactors p300/CREB-binding protein-assocd. factor and receptor-interacting protein 140 as well as its heterodimeric partner retinoid X receptor, suggesting that site-specific hydrophobicity at Lys347 enhanced mol. interaction of RARα with its modulators.  This study uncovers the first example of lysine trimethylation on a mammalian non-histone protein that has an important biol. consequence.  Our finding also provides the evidence for lysine methylation for the family of nuclear receptors for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpypHiOlk4R6bVg90H21EOLACvtfcHk0liZFmMYUO6Zdw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKgurw%253D&md5=204e2dcb913f021c0ae64245838c523d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M600223-MCP200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M600223-MCP200%26sid%3Dliteratum%253Aachs%26aulast%3DHuq%26aufirst%3DM.%2BD.%2BM.%26aulast%3DTsai%26aufirst%3DN.-P.%26aulast%3DKhan%26aufirst%3DS.%2BA.%26aulast%3DWei%26aufirst%3DL.-N.%26atitle%3DLysine%2520trimethylation%2520of%2520retinoic%2520acid%2520receptor-alpha%2520-%2520A%2520novel%2520means%2520to%2520regulate%2520receptor%2520function%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2007%26volume%3D6%26spage%3D677%26epage%3D688%26doi%3D10.1074%2Fmcp.M600223-MCP200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">504</span><span class="refDoi"> DOI: 10.1073/pnas.1016147107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes&doi=10.1073%2Fpnas.1016147107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lgWENAMJobAuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D504%26doi%3D10.1073%2Fpnas.1016147107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, F.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y.</span><span> </span><span class="NLM_article-title">Methylation of SUV39H1 by SET7/9 results in heterochromatin relaxation and genome instability</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">5516</span><span class="NLM_x">–</span> <span class="NLM_lpage">5521</span><span class="refDoi"> DOI: 10.1073/pnas.1216596110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1216596110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=5516-5521&author=D.+Wangauthor=J.+Zhouauthor=X.+Liuauthor=D.+Luauthor=C.+Shenauthor=Y.+Duauthor=F.-Z.+Weiauthor=B.+Songauthor=X.+Luauthor=Y.+Yu&title=Methylation+of+SUV39H1+by+SET7%2F9+results+in+heterochromatin+relaxation+and+genome+instability&doi=10.1073%2Fpnas.1216596110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1216596110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1216596110%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DF.-Z.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DY.%26atitle%3DMethylation%2520of%2520SUV39H1%2520by%2520SET7%252F9%2520results%2520in%2520heterochromatin%2520relaxation%2520and%2520genome%2520instability%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D5516%26epage%3D5521%26doi%3D10.1073%2Fpnas.1216596110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Albacker, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storer, N. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langdon, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dibiase, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langenau, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zon, L. I.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SUV39H1 suppresses embryonal rhabdomyosarcoma formation in zebrafish</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e64969</span><span class="refDoi"> DOI: 10.1371/journal.pone.0064969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1371%2Fjournal.pone.0064969" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e64969&author=C.+E.+Albackerauthor=N.+Y.+Storerauthor=E.+M.+Langdonauthor=A.+Dibiaseauthor=Y.+Zhouauthor=D.+M.+Langenauauthor=L.+I.+Zon&title=The+histone+methyltransferase+SUV39H1+suppresses+embryonal+rhabdomyosarcoma+formation+in+zebrafish&doi=10.1371%2Fjournal.pone.0064969"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0064969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0064969%26sid%3Dliteratum%253Aachs%26aulast%3DAlbacker%26aufirst%3DC.%2BE.%26aulast%3DStorer%26aufirst%3DN.%2BY.%26aulast%3DLangdon%26aufirst%3DE.%2BM.%26aulast%3DDibiase%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLangenau%26aufirst%3DD.%2BM.%26aulast%3DZon%26aufirst%3DL.%2BI.%26atitle%3DThe%2520histone%2520methyltransferase%2520SUV39H1%2520suppresses%2520embryonal%2520rhabdomyosarcoma%2520formation%2520in%2520zebrafish%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De64969%26doi%3D10.1371%2Fjournal.pone.0064969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Kouskouti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staub, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tora, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Gene-specific modulation of TAF10 function by SET9-mediated methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1016/S1097-2765(04)00182-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS1097-2765%2804%2900182-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=175-182&author=A.+Kouskoutiauthor=E.+Scheerauthor=A.+Staubauthor=L.+Toraauthor=I.+Talianidis&title=Gene-specific+modulation+of+TAF10+function+by+SET9-mediated+methylation&doi=10.1016%2FS1097-2765%2804%2900182-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2804%2900182-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252804%252900182-0%26sid%3Dliteratum%253Aachs%26aulast%3DKouskouti%26aufirst%3DA.%26aulast%3DScheer%26aufirst%3DE.%26aulast%3DStaub%26aufirst%3DA.%26aulast%3DTora%26aufirst%3DL.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DGene-specific%2520modulation%2520of%2520TAF10%2520function%2520by%2520SET9-mediated%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D14%26spage%3D175%26epage%3D182%26doi%3D10.1016%2FS1097-2765%2804%2900182-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Keating, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Osta, A.</span><span> </span><span class="NLM_article-title">Transcriptional regulation by the Set7 lysine methyltransferase</span> <span class="citation_source-journal">Epigenetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.4161/epi.24234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4161%2Fepi.24234" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=361-372&author=S.+T.+Keatingauthor=A.+El-Osta&title=Transcriptional+regulation+by+the+Set7+lysine+methyltransferase&doi=10.4161%2Fepi.24234"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Fepi.24234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.24234%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DS.%2BT.%26aulast%3DEl-Osta%26aufirst%3DA.%26atitle%3DTranscriptional%2520regulation%2520by%2520the%2520Set7%2520lysine%2520methyltransferase%26jtitle%3DEpigenetics%26date%3D2013%26volume%3D8%26spage%3D361%26epage%3D372%26doi%3D10.4161%2Fepi.24234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Maganti, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tersey, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosley, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozcan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pachaiyappan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Transcriptional activity of the islet beta cell factor Pdx1 is augmented by lysine methylation catalyzed by the methyltransferase Set7/9</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">9812</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1074/jbc.M114.616219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M114.616219" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9812-22&author=A.+V.+Magantiauthor=B.+Maierauthor=S.+A.+Terseyauthor=M.+L.+Sampleyauthor=A.+L.+Mosleyauthor=S.+Ozcanauthor=B.+Pachaiyappanauthor=P.+M.+Wosterauthor=C.+S.+Hunterauthor=R.+Steinauthor=R.+G.+Mirmira&title=Transcriptional+activity+of+the+islet+beta+cell+factor+Pdx1+is+augmented+by+lysine+methylation+catalyzed+by+the+methyltransferase+Set7%2F9&doi=10.1074%2Fjbc.M114.616219"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.616219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.616219%26sid%3Dliteratum%253Aachs%26aulast%3DMaganti%26aufirst%3DA.%2BV.%26aulast%3DMaier%26aufirst%3DB.%26aulast%3DTersey%26aufirst%3DS.%2BA.%26aulast%3DSampley%26aufirst%3DM.%2BL.%26aulast%3DMosley%26aufirst%3DA.%2BL.%26aulast%3DOzcan%26aufirst%3DS.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DHunter%26aufirst%3DC.%2BS.%26aulast%3DStein%26aufirst%3DR.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DTranscriptional%2520activity%2520of%2520the%2520islet%2520beta%2520cell%2520factor%2520Pdx1%2520is%2520augmented%2520by%2520lysine%2520methylation%2520catalyzed%2520by%2520the%2520methyltransferase%2520Set7%252F9%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D9812%26epage%3D22%26doi%3D10.1074%2Fjbc.M114.616219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, W.</span><span> </span><span class="NLM_article-title">Repression of hypoxia-inducible factor alpha signaling by Set7-mediated methylation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">5081</span><span class="NLM_x">–</span> <span class="NLM_lpage">98</span><span class="refDoi"> DOI: 10.1093/nar/gkv379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fnar%2Fgkv379" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=5081-98&author=X.+Liuauthor=Z.+Chenauthor=C.+Xuauthor=X.+Lengauthor=H.+Caoauthor=G.+Ouyangauthor=W.+Xiao&title=Repression+of+hypoxia-inducible+factor+alpha+signaling+by+Set7-mediated+methylation&doi=10.1093%2Fnar%2Fgkv379"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv379%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DLeng%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DOuyang%26aufirst%3DG.%26aulast%3DXiao%26aufirst%3DW.%26atitle%3DRepression%2520of%2520hypoxia-inducible%2520factor%2520alpha%2520signaling%2520by%2520Set7-mediated%2520methylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D5081%26epage%3D98%26doi%3D10.1093%2Fnar%2Fgkv379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Deering, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogihara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trace, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Methyltransferase Set7/9 Maintains Transcription and Euchromatin Structure at Islet-Enriched Genes</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.2337/db08-1150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2337%2Fdb08-1150" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2009&pages=185-193&author=T.+G.+Deeringauthor=T.+Ogiharaauthor=A.+P.+Traceauthor=B.+Maierauthor=R.+G.+Mirmira&title=Methyltransferase+Set7%2F9+Maintains+Transcription+and+Euchromatin+Structure+at+Islet-Enriched+Genes&doi=10.2337%2Fdb08-1150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2337%2Fdb08-1150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb08-1150%26sid%3Dliteratum%253Aachs%26aulast%3DDeering%26aufirst%3DT.%2BG.%26aulast%3DOgihara%26aufirst%3DT.%26aulast%3DTrace%26aufirst%3DA.%2BP.%26aulast%3DMaier%26aufirst%3DB.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DMethyltransferase%2520Set7%252F9%2520Maintains%2520Transcription%2520and%2520Euchromatin%2520Structure%2520at%2520Islet-Enriched%2520Genes%26jtitle%3DDiabetes%26date%3D2009%26volume%3D58%26spage%3D185%26epage%3D193%26doi%3D10.2337%2Fdb08-1150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Francis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garmey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Pdx-1 links histone H3-Lys-4 methylation to RNA polymerase II elongation during activation of insulin transcription</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">36244</span><span class="NLM_x">–</span> <span class="NLM_lpage">36253</span><span class="refDoi"> DOI: 10.1074/jbc.M505741200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M505741200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=36244-36253&author=J.+Francisauthor=S.+K.+Chakrabartiauthor=J.+C.+Garmeyauthor=R.+G.+Mirmira&title=Pdx-1+links+histone+H3-Lys-4+methylation+to+RNA+polymerase+II+elongation+during+activation+of+insulin+transcription&doi=10.1074%2Fjbc.M505741200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M505741200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M505741200%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DChakrabarti%26aufirst%3DS.%2BK.%26aulast%3DGarmey%26aufirst%3DJ.%2BC.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DPdx-1%2520links%2520histone%2520H3-Lys-4%2520methylation%2520to%2520RNA%2520polymerase%2520II%2520elongation%2520during%2520activation%2520of%2520insulin%2520transcription%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D36244%26epage%3D36253%26doi%3D10.1074%2Fjbc.M505741200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Diversity within the pRb pathway: is there a code of conduct?</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">4343</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1038/onc.2011.603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fonc.2011.603" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=4343-52&author=S.+Munroauthor=S.+M.+Carrauthor=N.+B.+La+Thangue&title=Diversity+within+the+pRb+pathway%3A+is+there+a+code+of+conduct%3F&doi=10.1038%2Fonc.2011.603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.603%26sid%3Dliteratum%253Aachs%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DDiversity%2520within%2520the%2520pRb%2520pathway%253A%2520is%2520there%2520a%2520code%2520of%2520conduct%253F%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D4343%26epage%3D52%26doi%3D10.1038%2Fonc.2011.603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Tao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neppl, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, R.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.-Z.</span><span> </span><span class="NLM_article-title">The histone methyltransferase Set7/9 promotes myoblast differentiation and myofibril assembly</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">194</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="NLM_x">–</span> <span class="NLM_lpage">565</span><span class="refDoi"> DOI: 10.1083/jcb.201010090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1083%2Fjcb.201010090" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2011&pages=551-565&author=Y.+Taoauthor=R.+L.+Nepplauthor=Z.-P.+Huangauthor=J.+Chenauthor=R.-H.+Tangauthor=R.+Caoauthor=Y.+Zhangauthor=S.-W.+Jinauthor=D.-Z.+Wang&title=The+histone+methyltransferase+Set7%2F9+promotes+myoblast+differentiation+and+myofibril+assembly&doi=10.1083%2Fjcb.201010090"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201010090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201010090%26sid%3Dliteratum%253Aachs%26aulast%3DTao%26aufirst%3DY.%26aulast%3DNeppl%26aufirst%3DR.%2BL.%26aulast%3DHuang%26aufirst%3DZ.-P.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DR.-H.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DS.-W.%26aulast%3DWang%26aufirst%3DD.-Z.%26atitle%3DThe%2520histone%2520methyltransferase%2520Set7%252F9%2520promotes%2520myoblast%2520differentiation%2520and%2520myofibril%2520assembly%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2011%26volume%3D194%26spage%3D551%26epage%3D565%26doi%3D10.1083%2Fjcb.201010090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Esteve, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feehery, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samaranayake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horwitz, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S.</span><span> </span><span class="NLM_article-title">Regulation of DNMT1 stability through SET7-mediated lysine methylation in mammalian cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">5076</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1073/pnas.0810362106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.0810362106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=5076-81&author=P.+O.+Esteveauthor=H.+G.+Chinauthor=J.+Bennerauthor=G.+R.+Feeheryauthor=M.+Samaranayakeauthor=G.+A.+Horwitzauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Regulation+of+DNMT1+stability+through+SET7-mediated+lysine+methylation+in+mammalian+cells&doi=10.1073%2Fpnas.0810362106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810362106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810362106%26sid%3Dliteratum%253Aachs%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DBenner%26aufirst%3DJ.%26aulast%3DFeehery%26aufirst%3DG.%2BR.%26aulast%3DSamaranayake%26aufirst%3DM.%26aulast%3DHorwitz%26aufirst%3DG.%2BA.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DRegulation%2520of%2520DNMT1%2520stability%2520through%2520SET7-mediated%2520lysine%2520methylation%2520in%2520mammalian%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D5076%26epage%3D81%26doi%3D10.1073%2Fpnas.0810362106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lh1Ypk_FB_lrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Ea, C.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baltimore, D.</span><span> </span><span class="NLM_article-title">Regulation of NF-kappa B activity through lysine monomethylation of p65</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">18972</span><span class="NLM_x">–</span> <span class="NLM_lpage">18977</span><span class="refDoi"> DOI: 10.1073/pnas.0910439106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.0910439106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18972-18977&author=C.-K.+Eaauthor=D.+Baltimore&title=Regulation+of+NF-kappa+B+activity+through+lysine+monomethylation+of+p65&doi=10.1073%2Fpnas.0910439106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910439106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910439106%26sid%3Dliteratum%253Aachs%26aulast%3DEa%26aufirst%3DC.-K.%26aulast%3DBaltimore%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520NF-kappa%2520B%2520activity%2520through%2520lysine%2520monomethylation%2520of%2520p65%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D18972%26epage%3D18977%26doi%3D10.1073%2Fpnas.0910439106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanmugam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, R.</span><span> </span><span class="NLM_article-title">Role of the histone H3 lysine 4 methyltransferase, SET7/9, in the regulation of NF-kappaB-dependent inflammatory genes. Relevance to diabetes and inflammation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">26771</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1074/jbc.M802800200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M802800200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=26771-81&author=Y.+Liauthor=M.+A.+Reddyauthor=F.+Miaoauthor=N.+Shanmugamauthor=J.+K.+Yeeauthor=D.+Hawkinsauthor=B.+Renauthor=R.+Natarajan&title=Role+of+the+histone+H3+lysine+4+methyltransferase%2C+SET7%2F9%2C+in+the+regulation+of+NF-kappaB-dependent+inflammatory+genes.+Relevance+to+diabetes+and+inflammation&doi=10.1074%2Fjbc.M802800200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802800200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802800200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DM.%2BA.%26aulast%3DMiao%26aufirst%3DF.%26aulast%3DShanmugam%26aufirst%3DN.%26aulast%3DYee%26aufirst%3DJ.%2BK.%26aulast%3DHawkins%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DNatarajan%26aufirst%3DR.%26atitle%3DRole%2520of%2520the%2520histone%2520H3%2520lysine%25204%2520methyltransferase%252C%2520SET7%252F9%252C%2520in%2520the%2520regulation%2520of%2520NF-kappaB-dependent%2520inflammatory%2520genes.%2520Relevance%2520to%2520diabetes%2520and%2520inflammation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D26771%26epage%3D81%26doi%3D10.1074%2Fjbc.M802800200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Paneni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battista, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capretti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiandotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scavone, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanza, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volpe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscher, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosentino, F.</span><span> </span><span class="NLM_article-title">Adverse Epigenetic Signatures by Histone Methyltransferase Set7 Contribute to Vascular Dysfunction in Patients With Type 2 Diabetes Mellitus</span> <span class="citation_source-journal">Circ.: Cardiovasc. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">U304</span><span class="refDoi"> DOI: 10.1161/CIRCGENETICS.114.000671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1161%2FCIRCGENETICS.114.000671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25472959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFaru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=150-U304&author=F.+Paneniauthor=S.+Costantinoauthor=R.+Battistaauthor=L.+Castelloauthor=G.+Caprettiauthor=S.+Chiandottoauthor=G.+Scavoneauthor=A.+Villanoauthor=D.+Pitoccoauthor=G.+Lanzaauthor=M.+Volpeauthor=T.+F.+Luscherauthor=F.+Cosentino&title=Adverse+Epigenetic+Signatures+by+Histone+Methyltransferase+Set7+Contribute+to+Vascular+Dysfunction+in+Patients+With+Type+2+Diabetes+Mellitus&doi=10.1161%2FCIRCGENETICS.114.000671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse Epigenetic Signatures by Histone Methyltransferase Set7 Contribute to Vascular Dysfunction in Patients With Type 2 Diabetes Mellitus</span></div><div class="casAuthors">Paneni, Francesco; Costantino, Sarah; Battista, Rodolfo; Castello, Lorenzo; Capretti, Giuliana; Chiandotto, Sergio; Scavone, Giuseppe; Villano, Angelo; Pitocco, Dario; Lanza, Gaetano; Volpe, Massimo; Luescher, Thomas F.; Cosentino, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Circulation: Cardiovascular Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-158</span>CODEN:
                <span class="NLM_cas:coden">CCGIAO</span>;
        ISSN:<span class="NLM_cas:issn">1942-3268</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">BACKGROUND-: Cellular studies showed that histone methyltransferase Set7 mediates high glucose-induced inflammation via epigenetic regulation of the transcription factor NF-kB.  However, the link between Set7 and vascular dysfunction in patients with diabetes mellitus remains unknown.  This study was designed to investigate whether Set7 contributes to vascular dysfunction in patients with type 2 diabetes mellitus (T2DM).  METHODS AND RESULTS-: Set7-driven epigenetic changes on NF-kB p65 promoter and expression of NF-kB-dependent genes, cyclooxygenase 2 and inducible endothelial nitric oxide synthase, were assessed in peripheral blood mononuclear cells isolated from 68 subjects (44 patients with T2DM and 24 age-matched controls).  Brachial artery flow-mediated dilation, 24-h urinary levels of 8-isoprostaglandin F2α, and plasma adhesion mols., intercellular cell adhesion mol.-1 and monocyte chemoattractant protein-1, were also detd.  Expts. in human aortic endothelial cells exposed to high glucose were performed to elucidate the mechanisms of Set7-driven inflammation and oxidative stress.  Set7 expression increased in peripheral blood mononuclear cells from patients with T2DM when compared with controls.  Patients with T2DM showed Set7-dependent monomethylation of lysine 4 of histone 3 on NF-kB p65 promoter.  This epigenetic signature was assocd. with upregulation of NF-kB, subsequent transcription of oxidant/inflammatory genes, and increased plasma levels of intercellular cell adhesion mol.-1 and monocyte chemoattractant protein-1.  Interestingly, we found that Set7 expression significantly correlated with oxidative marker 8-isoprostaglandin F2α (r=0.38; P=0.01) and flow-mediated dilation (r=-0.34; P=0.04).  In human aortic endothelial cells, silencing of Set7 prevented monomethylation of lysine 4 of histone 3 and abolished NF-kB-dependent oxidant and inflammatory signaling.  CONCLUSIONS-: Set7-induced epigenetic changes contribute to vascular dysfunction in patients with T2DM.  Targeting this chromatin-modifying enzyme may represent a novel therapeutic approach to prevent atherosclerotic vascular disease in this setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYhiQh5PrZfbVg90H21EOLACvtfcHk0ljSIWMoJWtOzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFaru7w%253D&md5=4cc7181c59c556d0c86c5cc62298af5e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1161%2FCIRCGENETICS.114.000671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCGENETICS.114.000671%26sid%3Dliteratum%253Aachs%26aulast%3DPaneni%26aufirst%3DF.%26aulast%3DCostantino%26aufirst%3DS.%26aulast%3DBattista%26aufirst%3DR.%26aulast%3DCastello%26aufirst%3DL.%26aulast%3DCapretti%26aufirst%3DG.%26aulast%3DChiandotto%26aufirst%3DS.%26aulast%3DScavone%26aufirst%3DG.%26aulast%3DVillano%26aufirst%3DA.%26aulast%3DPitocco%26aufirst%3DD.%26aulast%3DLanza%26aufirst%3DG.%26aulast%3DVolpe%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DT.%2BF.%26aulast%3DCosentino%26aufirst%3DF.%26atitle%3DAdverse%2520Epigenetic%2520Signatures%2520by%2520Histone%2520Methyltransferase%2520Set7%2520Contribute%2520to%2520Vascular%2520Dysfunction%2520in%2520Patients%2520With%2520Type%25202%2520Diabetes%2520Mellitus%26jtitle%3DCirc.%253A%2520Cardiovasc.%2520Genet.%26date%3D2015%26volume%3D8%26spage%3D150%26epage%3DU304%26doi%3D10.1161%2FCIRCGENETICS.114.000671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Q.</span><span> </span><span class="NLM_article-title">Abnormal epigenetic modifications in peripheral blood mononuclear cells from patients with alopecia areata</span> <span class="citation_source-journal">Br. J. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">166</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1111/j.1365-2133.2011.10646.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1111%2Fj.1365-2133.2011.10646.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2012&pages=266-273&author=M.+Zhaoauthor=G.+Liangauthor=X.+Wuauthor=S.+Wangauthor=P.+Zhangauthor=Y.+Suauthor=H.+Yinauthor=Y.+Tanauthor=J.+Zhangauthor=Q.+Lu&title=Abnormal+epigenetic+modifications+in+peripheral+blood+mononuclear+cells+from+patients+with+alopecia+areata&doi=10.1111%2Fj.1365-2133.2011.10646.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Abnormal epigenetic modifications in peripheral blood mononuclear cells from patients with alopecia areata</span></div><div class="casAuthors">Zhao, M.; Liang, G.; Wu, X.; Wang, S.; Zhang, P.; Su, Y.; Yin, H.; Tan, Y.; Zhang, J.; Lu, Q.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Dermatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">BJDEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0007-0963</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background: Alopecia areata (AA) is a hair loss disease caused by T-cell-mediated autoimmune reactions against anagen-stage hair follicles.  Although the exact etiol. is poorly understood, there is evidence to suggest that both genetic and environmental factors are involved in AA pathogenesis.  Objectives: To analyze DNA methylation and histone modification patterns in peripheral blood mononuclear cells (PBMCs) of patients with AA.  Methods: PBMC samples were obtained from 25 patients with AA and 20 healthy controls.  Global DNA methylcytosine levels, as well as histone acetylation and methylation levels, were measured by ELISA.  MRNA expression levels were detd. using real-time quant. reverse transcription-polymerase chain reaction.  Results: Genomic DNA methylation in PBMCs of patients with AA was increased relative to controls.  DNMT1, MBD1 and MBD4 expression levels were significantly higher in AA PBMCs than in controls, and DNMT1 transcription levels pos. correlated with global DNA methylation levels in patient samples.  Histone H3 acetylation was significantly increased and histone H3 lysine 4 methylation was significantly decreased in patient PBMCs compared with healthy controls.  Histone H3 acetylation levels were pos. correlated with AA disease severity, and with RANTES (CCL5) mRNA expression in PBMCs of patients with AA.  These changes were accompanied by increased p300 (EP300), histone deacetylase 1 (HDAC1), myeloid/lymphoid or mixed lineage leukemia (MLL), SET7/9 (SETD7), G9A (EHMT2), JMJD2C (KDM4C) and JARID1A (KDM5A) expression, as well as reduced HDAC2, HDAC7, LSD1 (KDM1A), JMJD2A (KDM4A) and JMJD2B (KDM4B) expression.  Conclusions: DNA methylation and histone modification status are altered in PBMCs of patients with AA, possibly due to the deregulation of epigenetic regulatory genes.  These changes may contribute to the activation of pathol. immune responses in AA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IswsWFb3FrVg90H21EOLACvtfcHk0ljSIWMoJWtOzQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVSjtr8%253D&md5=76737cd2408f75a2b3e315c93a0628f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2133.2011.10646.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2133.2011.10646.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DQ.%26atitle%3DAbnormal%2520epigenetic%2520modifications%2520in%2520peripheral%2520blood%2520mononuclear%2520cells%2520from%2520patients%2520with%2520alopecia%2520areata%26jtitle%3DBr.%2520J.%2520Dermatol.%26date%3D2012%26volume%3D166%26spage%3D266%26epage%3D273%26doi%3D10.1111%2Fj.1365-2133.2011.10646.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Adachi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furihata, M.</span><span> </span><span class="NLM_article-title">T-Cadherin Modulates Tumor-Associated Molecules in Gallbladder Cancer Cells</span> <span class="citation_source-journal">Cancer Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.3109/07357900903124472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.3109%2F07357900903124472" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=120-126&author=Y.+Adachiauthor=T.+Takeuchiauthor=T.+Nagayamaauthor=M.+Furihata&title=T-Cadherin+Modulates+Tumor-Associated+Molecules+in+Gallbladder+Cancer+Cells&doi=10.3109%2F07357900903124472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3109%2F07357900903124472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F07357900903124472%26sid%3Dliteratum%253Aachs%26aulast%3DAdachi%26aufirst%3DY.%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DNagayama%26aufirst%3DT.%26aulast%3DFurihata%26aufirst%3DM.%26atitle%3DT-Cadherin%2520Modulates%2520Tumor-Associated%2520Molecules%2520in%2520Gallbladder%2520Cancer%2520Cells%26jtitle%3DCancer%2520Invest.%26date%3D2009%26volume%3D28%26spage%3D120%26epage%3D126%26doi%3D10.3109%2F07357900903124472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Pagans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauder, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakane, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormeyer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">The Cellular Lysine Methyltransferase Set7/9-KMT7 Binds HIV-1 TAR RNA, Monomethylates the Viral Transactivator Tat, and Enhances HIV Transcription</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span><span class="refDoi"> DOI: 10.1016/j.chom.2010.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.chom.2010.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20227666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=234-244&author=S.+Pagansauthor=S.+E.+Kauderauthor=K.+Kaehlckeauthor=N.+Sakaneauthor=S.+Schroederauthor=W.+Dormeyerauthor=R.+C.+Trievelauthor=E.+Verdinauthor=M.+Schnolzerauthor=M.+Ott&title=The+Cellular+Lysine+Methyltransferase+Set7%2F9-KMT7+Binds+HIV-1+TAR+RNA%2C+Monomethylates+the+Viral+Transactivator+Tat%2C+and+Enhances+HIV+Transcription&doi=10.1016%2Fj.chom.2010.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular lysine methyltransferase Set7/9-KMT7 binds HIV-1 TAR RNA, monomethylates the viral transactivator Tat, and enhances HIV transcription</span></div><div class="casAuthors">Pagans, Sara; Kauder, Steven E.; Kaehlcke, Katrin; Sakane, Naoki; Schroeder, Sebastian; Dormeyer, Wilma; Trievel, Raymond C.; Verdin, Eric; Schnolzer, Martina; Ott, Melanie</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Tat protein of HIV-1 plays an essential role in HIV gene expression by promoting efficient elongation of viral transcripts.  Post-translational modifications of Tat fine-tune interactions of Tat with cellular cofactors and TAR RNA, a stem-loop structure at the 5' ends of viral transcripts.  Here, we identify the lysine methyltransferase Set7/9 (KMT7) as a coactivator of HIV transcription.  Set7/9-KMT7 assocs. with the HIV promoter in vivo and monomethylates lysine 51, a highly conserved residue located in the RNA-binding domain of Tat.  Knockdown of Set7/9-KMT7 suppresses Tat transactivation of the HIV promoter, but does not affect the transcriptional activity of methylation-deficient Tat (K51A).  Set7/9-KMT7 binds TAR RNA by itself and in complex with Tat and the pos. transcription elongation factor P-TEFb.  Our findings uncover a pos. role for Set7/9-KMT7 and Tat methylation during early steps of the Tat transactivation cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8AqerZRYELVg90H21EOLACvtfcHk0li9a0PYxpETXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWqu70%253D&md5=1d31622c99a7b4dc43b0908546cce975</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2010.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2010.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DPagans%26aufirst%3DS.%26aulast%3DKauder%26aufirst%3DS.%2BE.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DSakane%26aufirst%3DN.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DDormeyer%26aufirst%3DW.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DThe%2520Cellular%2520Lysine%2520Methyltransferase%2520Set7%252F9-KMT7%2520Binds%2520HIV-1%2520TAR%2520RNA%252C%2520Monomethylates%2520the%2520Viral%2520Transactivator%2520Tat%252C%2520and%2520Enhances%2520HIV%2520Transcription%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2010%26volume%3D7%26spage%3D234%26epage%3D244%26doi%3D10.1016%2Fj.chom.2010.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Han, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">She, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broering, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span> </span><span class="NLM_article-title">Set7 facilitates hepatitis C virus replication via enzymatic activity-dependent attenuation of the IFN-related pathway</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">194</span><span class="NLM_x">, </span> <span class="NLM_fpage">2757</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span><span class="refDoi"> DOI: 10.4049/jimmunol.1400583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4049%2Fjimmunol.1400583" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=2757-68&author=T.+Hanauthor=Y.+Wanauthor=J.+Wangauthor=P.+Zhaoauthor=Y.+Yuanauthor=L.+Wangauthor=Y.+Sheauthor=R.+Broeringauthor=M.+Luauthor=L.+Yeauthor=Y.+Zhu&title=Set7+facilitates+hepatitis+C+virus+replication+via+enzymatic+activity-dependent+attenuation+of+the+IFN-related+pathway&doi=10.4049%2Fjimmunol.1400583"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400583%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShe%26aufirst%3DY.%26aulast%3DBroering%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DSet7%2520facilitates%2520hepatitis%2520C%2520virus%2520replication%2520via%2520enzymatic%2520activity-dependent%2520attenuation%2520of%2520the%2520IFN-related%2520pathway%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D2757%26epage%3D68%26doi%3D10.4049%2Fjimmunol.1400583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Toledo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, G. M.</span><span> </span><span class="NLM_article-title">Regulating the p53 pathway: in vitro hypotheses, in vivo veritas</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1038/nrc2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrc2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17128209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Clsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=909-23&author=F.+Toledoauthor=G.+M.+Wahl&title=Regulating+the+p53+pathway%3A+in+vitro+hypotheses%2C+in+vivo+veritas&doi=10.1038%2Fnrc2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Regulating the p53 pathway: in vitro hypotheses, in vivo veritas</span></div><div class="casAuthors">Toledo, Franck; Wahl, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-923</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mutations in TP53, the gene that encodes the tumor suppressor p53, are found in 50% of human cancers, and increased levels of its neg. regulators MDM2 and MDM4 (also known as MDMX) down-regulate p53 function in many of the rest.  Understanding p53 regulation remains a crucial goal to design broadly applicable anticancer strategies based on this pathway.  This review of in vitro studies, human tumor data and recent mouse models shows that p53 post-translational modifications have modulatory roles, and MDM2 and MDM4 have more profound roles for regulating p53.  Importantly, MDM4 emerges as an independent target for drug development, as its inactivation is crucial for full p53 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqN3wMkB-VB7Vg90H21EOLACvtfcHk0li9a0PYxpETXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Clsr3E&md5=0f87a2342dd6dcd4ab3de2a0245eabd6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrc2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2012%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%26aufirst%3DF.%26aulast%3DWahl%26aufirst%3DG.%2BM.%26atitle%3DRegulating%2520the%2520p53%2520pathway%253A%2520in%2520vitro%2520hypotheses%252C%2520in%2520vivo%2520veritas%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D909%26epage%3D23%26doi%3D10.1038%2Fnrc2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanov, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinney, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prives, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlev, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Regulation of p53 activity through lysine methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span><span class="refDoi"> DOI: 10.1038/nature03117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnature03117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=15525938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=353-360&author=S.+Chuikovauthor=J.+K.+Kurashauthor=J.+R.+Wilsonauthor=B.+Xiaoauthor=N.+Justinauthor=G.+S.+Ivanovauthor=K.+McKinneyauthor=P.+Tempstauthor=C.+Privesauthor=S.+J.+Gamblinauthor=N.+A.+Barlevauthor=D.+Reinberg&title=Regulation+of+p53+activity+through+lysine+methylation&doi=10.1038%2Fnature03117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of p53 activity through lysine methylation</span></div><div class="casAuthors">Chuikov, Sergei; Kurash, Julia K.; Wilson, Jonathan R.; Xiao, Bing; Justin, Neil; Ivanov, Gleb S.; McKinney, Kristine; Tempst, Paul; Prives, Carol; Gamblin, Steven J.; Barlev, Nickolai A.; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53 is a tumor suppressor that regulates the cellular response to genotoxic stresses. p53 is a short-lived protein and its activity is regulated mostly by stabilization via different post-translational modifications.  Here we report a novel mechanism of p53 regulation through lysine methylation by Set9 methyltransferase.  Set9 specifically methylates p53 at one residue within the carboxyl-terminus regulatory region.  Methylated p53 is restricted to the nucleus and the modification pos. affects its stability.  Set9 regulates the expression of p53 target genes in a manner dependent on the p53-methylation site.  The crystal structure of a ternary complex of Set9 with a p53 peptide and the cofactor product S-adenosyl-L-homocysteine (AdoHcy) provides the mol. basis for recognition of p53 by this lysine methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQEN46GC7xzLVg90H21EOLACvtfcHk0liILzyXwl7Hug"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D&md5=eff3a7ec7f6adbf1fc43016325f074d0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03117%26sid%3Dliteratum%253Aachs%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DG.%2BS.%26aulast%3DMcKinney%26aufirst%3DK.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DPrives%26aufirst%3DC.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DBarlev%26aufirst%3DN.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520p53%2520activity%2520through%2520lysine%2520methylation%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D353%26epage%3D360%26doi%3D10.1038%2Fnature03117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marston, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granda, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span> </span><span class="NLM_article-title">Methylation of p53 by Set7/9 mediates p53 acetylation and activity in vivo</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2007.12.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=392-400&author=J.+K.+Kurashauthor=H.+Leiauthor=Q.+Shenauthor=W.+L.+Marstonauthor=B.+W.+Grandaauthor=H.+Fanauthor=D.+Wallauthor=E.+Liauthor=F.+Gaudet&title=Methylation+of+p53+by+Set7%2F9+mediates+p53+acetylation+and+activity+in+vivo&doi=10.1016%2Fj.molcel.2007.12.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.025%26sid%3Dliteratum%253Aachs%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DMarston%26aufirst%3DW.%2BL.%26aulast%3DGranda%26aufirst%3DB.%2BW.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DWall%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DGaudet%26aufirst%3DF.%26atitle%3DMethylation%2520of%2520p53%2520by%2520Set7%252F9%2520mediates%2520p53%2520acetylation%2520and%2520activity%2520in%2520vivo%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26spage%3D392%26epage%3D400%26doi%3D10.1016%2Fj.molcel.2007.12.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.-G.</span><span> </span><span class="NLM_article-title">Methyltransferase Set7/9 regulates p53 activity by interacting with Sirtuin 1 (SIRT1)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1925</span><span class="NLM_x">–</span> <span class="NLM_lpage">1930</span><span class="refDoi"> DOI: 10.1073/pnas.1019619108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1019619108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1925-1930&author=X.+Liuauthor=D.+Wangauthor=Y.+Zhaoauthor=B.+Tuauthor=Z.+Zhengauthor=L.+Wangauthor=H.+Wangauthor=W.+Guauthor=R.+G.+Roederauthor=W.-G.+Zhu&title=Methyltransferase+Set7%2F9+regulates+p53+activity+by+interacting+with+Sirtuin+1+%28SIRT1%29&doi=10.1073%2Fpnas.1019619108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1019619108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1019619108%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DZhu%26aufirst%3DW.-G.%26atitle%3DMethyltransferase%2520Set7%252F9%2520regulates%2520p53%2520activity%2520by%2520interacting%2520with%2520Sirtuin%25201%2520%2528SIRT1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D1925%26epage%3D1930%26doi%3D10.1073%2Fpnas.1019619108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">Regulation of p53 function by lysine methylation</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.2217/epi.11.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2217%2Fepi.11.21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=361-9&author=L.+E.+Westauthor=O.+Gozani&title=Regulation+of+p53+function+by+lysine+methylation&doi=10.2217%2Fepi.11.21"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2217%2Fepi.11.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.11.21%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DRegulation%2520of%2520p53%2520function%2520by%2520lysine%2520methylation%26jtitle%3DEpigenomics%26date%3D2011%26volume%3D3%26spage%3D361%26epage%3D9%26doi%3D10.2217%2Fepi.11.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span> </span><span class="NLM_article-title">p53-Dependent Transcription and Tumor Suppression Are Not Affected in Set7/9-Deficient Mice</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span><span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2011.08.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=673-680&author=B.+Lehnertzauthor=J.+C.+Rogalskiauthor=F.+M.+Schulzeauthor=L.+Yiauthor=S.+Linauthor=J.+Kastauthor=F.+M.+V.+Rossi&title=p53-Dependent+Transcription+and+Tumor+Suppression+Are+Not+Affected+in+Set7%2F9-Deficient+Mice&doi=10.1016%2Fj.molcel.2011.08.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DRogalski%26aufirst%3DJ.%2BC.%26aulast%3DSchulze%26aufirst%3DF.%2BM.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DKast%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26atitle%3Dp53-Dependent%2520Transcription%2520and%2520Tumor%2520Suppression%2520Are%2520Not%2520Affected%2520in%2520Set7%252F9-Deficient%2520Mice%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D43%26spage%3D673%26epage%3D680%26doi%3D10.1016%2Fj.molcel.2011.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Campaner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spreafico, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgold, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosato, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amati, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, G.</span><span> </span><span class="NLM_article-title">The Methyltransferase Set7/9 (Setd7) Is Dispensable for the p53-Mediated DNA Damage Response In Vivo</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">681</span><span class="NLM_x">–</span> <span class="NLM_lpage">688</span><span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2011.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21855806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVGku7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=681-688&author=S.+Campanerauthor=F.+Spreaficoauthor=T.+Burgoldauthor=M.+Doniauthor=U.+Rosatoauthor=B.+Amatiauthor=G.+Testa&title=The+Methyltransferase+Set7%2F9+%28Setd7%29+Is+Dispensable+for+the+p53-Mediated+DNA+Damage+Response+In+Vivo&doi=10.1016%2Fj.molcel.2011.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Methyltransferase Set7/9 (Setd7) Is Dispensable for the p53-Mediated DNA Damage Response In Vivo</span></div><div class="casAuthors">Campaner, Stefano; Spreafico, Fabio; Burgold, Thomas; Doni, Mirko; Rosato, Umberto; Amati, Bruno; Testa, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-688</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: p53 is the central regulator of cell fate following genotoxic stress and oncogene activation.  Its activity is controlled by several posttranslational modifications.  Originally defined as a crit. layer of p53 regulation in human cell lines, p53 lysine methylation by Set7/9 (also called Setd7) was proposed to fulfill a similar function in vivo in the mouse, promoting p53 acetylation, stabilization, and activation upon DNA damage ().  We tested the physiol. relevance of this circuit in an independent Set7/9 knockout mouse strain.  Deletion of Set7/9 had no effect on p53-dependent cell-cycle arrest or apoptosis following sublethal or lethal DNA damage induced by radiation or genotoxic agents.  Set7/9 was also dispensable for p53 acetylation following irradn. c-myc oncogene-induced apoptosis was also independent of Set7/9, and anal. of p53 target genes showed that Set7/9 is not required for the p53-dependent gene expression program.  Our data indicate that Set7/9 is dispensable for p53 function in the mouse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFipBxRO0rrVg90H21EOLACvtfcHk0lj-3xj4EGsQFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVGku7zO&md5=6ee511977f70e48a9fb509ad3e871701</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DCampaner%26aufirst%3DS.%26aulast%3DSpreafico%26aufirst%3DF.%26aulast%3DBurgold%26aufirst%3DT.%26aulast%3DDoni%26aufirst%3DM.%26aulast%3DRosato%26aufirst%3DU.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DTesta%26aufirst%3DG.%26atitle%3DThe%2520Methyltransferase%2520Set7%252F9%2520%2528Setd7%2529%2520Is%2520Dispensable%2520for%2520the%2520p53-Mediated%2520DNA%2520Damage%2520Response%2520In%2520Vivo%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D43%26spage%3D681%26epage%3D688%26doi%3D10.1016%2Fj.molcel.2011.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Mori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagechika, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, T.</span><span> </span><span class="NLM_article-title">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">8158</span><span class="NLM_x">–</span> <span class="NLM_lpage">8166</span><span class="refDoi"> DOI: 10.1016/j.bmc.2010.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2010.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21036620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8158-8166&author=S.+Moriauthor=K.+Iwaseauthor=N.+Iwanamiauthor=Y.+Tanakaauthor=H.+Kagechikaauthor=T.+Hirano&title=Development+of+novel+bisubstrate-type+inhibitors+of+histone+methyltransferase+SET7%2F9&doi=10.1016%2Fj.bmc.2010.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span></div><div class="casAuthors">Mori, Shuichi; Iwase, Kenta; Iwanami, Naoko; Tanaka, Yujiro; Kagechika, Hiroyuki; Hirano, Tomoya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8158-8166</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone modification, for example, by histone deacetylase (HDAC) and histone lysine methyltransferase (HMT), plays an important role in regulating gene expression.  To obtain novel inhibitors as tools for investigating the physiol. function of members of the HMT family, we designed and synthesized novel inhibitors, which are amine analogs of adenosylmethionine (AdoMet; the cofactor utilized in the methylation reaction) bearing various alkylamino groups coupled via an ethylene linker.  The inhibitory activities of these compds. towards SET7/9, an HMT, were evaluated.  It was found that introduction of an alkylamino group increased the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUp7neuWTg1rVg90H21EOLACvtfcHk0lhc_5Tg8YhdcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL&md5=76faa42f957d02ed47458c348b324929</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DS.%26aulast%3DIwase%26aufirst%3DK.%26aulast%3DIwanami%26aufirst%3DN.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520novel%2520bisubstrate-type%2520inhibitors%2520of%2520histone%2520methyltransferase%2520SET7%252F9%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8158%26epage%3D8166%26doi%3D10.1016%2Fj.bmc.2010.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Niwa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Handa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomabechi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohsawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirouzu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagechika, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structures of histone methyltransferase SET7/9 in complexes with adenosylmethionine derivatives</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">602</span><span class="refDoi"> DOI: 10.1107/S0907444912052092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1107%2FS0907444912052092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=595-602&author=H.+Niwaauthor=N.+Handaauthor=Y.+Tomabechiauthor=K.+Hondaauthor=M.+Toyamaauthor=N.+Ohsawaauthor=M.+Shirouzuauthor=H.+Kagechikaauthor=T.+Hiranoauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structures+of+histone+methyltransferase+SET7%2F9+in+complexes+with+adenosylmethionine+derivatives&doi=10.1107%2FS0907444912052092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444912052092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912052092%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DHanda%26aufirst%3DN.%26aulast%3DTomabechi%26aufirst%3DY.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DToyama%26aufirst%3DM.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructures%2520of%2520histone%2520methyltransferase%2520SET7%252F9%2520in%2520complexes%2520with%2520adenosylmethionine%2520derivatives%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D595%26epage%3D602%26doi%3D10.1107%2FS0907444912052092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Dowden, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S. G.</span><span> </span><span class="NLM_article-title">Toward the development of potent and selective bisubstrate inhibitors of protein arginine methyltransferases</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmcl.2010.02.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20219369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2103-5&author=J.+Dowdenauthor=W.+Hongauthor=R.+V.+Parryauthor=R.+A.+Pikeauthor=S.+G.+Ward&title=Toward+the+development+of+potent+and+selective+bisubstrate+inhibitors+of+protein+arginine+methyltransferases&doi=10.1016%2Fj.bmcl.2010.02.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the development of potent and selective bisubstrate inhibitors of protein arginine methyltransferases</span></div><div class="casAuthors">Dowden, James; Hong, Wei; Parry, Richard V.; Pike, Richard A.; Ward, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2103-2105</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Prototype inhibitors of protein arginine methyltransferases (PRMTs) have been constructed by attaching guanidine functionality via a variable linker to non-reactive amine analogs of the cellular co-factor (S)-adenosyl methionine (AdoMet).  Potent inhibition of PRMT1 (IC50 of ∼3-6 μM) combined with weak inhibition of the lysine methyltransferase SET7 (∼50% of activity at 100 μM) was obsd. for two such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj5FdIemXEXrVg90H21EOLACvtfcHk0lhc_5Tg8YhdcA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGmsb4%253D&md5=6401f6157aa931ae23ac27928d4bf1d1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.069%26sid%3Dliteratum%253Aachs%26aulast%3DDowden%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DParry%26aufirst%3DR.%2BV.%26aulast%3DPike%26aufirst%3DR.%2BA.%26aulast%3DWard%26aufirst%3DS.%2BG.%26atitle%3DToward%2520the%2520development%2520of%2520potent%2520and%2520selective%2520bisubstrate%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2103%26epage%3D5%26doi%3D10.1016%2Fj.bmcl.2010.02.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Bissinger, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassenboehler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavarelli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Acyl derivatives of p-aminosulfonamides and dapsone as new inhibitors of the arginine methyltransferase hPRMT1</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3717</span><span class="NLM_x">–</span> <span class="NLM_lpage">3731</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2011.02.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3717-3731&author=E.+M.+Bissingerauthor=R.+Heinkeauthor=A.+Spannhoffauthor=A.+Eberlinauthor=E.+Metzgerauthor=V.+Curaauthor=P.+Hassenboehlerauthor=J.+Cavarelliauthor=R.+Schuleauthor=M.+T.+Bedfordauthor=W.+Sipplauthor=M.+Jung&title=Acyl+derivatives+of+p-aminosulfonamides+and+dapsone+as+new+inhibitors+of+the+arginine+methyltransferase+hPRMT1&doi=10.1016%2Fj.bmc.2011.02.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.032%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.%2BM.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DEberlin%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DHassenboehler%26aufirst%3DP.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DAcyl%2520derivatives%2520of%2520p-aminosulfonamides%2520and%2520dapsone%2520as%2520new%2520inhibitors%2520of%2520the%2520arginine%2520methyltransferase%2520hPRMT1%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3717%26epage%3D3731%26doi%3D10.1016%2Fj.bmc.2011.02.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1021/ml3003346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+Potent%2C+Selective%2C+Cell-Active+Inhibitors+of+the+Histone+Lysine+Methyltransferase+EZH2&doi=10.1021%2Fml3003346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0ljYhBeAFdOBnw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520Potent%252C%2520Selective%252C%2520Cell-Active%2520Inhibitors%2520of%2520the%2520Histone%2520Lysine%2520Methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096%26doi%3D10.1021%2Fml3003346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Kashyap, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, T. M.</span><span> </span><span class="NLM_article-title">Using ’biased-privileged’ scaffolds to identify lysine methyltransferase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2253</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.02.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2014.02.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=2253-60&author=S.+Kashyapauthor=J.+Sandlerauthor=U.+Petersauthor=E.+J.+Martinezauthor=T.+M.+Kapoor&title=Using+%E2%80%99biased-privileged%E2%80%99+scaffolds+to+identify+lysine+methyltransferase+inhibitors&doi=10.1016%2Fj.bmc.2014.02.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.02.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.02.024%26sid%3Dliteratum%253Aachs%26aulast%3DKashyap%26aufirst%3DS.%26aulast%3DSandler%26aufirst%3DJ.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DMartinez%26aufirst%3DE.%2BJ.%26aulast%3DKapoor%26aufirst%3DT.%2BM.%26atitle%3DUsing%2520%25E2%2580%2599biased-privileged%25E2%2580%2599%2520scaffolds%2520to%2520identify%2520lysine%2520methyltransferase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D2253%26epage%3D60%26doi%3D10.1016%2Fj.bmc.2014.02.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Santo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Novel 3,5-Bis(bromohydroxybenzylidene)piperidin-4-ones as Coactivator-Associated Arginine Methyltransferase 1 Inhibitors: Enzyme Selectivity and Cellular Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4928</span><span class="NLM_x">–</span> <span class="NLM_lpage">4932</span><span class="refDoi"> DOI: 10.1021/jm200453n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200453n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4928-4932&author=D.+Chengauthor=S.+Valenteauthor=S.+Castellanoauthor=G.+Sbardellaauthor=R.+Di+Santoauthor=R.+Costiauthor=M.+T.+Bedfordauthor=A.+Mai&title=Novel+3%2C5-Bis%28bromohydroxybenzylidene%29piperidin-4-ones+as+Coactivator-Associated+Arginine+Methyltransferase+1+Inhibitors%3A+Enzyme+Selectivity+and+Cellular+Activity&doi=10.1021%2Fjm200453n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3,5-Bis(bromohydroxybenzylidene)piperidin-4-ones as Coactivator-Associated Arginine Methyltransferase 1 Inhibitors: Enzyme Selectivity and Cellular Activity</span></div><div class="casAuthors">Cheng, Donghang; Valente, Sergio; Castellano, Sabrina; Sbardella, Gianluca; Di Santo, Roberto; Costi, Roberta; Bedford, Mark T.; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4928-4932</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) represents a valuable target for hormone-dependent tumors such as prostate and breast cancers.  Here we report the enzyme and cellular characterization of the 1-benzyl-3,5-bis(3-bromo-4-hydroxybenzylidene)piperidin-4-one (7g) and its analogs 8a-l.  Among them, 7g, 8e, and 8l displayed high and selective CARM1 inhibition, with lower or no activity against a panel of different PRMTs or HKMTs.  In human LNCaP cells, 7g showed a significant dose-dependent redn. of the PSA promoter activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv4DaWjduarVg90H21EOLACvtfcHk0lhSuSrIil2Gew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVSjtrw%253D&md5=c9fba6d328440a5e41046ff0c47f661c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm200453n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200453n%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDi%2BSanto%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DNovel%25203%252C5-Bis%2528bromohydroxybenzylidene%2529piperidin-4-ones%2520as%2520Coactivator-Associated%2520Arginine%2520Methyltransferase%25201%2520Inhibitors%253A%2520Enzyme%2520Selectivity%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4928%26epage%3D4932%26doi%3D10.1021%2Fjm200453n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Synthesis and biological validation of novel synthetic histone/protein methyltransferase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">991</span><span class="refDoi"> DOI: 10.1002/cmdc.200700023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fcmdc.200700023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17458842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVyktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=987-991&author=A.+Maiauthor=S.+Valenteauthor=D.+Chengauthor=A.+Perroneauthor=R.+Ragnoauthor=S.+Simeoniauthor=G.+Sbardellaauthor=G.+Broschauthor=A.+Nebbiosoauthor=M.+Conteauthor=L.+Altucciauthor=M.+T.+Bedford&title=Synthesis+and+biological+validation+of+novel+synthetic+histone%2Fprotein+methyltransferase+inhibitors&doi=10.1002%2Fcmdc.200700023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological validation of novel synthetic histone/protein methyltransferase inhibitors</span></div><div class="casAuthors">Mai, Antonello; Valente, Sergio; Cheng, Donghang; Perrone, Andrea; Ragno, Rino; Simeoni, Silvia; Sbardella, Gianluca; Brosch, Gerald; Nebbioso, Angela; Conte, Mariarosaria; Altucci, Lucia; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">987-991</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The prepn. of novel histone/protein methyltransferase inhibitors is described and their potential as antitumor agents was evaluated in human leukemia cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCBG8vwlib8bVg90H21EOLACvtfcHk0lhSuSrIil2Gew"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVyktLk%253D&md5=cb0e3448e787d65760ee7d4563cfbb75</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700023%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DConte%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DSynthesis%2520and%2520biological%2520validation%2520of%2520novel%2520synthetic%2520histone%252Fprotein%2520methyltransferase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D987%26epage%3D991%26doi%3D10.1002%2Fcmdc.200700023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Nishino, N.; Takemoto, Y.; Ito, A.; Yoshida, M.; Yamori, T.</span> Preventive and therapeutic agent of disease containing inhibitor of protein ricin methylation enzyme Set7/9. Japanese Patent JP2012102087,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Nishino%2C+N.%3B+Takemoto%2C+Y.%3B+Ito%2C+A.%3B+Yoshida%2C+M.%3B+Yamori%2C+T.+Preventive+and+therapeutic+agent+of+disease+containing+inhibitor+of+protein+ricin+methylation+enzyme+Set7%2F9.+Japanese+Patent+JP2012102087%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DN.%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.+P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0ljPPbUT9MSeEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mok, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Structural chemistry of the histone methyltransferases cofactor binding site</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1021/ci100479z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100479z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Oksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=612-23&author=V.+Campagna-Slaterauthor=M.+W.+Mokauthor=K.+T.+Nguyenauthor=M.+Feherauthor=R.+Najmanovichauthor=M.+Schapira&title=Structural+chemistry+of+the+histone+methyltransferases+cofactor+binding+site&doi=10.1021%2Fci100479z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Chemistry of the Histone Methyltransferases Cofactor Binding Site</span></div><div class="casAuthors">Campagna-Slater, Valerie; Mok, Man Wai; Nguyen, Kong T.; Feher, Miklos; Najmanovich, Rafael; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">612-623</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) transfer a Me group from the cofactor S-adenosyl methionine to lysine or arginine residues on histone tails, thereby regulating chromatin compaction, binding of effector proteins and gene transcription.  HMTs constitute an emerging target class in diverse disease areas, and selective chem. probes are necessary for target validation.  Potent and selective competitors of the substrate peptide have been reported, but the chem. tractability of the cofactor binding site is poorly understood.  Here, a systematic anal. of this site across structures of 14 human HMTs or close homologues was conducted.  The druggability, interaction hotspots, and diversity of the cofactor binding pocket were dissected.  This anal. strongly suggests that this site is chem. tractable.  General principles underlying tight binding and specific guidelines to achieve selective inhibition are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnQVsPGdhHErVg90H21EOLACvtfcHk0ljPPbUT9MSeEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1Oksrk%253D&md5=73ea6e829571a9c610acd7391d8ad082</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci100479z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100479z%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DMok%26aufirst%3DM.%2BW.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DFeher%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520chemistry%2520of%2520the%2520histone%2520methyltransferases%2520cofactor%2520binding%2520site%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D612%26epage%3D23%26doi%3D10.1021%2Fci100479z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0ljPPbUT9MSeEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Tervo, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyrylenko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niskanen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salminen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leppanen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyronen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvinen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poso, A.</span><span> </span><span class="NLM_article-title">An in silico approach to discovering novel inhibitors of human sirtuin type 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6292</span><span class="NLM_x">–</span> <span class="NLM_lpage">6298</span><span class="refDoi"> DOI: 10.1021/jm049933m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049933m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6292-6298&author=A.+J.+Tervoauthor=S.+Kyrylenkoauthor=P.+Niskanenauthor=A.+Salminenauthor=J.+Leppanenauthor=T.+H.+Nyronenauthor=T.+Jarvinenauthor=A.+Poso&title=An+in+silico+approach+to+discovering+novel+inhibitors+of+human+sirtuin+type+2&doi=10.1021%2Fjm049933m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm049933m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049933m%26sid%3Dliteratum%253Aachs%26aulast%3DTervo%26aufirst%3DA.%2BJ.%26aulast%3DKyrylenko%26aufirst%3DS.%26aulast%3DNiskanen%26aufirst%3DP.%26aulast%3DSalminen%26aufirst%3DA.%26aulast%3DLeppanen%26aufirst%3DJ.%26aulast%3DNyronen%26aufirst%3DT.%2BH.%26aulast%3DJarvinen%26aufirst%3DT.%26aulast%3DPoso%26aufirst%3DA.%26atitle%3DAn%2520in%2520silico%2520approach%2520to%2520discovering%2520novel%2520inhibitors%2520of%2520human%2520sirtuin%2520type%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6292%26epage%3D6298%26doi%3D10.1021%2Fjm049933m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicidomini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caroli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tramontano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaccini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Histone Arginine Methyltransferases: Homology Modeling, Molecular Docking, Binding Mode Analysis, and Biological Evaluations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span><span class="refDoi"> DOI: 10.1021/jm061213n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061213n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1241-1253&author=R.+Ragnoauthor=S.+Simeoniauthor=S.+Castellanoauthor=C.+Vicidominiauthor=A.+Maiauthor=A.+Caroliauthor=A.+Tramontanoauthor=C.+Bonacciniauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=G.+Sbardella&title=Small+Molecule+Inhibitors+of+Histone+Arginine+Methyltransferases%3A+Homology+Modeling%2C+Molecular+Docking%2C+Binding+Mode+Analysis%2C+and+Biological+Evaluations&doi=10.1021%2Fjm061213n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm061213n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061213n%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DVicidomini%26aufirst%3DC.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCaroli%26aufirst%3DA.%26aulast%3DTramontano%26aufirst%3DA.%26aulast%3DBonaccini%26aufirst%3DC.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Histone%2520Arginine%2520Methyltransferases%253A%2520Homology%2520Modeling%252C%2520Molecular%2520Docking%252C%2520Binding%2520Mode%2520Analysis%252C%2520and%2520Biological%2520Evaluations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1241%26epage%3D1253%26doi%3D10.1021%2Fjm061213n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gust, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Target-based approach to inhibitors of histone arginine methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span><span class="refDoi"> DOI: 10.1021/jm061250e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061250e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2319-2325&author=A.+Spannhoffauthor=R.+Heinkeauthor=I.+Bauerauthor=P.+Trojerauthor=E.+Metzgerauthor=R.+Gustauthor=R.+Schueleauthor=G.+Broschauthor=W.+Sipplauthor=M.+Jung&title=Target-based+approach+to+inhibitors+of+histone+arginine+methyltransferases&doi=10.1021%2Fjm061250e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm061250e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061250e%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DGust%26aufirst%3DR.%26aulast%3DSchuele%26aufirst%3DR.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTarget-based%2520approach%2520to%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2319%26epage%3D2325%26doi%3D10.1021%2Fjm061250e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: Mixed Histone/Protein methyltransferase, acetyltransferase, and class III deacetylase (Sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span><span class="refDoi"> DOI: 10.1021/jm701595q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+Mixed+Histone%2FProtein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28Sirtuin%29+inhibitors&doi=10.1021%2Fjm701595q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520Mixed%2520Histone%252FProtein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528Sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290%26doi%3D10.1021%2Fjm701595q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Vadivelan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rambabu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boppana, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagarlapudi, S. A. R. P.</span><span> </span><span class="NLM_article-title">Pharmacophore modeling and virtual screening studies to design some potential histone deacetylase inhibitors as new leads</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">946</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2007.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.jmgm.2007.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOksL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=935-946&author=S.+Vadivelanauthor=B.+N.+Sinhaauthor=G.+Rambabuauthor=K.+Boppanaauthor=S.+A.+R.+P.+Jagarlapudi&title=Pharmacophore+modeling+and+virtual+screening+studies+to+design+some+potential+histone+deacetylase+inhibitors+as+new+leads&doi=10.1016%2Fj.jmgm.2007.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling and virtual screening studies to design some potential histone deacetylase inhibitors as new leads</span></div><div class="casAuthors">Vadivelan, S.; Sinha, B. N.; Rambabu, G.; Boppana, Kiran; Jagarlapudi, Sarma A. R. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">935-946</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylase is one of the important targets in the treatment of solid tumors and hematol. cancers.  A total of 20 well-defined inhibitors were used to generate Pharmacophore models using and HypoGen module of Catalyst.  These 20 mols. broadly represent 3 different chemotypes.  The best HypoGen model consists of four-pharmacophore features, one hydrogen bond acceptor, one hydrophobic aliph. and two ring arom. centers.  This model was validated against 378 known HDAC inhibitors with a correlation of 0.897 as well as enrichment factor of 2.68 against a max. value of 3.  This model was further used to retrieve mols. from NCI database with 238,819 mols.  A total of 4638 mols. from a pool of 238,819 mols. were identified as hits while 297 mols. were indicated as highly active.  Also, a Similarity anal. has been carried out for set of 4638 hits with respect to most active mol. of each chemotypes which validated not only the Virtual Screening potential of the model but also identified the possible new Chemotypes.  This type of Similarity anal. would prove to be efficient not only for lead generation but also for lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6-KqHzBn9lLVg90H21EOLACvtfcHk0lgOkE8OOVM8Hw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOksL7E&md5=6aa057815c5bf0cd21f77faaf1390844</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DVadivelan%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DB.%2BN.%26aulast%3DRambabu%26aufirst%3DG.%26aulast%3DBoppana%26aufirst%3DK.%26aulast%3DJagarlapudi%26aufirst%3DS.%2BA.%2BR.%2BP.%26atitle%3DPharmacophore%2520modeling%2520and%2520virtual%2520screening%2520studies%2520to%2520design%2520some%2520potential%2520histone%2520deacetylase%2520inhibitors%2520as%2520new%2520leads%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2008%26volume%3D26%26spage%3D935%26epage%3D946%26doi%3D10.1016%2Fj.jmgm.2007.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Evaluation of Virtual Screening as a Tool for Chemical Genetic Applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2082</span><span class="NLM_x">–</span> <span class="NLM_lpage">2091</span><span class="refDoi"> DOI: 10.1021/ci900219u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900219u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVartbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2082-2091&author=V.+Campagna-Slaterauthor=M.+Schapira&title=Evaluation+of+Virtual+Screening+as+a+Tool+for+Chemical+Genetic+Applications&doi=10.1021%2Fci900219u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of virtual screening as a tool for chemical genetic applications</span></div><div class="casAuthors">Campagna-Slater, Valerie; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2082-2091</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A collection of over 50,000 functionally annotated drugs, clin. candidates, and endogenous ligands was docked in silico against nine binding sites from seven protein targets, representing diverse function and structure, namely the sulfotransferases SULT1E1 and SULT1A3, the histone methyltransferase EHMT1, the histone acetyltransferase MYST3, and the nuclear hormone receptors ERα, PPARγ, and TRβ.  For 5 of the 9 virtual screens, compds. that docked best to the receptors clearly recapitulated known biol. functions of the genes or identified novel biol. subsequently validated in a sep. exptl. study.  In two cases, the hit list indicated some relevant but isolated biol. functions which would probably have been ignored a priori, and selected compds. were completely unrelated to gene function for the last two virtual screens.  This study demonstrates that virtual screening of pharmacol. annotated compd. libraries can be used to derive target biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWkRSwI9LGhbVg90H21EOLACvtfcHk0lhPbdHkFLYM0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVartbbK&md5=237625f754eb93134749f00b16bbd5f2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci900219u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900219u%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520Virtual%2520Screening%2520as%2520a%2520Tool%2520for%2520Chemical%2520Genetic%2520Applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2082%26epage%3D2091%26doi%3D10.1021%2Fci900219u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Novel Inhibitors of Human Histone Deacetylase (HDAC) Identified by QSAR Modeling of Known Inhibitors, Virtual Screening, and Experimental Validation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span><span class="refDoi"> DOI: 10.1021/ci800366f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800366f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1agt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=461-476&author=H.+Tangauthor=X.+S.+Wangauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=K.+V.+Butlerauthor=A.+P.+Kozikowskiauthor=M.+Jungauthor=A.+Tropsha&title=Novel+Inhibitors+of+Human+Histone+Deacetylase+%28HDAC%29+Identified+by+QSAR+Modeling+of+Known+Inhibitors%2C+Virtual+Screening%2C+and+Experimental+Validation&doi=10.1021%2Fci800366f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors of Human Histone Deacetylase (HDAC) Identified by QSAR Modeling of Known Inhibitors, Virtual Screening, and Experimental Validation</span></div><div class="casAuthors">Tang, Hao; Wang, Xiang S.; Huang, Xi-Ping; Roth, Bryan L.; Butler, Kyle V.; Kozikowski, Alan P.; Jung, Mira; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">461-476</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of histone deacetylases (HDACIs) have emerged as a new class of drugs for the treatment of human cancers and other diseases because of their effects on cell growth, differentiation, and apoptosis.  In this study we have developed several quant. structure-activity relationship (QSAR) models for 59 chem. diverse histone deacetylase class 1 (HDAC1) inhibitors.  The variable selection k nearest neighbor (kNN) and support vector machines (SVM) QSAR modeling approaches using both MolconnZ and MOE chem. descriptors generated from two-dimensional rendering of compds. as chem. graphs have been employed.  We have relied on a rigorous model development workflow including the division of the data set into training, test, and external sets and extensive internal and external validation.  Highly predictive QSAR models were generated with leave-one-out cross-validated (LOO-CV) q2 and external R2 values as high as 0.80 and 0.87, resp., using the kNN/MolconnZ approach and 0.93 and 0.87, resp., using the SVM/MolconnZ approach.  All validated QSAR models were employed concurrently for virtual screening (VS) of an inhouse compd. collection including 9.5 million mols. compiled from the ZINC7.0 database, the World Drug Index (WDI) database, the ASINEX Synergy libraries, and other com. databases.  VS resulted in 45 structurally unique consensus hits that were considered novel putative HDAC1 inhibitors.  These computational hits had several novel structural features that were not present in the original data set.  Four computational hits with novel scaffolds were tested exptl., and three of them were confirmed active against HDAC1, with IC50 values for the most active compd. of 1.00 μM.  The fourth compd. was later identified to be a selective inhibitor of HDAC6, a Class II HDAC.  Moreover, two of the confirmed hits are marketed drugs, which could potentially facilitate their further development as anticancer agents.  This study illustrates the power of the combined QSAR-VS method as a general approach for the effective identification of structurally novel bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dwScSdXT4bVg90H21EOLACvtfcHk0lhPbdHkFLYM0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1agt7g%253D&md5=764cd294ec422c0d9922b31e5941cb71</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci800366f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800366f%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%2BS.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DNovel%2520Inhibitors%2520of%2520Human%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Identified%2520by%2520QSAR%2520Modeling%2520of%2520Known%2520Inhibitors%252C%2520Virtual%2520Screening%252C%2520and%2520Experimental%2520Validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D461%26epage%3D476%26doi%3D10.1021%2Fci800366f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Kireev, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris-Drouin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Identification of non-peptide malignant brain tumor (MBT) repeat antagonists by virtual screening of commercially available compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7625</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1021/jm1007374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1007374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=7625-31&author=D.+Kireevauthor=T.+J.+Wigleauthor=J.+Norris-Drouinauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=S.+V.+Frye&title=Identification+of+non-peptide+malignant+brain+tumor+%28MBT%29+repeat+antagonists+by+virtual+screening+of+commercially+available+compounds&doi=10.1021%2Fjm1007374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm1007374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1007374%26sid%3Dliteratum%253Aachs%26aulast%3DKireev%26aufirst%3DD.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520of%2520non-peptide%2520malignant%2520brain%2520tumor%2520%2528MBT%2529%2520repeat%2520antagonists%2520by%2520virtual%2520screening%2520of%2520commercially%2520available%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D7625%26epage%3D31%26doi%3D10.1021%2Fjm1007374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Sundarapandian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugunadevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woo, L. K.</span><span> </span><span class="NLM_article-title">Docking-enabled pharmacophore model for histone deacetylase 8 inhibitors and its application in anti-cancer drug discovery</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2010.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.jmgm.2010.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=382-395&author=T.+Sundarapandianauthor=J.+Shaliniauthor=S.+Sugunadeviauthor=L.+K.+Woo&title=Docking-enabled+pharmacophore+model+for+histone+deacetylase+8+inhibitors+and+its+application+in+anti-cancer+drug+discovery&doi=10.1016%2Fj.jmgm.2010.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking-enabled pharmacophore model for histone deacetylase 8 inhibitors and its application in anti-cancer drug discovery</span></div><div class="casAuthors">Sundarapandian, Thangapandian; Shalini, John; Sugunadevi, Sakkiah; Woo, Lee Keun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">382-395</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Zinc-dependent histone deacetylase 8 removes the epsilon-acetyl groups present in the N-terminal lysine residues of histone proteins, thereby restricting various transcription factors from being expressed.  Inhibition of this enzyme has been reported to be a novel strategy in cancer treatment.  To identify novel and diverse leads for use in potent histone deacetylase 8 inhibitor design, a pharmacophore model showing high correlation between exptl. and estd. activities was generated using the best conformations of training set compds. from mol. docking expts.  The best pharmacophore model was validated using four different strategies and then used in database screening for novel virtual leads.  Hit compds. were selected and subjected to mol. docking using GOLD.  The top-scored compd. was further optimized for improved binding.  The optimization step led to a new set of compds. with both improved binding at the active site and estd. activities.  The identified virtual leads could be used for designing potent histone deacetylase 8 inhibitors as anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFZFFaiXEbGLVg90H21EOLACvtfcHk0lgtOFQ_vl6XdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsb%252FN&md5=87df1965105819c4d0f404e0297deca5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DSundarapandian%26aufirst%3DT.%26aulast%3DShalini%26aufirst%3DJ.%26aulast%3DSugunadevi%26aufirst%3DS.%26aulast%3DWoo%26aufirst%3DL.%2BK.%26atitle%3DDocking-enabled%2520pharmacophore%2520model%2520for%2520histone%2520deacetylase%25208%2520inhibitors%2520and%2520its%2520application%2520in%2520anti-cancer%2520drug%2520discovery%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D382%26epage%3D395%26doi%3D10.1016%2Fj.jmgm.2010.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine Demethylases Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+Demethylases+Inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lgtOFQ_vl6XdA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520Demethylases%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Yoo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span> </span><span class="NLM_article-title">Homology modeling, docking and structure-based pharmacophore of inhibitors of DNA methyltransferase</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">555</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1007/s10822-011-9441-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs10822-011-9441-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=555-567&author=J.+Yooauthor=J.+L.+Medina-Franco&title=Homology+modeling%2C+docking+and+structure-based+pharmacophore+of+inhibitors+of+DNA+methyltransferase&doi=10.1007%2Fs10822-011-9441-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9441-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9441-1%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DHomology%2520modeling%252C%2520docking%2520and%2520structure-based%2520pharmacophore%2520of%2520inhibitors%2520of%2520DNA%2520methyltransferase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D555%26epage%3D567%26doi%3D10.1007%2Fs10822-011-9441-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, T. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Progress in the Development and Application of Small Molecule Inhibitors of Bromodomain–Acetyl-lysine Interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9393</span><span class="NLM_x">–</span> <span class="NLM_lpage">9413</span><span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+C.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+Development+and+Application+of+Small+Molecule+Inhibitors+of+Bromodomain%E2%80%93Acetyl-lysine+Interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520Development%2520and%2520Application%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Bromodomain%25E2%2580%2593Acetyl-lysine%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-Based Virtual Screening and Biological Evaluation of Small Molecule Inhibitors for Protein Arginine Methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span><span class="refDoi"> DOI: 10.1021/jm300521m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.-W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=K.+K.+Liauthor=M.-F.+Linauthor=H.+Jiangauthor=Y.+G.+Zhengauthor=C.+Luo&title=Pharmacophore-Based+Virtual+Screening+and+Biological+Evaluation+of+Small+Molecule+Inhibitors+for+Protein+Arginine+Methylation&doi=10.1021%2Fjm300521m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.-W.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLin%26aufirst%3DM.-F.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-Based%2520Virtual%2520Screening%2520and%2520Biological%2520Evaluation%2520of%2520Small%2520Molecule%2520Inhibitors%2520for%2520Protein%2520Arginine%2520Methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987%26doi%3D10.1021%2Fjm300521m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Yoo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span> </span><span class="NLM_article-title">Inhibitors of DNA Methyltransferases: Insights from Computational Studies</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3475</span><span class="NLM_x">–</span> <span class="NLM_lpage">3487</span><span class="refDoi"> DOI: 10.2174/092986712801323289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2174%2F092986712801323289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22709005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2itb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3475-3487&author=J.+Yooauthor=J.+L.+Medina-Franco&title=Inhibitors+of+DNA+Methyltransferases%3A+Insights+from+Computational+Studies&doi=10.2174%2F092986712801323289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of DNA methyltransferases: insights from computational studies</span></div><div class="casAuthors">Yoo, J.; Medina-Franco, J. L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3475-3487</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs) are a family of epigenetic enzymes for which inhibition is an attractive strategy for the treatment of cancer and other diseases.  In synergy with exptl. approaches, computational methods are increasingly being used to identify and optimize the activity of inhibitors of DNMTs as well as to rationalize at the mol. level of the mechanism of established inhibitors.  Recently, a crystallog. structure of the methyltransferase domain of human DNMT1 bound to unmethylated DNA was published encouraging the application of structure-based approaches to design and optimize the activity of currently known inhibitors.  Herein, we review the progress in the discovery and optimization of inhibitors of DNMTs using computational approaches including homol. modeling, docking, pharmacophore modeling, mol. dynamics, and virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7J66x7abyxLVg90H21EOLACvtfcHk0lhOQPZ_wjyI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2itb3P&md5=ee28a21948b821fe08afbba97e56622e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.2174%2F092986712801323289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712801323289%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DInhibitors%2520of%2520DNA%2520Methyltransferases%253A%2520Insights%2520from%2520Computational%2520Studies%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3475%26epage%3D3487%26doi%3D10.2174%2F092986712801323289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">Discovery of a novel histone deacetylase 8 inhibitor by virtual screening</span> <span class="citation_source-journal">Med. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1007/s00044-010-9519-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs00044-010-9519-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2rs7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=152-156&author=L.+Zhangauthor=M.+Liauthor=J.+Fengauthor=H.+Fangauthor=W.+Xu&title=Discovery+of+a+novel+histone+deacetylase+8+inhibitor+by+virtual+screening&doi=10.1007%2Fs00044-010-9519-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel histone deacetylase 8 inhibitor by virtual screening</span></div><div class="casAuthors">Zhang, Lei; Li, Min-Yong; Feng, Jin-Hong; Fang, Hao; Xu, Wen-Fang</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-156</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Boston</span>)
        </div><div class="casAbstract">In order to develop potent histone deacetylase inhibitors, a virtual screening approach was performed to discover novel lead structures.  A com. database contg. about 167,000 mols. was in silico filtered by rule of five, zinc-binding groups, pharmacophore models, and binding pattern anal.  At last, three mols. were selected for enzyme inhibition assay, and one compd. 02 has IC50 of 1.6 μM against histone deacetylase 8 (HDAC8).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQFHm0bn6DLVg90H21EOLACvtfcHk0lhOQPZ_wjyI8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2rs7vJ&md5=457c889bd592a0913723386db3e796b4</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1007%2Fs00044-010-9519-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-010-9519-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520histone%2520deacetylase%25208%2520inhibitor%2520by%2520virtual%2520screening%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2012%26volume%3D21%26spage%3D152%26epage%3D156%26doi%3D10.1007%2Fs00044-010-9519-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in Vivo Chemical Probe of the Lysine Methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.-P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+Vivo+Chemical+Probe+of+the+Lysine+Methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lhtn7yixJ00Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520Vivo%2520Chemical%2520Probe%2520of%2520the%2520Lysine%2520Methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+Allosteric+Binding+Site+of+PRMT3+Yields+Potent+and+Selective+Inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhtn7yixJ00Cw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520Allosteric%2520Binding%2520Site%2520of%2520PRMT3%2520Yields%2520Potent%2520and%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, J.</span><span> </span><span class="NLM_article-title">Molecular Modeling and Virtual Screening of DNA Methyltransferase Inhibitors</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2138</span><span class="NLM_x">–</span> <span class="NLM_lpage">2147</span><span class="refDoi"> DOI: 10.2174/1381612811319120002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2174%2F1381612811319120002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=23016844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2138-2147&author=J.+L.+Medina-Francoauthor=J.+Yoo&title=Molecular+Modeling+and+Virtual+Screening+of+DNA+Methyltransferase+Inhibitors&doi=10.2174%2F1381612811319120002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling and virtual screening of DNA methyltransferase inhibitors</span></div><div class="casAuthors">Medina-Franco, Jose L.; Yoo, Jakyung</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2138-2147</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibition of DNA methyltransferases (DNMTs) is a promising approach for the therapeutic treatment of cancer and other diseases.  In this work, we review the recent progress on the mol. modeling and virtual screening toward the identification of key structural features assocd. with the enzyme inhibitory action of active compds. and to identify DNMT inhibitors with novel mol. scaffolds.  We discuss the mol. modeling with the co-factor binding site using a recent crystallog. structure of the methyltransferase domain of human DNMT1.  We also review the emerging synergy of mol. modeling and chemoinformatic approaches applied to epigenetic therapies targeting DNMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraut6rC0hKHbVg90H21EOLACvtfcHk0lgcl40sIF5w1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLw%253D&md5=7854d20bf9e11a343dd541c8225c7dde</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120002%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DYoo%26aufirst%3DJ.%26atitle%3DMolecular%2520Modeling%2520and%2520Virtual%2520Screening%2520of%2520DNA%2520Methyltransferase%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D2138%26epage%3D2147%26doi%3D10.2174%2F1381612811319120002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Fincke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7334</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-42&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Schuleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0lgcl40sIF5w1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D42%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-Throughput Virtual Screening Identifies Novel N ′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-Throughput+Virtual+Screening+Identifies+Novel+N+%E2%80%B2-%281-Phenylethylidene%29-benzohydrazides+as+Potent%2C+Specific%2C+and+Reversible+LSD1+Inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-Throughput%2520Virtual%2520Screening%2520Identifies%2520Novel%2520N%2520%25E2%2580%25B2-%25281-Phenylethylidene%2529-benzohydrazides%2520as%2520Potent%252C%2520Specific%252C%2520and%2520Reversible%2520LSD1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomsett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8073</span><span class="NLM_x">–</span> <span class="NLM_lpage">8088</span><span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+Novel+Small-Molecule+Inhibitors+of+BRD4+Using+Structure-Based+Virtual+Screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lgcmPdLnRuPTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Small-Molecule%2520Inhibitors%2520of%2520BRD4%2520Using%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibanez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shum, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djaballah, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of SETD8 with cellular activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2471</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1021/cb500515r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500515r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2471-8&author=G.+Blumauthor=G.+Ibanezauthor=X.+Raoauthor=D.+Shumauthor=C.+Raduauthor=H.+Djaballahauthor=J.+C.+Riceauthor=M.+Luo&title=Small-molecule+inhibitors+of+SETD8+with+cellular+activity&doi=10.1021%2Fcb500515r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of SETD8 with Cellular Activity</span></div><div class="casAuthors">Blum, Gil; Ibanez, Glorymar; Rao, Xiangjun; Shum, David; Radu, Constantin; Djaballah, Hakim; Rice, Judd C.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2471-2478</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SETD8/SET8/Pr-SET7/KMT5A is the sole protein lysine methyltransferase (PKMT) known to monomethylate lysine 20 of histone H4 in vivo.  SETD8's methyltransferase activity has been implicated in many essential cellular processes including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  Developing SETD8 inhibitors with cellular activity is a key step toward elucidating the diverse roles of SETD8 via convenient pharmacol. perturbation.  From the hits of a prior high throughput screen (HTS), SPS8I1-3 (NSC663284, BVT948, and ryuvidine) were validated as potent SETD8 inhibitors.  These compds. contain different structural motifs and inhibit SETD8 via distinct modes.  More importantly, these compds. show cellular activity by suppressing the H4K20me1 mark of SETD8 and recapitulate characteristic S/G2/M-phase cell cycle defects as obsd. for RNAi-mediated SETD8 knockdown.  The commonality of SPS8I1-3 against SETD8, together with their distinct structures and mechanisms for SETD8 inhibition, argues for the collective application of these compds. as SETD8 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquxqpOQLgFObVg90H21EOLACvtfcHk0lgcmPdLnRuPTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE&md5=87b7c415bf82577843fd5eeb09c2c746</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fcb500515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500515r%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DIbanez%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DX.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520SETD8%2520with%2520cellular%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2471%26epage%3D8%26doi%3D10.1021%2Fcb500515r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Kannan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melesina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heimburg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span> </span><span class="NLM_article-title">Discovery of inhibitors of Schistosoma mansoni HDAC8 by combining homology modeling, virtual screening, and in vitro validation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3005</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1021/ci5004653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5004653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3005-19&author=S.+Kannanauthor=J.+Melesinaauthor=A.+T.+Hauserauthor=A.+Chakrabartiauthor=T.+Heimburgauthor=K.+Schmidtkunzauthor=A.+Walterauthor=M.+Marekauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jungauthor=W.+Sippl&title=Discovery+of+inhibitors+of+Schistosoma+mansoni+HDAC8+by+combining+homology+modeling%2C+virtual+screening%2C+and+in+vitro+validation&doi=10.1021%2Fci5004653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of Schistosoma mansoni HDAC8 by Combining Homology Modeling, Virtual Screening, and in Vitro Validation</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Melesina, Jelena; Hauser, Alexander-Thomas; Chakrabarti, Alokta; Heimburg, Tino; Schmidtkunz, Karin; Walter, Alexandra; Marek, Martin; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3005-3019</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by S. mansoni, is a tropical disease that affects over 200 million people worldwide.  A novel approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during their life cycle.  We recently identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here we present results from a virtual screening campaign on smHDAC8.  Besides hydroxamates, several sulfonamide-thiazole derivs. were identified by a target-based virtual screening using a homol. model of smHDAC8.  In vitro testing of 75 compds. identified 8 hydroxamates as potent and lead-like inhibitors of the parasitic HDAC8.  Solving of the crystal structure of smHDAC8 with two of the virtual screening hits confirmed the predicted binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH_psd1ffmerVg90H21EOLACvtfcHk0lhveEeHyTuvUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO&md5=512119ba4e5cbcdd0d28570a3acb74b5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fci5004653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5004653%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DChakrabarti%26aufirst%3DA.%26aulast%3DHeimburg%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520HDAC8%2520by%2520combining%2520homology%2520modeling%252C%2520virtual%2520screening%252C%2520and%2520in%2520vitro%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3005%26epage%3D19%26doi%3D10.1021%2Fci5004653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleich, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6822</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/jm500871s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500871s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6822-6833&author=A.+Maauthor=W.+Yuauthor=F.+Liauthor=R.+M.+Bleichauthor=J.+M.+Heroldauthor=K.+V.+Butlerauthor=J.+L.+Norrisauthor=V.+Korboukhauthor=A.+Tripathyauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Jin&title=Discovery+of+a+Selective%2C+Substrate-Competitive+Inhibitor+of+the+Lysine+Methyltransferase+SETD8&doi=10.1021%2Fjm500871s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span></div><div class="casAuthors">Ma, Anqi; Yu, Wenyu; Li, Fengling; Bleich, Rachel M.; Herold, J. Martin; Butler, Kyle V.; Norris, Jacqueline L.; Korboukh, Victoria; Tripathy, Ashutosh; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Vedadi, Masoud; Brown, Peter J.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6822-6833</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20).  Monomethylation of H4K20 has been implicated in regulating diverse biol. processes including the DNA damage response.  In addn. to H4K20, SETD8 monomethylates nonhistone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression.  However, selective inhibitors of SETD8 are scarce.  The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor.  Here, the authors report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 I.  This small-mol. inhibitor is active in multiple biochem. assays.  Its affinity to SETD8 was confirmed by ITC (isothermal titrn. calorimetry) and SPR (surface plasmon resonance) studies.  Importantly, compd. I is selective for SETD8 over 15 other methyltransferases.  The authors also describe structure-activity relationships (SAR) of this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR9y31FWBfY7Vg90H21EOLACvtfcHk0lhveEeHyTuvUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI&md5=adca277f02033f96fe45a28087dca14e</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500871s%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBleich%26aufirst%3DR.%2BM.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520Selective%252C%2520Substrate-Competitive%2520Inhibitor%2520of%2520the%2520Lysine%2520Methyltransferase%2520SETD8%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6822%26epage%3D6833%26doi%3D10.1021%2Fjm500871s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Parenti, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grozio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galeno, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damonte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sociali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballestrero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruzzone, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rio, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nencioni, A.</span><span> </span><span class="NLM_article-title">Discovery of Novel and Selective SIRT6 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4796</span><span class="NLM_x">–</span> <span class="NLM_lpage">4804</span><span class="refDoi"> DOI: 10.1021/jm500487d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500487d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1Cnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4796-4804&author=M.+D.+Parentiauthor=A.+Grozioauthor=I.+Bauerauthor=L.+Galenoauthor=P.+Damonteauthor=E.+Milloauthor=G.+Socialiauthor=C.+Franceschiauthor=A.+Ballestreroauthor=S.+Bruzzoneauthor=A.+D.+Rioauthor=A.+Nencioni&title=Discovery+of+Novel+and+Selective+SIRT6+Inhibitors&doi=10.1021%2Fjm500487d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel and Selective SIRT6 Inhibitors</span></div><div class="casAuthors">Parenti, Marco Daniele; Grozio, Alessia; Bauer, Inga; Galeno, Lauretta; Damonte, Patrizia; Millo, Enrico; Sociali, Giovanna; Franceschi, Claudio; Ballestrero, Alberto; Bruzzone, Santina; Rio, Alberto Del; Nencioni, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4796-4804</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SIRT6 is an NAD+-dependent deacetylase with a role in the transcriptional control of metab. and aging but also in genome stability and inflammation.  Broad therapeutic applications are foreseen for SIRT6 inhibitors, including uses in diabetes, immune-mediated disorders, and cancer.  Here we report on the identification of the first selective SIRT6 inhibitors by in silico screening.  The most promising leads show micromolar IC50s, have significant selectivity for SIRT6 vs. SIRT1 and SIRT2, and are active in cells, as shown by increased acetylation at SIRT6 target lysines on histone 3, reduced TNF-α secretion, GLUT-1 upregulation, and increased glucose uptake.  Taken together, these results show the value of these compds. as starting leads for the development of new SIRT6-targeting therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvxmWnUj7yPrVg90H21EOLACvtfcHk0lhveEeHyTuvUg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1Cnsr4%253D&md5=f06f63bdfca06247ed5880deb50c5b13</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjm500487d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500487d%26sid%3Dliteratum%253Aachs%26aulast%3DParenti%26aufirst%3DM.%2BD.%26aulast%3DGrozio%26aufirst%3DA.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DGaleno%26aufirst%3DL.%26aulast%3DDamonte%26aufirst%3DP.%26aulast%3DMillo%26aufirst%3DE.%26aulast%3DSociali%26aufirst%3DG.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DRio%26aufirst%3DA.%2BD.%26aulast%3DNencioni%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520Novel%2520and%2520Selective%2520SIRT6%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4796%26epage%3D4804%26doi%3D10.1021%2Fjm500487d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Smadbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zee, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garapaty, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mago, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">De Novo Peptide Design and Experimental Validation of Histone Methyltransferase Inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e90095</span><span class="refDoi"> DOI: 10.1371/journal.pone.0090095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1371%2Fjournal.pone.0090095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24587223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e90095&author=J.+Smadbeckauthor=M.+B.+Petersonauthor=B.+M.+Zeeauthor=S.+Garapatyauthor=A.+Magoauthor=C.+Leeauthor=A.+Giannisauthor=P.+Trojerauthor=B.+A.+Garciaauthor=C.+A.+Floudas&title=De+Novo+Peptide+Design+and+Experimental+Validation+of+Histone+Methyltransferase+Inhibitors&doi=10.1371%2Fjournal.pone.0090095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">De novo peptide design and experimental validation of histone methyltransferase inhibitors</span></div><div class="casAuthors">Smadbeck, James; Peterson, Meghan B.; Zee, Barry M.; Garapaty, Shivani; Mago, Aashna; Lee, Christina; Giannis, Athanassios; Trojer, Patrick; Garcia, Benjamin A.; Floudas, Christodoulos A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e90095/1-e90095/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histones are small proteins crit. to the efficient packaging of DNA in the nucleus.  DNA-protein complexes, known as nucleosomes, are formed when the DNA winds itself around the surface of the histones.  The methylation of histone residues by enhancer of zeste homolog 2 (EZH2) maintains gene repression over successive cell generations.  Overexpression of EZH2 can silence important tumor suppressor genes leading to increased invasiveness of many types of cancers.  This makes the inhibition of EZH2 an important target in the development of cancer therapeutics.  We employed a three-stage computational de novo peptide design method to design inhibitory peptides of EZH2.  The method consists of a sequence selection stage and two validation stages for fold specificity and approx. binding affinity.  The sequence selection stage consists of an integer linear optimization model that was solved to produce a rank-ordered list of amino acid sequences with increased stability in the bound peptide-EZH2 structure.  These sequences were validated through the calcn. of the fold specificity and approx. binding affinity of the designed peptides.  Here we report the discovery of novel EZH2 inhibitory peptides using the de novo peptide design method.  The computationally discovered peptides were exptl. validated in vitro using dose titrns. and mechanism of action enzymic assays.  The peptide with the highest in vitro response, SQ037, was validated in nucleo using quant. mass spectrometry-based proteomics.  This peptide had an IC50 of 13.5 μM, demonstrated greater potency as an inhibitor when compared to the native and K27A mutant control peptides, and demonstrated competitive inhibition vs. the peptide substrate.  Addnl., this peptide demonstrated high specificity to the EZH2 target in comparison to other histone methyltransferases.  The validated peptides are the first computationally designed peptides that directly inhibit EZH2.  These inhibitors should prove useful for further chromatin biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-1ULxuWX0bbVg90H21EOLACvtfcHk0lg9DAGBJfMm_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjtL%252FM&md5=af0e5f85357302b0550d6352d449e2a2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090095%26sid%3Dliteratum%253Aachs%26aulast%3DSmadbeck%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DM.%2BB.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DGarapaty%26aufirst%3DS.%26aulast%3DMago%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DDe%2520Novo%2520Peptide%2520Design%2520and%2520Experimental%2520Validation%2520of%2520Histone%2520Methyltransferase%2520Inhibitors%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De90095%26doi%3D10.1371%2Fjournal.pone.0090095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span> </span><span class="NLM_article-title">Enhancing the Sensitivity of Pharmacophore-Based Virtual Screening by Incorporating Customized ZBG Features: A Case Study Using Histone Deacetylase 8</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/ci500762z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500762z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVegu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=861-71&author=X.+Houauthor=J.+Duauthor=R.+Liuauthor=Y.+Zhouauthor=M.+Liauthor=W.+Xuauthor=H.+Fang&title=Enhancing+the+Sensitivity+of+Pharmacophore-Based+Virtual+Screening+by+Incorporating+Customized+ZBG+Features%3A+A+Case+Study+Using+Histone+Deacetylase+8&doi=10.1021%2Fci500762z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing the Sensitivity of Pharmacophore-Based Virtual Screening by Incorporating Customized ZBG Features: A Case Study Using Histone Deacetylase 8</span></div><div class="casAuthors">Hou, Xuben; Du, Jintong; Liu, Renshuai; Zhou, Yi; Li, Minyong; Xu, Wenfang; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">861-871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As key regulators of epigenetic regulation, human histone deacetylases (HDACs) have been identified as drug targets for the treatment of several cancers.  The proper recognition of zinc-binding groups (ZBGs) will help improve the accuracy of virtual screening for novel HDAC inhibitors.  Here, we developed a high-specificity ZBG-based pharmacophore model for HDAC8 inhibitors by incorporating customized ZBG features.  Subsequently, pharmacophore-based virtual screening led to the discovery of three novel HDAC8 inhibitors with low micromole IC50 values (1.8-1.9 μM).  Further studies demonstrated that compd. H8-A5 was selective for HDAC8 over HDAC 1/4 and showed antiproliferation activity in MDA-MB-231 cancer cells.  Mol. docking and mol. dynamic studies suggested a possible binding mode for H8-A5, which provides a good starting point for the development of HDAC8 inhibitors in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4Iu2hmeMJF7Vg90H21EOLACvtfcHk0lg9DAGBJfMm_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVegu7g%253D&md5=06371ebb6d627ba47e81db3af2376bb3</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci500762z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500762z%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DEnhancing%2520the%2520Sensitivity%2520of%2520Pharmacophore-Based%2520Virtual%2520Screening%2520by%2520Incorporating%2520Customized%2520ZBG%2520Features%253A%2520A%2520Case%2520Study%2520Using%2520Histone%2520Deacetylase%25208%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D861%26epage%3D71%26doi%3D10.1021%2Fci500762z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective Inhibitors of Protein Methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+Inhibitors+of+Protein+Methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0lggsW-9Yc3a0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520Inhibitors%2520of%2520Protein%2520Methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Li, G.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, M.</span><span> </span><span class="NLM_article-title">Virtual screening in small molecule discovery for epigenetic targets</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">158</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2014.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.ymeth.2014.11.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=158-166&author=G.-B.+Liauthor=L.-L.+Yangauthor=Y.+Yuanauthor=J.+Zouauthor=Y.+Caoauthor=S.-Y.+Yangauthor=R.+Xiangauthor=M.+Xiang&title=Virtual+screening+in+small+molecule+discovery+for+epigenetic+targets&doi=10.1016%2Fj.ymeth.2014.11.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DYang%26aufirst%3DL.-L.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-Y.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DXiang%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520in%2520small%2520molecule%2520discovery%2520for%2520epigenetic%2520targets%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D158%26epage%3D166%26doi%3D10.1016%2Fj.ymeth.2014.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendez-Lucio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duenas-Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, J.</span><span> </span><span class="NLM_article-title">Discovery and development of DNA methyltransferase inhibitors using in silico approaches</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.drudis.2014.12.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=569-577&author=J.+L.+Medina-Francoauthor=O.+Mendez-Lucioauthor=A.+Duenas-Gonzalezauthor=J.+Yoo&title=Discovery+and+development+of+DNA+methyltransferase+inhibitors+using+in+silico+approaches&doi=10.1016%2Fj.drudis.2014.12.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMendez-Lucio%26aufirst%3DO.%26aulast%3DDuenas-Gonzalez%26aufirst%3DA.%26aulast%3DYoo%26aufirst%3DJ.%26atitle%3DDiscovery%2520and%2520development%2520of%2520DNA%2520methyltransferase%2520inhibitors%2520using%2520in%2520silico%2520approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D569%26epage%3D577%26doi%3D10.1016%2Fj.drudis.2014.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kathy Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-Based Virtual Screening and Biological Evaluation of Small Molecule Inhibitors for Protein Arginine Methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span><span class="refDoi"> DOI: 10.1021/jm300521m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.-W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=K.+Kathy+Liauthor=M.-F.+Linauthor=H.+Jiangauthor=Y.+George+Zhengauthor=C.+Luo&title=Pharmacophore-Based+Virtual+Screening+and+Biological+Evaluation+of+Small+Molecule+Inhibitors+for+Protein+Arginine+Methylation&doi=10.1021%2Fjm300521m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.-W.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKathy%2BLi%26aufirst%3DK.%26aulast%3DLin%26aufirst%3DM.-F.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGeorge%2BZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-Based%2520Virtual%2520Screening%2520and%2520Biological%2520Evaluation%2520of%2520Small%2520Molecule%2520Inhibitors%2520for%2520Protein%2520Arginine%2520Methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987%26doi%3D10.1021%2Fjm300521m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9028</span><span class="NLM_x">–</span> <span class="NLM_lpage">9041</span><span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.-C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+Novel+Selective+Non-Nucleoside+DNA+Methyltransferase+1+Inhibitors+through+Docking-Based+Virtual+Screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lh5QH1jx63_jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520Novel%2520Selective%2520Non-Nucleoside%2520DNA%2520Methyltransferase%25201%2520Inhibitors%2520through%2520Docking-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Kong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Astemizole arrests the proliferation of cancer cells by disrupting the EZH2-EED interaction of polycomb repressive complex 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9512</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1021/jm501230c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501230c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9512-21&author=X.+Kongauthor=L.+Chenauthor=L.+Jiaoauthor=X.+Jiangauthor=F.+Lianauthor=J.+Luauthor=K.+Zhuauthor=D.+Duauthor=J.+Liuauthor=H.+Dingauthor=N.+Zhangauthor=J.+Shenauthor=M.+Zhengauthor=K.+Chenauthor=X.+Liuauthor=H.+Jiangauthor=C.+Luo&title=Astemizole+arrests+the+proliferation+of+cancer+cells+by+disrupting+the+EZH2-EED+interaction+of+polycomb+repressive+complex+2&doi=10.1021%2Fjm501230c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Astemizole Arrests the Proliferation of Cancer Cells by Disrupting the EZH2-EED Interaction of Polycomb Repressive Complex 2</span></div><div class="casAuthors">Kong, Xiangqian; Chen, Limin; Jiao, Lianying; Jiang, Xiangrui; Lian, Fulin; Lu, Junyan; Zhu, Kongkai; Du, Daohai; Liu, Jingqiu; Ding, Hong; Zhang, Naixia; Shen, Jingshan; Zheng, Mingyue; Chen, Kaixian; Liu, Xin; Jiang, Hualiang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9512-9521</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb Repressive Complex 2 (PRC2) modulates the chromatin structure and transcriptional repression by trimethylation lysine 27 of histone H3 (H3K27me3), a process that necessitates the protein-protein interaction (PPI) between the catalytic subunit EZH2 and EED.  Deregulated PRC2 is intimately involved in tumorigenesis and progression, making it an invaluable target for epigenetic cancer therapy.  However, until now, there have been no reported small mol. compds. targeting the EZH2-EED interactions.  In the present study, we identified astemizole, an FDA-approved drug, as a small mol. inhibitor of the EZH2-EED interaction of PRC2.  The disruption of the EZH2-EED interaction by astemizole destabilizes the PRC2 complex and inhibits its methyltransferase activity in cancer cells.  Multiple lines of evidence have demonstrated that astemizole arrests the proliferation of PRC2-driven lymphomas primarily by disabling the PRC2 complex.  Our findings demonstrate the chem. tractability of the difficult PPI target by a small mol. compd., highlighting the therapeutic promise for PRC2-driven human cancers via targeted destruction of the EZH2-EED complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoHR0GppdWLVg90H21EOLACvtfcHk0lh5QH1jx63_jQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO&md5=9aa407fe354ba7e272bdfb6c6445f7f3</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjm501230c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501230c%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DAstemizole%2520arrests%2520the%2520proliferation%2520of%2520cancer%2520cells%2520by%2520disrupting%2520the%2520EZH2-EED%2520interaction%2520of%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9512%26epage%3D21%26doi%3D10.1021%2Fjm501230c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery of two aminoglycoside antibiotics as inhibitors targeting the menin-mixed lineage leukaemia interface</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2090</span><span class="NLM_x">–</span> <span class="NLM_lpage">2093</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.03.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmcl.2014.03.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24709560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=2090-2093&author=L.+Liauthor=R.+Zhouauthor=H.+Gengauthor=L.+Yueauthor=F.+Yeauthor=Y.+Xieauthor=J.+Liuauthor=X.+Kongauthor=H.+Jiangauthor=J.+Huangauthor=C.+Luo&title=Discovery+of+two+aminoglycoside+antibiotics+as+inhibitors+targeting+the+menin-mixed+lineage+leukaemia+interface&doi=10.1016%2Fj.bmcl.2014.03.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of two aminoglycoside antibiotics as inhibitors targeting the menin-mixed lineage leukaemia interface</span></div><div class="casAuthors">Li, Lianchun; Zhou, Ran; Geng, Heji; Yue, Liyan; Ye, Fei; Xie, Yiqian; Liu, Jingqiu; Kong, Xiangqian; Jiang, Hualiang; Huang, Jiandong; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2090-2093</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Menin functions as an oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in leukaemogenesis.  The menin-MLL interface is a potential therapeutic target in acute leukemia cases.  In this study, approx. 900 clin. compds. were evaluated and ranked using pharmacophore-based virtual screening, the top 29 hits were further evaluated by biochem. anal. to discover the inhibitors that target the menin-MLL interface.  Two aminoglycoside antibiotics, neomycin and tobramycin, were identified as menin-MLL inhibitors with binding affinities of 18.8 and 59.9 μM, resp.  The results of thermal shift assay validated the direct interactions between the two antibiotics and menin.  The results of isothermal titrn. calorimetry showed that the equil. dissocn. const. between menin and neomycin was approx. 15.6 μM.  We also predicted the binding modes of inhibitors at the menin-MLL interface through mol. docking anal.  The results indicated that neomycin and tobramycin competitively occupy the binding site of MLL.  This study has shed light on the development of powerful probes and new therapies for MLL-mediated leukaemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfV27zM7Fs5LVg90H21EOLACvtfcHk0lj3jcmaMzMFrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsLY%253D&md5=22b9b5de24bf1ff0f874ea018b6961cf</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.03.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.03.055%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520two%2520aminoglycoside%2520antibiotics%2520as%2520inhibitors%2520targeting%2520the%2520menin-mixed%2520lineage%2520leukaemia%2520interface%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D2090%26epage%3D2093%26doi%3D10.1016%2Fj.bmcl.2014.03.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">9665</span><span class="NLM_x">–</span> <span class="NLM_lpage">9673</span><span class="refDoi"> DOI: 10.1039/C4OB01591F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1039%2FC4OB01591F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25348815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=9665-9673&author=Y.+Xieauthor=R.+Zhouauthor=F.+Lianauthor=Y.+Liuauthor=L.+Chenauthor=Z.+Shiauthor=N.+Zhangauthor=M.+Zhengauthor=B.+Shenauthor=H.+Jiangauthor=Z.+Liangauthor=C.+Luo&title=Virtual+screening+and+biological+evaluation+of+novel+small+molecular+inhibitors+against+protein+arginine+methyltransferase+1+%28PRMT1%29&doi=10.1039%2FC4OB01591F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span></div><div class="casAuthors">Xie, Yiqian; Zhou, Ran; Lian, Fulin; Liu, Yan; Chen, Limin; Shi, Zhe; Zhang, Naixia; Zheng, Mingyue; Shen, Bairong; Jiang, Hualiang; Liang, Zhongjie; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">9665-9673</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification which is crucial for a variety of biol. processes.  Dysregulation of protein arginine methyltransferases (PRMTs) activity has been implicated in cancer and other serious diseases.  Thus, small mol. inhibitors against PRMT have great potential for therapeutic development.  Herein, through the combination of virtual screening and bioassays, six small mol. compds. were identified as PRMT1 inhibitors.  Amongst them, the binding affinity of compds. DCLX069 and DCLX078 with PRMT1 was further validated by T1ρ and satn. transfer difference (STD) NMR expts.  Most important of all, both compds. effectively blocked cell proliferation in breast cancer, liver cancer and acute myeloid leukemia cell lines.  The binding mode anal. from mol. docking simulations theor. indicated that both inhibitors occupied the SAM binding pocket to exert the inhibitory effect.  Taken together, our compds. enriched the structural scaffolds as PRMT1 inhibitors and afforded clues for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrinhNjy5ctKbVg90H21EOLACvtfcHk0lj3jcmaMzMFrg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F&md5=36cb4c315dad033dc9f10a1f21609ab4</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2FC4OB01591F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01591F%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520novel%2520small%2520molecular%2520inhibitors%2520against%2520protein%2520arginine%2520methyltransferase%25201%2520%2528PRMT1%2529%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D9665%26epage%3D9673%26doi%3D10.1039%2FC4OB01591F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">McInnes, C.</span><span> </span><span class="NLM_article-title">Virtual screening strategies in drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">494</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2007.08.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.cbpa.2007.08.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17936059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=494-502&author=C.+McInnes&title=Virtual+screening+strategies+in+drug+discovery&doi=10.1016%2Fj.cbpa.2007.08.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening strategies in drug discovery</span></div><div class="casAuthors">McInnes, Campbell</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">494-502</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The identification of novel therapeutic targets and characterization of their 3D structures is increasing at a dramatic rate.  Computational screening methods continue to be developed and improved as credible and complementary alternatives to high-throughput biochem. compd. screening (HTS).  While the majority of drug candidates currently being developed have been found using HTS methods, high-throughput docking and pharmacophore-based searching algorithms are gaining acceptance and becoming a major source of lead mols. in drug discovery.  Refinements and optimization of high-throughput docking methods have lead to improvements in reproducing exptl. data and in hit rates obtained, validating their use in hit identification.  In parallel with virtual screening methods, concomitant developments in cheminformatics including identification, design and manipulation of drug-like small mol. libraries have been achieved.  Herein, currently used in silico screening techniques and their utility on a comparative and target dependent basis is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3OTWaeATxk7Vg90H21EOLACvtfcHk0lhaIJk1kdxiVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrN&md5=8299cff4fa5156ddcdbf489cec894f7e</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.08.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.08.033%26sid%3Dliteratum%253Aachs%26aulast%3DMcInnes%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520strategies%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D494%26epage%3D502%26doi%3D10.1016%2Fj.cbpa.2007.08.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lhaIJk1kdxiVA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Baell, J. B.</span><span> </span><span class="NLM_article-title">Observations on screening-based research and some concerning trends in the literature</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1529</span><span class="NLM_x">–</span> <span class="NLM_lpage">1546</span><span class="refDoi"> DOI: 10.4155/fmc.10.237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4155%2Ffmc.10.237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21426147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1529-1546&author=J.+B.+Baell&title=Observations+on+screening-based+research+and+some+concerning+trends+in+the+literature&doi=10.4155%2Ffmc.10.237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Observations on screening-based research and some concerning trends in the literature</span></div><div class="casAuthors">Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1529-1546</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Academic drug discovery is being accompanied by a plethora of publications that report screening hits as good starting points for drug discovery or as useful tool compds., whereas in many cases this is not so.  These compds. may be protein-reactive but can also interfere in bioassays via a no. of other means, and it can be very hard to prove early on that they represent false starts.  This, for instance, makes it difficult for journals in their assessment of manuscripts submitted for publication.  Wider awareness and recognition of these problematic compds. will help the academic drug-discovery community focus on and publish genuinely optimizable screening hits.  This will be of general benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob88J0NH5Mx7Vg90H21EOLACvtfcHk0lgbtbn-2f5AdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjI&md5=14f916535540fc04e37c2eb21cfe7876</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.237%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DObservations%2520on%2520screening-based%2520research%2520and%2520some%2520concerning%2520trends%2520in%2520the%2520literature%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1529%26epage%3D1546%26doi%3D10.4155%2Ffmc.10.237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0lgbtbn-2f5AdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref90"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref90'); return false;" data-citation="" class="refNumLink">90</a></strong><div class="NLM_citation" id="cit90"><span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">Growing PAINS in academic drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">797</span><span class="NLM_x">–</span> <span class="NLM_lpage">801</span><span class="refDoi"> DOI: 10.4155/fmc.11.44</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4155%2Ffmc.11.44" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21644825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntV2jtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=797-801&author=A.+Whitty&title=Growing+PAINS+in+academic+drug+discovery&doi=10.4155%2Ffmc.11.44"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Growing PAINS in academic drug discovery</span></div><div class="casAuthors">Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">797-801</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">In a recent article it was argued that compds. published as drug leads by academic labs. commonly contain functionality that identifies them as nonspecific   pan-assay interference compds.'' (PAINS).  The article raises broad questions about why best practices for hit and lead qualification that are well known in industry are not more widely employed in academia, as well as about the role of journals in publishing manuscripts that report drug leads of little potential value.  Barriers to adoption of best practices for some academic drug-discovery researchers include knowledge gaps and infrastructure deficiencies, but they also arise from fundamental differences in how academic research is structured and how success is measured.  Academic drug discovery should not seek to become identical to com. pharmaceutical research, but we can do a better job of assessing and communicating the true potential of the drug leads we publish, thereby reducing the wastage of resources on nonviable compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHu052Dqa3DLVg90H21EOLACvtfcHk0ljDV83ktPNPTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntV2jtbo%253D&md5=3a773915ce7c352436eb9e2f4b8c6f4d</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.44&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.44%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DGrowing%2520PAINS%2520in%2520academic%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D797%26epage%3D801%26doi%3D10.4155%2Ffmc.11.44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref91"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref91'); return false;" data-citation="" class="refNumLink">91</a></strong><div class="NLM_citation" id="cit91"><span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span> </span><span class="NLM_article-title">Chemistry: Chemical con artists foil drug discovery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">481</span><span class="refDoi"> DOI: 10.1038/513481a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2F513481a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25254460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=481&author=J.+Baellauthor=M.+A.+Walters&title=Chemistry%3A+Chemical+con+artists+foil+drug+discovery&doi=10.1038%2F513481a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry: Chemical con artists foil drug discovery</span></div><div class="casAuthors">Baell, Jonathan; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7519</span>),
    <span class="NLM_cas:pages">481-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naivety about promiscuous, assay-duping mols. is polluting the literature and wasting resources, warn Jonathan Baell and Michael A. Walters.  Academic drug discoverers must be more vigilant.  Mols. that show the strongest activity in screening might not be the best starting points for drugs.  PAINS hits should almost always be ignored.  Even trained medicinal chemists have to be careful until they become experienced in screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFYQ0B4VVoLVg90H21EOLACvtfcHk0ljDV83ktPNPTA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M&md5=f5788d81d006460c688e4b78e6200503</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1038%2F513481a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F513481a%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DChemistry%253A%2520Chemical%2520con%2520artists%2520foil%2520drug%2520discovery%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D481%26doi%3D10.1038%2F513481a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref92"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref92'); return false;" data-citation="" class="refNumLink">92</a></strong><div class="NLM_citation" id="cit92"><span>LigPrep, version <span class="NLM_edition">2.3</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=LigPrep%2C+version+2.3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcommun%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref93"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref93'); return false;" data-citation="" class="refNumLink">93</a></strong><div class="NLM_citation" id="cit93"><span><span class="NLM_contrib-group">Shelley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cholleti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timlin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchimaya, M.</span><span> </span><span class="NLM_article-title">Epik: a software program for pK a prediction and protonation state generation for drug-like molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">681</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+Shelleyauthor=A.+Cholletiauthor=L.+Fryeauthor=J.+Greenwoodauthor=M.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0liW0IDp_yQtYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DTimlin%26aufirst%3DM.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref94"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref94'); return false;" data-citation="" class="refNumLink">94</a></strong><div class="NLM_citation" id="cit94"><span>Accelrys Discovery Studio 3.0; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Accelrys+Discovery+Studio+3.0%3B+Accelrys%3A+San+Diego%2C+CA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAccelrys%2520Discovery%2520Studio%25203.0%26pub%3DAccelrys%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref95"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref95'); return false;" data-citation="" class="refNumLink">95</a></strong><div class="NLM_citation" id="cit95"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehlers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma-O’Brien, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirchmair, J.</span><span> </span><span class="NLM_article-title">CAESAR: A New Conformer Generation Algorithm Based on Recursive Buildup and Local Rotational Symmetry Consideration</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1932</span><span class="refDoi"> DOI: 10.1021/ci700136x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700136x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1923-1932&author=J.+Liauthor=T.+Ehlersauthor=J.+Sutterauthor=S.+Varma-O%E2%80%99Brienauthor=J.+Kirchmair&title=CAESAR%3A+A+New+Conformer+Generation+Algorithm+Based+on+Recursive+Buildup+and+Local+Rotational+Symmetry+Consideration&doi=10.1021%2Fci700136x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">CAESAR: A New Conformer Generation Algorithm Based on Recursive Buildup and Local Rotational Symmetry Consideration</span></div><div class="casAuthors">Li, Jiabo; Ehlers, Tedman; Sutter, Jon; Varma-O'Brien, Shikha; Kirchmair, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1923-1932</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly efficient conformer search algorithm based on a divide-and-conquer and recursive conformer build-up approach is presented in this paper.  This approach is combined with consideration of local rotational symmetry so that conformer duplicates due to topol. symmetry in the systematic search can be efficiently eliminated.  This new algorithm, termed CAESAR (Conformer Algorithm based on Energy Screening and Recursive Buildup), has been implemented in Discovery Studio 1.7 as part of the Catalyst Component Collection.  CAESAR has been validated by comparing the conformer models generated by the new method and Catalyst/FAST.  CAESAR is consistently 5-20 times faster than Catalyst/FAST for all data sets investigated.  The speedup is even more dramatic for mols. with high topol. symmetry or for mols. that require a large no. of conformers to be sampled.  The quality of the conformer models generated by CAESAR has been validated by assessing the ability to reproduce the receptor-bound x-ray conformation of ligands extd. for the Protein Data Bank (PDB) and assessing the ability to adequately cover the pharmacophore space.  It is shown that CAESAR is able to reproduce the receptor-bound conformation slightly better than the Catalyst/FAST method for a data set of 918 ligands retrieved from the PDB.  In addn., it is shown that CAESAR covers the pharmacophore space as well or better than Catalyst/FAST.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd5K-SQHljYLVg90H21EOLACvtfcHk0liW0IDp_yQtYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOntLk%253D&md5=6e19e920c60c2944cc75034aee860440</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fci700136x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700136x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DEhlers%26aufirst%3DT.%26aulast%3DSutter%26aufirst%3DJ.%26aulast%3DVarma-O%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DKirchmair%26aufirst%3DJ.%26atitle%3DCAESAR%253A%2520A%2520New%2520Conformer%2520Generation%2520Algorithm%2520Based%2520on%2520Recursive%2520Buildup%2520and%2520Local%2520Rotational%2520Symmetry%2520Consideration%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1923%26epage%3D1932%26doi%3D10.1021%2Fci700136x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref96"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref96'); return false;" data-citation="" class="refNumLink">96</a></strong><div class="NLM_citation" id="cit96"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lj19crEfwzLuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref97"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref97'); return false;" data-citation="" class="refNumLink">97</a></strong><div class="NLM_citation" id="cit97"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein–Ligand Complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lj19crEfwzLuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref98"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref98'); return false;" data-citation="" class="refNumLink">98</a></strong><div class="NLM_citation" id="cit98"><span>Maestro, version <span class="NLM_edition">9.0</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro%2C+version+9.0%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref99"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref99'); return false;" data-citation="" class="refNumLink">99</a></strong><div class="NLM_citation" id="cit99"><span><span class="NLM_contrib-group">Siebel, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, A. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Osta, A.</span><span> </span><span class="NLM_article-title">Glycemic memory associated epigenetic changes</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">1853</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.bcp.2010.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bcp.2010.06.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2010&pages=1853-9&author=A.+L.+Siebelauthor=A.+Z.+Fernandezauthor=A.+El-Osta&title=Glycemic+memory+associated+epigenetic+changes&doi=10.1016%2Fj.bcp.2010.06.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DSiebel%26aufirst%3DA.%2BL.%26aulast%3DFernandez%26aufirst%3DA.%2BZ.%26aulast%3DEl-Osta%26aufirst%3DA.%26atitle%3DGlycemic%2520memory%2520associated%2520epigenetic%2520changes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D80%26spage%3D1853%26epage%3D9%26doi%3D10.1016%2Fj.bcp.2010.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref100"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref100'); return false;" data-citation="" class="refNumLink">100</a></strong><div class="NLM_citation" id="cit100"><span><span class="NLM_contrib-group">Yang, X.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.-F.</span><span> </span><span class="NLM_article-title">Negative regulation of NF-kappa B action by Set9-mediated lysine methylation of the RelA subunit</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1055</span><span class="NLM_x">–</span> <span class="NLM_lpage">1066</span><span class="refDoi"> DOI: 10.1038/emboj.2009.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Femboj.2009.55" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1055-1066&author=X.-D.+Yangauthor=B.+Huangauthor=M.+Liauthor=A.+Lambauthor=N.+L.+Kelleherauthor=L.-F.+Chen&title=Negative+regulation+of+NF-kappa+B+action+by+Set9-mediated+lysine+methylation+of+the+RelA+subunit&doi=10.1038%2Femboj.2009.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Femboj.2009.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2009.55%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DLamb%26aufirst%3DA.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DNegative%2520regulation%2520of%2520NF-kappa%2520B%2520action%2520by%2520Set9-mediated%2520lysine%2520methylation%2520of%2520the%2520RelA%2520subunit%26jtitle%3DEMBO%2520J.%26date%3D2009%26volume%3D28%26spage%3D1055%26epage%3D1066%26doi%3D10.1038%2Femboj.2009.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref101"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref101'); return false;" data-citation="" class="refNumLink">101</a></strong><div class="NLM_citation" id="cit101"><span><span class="NLM_contrib-group">Stelzer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalah, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satanower, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nativ, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oz-Levi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olender, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belinky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahir, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krug, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perco, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolker, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Safran, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancet, D.</span><span> </span><span class="NLM_article-title">In-silico human genomics with GeneCards</span> <span class="citation_source-journal">Hum. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">709</span><span class="NLM_x">–</span> <span class="NLM_lpage">717</span><span class="refDoi"> DOI: 10.1186/1479-7364-5-6-709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1186%2F1479-7364-5-6-709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22155609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVGrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=709-717&author=G.+Stelzerauthor=I.+Dalahauthor=T.+I.+Steinauthor=Y.+Satanowerauthor=N.+Rosenauthor=N.+Nativauthor=D.+Oz-Leviauthor=T.+Olenderauthor=F.+Belinkyauthor=I.+Bahirauthor=H.+Krugauthor=P.+Percoauthor=B.+Mayerauthor=E.+Kolkerauthor=M.+Safranauthor=D.+Lancet&title=In-silico+human+genomics+with+GeneCards&doi=10.1186%2F1479-7364-5-6-709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">In-silico human genomics with GeneCards</span></div><div class="casAuthors">Stelzer, Gil; Dalah, Irina; Stein, Tsippi Iny; Satanower, Yigeal; Rosen, Naomi; Nativ, Noam; Oz-Levi, Danit; Olender, Tsviya; Belinky, Frida; Bahir, Iris; Krug, Hagit; Perco, Paul; Mayer, Bernd; Lancet, Doron</div><div class="citationInfo"><span class="NLM_cas:title">Human Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">709-717</span>CODEN:
                <span class="NLM_cas:coden">HGUEAT</span>;
        ISSN:<span class="NLM_cas:issn">1479-7364</span>.
    
            (<span class="NLM_cas:orgname">Henry Stewart Publications</span>)
        </div><div class="casAbstract">Since 1998, the bioinformatics, systems biol., genomics and medical communities have enjoyed a synergistic relationship with the GeneCards database of human genes (http://www.genecards.org).  This human gene compendium was created to help to introduce order into the increasing chaos of information flow.  As a consequence of viewing details and deep links related to specific genes, users have often requested enhanced capabilities, such that, over time, GeneCards has blossomed into a suite of tools (including GeneDecks, GeneALaCart, GeneLoc, GeneNote and GeneAnnot) for a variety of analyses of both single human genes and sets thereof.  In this paper, we focus on inhouse and external research activities which have been enabled, enhanced, complemented and, in some cases, motivated by GeneCards.  In turn, such interactions have often inspired and propelled improvements in GeneCards.  We describe here the evolution and architecture of this project, including examples of synergistic applications in diverse areas such as synthetic lethality in cancer, the annotation of genetic variations in disease, omics integration in a systems biol. approach to kidney disease, and bioinformatics tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvTEBrDIxsxbVg90H21EOLACvtfcHk0lgX192zenjYlA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVGrtro%253D&md5=c36536c407de69bb50de1d765b746c81</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1186%2F1479-7364-5-6-709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-7364-5-6-709%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DG.%26aulast%3DDalah%26aufirst%3DI.%26aulast%3DStein%26aufirst%3DT.%2BI.%26aulast%3DSatanower%26aufirst%3DY.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DNativ%26aufirst%3DN.%26aulast%3DOz-Levi%26aufirst%3DD.%26aulast%3DOlender%26aufirst%3DT.%26aulast%3DBelinky%26aufirst%3DF.%26aulast%3DBahir%26aufirst%3DI.%26aulast%3DKrug%26aufirst%3DH.%26aulast%3DPerco%26aufirst%3DP.%26aulast%3DMayer%26aufirst%3DB.%26aulast%3DKolker%26aufirst%3DE.%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DLancet%26aufirst%3DD.%26atitle%3DIn-silico%2520human%2520genomics%2520with%2520GeneCards%26jtitle%3DHum.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D709%26epage%3D717%26doi%3D10.1186%2F1479-7364-5-6-709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref102"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref102'); return false;" data-citation="" class="refNumLink">102</a></strong><div class="NLM_citation" id="cit102"><span><span class="NLM_contrib-group">Belinky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahir, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stelzer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmerman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nativ, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalah, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iny Stein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mituyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Safran, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancet, D.</span><span> </span><span class="NLM_article-title">Non-redundant compendium of human ncRNA genes in GeneCards</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fbioinformatics%2Fbts676" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=255-61&author=F.+Belinkyauthor=I.+Bahirauthor=G.+Stelzerauthor=S.+Zimmermanauthor=N.+Rosenauthor=N.+Nativauthor=I.+Dalahauthor=T.+Iny+Steinauthor=N.+Rappaportauthor=T.+Mituyamaauthor=M.+Safranauthor=D.+Lancet&title=Non-redundant+compendium+of+human+ncRNA+genes+in+GeneCards&doi=10.1093%2Fbioinformatics%2Fbts676"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts676%26sid%3Dliteratum%253Aachs%26aulast%3DBelinky%26aufirst%3DF.%26aulast%3DBahir%26aufirst%3DI.%26aulast%3DStelzer%26aufirst%3DG.%26aulast%3DZimmerman%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DNativ%26aufirst%3DN.%26aulast%3DDalah%26aufirst%3DI.%26aulast%3DIny%2BStein%26aufirst%3DT.%26aulast%3DRappaport%26aufirst%3DN.%26aulast%3DMituyama%26aufirst%3DT.%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DLancet%26aufirst%3DD.%26atitle%3DNon-redundant%2520compendium%2520of%2520human%2520ncRNA%2520genes%2520in%2520GeneCards%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D255%26epage%3D61%26doi%3D10.1093%2Fbioinformatics%2Fbts676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref103"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref103'); return false;" data-citation="" class="refNumLink">103</a></strong><div class="NLM_citation" id="cit103"><span><span class="NLM_contrib-group">Oesterreich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, N. E.</span><span> </span><span class="NLM_article-title">The search for ESR1 mutations in breast cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1415</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1038/ng.2831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fng.2831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24270445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1415-6&author=S.+Oesterreichauthor=N.+E.+Davidson&title=The+search+for+ESR1+mutations+in+breast+cancer&doi=10.1038%2Fng.2831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">The search for ESR1 mutations in breast cancer</span></div><div class="casAuthors">Oesterreich, Steffi; Davidson, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1415-1416</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Two new studies report the identification of activating ESR1 gene mutations in aromatase inhibitor-resistant metastatic breast cancers.  This insight into therapeutic resistance suggests new approaches that may be useful in the management of endocrine-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFpXltTLXBR7Vg90H21EOLACvtfcHk0lik2vXvhxNbkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL7P&md5=e9cf29471564b577f3f8bcb1ee83f7b7</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fng.2831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2831%26sid%3Dliteratum%253Aachs%26aulast%3DOesterreich%26aufirst%3DS.%26aulast%3DDavidson%26aufirst%3DN.%2BE.%26atitle%3DThe%2520search%2520for%2520ESR1%2520mutations%2520in%2520breast%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1415%26epage%3D6%26doi%3D10.1038%2Fng.2831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref104"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref104'); return false;" data-citation="" class="refNumLink">104</a></strong><div class="NLM_citation" id="cit104"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lik2vXvhxNbkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref105"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref105'); return false;" data-citation="" class="refNumLink">105</a></strong><div class="NLM_citation" id="cit105"><span>Glide, version <span class="NLM_edition">5.5</span>; <span class="NLM_publisher-name">Schrödinger, Inc.</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide%2C+version+5.5%3B+Schr%C3%B6dinger%2C+Inc.%3A+New+York%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref106"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref106'); return false;" data-citation="" class="refNumLink">106</a></strong><div class="NLM_citation" id="cit106"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-61&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lik2vXvhxNbkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D61%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref107"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref107'); return false;" data-citation="" class="refNumLink">107</a></strong><div class="NLM_citation" id="cit107"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">DOCK 6: Impact of new features and current docking performance</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1132</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-56&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0lhj9_ErtUFUgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D56%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref108"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref108'); return false;" data-citation="" class="refNumLink">108</a></strong><div class="NLM_citation" id="cit108"><span><span class="NLM_contrib-group">Hamid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotshteyn, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabadi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, P.</span><span> </span><span class="NLM_article-title">Comparison of alamar blue and MTT assays for high through-put screening</span> <span class="citation_source-journal">Toxicol. In Vitro</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">703</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.tiv.2004.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.tiv.2004.03.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=703-10&author=R.+Hamidauthor=Y.+Rotshteynauthor=L.+Rabadiauthor=R.+Parikhauthor=P.+Bullock&title=Comparison+of+alamar+blue+and+MTT+assays+for+high+through-put+screening&doi=10.1016%2Fj.tiv.2004.03.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2004.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2004.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DHamid%26aufirst%3DR.%26aulast%3DRotshteyn%26aufirst%3DY.%26aulast%3DRabadi%26aufirst%3DL.%26aulast%3DParikh%26aufirst%3DR.%26aulast%3DBullock%26aufirst%3DP.%26atitle%3DComparison%2520of%2520alamar%2520blue%2520and%2520MTT%2520assays%2520for%2520high%2520through-put%2520screening%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2004%26volume%3D18%26spage%3D703%26epage%3D10%26doi%3D10.1016%2Fj.tiv.2004.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref109"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref109'); return false;" data-citation="" class="refNumLink">109</a></strong><div class="NLM_citation" id="cit109"><span><span class="NLM_contrib-group">Al-Nasiry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geusens, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanssens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luyten, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pijnenborg, R.</span><span> </span><span class="NLM_article-title">The use of Alamar Blue assay for quantitative analysis of viability, migration and invasion of choriocarcinoma cells</span> <span class="citation_source-journal">Hum. Reprod.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1093/humrep/dem011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fhumrep%2Fdem011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17307808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntlWms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2007&pages=1304-9&author=S.+Al-Nasiryauthor=N.+Geusensauthor=M.+Hanssensauthor=C.+Luytenauthor=R.+Pijnenborg&title=The+use+of+Alamar+Blue+assay+for+quantitative+analysis+of+viability%2C+migration+and+invasion+of+choriocarcinoma+cells&doi=10.1093%2Fhumrep%2Fdem011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">The use of Alamar Blue assay for quantitative analysis of viability, migration and invasion of choriocarcinoma cells</span></div><div class="casAuthors">Al-Nasiry, S.; Geusens, N.; Hanssens, M.; Luyten, C.; Pijnenborg, R.</div><div class="citationInfo"><span class="NLM_cas:title">Human Reproduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1304-1309</span>CODEN:
                <span class="NLM_cas:coden">HUREEE</span>;
        ISSN:<span class="NLM_cas:issn">0268-1161</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The current techniques for quantifying trophoblast viability, migration and invasion are mainly limited by the need to sacrifice the cells during the test procedure.  In this study, the vital dye AB was used to quantify cell no. and viability of BeWo and JEG-3 choriocarcinoma cells, as well as their migration and invasion through fibronectin-coated filters.  AB was directly added to culture medium of incubated test and control cells.  At various time intervals, the redox reaction, in which AB is reduced by the cells, was measured by absorbance readings at 540 and 630 nm.  For cell migration and invasion, cells were cultured onto uncoated or fibronectin-coated inserts, resp.  AB redn. of migrated cells was normalized to that of control cells to calc. percentages of migration.  This model was also tested in the presence of a reported inhibitor, transforming growth factor (TGF) β.  The curve of %AB redn. vs. cell no. was linear, with intra- and inter-assay Coeff. of Variations of 1.88% and 2.94%, resp.  AB redn. increased with both seeding concns. and incubation time with AB.  TGFβ treatment caused a modest decrease in AB redn. in both JEG-3 and BeWo cells.  TGFβ treatment also decreased migration in BeWo, but not in JEG-3, cells.  Thus, AB assay is a simple and reliable method for quantifying trophoblast viability, migration and invasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkaPtJrigCLVg90H21EOLACvtfcHk0lhj9_ErtUFUgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXntlWms7g%253D&md5=4ccf0048bb87d9c5d7667aa709d1b3f7</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1093%2Fhumrep%2Fdem011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhumrep%252Fdem011%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Nasiry%26aufirst%3DS.%26aulast%3DGeusens%26aufirst%3DN.%26aulast%3DHanssens%26aufirst%3DM.%26aulast%3DLuyten%26aufirst%3DC.%26aulast%3DPijnenborg%26aufirst%3DR.%26atitle%3DThe%2520use%2520of%2520Alamar%2520Blue%2520assay%2520for%2520quantitative%2520analysis%2520of%2520viability%252C%2520migration%2520and%2520invasion%2520of%2520choriocarcinoma%2520cells%26jtitle%3DHum.%2520Reprod.%26date%3D2007%26volume%3D22%26spage%3D1304%26epage%3D9%26doi%3D10.1093%2Fhumrep%2Fdem011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref110"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref110'); return false;" data-citation="" class="refNumLink">110</a></strong><div class="NLM_citation" id="cit110"><span>The PyMOL Molecular Graphics System, version <span class="NLM_edition">1.3r1</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+PyMOL+Molecular+Graphics+System%2C+version+1.3r1%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89'],'ref90':['cit90'],'ref91':['cit91'],'ref92':['cit92'],'ref93':['cit93'],'ref94':['cit94'],'ref95':['cit95'],'ref96':['cit96'],'ref97':['cit97'],'ref98':['cit98'],'ref99':['cit99'],'ref100':['cit100'],'ref101':['cit101'],'ref102':['cit102'],'ref103':['cit103'],'ref104':['cit104'],'ref105':['cit105'],'ref106':['cit106'],'ref107':['cit107'],'ref108':['cit108'],'ref109':['cit109'],'ref110':['cit110']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 43 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Fei Ye, Weiyao Zhang, Xiaoqing Ye, Jia Jin, Zhengbing Lv, <span class="NLM_string-name hlFld-ContribAuthor">Cheng Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Selective, Cell Active Inhibitors of Protein Arginine Methyltransferase 5 through Structure-Based Virtual Screening and Biological Assays. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 1066-1073. <a href="https://doi.org/10.1021/acs.jcim.8b00050" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00050%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BSelective%25252C%252BCell%252BActive%252BInhibitors%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B5%252Bthrough%252BStructure-Based%252BVirtual%252BScreening%252Band%252BBiological%252BAssays%26aulast%3DYe%26aufirst%3DFei%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01022018%26date%3D26042018%26date%3D19042018%26volume%3D58%26issue%3D5%26spage%3D1066%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ruifeng  Mao</span>, <span class="hlFld-ContribAuthor ">Jingwei  Shao</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Chenhua  Zhang</span>, <span class="hlFld-ContribAuthor ">Zhe  Shi</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Dequn  Sun</span>, <span class="hlFld-ContribAuthor ">Wenhu  Duan</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (14)
                                     , 6289-6304. <a href="https://doi.org/10.1021/acs.jmedchem.7b00587" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00587</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DPotent%25252C%252BSelective%25252C%252Band%252BCell%252BActive%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BInhibitor%252BDeveloped%252Bby%252BStructure-Based%252BVirtual%252BScreening%252Band%252BHit%252BOptimization%26aulast%3DMao%26aufirst%3DRuifeng%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D18042017%26date%3D12072017%26date%3D27072017%26date%3D26062017%26volume%3D60%26issue%3D14%26spage%3D6289%26epage%3D6304" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">John J.  Irwin</span> and <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span>  . </span><span class="cited-content_cbyCitation_article-title">Docking Screens for Novel Ligands Conferring New Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4103-4120. <a href="https://doi.org/10.1021/acs.jmedchem.5b02008" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b02008</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b02008&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b02008%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDocking%252BScreens%252Bfor%252BNovel%252BLigands%252BConferring%252BNew%252BBiology%26aulast%3DIrwin%26aufirst%3DJohn%2BJ.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D25122015%26date%3D15032016%26date%3D12052016%26date%3D25022016%26volume%3D59%26issue%3D9%26spage%3D4103%26epage%3D4120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yasushi  Takemoto</span>, <span class="hlFld-ContribAuthor ">Akihiro  Ito</span>, <span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Mutsumi  Okamura</span>, <span class="hlFld-ContribAuthor ">Takashi  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Noriko  Handa</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>, <span class="hlFld-ContribAuthor ">Takeshi  Sonoda</span>, <span class="hlFld-ContribAuthor ">Kenji  Ogawa</span>, <span class="hlFld-ContribAuthor ">Mohammad  Tariq</span>, <span class="hlFld-ContribAuthor ">Norikazu  Nishino</span>, <span class="hlFld-ContribAuthor ">Shingo  Dan</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>, <span class="hlFld-ContribAuthor ">Takao  Yamori</span>, <span class="hlFld-ContribAuthor ">Shigeyuki  Yokoyama</span>, and <span class="hlFld-ContribAuthor ">Minoru  Yoshida</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of Cyproheptadine as an Inhibitor of SET Domain Containing Lysine Methyltransferase 7/9 (Set7/9) That Regulates Estrogen-Dependent Transcription. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (8)
                                     , 3650-3660. <a href="https://doi.org/10.1021/acs.jmedchem.5b01732" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01732%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BCyproheptadine%252Bas%252Ban%252BInhibitor%252Bof%252BSET%252BDomain%252BContaining%252BLysine%252BMethyltransferase%252B7%25252F9%252B%252528Set7%25252F9%252529%252BThat%252BRegulates%252BEstrogen-Dependent%252BTranscription%26aulast%3DTakemoto%26aufirst%3DYasushi%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D06112015%26date%3D18042016%26date%3D28042016%26volume%3D59%26issue%3D8%26spage%3D3650%26epage%3D3660" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Shijie  Chen</span>, <span class="hlFld-ContribAuthor ">Linjuan  Li</span>, <span class="hlFld-ContribAuthor ">Yantao  Chen</span>, <span class="hlFld-ContribAuthor ">Junchi  Hu</span>, <span class="hlFld-ContribAuthor ">Jingqiu  Liu</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Junyan  Lu</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Jin  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Zhiyi  Yao</span>, and <span class="hlFld-ContribAuthor ">Cheng  Luo</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of Novel Disruptor of Telomeric Silencing 1-like (DOT1L) Inhibitors through Structure-Based Virtual Screening and Biological Assays. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (3)
                                     , 527-534. <a href="https://doi.org/10.1021/acs.jcim.5b00738" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00738</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00738%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BNovel%252BDisruptor%252Bof%252BTelomeric%252BSilencing%252B1-like%252B%252528DOT1L%252529%252BInhibitors%252Bthrough%252BStructure-Based%252BVirtual%252BScreening%252Band%252BBiological%252BAssays%26aulast%3DChen%26aufirst%3DShijie%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D17122015%26date%3D04032016%26date%3D28032016%26date%3D25022016%26volume%3D56%26issue%3D3%26spage%3D527%26epage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guijae  Yoo</span>, <span class="hlFld-ContribAuthor ">Jun Hyung  Park</span>, <span class="hlFld-ContribAuthor ">Seung Hyun  Kim</span>. </span><span class="cited-content_cbyCitation_article-title">Neuraminidase inhibitory diarylheptanoids from Alpinia officinarum: In vitro and molecular docking studies. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2021,</strong> <em>107 </em>, 104526. <a href="https://doi.org/10.1016/j.bioorg.2020.104526" title="DOI URL">https://doi.org/10.1016/j.bioorg.2020.104526</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2020.104526&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2020.104526%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNeuraminidase%252Binhibitory%252Bdiarylheptanoids%252Bfrom%252BAlpinia%252Bofficinarum%25253A%252BIn%252Bvitro%252Band%252Bmolecular%252Bdocking%252Bstudies%26aulast%3DYoo%26aufirst%3DGuijae%26date%3D2021%26volume%3D107%26spage%3D104526" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Poonam  Dhankhar</span>, <span class="hlFld-ContribAuthor ">Vikram  Dalal</span>, <span class="hlFld-ContribAuthor ">Vishakha  Singh</span>, <span class="hlFld-ContribAuthor ">Shailly  Tomar</span>, <span class="hlFld-ContribAuthor ">Pravindra  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Computational guided identification of novel potent inhibitors of N-terminal domain of nucleocapsid protein of severe acute respiratory syndrome coronavirus 2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biomolecular Structure and Dynamics</span><span> <strong>2020,</strong> <em>40 </em>, 1-16. <a href="https://doi.org/10.1080/07391102.2020.1852968" title="DOI URL">https://doi.org/10.1080/07391102.2020.1852968</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/07391102.2020.1852968&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F07391102.2020.1852968%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biomolecular%2520Structure%2520and%2520Dynamics%26atitle%3DComputational%252Bguided%252Bidentification%252Bof%252Bnovel%252Bpotent%252Binhibitors%252Bof%252BN-terminal%252Bdomain%252Bof%252Bnucleocapsid%252Bprotein%252Bof%252Bsevere%252Bacute%252Brespiratory%252Bsyndrome%252Bcoronavirus%252B2%26aulast%3DDhankhar%26aufirst%3DPoonam%26date%3D2020%26date%3D2020%26volume%3D40%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Meijia  Qian</span>, <span class="hlFld-ContribAuthor ">Fangjie  Yan</span>, <span class="hlFld-ContribAuthor ">Tao  Yuan</span>, <span class="hlFld-ContribAuthor ">Bo  Yang</span>, <span class="hlFld-ContribAuthor ">Qiaojun  He</span>, <span class="hlFld-ContribAuthor ">Hong  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting post-translational modification of transcription factors as cancer therapy. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2020,</strong> <em>25 </em>
                                    (8)
                                     , 1502-1512. <a href="https://doi.org/10.1016/j.drudis.2020.06.005" title="DOI URL">https://doi.org/10.1016/j.drudis.2020.06.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2020.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2020.06.005%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DTargeting%252Bpost-translational%252Bmodification%252Bof%252Btranscription%252Bfactors%252Bas%252Bcancer%252Btherapy%26aulast%3DQian%26aufirst%3DMeijia%26date%3D2020%26volume%3D25%26issue%3D8%26spage%3D1502%26epage%3D1512" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleni  Pitsillou</span>, <span class="hlFld-ContribAuthor ">Julia  Liang</span>, <span class="hlFld-ContribAuthor ">Andrew  Hung</span>, <span class="hlFld-ContribAuthor ">Tom C.  Karagiannis</span>. </span><span class="cited-content_cbyCitation_article-title">Chromatin modification by olive phenolics: In silico molecular docking studies utilising the phenolic groups categorised in the OliveNet™ database against lysine specific demethylase enzymes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2020,</strong> <em>97 </em>, 107575. <a href="https://doi.org/10.1016/j.jmgm.2020.107575" title="DOI URL">https://doi.org/10.1016/j.jmgm.2020.107575</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2020.107575&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2020.107575%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DChromatin%252Bmodification%252Bby%252Bolive%252Bphenolics%25253A%252BIn%252Bsilico%252Bmolecular%252Bdocking%252Bstudies%252Butilising%252Bthe%252Bphenolic%252Bgroups%252Bcategorised%252Bin%252Bthe%252BOliveNet%2525E2%252584%2525A2%252Bdatabase%252Bagainst%252Blysine%252Bspecific%252Bdemethylase%252Benzymes%26aulast%3DPitsillou%26aufirst%3DEleni%26date%3D2020%26volume%3D97%26spage%3D107575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laia  Richart</span>, <span class="hlFld-ContribAuthor ">Raphaël  Margueron</span>. </span><span class="cited-content_cbyCitation_article-title">Drugging histone methyltransferases in cancer. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>56 </em>, 51-62. <a href="https://doi.org/10.1016/j.cbpa.2019.11.009" title="DOI URL">https://doi.org/10.1016/j.cbpa.2019.11.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2019.11.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2019.11.009%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDrugging%252Bhistone%252Bmethyltransferases%252Bin%252Bcancer%26aulast%3DRichart%26aufirst%3DLaia%26date%3D2020%26volume%3D56%26spage%3D51%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zeng  Hou</span>, <span class="hlFld-ContribAuthor ">Wenjian  Min</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Ao  Niu</span>, <span class="hlFld-ContribAuthor ">Yuanqing  Li</span>, <span class="hlFld-ContribAuthor ">Liyuan  Cao</span>, <span class="hlFld-ContribAuthor ">Jie  Han</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Peng  Yang</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>. </span><span class="cited-content_cbyCitation_article-title">Lead discovery, chemical optimization, and biological evaluation studies of novel histone methyltransferase SET7 small-molecule inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (9)
                                     , 127061. <a href="https://doi.org/10.1016/j.bmcl.2020.127061" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127061%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLead%252Bdiscovery%25252C%252Bchemical%252Boptimization%25252C%252Band%252Bbiological%252Bevaluation%252Bstudies%252Bof%252Bnovel%252Bhistone%252Bmethyltransferase%252BSET7%252Bsmall-molecule%252Binhibitors%26aulast%3DHou%26aufirst%3DZeng%26date%3D2020%26volume%3D30%26issue%3D9%26spage%3D127061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenjian  Min</span>, <span class="hlFld-ContribAuthor ">Zeng  Hou</span>, <span class="hlFld-ContribAuthor ">Fang  Zhang</span>, <span class="hlFld-ContribAuthor ">Shengnan  Xie</span>, <span class="hlFld-ContribAuthor ">Kai  Yuan</span>, <span class="hlFld-ContribAuthor ">Haojie  Dong</span>, <span class="hlFld-ContribAuthor ">Liping  Wang</span>, <span class="hlFld-ContribAuthor ">Lianwen  Qi</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Peng  Yang</span>. </span><span class="cited-content_cbyCitation_article-title">Computational discovery and biological evaluation of novel inhibitors targeting histone-lysine N-methyltransferase SET7. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2020,</strong> <em>28 </em>
                                    (7)
                                     , 115372. <a href="https://doi.org/10.1016/j.bmc.2020.115372" title="DOI URL">https://doi.org/10.1016/j.bmc.2020.115372</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2020.115372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2020.115372%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DComputational%252Bdiscovery%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Binhibitors%252Btargeting%252Bhistone-lysine%252BN-methyltransferase%252BSET7%26aulast%3DMin%26aufirst%3DWenjian%26date%3D2020%26volume%3D28%26issue%3D7%26spage%3D115372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhe  Wang</span>, <span class="hlFld-ContribAuthor ">Huiyong  Sun</span>, <span class="hlFld-ContribAuthor ">Chao  Shen</span>, <span class="hlFld-ContribAuthor ">Xueping  Hu</span>, <span class="hlFld-ContribAuthor ">Junbo  Gao</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Cao</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">Combined strategies in structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2020,</strong> <em>22 </em>
                                    (6)
                                     , 3149-3159. <a href="https://doi.org/10.1039/C9CP06303J" title="DOI URL">https://doi.org/10.1039/C9CP06303J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CP06303J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CP06303J%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DCombined%252Bstrategies%252Bin%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DWang%26aufirst%3DZhe%26date%3D2020%26date%3D2020%26volume%3D22%26issue%3D6%26spage%3D3149%26epage%3D3159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ahmed H.E.  Hassan</span>, <span class="hlFld-ContribAuthor ">Kyung-Tae  Lee</span>, <span class="hlFld-ContribAuthor ">Yong Sup  Lee</span>. </span><span class="cited-content_cbyCitation_article-title">Flavone-based arylamides as potential anticancers: Design, synthesis and in vitro cell-based/cell-free evaluations. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>187 </em>, 111965. <a href="https://doi.org/10.1016/j.ejmech.2019.111965" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.111965</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.111965&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.111965%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFlavone-based%252Barylamides%252Bas%252Bpotential%252Banticancers%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bin%2525C2%2525A0vitro%252Bcell-based%25252Fcell-free%252Bevaluations%26aulast%3DHassan%26aufirst%3DAhmed%2BH.E.%26date%3D2020%26volume%3D187%26spage%3D111965" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yunjiang  Zhou</span>, <span class="hlFld-ContribAuthor ">Shi  Tang</span>, <span class="hlFld-ContribAuthor ">Tingting  Chen</span>, <span class="hlFld-ContribAuthor ">Miao-Miao  Niu</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Pharmacophore Modeling, Virtual Screening, Molecular Docking and Biological Evaluation for Identification of Potential Poly (ADP-Ribose) Polymerase-1 (PARP-1) Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (23)
                                     , 4258. <a href="https://doi.org/10.3390/molecules24234258" title="DOI URL">https://doi.org/10.3390/molecules24234258</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24234258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24234258%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DStructure-Based%252BPharmacophore%252BModeling%25252C%252BVirtual%252BScreening%25252C%252BMolecular%252BDocking%252Band%252BBiological%252BEvaluation%252Bfor%252BIdentification%252Bof%252BPotential%252BPoly%252B%252528ADP-Ribose%252529%252BPolymerase-1%252B%252528PARP-1%252529%252BInhibitors%26aulast%3DZhou%26aufirst%3DYunjiang%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D23%26spage%3D4258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangpu  Liu</span>, <span class="hlFld-ContribAuthor ">Yang  Jiao</span>, <span class="hlFld-ContribAuthor ">Chunxi  Huang</span>, <span class="hlFld-ContribAuthor ">Ping  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel and potent small-molecule inhibitors of tubulin with antitumor activities by virtual screening and biological evaluations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2019,</strong> <em>33 </em>
                                    (7)
                                     , 659-664. <a href="https://doi.org/10.1007/s10822-019-00206-y" title="DOI URL">https://doi.org/10.1007/s10822-019-00206-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00206-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00206-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIdentification%252Bof%252Bnovel%252Band%252Bpotent%252Bsmall-molecule%252Binhibitors%252Bof%252Btubulin%252Bwith%252Bantitumor%252Bactivities%252Bby%252Bvirtual%252Bscreening%252Band%252Bbiological%252Bevaluations%26aulast%3DLiu%26aufirst%3DGuangpu%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D7%26spage%3D659%26epage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K.  Rohini</span>, <span class="hlFld-ContribAuthor ">Roosha  Roy</span>, <span class="hlFld-ContribAuthor ">K.  Ramanathan</span>, <span class="hlFld-ContribAuthor ">V.  Shanthi</span>. </span><span class="cited-content_cbyCitation_article-title">E-pharmacophore hypothesis strategy to discover potent inhibitor for influenza treatment. </span><span class="cited-content_cbyCitation_journal-name">Journal of Theoretical and Computational Chemistry</span><span> <strong>2019,</strong> <em>18 </em>
                                    (04)
                                     , 1950021. <a href="https://doi.org/10.1142/S0219633619500214" title="DOI URL">https://doi.org/10.1142/S0219633619500214</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1142/S0219633619500214&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1142%2FS0219633619500214%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Theoretical%2520and%2520Computational%2520Chemistry%26atitle%3DE-pharmacophore%252Bhypothesis%252Bstrategy%252Bto%252Bdiscover%252Bpotent%252Binhibitor%252Bfor%252Binfluenza%252Btreatment%26aulast%3DRohini%26aufirst%3DK.%26date%3D2019%26date%3D2019%26volume%3D18%26issue%3D04%26spage%3D1950021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jie  Han</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Fulin  Lian</span>, <span class="hlFld-ContribAuthor ">Jun  Wang</span>, <span class="hlFld-ContribAuthor ">Tian  Lu</span>, <span class="hlFld-ContribAuthor ">Hongru  Tao</span>, <span class="hlFld-ContribAuthor ">Senhao  Xiao</span>, <span class="hlFld-ContribAuthor ">Fengcai  Zhang</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Naixia  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Chunshen  Zhao</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of alkoxy benzamide derivatives as novel BPTF bromodomain inhibitors via structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2019,</strong> <em>86 </em>, 494-500. <a href="https://doi.org/10.1016/j.bioorg.2019.01.035" title="DOI URL">https://doi.org/10.1016/j.bioorg.2019.01.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2019.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2019.01.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DDiscovery%252Bof%252Balkoxy%252Bbenzamide%252Bderivatives%252Bas%252Bnovel%252BBPTF%252Bbromodomain%252Binhibitors%252Bvia%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DZhang%26aufirst%3DDan%26date%3D2019%26volume%3D86%26spage%3D494%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bowen  Tang</span>, <span class="hlFld-ContribAuthor ">Baicun  Li</span>, <span class="hlFld-ContribAuthor ">Boqun  Li</span>, <span class="hlFld-ContribAuthor ">Jingbo  Qin</span>, <span class="hlFld-ContribAuthor ">Junming  Zhao</span>, <span class="hlFld-ContribAuthor ">Jianwenn  Xu</span>, <span class="hlFld-ContribAuthor ">Yingkun  Qiu</span>, <span class="hlFld-ContribAuthor ">Zhen  Wu</span>, <span class="hlFld-ContribAuthor ">Meijuan  Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Insights into the stereoselectivity of human SETD7 methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (16)
                                     , 9218-9227. <a href="https://doi.org/10.1039/C9RA00190E" title="DOI URL">https://doi.org/10.1039/C9RA00190E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9RA00190E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9RA00190E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DInsights%252Binto%252Bthe%252Bstereoselectivity%252Bof%252Bhuman%252BSETD7%252Bmethyltransferase%26aulast%3DTang%26aufirst%3DBowen%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D16%26spage%3D9218%26epage%3D9227" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Tino  Heimburg</span>, <span class="hlFld-ContribAuthor ">Berin  Karaman</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Computer‐based Lead Identification for Epigenetic Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 45-77. <a href="https://doi.org/10.1002/9783527809257.ch3" title="DOI URL">https://doi.org/10.1002/9783527809257.ch3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch3%26sid%3Dliteratum%253Aachs%26atitle%3DComputer%2525E2%252580%252590based%252BLead%252BIdentification%252Bfor%252BEpigenetic%252BTargets%26aulast%3DLuise%26aufirst%3DChiara%26date%3D2019%26date%3D2019%26spage%3D45%26epage%3D77%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongru  Tao</span>, <span class="hlFld-ContribAuthor ">Jun  Wang</span>, <span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Yiqian  Xie</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Rongfeng  Liu</span>, <span class="hlFld-ContribAuthor ">Liyan  Yue</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Xu</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of trisubstituted nicotinonitrile derivatives as novel human GCN5 inhibitors through AlphaScreen-based high throughput screening. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2019,</strong> <em>9 </em>
                                    (9)
                                     , 4917-4924. <a href="https://doi.org/10.1039/C8RA10074H" title="DOI URL">https://doi.org/10.1039/C8RA10074H</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8RA10074H&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8RA10074H%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DDiscovery%252Bof%252Btrisubstituted%252Bnicotinonitrile%252Bderivatives%252Bas%252Bnovel%252Bhuman%252BGCN5%252Binhibitors%252Bthrough%252BAlphaScreen-based%252Bhigh%252Bthroughput%252Bscreening%26aulast%3DTao%26aufirst%3DHongru%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D9%26spage%3D4917%26epage%3D4924" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Jelena  Melesina</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Based Design of Epigenetic Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 455-486. <a href="https://doi.org/10.1007/7355_2019_80" title="DOI URL">https://doi.org/10.1007/7355_2019_80</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_80&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_80%26sid%3Dliteratum%253Aachs%26atitle%3DStructure-Based%252BDesign%252Bof%252BEpigenetic%252BInhibitors%26aulast%3DRobaa%26aufirst%3DDina%26date%3D2019%26date%3D2019%26spage%3D455%26epage%3D486%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">K.  Rohini</span>, <span class="hlFld-ContribAuthor ">Pratika  Agarwal</span>, <span class="hlFld-ContribAuthor ">B.  Preethi</span>, <span class="hlFld-ContribAuthor ">V.  Shanthi</span>, <span class="hlFld-ContribAuthor ">K.  Ramanathan</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the Lead Compounds for Zika Virus NS2B-NS3 Protein: an e-Pharmacophore-Based Approach. </span><span class="cited-content_cbyCitation_journal-name">Applied Biochemistry and Biotechnology</span><span> <strong>2019,</strong> <em>187 </em>
                                    (1)
                                     , 194-210. <a href="https://doi.org/10.1007/s12010-018-2814-3" title="DOI URL">https://doi.org/10.1007/s12010-018-2814-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12010-018-2814-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12010-018-2814-3%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Biochemistry%2520and%2520Biotechnology%26atitle%3DExploring%252Bthe%252BLead%252BCompounds%252Bfor%252BZika%252BVirus%252BNS2B-NS3%252BProtein%25253A%252Ban%252Be-Pharmacophore-Based%252BApproach%26aulast%3DRohini%26aufirst%3DK.%26date%3D2019%26date%3D2018%26volume%3D187%26issue%3D1%26spage%3D194%26epage%3D210" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Shuichi  Mori</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Recent Advances in Chemical Tools for the Regulation and Study of Protein Lysine Methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1745-1759. <a href="https://doi.org/10.1002/tcr.201800034" title="DOI URL">https://doi.org/10.1002/tcr.201800034</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800034%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DRecent%252BAdvances%252Bin%252BChemical%252BTools%252Bfor%252Bthe%252BRegulation%252Band%252BStudy%252Bof%252BProtein%252BLysine%252BMethyltransferases%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1745%26epage%3D1759" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Chen</span>, <span class="hlFld-ContribAuthor ">Xiao  Lin</span>, <span class="hlFld-ContribAuthor ">Kyoung Jin  Park</span>, <span class="hlFld-ContribAuthor ">Kang Ro  Lee</span>, <span class="hlFld-ContribAuthor ">Hyun-Ju  Park</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of protoberberine alkaloids as novel histone methyltransferase G9a inhibitors by structure-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (9)
                                     , 917-928. <a href="https://doi.org/10.1007/s10822-018-0156-4" title="DOI URL">https://doi.org/10.1007/s10822-018-0156-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-018-0156-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-018-0156-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIdentification%252Bof%252Bprotoberberine%252Balkaloids%252Bas%252Bnovel%252Bhistone%252Bmethyltransferase%252BG9a%252Binhibitors%252Bby%252Bstructure-based%252Bvirtual%252Bscreening%26aulast%3DChen%26aufirst%3DJie%26date%3D2018%26date%3D2018%26volume%3D32%26issue%3D9%26spage%3D917%26epage%3D928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>, <span class="hlFld-ContribAuthor ">Takashi  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Hideaki  Niwa</span>, <span class="hlFld-ContribAuthor ">Michitake  Hirano</span>, <span class="hlFld-ContribAuthor ">Kasumi  Ohira</span>, <span class="hlFld-ContribAuthor ">Yusuke  Okazaki</span>, <span class="hlFld-ContribAuthor ">Shin  Sato</span>, <span class="hlFld-ContribAuthor ">Takashi  Umehara</span>, <span class="hlFld-ContribAuthor ">Yuki  Maemoto</span>, <span class="hlFld-ContribAuthor ">Akihiro  Ito</span>, <span class="hlFld-ContribAuthor ">Minoru  Yoshida</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>. </span><span class="cited-content_cbyCitation_article-title">Development of Novel Inhibitors for Histone Methyltransferase SET7/9 based on Cyproheptadine. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (15)
                                     , 1530-1540. <a href="https://doi.org/10.1002/cmdc.201800233" title="DOI URL">https://doi.org/10.1002/cmdc.201800233</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800233&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800233%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDevelopment%252Bof%252BNovel%252BInhibitors%252Bfor%252BHistone%252BMethyltransferase%252BSET7%25252F9%252Bbased%252Bon%252BCyproheptadine%26aulast%3DHirano%26aufirst%3DTomoya%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D15%26spage%3D1530%26epage%3D1540" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tewfik  Hamidi</span>, <span class="hlFld-ContribAuthor ">Anup Kumar  Singh</span>, <span class="hlFld-ContribAuthor ">Nicolas  Veland</span>, <span class="hlFld-ContribAuthor ">Vidyasiri  Vemulapalli</span>, <span class="hlFld-ContribAuthor ">Jianji  Chen</span>, <span class="hlFld-ContribAuthor ">Swanand  Hardikar</span>, <span class="hlFld-ContribAuthor ">Jianqiang  Bao</span>, <span class="hlFld-ContribAuthor ">Christopher J.  Fry</span>, <span class="hlFld-ContribAuthor ">Vicky  Yang</span>, <span class="hlFld-ContribAuthor ">Kimberly A.  Lee</span>, <span class="hlFld-ContribAuthor ">Ailan  Guo</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>, <span class="hlFld-ContribAuthor ">Mark T.  Bedford</span>, <span class="hlFld-ContribAuthor ">Taiping  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Rpl29 as a major substrate of the lysine methyltransferase Set7/9. </span><span class="cited-content_cbyCitation_journal-name">Journal of Biological Chemistry</span><span> <strong>2018,</strong> <em>293 </em>
                                    (33)
                                     , 12770-12780. <a href="https://doi.org/10.1074/jbc.RA118.002890" title="DOI URL">https://doi.org/10.1074/jbc.RA118.002890</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1074/jbc.RA118.002890&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1074%2Fjbc.RA118.002890%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Biological%2520Chemistry%26atitle%3DIdentification%252Bof%252BRpl29%252Bas%252Ba%252Bmajor%252Bsubstrate%252Bof%252Bthe%252Blysine%252Bmethyltransferase%252BSet7%25252F9%26aulast%3DHamidi%26aufirst%3DTewfik%26date%3D2018%26volume%3D293%26issue%3D33%26spage%3D12770%26epage%3D12780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danny C.  Lenstra</span>, <span class="hlFld-ContribAuthor ">Eddy  Damen</span>, <span class="hlFld-ContribAuthor ">Ruben G. G.  Leenders</span>, <span class="hlFld-ContribAuthor ">Richard H.  Blaauw</span>, <span class="hlFld-ContribAuthor ">Floris P. J. T.  Rutjes</span>, <span class="hlFld-ContribAuthor ">Anita  Wegert</span>, <span class="hlFld-ContribAuthor ">Jasmin  Mecinović</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Activity Relationship Studies on (
              R
              )-PFI-2 Analogues as Inhibitors of Histone Lysine Methyltransferase SETD7. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2018,</strong> <em>13 </em>
                                    (14)
                                     , 1405-1413. <a href="https://doi.org/10.1002/cmdc.201800242" title="DOI URL">https://doi.org/10.1002/cmdc.201800242</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201800242&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201800242%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DStructure-Activity%252BRelationship%252BStudies%252Bon%252B%252528%252BR%252B%252529-PFI-2%252BAnalogues%252Bas%252BInhibitors%252Bof%252BHistone%252BLysine%252BMethyltransferase%252BSETD7%26aulast%3DLenstra%26aufirst%3DDanny%2BC.%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D14%26spage%3D1405%26epage%3D1413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ryo  Tamura</span>, <span class="hlFld-ContribAuthor ">Shigehiro  Doi</span>, <span class="hlFld-ContribAuthor ">Ayumu  Nakashima</span>, <span class="hlFld-ContribAuthor ">Kensuke  Sasaki</span>, <span class="hlFld-ContribAuthor ">Kazuya  Maeda</span>, <span class="hlFld-ContribAuthor ">Toshinori  Ueno</span>, <span class="hlFld-ContribAuthor ">Takao  Masaki</span>, . </span><span class="cited-content_cbyCitation_article-title">Inhibition of the H3K4 methyltransferase SET7/9 ameliorates peritoneal fibrosis. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2018,</strong> <em>13 </em>
                                    (5)
                                     , e0196844. <a href="https://doi.org/10.1371/journal.pone.0196844" title="DOI URL">https://doi.org/10.1371/journal.pone.0196844</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0196844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0196844%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DInhibition%252Bof%252Bthe%252BH3K4%252Bmethyltransferase%252BSET7%25252F9%252Bameliorates%252Bperitoneal%252Bfibrosis%26aulast%3DTamura%26aufirst%3DRyo%26date%3D2018%26date%3D2018%26volume%3D13%26issue%3D5%26spage%3De0196844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenchao  Lu</span>, <span class="hlFld-ContribAuthor ">Rukang  Zhang</span>, <span class="hlFld-ContribAuthor ">Hao  Jiang</span>, <span class="hlFld-ContribAuthor ">Huimin  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Computer-Aided Drug Design in Epigenetics. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2018,</strong> <em>6 </em><a href="https://doi.org/10.3389/fchem.2018.00057" title="DOI URL">https://doi.org/10.3389/fchem.2018.00057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2018.00057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2018.00057%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DComputer-Aided%252BDrug%252BDesign%252Bin%252BEpigenetics%26aulast%3DLu%26aufirst%3DWenchao%26date%3D2018%26date%3D2018%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuela  Sabatino</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>, <span class="hlFld-ContribAuthor ">Alexandros  Patsilinakos</span>, <span class="hlFld-ContribAuthor ">Mariantonietta  Forgione</span>, <span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Fréderic  Alby</span>, <span class="hlFld-ContribAuthor ">Paola B.  Arimondo</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Rino  Ragno</span>. </span><span class="cited-content_cbyCitation_article-title">Disruptor of telomeric silencing 1-like (DOT1L): disclosing a new class of non-nucleoside inhibitors by means of ligand-based and structure-based approaches. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (3)
                                     , 435-458. <a href="https://doi.org/10.1007/s10822-018-0096-z" title="DOI URL">https://doi.org/10.1007/s10822-018-0096-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-018-0096-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-018-0096-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDisruptor%252Bof%252Btelomeric%252Bsilencing%252B1-like%252B%252528DOT1L%252529%25253A%252Bdisclosing%252Ba%252Bnew%252Bclass%252Bof%252Bnon-nucleoside%252Binhibitors%252Bby%252Bmeans%252Bof%252Bligand-based%252Band%252Bstructure-based%252Bapproaches%26aulast%3DSabatino%26aufirst%3DManuela%26date%3D2018%26date%3D2018%26volume%3D32%26issue%3D3%26spage%3D435%26epage%3D458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Wang</span>, <span class="hlFld-ContribAuthor ">Jiahui  Xu</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Jumin  Huang</span>, <span class="hlFld-ContribAuthor ">Zebo  Jiang</span>, <span class="hlFld-ContribAuthor ">Yuwei  Wang</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>, <span class="hlFld-ContribAuthor ">Elaine Lai Han  Leung</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Protein Arginine Methyltransferase 5 Inhibitor in Non-small Cell Lung Cancer by Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00173" title="DOI URL">https://doi.org/10.3389/fphar.2018.00173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00173%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BProtein%252BArginine%252BMethyltransferase%252B5%252BInhibitor%252Bin%252BNon-small%252BCell%252BLung%252BCancer%252Bby%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DWang%26aufirst%3DQianqian%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Wen  Lu</span>, <span class="hlFld-ContribAuthor ">Jun  Hu</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Chen  Zhang</span>, <span class="hlFld-ContribAuthor ">Fu  Lian</span>, <span class="hlFld-ContribAuthor ">Nai  Zhang</span>, <span class="hlFld-ContribAuthor ">Fan  Meng</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Kai  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Characterizations of Novel, Selective Histone Methyltransferase SET7 Inhibitors by Scaffold Hopping- and 2D-Molecular Fingerprint-Based Similarity Search. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2018,</strong> <em>23 </em>
                                    (3)
                                     , 567. <a href="https://doi.org/10.3390/molecules23030567" title="DOI URL">https://doi.org/10.3390/molecules23030567</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules23030567&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules23030567%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DIdentification%252Band%252BCharacterizations%252Bof%252BNovel%25252C%252BSelective%252BHistone%252BMethyltransferase%252BSET7%252BInhibitors%252Bby%252BScaffold%252BHopping-%252Band%252B2D-Molecular%252BFingerprint-Based%252BSimilarity%252BSearch%26aulast%3DDing%26aufirst%3DHong%26date%3D2018%26date%3D2018%26volume%3D23%26issue%3D3%26spage%3D567" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fangxia  Zou</span>, <span class="hlFld-ContribAuthor ">Yifei  Yang</span>, <span class="hlFld-ContribAuthor ">Tianfang  Ma</span>, <span class="hlFld-ContribAuthor ">Jiayue  Xi</span>, <span class="hlFld-ContribAuthor ">Jinpei  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiaoming  Zha</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel MEK1 inhibitors by pharmacophore and docking based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2017,</strong> <em>26 </em>
                                    (4)
                                     , 701-713. <a href="https://doi.org/10.1007/s00044-017-1788-y" title="DOI URL">https://doi.org/10.1007/s00044-017-1788-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-017-1788-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-017-1788-y%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DIdentification%252Bof%252Bnovel%252BMEK1%252Binhibitors%252Bby%252Bpharmacophore%252Band%252Bdocking%252Bbased%252Bvirtual%252Bscreening%26aulast%3DZou%26aufirst%3DFangxia%26date%3D2017%26date%3D2017%26volume%3D26%26issue%3D4%26spage%3D701%26epage%3D713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yan  Ye</span>, <span class="hlFld-ContribAuthor ">Bidong  Zhang</span>, <span class="hlFld-ContribAuthor ">Ruifeng  Mao</span>, <span class="hlFld-ContribAuthor ">Chenhua  Zhang</span>, <span class="hlFld-ContribAuthor ">Yulan  Wang</span>, <span class="hlFld-ContribAuthor ">Jing  Xing</span>, <span class="hlFld-ContribAuthor ">Yu-Chih  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaomin  Luo</span>, <span class="hlFld-ContribAuthor ">Hong  Ding</span>, <span class="hlFld-ContribAuthor ">Yaxi  Yang</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Mingyue  Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and optimization of selective inhibitors of protein arginine methyltransferase 5 by docking-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (17)
                                     , 3648-3661. <a href="https://doi.org/10.1039/C7OB00070G" title="DOI URL">https://doi.org/10.1039/C7OB00070G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB00070G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB00070G%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DDiscovery%252Band%252Boptimization%252Bof%252Bselective%252Binhibitors%252Bof%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252Bby%252Bdocking-based%252Bvirtual%252Bscreening%26aulast%3DYe%26aufirst%3DYan%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D17%26spage%3D3648%26epage%3D3661" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Leilei  Fu</span>, <span class="hlFld-ContribAuthor ">Shuya  Wang</span>, <span class="hlFld-ContribAuthor ">Xuan  Wang</span>, <span class="hlFld-ContribAuthor ">Peiqi  Wang</span>, <span class="hlFld-ContribAuthor ">Yaxin  Zheng</span>, <span class="hlFld-ContribAuthor ">Dahong  Yao</span>, <span class="hlFld-ContribAuthor ">Mingrui  Guo</span>, <span class="hlFld-ContribAuthor ">Lan  Zhang</span>, <span class="hlFld-ContribAuthor ">Liang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Crystal structure-based discovery of a novel synthesized PARP1 inhibitor (OL-1) with apoptosis-inducing mechanisms in triple-negative breast cancer. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2016,</strong> <em>6 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-016-0007-2" title="DOI URL">https://doi.org/10.1038/s41598-016-0007-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-016-0007-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-016-0007-2%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCrystal%252Bstructure-based%252Bdiscovery%252Bof%252Ba%252Bnovel%252Bsynthesized%252BPARP1%252Binhibitor%252B%252528OL-1%252529%252Bwith%252Bapoptosis-inducing%252Bmechanisms%252Bin%252Btriple-negative%252Bbreast%252Bcancer%26aulast%3DFu%26aufirst%3DLeilei%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yang  Zhou</span>, <span class="hlFld-ContribAuthor ">Tao  Nie</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Ming  Song</span>, <span class="hlFld-ContribAuthor ">Kuai  Li</span>, <span class="hlFld-ContribAuthor ">Mengxiao  Ding</span>, <span class="hlFld-ContribAuthor ">Ke  Ding</span>, <span class="hlFld-ContribAuthor ">Donghai  Wu</span>, <span class="hlFld-ContribAuthor ">Yong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">The discovery of novel and selective fatty acid binding protein 4 inhibitors by virtual screening and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (18)
                                     , 4310-4317. <a href="https://doi.org/10.1016/j.bmc.2016.07.022" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.07.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.07.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.07.022%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DThe%252Bdiscovery%252Bof%252Bnovel%252Band%252Bselective%252Bfatty%252Bacid%252Bbinding%252Bprotein%252B4%252Binhibitors%252Bby%252Bvirtual%252Bscreening%252Band%252Bbiological%252Bevaluation%26aulast%3DZhou%26aufirst%3DYang%26date%3D2016%26volume%3D24%26issue%3D18%26spage%3D4310%26epage%3D4317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Takashi  Fujiwara</span>, <span class="hlFld-ContribAuthor ">Kasumi  Ohira</span>, <span class="hlFld-ContribAuthor ">Ko  Urushibara</span>, <span class="hlFld-ContribAuthor ">Akihiro  Ito</span>, <span class="hlFld-ContribAuthor ">Minoru  Yoshida</span>, <span class="hlFld-ContribAuthor ">Misae  Kanai</span>, <span class="hlFld-ContribAuthor ">Aya  Tanatani</span>, <span class="hlFld-ContribAuthor ">Hiroyuki  Kagechika</span>, <span class="hlFld-ContribAuthor ">Tomoya  Hirano</span>. </span><span class="cited-content_cbyCitation_article-title">Steric structure–activity relationship of cyproheptadine derivatives as inhibitors of histone methyltransferase Set7/9. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2016,</strong> <em>24 </em>
                                    (18)
                                     , 4318-4323. <a href="https://doi.org/10.1016/j.bmc.2016.07.024" title="DOI URL">https://doi.org/10.1016/j.bmc.2016.07.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2016.07.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2016.07.024%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DSteric%252Bstructure%2525E2%252580%252593activity%252Brelationship%252Bof%252Bcyproheptadine%252Bderivatives%252Bas%252Binhibitors%252Bof%252Bhistone%252Bmethyltransferase%252BSet7%25252F9%26aulast%3DFujiwara%26aufirst%3DTakashi%26date%3D2016%26volume%3D24%26issue%3D18%26spage%3D4318%26epage%3D4323" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yu  Song</span>, <span class="hlFld-ContribAuthor ">Xiaoqian  Xue</span>, <span class="hlFld-ContribAuthor ">Xishan  Wu</span>, <span class="hlFld-ContribAuthor ">Rui  Wang</span>, <span class="hlFld-ContribAuthor ">Yanli  Xing</span>, <span class="hlFld-ContribAuthor ">Weiqun  Yan</span>, <span class="hlFld-ContribAuthor ">Yulai  Zhou</span>, <span class="hlFld-ContribAuthor ">Chao-Nan  Qian</span>, <span class="hlFld-ContribAuthor ">Yan  Zhang</span>, <span class="hlFld-ContribAuthor ">Yong  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of N -phenyl-2-( N -phenylphenylsulfonamido)acetamides as new RORγ inverse agonists: Virtual screening, structure-based optimization, and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>116 </em>, 13-26. <a href="https://doi.org/10.1016/j.ejmech.2016.03.052" title="DOI URL">https://doi.org/10.1016/j.ejmech.2016.03.052</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2016.03.052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2016.03.052%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIdentification%252Bof%252BN%252B-phenyl-2-%252528%252BN%252B-phenylphenylsulfonamido%252529acetamides%252Bas%252Bnew%252BROR%2525CE%2525B3%252Binverse%252Bagonists%25253A%252BVirtual%252Bscreening%25252C%252Bstructure-based%252Boptimization%25252C%252Band%252Bbiological%252Bevaluation%26aulast%3DSong%26aufirst%3DYu%26date%3D2016%26volume%3D116%26spage%3D13%26epage%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">B. M.  Hudson</span>, <span class="hlFld-ContribAuthor ">E.  Nguyen</span>, <span class="hlFld-ContribAuthor ">D. J.  Tantillo</span>. </span><span class="cited-content_cbyCitation_article-title">The influence of intramolecular sulfur–lone pair interactions on small-molecule drug design and receptor binding. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2016,</strong> <em>14 </em>
                                    (16)
                                     , 3975-3980. <a href="https://doi.org/10.1039/C6OB00254D" title="DOI URL">https://doi.org/10.1039/C6OB00254D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6OB00254D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6OB00254D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DThe%252Binfluence%252Bof%252Bintramolecular%252Bsulfur%2525E2%252580%252593lone%252Bpair%252Binteractions%252Bon%252Bsmall-molecule%252Bdrug%252Bdesign%252Band%252Breceptor%252Bbinding%26aulast%3DHudson%26aufirst%3DB.%2BM.%26date%3D2016%26date%3D2016%26volume%3D14%26issue%3D16%26spage%3D3975%26epage%3D3980" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Liyan  Yue</span>, <span class="hlFld-ContribAuthor ">Juanjuan  Du</span>, <span class="hlFld-ContribAuthor ">Fei  Ye</span>, <span class="hlFld-ContribAuthor ">Zhifeng  Chen</span>, <span class="hlFld-ContribAuthor ">Lianchun  Li</span>, <span class="hlFld-ContribAuthor ">Fulin  Lian</span>, <span class="hlFld-ContribAuthor ">Bidong  Zhang</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Yuanchao  Li</span>, <span class="hlFld-ContribAuthor ">Bing  Zhou</span>, <span class="hlFld-ContribAuthor ">Naixia  Zhang</span>, <span class="hlFld-ContribAuthor ">Yaxi  Yang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel small-molecule inhibitors targeting menin–MLL interaction, repurposing the antidiarrheal loperamide. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2016,</strong> <em>14 </em>
                                    (36)
                                     , 8503-8519. <a href="https://doi.org/10.1039/C6OB01248E" title="DOI URL">https://doi.org/10.1039/C6OB01248E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6OB01248E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6OB01248E%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DIdentification%252Bof%252Bnovel%252Bsmall-molecule%252Binhibitors%252Btargeting%252Bmenin%2525E2%252580%252593MLL%252Binteraction%25252C%252Brepurposing%252Bthe%252Bantidiarrheal%252Bloperamide%26aulast%3DYue%26aufirst%3DLiyan%26date%3D2016%26date%3D2016%26volume%3D14%26issue%3D36%26spage%3D8503%26epage%3D8519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_null" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0009.gif" alt="" id="_i1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=f_null"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Virtual screening procedures and activity assays for SET7 in vitro. (A) An integrated virtual screening procedure combining pharmacophore- and docking-based virtual screening. Numerals denote the number of molecules in each stage. (B) Inhibitory activity of the 127 candidate molecules based on virtual screening at 100 μM. The red columnar bar represents the activity of DC-S100, and the blue bar denotes the reference compound SAH. (C) Inhibitory activity of the 109 analogous compounds of DC-S100 selected from the similarity search, in which the inhibition rate was determined at 100 μM. The blue columnar bar represents the activity of the reference compound SAH.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Putative binding modes and activities of DC-S100 and DC-S237. (A) Binding mode of DC-S100 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). Carbon atoms of DC-S100 are displayed in yellow; nitrogen atoms are in blue and oxygen atoms in red. Key residues are shown as sticks, and carbon is in gray, nitrogen in blue, and oxygen in red. Key hydrogen bonds are shown as a red dashed line. All of the residues and compounds are labeled. (B) Binding mode of DC-S237 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). DC-S237 is shown as yellow sticks, and key interacting residues are shown as gray sticks. All of the oxygen atoms are in red, and nitrogen atoms are in blue. Hydrogen bonds between DC-S237 and SET7 are represented as a red dashed line. (C) Superimposition of the binding modes of DC-S100 and DC-S237 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). Both DC-S100 and DC-S237 share a similar binding conformation anchored in the cofactor binding pocket. DC-S237 is shown as blue sticks, and DC-S100 is depicted in cyan, with the surface of SET7 depicted in vacuum electrostatics. Key residue Lys-294 is highlighted. All of the binding mode figures were generated with PyMOL, version 1.3r1.<a onclick="showRef(event, 'ref110'); return false;" href="javascript:void(0);" class="ref ref110">(110)</a> (D) Inhibitory activities of DC-S100 and DC-S237 against SET7.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Activities of synthesized compounds. (A) Inhibitory activity of synthesized compounds. Activity data of reference compound SAH are represented as a blue columnar bar. (B) IC<sub>50</sub> curves of DC-S238 and DC-S239. The green sigmoidal curve graph depicts the activity profile of DC-S238 (IC<sub>50</sub> = 4.88 μM), and the pink sigmoidal curve graph displays that of DC-S239 (IC<sub>50</sub> = 4.59 μM).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cellular activities of DC-S239 against SET7 in different cell lines. (A) IC<sub>50</sub> curve of DC-S239 toward the MCF7 cell line. (B) IC<sub>50</sub> curve of DC-S239 toward the HL60 cell line. (C) Dose–response relationship of DC-S239 toward the MV4-11 cell line. (D) Activity of DC<b>-</b>S239 toward the HCT116 cell line. (E) Activity of DC-S239 toward the DHL4 cell. The antiproliferation activity of DC-S239 is determined by the MTT (MCF7, MV4-11) or alamarBlue assay (HL60, MV4-11, DHL4). The experimental data were analyzed with GraphPad Prism 5.0.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/medium/jm-2015-011546_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Predicted binding mode of DC-S239 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A">1N6A</a>). (A) Key interactions involved in stabilizing DC-S239 at the cofactor binding site. DC-S239 is depicted as yellow sticks, and key residues are labeled. Hydrogen bonds are shown as red lines, and the π–π stacking interaction involving residue Trp-352 is also highlighted. (B) Superimposition of the binding modes of DC-S239 and SAM. The structure of SET7 is displayed in vacuum electrostatics. DC-S239 is shown as yellow sticks, and SAM is displayed as gray sticks. Key residue Lys-294 is highlighted.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2015/jmcmar.2015.58.issue-20/acs.jmedchem.5b01154/20151016/images/large/jm-2015-011546_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.5b01154&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_36081" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_36081" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 110 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">49</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-49&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lgXWIqt4fOIgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D49%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Varier, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timmers, H. T.</span><span> </span><span class="NLM_article-title">Histone lysine methylation and demethylation pathways in cancer</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">1815</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">89</span><span class="refDoi"> DOI: 10.1016/j.bbcan.2010.10.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bbcan.2010.10.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20951770" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1815&publication_year=2011&pages=75-89&author=R.+A.+Varierauthor=H.+T.+Timmers&title=Histone+lysine+methylation+and+demethylation+pathways+in+cancer&doi=10.1016%2Fj.bbcan.2010.10.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Histone lysine methylation and demethylation pathways in cancer</span></div><div class="casAuthors">Varier, Radhika A.; Timmers, H. T. Marc</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Reviews on Cancer</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">1815</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-89</span>CODEN:
                <span class="NLM_cas:coden">BBACEU</span>;
        ISSN:<span class="NLM_cas:issn">0304-419X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The genetic changes leading to the development of human cancer are accompanied by alterations in the structure and modification status of chromatin, which represent powerful regulatory mechanisms for gene expression and genome stability.  These epigenetic alterations have sparked interest into deciphering the regulatory pathways and function of post-translational modifications of histones during the initiation and progression of cancer.  In this review we describe and summarize the current knowledge of several histone lysine methyltransferase and demethylase pathways relevant to cancer.  Mechanistic insight into histone modifications will pave the way for the development and therapeutic application of "epidrugs" in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp7MyfXQs2YR7Vg90H21EOLACvtfcHk0lgXWIqt4fOIgg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtF2lsA%253D%253D&md5=3331b133c4f1b15beab600420bb859f2</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.bbcan.2010.10.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbcan.2010.10.002%26sid%3Dliteratum%253Aachs%26aulast%3DVarier%26aufirst%3DR.%2BA.%26aulast%3DTimmers%26aufirst%3DH.%2BT.%26atitle%3DHistone%2520lysine%2520methylation%2520and%2520demethylation%2520pathways%2520in%2520cancer%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Rev.%2520Cancer%26date%3D2011%26volume%3D1815%26spage%3D75%26epage%3D89%26doi%3D10.1016%2Fj.bbcan.2010.10.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vasisht, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Structure and catalyticmechanism of the human histonemethyltransferase SET7/9</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">421</span><span class="NLM_x">, </span> <span class="NLM_fpage">652</span><span class="NLM_x">–</span> <span class="NLM_lpage">656</span><span class="refDoi"> DOI: 10.1038/nature01378</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnature01378" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=421&publication_year=2003&pages=652-656&author=B.+Xiaoauthor=C.+Jingauthor=J.+R.+Wilsonauthor=P.+A.+Walkerauthor=N.+Vasishtauthor=G.+Kellyauthor=S.+Howellauthor=I.+A.+Taylorauthor=G.+M.+Blackburnauthor=S.+J.+Gamblin&title=Structure+and+catalyticmechanism+of+the+human+histonemethyltransferase+SET7%2F9&doi=10.1038%2Fnature01378"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnature01378&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01378%26sid%3Dliteratum%253Aachs%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJing%26aufirst%3DC.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DVasisht%26aufirst%3DN.%26aulast%3DKelly%26aufirst%3DG.%26aulast%3DHowell%26aufirst%3DS.%26aulast%3DTaylor%26aufirst%3DI.%2BA.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructure%2520and%2520catalyticmechanism%2520of%2520the%2520human%2520histonemethyltransferase%2520SET7%252F9%26jtitle%3DNature%26date%3D2003%26volume%3D421%26spage%3D652%26epage%3D656%26doi%3D10.1038%2Fnature01378" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jing, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howell, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blackburn, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, B.</span><span> </span><span class="NLM_article-title">Crystal Structure and Functional Analysis of the Histone Methyltransferase SET7/9</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">105</span><span class="NLM_x">–</span> <span class="NLM_lpage">115</span><span class="refDoi"> DOI: 10.1016/S0092-8674(02)00964-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS0092-8674%2802%2900964-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2002&pages=105-115&author=J.+R.+Wilsonauthor=C.+Jingauthor=P.+A.+Walkerauthor=S.+R.+Martinauthor=S.+A.+Howellauthor=G.+M.+Blackburnauthor=S.+J.+Gamblinauthor=B.+Xiao&title=Crystal+Structure+and+Functional+Analysis+of+the+Histone+Methyltransferase+SET7%2F9&doi=10.1016%2FS0092-8674%2802%2900964-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1016%2FS0092-8674%2802%2900964-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0092-8674%252802%252900964-9%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DJing%26aufirst%3DC.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DHowell%26aufirst%3DS.%2BA.%26aulast%3DBlackburn%26aufirst%3DG.%2BM.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DXiao%26aufirst%3DB.%26atitle%3DCrystal%2520Structure%2520and%2520Functional%2520Analysis%2520of%2520the%2520Histone%2520Methyltransferase%2520SET7%252F9%26jtitle%3DCell%26date%3D2002%26volume%3D111%26spage%3D105%26epage%3D115%26doi%3D10.1016%2FS0092-8674%2802%2900964-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Kassner, I.; Andersson, A.; Fey, M.; Tomas, M.; Ferrando-May, E.; Hottiger, M. O.</span><span> </span><span class="NLM_article-title">SET7/9-dependent methylation of ARTD1 at K508 stimulates poly-ADP-ribose formation after oxidative stress</span>.  <span class="citation_source-journal">Open Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span><span class="refDoi"> DOI: 10.1098/rsob.120173</span> .</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1098%2Frsob.120173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24088713" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2013&author=I.+Kassner&author=A.+Andersson&author=M.+Fey&author=M.+Tomas&author=E.+Ferrando-May&author=M.+O.+Hottiger&title=SET7%2F9-dependent+methylation+of+ARTD1+at+K508+stimulates+poly-ADP-ribose+formation+after+oxidative+stress&doi=10.1098%2Frsob.120173"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1098%2Frsob.120173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1098%252Frsob.120173%26sid%3Dliteratum%253Aachs%26aulast%3DKassner%26aufirst%3DI.%26atitle%3DSET7%252F9-dependent%2520methylation%2520of%2520ARTD1%2520at%2520K508%2520stimulates%2520poly-ADP-ribose%2520formation%2520after%2520oxidative%2520stress%26jtitle%3DOpen%2520Biol.%26date%3D2013%26doi%3D10.1098%2Frsob.120173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Oppenheimer, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meir, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartz, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haze, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kandel, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mattan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liebergall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dvir-Ginzberg, M.</span><span> </span><span class="NLM_article-title">Set7/9 Impacts COL2A1 Expression Through Binding and Repression of SirT1 Histone Deacetylation</span> <span class="citation_source-journal">J. Bone Miner. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">348</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span><span class="refDoi"> DOI: 10.1002/jbmr.2052</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjbmr.2052" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=23873758" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ehsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2014&pages=348-360&author=H.+Oppenheimerauthor=A.+Kumarauthor=H.+Meirauthor=I.+Schwartzauthor=A.+Ziniauthor=A.+Hazeauthor=L.+Kandelauthor=Y.+Mattanauthor=M.+Liebergallauthor=M.+Dvir-Ginzberg&title=Set7%2F9+Impacts+COL2A1+Expression+Through+Binding+and+Repression+of+SirT1+Histone+Deacetylation&doi=10.1002%2Fjbmr.2052"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Set7/9 impacts COL2A1 expression through binding and repression of SirT1 histone deacetylation</span></div><div class="casAuthors">Oppenheimer, Hanna; Kumar, Ashok; Meir, Hadar; Schwartz, Israel; Zini, Avi; Haze, Amir; Kandel, Leonid; Mattan, Yoav; Liebergall, Meir; Dvir-Ginzberg, Mona</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Bone and Mineral Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">348-360</span>CODEN:
                <span class="NLM_cas:coden">JBMREJ</span>;
        ISSN:<span class="NLM_cas:issn">0884-0431</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Type II collagen is a key cartilaginous extracellular protein required for normal endochondral development and cartilage homeostasis.  COL2A1 gene expression is pos. regulated by the NAD-dependent protein deacetylase Sirtuin 1 (SirT1), through its ability to bind chromatin regions of the COL2A1 promoter and enhancer.  Although SirT1/Sox9 binding on the enhancer site of COL2A1 was previously demonstrated, little is known about its functional role on the gene promoter site.  Here, we examd. the mechanism by which promoter-assocd. SirT1 governs COL2A1 expression.  Human chondrocytes were encapsulated in three-dimensional (3D) alginate beads where they exhibited upregulated COL2A1 mRNA expression and increased levels of SirT1 occupancy on the promoter and enhancer regions, when compared to monolayer controls.  Chromatin immunopptn. (ChIP) analyses of 3D cultures showed augmented levels of the DNA-binding transcription factor SP1, and the histone methyltransferase Set7/9, on the COL2A1 promoter site.  ChIP reChIP assays revealed that SirT1 and Set7/9 form a protein complex on the COL2A1 promoter region of 3D-cultured chondrocytes, which also demonstrated elevated trimethylated lysine 4 on histone 3 (3MeH3K4), a hallmark of Set7/9 methyltransferase activity.  Advanced passaging of chondrocytes yielded a decrease in 3MeH3K4 and Set7/9 levels on the COL2A1 promoter and reduced COL2A1 expression, suggesting that the SirT1/Set7/9 complex is preferentially formed on the COL2A1 promoter and required for gene activation.  Interestingly, despite SirT1 occupancy, its deacetylation targets (ie, H3K9/14 and H4K16) were found acetylated on the COL2A1 promoter of 3D-cultured chondrocytes.  A possible explanation for this phenotype is the enrichment of the histone acetyltransferases P300 and GCN5 on the COL2A1 promoter of 3D-cultured chondrocytes.  Our study indicates that Set7/9 prevents the histone deacetylase activity of SirT1, potentiating euchromatin formation on the promoter site of COL2A1 and resulting in morphol.-dependent COL2A1 gene transactivation. © 2014 American Society for Bone and Mineral Research.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnUcjHmY7WRrVg90H21EOLACvtfcHk0ljcq4Cyrca6zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ehsLs%253D&md5=06cc70f1955098565d188ae475ce9cea</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1002%2Fjbmr.2052&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjbmr.2052%26sid%3Dliteratum%253Aachs%26aulast%3DOppenheimer%26aufirst%3DH.%26aulast%3DKumar%26aufirst%3DA.%26aulast%3DMeir%26aufirst%3DH.%26aulast%3DSchwartz%26aufirst%3DI.%26aulast%3DZini%26aufirst%3DA.%26aulast%3DHaze%26aufirst%3DA.%26aulast%3DKandel%26aufirst%3DL.%26aulast%3DMattan%26aufirst%3DY.%26aulast%3DLiebergall%26aufirst%3DM.%26aulast%3DDvir-Ginzberg%26aufirst%3DM.%26atitle%3DSet7%252F9%2520Impacts%2520COL2A1%2520Expression%2520Through%2520Binding%2520and%2520Repression%2520of%2520SirT1%2520Histone%2520Deacetylation%26jtitle%3DJ.%2520Bone%2520Miner.%2520Res.%26date%3D2014%26volume%3D29%26spage%3D348%26epage%3D360%26doi%3D10.1002%2Fjbmr.2052" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Subramanian, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jia, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor-Vazirani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Powell, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Collins, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vertino, P. M.</span><span> </span><span class="NLM_article-title">Regulation of Estrogen Receptor α by the SET7 Lysine Methyltransferase</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">336</span><span class="NLM_x">–</span> <span class="NLM_lpage">347</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.03.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2008.03.022" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2008&pages=336-347&author=K.+Subramanianauthor=D.+Jiaauthor=P.+Kapoor-Vaziraniauthor=D.+R.+Powellauthor=R.+E.+Collinsauthor=D.+Sharmaauthor=J.+Pengauthor=X.+Chengauthor=P.+M.+Vertino&title=Regulation+of+Estrogen+Receptor+%CE%B1+by+the+SET7+Lysine+Methyltransferase&doi=10.1016%2Fj.molcel.2008.03.022"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.03.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.03.022%26sid%3Dliteratum%253Aachs%26aulast%3DSubramanian%26aufirst%3DK.%26aulast%3DJia%26aufirst%3DD.%26aulast%3DKapoor-Vazirani%26aufirst%3DP.%26aulast%3DPowell%26aufirst%3DD.%2BR.%26aulast%3DCollins%26aufirst%3DR.%2BE.%26aulast%3DSharma%26aufirst%3DD.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DCheng%26aufirst%3DX.%26aulast%3DVertino%26aufirst%3DP.%2BM.%26atitle%3DRegulation%2520of%2520Estrogen%2520Receptor%2520%25CE%25B1%2520by%2520the%2520SET7%2520Lysine%2520Methyltransferase%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D30%26spage%3D336%26epage%3D347%26doi%3D10.1016%2Fj.molcel.2008.03.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Calnan, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webb, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">White, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stowe, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goswami, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Espejo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gygi, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brunet, A.</span><span> </span><span class="NLM_article-title">Methylation by Set9 modulates FoxO3 stability and transcriptional activity</span> <span class="citation_source-journal">Aging</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">462</span><span class="NLM_x">–</span> <span class="NLM_lpage">479</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.18632%2Faging.100471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22820736" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1anu7zE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2012&pages=462-479&author=D.+R.+Calnanauthor=A.+E.+Webbauthor=J.+L.+Whiteauthor=T.+R.+Stoweauthor=T.+Goswamiauthor=X.+Shiauthor=A.+Espejoauthor=M.+T.+Bedfordauthor=O.+Gozaniauthor=S.+P.+Gygiauthor=A.+Brunet&title=Methylation+by+Set9+modulates+FoxO3+stability+and+transcriptional+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation by Set9 modulates FoxO3 stability and transcriptional activity</span></div><div class="casAuthors">Calnan, Daniel R.; Webb, Ashley E.; White, Jamie L.; Stowe, Timothy R.; Goswami, Tapasree; Shi, Xiaobing; Espejo, Alexsandra; Bedford, Mark T.; Gozani, Or; Gygi, Steven P.; Brunet, Anne</div><div class="citationInfo"><span class="NLM_cas:title">Aging</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">462-479</span>CODEN:
                <span class="NLM_cas:coden">AGINCN</span>;
        ISSN:<span class="NLM_cas:issn">1945-4589</span>.
    
            (<span class="NLM_cas:orgname">Impact Journals LLC</span>)
        </div><div class="casAbstract">The FoxO family of transcription factors plays an important role in longevity and tumor suppression by regulating the expression of a wide range of target genes.  FoxO3 has recently been found to be assocd. with extreme longevity in humans and to regulate the homeostasis of adult stem cell pools in mammals, which may contribute to longevity.  The activity of FoxO3 is controlled by a variety of post-translational modifications that have been proposed to form a 'code' affecting FoxO3 subcellular localization, DNA binding ability, protein-protein interactions and protein stability.  Lysine methylation is a crucial post-translational modification on histones that regulates chromatin accessibility and is a key part of the 'histone code'.  However, whether lysine methylation plays a role in modulating FoxO3 activity has never been examd.  Here we show that the methyltransferase Set9 directly methylates FoxO3 in vitro and in cells.  Using a combination of tandem mass spectrometry and methyl-specific antibodies, we find that Set9 methylates FoxO3 at a single residue, lysine 271, a site previously known to be deacetylated by Sirt1.  Methylation of FoxO3 by Set9 decreases FoxO3 protein stability, while moderately increasing FoxO3 transcriptional activity.  The modulation of FoxO3 stability and activity by methylation may be crit. for fine-tuning cellular responses to stress stimuli, which may in turn affect FoxO3's ability to promote tumor suppression and longevity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpCIuFLql3YlLVg90H21EOLACvtfcHk0litJt2afXohYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1anu7zE&md5=a21545a042463e556b937198cbc51752</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.18632%2Faging.100471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Faging.100471%26sid%3Dliteratum%253Aachs%26aulast%3DCalnan%26aufirst%3DD.%2BR.%26aulast%3DWebb%26aufirst%3DA.%2BE.%26aulast%3DWhite%26aufirst%3DJ.%2BL.%26aulast%3DStowe%26aufirst%3DT.%2BR.%26aulast%3DGoswami%26aufirst%3DT.%26aulast%3DShi%26aufirst%3DX.%26aulast%3DEspejo%26aufirst%3DA.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DGozani%26aufirst%3DO.%26aulast%3DGygi%26aufirst%3DS.%2BP.%26aulast%3DBrunet%26aufirst%3DA.%26atitle%3DMethylation%2520by%2520Set9%2520modulates%2520FoxO3%2520stability%2520and%2520transcriptional%2520activity%26jtitle%3DAging%26date%3D2012%26volume%3D4%26spage%3D462%26epage%3D479" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Cell cycle control by a methylation-phosphorylation switch</span> <span class="citation_source-journal">Cell Cycle</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">734</span><span class="refDoi"> DOI: 10.4161/cc.10.5.14958</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4161%2Fcc.10.5.14958" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=733-734&author=S.+M.+Carrauthor=N.+B.+La+Thangue&title=Cell+cycle+control+by+a+methylation-phosphorylation+switch&doi=10.4161%2Fcc.10.5.14958"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.4161%2Fcc.10.5.14958&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fcc.10.5.14958%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DCell%2520cycle%2520control%2520by%2520a%2520methylation-phosphorylation%2520switch%26jtitle%3DCell%2520Cycle%26date%3D2011%26volume%3D10%26spage%3D733%26epage%3D734%26doi%3D10.4161%2Fcc.10.5.14958" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oppermann, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Interplay between lysine methylation and Cdk phosphorylation in growth control by the retinoblastoma protein</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">327</span><span class="refDoi"> DOI: 10.1038/emboj.2010.311</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Femboj.2010.311" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=317-327&author=S.+M.+Carrauthor=S.+Munroauthor=B.+Kesslerauthor=U.+Oppermannauthor=N.+B.+La+Thangue&title=Interplay+between+lysine+methylation+and+Cdk+phosphorylation+in+growth+control+by+the+retinoblastoma+protein&doi=10.1038%2Femboj.2010.311"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Femboj.2010.311&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2010.311%26sid%3Dliteratum%253Aachs%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DOppermann%26aufirst%3DU.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DInterplay%2520between%2520lysine%2520methylation%2520and%2520Cdk%2520phosphorylation%2520in%2520growth%2520control%2520by%2520the%2520retinoblastoma%2520protein%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D317%26epage%3D327%26doi%3D10.1038%2Femboj.2010.311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Huq, M. D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsai, N.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, L.-N.</span><span> </span><span class="NLM_article-title">Lysine trimethylation of retinoic acid receptor-alpha - A novel means to regulate receptor function</span> <span class="citation_source-journal">Mol. Cell. Proteomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">677</span><span class="NLM_x">–</span> <span class="NLM_lpage">688</span><span class="refDoi"> DOI: 10.1074/mcp.M600223-MCP200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fmcp.M600223-MCP200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17205979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXksVKgurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2007&pages=677-688&author=M.+D.+M.+Huqauthor=N.-P.+Tsaiauthor=S.+A.+Khanauthor=L.-N.+Wei&title=Lysine+trimethylation+of+retinoic+acid+receptor-alpha+-+A+novel+means+to+regulate+receptor+function&doi=10.1074%2Fmcp.M600223-MCP200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine trimethylation of retinoic acid receptor-α: a novel means to regulate receptor function</span></div><div class="casAuthors">Huq, M. D. Mostaqul; Tsai, Nien-Pei; Khan, Shaukat Ali; Wei, Li-Na</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Proteomics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">677-688</span>CODEN:
                <span class="NLM_cas:coden">MCPOBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-9476</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Retinoic acid receptors (RARs) belong to the nuclear receptor superfamily.  The mechanism of ligand-dependent activation of RARs is well known.  The effect of protein phosphorylation on the activity of RARs has also been demonstrated.  However, it is unclear whether other types of modifications exist and if so whether they can affect the activity of RARs.  In a mass spectrometric anal. of mouse RARα expressed in insect cells, we identified a trimethylation site on Lys347 in the ligand binding domain.  The modification site was verified in mammalian cells, and site-directed mutagenesis studies revealed the functionality of Lys347 methylation in vivo.  Constitutive neg. mutants, mimicking hypomethylated RARα, were prepd. by replacing methylated Lys347 with either alanine or glutamine.  A constitutive pos. mutant partially mimicking the hypermethylated RARα was generated by replacing the methylated lysine residue with phenylalanine, a bulky hydrophobic amino acid, to introduce a site-specific hydrophobicity similar to that contributed by lysine methylation.  Studies of these mutants revealed that trimethylation of Lys347 of RARα facilitated its interactions with cofactors p300/CREB-binding protein-assocd. factor and receptor-interacting protein 140 as well as its heterodimeric partner retinoid X receptor, suggesting that site-specific hydrophobicity at Lys347 enhanced mol. interaction of RARα with its modulators.  This study uncovers the first example of lysine trimethylation on a mammalian non-histone protein that has an important biol. consequence.  Our finding also provides the evidence for lysine methylation for the family of nuclear receptors for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpypHiOlk4R6bVg90H21EOLACvtfcHk0litJt2afXohYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXksVKgurw%253D&md5=204e2dcb913f021c0ae64245838c523d</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1074%2Fmcp.M600223-MCP200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fmcp.M600223-MCP200%26sid%3Dliteratum%253Aachs%26aulast%3DHuq%26aufirst%3DM.%2BD.%2BM.%26aulast%3DTsai%26aufirst%3DN.-P.%26aulast%3DKhan%26aufirst%3DS.%2BA.%26aulast%3DWei%26aufirst%3DL.-N.%26atitle%3DLysine%2520trimethylation%2520of%2520retinoic%2520acid%2520receptor-alpha%2520-%2520A%2520novel%2520means%2520to%2520regulate%2520receptor%2520function%26jtitle%3DMol.%2520Cell.%2520Proteomics%26date%3D2007%26volume%3D6%26spage%3D677%26epage%3D688%26doi%3D10.1074%2Fmcp.M600223-MCP200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Yang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dasgupta, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sears, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyagi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chance, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stark, G. R.</span><span> </span><span class="NLM_article-title">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">21499</span><span class="NLM_x">–</span> <span class="NLM_lpage">504</span><span class="refDoi"> DOI: 10.1073/pnas.1016147107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1016147107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21098664" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=21499-504&author=J.+Yangauthor=J.+Huangauthor=M.+Dasguptaauthor=N.+Searsauthor=M.+Miyagiauthor=B.+Wangauthor=M.+R.+Chanceauthor=X.+Chenauthor=Y.+Duauthor=Y.+Wangauthor=L.+Anauthor=Q.+Wangauthor=T.+Luauthor=X.+Zhangauthor=Z.+Wangauthor=G.+R.+Stark&title=Reversible+methylation+of+promoter-bound+STAT3+by+histone-modifying+enzymes&doi=10.1073%2Fpnas.1016147107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible methylation of promoter-bound STAT3 by histone-modifying enzymes</span></div><div class="casAuthors">Yang, Jinbo; Huang, Jing; Dasgupta, Maupali; Sears, Nathan; Miyagi, Masaru; Wang, Benlian; Chance, Mark R.; Chen, Xing; Du, Yuping; Wang, Yuxin; An, Lizhe; Wang, Qin; Lu, Tao; Zhang, Xiaodong; Wang, Zhenghe; Stark, George R.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">50</span>),
    <span class="NLM_cas:pages">21499-21504, S21499/1-S21499/8</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Following its tyrosine phosphorylation, STAT3 is methylated on K140 by the histone Me transferase SET9 and demethylated by LSD1 when it is bound to a subset of the promoters that it activates.  Methylation of K140 is a neg. regulatory event, because its blockade greatly increases the steady-state amt. of activated STAT3 and the expression of many (i.e., SOCS3) but not all (i.e., CD14) STAT3 target genes.  Biol. relevance is shown by the observation that overexpression of SOCS3 when K140 cannot be methylated blocks the ability of cells to activate STAT3 in response to IL-6.  K140 methylation does not occur with mutants of STAT3 that do not enter nuclei or bind to DNA.  Following treatment with IL-6, events at the SOCS3 promoter occur in an ordered sequence, as shown by chromatin immunopptns.  Y705-phosphoryl-STAT3 binds first and S727 is then phosphorylated, followed by the coincident binding of SET9 and dimethylation of K140, and lastly by the binding of LSD1.  We conclude that the lysine methylation of promoter-bound STAT3 leads to biol. important down-regulation of the dependent responses and that SET9, which is known to help provide an activating methylation mark to H3K4, is recruited to the newly activated SOCS3 promoter by STAT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTNJmJCGOKwbVg90H21EOLACvtfcHk0lgNKn7P8lm5QA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1WisL%252FM&md5=255afed29b2b72fde8e2baf3d09a85dd</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1016147107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1016147107%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DDasgupta%26aufirst%3DM.%26aulast%3DSears%26aufirst%3DN.%26aulast%3DMiyagi%26aufirst%3DM.%26aulast%3DWang%26aufirst%3DB.%26aulast%3DChance%26aufirst%3DM.%2BR.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DAn%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DStark%26aufirst%3DG.%2BR.%26atitle%3DReversible%2520methylation%2520of%2520promoter-bound%2520STAT3%2520by%2520histone-modifying%2520enzymes%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D21499%26epage%3D504%26doi%3D10.1073%2Fpnas.1016147107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wei, F.-Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Y.</span><span> </span><span class="NLM_article-title">Methylation of SUV39H1 by SET7/9 results in heterochromatin relaxation and genome instability</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">5516</span><span class="NLM_x">–</span> <span class="NLM_lpage">5521</span><span class="refDoi"> DOI: 10.1073/pnas.1216596110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1216596110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=5516-5521&author=D.+Wangauthor=J.+Zhouauthor=X.+Liuauthor=D.+Luauthor=C.+Shenauthor=Y.+Duauthor=F.-Z.+Weiauthor=B.+Songauthor=X.+Luauthor=Y.+Yu&title=Methylation+of+SUV39H1+by+SET7%2F9+results+in+heterochromatin+relaxation+and+genome+instability&doi=10.1073%2Fpnas.1216596110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1216596110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1216596110%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DLu%26aufirst%3DD.%26aulast%3DShen%26aufirst%3DC.%26aulast%3DDu%26aufirst%3DY.%26aulast%3DWei%26aufirst%3DF.-Z.%26aulast%3DSong%26aufirst%3DB.%26aulast%3DLu%26aufirst%3DX.%26aulast%3DYu%26aufirst%3DY.%26atitle%3DMethylation%2520of%2520SUV39H1%2520by%2520SET7%252F9%2520results%2520in%2520heterochromatin%2520relaxation%2520and%2520genome%2520instability%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D5516%26epage%3D5521%26doi%3D10.1073%2Fpnas.1216596110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Albacker, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Storer, N. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langdon, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dibiase, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Langenau, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zon, L. I.</span><span> </span><span class="NLM_article-title">The histone methyltransferase SUV39H1 suppresses embryonal rhabdomyosarcoma formation in zebrafish</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">e64969</span><span class="refDoi"> DOI: 10.1371/journal.pone.0064969</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1371%2Fjournal.pone.0064969" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=e64969&author=C.+E.+Albackerauthor=N.+Y.+Storerauthor=E.+M.+Langdonauthor=A.+Dibiaseauthor=Y.+Zhouauthor=D.+M.+Langenauauthor=L.+I.+Zon&title=The+histone+methyltransferase+SUV39H1+suppresses+embryonal+rhabdomyosarcoma+formation+in+zebrafish&doi=10.1371%2Fjournal.pone.0064969"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0064969&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0064969%26sid%3Dliteratum%253Aachs%26aulast%3DAlbacker%26aufirst%3DC.%2BE.%26aulast%3DStorer%26aufirst%3DN.%2BY.%26aulast%3DLangdon%26aufirst%3DE.%2BM.%26aulast%3DDibiase%26aufirst%3DA.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLangenau%26aufirst%3DD.%2BM.%26aulast%3DZon%26aufirst%3DL.%2BI.%26atitle%3DThe%2520histone%2520methyltransferase%2520SUV39H1%2520suppresses%2520embryonal%2520rhabdomyosarcoma%2520formation%2520in%2520zebrafish%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26spage%3De64969%26doi%3D10.1371%2Fjournal.pone.0064969" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Kouskouti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheer, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Staub, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tora, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Talianidis, I.</span><span> </span><span class="NLM_article-title">Gene-specific modulation of TAF10 function by SET9-mediated methylation</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">175</span><span class="NLM_x">–</span> <span class="NLM_lpage">182</span><span class="refDoi"> DOI: 10.1016/S1097-2765(04)00182-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS1097-2765%2804%2900182-0" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2004&pages=175-182&author=A.+Kouskoutiauthor=E.+Scheerauthor=A.+Staubauthor=L.+Toraauthor=I.+Talianidis&title=Gene-specific+modulation+of+TAF10+function+by+SET9-mediated+methylation&doi=10.1016%2FS1097-2765%2804%2900182-0"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1016%2FS1097-2765%2804%2900182-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1097-2765%252804%252900182-0%26sid%3Dliteratum%253Aachs%26aulast%3DKouskouti%26aufirst%3DA.%26aulast%3DScheer%26aufirst%3DE.%26aulast%3DStaub%26aufirst%3DA.%26aulast%3DTora%26aufirst%3DL.%26aulast%3DTalianidis%26aufirst%3DI.%26atitle%3DGene-specific%2520modulation%2520of%2520TAF10%2520function%2520by%2520SET9-mediated%2520methylation%26jtitle%3DMol.%2520Cell%26date%3D2004%26volume%3D14%26spage%3D175%26epage%3D182%26doi%3D10.1016%2FS1097-2765%2804%2900182-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Keating, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Osta, A.</span><span> </span><span class="NLM_article-title">Transcriptional regulation by the Set7 lysine methyltransferase</span> <span class="citation_source-journal">Epigenetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.4161/epi.24234</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4161%2Fepi.24234" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=361-372&author=S.+T.+Keatingauthor=A.+El-Osta&title=Transcriptional+regulation+by+the+Set7+lysine+methyltransferase&doi=10.4161%2Fepi.24234"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Fepi.24234&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Fepi.24234%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DS.%2BT.%26aulast%3DEl-Osta%26aufirst%3DA.%26atitle%3DTranscriptional%2520regulation%2520by%2520the%2520Set7%2520lysine%2520methyltransferase%26jtitle%3DEpigenetics%26date%3D2013%26volume%3D8%26spage%3D361%26epage%3D372%26doi%3D10.4161%2Fepi.24234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Maganti, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tersey, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sampley, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mosley, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ozcan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pachaiyappan, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woster, P. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Transcriptional activity of the islet beta cell factor Pdx1 is augmented by lysine methylation catalyzed by the methyltransferase Set7/9</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">290</span><span class="NLM_x">, </span> <span class="NLM_fpage">9812</span><span class="NLM_x">–</span> <span class="NLM_lpage">22</span><span class="refDoi"> DOI: 10.1074/jbc.M114.616219</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M114.616219" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=9812-22&author=A.+V.+Magantiauthor=B.+Maierauthor=S.+A.+Terseyauthor=M.+L.+Sampleyauthor=A.+L.+Mosleyauthor=S.+Ozcanauthor=B.+Pachaiyappanauthor=P.+M.+Wosterauthor=C.+S.+Hunterauthor=R.+Steinauthor=R.+G.+Mirmira&title=Transcriptional+activity+of+the+islet+beta+cell+factor+Pdx1+is+augmented+by+lysine+methylation+catalyzed+by+the+methyltransferase+Set7%2F9&doi=10.1074%2Fjbc.M114.616219"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.616219&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.616219%26sid%3Dliteratum%253Aachs%26aulast%3DMaganti%26aufirst%3DA.%2BV.%26aulast%3DMaier%26aufirst%3DB.%26aulast%3DTersey%26aufirst%3DS.%2BA.%26aulast%3DSampley%26aufirst%3DM.%2BL.%26aulast%3DMosley%26aufirst%3DA.%2BL.%26aulast%3DOzcan%26aufirst%3DS.%26aulast%3DPachaiyappan%26aufirst%3DB.%26aulast%3DWoster%26aufirst%3DP.%2BM.%26aulast%3DHunter%26aufirst%3DC.%2BS.%26aulast%3DStein%26aufirst%3DR.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DTranscriptional%2520activity%2520of%2520the%2520islet%2520beta%2520cell%2520factor%2520Pdx1%2520is%2520augmented%2520by%2520lysine%2520methylation%2520catalyzed%2520by%2520the%2520methyltransferase%2520Set7%252F9%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D9812%26epage%3D22%26doi%3D10.1074%2Fjbc.M114.616219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, W.</span><span> </span><span class="NLM_article-title">Repression of hypoxia-inducible factor alpha signaling by Set7-mediated methylation</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">5081</span><span class="NLM_x">–</span> <span class="NLM_lpage">98</span><span class="refDoi"> DOI: 10.1093/nar/gkv379</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fnar%2Fgkv379" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=5081-98&author=X.+Liuauthor=Z.+Chenauthor=C.+Xuauthor=X.+Lengauthor=H.+Caoauthor=G.+Ouyangauthor=W.+Xiao&title=Repression+of+hypoxia-inducible+factor+alpha+signaling+by+Set7-mediated+methylation&doi=10.1093%2Fnar%2Fgkv379"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv379%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DLeng%26aufirst%3DX.%26aulast%3DCao%26aufirst%3DH.%26aulast%3DOuyang%26aufirst%3DG.%26aulast%3DXiao%26aufirst%3DW.%26atitle%3DRepression%2520of%2520hypoxia-inducible%2520factor%2520alpha%2520signaling%2520by%2520Set7-mediated%2520methylation%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3D5081%26epage%3D98%26doi%3D10.1093%2Fnar%2Fgkv379" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Deering, T. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ogihara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trace, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maier, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Methyltransferase Set7/9 Maintains Transcription and Euchromatin Structure at Islet-Enriched Genes</span> <span class="citation_source-journal">Diabetes</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">185</span><span class="NLM_x">–</span> <span class="NLM_lpage">193</span><span class="refDoi"> DOI: 10.2337/db08-1150</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2337%2Fdb08-1150" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2009&pages=185-193&author=T.+G.+Deeringauthor=T.+Ogiharaauthor=A.+P.+Traceauthor=B.+Maierauthor=R.+G.+Mirmira&title=Methyltransferase+Set7%2F9+Maintains+Transcription+and+Euchromatin+Structure+at+Islet-Enriched+Genes&doi=10.2337%2Fdb08-1150"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.2337%2Fdb08-1150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2337%252Fdb08-1150%26sid%3Dliteratum%253Aachs%26aulast%3DDeering%26aufirst%3DT.%2BG.%26aulast%3DOgihara%26aufirst%3DT.%26aulast%3DTrace%26aufirst%3DA.%2BP.%26aulast%3DMaier%26aufirst%3DB.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DMethyltransferase%2520Set7%252F9%2520Maintains%2520Transcription%2520and%2520Euchromatin%2520Structure%2520at%2520Islet-Enriched%2520Genes%26jtitle%3DDiabetes%26date%3D2009%26volume%3D58%26spage%3D185%26epage%3D193%26doi%3D10.2337%2Fdb08-1150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Francis, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garmey, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mirmira, R. G.</span><span> </span><span class="NLM_article-title">Pdx-1 links histone H3-Lys-4 methylation to RNA polymerase II elongation during activation of insulin transcription</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">280</span><span class="NLM_x">, </span> <span class="NLM_fpage">36244</span><span class="NLM_x">–</span> <span class="NLM_lpage">36253</span><span class="refDoi"> DOI: 10.1074/jbc.M505741200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M505741200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=280&publication_year=2005&pages=36244-36253&author=J.+Francisauthor=S.+K.+Chakrabartiauthor=J.+C.+Garmeyauthor=R.+G.+Mirmira&title=Pdx-1+links+histone+H3-Lys-4+methylation+to+RNA+polymerase+II+elongation+during+activation+of+insulin+transcription&doi=10.1074%2Fjbc.M505741200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M505741200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M505741200%26sid%3Dliteratum%253Aachs%26aulast%3DFrancis%26aufirst%3DJ.%26aulast%3DChakrabarti%26aufirst%3DS.%2BK.%26aulast%3DGarmey%26aufirst%3DJ.%2BC.%26aulast%3DMirmira%26aufirst%3DR.%2BG.%26atitle%3DPdx-1%2520links%2520histone%2520H3-Lys-4%2520methylation%2520to%2520RNA%2520polymerase%2520II%2520elongation%2520during%2520activation%2520of%2520insulin%2520transcription%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2005%26volume%3D280%26spage%3D36244%26epage%3D36253%26doi%3D10.1074%2Fjbc.M505741200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">La Thangue, N. B.</span><span> </span><span class="NLM_article-title">Diversity within the pRb pathway: is there a code of conduct?</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">4343</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span><span class="refDoi"> DOI: 10.1038/onc.2011.603</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fonc.2011.603" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=4343-52&author=S.+Munroauthor=S.+M.+Carrauthor=N.+B.+La+Thangue&title=Diversity+within+the+pRb+pathway%3A+is+there+a+code+of+conduct%3F&doi=10.1038%2Fonc.2011.603"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fonc.2011.603&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2011.603%26sid%3Dliteratum%253Aachs%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DLa%2BThangue%26aufirst%3DN.%2BB.%26atitle%3DDiversity%2520within%2520the%2520pRb%2520pathway%253A%2520is%2520there%2520a%2520code%2520of%2520conduct%253F%26jtitle%3DOncogene%26date%3D2012%26volume%3D31%26spage%3D4343%26epage%3D52%26doi%3D10.1038%2Fonc.2011.603" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Tao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neppl, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tang, R.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, S.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.-Z.</span><span> </span><span class="NLM_article-title">The histone methyltransferase Set7/9 promotes myoblast differentiation and myofibril assembly</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">194</span><span class="NLM_x">, </span> <span class="NLM_fpage">551</span><span class="NLM_x">–</span> <span class="NLM_lpage">565</span><span class="refDoi"> DOI: 10.1083/jcb.201010090</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1083%2Fjcb.201010090" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2011&pages=551-565&author=Y.+Taoauthor=R.+L.+Nepplauthor=Z.-P.+Huangauthor=J.+Chenauthor=R.-H.+Tangauthor=R.+Caoauthor=Y.+Zhangauthor=S.-W.+Jinauthor=D.-Z.+Wang&title=The+histone+methyltransferase+Set7%2F9+promotes+myoblast+differentiation+and+myofibril+assembly&doi=10.1083%2Fjcb.201010090"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1083%2Fjcb.201010090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.201010090%26sid%3Dliteratum%253Aachs%26aulast%3DTao%26aufirst%3DY.%26aulast%3DNeppl%26aufirst%3DR.%2BL.%26aulast%3DHuang%26aufirst%3DZ.-P.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DTang%26aufirst%3DR.-H.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJin%26aufirst%3DS.-W.%26aulast%3DWang%26aufirst%3DD.-Z.%26atitle%3DThe%2520histone%2520methyltransferase%2520Set7%252F9%2520promotes%2520myoblast%2520differentiation%2520and%2520myofibril%2520assembly%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2011%26volume%3D194%26spage%3D551%26epage%3D565%26doi%3D10.1083%2Fjcb.201010090" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Esteve, P. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chin, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benner, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feehery, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Samaranayake, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horwitz, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobsen, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pradhan, S.</span><span> </span><span class="NLM_article-title">Regulation of DNMT1 stability through SET7-mediated lysine methylation in mammalian cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">5076</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1073/pnas.0810362106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.0810362106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=5076-81&author=P.+O.+Esteveauthor=H.+G.+Chinauthor=J.+Bennerauthor=G.+R.+Feeheryauthor=M.+Samaranayakeauthor=G.+A.+Horwitzauthor=S.+E.+Jacobsenauthor=S.+Pradhan&title=Regulation+of+DNMT1+stability+through+SET7-mediated+lysine+methylation+in+mammalian+cells&doi=10.1073%2Fpnas.0810362106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0810362106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0810362106%26sid%3Dliteratum%253Aachs%26aulast%3DEsteve%26aufirst%3DP.%2BO.%26aulast%3DChin%26aufirst%3DH.%2BG.%26aulast%3DBenner%26aufirst%3DJ.%26aulast%3DFeehery%26aufirst%3DG.%2BR.%26aulast%3DSamaranayake%26aufirst%3DM.%26aulast%3DHorwitz%26aufirst%3DG.%2BA.%26aulast%3DJacobsen%26aufirst%3DS.%2BE.%26aulast%3DPradhan%26aufirst%3DS.%26atitle%3DRegulation%2520of%2520DNMT1%2520stability%2520through%2520SET7-mediated%2520lysine%2520methylation%2520in%2520mammalian%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D5076%26epage%3D81%26doi%3D10.1073%2Fpnas.0810362106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gay, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bajko, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span> </span><span class="NLM_article-title">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">125</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1038/ng.268</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fng.268" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19098913" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2009&pages=125-129&author=J.+Wangauthor=S.+Heviauthor=J.+K.+Kurashauthor=H.+Leiauthor=F.+Gayauthor=J.+Bajkoauthor=H.+Suauthor=W.+Sunauthor=H.+Changauthor=G.+Xuauthor=F.+Gaudetauthor=E.+Liauthor=T.+Chen&title=The+lysine+demethylase+LSD1+%28KDM1%29+is+required+for+maintenance+of+global+DNA+methylation&doi=10.1038%2Fng.268"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">The lysine demethylase LSD1 (KDM1) is required for maintenance of global DNA methylation</span></div><div class="casAuthors">Wang, Jing; Hevi, Sarah; Kurash, Julia K.; Lei, Hong; Gay, Frederique; Bajko, Jeffrey; Su, Hui; Sun, Weitao; Chang, Hua; Xu, Guoliang; Gaudet, Francois; Li, En; Chen, Taiping</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">125-129</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Histone methylation and DNA methylation cooperatively regulate chromatin structure and gene activity.  How these two systems coordinate with each other remains unclear.  Here we study the biol. function of lysine-specific demethylase 1 (LSD1, also known as KDM1 and AOF2), which has been shown to demethylate histone H3 on lysine 4 (H3K4) and lysine 9 (H3K9).  We show that LSD1 is required for gastrulation during mouse embryogenesis.  Notably, targeted deletion of the gene encoding LSD1 (namely, Aof2) in embryonic stem (ES) cells induces progressive loss of DNA methylation.  This loss correlates with a decrease in DNA methyltransferase 1 (Dnmt1) protein, as a result of reduced Dnmt1 stability.  Dnmt1 protein is methylated in vivo, and its methylation is enhanced in the absence of LSD1.  Furthermore, Dnmt1 can be methylated by Set7/9 (also known as KMT7) and demethylated by LSD1 in vitro.  Our findings suggest that LSD1 demethylates and stabilizes Dnmt1, thus providing a previously unknown mechanistic link between the histone and DNA methylation systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSJgebl5_48LVg90H21EOLACvtfcHk0lg3MCNHNXWdrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFajtbbK&md5=37722ddd4564a79a0cd1c7b148c370d8</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1038%2Fng.268&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.268%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DHevi%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DGay%26aufirst%3DF.%26aulast%3DBajko%26aufirst%3DJ.%26aulast%3DSu%26aufirst%3DH.%26aulast%3DSun%26aufirst%3DW.%26aulast%3DChang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DG.%26aulast%3DGaudet%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DT.%26atitle%3DThe%2520lysine%2520demethylase%2520LSD1%2520%2528KDM1%2529%2520is%2520required%2520for%2520maintenance%2520of%2520global%2520DNA%2520methylation%26jtitle%3DNat.%2520Genet.%26date%3D2009%26volume%3D41%26spage%3D125%26epage%3D129%26doi%3D10.1038%2Fng.268" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Ea, C.-K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baltimore, D.</span><span> </span><span class="NLM_article-title">Regulation of NF-kappa B activity through lysine monomethylation of p65</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">106</span><span class="NLM_x">, </span> <span class="NLM_fpage">18972</span><span class="NLM_x">–</span> <span class="NLM_lpage">18977</span><span class="refDoi"> DOI: 10.1073/pnas.0910439106</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.0910439106" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=106&publication_year=2009&pages=18972-18977&author=C.-K.+Eaauthor=D.+Baltimore&title=Regulation+of+NF-kappa+B+activity+through+lysine+monomethylation+of+p65&doi=10.1073%2Fpnas.0910439106"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0910439106&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0910439106%26sid%3Dliteratum%253Aachs%26aulast%3DEa%26aufirst%3DC.-K.%26aulast%3DBaltimore%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520NF-kappa%2520B%2520activity%2520through%2520lysine%2520monomethylation%2520of%2520p65%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2009%26volume%3D106%26spage%3D18972%26epage%3D18977%26doi%3D10.1073%2Fpnas.0910439106" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shanmugam, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hawkins, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Natarajan, R.</span><span> </span><span class="NLM_article-title">Role of the histone H3 lysine 4 methyltransferase, SET7/9, in the regulation of NF-kappaB-dependent inflammatory genes. Relevance to diabetes and inflammation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">283</span><span class="NLM_x">, </span> <span class="NLM_fpage">26771</span><span class="NLM_x">–</span> <span class="NLM_lpage">81</span><span class="refDoi"> DOI: 10.1074/jbc.M802800200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1074%2Fjbc.M802800200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=283&publication_year=2008&pages=26771-81&author=Y.+Liauthor=M.+A.+Reddyauthor=F.+Miaoauthor=N.+Shanmugamauthor=J.+K.+Yeeauthor=D.+Hawkinsauthor=B.+Renauthor=R.+Natarajan&title=Role+of+the+histone+H3+lysine+4+methyltransferase%2C+SET7%2F9%2C+in+the+regulation+of+NF-kappaB-dependent+inflammatory+genes.+Relevance+to+diabetes+and+inflammation&doi=10.1074%2Fjbc.M802800200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M802800200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M802800200%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DY.%26aulast%3DReddy%26aufirst%3DM.%2BA.%26aulast%3DMiao%26aufirst%3DF.%26aulast%3DShanmugam%26aufirst%3DN.%26aulast%3DYee%26aufirst%3DJ.%2BK.%26aulast%3DHawkins%26aufirst%3DD.%26aulast%3DRen%26aufirst%3DB.%26aulast%3DNatarajan%26aufirst%3DR.%26atitle%3DRole%2520of%2520the%2520histone%2520H3%2520lysine%25204%2520methyltransferase%252C%2520SET7%252F9%252C%2520in%2520the%2520regulation%2520of%2520NF-kappaB-dependent%2520inflammatory%2520genes.%2520Relevance%2520to%2520diabetes%2520and%2520inflammation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2008%26volume%3D283%26spage%3D26771%26epage%3D81%26doi%3D10.1074%2Fjbc.M802800200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Paneni, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costantino, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Battista, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castello, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capretti, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiandotto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scavone, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Villano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pitocco, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanza, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volpe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luscher, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cosentino, F.</span><span> </span><span class="NLM_article-title">Adverse Epigenetic Signatures by Histone Methyltransferase Set7 Contribute to Vascular Dysfunction in Patients With Type 2 Diabetes Mellitus</span> <span class="citation_source-journal">Circ.: Cardiovasc. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">150</span><span class="NLM_x">–</span> <span class="NLM_lpage">U304</span><span class="refDoi"> DOI: 10.1161/CIRCGENETICS.114.000671</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1161%2FCIRCGENETICS.114.000671" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25472959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtFaru7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2015&pages=150-U304&author=F.+Paneniauthor=S.+Costantinoauthor=R.+Battistaauthor=L.+Castelloauthor=G.+Caprettiauthor=S.+Chiandottoauthor=G.+Scavoneauthor=A.+Villanoauthor=D.+Pitoccoauthor=G.+Lanzaauthor=M.+Volpeauthor=T.+F.+Luscherauthor=F.+Cosentino&title=Adverse+Epigenetic+Signatures+by+Histone+Methyltransferase+Set7+Contribute+to+Vascular+Dysfunction+in+Patients+With+Type+2+Diabetes+Mellitus&doi=10.1161%2FCIRCGENETICS.114.000671"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Adverse Epigenetic Signatures by Histone Methyltransferase Set7 Contribute to Vascular Dysfunction in Patients With Type 2 Diabetes Mellitus</span></div><div class="casAuthors">Paneni, Francesco; Costantino, Sarah; Battista, Rodolfo; Castello, Lorenzo; Capretti, Giuliana; Chiandotto, Sergio; Scavone, Giuseppe; Villano, Angelo; Pitocco, Dario; Lanza, Gaetano; Volpe, Massimo; Luescher, Thomas F.; Cosentino, Francesco</div><div class="citationInfo"><span class="NLM_cas:title">Circulation: Cardiovascular Genetics</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150-158</span>CODEN:
                <span class="NLM_cas:coden">CCGIAO</span>;
        ISSN:<span class="NLM_cas:issn">1942-3268</span>.
    
            (<span class="NLM_cas:orgname">Lippincott Williams & Wilkins</span>)
        </div><div class="casAbstract">BACKGROUND-: Cellular studies showed that histone methyltransferase Set7 mediates high glucose-induced inflammation via epigenetic regulation of the transcription factor NF-kB.  However, the link between Set7 and vascular dysfunction in patients with diabetes mellitus remains unknown.  This study was designed to investigate whether Set7 contributes to vascular dysfunction in patients with type 2 diabetes mellitus (T2DM).  METHODS AND RESULTS-: Set7-driven epigenetic changes on NF-kB p65 promoter and expression of NF-kB-dependent genes, cyclooxygenase 2 and inducible endothelial nitric oxide synthase, were assessed in peripheral blood mononuclear cells isolated from 68 subjects (44 patients with T2DM and 24 age-matched controls).  Brachial artery flow-mediated dilation, 24-h urinary levels of 8-isoprostaglandin F2α, and plasma adhesion mols., intercellular cell adhesion mol.-1 and monocyte chemoattractant protein-1, were also detd.  Expts. in human aortic endothelial cells exposed to high glucose were performed to elucidate the mechanisms of Set7-driven inflammation and oxidative stress.  Set7 expression increased in peripheral blood mononuclear cells from patients with T2DM when compared with controls.  Patients with T2DM showed Set7-dependent monomethylation of lysine 4 of histone 3 on NF-kB p65 promoter.  This epigenetic signature was assocd. with upregulation of NF-kB, subsequent transcription of oxidant/inflammatory genes, and increased plasma levels of intercellular cell adhesion mol.-1 and monocyte chemoattractant protein-1.  Interestingly, we found that Set7 expression significantly correlated with oxidative marker 8-isoprostaglandin F2α (r=0.38; P=0.01) and flow-mediated dilation (r=-0.34; P=0.04).  In human aortic endothelial cells, silencing of Set7 prevented monomethylation of lysine 4 of histone 3 and abolished NF-kB-dependent oxidant and inflammatory signaling.  CONCLUSIONS-: Set7-induced epigenetic changes contribute to vascular dysfunction in patients with T2DM.  Targeting this chromatin-modifying enzyme may represent a novel therapeutic approach to prevent atherosclerotic vascular disease in this setting.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYhiQh5PrZfbVg90H21EOLACvtfcHk0lgt-ppfc9zVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtFaru7w%253D&md5=4cc7181c59c556d0c86c5cc62298af5e</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1161%2FCIRCGENETICS.114.000671&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1161%252FCIRCGENETICS.114.000671%26sid%3Dliteratum%253Aachs%26aulast%3DPaneni%26aufirst%3DF.%26aulast%3DCostantino%26aufirst%3DS.%26aulast%3DBattista%26aufirst%3DR.%26aulast%3DCastello%26aufirst%3DL.%26aulast%3DCapretti%26aufirst%3DG.%26aulast%3DChiandotto%26aufirst%3DS.%26aulast%3DScavone%26aufirst%3DG.%26aulast%3DVillano%26aufirst%3DA.%26aulast%3DPitocco%26aufirst%3DD.%26aulast%3DLanza%26aufirst%3DG.%26aulast%3DVolpe%26aufirst%3DM.%26aulast%3DLuscher%26aufirst%3DT.%2BF.%26aulast%3DCosentino%26aufirst%3DF.%26atitle%3DAdverse%2520Epigenetic%2520Signatures%2520by%2520Histone%2520Methyltransferase%2520Set7%2520Contribute%2520to%2520Vascular%2520Dysfunction%2520in%2520Patients%2520With%2520Type%25202%2520Diabetes%2520Mellitus%26jtitle%3DCirc.%253A%2520Cardiovasc.%2520Genet.%26date%3D2015%26volume%3D8%26spage%3D150%26epage%3DU304%26doi%3D10.1161%2FCIRCGENETICS.114.000671" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Q.</span><span> </span><span class="NLM_article-title">Abnormal epigenetic modifications in peripheral blood mononuclear cells from patients with alopecia areata</span> <span class="citation_source-journal">Br. J. Dermatol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">166</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">–</span> <span class="NLM_lpage">273</span><span class="refDoi"> DOI: 10.1111/j.1365-2133.2011.10646.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1111%2Fj.1365-2133.2011.10646.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVSjtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=166&publication_year=2012&pages=266-273&author=M.+Zhaoauthor=G.+Liangauthor=X.+Wuauthor=S.+Wangauthor=P.+Zhangauthor=Y.+Suauthor=H.+Yinauthor=Y.+Tanauthor=J.+Zhangauthor=Q.+Lu&title=Abnormal+epigenetic+modifications+in+peripheral+blood+mononuclear+cells+from+patients+with+alopecia+areata&doi=10.1111%2Fj.1365-2133.2011.10646.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Abnormal epigenetic modifications in peripheral blood mononuclear cells from patients with alopecia areata</span></div><div class="casAuthors">Zhao, M.; Liang, G.; Wu, X.; Wang, S.; Zhang, P.; Su, Y.; Yin, H.; Tan, Y.; Zhang, J.; Lu, Q.</div><div class="citationInfo"><span class="NLM_cas:title">British Journal of Dermatology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">166</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">BJDEAZ</span>;
        ISSN:<span class="NLM_cas:issn">0007-0963</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Background: Alopecia areata (AA) is a hair loss disease caused by T-cell-mediated autoimmune reactions against anagen-stage hair follicles.  Although the exact etiol. is poorly understood, there is evidence to suggest that both genetic and environmental factors are involved in AA pathogenesis.  Objectives: To analyze DNA methylation and histone modification patterns in peripheral blood mononuclear cells (PBMCs) of patients with AA.  Methods: PBMC samples were obtained from 25 patients with AA and 20 healthy controls.  Global DNA methylcytosine levels, as well as histone acetylation and methylation levels, were measured by ELISA.  MRNA expression levels were detd. using real-time quant. reverse transcription-polymerase chain reaction.  Results: Genomic DNA methylation in PBMCs of patients with AA was increased relative to controls.  DNMT1, MBD1 and MBD4 expression levels were significantly higher in AA PBMCs than in controls, and DNMT1 transcription levels pos. correlated with global DNA methylation levels in patient samples.  Histone H3 acetylation was significantly increased and histone H3 lysine 4 methylation was significantly decreased in patient PBMCs compared with healthy controls.  Histone H3 acetylation levels were pos. correlated with AA disease severity, and with RANTES (CCL5) mRNA expression in PBMCs of patients with AA.  These changes were accompanied by increased p300 (EP300), histone deacetylase 1 (HDAC1), myeloid/lymphoid or mixed lineage leukemia (MLL), SET7/9 (SETD7), G9A (EHMT2), JMJD2C (KDM4C) and JARID1A (KDM5A) expression, as well as reduced HDAC2, HDAC7, LSD1 (KDM1A), JMJD2A (KDM4A) and JMJD2B (KDM4B) expression.  Conclusions: DNA methylation and histone modification status are altered in PBMCs of patients with AA, possibly due to the deregulation of epigenetic regulatory genes.  These changes may contribute to the activation of pathol. immune responses in AA.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq6IswsWFb3FrVg90H21EOLACvtfcHk0lgt-ppfc9zVCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVSjtr8%253D&md5=76737cd2408f75a2b3e315c93a0628f3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1111%2Fj.1365-2133.2011.10646.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1365-2133.2011.10646.x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DLiang%26aufirst%3DG.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DZhang%26aufirst%3DP.%26aulast%3DSu%26aufirst%3DY.%26aulast%3DYin%26aufirst%3DH.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DQ.%26atitle%3DAbnormal%2520epigenetic%2520modifications%2520in%2520peripheral%2520blood%2520mononuclear%2520cells%2520from%2520patients%2520with%2520alopecia%2520areata%26jtitle%3DBr.%2520J.%2520Dermatol.%26date%3D2012%26volume%3D166%26spage%3D266%26epage%3D273%26doi%3D10.1111%2Fj.1365-2133.2011.10646.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Adachi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takeuchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagayama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furihata, M.</span><span> </span><span class="NLM_article-title">T-Cadherin Modulates Tumor-Associated Molecules in Gallbladder Cancer Cells</span> <span class="citation_source-journal">Cancer Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">120</span><span class="NLM_x">–</span> <span class="NLM_lpage">126</span><span class="refDoi"> DOI: 10.3109/07357900903124472</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.3109%2F07357900903124472" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=120-126&author=Y.+Adachiauthor=T.+Takeuchiauthor=T.+Nagayamaauthor=M.+Furihata&title=T-Cadherin+Modulates+Tumor-Associated+Molecules+in+Gallbladder+Cancer+Cells&doi=10.3109%2F07357900903124472"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.3109%2F07357900903124472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F07357900903124472%26sid%3Dliteratum%253Aachs%26aulast%3DAdachi%26aufirst%3DY.%26aulast%3DTakeuchi%26aufirst%3DT.%26aulast%3DNagayama%26aufirst%3DT.%26aulast%3DFurihata%26aufirst%3DM.%26atitle%3DT-Cadherin%2520Modulates%2520Tumor-Associated%2520Molecules%2520in%2520Gallbladder%2520Cancer%2520Cells%26jtitle%3DCancer%2520Invest.%26date%3D2009%26volume%3D28%26spage%3D120%26epage%3D126%26doi%3D10.3109%2F07357900903124472" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Pagans, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kauder, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaehlcke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sakane, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schroeder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dormeyer, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trievel, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verdin, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schnolzer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, M.</span><span> </span><span class="NLM_article-title">The Cellular Lysine Methyltransferase Set7/9-KMT7 Binds HIV-1 TAR RNA, Monomethylates the Viral Transactivator Tat, and Enhances HIV Transcription</span> <span class="citation_source-journal">Cell Host Microbe</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">234</span><span class="NLM_x">–</span> <span class="NLM_lpage">244</span><span class="refDoi"> DOI: 10.1016/j.chom.2010.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.chom.2010.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20227666" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXlsVWqu70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2010&pages=234-244&author=S.+Pagansauthor=S.+E.+Kauderauthor=K.+Kaehlckeauthor=N.+Sakaneauthor=S.+Schroederauthor=W.+Dormeyerauthor=R.+C.+Trievelauthor=E.+Verdinauthor=M.+Schnolzerauthor=M.+Ott&title=The+Cellular+Lysine+Methyltransferase+Set7%2F9-KMT7+Binds+HIV-1+TAR+RNA%2C+Monomethylates+the+Viral+Transactivator+Tat%2C+and+Enhances+HIV+Transcription&doi=10.1016%2Fj.chom.2010.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">The cellular lysine methyltransferase Set7/9-KMT7 binds HIV-1 TAR RNA, monomethylates the viral transactivator Tat, and enhances HIV transcription</span></div><div class="casAuthors">Pagans, Sara; Kauder, Steven E.; Kaehlcke, Katrin; Sakane, Naoki; Schroeder, Sebastian; Dormeyer, Wilma; Trievel, Raymond C.; Verdin, Eric; Schnolzer, Martina; Ott, Melanie</div><div class="citationInfo"><span class="NLM_cas:title">Cell Host & Microbe</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">234-244</span>CODEN:
                <span class="NLM_cas:coden">CHMECB</span>;
        ISSN:<span class="NLM_cas:issn">1931-3128</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The Tat protein of HIV-1 plays an essential role in HIV gene expression by promoting efficient elongation of viral transcripts.  Post-translational modifications of Tat fine-tune interactions of Tat with cellular cofactors and TAR RNA, a stem-loop structure at the 5' ends of viral transcripts.  Here, we identify the lysine methyltransferase Set7/9 (KMT7) as a coactivator of HIV transcription.  Set7/9-KMT7 assocs. with the HIV promoter in vivo and monomethylates lysine 51, a highly conserved residue located in the RNA-binding domain of Tat.  Knockdown of Set7/9-KMT7 suppresses Tat transactivation of the HIV promoter, but does not affect the transcriptional activity of methylation-deficient Tat (K51A).  Set7/9-KMT7 binds TAR RNA by itself and in complex with Tat and the pos. transcription elongation factor P-TEFb.  Our findings uncover a pos. role for Set7/9-KMT7 and Tat methylation during early steps of the Tat transactivation cycle.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpu8AqerZRYELVg90H21EOLACvtfcHk0ljxTGUaujjU1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXlsVWqu70%253D&md5=1d31622c99a7b4dc43b0908546cce975</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.chom.2010.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chom.2010.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DPagans%26aufirst%3DS.%26aulast%3DKauder%26aufirst%3DS.%2BE.%26aulast%3DKaehlcke%26aufirst%3DK.%26aulast%3DSakane%26aufirst%3DN.%26aulast%3DSchroeder%26aufirst%3DS.%26aulast%3DDormeyer%26aufirst%3DW.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26aulast%3DVerdin%26aufirst%3DE.%26aulast%3DSchnolzer%26aufirst%3DM.%26aulast%3DOtt%26aufirst%3DM.%26atitle%3DThe%2520Cellular%2520Lysine%2520Methyltransferase%2520Set7%252F9-KMT7%2520Binds%2520HIV-1%2520TAR%2520RNA%252C%2520Monomethylates%2520the%2520Viral%2520Transactivator%2520Tat%252C%2520and%2520Enhances%2520HIV%2520Transcription%26jtitle%3DCell%2520Host%2520Microbe%26date%3D2010%26volume%3D7%26spage%3D234%26epage%3D244%26doi%3D10.1016%2Fj.chom.2010.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Han, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">She, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Broering, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y.</span><span> </span><span class="NLM_article-title">Set7 facilitates hepatitis C virus replication via enzymatic activity-dependent attenuation of the IFN-related pathway</span> <span class="citation_source-journal">J. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">194</span><span class="NLM_x">, </span> <span class="NLM_fpage">2757</span><span class="NLM_x">–</span> <span class="NLM_lpage">68</span><span class="refDoi"> DOI: 10.4049/jimmunol.1400583</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4049%2Fjimmunol.1400583" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=194&publication_year=2015&pages=2757-68&author=T.+Hanauthor=Y.+Wanauthor=J.+Wangauthor=P.+Zhaoauthor=Y.+Yuanauthor=L.+Wangauthor=Y.+Sheauthor=R.+Broeringauthor=M.+Luauthor=L.+Yeauthor=Y.+Zhu&title=Set7+facilitates+hepatitis+C+virus+replication+via+enzymatic+activity-dependent+attenuation+of+the+IFN-related+pathway&doi=10.4049%2Fjimmunol.1400583"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.4049%2Fjimmunol.1400583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4049%252Fjimmunol.1400583%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DT.%26aulast%3DWan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DShe%26aufirst%3DY.%26aulast%3DBroering%26aufirst%3DR.%26aulast%3DLu%26aufirst%3DM.%26aulast%3DYe%26aufirst%3DL.%26aulast%3DZhu%26aufirst%3DY.%26atitle%3DSet7%2520facilitates%2520hepatitis%2520C%2520virus%2520replication%2520via%2520enzymatic%2520activity-dependent%2520attenuation%2520of%2520the%2520IFN-related%2520pathway%26jtitle%3DJ.%2520Immunol.%26date%3D2015%26volume%3D194%26spage%3D2757%26epage%3D68%26doi%3D10.4049%2Fjimmunol.1400583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Toledo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wahl, G. M.</span><span> </span><span class="NLM_article-title">Regulating the p53 pathway: in vitro hypotheses, in vivo veritas</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1038/nrc2012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrc2012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17128209" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Clsr3E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=909-23&author=F.+Toledoauthor=G.+M.+Wahl&title=Regulating+the+p53+pathway%3A+in+vitro+hypotheses%2C+in+vivo+veritas&doi=10.1038%2Fnrc2012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Regulating the p53 pathway: in vitro hypotheses, in vivo veritas</span></div><div class="casAuthors">Toledo, Franck; Wahl, Geoffrey M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-923</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Mutations in TP53, the gene that encodes the tumor suppressor p53, are found in 50% of human cancers, and increased levels of its neg. regulators MDM2 and MDM4 (also known as MDMX) down-regulate p53 function in many of the rest.  Understanding p53 regulation remains a crucial goal to design broadly applicable anticancer strategies based on this pathway.  This review of in vitro studies, human tumor data and recent mouse models shows that p53 post-translational modifications have modulatory roles, and MDM2 and MDM4 have more profound roles for regulating p53.  Importantly, MDM4 emerges as an independent target for drug development, as its inactivation is crucial for full p53 activation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqqN3wMkB-VB7Vg90H21EOLACvtfcHk0ljxTGUaujjU1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Clsr3E&md5=0f87a2342dd6dcd4ab3de2a0245eabd6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1038%2Fnrc2012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2012%26sid%3Dliteratum%253Aachs%26aulast%3DToledo%26aufirst%3DF.%26aulast%3DWahl%26aufirst%3DG.%2BM.%26atitle%3DRegulating%2520the%2520p53%2520pathway%253A%2520in%2520vitro%2520hypotheses%252C%2520in%2520vivo%2520veritas%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D909%26epage%3D23%26doi%3D10.1038%2Fnrc2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Chuikov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ivanov, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McKinney, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tempst, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prives, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barlev, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">Regulation of p53 activity through lysine methylation</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">432</span><span class="NLM_x">, </span> <span class="NLM_fpage">353</span><span class="NLM_x">–</span> <span class="NLM_lpage">360</span><span class="refDoi"> DOI: 10.1038/nature03117</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnature03117" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=15525938" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=432&publication_year=2004&pages=353-360&author=S.+Chuikovauthor=J.+K.+Kurashauthor=J.+R.+Wilsonauthor=B.+Xiaoauthor=N.+Justinauthor=G.+S.+Ivanovauthor=K.+McKinneyauthor=P.+Tempstauthor=C.+Privesauthor=S.+J.+Gamblinauthor=N.+A.+Barlevauthor=D.+Reinberg&title=Regulation+of+p53+activity+through+lysine+methylation&doi=10.1038%2Fnature03117"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of p53 activity through lysine methylation</span></div><div class="casAuthors">Chuikov, Sergei; Kurash, Julia K.; Wilson, Jonathan R.; Xiao, Bing; Justin, Neil; Ivanov, Gleb S.; McKinney, Kristine; Tempst, Paul; Prives, Carol; Gamblin, Steven J.; Barlev, Nickolai A.; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">432</span>
        (<span class="NLM_cas:issue">7015</span>),
    <span class="NLM_cas:pages">353-360</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">P53 is a tumor suppressor that regulates the cellular response to genotoxic stresses. p53 is a short-lived protein and its activity is regulated mostly by stabilization via different post-translational modifications.  Here we report a novel mechanism of p53 regulation through lysine methylation by Set9 methyltransferase.  Set9 specifically methylates p53 at one residue within the carboxyl-terminus regulatory region.  Methylated p53 is restricted to the nucleus and the modification pos. affects its stability.  Set9 regulates the expression of p53 target genes in a manner dependent on the p53-methylation site.  The crystal structure of a ternary complex of Set9 with a p53 peptide and the cofactor product S-adenosyl-L-homocysteine (AdoHcy) provides the mol. basis for recognition of p53 by this lysine methyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQEN46GC7xzLVg90H21EOLACvtfcHk0ljxTGUaujjU1A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXpvVCjtbo%253D&md5=eff3a7ec7f6adbf1fc43016325f074d0</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03117&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03117%26sid%3Dliteratum%253Aachs%26aulast%3DChuikov%26aufirst%3DS.%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DXiao%26aufirst%3DB.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DIvanov%26aufirst%3DG.%2BS.%26aulast%3DMcKinney%26aufirst%3DK.%26aulast%3DTempst%26aufirst%3DP.%26aulast%3DPrives%26aufirst%3DC.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26aulast%3DBarlev%26aufirst%3DN.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DRegulation%2520of%2520p53%2520activity%2520through%2520lysine%2520methylation%26jtitle%3DNature%26date%3D2004%26volume%3D432%26spage%3D353%26epage%3D360%26doi%3D10.1038%2Fnature03117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Kurash, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lei, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marston, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granda, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wall, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, F.</span><span> </span><span class="NLM_article-title">Methylation of p53 by Set7/9 mediates p53 acetylation and activity in vivo</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">392</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span><span class="refDoi"> DOI: 10.1016/j.molcel.2007.12.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2007.12.025" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2008&pages=392-400&author=J.+K.+Kurashauthor=H.+Leiauthor=Q.+Shenauthor=W.+L.+Marstonauthor=B.+W.+Grandaauthor=H.+Fanauthor=D.+Wallauthor=E.+Liauthor=F.+Gaudet&title=Methylation+of+p53+by+Set7%2F9+mediates+p53+acetylation+and+activity+in+vivo&doi=10.1016%2Fj.molcel.2007.12.025"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2007.12.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2007.12.025%26sid%3Dliteratum%253Aachs%26aulast%3DKurash%26aufirst%3DJ.%2BK.%26aulast%3DLei%26aufirst%3DH.%26aulast%3DShen%26aufirst%3DQ.%26aulast%3DMarston%26aufirst%3DW.%2BL.%26aulast%3DGranda%26aufirst%3DB.%2BW.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DWall%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DGaudet%26aufirst%3DF.%26atitle%3DMethylation%2520of%2520p53%2520by%2520Set7%252F9%2520mediates%2520p53%2520acetylation%2520and%2520activity%2520in%2520vivo%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D29%26spage%3D392%26epage%3D400%26doi%3D10.1016%2Fj.molcel.2007.12.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roeder, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W.-G.</span><span> </span><span class="NLM_article-title">Methyltransferase Set7/9 regulates p53 activity by interacting with Sirtuin 1 (SIRT1)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">108</span><span class="NLM_x">, </span> <span class="NLM_fpage">1925</span><span class="NLM_x">–</span> <span class="NLM_lpage">1930</span><span class="refDoi"> DOI: 10.1073/pnas.1019619108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1019619108" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=1925-1930&author=X.+Liuauthor=D.+Wangauthor=Y.+Zhaoauthor=B.+Tuauthor=Z.+Zhengauthor=L.+Wangauthor=H.+Wangauthor=W.+Guauthor=R.+G.+Roederauthor=W.-G.+Zhu&title=Methyltransferase+Set7%2F9+regulates+p53+activity+by+interacting+with+Sirtuin+1+%28SIRT1%29&doi=10.1073%2Fpnas.1019619108"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1019619108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1019619108%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DD.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DTu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DGu%26aufirst%3DW.%26aulast%3DRoeder%26aufirst%3DR.%2BG.%26aulast%3DZhu%26aufirst%3DW.-G.%26atitle%3DMethyltransferase%2520Set7%252F9%2520regulates%2520p53%2520activity%2520by%2520interacting%2520with%2520Sirtuin%25201%2520%2528SIRT1%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2011%26volume%3D108%26spage%3D1925%26epage%3D1930%26doi%3D10.1073%2Fpnas.1019619108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">West, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gozani, O.</span><span> </span><span class="NLM_article-title">Regulation of p53 function by lysine methylation</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">361</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.2217/epi.11.21</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2217%2Fepi.11.21" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=361-9&author=L.+E.+Westauthor=O.+Gozani&title=Regulation+of+p53+function+by+lysine+methylation&doi=10.2217%2Fepi.11.21"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.2217%2Fepi.11.21&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.11.21%26sid%3Dliteratum%253Aachs%26aulast%3DWest%26aufirst%3DL.%2BE.%26aulast%3DGozani%26aufirst%3DO.%26atitle%3DRegulation%2520of%2520p53%2520function%2520by%2520lysine%2520methylation%26jtitle%3DEpigenomics%26date%3D2011%26volume%3D3%26spage%3D361%26epage%3D9%26doi%3D10.2217%2Fepi.11.21" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Lehnertz, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rogalski, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kast, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M. V.</span><span> </span><span class="NLM_article-title">p53-Dependent Transcription and Tumor Suppression Are Not Affected in Set7/9-Deficient Mice</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">673</span><span class="NLM_x">–</span> <span class="NLM_lpage">680</span><span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2011.08.006" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=673-680&author=B.+Lehnertzauthor=J.+C.+Rogalskiauthor=F.+M.+Schulzeauthor=L.+Yiauthor=S.+Linauthor=J.+Kastauthor=F.+M.+V.+Rossi&title=p53-Dependent+Transcription+and+Tumor+Suppression+Are+Not+Affected+in+Set7%2F9-Deficient+Mice&doi=10.1016%2Fj.molcel.2011.08.006"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.006%26sid%3Dliteratum%253Aachs%26aulast%3DLehnertz%26aufirst%3DB.%26aulast%3DRogalski%26aufirst%3DJ.%2BC.%26aulast%3DSchulze%26aufirst%3DF.%2BM.%26aulast%3DYi%26aufirst%3DL.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DKast%26aufirst%3DJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%2BV.%26atitle%3Dp53-Dependent%2520Transcription%2520and%2520Tumor%2520Suppression%2520Are%2520Not%2520Affected%2520in%2520Set7%252F9-Deficient%2520Mice%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D43%26spage%3D673%26epage%3D680%26doi%3D10.1016%2Fj.molcel.2011.08.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Campaner, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spreafico, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgold, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doni, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosato, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amati, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Testa, G.</span><span> </span><span class="NLM_article-title">The Methyltransferase Set7/9 (Setd7) Is Dispensable for the p53-Mediated DNA Damage Response In Vivo</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">681</span><span class="NLM_x">–</span> <span class="NLM_lpage">688</span><span class="refDoi"> DOI: 10.1016/j.molcel.2011.08.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.molcel.2011.08.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21855806" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtVGku7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2011&pages=681-688&author=S.+Campanerauthor=F.+Spreaficoauthor=T.+Burgoldauthor=M.+Doniauthor=U.+Rosatoauthor=B.+Amatiauthor=G.+Testa&title=The+Methyltransferase+Set7%2F9+%28Setd7%29+Is+Dispensable+for+the+p53-Mediated+DNA+Damage+Response+In+Vivo&doi=10.1016%2Fj.molcel.2011.08.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">The Methyltransferase Set7/9 (Setd7) Is Dispensable for the p53-Mediated DNA Damage Response In Vivo</span></div><div class="casAuthors">Campaner, Stefano; Spreafico, Fabio; Burgold, Thomas; Doni, Mirko; Rosato, Umberto; Amati, Bruno; Testa, Giuseppe</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">681-688</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Summary: p53 is the central regulator of cell fate following genotoxic stress and oncogene activation.  Its activity is controlled by several posttranslational modifications.  Originally defined as a crit. layer of p53 regulation in human cell lines, p53 lysine methylation by Set7/9 (also called Setd7) was proposed to fulfill a similar function in vivo in the mouse, promoting p53 acetylation, stabilization, and activation upon DNA damage ().  We tested the physiol. relevance of this circuit in an independent Set7/9 knockout mouse strain.  Deletion of Set7/9 had no effect on p53-dependent cell-cycle arrest or apoptosis following sublethal or lethal DNA damage induced by radiation or genotoxic agents.  Set7/9 was also dispensable for p53 acetylation following irradn. c-myc oncogene-induced apoptosis was also independent of Set7/9, and anal. of p53 target genes showed that Set7/9 is not required for the p53-dependent gene expression program.  Our data indicate that Set7/9 is dispensable for p53 function in the mouse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvFipBxRO0rrVg90H21EOLACvtfcHk0lhDGXhz_qD69w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtVGku7zO&md5=6ee511977f70e48a9fb509ad3e871701</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2011.08.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2011.08.007%26sid%3Dliteratum%253Aachs%26aulast%3DCampaner%26aufirst%3DS.%26aulast%3DSpreafico%26aufirst%3DF.%26aulast%3DBurgold%26aufirst%3DT.%26aulast%3DDoni%26aufirst%3DM.%26aulast%3DRosato%26aufirst%3DU.%26aulast%3DAmati%26aufirst%3DB.%26aulast%3DTesta%26aufirst%3DG.%26atitle%3DThe%2520Methyltransferase%2520Set7%252F9%2520%2528Setd7%2529%2520Is%2520Dispensable%2520for%2520the%2520p53-Mediated%2520DNA%2520Damage%2520Response%2520In%2520Vivo%26jtitle%3DMol.%2520Cell%26date%3D2011%26volume%3D43%26spage%3D681%26epage%3D688%26doi%3D10.1016%2Fj.molcel.2011.08.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Mori, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwase, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwanami, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanaka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagechika, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, T.</span><span> </span><span class="NLM_article-title">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">8158</span><span class="NLM_x">–</span> <span class="NLM_lpage">8166</span><span class="refDoi"> DOI: 10.1016/j.bmc.2010.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2010.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21036620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8158-8166&author=S.+Moriauthor=K.+Iwaseauthor=N.+Iwanamiauthor=Y.+Tanakaauthor=H.+Kagechikaauthor=T.+Hirano&title=Development+of+novel+bisubstrate-type+inhibitors+of+histone+methyltransferase+SET7%2F9&doi=10.1016%2Fj.bmc.2010.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span></div><div class="casAuthors">Mori, Shuichi; Iwase, Kenta; Iwanami, Naoko; Tanaka, Yujiro; Kagechika, Hiroyuki; Hirano, Tomoya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8158-8166</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone modification, for example, by histone deacetylase (HDAC) and histone lysine methyltransferase (HMT), plays an important role in regulating gene expression.  To obtain novel inhibitors as tools for investigating the physiol. function of members of the HMT family, we designed and synthesized novel inhibitors, which are amine analogs of adenosylmethionine (AdoMet; the cofactor utilized in the methylation reaction) bearing various alkylamino groups coupled via an ethylene linker.  The inhibitory activities of these compds. towards SET7/9, an HMT, were evaluated.  It was found that introduction of an alkylamino group increased the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUp7neuWTg1rVg90H21EOLACvtfcHk0lhDGXhz_qD69w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL&md5=76faa42f957d02ed47458c348b324929</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DS.%26aulast%3DIwase%26aufirst%3DK.%26aulast%3DIwanami%26aufirst%3DN.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520novel%2520bisubstrate-type%2520inhibitors%2520of%2520histone%2520methyltransferase%2520SET7%252F9%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8158%26epage%3D8166%26doi%3D10.1016%2Fj.bmc.2010.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Niwa, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Handa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomabechi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohsawa, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shirouzu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kagechika, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirano, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Umehara, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoyama, S.</span><span> </span><span class="NLM_article-title">Structures of histone methyltransferase SET7/9 in complexes with adenosylmethionine derivatives</span> <span class="citation_source-journal">Acta Crystallogr., Sect. D: Biol. Crystallogr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">69</span><span class="NLM_x">, </span> <span class="NLM_fpage">595</span><span class="NLM_x">–</span> <span class="NLM_lpage">602</span><span class="refDoi"> DOI: 10.1107/S0907444912052092</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1107%2FS0907444912052092" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=595-602&author=H.+Niwaauthor=N.+Handaauthor=Y.+Tomabechiauthor=K.+Hondaauthor=M.+Toyamaauthor=N.+Ohsawaauthor=M.+Shirouzuauthor=H.+Kagechikaauthor=T.+Hiranoauthor=T.+Umeharaauthor=S.+Yokoyama&title=Structures+of+histone+methyltransferase+SET7%2F9+in+complexes+with+adenosylmethionine+derivatives&doi=10.1107%2FS0907444912052092"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444912052092&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444912052092%26sid%3Dliteratum%253Aachs%26aulast%3DNiwa%26aufirst%3DH.%26aulast%3DHanda%26aufirst%3DN.%26aulast%3DTomabechi%26aufirst%3DY.%26aulast%3DHonda%26aufirst%3DK.%26aulast%3DToyama%26aufirst%3DM.%26aulast%3DOhsawa%26aufirst%3DN.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26aulast%3DUmehara%26aufirst%3DT.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DStructures%2520of%2520histone%2520methyltransferase%2520SET7%252F9%2520in%2520complexes%2520with%2520adenosylmethionine%2520derivatives%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D595%26epage%3D602%26doi%3D10.1107%2FS0907444912052092" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Dowden, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parry, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pike, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, S. G.</span><span> </span><span class="NLM_article-title">Toward the development of potent and selective bisubstrate inhibitors of protein arginine methyltransferases</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">2103</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2010.02.069</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmcl.2010.02.069" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=20219369" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXjslGmsb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2010&pages=2103-5&author=J.+Dowdenauthor=W.+Hongauthor=R.+V.+Parryauthor=R.+A.+Pikeauthor=S.+G.+Ward&title=Toward+the+development+of+potent+and+selective+bisubstrate+inhibitors+of+protein+arginine+methyltransferases&doi=10.1016%2Fj.bmcl.2010.02.069"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Toward the development of potent and selective bisubstrate inhibitors of protein arginine methyltransferases</span></div><div class="casAuthors">Dowden, James; Hong, Wei; Parry, Richard V.; Pike, Richard A.; Ward, Stephen G.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2103-2105</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Prototype inhibitors of protein arginine methyltransferases (PRMTs) have been constructed by attaching guanidine functionality via a variable linker to non-reactive amine analogs of the cellular co-factor (S)-adenosyl methionine (AdoMet).  Potent inhibition of PRMT1 (IC50 of ∼3-6 μM) combined with weak inhibition of the lysine methyltransferase SET7 (∼50% of activity at 100 μM) was obsd. for two such compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrj5FdIemXEXrVg90H21EOLACvtfcHk0lhDGXhz_qD69w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXjslGmsb4%253D&md5=6401f6157aa931ae23ac27928d4bf1d1</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2010.02.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2010.02.069%26sid%3Dliteratum%253Aachs%26aulast%3DDowden%26aufirst%3DJ.%26aulast%3DHong%26aufirst%3DW.%26aulast%3DParry%26aufirst%3DR.%2BV.%26aulast%3DPike%26aufirst%3DR.%2BA.%26aulast%3DWard%26aufirst%3DS.%2BG.%26atitle%3DToward%2520the%2520development%2520of%2520potent%2520and%2520selective%2520bisubstrate%2520inhibitors%2520of%2520protein%2520arginine%2520methyltransferases%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2010%26volume%3D20%26spage%3D2103%26epage%3D5%26doi%3D10.1016%2Fj.bmcl.2010.02.069" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Bissinger, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlin, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cura, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hassenboehler, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cavarelli, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Acyl derivatives of p-aminosulfonamides and dapsone as new inhibitors of the arginine methyltransferase hPRMT1</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3717</span><span class="NLM_x">–</span> <span class="NLM_lpage">3731</span><span class="refDoi"> DOI: 10.1016/j.bmc.2011.02.032</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2011.02.032" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2011&pages=3717-3731&author=E.+M.+Bissingerauthor=R.+Heinkeauthor=A.+Spannhoffauthor=A.+Eberlinauthor=E.+Metzgerauthor=V.+Curaauthor=P.+Hassenboehlerauthor=J.+Cavarelliauthor=R.+Schuleauthor=M.+T.+Bedfordauthor=W.+Sipplauthor=M.+Jung&title=Acyl+derivatives+of+p-aminosulfonamides+and+dapsone+as+new+inhibitors+of+the+arginine+methyltransferase+hPRMT1&doi=10.1016%2Fj.bmc.2011.02.032"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2011.02.032&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2011.02.032%26sid%3Dliteratum%253Aachs%26aulast%3DBissinger%26aufirst%3DE.%2BM.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DEberlin%26aufirst%3DA.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DCura%26aufirst%3DV.%26aulast%3DHassenboehler%26aufirst%3DP.%26aulast%3DCavarelli%26aufirst%3DJ.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DAcyl%2520derivatives%2520of%2520p-aminosulfonamides%2520and%2520dapsone%2520as%2520new%2520inhibitors%2520of%2520the%2520arginine%2520methyltransferase%2520hPRMT1%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2011%26volume%3D19%26spage%3D3717%26epage%3D3731%26doi%3D10.1016%2Fj.bmc.2011.02.032" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suarez, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Newlander, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romeril, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burgess, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grant, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brackley, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scherzer, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aller, G. S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machutta, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knight, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span> </span><span class="NLM_article-title">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span> <span class="citation_source-journal">ACS Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1091</span><span class="NLM_x">–</span> <span class="NLM_lpage">1096</span><span class="refDoi"> DOI: 10.1021/ml3003346</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ml3003346" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2012&pages=1091-1096&author=S.+K.+Vermaauthor=X.+Tianauthor=L.+V.+LaFranceauthor=C.+Duquenneauthor=D.+P.+Suarezauthor=K.+A.+Newlanderauthor=S.+P.+Romerilauthor=J.+L.+Burgessauthor=S.+W.+Grantauthor=J.+A.+Brackleyauthor=A.+P.+Gravesauthor=D.+A.+Scherzerauthor=A.+Shuauthor=C.+Thompsonauthor=H.+M.+Ottauthor=G.+S.+V.+Allerauthor=C.+A.+Machuttaauthor=E.+Diazauthor=Y.+Jiangauthor=N.+W.+Johnsonauthor=S.+D.+Knightauthor=R.+G.+Krugerauthor=M.+T.+McCabeauthor=D.+Dhanakauthor=P.+J.+Tumminoauthor=C.+L.+Creasyauthor=W.+H.+Miller&title=Identification+of+Potent%2C+Selective%2C+Cell-Active+Inhibitors+of+the+Histone+Lysine+Methyltransferase+EZH2&doi=10.1021%2Fml3003346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Potent, Selective, Cell-Active Inhibitors of the Histone Lysine Methyltransferase EZH2</span></div><div class="casAuthors">Verma, Sharad K.; Tian, Xinrong; LaFrance, Louis V.; Duquenne, Celine; Suarez, Dominic P.; Newlander, Kenneth A.; Romeril, Stuart P.; Burgess, Joelle L.; Grant, Seth W.; Brackley, James A.; Graves, Alan P.; Scherzer, Daryl A.; Shu, Art; Thompson, Christine; Ott, Heidi M.; Van Aller, Glenn S.; Machutta, Carl A.; Diaz, Elsie; Jiang, Yong; Johnson, Neil W.; Knight, Steven D.; Kruger, Ryan G.; McCabe, Michael T.; Dhanak, Dashyant; Tummino, Peter J.; Creasy, Caretha L.; Miller, William H.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1091-1096</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The histone H3-lysine 27 (H3K27) methyltransferase EZH2 plays a crit. role in regulating gene expression, and its aberrant activity is linked to the onset and progression of cancer.  As part of a drug discovery program targeting EZH2, we have identified highly potent, selective, S-(S'-adenosyl)-L-methionine (SAM)-competitive, and cell-active EZH2 inhibitors, including GSK926 I (R1 = Me, R2 = Q) and GSK343 I (R1 = n-Pr, R2 = Q1).  These compds. are small mol. chem. tools that would be useful to further explore the biol. of EZH2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8NUzf0zjo0bVg90H21EOLACvtfcHk0lgnKJlr95kAnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsFChs77K&md5=a6540bb712ee52cba05bd0b074d8b26a</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1021%2Fml3003346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fml3003346%26sid%3Dliteratum%253Aachs%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DSuarez%26aufirst%3DD.%2BP.%26aulast%3DNewlander%26aufirst%3DK.%2BA.%26aulast%3DRomeril%26aufirst%3DS.%2BP.%26aulast%3DBurgess%26aufirst%3DJ.%2BL.%26aulast%3DGrant%26aufirst%3DS.%2BW.%26aulast%3DBrackley%26aufirst%3DJ.%2BA.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DScherzer%26aufirst%3DD.%2BA.%26aulast%3DShu%26aufirst%3DA.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DAller%26aufirst%3DG.%2BS.%2BV.%26aulast%3DMachutta%26aufirst%3DC.%2BA.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DJiang%26aufirst%3DY.%26aulast%3DJohnson%26aufirst%3DN.%2BW.%26aulast%3DKnight%26aufirst%3DS.%2BD.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26atitle%3DIdentification%2520of%2520Potent%252C%2520Selective%252C%2520Cell-Active%2520Inhibitors%2520of%2520the%2520Histone%2520Lysine%2520Methyltransferase%2520EZH2%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2012%26volume%3D3%26spage%3D1091%26epage%3D1096%26doi%3D10.1021%2Fml3003346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Kashyap, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandler, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kapoor, T. M.</span><span> </span><span class="NLM_article-title">Using ’biased-privileged’ scaffolds to identify lysine methyltransferase inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">2253</span><span class="NLM_x">–</span> <span class="NLM_lpage">60</span><span class="refDoi"> DOI: 10.1016/j.bmc.2014.02.024</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmc.2014.02.024" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2014&pages=2253-60&author=S.+Kashyapauthor=J.+Sandlerauthor=U.+Petersauthor=E.+J.+Martinezauthor=T.+M.+Kapoor&title=Using+%E2%80%99biased-privileged%E2%80%99+scaffolds+to+identify+lysine+methyltransferase+inhibitors&doi=10.1016%2Fj.bmc.2014.02.024"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2014.02.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2014.02.024%26sid%3Dliteratum%253Aachs%26aulast%3DKashyap%26aufirst%3DS.%26aulast%3DSandler%26aufirst%3DJ.%26aulast%3DPeters%26aufirst%3DU.%26aulast%3DMartinez%26aufirst%3DE.%2BJ.%26aulast%3DKapoor%26aufirst%3DT.%2BM.%26atitle%3DUsing%2520%25E2%2580%2599biased-privileged%25E2%2580%2599%2520scaffolds%2520to%2520identify%2520lysine%2520methyltransferase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2014%26volume%3D22%26spage%3D2253%26epage%3D60%26doi%3D10.1016%2Fj.bmc.2014.02.024" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Santo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Costi, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span> </span><span class="NLM_article-title">Novel 3,5-Bis(bromohydroxybenzylidene)piperidin-4-ones as Coactivator-Associated Arginine Methyltransferase 1 Inhibitors: Enzyme Selectivity and Cellular Activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">4928</span><span class="NLM_x">–</span> <span class="NLM_lpage">4932</span><span class="refDoi"> DOI: 10.1021/jm200453n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm200453n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnsVSjtrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=4928-4932&author=D.+Chengauthor=S.+Valenteauthor=S.+Castellanoauthor=G.+Sbardellaauthor=R.+Di+Santoauthor=R.+Costiauthor=M.+T.+Bedfordauthor=A.+Mai&title=Novel+3%2C5-Bis%28bromohydroxybenzylidene%29piperidin-4-ones+as+Coactivator-Associated+Arginine+Methyltransferase+1+Inhibitors%3A+Enzyme+Selectivity+and+Cellular+Activity&doi=10.1021%2Fjm200453n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Novel 3,5-Bis(bromohydroxybenzylidene)piperidin-4-ones as Coactivator-Associated Arginine Methyltransferase 1 Inhibitors: Enzyme Selectivity and Cellular Activity</span></div><div class="casAuthors">Cheng, Donghang; Valente, Sergio; Castellano, Sabrina; Sbardella, Gianluca; Di Santo, Roberto; Costi, Roberta; Bedford, Mark T.; Mai, Antonello</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">4928-4932</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Coactivator-assocd. arginine methyltransferase 1 (CARM1) represents a valuable target for hormone-dependent tumors such as prostate and breast cancers.  Here we report the enzyme and cellular characterization of the 1-benzyl-3,5-bis(3-bromo-4-hydroxybenzylidene)piperidin-4-one (7g) and its analogs 8a-l.  Among them, 7g, 8e, and 8l displayed high and selective CARM1 inhibition, with lower or no activity against a panel of different PRMTs or HKMTs.  In human LNCaP cells, 7g showed a significant dose-dependent redn. of the PSA promoter activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpXv4DaWjduarVg90H21EOLACvtfcHk0li4t9sCdE_Q4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnsVSjtrw%253D&md5=c9fba6d328440a5e41046ff0c47f661c</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fjm200453n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm200453n%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDi%2BSanto%26aufirst%3DR.%26aulast%3DCosti%26aufirst%3DR.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DMai%26aufirst%3DA.%26atitle%3DNovel%25203%252C5-Bis%2528bromohydroxybenzylidene%2529piperidin-4-ones%2520as%2520Coactivator-Associated%2520Arginine%2520Methyltransferase%25201%2520Inhibitors%253A%2520Enzyme%2520Selectivity%2520and%2520Cellular%2520Activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D4928%26epage%3D4932%26doi%3D10.1021%2Fjm200453n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conte, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span> </span><span class="NLM_article-title">Synthesis and biological validation of novel synthetic histone/protein methyltransferase inhibitors</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">987</span><span class="NLM_x">–</span> <span class="NLM_lpage">991</span><span class="refDoi"> DOI: 10.1002/cmdc.200700023</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fcmdc.200700023" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17458842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXotVyktLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=987-991&author=A.+Maiauthor=S.+Valenteauthor=D.+Chengauthor=A.+Perroneauthor=R.+Ragnoauthor=S.+Simeoniauthor=G.+Sbardellaauthor=G.+Broschauthor=A.+Nebbiosoauthor=M.+Conteauthor=L.+Altucciauthor=M.+T.+Bedford&title=Synthesis+and+biological+validation+of+novel+synthetic+histone%2Fprotein+methyltransferase+inhibitors&doi=10.1002%2Fcmdc.200700023"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and biological validation of novel synthetic histone/protein methyltransferase inhibitors</span></div><div class="casAuthors">Mai, Antonello; Valente, Sergio; Cheng, Donghang; Perrone, Andrea; Ragno, Rino; Simeoni, Silvia; Sbardella, Gianluca; Brosch, Gerald; Nebbioso, Angela; Conte, Mariarosaria; Altucci, Lucia; Bedford, Mark T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">987-991</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The prepn. of novel histone/protein methyltransferase inhibitors is described and their potential as antitumor agents was evaluated in human leukemia cell line.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoCBG8vwlib8bVg90H21EOLACvtfcHk0li4t9sCdE_Q4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXotVyktLk%253D&md5=cb0e3448e787d65760ee7d4563cfbb75</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200700023&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200700023%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DConte%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26atitle%3DSynthesis%2520and%2520biological%2520validation%2520of%2520novel%2520synthetic%2520histone%252Fprotein%2520methyltransferase%2520inhibitors%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D987%26epage%3D991%26doi%3D10.1002%2Fcmdc.200700023" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Nishino, N.; Takemoto, Y.; Ito, A.; Yoshida, M.; Yamori, T.</span> Preventive and therapeutic agent of disease containing inhibitor of protein ricin methylation enzyme Set7/9. Japanese Patent JP2012102087,<span class="NLM_x"> </span><span class="NLM_year">2011</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Nishino%2C+N.%3B+Takemoto%2C+Y.%3B+Ito%2C+A.%3B+Yoshida%2C+M.%3B+Yamori%2C+T.+Preventive+and+therapeutic+agent+of+disease+containing+inhibitor+of+protein+ricin+methylation+enzyme+Set7%2F9.+Japanese+Patent+JP2012102087%2C+2011."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DNishino%26aufirst%3DN.%26date%3D2011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oudhoff, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tatlock, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zeng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Freeman, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Senisterra, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcellus, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kennedy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lambert, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couzens, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aman, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gingras, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Awar, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fish, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerstenberger, B. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benn, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grimley, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braam, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rossi, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudol, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bunnage, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owen, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaph, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span> </span><span class="NLM_article-title">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">12853</span><span class="NLM_x">–</span> <span class="NLM_lpage">12858</span><span class="refDoi"> DOI: 10.1073/pnas.1407358111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1073%2Fpnas.1407358111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25136132" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12853-12858&author=D.+Barsyte-Lovejoyauthor=F.+Liauthor=M.+J.+Oudhoffauthor=J.+H.+Tatlockauthor=A.+Dongauthor=H.+Zengauthor=H.+Wuauthor=S.+A.+Freemanauthor=M.+Schapiraauthor=G.+A.+Senisterraauthor=E.+Kuznetsovaauthor=R.+Marcellusauthor=A.+Allali-Hassaniauthor=S.+Kennedyauthor=J.+P.+Lambertauthor=A.+L.+Couzensauthor=A.+Amanauthor=A.+C.+Gingrasauthor=R.+Al-Awarauthor=P.+V.+Fishauthor=B.+S.+Gerstenbergerauthor=L.+Robertsauthor=C.+L.+Bennauthor=R.+L.+Grimleyauthor=M.+J.+Braamauthor=F.+M.+Rossiauthor=M.+Sudolauthor=P.+J.+Brownauthor=M.+E.+Bunnageauthor=D.+R.+Owenauthor=C.+Zaphauthor=M.+Vedadiauthor=C.+H.+Arrowsmith&title=%28R%29-PFI-2+is+a+potent+and+selective+inhibitor+of+SETD7+methyltransferase+activity+in+cells&doi=10.1073%2Fpnas.1407358111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">(R)-PFI-2 is a potent and selective inhibitor of SETD7 methyltransferase activity in cells</span></div><div class="casAuthors">Barsyte-Lovejoy, Dalia; Li, Fengling; Oudhoff, Menno J.; Tatlock, John H.; Dong, Aiping; Zeng, Hong; Wu, Hong; Freeman, Spencer A.; Schapira, Matthieu; Senisterra, Guillermo A.; Kuznetsova, Ekaterina; Marcellus, Richard; Allali-Hassani, Abdellah; Kennedy, Steven; Lambert, Jean-Philippe; Couzens, Amber L.; Aman, Ahmed; Gingras, Anne-Claude; Al-Awar, Rima; Fish, Paul V.; Gerstenberger, Brian S.; Roberts, Lee; Benn, Caroline L.; Grimley, Rachel L.; Braam, Mitchell J. S.; Rossi, Fabio M. V.; Sudol, Marius; Brown, Peter J.; Bunnage, Mark E.; Owen, Dafydd R.; Zaph, Colby; Vedadi, Masoud; Arrowsmith, Cheryl H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12853-12858</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">SET domain contg. (lysine methyltransferase) 7 (SETD7) is implicated in multiple signaling and disease related pathways with a broad diversity of reported substrates.  Here, we report the discovery of (R)-PFI-2-a first-in-class, potent (Kiapp = 0.33 nM), selective, and cell-active inhibitor of the methyltransferase activity of human SETD7-and its 500-fold less active enantiomer, (S)-PFI-2.  (R)-PFI-2 exhibits an unusual cofactor-dependent and substrate-competitive inhibitory mechanism by occupying the substrate peptide binding groove of SETD7, including the catalytic lysine-binding channel, and by making direct contact with the donor Me group of the cofactor, S-adenosylmethionine.  Chemoproteomics expts. using a biotinylated deriv. of (R)-PFI-2 demonstrated dose-dependent competition for binding to endogenous SETD7 in MCF7 cells pretreated with (R)-PFI-2.  In murine embryonic fibroblasts, (R)-PFI-2 treatment phenocopied the effects of Setd7 deficiency on Hippo pathway signaling, via modulation of the transcriptional coactivator Yes-assocd. protein (YAP) and regulation of YAP target genes.  In confluent MCF7 cells, (R)-PFI-2 rapidly altered YAP localization, suggesting continuous and dynamic regulation of YAP by the methyltransferase activity of SETD7.  These data establish (R)-PFI-2 and related compds. as a valuable tool-kit for the study of the diverse roles of SETD7 in cells and further validate protein methyltransferases as a druggable target class.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv-HTBIcjMnbVg90H21EOLACvtfcHk0lhOfbicYOX8HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVartr7E&md5=0b8ad07ae23c4eba5cf11060d231b3f4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1407358111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1407358111%26sid%3Dliteratum%253Aachs%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DOudhoff%26aufirst%3DM.%2BJ.%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DZeng%26aufirst%3DH.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DFreeman%26aufirst%3DS.%2BA.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DSenisterra%26aufirst%3DG.%2BA.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DMarcellus%26aufirst%3DR.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DKennedy%26aufirst%3DS.%26aulast%3DLambert%26aufirst%3DJ.%2BP.%26aulast%3DCouzens%26aufirst%3DA.%2BL.%26aulast%3DAman%26aufirst%3DA.%26aulast%3DGingras%26aufirst%3DA.%2BC.%26aulast%3DAl-Awar%26aufirst%3DR.%26aulast%3DFish%26aufirst%3DP.%2BV.%26aulast%3DGerstenberger%26aufirst%3DB.%2BS.%26aulast%3DRoberts%26aufirst%3DL.%26aulast%3DBenn%26aufirst%3DC.%2BL.%26aulast%3DGrimley%26aufirst%3DR.%2BL.%26aulast%3DBraam%26aufirst%3DM.%2BJ.%26aulast%3DRossi%26aufirst%3DF.%2BM.%26aulast%3DSudol%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DBunnage%26aufirst%3DM.%2BE.%26aulast%3DOwen%26aufirst%3DD.%2BR.%26aulast%3DZaph%26aufirst%3DC.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26atitle%3D%2528R%2529-PFI-2%2520is%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520SETD7%2520methyltransferase%2520activity%2520in%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D12853%26epage%3D12858%26doi%3D10.1073%2Fpnas.1407358111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mok, M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feher, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najmanovich, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Structural chemistry of the histone methyltransferases cofactor binding site</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">612</span><span class="NLM_x">–</span> <span class="NLM_lpage">23</span><span class="refDoi"> DOI: 10.1021/ci100479z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci100479z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXis1Oksrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=612-23&author=V.+Campagna-Slaterauthor=M.+W.+Mokauthor=K.+T.+Nguyenauthor=M.+Feherauthor=R.+Najmanovichauthor=M.+Schapira&title=Structural+chemistry+of+the+histone+methyltransferases+cofactor+binding+site&doi=10.1021%2Fci100479z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Chemistry of the Histone Methyltransferases Cofactor Binding Site</span></div><div class="casAuthors">Campagna-Slater, Valerie; Mok, Man Wai; Nguyen, Kong T.; Feher, Miklos; Najmanovich, Rafael; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">612-623</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) transfer a Me group from the cofactor S-adenosyl methionine to lysine or arginine residues on histone tails, thereby regulating chromatin compaction, binding of effector proteins and gene transcription.  HMTs constitute an emerging target class in diverse disease areas, and selective chem. probes are necessary for target validation.  Potent and selective competitors of the substrate peptide have been reported, but the chem. tractability of the cofactor binding site is poorly understood.  Here, a systematic anal. of this site across structures of 14 human HMTs or close homologues was conducted.  The druggability, interaction hotspots, and diversity of the cofactor binding pocket were dissected.  This anal. strongly suggests that this site is chem. tractable.  General principles underlying tight binding and specific guidelines to achieve selective inhibition are presented.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrnQVsPGdhHErVg90H21EOLACvtfcHk0lhOfbicYOX8HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXis1Oksrk%253D&md5=73ea6e829571a9c610acd7391d8ad082</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fci100479z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci100479z%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DMok%26aufirst%3DM.%2BW.%26aulast%3DNguyen%26aufirst%3DK.%2BT.%26aulast%3DFeher%26aufirst%3DM.%26aulast%3DNajmanovich%26aufirst%3DR.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DStructural%2520chemistry%2520of%2520the%2520histone%2520methyltransferases%2520cofactor%2520binding%2520site%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D612%26epage%3D23%26doi%3D10.1021%2Fci100479z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0lhOfbicYOX8HA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Tervo, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyrylenko, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niskanen, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salminen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leppanen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nyronen, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarvinen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poso, A.</span><span> </span><span class="NLM_article-title">An in silico approach to discovering novel inhibitors of human sirtuin type 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">6292</span><span class="NLM_x">–</span> <span class="NLM_lpage">6298</span><span class="refDoi"> DOI: 10.1021/jm049933m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm049933m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=6292-6298&author=A.+J.+Tervoauthor=S.+Kyrylenkoauthor=P.+Niskanenauthor=A.+Salminenauthor=J.+Leppanenauthor=T.+H.+Nyronenauthor=T.+Jarvinenauthor=A.+Poso&title=An+in+silico+approach+to+discovering+novel+inhibitors+of+human+sirtuin+type+2&doi=10.1021%2Fjm049933m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Fjm049933m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm049933m%26sid%3Dliteratum%253Aachs%26aulast%3DTervo%26aufirst%3DA.%2BJ.%26aulast%3DKyrylenko%26aufirst%3DS.%26aulast%3DNiskanen%26aufirst%3DP.%26aulast%3DSalminen%26aufirst%3DA.%26aulast%3DLeppanen%26aufirst%3DJ.%26aulast%3DNyronen%26aufirst%3DT.%2BH.%26aulast%3DJarvinen%26aufirst%3DT.%26aulast%3DPoso%26aufirst%3DA.%26atitle%3DAn%2520in%2520silico%2520approach%2520to%2520discovering%2520novel%2520inhibitors%2520of%2520human%2520sirtuin%2520type%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D6292%26epage%3D6298%26doi%3D10.1021%2Fjm049933m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Ragno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simeoni, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Castellano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vicidomini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caroli, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tramontano, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonaccini, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span> </span><span class="NLM_article-title">Small Molecule Inhibitors of Histone Arginine Methyltransferases: Homology Modeling, Molecular Docking, Binding Mode Analysis, and Biological Evaluations</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">1241</span><span class="NLM_x">–</span> <span class="NLM_lpage">1253</span><span class="refDoi"> DOI: 10.1021/jm061213n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061213n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=1241-1253&author=R.+Ragnoauthor=S.+Simeoniauthor=S.+Castellanoauthor=C.+Vicidominiauthor=A.+Maiauthor=A.+Caroliauthor=A.+Tramontanoauthor=C.+Bonacciniauthor=P.+Trojerauthor=I.+Bauerauthor=G.+Broschauthor=G.+Sbardella&title=Small+Molecule+Inhibitors+of+Histone+Arginine+Methyltransferases%3A+Homology+Modeling%2C+Molecular+Docking%2C+Binding+Mode+Analysis%2C+and+Biological+Evaluations&doi=10.1021%2Fjm061213n"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1021%2Fjm061213n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061213n%26sid%3Dliteratum%253Aachs%26aulast%3DRagno%26aufirst%3DR.%26aulast%3DSimeoni%26aufirst%3DS.%26aulast%3DCastellano%26aufirst%3DS.%26aulast%3DVicidomini%26aufirst%3DC.%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCaroli%26aufirst%3DA.%26aulast%3DTramontano%26aufirst%3DA.%26aulast%3DBonaccini%26aufirst%3DC.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26atitle%3DSmall%2520Molecule%2520Inhibitors%2520of%2520Histone%2520Arginine%2520Methyltransferases%253A%2520Homology%2520Modeling%252C%2520Molecular%2520Docking%252C%2520Binding%2520Mode%2520Analysis%252C%2520and%2520Biological%2520Evaluations%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D1241%26epage%3D1253%26doi%3D10.1021%2Fjm061213n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Spannhoff, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heinke, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gust, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuele, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Target-based approach to inhibitors of histone arginine methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">2319</span><span class="NLM_x">–</span> <span class="NLM_lpage">2325</span><span class="refDoi"> DOI: 10.1021/jm061250e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm061250e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=2319-2325&author=A.+Spannhoffauthor=R.+Heinkeauthor=I.+Bauerauthor=P.+Trojerauthor=E.+Metzgerauthor=R.+Gustauthor=R.+Schueleauthor=G.+Broschauthor=W.+Sipplauthor=M.+Jung&title=Target-based+approach+to+inhibitors+of+histone+arginine+methyltransferases&doi=10.1021%2Fjm061250e"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1021%2Fjm061250e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm061250e%26sid%3Dliteratum%253Aachs%26aulast%3DSpannhoff%26aufirst%3DA.%26aulast%3DHeinke%26aufirst%3DR.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DGust%26aufirst%3DR.%26aulast%3DSchuele%26aufirst%3DR.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DTarget-based%2520approach%2520to%2520inhibitors%2520of%2520histone%2520arginine%2520methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D2319%26epage%3D2325%26doi%3D10.1021%2Fjm061250e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Mai, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Valente, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebbioso, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perrone, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosch, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sbardella, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Bellis, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miceli, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Altucci, L.</span><span> </span><span class="NLM_article-title">Epigenetic multiple ligands: Mixed Histone/Protein methyltransferase, acetyltransferase, and class III deacetylase (Sirtuin) inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">2279</span><span class="NLM_x">–</span> <span class="NLM_lpage">2290</span><span class="refDoi"> DOI: 10.1021/jm701595q</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm701595q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=2279-2290&author=A.+Maiauthor=D.+Chengauthor=M.+T.+Bedfordauthor=S.+Valenteauthor=A.+Nebbiosoauthor=A.+Perroneauthor=G.+Broschauthor=G.+Sbardellaauthor=F.+De+Bellisauthor=M.+Miceliauthor=L.+Altucci&title=Epigenetic+multiple+ligands%3A+Mixed+Histone%2FProtein+methyltransferase%2C+acetyltransferase%2C+and+class+III+deacetylase+%28Sirtuin%29+inhibitors&doi=10.1021%2Fjm701595q"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm701595q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701595q%26sid%3Dliteratum%253Aachs%26aulast%3DMai%26aufirst%3DA.%26aulast%3DCheng%26aufirst%3DD.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DValente%26aufirst%3DS.%26aulast%3DNebbioso%26aufirst%3DA.%26aulast%3DPerrone%26aufirst%3DA.%26aulast%3DBrosch%26aufirst%3DG.%26aulast%3DSbardella%26aufirst%3DG.%26aulast%3DDe%2BBellis%26aufirst%3DF.%26aulast%3DMiceli%26aufirst%3DM.%26aulast%3DAltucci%26aufirst%3DL.%26atitle%3DEpigenetic%2520multiple%2520ligands%253A%2520Mixed%2520Histone%252FProtein%2520methyltransferase%252C%2520acetyltransferase%252C%2520and%2520class%2520III%2520deacetylase%2520%2528Sirtuin%2529%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D2279%26epage%3D2290%26doi%3D10.1021%2Fjm701595q" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Vadivelan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, B. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rambabu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boppana, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagarlapudi, S. A. R. P.</span><span> </span><span class="NLM_article-title">Pharmacophore modeling and virtual screening studies to design some potential histone deacetylase inhibitors as new leads</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">935</span><span class="NLM_x">–</span> <span class="NLM_lpage">946</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2007.07.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.jmgm.2007.07.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVOksL7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=935-946&author=S.+Vadivelanauthor=B.+N.+Sinhaauthor=G.+Rambabuauthor=K.+Boppanaauthor=S.+A.+R.+P.+Jagarlapudi&title=Pharmacophore+modeling+and+virtual+screening+studies+to+design+some+potential+histone+deacetylase+inhibitors+as+new+leads&doi=10.1016%2Fj.jmgm.2007.07.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore modeling and virtual screening studies to design some potential histone deacetylase inhibitors as new leads</span></div><div class="casAuthors">Vadivelan, S.; Sinha, B. N.; Rambabu, G.; Boppana, Kiran; Jagarlapudi, Sarma A. R. P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">935-946</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone deacetylase is one of the important targets in the treatment of solid tumors and hematol. cancers.  A total of 20 well-defined inhibitors were used to generate Pharmacophore models using and HypoGen module of Catalyst.  These 20 mols. broadly represent 3 different chemotypes.  The best HypoGen model consists of four-pharmacophore features, one hydrogen bond acceptor, one hydrophobic aliph. and two ring arom. centers.  This model was validated against 378 known HDAC inhibitors with a correlation of 0.897 as well as enrichment factor of 2.68 against a max. value of 3.  This model was further used to retrieve mols. from NCI database with 238,819 mols.  A total of 4638 mols. from a pool of 238,819 mols. were identified as hits while 297 mols. were indicated as highly active.  Also, a Similarity anal. has been carried out for set of 4638 hits with respect to most active mol. of each chemotypes which validated not only the Virtual Screening potential of the model but also identified the possible new Chemotypes.  This type of Similarity anal. would prove to be efficient not only for lead generation but also for lead optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6-KqHzBn9lLVg90H21EOLACvtfcHk0liXM36OhEklkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVOksL7E&md5=6aa057815c5bf0cd21f77faaf1390844</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2007.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2007.07.002%26sid%3Dliteratum%253Aachs%26aulast%3DVadivelan%26aufirst%3DS.%26aulast%3DSinha%26aufirst%3DB.%2BN.%26aulast%3DRambabu%26aufirst%3DG.%26aulast%3DBoppana%26aufirst%3DK.%26aulast%3DJagarlapudi%26aufirst%3DS.%2BA.%2BR.%2BP.%26atitle%3DPharmacophore%2520modeling%2520and%2520virtual%2520screening%2520studies%2520to%2520design%2520some%2520potential%2520histone%2520deacetylase%2520inhibitors%2520as%2520new%2520leads%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2008%26volume%3D26%26spage%3D935%26epage%3D946%26doi%3D10.1016%2Fj.jmgm.2007.07.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Campagna-Slater, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Evaluation of Virtual Screening as a Tool for Chemical Genetic Applications</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">2082</span><span class="NLM_x">–</span> <span class="NLM_lpage">2091</span><span class="refDoi"> DOI: 10.1021/ci900219u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci900219u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVartbbK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=2082-2091&author=V.+Campagna-Slaterauthor=M.+Schapira&title=Evaluation+of+Virtual+Screening+as+a+Tool+for+Chemical+Genetic+Applications&doi=10.1021%2Fci900219u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Evaluation of virtual screening as a tool for chemical genetic applications</span></div><div class="casAuthors">Campagna-Slater, Valerie; Schapira, Matthieu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2082-2091</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A collection of over 50,000 functionally annotated drugs, clin. candidates, and endogenous ligands was docked in silico against nine binding sites from seven protein targets, representing diverse function and structure, namely the sulfotransferases SULT1E1 and SULT1A3, the histone methyltransferase EHMT1, the histone acetyltransferase MYST3, and the nuclear hormone receptors ERα, PPARγ, and TRβ.  For 5 of the 9 virtual screens, compds. that docked best to the receptors clearly recapitulated known biol. functions of the genes or identified novel biol. subsequently validated in a sep. exptl. study.  In two cases, the hit list indicated some relevant but isolated biol. functions which would probably have been ignored a priori, and selected compds. were completely unrelated to gene function for the last two virtual screens.  This study demonstrates that virtual screening of pharmacol. annotated compd. libraries can be used to derive target biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWkRSwI9LGhbVg90H21EOLACvtfcHk0liXM36OhEklkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVartbbK&md5=237625f754eb93134749f00b16bbd5f2</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1021%2Fci900219u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci900219u%26sid%3Dliteratum%253Aachs%26aulast%3DCampagna-Slater%26aufirst%3DV.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DEvaluation%2520of%2520Virtual%2520Screening%2520as%2520a%2520Tool%2520for%2520Chemical%2520Genetic%2520Applications%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D2082%26epage%3D2091%26doi%3D10.1021%2Fci900219u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Tang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kozikowski, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Novel Inhibitors of Human Histone Deacetylase (HDAC) Identified by QSAR Modeling of Known Inhibitors, Virtual Screening, and Experimental Validation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">476</span><span class="refDoi"> DOI: 10.1021/ci800366f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci800366f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXht1agt7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2009&pages=461-476&author=H.+Tangauthor=X.+S.+Wangauthor=X.-P.+Huangauthor=B.+L.+Rothauthor=K.+V.+Butlerauthor=A.+P.+Kozikowskiauthor=M.+Jungauthor=A.+Tropsha&title=Novel+Inhibitors+of+Human+Histone+Deacetylase+%28HDAC%29+Identified+by+QSAR+Modeling+of+Known+Inhibitors%2C+Virtual+Screening%2C+and+Experimental+Validation&doi=10.1021%2Fci800366f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Inhibitors of Human Histone Deacetylase (HDAC) Identified by QSAR Modeling of Known Inhibitors, Virtual Screening, and Experimental Validation</span></div><div class="casAuthors">Tang, Hao; Wang, Xiang S.; Huang, Xi-Ping; Roth, Bryan L.; Butler, Kyle V.; Kozikowski, Alan P.; Jung, Mira; Tropsha, Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">461-476</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Inhibitors of histone deacetylases (HDACIs) have emerged as a new class of drugs for the treatment of human cancers and other diseases because of their effects on cell growth, differentiation, and apoptosis.  In this study we have developed several quant. structure-activity relationship (QSAR) models for 59 chem. diverse histone deacetylase class 1 (HDAC1) inhibitors.  The variable selection k nearest neighbor (kNN) and support vector machines (SVM) QSAR modeling approaches using both MolconnZ and MOE chem. descriptors generated from two-dimensional rendering of compds. as chem. graphs have been employed.  We have relied on a rigorous model development workflow including the division of the data set into training, test, and external sets and extensive internal and external validation.  Highly predictive QSAR models were generated with leave-one-out cross-validated (LOO-CV) q2 and external R2 values as high as 0.80 and 0.87, resp., using the kNN/MolconnZ approach and 0.93 and 0.87, resp., using the SVM/MolconnZ approach.  All validated QSAR models were employed concurrently for virtual screening (VS) of an inhouse compd. collection including 9.5 million mols. compiled from the ZINC7.0 database, the World Drug Index (WDI) database, the ASINEX Synergy libraries, and other com. databases.  VS resulted in 45 structurally unique consensus hits that were considered novel putative HDAC1 inhibitors.  These computational hits had several novel structural features that were not present in the original data set.  Four computational hits with novel scaffolds were tested exptl., and three of them were confirmed active against HDAC1, with IC50 values for the most active compd. of 1.00 μM.  The fourth compd. was later identified to be a selective inhibitor of HDAC6, a Class II HDAC.  Moreover, two of the confirmed hits are marketed drugs, which could potentially facilitate their further development as anticancer agents.  This study illustrates the power of the combined QSAR-VS method as a general approach for the effective identification of structurally novel bioactive compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0dwScSdXT4bVg90H21EOLACvtfcHk0liXM36OhEklkQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXht1agt7g%253D&md5=764cd294ec422c0d9922b31e5941cb71</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci800366f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci800366f%26sid%3Dliteratum%253Aachs%26aulast%3DTang%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DX.%2BS.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DKozikowski%26aufirst%3DA.%2BP.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DNovel%2520Inhibitors%2520of%2520Human%2520Histone%2520Deacetylase%2520%2528HDAC%2529%2520Identified%2520by%2520QSAR%2520Modeling%2520of%2520Known%2520Inhibitors%252C%2520Virtual%2520Screening%252C%2520and%2520Experimental%2520Validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2009%26volume%3D49%26spage%3D461%26epage%3D476%26doi%3D10.1021%2Fci800366f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Kireev, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris-Drouin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span> </span><span class="NLM_article-title">Identification of non-peptide malignant brain tumor (MBT) repeat antagonists by virtual screening of commercially available compounds</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7625</span><span class="NLM_x">–</span> <span class="NLM_lpage">31</span><span class="refDoi"> DOI: 10.1021/jm1007374</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm1007374" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=7625-31&author=D.+Kireevauthor=T.+J.+Wigleauthor=J.+Norris-Drouinauthor=J.+M.+Heroldauthor=W.+P.+Janzenauthor=S.+V.+Frye&title=Identification+of+non-peptide+malignant+brain+tumor+%28MBT%29+repeat+antagonists+by+virtual+screening+of+commercially+available+compounds&doi=10.1021%2Fjm1007374"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fjm1007374&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm1007374%26sid%3Dliteratum%253Aachs%26aulast%3DKireev%26aufirst%3DD.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DNorris-Drouin%26aufirst%3DJ.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26atitle%3DIdentification%2520of%2520non-peptide%2520malignant%2520brain%2520tumor%2520%2528MBT%2529%2520repeat%2520antagonists%2520by%2520virtual%2520screening%2520of%2520commercially%2520available%2520compounds%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D7625%26epage%3D31%26doi%3D10.1021%2Fjm1007374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Sundarapandian, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shalini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugunadevi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woo, L. K.</span><span> </span><span class="NLM_article-title">Docking-enabled pharmacophore model for histone deacetylase 8 inhibitors and its application in anti-cancer drug discovery</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">382</span><span class="NLM_x">–</span> <span class="NLM_lpage">395</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2010.07.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.jmgm.2010.07.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsb%252FN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2010&pages=382-395&author=T.+Sundarapandianauthor=J.+Shaliniauthor=S.+Sugunadeviauthor=L.+K.+Woo&title=Docking-enabled+pharmacophore+model+for+histone+deacetylase+8+inhibitors+and+its+application+in+anti-cancer+drug+discovery&doi=10.1016%2Fj.jmgm.2010.07.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">Docking-enabled pharmacophore model for histone deacetylase 8 inhibitors and its application in anti-cancer drug discovery</span></div><div class="casAuthors">Sundarapandian, Thangapandian; Shalini, John; Sugunadevi, Sakkiah; Woo, Lee Keun</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">382-395</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Zinc-dependent histone deacetylase 8 removes the epsilon-acetyl groups present in the N-terminal lysine residues of histone proteins, thereby restricting various transcription factors from being expressed.  Inhibition of this enzyme has been reported to be a novel strategy in cancer treatment.  To identify novel and diverse leads for use in potent histone deacetylase 8 inhibitor design, a pharmacophore model showing high correlation between exptl. and estd. activities was generated using the best conformations of training set compds. from mol. docking expts.  The best pharmacophore model was validated using four different strategies and then used in database screening for novel virtual leads.  Hit compds. were selected and subjected to mol. docking using GOLD.  The top-scored compd. was further optimized for improved binding.  The optimization step led to a new set of compds. with both improved binding at the active site and estd. activities.  The identified virtual leads could be used for designing potent histone deacetylase 8 inhibitors as anti-cancer therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFZFFaiXEbGLVg90H21EOLACvtfcHk0lgovWR1-ZJw9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVyhsb%252FN&md5=87df1965105819c4d0f404e0297deca5</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2010.07.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2010.07.007%26sid%3Dliteratum%253Aachs%26aulast%3DSundarapandian%26aufirst%3DT.%26aulast%3DShalini%26aufirst%3DJ.%26aulast%3DSugunadevi%26aufirst%3DS.%26aulast%3DWoo%26aufirst%3DL.%2BK.%26atitle%3DDocking-enabled%2520pharmacophore%2520model%2520for%2520histone%2520deacetylase%25208%2520inhibitors%2520and%2520its%2520application%2520in%2520anti-cancer%2520drug%2520discovery%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2010%26volume%3D29%26spage%3D382%26epage%3D395%26doi%3D10.1016%2Fj.jmgm.2010.07.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Suzuki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miyata, N.</span><span> </span><span class="NLM_article-title">Lysine Demethylases Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">8236</span><span class="NLM_x">–</span> <span class="NLM_lpage">8250</span><span class="refDoi"> DOI: 10.1021/jm201048w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm201048w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=8236-8250&author=T.+Suzukiauthor=N.+Miyata&title=Lysine+Demethylases+Inhibitors&doi=10.1021%2Fjm201048w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">Lysine Demethylases Inhibitors</span></div><div class="casAuthors">Suzuki, Takayoshi; Miyata, Naoki</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8236-8250</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review, on the use of lysine demethylases inhibitors in therapy of various diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGplp9MjSdvnn7Vg90H21EOLACvtfcHk0lgovWR1-ZJw9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1OrtrnF&md5=194d4bbc2e52672f5d5f019de0bb760c</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1021%2Fjm201048w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm201048w%26sid%3Dliteratum%253Aachs%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DMiyata%26aufirst%3DN.%26atitle%3DLysine%2520Demethylases%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D8236%26epage%3D8250%26doi%3D10.1021%2Fjm201048w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Yoo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span> </span><span class="NLM_article-title">Homology modeling, docking and structure-based pharmacophore of inhibitors of DNA methyltransferase</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">555</span><span class="NLM_x">–</span> <span class="NLM_lpage">567</span><span class="refDoi"> DOI: 10.1007/s10822-011-9441-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs10822-011-9441-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=555-567&author=J.+Yooauthor=J.+L.+Medina-Franco&title=Homology+modeling%2C+docking+and+structure-based+pharmacophore+of+inhibitors+of+DNA+methyltransferase&doi=10.1007%2Fs10822-011-9441-1"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9441-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9441-1%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DHomology%2520modeling%252C%2520docking%2520and%2520structure-based%2520pharmacophore%2520of%2520inhibitors%2520of%2520DNA%2520methyltransferase%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D555%26epage%3D567%26doi%3D10.1007%2Fs10822-011-9441-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Hewings, D. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rooney, T. P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jennings, L. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hay, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schofield, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brennan, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Conway, S. J.</span><span> </span><span class="NLM_article-title">Progress in the Development and Application of Small Molecule Inhibitors of Bromodomain–Acetyl-lysine Interactions</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">9393</span><span class="NLM_x">–</span> <span class="NLM_lpage">9413</span><span class="refDoi"> DOI: 10.1021/jm300915b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300915b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=9393-9413&author=D.+S.+Hewingsauthor=T.+P.+C.+Rooneyauthor=L.+E.+Jenningsauthor=D.+A.+Hayauthor=C.+J.+Schofieldauthor=P.+E.+Brennanauthor=S.+Knappauthor=S.+J.+Conway&title=Progress+in+the+Development+and+Application+of+Small+Molecule+Inhibitors+of+Bromodomain%E2%80%93Acetyl-lysine+Interactions&doi=10.1021%2Fjm300915b"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm300915b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300915b%26sid%3Dliteratum%253Aachs%26aulast%3DHewings%26aufirst%3DD.%2BS.%26aulast%3DRooney%26aufirst%3DT.%2BP.%2BC.%26aulast%3DJennings%26aufirst%3DL.%2BE.%26aulast%3DHay%26aufirst%3DD.%2BA.%26aulast%3DSchofield%26aufirst%3DC.%2BJ.%26aulast%3DBrennan%26aufirst%3DP.%2BE.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DConway%26aufirst%3DS.%2BJ.%26atitle%3DProgress%2520in%2520the%2520Development%2520and%2520Application%2520of%2520Small%2520Molecule%2520Inhibitors%2520of%2520Bromodomain%25E2%2580%2593Acetyl-lysine%2520Interactions%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D9393%26epage%3D9413%26doi%3D10.1021%2Fjm300915b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, Y. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-Based Virtual Screening and Biological Evaluation of Small Molecule Inhibitors for Protein Arginine Methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span><span class="refDoi"> DOI: 10.1021/jm300521m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.-W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=K.+K.+Liauthor=M.-F.+Linauthor=H.+Jiangauthor=Y.+G.+Zhengauthor=C.+Luo&title=Pharmacophore-Based+Virtual+Screening+and+Biological+Evaluation+of+Small+Molecule+Inhibitors+for+Protein+Arginine+Methylation&doi=10.1021%2Fjm300521m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.-W.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DK.%2BK.%26aulast%3DLin%26aufirst%3DM.-F.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DZheng%26aufirst%3DY.%2BG.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-Based%2520Virtual%2520Screening%2520and%2520Biological%2520Evaluation%2520of%2520Small%2520Molecule%2520Inhibitors%2520for%2520Protein%2520Arginine%2520Methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987%26doi%3D10.1021%2Fjm300521m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Yoo, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span> </span><span class="NLM_article-title">Inhibitors of DNA Methyltransferases: Insights from Computational Studies</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">3475</span><span class="NLM_x">–</span> <span class="NLM_lpage">3487</span><span class="refDoi"> DOI: 10.2174/092986712801323289</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2174%2F092986712801323289" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22709005" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtF2itb3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2012&pages=3475-3487&author=J.+Yooauthor=J.+L.+Medina-Franco&title=Inhibitors+of+DNA+Methyltransferases%3A+Insights+from+Computational+Studies&doi=10.2174%2F092986712801323289"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of DNA methyltransferases: insights from computational studies</span></div><div class="casAuthors">Yoo, J.; Medina-Franco, J. L.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">3475-3487</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  DNA methyltransferases (DNMTs) are a family of epigenetic enzymes for which inhibition is an attractive strategy for the treatment of cancer and other diseases.  In synergy with exptl. approaches, computational methods are increasingly being used to identify and optimize the activity of inhibitors of DNMTs as well as to rationalize at the mol. level of the mechanism of established inhibitors.  Recently, a crystallog. structure of the methyltransferase domain of human DNMT1 bound to unmethylated DNA was published encouraging the application of structure-based approaches to design and optimize the activity of currently known inhibitors.  Herein, we review the progress in the discovery and optimization of inhibitors of DNMTs using computational approaches including homol. modeling, docking, pharmacophore modeling, mol. dynamics, and virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7J66x7abyxLVg90H21EOLACvtfcHk0liNVRZqpmc7fw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtF2itb3P&md5=ee28a21948b821fe08afbba97e56622e</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.2174%2F092986712801323289&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986712801323289%26sid%3Dliteratum%253Aachs%26aulast%3DYoo%26aufirst%3DJ.%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26atitle%3DInhibitors%2520of%2520DNA%2520Methyltransferases%253A%2520Insights%2520from%2520Computational%2520Studies%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2012%26volume%3D19%26spage%3D3475%26epage%3D3487%26doi%3D10.2174%2F092986712801323289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span> </span><span class="NLM_article-title">Discovery of a novel histone deacetylase 8 inhibitor by virtual screening</span> <span class="citation_source-journal">Med. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">152</span><span class="NLM_x">–</span> <span class="NLM_lpage">156</span><span class="refDoi"> DOI: 10.1007/s00044-010-9519-7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs00044-010-9519-7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsV2rs7vJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2012&pages=152-156&author=L.+Zhangauthor=M.+Liauthor=J.+Fengauthor=H.+Fangauthor=W.+Xu&title=Discovery+of+a+novel+histone+deacetylase+8+inhibitor+by+virtual+screening&doi=10.1007%2Fs00044-010-9519-7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel histone deacetylase 8 inhibitor by virtual screening</span></div><div class="casAuthors">Zhang, Lei; Li, Min-Yong; Feng, Jin-Hong; Fang, Hao; Xu, Wen-Fang</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">152-156</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Boston</span>)
        </div><div class="casAbstract">In order to develop potent histone deacetylase inhibitors, a virtual screening approach was performed to discover novel lead structures.  A com. database contg. about 167,000 mols. was in silico filtered by rule of five, zinc-binding groups, pharmacophore models, and binding pattern anal.  At last, three mols. were selected for enzyme inhibition assay, and one compd. 02 has IC50 of 1.6 μM against histone deacetylase 8 (HDAC8).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquQFHm0bn6DLVg90H21EOLACvtfcHk0lgfVwTPTZg_hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsV2rs7vJ&md5=457c889bd592a0913723386db3e796b4</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1007%2Fs00044-010-9519-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-010-9519-7%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DFang%26aufirst%3DH.%26aulast%3DXu%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520a%2520novel%2520histone%2520deacetylase%25208%2520inhibitor%2520by%2520virtual%2520screening%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2012%26volume%3D21%26spage%3D152%26epage%3D156%26doi%3D10.1007%2Fs00044-010-9519-7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiong, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allali-Hassani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of an in Vivo Chemical Probe of the Lysine Methyltransferases G9a and GLP</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8931</span><span class="NLM_x">–</span> <span class="NLM_lpage">8942</span><span class="refDoi"> DOI: 10.1021/jm401480r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401480r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8931-8942&author=F.+Liuauthor=D.+Barsyte-Lovejoyauthor=F.+Liauthor=Y.+Xiongauthor=V.+Korboukhauthor=X.-P.+Huangauthor=A.+Allali-Hassaniauthor=W.+P.+Janzenauthor=B.+L.+Rothauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=P.+J.+Brownauthor=M.+Vedadiauthor=J.+Jin&title=Discovery+of+an+in+Vivo+Chemical+Probe+of+the+Lysine+Methyltransferases+G9a+and+GLP&doi=10.1021%2Fjm401480r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of an in vivo chemical probe of the lysine methyltransferases G9a and GLP</span></div><div class="casAuthors">Liu, Feng; Barsyte-Lovejoy, Dalia; Li, Fengling; Xiong, Yan; Korboukh, Victoria; Huang, Xi-Ping; Allali-Hassani, Abdellah; Janzen, William P.; Roth, Bryan L.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Brown, Peter J.; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">8931-8942</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Among epigenetic writers, readers, and erasers, the lysine methyltransferases G9a and GLP, which catalyze mono- and dimethylation of histone H3 lysine 9 (H3K9me2) and nonhistone proteins, have been implicated in a variety of human diseases.  A toolkit of well-characterized chem. probes will allow biol. and disease hypotheses concerning these proteins to be tested in cell-based and animal models with high confidence.  We previously discovered potent and selective G9a/GLP inhibitors including the cellular chem. probe UNC0638, which displays an excellent sepn. of functional potency and cell toxicity.  However, this inhibitor is not suitable for animal studies due to its poor pharmacokinetic (PK) properties.  Here, we report the discovery of the first G9a and GLP in vivo chem. probe UNC0642, which not only maintains high in vitro and cellular potency, low cell toxicity, and excellent selectivity, but also displays improved in vivo PK properties, making it suitable for animal studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowOI3TSEz3ZbVg90H21EOLACvtfcHk0lgfVwTPTZg_hg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsF2htrvK&md5=933f0c49a938b4dd8146f6d09aa21e21</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1021%2Fjm401480r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401480r%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DXiong%26aufirst%3DY.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520an%2520in%2520Vivo%2520Chemical%2520Probe%2520of%2520the%2520Lysine%2520Methyltransferases%2520G9a%2520and%2520GLP%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8931%26epage%3D8942%26doi%3D10.1021%2Fjm401480r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dobrovetsky, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dong, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smil, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">2110</span><span class="NLM_x">–</span> <span class="NLM_lpage">2124</span><span class="refDoi"> DOI: 10.1021/jm3018332</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3018332" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=2110-2124&author=F.+Liuauthor=F.+Liauthor=A.+Maauthor=E.+Dobrovetskyauthor=A.+Dongauthor=C.+Gaoauthor=I.+Korboukhauthor=J.+Liuauthor=D.+Smilauthor=P.+J.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=M.+Schapiraauthor=M.+Vedadiauthor=J.+Jin&title=Exploiting+an+Allosteric+Binding+Site+of+PRMT3+Yields+Potent+and+Selective+Inhibitors&doi=10.1021%2Fjm3018332"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit67R"><div class="casContent"><span class="casTitleNuber">67</span><div class="casTitle"><span class="NLM_cas:atitle">Exploiting an Allosteric Binding Site of PRMT3 Yields Potent and Selective Inhibitors</span></div><div class="casAuthors">Liu, Feng; Li, Fengling; Ma, Anqi; Dobrovetsky, Elena; Dong, Aiping; Gao, Cen; Korboukh, Ilia; Liu, Jing; Smil, David; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Schapira, Matthieu; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">2110-2124</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play an important role in diverse biol. processes.  Among the nine known human PRMTs, PRMT3 has been implicated in ribosomal biosynthesis via asym. dimethylation of the 40S ribosomal protein S2 and in cancer via interaction with the DAL-1 tumor suppressor protein.  However, few selective inhibitors of PRMTs have been discovered.  We recently disclosed the first selective PRMT3 inhibitor, which occupies a novel allosteric binding site and is noncompetitive with both the peptide substrate and cofactor.  Here we report comprehensive structure-activity relationship studies of this series, which resulted in the discovery of multiple PRMT3 inhibitors with submicromolar potencies.  An X-ray crystal structure of compd. I in complex with PRMT3 confirmed that this inhibitor occupied the same allosteric binding site as our initial lead compd.  These studies provide the first exptl. evidence that potent and selective inhibitors can be created by exploiting the allosteric binding site of PRMT3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqgc4z8jYHwWbVg90H21EOLACvtfcHk0lhVuoRY8i2t8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtlWgt74%253D&md5=d538b236cbe1326ed74d84ab12778e47</span></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1021%2Fjm3018332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3018332%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DDobrovetsky%26aufirst%3DE.%26aulast%3DDong%26aufirst%3DA.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DSmil%26aufirst%3DD.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DSchapira%26aufirst%3DM.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DExploiting%2520an%2520Allosteric%2520Binding%2520Site%2520of%2520PRMT3%2520Yields%2520Potent%2520and%2520Selective%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D2110%26epage%3D2124%26doi%3D10.1021%2Fjm3018332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, J.</span><span> </span><span class="NLM_article-title">Molecular Modeling and Virtual Screening of DNA Methyltransferase Inhibitors</span> <span class="citation_source-journal">Curr. Pharm. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2138</span><span class="NLM_x">–</span> <span class="NLM_lpage">2147</span><span class="refDoi"> DOI: 10.2174/1381612811319120002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.2174%2F1381612811319120002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=23016844" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2013&pages=2138-2147&author=J.+L.+Medina-Francoauthor=J.+Yoo&title=Molecular+Modeling+and+Virtual+Screening+of+DNA+Methyltransferase+Inhibitors&doi=10.2174%2F1381612811319120002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular modeling and virtual screening of DNA methyltransferase inhibitors</span></div><div class="casAuthors">Medina-Franco, Jose L.; Yoo, Jakyung</div><div class="citationInfo"><span class="NLM_cas:title">Current Pharmaceutical Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2138-2147</span>CODEN:
                <span class="NLM_cas:coden">CPDEFP</span>;
        ISSN:<span class="NLM_cas:issn">1381-6128</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Inhibition of DNA methyltransferases (DNMTs) is a promising approach for the therapeutic treatment of cancer and other diseases.  In this work, we review the recent progress on the mol. modeling and virtual screening toward the identification of key structural features assocd. with the enzyme inhibitory action of active compds. and to identify DNMT inhibitors with novel mol. scaffolds.  We discuss the mol. modeling with the co-factor binding site using a recent crystallog. structure of the methyltransferase domain of human DNMT1.  We also review the emerging synergy of mol. modeling and chemoinformatic approaches applied to epigenetic therapies targeting DNMTs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraut6rC0hKHbVg90H21EOLACvtfcHk0lhVuoRY8i2t8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlvVKqtLw%253D&md5=7854d20bf9e11a343dd541c8225c7dde</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.2174%2F1381612811319120002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1381612811319120002%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DYoo%26aufirst%3DJ.%26atitle%3DMolecular%2520Modeling%2520and%2520Virtual%2520Screening%2520of%2520DNA%2520Methyltransferase%2520Inhibitors%26jtitle%3DCurr.%2520Pharm.%2520Des.%26date%3D2013%26volume%3D19%26spage%3D2138%26epage%3D2147%26doi%3D10.2174%2F1381612811319120002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Schmitt, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carlino, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pippel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulz-Fincke, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Metzger, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willmann, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yiu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barton, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schule, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span> </span><span class="NLM_article-title">Nonpeptidic propargylamines as inhibitors of lysine specific demethylase 1 (LSD1) with cellular activity</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">7334</span><span class="NLM_x">–</span> <span class="NLM_lpage">42</span><span class="refDoi"> DOI: 10.1021/jm400792m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400792m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=7334-42&author=M.+L.+Schmittauthor=A.+T.+Hauserauthor=L.+Carlinoauthor=M.+Pippelauthor=J.+Schulz-Finckeauthor=E.+Metzgerauthor=D.+Willmannauthor=T.+Yiuauthor=M.+Bartonauthor=R.+Schuleauthor=W.+Sipplauthor=M.+Jung&title=Nonpeptidic+propargylamines+as+inhibitors+of+lysine+specific+demethylase+1+%28LSD1%29+with+cellular+activity&doi=10.1021%2Fjm400792m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit69R"><div class="casContent"><span class="casTitleNuber">69</span><div class="casTitle"><span class="NLM_cas:atitle">Nonpeptidic Propargylamines as Inhibitors of Lysine Specific Demethylase 1 (LSD1) with Cellular Activity</span></div><div class="casAuthors">Schmitt, Martin L.; Hauser, Alexander-Thomas; Carlino, Luca; Pippel, Martin; Schulz-Fincke, Johannes; Metzger, Eric; Willmann, Dominica; Yiu, Teresa; Barton, Michelle; Schuele, Roland; Sippl, Wolfgang; Jung, Manfred</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">7334-7342</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Lysine demethylases play an important role in epigenetic regulation and thus in the development of diseases like cancer or neurodegenerative disorders.  As the lysine specific demethylase 1 (LSD1/KDM1) has been strongly connected to androgen and estrogen dependent gene expression, it serves as a promising target for the therapy of hormone dependent cancer.  Here, we report on the discovery of new small mol. inhibitors of LSD1 contg. a propargylamine warhead, starting out from lysine contg. substrate analogs.  On the basis of these substrate mimicking inhibitors, we were able to increase potency by a combination of similarity-based virtual screening and subsequent synthetic optimization resulting in more druglike LSD1 inhibitors that led to histone hypermethylation in breast cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRGRr4rUPfsLVg90H21EOLACvtfcHk0lhVuoRY8i2t8Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlSjurnO&md5=d4de914dec37f011a7b9488ec539a395</span></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1021%2Fjm400792m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400792m%26sid%3Dliteratum%253Aachs%26aulast%3DSchmitt%26aufirst%3DM.%2BL.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DCarlino%26aufirst%3DL.%26aulast%3DPippel%26aufirst%3DM.%26aulast%3DSchulz-Fincke%26aufirst%3DJ.%26aulast%3DMetzger%26aufirst%3DE.%26aulast%3DWillmann%26aufirst%3DD.%26aulast%3DYiu%26aufirst%3DT.%26aulast%3DBarton%26aufirst%3DM.%26aulast%3DSchule%26aufirst%3DR.%26aulast%3DSippl%26aufirst%3DW.%26aulast%3DJung%26aufirst%3DM.%26atitle%3DNonpeptidic%2520propargylamines%2520as%2520inhibitors%2520of%2520lysine%2520specific%2520demethylase%25201%2520%2528LSD1%2529%2520with%2520cellular%2520activity%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D7334%26epage%3D42%26doi%3D10.1021%2Fjm400792m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Sorna, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Theisen, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stephens, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warner, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vankayalapati, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span> </span><span class="NLM_article-title">High-Throughput Virtual Screening Identifies Novel N ′-(1-Phenylethylidene)-benzohydrazides as Potent, Specific, and Reversible LSD1 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">9496</span><span class="NLM_x">–</span> <span class="NLM_lpage">9508</span><span class="refDoi"> DOI: 10.1021/jm400870h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400870h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9496-9508&author=V.+Sornaauthor=E.+R.+Theisenauthor=B.+Stephensauthor=S.+L.+Warnerauthor=D.+J.+Bearssauthor=H.+Vankayalapatiauthor=S.+Sharma&title=High-Throughput+Virtual+Screening+Identifies+Novel+N+%E2%80%B2-%281-Phenylethylidene%29-benzohydrazides+as+Potent%2C+Specific%2C+and+Reversible+LSD1+Inhibitors&doi=10.1021%2Fjm400870h"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1021%2Fjm400870h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400870h%26sid%3Dliteratum%253Aachs%26aulast%3DSorna%26aufirst%3DV.%26aulast%3DTheisen%26aufirst%3DE.%2BR.%26aulast%3DStephens%26aufirst%3DB.%26aulast%3DWarner%26aufirst%3DS.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DSharma%26aufirst%3DS.%26atitle%3DHigh-Throughput%2520Virtual%2520Screening%2520Identifies%2520Novel%2520N%2520%25E2%2580%25B2-%25281-Phenylethylidene%2529-benzohydrazides%2520as%2520Potent%252C%2520Specific%252C%2520and%2520Reversible%2520LSD1%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9496%26epage%3D9508%26doi%3D10.1021%2Fjm400870h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Vidler, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filippakopoulos, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fedorov, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Picaud, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomsett, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodward, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hoelder, S.</span><span> </span><span class="NLM_article-title">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">8073</span><span class="NLM_x">–</span> <span class="NLM_lpage">8088</span><span class="refDoi"> DOI: 10.1021/jm4011302</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4011302" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=8073-8088&author=L.+R.+Vidlerauthor=P.+Filippakopoulosauthor=O.+Fedorovauthor=S.+Picaudauthor=S.+Martinauthor=M.+Tomsettauthor=H.+Woodwardauthor=N.+Brownauthor=S.+Knappauthor=S.+Hoelder&title=Discovery+of+Novel+Small-Molecule+Inhibitors+of+BRD4+Using+Structure-Based+Virtual+Screening&doi=10.1021%2Fjm4011302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Small-Molecule Inhibitors of BRD4 Using Structure-Based Virtual Screening</span></div><div class="casAuthors">Vidler, Lewis R.; Filippakopoulos, Panagis; Fedorov, Oleg; Picaud, Sarah; Martin, Sarah; Tomsett, Michael; Woodward, Hannah; Brown, Nathan; Knapp, Stefan; Hoelder, Swen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8073-8088</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bromodomains (BRDs) are epigenetic readers that recognize acetylated-lysine (KAc) on proteins and are implicated in a no. of diseases.  We describe a virtual screening approach to identify BRD inhibitors.  Key elements of this approach are the extensive design and use of substructure queries to compile a set of com. available compds. featuring novel putative KAc mimetics and docking this set for final compd. selection.  We describe the validation of this approach by applying it to the first BRD of BRD4.  The selection and testing of 143 compds. lead to the discovery of six novel hits, including four unprecedented KAc mimetics.  We solved the crystal structure of four hits, detd. their binding mode, and improved their potency through synthesis and the purchase of derivs.  This work provides a validated virtual screening approach that is applicable to other BRDs and describes novel KAc mimetics that can be further explored to design more potent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGow5yj8bnohUbVg90H21EOLACvtfcHk0lgBhZsTKKcuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOjurrI&md5=c1f76e4838713335be7a93ebe4c68701</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fjm4011302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4011302%26sid%3Dliteratum%253Aachs%26aulast%3DVidler%26aufirst%3DL.%2BR.%26aulast%3DFilippakopoulos%26aufirst%3DP.%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DPicaud%26aufirst%3DS.%26aulast%3DMartin%26aufirst%3DS.%26aulast%3DTomsett%26aufirst%3DM.%26aulast%3DWoodward%26aufirst%3DH.%26aulast%3DBrown%26aufirst%3DN.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DHoelder%26aufirst%3DS.%26atitle%3DDiscovery%2520of%2520Novel%2520Small-Molecule%2520Inhibitors%2520of%2520BRD4%2520Using%2520Structure-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D8073%26epage%3D8088%26doi%3D10.1021%2Fjm4011302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Blum, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ibanez, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shum, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Djaballah, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rice, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, M.</span><span> </span><span class="NLM_article-title">Small-molecule inhibitors of SETD8 with cellular activity</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">2471</span><span class="NLM_x">–</span> <span class="NLM_lpage">8</span><span class="refDoi"> DOI: 10.1021/cb500515r</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb500515r" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=2471-8&author=G.+Blumauthor=G.+Ibanezauthor=X.+Raoauthor=D.+Shumauthor=C.+Raduauthor=H.+Djaballahauthor=J.+C.+Riceauthor=M.+Luo&title=Small-molecule+inhibitors+of+SETD8+with+cellular+activity&doi=10.1021%2Fcb500515r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Inhibitors of SETD8 with Cellular Activity</span></div><div class="casAuthors">Blum, Gil; Ibanez, Glorymar; Rao, Xiangjun; Shum, David; Radu, Constantin; Djaballah, Hakim; Rice, Judd C.; Luo, Minkui</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2471-2478</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SETD8/SET8/Pr-SET7/KMT5A is the sole protein lysine methyltransferase (PKMT) known to monomethylate lysine 20 of histone H4 in vivo.  SETD8's methyltransferase activity has been implicated in many essential cellular processes including DNA replication, DNA damage response, transcription modulation, and cell cycle regulation.  Developing SETD8 inhibitors with cellular activity is a key step toward elucidating the diverse roles of SETD8 via convenient pharmacol. perturbation.  From the hits of a prior high throughput screen (HTS), SPS8I1-3 (NSC663284, BVT948, and ryuvidine) were validated as potent SETD8 inhibitors.  These compds. contain different structural motifs and inhibit SETD8 via distinct modes.  More importantly, these compds. show cellular activity by suppressing the H4K20me1 mark of SETD8 and recapitulate characteristic S/G2/M-phase cell cycle defects as obsd. for RNAi-mediated SETD8 knockdown.  The commonality of SPS8I1-3 against SETD8, together with their distinct structures and mechanisms for SETD8 inhibition, argues for the collective application of these compds. as SETD8 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGquxqpOQLgFObVg90H21EOLACvtfcHk0lgBhZsTKKcuFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlOnurbE&md5=87b7c415bf82577843fd5eeb09c2c746</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1021%2Fcb500515r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb500515r%26sid%3Dliteratum%253Aachs%26aulast%3DBlum%26aufirst%3DG.%26aulast%3DIbanez%26aufirst%3DG.%26aulast%3DRao%26aufirst%3DX.%26aulast%3DShum%26aufirst%3DD.%26aulast%3DRadu%26aufirst%3DC.%26aulast%3DDjaballah%26aufirst%3DH.%26aulast%3DRice%26aufirst%3DJ.%2BC.%26aulast%3DLuo%26aufirst%3DM.%26atitle%3DSmall-molecule%2520inhibitors%2520of%2520SETD8%2520with%2520cellular%2520activity%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2014%26volume%3D9%26spage%3D2471%26epage%3D8%26doi%3D10.1021%2Fcb500515r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Kannan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Melesina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hauser, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakrabarti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heimburg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidtkunz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walter, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marek, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pierce, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romier, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jung, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sippl, W.</span><span> </span><span class="NLM_article-title">Discovery of inhibitors of Schistosoma mansoni HDAC8 by combining homology modeling, virtual screening, and in vitro validation</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">3005</span><span class="NLM_x">–</span> <span class="NLM_lpage">19</span><span class="refDoi"> DOI: 10.1021/ci5004653</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci5004653" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2014&pages=3005-19&author=S.+Kannanauthor=J.+Melesinaauthor=A.+T.+Hauserauthor=A.+Chakrabartiauthor=T.+Heimburgauthor=K.+Schmidtkunzauthor=A.+Walterauthor=M.+Marekauthor=R.+J.+Pierceauthor=C.+Romierauthor=M.+Jungauthor=W.+Sippl&title=Discovery+of+inhibitors+of+Schistosoma+mansoni+HDAC8+by+combining+homology+modeling%2C+virtual+screening%2C+and+in+vitro+validation&doi=10.1021%2Fci5004653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Inhibitors of Schistosoma mansoni HDAC8 by Combining Homology Modeling, Virtual Screening, and in Vitro Validation</span></div><div class="casAuthors">Kannan, Srinivasaraghavan; Melesina, Jelena; Hauser, Alexander-Thomas; Chakrabarti, Alokta; Heimburg, Tino; Schmidtkunz, Karin; Walter, Alexandra; Marek, Martin; Pierce, Raymond J.; Romier, Christophe; Jung, Manfred; Sippl, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">54</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">3005-3019</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Schistosomiasis, caused by S. mansoni, is a tropical disease that affects over 200 million people worldwide.  A novel approach for targeting eukaryotic parasites is to tackle their dynamic epigenetic machinery that is necessary for the extensive phenotypic changes during their life cycle.  We recently identified S. mansoni histone deacetylase 8 (smHDAC8) as a potential target for antiparasitic therapy.  Here we present results from a virtual screening campaign on smHDAC8.  Besides hydroxamates, several sulfonamide-thiazole derivs. were identified by a target-based virtual screening using a homol. model of smHDAC8.  In vitro testing of 75 compds. identified 8 hydroxamates as potent and lead-like inhibitors of the parasitic HDAC8.  Solving of the crystal structure of smHDAC8 with two of the virtual screening hits confirmed the predicted binding mode.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoH_psd1ffmerVg90H21EOLACvtfcHk0liwb3mQIAzJKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsFygtLvO&md5=512119ba4e5cbcdd0d28570a3acb74b5</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fci5004653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci5004653%26sid%3Dliteratum%253Aachs%26aulast%3DKannan%26aufirst%3DS.%26aulast%3DMelesina%26aufirst%3DJ.%26aulast%3DHauser%26aufirst%3DA.%2BT.%26aulast%3DChakrabarti%26aufirst%3DA.%26aulast%3DHeimburg%26aufirst%3DT.%26aulast%3DSchmidtkunz%26aufirst%3DK.%26aulast%3DWalter%26aufirst%3DA.%26aulast%3DMarek%26aufirst%3DM.%26aulast%3DPierce%26aufirst%3DR.%2BJ.%26aulast%3DRomier%26aufirst%3DC.%26aulast%3DJung%26aufirst%3DM.%26aulast%3DSippl%26aufirst%3DW.%26atitle%3DDiscovery%2520of%2520inhibitors%2520of%2520Schistosoma%2520mansoni%2520HDAC8%2520by%2520combining%2520homology%2520modeling%252C%2520virtual%2520screening%252C%2520and%2520in%2520vitro%2520validation%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2014%26volume%3D54%26spage%3D3005%26epage%3D19%26doi%3D10.1021%2Fci5004653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bleich, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herold, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tripathy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janzen, W. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6822</span><span class="NLM_x">–</span> <span class="NLM_lpage">6833</span><span class="refDoi"> DOI: 10.1021/jm500871s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500871s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6822-6833&author=A.+Maauthor=W.+Yuauthor=F.+Liauthor=R.+M.+Bleichauthor=J.+M.+Heroldauthor=K.+V.+Butlerauthor=J.+L.+Norrisauthor=V.+Korboukhauthor=A.+Tripathyauthor=W.+P.+Janzenauthor=C.+H.+Arrowsmithauthor=S.+V.+Fryeauthor=M.+Vedadiauthor=P.+J.+Brownauthor=J.+Jin&title=Discovery+of+a+Selective%2C+Substrate-Competitive+Inhibitor+of+the+Lysine+Methyltransferase+SETD8&doi=10.1021%2Fjm500871s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective, Substrate-Competitive Inhibitor of the Lysine Methyltransferase SETD8</span></div><div class="casAuthors">Ma, Anqi; Yu, Wenyu; Li, Fengling; Bleich, Rachel M.; Herold, J. Martin; Butler, Kyle V.; Norris, Jacqueline L.; Korboukh, Victoria; Tripathy, Ashutosh; Janzen, William P.; Arrowsmith, Cheryl H.; Frye, Stephen V.; Vedadi, Masoud; Brown, Peter J.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">6822-6833</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The lysine methyltransferase SETD8 is the only known methyltransferase that catalyzes monomethylation of histone H4 lysine 20 (H4K20).  Monomethylation of H4K20 has been implicated in regulating diverse biol. processes including the DNA damage response.  In addn. to H4K20, SETD8 monomethylates nonhistone substrates including proliferating cell nuclear antigen (PCNA) and promotes carcinogenesis by deregulating PCNA expression.  However, selective inhibitors of SETD8 are scarce.  The only known selective inhibitor of SETD8 to date is nahuoic acid A, a marine natural product, which is competitive with the cofactor.  Here, the authors report the discovery of the first substrate-competitive inhibitor of SETD8, UNC0379 I.  This small-mol. inhibitor is active in multiple biochem. assays.  Its affinity to SETD8 was confirmed by ITC (isothermal titrn. calorimetry) and SPR (surface plasmon resonance) studies.  Importantly, compd. I is selective for SETD8 over 15 other methyltransferases.  The authors also describe structure-activity relationships (SAR) of this series.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrR9y31FWBfY7Vg90H21EOLACvtfcHk0liwb3mQIAzJKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtFyitbjI&md5=adca277f02033f96fe45a28087dca14e</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fjm500871s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500871s%26sid%3Dliteratum%253Aachs%26aulast%3DMa%26aufirst%3DA.%26aulast%3DYu%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBleich%26aufirst%3DR.%2BM.%26aulast%3DHerold%26aufirst%3DJ.%2BM.%26aulast%3DButler%26aufirst%3DK.%2BV.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DKorboukh%26aufirst%3DV.%26aulast%3DTripathy%26aufirst%3DA.%26aulast%3DJanzen%26aufirst%3DW.%2BP.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DBrown%26aufirst%3DP.%2BJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520Selective%252C%2520Substrate-Competitive%2520Inhibitor%2520of%2520the%2520Lysine%2520Methyltransferase%2520SETD8%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6822%26epage%3D6833%26doi%3D10.1021%2Fjm500871s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Parenti, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grozio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galeno, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Damonte, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Millo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sociali, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Franceschi, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballestrero, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bruzzone, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rio, A. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nencioni, A.</span><span> </span><span class="NLM_article-title">Discovery of Novel and Selective SIRT6 Inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">4796</span><span class="NLM_x">–</span> <span class="NLM_lpage">4804</span><span class="refDoi"> DOI: 10.1021/jm500487d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500487d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXnt1Cnsr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=4796-4804&author=M.+D.+Parentiauthor=A.+Grozioauthor=I.+Bauerauthor=L.+Galenoauthor=P.+Damonteauthor=E.+Milloauthor=G.+Socialiauthor=C.+Franceschiauthor=A.+Ballestreroauthor=S.+Bruzzoneauthor=A.+D.+Rioauthor=A.+Nencioni&title=Discovery+of+Novel+and+Selective+SIRT6+Inhibitors&doi=10.1021%2Fjm500487d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel and Selective SIRT6 Inhibitors</span></div><div class="casAuthors">Parenti, Marco Daniele; Grozio, Alessia; Bauer, Inga; Galeno, Lauretta; Damonte, Patrizia; Millo, Enrico; Sociali, Giovanna; Franceschi, Claudio; Ballestrero, Alberto; Bruzzone, Santina; Rio, Alberto Del; Nencioni, Alessio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4796-4804</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">SIRT6 is an NAD+-dependent deacetylase with a role in the transcriptional control of metab. and aging but also in genome stability and inflammation.  Broad therapeutic applications are foreseen for SIRT6 inhibitors, including uses in diabetes, immune-mediated disorders, and cancer.  Here we report on the identification of the first selective SIRT6 inhibitors by in silico screening.  The most promising leads show micromolar IC50s, have significant selectivity for SIRT6 vs. SIRT1 and SIRT2, and are active in cells, as shown by increased acetylation at SIRT6 target lysines on histone 3, reduced TNF-α secretion, GLUT-1 upregulation, and increased glucose uptake.  Taken together, these results show the value of these compds. as starting leads for the development of new SIRT6-targeting therapeutic agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqvxmWnUj7yPrVg90H21EOLACvtfcHk0liwb3mQIAzJKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXnt1Cnsr4%253D&md5=f06f63bdfca06247ed5880deb50c5b13</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fjm500487d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500487d%26sid%3Dliteratum%253Aachs%26aulast%3DParenti%26aufirst%3DM.%2BD.%26aulast%3DGrozio%26aufirst%3DA.%26aulast%3DBauer%26aufirst%3DI.%26aulast%3DGaleno%26aufirst%3DL.%26aulast%3DDamonte%26aufirst%3DP.%26aulast%3DMillo%26aufirst%3DE.%26aulast%3DSociali%26aufirst%3DG.%26aulast%3DFranceschi%26aufirst%3DC.%26aulast%3DBallestrero%26aufirst%3DA.%26aulast%3DBruzzone%26aufirst%3DS.%26aulast%3DRio%26aufirst%3DA.%2BD.%26aulast%3DNencioni%26aufirst%3DA.%26atitle%3DDiscovery%2520of%2520Novel%2520and%2520Selective%2520SIRT6%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D4796%26epage%3D4804%26doi%3D10.1021%2Fjm500487d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Smadbeck, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peterson, M. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zee, B. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garapaty, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mago, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giannis, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Floudas, C. A.</span><span> </span><span class="NLM_article-title">De Novo Peptide Design and Experimental Validation of Histone Methyltransferase Inhibitors</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e90095</span><span class="refDoi"> DOI: 10.1371/journal.pone.0090095</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1371%2Fjournal.pone.0090095" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24587223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVKjtL%252FM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2014&pages=e90095&author=J.+Smadbeckauthor=M.+B.+Petersonauthor=B.+M.+Zeeauthor=S.+Garapatyauthor=A.+Magoauthor=C.+Leeauthor=A.+Giannisauthor=P.+Trojerauthor=B.+A.+Garciaauthor=C.+A.+Floudas&title=De+Novo+Peptide+Design+and+Experimental+Validation+of+Histone+Methyltransferase+Inhibitors&doi=10.1371%2Fjournal.pone.0090095"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">De novo peptide design and experimental validation of histone methyltransferase inhibitors</span></div><div class="casAuthors">Smadbeck, James; Peterson, Meghan B.; Zee, Barry M.; Garapaty, Shivani; Mago, Aashna; Lee, Christina; Giannis, Athanassios; Trojer, Patrick; Garcia, Benjamin A.; Floudas, Christodoulos A.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">e90095/1-e90095/14, 14 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Histones are small proteins crit. to the efficient packaging of DNA in the nucleus.  DNA-protein complexes, known as nucleosomes, are formed when the DNA winds itself around the surface of the histones.  The methylation of histone residues by enhancer of zeste homolog 2 (EZH2) maintains gene repression over successive cell generations.  Overexpression of EZH2 can silence important tumor suppressor genes leading to increased invasiveness of many types of cancers.  This makes the inhibition of EZH2 an important target in the development of cancer therapeutics.  We employed a three-stage computational de novo peptide design method to design inhibitory peptides of EZH2.  The method consists of a sequence selection stage and two validation stages for fold specificity and approx. binding affinity.  The sequence selection stage consists of an integer linear optimization model that was solved to produce a rank-ordered list of amino acid sequences with increased stability in the bound peptide-EZH2 structure.  These sequences were validated through the calcn. of the fold specificity and approx. binding affinity of the designed peptides.  Here we report the discovery of novel EZH2 inhibitory peptides using the de novo peptide design method.  The computationally discovered peptides were exptl. validated in vitro using dose titrns. and mechanism of action enzymic assays.  The peptide with the highest in vitro response, SQ037, was validated in nucleo using quant. mass spectrometry-based proteomics.  This peptide had an IC50 of 13.5 μM, demonstrated greater potency as an inhibitor when compared to the native and K27A mutant control peptides, and demonstrated competitive inhibition vs. the peptide substrate.  Addnl., this peptide demonstrated high specificity to the EZH2 target in comparison to other histone methyltransferases.  The validated peptides are the first computationally designed peptides that directly inhibit EZH2.  These inhibitors should prove useful for further chromatin biol. investigations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-1ULxuWX0bbVg90H21EOLACvtfcHk0lj9IDtVivSFEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVKjtL%252FM&md5=af0e5f85357302b0550d6352d449e2a2</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0090095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0090095%26sid%3Dliteratum%253Aachs%26aulast%3DSmadbeck%26aufirst%3DJ.%26aulast%3DPeterson%26aufirst%3DM.%2BB.%26aulast%3DZee%26aufirst%3DB.%2BM.%26aulast%3DGarapaty%26aufirst%3DS.%26aulast%3DMago%26aufirst%3DA.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DGiannis%26aufirst%3DA.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DGarcia%26aufirst%3DB.%2BA.%26aulast%3DFloudas%26aufirst%3DC.%2BA.%26atitle%3DDe%2520Novo%2520Peptide%2520Design%2520and%2520Experimental%2520Validation%2520of%2520Histone%2520Methyltransferase%2520Inhibitors%26jtitle%3DPLoS%2520One%26date%3D2014%26volume%3D9%26spage%3De90095%26doi%3D10.1371%2Fjournal.pone.0090095" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Hou, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fang, H.</span><span> </span><span class="NLM_article-title">Enhancing the Sensitivity of Pharmacophore-Based Virtual Screening by Incorporating Customized ZBG Features: A Case Study Using Histone Deacetylase 8</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">71</span><span class="refDoi"> DOI: 10.1021/ci500762z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci500762z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktVegu7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=861-71&author=X.+Houauthor=J.+Duauthor=R.+Liuauthor=Y.+Zhouauthor=M.+Liauthor=W.+Xuauthor=H.+Fang&title=Enhancing+the+Sensitivity+of+Pharmacophore-Based+Virtual+Screening+by+Incorporating+Customized+ZBG+Features%3A+A+Case+Study+Using+Histone+Deacetylase+8&doi=10.1021%2Fci500762z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Enhancing the Sensitivity of Pharmacophore-Based Virtual Screening by Incorporating Customized ZBG Features: A Case Study Using Histone Deacetylase 8</span></div><div class="casAuthors">Hou, Xuben; Du, Jintong; Liu, Renshuai; Zhou, Yi; Li, Minyong; Xu, Wenfang; Fang, Hao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">861-871</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">As key regulators of epigenetic regulation, human histone deacetylases (HDACs) have been identified as drug targets for the treatment of several cancers.  The proper recognition of zinc-binding groups (ZBGs) will help improve the accuracy of virtual screening for novel HDAC inhibitors.  Here, we developed a high-specificity ZBG-based pharmacophore model for HDAC8 inhibitors by incorporating customized ZBG features.  Subsequently, pharmacophore-based virtual screening led to the discovery of three novel HDAC8 inhibitors with low micromole IC50 values (1.8-1.9 μM).  Further studies demonstrated that compd. H8-A5 was selective for HDAC8 over HDAC 1/4 and showed antiproliferation activity in MDA-MB-231 cancer cells.  Mol. docking and mol. dynamic studies suggested a possible binding mode for H8-A5, which provides a good starting point for the development of HDAC8 inhibitors in cancer treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp4Iu2hmeMJF7Vg90H21EOLACvtfcHk0lj9IDtVivSFEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktVegu7g%253D&md5=06371ebb6d627ba47e81db3af2376bb3</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fci500762z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci500762z%26sid%3Dliteratum%253Aachs%26aulast%3DHou%26aufirst%3DX.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DXu%26aufirst%3DW.%26aulast%3DFang%26aufirst%3DH.%26atitle%3DEnhancing%2520the%2520Sensitivity%2520of%2520Pharmacophore-Based%2520Virtual%2520Screening%2520by%2520Incorporating%2520Customized%2520ZBG%2520Features%253A%2520A%2520Case%2520Study%2520Using%2520Histone%2520Deacetylase%25208%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D861%26epage%3D71%26doi%3D10.1021%2Fci500762z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Kaniskan, H. U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Selective Inhibitors of Protein Methyltransferases</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1596</span><span class="NLM_x">–</span> <span class="NLM_lpage">1629</span><span class="refDoi"> DOI: 10.1021/jm501234a</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501234a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=1596-1629&author=H.+U.+Kaniskanauthor=K.+D.+Konzeauthor=J.+Jin&title=Selective+Inhibitors+of+Protein+Methyltransferases&doi=10.1021%2Fjm501234a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Selective inhibitors of protein methyltransferases</span></div><div class="casAuthors">Kaniskan, H. Umit; Konze, Kyle D.; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1596-1629</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Mounting evidence suggests that protein methyltransferases (PMTs), which catalyze the methylation of histone and non-histone proteins, play a crucial role in diverse biol. processes and human diseases.  In particular, PMTs have been recognized as major players in regulating gene expression and chromatin state.  PMTs are divided into 2 categories: protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs).  There has been a steadily growing interest in these enzymes as potential therapeutic targets and therefore discovery of PMT inhibitors has also been pursued increasingly over the past decade.  Here, the authors present a perspective on selective, small-mol. inhibitors of PMTs with an emphasis on their discovery, characterization, and applicability as chem. tools for deciphering the target PMTs' physiol. functions and involvement in human diseases.  The authors highlight the current state of PMT inhibitors and discuss future directions and opportunities for PMT inhibitor discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEWF_XeLEY1rVg90H21EOLACvtfcHk0liVRPE6TfHQdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvFKqtbbJ&md5=8fb2b2d21d306c02a2da60a3fb88c430</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1021%2Fjm501234a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501234a%26sid%3Dliteratum%253Aachs%26aulast%3DKaniskan%26aufirst%3DH.%2BU.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DSelective%2520Inhibitors%2520of%2520Protein%2520Methyltransferases%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D1596%26epage%3D1629%26doi%3D10.1021%2Fjm501234a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Li, G.-B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiang, M.</span><span> </span><span class="NLM_article-title">Virtual screening in small molecule discovery for epigenetic targets</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">158</span><span class="NLM_x">–</span> <span class="NLM_lpage">166</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2014.11.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.ymeth.2014.11.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2015&pages=158-166&author=G.-B.+Liauthor=L.-L.+Yangauthor=Y.+Yuanauthor=J.+Zouauthor=Y.+Caoauthor=S.-Y.+Yangauthor=R.+Xiangauthor=M.+Xiang&title=Virtual+screening+in+small+molecule+discovery+for+epigenetic+targets&doi=10.1016%2Fj.ymeth.2014.11.010"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2014.11.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2014.11.010%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DG.-B.%26aulast%3DYang%26aufirst%3DL.-L.%26aulast%3DYuan%26aufirst%3DY.%26aulast%3DZou%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-Y.%26aulast%3DXiang%26aufirst%3DR.%26aulast%3DXiang%26aufirst%3DM.%26atitle%3DVirtual%2520screening%2520in%2520small%2520molecule%2520discovery%2520for%2520epigenetic%2520targets%26jtitle%3DMethods%26date%3D2015%26volume%3D71%26spage%3D158%26epage%3D166%26doi%3D10.1016%2Fj.ymeth.2014.11.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Medina-Franco, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendez-Lucio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duenas-Gonzalez, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoo, J.</span><span> </span><span class="NLM_article-title">Discovery and development of DNA methyltransferase inhibitors using in silico approaches</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">577</span><span class="refDoi"> DOI: 10.1016/j.drudis.2014.12.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.drudis.2014.12.007" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2015&pages=569-577&author=J.+L.+Medina-Francoauthor=O.+Mendez-Lucioauthor=A.+Duenas-Gonzalezauthor=J.+Yoo&title=Discovery+and+development+of+DNA+methyltransferase+inhibitors+using+in+silico+approaches&doi=10.1016%2Fj.drudis.2014.12.007"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2014.12.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2014.12.007%26sid%3Dliteratum%253Aachs%26aulast%3DMedina-Franco%26aufirst%3DJ.%2BL.%26aulast%3DMendez-Lucio%26aufirst%3DO.%26aulast%3DDuenas-Gonzalez%26aufirst%3DA.%26aulast%3DYoo%26aufirst%3DJ.%26atitle%3DDiscovery%2520and%2520development%2520of%2520DNA%2520methyltransferase%2520inhibitors%2520using%2520in%2520silico%2520approaches%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2015%26volume%3D20%26spage%3D569%26epage%3D577%26doi%3D10.1016%2Fj.drudis.2014.12.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sinha, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chai, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, Y.-W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kathy Li, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, M.-F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">George Zheng, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Pharmacophore-Based Virtual Screening and Biological Evaluation of Small Molecule Inhibitors for Protein Arginine Methylation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">7978</span><span class="NLM_x">–</span> <span class="NLM_lpage">7987</span><span class="refDoi"> DOI: 10.1021/jm300521m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm300521m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=7978-7987&author=J.+Wangauthor=L.+Chenauthor=S.+H.+Sinhaauthor=Z.+Liangauthor=H.+Chaiauthor=S.+Muniyanauthor=Y.-W.+Chouauthor=C.+Yangauthor=L.+Yanauthor=Y.+Fengauthor=K.+Kathy+Liauthor=M.-F.+Linauthor=H.+Jiangauthor=Y.+George+Zhengauthor=C.+Luo&title=Pharmacophore-Based+Virtual+Screening+and+Biological+Evaluation+of+Small+Molecule+Inhibitors+for+Protein+Arginine+Methylation&doi=10.1021%2Fjm300521m"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fjm300521m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm300521m%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DSinha%26aufirst%3DS.%2BH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DChai%26aufirst%3DH.%26aulast%3DMuniyan%26aufirst%3DS.%26aulast%3DChou%26aufirst%3DY.-W.%26aulast%3DYang%26aufirst%3DC.%26aulast%3DYan%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DY.%26aulast%3DKathy%2BLi%26aufirst%3DK.%26aulast%3DLin%26aufirst%3DM.-F.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DGeorge%2BZheng%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DPharmacophore-Based%2520Virtual%2520Screening%2520and%2520Biological%2520Evaluation%2520of%2520Small%2520Molecule%2520Inhibitors%2520for%2520Protein%2520Arginine%2520Methylation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D7978%26epage%3D7987%26doi%3D10.1021%2Fjm300521m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9028</span><span class="NLM_x">–</span> <span class="NLM_lpage">9041</span><span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+Chenauthor=Y.+Wangauthor=W.+Zhouauthor=S.+Liauthor=J.+Pengauthor=Z.+Shiauthor=J.+Huauthor=Y.-C.+Liuauthor=H.+Dingauthor=Y.+Linauthor=L.+Liauthor=S.+Chengauthor=J.+Liuauthor=T.+Luauthor=H.+Jiangauthor=B.+Liuauthor=M.+Zhengauthor=C.+Luo&title=Identifying+Novel+Selective+Non-Nucleoside+DNA+Methyltransferase+1+Inhibitors+through+Docking-Based+Virtual+Screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0liVRPE6TfHQdg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPeng%26aufirst%3DJ.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DY.-C.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DCheng%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520Novel%2520Selective%2520Non-Nucleoside%2520DNA%2520Methyltransferase%25201%2520Inhibitors%2520through%2520Docking-Based%2520Virtual%2520Screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Kong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Astemizole arrests the proliferation of cancer cells by disrupting the EZH2-EED interaction of polycomb repressive complex 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9512</span><span class="NLM_x">–</span> <span class="NLM_lpage">21</span><span class="refDoi"> DOI: 10.1021/jm501230c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501230c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9512-21&author=X.+Kongauthor=L.+Chenauthor=L.+Jiaoauthor=X.+Jiangauthor=F.+Lianauthor=J.+Luauthor=K.+Zhuauthor=D.+Duauthor=J.+Liuauthor=H.+Dingauthor=N.+Zhangauthor=J.+Shenauthor=M.+Zhengauthor=K.+Chenauthor=X.+Liuauthor=H.+Jiangauthor=C.+Luo&title=Astemizole+arrests+the+proliferation+of+cancer+cells+by+disrupting+the+EZH2-EED+interaction+of+polycomb+repressive+complex+2&doi=10.1021%2Fjm501230c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">Astemizole Arrests the Proliferation of Cancer Cells by Disrupting the EZH2-EED Interaction of Polycomb Repressive Complex 2</span></div><div class="casAuthors">Kong, Xiangqian; Chen, Limin; Jiao, Lianying; Jiang, Xiangrui; Lian, Fulin; Lu, Junyan; Zhu, Kongkai; Du, Daohai; Liu, Jingqiu; Ding, Hong; Zhang, Naixia; Shen, Jingshan; Zheng, Mingyue; Chen, Kaixian; Liu, Xin; Jiang, Hualiang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9512-9521</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb Repressive Complex 2 (PRC2) modulates the chromatin structure and transcriptional repression by trimethylation lysine 27 of histone H3 (H3K27me3), a process that necessitates the protein-protein interaction (PPI) between the catalytic subunit EZH2 and EED.  Deregulated PRC2 is intimately involved in tumorigenesis and progression, making it an invaluable target for epigenetic cancer therapy.  However, until now, there have been no reported small mol. compds. targeting the EZH2-EED interactions.  In the present study, we identified astemizole, an FDA-approved drug, as a small mol. inhibitor of the EZH2-EED interaction of PRC2.  The disruption of the EZH2-EED interaction by astemizole destabilizes the PRC2 complex and inhibits its methyltransferase activity in cancer cells.  Multiple lines of evidence have demonstrated that astemizole arrests the proliferation of PRC2-driven lymphomas primarily by disabling the PRC2 complex.  Our findings demonstrate the chem. tractability of the difficult PPI target by a small mol. compd., highlighting the therapeutic promise for PRC2-driven human cancers via targeted destruction of the EZH2-EED complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoHR0GppdWLVg90H21EOLACvtfcHk0lhWXALfcE10DQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO&md5=9aa407fe354ba7e272bdfb6c6445f7f3</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjm501230c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501230c%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DDu%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DShen%26aufirst%3DJ.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DAstemizole%2520arrests%2520the%2520proliferation%2520of%2520cancer%2520cells%2520by%2520disrupting%2520the%2520EZH2-EED%2520interaction%2520of%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9512%26epage%3D21%26doi%3D10.1021%2Fjm501230c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kong, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery of two aminoglycoside antibiotics as inhibitors targeting the menin-mixed lineage leukaemia interface</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2090</span><span class="NLM_x">–</span> <span class="NLM_lpage">2093</span><span class="refDoi"> DOI: 10.1016/j.bmcl.2014.03.055</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bmcl.2014.03.055" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24709560" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsLY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2014&pages=2090-2093&author=L.+Liauthor=R.+Zhouauthor=H.+Gengauthor=L.+Yueauthor=F.+Yeauthor=Y.+Xieauthor=J.+Liuauthor=X.+Kongauthor=H.+Jiangauthor=J.+Huangauthor=C.+Luo&title=Discovery+of+two+aminoglycoside+antibiotics+as+inhibitors+targeting+the+menin-mixed+lineage+leukaemia+interface&doi=10.1016%2Fj.bmcl.2014.03.055"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of two aminoglycoside antibiotics as inhibitors targeting the menin-mixed lineage leukaemia interface</span></div><div class="casAuthors">Li, Lianchun; Zhou, Ran; Geng, Heji; Yue, Liyan; Ye, Fei; Xie, Yiqian; Liu, Jingqiu; Kong, Xiangqian; Jiang, Hualiang; Huang, Jiandong; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2090-2093</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Menin functions as an oncogenic cofactor of mixed lineage leukemia (MLL) fusion proteins in leukaemogenesis.  The menin-MLL interface is a potential therapeutic target in acute leukemia cases.  In this study, approx. 900 clin. compds. were evaluated and ranked using pharmacophore-based virtual screening, the top 29 hits were further evaluated by biochem. anal. to discover the inhibitors that target the menin-MLL interface.  Two aminoglycoside antibiotics, neomycin and tobramycin, were identified as menin-MLL inhibitors with binding affinities of 18.8 and 59.9 μM, resp.  The results of thermal shift assay validated the direct interactions between the two antibiotics and menin.  The results of isothermal titrn. calorimetry showed that the equil. dissocn. const. between menin and neomycin was approx. 15.6 μM.  We also predicted the binding modes of inhibitors at the menin-MLL interface through mol. docking anal.  The results indicated that neomycin and tobramycin competitively occupy the binding site of MLL.  This study has shed light on the development of powerful probes and new therapies for MLL-mediated leukaemogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfV27zM7Fs5LVg90H21EOLACvtfcHk0liEvsizZ1SlmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXlvVSlsLY%253D&md5=22b9b5de24bf1ff0f874ea018b6961cf</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2014.03.055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2014.03.055%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DGeng%26aufirst%3DH.%26aulast%3DYue%26aufirst%3DL.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DXie%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DKong%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520of%2520two%2520aminoglycoside%2520antibiotics%2520as%2520inhibitors%2520targeting%2520the%2520menin-mixed%2520lineage%2520leukaemia%2520interface%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2014%26volume%3D24%26spage%3D2090%26epage%3D2093%26doi%3D10.1016%2Fj.bmcl.2014.03.055" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Xie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">9665</span><span class="NLM_x">–</span> <span class="NLM_lpage">9673</span><span class="refDoi"> DOI: 10.1039/C4OB01591F</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1039%2FC4OB01591F" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25348815" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2014&pages=9665-9673&author=Y.+Xieauthor=R.+Zhouauthor=F.+Lianauthor=Y.+Liuauthor=L.+Chenauthor=Z.+Shiauthor=N.+Zhangauthor=M.+Zhengauthor=B.+Shenauthor=H.+Jiangauthor=Z.+Liangauthor=C.+Luo&title=Virtual+screening+and+biological+evaluation+of+novel+small+molecular+inhibitors+against+protein+arginine+methyltransferase+1+%28PRMT1%29&doi=10.1039%2FC4OB01591F"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening and biological evaluation of novel small molecular inhibitors against protein arginine methyltransferase 1 (PRMT1)</span></div><div class="casAuthors">Xie, Yiqian; Zhou, Ran; Lian, Fulin; Liu, Yan; Chen, Limin; Shi, Zhe; Zhang, Naixia; Zheng, Mingyue; Shen, Bairong; Jiang, Hualiang; Liang, Zhongjie; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">47</span>),
    <span class="NLM_cas:pages">9665-9673</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Protein arginine methylation is a common post-translational modification which is crucial for a variety of biol. processes.  Dysregulation of protein arginine methyltransferases (PRMTs) activity has been implicated in cancer and other serious diseases.  Thus, small mol. inhibitors against PRMT have great potential for therapeutic development.  Herein, through the combination of virtual screening and bioassays, six small mol. compds. were identified as PRMT1 inhibitors.  Amongst them, the binding affinity of compds. DCLX069 and DCLX078 with PRMT1 was further validated by T1ρ and satn. transfer difference (STD) NMR expts.  Most important of all, both compds. effectively blocked cell proliferation in breast cancer, liver cancer and acute myeloid leukemia cell lines.  The binding mode anal. from mol. docking simulations theor. indicated that both inhibitors occupied the SAM binding pocket to exert the inhibitory effect.  Taken together, our compds. enriched the structural scaffolds as PRMT1 inhibitors and afforded clues for further optimization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrinhNjy5ctKbVg90H21EOLACvtfcHk0liEvsizZ1SlmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVemsr7F&md5=36cb4c315dad033dc9f10a1f21609ab4</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1039%2FC4OB01591F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC4OB01591F%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DY.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DZheng%26aufirst%3DM.%26aulast%3DShen%26aufirst%3DB.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DLiang%26aufirst%3DZ.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520and%2520biological%2520evaluation%2520of%2520novel%2520small%2520molecular%2520inhibitors%2520against%2520protein%2520arginine%2520methyltransferase%25201%2520%2528PRMT1%2529%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2014%26volume%3D12%26spage%3D9665%26epage%3D9673%26doi%3D10.1039%2FC4OB01591F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">McInnes, C.</span><span> </span><span class="NLM_article-title">Virtual screening strategies in drug discovery</span> <span class="citation_source-journal">Curr. Opin. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">494</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.cbpa.2007.08.033</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.cbpa.2007.08.033" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17936059" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2007&pages=494-502&author=C.+McInnes&title=Virtual+screening+strategies+in+drug+discovery&doi=10.1016%2Fj.cbpa.2007.08.033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit86R"><div class="casContent"><span class="casTitleNuber">86</span><div class="casTitle"><span class="NLM_cas:atitle">Virtual screening strategies in drug discovery</span></div><div class="casAuthors">McInnes, Campbell</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Chemical Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">494-502</span>CODEN:
                <span class="NLM_cas:coden">COCBF4</span>;
        ISSN:<span class="NLM_cas:issn">1367-5931</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The identification of novel therapeutic targets and characterization of their 3D structures is increasing at a dramatic rate.  Computational screening methods continue to be developed and improved as credible and complementary alternatives to high-throughput biochem. compd. screening (HTS).  While the majority of drug candidates currently being developed have been found using HTS methods, high-throughput docking and pharmacophore-based searching algorithms are gaining acceptance and becoming a major source of lead mols. in drug discovery.  Refinements and optimization of high-throughput docking methods have lead to improvements in reproducing exptl. data and in hit rates obtained, validating their use in hit identification.  In parallel with virtual screening methods, concomitant developments in cheminformatics including identification, design and manipulation of drug-like small mol. libraries have been achieved.  Herein, currently used in silico screening techniques and their utility on a comparative and target dependent basis is discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3OTWaeATxk7Vg90H21EOLACvtfcHk0lh9lrixF-KtLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtF2rsbrN&md5=8299cff4fa5156ddcdbf489cec894f7e</span></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1016%2Fj.cbpa.2007.08.033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cbpa.2007.08.033%26sid%3Dliteratum%253Aachs%26aulast%3DMcInnes%26aufirst%3DC.%26atitle%3DVirtual%2520screening%2520strategies%2520in%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2007%26volume%3D11%26spage%3D494%26epage%3D502%26doi%3D10.1016%2Fj.cbpa.2007.08.033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Lipinski, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lombardo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dominy, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feeney, P. J.</span><span> </span><span class="NLM_article-title">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span> <span class="citation_source-journal">Adv. Drug Delivery Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">26</span><span class="refDoi"> DOI: 10.1016/S0169-409X(00)00129-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2FS0169-409X%2800%2900129-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=11259830" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2001&pages=3-26&author=C.+A.+Lipinskiauthor=F.+Lombardoauthor=B.+W.+Dominyauthor=P.+J.+Feeney&title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&doi=10.1016%2FS0169-409X%2800%2900129-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings</span></div><div class="casAuthors">Lipinski, C. A.; Lombardo, F.; Dominy, B. W.; Feeney, P. J.</div><div class="citationInfo"><span class="NLM_cas:title">Advanced Drug Delivery Reviews</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">3-26</span>CODEN:
                <span class="NLM_cas:coden">ADDREP</span>;
        ISSN:<span class="NLM_cas:issn">0169-409X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A review with 50 refs.  Exptl. and computational approaches to est. soly. and permeability in discovery and development settings are described.  In the discovery setting 'the rule of 5' predicts that poor absorption or permeation is more likely when there are more than 5 H-bond donors, 10 H-bond acceptors, the mol. wt. (MWT) is greater than 500 and the calcd. Log P (CLogP) is greater than 5 (or MlogP >4.15).  Computational methodol. for the rule-based Moriguchi Log P (MLogP) calcn. is described.  Turbidimetric soly. measurement is described and applied to known drugs.  High throughput screening (HTS) leads tend to have higher MWT and Log P and lower turbidimetric soly. than leads in the pre-HTS era.  In the development setting, soly. calcns. focus on exact value prediction and are difficult because of polymorphism.  Recent work on linear free energy relationships and Log P approaches are critically reviewed.  Useful predictions are possible in closely related analog series when coupled with exptl. thermodn. soly. measurements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYDqa02g16urVg90H21EOLACvtfcHk0lh9lrixF-KtLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXitVOhs7o%253D&md5=c60bb89da68f051c0ee7ac4c0468a0e4</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1016%2FS0169-409X%2800%2900129-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0169-409X%252800%252900129-0%26sid%3Dliteratum%253Aachs%26aulast%3DLipinski%26aufirst%3DC.%2BA.%26aulast%3DLombardo%26aufirst%3DF.%26aulast%3DDominy%26aufirst%3DB.%2BW.%26aulast%3DFeeney%26aufirst%3DP.%2BJ.%26atitle%3DExperimental%2520and%2520computational%2520approaches%2520to%2520estimate%2520solubility%2520and%2520permeability%2520in%2520drug%2520discovery%2520and%2520development%2520settings%26jtitle%3DAdv.%2520Drug%2520Delivery%2520Rev.%26date%3D2001%26volume%3D46%26spage%3D3%26epage%3D26%26doi%3D10.1016%2FS0169-409X%2800%2900129-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Baell, J. B.</span><span> </span><span class="NLM_article-title">Observations on screening-based research and some concerning trends in the literature</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">1529</span><span class="NLM_x">–</span> <span class="NLM_lpage">1546</span><span class="refDoi"> DOI: 10.4155/fmc.10.237</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4155%2Ffmc.10.237" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21426147" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2010&pages=1529-1546&author=J.+B.+Baell&title=Observations+on+screening-based+research+and+some+concerning+trends+in+the+literature&doi=10.4155%2Ffmc.10.237"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Observations on screening-based research and some concerning trends in the literature</span></div><div class="casAuthors">Baell, Jonathan B.</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1529-1546</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Academic drug discovery is being accompanied by a plethora of publications that report screening hits as good starting points for drug discovery or as useful tool compds., whereas in many cases this is not so.  These compds. may be protein-reactive but can also interfere in bioassays via a no. of other means, and it can be very hard to prove early on that they represent false starts.  This, for instance, makes it difficult for journals in their assessment of manuscripts submitted for publication.  Wider awareness and recognition of these problematic compds. will help the academic drug-discovery community focus on and publish genuinely optimizable screening hits.  This will be of general benefit.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGob88J0NH5Mx7Vg90H21EOLACvtfcHk0liwfsFJs5Mj5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1yqsbjI&md5=14f916535540fc04e37c2eb21cfe7876</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.4155%2Ffmc.10.237&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.10.237%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26atitle%3DObservations%2520on%2520screening-based%2520research%2520and%2520some%2520concerning%2520trends%2520in%2520the%2520literature%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2010%26volume%3D2%26spage%3D1529%26epage%3D1546%26doi%3D10.4155%2Ffmc.10.237" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Baell, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holloway, G. A.</span><span> </span><span class="NLM_article-title">New substructure filters for removal of pan assay interference compounds (PAINS) from screening libraries and for their exclusion in bioassays</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">2719</span><span class="NLM_x">–</span> <span class="NLM_lpage">2740</span><span class="refDoi"> DOI: 10.1021/jm901137j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm901137j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2010&pages=2719-2740&author=J.+B.+Baellauthor=G.+A.+Holloway&title=New+substructure+filters+for+removal+of+pan+assay+interference+compounds+%28PAINS%29+from+screening+libraries+and+for+their+exclusion+in+bioassays&doi=10.1021%2Fjm901137j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">New Substructure Filters for Removal of Pan Assay Interference Compounds (PAINS) from Screening Libraries and for Their Exclusion in Bioassays</span></div><div class="casAuthors">Baell, Jonathan B.; Holloway, Georgina A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">2719-2740</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This report describes a no. of substructural features which can help to identify compds. that appear as frequent hitters (promiscuous compds.) in many biochem. high throughput screens.  The compds. identified by such substructural features are not recognized by filters commonly used to identify reactive compds.  Even though these substructural features were identified using only one assay detection technol., such compds. have been reported to be active from many different assays.  In fact, these compds. are increasingly prevalent in the literature as potential starting points for further exploration, whereas they may not be.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGro_JARcceWjLVg90H21EOLACvtfcHk0liwfsFJs5Mj5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsF2qsLw%253D&md5=fbf397aa4910753c550425708c866fd2</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1021%2Fjm901137j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm901137j%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%2BB.%26aulast%3DHolloway%26aufirst%3DG.%2BA.%26atitle%3DNew%2520substructure%2520filters%2520for%2520removal%2520of%2520pan%2520assay%2520interference%2520compounds%2520%2528PAINS%2529%2520from%2520screening%2520libraries%2520and%2520for%2520their%2520exclusion%2520in%2520bioassays%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2010%26volume%3D53%26spage%3D2719%26epage%3D2740%26doi%3D10.1021%2Fjm901137j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref90"><div class="reference"><strong class="refLabel"><a href="#ref90" class="rightTabRefNumLink">90</a></strong><div class="NLM_citation" id="rightTab-cit90"><span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">Growing PAINS in academic drug discovery</span> <span class="citation_source-journal">Future Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">797</span><span class="NLM_x">–</span> <span class="NLM_lpage">801</span><span class="refDoi"> DOI: 10.4155/fmc.11.44</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.4155%2Ffmc.11.44" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=21644825" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref90/cit90&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3MXntV2jtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=797-801&author=A.+Whitty&title=Growing+PAINS+in+academic+drug+discovery&doi=10.4155%2Ffmc.11.44"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit90R"><div class="casContent"><span class="casTitleNuber">90</span><div class="casTitle"><span class="NLM_cas:atitle">Growing PAINS in academic drug discovery</span></div><div class="casAuthors">Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Future Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">797-801</span>CODEN:
                <span class="NLM_cas:coden">FMCUA7</span>;
        ISSN:<span class="NLM_cas:issn">1756-8919</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">In a recent article it was argued that compds. published as drug leads by academic labs. commonly contain functionality that identifies them as nonspecific   pan-assay interference compds.'' (PAINS).  The article raises broad questions about why best practices for hit and lead qualification that are well known in industry are not more widely employed in academia, as well as about the role of journals in publishing manuscripts that report drug leads of little potential value.  Barriers to adoption of best practices for some academic drug-discovery researchers include knowledge gaps and infrastructure deficiencies, but they also arise from fundamental differences in how academic research is structured and how success is measured.  Academic drug discovery should not seek to become identical to com. pharmaceutical research, but we can do a better job of assessing and communicating the true potential of the drug leads we publish, thereby reducing the wastage of resources on nonviable compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqHu052Dqa3DLVg90H21EOLACvtfcHk0liwfsFJs5Mj5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXntV2jtbo%253D&md5=3a773915ce7c352436eb9e2f4b8c6f4d</span></div><a href="/servlet/linkout?suffix=cit90&amp;dbid=16384&amp;doi=10.4155%2Ffmc.11.44&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Ffmc.11.44%26sid%3Dliteratum%253Aachs%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DGrowing%2520PAINS%2520in%2520academic%2520drug%2520discovery%26jtitle%3DFuture%2520Med.%2520Chem.%26date%3D2011%26volume%3D3%26spage%3D797%26epage%3D801%26doi%3D10.4155%2Ffmc.11.44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref91"><div class="reference"><strong class="refLabel"><a href="#ref91" class="rightTabRefNumLink">91</a></strong><div class="NLM_citation" id="rightTab-cit91"><span><span class="NLM_contrib-group">Baell, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walters, M. A.</span><span> </span><span class="NLM_article-title">Chemistry: Chemical con artists foil drug discovery</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">513</span><span class="NLM_x">, </span> <span class="NLM_fpage">481</span><span class="refDoi"> DOI: 10.1038/513481a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2F513481a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25254460" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref91/cit91&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=513&publication_year=2014&pages=481&author=J.+Baellauthor=M.+A.+Walters&title=Chemistry%3A+Chemical+con+artists+foil+drug+discovery&doi=10.1038%2F513481a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit91R"><div class="casContent"><span class="casTitleNuber">91</span><div class="casTitle"><span class="NLM_cas:atitle">Chemistry: Chemical con artists foil drug discovery</span></div><div class="casAuthors">Baell, Jonathan; Walters, Michael A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">513</span>
        (<span class="NLM_cas:issue">7519</span>),
    <span class="NLM_cas:pages">481-483</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Naivety about promiscuous, assay-duping mols. is polluting the literature and wasting resources, warn Jonathan Baell and Michael A. Walters.  Academic drug discoverers must be more vigilant.  Mols. that show the strongest activity in screening might not be the best starting points for drugs.  PAINS hits should almost always be ignored.  Even trained medicinal chemists have to be careful until they become experienced in screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqJFYQ0B4VVoLVg90H21EOLACvtfcHk0lipelnToT-VDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1WjsL7M&md5=f5788d81d006460c688e4b78e6200503</span></div><a href="/servlet/linkout?suffix=cit91&amp;dbid=16384&amp;doi=10.1038%2F513481a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F513481a%26sid%3Dliteratum%253Aachs%26aulast%3DBaell%26aufirst%3DJ.%26aulast%3DWalters%26aufirst%3DM.%2BA.%26atitle%3DChemistry%253A%2520Chemical%2520con%2520artists%2520foil%2520drug%2520discovery%26jtitle%3DNature%26date%3D2014%26volume%3D513%26spage%3D481%26doi%3D10.1038%2F513481a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref92"><div class="reference"><strong class="refLabel"><a href="#ref92" class="rightTabRefNumLink">92</a></strong><div class="NLM_citation" id="rightTab-cit92"><span>LigPrep, version <span class="NLM_edition">2.3</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=LigPrep%2C+version+2.3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit92&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcommun%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref93"><div class="reference"><strong class="refLabel"><a href="#ref93" class="rightTabRefNumLink">93</a></strong><div class="NLM_citation" id="rightTab-cit93"><span><span class="NLM_contrib-group">Shelley, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cholleti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Timlin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uchimaya, M.</span><span> </span><span class="NLM_article-title">Epik: a software program for pK a prediction and protonation state generation for drug-like molecules</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">681</span><span class="NLM_x">–</span> <span class="NLM_lpage">691</span><span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref93/cit93&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+Shelleyauthor=A.+Cholletiauthor=L.+Fryeauthor=J.+Greenwoodauthor=M.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit93R"><div class="casContent"><span class="casTitleNuber">93</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lipelnToT-VDg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit93&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%26aulast%3DGreenwood%26aufirst%3DJ.%26aulast%3DTimlin%26aufirst%3DM.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref94"><div class="reference"><strong class="refLabel"><a href="#ref94" class="rightTabRefNumLink">94</a></strong><div class="NLM_citation" id="rightTab-cit94"><span>Accelrys Discovery Studio 3.0; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Accelrys+Discovery+Studio+3.0%3B+Accelrys%3A+San+Diego%2C+CA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit94&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAccelrys%2520Discovery%2520Studio%25203.0%26pub%3DAccelrys%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref95"><div class="reference"><strong class="refLabel"><a href="#ref95" class="rightTabRefNumLink">95</a></strong><div class="NLM_citation" id="rightTab-cit95"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ehlers, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sutter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varma-O’Brien, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirchmair, J.</span><span> </span><span class="NLM_article-title">CAESAR: A New Conformer Generation Algorithm Based on Recursive Buildup and Local Rotational Symmetry Consideration</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1923</span><span class="NLM_x">–</span> <span class="NLM_lpage">1932</span><span class="refDoi"> DOI: 10.1021/ci700136x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci700136x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref95/cit95&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXovFOntLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1923-1932&author=J.+Liauthor=T.+Ehlersauthor=J.+Sutterauthor=S.+Varma-O%E2%80%99Brienauthor=J.+Kirchmair&title=CAESAR%3A+A+New+Conformer+Generation+Algorithm+Based+on+Recursive+Buildup+and+Local+Rotational+Symmetry+Consideration&doi=10.1021%2Fci700136x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit95R"><div class="casContent"><span class="casTitleNuber">95</span><div class="casTitle"><span class="NLM_cas:atitle">CAESAR: A New Conformer Generation Algorithm Based on Recursive Buildup and Local Rotational Symmetry Consideration</span></div><div class="casAuthors">Li, Jiabo; Ehlers, Tedman; Sutter, Jon; Varma-O'Brien, Shikha; Kirchmair, Johannes</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1923-1932</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A highly efficient conformer search algorithm based on a divide-and-conquer and recursive conformer build-up approach is presented in this paper.  This approach is combined with consideration of local rotational symmetry so that conformer duplicates due to topol. symmetry in the systematic search can be efficiently eliminated.  This new algorithm, termed CAESAR (Conformer Algorithm based on Energy Screening and Recursive Buildup), has been implemented in Discovery Studio 1.7 as part of the Catalyst Component Collection.  CAESAR has been validated by comparing the conformer models generated by the new method and Catalyst/FAST.  CAESAR is consistently 5-20 times faster than Catalyst/FAST for all data sets investigated.  The speedup is even more dramatic for mols. with high topol. symmetry or for mols. that require a large no. of conformers to be sampled.  The quality of the conformer models generated by CAESAR has been validated by assessing the ability to reproduce the receptor-bound x-ray conformation of ligands extd. for the Protein Data Bank (PDB) and assessing the ability to adequately cover the pharmacophore space.  It is shown that CAESAR is able to reproduce the receptor-bound conformation slightly better than the Catalyst/FAST method for a data set of 918 ligands retrieved from the PDB.  In addn., it is shown that CAESAR covers the pharmacophore space as well or better than Catalyst/FAST.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpd5K-SQHljYLVg90H21EOLACvtfcHk0ljV4mgm7vzjIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXovFOntLk%253D&md5=6e19e920c60c2944cc75034aee860440</span></div><a href="/servlet/linkout?suffix=cit95&amp;dbid=16384&amp;doi=10.1021%2Fci700136x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700136x%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DEhlers%26aufirst%3DT.%26aulast%3DSutter%26aufirst%3DJ.%26aulast%3DVarma-O%25E2%2580%2599Brien%26aufirst%3DS.%26aulast%3DKirchmair%26aufirst%3DJ.%26atitle%3DCAESAR%253A%2520A%2520New%2520Conformer%2520Generation%2520Algorithm%2520Based%2520on%2520Recursive%2520Buildup%2520and%2520Local%2520Rotational%2520Symmetry%2520Consideration%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1923%26epage%3D1932%26doi%3D10.1021%2Fci700136x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref96"><div class="reference"><strong class="refLabel"><a href="#ref96" class="rightTabRefNumLink">96</a></strong><div class="NLM_citation" id="rightTab-cit96"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: A New Approach for Rapid, Accurate Docking and Scoring. 1. Method and Assessment of Docking Accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref96/cit96&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+A+New+Approach+for+Rapid%2C+Accurate+Docking+and+Scoring.+1.+Method+and+Assessment+of+Docking+Accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit96R"><div class="casContent"><span class="casTitleNuber">96</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0ljV4mgm7vzjIA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit96&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520A%2520New%2520Approach%2520for%2520Rapid%252C%2520Accurate%2520Docking%2520and%2520Scoring.%25201.%2520Method%2520and%2520Assessment%2520of%2520Docking%2520Accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref97"><div class="reference"><strong class="refLabel"><a href="#ref97" class="rightTabRefNumLink">97</a></strong><div class="NLM_citation" id="rightTab-cit97"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanschagrin, P. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span> </span><span class="NLM_article-title">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein–Ligand Complexes</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">49</span><span class="NLM_x">, </span> <span class="NLM_fpage">6177</span><span class="NLM_x">–</span> <span class="NLM_lpage">6196</span><span class="refDoi"> DOI: 10.1021/jm051256o</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm051256o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref97/cit97&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=6177-6196&author=R.+A.+Friesnerauthor=R.+B.+Murphyauthor=M.+P.+Repaskyauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=T.+A.+Halgrenauthor=P.+C.+Sanschagrinauthor=D.+T.+Mainz&title=Extra+Precision+Glide%3A+Docking+and+Scoring+Incorporating+a+Model+of+Hydrophobic+Enclosure+for+Protein%E2%80%93Ligand+Complexes&doi=10.1021%2Fjm051256o"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit97R"><div class="casContent"><span class="casTitleNuber">97</span><div class="casTitle"><span class="NLM_cas:atitle">Extra Precision Glide: Docking and Scoring Incorporating a Model of Hydrophobic Enclosure for Protein-Ligand Complexes</span></div><div class="casAuthors">Friesner, Richard A.; Murphy, Robert B.; Repasky, Matthew P.; Frye, Leah L.; Greenwood, Jeremy R.; Halgren, Thomas A.; Sanschagrin, Paul C.; Mainz, Daniel T.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">6177-6196</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A novel scoring function to est. protein-ligand binding affinities has been developed and implemented as the Glide 4.0 XP scoring function and docking protocol.  In addn. to unique water desolvation energy terms, protein-ligand structural motifs leading to enhanced binding affinity are included:(1) hydrophobic enclosure where groups of lipophilic ligand atoms are enclosed on opposite faces by lipophilic protein atoms, (2) neutral-neutral single or correlated hydrogen bonds in a hydrophobically enclosed environment, and (3) five categories of charged-charged hydrogen bonds.  The XP scoring function and docking protocol have been developed to reproduce exptl. binding affinities for a set of 198 complexes (RMSDs of 2.26 and 1.73 kcal/mol over all and well-docked ligands, resp.) and to yield quality enrichments for a set of fifteen screens of pharmaceutical importance.  Enrichment results demonstrate the importance of the novel XP mol. recognition and water scoring in sepg. active and inactive ligands and avoiding false positives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohZiwN12vaY7Vg90H21EOLACvtfcHk0lj0XF6t-MiQLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVGmurg%253D&md5=ea428c82ead0d8c27f8c1a7b694a1edf</span></div><a href="/servlet/linkout?suffix=cit97&amp;dbid=16384&amp;doi=10.1021%2Fjm051256o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm051256o%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DSanschagrin%26aufirst%3DP.%2BC.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26atitle%3DExtra%2520Precision%2520Glide%253A%2520Docking%2520and%2520Scoring%2520Incorporating%2520a%2520Model%2520of%2520Hydrophobic%2520Enclosure%2520for%2520Protein%25E2%2580%2593Ligand%2520Complexes%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D6177%26epage%3D6196%26doi%3D10.1021%2Fjm051256o" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref98"><div class="reference"><strong class="refLabel"><a href="#ref98" class="rightTabRefNumLink">98</a></strong><div class="NLM_citation" id="rightTab-cit98"><span>Maestro, version <span class="NLM_edition">9.0</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro%2C+version+9.0%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit98&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref99"><div class="reference"><strong class="refLabel"><a href="#ref99" class="rightTabRefNumLink">99</a></strong><div class="NLM_citation" id="rightTab-cit99"><span><span class="NLM_contrib-group">Siebel, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandez, A. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">El-Osta, A.</span><span> </span><span class="NLM_article-title">Glycemic memory associated epigenetic changes</span> <span class="citation_source-journal">Biochem. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">80</span><span class="NLM_x">, </span> <span class="NLM_fpage">1853</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1016/j.bcp.2010.06.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref99/cit99&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.bcp.2010.06.005" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=80&publication_year=2010&pages=1853-9&author=A.+L.+Siebelauthor=A.+Z.+Fernandezauthor=A.+El-Osta&title=Glycemic+memory+associated+epigenetic+changes&doi=10.1016%2Fj.bcp.2010.06.005"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit99&amp;dbid=16384&amp;doi=10.1016%2Fj.bcp.2010.06.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bcp.2010.06.005%26sid%3Dliteratum%253Aachs%26aulast%3DSiebel%26aufirst%3DA.%2BL.%26aulast%3DFernandez%26aufirst%3DA.%2BZ.%26aulast%3DEl-Osta%26aufirst%3DA.%26atitle%3DGlycemic%2520memory%2520associated%2520epigenetic%2520changes%26jtitle%3DBiochem.%2520Pharmacol.%26date%3D2010%26volume%3D80%26spage%3D1853%26epage%3D9%26doi%3D10.1016%2Fj.bcp.2010.06.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref100"><div class="reference"><strong class="refLabel"><a href="#ref100" class="rightTabRefNumLink">100</a></strong><div class="NLM_citation" id="rightTab-cit100"><span><span class="NLM_contrib-group">Yang, X.-D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelleher, N. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.-F.</span><span> </span><span class="NLM_article-title">Negative regulation of NF-kappa B action by Set9-mediated lysine methylation of the RelA subunit</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">1055</span><span class="NLM_x">–</span> <span class="NLM_lpage">1066</span><span class="refDoi"> DOI: 10.1038/emboj.2009.55</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref100/cit100&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Femboj.2009.55" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2009&pages=1055-1066&author=X.-D.+Yangauthor=B.+Huangauthor=M.+Liauthor=A.+Lambauthor=N.+L.+Kelleherauthor=L.-F.+Chen&title=Negative+regulation+of+NF-kappa+B+action+by+Set9-mediated+lysine+methylation+of+the+RelA+subunit&doi=10.1038%2Femboj.2009.55"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit100&amp;dbid=16384&amp;doi=10.1038%2Femboj.2009.55&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2009.55%26sid%3Dliteratum%253Aachs%26aulast%3DYang%26aufirst%3DX.-D.%26aulast%3DHuang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DM.%26aulast%3DLamb%26aufirst%3DA.%26aulast%3DKelleher%26aufirst%3DN.%2BL.%26aulast%3DChen%26aufirst%3DL.-F.%26atitle%3DNegative%2520regulation%2520of%2520NF-kappa%2520B%2520action%2520by%2520Set9-mediated%2520lysine%2520methylation%2520of%2520the%2520RelA%2520subunit%26jtitle%3DEMBO%2520J.%26date%3D2009%26volume%3D28%26spage%3D1055%26epage%3D1066%26doi%3D10.1038%2Femboj.2009.55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref101"><div class="reference"><strong class="refLabel"><a href="#ref101" class="rightTabRefNumLink">101</a></strong><div class="NLM_citation" id="rightTab-cit101"><span><span class="NLM_contrib-group">Stelzer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalah, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stein, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Satanower, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nativ, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oz-Levi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olender, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Belinky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahir, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krug, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perco, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mayer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kolker, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Safran, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancet, D.</span><span> </span><span class="NLM_article-title">In-silico human genomics with GeneCards</span> <span class="citation_source-journal">Hum. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">709</span><span class="NLM_x">–</span> <span class="NLM_lpage">717</span><span class="refDoi"> DOI: 10.1186/1479-7364-5-6-709</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1186%2F1479-7364-5-6-709" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=22155609" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref101/cit101&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtVGrtro%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=709-717&author=G.+Stelzerauthor=I.+Dalahauthor=T.+I.+Steinauthor=Y.+Satanowerauthor=N.+Rosenauthor=N.+Nativauthor=D.+Oz-Leviauthor=T.+Olenderauthor=F.+Belinkyauthor=I.+Bahirauthor=H.+Krugauthor=P.+Percoauthor=B.+Mayerauthor=E.+Kolkerauthor=M.+Safranauthor=D.+Lancet&title=In-silico+human+genomics+with+GeneCards&doi=10.1186%2F1479-7364-5-6-709"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit101R"><div class="casContent"><span class="casTitleNuber">101</span><div class="casTitle"><span class="NLM_cas:atitle">In-silico human genomics with GeneCards</span></div><div class="casAuthors">Stelzer, Gil; Dalah, Irina; Stein, Tsippi Iny; Satanower, Yigeal; Rosen, Naomi; Nativ, Noam; Oz-Levi, Danit; Olender, Tsviya; Belinky, Frida; Bahir, Iris; Krug, Hagit; Perco, Paul; Mayer, Bernd; Lancet, Doron</div><div class="citationInfo"><span class="NLM_cas:title">Human Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">709-717</span>CODEN:
                <span class="NLM_cas:coden">HGUEAT</span>;
        ISSN:<span class="NLM_cas:issn">1479-7364</span>.
    
            (<span class="NLM_cas:orgname">Henry Stewart Publications</span>)
        </div><div class="casAbstract">Since 1998, the bioinformatics, systems biol., genomics and medical communities have enjoyed a synergistic relationship with the GeneCards database of human genes (http://www.genecards.org).  This human gene compendium was created to help to introduce order into the increasing chaos of information flow.  As a consequence of viewing details and deep links related to specific genes, users have often requested enhanced capabilities, such that, over time, GeneCards has blossomed into a suite of tools (including GeneDecks, GeneALaCart, GeneLoc, GeneNote and GeneAnnot) for a variety of analyses of both single human genes and sets thereof.  In this paper, we focus on inhouse and external research activities which have been enabled, enhanced, complemented and, in some cases, motivated by GeneCards.  In turn, such interactions have often inspired and propelled improvements in GeneCards.  We describe here the evolution and architecture of this project, including examples of synergistic applications in diverse areas such as synthetic lethality in cancer, the annotation of genetic variations in disease, omics integration in a systems biol. approach to kidney disease, and bioinformatics tools.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvTEBrDIxsxbVg90H21EOLACvtfcHk0lheHHL6P6_1ng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtVGrtro%253D&md5=c36536c407de69bb50de1d765b746c81</span></div><a href="/servlet/linkout?suffix=cit101&amp;dbid=16384&amp;doi=10.1186%2F1479-7364-5-6-709&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1479-7364-5-6-709%26sid%3Dliteratum%253Aachs%26aulast%3DStelzer%26aufirst%3DG.%26aulast%3DDalah%26aufirst%3DI.%26aulast%3DStein%26aufirst%3DT.%2BI.%26aulast%3DSatanower%26aufirst%3DY.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DNativ%26aufirst%3DN.%26aulast%3DOz-Levi%26aufirst%3DD.%26aulast%3DOlender%26aufirst%3DT.%26aulast%3DBelinky%26aufirst%3DF.%26aulast%3DBahir%26aufirst%3DI.%26aulast%3DKrug%26aufirst%3DH.%26aulast%3DPerco%26aufirst%3DP.%26aulast%3DMayer%26aufirst%3DB.%26aulast%3DKolker%26aufirst%3DE.%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DLancet%26aufirst%3DD.%26atitle%3DIn-silico%2520human%2520genomics%2520with%2520GeneCards%26jtitle%3DHum.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D709%26epage%3D717%26doi%3D10.1186%2F1479-7364-5-6-709" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref102"><div class="reference"><strong class="refLabel"><a href="#ref102" class="rightTabRefNumLink">102</a></strong><div class="NLM_citation" id="rightTab-cit102"><span><span class="NLM_contrib-group">Belinky, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bahir, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stelzer, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zimmerman, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosen, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nativ, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalah, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iny Stein, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappaport, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mituyama, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Safran, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lancet, D.</span><span> </span><span class="NLM_article-title">Non-redundant compendium of human ncRNA genes in GeneCards</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">255</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1093/bioinformatics/bts676</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref102/cit102&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fbioinformatics%2Fbts676" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2013&pages=255-61&author=F.+Belinkyauthor=I.+Bahirauthor=G.+Stelzerauthor=S.+Zimmermanauthor=N.+Rosenauthor=N.+Nativauthor=I.+Dalahauthor=T.+Iny+Steinauthor=N.+Rappaportauthor=T.+Mituyamaauthor=M.+Safranauthor=D.+Lancet&title=Non-redundant+compendium+of+human+ncRNA+genes+in+GeneCards&doi=10.1093%2Fbioinformatics%2Fbts676"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit102&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbts676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbts676%26sid%3Dliteratum%253Aachs%26aulast%3DBelinky%26aufirst%3DF.%26aulast%3DBahir%26aufirst%3DI.%26aulast%3DStelzer%26aufirst%3DG.%26aulast%3DZimmerman%26aufirst%3DS.%26aulast%3DRosen%26aufirst%3DN.%26aulast%3DNativ%26aufirst%3DN.%26aulast%3DDalah%26aufirst%3DI.%26aulast%3DIny%2BStein%26aufirst%3DT.%26aulast%3DRappaport%26aufirst%3DN.%26aulast%3DMituyama%26aufirst%3DT.%26aulast%3DSafran%26aufirst%3DM.%26aulast%3DLancet%26aufirst%3DD.%26atitle%3DNon-redundant%2520compendium%2520of%2520human%2520ncRNA%2520genes%2520in%2520GeneCards%26jtitle%3DBioinformatics%26date%3D2013%26volume%3D29%26spage%3D255%26epage%3D61%26doi%3D10.1093%2Fbioinformatics%2Fbts676" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref103"><div class="reference"><strong class="refLabel"><a href="#ref103" class="rightTabRefNumLink">103</a></strong><div class="NLM_citation" id="rightTab-cit103"><span><span class="NLM_contrib-group">Oesterreich, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davidson, N. E.</span><span> </span><span class="NLM_article-title">The search for ESR1 mutations in breast cancer</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1415</span><span class="NLM_x">–</span> <span class="NLM_lpage">6</span><span class="refDoi"> DOI: 10.1038/ng.2831</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1038%2Fng.2831" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=24270445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref103/cit103&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL7P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=1415-6&author=S.+Oesterreichauthor=N.+E.+Davidson&title=The+search+for+ESR1+mutations+in+breast+cancer&doi=10.1038%2Fng.2831"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit103R"><div class="casContent"><span class="casTitleNuber">103</span><div class="casTitle"><span class="NLM_cas:atitle">The search for ESR1 mutations in breast cancer</span></div><div class="casAuthors">Oesterreich, Steffi; Davidson, Nancy E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1415-1416</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Two new studies report the identification of activating ESR1 gene mutations in aromatase inhibitor-resistant metastatic breast cancers.  This insight into therapeutic resistance suggests new approaches that may be useful in the management of endocrine-resistant breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFpXltTLXBR7Vg90H21EOLACvtfcHk0lhpGtbu66mIXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL7P&md5=e9cf29471564b577f3f8bcb1ee83f7b7</span></div><a href="/servlet/linkout?suffix=cit103&amp;dbid=16384&amp;doi=10.1038%2Fng.2831&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.2831%26sid%3Dliteratum%253Aachs%26aulast%3DOesterreich%26aufirst%3DS.%26aulast%3DDavidson%26aufirst%3DN.%2BE.%26atitle%3DThe%2520search%2520for%2520ESR1%2520mutations%2520in%2520breast%2520cancer%26jtitle%3DNat.%2520Genet.%26date%3D2013%26volume%3D45%26spage%3D1415%26epage%3D6%26doi%3D10.1038%2Fng.2831" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref104"><div class="reference"><strong class="refLabel"><a href="#ref104" class="rightTabRefNumLink">104</a></strong><div class="NLM_citation" id="rightTab-cit104"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref104/cit104&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+A+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit104R"><div class="casContent"><span class="casTitleNuber">104</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhpGtbu66mIXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit104&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520A%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref105"><div class="reference"><strong class="refLabel"><a href="#ref105" class="rightTabRefNumLink">105</a></strong><div class="NLM_citation" id="rightTab-cit105"><span>Glide, version <span class="NLM_edition">5.5</span>; <span class="NLM_publisher-name">Schrödinger, Inc.</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Glide%2C+version+5.5%3B+Schr%C3%B6dinger%2C+Inc.%3A+New+York%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit105&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DGlide%26pub%3DSchr%25C3%25B6dinger%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref106"><div class="reference"><strong class="refLabel"><a href="#ref106" class="rightTabRefNumLink">106</a></strong><div class="NLM_citation" id="rightTab-cit106"><span><span class="NLM_contrib-group">Trott, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span> </span><span class="NLM_article-title">AutoDock Vina: improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">455</span><span class="NLM_x">–</span> <span class="NLM_lpage">61</span><span class="refDoi"> DOI: 10.1002/jcc.21334</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjcc.21334" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=19499576" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref106/cit106&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2010&pages=455-61&author=O.+Trottauthor=A.+J.+Olson&title=AutoDock+Vina%3A+improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function%2C+efficient+optimization%2C+and+multithreading&doi=10.1002%2Fjcc.21334"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit106R"><div class="casContent"><span class="casTitleNuber">106</span><div class="casTitle"><span class="NLM_cas:atitle">AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading</span></div><div class="casAuthors">Trott, Oleg; Olson, Arthur J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">31</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">455-461</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">AutoDock Vina, a new program for mol. docking and virtual screening, is presented.  AutoDock Vina achieves an approx. 2 orders of magnitude speed-up compared with the mol. docking software previously developed in the authors' lab (AutoDock 4), while also significantly improving the accuracy of the binding mode predictions, judging by the authors' tests on the training set used in AutoDock 4 development.  Further speed-up is achieved from parallelism, by using multithreading on multicore machines.  AutoDock Vina automatically calcs. the grid maps and clusters the results in a way transparent to the user.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLecOIkUxIlrVg90H21EOLACvtfcHk0lhpGtbu66mIXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFGnur3O&md5=c6974af8a1235f7aa09918d3e6f70dc4</span></div><a href="/servlet/linkout?suffix=cit106&amp;dbid=16384&amp;doi=10.1002%2Fjcc.21334&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.21334%26sid%3Dliteratum%253Aachs%26aulast%3DTrott%26aufirst%3DO.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26atitle%3DAutoDock%2520Vina%253A%2520improving%2520the%2520speed%2520and%2520accuracy%2520of%2520docking%2520with%2520a%2520new%2520scoring%2520function%252C%2520efficient%2520optimization%252C%2520and%2520multithreading%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2010%26volume%3D31%26spage%3D455%26epage%3D61%26doi%3D10.1002%2Fjcc.21334" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref107"><div class="reference"><strong class="refLabel"><a href="#ref107" class="rightTabRefNumLink">107</a></strong><div class="NLM_citation" id="rightTab-cit107"><span><span class="NLM_contrib-group">Allen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balius, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mukherjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brozell, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moustakas, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lang, P. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rizzo, R. C.</span><span> </span><span class="NLM_article-title">DOCK 6: Impact of new features and current docking performance</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">1132</span><span class="NLM_x">–</span> <span class="NLM_lpage">56</span><span class="refDoi"> DOI: 10.1002/jcc.23905</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1002%2Fjcc.23905" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=25914306" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref107/cit107&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2015&pages=1132-56&author=W.+J.+Allenauthor=T.+E.+Baliusauthor=S.+Mukherjeeauthor=S.+R.+Brozellauthor=D.+T.+Moustakasauthor=P.+T.+Langauthor=D.+A.+Caseauthor=I.+D.+Kuntzauthor=R.+C.+Rizzo&title=DOCK+6%3A+Impact+of+new+features+and+current+docking+performance&doi=10.1002%2Fjcc.23905"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit107R"><div class="casContent"><span class="casTitleNuber">107</span><div class="casTitle"><span class="NLM_cas:atitle">DOCK 6: Impact of new features and current docking performance</span></div><div class="casAuthors">Allen, William J.; Balius, Trent E.; Mukherjee, Sudipto; Brozell, Scott R.; Moustakas, Demetri T.; Lang, P. Therese; Case, David A.; Kuntz, Irwin D.; Rizzo, Robert C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1132-1156</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">This manuscript presents the latest algorithmic and methodol. developments to the structure-based design program DOCK 6.7 focused on an updated internal energy function, new anchor selection control, enhanced minimization options, a footprint similarity scoring function, a symmetry-cor. root-mean-square deviation algorithm, a database filter, and docking forensic tools.  An important strategy during development involved use of three orthogonal metrics for assessment and validation: pose reprodn. over a large database of 1043 protein-ligand complexes (SB2012 test set), cross-docking to 24 drug-target protein families, and database enrichment using large active and decoy datasets (Directory of Useful Decoys [DUD]-E test set) for five important proteins including HIV protease and IGF-1R.  Relative to earlier versions, a key outcome of the work is a significant increase in pose reprodn. success in going from DOCK 4.0.2 (51.4%) → 5.4 (65.2%) → 6.7 (73.3%) as a result of significant decreases in failure arising from both sampling 24.1% → 13.6% → 9.1% and scoring 24.4% → 21.1% → 17.5%.  Companion cross-docking and enrichment studies with the new version highlight other strengths and remaining areas for improvement, esp. for systems contg. metal ions.  The source code for DOCK 6.7 is available for download and free for academic users at. © 2015 Wiley Periodicals, Inc.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBNgMYHDmTu7Vg90H21EOLACvtfcHk0liVgx8_gv_AvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnt1Smtbs%253D&md5=8b5fc6cc7533f975e1e740acb3688807</span></div><a href="/servlet/linkout?suffix=cit107&amp;dbid=16384&amp;doi=10.1002%2Fjcc.23905&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.23905%26sid%3Dliteratum%253Aachs%26aulast%3DAllen%26aufirst%3DW.%2BJ.%26aulast%3DBalius%26aufirst%3DT.%2BE.%26aulast%3DMukherjee%26aufirst%3DS.%26aulast%3DBrozell%26aufirst%3DS.%2BR.%26aulast%3DMoustakas%26aufirst%3DD.%2BT.%26aulast%3DLang%26aufirst%3DP.%2BT.%26aulast%3DCase%26aufirst%3DD.%2BA.%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26aulast%3DRizzo%26aufirst%3DR.%2BC.%26atitle%3DDOCK%25206%253A%2520Impact%2520of%2520new%2520features%2520and%2520current%2520docking%2520performance%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2015%26volume%3D36%26spage%3D1132%26epage%3D56%26doi%3D10.1002%2Fjcc.23905" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref108"><div class="reference"><strong class="refLabel"><a href="#ref108" class="rightTabRefNumLink">108</a></strong><div class="NLM_citation" id="rightTab-cit108"><span><span class="NLM_contrib-group">Hamid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rotshteyn, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabadi, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, P.</span><span> </span><span class="NLM_article-title">Comparison of alamar blue and MTT assays for high through-put screening</span> <span class="citation_source-journal">Toxicol. In Vitro</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">703</span><span class="NLM_x">–</span> <span class="NLM_lpage">10</span><span class="refDoi"> DOI: 10.1016/j.tiv.2004.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref108/cit108&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1016%2Fj.tiv.2004.03.012" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2004&pages=703-10&author=R.+Hamidauthor=Y.+Rotshteynauthor=L.+Rabadiauthor=R.+Parikhauthor=P.+Bullock&title=Comparison+of+alamar+blue+and+MTT+assays+for+high+through-put+screening&doi=10.1016%2Fj.tiv.2004.03.012"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit108&amp;dbid=16384&amp;doi=10.1016%2Fj.tiv.2004.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tiv.2004.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DHamid%26aufirst%3DR.%26aulast%3DRotshteyn%26aufirst%3DY.%26aulast%3DRabadi%26aufirst%3DL.%26aulast%3DParikh%26aufirst%3DR.%26aulast%3DBullock%26aufirst%3DP.%26atitle%3DComparison%2520of%2520alamar%2520blue%2520and%2520MTT%2520assays%2520for%2520high%2520through-put%2520screening%26jtitle%3DToxicol.%2520In%2520Vitro%26date%3D2004%26volume%3D18%26spage%3D703%26epage%3D10%26doi%3D10.1016%2Fj.tiv.2004.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref109"><div class="reference"><strong class="refLabel"><a href="#ref109" class="rightTabRefNumLink">109</a></strong><div class="NLM_citation" id="rightTab-cit109"><span><span class="NLM_contrib-group">Al-Nasiry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geusens, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hanssens, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luyten, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pijnenborg, R.</span><span> </span><span class="NLM_article-title">The use of Alamar Blue assay for quantitative analysis of viability, migration and invasion of choriocarcinoma cells</span> <span class="citation_source-journal">Hum. Reprod.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">1304</span><span class="NLM_x">–</span> <span class="NLM_lpage">9</span><span class="refDoi"> DOI: 10.1093/humrep/dem011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=10.1093%2Fhumrep%2Fdem011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=17307808" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref109/cit109&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntlWms7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2007&pages=1304-9&author=S.+Al-Nasiryauthor=N.+Geusensauthor=M.+Hanssensauthor=C.+Luytenauthor=R.+Pijnenborg&title=The+use+of+Alamar+Blue+assay+for+quantitative+analysis+of+viability%2C+migration+and+invasion+of+choriocarcinoma+cells&doi=10.1093%2Fhumrep%2Fdem011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit109R"><div class="casContent"><span class="casTitleNuber">109</span><div class="casTitle"><span class="NLM_cas:atitle">The use of Alamar Blue assay for quantitative analysis of viability, migration and invasion of choriocarcinoma cells</span></div><div class="casAuthors">Al-Nasiry, S.; Geusens, N.; Hanssens, M.; Luyten, C.; Pijnenborg, R.</div><div class="citationInfo"><span class="NLM_cas:title">Human Reproduction</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1304-1309</span>CODEN:
                <span class="NLM_cas:coden">HUREEE</span>;
        ISSN:<span class="NLM_cas:issn">0268-1161</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The current techniques for quantifying trophoblast viability, migration and invasion are mainly limited by the need to sacrifice the cells during the test procedure.  In this study, the vital dye AB was used to quantify cell no. and viability of BeWo and JEG-3 choriocarcinoma cells, as well as their migration and invasion through fibronectin-coated filters.  AB was directly added to culture medium of incubated test and control cells.  At various time intervals, the redox reaction, in which AB is reduced by the cells, was measured by absorbance readings at 540 and 630 nm.  For cell migration and invasion, cells were cultured onto uncoated or fibronectin-coated inserts, resp.  AB redn. of migrated cells was normalized to that of control cells to calc. percentages of migration.  This model was also tested in the presence of a reported inhibitor, transforming growth factor (TGF) β.  The curve of %AB redn. vs. cell no. was linear, with intra- and inter-assay Coeff. of Variations of 1.88% and 2.94%, resp.  AB redn. increased with both seeding concns. and incubation time with AB.  TGFβ treatment caused a modest decrease in AB redn. in both JEG-3 and BeWo cells.  TGFβ treatment also decreased migration in BeWo, but not in JEG-3, cells.  Thus, AB assay is a simple and reliable method for quantifying trophoblast viability, migration and invasion.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpFkaPtJrigCLVg90H21EOLACvtfcHk0lhCsoeLL0_xkg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXntlWms7g%253D&md5=4ccf0048bb87d9c5d7667aa709d1b3f7</span></div><a href="/servlet/linkout?suffix=cit109&amp;dbid=16384&amp;doi=10.1093%2Fhumrep%2Fdem011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fhumrep%252Fdem011%26sid%3Dliteratum%253Aachs%26aulast%3DAl-Nasiry%26aufirst%3DS.%26aulast%3DGeusens%26aufirst%3DN.%26aulast%3DHanssens%26aufirst%3DM.%26aulast%3DLuyten%26aufirst%3DC.%26aulast%3DPijnenborg%26aufirst%3DR.%26atitle%3DThe%2520use%2520of%2520Alamar%2520Blue%2520assay%2520for%2520quantitative%2520analysis%2520of%2520viability%252C%2520migration%2520and%2520invasion%2520of%2520choriocarcinoma%2520cells%26jtitle%3DHum.%2520Reprod.%26date%3D2007%26volume%3D22%26spage%3D1304%26epage%3D9%26doi%3D10.1093%2Fhumrep%2Fdem011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref110"><div class="reference"><strong class="refLabel"><a href="#ref110" class="rightTabRefNumLink">110</a></strong><div class="NLM_citation" id="rightTab-cit110"><span>The PyMOL Molecular Graphics System, version <span class="NLM_edition">1.3r1</span>; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=The+PyMOL+Molecular+Graphics+System%2C+version+1.3r1%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit110&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%2520System%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1N6A','PDB','1N6A'); return false;">PDB: 1N6A</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.5b01154">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68386"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.5b01154">10.1021/acs.jmedchem.5b01154</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Additional figures and tables illustrating the inhibition data, assessment of docking programs, validation of docking procedure, and 2D interaction analysis of DC-S100, DC-S237, and DC-S239 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_001.pdf">jm5b01154_si_001.pdf (957.0 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.5b01154/suppl_file/jm5b01154_si_002.csv">jm5b01154_si_002.csv (16.11 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2015.58.issue-20%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.5b01154%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.5b01154" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"66e0e7e6d81a23f8","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.6.0","si":10}'></script>
</body>
</html>
